<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/22/2019.12.20.884403.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.20.884403v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Diverse demographic histories in a guild of hymenopteran parasitoids" />
<meta name="DC.Identifier" content="10.1101/2019.12.20.884403" />
<meta name="DC.Date" content="2019-12-22" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Signatures of changes in population size have been detected in genome-wide variation in many species. However, the causes of such changes and the extent to which they are shared across co-distributed species remain poorly understood. During Pleistocene glacial maxima, many temperate European species were confined to southern refugia. While vicariance and range expansion processes associated with glacial cycles have been widely studied, little is known about the demographic history of refugial populations, and the extent and causes of demographic variation among codistributed species. We used whole genome sequence data to reconstruct and compare demographic histories during the Quaternary for Iberian refuge populations in a single ecological guild (seven species of chalcid parasitoid wasps associated with oak cynipid galls). We find support for large changes in effective population size ( N  e ) through the Pleistocene that coincide with major climate change events. However, there is little evidence that the timing, direction and magnitude of demographic change are shared across species, suggesting that demographic histories are largely idiosyncratic. Our results are compatible with the idea that specialist parasitoids attacking a narrow range of hosts experience greater fluctuations in N  e  than generalists." />
<meta name="DC.Contributor" content="William Walton" />
<meta name="DC.Contributor" content="Graham N Stone" />
<meta name="DC.Contributor" content="Konrad Lohse" />
<meta name="article:published_time" content="2019-12-22" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Diverse demographic histories in a guild of hymenopteran parasitoids" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Signatures of changes in population size have been detected in genome-wide variation in many species. However, the causes of such changes and the extent to which they are shared across co-distributed species remain poorly understood. During Pleistocene glacial maxima, many temperate European species were confined to southern refugia. While vicariance and range expansion processes associated with glacial cycles have been widely studied, little is known about the demographic history of refugial populations, and the extent and causes of demographic variation among codistributed species. We used whole genome sequence data to reconstruct and compare demographic histories during the Quaternary for Iberian refuge populations in a single ecological guild (seven species of chalcid parasitoid wasps associated with oak cynipid galls). We find support for large changes in effective population size (&lt;i&gt;N&lt;/i&gt;&lt;sub&gt;&lt;i&gt;e&lt;/i&gt;&lt;/sub&gt;) through the Pleistocene that coincide with major climate change events. However, there is little evidence that the timing, direction and magnitude of demographic change are shared across species, suggesting that demographic histories are largely idiosyncratic. Our results are compatible with the idea that specialist parasitoids attacking a narrow range of hosts experience greater fluctuations in &lt;i&gt;N&lt;/i&gt;&lt;sub&gt;&lt;i&gt;e&lt;/i&gt;&lt;/sub&gt; than generalists.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.20.884403v1" />
<meta name="citation_id" content="2019.12.20.884403v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.20.884403v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.20.884403v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.20.884403v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.20.884403" />
<meta name="citation_num_pages" content="30" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.20.884403" />
<meta name="citation_author" content="William Walton" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh" />
<meta name="citation_author" content="Graham N Stone" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2737-696X" />
<meta name="citation_author" content="Konrad Lohse" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, University of Edinburgh" />
<meta name="citation_author_email" content="konrad.lohse@ed.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9918-058X" />
<meta name="citation_reference" content="Askew, R. R., Melika, G., Pujade-Villar, J., Schönrogge, K., Stone, G. N., and Nieves-Aldrey, J. L. (2013). Catalogue of parasitoids and inquilines in cynipid oak galls in the West Palaearctic. Zootaxa, 3643(1):1–133." />
<meta name="citation_reference" content="Beichman, A. C., Huerta-Sanchez, E., and Lohmueller, K. E. (2018). Using genomic data to infer historic population dynamics of nonmodel organisms. Annual Review of Ecology, Evolution, and Systematics, 49(1):433–456." />
<meta name="citation_reference" content="Brewer, S., Cheddadi, R., de Beaulieu, J., and Reille, M. (2002). The spread of deciduous Quercus throughout Europe since the last glacial period. Forest Ecology and Management, 156(1-3):27–48." />
<meta name="citation_reference" content="Bunnefeld, L., Frantz, L. A. F., and Lohse, K. (2015). Inferring bottlenecks from genome-wide samples of short sequence blocks. Genetics, 201(3):1157–1169." />
<meta name="citation_reference" content="Bunnefeld, L., Hearn, J., Stone, G. N., and Lohse, K. (2018). Whole-genome data reveal the complex history of a diverse ecological community. Proceedings of the National Academy of Sciences of the United States of America, 115(28):E6507–E6515." />
<meta name="citation_reference" content="Charlesworth, B., Morgan, M. T., and Charlesworth, D. (1993). The effect of deleterious mutations on neutral molecular variation. Genetics, 134(4):1289–1303." />
<meta name="citation_reference" content="Cizauskas, C. A., Carlson, C. J., Burgio, K. R., Clements, C. F., Dougherty, E. R., Harris, N. C., and Phillips, A. J. (2017). Parasite vulnerability to climate change: an evidence-based functional trait approach. Royal Society Open Science, 4(1):160535." />
<meta name="citation_reference" content="Colles, A., Liow, L. H., and Prinzing, A. (2009). Are specialists at risk under environmental change? Neoecological, paleoecological and phylogenetic approaches. Ecology letters, 12(8):849–63." />
<meta name="citation_reference" content="Comps, B., Gömöry, D., Letouzey, J., Thiébaut, B., and Petit, R. J. (2001). Diverging trends between heterozygosity and allelic richness during postglacial colonization in the european beech. Genetics, 157(1):389–397." />
<meta name="citation_reference" content="Compton, S. G., Ellwood, M. D. F., Davis, A. J., and Welch, K. (2000). The Flight Heights of Chalcid Wasps (Hymenoptera, Chalcidoidea) in a Lowland Bornean Rain Forest: Fig Wasps are the High Fliers. Biotropica, 32(3):515–522." />
<meta name="citation_reference" content="Cooper, A., Turney, C., Hughen, K. A., Brook, B. W., McDonald, H. G., and Bradshaw, C. J. A. (2015). Abrupt warming events drove Late Pleistocene Holarctic megafaunal turnover. Science, 349(6248):602–6." />
<meta name="citation_reference" content="Ewing, G. B. and Jensen, J. D. (2016). The consequences of not accounting for background selection in demographic inference. Molecular Ecology, 25(1):135–141." />
<meta name="citation_reference" content="Feliner, G. N. (2011). Southern European glacial refugia: A tale of tales. Taxon, 60(2):365–372." />
<meta name="citation_reference" content="Gattepaille, L. M., Jakobsson, M., and Blum, M. G. B. (2013). Inferring population size changes with sequence and SNP data: lessons from human bottlenecks. Heredity, 110(5):409–19." />
<meta name="citation_reference" content="Gómez, A. and Lunt, D. H. (2007). Refugia within Refugia: Patterns of Phylogeographic Concordance in the Iberian Peninsula. In Phylogeography of Southern European Refugia, pages 155–188. Springer Netherlands, Dordrecht." />
<meta name="citation_reference" content="Gutenkunst, R. N., Hernandez, R. D., Williamson, S. H., Bustamante, C. D., and Stephan, W. (2009). Inferring the Joint Demographic History of Multiple Populations from Multidimensional SNP Frequency Data. PLoS Genetics, 5(10):e1000695." />
<meta name="citation_reference" content="Hayward, A. and Stone, G. N. (2006). Comparative phylogeography across two trophic levels: the oak gall wasp Andricus kollari and its chalcid parasitoid Megastigmus stigmatizans. Molecular Ecology, 15(2):479–489." />
<meta name="citation_reference" content="Hearn, J., Stone, G. N., Bunnefeld, L., Nicholls, J. A., Barton, N. H., and Lohse, K. (2014). Likelihood-based inference of population history from low-coverage de novo genome assemblies. Molecular Ecology, 23(1):198–211." />
<meta name="citation_reference" content="Hewitt, G. (2000). The genetic legacy of the Quaternary ice ages. Nature, 405(6789):907–913." />
<meta name="citation_reference" content="Hofreiter, M. and Stewart, J. (2009). Ecological Change, Range Fluctuations and Population Dynamics during the Pleistocene. Current Biology, 19(14):R584–R594." />
<meta name="citation_reference" content="Janecka, J. E., Tewes, M. E., Davis, I. A., Haines, A. M., Caso, A., Blankenship, T. L., and Honeycutt, R. L. (2016). Genetic differences in the response to landscape fragmentation by a habitat generalist, the bobcat, and a habitat specialist, the ocelot. Conservation Genetics, 17(5):1093–1108." />
<meta name="citation_reference" content="Keightley, P. D., Trivedi, U., Thomson, M., Oliver, F., Kumar, S., and Blaxter, M. L. (2009). Analysis of the genome sequences of three Drosophila melanogaster spontaneous mutation accumulation lines. Genome research, 19(7):1195–201." />
<meta name="citation_reference" content="Kelleher, J., Etheridge, A. M., and McVean, G. (2016). Efficient Coalescent Simulation and Genealogical Analysis for Large Sample Sizes. PLoS Computational Biology, 12(5):1–22." />
<meta name="citation_reference" content="Kelleher, J., Wong, Y., Wohns, A. W., Fadil, C., Albers, P. K., and McVean, G. (2019). Inferring whole-genome histories in large population dataset. Nature Genetics, 51(9):1330–1338." />
<meta name="citation_reference" content="Li, H. and Durbin, R. (2009). Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics, 25(14):1754–1760." />
<meta name="citation_reference" content="Li, H. and Durbin, R. (2011). Inference of human population history from individual whole-genome sequences. Nature, 475(7357):493–496." />
<meta name="citation_reference" content="Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., and Durbin, R. (2009). The Sequence Alignment/Map format and SAMtools. Bioinformatics, 25(16):2078–2079." />
<meta name="citation_reference" content="Lohse, K., Barton, N., Melika, G., and Stone, G. (2012). A likelihood-based comparison of population histories in a parasitoid guild. Molecular Ecology, 21:4605–4617." />
<meta name="citation_reference" content="Lohse, K., Harrison, R. J., and Barton, N. H. (2011). A general method for calculating likelihoods under the coalescent process. Genetics, 189(3):977–87." />
<meta name="citation_reference" content="Lohse, K., Sharanowski, B., and Stone, G. N. (2010). Quantifying the pleistocene history of the oak gall parasitoid Cecidostiba fungosa using twenty intron loci. Evolution, 64(9):2664–2681." />
<meta name="citation_reference" content="Mackintosh, A., Laetsch, D. R., Hayward, A., Charlesworth, B., Waterfall, M., Vila, R., and Lohse, K. (2019). The determinants of genetic diversity in butterflies. Nature Genetics, 10(1):3466." />
<meta name="citation_reference" content="McKinney, M. L. (1997). Extinction vulnerability and selectivity: Combining ecological and paleontological views. Annual Review of Ecology and Systematics, 28(1):495–516." />
<meta name="citation_reference" content="Nei, M. (1972). Genetic distance between populations. The American Naturalist, 106(949):283–292." />
<meta name="citation_reference" content="Nicholls, J. A., Challis, R. J., Mutun, S., and Stone, G. N. (2012). Mitochondrial barcodes are diagnostic of shared refugia but not species in hybridizing oak gallwasps. Molecular Ecology, 21(16):4051–4062." />
<meta name="citation_reference" content="Nicholls, J. A., Preuss, S., Hayward, A., Melika, G., Csóka, G., Nieves-Aldrey, J.-L., Askew, R. R., Tavakoli, M., Schönrogge, K., and Stone, G. N. (2010). Concordant phylogeography and cryptic speciation in two Western Palaearctic oak gall parasitoid species complexes. Molecular Ecology, 19(3):592–609." />
<meta name="citation_reference" content="Nordén, J., Penttilä, R., Siitonen, J., Tomppo, E., and Ovaskainen, O. (2013). Specialist species of wood-inhabiting fungi struggle while generalists thrive in fragmented boreal forests. Journal of Ecology, 101(3):701–712." />
<meta name="citation_reference" content="Östergård, H. and Ehrlén, J. (2005). Among population variation in specialist and generalist seed predation - the importance of host plant distribution, alternative hosts and environmental variation. Oikos, 111(1):39–46." />
<meta name="citation_reference" content="Petit, R. J., Brewer, S., Bordács, S., Burg, K., Cheddadi, R., Coart, E., Cottrell, J., Csaikl, U. M., van Dam, B., Deans, J. D., Espinel, S., Fineschi, S., Finkeldey, R., Glaz, I., Goicoechea, P. G., Jensen, J. S., König, A. O., Lowe, A. J., Madsen, S. F., Mátyás, G., Munro, R. C., Popescu, F., Slade, D., Tabbener, H., de Vries, S. G., Ziegenhagen, B., de Beaulieu, J.-L., and Kremer, A. (2002). Identification of refugia and post-glacial colonisation routes of European white oaks based on chloroplast DNA and fossil pollen evidence. Forest Ecology and Management, 156(1-3):49–74." />
<meta name="citation_reference" content="Rand, T. A. and Tscharntke, T. (2007). Contrasting effects of natural habitat loss on generalist and specialist aphid natural enemies. Oikos, 116(8):1353–1362." />
<meta name="citation_reference" content="Rokas, A., Atkinson, R. J., Brown, G. S., West, S. A., and Stone, G. N. (2001). Understanding patterns of genetic diversity in the oak gallwasp Biorhiza pallida: demographic history or a Wolbachia selective sweep? Heredity, 87(3):294–304." />
<meta name="citation_reference" content="Rokas, A., Atkinson, R. J., Webster, L., Csoka, G., and Stone, G. N. (2003). Out of Anatolia: longitudinal gradients in genetic diversity support an eastern origin for a circum-Mediterranean oak gallwasp Andricus quercustozae. Molecular Ecology, 12(8):2153–2174." />
<meta name="citation_reference" content="Schrider, D. R., Shanku, A. G., and Kern, A. D. (2016). Effects of linked selective sweeps on demographic inference and model selection. Genetics, 204(3):1207–1223." />
<meta name="citation_reference" content="Smith, J. M. and Haigh, J. (1974). The hitch-hiking effect of a favourable gene. Genet. Res., Camb, 23:23–35." />
<meta name="citation_reference" content="Speidel, L., Forest, M., Shi, S., and Myers, S. R. (2019). A method for genome-wide genealogy estimation for thousands of samples. Nature Genetics, 51(9):1321–1329." />
<meta name="citation_reference" content="Stanke, M. and Morgenstern, B. (2005). AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints. Nucleic acids research, 33(Web Server issue):W465–7." />
<meta name="citation_reference" content="Stone, G., Lohse, K., Nicholls, J., Fuentes-Utrilla, P., Sinclair, F., Schönrogge, K., Csóka, G., Melika, G., Nieves-Aldrey, J.-L., Pujade-Villar, J., Tavakoli, M., Askew, R., and Hickerson, M. (2012). Reconstructing Community Assembly in Time and Space Reveals Enemy Escape in a Western Palearctic Insect Community. Current Biology, 22(6):532–537." />
<meta name="citation_reference" content="Stone, G. N., Challis, R. J., Atkinson, R. J., Csóka, G., Hayward, A., Melika, G., Mutun, S., Preuss, S., Rokas, A., Sadeghi, E., and Schönrogge, K. (2007). The phylogeographical clade trade: tracing the impact of human-mediated dispersal on the colonization of northern Europe by the oak gallwasp Andricus kollari. Molecular Ecology, 16(13):2768–2781." />
<meta name="citation_reference" content="Stone, G. N., Schönrogge, K., Atkinson, R. J., Bellido, D., and Pujade-Villar, J. (2002). The population biology of oak gall wasps (Hymenoptera: Cynipidae). Annual Review of Entomology, 47(1):633–668." />
<meta name="citation_reference" content="Stone, G. N., White, S. C., Csóka, G., Melika, G., Mutun, S., Pénzes, Z., Sadeghi, S. E., Schönrogge, K., Tavakoli, M., and Nicholls, J. A. (2017). Tournament ABC analysis of the western Palaearctic population history of an oak gall wasp, Synergus umbraculus. Molecular Ecology, 26(23):6685–6703." />
<meta name="citation_reference" content="Tallavaara, M., Luoto, M., Korhonen, N., Järvinen, H., and Seppä, H. (2015). Human population dynamics in Europe over the Last Glacial Maximum. Proceedings of the National Academy of Sciences, 112(27):8232–8237." />
<meta name="citation_reference" content="Tison, J.-L., Edmark, V. N., Sandoval-Castellanos, E., Van Dyck, H., Tammaru, T., Välimäki, P., Dalén, L., and Gotthard, K. (2014). Signature of post-glacial expansion and genetic structure at the northern range limit of the speckled wood butterfly. Biological Journal of the Linnean Society, 113(1):136–148." />
<meta name="citation_reference" content="Van der Auwera, G. A., Carneiro, M. O., Hartl, C., Poplin, R., del Angel, G., Levy-Moonshine, A., Jordan, T., Shakir, K., Roazen, D., Thibault, J., Banks, E., Garimella, K. V., Altshuler, D., Gabriel, S., and DePristo, A. (2013). From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline. Current Protocols in Bioinformatics, 43(1):11.10.1–11.10.33." />
<meta name="citation_reference" content="Vitales, D., García-Fernández, A., Garnatje, T., Pellicer, J., and Vallès, J. (2016). Phylogeographic insights of the lowland species cheirolophus sempervirens in the southwestern iberian peninsula. Journal of Systematics and Evolution, 54(1):65–74." />
<meta name="citation_reference" content="Wakeley, J. (2008). Coalescent Theory: An Introduction. W. H. Freeman." />
<meta name="citation_reference" content="Wall, J. D. (2003). Estimating ancestral population sizes and divergence times. Genetics, 163(1):395–404." />
<meta name="citation_reference" content="Wolfram Research, I. (2016). Mathematica." />
<meta name="twitter:title" content="Diverse demographic histories in a guild of hymenopteran parasitoids" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Signatures of changes in population size have been detected in genome-wide variation in many species. However, the causes of such changes and the extent to which they are shared across co-distributed species remain poorly understood. During Pleistocene glacial maxima, many temperate European species were confined to southern refugia. While vicariance and range expansion processes associated with glacial cycles have been widely studied, little is known about the demographic history of refugial populations, and the extent and causes of demographic variation among codistributed species. We used whole genome sequence data to reconstruct and compare demographic histories during the Quaternary for Iberian refuge populations in a single ecological guild (seven species of chalcid parasitoid wasps associated with oak cynipid galls). We find support for large changes in effective population size ( N  e ) through the Pleistocene that coincide with major climate change events. However, there is little evidence that the timing, direction and magnitude of demographic change are shared across species, suggesting that demographic histories are largely idiosyncratic. Our results are compatible with the idea that specialist parasitoids attacking a narrow range of hosts experience greater fluctuations in N  e  than generalists." />
<meta name="og-title" property="og:title" content="Diverse demographic histories in a guild of hymenopteran parasitoids" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.20.884403v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Signatures of changes in population size have been detected in genome-wide variation in many species. However, the causes of such changes and the extent to which they are shared across co-distributed species remain poorly understood. During Pleistocene glacial maxima, many temperate European species were confined to southern refugia. While vicariance and range expansion processes associated with glacial cycles have been widely studied, little is known about the demographic history of refugial populations, and the extent and causes of demographic variation among codistributed species. We used whole genome sequence data to reconstruct and compare demographic histories during the Quaternary for Iberian refuge populations in a single ecological guild (seven species of chalcid parasitoid wasps associated with oak cynipid galls). We find support for large changes in effective population size ( N  e ) through the Pleistocene that coincide with major climate change events. However, there is little evidence that the timing, direction and magnitude of demographic change are shared across species, suggesting that demographic histories are largely idiosyncratic. Our results are compatible with the idea that specialist parasitoids attacking a narrow range of hosts experience greater fluctuations in N  e  than generalists." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-22" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.20.884403v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.20.884403v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.20.884403v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.20.884403v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1066333" />
    <title>Diverse demographic histories in a guild of hymenopteran parasitoids | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"vkCu4Xxk10uxAMezT9zssaEJMzbmiYZ8M5uocvHaxdg","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1066333","apath":"\/biorxiv\/early\/2019\/12\/22\/2019.12.20.884403.atom","pisa":"biorxiv;2019.12.20.884403v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.20.884403v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.20.884403v1","title":"Diverse demographic histories in a guild of hymenopteran parasitoids","identifier":"node\/1066333"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iRXB4YgZZfM+vDaeMu8EmYn7DB4xm8X0Q4m8OBPJNCaoteYgZQqb9izhTaFj2SpSQFpl54puKQcI6KQHtqwcnhWoqcvh6wC1Lz4AGyh9vukAI1zmQBemE9RAEufM5CX\\\\\\\/Qaa5hBwNSu4RQWmxITOnkUmvRsfVROm+rstjskg5KXjhXzBQUSG8xnQLYGKj8RkXlxxcAkFwf7obdB9usEfd+y19I7iIA3hTdFRocTuq\\\\\\\/yKWK02SpNnZfn1dDjAXC5YHrug4n4Zp8vmUdfHfi++8kru2v179tncRqOnCOkzYGQWFSVxSA5OzUL\\\\\\\/UM2vK0Yd6GrA5IskNH5uZUf2pAKHH39EwIuB3dIjRprxzD2cdQmUEI06ctIMuoQuPNhcAAKukI9Vqr1Q2r3Z6u8Geez3nVKdtT4SEFcMgTBxPj5742L3nIAFKh3W5Zswhh7a9p84vjqOorcYSmTHeDgbRRJfzutkhq+DSmVQ\\\\\\\/h+fbp\\\\\\\/bBhqcZSRjXNSD817mgoDr7uRRqk85paVAz8AKrpRrg2Ljx6gJReHURJqota5G9okxNi2l+iF0oXe6qrAwGjLUWDlWVWKDKVXKClpHEK\\\\\\\/VtTKFxh21PWcD5JlfwRV1\\\\\\\/3isHpgugGbq5uL6H5+4cZiVtc4n9xhY6Fo4ea9tPQIhMX5Gl+y+4MrcBzmKrgot9aPoIxwbjFdIKIQ8sqvASjat8AOKUPD4YPb2JZtxUwR2vtY8MUb9lyAGUeOg6yMT+ELh\\\\\\\/pvwJHuMiFXmEkPfB47ELbHFszQym8M8vi5iLMM9wStFtka6tkDINLpoTQXVX6AClGCsUErQywTgGsImTjUf72d3GZhThVSP4\\\\\\\/DY03QODDH7eyM2zd0\\\\\\\/IT9cZMLPzwA873XX4iL\\\\\\\/ckiPYzz9Kk6cu1J2CvsExSASmG6GiD03Txs2RUWBOyp9GweJELQy7G9EyPO11hdKaHXfJ9YttYP2jGjyjahW8TDhI0r6K6JS\\\\\\\/UkccOnbh\\\\\\\/bbDQQ+VMh8IYYEwGBTrPwxq3RYa+TQedWSKqEbe40RK5suwBUkpT4pw1jYuhQ0wIjWHIFuubnzA\\\\\\\/QhyOizu0oV5s4VNh6tQ3PiCId8tAMbstGGQBX\\\\\\\/wPHKSM4uMhUjfzQxx7Xm+dXM5reH0KIjVYzo22k31B7IEjnqvMHiJ2mninIMNAZIx6e\\\\\\\/ugBbe1XEed+L3vIvIDeJQzFZJOOhwzKDfFV3beFKqu04J8WMQnvYFScoIAoDmVM+nykNEM8UIwoXJMydC2rUOsot5rksR4P42BVfsSFura6J9W3ZFDm3GFmqt9Bjw+HeZUdYAf1sdSangcmCVHEYMdMNSiwRUbYXU4gCJHl8vqLwbstR1\\\u0022,\\\u0022iv\\\u0022:\\\u0022wfytWsKCxbL+qnosamSjEQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f744c81946488d5564a42b1025c1a8caa516ca4d6db18077df7eb23fad68af97\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.20.884403v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1066333 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.20.884403v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-r7KR47c2StvCKnNvWQUzBsc7mtOf49dBVPa4oyskxfs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1066333" id="node1066333" data-pisa="biorxiv;2019.12.20.884403v1" data-pisa-master="biorxiv;2019.12.20.884403" data-apath="/biorxiv/early/2019/12/22/2019.12.20.884403.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Diverse demographic histories in a guild of hymenopteran parasitoids</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">William</span> <span class="nlm-surname">Walton</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2737-696X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Graham N</span> <span class="nlm-surname">Stone</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9918-058X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Konrad</span> <span class="nlm-surname">Lohse</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.20.884403 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1066333" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">William Walton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute of Evolutionary Biology, University of Edinburgh</span>, EH9 3FL, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BWalton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walton%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BWalton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Graham N Stone </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute of Evolutionary Biology, University of Edinburgh</span>, EH9 3FL, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Graham%2BN%2BStone%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stone%20GN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGraham%2BN%2BStone%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2737-696X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Graham N Stone</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Konrad Lohse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute of Evolutionary Biology, University of Edinburgh</span>, EH9 3FL, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Konrad%2BLohse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lohse%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKonrad%2BLohse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9918-058X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Konrad Lohse</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:konrad.lohse@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">konrad.lohse@ed.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.20.884403v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1066333" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1066333/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.20.884403v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1066333" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1066333/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.20.884403v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1066333" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1066333/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.20.884403v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1066333" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1066333/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.20.884403v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1066333" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1066333/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.20.884403v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1066333" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1066333/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Signatures of changes in population size have been detected in genome-wide variation in many species. However, the causes of such changes and the extent to which they are shared across co-distributed species remain poorly understood. During Pleistocene glacial maxima, many temperate European species were confined to southern refugia. While vicariance and range expansion processes associated with glacial cycles have been widely studied, little is known about the demographic history of refugial populations, and the extent and causes of demographic variation among codistributed species. We used whole genome sequence data to reconstruct and compare demographic histories during the Quaternary for Iberian refuge populations in a single ecological guild (seven species of chalcid parasitoid wasps associated with oak cynipid galls). We find support for large changes in effective population size (<em>N</em><sub><em>e</em></sub>) through the Pleistocene that coincide with major climate change events. However, there is little evidence that the timing, direction and magnitude of demographic change are shared across species, suggesting that demographic histories are largely idiosyncratic. Our results are compatible with the idea that specialist parasitoids attacking a narrow range of hosts experience greater fluctuations in <em>N</em><sub><em>e</em></sub> than generalists.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Natural populations change in size and distribution in response to biotic and abiotic factors over both ecological and evolutionary timescales. Given that population size is a fundamental parameter in the evolutionary process, there is considerable interest in using genetic variation in modern samples to reconstruct the demographic history of populations (see <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beichman et al., 2018</a>, for a recent review). A key question is to what extent demographic history has been shaped by past climatic events and, if so, whether such responses are concordant across taxa, or vary with species traits.</p><p id="p-4">During Pleistocene glacial maxima, the ranges of many temperate European plant and animal taxa were restricted to southern refugia, primarily in Iberia, Italy and the Balkans (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hewitt, 2000</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Feliner, 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hofreiter and Stewart, 2009</a>). Modern refugial populations of such taxa commonly harbour higher genetic diversity than northern populations founded by postglacial range expansion (e.g. <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hewitt, 2000</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rokas et al., 2003</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tison et al., 2014</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Vitales et al., 2016</a>) (but see <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Comps et al., 2001</a>). However, little is known about how refugial population sizes varied through glacial cycles, and the extent to which such variation is concordant across co-distributed species. One hypothesis is that contraction of suitable habitat during glacial maxima reduced population sizes simultaneously across suites of ecologically interacting species. Alternatively, the long term size of refugial populations may be shaped by gene flow both within and between major refugia during brief warmer periods within glacial maxima (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cooper et al., 2015</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hofreiter and Stewart, 2009</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tallavaara et al., 2015</a>). Persistent spatial structure within refugia has been discovered in some species, leading to a hypothesis of ‘refugia within refugia’ (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gómez and Lunt, 2007</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Feliner, 2011</a>). In this case, the long term genetic diversity of a species is predicted to be a function of its dispersal ability. We might also expect ecologically specialist species (whose biology is critically dependent on interactions with a small number of other taxa) to experience greater or more frequent demographic changes than generalists (whose demography is less tightly coupled to abundance of any specific interaction) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rand and Tscharntke, 2007</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Östergård and Ehrlén, 2005</a>). While specialists have been shown to be more vulnerable than generalists to habitat fragmentation and climate change in taxa ranging from parasites and parasitoids (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cizauskas et al., 2017</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Rand and Tscharntke, 2007</a>) to fungi (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nordén et al., 2013</a>) and mammalian predators (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Janecka et al., 2016</a>), few studies have investigated whether specialists have experienced stronger or more frequent <em>N</em><sub><em>e</em></sub> changes than generalists over evolutionary timescales (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mackintosh et al., 2019</a>). A third alternative is associated with a pattern of longitudinal range expansion across Europe seen in some taxa (including the system studied here) through the Pleistocene (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Stone et al., 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bunnefeld et al., 2018</a>). For refugial populations that formed during the Pleistocene, we might expect signatures of past increases in population size. Here we discriminate between these alternative hypotheses for Iberian refuge populations of multiple species in a single ecological guild - chalcid parasitoid wasps associated with herbivorous cynipid gall wasp hosts on oak.</p><p id="p-5">Oak gall wasp communities are multitrophic systems comprising oak (<em>Quercus</em>) host plants, cynipid gall wasp herbivores and chalcid parasitoid natural enemies (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stone et al., 2002</a>). The latter are obligate specialists of cynipid galls, allowing the guild of parasitoids to be considered in ecological isolation. Many of the component species show broad longitudinal distributions, extending from Iberia in the West to Iran in the East, and all species studied to date are genetically structured into major southern refuge populations (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Stone et al., 2012</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">2017</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nicholls et al., 2010</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lohse et al., 2010</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Petit et al., 2002</a>). <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> showed that refugia in the Balkans and Iberia were colonized primarily through westwards range expansion from an eastern origin one or more glacial cycles in the past. Iberian populations have lower genetic diversity than more eastern refugia, show little evidence of population structure (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Rokas et al., 2003</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stone et al., 2007</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Nicholls et al., 2010</a>) (for an exception see <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rokas et al. (2001)</a>, and are the source of most post-colonisation gene flow (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Stone et al., 2017</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bunnefeld et al., 2018</a>). Thus for most species in this community the Iberian refuge population represents the end-point of a longitudinal expansion history.</p><p id="p-6">Here we use whole genome sequence (WGS) data to quantify and compare the demographic histories of Iberian refuge populations in seven chalcid parasitoid species in the oak gall wasp community. Our sampling targets males (five from Iberia, one from the Balkans in each species), whose haploid genome facilitates analysis (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Bunnefeld et al., 2018</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hearn et al., 2014</a>). WGS data allow for powerful demographic inference even for non-model organisms. However, given fragmented reference genomes and limited sample sizes available for these species, inference methods must be chosen carefully and, ideally, should use linkage as well as allele frequency information. In particular, inference based only on the site frequency spectrum (SFS) (e.g. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gutenkunst et al., 2009</a>) requires larger samples for accurate inference of recent population history. We investigate demographic history using two approaches that use the signal contained in genome-wide variation of a small sample of individuals in different ways (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Bunnefeld et al., 2018</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lohse et al., 2011</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li and Durbin, 2011</a>): we use a parametric maximum-composite likelihood (MCL) method based on a blockwise summary of sequence variation (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lohse et al., 2011</a>) (hereafter termed the ‘blockwise method’) to fit a model of a single instantaneous step change in <em>N</em><sub><em>e</em></sub>. Using analytic likelihood calculations to fit such fully specified but minimally complex histories utilises both frequency and linkage information and facilitates comparison between species for the timing and magnitude of <em>N</em><sub><em>e</em></sub> change. We also applied a non-parametric method, the Pairwise Sequentially Markovian Coalescent (PSMC) (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Li and Durbin, 2011</a>), based on the distribution of pairwise differences in minimal samples of two haploid genomes. PSMC generates a more resolved picture of <em>N</em><sub><em>e</em></sub> change through time, including an estimate of the time at which populations have diverged. As the two methods use different data properties and sampling strategies, we can have high confidence in inferences supported by both. Both methods have contrasting limitations: while the blockwise method – by design – cannot detect gradual <em>N</em><sub><em>e</em></sub> changes or resolve population histories involving multiple changes, PSMC is known to smooth out very sudden changes. PSMC is based on fewer samples which contain less information (i.e. fewer coalescence events) about the recent <em>N</em><sub><em>e</em></sub> (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Li and Durbin, 2011</a>). The two methods therefore complement each other and together provide a comprehensive picture of population history.</p><p id="p-7">We address the following questions: <em>(i)</em> What signatures of <em>N</em><sub><em>e</em></sub> change, if any, are present for each species? <em>(ii)</em> To what extent are the direction and timing of changes in <em>N</em><sub><em>e</em></sub> concordant across species? Do species show evidence for simultaneous <em>N</em><sub><em>e</em></sub> change suggesting concordant responses to a shared underlying driver such as climate change, or are their demographic histories largely idiosyncratic? <em>(iii)</em> Do major demographic changes coincide with specific glacial or interglacial periods in the Pleistocene? <em>(iv)</em> Did inferred demographic changes occur after the colonisation of Iberia, or are they shared with other refugial populations, suggesting that they took place in a shared ancestral population prior to the colonisation of Iberia?</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Samples and sequencing</h3><p id="p-8">We analysed whole genome Illumina paired end resequencing data for six (haploid) male individuals in each of seven species of chalcid parasitoids (spanning five families): <em>Megastigmus dorsalis</em> and <em>M. stigmatizans</em> (Megastigmidae), <em>Torymus auratus</em> (Torymidae), <em>Ormyrus nitidulus</em> and <em>O. pomaceus</em> (Ormyridae), <em>Eurytoma brunniventris</em> (Eurytomidae) and <em>Cecidostiba fungosa</em> (Pteromalidae) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table S1</a>). The <em>M. dorsalis</em> individuals correspond to cryptic species 1 for this complex, as defined by <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Nicholls et al. (2010)</a>. For each species, we sampled five individuals from Iberia (the focal refugial population) and one from Hungary (the Balkan refugium). Data for the Hungarian and two Iberian samples of each species were generated previously by <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a>. We generated analogous data for the remaining three Iberian individuals using the protocols described by <a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a>. DNA was extracted from individual wasps using the Qiagen DNeasy kit. Individual Nextera genomic libraries were generated and sequenced on an Illumina HiSeq 2000 by the NERC Edinburgh Genomics facility, UK. Raw reads were deposited at the SRA (PRJEB20883). Mean coverage per haploid individual ranged from 3.5x to 18.5x. Raw reads were mapped back to reference genomes assembled by <a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> using <em>BWA</em> (0.7.15-r1140) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li and Durbin, 2009</a>), duplicates were marked with <em>picard</em> (V2.9.0) MarkDuplicates (broadinstitute.github.io/picard/), variants were called using <em>Freebayes</em> (v1.1.0-3-g961e5f3) (<a href="https://github.com/ekg/freebayes">https://github.com/ekg/freebayes</a>) with a minimum base quality of 10 and a minimum mapping quality of 20 (see <a id="xref-ref-5-9" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> for further details).</p></div><div id="sec-4" class="subsection"><h3>Inferring step changes in population size</h3><p id="p-9">We fitted a model of a single instantaneous step change in <em>N</em><sub><em>e</em></sub> using the framework for blockwise likelihood calculations developed by <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Lohse et al. (2011)</a>. The model includes three parameters: the scaled mutation rate <em>θ</em> = 4<em>N</em>0<em>µ× l</em> (where <em>N</em>0 is the current <em>N</em><sub><em>e</em></sub> and <em>l</em> is block length); <em>T</em>, the time of <em>N</em><sub><em>e</em></sub> change measured in 2<em>N</em><sub><em>e</em></sub> generations; and <em>λ</em> = <em>N</em><sub>0</sub><em>/N</em><sub>1</sub>, the relative magnitude of the <em>N</em><sub><em>e</em></sub> change, (where <em>N</em><sub>1</sub> denotes the <em>N</em><sub><em>e</em></sub> prior to the step change).</p><p id="p-10">Following <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bunnefeld et al. (2015)</a>, we summarized sequence variation in short blocks of a fixed length <em>l</em> by the (folded) blockwise site frequency spectrum (bSFS). Given our sampling scheme of <em>n</em> = 5 haploid males, the bSFS consists of counts of two types of variants: those for which the minor allele occurs once or twice in the sample. The probability of observing a particular set of mutations in a block can be computed analytically as a higher-order derivative of the generating function of genealogies (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Lohse et al., 2011</a>). The product of probabilities of bSFS configurations across blocks can be interpreted as the composite likelihood (<em>CL</em>) of the model. We maximised <em>lnCL</em> in <em>Mathematica</em> (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wolfram Research, 2016</a>) using the function <em>NMaximise</em> (Supplementary File 1).</p><p id="p-11">To generate blockwise data for each species, we applied the same quality filters used for calling SNPs to all sites, i.e. we identified regions of the genome with a base quality <em>&gt;</em>10 and mapping quality <em>&gt;</em>20 in each individual from bam files by the CallableLoci walker of <em>GATK</em> (v3.4) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Van der Auwera et al., 2013</a>). Only regions meeting these criteria in all five Iberian individuals were included in further analyses. Custom scripts were used to partition the data into blocks of a fixed length <em>l</em> of callable sites. <em>l</em> was chosen to be inversely proportional to the pairwise genetic diversity of each species (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), such that blocks contained on average two pairwise differences. This ensures that the information content per block as well as any effect of intra-block recombination is consistent across species. Blocks with a physical span (including non-callable sites) of <em>&gt;</em> 2<em>l</em>, contigs with length <em>&lt;</em> 2<em>l</em>, and blocks with more than five “None” (uncalled) sites were removed.</p><p id="p-12">We assessed support for a step change in <em>N</em><sub><em>e</em></sub> relative to the (nested) null model of constant <em>N</em><sub><em>e</em></sub> by generating parametric bootstraps with the coalescent simulator <em>msprime</em> (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kelleher et al., 2016</a>). For each species, 100 replicate data sets were simulated under the null model assuming estimates of recombination inferred by <a id="xref-ref-5-10" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table S3</a>). Each dataset had the same total length as the real data (after filtering) and was partitioned into 5,000 windows of sequences (for computational efficiency). Both a null model of constant <em>N</em><sub><em>e</em></sub> and a step change model were fitted to each replicate, and the 95% quantile of the difference in support between models <em>lnCL</em>s was compared to that of the real data. Confidence intervals (CIs) of parameter estimates were obtained via an analogous parametric bootstrapping procedure: we simulated 100 datasets with recombination under the inferred step change model and fitted that model to each simulation replicate. CIs were obtained as 2.5% and 97.5% quantiles of the distribution of parameter estimates, and were centred around the point estimates of parameters obtained from the real data.</p></div><div id="sec-5" class="subsection"><h3>Reconstructing ancestral population size with PSMC</h3><p id="p-13">The Pairwise Sequentially Markovian Coalescent (PSMC) (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Li and Durbin, 2011</a>) was used to infer a history of population size change in each species. PSMC is a non-parametric method that reconstructs a trajectory of past <em>N</em><sub><em>e</em></sub> from the density of pairwise differences along the genome via a hidden Markov model in which the hidden states are pairwise coalescence times, the distribution of which is used to estimate <em>N</em><sub><em>e</em></sub> in discrete time intervals. In each species, the two Iberian individuals with the greatest average read depth were used as the focal pair. Fastq files were generated using <em>samtools mpileup</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li et al., 2009</a>), and regions covered in both individuals were combined into fastq files using <em>seqtk mergefa</em> (<a href="https://github.com/lh3/seqtk">https://github.com/lh3/seqtk</a>) and converted into PSMC input files. PSMC, by default, discretizes pairwise alignments into blocks of 100bp which are encoded as variant if they contain at least one variant. While this makes analyses of large genomes with low diversity (e.g. humans) computationally efficient, this discretisation is too coarse when considering more diverse genomes where the chance of several pairwise differences occurring in the same 100 bp block is non-negligible (which biases <em>N</em><sub><em>e</em></sub> estimates downwards). We investigated the effect of varying block length (100, 50, 25 and 1bp) on <em>N</em><sub><em>e</em></sub> inference for the most diverse (<em>E. brunniventris</em>, <em>π</em> = 0.0071) and the least diverse (<em>M. stigmatizans π</em> = 0.00067) species in our set. As expected, population trajectories showed higher <em>N</em><sub><em>e</em></sub> and were pushed back in time with smaller block lengths (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure S1</a>). We chose a block length of 25bp for all analyses which minimizes these biases without excessive computational cost.</p><p id="p-14">We inferred 30 free interval parameters across 64 time intervals (with the option -p “28*2+3+5”). The maximum coalescence time (-t) was set to 5, the initial value of <em>θ/ρ</em> (-r) to 1. 100 bootstraps were performed for each run. Times of peak <em>N</em><sub><em>e</em></sub> and values of <em>N</em><sub><em>e</em></sub> in each time interval were considered to be different between species if there was no overlap in bootstrap replicates. To be able to compare the magnitude of inferred <em>N</em><sub><em>e</em></sub> change between PSMC and blockwise analyses, we normalised the maximum <em>N</em><sub><em>e</em></sub> inferred by each method by the long-term average <em>N</em><sub><em>e</em></sub> estimated simply from <em>π</em> (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/22/2019.12.20.884403/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/22/2019.12.20.884403/embed/inline-graphic-1.gif"/></noscript></span></span> and computed the following measure of <em>N</em><sub><em>e</em></sub> change: <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/22/2019.12.20.884403/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/22/2019.12.20.884403/embed/inline-graphic-2.gif"/></noscript></span></span>. Unlike the size of the step change (<em>λ</em>), this measures the maximum <em>N</em><sub><em>e</em></sub> relative to the average over the entire history of a sample and is therefore expected to be smaller than <em>λ</em>.</p></div><div id="sec-6" class="subsection"><h3>Calibrating the timing of events</h3><p id="p-15">Time estimates were converted into years assuming a mutation rate of 3.46 <em>×</em> 10<sup><em>−9</em></sup> mutations per base per generation estimated for <em>Drosophila melanogaster</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Keightley et al., 2009</a>). We assumed two generations per year for all species with the exception of <em>M. stigmatizans</em> which has a single generation per year (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Stone et al., 2012</a>). While this calibration allows comparisons with the estimates obtained by <a id="xref-ref-5-11" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> (calibrated in the same way), these absolute times are likely underestimates and should be treated with caution (given that we use a spontaneous mutation rate estimate and not all <em>de novo</em> mutations are selectively neutral). We stress, however, that comparing the relative timing of demographic events across this set of parasitoids only relies on the assumption of the same per generation mutation rate across species rather than any absolute calibration.</p></div><div id="sec-7" class="subsection"><h3>Cross-population PSMC analyses</h3><p id="p-16">To test whether population size changes in each species occurred before or after the divergence between the Iberian and Balkan refuge populations, we compared the PSMC trajectories of Iberian pairs and cross-population pairs (one Iberian and one Hungarian individual). Any divergence between Iberian and the Balkan refuge populations should be visible as a separation between the within-population (Iberian pair) and the cross-population (Iberia-Balkan pair) PSMC trajectories. Specifically, in the absence of post-divergence gene flow we expect <em>N</em><sub><em>e</em></sub> estimates for the cross-population pairs to increase from the time of the population split due to the accumulation of genetic differences between the populations. Thus population size changes that occurred in an ancestral population (potentially outside Iberia) should predate the divergence of within and cross-population trajectories. In contrast, we would expect demographic events unique to the Iberian population to happen after divergence of the within- and cross-population PSMC trajectories. We also compared our PSMC divergence time estimates to those made by <a id="xref-ref-5-12" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> under explicit models of divergence and admixture.</p></div><div id="sec-8" class="subsection"><h3>Potential population structure in Iberia</h3><p id="p-17">Population structure within any assumed panmictic population (here, Iberia) can confound inferences of past demography (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gattepaille et al., 2013</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bunnefeld et al., 2015</a>). In particular, <em>N</em><sub><em>e</em></sub> estimates may be inflated due to divergence between demes (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gattepaille et al., 2013</a>). Likewise, when samples are taken from the same deme, the (structured) coalescent generates a mixture of very recent (within-deme) and older ancestry resulting from migration between demes (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wakeley, 2008</a>), which can mimic signatures of a bottleneck. To test for population structure within Iberia, we repeated PSMC analyses on every pairwise combination of our five individuals. In a structured population, and given that the sampling locations were widely spaced across Iberia (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table S1</a>), <em>N</em><sub><em>e</em></sub> trajectories are expected to differ between pairs of haploid males sampled from the same <em>versus</em> different sub-populations (analogous to the within- and between-population analyses involving Iberian and Hungarian samples).</p></div><div id="sec-9" class="subsection"><h3>Sensitivity analyses</h3><p id="p-18">The assumption of no recombination within blocks (which is required to make the composite likelihood estimation tractable) potentially leads to biases in parameter estimates. Specifically, recombination may lead to a downward bias of <em>N</em><sub><em>e</em></sub> estimates (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wall, 2003</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bunnefeld et al., 2015</a>). We used simulations in <em>msprime</em> (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kelleher et al., 2016</a>) to quantify the effect of recombination on parameter estimates (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table S2</a>). One million unlinked blocks of 586bp (corresponding to the block length used for <em>O. nitidulus</em>) were simulated under a step change model with different recombination rates and step sizes.</p><p id="p-19">It is well known that both selective sweeps (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Smith and Haigh, 1974</a>) and background selection (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charlesworth et al., 1993</a>) affect variation at neutral sites in the genome which, in turn, can bias demographic inference (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ewing and Jensen, 2016</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Schrider et al., 2016</a>). To explore the effect of selection on the blockwise analyses, we fitted step change models separately to blockwise data generated from genic and intergenic regions of the <em>O. nitidulus</em> genome. Genes were predicted <em>ab initio</em> with <em>Augustus</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stanke and Morgenstern, 2005</a>). <em>Nasonia vitripennis</em>, a chalcid parasitoid (family Pteromalidae), was used as a training set. If selection has a strong effect on demographic inference, estimates of <em>θ</em> are expected to be much lower for genic compared to intergenic regions, as most selection tends to reduce diversity both at selective targets and linked regions of the genome (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Smith and Haigh, 1974</a>). Similarly, we would expect estimates for the time of the step change, <em>T</em>, to be biased towards the present in genic compared to intergenic regions.</p></div></div><div class="section" id="sec-10"><h2 class="">Results</h2><p id="p-20">After filtering, the blockwise datasets ranged in total length from 54 to 151 Mb (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table S3</a>). The pairwise alignments used as input for PSMC spanned a total length of 161 to 379 Mb (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table S3</a>). Despite this difference in overall length, which is mainly a due to missing data and the difference in sample size (two <em>versus</em> five individuals), estimates of average pairwise diversity, as measured by <em>π</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nei, 1972</a>), agreed broadly between the two datasets and were slightly lower for the blockwise data (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) compared to the pairwise alignments (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Across species, <em>π</em> estimates spanned an order of magnitude, from 0.00067 in <em>M. stigmatizans</em> to 0.0071 in <em>E. brunniventris</em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1076777/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1076777/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1076777" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <span class="caption-title">Maximum composite likelihood estimates (MCLE) of demographic parameters for Iberian populations of seven species of chalcid parasitoid wasps under a model of a single step change.</span><p id="p-21" class="first-child">Estimates for the time of <em>N</em><sub><em>e</em></sub> change (<em>T</em>) are given in thousands of years ago (kya). 95% CIs are shown in brackets. The maximum <em>N</em><sub><em>e</em></sub> is scaled relative to <em>π</em> as <em>Max</em>[<em>N</em><sub><em>e</em></sub>]/<em>N</em><sub><em>e,π</em></sub>.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1076784/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1076784/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1076784" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <span class="caption-title">Summaries of PSMC trajectories for Iberian populations of seven species of chalcid parasitoid wasps.</span><p id="p-22" class="first-child">Times are given in thousands of years ago (kya). 95 % CIs of peak <em>N</em><sub><em>e</em></sub> times are shown in brackets. The maximum <em>N</em><sub><em>e</em></sub> is scaled relative to <em>pi</em> as <em>Max</em>[<em>N</em><sub><em>e</em></sub>]/<em>N</em><sub><em>e,π</em></sub>.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-11" class="subsection"><h3>Large changes in <em>N</em><sub><em>e</em></sub> detected in four species</h3><p id="p-23">Taking the results of the blockwise and PSMC analyses across all seven parasitoid species together, four species (<em>M. stigmatizans, M. dorsalis, O. nitidulus</em> and <em>T. auratus</em>) show evidence for large (at least a factor of three) changes in population size during the Quaternary period (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In these species, an instantaneous step change in <em>N</em><sub><em>e</em></sub> fits the blockwise data significantly better than a null model of a constant <em>N</em><sub><em>e</em></sub>. We infer a decrease in <em>N</em><sub><em>e</em></sub> towards the present (<em>λ &lt;</em> 1) in three species (<em>M. stigmatizans, M. dorsalis</em> and <em>O. nitidulus</em>) and an increase in one species (<em>T. auratus</em>) (<em>λ &gt;</em> 1) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). In all four species, the change in <em>N</em><sub><em>e</em></sub> in the PSMC trajectory (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) agrees both in direction and timescale with the inference under the single step-change model. However, the <em>N</em><sub><em>e</em></sub> changes visible in the PSMC trajectory for these species are not equally abrupt. For example, <em>T. auratus</em> shows a gradual increase of <em>N</em><sub><em>e</em></sub> over a period of more than 200 ky, while the decreases in the PSMC trajectories of <em>M. dorsalis</em> and <em>M. stigmatizans</em> are comparatively sudden. With the exception of <em>M. stigmatizans</em>, the magnitude of <em>N</em><sub><em>e</em></sub> change inferred under the step-change model is greater than the relative magnitude of peak <em>N</em><sub><em>e</em></sub> in the PSMC trajectories (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">2</a>). Inferences based on PSMC and blockwise analyses are also broadly similar for <em>O. pomaceus</em> and <em>C. fungosa</em>: a step change in <em>N</em><sub><em>e</em></sub> is supported in neither species, and both show comparatively small population size changes in the PSMC analysis (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PSMC and maximum composite likelihood estimates (MCLE) of Ne for Iberian populations of seven species of chalcid parasitoid wasps. PSMC estimates and bootstrap replicates are shown in dark red and pale red, respectively. Population sizes estimated using the composite likelihood step change model are shown in blue with 95% CIs in grey. Species for which a step change model is not supported are outlined in green. Results for M. stigmatizans are also plotted on an alternative timescale to reveal recent Ne change (bottom right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-827608863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;PSMC and maximum composite likelihood estimates (MCLE) of &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; for Iberian populations of seven species of chalcid parasitoid wasps. PSMC estimates and bootstrap replicates are shown in dark red and pale red, respectively. Population sizes estimated using the composite likelihood step change model are shown in blue with 95% CIs in grey. Species for which a step change model is not supported are outlined in green. Results for &lt;em&gt;M. stigmatizans&lt;/em&gt; are also plotted on an alternative timescale to reveal recent &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; change (bottom right).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F1.medium.gif" width="440" height="320"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F1.medium.gif" width="440" height="320"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-24" class="first-child">PSMC and maximum composite likelihood estimates (MCLE) of <em>N</em><sub><em>e</em></sub> for Iberian populations of seven species of chalcid parasitoid wasps. PSMC estimates and bootstrap replicates are shown in dark red and pale red, respectively. Population sizes estimated using the composite likelihood step change model are shown in blue with 95% CIs in grey. Species for which a step change model is not supported are outlined in green. Results for <em>M. stigmatizans</em> are also plotted on an alternative timescale to reveal recent <em>N</em><sub><em>e</em></sub> change (bottom right).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25"><em>E. brunniventris</em> is an exception to the broad agreement between blockwise and PSMC analyses: despite showing no support for a step change, the PMSC trajectory of this species indicates a substantial (but gradual) change in <em>N</em><sub><em>e</em></sub> (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). However, additional analyses for this species (see Discussion) suggest that the PSMC analyses for <em>E. brunniventris</em> may be affected by genetic structure and/or the low contiguity of its reference genome.</p></div><div id="sec-12" class="subsection"><h3>No support for temporal clustering of demographic events</h3><p id="p-26">The four species with support for a step change in <em>N</em><sub><em>e</em></sub> (<em>M. stigmatizans, M. dorsalis, O. nitidulus</em> and <em>T. auratus</em>) show no overlap in the estimated times of <em>N</em><sub><em>e</em></sub> change. Using an insect spontaneous mutation rate (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Keightley et al., 2009</a>) to calibrate <em>T</em> estimates (see Methods for caveats), Iberian populations of <em>M. stigmatizans</em> and <em>M. dorsalis</em> most likely decreased in size at the start of the current interglacial (10 and 12.5 kya respectively). In contrast, the decrease in <em>N</em><sub><em>e</em></sub> inferred for <em>O. nitidulus</em> most likely dates to late in the last glacial period (33 kya) and the increase in <em>N</em><sub><em>e</em></sub> in <em>T. auratus</em> at 100 kya dates to just after the end of the (Eemian) interglacial (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-27">Paralleling the blockwise model results, the times of peak <em>N</em><sub><em>e</em></sub> in the PSMC trajectories also had non-overlapping CIs (across bootstrap replicates) in <em>M. stigmatizans, M. dorsalis, O. nitidulus</em> and <em>T. auratus</em> (although values for the two <em>Megastigmus</em> species are close, see <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). Interestingly, the three species with no support for a step change (<em>O. pomaceus, C. fungosa</em> and <em>E. brunniventris</em>) as well <em>M. dorsalis</em> show highly similar (overlapping CI) times of peak <em>N</em><sub><em>e</em></sub> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ne trajectories inferred by PSMC for Iberian populations of seven species of parasitoids. The vertical grey bar indicates the overlap in peak Ne of bootstrap replicates across four species (solid lines) (M. dorsalis, O. pomaceus, C. fungosa and E. brunniventris). The remaining three species (dashed lines) have unique peak Ne times." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-827608863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; trajectories inferred by PSMC for Iberian populations of seven species of parasitoids. The vertical grey bar indicates the overlap in peak &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; of bootstrap replicates across four species (solid lines) (&lt;em&gt;M. dorsalis, O. pomaceus, C. fungosa&lt;/em&gt; and &lt;em&gt;E. brunniventris&lt;/em&gt;). The remaining three species (dashed lines) have unique peak &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; times.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F2.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F2.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-28" class="first-child"><em>N</em><sub><em>e</em></sub> trajectories inferred by PSMC for Iberian populations of seven species of parasitoids. The vertical grey bar indicates the overlap in peak <em>N</em><sub><em>e</em></sub> of bootstrap replicates across four species (solid lines) (<em>M. dorsalis, O. pomaceus, C. fungosa</em> and <em>E. brunniventris</em>). The remaining three species (dashed lines) have unique peak <em>N</em><sub><em>e</em></sub> times.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Changes in <em>N</em><sub><em>e</em></sub> occur after the divergence of Iberian populations</h3><p id="p-29">In five species the PSMC trajectories of within-population (Iberia) pairs diverge clearly from the cross-population (Iberia <em>versus</em> Hungarian) trajectories. In contrast, little divergence between within and cross-population PSMC trajectories is visible for <em>E. brunniventris</em> and none for <em>T. auratus</em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In the three species for which the blockwise analysis supports a decline in population size, the inferred time of <em>N</em><sub><em>e</em></sub> change is more recent than the divergence of within- and cross-population PSMC trajectories (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>) and so must have occurred after Iberian and Balkan populations split. For <em>O. pomaceus</em>, <em>C. fungosa</em> and <em>M. stigmatizans</em> the split times between Iberian and the Balkan populations inferred here <em>post hoc</em> by comparing PSMCs trajectories are broadly compatible with the divergence estimates obtained by <a id="xref-ref-5-13" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> under explicit models of population divergence (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). In contrast, for both <em>M. dorsalis</em> and <em>O. nitidulus</em> PSMC based estimates of population divergence substantially predate the split times inferred by <a id="xref-ref-5-14" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a>. In both cases, the cross-population PSMC trajectories decrease after divergence between the Iberian and Hungarian populations, suggesting that these populations have been connected by some level of post-divergence gene flow.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PSMC Ne trajectories for Iberian (red) and cross-population (Iberian vs Balkans) pairs (blue) of seven species of parasitoids. Population splits can be inferred as the time at which the trajectories of within-population and cross-population pairs diverge. Vertical bars indicate the split times estimated by Bunnefeld et al. (2018) using a MCL method based on the blockwise data. The split time estimated by Bunnefeld et al. (2018) for M. dorsalis is too recent (8 kya) to be visible in this figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-827608863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;PSMC &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; trajectories for Iberian (red) and cross-population (Iberian vs Balkans) pairs (blue) of seven species of parasitoids. Population splits can be inferred as the time at which the trajectories of within-population and cross-population pairs diverge. Vertical bars indicate the split times estimated by Bunnefeld et al. (2018) using a MCL method based on the blockwise data. The split time estimated by Bunnefeld et al. (2018) for &lt;em&gt;M. dorsalis&lt;/em&gt; is too recent (8 kya) to be visible in this figure.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F3.medium.gif" width="440" height="317"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F3.medium.gif" width="440" height="317"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-30" class="first-child">PSMC <em>N</em><sub><em>e</em></sub> trajectories for Iberian (red) and cross-population (Iberian vs Balkans) pairs (blue) of seven species of parasitoids. Population splits can be inferred as the time at which the trajectories of within-population and cross-population pairs diverge. Vertical bars indicate the split times estimated by <a id="xref-ref-5-15" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> using a MCL method based on the blockwise data. The split time estimated by <a id="xref-ref-5-16" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> for <em>M. dorsalis</em> is too recent (8 kya) to be visible in this figure.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>No evidence for structure within Iberia</h3><p id="p-31">We find little variation in PSMC trajectories between different Iberian pairs in almost all species (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure S2</a>), suggesting that our demographic inferences are unlikely to be substantially influenced by population structure within Iberia. For all species, with the exception of <em>E. brunniventris</em>, the differences in PSMC trajectories between Iberian pairs are similar in magnitude to the variation across bootstrap replicates and so likely reflect variation in coverage between individuals. <em>E. brunniventris</em> is the only species that showed clear variation in PSMC trajectories between Iberian pairs. While our additional analyses for <em>E. brunniventris</em> suggest other potential reasons for this finding (see Discussion), we cannot rule out that this variation is in part driven by population structure.</p></div><div id="sec-15" class="subsection"><h3>Sensitivity analyses</h3><p id="p-32">Both demographic inference methods used here assume selective neutrality but make different simplifying assumptions about recombination: while PSMC approximates recombination as a Markov process along the genome, the blockwise composite likelihood framework assumes no recombination within blocks. To check the extent to which recombination and selection may bias parameter estimates, we fitted histories of a step change to data simulated with recombination. Our exploration of simulated data shows that both <em>N</em><sub><em>e</em></sub> and <em>λ</em> are underestimated with increasing recombination rate whereas the time of step change (<em>T</em>) is biased downwards when <em>N</em><sub><em>e</em></sub> declines towards the present (<em>λ &lt;</em> 1) and upwards when <em>N</em><sub><em>e</em></sub> increases (<em>λ &gt;</em> 1) (<a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table S3</a>). These biases are an expected consequence of the shuffling of genealogical histories within a block in the presence of recombination, which reduces the variance in bSFS configurations. Given a recombination rate of <em>r</em> = 3 <em>×</em> 10<sup><em>−</em>9</sup> estimated for the parasitoids, our simulation check suggests that while estimates of <em>T</em>, the time of the step change may be underestimated by up to a factor of two, the ability to detect a step change and estimate its magnitude (<em>λ</em>) is little affected.</p><p id="p-33">To investigate the potential effect of selective constraint on the block based inference we fitted a history of a single step change separately to genic and intergenic regions. Parameter estimates based on genic data for <em>O. nitidulus</em> were similar to the full data set (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table S2</a>). While <em>N</em><sub><em>e</em></sub> is slightly lower and <em>T</em> is slightly greater for genic regions, as would be expected as a result of selective constraints, the timing and relative magnitude of demographic change was little affected by partitioning the data by annotation.</p></div></div><div class="section" id="sec-16"><h2 class="">Discussion</h2><p id="p-34">We analysed genome-wide sequence variation using two contrasting inference approaches to test if and how demographic histories vary within a guild of insect parasitoids in a single Pleistocene refugium. We find evidence for drastic declines in population size in three out of seven species (<em>M. stigmatizans, M. dorsalis, O. nitidulus</em>), and a large increase in population size one species (<em>T. auratus</em>). However, contrary to simple models of concordant community-wide responses to Pleistocene climatic events, we find that the demographic histories of species in this guild differ markedly both in direction and timescale.</p><p id="p-35">Interestingly, the four species that show the smallest change in past population size overlap in terms of the time of peak <em>N</em><sub><em>e</em></sub>. The apparent congruence in these species may well be due to a shared background demography, the signatures of which are masked in species that show more drastic and idiosyncratic changes in <em>N</em><sub><em>e</em></sub>. However, given that PSMC assumes selective neutrality, which is problematic for compact insect genomes, the genome-wide effect of selection is an alternative (and not mutually exclusive) explanation for the shared timing of the “hump” in the PSMC trajectories. <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Schrider et al. (2016)</a> have shown that selective sweeps can generate troughs in PSMC trajectories and background selection has been shown to lead to erroneous inference of population growth (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Ewing and Jensen, 2016</a>). We therefore cannot rule out the possibility that congruent peaks in <em>N</em><sub><em>e</em></sub> are an artefact of selective effects which, assuming similar genome composition, mutation, and recombination rates of these species, may distort PSMC inferences in similar ways. However, the fact that we have reconstructed a set of very different PSMC trajectories, most of which differ markedly from the selection-induced PSMC trajectories of <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Schrider et al. (2016)</a> (in that they show large declines rather than increases in <em>N</em><sub><em>e</em></sub> toward the present), suggests that it is unlikely that selective effects are the main cause of the inferred <em>N</em><sub><em>e</em></sub> changes. For the blockwise analyses we find little difference in parameter estimates between genic and intergenic regions of <em>O. nitidulus</em> (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table S2</a>), which again argues against a major effect of selection on our inferences. Furthermore, one would expect genomes with a shorter map length (physical length <em>×</em> recombination rate) to be disproportionately affected by linked selection (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Smith and Haigh, 1974</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Mackintosh et al., 2019</a>). However, if anything we observe the opposite pattern: the two species with shortest map lengths (<em>O. nitidulus</em> and <em>C. fungosa</em>) show less pronounced <em>N</em><sub><em>e</em></sub> change than the two species with the longest map length (<em>M. stigmatizans</em> and <em>M. dorsalis</em>).</p><div id="sec-17" class="subsection"><h3>Concordance between step change and PSMC analyses</h3><p id="p-36">PSMC and blockwise analyses exploit different aspects of the data and differ drastically in sampling schemes (contiguous pairwise alignments vs short blocks sampled across five individuals) and underlying assumptions: while the blockwise composite likelihood framework fits a single instantaneous step change in <em>N</em><sub><em>e</em></sub>, PSMC reconstructs population size change as a continuous trajectory. Despite these differences, both methods yield broadly congruent conclusions: the four species for which the blockwise analyses diagnose an abrupt change in <em>N</em><sub><em>e</em></sub> also show PSMC trajectories with large <em>N</em><sub><em>e</em></sub> changes in the same direction and at similar times. The slightly greater magnitude of <em>N</em><sub><em>e</em></sub> change under the step change history compared to the PSMC analyses may be a consequence of the fact that larger samples contain more information about recent demography. In other words, the change in <em>N</em><sub><em>e</em></sub> in the recent past detectable with larger samples (five haploid individuals) in the blockwise framework may simply not be detectable by PSMC analysis (two individuals).</p><p id="p-37">In general, one may view the fact that PSMC is essentially assumption-free as an advantage over model based inference, because it enables straightforward inference of <em>N</em><sub><em>e</em></sub> changes. Comparison of PSMC trajectories between pairs of individuals and populations allows qualitative assessment of divergence and admixture (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). However, the flip-side of this flexibility is that PSMC has limited resolution in the recent past and provides no obvious route for quantitatively testing (necessarily) simple demographic hypotheses across species. We have sought to overcome the latter limitation by comparing PSMC trajectories in terms of simple summaries (such as the timing and relative magnitude of peak <em>N</em><sub><em>e</em></sub>).</p></div><div id="sec-18" class="subsection"><h3>Changes in population size coincide with late Pleistocene climatic transitions</h3><p id="p-38">Our estimates of both the timing of drastic <em>N</em><sub><em>e</em></sub> changes and the cluster of peak <em>N</em><sub><em>e</em></sub> coincide with climatic events in the Quaternary: the start of the Holocene around 11 kya, a Dansgaard-Oeschger interstadial event – a brief warm period during the last glacial – 37 kya and the end of the Eemian interglacial around 106 kya. Previous studies on these and other species in the gallwasp community have inferred an increase in gene flow between refugia during these three time periods (<a id="xref-ref-5-17" class="xref-bibr" href="#ref-5">Bunnefeld et al., 2018</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lohse et al., 2010</a>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">2012</a>). In particular, the cluster of peak <em>N</em><sub><em>e</em></sub> times during the Dansgaard-Oeschger interstadial event coincides with a large community-wide peak in the frequency of admixture between refugia inferred by <a id="xref-ref-5-18" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a>. Similarly, the increase in <em>N</em><sub><em>e</em></sub> for <em>T. auratus</em> at the end of the Eemian interglacial period coincides with the divergence of the Iberian population of this species inferred previously (<a id="xref-ref-5-19" class="xref-bibr" href="#ref-5">Bunnefeld et al., 2018</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Stone et al., 2012</a>). It seems plausible that both events are associated with the geographic expansion of suitable oak habitat across refugial barriers during these times (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brewer et al., 2002</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Petit et al., 2002</a>), which may have triggered parasitoid range expansions.</p></div><div id="sec-19" class="subsection"><h3>Population structure within Iberia and gene flow from Eastern refugia</h3><p id="p-39">Population structure within southern European refugia has previously been demonstrated for some species, and it has been suggested that given its complex topography Iberia should be considered a mosaic of multiple micro-refugia rather than a single entity (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Feliner, 2011</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hearn et al., 2014</a>). However, our PSMC results suggest a complete lack of population structure in six out of seven species in the parasitoid guild (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure S2</a>) and imply high gene flow across Iberia. This is perhaps unsurprising given that gallwasp-associated parasitoid wasps (and other chalcids) are able to disperse long distances even across patchy habitats and host distributions (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hayward and Stone, 2006</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Compton et al., 2000</a>).</p><p id="p-40">Our estimates of population split times based on comparisons of within and cross-population PSMC trajectories agree broadly with those of <a id="xref-ref-5-20" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> for most species. <em>M. dorsalis</em> and <em>O. nitidulus</em>, the two species that show the least agreement with past estimates, both show decreases in cross-population <em>N</em><sub><em>e</em></sub> after divergence that are compatible with ongoing gene flow between refugia. The model space considered by <a id="xref-ref-5-21" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> was limited to histories involving a single burst of instantaneous admixture between refugial populations, with no potential to detect such post divergence gene flow.</p></div><div id="sec-20" class="subsection"><h3><em>E. brunniventris</em> is an outlier</h3><p id="p-41"><em>E. brunniventris</em> is an outlier in our results in several ways: it is the species with the highest genetic diversity, shows signals of population structure and is the only species for which our two inference approaches disagree. While the blockwise analysis gives no support for a change in <em>N</em><sub><em>e</em></sub>, the PSMC trajectory shows a steady increase by a factor of four. To explore the ability of the blockwise analyses to detect gradual changes in population size, we simulated 100 replicate data sets (assuming the same size and block length as the real data) under the gradual change in <em>N</em><sub><em>e</em></sub> inferred via PSMC for <em>E. brunniventris</em>. We find that in each case, a history of a single step change fits the data better than a null model of constant <em>N</em><sub><em>e</em></sub>, indicating that the blockwise analysis is indeed sensitive to gradual increases in population size. A possible explanation for the discrepancy between the PSMC and blockwise inferences for <em>E. brunniventris</em> is that its genome assembly, the least contiguous among our set of taxa (N50 of 3.8kb, <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table S1</a>), is simply too fragmented for reliable PSMC inference. Since by default PSMC does not consider contigs <em>&lt;</em> 10kb long, PSMC inference for <em>E. brunniventris</em> is based on a likely conserved subset of the genome. However, even when the blockwise analysis is limited to contigs <em>&gt;</em> 10kb, a null model of constant <em>N</em><sub><em>e</em></sub> cannot be rejected. Interestingly, both a history of constant <em>N</em><sub><em>e</em></sub> and a step change model give a poor fit to the observed frequency of bSFS configurations in <em>E. brunniventris</em>. In particular, <em>E. brunniventris</em> shows an excess of both monomorphic blocks and blocks with a large number of variants (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S4</a>), suggesting that its history is not well approximated by any model that assumes a single panmictic population. The lack of divergence of within and cross-population PSMC trajectories for <em>E. brunniventris</em> would be compatible with substantial gene flow between the Iberian and Hungarian populations (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). Alternatively <em>E. brunniventris</em> – an extreme generalist attacking a wide range of oak gallwasp hosts (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Askew et al., 2013</a>) – may harbour genetic structure as a result of recent divergence into cryptic host races. However, in the absence of a better reference genome and larger samples it remains unclear to what extent the disagreement between PSMC and blockwise analyses for <em>E. brunniventris</em> is indicative of a more complex history.</p></div><div id="sec-21" class="subsection"><h3>Outlook</h3><p id="p-42">We have uncovered considerable diversity in the demographic histories of a guild of insect parasitoids at the scale of a single glacial refugium. Our results mirror the findings of <a id="xref-ref-5-22" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a> who showed that the relationships between refugia in this guild are to a large extent idiosyncratic. An obvious question is to what extent differences in demographic history correlate with ecological traits. Intriguingly, the four parasitoid species with support for a step change (in either direction) have a narrower host range (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Askew et al., 2013</a>) and a lower ancestral <em>N</em><sub><em>e</em></sub> than those for which a null model of constant <em>N</em><sub><em>e</em></sub> could not be rejected (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S3</a>, <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table S1</a>). Although these trends are not significant, they are compatible with the idea that specialists are more prone to large changes in <em>N</em><sub><em>e</em></sub> than generalists and, as a consequence, may also be at a higher risk of extinction (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Colles et al., 2009</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">McKinney, 1997</a>). Larger samples of species, incorporating wide diversity in host number and other relevant traits (such as dispersal ability) are required to explore the potential relationships between species characteristics and demographic history.</p><p id="p-43">Parasitoid wasps are particularly suited for such systematic comparisons across co-distributed species given their rich biology and manageable genomes, which can be sampled in a haploid state by targeting males. It will be particularly interesting to interrogate such data with a new generation of inference approaches that reconstruct ancestral recombination graphs (ARG) from phased genomes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kelleher et al., 2019</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Speidel et al., 2019</a>). Given sufficiently large samples, it may be possible to infer much more detailed trajectories of <em>N</em><sub><em>e</em></sub> change and population divergence times directly from such reconstructed ARGs.</p></div></div><div class="section" id="sec-23"><h2 class="">Data Accessibility</h2><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-45">Raw reads have been deposited in the European Nucleotide Archive (ENA) (ERP023079) and the SRA (PRJEB20883)</p></li><li id="list-item-2"><p id="p-46">Genome assemblies are deposited in the ENA (PRJEB27189 and ERP109243)</p></li><li id="list-item-3"><p id="p-47"><em>Mathematica</em> notebook and blockwise data are available as Supporting Information</p></li></ul></div><div class="section" id="sec-24"><h2 class="">Author Contributions</h2><p id="p-48">KL and GS designed the project; WW analysed the sequence data with contributions from KL; all authors wrote the manuscript.</p></div><div class="section" id="sec-25"><h2 class="">Supplementary information</h2><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ne trajectories inferred by PSMC using different block lengths for E. brunniventris and M. stigmatizans. The mutation rate is adjusted accordingly for each block length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-827608863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; trajectories inferred by PSMC using different block lengths for &lt;em&gt;E. brunniventris&lt;/em&gt; and &lt;em&gt;M. stigmatizans&lt;/em&gt;. The mutation rate is adjusted accordingly for each block length.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F4.medium.gif" width="440" height="386"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F4.medium.gif" width="440" height="386"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1:</span> <p id="p-49" class="first-child"><em>N</em><sub><em>e</em></sub> trajectories inferred by PSMC using different block lengths for <em>E. brunniventris</em> and <em>M. stigmatizans</em>. The mutation rate is adjusted accordingly for each block length.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ne trajectories inferred by PSMC for all pairwise combinations of the five Iberian haploid males of each species. Individuals 1 and 2 for each species correspond to the focal pair in figures 1 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-827608863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; trajectories inferred by PSMC for all pairwise combinations of the five Iberian haploid males of each species. Individuals 1 and 2 for each species correspond to the focal pair in figures 1 2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F5.medium.gif" width="440" height="317"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F5.medium.gif" width="440" height="317"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2:</span> <p id="p-50" class="first-child"><em>N</em><sub><em>e</em></sub> trajectories inferred by PSMC for all pairwise combinations of the five Iberian haploid males of each species. Individuals 1 and 2 for each species correspond to the focal pair in <a id="xref-fig-1-5" class="xref-fig" href="#F1">figures 1</a> <a id="xref-fig-2-2" class="xref-fig" href="#F2">2</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1076769/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1076769/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1076769" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1:</span> <p id="p-51" class="first-child">Sampling locations and gallwasp hosts of parasitoid individuals used for whole genome sequencing. *: The two focal Iberian individuals used for PSMC analysis <em>†</em>: Individuals previously analysed by <a id="xref-ref-5-23" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a></p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1076786/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1076786/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1076786" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S2:</span> <p id="p-52" class="first-child">Simulation investigation of recombination bias in the likelihood method. Recombination rate for simulations with varying <em>λ</em> and block length is 2.3 × 10<sup><em>−</em>9</sup>. Comparison of genic and intergenic regions is performed on real data</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1076782/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1076782/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1076782" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S3:</span> <p id="p-53" class="first-child">Assembly summaries. N50: 50% of the assembly is contained in contigs of length equal to or greater than this value. CEGMA (Core Eukaryotic Genes Mapping Approach): full or partial presence of a core set of eukaryotic genes. <em>r</em>: recombination rate per base pair per generation estimated by <a id="xref-ref-5-24" class="xref-bibr" href="#ref-5">Bunnefeld et al. (2018)</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Goodness of fit of the best fitting demographic histories for seven species of parasitoids. The observed frequencies of blockwise configurations defined via the folded SFS are shown in orange (left). The expected frequency under the best fitting history is shown on the right for four species fitting a step change model (top, black) and three species for which a constant Ne could not be rejected (bottom, gray). For a sample of n = 5 the folded SFS has two entries j2,3, j1,4. The log composite likelihood, lnCL (per block), a measure of absolute goodness of fit of the data to the inferred model, is shown for each species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-827608863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Goodness of fit of the best fitting demographic histories for seven species of parasitoids. The observed frequencies of blockwise configurations defined via the folded SFS are shown in orange (left). The expected frequency under the best fitting history is shown on the right for four species fitting a step change model (top, black) and three species for which a constant &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; could not be rejected (bottom, gray). For a sample of &lt;em&gt;n&lt;/em&gt; = 5 the folded SFS has two entries &lt;em&gt;j&lt;/em&gt;&lt;sub&gt;2,3&lt;/sub&gt;, &lt;em&gt;j&lt;/em&gt;&lt;sub&gt;1,4&lt;/sub&gt;. The log composite likelihood, &lt;em&gt;lnCL&lt;/em&gt; (per block), a measure of absolute goodness of fit of the data to the inferred model, is shown for each species.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F6.medium.gif" width="440" height="140"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F6.medium.gif" width="440" height="140"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3:</span> <p id="p-54" class="first-child">Goodness of fit of the best fitting demographic histories for seven species of parasitoids. The observed frequencies of blockwise configurations defined via the folded SFS are shown in orange (left). The expected frequency under the best fitting history is shown on the right for four species fitting a step change model (top, black) and three species for which a constant <em>N</em><sub><em>e</em></sub> could not be rejected (bottom, gray). For a sample of <em>n</em> = 5 the folded SFS has two entries <em>j</em><sub>2,3</sub>, <em>j</em><sub>1,4</sub>. The log composite likelihood, <em>lnCL</em> (per block), a measure of absolute goodness of fit of the data to the inferred model, is shown for each species.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) Host range and (b) ancestral Ne of species with and without Ne changes inferred by the likelihood method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-827608863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(a) Host range and (b) ancestral &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; of species with and without &lt;em&gt;N&lt;/em&gt;&lt;sub&gt;&lt;em&gt;e&lt;/em&gt;&lt;/sub&gt; changes inferred by the likelihood method.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F7.medium.gif" width="440" height="203"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F7.medium.gif" width="440" height="203"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/22/2019.12.20.884403/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4:</span> <p id="p-55" class="first-child">(a) Host range and (b) ancestral <em>N</em><sub><em>e</em></sub> of species with and without <em>N</em><sub><em>e</em></sub> changes inferred by the likelihood method.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-44">We thank Lynsey Bunnefeld for help in the molecular lab and useful discussions and José Luis Nieves-Aldrey and Juli Pujade-villar for contributing samples and Sam Ebdon for comments on the manuscript. This work was supported by a Natural Environment Research Council to GNS and KL (NE/J010499/1). WW was supported by an E3 doctoral training studentship from the Natural Environment Research Council (NERC) UK, KL is supported by a NERC fellowship (NE/L011522/1) and an ERC starting grant (ModelGenomLand).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Askew</span>,  <span class="cit-name-given-names">R. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melika</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pujade-Villar</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schönrogge</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nieves-Aldrey</span>,  <span class="cit-name-given-names">J. L.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Catalogue of parasitoids and inquilines in cynipid oak galls in the West Palaearctic</span>. <abbr class="cit-jnl-abbrev">Zootaxa</abbr>, <span class="cit-vol">3643</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZootaxa%26rft.volume%253D3643%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beichman</span>,  <span class="cit-name-given-names">A. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huerta-Sanchez</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>,  <span class="cit-name-given-names">K. E.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Using genomic data to infer historic population dynamics of nonmodel organisms</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">49</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">433</span>–<span class="cit-lpage">456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D49%26rft.spage%253D433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.3" data-doi="10.1016/S0378-1127(01)00646-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brewer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheddadi</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Beaulieu</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Reille</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">The spread of deciduous Quercus throughout Europe since the last glacial period</span>. <abbr class="cit-jnl-abbrev">Forest Ecology and Management</abbr>, <span class="cit-vol">156</span>(<span class="cit-issue">1-3</span>):<span class="cit-fpage">27</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2BEcology%2Band%2BManagement%26rft.volume%253D156%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1127%252801%252900646-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1127(01)00646-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000174149400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.4" data-doi="10.1534/genetics.115.179861"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bunnefeld</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frantz</span>,  <span class="cit-name-given-names">L. A. F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lohse</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Inferring bottlenecks from genome-wide samples of short sequence blocks</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">201</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">1157</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179861%26rft_id%253Dinfo%253Apmid%252F26341659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS8zLzExNTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8yMi8yMDE5LjEyLjIwLjg4NDQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.5" data-doi="10.1073/pnas.1800334115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bunnefeld</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hearn</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lohse</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Whole-genome data reveal the complex history of a diverse ecological community</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, <span class="cit-vol">115</span>(<span class="cit-issue">28</span>):<span class="cit-fpage">E6507</span>–<span class="cit-lpage">E6515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1800334115%26rft_id%253Dinfo%253Apmid%252F29946026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzI4L0U2NTA3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMjIvMjAxOS4xMi4yMC44ODQ0MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.6" data-doi="10.1111/j.0014-3820.2002.tb00188.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgan</span>,  <span class="cit-name-given-names">M. T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">1993</span>). <span class="cit-article-title">The effect of deleterious mutations on neutral molecular variation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">134</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1289</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D134%26rft.issue%253D4%26rft.spage%253D1289%26rft.epage%253D1303%26rft.atitle%253DThe%2Beffect%2Bof%2Bdeleterious%2Bmutations%2Bon%2Bneutral%2Bmolecular%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2002.tb00188.x%26rft_id%253Dinfo%253Apmid%252F8375663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNC80LzEyODkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8yMi8yMDE5LjEyLjIwLjg4NDQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.7" data-doi="10.1098/rsos.160535"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cizauskas</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">C. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burgio</span>,  <span class="cit-name-given-names">K. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clements</span>,  <span class="cit-name-given-names">C. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dougherty</span>,  <span class="cit-name-given-names">E. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">N. C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">A. J.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Parasite vulnerability to climate change: an evidence-based functional trait approach</span>. <abbr class="cit-jnl-abbrev">Royal Society Open Science</abbr>, <span class="cit-vol">4</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">160535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRoyal%2BSociety%2BOpen%2BScience%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsos.160535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsos.160535&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.8" data-doi="10.1111/j.1461-0248.2009.01336.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Colles</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liow</span>,  <span class="cit-name-given-names">L. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Prinzing</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Are specialists at risk under environmental change? Neoecological, paleoecological and phylogenetic approaches</span>. <abbr class="cit-jnl-abbrev">Ecology letters</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">849</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DColles%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D8%26rft.spage%253D849%26rft.epage%253D863%26rft.atitle%253DAre%2Bspecialists%2Bat%2Brisk%2Bunder%2Benvironmental%2Bchange%253F%2BNeoecological%252C%2Bpaleoecological%2Band%2Bphylogenetic%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2009.01336.x%26rft_id%253Dinfo%253Apmid%252F19580588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2009.01336.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19580588&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267660600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Comps</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gömöry</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Letouzey</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thiébaut</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Petit</span>,  <span class="cit-name-given-names">R. J.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Diverging trends between heterozygosity and allelic richness during postglacial colonization in the european beech</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">157</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">389</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DComps%26rft.auinit1%253DB.%26rft.volume%253D157%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D397%26rft.atitle%253DDiverging%2BTrends%2BBetween%2BHeterozygosity%2Band%2BAllelic%2BRichness%2BDuring%2BPostglacial%2BColonization%2Bin%2Bthe%2BEuropean%2BBeech%26rft_id%253Dinfo%253Apmid%252F11139519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU3LzEvMzg5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMjIvMjAxOS4xMi4yMC44ODQ0MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.10" data-doi="10.1111/j.1744-7429.2000.tb00497.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Compton</span>,  <span class="cit-name-given-names">S. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellwood</span>,  <span class="cit-name-given-names">M. D. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">A. J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Welch</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">The Flight Heights of Chalcid Wasps (Hymenoptera, Chalcidoidea) in a Lowland Bornean Rain Forest: Fig Wasps are the High Fliers</span>. <abbr class="cit-jnl-abbrev">Biotropica</abbr>, <span class="cit-vol">32</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">515</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotropica%26rft.volume%253D32%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7429.2000.tb00497.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7429.2000.tb00497.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000089651700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.11" data-doi="10.1126/science.aac4315"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Turney</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughen</span>,  <span class="cit-name-given-names">K. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brook</span>,  <span class="cit-name-given-names">B. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McDonald</span>,  <span class="cit-name-given-names">H. G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>,  <span class="cit-name-given-names">C. J. A.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Abrupt warming events drove Late Pleistocene Holarctic megafaunal turnover</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">349</span>(<span class="cit-issue">6248</span>):<span class="cit-fpage">602</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aac4315%26rft_id%253Dinfo%253Apmid%252F26250679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDkvNjI0OC82MDIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8yMi8yMDE5LjEyLjIwLjg4NDQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.12" data-doi="10.1111/mec.13390"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ewing</span>,  <span class="cit-name-given-names">G. B.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">J. D.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">The consequences of not accounting for background selection in demographic inference</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">25</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">135</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D25%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13390&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feliner</span>,  <span class="cit-name-given-names">G. N.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Southern European glacial refugia: A tale of tales</span>. <abbr class="cit-jnl-abbrev">Taxon</abbr>, <span class="cit-vol">60</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">365</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTaxon%26rft.volume%253D60%26rft.spage%253D365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000289587100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.14" data-doi="10.1038/hdy.2012.120"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gattepaille</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Blum</span>,  <span class="cit-name-given-names">M. G. B.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Inferring population size changes with sequence and SNP data: lessons from human bottlenecks</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr>, <span class="cit-vol">110</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">409</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D110%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.120%26rft_id%253Dinfo%253Apmid%252F23423148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23423148&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.20.884403v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gómez</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lunt</span>,  <span class="cit-name-given-names">D. H.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-chapter-title">Refugia within Refugia: Patterns of Phylogeographic Concordance in the Iberian Peninsula</span>. In <span class="cit-source">Phylogeography of Southern European Refugia</span>, pages <span class="cit-fpage">155</span>–<span class="cit-lpage">188</span>. <span class="cit-publ-name">Springer Netherlands</span>, <span class="cit-publ-loc">Dordrecht</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>,  <span class="cit-name-given-names">R. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hernandez</span>,  <span class="cit-name-given-names">R. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williamson</span>,  <span class="cit-name-given-names">S. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bustamante</span>,  <span class="cit-name-given-names">C. D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stephan</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Inferring the Joint Demographic History of Multiple Populations from Multidimensional SNP Frequency Data</span>. <abbr class="cit-jnl-abbrev">PLoS Genetics</abbr>, <span class="cit-vol">5</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">e1000695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D5%26rft.spage%253De1000695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayward</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Comparative phylogeography across two trophic levels: the oak gall wasp Andricus kollari and its chalcid parasitoid Megastigmus stigmatizans</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">479</span>–<span class="cit-lpage">489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHayward%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D479%26rft.epage%253D489%26rft.atitle%253DComparative%2Bphylogeography%2Bacross%2Btwo%2Btrophic%2Blevels%253A%2Bthe%2Boak%2Bgall%2Bwasp%2BAndricus%2Bkollari%2Band%2Bits%2Bchalcid%2Bparasitoid%2BMegastigmus%2Bstigmatizans.%26rft_id%253Dinfo%253Apmid%252F16448414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16448414&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.18" data-doi="10.1111/mec.12578"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hearn</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bunnefeld</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicholls</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lohse</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Likelihood-based inference of population history from low-coverage <em>de novo</em> genome assemblies</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">23</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">198</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D23%26rft.spage%253D198%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12578&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.19" data-doi="10.1038/35016000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hewitt</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">The genetic legacy of the Quaternary ice ages</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">405</span>(<span class="cit-issue">6789</span>):<span class="cit-fpage">907</span>–<span class="cit-lpage">913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D405%26rft.issue%253D6789%26rft.spage%253D907%26rft.epage%253D913%26rft.atitle%253DThe%2Bgenetic%2Blegacy%2Bof%2Bthe%2BQuaternary%2Bice%2Bages%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35016000%26rft_id%253Dinfo%253Apmid%252F10879524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35016000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2000073425&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=10879524&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087732700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.20" data-doi="10.1016/j.cub.2009.06.030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hofreiter</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Stewart</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Ecological Change, Range Fluctuations and Population Dynamics during the Pleistocene</span>. <abbr class="cit-jnl-abbrev">Current Biology</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">14</span>):<span class="cit-fpage">R584</span>–<span class="cit-lpage">R594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DHofreiter%26rft.auinit1%253DM.%26rft.volume%253D19%26rft.issue%253D14%26rft.spage%253DR584%26rft.epage%253DR594%26rft.atitle%253DEcological%2Bchange%252C%2Brange%2Bfluctuations%2Band%2Bpopulation%2Bdynamics%2Bduring%2Bthe%2BPleistocene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.06.030%26rft_id%253Dinfo%253Apmid%252F19640497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.06.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19640497&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268530200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Janecka</span>,  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tewes</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">I. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haines</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caso</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blankenship</span>,  <span class="cit-name-given-names">T. L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Honeycutt</span>,  <span class="cit-name-given-names">R. L.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Genetic differences in the response to landscape fragmentation by a habitat generalist, the bobcat, and a habitat specialist, the ocelot</span>. <abbr class="cit-jnl-abbrev">Conservation Genetics</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1093</span>–<span class="cit-lpage">1108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BGenetics%26rft.volume%253D17%26rft.spage%253D1093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.22" data-doi="10.1101/gr.091231.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keightley</span>,  <span class="cit-name-given-names">P. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trivedi</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomson</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oliver</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Blaxter</span>,  <span class="cit-name-given-names">M. L.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Analysis of the genome sequences of three Drosophila melanogaster spontaneous mutation accumulation lines</span>. <abbr class="cit-jnl-abbrev">Genome research</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1195</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.091231.109%26rft_id%253Dinfo%253Apmid%252F19439516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzcvMTE5NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzIyLzIwMTkuMTIuMjAuODg0NDAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelleher</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Etheridge</span>,  <span class="cit-name-given-names">A. M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Efficient Coalescent Simulation and Genealogical Analysis for Large Sample Sizes</span>. <abbr class="cit-jnl-abbrev">PLoS Computational Biology</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D12%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.24" data-doi="10.1038/s41588-019-0483-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelleher</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wohns</span>,  <span class="cit-name-given-names">A. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fadil</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albers</span>,  <span class="cit-name-given-names">P. K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Inferring whole-genome histories in large population dataset</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>, <span class="cit-vol">51</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">1330</span>–<span class="cit-lpage">1338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D51%26rft.spage%253D1330%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-019-0483-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-019-0483-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.25" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">25</span>(<span class="cit-issue">14</span>):<span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.26" data-doi="10.1038/nature10231"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Inference of human population history from individual whole-genome sequences</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">475</span>(<span class="cit-issue">7357</span>):<span class="cit-fpage">493</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLi%26rft.auinit1%253DH.%26rft.volume%253D475%26rft.issue%253D7357%26rft.spage%253D493%26rft.epage%253D496%26rft.atitle%253DInference%2Bof%2Bhuman%2Bpopulation%2Bhistory%2Bfrom%2Bindividual%2Bwhole-genome%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10231%26rft_id%253Dinfo%253Apmid%252F21753753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21753753&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293167900039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.27" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handsaker</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wysoker</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fennell</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruan</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Homer</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marth</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abecasis</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">25</span>(<span class="cit-issue">16</span>):<span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.28" data-doi="10.1111/j.1365-294X.2012.05700.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lohse</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melika</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">A likelihood-based comparison of population histories in a parasitoid guild</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">21</span>:<span class="cit-fpage">4605</span>–<span class="cit-lpage">4617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLohse%26rft.auinit1%253DK.%26rft.volume%253D21%26rft.issue%253D18%26rft.spage%253D4605%26rft.epage%253D4617%26rft.atitle%253DA%2Blikelihood-based%2Bcomparison%2Bof%2Bpopulation%2Bhistories%2Bin%2Ba%2Bparasitoid%2Bguild.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05700.x%26rft_id%253Dinfo%253Apmid%252F22891657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22891657&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.29" data-doi="10.1534/genetics.111.129569"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lohse</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">R. J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">A general method for calculating likelihoods under the coalescent process</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">189</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">977</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.129569%26rft_id%253Dinfo%253Apmid%252F21900266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzMvOTc3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMjIvMjAxOS4xMi4yMC44ODQ0MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.30" data-doi="10.1111/j.1558-5646.2010.01024.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lohse</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharanowski</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Quantifying the pleistocene history of the oak gall parasitoid Cecidostiba fungosa using twenty intron loci</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">64</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">2664</span>–<span class="cit-lpage">2681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DLohse%26rft.auinit1%253DK.%26rft.volume%253D64%26rft.issue%253D9%26rft.spage%253D2664%26rft.epage%253D2681%26rft.atitle%253DQuantifying%2Bthe%2Bpleistocene%2Bhistory%2Bof%2Bthe%2Boak%2Bgall%2Bparasitoid%2BCecidostiba%2Bfungosa%2Busing%2Btwenty%2Bintron%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2010.01024.x%26rft_id%253Dinfo%253Apmid%252F20455927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2010.01024.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20455927&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281636400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mackintosh</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laetsch</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayward</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waterfall</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vila</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lohse</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">The determinants of genetic diversity in butterflies</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">3466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D10%26rft.spage%253D3466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.32" data-doi="10.1146/annurev.ecolsys.28.1.495"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKinney</span>,  <span class="cit-name-given-names">M. L.</span></span> (<span class="cit-pub-date">1997</span>). <span class="cit-article-title">Extinction vulnerability and selectivity: Combining ecological and paleontological views</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology and Systematics</abbr>, <span class="cit-vol">28</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">495</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DMcKinney%26rft.auinit1%253DM.%2BL.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D495%26rft.epage%253D516%26rft.atitle%253DEXTINCTION%2BVULNERABILITY%2BAND%2BSELECTIVITY%253A%2BCombining%2BEcological%2Band%2BPaleontological%2BViews%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.28.1.495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.28.1.495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000070961400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.33" data-doi="10.1086/282771"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nei</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">1972</span>). <span class="cit-article-title">Genetic distance between populations</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">106</span>(<span class="cit-issue">949</span>):<span class="cit-fpage">283</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D106%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972M475000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.34" data-doi="10.1111/j.1365-294X.2012.05683.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nicholls</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Challis</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mutun</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Mitochondrial barcodes are diagnostic of shared refugia but not species in hybridizing oak gallwasps</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">16</span>):<span class="cit-fpage">4051</span>–<span class="cit-lpage">4062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DNicholls%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D21%26rft.issue%253D16%26rft.spage%253D4051%26rft.epage%253D4062%26rft.atitle%253DMitochondrial%2Bbarcodes%2Bare%2Bdiagnostic%2Bof%2Bshared%2Brefugia%2Bbut%2Bnot%2Bspecies%2Bin%2Bhybridizing%2Boak%2Bgallwasps.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05683.x%26rft_id%253Dinfo%253Apmid%252F22724511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05683.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22724511&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.35" data-doi="10.1111/j.1365-294X.2009.04499.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nicholls</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preuss</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayward</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melika</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csóka</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nieves-Aldrey</span>,  <span class="cit-name-given-names">J.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Askew</span>,  <span class="cit-name-given-names">R. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tavakoli</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schönrogge</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Concordant phylogeography and cryptic speciation in two Western Palaearctic oak gall parasitoid species complexes</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">592</span>–<span class="cit-lpage">609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DNicholls%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D592%26rft.epage%253D609%26rft.atitle%253DConcordant%2Bphylogeography%2Band%2Bcryptic%2Bspeciation%2Bin%2Btwo%2BWestern%2BPalaearctic%2Boak%2Bgall%2Bparasitoid%2Bspecies%2Bcomplexes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04499.x%26rft_id%253Dinfo%253Apmid%252F20070516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04499.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20070516&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.36" data-doi="10.1111/1365-2745.12085"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nordén</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Penttilä</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siitonen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tomppo</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ovaskainen</span>,  <span class="cit-name-given-names">O.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Specialist species of wood-inhabiting fungi struggle while generalists thrive in fragmented boreal forests</span>. <abbr class="cit-jnl-abbrev">Journal of Ecology</abbr>, <span class="cit-vol">101</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">701</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEcology%26rft.volume%253D101%26rft.spage%253D701%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1365-2745.12085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1365-2745.12085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.37" data-doi="10.1111/j.0030-1299.2005.13902.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Östergård</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ehrlén</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Among population variation in specialist and generalist seed predation - the importance of host plant distribution, alternative hosts and environmental variation</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr>, <span class="cit-vol">111</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">39</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D111%26rft.spage%253D39%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0030-1299.2005.13902.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0030-1299.2005.13902.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000231864900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.38" data-doi="10.1016/S0378-1127(01)00634-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petit</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brewer</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bordács</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burg</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheddadi</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coart</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cottrell</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csaikl</span>,  <span class="cit-name-given-names">U. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Dam</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deans</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Espinel</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fineschi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finkeldey</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glaz</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goicoechea</span>,  <span class="cit-name-given-names">P. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">J. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">König</span>,  <span class="cit-name-given-names">A. O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowe</span>,  <span class="cit-name-given-names">A. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Madsen</span>,  <span class="cit-name-given-names">S. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mátyás</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munro</span>,  <span class="cit-name-given-names">R. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Popescu</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Slade</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tabbener</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Vries</span>,  <span class="cit-name-given-names">S. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziegenhagen</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Beaulieu</span>,  <span class="cit-name-given-names">J.-L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kremer</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Identification of refugia and post-glacial colonisation routes of European white oaks based on chloroplast DNA and fossil pollen evidence</span>. <abbr class="cit-jnl-abbrev">Forest Ecology and Management</abbr>, <span class="cit-vol">156</span>(<span class="cit-issue">1-3</span>):<span class="cit-fpage">49</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2BEcology%2Band%2BManagement%26rft.volume%253D156%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1127%252801%252900634-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1127(01)00634-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000174149400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.39" data-doi="10.1111/j.0030-1299.2007.15871.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rand</span>,  <span class="cit-name-given-names">T. A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tscharntke</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Contrasting effects of natural habitat loss on generalist and specialist aphid natural enemies</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr>, <span class="cit-vol">116</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1353</span>–<span class="cit-lpage">1362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D116%26rft.spage%253D1353%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0030-1299.2007.15871.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0030-1299.2007.15871.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.40" data-doi="10.1046/j.1365-2540.2001.00872.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rokas</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkinson</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">G. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">West</span>,  <span class="cit-name-given-names">S. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Understanding patterns of genetic diversity in the oak gallwasp Biorhiza pallida: demographic history or a Wolbachia selective sweep?</span> <abbr class="cit-jnl-abbrev">Heredity</abbr>, <span class="cit-vol">87</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">294</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DRokas%26rft.auinit1%253DA.%26rft.volume%253D87%26rft.issue%253DPt%2B3%26rft.spage%253D294%26rft.epage%253D304%26rft.atitle%253DUnderstanding%2Bpatterns%2Bof%2Bgenetic%2Bdiversity%2Bin%2Bthe%2Boak%2Bgallwasp%2BBiorhiza%2Bpallida%253A%2Bdemographic%2Bhistory%2Bor%2Ba%2BWolbachia%2Bselective%2Bsweep%253F%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2001.00872.x%26rft_id%253Dinfo%253Apmid%252F11737276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2001.00872.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11737276&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172343300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rokas</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkinson</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Webster</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csoka</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Out of Anatolia: longitudinal gradients in genetic diversity support an eastern origin for a circum-Mediterranean oak gallwasp Andricus quercustozae</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">2153</span>–<span class="cit-lpage">2174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DRokas%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D8%26rft.spage%253D2153%26rft.epage%253D2174%26rft.atitle%253DOut%2Bof%2BAnatolia%253A%2Blongitudinal%2Bgradients%2Bin%2Bgenetic%2Bdiversity%2Bsupport%2Ban%2Beastern%2Borigin%2Bfor%2Ba%2Bcircum-Mediterranean%2Boak%2Bgallwasp%2BAndricus%2Bquercustozae.%26rft_id%253Dinfo%253Apmid%252F12859636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12859636&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.42" data-doi="10.1534/genetics.116.190223"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shanku</span>,  <span class="cit-name-given-names">A. G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kern</span>,  <span class="cit-name-given-names">A. D.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Effects of linked selective sweeps on demographic inference and model selection</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">204</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">1207</span>–<span class="cit-lpage">1223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.190223%26rft_id%253Dinfo%253Apmid%252F27605051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC8zLzEyMDciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8yMi8yMDE5LjEyLjIwLjg4NDQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.43" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">J. M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Haigh</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1974</span>). <span class="cit-article-title">The hitch-hiking effect of a favourable gene</span>. <abbr class="cit-jnl-abbrev">Genet. Res., Camb</abbr>, <span class="cit-vol">23</span>:<span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.44" data-doi="10.1038/s41588-019-0484-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Speidel</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forest</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Myers</span>,  <span class="cit-name-given-names">S. R.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">A method for genome-wide genealogy estimation for thousands of samples</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>, <span class="cit-vol">51</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">1321</span>–<span class="cit-lpage">1329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D51%26rft.spage%253D1321%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-019-0484-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-019-0484-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.45" data-doi="10.1093/nar/gki458"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stanke</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Morgenstern</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr>, <span class="cit-vol">33</span>(Web Server issue):<span class="cit-fpage">W465</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW465%26rft.epage%253DW467%26rft.atitle%253DAUGUSTUS%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bgene%2Bprediction%2Bin%2Beukaryotes%2Bthat%2Ballows%2Buser-defined%2Bconstraints%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki458%26rft_id%253Dinfo%253Apmid%252F15980513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980513&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400094&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.46" data-doi="10.1016/j.cub.2012.01.059"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lohse</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicholls</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fuentes-Utrilla</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sinclair</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schönrogge</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csóka</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melika</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nieves-Aldrey</span>,  <span class="cit-name-given-names">J.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pujade-Villar</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tavakoli</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Askew</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hickerson</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Reconstructing Community Assembly in Time and Space Reveals Enemy Escape in a Western Palearctic Insect Community</span>. <abbr class="cit-jnl-abbrev">Current Biology</abbr>, <span class="cit-vol">22</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">532</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DStone%26rft.auinit1%253DG.%2BN.%26rft.volume%253D22%26rft.issue%253D6%26rft.spage%253D532%26rft.epage%253D537%26rft.atitle%253DReconstructing%2Bcommunity%2Bassembly%2Bin%2Btime%2Band%2Bspace%2Breveals%2Benemy%2Bescape%2Bin%2Ba%2BWestern%2BPalearctic%2Binsect%2Bcommunity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2012.01.059%26rft_id%253Dinfo%253Apmid%252F22405865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2012.01.059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22405865&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Challis</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkinson</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csóka</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayward</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melika</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mutun</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preuss</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rokas</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sadeghi</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Schönrogge</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">The phylogeographical clade trade: tracing the impact of human-mediated dispersal on the colonization of northern Europe by the oak gallwasp Andricus kollari</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">16</span>(<span class="cit-issue">13</span>):<span class="cit-fpage">2768</span>–<span class="cit-lpage">2781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DStone%26rft.auinit1%253DG.%2BN.%26rft.volume%253D16%26rft.issue%253D13%26rft.spage%253D2768%26rft.epage%253D2781%26rft.atitle%253DThe%2Bphylogeographical%2Bclade%2Btrade%253A%2Btracing%2Bthe%2Bimpact%2Bof%2Bhuman-mediated%2Bdispersal%2Bon%2Bthe%2Bcolonization%2Bof%2Bnorthern%2BEurope%2Bby%2Bthe%2Boak%2Bgallwasp%2BAndricus%2Bkollari.%26rft_id%253Dinfo%253Apmid%252F17594446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17594446&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247561800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.48" data-doi="10.1146/annurev.ento.47.091201.145247"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schönrogge</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atkinson</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bellido</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pujade-Villar</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">The population biology of oak gall wasps (Hymenoptera: Cynipidae)</span>. <abbr class="cit-jnl-abbrev">Annual Review of Entomology</abbr>, <span class="cit-vol">47</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">633</span>–<span class="cit-lpage">668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bentomology%26rft.stitle%253DAnnu%2BRev%2BEntomol%26rft.aulast%253DStone%26rft.auinit1%253DG.%2BN.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D633%26rft.epage%253D668%26rft.atitle%253DTHE%2BPOPULATION%2BBIOLOGY%2BOF%2BOAK%2BGALL%2BWASPS%2B%2528HYMENOPTERA%253A%2BCYNIPIDAE%2529%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ento.47.091201.145247%26rft_id%253Dinfo%253Apmid%252F11729087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ento.47.091201.145247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11729087&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F22%2F2019.12.20.884403.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173421900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stone</span>,  <span class="cit-name-given-names">G. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">S. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csóka</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melika</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mutun</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pénzes</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sadeghi</span>,  <span class="cit-name-given-names">S. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schönrogge</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tavakoli</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nicholls</span>,  <span class="cit-name-given-names">J. A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Tournament ABC analysis of the western Palaearctic population history of an oak gall wasp, <em>Synergus umbraculus</em></span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">26</span>(<span class="cit-issue">23</span>):<span class="cit-fpage">6685</span>–<span class="cit-lpage">6703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D26%26rft.spage%253D6685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.50" data-doi="10.1073/pnas.1503784112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tallavaara</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luoto</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korhonen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Järvinen</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Seppä</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Human population dynamics in Europe over the Last Glacial Maximum</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, <span class="cit-vol">112</span>(<span class="cit-issue">27</span>):<span class="cit-fpage">8232</span>–<span class="cit-lpage">8237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1503784112%26rft_id%253Dinfo%253Apmid%252F26100880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzI3LzgyMzIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8yMi8yMDE5LjEyLjIwLjg4NDQwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tison</span>,  <span class="cit-name-given-names">J.-L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edmark</span>,  <span class="cit-name-given-names">V. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandoval-Castellanos</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Dyck</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tammaru</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Välimäki</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalén</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gotthard</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Signature of post-glacial expansion and genetic structure at the northern range limit of the speckled wood butterfly</span>. <abbr class="cit-jnl-abbrev">Biological Journal of the Linnean Society</abbr>, <span class="cit-vol">113</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">136</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BJournal%2Bof%2Bthe%2BLinnean%2BSociety%26rft.volume%253D113%26rft.spage%253D136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van der Auwera</span>,  <span class="cit-name-given-names">G. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carneiro</span>,  <span class="cit-name-given-names">M. O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartl</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poplin</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">del Angel</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levy-Moonshine</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jordan</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shakir</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roazen</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thibault</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banks</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garimella</span>,  <span class="cit-name-given-names">K. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altshuler</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabriel</span>,  <span class="cit-name-given-names">S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">DePristo</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline</span>. <abbr class="cit-jnl-abbrev">Current Protocols in Bioinformatics</abbr>, <span class="cit-vol">43</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">11.10.1</span>–<span class="cit-lpage">11.10.33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BProtocols%2Bin%2BBioinformatics%26rft.volume%253D43%26rft.spage%253D11101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vitales</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">García-Fernández</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garnatje</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pellicer</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Vallès</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Phylogeographic insights of the lowland species cheirolophus sempervirens in the southwestern iberian peninsula</span>. <abbr class="cit-jnl-abbrev">Journal of Systematics and Evolution</abbr>, <span class="cit-vol">54</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">65</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BSystematics%2Band%2BEvolution%26rft.volume%253D54%26rft.spage%253D65%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.20.884403v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-chapter-title">Coalescent Theory: An Introduction</span>. <span class="cit-publ-name">W. H. Freeman</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wall</span>,  <span class="cit-name-given-names">J. D.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Estimating ancestral population sizes and divergence times</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">163</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">395</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWall%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D395%26rft.epage%253D404%26rft.atitle%253DEstimating%2BAncestral%2BPopulation%2BSizes%2Band%2BDivergence%2BTimes%26rft_id%253Dinfo%253Apmid%252F12586724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvMzk1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMjIvMjAxOS4xMi4yMC44ODQ0MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884403v1.56"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Wolfram Research, I.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Mathematica</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.20.884403v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.20.884403v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2019.12.18.881268v1" title="Concerted cutting by Spo11 illuminates DNA break mechanisms and initiates gap repair during meiosis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 22, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.20.884403v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2019.12.20.884403v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.20.884403v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.20.884403v1" class="active" data-icon-position="" data-hide-link-title="0">Diverse demographic histories in a guild of hymenopteran parasitoids</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1066333" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Ksh6NJ_nmKgT4nioo5VzeqEi_opjPVKPz4rK4-WrGf4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1066333" id="node1066333--2" data-pisa="biorxiv;2019.12.20.884403v1" data-pisa-master="biorxiv;2019.12.20.884403" data-seqnum="1066333" data-apath="/biorxiv/early/2019/12/22/2019.12.20.884403.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Diverse demographic histories in a guild of hymenopteran parasitoids</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">William</span> <span class="nlm-surname">Walton</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Graham N</span> <span class="nlm-surname">Stone</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Konrad</span> <span class="nlm-surname">Lohse</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.20.884403; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.20.884403 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.20.884403v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1&amp;title=Diverse%20demographic%20histories%20in%20a%20guild%20of%20hymenopteran%20parasitoids" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1&amp;title=Diverse%20demographic%20histories%20in%20a%20guild%20of%20hymenopteran%20parasitoids" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1&amp;text=Diverse%20demographic%20histories%20in%20a%20guild%20of%20hymenopteran%20parasitoids" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1&amp;title=Diverse%20demographic%20histories%20in%20a%20guild%20of%20hymenopteran%20parasitoids" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1&amp;t=Diverse%20demographic%20histories%20in%20a%20guild%20of%20hymenopteran%20parasitoids" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1&amp;title=Diverse%20demographic%20histories%20in%20a%20guild%20of%20hymenopteran%20parasitoids" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1&amp;title=Diverse%20demographic%20histories%20in%20a%20guild%20of%20hymenopteran%20parasitoids" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1066333" id="node1066333--3" data-pisa="biorxiv;2019.12.20.884403v1" data-pisa-master="biorxiv;2019.12.20.884403" data-seqnum="1066333" data-apath="/biorxiv/early/2019/12/22/2019.12.20.884403.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Diverse demographic histories in a guild of hymenopteran parasitoids</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">William</span> <span class="nlm-surname">Walton</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Graham N</span> <span class="nlm-surname">Stone</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Konrad</span> <span class="nlm-surname">Lohse</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.20.884403; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.20.884403 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Diverse%20demographic%20histories%20in%20a%20guild%20of%20hymenopteran%20parasitoids&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Walton%2C+William&amp;rft.au=Stone%2C+Graham+N&amp;rft.au=Lohse%2C+Konrad"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1066333/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1066333/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1066333/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1066333/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1066333/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1066333/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1066333/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1066333/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1066333/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1066333/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1066333/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1066333/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1&amp;count=horizontal&amp;via=&amp;text=Diverse%20demographic%20histories%20in%20a%20guild%20of%20hymenopteran%20parasitoids&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884403v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.20.884403v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
