<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/embed/graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/11/06/522631.atom" />
<meta name="HW.pisa" content="biorxiv;522631v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations" />
<meta name="DC.Identifier" content="10.1101/522631" />
<meta name="DC.Date" content="2019-11-06" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Environmental factors interact with internal rules of population regulation, sometimes perturbing systems to alternate dynamics though changes in parameter values. Yet, pinpointing when such changes occur in naturally fluctuating populations is difficult. An algorithmic approach that can identify the timing and magnitude of parameter shifts would facilitate understanding of abrupt ecological transitions with potential to inform conservation and management of species.

The “Dynamic Shift Detector” is an algorithm to identify changes in parameter values governing temporal fluctuations in populations with nonlinear dynamics. The algorithm examines population time series data for the presence, location, and magnitude of parameter shifts. It uses an iterative approach to fitting subsets of time series data, then ranks the fit of break point combinations using model selection, assigning a relative weight to each break. We examined the performance of the Dynamic Shift Detector with simulations and two case studies. Under low environmental/sampling noise, the break point sets selected by the Dynamic Shift Detector contained the true simulated breaks with 70-100% accuracy. The weighting tool generally assigned breaks intentionally placed in simulated data (i.e., true breaks) with weights averaging &gt;0.8 and those due to sampling error (i.e., erroneous breaks) with weights averaging &lt;0.2. In our case study examining an invasion process, the algorithm identified shifts in population cycling associated with variations in resource availability. The shifts identified for the conservation case study highlight a decline process that generally coincided with changing management practices affecting the availability of hostplant resources.

When interpreted in the context of species biology, the Dynamic Shift Detector algorithm can aid management decisions and identify critical time periods related to species’ dynamics. In an era of rapid global change, such tools can provide key insights into the conditions under which population parameters, and their corresponding dynamics, can shift.

Author Summary Populations naturally fluctuate in abundance, and the rules governing these fluctuations are a result of both internal (density dependent) and external (environmental) processes. For these reasons, pinpointing when changes in populations occur is difficult. In this study, we develop a novel break-point analysis tool for population time series data. Using a density dependent model to describe a population’s underlying dynamic process, our tool iterates through all possible break point combinations (i.e., abrupt changes in parameter values) and applies information-theoretic decision tools (i.e. Akaike’s Information Criterion corrected for small sample sizes) to determine best fits. Here, we develop the approach, simulate data under a variety of conditions to demonstrate its utility, and apply the tool to two case studies: an invasion of multicolored Asian ladybeetle and declining monarch butterflies. The Dynamic Shift Detector algorithm identified parameter changes that correspond to known environmental change events in both case studies." />
<meta name="DC.Contributor" content="Christie A. Bahlai" />
<meta name="DC.Contributor" content="Elise F. Zipkin" />
<meta name="article:published_time" content="2019-11-06" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Environmental factors interact with internal rules of population regulation, sometimes perturbing systems to alternate dynamics though changes in parameter values. Yet, pinpointing when such changes occur in naturally fluctuating populations is difficult. An algorithmic approach that can identify the timing and magnitude of parameter shifts would facilitate understanding of abrupt ecological transitions with potential to inform conservation and management of species.&lt;/p&gt;&lt;p&gt;The “Dynamic Shift Detector” is an algorithm to identify changes in parameter values governing temporal fluctuations in populations with nonlinear dynamics. The algorithm examines population time series data for the presence, location, and magnitude of parameter shifts. It uses an iterative approach to fitting subsets of time series data, then ranks the fit of break point combinations using model selection, assigning a relative weight to each break. We examined the performance of the Dynamic Shift Detector with simulations and two case studies. Under low environmental/sampling noise, the break point sets selected by the Dynamic Shift Detector contained the true simulated breaks with 70-100% accuracy. The weighting tool generally assigned breaks intentionally placed in simulated data (i.e., true breaks) with weights averaging &amp;gt;0.8 and those due to sampling error (i.e., erroneous breaks) with weights averaging &amp;lt;0.2. In our case study examining an invasion process, the algorithm identified shifts in population cycling associated with variations in resource availability. The shifts identified for the conservation case study highlight a decline process that generally coincided with changing management practices affecting the availability of hostplant resources.&lt;/p&gt;&lt;p&gt;When interpreted in the context of species biology, the Dynamic Shift Detector algorithm can aid management decisions and identify critical time periods related to species’ dynamics. In an era of rapid global change, such tools can provide key insights into the conditions under which population parameters, and their corresponding dynamics, can shift.&lt;/p&gt;&lt;h3&gt;Author Summary&lt;/h3&gt;
&lt;p&gt;Populations naturally fluctuate in abundance, and the rules governing these fluctuations are a result of both internal (density dependent) and external (environmental) processes. For these reasons, pinpointing when changes in populations occur is difficult. In this study, we develop a novel break-point analysis tool for population time series data. Using a density dependent model to describe a population’s underlying dynamic process, our tool iterates through all possible break point combinations (i.e., abrupt changes in parameter values) and applies information-theoretic decision tools (i.e. Akaike’s Information Criterion corrected for small sample sizes) to determine best fits. Here, we develop the approach, simulate data under a variety of conditions to demonstrate its utility, and apply the tool to two case studies: an invasion of multicolored Asian ladybeetle and declining monarch butterflies. The Dynamic Shift Detector algorithm identified parameter changes that correspond to known environmental change events in both case studies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;522631v2" />
<meta name="citation_id" content="522631v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/522631v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/522631v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/522631v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631.full.pdf" />
<meta name="citation_doi" content="10.1101/522631" />
<meta name="citation_num_pages" content="22" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="522631" />
<meta name="citation_author" content="Christie A. Bahlai" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Kent State University" />
<meta name="citation_author_email" content="cbahlai@kent.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8937-8709" />
<meta name="citation_author" content="Elise F. Zipkin" />
<meta name="citation_author_institution" content="Department of Integrative Biology; Program in Ecology, Evolutionary Biology and Behavior, Michigan State University" />
<meta name="citation_reference" content="Andersen T, Carstensen J, Hernández-García E, Duarte CM. Ecological thresholds and regime shifts: approaches to identification. Trends in Ecology &amp; Evolution. 2009;24: 49–57. doi:10.1016/j.tree.2008.07.014" />
<meta name="citation_reference" content="Bestelmeyer BT, Ellison AM, Fraser WR, Gorman KB, Holbrook SJ, Laney CM, et al. Analysis of abrupt transitions in ecological systems. Ecosphere. 2011;2: art129. doi:10.1890/es11-00216.1" />
<meta name="citation_reference" content="Scheffer M, Carpenter S, Foley JA, Folke C, Walker B. Catastrophic shifts in ecosystems. Nature. 2001;413: 591–596. doi:10.1038/35098000" />
<meta name="citation_reference" content="Scheffer M, Carpenter SR. Catastrophic regime shifts in ecosystems: linking theory to observation. Trends in Ecology &amp; Evolution. 2003;18: 648–656. doi:10.1016/j.tree.2003.09.002" />
<meta name="citation_reference" content="May RM. Thresholds and breakpoints in ecosystems with a multiplicity of stable states. Nature. 1977;269: 471–477. doi:10.1038/269471a0" />
<meta name="citation_reference" content="Ducré-Robitaille J-F, Vincent LA, Boulet G. Comparison of techniques for detection of discontinuities in temperature series: DETECTING DISCONTINUITIES IN TEMPERATURE SERIES. International Journal of Climatology. 2003;23: 1087–1101. doi:10.1002/joc.924" />
<meta name="citation_reference" content="Rodionov SN. A sequential algorithm for testing climate regime shifts: ALGORITHM FOR TESTING REGIME SHIFTS. Geophysical Research Letters. 2004;31: n/a-n/a. doi:10.1029/2004GL019448" />
<meta name="citation_reference" content="Beaugrand G., Conversi A., Chiba S., Edwards M., Fonda-Umani S., Greene C., et al. Synchronous marine pelagic regime shifts in the Northern Hemisphere. Philosophical Transactions of the Royal Society B: Biological Sciences. 2015;370: 20130272. doi:10.1098/rstb.2013.0272" />
<meta name="citation_reference" content="Zampolli F. Optimal monetary policy in a regime-switching economy: The response to abrupt shifts in exchange rate dynamics. Journal of Economic Dynamics and Control. 2006;30: 1527–1567. doi:10.1016/j.jedc.2005.10.013" />
<meta name="citation_reference" content="Holling CS. Understanding the Complexity of Economic, Ecological, and Social Systems. Ecosystems. 2001;4: 390–405. doi:10.1007/s10021-001-0101-5" />
<meta name="citation_reference" content="Bjørnstad ON, Grenfell BT. Noisy Clockwork: Time Series Analysis of Population Fluctuations in Animals. Science. 2001;293: 638. doi:10.1126/science.1062226" />
<meta name="citation_reference" content="Hare SR, Mantua NJ. Empirical evidence for North Pacific regime shifts in 1977 and 1989. Progress in Oceanography. 2000;47: 103–145. doi:10.1016/S0079-6611(00)00033-1" />
<meta name="citation_reference" content="Carpenter SR, Brock WA, Cole JJ, Kitchell JF, Pace ML. Leading indicators of trophic cascades. Ecology Letters. 2008;11: 128–138. doi:10.1111/j.1461-0248.2007.01131.x" />
<meta name="citation_reference" content="Weimerskirch H, Inchausti P, Guinet C, Barbraud C. Trends in bird and seal populations as indicators of a system shift in the Southern Ocean. Antarctic Science. 2003;15: 249–256." />
<meta name="citation_reference" content="Berryman A, Lima M. Deciphering the effects of climate on animal populations: diagnostic analysis provides new interpretation of Soay sheep dynamics. The American Naturalist. 2006;168: 784–795." />
<meta name="citation_reference" content="Toms JD, Lesperance ML. Piecewise regression: a tool for identifying ecological thresholds. Ecology. 2003;84: 2034–2041. doi:10.1890/02-0472" />
<meta name="citation_reference" content="Knapp AK, Smith MD, Hobbie SE, Collins SL, Fahey TJ, Hansen GJA, et al. Past, present, and future roles of long-term experiments in the LTER Network. Bioscience. 2012;62: 377–389. doi:10.1029/2008gb003336" />
<meta name="citation_reference" content="Braun JV, Muller H-G. Statistical Methods for DNA Sequence Segmentation. Statistical Science. 1998;13: 142–162." />
<meta name="citation_reference" content="Zeileis A, Leisch F, Hornik K, Kleiber C. strucchange. An R package for testing for structural change in linear regression models. 2001." />
<meta name="citation_reference" content="Killick R, Eckley I. changepoint: An R package for changepoint analysis. Journal of Statistical Software. 2014;58: 1–19." />
<meta name="citation_reference" content="Priyadarshana W, Sofronov G. Multiple break-points detection in array CGH data via the cross-entropy method. IEEE/ACM Transactions on Computational Biology and Bioinformatics (TCBB). 2015;12: 487–498." />
<meta name="citation_reference" content="Mauget SA. Multidecadal Regime Shifts in U.S. Streamflow, Precipitation, and Temperature at the End of the Twentieth Century. Journal of Climate. 2003;16: 3905–3916. doi:10.1175/1520-0442(2003)016&lt;3905:MRSIUS&gt;2.0.CO;2" />
<meta name="citation_reference" content="Jenouvrier S, Weimerskirch H, Barbraud C, Park Y-H, Cazelles B. Evidence of a shift in the cyclicity of Antarctic seabird dynamics linked to climate. Proceedings of the Royal Society B: Biological Sciences. 2005;272: 887–895. doi:10.1098/rspb.2004.2978" />
<meta name="citation_reference" content="Cazelles B, Chavez M, Berteaux D, Ménard F, Vik JO, Jenouvrier S, et al. Wavelet analysis of ecological time series. Oecologia. 2008;156: 287–304. doi:10.1007/s00442-008-0993-2" />
<meta name="citation_reference" content="Carl Boettiger, Alan Hastings. Early warning signals and the prosecutor’s fallacy. Proceedings of the Royal Society B: Biological Sciences. 2012;279: 4734–4739. doi:10.1098/rspb.2012.2085" />
<meta name="citation_reference" content="citation_title=The nature of scientific evidence;citation_year=2004" />
<meta name="citation_reference" content="Turchin P. Complex population dynamics: a theoretical/empirical synthesis. Princeton University Press; 2003." />
<meta name="citation_reference" content="May RM. Simple mathematical models with very complicated dynamics. Nature. 1976;261: 459–467. doi:10.1038/261459a0" />
<meta name="citation_reference" content="Barraquand F, Louca S, Abbott KC, Cobbold CA, Cordoleani F, DeAngelis DL, et al. Moving forward in circles: challenges and opportunities in modelling population cycles. Ecology letters. 2017;20: 1074–1092." />
<meta name="citation_reference" content="Boettiger C. From noise to knowledge: how randomness generates novel phenomena and reveals information. Ecology Letters. 2018;21: 1255–1267. doi:10.1111/ele.13085" />
<meta name="citation_reference" content="Gadrich T, Katriel G. A Mechanistic Stochastic Ricker Model: Analytical and Numerical Investigations. Int J Bifurcation Chaos. 2016;26: 1650067. doi:10.1142/S021812741650067X" />
<meta name="citation_reference" content="Ricker WE. Stock and Recruitment. J Fish Res Bd Can. 1954;11: 559–623. doi:10.1139/f54-039" />
<meta name="citation_reference" content="Brook BW, Bradshaw CJ. Strength of evidence for density dependence in abundance time series of 1198 species. Ecology. 2006;87: 1445–1451." />
<meta name="citation_reference" content="Hall AR, Osborn DR, Sakkas N. Inference on Structural Breaks using Information Criteria. The Manchester School. 2013;81: 54–81. doi:10.1111/manc.12017" />
<meta name="citation_reference" content="Burnham KP, Anderson DR. Model selection and multimodal inference: a practical information-theoretic approach. 2nd ed. New York: Springer Science + Business Media, LLC; 2002." />
<meta name="citation_reference" content="Wagenmakers E-J, Farrell S. AIC model selection using Akaike weights. Psychonomic Bulletin &amp; Review. 2004;11: 192–196. doi:10.3758/BF03206482" />
<meta name="citation_reference" content="R Development Core Team. R: A Language and Environment for Statistical Computing 3.3.3. R Foundation for Statistical Computing. 2017. Available: http://www.R-project.org" />
<meta name="citation_reference" content="Elzhov TV, Mullen KM, Spiess A-N, Bolker B. minpack. lm: R Interface to the Levenberg-Marquardt Nonlinear Least-Squares Algorithm Found in MINPACK. 2016." />
<meta name="citation_reference" content="Bahlai C. cbahlai/dynamic_shift_detector: The Dynamic Shift Detector. Zenodo; 2019. doi:10.5281/zenodo.3368486" />
<meta name="citation_reference" content="Bahlai CA, Colunga-Garcia M, Gage SH, Landis DA. Long term functional dynamics of an aphidophagous coccinellid community are unchanged in response to repeated invasion. PLoS One. 2013;8: e83407. doi:10.1371/journal.pone.0083407" />
<meta name="citation_reference" content="Bahlai C, Colunga-Garcia M, Gage S, Landis D. The role of exotic ladybeetles in the decline of native ladybeetle populations: evidence from long-term monitoring. Biol Invasions. 2015;17: 1005–1024. doi:10.1007/s10530-014-0772-4" />
<meta name="citation_reference" content="Bahlai CA, vander Werf W, O’Neal M, Hemerik L, Landis DA. Shifts in dynamic regime of an invasive lady beetle are linked to the invasion and insecticidal management of its prey. Ecological Applications. 2015 [cited 14 May 2015]. doi:10.1890/14-2022.1" />
<meta name="citation_reference" content="Bahlai CA, Sears MK. Population dynamics of Harmonia axyridis and Aphis glycines in Niagara Peninsula soybean fields and vineyards. Journal of the Entomological Society of Ontario. 2009;140: 27–39." />
<meta name="citation_reference" content="Heimpel G, Frelich L, Landis D, Hopper K, Hoelmer K, Sezen Z, et al. European buckthorn and Asian soybean aphid as components of an extensive invasional meltdown in North America. Biological Invasions. 2010;12: 2913–2931. doi:10.1007/s10530-010-9736-5" />
<meta name="citation_reference" content="Rhainds M, Yoo HJS, Kindlmann P, Voegtlin D, Castillo D, Rutledge C, et al. Two-year oscillation cycle in abundance of soybean aphid in Indiana. Agricultural and Forest Entomology. 2010;12: 251–257." />
<meta name="citation_reference" content="Ragsdale DW, Voegtlin DJ, O’Neil RJ. Soybean aphid biology in North America. Annals of the Entomological Society of America. 2004;97: 204–208." />
<meta name="citation_reference" content="Wu Z, Schenk-Hamlin D, Zhan W, Ragsdale DW, Heimpel GE. The soybean aphid in China: a historical review. Annals of the Entomological Society of America. 2004;97: 209–218." />
<meta name="citation_reference" content="Urquhart FA, Urquhart NR. Autumnal migration routes of the eastern population of the monarch butterfly (Danaus p. plexippus L.; Danaidae; Lepidoptera) in North America to the overwintering site in the Neovolcanic Plateau of Mexico. Can J Zool. 1978;56: 1759–1764. doi:10.1139/z78-240" />
<meta name="citation_reference" content="Flockhart DTT, Brower LP, Ramirez MI, Hobson KA, Wassenaar LI, Altizer S, et al. Regional climate on the breeding grounds predicts variation in the natal origin of monarch butterflies overwintering in Mexico over 38 years. Glob Change Biol. 2017;23: 2565–2576. doi:10.1111/gcb.13589" />
<meta name="citation_reference" content="Prysby MD, Oberhauser KS. Temporal and geographic variation in monarch densities: citizen scientists document monarch population patterns. The monarch butterfly: Biology and conservation. 2004; 9–20." />
<meta name="citation_reference" content="Sarkar S. What Is Threatening Monarchs? BioScience. 2017;67: 1080–1080. doi:10.1093/biosci/bix120" />
<meta name="citation_reference" content="Lovett J. Monarch Population Status. Monarch Watch; 2017. Available: http://monarchwatch.org/blog/2017/02/11/monarch-population-status-30/" />
<meta name="citation_reference" content="Hartzler RG. Reduction in common milkweed (Asclepias syriaca) occurrence in Iowa cropland from 1999 to 2009. Crop Protection. 2010;29: 1542–1544. doi:10.1016/j.cropro.2010.07.018" />
<meta name="citation_reference" content="Pleasants JM, Oberhauser KS. Milkweed loss in agricultural fields because of herbicide use: effect on the monarch butterfly population. Insect Conservation and Diversity. 2013;6: 135–144. doi:10.1111/j.1752-4598.2012.00196.x" />
<meta name="citation_reference" content="Zaya DN, Pearse IS, Spyreas G. Long-Term Trends in Midwestern Milkweed Abundances and Their Relevance to Monarch Butterfly Declines. BioScience. 2017;67: 343–356. doi:10.1093/biosci/biw186" />
<meta name="citation_reference" content="Powles SODSB. Glyphosate-resistant crops and weeds: now and in the future. 2010." />
<meta name="citation_reference" content="Baker NT. Estimated annual agricultural pesticide use by crop group for states of the conterminous United States, 1992-2014. National Water Quality Assessment Program. 2017. doi:10.5066/F7NP22KM" />
<meta name="citation_reference" content="Saunders SP, Ries L, Oberhauser KS, Thogmartin WE, Zipkin EF. Local and cross-seasonal associations of climate and land use with abundance of monarch butterflies Danaus plexippus. Ecography. 2017; n/a-n/a. doi:10.1111/ecog.02719" />
<meta name="citation_reference" content="Zipkin EF, Ries L, Reeves R, Regetz J, Oberhauser KS. Tracking climate impacts on the migratory monarch butterfly. Glob Change Biol. 2012;18: 3039–3049. doi:10.1111/j.1365-2486.2012.02751.x" />
<meta name="citation_reference" content="Brower LP, Kust DR, Rendon-Salinas E, Serrano EG, Kust KR, Miller J, et al. Catastrophic winter storm mortality of monarch butterflies in Mexico during January 2002. The Monarch butterfly: biology and conservation. 2004; 151–166." />
<meta name="citation_reference" content="Vidal O, Rendón-Salinas E. Dynamics and trends of overwintering colonies of the monarch butterfly in Mexico. Biological Conservation. 2014;180: 165–175. doi:10.1016/j.biocon.2014.09.041" />
<meta name="citation_reference" content="Davis AK, Altizer S. New Perspectives on Monarch Migration, Evolution, and Population Biology. Monarchs in a Changing World: Biology and Conservation of an Iconic Butterfly. 2015; 203." />
<meta name="citation_reference" content="Saunders SP, Ries L, Neupane N, Ramírez MI, García-Serrano E, Rendón-Salinas E, et al. Multiscale seasonal factors drive the size of winter monarch colonies. Proc Natl Acad Sci USA. 2019;116: 8609. doi:10.1073/pnas.1805114116" />
<meta name="citation_reference" content="Abbott KC, Dwyer G. Food Limitation and Insect Outbreaks: Complex Dynamics in Plant-Herbivore Models. Journal of Animal Ecology. 2007;76: 1004–1014." />
<meta name="twitter:title" content="The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Environmental factors interact with internal rules of population regulation, sometimes perturbing systems to alternate dynamics though changes in parameter values. Yet, pinpointing when such changes occur in naturally fluctuating populations is difficult. An algorithmic approach that can identify the timing and magnitude of parameter shifts would facilitate understanding of abrupt ecological transitions with potential to inform conservation and management of species.

The “Dynamic Shift Detector” is an algorithm to identify changes in parameter values governing temporal fluctuations in populations with nonlinear dynamics. The algorithm examines population time series data for the presence, location, and magnitude of parameter shifts. It uses an iterative approach to fitting subsets of time series data, then ranks the fit of break point combinations using model selection, assigning a relative weight to each break. We examined the performance of the Dynamic Shift Detector with simulations and two case studies. Under low environmental/sampling noise, the break point sets selected by the Dynamic Shift Detector contained the true simulated breaks with 70-100% accuracy. The weighting tool generally assigned breaks intentionally placed in simulated data (i.e., true breaks) with weights averaging &gt;0.8 and those due to sampling error (i.e., erroneous breaks) with weights averaging &lt;0.2. In our case study examining an invasion process, the algorithm identified shifts in population cycling associated with variations in resource availability. The shifts identified for the conservation case study highlight a decline process that generally coincided with changing management practices affecting the availability of hostplant resources.

When interpreted in the context of species biology, the Dynamic Shift Detector algorithm can aid management decisions and identify critical time periods related to species’ dynamics. In an era of rapid global change, such tools can provide key insights into the conditions under which population parameters, and their corresponding dynamics, can shift.

Author Summary Populations naturally fluctuate in abundance, and the rules governing these fluctuations are a result of both internal (density dependent) and external (environmental) processes. For these reasons, pinpointing when changes in populations occur is difficult. In this study, we develop a novel break-point analysis tool for population time series data. Using a density dependent model to describe a population’s underlying dynamic process, our tool iterates through all possible break point combinations (i.e., abrupt changes in parameter values) and applies information-theoretic decision tools (i.e. Akaike’s Information Criterion corrected for small sample sizes) to determine best fits. Here, we develop the approach, simulate data under a variety of conditions to demonstrate its utility, and apply the tool to two case studies: an invasion of multicolored Asian ladybeetle and declining monarch butterflies. The Dynamic Shift Detector algorithm identified parameter changes that correspond to known environmental change events in both case studies." />
<meta name="og-title" property="og:title" content="The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/522631v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Environmental factors interact with internal rules of population regulation, sometimes perturbing systems to alternate dynamics though changes in parameter values. Yet, pinpointing when such changes occur in naturally fluctuating populations is difficult. An algorithmic approach that can identify the timing and magnitude of parameter shifts would facilitate understanding of abrupt ecological transitions with potential to inform conservation and management of species.

The “Dynamic Shift Detector” is an algorithm to identify changes in parameter values governing temporal fluctuations in populations with nonlinear dynamics. The algorithm examines population time series data for the presence, location, and magnitude of parameter shifts. It uses an iterative approach to fitting subsets of time series data, then ranks the fit of break point combinations using model selection, assigning a relative weight to each break. We examined the performance of the Dynamic Shift Detector with simulations and two case studies. Under low environmental/sampling noise, the break point sets selected by the Dynamic Shift Detector contained the true simulated breaks with 70-100% accuracy. The weighting tool generally assigned breaks intentionally placed in simulated data (i.e., true breaks) with weights averaging &gt;0.8 and those due to sampling error (i.e., erroneous breaks) with weights averaging &lt;0.2. In our case study examining an invasion process, the algorithm identified shifts in population cycling associated with variations in resource availability. The shifts identified for the conservation case study highlight a decline process that generally coincided with changing management practices affecting the availability of hostplant resources.

When interpreted in the context of species biology, the Dynamic Shift Detector algorithm can aid management decisions and identify critical time periods related to species’ dynamics. In an era of rapid global change, such tools can provide key insights into the conditions under which population parameters, and their corresponding dynamics, can shift.

Author Summary Populations naturally fluctuate in abundance, and the rules governing these fluctuations are a result of both internal (density dependent) and external (environmental) processes. For these reasons, pinpointing when changes in populations occur is difficult. In this study, we develop a novel break-point analysis tool for population time series data. Using a density dependent model to describe a population’s underlying dynamic process, our tool iterates through all possible break point combinations (i.e., abrupt changes in parameter values) and applies information-theoretic decision tools (i.e. Akaike’s Information Criterion corrected for small sample sizes) to determine best fits. Here, we develop the approach, simulate data under a variety of conditions to demonstrate its utility, and apply the tool to two case studies: an invasion of multicolored Asian ladybeetle and declining monarch butterflies. The Dynamic Shift Detector algorithm identified parameter changes that correspond to known environmental change events in both case studies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-11-06" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/522631v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/522631v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/522631v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/522631v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/982714" />
    <title>The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"vfWvTIdqXb8IAbZX7_xVEJimuwj3V0QpnxI8e-gET4Y","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;522631v2"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;522631v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;522631v2"}],"nid":"982714","apath":"\/biorxiv\/early\/2019\/11\/06\/522631.atom","pisa":"biorxiv;522631v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/522631v2","title":"The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations","identifier":"node\/982714"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VqKgEkIq9mGIqdK7KJ7w4PKmk4nvr3ubGezlbln+kYObVaD8eAykTsJaULxYnBTzpGjaWT1yBiFZMWK8mU+xrRqkAUq3geHDchKxZraSZJRdhyNiknhTjs1Ka3Of8appF2hhPdI+3IAtmKPP\\\\\\\/vdnGi6KLxWgXOLakB4oWwaFKEHaSK05pN8bEL2ABg4MtXjelSl93Bd1p9yvPTByc+ZDmurbTYWLshyz3sjpZbP0zF7B\\\\\\\/iutl5f5PAoUgDJdNY5IGKZCZqXNJEDYOR0SxCJDgjXaPmteSAQpOBkf+TALZ9oD7+bqGtknCt2ZUOgrYP6TyVGLdOUBMd368j7KAQUyG872z1RaMiCIIv8jPc9Fe3SK8mqwkhR\\\\\\\/npxJ8ezNqX3Y+1zM\\\\\\\/JxdAa5bjg9lADeV8frIKkOmdszCu8v7CHBP3baaqNJDCKkW21yGf+zlK7uEprgBPTQCLw2oh3CzAcoWQuR+D6O\\\\\\\/asNdQ86CHPDIBgkxhRUkG35fBdmPKhPAdPAka67XUWWGlh5sNKDgV00KbUvtmLBp0q8TcWGd9RpUIifiHknuy4loG7YX01443JTEPqPGISz\\\\\\\/mkl5QEmfDDT5VdyfmJNQvWdsJLVu2APAhBSJqvKt8pgyDBkNYg1FjUtxcfO23orcDuPL1QDr0crNDVKnjT9q+K3jtSkyS8KHehMaWT6zTToM\\\\\\\/3cAnu1FQiqXtYRLCpX\\\\\\\/2n2wJgjREi4Ipb1iWM3hCQPlAJFpYTITmU\\\\\\\/CppA3kQk9Fahs1LbdjYro0VUcGNclpWLLXvp4jXOBH4MPF2KHVzERvr+ObaxvRd6iBSz0wgDxZAVhecc+g1t\\\\\\\/8bvd6n9WtCe0kEl\\\\\\\/n1rr5qO5\\\\\\\/i1abDXRNeSagbUwW01fBiUqWqcpknARlO6OqGW4+qHi5L63DJllMCrSyYYn8boiDR44cMpsYYUmIzECYTm2c\\\\\\\/DOP1xZNIhWwghFFGLs0hkrnZprWsfRjTOIBTdvRZZdg9hSyDnZ6imTM4+9JswBh3dBu7AkgOulQnVgc2j060fj7cERAZTQqCSWUgtQ0zBj1ILq81x\\\\\\\/qr81f1DWrGsDygqACm2H1Pf6EvrX8MNGGOxrvIWfS8O+9u8DfdSmH7mJXod1trM32f8LjeZa7Giymo4NCKnFFKvsGeuh1McG9HedANwjXIccO3sckvku3gwe2fx9GTxZ+zUgnjKPv6NP32DF\\\\\\\/yfiXhPI2WoiVgHqxeO2eIPCWIn6YUf9D9YiywoCa+3jOl4HA7\\\\\\\/X39YLKlo8oAALdyAajQToFpPF\\\\\\\/63u8iThIm\\\\\\\/Bv4raV0ct1Zo9gG3ddLPxvuG5rIx2ETBDpcM=\\\u0022,\\\u0022iv\\\u0022:\\\u002285illXIUsxTkHOh0aegUlg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00229f5351aa63697f18a5c8e1f0da3560bdf6f44cbd4f075cb61ee0c7a99a185f28\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/522631v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-982714 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/522631v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-q4hfALGP65Fg54pzYTqES4Yb6TwMZXh6v1xvOecOpM4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="982714" id="node982714" data-pisa="biorxiv;522631v2" data-pisa-master="biorxiv;522631" data-apath="/biorxiv/early/2019/11/06/522631.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8937-8709" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Christie A.</span> <span class="nlm-surname">Bahlai</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Elise F.</span> <span class="nlm-surname">Zipkin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/522631 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node982714" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Christie A. Bahlai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biological Sciences, Kent State University</span>. Kent, Ohio, <span class='nlm-country'>United States of America</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christie%2BA.%2BBahlai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bahlai%20CA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristie%2BA.%2BBahlai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8937-8709" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christie A. Bahlai</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:cbahlai@kent.edu" class="" data-icon-position="" data-hide-link-title="0">cbahlai@kent.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elise F. Zipkin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Integrative Biology; Program in Ecology, Evolutionary Biology and Behavior, Michigan State University</span>, East Lansing, Michigan, <span class='nlm-country'>United States of America</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elise%2BF.%2BZipkin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zipkin%20EF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AElise%2BF.%2BZipkin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/522631v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:982714" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:982714/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/522631v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:982714" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:982714/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/522631v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:982714" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:982714/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/522631v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:982714" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:982714/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/522631v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:982714" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:982714/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/522631v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:982714" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:982714/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/522631v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:982714" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:982714/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Environmental factors interact with internal rules of population regulation, sometimes perturbing systems to alternate dynamics though changes in parameter values. Yet, pinpointing when such changes occur in naturally fluctuating populations is difficult. An algorithmic approach that can identify the timing and magnitude of parameter shifts would facilitate understanding of abrupt ecological transitions with potential to inform conservation and management of species.</p><p id="p-3">The “Dynamic Shift Detector” is an algorithm to identify changes in parameter values governing temporal fluctuations in populations with nonlinear dynamics. The algorithm examines population time series data for the presence, location, and magnitude of parameter shifts. It uses an iterative approach to fitting subsets of time series data, then ranks the fit of break point combinations using model selection, assigning a relative weight to each break. We examined the performance of the Dynamic Shift Detector with simulations and two case studies. Under low environmental/sampling noise, the break point sets selected by the Dynamic Shift Detector contained the true simulated breaks with 70-100% accuracy. The weighting tool generally assigned breaks intentionally placed in simulated data (i.e., true breaks) with weights averaging &gt;0.8 and those due to sampling error (i.e., erroneous breaks) with weights averaging &lt;0.2. In our case study examining an invasion process, the algorithm identified shifts in population cycling associated with variations in resource availability. The shifts identified for the conservation case study highlight a decline process that generally coincided with changing management practices affecting the availability of hostplant resources.</p><p id="p-4">When interpreted in the context of species biology, the Dynamic Shift Detector algorithm can aid management decisions and identify critical time periods related to species’ dynamics. In an era of rapid global change, such tools can provide key insights into the conditions under which population parameters, and their corresponding dynamics, can shift.</p><div id="sec-1" class="subsection"><p id="p-5"><strong>Author Summary</strong> Populations naturally fluctuate in abundance, and the rules governing these fluctuations are a result of both internal (density dependent) and external (environmental) processes. For these reasons, pinpointing when changes in populations occur is difficult. In this study, we develop a novel break-point analysis tool for population time series data. Using a density dependent model to describe a population’s underlying dynamic process, our tool iterates through all possible break point combinations (i.e., abrupt changes in parameter values) and applies information-theoretic decision tools (i.e. Akaike’s Information Criterion corrected for small sample sizes) to determine best fits. Here, we develop the approach, simulate data under a variety of conditions to demonstrate its utility, and apply the tool to two case studies: an invasion of multicolored Asian ladybeetle and declining monarch butterflies. The Dynamic Shift Detector algorithm identified parameter changes that correspond to known environmental change events in both case studies.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-8">Abrupt and persistent changes in ecological processes, and methods to detect them, have long interested ecologists [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Changes to the rules governing system dynamics are often associated with substantial changes to biodiversity and ecosystem functions. Understanding when, and how these changes occur is critically important to broader evaluation of system behaviors. The study of abrupt changes, discontinuities, and regime shifts is highly interdisciplinary, and has been examined for a variety of processes related to climate [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>], ecology [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>], and economic and social systems [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>].</p><p id="p-9">Population dynamics are determined by internal, biotic rules and also external abiotic factors, leading to both stochastic and deterministic forces affecting abundance through time [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. External perturbations can lead to shifts in population dynamics, such that the parameters governing population abundance transition to other values [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>]. In the context of this study, we define the set of parameters controlling dynamics as a population’s <em>dynamic rule</em>, and an abrupt shift in these parameter values as a <em>dynamic shift</em>. We use the term <em>break point</em> to describe the location in time series data where the dynamic shift occurs.</p><p id="p-10">Although theoretically straightforward, identifying dynamic shifts in noisy ecological systems is challenging using real-world data due to a lack of systematic, adaptable tools [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. Arguably, the most common approach to identify break points is through the use of segmented regressions [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. However, these models don’t adequately account for nonlinearities, and uncertainties in the existence and location of breaks are typically not quantified [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>]. Break points are often applied to time series data <em>ad hoc</em>, based on data visualization or specific hypotheses surrounding factors affecting population changes [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>–<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>], creating the potential for biases in break point selection.</p><p id="p-11">Several break point detection methods have been developed to address issues associated with <em>ad hoc</em> approaches. Such dynamic shift analysis tools use a variety of statistical optimization strategies, including linear and moving average methods [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>–<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. For example, climatological and econometrics time series data are frequently examined for stepwise statistical deviations from the mean or variance [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>,<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>]. To fit the periodicity of population time series data, wavelet analyses have also been used to detect break points [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>] but this method does not mechanistically account for density-dependent processes such that changes in parameter values are not easily interpretable [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Dynamic shift detection methods that explicitly account for non-linear population processes may be less likely to yield false positives than methods based on summary statistics [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>].</p><p id="p-12">Break point detection methods based on statistical measures tend to rely on null hypothesis testing (i.e., that no dynamic shift occurred) and thus they have low sensitivity in situations where statistical power is limited. Additionally, such methods do not provide a means for assessing uncertainty in the existence and magnitude of break points [<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>]. In a 2009 review, Andersen and colleagues note that if common break point methods were used on typical ecological time series with 20-40 time steps, only the most extreme transitions occurring near the midpoint of the time series would be deemed ‘significant’ [<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">1</a>]. They concluded that break point analyses could be enhanced with respect to both sensitivity and parsimony by use of model selection procedures. Thus, to address these limitations in the ability to identify dynamic shifts in population processes, it is necessary to develop rigorous tools that allow users to accommodate non-linear population models and quantify uncertainties associated with the existence of potential break points.</p><p id="p-13">In this paper, we develop a generalizable algorithm, the Dynamic Shift Detector (DSD), to identify dynamic shifts in populations with density-dependent growth using time series data. The DSD algorithm uses an iterative approach, grounded in information theoretic model selection. We illustrate the DSD using the Ricker model because of this model’s simplicity and high performance under a variety of realistic environmental scenarios. Density-dependent population models such as the Ricker provide a convenient, generalizable proxy for a variety of ecological processes because of their relatively simple parameterization and potential to explain complex dynamics [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>]. Although deterministic approaches to population modelling have largely fallen out of favor for more complex structures and stochastic elements [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>–<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>], simple dynamic models remain useful due to their easily interpretable and ecologically meaningful parameters [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. Further, the techniques described in our paper can be readily adapted to other model structures, including more complicated processes such as seasonal periodicity or lag effects.</p><p id="p-14">We describe the basic structure of our DSD algorithm and how it can be used to evaluate the presence, location, and magnitude of dynamic shifts in population parameters (i.e., break points in time series abundance data). We demonstrate the utility of our algorithm through a series of simulations and apply the algorithm to empirical case studies of two populations of economic and conservation concern. First, we examine the invasion process of the multicolored Asian ladybeetle (<em>Harmonia axyridis</em>), a cosmopolitan invasive, in the two decades following its arrival in Midwestern US agricultural ecosystems. Then, we examine the declining eastern monarch butterfly (<em>Danaus plexippus</em>) population using census data collected on its overwintering grounds in Mexico over a similar two-decade period. In our ladybeetle case study, the DSD algorithm identified dynamic shifts associated with known variation in prey availability, with moderately high weight for a break coinciding with prey arrival and moderately low weight for a break coinciding with management actions aimed to control the prey. The results for the monarch population were more ambiguous, with greater uncertainty about the number and location of breaks in the time series. Several equivalently performing break point combinations had divergent weights for various break points, suggesting that multiple, super-imposed biological processes drive the dynamics of this population.</p></div><div class="section" id="sec-3"><h2 class="">Models</h2><div id="sec-4" class="subsection"><h3>The Dynamic Shift Detector algorithm</h3><p id="p-15">Although any time series population model can be used with our tool, we illustrate the DSD algorithm with a Ricker population model. To do this, we assume that the population size in time t+1, <em>N</em><sub><em>t</em>+1</sub>, is dependent on the population size in time t, <em>N</em><sub><em>t</em></sub> and regulated by two parameters: the carrying capacity of the system, <em>K</em>, and the annual intrinsic growth rate, <em>r</em> [<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>]:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/06/522631/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/06/522631/embed/graphic-1.gif"/></noscript></span>

</span></p><p id="p-16">We further assume that observed annual population abundance is partially stochastic and may be influenced by environmental variation and sampling error. We include an error term <em>ε</em><sub><em>t</em></sub> to represent this noise, which follows a normal distribution centered around zero with a variance of <em>σ</em><sup>2</sup>. The parameters K, r, and <em>ε</em><sub><em>t</em></sub> are estimated from the population time series data (N<sub>1</sub>, N<sub>2</sub>, … N<sub><em>t</em></sub>). The Ricker model is a useful starting point for break point analyses because 1) it does not rely on any external information (abundance in time t is a function of only abundance in time <em>t</em>-1); 2) only three parameters (including the error) need to be estimated, and those parameters have ecologically meaningful interpretations; and 3) it is an extremely flexible distribution, taking a variety of forms, from linear to compensatory to over-compensatory, and thus has a wide range of applications across a variety of taxa [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>,<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. Subsequent applications of the DSD algorithm can incorporate other population models if the life history of the target organism requires a different structure.</p><p id="p-17">To build the DSD algorithm, we use an iterative, model-selection process to determine if, and when, shifts in parameter values occur within a given time series. To achieve this, we first fit the Ricker model to the entire time series of available data. Then the population time series is subdivided into all possible combinations of 2, 3, …, n subsets of sequential data points (hereafter, ‘break point combinations’) and the Ricker model is fit to each of the subsets produced for each break point combination. To avoid overfitting, we constrain break point combinations to include only subsets with a minimum of four sequential data points.</p><p id="p-18">After fitting all break point combinations, we evaluate the candidate set of models by calculating the Akaike Information Criteria for small sample sizes (AICc) for each segment and summing them accordingly [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. Fits for break point combinations with comparatively lower AICc values are considered to have better performance. For a given time series, the DSD algorithm produces a set of top performing break point combinations for cases in which model fits produce equivalent AICc values (i.e. within 2 units of the best-performing fit; [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]). To evaluate the strength of evidence for an identified break in the time series, we use the relative variable importance method [<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>]. We compute the Akaike weight <em>w</em><sub>i</sub> (a measure of the relative likelihood of a break point combination, given the data and the set of break point combinations being tested) for every identified break point across all combinations. Commonly used in model averaging, the <em>w</em><sub>1</sub>, <em>w</em><sub>2</sub>,… <em>w</em><sub>n</sub> are interpreted as the respective conditional probabilities for each model in a set of <em>n</em> models [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. Break weight (i.e., relative variable importance, <em>sensu</em> [<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">35</a>]) is computed as the sum of the Akaike weights for all break point combinations, where that break point appears. Break point combinations with weights &lt;0.001 were excluded to increase computational efficiency.</p><p id="p-19">We selected AICc as our information criterion for model selection within the DSD algorithm because it provides a balance of specificity and sensitivity. However, we also completed a parallel analysis with an identical procedure using AIC as the information criterion for decision-making, which is documented in Appendix S1. AICc is a function of AIC with a correction for small sample bias, which is appropriate for the sample sizes typical to contemporary population time series data (i.e., 15-30 years/data points) and is designed to minimize the risk of overfitting during model selection [<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">35</a>]. However, use of AIC for model selection may be desirable when increased algorithmic sensitivity to dynamic shifts is desired.</p><p id="p-20">The DSD algorithm is implemented as a series of R functions to enable a user to quickly generate a list of potential break points for a population time series dataset. The algorithm (and all subsequent simulations and case studies) were scripted and run in R Version 3.3.3 [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. For fitting the Ricker model, we used the Levenberg-Marquardt nonlinear least-squares algorithm as implemented in the package minpack.LM [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. All data manipulations, analyses and figure scripts, including the complete development history, are publicly available in a Github repository at <a href="https://github.com/cbahlai/dynamic_shift_detector">https://github.com/cbahlai/dynamic_shift_detector</a> [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. We summarize the role of each function used in the algorithm within Appendix S2.</p></div><div id="sec-5" class="subsection"><h3>Simulation study</h3><p id="p-21">We conducted a series of simulations to test the accuracy of the DSD algorithm under a variety of plausible parameter spaces. For all scenarios, we fix <em>N</em><sub><em>1</em></sub> = 3000, and <em>K</em> = 2000 in the initial conditions, as the Ricker model is most reliably fit for populations fluctuating around their carrying capacity. As the dynamic observed in a Ricker population is driven primarily by the relationship of other parameters to <em>K</em> than by the absolute value of <em>K</em> itself, we held the starting value of <em>K</em> constant for all simulations. For each set of simulations, we held the variables not being varied at “base values” defined as: starting value of <em>r</em> = 2, change in <em>r</em> = ±25%, change in <em>K</em> = ±75%, 2% noise (<em>τ</em>; described below), with time series length of 20 years. We examined the effect of the size of initial <em>r</em> on algorithm performance by creating scenarios with different starting values of <em>r</em> = 0.5, 1, 1.5, 2. For each value of initial <em>r</em>, we modified the percent change in r at break points from the starting values (± no change, 10%, 25%, 50%, 75%) while holding all other parameters at base values. We then ran a set of simulations examining the percent change in K at break points from its starting value (± no change, 10%, 25%, 50%, 75%) while holding all other parameters (including <em>r</em>) at base values. This led to a total of 40 scenarios (four starting values of <em>r</em> with five percent changes in <em>r</em> and five percent changes in <em>K</em>).</p><p id="p-22">We further evaluated how the magnitude of stochasticity in the system (as measured by the error term <em>ε</em><sub><em>t</em></sub>) influenced algorithm performance. For generalizability of our simulation results, we simulated error as a percentage of the mean population size, rather than as absolute value (as described in the model above that we used for fitting the DSD). For each annual population size in the simulated dataset, a random value was selected from a normal curve of mean 0 and standard deviation of <em>τ</em> (where <em>τ</em> = 1%, 2%, 5%, 10%, 15%) and multiplied by the expected population size generated from the deterministic portion of the model.</p><p id="p-23">We ran these simulations with all noise (<em>τ</em>) levels across all percent change values for <em>r</em> and <em>K</em> (with other parameters held at base values) for a total of 50 additional scenarios (five percent noise values with five percent changes in <em>r</em> and five percent changes in <em>K</em>). Finally, we tested the impact of time series length by modifying the length of the simulated time series at five-year intervals over a range from 15 – 30 years (as the number of break point allows) while holding all other parameters constant, for four additional scenarios. We generated 250 simulated datasets for each of the 94 possible scenarios assuming breakpoint combinations with 0, 1, 2 and 3 breaks. Break point locations were randomly selected from within the set of possible time points. In total, we generated 93,572 data sets that we examined with our DSD algorithm. (Note that 94,000 simulations were run but simulations for higher numbers of break points in shorter time series occasionally failed to converge; results for such combinations are not presented).</p><p id="p-24">We evaluated the DSD algorithm’s performance for all test scenarios by examining its ability to identify the true break points within the set of the best fitting break point combinations (i.e. the top ranked break point combination and those break point combinations whose AICc values fell within two units of the top ranked). We also examined the performance of the break-point weighting tool by calculating the average weightings of all true and erroneous break points identified in the top performing model(s) across all runs of a given scenario.</p></div></div><div class="section" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Simulation Study</h3><p id="p-25">The scenario with the correct number of breaks and their locations was detected within the top performing break point combination sets with &gt;70% accuracy under nearly all parameterizations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). Accuracy was generally lowest in time series with three break points but above 70% for most scenarios. These results remained roughly consistent regardless of the value of the variance (σ<sup>2</sup>) determining the annual amount of environmental/sampling noise (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1 A</a>). Results were similar across all r values tested but performance of the DSD declined slightly when initial r was large (&gt;2.0; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1 B</a>). The DSD algorithm had the highest accuracy with larger shifts in K (≥25%; <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig 1. C</a>) and relatively smaller changes to <em>r</em> (≤25%; <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1 D</a>). This result is somewhat counter-intuitive, as we would generally expect large shifts in all parameters to be more easily detected. However, because nonlinear models produce chaotic dynamics with high population growth rates (e.g., r &gt; 2.3 in the Ricker model), a large shift in parameters could potentially result in a situation where multiple break point fits would perform equally well. Finally, the accuracy of the DSD algorithm decreased as scenario length increased, likely because of the factorial increase in potential break point combinations with additional data in the time series (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1 E</a>). Accuracy was also lower in cases where the number of break points was high relative to the time series length (e.g., 20 years and three breaks).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance of the Dynamic Shift Detector (DSD) algorithm under varying parameter values. Proportion of simulation results in which the true break scenario was detected within the top break point combinations as identified by the DSD implemented with an underlying Ricker model with varied A) noise (in the form of normally distributed error), B) starting values of the r parameter, C) percent changes in the K parameter, D) percent changes in r, and E) simulated time series length. Sets of 0, 1, 2 and 3 break points were randomly generated from within the set of possible values, and 250 datasets were simulated for each scenario. In each panel, other variables (that were not being varied) were held constant at their base values (i.e., noise=2%; starting value of r = 2; change in r = &#xB1;25%; change in K = &#xB1;75%; time series length = 20 years). Trends within a set of scenarios (grey lines) are illustrated with a third-order GAM smoothing line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1631030435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Performance of the Dynamic Shift Detector (DSD) algorithm under varying parameter values.&lt;/span&gt; Proportion of simulation results in which the true break scenario was detected within the top break point combinations as identified by the DSD implemented with an underlying Ricker model with varied A) noise (in the form of normally distributed error), B) starting values of the &lt;em&gt;r&lt;/em&gt; parameter, C) percent changes in the &lt;em&gt;K&lt;/em&gt; parameter, D) percent changes in &lt;em&gt;r&lt;/em&gt;, and E) simulated time series length. Sets of 0, 1, 2 and 3 break points were randomly generated from within the set of possible values, and 250 datasets were simulated for each scenario. In each panel, other variables (that were not being varied) were held constant at their base values (i.e., noise=2%; starting value of &lt;em&gt;r&lt;/em&gt; = 2; change in &lt;em&gt;r&lt;/em&gt; = &#xB1;25%; change in &lt;em&gt;K&lt;/em&gt; = &#xB1;75%; time series length = 20 years). Trends within a set of scenarios (grey lines) are illustrated with a third-order GAM smoothing line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F1.medium.gif" width="440" height="119"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F1.medium.gif" width="440" height="119"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <span class="caption-title">Performance of the Dynamic Shift Detector (DSD) algorithm under varying parameter values.</span><p id="p-26" class="first-child">Proportion of simulation results in which the true break scenario was detected within the top break point combinations as identified by the DSD implemented with an underlying Ricker model with varied A) noise (in the form of normally distributed error), B) starting values of the <em>r</em> parameter, C) percent changes in the <em>K</em> parameter, D) percent changes in <em>r</em>, and E) simulated time series length. Sets of 0, 1, 2 and 3 break points were randomly generated from within the set of possible values, and 250 datasets were simulated for each scenario. In each panel, other variables (that were not being varied) were held constant at their base values (i.e., noise=2%; starting value of <em>r</em> = 2; change in <em>r</em> = ±25%; change in <em>K</em> = ±75%; time series length = 20 years). Trends within a set of scenarios (grey lines) are illustrated with a third-order GAM smoothing line.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">The breakpoint weighting analysis revealed that in the vast majority of cases, the average weight of a true break exceeded a value of 0.8 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A-E</a>), whereas the weight of erroneous breaks averaged less than 0.2 in weight. The notable exception occurred when true breaks resulted from very small shifts in K (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2 C</a>). Thus, our results suggest the following decision rules to evaluate strength of evidence for a break occurring at a given time point: when a weight of &gt;0.8 is indicated for a break found by the DSD algorithm, we can reasonably conclude this is a true break, and likewise, a break with a weight of &lt;0.2 can reasonably assumed to be erroneous. Weight values intermediate to those two thresholds can be interpreted as a quantification of the strength of evidence that a break occurred.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average break weight of break points detected under varying parameterization conditions. Average weights of break points identified by the Dynamic Shift Detector algorithm reflecting true parameterization conditions (diamonds) or erroneous breaks suggested by the algorithm (triangles) under varied A) noise (in the form of normally distributed error), B) starting values of the r parameter, C) percent changes in the K parameter, D) percent changes in r, and E) simulated time series length. Sets of 0, 1, 2 and 3 break points were randomly generated from within the set of possible values, and 250 datasets were simulated for each scenario. In each panel, other variables (that were not being varied) were held constant at their base values (i.e., noise=2%; starting value of r = 2; change in r = &#xB1;25%; change in K = &#xB1;75%; time series length = 20 years). Trends within a set of scenarios (grey lines) are illustrated with a third-order GAM smoothing line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1631030435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Average break weight of break points detected under varying parameterization conditions.&lt;/span&gt; Average weights of break points identified by the Dynamic Shift Detector algorithm reflecting true parameterization conditions (diamonds) or erroneous breaks suggested by the algorithm (triangles) under varied A) noise (in the form of normally distributed error), B) starting values of the &lt;em&gt;r&lt;/em&gt; parameter, C) percent changes in the &lt;em&gt;K&lt;/em&gt; parameter, D) percent changes in &lt;em&gt;r&lt;/em&gt;, and E) simulated time series length. Sets of 0, 1, 2 and 3 break points were randomly generated from within the set of possible values, and 250 datasets were simulated for each scenario. In each panel, other variables (that were not being varied) were held constant at their base values (i.e., noise=2%; starting value of &lt;em&gt;r&lt;/em&gt; = 2; change in &lt;em&gt;r&lt;/em&gt; = &#xB1;25%; change in &lt;em&gt;K&lt;/em&gt; = &#xB1;75%; time series length = 20 years). Trends within a set of scenarios (grey lines) are illustrated with a third-order GAM smoothing line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F2.medium.gif" width="440" height="90"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F2.medium.gif" width="440" height="90"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <span class="caption-title">Average break weight of break points detected under varying parameterization conditions.</span><p id="p-28" class="first-child">Average weights of break points identified by the Dynamic Shift Detector algorithm reflecting true parameterization conditions (diamonds) or erroneous breaks suggested by the algorithm (triangles) under varied A) noise (in the form of normally distributed error), B) starting values of the <em>r</em> parameter, C) percent changes in the <em>K</em> parameter, D) percent changes in <em>r</em>, and E) simulated time series length. Sets of 0, 1, 2 and 3 break points were randomly generated from within the set of possible values, and 250 datasets were simulated for each scenario. In each panel, other variables (that were not being varied) were held constant at their base values (i.e., noise=2%; starting value of <em>r</em> = 2; change in <em>r</em> = ±25%; change in <em>K</em> = ±75%; time series length = 20 years). Trends within a set of scenarios (grey lines) are illustrated with a third-order GAM smoothing line.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Case study applications</h3><p id="p-29">We tested the performance of the DSD algorithm with two case studies using population time series data from field observations. Both case studies involve approximately two decades of observations of economically or culturally important insect species: one examines an invasion process and the other examines a population decline, both occurring over the same time period in recent history.</p><div id="sec-9" class="subsection"><h4>Multicolored Asian ladybeetles in southwestern Michigan</h4><p id="p-30">The 1994 invasion of multicolored Asian ladybeetles to southwestern Michigan, United States was documented in monitoring data collected on agriculturally-important Coccinellidae (ladybeetles) in landscapes dominated by field crops. Population density of ladybeetles is monitored in ten plant communities weekly over the growing season using yellow sticky card glue traps starting in 1989 at the Kellogg Biological Station at Michigan State University. We used data on the captures of adults at the site from 1994-2017, culled at day of year 222 (August 10) to minimize the effect of year-to-year variation in the sampling period. We then calculated the average number of adults captured per trap, across all traps deployed within a sampling year, and used this value in our analysis. Detailed sampling methodology is available in previous work [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>–<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>].</p><p id="p-31">Two break points, one occurring after 2000 and one occurring after 2005, were observed in the top performing break point combination (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3 A</a>, AICc=−18.02). However, the DSD algorithm indicated that two additional break point combinations, a single break after 2000 (AICc=−17.46), and a no break series (AICc=−17.64), had equivalent performance. Break weight analysis suggested a weight of 0.56 for the 2000 break, and a weight of 0.29 for the break after 2005. As these weights fall into a range intermediate to the 0.2 and 0.8 decision rules, we conclude that there is moderately strong evidence of a shift in dynamic rule after 2000, and moderate-weak evidence for a shift after 2005. The shift in 2000 is characterized by substantial increases in the values of <em>K</em> and <em>r</em>, with approximate increases of 75% and 40% over their initial estimates, respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The shift in 2005 is characterized by a return to parameter estimates that were nearly identical to those observed at the beginning of the time series (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3 B</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/985367/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/985367/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/985367" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-32" class="first-child">Ricker model parameter values for each phase between break points resulting from fitting population data of 1) multicolored Asian ladybeetles from Michigan, USA (1994-2017), and 2) the area occupied by monarch butterflies in their winter habitat in central Mexico (1995-2017). The parameter <em>r</em> is the per capita yearly intrinsic rate of increase and <em>K</em> is the carrying capacity (e.g., average number of adult ladybeetles captured per trap annually and hectares occupied by monarchs annually).</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dynamic Shift Detector breaks and Ricker model fits for an invasive species. Population data documenting the invasion of multicolored Asian ladybeetle in Michigan, USA from 1994-2017. A) Time series data showing the average number of adults captured, per trap, per year. Vertical blue lines indicate years in which dynamic shifts occurred, as estimated by the Dynamic Shift Detector algorithm. B) Ricker fits of time series data segments. Ladybeetle art by M. Broussard, used under a CC-BY 3.0 license." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1631030435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Dynamic Shift Detector breaks and Ricker model fits for an invasive species.&lt;/span&gt; Population data documenting the invasion of multicolored Asian ladybeetle in Michigan, USA from 1994-2017. A) Time series data showing the average number of adults captured, per trap, per year. Vertical blue lines indicate years in which dynamic shifts occurred, as estimated by the Dynamic Shift Detector algorithm. B) Ricker fits of time series data segments. Ladybeetle art by M. Broussard, used under a CC-BY 3.0 license.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F3.medium.gif" width="362" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F3.medium.gif" width="362" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <span class="caption-title">Dynamic Shift Detector breaks and Ricker model fits for an invasive species.</span><p id="p-33" class="first-child">Population data documenting the invasion of multicolored Asian ladybeetle in Michigan, USA from 1994-2017. A) Time series data showing the average number of adults captured, per trap, per year. Vertical blue lines indicate years in which dynamic shifts occurred, as estimated by the Dynamic Shift Detector algorithm. B) Ricker fits of time series data segments. Ladybeetle art by M. Broussard, used under a CC-BY 3.0 license.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Our results can be explained in the context of the known ecology of this ladybeetle. Dynamics of the ladybeetle invasion appear to be closely coupled with prey availability [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>,<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>–<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>], which, in turn, is driven by documented pest management practices (neonicotinoid insecticide use; [<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">42</a>]) leading to a relatively simple pulsed change. The first shift in the dynamics of the Asian ladybeetle, after 2000, corresponds to the well documented arrival and establishment of soybean aphid to North America, a preferred prey item from the ladybeetle’s native range [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>,<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. The invasion of this aphid dramatically increased resources available to the ladybeetle in habitats where the beetles were already well-established [<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>], supporting both a higher carrying capacity and a greater intrinsic growth rate. The second shift, after 2005, was less strongly supported, but coincides with the introduction and uptake of a management strategy for aphids that incompletely controlled the prey item. Landscape-scale use of neonicotinoid insecticides decreased prey numbers, particularly during the spring when aphids colonize new hosts, which could limit early season reproduction of ladybeetles [<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">42</a>]. Indeed, in this case, we would expect a weaker shift in dynamics as the prey item is incompletely controlled, and control tactics were not uniformly adopted across the prey’s range simultaneously.</p></div><div id="sec-10" class="subsection"><h4>Monarch butterflies in Mexican overwintering grounds</h4><p id="p-35">The eastern population of the North American monarch butterfly (<em>Danaus plexippus</em>) is migratory, with the majority of individuals overwintering in large aggregations in Oyamel fir forests within the transvolcanic mountains in the central region of Mexico [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. Monarchs are highly dispersed over their breeding season, occupying landscapes throughout the agricultural belt in central and eastern United States and southern Canada [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>]. As such, estimates of the overwintering population size can provide a convenient and inclusive annual metric of the size of the eastern migratory population [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. This population of monarchs has been in dramatic decline in recent decades, although the degree and cause of this decline is hotly debated [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>]. We used data on the total area occupied by monarchs from 1995-2017 (based on early winter surveys conducted in December) compiled by the World Wildlife Fund Mexico (available at MonarchWatch; Lovett 2017).</p><p id="p-36">The DSD algorithm estimated that the best break point combination fit for the monarch overwintering data was a single break after 2003 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>; AICc=120.18). However, the algorithm indicated that two additional break point combinations, a single break after 2006 (AICc=121.87) and a two-break combination of 2003 and 2008 (AICc=-121.86), had equivalent performance. The weight analysis computed weights of 0.49, 0.14, and 0.26, for 2003, 2006, and 2008 respectively, suggesting that the break at 2006 is erroneous and providing intermediate support for the 2003 and 2008 breaks. As with our ladybeetle case study, the strength of evidence was strongest for the first break, and weaker for the second break. The shift corresponds with a &gt;50% reduction in K in 2003, and, if the secondary break is taken at 2008, a further reduction of K nearing 50% again at that point (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4 B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dynamic Shift Detector breaks and Ricker model fits for a species of conservation concern. Population data documenting the area occupied by monarch butterflies in their winter habitat in central Mexico from 1995-2017. A) Time series data showing the total area occupied by overwintering monarchs each year in December. Vertical blue lines indicate years in which dynamic shifts occurred, as estimated by the Dynamic Shift Detector algorithm. B) Ricker fits of time series data segments. Butterfly art by D. Descouens and T.M. Seesey, used under a CC-BY 3.0 license." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1631030435" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Dynamic Shift Detector breaks and Ricker model fits for a species of conservation concern.&lt;/span&gt; Population data documenting the area occupied by monarch butterflies in their winter habitat in central Mexico from 1995-2017. A) Time series data showing the total area occupied by overwintering monarchs each year in December. Vertical blue lines indicate years in which dynamic shifts occurred, as estimated by the Dynamic Shift Detector algorithm. B) Ricker fits of time series data segments. Butterfly art by D. Descouens and T.M. Seesey, used under a CC-BY 3.0 license.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F4.medium.gif" width="375" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F4.medium.gif" width="375" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/06/522631/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <span class="caption-title">Dynamic Shift Detector breaks and Ricker model fits for a species of conservation concern.</span><p id="p-37" class="first-child">Population data documenting the area occupied by monarch butterflies in their winter habitat in central Mexico from 1995-2017. A) Time series data showing the total area occupied by overwintering monarchs each year in December. Vertical blue lines indicate years in which dynamic shifts occurred, as estimated by the Dynamic Shift Detector algorithm. B) Ricker fits of time series data segments. Butterfly art by D. Descouens and T.M. Seesey, used under a CC-BY 3.0 license.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The patterns we observe are consistent with a leading hypothesis to explain monarch population decline. Loss of milkweed hostplants due to changing agricultural practices on Midwestern breeding grounds [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>] is hypothesized to be a major driver in the dynamics of this species. Changing herbicide practices in central North America have largely eliminated milkweed hostplants from agricultural field crops, with fairly consistent, low levels of milkweed on the landscape starting from about 2003-2005 [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>]. Although glyphosate tolerant soybeans and maize were introduced to the US market in 1996 and 1998 respectively [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>], actual glyphosate use lagged behind, with dramatic increases in use of the pesticide in 1998-2003 in soybean, and 2007-2008 in maize [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>].</p><p id="p-39">However, additional drivers likely play a role in monarch processes given the uncertainty in our results. Abiotic drivers of monarch population dynamics are complex and can interact at local, regional, and continental scales [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>]. Other studies have implicated climate [<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>], extreme weather events [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>], changing habitat availability on wintering grounds [<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>], and mortality during the fall migration [<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>,<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>] as possible factors influencing monarch population dynamics. With many super-imposed drivers, monarch dynamics are likely driven by both press and pulsed processes, making the detection of discrete break points associated with dynamic shifts complicated.</p></div></div></div><div class="section" id="sec-11"><h2 class="">Discussion</h2><p id="p-40">The DSD algorithm provides a novel tool for evaluating shifts in parameter values that govern density-dependent populations, such as carrying capacity and population growth rates. As illustrated with our simulations and case studies, the DSD algorithm can not only identify and quantify parameter changes but also assess uncertainty in potential break points and help detect time frames where additional research should be focused. We recommend that the model selection approach be used to identify a list of potential break points and break point combinations, and the weighting tool be used to evaluate the strength of evidence for each potential break, providing a clear direction to focus downstream research on changing dynamic processes. Characterizing dynamic transitions in population time series data has been hindered by a lack of a common, accessible, and empirical approaches [<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>]. Most assessments of break points in ecological research are <em>ad hoc</em> [<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>–<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>], introducing the potential for bias in break point selection, particularly in cases where nonlinear dynamics are likely to occur. Our DSD algorithm directly addresses this gap in tools, using an information-theoretic, model selection approach and a framework that can incorporate a variety nonlinear population models to assess ecological processes.</p><p id="p-41">The core novelty of our tool lies in the model selection procedure used within the DSD algorithm, which allows for greater accuracy than common break point detection models [<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">1</a>]. The DSD additionally allows users to assess the confidence in a given break point, as well as providing a measure of which break points are likely to appear together, if multiple dynamic shifts have occurred. Information-theoretic approaches such as model selection using AIC, may be prone to over-fitting, particularly when data are limited [<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">35</a>]. Thus, we used AICc, the Akaike Information Criterion corrected for small sample sizes, as the selection criterion within the DSD algorithm. AICc is generally recommended in place of AIC in situations where small samples sizes are being examined as it incorporates a penalty term that is inversely related to the number of observations. As sample size increases, the penalty for model complexity is reduced and AICc approaches AIC [<a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">35</a>]. Break point detection methods must incorporate a compromise between sensitivity and penalty for over-parameterization [<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">1</a>]. We examined the performance of both AICc (here) and AIC (in Appendix S1) and found that using AICc performed best for our simulated data. The DSD algorithm aides in the interpretation of break points by incorporating a metric based on Akaike weights, which allows users to assess the relative ‘strength’ of multiple breaks. Where many tools aim to identify points at which parameter changes occur, the DSD algorithm provides a measure of the confidence in each break, as well as a measure of how differing break sets perform in explaining variation in the data relative to each other.</p><p id="p-42">The performance of the DSD algorithm was fairly stable among the break point simulations we tested. We found that the amount of environmental/sampling noise (ranging from 1-15% of the population size) had little effect on algorithm performance (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1 A</a>). Other input conditions had relatively greater impacts on the performance of the DSD algorithm, depending on which parameter was changed and by how much. Large shifts in a population’s carry capacity were more easily detected than smaller shifts (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1 C</a>). However, large changes in population growth rate were harder to detect, but this effect was most pronounced when simulated data contained multiple breaks (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figs. 1</a>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">2D</a>). Although larger shifts in regression parameters would, intuitively, lead to a higher likelihood of detection, large shifts in growth rate are also more likely to induce variations in transient dynamics in the years immediately following the shift, potentially making the timing of shifts more difficult to pinpoint. Similarly, longer time series yielded results that were more error prone (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1</a>–<a id="xref-fig-2-4" class="xref-fig" href="#F2">2E</a>). This is likely because there were simply more possible break-point combinations for the algorithm to select from and because the penalty for increasing parameterization (i.e. AICc) decreases as sample sizes grow (leading to increasing likelihood of identifying extra, erroneous breaks).</p><p id="p-43">In our case studies, we found interpretation of the ladybeetle example was straightforward (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3</a>). Our top break point combination and the equivalently-performing set did not contain contradictory information: each candidate set was simply a subset of breakpoints from the most complex set, and only two break points were found. Both of these break points were associated with moderate or greater weights, although the values of these break weights were in the intermediate range (i.e., between 0.2 and 0.8), suggesting breaks in natural systems may not be as well behaved as those in simulated data. The monarch butterfly case study results were slightly more ambiguous, as the model selection tool identified a break that the weighting tool suggested was erroneous (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4</a>). Weights of the two most strongly-supported breaks were numerically similar to those of the ladybeetle case study, and are also interpretable with knowledge of the study system. However, the model selection results suggest additional, superimposed processes may be affecting monarch population dynamics, creating a noisier signal.</p><p id="p-44">The DSD algorithm is readily adaptable to other population models and, indeed, potentially to other nonlinear processes. Density-dependent population growth has the potential to mask transition points because of its inherent nonlinear structure. For example, transient dynamics occurring immediately after a temporary disturbance can result in a change in population size, but not necessarily in the rules governing dynamics. We used the Ricker model as the core population model within the algorithm because it had a number of useful characteristics, namely its simple parameterization and realistic behavior [<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>]. However, simple density-dependent population models including Ricker, Beverton-Holt, and logistic models have similar performance in predicting outbreaks of insects driven by food limitation [<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. Indeed, in early iterations examining the DSD, we fit both logistic and Ricker models to real data and found that the two models ranked break point combinations nearly identically, even while the Ricker model generally provided a better fit for the data [<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">42</a>]. Thus, we expect the DSD algorithm would have similar performance across models with similar structures, but performance may vary with other model structures, particularly those that incorporate additional terms.</p><p id="p-45">We recommend users carefully consider the strengths and limitations of the DSD algorithm in the context of their own data. For example, if changes to parameter values occur frequently (e.g., less than 3-4 years or time periods), the frequency of shifts would violate the constraints placed on the DSD to prevent overfitting. We also observed that the likelihood of identifying erroneous break points increased as time series length increased. Thus, in cases where a long time series exists, but a particular time period is of interest, the DSD algorithm could be used on the time period of interest alone to minimize the likelihood of distracting or erroneous results. The DSD algorithm functions as a method for identifying break points within time series data and quantifying the strength of evidence for each potential break point. When interpreted in the context of species biology, the DSD algorithm has the potential to aid management decisions, identify critical drivers of change in species’ dynamics, and help determine where best to focus additional research efforts.</p></div><div class="section" id="sec-13"><h2 class="">Supporting Information</h2><p id="p-47">S1– Additional Simulations using AIC for break point selection</p><p id="p-48">S2– Description of functions used in the Dynamic Shift Detector (DSD) algorithm</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-46">The conception of an earlier version of this algorithm came about out of conversations with Wopke van der Werf and Douglas Landis, and the algorithm has incorporated feedback from colleagues from the US-Long Term Ecological Research network throughout its development.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-6">Manuscript text has been updated following peer review; title and algorithm name have been changed</p></li><li class="fn-dataset fn-group-external-links" id="fn-2"><p id="p-7">
<a href="https://github.com/cbahlai/dynamic_shift_detector">https://github.com/cbahlai/dynamic_shift_detector</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.1" data-doi="10.1016/j.tree.2008.07.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carstensen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hernández-García</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duarte</span>  <span class="cit-name-given-names">CM.</span></span> <span class="cit-article-title">Ecological thresholds and regime shifts: approaches to identification</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">24</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tree.2008.07.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DAndersen%26rft.auinit1%253DT.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D57%26rft.atitle%253DEcological%2Bthresholds%2Band%2Bregime%2Bshifts%253A%2Bapproaches%2Bto%2Bidentification.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2008.07.014%26rft_id%253Dinfo%253Apmid%252F18952317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2008.07.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18952317&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262664700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.2" data-doi="10.1890/ES11-00216.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bestelmeyer</span>  <span class="cit-name-given-names">BT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellison</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">WR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gorman</span>  <span class="cit-name-given-names">KB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holbrook</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laney</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Analysis of abrupt transitions in ecological systems</span>. <abbr class="cit-jnl-abbrev">Ecosphere</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">2</span>: <span class="cit-fpage">art129</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1890/es11-00216.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcosphere%26rft.volume%253D2%26rft.spage%253D129art%26rft_id%253Dinfo%253Adoi%252F10.1890%252FES11-00216.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/ES11-00216.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-522631v2.3" data-doi="10.1038/35098000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Folke</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Catastrophic shifts in ecosystems</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">2001</span>;<span class="cit-vol">413</span>: <span class="cit-fpage">591</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/35098000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DScheffer%26rft.auinit1%253DM.%26rft.volume%253D413%26rft.issue%253D6856%26rft.spage%253D591%26rft.epage%253D596%26rft.atitle%253DCatastrophic%2Bshifts%2Bin%2Becosystems.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35098000%26rft_id%253Dinfo%253Apmid%252F11595939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35098000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11595939&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171485700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">4.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-522631v2.4" data-doi="10.1016/j.tree.2003.09.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scheffer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">SR.</span></span> <span class="cit-article-title">Catastrophic regime shifts in ecosystems: linking theory to observation</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">18</span>: <span class="cit-fpage">648</span>–<span class="cit-lpage">656</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tree.2003.09.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D18%26rft.spage%253D648%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2003.09.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2003.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000187223100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.5" data-doi="10.1038/269471a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">RM.</span></span> <span class="cit-article-title">Thresholds and breakpoints in ecosystems with a multiplicity of stable states</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">1977</span>;<span class="cit-vol">269</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/269471a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D269%26rft.spage%253D471%26rft_id%253Dinfo%253Adoi%252F10.1038%252F269471a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/269471a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.6" data-doi="10.1002/joc.924"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ducré-Robitaille</span>  <span class="cit-name-given-names">J-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boulet</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Comparison of techniques for detection of discontinuities in temperature series: DETECTING DISCONTINUITIES IN TEMPERATURE SERIES</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">23</span>: <span class="cit-fpage">1087</span>–<span class="cit-lpage">1101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/joc.924</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D23%26rft.spage%253D1087%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.924&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000184388400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.7" data-doi="10.1029/2004GL019448"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rodionov</span>  <span class="cit-name-given-names">SN.</span></span> <span class="cit-article-title">A sequential algorithm for testing climate regime shifts: ALGORITHM FOR TESTING REGIME SHIFTS</span>. <abbr class="cit-jnl-abbrev">Geophysical Research Letters</abbr>. <span class="cit-pub-date">2004</span>;<span class="cit-vol">31</span>: n/a-n/a.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1029/2004GL019448</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeophysical%2BResearch%2BLetters%26rft.volume%253D31%26rft.spage%253DL09204%26rft_id%253Dinfo%253Adoi%252F10.1029%252F2004GL019448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1029/2004GL019448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.8" data-doi="10.1098/rstb.2013.0272"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beaugrand</span>  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conversi</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chiba</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fonda-Umani</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">C.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Synchronous marine pelagic regime shifts in the Northern Hemisphere</span>. <abbr class="cit-jnl-abbrev">Philosophical Transactions of the Royal Society B: Biological Sciences</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">370</span>: <span class="cit-fpage">20130272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1098/rstb.2013.0272</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2013.0272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2013.0272&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.9" data-doi="10.1016/j.jedc.2005.10.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zampolli</span>  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Optimal monetary policy in a regime-switching economy: The response to abrupt shifts in exchange rate dynamics</span>. <abbr class="cit-jnl-abbrev">Journal of Economic Dynamics and Control</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">30</span>: <span class="cit-fpage">1527</span>–<span class="cit-lpage">1567</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jedc.2005.10.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEconomic%2BDynamics%2Band%2BControl%26rft.volume%253D30%26rft.spage%253D1527%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jedc.2005.10.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jedc.2005.10.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.10" data-doi="10.1007/s10021-001-0101-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holling</span>  <span class="cit-name-given-names">CS.</span></span> <span class="cit-article-title">Understanding the Complexity of Economic, Ecological, and Social Systems</span>. <abbr class="cit-jnl-abbrev">Ecosystems</abbr>. <span class="cit-pub-date">2001</span>;<span class="cit-vol">4</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">405</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s10021-001-0101-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcosystems%26rft.volume%253D4%26rft.spage%253D390%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10021-001-0101-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10021-001-0101-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.11" data-doi="10.1126/science.1062226"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bjørnstad</span>  <span class="cit-name-given-names">ON</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grenfell</span>  <span class="cit-name-given-names">BT.</span></span> <span class="cit-article-title">Noisy Clockwork: Time Series Analysis of Population Fluctuations in Animals</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2001</span>;<span class="cit-vol">293</span>: <span class="cit-fpage">638</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1062226</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBjornstad%26rft.auinit1%253DO.%2BN.%26rft.volume%253D293%26rft.issue%253D5530%26rft.spage%253D638%26rft.epage%253D643%26rft.atitle%253DNoisy%2BClockwork%253A%2BTime%2BSeries%2BAnalysis%2Bof%2BPopulation%2BFluctuations%2Bin%2BAnimals%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1062226%26rft_id%253Dinfo%253Apmid%252F11474099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTMvNTUzMC82MzgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8wNi81MjI2MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.12" data-doi="10.1016/S0079-6611(00)00033-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hare</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mantua</span>  <span class="cit-name-given-names">NJ.</span></span> <span class="cit-article-title">Empirical evidence for North Pacific regime shifts in 1977 and 1989</span>. <abbr class="cit-jnl-abbrev">Progress in Oceanography</abbr>. <span class="cit-pub-date">2000</span>;<span class="cit-vol">47</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/S0079-6611(00)00033-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2BOceanography%26rft.volume%253D47%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0079-6611%252800%252900033-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0079-6611(00)00033-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.13" data-doi="10.1111/j.1461-0248.2007.01131.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>  <span class="cit-name-given-names">WA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cole</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitchell</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pace</span>  <span class="cit-name-given-names">ML.</span></span> <span class="cit-article-title">Leading indicators of trophic cascades</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">11</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">138</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1461-0248.2007.01131.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DCarpenter%26rft.auinit1%253DS.%2BR.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D128%26rft.epage%253D138%26rft.atitle%253DLeading%2Bindicators%2Bof%2Btrophic%2Bcascades.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2007.01131.x%26rft_id%253Dinfo%253Apmid%252F18021242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2007.01131.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18021242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252206800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.14" data-doi="10.1017/S0954102003001202"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weimerskirch</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Inchausti</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guinet</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barbraud</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">Trends in bird and seal populations as indicators of a system shift in the Southern Ocean</span>. <abbr class="cit-jnl-abbrev">Antarctic Science</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">15</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntarctic%2BScience%26rft.volume%253D15%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0954102003001202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0954102003001202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000183658000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.15" data-doi="10.1086/508670"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berryman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lima</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Deciphering the effects of climate on animal populations: diagnostic analysis provides new interpretation of Soay sheep dynamics</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">168</span>: <span class="cit-fpage">784</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DBerryman%26rft.auinit1%253DA.%26rft.volume%253D168%26rft.issue%253D6%26rft.spage%253D784%26rft.epage%253D795%26rft.atitle%253DDeciphering%2Bthe%2Beffects%2Bof%2Bclimate%2Bon%2Banimal%2Bpopulations%253A%2Bdiagnostic%2Banalysis%2Bprovides%2Bnew%2Binterpretation%2Bof%2BSoay%2Bsheep%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F508670%26rft_id%253Dinfo%253Apmid%252F17109320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/508670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17109320&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242126900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-522631v2.16" data-doi="10.1890/02-0472"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Toms</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lesperance</span>  <span class="cit-name-given-names">ML.</span></span> <span class="cit-article-title">Piecewise regression: a tool for identifying ecological thresholds</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">84</span>: <span class="cit-fpage">2034</span>–<span class="cit-lpage">2041</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1890/02-0472</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D84%26rft.spage%253D2034%26rft_id%253Dinfo%253Adoi%252F10.1890%252F02-0472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/02-0472&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000185073100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.17" data-doi="10.1525/bio.2012.62.4.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hobbie</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fahey</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">GJA</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Past, present, and future roles of long-term experiments in the LTER Network</span>. <abbr class="cit-jnl-abbrev">Bioscience</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">62</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">389</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1029/2008gb003336</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioscience%26rft_id%253Dinfo%253Adoi%252F10.1525%252Fbio.2012.62.4.9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1525/bio.2012.62.4.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000302927600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.18" data-doi="10.1214/ss/1028905933"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">JV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H-G.</span></span> <span class="cit-article-title">Statistical Methods for DNA Sequence Segmentation</span>. <abbr class="cit-jnl-abbrev">Statistical Science</abbr>. <span class="cit-pub-date">1998</span>;<span class="cit-vol">13</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistical%2BScience%26rft.volume%253D13%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1214%252Fss%252F1028905933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/ss/1028905933&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000074917200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-522631v2.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeileis</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leisch</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hornik</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kleiber</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">strucchange</span>. <abbr class="cit-jnl-abbrev">An R package for testing for structural change in linear regression models</abbr>. <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-522631v2.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Killick</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckley</span>  <span class="cit-name-given-names">I.</span></span> <span class="cit-article-title">changepoint: An R package for changepoint analysis</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">58</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D58%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Priyadarshana</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sofronov</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Multiple break-points detection in array CGH data via the cross-entropy method</span>. <abbr class="cit-jnl-abbrev">IEEE/ACM Transactions on Computational Biology and Bioinformatics (TCBB)</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">12</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%252FACM%2BTransactions%2Bon%2BComputational%2BBiology%2Band%2BBioinformatics%2B%2528TCBB%2529%26rft.volume%253D12%26rft.spage%253D487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.22" data-doi="10.1175/1520-0442(2003)016&lt;3905:MRSIUS&gt;2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mauget</span>  <span class="cit-name-given-names">SA.</span></span> <span class="cit-article-title">Multidecadal Regime Shifts in U.S. Streamflow, Precipitation, and Temperature at the End of the Twentieth Century</span>. <abbr class="cit-jnl-abbrev">Journal of Climate</abbr>. <span class="cit-pub-date">2003</span>;<span class="cit-vol">16</span>: <span class="cit-fpage">3905</span>–<span class="cit-lpage">3916</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1175/1520-0442(2003)016&lt;3905:MRSIUS&gt;2.0.CO;2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClimate%26rft.volume%253D16%26rft.spage%253D3905%26rft_id%253Dinfo%253Adoi%252F10.1175%252F1520-0442%25282003%2529016%253C3905%253AMRSIUS%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/1520-0442(2003)016&lt;3905:MRSIUS&gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.23" data-doi="10.1098/rspb.2004.2978"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jenouvrier</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weimerskirch</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barbraud</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">Y-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cazelles</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Evidence of a shift in the cyclicity of Antarctic seabird dynamics linked to climate</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">272</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">895</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1098/rspb.2004.2978</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DJenouvrier%26rft.auinit1%253DS.%26rft.volume%253D272%26rft.issue%253D1566%26rft.spage%253D887%26rft.epage%253D895%26rft.atitle%253DEvidence%2Bof%2Ba%2Bshift%2Bin%2Bthe%2Bcyclicity%2Bof%2BAntarctic%2Bseabird%2Bdynamics%2Blinked%2Bto%2Bclimate%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2004.2978%26rft_id%253Dinfo%253Apmid%252F16024342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2004.2978&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16024342&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.24" data-doi="10.1007/s00442-008-0993-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cazelles</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chavez</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berteaux</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ménard</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vik</span>  <span class="cit-name-given-names">JO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenouvrier</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Wavelet analysis of ecological time series</span>. <abbr class="cit-jnl-abbrev">Oecologia</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">156</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s00442-008-0993-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOecologia%26rft.stitle%253DOecologia%26rft.aulast%253DCazelles%26rft.auinit1%253DB.%26rft.volume%253D156%26rft.issue%253D2%26rft.spage%253D287%26rft.epage%253D304%26rft.atitle%253DWavelet%2Banalysis%2Bof%2Becological%2Btime%2Bseries.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00442-008-0993-2%26rft_id%253Dinfo%253Apmid%252F18322705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00442-008-0993-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18322705&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255954100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.25" data-doi="10.1098/rspb.2012.2085"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carl</span>  <span class="cit-name-given-names">Boettiger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Alan</span> <span class="cit-name-surname">Hastings</span></span>. <span class="cit-article-title">Early warning signals and the prosecutor’s fallacy</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">279</span>: <span class="cit-fpage">4734</span>–<span class="cit-lpage">4739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1098/rspb.2012.2085</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2012.2085%26rft_id%253Dinfo%253Apmid%252F23055060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2012.2085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23055060&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-522631v2.26"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"> <span class="cit-name-given-names">ML</span> <span class="cit-name-surname">Taper</span></span> and </li><li><span class="cit-ed"> <span class="cit-name-given-names">SR</span> <span class="cit-name-surname">Lele</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Royall</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-chapter-title">The Likelihood Paradigm for Statistical Evidence</span>. in <span class="cit-ed"> <span class="cit-name-given-names">ML</span> <span class="cit-name-surname">Taper</span></span> and <span class="cit-ed"> <span class="cit-name-given-names">SR</span> <span class="cit-name-surname">Lele</span></span>, <span class="cit-edition">eds</span>. <span class="cit-source">The nature of scientific evidence</span>. <span class="cit-publ-name">University of Chicago Press</span>, <span class="cit-publ-loc">Chicago</span>; <span class="cit-pub-date">2004</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-522631v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turchin</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-chapter-title">Complex population dynamics: a theoretical/empirical synthesis</span>. <span class="cit-publ-name">Princeton University Press</span>; <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.28" data-doi="10.1038/261459a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">RM.</span></span> <span class="cit-article-title">Simple mathematical models with very complicated dynamics</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>. <span class="cit-pub-date">1976</span>;<span class="cit-vol">261</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/261459a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMay%26rft.auinit1%253DR.%2BM.%26rft.volume%253D261%26rft.issue%253D5560%26rft.spage%253D459%26rft.epage%253D467%26rft.atitle%253DSimple%2Bmathematical%2Bmodels%2Bwith%2Bvery%2Bcomplicated%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F261459a0%26rft_id%253Dinfo%253Apmid%252F934280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/261459a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=934280&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976BT72500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">29.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-522631v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barraquand</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Louca</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">KC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cobbold</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cordoleani</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeAngelis</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Moving forward in circles: challenges and opportunities in modelling population cycles</span>. <abbr class="cit-jnl-abbrev">Ecology letters</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">20</span>: <span class="cit-fpage">1074</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2Bletters%26rft.volume%253D20%26rft.spage%253D1074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.30" data-doi="10.1111/ele.13085"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boettiger</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">From noise to knowledge: how randomness generates novel phenomena and reveals information</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>. <span class="cit-pub-date">2018</span>;<span class="cit-vol">21</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1267</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/ele.13085</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D21%26rft.spage%253D1255%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.13085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.13085&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.31" data-doi="10.1142/S021812741650067X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gadrich</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katriel</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">A Mechanistic Stochastic Ricker Model: Analytical and Numerical Investigations</span>. <abbr class="cit-jnl-abbrev">Int J Bifurcation Chaos</abbr>. <span class="cit-pub-date">2016</span>;<span class="cit-vol">26</span>: <span class="cit-fpage">1650067</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1142/S021812741650067X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BBifurcation%2BChaos%26rft.volume%253D26%26rft.spage%253D1650067%26rft_id%253Dinfo%253Adoi%252F10.1142%252FS021812741650067X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/S021812741650067X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.32" data-doi="10.1139/f54-039"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ricker</span>  <span class="cit-name-given-names">WE.</span></span> <span class="cit-article-title">Stock and Recruitment</span>. <abbr class="cit-jnl-abbrev">J Fish Res Bd Can</abbr>. <span class="cit-pub-date">1954</span>;<span class="cit-vol">11</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">623</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1139/f54-039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BFish%2BRes%2BBd%2BCan%26rft.volume%253D11%26rft.spage%253D559%26rft_id%253Dinfo%253Adoi%252F10.1139%252Ff54-039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/f54-039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.33" data-doi="10.1890/0012-9658(2006)87[1445:SOEFDD]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brook</span>  <span class="cit-name-given-names">BW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">CJ.</span></span> <span class="cit-article-title">Strength of evidence for density dependence in abundance time series of 1198 species</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">87</span>: <span class="cit-fpage">1445</span>–<span class="cit-lpage">1451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DBrook%26rft.auinit1%253DB.%2BW.%26rft.volume%253D87%26rft.issue%253D6%26rft.spage%253D1445%26rft.epage%253D1451%26rft.atitle%253DStrength%2Bof%2Bevidence%2Bfor%2Bdensity%2Bdependence%2Bin%2Babundance%2Btime%2Bseries%2Bof%2B1198%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9658%25282006%252987%255B1445%253ASOEFDD%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F16869419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9658(2006)87[1445:SOEFDD]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16869419&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238658400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.34" data-doi="10.1111/manc.12017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sakkas</span>  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Inference on Structural Breaks using Information Criteria</span>. <abbr class="cit-jnl-abbrev">The Manchester School</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">81</span>: <span class="cit-fpage">54</span>–<span class="cit-lpage">81</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/manc.12017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BManchester%2BSchool%26rft.volume%253D81%26rft.spage%253D54%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmanc.12017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/manc.12017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-book" id="cit-522631v2.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burnham</span>  <span class="cit-name-given-names">KP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">DR.</span></span> <span class="cit-chapter-title">Model selection and multimodal inference: a practical information-theoretic approach</span>. <span class="cit-edition">2nd ed</span>. <span class="cit-publ-loc">New York</span>: <span class="cit-publ-name">Springer Science + Business Media, LLC</span>; <span class="cit-pub-date">2002</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.36" data-doi="10.3758/BF03206482"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wagenmakers</span>  <span class="cit-name-given-names">E-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farrell</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">AIC model selection using Akaike weights</span>. <abbr class="cit-jnl-abbrev">Psychonomic Bulletin &amp; Review</abbr>. <span class="cit-pub-date">2004</span>;<span class="cit-vol">11</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3758/BF03206482</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychonomic%2BBulletin%2B%2526%2BReview%26rft.stitle%253DPsychon%2BBull%2BRev%26rft.aulast%253DWagenmakers%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D192%26rft.epage%253D196%26rft.atitle%253DAIC%2Bmodel%2Bselection%2Busing%2BAkaike%2Bweights.%26rft_id%253Dinfo%253Adoi%252F10.3758%252FBF03206482%26rft_id%253Dinfo%253Apmid%252F15117008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3758/BF03206482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15117008&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220674200028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-web" id="cit-522631v2.37"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span>. <span class="cit-article-title">R: A Language and Environment for Statistical Computing 3.3.3</span>. <span class="cit-source">R Foundation for Statistical Computing</span>. <span class="cit-pub-date">2017</span>. Available: <a href="http://www.R-project.org">http://www.R-project.org</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elzhov</span>  <span class="cit-name-given-names">TV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mullen</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spiess</span>  <span class="cit-name-given-names">A-N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">minpack. lm: R Interface to the Levenberg-Marquardt Nonlinear Least-Squares Algorithm Found in MINPACK</span>. <span class="cit-pub-date">2016</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.39" data-doi="10.5281/zenodo.3368486"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bahlai</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">cbahlai/dynamic_shift_detector: The Dynamic Shift Detector</span>. <abbr class="cit-jnl-abbrev">Zenodo</abbr>; <span class="cit-pub-date">2019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.5281/zenodo.3368486</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZenodo%26rft_id%253Dinfo%253Adoi%252F10.5281%252Fzenodo.3368486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5281/zenodo.3368486&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.40" data-doi="10.1371/journal.pone.0083407"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bahlai</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colunga-Garcia</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landis</span>  <span class="cit-name-given-names">DA.</span></span> <span class="cit-article-title">Long term functional dynamics of an aphidophagous coccinellid community are unchanged in response to repeated invasion</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">8</span>: <span class="cit-fpage">e83407</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0083407</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De83407%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0083407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0083407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.41" data-doi="10.1007/s10530-014-0772-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bahlai</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colunga-Garcia</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gage</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landis</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">The role of exotic ladybeetles in the decline of native ladybeetle populations: evidence from long-term monitoring</span>. <abbr class="cit-jnl-abbrev">Biol Invasions</abbr>. <span class="cit-pub-date">2015</span>;<span class="cit-vol">17</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1024</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s10530-014-0772-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BInvasions%26rft.volume%253D17%26rft.spage%253D1005%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10530-014-0772-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10530-014-0772-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.42" data-doi="10.1890/14-2022.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bahlai</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">vander Werf</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Neal</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hemerik</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landis</span>  <span class="cit-name-given-names">DA.</span></span> <span class="cit-article-title">Shifts in dynamic regime of an invasive lady beetle are linked to the invasion and insecticidal management of its prey</span>. <abbr class="cit-jnl-abbrev">Ecological Applications</abbr>. <span class="cit-pub-date">2015</span> [cited <span class="cit-date-in-citation">14 May 2015</span>].<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1890/14-2022.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.volume%253D25%26rft.spage%253D1807%26rft_id%253Dinfo%253Adoi%252F10.1890%252F14-2022.1%26rft_id%253Dinfo%253Apmid%252F26591447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/14-2022.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26591447&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bahlai</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sears</span>  <span class="cit-name-given-names">MK.</span></span> <span class="cit-article-title">Population dynamics of Harmonia axyridis and Aphis glycines in Niagara Peninsula soybean fields and vineyards</span>. <abbr class="cit-jnl-abbrev">Journal of the Entomological Society of Ontario</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">140</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BEntomological%2BSociety%2Bof%2BOntario%26rft.volume%253D140%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">44.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-522631v2.44" data-doi="10.1007/s10530-010-9736-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heimpel</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frelich</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landis</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hopper</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoelmer</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sezen</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">European buckthorn and Asian soybean aphid as components of an extensive invasional meltdown in North America</span>. <abbr class="cit-jnl-abbrev">Biological Invasions</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">12</span>: <span class="cit-fpage">2913</span>–<span class="cit-lpage">2931</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s10530-010-9736-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BInvasions%26rft.volume%253D12%26rft.spage%253D2913%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10530-010-9736-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10530-010-9736-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rhainds</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">HJS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kindlmann</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Voegtlin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castillo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rutledge</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Two-year oscillation cycle in abundance of soybean aphid in Indiana</span>. <abbr class="cit-jnl-abbrev">Agricultural and Forest Entomology</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">12</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgricultural%2Band%2BForest%2BEntomology%26rft.volume%253D12%26rft.spage%253D251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.46" data-doi="10.1093/aesa/97.2.204"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ragsdale</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Voegtlin</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Neil</span>  <span class="cit-name-given-names">RJ.</span></span> <span class="cit-article-title">Soybean aphid biology in North America</span>. <abbr class="cit-jnl-abbrev">Annals of the Entomological Society of America</abbr>. <span class="cit-pub-date">2004</span>;<span class="cit-vol">97</span>: <span class="cit-fpage">204</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BEntomological%2BSociety%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faesa%252F97.2.204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aesa/97.2.204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.47" data-doi="10.1093/aesa/97.2.209"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schenk-Hamlin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhan</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ragsdale</span>  <span class="cit-name-given-names">DW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heimpel</span>  <span class="cit-name-given-names">GE.</span></span> <span class="cit-article-title">The soybean aphid in China: a historical review</span>. <abbr class="cit-jnl-abbrev">Annals of the Entomological Society of America</abbr>. <span class="cit-pub-date">2004</span>;<span class="cit-vol">97</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BEntomological%2BSociety%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faesa%252F97.2.209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aesa/97.2.209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.48" data-doi="10.1139/z78-240"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Urquhart</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urquhart</span>  <span class="cit-name-given-names">NR.</span></span> <span class="cit-article-title">Autumnal migration routes of the eastern population of the monarch butterfly (Danaus p. plexippus L.; Danaidae; Lepidoptera) in North America to the overwintering site in the Neovolcanic Plateau of Mexico</span>. <abbr class="cit-jnl-abbrev">Can J Zool</abbr>. <span class="cit-pub-date">1978</span>;<span class="cit-vol">56</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1764</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1139/z78-240</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan%2BJ%2BZool%26rft.volume%253D56%26rft.spage%253D1759%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fz78-240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/z78-240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978FN22600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.49" data-doi="10.1111/gcb.13589"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Flockhart</span>  <span class="cit-name-given-names">DTT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brower</span>  <span class="cit-name-given-names">LP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramirez</span>  <span class="cit-name-given-names">MI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hobson</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wassenaar</span>  <span class="cit-name-given-names">LI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altizer</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Regional climate on the breeding grounds predicts variation in the natal origin of monarch butterflies overwintering in Mexico over 38 years</span>. <abbr class="cit-jnl-abbrev">Glob Change Biol</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">23</span>: <span class="cit-fpage">2565</span>–<span class="cit-lpage">2576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/gcb.13589</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChange%2BBiol%26rft.volume%253D23%26rft.spage%253D2565%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.13589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.13589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prysby</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oberhauser</span>  <span class="cit-name-given-names">KS.</span></span> <span class="cit-article-title">Temporal and geographic variation in monarch densities: citizen scientists document monarch population patterns</span>. <abbr class="cit-jnl-abbrev">The monarch butterfly: Biology and conservation</abbr>. <span class="cit-pub-date">2004</span>; <span class="cit-fpage">9</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.51" data-doi="10.1093/biosci/bix120"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sarkar</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">What Is Threatening Monarchs</span>? <abbr class="cit-jnl-abbrev">BioScience</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">67</span>: <span class="cit-fpage">1080</span>–<span class="cit-lpage">1080</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/biosci/bix120</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioScience%26rft.volume%253D67%26rft.spage%253D1080%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiosci%252Fbix120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biosci/bix120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">52.</span><div class="cit ref-cit ref-web no-rev-xref" id="cit-522631v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lovett</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Monarch Population Status</span>. <span class="cit-publ-name">Monarch Watch</span>; <span class="cit-pub-date">2017</span>. Available: <a href="http://monarchwatch.org/blog/2017/02/11/monarch-population-status-30/">http://monarchwatch.org/blog/2017/02/11/monarch-population-status-30/</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.53" data-doi="10.1016/j.cropro.2010.07.018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hartzler</span>  <span class="cit-name-given-names">RG.</span></span> <span class="cit-article-title">Reduction in common milkweed (Asclepias syriaca) occurrence in Iowa cropland from 1999 to 2009</span>. <abbr class="cit-jnl-abbrev">Crop Protection</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">29</span>: <span class="cit-fpage">1542</span>–<span class="cit-lpage">1544</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.cropro.2010.07.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BProtection%26rft.volume%253D29%26rft.spage%253D1542%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cropro.2010.07.018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cropro.2010.07.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000284442700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.54" data-doi="10.1111/j.1752-4598.2012.00196.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pleasants</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oberhauser</span>  <span class="cit-name-given-names">KS.</span></span> <span class="cit-article-title">Milkweed loss in agricultural fields because of herbicide use: effect on the monarch butterfly population</span>. <abbr class="cit-jnl-abbrev">Insect Conservation and Diversity</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">6</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1752-4598.2012.00196.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2BConservation%2Band%2BDiversity%26rft.volume%253D6%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4598.2012.00196.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4598.2012.00196.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.55" data-doi="10.1093/biosci/biw186"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zaya</span>  <span class="cit-name-given-names">DN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pearse</span>  <span class="cit-name-given-names">IS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spyreas</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Long-Term Trends in Midwestern Milkweed Abundances and Their Relevance to Monarch Butterfly Declines</span>. <abbr class="cit-jnl-abbrev">BioScience</abbr>. <span class="cit-pub-date">2017</span>;<span class="cit-vol">67</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/biosci/biw186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioScience%26rft.volume%253D67%26rft.spage%253D343%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiosci%252Fbiw186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biosci/biw186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Powles</span>  <span class="cit-name-given-names">SODSB.</span></span> <span class="cit-article-title">Glyphosate-resistant crops and weeds: now and in the future</span>. <span class="cit-pub-date">2010</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.57" data-doi="10.5066/F7NP22KM"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">NT.</span></span> <span class="cit-article-title">Estimated annual agricultural pesticide use by crop group for states of the conterminous United States, 1992-2014</span>. <abbr class="cit-jnl-abbrev">National Water Quality Assessment Program</abbr>. <span class="cit-pub-date">2017</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.5066/F7NP22KM</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNational%2BWater%2BQuality%2BAssessment%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.5066%252FF7NP22KM%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5066/F7NP22KM&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.58" data-doi="10.1111/ecog.02719"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ries</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oberhauser</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thogmartin</span>  <span class="cit-name-given-names">WE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zipkin</span>  <span class="cit-name-given-names">EF.</span></span> <span class="cit-article-title">Local and cross-seasonal associations of climate and land use with abundance of monarch butterflies Danaus plexippus</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr>. <span class="cit-pub-date">2017</span>; n/a-n/a.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/ecog.02719</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fecog.02719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ecog.02719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.59" data-doi="10.1111/j.1365-2486.2012.02751.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zipkin</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ries</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reeves</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regetz</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oberhauser</span>  <span class="cit-name-given-names">KS.</span></span> <span class="cit-article-title">Tracking climate impacts on the migratory monarch butterfly</span>. <abbr class="cit-jnl-abbrev">Glob Change Biol</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">18</span>: <span class="cit-fpage">3039</span>–<span class="cit-lpage">3049</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1365-2486.2012.02751.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChange%2BBiol%26rft.volume%253D18%26rft.spage%253D3039%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2486.2012.02751.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2486.2012.02751.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000308443800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brower</span>  <span class="cit-name-given-names">LP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kust</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rendon-Salinas</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Serrano</span>  <span class="cit-name-given-names">EG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kust</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Catastrophic winter storm mortality of monarch butterflies in Mexico during January 2002</span>. <abbr class="cit-jnl-abbrev">The Monarch butterfly: biology and conservation</abbr>. <span class="cit-pub-date">2004</span>; <span class="cit-fpage">151</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.61" data-doi="10.1016/j.biocon.2014.09.041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rendón-Salinas</span>  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Dynamics and trends of overwintering colonies of the monarch butterfly in Mexico</span>. <abbr class="cit-jnl-abbrev">Biological Conservation</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">180</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.biocon.2014.09.041</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BConservation%26rft.volume%253D180%26rft.spage%253D165%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biocon.2014.09.041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biocon.2014.09.041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altizer</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">New Perspectives on Monarch Migration, Evolution, and Population Biology</span>. <abbr class="cit-jnl-abbrev">Monarchs in a Changing World: Biology and Conservation of an Iconic Butterfly</abbr>. <span class="cit-pub-date">2015</span>; <span class="cit-vol">203</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.63" data-doi="10.1073/pnas.1805114116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ries</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neupane</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramírez</span>  <span class="cit-name-given-names">MI</span></span>, <span class="cit-auth"><span class="cit-name-surname">García-Serrano</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rendón-Salinas</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> <span class="cit-article-title">Multiscale seasonal factors drive the size of winter monarch colonies</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr>. <span class="cit-pub-date">2019</span>;<span class="cit-vol">116</span>: <span class="cit-fpage">8609</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1805114116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1805114116%26rft_id%253Dinfo%253Apmid%252F30886097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE2LzE3Lzg2MDkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8wNi81MjI2MzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-522631v2.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">KC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dwyer</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Food Limitation and Insect Outbreaks: Complex Dynamics in Plant-Herbivore Models</span>. <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">76</span>: <span class="cit-fpage">1004</span>–<span class="cit-lpage">1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.stitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.aulast%253DAbbott%26rft.auinit1%253DK.%2BC.%26rft.volume%253D76%26rft.issue%253D5%26rft.spage%253D1004%26rft.epage%253D1014%26rft.atitle%253DFood%2Blimitation%2Band%2Binsect%2Boutbreaks%253A%2Bcomplex%2Bdynamics%2Bin%2Bplant-herbivore%2Bmodels.%26rft_id%253Dinfo%253Apmid%252F17714279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17714279&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F06%2F522631.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/522631v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F522631v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/819276v2" title="Brain networks processing temporal information in dynamic facial expressions" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;November 06, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/522631v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/522631v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/11/06/522631.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/522631v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/522631v2" class="active" data-icon-position="" data-hide-link-title="0">The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/982714" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-mqIO0bvdCwrhcyQ81lE5KdKt588bwq6ZgKu0hkHK_N8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="982714" id="node982714--2" data-pisa="biorxiv;522631v2" data-pisa-master="biorxiv;522631" data-seqnum="982714" data-apath="/biorxiv/early/2019/11/06/522631.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Christie A.</span> <span class="nlm-surname">Bahlai</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Elise F.</span> <span class="nlm-surname">Zipkin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">522631; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/522631 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/522631v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/522631v2&amp;title=The%20Dynamic%20Shift%20Detector%3A%20an%20algorithm%20to%20identify%20changes%20in%20parameter%20values%20governing%20populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/522631v2&amp;title=The%20Dynamic%20Shift%20Detector%3A%20an%20algorithm%20to%20identify%20changes%20in%20parameter%20values%20governing%20populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/522631v2&amp;text=The%20Dynamic%20Shift%20Detector%3A%20an%20algorithm%20to%20identify%20changes%20in%20parameter%20values%20governing%20populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/522631v2&amp;title=The%20Dynamic%20Shift%20Detector%3A%20an%20algorithm%20to%20identify%20changes%20in%20parameter%20values%20governing%20populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/522631v2&amp;t=The%20Dynamic%20Shift%20Detector%3A%20an%20algorithm%20to%20identify%20changes%20in%20parameter%20values%20governing%20populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/522631v2&amp;title=The%20Dynamic%20Shift%20Detector%3A%20an%20algorithm%20to%20identify%20changes%20in%20parameter%20values%20governing%20populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/522631v2&amp;title=The%20Dynamic%20Shift%20Detector%3A%20an%20algorithm%20to%20identify%20changes%20in%20parameter%20values%20governing%20populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="982714" id="node982714--3" data-pisa="biorxiv;522631v2" data-pisa-master="biorxiv;522631" data-seqnum="982714" data-apath="/biorxiv/early/2019/11/06/522631.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">The Dynamic Shift Detector: an algorithm to identify changes in parameter values governing populations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Christie A.</span> <span class="nlm-surname">Bahlai</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Elise F.</span> <span class="nlm-surname">Zipkin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">522631; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/522631 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=The%20Dynamic%20Shift%20Detector%3A%20an%20algorithm%20to%20identify%20changes%20in%20parameter%20values%20governing%20populations&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Bahlai%2C+Christie+A.&amp;rft.au=Zipkin%2C+Elise+F."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/982714/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/982714/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/982714/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/982714/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/982714/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/982714/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/982714/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/982714/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/982714/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/982714/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/982714/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/982714/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/522631v2&amp;count=horizontal&amp;via=&amp;text=The%20Dynamic%20Shift%20Detector%3A%20an%20algorithm%20to%20identify%20changes%20in%20parameter%20values%20governing%20populations&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/522631v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/522631v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/522631v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
