<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/13/2019.12.12.873679.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.12.873679v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of Neodon voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing" />
<meta name="DC.Identifier" content="10.1101/2019.12.12.873679" />
<meta name="DC.Date" content="2019-12-13" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Neodon , genus of a short time evolutionary history, was reported to be diverged from its relatives in early stage of Pleistocene. Only 4 species were well documented in Neodon for a long period of time until last years when a systematic work described and added three new species, adjusted three species used to belong to Lasiopodomys, Phaiomys, Microtus to Neodon and removed one species ( Neodon juldaschi ) to genus Blanfordimys , leading to a total of eight species recorded in Neodon . To gain a better insight into the phylogeny and ecology of Neodon , we have systematically sampled Neodon species along the whole Hengduan and Himalayan Mountains in the last 20 years. In addition to morphological identification, we generated 1x - 15x whole genome sequencing (WGS) data and achieved the mitochondrial genomes and an average of 5,382 nuclear genes for each morpho-species. Both morphology and phylogeny results supported an extra six new species in Neodon (nominated Neodon shergylaensis sp. nov., N. namchabarwaensis sp. nov., N. liaoruii sp. nov., N. chayuensis sp. nov., N. bomiensis sp. nov., and N. bershulaensis sp. nov.). This is the first study that included Neodon samples covering its entire distribution area in China and this systematic sampling also revealed a long-time underestimation of Neodon ’s diversity, and suggested its speciation events linked highly to founder event via dispersal (from Plateau to surrounding mountains). The results also revealed that the Qinghai-Tibetan Plateau is the center of origin of Neodon , and the impetus of speciation include climate change, isolation of rivers and mountains." />
<meta name="DC.Contributor" content="Shaoying Liu" />
<meta name="DC.Contributor" content="Chengran Zhou" />
<meta name="DC.Contributor" content="Tao Wan" />
<meta name="DC.Contributor" content="Guanliang Meng" />
<meta name="DC.Contributor" content="W. Robert W. Murphy" />
<meta name="DC.Contributor" content="Zhengxin Fan" />
<meta name="DC.Contributor" content="Mingkun Tang" />
<meta name="DC.Contributor" content="Yang Liu" />
<meta name="DC.Contributor" content="Tao Zeng" />
<meta name="DC.Contributor" content="Shunde Chen" />
<meta name="DC.Contributor" content="Yun Zhao" />
<meta name="DC.Contributor" content="Shanlin Liu" />
<meta name="article:published_time" content="2019-12-13" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of Neodon voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;&lt;i&gt;Neodon&lt;/i&gt;, genus of a short time evolutionary history, was reported to be diverged from its relatives in early stage of Pleistocene. Only 4 species were well documented in &lt;i&gt;Neodon&lt;/i&gt; for a long period of time until last years when a systematic work described and added three new species, adjusted three species used to belong to &lt;i&gt;Lasiopodomys, Phaiomys, Microtus&lt;/i&gt; to &lt;i&gt;Neodon&lt;/i&gt; and removed one species (&lt;i&gt;Neodon juldaschi&lt;/i&gt;) to genus &lt;i&gt;Blanfordimys&lt;/i&gt;, leading to a total of eight species recorded in &lt;i&gt;Neodon&lt;/i&gt;. To gain a better insight into the phylogeny and ecology of &lt;i&gt;Neodon&lt;/i&gt;, we have systematically sampled &lt;i&gt;Neodon&lt;/i&gt; species along the whole Hengduan and Himalayan Mountains in the last 20 years. In addition to morphological identification, we generated 1x - 15x whole genome sequencing (WGS) data and achieved the mitochondrial genomes and an average of 5,382 nuclear genes for each morpho-species. Both morphology and phylogeny results supported an extra six new species in &lt;i&gt;Neodon&lt;/i&gt; (nominated &lt;i&gt;Neodon shergylaensis&lt;/i&gt; sp. nov., &lt;i&gt;N. namchabarwaensis&lt;/i&gt; sp. nov., &lt;i&gt;N. liaoruii&lt;/i&gt; sp. nov., &lt;i&gt;N. chayuensis&lt;/i&gt; sp. nov., &lt;i&gt;N. bomiensis&lt;/i&gt; sp. nov., and &lt;i&gt;N. bershulaensis&lt;/i&gt; sp. nov.). This is the first study that included &lt;i&gt;Neodon&lt;/i&gt; samples covering its entire distribution area in China and this systematic sampling also revealed a long-time underestimation of &lt;i&gt;Neodon&lt;/i&gt;’s diversity, and suggested its speciation events linked highly to founder event via dispersal (from Plateau to surrounding mountains). The results also revealed that the Qinghai-Tibetan Plateau is the center of origin of &lt;i&gt;Neodon&lt;/i&gt;, and the impetus of speciation include climate change, isolation of rivers and mountains.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.12.873679v1" />
<meta name="citation_id" content="2019.12.12.873679v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.12.873679v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.12.873679v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.12.873679v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.12.873679" />
<meta name="citation_num_pages" content="39" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.12.873679" />
<meta name="citation_author" content="Shaoying Liu" />
<meta name="citation_author_institution" content="Sichuan Academy of Forestry" />
<meta name="citation_author_email" content="shaoyliu@163.com" />
<meta name="citation_author_email" content="shanlin1115@gmail.com" />
<meta name="citation_author" content="Chengran Zhou" />
<meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University" />
<meta name="citation_author_institution" content="BGI-Shenzhen" />
<meta name="citation_author_institution" content="China National GeneBank" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9468-5973" />
<meta name="citation_author" content="Tao Wan" />
<meta name="citation_author_institution" content="Sichuan Academy of Forestry" />
<meta name="citation_author_institution" content="College of Life Sciences, Shaanxi Normal University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6528-8672" />
<meta name="citation_author" content="Guanliang Meng" />
<meta name="citation_author_institution" content="BGI-Shenzhen" />
<meta name="citation_author_institution" content="China National GeneBank" />
<meta name="citation_author" content="W. Robert W. Murphy" />
<meta name="citation_author_institution" content="Centre for Biodiversity and Conservation Biology" />
<meta name="citation_author" content="Zhengxin Fan" />
<meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University" />
<meta name="citation_author" content="Mingkun Tang" />
<meta name="citation_author_institution" content="Sichuan Academy of Forestry" />
<meta name="citation_author" content="Yang Liu" />
<meta name="citation_author_institution" content="Sichuan Academy of Forestry" />
<meta name="citation_author" content="Tao Zeng" />
<meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University" />
<meta name="citation_author" content="Shunde Chen" />
<meta name="citation_author_institution" content="College of Life Science, Sichuan Normal University" />
<meta name="citation_author" content="Yun Zhao" />
<meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University" />
<meta name="citation_author" content="Shanlin Liu" />
<meta name="citation_author_institution" content="BGI-Shenzhen" />
<meta name="citation_author_institution" content="China National GeneBank" />
<meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Food Nutrition and Human Health, China Agricultural University" />
<meta name="citation_author_email" content="shaoyliu@163.com" />
<meta name="citation_author_email" content="shanlin1115@gmail.com" />
<meta name="citation_reference" content="citation_journal_title=Doklady Biological Sciences;citation_year=2009" />
<meta name="citation_reference" content="Allen GM 1940. Mammals of China and Mongolia. Part 2. American Museum of Natural History, Central Asiatic Expeditions 1921-1930 11: 621–1350." />
<meta name="citation_reference" content="Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ 1990. Basic local alignment search tool. Journal of Molecular Biology 215: 403–410." />
<meta name="citation_reference" content="An Z, John E K, Warren L P, Stephen C P 2001. Evolution of Asian monsoons and phased uplift of the Himalaya–Tibetan plateau since Late Miocene times. Nature 411: 62." />
<meta name="citation_reference" content="Carleton MD, Musser GG 1995. Systematic Studies of Oryzomyine Rodents (Muridae, Sigmodontinae)-Definition and Distribution of Oligoryzomys-Vegetus (Bangs, 1902). Proceedings of the Biological Society of Washington." />
<meta name="citation_reference" content="Chen J, Ji J, Gong J, Qing J 2008. Formation of the Yarlung Zangbo Grand Canyon, Tibet, China. Geological Bulletin of China 27: 491–499." />
<meta name="citation_reference" content="Chifman J, Kubatko L 2014. Quartet inference from SNP data under the coalescent model. Bioinformatics 30: 3317–3324." />
<meta name="citation_reference" content="Corbet GB 1978. The mammals of the Palaearctic region: a taxonomic review. British Museum (Natural History)." />
<meta name="citation_reference" content="Coyne J, Orr H 2004. Speciation Sinauer Associates. Sunderland, MA: 276–281." />
<meta name="citation_reference" content="Ellerman J 1949. The Families and Genera of Living Rodents Volume III, Part I. British Museum of Natural History, London." />
<meta name="citation_reference" content="citation_journal_title=Family Muridae. The Families and Genera of Living Rodents. With a List of Named Forms (1758-1936);citation_title=The Families and Genera of Living Rodents. With a List of Named Forms (1758-1936);citation_author=by RW HAYMAN &amp; GWC HOLT. Volume II. Family Muridae;citation_volume=II;citation_year=1941" />
<meta name="citation_reference" content="Ellerman JR, Morrison-Scott TCS. 1951. Checklist of Palaearctic and Indian mammals, 1758–1946: British Museum (Natural History)." />
<meta name="citation_reference" content="Feng Z-J, Cai G-Q, Zheng C-L 1986. The mammals of Xizang. Science, Beijing: 1–441." />
<meta name="citation_reference" content="Field A, Miles J, Field Z. 2012. Discovering statistics using R: Sage publications." />
<meta name="citation_reference" content="Gromov I, Polyakov IY 1977. Mammals: Voles (Microtinae). Fauna SSSR 3: 1–504." />
<meta name="citation_reference" content="Guo Z, Sun B, Zhang Z, Peng S, Xiao G, Ge J, Hao Q, Qiao Y, Liang M, Liu J 2008. A major reorganization of Asian climate regime by the Early Miocene. Climate of the past Discussions 4: 535–584." />
<meta name="citation_reference" content="Hooper ET 1958. The male phallus in mice of the genus Peromyscus. MUSEUM OF ZOOLOGY, UNIVERSITY of MICHIGAN." />
<meta name="citation_reference" content="Hooper ET, Hart BS 1962. A synopsis of recent North American microtine rodents. MUSEUM OF ZOOLOGY, UNIVERSITY of MICHIGAN." />
<meta name="citation_reference" content="Hubbard T, Barker D, Birney E, Cameron G, Chen Y, Clark L, Cox T, Cuff J, Curwen V, Down T 2002. The Ensembl genome database project. Nucleic Acids Research 30: 38–41." />
<meta name="citation_reference" content="Huelsenbeck JP, Ronquist F 2001. MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics 17: 754–755." />
<meta name="citation_reference" content="Jia DR, Abbott RJ, Liu TL, Mao KS, Bartish IV, Liu JQ 2012. Out of the Qinghai–Tibet Plateau: evidence for the origin and dispersal of Eurasian temperate plants from a phylogeographic study of Hippophaë rhamnoides (Elaeagnaceae). New Phytologist 194: 1123–1133." />
<meta name="citation_reference" content="Katoh K, Standley DM 2013. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Molecular Biology and Evolution 30: 772–780." />
<meta name="citation_reference" content="Kozlov AM, Aberer AJ, Stamatakis A 2015. ExaML version 3: a tool for phylogenomic analyses on supercomputers. Bioinformatics 31: 2577–2579." />
<meta name="citation_reference" content="Kumar S, Stecher G, Suleski M, Hedges SB 2017. TimeTree: a resource for timelines, timetrees, and divergence times. Molecular Biology and Evolution 34: 1812–1819." />
<meta name="citation_reference" content="Landis MJ, Matzke NJ, Moore BR, Huelsenbeck JP 2013. Bayesian analysis of biogeography when the number of areas is large. Systematic Biology 62: 789–804." />
<meta name="citation_reference" content="Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth GT, Abecasis GR, Durbin R 2009. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25: 2078–2079." />
<meta name="citation_reference" content="Lidicker Jr W 1968. A phylogeny of New Guinea rodent genera based on phallic morphology. Journal of Mammalogy 49: 609–643." />
<meta name="citation_reference" content="Liu S, Jin W, Liu Y, Murphy RW, Lv B, Hao H, Liao R, Sun Z, Tang M, Chen W 2017. Taxonomic position of Chinese voles of the tribe Arvicolini and the description of 2 new species from Xizang, China. Journal of Mammalogy 98: 166–182." />
<meta name="citation_reference" content="Liu SY, Sun ZY, Liu Y, Wang H, Guo P, Murphy RW 2012. A new vole from Xizang, China and the molecular phylogeny of the genus Neodon (Cricetidae: Arvicolinae). Zootaxa 3235: 1–22." />
<meta name="citation_reference" content="Liu Y, Montgomery D, Hallet B, Tang W, Zhang J, Zhang X 2006. Quaternary glacier blocking events at the entrance of Yarlung Zangbo Great Canyon, Southeast Tibet. Quaternary Sciences 26: 52–62." />
<meta name="citation_reference" content="Luo Z 2000. Rodentia Part III: Cricetidae. Fauna Sinica Mammalia 6: 121–128." />
<meta name="citation_reference" content="Lv X, Cheng J, Meng Y, Chang Y, Xia L, Wen Z, Ge D, Liu S, Yang Q 2018. Disjunct distribution and distinct intraspecific diversification of Eothenomys melanogaster in South China. BMC Evolutionary Biology 18: 50." />
<meta name="citation_reference" content="Lv X, Xia L, Ge D, Wu Y, Yang Q 2016. Climatic niche conservatism and ecological opportunity in the explosive radiation of arvicoline rodents (Arvicolinae, Cricetidae). Evolution 70: 1094–1104." />
<meta name="citation_reference" content="Marchese C 2015. Biodiversity hotspots: A shortcut for a more complicated concept. Global Ecology and Conservation 3: 297–309." />
<meta name="citation_reference" content="Matzke NJ 2013a. BioGeoBEARS: BioGeography with Bayesian (and likelihood) evolutionary analysis in R Scripts. R package, version 0.2 1: 2013." />
<meta name="citation_reference" content="Matzke NJ 2014. Model selection in historical biogeography reveals that founder-event speciation is a crucial process in island clades. Systematic Biology 63: 951–970." />
<meta name="citation_reference" content="Matzke NJ 2013b. Probabilistic historical biogeography: new models for founder-event speciation, imperfect detection, and fossils allow improved accuracy and model-testing. Frontiers of Biogeography 5." />
<meta name="citation_reference" content="Meng G, Li Y, Yang C, Liu S 2019. MitoZ: a toolkit for animal mitochondrial genome assembly, annotation and visualization. Nucleic Acids Research 47: e63–e63." />
<meta name="citation_reference" content="Moreau CS, Bell CD 2013. Testing the museum versus cradle tropical biological diversity hypothesis: phylogeny, diversification, and ancestral biogeographic range evolution of the ants. Evolution 67: 2240–2257." />
<meta name="citation_reference" content="Mosbrugger V, Favre A, Muellner-Riehl AN, Päckert M, Mulch A 2018. Cenozoic evolution of geo-biodiversity in the Tibeto-Himalayan region. Mountains, Climate, and Biodiversity: 429–448." />
<meta name="citation_reference" content="Muellner-Riehl AN 2019. Mountains as evolutionary arenas: patterns, emerging approaches, paradigm shifts, and their implications for plant phylogeographic research in the Tibeto-Himalayan region. Frontiers in Plant Science 10: 195." />
<meta name="citation_reference" content="Musser G, Carleton M 2005. Order rodentia. Mammal species of the world: a taxonomic and geographic reference 2." />
<meta name="citation_reference" content="Myers N, Mittermeier RA, Mittermeier CG, Da Fonseca GA, Kent J 2000. Biodiversity hotspots for conservation priorities. Nature 403: 853." />
<meta name="citation_reference" content="Paradis E, Claude J, Strimmer K 2004. APE: analyses of phylogenetics and evolution in R language. Bioinformatics 20: 289–290." />
<meta name="citation_reference" content="Pisano J, Condamine FL, Lebedev V, Bannikova A, Quéré JP, Shenbrot GI, Pages M, Michaux JR 2015. Out of Himalaya: the impact of past Asian environmental changes on the evolutionary and biogeographical history of Dipodoidea (Rodentia). Journal of Biogeography 42: 856–870." />
<meta name="citation_reference" content="Puillandre N, Lambert A, Brouillet S, Achaz G 2012. ABGD, Automatic Barcode Gap Discovery for primary species delimitation. Molecular Ecology 21: 1864–1877." />
<meta name="citation_reference" content="Rambaut A, Drummond AJ, Xie D, Baele G, Suchard MA 2018. Posterior summarization in Bayesian phylogenetics using Tracer 1.7. Systematic Biology 67: 901–904." />
<meta name="citation_reference" content="Ree RH, Smith SA 2008. Maximum likelihood inference of geographic range evolution by dispersal, local extinction, and cladogenesis. Systematic Biology 57: 4–14." />
<meta name="citation_reference" content="Revell LJ 2012. phytools: an R package for phylogenetic comparative biology (and other things). Methods in Ecology and Evolution 3: 217–223." />
<meta name="citation_reference" content="Ronquist F 1997. Dispersal-vicariance analysis: a new approach to the quantification of historical biogeography. Systematic Biology 46: 195–203." />
<meta name="citation_reference" content="Sayyari E, Whitfield JB, Mirarab S 2018. DiscoVista: Interpretable visualizations of gene tree discordance. Molecular Phylogenetics and Evolution 122: 110–115." />
<meta name="citation_reference" content="Sikes RS, Gannon WL 2011. Guidelines of the American Society of Mammalogists for the use of wild mammals in research. Journal of Mammalogy 92: 235–253." />
<meta name="citation_reference" content="Simão FA, Waterhouse RM, Ioannidis P, Kriventseva EV, Zdobnov EM 2015. BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs. Bioinformatics 31: 3210–3212." />
<meta name="citation_reference" content="Stamatakis A 2014. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30: 1312–1313." />
<meta name="citation_reference" content="Suyama M, Torrents D, Bork P 2006. PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments. Nucleic acids research 34: W609–W612." />
<meta name="citation_reference" content="Swofford DL 2001. Paup*: Phylogenetic analysis using parsimony (and other methods) 4.0. B5." />
<meta name="citation_reference" content="Tamma K, Ramakrishnan U 2015. Higher speciation and lower extinction rates influence mammal diversity gradients in Asia. BMC Evolutionary Biology 15: 11." />
<meta name="citation_reference" content="Wang E-q, Chen L-z, Chen Z-l 2002. Tectonic and climatic element-controlled evolution of the Yalungzangbu River in southern Tibet. Quaternary Sciences 22: 365–373." />
<meta name="citation_reference" content="Wang X, Tseng ZJ, Li Q, Takeuchi GT, Xie G 2014. From ‘third pole’to north pole: a Himalayan origin for the arctic fox. Proceedings of the Royal Society B: Biological Sciences 281: 20140893." />
<meta name="citation_reference" content="Weigold H. 2005. Die biogeographie Tibets und seiner vorländer: Verein Sächsischer Ornithologen." />
<meta name="citation_reference" content="Wickham H. 2016. ggplot2: elegant graphics for data analysis: Springer." />
<meta name="citation_reference" content="Winger BM, Bates JM 2015. The tempo of trait divergence in geographic isolation: Avian speciation across the Marañon Valley of Peru. Evolution 69: 772–787." />
<meta name="citation_reference" content="Xing Y, Ree RH 2017. Uplift-driven diversification in the Hengduan Mountains, a temperate biodiversity hotspot. Proceedings of the National Academy of Sciences 114: E3444–E3451." />
<meta name="citation_reference" content="Yang A, Fang L 1988. Phallic morphology of 13 species of the family Muridae from China, with comments on its taxonomic significance. Acta Theriologica Sinica 4: 275–285." />
<meta name="citation_reference" content="Yang A, Liu S, Fang L 1992. Phallic morphology of eight species in Gerbillinae and Microtinae from China. Acta Theriologica Sinica 1: 31–38." />
<meta name="citation_reference" content="Yang Z 2007. PAML 4: phylogenetic analysis by maximum likelihood. Molecular Biology and Evolution 24: 1586–1591." />
<meta name="citation_reference" content="Yu G, Smith DK, Zhu H, Guan Y, Lam TTY 2017. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods in Ecology and Evolution 8: 28–36." />
<meta name="citation_reference" content="Zhang C, Rabiee M, Sayyari E, Mirarab S 2018. ASTRAL-III: polynomial time species tree reconstruction from partially resolved gene trees. BMC Bioinformatics 19: 153." />
<meta name="citation_reference" content="Zhang J, Kapli P, Pavlidis P, Stamatakis A 2013. A general species delimitation method with applications to phylogenetic placements. Bioinformatics 29: 2869–2876." />
<meta name="citation_reference" content="Zhang Q, He T, Wei H, Li F, Feng Y, Zong H, Chen S 2016. Characterization of the complete mitochondrial genome and phylogenetic relationship of Neodon sikimensis (Rodentia: Arvicolinae). Mitochondrial DNA Part B 1: 445–446." />
<meta name="twitter:title" content="Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of Neodon voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Neodon , genus of a short time evolutionary history, was reported to be diverged from its relatives in early stage of Pleistocene. Only 4 species were well documented in Neodon for a long period of time until last years when a systematic work described and added three new species, adjusted three species used to belong to Lasiopodomys, Phaiomys, Microtus to Neodon and removed one species ( Neodon juldaschi ) to genus Blanfordimys , leading to a total of eight species recorded in Neodon . To gain a better insight into the phylogeny and ecology of Neodon , we have systematically sampled Neodon species along the whole Hengduan and Himalayan Mountains in the last 20 years. In addition to morphological identification, we generated 1x - 15x whole genome sequencing (WGS) data and achieved the mitochondrial genomes and an average of 5,382 nuclear genes for each morpho-species. Both morphology and phylogeny results supported an extra six new species in Neodon (nominated Neodon shergylaensis sp. nov., N. namchabarwaensis sp. nov., N. liaoruii sp. nov., N. chayuensis sp. nov., N. bomiensis sp. nov., and N. bershulaensis sp. nov.). This is the first study that included Neodon samples covering its entire distribution area in China and this systematic sampling also revealed a long-time underestimation of Neodon ’s diversity, and suggested its speciation events linked highly to founder event via dispersal (from Plateau to surrounding mountains). The results also revealed that the Qinghai-Tibetan Plateau is the center of origin of Neodon , and the impetus of speciation include climate change, isolation of rivers and mountains." />
<meta name="og-title" property="og:title" content="Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of Neodon voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.12.873679v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Neodon , genus of a short time evolutionary history, was reported to be diverged from its relatives in early stage of Pleistocene. Only 4 species were well documented in Neodon for a long period of time until last years when a systematic work described and added three new species, adjusted three species used to belong to Lasiopodomys, Phaiomys, Microtus to Neodon and removed one species ( Neodon juldaschi ) to genus Blanfordimys , leading to a total of eight species recorded in Neodon . To gain a better insight into the phylogeny and ecology of Neodon , we have systematically sampled Neodon species along the whole Hengduan and Himalayan Mountains in the last 20 years. In addition to morphological identification, we generated 1x - 15x whole genome sequencing (WGS) data and achieved the mitochondrial genomes and an average of 5,382 nuclear genes for each morpho-species. Both morphology and phylogeny results supported an extra six new species in Neodon (nominated Neodon shergylaensis sp. nov., N. namchabarwaensis sp. nov., N. liaoruii sp. nov., N. chayuensis sp. nov., N. bomiensis sp. nov., and N. bershulaensis sp. nov.). This is the first study that included Neodon samples covering its entire distribution area in China and this systematic sampling also revealed a long-time underestimation of Neodon ’s diversity, and suggested its speciation events linked highly to founder event via dispersal (from Plateau to surrounding mountains). The results also revealed that the Qinghai-Tibetan Plateau is the center of origin of Neodon , and the impetus of speciation include climate change, isolation of rivers and mountains." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-13" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.12.873679v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.12.873679v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.12.873679v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.12.873679v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1036085" />
    <title>Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of Neodon voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zKVaaxrfwVIkJfhJNtyOEVbkfKBH6nrClUE6HgfIqoA","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1036085","apath":"\/biorxiv\/early\/2019\/12\/13\/2019.12.12.873679.atom","pisa":"biorxiv;2019.12.12.873679v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.12.873679v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.12.873679v1","title":"Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of Neodon voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing","identifier":"node\/1036085"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O0BRNKQcj5kQpBMdXXAFvkXcmwhRlK36c\\\\\\\/Gc\\\\\\\/ny++hYwfrZigshLA6GTSD9mGDaanTWPYW+i6+c6IQ0aFduTz2JHhhH\\\\\\\/2zDr719A9sC2ZBPK6M63w1UbykxGydgVc9rqt5mE6LLVUPJZzeF6IxuUNWPzf+NCD8IswOKtQqUyAzHNxPeOce1kyFo9A1eT3Y4rmAU08pGOkgdyUPBUU5U8Hrj79uqn4pmZxwkl6e0nkLPqPlz5dMR3j86iTHAnVelSSiDkptgINDxd+Y3hMQVncWc65dHa64CwYvzyFnncMOTN7qgGwUuv+It+BCiYKvCQtBLqvIbmOmsF9S4LrWfkdlsxpymUhG\\\\\\\/niNm\\\\\\\/0OqT8a66IyVTxRKEd44uNWuOEXqu9wN4gVgkmt7WDSIx2p2py2VYECFhn60qR9QINz8dVRCzG6++kCJCHUnZcjIHynQbOrl70l3BOUCveo6xYShV2eodow4QwQ3fR8PtZLsNGjRzfUO8F5GThZ5dCJ\\\\\\\/agHpp3pTXxMRpOgaBiAu82H5SWO4q9ONE0f2Ec+ndw803bojDwUgyJsdfHS8dz+4ezF\\\\\\\/D8jxrh6Gpt6JDIyRz55JqmbK3VUUuEz+e8p0B2T4ZdoOG9aaXbMKo82b7qk4UM1prDCkdqLlVD41y0pgI47c\\\\\\\/uXWJgGK2LzNZcZOEnJMFGg6IOwDZHUlMhvMjb6UTO7GbBFnQNXGJuyRo63g53lQEE+C\\\\\\\/fcf1A3I4aQJomfHJ0pcgVz3Kyk3gHcSquvoK6V0JiGCm5oLZGktD48rYQGO44r\\\\\\\/QlupEQbqu7wdevFwQSDjlXxtzxzy7gS1xQiXhCH3QqLgVd1X1x1OChYHxzTtGB\\\\\\\/F6jlKxlJ2nkHOhGd6B6sImLYNyfWOZLthDRGHKeMaoHpuTYnz\\\\\\\/1+TJmGKq3IPc1mKqP4Suuvw2DrZg7u0j1qVYUPucKi9gTtFCZpIc5g5mL6G97dM7CZ6cJg5qqAYnAv8WNyzcB29+fnFWSMia+4ldNfySaaRBvOsm1LFmPztDY\\\\\\\/aooGyQqxPB2xO8SssNQdvewygjHL+K0LUZMI0rOmfl1eeB+6gg9FqZmoFQU2QpDW+b4YXflAVl\\\\\\\/w8bLpMBAcXh5fylYEGBnhr2Epb38bsBfXkyUvXdGjWwc+2PnqjF28IBS2uedMIm2AzLkBSHLHgnh701ytQ+YkrdHGN\\\\\\\/tn9pVZKv4wMTbCOto5GSkQHfthgqaRI6sRhNEH8Qy3qvc7oiU3+9727TvLllHfxmm1A1VNozDNCukg7l183BPXivOPiE+8R8dmH64Dvxd08USJFd2l+EpTSAhCrJZF098XslmJz8ES4DlB7kjZ8pbZdP\\\u0022,\\\u0022iv\\\u0022:\\\u0022i7WVJa4RdUZK5U36jNazHA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022ecee40eba69f5ce299b83384866f830c3cb940b765ce6383fd1c04fbc54184cf\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.12.873679v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1036085 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.12.873679v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-yRitqLouP9T9Hzq8tNdTaoLz2KET4I_Y3fKXiPsmmes" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1036085" id="node1036085" data-pisa="biorxiv;2019.12.12.873679v1" data-pisa-master="biorxiv;2019.12.12.873679" data-apath="/biorxiv/early/2019/12/13/2019.12.12.873679.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of <em>Neodon</em> voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shaoying</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9468-5973" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Chengran</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-6528-8672" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Wan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Guanliang</span> <span class="nlm-surname">Meng</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">W. Robert W.</span> <span class="nlm-surname">Murphy</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Zhengxin</span> <span class="nlm-surname">Fan</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Mingkun</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Yang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Zeng</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Shunde</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Yun</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Shanlin</span> <span class="nlm-surname">Liu</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.12.873679 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1036085" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shaoying Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Sichuan Academy of Forestry</span>, No.18, Xinhui xilu, Chengdu, Sichuan, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shaoying%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShaoying%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shaoyliu@163.com" class="" data-icon-position="" data-hide-link-title="0">shaoyliu@163.com</a>
<a href="mailto:shanlin1115@gmail.com" class="" data-icon-position="" data-hide-link-title="0">shanlin1115@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Chengran Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University</span>, Chengdu 610065, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>BGI-Shenzhen</span>, Shenzhen 518083, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>China National GeneBank</span>, BGI-Shenzhen, Shenzhen 518120, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chengran%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChengran%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9468-5973" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chengran Zhou</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Tao Wan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Sichuan Academy of Forestry</span>, No.18, Xinhui xilu, Chengdu, Sichuan, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>College of Life Sciences, Shaanxi Normal University</span>, Xi’an, Shaanxi 710119, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tao%2BWan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wan%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATao%2BWan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6528-8672" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Tao Wan</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Guanliang Meng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>BGI-Shenzhen</span>, Shenzhen 518083, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>China National GeneBank</span>, BGI-Shenzhen, Shenzhen 518120, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guanliang%2BMeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meng%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuanliang%2BMeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">W. Robert W. Murphy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centre for Biodiversity and Conservation Biology</span>, Royal Ontario Museum, 100 Queen’s Park, Toronto, Ontario M5S 2C6, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=W.%2BRobert%2BW.%2BMurphy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Murphy%20WR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AW.%2BRobert%2BW.%2BMurphy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhengxin Fan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University</span>, Chengdu 610065, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhengxin%2BFan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fan%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhengxin%2BFan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mingkun Tang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Sichuan Academy of Forestry</span>, No.18, Xinhui xilu, Chengdu, Sichuan, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mingkun%2BTang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tang%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMingkun%2BTang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Yang Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Sichuan Academy of Forestry</span>, No.18, Xinhui xilu, Chengdu, Sichuan, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yang%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYang%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Tao Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University</span>, Chengdu 610065, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tao%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATao%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Shunde Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>College of Life Science, Sichuan Normal University</span>, Chengdu 610066, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shunde%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShunde%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Yun Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University</span>, Chengdu 610065, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yun%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYun%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Shanlin Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>BGI-Shenzhen</span>, Shenzhen 518083, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>China National GeneBank</span>, BGI-Shenzhen, Shenzhen 518120, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>Beijing Advanced Innovation Center for Food Nutrition and Human Health, China Agricultural University</span>, Beijing 100193, <span class='nlm-country'>People’s Republic of China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shanlin%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShanlin%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:shaoyliu@163.com" class="" data-icon-position="" data-hide-link-title="0">shaoyliu@163.com</a>
<a href="mailto:shanlin1115@gmail.com" class="" data-icon-position="" data-hide-link-title="0">shanlin1115@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.12.873679v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1036085" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1036085/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.12.873679v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1036085" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1036085/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.12.873679v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1036085" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1036085/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.12.873679v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1036085" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1036085/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.12.873679v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1036085" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1036085/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.12.873679v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1036085" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1036085/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3"><em>Neodon</em>, genus of a short time evolutionary history, was reported to be diverged from its relatives in early stage of Pleistocene. Only 4 species were well documented in <em>Neodon</em> for a long period of time until last years when a systematic work described and added three new species, adjusted three species used to belong to <em>Lasiopodomys, Phaiomys, Microtus</em> to <em>Neodon</em> and removed one species (<em>Neodon juldaschi</em>) to genus <em>Blanfordimys</em>, leading to a total of eight species recorded in <em>Neodon</em>. To gain a better insight into the phylogeny and ecology of <em>Neodon</em>, we have systematically sampled <em>Neodon</em> species along the whole Hengduan and Himalayan Mountains in the last 20 years. In addition to morphological identification, we generated 1x - 15x whole genome sequencing (WGS) data and achieved the mitochondrial genomes and an average of 5,382 nuclear genes for each morpho-species. Both morphology and phylogeny results supported an extra six new species in <em>Neodon</em> (nominated <em>Neodon shergylaensis</em> sp. nov., <em>N. namchabarwaensis</em> sp. nov., <em>N. liaoruii</em> sp. nov., <em>N. chayuensis</em> sp. nov., <em>N. bomiensis</em> sp. nov., and <em>N. bershulaensis</em> sp. nov.). This is the first study that included <em>Neodon</em> samples covering its entire distribution area in China and this systematic sampling also revealed a long-time underestimation of <em>Neodon</em>’s diversity, and suggested its speciation events linked highly to founder event via dispersal (from Plateau to surrounding mountains). The results also revealed that the Qinghai-Tibetan Plateau is the center of origin of <em>Neodon</em>, and the impetus of speciation include climate change, isolation of rivers and mountains.</p></div><div id="sec-1" class="subsection"><p id="p-4">The Tibetan-Himalayan region (THR), comprised of the Himalayas, the Hengduan Mountains (HD) biodiversity hotspot and the Qinghai-Tibetan Plateau (QTP), consists of a series of parallel alpine ridges and deep river valleys forming dramatic ecological stratification and environmental heterogeneity (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Marchese 2015</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Muellner-Riehl 2019</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Myers, et al. 2000</a>). The complex topographical features and physical boundaries lead to the geographic isolation of biota that limits or ceases gene flow. This can drive speciation. The topographical complexity and vast territory also constrain fieldwork, and this affects estimates of species richness and testing hypotheses on species’ interactions, and especially for species with limited dispersal abilities. The QTP is regarded to be a “museum of evolution” and a “cradle of evolution” (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Moreau and Bell 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mosbrugger, et al. 2018</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Tamma and Ramakrishnan 2015</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Xing and Ree 2017</a>) because of its high percentages of ancient species and biodiversity. Evidence is accumulating that the QTP is the center of origin and accumulation for many organisms with particular biogeographical relationships to other Palearctic regions, and this support the “out of Tibet” hypothesis (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jia, et al. 2012</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Mosbrugger, et al. 2018</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pisano, et al. 2015</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wang, et al. 2014</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Weigold 2005</a>). Thus, the Tibetan-Himalayan region provides critical clues to how geology and climate together drive the evolution.</p><p id="p-5">Voles and lemmings are one of the youngest groups of rodents and the most recent ancestor of <em>Neodon</em> and <em>Microtus</em> (Rodentia: Cricetidae) was dated at about 7 million years ago (Mya) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abramson, et al. 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lv, et al. 2016</a>). Speciation within <em>Neodon</em> was driven by orogenesis in the HD region from the late Miocene, when mountains surrounding QTP have reached its current elevations (~ 8 Mya), to the late Pliocene (~2 Mya) (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Mosbrugger, et al. 2018</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Muellner-Riehl 2019</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Xing and Ree 2017</a>). <em>Neodon</em> was erected by Horsfield in 1841 with only four well-documented species (<em>N. sikimensis, N. irene, N. forresti</em> and <em>N. juldaschi</em>). The genus occurs only in the Himalayas, HD and QTP (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu, et al. 2017</a>). A long-running taxonomic debate involved <em>Neodon</em>’s phylogenetic position—either as a subgenus of <em>Microtus</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Allen 1940</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carleton and Musser 1995</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gromov and Polyakov 1977</a>) or as a subgenus of <em>Pitymys</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Corbet 1978</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ellerman 1949</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ellerman and Morrison-Scott 1951</a>). However, recent morphological and molecular evidence confirmed <em>Neodon</em> to be a monophyletic genus (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ellerman 1941</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Liu, et al. 2017</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu, et al. 2012</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Musser and Carleton 2005</a>), and having far more than four species. For example, all but two species in Arvicolinae (Cricetidae) on the QTP and surrounding high elevation areas belong to <em>Neodon</em>.</p><p id="p-6">To gain better insights into the phylogenetic status and diversity of <em>Neodon</em>, as well as the role played by the QTP in driving biogeography and diversification, we report on a collection of specimens of small mammals taken from throughout the distribution <em>Neodon</em>, and especially in the Himalayas, over the past 20 years. Sampling, which covers tens of thousands of square kilometers, achieved more than 2,000 samples, of which 193 samples were morphologically identified to be <em>Neodon</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>, Supplementary Appendix S1). In addition to morphological and geographic data, we provide 1X–15X whole genome sequencing (WGS) data for each representative morphological species (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>, Supplementary Table S1), identify and describe six new species, demonstrate the underestimation of diversity in China and reveal that drastic climate change and topography of THR, as well as the founder event via dispersal, influence the diversification. These results suggest that similar investigations of other small mammals will discover greater diversity and lead to identifying the common driver(s) of patterns and processes in the THR ecosystem.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Geographical distribution of Neodon in this study. Approximate extent of occurrence of each clade is shown as colored lines (clade 1: yellow; clade 2: green, clade 3: purple, refer to Fig. 4 for clade information). Stars show the type localities of former species. Circles show the historical collection sites. Squares show the distribution of newly collected specimens." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1196413288" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Geographical distribution of &lt;em&gt;Neodon&lt;/em&gt; in this study. Approximate extent of occurrence of each clade is shown as colored lines (clade 1: yellow; clade 2: green, clade 3: purple, refer to Fig. 4 for clade information). Stars show the type localities of former species. Circles show the historical collection sites. Squares show the distribution of newly collected specimens.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F1.medium.gif" width="440" height="344"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F1.medium.gif" width="440" height="344"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Geographical distribution of <em>Neodon</em> in this study. Approximate extent of occurrence of each clade is shown as colored lines (clade 1: yellow; clade 2: green, clade 3: purple, refer to <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a> for clade information). Stars show the type localities of former species. Circles show the historical collection sites. Squares show the distribution of newly collected specimens.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic pipeline illustrating the workflow of morphological analysis and molecular phylogeny analysis after systematic sampling. a) Workflow of morphological analysis. b) Workflow of molecular analysis, including: A. sample preparation for sequencing and reference data set construction; B. mitochondrial and nuclear data set construction; and C. phylogenetic analysis, divergence time estimation, diversification analysis, incomplete lineage sorting (ILS) testing, species delimitation, et al. The combined results of both morphological analysis and molecular phylogeny analysis shows that Neodon has 15 species, including six new species identified herein." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1196413288" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic pipeline illustrating the workflow of morphological analysis and molecular phylogeny analysis after systematic sampling. a) Workflow of morphological analysis. b) Workflow of molecular analysis, including: A. sample preparation for sequencing and reference data set construction; B. mitochondrial and nuclear data set construction; and C. phylogenetic analysis, divergence time estimation, diversification analysis, incomplete lineage sorting (ILS) testing, species delimitation, et al. The combined results of both morphological analysis and molecular phylogeny analysis shows that &lt;em&gt;Neodon&lt;/em&gt; has 15 species, including six new species identified herein.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F2.medium.gif" width="440" height="242"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F2.medium.gif" width="440" height="242"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-8" class="first-child">Schematic pipeline illustrating the workflow of morphological analysis and molecular phylogeny analysis after systematic sampling. a) Workflow of morphological analysis. b) Workflow of molecular analysis, including: A. sample preparation for sequencing and reference data set construction; B. mitochondrial and nuclear data set construction; and C. phylogenetic analysis, divergence time estimation, diversification analysis, incomplete lineage sorting (ILS) testing, species delimitation, et al. The combined results of both morphological analysis and molecular phylogeny analysis shows that <em>Neodon</em> has 15 species, including six new species identified herein.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">M<span class="sc">aterials and</span> M<span class="sc">ethods</span></h2><div id="sec-3" class="subsection"><h3>Ethics statement</h3><p id="p-9">All samples were obtained following Guidelines of the American Society of Mammalogists (ASM guidelines) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sikes and Gannon 2011</a>) and the laws and regulations of China for the implementation of the protection of terrestrial wild animals (State Council Decree 1992). Collecting protocols were approved by the Ethics Committee of the Sichuan Academy of Forestry (no specific permit number). Voucher specimens were deposited in the Sichuan Academy of Forestry, Chengdu, China.</p></div><div id="sec-4" class="subsection"><h3>Sample information</h3><p id="p-10">We included a total of 193 specimens of <em>Neodon</em> that collected in QTP for analyses. The collection comprised 54 juveniles and 139 adults, of which 99 had intact skulls and were used for statistical analysis of morphology. To verify the morphological identification and further investigate the evolutionary history of <em>Neodon</em>, we generated WGS data for a total of 48 specimens, representing 15 putative species of <em>Neodon</em>, four species of <em>Lasiopodomys</em>, three species of <em>Eothenomys</em>, one species of <em>Craseomys</em>, one species of <em>Caryomys</em>, one species of <em>Myodes</em>, and seven species of <em>Microtus</em> (Supplementary Table S1). All specimens were housed at the Sichuan Academy of Forestry (SAF).</p></div><div id="sec-5" class="subsection"><h3>Morphology analyses</h3><p id="p-11">Morphological evaluations for the 92 intact adult specimens identified, <em>N. clarkei</em>, <em>N. fuscus</em>, <em>N. irene</em>, <em>N. leucurus</em>, <em>N. linzhiensis</em>, <em>N. nyalamensus</em>, <em>N. medogensis</em> and <em>N. sikimensis</em>, plus one previously evaluated unnamed (<em>N. forresti</em>) and six tentatively new species. We also included an additional 41 broken adult skulls and 49 juvenile skulls of the putative six new species for comparison.</p><div id="sec-6" class="subsection"><h4>Measurement of morphological characteristics</h4><p id="p-12">For all the specimens, we collected the external, cranial, and dental characteristics of which the external measurements were recorded in the field on freshly captured specimens to an accuracy of 0.5 mm and the cranial and dental characteristics were measured using a Vernier caliper to an accuracy of 0.02 mm in lab. The measurements included previously used characteristics (Supplementary Table S2) (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Liu, et al. 2017</a>). For males, we recorded characteristics of genitalium, which provides significant clues to interrelationships of species (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hooper 1958</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hooper and Hart 1962</a>). We prepared the glans penes following the canonical methods (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hooper 1958</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lidicker Jr 1968</a>) and characterized bacular structures (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Hooper 1958</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yang and Fang 1988</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Yang, et al. 1992</a>) with several measurements (Supplementary Table S2).</p><p id="p-13">Morphometric variation of 17 non-gender-related measurements of adult specimens was analyzed using principal component analyses (PCAs) in SPSS v.17.0. We employed Kaiser-Meyer-Olkin (KMO) and Bartlett’s tests to check the fitness of the PCA analysis followed by Tukey’s test and independent samples <em>t</em>-tests to assess statistical differences.</p></div></div><div id="sec-7" class="subsection"><h3>High throughput sequencing (HTS)</h3><p id="p-14">We extracted genomic DNA for each specimen from muscle tissues using Gentra Puregene Tissus Kit (Qiagen, Valencia, CA) according to the manufacturer’s protocol, and then generated &gt; 10 Gb data for one representative specimen of each morphological species and about 3 Gb data (1X) for the other specimens from the same species. Low-quality reads of high-coverage specimens were removed if they met one or more following criteria: 1) an N-content of more than 10%; 2) adapter contaminated reads (reads overlapping more than 50% with the adapter sequence, with a maximum of 1 bp mismatches to the adaptor sequence); and 3) more than 20% of the read-length below Q10. The criteria of low-quality read-filtering of other specimens was put in Supplementary Table S1.</p><div id="sec-8" class="subsection"><h4>Mitochondrial genome assembly and annotation</h4><p id="p-15">We assembled and annotated the mitogenome for each sample using MitoZ (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Meng, et al. 2019</a>) with about 3 Gb (~1X) clean data. For <em>Neodon</em> and other closely-related genera, we also downloaded available cytochrome b (<em>cytb</em>) and cytochrome c oxidase subunit I (<em>cox1</em>) genes from GenBank (accessed in Oct. 2018), the two most widely used genetic marker for small mammals (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Jia, et al. 2012</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Liu, et al. 2017</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lv, et al. 2018</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Zhang, et al. 2016</a>).</p></div><div id="sec-9" class="subsection"><h4>Construction of orthologues data sets</h4><p id="p-16">To obtain orthologous nuclear genes for each sample (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>), we at first obtained 6,192 lineage-specific single-copy orthologs for the euarchontoglires group by using the Benchmarking Universal Single-Copy Orthologs (BUSCO) database (v9) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Simão, et al. 2015</a>). Then, we downloaded the genome of the North American deer mouse (Creicetidae, <em>Peromyscus maniculatus</em>), the most closely related species of <em>Neodon</em> available in the BUSCO database, from Ensemble (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hubbard, et al. 2002</a>) to obtain its corresponding full gene regions (including exons and introns). Next, genes with high similarity homologs (BLASTn v2.6.0+ with e value &lt; 1e-5) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul, et al. 1990</a>) within the current gene set were removed to avoid mapping uncertainty in the next step. Next, we aligned WGS data of each sample to the <em>Peromyscus maniculatus</em> genome using BWA-MEM (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li 2013</a>) with default parameters and obtained their corresponding genes using consensus calling function in bcftools v.1.8 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li, et al. 2009</a>) (detailed in Supplementary Appendix S1). Finally, high quality CDS—no internal stop codons, ‘N’ content smaller than 20% and present in more than 50% of the total samples—were filtered out for subsequent analyses. We firstly aligned their corresponding protein sequences using MAFFT v.7.313 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Katoh and Standley 2013</a>) and then achieved the CDS alignments using PAL2NAL (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Suyama, et al. 2006</a>) based on the protein alignments. Final alignments were given in Supplementary Appendices S2-S5, available on Dryad.</p></div><div id="sec-10" class="subsection"><h4>Genetic distance calculation</h4><p id="p-17">We calculated the Kimura 2-parameter genetic distances of each nuclear and mitochondrial gene using the dist.dna function in the R ape v.1.1-1 package (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Paradis, et al. 2004</a>). The average genetic distance of each taxonomic group (e.g. within species, within genus) was calculated.</p></div><div id="sec-11" class="subsection"><h4>Species Delimitation</h4><p id="p-18">In addition to morphological identification, we conducted species delimitation using two sequence-based methods: the clustering-based Bayesian implementation of the Poisson Tree Processes (bPTP) analysis (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhang, et al. 2013</a>) using both mitochondrial and nuclear trees and the similarity-based Automatic Barcode Gap Discovery (ABGD) analysis using mitochondrial data (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Puillandre, et al. 2012</a>).</p></div><div id="sec-12" class="subsection"><h4>Phylogenetic inference</h4><p id="p-19">We inferred the phylogeny using RAxML (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Stamatakis 2014</a>) with GTR+GAMMA+I model and 100 bootstrap replicates for each gene. Next, the final species tree was achieved using ASTRAL-III (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zhang, et al. 2018</a>) based on the multispecies coalescent model and the bootstrap support of each node was estimated by the multi-locus resampling method. SVDquartets (parameters of “eval Quartets=1e+6 bootstrap=standard”) implemented in PAUP v.4.0a164 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chifman and Kubatko 2014</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Swofford 2001</a>) was also utilized to estimate the species tree with the same dataset to validate the results. Simultaneously, we concatenated the CDS alignments to generate a “supergene” alignment for each species and used MrBayes v.3 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huelsenbeck and Ronquist 2001</a>) and RAxML to construct concatenated trees. Alignments included a “mitochondrial Gene Set” and “nuclear Gene Set”. Branch lengths of final species tree were re-estimated in units of substitutions per site by constraining alignments to the species tree topology using ExaML v.3.0.21 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kozlov, et al. 2015</a>). Trees were outgroup-rooted with species in <em>Eothenomys, Craseomys</em>, <em>Caryomys</em> and <em>Myodes</em>.</p><p id="p-20">The divergence time of the species tree were estimated based on the second codon sites of the nuclear genes using the Bayesian relaxed clock method MCMCTree implemented in the PAML v.4.9h package (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Yang 2007</a>) with the approximate likelihood calculation of ‘REV’ (GTR, model=7) model and with fossil calibration points taken from records in the Paleobiology Database (Available: <a href="https://paleobiodb.org">https://paleobiodb.org</a>, Accessed 2018 Dec 12) and the timetree database (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kumar, et al. 2017</a>) (detailed in Supplementary Appendix S1 and Table S3).</p></div></div><div id="sec-13" class="subsection"><h3>Evolutionary and biogeographic analyses</h3><div id="sec-14" class="subsection"><h4>Incomplete lineage sorting investigation</h4><p id="p-21">Because incomplete lineage sorting (ILS) may cause incongruence in phylogenetic trees inferred from the different datasets, we scanned for the presence of ILS spanning the evolution of <em>Neodon</em> with the nuclear data set of all 28 taxa (including outgroups) using DiscoVista (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sayyari, et al. 2018</a>). The correlation between ILS content and the inner-node branch length were calculated based on the linear models and Pearson test in R (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Field, et al. 2012</a>) and visualized using ggplot2 (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wickham 2016</a>).</p></div><div id="sec-15" class="subsection"><h4>Evolutionary rate analysis</h4><p id="p-22">To investigate the evolutionary rate of different clades within <em>Neodon</em> and the relationship to their corresponding living conditions, we calculated the evolutionary rates of mitochondrial genes and a nuclear gene set including 100 genes with top ASTRAL gene tree scores using the “several ω ratio” branch model (model = 2) implemented in the PAML package (v.4.9h) (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Yang 2007</a>) with the external and internal branches being set as foreground and background, respectively. The evolutionary rate of each branch was visualized using R package ggtree (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Yu, et al. 2017</a>).</p></div><div id="sec-16" class="subsection"><h4>Diversification rate analysis</h4><p id="p-23">To assess the diversification of <em>Neodon</em> through time, we generated Log-Lineage through time (LTT) plots for the time-calibrated phylogeny (non-<em>Neodon</em> species were pruned), as well as for 100 simulated trees of the same age and taxon richness, using Phytools (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Revell 2012</a>). For generating simulated trees, the ‘Yule’ (pure-birth) model and birth-death model were compared using Akaike information criterion (AIC). Next, 100 simulated trees were generated using ‘pbtree’ implemented in the Phytools under a Yule model and a constant rate model with the speciation rate of 0.487955 estimated using ‘fit.bd’.</p></div><div id="sec-17" class="subsection"><h4>Biogeographic analysis</h4><p id="p-24">Extended outgroups were pruned from the tree so that only <em>Neodon</em>, <em>Alexandormys</em>, <em>Microtus</em> and <em>Lasiopodomys</em> were analyzed. We used B<span class="sc">io</span>G<span class="sc">eo</span>B<span class="sc">ears</span> v.1.1.2 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Matzke 2013a</a>) for biogeographic reconstruction based on the species tree and assigned the species to one or two of the following biogeographical regions according to their distributions with the Tsangpo River and the Mekong-Salween rivers divide being used as the border for EH-H and H-HD, respectively: P (the QTP); EH (Eastern Himalayan Mountains); H (Himalayas); HD (Hengduan Mountains); O (area out of THR). The maximum range size was set to 2 because no extant species occurs in ≥ 3 biogeographical regions as defined here. We tested a total of six models of biogeographical reconstruction in the likelihood framework in B<span class="sc">io</span>G<span class="sc">eo</span>B<span class="sc">ears</span>, including dispersal‐extinction cladogenesis model (DEC) (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ree and Smith 2008</a>), dispersal–vicariance analysis (DIVA) (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ronquist 1997</a>) and the BayArea model (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Landis, et al. 2013</a>), plus all three models separately under the possibility of founder events (+J) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Matzke 2014</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Matzke 2013b</a>). AIC scores was employed to compare the fit of different models.</p></div><div id="sec-18" class="subsection"><h4>Data records</h4><p id="p-25">Data that support our findings were published under the International Nucleotide Sequence Database Collaboration BioProject PRJNA564473 (ncbi.nlm.nih.gov/bioproject/?term=PRJNA564473) and CNGB Nucleotide Sequence Archive (CNSA) project CNP0000173 (db.cngb.org/search/project/CNP0000173).</p></div></div></div><div class="section" id="sec-19"><h2 class="">R<span class="sc">esults</span></h2><div id="sec-20" class="subsection"><h3>Morphological comparison</h3><div id="sec-21" class="subsection"><h4>Comparisons of skull, teeth and bacular structures</h4><p id="p-26">Initial observations of skulls, teeth and bacular structures revealed 15 distinct patterns (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>–<a id="xref-fig-4-2" class="xref-fig" href="#F4">4</a>, Supplementary Fig. S1-2) each representing a putative species of <em>Neodon</em>. This included eight described species, one previously evaluated unnamed taxon and six tentatively new species. Skulls were compared in Supplementary Fig. 1 and molars and glans penes for all species of <em>Neodon</em> in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Molar patterns and morphology of glans penes clearly distinguished all unidentified species of <em>Neodon</em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1038904/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/1038904/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <p id="p-27" class="first-child">Morphology comparison of 15 species of <em>Neodon</em></p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PCA result. Unidentified Taxon 1: unidentified taxon from Bershula Mountains; Unidentified Taxon 2: unidentified taxon from Chayu County; Unidentified Taxon 3: unidentified taxon from southern Namchabarwa Mountains; Unidentified Taxon 4: unidentified taxon from north of Yarlung Zangbo River; Unidentified Taxon 5: unidentified taxon distributed between south of Yarlung Zangbo River and north of Namchabarwa Mountains; Unidentified Taxon 6: unidentified taxon from Bomi County." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1196413288" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;PCA result. Unidentified Taxon 1: unidentified taxon from Bershula Mountains; Unidentified Taxon 2: unidentified taxon from Chayu County; Unidentified Taxon 3: unidentified taxon from southern Namchabarwa Mountains; Unidentified Taxon 4: unidentified taxon from north of Yarlung Zangbo River; Unidentified Taxon 5: unidentified taxon distributed between south of Yarlung Zangbo River and north of Namchabarwa Mountains; Unidentified Taxon 6: unidentified taxon from Bomi County.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F3.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F3.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child">PCA result. Unidentified Taxon 1: unidentified taxon from Bershula Mountains; Unidentified Taxon 2: unidentified taxon from Chayu County; Unidentified Taxon 3: unidentified taxon from southern Namchabarwa Mountains; Unidentified Taxon 4: unidentified taxon from north of Yarlung Zangbo River; Unidentified Taxon 5: unidentified taxon distributed between south of Yarlung Zangbo River and north of Namchabarwa Mountains; Unidentified Taxon 6: unidentified taxon from Bomi County.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Divergence time tree, diversification patterns, ILS contents and morphologic photos for Neodon. a) Divergence time tree with the Astral branch supports number of nuclear gene tree were also showed near the branches. New species were marked in bold. Clades of Neodon were marked in shading with different colors on the tree (clade 1: yellow; clade 2: green; clade 3 purple). Branches with high ILS occurrence frequency (&gt;50%) was marked with the ILS content number. Log-Lineage-through-time (LTT) plots for Neodon were estimated from the time-calibrated phylogeny of Neodon (red curve) and the simulated trees under a Yule model (gray curve), and the semilucent red dashed line indicates the null distribution under a Yule process. b) The divergence time, geologic timescale, earth impacts, carbon dioxide, geologic events and climate events were showed at the bottom of the figure. c) Comparison of the tooth rows and glans penis of the Neodon species were showed in the right of the figure. Numbered views are 1: glans; 2: midventral cut view; 3: urethral lappet; 4: dorsal papilla (detailed in Supplementary Fig. S2). For N. linzhiensis, lettered structural features are: a. distal baculum; b. outer crater; c. inner crater; d. ventral groove; e. glans; f. prepuce; g. penis body; h. station of dorsal papilla; i. lateral baculum (cartilage); j. urethral lappet; k. lateral baculum (bony part); l. distal baculum (bony part); and m. proximal baculum (Jia, et al. 2012)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1196413288" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Divergence time tree, diversification patterns, ILS contents and morphologic photos for &lt;em&gt;Neodon&lt;/em&gt;. a) Divergence time tree with the Astral branch supports number of nuclear gene tree were also showed near the branches. New species were marked in bold. Clades of &lt;em&gt;Neodon&lt;/em&gt; were marked in shading with different colors on the tree (clade 1: yellow; clade 2: green; clade 3 purple). Branches with high ILS occurrence frequency (&gt;50%) was marked with the ILS content number. Log-Lineage-through-time (LTT) plots for &lt;em&gt;Neodon&lt;/em&gt; were estimated from the time-calibrated phylogeny of &lt;em&gt;Neodon&lt;/em&gt; (red curve) and the simulated trees under a Yule model (gray curve), and the semilucent red dashed line indicates the null distribution under a Yule process. b) The divergence time, geologic timescale, earth impacts, carbon dioxide, geologic events and climate events were showed at the bottom of the figure. c) Comparison of the tooth rows and glans penis of the &lt;em&gt;Neodon&lt;/em&gt; species were showed in the right of the figure. Numbered views are 1: glans; 2: midventral cut view; 3: urethral lappet; 4: dorsal papilla (detailed in Supplementary Fig. S2). For &lt;em&gt;N. linzhiensis&lt;/em&gt;, lettered structural features are: a. distal baculum; b. outer crater; c. inner crater; d. ventral groove; e. glans; f. prepuce; g. penis body; h. station of dorsal papilla; i. lateral baculum (cartilage); j. urethral lappet; k. lateral baculum (bony part); l. distal baculum (bony part); and m. proximal baculum (Jia, et al. 2012).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F4.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F4.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Divergence time tree, diversification patterns, ILS contents and morphologic photos for <em>Neodon</em>. a) Divergence time tree with the Astral branch supports number of nuclear gene tree were also showed near the branches. New species were marked in bold. Clades of <em>Neodon</em> were marked in shading with different colors on the tree (clade 1: yellow; clade 2: green; clade 3 purple). Branches with high ILS occurrence frequency (&gt;50%) was marked with the ILS content number. Log-Lineage-through-time (LTT) plots for <em>Neodon</em> were estimated from the time-calibrated phylogeny of <em>Neodon</em> (red curve) and the simulated trees under a Yule model (gray curve), and the semilucent red dashed line indicates the null distribution under a Yule process. b) The divergence time, geologic timescale, earth impacts, carbon dioxide, geologic events and climate events were showed at the bottom of the figure. c) Comparison of the tooth rows and glans penis of the <em>Neodon</em> species were showed in the right of the figure. Numbered views are 1: glans; 2: midventral cut view; 3: urethral lappet; 4: dorsal papilla (detailed in Supplementary Fig. S2). For <em>N. linzhiensis</em>, lettered structural features are: a. distal baculum; b. outer crater; c. inner crater; d. ventral groove; e. glans; f. prepuce; g. penis body; h. station of dorsal papilla; i. lateral baculum (cartilage); j. urethral lappet; k. lateral baculum (bony part); l. distal baculum (bony part); and m. proximal baculum (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Jia, et al. 2012</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h4>PCA</h4><p id="p-30">Taxonomic identifications were corroborated by the PCA analysis (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>). The analysis included 17 non-gender-related measurements of external, cranial and dental characteristics of adults. Fitness testing delivered a Kaiser-Meyer-Olkin value of 0.941 and a Bartlett’s test of &lt; 0.001; these demonstrated the robustness of the inference.</p><p id="p-31">The first two principal components (PCs) explained 82.129% of the total variance. Thirteen measurements (LM, ZB, SGL, MB, SBL, SH, LIL, ABL, MM, CBL, LMxT, LMbT and HBL) contributed 60.653% to PC1, and four measurements (TL, IOW, EL and HFL) contributed 21.476% to PC2. Specimens of <em>N. irene</em>, <em>N. fuscus</em>, <em>N. leucurus</em> clearly differed from all other specimens. <em>Neodon forresti</em> overlapped slightly with <em>N. linzhiensis</em> and these two species clearly separated from remaining taxa. Specimens of <em>N. sikimensis</em> separated clearly from congeners from north of Yarlung Zangbo River and specimens from north of Yarlung Zangbo River separated unambiguously from specimens distributed south of Namchabarwa Mountains. Specimens from south of Yarlung Zangbo River differed from those north of Yarlung Zangbo River and specimens from south of this river differed with those from south of Namchabarwa Mountains. Specimens from south of Yarlung Zangbo River and <em>N. sikimensis</em> slightly overlapped with each other. Among specimens having four closed triangles in the first lower molar, <em>N. fuscus</em> and specimens from Chayu County clearly separated from each other and all other specimens. Specimens from Bomi County mixed with <em>N. medogensis</em>. For the specimens that have five closed triangles in the first lower molar, <em>N. clarkei</em>, <em>N. linzhiensis</em> and specimens from the Bershula Mountains separated from each other distinctively. ANOVAs for the scores of PC1 (F = 45.720, P &lt; 0.001) and PC2 (F = 53.848, P &lt; 0.001) exhibited highly significant differences among species of <em>Neodon</em>.</p><p id="p-32">Tukey’s post hoc tests revealed that the PC1 or PC2 scores differentiated the six unidentified taxa. Among taxa that have 3 closed triangles on the first lower molar, the taxon from north of Yarlung Zangbo River differed significantly from the taxon from southern Namchabarwa Mountains (P &lt; 0.001), <em>N. irene</em> (P &lt; 0.001), <em>N. leucurus</em> (P &lt; 0.001), and <em>N. forresti</em> (P=0.07), but did not differ significantly from the taxon from south of Yarlung Zangbo River, <em>N. sikimensis</em> and <em>N. nyalamensis</em>. The taxon from south of Yarlung Zangbo River also differed significantly from the taxon from Namchabarwa Mountains (P &lt; 0.001), <em>N. irene</em> (P &lt; 0.001), <em>N. leucurus</em> (P &lt; 0.001), and <em>N. forresti</em> (P &lt; 0.001), but not from the taxon from north of Yarlung Zangbo River, <em>N. sikimensis</em> and <em>N. nyalamensis</em>. The taxon from the southern Namchabarwa Mountains differed significantly of all taxa that have 3 closed triangles on the first lower molar. For the taxa that have 5 closed triangles in the first lower molar, the taxon from Bershula Mountains differed significantly from <em>N. linzhiensis</em> (P = 0.003) and <em>N. clarkei</em> (P &lt; 0.001). For the taxa that have 4 closed triangles in the first lower molar, (a) the taxon from Chayu County differed significantly of <em>N. fuscus</em> (P &lt; 0.001), but from the taxon from Bomi County and <em>N. medogensis</em>. The taxon from Bomi County also differed significantly from <em>N. fuscus</em> (P &lt; 0.001). (4) For taxa that overlapped in PCA, <em>t</em>-tests were calculated and the results showed that at least 2 measurements had significant differences in one-to-one comparisons.</p></div></div><div id="sec-23" class="subsection"><h3>HTS data</h3><p id="p-33">We obtained a total of 620.45 Gb clean paired-end reads for the 47 samples with each having a data size ranging from 2.21 Gb to 43.38 Gb (Supplementary Table S1). All samples achieved full mitochondrial genomes that contained 13 protein coding genes (PCGs) and two rRNAs for phylogenetic analyses. 6,016 full-length single-copy orthologous genes that spanned a total length of 288,415,615 nucleotides were obtained from <em>Peromyscus maniculatus</em> and used for nuclear gene dataset construction. After the removal of the low confidential genes, we obtained 5,382 coding genes (the “nuclear gene set”), with an average, maximal, minimal length of 1,879, 18,018 and 60 nt, respectively, for all the samples (Supplementary Fig. S3 and Table S4).</p></div><div id="sec-24" class="subsection"><h3>Phylogenetic analysis</h3><div id="sec-25" class="subsection"><h4>Sequence divergence and species delimitation</h4><p id="p-34">For the mitochondrial genes, the results from <em>Neodon cox1</em> and <em>cytb</em> showed an average of 0.94% similarity, 1.11% for intra-species genetic distance and 11.00%, 11.30% for inter-species distance, among which, <em>N. sikimensis</em> showed the greatest intra-species genetic distances with an average of 5.23% (<em>cox1</em>) and 5.20% (<em>cytb</em>). Individuals of the taxon from Bomi County and <em>N. forresti</em> did not show any genetic variation (Supplementary Table S5), and the smallest congenic inter-species genetic distances of 3.55% (<em>cox1</em>) and 3.70% (<em>cytb</em>) occurred between taxa from Bomi and Chayu counties (Supplementary Table S6-8, Supplementary Fig. S4). The results of molecular-based species delimitation agreed with the morphology analyses (Supplementary Appendix S1).</p></div><div id="sec-26" class="subsection"><h4>Phylogenetic Relationship</h4><p id="p-35">The phylogeny for species of <em>Neodon</em> was inferred using concatenated methods MrBayes and RAxML, and coalescent-based ASTRAL-III and SVDquartets (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig4</a>, Supplementary Fig. S5-8). Trees produced by all phylogenetic analyses based on nuclear genes yielded same topologies with several small-scale incongruences compared to that based on mitochondrial genes (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig4</a>, Supplementary Fig. S4-6). Both genomes resolved three clades. The species-tree inferred using ASTRAL III with a normalized quartet score of 70.13% and high branch support (BS ≥ 92) from the most comprehensive dataset of 5,328 nuclear genes was used for the following analyses; more details on the other trees were placed in Supplementary Fig. S5-8.</p><p id="p-36">The first clade included five named taxa distributed in almost all of the HTR, especially in the eastern and western regions (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">4</a>, Supplementary Fig. S9a). The most wildly distributed species, <em>N. leucurus</em>, was the root taxon. The second clade included two described and three new taxa distributed mainly in the Himalayas (Supplementary Fig. S9b). <em>N. sikimensis</em> and <em>N. nyalamensis</em> were the root taxa and these specimens were collected near their type localities. The three new taxa occurred around the Yalung Zangbo River, Nachabarwa Mountains and Duoxiongla Peak area. Clade three, distributed in the eastern Himalayan and eastern HD mountains, included two described and three new species that were also collected from new sites: Bershula Mountains, Bomi County and Chayu County. Galongla Peak and the Gangrigabu Mountains separated the samples from Bomi and <em>N. medogensis</em> (Supplementary Fig. S9c).</p></div></div><div id="sec-27" class="subsection"><h3>Species nomination</h3><p id="p-37">Comparisons of molar pattern and glans penes, principal component analysis of morphology, gene distance as well as the phylogenetic analyses all confirmed that six unidentified taxa were new species that were not described literally. We nominated them as <em>Neodon shergylaensis</em> sp. nov. (unidentified taxon from north of Yarlung Zangbo River), <em>Neodon namchabarwaensis</em> sp. nov. (unidentified taxon from between south of Yarlung Zangbo River and north of Namchabarwa Mountains), <em>Neodon liaoruii</em> sp. nov. (unidentified taxon from southern Namchabarwa Mountains), <em>Neodon chayuensis</em> sp. nov. (unidentified taxon from Chayu county), <em>Neodon bomiensis</em> sp. nov. (unidentified taxon from Bomi county), and <em>Neodon bershulaensis</em> sp. nov. (unidentified taxon from Bershula Mountains). Details of description could be found in Supplementary Appendix S1.</p></div><div id="sec-28" class="subsection"><h3>Divergence time and diversification rate</h3><p id="p-38">Given the 95% credibility intervals of estimated date based on nuclear gene sets, the last common ancestor (LCA) of <em>Neodon</em> and outgroups lived in the late Miocene about 6.5 Mya (95% CI=7.7–5.5 Mya), and <em>Neodon</em> and <em>Lasiopodomys</em> diverged 3.4 Mya (3.8–3.1 Mya). <em>Neodon</em> experienced an explosive radiation in the late Neogene, and this coincided with changes in the Asian monsoons in the late Pliocene–early Pleistocene glacial event and uplifting events of the HD. The LTT plot suggested a deceleration in the rate of speciation after the initial radiation, and this was corroborated by the gamma statistic (γ= −3.6521, p &lt; 0.05) (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4</a>).</p><p id="p-39">Although the three clades existed in different regions with different climates, the Kruskal-Wallis test of the substitution ratio did not show significant differences between them (Supplementary Fig. S10, Table S9-10) revealing that they still had similar evolutionary rates among genes and were not significantly influenced by the conditions of their distributions.</p></div><div id="sec-29" class="subsection"><h3>Incomplete lineage sorting</h3><p id="p-40">Scans for the presence of ILS spanning the evolution of <em>Neodon</em> used 5,382 nuclear gene trees of 27 species (Supplementary Fig. S11a, b). ILS occurrence-frequency ranged from 1.49% to 64.55% for the 24 branches, while 37.50% of branches were affected by high levels of ILS (ILS rate &gt; 50%) (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4</a>). Thus, conflicting branches among gene trees were likely caused by high ILS content. A strong significant negative correlation was detected between the frequency of ILS occurrence and inner-node branch length (Pearson coefficient r = −0.98, P=6.12e-18) indicating that the high levels of ILS related to the rapid radiation (Supplementary Fig. S11c).</p></div><div id="sec-30" class="subsection"><h3>Ancestral range reconstruction</h3><p id="p-41">The ancestral area estimation was conducted using BioGeoBEARS with the best model DEC+J detected by AIC testing. The results (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>), with a j value of 0.043 and d value of 0.012 (Supplementary Table S11), indicated that (1) the common ancestors of <em>Neodon</em> were located in the QTP, followed in some lineages by rapid radiations, coinciding with recent climate change and uplifts of the HD; (2) several clades exhibited dispersal events to other regions, and thus dispersed out of the QTP.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ancestral range estimation results. a) Area delineation. b) Ancestral range estimation based on DEC + J model implemented in BioGeoBEARS." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1196413288" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Ancestral range estimation results. a) Area delineation. b) Ancestral range estimation based on DEC + J model implemented in BioGeoBEARS.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F5.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F5.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/2019.12.12.873679/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-42" class="first-child">Ancestral range estimation results. a) Area delineation. b) Ancestral range estimation based on DEC + J model implemented in BioGeoBEARS.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-31"><h2 class="">D<span class="sc">iscussion</span></h2><p id="p-43">Both the morphological and genetic data strongly support the long-term underestimated diversity in <em>Neodon</em>. Our sampling contributes six new species and confirms a total of 15 extant species of <em>Neodon</em>, which is much greater diversity than previously documented.</p><p id="p-44">Resolution of the number of closed triangles in the first lower molar (CTFLM) was thought to identify species of <em>Neodon</em> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Feng, et al. 1986</a>; <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Jia, et al. 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Luo 2000</a>). However, our analyses show its failure to do so because, for example, all specimens with three CTFLM (distributed in southern Tibet) were once regarded as <em>N. sikimensis</em>, but now includes at least five species: <em>N. sikimensis, N. nyalamensis, N. linzhiensis, N. liaoruii</em> sp. nov., <em>N. shergylaensis</em> sp. nov., and <em>N. namchabarwaensis</em> sp. nov. The genetic distances between these species range from 7.61% to 13.17% for <em>cytb</em> and 8.01% to 12.74% for <em>cox1</em>. Analyses obtain the same result for four and five CTFLMs (detailed in Supplementary Table 5-8). In addition to CTFLM, our results show that several other traits are critical for classifying species of Arvicoloni (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Jia, et al. 2012</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Liu, et al. 2017</a>) including external, cranial, dental and penis characters, which differ at various levels between species of <em>Neodon</em>, and especially glans penes and bacula (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig. 4</a>).</p><p id="p-45">Our phylogenetic analyses take advantage of thousands of nuclear genes as well as mitochondrial protein-coding genes and it is by far the largest molecular dataset for the study for small mammals. The results using coalescent and concatenation methods show good consistency and corroborate <em>Lasiopodomys</em> as the sister group of <em>Neodon</em>. They share a common ancestor with a clade consisting of <em>Microtus</em> and <em>Alexandromys</em>. While species in <em>Neodon</em> form three clades, the newly discovered species occur mainly in clades two and three (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig. 4</a>). Speciation in <em>Neodon</em> appears to have occurred in a very short time and resulted in at least 15 species. The rapid speciation also likely led to the negative correlation between percentage of ILS and inner-node branch length of the dated tree.</p><p id="p-46">Geographical isolation as a driver for speciation has been studied intensively in a myriad of taxa (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Coyne and Orr 2004</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Winger and Bates 2015</a>; <a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Xing and Ree 2017</a>). Our results demonstrate that several critical geographical events during the orogenesis of HD and repetitive climate changes with glacial cycles play crucial roles in the diversification of these voles. Global cooling, irreversible aridification of inland Asia and uplifting episodes of the HD coincide with the divergence of <em>Neodon</em> and <em>Lasiopodomys</em>; subsequent climate change and dispersal events and of <em>Neodon</em> led to the adaptive radiation. The coincidence of <em>Neodon</em>’s evolutionary history with the several climate changes and orogenic events of HD reveal that the QTP is the origin and evolution center for <em>Neodon</em>. This corresponds with the “out of QTP” hypothesis. The two most widely distributed species of <em>Neodon</em>, <em>N. leucurus</em> and <em>N. fuscus</em>, occupy root positions within <em>Neodon</em>, confirming <em>Neodon</em>’s origin in the QTP. Further, climate oscillations and the formation of rivers and mountains on the HD gave rise to physical boundaries for dispersal of <em>Neodon</em>, and subsequent speciation. This fits well with the concept of “sky island effects” (Supplementary Fig. S9). For instance, at the eastern THR (including HD and the eastern margins of QTP and HM) where mountain ranges or single summits served as glacial period sky island refugia (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Mosbrugger, et al. 2018</a>), <em>N. namchabarwaensis</em> sp. nov. and <em>N. liaoruii</em> sp. nov. appear to have speciated due to the obstruction posed by the Duoxiongla Mountain pass (4,200 m above sea level (a.s.l.)), and <em>N. medogensis</em> and <em>N. bomiensis</em> differentiated due to the barrier effect of Galongla Snow Mountain pass (4,200 m a.s.l.) (Supplementary Appendix S1). Finally, systematic sampling and integrative approaches are needed for other relatively sedentary species, such as small mammals, reptiles, amphibians, and others in the THR, to clarify their biodiversity.</p></div><div class="section" id="sec-33"><h2 class="">SUPPLEMENTARY MATERIAL</h2><p id="p-47">Supplementary materials and data files are available from Dryad data repository doi. Trees conducted by this study also could be found in <a href="http://purl.org/phylo/treebase/phylows/study/TB2:S25507">http://purl.org/phylo/treebase/phylows/study/TB2:S25507</a>.</p></div><div class="section" id="sec-34"><h2 class="">FUNDING</h2><p id="p-48">This work was supported by the National Natural Science Foundation of China (31470110, 31970399).</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-49">We thank Rui Liao for assistance in collecting specimens in the field. Special thanks to Yinjuan Mao and Junhua Bai for drawing the morphological figures.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.1"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Abramson</span>  <span class="cit-name-given-names">N</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Lebedev</span>  <span class="cit-name-given-names">V</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Bannikova</span>  <span class="cit-name-given-names">A</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Tesakov</span>  <span class="cit-name-given-names">A</span></span></li></ol><cite> editors. <abbr class="cit-jnl-abbrev">Doklady Biological Sciences</abbr>. <span class="cit-pub-date">2009</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">GM</span></span> <span class="cit-pub-date">1940</span>. <span class="cit-article-title">Mammals of China and Mongolia. Part 2. American Museum of Natural History</span>, <abbr class="cit-jnl-abbrev">Central Asiatic Expeditions 1921-1930</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">1350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCentral%2BAsiatic%2BExpeditions%2B1921-1930%26rft.volume%253D11%26rft.spage%253D621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.3" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">EW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">DJ</span></span> <span class="cit-pub-date">1990</span>. <span class="cit-article-title">Basic local alignment search tool</span>. <abbr class="cit-jnl-abbrev">Journal of Molecular Biology</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873679v1.4" data-doi="10.1038/35075035"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">E K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">L P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">C P</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Evolution of Asian monsoons and phased uplift of the Himalaya–Tibetan plateau since Late Miocene times</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D411%26rft.issue%253D6833%26rft.spage%253D62%26rft.epage%253D66%26rft.atitle%253DEvolution%2Bof%2BAsian%2Bmonsoons%2Band%2Bphased%2Buplift%2Bof%2Bthe%2BHimalaya-Tibetan%2BPlateau%2Bsince%2Blate%2BMiocene%2Btimes%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35075035%26rft_id%253Dinfo%253Apmid%252F11333976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35075035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2001042251&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=11333976&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168432800042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carleton</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Musser</span>  <span class="cit-name-given-names">GG</span></span> <span class="cit-pub-date">1995</span>. <span class="cit-article-title">Systematic Studies of Oryzomyine Rodents (Muridae, Sigmodontinae)-Definition and Distribution of Oligoryzomys-Vegetus (Bangs, 1902)</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Biological Society of Washington</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873679v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qing</span>  <span class="cit-name-given-names">J</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Formation of the Yarlung Zangbo Grand Canyon, Tibet, China</span>. <abbr class="cit-jnl-abbrev">Geological Bulletin of China</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeological%2BBulletin%2Bof%2BChina%26rft.volume%253D27%26rft.spage%253D491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.7" data-doi="10.1093/bioinformatics/btu530"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chifman</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kubatko</span>  <span class="cit-name-given-names">L</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Quartet inference from SNP data under the coalescent model</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3317</span>–<span class="cit-lpage">3324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu530%26rft_id%253Dinfo%253Apmid%252F25104814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu530&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25104814&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000345827400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Corbet</span>  <span class="cit-name-given-names">GB</span></span> <span class="cit-pub-date">1978</span>. <span class="cit-article-title">The mammals of the Palaearctic region: a taxonomic review</span>. <abbr class="cit-jnl-abbrev">British Museum (Natural History)</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Speciation Sinauer Associates</span>. <abbr class="cit-jnl-abbrev">Sunderland, MA</abbr>: <span class="cit-fpage">276</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellerman</span>  <span class="cit-name-given-names">J</span></span> <span class="cit-pub-date">1949</span>. <span class="cit-article-title">The Families and Genera of Living Rodents Volume III, Part I</span>. <abbr class="cit-jnl-abbrev">British Museum of Natural History, London</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.11"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-auth"> <span class="cit-name-given-names">RW</span> <span class="cit-name-surname">Hayman</span></span> &amp; </li><li><span class="cit-auth"> <span class="cit-name-given-names">GWC</span> <span class="cit-name-surname">Holt</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Ellerman</span>  <span class="cit-name-given-names">JR</span></span> <span class="cit-pub-date">1941</span>. <span class="cit-article-title">The Families and Genera of Living Rodents. With a List of Named Forms (1758-1936)</span>. by <span class="cit-auth"> <span class="cit-name-given-names">RW</span> <span class="cit-name-surname">Hayman</span></span> &amp; <span class="cit-auth"> <span class="cit-name-given-names">GWC</span> <span class="cit-name-surname">Holt</span></span>. Volume <span class="cit-vol">II</span>. <abbr class="cit-jnl-abbrev">Family Muridae. The Families and Genera of Living Rodents. With a List of Named Forms (1758-1936)</abbr>. <span class="cit-auth cit-collab">by RW HAYMAN &amp; GWC HOLT. Volume II. Family Muridae</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellerman</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morrison-Scott</span>  <span class="cit-name-given-names">TCS</span></span>. <span class="cit-pub-date">1951</span>. <span class="cit-article-title">Checklist of Palaearctic and Indian mammals</span>, <span class="cit-fpage">1758</span>–<span class="cit-lpage">1946</span>: <abbr class="cit-jnl-abbrev">British Museum (Natural History)</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Z-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">G-Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C-L</span></span> <span class="cit-pub-date">1986</span>. <span class="cit-article-title">The mammals of Xizang</span>. <abbr class="cit-jnl-abbrev">Science, Beijing</abbr>: <span class="cit-fpage">1</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miles</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">Z</span></span>. <span class="cit-pub-date">2012</span>. <abbr class="cit-jnl-abbrev">Discovering statistics using R: Sage publications</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gromov</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Polyakov</span>  <span class="cit-name-given-names">IY</span></span> <span class="cit-pub-date">1977</span>. <span class="cit-article-title">Mammals: Voles (Microtinae)</span>. <abbr class="cit-jnl-abbrev">Fauna SSSR</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFauna%2BSSSR%26rft.volume%253D3%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873679v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">A major reorganization of Asian climate regime by the Early Miocene</span>. <abbr class="cit-jnl-abbrev">Climate of the past Discussions</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClimate%2Bof%2Bthe%2Bpast%2BDiscussions%26rft.volume%253D4%26rft.spage%253D535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hooper</span>  <span class="cit-name-given-names">ET</span></span> <span class="cit-pub-date">1958</span>. <span class="cit-article-title">The male phallus in mice of the genus <em>Peromyscus</em></span>. <abbr class="cit-jnl-abbrev">MUSEUM OF ZOOLOGY, UNIVERSITY of MICHIGAN</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hooper</span>  <span class="cit-name-given-names">ET</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">BS</span></span> <span class="cit-pub-date">1962</span>. <span class="cit-article-title">A synopsis of recent North American microtine rodents</span>. <abbr class="cit-jnl-abbrev">MUSEUM OF ZOOLOGY, UNIVERSITY of MICHIGAN</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.19" data-doi="10.1093/nar/30.1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hubbard</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuff</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Curwen</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Down</span>  <span class="cit-name-given-names">T</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">The Ensembl genome database project</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHubbard%26rft.auinit1%253DT.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D41%26rft.atitle%253DThe%2BEnsembl%2Bgenome%2Bdatabase%2Bproject%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.1.38%26rft_id%253Dinfo%253Apmid%252F11752248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.1.38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11752248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173077100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.20" data-doi="10.1093/bioinformatics/17.8.754"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ronquist</span>  <span class="cit-name-given-names">F</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">MRBAYES: Bayesian inference of phylogenetic trees</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHuelsenbeck%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D754%26rft.epage%253D755%26rft.atitle%253DMRBAYES%253A%2BBayesian%2Binference%2Bof%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.8.754%26rft_id%253Dinfo%253Apmid%252F11524383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.8.754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11524383&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171021000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">KS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bartish</span>  <span class="cit-name-given-names">IV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">JQ</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Out of the Qinghai–Tibet Plateau: evidence for the origin and dispersal of Eurasian temperate plants from a phylogeographic study of Hippophaë rhamnoides (Elaeagnaceae)</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">1123</span>–<span class="cit-lpage">1133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DJia%26rft.auinit1%253DD.%2BR.%26rft.volume%253D194%26rft.issue%253D4%26rft.spage%253D1123%26rft.epage%253D1133%26rft.atitle%253DOut%2Bof%2Bthe%2BQinghai-Tibet%2BPlateau%253A%2Bevidence%2Bfor%2Bthe%2Borigin%2Band%2Bdispersal%2Bof%2BEurasian%2Btemperate%2Bplants%2Bfrom%2Ba%2Bphylogeographic%2Bstudy%2Bof%2BHippophae%2Brhamnoides%2B%2528Elaeagnaceae%2529.%26rft_id%253Dinfo%253Apmid%252F22432741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22432741&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.22" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">DM</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.23" data-doi="10.1093/bioinformatics/btv184"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kozlov</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aberer</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">ExaML version 3: a tool for phylogenomic analyses on supercomputers</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2577</span>–<span class="cit-lpage">2579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv184%26rft_id%253Dinfo%253Apmid%252F25819675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25819675&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.24" data-doi="doi:10.1093/molbev/msx116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suleski</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hedges</span>  <span class="cit-name-given-names">SB</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">TimeTree: a resource for timelines, timetrees, and divergence times</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1812</span>–<span class="cit-lpage">1819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D34%26rft.spage%253D1812%26rft_id%253Dinfo%253Adoi%252Fdoi%253A10.1093%252Fmolbev%252Fmsx116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi:10.1093/molbev/msx116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.25" data-doi="10.1093/sysbio/syt040"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Landis</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matzke</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>  <span class="cit-name-given-names">JP</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Bayesian analysis of biogeography when the number of areas is large</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">804</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyt040%26rft_id%253Dinfo%253Apmid%252F23736102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syt040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23736102&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM</span>. <abbr class="cit-jnl-abbrev">arXiv preprint</abbr><span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id-scheme">arXiv:</span><span class="cit-pub-id cit-pub-id-arxiv">1303.3997</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.27" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Homer</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">GT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">GR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.28" data-doi="10.2307/1378724"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lidicker</span>   <span class="cit-name-suffix">Jr</span>  <span class="cit-name-given-names">W</span></span> <span class="cit-pub-date">1968</span>. <span class="cit-article-title">A phylogeny of New Guinea rodent genera based on phallic morphology</span>. <abbr class="cit-jnl-abbrev">Journal of Mammalogy</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMammalogy%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1378724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1378724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">RW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Taxonomic position of Chinese voles of the tribe Arvicolini and the description of 2 new species from Xizang, China</span>. <abbr class="cit-jnl-abbrev">Journal of Mammalogy</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMammalogy%26rft.volume%253D98%26rft.spage%253D166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.30" data-doi="10.11646/zootaxa.3235.1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">SY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">ZY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">RW</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">A new vole from Xizang, China and the molecular phylogeny of the genus Neodon (Cricetidae: Arvicolinae)</span>. <abbr class="cit-jnl-abbrev">Zootaxa</abbr> <span class="cit-vol">3235</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZootaxa%26rft.volume%253D3235%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.11646%252Fzootaxa.3235.1.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.11646/zootaxa.3235.1.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873679v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hallet</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Quaternary glacier blocking events at the entrance of Yarlung Zangbo Great Canyon, Southeast Tibet</span>. <abbr class="cit-jnl-abbrev">Quaternary Sciences</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BSciences%26rft.volume%253D26%26rft.spage%253D52%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Rodentia Part III: Cricetidae</span>. <abbr class="cit-jnl-abbrev">Fauna Sinica Mammalia</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFauna%2BSinica%2BMammalia%26rft.volume%253D6%26rft.spage%253D121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Q</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Disjunct distribution and distinct intraspecific diversification of Eothenomys melanogaster in South China</span>. <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D18%26rft.spage%253D50%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lv</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Q</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Climatic niche conservatism and ecological opportunity in the explosive radiation of arvicoline rodents (Arvicolinae, Cricetidae)</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">1094</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D70%26rft.spage%253D1094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marchese</span>  <span class="cit-name-given-names">C</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Biodiversity hotspots: A shortcut for a more complicated concept</span>. <abbr class="cit-jnl-abbrev">Global Ecology and Conservation</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BEcology%2Band%2BConservation%26rft.volume%253D3%26rft.spage%253D297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matzke</span>  <span class="cit-name-given-names">NJ</span></span> <span class="cit-pub-date">2013a</span>. <span class="cit-article-title">BioGeoBEARS: BioGeography with Bayesian (and likelihood) evolutionary analysis in R Scripts</span>. <abbr class="cit-jnl-abbrev">R package, version 0.2</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">2013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2Bpackage%252C%2Bversion%2B0.2%26rft.volume%253D1%26rft.spage%253D2013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.37" data-doi="10.1093/sysbio/syu056"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matzke</span>  <span class="cit-name-given-names">NJ</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Model selection in historical biogeography reveals that founder-event speciation is a crucial process in island clades</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">951</span>–<span class="cit-lpage">970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyu056%26rft_id%253Dinfo%253Apmid%252F25123369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syu056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25123369&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matzke</span>  <span class="cit-name-given-names">NJ</span></span> <span class="cit-pub-date">2013b</span>. <span class="cit-article-title">Probabilistic historical biogeography: new models for founder-event speciation, imperfect detection, and fossils allow improved accuracy and model-testing</span>. <abbr class="cit-jnl-abbrev">Frontiers of Biogeography</abbr> <span class="cit-fpage">5</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">MitoZ: a toolkit for animal mitochondrial genome assembly, annotation and visualization</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Research</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">e63</span>–<span class="cit-lpage">e63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D47%26rft.spage%253De63%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.40" data-doi="10.1111/evo.12105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moreau</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">CD</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Testing the museum versus cradle tropical biological diversity hypothesis: phylogeny, diversification, and ancestral biogeographic range evolution of the ants</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">2240</span>–<span class="cit-lpage">2257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D2240%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12105%26rft_id%253Dinfo%253Apmid%252F23888848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23888848&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322329500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mosbrugger</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Favre</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muellner-Riehl</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Päckert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mulch</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Cenozoic evolution of geo-biodiversity in the Tibeto-Himalayan region</span>. <abbr class="cit-jnl-abbrev">Mountains, Climate, and Biodiversity</abbr>: <span class="cit-fpage">429</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Muellner-Riehl</span>  <span class="cit-name-given-names">AN</span></span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Mountains as evolutionary arenas: patterns, emerging approaches, paradigm shifts, and their implications for plant phylogeographic research in the Tibeto-Himalayan region</span>. <abbr class="cit-jnl-abbrev">Frontiers in Plant Science</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BPlant%2BScience%26rft.volume%253D10%26rft.spage%253D195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Musser</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carleton</span>  <span class="cit-name-given-names">M</span></span> <span class="cit-pub-date">2005</span>. <abbr class="cit-jnl-abbrev">Order rodentia</abbr>. <span class="cit-article-title">Mammal species of the world: a taxonomic and geographic reference 2</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.44" data-doi="10.1038/35002501"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mittermeier</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mittermeier</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Da Fonseca</span>  <span class="cit-name-given-names">GA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">J</span></span> <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Biodiversity hotspots for conservation priorities</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">403</span>: <span class="cit-fpage">853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMyers%26rft.auinit1%253DN.%26rft.volume%253D403%26rft.issue%253D6772%26rft.spage%253D853%26rft.epage%253D858%26rft.atitle%253DBiodiversity%2Bhotspots%2Bfor%2Bconservation%2Bpriorities.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35002501%26rft_id%253Dinfo%253Apmid%252F10706275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35002501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10706275&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085559200046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.45" data-doi="10.1093/bioinformatics/btg412"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paradis</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claude</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K</span></span> <span class="cit-pub-date">2004</span>. <span class="cit-article-title">APE: analyses of phylogenetics and evolution in R language</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DParadis%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D289%26rft.epage%253D290%26rft.atitle%253DAPE%253A%2BAnalyses%2Bof%2BPhylogenetics%2Band%2BEvolution%2Bin%2BR%2Blanguage%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg412%26rft_id%253Dinfo%253Apmid%252F14734327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg412&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14734327&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188389700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pisano</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Condamine</span>  <span class="cit-name-given-names">FL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lebedev</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bannikova</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quéré</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shenbrot</span>  <span class="cit-name-given-names">GI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pages</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michaux</span>  <span class="cit-name-given-names">JR</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Out of Himalaya: the impact of past Asian environmental changes on the evolutionary and biogeographical history of Dipodoidea (Rodentia)</span>. <abbr class="cit-jnl-abbrev">Journal of Biogeography</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">856</span>–<span class="cit-lpage">870</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiogeography%26rft.stitle%253DJournal%2Bof%2BBiogeography%26rft.aulast%253DPisano%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D5%26rft.spage%253D856%26rft.epage%253D870%26rft.atitle%253DOut%2Bof%2BHimalaya%253B%2Bthe%2Bimpact%2Bof%2Bpast%2BAsian%2Benvironmental%2Bchanges%2Bon%2Bthe%2Bevolutionary%2Band%2Bbiogeographical%2Bhistory%2Bof%2BDipodoidea%2B%2528Rodentia%2529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2015071120&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.47" data-doi="10.1111/j.1365-294X.2011.05239.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Puillandre</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brouillet</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Achaz</span>  <span class="cit-name-given-names">G</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">ABGD, Automatic Barcode Gap Discovery for primary species delimitation</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1864</span>–<span class="cit-lpage">1877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPuillandre%26rft.auinit1%253DN.%26rft.volume%253D21%26rft.issue%253D8%26rft.spage%253D1864%26rft.epage%253D1877%26rft.atitle%253DABGD%252C%2BAutomatic%2BBarcode%2BGap%2BDiscovery%2Bfor%2Bprimary%2Bspecies%2Bdelimitation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05239.x%26rft_id%253Dinfo%253Apmid%252F21883587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05239.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21883587&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302616200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873679v1.48" data-doi="10.1093/sysbio/syy032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Posterior summarization in Bayesian phylogenetics using Tracer 1.7</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">904</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.volume%253D67%26rft.spage%253D901%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyy032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syy032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.49" data-doi="10.1080/10635150701883881"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ree</span>  <span class="cit-name-given-names">RH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">SA</span></span> <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Maximum likelihood inference of geographic range evolution by dispersal, local extinction, and cladogenesis</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701883881%26rft_id%253Dinfo%253Apmid%252F18253896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150701883881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2009007777&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=18253896&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253657800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Revell</span>  <span class="cit-name-given-names">LJ</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">phytools: an R package for phylogenetic comparative biology (and other things)</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.51" data-doi="10.1093/sysbio/46.1.195"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>  <span class="cit-name-given-names">F</span></span> <span class="cit-pub-date">1997</span>. <span class="cit-article-title">Dispersal-vicariance analysis: a new approach to the quantification of historical biogeography</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252F46.1.195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/46.1.195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997WZ11000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.52" data-doi="10.1016/j.ympev.2018.01.019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sayyari</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitfield</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mirarab</span>  <span class="cit-name-given-names">S</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">DiscoVista: Interpretable visualizations of gene tree discordance</span>. <abbr class="cit-jnl-abbrev">Molecular Phylogenetics and Evolution</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%26rft.volume%253D122%26rft.spage%253D110%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2018.01.019%26rft_id%253Dinfo%253Apmid%252F29421312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2018.01.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29421312&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.53" data-doi="10.1644/10-MAMM-F-355.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sikes</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gannon</span>  <span class="cit-name-given-names">WL</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Guidelines of the American Society of Mammalogists for the use of wild mammals in research</span>. <abbr class="cit-jnl-abbrev">Journal of Mammalogy</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMammalogy%26rft_id%253Dinfo%253Adoi%252F10.1644%252F10-MAMM-F-355.1%26rft_id%253Dinfo%253Apmid%252F29692469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1644/10-MAMM-F-355.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29692469&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287898700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.54" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">EV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">EM</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.55" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.56" data-doi="10.1093/nar/gkl315"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Suyama</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torrents</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments</span>. <abbr class="cit-jnl-abbrev">Nucleic acids research</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W609</span>–<span class="cit-lpage">W612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSuyama%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW609%26rft.epage%253DW612%26rft.atitle%253DPAL2NAL%253A%2Brobust%2Bconversion%2Bof%2Bprotein%2Bsequence%2Balignments%2Binto%2Bthe%2Bcorresponding%2Bcodon%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl315%26rft_id%253Dinfo%253Apmid%252F16845082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845082&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245650200122&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Swofford</span>  <span class="cit-name-given-names">DL</span></span> <span class="cit-pub-date">2001</span>. <abbr class="cit-jnl-abbrev">Paup*: Phylogenetic analysis using parsimony (and other methods) 4.0. B5</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tamma</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">U</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Higher speciation and lower extinction rates influence mammal diversity gradients in Asia</span>. <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D15%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873679v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">E-q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L-z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z-l</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Tectonic and climatic element-controlled evolution of the Yalungzangbu River in southern Tibet</span>. <abbr class="cit-jnl-abbrev">Quaternary Sciences</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BSciences%26rft.volume%253D22%26rft.spage%253D365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.60" data-doi="10.1098/rspb.2014.0893"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">ZJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takeuchi</span>  <span class="cit-name-given-names">GT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">G</span></span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">From ‘third pole’to north pole: a Himalayan origin for the arctic fox</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20140893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2014.0893%26rft_id%253Dinfo%253Apmid%252F24920475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2014.0893&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24920475&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weigold</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2005</span>. <abbr class="cit-jnl-abbrev">Die biogeographie Tibets und seiner vorländer: Verein Sächsischer Ornithologen</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.12.873679v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-chapter-title">ggplot2: elegant graphics for data analysis</span>: <span class="cit-publ-name">Springer</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.63" data-doi="10.1111/evo.12607"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Winger</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">JM</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The tempo of trait divergence in geographic isolation: Avian speciation across the Marañon Valley of Peru</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D69%26rft.spage%253D772%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12607%26rft_id%253Dinfo%253Apmid%252F25611790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12607&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25611790&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.64" data-doi="10.1073/pnas.1616063114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ree</span>  <span class="cit-name-given-names">RH</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Uplift-driven diversification in the Hengduan Mountains, a temperate biodiversity hotspot</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">E3444</span>–<span class="cit-lpage">E3451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1616063114%26rft_id%253Dinfo%253Apmid%252F28373546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzE3L0UzNDQ0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMTMvMjAxOS4xMi4xMi44NzM2NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L</span></span> <span class="cit-pub-date">1988</span>. <span class="cit-article-title">Phallic morphology of 13 species of the family Muridae from China, with comments on its taxonomic significance</span>. <abbr class="cit-jnl-abbrev">Acta Theriologica Sinica</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BTheriologica%2BSinica%26rft.volume%253D4%26rft.spage%253D275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">L</span></span> <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Phallic morphology of eight species in Gerbillinae and Microtinae from China</span>. <abbr class="cit-jnl-abbrev">Acta Theriologica Sinica</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BTheriologica%2BSinica%26rft.volume%253D1%26rft.spage%253D31%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.67" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.68"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">TTY</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D8%26rft.spage%253D28%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.69" data-doi="10.1186/s12859-018-2129-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabiee</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sayyari</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mirarab</span>  <span class="cit-name-given-names">S</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">ASTRAL-III: polynomial time species tree reconstruction from partially resolved gene trees</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D19%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2129-y%26rft_id%253Dinfo%253Apmid%252F29745866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-018-2129-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29745866&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.70" data-doi="10.1093/bioinformatics/btt499"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kapli</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">A general species delimitation method with applications to phylogenetic placements</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2869</span>–<span class="cit-lpage">2876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt499%26rft_id%253Dinfo%253Apmid%252F23990417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt499&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23990417&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F2019.12.12.873679.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326643600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873679v1.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zong</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S</span></span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Characterization of the complete mitochondrial genome and phylogenetic relationship of Neodon sikimensis (Rodentia: Arvicolinae)</span>. <abbr class="cit-jnl-abbrev">Mitochondrial DNA Part B</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMitochondrial%2BDNA%2BPart%2BB%26rft.volume%253D1%26rft.spage%253D445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.12.873679v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.12.873679v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2019.12.13.875138v1" title="Recent hybridization and allopolyploidy reprogrammed Spartina microRNA expression under xenobiotic induced stress" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 13, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.12.873679v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2019.12.12.873679v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.12.873679v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.12.873679v1" class="active" data-icon-position="" data-hide-link-title="0">Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of Neodon voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1036085" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5SiVL_LdKQgjfgpN_cb5I_sBDs119nsmJ343IMGvFiw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1036085" id="node1036085--2" data-pisa="biorxiv;2019.12.12.873679v1" data-pisa-master="biorxiv;2019.12.12.873679" data-seqnum="1036085" data-apath="/biorxiv/early/2019/12/13/2019.12.12.873679.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of <em>Neodon</em> voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shaoying</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Chengran</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Wan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Guanliang</span> <span class="nlm-surname">Meng</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">W. Robert W.</span> <span class="nlm-surname">Murphy</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Zhengxin</span> <span class="nlm-surname">Fan</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Mingkun</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Yang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Zeng</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Shunde</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Yun</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Shanlin</span> <span class="nlm-surname">Liu</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.12.873679; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.12.873679 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.12.873679v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1&amp;title=Out%20of%20the%20Qinghai-Tibetan%20Plateau%20and%20get%20flourishing%20-%20the%20evolution%20of%20Neodon%20voles%20%28Rodentia%3A%20Cricetidae%29%20revealed%20by%20systematic%20sampling%20and%20low%20coverage%20whole%20genome%20sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1&amp;title=Out%20of%20the%20Qinghai-Tibetan%20Plateau%20and%20get%20flourishing%20-%20the%20evolution%20of%20Neodon%20voles%20%28Rodentia%3A%20Cricetidae%29%20revealed%20by%20systematic%20sampling%20and%20low%20coverage%20whole%20genome%20sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1&amp;text=Out%20of%20the%20Qinghai-Tibetan%20Plateau%20and%20get%20flourishing%20-%20the%20evolution%20of%20Neodon%20voles%20%28Rodentia%3A%20Cricetidae%29%20revealed%20by%20systematic%20sampling%20and%20low%20coverage%20whole%20genome%20sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1&amp;title=Out%20of%20the%20Qinghai-Tibetan%20Plateau%20and%20get%20flourishing%20-%20the%20evolution%20of%20Neodon%20voles%20%28Rodentia%3A%20Cricetidae%29%20revealed%20by%20systematic%20sampling%20and%20low%20coverage%20whole%20genome%20sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1&amp;t=Out%20of%20the%20Qinghai-Tibetan%20Plateau%20and%20get%20flourishing%20-%20the%20evolution%20of%20Neodon%20voles%20%28Rodentia%3A%20Cricetidae%29%20revealed%20by%20systematic%20sampling%20and%20low%20coverage%20whole%20genome%20sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1&amp;title=Out%20of%20the%20Qinghai-Tibetan%20Plateau%20and%20get%20flourishing%20-%20the%20evolution%20of%20Neodon%20voles%20%28Rodentia%3A%20Cricetidae%29%20revealed%20by%20systematic%20sampling%20and%20low%20coverage%20whole%20genome%20sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1&amp;title=Out%20of%20the%20Qinghai-Tibetan%20Plateau%20and%20get%20flourishing%20-%20the%20evolution%20of%20Neodon%20voles%20%28Rodentia%3A%20Cricetidae%29%20revealed%20by%20systematic%20sampling%20and%20low%20coverage%20whole%20genome%20sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1036085" id="node1036085--3" data-pisa="biorxiv;2019.12.12.873679v1" data-pisa-master="biorxiv;2019.12.12.873679" data-seqnum="1036085" data-apath="/biorxiv/early/2019/12/13/2019.12.12.873679.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Out of the Qinghai-Tibetan Plateau and get flourishing - the evolution of <em>Neodon</em> voles (Rodentia: Cricetidae) revealed by systematic sampling and low coverage whole genome sequencing</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Shaoying</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Chengran</span> <span class="nlm-surname">Zhou</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Wan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Guanliang</span> <span class="nlm-surname">Meng</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">W. Robert W.</span> <span class="nlm-surname">Murphy</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Zhengxin</span> <span class="nlm-surname">Fan</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Mingkun</span> <span class="nlm-surname">Tang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Yang</span> <span class="nlm-surname">Liu</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Zeng</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Shunde</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Yun</span> <span class="nlm-surname">Zhao</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Shanlin</span> <span class="nlm-surname">Liu</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.12.873679; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.12.873679 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Out%20of%20the%20Qinghai-Tibetan%20Plateau%20and%20get%20flourishing%20-%20the%20evolution%20of%20Neodon%20voles%20%28Rodentia%3A%20Cricetidae%29%20revealed%20by%20systematic%20sampling%20and%20low%20coverage%20whole%20genome%20sequencing&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Liu%2C+Shaoying&amp;rft.au=Zhou%2C+Chengran&amp;rft.au=Wan%2C+Tao&amp;rft.au=Meng%2C+Guanliang&amp;rft.au=Murphy%2C+W.+Robert+W.&amp;rft.au=Fan%2C+Zhengxin&amp;rft.au=Tang%2C+Mingkun&amp;rft.au=Liu%2C+Yang&amp;rft.au=Zeng%2C+Tao&amp;rft.au=Chen%2C+Shunde&amp;rft.au=Zhao%2C+Yun&amp;rft.au=Liu%2C+Shanlin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1036085/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1036085/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1036085/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1036085/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1036085/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1036085/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1036085/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1036085/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1036085/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1036085/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1036085/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1036085/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1&amp;count=horizontal&amp;via=&amp;text=Out%20of%20the%20Qinghai-Tibetan%20Plateau%20and%20get%20flourishing%20-%20the%20evolution%20of%20Neodon%20voles%20%28Rodentia%3A%20Cricetidae%29%20revealed%20by%20systematic%20sampling%20and%20low%20coverage%20whole%20genome%20sequencing&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873679v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.12.873679v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/zoology" class="highlight" data-icon-position="" data-hide-link-title="0">Zoology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
