<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/embed/graphic-5.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/10/24/817411.atom" />
<meta name="HW.pisa" content="biorxiv;817411v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Climatic niche predicts the landscape structure of locally adaptive standing genetic variation" />
<meta name="DC.Identifier" content="10.1101/817411" />
<meta name="DC.Date" content="2019-10-24" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Within a species’ range, intraspecific diversity in the form of adaptive standing genetic variation (SGV) may be non-randomly clustered into different geographic regions, reflecting the combined effects of historical range movements and spatially-varying natural selection. As a consequence of a patchy distribution of adaptive SGV, populations in different parts of the range are likely to vary in their capacity to respond to changing selection pressures, especially long-lived sessile organisms like forest trees. However, the spatial distribution of adaptive SGV across the landscape is rarely considered when predicting species responses to environmental change. Here, we use a landscape genomics approach to estimate the distribution of adaptive SGV along spatial gradients reflecting the expansion history and contemporary climatic niche of balsam poplar, Populus balsamifera (Salicaceae), a widely distributed forest tree with a transcontinental distribution in North America. By scanning the genome for signatures of spatially varying local adaptation, we estimated how adaptive SGV has been shaped by geographic distance from the rear range edge (expansion history) versus proximity to the current center of the climatic niche (environmental selection). We found that adaptive SGV was strongly structured by the current climatic niche, with surprisingly little importance attributable to historical effects such as migration out of southern refugia. As expected, the effect of the climatic niche on SGV was strong for genes whose expression is responsive to abiotic stress (drought), although genes upregulated under biotic (wounding) stress also contained SGV that followed climatic and latitudinal gradients. The latter result could reflect parallel selection pressures, or co-regulation of functional pathways involved in both abiotic and biotic stress responses. Our study in balsam poplar suggests that clustering of locally adaptive SGV within ranges primarily reflects spatial proximity within the contemporary climatic niche – an important consideration for the design of effective strategies for biodiversity conservation and avoidance of maladaptation under climate change." />
<meta name="DC.Contributor" content="Vikram E. Chhatre" />
<meta name="DC.Contributor" content="Karl C. Fetter" />
<meta name="DC.Contributor" content="Andrew V. Gougherty" />
<meta name="DC.Contributor" content="Matthew C. Fitzpatrick" />
<meta name="DC.Contributor" content="Raju Y. Soolanayakanahally" />
<meta name="DC.Contributor" content="Ronald S. Zalesny" />
<meta name="DC.Contributor" content="Stephen R. Keller" />
<meta name="article:published_time" content="2019-10-24" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Climatic niche predicts the landscape structure of locally adaptive standing genetic variation" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Within a species’ range, intraspecific diversity in the form of adaptive standing genetic variation (SGV) may be non-randomly clustered into different geographic regions, reflecting the combined effects of historical range movements and spatially-varying natural selection. As a consequence of a patchy distribution of adaptive SGV, populations in different parts of the range are likely to vary in their capacity to respond to changing selection pressures, especially long-lived sessile organisms like forest trees. However, the spatial distribution of adaptive SGV across the landscape is rarely considered when predicting species responses to environmental change. Here, we use a landscape genomics approach to estimate the distribution of adaptive SGV along spatial gradients reflecting the expansion history and contemporary climatic niche of balsam poplar, &lt;i&gt;Populus balsamifera&lt;/i&gt; (Salicaceae), a widely distributed forest tree with a transcontinental distribution in North America. By scanning the genome for signatures of spatially varying local adaptation, we estimated how adaptive SGV has been shaped by geographic distance from the rear range edge (expansion history) versus proximity to the current center of the climatic niche (environmental selection). We found that adaptive SGV was strongly structured by the current climatic niche, with surprisingly little importance attributable to historical effects such as migration out of southern refugia. As expected, the effect of the climatic niche on SGV was strong for genes whose expression is responsive to abiotic stress (drought), although genes upregulated under biotic (wounding) stress also contained SGV that followed climatic and latitudinal gradients. The latter result could reflect parallel selection pressures, or co-regulation of functional pathways involved in both abiotic and biotic stress responses. Our study in balsam poplar suggests that clustering of locally adaptive SGV within ranges primarily reflects spatial proximity within the contemporary climatic niche – an important consideration for the design of effective strategies for biodiversity conservation and avoidance of maladaptation under climate change.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;817411v1" />
<meta name="citation_id" content="817411v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/817411v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/817411v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/817411v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411.full.pdf" />
<meta name="citation_doi" content="10.1101/817411" />
<meta name="citation_num_pages" content="38" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="817411" />
<meta name="citation_author" content="Vikram E. Chhatre" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Vermont" />
<meta name="citation_author_institution" content="Wyoming INBRE Bioinformatics Core, University of Wyoming" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6627-2832" />
<meta name="citation_author" content="Karl C. Fetter" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Vermont" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9234-9300" />
<meta name="citation_author" content="Andrew V. Gougherty" />
<meta name="citation_author_institution" content="University of Maryland Center for Environmental Science" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3905-8539" />
<meta name="citation_author" content="Matthew C. Fitzpatrick" />
<meta name="citation_author_institution" content="University of Maryland Center for Environmental Science" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1911-8407" />
<meta name="citation_author" content="Raju Y. Soolanayakanahally" />
<meta name="citation_author_institution" content="Indian Head Research Farm, Agriculture and Agri-Food Canada" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9345-9640" />
<meta name="citation_author" content="Ronald S. Zalesny" />
<meta name="citation_author_institution" content="USDA Forest Service, Northern Research Station, Institute for Applied Ecosystem Studies" />
<meta name="citation_author" content="Stephen R. Keller" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Vermont" />
<meta name="citation_author_email" content="srkeller@uvm.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8887-9213" />
<meta name="citation_reference" content="Fisher, R. A. The genetical theory of natural selection (Oxford University Press, Oxford, UK, 1930)." />
<meta name="citation_reference" content="Orr, H. A. The genetic theory of adaptation: a brief history. Nature Reviews Genetics 6, 119 (2005)." />
<meta name="citation_reference" content="Charlesworth, B. The evolutionary biology of sex. Current Biology 16, R693–5 (2006)." />
<meta name="citation_reference" content="Delph, L. F. &amp; Kelly, J. K. On the importance of balancing selection in plants. New Phytologist 201, 45–56 (2014)." />
<meta name="citation_reference" content="Mauricio, R. et al. Natural selection for polymorphism in the disease resistance gene Rps2 of Arabidopsis thaliana. Genetics 163, 735–746 (2003)." />
<meta name="citation_reference" content="Fumagalli, M. et al. A population genetics study of the familial Mediterranean fever gene: evidence of balancing selection under an overdominance regime. Genes and Immunity 10, 678 (2009)." />
<meta name="citation_reference" content="Gos, G., Slotte, T. &amp; Wright, S. I. Signatures of balancing selection are maintained at disease resistance loci following mating system evolution and a population bottleneck in the genus Capsella. BMC Evolutionary Biology 12, 152 (2012)." />
<meta name="citation_reference" content="Hancock, A. M. et al. Human adaptations to diet, subsistence, and ecoregion are due to subtle shifts in allele frequency. Proceedings of the National Academy of Sciences 107, 8924–8930 (2010)." />
<meta name="citation_reference" content="Messer, P. W. &amp; Petrov, D. A. Population genomics of rapid adaptation by soft selective sweeps. Trends in Ecology &amp; Evolution 28, 659–669 (2013)." />
<meta name="citation_reference" content="Hewitt, G. The genetic legacy of the Quaternary ice ages. Nature 405, 907–913 (2000)." />
<meta name="citation_reference" content="Excoffier, L., Foll, M. &amp; Petit, R. J. Genetic consequences of range expansions. Annual Review of Ecology, Evolution, and Systematics 40, 481–501 (2009)." />
<meta name="citation_reference" content="Hampe, A. &amp; Jump, A. S. Climate relicts: past, present, future. Annual Review of Ecology, Evolution, and Systematics 42, 313–333 (2011)." />
<meta name="citation_reference" content="Pironon, S. et al. Geographic variation in genetic and demographic performance: new insights from an old biogeographical paradigm. Biological Reviews 92, 1877–1909 (2017)." />
<meta name="citation_reference" content="Hampe, A. &amp; Petit, R. J. Conserving biodiversity under climate change: the rear edge matters. Ecology Letters 8, 461–467 (2005)." />
<meta name="citation_reference" content="Rehm, E. M., Olivas, P., Stroud, J. &amp; Feeley, K. J. Losing your edge: climate change and the conservation value of range-edge populations. Ecology and Evolution 5, 4315–4326 (2015)." />
<meta name="citation_reference" content="Slatkin, M. &amp; Excoffier, L. Serial founder effects during range expansion: a spatial analog of genetic drift. Genetics 191, 171–181 (2012)." />
<meta name="citation_reference" content="Pujol, B. &amp; Pannell, J. R. Reduced responses to selection after species range expansion. Science 321, 96–96 (2008)." />
<meta name="citation_reference" content="Louthan, A. M., Doak, D. F. &amp; Angert, A. L. Where and when do species interactions set range limits. Trends in Ecology &amp; Evolution 30, 780–792 (2015)." />
<meta name="citation_reference" content="Chhatre, V. E., Evans, L. M., DiFazio, S. P. &amp; Keller, S. R. Adaptive introgression and maintenance of a trispecies hybrid complex in range-edge populations of Populus. Molecular Ecology 27, 4820–4838 (2018)." />
<meta name="citation_reference" content="Fetter, K. C., Nelson, D. M. &amp; Keller, S. R. Trade-offs and selection conflicts in hybrid poplars indicate the stomatal ratio as an important trait regulating disease resistance. bioRxiv, 814046 (2019)." />
<meta name="citation_reference" content="Schoville, S. D. et al. Adaptive genetic variation on the landscape: methods and cases. Annual Review of Ecology, Evolution, and Systematics 43, 23–43 (2012)." />
<meta name="citation_reference" content="Macdonald, S. L., Llewelyn, J., Moritz, C. &amp; Phillips, B. L. Peripheral isolates as sources of adaptive diversity under climate change. Frontiers in Ecology and Evolution 5, 88 (2017)." />
<meta name="citation_reference" content="Antonovics, J. The nature of limits to natural selection. Annals of the Missouri Botanical Garden, 224–247 (1976)." />
<meta name="citation_reference" content="Bridle, J. R. &amp; Vines, T. H. Limits to evolution at range margins: when and why does adaptation fail. Trends in Ecology &amp; Evolution 22, 140–147 (2007)." />
<meta name="citation_reference" content="Mimura, M. &amp; Aitken, S. N. Adaptive gradients and isolation-by-distance with postglacial migration in Picea sitchensis. Heredity 99, 224–232 (2007)." />
<meta name="citation_reference" content="Sexton, J. P., Strauss, S. Y. &amp; Rice, K. J. Gene flow increases fitness at the warm edge of a species’ range. Proceedings of the National Academy of Sciences 108, 11704–11709 (2011)." />
<meta name="citation_reference" content="Holliday, J. A., Suren, H. &amp; Aitken, S. N. Divergent selection and heterogeneous migration rates across the range of Sitka spruce (Picea sitchensis). Proceedings of the Royal Society B 279, 1675–1683 (2012)." />
<meta name="citation_reference" content="Gould, B. et al. Local adaptation and range boundary formation in response to complex environmental gradients across the geographical range of Clarkia xantiana ssp. xantiana. Journal of Ecology 102, 95–107 (2014)." />
<meta name="citation_reference" content="De Lafontaine, G., Napier, J. D., Petit, R. J. &amp; Hu, F. S. Invoking adaptation to decipher the genetic legacy of past climate change. Ecology 99, 1530–1546 (2018)." />
<meta name="citation_reference" content="Schiffers, K. et al. Landscape structure and genetic architecture jointly impact rates of niche evolution. Ecography 37, 1218–1229 (2014)." />
<meta name="citation_reference" content="Polechová, J. &amp; Barton, N. H. Limits to adaptation along environmental gradients. Proceedings of the National Academy of Sciences 112, 6401–6406 (2015)." />
<meta name="citation_reference" content="Kirkpatrick, M. &amp; Barton, N. H. Evolution of a species’ range. The American Naturalist 150, 1–23 (1997)." />
<meta name="citation_reference" content="Holt, R. D. &amp; Barfield, M. Theoretical perspectives on the statics and dynamics of species’ borders in patchy environments. The American Naturalist 178 Suppl 1, S6–25 (2011)." />
<meta name="citation_reference" content="Chhatre, V. E. &amp; Rajora, O. P. Genetic divergence and signatures of natural selection in marginal populations of a keystone, long-lived conifer, eastern white pine (Pinus strobus) from Northern Ontario. PLoS ONE 9(5), e97291 (2014)." />
<meta name="citation_reference" content="Bellon, M. R., Dulloo, E., Sardos, J., Thormann, I. &amp; Burdon, J. J. In situ conservationharnessing natural and human-derived evolutionary forces to ensure future crop adaptation. Evolutionary Applications 10, 965–977 (2017)." />
<meta name="citation_reference" content="Garcia, R. A., Cabeza, M., Rahbek, C. &amp; Araüjo, M. B. Multiple dimensions of climate change and their implications for biodiversity. Science 344, 1247579 (2014)." />
<meta name="citation_reference" content="National Academies of Sciences, E. &amp; Medicine, Forest Health and Biotechnology: Possibilities and Considerations https://doi.org/10.17226/25221 (National Academies Press, 2019)." />
<meta name="citation_reference" content="Davis, M. B. &amp; Shaw, R. G. Range shifts and adaptive responses to Quaternary climate change. Science 292, 673–679 (2001)." />
<meta name="citation_reference" content="Gavin, D. G. et al. Climate refugia: joint inference from fossil records, species distribution models and phylogeography. New Phytologist 204, 37–54 (2014)." />
<meta name="citation_reference" content="Petit, R. J. &amp; Hampe, A. Some evolutionary consequences of being a tree. Annual Review of Ecology, Evolution, and Systematics 37, 187–214 (2006)." />
<meta name="citation_reference" content="Savolainen, O., Pyhäjärvi, T. &amp; Knürr, T. Gene flow and local adaptation in trees. Annual Review of Ecology, Evolution, and Systematics 38, 595–619 (2007)." />
<meta name="citation_reference" content="Eckert, A. J. et al. Patterns of population structure and environmental associations to aridity across the range of loblolly pine (Pinus taeda L., Pinaceae). Genetics. https://doi.org/10.1534/genetics.110.115543 (2010)." />
<meta name="citation_reference" content="Keller, S. R., Levsen, N., Olson, M. S. &amp; Tiffin, P. Local adaptation in the flowering-time gene network of balsam poplar, Populus balsamifera L. Molecular Biology and Evolution 29, 3143–3152 (2012)." />
<meta name="citation_reference" content="Sork, V. L. et al. Landscape genomic analysis of candidate genes for climate adaptation in a California endemic oak, Quercus lobata. American Journal of Botany 103, 33–46 (2016)." />
<meta name="citation_reference" content="Yeaman, S. et al. Convergent local adaptation to climate in distantly related conifers. Science 353, 1431–1433 (2016)." />
<meta name="citation_reference" content="Sork, V. et al. Putting the landscape into the genomics of trees: approaches for understanding local adaptation and population responses to changing climate. Tree Genetics &amp; Genomes 9, 901–911 (2013)." />
<meta name="citation_reference" content="Little, E. L. Atlas of United States trees. Volume 1. Conifers and important hardwoods. Miscellaneous publication 1146. US Department of Agriculture, Forest Service, Washington, DC (1971)." />
<meta name="citation_reference" content="Keller, S. R., Chhatre, V. E. &amp; Fitzpatrick, M. C. Influence of range position on locally adaptive gene–environment associations in Populus flowering time genes. Journal of Heredity 109, 47–58 (2017)." />
<meta name="citation_reference" content="Keller, S. R., Olson, M. S., Silim, S., Schroeder, W. &amp; Tiffin, P. Genomic diversity, population structure, and migration following rapid range expansion in the Balsam Poplar, Populus balsamifera. Molecular Ecology 19, 1212–1226 (2010)." />
<meta name="citation_reference" content="Levsen, N. D., Tiffin, P. &amp; Olson, M. S. Pleistocene speciation in the genus Populus (Salicaceae). Systematic Biology 61, 401–412 (2012)." />
<meta name="citation_reference" content="Soolanayakanahally, R. Y., Guy, R. D., Silim, S. N., Drewes, E. C. &amp; Schroeder, W. R. Enhanced assimilation rate and water use efficiency with latitude through increased photosynthetic capacity and internal conductance in balsam poplar (Populus balsamifera L. Plant, Cell &amp; Environment 32, 1821–1832 (2009)." />
<meta name="citation_reference" content="Keller, S. R. et al. Climate-driven local adaptation of ecophysiology and phenology in balsam poplar, Populus balsamifera L. (Salicaceae). American Journal of Botany 98, 99–108 (2011)." />
<meta name="citation_reference" content="Olson, M. S. et al. The adaptive potential of Populus balsamifera L. to phenology requirements in a warmer global climate. Molecular Ecology 22, 1214–1230 (2013)." />
<meta name="citation_reference" content="Keller, S. R., Levsen, N., Ingvarsson, P. K., Olson, M. S. &amp; Tiffin, P. Local selection across a latitudinal gradient shapes nucleotide diversity in balsam poplar, Populus balsamifera L. Genetics 188, 941–952 (2011)." />
<meta name="citation_reference" content="Glaubitz, J. C. et al. TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline. PLoS One 9, e90346 (2014)." />
<meta name="citation_reference" content="Bonin, A., Nicole, F., Pompanon, F., Miaud, C. &amp; Taberlet, P. Population adaptive index: a new method to help measure intraspecific genetic diversity and prioritize populations for conservation. Conservation Biology 21, 697–708 (2007)." />
<meta name="citation_reference" content="Sjdin, A., Street, N. R., Sandberg, G., Gustafsson, P. &amp; Jansson, S. The Populus Genome Integrative Explorer (PopGenIE): a new resource for exploring the Populus genome. New Phytologist 182, 1013–1025 (2009)." />
<meta name="citation_reference" content="Eckert, C., Samis, K. &amp; Lougheed, S. Genetic variation across species geographical ranges: the central–marginal hypothesis and beyond. Molecular ecology 17, 1170–1188 (2008)." />
<meta name="citation_reference" content="Sork, V. L. Genomic studies of local adaptation in natural plant populations. Journal of Heredity 109, 3–15 (2017)." />
<meta name="citation_reference" content="Fetter, K. C., Gugger, P. F. &amp; Keller, S. R. Landscape Genomics of Angiosperm Trees: From Historic Roots to Discovering New Branches of Adaptive Evolution in Comparative and Evolutionary Genomics of Angiosperm Trees 303–333 (Springer, 2017). https://doi.org/10.1007/7397201619." />
<meta name="citation_reference" content="Travisano, M., Mongold, J. A., Bennett, A. F. &amp; Lenski, R. E. Experimental tests of the roles of adaptation, chance, and history in evolution. Science 267, 87–90 (1995)." />
<meta name="citation_reference" content="Keller, S. R. &amp; Taylor, D. R. History, chance and adaptation during biological invasion: separating stochastic phenotypic evolution from response to selection. Ecology Letters 11, 852–866 (2008)." />
<meta name="citation_reference" content="Gougherty A, C. V.K. S. &amp; Fitzpatrick, M. Contemporary range position predicts the range-wide pattern of genetic diversity in balsam poplar, Populus blasmiafera L.) (unpublished ms)." />
<meta name="citation_reference" content="Meirmans, P. G., Godbout, J., Lamothe, M., Thompson, S. L. &amp; Isabel, N. History rather than hybridization determines population structure and adaptation in Populus balsamifera. Journal of Evolutionary Biology 30, 2044–2058 (2017)." />
<meta name="citation_reference" content="Schuler, M. A. &amp; Werck-Reichhart, D. Functional Genomics of P450S. Annual Review of Plant Biology 54. PMID: 14503006, 629–667 (2003)." />
<meta name="citation_reference" content="Nelson, D. &amp; Werck-Reichhart, D. A P450-centric view of plant evolution. The Plant Journal 66, 194–211 (2011)." />
<meta name="citation_reference" content="Lindroth, R. L. &amp; Hwang, S.-Y. Diversity, redundancy, and multiplicity in chemical defense systems of aspen in Phytochemical Diversity and Redundancy in Ecological Interactions 25–56 (Springer, 1996)." />
<meta name="citation_reference" content="Hwang, S.-Y. &amp; Lindroth, R. L. Clonal variation in foliar chemistry of aspen: effects on gypsy moths and forest tent caterpillars. Oecologia 111, 99–108 (1997)." />
<meta name="citation_reference" content="Gould, K. S. Nature’s Swiss army knife: the diverse protective roles of anthocyanins in leaves. BioMed Research International 2004, 314–320 (2004)." />
<meta name="citation_reference" content="Miranda, M. et al. The transcriptional response of hybrid poplar (Populus trichocarpa × P. deltoides) to infection by Melampsora medusae leaf rust involves induction of flavonoid pathway genes leading to the accumulation of proanthocyanidins. Molecular Plant-Microbe Interactions 20, 816–831 (2007)." />
<meta name="citation_reference" content="Chen, W. et al. Expression profile matrix of Arabidopsis transcription factor genes suggests their putative functions in response to environmental stresses. The Plant Cell 14, 559–574 (2002)." />
<meta name="citation_reference" content="Sun, Y. et al. Isolation and promoter analysis of a chalcone synthase gene PtrCHS4 from Populus trichocarpa. Plant Cell Reports 30, 1661 (2011)." />
<meta name="citation_reference" content="Tsai, C.-J., Harding, S. A., Tschaplinski, T. J., Lindroth, R. L. &amp; Yuan, Y. Genome-wide analysis of the structural genes regulating defense phenylpropanoid metabolism in Populus. New Phytologist 172, 47–62 (2006)." />
<meta name="citation_reference" content="Cole, C. T. &amp; Ingvarsson, P. K. Pathway position constrains the evolution of an ecologically important pathway in aspens (Populus tremula L. Molecular Ecology 27, 3317–3330 (2018)." />
<meta name="citation_reference" content="Plett, J. M., Wilkins, O., Campbell, M. M., Ralph, S. G. &amp; Regan, S. Endogenous overexpression of Populus MYB186 increases trichome density, improves insect pest resistance, and impacts plant growth. The Plant Journal 64, 419–432 (2010)." />
<meta name="citation_reference" content="Aitken, S. N. &amp; Whitlock, M. C. Assisted gene flow to facilitate local adaptation to climate change. Annual Review of Ecology, Evolution, and Systematics 44. https://doi.org/10.1146/annurev-ecolsys-110512-135747 (2013)." />
<meta name="citation_reference" content="Urban, M. C. et al. Improving the forecast for biodiversity under climate change. Science 353, aad8466–1–aad8466-9 (2016)." />
<meta name="citation_reference" content="Breed, M. F. et al. The potential of genomics for restoring ecosystems and biodiversity. Nature Reviews Genetics, 1 (2019)." />
<meta name="citation_reference" content="Gillis, M., Omule, A. &amp; Brierley, T. Monitoring Canada’s forests: the national forest inventory. The Forestry Chronicle 81, 214–221 (2005)." />
<meta name="citation_reference" content="Woudenberg, S. W. et al. The Forest Inventory and Analysis Database: Database description and users manual version 4.0 for Phase 2. Gen. Tech. Rep. RMRS-GTR-245. Fort Collins, CO: US Department of Agriculture, Forest Service, Rocky Mountain Research Station. 336 p. 245 (2010)." />
<meta name="citation_reference" content="Aiello-Lammens, M. E., Boria, R. A., Radosavljevic, A., Vilela, B. &amp; Anderson, R. P. spThin: an R package for spatial thinning of species occurrence records for use in ecological niche models. Ecography 38, 541–545 (2015)." />
<meta name="citation_reference" content="Burgman, M. A. &amp; Fox, J. C. Bias in species range estimates from minimum convex polygons: implications for conservation and options for improved planning in Animal Conservation forum 6 (2003), 19–28. https://doi.org/10.1017/S1367943003003044." />
<meta name="citation_reference" content="R Core Team. R: A Language and Environment for Statistical Computing R Foundation for Statistical Computing (Vienna, Austria, 2018). https://www.R-project.org/." />
<meta name="citation_reference" content="Elshire, R. J. et al. A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species. PLoS One 6, e19379 (2011)." />
<meta name="citation_reference" content="Tuskan, G. A. et al. The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray). Science 313, 1596–1604 (2006)." />
<meta name="citation_reference" content="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)." />
<meta name="citation_reference" content="Danecek, P. et al. The variant call format and VCFtools. Bioinformatics 27, 2156–2158 (2011)." />
<meta name="citation_reference" content="ARCC. Teton Computing Environment: Intel x86 64 Cluster, Advanced Research Computing Center, University of Wyoming, Laramie, WY. 2018. https://doi.org/10.15786/M2FY47." />
<meta name="citation_reference" content="Alexander, D. H., Novembre, J. &amp; Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Genome Research 19, 1655–1664 (2009)." />
<meta name="citation_reference" content="Raj, A., Stephens, M. &amp; Pritchard, J. K. fastSTRUCTURE: variational inference of population structure in large SNP data sets. Genetics 197, 573–589 (2014)." />
<meta name="citation_reference" content="Chhatre, V. E. Distruct 2.3: A modified admixture plotting script (2018). http://distruct2.popgen.org." />
<meta name="citation_reference" content="Jombart, T., Devillard, S. &amp; Balloux, F. Discriminant analysis of principal components: a new method for the analysis of genetically structured populations. BMC Genetics 11, 94 (2010)." />
<meta name="citation_reference" content="Lotterhos, K. E. &amp; Whitlock, M. C. Evaluation of demographic history and neutral parameterization on the performance of FST outlier tests. Molecular Ecology 23, 2178–2192 (2014)." />
<meta name="citation_reference" content="Foll, M. &amp; Gaggiotti, O. A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: A Bayesian perspective. Genetics 180, 977–993 (2008)." />
<meta name="citation_reference" content="Storey, J. D., Bass, A. J., Dabney, A. &amp; Robinson, D. qvalue: Q-value estimation for false discovery rate control R package version 2.14.1 (2019). https://github.com/jdstorey/qvalue." />
<meta name="citation_reference" content="Frichot, E., Schoville, S. D., Bouchard, G. &amp; François, O. Testing for associations between loci and environmental gradients using latent factor mixed models. Molecular Biology and Evolution 30, 1687–99 (2013)." />
<meta name="citation_reference" content="Caye, K. &amp; Franois, O. LFMM: Latent Factor Mixed Models R package version 0.0 (2017)." />
<meta name="citation_reference" content="Günther, T. &amp; Coop, G. Robust identification of local adaptation from allele frequencies. Genetics 195, 205–220 (2013)." />
<meta name="citation_reference" content="Szkiba, D., Kapun, M., von Haeseler, A. &amp; Gallach, M. SNP2GO: functional analysis of genome-wide association studies. Genetics 197, 285–289 (2014)." />
<meta name="citation_reference" content="Supek, F., Bošnjak, M., Škunca, N. &amp; Šmuc, T. REVIGO summarizes and visualizes long lists of gene ontology terms. PLoS One 6, e21800 (2011)." />
<meta name="twitter:title" content="Climatic niche predicts the landscape structure of locally adaptive standing genetic variation" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Within a species’ range, intraspecific diversity in the form of adaptive standing genetic variation (SGV) may be non-randomly clustered into different geographic regions, reflecting the combined effects of historical range movements and spatially-varying natural selection. As a consequence of a patchy distribution of adaptive SGV, populations in different parts of the range are likely to vary in their capacity to respond to changing selection pressures, especially long-lived sessile organisms like forest trees. However, the spatial distribution of adaptive SGV across the landscape is rarely considered when predicting species responses to environmental change. Here, we use a landscape genomics approach to estimate the distribution of adaptive SGV along spatial gradients reflecting the expansion history and contemporary climatic niche of balsam poplar, Populus balsamifera (Salicaceae), a widely distributed forest tree with a transcontinental distribution in North America. By scanning the genome for signatures of spatially varying local adaptation, we estimated how adaptive SGV has been shaped by geographic distance from the rear range edge (expansion history) versus proximity to the current center of the climatic niche (environmental selection). We found that adaptive SGV was strongly structured by the current climatic niche, with surprisingly little importance attributable to historical effects such as migration out of southern refugia. As expected, the effect of the climatic niche on SGV was strong for genes whose expression is responsive to abiotic stress (drought), although genes upregulated under biotic (wounding) stress also contained SGV that followed climatic and latitudinal gradients. The latter result could reflect parallel selection pressures, or co-regulation of functional pathways involved in both abiotic and biotic stress responses. Our study in balsam poplar suggests that clustering of locally adaptive SGV within ranges primarily reflects spatial proximity within the contemporary climatic niche – an important consideration for the design of effective strategies for biodiversity conservation and avoidance of maladaptation under climate change." />
<meta name="og-title" property="og:title" content="Climatic niche predicts the landscape structure of locally adaptive standing genetic variation" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/817411v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Within a species’ range, intraspecific diversity in the form of adaptive standing genetic variation (SGV) may be non-randomly clustered into different geographic regions, reflecting the combined effects of historical range movements and spatially-varying natural selection. As a consequence of a patchy distribution of adaptive SGV, populations in different parts of the range are likely to vary in their capacity to respond to changing selection pressures, especially long-lived sessile organisms like forest trees. However, the spatial distribution of adaptive SGV across the landscape is rarely considered when predicting species responses to environmental change. Here, we use a landscape genomics approach to estimate the distribution of adaptive SGV along spatial gradients reflecting the expansion history and contemporary climatic niche of balsam poplar, Populus balsamifera (Salicaceae), a widely distributed forest tree with a transcontinental distribution in North America. By scanning the genome for signatures of spatially varying local adaptation, we estimated how adaptive SGV has been shaped by geographic distance from the rear range edge (expansion history) versus proximity to the current center of the climatic niche (environmental selection). We found that adaptive SGV was strongly structured by the current climatic niche, with surprisingly little importance attributable to historical effects such as migration out of southern refugia. As expected, the effect of the climatic niche on SGV was strong for genes whose expression is responsive to abiotic stress (drought), although genes upregulated under biotic (wounding) stress also contained SGV that followed climatic and latitudinal gradients. The latter result could reflect parallel selection pressures, or co-regulation of functional pathways involved in both abiotic and biotic stress responses. Our study in balsam poplar suggests that clustering of locally adaptive SGV within ranges primarily reflects spatial proximity within the contemporary climatic niche – an important consideration for the design of effective strategies for biodiversity conservation and avoidance of maladaptation under climate change." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-10-24" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/817411v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/817411v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/817411v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/817411v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/964641" />
    <title>Climatic niche predicts the landscape structure of locally adaptive standing genetic variation | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__AQm9HArb2Y_D7UzgraiLIwAB7088fIq2iZD0Uigie_s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"I02YAnNiZHL87yJrEf_0UOd87h4Pel6bv89tN8o6LV0","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;817411v1"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;817411v1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;817411v1"}],"nid":"964641","apath":"\/biorxiv\/early\/2019\/10\/24\/817411.atom","pisa":"biorxiv;817411v1","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/817411v1","title":"Climatic niche predicts the landscape structure of locally adaptive standing genetic variation","identifier":"node\/964641"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222oMet+GbWOSnEZO2A4zTkTDzSSm\\\\\\\/w+h8rXRnQtZqOYtqtl9wf8ZkMj9hDY2pEWDAdeQS7MgnAYxh\\\\\\\/fSPoOGNjQVwa2GqYIPOg3bn3eh\\\\\\\/z77Uwoz91ERWDlO9\\\\\\\/9b2c6r\\\\\\\/qVVYwTBy88HlVTPZpyoZCsQe0EFl7fCkgAnwdHsQpaMIZKd4LAczoI7dlSSigNEzRKA5Ym9QZS2jrIorujKrRisUKCwT1xdkSHx93eyoXDU+1R1juZH0N1S3hnGYHw7gWL0tl6m6si6G8y1wZK0g1OV2irPZrjoXUo24X3WnFkkDzEPn7ODbEl\\\\\\\/9rUXRAhRlpcv4asGNffe5bbg53XygZpz1Aq5BS3wO6mf4Zy6D9gCL\\\\\\\/hu24HYkg70d9G+oB3ll3jbn1t\\\\\\\/SAiFKkY0SppasgHxDJC6iAfaSOdUE7geTCgbC+domP1A5YOEh3bI1UCYlXuylxg9BAfUWFyQpqMIPL1XeULALwsNRgN\\\\\\\/Nzk7sclkf4dW\\\\\\\/6rQ+5V\\\\\\\/5pXMvo1srCqj1Q6jqTVx39yAHK8Dew66mRMmTn0hJMX\\\\\\\/b0IlWvUTvuGIiVHCDrVYNdMxZ8VJUuCsdGoW22kOy1sdG4xsd341c\\\\\\\/AqNkyYZjMPpFYOVU5bJ98oMPg8aTLrAOUKpTRbT6nZWxb6IlXpNa+umTB+ge64OEElYXQok1dSsESZeNVtgBwLf+6NmZ9qTy263VqpCpeCDoeUDzzHCVYLYT8NcoEtTvqVE2B\\\\\\\/Lnl5H74Y\\\\\\\/Y2Cc1B7wrghhPeY1ZiPTbbC4zbtvYPyT7xj7grADt5vtMtmDewc5aWUeS5xwFwyTzM7A0QF8Llz+r12eXLpo6vchpv\\\\\\\/uz+iiF3qMhY7pr\\\\\\\/ahUqRfAkoL3DWp3tQRZoE31NrwyGQQ1TJspBk1DW2QJwJ+atkn+z8gfF5oQwBh3vnmWKnU6DnpSMFbepAYiP1STs4jdOtU\\\\\\\/sTufpa+k5pSx5aZnlzuubO5JPFup+9IWOtyGM2pNtqz5tP2Qd8KoEvvZKNlQuPbP\\\\\\\/uoZiEf1gUTw4wzhc5EjfxZc3s4g+FQU3yxnVYAcEFwIktSpND2ApmSkOiSpux9F4RNbdf1zLntrTSd73FfqVN5Lu0vpyqMGMxRmy103XaVScqQNw0Wgu6qikv7SHgaEwCNcrE4U4XYGXqPMOcOxTpwH7Xi6Z0IniGyp0Q9BaTFUgSl4BVKlnLC0WBvK5YDXFbI1lHk4hiUKABBVkb9mAsEmgu8FZlglCRCeGFZyuJrX8ANu\\\\\\\/X3k+naFSTSqfbELxkp5jcXfNyxUFPHbNjhftCxS8gB3lMZibsWMXqk6KvVCLY7AYIM33v9+Uo=\\\u0022,\\\u0022iv\\\u0022:\\\u00228YO9eSxabXiI8YYU+6eCFw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f31759f50b2f50ce2c74e4987557fd9f3f8af4c7275a347ed9c98e3cf39a8dd9\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/817411v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-964641 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/817411v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-z6jQ93zCjkXLmnBvz3a83MFkj0V8BfP6cR_AIpnJ4uk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="964641" id="node964641" data-pisa="biorxiv;817411v1" data-pisa-master="biorxiv;817411" data-apath="/biorxiv/early/2019/10/24/817411.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Climatic niche predicts the landscape structure of locally adaptive standing genetic variation</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6627-2832" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Vikram E.</span> <span class="nlm-surname">Chhatre</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9234-9300" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Karl C.</span> <span class="nlm-surname">Fetter</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3905-8539" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andrew V.</span> <span class="nlm-surname">Gougherty</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-1911-8407" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Matthew C.</span> <span class="nlm-surname">Fitzpatrick</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-9345-9640" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Raju Y.</span> <span class="nlm-surname">Soolanayakanahally</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ronald S.</span> <span class="nlm-surname">Zalesny</span> <span class="nlm-suffix">Jr.</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-8887-9213" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Stephen R.</span> <span class="nlm-surname">Keller</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/817411 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node964641" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vikram E. Chhatre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Plant Biology, University of Vermont</span>, Burlington, VT 05405, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>Wyoming INBRE Bioinformatics Core, University of Wyoming</span>, Laramie, WY 82071, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vikram%2BE.%2BChhatre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chhatre%20VE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVikram%2BE.%2BChhatre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6627-2832" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vikram E. Chhatre</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Karl C. Fetter </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Plant Biology, University of Vermont</span>, Burlington, VT 05405, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BC.%2BFetter%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fetter%20KC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BC.%2BFetter%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9234-9300" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl C. Fetter</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Andrew V. Gougherty </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>University of Maryland Center for Environmental Science</span>, Frostburg, MD 21532, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BV.%2BGougherty%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gougherty%20AV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BV.%2BGougherty%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3905-8539" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew V. Gougherty</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Matthew C. Fitzpatrick </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>University of Maryland Center for Environmental Science</span>, Frostburg, MD 21532, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BC.%2BFitzpatrick%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fitzpatrick%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BC.%2BFitzpatrick%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1911-8407" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew C. Fitzpatrick</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Raju Y. Soolanayakanahally </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Indian Head Research Farm, Agriculture and Agri-Food Canada</span>, Indian Head, SK S0G 2K0, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raju%2BY.%2BSoolanayakanahally%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Soolanayakanahally%20RY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARaju%2BY.%2BSoolanayakanahally%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9345-9640" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Raju Y. Soolanayakanahally</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ronald S. Zalesny <span class="nlm-suffix">Jr.</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>USDA Forest Service, Northern Research Station, Institute for Applied Ecosystem Studies</span>, Rhinelander, WI 54501, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronald%2BS.%2BZalesny%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EJr.%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zalesny%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonald%2BS.%2BZalesny%2BJr." rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Stephen R. Keller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Plant Biology, University of Vermont</span>, Burlington, VT 05405, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BR.%2BKeller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Keller%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BR.%2BKeller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8887-9213" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stephen R. Keller</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srkeller@uvm.edu" class="" data-icon-position="" data-hide-link-title="0">srkeller@uvm.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/817411v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:964641" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:964641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/817411v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:964641" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:964641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/817411v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:964641" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:964641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/817411v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:964641" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:964641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/817411v1.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:964641" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:964641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/817411v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:964641" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:964641/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-4">Within a species’ range, intraspecific diversity in the form of adaptive standing genetic variation (SGV) may be non-randomly clustered into different geographic regions, reflecting the combined effects of historical range movements and spatially-varying natural selection. As a consequence of a patchy distribution of adaptive SGV, populations in different parts of the range are likely to vary in their capacity to respond to changing selection pressures, especially long-lived sessile organisms like forest trees. However, the spatial distribution of adaptive SGV across the landscape is rarely considered when predicting species responses to environmental change. Here, we use a landscape genomics approach to estimate the distribution of adaptive SGV along spatial gradients reflecting the expansion history and contemporary climatic niche of balsam poplar, <em>Populus balsamifera</em> (Salicaceae), a widely distributed forest tree with a transcontinental distribution in North America. By scanning the genome for signatures of spatially varying local adaptation, we estimated how adaptive SGV has been shaped by geographic distance from the rear range edge (expansion history) versus proximity to the current center of the climatic niche (environmental selection). We found that adaptive SGV was strongly structured by the current climatic niche, with surprisingly little importance attributable to historical effects such as migration out of southern refugia. As expected, the effect of the climatic niche on SGV was strong for genes whose expression is responsive to abiotic stress (drought), although genes upregulated under biotic (wounding) stress also contained SGV that followed climatic and latitudinal gradients. The latter result could reflect parallel selection pressures, or co-regulation of functional pathways involved in both abiotic and biotic stress responses. Our study in balsam poplar suggests that clustering of locally adaptive SGV within ranges primarily reflects spatial proximity within the contemporary climatic niche – an important consideration for the design of effective strategies for biodiversity conservation and avoidance of maladaptation under climate change.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-7">Adaptive standing genetic variation (SGV) is of fundamental importance to the response of populations to environmental change. R.A. Fisher<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a></sup> first identified that the response to selection is proportional to the genetic variance in fitness, but also assumed that most populations exist at or near their current fitness optima such that selection primarily acts on new mutations as opposed to SGV<sup><a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>. However, several processes are known to maintain SGV over broader temporal and spatial scales. For example, fluctuating or balancing selection may favor the long-term persistence of multiple alleles within populations, and this has been particularly well documented for genes associated with mating type<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> or immunity<sup><a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>–<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. Similarly, spatial population structure and local selection can maintain adaptive SGV at range-wide scales, for example due to polygenic selection shifting adaptive allele frequencies along environmental gradients<sup><a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup> or when hard or soft selective sweeps act locally within different populations<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup>.</p><p id="p-8">For many species, the current spatial distribution of adaptive SGV will in part reflect historical events such as range contractions and expansions in response to past glaciation, as well as effects of environmental selection during or since these events<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>–<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup>. Hampe &amp; Petit<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup> argued that populations distributed along the rear edge of a species’ range may be closely descended from refugial populations, and hence are predicted to be older and harbor higher levels of SGV compared to younger populations in areas of recent expansion<sup><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Reduced SGV with distance from the rear range edge is also a predicted consequence of serial founder effects during range expansion<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>, which can lower adaptive potential with distance from rear edge source populations<sup><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a></sup>. Moreover, rear edge populations at lower latitudes may be subject to increased levels of abiotic stress, hybridization with congeners, or experience higher disease or pathogen incidence compared to populations in the core of the species’ range<sup><a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>–<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a></sup>, suggesting that the strength or nature of selection may vary with proximity to the rear range edge<sup><a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>,<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a></sup>. However, the effect of range limits on adaptive genetic variation remains poorly understood<sup><a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup>, and only a handful of experimental studies have quantified how SGV under selection varies along a continuum from the core to the edge of a species’ range (e.g.<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>–<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>).</p><p id="p-9">While proximity to the rear range edge may predict if SGV for ancestral polymorphism has been reduced by colonization bottlenecks, it may not adequately capture how SGV has been shaped by spatially-varying selection after expansion<sup><a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. In this context, populations inhabiting ecologically marginal environments are of particular interest<sup><a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>. Marginal environments may impose intense selection pressures on populations to locally adapt, but such adaptation may be constrained by demographic effects of genetic drift<sup><a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>,<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup> or the immigration of maladapted alleles from more abundant populations elsewhere in the range<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. On the other hand, gene flow of beneficial alleles may facilitate adaptation to marginal environments<sup><a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>, and could contribute substantially to the global pool of adaptive SGV present across a species’ range. Identifying local populations containing unique adaptive variation relative to the rest of the range has gained increased importance in the context of contemporary global change<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a></sup>, as efforts are needed to identify and conserve adaptive germplasm <em>in situ</em> for crop breeding<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>, or to forecast the adaptive response of natural populations to climate change<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>.</p><p id="p-10">North American tree species provide excellent study systems for addressing the effects of range context on locally adaptive SGV, as their expansion history into formerly glaciated regions is well documented in the fossil pollen record, and is known to have tracked the changing climate during the Holocene<sup><a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a></sup>. Due to their long individual lifespans, frequently spanning hundreds of years, trees occur in populations characterized by overlapping generations in which SGV for fitness traits can be maintained by demographic or selective processes over eons<sup><a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>. Furthermore, many tree species show strong clines across their ranges for functional traits and adaptive allele frequencies that occur across complex spatial gradients of expansion history and climatic environment<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>,<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>–<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>. This makes forest trees important systems in which to understand the spatial landscape of adaptive variation<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>, and the roles of history and selection in shaping where SGV is concentrated within the range<sup><a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a></sup>.</p><p id="p-11">In this study, we report a landscape genomics analysis investigating the importance of range context for the distribution of locally adaptive genomic variation in the widely distributed boreal tree, <em>Populus balsamifera</em> L. (Salicaceae). The occurrence of <em>P. balsamifera</em> is spread across 30 degrees of latitude and over 100 degrees of longitude<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>, and as such populations span a massive range in climate and growing season length<sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup>. Previous genetic studies place the likely refugial location of <em>P. balsamifera</em> during the last glacial maximum (LGM) in the central Rocky Mountains<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>,<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a></sup>, where <em>P. balsamifera</em> currently exists in relatively isolated populations along its southern range edge<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>,<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">48</a></sup>. The current center of abundance of the species lies to the north of these rear edge populations across central Canada, where population genetic and species distribution models suggest the presence of a large effective population size (<em>N</em><sub><em>e</em></sub>) with high historic levels of gene flow to the northern and eastern peripheries of the range<sup><a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">49</a></sup>. <em>Populus balsamifera</em> is also known to exhibit strong clinal variation for phenology and ecophysiological traits<sup><a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>–<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup> and for nucleotide diversity and divergence in flowering time genes, suggesting pervasive local adaptation to climate and photoperiod across its range<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>,<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup>.</p><p id="p-12">Here, we use a combination of approaches to scan the genome for signals of local adaptation, conditioned on the spatial pattern of neutral population structure, and show that adaptive SGV varies strongly with proximity to the species’ core climatic niche, with weaker effects of distance from the rear range edge. The predominant influence of the contemporary climatic niche shaping adaptive SGV is largely robust across genes involved in defense vs. drought stress, suggesting similar processes have shaped variation in genomic pathways involved in response to biotic and abiotic stress. Our landscape genomics approach gives spatially-explicit insights into where and why SGV is clustered within a species’ range, and serves as a guide for developing strategies aimed at identifying and conserving areas of unique genetic diversity and mitigating loss of adaptation under environmental change.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-13">Our GBS libraries for 508 individuals generated 3.9 billion single-end reads passing filter, of which 1.35 billion passed QC in the GBS pipeline as “good, barcoded reads” <em>sensu</em> Glaubitz et al.<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>, consisting of a perfect match to one of the barcodes and no ambiguous base calls. Reads passing QC clustered into 3,755,656 sequence tags that mapped uniquely to 220.9 Mb of the <em>P. trichocarpa</em> reference genome v3.0. Variant calling produced 1,107,538 sites which, after filtering, resulted in 167,324 biallelic SNPs for inference of population structure and selection, with a median (across individuals) per-site depth of 21.8X.</p><div id="sec-3" class="subsection"><h3>Population Structure</h3><p id="p-14">Inference of population structure via discriminant analysis of principal components (DAPC) revealed support for 3 to 4 genetically distinct clusters, with mean BIC over the 1000 iterations minimized at <em>K</em>=3 (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig. S1A, B</a>). Clustering assignments at <em>K</em>=3 separated individuals into a northwestern cluster in Alberta and British Columbia, a central cluster throughout the range core from Saskatchewan to Ontario including the Great Lakes region, and an Atlantic Canada cluster along the eastern coast (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1C</a> inset). In about 10% of the iterations, DAPC favored a model of <em>K</em>=4 that showed a slightly higher BIC averaged across iterations compared to <em>K</em>=3. At <em>K</em>=4, the dominant solution split the range core into eastern and western clusters that formed distinct groups but were only weakly differentiated (<em>F</em><sub>ST</sub> = 0.008) (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1C</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DAPC model selection output from 1000 iterations of find.clusters from K=1 to K=10 on the range-wide (N=437) data set (A). The best value of K is defined as the cluster with the lowest BIC. K=3 most frequently had the lowest BIC score (B). For the K=3 clustering results, three unique solutions were recovered (C). The BIC distribution of result A, the most common K=3 result (D). For the K=4 clustering results, more unique solutions were recovered (E). The BIC distribution of result A, the most common result at K=4 (F)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;DAPC model selection output from 1000 iterations of &lt;kbd&gt;find.clusters&lt;/kbd&gt; from &lt;em&gt;K&lt;/em&gt;=1 to &lt;em&gt;K&lt;/em&gt;=10 on the range-wide (N=437) data set (A). The best value of &lt;em&gt;K&lt;/em&gt; is defined as the cluster with the lowest BIC. &lt;em&gt;K&lt;/em&gt;=3 most frequently had the lowest BIC score (B). For the &lt;em&gt;K&lt;/em&gt;=3 clustering results, three unique solutions were recovered (C). The BIC distribution of result A, the most common &lt;em&gt;K&lt;/em&gt;=3 result (D). For the &lt;em&gt;K&lt;/em&gt;=4 clustering results, more unique solutions were recovered (E). The BIC distribution of result A, the most common result at &lt;em&gt;K&lt;/em&gt;=4 (F).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F9.medium.gif" width="405" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F9.medium.gif" width="405" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure S1</span> <p id="p-64" class="first-child">DAPC model selection output from 1000 iterations of <kbd>find.clusters</kbd> from <em>K</em>=1 to <em>K</em>=10 on the range-wide (N=437) data set (A). The best value of <em>K</em> is defined as the cluster with the lowest BIC. <em>K</em>=3 most frequently had the lowest BIC score (B). For the <em>K</em>=3 clustering results, three unique solutions were recovered (C). The BIC distribution of result A, the most common <em>K</em>=3 result (D). For the <em>K</em>=4 clustering results, more unique solutions were recovered (E). The BIC distribution of result A, the most common result at <em>K</em>=4 (F).</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sample collection map of the 437 individuals analyzed in selection scans. The red line represents the estimated location of the southern range edge based on occurrence records (A). The range boundary of P. balsamifera is denoted by the black line and is adapted from47. 19 BIOCLIM variables and latitude were included in a PCA and PC1 and PC2 were used in gene-environment analyses (B). Discriminant analysis of principal components (DAPC) at K =4 separates Northwestern Rocky Mountains from populations east of Saskatchewan along DAPC axis 1. DAPC axis 2 separates an Atlantic Canada clusters and two range core clusters. Shapes in the sample map, PCA plot correspond to the DAPC (C). ADMIXTURE clustering of all 508 individuals sequenced in the study at K =7, the level of K with the lowest cross validation error. Colors represent the following clusters: blue &amp; goldenrod: range core, lightgreen: Rocky Mountain region, red: Atlantic Canada, yellow: New Glasgow, NS, Canada with introgression from P. deltoides, pink: admixture from Northwestern cluster reported by Keller et al49. Individuals with less than 90% P. balsamifera ancestry were removed prior to performing selection scans." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sample collection map of the 437 individuals analyzed in selection scans. The red line represents the estimated location of the southern range edge based on occurrence records (A). The range boundary of &lt;em&gt;P. balsamifera&lt;/em&gt; is denoted by the black line and is adapted from&lt;sup&gt;47&lt;/sup&gt;. 19 BIOCLIM variables and latitude were included in a PCA and PC1 and PC2 were used in gene-environment analyses (B). Discriminant analysis of principal components (DAPC) at &lt;em&gt;K&lt;/em&gt; =4 separates Northwestern Rocky Mountains from populations east of Saskatchewan along DAPC axis 1. DAPC axis 2 separates an Atlantic Canada clusters and two range core clusters. Shapes in the sample map, PCA plot correspond to the DAPC (C). ADMIXTURE clustering of all 508 individuals sequenced in the study at &lt;em&gt;K&lt;/em&gt; =7, the level of K with the lowest cross validation error. Colors represent the following clusters: blue &amp; goldenrod: range core, lightgreen: Rocky Mountain region, red: Atlantic Canada, yellow: New Glasgow, NS, Canada with introgression from &lt;em&gt;P. deltoides&lt;/em&gt;, pink: admixture from Northwestern cluster reported by Keller &lt;em&gt;et al&lt;/em&gt;&lt;sup&gt;49&lt;/sup&gt;. Individuals with less than 90% &lt;em&gt;P. balsamifera&lt;/em&gt; ancestry were removed prior to performing selection scans.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F1.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F1.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Sample collection map of the 437 individuals analyzed in selection scans. The red line represents the estimated location of the southern range edge based on occurrence records (A). The range boundary of <em>P. balsamifera</em> is denoted by the black line and is adapted from<sup><a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a></sup>. 19 BIOCLIM variables and latitude were included in a PCA and PC1 and PC2 were used in gene-environment analyses (B). Discriminant analysis of principal components (DAPC) at <em>K</em> =4 separates Northwestern Rocky Mountains from populations east of Saskatchewan along DAPC axis 1. DAPC axis 2 separates an Atlantic Canada clusters and two range core clusters. Shapes in the sample map, PCA plot correspond to the DAPC (C). ADMIXTURE clustering of all 508 individuals sequenced in the study at <em>K</em> =7, the level of K with the lowest cross validation error. Colors represent the following clusters: blue &amp; goldenrod: range core, lightgreen: Rocky Mountain region, red: Atlantic Canada, yellow: New Glasgow, NS, Canada with introgression from <em>P. deltoides</em>, pink: admixture from Northwestern cluster reported by Keller <em>et al</em><sup><a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">49</a></sup>. Individuals with less than 90% <em>P. balsamifera</em> ancestry were removed prior to performing selection scans.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">The genetic clusters from DAPC aligned along major gradients of climate and growing season length, as captured by the first two principal components of the climate PCA (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>). Large values of PC1 represented dry environments with high temperature and precipitation seasonality, and were primarily occupied by the western core and northwestern Rockies groups, while small values of PC1 (wet, low seasonality) were occupied by individuals with ancestry in the Atlantic Canada cluster. PC2 represented a north-south gradient of growing season length, with large values representing high latitude, cool sites occupied by the northwestern Rockies and a subset of the Atlantic Canada clusters, while small values of PC2 (low latitude, warm sites) were dominated by members of the eastern core cluster (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1B</a>). Ancestry clusters were also non-randomly distributed with respect to distance from the rear range edge and proximity to the species-wide climatic niche centroid (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1A</a>). Notably, the Atlantic Canada cluster was farthest from the climatic centroid while spanning a range of distances from the rear edge. In contrast, the western and eastern core clusters were close to the climatic centroid, with many eastern core populations also proximate to the rear range edge (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1A</a>).</p><p id="p-17">ADMIXTURE analysis largely corroborated the population structure found by DAPC, but also revealed extensive mixed ancestry among <em>P. balsamifera</em> clusters, as well as introgression with other <em>Populus</em> species (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1D</a>). At lower levels of <em>K</em> (2–3), introgression was especially prevalent with <em>P. trichocarpa</em> along the rear range edge and in the northwestern Rockies (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Fig. S2</a>; see also Chhatre <em>et al</em><sup><a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a></sup>). At <em>K</em>=4, a separate source of introgressed ancestry was apparent in the Atlantic Canada population of NEG, representing hybridization with <em>P. deltoides</em><sup><a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">20</a></sup>. Scattered introgression was also apparent at a low frequency throughout other portions of the range. At higher levels of <em>K</em> (5–6), ancestry within <em>P. balsamifera</em> separated into an Atlantic Canada cluster and a subdivided central range core split into eastern and western clusters, as observed in DAPC (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1D</a>). Individual ancestry assignments to the eastern and western range core revealed a longitudinal cline, indicating extensive shared ancestral polymorphism and/or gene flow between these two weakly differentiated groups. At <em>K</em>=7, the best model supported by ADMIXTURE cross-validation, an additional distinct ancestry cluster was evident in the far northwestern Rockies (WLK and FNO; <a id="xref-fig-1-9" class="xref-fig" href="#F1">Fig. 1C</a>).</p><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Admixture analysis of 508 individuals collected from across the range of P. balsamifera from K=2 to K=6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Admixture analysis of 508 individuals collected from across the range of &lt;em&gt;P. balsamifera&lt;/em&gt; from &lt;em&gt;K&lt;/em&gt;=2 to &lt;em&gt;K&lt;/em&gt;=6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F10.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F10.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2</span> <p id="p-65" class="first-child">Admixture analysis of 508 individuals collected from across the range of <em>P. balsamifera</em> from <em>K</em>=2 to <em>K</em>=6.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Genomic Signals of Local Adaptation</h3><p id="p-18">After eliminating individuals with less than 90% <em>P. balsamifera</em> ancestry (N=26 individuals; <a id="xref-fig-1-10" class="xref-fig" href="#F1">Fig. 1D</a>), we retained 437 individuals and 129,251 SNPs to test for genomic signatures of local adaptation, with 67% of SNPs within 5 kb of an annotated gene model. Genome scans identified 397 outlier SNPs in tests for elevated population differentiation (Bayescan and <em>X</em><sup><em>T</em></sup> <em>X</em>) that exceeded the empirical Type I error rate based on the distribution of intergenic SNPs (<em>α</em>=0.0006) – an enrichment of more than 2.5X over neutral expectations. Most differentiation outliers were significant only in Bayescan (321/397 loci, 81%) or to a lesser extent, in <em>X</em><sup><em>T</em></sup> <em>X</em> (5 loci, 1%), while 71 outliers (18%) overlapped between the two differentiation tests (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>). Gene-environment association (GEA) tests (LFMM and Bayenv2) returned 535 outliers associated with either climate PC1 or PC2, more than 1.7X greater than neutral expectations. Similar to differentiation tests, most outliers in GEA tests were sensitive to the method used to detect them, with 16 of 535 loci (3%) in the range-wide set shared between Bayenv2 and LFMM (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2a</a>). Combining across all four genome scan approaches, we identified 814 loci within 5kb of 724 genes that showed evidence of either elevated differentiation or gene-environment association relative to the empirical neutral distribution, with most loci unique to a single analysis (85%) and only 6 loci (0.7%) shared across all four methods.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn diagrams showing overlap of local adaptation outliers among four selection scans using the Range-Wide and Range-Core samples. Bayenv2 and LFMM test for gene-environment associations (GEA), whereas Bayescan and XT X test for elevated population differentiation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Venn diagrams showing overlap of local adaptation outliers among four selection scans using the Range-Wide and Range-Core samples. Bayenv2 and LFMM test for gene-environment associations (GEA), whereas Bayescan and &lt;em&gt;X&lt;/em&gt;&lt;sup&gt;&lt;em&gt;T&lt;/em&gt;&lt;/sup&gt; &lt;em&gt;X&lt;/em&gt; test for elevated population differentiation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F2.medium.gif" width="440" height="199"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F2.medium.gif" width="440" height="199"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Venn diagrams showing overlap of local adaptation outliers among four selection scans using the Range-Wide and Range-Core samples. Bayenv2 and LFMM test for gene-environment associations (GEA), whereas Bayescan and <em>X</em><sup><em>T</em></sup> <em>X</em> test for elevated population differentiation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Outliers unique to a single method may be false positives caused by inadequate control of background structure due to residual relatedness or departure from the assumed demographic history. Alternatively, these loci may be true positives that are not detected by the complimentary GEA or differentiation method, for example if the statistical modeling of background genetic structure over-corrects the allele frequencies and dilutes the selection signal. To explore this possibility further, we performed each selection scan again on just those populations belonging to the eastern and western range core clusters, where background population structure was minimal (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Fig. 1</a>; N=336 individuals; <em>F</em><sub>ST</sub> <em>&lt;</em> 0.01). This greatly increased the frequency of shared outliers among differentiation methods to 44% (106/242 loci; <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2b</a>), and confirmed several prominent genomic regions of high differentiation in both the range-wide and core sets (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). These genomic regions included a large peak encompassing ca. 168 kb on the distal arm of chromosome 3, a second prominent peak spanning ca. 369 kb on chromosome 4, and an outlier marked by a single SNP on chromosome 8 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots of XT X outlier scan for the set of range-wide individuals (A) and the range core individuals (B). Highlighted in each panel are outlier loci that were also identified in Bayescan; 106 loci in panel A and 71 in panel B. The horizontal black line indicates the -log10P-value significance threshold based on the empirical neutral distribution" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots of &lt;em&gt;X&lt;/em&gt;&lt;sup&gt;&lt;em&gt;T&lt;/em&gt;&lt;/sup&gt; &lt;em&gt;X&lt;/em&gt; outlier scan for the set of range-wide individuals (A) and the range core individuals (B). Highlighted in each panel are outlier loci that were also identified in Bayescan; 106 loci in panel A and 71 in panel B. The horizontal black line indicates the -log&lt;sub&gt;10&lt;/sub&gt;&lt;em&gt;P&lt;/em&gt;-value significance threshold based on the empirical neutral distribution&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F3.medium.gif" width="440" height="255"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F3.medium.gif" width="440" height="255"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Manhattan plots of <em>X</em><sup><em>T</em></sup> <em>X</em> outlier scan for the set of range-wide individuals (A) and the range core individuals (B). Highlighted in each panel are outlier loci that were also identified in Bayescan; 106 loci in panel A and 71 in panel B. The horizontal black line indicates the -log<sub>10</sub><em>P</em>-value significance threshold based on the empirical neutral distribution</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">A more modest increase in the frequency of shared outliers was evident between GEA methods performed on the range core set (13%; 71/559 loci); however, the number of outliers identified by Bayenv2 increased dramatically, suggesting the selection signal in the range-wide set likely contained false negatives due to over-correction of population structure (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2b</a>). Genomic regions of shared outliers in the range-wide set were limited to a few scattered chromosomal regions, whereas in the range core, shared outliers were clustered into prominent peaks on chromosomes 3, 4, 8, 12, 15, and 17 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LFMM results for gene-environment association tests for the range-wide (A,B) and range core (C,D) sample sets with climate PC1 (A,C) and PC2 (B, D). Loci in orange were also significant in the Bayenv2 gene-environment test. The horizontal black line indicates the -log10P-value significance threshold based on the empirical neutral distribution" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;LFMM results for gene-environment association tests for the range-wide (A,B) and range core (C,D) sample sets with climate PC1 (A,C) and PC2 (B, D). Loci in orange were also significant in the Bayenv2 gene-environment test. The horizontal black line indicates the -log&lt;sub&gt;10&lt;/sub&gt;&lt;em&gt;P&lt;/em&gt;-value significance threshold based on the empirical neutral distribution&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F4.medium.gif" width="337" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F4.medium.gif" width="337" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-23" class="first-child">LFMM results for gene-environment association tests for the range-wide (A,B) and range core (C,D) sample sets with climate PC1 (A,C) and PC2 (B, D). Loci in orange were also significant in the Bayenv2 gene-environment test. The horizontal black line indicates the -log<sub>10</sub><em>P</em>-value significance threshold based on the empirical neutral distribution</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Across all four methods, inference of loci under local adaptation appeared to be sensitive to the genome scan approach (differentiation vs. GEA) and the specific method used. Despite multiple controls in place for background genetic structure, false positives are inevitably part of these differences. However, the genetic architecture of local adaptation is likely to be dominated by many loci of small effect that exhibit weak allele frequency (co)variance among populations, making it difficult to distinguish these loci from the genomic background.</p><p id="p-25">Additionally, some loci likely experienced selection along portions of the climatic gradients that run parallel with the neutral genetic structure (e.g, climate PC2 and the Atlantic Canada cluster; <a id="xref-fig-1-12" class="xref-fig" href="#F1">Fig. 1</a>). Accordingly, we retained two sets of outliers for further investigation: (1) an inclusive set of all 814 outlier loci identified by one or more selection scan methods in the range-wide analyses, and (2) only those outliers corroborated by both methods within a given genome scan approach (differentiation or GEA), resulting in a reduced set of 203 loci. The former is more likely to retain true positive cases of weakly selected loci or those spatially confounded with neutral structure at the expense of including more false positives, while the latter is less likely to contain fewer false positives but will be biased towards loci of large effect (e.g., large allele frequency differences) that are not closely aligned with neutral population structure.</p></div><div id="sec-5" class="subsection"><h3>Distribution of adaptive SGV along geographic and climatic range edges</h3><p id="p-26">We investigated the spatial turnover in SGV by determining the relative contribution of individual populations to the adaptive variation we observed across the entire set of populations. For each population, we estimated the population adaptive index (PAI) of Bonin <em>et al.</em><sup><a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a></sup>:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/24/817411/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/24/817411/embed/graphic-5.gif"/></noscript></span>

</span>
where <em>p</em><sub><em>ij</em></sub> is the minor allele frequency of the <em>i</em>th outlier locus in the <em>j</em> th population, and <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/24/817411/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/24/817411/embed/inline-graphic-1.gif"/></noscript></span></span> is the mean frequency across all populations. Populations with high values of PAI contribute disproportionately to the pool of adaptive allele frequencies; while populations with low PAI have allele frequencies closer to the range-wide average. To assess how this among-population component of SGV varied across the landscape, we modelled PAI as a function of distance from the rear edge and the climatic niche centroid. When considering the inclusive set of 814 selection outliers as an index of genomic beta diversity, over 75% of the variability in PAI was explained by jointly accounting for geographic distance from the rear edge and the euclidean distance from the climatic niche centroid (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Contrary to predictions based on expansion history<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">14</a></sup>, PAI showed little effect of distance from the rear range edge except through an interaction with climate distance, which revealed a dramatic increase in PAI with combined distance away from both the rear edge and climatic niche centroid, consistent with divergent local selection in climatically marginal environments that are geographically located far from the current rear range edge (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5a</a>). Driving this trend were large values of PAI for populations in Atlantic Canada that inhabited the easternmost portion of the species’ range – a region climatically distinct from the majority of the species’ niche (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Fig. 1A, B</a>; <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5a</a>). The overall trend in PAI was similar for the large effect outliers (N=203 loci), although this set of loci showed some rear edge populations exhibiting slightly elevated levels of PAI but still far below levels seen in marginal climates (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>; <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5b</a>).</p><p id="p-27">The historic and selective processes shaping the geographic distribution of SGV may differ for adaptive variation underlying responses to abiotic vs. biotic stress<sup><a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a></sup>. To test this prediction, we calculated PAI separately for outlier SNPs that tagged genes showing differential expression under controlled drought (abiotic) or beetle/mechanical wounding (biotic) based on the gene expression atlas on the Populus Genome Integrative Explorer (PopGenIE)<sup><a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup>. Among the 724 genes associated with the set of inclusive outliers, 105 genes were differentially expressed under drought (N=174 outlier SNPs), and 30 were differentially expressed under beetle or mechanical wounding (48 outlier SNPs). Both types of stressor showed a strong increase in PAI towards climatically marginal range edges, although the effects were more pronounced for loci associated with abiotic stress response (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5c,d</a>). Interestingly, PAI for biotic stress outliers increased consistently with distance from the rear edge in a trend that was not evident among drought-associated loci (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5c</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/966781/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/966781/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/966781" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <p id="p-28" class="first-child">Response of population adaptive index (PAI) to distance from the rear range edge and from the climatic niche centroid. (a) The inclusive set of significant outliers identified across all four selection scan methods; (b) large effect outliers corroborated by both GEA-based tests and/or both <em>F</em><sub>ST</sub>-based tests; (c,d) outliers proximate to (¡5kb) genes differentially expressed in response to (c) biotic (beetle or mechanical) or (d) abiotic (drought) stress. Values are coefficients (SE) from multiple regression.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted model fit surface of the relationship between population adaptive index (PAI) and two predictor distances (distance from the rear range edge and distance from the climate centroid). PAI estimates are summed over all outliers and normalized for comparison between data sets (see definitions in Table 1). Surface colors show PAI magnitude from low (blue) to high (red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predicted model fit surface of the relationship between population adaptive index (PAI) and two predictor distances (distance from the rear range edge and distance from the climate centroid). PAI estimates are summed over all outliers and normalized for comparison between data sets (see definitions in Table 1). Surface colors show PAI magnitude from low (blue) to high (red).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F5.medium.gif" width="440" height="403"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F5.medium.gif" width="440" height="403"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">Predicted model fit surface of the relationship between population adaptive index (PAI) and two predictor distances (distance from the rear range edge and distance from the climate centroid). PAI estimates are summed over all outliers and normalized for comparison between data sets (see definitions in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Surface colors show PAI magnitude from low (blue) to high (red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">As an index of genetic beta diversity, PAI identifies populations whose allele frequencies deviate most from the sample-wide mean, and thus this estimator is inherently on the <em>among</em>-population component of SGV. To investigate the effects of range position on <em>within</em>-population SGV (i.e., alpha diversity), we estimated the variance in allele frequencies at biallelic sites (<em>pq</em> =<em>p</em>*(1-<em>p</em>)) for each population averaged across outlier loci. The inclusive set of 814 outliers showed a highly significant interaction between rear edge and climatic distances (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), with <em>pq</em> for locally-adaptive alleles highest with increasing distance from the rear edge but close to the center of the climatic niche (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6a</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/966783/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/966783/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/966783" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <p id="p-31" class="first-child">Response of allele frequency variance (<em>pq</em>) averaged over outlier loci to distance from the rear range edge and from the climatic niche centroid. Variable definitions and outlier groups are as described in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a></p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted model fit surface of the relationship between allele frequency variance (pq) and two predictor distances (distance from the rear (southern) range edge and distance from the climate centroid). Panels (a) through (d) correspond to categories of outliers decribed in Figure 5. The z-axis is scaled to span the range of allele frequency variance observed across all four comparisons (min=0.04; max=0.17). Surface colors indicate strength of allele frequency variance from low (blue) to high (red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Predicted model fit surface of the relationship between allele frequency variance (&lt;em&gt;pq&lt;/em&gt;) and two predictor distances (distance from the rear (southern) range edge and distance from the climate centroid). Panels (a) through (d) correspond to categories of outliers decribed in Figure 5. The z-axis is scaled to span the range of allele frequency variance observed across all four comparisons (min=0.04; max=0.17). Surface colors indicate strength of allele frequency variance from low (blue) to high (red).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F6.medium.gif" width="440" height="403"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F6.medium.gif" width="440" height="403"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-32" class="first-child">Predicted model fit surface of the relationship between allele frequency variance (<em>pq</em>) and two predictor distances (distance from the rear (southern) range edge and distance from the climate centroid). Panels (a) through (d) correspond to categories of outliers decribed in <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>. The z-axis is scaled to span the range of allele frequency variance observed across all four comparisons (min=0.04; max=0.17). Surface colors indicate strength of allele frequency variance from low (blue) to high (red).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">For large effect outliers and those associated with drought stress, allele frequency variance was lowest on average for climatically distant populations where PAI was high, suggesting selection has depleted SGV locally (<em>pq</em>) while increasing the genetic distinctiveness (PAI) of climatically marginal populations (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6</a>). In contrast, within-population SGV for outliers associated with biotic stress increased significantly with distance from the rear edge (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), but showed no consistent effect of position along the climate gradient (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6</a>).</p><p id="p-34">The effects of distance from the rear range edge and climatic niche on components of adaptive SGV are evident when population values of <em>pq</em> and PAI are mapped onto the landscape (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a>). With the exception of the biotic outliers, the overall trend showed higher within-population SGV and lower among-population SGV nearer to the geographic and climatic center of the range, per theoretical expectation of the ecological and evolutionary processes structuring range limits<sup><a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. Two exceptions to this general trend were evident. First was the clearly elevated among-population SGV seen in the Atlantic Canada populations, and that featured prominently in all outlier sets. Second was the unusual combination of SGV associated with biotic stress response genes, in which Atlantic Canada populations harbored high levels of SGV both within- and among-population (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Landscape level comparison of the relationship between allele frequency variance (color: blue to red) and population adaptive index (size: small to large) among four groups of selection scan outliers. PAI point sizes are not comparable between individual subplots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Landscape level comparison of the relationship between allele frequency variance (color: blue to red) and population adaptive index (size: small to large) among four groups of selection scan outliers. PAI point sizes are not comparable between individual subplots.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F7.medium.gif" width="440" height="303"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F7.medium.gif" width="440" height="303"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-35" class="first-child">Landscape level comparison of the relationship between allele frequency variance (color: blue to red) and population adaptive index (size: small to large) among four groups of selection scan outliers. PAI point sizes are not comparable between individual subplots.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Functional enrichment of adaptation outliers</h3><p id="p-36">We found 52 gene ontology (GO) terms significantly enriched among genes associated with the 814 outlier set (Supplementary Information). These included clusters of terms relating to enzyme transferase activity, biosynthesis of cell wall components such as cellulose and lipids, signal transduction and ROS activity, response to stress and/or DNA damage repair, and transport across cell membranes (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. 8</a>). Genes associated with the 203 outlier set showed significant enrichment for 28 GO terms including many similar functions as the 814 outliers set, but with the addition of several terms associated with vessicle transport within and across cell membranes, and response to foreign bodies (Supplementary File S1).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Redundancy clustering of functionally enriched GO terms associated with the 814 loci significant in the range-wide local adaptation scans. Color indicates significance of the enrichment for each GO term, and symbol size is proportional to the frequency of each GO term in the P. trichocarpa v3.0 genome annotation" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-39797360" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Redundancy clustering of functionally enriched GO terms associated with the 814 loci significant in the range-wide local adaptation scans. Color indicates significance of the enrichment for each GO term, and symbol size is proportional to the frequency of each GO term in the &lt;em&gt;P. trichocarpa&lt;/em&gt; v3.0 genome annotation&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F8.medium.gif" width="292" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F8.medium.gif" width="292" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/24/817411/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-37" class="first-child">Redundancy clustering of functionally enriched GO terms associated with the 814 loci significant in the range-wide local adaptation scans. Color indicates significance of the enrichment for each GO term, and symbol size is proportional to the frequency of each GO term in the <em>P. trichocarpa</em> v3.0 genome annotation</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Several of the enriched GO terms were associated with candidate genes tagged by SNPs that were selection outliers in all four genome scan methods in the range-wide and core sets (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) These included candidate genes within the selected region on chromsome 3: the flavonoid biosynthesis gene CHALCONE SYNTHASE, CELLULOSE SYNTHASE-LIKE D1.1, and two FERRODOXIN genes. Chromosome 12 also contained candidate genes with enriched GO terms associated with DNA repair: an ATPase-like family protein (Potri.012G096300), RESPIRATORY BURST OXIDASE HOMOLOG C, a Tyrosyl-DNA phosphodiesterase, and a UvrD DNA helicase. Chromosomes 15 and 17 each contained a single gene with enriched GO terms: the MADS-box transcription factor TRANSPARENT TESTA 16, and the stress-responsive molecular chaperone HEAT SHOCK PROTEIN 90-1 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/966774/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/966774/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/966774" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <p id="p-39" class="first-child">Candidate genes for local adaptation in range-wide and range core datasets. Annotated genes are located within 5kb of outlier SNPs identifed by all four selection scan methods.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-7"><h2 class="">Discussion</h2><p id="p-40">The current period of rapid global climate change has created an immediate need to understand the genetic basis of climate adaptation in natural populations. For long-lived sessile organisms such as forest trees, adaptation to ongoing climate change will depend in large part on the availability of adaptive standing genetic variation (SGV) within the species, as well as the distribution of SGV across a potentially expansive landscape. Numerous studies have shown the post-glacial biogeography of forest trees shaped SGV over millennia, involving complex cycles of range contraction and expansion, and also local adaptation along climatic gradients<sup><a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>,<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">14</a>,<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>,<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a></sup>. But while forest trees have become important models for understanding the genetic basis of climate adaptation at landscape scales<sup><a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a>,<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>,<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup>, few studies have explicitly addressed how adaptive SGV is partitioned along spatial gradients reflecting expansion history and current climatic niche<sup><a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">13</a></sup>.</p><p id="p-41">Our approach fills an important gap in identifying geographic regions where adaptive genomic diversity is concentrated within the range, while also revealing the relative importance of historic and selective processes shaping the current distribution of SGV<sup><a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>,<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>,<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup>. Our findings in a widespread forest tree species suggest that adaptive SGV clusters non-randomly along geographic and climatic gradients, with decreasing alpha diversity and increasing compositional turnover (i.e., beta diversity) with distance away from the center of the current climatic niche (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 5</a> and <a id="xref-fig-6-4" class="xref-fig" href="#F6">6</a>). Contrary to expectations of loss of diversity due to founder effects during expansion from refugia, we observed relatively low SGV in low latitude populations<sup><a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">14</a></sup>. These findings strongly suggest that regionally common SGV is maintained in the climatic center of the range, while climatically marginal populations are unique in harboring locally reduced but regionally rare adaptive variation.</p><p id="p-42">In many ways, our findings mirror the Centre-Periphery Hypothesis (CPH), that predicts within-population genetic diversity should be highest near the abundant center of the range, while population differentiation should increase towards the range periphery where environments become less optimal and abundance drops<sup><a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">58</a></sup>. Gougherty et al.<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup> also found support for the CPH in poplar, using a larger range-wide sample of populations and focusing exclusively on the alpha diversity of putatively neutral SNPs sampled randomly from the genomic background. This suggests that both neutral and adaptive genomic diversity follows the predictions of the CPH because both are jointly influenced by <em>N</em> e. That is, we might expect higher alpha and lower beta diversity across the genome where the region of largest <em>N</em> e (which presumably reflects the demographic abundance of the species) is centered with the species’ environmental niche. This assumes the major selective influences shaping range limits are climatic in origin, thus aligning climate-adaptive variation with spatial locations of demographic abundance; however, this expectation might break down for genes under strong selection for functional responses not directly related to climate<sup><a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">13</a></sup>. The majority of the genomic outliers for adaptation studied here fit this prediction, with high alpha diversity (<em>pq</em>) and low beta diversity (PAI) near the geographic and climatic center of the range, consistent with the CPH.</p><p id="p-43">One notable exception occurred for outliers associated with response to biotic stress, which showed alpha diversity increased monotonically away from the rear edge, while beta diversity also increased with distance from the climatic niche (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Fig. 6</a>). Thus, genes associated with defense response appear to harbor regionally high levels of diversity in the same climatically marginal region of Atlantic Canada, while also maintaining high levels of within-population polymorphism for these genes. High levels of within-population polymorphism that are also regionally unique could be caused by several evolutionary processes, such as a history of spatial or temporally varying balancing selection, or selection-driven introgression from congeners. Both processees would be expected to lead to high levels of polymorphism within populations that may also be unique to the region where the selection or introgression is occurring. Regardless of the mechanism, the high alpha and beta diversity for biotic response in Atlantic Canada suggests the selective history on defense-related genes may be unique in this portion of the range.</p><p id="p-44">The Atlantic Canada populations (in our study, populations CBI, HWK, MBK &amp; FIS) occupy a maritime climate quite different from the rest of the range, characterized by cool temperatures and high precipitation amounts with low seasonality<sup><a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">48</a></sup>. We previously showed that balsam poplar in this region were divergent from the rest of the range for a suite of phenology and ecophysiological traits associated with climate adaptation, and that this phenotypic divergence was in excess of that predicted by background genetic structure, suggesting a unique history of local selection in Atlantic Canada<sup><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup>. In a separate study of <em>P. balsamifera</em>, Meirmans <em>et al</em><sup><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></sup> similarly reported a genetically distinct Atlantic Canada group that showed little admixture with the nearby range core ancestry group to its west, which they suggested was likely rooted in the local adaptation of the Atlantic Canada group to the unique maritime climate. Consistent with this proposition, the PAI for most of the local adaptation outliers in our study peaked in the Atlantic Canada group, whereas within-population allele frequency variances were lowest in this group, suggesting local selection driving SGV adaptive under these climate conditions towards fixation (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Fig 5b, d</a> &amp; <a id="xref-fig-6-6" class="xref-fig" href="#F6">6b, d</a>).</p><p id="p-45">It is important to note that landscape genomic studies of local adaptation invariably capture other selective gradients that are geographically aligned with climate; thus, what we infer as genes under climate-driven selection based on associations with temperature or precipitation gradients may actually reflect other unknown causal agents of selection. In our range-wide analysis, we found outlier genomic regions were significantly enriched for a variety of functions, including oxidoreductase activity, response to stress/DNA damage repair, cell signaling/transport, metal (iron) binding, and biosynthesis of cellulose, lipids, fatty acids, and flavonols (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Fig. 8</a>). Many of these enriched functions suggest a shared genetic architecture of response to abiotic and biotic stress, mediated through the action of heme oxygenases such as the diverse and biochemically active family of plant cytochrome P450 monooxygenases (450s)<sup><a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a></sup>. In plants, P450s are known to be key components of biosynthesis, metabolism, and signaling, involving responses to osmotic and temperature stress and the production of a variety of growth and defense compounds including phenylpropanoids, fatty acids, and hormones such as auxin and jasmonic acid<sup><a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">65</a>,<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a></sup>. Among the candidate genes repeatedly identified by all four selection scan methods, several are involved in phenylpropanoid biosynthesis (orthologs of <em>Arabidopsis thaliana</em> MYB4, CHALCONE SYNTHASE 4 (CHS4), a UDP-glycosyltransferase (UGT), and TRANSPARENT TESTA 16 (TT16) (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Phenylpropanoids are physiologically and ecologically important compounds that include lignins, anthocyanins, and condensed tannins (proanthocyanidins). Many of these compounds are known to be inducible in response to light and osmotic stress, and are also active as plant defenses against fungi, insects, and other natural enemies<sup><a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>–<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a></sup>. For example, MYB4 and CHS4 are both known to be responsive in <em>Populus</em> to infection by the fungal rust <em>Melampsora medusae</em>, a common leaf pathogen of poplars<sup><a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a></sup>. MYB4 expression is also responsive to jasmonic and salicylic acid, and UV-B light treatments in <em>Arabidopsis thaliana</em><sup><a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a></sup>, while in <em>Populus</em>, CHS4 expression is induced by wounding<sup><a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a>,<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a></sup>. A recent study of selection in 25 genes in ten steps of the condensed tannin synthesis pathway that included six CHS genes in <em>Populus</em> found elevated levels of nonsynonymous divergence at CHS genes<sup><a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a></sup>. The transcription factor PtMYB186 presents another intriguing example suggesting the action of local selection on genes with dual roles in both growth and defense (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). This homolog of AtMYB206 is associated with trichome initiation – an established defense against plant herbivores; however, over-expression of PtMYB186 in transgenic poplar results in pleiotropic effects on plant growth rate, even in the absence of herbivory<sup><a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a></sup>. Overall, the functional enrichment we observed among outlier genomic regions implicates pathways, individual genes, and functional domains associated with responses to environment and plant defense, suggesting the genetic architecture of locally adaptive SGV in <em>Populus</em> may have a large pleiotropic component, shared between both abiotic and biotic stress responses.</p><p id="p-46">The landscape of natural genetic variation involved in adaptation has become the increasing focus of attention of breeders, conservation biologists, and natural resource managers alike, who are facing the uncertain question of how existing populations are likely to respond to the novel pressures of global change, and where adaptive variation will be most needed in the future. Climate change conservation and mitigation strategies such as restoration or assisted gene flow<sup><a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a></sup> increasingly require spatially-explicit solutions that rely on identifying and preserving locally adaptive SGV in different parts of a species’ range. Thus, understanding local adaptation and the distribution of adaptive SGV in the context of range position has become prerequisite for global biodiversity conservation<sup><a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>,<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>,<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a></sup>. Our novel combination of landscape genomics analyses has revealed the current climatic niche is of primary importance in shaping the spatial distribution of locally adaptive SGV across the range of <em>Populus balsamifera</em>, and offers a promising general approach that could be used to prioritize regions for the conservation of unique genetic resources in other crop and wild species.</p></div><div class="section" id="sec-8"><h2 class="">Methods</h2><div id="sec-9" class="subsection"><h3>Biological Material</h3><p id="p-47">We targeted southern (rear) edge and nearby regions in the core of <em>Populus balsamifera</em>’s range for sampling, spanning from northern BC to Atlantic Canada (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Fig. 1</a>). The current sampling scheme therefore covers both the rear range edge, the central core of the range, and the climatically maritime region to the east. During winter 2013, we collected dormant stem cuttings from 508 trees from 57 populations (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Fig. 1</a>) spanning 9 Canadian Provinces and 7 US States (Longitudes 55-128<sup><em>◦</em></sup>W and Latitudes 39-60<sup><em>◦</em></sup>N; <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table S1</a>). The cuttings were flushed under greenhouse conditions and fresh foliage was used for extracting whole genomic DNA using DNeasy 96 Plant Mini Kits (Qiagen, Valencia, CA, USA). DNA was quantified using a fluorometric assay (Qubit BR; Invitrogen) and confirmed for high molecular weight using 1% agarose gel electrophoresis.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/966782/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/966782/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/966782" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1</span> <p id="p-66" class="first-child">Population locality information for 508 <em>Populus balsamifera</em> individuals used in this study.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Estimation of Geographic and Climatic Range Edges</h3><p id="p-48">We used two distance metrics to describe the position of each <em>P. balsamifera</em> population relative to the broader range of the species: geographic distance from the southern range edge, and a measure of climatic distance from the center of the range in climatic space (Supplementary Information). Both metrics were calculated from a set of <em>P. balsamifera</em> occurrences distributed throughout the range, collected from the Global Biodiversity Information Facility and US and Canadian forest inventory programs<sup><a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>,<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a></sup>. To reduce spatial bias in the occurrences, we thinned the occurrences in geographic space, such that occurrences were no nearer than 75 km, resulting in 401 unique occurrences throughout the range. Occurrences were thinned using the spThin<sup><a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a></sup> package in R.</p><p id="p-49">Geographic distances from the southern range edge were calculated by first fitting an alpha hull around the 401 occurrences. Alpha hulls are similar to convex hulls, but are recommended as a way to minimize effects of spatial bias and outliers when defining a species range polygon<sup><a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a></sup>. The southern range edge was defined as the southern-most line segments of the alpha hull, from which we calculated the minimum great circle distance to each <em>P. balsamifera</em> population. Next, we used Mahalanobis distance to quantify the distance of each population from the climatic centroid of the species. We used 19 bioclimatic variables (<a href="http://www.worldclim.org/bioclim">http://www.worldclim.org/bioclim</a>), elevation, and latitude to calculate the Mahalanobis distance, with the climatic center being defined as the average value of these variables across the 401 <em>P. balsamifera</em> occurrences. Mahalanobis distance is a unit-less, scale-invariant, multivariate distance metric that accounts for correlation among variables by scaling distances by their covariance (thereby ensuring correlated variables do not artificially inflate the measure of distance between locations). We used squared distances calculated with the Mahalanobis function in R<sup><a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a></sup>.</p></div><div id="sec-11" class="subsection"><h3>GBS Sequencing</h3><p id="p-50">We used genotyping-by-sequencing (GBS)<sup><a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a></sup> to obtain genome-wide polymorphism data for 508 trees. Genomic sequencing libraries were prepared from 100ng of genomic DNA per sample digested with EcoT22I restriction endonuclease followed by ligation of barcoded adapters of varying length from 4–8bp, following Elshire <em>et al</em><sup><a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">84</a></sup>. Equimolar concentrations of barcoded fragments were pooled and purified with QIAquick PCR purification kit. Purified products were amplified with 18 PCR cycles to append Illumina sequencing primers, cleaned again using PCR purification kit, and the resulting library was screened for distribution of fragment sizes using a Bioanalyzer. These libraries were sequenced at 48plex (i.e., each library sequenced twice) using Illumina HiSeq 2500 to generate 100bp single end reads. Cornell University Institute of Genomic Diversity (Ithaca, NY) performed the library construction and sequencing steps.</p><p id="p-51">We employed the Tassel GBS Pipeline<sup><a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">55</a></sup> to process raw sequence reads and call variants and genotypes. In order to pass the quality control, sequence reads had to have perfect barcode matches, presence of restriction site overhang and no undecipherable nucleotides (N’s). Filtered reads were trimmed to 64bp and aligned to the <em>P. trichocarpa</em> reference assembly version 3.0<sup><a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a></sup> using the Burrows-Wheeler Aligner (BWA)<sup><a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a></sup>. Single Nucleotide Polymorphisms (SNPs) were determined based on aligned positions to the <em>P. trichocarpa</em> reference, and genotypes called under the maximum likelihood framework in Tassel<sup><a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">55</a></sup>. SNP genotype data along with sequence quality scores were stored in Variant Call Format v4.1 (VCF) files, which were further processed with VCFTools 0.1.11<sup><a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a></sup>. SNP genotypes were filtered to retain only biallelic sites with no indels, individuals with <em>&lt;</em>50% missingness, sites with <em>&lt;</em>20% missingness, genotype quality (GQ) <em>&gt;</em>90, and site depth <em>&gt;</em>5. We further removed SNPs that showed excess heterozygosity in tests of Hardy Weinberg Equilibrium (<em>P &lt;</em>0.001). After filtering, the final dataset consisted of 167,324 SNPs for downstream analyses. Bioinformatic analyses were performed on a local Dell PowerEdge Linux server (24 cpu, 64 Gb RAM) at the University of Vermont and on the Teton computing environment<sup><a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a></sup> at the University of Wyoming.</p></div><div id="sec-12" class="subsection"><h3>Analysis of Population Structure</h3><p id="p-52">We characterized the genome-wide background genetic structure within our sample using the maximum likelihood clustering algorithm ADMIXTURE<sup><a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">89</a></sup>. We evaluated the likelihood models for presence of up to 7 Hardy-Weinberg clusters (<em>K</em>=1 through <em>K</em>=7; <a id="xref-fig-10-2" class="xref-fig" href="#F10">Fig S2</a>). Optimal <em>K</em> was inferred as the model that minimized the error rate based on 2000 bootstrap replicates across 10X cross-validation iterations. Cluster membership barplots were visualized using Distruct v2.3 Python script<sup><a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">90</a>,<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">91</a></sup>. Individuals exhibiting ancestry of 10% or more from other <em>Populus</em> sp. were removed from further analysis (N=26).</p><p id="p-53">As a complementary approach to ADMIXTURE, we used Discriminant Analysis of Principal Components (DAPC)<sup><a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">92</a></sup> to partition individuals along the major axes of genome-wide background genetic variation. The <em>K</em>-means clustering function used in DAPC (find.clusters()) can return different results from run to run; thus, we performed 1000 iterations of the function from <em>K</em>=1 to <em>K</em>=10, retaining the best unique solution that was most frequently observed (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Fig. S1</a>). Discriminant analyses using the most commonly observed unique solution was performed using 60% of the principal components and retaining two discriminant axes.</p></div><div id="sec-13" class="subsection"><h3>Identifying Genomic Regions of Local Adaptation</h3><p id="p-54">Genomic signals of natural selection appear as regions of elevated population differentiation across the genomic background. Quantifying background levels of differentiation is necessary for accommodating the highly variable levels of polymorphism found within the genome, and identifying SNPs potentially be under selection. In order to generate a neutral distribution from which to infer putatively locally adapted variants, we annotated the 167,324 SNPs with VCFCodingSNPs v1.5 using the <em>P. trichocarpa</em> reference genome annotation v3.0, classifying SNPs as Downstream, Upstream, 3’ UTR, 5’ UTR, Synonymous, Non-Synonymous and Intronic. SNPs <em>&gt;</em>5kb away from an annotated gene (hereafter called intergenic) were selected as the null set of loci for neutral parameterization of local adaptation genome scans following Lotterhos &amp; Whitlock<sup><a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">93</a></sup>.</p><p id="p-55">For selection scans, we ranked every candidate outlier using its test statistic among the empirical null distribution of intergenic SNPs (N=1,649 for range-wide and N=1,253 for range-core data sets) and determined its empirical <em>P</em>-value. This approach caps the lower end of the empirical <em>P</em>-value proportionate with the size of the null distribution. For example, the lowest possible empirical <em>P</em>-value for the range-wide data set is <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/24/817411/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/24/817411/embed/inline-graphic-2.gif"/></noscript></span></span> and that for the range-core data set is <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/24/817411/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/24/817411/embed/inline-graphic-3.gif"/></noscript></span></span>. To make the inference robust, we considered an outlier a candidate if its empirical <em>P</em>-value equaled the lowest possible.</p><p id="p-56">To scan for genomic regions showing elevated population differentiation, we applied the <em>F</em>-model implemented in BAYESCAN v2.1<sup><a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">94</a></sup> to identify SNPs under selection in range-wide as well as core groups of populations. We optimized run conditions with 20 pilot runs of 5000 MCMC sweeps each, followed by 100,000 MCMC sweeps after discarding the first 50,000 sweeps as burn-in. To minimize false positives, we used the recommendation of Lotterhos &amp; Whitlock<sup><a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">93</a></sup> by setting the prior expectation of selection to one in 10,000. Candidate SNPs were then assigned empirical <em>P</em>-values based on the empirical null distribution of intergenic SNPS at an FDR of 0.01 using the empPvals() function in the R package qvalue<sup><a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">95</a></sup>.</p><p id="p-57">Additionally we estimated <em>X</em><sup><em>T</em></sup><em>X</em>, analogous to <em>F</em><sub>ST</sub> using the correlated allele frequencies model in BAYENV2. The maximum expected value of this statistic assuming neutrality is equal to the total number of populations being tested. To avoid spurious selection candidates, we again performed neutral parameterization to obtain empirical <em>P</em>-values for all <em>X</em><sup><em>T</em></sup> <em>X</em> candidate loci and only retained those with lowest empirical <em>P</em>-value.</p><p id="p-58">We tested for association between SNPs and the environment using latent factor mixed models<sup><a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">96</a></sup> with the R package LFMM<sup><a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">97</a></sup>. For environmental predictors, we ordinated each individual in multivariate climate space based on a PCA of the 19 bioclimatic variables and latitude. Climatic PC1 and PC2 were then used as separate predictors in association tests. Analyses were conducted on the full set of individuals with <em>&gt;</em>90% <em>P. baslamifera</em> ancestry (N=437) and on a subset of individuals from the range core (N=336) that showed minimal background genetic structure (<a id="xref-fig-1-16" class="xref-fig" href="#F1">Fig. 1</a>). To determine appropriate values for the number of latent factors and genomic inflation factors for our datasets, we performed a cross-validation analysis across a range of latent factors (<em>K</em>=1 to <em>K</em>=10) and at three values of the genomic inflation factor (1e-10, 1, and 1e+20). We retained parameter values that minimized the cross-validation error, and used them to construct LFMMs with the ridge penalty on a SNP-wise basis. The statistical tests of each model produced lambda-adjusted z-scores, which we used to generate empirical <em>P</em>-values based on the empirical null distribution of intergenic SNPs. We accepted only candidate SNPs with the most extreme empirical <em>P</em>-values as significant.</p><p id="p-59">To allow for robust assessment of candidate outliers obtained from LFMM, we also performed gene-environment association analysis with BAYENV2<sup><a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">98</a></sup> on the same set of populations and climate PCs as above. First, a co-variance matrix of allele frequencies was estimated using the intergenic SNPs for range-wide and range-core data sets. Two hundred matrices were drawn (once every 5000 iterations) and the final matrix (200<sup>th</sup> after 100,000 iterations) was used as a control for population relatedness in our sample. We then ran BAYENV2 on a SNP-wise basis to estimate Bayes Factors for each SNP, and ranked the outliers within the neutral distribution of intergenic SNPs to obtain empirical <em>P</em>-values. Similar to LFMM, we only retained those outliers as candidates with the lowest empirical <em>P</em>-value.</p></div><div id="sec-14" class="subsection"><h3>Testing the Landscape Distribution of SGV</h3><p id="p-60">To investigate how locally adaptive standing genetic variation (SGV) was spatially distributed along landscape gradients of distance from the rear edge and from the climatic niche centroid, we calculated two populationlevel metrics of SGV based on the set of identified local adaptation outliers. We estimated a genetic index of alpha diversity (i.e., within-population variation) by calculating the allele frequency variance (<em>p × q</em>). We also calculated a genetic index of beta diversity (i.e., compositional turnover across regions) by calculating the Population Adaptive Index (PAI) <em>sensu</em> Bonin <em>et al</em><sup><a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a></sup>. Briefly, PAI is the absolute difference between adaptive allele frequencies of a focal population and that of the entire metapopulation. It therefore represents the among-population component of SGV contributed by each focal population compared to the rest of the range. To test whether the alpha and beta diversity components of SGV varied with proximity to the rear range edge or climatic niche center, we performed a multiple linear regression between each measure of SGV and the two predictor distances. Regression models were weighted so that residual errors were inversely proportional to the sample sizes (numbers of individuals) per population (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table S1</a>).</p></div><div id="sec-15" class="subsection"><h3>Functional Enrichment</h3><p id="p-61">We tested for functional enrichment of genomic regions implicated in local adaptation based on the Gene Ontology (GO) terms associated with the <em>P. trichocarpa</em> reference annotation v3.0. Enrichment of GO terms associated with genes nearby to candidate SNPs was assessed based on the hypergeometric test implemented in the R package SNP2GO<sup><a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">99</a></sup>. We tested for enrichment within a 50kb window upstream and downstream of each candidate SNP, and determined significance based on 1e+05 permutations and a false discovery rate (FDR) of 10%. Significantly enriched GO terms were then clustered for redundancy with REVIGO<sup><a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">100</a></sup>, using the SimRel semantic similarity metric and a clustering threshold of 0.7.</p></div></div><div class="section" id="sec-16"><h2 class="">Data Archiving</h2><p id="p-62">Illumina sequencing data are archived under NCBI Bioproject SAMN04517823 and NCBI SRA Accession Numbers SRX1605454-68. SNP genotyping data, climate PCA loadings, and outputs from selection scans and enrichment tests are available at <a href="https://github.com/stephenrkeller/Pbalsamifera">https://github.com/stephenrkeller/Pbalsamifera</a> SGV.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-63">We are grateful to the following individuals and agencies for assistance with germplasm collections: Andrew Elmore, George Howlett Jr., Steven Guinn, Garth Inouye, and the USDA National Forest Service. We thank members of the Keller lab for feedback on the manuscript. This study was funded by a National Science Foundation award 1461868 to S.R.K. and M.C.F.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-3"><p id="p-5">
<a href="https://github.com/stephenrkeller/Pbalsamifera_SGV">https://github.com/stephenrkeller/Pbalsamifera_SGV</a>
</p></li><li class="fn-dataset fn-group-external-links" id="fn-4"><p id="p-6">
<a href="https://www.ncbi.nlm.nih.gov/bioproject/306143">https://www.ncbi.nlm.nih.gov/bioproject/306143</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-817411v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R. A.</span></span> <span class="cit-source">The genetical theory of natural selection</span> (<span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, UK</span>, <span class="cit-pub-date">1930</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.2" data-doi="10.1038/nrg1523"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Orr</span>,  <span class="cit-name-given-names">H. A.</span></span> <span class="cit-article-title">The genetic theory of adaptation: a brief history</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">119</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D119%26rft.epage%253D127%26rft.atitle%253DThe%2Bgenetic%2Btheory%2Bof%2Badaptation%253A%2Ba%2Bbrief%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1523%26rft_id%253Dinfo%253Apmid%252F15716908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15716908&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226721300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">The evolutionary biology of sex</span>. <abbr class="cit-jnl-abbrev">Current Biology</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">R693</span>–<span class="cit-lpage">5</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D16%26rft.issue%253D17%26rft.spage%253DR693%26rft.epage%253DR695%26rft.atitle%253DThe%2Bevolutionary%2Bbiology%2Bof%2Bsex.%26rft_id%253Dinfo%253Apmid%252F16950095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16950095&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.4" data-doi="10.1111/nph.12441"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Delph</span>,  <span class="cit-name-given-names">L. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">J. K.</span></span> <span class="cit-article-title">On the importance of balancing selection in plants</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">201</span>, <span class="cit-fpage">45</span>–<span class="cit-lpage">56</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D201%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12441%26rft_id%253Dinfo%253Apmid%252F23952298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12441&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23952298&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mauricio</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Natural selection for polymorphism in the disease resistance gene Rps2 of <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>, <span class="cit-fpage">735</span>–<span class="cit-lpage">746</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMauricio%26rft.auinit1%253DR.%26rft.volume%253D163%26rft.issue%253D2%26rft.spage%253D735%26rft.epage%253D746%26rft.atitle%253DNatural%2BSelection%2Bfor%2BPolymorphism%2Bin%2Bthe%2BDisease%2BResistance%2BGene%2BRps2%2Bof%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Apmid%252F12618410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzIvNzM1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-817411v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fumagalli</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A population genetics study of the familial Mediterranean fever gene: evidence of balancing selection under an overdominance regime</span>. <abbr class="cit-jnl-abbrev">Genes and Immunity</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">678</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Band%2Bimmunity%26rft.stitle%253DGenes%2BImmun%26rft.aulast%253DFumagalli%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D678%26rft.epage%253D686%26rft.atitle%253DA%2Bpopulation%2Bgenetics%2Bstudy%2Bof%2Bthe%2Bfamilial%2BMediterranean%2Bfever%2Bgene%253A%2Bevidence%2Bof%2Bbalancing%2Bselection%2Bunder%2Ban%2Boverdominance%2Bregime.%26rft_id%253Dinfo%253Apmid%252F19675583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19675583&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gos</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Slotte</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">S. I.</span></span> <span class="cit-article-title">Signatures of balancing selection are maintained at disease resistance loci following mating system evolution and a population bottleneck in the genus <em>Capsella</em></span>. <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">152</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D12%26rft.spage%253D152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.8" data-doi="10.1073/pnas.0914625107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hancock</span>,  <span class="cit-name-given-names">A. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Human adaptations to diet, subsistence, and ecoregion are due to subtle shifts in allele frequency</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">107</span>, <span class="cit-fpage">8924</span>–<span class="cit-lpage">8930</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0914625107%26rft_id%253Dinfo%253Apmid%252F20445095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoyMToiMTA3L1N1cHBsZW1lbnRfMi84OTI0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Messer</span>,  <span class="cit-name-given-names">P. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Petrov</span>,  <span class="cit-name-given-names">D. A.</span></span> <span class="cit-article-title">Population genomics of rapid adaptation by soft selective sweeps</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">659</span>–<span class="cit-lpage">669</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D28%26rft.spage%253D659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.10" data-doi="10.1038/35016000"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hewitt</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">The genetic legacy of the Quaternary ice ages</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">405</span>, <span class="cit-fpage">907</span>–<span class="cit-lpage">913</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D405%26rft.issue%253D6789%26rft.spage%253D907%26rft.epage%253D913%26rft.atitle%253DThe%2Bgenetic%2Blegacy%2Bof%2Bthe%2BQuaternary%2Bice%2Bages%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35016000%26rft_id%253Dinfo%253Apmid%252F10879524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35016000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2000073425&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=10879524&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087732700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-817411v1.11" data-doi="10.1146/annurev.ecolsys.39.110707.173414"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foll</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Petit</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-article-title">Genetic consequences of range expansions</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">40</span>, <span class="cit-fpage">481</span>–<span class="cit-lpage">501</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D40%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000272455700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hampe</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jump</span>,  <span class="cit-name-given-names">A. S.</span></span> <span class="cit-article-title">Climate relicts: past, present, future</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">42</span>, <span class="cit-fpage">313</span>–<span class="cit-lpage">333</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D42%26rft.spage%253D313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pironon</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Geographic variation in genetic and demographic performance: new insights from an old biogeographical paradigm</span>. <abbr class="cit-jnl-abbrev">Biological Reviews</abbr> <span class="cit-vol">92</span>, <span class="cit-fpage">1877</span>–<span class="cit-lpage">1909</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%26rft.volume%253D92%26rft.spage%253D1877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.14" data-doi="10.1111/j.1461-0248.2005.00739.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hampe</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Petit</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-article-title">Conserving biodiversity under climate change: the rear edge matters</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">461</span>–<span class="cit-lpage">467</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DHampe%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D461%26rft.epage%253D467%26rft.atitle%253DConserving%2Bbiodiversity%2Bunder%2Bclimate%2Bchange%253A%2Bthe%2Brear%2Bedge%2Bmatters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2005.00739.x%26rft_id%253Dinfo%253Apmid%252F21352449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2005.00739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21352449&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rehm</span>,  <span class="cit-name-given-names">E. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olivas</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stroud</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Feeley</span>,  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-article-title">Losing your edge: climate change and the conservation value of range-edge populations</span>. <abbr class="cit-jnl-abbrev">Ecology and Evolution</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">4315</span>–<span class="cit-lpage">4326</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2Band%2BEvolution%26rft.volume%253D5%26rft.spage%253D4315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.16" data-doi="10.1534/genetics.112.139022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Excoffier</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Serial founder effects during range expansion: a spatial analog of genetic drift</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>, <span class="cit-fpage">171</span>–<span class="cit-lpage">181</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.139022%26rft_id%253Dinfo%253Apmid%252F22367031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzEvMTcxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.17" data-doi="10.1126/science.1157570"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pujol</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pannell</span>,  <span class="cit-name-given-names">J. R.</span></span> <span class="cit-article-title">Reduced responses to selection after species range expansion</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">321</span>, <span class="cit-fpage">96</span>–<span class="cit-lpage">96</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPujol%26rft.auinit1%253DB.%26rft.volume%253D321%26rft.issue%253D5885%26rft.spage%253D96%26rft.epage%253D96%26rft.atitle%253DReduced%2BResponses%2Bto%2BSelection%2BAfter%2BSpecies%2BRange%2BExpansion%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1157570%26rft_id%253Dinfo%253Apmid%252F18599779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjEvNTg4NS85NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzI0LzgxNzQxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Louthan</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doak</span>,  <span class="cit-name-given-names">D. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Angert</span>,  <span class="cit-name-given-names">A. L.</span></span> <span class="cit-article-title">Where and when do species interactions set range limits</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">780</span>–<span class="cit-lpage">792</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D30%26rft.spage%253D780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chhatre</span>,  <span class="cit-name-given-names">V. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">DiFazio</span>,  <span class="cit-name-given-names">S. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span> <span class="cit-article-title">Adaptive introgression and maintenance of a trispecies hybrid complex in range-edge populations of <em>Populus</em></span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">4820</span>–<span class="cit-lpage">4838</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D27%26rft.spage%253D4820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fetter</span>,  <span class="cit-name-given-names">K. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">D. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span> <span class="cit-article-title">Trade-offs and selection conflicts in hybrid poplars indicate the stomatal ratio as an important trait regulating disease resistance</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, <span class="cit-fpage">814046</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schoville</span>,  <span class="cit-name-given-names">S. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Adaptive genetic variation on the landscape: methods and cases</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">43</span>, <span class="cit-fpage">23</span>–<span class="cit-lpage">43</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D43%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>,  <span class="cit-name-given-names">S. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Llewelyn</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moritz</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">B. L.</span></span> <span class="cit-article-title">Peripheral isolates as sources of adaptive diversity under climate change</span>. <abbr class="cit-jnl-abbrev">Frontiers in Ecology and Evolution</abbr> <span class="cit-vol">5</span>, <span class="cit-fpage">88</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D5%26rft.spage%253D88%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Antonovics</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">The nature of limits to natural selection</span>. <abbr class="cit-jnl-abbrev">Annals of the Missouri Botanical Garden</abbr>, <span class="cit-fpage">224</span>–<span class="cit-lpage">247</span> (<span class="cit-pub-date">1976</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bridle</span>,  <span class="cit-name-given-names">J. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Vines</span>,  <span class="cit-name-given-names">T. H.</span></span> <span class="cit-article-title">Limits to evolution at range margins: when and why does adaptation fail</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">140</span>–<span class="cit-lpage">147</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D22%26rft.spage%253D140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-817411v1.25" data-doi="10.1038/sj.hdy.6800987"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mimura</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Aitken</span>,  <span class="cit-name-given-names">S. N.</span></span> <span class="cit-article-title">Adaptive gradients and isolation-by-distance with postglacial migration in <em>Picea sitchensis</em></span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">99</span>, <span class="cit-fpage">224</span>–<span class="cit-lpage">232</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMimura%26rft.auinit1%253DM.%26rft.volume%253D99%26rft.issue%253D2%26rft.spage%253D224%26rft.epage%253D232%26rft.atitle%253DAdaptive%2Bgradients%2Band%2Bisolation-by-distance%2Bwith%2Bpostglacial%2Bmigration%2Bin%2BPicea%2Bsitchensis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800987%26rft_id%253Dinfo%253Apmid%252F17487214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17487214&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248301900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-817411v1.26" data-doi="10.1073/pnas.1100404108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sexton</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strauss</span>,  <span class="cit-name-given-names">S. Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rice</span>,  <span class="cit-name-given-names">K. J.</span></span> <span class="cit-article-title">Gene flow increases fitness at the warm edge of a species’ range</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">108</span>, <span class="cit-fpage">11704</span>–<span class="cit-lpage">11709</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1100404108%26rft_id%253Dinfo%253Apmid%252F21709253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI4LzExNzA0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.27" data-doi="10.1098/rspb.2011.1805"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holliday</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suren</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Aitken</span>,  <span class="cit-name-given-names">S. N.</span></span> <span class="cit-article-title">Divergent selection and heterogeneous migration rates across the range of Sitka spruce (<em>Picea sitchensis</em>)</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B</abbr> <span class="cit-vol">279</span>, <span class="cit-fpage">1675</span>–<span class="cit-lpage">1683</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2011.1805%26rft_id%253Dinfo%253Apmid%252F22113032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2011.1805&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22113032&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gould</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Local adaptation and range boundary formation in response to complex environmental gradients across the geographical range of <em>Clarkia xantiana</em> ssp. <em>xantiana</em></span>. <abbr class="cit-jnl-abbrev">Journal of Ecology</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">95</span>–<span class="cit-lpage">107</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEcology%26rft.volume%253D102%26rft.spage%253D95%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.29" data-doi="10.1002/ecy.2382"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Lafontaine</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Napier</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petit</span>,  <span class="cit-name-given-names">R. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">F. S.</span></span> <span class="cit-article-title">Invoking adaptation to decipher the genetic legacy of past climate change</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">99</span>, <span class="cit-fpage">1530</span>–<span class="cit-lpage">1546</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D99%26rft.spage%253D1530%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fecy.2382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ecy.2382&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schiffers</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Landscape structure and genetic architecture jointly impact rates of niche evolution</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">1218</span>–<span class="cit-lpage">1229</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D37%26rft.spage%253D1218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.31" data-doi="10.1073/pnas.1421515112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Polechová</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span> <span class="cit-article-title">Limits to adaptation along environmental gradients</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">112</span>, <span class="cit-fpage">6401</span>–<span class="cit-lpage">6406</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1421515112%26rft_id%253Dinfo%253Apmid%252F25941385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzIwLzY0MDEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8yNC84MTc0MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.32" data-doi="10.1086/286054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span> <span class="cit-article-title">Evolution of a species’ range</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">150</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D150%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1086%252F286054%26rft_id%253Dinfo%253Apmid%252F18811273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/286054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18811273&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XE52000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">R. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Barfield</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Theoretical perspectives on the statics and dynamics of species’ borders in patchy environments</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">178</span> <span class="cit-issue">Suppl 1</span>, S<span class="cit-fpage">6</span>–<span class="cit-lpage">25</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D178%26rft.spage%253D6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chhatre</span>,  <span class="cit-name-given-names">V. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rajora</span>,  <span class="cit-name-given-names">O. P.</span></span> <span class="cit-article-title">Genetic divergence and signatures of natural selection in marginal populations of a keystone, long-lived conifer, eastern white pine (<em>Pinus strobus</em>) from Northern Ontario</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">e97291</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D9%26rft.spage%253De97291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bellon</span>,  <span class="cit-name-given-names">M. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dulloo</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sardos</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thormann</span>,  <span class="cit-name-given-names">I.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Burdon</span>,  <span class="cit-name-given-names">J. J.</span></span> <span class="cit-article-title"><em>In situ</em> conservationharnessing natural and human-derived evolutionary forces to ensure future crop adaptation</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">965</span>–<span class="cit-lpage">977</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D10%26rft.spage%253D965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.36" data-doi="10.1126/science.1247579"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garcia</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cabeza</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rahbek</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Araüjo</span>,  <span class="cit-name-given-names">M. B.</span></span> <span class="cit-article-title">Multiple dimensions of climate change and their implications for biodiversity</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">344</span>, <span class="cit-fpage">1247579</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGarcia%26rft.auinit1%253DR.%2BA.%26rft.volume%253D344%26rft.issue%253D6183%26rft.spage%253D1247579%26rft.epage%253D1247579%26rft.atitle%253DMultiple%2BDimensions%2Bof%2BClimate%2BChange%2Band%2BTheir%2BImplications%2Bfor%2BBiodiversity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1247579%26rft_id%253Dinfo%253Apmid%252F24786084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDQvNjE4My8xMjQ3NTc5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-web" id="cit-817411v1.37"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">National Academies of Sciences, E. &amp; Medicine</span>, <span class="cit-source">Forest Health and Biotechnology: Possibilities and Considerations</span> <a href="https://doi.org/10.17226/25221">https://doi.org/10.17226/25221</a> (<span class="cit-publ-name">National Academies Press</span>, <span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.38" data-doi="10.1126/science.292.5517.673"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">M. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Shaw</span>,  <span class="cit-name-given-names">R. G.</span></span> <span class="cit-article-title">Range shifts and adaptive responses to Quaternary climate change</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">292</span>, <span class="cit-fpage">673</span>–<span class="cit-lpage">679</span> (<span class="cit-pub-date">2001</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDavis%26rft.auinit1%253DM.%2BB.%26rft.volume%253D292%26rft.issue%253D5517%26rft.spage%253D673%26rft.epage%253D679%26rft.atitle%253DRange%2BShifts%2Band%2BAdaptive%2BResponses%2Bto%2BQuaternary%2BClimate%2BChange%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.292.5517.673%26rft_id%253Dinfo%253Apmid%252F11326089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTIvNTUxNy82NzMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8yNC84MTc0MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.39" data-doi="10.1111/nph.12929"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gavin</span>,  <span class="cit-name-given-names">D. G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Climate refugia: joint inference from fossil records, species distribution models and phylogeography</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">204</span>, <span class="cit-fpage">37</span>–<span class="cit-lpage">54</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D204%26rft.spage%253D37%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12929%26rft_id%253Dinfo%253Apmid%252F25039238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25039238&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000341193500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.40" data-doi="10.1146/annurev.ecolsys.37.091305.110215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petit</span>,  <span class="cit-name-given-names">R. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hampe</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Some evolutionary consequences of being a tree</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">187</span>–<span class="cit-lpage">214</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D37%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.37.091305.110215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.37.091305.110215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000243038500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.41" data-doi="10.1146/annurev.ecolsys.38.091206.095646"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Knürr</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Gene flow and local adaptation in trees</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">38</span>, <span class="cit-fpage">595</span>–<span class="cit-lpage">619</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D38%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.38.091206.095646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.38.091206.095646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252359800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-817411v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckert</span>,  <span class="cit-name-given-names">A. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Patterns of population structure and environmental associations to aridity across the range of loblolly pine (<em>Pinus taeda</em> L., <em>Pinaceae</em>)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <a href="https://doi.org/10.1534/genetics.110.115543">https://doi.org/10.1534/genetics.110.115543</a> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.43" data-doi="10.1093/molbev/mss121"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levsen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olson</span>,  <span class="cit-name-given-names">M. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tiffin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Local adaptation in the flowering-time gene network of balsam poplar, <em>Populus balsamifera</em> L</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr> <span class="cit-vol">29</span>, <span class="cit-fpage">3143</span>–<span class="cit-lpage">3152</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss121%26rft_id%253Dinfo%253Apmid%252F22513286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22513286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309927900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-817411v1.44" data-doi="10.3732/ajb.1500162"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sork</span>,  <span class="cit-name-given-names">V. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Landscape genomic analysis of candidate genes for climate adaptation in a California endemic oak, <em>Quercus lobata</em></span>. <abbr class="cit-jnl-abbrev">American Journal of Botany</abbr> <span class="cit-vol">103</span>, <span class="cit-fpage">33</span>–<span class="cit-lpage">46</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1500162%26rft_id%253Dinfo%253Apmid%252F26744482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6IjEwMy8xLzMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.45" data-doi="10.1126/science.aaf7812"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Convergent local adaptation to climate in distantly related conifers</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>, <span class="cit-fpage">1431</span>–<span class="cit-lpage">1433</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYeaman%26rft.auinit1%253DS.%26rft.volume%253D353%26rft.issue%253D6306%26rft.spage%253D1431%26rft.epage%253D1433%26rft.atitle%253DConvergent%2Blocal%2Badaptation%2Bto%2Bclimate%2Bin%2Bdistantly%2Brelated%2Bconifers%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf7812%26rft_id%253Dinfo%253Apmid%252F27708038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTMvNjMwNi8xNDMxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sork</span>,  <span class="cit-name-given-names">V.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Putting the landscape into the genomics of trees: approaches for understanding local adaptation and population responses to changing climate</span>. <abbr class="cit-jnl-abbrev">Tree Genetics &amp; Genomes</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">901</span>–<span class="cit-lpage">911</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenetics%2B%2526%2BGenomes%26rft.volume%253D9%26rft.spage%253D901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-817411v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Little</span>,  <span class="cit-name-given-names">E. L.</span></span> <span class="cit-chapter-title">Atlas of United States trees</span>. Volume <span class="cit-vol">1</span>. <span class="cit-source">Conifers and important hardwoods. Miscellaneous publication</span> <span class="cit-fpage">1146</span>. <span class="cit-publ-name">US Department of Agriculture, Forest Service</span>, <span class="cit-publ-loc">Washington, DC</span> (<span class="cit-pub-date">1971</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConifers%2Band%2Bimportant%2Bhardwoods.%2BMiscellaneous%2Bpublication%26rft.volume%253D1%26rft.spage%253D1146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chhatre</span>,  <span class="cit-name-given-names">V. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>,  <span class="cit-name-given-names">M. C.</span></span> <span class="cit-article-title">Influence of range position on locally adaptive gene–environment associations in <em>Populus</em> flowering time genes</span>. <abbr class="cit-jnl-abbrev">Journal of Heredity</abbr> <span class="cit-vol">109</span>, <span class="cit-fpage">47</span>–<span class="cit-lpage">58</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.volume%253D109%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.49" data-doi="10.1111/j.1365-294X.2010.04546.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olson</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silim</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schroeder</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tiffin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Genomic diversity, population structure, and migration following rapid range expansion in the Balsam Poplar, <em>Populus balsamifera</em></span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">1212</span>–<span class="cit-lpage">1226</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DKeller%26rft.auinit1%253DS.%2BR.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D1212%26rft.epage%253D1226%26rft.atitle%253DGenomic%2Bdiversity%252C%2Bpopulation%2Bstructure%252C%2Band%2Bmigration%2Bfollowing%2Brapid%2Brange%2Bexpansion%2Bin%2Bthe%2BBalsam%2Bpoplar%252C%2BPopulus%2Bbalsamifera.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04546.x%26rft_id%253Dinfo%253Apmid%252F20163548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04546.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20163548&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.50" data-doi="10.1093/sysbio/syr120"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Levsen</span>,  <span class="cit-name-given-names">N. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tiffin</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Olson</span>,  <span class="cit-name-given-names">M. S.</span></span> <span class="cit-article-title">Pleistocene speciation in the genus <em>Populus</em> (<em>Salicaceae</em>)</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr> <span class="cit-vol">61</span>, <span class="cit-fpage">401</span>–<span class="cit-lpage">412</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyr120%26rft_id%253Dinfo%253Apmid%252F22213709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syr120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22213709&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.51" data-doi="10.1111/j.1365-3040.2009.02042.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soolanayakanahally</span>,  <span class="cit-name-given-names">R. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guy</span>,  <span class="cit-name-given-names">R. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silim</span>,  <span class="cit-name-given-names">S. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drewes</span>,  <span class="cit-name-given-names">E. C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schroeder</span>,  <span class="cit-name-given-names">W. R.</span></span> <span class="cit-article-title">Enhanced assimilation rate and water use efficiency with latitude through increased photosynthetic capacity and internal conductance in balsam poplar (<em>Populus balsamifera</em> L</span>. <abbr class="cit-jnl-abbrev">Plant, Cell &amp; Environment</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">1821</span>–<span class="cit-lpage">1832</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.stitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.aulast%253DSoolanayakanahally%26rft.auinit1%253DR.%2BY.%26rft.volume%253D32%26rft.issue%253D12%26rft.spage%253D1821%26rft.epage%253D1832%26rft.atitle%253DEnhanced%2Bassimilation%2Brate%2Band%2Bwater%2Buse%2Befficiency%2Bwith%2Blatitude%2Bthrough%2Bincreased%2Bphotosynthetic%2Bcapacity%2Band%2Binternal%2Bconductance%2Bin%2Bbalsam%2Bpoplar%2B%2528Populus%2Bbalsamifera%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3040.2009.02042.x%26rft_id%253Dinfo%253Apmid%252F19712064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3040.2009.02042.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19712064&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271526500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.52" data-doi="10.3732/ajb.1000317"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Climate-driven local adaptation of ecophysiology and phenology in balsam poplar, <em>Populus balsamifera</em> L. (<em>Salicaceae</em>)</span>. <abbr class="cit-jnl-abbrev">American Journal of Botany</abbr> <span class="cit-vol">98</span>, <span class="cit-fpage">99</span>–<span class="cit-lpage">108</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1000317%26rft_id%253Dinfo%253Apmid%252F21613088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjc6Ijk4LzEvOTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8yNC84MTc0MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.53" data-doi="10.1111/mec.12067"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olson</span>,  <span class="cit-name-given-names">M. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The adaptive potential of <em>Populus balsamifera</em> L. to phenology requirements in a warmer global climate</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">22</span>, <span class="cit-fpage">1214</span>–<span class="cit-lpage">1230</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D22%26rft.spage%253D1214%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.54" data-doi="10.1534/genetics.111.128041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levsen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>,  <span class="cit-name-given-names">P. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olson</span>,  <span class="cit-name-given-names">M. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tiffin</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Local selection across a latitudinal gradient shapes nucleotide diversity in balsam poplar, <em>Populus balsamifera</em> L</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>, <span class="cit-fpage">941</span>–<span class="cit-lpage">952</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128041%26rft_id%253Dinfo%253Apmid%252F21624997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzQvOTQxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.55" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">e90346</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.56" data-doi="10.1111/j.1523-1739.2007.00685.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonin</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicole</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pompanon</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miaud</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Taberlet</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-article-title">Population adaptive index: a new method to help measure intraspecific genetic diversity and prioritize populations for conservation</span>. <abbr class="cit-jnl-abbrev">Conservation Biology</abbr> <span class="cit-vol">21</span>, <span class="cit-fpage">697</span>–<span class="cit-lpage">708</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BBiology%26rft.stitle%253DConservation%2BBiology%26rft.aulast%253DBonin%26rft.auinit1%253DA.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D697%26rft.epage%253D708%26rft.atitle%253DPopulation%2Badaptive%2Bindex%253A%2Ba%2Bnew%2Bmethod%2Bto%2Bhelp%2Bmeasure%2Bintraspecific%2Bgenetic%2Bdiversity%2Band%2Bprioritize%2Bpopulations%2Bfor%2Bconservation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1523-1739.2007.00685.x%26rft_id%253Dinfo%253Apmid%252F17531048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1523-1739.2007.00685.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17531048&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246843200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.57" data-doi="10.1111/j.1469-8137.2009.02807.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sjdin</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Street</span>,  <span class="cit-name-given-names">N. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandberg</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gustafsson</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Jansson</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">The Populus Genome Integrative Explorer (PopGenIE): a new resource for exploring the Populus genome</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">182</span>, <span class="cit-fpage">1013</span>–<span class="cit-lpage">1025</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSjodin%26rft.auinit1%253DA.%26rft.volume%253D182%26rft.issue%253D4%26rft.spage%253D1013%26rft.epage%253D1025%26rft.atitle%253DThe%2BPopulus%2BGenome%2BIntegrative%2BExplorer%2B%2528PopGenIE%2529%253A%2Ba%2Bnew%2Bresource%2Bfor%2Bexploring%2Bthe%2BPopulus%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.02807.x%26rft_id%253Dinfo%253Apmid%252F19383103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.02807.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19383103&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265936500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.58" data-doi="10.1111/j.1365-294X.2007.03659.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eckert</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samis</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lougheed</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Genetic variation across species geographical ranges: the central–marginal hypothesis and beyond</span>. <abbr class="cit-jnl-abbrev">Molecular ecology</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">1170</span>–<span class="cit-lpage">1188</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEckert%26rft.auinit1%253DC.%2BG.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D1170%26rft.epage%253D1188%26rft.atitle%253DGenetic%2Bvariation%2Bacross%2Bspecies%2527%2Bgeographical%2Branges%253A%2Bthe%2Bcentral-marginal%2Bhypothesis%2Band%2Bbeyond.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03659.x%26rft_id%253Dinfo%253Apmid%252F18302683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03659.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18302683&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253481600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sork</span>,  <span class="cit-name-given-names">V. L.</span></span> <span class="cit-article-title">Genomic studies of local adaptation in natural plant populations</span>. <abbr class="cit-jnl-abbrev">Journal of Heredity</abbr> <span class="cit-vol">109</span>, <span class="cit-fpage">3</span>–<span class="cit-lpage">15</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.volume%253D109%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-book" id="cit-817411v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fetter</span>,  <span class="cit-name-given-names">K. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gugger</span>,  <span class="cit-name-given-names">P. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span> <span class="cit-chapter-title">Landscape Genomics of Angiosperm Trees: From Historic Roots to Discovering New Branches of Adaptive Evolution</span> in <span class="cit-source">Comparative and Evolutionary Genomics of Angiosperm Trees</span> <span class="cit-fpage">303</span>–<span class="cit-lpage">333</span> (<span class="cit-publ-name">Springer</span>, <span class="cit-pub-date">2017</span>). <a href="https://doi.org/10.1007/7397201619">https://doi.org/10.1007/7397201619</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.61" data-doi="10.1126/science.7809610"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Travisano</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mongold</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bennett</span>,  <span class="cit-name-given-names">A. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lenski</span>,  <span class="cit-name-given-names">R. E.</span></span> <span class="cit-article-title">Experimental tests of the roles of adaptation, chance, and history in evolution</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">267</span>, <span class="cit-fpage">87</span>–<span class="cit-lpage">90</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTravisano%26rft.auinit1%253DM%26rft.volume%253D267%26rft.issue%253D5194%26rft.spage%253D87%26rft.epage%253D90%26rft.atitle%253DExperimental%2Btests%2Bof%2Bthe%2Broles%2Bof%2Badaptation%252C%2Bchance%252C%2Band%2Bhistory%2Bin%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.7809610%26rft_id%253Dinfo%253Apmid%252F7809610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyNjcvNTE5NC84NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzI0LzgxNzQxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.62" data-doi="10.1111/j.1461-0248.2008.01188.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">S. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Taylor</span>,  <span class="cit-name-given-names">D. R.</span></span> <span class="cit-article-title">History, chance and adaptation during biological invasion: separating stochastic phenotypic evolution from response to selection</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">852</span>–<span class="cit-lpage">866</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DKeller%26rft.auinit1%253DS.%2BR.%26rft.volume%253D11%26rft.issue%253D8%26rft.spage%253D852%26rft.epage%253D866%26rft.atitle%253DHistory%252C%2Bchance%2Band%2Badaptation%2Bduring%2Bbiological%2Binvasion%253A%2Bseparating%2Bstochastic%2Bphenotypic%2Bevolution%2Bfrom%2Bresponse%2Bto%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2008.01188.x%26rft_id%253Dinfo%253Apmid%252F18422638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2008.01188.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18422638&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257655400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gougherty A</span>,  <span class="cit-name-given-names">C. V.K. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Contemporary range position predicts the range-wide pattern of genetic diversity in balsam poplar, <em>Populus blasmiafera</em> L.) (unpublished ms)</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meirmans</span>,  <span class="cit-name-given-names">P. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Godbout</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lamothe</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">S. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Isabel</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">History rather than hybridization determines population structure and adaptation in <em>Populus balsamifera</em></span>. <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">2044</span>–<span class="cit-lpage">2058</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEvolutionary%2BBiology%26rft.volume%253D30%26rft.spage%253D2044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.65" data-doi="10.1146/annurev.arplant.54.031902.134840"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schuler</span>,  <span class="cit-name-given-names">M. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Werck-Reichhart</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Functional Genomics of P450S</span>. <abbr class="cit-jnl-abbrev">Annual Review of Plant Biology</abbr> <span class="cit-vol">54</span>.<span class="cit-pub-id-sep cit-pub-id-pmid-sep"> </span><span class="cit-pub-id-scheme">PMID: </span><span class="cit-pub-id cit-pub-id-pmid">14503006</span>, <span class="cit-fpage">629</span>–<span class="cit-lpage">667</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BPlant%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.54.031902.134840%26rft_id%253Dinfo%253Apmid%252F14503006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.54.031902.134840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14503006&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185094100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.66" data-doi="10.1111/j.1365-313X.2011.04529.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Werck-Reichhart</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">A P450-centric view of plant evolution</span>. <abbr class="cit-jnl-abbrev">The Plant Journal</abbr> <span class="cit-vol">66</span>, <span class="cit-fpage">194</span>–<span class="cit-lpage">211</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DNelson%26rft.auinit1%253DD.%26rft.volume%253D66%26rft.issue%253D1%26rft.spage%253D194%26rft.epage%253D211%26rft.atitle%253DA%2BP450-centric%2Bview%2Bof%2Bplant%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2011.04529.x%26rft_id%253Dinfo%253Apmid%252F21443632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2011.04529.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21443632&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288862500014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-book" id="cit-817411v1.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lindroth</span>,  <span class="cit-name-given-names">R. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hwang</span>,  <span class="cit-name-given-names">S.-Y.</span></span> <span class="cit-chapter-title">Diversity, redundancy, and multiplicity in chemical defense systems of aspen</span> in <span class="cit-source">Phytochemical Diversity and Redundancy in Ecological Interactions</span> <span class="cit-fpage">25</span>–<span class="cit-lpage">56</span> (<span class="cit-publ-name">Springer</span>, <span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">68.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-817411v1.68" data-doi="10.1007/s004420050213"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hwang</span>,  <span class="cit-name-given-names">S.-Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lindroth</span>,  <span class="cit-name-given-names">R. L.</span></span> <span class="cit-article-title">Clonal variation in foliar chemistry of aspen: effects on gypsy moths and forest tent caterpillars</span>. <abbr class="cit-jnl-abbrev">Oecologia</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">99</span>–<span class="cit-lpage">108</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOecologia%26rft.volume%253D111%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004420050213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004420050213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XF86000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gould</span>,  <span class="cit-name-given-names">K. S.</span></span> <span class="cit-article-title">Nature’s Swiss army knife: the diverse protective roles of anthocyanins in leaves</span>. <abbr class="cit-jnl-abbrev">BioMed Research International</abbr> <span class="cit-vol">2004</span>, <span class="cit-fpage">314</span>–<span class="cit-lpage">320</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioMed%2BResearch%2BInternational%26rft.volume%253D2004%26rft.spage%253D314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.70" data-doi="10.1094/MPMI-20-7-0816"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miranda</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The transcriptional response of hybrid poplar (<em>Populus trichocarpa × P. deltoides</em>) to infection by <em>Melampsora medusae</em> leaf rust involves induction of flavonoid pathway genes leading to the accumulation of proanthocyanidins</span>. <abbr class="cit-jnl-abbrev">Molecular Plant-Microbe Interactions</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">816</span>–<span class="cit-lpage">831</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DMiranda%26rft.auinit1%253DM.%26rft.volume%253D20%26rft.issue%253D7%26rft.spage%253D816%26rft.epage%253D831%26rft.atitle%253DThe%2Btranscriptional%2Bresponse%2Bof%2Bhybrid%2Bpoplar%2B%2528Populus%2Btrichocarpa%2Bx%2BP.%2Bdeltoides%2529%2Bto%2Binfection%2Bby%2BMelampsora%2Bmedusae%2Bleaf%2Brust%2Binvolves%2Binduction%2Bof%2Bflavonoid%2Bpathway%2Bgenes%2Bleading%2Bto%2Bthe%2Baccumulation%2Bof%2Bproanthocyanidins.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-20-7-0816%26rft_id%253Dinfo%253Apmid%252F17601169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-20-7-0816&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17601169&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247352400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.71" data-doi="10.1105/tpc.010410"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Expression profile matrix of <em>Arabidopsis</em> transcription factor genes suggests their putative functions in response to environmental stresses</span>. <abbr class="cit-jnl-abbrev">The Plant Cell</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">559</span>–<span class="cit-lpage">574</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.010410%26rft_id%253Dinfo%253Apmid%252F11910004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE0LzMvNTU5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.72" data-doi="10.1007/s00299-011-1075-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Isolation and promoter analysis of a chalcone synthase gene PtrCHS4 from <em>Populus trichocarpa</em></span>. <abbr class="cit-jnl-abbrev">Plant Cell Reports</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1661</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bcell%2Breports%26rft.stitle%253DPlant%2BCell%2BRep%26rft.aulast%253DSun%26rft.auinit1%253DY.%26rft.volume%253D30%26rft.issue%253D9%26rft.spage%253D1661%26rft.epage%253D1671%26rft.atitle%253DIsolation%2Band%2Bpromoter%2Banalysis%2Bof%2Ba%2Bchalcone%2Bsynthase%2Bgene%2BPtrCHS4%2Bfrom%2BPopulus%2Btrichocarpa.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00299-011-1075-1%26rft_id%253Dinfo%253Apmid%252F21553109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00299-011-1075-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21553109&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293794500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.73" data-doi="10.1111/j.1469-8137.2006.01798.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">C.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harding</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tschaplinski</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindroth</span>,  <span class="cit-name-given-names">R. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-article-title">Genome-wide analysis of the structural genes regulating defense phenylpropanoid metabolism in <em>Populus</em></span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">172</span>, <span class="cit-fpage">47</span>–<span class="cit-lpage">62</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DTsai%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D172%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D62%26rft.atitle%253DGenome-wide%2Banalysis%2Bof%2Bthe%2Bstructural%2Bgenes%2Bregulating%2Bdefense%2Bphenylpropanoid%2Bmetabolism%2Bin%2BPopulus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2006.01798.x%26rft_id%253Dinfo%253Apmid%252F16945088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2006.01798.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16945088&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239988100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cole</span>,  <span class="cit-name-given-names">C. T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>,  <span class="cit-name-given-names">P. K.</span></span> <span class="cit-article-title">Pathway position constrains the evolution of an ecologically important pathway in aspens (<em>Populus tremula</em> L</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">3317</span>–<span class="cit-lpage">3330</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D27%26rft.spage%253D3317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.75" data-doi="10.1111/j.1365-313X.2010.04343.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plett</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilkins</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campbell</span>,  <span class="cit-name-given-names">M. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ralph</span>,  <span class="cit-name-given-names">S. G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Regan</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Endogenous overexpression of <em>Populus</em> MYB186 increases trichome density, improves insect pest resistance, and impacts plant growth</span>. <abbr class="cit-jnl-abbrev">The Plant Journal</abbr> <span class="cit-vol">64</span>, <span class="cit-fpage">419</span>–<span class="cit-lpage">432</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DPlett%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D64%26rft.issue%253D3%26rft.spage%253D419%26rft.epage%253D432%26rft.atitle%253DEndogenous%2Boverexpression%2Bof%2BPopulus%2BMYB186%2Bincreases%2Btrichome%2Bdensity%252C%2Bimproves%2Binsect%2Bpest%2Bresistance%252C%2Band%2Bimpacts%2Bplant%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2010.04343.x%26rft_id%253Dinfo%253Apmid%252F20807210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2010.04343.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20807210&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284257900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aitken</span>,  <span class="cit-name-given-names">S. N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>,  <span class="cit-name-given-names">M. C.</span></span> <span class="cit-article-title">Assisted gene flow to facilitate local adaptation to climate change</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">44</span>. <a href="https://doi.org/10.1146/annurev-ecolsys-110512-135747">https://doi.org/10.1146/annurev-ecolsys-110512-135747</a> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.77"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Urban</span>,  <span class="cit-name-given-names">M. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Improving the forecast for biodiversity under climate change</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>, aad<span class="cit-fpage">8466</span>–<span class="cit-lpage">1</span>–aad8466-9 (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D353%26rft.spage%253D8466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Breed</span>,  <span class="cit-name-given-names">M. F.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The potential of genomics for restoring ecosystems and biodiversity</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr>, <span class="cit-vol">1</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gillis</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Omule</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Brierley</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Monitoring Canada’s forests: the national forest inventory</span>. <abbr class="cit-jnl-abbrev">The Forestry Chronicle</abbr> <span class="cit-vol">81</span>, <span class="cit-fpage">214</span>–<span class="cit-lpage">221</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BForestry%2BChronicle%26rft.volume%253D81%26rft.spage%253D214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woudenberg</span>,  <span class="cit-name-given-names">S. W.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Forest Inventory and Analysis Database: Database description and users manual version 4.0 for Phase 2</span>. <abbr class="cit-jnl-abbrev">Gen. Tech. Rep. RMRS-GTR-245. Fort Collins, CO: US Department of Agriculture, Forest Service, Rocky Mountain Research Station</abbr>. <span class="cit-vol">336</span> p. <span class="cit-fpage">245</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGen.%2BTech.%2BRep.%2BRMRS-GTR-245.%2BFort%2BCollins%252C%2BCO%253A%2BUS%2BDepartment%2Bof%2BAgriculture%252C%2BForest%2BService%252C%2BRocky%2BMountain%2BResearch%2BStation%26rft.volume%253D336%26rft.spage%253D245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.81" data-doi="10.1111/ecog.01132"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aiello-Lammens</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boria</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Radosavljevic</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vilela</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">R. P.</span></span> <span class="cit-article-title">spThin: an R package for spatial thinning of species occurrence records for use in ecological niche models</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">38</span>, <span class="cit-fpage">541</span>–<span class="cit-lpage">545</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D38%26rft.spage%253D541%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fecog.01132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ecog.01132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.82"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burgman</span>,  <span class="cit-name-given-names">M. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fox</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-article-title">Bias in species range estimates from minimum convex polygons: implications for conservation and options for improved planning</span> in <abbr class="cit-jnl-abbrev">Animal Conservation forum</abbr> <span class="cit-vol">6</span> (<span class="cit-pub-date">2003</span>), <span class="cit-fpage">19</span>–<span class="cit-lpage">28</span>. <a href="https://doi.org/10.1017/S1367943003003044">https://doi.org/10.1017/S1367943003003044</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2BConservation%2Bforum%26rft.volume%253D6%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-web" id="cit-817411v1.83"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>. <span class="cit-chapter-title">R: A Language and Environment for Statistical Computing</span> <span class="cit-publ-name">R Foundation for Statistical Computing</span> (<span class="cit-publ-loc">Vienna, Austria</span>, <span class="cit-pub-date">2018</span>). <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">84.</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference 84. in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.84" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e19379</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.85" data-doi="10.1126/science.1128691"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tuskan</span>,  <span class="cit-name-given-names">G. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">313</span>, <span class="cit-fpage">1596</span>–<span class="cit-lpage">1604</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTuskan%26rft.auinit1%253DG.%2BA.%26rft.volume%253D313%26rft.issue%253D5793%26rft.spage%253D1596%26rft.epage%253D1604%26rft.atitle%253DThe%2Bgenome%2Bof%2Bblack%2Bcottonwood%252C%2BPopulus%2Btrichocarpa%2B%2528Torr.%2B%2526amp%253B%2BGray%2529.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1128691%26rft_id%253Dinfo%253Apmid%252F16973872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTMvNTc5My8xNTk2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.86" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.87" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The variant call format and VCFtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-web" id="cit-817411v1.88"><div class="cit-metadata"><cite><span class="cit-article-title">ARCC</span>. <span class="cit-source">Teton Computing Environment: Intel x86 64 Cluster, Advanced Research Computing Center, University of Wyoming, Laramie, WY</span>. <span class="cit-pub-date">2018</span>. <a href="https://doi.org/10.15786/M2FY47">https://doi.org/10.15786/M2FY47</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">89.</span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference 89. in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.89" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">D. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Novembre</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lange</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals</span>. <abbr class="cit-jnl-abbrev">Genome Research</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzI0LzgxNzQxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">90.</span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference 90. in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.90" data-doi="10.1534/genetics.114.164350"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raj</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J. K.</span></span> <span class="cit-article-title">fastSTRUCTURE: variational inference of population structure in large SNP data sets</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>, <span class="cit-fpage">573</span>–<span class="cit-lpage">589</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164350%26rft_id%253Dinfo%253Apmid%252F24700103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNTczIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">91.</span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference 91. in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.91"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chhatre</span>,  <span class="cit-name-given-names">V. E.</span></span> <abbr class="cit-jnl-abbrev">Distruct 2.3: A modified admixture plotting script</abbr> (<span class="cit-pub-date">2018</span>). <a href="http://distruct2.popgen.org">http://distruct2.popgen.org</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">92.</span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference 92. in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jombart</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Devillard</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Balloux</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Discriminant analysis of principal components: a new method for the analysis of genetically structured populations</span>. <abbr class="cit-jnl-abbrev">BMC Genetics</abbr> <span class="cit-vol">11</span>, <span class="cit-fpage">94</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenetics%26rft.volume%253D11%26rft.spage%253D94%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">93.</span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference 93. in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.93" data-doi="10.1111/mec.12725"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>,  <span class="cit-name-given-names">K. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>,  <span class="cit-name-given-names">M. C.</span></span> <span class="cit-article-title">Evaluation of demographic history and neutral parameterization on the performance of <em>F</em><sub>ST</sub> outlier tests</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">2178</span>–<span class="cit-lpage">2192</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D23%26rft.spage%253D2178%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12725%26rft_id%253Dinfo%253Apmid%252F24655127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24655127&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">94.</span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference 94. in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.94" data-doi="10.1534/genetics.108.092221"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foll</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: A Bayesian perspective</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>, <span class="cit-fpage">977</span>–<span class="cit-lpage">993</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092221%26rft_id%253Dinfo%253Apmid%252F18780740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzIvOTc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">95.</span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference 95. in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.95"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Storey</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bass</span>,  <span class="cit-name-given-names">A. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dabney</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">D.</span></span> <abbr class="cit-jnl-abbrev">qvalue: Q-value estimation for false discovery rate control</abbr> <span class="cit-article-title">R package version 2.14.1</span> (<span class="cit-pub-date">2019</span>). <a href="https://github.com/jdstorey/qvalue">https://github.com/jdstorey/qvalue</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">96.</span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference 96. in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.96" data-doi="10.1093/molbev/mst063"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frichot</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoville</span>,  <span class="cit-name-given-names">S. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouchard</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">François</span>,  <span class="cit-name-given-names">O.</span></span> <span class="cit-article-title">Testing for associations between loci and environmental gradients using latent factor mixed models</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1687</span>–<span class="cit-lpage">99</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst063%26rft_id%253Dinfo%253Apmid%252F23543094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23543094&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321056200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">97.</span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference 97. in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.97"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caye</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Franois</span>,  <span class="cit-name-given-names">O.</span></span> <abbr class="cit-jnl-abbrev">LFMM: Latent Factor Mixed Models</abbr> <span class="cit-article-title">R package version 0.0</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">98.</span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference 98. in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.98" data-doi="10.1534/genetics.113.152462"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Günther</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Coop</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Robust identification of local adaptation from allele frequencies</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>, <span class="cit-fpage">205</span>–<span class="cit-lpage">220</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152462%26rft_id%253Dinfo%253Apmid%252F23821598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzEvMjA1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">99.</span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference 99. in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.99" data-doi="10.1534/genetics.113.160341"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Szkiba</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kapun</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">von Haeseler</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gallach</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">SNP2GO: functional analysis of genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>, <span class="cit-fpage">285</span>–<span class="cit-lpage">289</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160341%26rft_id%253Dinfo%253Apmid%252F24561481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzEvMjg1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTAvMjQvODE3NDExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">100.</span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference 100. in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-817411v1.100" data-doi="10.1371/journal.pone.0021800"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Supek</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bošnjak</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Škunca</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Šmuc</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">REVIGO summarizes and visualizes long lists of gene ontology terms</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e21800</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSupek%26rft.auinit1%253DF.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253De21800%26rft.epage%253De21800%26rft.atitle%253DREVIGO%2Bsummarizes%2Band%2Bvisualizes%2Blong%2Blists%2Bof%2Bgene%2Bontology%2Bterms.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0021800%26rft_id%253Dinfo%253Apmid%252F21789182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0021800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21789182&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F24%2F817411.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/817411v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F817411v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/817163v1" title="Decoupling Genetics from Attainments: The Role of Social Environments" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 24, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/817411v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2019/10/24/817411.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/817411v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/817411v1" class="active" data-icon-position="" data-hide-link-title="0">Climatic niche predicts the landscape structure of locally adaptive standing genetic variation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/964641" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3kq2CQHCMAbzr99MejBqDxIvBtE8SgSsjG2ctuwgPFg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="964641" id="node964641--2" data-pisa="biorxiv;817411v1" data-pisa-master="biorxiv;817411" data-seqnum="964641" data-apath="/biorxiv/early/2019/10/24/817411.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Climatic niche predicts the landscape structure of locally adaptive standing genetic variation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Vikram E.</span> <span class="nlm-surname">Chhatre</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Karl C.</span> <span class="nlm-surname">Fetter</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew V.</span> <span class="nlm-surname">Gougherty</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Matthew C.</span> <span class="nlm-surname">Fitzpatrick</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Raju Y.</span> <span class="nlm-surname">Soolanayakanahally</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ronald S.</span> <span class="nlm-surname">Zalesny</span> <span class="nlm-suffix">Jr.</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Stephen R.</span> <span class="nlm-surname">Keller</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">817411; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/817411 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/817411v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/817411v1&amp;title=Climatic%20niche%20predicts%20the%20landscape%20structure%20of%20locally%20adaptive%20standing%20genetic%20variation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/817411v1&amp;title=Climatic%20niche%20predicts%20the%20landscape%20structure%20of%20locally%20adaptive%20standing%20genetic%20variation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/817411v1&amp;text=Climatic%20niche%20predicts%20the%20landscape%20structure%20of%20locally%20adaptive%20standing%20genetic%20variation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/817411v1&amp;title=Climatic%20niche%20predicts%20the%20landscape%20structure%20of%20locally%20adaptive%20standing%20genetic%20variation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/817411v1&amp;t=Climatic%20niche%20predicts%20the%20landscape%20structure%20of%20locally%20adaptive%20standing%20genetic%20variation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/817411v1&amp;title=Climatic%20niche%20predicts%20the%20landscape%20structure%20of%20locally%20adaptive%20standing%20genetic%20variation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/817411v1&amp;title=Climatic%20niche%20predicts%20the%20landscape%20structure%20of%20locally%20adaptive%20standing%20genetic%20variation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="964641" id="node964641--3" data-pisa="biorxiv;817411v1" data-pisa-master="biorxiv;817411" data-seqnum="964641" data-apath="/biorxiv/early/2019/10/24/817411.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Climatic niche predicts the landscape structure of locally adaptive standing genetic variation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Vikram E.</span> <span class="nlm-surname">Chhatre</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Karl C.</span> <span class="nlm-surname">Fetter</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Andrew V.</span> <span class="nlm-surname">Gougherty</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Matthew C.</span> <span class="nlm-surname">Fitzpatrick</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Raju Y.</span> <span class="nlm-surname">Soolanayakanahally</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Ronald S.</span> <span class="nlm-surname">Zalesny</span> <span class="nlm-suffix">Jr.</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Stephen R.</span> <span class="nlm-surname">Keller</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">817411; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/817411 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Climatic%20niche%20predicts%20the%20landscape%20structure%20of%20locally%20adaptive%20standing%20genetic%20variation&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Chhatre%2C+Vikram+E.&amp;rft.au=Fetter%2C+Karl+C.&amp;rft.au=Gougherty%2C+Andrew+V.&amp;rft.au=Fitzpatrick%2C+Matthew+C.&amp;rft.au=Soolanayakanahally%2C+Raju+Y.&amp;rft.au=Zalesny%2C+Ronald+S.&amp;rft.au=Keller%2C+Stephen+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/964641/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/964641/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/964641/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/964641/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/964641/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/964641/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/964641/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/964641/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/964641/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/964641/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/964641/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/964641/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/817411v1&amp;count=horizontal&amp;via=&amp;text=Climatic%20niche%20predicts%20the%20landscape%20structure%20of%20locally%20adaptive%20standing%20genetic%20variation&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/817411v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/817411v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/817411v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
