<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/08/17/642306.atom" />
<meta name="HW.pisa" content="biorxiv;642306v5" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="In the presence of population structure: From genomics to candidate genes underlying local adaptation" />
<meta name="DC.Identifier" content="10.1101/642306" />
<meta name="DC.Date" content="2019-08-17" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Understanding the genomic signatures, genes, and traits underlying local adaptation of organisms to heterogeneous environments is of central importance to the field evolutionary biology. Mixed linear mrsodels that identify allele associations to environment, while controlling for genome-wide variation at other loci, have emerged as the method of choice when studying local adaptation. Despite their importance, it is unclear whether this approach performs better than identifying environmentally-associated SNPs without accounting for population structure. To examine this, we first use the mixed linear model GEMMA, and simple Spearman correlations, to identify SNPs showing significant associations to climate with and without accounting for population structure. Subsequently, using Italy and Sweden populations, we compare evidence of allele frequency differentiation ( FST ), linkage disequilibrium (LD), fitness variation, and functional constraint, underlying these SNPs. Using a lenient cut-off for significance, we find that SNPs identified by both approaches, and SNPs uniquely identified by Spearman correlations, were enriched at sites showing genomic evidence of local adaptation and function but were limited across Quantitative Trait Loci (QTL) explaining fitness variation. SNPs uniquely identified by GEMMA, showed no direct or indirect evidence of local adaptation, and no enrichment along putative functional sites. Finally, SNPs that showed significantly high FST and LD, were enriched along fitness QTL peaks and cis-regulatory/nonsynonymous sites showing significant functional constraint. Using these SNPs, we identify genes underlying fitness QTL, and genes linking flowering time to local adaptation. These include a regulator of abscisic-acid ( FLDH ) and flowering time genes PIF3, FIO1 , and COL5 ." />
<meta name="DC.Contributor" content="Nicholas Price" />
<meta name="DC.Contributor" content="Lua Lopez" />
<meta name="DC.Contributor" content="Adrian E. Platts" />
<meta name="DC.Contributor" content="Jesse R. Lasky" />
<meta name="DC.Contributor" content="John K. McKay" />
<meta name="article:published_time" content="2019-08-17" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="In the presence of population structure: From genomics to candidate genes underlying local adaptation" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Understanding the genomic signatures, genes, and traits underlying local adaptation of organisms to heterogeneous environments is of central importance to the field evolutionary biology. Mixed linear mrsodels that identify allele associations to environment, while controlling for genome-wide variation at other loci, have emerged as the method of choice when studying local adaptation. Despite their importance, it is unclear whether this approach performs better than identifying environmentally-associated SNPs without accounting for population structure. To examine this, we first use the mixed linear model GEMMA, and simple Spearman correlations, to identify SNPs showing significant associations to climate with and without accounting for population structure. Subsequently, using Italy and Sweden populations, we compare evidence of allele frequency differentiation (&lt;i&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/i&gt;), linkage disequilibrium (LD), fitness variation, and functional constraint, underlying these SNPs. Using a lenient cut-off for significance, we find that SNPs identified by both approaches, and SNPs uniquely identified by Spearman correlations, were enriched at sites showing genomic evidence of local adaptation and function but were limited across Quantitative Trait Loci (QTL) explaining fitness variation. SNPs uniquely identified by GEMMA, showed no direct or indirect evidence of local adaptation, and no enrichment along putative functional sites. Finally, SNPs that showed significantly high &lt;i&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/i&gt; and LD, were enriched along fitness QTL peaks and cis-regulatory/nonsynonymous sites showing significant functional constraint. Using these SNPs, we identify genes underlying fitness QTL, and genes linking flowering time to local adaptation. These include a regulator of abscisic-acid (&lt;i&gt;FLDH&lt;/i&gt;) and flowering time genes &lt;i&gt;PIF3, FIO1&lt;/i&gt;, and &lt;i&gt;COL5&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;642306v5" />
<meta name="citation_id" content="642306v5" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/642306v5" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/642306v5.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/642306v5.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306.full.pdf" />
<meta name="citation_doi" content="10.1101/642306" />
<meta name="citation_num_pages" content="36" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="642306" />
<meta name="citation_author" content="Nicholas Price" />
<meta name="citation_author_institution" content="Department of Bioagricultural Sciences &amp; Pest Management, Colorado State University" />
<meta name="citation_author_email" content="price4890@gmail.com" />
<meta name="citation_author" content="Lua Lopez" />
<meta name="citation_author_institution" content="Department of Biology, Binghamton University (State University of New York)" />
<meta name="citation_author" content="Adrian E. Platts" />
<meta name="citation_author_institution" content="Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory" />
<meta name="citation_author_institution" content="Center for Genomics and Systems Biology, Department of Biology, New York University" />
<meta name="citation_author" content="Jesse R. Lasky" />
<meta name="citation_author_institution" content="Department of Biology, Pennsylvania State University" />
<meta name="citation_author" content="John K. McKay" />
<meta name="citation_author_institution" content="Department of Bioagricultural Sciences &amp; Pest Management, Colorado State University" />
<meta name="citation_reference" content="1001 Genomes Consortium. 2016. 1,135 Genomes Reveal the Global Pattern of Polymorphism in Arabidopsis thaliana. Cell 166:481–491." />
<meta name="citation_reference" content="Ågren J, Oakley CG, Lundemo S, Schemske DW. 2017. Adaptive divergence in flowering time among natural populations of Arabidopsis thaliana: Estimates of selection and QTL mapping. Evolution 71:550–564." />
<meta name="citation_reference" content="Ågren J, Oakley CG, McKay JK, Lovell JT, Schemske DW. 2013. Genetic mapping of adaptation reveals fitness tradeoffs in Arabidopsis thaliana. Proc Natl Acad Sci U S A 110:21077–21082." />
<meta name="citation_reference" content="Ågren J, Schemske DW. 2012. Reciprocal transplants demonstrate strong adaptive differentiation of the model organism Arabidopsis thaliana in its native range. New Phytologist 194:1112–1122." />
<meta name="citation_reference" content="Anderson JT, Lee CR, Rushworth CA, Colautti RI, Mitchell-Olds T. 2013. Genetic trade-offs and conditional neutrality contribute to local adaptation. Mol Ecol 22:699–708." />
<meta name="citation_reference" content="Anderson JT, Willis JH, Mitchell-Olds T. 2011. Evolutionary genetics of plant adaptation. Trends Genet 27:258–266." />
<meta name="citation_reference" content="Arguello JR, Cardoso-Moreira M, Grenier JK, Gottipati S, Clark AG, Benton R. 2016. Extensive local adaptation within the chemosensory system following Drosophila melanogaster’s global expansion. Nat Commun 7:ncomms11855." />
<meta name="citation_reference" content="Barton N, Bengtsson BO. 1986. The barrier to genetic exchange between hybridising populations. Heredity (Edinb) 57 (Pt 3):357–376." />
<meta name="citation_reference" content="Beaumont MA, Balding DJ. 2004. Identifying adaptive genetic divergence among populations from genome scans. Mol Ecol 13:969–980." />
<meta name="citation_reference" content="Berardini TZ, Reiser L, Li D, Mezheritsky Y, Muller R, Strait E, Huala E. 2015. The Arabidopsis information resource: Making and mining the “gold standard” annotated reference plant genome. Genesis 53:474–485." />
<meta name="citation_reference" content="Bergelson J, Roux F. 2010. Towards identifying genes underlying ecologically relevant traits in Arabidopsis thaliana. Nat Rev Genet 11:867–879." />
<meta name="citation_reference" content="Bhandari J, Fitzpatrick AH, Crowell DN. 2010. Identification of a novel abscisic acid-regulated farnesol dehydrogenase from Arabidopsis. Plant Physiol 154:1116–1127." />
<meta name="citation_reference" content="Caicedo AL, Richards C, Ehrenreich IM, Purugganan MD. 2009. Complex rearrangements lead to novel chimeric gene fusion polymorphisms at the Arabidopsis thaliana MAF2-5 flowering time gene cluster. Mol Biol Evol 26:699–711." />
<meta name="citation_reference" content="Charlesworth B, Morgan MT, Charlesworth D. 1993. The effect of deleterious mutations on neutral molecular variation. Genetics 134:1289–1303." />
<meta name="citation_reference" content="Colosimo PF, Peichel CL, Nereng K, Blackman BK, Shapiro MD, Schluter D, Kingsley DM. 2004. The genetic architecture of parallel armor plate reduction in threespine sticklebacks. PLoS Biol 2:E109." />
<meta name="citation_reference" content="Conover DO, Duffy TA, Hice LA. 2009. The covariance between genetic and environmental influences across ecological gradients: reassessing the evolutionary significance of countergradient and cogradient variation. Ann N Y Acad Sci 1168:100–129." />
<meta name="citation_reference" content="Coop G, Pickrell JK, Novembre J, Kudaravalli S, Li J, Absher D, Myers RM, Cavalli-Sforza LL, Feldman MW, Pritchard JK. 2009. The role of geography in human adaptation. PLoS Genet 5:e1000500." />
<meta name="citation_reference" content="de Villemereuil P, Gaggiotti OE. 2015. A new FST-based method to uncover local adaptation using environmental variables. Methods in Ecology and Evolution 6:1248–1258." />
<meta name="citation_reference" content="DeGiorgio M, Huber CD, Hubisz MJ, Hellmann I, Nielsen R. 2016. SweepFinder2: increased sensitivity, robustness and flexibility. Bioinformatics 32:1895–1897." />
<meta name="citation_reference" content="Exposito-Alonso M, Vasseur F, Ding W, Wang G, Burbano HA, Weigel D. 2018. Genomic basis and evolutionary potential for extreme drought adaptation in Arabidopsis thaliana. Nat Ecol Evol 2:352–358." />
<meta name="citation_reference" content="Fischer MC, Rellstab C, Tedder A, Zoller S, Gugerli F, Shimizu KK, Holderegger R, Widmer A. 2013. Population genomic footprints of selection and associations with climate in natural populations of Arabidopsis halleri from the Alps. Mol Ecol 22:5594–5607." />
<meta name="citation_reference" content="Foll M, Gaggiotti O. 2008. A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective. Genetics 180:977–993." />
<meta name="citation_reference" content="Forester BR, Lasky JR, Wagner HH, Urban DL. 2018. Comparing methods for detecting multilocus adaptation with multivariate genotype–environment associations. Molecular Ecology 27:2215–2233." />
<meta name="citation_reference" content="Frachon L, Bartoli C, Carrere S, Bouchez O, Chaubet A, Gautier M, Roby D, Roux F. 2018. A Genomic Map of Climate Adaptation in Arabidopsis thaliana at a Micro-Geographic Scale. Front Plant Sci 9:967." />
<meta name="citation_reference" content="Gehan MA, Park S, Gilmour SJ, An C, Lee CM, Thomashow MF. 2015. Natural variation in the C-repeat binding factor cold response pathway correlates with local adaptation of Arabidopsis ecotypes. Plant J 84:682–693." />
<meta name="citation_reference" content="Gienapp P, Fior S, Guillaume F, Lasky JR, Sork VL, Csilléry K. 2017. Genomic quantitative genetics to study evolution in the wild. Trends in Ecology &amp; Evolution." />
<meta name="citation_reference" content="Gillespie JH. 2000. Genetic drift in an infinite population. The pseudohitchhiking model. Genetics 155:909–919." />
<meta name="citation_reference" content="Goodstein DM, Shu S, Howson R, Neupane R, Hayes RD, Fazo J, Mitros T, Dirks W, Hellsten U, Putnam N, et al. 2012. Phytozome: a comparative platform for green plant genomics. Nucleic Acids Res 40:D1178–1186." />
<meta name="citation_reference" content="Graur D. 2016. Molecular and Genome Evolution. Sunderland, MA: Sinauer." />
<meta name="citation_reference" content="Hancock AM, Brachi B, Faure N, Horton MW, Jarymowycz LB, Sperone FG, Toomajian C, Roux F, Bergelson J. 2011. Adaptation to climate across the Arabidopsis thaliana genome. Science 334:83–86." />
<meta name="citation_reference" content="Hassidim M, Harir Y, Yakir E, Kron I, Green RM. 2009. Over-expression of CONSTANS-LIKE 5 can induce flowering in short-day grown Arabidopsis. Planta 230:481–491." />
<meta name="citation_reference" content="Haudry A, Platts AE, Vello E, Hoen DR, Leclercq M, Williamson RJ, Forczek E, Joly-Lopez Z, Steffen JG, Hazzouri KM, et al. 2013. An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions. Nat Genet 45:891–898." />
<meta name="citation_reference" content="Hendry AP, Taylor EB, McPhail JD. 2002. Adaptive divergence and the balance between selection and gene flow: lake and stream stickleback in the Misty system. Evolution 56:1199–1216." />
<meta name="citation_reference" content="Henson J, Tischler G, Ning Z. 2012. Next-generation sequencing and large genome assemblies. Pharmacogenomics 13:901–915." />
<meta name="citation_reference" content="Hereford J. 2009. A quantitative survey of local adaptation and fitness trade-offs. Am Nat 173:579–588." />
<meta name="citation_reference" content="Hoban S, Kelley JL, Lotterhos KE, Antolin MF, Bradburd G, Lowry DB, Poss ML, Reed LK, Storfer A, Whitlock MC. 2016. Finding the Genomic Basis of Local Adaptation: Pitfalls, Practical Solutions, and Future Directions. Am Nat 188:379–397." />
<meta name="citation_reference" content="Huber CD, DeGiorgio M, Hellmann I, Nielsen R. 2016. Detecting recent selective sweeps while controlling for mutation rate and background selection. Mol Ecol 25:142–156." />
<meta name="citation_reference" content="Huber CD, Nordborg M, Hermisson J, Hellmann I. 2014. Keeping it local: evidence for positive selection in Swedish Arabidopsis thaliana. Mol Biol Evol 31:3026–3039." />
<meta name="citation_reference" content="Hupalo D, Kern AD. 2013. Conservation and functional element discovery in 20 angiosperm plant genomes. Mol Biol Evol 30:1729–1744." />
<meta name="citation_reference" content="Jacobs GS, Sluckin TJ, Kivisild T. 2016. Refining the Use of Linkage Disequilibrium as a Robust Signature of Selective Sweeps. Genetics 203:1807–1825." />
<meta name="citation_reference" content="Jeong C, Di Rienzo A. 2014. Adaptations to local environments in modern human populations. Curr Opin Genet Dev 29:1–8." />
<meta name="citation_reference" content="Jiang B, Shi Y, Zhang X, Xin X, Qi L, Guo H, Li J, Yang S. 2017. PIF3 is a negative regulator of the CBF pathway and freezing tolerance in Arabidopsis. Proc Natl Acad Sci U S A 114:E6695–E6702." />
<meta name="citation_reference" content="Jones FC, Grabherr MG, Chan YF, Russell P, Mauceli E, Johnson J, Swofford R, Pirun M, Zody MC, White S, et al. 2012. The genomic basis of adaptive evolution in threespine sticklebacks. Nature 484:55–61." />
<meta name="citation_reference" content="Joosen RV, Ligterink W, Hilhorst HW, Keurentjes JJ. 2009. Advances in genetical genomics of plants. Current genomics 10:540–549." />
<meta name="citation_reference" content="Kalladan R, Lasky JR, Chang TZ, Sharma S, Juenger TE, Verslues PE. 2017. Natural variation identifies genes affecting drought-induced abscisic acid accumulation in Arabidopsis thaliana. Proc Natl Acad Sci U S A 114:11536–11541." />
<meta name="citation_reference" content="Kang HM, Sul JH, Service SK, Zaitlen NA, Kong SY, Freimer NB, Sabatti C, Eskin E. 2010. Variance component model to account for sample structure in genome-wide association studies. Nat Genet 42:348–354." />
<meta name="citation_reference" content="Kang HM, Zaitlen NA, Wade CM, Kirby A, Heckerman D, Daly MJ, Eskin E. 2008. Efficient control of population structure in model organism association mapping. Genetics 178:1709–1723." />
<meta name="citation_reference" content="Katoh K, Toh H. 2008. Recent developments in the MAFFT multiple sequence alignment program. Brief Bioinform 9:286–298." />
<meta name="citation_reference" content="Kaufmann J, Lenz TL, Kalbe M, Milinski M, Eizaguirre C. 2017. A field reciprocal transplant experiment reveals asymmetric costs of migration between lake and river ecotypes of three-spined sticklebacks (Gasterosteus aculeatus). J Evol Biol 30:938–950." />
<meta name="citation_reference" content="Kawecki TJ, Ebert D. 2004. Conceptual issues in local adaptation. Ecology Letters 7:1225–1241." />
<meta name="citation_reference" content="Keller SR, Levsen N, Olson MS, Tiffin P. 2012. Local adaptation in the flowering-time gene network of balsam poplar, Populus balsamifera L. Mol Biol Evol 29:3143–3152." />
<meta name="citation_reference" content="Kryvokhyzha D, Holm K, Chen J, Cornille A, Glémin S, Wright SI, Lagercrantz U, Lascoux M. 2016. The influence of population structure on gene expression and flowering time variation in the ubiquitous weed Capsella bursa-pastoris (Brassicaceae). Molecular Ecology 25:1106–1121." />
<meta name="citation_reference" content="Lachance J, Tishkoff SA. 2013. Population Genomics of Human Adaptation. Annu Rev Ecol Evol Syst 44:123–143." />
<meta name="citation_reference" content="Lasky JR, Des Marais DL, Lowry DB, Povolotskaya I, McKay JK, Richards JH, Keitt TH, Juenger TE. 2014. Natural variation in abiotic stress responsive gene expression and local adaptation to climate in Arabidopsis thaliana. Mol Biol Evol 31:2283–2296." />
<meta name="citation_reference" content="Lasky JR, Des Marais DL, McKay JK, Richards JH, Juenger TE, Keitt TH. 2012. Characterizing genomic variation of Arabidopsis thaliana: the roles of geography and climate. Molecular Ecology 21:5512–5529." />
<meta name="citation_reference" content="Lasky JR, Forester BR, Reimherr M. 2018. Coherent synthesis of genomic associations with phenotypes and home environments. Mol Ecol Resour." />
<meta name="citation_reference" content="Leimu R, Fischer M. 2008. A meta-analysis of local adaptation in plants. PLoS One 3:e4010." />
<meta name="citation_reference" content="Li XC, Fay JC. 2017. Cis-Regulatory Divergence in Gene Expression between Two Thermally Divergent Yeast Species. Genome Biol Evol 9:1120–1129." />
<meta name="citation_reference" content="Long Q, Rabanal FA, Meng D, Huber CD, Farlow A, Platzer A, Zhang Q, Vilhjalmsson BJ, Korte A, Nizhynska V, et al. 2013. Massive genomic variation and strong selection in Arabidopsis thaliana lines from Sweden. Nat Genet 45:884–890." />
<meta name="citation_reference" content="Lotterhos KE, Whitlock MC. 2014. Evaluation of demographic history and neutral parameterization on the performance of FST outlier tests. Molecular Ecology 23:2178–2192." />
<meta name="citation_reference" content="Maddamsetti R, Hatcher PJ, Green AG, Williams BL, Marks DS, Lenski RE. 2017. Core Genes Evolve Rapidly in the Long-term Evolution Experiment with Escherichia coli. Genome Biol Evol." />
<meta name="citation_reference" content="McKay JK, Latta RG. 2002. Adaptive population divergence: markers, QTL and traits. Trends in Ecology &amp; Evolution 17:285–291." />
<meta name="citation_reference" content="Miller W, Rosenbloom K, Hardison RC, Hou M, Taylor J, Raney B, Burhans R, King DC, Baertsch R, Blankenberg D, et al. 2007. 28-way vertebrate alignment and conservation track in the UCSC Genome Browser. Genome Res 17:1797–1808." />
<meta name="citation_reference" content="Monroe JG, McGovern C, Lasky JR, Grogan K, Beck J, McKay JK. 2016. Adaptation to warmer climates by parallel functional evolution of CBF genes in Arabidopsis thaliana. Mol Ecol 25:3632–3644." />
<meta name="citation_reference" content="Monroe JG, Powell T, Price N, Mullen JL, Howard A, Evans K, Lovell JT, McKay JK. 2018. Drought adaptation in Arabidopsis thaliana by extensive genetic loss-of-function. Elife 7:e41038." />
<meta name="citation_reference" content="Monte E, Tepperman JM, Al-Sady B, Kaczorowski KA, Alonso JM, Ecker JR, Li X, Zhang Y, Quail PH. 2004. The phytochrome-interacting transcription factor, PIF3, acts early, selectively, and positively in light-induced chloroplast development. Proc Natl Acad Sci U S A 101:16091–16098." />
<meta name="citation_reference" content="Nachman MW, Hoekstra HE, D’Agostino SL. 2003. The genetic basis of adaptive melanism in pocket mice. Proc Natl Acad Sci U S A 100:5268–5273." />
<meta name="citation_reference" content="Oakley CG, Agren J, Atchison RA, Schemske DW. 2014. QTL mapping of freezing tolerance: links to fitness and adaptive trade-offs. Mol Ecol 23:4304–4315." />
<meta name="citation_reference" content="Oakley CG, Savage L, Lotz S, Larson GR, Thomashow MF, Kramer DM, Schemske DW. 2018. Genetic basis of photosynthetic responses to cold in two locally adapted populations of Arabidopsis thaliana. J Exp Bot 69:699–709." />
<meta name="citation_reference" content="Oda A, Fujiwara S, Kamada H, Coupland G, Mizoguchi T. 2004. Antisense suppression of the Arabidopsis PIF3 gene does not affect circadian rhythms but causes early flowering and increases FT expression. FEBS Lett 557:259–264." />
<meta name="citation_reference" content="Pass DA, Sornay E, Marchbank A, Crawford MR, Paszkiewicz K, Kent NA, Murray JAH. 2017. Genome-wide chromatin mapping with size resolution reveals a dynamic sub-nucleosomal landscape in Arabidopsis. PLoS Genet 13:e1006988." />
<meta name="citation_reference" content="Phifer-Rixey M, Bi K, Ferris KG, Sheehan MJ, Lin D, Mack KL, Keeble SM, Suzuki TA, Good JM, Nachman MW. 2018. The genomic basis of environmental adaptation in house mice. PLoS Genet 14:e1007672." />
<meta name="citation_reference" content="Pisupati R, Reichardt I, Seren U, Korte P, Nizhynska V, Kerdaffrec E, Uzunova K, Rabanal FA, Filiault DL, Nordborg M. 2017. Verification of Arabidopsis stock collections using SNPmatch, a tool for genotyping high-plexed samples. Sci Data 4:170184." />
<meta name="citation_reference" content="Platt A, Vilhjalmsson BJ, Nordborg M. 2010. Conditions under which genome-wide association studies will be positively misleading. Genetics 186:1045–1052." />
<meta name="citation_reference" content="Pluess AR, Frank A, Heiri C, Lalagüe H, Vendramin GG, Oddou-Muratorio S. 2016. Genome– environment association study suggests local adaptation to climate at the regional scale in Fagus sylvatica. New Phytologist 210:589–601." />
<meta name="citation_reference" content="Price AL, Zaitlen NA, Reich D, Patterson N. 2010. New approaches to population stratification in genome-wide association studies. Nat Rev Genet 11:459–463." />
<meta name="citation_reference" content="Price N, Moyers BT, Lopez L, Lasky JR, Monroe JG, Mullen JL, Oakley CG, Lin J, Agren J, Schrider DR, et al. 2018. Combining population genomics and fitness QTLs to identify the genetics of local adaptation in Arabidopsis thaliana. Proc Natl Acad Sci U S A 115:5028–5033." />
<meta name="citation_reference" content="Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, et al. 2007. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 81:559–575." />
<meta name="citation_reference" content="Rellstab C, Fischer MC, Zoller S, Graf R, Tedder A, Shimizu KK, Widmer A, Holderegger R, Gugerli F. 2017. Local adaptation (mostly) remains local: reassessing environmental associations of climate-related candidate SNPs in Arabidopsis halleri. Heredity (Edinb) 118:193–201." />
<meta name="citation_reference" content="Ristova D, Metesch K, Busch W. 2017. Natural Genetic Variation Shapes Root System Responses to Phytohormones in Arabidopsis. bioRxiv: 155184." />
<meta name="citation_reference" content="Sackton TB, Grayson P, Cloutier A, Hu Z, Liu JS, Wheeler NE, Gardner PP, Clarke JA, Baker AJ, Clamp M, et al. 2019. Convergent regulatory evolution and loss of flight in paleognathous birds. Science 364:74–78." />
<meta name="citation_reference" content="Sasaki E, Zhang P, Atwell S, Meng D, Nordborg M. 2015. “Missing” G × E Variation Controls Flowering Time in Arabidopsis thaliana. PLoS Genet 11:e1005597." />
<meta name="citation_reference" content="Savolainen O, Lascoux M, Merila J. 2013. Ecological genomics of local adaptation. Nat Rev Genet 14:807–820." />
<meta name="citation_reference" content="Savolainen O, Pyhäjärvi T, Knürr T. 2007. Gene Flow and Local Adaptation in Trees. Annual Review of Ecology, Evolution, and Systematics 38:595–619." />
<meta name="citation_reference" content="Siepel A, Arbiza L. 2014. Cis-regulatory elements and human evolution. Curr Opin Genet Dev 29:81–89." />
<meta name="citation_reference" content="Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier LW, Richards S, et al. 2005. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res 15:1034–1050." />
<meta name="citation_reference" content="Sork VL. 2017. Genomic Studies of Local Adaptation in Natural Plant Populations. J Hered 109:3–15." />
<meta name="citation_reference" content="Soy J, Leivar P, Gonzalez-Schain N, Sentandreu M, Prat S, Quail PH, Monte E. 2012. Phytochrome-imposed oscillations in PIF3 protein abundance regulate hypocotyl growth under diurnal light/dark conditions in Arabidopsis. Plant J 71:390–401." />
<meta name="citation_reference" content="Spearman C. 1987. The Proof and Measurement of Association between Two Things. The American Journal of Psychology 100:441–471." />
<meta name="citation_reference" content="Svetec N, Cridland JM, Zhao L, Begun DJ. 2016. The Adaptive Significance of Natural Genetic Variation in the DNA Damage Response of Drosophila melanogaster. PLoS Genet 12:e1005869." />
<meta name="citation_reference" content="Tamura K, Stecher G, Peterson D, Filipski A, Kumar S. 2013. MEGA6: Molecular Evolutionary Genetics Analysis version 6.0. Mol Biol Evol 30:2725–2729." />
<meta name="citation_reference" content="Team RDC. 2009. R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing. Vienna, Austria: R Foundation for Statistical Computing." />
<meta name="citation_reference" content="Tiffin P, Ross-Ibarra J. 2014. Advances and limits of using population genetics to understand local adaptation. Trends Ecol Evol 29:673–680." />
<meta name="citation_reference" content="Tuteja N. 2007. Abscisic Acid and abiotic stress signaling. Plant Signal Behav 2:135–138." />
<meta name="citation_reference" content="Via S. 1991. The Genetic Structure of Host Plant Adaptation in a Spatial Patchwork: Demographic Variability among Reciprocally Transplanted Pea Aphid Clones. Evolution 45:827–852." />
<meta name="citation_reference" content="Wadgymar SM, Lowry DB, Gould BA, Byron CN, Mactavish RM, Anderson JT. 2017. Identifying targets and agents of selection: innovative methods to evaluate the processes that contribute to local adaptation. Methods in Ecology and Evolution 8:738–749." />
<meta name="citation_reference" content="Wang L, Jia P, Wolfinger RD, Chen X, Grayson BL, Aune TM, Zhao Z. 2011. An efficient hierarchical generalized linear mixed model for pathway analysis of genome-wide association studies. Bioinformatics 27:686–692." />
<meta name="citation_reference" content="Yang J, Guo B, Shikano T, Liu X, Merila J. 2016. Quantitative trait locus analysis of body shape divergence in nine-spined sticklebacks based on high-density SNP-panel. Sci Rep 6:26632." />
<meta name="citation_reference" content="Yeaman S, Hodgins KA, Lotterhos KE, Suren H, Nadeau S, Degner JC, Nurkowski KA, Smets P, Wang T, Gray LK, et al. 2016. Convergent local adaptation to climate in distantly related conifers. Science 353:1431–1433." />
<meta name="citation_reference" content="Yeaman S, Whitlock MC. 2011. The genetic architecture of adaptation under migration-selection balance. Evolution 65:1897–1911." />
<meta name="citation_reference" content="Yu J, Pressoir G, Briggs WH, Vroh Bi I, Yamasaki M, Doebley JF, McMullen MD, Gaut BS, Nielsen DM, Holland JB, et al. 2006. A unified mixed-model method for association mapping that accounts for multiple levels of relatedness. Nat Genet 38:203–208." />
<meta name="citation_reference" content="Zhou X, Stephens M. 2012. Genome-wide efficient mixed-model analysis for association studies. Nat Genet 44:821–824." />
<meta name="citation_reference" content="Zou C, Sun K, Mackaluso JD, Seddon AE, Jin R, Thomashow MF, Shiu SH. 2011. Cis-regulatory code of stress-responsive transcription in Arabidopsis thaliana. Proc Natl Acad Sci U S A 108:14992–14997." />
<meta name="twitter:title" content="In the presence of population structure: From genomics to candidate genes underlying local adaptation" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Understanding the genomic signatures, genes, and traits underlying local adaptation of organisms to heterogeneous environments is of central importance to the field evolutionary biology. Mixed linear mrsodels that identify allele associations to environment, while controlling for genome-wide variation at other loci, have emerged as the method of choice when studying local adaptation. Despite their importance, it is unclear whether this approach performs better than identifying environmentally-associated SNPs without accounting for population structure. To examine this, we first use the mixed linear model GEMMA, and simple Spearman correlations, to identify SNPs showing significant associations to climate with and without accounting for population structure. Subsequently, using Italy and Sweden populations, we compare evidence of allele frequency differentiation ( FST ), linkage disequilibrium (LD), fitness variation, and functional constraint, underlying these SNPs. Using a lenient cut-off for significance, we find that SNPs identified by both approaches, and SNPs uniquely identified by Spearman correlations, were enriched at sites showing genomic evidence of local adaptation and function but were limited across Quantitative Trait Loci (QTL) explaining fitness variation. SNPs uniquely identified by GEMMA, showed no direct or indirect evidence of local adaptation, and no enrichment along putative functional sites. Finally, SNPs that showed significantly high FST and LD, were enriched along fitness QTL peaks and cis-regulatory/nonsynonymous sites showing significant functional constraint. Using these SNPs, we identify genes underlying fitness QTL, and genes linking flowering time to local adaptation. These include a regulator of abscisic-acid ( FLDH ) and flowering time genes PIF3, FIO1 , and COL5 ." />
<meta name="og-title" property="og:title" content="In the presence of population structure: From genomics to candidate genes underlying local adaptation" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/642306v5" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Understanding the genomic signatures, genes, and traits underlying local adaptation of organisms to heterogeneous environments is of central importance to the field evolutionary biology. Mixed linear mrsodels that identify allele associations to environment, while controlling for genome-wide variation at other loci, have emerged as the method of choice when studying local adaptation. Despite their importance, it is unclear whether this approach performs better than identifying environmentally-associated SNPs without accounting for population structure. To examine this, we first use the mixed linear model GEMMA, and simple Spearman correlations, to identify SNPs showing significant associations to climate with and without accounting for population structure. Subsequently, using Italy and Sweden populations, we compare evidence of allele frequency differentiation ( FST ), linkage disequilibrium (LD), fitness variation, and functional constraint, underlying these SNPs. Using a lenient cut-off for significance, we find that SNPs identified by both approaches, and SNPs uniquely identified by Spearman correlations, were enriched at sites showing genomic evidence of local adaptation and function but were limited across Quantitative Trait Loci (QTL) explaining fitness variation. SNPs uniquely identified by GEMMA, showed no direct or indirect evidence of local adaptation, and no enrichment along putative functional sites. Finally, SNPs that showed significantly high FST and LD, were enriched along fitness QTL peaks and cis-regulatory/nonsynonymous sites showing significant functional constraint. Using these SNPs, we identify genes underlying fitness QTL, and genes linking flowering time to local adaptation. These include a regulator of abscisic-acid ( FLDH ) and flowering time genes PIF3, FIO1 , and COL5 ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-08-17" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/642306v5.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/642306v5.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/642306v5.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/642306v5" />
<link rel="shortlink" href="https://www.biorxiv.org/node/872420" />
    <title>In the presence of population structure: From genomics to candidate genes underlying local adaptation | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"6MOkimZDTfaguBl25uyLYPp7NK3R9DoVNgcCYy3TAtI","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;642306v5"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;642306v5"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;642306v5"}],"nid":"872420","apath":"\/biorxiv\/early\/2019\/08\/17\/642306.atom","pisa":"biorxiv;642306v5","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/642306v5","title":"In the presence of population structure: From genomics to candidate genes underlying local adaptation","identifier":"node\/872420"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227MCJkQVIX0ITD6cAaoAgI2bD5CHFbxAOKzIJTdmgEC1Q9NpNXYViocBrt6lkqFDE7zLrk6\\\\\\\/wdHzK+tHJFj\\\\\\\/bRUgqGLldw\\\\\\\/KBqXjCbARO20ainGwj594o4NTvrr+KAy+9VZLLoAkPaX80Zrx+88uGovlyva6suLFdDxkwf8PKSqDDC5QN1pHD5DQwlcRFXonYtrJVBf\\\\\\\/P0AknuOWJQDw0QwIqOGthGfuzhLDb5wBuc4mHcvAYId\\\\\\\/LplVV7WK\\\\\\\/JFquMsHJqv5is9iihNd9fBH5PNdyBKBVDJW\\\\\\\/\\\\\\\/yR5J+QxlQwcYREM8xyPz7huQ8oWQ\\\\\\\/\\\\\\\/eYFGoM8aYOPpAHCU7JQZHCBwv2v\\\\\\\/TAkgRaPwiuU0mei7KnHcdAAQIoZ8pRL2GGSXY7PHwFBSl4M282FG9N5\\\\\\\/dCnnWyATmF0MMdEahuB\\\\\\\/T7xYl+Hrf98JPjQTLQfQHSxl5qG3nvLfwQtZ0O3lwr8BpDIw50GcRcQnv+ZdhFkvCUBXkLGvakzDxjibbOZ6TILWY4k7tO1e6Icr8rDFnE8K6SN1et0RaSwFrTtAbQucvReT0SBl74IeyituHyD2HtY4pK\\\\\\\/3OZaPcmF\\\\\\\/ET44dHnBE6LfyFo7MWZ6dPrS2MOqGA2TuytTD33FJel4hMK04EAGHj9v\\\\\\\/OCnlHF\\\\\\\/6\\\\\\\/ORD27dBXAxyISA69gYiA7z7Nt\\\\\\\/sEzBf16M7C4\\\\\\\/V5OyrlXvNAAhbktML49cfHmnVZG7w7B01Kq5TjH+DeNuyYTQPKUmJgBKk21d0trUpfT9iVR4VTDyJrac0tzpFOyw1agA1RrZzujqnWjTEwhysUW4kYnIubOw8sCk\\\\\\\/+PwrRh+frX2Kt1QS8SrWGe3Er1y4CZOMZhi2NIToY53BJTp3fMFUwTqa0Wt806aibo2PJSL2f9icpaqAgD+qDfPdwOee6Ksn2VgkAI4GJJ7VMolgOyIGO8F2FjUvOaZqdyfXCu\\\\\\\/h7cQBO4mkp7mPAAY0wzxlbkuAU1WZmLD0QyfS9dxv3m3yope+PQnuLjNxNuYYzm9EZNWT4W9VvIFlRlCfUp8WHQgvOhjkHwmxuFRZBuEGaMO6EHTBcnkOscGD7dDhoyUjyqFIi3yNc3Q+BzW9D3THbz\\\\\\\/ExfApEXjWfQk5VtNh7cRePLarNuY85OwVA3pIJ1K3e8einB3DABPJqxFCcu59hgC0mBKWg5vB\\\\\\\/J6TbMqWurYtR63\\\\\\\/nGWA7pwuBov+j8rHpig43o+cMHg3kLXzAGYmYBS0cVZunK6\\\\\\\/LkwWHeqgtmcYoZPKDjDyw2zyPxCj7NOVQj5ZWCvMGf1HAfIvn+AJ6gLwwq470KlYx7KqOi2svjX5KxPpC6Q=\\\u0022,\\\u0022iv\\\u0022:\\\u0022BywPQijFCZLV7ScNt\\\\\\\/t\\\\\\\/fQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226535b95e308bdfd80b1a826a748d188328a53ad9373db228404af7dad2bb05d7\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/642306v5.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-872420 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/642306v5.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RIOBRzF1K3IRh3_uzdThrvr94yaBEGY0a9NmpuWw30s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="872420" id="node872420" data-pisa="biorxiv;642306v5" data-pisa-master="biorxiv;642306" data-apath="/biorxiv/early/2019/08/17/642306.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">In the presence of population structure: From genomics to candidate genes underlying local adaptation</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Nicholas</span> <span class="nlm-surname">Price</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Lua</span> <span class="nlm-surname">Lopez</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Adrian E.</span> <span class="nlm-surname">Platts</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jesse R.</span> <span class="nlm-surname">Lasky</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">John K.</span> <span class="nlm-surname">McKay</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/642306 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node872420" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicholas Price </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Bioagricultural Sciences &#38; Pest Management, Colorado State University</span>, Fort Collins, CO 80523, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BPrice%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Price%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BPrice%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:price4890@gmail.com" class="" data-icon-position="" data-hide-link-title="0">price4890@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lua Lopez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, Binghamton University (State University of New York)</span>, Binghamton, NY 13902, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lua%2BLopez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALua%2BLopez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Adrian E. Platts </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory</span>, Cold Spring Harbor, NY, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>Center for Genomics and Systems Biology, Department of Biology, New York University</span>, New York 10003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrian%2BE.%2BPlatts%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Platts%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdrian%2BE.%2BPlatts%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jesse R. Lasky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, Pennsylvania State University</span>, University Park, PA 16802, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BR.%2BLasky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lasky%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJesse%2BR.%2BLasky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John K. McKay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Bioagricultural Sciences &#38; Pest Management, Colorado State University</span>, Fort Collins, CO 80523, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BK.%2BMcKay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McKay%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BK.%2BMcKay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/642306v5" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:872420" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:872420/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/642306v5.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:872420" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:872420/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/642306v5.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:872420" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:872420/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/642306v5.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:872420" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:872420/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/642306v5.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:872420" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:872420/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/642306v5.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:872420" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:872420/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Understanding the genomic signatures, genes, and traits underlying local adaptation of organisms to heterogeneous environments is of central importance to the field evolutionary biology. Mixed linear mrsodels that identify allele associations to environment, while controlling for genome-wide variation at other loci, have emerged as the method of choice when studying local adaptation. Despite their importance, it is unclear whether this approach performs better than identifying environmentally-associated SNPs without accounting for population structure. To examine this, we first use the mixed linear model GEMMA, and simple Spearman correlations, to identify SNPs showing significant associations to climate with and without accounting for population structure. Subsequently, using Italy and Sweden populations, we compare evidence of allele frequency differentiation (<em>F<sub>ST</sub></em>), linkage disequilibrium (LD), fitness variation, and functional constraint, underlying these SNPs. Using a lenient cut-off for significance, we find that SNPs identified by both approaches, and SNPs uniquely identified by Spearman correlations, were enriched at sites showing genomic evidence of local adaptation and function but were limited across Quantitative Trait Loci (QTL) explaining fitness variation. SNPs uniquely identified by GEMMA, showed no direct or indirect evidence of local adaptation, and no enrichment along putative functional sites. Finally, SNPs that showed significantly high <em>F<sub>ST</sub></em> and LD, were enriched along fitness QTL peaks and cis-regulatory/nonsynonymous sites showing significant functional constraint. Using these SNPs, we identify genes underlying fitness QTL, and genes linking flowering time to local adaptation. These include a regulator of abscisic-acid (<em>FLDH</em>) and flowering time genes <em>PIF3, FIO1</em>, and <em>COL5</em>.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Populations of a species may inhabit different environments where local selection pressures favor a combination of (multivariate) phenotypes (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Leimu and Fischer 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Conover, et al. 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hereford 2009</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Savolainen, et al. 2013</a>). Once locally adapted, the resident genotype is expected, on average, to have a higher relative fitness than a foreign genotype (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Kawecki and Ebert 2004</a>). Despite the widespread evidence of local adaptation in many taxa (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Leimu and Fischer 2008</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Jeong and Di Rienzo 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Arguello, et al. 2016</a>), our understanding of the traits involved, its genetic basis, and its environmental underpinnings is still at an infant stage (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Savolainen, et al. 2013</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Tiffin and Ross-Ibarra 2014</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Wadgymar, et al. 2017</a>).</p><p id="p-5">In a variety of species, reciprocal transplant and common garden/laboratory experiments have showed significant adaptive differentiation between natural populations inhabiting different environments (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Via 1991</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hendry, et al. 2002</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Savolainen, et al. 2007</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ågren and Schemske 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Kaufmann, et al. 2017</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Phifer-Rixey, et al. 2018</a>). Furthermore, in plants and animals, mapping experiments have uncovered Quantitative Trait Loci (QTL) for traits that are thought to underlie local adaptation (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Colosimo, et al. 2004</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Oakley, et al. 2014</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Yang, et al. 2016</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ågren, et al. 2017</a>), in addition to QTL explaining fitness differences across environments (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ågren, et al. 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Anderson, et al. 2013</a>). Despite the importance of QTL studies in providing direct evidence for local adaptation (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Ågren, et al. 2013</a>), in many instances they provide a low resolution for its genetic basis, and in practical terms are time consuming, expensive, and labor intense (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Joosen, et al. 2009</a>).</p><p id="p-6">With the advent of low-cost, and fast, next generation sequencing (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Henson, et al. 2012</a>), higher resolution population genomics approaches have emerged as the new means for examining the genetic basis of local adaptation. (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lachance and Tishkoff 2013</a>; <a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">Savolainen, et al. 2013</a>; <a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Tiffin and Ross-Ibarra 2014</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Sork 2017</a>). In brief, these methods include: (a) identifying single nucleotide polymorphisms (SNPs) showing significant allele frequency differentiation between populations (<em>F<sub>ST</sub></em>) (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Beaumont and Balding 2004</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Foll and Gaggiotti 2008</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">de Villemereuil and Gaggiotti 2015</a>); (b) identifying genomic regions showing significant increases in linkage disequilibrium (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Jacobs, et al. 2016</a>) or composite likelihood ratios for recent sweeps (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">DeGiorgio, et al. 2016</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Huber, et al. 2016</a>); and (c) alleles showing significant correlations to environment/climate (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hancock, et al. 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Jones, et al. 2012</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Lasky, et al. 2012</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Lasky, et al. 2014</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Pluess, et al. 2016</a>; <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Yeaman, et al. 2016</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Monroe, et al. 2018</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Price, et al. 2018</a>). The latter approach has gained particular attention because it can be implemented on the basis of individual (as opposed to population-based) sampling, and furthermore it provides a direct link to ecologically relevant factors (e.g., climate).</p><p id="p-7">Despite the ability of population genomic methods to identify candidate genetic variation underlying local adaptation, it is hard to disentangle the effects of selection from demographic history (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Lotterhos and Whitlock 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hoban, et al. 2016</a>). Geographically varying environments can generate population structure in the regions of genes involved in adaptation, even under conditions (e.g. high gene flow) that do not generate population structure at a genome-wide level (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">McKay and Latta 2002</a>). The degree to which population structure influences the patterns of genome-wide LD and therefore the occurrence of false positives has emerged as a critical hurdle (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Platt, et al. 2010</a>). To limit the number of spurious associations, studies usually estimate population structure using different methods (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Price, et al. 2010</a>), incorporate population structure (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Yu, et al. 2006</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Kang, et al. 2010</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Wang, et al. 2011</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Zhou and Stephens 2012</a>) and/or geographic structure (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Lasky et al. 2012</a>) into statistical models, and finally test whether certain loci explain significantly higher variation in environment/climate than population structure itself (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Hancock, et al. 2011</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Lasky, et al. 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fischer, et al. 2013</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Huber, et al. 2014</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Lasky, et al. 2014</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Monroe, et al. 2016</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Rellstab, et al. 2017</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Frachon, et al. 2018</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Lasky, et al. 2018</a>; <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Price, et al. 2018</a>). While such approaches may limit the number of false positives, they can also lead to false negatives (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bergelson and Roux 2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Anderson, et al. 2011</a>). According to simulations (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Forester, et al. 2018</a>), when selection is spatially autocorrelated, accounting for population structure reduces the power to detect loci under selection (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Forester, et al. 2018</a>).</p><p id="p-8">The negative effects of accounting for population structure, may explain the reduced signal of genetic convergence to climate among distantly related conifers (<a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">Yeaman, et al. 2016</a>). Furthermore, in <em>Arabidopsis thaliana</em>, it may underlie the lack of significant associations between climate-correlated SNPs and fitness QTL exhibiting genetic tradeoffs (<a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">Price, et al. 2018</a>); and the lack of SNPs showing significant associations to drought survival among Eurasian accessions (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Exposito-Alonso, et al. 2018</a>). Finally, in the plant <em>Capsella bursa-pastoris</em>, accounting for population structure explained all variation in gene expression among ecotypes that differed in important life-history traits such as flowering time and circadian rhythm (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Kryvokhyzha, et al. 2016</a>).</p><p id="p-9">In conjunction to population genomic signatures of selection or significant associations to climate, genetic variation underlying local adaptation is expected to be enriched along sites that are functional and influence fitness. SNPs showing significant associations to climate were found to be significantly enriched among nonsynonymous, but also synonymous variation (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Hancock, et al. 2011</a>; <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Lasky, et al. 2012</a>). The enrichment among synonymous variation (which largely evolve neutrally) maybe the result of linkage disequilibrium due to neutral processes but also background selection (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Charlesworth, et al. 1993</a>) and/or hitchhiking (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gillespie 2000</a>). A stricter enrichment test will be one that controls for sequence conservation along coding and non-coding sites. Sites that are highly conserved among species (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Miller, et al. 2007</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Haudry, et al. 2013</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hupalo and Kern 2013</a>), are assumed to be under functional/selective constraint and functionally important — that is, due to purifying selection the number of tolerated mutations is very limited (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Graur 2016</a>). Therefore, SNPs showing significant evidence of local adaptation across highly constraint sites are more likely to be true positives.</p><p id="p-10">In the current study, we first examine how accounting for population structure in genome-wide associations to climate may affect our ability to provide a comprehensive picture on the genetic basis of local adaptation; and secondly, we use an approach that accounts for genetic signatures of selection and functional constraint to detect potential genes underlying local adaptation.</p><p id="p-11">More specifically, during the first phase we use 875 <em>A. thaliana</em> Eurasian accessions and identify SNPs showing significant correlations to Minimum Temperature of Coldest Month (Min.Tmp.Cld.M) using a mixed linear model that accounts for population structure (<a id="xref-ref-102-2" class="xref-bibr" href="#ref-102">Zhou and Stephens 2012</a>) (GEMMA-Genome-wide Efficient Mixed Model Association) and simple Spearman correlations (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Spearman 1987</a>) that did not account for population structure (we mainly focused on Min.Tmp.Cld.M —a proxy to winter temperature—because of significant evidence linking cold acclimation to local adaptation in wild populations (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Ågren and Schemske 2012</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Oakley, et al. 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gienapp, et al. 2017</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Oakley, et al. 2018</a>)). Using the two sets of SNPs, we first separated them into ones identified by both methods (referred to as “Common” hereafter), and ones uniquely identified by each approach (referred to as “GEMMA” and “Spearman” hereafter). Thereafter, we compared evidence of local adaptation and function underlying the three sets of SNPs. More specifically, using Italy and Sweden re-sequenced genomes, and QTL explaining fitness variation between these populations (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Ågren, et al. 2013</a>), we examined: (a) the level of allele frequency differentiation and linkage disequilibrium underlying SNPs showing significant associations to climate before and after accounting for population structure; (b) how these SNPs were distributed along LOD score peaks of 20 fitness QTLs (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Ågren, et al. 2013</a>); and (c) how they were distributed along nonsynonymous and cis-regulatory sites showing significant functional constraint among plants species of the Brassicaceae family (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Haudry, et al. 2013</a>).</p><p id="p-12">During the second phase, we use SNPs showing significant evidence of local adaptation and function, to identify potential genes underlying fitness QTL and flowering time variation between Italy and Sweden populations. Flowering time is a life history trait that is thought to play a significant role in local adaptation to climate (Hall and Willis 2006; Verhoeven, et al. 2008; Sandring and Agren 2009; Dittmar, et al. 2014; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Ågren, et al. 2017</a>), and whose genetic basis has been thoroughly studied (Salomé, et al. 2011; Sasaki, et al. 2017). To re-examine evidence linking flowering time to climate adaptation we used the following data: (1) a list of genes that were experimentally shown to affect flowering time; (2) high confidence QTL explaining flowering time variation between Italy and Sweden populations (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Ågren, et al. 2017</a>), and (3) flowering time estimates for Arabidopsis Eurasian accessions (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1001 Genomes Consortium 2016</a>).</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Detecting associations to climate with and without accounting for population structure</h3><p id="p-13">To compare allele associations to climate with and without accounting for population structure we focused on the climate variable Minimum Temperature of Coldest Month (Min.Tmp.Cld.M). Using a SNP genotype matrix for a panel of 1,135 globally distributed accessions downloaded from the 1001 Genomes database, we filtered out accessions from outside the native Eurasian and North African range of <em>A. thaliana</em>, as these accessions may have weaker patterns of local adaptation (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Lasky, et al. 2012</a>). We also filtered out accessions that were likely laboratory escapees or contaminants (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Pisupati, et al. 2017</a>), leaving 875 accessions. After we filtered for biallelic SNPs with minor allele frequency &gt;0.05, we tested association with home climate of ecotype and tested for potential confounding effects of population structure using the software “gemma” (<a id="xref-ref-102-3" class="xref-bibr" href="#ref-102">Zhou and Stephens 2012</a>). The parameters used in gemma were a MAF of 0.05 (default 0.01) and a missingness threshold of 0.05. For the linear mixed model option, we used Wald test (default) to test for significant associations to climate. We tested models where home climate was a function of SNP allele, and the association p-values we report are for the null hypothesis that the mean climate occupied by the two alleles is equal (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Lasky, et al. 2014</a>). Using the same set of SNPs we estimated correlations to climate using simple “Spearman” correlations (<a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">Spearman 1987</a>) and not accounting for population structure. To estimate p-values for the Spearman correlations we used the ‘cor.test’ function implemented in R (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Team 2009</a>).</p></div><div id="sec-4" class="subsection"><h3>Estimates of selection and candidate functional variation underlying populations in North Sweden and South Italy</h3><p id="p-14">In addition to associations to climate, genomic signatures of selection were examined in North Sweden and South Italy populations that represent the most northern and southern tips of Eurasia. The accessions used, including latitude and longitude coordinates are found in the Supplementary data file. Evidence of local adaptation/selection across SNPs between Italy and Sweden populations was measured using absolute allele frequency differentiation (<em>F<sub>ST≈</sub></em> |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>|) and linkage disequilibrium (LD) between a SNP and its neighboring SNPs with a 20kb window. LD was measured using the package ‘PLINK’ (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Purcell, et al. 2007</a>) and it was estimated as the mean square coefficient of correlation <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/embed/inline-graphic-1.gif"/></noscript></span></span>. For evidence of recent sweeps we used previously calculated (<a id="xref-ref-77-4" class="xref-bibr" href="#ref-77">Price, et al. 2018</a>) Composite likelihood Ratios (CLR’s) that were computed using Sweepfinder2 (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">DeGiorgio, et al. 2016</a>). We focused on CLR’s in North Sweden because in other populations signals identified were very weak (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Long, et al. 2013</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Huber, et al. 2014</a>; <a id="xref-ref-77-5" class="xref-bibr" href="#ref-77">Price, et al. 2018</a>).</p><p id="p-15">To narrow down SNPs to ones that are more likely to underlie differences in function/expression of protein-coding genes we focused on cis-regulatory and nonsynonymous variation that was found along sites showing significant functional constraint. We regarded cis-regulatory SNPs as those found within 1 kb upstream from the transcriptional start site of a gene (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Zou, et al. 2011</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Pass, et al. 2017</a>), unless these sites were found in transcribed regions of other genes (in which case they were excluded). To call nonsynonymous variation we used bi-allelic sites, we used a publicly available python script (callSynNonSyn.py; archived at <a href="https://github.com/kern-lab/">https://github.com/kern-lab/</a>), and gene models downloaded from the TAIR database (TAR10 genome release) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Berardini, et al. 2015</a>). To annotate regions showing significant functional constraint across the <em>A. thaliana</em> genome we used phastCons scores (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Siepel, et al. 2005</a>) derived using a nine-way alignment of Brassicaceae species from the study by <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Haudry et al. (2013)</a>. We defined conserved regions as those with a score &gt;=0.8 over blocks of &gt;=10 nucleotides.</p></div><div id="sec-5" class="subsection"><h3>Fitness and flowering time QTL underlying Italy and Sweden populations</h3><p id="p-16">Quantitative trait loci explaining fitness variation between natural <em>A. thaliana</em> Italy and Sweden populations were retrieved by the study of <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Ågren et al. (2013)</a>. These 20 fitness QTL were assembled into 6 genetic tradeoff QTL (<a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Ågren, et al. 2013</a>), however we treated them as independent given the very long genetic distances between fitness QTL peaks (Supplementary data). Furthermore, we retrieved high confidence QTL explaining flowering time variation between these populations (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Ågren, et al. 2017</a>) (Supplementary data).</p></div><div id="sec-6" class="subsection"><h3>Circular permutation tests</h3><p id="p-17">To test whether SNPs showing significant correlations to Min.Tmp.Cld.M and/or SNPs showing high <em>F<sub>ST</sub></em> and LD between Italy and Sweden populations are enriched along QTL peaks and among cis-regulatory/nonsynonymous variation at sites showing significant functional constraint we used a 1,000 circular permutations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagram of circular permutations used to build null distributions for various measures. The first step involves choosing a random location along the genome and shift p-values or allele frequency differentiations. In the next step, when examining climate associations, we chose SNPs with a p-value less than a specified threshold. We also samples of SNPs showing a high FST and LD. SNPs showing &#x201C;significant&#x201D; correlations to climate were further partitioned into those showing &#x201C;significance&#x201D; using both Spearman and GEMMA correlations (&#x201C;Common&#x201D;) or were unique to each approach (&#x201C;GEMMA&#x201D; or &#x201C;Spearman&#x201D;). Using the final set of SNPs, we computed various measures of interest." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-544092932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diagram of circular permutations used to build null distributions for various measures. The first step involves choosing a random location along the genome and shift p-values or allele frequency differentiations. In the next step, when examining climate associations, we chose SNPs with a p-value less than a specified threshold. We also samples of SNPs showing a high &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; and LD. SNPs showing &#x201C;significant&#x201D; correlations to climate were further partitioned into those showing &#x201C;significance&#x201D; using both Spearman and GEMMA correlations (&#x201C;Common&#x201D;) or were unique to each approach (&#x201C;GEMMA&#x201D; or &#x201C;Spearman&#x201D;). Using the final set of SNPs, we computed various measures of interest.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F1.medium.gif" width="427" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F1.medium.gif" width="427" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <p id="p-18" class="first-child">Diagram of circular permutations used to build null distributions for various measures. The first step involves choosing a random location along the genome and shift p-values or allele frequency differentiations. In the next step, when examining climate associations, we chose SNPs with a p-value less than a specified threshold. We also samples of SNPs showing a high <em>F<sub>ST</sub></em> and LD. SNPs showing “significant” correlations to climate were further partitioned into those showing “significance” using both Spearman and GEMMA correlations (“Common”) or were unique to each approach (“GEMMA” or “Spearman”). Using the final set of SNPs, we computed various measures of interest.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19"><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a> depicts the steps involved in the circular permutations. In brief, during the first step p-values associated with climate correlations or allele frequency differentiations were shifted based on a random SNP along the genome. The second step involved choosing SNPs showing “significant” associations to climate (p-value &lt; 1<sup>st</sup> percentile of Spearman/GEMMA p-value distributions), or high allele frequency differentiations and LD (|<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>|&gt;0.70 and LD&gt;0.19: 0.70 and 0.19 represent the 95<sup>th</sup> percentiles of the distributions). For climate associations, SNPs were partitioned into those showing “significance” using both Spearman and GEMMA associations (“Common”) or those showing “significance” using only one approach (“GEMMA” or “Spearman”).</p><p id="p-20">The final sets of SNPs were used to estimate the following measures: (a): % of SNPs found within a certain distance (100, 200, ….600 kb) upstream and downstream of the 20 fitness QTL peaks; and (b) the proportion of cis-regulatory and nonsynonymous SNPs that were within regions showing significant functional constraint.</p></div><div id="sec-7" class="subsection"><h3>Sliding window analysis of chromosomal variation SNPs showing evidence of local adaptation</h3><p id="p-21">To detect chromosomal regions with a high proportion of SNPs showing significant evidence of local adaptation we used a sliding window approach. Specifically, for a window size of 20kb and a step size of 1kb we estimated the ratio of SNPs showing a specific requirement (e.g., <em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>|&gt;0.70 and LD&gt;0.19) over the total number of SNPs within a 20kb window.</p></div><div id="sec-8" class="subsection"><h3>Flowering time estimates for A. thaliana Eurasian accessions and candidate flowering time genes</h3><p id="p-22">Estimates of flowering time for the 835 Eurasian <em>A. thaliana</em> accessions were downloaded from the study by Alonso-Blanco et. al (2016) (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1001 Genomes Consortium 2016</a>). In brief, plants were grown in growth chambers with the following settings: after 6 days of stratification in the dark at 4°C, constant temperature of 16°C with 16 hours light / 8 hours darkness, 65% humidity. Flowering time was scored as days until first open flower. See Alonso-Blanco et al. (2016) for further details. A set of genes that were experimentally verified to affect flowering time was downloaded from Prof. Dr. George Coupland website (<a href="https://www.mpipz.mpg.de/14637/Arabidopsis_flowering_genes">https://www.mpipz.mpg.de/14637/Arabidopsis_flowering_genes</a>)</p></div><div id="sec-9" class="subsection"><h3>Constructing rooted gene trees</h3><p id="p-23">To build neighbor joining trees of genes showing significant local adaptation we downloaded 1:1 orthologs between <em>Arabidopsis thaliana</em> and outgroups <em>Arabidopsis lyrata</em> and <em>Capsella rubella</em> from the Phytozome database (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Goodstein, et al. 2012</a>) and after aligning the coding sequences with MAFFT (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Katoh and Toh 2008</a>) we used MEGA (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Tamura, et al. 2013</a>) to build a rooted gene trees.</p></div></div><div class="section" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Genomic signatures of local adaptation and selection captured by climate associations in Italy and Sweden populations</h3><p id="p-24">Genome wide correlations to Minimum Temperature of Coldest Month (Min.Tmp.Cld.M) were examined using simple Spearman correlations (<a id="xref-ref-89-3" class="xref-bibr" href="#ref-89">Spearman 1987</a>) and a mixed model called GEMMA (<a id="xref-ref-102-4" class="xref-bibr" href="#ref-102">Zhou and Stephens 2012</a>) that accounted for putative population structure. Fig. S1 depicts the p-value distributions that were obtained when testing for significance using GEMMA and Spearman correlations. P-values underlying Spearman correlation were skewed towards very low p-values; many of which are likely false positives. Nonetheless, we compared the results obtained by the two methods using different percentiles of the p-value distributions as cutoffs for significance (10<sup>th</sup>, 5<sup>th</sup>, 1<sup>st</sup> percentiles). For significant SNPs that were segregating between North Sweden and South Italy populations the percent of SNPs that were identified by both approaches (“Common”) at the three significance levels was ~20% (Fig. S2).</p><p id="p-25">When comparing the average absolute allele frequency differentiation |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| and LD <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/embed/inline-graphic-2.gif"/></noscript></span></span> across SNPs that were unique to each approach (“Spearman” or “GEMMA”) and SNPs identified by both methods (“Common”) the set of “Common” and “Spearman” SNPs showed significantly higher allele frequency differentiation and LD than the genome average (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig 2A</a>-<a id="xref-fig-2-2" class="xref-fig" href="#F2">2B</a>). On the other hand, SNPs that were unique to “GEMMA” did not show any large differences in <em>F<sub>ST</sub></em> and LD from the genome average (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figs 2A</a>-<a id="xref-fig-2-4" class="xref-fig" href="#F2">2B</a>). Furthermore, across “Common” and “Spearman” SNPs we see a decrease in (|<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>|) and LD as the threshold of significance becomes less stringent (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figs 2A</a>-<a id="xref-fig-2-6" class="xref-fig" href="#F2">2B</a>). This is indicative that stronger associations capture SNPs showing stronger evidence of local adaptation and recent selection. Therefore, for any further tests we used the set of SNPs that were significant using the 1<sup>st</sup> percentile of the p-value distributions as the cutoff.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing allele frequency differentiation and linkage disequilibrium across SNPs showing significant associations to Min.Tmp.Cld.M. (A-B) Average absolute allele frequency differentiation (|fN.Sweden &#x2013; fS.Italy|) and linkage disequilibrium () was significantly higher the genome average (dotted lines) across SNPs identified by both Spearman and GEMMA correlations (&#x201C;Common&#x201D;) and SNPs uniquely identified by Spearman correlations (&#x201C;Spearman&#x201D;). These sets of SNPs also showed a decrease in |fN.Sweden &#x2013; fS.Italy| and  decreased as the the threshold for significance became more lenient (1&#x2192;10 %) (95% CI&#x2019;s which are not visible were estimated using a bootstrap approach). (C-E) A positive association between |fN.Sweden &#x2013; fS.Italy| and  across the three sets of SNPs showing significant correlations to Min.Tmp.Cld.M when using the 1st percentile of the p-value distributions (Fig. S1) as the significance level." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-544092932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparing allele frequency differentiation and linkage disequilibrium across SNPs showing significant associations to Min.Tmp.Cld.M. (A-B) Average absolute allele frequency differentiation (|&lt;em&gt;f&lt;sub&gt;N.Sweden&lt;/sub&gt;&lt;/em&gt; &#x2013; &lt;em&gt;f&lt;sub&gt;S.Italy&lt;/sub&gt;&lt;/em&gt;|) and linkage disequilibrium () was significantly higher the genome average (dotted lines) across SNPs identified by both Spearman and GEMMA correlations (&#x201C;Common&#x201D;) and SNPs uniquely identified by Spearman correlations (&#x201C;Spearman&#x201D;). These sets of SNPs also showed a decrease in |&lt;em&gt;f&lt;sub&gt;N.Sweden&lt;/sub&gt;&lt;/em&gt; &#x2013; &lt;em&gt;f&lt;sub&gt;S.Italy&lt;/sub&gt;&lt;/em&gt;| and  decreased as the the threshold for significance became more lenient (1&#x2192;10 %) (95% CI&#x2019;s which are not visible were estimated using a bootstrap approach). (C-E) A positive association between |&lt;em&gt;f&lt;sub&gt;N.Sweden&lt;/sub&gt;&lt;/em&gt; &#x2013; &lt;em&gt;f&lt;sub&gt;S.Italy&lt;/sub&gt;&lt;/em&gt;| and  across the three sets of SNPs showing significant correlations to Min.Tmp.Cld.M when using the 1&lt;sup&gt;st&lt;/sup&gt; percentile of the p-value distributions (Fig. S1) as the significance level.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F2.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F2.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <p id="p-26" class="first-child">Comparing allele frequency differentiation and linkage disequilibrium across SNPs showing significant associations to Min.Tmp.Cld.M. (A-B) Average absolute allele frequency differentiation (|<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>|) and linkage disequilibrium (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F2/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F2/embed/inline-graphic-3.gif"/></noscript></span></span>) was significantly higher the genome average (dotted lines) across SNPs identified by both Spearman and GEMMA correlations (“Common”) and SNPs uniquely identified by Spearman correlations (“Spearman”). These sets of SNPs also showed a decrease in |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F2/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F2/embed/inline-graphic-4.gif"/></noscript></span></span> decreased as the the threshold for significance became more lenient (1→10 %) (95% CI’s which are not visible were estimated using a bootstrap approach). (C-E) A positive association between |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F2/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F2/embed/inline-graphic-5.gif"/></noscript></span></span> across the three sets of SNPs showing significant correlations to Min.Tmp.Cld.M when using the 1<sup>st</sup> percentile of the p-value distributions (Fig. S1) as the significance level.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Local adaptation is expected to lead to an increase in allele frequency differentiation and LD at and near the site under selection; in other words, we would expect a positive correlation between |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| and LD. As shown in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figs 2C</a>-<a id="xref-fig-2-8" class="xref-fig" href="#F2">2E</a> all three sets of SNPs showed significant associations between |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| and LD. On the other hand, while “Spearman” SNPs were enriched in SNPs showing a |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| near fixation (&gt;0.90) the level of LD was lower (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2D</a>) (also reflected in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2B</a>). Finally, as expected given the small deviations from the genome average (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figs 2A</a>-<a id="xref-fig-2-12" class="xref-fig" href="#F2">2B</a>), “GEMMA” SNPs showed the poorest associations with a small sample of SNPs showing a |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| between 0.60-0.80 and LD between 0.30-0.40 (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Fig. 2E</a>). Taken together, indicate indicate that “Common” and “Spearman” SNPs capture higher population genomic evidence of local adaptation, than SNPs uniquely identified by GEMMA.</p></div><div id="sec-12" class="subsection"><h3>Detecting recent selection underlying fitness QTL is significantly reduced when accounting for population structure in GWA to climate</h3><p id="p-28">Direct evidence of local adaptation was shown in a study by <a id="xref-ref-3-7" class="xref-bibr" href="#ref-3">Ågren et al. (2013)</a> in which they identified 20 QTL explaining fitness variation between Italy and Sweden populations (Supplementary data). To examine population genomic evidence of local adaptation underlying these QTL we first examined |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| and LD which are measures specific to Italy and Sweden populations. If the peaks of the QTL identified, are on average, near the site under selection we would expect a significant enrichment of high <em>F<sub>ST</sub></em> and LD SNPs. Using different window sizes (100, 200, ….600 kb) from the QTL peaks we compared the observed proportion of high <em>F<sub>ST</sub></em> SNPs (|<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| &gt;0.70 —0.70 represents the 95<sup>th</sup> percentile of the distribution), and the proportion of high <em>F<sub>ST</sub></em> and LD SNPs (LD&gt;0.19 — 0.19 represents the 95<sup>th</sup> percentile of the distribution) to the expected proportion derived using a circular permutation test. As expected, we observed a significantly high proportion of high <em>F<sub>ST</sub></em> (&gt;<em>0.70</em>) SNPs and high <em>F<sub>ST</sub></em> and high LD SNPs (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figs 3A</a>-<a id="xref-fig-3-2" class="xref-fig" href="#F3">3B</a>). A significant proportion was also observed at 200 kb, but as window sizes became larger the proportions became insignificant (&lt;95<sup>th</sup> percentile) and normalized to the genome average.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of SNPs showing population genomic evidence of local adaptation along fitness QTL peaks. (A) The observed proportion of SNPs showing significantly high allele frequency differentiation (|fN.Sweden &#x2013; fS.Italy| &gt; 0.70) (red line) in comparison to a permutation distribution of random proportions. The observed proportion was greater than the 95th percentile. (B) The observed proportion of SNPs showing a |fN.Sweden &#x2013; fS.Italy| &gt; 0.70 and significantly high linkage disequilibrium , was also greater than the 95th percentile of random proportions. (C) The observed proportions of SNPs showing significant associations to climate, in addition to a |fN.Sweden &#x2013; fS.Italy| &gt;0.70 and a . The proportion of &#x201C;Common&#x201D; and &#x201C;GEMMA&#x201D; SNPs near QTL peaks was very low, while the proportion &#x201C;Spearman&#x201D; SNPs was greater than the 80th percentile of the distribution but less than the 95th." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-544092932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of SNPs showing population genomic evidence of local adaptation along fitness QTL peaks. (A) The observed proportion of SNPs showing significantly high allele frequency differentiation (|&lt;em&gt;f&lt;sub&gt;N.Sweden&lt;/sub&gt;&lt;/em&gt; &#x2013; &lt;em&gt;f&lt;sub&gt;S.Italy&lt;/sub&gt;&lt;/em&gt;| &gt; 0.70) (red line) in comparison to a permutation distribution of random proportions. The observed proportion was greater than the 95&lt;sup&gt;th&lt;/sup&gt; percentile. (B) The observed proportion of SNPs showing a |&lt;em&gt;f&lt;sub&gt;N.Sweden&lt;/sub&gt;&lt;/em&gt; &#x2013; &lt;em&gt;f&lt;sub&gt;S.Italy&lt;/sub&gt;&lt;/em&gt;| &gt; 0.70 and significantly high linkage disequilibrium , was also greater than the 95&lt;sup&gt;th&lt;/sup&gt; percentile of random proportions. (C) The observed proportions of SNPs showing significant associations to climate, in addition to a |&lt;em&gt;f&lt;sub&gt;N.Sweden&lt;/sub&gt;&lt;/em&gt; &#x2013; &lt;em&gt;f&lt;sub&gt;S.Italy&lt;/sub&gt;&lt;/em&gt;| &gt;0.70 and a . The proportion of &#x201C;Common&#x201D; and &#x201C;GEMMA&#x201D; SNPs near QTL peaks was very low, while the proportion &#x201C;Spearman&#x201D; SNPs was greater than the 80&lt;sup&gt;th&lt;/sup&gt; percentile of the distribution but less than the 95&lt;sup&gt;th&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F3.medium.gif" width="440" height="262"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F3.medium.gif" width="440" height="262"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <p id="p-29" class="first-child">Distribution of SNPs showing population genomic evidence of local adaptation along fitness QTL peaks. (A) The observed proportion of SNPs showing significantly high allele frequency differentiation (|<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| &gt; 0.70) (red line) in comparison to a permutation distribution of random proportions. The observed proportion was greater than the 95<sup>th</sup> percentile. (B) The observed proportion of SNPs showing a |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| &gt; 0.70 and significantly high linkage disequilibrium <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F3/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F3/embed/inline-graphic-6.gif"/></noscript></span></span>, was also greater than the 95<sup>th</sup> percentile of random proportions. (C) The observed proportions of SNPs showing significant associations to climate, in addition to a |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| &gt;0.70 and a <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F3/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/F3/embed/inline-graphic-7.gif"/></noscript></span></span>. The proportion of “Common” and “GEMMA” SNPs near QTL peaks was very low, while the proportion “Spearman” SNPs was greater than the 80<sup>th</sup> percentile of the distribution but less than the 95<sup>th</sup>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Next, we performed the same test using SNPs showing significant associations to climate, and SNPs that showing significant associations to climate and a high <em>F<sub>ST</sub></em> and high LD. When considering only associations to climate (Fig. S3) we find an enrichment of “Common” and “Spearman” SNPs (Fig. S3), but not “GEMMA” SNPs. On the other hand, when we also considered high <em>F<sub>ST</sub></em> and high LD, none of the SNPs showed an enrichment (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3C</a>). Among the three sets, Spearman SNPs showed the highest observed proportion (&gt;80<sup>th</sup> percentile). To determine whether the significantly lower proportion of “Common” SNPs showing a high <em>F<sub>ST</sub></em> and high LD was due to the stringency of our cutoff, we calculated the average <em>F<sub>ST</sub></em> and LD for SNPs with an |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| and &lt;0.70 and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/08/17/642306/embed/inline-graphic-8.gif"/></noscript></span></span>. The average |<em>f<sub>N.Sweden</sub></em> – <em>f<sub>S.Italy</sub></em>| (≈0.26) and LD (≈0.06) were near or lower than the genome average (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Fig. 2A</a>-<a id="xref-fig-2-15" class="xref-fig" href="#F2">2B</a>) to be considered as strong candidates underlying local adaptation.</p><p id="p-31">Given that “Common” SNPs did not show an enrichment within 100kb of the 20 fitness QTL peaks examined (Supplementary data), we searched whether any of these were within the six tradeoff QTL that each span a large genomic region (Supplementary data). As shown in Fig. S4, most of the high <em>F<sub>ST</sub></em> and high LD “Common” SNPs (181/209) were found within a 60kb region (14,719-14,781Mb) of a single genetic tradeoff (GT) QTL (GT QTL 2:2) (Fig. S4). Apart from being limited to a single GT QTL, given the multiple fitness QTL within GT QTL 2:2 (which spanned a length of ~4Mb); the multiple regions showing significant recent sweeps in Sweden; and the large number of windows showing a high proportion of high <em>F<sub>ST</sub></em> and LD SNPs; it is highly unlikely that the “Common” set of SNPs cover all the causative genetic variation within GT QTL 2:2.</p><p id="p-32">To further test for enrichment of “GEMMA” SNPs showing significant associations to climate and within 100 kb of fitness QTL peaks, we used an additional three climate variables (Table S1). Using a lenient cutoff for significance (FDR&lt;0.1), we only identified a few additional SNPs within 100 kb of fitness QTL.</p><p id="p-33">All in all, our results indicate that accounting for population structure when performing GWA to climate significantly reduces our ability to capture recent selection underlying fitness QTL.</p></div><div id="sec-13" class="subsection"><h3>All but “GEMMA” SNPs show enrichment at cis-regulatory and nonsynonymous sites showing significant functional constraint</h3><p id="p-34">SNPs underlying local adaptation are expected to be significantly enriched at sites that affect function and/or expression of protein coding genes. Using the three sets of SNPs showing significant associations to climate (“Common”, “GEMMA”, and “Spearman”) and SNPs showing high <em>F<sub>ST</sub></em> and LD (abbreviated as “<em>F<sub>ST</sub></em>&amp;LD”) we examined their distribution among cis-regulatory and nonsynonymous SNPs at sites showing significant functional constraint among Brassicaceae plants (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Haudry, et al. 2013</a>) (phastCons&gt;0.8).</p><p id="p-35">As shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figs 4A</a> &amp; <a id="xref-fig-4-2" class="xref-fig" href="#F4">4B</a>, the proportions of nonsynonymous/cis-regulatory “Common”, “Spearman”, and high “<em>F<sub>ST</sub></em>&amp;LD” SNPs, was significantly higher than expected by chance. On the other hand, “GEMMA” SNPs did not show a significant enrichment in any of the categories. When considering all the results obtained so far (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Figs 2</a>-<a id="xref-fig-4-3" class="xref-fig" href="#F4">4</a>), unique “Spearman” SNPs seem to capture additional genetic variation underlying local adaptation (i.e., in addition to “Common” SNPs), while unique “GEMMA” SNPs do not.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion of cis-regulatory and nonsynonymous SNPs showing population genomic evidence of local adaptation and function (A) Red dots indicate the observed proportion of nonsynonymous SNPs showing associations to climate (&#x201C;Common&#x201D;, &#x201C;GEMMA&#x201D;, &#x201C;Spearman&#x201D;) or high FST and LD (&#x201C;FST&amp;LD&#x201D;) and within coding regions showing significant functional constraint (phastCons&gt;0.8). Expectations and 95% CI&#x2019;s were derived using circular permutations (B) The observed proportions of cis-regulatory SNPs showing population genomic evidence of local adaptation and found within conserved regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-544092932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion of cis-regulatory and nonsynonymous SNPs showing population genomic evidence of local adaptation and function (A) Red dots indicate the observed proportion of nonsynonymous SNPs showing associations to climate (&#x201C;Common&#x201D;, &#x201C;GEMMA&#x201D;, &#x201C;Spearman&#x201D;) or high &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; and LD (&#x201C;&lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt;&amp;LD&#x201D;) and within coding regions showing significant functional constraint (phastCons&gt;0.8). Expectations and 95% CI&#x2019;s were derived using circular permutations (B) The observed proportions of cis-regulatory SNPs showing population genomic evidence of local adaptation and found within conserved regions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F4.medium.gif" width="440" height="175"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F4.medium.gif" width="440" height="175"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <p id="p-36" class="first-child">Proportion of cis-regulatory and nonsynonymous SNPs showing population genomic evidence of local adaptation and function (A) Red dots indicate the observed proportion of nonsynonymous SNPs showing associations to climate (“Common”, “GEMMA”, “Spearman”) or high <em>F<sub>ST</sub></em> and LD (“<em>F<sub>ST</sub></em>&amp;LD”) and within coding regions showing significant functional constraint (phastCons&gt;0.8). Expectations and 95% CI’s were derived using circular permutations (B) The observed proportions of cis-regulatory SNPs showing population genomic evidence of local adaptation and found within conserved regions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Flowering time genes showing significant evidence of local adaptation and underlying flowering time QTL</h3><p id="p-37">Given the significant enrichment of high <em>F<sub>ST</sub></em> and LD SNPs, along QTL peaks (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3B</a>) and conserved cis-regulatory and nonsynonymous sites (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figs 4A</a> &amp; <a id="xref-fig-4-5" class="xref-fig" href="#F4">4B</a>), we used them to detect potential genes that may underlie fitness QTL (note: we only focused on variants within 100kb of their peaks), in addition to examining evidence of local adaptation underlying a list of ~170 genes affecting flowering time (Supplementary data). To further narrow down on SNPs that are more likely to underlie the fitness QTL examined, we only considered variation that segregated between the parents used to derive the RIL’s (<a id="xref-ref-3-8" class="xref-bibr" href="#ref-3">Ågren, et al. 2013</a>). SNPs between the parental genomes were called in a previous study (<a id="xref-ref-77-6" class="xref-bibr" href="#ref-77">Price, et al. 2018</a>).</p><p id="p-38">Our analysis resulted in 25 genes within 100 kb of fitness QTL peaks and spanning three genetic tradeoff QTL (2:2, 4:2, 5:5) (Table S2). Many of these were involved in interesting biological processes such as: response to different abiotic stress factors and the abscisic-acid signaling pathway which is important in abiotic stress response (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Tuteja 2007</a>) (Table S2). Among these genes, two of them (AT4G33360 (<em>FLDH</em>), AT4G33470 (<em>HDA14</em>)) showed strong expression GxE interactions (GxE interactions were identified in a previous study (<a id="xref-ref-77-7" class="xref-bibr" href="#ref-77">Price, et al. 2018</a>)) when Italy and Sweden plants were grown under cold acclimation conditions (4 °C) for two weeks (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gehan, et al. 2015</a>). Interestingly, <em>FLDH</em> is a negative regulator of the abscisic acid signaling pathway (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bhandari, et al. 2010</a>). As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a> this gene was within a region of a genetic tradeoff QTL that showed a significantly high proportion of high <em>F<sub>ST</sub></em> and high LD SNPs. Expression of <em>FLDH</em> under control and cold acclimation conditions was significantly lower in Sweden than Italy plants (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5A</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness QTL and flowering time genes showing significant evidence of local adaptation along conserved cis-regulatory and nonsynonymous sites. (A) FLDH a negative regulator of ABA (Bhandari, et al. 2010) was found within a genetic tradeoff QTL 4:2 and 100 kb from a fitness QTL peak (red and blue arrows represent QTL where the Sweden genotype had lower fitness in Italy and higher fitness in Sweden, respectively). FLDH was found within a region showing a high proportion of high FST and high LD SNPs. In Italy and Sweden plants it showed strong expression GxE interactions under control and cold acclimation conditions for two weeks (FPKM: Fragments Per Kilobase Million). (B) PIF3 is a phytochrome interacting factor that has been found to affect flowering time (Oda, et al. 2004) and was found underlying a region along chromosome 1 that showed the largest Composite Likelihood Ratio (CLR) of a recent sweep in Sweden and windows with a high proportion of high FST and high LD SNPs. A rooted phylogeny of the PIF3 coding region indicated that Eurasian accessions sharing the same allele as the Sweden parent (blue dot) show significantly higher flowering time than accessions sharing the same allele as the Italy parent (red dot). (C) COL5 is another gene that has been found to affect flowering time (Hassidim, et al. 2009) and in which Eurasian accessions show significant genetic differentiation and segregation in flowering time. This gene is also found within previously identified flowering time QTL (FlrT-5:4) (&#xC5;gren, et al. 2017) in which the Sweden genotype was associated with longer flowering time in both Italy and Sweden" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-544092932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fitness QTL and flowering time genes showing significant evidence of local adaptation along conserved cis-regulatory and nonsynonymous sites. (A) &lt;em&gt;FLDH&lt;/em&gt; a negative regulator of ABA (Bhandari, et al. 2010) was found within a genetic tradeoff QTL 4:2 and 100 kb from a fitness QTL peak (red and blue arrows represent QTL where the Sweden genotype had lower fitness in Italy and higher fitness in Sweden, respectively). &lt;em&gt;FLDH&lt;/em&gt; was found within a region showing a high proportion of high &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; and high LD SNPs. In Italy and Sweden plants it showed strong expression GxE interactions under control and cold acclimation conditions for two weeks (FPKM: Fragments Per Kilobase Million). (B) &lt;em&gt;PIF3&lt;/em&gt; is a phytochrome interacting factor that has been found to affect flowering time (Oda, et al. 2004) and was found underlying a region along chromosome 1 that showed the largest Composite Likelihood Ratio (CLR) of a recent sweep in Sweden and windows with a high proportion of high &lt;em&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/em&gt; and high LD SNPs. A rooted phylogeny of the &lt;em&gt;PIF3&lt;/em&gt; coding region indicated that Eurasian accessions sharing the same allele as the Sweden parent (blue dot) show significantly higher flowering time than accessions sharing the same allele as the Italy parent (red dot). (C) &lt;em&gt;COL5&lt;/em&gt; is another gene that has been found to affect flowering time (Hassidim, et al. 2009) and in which Eurasian accessions show significant genetic differentiation and segregation in flowering time. This gene is also found within previously identified flowering time QTL (FlrT-5:4) (&#xC5;gren, et al. 2017) in which the Sweden genotype was associated with longer flowering time in both Italy and Sweden&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F5.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F5.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/17/642306/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5.</span> <p id="p-39" class="first-child">Fitness QTL and flowering time genes showing significant evidence of local adaptation along conserved cis-regulatory and nonsynonymous sites. (A) <em>FLDH</em> a negative regulator of ABA (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Bhandari, et al. 2010</a>) was found within a genetic tradeoff QTL 4:2 and 100 kb from a fitness QTL peak (red and blue arrows represent QTL where the Sweden genotype had lower fitness in Italy and higher fitness in Sweden, respectively). <em>FLDH</em> was found within a region showing a high proportion of high <em>F<sub>ST</sub></em> and high LD SNPs. In Italy and Sweden plants it showed strong expression GxE interactions under control and cold acclimation conditions for two weeks (FPKM: Fragments Per Kilobase Million). (B) <em>PIF3</em> is a phytochrome interacting factor that has been found to affect flowering time (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Oda, et al. 2004</a>) and was found underlying a region along chromosome 1 that showed the largest Composite Likelihood Ratio (CLR) of a recent sweep in Sweden and windows with a high proportion of high <em>F<sub>ST</sub></em> and high LD SNPs. A rooted phylogeny of the <em>PIF3</em> coding region indicated that Eurasian accessions sharing the same allele as the Sweden parent (blue dot) show significantly higher flowering time than accessions sharing the same allele as the Italy parent (red dot). (C) <em>COL5</em> is another gene that has been found to affect flowering time (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hassidim, et al. 2009</a>) and in which Eurasian accessions show significant genetic differentiation and segregation in flowering time. This gene is also found within previously identified flowering time QTL (FlrT-5:4) (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Ågren, et al. 2017</a>) in which the Sweden genotype was associated with longer flowering time in both Italy and Sweden</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Among the set of flowering time genes, we identified three (AT1G09530 (<em>PIF3</em>), AT2G21070 (<em>FIO1</em>), and AT5G57660 (<em>COL5</em>)) that contained high <em>F<sub>ST</sub></em> and LD nonsynonymous SNPs within conserved coding regions. Among the three genes, <em>PIF3</em> was found along a chromosomal region that showed the highest CLR for a recent sweep in Sweden and a high density of SNPs showing high <em>F<sub>ST</sub></em> and high LD (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5B</a>).</p><p id="p-41">Eurasian accessions sharing a similar allele as the Sweden parent showed longer flowering time than accessions sharing the same allele as the Italy parent (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5B</a>). The same pattern was observed when examining <em>COL5</em> (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5C</a>), a flowering time gene which was also found within a flowering time QTL (FlrT-5:4, Supplementary data). According to FlrT-5:4, the Sweden genotype was associated with longer flowering time in both Italy and Sweden (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Ågren, et al. 2017</a>). In conjunction, with its overlap to a genetic tradeoff QTL (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Ågren, et al. 2017</a>), indicates a possible role in fitness tradeoffs. Studies have attributed flowering time variation within FlrT-5:4 to <em>VIN3</em> (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1001 Genomes Consortium 2016</a>; <a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Ågren, et al. 2017</a>). Although it may be an additional candidate we did not find any significant genetic differentiation and selection along coding and cis-regulatory sites of <em>VIN3</em>.</p><p id="p-42">When examining flowering time genes with high <em>F<sub>ST</sub></em> and LD along cis-regulatory/nonsynonymous sites that did not show significant functional constraint we identified an additional nine genes; four of which were found within flowering time QTL (<strong>FlrT</strong>): AT1G14920 (GAI); AT1G53090 (SPA4); AT2G22540 (SVP); AT2G47700 (RFI2); AT4G32980 (ATH1-<strong>FlrT4:1</strong>); AT5G24470 (PRR5-<strong>FlrT5:2</strong>); AT5G62640 (ELF5); AT5G65050 (MAF2-<strong>FlrT5:5</strong>); and AT5G65060 (MAF3-<strong>FlrT5:5</strong>). ATH1 was found in genetic tradeoff QTL 4:2, while genes MAF2, and MAF3 were found within genetic tradeoff QTL 5:5 and within 100 kb of fitness QTL peaks.</p></div></div><div class="section" id="sec-15"><h2 class="">Discussion</h2><p id="p-43">In the quest to study the genetic basis of local adaptation using genome wide associations to environment, linear mixed models have emerged as a powerful tool given their ability to account for population structure while testing for significant associations (<a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">Yu, et al. 2006</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kang, et al. 2008</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Kang, et al. 2010</a>; <a id="xref-ref-102-5" class="xref-bibr" href="#ref-102">Zhou and Stephens 2012</a>). Although they provide a robust statistical framework for removing many false positives, the current study shows that such an approach may significantly limit our ability to understand the polygenic basis of local adaptation.</p><p id="p-44">SNPs showing significant associations to climate after accounting for population structure, were not in line with high <em>F<sub>ST</sub></em> and LD SNPs that showed a significant enrichment along fitness QTL peaks and coding/noncoding sites under functional constraint. Since these QTL are of large effect (<a id="xref-ref-3-9" class="xref-bibr" href="#ref-3">Ågren, et al. 2013</a>), the lack of significant associations is less likely to be a result of many alleles having a small effect on fitness. A more likely explanation is that accounting for population structure using genetic relatedness estimated from genome-wide SNPs, can lead to a significant number of false negatives. <em>Arabidopsis thaliana</em> however, is a simple, highly inbred species —to correctly assess the impact of accounting for population structure when examining the genetic basis of local adaptation there needs to be examination of other species with more complex life-history traits and evolutionary dynamics.</p><p id="p-45">A large portion of SNPs that showed significant associations to climate, and SNPs that showed high <em>F<sub>ST</sub></em> and LD between Italy and Sweden populations were enrichment among nonsynonymous/cis-regulatory variation at sites showing significant functional constraint. These results, further support an important role of cis-regulatory (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Lasky, et al. 2014</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Siepel and Arbiza 2014</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Li and Fay 2017</a>; <a id="xref-ref-77-8" class="xref-bibr" href="#ref-77">Price, et al. 2018</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Sackton, et al. 2019</a>) and nonsynonymous variation (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Nachman, et al. 2003</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Coop, et al. 2009</a>; <a id="xref-ref-55-6" class="xref-bibr" href="#ref-55">Lasky, et al. 2012</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Huber, et al. 2014</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Svetec, et al. 2016</a>; <a id="xref-ref-77-9" class="xref-bibr" href="#ref-77">Price, et al. 2018</a>) in adaptation. Among the list of candidate genes underlying fitness QTL and showing significant evidence of local adaptation at functionally constraint sites, we identified <em>FLDH. FLDH</em> is a negative regulator of abscisic-acid signaling (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Bhandari, et al. 2010</a>), that showed strong GxE interactions between Italy and Sweden plants under cold acclimation conditions. Abscisic-acid signaling is known to play an important role in abiotic stress response (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Tuteja 2007</a>), with many studies supporting its role in local adaptation to climate (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Keller, et al. 2012</a>; <a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Lasky, et al. 2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kalladan, et al. 2017</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Ristova, et al. 2017</a>).</p><p id="p-46">In addition to abscisic-acid signaling, our results provide further support for the important role of flowering time in local adaptation to climate. Among a list of genes that were experimentally shown to affect flowering time, we identified three genes (<em>PIF3</em>, <em>FIO1</em>, and <em>COL5</em>) that showed significant evidence of local adaptation and functional constraint along nonsynonymous sites. <em>FIO1</em> was previously shown to contain SNPs that showed significant associations to flowering time among natural Swedish lines (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Sasaki, et al. 2015</a>) and <em>COL5</em> was located within a QTL that explains flowering time variation among Sweden and Italy recombinant inbred lines (<a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">Ågren, et al. 2017</a>). Finally, <em>PIF3</em>, a transcription factor that interacts with phytochromes (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Soy, et al. 2012</a>), has been implicated in multiple biological processes including early hypocotyl growth (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Monte, et al. 2004</a>), photomorphogenesis (Dong, et al. 2017), flowering time (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Oda, et al. 2004</a>), and regulation of physiological responses to temperature (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Jiang, et al. 2017</a>). This highly conserved transcription factor was found within a large region that showed significant evidence of local adaptation. This chromosomal region may involve a single causative variant, or a group of linked genes that interact with <em>PIF3 and</em> were under selection because they contributed to building an advantageous phenotype (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Barton and Bengtsson 1986</a>; <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Yeaman and Whitlock 2011</a>).</p><p id="p-47">When ignoring functional constraint, we identify a list of addition flowering time genes showing significant evidence of local adaptation along nonsynonymous/cis-regulatory sites. Genes such as SVP, MAF2, and MAF3 were previously associated with flowering time variation among natural <em>Arabidopsis</em> accessions (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Caicedo, et al. 2009</a>; <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Sasaki, et al. 2015</a>). Although adaptation may involve sites that are not deeply rooted and/or under strong functional constraint, including additional plant genomes when estimating sequence conservation across species may increase our power to detect functionally important regions. As shown by studies examining adaptation in species ranging from bacteria (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Maddamsetti, et al. 2017</a>) to birds (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Sackton, et al. 2019</a>), addressing functional constraint can improve our understanding of its genetic basis.</p><p id="p-48">Finally, the current study shows that we need a new statistical framework to examine genome wide associations to environment, and furthermore, it increases our understanding on the genes and traits that may underlie local adaptation in <em>A. thaliana</em>.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">modified Figure</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.1" data-doi="10.1016/j.cell.2016.05.063"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">1001 Genomes Consortium</span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">1,135 Genomes Reveal the Global Pattern of Polymorphism in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>:<span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.063%26rft_id%253Dinfo%253Apmid%252F27293186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27293186&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ågren</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oakley</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lundemo</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">DW</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Adaptive divergence in flowering time among natural populations of Arabidopsis thaliana: Estimates of selection and QTL mapping</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">71</span>:<span class="cit-fpage">550</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D71%26rft.spage%253D550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.3" data-doi="10.1073/pnas.1316773110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ågren</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oakley</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lovell</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">DW</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Genetic mapping of adaptation reveals fitness tradeoffs in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">110</span>:<span class="cit-fpage">21077</span>–<span class="cit-lpage">21082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1316773110%26rft_id%253Dinfo%253Apmid%252F24324156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzUyLzIxMDc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMTcvNjQyMzA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.4" data-doi="10.1111/j.1469-8137.2012.04112.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ågren</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">DW</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Reciprocal transplants demonstrate strong adaptive differentiation of the model organism Arabidopsis thaliana in its native range</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">194</span>:<span class="cit-fpage">1112</span>–<span class="cit-lpage">1122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DAgren%26rft.auinit1%253DJ.%26rft.volume%253D194%26rft.issue%253D4%26rft.spage%253D1112%26rft.epage%253D1122%26rft.atitle%253DReciprocal%2Btransplants%2Bdemonstrate%2Bstrong%2Badaptive%2Bdifferentiation%2Bof%2Bthe%2Bmodel%2Borganism%2BArabidopsis%2Bthaliana%2Bin%2Bits%2Bnative%2Brange.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2012.04112.x%26rft_id%253Dinfo%253Apmid%252F22432639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2012.04112.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22432639&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303435400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.5" data-doi="10.1111/j.1365-294X.2012.05522.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rushworth</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colautti</span>  <span class="cit-name-given-names">RI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Genetic trade-offs and conditional neutrality contribute to local adaptation</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">22</span>:<span class="cit-fpage">699</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D22%26rft.spage%253D699%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05522.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05522.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000313726300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.6" data-doi="10.1016/j.tig.2011.04.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Evolutionary genetics of plant adaptation</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr> <span class="cit-vol">27</span>:<span class="cit-fpage">258</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DAnderson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D27%26rft.issue%253D7%26rft.spage%253D258%26rft.epage%253D266%26rft.atitle%253DEvolutionary%2Bgenetics%2Bof%2Bplant%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.04.001%26rft_id%253Dinfo%253Apmid%252F21550682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.04.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21550682&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292413300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.7" data-doi="10.1038/ncomms11855"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Arguello</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cardoso-Moreira</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grenier</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gottipati</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benton</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Extensive local adaptation within the chemosensory system following Drosophila melanogaster’s global expansion</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">ncomms11855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D7%26rft.spage%253Dncomms11855%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11855%26rft_id%253Dinfo%253Apmid%252F27292132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27292132&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bengtsson</span>  <span class="cit-name-given-names">BO</span></span>. <span class="cit-pub-date">1986</span>. <span class="cit-article-title">The barrier to genetic exchange between hybridising populations</span>. <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">57</span> (<span class="cit-issue">Pt 3</span>):<span class="cit-fpage">357</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D57%26rft.spage%253D357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.9" data-doi="10.1111/j.1365-294X.2004.02125.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Identifying adaptive genetic divergence among populations from genome scans</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">969</span>–<span class="cit-lpage">980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBeaumont%26rft.auinit1%253DM.%2BA.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D969%26rft.epage%253D980%26rft.atitle%253DIdentifying%2Badaptive%2Bgenetic%2Bdivergence%2Bamong%2Bpopulations%2Bfrom%2Bgenome%2Bscans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02125.x%26rft_id%253Dinfo%253Apmid%252F15012769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02125.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012769&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220153000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.10" data-doi="10.1002/dvg.22877"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berardini</span>  <span class="cit-name-given-names">TZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reiser</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mezheritsky</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strait</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huala</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The Arabidopsis information resource: Making and mining the “gold standard” annotated reference plant genome</span>. <abbr class="cit-jnl-abbrev">Genesis</abbr> <span class="cit-vol">53</span>:<span class="cit-fpage">474</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenesis%26rft.volume%253D53%26rft.spage%253D474%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvg.22877%26rft_id%253Dinfo%253Apmid%252F26201819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvg.22877&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26201819&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.11" data-doi="10.1038/nrg2896"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">F</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Towards identifying genes underlying ecologically relevant traits in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">867</span>–<span class="cit-lpage">879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBergelson%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D867%26rft.epage%253D879%26rft.atitle%253DTowards%2Bidentifying%2Bgenes%2Bunderlying%2Becologically%2Brelevant%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2896%26rft_id%253Dinfo%253Apmid%252F21085205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085205&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.12" data-doi="10.1104/pp.110.157784"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhandari</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">AH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crowell</span>  <span class="cit-name-given-names">DN</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Identification of a novel abscisic acid-regulated farnesol dehydrogenase from Arabidopsis</span>. <abbr class="cit-jnl-abbrev">Plant Physiol</abbr> <span class="cit-vol">154</span>:<span class="cit-fpage">1116</span>–<span class="cit-lpage">1127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.157784%26rft_id%253Dinfo%253Apmid%252F20807998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU0LzMvMTExNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzE3LzY0MjMwNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.13" data-doi="10.1093/molbev/msn300"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caicedo</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">IM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Purugganan</span>  <span class="cit-name-given-names">MD</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Complex rearrangements lead to novel chimeric gene fusion polymorphisms at the Arabidopsis thaliana MAF2-5 flowering time gene cluster</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">26</span>:<span class="cit-fpage">699</span>–<span class="cit-lpage">711</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn300%26rft_id%253Dinfo%253Apmid%252F19139056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msn300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19139056&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263420900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.14" data-doi="10.1111/j.0014-3820.2002.tb00188.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">MT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">1993</span>. <span class="cit-article-title">The effect of deleterious mutations on neutral molecular variation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">134</span>:<span class="cit-fpage">1289</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB.%26rft.volume%253D134%26rft.issue%253D4%26rft.spage%253D1289%26rft.epage%253D1303%26rft.atitle%253DThe%2Beffect%2Bof%2Bdeleterious%2Bmutations%2Bon%2Bneutral%2Bmolecular%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2002.tb00188.x%26rft_id%253Dinfo%253Apmid%252F8375663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNC80LzEyODkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOC8xNy82NDIzMDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.15" data-doi="10.1371/journal.pbio.0020109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Colosimo</span>  <span class="cit-name-given-names">PF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peichel</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nereng</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blackman</span>  <span class="cit-name-given-names">BK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schluter</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kingsley</span>  <span class="cit-name-given-names">DM</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">The genetic architecture of parallel armor plate reduction in threespine sticklebacks</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr> <span class="cit-vol">2</span>:<span class="cit-fpage">E109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DColosimo%26rft.auinit1%253DP.%2BF.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253DE109%26rft.epage%253DE109%26rft.atitle%253DThe%2Bgenetic%2Barchitecture%2Bof%2Bparallel%2Barmor%2Bplate%2Breduction%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020109%26rft_id%253Dinfo%253Apmid%252F15069472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15069472&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.16" data-doi="10.1111/j.1749-6632.2009.04575.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Conover</span>  <span class="cit-name-given-names">DO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duffy</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hice</span>  <span class="cit-name-given-names">LA</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The covariance between genetic and environmental influences across ecological gradients: reassessing the evolutionary significance of countergradient and cogradient variation</span>. <abbr class="cit-jnl-abbrev">Ann N Y Acad Sci</abbr> <span class="cit-vol">1168</span>:<span class="cit-fpage">100</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DConover%26rft.auinit1%253DD.%2BO.%26rft.volume%253D1168%26rft.spage%253D100%26rft.epage%253D129%26rft.atitle%253DThe%2Bcovariance%2Bbetween%2Bgenetic%2Band%2Benvironmental%2Binfluences%2Bacross%2Becological%2Bgradients%253A%2Breassessing%2Bthe%2Bevolutionary%2Bsignificance%2Bof%2Bcountergradient%2Band%2Bcogradient%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2009.04575.x%26rft_id%253Dinfo%253Apmid%252F19566705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2009.04575.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19566705&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268507900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.17" data-doi="10.1371/journal.pgen.1000500"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Absher</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The role of geography in human adaptation</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">e1000500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCoop%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000500%26rft.epage%253De1000500%26rft.atitle%253DThe%2Brole%2Bof%2Bgeography%2Bin%2Bhuman%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000500%26rft_id%253Dinfo%253Apmid%252F19503611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19503611&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Villemereuil</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">OE</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">A new FST-based method to uncover local adaptation using environmental variables</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">1248</span>–<span class="cit-lpage">1258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D6%26rft.spage%253D1248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.19" data-doi="10.1093/bioinformatics/btw051"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">DeGiorgio</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">SweepFinder2: increased sensitivity, robustness and flexibility</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>:<span class="cit-fpage">1895</span>–<span class="cit-lpage">1897</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw051%26rft_id%253Dinfo%253Apmid%252F27153702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153702&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Exposito-Alonso</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vasseur</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burbano</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Genomic basis and evolutionary potential for extreme drought adaptation in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Nat Ecol Evol</abbr> <span class="cit-vol">2</span>:<span class="cit-fpage">352</span>–<span class="cit-lpage">358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BEcol%2BEvol%26rft.volume%253D2%26rft.spage%253D352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.21" data-doi="10.1111/mec.12521"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rellstab</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tedder</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zoller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gugerli</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">KK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holderegger</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Widmer</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Population genomic footprints of selection and associations with climate in natural populations of Arabidopsis halleri from the Alps</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">22</span>:<span class="cit-fpage">5594</span>–<span class="cit-lpage">5607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D22%26rft.spage%253D5594%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000329786700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.22" data-doi="10.1534/genetics.108.092221"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">O</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>:<span class="cit-fpage">977</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092221%26rft_id%253Dinfo%253Apmid%252F18780740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzIvOTc3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMTcvNjQyMzA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.23" data-doi="10.1111 /mec. 14584"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">HH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">DL</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Comparing methods for detecting multilocus adaptation with multivariate genotype–environment associations</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">27</span>:<span class="cit-fpage">2215</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D27%26rft.spage%253D2215%26rft_id%253Dinfo%253Adoi%252F10.1111%2B%252Fmec.%2B14584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111 /mec. 14584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frachon</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bartoli</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carrere</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bouchez</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chaubet</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roby</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">F</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">A Genomic Map of Climate Adaptation in Arabidopsis thaliana at a Micro-Geographic Scale</span>. <abbr class="cit-jnl-abbrev">Front Plant Sci</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BPlant%2BSci%26rft.volume%253D9%26rft.spage%253D967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.25" data-doi="10.1111/tpj.13027"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gehan</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomashow</span>  <span class="cit-name-given-names">MF</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Natural variation in the C-repeat binding factor cold response pathway correlates with local adaptation of Arabidopsis ecotypes</span>. <abbr class="cit-jnl-abbrev">Plant J</abbr> <span class="cit-vol">84</span>:<span class="cit-fpage">682</span>–<span class="cit-lpage">693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ%26rft.volume%253D84%26rft.spage%253D682%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13027%26rft_id%253Dinfo%253Apmid%252F26369909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26369909&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gienapp</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fior</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">VL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csilléry</span>  <span class="cit-name-given-names">K</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Genomic quantitative genetics to study evolution in the wild</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gillespie</span>  <span class="cit-name-given-names">JH</span></span>. <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Genetic drift in an infinite population. The pseudohitchhiking model</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>:<span class="cit-fpage">909</span>–<span class="cit-lpage">919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGillespie%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D909%26rft.epage%253D919%26rft.atitle%253DGenetic%2BDrift%2Bin%2Ban%2BInfinite%2BPopulation%253A%2BThe%2BPseudohitchhiking%2BModel%26rft_id%253Dinfo%253Apmid%252F10835409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTA5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMTcvNjQyMzA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.28" data-doi="10.1093/nar/gkr944"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goodstein</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neupane</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fazo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitros</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dirks</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hellsten</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Putnam</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Phytozome: a comparative platform for green plant genomics</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">40</span>:<span class="cit-fpage">D1178</span>–<span class="cit-lpage">1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr944%26rft_id%253Dinfo%253Apmid%252F22110026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22110026&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300176&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-642306v5.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Graur</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-source">Molecular and Genome Evolution</span>. <span class="cit-publ-loc">Sunderland, MA</span>: <span class="cit-publ-name">Sinauer</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.30" data-doi="10.1126/science.1209244"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarymowycz</span>  <span class="cit-name-given-names">LB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperone</span>  <span class="cit-name-given-names">FG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Adaptation to climate across the Arabidopsis thaliana genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>:<span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzE3LzY0MjMwNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.31" data-doi="10.1007/s00425-009-0958-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hassidim</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harir</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yakir</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kron</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">RM</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Over-expression of CONSTANS-LIKE 5 can induce flowering in short-day grown Arabidopsis</span>. <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">230</span>:<span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.volume%253D230%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-009-0958-7%26rft_id%253Dinfo%253Apmid%252F19504268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-009-0958-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19504268&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268393600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.32" data-doi="10.1038/ng.2684"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haudry</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Platts</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vello</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoen</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leclercq</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forczek</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joly-Lopez</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steffen</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hazzouri</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">45</span>:<span class="cit-fpage">891</span>–<span class="cit-lpage">898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D45%26rft.spage%253D891%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2684%26rft_id%253Dinfo%253Apmid%252F23817568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23817568&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.33" data-doi="10.1554/0014-3820(2002)056[1199:ADATBB]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hendry</span>  <span class="cit-name-given-names">AP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">McPhail</span>  <span class="cit-name-given-names">JD</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Adaptive divergence and the balance between selection and gene flow: lake and stream stickleback in the Misty system</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">56</span>:<span class="cit-fpage">1199</span>–<span class="cit-lpage">1216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHendry%26rft.auinit1%253DA.%2BP.%26rft.volume%253D56%26rft.issue%253D6%26rft.spage%253D1199%26rft.epage%253D1216%26rft.atitle%253DAdaptive%2Bdivergence%2Band%2Bthe%2Bbalance%2Bbetween%2Bselection%2Band%2Bgene%2Bflow%253A%2Blake%2Band%2Bstream%2Bstickleback%2Bin%2Bthe%2BMisty%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282002%2529056%255B1199%253AADATBB%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12144020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2002)056[1199:ADATBB]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12144020&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176820900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.34" data-doi="10.2217/pgs.12.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Henson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tischler</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Z</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Next-generation sequencing and large genome assemblies</span>. <abbr class="cit-jnl-abbrev">Pharmacogenomics</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">901</span>–<span class="cit-lpage">915</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacogenomics%26rft.stitle%253DPharmacogenomics%26rft.aulast%253DHenson%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D901%26rft.epage%253D915%26rft.atitle%253DNext-generation%2Bsequencing%2Band%2Blarge%2Bgenome%2Bassemblies.%26rft_id%253Dinfo%253Adoi%252F10.2217%252Fpgs.12.72%26rft_id%253Dinfo%253Apmid%252F22676195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2217/pgs.12.72&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22676195&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305380200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.35" data-doi="10.1086/597611"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hereford</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">A quantitative survey of local adaptation and fitness trade-offs</span>. <abbr class="cit-jnl-abbrev">Am Nat</abbr> <span class="cit-vol">173</span>:<span class="cit-fpage">579</span>–<span class="cit-lpage">588</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DHereford%26rft.auinit1%253DJ.%26rft.volume%253D173%26rft.issue%253D5%26rft.spage%253D579%26rft.epage%253D588%26rft.atitle%253DA%2Bquantitative%2Bsurvey%2Bof%2Blocal%2Badaptation%2Band%2Bfitness%2Btrade-offs.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F597611%26rft_id%253Dinfo%253Apmid%252F19272016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/597611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19272016&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264812800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.36" data-doi="10.1086/688018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoban</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antolin</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradburd</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poss</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">LK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Storfer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Finding the Genomic Basis of Local Adaptation: Pitfalls, Practical Solutions, and Future Directions</span>. <abbr class="cit-jnl-abbrev">Am Nat</abbr> <span class="cit-vol">188</span>:<span class="cit-fpage">379</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BNat%26rft.volume%253D188%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1086%252F688018%26rft_id%253Dinfo%253Apmid%252F27622873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/688018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27622873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.37" data-doi="10.1111/mec.13351"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeGiorgio</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Detecting recent selective sweeps while controlling for mutation rate and background selection</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">142</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D25%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13351%26rft_id%253Dinfo%253Apmid%252F26290347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26290347&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.38" data-doi="10.1093/molbev/msu247"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hermisson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Keeping it local: evidence for positive selection in Swedish Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">31</span>:<span class="cit-fpage">3026</span>–<span class="cit-lpage">3039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu247%26rft_id%253Dinfo%253Apmid%252F25158800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25158800&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.39" data-doi="10.1093/molbev/mst082"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hupalo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kern</span>  <span class="cit-name-given-names">AD</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Conservation and functional element discovery in 20 angiosperm plant genomes</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">30</span>:<span class="cit-fpage">1729</span>–<span class="cit-lpage">1744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst082%26rft_id%253Dinfo%253Apmid%252F23640124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23640124&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321056200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.40" data-doi="10.1534/genetics.115.185900"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">GS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sluckin</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kivisild</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Refining the Use of Linkage Disequilibrium as a Robust Signature of Selective Sweeps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>:<span class="cit-fpage">1807</span>–<span class="cit-lpage">1825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185900%26rft_id%253Dinfo%253Apmid%252F27516617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMy80LzE4MDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOC8xNy82NDIzMDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.41" data-doi="10.1016/j.gde.2014.06.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di</span>  <span class="cit-name-given-names">Rienzo A</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Adaptations to local environments in modern human populations</span>. <abbr class="cit-jnl-abbrev">Curr Opin Genet Dev</abbr> <span class="cit-vol">29</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.volume%253D29%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2014.06.011%26rft_id%253Dinfo%253Apmid%252F25129844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2014.06.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25129844&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.42" data-doi="10.1073/pnas.1706226114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">PIF3 is a negative regulator of the CBF pathway and freezing tolerance in Arabidopsis</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">114</span>:<span class="cit-fpage">E6695</span>–<span class="cit-lpage">E6702</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1706226114%26rft_id%253Dinfo%253Apmid%252F28739888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzMyL0U2Njk1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMTcvNjQyMzA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.43" data-doi="10.1038/nature10944"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">FC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">YF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mauceli</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swofford</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pirun</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zody</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">The genomic basis of adaptive evolution in threespine sticklebacks</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>:<span class="cit-fpage">55</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DF.%2BC.%26rft.volume%253D484%26rft.issue%253D7392%26rft.spage%253D55%26rft.epage%253D61%26rft.atitle%253DThe%2Bgenomic%2Bbasis%2Bof%2Badaptive%2Bevolution%2Bin%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10944%26rft_id%253Dinfo%253Apmid%252F22481358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22481358&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302343400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Joosen</span>  <span class="cit-name-given-names">RV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ligterink</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hilhorst</span>  <span class="cit-name-given-names">HW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keurentjes</span>  <span class="cit-name-given-names">JJ</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Advances in genetical genomics of plants</span>. <abbr class="cit-jnl-abbrev">Current genomics</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">540</span>–<span class="cit-lpage">549</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenomics%26rft.volume%253D10%26rft.spage%253D540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.45" data-doi="10.1073/pnas.1705884114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kalladan</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">TZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juenger</span>  <span class="cit-name-given-names">TE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verslues</span>  <span class="cit-name-given-names">PE</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Natural variation identifies genes affecting drought-induced abscisic acid accumulation in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">114</span>:<span class="cit-fpage">11536</span>–<span class="cit-lpage">11541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1705884114%26rft_id%253Dinfo%253Apmid%252F29073083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQzLzExNTM2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMTcvNjQyMzA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.46" data-doi="10.1038/ng.548"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">HM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sul</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Service</span>  <span class="cit-name-given-names">SK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">SY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freimer</span>  <span class="cit-name-given-names">NB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabatti</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Variance component model to account for sample structure in genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">42</span>:<span class="cit-fpage">348</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D348%26rft.epage%253D354%26rft.atitle%253DVariance%2Bcomponent%2Bmodel%2Bto%2Baccount%2Bfor%2Bsample%2Bstructure%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.548%26rft_id%253Dinfo%253Apmid%252F20208533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208533&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.47" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">HM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Efficient control of population structure in model organism association mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>:<span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOC8xNy82NDIzMDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.48" data-doi="10.1093/bib/bbn013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Recent developments in the MAFFT multiple sequence alignment program</span>. <abbr class="cit-jnl-abbrev">Brief Bioinform</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">286</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief%2BBioinform%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbn013%26rft_id%253Dinfo%253Apmid%252F18372315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbn013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18372315&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256756400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kaufmann</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lenz</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalbe</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milinski</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eizaguirre</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">A field reciprocal transplant experiment reveals asymmetric costs of migration between lake and river ecotypes of three-spined sticklebacks (Gasterosteus aculeatus)</span>. <abbr class="cit-jnl-abbrev">J Evol Biol</abbr> <span class="cit-vol">30</span>:<span class="cit-fpage">938</span>–<span class="cit-lpage">950</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BEvol%2BBiol%26rft.volume%253D30%26rft.spage%253D938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.50" data-doi="10.1111/j.1461-0248.2004.00684.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kawecki</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Conceptual issues in local adaptation</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">1225</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D7%26rft.spage%253D1225%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2004.00684.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2004.00684.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225078000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.51" data-doi="10.1093/molbev/mss121"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levsen</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">MS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Local adaptation in the flowering-time gene network of balsam poplar, Populus balsamifera L</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">29</span>:<span class="cit-fpage">3143</span>–<span class="cit-lpage">3152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss121%26rft_id%253Dinfo%253Apmid%252F22513286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22513286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309927900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.52" data-doi="10.1111/mec.13537"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kryvokhyzha</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornille</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glémin</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">SI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lagercrantz</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The influence of population structure on gene expression and flowering time variation in the ubiquitous weed Capsella bursa-pastoris (Brassicaceae)</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">1106</span>–<span class="cit-lpage">1121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D25%26rft.spage%253D1106%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.53" data-doi="10.1146/annurev-ecolsys-110512-135833"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lachance</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>  <span class="cit-name-given-names">SA</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Population Genomics of Human Adaptation</span>. <abbr class="cit-jnl-abbrev">Annu Rev Ecol Evol Syst</abbr> <span class="cit-vol">44</span>:<span class="cit-fpage">123</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BEcol%2BEvol%2BSyst%26rft.volume%253D44%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110512-135833%26rft_id%253Dinfo%253Apmid%252F25383060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110512-135833&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25383060&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.54" data-doi="10.1093/molbev/msu170"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Des Marais</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Povolotskaya</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keitt</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juenger</span>  <span class="cit-name-given-names">TE</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Natural variation in abiotic stress responsive gene expression and local adaptation to climate in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">31</span>:<span class="cit-fpage">2283</span>–<span class="cit-lpage">2296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu170%26rft_id%253Dinfo%253Apmid%252F24850899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24850899&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343401100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.55" data-doi="10.1111/j.1365-294X.2012.05709.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Des Marais</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juenger</span>  <span class="cit-name-given-names">TE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keitt</span>  <span class="cit-name-given-names">TH</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Characterizing genomic variation of Arabidopsis thaliana: the roles of geography and climate</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">21</span>:<span class="cit-fpage">5512</span>–<span class="cit-lpage">5529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLasky%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D21%26rft.issue%253D22%26rft.spage%253D5512%26rft.epage%253D5529%26rft.atitle%253DCharacterizing%2Bgenomic%2Bvariation%2Bof%2BArabidopsis%2Bthaliana%253A%2Bthe%2Broles%2Bof%2Bgeography%2Band%2Bclimate.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05709.x%26rft_id%253Dinfo%253Apmid%252F22857709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05709.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22857709&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310583800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reimherr</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Coherent synthesis of genomic associations with phenotypes and home environments</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Resour</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.57" data-doi="10.1371/journal.pone.0004010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leimu</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">A meta-analysis of local adaptation in plants</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">e4010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLeimu%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253De4010%26rft.epage%253De4010%26rft.atitle%253DA%2Bmeta-analysis%2Bof%2Blocal%2Badaptation%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0004010%26rft_id%253Dinfo%253Apmid%252F19104660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0004010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19104660&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.58" data-doi="10.1093/gbe/evx072"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">XC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">JC</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Cis-Regulatory Divergence in Gene Expression between Two Thermally Divergent Yeast Species</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr> <span class="cit-vol">9</span>:<span class="cit-fpage">1120</span>–<span class="cit-lpage">1129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%2BEvol%26rft.volume%253D9%26rft.spage%253D1120%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevx072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evx072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.59" data-doi="10.1038/ng.2678"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabanal</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farlow</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Platzer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nizhynska</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Massive genomic variation and strong selection in Arabidopsis thaliana lines from Sweden</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">45</span>:<span class="cit-fpage">884</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D45%26rft.spage%253D884%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2678%26rft_id%253Dinfo%253Apmid%252F23793030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23793030&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.60" data-doi="10.1111/mec.12725"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Evaluation of demographic history and neutral parameterization on the performance of FST outlier tests</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">23</span>:<span class="cit-fpage">2178</span>–<span class="cit-lpage">2192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D23%26rft.spage%253D2178%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12725%26rft_id%253Dinfo%253Apmid%252F24655127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24655127&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maddamsetti</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hatcher</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">RE</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Core Genes Evolve Rapidly in the Long-term Evolution Experiment with Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Latta</span>  <span class="cit-name-given-names">RG</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Adaptive population divergence: markers, QTL and traits</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">17</span>:<span class="cit-fpage">285</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D17%26rft.spage%253D285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.63" data-doi="10.1101/gr.6761107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hardison</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raney</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burhans</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baertsch</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blankenberg</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">28-way vertebrate alignment and conservation track in the UCSC Genome Browser</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">17</span>:<span class="cit-fpage">1797</span>–<span class="cit-lpage">1808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6761107%26rft_id%253Dinfo%253Apmid%252F17984227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8xNzk3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMTcvNjQyMzA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Monroe</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGovern</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grogan</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">JK</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Adaptation to warmer climates by parallel functional evolution of CBF genes in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">25</span>:<span class="cit-fpage">3632</span>–<span class="cit-lpage">3644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D25%26rft.spage%253D3632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.65" data-doi="10.7554/eLife.41038"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Monroe</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mullen</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lovell</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">JK</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Drought adaptation in Arabidopsis thaliana by extensive genetic loss-of-function</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">e41038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D7%26rft.spage%253De41038%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.41038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.41038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.66" data-doi="10.1073/pnas.0407107101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Monte</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tepperman</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Al-Sady</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaczorowski</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alonso</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">PH</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">The phytochrome-interacting transcription factor, PIF3, acts early, selectively, and positively in light-induced chloroplast development</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">101</span>:<span class="cit-fpage">16091</span>–<span class="cit-lpage">16098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0407107101%26rft_id%253Dinfo%253Apmid%252F15505214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQ2LzE2MDkxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMTcvNjQyMzA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.67" data-doi="10.1073/pnas.0431157100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">D’Agostino</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">The genetic basis of adaptive melanism in pocket mice</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">100</span>:<span class="cit-fpage">5268</span>–<span class="cit-lpage">5273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0431157100%26rft_id%253Dinfo%253Apmid%252F12704245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzkvNTI2OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzE3LzY0MjMwNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.68" data-doi="10.1111/mec.12862"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oakley</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agren</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atchison</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">DW</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">QTL mapping of freezing tolerance: links to fitness and adaptive trade-offs</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">23</span>:<span class="cit-fpage">4304</span>–<span class="cit-lpage">4315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D23%26rft.spage%253D4304%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.69" data-doi="10.1093/jxb/erx437"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oakley</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savage</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lotz</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">GR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomashow</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">DW</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Genetic basis of photosynthetic responses to cold in two locally adapted populations of Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">J Exp Bot</abbr> <span class="cit-vol">69</span>:<span class="cit-fpage">699</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BBot%26rft.volume%253D69%26rft.spage%253D699%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferx437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erx437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.70" data-doi="10.1016/S0014-5793(03)01470-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oda</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fujiwara</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kamada</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mizoguchi</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Antisense suppression of the Arabidopsis PIF3 gene does not affect circadian rhythms but causes early flowering and increases FT expression</span>. <abbr class="cit-jnl-abbrev">FEBS Lett</abbr> <span class="cit-vol">557</span>:<span class="cit-fpage">259</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DOda%26rft.auinit1%253DA.%26rft.volume%253D557%26rft.issue%253D1-3%26rft.spage%253D259%26rft.epage%253D264%26rft.atitle%253DAntisense%2Bsuppression%2Bof%2Bthe%2BArabidopsis%2BPIF3%2Bgene%2Bdoes%2Bnot%2Baffect%2Bcircadian%2Brhythms%2Bbut%2Bcauses%2Bearly%2Bflowering%2Band%2Bincreases%2BFT%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252803%252901470-4%26rft_id%253Dinfo%253Apmid%252F14741378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(03)01470-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14741378&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188648100047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pass</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sornay</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marchbank</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paszkiewicz</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">JAH</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Genome-wide chromatin mapping with size resolution reveals a dynamic sub-nucleosomal landscape in Arabidopsis</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">13</span>:<span class="cit-fpage">e1006988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D13%26rft.spage%253De1006988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phifer-Rixey</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">KG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sheehan</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mack</span>  <span class="cit-name-given-names">KL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keeble</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">MW</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">The genomic basis of environmental adaptation in house mice</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">e1007672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D14%26rft.spage%253De1007672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pisupati</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reichardt</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seren</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nizhynska</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kerdaffrec</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uzunova</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabanal</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Filiault</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Verification of Arabidopsis stock collections using SNPmatch, a tool for genotyping high-plexed samples</span>. <abbr class="cit-jnl-abbrev">Sci Data</abbr> <span class="cit-vol">4</span>:<span class="cit-fpage">170184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BData%26rft.volume%253D4%26rft.spage%253D170184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.74" data-doi="10.1534/genetics.110.121665"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Conditions under which genome-wide association studies will be positively misleading</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>:<span class="cit-fpage">1045</span>–<span class="cit-lpage">1052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121665%26rft_id%253Dinfo%253Apmid%252F20813880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni8zLzEwNDUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOC8xNy82NDIzMDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pluess</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heiri</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lalagüe</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vendramin</span>  <span class="cit-name-given-names">GG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oddou-Muratorio</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Genome– environment association study suggests local adaptation to climate at the regional scale in Fagus sylvatica</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">210</span>:<span class="cit-fpage">589</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D210%26rft.spage%253D589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.76" data-doi="10.1038/nrg2813"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">NA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">New approaches to population stratification in genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">459</span>–<span class="cit-lpage">463</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPrice%26rft.auinit1%253DA.%2BL.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D459%26rft.epage%253D463%26rft.atitle%253DNew%2Bapproaches%2Bto%2Bpopulation%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2813%26rft_id%253Dinfo%253Apmid%252F20548291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20548291&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278998500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.77" data-doi="10.1073/pnas.1719998115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moyers</span>  <span class="cit-name-given-names">BT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monroe</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mullen</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oakley</span>  <span class="cit-name-given-names">CG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agren</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Combining population genomics and fitness QTLs to identify the genetics of local adaptation in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">115</span>:<span class="cit-fpage">5028</span>–<span class="cit-lpage">5033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1719998115%26rft_id%253Dinfo%253Apmid%252F29686078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzE5LzUwMjgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOC8xNy82NDIzMDYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.78" data-doi="10.1086/519795"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maller</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sklar</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Bakker</span>  <span class="cit-name-given-names">PI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <abbr class="cit-jnl-abbrev">Am J Hum Genet</abbr> <span class="cit-vol">81</span>:<span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rellstab</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zoller</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graf</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tedder</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">KK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Widmer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holderegger</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gugerli</span>  <span class="cit-name-given-names">F</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Local adaptation (mostly) remains local: reassessing environmental associations of climate-related candidate SNPs in Arabidopsis halleri</span>. <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">118</span>:<span class="cit-fpage">193</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D118%26rft.spage%253D193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ristova</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Metesch</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Busch</span>  <span class="cit-name-given-names">W</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Natural Genetic Variation Shapes Root System Responses to Phytohormones in Arabidopsis</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: <span class="cit-fpage">155184</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.81" data-doi="10.1126/science.aat7244"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sackton</span>  <span class="cit-name-given-names">TB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grayson</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cloutier</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">NE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">PP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clamp</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Convergent regulatory evolution and loss of flight in paleognathous birds</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">364</span>:<span class="cit-fpage">74</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSackton%26rft.auinit1%253DT.%2BB.%26rft.volume%253D364%26rft.issue%253D6435%26rft.spage%253D74%26rft.epage%253D78%26rft.atitle%253DConvergent%2Bregulatory%2Bevolution%2Band%2Bloss%2Bof%2Bflight%2Bin%2Bpaleognathous%2Bbirds%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aat7244%26rft_id%253Dinfo%253Apmid%252F30948549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNjQvNjQzNS83NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzE3LzY0MjMwNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.82" data-doi="10.1371/journal.pgen.1005597"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">“Missing” G × E Variation Controls Flowering Time in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">11</span>:<span class="cit-fpage">e1005597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D11%26rft.spage%253De1005597%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005597%26rft_id%253Dinfo%253Apmid%252F26473359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005597&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26473359&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.83" data-doi="10.1038/nrg3522"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merila</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Ecological genomics of local adaptation</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">14</span>:<span class="cit-fpage">807</span>–<span class="cit-lpage">820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D14%26rft.spage%253D807%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3522%26rft_id%253Dinfo%253Apmid%252F24136507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24136507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.84" data-doi="10.1146/annurev.ecolsys.38.091206.095646"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knürr</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Gene Flow and Local Adaptation in Trees</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">38</span>:<span class="cit-fpage">595</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D38%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.38.091206.095646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.38.091206.095646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252359800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arbiza</span>  <span class="cit-name-given-names">L</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Cis-regulatory elements and human evolution</span>. <abbr class="cit-jnl-abbrev">Curr Opin Genet Dev</abbr> <span class="cit-vol">29</span>:<span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.volume%253D29%26rft.spage%253D81%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.86" data-doi="10.1101/gr.3715005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bejerano</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hinrichs</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clawson</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spieth</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hillier</span>  <span class="cit-name-given-names">LW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">15</span>:<span class="cit-fpage">1034</span>–<span class="cit-lpage">1050</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3715005%26rft_id%253Dinfo%253Apmid%252F16024819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvMTAzNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzE3LzY0MjMwNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.87"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">VL</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Genomic Studies of Local Adaptation in Natural Plant Populations</span>. <abbr class="cit-jnl-abbrev">J Hered</abbr> <span class="cit-vol">109</span>:<span class="cit-fpage">3</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BHered%26rft.volume%253D109%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soy</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leivar</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez-Schain</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sentandreu</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prat</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quail</span>  <span class="cit-name-given-names">PH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monte</span>  <span class="cit-name-given-names">E</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Phytochrome-imposed oscillations in PIF3 protein abundance regulate hypocotyl growth under diurnal light/dark conditions in Arabidopsis</span>. <abbr class="cit-jnl-abbrev">Plant J</abbr> <span class="cit-vol">71</span>:<span class="cit-fpage">390</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DSoy%26rft.auinit1%253DJ.%26rft.volume%253D71%26rft.issue%253D3%26rft.spage%253D390%26rft.epage%253D401%26rft.atitle%253DPhytochrome-imposed%2Boscillations%2Bin%2BPIF3%2Bprotein%2Babundance%2Bregulate%2Bhypocotyl%2Bgrowth%2Bunder%2Bdiurnal%2Blight%252Fdark%2Bconditions%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Apmid%252F22409654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22409654&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306893500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.89" data-doi="10.2307/1422689"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spearman</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-pub-date">1987</span>. <span class="cit-article-title">The Proof and Measurement of Association between Two Things</span>. <abbr class="cit-jnl-abbrev">The American Journal of Psychology</abbr> <span class="cit-vol">100</span>:<span class="cit-fpage">441</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2Bjournal%2Bof%2Bpsychology%26rft.stitle%253DAm%2BJ%2BPsychol%26rft.aulast%253DSpearman%26rft.auinit1%253DC.%26rft.volume%253D100%26rft.issue%253D3-4%26rft.spage%253D441%26rft.epage%253D471%26rft.atitle%253DThe%2Bproof%2Band%2Bmeasurement%2Bof%2Bassociation%2Bbetween%2Btwo%2Bthings.%2BBy%2BC.%2BSpearman%252C%2B1904.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1422689%26rft_id%253Dinfo%253Apmid%252F3322052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1422689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3322052&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K868500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.90" data-doi="10.1371/journal.pgen.1005869"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Svetec</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cridland</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Begun</span>  <span class="cit-name-given-names">DJ</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The Adaptive Significance of Natural Genetic Variation in the DNA Damage Response of Drosophila melanogaster</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">e1005869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D12%26rft.spage%253De1005869%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005869&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.91" data-doi="10.1093/molbev/mst197"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">MEGA6: Molecular Evolutionary Genetics Analysis version 6.0</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">30</span>:<span class="cit-fpage">2725</span>–<span class="cit-lpage">2729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst197%26rft_id%253Dinfo%253Apmid%252F24132122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24132122&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327793000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-book" id="cit-642306v5.92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Team</span>  <span class="cit-name-given-names">RDC</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-chapter-title">R: A Language and Environment for Statistical Computing</span>. <span class="cit-source">R Foundation for Statistical Computing</span>. <span class="cit-publ-loc">Vienna, Austria</span>: <span class="cit-publ-name">R Foundation for Statistical Computing</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.93" data-doi="10.1016/j.tree.2014.10.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Advances and limits of using population genetics to understand local adaptation</span>. <abbr class="cit-jnl-abbrev">Trends Ecol Evol</abbr> <span class="cit-vol">29</span>:<span class="cit-fpage">673</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol%2BEvol%26rft.volume%253D29%26rft.spage%253D673%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2014.10.004%26rft_id%253Dinfo%253Apmid%252F25454508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2014.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25454508&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.94" data-doi="10.4161/psb.2.3.4156"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tuteja</span>  <span class="cit-name-given-names">N</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Abscisic Acid and abiotic stress signaling</span>. <abbr class="cit-jnl-abbrev">Plant Signal Behav</abbr> <span class="cit-vol">2</span>:<span class="cit-fpage">135</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BSignal%2BBehav%26rft.aulast%253DTuteja%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253D135%26rft.epage%253D138%26rft.atitle%253DAbscisic%2BAcid%2Band%2Babiotic%2Bstress%2Bsignaling.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fpsb.2.3.4156%26rft_id%253Dinfo%253Apmid%252F19516981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/psb.2.3.4156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19516981&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.95" data-doi="10.2307/2409692"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">1991</span>. <span class="cit-article-title">The Genetic Structure of Host Plant Adaptation in a Spatial Patchwork: Demographic Variability among Reciprocally Transplanted Pea Aphid Clones</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">45</span>:<span class="cit-fpage">827</span>–<span class="cit-lpage">852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D45%26rft.spage%253D827%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991FW83600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.96"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wadgymar</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">BA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byron</span>  <span class="cit-name-given-names">CN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mactavish</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">JT</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Identifying targets and agents of selection: innovative methods to evaluate the processes that contribute to local adaptation</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">8</span>:<span class="cit-fpage">738</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D8%26rft.spage%253D738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.97" data-doi="10.1093/bioinformatics/btq728"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolfinger</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grayson</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aune</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">An efficient hierarchical generalized linear mixed model for pathway analysis of genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>:<span class="cit-fpage">686</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq728%26rft_id%253Dinfo%253Apmid%252F21266443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq728&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21266443&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287758400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.98"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shikano</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merila</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Quantitative trait locus analysis of body shape divergence in nine-spined sticklebacks based on high-density SNP-panel</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">26632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D26632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.99" data-doi="10.1126/science.aaf7812"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodgins</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suren</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nadeau</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Degner</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nurkowski</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smets</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">LK</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Convergent local adaptation to climate in distantly related conifers</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">353</span>:<span class="cit-fpage">1431</span>–<span class="cit-lpage">1433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYeaman%26rft.auinit1%253DS.%26rft.volume%253D353%26rft.issue%253D6306%26rft.spage%253D1431%26rft.epage%253D1433%26rft.atitle%253DConvergent%2Blocal%2Badaptation%2Bto%2Bclimate%2Bin%2Bdistantly%2Brelated%2Bconifers%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf7812%26rft_id%253Dinfo%253Apmid%252F27708038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTMvNjMwNi8xNDMxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMTcvNjQyMzA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.100" data-doi="10.1111/j.1558-5646.2011.01269.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">The genetic architecture of adaptation under migration-selection balance</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>:<span class="cit-fpage">1897</span>–<span class="cit-lpage">1911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DYeaman%26rft.auinit1%253DS.%26rft.volume%253D65%26rft.issue%253D7%26rft.spage%253D1897%26rft.epage%253D1911%26rft.atitle%253DThe%2Bgenetic%2Barchitecture%2Bof%2Badaptation%2Bunder%2Bmigration-selection%2Bbalance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01269.x%26rft_id%253Dinfo%253Apmid%252F21729046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01269.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729046&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292447000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.101" data-doi="10.1038/ng1702"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">WH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vroh Bi</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">38</span>:<span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.102" data-doi="10.1038/ng.2310"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Genome-wide efficient mixed-model analysis for association studies</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">44</span>:<span class="cit-fpage">821</span>–<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D821%26rft.epage%253D824%26rft.atitle%253DGenome-wide%2Befficient%2Bmixed-model%2Banalysis%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2310%26rft_id%253Dinfo%253Apmid%252F22706312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706312&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F17%2F642306.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-642306v5.103" data-doi="10.1073/pnas.1103202108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackaluso</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seddon</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomashow</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shiu</span>  <span class="cit-name-given-names">SH</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Cis-regulatory code of stress-responsive transcription in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">108</span>:<span class="cit-fpage">14992</span>–<span class="cit-lpage">14997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1103202108%26rft_id%253Dinfo%253Apmid%252F21849619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM2LzE0OTkyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMTcvNjQyMzA2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/642306v5.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F642306v5" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/364778v5" title="Mutant lamins cause nuclear envelope rupture and DNA damage in skeletal muscle cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;August 17, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/642306v5.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/642306v5.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/642306v5.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/642306v5" class="active" data-icon-position="" data-hide-link-title="0">In the presence of population structure: From genomics to candidate genes underlying local adaptation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/872420" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8IU00KfheAlPKnUEO_Cd8sC410RnjfU5WwaIspdfO5U" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="872420" id="node872420--2" data-pisa="biorxiv;642306v5" data-pisa-master="biorxiv;642306" data-seqnum="872420" data-apath="/biorxiv/early/2019/08/17/642306.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">In the presence of population structure: From genomics to candidate genes underlying local adaptation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Nicholas</span> <span class="nlm-surname">Price</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Lua</span> <span class="nlm-surname">Lopez</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Adrian E.</span> <span class="nlm-surname">Platts</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jesse R.</span> <span class="nlm-surname">Lasky</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">John K.</span> <span class="nlm-surname">McKay</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">642306; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/642306 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/642306v5" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/642306v5&amp;title=In%20the%20presence%20of%20population%20structure%3A%20From%20genomics%20to%20candidate%20genes%20underlying%20local%20adaptation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/642306v5&amp;title=In%20the%20presence%20of%20population%20structure%3A%20From%20genomics%20to%20candidate%20genes%20underlying%20local%20adaptation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/642306v5&amp;text=In%20the%20presence%20of%20population%20structure%3A%20From%20genomics%20to%20candidate%20genes%20underlying%20local%20adaptation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/642306v5&amp;title=In%20the%20presence%20of%20population%20structure%3A%20From%20genomics%20to%20candidate%20genes%20underlying%20local%20adaptation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/642306v5&amp;t=In%20the%20presence%20of%20population%20structure%3A%20From%20genomics%20to%20candidate%20genes%20underlying%20local%20adaptation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/642306v5&amp;title=In%20the%20presence%20of%20population%20structure%3A%20From%20genomics%20to%20candidate%20genes%20underlying%20local%20adaptation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/642306v5&amp;title=In%20the%20presence%20of%20population%20structure%3A%20From%20genomics%20to%20candidate%20genes%20underlying%20local%20adaptation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="872420" id="node872420--3" data-pisa="biorxiv;642306v5" data-pisa-master="biorxiv;642306" data-seqnum="872420" data-apath="/biorxiv/early/2019/08/17/642306.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">In the presence of population structure: From genomics to candidate genes underlying local adaptation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Nicholas</span> <span class="nlm-surname">Price</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Lua</span> <span class="nlm-surname">Lopez</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Adrian E.</span> <span class="nlm-surname">Platts</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jesse R.</span> <span class="nlm-surname">Lasky</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">John K.</span> <span class="nlm-surname">McKay</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">642306; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/642306 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=In%20the%20presence%20of%20population%20structure%3A%20From%20genomics%20to%20candidate%20genes%20underlying%20local%20adaptation&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Price%2C+Nicholas&amp;rft.au=Lopez%2C+Lua&amp;rft.au=Platts%2C+Adrian+E.&amp;rft.au=Lasky%2C+Jesse+R.&amp;rft.au=McKay%2C+John+K."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/872420/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/872420/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/872420/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/872420/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/872420/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/872420/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/872420/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/872420/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/872420/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/872420/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/872420/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/872420/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/642306v5&amp;count=horizontal&amp;via=&amp;text=In%20the%20presence%20of%20population%20structure%3A%20From%20genomics%20to%20candidate%20genes%20underlying%20local%20adaptation&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/642306v5" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/642306v5&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/642306v5" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
