<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2015/02/23/015495.atom" />
<meta name="HW.pisa" content="biorxiv;015495v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Diversity and Climate Adaptation in Brachypodium" />
<meta name="DC.Identifier" content="10.1101/015495" />
<meta name="DC.Date" content="2015-02-23" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2015, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The Brachypodium genus contains the model grasses B. distachyon , B. stacei and B. hybridum , that are useful for molecular and physiological studies relevant to grain, pasture and bioenergy crops, as well as ecology. In this chapter we discuss the natural variation in climate/geography, genotypic and phenotypic diversity that exists within these species. We describe utilisation of this diversity via two methods, Genome Wide Association Studies and Landscape Genomics, to examine the interaction between specific genetic variants, phenotype, and environment. The aim is to identify adaptive loci that control specific traits in specific environments and understand the contribution of background polygenetic variation shaped by demographic processes. With recent developments in high throughput phenotyping, cheaper genotyping by sequencing and higher spatial/temporal resolution of climate data, these approaches can exploit the diversity of the Brachypodium. Experiments using this toolkit will reveal alleles, genes and pathways underlying agriculturally important and environmentally sensitive traits for use in grass breeding.

*   GBS
    :   genotyping by sequencing
    GWAS
    :   genome wide association studies
    QTL
    :   quantitative trait loci
    MaxEnt
    :   Maximum Entropy
    SNPs
    :   single nucleotide polymorphisms

*   Accession
    :   A collection of seeds from one location. This includes bulk collections and maternal descent lines
    Ecotype
    :   An individual or group whose genetic distinction is strongly associated to an environment or type
    Genotype
    :   This general term is used either to describe the genotype at a locus such as a SNP (AA, Aa, aa) or a background whole genome genotype which can have levels of species, subspecies, population genetic structure group, family, individual maternal line)
    Phenotype (qualitative and quantitative)
    :   Measurable traits expressed by plants
    Population
    :   Non-random mating between groups within a specified geographic space
    Subspecies
    :   In this paper, subspecies is a major hierarchical cluster of genotype groups and their respective families and/or genotypes. Subspecies could interbreed but don’t in natural environment due to some sort of natural barrier" />
<meta name="DC.Contributor" content="Pip Wilson" />
<meta name="DC.Contributor" content="Jared Streich" />
<meta name="DC.Contributor" content="Justin Borevitz" />
<meta name="article:published_time" content="2015-02-23" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Genomic Diversity and Climate Adaptation in Brachypodium" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;The Brachypodium genus contains the model grasses &lt;i&gt;B. distachyon&lt;/i&gt;, &lt;i&gt;B. stacei&lt;/i&gt; and &lt;i&gt;B. hybridum&lt;/i&gt;, that are useful for molecular and physiological studies relevant to grain, pasture and bioenergy crops, as well as ecology. In this chapter we discuss the natural variation in climate/geography, genotypic and phenotypic diversity that exists within these species. We describe utilisation of this diversity via two methods, Genome Wide Association Studies and Landscape Genomics, to examine the interaction between specific genetic variants, phenotype, and environment. The aim is to identify adaptive loci that control specific traits in specific environments and understand the contribution of background polygenetic variation shaped by demographic processes. With recent developments in high throughput phenotyping, cheaper genotyping by sequencing and higher spatial/temporal resolution of climate data, these approaches can exploit the diversity of the Brachypodium. Experiments using this toolkit will reveal alleles, genes and pathways underlying agriculturally important and environmentally sensitive traits for use in grass breeding.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2015/01/01" />
<meta name="citation_mjid" content="biorxiv;015495v1" />
<meta name="citation_id" content="015495v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/015495v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/015495v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/015495v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495.full.pdf" />
<meta name="citation_doi" content="10.1101/015495" />
<meta name="citation_num_pages" content="23" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="015495" />
<meta name="citation_author" content="Pip Wilson" />
<meta name="citation_author_institution" content="Centre of Excellence in Plant Energy Biology" />
<meta name="citation_author_email" content="Justin.Borevitz@anu.edu.au" />
<meta name="citation_author" content="Jared Streich" />
<meta name="citation_author_institution" content="Centre of Excellence in Plant Energy Biology" />
<meta name="citation_author_email" content="Justin.Borevitz@anu.edu.au" />
<meta name="citation_author" content="Justin Borevitz" />
<meta name="citation_author_institution" content="Centre of Excellence in Plant Energy Biology" />
<meta name="citation_author_email" content="Justin.Borevitz@anu.edu.au" />
<meta name="citation_reference" content="ALA [Internet] (Atlas of Living Australia) ala.org.au, [Cited 27Nov2014] Available from: http://avh.ala.org.au/occurrences/search?taxa=Brachypodium+distachyon" />
<meta name="citation_reference" content="Atwell S, Huang YS, Villhjálmsson BJ, Willems G, Horton M, Li Y, et al. Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines. Nature letters 2010;465:627-31" />
<meta name="citation_reference" content="Banta JA, Ehrenreich IM, Gerard S, Chou L, Wilczek A, Schmitt J, et al. Climate envelope modeling reveals intraspecific relationships among phenology, niche breadth and potential range size in Arabidopsis thaliana. Ecology Letters. 2012;15:769-77" />
<meta name="citation_reference" content="Barrero JM, Jacobsen JV, Talbot MJ, White RG, Swain SM, Garvin DF, et al. Grain dormancy and light quality effects on germination in the model grass Brachypodium distachyon. New Phytologist 2012;193:376-86" />
<meta name="citation_reference" content="Baxter I, Brazelton JN, Yu D, Huang YS, Lahner B, Yukubova E, et al. A coastal cline in sodium accumulation in Arabidopsis thaliana is driven by natural variation of the sodium transporter AtHKT1;1. PlosGenetics 2010;DOI:10.1371/journal.pgen.1001193" />
<meta name="citation_reference" content="Betekhtin A, Jenkins G, Hasterok R. Reconstructing the Evolution of Brachypodium Genomes Using Comparative Chromosome Painting. PLoS One. 2014;DOI:10.1371/journal.pone.0115108" />
<meta name="citation_reference" content="BioClim [Internet]. WorldClim - Global Climate Data, free climate data for ecology. [Cited 5Jan2015] Available from: http://www.bioclim.org/" />
<meta name="citation_reference" content="Brachi B, Morris GP, Borevitz JO. Genome-wide association studies in plants: the missing heritability in in the field. Genome Biology. 2011;12:232" />
<meta name="citation_reference" content="Brown TB, Cheng R, Sirault XRR, Rungrat T, Murray KD, Trtilek M, et al. TraitCapture: genomic and environmental modelling of plant phenomic data. Current Opinion in Plant Biology. 2014;18:73-9" />
<meta name="citation_reference" content="citation_title=Flowering Plant Families of the World;citation_pages=167;citation_year=2007" />
<meta name="citation_reference" content="Clausen J, Keck DD, Hiesey WM. Experimental studies on the nature of species. Washington D.C.:Carnegie Institution of Washington; 1940." />
<meta name="citation_reference" content="Catalán P, Müller J, Hasterok R, Jenkins G, Mur LAJ, Langdon T, et al. Evolution and taxonomic split of the model grass Brachypodium distachyon. Ann Bot. 2012;109(2):385-405" />
<meta name="citation_reference" content="Elshire RJ, Glaubitz JC, Sun Q, Poland JA, Kawamoto K, Buckler ES, et al. A robust simple genotyping-by-sequencing (GBS) approach for high diversity species. PLoS One. 2011;doi: 10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="Fournier-Level A, Korte A, Cooper MD, Nordborg M, Schmitt J, Wilczek AM. A map of local adaptation in Arabidopsis thaliana. Science. 2011 Oct 7;334(6052):86-9." />
<meta name="citation_reference" content="Garrick RC, Nason JD, Fernández-Manjarrés JF, Dyer RJ, Ecological co-associations influence species’ responses to past climatic change: an example from a Sonoran Desert bark beetle. Molecular ecology. 2013;22(12):3345-61" />
<meta name="citation_reference" content="Garvin DF, Gu YQ, Hasterok R, Hazen SP, Jenkins G, Mockler TC, et al. Development of genetic and genomic research resources for Brachypodium distachyon, a new model system for grass crop research. Crop Science. 2008;48:S69–S84" />
<meta name="citation_reference" content="Grime JP. Evidence for the existence of three primary strategies in plants and its relevance to ecological and evolutionary theory. The American Naturalist. 1977;111(982):1169-94" />
<meta name="citation_reference" content="Groombridge B, Jenkins M. World Atlas of Biodiversity: Earth’s Living Resources In the 21st Century. Berkeley, CA:University of California Press; 2002" />
<meta name="citation_reference" content="Halperin E, Stephan DA. SNP imputation in association studies. Nature Biotechnology. 2009;27:349-51" />
<meta name="citation_reference" content="Hancock AM, Brachi B, Faure N, Horton MW, Jarymowycz LB, Sperone FG, et al. Adaptation to climate across the Arabidopsis thaliana genome. Science. 2011;334(6052):83-6." />
<meta name="citation_reference" content="Hartley W. Studies on the origin, evolution and distribution of the Gramineae. V. The subfamily Festucoideae. Australian Journal of Botany. 1973;21:201-34" />
<meta name="citation_reference" content="Hoffmann A, Griffin P, Dillon S, Catullo R, Rane R, Byrne M, et al. A framework for incorporating evolutionary genomics into biodiversity conservation and management. Climate Change Responses 2015;2:1 doi:10.1186/s40665-014-0009-x" />
<meta name="citation_reference" content="Horton MW, Hancock AM, Huang YS, Toomajian C, Atwell S, Auton A, et al. Genome-wide patterns of genetic variation in worldwide Arabidopsis thaliana accessions from the RegMap panel. Nature Genetics. 2012;44(2):212-7" />
<meta name="citation_reference" content="Huang X, Han B. Natural variations and genome-wide association studies in crop plants. Annual Review in Plant Biology. 2014;65:531-51" />
<meta name="citation_reference" content="Huang X, Wei X, Sang T, Zhao T, Zhao Q, Feng Q, et al. Genome-wide association studies of 14 agronomic traits in rice landraces. Nature Genetics. 2010;42:961-7" />
<meta name="citation_reference" content="Joost S, Bonin A, Bruford M W, Després L, Conord C, Erhardt G et al. A spatial analysis method (SAM) to detect candidate loci for selection: towards a landscape genomics approach to adaptation. Molecular Ecology. 2007;16:3955–69 doi: 10.1111/j.1365-294X.2007.03442.x" />
<meta name="citation_reference" content="Klein RJ, Zeiss C, Chew EY, Tsai JY, Sackler RS, Haynes C, et al. Complement Factor H polymorphism in age-related macular degeneration. Science. 2005;308(5720):385–9" />
<meta name="citation_reference" content="Li Y, Roycewicz P, Smith E, Borevitz JO. Genetics of local adaptation in the laboratory: flowering time quantitative trait loci under geographic and seasonal conditions in Arabidopsis. PLoS One. 2006;1(1),e105" />
<meta name="citation_reference" content="Li Y, Huang Y, Bergelson J, Nordborg M, Borevitz JO. Association mapping of local climate-sensitive quantitative trait loci in Arabidopsis thaliana. PNAS. 2010;107(49): 21199-204" />
<meta name="citation_reference" content="Li Y, Cheng R, Spokas KA, Genetic variation for life history sensitivity to seasonal warming in Arabidopsis thaliana. Genetics. 2014;196:569-77" />
<meta name="citation_reference" content="Morris GP, Ramu P, Deshpande SP, Hash CT, Shah T, Upadhyaya HD, et al. Population genomics and genome-wide association studies of agroclimatic traits in sorghum. PNAS. 2013;110(2):453-8" />
<meta name="citation_reference" content="Pasam RK, Sharma R, Malosetti M, van Eeuwijk FA, Haseneyer G, Kilian B, et al. Genome-wide association studies for agronomical traits in a world wide spring barley collection. BMC Plant Biology. 2012;12:16" />
<meta name="citation_reference" content="Peel MC, Finlayson BL, McMahon TA. Updated world map of the Köppen-Geiger climate classification, Hydrol. Earth Syst. Sci., 2007;11:1633-44" />
<meta name="citation_reference" content="Phillips SJ, Dudík M. Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation. Ecography. 2008;31:161–175" />
<meta name="citation_reference" content="Phillips SJ, Anderson RP, Schapire RE. Maximum entropy modeling of species geographic distributions. Ecological Modelling. 2006;190(3-4):231-59" />
<meta name="citation_reference" content="Platt A, Horton M, Huang YS, Li Y, Anastasio AE, Mulyati NW, et al. The Scale of Population Structure in Arabidopsis thaliana. PLoS Genetics. 2010;6(2):e1000843" />
<meta name="citation_reference" content="Revedin A, Aranguren B, Becattini R, Longo L, Marconi E, Lippi MM, et al. Thirty thousand-year-old evidence of plant food processing. PNAS. 2010;107(44):18815-9" />
<meta name="citation_reference" content="Schwartz CJ, Doyle MR, Manzaneda AJ, Rey PJ, Mitchell-Olds T, Amasino RM. Natural Variation of Flowering Time and Vernalization Responsiveness in Brachypodium distachyon. Bioenerg. Res. 2010;3:38-46" />
<meta name="citation_reference" content="Sharbel T, Haubold B, Mitchell-Olds T. Genetic isolation by distance in Arabidopsis thaliana: biogeography and postglacial colonization of Europe. Molecular Ecology. 2000;9:2109-18" />
<meta name="citation_reference" content="Storz JF. Using genome scans of DNA polymorphism to infer adaptive population divergence. Molecular Ecology, 2005;14:671–88." />
<meta name="citation_reference" content="Vogel JP, Tuna M, Budak H, Huo N, Gu YQ, Steinwand MA. Development of SSR markers and analysis of diversity in Turkish populations of Brachypodium distachyon. BMC Plant Biology. 2009;9:88" />
<meta name="citation_reference" content="Wen W, Li D, Li W, Gao Y, Li W, Li H, et al. Metabolome-based genome-wide association study of maize kernel leads to novel biochemical insights. Nature communications. 2014;5(3438):1-10" />
<meta name="citation_reference" content="WorldClim [Internet], WorldClim - Global Climate Data, free climate data for ecology, [cited 5Jan2014] Available from: http://www.worldclim.org/" />
<meta name="twitter:title" content="Genomic Diversity and Climate Adaptation in Brachypodium" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The Brachypodium genus contains the model grasses B. distachyon , B. stacei and B. hybridum , that are useful for molecular and physiological studies relevant to grain, pasture and bioenergy crops, as well as ecology. In this chapter we discuss the natural variation in climate/geography, genotypic and phenotypic diversity that exists within these species. We describe utilisation of this diversity via two methods, Genome Wide Association Studies and Landscape Genomics, to examine the interaction between specific genetic variants, phenotype, and environment. The aim is to identify adaptive loci that control specific traits in specific environments and understand the contribution of background polygenetic variation shaped by demographic processes. With recent developments in high throughput phenotyping, cheaper genotyping by sequencing and higher spatial/temporal resolution of climate data, these approaches can exploit the diversity of the Brachypodium. Experiments using this toolkit will reveal alleles, genes and pathways underlying agriculturally important and environmentally sensitive traits for use in grass breeding.

*   GBS
    :   genotyping by sequencing
    GWAS
    :   genome wide association studies
    QTL
    :   quantitative trait loci
    MaxEnt
    :   Maximum Entropy
    SNPs
    :   single nucleotide polymorphisms

*   Accession
    :   A collection of seeds from one location. This includes bulk collections and maternal descent lines
    Ecotype
    :   An individual or group whose genetic distinction is strongly associated to an environment or type
    Genotype
    :   This general term is used either to describe the genotype at a locus such as a SNP (AA, Aa, aa) or a background whole genome genotype which can have levels of species, subspecies, population genetic structure group, family, individual maternal line)
    Phenotype (qualitative and quantitative)
    :   Measurable traits expressed by plants
    Population
    :   Non-random mating between groups within a specified geographic space
    Subspecies
    :   In this paper, subspecies is a major hierarchical cluster of genotype groups and their respective families and/or genotypes. Subspecies could interbreed but don’t in natural environment due to some sort of natural barrier" />
<meta name="og-title" property="og:title" content="Genomic Diversity and Climate Adaptation in Brachypodium" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/015495v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The Brachypodium genus contains the model grasses B. distachyon , B. stacei and B. hybridum , that are useful for molecular and physiological studies relevant to grain, pasture and bioenergy crops, as well as ecology. In this chapter we discuss the natural variation in climate/geography, genotypic and phenotypic diversity that exists within these species. We describe utilisation of this diversity via two methods, Genome Wide Association Studies and Landscape Genomics, to examine the interaction between specific genetic variants, phenotype, and environment. The aim is to identify adaptive loci that control specific traits in specific environments and understand the contribution of background polygenetic variation shaped by demographic processes. With recent developments in high throughput phenotyping, cheaper genotyping by sequencing and higher spatial/temporal resolution of climate data, these approaches can exploit the diversity of the Brachypodium. Experiments using this toolkit will reveal alleles, genes and pathways underlying agriculturally important and environmentally sensitive traits for use in grass breeding.

*   GBS
    :   genotyping by sequencing
    GWAS
    :   genome wide association studies
    QTL
    :   quantitative trait loci
    MaxEnt
    :   Maximum Entropy
    SNPs
    :   single nucleotide polymorphisms

*   Accession
    :   A collection of seeds from one location. This includes bulk collections and maternal descent lines
    Ecotype
    :   An individual or group whose genetic distinction is strongly associated to an environment or type
    Genotype
    :   This general term is used either to describe the genotype at a locus such as a SNP (AA, Aa, aa) or a background whole genome genotype which can have levels of species, subspecies, population genetic structure group, family, individual maternal line)
    Phenotype (qualitative and quantitative)
    :   Measurable traits expressed by plants
    Population
    :   Non-random mating between groups within a specified geographic space
    Subspecies
    :   In this paper, subspecies is a major hierarchical cluster of genotype groups and their respective families and/or genotypes. Subspecies could interbreed but don’t in natural environment due to some sort of natural barrier" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2015-02-23" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/015495v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/015495v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/015495v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/015495v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/3950" />
    <title>Genomic Diversity and Climate Adaptation in Brachypodium | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"4FIythqSg8-sXdmlPT6ccNTsgmROc0yyzmMr2UbLjxw","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;015495v1"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;015495v1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;015495v1"}],"nid":"3950","apath":"\/biorxiv\/early\/2015\/02\/23\/015495.atom","pisa":"biorxiv;015495v1","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/015495v1","title":"Genomic Diversity and Climate Adaptation in Brachypodium","identifier":"node\/3950"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220sStd7OHm\\\\\\\/SIfWtUqE9XqeQkqEIuv4il13WySIqPsJA2dQLkTkqXSlB6xA0Bd\\\\\\\/7qakv5L4Em\\\\\\\/s3Fup35dqvspOxtTyvLlhK8p4jEMeRanLmv1oF+QPhfbczuQMLfpowLGSsFQdkBqrGkInZ6VPB1gAtH1RVXMFcnBK19kXylyvQl1v9QI5mH3CAmizDWLM\\\\\\\/XZHD0uBUdLefuo4KLbWNBuPsgup+Vrdn30ueEKJEGt7kdrGNRe8QxdUKODG0g6KIIhA\\\\\\\/B4uA\\\\\\\/Jr1vzpM605VGBsDOq7wxph7lI774WGohk8KXdIBVVBNU6INnJw+Pg3b0By1bg7rqGX36WcgQk6A\\\\\\\/u8yzFzf6eeJ\\\\\\\/JbCzwuBy0uwecq7AUcdCyJWQSKWG8IMmABFGeuljPFYAmTQMx3wy3h9v399Vg6SjJ3IZxD5CKrdxZbE0fgOcYIOURB5NlfUlOnjv0KFwKQdgpToiBbDMq05VLQUU5\\\\\\\/+FcdcXxQEFxzSit9kP2fGqvyFkSOGEAgrPTbWpGyN7F0S9Fjejeg9FqK7PKrhUBf5nZ7t1o\\\\\\\/MVgpYNeH\\\\\\\/j9RrZLmAqm7sFigwsnbHjQ3eTdtSLN33fioR7rl24vW\\\\\\\/uzWd\\\\\\\/wzscpRLMsoIKjmriMZq\\\\\\\/1l601J\\\\\\\/06UUdW2LWKyelTxHlstWzt7WwxQaG73ETaNuKchd6cnI7PB6VJZbrE+Cw565PFwhevhS7Wm31tZLkuTSIeen95OZlhSvOCCUxt8wIhEhL\\\\\\\/pJewECUqLBX7gvNMCrFolEBXbFr7f7ejR7shX1jEWi\\\\\\\/c\\\\\\\/a7V+\\\\\\\/hJ2EV4nL+xohg3bdtva5Vl\\\\\\\/ZV54fFHJXfCEX94rTmrs1hOnIDY+nyFcumLb+BWjJIBG1zA1bpyOR1Fvk+l1srP9xfNo\\\\\\\/+eex8fCLIiifNAD2VTaO6c5+75QHr\\\\\\\/wQ4GDnFNe90yEYsXaPt\\\\\\\/cn9xvJqTZtUsNyJlHQvDasCX4c+g4o2TKQ\\\\\\\/rTyybpvfD2YffPOvUVJ04PLWV\\\\\\\/uMrrejL8kaT1PnDR9JnoIu70NcjDb+Hqg7WSTD3bBnv\\\\\\\/LG1ZEAQpSp46NWb1ZlfB6asBEgc2QpYG0yEpnUVgpkK7FVJCeK2dnBBFGpfZ+PZJZap+gXBt87Z\\\\\\\/BlPca\\\\\\\/gcYBO51q4duXO5BzAavNP7LubfnVwg9XD3zni6vZhAiDMQVrjQ3Ihmsit9zEy7EYN9rwk4Us9hGjzx6C8il2JNY23lg\\\\\\\/Bv3vc+Xl18l2sqNNOoWWT8h67bh9YDsHVByxoAMqZl3l74UrNYabr5z2DjUwBSblah0pelXzpIcJhxiNd9rsWrGZwB71zPIoYqK+0mDAqsRwrZU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Y86mBMmUM8NqT\\\\\\\/Fo\\\\\\\/6Yv8A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226b718219a6ccc3b5bff38facc30dd9f069385438b8ae1108ed1726b682e78207\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/015495v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-3950 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/015495v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JpR9Jcvgqdwiq_D4fZu4trNVEHq4QFfT8gWsdst8LpA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="3950" id="node3950" data-pisa="biorxiv;015495v1" data-pisa-master="biorxiv;015495" data-apath="/biorxiv/early/2015/02/23/015495.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Genomic Diversity and Climate Adaptation in Brachypodium</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Pip</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jared</span> <span class="nlm-surname">Streich</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Justin</span> <span class="nlm-surname">Borevitz</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/015495 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node3950" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Pip Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centre of Excellence in Plant Energy Biology</span>, 134 Linnaeus Way, Australian National University, Canberra, ACT 2601 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pip%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APip%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Justin.Borevitz@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">Justin.Borevitz@anu.edu.au</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jared Streich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centre of Excellence in Plant Energy Biology</span>, 134 Linnaeus Way, Australian National University, Canberra, ACT 2601 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jared%2BStreich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Streich%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJared%2BStreich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Justin.Borevitz@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">Justin.Borevitz@anu.edu.au</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Justin Borevitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centre of Excellence in Plant Energy Biology</span>, 134 Linnaeus Way, Australian National University, Canberra, ACT 2601 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BBorevitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Borevitz%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BBorevitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Justin.Borevitz@anu.edu.au" class="" data-icon-position="" data-hide-link-title="0">Justin.Borevitz@anu.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/015495v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:3950" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:3950/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/015495v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:3950" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:3950/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/015495v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:3950" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:3950/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/015495v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:3950" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:3950/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/015495v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:3950" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:3950/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The Brachypodium genus contains the model grasses <em>B. distachyon</em>, <em>B. stacei</em> and <em>B. hybridum</em>, that are useful for molecular and physiological studies relevant to grain, pasture and bioenergy crops, as well as ecology. In this chapter we discuss the natural variation in climate/geography, genotypic and phenotypic diversity that exists within these species. We describe utilisation of this diversity via two methods, Genome Wide Association Studies and Landscape Genomics, to examine the interaction between specific genetic variants, phenotype, and environment. The aim is to identify adaptive loci that control specific traits in specific environments and understand the contribution of background polygenetic variation shaped by demographic processes. With recent developments in high throughput phenotyping, cheaper genotyping by sequencing and higher spatial/temporal resolution of climate data, these approaches can exploit the diversity of the Brachypodium. Experiments using this toolkit will reveal alleles, genes and pathways underlying agriculturally important and environmentally sensitive traits for use in grass breeding.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Brachypodium is part of the Poaceae plant family, one of the five largest. Poaceae are one of two plant families found on all seven continents and grow in most habitats from alpine, to freshwater, and marine environments (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Groombridge and Jenkins 2002</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brummitt and Cheek 2007</a>). The Pooideae subfamily of the Poaceae are also widespread, especially in temperate zones, and many genera extend to cold climates in mountainous regions or high latitudes. Similarly to other Pooideae genera, the Brachypodium genus also has a wide distribution in temperate regions, but is less common in cold climates and more abundant in hot arid regions (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hartley 1973</a>). Brachypodium native range is the Mediterranean region with accessions commonly found through southern Europe, North Africa and Eurasia. However, worldwide herbarium records show Brachypodium is now present on all six continents except Antarctica (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Garvin et al. 2008</a>). This wide geographic range and climate tolerance in both the native and introduced range, as well as a key evolutionary position, mean Brachypodium is a rich resource of allelic diversity.</p><div id="sec-2" class="subsection"><h3>1 Diversity in Natural Populations of Brachypodium</h3><div id="sec-3" class="subsection"><h4>Climate and Geographic Diversity</h4><p id="p-4">A large collection of thousands of Brachypodium accessions are now available. They come from a range of seasonal temperature profiles that span from alpine to hot arid deserts (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The majority of accessions have been collected from temperate grasslands with hot dry summers and wet winters (Mediterranean). These include accessions from southern Europe and South-Western USA. Many accessions come from temperate regions that also get summer rainfall, such as France and South-Eastern Australia. A few accessions come from regions with colder climates, such as mountainous regions in Turkey and on the Spanish/French border. Lastly, several accessions come from arid regions in the Middle East. These different climate types favour the development of different life history strategies allowing optimization of reproductive success, as well as abiotic stress tolerance mechanisms to deal with heat, cold and drought.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Geographic and climatic distribution of Brachypodium accessions Distribution of the 350 populations, &gt;1,400 accessions, of Brachypodium overlaid on the K&#xF6;ppen-Geiger climatic regions (Peel et al. 2007). Many other private collections exist." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1144555808" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Geographic and climatic distribution of Brachypodium accessions&lt;/span&gt; Distribution of the 350 populations, &gt;1,400 accessions, of Brachypodium overlaid on the K&#xF6;ppen-Geiger climatic regions (Peel et al. 2007). Many other private collections exist.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F1.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F1.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <span class="caption-title">Geographic and climatic distribution of Brachypodium accessions</span><p id="p-5" class="first-child">Distribution of the 350 populations, &gt;1,400 accessions, of Brachypodium overlaid on the Köppen-Geiger climatic regions (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Peel et al. 2007</a>). Many other private collections exist.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">The geographic distribution of Brachypodium is dependent on climate and the history of its dispersal. Brachypodium species have had a long association with human civilisation. There is evidence of use of Brachypodium species as food in Paleolithic societies in the region of Italy as long as 30 000 years ago (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Revedin et al. 2010</a>). This association may have aided its dispersal through Europe and Eurasia. In more modern times, Brachypodium co-localisation with cereal crops may have aided its dispersal through joint harvest and planting, especially from its native range in the Mediterranean to the “New World” of the Americas and Australia.</p></div><div id="sec-4" class="subsection"><h4>Genomic Diversity</h4><p id="p-7">There is a large amount of genomic diversity in the <em>Brachypodium distachyon</em> complex. Firstly there are three species now described. <em>B. distachyon</em> (2n=10) and <em>B. stacei</em> (2n=20) share common ancestry at large fractions of their genome and <em>B. hybridum</em> (2n=30) is an allopolyploid of <em>B. stacei</em> and <em>B. distachyon</em>. Secondly there is major population genomic structure within each species. Genotyping By Sequencing (GBS) is an efficient way to characterize population and phylogenomic variation within and between populations and species in a complex. It uses restriction digestion, barcoded adaptor ligation, pooling, second generation sequencing, and de-multiplexing to identify SNPs and call genotypes at &gt;10,000s loci (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Elshire et al. 2011</a>). Multiplexing reduces the individual sample cost of sequencing allowing many more accessions to be sequenced, up to 384 in a single sequencing lane.</p><p id="p-8">Preliminary results are showing that <em>B. distachyon</em> can be divided into two subspecies, those accessions that are similar to the Bd21 accessions, denoted as subspecies A, and those that are similar to the Bd1-1 accession, denoted as subspecies B. While the two subspecies can be intercrossed in the laboratory, the barriers seem to be maintained in nature. Within each of these subspecies there is further geographic subdivision in genetic diversity between Eastern and Western Europe (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Subspecies and population genomic subdivision with Brachypodium The first division in B. distachyon is that between the A and the B subspecies. Second is the geographic split between Eastern and Western Europe. This implies independent re-colonization of Europe, for each subspecies via both the east and west routes after glacial retreat." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1144555808" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Subspecies and population genomic subdivision with Brachypodium&lt;/span&gt; The first division in &lt;em&gt;B. distachyon&lt;/em&gt; is that between the A and the B subspecies. Second is the geographic split between Eastern and Western Europe. This implies independent re-colonization of Europe, for each subspecies via both the east and west routes after glacial retreat.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F2.medium.gif" width="440" height="267"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F2.medium.gif" width="440" height="267"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <span class="caption-title">Subspecies and population genomic subdivision with Brachypodium</span><p id="p-9" class="first-child">The first division in <em>B. distachyon</em> is that between the A and the B subspecies. Second is the geographic split between Eastern and Western Europe. This implies independent re-colonization of Europe, for each subspecies via both the east and west routes after glacial retreat.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">This East-West division is most likely due to isolation by distance. Arabidopsis also shows a weak east west divide due to recolonisation of Europe, after the last ice age from refugia on the Iberian Peninsula and Western Asia (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sharbel et al. 2000</a>). Brachypodium has a lot of parallels with Arabidopsis including its highly selfing nature, similar native range of Europe and Eurasia, similar habitats in disturbed sites and similar annual life strategy. The pattern of range retraction and subsequent recolonisation likely explains the genetic differentiation among Eastern and Western Europe genotypes seen in <em>Brachypodium distachyon, Arabidopsis thaliana</em>, and many other species. The presence of this pattern in both the A and B subspecies indicates parallel re-colonisations of subspecies that diverged before the last ice age.</p><p id="p-11">Though currently under represented in germplasm collections, the evolution that created <em>B. stacei</em> appears to have one prominent haplotype, and many rare haplotypes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Catalan et al. 2012</a>). A substantial portion of the <em>B. distachyon</em> genome is shared with <em>B. stacei</em> across many chromosome segments. <em>B. hybridum</em> are allotetraploids, resulting from a recent hybridization events fusing <em>B. distachyon</em> and <em>B. stacei</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Betekhtin et al. 2014</a>). This polyploidisation with <em>B. stacei</em> may have occurred independently with both A and B subspecies of <em>B. distachyon</em> (further discussed in <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Catalan et al. 2012</a>). All three Brachypodium species are morphologically similar to each other, but express different traits. <em>B. stacei</em> and <em>B. hybridum</em> have higher seed yield, more biomass and quicker growth rate than <em>B. distachyon</em> (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Catalan et al. 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Vogel et al. 2009</a>). They also have different ranges with the majority of accessions found in North America and Australia being <em>B. hybridum</em> while <em>B. stacei</em> has been found throughout Mediterranean regions. These species provide ideal models for studies of allopolyploidy with parallels particular to wheat, <em>Triticum aestivum</em>.</p></div><div id="sec-5" class="subsection"><h4>Adaptive Phenotypic Variation</h4><p id="p-12">The genetic makeup, and resulting phenotypic expression, of a given accession is affected by the genotypic and climatic history of its ancestors as this determines the suite of alleles present in its genome. Past and present selective pressures, isolation by distance, and the prevalence of intercrossing all affect this genotypic diversity.</p><p id="p-13">Selective pressures can include biotic and abiotic stresses, the prevalence of which can vary at different times of year and from year to year. These pressure can select for favourable alleles in a mixed population or favour new advantageous mutations over the standing genetic variation. <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Grime (1977)</a> defined classic strategies for plant survival in selective environments including competition (C), stress tolerant (S) and tolerant to ruderal/frequently-disturbed environments (R) the CSR model. Plants that are competitive tend to have rapid growth with extensive lateral spread above and below ground and low seed yield. Stress tolerant plants tend to have slow growth with small leaves and low seed yields. Ruderal plants tend to have rapid growth, with small stature and large yield. Most Brachypodium accessions seem to favour a ruderal strategy, however the diversity of life strategies, growth rates, and plant architectural traits seen suggest that competition and stress tolerance are also important selective pressures.</p><p id="p-14">Isolation by distance can cause populations to diverge over time and has a major impact on Brachypodium diversity. This differentiation may occur via physical barriers, such as a mountain range, ice sheet or seas. This can be seen in the East West divide within each <em>B. distachyon</em> subspecies. However, isolation by distance may also occur from a difference in reproductive timing as different life strategies initiate flowering at different times of year, eliminating the chance of cross-pollination. This could be a factor in the division of the A and B subspecies in <em>B. distachyon</em>. Post reproductive barriers could also be involved, such as genetic incompatibilities and hybrid breakdown.</p><p id="p-15">The timing of germination and reproduction, also know as life strategy, are good examples of phenological variation that are strongly shaped by natural selection in local climates. Brachypodium accessions collected from high altitude and high latitude sites are generally strong ‘vernalisation requiring’ ecotypes (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schwartz et al. 2010</a>). This allows extended vegetative growth and the synchronisation of flowering with the more favourable conditions of late spring rather than risking damage to cold-sensitive floral structures in winter. In Mediterranean regions, winter rainfall determines the growing season for ‘rapid cycling’ ecotypes to avoid the hot dry summers. Accessions from hot arid climates with intermittent rainfall as well as disturbed agricultural site also tend to harbour ‘rapid cycling’ ecotypes with fast generation times, low dormancy and low vernalisation requirements, e.g. Bd21-1 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrero et al. 2012</a>). This fast generation time allows a full life cycle to occur when moisture is available, possibly two times a year, while enduring dry periods as seed. Though rare, Brachypodium accessions from warmer, wet summer regions are later flowering allowing more vegetative growth and increased summer seed set. Some natural accessions may also have the ability to produce different phenotypes in different environmental conditions because of phenotypic plasticity. For example, accessions from variable climates may have some vernalisation requirements, but also a strong photoperiod response such that if there is a cold winter, flowering can be promoted, but in a mild winter flowering is initiated by a strong photoperiod response.</p></div><div id="sec-6" class="subsection"><h4>Case Study: Flowering Time</h4><p id="p-16">An experiment to investigate the genotypic and environmental basis for the diversity of flowering time of <em>B. distachyon</em> was undertaken at ANU in 2014. Plants were grown in specially modified growth chambers to simulate a temperate region with a cold winter and hot summer. Chambers were set for either a Winter or Spring germination. The modified chamber controls allowed us to program down to one minute changes in light intensity, light spectrum, temperature and humidity; mimicking diurnal and seasonal changes in climate. In this experiment 256 accessions were grown in each chamber. Ear emergence was chosen as an indicator of transition to reproduction as many accessions flowered within the ear and hence were difficult to score for flowering. The Winter germination chamber had cold (5°C nights) resulted in the vernalisation requirements of all lines being met and the majority of lines reaching ear emergence between 100-160 days post emergence (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a</a>). However, in the Spring germination treatment the transition to reproduction was more variable. The majority of lines in the Spring germination reached ear emergence fairly quickly between 60 - 120 days post emergence. However, although there were still a few weeks with cold nights, the accessions requiring stronger vernalisation did not reach ear emergence by the end of the experiment at 180 days (e.g. Tek1, ABR4, UKR-99-137). Interestingly, while the life strategy of the majority of lines remained fairly unchanged between environments (early flowering), those accessions from Western Europe tended to show more phenotypic plasticity between environments with a wider range of delay in ear emergence with the Winter sowing than the Spring sowing.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Life strategy of B. distachyon accessions is associated with subspecies, region and climate type at the collection site 256 Brachypodium accessions were grown in simulated growth conditions to mimic a Winter and Spring sowing date. Days from seedling emergence to ear emergence was monitored as an indicator of transition to the reproductive stage. a) Comparison of life strategy in the two environments; dashed lines indicate the cutoffs for classification of accessions as early, mid or late flowering based on their flowering time under the Spring germination. b) Subspecies and geographic classification of accession show varying distributions of early, mid and late flowering accessions; numbers in columns indicate number of accessions in each category and c) Early, middle and late flowering groups of accessions are sourced from different proportions of climate types." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1144555808" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Life strategy of &lt;em&gt;B. distachyon&lt;/em&gt; accessions is associated with subspecies, region and climate type at the collection site&lt;/span&gt; 256 Brachypodium accessions were grown in simulated growth conditions to mimic a Winter and Spring sowing date. Days from seedling emergence to ear emergence was monitored as an indicator of transition to the reproductive stage. a) Comparison of life strategy in the two environments; dashed lines indicate the cutoffs for classification of accessions as early, mid or late flowering based on their flowering time under the Spring germination. b) Subspecies and geographic classification of accession show varying distributions of early, mid and late flowering accessions; numbers in columns indicate number of accessions in each category and c) Early, middle and late flowering groups of accessions are sourced from different proportions of climate types.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F3.medium.gif" width="189" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F3.medium.gif" width="189" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <span class="caption-title">Life strategy of <em>B. distachyon</em> accessions is associated with subspecies, region and climate type at the collection site</span><p id="p-17" class="first-child">256 Brachypodium accessions were grown in simulated growth conditions to mimic a Winter and Spring sowing date. Days from seedling emergence to ear emergence was monitored as an indicator of transition to the reproductive stage. a) Comparison of life strategy in the two environments; dashed lines indicate the cutoffs for classification of accessions as early, mid or late flowering based on their flowering time under the Spring germination. b) Subspecies and geographic classification of accession show varying distributions of early, mid and late flowering accessions; numbers in columns indicate number of accessions in each category and c) Early, middle and late flowering groups of accessions are sourced from different proportions of climate types.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">The accessions were then classified as early, mid or later flowering based on the days to ear emergence in the Spring sowing (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3a</a>). When these groupings are compared to the subspecies classifications we can see there is no firm trend for flowering time between the subspecies, as the patterns differ significantly in the Eastern European and Western European subsets (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3b</a>). For example, while accessions from the A subspecies tend to be early flowering in Eastern Europe, a mix of life strategies are present in Western Europe. This mix would indicate that different flowering time alleles have evolved in the different subspecies. Making crosses between accessions from different geographic regions and also between subspecies within a region would be needed to genetically map divergent flowering time alleles. Fortunately multiple RIL sets are available spanning this natural divide. Just as flowering time is variable within subspecies and geographic regions, it also varies with climatic type at the site of collection. When we categorise our accessions by climatic region, the life strategy patterns are not fixed based one on selective pressures in these climates alone. Accessions from climates with dry summers can be early or late flowering (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3c</a>). A similar trend is seen for accessions from arid regions, where a wide range of flowering times are seen. Mid flowering accessions tend to be from climates with rainfall evenly distributed through the year, indicating that the accessions with long growing seasons and no vernalisation requirements have a fitness advantage in this climate type. An alpine accessions was late flowering, suggesting that vernalisation is needed for survival in this environments. We must keep in mind sampling bias in any collection. The observed trends can be influenced by when and where the collections are made. Netherless, there are trends with exceptions, illustrating the impact of the genotype and climate on the geographic distribution and phenotypic expression of accessions.</p><p id="p-19">Understanding fitness of an individual accession in a particular climate is complex, complicating studies aimed at elucidating the genetic architecture of adaptive traits. Luckily, new approaches have been developed that control for genetic and spatial structure, which can result in outcomes that have applications in crops species breeding programs, weed management and ecosystem restoration. Next we outline two of these approaches: Genome Wide Association Studies (GWAS) and Landscape Genomics.</p></div></div><div id="sec-7" class="subsection"><h3>2 Using Local Adaptation to Identify Causative Alleles</h3><div id="sec-8" class="subsection"><h4>Genome Wide Association Studies</h4><p id="p-20">Natural populations provide a valuable resource in the diversity of genotypic variation that has been found in extreme environments. Historically these populations have been studied for phenotypic diversity in small numbers of accessions using SSR and microsatellites. The development of comprehensive next generation sequencing has meant that thousands of accessions can now be screened and powerful subsets selected for Genome Wide Association Studies.</p><p id="p-21">Genome wide association studies (GWAS) were originally developed for studies of human disease to find causative mutations (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Klein et al. 2005</a>). The aim of GWAS is to identify the genetic architecture of phenotypic traits of interest in a certain population. This can include the number of alleles associated with the trait, whether one allele affects multiple traits and the strength of individual alleles in accounting for the trait of interest. By comparing different environments we can also identify which alleles are important in each environment, which are robust across environments and which allow for plasticity of the trait between environments. GWAS have been performed in a number of plant species including Arabidopsis, maize, rice, barley, and sorghum (e.g. <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Atwell et al. 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wen et al. 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huang et al. 2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pasam et al. 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Morris et al. 2013</a>). These studies have successfully identified causative QTLs for many traits including disease resistance, proline accumulation, flowering time, starch profile, plant height and grain weight.</p><div id="sec-9" class="subsection"><h5>Genotype Data</h5><p id="p-22">To perform GWAS you must have genotyped a large number of SNPs tagging the many haplotype blocks that are differentially assorted in the sample being studied. The density of SNPs needed depends on ancestral recombination in the genome and will affect the resolution of the QTLs identified. To get high density SNP genotyping whole genome re-sequencing can be used by randomly shearing DNA and ligating barcoded adaptors. Like GBS, samples are then multiplexed and run on an Illumina HiSeq platform. Low pass sequencing (∼1-2X) is sufficient as missing genotypes at common SNPs can be imputed based on linkage disequilibrium (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Halperin and Stephan, 2009</a>). GWAS must also be performed on a sizable number of accessions, at least 100 but optimally ∼300, to maximise statistical power. SNPs are usually filtered at 5% minor allele frequency. A genome scan is then performed testing each SNP for association with the quantitative trait while controlling for ancestral relatedness among samples. Once a QTL are identified to perhaps 100kb resolution, candidate genes can be selected. Ultimately the causative alleles can be confirmed by transgenic complementation of knockout lines with different functional alleles.</p><p id="p-23">GWAS has some advantages over traditional bi-parental mapping (e.g. RILs), nested association mapping (NAM) or multi-parent advanced generation intercrossed (MAGIC) populations in that the source of genetic variation is much greater as its not just limited up to a couple dozen founder accessions, but comes from hundreds. Furthermore, the resolution of mapping of the causative allele can be much greater due to the higher frequency of recombination events in a large ancestral population evolved over centuries compared to a hand-crossed population with only a handful of generations of crossing. However, rare alleles of large effect contribution noise for GWAS. In addition, population structure from non-random mating can cause false-positives at loci associated with the causative one. Even when appropriately controlled for statistically, population structure can cause false-negative results by downplaying the effect at the truly causative locus.</p><p id="p-24">There are a couple approaches to mitigate the effects of population structure. The first is to balance the structure. GBS can be performed on a very large number of diverse lines from across the world. Once the structure of the global population is known, a core GWAS set can be selected balancing membership roughly equally from across different lineages (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4a</a>). Whole genome sequencing is performed on this GWAS set, to increase resolution of mapping causative alleles. A kinship matrix is also made from the SNP data to include in the statistical analysis to reduce the rate of false positives. The second approach is to perform the same initial GBS screen of a wide range of accessions, but then to select a region within a population structure group or a hybrid zone where the groups have become well mixed. The resulting population sample be less confounded such that segregation at the causative alleles will be largely uncoupled from the background genetic variation. These approaches are further described in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brachi et al (2011)</a> and an example of its success in Arabidopsis is given by (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li et al. 2010</a>: <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baxter et al. 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Horton et al. 2012</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Considerations when performing GWAS in Brachypodium a) Selection of a balanced set of accessions is key for successful GWAS. Initial GBS of all available accessions facilitates selection of a subset of lines for intense phenotyping and whole genome sequencing b) Replication of a few key lines allow for the heritability of a phenotypic trait to be estimated. Traits with large environmental variance will return poor GWAS results. Points are the average time to ear emergence of 3-6 biological replicates &#xB1; S.D.; * did not flower in the Spring germination c) A genome wide screen of SNPs identifies those associated with a quantitative trait. Manhattan plot with arrows showing SNPs associated with days to ear emergence with a Spring germination. A balanced set of 95 accessions from the A subspecies was used in analysis. The dotted line indicates the 5% empirical genome-wide significance threshold." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1144555808" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Considerations when performing GWAS in Brachypodium&lt;/span&gt; a) Selection of a balanced set of accessions is key for successful GWAS. Initial GBS of all available accessions facilitates selection of a subset of lines for intense phenotyping and whole genome sequencing b) Replication of a few key lines allow for the heritability of a phenotypic trait to be estimated. Traits with large environmental variance will return poor GWAS results. Points are the average time to ear emergence of 3-6 biological replicates &#xB1; S.D.; * did not flower in the Spring germination c) A genome wide screen of SNPs identifies those associated with a quantitative trait. Manhattan plot with arrows showing SNPs associated with days to ear emergence with a Spring germination. A balanced set of 95 accessions from the A subspecies was used in analysis. The dotted line indicates the 5% empirical genome-wide significance threshold.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F4.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F4.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <span class="caption-title">Considerations when performing GWAS in Brachypodium</span><p id="p-25" class="first-child">a) Selection of a balanced set of accessions is key for successful GWAS. Initial GBS of all available accessions facilitates selection of a subset of lines for intense phenotyping and whole genome sequencing b) Replication of a few key lines allow for the heritability of a phenotypic trait to be estimated. Traits with large environmental variance will return poor GWAS results. Points are the average time to ear emergence of 3-6 biological replicates ± S.D.; * did not flower in the Spring germination c) A genome wide screen of SNPs identifies those associated with a quantitative trait. Manhattan plot with arrows showing SNPs associated with days to ear emergence with a Spring germination. A balanced set of 95 accessions from the A subspecies was used in analysis. The dotted line indicates the 5% empirical genome-wide significance threshold.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">In consideration to the first approach outlined above, GWAS cannot be performed across the A and B subspecies due to a high level of genome level differentiation. Hence we suggest that GWAS sets are developed for each subspecies separately. Furthermore, the current <em>B. distachyon</em> public collections contain a lot of groups of very closely related accessions, e.g. BdTR1_ lines. Selection of a GWAS set must also limit family structure. Including closely related accessions limits recombination and introduces bias towards the common alleles in that family. The second approach of finding a well-admixed population may be challenging due to the selfing nature of Brachypodium. However, it is our experience that several field sites and even some maternal lines are segregating both genotypes and phenotypes, indicating that this strategy is a possibility.</p></div><div id="sec-10" class="subsection"><h5>Phenotyping Data</h5><p id="p-27">There are a number of GWAS requirements for phenotype data. The traits must be due to common genetic variation rather than multiple rare variants. It is helpful to measure heritability, or the proportion of total phenotypic variation (Vp) due to genetic variation (Vg) among lines. A simple way to estimate heritability (H2) is to subtract environmental variation (Ve) within replicate inbred lines, H2 = Vg/Vp = (Vp-Ve)/Vp. If the variability within a genotype is low then more of the variability seen in the phenotype will be due to genetic factors (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4b</a>). Alternatively the pseudo-heritability is the amount of phenotypic variation explained by the kinship or pairwise genetic diversity among lines and can be calculated prior to GWAS.</p><p id="p-28">Due to the large number of accessions to be phenotyped for GWAS, the phenotypes must also be quantifiable in a high throughput manner. For some measures, such as plant height or ear emergence this can be done manually with relative ease. However, for traits that require continuous measurement, such as growth rates, or those where measurements and analysis are very time consuming, such as photosynthetic measures, high throughput phenotyping systems can be very beneficial. One such system is the PlantScreen phenotyping platform (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brown et al, 2014</a>). This platform allows up to 300 plants to be screened at a time; including individual pot weighing and watering, thermal imaging, RGB stereo imaging and chlorophyll fluorescence. This system allows us to monitor abiotic stress tolerance of accessions by measuring photosynthetic efficiency and photoinhibition; water use; stomatal conductance/transpiration cooling; and photo-protective mechanisms such as pigment accumulation and non-photochemical quenching (NPQ).</p></div><div id="sec-11" class="subsection"><h5>Genotype by Environment Interactions</h5><p id="p-29">An important extension of GWAS is to understand how the growth environment affects the genetic architecture of traits. Certain loci may be more important in the expression of a trait of interest in particular environments. A simple example is where a vernalisation-sensitive allele may be important in controlling flowering time in an alpine climate, but a photoperiod-sensitive allele may be more important in a lowland climate. The interaction between genetic loci may also vary between environments. Understanding the genetic architecture of phenotypic plasticity is also of great interest, especially in a world where climate variability and change is accelerating. Hence it is important to consider how the growth environment will influence the phenotypes seen in GWAS experiments and how this will relate to extrapolation of results to applications of plant breeding or ecological work.</p><p id="p-30">Using dynamic growth chambers, such as the SpectralPhenoClimatron (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Brown et al, 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li et al, 2006</a>), in combination with climate modelling software such as SolarCalc, we can undertake GWAS to simultaneously compare contrasts such as life strategies at different times of year (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Li et al, 2010</a>); growth and yield in current and future climates (e.g. <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li et al, 2014</a>); and the effect of fluctuating light, as would be seen under a forest canopy, on growth and initiation of photoprotective mechanisms.</p></div><div id="sec-12" class="subsection"><h5>Applications</h5><p id="p-31">GWAS allows us to understand the genetic architecture of interesting phenotypic traits, whether it is flowering time, biomass production, components crop yield or abiotic stress tolerance. With this approach we can identify segregating alleles in natural populations causing advantageous phenotypes (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4c</a>). This understanding can then be applied to breeding better crops by selecting genotypes with predicted to have optimal phenotypes, aka crop design (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Huang and Han, 2014</a>). Understanding how the genetic architecture of traits is affected by environment allows us to select accessions with the appropriate genetic makeup to thrive in specific environments, current or future. This can apply to both agriculture and foundation species of natural systems as climate change is occurring more rapidly than traditional breeding by phenotypic selection keep up with (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hoffmann et al, 2015</a>).</p><p id="p-32">In summary, GWAS is a powerful method to dissect the genetic basis of complex traits and the dependence on the growth environment. It is of interest to know if these are adaptive traits and contribute to yield and fitness in particular environments as this can help predict invasive potential and resilience under environmental change. Genes controlling adaptive traits may show a signature of selection on the landscape if one allele provided an advantage in a particular environment and/or the other allele was detrimental in an opposing environment. Indeed direct associations between environmental variables at geographic locations and alleles at quantitative trait loci have been found, confirming their role in adaptation (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Li et al, 2010</a> and others). This opens the door to a genome scan to naively identify adaptive loci associated with environmental variables. This could work without knowledge of what traits are under selection. Though promising, this so called Landscape Genomics approach, has caveats including needing to control for background variation due to population structure. Geographic location is confounded with many environmental variables limiting our ability to separate selection at adaptive loci from isolation by distance that shapes the entire genome.</p></div></div><div id="sec-13" class="subsection"><h4>Landscape Genomics</h4><p id="p-33">Landscape Genomics is a growing field in plant biology integrating the sciences of geographic and climate mapping with population genomics and association studies. Teams of researchers in this field must have a diverse set of skills ranging from the lab to field studies uniting theory and computational biology. They must collect and analyse large data sets to correlate spatial and environmental variables with perhaps millions of genetic loci while accounting for genetic relatedness and demographic processes affecting the whole genome. Landscape genomics combines high resolution spatial/temporal climate data from the sites of collection with second generation sequencing technology to 1) explore the differences in climatic niche breadth among genotypes; to 2) select locations with variable microclimates and well mixed genetic variation; and 3) to associate particular climatic variables (e.g. winter temperature or precipitation levels) with specific adaptive alleles. Landscape genomics can be applied to almost any species in the natural environment, including animals, fungi, and plants. Some of the most successful studies are often of model species and their close relatives, which usually have annotated reference genomes and large developed collections that are already sequenced (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hancock et al, 2011</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fournier-Level et al, 2011</a>).</p><p id="p-34">Landscapes are longstanding experiments of natural and, increasingly, artificial selection. Landscape genomics dissects and describes the underlying adaptive genetic loci through association with a suite of environmental variables describing the microclimate within a species range. Initially, populations are founded with limited genetic diversity. New mutations are initially rare and most disappear due to genetic drift. When new genetic variation arrives via migration, it brings a genome’s worth of standing genetic variation. Even in largely selfing species, outcrossing will eventually happen and new allelic-combinations will be released for selection across the environment. As in synthetic crosses, the recombinant chromosomes initially have extended haplotypes from the founders that are ultimately broken down through subsequent generations and further outcrossing. There may now be opportunity for local range expansion as the diverse population spreads into new microclimate, soil and/or habitat types. As the ecological space fills, competition will favour plants with adaptive alleles in their most suitable habitat. Neutral genetic variation will have a largely random pattern across the landscape. Across the genome, a trend of isolation by distance may develop if gene flow is limited. Ideally this would be in a direction orthogonal to the environmental gradient. At this stage a genome scan for SNP alleles in association with local environment can identify adaptive loci. The allele effect at a locus multiplied by the strength of selection determines the ability to identify and adaptive QTL. The size of the adaptive locus is determined by the amount of recombination, which should narrow as the population ages. Control of false positives due to population structure can be performed by statistically accounting for overall relatedness across the genome in a manner analogous to GWAS. Across large geographic regions multiple rare variants explain adaptation limiting Landscape Genomics in much the same way as GWAS.</p><p id="p-35">One advantage of annual plants is selection acts quickly at each generation. It can be observed at a locus when a few ancestral haplotypes are competing and gene flow mixes the background genetic variation. By understanding the genetic basis of local adaptation, we may finding allelic variants that allow some groups to be specialists, while other alleles may be important for generalists (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Storz, 2005</a>). Brachypodium has a large geographic range, mostly spanning Mediterranean-like climates in Europe, North Africa and Eurasia. It is similar to <em>Arabidopsis thaliana</em>, which has genetic diversity associated with geographic and climatic space (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Platt et al. 2010</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Horton et al. 2012</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Banta et al. 2012</a>). Each subspecies of Brachypodium distachyon would be expected to show similar associations with adaptive alleles across geographic and climatic space.</p><p id="p-36">So the next step is identifying which populations and climate regions are suitable for Landscape Genomics. Different genetic lineages may occupy different geographic ranges and corresponding climate envelopes. These can highlight potential adaptive differences however alleles providing the advantage may be fixed within each lineage. Hybrid zones between groups or long range dispersal followed by admixture could provide the opportunity to uncouple adaptive alleles from their genomic background. Digital herbarium records are a great place to start describing the climate range of a species utilizing existing collections. Records often have meta data about microclimate, some phenotype data including whether the plant was flowering, in addition to when and where the plant was collected. This can aid a researcher’s decision about when to travel for collecting, what traits to look for and what locations they occupy. With collection points, researchers can also travel back to the same location or similar locations based on model predictions like those created in computer programs to predict niche breadth using climate envelopes (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Phillips, 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Joost, 2007</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Banta, 2012</a>).</p></div><div id="sec-14" class="subsection"><h4>Determining the geographic range and climatic niche breadth of divergent genetic lineages</h4><p id="p-37">While the geographic range and environmental tolerance of a species can be wide, the breadth of environmental conditions a subspecies, or major genetic lineage, can inhabit may be relatively small. Understanding climatic breadth within species can inform growers about what crop varieties are suitable for their locations and can also aid seed collectors about new locations for future collections. Maximum Entropy (MaxEnt) software is a common research tool for inferring and predicting species distributions and tolerances of environmental factors (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Phillips, 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Phillips, 2008</a>). The inputs are flexible and can be manipulated by the user. Data is often sourced from WorldClim, a global grid of climate layers (WorldClim, 2015). A subset of biologically relevant variables, BioClim, are often used in ecological studies including species range modeling programs (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">BioClim, 2015</a>). These variables represent growth limiting factors such as temperature and precipitation and their seasonality. MaxEnt couples environmental data from training locations as inputs and compares these realised niche variables with all locations outputting geographical heat maps indicating where similar ecological niche exist. The results describe a species fundamental niche, which is the climate limited range where a species might be able to persist. MaxEnt outputs response curves of climate variables and uses permutations to rank the relative contribution of each variable when determining the niche. The program uses the model to predict sites left out of the training data. This cross validation also allows comparisons between models. Different models can be made and compared which incorporate population genetic information. This is done by defining different sets of training locations based on the occurrence of particular genetic lineages. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5a</a> shows the geographic range of the Australian Brachypodium fundamental niche generated by MaxEnt trained with the Bioclim climate variables at 80 locations that comprise our current collections. Mean Temperature of Wettest Quarter (WorldClim 8) was the strongest factor determining the distribution (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">BioClim, 2015</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicting species distribution and detecting segregating SNPs by climate input a) MaxEnt produced probable distribution of Brachypodium in Australia using 80 geographic locations as presence only data correlated with WorldClim/BioClim variables. b) 80 Australian populations collected by the Borevitz lab of B. distachyon complex to show the environmental relatedness of each population (no genetic data used). The vertical dendrogram describes environmental variable correlation across all populations. The horizontal dendrogram describes individual population clustering by their environmental relatedness to inform researchers about where certain genotypes/haplotypes might grow. If specific haplotypes are found in multiple locations one can dissect which strains are more likely to be pervasive. Likewise the variation and possible selection pressure one or more environmental variable(s) may have across many populations. Using a custom R script, we normalize 19 WorldClim environmental variables point sampled across locations. Variables with the smallest normalised distribution from the mean are likely to be a strong selection pressure species-wide. Variables with larger distribution could be random, but also could be associated with specific alleles. c) A cartoon representation of SNPs associated with environmental variables, calculated much the same way as a LOD with phenotype data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1144555808" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Predicting species distribution and detecting segregating SNPs by climate input&lt;/span&gt; a) MaxEnt produced probable distribution of Brachypodium in Australia using 80 geographic locations as presence only data correlated with WorldClim/BioClim variables. b) 80 Australian populations collected by the Borevitz lab of &lt;em&gt;B. distachyon&lt;/em&gt; complex to show the environmental relatedness of each population (no genetic data used). The vertical dendrogram describes environmental variable correlation across all populations. The horizontal dendrogram describes individual population clustering by their environmental relatedness to inform researchers about where certain genotypes/haplotypes might grow. If specific haplotypes are found in multiple locations one can dissect which strains are more likely to be pervasive. Likewise the variation and possible selection pressure one or more environmental variable(s) may have across many populations. Using a custom R script, we normalize 19 WorldClim environmental variables point sampled across locations. Variables with the smallest normalised distribution from the mean are likely to be a strong selection pressure species-wide. Variables with larger distribution could be random, but also could be associated with specific alleles. c) A cartoon representation of SNPs associated with environmental variables, calculated much the same way as a LOD with phenotype data.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F5.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F5.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2015/02/23/015495/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5.</span> <span class="caption-title">Predicting species distribution and detecting segregating SNPs by climate input</span><p id="p-38" class="first-child">a) MaxEnt produced probable distribution of Brachypodium in Australia using 80 geographic locations as presence only data correlated with WorldClim/BioClim variables. b) 80 Australian populations collected by the Borevitz lab of <em>B. distachyon</em> complex to show the environmental relatedness of each population (no genetic data used). The vertical dendrogram describes environmental variable correlation across all populations. The horizontal dendrogram describes individual population clustering by their environmental relatedness to inform researchers about where certain genotypes/haplotypes might grow. If specific haplotypes are found in multiple locations one can dissect which strains are more likely to be pervasive. Likewise the variation and possible selection pressure one or more environmental variable(s) may have across many populations. Using a custom R script, we normalize 19 WorldClim environmental variables point sampled across locations. Variables with the smallest normalised distribution from the mean are likely to be a strong selection pressure species-wide. Variables with larger distribution could be random, but also could be associated with specific alleles. c) A cartoon representation of SNPs associated with environmental variables, calculated much the same way as a LOD with phenotype data.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-15" class="subsection"><h5>How do neutral forces affect the genetic signal of climate adaptation?</h5><p id="p-39">The climatic niche breadth of a genetic lineage is the result of both adaptive and neutral forces. With limited migration, founder effects and isolation by distance, prevent our ability to separate the chance historical demographic effects on the entire genome from the adaptive genes. We must consider the genetic divergence and populations structure when selecting locations for intensive sampling. Recently founded populations may not have enough diversity to identify haplotype blocks associated with climate variables while diverged and structured populations may not allow adaptive loci to be partitioned from background variation. Geographic distance can often explain genomic isolation (eg Mantel test) within a species and can be the major factor at large, intercontinental scales leaving little ability to identify adaptive loci. Mantel tests can be performed at regional scales across clusters of sites to identify groups where isolation by distance is not a dominant factor. Partial mantel tests can be used to further account for genomic isolation by environment in addition to geographic distance. These background effects are essential to control for when looking to associate particular adaptive genetic loci with specific environmental variables.</p><p id="p-40">A more advanced approach than Mantel tests takes advantage of the software package Popgraph in R. It is a valuable tool for describing genetic relatedness between populations and environmental layers. The software works similarly to the PCoA algorithm, only it keeps the collection locations static on a map and uses visual tools to show genomic relatedness across geographic locations (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Garrick et al, 2013</a>). MaxEnt, Mantel tests, Popgraph, and other methods can provide comparisons of environments, geographic space, and genomic variation.</p></div><div id="sec-16" class="subsection"><h5>Identifying which climate variables</h5><p id="p-41">When overall climate is found to be associated with genomic variation, it is of interest to investigate which particular environmental variables are the drivers of natural selection. This can be done by clustering environmental variables across geographic locations. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5b</a> shows an ordered heat-map of 80 collections locations by 19 WorldClim environmental variables. The climate data is provided via Atlas of Living Australia (ALA) for points at sample locations. In this heatmap there are two large clades of environment types with 5 sub-groups. While there is similarity among environments at a local scale extending East and West, there is large split North to South. Three geographic regions are highlighted which represent three major climate types. The Mediterranean regions (green) cluster in climate space near desert regions (grey) while temperate regions (red) are separated. This is despite the fact that Mediterranean regions are geographically separated from desert regions.</p></div><div id="sec-17" class="subsection"><h5>Testing adaptation to climatic range</h5><p id="p-42">Typically common gardens and reciprocal transplant studies are used to test for local adaptation where the home genotype performs superior to the genotype from farther away (e.g. <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Clausen et al. 1940</a>). Provenance trials do this on a larger scale, evaluating phenotypes from a broader range of genotypes across many locations. These studies are massive and are certainly hampered by starting conditions and weather variation. As described above, one solution is to use growth chambers that can mimic natural climatic conditions, without weather noise. The SpectralPhenoClimatron, allow fitness traits to be measured in multiple target environments and using high-throughput phenotyping techniques.</p></div><div id="sec-18" class="subsection"><h5>Scanning for Adaptive Genetic loci controlling survival or fitness</h5><p id="p-43">Measuring fitness directly is difficult. It is the result of selection at many different life stages and often differs across environments and variation at many loci can contribute. An indirect way is to use the presence of a plant in a location as an observation that a particular combination of alleles can survive there. When a large collection of plants spanning a suitable range of climate variables are sequenced, one can use environment data as a fitness phenotype to and test if certain alleles associated. A genome wide association study scan can then be performed with climate data to identify adaptive loci. <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5c</a> shows a cartoon example of specific loci associating with Precipitation in the Warmest Quarter.</p></div></div><div id="sec-19" class="subsection"><h4>Future Collections</h4><p id="p-44">Extensive public collections exist, however many geographic regions and climate niche remain unrepresented or undersampled. Having those spaces filled would be beneficial for the whole Brachypodium community. As mentioned above there is a strong division in the current public collection between Eastern and Western European accessions. Hence, it would be advantageous to have collections across the Middle East and North Africa as it is highly possible that <em>B. distachyon</em> complex species were pushed south during the last ice age and extant lineages could be sources of new maternal lines for research. Central Southern Europe might provide a source of admixture populations between the Eastern and Western genotypes or completely new genotype groups. With genotype data, we can focus further collections on hybrid zones and polymorphic sites to increase the number of recombinant genotypes. These genotypes provide unique opportunities to study natural selection on adaptive traits. They provide natural genetic mapping resources to dissect complex adaptive traits. Admixed populations also inform the evolutionary history of Brachypodium and provide an opportunity to test natural selection on segregating variation.</p><p id="p-45">To study the natural variation of stress tolerance in <em>B. distachyon</em>, it would be good to have further collections from more extreme environments. This would include higher altitude/latitude environments such as Northern Europe and more arid environments in the Middle East and North Africa. Areas with multiple, recent introductions are also of interest as they may provide examples of strong selection on standing variation.</p></div></div></div><div class="section" id="sec-20"><h2 class="">Conclusion</h2><p id="p-46">Brachypodium has a large genetic diversity with its distribution spanning a wide range of climatic regions. Through Landscape Genomics we can use this diversity to predict the niche breadth and potential range size of a particular ecotype. We can also start to understand the genetic basis of fitness/adaptation to this niche. Genome Wide Association Studies utilises Brachypodium genetic diversity in combination with its broad phenotypic diversity to elucidate the genetic architecture of traits of interest and how this architecture varies in different climate types. Both techniques rely on a thorough understanding of the population structure of available collections with particular attention to sub-genomic divisions. Future developments in these techniques will rely heavily on the expansion of the collection of publicly available accessions and development of bioinformatic and statistical methods to increase the power and accuracy of both the Landscape Genomics and GWAS approaches. However, via recently developed techniques, the use of these approaches in Brachypodium has much potential for applications in crop design and ecological restoration projects for future climates.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We would like to acknowledge our collaborators: Location data and seed - Shuangshuang Liu of the Kent Bradford lab at UC Davis United states, Pilar Catalan from the University of Zaragoza in Huesca Spain, Luis Mur at Aberystwyth University at Aberystwyth Wales, Dave Garvin from University of Minnesota/USDA United States and John Vogel from JGI/UC Berkeley.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-59"><strong>More details can be found here:</strong></p><p id="p-60"><a href="http://borevitzlab.anu.edu.au/resources/populations/">http://borevitzlab.anu.edu.au/resources/populations/</a></p></li></ul></div><div class="section glossary"><h2 class="">Abbreviations</h2><dl id="def-list-1"><dt id="def-item-1">GBS<span class="def-item-dt-sep"></span></dt><dd>genotyping by sequencing<span class="def-item-dd-sep"></span></dd><dt id="def-item-2">GWAS<span class="def-item-dt-sep"></span></dt><dd>genome wide association studies<span class="def-item-dd-sep"></span></dd><dt id="def-item-3">QTL<span class="def-item-dt-sep"></span></dt><dd>quantitative trait loci<span class="def-item-dd-sep"></span></dd><dt id="def-item-4">MaxEnt<span class="def-item-dt-sep"></span></dt><dd>Maximum Entropy<span class="def-item-dd-sep"></span></dd><dt id="def-item-5">SNPs<span class="def-item-dt-sep"></span></dt><dd>single nucleotide polymorphisms</dd></dl></div><div class="section glossary"><h2 class="">Glossary</h2><dl id="def-list-2"><dt id="def-item-6">Accession<span class="def-item-dt-sep"></span></dt><dd>A collection of seeds from one location. This includes bulk collections and maternal descent lines<span class="def-item-dd-sep"></span></dd><dt id="def-item-7">Ecotype<span class="def-item-dt-sep"></span></dt><dd>An individual or group whose genetic distinction is strongly associated to an environment or type<span class="def-item-dd-sep"></span></dd><dt id="def-item-8">Genotype<span class="def-item-dt-sep"></span></dt><dd>This general term is used either to describe the genotype at a locus such as a SNP (AA, Aa, aa) or a background whole genome genotype which can have levels of species, subspecies, population genetic structure group, family, individual maternal line)<span class="def-item-dd-sep"></span></dd><dt id="def-item-9">Phenotype (qualitative and quantitative)<span class="def-item-dt-sep"></span></dt><dd>Measurable traits expressed by plants<span class="def-item-dd-sep"></span></dd><dt id="def-item-10">Population<span class="def-item-dt-sep"></span></dt><dd>Non-random mating between groups within a specified geographic space<span class="def-item-dd-sep"></span></dd><dt id="def-item-11">Subspecies<span class="def-item-dt-sep"></span></dt><dd>In this paper, subspecies is a major hierarchical cluster of genotype groups and their respective families and/or genotypes. Subspecies could interbreed but don’t in natural environment due to some sort of natural barrier</dd></dl></div><div class="section ref-list" id="ref-list-1"><h2 class="">Citations</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-015495v1.1"><div class="cit-metadata"><cite><span class="cit-comment">ALA [Internet] (Atlas of Living Australia) ala.org.au, [Cited 27Nov2014] Available from: <a href="http://avh.ala.org.au/occurrences/search?taxa=Brachypodium+distachyon">http://avh.ala.org.au/occurrences/search?taxa=Brachypodium+distachyon</a></span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">YS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Villhjálmsson</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Genome-wide association study of 107 phenotypes in <em>Arabidopsis thaliana</em> inbred lines</span>. <abbr class="cit-jnl-abbrev">Nature letters</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">465</span>:<span class="cit-fpage">627</span>-<span class="cit-lpage">31</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bletters%26rft.volume%253D465%26rft.spage%253D627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.3" data-doi="10.1111/j.1461-0248.2012.01796.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Banta</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">IM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerard</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilczek</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Climate envelope modeling reveals intraspecific relationships among phenology, niche breadth and potential range size in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">15</span>:<span class="cit-fpage">769</span>-<span class="cit-lpage">77</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DBanta%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D769%26rft.epage%253D777%26rft.atitle%253DClimate%2Benvelope%2Bmodelling%2Breveals%2Bintraspecific%2Brelationships%2Bamong%2Bflowering%2Bphenology%252C%2Bniche%2Bbreadth%2Band%2Bpotential%2Brange%2Bsize%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2012.01796.x%26rft_id%253Dinfo%253Apmid%252F22583905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2012.01796.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22583905&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.4" data-doi="10.1111/j.1469-8137.2011.03938.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barrero</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">JV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Swain</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garvin</span>  <span class="cit-name-given-names">DF</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Grain dormancy and light quality effects on germination in the model grass <em>Brachypodium distachyon</em></span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">193</span>:<span class="cit-fpage">376</span>-<span class="cit-lpage">86</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DBarrero%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D193%26rft.issue%253D2%26rft.spage%253D376%26rft.epage%253D386%26rft.atitle%253DGrain%2Bdormancy%2Band%2Blight%2Bquality%2Beffects%2Bon%2Bgermination%2Bin%2Bthe%2Bmodel%2Bgrass%2BBrachypodium%2Bdistachyon.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03938.x%26rft_id%253Dinfo%253Apmid%252F22039925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03938.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039925&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298591800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.5" data-doi="10.1371/journal.pgen.1001193"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brazelton</span>  <span class="cit-name-given-names">JN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">YS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lahner</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yukubova</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">A coastal cline in sodium accumulation in <em>Arabidopsis thaliana</em> is driven by natural variation of the sodium transporter AtHKT1;1</span>. <abbr class="cit-jnl-abbrev">PlosGenetics</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pgen.1001193</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBaxter%26rft.auinit1%253DI.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De1001193%26rft.epage%253De1001193%26rft.atitle%253DA%2Bcoastal%2Bcline%2Bin%2Bsodium%2Baccumulation%2Bin%2BArabidopsis%2Bthaliana%2Bis%2Bdriven%2Bby%2Bnatural%2Bvariation%2Bof%2Bthe%2Bsodium%2Btransporter%2BAtHKT1%253B1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001193%26rft_id%253Dinfo%253Apmid%252F21085628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085628&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.6" data-doi="10.1371/journal.pone.0115108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Betekhtin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hasterok</span>  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Reconstructing the Evolution of Brachypodium Genomes Using Comparative Chromosome Painting</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0115108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253D115108%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0115108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0115108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-015495v1.7"><div class="cit-metadata"><cite><span class="cit-comment">BioClim [Internet]. WorldClim - Global Climate Data, free climate data for ecology. [Cited 5Jan2015] Available from: <a href="http://www.bioclim.org/">http://www.bioclim.org/</a></span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.8" data-doi="10.1186/gb-2011-12-10-232"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">GP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">JO</span></span>. <span class="cit-article-title">Genome-wide association studies in plants: the missing heritability in in the field</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">12</span>:<span class="cit-fpage">232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBrachi%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D232%26rft.epage%253D232%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bplants%253A%2Bthe%2Bmissing%2Bheritability%2Bis%2Bin%2Bthe%2Bfield.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-10-232%26rft_id%253Dinfo%253Apmid%252F22035733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-10-232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22035733&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.9" data-doi="10.1016/j.pbi.2014.02.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">TB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sirault</span>  <span class="cit-name-given-names">XRR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rungrat</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">KD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trtilek</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">TraitCapture: genomic and environmental modelling of plant phenomic data</span>. <abbr class="cit-jnl-abbrev">Current Opinion in Plant Biology</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">18</span>:<span class="cit-fpage">73</span>-<span class="cit-lpage">9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BPlant%2BBiology%26rft.volume%253D18%26rft.spage%253D73%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2014.02.002%26rft_id%253Dinfo%253Apmid%252F24646691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2014.02.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24646691&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-015495v1.10"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Heywood</span>  <span class="cit-name-given-names">VH</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Brummitt</span>  <span class="cit-name-given-names">RK</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Culham</span>  <span class="cit-name-given-names">A</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Seberg</span>  <span class="cit-name-given-names">O</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Brummitt</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheek</span>  <span class="cit-name-given-names">MR</span></span>. <span class="cit-chapter-title">Hoplestigmataceae</span>. In: <span class="cit-ed"><span class="cit-name-surname">Heywood</span>  <span class="cit-name-given-names">VH</span></span>, <span class="cit-ed"><span class="cit-name-surname">Brummitt</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-ed"><span class="cit-name-surname">Culham</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-ed"><span class="cit-name-surname">Seberg</span>  <span class="cit-name-given-names">O</span></span>, editors. <span class="cit-source">Flowering Plant Families of the World</span>. <span class="cit-publ-loc">Kew</span>:<span class="cit-publ-name">Royal Botanic Gardens</span>; <span class="cit-pub-date">2007</span>. p. <span class="cit-fpage">167</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-015495v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clausen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keck</span>  <span class="cit-name-given-names">DD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hiesey</span>  <span class="cit-name-given-names">WM</span></span>. <span class="cit-source">Experimental studies on the nature of species</span>. <span class="cit-publ-loc">Washington D.C.</span>:<span class="cit-publ-name">Carnegie Institution of Washington</span>; <span class="cit-pub-date">1940</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.12" data-doi="10.1093/aob/mcr294"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Catalán</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hasterok</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mur</span>  <span class="cit-name-given-names">LAJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Langdon</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Evolution and taxonomic split of the model grass <em>Brachypodium distachyon</em></span>. <abbr class="cit-jnl-abbrev">Ann Bot</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">109</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">385</span>-<span class="cit-lpage">405</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn%2BBot%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcr294%26rft_id%253Dinfo%253Apmid%252F22213013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aob/mcr294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22213013&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.13" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">ES</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">A robust simple genotyping-by-sequencing (GBS) approach for high diversity species</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.14" data-doi="10.1126/science.1209271"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fournier-Level</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilczek</span>  <span class="cit-name-given-names">AM</span></span>. <span class="cit-article-title">A map of local adaptation in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2011</span> Oct 7;<span class="cit-vol">334</span>(<span class="cit-issue">6052</span>):<span class="cit-fpage">86</span>-<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFournier-Level%26rft.auinit1%253DA.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D86%26rft.epage%253D89%26rft.atitle%253DA%2BMap%2Bof%2BLocal%2BAdaptation%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209271%26rft_id%253Dinfo%253Apmid%252F21980109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzAyLzIzLzAxNTQ5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.15" data-doi="10.1111/mec.12318"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nason</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernández-Manjarrés</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dyer</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-article-title">Ecological co-associations influence species’ responses to past climatic change: an example from a Sonoran Desert bark beetle</span>. <abbr class="cit-jnl-abbrev">Molecular ecology</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">22</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">3345</span>-<span class="cit-lpage">61</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Becology%26rft.volume%253D22%26rft.spage%253D3345%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000320396100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.16" data-doi="10.2135/cropsci2007.04.0208"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garvin</span>  <span class="cit-name-given-names">DF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">YQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hasterok</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hazen</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mockler</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Development of genetic and genomic research resources for <em>Brachypodium distachyon</em>, a new model system for grass crop research</span>. <abbr class="cit-jnl-abbrev">Crop Science</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">48</span>:<span class="cit-fpage">S69</span>–<span class="cit-lpage">S84</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.volume%253D48%26rft.spage%253DS69%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2007.04.0208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2007.04.0208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254951500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.17" data-doi="10.1086/283244"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grime</span>  <span class="cit-name-given-names">JP</span></span>. <span class="cit-article-title">Evidence for the existence of three primary strategies in plants and its relevance to ecological and evolutionary theory</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>. <span class="cit-pub-date">1977</span>;<span class="cit-vol">111</span>(<span class="cit-issue">982</span>):<span class="cit-fpage">1169</span>-<span class="cit-lpage">94</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D111%26rft.spage%253D1169%26rft_id%253Dinfo%253Adoi%252F10.1086%252F283244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/283244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977EF94400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-015495v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Groombridge</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-source">World Atlas of Biodiversity: Earth’s Living Resources In the 21st Century</span>. <span class="cit-publ-loc">Berkeley, CA</span>:<span class="cit-publ-name">University of California Press</span>; <span class="cit-pub-date">2002</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.19" data-doi="10.1038/nbt0409-349"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Halperin</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephan</span>  <span class="cit-name-given-names">DA</span></span>. <span class="cit-article-title">SNP imputation in association studies</span>. <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">27</span>:<span class="cit-fpage">349</span>-<span class="cit-lpage">51</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHalperin%26rft.auinit1%253DE.%26rft.volume%253D27%26rft.issue%253D4%26rft.spage%253D349%26rft.epage%253D351%26rft.atitle%253DSNP%2Bimputation%2Bin%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0409-349%26rft_id%253Dinfo%253Apmid%252F19352374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0409-349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19352374&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264971800020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.20" data-doi="10.1126/science.1209244"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarymowycz</span>  <span class="cit-name-given-names">LB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperone</span>  <span class="cit-name-given-names">FG</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Adaptation to climate across the Arabidopsis thaliana genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2011</span>;<span class="cit-vol">334</span>(<span class="cit-issue">6052</span>):<span class="cit-fpage">83</span>-<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE1LzAyLzIzLzAxNTQ5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hartley</span>  <span class="cit-name-given-names">W.</span></span> <span class="cit-article-title">Studies on the origin, evolution and distribution of the <em>Gramineae</em></span>. <abbr class="cit-jnl-abbrev">V. The subfamily Festucoideae. Australian Journal of Botany</abbr>. <span class="cit-pub-date">1973</span>;<span class="cit-vol">21</span>:<span class="cit-fpage">201</span>-<span class="cit-lpage">34</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DV.%2BThe%2Bsubfamily%2BFestucoideae.%2BAustralian%2BJournal%2Bof%2BBotany%26rft.volume%253D21%26rft.spage%253D201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.22" data-doi="10.1186/s40665-014-0009-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Griffin</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Catullo</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rane</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">A framework for incorporating evolutionary genomics into biodiversity conservation and management</span>. <abbr class="cit-jnl-abbrev">Climate Change Responses</abbr> <span class="cit-pub-date">2015</span>;<span class="cit-vol">2</span>:<span class="cit-fpage">1</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s40665-014-0009-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClimate%2BChange%2BResponses%26rft.volume%253D2%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40665-014-0009-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40665-014-0009-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.23" data-doi="10.1038/ng.1042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">YS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Genome-wide patterns of genetic variation in worldwide <em>Arabidopsis thaliana</em> accessions from the RegMap panel</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">44</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">212</span>-<span class="cit-lpage">7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHorton%26rft.auinit1%253DM.%2BW.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D212%26rft.epage%253D216%26rft.atitle%253DGenome-wide%2Bpatterns%2Bof%2Bgenetic%2Bvariation%2Bin%2Bworldwide%2BArabidopsis%2Bthaliana%2Baccessions%2Bfrom%2Bthe%2BRegMap%2Bpanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1042%26rft_id%253Dinfo%253Apmid%252F22231484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22231484&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Natural variations and genome-wide association studies in crop plants</span>. <abbr class="cit-jnl-abbrev">Annual Review in Plant Biology</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">65</span>:<span class="cit-fpage">531</span>-<span class="cit-lpage">51</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bin%2BPlant%2BBiology%26rft.volume%253D65%26rft.spage%253D531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.25" data-doi="10.1038/ng.695"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Genome-wide association studies of 14 agronomic traits in rice landraces</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">42</span>:<span class="cit-fpage">961</span>-<span class="cit-lpage">7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D42%26rft.issue%253D11%26rft.spage%253D961%26rft.epage%253D967%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bof%2B14%2Bagronomic%2Btraits%2Bin%2Brice%2Blandraces.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.695%26rft_id%253Dinfo%253Apmid%252F20972439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20972439&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283540500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.26" data-doi="10.1111/j.1365-294X.2007.03442.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Joost</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruford</span>  <span class="cit-name-given-names">M W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Després</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conord</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erhardt</span>  <span class="cit-name-given-names">G</span></span> <span class="cit-etal">et al</span>. <span class="cit-article-title">A spatial analysis method (SAM) to detect candidate loci for selection: towards a landscape genomics approach to adaptation</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>. <span class="cit-pub-date">2007</span>;<span class="cit-vol">16</span>:<span class="cit-fpage">3955</span>–<span class="cit-lpage">69</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi: </span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1365-294X.2007.03442.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DJoost%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D18%26rft.spage%253D3955%26rft.epage%253D3969%26rft.atitle%253DA%2Bspatial%2Banalysis%2Bmethod%2B%2528SAM%2529%2Bto%2Bdetect%2Bcandidate%2Bloci%2Bfor%2Bselection%253A%2Btowards%2Ba%2Blandscape%2Bgenomics%2Bapproach%2Bto%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03442.x%26rft_id%253Dinfo%253Apmid%252F17850556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03442.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17850556&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249433400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.27" data-doi="10.1126/science.1109557"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeiss</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chew</span>  <span class="cit-name-given-names">EY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">JY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sackler</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haynes</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Complement Factor H polymorphism in age-related macular degeneration</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-pub-date">2005</span>;<span class="cit-vol">308</span>(<span class="cit-issue">5720</span>):<span class="cit-fpage">385</span>–<span class="cit-lpage">9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1109557%26rft_id%253Dinfo%253Apmid%252F15761122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDgvNTcyMC8zODUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8wMi8yMy8wMTU0OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.28" data-doi="10.1371/journal.pone.0000105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roycewicz</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">JO</span></span>. <span class="cit-article-title">Genetics of local adaptation in the laboratory: flowering time quantitative trait loci under geographic and seasonal conditions in Arabidopsis</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">1</span>(<span class="cit-issue">1</span>),<span class="cit-fpage">e105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D1%26rft.spage%253De105%26rft.epage%253De105%26rft.atitle%253DGenetics%2Bof%2Blocal%2Badaptation%2Bin%2Bthe%2Blaboratory%253A%2Bflowering%2Btime%2Bquantitative%2Btrait%2Bloci%2Bunder%2Bgeographic%2Band%2Bseasonal%2Bconditions%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000105%26rft_id%253Dinfo%253Apmid%252F17205109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17205109&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.29" data-doi="10.1073/pnas.1007431107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">JO</span></span>. <span class="cit-article-title">Association mapping of local climate-sensitive quantitative trait loci in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">PNAS</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">107</span>(<span class="cit-issue">49</span>): <span class="cit-fpage">21199</span>-<span class="cit-lpage">204</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1007431107%26rft_id%253Dinfo%253Apmid%252F21078970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQ5LzIxMTk5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDIvMjMvMDE1NDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.30" data-doi="10.1534/genetics.113.157628"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spokas</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-article-title">Genetic variation for life history sensitivity to seasonal warming in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">196</span>:<span class="cit-fpage">569</span>-<span class="cit-lpage">77</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.157628%26rft_id%253Dinfo%253Apmid%252F24281156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzIvNTY5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDIvMjMvMDE1NDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.31" data-doi="10.1073/pnas.1215985110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">GP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hash</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">HD</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Population genomics and genome-wide association studies of agroclimatic traits in sorghum</span>. <abbr class="cit-jnl-abbrev">PNAS</abbr>. <span class="cit-pub-date">2013</span>;<span class="cit-vol">110</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">453</span>-<span class="cit-lpage">8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1215985110%26rft_id%253Dinfo%253Apmid%252F23267105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMi80NTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNS8wMi8yMy8wMTU0OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pasam</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haseneyer</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Genome-wide association studies for agronomical traits in a world wide spring barley collection</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biology</abbr>. <span class="cit-pub-date">2012</span>;<span class="cit-vol">12</span>:<span class="cit-fpage">16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiology%26rft.volume%253D12%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.33" data-doi="10.5194/hess-11-1633-2007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peel</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finlayson</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">McMahon</span>  <span class="cit-name-given-names">TA</span></span>. <span class="cit-article-title">Updated world map of the Köppen-Geiger climate classification</span>, <abbr class="cit-jnl-abbrev">Hydrol. Earth Syst. Sci.</abbr>, <span class="cit-pub-date">2007</span>;<span class="cit-vol">11</span>:<span class="cit-fpage">1633</span>-<span class="cit-lpage">44</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHydrol.%2BEarth%2BSyst.%2BSci.%26rft.volume%253D11%26rft.spage%253D1633%26rft_id%253Dinfo%253Adoi%252F10.5194%252Fhess-11-1633-2007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5194/hess-11-1633-2007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000251516100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.34" data-doi="10.1111/j.0906-7590.2008.5203.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudík</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr>. <span class="cit-pub-date">2008</span>;<span class="cit-vol">31</span>:<span class="cit-fpage">161</span>–<span class="cit-lpage">175</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D31%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0906-7590.2008.5203.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0906-7590.2008.5203.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254499200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.35" data-doi="10.1016/j.ecolmodel.2005.03.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">RP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schapire</span>  <span class="cit-name-given-names">RE</span></span>. <span class="cit-article-title">Maximum entropy modeling of species geographic distributions</span>. <abbr class="cit-jnl-abbrev">Ecological Modelling</abbr>. <span class="cit-pub-date">2006</span>;<span class="cit-vol">190</span>(<span class="cit-issue">3-4</span>):<span class="cit-fpage">231</span>-<span class="cit-lpage">59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BModelling%26rft.volume%253D190%26rft.spage%253D231%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ecolmodel.2005.03.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ecolmodel.2005.03.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233859600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.36" data-doi="10.1371/journal.pgen.1000843"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">YS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anastasio</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mulyati</span>  <span class="cit-name-given-names">NW</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">The Scale of Population Structure in <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genetics</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">6</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">e1000843</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D6%26rft.spage%253De1000843%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000843%26rft_id%253Dinfo%253Apmid%252F20169178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000843&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20169178&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.37" data-doi="10.1073/pnas.1006993107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Revedin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aranguren</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becattini</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Longo</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marconi</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lippi</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Thirty thousand-year-old evidence of plant food processing</span>. <abbr class="cit-jnl-abbrev">PNAS</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">107</span>(<span class="cit-issue">44</span>):<span class="cit-fpage">18815</span>-<span class="cit-lpage">9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1006993107%26rft_id%253Dinfo%253Apmid%252F20956317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQ0LzE4ODE1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTUvMDIvMjMvMDE1NDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.38" data-doi="10.1007/s12155-009-9069-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manzaneda</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rey</span>  <span class="cit-name-given-names">PJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amasino</span>  <span class="cit-name-given-names">RM</span></span>. <span class="cit-article-title">Natural Variation of Flowering Time and Vernalization Responsiveness in <em>Brachypodium distachyon</em></span>. <abbr class="cit-jnl-abbrev">Bioenerg. Res</abbr>. <span class="cit-pub-date">2010</span>;<span class="cit-vol">3</span>:<span class="cit-fpage">38</span>-<span class="cit-lpage">46</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioenerg.%2BRes%26rft.volume%253D3%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12155-009-9069-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12155-009-9069-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.39" data-doi="10.1046/j.1365-294X.2000.01122.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharbel</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haubold</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T.</span></span> <span class="cit-article-title">Genetic isolation by distance in <em>Arabidopsis thaliana</em>: biogeography and postglacial colonization of Europe</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>. <span class="cit-pub-date">2000</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">2109</span>-<span class="cit-lpage">18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSharbel%26rft.auinit1%253DT.%2BF.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D2109%26rft.epage%253D2118%26rft.atitle%253DGenetic%2Bisolation%2Bby%2Bdistance%2Bin%2BArabidopsis%2Bthaliana%253A%2Bbiogeography%2Band%2Bpostglacial%2Bcolonization%2Bof%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2000.01122.x%26rft_id%253Dinfo%253Apmid%252F11123622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2000.01122.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11123622&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166112700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.40" data-doi="10.1111/j.1365-294X.2005.02437.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Storz</span>  <span class="cit-name-given-names">JF</span></span>. <span class="cit-article-title">Using genome scans of DNA polymorphism to infer adaptive population divergence</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-pub-date">2005</span>;<span class="cit-vol">14</span>:<span class="cit-fpage">671</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DStorz%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D671%26rft.epage%253D688%26rft.atitle%253DUsing%2Bgenome%2Bscans%2Bof%2BDNA%2Bpolymorphism%2Bto%2Binfer%2Badaptive%2Bpopulation%2Bdivergence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02437.x%26rft_id%253Dinfo%253Apmid%252F15723660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02437.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15723660&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227132300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.41" data-doi="10.1186/1471-2229-9-88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tuna</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Budak</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huo</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">YQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steinwand</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-article-title">Development of SSR markers and analysis of diversity in Turkish populations of <em>Brachypodium distachyon</em></span>. <abbr class="cit-jnl-abbrev">BMC Plant Biology</abbr>. <span class="cit-pub-date">2009</span>;<span class="cit-vol">9</span>:<span class="cit-fpage">88</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiology%26rft.volume%253D9%26rft.spage%253D88%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-88%26rft_id%253Dinfo%253Apmid%252F19594938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19594938&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2015%2F02%2F23%2F015495.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-015495v1.42" data-doi="10.1038/ncomms5737"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al</span>. <span class="cit-article-title">Metabolome-based genome-wide association study of maize kernel leads to novel biochemical insights</span>. <abbr class="cit-jnl-abbrev">Nature communications</abbr>. <span class="cit-pub-date">2014</span>;<span class="cit-vol">5</span>(<span class="cit-issue">3438</span>):<span class="cit-fpage">1</span>-<span class="cit-lpage">10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5737&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-015495v1.43"><div class="cit-metadata"><cite><span class="cit-comment">WorldClim [Internet], WorldClim - Global Climate Data, free climate data for ecology, [cited 5Jan2014] Available from: <a href="http://www.worldclim.org/">http://www.worldclim.org/</a></span></cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/015495v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F015495v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/010843v2" title="Sharing and specificity of co-expression networks across 35 human tissues" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/011361v2" title="The developmental transcriptome of contrasting Arctic charr (Salvelinus alpinus) morphs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 23, 2015.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/015495v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/015495v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/015495v1" class="active" data-icon-position="" data-hide-link-title="0">Genomic Diversity and Climate Adaptation in Brachypodium</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/3950" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zjBJuImmQBnZ9Bw_BGcGR1FY_unNa20ESFTAQKWH81w" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="3950" id="node3950--2" data-pisa="biorxiv;015495v1" data-pisa-master="biorxiv;015495" data-seqnum="3950" data-apath="/biorxiv/early/2015/02/23/015495.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Genomic Diversity and Climate Adaptation in Brachypodium</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Pip</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jared</span> <span class="nlm-surname">Streich</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Justin</span> <span class="nlm-surname">Borevitz</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">015495; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/015495 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/015495v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/015495v1&amp;title=Genomic%20Diversity%20and%20Climate%20Adaptation%20in%20Brachypodium" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/015495v1&amp;title=Genomic%20Diversity%20and%20Climate%20Adaptation%20in%20Brachypodium" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/015495v1&amp;text=Genomic%20Diversity%20and%20Climate%20Adaptation%20in%20Brachypodium" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/015495v1&amp;title=Genomic%20Diversity%20and%20Climate%20Adaptation%20in%20Brachypodium" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/015495v1&amp;t=Genomic%20Diversity%20and%20Climate%20Adaptation%20in%20Brachypodium" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/015495v1&amp;title=Genomic%20Diversity%20and%20Climate%20Adaptation%20in%20Brachypodium" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/015495v1&amp;title=Genomic%20Diversity%20and%20Climate%20Adaptation%20in%20Brachypodium" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="3950" id="node3950--3" data-pisa="biorxiv;015495v1" data-pisa-master="biorxiv;015495" data-seqnum="3950" data-apath="/biorxiv/early/2015/02/23/015495.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Genomic Diversity and Climate Adaptation in Brachypodium</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Pip</span> <span class="nlm-surname">Wilson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jared</span> <span class="nlm-surname">Streich</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Justin</span> <span class="nlm-surname">Borevitz</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">015495; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/015495 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Genomic%20Diversity%20and%20Climate%20Adaptation%20in%20Brachypodium&amp;rft.volume&amp;rft.issue&amp;rft.date=2015-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Wilson%2C+Pip&amp;rft.au=Streich%2C+Jared&amp;rft.au=Borevitz%2C+Justin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/3950/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/3950/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/3950/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/3950/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/3950/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/3950/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/3950/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/3950/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/3950/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/3950/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/3950/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/3950/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/015495v1&amp;count=horizontal&amp;via=&amp;text=Genomic%20Diversity%20and%20Climate%20Adaptation%20in%20Brachypodium&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/015495v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/015495v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/015495v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/plant-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Plant Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
