<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/02/04/834234.atom" />
<meta name="HW.pisa" content="biorxiv;834234v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A tradeoff between robustness to environmental fluctuations and speed of evolution" />
<meta name="DC.Identifier" content="10.1101/834234" />
<meta name="DC.Date" content="2020-02-04" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Organisms must cope with both short- and long-term environmental changes to persist. In this study we investigated theoretically if life histories trade off between their robustness to short-term environmental perturbations and their ability to evolve directional trait changes. With mathematical models and stochastic individual-based simulations, we could confirm the tradeoff, at least for comparisons between standardized life histories. Offspring dormancy and high adult survival rates allowed large population sizes to be maintained in face of interannual environmental fluctuations but limited the speed of trait evolution with ongoing environmental change. In contrast, precocious offspring maturation and short-living adults promoted evolvability while lowering demographic ro-bustness. This tradeoff had immediate consequences on extinction dynamics in variable environments. High evolvability allowed to cope best with long-lasting gradual environmental change, but came at the expense of more pronounced population declines from environmental variability and during the initial phase of environmental change." />
<meta name="DC.Contributor" content="Max Schmid" />
<meta name="DC.Contributor" content="Maria Paniw" />
<meta name="DC.Contributor" content="Maarten Postuma" />
<meta name="DC.Contributor" content="Arpat Ozgul" />
<meta name="DC.Contributor" content="Frédéric Guillaume" />
<meta name="article:published_time" content="2020-02-04" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A tradeoff between robustness to environmental fluctuations and speed of evolution" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Organisms must cope with both short- and long-term environmental changes to persist. In this study we investigated theoretically if life histories trade off between their robustness to short-term environmental perturbations and their ability to evolve directional trait changes. With mathematical models and stochastic individual-based simulations, we could confirm the tradeoff, at least for comparisons between standardized life histories. Offspring dormancy and high adult survival rates allowed large population sizes to be maintained in face of interannual environmental fluctuations but limited the speed of trait evolution with ongoing environmental change. In contrast, precocious offspring maturation and short-living adults promoted evolvability while lowering demographic robustness. This tradeoff had immediate consequences on extinction dynamics in variable environments. High evolvability allowed to cope best with long-lasting gradual environmental change, but came at the expense of more pronounced population declines from environmental variability and during the initial phase of environmental change.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;834234v2" />
<meta name="citation_id" content="834234v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/834234v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/834234v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/834234v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234.full.pdf" />
<meta name="citation_doi" content="10.1101/834234" />
<meta name="citation_num_pages" content="70" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="834234" />
<meta name="citation_author" content="Max Schmid" />
<meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7197-4775" />
<meta name="citation_author" content="Maria Paniw" />
<meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich" />
<meta name="citation_author_institution" content="Ecological and Forestry Applications Research Centre (CREAF), Campus de Bellaterra (UAB) Edifici C" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1949-4448" />
<meta name="citation_author" content="Maarten Postuma" />
<meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich" />
<meta name="citation_author_institution" content="Plant Ecology and Nature Conservation Group, Wageningen University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4317-2721" />
<meta name="citation_author" content="Arpat Ozgul" />
<meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7477-2642" />
<meta name="citation_author" content="Frédéric Guillaume" />
<meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich" />
<meta name="citation_author_email" content="frederic.guillaume@ieu.uzh.ch" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0874-0081" />
<meta name="citation_reference" content="Aitken, S. N., Yeaman, S., Holliday, J. a., Wang, T., and Curtis-McLane, S. (2008). Adaptation, migration or extirpation: Climate change outcomes for tree populations. Evolutionary Applications, 1(1):95–111." />
<meta name="citation_reference" content="Barfield, M., Holt, R. D., and Gomulkiewicz, R. (2011). Evolution in stage-structured populations. The American Naturalist, 177(4)." />
<meta name="citation_reference" content="Beckman, N., Bullock, J. M., and Salguero-Gómez, R. (2018). High dispersal ability is related to fast life history strategies. Journal of Ecology, 106(March):1349–1362." />
<meta name="citation_reference" content="Bell, G. and Gonzalez, A. (2009). Evolutionary rescue can prevent extinction following environmental change. Ecology Letters, 12(9):942–948." />
<meta name="citation_reference" content="Boulding, E. G. and Hay, T. (2001). Genetic and demographic parameters determining population persistence after a discrete change in the environment. Heredity, 86(3):313–324." />
<meta name="citation_reference" content="Boyce, M. S., Haridas, C. V., Lee, C. T., Boggs, C. L., Bruna, E. M., Coulson, T., Doak, D., Drake, J. M., Gaillard, J. M., Horvitz, C. C., Kalisz, S., Kendall, B. E., Knight, T., Mastrandrea, M., Menges, E. S., Morris, W. F., Pfister, C. A., and Tuljapurkar, S. (2006). Demography in an increasingly variable world. Trends in Ecology and Evolution, 21(3):141–148." />
<meta name="citation_reference" content="Buoro, M. and Carlson, S. M. (2014). Life-history syndromes: Integrating dispersal through space and time. Ecology Letters, 17(6):756–767." />
<meta name="citation_reference" content="Bürger, R. (1999). Evolution of genetic variability and the advantage of sex and recombination in changing environments. Genetics, 153(2):1055–1069." />
<meta name="citation_reference" content="Bürger, R. and Lynch, M. (1995). Evolution and extinction in a changing environment: A quantitative-genetic analysis. Evolution, 49(1):151–163." />
<meta name="citation_reference" content="Carlson, S. M., Cunningham, C. J., and Westley, P. A. (2014). Evolutionary rescue in a changing world. Trends in Ecology and Evolution, 29(9):521–530." />
<meta name="citation_reference" content="Caswell, H. (2001). Matrix Population Models. Construction, Analysis, and Interpretation. Sinauer Associates, Inc., Sunderland, MA., 2nd edition." />
<meta name="citation_reference" content="Caswell, H., Takada, T., and Hunter, C. M. (2004). Sensitivity analysis of equilibrium in density-dependent matrix population models. Ecology Letters, 7(5):380–387." />
<meta name="citation_reference" content="Cayuela, H., Joly, P., Schmidt, B. R., Pichenot, J., Bonnaire, E., Priol, P., Peyronel, O., Laville, M., and Besnard, A. (2017). Life history tactics shape amphibians’ demographic responses to the North Atlantic Oscillation. Global Change Biology, 23(11):4620–4638." />
<meta name="citation_reference" content="Charlesworth, B. (1994). Evolution in age-structured populations. Cambridge University Press, Cambridge, 2nd edition." />
<meta name="citation_reference" content="Chevin, L.-M. (2013). Genetic constraints on adaptation to a changing environment. Evolution, 67(3):708–721." />
<meta name="citation_reference" content="Chevin, L.-M., Cotto, O., and Ashander, J. (2017). Stochastic evolutionary demography under a fluctuating optimum phenotype. The American Naturalist, 190(6):000–000." />
<meta name="citation_reference" content="Chevin, L.-M., Lande, R., and Mace, G. M. (2010). Adaptation, plasticity, and extinction in a changing environment: Towards a predictive theory. PLoS Biology, 8(4):e1000357." />
<meta name="citation_reference" content="Cotto, O., Sandell, L., Chevin, L.-M., and Ronce, O. (2019). Maladaptive shifts in life history in a changing environment. The American Naturalist, 194(2):000–000." />
<meta name="citation_reference" content="Cotto, O., Schmid, M., and Guillaume, F. (2020). Nemo-age: Spatially explicit simulations of eco-evolutionary dynamics in stage-structured populations under changing environments. submitted." />
<meta name="citation_reference" content="Cotto, O., Wessely, J., Georges, D., Klonner, G., Schmid, M., Dullinger, S., Thuiller, W., and Guillaume, F. (2017). A dynamic eco-evolutionary model predicts slow response of alpine plants to climate warming. Nature Communications, 8(May):15399." />
<meta name="citation_reference" content="Dalgleish, H. J., Koons, D. N., and Adler, P. B. (2010). Can life-history traits predict the response of forb populations to changes in climate variability? Journal of Ecology, 98(1):209–217." />
<meta name="citation_reference" content="Doak, D. F. and Morris, W. F. (2010). Demographic compensation and tipping points in climate-induced range shifts. Nature, 467(7318):959–962." />
<meta name="citation_reference" content="Dullinger, S., Gattringer, A., Thuiller, W., Moser, D., Zimmermann, N. E., Guisan, A., Willner, W., Plutzar, C., Leitner, M., Mang, T., Caccianiga, M., Dirnböck, T., Ertl, S., Fischer, A., Lenoir, J., Svenning, J.-C., Psomas, A., Schmatz, D. R., Silc, U., Vittoz, P., and Hülber, K. (2012). Extinction debt of high-mountain plants under twenty-first-century climate change. Nature Climate Change, 2(8):619–622." />
<meta name="citation_reference" content="Duputié, A., Massol, F., Chuine, I., Kirkpatrick, M., and Ronce, O. (2012). How do genetic correlations affect species range shifts in a changing environment? Ecology Letters, 15(3):251–259." />
<meta name="citation_reference" content="Duputié, A., Rutschmann, A., Ronce, O., and Chuine, I. (2015). Phenological plasticity will not help all species adapt to climate change. Global Change Biology, 21(8):3062–3073." />
<meta name="citation_reference" content="Ellner, S. P. and Hairston, N. G. (1994). Role of overlapping generations in maintaining genetic variation in a fluctuating environment. The American Naturalist, 143(3):403–417." />
<meta name="citation_reference" content="Engen, S., Lande, R., and Saether, B.-E. (2009). Reproductive value and fluctuating selection in an age-structured population. Genetics, 183(2):629–637." />
<meta name="citation_reference" content="Engen, S., Lande, R., and Saether, B.-E. (2011). Evolution of a plastic quantitative trait in an age-structured population in a fluctuating environment. Evolution, 65(10):2893–2906." />
<meta name="citation_reference" content="Engen, S., Lande, R., and Sæther, B.-E. (2013a). A quantitative genetic model of r- and K-selection in a fluctuating population. The American Naturalist, 181(6):725–736." />
<meta name="citation_reference" content="Engen, S., Lande, R., Sæther, B.-E., and Weimerskirch, H. (2005). Extinction in relation to demographic and environmental stochasticity in age-structured models. Mathematical Biosciences, 195(2):210–227." />
<meta name="citation_reference" content="Engen, S., Sæther, B.-E., Armitage, K. B., Blumstein, D. T., Clutton-Brock, T. H., Dobson, F. S., Festa-Bianchet, M., Oli, M. K., and Ozgul, A. (2013b). Estimating the effect of temporally autocorrelated environments on the demography of density-independent age-structured populations. Methods in Ecology and Evolution, 4(6):573–584." />
<meta name="citation_reference" content="Evans, M. E. K. and Dennehy, J. J. (2005). Germ banking : Bet-hedging and variable release from egg and seed dormancy. The Quarterly Review of Biology, 80(4):431–451." />
<meta name="citation_reference" content="Gaillard, J., Yoccoz, N. G., Lebreton, J., Bonenfant, C., Devillard, S., Loison, A., Pontier, D., and Allaine, D. (2005). Generation time: A reliable metric to measure life-history variation among mammalian populations. The American Naturalist, 166(1):119–123." />
<meta name="citation_reference" content="Gaillard, J.-M., Pontier, D., Allainé, D., Lebreton, J. D., Trouvilliez, J., and Clobert, J. (1989). An analysis of demographic tactics in birds and mammals. Oikos, 56(1):59–76." />
<meta name="citation_reference" content="Gomulkiewicz, R. and Holt, R. D. (1995). When does evolution by natural selection prevent extinction? Evolution, 49(1):201–207." />
<meta name="citation_reference" content="Gomulkiewicz, R. and Houle, D. (2009). Demographic and genetic constraints on evolution. The American Naturalist, 174(6)." />
<meta name="citation_reference" content="Grant, A. and Benton, T. G. (2000). Elasticity analysis for density-dependent populations in stochastic environments. Ecology, 81(3):680–693." />
<meta name="citation_reference" content="Grant, A. and Benton, T. G. (2003). Density-dependent populations require density-dependent elasticity analysis: An illustration using the LPA model of Tribolium. Journal of Animal Ecology, 72(1):94–105." />
<meta name="citation_reference" content="Guillaume, F. and Rougemont, J. (2006). Nemo: An evolutionary and population genetics programming framework. Bioinformatics, 22(20):2556–2557." />
<meta name="citation_reference" content="Hairston, N. G. and De Stasio Jr., B. T. (1988). Rate of evolution slowed by a dormant propagule pool. Nature, 336:239–242." />
<meta name="citation_reference" content="Hallett, L. M., Farrer, E. C., Suding, K. N., Mooney, H. A., and Hobbs, R. J. (2018). Tradeoffs in demographic mechanisms underlie differences in species abundance and stability. Nature Communications, 9(1):1–6." />
<meta name="citation_reference" content="Hansen, T. F. and Houle, D. (2008). Measuring and comparing evolvability and constraint in multivariate characters. Journal of Evolutionary Biology, 21(5):1201–1219." />
<meta name="citation_reference" content="Hansen, T. F., Solvin, T. M., and Pavlicev, M. (2019). Predicting evolutionary potential: A numerical test of evolvability measures. Evolution, 73(4):689–703." />
<meta name="citation_reference" content="Houle, D. (1992). Comparing evolvability and variability of quantitative traits. Genetics, 130(1):195–204." />
<meta name="citation_reference" content="Koons, D. N., Metcalf, C. J. E., and Tuljapurkar, S. (2008). Evolution of delayed reproduction in uncertain environments: A life-history perspective. The American Naturalist, 172(6):797–805." />
<meta name="citation_reference" content="Kopp, M. and Matuszewski, S. (2014). Rapid evolution of quantitative traits: Theoretical perspectives. Evolutionary Applications, 7(1):169–191." />
<meta name="citation_reference" content="Kuparinen, A., Savolainen, O., and Schurr, F. M. (2010). Increased mortality can promote evolutionary adaptation of forest trees to climate change. Forest Ecology and Management, 259(5):1003–1008." />
<meta name="citation_reference" content="Lande, R. (1982). A quantitative genetic theory of life history evolution. Ecology, 63(3):607–615." />
<meta name="citation_reference" content="Lande, R. (1993). Risks of population extinction from demographic and environmental stochasticity and random catastrophes. The American Naturalist, 142(6):911–927." />
<meta name="citation_reference" content="Lande, R. and Orzack, S. H. (1988). Extinction dynamics of age-structured populations in a fluctuating environment. Proceedings of the National Academy of Sciences of the United States of America, 85(19):7418–21." />
<meta name="citation_reference" content="Lanfear, R., Kokko, H., and Eyre-Walker, A. (2014). Population size and the rate of evolution. Trends in Ecology and Evolution, 29(1):33–41." />
<meta name="citation_reference" content="Lynch, M., Gabriel, W., and Wood, A. M. (1991). Adaptive and demographic responses of plankton populations to environmental change. Limnology and Oceanography, 36(7):1301–1312." />
<meta name="citation_reference" content="citation_title=Biotic Interactions and Global Change;citation_pages=234-250;citation_year=1993" />
<meta name="citation_reference" content="Marshall, D. J., Burgess, S. C., and Connallon, T. (2016). Global change, life-history complexity and the potential for evolutionary rescue. Evolutionary Applications, 9(9):1189–1201." />
<meta name="citation_reference" content="McDonald, J. L., Franco, M., Townley, S., Ezard, T. H., Jelbert, K., and Hodgson, D. J. (2017). Divergent demographic strategies of plants in variable environments. Nature Ecology and Evolution, 1(2)." />
<meta name="citation_reference" content="Morris, W. F., Pfister, C. A., Tuljapurkar, S., Haridas, C. V., Boggs, C. L., Boyce, M. S., Bruna, E. M., Church, D. R., Coulson, T., Doak, D. F., Forsyth, S., Gaillard, J.-m., Horvitz, C. C., Kalisz, S., Kendall, B. E., Knight, T. M., Lee, C. T., Menges, E. S., Chirrakal, V., Boggs, C. L., Boyce, M. S., Bruna, E. M., Church, D. R., Coulson, T., Doak, D. F., Forsyth, S., Gaillard, J.-m., Horvitz, C. C., Kalisz, S., Kendall, E., Knight, T. M., Lee, C. T., and Menges, E. S. (2008). Longevity can buffer plant and animal populations against changing climatic variability. Ecology, 89(1):19–25." />
<meta name="citation_reference" content="Orive, M. E., Barfield, M., Fernandez, C., and Holt, R. D. (2017). Effects of clonal reproduction on evolutionary lag and evolutionary rescue. The American Naturalist, 190(4)." />
<meta name="citation_reference" content="Orzack, S. H. and Tuljapurkar, S. (1989). Population Dynamics in Variable Environments. VII. The Demography and Evolution of Iteroparity. The American Naturalist, 133(6):901–923." />
<meta name="citation_reference" content="Osmond, M. M., Otto, S. P., and Klausmeier, C. A. (2017). When predators help prey adapt and persist in a changing environment. The American Naturalist, 190(1):000–000." />
<meta name="citation_reference" content="Paniw, M., Ozgul, A., and Salguero-Gómez, R. (2017). Interactive life-history traits predict sensitivity of plants and animals to temporal autocorrelation. Ecology Letters, pages 275–286." />
<meta name="citation_reference" content="Parmesan, C. (2006). Ecological and evolutionary responses to recent climate change. Annual Review of Ecology, Evolution, and Systematics, 37(1):637–669." />
<meta name="citation_reference" content="Petit, R. J. and Hampe, A. (2006). Some evolutionary consequences of being a tree. Annual Review of Ecology, Evolution, and Systematics, 37(May):187–214." />
<meta name="citation_reference" content="Polechová, J., Barton, N., and Marion, G. (2009). Species’ range: Adaptation in space and time. The American Naturalist, 174(5):E186–-E204." />
<meta name="citation_reference" content="R Development Core Team (2008). R: a language and environment for statistical computing." />
<meta name="citation_reference" content="Sæther, B.-E., Coulson, T., Grøtan, V., Engen, S., Altwegg, R., Armitage, K. B., Barbraud, C., Becker, P. H., Blumstein, D. T., Dobson, F. S., Festa-Bianchet, M., Gaillard, J.-M., Jenkins, A., Jones, C., Nicoll, M. A. C., Norris, K., Oli, M. K., Ozgul, A., and Weimerskirch, H. (2013). How life history influences population dynamics in fluctuating environments. The American Naturalist, 182(6):743–759." />
<meta name="citation_reference" content="Salguero-Gómez, R. (2017). Applications of the fast–slow continuum and reproductive strategy framework of plant life histories. New Phytologist, 213(4):1618–1624." />
<meta name="citation_reference" content="Salguero-Gómez, R., Jones, O. R., Blomberg, S. P., Hodgson, D. J., Zuidema, P. A., and Kroon, H. D. (2017). Fast–slow continuum and reproductive strategies structure plant life-history variation worldwide. Proceedings of the National Academy of Sciences, 114(45):E9753–E9753." />
<meta name="citation_reference" content="Sasaki, A. and Ellner, S. (1997). Quantitative genetic variance maintained by fluctuating selection with overlapping generations : Variance components and covariances. Evolution, 51(3):682–696." />
<meta name="citation_reference" content="Stearns, S. C. (1983). The influence of size and phylogeny on patterns of covariation among life-history traits in the mammals. Oikos, 41(2):173–187." />
<meta name="citation_reference" content="Stocker, T., Qin, D., Plattner, G.-K., Tignor, M., Allen, S., Boschung, J., Nauels, A., Xia, Y., Bex, V., and Midgley, P. (2013). IPCC, 2013: Climate Change 2013: The Physical Science Basis. Contribution of Working Group I to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change. Technical report, IPCC, Cambridge University Press, Cambridge, United Kingdom and New York, NY, USA." />
<meta name="citation_reference" content="Stubben, C. J. and Milligan, B. G. (2007). Estimating and Analyzing Demographic Models Using the popbio Package in R. Journal of Statistical Software, 22:11." />
<meta name="citation_reference" content="Svardal, H., Rueffler, C., and Hermisson, J. (2015). A general condition for adaptive genetic polymorphism in temporally and spatially heterogeneous environments. Theoretical Population Biology, 99:76–97." />
<meta name="citation_reference" content="Takada, T. and Nakajima, H. (1998). Theorems on the invasion process in stage-structured populations with density-dependent dynamics. Journal of Mathematical Biology, 36(5):497–514." />
<meta name="citation_reference" content="Templeton, A. R. and Levin, D. A. (1979). Evolutionary consequences of seed pools. The American Naturalist, 114(2):232–249." />
<meta name="citation_reference" content="Tuljapurkar, S. (1990). Population Dynamics in Variable Environments. Springer Verlag, Berlin Heidelberg." />
<meta name="citation_reference" content="Tuljapurkar, S., Gaillard, J. M., and Coulson, T. (2009). From stochastic environments to life histories and back. Philosophical Transactions of the Royal Society B: Biological Sciences, 364(1523):1499–1509." />
<meta name="citation_reference" content="Tuljapurkar, S., Horvitz, C. C., and Pascarella, J. B. (2003). The many growth rates and elasticities of populations in random environments. The American Naturalist, 162(4):489–502." />
<meta name="citation_reference" content="Ummenhofer, C. C. and Meehl, G. A. (2017). Extreme weather and climate events with ecological relevance: a review. Philosophical Transactions of the Royal Society B: Biological Sciences, 372(1723):20160135." />
<meta name="citation_reference" content="Van De Pol, M., Vindenes, Y., Sæther, B.-E., Engen, S., Ens, B. J., Oosterbeek, K., and Tinbergen, J. M. (2010). Effects of climate change and variability on population dynamics in a long-lived shorebird. Ecology, 91(4):1192–1204." />
<meta name="citation_reference" content="Van Mantgem, P. J. and Stephenson, N. L. (2007). Apparent climatically induced increase of tree mortality rates in a temperate forest. Ecology Letters, 10(10):909–916." />
<meta name="citation_reference" content="Vander Wal, E., Garant, D., Festa-Bianchet, M., and Pelletier, F. (2013). Evolutionary rescue in vertebrates: Evidence, applications and uncertainty. Philosophical Transactions of the Royal Society B: Biological Sciences, 368(1610):20120090." />
<meta name="citation_reference" content="Vázquez, D. P., Gianoli, E., Morris, W. F., and Bozinovic, F. (2015). Ecological and evolutionary impacts of changing climatic variability. Biological Reviews, 92:22–42." />
<meta name="citation_reference" content="Via, S. and Lande, R. (1985). Genotype-environment interaction and the evolution of phenotypic plasticity. Evolution, 39(3):505–522." />
<meta name="citation_reference" content="Willi, Y. and Hoffmann, A. A. (2009). Demographic factors and genetic variation influence population persistence under environmental change. Journal of Evolutionary Biology, 22(1):124–133." />
<meta name="citation_reference" content="Yamamichi, M., Jr, N. G. H., Rees, M., and Ellner, S. P. (2019). Rapid evolution with generation overlap : The double-edged effect of dormancy. Theoretical Ecology, 12(2):179–195." />
<meta name="citation_reference" content="Zeineddine, M. and Jansen, V. A. A. (2009). To age, to die: Parity, evolutionary tracking and Cole’s paradox. Evolution, 63(6):1498–1507." />
<meta name="twitter:title" content="A tradeoff between robustness to environmental fluctuations and speed of evolution" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Organisms must cope with both short- and long-term environmental changes to persist. In this study we investigated theoretically if life histories trade off between their robustness to short-term environmental perturbations and their ability to evolve directional trait changes. With mathematical models and stochastic individual-based simulations, we could confirm the tradeoff, at least for comparisons between standardized life histories. Offspring dormancy and high adult survival rates allowed large population sizes to be maintained in face of interannual environmental fluctuations but limited the speed of trait evolution with ongoing environmental change. In contrast, precocious offspring maturation and short-living adults promoted evolvability while lowering demographic ro-bustness. This tradeoff had immediate consequences on extinction dynamics in variable environments. High evolvability allowed to cope best with long-lasting gradual environmental change, but came at the expense of more pronounced population declines from environmental variability and during the initial phase of environmental change." />
<meta name="og-title" property="og:title" content="A tradeoff between robustness to environmental fluctuations and speed of evolution" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/834234v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Organisms must cope with both short- and long-term environmental changes to persist. In this study we investigated theoretically if life histories trade off between their robustness to short-term environmental perturbations and their ability to evolve directional trait changes. With mathematical models and stochastic individual-based simulations, we could confirm the tradeoff, at least for comparisons between standardized life histories. Offspring dormancy and high adult survival rates allowed large population sizes to be maintained in face of interannual environmental fluctuations but limited the speed of trait evolution with ongoing environmental change. In contrast, precocious offspring maturation and short-living adults promoted evolvability while lowering demographic ro-bustness. This tradeoff had immediate consequences on extinction dynamics in variable environments. High evolvability allowed to cope best with long-lasting gradual environmental change, but came at the expense of more pronounced population declines from environmental variability and during the initial phase of environmental change." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-02-04" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/834234v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/834234v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/834234v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/834234v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1134918" />
    <title>A tradeoff between robustness to environmental fluctuations and speed of evolution | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__AQm9HArb2Y_D7UzgraiLIwAB7088fIq2iZD0Uigie_s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"gHw0npZJG59OasEAfdljs1BNLyh_TRd5Me3bcouJFCk","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1134918","apath":"\/biorxiv\/early\/2020\/02\/04\/834234.atom","pisa":"biorxiv;834234v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;834234v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/834234v2","title":"A tradeoff between robustness to environmental fluctuations and speed of evolution","identifier":"node\/1134918"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ius35CBNtJ5+jMA8lJepF\\\\\\\/fZ7WDGEG3gFY6Yu27W6vkDP0wJZhXLA2su9YXtKubpXRbGcFc3P0oJlq1+el38BdmR48G6Y5LI59xkAWhs9UwnmMXFl4U414Dbvpj2XT9YTQWya9Op6wjxbVlUTMEUzPtJZaPzUiyYdrt3xoB6lqtLAERdvLxrXABr8mB+fUTKCO6mCPmZQQAZ2DsT61bzkGS5L8OWKbIOn+9\\\\\\\/WJ5ZkjIKNr\\\\\\\/q4opWMyebUDXMDXSwi\\\\\\\/JXOSPm0LfzJ89aX69wFbwDS\\\\\\\/DH8PoPCsREq+RQcsVzAbBUgGezFJPpxN0WeWgMuJQz4xn3ApkDPDss2arvbLEDClJNi4g5dYTdi1b\\\\\\\/799+\\\\\\\/Cd\\\\\\\/dLXfYADAIr4kjTdAHf3LrdMexWxZkE6UeiunduaOu8LGKGtCbJSCOR8I5QPAXHbJsTRxyH3UrA8OPyPU7Vj+ctCxh20mfIXk2XHorkG2o2TELfQrhFf\\\\\\\/5uQZAMk8YbyDrcfILP8oI2oMZ5+2h99Khx7Y2bwE2YDcbhCY4hfPif9j5lI0Kakaeug4rYa+zHm2YnoIi1UFCll1h+u6US4MTd1r\\\\\\\/sMvXnHufoIL8Ael7ePPaxYoyRq7VEvpbEXkqHYM4B6o6LETa\\\\\\\/M2bZ7Q0hbTHtonUSZ6azG7jsJ51\\\\\\\/eO\\\\\\\/JYiJC3XQG5PGS1Wd9s2L+KY3mKW87jpfS1AiPkx9x8wx7RK\\\\\\\/gtFYhqmY0+OEEMIXKnljanHV9QVq5HuELFH6ss+9JAlUqNOVv4PcP1qQHoEXBjDK9fUaJsRVlzVZyxHnV4s7MS\\\\\\\/4vdD9uDL\\\\\\\/5P11zC8ZQiSnhin+m6mKVibvLOvJ459zxnnlcibhkY81DiHcI5BRy0zO8gCZVBsOlFU\\\\\\\/ZNFgUHmTJ49V4XQ0Of\\\\\\\/bhXtGQa\\\\\\\/2uVXLCLzTr9WkS6tsQUWR2G6ypZ\\\\\\\/WjKhhfUkLRVlAxSxJCGcazuD8tUjDxMPvT1EGr1ELnDUx2P+oRQJIMQ\\\\\\\/HXKvYPDrBOAJqe053axDIwuT51QpRaotdhlccOoS7FrPwNtSHw2ji\\\\\\\/6TZ0OQMAmbDuMbtjgfbxz97Zp3kznrTigzraeQmAQXYoC\\\\\\\/Msaoe7SFSW7JXHhhEkds+v4H6l5Dvtm8Cf55ynYns2HLfCuNMleA2Kgv\\\\\\\/QkaRinCPKLraUXW6KVMNG3sN9z6iJvVVX6fLXCwAlLRLYfvDIn82rfhfTRpXWH\\\\\\\/i1CkDg\\\\\\\/B1CHvQlViwFs8JpaDmU8g4ciLhdf+QzjpFOAw2IrWLukQRgGPsU1Raudtu\\\\\\\/kI8rCWdM5n9eQ4eoXmd7Kt4KOB5jC7gg\\\\\\\/7ulBTXkvFOwoM9gFShqG3vkjjRPgOj8e16rZq\\\u0022,\\\u0022iv\\\u0022:\\\u0022BqjlPWNbC\\\\\\\/f2v8utpC4UJA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022ceaf33dfb77d84db9298123face03ce6b1b0e1a0778330fa379dd00bd10e9e6b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/834234v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1134918 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/834234v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NKqVGH1K_gB4o8fqN0A7ew97pzFnjZXzgt0wKDSG25k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1134918" id="node1134918" data-pisa="biorxiv;834234v2" data-pisa-master="biorxiv;834234" data-apath="/biorxiv/early/2020/02/04/834234.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A tradeoff between robustness to environmental fluctuations and speed of evolution</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7197-4775" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Max</span> <span class="nlm-surname">Schmid</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1949-4448" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Paniw</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4317-2721" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Maarten</span> <span class="nlm-surname">Postuma</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7477-2642" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Arpat</span> <span class="nlm-surname">Ozgul</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0874-0081" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Frédéric</span> <span class="nlm-surname">Guillaume</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/834234 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1134918" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Max Schmid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Evolutionary Biology and Environmental Studies, University of Zurich</span>, Winterthurerstrasse 190, CH-8057, Zurich, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Max%2BSchmid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmid%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMax%2BSchmid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7197-4775" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Max Schmid</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Maria Paniw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Evolutionary Biology and Environmental Studies, University of Zurich</span>, Winterthurerstrasse 190, CH-8057, Zurich, <span class='nlm-country'>Switzerland</span></div><div class='author-affiliation'><span class='nlm-institution'>Ecological and Forestry Applications Research Centre (CREAF), Campus de Bellaterra (UAB) Edifici C</span>, ES-08193 Cerdanyola del Vallès, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BPaniw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paniw%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaria%2BPaniw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1949-4448" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maria Paniw</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maarten Postuma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Evolutionary Biology and Environmental Studies, University of Zurich</span>, Winterthurerstrasse 190, CH-8057, Zurich, <span class='nlm-country'>Switzerland</span></div><div class='author-affiliation'><span class='nlm-institution'>Plant Ecology and Nature Conservation Group, Wageningen University</span>, 6700 AA Wageningen, <span class='nlm-country'>the Netherlands</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maarten%2BPostuma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Postuma%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaarten%2BPostuma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4317-2721" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maarten Postuma</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Arpat Ozgul </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Evolutionary Biology and Environmental Studies, University of Zurich</span>, Winterthurerstrasse 190, CH-8057, Zurich, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arpat%2BOzgul%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ozgul%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArpat%2BOzgul%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7477-2642" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arpat Ozgul</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Frédéric Guillaume </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Evolutionary Biology and Environmental Studies, University of Zurich</span>, Winterthurerstrasse 190, CH-8057, Zurich, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fr%C3%A9d%C3%A9ric%2BGuillaume%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guillaume%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFr%25C3%25A9d%25C3%25A9ric%2BGuillaume%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0874-0081" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frédéric Guillaume</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frederic.guillaume@ieu.uzh.ch" class="" data-icon-position="" data-hide-link-title="0">frederic.guillaume@ieu.uzh.ch</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/834234v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1134918" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1134918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/834234v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1134918" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1134918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/834234v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1134918" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1134918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/834234v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1134918" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1134918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/834234v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1134918" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1134918/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-5">Organisms must cope with both short- and long-term environmental changes to persist. In this study we investigated theoretically if life histories trade off between their robustness to short-term environmental perturbations and their ability to evolve directional trait changes. With mathematical models and stochastic individual-based simulations, we could confirm the tradeoff, at least for comparisons between standardized life histories. Offspring dormancy and high adult survival rates allowed large population sizes to be maintained in face of interannual environmental fluctuations but limited the speed of trait evolution with ongoing environmental change. In contrast, precocious offspring maturation and short-living adults promoted evolvability while lowering demographic ro-bustness. This tradeoff had immediate consequences on extinction dynamics in variable environments. High evolvability allowed to cope best with long-lasting gradual environmental change, but came at the expense of more pronounced population declines from environmental variability and during the initial phase of environmental change.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-6">Environmental conditions are rarely constant but fluctuate between years and average conditions change gradually over time, and both dynamics pose a considerable challenge for species (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bürger and Lynch, 1995</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boyce et al., 2006</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Van De Pol et al., 2010</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Vázquez et al., 2015</a>). Average global temperatures change successively in the course of long-term glacial cycles or during ongoing anthropogenic climate change, but also vary on shorter time scales, for instance with variations between years (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Stocker et al., 2013</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Ummenhofer and Meehl, 2017</a>). Given that the environmental tolerance of all species is finite, the exposure to fluctuating environmental conditions can result in higher mortality, lower recruitment, and population declines. A dynamic environment thus demands species to constantly adapt and adjust. While several adaptive strategies exist to cope with environmental changes (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Parmesan, 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aitken et al., 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bell and Gonzalez, 2009</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Doak and Morris, 2010</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Duputié et al., 2015</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cayuela et al., 2017</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">McDonald et al., 2017</a>), some strategies might not be adaptive with every kind of temporal change. For instance life-history characteristics affect the robustness of a population against short-term environmental fluctuations (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Caswell, 2001</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Boyce et al., 2006</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Morris et al., 2008</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Tuljapurkar et al., 2009</a>), and the speed of adaptive evolution in response to gradual environmental change (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Engen et al., 2011</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Orive et al., 2017</a>). However, the ability of life histories to cope with one kind of environmental variability might come at a cost to cope with another kind of dynamic environment.</p><p id="p-7">Stochasticity in vital rates (survival and fecundity) from random environmental fluctuations reduces the long-run fitness of populations and may depress population sizes, thereby increasing extinction risks (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lande and Orzack, 1988</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Tuljapurkar et al., 2003</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Engen et al., 2005</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sæther et al., 2013</a>). How severely environmental variability feeds back on population growth then depends on the species’ life cycle, especially on its position along the fast-slow continuum and its reproductive strategy. Theory shows that species with longer generation times are more robust to fluctuations than short lived species, as are more iteroparous compared to semelparous species (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Tuljapurkar, 1990</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Tuljapurkar et al., 2009</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Salguero-Gómez et al., 2017</a>). Empirically, <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Morris et al. (2008)</a> found a relation between the life expectancy of 36 plant and animal species and their sensitivity to environmentally driven variation in vital rates. Life histories with long generation times experienced less pronounced demographic consequences from stochasticity in vital rates than short-lived species (see also <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dalgleish et al., 2010</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Sæther et al., 2013</a>). The temporal patterning of fluctuations is also crucial for population dynamics (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Engen et al., 2013b</a>). The growth rates of fast life histories were shown to be more sensitive to temporal autocorrelation in environmental fluctuations than slow life histories (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Paniw et al., 2017</a>). A life history’s robustness against environmental fluctuations further depends on the proportion of individuals exposed to environmental extremes. For instance, long-lived species often exhibit highly tolerant age or stage classes. Plant seeds or adult trees are good examples of such stages because of their high tolerance against extreme temperatures or drought events, and can overcome detrimental conditions much better than seedlings or juvenile plants (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Petit and Hampe, 2006</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Buoro and Carlson, 2014</a>). Delayed maturation of seeds or egg dormancy then can allow species to cope with temporally fluctuating environments by “dispersal through time” (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Evans and Dennehy, 2005</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Buoro and Carlson, 2014</a>). Consequently, a species’ life history is a critical factor affecting population dynamics in fluctuating environments with interannual variation (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Tuljapurkar, 1990</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Tuljapurkar et al., 2009</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Morris et al., 2008</a>).</p><p id="p-8">Life-history strategies also shape the rate of trait evolution when the environment changes gradually (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lande, 1982</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Charlesworth, 1994</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ellner and Hairston, 1994</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Engen et al., 2009</a>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">2011</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Orive et al., 2017</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cotto et al., 2019</a>). The generation time of a species, which affects the pace of life, is an indicator for how fast selection can translate into changes of the gene pool. The lower the generation time, the greater is the evolutionary response to environmental change per unit of time (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Lande, 1982</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Vander Wal et al., 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Carlson et al., 2014</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Orive et al., 2017</a>). With life stages highly tolerant to environmental variation, life-history strategies further control the proportion of individuals exposed to the environment. Those genotypes “hidden” in the seedbank or in highly tolerant adult stages are not under selection and thereby delay genetic changes at the population level, lowering its rate of evolutionary trait change (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Templeton and Levin, 1979</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hairston and De Stasio Jr., 1988</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>; <a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Orive et al., 2017</a>). Together with the life-history effects on robustness mentioned above, we can expect that short-lived species will have faster evolutionary trait responses but will suffer more from a higher de-mographic sensitivity to random fluctuations in vital rates. In contrast, life histories with longer generation times should be demographically more robust but less evolutionarily responsive to changes in their environments. This tradeoff between demographic robustness and evolvability is in fact implicit from previous theoretical work. <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Lande (1982)</a> and <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> described the evolutionary response to selection as a function of the growth rate sensitivity to vital rate changes, which is a quantity related to robustness against short-term fluctuations. The tradeoff was also suggested by the results of <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Hairston and De Stasio Jr. (1988)</a> and <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Templeton and Levin (1979)</a>, who found that the presence of a seed bank, a mechanism to cope with stochastic environments, slowed down the rate of phenotypic trait evolution. In a study by <a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Orive et al. (2017)</a>, elevated iteroparity (with adults shielded from selection) favored population recovery after a sudden step change in the environment, but made evolutionary rescue with gradual environmental change less likely.</p><p id="p-9">While several findings point towards a robustness-evolvability tradeoff for life-history strategies, other studies hint at potential mechanisms to overcome this tradeoff. Life histories with overlapping generations were suggested to harbour higher levels of additive genetic variance in fluctuating environments than life histories with non-overlapping generations, a process known as the genetic storage effect (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Ellner and Hairston, 1994</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sasaki and Ellner, 1997</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Svardal et al., 2015</a>). Given that the likelihood of a successful evolutionary rescue increases with the extent of standing genetic variation (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Carlson et al., 2014</a>), life histories with highly robust life stages might be able to accumulate higher levels of genetic variance, and thus also evolve faster (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Yamamichi et al., 2019</a>). All in all, this shows that life-history strategies can have large effects on evolutionary and demographic dynamics, and on the success of evolutionary rescue. However, the exact resolution of the tradeoff between evolvability and robustness is not solved yet and will likely depend on the details of the species’ life cycle and on the type of environmental variation.</p><p id="p-10">Here, we first characterized the tradeoff between demographic robustness and evolvability for four 3-stage life histories spanning the fast-slow continuum and different levels of iteroparity in stage-structured populations. Our analytical analysis of the tradeoff is based on classical matrix population modeling (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Caswell, 2001</a>) and on the quantitative genetics formalism developed by <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Lande (1982)</a> and <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> to compute the phenotypic response to selection in stage-structured populations. We show that a linear tradeoff exists between sensitivities of the equilibrium population size (and growth rate), and the rate of evolution of a quantitative trait. In a second step, we used individual-based simulations to model the eco-evolutionary dynamics of a single population for the same four life histories subject to both random inter-annual fluctuations and long-term gradual changes of the optimum trait value. The population reached its carrying capacity from negative density dependent survival. Only juveniles were under selection such that only their survival rate varied with the environment, while offspring and adults had very high environmental tolerance levels. The simulations further allowed us to account for demographic and genetic stochasticity and evolving additive genetic variance. We could thus study the effect of the tradeoff between robustness and evolvability on the population extinction risk.</p></div><div class="section" id="sec-2"><h2 class="">Models</h2><p id="p-11">We modeled a hermaphroditic species with three life stages (offspring (<em>n</em><sub>1</sub>), juveniles (<em>n</em><sub>2</sub>), and adults (<em>n</em><sub>3</sub>)) in a single random mating population. The population reached an equilibrium population size <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-1.gif"/></noscript></span></span> from intra-specific density regulation. Environmental fluctuations only affected juvenile survival via stabilizing selection on a quantitative trait. This corresponds to situations where the focal trait is only expressed at the juvenile stage (e.g., growth), and where offspring and adult individuals have much higher environmental tolerance (e.g., frost or drought tolerance), or dwell in much more stable habitats (e.g., <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Petit and Hampe, 2006</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Van De Pol et al., 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Marshall et al., 2016</a>). As a consequence of the negative density dependence, a reduction in juvenile survival from environmental variability or directional selection resulted in a lower equilibrium population size (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1d,e</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The life-history plot in a) illustrates the three-stage life cycle with delayed maturation and long-living adults (del-sem, del-ite, pre-ite), while b) shows the biennial two-stage life cycle (pre-sem), together with the relevant vital rates. Graph c) demonstrates the effect of negative density dependence on juvenile survival in the three-stage life cycle based on the Ricker function. Graph d) illustrates the temporal change of total population size over time following logistic growth such that an equilibrium mean population size is achieved from negative density dependence. By reducing the average juvenile survival &#x3C3;2, for instance from environmental perturbations, the population size at equilibrium  is reduced. We used the sensitivity of the equilibrium population size to juvenile survival  as a measure for environmental robustness. In graph e), two life histories are shown with the same equilibrium population size (, the two solid lines &#x2014;) when environmental fluctuations are absent (at &#x3C3;2 = 0.8) but with different sensitivities to reductions in juvenile survival from interannual fluctuations. The slope of the dotted lines (&#x2026;) in graph e) represents  as how quickly  responds to small reductions in &#x3C3;2. Graph f) illustrates the scenario with directional environmental change and the the corresponding trait evolution of two life histories (the two solid lines &#x2014;). Environmental change is realized via directional changes of the phenotypic optima (&#x3B8;, dashed line &#x2013;&#x2013;)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The life-history plot in a) illustrates the three-stage life cycle with delayed maturation and long-living adults (&lt;em&gt;del-sem&lt;/em&gt;, &lt;em&gt;del-ite&lt;/em&gt;, &lt;em&gt;pre-ite&lt;/em&gt;), while b) shows the biennial two-stage life cycle (&lt;em&gt;pre-sem&lt;/em&gt;), together with the relevant vital rates. Graph c) demonstrates the effect of negative density dependence on juvenile survival in the three-stage life cycle based on the Ricker function. Graph d) illustrates the temporal change of total population size over time following logistic growth such that an equilibrium mean population size is achieved from negative density dependence. By reducing the average juvenile survival &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, for instance from environmental perturbations, the population size at equilibrium  is reduced. We used the sensitivity of the equilibrium population size to juvenile survival  as a measure for environmental robustness. In graph e), two life histories are shown with the same equilibrium population size (, the two solid lines &#x2014;) when environmental fluctuations are absent (at &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8) but with different sensitivities to reductions in juvenile survival from interannual fluctuations. The slope of the dotted lines (&#x2026;) in graph e) represents  as how quickly  responds to small reductions in &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;. Graph f) illustrates the scenario with directional environmental change and the the corresponding trait evolution of two life histories (the two solid lines &#x2014;). Environmental change is realized via directional changes of the phenotypic optima (&lt;em&gt;&#x3B8;&lt;/em&gt;, dashed line &#x2013;&#x2013;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F1.medium.gif" width="440" height="427"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F1.medium.gif" width="440" height="427"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-12" class="first-child">The life-history plot in a) illustrates the three-stage life cycle with delayed maturation and long-living adults (<em>del-sem</em>, <em>del-ite</em>, <em>pre-ite</em>), while b) shows the biennial two-stage life cycle (<em>pre-sem</em>), together with the relevant vital rates. Graph c) demonstrates the effect of negative density dependence on juvenile survival in the three-stage life cycle based on the Ricker function. Graph d) illustrates the temporal change of total population size over time following logistic growth such that an equilibrium mean population size is achieved from negative density dependence. By reducing the average juvenile survival <em>σ</em><sub>2</sub>, for instance from environmental perturbations, the population size at equilibrium <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-2.gif"/></noscript></span></span> is reduced. We used the sensitivity of the equilibrium population size to juvenile survival <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-3.gif"/></noscript></span></span> as a measure for environmental robustness. In graph e), two life histories are shown with the same equilibrium population size (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-4.gif"/></noscript></span></span>, the two solid lines —) when environmental fluctuations are absent (at <em>σ</em><sub>2</sub> = 0.8) but with different sensitivities to reductions in juvenile survival from interannual fluctuations. The slope of the dotted lines (…) in graph e) represents <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-5.gif"/></noscript></span></span> as how quickly <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F1/embed/inline-graphic-6.gif"/></noscript></span></span> responds to small reductions in <em>σ</em><sub>2</sub>. Graph f) illustrates the scenario with directional environmental change and the the corresponding trait evolution of two life histories (the two solid lines —). Environmental change is realized via directional changes of the phenotypic optima (<em>θ</em>, dashed line ––).</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">We focused on four distinct life-history strategies along two axes, timing of maturation and degree of iteroparity, with two cases each: precocious (<strong>pre</strong>) and delayed (<strong>del</strong>) maturation, iteroparity (<strong>ite</strong>) and semelparity (<strong>sem</strong>). The life histories with combinations of delayed maturation and iteroparity (<em>del-sem</em>, <em>del-ite</em>, <em>pre-ite</em>) had three stages and overlapping generations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1a</a>). The precocious, semelparous life cycle (<em>per-sem</em>) had non-overlapping generations modeled with only two stages, juveniles (offspring maturing immediately) and adults reproducing once and dying (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1b</a>).</p><div id="sec-3" class="subsection"><h3>Life-history parameters</h3><p id="p-14">Offspring, juvenile and adult individuals survived to the next year with probability <em>σ</em><sub>1</sub>, <em>σ</em><sub>2</sub>, and <em>σ</em><sub>3</sub>, respectively (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1a,b</a>). Offspring exclusively resulted from the sexual reproduction of hermaphroditic adults with an average per capita fecundity of <em>φ</em>. Offspring matured to juveniles with a maturation probability <em>γ</em> and stayed in the offspring stage with probability 1 − <em>γ</em>. Survival of juveniles was reduced by intra-specific competition <em>c</em> following the Ricker function (<em>c</em> = <em>exp</em>(−<em>b</em> ∗ <em>n</em><sub>3</sub>)) and thus declined with increasing adult number <em>n</em><sub>3</sub> for overlapping generations, and with juvenile number <em>n</em><sub>2</sub> for non-overlapping generations, depending on the competition coefficient <em>b</em>. We also modeled a Beverton-Holt density regulation and present the results in the Supplementary Material.</p><p id="p-15">The demographic dynamics of the three-stage life cycles <em>del-sem</em>, <em>del-ite</em>, and <em>pre-ite</em> were modeled by the following matrix population model (MPM):
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-2.gif"/></noscript></span>

</span>
and the two-stage life cycle <em>pre-sem</em> was represented by the following MPM:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-3.gif"/></noscript></span>

</span>
</p></div><div id="sec-4" class="subsection"><h3>Population sensitivity to environmental fluctuations</h3><p id="p-16">The robustness of a life history against environmental fluctuations can be seen as the sensitivity of the total equilibrium population size <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-7.gif"/></noscript></span></span> to juvenile survival <em>σ</em><sub>2</sub> (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-8.gif"/></noscript></span></span>, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1e</a>). Assuming that the detrimental environmental conditions exclusively reduces average <em>σ</em><sub>2</sub>, but not <em>σ</em><sub>1</sub> nor <em>σ</em><sub>3</sub>, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-9.gif"/></noscript></span></span> is indicative of the effect of environmental fluctuations on the equilibrium population size (at λ = 1).</p><p id="p-17">For the three-stage life cycle, we first derived the total population size at equilibrium <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-10.gif"/></noscript></span></span> by solving <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-11.gif"/></noscript></span></span> with population vector <em>N</em> = (<em>n</em><sub>1</sub>, <em>n</em><sub>2</sub>, <em>n</em><sub>3</sub>)<sup><em>T</em></sup> (T meaning transposed, see Appendix A for the derivation):
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-4.gif"/></noscript></span>

</span>
</p><p id="p-18">From <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">equation 3</a> we derived the partial derivative of <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-12.gif"/></noscript></span></span> with respect to <em>σ</em><sub>2</sub> (see Appendix A8):
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-5.gif"/></noscript></span>

</span>
</p><p id="p-19">The derivations for the two-stage life cycle are detailed in Appendix B. From <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">equation (4)</a>, we can readily see that for a given match of the population to its local optimum (for a given <em>σ</em><sub>2</sub>), different life histories will have different rates of declines of their population size away from its equilibrium value depending mostly on the fecundity and less on other rates.</p><p id="p-20">An alternative measure for the robustness to environmental variability is the sensitivity of the population growth rate <em>λ</em> to juvenile survival (<em>∂λ</em>/<em>∂σ</em><sub>2</sub>), which was shown to correlate with the sensitivity of the effective population size <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-13.gif"/></noscript></span></span>, instead of the total population size <em>n</em><sub><em>t</em></sub> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Grant and Benton, 2000</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">2003</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Caswell et al., 2004</a>). Here, the effective population size represents the number of individuals weighted by the contribution of their stage class to density regulation and population growth (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Takada and Nakajima, 1998</a>). We present the results for <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-14.gif"/></noscript></span></span> together with those of <em>∂λ</em>/<em>∂σ</em><sub>2</sub> (both estimated at equilibrium population density when <em>λ</em>=1), and discuss the differences between these two measures in Appendix C.</p></div><div id="sec-5" class="subsection"><h3>The rate of trait evolution</h3><p id="p-21">We quantified the speed of evolution for a life history as the average trait change Δ<em>z</em> per time step <em>t</em> from genetic evolution (not plasticity) in response to directional selection. The speed of evolution for quantitative traits in a stage-structured population was derived by <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a>:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-6.gif"/></noscript></span>

</span>
where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-15.gif"/></noscript></span></span> is the sensitivity of the growth rate <em>λ</em> to changes in the vital rates <em>a</em><sub><em>ij</em></sub> (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Caswell, 2001</a>). <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a> (also known as Lande’s theorem; <a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Lande, 1982</a>) was obtained under the assumption of weak selection and same mean phenotype <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-16.gif"/></noscript></span></span> and genetic variance <strong>G</strong> for all stages <em>j</em> (see discussion in <a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>, Appendix B). Importantly, <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">equation 5</a> makes explicit the link between rate of evolution and population sensitivity to variation in vital rates. It also accounts for the changes of vital rates caused by changes in trait value <em>z</em>. We consider here that variation in survival is caused by the phenotypic mismatch of the population with its environment, which only affects the survival rate of juveniles. The relationship between phenotypic trait value <em>z</em> and juvenile survival <em>σ</em><sub>2</sub> is given by a Gaussian fitness function: <em>w</em>(<em>z</em>) = <em>exp</em>(−(<em>z</em> − <em>θ</em>)<sup>2</sup>/2<em>ω</em><sup>2</sup>), where <em>σ</em><sub>2</sub>(<em>z</em>) = <em>w</em>(<em>z</em>), with phenotypic optimum value <em>θ</em>, and variance of the fitness function <em>ω</em><sup>2</sup>, inversely related to the strength of selection on the trait.</p><p id="p-22">For a species with three life-history stages, that reached the equilibrium population size from negative density dependence, with a stable stage distribution (SSD), and with one trait under selection in juveniles (<em>z</em><sub>2</sub>), the asymptotic rate of evolution can be derived from <a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">Equation 5</a>:
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-7.gif"/></noscript></span>

</span>
</p><p id="p-23">The asymptotic rate of trait change <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-17.gif"/></noscript></span></span> thus varies with the additive genetic variance in juveniles <em>G</em><sub>2</sub>, the selection gradient on a single juvenile trait <em>β</em>(<em>z</em><sub>2</sub>), the fraction of individuals exposed to selection <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-18.gif"/></noscript></span></span>, the survival probability of these individuals <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-19.gif"/></noscript></span></span>, and their contribution to the subsequent generations <em>v</em><sub>3</sub> (the reproductive value of adults standardized such that <em>v</em><sup><em>T</em></sup> <em>w</em> = 1) (<a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">equation 8</a>, <a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>). The rate of evolution <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-20.gif"/></noscript></span></span> can thus be expressed as a function of the vital rates by deriving expressions for <em>w</em><sub>2</sub>, <em>c</em>, and <em>v</em><sub>3</sub>:
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-8.gif"/></noscript></span>

</span>
as detailed in Appendix A. <a id="xref-disp-formula-5-4" class="xref-disp-formula" href="#disp-formula-5">Equations 5</a>–<a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">7</a> assume a population growth rate of <em>λ</em> = 1 at equilibrium population density, sexual (non-clonal) reproduction, weak selection, population dynamics that are faster than evolutionary dynamics, a Gaussian genotype-phenotype distribution, and the Gaussian fitness function mentioned above (<a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>; <a id="xref-ref-57-6" class="xref-bibr" href="#ref-57">Orive et al., 2017</a>). Under these assumptions, the selection gradient is: <em>β</em>(<em>z</em>) = −(<em>z</em> − <em>θ</em>)/(<em>ω</em><sup>2</sup> + <em>P</em>), a function of the distance between the phenotype <em>z</em> and its optimum <em>θ</em>, with <em>P</em> the phenotypic variance of the population (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Via and Lande, 1985</a>). Further assuming, for simplicity, that <em>β</em>(<em>z</em>) = 1, the evolvability of the population, defined by <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hansen et al. (2019)</a> as <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-21.gif"/></noscript></span></span>, becomes <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-22.gif"/></noscript></span></span>. We thus use the rate of change of the population mean phenotype as our measure of evolvability.</p></div><div id="sec-6" class="subsection"><h3>Standardization for numerical exploration</h3><p id="p-24">We standardized our generic life histories for uniform maximum population sizes <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-23.gif"/></noscript></span></span> in absence of environmental variation and for uniform maximum population growth rates (<em>λ</em><sub><em>max</em></sub>, the intrinsic population growth rate in absence of density regulation when <em>c</em>=<em>1</em>). With this standardization we could compare our four life histories while keeping key demographic properties constant (see Appendix C for a detailed discussion). Using a numeric approach, we adjusted the fecundity <em>ϕ</em> to reach equal <em>λ<sub>max</sub></em> across life histories, and a standardized form of the competition coefficient <em>b</em> (see <a id="xref-disp-formula-17-1" class="xref-disp-formula" href="#disp-formula-17">Equations A9</a>, <a id="xref-disp-formula-33-1" class="xref-disp-formula" href="#disp-formula-33">B8</a>) to reach equal <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-24.gif"/></noscript></span></span>. Using this standardization, <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">Equation 4</a> becomes:
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-9.gif"/></noscript></span>

</span>
</p><p id="p-25">Using <a id="xref-disp-formula-7-2" class="xref-disp-formula" href="#disp-formula-7">Equations 7</a> and <a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">8</a>, we explored the effect of the vital rates <em>σ</em><sub>1</sub>, <em>σ</em><sub>3</sub>, and <em>γ</em> on the tradeoff between population size sensitivity <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-25.gif"/></noscript></span></span> and evolvability Δ<em>z</em> (with <em>G</em> = <em>β</em>(<em>z</em><sub>2</sub>) = 1). As the sensitivity of the total population size varied with <em>σ</em><sub>2</sub> (see <a id="xref-disp-formula-8-3" class="xref-disp-formula" href="#disp-formula-8">equ. 8</a>),<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-26.gif"/></noscript></span></span> was always quantified with <em>σ</em><sub>2</sub> = 0.8. In addition we provide results for <em>∂λ</em>/<em>∂σ</em><sub>2</sub> calculated for each combination of vital rates using the function sensitivity() from the <em>popbio</em> package (version 2.4.3, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Stubben and Milligan, 2007</a>) in R 3.4.2 (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">R Development Core Team, 2008</a>).</p></div><div id="sec-7" class="subsection"><h3>Individual-based simulations</h3><p id="p-26">We ran individual-based simulations to test our mathematical derivations and account for genetic and demographic stochasticity, and an evolving additive genetic variance. Simulations were run with Nemo (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Guillaume and Rougemont, 2006</a>) extended for stage-structured populations (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Cotto et al., 2017</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2020</a>). We simulated a single panmictic population of hermaphroditic individuals. The population reached an equilibrium population size by density-dependent juvenile survival. A single quantitative trait was under hard selection in the juvenile stage such that changes in juvenile fitness (survival) translated into changes of the population size. The following life cycle events occurred consecutively each year: density-dependent regulation (of juvenile survival using the Ricker function), viability selection (removal of juveniles according to their survival probability which varied with the distance between trait value and environment), sexual reproduction (independent from the environmental state or population density), and stage transitions according to the MPM.</p></div><div id="sec-8" class="subsection"><h3>Life-history parameters</h3><p id="p-27">We modeled the four generic life-history strategies presented in the previous section. We standardized all four life-history strategies for the same total population size in absence of environmental fluctuations (<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-27.gif"/></noscript></span></span> by adjusting <em>b</em>) and the same maximum population growth rate (<em>λ</em> = 1.10 or <em>λ</em><sub><em>max</em></sub> = 1.15, by adjusting <em>ϕ</em>). The single vital rates are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">table 1</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1140679/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1140679/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1140679" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-28" class="first-child">The following table lists the vital rates of the simulated life-history strategies, which are combinations of delayed (<em>del</em>, maturation probability <em>γ</em> = 0.5) or precocious (<em>pre</em>, <em>γ</em> = 1.0) offspring maturation, and semelparous (<em>sem</em>, adult survival <em>σ</em><sub>3</sub> = 0.0) or iteroparous (<em>ite</em>, <em>σ</em><sub>3</sub> = 0.5) adults, standardized for <em>λ</em><sub><em>max</em></sub> = 1.10 or <em>λ</em><sub><em>max</em></sub> = 11.5 via the annual fecundity <em>ϕ</em>. Offspring and juvenile survival were set to <em>σ</em><sub>1</sub> = <em>σ</em><sub>2</sub> = 0.8 for all life histories. The generation time (<em>T</em>) and the net reproductive rate (<em>R</em><sub>0</sub>) were calculated for the MPMs in absence of density regulation using the functions generation.time() and net.reproductive.rate() from the popbio package in R (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Stubben and Milligan, 2007</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">R Development Core Team, 2008</a>). The growth rate sensitivity (<em>∂λ</em>/<em>∂σ</em><sub>2</sub>) was calculated for the MPMs at <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/T1/embed/inline-graphic-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/T1/embed/inline-graphic-28.gif"/></noscript></span></span> using the function eigen.analysis() from the popbio package in R (<a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Stubben and Milligan, 2007</a>; <a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">R Development Core Team, 2008</a>). The rate of evolution Δ<em>z</em> was calculated with <em>G</em><sub>2</sub><em>β</em>(<em>z</em><sub>2</sub>) = 1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">With these simulation scenarios we tried to span a range of key life-history characteristics by accounting for differences in “pace of life” along the slow-fast continuum and for differences in the reproductive strategy (see <a id="xref-fig-19-1" class="xref-fig" href="#F19">Fig. S13</a>, and <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Stearns, 1983</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Gaillard et al., 1989</a>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">2005</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Salguero-Gómez, 2017</a>).</p></div><div id="sec-9" class="subsection"><h3>Genetic setup and selection model</h3><p id="p-30">We simulated the evolutionary dynamics of adaptation to a single environmental condition via the evolution of a single quantitative trait (<em>z</em>) under stabilizing selection in the juvenile stage using the classical Gaussian fitness function <em>w</em>(<em>z</em>) = <em>exp</em>(−(<em>z</em> − <em>θ</em>)<sup>2</sup>/2<em>ω</em><sup>2</sup>) (see above). We set the width of the fitness function to <em>ω</em><sup>2</sup> = 4 (corresponding to a stronger selection strength) or <em>ω</em><sup>2</sup> = 16 (representing a weaker selection strength). Each individual carried <em>l</em> = 20 unlinked loci, contributing additively to the trait <em>z</em>. We used a continuum-of-allele model where mutational effects, picked from a Gaussian distribution centered on zero with mutational variance <em>α</em> = 0.1, were added to the existing allelic values. The mutation rate was <em>μ</em> = 0.0001 so that the mutational phenotypic variance was <em>V</em><sub><em>M</em></sub> = 2<em>lμα</em> = 0.001. We excluded random environmental effects on phenotype expression (e.g., developmental noise) such that the phenotype <em>z</em> was the sole product of the additive genetic effects for simplicity.</p></div><div id="sec-10" class="subsection"><h3>Environmental scenarios</h3><p id="p-31">We simulated environmental variation between years together with gradual changes of the average condition by perturbing the phenotypic optimum <em>θ</em> (e.g., see <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chevin et al., 2017</a>). Each year, the position of the phenotypic optimum for the entire population was picked from a Gaussian distribution <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-29.gif"/></noscript></span></span> with the average environment <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-30.gif"/></noscript></span></span> and variance <em>ϵ</em>. We thus simulated environmental fluctuations between years that were independently and identically distributed (e.g., <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Caswell, 2001</a>).</p><p id="p-32">To allow the population genetic variance to reach mutation-selection-drift balance, we ran burn-in simulations for 50’000 years for each life history. Burn-in simulations were run with a constant average environment of <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-31.gif"/></noscript></span></span> for six different degrees of environmental fluctuations (<em>ϵ</em> = 0.0, 0.2, 0.5, 1.0, 1.5, 2.0).</p><p id="p-33">To asses the population size sensitivity of life-history strategies to interannual environmental stochasticity, we recorded population demographic characteristic (e.g., population size <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-32.gif"/></noscript></span></span>) every 10 years for an additional period of 500 years. A life history’s population size sensitivity was assessed as the steepness of declining average population sizes<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-33.gif"/></noscript></span></span> with increasing degrees of interannual environmental fluctuations.</p><p id="p-34">To asses the speed of trait evolution, we then initiated directional environmental change of the average environmental condition <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-34.gif"/></noscript></span></span> after burn-in, while still simulating interannual variation on top of the changing mean. The average phenotypic optimum started to change at <em>t</em>=100 with a rate of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-35.gif"/></noscript></span></span>, for 200 and 800 years. We quantified the speed of evolution as the time to reach 90% of the novel phenotypic optimum <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-36.gif"/></noscript></span></span>. We did not use the maximum rate of phenotypic change per time step because all life histories either went extinct or adopted trait changes as fast as the rate of environmental change <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-37.gif"/></noscript></span></span>.</p></div></div><div class="section" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Mathematical models</h3><p id="p-35">We found a tradeoff between life-history effects on the rate of evolution and their robustness against environmental perturbations, at least when comparing life histories with the same maximum growth rate. The tradeoff is explicit in the positive, linear relationship between the rate of evolution (Δ<em>z</em>) and the sensitivity of the population size to juvenile survival (<em>∂n</em><sub><em>t</em></sub>/<em>∂σ</em><sub>2</sub> and <em>∂λ</em>/<em>∂σ</em><sub>2</sub>) for both the three-stage and the two-stage model (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). Changes in vital rates that enhanced the rate of evolution also increased the sensitivity to environmental perturbations, and vice versa.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A life history&#x2019;s tradeoff between evolvability (&#x394;z) and robustness ( in a, and &#x2202;&#x3BB;/&#x2202;&#x3C3;2 in b). Each circle illustrates a random combination of vital rates (with 0.2 &#x2264; &#x3C3;1 &#x2264; 1.0, &#x3C3;2 = 0.8, 0.0 &#x2264; &#x3C3;3 &#x2264; 0.9, 0.2 &#x2264; &#x3B3; &#x2264; 1.0). The fecundity (&#x3D5;) and the strength of competition (b) were re-calculated for each vital rate combination to standardize the maximum population growth rate (&#x3BB;max = 1.10 &#x2212; black, &#x3BB;max = 1.15 &#x2212; darkgray, &#x3BB;max = 1.20 &#x2212; lightgray) and the total population size at stable conditions (nt=1&#x2019;500). We then calculated the rate of evolution (&#x394;z, with G2&#x3B2;(z2) = 1), the population size sensitivity , and the growth rate sensitivity (&#x2202;&#x3BB;/&#x2202;&#x3C3;2) for each life-history variant. The three single points on the right of the plots were derived for the 2-stage life cycle (with &#x3C3;2 = 0.8), while the rest of the points illustrate parameter combinations for 3-stage life cycles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A life history&#x2019;s tradeoff between evolvability (&#x394;&lt;em&gt;z&lt;/em&gt;) and robustness ( in a, and &lt;em&gt;&#x2202;&#x3BB;&lt;/em&gt;/&lt;em&gt;&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; in b). Each circle illustrates a random combination of vital rates (with 0.2 &#x2264; &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; &#x2264; 1.0, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8, 0.0 &#x2264; &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt; &#x2264; 0.9, 0.2 &#x2264; &lt;em&gt;&#x3B3;&lt;/em&gt; &#x2264; 1.0). The fecundity (&lt;em&gt;&#x3D5;&lt;/em&gt;) and the strength of competition (&lt;em&gt;b&lt;/em&gt;) were re-calculated for each vital rate combination to standardize the maximum population growth rate (&lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.10 &#x2212; &lt;em&gt;black&lt;/em&gt;, &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.15 &#x2212; &lt;em&gt;darkgray&lt;/em&gt;, &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.20 &#x2212; &lt;em&gt;lightgray&lt;/em&gt;) and the total population size at stable conditions (&lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt;=1&#x2019;500). We then calculated the rate of evolution (&#x394;&lt;em&gt;z&lt;/em&gt;, with &lt;em&gt;G&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;&lt;em&gt;&#x3B2;&lt;/em&gt;(&lt;em&gt;z&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) = 1), the population size sensitivity , and the growth rate sensitivity (&lt;em&gt;&#x2202;&#x3BB;&lt;/em&gt;/&lt;em&gt;&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) for each life-history variant. The three single points on the right of the plots were derived for the 2-stage life cycle (with &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8), while the rest of the points illustrate parameter combinations for 3-stage life cycles.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F2.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F2.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-36" class="first-child">A life history’s tradeoff between evolvability (Δ<em>z</em>) and robustness (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F2/embed/inline-graphic-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F2/embed/inline-graphic-38.gif"/></noscript></span></span> in a, and <em>∂λ</em>/<em>∂σ</em><sub>2</sub> in b). Each circle illustrates a random combination of vital rates (with 0.2 ≤ <em>σ</em><sub>1</sub> ≤ 1.0, <em>σ</em><sub>2</sub> = 0.8, 0.0 ≤ <em>σ</em><sub>3</sub> ≤ 0.9, 0.2 ≤ <em>γ</em> ≤ 1.0). The fecundity (<em>ϕ</em>) and the strength of competition (<em>b</em>) were re-calculated for each vital rate combination to standardize the maximum population growth rate (<em>λ</em><sub><em>max</em></sub> = 1.10 − <em>black</em>, <em>λ</em><sub><em>max</em></sub> = 1.15 − <em>darkgray</em>, <em>λ</em><sub><em>max</em></sub> = 1.20 − <em>lightgray</em>) and the total population size at stable conditions (<em>n</em><sub><em>t</em></sub>=1’500). We then calculated the rate of evolution (Δ<em>z</em>, with <em>G</em><sub>2</sub><em>β</em>(<em>z</em><sub>2</sub>) = 1), the population size sensitivity <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F2/embed/inline-graphic-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F2/embed/inline-graphic-39.gif"/></noscript></span></span>, and the growth rate sensitivity (<em>∂λ</em>/<em>∂σ</em><sub>2</sub>) for each life-history variant. The three single points on the right of the plots were derived for the 2-stage life cycle (with <em>σ</em><sub>2</sub> = 0.8), while the rest of the points illustrate parameter combinations for 3-stage life cycles.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">A higher degree of iteroparity (high adult survival <em>σ</em><sub>3</sub>; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a,d,g</a>) and delayed maturation (low maturation rate <em>γ</em>; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3c,f</a>) increased the robustness of the population size to environmental fluctuations, but reduced the rate of trait evolution. In contrast, precocious maturation and semelparous adults allowed for fast evolution but reduced the robustness to environmental variability. We obtained qualitatively similar results for the sensitivity of the growth rate <em>∂λ</em>/<em>∂σ</em><sub>2</sub>, at least for a given maximal growth rate (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The graphs illustrate the analytical results for the sensitivity of the total population size to juvenile survival (, a-c), the rate of evolution (&#x394;z, d-f), and the sensitivity of the population growth rate (&#x2202;&#x3BB;/&#x2202;&#x3C3;2, g-i) for three-stage life cycle depending on survival rate (&#x3C3;1, &#x3C3;3), fecundity (&#x3D5;), and maturation probability (&#x3B3;). The sensitivity measures (&#x2202;&#x3BB;/&#x2202;&#x3C3;2, ) were estimated at equilibrium population size in absence of interannual environmental variation at &#x3C3;2 = 0.8. The maximum growth rate has been standardized for &#x3BB;max = 1.10 by fecundity adjustments, except for the case with varying fecundity (b,e,h). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (b). If not specified otherwise, the following parameter values have been used : &#x3C3;1 = 0.8, &#x3C3;3 = 0.5, &#x3B3; = 0.5, G2 = 1, and &#x3B2;(z2) = 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The graphs illustrate the analytical results for the sensitivity of the total population size to juvenile survival (, a-c), the rate of evolution (&#x394;&lt;em&gt;z&lt;/em&gt;, d-f), and the sensitivity of the population growth rate (&lt;em&gt;&#x2202;&#x3BB;&lt;/em&gt;/&lt;em&gt;&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, g-i) for three-stage life cycle depending on survival rate (&lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt;), fecundity (&lt;em&gt;&#x3D5;&lt;/em&gt;), and maturation probability (&lt;em&gt;&#x3B3;&lt;/em&gt;). The sensitivity measures (&lt;em&gt;&#x2202;&#x3BB;&lt;/em&gt;/&lt;em&gt;&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, ) were estimated at equilibrium population size in absence of interannual environmental variation at &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8. The maximum growth rate has been standardized for &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.10 by fecundity adjustments, except for the case with varying fecundity (b,e,h). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (&lt;em&gt;b&lt;/em&gt;). If not specified otherwise, the following parameter values have been used : &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = 0.8, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt; = 0.5, &lt;em&gt;&#x3B3;&lt;/em&gt; = 0.5, &lt;em&gt;G&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 1, and &lt;em&gt;&#x3B2;&lt;/em&gt;(&lt;em&gt;z&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) = 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F3.medium.gif" width="428" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F3.medium.gif" width="428" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-38" class="first-child">The graphs illustrate the analytical results for the sensitivity of the total population size to juvenile survival (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F3/embed/inline-graphic-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F3/embed/inline-graphic-40.gif"/></noscript></span></span>, a-c), the rate of evolution (Δ<em>z</em>, d-f), and the sensitivity of the population growth rate (<em>∂λ</em>/<em>∂σ</em><sub>2</sub>, g-i) for three-stage life cycle depending on survival rate (<em>σ</em><sub>1</sub>, <em>σ</em><sub>3</sub>), fecundity (<em>ϕ</em>), and maturation probability (<em>γ</em>). The sensitivity measures (<em>∂λ</em>/<em>∂σ</em><sub>2</sub>, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F3/embed/inline-graphic-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F3/embed/inline-graphic-41.gif"/></noscript></span></span>) were estimated at equilibrium population size in absence of interannual environmental variation at <em>σ</em><sub>2</sub> = 0.8. The maximum growth rate has been standardized for <em>λ</em><sub><em>max</em></sub> = 1.10 by fecundity adjustments, except for the case with varying fecundity (b,e,h). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F3/embed/inline-graphic-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F3/embed/inline-graphic-42.gif"/></noscript></span></span> via adjustments of the strength of competition (<em>b</em>). If not specified otherwise, the following parameter values have been used : <em>σ</em><sub>1</sub> = 0.8, <em>σ</em><sub>3</sub> = 0.5, <em>γ</em> = 0.5, <em>G</em><sub>2</sub> = 1, and <em>β</em>(<em>z</em><sub>2</sub>) = 1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The maximum rate of population growth (<em>λ</em><sub><em>max</em></sub>) shaped both the population size and growth rate sensitivity to environmental fluctuations, even though it had no effect on the rate of evolution. Higher <em>λ</em><sub><em>max</em></sub> reduced <em>∂n</em><sub>t</sub>/<em>∂σ</em><sub>2</sub> and thus helped to maintain higher population densities with short-term environmental perturbations (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a>, <a id="xref-disp-formula-8-4" class="xref-disp-formula" href="#disp-formula-8">equation 8</a>). However, higher <em>λ</em><sub><em>max</em></sub> increased <em>∂λ</em>/<em>∂σ</em><sub>2</sub> (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2b</a>). Therefore, vital rate changes that increased the populations’ maximal growth rate and helped to maintain higher population sizes in the face of interannual environmental fluctuations caused stronger declines in the effective population size (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3h</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3g-i</a>).</p><p id="p-40">Results for a Beverton-Holt instead of a Ricker function for density regulation were qualitatively similar. The linear robustness-evolvability tradeoff remained (<a id="xref-fig-11-1" class="xref-fig" href="#F11">Fig. S5</a>). Only the population size sensitivity <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-43.gif"/></noscript></span></span> was higher with a Beverton-Holt regulation function (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Fig. S6</a>).</p></div><div id="sec-13" class="subsection"><h3>Individual-based simulations</h3><p id="p-41">We could confirm the tradeoff between evolvability and robustness also in our individual-based simulations as the genetic storage effect was not strong enough to overcome the tradeoff. Life-history strategies with high robustness had the slowest rates of evolution, and vice versa. The average population size maintained at constant average conditions during burn-in declined with increasing environmental fluctuations for all life cycles below <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-44.gif"/></noscript></span></span>, while life-history strategies differed from each other in the extent of <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-45.gif"/></noscript></span></span> reductions (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4b</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5a</a>). The onset of gradual environmental change further reduced juvenile survival and evoked a decline in juvenile number and total population size (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4b,d</a>), before trait evolution allowed to adapt to the changing conditions (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4a</a>) and recover in a later phase of environmental change (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4d</a>). Evolutionary rescue resulted in a u-shaped curve in total population size <em>n</em><sub><em>t</em></sub>, with a population size recovery already during environmental change or only after environmental change has stopped (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4b</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The graphs show the results of individual-based simulations for the mean trait value z (a), the total population size nt (b), the additive genetic variance G2 (c), and the juvenile number n2 (d) for the four life histories (del-sem, del-ite, pre-ite, pre-sem) over 600 years. Environmental change started at t=100 and was realized by a gradual shift in the phenotypic optimum (&#x2013;&#x2013; black dashed lines) from &#x3B8; = 10 to &#x3B8; = 12 by t=300 (a). The between-year environmental fluctuations were set to &#x3F5; = 0.2 and life histories were standardized for &#x3BB;max = 1.15. Average population sizes (nt) were standardized to nt = 1500 in absence of interannual environmental fluctuations, and were depressed as result of between-year fluctuations and directional environmental change as result of a fitness reduction in juveniles (a). All values of the total population size and trait values represent the arithmetic means over all of the 100 replicates that did not go extinct." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The graphs show the results of individual-based simulations for the mean trait value &lt;em&gt;z&lt;/em&gt; (a), the total population size &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; (b), the additive genetic variance &lt;em&gt;G&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; (c), and the juvenile number &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; (d) for the four life histories (&lt;em&gt;del-sem&lt;/em&gt;, &lt;em&gt;del-ite&lt;/em&gt;, &lt;em&gt;pre-ite&lt;/em&gt;, &lt;em&gt;pre-sem&lt;/em&gt;) over 600 years. Environmental change started at t=100 and was realized by a gradual shift in the phenotypic optimum (&#x2013;&#x2013; black dashed lines) from &lt;em&gt;&#x3B8;&lt;/em&gt; = 10 to &lt;em&gt;&#x3B8;&lt;/em&gt; = 12 by t=300 (a). The between-year environmental fluctuations were set to &lt;em&gt;&#x3F5;&lt;/em&gt; = 0.2 and life histories were standardized for &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.15. Average population sizes (&lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt;) were standardized to &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; = 1500 in absence of interannual environmental fluctuations, and were depressed as result of between-year fluctuations and directional environmental change as result of a fitness reduction in juveniles (a). All values of the total population size and trait values represent the arithmetic means over all of the 100 replicates that did not go extinct.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F4.medium.gif" width="440" height="422"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F4.medium.gif" width="440" height="422"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-42" class="first-child">The graphs show the results of individual-based simulations for the mean trait value <em>z</em> (a), the total population size <em>n</em><sub><em>t</em></sub> (b), the additive genetic variance <em>G</em><sub>2</sub> (c), and the juvenile number <em>n</em><sub>2</sub> (d) for the four life histories (<em>del-sem</em>, <em>del-ite</em>, <em>pre-ite</em>, <em>pre-sem</em>) over 600 years. Environmental change started at t=100 and was realized by a gradual shift in the phenotypic optimum (–– black dashed lines) from <em>θ</em> = 10 to <em>θ</em> = 12 by t=300 (a). The between-year environmental fluctuations were set to <em>ϵ</em> = 0.2 and life histories were standardized for <em>λ</em><sub><em>max</em></sub> = 1.15. Average population sizes (<em>n</em><sub><em>t</em></sub>) were standardized to <em>n</em><sub><em>t</em></sub> = 1500 in absence of interannual environmental fluctuations, and were depressed as result of between-year fluctuations and directional environmental change as result of a fitness reduction in juveniles (a). All values of the total population size and trait values represent the arithmetic means over all of the 100 replicates that did not go extinct.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The graphs illustrate the total population size nt (a), the variability in nt over time (90 % quantile of nt over 500 years, b), the speed of evolution (the average number of years to reach the trait value z = 11.8, c), and the additive genetic variance (G2, d) are shown depending on the interannual environmental variability (&#x3F5;). The simulation results are shown for four life histories (del-sem, del-ite, pre-ite, pre-sem) that have been standardized for two maximal population growth rates (&#x3BB;max = 1.10, &#x3BB;max = 1.15) with environmental change lasting for 200 years." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The graphs illustrate the total population size &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; (a), the variability in &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; over time (90 % quantile of &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; over 500 years, b), the speed of evolution (the average number of years to reach the trait value &lt;em&gt;z&lt;/em&gt; = 11.8, c), and the additive genetic variance (&lt;em&gt;G&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, d) are shown depending on the interannual environmental variability (&lt;em&gt;&#x3F5;&lt;/em&gt;). The simulation results are shown for four life histories (&lt;em&gt;del-sem&lt;/em&gt;, &lt;em&gt;del-ite&lt;/em&gt;, &lt;em&gt;pre-ite&lt;/em&gt;, &lt;em&gt;pre-sem&lt;/em&gt;) that have been standardized for two maximal population growth rates (&lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.10, &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.15) with environmental change lasting for 200 years.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F5.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F5.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-43" class="first-child">The graphs illustrate the total population size <em>n</em><sub><em>t</em></sub> (a), the variability in <em>n</em><sub><em>t</em></sub> over time (90 % quantile of <em>n</em><sub><em>t</em></sub> over 500 years, b), the speed of evolution (the average number of years to reach the trait value <em>z</em> = 11.8, c), and the additive genetic variance (<em>G</em><sub>2</sub>, d) are shown depending on the interannual environmental variability (<em>ϵ</em>). The simulation results are shown for four life histories (<em>del-sem</em>, <em>del-ite</em>, <em>pre-ite</em>, <em>pre-sem</em>) that have been standardized for two maximal population growth rates (<em>λ</em><sub><em>max</em></sub> = 1.10, <em>λ</em><sub><em>max</em></sub> = 1.15) with environmental change lasting for 200 years.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Population size sensitivity to environmental variability</h3><p id="p-44">The sensitivity of the total population size <em>n</em><sub><em>t</em></sub> to juvenile survival (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5a,b</a>) matched with our analytical predictions (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Tab. 1</a>). The biennial, two-stage life cycle (<em>pre-sem</em>) had the greatest sensitivity, shown in the decline of <em>n</em><sub><em>t</em></sub> with the strength of interannual environmental fluctuations (<em>ϵ</em>, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5a</a>) and in the fluctuations of <em>n</em><sub><em>t</em></sub> over time when the population was under stabilizing selection (i.e., 90% quantile of total population size, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5b</a>). Sensitivity of <em>n</em><sub><em>t</em></sub> to fluctuations decreased among life histories as iteroparity (<em>σ</em><sub>3</sub>) and maturation time (1 − <em>γ</em>) increased (see <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), from <em>pre-sem</em> to <em>pre-ite</em>, <em>del-sem</em>, and <em>del-ite</em> life histories. The standardization of the life histories for maximum growth rate <em>λ</em><sub><em>max</em></sub> did not affect the rank order of life-history strategies. Overall, the sensitivity and the temporal fluctuations of <em>n</em><sub><em>t</em></sub> decreased when we increased <em>λ</em><sub><em>max</em></sub> from 1.10 to 1.15 (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5a,b</a>). A weaker strength of selection (<em>ω</em><sup>2</sup> = 16) also attenuated the sensitivity and stochasticity of population size for all life histories (<a id="xref-fig-17-1" class="xref-fig" href="#F17">Fig. S11</a>).</p></div><div id="sec-15" class="subsection"><h3>Rate of directional trait evolution</h3><p id="p-45">As expected from our analytical predictions (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), the biennial life cycle (<em>pre-sem</em>) had the fastest evolutionary trait response to directional environmental change, while the <em>del-ite</em> life history evolved the slowest (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5c</a>). However, even though the differences were small, life histories standardized for <em>λ</em><sub><em>max</em></sub> = 1.15 evolved faster than life histories with <em>λ</em><sub><em>max</em></sub> = 1.10 (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 5c</a>), which was a deviation from the analytical predictions. Nevertheless, differences in evolvability were mainly found on a yearly, absolute time scales. Once scaled to generation time, all life histories took a similar number of <em>generations</em> to reach the same phenotypic threshold (<a id="xref-fig-16-1" class="xref-fig" href="#F16">Fig. S10c</a>).</p><p id="p-46">Although standing genetic variation <em>G</em><sub>2</sub> was similar across life-history strategies after burn-in (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Fig. 5d</a>), its temporal dynamics was strongly affected by the degree of iteroparity and delayed maturation. <em>G</em><sub>2</sub> increased fastest during adaptation to new average local conditions in the <em>pre-sem</em> and slowest in the <em>del-ite</em> life cycle (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4c</a>). The dynamics of <em>G</em><sub>2</sub> were also slowed down by delayed maturation (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4c</a>). Overall, larger interannual fluctuations tended to increase standing genetic variation in all life histories (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Fig. 5d</a>).</p></div><div id="sec-16" class="subsection"><h3>Extinction risk</h3><p id="p-47">The extinction risk of the population, evaluated as the total number of extinct replicates at a given time during the simulations (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6</a>), varied among life histories. In general, population extinctions were delayed relative to the onset of directional environmental change and still occurred after the environmental change stopped (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6a,c</a>). The less robust life-history strategies (e.g., <em>pre-sem</em>) experienced earlier extinction because of larger demographic stochasticity leading to earlier declines in <em>n</em><sub><em>t</em></sub> than robust life histories (e.g., <em>del-ite</em>; <a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4b,d</a>). Nevertheless, the iteroparous life histories (<em>pre-ite, del-ite</em>) had larger extinction risks when we prolonged the period of environmental change from 200 to 800 years, at least at low rate of random environmental fluctuations (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6c</a>). Under these conditions, the semelparous life cycles could adapt faster to the shifting trait optimum and reduce their phenotypic lag with it (results not shown). However, at high rates of interannual fluctuations, the bi-annual life cycle <em>pre-sem</em> always had the highest rate of extinction (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6b,d</a>). Overall, extinction risks were attenuated by having larger fecundity when setting <em>λ</em><sub><em>max</em></sub> = 1.15 (<a id="xref-fig-15-1" class="xref-fig" href="#F15">Fig. S9</a>) and when reducing the strength of selection to <em>ω</em><sup>2</sup> = 16, where no extinction occurred (<a id="xref-fig-18-1" class="xref-fig" href="#F18">Fig. S12</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="These graphs show the number of extinct replicates over time t (a,c) and the total number of extinct replicates depending on environmental variability &#x3F5; (b,d) for each of the four life histories. Extinction dynamics are shown for two different scenarios with environmental change either lasting for 200 years (a,b) or for 800 years (c,d). The gray dashed lines indicate the start of environmental change (t = 100) and its end (t = 300 for a,b; t = 900 for c,d). In both scenarios, the rate of environmental change per year was set to  and all life histories were standardized for a maximum growth rate of &#x3BB; = 1.10. The temporal dynamics are shown for an environmental variability of &#x3F5; = 0.5 (a) and &#x3F5; = 0.2 (c)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;These graphs show the number of extinct replicates over time &lt;em&gt;t&lt;/em&gt; (a,c) and the total number of extinct replicates depending on environmental variability &lt;em&gt;&#x3F5;&lt;/em&gt; (b,d) for each of the four life histories. Extinction dynamics are shown for two different scenarios with environmental change either lasting for 200 years (a,b) or for 800 years (c,d). The gray dashed lines indicate the start of environmental change (&lt;em&gt;t&lt;/em&gt; = 100) and its end (&lt;em&gt;t&lt;/em&gt; = 300 for a,b; &lt;em&gt;t&lt;/em&gt; = 900 for c,d). In both scenarios, the rate of environmental change per year was set to  and all life histories were standardized for a maximum growth rate of &lt;em&gt;&#x3BB;&lt;/em&gt; = 1.10. The temporal dynamics are shown for an environmental variability of &lt;em&gt;&#x3F5;&lt;/em&gt; = 0.5 (a) and &lt;em&gt;&#x3F5;&lt;/em&gt; = 0.2 (c).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F6.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F6.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <p id="p-48" class="first-child">These graphs show the number of extinct replicates over time <em>t</em> (a,c) and the total number of extinct replicates depending on environmental variability <em>ϵ</em> (b,d) for each of the four life histories. Extinction dynamics are shown for two different scenarios with environmental change either lasting for 200 years (a,b) or for 800 years (c,d). The gray dashed lines indicate the start of environmental change (<em>t</em> = 100) and its end (<em>t</em> = 300 for a,b; <em>t</em> = 900 for c,d). In both scenarios, the rate of environmental change per year was set to <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F6/embed/inline-graphic-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F6/embed/inline-graphic-46.gif"/></noscript></span></span> and all life histories were standardized for a maximum growth rate of <em>λ</em> = 1.10. The temporal dynamics are shown for an environmental variability of <em>ϵ</em> = 0.5 (a) and <em>ϵ</em> = 0.2 (c).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-17"><h2 class="">Discussion</h2><p id="p-49">Life-history strategies can impact a species’ persistence in a dynamic environment, especially by helping it to cope with short-term interannual fluctuations and by shaping its evolutionary response to long-term gradual changes (<a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Lande, 1982</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Hairston and De Stasio Jr., 1988</a>; <a id="xref-ref-2-10" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>). In this study, we asked (i) whether a tradeoff between a life history’s demographic robustness against environmental variability and its rate of trait evolution exists, (ii) how life-history characteristics determine the position of a species along the robustness-evolvability tradeoff, focusing on fast versus slow life cycles in species with overlapping generations, different degrees of iteroparity and timing of offspring maturation (dormancy), (iii) whether a genetic storage effect could alleviate the tradeoff, and (iv) how the tradeoff affects species persistence in fluctuating environments. Using existing theory from matrix population modeling and quantitative genetics, we confirmed numerically that a tradeoff exists. However, current theory does not account for stochasticity, density dependence, and demographic feedback on the genetic composition of a population. For these reasons, we ran stochastic individual-based simulations for a single population reaching a carrying capacity under negative density dependence. Here too, we could confirm the hypothesized tradeoff between a life history’s robustness and evolvability. Overall, we could show that, along the fast-slow continuum, short-lived semelparous species were more “evolvable” and could cope best with long-lasting directional environmental change, but experienced the highest extinction risk from interannual fluctuations and during the initial phase of environmental changes. They should thus suffer more from rapid anthropogenic climate changes than long-lived iteroparous species. We now discuss the effects and implications of the details of our models.</p><p id="p-50">A higher robustness against random and uncorrelated between-year environmental fluctuations was achieved by three life-history characteristics: Offspring dormancy, higher adult survival (leading to iteroparity), and a larger maximum growth rate. Offspring dormancy and higher adult survival helped to tolerate a variable environment by shielding a higher proportion of individuals from detrimental environmental conditions and spreading the risk of failed survival across years (e.g., see also <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Orzack and Tuljapurkar, 1989</a>; <a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Tuljapurkar, 1990</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Koons et al., 2008</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Zeineddine and Jansen, 2009</a>). In our simulations, offspring dormancy and iteroparity allowed the population to maintain larger sizes in face of fluctuations, reducing its demographic stochasticity, and thereby helping the population to cope with reduced offspring survival caused by interannual environmental variation. In addition, a larger maximum growth rate improved the robustness to environmental variability by helping to recover faster from extreme events (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Hallett et al., 2018</a>).</p><p id="p-51">Opposed to the means of promoting robustness, evolvability of life-history strategies benefited from lower dormancy in offspring and reduced adult survival, following two main effects. First, the rate of trait evolution in stage-structured populations generally increases when exposing more individuals to selection and improving their contribution to the next generation. From <a id="xref-disp-formula-5-5" class="xref-disp-formula" href="#disp-formula-5">Equations 5</a> and <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">6</a>, these two factors maximize the weight put on the amount of additive genetic variation <em>G</em><sub>2</sub> available for selection (i.e, the evolvability of the trait, see <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Houle, 1992</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hansen and Houle, 2008</a>). Therefore, life-histories without dormancy could mobilize a larger amount of genetic variation per unit of time and adapt faster to shifting environmental conditions. Increased adult mortality has a similar effect by increasing the rate of juvenile recruitment. <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kuparinen et al. (2010)</a> also showed this effect when simulating the evolution of a quantitative trait in Scots pine and Silver birch in populations with a fixed age structure. A higher mortality in adult trees (e.g., from senescence or forest management) favored faster recruitment of adapted juvenile trees and accelerated trait evolution in their simulations. A second effect favoring evolvability was shorter generation times (also described in <a id="xref-ref-2-11" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>; <a id="xref-ref-57-7" class="xref-bibr" href="#ref-57">Orive et al., 2017</a>). In our case, increased generation time, mostly controlled by the degree of iteroparity, was the main factor lowering the pace of adaptation to the shifting environment. Per unit of generation time, long-lived species experienced larger environmental shifts causing larger maladaptation and thus lower growth rate, eventually reducing juvenile recruitment rates sufficiently to cause larger extinction rates than for fast evolving, short-lived species. <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Osmond et al. (2017)</a> described this mechanism in predator-prey systems when predation reduced the generation time of the prey species and thus helped the prey to better adapt to novel environmental conditions. Overall, we could thus confirm our initial hypothesis that those life-history changes favoring evolvability simultaneously reduce robustness to interannual environmental variability, and vice versa.</p><p id="p-52">In our simulations, the genetic storage effect did not affect the amount of standing genetic variation at the start of directional environmental change and did not help to overcome the tradeoff between robustness and evolvability in long-lived species. Although all life histories had same standing genetic variation, they differed markedly in the evolution of additive genetic variance <em>G</em><sub>2</sub> during environmental change. The genetic variance <em>G</em><sub>2</sub> increased during directional selection in all simulations, as expected from the increase of allele frequencies of rare variants during adaptation towards a new trait optimum (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bürger, 1999</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Kopp and Matuszewski, 2014</a>). This rise in <em>G</em><sub>2</sub> happened most rapidly for the biennial life cycle and slowest for the life history with offspring dormancy and iteroparity. When a higher proportion of juveniles under selection allowed for faster allele frequency changes over time, the additive genetic variance rose more quickly, which further boosted evolution. Life histories thus shaped the rate of evolution via <em>G</em><sub>2</sub> differently from our initial expectations, in fact in the opposite direction, such that the tradeoff was not mitigated but enforced. Yet, the storage effect has been shown to speed up evolution with overlapping generations for very large levels of environmental fluctuations <em>ϵ</em> (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Yamamichi et al., 2019</a>). We could not impose such high levels of <em>ϵ</em> without driving populations to extinction. In our case, populations were under hard selection such that decreases in mean fitness translated into reduced offspring numbers, while <a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Yamamichi et al. (2019)</a> simulated a yearly offspring production that was insensitive to <em>ϵ</em>.</p><p id="p-53">To overcome the tradeoff, a higher maximum growth rate could allow life-history strategies to be more robust and at the same time more evolvable. In our models, a higher population growth rate <em>λ</em><sub><em>max</em></sub> improved robustness against environmental fluctuations by helping to recover faster from detrimental environmental events and avoid extinction, in line with previous theory (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lynch and Lande, 1993</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lande, 1993</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bürger and Lynch, 1995</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bürger, 1999</a>). Interestingly, a higher growth rate also caused faster trait evolution in our simulations, even though it had no effect on Δ<em>z</em> in our mathematical models. In the simulations, a higher <em>λ</em><sub><em>max</em></sub>, when realized by larger fecundity, was associated with more frequent mutation and recombination events. This effect of life-history parameters on the amount of mutational input (<a id="xref-fig-14-1" class="xref-fig" href="#F14">Fig. S8</a>) is a process not accounted for in previous analytical models, but should be of importance in natural systems.</p><p id="p-54">In conclusion, our results show that the feedback between evolutionary and demographic processes are critical for species persistence. A similar conclusion can be reached from models of evolutionary rescue (i.e., adaptation to single-shift environmental scenarios: <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lynch et al., 1991</a>; Lynch and <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Lande, 1993</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Bürger and Lynch, 1995</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gomulkiewicz and Holt, 1995</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Gomulkiewicz and Houle, 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chevin, 2013</a>), or from models of species persistence on temporally shifting environmental gradients (i.e., models of species’ range evolution: <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Polechová et al., 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Duputié et al., 2012</a>). However, these models often ignore short-term perturbations (but see <a id="xref-ref-57-8" class="xref-bibr" href="#ref-57">Orive et al., 2017</a>), assume constant population growth (i.e., with density-independent growth, but see <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boulding and Hay, 2001</a>), and do not incorporate explicit life cycles. On the other hand, quantitative genetics models of trait evolution in stage-structured populations with (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Engen et al., 2011</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">2013a</a>) or without (<a id="xref-ref-2-12" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>) stochastic environmental fluctuations do not model species persistence, to a few recent exceptions (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Marshall et al., 2016</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Cotto et al., 2019</a>). Importantly, no study to date explicitly addressed the tradeoff between evolvability and demographic robustness. We have shown that such a tradeoff exists and may limit the odds of evolutionary rescue in fast life-histories, while slow life-histories may persist longer while accumulating large adaptation lags at the cost of increased extinction debts (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Dullinger et al., 2012</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Cotto et al., 2017</a>).</p><div id="sec-18" class="subsection"><h3>Implications</h3><p id="p-55">We highlighted a fundamental property of eco-evolutionary dynamics whereby maximizing the evolutionary response of a population adapting to a fluctuating environment comes at the expense of its demographic robustness to variation in vital rates. Our conclusions are based on comparisons of eco-evolutionary dynamics of standardized life cycles. However, life-history strategies in natural systems are not standardized. In fact, systematic differences in more than one life-history characteristic exist. Species with low generation times often exhibit systematically higher population sizes <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-47.gif"/></noscript></span></span>, greater intrinsic growth rates, larger dispersal abilities, but lower competitive abilities than species with long generation times (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Petit and Hampe, 2006</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Buoro and Carlson, 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beckman et al., 2018</a>). They might thus be more robust to changes than predicted here. Similarly, long-lived species may suffer more from climate changes if adult mortality is increased, reducing their robustness. For instance, adult trees may be sensitive to extreme weather events such as drought or forest fires (e.g., <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Van Mantgem and Stephenson, 2007</a>). As we have shown, reducing adult survival would decrease robustness by modifying the life history strategy of the species towards semelparity. By simplifying the action of selection on a single life stage (e.g., on tree seedlings), we have not incorporated possible feedback between environmentally induced selection and life history characteristics. Such feed-back could change the life history of the species, as shown by <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Cotto et al. (2019)</a>, and move the species along the fast-slow life history tradeoff between evolvability and robustness. Instead, we have characterized archetype life-history strategies that span the fast-slow continuum and exemplified those possible outcomes. Our work thus shows the importance of merging population ecology with evolutionary quantitative genetics in a dynamical and stochastic modelling framework to characterize species’ vulnerability to environmental changes.</p></div></div><div class="section" id="sec-19"><h2 class="">Conflicts of interest</h2><p id="p-57">The authors declare no conflict of interest.</p></div><div class="section" id="sec-28"><h2 class="">Supplementary Material</h2><div class="section" id="sec-29"><h3>Supp A: 3-stage model with Beverton-Holt density regulation</h3><div class="section" id="sec-30"><h4>Equilibrium population size</h4><p id="p-78">The demographic dynamics were modeled via the population vector
<span class="disp-formula" id="disp-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-50.gif"/></noscript></span>

</span>
and the MPM
<span class="disp-formula" id="disp-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-51.gif"/></noscript></span>

</span>
such that <em>N</em> (<em>t</em> + 1) = <em>AN</em> (<em>t</em>). At equilibrium <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-66.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-67.gif"/></noscript></span></span>. The number of offspring, juveniles, and adults at equilibrium then could be derived as
<span class="disp-formula" id="disp-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-52.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-53.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-54.gif"/></noscript></span>

</span>
</p><p id="p-79">The total population size at equilibrium was computed from <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-68.gif"/></noscript></span></span>as
<span class="disp-formula" id="disp-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-55.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-31"><h4>Sensitivity of total population size to juvenile survival</h4><p id="p-80">The partial derivative of the total population size at equilibrium <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-69.gif"/></noscript></span></span> to juvenile survival (<em>σ</em><sub>2</sub>) was calculated as
<span class="disp-formula" id="disp-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-56.gif"/></noscript></span>

</span>
and could be simplified to
<span class="disp-formula" id="disp-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-57.gif"/></noscript></span>

</span>
</p><p id="p-81">To reach a certain total equilibrium population size <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-70.gif"/></noscript></span></span> the strenght of intra-specific competition can be standardized as follows
<span class="disp-formula" id="disp-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-58.gif"/></noscript></span>

</span>
</p><p id="p-82">By inserting <a id="xref-disp-formula-51-1" class="xref-disp-formula" href="#disp-formula-51">equation S9</a> in <a id="xref-disp-formula-50-1" class="xref-disp-formula" href="#disp-formula-50">S8</a> the population size sensitivity to juvenile survival could be standardized for <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-71.gif"/></noscript></span></span> as
<span class="disp-formula" id="disp-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-59.gif"/></noscript></span>

</span>
 such that the following partial derivative is obtained
<span class="disp-formula" id="disp-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-60.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-32"><h4>Rate of evolution at stable population size</h4><p id="p-83">The equation from <a id="xref-ref-2-17" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> could be adapted for our 3-stage life cycle as
<span class="disp-formula" id="disp-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-61.gif"/></noscript></span>

</span>
</p><p id="p-84">The juvenile proportion (<em>w</em><sub>2</sub>) can be derived from <a id="xref-disp-formula-46-1" class="xref-disp-formula" href="#disp-formula-46">equation S4</a> and <a id="xref-disp-formula-14-2" class="xref-disp-formula" href="#disp-formula-14">A6</a> as
<span class="disp-formula" id="disp-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-62.gif"/></noscript></span>

</span>
</p><p id="p-85">The probability of surviving intra-specific competition (<em>c</em>) can be derived by combining the Beverton-Holt function <em>c</em> = 1<em>/</em>(1 + <em>bn</em><sub>3</sub>) and <a id="xref-disp-formula-47-1" class="xref-disp-formula" href="#disp-formula-47">equation S5</a>
<span class="disp-formula" id="disp-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-63.gif"/></noscript></span>

</span>
</p><p id="p-86">The reproductive value of adults (<em>v</em><sub>3</sub>) can be derived from the left eigenvector (<em>v</em>) of the matrix population model (<em>A</em>) when solving <em>v</em><sup><em>T</em></sup> <em>A</em> = <em>λv</em><sup><em>T</em></sup> (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Caswell 2001</a>, chapter 4.6, equation 4.81). For a population at its stable population size (<em>λ</em> = 1) the reproductive value of adults is
<span class="disp-formula" id="disp-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-64.gif"/></noscript></span>

</span>
</p><p id="p-87">The normalization of <em>v</em><sub>3</sub> such that <em>v</em><sup><em>T</em></sup> <em>w</em> = 1 could be achieved as
<span class="disp-formula" id="disp-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-65.gif"/></noscript></span>

</span>
</p><p id="p-88">The rate of evolution then can be expressed as
<span class="disp-formula" id="disp-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-66.gif"/></noscript></span>

</span>
</p></div></div><div class="section" id="sec-33"><h3>Supp B: 2-stage model with Beverton-Holt density regulation</h3><div class="section" id="sec-34"><h4>Equilibrium population size</h4><p id="p-89">Population dynamics were modeled with the population vector
<span class="disp-formula" id="disp-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-67.gif"/></noscript></span>

</span>
 and the matrix population model
<span class="disp-formula" id="disp-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-68.gif"/></noscript></span>

</span>
 such that <em>N</em> (<em>t</em> + 1) = <em>An</em> with equilibrium <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-72.gif"/></noscript></span></span>.</p><p id="p-90">The number of juveniles and adults at equilibrium then could be derived as
<span class="disp-formula" id="disp-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-69.gif"/></noscript></span>

</span>
and
<span class="disp-formula" id="disp-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-70.gif"/></noscript></span>

</span>
with a total population size at equilibrium of
<span class="disp-formula" id="disp-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-71.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-35"><h4>Sensitivity of total population size to juvenile survival</h4><p id="p-91">The partial derivative of the total population size at equilibrium <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-73.gif"/></noscript></span></span> with respect to juvenile survival (<em>σ</em><sub>2</sub>) can be computed as
<span class="disp-formula" id="disp-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-72.gif"/></noscript></span>

</span>
 and simplified to
<span class="disp-formula" id="disp-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-73.gif"/></noscript></span>

</span>
</p><p id="p-92">The strength of intra-specific competition (<em>b</em>) could be standardized to reach a certain equilibrium population size <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-74.gif"/></noscript></span></span> via
<span class="disp-formula" id="disp-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-74.gif"/></noscript></span>

</span>
 while the fecundity (<em>ϕ</em>) could be used to standardize a life history for a specific maximum growth rate (<em>λ</em><sub><em>max</em></sub>) when
<span class="disp-formula" id="disp-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-75.gif"/></noscript></span>

</span>
and
<span class="disp-formula" id="disp-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-76.gif"/></noscript></span>

</span>
</p><p id="p-93">The sensitivity <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-75.gif"/></noscript></span></span> then could be reformulated as
<span class="disp-formula" id="disp-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-77.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-36"><h4>Rate of evolution at stable population size</h4><p id="p-94">The equation from <a id="xref-ref-2-18" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> can be adapted for our two-stage life cycle as
<span class="disp-formula" id="disp-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-78.gif"/></noscript></span>

</span>
</p><p id="p-95">The juvenile proportion can be derived from equation B3 and B5:
<span class="disp-formula" id="disp-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-79.gif"/></noscript></span>

</span>
and the probability of surviving intra-specific competition (<em>c</em>) can be derived by combining the Beverton-Holt function <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-76.gif"/></noscript></span></span> and equation B3:
<span class="disp-formula" id="disp-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-80.gif"/></noscript></span>

</span>
</p><p id="p-96">The reproductive value of adults for the 2-stage life history with Beverton-Holt density regulation is
<span class="disp-formula" id="disp-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-81.gif"/></noscript></span>

</span>
and can be normalize (<em>v</em><sup><em>T</em></sup> <em>w</em> = 1) to
<span class="disp-formula" id="disp-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-82.gif"/></noscript></span>

</span>
</p><p id="p-97">The rate of evolution for two-stage life history at population size equilibrium with BH density regulation then can be expressed as
<span class="disp-formula" id="disp-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-83.gif"/></noscript></span>

</span>
</p></div></div><div class="section" id="sec-37"><h3>Supp C: Plots mathematical model</h3><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="3 stages (standardized for nt + &#x3BB;max) The model results for the sensitivity of the total population size (, a-c), the rate of evolution (&#x394;z, d-f), the juvenile ratio (, g-i), and the sensitivity of the max. growth rate (&#x2202;&#x3BB;/&#x2202;&#x3C3;2, j-l) are shown depending on a life histories&#x2019; survival rate (&#x3C3;1,&#x3C3;3), fecundity (&#x3D5;), and maturation probability (&#x3B3;). The maximum growth rate has been standardized for &#x3BB;max = 1.1 by fecundity adjustments, except for the case when we varied fecundity (b,e,h,k). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (b). The following parameter values have been used if not other specified: &#x3C3;1 = 0.8, &#x3C3;2 = 0.8, &#x3C3;3 = 0.5, &#x3B3; = 0.5, G = 1, and &#x3B2;(z2) = 1. This graph is identical to figure 3 but shows additionally the juvenile ratio." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;3 stages (standardized for &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; + &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt;)&lt;/span&gt; The model results for the sensitivity of the total population size (, a-c), the rate of evolution (&#x394;&lt;em&gt;z&lt;/em&gt;, d-f), the juvenile ratio (, g-i), and the sensitivity of the max. growth rate (&lt;em&gt;&#x2202;&#x3BB;&lt;/em&gt;/&lt;em&gt;&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, j-l) are shown depending on a life histories&#x2019; survival rate (&lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;,&lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt;), fecundity (&lt;em&gt;&#x3D5;&lt;/em&gt;), and maturation probability (&lt;em&gt;&#x3B3;&lt;/em&gt;). The maximum growth rate has been standardized for &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.1 by fecundity adjustments, except for the case when we varied fecundity (b,e,h,k). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (&lt;strong&gt;b&lt;/strong&gt;). The following parameter values have been used if not other specified: &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = 0.8, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt; = 0.5, &lt;em&gt;&#x3B3;&lt;/em&gt; = 0.5, &lt;em&gt;G&lt;/em&gt; = 1, and &lt;em&gt;&#x3B2;&lt;/em&gt;(&lt;em&gt;z&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) = 1. This graph is identical to figure 3 but shows additionally the juvenile ratio.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F7.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F7.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1:</span> <span class="caption-title">3 stages (standardized for <em>n</em><sub><em>t</em></sub> + <em>λ</em><sub><em>max</em></sub>)</span><p id="p-98" class="first-child">The model results for the sensitivity of the total population size (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F7/embed/inline-graphic-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F7/embed/inline-graphic-77.gif"/></noscript></span></span>, a-c), the rate of evolution (Δ<em>z</em>, d-f), the juvenile ratio (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F7/embed/inline-graphic-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F7/embed/inline-graphic-78.gif"/></noscript></span></span>, g-i), and the sensitivity of the max. growth rate (<em>∂λ</em>/<em>∂σ</em><sub>2</sub>, j-l) are shown depending on a life histories’ survival rate (<em>σ</em><sub>1</sub>,<em>σ</em><sub>3</sub>), fecundity (<em>ϕ</em>), and maturation probability (<em>γ</em>). The maximum growth rate has been standardized for <em>λ</em><sub><em>max</em></sub> = 1.1 by fecundity adjustments, except for the case when we varied fecundity (b,e,h,k). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F7/embed/inline-graphic-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F7/embed/inline-graphic-79.gif"/></noscript></span></span> via adjustments of the strength of competition (<strong>b</strong>). The following parameter values have been used if not other specified: <em>σ</em><sub>1</sub> = 0.8, <em>σ</em><sub>2</sub> = 0.8, <em>σ</em><sub>3</sub> = 0.5, <em>γ</em> = 0.5, <em>G</em> = 1, and <em>β</em>(<em>z</em><sub>2</sub>) = 1. This graph is identical to <a id="xref-fig-3-5" class="xref-fig" href="#F3">figure 3</a> but shows additionally the juvenile ratio.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="2 stages (standardized for nt + &#x3BB;max) The sensitivity of the total population size (, a-b), the rate of evolution (&#x394;z, c-d), the juvenile ratio (, e-f), and the sensitivity of the max. growth rate (&#x2202;&#x3BB;/&#x2202;&#x3C3;2, g-h) are shown depending on the two-stage life cycles&#x2019; survival rate (&#x3C3;2) and fecundity (&#x3D5;). The maximum growth rate has been standardized for &#x3BB;max = 1.1 by fecundity adjustments, except for the case when we varied fecundity (b,d,f,h). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (b)., G = 1, and &#x3B2;(z2) = 1" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;2 stages (standardized for &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; + &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt;)&lt;/span&gt; The sensitivity of the total population size (, a-b), the rate of evolution (&#x394;&lt;em&gt;z&lt;/em&gt;, c-d), the juvenile ratio (, e-f), and the sensitivity of the max. growth rate (&lt;em&gt;&#x2202;&#x3BB;/&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, g-h) are shown depending on the two-stage life cycles&#x2019; survival rate (&lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) and fecundity (&lt;em&gt;&#x3D5;&lt;/em&gt;). The maximum growth rate has been standardized for &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.1 by fecundity adjustments, except for the case when we varied fecundity (b,d,f,h). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (&lt;strong&gt;b&lt;/strong&gt;)., &lt;em&gt;G&lt;/em&gt; = 1, and &lt;em&gt;&#x3B2;&lt;/em&gt;(&lt;em&gt;z&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) = 1&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F8.medium.gif" width="214" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F8.medium.gif" width="214" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2:</span> <span class="caption-title">2 stages (standardized for <em>n</em><sub><em>t</em></sub> + <em>λ</em><sub><em>max</em></sub>)</span><p id="p-99" class="first-child">The sensitivity of the total population size (<span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F8/embed/inline-graphic-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F8/embed/inline-graphic-80.gif"/></noscript></span></span>, a-b), the rate of evolution (Δ<em>z</em>, c-d), the juvenile ratio (<span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F8/embed/inline-graphic-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F8/embed/inline-graphic-81.gif"/></noscript></span></span>, e-f), and the sensitivity of the max. growth rate (<em>∂λ/∂σ</em><sub>2</sub>, g-h) are shown depending on the two-stage life cycles’ survival rate (<em>σ</em><sub>2</sub>) and fecundity (<em>ϕ</em>). The maximum growth rate has been standardized for <em>λ</em><sub><em>max</em></sub> = 1.1 by fecundity adjustments, except for the case when we varied fecundity (b,d,f,h). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F8/embed/inline-graphic-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F8/embed/inline-graphic-82.gif"/></noscript></span></span> via adjustments of the strength of competition (<strong>b</strong>)., <em>G</em> = 1, and <em>β</em>(<em>z</em><sub>2</sub>) = 1</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="3 stages (standardized for nt only) The model results for three-stage life cycles are shown for the sensitivity of the total population size (, a-c), the rate of evolution (&#x394;z, d-f), the juvenile ratio (, g-i), and the sensitivity of the max. growth rate (&#x2202;&#x3BB;/&#x2202;&#x3C3;2, j-l) depending on a life histories&#x2019; survival rate (&#x3C3;1,&#x3C3;3), fecundity (&#x3D5;), and maturation probability (&#x3B3;). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (b). However, the maximum growth rate has NOT been standardized in this case. The following parameter values have been used if not other specified: &#x3C3;1 = 0.8, &#x3C3;2 = 0.8, &#x3C3;3 = 0.5, &#x3D5; = 2, G = 1, and &#x3B2;(z2) = 1. This figure shows the same results as figure 3 but adds the proportion of juveniles (g-i)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;3 stages (standardized for &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; only)&lt;/span&gt; The model results for three-stage life cycles are shown for the sensitivity of the total population size (, a-c), the rate of evolution (&#x394;&lt;em&gt;z&lt;/em&gt;, d-f), the juvenile ratio (, g-i), and the sensitivity of the max. growth rate (&lt;em&gt;&#x2202;&#x3BB;&lt;/em&gt;/&lt;em&gt;&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, j-l) depending on a life histories&#x2019; survival rate (&lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;,&lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt;), fecundity (&lt;em&gt;&#x3D5;&lt;/em&gt;), and maturation probability (&lt;em&gt;&#x3B3;&lt;/em&gt;). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (&lt;strong&gt;b&lt;/strong&gt;). However, the maximum growth rate has NOT been standardized in this case. The following parameter values have been used if not other specified: &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = 0.8, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt; = 0.5, &lt;em&gt;&#x3D5;&lt;/em&gt; = 2, &lt;em&gt;G&lt;/em&gt; = 1, and &lt;em&gt;&#x3B2;&lt;/em&gt;(&lt;em&gt;z&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) = 1. This figure shows the same results as figure 3 but adds the proportion of juveniles (g-i).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F9.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F9.medium.gif" width="332" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3:</span> <span class="caption-title">3 stages (standardized for <em>n</em><sub><em>t</em></sub> only)</span><p id="p-100" class="first-child">The model results for three-stage life cycles are shown for the sensitivity of the total population size (<span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F9/embed/inline-graphic-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F9/embed/inline-graphic-83.gif"/></noscript></span></span>, a-c), the rate of evolution (Δ<em>z</em>, d-f), the juvenile ratio (<span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F9/embed/inline-graphic-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F9/embed/inline-graphic-84.gif"/></noscript></span></span>, g-i), and the sensitivity of the max. growth rate (<em>∂λ</em>/<em>∂σ</em><sub>2</sub>, j-l) depending on a life histories’ survival rate (<em>σ</em><sub>1</sub>,<em>σ</em><sub>3</sub>), fecundity (<em>ϕ</em>), and maturation probability (<em>γ</em>). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F9/embed/inline-graphic-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F9/embed/inline-graphic-85.gif"/></noscript></span></span> via adjustments of the strength of competition (<strong>b</strong>). However, the maximum growth rate has NOT been standardized in this case. The following parameter values have been used if not other specified: <em>σ</em><sub>1</sub> = 0.8, <em>σ</em><sub>2</sub> = 0.8, <em>σ</em><sub>3</sub> = 0.5, <em>ϕ</em> = 2, <em>G</em> = 1, and <em>β</em>(<em>z</em><sub>2</sub>) = 1. This figure shows the same results as <a id="xref-fig-3-6" class="xref-fig" href="#F3">figure 3</a> but adds the proportion of juveniles (g-i).</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="2 stages (standardized for nt only) The model results for two-stage life cycles illustrate the sensitivity of the total population size (, a-b), the rate of evolution (&#x394;z, c-d), the juvenile ratio (, e-f), and the sensitivity of the max. growth rate (&#x2202;&#x3BB;/&#x2202;&#x3C3;2, g-h) depending on a life histories&#x2019; survival rate (&#x3C3;2) and fecundity (&#x3D5;). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (b). However, the maximum growth rate has NOT been standardized for &#x3BB;max = 1.1 by fecundity adjustments. Juvenile survival was fixed to &#x3C3;2 = 0.8 with varying fecundity (b,d,f,h) and fecundity was set to &#x3D5; = 2 with varying &#x3C3;2 (a,c,e,g)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;2 stages (standardized for &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; only)&lt;/span&gt; The model results for two-stage life cycles illustrate the sensitivity of the total population size (, a-b), the rate of evolution (&#x394;&lt;em&gt;z&lt;/em&gt;, c-d), the juvenile ratio (, e-f), and the sensitivity of the max. growth rate (&lt;em&gt;&#x2202;&#x3BB;/&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, g-h) depending on a life histories&#x2019; survival rate (&lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) and fecundity (&lt;em&gt;&#x3D5;&lt;/em&gt;). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations  via adjustments of the strength of competition (&lt;strong&gt;b&lt;/strong&gt;). However, the maximum growth rate has NOT been standardized for &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.1 by fecundity adjustments. Juvenile survival was fixed to &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8 with varying fecundity (b,d,f,h) and fecundity was set to &lt;em&gt;&#x3D5;&lt;/em&gt; = 2 with varying &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; (a,c,e,g).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F10.medium.gif" width="214" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F10.medium.gif" width="214" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4:</span> <span class="caption-title">2 stages (standardized for <em>n</em><sub><em>t</em></sub> only)</span><p id="p-101" class="first-child">The model results for two-stage life cycles illustrate the sensitivity of the total population size (<span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F10/embed/inline-graphic-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F10/embed/inline-graphic-86.gif"/></noscript></span></span>, a-b), the rate of evolution (Δ<em>z</em>, c-d), the juvenile ratio (<span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F10/embed/inline-graphic-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F10/embed/inline-graphic-87.gif"/></noscript></span></span>, e-f), and the sensitivity of the max. growth rate (<em>∂λ/∂σ</em><sub>2</sub>, g-h) depending on a life histories’ survival rate (<em>σ</em><sub>2</sub>) and fecundity (<em>ϕ</em>). Each parameter combination has been standardized for the same equilibrium population size in absence of interannual environmental fluctuations <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F10/embed/inline-graphic-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F10/embed/inline-graphic-88.gif"/></noscript></span></span> via adjustments of the strength of competition (<strong>b</strong>). However, the maximum growth rate has NOT been standardized for <em>λ</em><sub><em>max</em></sub> = 1.1 by fecundity adjustments. Juvenile survival was fixed to <em>σ</em><sub>2</sub> = 0.8 with varying fecundity (b,d,f,h) and fecundity was set to <em>ϕ</em> = 2 with varying <em>σ</em><sub>2</sub> (a,c,e,g).</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="evolvability-robustness tradeoff with Beverton-Holt density regulation A life history&#x2019;s tradeoff between evolvability (&#x394;z) and robustness ( in a, and &#x2202;&#x3BB;/&#x2202;&#x3C3;2 in b) using a Beverton-Holt function for density regulation. Each circle illustrates a random combination of vital rates (with 0.2 &#x2264; &#x3C3;1 &#x2264; 1.0, &#x3C3;2 = 0.8, 0.0 &#x2264; &#x3C3;3 &#x2264; 0.9, 0.2 &#x2264; &#x3B3; &#x2264; 1.0). The fecundity (&#x3D5;) and the strength of competition (b) were re-calculated for each vital rate combination to standardize the maximum population growth rate (&#x3BB;max = 1.10 &#x2212; black, &#x3BB;max = 1.15 &#x2212; darkgray, &#x3BB;max = 1.20 &#x2212; lightgray) and the total population size at stable conditions (nt=1&#x2019;500). We then calculated the rate of evolution (&#x394;z, with G2&#x3B2;(z2) = 1), the population size sensitivity , and the growth rate sensitivity (&#x2202;&#x3BB;/&#x2202;&#x3C3;2) for each life-history variant. The three single points on the right of the plots were derived for the 2-stage life cycle (with &#x3C3;2 = 0.8), while the rest of the points illustrate parameter combinations for 3-stage life cycles." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;evolvability-robustness tradeoff with Beverton-Holt density regulation&lt;/span&gt; A life history&#x2019;s tradeoff between evolvability (&#x394;&lt;em&gt;z&lt;/em&gt;) and robustness ( in a, and &lt;em&gt;&#x2202;&#x3BB;&lt;/em&gt;/&lt;em&gt;&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; in b) using a Beverton-Holt function for density regulation. Each circle illustrates a random combination of vital rates (with 0.2 &#x2264; &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; &#x2264; 1.0, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8, 0.0 &#x2264; &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt; &#x2264; 0.9, 0.2 &#x2264; &lt;em&gt;&#x3B3;&lt;/em&gt; &#x2264; 1.0). The fecundity (&lt;em&gt;&#x3D5;&lt;/em&gt;) and the strength of competition (&lt;em&gt;b&lt;/em&gt;) were re-calculated for each vital rate combination to standardize the maximum population growth rate (&lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.10 &#x2212; black, &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.15 &#x2212; darkgray, &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.20 &#x2212; &lt;em&gt;lightgray&lt;/em&gt;) and the total population size at stable conditions (&lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt;=1&#x2019;500). We then calculated the rate of evolution (&#x394;&lt;em&gt;z&lt;/em&gt;, with &lt;em&gt;G&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;&lt;em&gt;&#x3B2;&lt;/em&gt;(&lt;em&gt;z&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) = 1), the population size sensitivity , and the growth rate sensitivity (&lt;em&gt;&#x2202;&#x3BB;/&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;) for each life-history variant. The three single points on the right of the plots were derived for the 2-stage life cycle (with &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8), while the rest of the points illustrate parameter combinations for 3-stage life cycles.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F11.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F11.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5:</span> <span class="caption-title">evolvability-robustness tradeoff with Beverton-Holt density regulation</span><p id="p-102" class="first-child">A life history’s tradeoff between evolvability (Δ<em>z</em>) and robustness (<span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F11/embed/inline-graphic-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F11/embed/inline-graphic-89.gif"/></noscript></span></span> in a, and <em>∂λ</em>/<em>∂σ</em><sub>2</sub> in b) using a Beverton-Holt function for density regulation. Each circle illustrates a random combination of vital rates (with 0.2 ≤ <em>σ</em><sub>1</sub> ≤ 1.0, <em>σ</em><sub>2</sub> = 0.8, 0.0 ≤ <em>σ</em><sub>3</sub> ≤ 0.9, 0.2 ≤ <em>γ</em> ≤ 1.0). The fecundity (<em>ϕ</em>) and the strength of competition (<em>b</em>) were re-calculated for each vital rate combination to standardize the maximum population growth rate (<em>λ</em><sub><em>max</em></sub> = 1.10 − black, <em>λ</em><sub><em>max</em></sub> = 1.15 − darkgray, <em>λ</em><sub><em>max</em></sub> = 1.20 − <em>lightgray</em>) and the total population size at stable conditions (<em>n</em><sub><em>t</em></sub>=1’500). We then calculated the rate of evolution (Δ<em>z</em>, with <em>G</em><sub>2</sub><em>β</em>(<em>z</em><sub>2</sub>) = 1), the population size sensitivity <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F11/embed/inline-graphic-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F11/embed/inline-graphic-90.gif"/></noscript></span></span>, and the growth rate sensitivity (<em>∂λ/∂σ</em><sub>2</sub>) for each life-history variant. The three single points on the right of the plots were derived for the 2-stage life cycle (with <em>σ</em><sub>2</sub> = 0.8), while the rest of the points illustrate parameter combinations for 3-stage life cycles.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="compare results for &#x394;z,, and &#x2202;&#x3BB;/&#x2202;&#x3C3;2 between models with Beverton-Holt and Ricker density regulation These graphs compare the model results for density regulation based on the Ricker function with the models based on the Beverton-Holt function, contrasting the rate of evolution (&#x394;z), the sensitivity of the total population size to juvenile survival , and the sensitivity of the population growth rate to juvenile survival (&#x2202;&#x3BB;/&#x2202;&#x3C3;2). Each dot is a random combination of vital rates with 0.2 &#x2264; &#x3C3;1 &#x2264; 1.0, &#x3C3;2 = 0.8, 0.0 &#x2264; &#x3C3;3 &#x2264; 0.9, 0.2 &#x2264; &#x3B3; &#x2264; 1.0." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;compare results for &#x394;&lt;em&gt;z&lt;/em&gt;,, and &lt;em&gt;&#x2202;&#x3BB;&lt;/em&gt;/&lt;em&gt;&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; between models with Beverton-Holt and Ricker density regulation&lt;/span&gt; These graphs compare the model results for density regulation based on the Ricker function with the models based on the Beverton-Holt function, contrasting the rate of evolution (&#x394;&lt;em&gt;z&lt;/em&gt;), the sensitivity of the total population size to juvenile survival , and the sensitivity of the population growth rate to juvenile survival (&lt;em&gt;&#x2202;&#x3BB;&lt;/em&gt;/&lt;em&gt;&#x2202;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;). Each dot is a random combination of vital rates with 0.2 &lt;em&gt;&#x2264; &#x3C3;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; &lt;em&gt;&#x2264;&lt;/em&gt; 1.0, &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 0.8, 0.0 &#x2264; &lt;em&gt;&#x3C3;&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt; &#x2264; 0.9, 0.2 &#x2264; &lt;em&gt;&#x3B3;&lt;/em&gt; &#x2264; 1.0.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F12.medium.gif" width="440" height="156"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F12.medium.gif" width="440" height="156"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S6:</span> <span class="caption-title">compare results for Δ<em>z</em>,<span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F12/embed/inline-graphic-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F12/embed/inline-graphic-91.gif"/></noscript></span></span>, and <em>∂λ</em>/<em>∂σ</em><sub>2</sub> between models with Beverton-Holt and Ricker density regulation</span><p id="p-103" class="first-child">These graphs compare the model results for density regulation based on the Ricker function with the models based on the Beverton-Holt function, contrasting the rate of evolution (Δ<em>z</em>), the sensitivity of the total population size to juvenile survival <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F12/embed/inline-graphic-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F12/embed/inline-graphic-92.gif"/></noscript></span></span>, and the sensitivity of the population growth rate to juvenile survival (<em>∂λ</em>/<em>∂σ</em><sub>2</sub>). Each dot is a random combination of vital rates with 0.2 <em>≤ σ</em><sub>1</sub> <em>≤</em> 1.0, <em>σ</em><sub>2</sub> = 0.8, 0.0 ≤ <em>σ</em><sub>3</sub> ≤ 0.9, 0.2 ≤ <em>γ</em> ≤ 1.0.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population growth over time for all four life histories and both levels of &#x3BB;max This graph illustrates the growth trajectories for all four life histories (del-ite, del-sem, pre-sem, pre-ite) and both maximum growth rates (&#x3BB;max = 1.10 and &#x3BB;max = 1.15) over time. All life histories here start at the same total population size (nt = 30) and grow to the same carrying capacity of 1500 individuals (nt = n1 + n2 + n3). The population growth trajectories are quite similar across life histories with the same &#x3BB;max, while &#x3BB;max = 1.15 results in a faster increase in nt than a standardization for &#x3BB;max = 1.10. In the beginning, stage structure was adjusted to the SSD of the respective life history." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Population growth over time for all four life histories and both levels of &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt;&lt;/span&gt; This graph illustrates the growth trajectories for all four life histories (&lt;em&gt;del-ite&lt;/em&gt;, &lt;em&gt;del-sem&lt;/em&gt;, &lt;em&gt;pre-sem&lt;/em&gt;, &lt;em&gt;pre-ite&lt;/em&gt;) and both maximum growth rates (&lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.10 and &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.15) over time. All life histories here start at the same total population size (&lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; = 30) and grow to the same carrying capacity of 1500 individuals (&lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; = &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; + &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; + &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt;). The population growth trajectories are quite similar across life histories with the same &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt;, while &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.15 results in a faster increase in &lt;em&gt;n&lt;/em&gt;&lt;sub&gt;&lt;em&gt;t&lt;/em&gt;&lt;/sub&gt; than a standardization for &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.10. In the beginning, stage structure was adjusted to the SSD of the respective life history.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F13.medium.gif" width="428" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S7:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F13.medium.gif" width="428" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S7:</span> <span class="caption-title">Population growth over time for all four life histories and both levels of <em>λ</em><sub><em>max</em></sub></span><p id="p-104" class="first-child">This graph illustrates the growth trajectories for all four life histories (<em>del-ite</em>, <em>del-sem</em>, <em>pre-sem</em>, <em>pre-ite</em>) and both maximum growth rates (<em>λ</em><sub><em>max</em></sub> = 1.10 and <em>λ</em><sub><em>max</em></sub> = 1.15) over time. All life histories here start at the same total population size (<em>n</em><sub><em>t</em></sub> = 30) and grow to the same carrying capacity of 1500 individuals (<em>n</em><sub><em>t</em></sub> = <em>n</em><sub>1</sub> + <em>n</em><sub>2</sub> + <em>n</em><sub>3</sub>). The population growth trajectories are quite similar across life histories with the same <em>λ</em><sub><em>max</em></sub>, while <em>λ</em><sub><em>max</em></sub> = 1.15 results in a faster increase in <em>n</em><sub><em>t</em></sub> than a standardization for <em>λ</em><sub><em>max</em></sub> = 1.10. In the beginning, stage structure was adjusted to the SSD of the respective life history.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-38"><h2 class="">Supp D: Plots individual-based simulations</h2><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolution of G2 with &#x3BB; = 1.15 Simulation results for the evolution of additive genetic variance during environmental change are shown for the four life histories, the two different duration of environmental change (200 years - a,c; 800 years - b,d) and two degrees of interannual environmental variability (&#x3F5; = 0.2 - a,b; &#x3F5; = 1.0 - c,d). The simulation results were averaged over all of the 100 replicates that did not go extinct." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Evolution of &lt;em&gt;G&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; with &lt;em&gt;&#x3BB;&lt;/em&gt; = 1.15&lt;/span&gt; Simulation results for the evolution of additive genetic variance during environmental change are shown for the four life histories, the two different duration of environmental change (200 years - a,c; 800 years - b,d) and two degrees of interannual environmental variability (&lt;em&gt;&#x3F5;&lt;/em&gt; = 0.2 - a,b; &lt;em&gt;&#x3F5;&lt;/em&gt; = 1.0 - c,d). The simulation results were averaged over all of the 100 replicates that did not go extinct.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F14.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S8:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F14.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S8:</span> <span class="caption-title">Evolution of <em>G</em><sub>2</sub> with <em>λ</em> = 1.15</span><p id="p-105" class="first-child">Simulation results for the evolution of additive genetic variance during environmental change are shown for the four life histories, the two different duration of environmental change (200 years - a,c; 800 years - b,d) and two degrees of interannual environmental variability (<em>ϵ</em> = 0.2 - a,b; <em>ϵ</em> = 1.0 - c,d). The simulation results were averaged over all of the 100 replicates that did not go extinct.</p><div class="sb-div caption-clear"></div></div></div><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Extinction dynamics with &#x3BB; = 1.15 These graphs show the same simulation results on the number of extinct replicates as figure 6, but illustrate additionally the results for the life-history standardization of &#x3BB;max = 1.15. These graphs show for each of the four life histories the number of extinct replicates over time t (a,c) and the total number of extinct replicates depending on environmental variability &#x3F5; (b,d). Extinction dynamics are shown for two different scenarios with environmental change either lasting for 200 years (a,b) or for 800 years (c,d). The gray dashed lines indicate the start of environmental change (t = 100) and its end (t = 300 for a,b; t = 900 for c,d). In both scenarios, the rate of environmental change per year was set to . The temporal dynamics are shown for an environmental variability of &#x3F5; = 0.5 (a) and &#x3F5; = 0.2 (c)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Extinction dynamics with &lt;em&gt;&#x3BB;&lt;/em&gt; = 1.15&lt;/span&gt; These graphs show the same simulation results on the number of extinct replicates as figure 6, but illustrate additionally the results for the life-history standardization of &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.15. These graphs show for each of the four life histories the number of extinct replicates over time &lt;em&gt;t&lt;/em&gt; (a,c) and the total number of extinct replicates depending on environmental variability &lt;em&gt;&#x3F5;&lt;/em&gt; (b,d). Extinction dynamics are shown for two different scenarios with environmental change either lasting for 200 years (a,b) or for 800 years (c,d). The gray dashed lines indicate the start of environmental change (&lt;em&gt;t&lt;/em&gt; = 100) and its end (&lt;em&gt;t&lt;/em&gt; = 300 for a,b; &lt;em&gt;t&lt;/em&gt; = 900 for c,d). In both scenarios, the rate of environmental change per year was set to . The temporal dynamics are shown for an environmental variability of &lt;em&gt;&#x3F5;&lt;/em&gt; = 0.5 (a) and &lt;em&gt;&#x3F5;&lt;/em&gt; = 0.2 (c).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F15.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S9:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F15.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S9:</span> <span class="caption-title">Extinction dynamics with <em>λ</em> = 1.15</span><p id="p-106" class="first-child">These graphs show the same simulation results on the number of extinct replicates as <a id="xref-fig-6-5" class="xref-fig" href="#F6">figure 6</a>, but illustrate additionally the results for the life-history standardization of <em>λ</em><sub><em>max</em></sub> = 1.15. These graphs show for each of the four life histories the number of extinct replicates over time <em>t</em> (a,c) and the total number of extinct replicates depending on environmental variability <em>ϵ</em> (b,d). Extinction dynamics are shown for two different scenarios with environmental change either lasting for 200 years (a,b) or for 800 years (c,d). The gray dashed lines indicate the start of environmental change (<em>t</em> = 100) and its end (<em>t</em> = 300 for a,b; <em>t</em> = 900 for c,d). In both scenarios, the rate of environmental change per year was set to <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F15/embed/inline-graphic-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/F15/embed/inline-graphic-93.gif"/></noscript></span></span>. The temporal dynamics are shown for an environmental variability of <em>ϵ</em> = 0.5 (a) and <em>ϵ</em> = 0.2 (c).</p><div class="sb-div caption-clear"></div></div></div><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolvability per generation These graphs are identical to Fig. 5, except that the speed of evolution (c) is quantified as the number of generations (not years!) to reach 90% of the novel phenotypic optimum after climate change. For this purpose, the number of years to reach the threshold was divided by the average generation time of each life-history strategy at stable conditions from table 1. Please not that the realized generation time in the course of environmental change might differ from these estimates that have been calculated at equilibrium." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Evolvability per generation&lt;/span&gt; These graphs are identical to Fig. 5, except that the speed of evolution (c) is quantified as the number of generations (not years!) to reach 90% of the novel phenotypic optimum after climate change. For this purpose, the number of years to reach the threshold was divided by the average generation time of each life-history strategy at stable conditions from table 1. Please not that the realized generation time in the course of environmental change might differ from these estimates that have been calculated at equilibrium.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F16.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S10:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F16.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S10:</span> <span class="caption-title">Evolvability per generation</span><p id="p-107" class="first-child">These graphs are identical to <a id="xref-fig-5-10" class="xref-fig" href="#F5">Fig. 5</a>, except that the speed of evolution (c) is quantified as the number of generations (not years!) to reach 90% of the novel phenotypic optimum after climate change. For this purpose, the number of years to reach the threshold was divided by the average generation time of each life-history strategy at stable conditions from <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">table 1</a>. Please not that the realized generation time in the course of environmental change might differ from these estimates that have been calculated at equilibrium.</p><div class="sb-div caption-clear"></div></div></div><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Robustness and evolvability with &#x3C9;2 = 16 The simulation results for strong selection (&#x3C9;2 = 4) are plotted together with the results for weak selection (&#x3C9;2 = 16). Weaker selection strength leads to higher population sizes after burn-in (a), lower stochasticity in total population sizes over time (b), a slower speed of evolution (c), and higher levels of additive genetic variance in the juvenile trait (d). However, the rank order of life-history strategies with regard to robustness and evolvability is preserved." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Robustness and evolvability with &lt;em&gt;&#x3C9;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 16&lt;/span&gt; The simulation results for strong selection (&lt;em&gt;&#x3C9;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 4) are plotted together with the results for weak selection (&lt;em&gt;&#x3C9;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 16). Weaker selection strength leads to higher population sizes after burn-in (a), lower stochasticity in total population sizes over time (b), a slower speed of evolution (c), and higher levels of additive genetic variance in the juvenile trait (d). However, the rank order of life-history strategies with regard to robustness and evolvability is preserved.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F17.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S11:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F17.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S11:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S11:</span> <span class="caption-title">Robustness and evolvability with <em>ω</em><sup>2</sup> = 16</span><p id="p-108" class="first-child">The simulation results for strong selection (<em>ω</em><sup>2</sup> = 4) are plotted together with the results for weak selection (<em>ω</em><sup>2</sup> = 16). Weaker selection strength leads to higher population sizes after burn-in (a), lower stochasticity in total population sizes over time (b), a slower speed of evolution (c), and higher levels of additive genetic variance in the juvenile trait (d). However, the rank order of life-history strategies with regard to robustness and evolvability is preserved.</p><div class="sb-div caption-clear"></div></div></div><div id="F18" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F18.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Extinction dynamics with &#x3C9;2 = 16 The effects of a stronger (&#x3C9;2 = 4, a-b) versus weaker (&#x3C9;2 = 16, c-d) selection on extinction probabilities is shown for the environmental change scenario over 200 years, similar to Fig. 6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Extinction dynamics with &lt;em&gt;&#x3C9;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 16&lt;/span&gt; The effects of a stronger (&lt;em&gt;&#x3C9;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 4, a-b) versus weaker (&lt;em&gt;&#x3C9;&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 16, c-d) selection on extinction probabilities is shown for the environmental change scenario over 200 years, similar to Fig. 6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S12:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F18.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S12:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F18.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S12:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S12:</span> <span class="caption-title">Extinction dynamics with <em>ω</em><sup>2</sup> = 16</span><p id="p-109" class="first-child">The effects of a stronger (<em>ω</em><sup>2</sup> = 4, a-b) versus weaker (<em>ω</em><sup>2</sup> = 16, c-d) selection on extinction probabilities is shown for the environmental change scenario over 200 years, similar to <a id="xref-fig-6-6" class="xref-fig" href="#F6">Fig. 6</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F19" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F19.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulated life histories along fast-slow continuum and reproductive strategy The generation time (T) of each simulated life history is plotted against its net reproductive rate (R0) from table 1. While the blue circles illustrate life histories standardized for &#x3BB;max = 1.10, orange dots represent life histories with &#x3BB;max = 1.15. While the x-axis could be interpreted to represent the fast-slow continuum, the y-axis might rather illustrate the reproductive strategy axis as described in meta-analyses (Salguero-G&#xF3;mez et al., 2017). Please note that both quantities (T and R0) were computed in absence of density regulation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2082123611" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Simulated life histories along fast-slow continuum and reproductive strategy&lt;/span&gt; The generation time (&lt;em&gt;T&lt;/em&gt;) of each simulated life history is plotted against its net reproductive rate (&lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt;) from table 1. While the blue circles illustrate life histories standardized for &lt;em&gt;&#x3BB;&lt;sub&gt;max&lt;/sub&gt;&lt;/em&gt; = 1.10, orange dots represent life histories with &lt;em&gt;&#x3BB;&lt;/em&gt;&lt;sub&gt;&lt;em&gt;max&lt;/em&gt;&lt;/sub&gt; = 1.15. While the x-axis could be interpreted to represent the fast-slow continuum, the y-axis might rather illustrate the reproductive strategy axis as described in meta-analyses (Salguero-G&#xF3;mez et al., 2017). Please note that both quantities (&lt;em&gt;T&lt;/em&gt; and &lt;em&gt;R&lt;/em&gt;&lt;sub&gt;0&lt;/sub&gt;) were computed in absence of density regulation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S13:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F19.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S13:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F19.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S13:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/04/834234/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S13:</span> <span class="caption-title">Simulated life histories along fast-slow continuum and reproductive strategy</span><p id="p-110" class="first-child">The generation time (<em>T</em>) of each simulated life history is plotted against its net reproductive rate (<em>R</em><sub>0</sub>) from <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">table 1</a>. While the blue circles illustrate life histories standardized for <em>λ<sub>max</sub></em> = 1.10, orange dots represent life histories with <em>λ</em><sub><em>max</em></sub> = 1.15. While the x-axis could be interpreted to represent the fast-slow continuum, the y-axis might rather illustrate the reproductive strategy axis as described in meta-analyses (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Salguero-Gómez et al., 2017</a>). Please note that both quantities (<em>T</em> and <em>R</em><sub>0</sub>) were computed in absence of density regulation.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-56">We thank Thomas F. Hansen, Luis-Miguel Chevin, and Richard Gomulkiewicz for insightful discussions and comments on the manuscript. MS and FG were supported by grant PP00P3 176965 from the Swiss National Science Foundation to FG.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2></div><div class="section app" id="app-2"><h2 class="">Appendix A: 3-stage model</h2><div class="section" id="sec-20"><h3>Equilibrium population size</h3><p id="p-58">The populations’ demographic dynamics were modeled via the population vector
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-16.gif"/></noscript></span>

</span>
and the matrix population model
<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-17.gif"/></noscript></span>

</span>
such that <em>N</em> (<em>t</em> + 1) = <em>AN</em> (<em>t</em>). By solving for the stage densities at equilibrium when <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-48.gif"/></noscript></span></span>,with <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-49.gif"/></noscript></span></span>, the individual numbers within the offspring, juvenile, and adult stage at equilibrium could be derived as
<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-18.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-19.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-20.gif"/></noscript></span>

</span>
</p><p id="p-59">The total population size at equilibrium was computed from <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-50.gif"/></noscript></span></span> as
<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-21.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-21"><h3>Sensitivity of total population size to juvenile survival</h3><p id="p-60">The partial derivative of the total population size at equilibrium <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-51.gif"/></noscript></span></span> to juvenile survival (<em>σ</em><sub>2</sub>) was calculated as
<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-22.gif"/></noscript></span>

</span>
which could be simplified to
<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-23.gif"/></noscript></span>

</span>
</p><p id="p-61">To standardize a life history for a specific total equilibrium population size <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-52.gif"/></noscript></span></span> the strength of intra-specific competition was adjusted as follows
<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-24.gif"/></noscript></span>

</span>
</p><p id="p-62">By inserting <a id="xref-disp-formula-17-2" class="xref-disp-formula" href="#disp-formula-17">equation A9</a> in <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">equation A8</a>, the population size sensitivity can be expressed depending on the equilibrium population size as
<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-25.gif"/></noscript></span>

</span>
such that the following partial derivative is obtained
<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-26.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-22"><h3>Rate of evolution at equilibrium population size</h3><p id="p-63">We adapted <a id="xref-disp-formula-8-5" class="xref-disp-formula" href="#disp-formula-8">equation 8</a> from <a id="xref-ref-2-13" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> for our three-stage life cycle to describe the rate of evolution depending on the juvenile proportion (<em>w</em><sub>2</sub>), juvenile survival (<em>σ</em><sub>2</sub>), competition survival (<em>c</em>), the reproductive value of adults (<em>v</em><sub>3</sub>), the additive genetic variance in juveniles (<em>G</em><sub>2</sub>), and the selection gradient on a juvenile trait (<em>β</em>(<em>z</em><sub>2</sub>)) as
<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-27.gif"/></noscript></span>

</span>
</p><p id="p-64">The juvenile proportion (<em>w</em><sub>2</sub>) could be derived from <a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">equation A4</a> and <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">A6</a> as
<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-28.gif"/></noscript></span>

</span>
</p><p id="p-65">The probability of surviving intra-specific competition (<em>c</em>) was derived by combining the Ricker function <em>c</em> = <em>exp</em>(−<em>bn</em><sub>3</sub>) and <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">equation A5</a> as
<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-29.gif"/></noscript></span>

</span>
</p><p id="p-66">The reproductive value of adults (<em>v</em><sub>3</sub>) could be derived from the left eigenvector (<em>v</em>) of the matrix population model (<em>A</em>) when solving <em>vA</em> = <em>λv</em> (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Caswell, 2001</a>, equation 4.81). For a population at equilibrium population size (<em>λ</em> = 1) the reproductive value of adults can be derived as
<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-30.gif"/></noscript></span>

</span>
and normalized to
<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-31.gif"/></noscript></span>

</span>
such that <em>v</em> ∗ <em>w</em> = 1 (as in <a id="xref-ref-2-14" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>). The rate of evolution depending on the single vital rates then could be expressed as
<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-32.gif"/></noscript></span>

</span>
</p></div></div><div class="section app" id="app-3"><h2 class="">Appendix B: 2-stage model</h2><div class="section" id="sec-23"><h3>Equilibrium population size</h3><p id="p-67">The demographic dynamics for a 2-stage life history with non-overlapping generations were modeled by the population vector
<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-33.gif"/></noscript></span>

</span>
 and the matrix population model
<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-34.gif"/></noscript></span>

</span>
such that <em>N</em> (<em>t</em> + 1) = <em>AN</em>. By solving for the individual numbers within stages at equilibrium when <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-54.gif"/></noscript></span></span> the juvenile and adult numbers at equilibrium were derived as
<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-35.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-36.gif"/></noscript></span>

</span>
 (with natural logarithm log()). The total population size at equilibrium was computed from <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-55.gif"/></noscript></span></span> as
<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-37.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-24"><h3>Sensitivity of total population size to juvenile survival</h3><p id="p-68">The partial derivative of the total population size at equilibrium <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-56.gif"/></noscript></span></span> to juvenile survival <em>σ</em><sub>2</sub> was derived as
<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-38.gif"/></noscript></span>

</span>
 which could be simplified to
<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-39.gif"/></noscript></span>

</span>
</p><p id="p-69">To reach a certain total equilibrium population size <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-57.gif"/></noscript></span></span> the strength of intra-specific competition (<em>b</em>) was standardized as follows
<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-40.gif"/></noscript></span>

</span>
</p><p id="p-70">Given that the maximum growth rate <em>λ</em><sub><em>max</em></sub> could be calculated from
<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-41.gif"/></noscript></span>

</span>
life histories could be standardized for <em>λ</em><sub><em>max</em></sub> via <em>ϕ</em> adjustments as
<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-42.gif"/></noscript></span>

</span>
such that the sensitivity <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-58.gif"/></noscript></span></span> could be generalized to
<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-43.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-25"><h3>Rate of evolution at stable population size</h3><p id="p-71">To describe the rate of evolution we adapted <a id="xref-disp-formula-8-6" class="xref-disp-formula" href="#disp-formula-8">equation 8</a> from <a id="xref-ref-2-15" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> for our two-stage life cycle as
<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-44.gif"/></noscript></span>

</span>
</p><p id="p-72">The juvenile proportion could be derived from <a id="xref-disp-formula-28-1" class="xref-disp-formula" href="#disp-formula-28">equation B3</a> and <a id="xref-disp-formula-30-1" class="xref-disp-formula" href="#disp-formula-30">B5</a>:
<span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-45.gif"/></noscript></span>

</span>
</p><p id="p-73">The probability of surviving intra-specific competition (<em>c</em>) was derived by combining the Ricker function <em>c</em> = <em>exp</em>(−<em>bn</em><sub>2</sub>) and <a id="xref-disp-formula-28-2" class="xref-disp-formula" href="#disp-formula-28">equation B3</a>:
<span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-46.gif"/></noscript></span>

</span>
</p><p id="p-74">The reproductive value of adults (<em>v</em><sub>3</sub>) could be derived from the left eigenvector (<em>v</em>) of the matrix population model (<em>A</em>) when solving <em>vA</em> = <em>λv</em> (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Caswell, 2001</a>, equation 4.81). For a population at equilibrium (<em>λ</em> = 1) the reproductive value of adults could be derived as
<span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-47.gif"/></noscript></span>

</span>
and normalized to
<span class="disp-formula" id="disp-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-48.gif"/></noscript></span>

</span>
such that <em>v</em> ∗ <em>w</em> = 1 (as in <a id="xref-ref-2-16" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>). The rate of evolution depending on the single vital rates then could be expressed as
<span class="disp-formula" id="disp-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/graphic-49.gif"/></noscript></span>

</span>
</p></div></div><div class="section app" id="app-4"><h2 class="">Appendix C: The standardization of life-history strategies</h2><div class="section" id="sec-26"><h3>standardization</h3><p id="p-75">A critical aspect for the interpretation of our model and simulation results was the standardization of life-history strategies for uniform total population sizes <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-59.gif"/></noscript></span></span> at stable conditions and for homogeneous maximum growth rates <em>λ</em><sub><em>max</em></sub>. The population size standardization was necessary for two reasons. First, the environmental sensitivity <em>∂n</em><sub><em>t</em></sub>/<em>∂σ</em><sub>2</sub> varied with the total population size (see <a id="xref-disp-formula-8-7" class="xref-disp-formula" href="#disp-formula-8">equation 8</a>), such that we had to measure it in the mathematical models at the same <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-60.gif"/></noscript></span></span> to achieve informative comparisons between life histories. Second, the population size standardization allowed us to make <em>∂n</em><sub><em>t</em></sub>/<em>∂σ</em><sub>2</sub> an informative proxy for extinction risk from short-term environmental variability. The population size reduction from environmental fluctuations determined the distance to critically low population sizes, the levels of genetic and demographic stochasticity, and thus fed back on the success rate of evolutionary rescue (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Gomulkiewicz and Holt, 1995</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Willi and Hoffmann, 2009</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lanfear et al., 2014</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Carlson et al., 2014</a>; <a id="xref-ref-57-9" class="xref-bibr" href="#ref-57">Orive et al., 2017</a>). While we are convinced that a population size standardization was necessary for this kind of study, we want to highlight that other kinds of standardization, for instance for the effective population size <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-61.gif"/></noscript></span></span> (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Caswell et al., 2004</a>) or for uniform adult numbers <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-62.gif"/></noscript></span></span>, could make sense as well.</p><p id="p-76">A life-history standardization for uniform maximum population growth rates was not mandatory, but highly useful. While the growth rate at population size equilibrium is always unity (<em>λ</em> = 1), the maximum population growth rate <em>λ</em><sub><em>max</em></sub> is an important demographic parameter with regard to a populations’ recovery subsequent to detrimental environmental conditions (Lynch and <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Lande, 1993</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Chevin et al., 2010</a>). By running two sets of simulations with distinct levels of <em>λ</em><sub><em>max</em></sub> we could study the levels of robustness and evolvability not only along the axis of the fast-slow continuum (e.g., generation time <em>T</em>) but also along the axis of reproductive strategy as well (e.g., net reproductive rate <em>R</em><sub>0</sub>). This distinction revealed that the tradeoff between robustness and evolvability was only present when life histories with the same <em>λ</em><sub><em>max</em></sub> were compared to each other. Instead, both properties could be increased or decreased simultaneously when changes in vital rates translated into changes in <em>λ</em><sub><em>max</em></sub> (e.g., when moving to the bottom right or top-left corner in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2a</a>). Thus, some life histories could be more evolvable and, at the same time, more robust than others. We thus conclude that the described tradeoff is not a “hard” tradeoff but could be overcome, at least partly, by life-history evolution towards higher maximum growth rates (<em>λ</em><sub><em>max</em></sub>) along the reproductive strategy axis of life histories.</p></div><div class="section" id="sec-27"><h3>Two measures to quantify robustness against environmental variability</h3><p id="p-77">Even though we used two different robustness measures, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-63.gif"/></noscript></span></span> and <em>∂λ</em>/<em>∂σ</em><sub>2</sub>, we suggest to not use them interchangeably given that they responded in opposite directions to some vital rate changes (see also <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Takada and Nakajima, 1998</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Grant and Benton, 2000</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Caswell et al., 2004</a>). In our mathematical models, increasing a life history’s maximum growth rate (e.g., to <em>λ</em><sub><em>max</em></sub>=1.15) resulted in a lower sensitivity of total population size <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-64.gif"/></noscript></span></span> but in a higher sensitivity of the effective population size (<em>∂λ</em>/<em>∂σ</em><sub>2</sub>), such that one measure indicated an increase in robustness, and the other a decrease. Hence, it is unclear which measure is better suited to quantify a life history’s robustness against interannual environmental fluctuations. On the one hand, <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/04/834234/embed/inline-graphic-65.gif"/></noscript></span></span> was much better correlated to the extinction risk from short-term environmental variation than <em>∂λ</em>/<em>∂σ</em><sub>2</sub> levels. On the other hand, this correlation might vanish, or even be reversed, when life-history strategies would have been standardized for the same ‘effective’ population size, instead of the same ‘total’ population size. We thus don’t suggest to favor one measure over the other, but recommend a careful interpretation and further theoretical investigations.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><span class="em-link"><span class="em-addr">maxschmid{at}posteo.de</span></span>, <span class="em-link"><span class="em-addr">maria.paniw{at}ieu.uzh.ch</span></span>,<span class="em-link"><span class="em-addr">maartenpostuma{at}outlook.com</span></span>,<span class="em-link"><span class="em-addr">arpat.ozgul{at}ieu.uzh.ch</span></span>, <span class="em-link"><span class="em-addr">frederic.guillaume{at}ieu.uzh.ch</span></span></p></li><li class="fn-study-group-members" id="fn-2"><p id="p-2"><strong>statement of authorship</strong>: MS and FG conceived the study, and all authors further developed the approach. MS derived the mathematical equations, performed the simulations, and analyzed output data. M.S. and F.G. wrote the paper, with substantial help from all co-authors.</p></li><li class="fn-other" id="fn-3"><p id="p-3"><strong>data accessibility statement</strong>: The authors hereby confirm that the simulation data supporting the results will be archived in an appropriate public repository and the data DOI will be included at the end of the article.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aitken</span>,  <span class="cit-name-given-names">S. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeaman</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holliday</span>,  <span class="cit-name-given-names">J. a.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Curtis-McLane</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Adaptation, migration or extirpation: Climate change outcomes for tree populations</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>, <span class="cit-vol">1</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">95</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D1%26rft.spage%253D95%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barfield</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">R. D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gomulkiewicz</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Evolution in stage-structured populations</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">177</span>(<span class="cit-issue">4</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beckman</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bullock</span>,  <span class="cit-name-given-names">J. M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Salguero-Gómez</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">High dispersal ability is related to fast life history strategies</span>. <abbr class="cit-jnl-abbrev">Journal of Ecology</abbr>, <span class="cit-vol">106</span>(<span class="cit-month">March</span>):<span class="cit-fpage">1349</span>–<span class="cit-lpage">1362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEcology%26rft.volume%253D106%26rft.spage%253D1349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.4" data-doi="10.1111/j.1461-0248.2009.01350.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bell</span>,  <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Evolutionary rescue can prevent extinction following environmental change</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">942</span>–<span class="cit-lpage">948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DBell%26rft.auinit1%253DG.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253D942%26rft.epage%253D948%26rft.atitle%253DEvolutionary%2Brescue%2Bcan%2Bprevent%2Bextinction%2Bfollowing%2Benvironmental%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2009.01350.x%26rft_id%253Dinfo%253Apmid%252F19659574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2009.01350.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19659574&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268969100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.5" data-doi="10.1046/j.1365-2540.2001.00829.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boulding</span>,  <span class="cit-name-given-names">E. G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hay</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Genetic and demographic parameters determining population persistence after a discrete change in the environment</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr>, <span class="cit-vol">86</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">313</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DBoulding%26rft.auinit1%253DE.%2BG.%26rft.volume%253D86%26rft.issue%253DPt%2B3%26rft.spage%253D313%26rft.epage%253D324%26rft.atitle%253DGenetic%2Band%2Bdemographic%2Bparameters%2Bdetermining%2Bpopulation%2Bpersistence%2Bafter%2Ba%2Bdiscrete%2Bchange%2Bin%2Bthe%2Benvironment.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.2001.00829.x%26rft_id%253Dinfo%253Apmid%252F11488968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.2001.00829.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11488968&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170425800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boyce</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haridas</span>,  <span class="cit-name-given-names">C. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">C. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boggs</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruna</span>,  <span class="cit-name-given-names">E. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coulson</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doak</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaillard</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horvitz</span>,  <span class="cit-name-given-names">C. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalisz</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kendall</span>,  <span class="cit-name-given-names">B. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knight</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mastrandrea</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Menges</span>,  <span class="cit-name-given-names">E. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">W. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfister</span>,  <span class="cit-name-given-names">C. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tuljapurkar</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Demography in an increasingly variable world</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology and Evolution</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">141</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D21%26rft.spage%253D141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.7" data-doi="10.1111/ele.12275"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Buoro</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">S. M.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Life-history syndromes: Integrating dispersal through space and time</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">17</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">756</span>–<span class="cit-lpage">767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D17%26rft.spage%253D756%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12275%26rft_id%253Dinfo%253Apmid%252F24690406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12275&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24690406&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bürger</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">1999</span>). <span class="cit-article-title">Evolution of genetic variability and the advantage of sex and recombination in changing environments</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">153</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">1055</span>–<span class="cit-lpage">1069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBurger%26rft.auinit1%253DR.%26rft.volume%253D153%26rft.issue%253D2%26rft.spage%253D1055%26rft.epage%253D1069%26rft.atitle%253DEvolution%2Bof%2BGenetic%2BVariability%2Band%2Bthe%2BAdvantage%2Bof%2BSex%2Band%2BRecombination%2Bin%2BChanging%2BEnvironments%26rft_id%253Dinfo%253Apmid%252F10511578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1My8yLzEwNTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wNC84MzQyMzQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.9" data-doi="10.2307/2410301"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bürger</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">1995</span>). <span class="cit-article-title">Evolution and extinction in a changing environment: A quantitative-genetic analysis</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">49</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">151</span>–<span class="cit-lpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D49%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410301%26rft_id%253Dinfo%253Apmid%252F28593664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28593664&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RH53700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">S. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cunningham</span>,  <span class="cit-name-given-names">C. J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Westley</span>,  <span class="cit-name-given-names">P. A.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Evolutionary rescue in a changing world</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology and Evolution</abbr>, <span class="cit-vol">29</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">521</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D29%26rft.spage%253D521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-834234v2.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caswell</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-source">Matrix Population Models. Construction, Analysis, and Interpretation</span>. <span class="cit-publ-name">Sinauer Associates, Inc</span>., <span class="cit-publ-loc">Sunderland, MA</span>., <span class="cit-edition">2nd</span> edition.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.12" data-doi="10.1111/j.1461-0248.2004.00595.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caswell</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takada</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hunter</span>,  <span class="cit-name-given-names">C. M.</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Sensitivity analysis of equilibrium in density-dependent matrix population models</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">380</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D7%26rft.spage%253D380%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2004.00595.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2004.00595.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.13" data-doi="10.1111/gcb.13672"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cayuela</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joly</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">B. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pichenot</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonnaire</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Priol</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peyronel</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laville</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Besnard</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Life history tactics shape amphibians’ demographic responses to the North Atlantic Oscillation</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr>, <span class="cit-vol">23</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">4620</span>–<span class="cit-lpage">4638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D23%26rft.spage%253D4620%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.13672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.13672&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-834234v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-source">Evolution in age-structured populations</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>, <span class="cit-edition">2nd</span> edition.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.15" data-doi="10.1111/j.1558-5646.2012.01809.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chevin</span>,  <span class="cit-name-given-names">L.-M.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Genetic constraints on adaptation to a changing environment</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">67</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">708</span>–<span class="cit-lpage">721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D708%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01809.x%26rft_id%253Dinfo%253Apmid%252F23461322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01809.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23461322&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.16" data-doi="10.1086/694123"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chevin</span>,  <span class="cit-name-given-names">L.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cotto</span>,  <span class="cit-name-given-names">O.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ashander</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Stochastic evolutionary demography under a fluctuating optimum phenotype</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">190</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">000</span>–<span class="cit-lpage">000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D190%26rft.spage%253D000%26rft_id%253Dinfo%253Adoi%252F10.1086%252F694123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/694123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.17" data-doi="10.1371/journal.pbio.1000357"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chevin</span>,  <span class="cit-name-given-names">L.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mace</span>,  <span class="cit-name-given-names">G. M.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Adaptation, plasticity, and extinction in a changing environment: Towards a predictive theory</span>. <abbr class="cit-jnl-abbrev">PLoS Biology</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">e1000357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DChevin%26rft.auinit1%253DL.%2BM.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253De1000357%26rft.epage%253De1000357%26rft.atitle%253DAdaptation%252C%2Bplasticity%252C%2Band%2Bextinction%2Bin%2Ba%2Bchanging%2Benvironment%253A%2Btowards%2Ba%2Bpredictive%2Btheory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000357%26rft_id%253Dinfo%253Apmid%252F20463950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20463950&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotto</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandell</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chevin</span>,  <span class="cit-name-given-names">L.-M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ronce</span>,  <span class="cit-name-given-names">O.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Maladaptive shifts in life history in a changing environment</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">194</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">000</span>–<span class="cit-lpage">000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D194%26rft.spage%253D000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotto</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmid</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Guillaume</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2020</span>). <span class="cit-article-title">Nemo-age: Spatially explicit simulations of eco-evolutionary dynamics in stage-structured populations under changing environments</span>. <abbr class="cit-jnl-abbrev">submitted</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cotto</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wessely</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Georges</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klonner</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmid</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dullinger</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thuiller</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Guillaume</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">A dynamic eco-evolutionary model predicts slow response of alpine plants to climate warming</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr>, <span class="cit-vol">8</span>(<span class="cit-month">May</span>):<span class="cit-fpage">15399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D8%26rft.spage%253D15399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.21" data-doi="10.1111/j.1365-2745.2009.01585.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dalgleish</span>,  <span class="cit-name-given-names">H. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koons</span>,  <span class="cit-name-given-names">D. N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Adler</span>,  <span class="cit-name-given-names">P. B.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Can life-history traits predict the response of forb populations to changes in climate variability?</span> <abbr class="cit-jnl-abbrev">Journal of Ecology</abbr>, <span class="cit-vol">98</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">209</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEcology%26rft.volume%253D98%26rft.spage%253D209%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2745.2009.01585.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2745.2009.01585.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000272657400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.22" data-doi="10.1038/nature09439"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Doak</span>,  <span class="cit-name-given-names">D. F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">W. F.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Demographic compensation and tipping points in climate-induced range shifts</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">467</span>(<span class="cit-issue">7318</span>):<span class="cit-fpage">959</span>–<span class="cit-lpage">962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDoak%26rft.auinit1%253DD.%2BF.%26rft.volume%253D467%26rft.issue%253D7318%26rft.spage%253D959%26rft.epage%253D962%26rft.atitle%253DDemographic%2Bcompensation%2Band%2Btipping%2Bpoints%2Bin%2Bclimate-induced%2Brange%2Bshifts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09439%26rft_id%253Dinfo%253Apmid%252F20962844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09439&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20962844&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283254700037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dullinger</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gattringer</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thuiller</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moser</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zimmermann</span>,  <span class="cit-name-given-names">N. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guisan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willner</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plutzar</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leitner</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mang</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caccianiga</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dirnböck</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ertl</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fischer</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lenoir</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Svenning</span>,  <span class="cit-name-given-names">J.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Psomas</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmatz</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silc</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vittoz</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hülber</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Extinction debt of high-mountain plants under twenty-first-century climate change</span>. <abbr class="cit-jnl-abbrev">Nature Climate Change</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">619</span>–<span class="cit-lpage">622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BClimate%2BChange%26rft.volume%253D2%26rft.spage%253D619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.24" data-doi="10.1111/j.1461-0248.2011.01734.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duputié</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Massol</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chuine</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ronce</span>,  <span class="cit-name-given-names">O.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">How do genetic correlations affect species range shifts in a changing environment?</span> <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">251</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DDuputie%26rft.auinit1%253DA.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D251%26rft.epage%253D259%26rft.atitle%253DHow%2Bdo%2Bgenetic%2Bcorrelations%2Baffect%2Bspecies%2Brange%2Bshifts%2Bin%2Ba%2Bchanging%2Benvironment%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2011.01734.x%26rft_id%253Dinfo%253Apmid%252F22248112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2011.01734.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22248112&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duputié</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rutschmann</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ronce</span>,  <span class="cit-name-given-names">O.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chuine</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Phenological plasticity will not help all species adapt to climate change</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">3062</span>–<span class="cit-lpage">3073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D21%26rft.spage%253D3062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.26" data-doi="10.1086/285610"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellner</span>,  <span class="cit-name-given-names">S. P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hairston</span>,  <span class="cit-name-given-names">N. G.</span></span> (<span class="cit-pub-date">1994</span>). <span class="cit-article-title">Role of overlapping generations in maintaining genetic variation in a fluctuating environment</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">143</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">403</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D143%26rft.spage%253D403%26rft_id%253Dinfo%253Adoi%252F10.1086%252F285610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/285610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NC03300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.27" data-doi="10.1534/genetics.109.105841"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Engen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Saether</span>,  <span class="cit-name-given-names">B.-E.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Reproductive value and fluctuating selection in an age-structured population</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">183</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">629</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.105841%26rft_id%253Dinfo%253Apmid%252F19620391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzIvNjI5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDQvODM0MjM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Engen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Saether</span>,  <span class="cit-name-given-names">B.-E.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Evolution of a plastic quantitative trait in an age-structured population in a fluctuating environment</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">65</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">2893</span>–<span class="cit-lpage">2906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DEngen%26rft.auinit1%253DS.%26rft.volume%253D65%26rft.issue%253D10%26rft.spage%253D2893%26rft.epage%253D2906%26rft.atitle%253DEvolution%2Bof%2Ba%2Bplastic%2Bquantitative%2Btrait%2Bin%2Ban%2Bage-structured%2Bpopulation%2Bin%2Ba%2Bfluctuating%2Benvironment.%26rft_id%253Dinfo%253Apmid%252F21967430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21967430&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.29" data-doi="10.1086/670257"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Engen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sæther</span>,  <span class="cit-name-given-names">B.-E.</span></span> (<span class="cit-pub-date">2013a</span>). <span class="cit-article-title">A quantitative genetic model of r- and K-selection in a fluctuating population</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">181</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">725</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D181%26rft.spage%253D725%26rft_id%253Dinfo%253Adoi%252F10.1086%252F670257%26rft_id%253Dinfo%253Apmid%252F23669536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/670257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23669536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.30" data-doi="10.1016/j.mbs.2005.02.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Engen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sæther</span>,  <span class="cit-name-given-names">B.-E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Weimerskirch</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Extinction in relation to demographic and environmental stochasticity in age-structured models</span>. <abbr class="cit-jnl-abbrev">Mathematical Biosciences</abbr>, <span class="cit-vol">195</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">210</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bbiosciences%26rft.stitle%253DMath%2BBiosci%26rft.aulast%253DEngen%26rft.auinit1%253DS.%26rft.volume%253D195%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D227%26rft.atitle%253DExtinction%2Bin%2Brelation%2Bto%2Bdemographic%2Band%2Benvironmental%2Bstochasticity%2Bin%2Bage-structured%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mbs.2005.02.003%26rft_id%253Dinfo%253Apmid%252F15907948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mbs.2005.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15907948&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230041400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Engen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sæther</span>,  <span class="cit-name-given-names">B.-E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Armitage</span>,  <span class="cit-name-given-names">K. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blumstein</span>,  <span class="cit-name-given-names">D. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clutton-Brock</span>,  <span class="cit-name-given-names">T. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobson</span>,  <span class="cit-name-given-names">F. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Festa-Bianchet</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oli</span>,  <span class="cit-name-given-names">M. K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ozgul</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2013b</span>). <span class="cit-article-title">Estimating the effect of temporally autocorrelated environments on the demography of density-independent age-structured populations</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>, <span class="cit-vol">4</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">573</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D4%26rft.spage%253D573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.32" data-doi="10.1086/498282"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">M. E. K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Dennehy</span>,  <span class="cit-name-given-names">J. J.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Germ banking : Bet-hedging and variable release from egg and seed dormancy</span>. <abbr class="cit-jnl-abbrev">The Quarterly Review of Biology</abbr>, <span class="cit-vol">80</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">431</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BQuarterly%2BReview%2Bof%2BBiology%26rft.stitle%253DThe%2BQuarterly%2BReview%2Bof%2BBiology%26rft.aulast%253DEvans%26rft.auinit1%253DM.%2BE.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D431%26rft.epage%253D451%26rft.atitle%253DGerm%2Bbanking%253A%2Bbet-hedging%2Band%2Bvariable%2Brelease%2Bfrom%2Begg%2Band%2Bseed%2Bdormancy.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F498282%26rft_id%253Dinfo%253Apmid%252F16519139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/498282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16519139&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-834234v2.33" data-doi="10.1086/430330"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gaillard</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoccoz</span>,  <span class="cit-name-given-names">N. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lebreton</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonenfant</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Devillard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loison</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pontier</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Allaine</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Generation time: A reliable metric to measure life-history variation among mammalian populations</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">166</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">119</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DGaillard%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D166%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D123%26rft.atitle%253DGeneration%2Btime%253A%2Ba%2Breliable%2Bmetric%2Bto%2Bmeasure%2Blife-history%2Bvariation%2Bamong%2Bmammalian%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F430330%26rft_id%253Dinfo%253Apmid%252F15937795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/430330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15937795&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229794700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.34" data-doi="10.2307/3566088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gaillard</span>,  <span class="cit-name-given-names">J.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pontier</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Allainé</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lebreton</span>,  <span class="cit-name-given-names">J. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trouvilliez</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Clobert</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1989</span>). <span class="cit-article-title">An analysis of demographic tactics in birds and mammals</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr>, <span class="cit-vol">56</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">59</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D56%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3566088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3566088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989AQ56300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.35" data-doi="10.2307/2410305"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gomulkiewicz</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">R. D.</span></span> (<span class="cit-pub-date">1995</span>). <span class="cit-article-title">When does evolution by natural selection prevent extinction?</span> <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">49</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">201</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D49%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2410305%26rft_id%253Dinfo%253Apmid%252F28593677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2410305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28593677&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RH53700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gomulkiewicz</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Houle</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Demographic and genetic constraints on evolution</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">174</span>(<span class="cit-issue">6</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Benton</span>,  <span class="cit-name-given-names">T. G.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Elasticity analysis for density-dependent populations in stochastic environments</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-vol">81</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">680</span>–<span class="cit-lpage">693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D81%26rft.spage%253D680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Benton</span>,  <span class="cit-name-given-names">T. G.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Density-dependent populations require density-dependent elasticity analysis: An illustration using the LPA model of Tribolium</span>. <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>, <span class="cit-vol">72</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">94</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D72%26rft.spage%253D94%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.39" data-doi="10.1093/bioinformatics/btl415"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Rougemont</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Nemo: An evolutionary and population genetics programming framework</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">22</span>(<span class="cit-issue">20</span>):<span class="cit-fpage">2556</span>–<span class="cit-lpage">2557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl415%26rft_id%253Dinfo%253Apmid%252F16882649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16882649&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241271300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.40" data-doi="10.1038/336239a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hairston</span>,  <span class="cit-name-given-names">N. G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">De Stasio</span>   <span class="cit-name-suffix">Jr.</span>,  <span class="cit-name-given-names">B. T.</span></span> (<span class="cit-pub-date">1988</span>). <span class="cit-article-title">Rate of evolution slowed by a dormant propagule pool</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">336</span>:<span class="cit-fpage">239</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D336%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1038%252F336239a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/336239a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988Q954200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.41" data-doi="10.1038/s41467-018-02922-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hallett</span>,  <span class="cit-name-given-names">L. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Farrer</span>,  <span class="cit-name-given-names">E. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suding</span>,  <span class="cit-name-given-names">K. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mooney</span>,  <span class="cit-name-given-names">H. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hobbs</span>,  <span class="cit-name-given-names">R. J.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Tradeoffs in demographic mechanisms underlie differences in species abundance and stability</span>. <abbr class="cit-jnl-abbrev">Nature Communications</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-02922-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-02922-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.42" data-doi="10.1111/j.1420-9101.2008.01573.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hansen</span>,  <span class="cit-name-given-names">T. F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Houle</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Measuring and comparing evolvability and constraint in multivariate characters</span>. <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr>, <span class="cit-vol">21</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1201</span>–<span class="cit-lpage">1219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DHansen%26rft.auinit1%253DT.%2BF.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D1201%26rft.epage%253D1219%26rft.atitle%253DMeasuring%2Band%2Bcomparing%2Bevolvability%2Band%2Bconstraint%2Bin%2Bmultivariate%2Bcharacters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2008.01573.x%26rft_id%253Dinfo%253Apmid%252F18662244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2008.01573.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18662244&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258379600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hansen</span>,  <span class="cit-name-given-names">T. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solvin</span>,  <span class="cit-name-given-names">T. M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pavlicev</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Predicting evolutionary potential: A numerical test of evolvability measures</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">73</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">689</span>–<span class="cit-lpage">703</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D73%26rft.spage%253D689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Houle</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">1992</span>). <span class="cit-article-title">Comparing evolvability and variability of quantitative traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">130</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">195</span>–<span class="cit-lpage">204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHoule%26rft.auinit1%253DD.%26rft.volume%253D130%26rft.issue%253D1%26rft.spage%253D195%26rft.epage%253D204%26rft.atitle%253DComparing%2Bevolvability%2Band%2Bvariability%2Bof%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Apmid%252F1732160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMwLzEvMTk1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDQvODM0MjM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.45" data-doi="10.1086/592867"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Koons</span>,  <span class="cit-name-given-names">D. N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Metcalf</span>,  <span class="cit-name-given-names">C. J. E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tuljapurkar</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Evolution of delayed reproduction in uncertain environments: A life-history perspective</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">172</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">797</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DKoons%26rft.auinit1%253DD.%2BN.%26rft.volume%253D172%26rft.issue%253D6%26rft.spage%253D797%26rft.epage%253D805%26rft.atitle%253DEvolution%2Bof%2Bdelayed%2Breproduction%2Bin%2Buncertain%2Benvironments%253A%2Ba%2Blife-history%2Bperspective.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F592867%26rft_id%253Dinfo%253Apmid%252F18959491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/592867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18959491&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261235500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kopp</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Matuszewski</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Rapid evolution of quantitative traits: Theoretical perspectives</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">169</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D7%26rft.spage%253D169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuparinen</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savolainen</span>,  <span class="cit-name-given-names">O.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Schurr</span>,  <span class="cit-name-given-names">F. M.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Increased mortality can promote evolutionary adaptation of forest trees to climate change</span>. <abbr class="cit-jnl-abbrev">Forest Ecology and Management</abbr>, <span class="cit-vol">259</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1003</span>–<span class="cit-lpage">1008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2BEcology%2Band%2BManagement%26rft.volume%253D259%26rft.spage%253D1003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.48" data-doi="10.2307/1936778"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">1982</span>). <span class="cit-article-title">A quantitative genetic theory of life history evolution</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-vol">63</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">607</span>–<span class="cit-lpage">615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D63%26rft.spage%253D607%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1936778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1936778&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1982NV13400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.49" data-doi="10.1086/285580"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">1993</span>). <span class="cit-article-title">Risks of population extinction from demographic and environmental stochasticity and random catastrophes</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">142</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">911</span>–<span class="cit-lpage">927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D142%26rft.spage%253D911%26rft_id%253Dinfo%253Adoi%252F10.1086%252F285580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/285580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993MQ51700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.50" data-doi="10.1073/pnas.85.19.7418"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Orzack</span>,  <span class="cit-name-given-names">S. H.</span></span> (<span class="cit-pub-date">1988</span>). <span class="cit-article-title">Extinction dynamics of age-structured populations in a fluctuating environment</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, <span class="cit-vol">85</span>(<span class="cit-issue">19</span>):<span class="cit-fpage">7418</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLande%26rft.auinit1%253DR.%26rft.volume%253D85%26rft.issue%253D19%26rft.spage%253D7418%26rft.epage%253D7421%26rft.atitle%253DExtinction%2Bdynamics%2Bof%2Bage-structured%2Bpopulations%2Bin%2Ba%2Bfluctuating%2Benvironment.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.19.7418%26rft_id%253Dinfo%253Apmid%252F3174643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODUvMTkvNzQxOCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzA0LzgzNDIzNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lanfear</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kokko</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Eyre-Walker</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Population size and the rate of evolution</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology and Evolution</abbr>, <span class="cit-vol">29</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">33</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D29%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabriel</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wood</span>,  <span class="cit-name-given-names">A. M.</span></span> (<span class="cit-pub-date">1991</span>). <span class="cit-article-title">Adaptive and demographic responses of plankton populations to environmental change</span>. <abbr class="cit-jnl-abbrev">Limnology and Oceanography</abbr>, <span class="cit-vol">36</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1301</span>–<span class="cit-lpage">1312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLimnology%2Band%2BOceanography%26rft.volume%253D36%26rft.spage%253D1301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-834234v2.53"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Kareiva</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Kingsolver</span>,  <span class="cit-name-given-names">J. G.</span></span>, and </li><li><span class="cit-ed"><span class="cit-name-surname">Huey</span>,  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">1993</span>). <span class="cit-chapter-title">Evolution and extinction in response to environmental change</span>. In <span class="cit-ed"><span class="cit-name-surname">Kareiva</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kingsolver</span>,  <span class="cit-name-given-names">J. G.</span></span>, and <span class="cit-ed"><span class="cit-name-surname">Huey</span>,  <span class="cit-name-given-names">R. B.</span></span>, editors, <span class="cit-source">Biotic Interactions and Global Change</span>, pages <span class="cit-fpage">234</span>–<span class="cit-lpage">250</span>. <span class="cit-publ-name">Sinauer</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marshall</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burgess</span>,  <span class="cit-name-given-names">S. C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Connallon</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Global change, life-history complexity and the potential for evolutionary rescue</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">1189</span>–<span class="cit-lpage">1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D9%26rft.spage%253D1189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McDonald</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franco</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Townley</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ezard</span>,  <span class="cit-name-given-names">T. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jelbert</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hodgson</span>,  <span class="cit-name-given-names">D. J.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Divergent demographic strategies of plants in variable environments</span>. <abbr class="cit-jnl-abbrev">Nature Ecology and Evolution</abbr>, <span class="cit-vol">1</span>(<span class="cit-issue">2</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.56" data-doi="10.1890/07-0774.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">W. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfister</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tuljapurkar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haridas</span>,  <span class="cit-name-given-names">C. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boggs</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boyce</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruna</span>,  <span class="cit-name-given-names">E. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Church</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coulson</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doak</span>,  <span class="cit-name-given-names">D. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forsyth</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaillard</span>,  <span class="cit-name-given-names">J.-m.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horvitz</span>,  <span class="cit-name-given-names">C. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalisz</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kendall</span>,  <span class="cit-name-given-names">B. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knight</span>,  <span class="cit-name-given-names">T. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">C. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Menges</span>,  <span class="cit-name-given-names">E. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chirrakal</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boggs</span>,  <span class="cit-name-given-names">C. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boyce</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruna</span>,  <span class="cit-name-given-names">E. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Church</span>,  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coulson</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doak</span>,  <span class="cit-name-given-names">D. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forsyth</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaillard</span>,  <span class="cit-name-given-names">J.-m.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horvitz</span>,  <span class="cit-name-given-names">C. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalisz</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kendall</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knight</span>,  <span class="cit-name-given-names">T. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">C. T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Menges</span>,  <span class="cit-name-given-names">E. S.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Longevity can buffer plant and animal populations against changing climatic variability</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-vol">89</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">19</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DMorris%26rft.auinit1%253DW.%2BF.%26rft.volume%253D89%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D25%26rft.atitle%253DLongevity%2Bcan%2Bbuffer%2Bplant%2Band%2Banimal%2Bpopulations%2Bagainst%2Bchanging%2Bclimatic%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F07-0774.1%26rft_id%253Dinfo%253Apmid%252F18376542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/07-0774.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18376542&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253717200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Orive</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barfield</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernandez</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">R. D.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Effects of clonal reproduction on evolutionary lag and evolutionary rescue</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">190</span>(<span class="cit-issue">4</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.58" data-doi="10.1086/284959"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Orzack</span>,  <span class="cit-name-given-names">S. H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tuljapurkar</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1989</span>). <span class="cit-article-title">Population Dynamics in Variable Environments. VII. The Demography and Evolution of Iteroparity</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">133</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">901</span>–<span class="cit-lpage">923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D133%26rft.spage%253D901%26rft_id%253Dinfo%253Adoi%252F10.1086%252F284959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/284959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989AG01500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.59" data-doi="10.1086/694123"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Osmond</span>,  <span class="cit-name-given-names">M. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Otto</span>,  <span class="cit-name-given-names">S. P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Klausmeier</span>,  <span class="cit-name-given-names">C. A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">When predators help prey adapt and persist in a changing environment</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">190</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">000</span>–<span class="cit-lpage">000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D190%26rft.spage%253D000%26rft_id%253Dinfo%253Adoi%252F10.1086%252F694123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/694123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Paniw</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozgul</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Salguero-Gómez</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Interactive life-history traits predict sensitivity of plants and animals to temporal autocorrelation</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, pages <span class="cit-fpage">275</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.61" data-doi="10.1146/annurev.ecolsys.37.091305.110100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parmesan</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Ecological and evolutionary responses to recent climate change</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">37</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">637</span>–<span class="cit-lpage">669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D37%26rft.spage%253D637%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.37.091305.110100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.37.091305.110100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000243038500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.62" data-doi="10.1146/annurev.ecolsys.37.091305.110215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petit</span>,  <span class="cit-name-given-names">R. J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hampe</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Some evolutionary consequences of being a tree</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">37</span>(<span class="cit-month">May</span>):<span class="cit-fpage">187</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D37%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.37.091305.110215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.37.091305.110215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000243038500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.63" data-doi="10.1086/605958"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Polechová</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Marion</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Species’ range: Adaptation in space and time</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">174</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">E186</span>–<span class="cit-lpage">-E204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DPolechova%26rft.auinit1%253DJ.%26rft.volume%253D174%26rft.issue%253D5%26rft.spage%253DE186%26rft.epage%253DE204%26rft.atitle%253DSpecies%2527%2Brange%253A%2Badaptation%2Bin%2Bspace%2Band%2Btime.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F605958%26rft_id%253Dinfo%253Apmid%252F19788353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/605958&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19788353&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271021900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.64"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span> (<span class="cit-pub-date">2008</span>). <abbr class="cit-jnl-abbrev">R: a language and environment for statistical computing</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.65" data-doi="10.1086/673497"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sæther</span>,  <span class="cit-name-given-names">B.-E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coulson</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grøtan</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Engen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Altwegg</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Armitage</span>,  <span class="cit-name-given-names">K. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barbraud</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">P. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blumstein</span>,  <span class="cit-name-given-names">D. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dobson</span>,  <span class="cit-name-given-names">F. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Festa-Bianchet</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaillard</span>,  <span class="cit-name-given-names">J.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jenkins</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicoll</span>,  <span class="cit-name-given-names">M. A. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Norris</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oli</span>,  <span class="cit-name-given-names">M. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ozgul</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Weimerskirch</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">How life history influences population dynamics in fluctuating environments</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">182</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">743</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D182%26rft.spage%253D743%26rft_id%253Dinfo%253Adoi%252F10.1086%252F673497%26rft_id%253Dinfo%253Apmid%252F24231536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/673497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24231536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326907300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salguero-Gómez</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Applications of the fast–slow continuum and reproductive strategy framework of plant life histories</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr>, <span class="cit-vol">213</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1618</span>–<span class="cit-lpage">1624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D213%26rft.spage%253D1618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.67" data-doi="10.1073/pnas.1717717114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salguero-Gómez</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">O. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blomberg</span>,  <span class="cit-name-given-names">S. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodgson</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zuidema</span>,  <span class="cit-name-given-names">P. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kroon</span>,  <span class="cit-name-given-names">H. D.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Fast–slow continuum and reproductive strategies structure plant life-history variation worldwide</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, <span class="cit-vol">114</span>(<span class="cit-issue">45</span>):<span class="cit-fpage">E9753</span>–<span class="cit-lpage">E9753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1717717114%26rft_id%253Dinfo%253Apmid%252F29087354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQ1L0U5NzUzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDQvODM0MjM0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.68" data-doi="10.2307/2411145"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ellner</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1997</span>). <span class="cit-article-title">Quantitative genetic variance maintained by fluctuating selection with overlapping generations : Variance components and covariances</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">51</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">682</span>–<span class="cit-lpage">696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D51%26rft.spage%253D682%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XM06500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.69" data-doi="10.2307/3544261"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stearns</span>,  <span class="cit-name-given-names">S. C.</span></span> (<span class="cit-pub-date">1983</span>). <span class="cit-article-title">The influence of size and phylogeny on patterns of covariation among life-history traits in the mammals</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr>, <span class="cit-vol">41</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">173</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D41%26rft.spage%253D173%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3544261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3544261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983RL35800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-book" id="cit-834234v2.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stocker</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plattner</span>,  <span class="cit-name-given-names">G.-K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tignor</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boschung</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nauels</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bex</span>,  <span class="cit-name-given-names">V.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Midgley</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-chapter-title">IPCC, 2013: Climate Change 2013: The Physical Science Basis</span>. <span class="cit-source">Contribution of Working Group I to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change</span>. <span class="cit-publ-name">Technical report, IPCC, Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, United Kingdom and New York, NY, USA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stubben</span>,  <span class="cit-name-given-names">C. J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Milligan</span>,  <span class="cit-name-given-names">B. G.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Estimating and Analyzing Demographic Models Using the popbio Package in R</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">22</span>:<span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D22%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.72" data-doi="10.1016/j.tpb.2014.11.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Svardal</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rueffler</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hermisson</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">A general condition for adaptive genetic polymorphism in temporally and spatially heterogeneous environments</span>. <abbr class="cit-jnl-abbrev">Theoretical Population Biology</abbr>, <span class="cit-vol">99</span>:<span class="cit-fpage">76</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2BPopulation%2BBiology%26rft.volume%253D99%26rft.spage%253D76%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2014.11.002%26rft_id%253Dinfo%253Apmid%252F25446960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2014.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25446960&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takada</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nakajima</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">Theorems on the invasion process in stage-structured populations with density-dependent dynamics</span>. <abbr class="cit-jnl-abbrev">Journal of Mathematical Biology</abbr>, <span class="cit-vol">36</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">497</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMathematical%2BBiology%26rft.volume%253D36%26rft.spage%253D497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.74" data-doi="10.1086/283471"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Templeton</span>,  <span class="cit-name-given-names">A. R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Levin</span>,  <span class="cit-name-given-names">D. A.</span></span> (<span class="cit-pub-date">1979</span>). <span class="cit-article-title">Evolutionary consequences of seed pools</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">114</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">232</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D114%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1086%252F283471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/283471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979HK47000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-book" id="cit-834234v2.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tuljapurkar</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1990</span>). <span class="cit-source">Population Dynamics in Variable Environments</span>. <span class="cit-publ-name">Springer Verlag</span>, <span class="cit-publ-loc">Berlin Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.76" data-doi="10.1098/rstb.2009.0021"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tuljapurkar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaillard</span>,  <span class="cit-name-given-names">J. M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Coulson</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">From stochastic environments to life histories and back</span>. <abbr class="cit-jnl-abbrev">Philosophical Transactions of the Royal Society B: Biological Sciences</abbr>, <span class="cit-vol">364</span>(<span class="cit-issue">1523</span>):<span class="cit-fpage">1499</span>–<span class="cit-lpage">1509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DTuljapurkar%26rft.auinit1%253DS.%26rft.volume%253D364%26rft.issue%253D1523%26rft.spage%253D1499%26rft.epage%253D1509%26rft.atitle%253DFrom%2Bstochastic%2Benvironments%2Bto%2Blife%2Bhistories%2Band%2Bback%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0021%26rft_id%253Dinfo%253Apmid%252F19414465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2009.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19414465&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.77" data-doi="10.1086/378648"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tuljapurkar</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horvitz</span>,  <span class="cit-name-given-names">C. C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pascarella</span>,  <span class="cit-name-given-names">J. B.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">The many growth rates and elasticities of populations in random environments</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">162</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">489</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D162%26rft.spage%253D489%26rft_id%253Dinfo%253Adoi%252F10.1086%252F378648%26rft_id%253Dinfo%253Apmid%252F14582010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/378648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14582010&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186172300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.78" data-doi="10.1098/rstb.2016.0135"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ummenhofer</span>,  <span class="cit-name-given-names">C. C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Meehl</span>,  <span class="cit-name-given-names">G. A.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Extreme weather and climate events with ecological relevance: a review</span>. <abbr class="cit-jnl-abbrev">Philosophical Transactions of the Royal Society B: Biological Sciences</abbr>, <span class="cit-vol">372</span>(<span class="cit-issue">1723</span>):<span class="cit-fpage">20160135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0135%26rft_id%253Dinfo%253Apmid%252F28483866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2016.0135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28483866&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.79" data-doi="10.1890/09-0410.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van De Pol</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vindenes</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sæther</span>,  <span class="cit-name-given-names">B.-E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Engen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ens</span>,  <span class="cit-name-given-names">B. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oosterbeek</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Tinbergen</span>,  <span class="cit-name-given-names">J. M.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Effects of climate change and variability on population dynamics in a long-lived shorebird</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-vol">91</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">1192</span>–<span class="cit-lpage">1204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253Dvan%2Bde%2BPol%26rft.auinit1%253DM.%26rft.volume%253D91%26rft.issue%253D4%26rft.spage%253D1192%26rft.epage%253D1204%26rft.atitle%253DEffects%2Bof%2Bclimate%2Bchange%2Band%2Bvariability%2Bon%2Bpopulation%2Bdynamics%2Bin%2Ba%2Blong-lived%2Bshorebird.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F09-0410.1%26rft_id%253Dinfo%253Apmid%252F20462133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/09-0410.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20462133&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277525300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.80" data-doi="10.1111/j.1461-0248.2007.01080.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Mantgem</span>,  <span class="cit-name-given-names">P. J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Stephenson</span>,  <span class="cit-name-given-names">N. L.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Apparent climatically induced increase of tree mortality rates in a temperate forest</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">909</span>–<span class="cit-lpage">916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253Dvan%2BMantgem%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D909%26rft.epage%253D916%26rft.atitle%253DApparent%2Bclimatically%2Binduced%2Bincrease%2Bof%2Btree%2Bmortality%2Brates%2Bin%2Ba%2Btemperate%2Bforest.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2007.01080.x%26rft_id%253Dinfo%253Apmid%252F17845291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2007.01080.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17845291&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249275900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.81" data-doi="10.1098/rstb.2012.0090"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vander Wal</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garant</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Festa-Bianchet</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Pelletier</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Evolutionary rescue in vertebrates: Evidence, applications and uncertainty</span>. <abbr class="cit-jnl-abbrev">Philosophical Transactions of the Royal Society B: Biological Sciences</abbr>, <span class="cit-vol">368</span>(<span class="cit-issue">1610</span>):<span class="cit-fpage">20120090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.volume%253D368%26rft.spage%253D20120090%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2012.0090%26rft_id%253Dinfo%253Apmid%252F23209171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2012.0090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23209171&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.82" data-doi="10.1111/brv.12216"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vázquez</span>,  <span class="cit-name-given-names">D. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gianoli</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">W. F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Bozinovic</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Ecological and evolutionary impacts of changing climatic variability</span>. <abbr class="cit-jnl-abbrev">Biological Reviews</abbr>, <span class="cit-vol">92</span>:<span class="cit-fpage">22</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%26rft.volume%253D92%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fbrv.12216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/brv.12216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.83" data-doi="10.2307/2408649"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Via</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">1985</span>). <span class="cit-article-title">Genotype-environment interaction and the evolution of phenotypic plasticity</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">39</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">505</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D39%26rft.spage%253D505%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AJY2600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.84" data-doi="10.1111/j.1420-9101.2008.01631.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Willi</span>,  <span class="cit-name-given-names">Y.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">A. A.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Demographic factors and genetic variation influence population persistence under environmental change</span>. <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr>, <span class="cit-vol">22</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">124</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DWilli%26rft.auinit1%253DY.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D124%26rft.epage%253D133%26rft.atitle%253DDemographic%2Bfactors%2Band%2Bgenetic%2Bvariation%2Binfluence%2Bpopulation%2Bpersistence%2Bunder%2Benvironmental%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2008.01631.x%26rft_id%253Dinfo%253Apmid%252F19120814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2008.01631.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19120814&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261837300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yamamichi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth">  <span class="cit-name-suffix">Jr</span>,  <span class="cit-name-given-names">N. G.</span> <span class="cit-name-surname">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rees</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ellner</span>,  <span class="cit-name-given-names">S. P.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Rapid evolution with generation overlap : The double-edged effect of dormancy</span>. <abbr class="cit-jnl-abbrev">Theoretical Ecology</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">179</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2BEcology%26rft.volume%253D12%26rft.spage%253D179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-834234v2.86"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeineddine</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Jansen</span>,  <span class="cit-name-given-names">V. A. A.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">To age, to die: Parity, evolutionary tracking and Cole’s paradox</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">63</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">1498</span>–<span class="cit-lpage">1507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DZeineddine%26rft.auinit1%253DM.%26rft.volume%253D63%26rft.issue%253D6%26rft.spage%253D1498%26rft.epage%253D1507%26rft.atitle%253DTo%2Bage%252C%2Bto%2Bdie%253A%2Bparity%252C%2Bevolutionary%2Btracking%2Band%2BCole%2527s%2Bparadox.%26rft_id%253Dinfo%253Apmid%252F19492994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19492994&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F04%2F834234.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/834234v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F834234v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/830406v3" title="The immunocytokine L19-TNF eradicates Sarcomas in combination with Chemotherapy agents or with immune check-point inhibitors" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/852442v2" title="A novel transplantable model of lung cancer associated tissue loss and disrupted muscle regeneration" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;February 04, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/834234v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/834234v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/834234v2" class="active" data-icon-position="" data-hide-link-title="0">A tradeoff between robustness to environmental fluctuations and speed of evolution</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1134918" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-d4_s4pyUFruFvqDUY-OHTOHOq9b9DwoQil44LadGH_I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1134918" id="node1134918--2" data-pisa="biorxiv;834234v2" data-pisa-master="biorxiv;834234" data-seqnum="1134918" data-apath="/biorxiv/early/2020/02/04/834234.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A tradeoff between robustness to environmental fluctuations and speed of evolution</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Max</span> <span class="nlm-surname">Schmid</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Paniw</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Maarten</span> <span class="nlm-surname">Postuma</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Arpat</span> <span class="nlm-surname">Ozgul</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Frédéric</span> <span class="nlm-surname">Guillaume</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">834234; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/834234 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/834234v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/834234v2&amp;title=A%20tradeoff%20between%20robustness%20to%20environmental%20fluctuations%20and%20speed%20of%20evolution" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/834234v2&amp;title=A%20tradeoff%20between%20robustness%20to%20environmental%20fluctuations%20and%20speed%20of%20evolution" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/834234v2&amp;text=A%20tradeoff%20between%20robustness%20to%20environmental%20fluctuations%20and%20speed%20of%20evolution" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/834234v2&amp;title=A%20tradeoff%20between%20robustness%20to%20environmental%20fluctuations%20and%20speed%20of%20evolution" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/834234v2&amp;t=A%20tradeoff%20between%20robustness%20to%20environmental%20fluctuations%20and%20speed%20of%20evolution" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/834234v2&amp;title=A%20tradeoff%20between%20robustness%20to%20environmental%20fluctuations%20and%20speed%20of%20evolution" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/834234v2&amp;title=A%20tradeoff%20between%20robustness%20to%20environmental%20fluctuations%20and%20speed%20of%20evolution" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1134918" id="node1134918--3" data-pisa="biorxiv;834234v2" data-pisa-master="biorxiv;834234" data-seqnum="1134918" data-apath="/biorxiv/early/2020/02/04/834234.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A tradeoff between robustness to environmental fluctuations and speed of evolution</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Max</span> <span class="nlm-surname">Schmid</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Maria</span> <span class="nlm-surname">Paniw</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Maarten</span> <span class="nlm-surname">Postuma</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Arpat</span> <span class="nlm-surname">Ozgul</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Frédéric</span> <span class="nlm-surname">Guillaume</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">834234; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/834234 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20tradeoff%20between%20robustness%20to%20environmental%20fluctuations%20and%20speed%20of%20evolution&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Schmid%2C+Max&amp;rft.au=Paniw%2C+Maria&amp;rft.au=Postuma%2C+Maarten&amp;rft.au=Ozgul%2C+Arpat&amp;rft.au=Guillaume%2C+Fr%C3%A9d%C3%A9ric"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1134918/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1134918/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1134918/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1134918/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1134918/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1134918/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1134918/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1134918/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1134918/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1134918/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1134918/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1134918/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/834234v2&amp;count=horizontal&amp;via=&amp;text=A%20tradeoff%20between%20robustness%20to%20environmental%20fluctuations%20and%20speed%20of%20evolution&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/834234v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/834234v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/834234v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
