<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/16/2020.01.14.902643.atom" />
<meta name="HW.pisa" content="biorxiv;2020.01.14.902643v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual Circaeaster agrestis (Circaeasteraceae, Ranunculales)" />
<meta name="DC.Identifier" content="10.1101/2020.01.14.902643" />
<meta name="DC.Date" content="2020-01-16" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="A primary goal of phylogeography is understanding how geographical and ecological factors shape the genetic structure and evolutionary history of species. Environmental factors can act in concert and vary in space, resulting in divergent natural selection leading to genetic divergence during" />
<meta name="DC.Contributor" content="Xu Zhang" />
<meta name="DC.Contributor" content="Yanxia Sun" />
<meta name="DC.Contributor" content="Jacob B. Landis" />
<meta name="DC.Contributor" content="Jianwen Zhang" />
<meta name="DC.Contributor" content="Linsen Yang" />
<meta name="DC.Contributor" content="Nan Lin" />
<meta name="DC.Contributor" content="Huajie Zhang" />
<meta name="DC.Contributor" content="Rui Guo" />
<meta name="DC.Contributor" content="Lijuan Li" />
<meta name="DC.Contributor" content="Yonghong Zhang" />
<meta name="DC.Contributor" content="Tao Deng" />
<meta name="DC.Contributor" content="Hang Sun" />
<meta name="DC.Contributor" content="Hengchang Wang" />
<meta name="article:published_time" content="2020-01-16" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual Circaeaster agrestis (Circaeasteraceae, Ranunculales)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Investigating the interaction between environmental heterogeneity and local adaptation is critical to understand the evolutionary history of a species, providing the premise for studying the response of organisms to rapid climate change. However, for most species how exactly the spatial heterogeneity promotes population divergence and how genomic variations contribute to adaptive evolution remain poorly understood. We examine the contributions of geographical and environmental variables to population divergence of the relictual, alpine herb Circaeaster agrestis as well as genetic basis of local adaptation using RAD-seq and plastome data. We detected significant genetic structure with an extraordinary disequilibrium of genetic diversity among regions, and signals of isolation-by-distance along with isolation-by-resistance. The populations were estimated to begin diverging in the late Miocene, along with a possible ancestral distribution of the Hengduan Mountains and adjacent regions. Both environmental gradient and redundancy analyses revealed significant association between genetic variation and temperature variables. Genome‐environment association analyses identified 16 putatively adaptive loci related to biotic and abiotic stress resistance. Our genome wide data provide new insights into the important role of environmental heterogeneity in shaping genetic structure, and access the footprints of local adaptation in an ancient relictual species. Keywords: local adaptation, Circaeaster agrestis, RAD sequencing, genetic structure, environmental heterogeneity, temperature variables&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;2020.01.14.902643v2" />
<meta name="citation_id" content="2020.01.14.902643v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.14.902643v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.14.902643v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.14.902643v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643.full.pdf" />
<meta name="citation_doi" content="10.1101/2020.01.14.902643" />
<meta name="citation_num_pages" content="37" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2020.01.14.902643" />
<meta name="citation_author" content="Xu Zhang" />
<meta name="citation_author_institution" content="CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5743-3497" />
<meta name="citation_author" content="Yanxia Sun" />
<meta name="citation_author_institution" content="CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences" />
<meta name="citation_author_email" content="sunyanxia@wbgcas.cn" />
<meta name="citation_author_email" content="sunhang@mail.kib.ac.cn" />
<meta name="citation_author_email" content="hcwang@wbgcas.cn" />
<meta name="citation_author" content="Jacob B. Landis" />
<meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California Riverside" />
<meta name="citation_author" content="Jianwen Zhang" />
<meta name="citation_author_institution" content="Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences" />
<meta name="citation_author" content="Linsen Yang" />
<meta name="citation_author_institution" content="Hubei Key Laboratory of Shennongjia Golden Monkey Conservation Biology, Administration of Shennongjia National Park" />
<meta name="citation_author" content="Nan Lin" />
<meta name="citation_author_institution" content="CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences" />
<meta name="citation_author" content="Huajie Zhang" />
<meta name="citation_author_institution" content="CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences" />
<meta name="citation_author" content="Rui Guo" />
<meta name="citation_author_institution" content="CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences" />
<meta name="citation_author" content="Lijuan Li" />
<meta name="citation_author_institution" content="CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences" />
<meta name="citation_author" content="Yonghong Zhang" />
<meta name="citation_author_institution" content="Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences" />
<meta name="citation_author" content="Tao Deng" />
<meta name="citation_author_institution" content="Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences" />
<meta name="citation_author" content="Hang Sun" />
<meta name="citation_author_institution" content="Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences" />
<meta name="citation_author_email" content="sunyanxia@wbgcas.cn" />
<meta name="citation_author_email" content="sunhang@mail.kib.ac.cn" />
<meta name="citation_author_email" content="hcwang@wbgcas.cn" />
<meta name="citation_author" content="Hengchang Wang" />
<meta name="citation_author_institution" content="CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences" />
<meta name="citation_author_institution" content="Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences" />
<meta name="citation_author_email" content="sunyanxia@wbgcas.cn" />
<meta name="citation_author_email" content="sunhang@mail.kib.ac.cn" />
<meta name="citation_author_email" content="hcwang@wbgcas.cn" />
<meta name="citation_reference" content="Ahrens CW, Byrne M, Rymer PD. 2019. Standing genomic variation within coding and regulatory regions contributes to the adaptive capacity to climate in a foundation tree species. Mol Ecol 28 (10): 2502–2516" />
<meta name="citation_reference" content="Aitken SN, Yeaman S, Holliday JA, Wang T, Curtis-McLane S. 2008. Adaptation, migration or extirpation: climate change outcomes for tree populations. Evol Appl 1 (1): 95–111" />
<meta name="citation_reference" content="Anderson JT, Willis JH, Mitchell-Olds T. 2011. Evolutionary genetics of plant adaptation. Trends Genet 27 (7): 258–266" />
<meta name="citation_reference" content="Bay RA, Harrigan RJ, Underwood VL, Gibbs HL, Smith TB, Ruegg K. 2018. Genomic signals of selection predict climate-driven population declines in a migratory bird. Science 359 (6371): 83." />
<meta name="citation_reference" content="Bolger AM, Lohse M, Usadel B. 2014. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30 (15): 2114–2120" />
<meta name="citation_reference" content="Bouckaert R, Heled J, Kuhnert D, Vaughan T, Wu CH, Xie D, Suchard MA, Rambaut A, Drummond AJ. 2014. BEAST 2: a software platform for Bayesian evolutionary analysis. PLoS Comput Biol 10 (4): e1003537." />
<meta name="citation_reference" content="Catchen J, Hohenlohe PA, Bassham S, Amores A, Cresko WA. 2013. Stacks: an analysis tool set for population genomics. Molecular ecology 22 (11): 3124–3140" />
<meta name="citation_reference" content="Chen Y-S, Deng T, Zhou Z, Sun H. 2017. Is the East Asian flora ancient or not? National Science Review 5 (6): 920–932" />
<meta name="citation_reference" content="Colautti RI, Barrett SCH. 2013. Rapid Adaptation to Climate Facilitates Range Expansion of an Invasive Plant. Science 342 (6156): 364." />
<meta name="citation_reference" content="Conover DO, Duffy TA, Hice LA. 2009. The Covariance between Genetic and Environmental Influences across Ecological Gradients. Annals of the New York Academy of Sciences 1168 (1): 100–129" />
<meta name="citation_reference" content="Coop G, Witonsky D, Di Rienzo A, Pritchard JK. 2010. Using Environmental Correlations to Identify Loci Underlying Local Adaptation. Genetics 185 (4): 1411." />
<meta name="citation_reference" content="Coyne JA, Orr HA. 2004. Speciation. Sunderland, MA, USA: Sinauer Associates." />
<meta name="citation_reference" content="Darriba D, Taboada GL, Doallo R, Posada D. 2012. jModelTest 2: more models, new heuristics and parallel computing. Nat Methods 9 (8): 772." />
<meta name="citation_reference" content="Darwin C. 1859. On the origin of species by means of natural selection, or the preservation of favoured races in the struggle for life. London, UK: John Murray." />
<meta name="citation_reference" content="de Villemereuil P, Gaggiotti OE. 2015. A new FST-based method to uncover local adaptation using environmental variables. Methods in Ecology and Evolution 6 (11): 1248–1258" />
<meta name="citation_reference" content="Dickson BG, Albano CM, Anantharaman R, Beier P, Fargione J, Graves TA, Gray ME, Hall KR, Lawler JJ, Leonard PB, et al. 2019. Circuit-theory applications to connectivity science and conservation. Conservation Biology 33 (2): 239–249" />
<meta name="citation_reference" content="Earl DA, vonHoldt BM. 2012. STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method. Conservation Genetics Resources 4 (2): 359–361" />
<meta name="citation_reference" content="Ellegren H. 2014. Genome sequencing and population genomics in non-model organisms. Trends in Ecology &amp; Evolution 29 (1): 51–63" />
<meta name="citation_reference" content="Excoffier L, Laval G, Schneider S. 2007. Arlequin (version 3.0): an integrated software package for population genetics data analysis. Evolutionary bioinformatics online 1 : 47–50." />
<meta name="citation_reference" content="Favre A, Packert M, Pauls SU, Jahnig SC, Uhl D, Michalak I, Muellner-Riehl AN. 2015. The role of the uplift of the Qinghai-Tibetan Plateau for the evolution of Tibetan biotas. Biol Rev Camb Philos Soc 90 (1): 236–253" />
<meta name="citation_reference" content="Fick SE, Hijmans RJ. 2017. WorldClim 2: new 1-km spatial resolution climate surfaces for global land areas. International Journal of Climatology 37 (12): 4302–4315" />
<meta name="citation_reference" content="Forester BR, Lasky JR, Wagner HH, Urban DL. 2018. Comparing methods for detecting multilocus adaptation with multivariate genotype–environment associations. Molecular Ecology 27 (9): 2215–2233" />
<meta name="citation_reference" content="Foster AS. 1971. Additional studies on the morphology of blind vein-endings in the leaf of circaeaster agrestis. American Journal of Botany 58 (3): 263–272" />
<meta name="citation_reference" content="Friis G, Fandos G, Zellmer AJ, McCormack JE, Faircloth BC, Milá B. 2018. Genome-wide signals of drift and local adaptation during rapid lineage divergence in a songbird. Molecular Ecology 27(24): 5137–5153." />
<meta name="citation_reference" content="citation_title=Flora of China;citation_pages=439;citation_year=2001" />
<meta name="citation_reference" content="Gao LM, MÖLler M, Zhang XM, Hollingsworth ML, Liu J, Mill RR, Gibby M, Li DZ. 2007. High variation and strong phylogeographic pattern among cpDNA haplotypes in Taxus wallichiana (Taxaceae) in China and North Vietnam. Molecular Ecology 16 (22): 4684–4698" />
<meta name="citation_reference" content="Goicoechea PG, Guillardín L, Fernández-Ibarrodo L, Valbuena-Carabaña M, González-Martínez SC, Alía R, Kremer A. 2019. Adaptive Introgression Promotes Fast Adaptation In Oaks Marginal Populations. bioRxiv: 731919." />
<meta name="citation_reference" content="Gomulkiewicz R, Holt RD, Barfield M, Nuismer SL. 2010. Genetics, adaptation, and invasion in harsh environments. Evolutionary Applications 3 (2): 97–108" />
<meta name="citation_reference" content="Gotz S, Garcia-Gomez JM, Terol J, Williams TD, Nagaraj SH, Nueda MJ, Robles M, Talon M, Dopazo J, Conesa A. 2008. High-throughput functional annotation and data mining with the Blast2GO suite. Nucleic Acids Res 36 (10): 3420–3435" />
<meta name="citation_reference" content="Goudet J, Raymond M, de Meeüs T, Rousset F. 1996. Testing Differentiation in Diploid Populations. Genetics 144 (4): 1933." />
<meta name="citation_reference" content="Gu Y, Kaplinsky N, Bringmann M, Cobb A, Carroll A, Sampathkumar A, Baskin TI, Persson S, Somerville CR. 2010. Identification of a cellulose synthase-associated protein required for cellulose biosynthesis. Proceedings of the National Academy of Sciences 107 (29): 12866." />
<meta name="citation_reference" content="Gugger PF, Liang CT, Sork VL, Hodgskiss P, Wright JW. 2017. Applying landscape genomic tools to forest management and restoration of Hawaiian koa (Acacia koa) in a changing environment. Evolutionary Applications 11 (2): 231–242" />
<meta name="citation_reference" content="Heled J. 2010. Extended Bayesian Skyline Plots Tutorial." />
<meta name="citation_reference" content="Heled J, Drummond AJ. 2008. Bayesian inference of population size history from multiple loci. BMC Evolutionary Biology 8 (1): 289." />
<meta name="citation_reference" content="Ho SYW, Phillips MJ. 2009. Accounting for Calibration Uncertainty in Phylogenetic Estimation of Evolutionary Divergence Times. Systematic Biology 58 (3): 367–380." />
<meta name="citation_reference" content="Hoban S, Kelley JL, Lotterhos KE, Antolin MF, Bradburd G, Lowry DB, Poss ML, Reed LK, Storfer A, Whitlock MC. 2016. Finding the Genomic Basis of Local Adaptation: Pitfalls, Practical Solutions, and Future Directions. The American Naturalist 188 (4): 379–397" />
<meta name="citation_reference" content="Jakobsson M, Rosenberg NA. 2007. CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure. Bioinformatics 23 (14): 1801–1806" />
<meta name="citation_reference" content="Jia K-H, Zhao W, Maier PA, Hu X-G, Jin Y, Zhou S-S, Jiao S-Q, El-Kassaby YA, Wang T, Wang X-R, et al. 2019. Landscape genomics predicts climate change-related genetic offset for the widespread Platycladus orientalis (Cupressaceae). Evolutionary Applications https://doi.org/10.1111/eva.12891." />
<meta name="citation_reference" content="Jiang X-L, Gardner EM, Meng H-H, Deng M, Xu G-B. 2019. Land bridges in the Pleistocene contributed to flora assembly on the continental islands of South China: Insights from the evolutionary history of Quercus championii. Molecular Phylogenetics and Evolution 132 : 36–45." />
<meta name="citation_reference" content="Jombart T. 2008. adegenet: a R package for the multivariate analysis of genetic markers. Bioinformatics 24 (11): 1403–1405" />
<meta name="citation_reference" content="Katoh K, Standley DM. 2013. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol 30 (4): 772–780" />
<meta name="citation_reference" content="Kawecki TJ, Ebert D. 2004. Conceptual issues in local adaptation. Ecology Letters 7(12): 1225–1241." />
<meta name="citation_reference" content="Kearse M, Moir R, Wilson A, Stones-Havas S, Cheung M, Sturrock S, Buxton S, Cooper A, Markowitz S, Duran C, et al. 2012. Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics 28 (12): 1647–1649" />
<meta name="citation_reference" content="Kimura M, Crow JF. 1964. The number of alleles that can be maintained in a finite population. Genetics 49 (4): 725–738" />
<meta name="citation_reference" content="Kramer AT, Havens K. 2009. Plant conservation genetics in a changing world. Trends in Plant Science 14 (11): 599–607" />
<meta name="citation_reference" content="Leinonen PH, Remington DL, LeppÄLÄ J, Savolainen O. 2013. Genetic basis of local adaptation and flowering time variation in Arabidopsis lyrata. Molecular Ecology 22 (3): 709–723" />
<meta name="citation_reference" content="Li H, Luan S. 2011. The cyclophilin AtCYP71 interacts with CAF-1 and LHP1 and functions in multiple chromatin remodeling processes. Mol Plant 4 (4): 748–758" />
<meta name="citation_reference" content="Li Y, Zhai S-N, Qiu Y-X, Guo Y-P, Ge X-J, Comes HP. 2011. Glacial survival east and west of the ‘Mekong–Salween Divide’in the Himalaya–Hengduan Mountains region as revealed by AFLPs and cpDNA sequence variation in Sinopodophyllum hexandrum (Berberidaceae). Molecular Phylogenetics and Evolution 59 (2): 412–424" />
<meta name="citation_reference" content="Liu J, Möller M, Provan J, Gao L-M, Poudel RC, Li D-Z. 2013. Geological and ecological factors drive cryptic speciation of yews in a biodiversity hotspot. New Phytologist 199 (4): 1093–1108" />
<meta name="citation_reference" content="Lowry DB, Lovell JT, Zhang L, Bonnette J, Fay PA, Mitchell RB, Lloyd-Reilley J, Boe AR, Wu Y, Rouquette FM, et al. 2019. QTL × environment interactions underlie adaptive divergence in switchgrass across a large latitudinal gradient. Proceedings of the National Academy of Sciences 116 (26): 12933." />
<meta name="citation_reference" content="McCormack JE, Hird SM, Zellmer AJ, Carstens BC, Brumfield RT. 2013. Applications of next-generation sequencing to phylogeography and phylogenetics. Molecular Phylogenetics and Evolution 66 (2): 526–538" />
<meta name="citation_reference" content="McRae BH, Beier P. 2007. Circuit theory predicts gene flow in plant and animal populations. Proceedings of the National Academy of Sciences 104 (50): 19885." />
<meta name="citation_reference" content="Mitchell-Olds T, Willis JH, Goldstein DB. 2007. Which evolutionary processes influence natural genetic variation for phenotypic traits? Nature Reviews Genetics 8 : 845." />
<meta name="citation_reference" content="Mustilli AC, Merlot S, Vavasseur A, Fenzi F, Giraudat J. 2002. Arabidopsis OST1 protein kinase mediates the regulation of stomatal aperture by abscisic acid and acts upstream of reactive oxygen species production. The Plant cell 14 (12): 3089–3099" />
<meta name="citation_reference" content="Nevado B, Wong ELY, Osborne OG, Filatov DA. 2019. Adaptive Evolution Is Common in Rapid Evolutionary Radiations. Current Biology." />
<meta name="citation_reference" content="Nielsen R. 2005. Molecular Signatures of Natural Selection. Annual Review of Genetics 39 (1): 197–218" />
<meta name="citation_reference" content="Nowakowski AJ, DeWoody JA, Fagan ME, Willoughby JR, Donnelly MA. 2015. Mechanistic insights into landscape genetic structure of two tropical amphibians using field-derived resistance surfaces. Mol Ecol 24(3): 580–595." />
<meta name="citation_reference" content="Oksanen J, Blanchet FG, Friendly M, Kindt R, Legendre P, McGlinn D, Minchin PR, O’Hara RB, Simpson GL, Solymos P, et al. 2018. Vegan: Community Ecology Package. https://CRAN.R-project.org/package=vegan." />
<meta name="citation_reference" content="Peakall R, Smouse PE. 2012. GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research--an update. Bioinformatics 28 (19): 2537–2539" />
<meta name="citation_reference" content="Pham J, Liu J, Bennett MH, Mansfield JW, Desikan R. 2012. Arabidopsis histidine kinase 5 regulates salt sensitivity and resistance against bacterial and fungal infection. New Phytol 194 (1): 168–180" />
<meta name="citation_reference" content="Phillips SJ, Dudik M. 2008. Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation. Ecography 31 (2): 161–175" />
<meta name="citation_reference" content="Plummer M, Best N, Cowles K, Vines K. 2006. CODA: convergence diagnosis and output analysis for MCMC. R News 6 (1): 7–11" />
<meta name="citation_reference" content="Poudel RC, Möller M, Liu J, Gao L-M, Baral SR, Li D-Z. 2014. Low genetic diversity and high inbreeding of the endangered yews in Central Himalaya: implications for conservation of their highly fragmented populations. Diversity and Distributions 20 (11): 1270–1284" />
<meta name="citation_reference" content="Pritchard JK, Stephens M, Donnelly P. 2000. Inference of population structure using multilocus genotype data. Genetics 155 (2): 945–959" />
<meta name="citation_reference" content="Qu XJ, Moore MJ, Li DZ, Yi TS. 2019. PGA: a software package for rapid, accurate, and flexible batch annotation of plastomes. Plant Methods 15 (1): 50." />
<meta name="citation_reference" content="R Team C 2014. R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna, Austria." />
<meta name="citation_reference" content="Rambaut A, Drummond A. 2010. TreeAnnotator version 1.6. 1. University of Edinburgh, Edinburgh, UK. Available at: http://beast.bio.ed.ac.uk." />
<meta name="citation_reference" content="Rambaut A, Drummond AJ, Xie D, Baele G, Suchard MA. 2018. Posterior Summarization in Bayesian Phylogenetics Using Tracer 1.7. Syst Biol 67 (5): 901–904" />
<meta name="citation_reference" content="Ree RH, Smith SA. 2008. Maximum likelihood inference of geographic range evolution by dispersal, local extinction, and cladogenesis. Syst Biol 57 (1): 4–14" />
<meta name="citation_reference" content="Rellstab C, Gugerli F, Eckert AJ, Hancock AM, Holderegger R. 2015. A practical guide to environmental association analysis in landscape genomics. Molecular Ecology 24 (17): 4348–4370" />
<meta name="citation_reference" content="Ren G, Mateo RG, Liu J, Suchan T, Alvarez N, Guisan A, Conti E, Salamin N. 2017. Genetic consequences of Quaternary climatic oscillations in the Himalayas: Primula tibetica as a case study based on restriction site-associated DNA sequencing. New Phytologist 213 (3): 1500–1512" />
<meta name="citation_reference" content="Ren Y, Li Z, Hu Z. 2003. Approaches to the systematic position of Circaeaster based on the morphological data. Acta Botanica Boreali-occidentalia Sinica 23 (7): 1091–1097" />
<meta name="citation_reference" content="Rochette NC, Catchen JM. 2017. Deriving genotypes from RAD-seq short-read data using Stacks. Nature Protocols 12 : 2640." />
<meta name="citation_reference" content="Rosenberg NA. 2004. Distruct: a program for the graphical display of population structure. Molecular Ecology Notes 4 (1): 137–138" />
<meta name="citation_reference" content="Rozas J, Ferrer-Mata A, Sanchez-DelBarrio JC, Guirao-Rico S, Librado P, Ramos-Onsins SE, Sanchez-Gracia A. 2017. DnaSP 6: DNA Sequence Polymorphism Analysis of Large Data Sets. Mol Biol Evol 34 (12): 3299–3302" />
<meta name="citation_reference" content="Ruiz-Sanchez E, Rodriguez-Gomez F, Sosa V. 2012. Refugia and geographic barriers of populations of the desert poppy, Hunnemannia fumariifolia (Papaveraceae). Organisms Diversity &amp; Evolution 12 (2): 133–143" />
<meta name="citation_reference" content="Savolainen O, Lascoux M, Merilä J. 2013. Ecological genomics of local adaptation. Nature Reviews Genetics 14 : 807." />
<meta name="citation_reference" content="Savolainen O, Pyhäjärvi T, Knürr T. 2007. Gene Flow and Local Adaptation in Trees. Annual Review of Ecology, Evolution, and Systematics 38 (1): 595–619" />
<meta name="citation_reference" content="Seehausen O, Butlin RK, Keller I, Wagner CE, Boughman JW, Hohenlohe PA, Peichel CL, Saetre G-P, Bank C, Brännström Å, et al. 2014. Genomics and the origin of species. Nature Reviews Genetics 15 : 176." />
<meta name="citation_reference" content="Sierla M, Horak H, Overmyer K, Waszczak C, Yarmolinsky D, Maierhofer T, Vainonen JP, Salojarvi J, Denessiouk K, Laanemets K, et al. 2018. The Receptor-like Pseudokinase GHR1 Is Required for Stomatal Closure. The Plant cell 30 (11): 2813–2837" />
<meta name="citation_reference" content="Spieth PT. 1974. Gene flow and genetic differentiation. Genetics 78 (3): 961." />
<meta name="citation_reference" content="Stroud DA, Surgenor EE, Formosa LE, Reljic B, Frazier AE, Dibley MG, Osellame LD, Stait T, Beilharz TH, Thorburn DR, et al. 2016. Accessory subunits are integral for assembly and function of human mitochondrial complex I. Nature 538 (7623): 123–126" />
<meta name="citation_reference" content="Sun B-N, Wu J-Y, Liu Y-S, Ding S-T, Li X-C, Xie S-P, Yan D-F, Lin Z-C. 2011. Reconstructing Neogene vegetation and climates to infer tectonic uplift in western Yunnan, China. Palaeogeography, Palaeoclimatology, Palaeoecology 304 (3): 328–336" />
<meta name="citation_reference" content="Sun H, Zhang J, Deng T, Boufford DE. 2017. Origins and evolution of plant diversity in the Hengduan Mountains, China. Plant Diversity 39 (4): 161–166" />
<meta name="citation_reference" content="Sun Y, Deng T, Zhang A, Moore MJ, Landis JB, Lin N, Zhang H, Zhang X, Huang J, Zhang X, et al. 2020. The draft genome of the endangered, relictual plant Kingdonia uniflora (Circaeasteraceae, Ranunculales) reveals potential mechanisms and perils of evolutionary specialization. bioRxiv: 2020.2001.2008.898460." />
<meta name="citation_reference" content="Sun Y, Moore MJ, Lin N, Adelalu KF, Meng A, Jian S, Yang L, Li J, Wang H. 2017. Complete plastome sequencing of both living species of Circaeasteraceae (Ranunculales) reveals unusual rearrangements and the loss of the ndh gene family. BMC Genomics 18(1): 592." />
<meta name="citation_reference" content="Tognetti, V. B., Van Aken, O., Morreel, K., Vandenbroucke, K., van de Cotte, B., De Clercq, I., et al. 2010. Perturbation of indole-3-butyric acid homeostasis by the UDP-glucosyltransferase UGT74E2 modulates Arabidopsis architecture and water stress tolerance. The Plant cell, 22 (8), 2660–2679." />
<meta name="citation_reference" content="Weigel D, Nordborg M. 2015. Population Genomics for Understanding Adaptation in Wild Plant Species. Annual Review of Genetics 49 (1): 315–338" />
<meta name="citation_reference" content="Weir BS, Cockerham CC. 1984. Estimating F-statistics for the analysis of population structure. Evolution 38 (6): 1358–1370" />
<meta name="citation_reference" content="Wen J, Zhang JQ, Nie ZL, Zhong Y, Sun H. 2014. Evolutionary diversifications of plants on the Qinghai-Tibetan Plateau. Front Genet 5 : 4." />
<meta name="citation_reference" content="Wright S. 1946. Isolation by distance under diverse systems of mating. Genetics 31 (1): 39–59" />
<meta name="citation_reference" content="Yang JB, Li DZ, Li HT. 2014. Highly effective sequencing whole chloroplast genomes of angiosperms by nine novel universal primer pairs. Mol Ecol Resour 14 (5): 1024–1031" />
<meta name="citation_reference" content="Yu Y, Harris AJ, Blair C, He X. 2015. RASP (Reconstruct Ancestral State in Phylogenies): a tool for historical biogeography. Mol Phylogenet Evol 87 : 46–49." />
<meta name="citation_reference" content="Zhang D, Gao F, Jakovlic I, Zou H, Zhang J, Li WX, Wang GT. 2019. PhyloSuite: An integrated and scalable desktop platform for streamlined molecular sequence data management and evolutionary phylogenetics studies. Mol Ecol Resour doi:10.1111/1755-0998.13096." />
<meta name="citation_reference" content="Zou Y-P, Hou X-H, Wu Q, Chen J-F, Li Z-W, Han T-S, Niu X-M, Yang L, Xu Y-C, Zhang J, et al. 2017. Adaptation of Arabidopsis thaliana to the Yangtze River basin. Genome Biology 18 (1): 239." />
<meta name="twitter:title" content="Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual Circaeaster agrestis (Circaeasteraceae, Ranunculales)" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="A primary goal of phylogeography is understanding how geographical and ecological factors shape the genetic structure and evolutionary history of species. Environmental factors can act in concert and vary in space, resulting in divergent natural selection leading to genetic divergence during" />
<meta name="og-title" property="og:title" content="Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual Circaeaster agrestis (Circaeasteraceae, Ranunculales)" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.14.902643v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="A primary goal of phylogeography is understanding how geographical and ecological factors shape the genetic structure and evolutionary history of species. Environmental factors can act in concert and vary in space, resulting in divergent natural selection leading to genetic divergence during " />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-16" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.14.902643v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.14.902643v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.14.902643v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.14.902643v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1107107" />
    <title>Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual Circaeaster agrestis (Circaeasteraceae, Ranunculales) | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"tPhhIhJwvzt0o1W5JVHha8MdmJlvXMUTTRgB2zElZtU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.01.14.902643v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.14.902643v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.14.902643v2"}],"nid":"1107107","apath":"\/biorxiv\/early\/2020\/01\/16\/2020.01.14.902643.atom","pisa":"biorxiv;2020.01.14.902643v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.14.902643v2","title":"Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual Circaeaster agrestis (Circaeasteraceae, Ranunculales)","identifier":"node\/1107107"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002258XSb0fFrb5nirvBlsE2gTrPZQwLEBNTY6DkMOT+R6I4flz5IVVXgn7y3N370I5OPSa7nS2dYOWZp1ib+T6hJEtMGmWI0+tgg6He+k1tNETXysd1+Yfr1Rln7BjFd8GtreSvm+GJ0pg+ung3rEZ15jSunRa7KBjbxCfApIAF6cBURly1GCmzAElKQh0Po6xGorgXbI7SgnCFKJzM8HSjsTBIn+YankkZVIiNaFYHNmYIm+f1SSrYB6fMr5CHhl0cTdTyJ\\\\\\\/7BLM2LCp7AEU6cxdNwWMYyygEFuz0Zo1jGuQYHw\\\\\\\/l5f3UWQFFwjQgtETMbLfQci67XZaMOeP6v09JLQjTMF6aZ\\\\\\\/5PSWpN7IxAMbjQw1lcGZAHTtHr3NY5ua92iPW2VuRll8QZZsd\\\\\\\/ii2ZbnlBIoyq42yRD4kH7fftEhNvploSk0gDAt1dGJhKk6teLcaE0ICnP3uQ0\\\\\\\/UuuQgLiQgC4hV0W\\\\\\\/qWXu9OdfNCMYoDrqE0fot2WXaUxFYlcYIzyXlkm66slB9ZbRbyZ5fm3dVWInr6e\\\\\\\/ZermRk0mS0GUkWjaD\\\\\\\/ED6C+uqwKCJ9yDUGvQWzFzcUyDObTJN9l1OzXtfXmcIjvLhnbQuX9e8gooG55+U4rcVplpJjT5yacNUuBN6J\\\\\\\/bnmlJJ38zfOAS5SX+0cw2VwUWVZRrfdttjmwTA97cD\\\\\\\/QiFPdr6qyFOxffTC1qEPmNbIJiUu02pNG0MPzA8p8a+w+QA0Xcdrdn\\\\\\\/bzHBeBxqIpWnVaSTrfSE2clONt919s15PtyzDbjlgcygInwnw7Bv0FmpbbPq8GQjfl1eAdGkPjqBv6lBPdbjaSEjEpxcTSrxj61hx8zwe2icHeNO7c\\\\\\\/RQwFdy3MxyNPIuPSckqIQQzSJyvwF7nLqm3tRzcXguxDuZ6WN9QH1cX38A7zAwSBkG\\\\\\\/w2CXz7GG6WrxrYNfc\\\\\\\/KnvP1daH\\\\\\\/\\\\\\\/qxTxkbpo2yPE85v3ZB0VHtdnrGw9ZuJUXz+wTqPqefyPRpNCMaBrM6IHlYAVi\\\\\\\/P\\\\\\\/Ofpf0babZO5u6CSliT2YghnzJTUTVtKxTdyin1qB\\\\\\\/m9RdU2bPBk0TXfeuGL0MqAoB63nARrVCnfKUB5KYWrJdsYrnTEHNkrg26MrH0Ch\\\\\\\/KELv3VViVNk8vCC6zgOGVpWXz2uw5kRqbN0AIz85TKuIyXy4cURg5vdOTn6RbHr6h8LnPCZHyK0DDrRoOFuK51+xTWeYX7EQZEc5NAKnZxRgDJhhVKw7GCNlvrQOXIU6DLxvRG6KT7gzW+pcLeiqn5zUXfJPE2UWh3VC8sxu5ZsWa8zVC4lP6UygLaGyqFApsWKR10eCRP05jeFPoDiO02c5njtbqCp\\\u0022,\\\u0022iv\\\u0022:\\\u0022paKGZEU19jTpkoqug41pvw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00227ff800a1a14b99125b88c22e7b451a3913fa8717a781ad3acdfafe0ce5d908a6\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.14.902643v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1107107 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.14.902643v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y7tU54WwsRKaiMx5sbddMyh5vwr-6K5XGazOPAAKH7M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107107" id="node1107107" data-pisa="biorxiv;2020.01.14.902643v2" data-pisa-master="biorxiv;2020.01.14.902643" data-apath="/biorxiv/early/2020/01/16/2020.01.14.902643.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual <em>Circaeaster agrestis</em> (Circaeasteraceae, Ranunculales)</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5743-3497" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Xu</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yanxia</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jacob B.</span> <span class="nlm-surname">Landis</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianwen</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Linsen</span> <span class="nlm-surname">Yang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Nan</span> <span class="nlm-surname">Lin</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Huajie</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Rui</span> <span class="nlm-surname">Guo</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Lijuan</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Yonghong</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Deng</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Hang</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Hengchang</span> <span class="nlm-surname">Wang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2020.01.14.902643 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1107107" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xu Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>University of Chinese Academy of Sciences</span>, Beijing, 100049 <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xu%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXu%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5743-3497" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xu Zhang</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yanxia Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yanxia%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYanxia%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sunyanxia@wbgcas.cn" class="" data-icon-position="" data-hide-link-title="0">sunyanxia@wbgcas.cn</a>
<a href="mailto:sunhang@mail.kib.ac.cn" class="" data-icon-position="" data-hide-link-title="0">sunhang@mail.kib.ac.cn</a>
<a href="mailto:hcwang@wbgcas.cn" class="" data-icon-position="" data-hide-link-title="0">hcwang@wbgcas.cn</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jacob B. Landis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Botany and Plant Sciences, University of California Riverside</span>, Riverside, CA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacob%2BB.%2BLandis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Landis%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacob%2BB.%2BLandis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jianwen Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences</span>, Kunming, Yunnan, 650201 <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianwen%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianwen%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Linsen Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Hubei Key Laboratory of Shennongjia Golden Monkey Conservation Biology, Administration of Shennongjia National Park</span>, Shennongjia, Hubei, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linsen%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinsen%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Nan Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>University of Chinese Academy of Sciences</span>, Beijing, 100049 <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nan%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANan%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Huajie Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huajie%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuajie%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Rui Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>University of Chinese Academy of Sciences</span>, Beijing, 100049 <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rui%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARui%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Lijuan Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>University of Chinese Academy of Sciences</span>, Beijing, 100049 <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lijuan%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALijuan%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Yonghong Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences</span>, Kunming, Yunnan, 650201 <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yonghong%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYonghong%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Tao Deng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences</span>, Kunming, Yunnan, 650201 <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tao%2BDeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deng%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATao%2BDeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Hang Sun </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Key Laboratory for Plant Diversity and Biogeography of East Asia, Kunming Institute of Botany, Chinese Academy of Sciences</span>, Kunming, Yunnan, 650201 <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hang%2BSun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sun%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHang%2BSun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sunyanxia@wbgcas.cn" class="" data-icon-position="" data-hide-link-title="0">sunyanxia@wbgcas.cn</a>
<a href="mailto:sunhang@mail.kib.ac.cn" class="" data-icon-position="" data-hide-link-title="0">sunhang@mail.kib.ac.cn</a>
<a href="mailto:hcwang@wbgcas.cn" class="" data-icon-position="" data-hide-link-title="0">hcwang@wbgcas.cn</a></span></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Hengchang Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CAS Key Laboratory of Plant Germplasm Enhancement and Specialty Agriculture, Wuhan Botanical Garden, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div><div class='author-affiliation'><span class='nlm-institution'>Center of Conservation Biology, Core Botanical Gardens, Chinese Academy of Sciences</span>, Wuhan 430074, Hubei, <span class='nlm-country'>China</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hengchang%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHengchang%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sunyanxia@wbgcas.cn" class="" data-icon-position="" data-hide-link-title="0">sunyanxia@wbgcas.cn</a>
<a href="mailto:sunhang@mail.kib.ac.cn" class="" data-icon-position="" data-hide-link-title="0">sunhang@mail.kib.ac.cn</a>
<a href="mailto:hcwang@wbgcas.cn" class="" data-icon-position="" data-hide-link-title="0">hcwang@wbgcas.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2020.01.14.902643v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1107107" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1107107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.14.902643v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1107107" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1107107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.14.902643v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1107107" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1107107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.14.902643v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1107107" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1107107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2020.01.14.902643v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1107107" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1107107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2020.01.14.902643v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1107107" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1107107/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Summary</h2><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-3">Investigating the interaction between environmental heterogeneity and local adaptation is critical to understand the evolutionary history of a species, providing the premise for studying the response of organisms to rapid climate change. However, for most species how exactly the spatial heterogeneity promotes population divergence and how genomic variations contribute to adaptive evolution remain poorly understood.</p></li><li id="list-item-2"><p id="p-4">We examine the contributions of geographical and environmental variables to population divergence of the relictual, alpine herb <em>Circaeaster agrestis</em> as well as genetic basis of local adaptation using RAD-seq and plastome data.</p></li><li id="list-item-3"><p id="p-5">We detected significant genetic structure with an extraordinary disequilibrium of genetic diversity among regions, and signals of isolation-by-distance along with isolation-by-resistance. The populations were estimated to begin diverging in the late Miocene, along with a possible ancestral distribution of the Hengduan Mountains and adjacent regions. Both environmental gradient and redundancy analyses revealed significant association between genetic variation and temperature variables. Genome-environment association analyses identified 16 putatively adaptive loci related to biotic and abiotic stress resistance.</p></li><li id="list-item-4"><p id="p-6">Our genome wide data provide new insights into the important role of environmental heterogeneity in shaping genetic structure, and access the footprints of local adaptation in an ancient relictual species.</p></li></ul></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-7">A primary goal of phylogeography is understanding how geographical and ecological factors shape the genetic structure and evolutionary history of species. Environmental factors can act in concert and vary in space, resulting in divergent natural selection leading to genetic divergence during adaptation to heterogeneous environments (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Savolainen <em>et al.</em>, 2007</a>). Such genetic divergence along environmental gradients, or across varied ecological habitats, can be indicative of local adaptation, a mechanism that is beneficial to the long persistence of a population and intensify differentiation among populations (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kawecki &amp; Ebert, 2004</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Conover <em>et al.</em>, 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Colautti &amp; Barrett, 2013</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Savolainen <em>et al.</em>, 2013</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lowry <em>et al.</em>, 2019</a>). Understanding the genetic basis of local adaptation will provide deeper insights into the response of organisms to ongoing climate change, aiding efforts to the conservation of genetic resources (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aitken <em>et al.</em>, 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahrens <em>et al.</em>, 2019</a>).</p><p id="p-8">On the one hand, divergent natural selection is the driving force of genetic divergence (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Darwin, 1859</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Coyne &amp; Orr, 2004</a>), while on the other hand, gene flow can buffer genetic differentiation (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Spieth, 1974</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Goicoechea <em>et al.</em>, 2019</a>). Evolution resulting from local adaptation depends on both the strength of local selective pressures and the homogenizing effect of connectivity among different localities. Additionally, genetic drift may reinforce the establishment and maintenance of genetic divergence, particularly in populations with small effective size (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Wright, 1946</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kimura &amp; Crow, 1964</a>). Therefore, quantifying the relative contributions among natural selection, genetic drift and population connectivity to genetic divergence is crucial for understanding the role of local adaptation in adaptive evolution of a species (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Savolainen <em>et al.</em>, 2007</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Friis <em>et al.</em>, 2018</a>).</p><p id="p-9">Phenotypic patterns that are associated with divergent selection have been observed in model plants (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Mitchell-Olds <em>et al.</em>, 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em>, 2011</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Leinonen <em>et al.</em>, 2013</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Zou <em>et al.</em>, 2017</a>). However, identification of genetic loci that govern phenotypic variations in response to environmental pressures is a challenging task, especially for non-model species with limited genomic resources. Facilitated by advancements in sequencing technology and population genomics, it is now possible to study hundreds of thousands of loci improving our understanding of genome-wide effects of accumulating genetic divergence and genomic properties that influence the process of local adaptation (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">McCormack <em>et al.</em>, 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ellegren, 2014</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Seehausen <em>et al.</em>, 2014</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Weigel &amp; Nordborg, 2015</a>). In particular, highly divergent loci identified by Genome-Environment Association (GEA) analyses can be interpreted as potential targets of divergent selection associated with population-specific environmental covariables (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Coop <em>et al.</em>, 2010</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Rellstab <em>et al.</em>, 2015</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hoban <em>et al.</em>, 2016</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Forester <em>et al.</em>, 2018</a>). In this context, plant studies are poised to reveal the genomic signatures underlying the evolution of local adaptation (e.g. <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Seehausen <em>et al.</em>, 2014</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ahrens <em>et al.</em>, 2019</a>). However, for most species, such as relictual alpine herbs, how exactly the spatial heterogeneity promotes population diversification and how genomic variations contribute to adaptive evolution, in particular their interaction, remains poorly understood.</p><p id="p-10">In the present study, we focus on investigating the evolutionary history and local adaptation of <em>Circaeaster agrestis</em> Maxim, an annual alpine herb with a relatively narrow habitat (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fu &amp; Bartholomew, 2001</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Sun, Y <em>et al.</em>, 2017</a>). <em>Circaeaster</em> Maxim., with its only alliance <em>Kingdonia</em> Balf.f. &amp; W.W. Smith, constitute the early-diverging eudicot family Circaeasteraceae (Ranunculales) (The Angiosperm Phylogeny et al., 2016). Due to special morphological characters such as open dichotomous leaf venation, this genus is of great interest to many botanists (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Foster, 1971</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Ren <em>et al.</em>, 2003</a>). <em>Circaeaster</em> exhibits low morphological diversity with only one species, garnering a status of critical endangerment (Wild Plants Under State Protection in China). The distributions of <em>C. agrestis</em> is confined to the Qinghai-Tibet Plateau (QTP) and adjacent areas where exhibit a wide range of elevations from 2,100-5,000 m (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Fu &amp; Bartholomew, 2001</a>). The uplift of QTP has created extraordinarily geomorphological and climatic diversity in this area. Due to the high mountain barrier formed by the Himalayas in the south, the central and western QTP (i.e. the Himalayas) are characterized by a cold and dry climate (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Favre <em>et al.</em>, 2015</a>). In contrast, the eastern QTP (i.e. the Hengduan Mountains) and adjacent arears are associated with deep valleys and characterized mainly by a warm and wet climate (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Ren <em>et al.</em>, 2017</a>). Given the high degree of geographical and ecological heterogeneity, QTP comprises a promising model system for studying genetic signatures of local adaptation.</p><p id="p-11">Specifically, we generated and analyzed two types of genomic datasets: single nucleotide polymorphisms (SNPs) derived from the restriction site associated DNA sequencing (RAD-Seq) of 18 <em>C. agrestis</em> populations and 20 plastome sequences of <em>C. agrestis</em> populations with other Ranunculales representatives. We aim to illustrate the evolutionary patterns of population divergence interacting with environmental variables and access the potential genetic basis of local adaptation. Here, we raise the following questions: 1) how alpine environments affect genetic structure within species and drive population divergence, and 2) how genetic variation is associated with local adaptation? Answering these questions will help us understand species adaptation to rapidly changing climate, and aid in conservation of small patchy diversity in biodiversity hotspots.</p></div><div class="section" id="sec-2"><h2 class="">Materials and Methods</h2><div id="sec-3" class="subsection"><h3>Sampling, RAD library preparation and sequencing</h3><p id="p-12">A total of 139 individuals of <em>C. agrestis</em> from 18 localities were sampled (Table S1). Sampling locations were chosen based on existing occurrence records covering the geographic and climatic distribution of the species (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). All materials were dried and stored in silica gel, and all the voucher specimens were deposited in Herbarium of Wuhan Botanical Garden (HIB) (Table S1). Total DNA was extracted from silica gel-dried leaves with a modified CTAB (Cetyl trimethylammonium bromide) method (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Yang <em>et al.</em>, 2014</a>). For RAD library construction and sequencing, genomic DNA was digested with the restriction enzyme EcoRI in a 30 ul reaction. T4 ligase was used to ligate a P1 adapter to the fragments. The adapter-ligated fragments were subsequently pooled, randomly sheared, and size-selected with a length of 350–550 bp. DNA was then ligated to a second adapter (P2), a Y adapter hat has divergent ends to the chosen DNA fragments. The ligation products were purified and PCR amplified, followed by gel purification, and finally size selected to 350-550 bp. Agilent 2100 Bioanaylzer and qPCR were used to qualify and quantify the sample library. Paired-end sequencing was performed on two lanes of Illumina HiSeq 2000 platform at BGI-Shenzhen (Shenzhen, Guangdong, China).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Geographical distribution of haplotypes detected in polymorphic sites of Circaeaster agrestis populations. Colors indicate genetic groups based on STRUCTURE and PoCA results (see also Fig. 2), with the Eastern clade being made up of the orange and dark green circles, with the Western clade consisting of the light green, light pink, purple, and blue circles. Circle sizes reflect the number of individuals sampled." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-983153512" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Geographical distribution of haplotypes detected in polymorphic sites of Circaeaster agrestis populations. Colors indicate genetic groups based on STRUCTURE and PoCA results (see also Fig. 2), with the Eastern clade being made up of the orange and dark green circles, with the Western clade consisting of the light green, light pink, purple, and blue circles. Circle sizes reflect the number of individuals sampled.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F1.medium.gif" width="440" height="339"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F1.medium.gif" width="440" height="339"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1</span> <p id="p-13" class="first-child">Geographical distribution of haplotypes detected in polymorphic sites of Circaeaster agrestis populations. Colors indicate genetic groups based on STRUCTURE and PoCA results (see also <a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>), with the Eastern clade being made up of the orange and dark green circles, with the Western clade consisting of the light green, light pink, purple, and blue circles. Circle sizes reflect the number of individuals sampled.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">For plastome sequencing, 20 samples covering all 18 localities were chosen. For each sample, a 500-bp DNA TruSeq Illumina (Illumina Inc., San Diego, CA, USA) sequencing library was constructed using 2.5-5.0 ng sonicated DNA as input. Then the libraries were quantified using an Agilent 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA) and real-time quantitative PCR. Libraries were multiplexed and sequenced using a 2×125 bp run on one lane of Illumina HiSeq 2000 platform at BGI-Shenzhen (Shenzhen, Guangdong, China).</p></div><div id="sec-4" class="subsection"><h3>Processing of Illumina data</h3><p id="p-15">For RAD-sequencing, pair-end Illumina reads were processed into RAD-tags using the STACKS v.2.30 software pipeline (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Catchen <em>et al.</em>, 2013</a>). Samples were initially demultiplexed with PROCESS_RADTAGS. Reads with an average Phred score of at least 30, an unambiguous barcode, and restriction cut site were retained. Then the perl wrapper DENOVO_MAP.PL was used to identify optimal parameters by executing USTACKS, CSTACKS, and SSTACKS (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Catchen <em>et al.</em>, 2013</a>). Following the suggested protocol, we investigated a range of parameter values with <em>M</em> and <em>n</em> values ranging from 1 to 9 (fixing <em>M</em> = <em>n</em>) and <em>m</em> = 3 (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Rochette &amp; Catchen, 2017</a>). We plotted two statistics that were useful in making the appropriate parameter choice: the number of polymorphic loci shared across most samples (the r80 loci) and the distribution of the number of SNPs per locus.</p><p id="p-16">We filtered the catalog of reads using the POPULATIONS module in the STACKS pipeline to produce datasets for downstream population genetic analyses. Polymorphic RAD loci that presented in all 139 individuals were retained for subsequent analyses. To validate the influence of missing data, we also employed a loose parameter of <em>- r</em> = 0.8, in which at least 80% of individuals in a population were required to process a locus for that population. Potential homologs were excluded by removing loci showing heterozygosity &gt; 0.5 within samples. We further filtered our dataset with a minor allele frequency (MAF) &gt; 0.01 and kept only biallelic SNPs. Finally, per locus, we filtered our dataset by selecting the most informative SNP based on the number of minor alleles.</p><p id="p-17">For genome-skimming sequencing, raw sequence reads were subsequently filtered using Trimmomatic v.0.36 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em>, 2014</a>) by removing duplicate reads and adapter-contaminated reads. Remaining high-quality reads were directly mapped to the reported plastome of <em>C. agrestis</em> (NCBI accession number: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KY908400&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom">KY908400</a>) using Geneious v9.0.2 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kearse <em>et al.</em>, 2012</a>). Initial annotations were implemented in the Plastid Genome Annotator (PGA) (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Qu <em>et al.</em>, 2019</a>), and then were refined by manual corrections in Geneious. The annotated plastomes were deposited in GenBank under accession numbers xxx-xxx.</p></div><div id="sec-5" class="subsection"><h3>Genetic diversity and population structure</h3><p id="p-18">We estimated genetic diversity indices including nucleotide diversity (<em>π</em>), expected heterozygosity (<em>H<sub>e</sub></em>), observed heterozygosity (<em>H<sub>o</sub></em>) and inbreeding coefficients (<em>F</em><sub>IS</sub>) using POPULATIONS based on the RAD-seq data. The polymorphic sites in <em>Phylip</em> format were exported from POPULATIONS and were used for calculating haplotypes in DnaSP v6.12 (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Rozas <em>et al.</em>, 2017</a>). The <em>hierfstat</em> package (Goudet, 2005) in R v3.6.1 (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">R Team, 2014</a>) was used to estimate population and group level <em>F</em><sub>ST</sub> using Weir and Cockerham’s method (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Weir &amp; Cockerham, 1984</a>).</p><p id="p-19">Population genetic structure was estimated using two methods, Bayesian clustering and principal coordinate analysis (PCoA). Bayesian clustering was performed in STRUCTURE v2.3.4 (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Pritchard <em>et al.</em>, 2000</a>) with the admixture model. As STRUCTURE analysis assumes that loci are unlinked, we thus filtered out linked SNPs using the <em>–write_single_snp</em> option in the POPULATIONS module (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Catchen <em>et al.</em>, 2013</a>). To determine the optimal number of groups (<em>K</em>), we ran STRUCTURE 10 times for each <em>K</em> value from 1 through 10. Each run was performed for 200,000 Markov Chain Monte Carlo (MCMC) generations with a burn-in period of 100,000 generations. The optimal <em>K</em> was chosen using both delta-<em>K</em> method implemented in STRUCTURE HARVESTER (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Earl &amp; vonHoldt, 2012</a>) and cross-entropy criterion implemented in R package <em>LEA</em> v2.8.0 (<em>snmf</em> function) (<a href="https://bioconductor.org/packages/LEA/">https://bioconductor.org/packages/LEA/</a>). The coefficient for cluster membership of each individual was averaged across the 10 independent runs using CLUMPP (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Jakobsson &amp; Rosenberg, 2007</a>) and plotted using DISTRUCT (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Rosenberg, 2004</a>). We performed PCoA analyses using the <em>dartR</em> v1.1.11 package (<em>gl.pcoa</em> function) in R. We tested the overall population structure using the G-statistic test (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Goudet <em>et al.</em>, 1996</a>) by <em>gstat.randtest</em> function in <em>adegenet</em> v2.1.1 package (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Jombart, 2008</a>). Additionally, an analysis of molecular variance (AMOVA) was performed to quantify genetic differentiation among and within populations and genetic groups (as identified by STRUCTURE and PCoA) using Arlequin v3.5 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Excoffier <em>et al.</em>, 2007</a>) with significance tests of variance components based on 10,000 permutations.</p></div><div id="sec-6" class="subsection"><h3>Population divergence history</h3><p id="p-20">We adopted a two-step strategy for investigating the divergent time of <em>C. agrestis</em>. First, we used plastome sequences assembled from Ranunculales and newly sampled <em>C. agrestis</em> populations to estimate the diversification of <em>C. agrestis</em>. A total of 47 plastomes were included (Table S1). The 79 protein coding regions (CDS) were extracted and aligned using MAFFT v7.313 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Katoh &amp; Standley, 2013</a>) and were then concatenated using PhyloSuite v1.1.15 (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Zhang <em>et al.</em>, 2019</a>). The program BEAST2 v2.5.2 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bouckaert <em>et al.</em>, 2014</a>) was employed for molecular dating with concatenated matrix using a GTR+G+I substitution model selected by jModelTest v2.0.1 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Darriba <em>et al.</em>, 2012</a>) under the Bayesian information criterion (BIC). A log-normal relaxed-clock mode and birth-death process were applied. According to fossil records, the time to most recent common ancestors (TMRCA) of Ranunculales and Berberidaceae were constrained to the minimum age of 112 Ma and 72 Ma, respectively. We set both calibrations as an exponential distribution (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Ho &amp; Phillips, 2009</a>) with a mean of 1 and an offset (hard bound constraint) that equaled the minimum age of the calibrations. The Markov Chain Monte Carlo (MCMC) run was set as 100 million generations with sampling every 1,000 generations. Tracer v1.7.1 (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Rambaut <em>et al.</em>, 2018</a>) was used to assess the effective sample size (ESS &gt; 200) of each parameter. A maximum clade credibility tree was built by TreeAnnotator v1.8.0 (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Rambaut &amp; Drummond, 2010</a>) with the initial 20% of trees discarded as burn-in.</p><p id="p-21">Second, we estimated divergent time between haplotypes of <em>C. agrestis</em> using the SNP sequences generated from RAD-seq. All the parameters of BEAST2 were kept the same as above, with the exception of the TMRCA of <em>C. agrestis</em> populations was constrained based on the previous results. To infer the biogeographic history, ancestral distributions of C. agrestis were estimated by statistical dispersal-extinction-cladogenesis (S-DEC) (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Ree &amp; Smith, 2008</a>), as implemented in Reconstruct Ancestral State in Phylogenies (RASP) v4.0 (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Yu <em>et al.</em>, 2015</a>), using 50,000 randomly sampled trees from posterior distribution generated by BEAST2. We defined the following five areas according to geographic distributions of <em>C. agrestis</em>: A, Qinling Mountain and adjacent regions; B, Northwestern Sichuan-South of Gansu; C, North of Qinghai; D, Hengduan Mountains; E, Himalayas. To detect potential demographic expansions, temporal changes in the effective population size (Ne) were inferred with Extended Bayesian skyline plots (EBSP) as implemented in BEAST2 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Heled &amp; Drummond, 2008</a>). We used the SNP sequences of 139 individuals and the default substitution model. The MCMC was set to 50 million generations with sampling every 5000 generations, and the first 20% was discarded as burn-in. Final graphs were produced with the custom R script provided by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Heled (2010)</a>, with a burn-in cutoff of 20%.</p></div><div id="sec-7" class="subsection"><h3>Effects of heterogeneous landscapes on genetic structure</h3><p id="p-22">To illustrate the effects of the heterogeneous landscapes on shaping genetic structure, we conducted Mantel tests for the presence of isolation-by-distance (IBD) and isolation-by-resistance (IBR). To avoid the potential correlation between geographical distance and resistance distance, a partial Mantel test was also conducted to test IBD and IBR by controlling the resistance and geographical distance, respectively. GenAlEx v6.5 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Peakall &amp; Smouse, 2012</a>) was employed to compute the pairwise geographic distance among 18 populations. Resistance distance was generated in CIRCUITSCAPE v4.0.5 based on circuit theory (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">McRae &amp; Beier, 2007</a>). Circuit resistance provide models of connectivity or resistance to predict patterns of dispersal among sites in a heterogeneous landscape (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dickson <em>et al.</em>, 2019</a>). The current ecological niche model (ENM) of environmental niche was calculated in MAXENT (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Phillips &amp; Dudik, 2008</a>). We used 19 bioclimatic variables available from the WorldClim2 database (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fick &amp; Hijmans, 2017</a>) at a 30 arc□seconds resolution (Table S1), and actual evapotranspiration (AET; <a href="http://www.physicalgeography.net/fundamentals/8j.html">http://www.physicalgeography.net/fundamentals/8j.html</a>) as environmental predictors. To avoid multicollinearity, we ran a Pearson correlation analysis to eliminate one of the variables in each pair with a correlation value higher than 0.9 paired. A total 12 climatic layers was retained for analyses (Table S1).</p><p id="p-23">We used environmental niches representing connectivity between populations as conductance grids to produce pairwise resistance distance, as high habitat suitability was assumed to have low resistance (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Nowakowski <em>et al.</em>, 2015</a>). Tests of the significance for the relationship between geographical/resistance distances and genetic distance among populations were implemented in the R package <em>ade4</em> v1.7 (<a href="https://CRAN.R-project.org/package=ade4">https://CRAN.R-project.org/package=ade4</a>) using <em>mantel.rtest</em> function with 9,999 permutations.</p></div><div id="sec-8" class="subsection"><h3>Environmental variables associated with genetic structure</h3><p id="p-24">To estimate the contributions of environmental variables to genetic differentiation and to understand the turnover of allele frequencies along an environmental gradient, a gradient forest (GF) analysis was performed using the R package <em>gradientForest</em> v0.1(<a href="http://gradientforest.r-forge.r-project.org/">http://gradientforest.r-forge.r-project.org/</a>). GF is a nonparametric, machine□learning regression tree approach that allows for exploration of nonlinear associations of spatial, environmental, and allelic variables (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gugger <em>et al.</em>, 2017</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bay <em>et al.</em>, 2018</a>). The analysis partitions the allele frequency data at split values along the environmental gradients and defines the amount of variation explained as ‘split importance’ values (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Jiang <em>et al.</em>, 2019</a>). The overall predictor importance plot of GF shows the mean importance weighted by allele <em>R</em><sup><em>2</em></sup>, and the cumulative plot for each predictor shows cumulative change along the environmental gradient. The split importance values are added cumulatively to produce a step-like curve, thus, areas with large steps in a row indicate significant influence on allelic change. The 12 climatic layers used in above analysis was employed as environmental variables in the GF analysis.</p><p id="p-25">A redundancy analysis (RDA) was performed to understand the associations between genetic structure and environmental variables. We estimated the proportion of genetic variance in the populations that is explained by environmental variables using six important variables identified by GF (bio03: isothermality; AET: actual evapotranspiration; bio04: temperature seasonality; bio07: temperature annual range; bio12: annual precipitation and bio15: seasonality precipitation, see results). In our analysis, we constrained the dependent variables (individuals) by the explanatory variables (climate). The RDA analysis was performed using the <em>rda</em> function in the <em>vegan</em> v2.5 package (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Oksanen <em>et al.</em>, 2018</a>) in R (<a href="http://CRAN.R-project.org/package=vegan">http://CRAN.R-project.org/package=vegan</a>). The <em>anova.cca</em> function was used to calculate overall significance of the RDA plot, and if this was found to be significant, then significance of each climate variable was calculated, using a significance tests with 9,999 permutations.</p></div><div id="sec-9" class="subsection"><h3>Genome□Environment Association (GEA) analysis</h3><p id="p-26">To access the evidence of divergent selection acting on the genome and the genetic signatures of local adaptation, tests of associated outlier loci with environmental variation were carried out using BAYESCENV v1.1 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">de Villemereuil &amp; Gaggiotti, 2015</a>), which extends the capabilities of the BAYESCAN algorithm by including a model that incorporates environmental data. BAYESCENV has been shown to be fairly robust to isolation by distance and a hierarchically structured scenario (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">de Villemereuil &amp; Gaggiotti, 2015</a>). For a comprehensive consideration of the environmental effect, the environmental covariable layer was calculated with principal component analysis (PCA) of the six important bioclimatic variables, and the first PC axe (PC1), which explained most of the variability (48.62%), was extracted as the input for BAYESCENV. We ran 20 pilot runs of 5,000 iterations and a burn□in of 50,000 iterations, ultimately obtaining 5,000 MCMC iterations for the analysis. Diagnostics of the log likelihoods and <em>F</em><sub>ST</sub> values for the 5,000 sampled iterations were checked using the R package <em>coda</em> v0.19 (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Plummer <em>et al.</em>, 2006</a>) to confirm convergence and sample sizes of at least 2,500.</p><p id="p-27">For functional annotation, the locus consensus sequences of all 6,120 SNPs that were initially generated by POPULATIONS were exported in <em>fasta</em> format. Candidate loci identified in GEA were annotated with Blast2Go v2 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gotz <em>et al.</em>, 2008</a>), which performs automated annotation augmentation by searching the non-redundant <em>Arabidopsis</em> protein database with records from the NCBI ESTs databases (Blastx). All the parameters were set as default with e-value &lt; 1.0E-5.</p></div></div><div class="section" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Sequence data quality and processing</h3><p id="p-28">We sequenced 139 individuals of <em>C. agrestis</em> using Illumina that produced a total of 133 gigabyte (Gb) data containing 845,396,236 raw reads (Table S2). After filtering out low quality reads, the average number of used reads per sample was 4,542,944 (median: 4,883,567; Table S2). The depths of coverage for processed samples were ranged from 5.77x (WLG-2) to 22.44x (HZ2-3), with a mean coverage of 11.99x (Table S2). We optimized the parameters for STACKS analysis using five representative samples and identified optimal parameters for our dataset to be <em>M</em> = <em>n</em> =3 (Fig. S1 and S2). There were 3,640,060 loci containing 2,519,268 variant sites after all samples <em>de novo</em> assembling. After strict filtering (<em>- r</em> = 1), we obtained 6,959 RAD loci containing 6,120 SNPs that were used for population genetic analyses. The mean genotyped sites per locus was 146.59bp (stderr 0.03). After a looser filtering (<em>- r</em> = 0.8), we obtained 24,247 RAD loci containing 22,915 variant sites. The sizes of 20 sequenced plastomes from 18 populations of <em>C. agrestis</em> were ranged from 150,979 bp (ZF) to 151,056 bp (SNJ) possessing the same 79 protein coding genes arranged in the same order (Table S3).</p></div><div id="sec-12" class="subsection"><h3>Genetic diversity and structure</h3><p id="p-29">We detected an extraordinary difference of genetic diversity at population level, especially for the nucleotide diversity <em>π</em>, which ranged from 0.0008 (HZ2) to 0.1646 (DDL). The expected heterozygosity (<em>H</em><sub>e</sub>) and observed heterozygosity (<em>H</em><sub>o</sub>) were 0.0007 (HZ2) - 0.1555 (DDL) and 0.0014 (HZ2) - 0.2650 (DDL; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), respectively. When a loose filtering strategy was employed, we obtained a similar genetic diversity pattern (Table S4). In total, 66 haplotypes from 139 individuals (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>; Table S5) were detected, and the haplotype diversity (<em>H</em><sub>d</sub>) was 0.9798. Genetic differentiation between YL and HZ3 is the highest (<em>F</em><sub>ST</sub>=0.9913), and that between ZG and DDL is the lowest (<em>F</em><sub>ST</sub>=0.0640; Table S6; Fig. S3a).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111346/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111346/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111346" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <p id="p-30" class="first-child">Summary of statistics calculated for the 6120 variant positions. n, number of genotype;, average nucleotide diversity; <em>H<sub>e</sub></em>, average expected heterozygosity per locus; <em>H<sub>o</sub></em>, average observed heterozygosity per locus; the inbreeding coefficients (<em>F</em><sub>IS</sub>). The statistics of six defined groups are bolded. E1, eastern group1; E2, eastern group2; W1, western group1; W2, western group2; W3, western group3; W4, western group4</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">We conducted a Bayesian clustering analysis to explore the genetic structure of <em>C. agrestis</em>. The delta-<em>K</em> method identified best-fit number for clustering (highest Δ<em>K</em> value) was two (Fig. S4a), representing the East and West clades. While the method based on the cross-entropy criterion indicated a better number of clusters was six (Fig. S4b). Considering high <em>F</em>ST among populations, we further subdivided two clades into two and four groups (E1, E2, W1, W2, W3 and W4), respectively, which was also supported by the PoCA results (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). The AMOVA revealed that 77.59% of the overall variation was distributed among six groups, with 11.27% explained by variation among populations within groups (Table S7). Within the genetic groups, W2 exhibited the highest nucleotide diversity (<em>π</em>; 0.1646), while W3 had the lowest (0.0206). All genetic groups had lower observed heterozygosity than expected, with the exception of W2. In general, the East clade has lower levels of observed heterozygosity (<em>H</em><sub>o</sub>, E1=0.0029, E2=0.0034, W1=0.0548, W2=0.2650, W3=0.0134, W4=0.0088) and higher levels of inbreeding coefficient (<em>F</em><sub>IS</sub>, E1=0.1952, E2=0.1011, W1=0.0384, W2=-0.1806, W3=0.0227, W4=0.0510) than the West clade (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Among genetic groups, genetic differentiation between W3 and W4 is the highest (<em>F</em><sub>ST</sub>=0.8908), followed by E2 and W4 (<em>F</em><sub>ST</sub>=0.8451), and that between E1 and E2 is lowest (<em>F</em><sub>ST</sub>=0.1284; Table S8; Fig. S3b). The G-statistic test suggested a significant population structure (<em>p</em> = 0.01**; Fig. S3c).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population genetic structure in Circaeaster agrestis based on 6,120 SNPs from (a) PCoA for all populations, east-clade and west-clade; and (b) STRUCTURE with K=2 (showing the highest &#x394;K; Fig. S4a) and 6 (representing the divergence within east and west clades)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-983153512" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Population genetic structure in Circaeaster agrestis based on 6,120 SNPs from (a) PCoA for all populations, east-clade and west-clade; and (b) STRUCTURE with K=2 (showing the highest &#x394;K; Fig. S4a) and 6 (representing the divergence within east and west clades).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F2.medium.gif" width="440" height="247"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F2.medium.gif" width="440" height="247"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2</span> <p id="p-32" class="first-child">Population genetic structure in Circaeaster agrestis based on 6,120 SNPs from (a) PCoA for all populations, east-clade and west-clade; and (b) STRUCTURE with K=2 (showing the highest ΔK; Fig. S4a) and 6 (representing the divergence within east and west clades).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Population divergence history</h3><p id="p-33">The 47-taxa 79-CDS matrix of Ranunculales used for estimating the original divergence of <em>C. agrestis</em> was 66,117 bp in length containing 12,998 parsimony-informative sites. The divergence of <em>C. agrestis</em> from its closely related <em>Kingdonia</em> was estimated to occur in the early Eocene (stem age, ca. 52.19 Ma; 95% highest posterior density [HPD] intervals, 26.12-83.13 Ma), whereas the early divergence within <em>C. agrestis</em> started in the late□Miocene (ca. 6.33 Ma; 95% HPD, 1.96-23.35 Ma) (Fig. S5). The tree topology inferred by BEAST2 using the SNP sequences generated from RAD-seq identified two main clades (East and West clades) and six sub-groups (E1, E2, W1, W2, W3 and W4 groups), which was consistent with STRUCTURE and PoCA results. The divergence of two main clades was estimated at appropriately 6.71 Ma (95% HPD, 5.18-7.89 Ma). The subsequent divergences between six groups occurred mainly during the Pliocene to late Pleistocene (ca. 5.05Ma-2.90Ma) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a>). Biogeographic analysis supported Hengduan Mountains and adjacent regions as the most likely ancestral region for C. agrestise (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3a</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">3b</a>). For the demographic inference, EBSP analysis showed no sign of population expansion or contraction in demographic history of <em>C. agrestis</em> (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3c</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolutionary history of Circaeaster agrestis. (a) Divergent time estimation and ancestral area reconstructions of 66 haplotypes based on SNPs sequences. Mean divergence dates and 95% HPDs for major nodes (1&#x2013;5) are summarized in top left. Pie charts on each node indicate marginal probabilities for each alternative ancestral area derived from S-DEC. Results are based on a maximum area number of two. Inferred dispersal and vicariance events are indicated by green and red circle respectively. (b) Five defined areas according to geographic distributions of C. agrestis: A, Qinling Mountains; B, Sichuan-South of Gansu; C, North of Qinghai; D, Hengduan Mountains; E, Himalayas. (c) Temporal changes in the effective population size (Ne) changes inferred with Extended Bayesian skyline plots (EBSP) using BEAST2. Re, reproductive number, &#x3B4;, death rate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-983153512" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Evolutionary history of Circaeaster agrestis. (a) Divergent time estimation and ancestral area reconstructions of 66 haplotypes based on SNPs sequences. Mean divergence dates and 95% HPDs for major nodes (1&#x2013;5) are summarized in top left. Pie charts on each node indicate marginal probabilities for each alternative ancestral area derived from S-DEC. Results are based on a maximum area number of two. Inferred dispersal and vicariance events are indicated by green and red circle respectively. (b) Five defined areas according to geographic distributions of C. agrestis: A, Qinling Mountains; B, Sichuan-South of Gansu; C, North of Qinghai; D, Hengduan Mountains; E, Himalayas. (c) Temporal changes in the effective population size (Ne) changes inferred with Extended Bayesian skyline plots (EBSP) using BEAST2. Re, reproductive number, &#x3B4;, death rate.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F3.medium.gif" width="440" height="428"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F3.medium.gif" width="440" height="428"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3</span> <p id="p-34" class="first-child">Evolutionary history of Circaeaster agrestis. (a) Divergent time estimation and ancestral area reconstructions of 66 haplotypes based on SNPs sequences. Mean divergence dates and 95% HPDs for major nodes (1–5) are summarized in top left. Pie charts on each node indicate marginal probabilities for each alternative ancestral area derived from S-DEC. Results are based on a maximum area number of two. Inferred dispersal and vicariance events are indicated by green and red circle respectively. (b) Five defined areas according to geographic distributions of C. agrestis: A, Qinling Mountains; B, Sichuan-South of Gansu; C, North of Qinghai; D, Hengduan Mountains; E, Himalayas. (c) Temporal changes in the effective population size (Ne) changes inferred with Extended Bayesian skyline plots (EBSP) using BEAST2. Re, reproductive number, δ, death rate.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Effects of topography and ecology</h3><p id="p-35">Given that the distribution range of <em>C. agrestis</em> is topographically complex, we evaluated the influence of the heterogeneous landscape in shaping genetic structure via Mantel test. Our analyses suggested both significant patterns of IBD (<em>R</em><sup>2</sup>= 0.443, <em>P</em> = 0.001**) and IBR (<em>R</em><sup>2</sup>= 0.315, <em>P</em> = 0.017*) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4a</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">b</a>), which were also supported by a partial Mantel test (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/2020.01.14.902643/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/2020.01.14.902643/embed/inline-graphic-1.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/2020.01.14.902643/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/2020.01.14.902643/embed/inline-graphic-2.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/2020.01.14.902643/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/2020.01.14.902643/embed/inline-graphic-3.gif"/></noscript></span></span>). In our case, geographical distance explained the genetic structure better than resistance distances based on current distribution of habitats. We further plotted the conductance grid derived from the IBR model to illustrate the connectivity among populations and thus represent the empirical costs of gene flow through a complex landscape. In general, connectivity gaps among populations were observed in the western clade, reflecting increased habitat isolation in this area resulted from complex topography. In contrast, locations of eastern clade had low resistance to dispersal thus high connectivity among populations (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4c</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Topographic and ecological effects on genetic structure. Relationship of genetic distance and (a) geographical distance, (b) resistance distance based on climatic niche suitability, as tested by a Mantel test. (c) Conductance grid derived from the Isolation&#x2010; by&#x2010; Resistance (IBR) model. Grid with warm colour indicates high conductance, cold colour indicates high resistance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-983153512" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Topographic and ecological effects on genetic structure. Relationship of genetic distance and (a) geographical distance, (b) resistance distance based on climatic niche suitability, as tested by a Mantel test. (c) Conductance grid derived from the Isolation&#x2010; by&#x2010; Resistance (IBR) model. Grid with warm colour indicates high conductance, cold colour indicates high resistance.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F4.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F4.medium.gif" width="440" height="430"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4</span> <p id="p-36" class="first-child">Topographic and ecological effects on genetic structure. Relationship of genetic distance and (a) geographical distance, (b) resistance distance based on climatic niche suitability, as tested by a Mantel test. (c) Conductance grid derived from the Isolation‐ by‐ Resistance (IBR) model. Grid with warm colour indicates high conductance, cold colour indicates high resistance.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Environmental variables associated with genetic structure</h3><p id="p-37">Among the 12 variables used for GF analysis, isothermality (bio03) was indicated as the most important predictor. Actual evapotranspiration (AET), temperature annual range (bio07) and temperature seasonality (bio04) were also of high importance. Seasonality precipitation (bio15), and annual precipitation (bio12) showed moderate importance to allele frequencies. The other six environmental variables had relatively lower contributions (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5a</a>). <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5b</a> showed the cumulative change in overall allele frequencies, where changes evolved with the environmental gradient.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(a) R2-weighted importance of environmental variables that explain genetic gradients from gradient forest (GF) analysis. (b) Cumulative importance of allelic change along the first six environmental gradients." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-983153512" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(a) R2-weighted importance of environmental variables that explain genetic gradients from gradient forest (GF) analysis. (b) Cumulative importance of allelic change along the first six environmental gradients.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F5.medium.gif" width="440" height="299"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F5.medium.gif" width="440" height="299"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5</span> <p id="p-38" class="first-child">(a) R2-weighted importance of environmental variables that explain genetic gradients from gradient forest (GF) analysis. (b) Cumulative importance of allelic change along the first six environmental gradients.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The RDA analysis revealed that a significant amount of genetic variation among populations is associated with the six important environmental variables (46.58%, <em>p</em> = 0.001). Each of the two axes explained a significant amount of variation (Axis 1: 58.55%, <em>P</em> = 0.001**; Axis 2: 27.31%, <em>P</em> = 0.001**; <a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6</a>). We also conducted RDA analysis using each six environmental variables separately and found that all six variables were significant. The contribution on genetic variation of each variable generally consistent with the GF analysis. Isothermality (bio03; 18.93%, <em>P</em> = 0.001**) was the most important predictor, whereas the temperature annual range (bio07) was less important in RDA analysis (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111343/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111343/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111343" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <p id="p-40" class="first-child">RDA results based on six important environmental variables identified by GF analysis.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Redundancy analysis showing the relationship between the independent climate parameters and population structure. Individuals are colored points and colors represent six groups (E1, E2, W1, W2, W3, W4). Small black points are SNPs. The percentage of explained variance for each axis is provided." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-983153512" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Redundancy analysis showing the relationship between the independent climate parameters and population structure. Individuals are colored points and colors represent six groups (E1, E2, W1, W2, W3, W4). Small black points are SNPs. The percentage of explained variance for each axis is provided.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F6.medium.gif" width="440" height="402"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 6" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F6.medium.gif" width="440" height="402"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/2020.01.14.902643/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 6</span> <p id="p-41" class="first-child">Redundancy analysis showing the relationship between the independent climate parameters and population structure. Individuals are colored points and colors represent six groups (E1, E2, W1, W2, W3, W4). Small black points are SNPs. The percentage of explained variance for each axis is provided.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Genome□Environment Association (GEA) analysis</h3><p id="p-42">The BAYESCENV result revealed a total of 61 loci have a significant association with environmental variables. We then aligned consensus sequences of these candidate loci for annotation. Sixteen of the 61 loci were successfully annotated by Blast2Go with cut-off of e-value &lt;1E-5 and a mean similarity more than 80 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>; Table S9). Among these were genes related to abiotic stress responses and stress-induced morphological adaptations, such as <em>UGT74E2</em> (Loci ID: 2763) regulating auxin homeostasis (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Tognetti <em>et al</em>., 2010</a>), <em>SRK2E</em> (Loci ID: 8038) involving in the resistance to drought by avoiding water loss (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Mustilli <em>et al.</em>, 2002</a>) and <em>AHK5</em> (Loci ID: 203056) regulating stomatal state and transmitting the stress signal (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pham <em>et al.</em>, 2012</a>), as well as genes required for the formation and development of leaves (<em>CYP71</em>, loci ID: 293398; <em>CSI1</em>, loci ID: 218068) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Li &amp; Luan, 2011</a>). These loci also included genes related to oxidation-reduction process involved in cellular respiration, as for example, <em>NDUFS7</em> (Loci ID: 83859) and <em>NDUFB3</em> (Loci ID: 219291); and genes implicated in the defense of plants against pathogens, such as Glucan endo-1,3-beta-glucosidase B (Loci ID: 296907), <em>RPPL1</em> (Loci ID:146932) and <em>NUDT8</em> (Loci ID: 243120) (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Stroud <em>et al.</em>, 2016</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111349/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111349/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111349" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <p id="p-43" class="first-child">Annotation information of 16 candidate loci under selection that the E-value of blastx is less than 1E-5. Detailed functional annotation is provided in Table S9.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-17"><h2 class="">Discussion</h2><div id="sec-18" class="subsection"><h3>Effects of topography on genetic structure</h3><p id="p-44">Our study presents the genetic structure of the palaeoendemic alpine <em>C. agrestis</em>. High haplotype diversity, uneven distribution of genetic diversity, as well as significant IBD and IBR patterns suggest restricted gene flow is likely a key factor in the high genetic differentiation among <em>C. agrestis</em> populations. The distribution area of <em>C. agrestis</em> is remarkable for its extraordinarily topographic diversity with some of the world’s most rugged mountain ranges and altitudinal gradients spanning over 5,000 m (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Favre <em>et al.</em>, 2015</a>). As a direct consequence of the uplift of the QTP, habitat fragmentation induced by complex topography and frequented climatic fluctuation was invoked as an important mechanism for the high diversity of alpine plants in this area (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Wen <em>et al.</em>, 2014</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Sun, H <em>et al.</em>, 2017</a>). Hence, dramatically topographic and climatic changes triggered by mountain uplift probably have influenced genetic connectivity and shaped current genetic structure of <em>C. agrestis</em>.</p><p id="p-45">The high IBR pattern of western clade, reflecting the habitat isolation and perhaps caused by complex topography, may be the cause for limited gene flow as well as observed high differentiation. The Himalayan orogeny likely triggered the vicariance among different <em>C. agrestis</em> groups. The north-south mountain chains and valleys may provide corridors for flora exchange between the north and south, but represent barriers to migration between the east and west (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em>, 2017</a>); this likely resulted in the observed meridional differentiation of <em>C. agrestis</em> in the western clade. Previous studies (e.g., <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gao <em>et al.</em>, 2007</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Li <em>et al.</em>, 2011</a>) have also suggested phylogeographic isolation associated with mountains and valleys with north-south direction. Notably, the populations located in the Hengduan Mountains (HM), exhibit high levels of genetic diversity (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). As the QTP blocks the cold and dry air toward HM (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Liu <em>et al.</em>, 2013</a>), the HM region is characterized by a much warmer and wetter climate, providing a comparatively stable environment for species. Although the western individuals are likely to possess higher genetic diversity, they are also likely to be more vulnerable to shifting disturbance regimes and environmental changes due to lack of genetic connectivity resulted from habitats isolation.</p><p id="p-46">Compared to the western clade, north-south mountain barriers among different latitudes may be the reason for the divergence among the two groups in the eastern clade. A conductance grid derived from the IBR model showed low resistance to dispersal, thus high connectivity, among populations in the eastern clade. In addition, E1 and E2 exhibit a relatively low level of genetic differentiation. This phenomenon is likely due to the similar local environments among different populations resulting from the quite moderate topographical and climatic characteristics in this region (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chen <em>et al.</em>, 2017</a>). We detected a significantly high <em>F</em><sub>IS</sub> values combined with low level of observed heterozygosity in the eastern clade (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), indicating high levels of inbreeding, which may attribute to disturbance caused by human activities or the possible bottleneck events (Rands et al., 2010). There is consensus that breeding between related individuals can lead to a decrease in heterozygosity and fixation of deleterious alleles that may reduce reproductive fitness and increase the population susceptibility to environmental pressure (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kramer &amp; Havens, 2009</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Poudel <em>et al.</em>, 2014</a>), making populations vulnerable to cope with rapid climate changes. Notably, the highest <em>F</em><sub>IS</sub> value was detected in WLG (<em>F</em><sub>IS</sub>: 0.0621), indicating a key protected population for <em>C. agrestis</em> populations in this region.</p></div><div id="sec-19" class="subsection"><h3>Impact of environmental heterogeneity in population divergence</h3><p id="p-47">Our molecular dating based on plastome sequences suggested an early Eocene origin of <em>C. agrestis</em>, which is accordant with the previous estimation (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Ruiz-Sanchez <em>et al.</em>, 2012</a>) and our results from nuclear data (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Sun <em>et al.</em>, 2020</a>). The diversification of <em>C. agrestis</em> has increased since the Pliocene (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3</a>), which is likely related with this period of intense uplift of the HM during the late Miocene to late Pliocene (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Sun <em>et al.</em>, 2011</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Favre <em>et al.</em>, 2015</a>). The environmental and climatic fluctuation triggered by mountain uplifts can always led to divergent selection and species adaptation associated with dramatic ecological niches changes (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Ren <em>et al.</em>, 2017</a>). Orogenic activities not only provided numerous new habitats favoring for diversification, but also promoted population divergence due to the hindered gene flow across a continuous altitudinal difference. Our biogeographic analysis revealed that the demographic evolution of <em>C. agrestis</em> has not simply been influenced by the orogeny at a particular time in history, but is more complex, involving repeated episodes of dispersal and subsequent vicariance events over multiple timescales. Given the EBSP analysis showing a relatively stable population size in demographic history of <em>C. agrestis</em>, geographical events induced environmental and climatic fluctuation that promotes population divergence, while likely had a small effect on demographic changes. Coupled with the evidence of significant IBD and IBR patterns, we speculate natural selection driven by extraordinary environmental heterogeneity and geomorphological dynamics may have profound impact in the diversification of <em>C. agrestis</em>. In addition, during our field work we found population sizes of <em>C. agrestis</em> in the Himalaya-Hengduan Mountains to be much smaller than other sites. Previous studies have indicated a significant negative effect of small population size on adaptive evolution (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Nevado <em>et al.</em>, 2019</a>). Hence, we speculate that natural selection may play double-edged roles in the evolutionary history of any given species. On one hand, heterogeneous environments could provide more new ecological niches conducive for speciation, on the other hand, it may put the populations with weak adaptive ability to new environment in danger of extinction.</p><p id="p-48">Reproductive isolation induced by geographical distance is likely to be another important factor driving the population divergence of <em>C. agrestis.</em> The significant population structure from AMOVA analysis, along with high level of <em>F</em><sub>ST</sub> values, and small population numbers discovered in the field commonly reflect there is a potential for reproductive isolation among different <em>C. agrestis</em> populations, especially in the western clade. For populations with small size, deleterious mutations and compensatory adaptive mutations can be quickly and independently fixed, causing reproductive isolation coupled with high level of genetic differentiation (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Nielsen, 2005</a>). However, <em>C. agrestis</em> exhibits low morphological diversity and no sign of expanding population demographic history. We speculate that strong environmental selective pressures are constraining morphological diversity while mutations are leading to observable population differentiation. Nonetheless, cryptic speciation cannot be ruled out, which needs further crossing experiments or/and testing for ecological niche divergence/overlap. Furthermore, significant IBR pattern suggest that genetic structure would be better explained by habitat isolation between populations than just by geographical distance alone. Therefore, habitat changes may have large effect on genetic structure for <em>C. agrestis</em>, indicating its sensitivity to environmental niche.</p><p id="p-49">Studying local adaptation contributes to understanding the ability of populations to sustain or adapt to rapid climate change (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Jia <em>et al.</em>, 2019</a>). We associated environmental variables with genetic structure to demonstrate the impact of environmental heterogeneity on population divergence. GF and RDA analyses indicated that covariation between genetic structure and environmental variables have played a role in promoting the diversification of <em>C. agrestis</em>. Specifically, both analyses found a significant association between genetic variation and temperature variables (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4</a>; bio03, AET, bio04, bio07), suggesting temperature is potentially an important driver of genetic variation within <em>C. agrestis</em>, as detected in previous studies of angiosperm trees (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Ahrens <em>et al.</em>, 2019</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Jia <em>et al.</em>, 2019</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Jiang <em>et al.</em>, 2019</a>). In fact, there is growing evidence that high-mountain environments experience more rapid changes in temperature than environments at lower elevations (Pepin et al., 2015; Palazzi, Mortarini, Terzago, &amp; von Hardenberg, 2019). Despite that, little attention has been given to understand the mechanisms of adaptive response to climate changes in high altitude areas. <em>C. agrestis</em> is habitually confined to grow under trees or shrubs, indicating it is sensitive to temperature. Moreover, the vertical distribution of <em>C. agrestis</em> is extensively wide ranging from 2,100 to 5,000 meters above sea level (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Fu &amp; Bartholomew, 2001</a>), exhibiting a considerable differentiation in temperature. Therefore, it is reasonable that temperature variables play a vital role in promoting genetic divergence within <em>C. agrestis.</em> Furthermore, given the ongoing climate warming, temperature is likely still to be a key driver for adaptation of <em>C. agrestis</em> in future.</p></div><div id="sec-20" class="subsection"><h3>Genomic signatures associated with local adaptation</h3><p id="p-50">Simulation studies showed that the GEA method, which incorporates environmental information in the form of “environmental differentiation”, has a much lower false positive rate than genome-scan methods, and is robust to IBD pattern as well as a hierarchically structured scenario (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">de Villemereuil &amp; Gaggiotti, 2015</a>). In the present study, we first employed PCA to calculate the overall environmental differentiation and provide the first axis (PC1) to BAYESCENV as a description of the environmental distance between populations. Considering that all environmental pressures drive species divergence together, this approach could provide a realistically and comprehensively environmental gradient for studying local adaptation.</p><p id="p-51">In our analysis, BAYESCENV identified a set of genes that may be involved in a continuously adaptive process. Some genes as pivotal kinases involved in signaling pathway regulation, may induce abiotic stress responses for defending harsh climate in QTP. For example, genes associated with abscisic acid (ABA) signaling pathway (<em>SRK2EA</em>) can regulate numerous ABA responses, such as stomata closure, to response to high altitudes with extreme difference in temperature (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Sierla <em>et al.</em>, 2018</a>). Considering the tiny leaves of <em>C. agrestis</em> and their sensitivity to temperature, we speculate that stomatal regulation would be a vital process for maintaining hydration and regulating temperature. As altitude increases, hypobaric hypoxia becomes the main factor interfering with life activity. Genes related to oxidation-reduction processes involved in cellular respiration, such as <em>NDUFS7</em> and <em>NDUFB3</em>, are of importance in facilitating adaptation of <em>C. agrestis</em> to high altitude areas of hypoxia. In addition, some putatively adaptive genes (<em>CYP71</em> and <em>CSI1</em>) are associated with physiological trait regulation of vegetative and reproductive organs (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gu <em>et al.</em>, 2010</a>). Thus, morphological adaptations may be an essential alternative for <em>C. agrestis</em> to respond to environmental stress.</p><p id="p-52">Despite limited available gene information in this study, it should be noted that genetic patterns of adaptation to environmental heterogeneity are likely driven by a wide range of alleles of small-effect for complex traits (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Savolainen <em>et al.</em>, 2013</a>). The physiological processes involved in the adaptation of <em>C. agrestis</em> to alpine environments are most likely complex traits determined by multiple genes with potentially adaptive mechanisms including regulation of expression and epigenetic alteration (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gomulkiewicz <em>et al.</em>, 2010</a>; <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Ahrens <em>et al.</em>, 2019</a>). The genes identified as putatively adaptive signals in <em>C. agrestis</em> are most likely a part of the broader process of adaptation to climate and should constitute the first step in studying genetic mechanism of local adaptation in alpine environments. Further work with characterizing whole genome could further illuminate patterns of functional genes controlling the response to climate changes. Generally, these patterns can serve as a proxy for understanding and monitoring the adaptive process in rapidly changing environment (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Ahrens <em>et al.</em>, 2019</a>) and inform future management strategies.</p></div></div><div class="section" id="sec-21"><h2 class="">Conclusions</h2><p id="p-53">In this study, we demonstrate two major advances. First, we provide new insights into the genetic structure and evolutionary history of a relictual alpine herb surviving in heterogeneous environment, which can inform further conservation. Our results shed light on vital roles of mountain uplift in promoting population diversification and the dual effect of environmental heterogeneity on species evolutionary history. Second, our analyses provide a roadmap to study evolution of local adaptation of non-model species for which common gardens and genomic resources are limited. By conducting multiple analyses, we associated environmental variables with genetic variation and explicitly measured the relative influence of bioclimate on population divergence. In turn, the important environmental predictors were implemented in identifying potential loci that may under divergent selection to gain in-depth knowledge of genetic basis involved in local adaptation. More importantly, our results improve our understanding of drivers and mechanisms of adaptive evolution, aiding efforts on developing sound conservation programs in facing of rapid changing climate.</p></div><div class="section" id="sec-23"><h2 class="">Author contributions</h2><p id="p-55">HCW, HS, and YXS developed the idea and designed the experiment. JWZ, LSY, NL, HJZ, RG, LJL, YHZ and TD collected the leaf materials. XZ and YXS performed the statistical analyses; XZ, YXS, JBL, HS and HCW interpreted the results and wrote the manuscript. All authors read, edited and approved the final manuscript. XZ and YXS contributed equally to this work.</p></div><div class="section" id="sec-24"><h2 class="">Data accessibility</h2><p id="p-56">All newly sequenced plastomes were deposited in National Center for Biotechnology Information (NCBI) with accession numbers xxx-xxx. RAD raw read data are stored at the NCBI Sequence Read Archive in the Bioproject xxx. Genomic data sets and R scripts used in this study are available in the DRYAD archives under accession xxx.</p></div><div class="section" id="sec-25"><h2 class="">Supporting information</h2><p id="p-57"><strong>Fig. S1</strong> The number of 80% polymorphic loci shared across most samples (the r80 loci) as M=n increases.</p><p id="p-58"><strong>Fig. S2</strong> The distribution of the number of SNPs per locus for a range of M=n values.</p><p id="p-59"><strong>Fig. S3</strong> (a) <em>F</em><sub>ST</sub> values within populations estimated by <em>hierfstat,</em> Group names of populations were labeled in the top of boxes. (b) <em>F</em><sub>ST</sub> values within genetic groups estimated by <em>hierfstat.</em> (c) The results of G-statistic test.</p><p id="p-60"><strong>Fig. S4</strong> The optimal <em>K</em> value identified using (a) delta-<em>K</em> method in STRUCTURE HARVESTER, and (b) cross-entropy criterion in R package <em>LEA.</em></p><p id="p-61"><strong>Fig. S5</strong> Divergence times of Ranunculales estimated by BEAST2 with a relaxed molecular clock based on the combined protein-coding region sequences. A and B indicate fossil calibration points. C and D indicate the origination and diversification of Circaeaster agrestis, respectively. Median ages of nodes are shown with bars indicating the 95% highest posterior density intervals for each node.</p><p id="p-62"><strong>Table S1</strong> Summary of population information of Circaeaster agrestis, including the number of individuals genotyped at each localities (n), geographic information,voucher specimens information, and 20 bioclimate values (bio01-bio19 and AET) of each location. AET, the actual evapotranspiration.</p><p id="p-63"><strong>Table S2</strong> Numbers of reads of RAD-seq for each individual of Circaeaster agrestis.</p><p id="p-64"><strong>Table S3</strong> Assemble and annotation information of newly sequenced plastomes of Circaeaster agrestis populations.</p><p id="p-65"><strong>Table S4</strong> Summary of statistics calculated for the 22915 variant positions. n, number of genotype;, average nucleotide diversity; He, average expected heterozygosity per locus; Ho, average observed heterozygosity per locus; the inbreeding coefficients (FIS). The statistics of six defined groups are bolded. E1, eastern group1; E2, eastern group2; W1, western group1; W2, western group2; W3, western group3; W4, western group4.</p><p id="p-66"><strong>Table S5</strong> Haplotypes information detected in 18 populations based on 6120 SNPs.</p><p id="p-67"><strong>Table S6</strong> Paired population‐level FST estimated by hierfstat using Weir and Cockerham’s method.</p><p id="p-68"><strong>Table S7</strong> The analysis of molecular variance (AMOVA) for SNP data among six genetic groups (E1, E2, W1, W2, W3 and W4)</p><p id="p-69"><strong>Table S8</strong> Paired group‐level FST estimated by hierfstat using Weir and Cockerham’s method.</p><p id="p-70"><strong>Table S9</strong> Detailed annotation information of sixteen genes under potential divergent selection.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">We are grateful to Jie Cai for help with sample collection in Tibet. This work was supported by the Strategic Priority Research Program of Chinese Academy of Sciences (XDA20050203), the Programme Foundation for the Backbone of Scientific Research by Wuhan Botanical Garden, Chinese Academy of Sciences (Y855241G01), the Major Program of National Natural Science Foundation of China (31590823), and the National Key R and D Program of China (2017YFC0505200).</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCE</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.1" data-doi="10.1111/mec.15092"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahrens</span>  <span class="cit-name-given-names">CW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rymer</span>  <span class="cit-name-given-names">PD</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Standing genomic variation within coding and regulatory regions contributes to the adaptive capacity to climate in a foundation tree species</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">28</span> (<span class="cit-issue">10</span>): <span class="cit-fpage">2502</span>–<span class="cit-lpage">2516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D28%26rft.spage%253D2502%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.15092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.15092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.2" data-doi="10.1111/j.1752-4571.2007.00013.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Curtis-McLane</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Adaptation, migration or extirpation: climate change outcomes for tree populations</span>. <abbr class="cit-jnl-abbrev">Evol Appl</abbr> <span class="cit-vol">1</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">95</span>–<span class="cit-lpage">111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BAppl%26rft.volume%253D1%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2007.00013.x%26rft_id%253Dinfo%253Apmid%252F25567494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2007.00013.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25567494&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262783400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.3" data-doi="10.1016/j.tig.2011.04.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Evolutionary genetics of plant adaptation</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr> <span class="cit-vol">27</span> (<span class="cit-issue">7</span>): <span class="cit-fpage">258</span>–<span class="cit-lpage">266</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DAnderson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D27%26rft.issue%253D7%26rft.spage%253D258%26rft.epage%253D266%26rft.atitle%253DEvolutionary%2Bgenetics%2Bof%2Bplant%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.04.001%26rft_id%253Dinfo%253Apmid%252F21550682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.04.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21550682&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292413300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.4" data-doi="10.1126/science.aan4380"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bay</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrigan</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Underwood</span>  <span class="cit-name-given-names">VL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">HL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">TB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruegg</span>  <span class="cit-name-given-names">K</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Genomic signals of selection predict climate-driven population declines in a migratory bird</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">359</span> (<span class="cit-issue">6371</span>): <span class="cit-fpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBay%26rft.auinit1%253DR.%2BA.%26rft.volume%253D359%26rft.issue%253D6371%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DGenomic%2Bsignals%2Bof%2Bselection%2Bpredict%2Bclimate-driven%2Bpopulation%2Bdeclines%2Bin%2Ba%2Bmigratory%2Bbird%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aan4380%26rft_id%253Dinfo%253Apmid%252F29302012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNTkvNjM3MS84MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2LzIwMjAuMDEuMTQuOTAyNjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.5" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span> (<span class="cit-issue">15</span>): <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.6" data-doi="10.1371/journal.pcbi.1003537"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhnert</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">BEAST 2: a software platform for Bayesian evolutionary analysis</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-vol">10</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">e1003537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D10%26rft.spage%253De1003537%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003537%26rft_id%253Dinfo%253Apmid%252F24722319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24722319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.7" data-doi="10.1111/mec.12354"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">WA</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Stacks: an analysis tool set for population genomics</span>. <abbr class="cit-jnl-abbrev">Molecular ecology</abbr> <span class="cit-vol">22</span> (<span class="cit-issue">11</span>): <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Becology%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y-S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Is the East Asian flora ancient or not?</span> <abbr class="cit-jnl-abbrev">National Science Review</abbr> <span class="cit-vol">5</span> (<span class="cit-issue">6</span>): <span class="cit-fpage">920</span>–<span class="cit-lpage">932</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNational%2BScience%2BReview%26rft.volume%253D5%26rft.spage%253D920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.9" data-doi="10.1126/science.1242121"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Colautti</span>  <span class="cit-name-given-names">RI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">SCH</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Rapid Adaptation to Climate Facilitates Range Expansion of an Invasive Plant</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span> (<span class="cit-issue">6156</span>): <span class="cit-fpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColautti%26rft.auinit1%253DR.%2BI.%26rft.volume%253D342%26rft.issue%253D6156%26rft.spage%253D364%26rft.epage%253D366%26rft.atitle%253DRapid%2BAdaptation%2Bto%2BClimate%2BFacilitates%2BRange%2BExpansion%2Bof%2Ban%2BInvasive%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1242121%26rft_id%253Dinfo%253Apmid%252F24136968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE1Ni8zNjQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE0LjkwMjY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.10" data-doi="10.1111/j.1749-6632.2009.04575.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Conover</span>  <span class="cit-name-given-names">DO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duffy</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hice</span>  <span class="cit-name-given-names">LA</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">The Covariance between Genetic and Environmental Influences across Ecological Gradients</span>. <abbr class="cit-jnl-abbrev">Annals of the New York Academy of Sciences</abbr> <span class="cit-vol">1168</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">100</span>–<span class="cit-lpage">129</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DConover%26rft.auinit1%253DD.%2BO.%26rft.volume%253D1168%26rft.spage%253D100%26rft.epage%253D129%26rft.atitle%253DThe%2Bcovariance%2Bbetween%2Bgenetic%2Band%2Benvironmental%2Binfluences%2Bacross%2Becological%2Bgradients%253A%2Breassessing%2Bthe%2Bevolutionary%2Bsignificance%2Bof%2Bcountergradient%2Band%2Bcogradient%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2009.04575.x%26rft_id%253Dinfo%253Apmid%252F19566705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2009.04575.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19566705&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268507900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.11" data-doi="10.1534/genetics.110.114819"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Witonsky</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Rienzo</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Using Environmental Correlations to Identify Loci Underlying Local Adaptation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">1411</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.114819%26rft_id%253Dinfo%253Apmid%252F20516501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS80LzE0MTEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE0LjkwMjY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.14.902643v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">HA</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-source">Speciation</span>. <span class="cit-publ-loc">Sunderland, MA, USA</span>: <span class="cit-publ-name">Sinauer Associates</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.13" data-doi="10.1038/nmeth.2109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darriba</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">GL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doallo</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">jModelTest 2: more models, new heuristics and parallel computing</span>. <abbr class="cit-jnl-abbrev">Nat Methods</abbr> <span class="cit-vol">9</span> (<span class="cit-issue">8</span>): <span class="cit-fpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPatel%26rft.auinit1%253DV.%2BR.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D772%26rft.epage%253D772%26rft.atitle%253DjModelTest%2B2%253A%2Bmore%2Bmodels%252C%2Bnew%2Bheuristics%2Band%2Bparallel%2Bcomputing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2109%26rft_id%253Dinfo%253Apmid%252F22847109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22847109&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.14.902643v2.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darwin</span>  <span class="cit-name-given-names">C</span></span>. <span class="cit-pub-date">1859</span>. <span class="cit-source">On the origin of species by means of natural selection, or the preservation of favoured races in the struggle for life</span>. <span class="cit-publ-loc">London, UK</span>: <span class="cit-publ-name">John Murray</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Villemereuil</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">OE</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">A new FST-based method to uncover local adaptation using environmental variables</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">6</span> (<span class="cit-issue">11</span>): <span class="cit-fpage">1248</span>–<span class="cit-lpage">1258</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D6%26rft.spage%253D1248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">BG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albano</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anantharaman</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beier</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fargione</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graves</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lawler</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">PB</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Circuit-theory applications to connectivity science and conservation</span>. <abbr class="cit-jnl-abbrev">Conservation Biology</abbr> <span class="cit-vol">33</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">239</span>–<span class="cit-lpage">249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BBiology%26rft.volume%253D33%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Earl</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">vonHoldt</span>  <span class="cit-name-given-names">BM</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method</span>. <abbr class="cit-jnl-abbrev">Conservation Genetics Resources</abbr> <span class="cit-vol">4</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">359</span>–<span class="cit-lpage">361</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BGenetics%2BResources%26rft.volume%253D4%26rft.spage%253D359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Genome sequencing and population genomics in non-model organisms</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">29</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">51</span>–<span class="cit-lpage">63</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D29%26rft.spage%253D51%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laval</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Arlequin (version 3.0): an integrated software package for population genetics data analysis</span>. <abbr class="cit-jnl-abbrev">Evolutionary bioinformatics online</abbr> <span class="cit-vol">1</span> : <span class="cit-fpage">47</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2Bbioinformatics%2Bonline%26rft.volume%253D1%26rft.spage%253D47%26rft_id%253Dinfo%253Apmid%252F19325852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19325852&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000207065900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.20" data-doi="10.1111/brv.12107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Favre</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Packert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pauls</span>  <span class="cit-name-given-names">SU</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jahnig</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uhl</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michalak</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muellner-Riehl</span>  <span class="cit-name-given-names">AN</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">The role of the uplift of the Qinghai-Tibetan Plateau for the evolution of Tibetan biotas</span>. <abbr class="cit-jnl-abbrev">Biol Rev Camb Philos Soc</abbr> <span class="cit-vol">90</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">236</span>–<span class="cit-lpage">253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol%2BRev%2BCamb%2BPhilos%2BSoc%26rft.volume%253D90%26rft.spage%253D236%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fbrv.12107%26rft_id%253Dinfo%253Apmid%252F24784793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/brv.12107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24784793&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.21" data-doi="https://doi.org/10.1002/joc.5086"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fick</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">RJ</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">WorldClim 2: new 1-km spatial resolution climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr> <span class="cit-vol">37</span> (<span class="cit-issue">12</span>): <span class="cit-fpage">4302</span>–<span class="cit-lpage">4315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D37%26rft.spage%253D4302%26rft_id%253Dinfo%253Adoi%252Fhttps%253A%252F%252Fdoi.org%252F10.1002%252Fjoc.5086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=https://doi.org/10.1002/joc.5086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.22" data-doi="10.1111 /mec. 14584"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">HH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">DL</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Comparing methods for detecting multilocus adaptation with multivariate genotype–environment associations</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">27</span> (<span class="cit-issue">9</span>): <span class="cit-fpage">2215</span>–<span class="cit-lpage">2233</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D27%26rft.spage%253D2215%26rft_id%253Dinfo%253Adoi%252F10.1111%2B%252Fmec.%2B14584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111 /mec. 14584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foster</span>  <span class="cit-name-given-names">AS</span></span>. <span class="cit-pub-date">1971</span>. <span class="cit-article-title">Additional studies on the morphology of blind vein-endings in the leaf of circaeaster agrestis</span>. <abbr class="cit-jnl-abbrev">American Journal of Botany</abbr> <span class="cit-vol">58</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">263</span>–<span class="cit-lpage">272</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.volume%253D58%26rft.spage%253D263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.24" data-doi="10.1111/mec.14946"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Friis</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fandos</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zellmer</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCormack</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milá</span>  <span class="cit-name-given-names">B</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Genome-wide signals of drift and local adaptation during rapid lineage divergence in a songbird</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">24</span>): <span class="cit-fpage">5137</span>–<span class="cit-lpage">5153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D27%26rft.spage%253D5137%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14946&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.14.902643v2.25"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">ZY</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Raven</span>,  <span class="cit-name-given-names">P. H.</span></span> &amp; </li><li><span class="cit-ed"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">D. Y.</span></span> ed</li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bartholomew</span>  <span class="cit-name-given-names">B</span></span> <span class="cit-pub-date">2001</span>. <span class="cit-chapter-title">CIRCAEASTERACEAE</span>. In: <span class="cit-ed"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">ZY</span></span>, <span class="cit-ed"><span class="cit-name-surname">Raven</span>,  <span class="cit-name-given-names">P. H.</span></span> &amp; <span class="cit-ed"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">D. Y.</span></span> ed. <span class="cit-source">Flora of China</span>. <span class="cit-publ-loc">Beijing &amp; St. Louis</span>: <span class="cit-publ-name">Science Press &amp; Missouri Botanical Garden Press</span>, <span class="cit-fpage">439</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.26" data-doi="10.1111/j.1365-294X.2007.03537.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">MÖLler</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">XM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mill</span>  <span class="cit-name-given-names">RR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibby</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">DZ</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">High variation and strong phylogeographic pattern among cpDNA haplotypes in <em>Taxus wallichiana</em> (Taxaceae) in China and North Vietnam</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">16</span> (<span class="cit-issue">22</span>): <span class="cit-fpage">4684</span>–<span class="cit-lpage">4698</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGao%26rft.auinit1%253DL.%2BM.%26rft.volume%253D16%26rft.issue%253D22%26rft.spage%253D4684%26rft.epage%253D4698%26rft.atitle%253DHigh%2Bvariation%2Band%2Bstrong%2Bphylogeographic%2Bpattern%2Bamong%2BcpDNA%2Bhaplotypes%2Bin%2BTaxus%2Bwallichiana%2B%2528Taxaceae%2529%2Bin%2BChina%2Band%2BNorth%2BVietnam.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03537.x%26rft_id%253Dinfo%253Apmid%252F17908214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03537.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17908214&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251028200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goicoechea</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guillardín</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernández-Ibarrodo</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valbuena-Carabaña</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">González-Martínez</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alía</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Adaptive Introgression Promotes Fast Adaptation In Oaks Marginal Populations</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: <span class="cit-fpage">731919</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gomulkiewicz</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barfield</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nuismer</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Genetics, adaptation, and invasion in harsh environments</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <span class="cit-vol">3</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">97</span>–<span class="cit-lpage">108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D3%26rft.spage%253D97%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.29" data-doi="10.1093/nar/gkn176"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gotz</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia-Gomez</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Terol</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">TD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagaraj</span>  <span class="cit-name-given-names">SH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nueda</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robles</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Talon</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dopazo</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conesa</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">High-throughput functional annotation and data mining with the Blast2GO suite</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">36</span> (<span class="cit-issue">10</span>): <span class="cit-fpage">3420</span>–<span class="cit-lpage">3435</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn176%26rft_id%253Dinfo%253Apmid%252F18445632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18445632&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257183200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Meeüs</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rousset</span>  <span class="cit-name-given-names">F</span></span>. <span class="cit-pub-date">1996</span>. <span class="cit-article-title">Testing Differentiation in Diploid Populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">1933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGoudet%26rft.auinit1%253DJ.%26rft.volume%253D144%26rft.issue%253D4%26rft.spage%253D1933%26rft.epage%253D1940%26rft.atitle%253DTesting%2BDifferentiation%2Bin%2BDiploid%2BPopulations%26rft_id%253Dinfo%253Apmid%252F8978076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NC80LzE5MzMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE0LjkwMjY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.31" data-doi="10.1073/pnas.1007092107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaplinsky</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bringmann</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cobb</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sampathkumar</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baskin</span>  <span class="cit-name-given-names">TI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Persson</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Somerville</span>  <span class="cit-name-given-names">CR</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Identification of a cellulose synthase-associated protein required for cellulose biosynthesis</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">107</span> (<span class="cit-issue">29</span>): <span class="cit-fpage">12866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1007092107%26rft_id%253Dinfo%253Apmid%252F20616083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzI5LzEyODY2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNC45MDI2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gugger</span>  <span class="cit-name-given-names">PF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">VL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodgskiss</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">JW</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Applying landscape genomic tools to forest management and restoration of Hawaiian koa (<em>Acacia koa</em>) in a changing environment</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <span class="cit-vol">11</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">231</span>–<span class="cit-lpage">242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D11%26rft.spage%253D231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Extended Bayesian Skyline Plots Tutorial</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Bayesian inference of population size history from multiple loci</span>. <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr> <span class="cit-vol">8</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D8%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.35" data-doi="10.1093/sysbio/syp035"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">SYW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">MJ</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Accounting for Calibration Uncertainty in Phylogenetic Estimation of Evolutionary Divergence Times</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr> <span class="cit-vol">58</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">367</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyp035%26rft_id%253Dinfo%253Apmid%252F20525591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syp035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525591&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270004400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.36" data-doi="10.1086/688018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoban</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antolin</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradburd</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poss</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">LK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Storfer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Finding the Genomic Basis of Local Adaptation: Pitfalls, Practical Solutions, and Future Directions</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">188</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">379</span>–<span class="cit-lpage">397</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D188%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1086%252F688018%26rft_id%253Dinfo%253Apmid%252F27622873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/688018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27622873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.37" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">NA</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span> (<span class="cit-issue">14</span>): <span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485429&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249248300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">K-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">X-G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S-S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">S-Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">El-Kassaby</span>  <span class="cit-name-given-names">YA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X-R</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Landscape genomics predicts climate change-related genetic offset for the widespread <em>Platycladus orientalis</em> (Cupressaceae)</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <a href="https://doi.org/10.1111/eva.12891">https://doi.org/10.1111/eva.12891</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">X-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">EM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">H-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">G-B</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Land bridges in the Pleistocene contributed to flora assembly on the continental islands of South China: Insights from the evolutionary history of <em>Quercus championii</em></span>. <abbr class="cit-jnl-abbrev">Molecular Phylogenetics and Evolution</abbr> <span class="cit-vol">132</span> : <span class="cit-fpage">36</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%26rft.volume%253D132%26rft.spage%253D36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.40" data-doi="10.1093/bioinformatics/btn129"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">adegenet: a R package for the multivariate analysis of genetic markers</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span> (<span class="cit-issue">11</span>): <span class="cit-fpage">1403</span>–<span class="cit-lpage">1405</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn129%26rft_id%253Dinfo%253Apmid%252F18397895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397895&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256169300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.41" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">DM</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">30</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23329690&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000317002300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.42" data-doi="10.1111/j.1461-0248.2004.00684.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kawecki</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">D</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Conceptual issues in local adaptation</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1225</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D7%26rft.spage%253D1225%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2004.00684.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2004.00684.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225078000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.43" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sturrock</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buxton</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Markowitz</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duran</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span> (<span class="cit-issue">12</span>): <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">JF</span></span>. <span class="cit-pub-date">1964</span>. <span class="cit-article-title">The number of alleles that can be maintained in a finite population</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">49</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">725</span>–<span class="cit-lpage">738</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D49%26rft.issue%253D4%26rft.spage%253D725%26rft.epage%253D738%26rft.atitle%253DTHE%2BNUMBER%2BOF%2BALLELES%2BTHAT%2BCAN%2BBE%2BMAINTAINED%2BIN%2BA%2BFINITE%2BPOPULATION%26rft_id%253Dinfo%253Apmid%252F14156929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI0OS80LzcyNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2LzIwMjAuMDEuMTQuOTAyNjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.45" data-doi="10.1016/j.tplants.2009.08.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kramer</span>  <span class="cit-name-given-names">AT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Havens</span>  <span class="cit-name-given-names">K</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Plant conservation genetics in a changing world</span>. <abbr class="cit-jnl-abbrev">Trends in Plant Science</abbr> <span class="cit-vol">14</span> (<span class="cit-issue">11</span>): <span class="cit-fpage">599</span>–<span class="cit-lpage">607</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DKramer%26rft.auinit1%253DA.%2BT.%26rft.volume%253D14%26rft.issue%253D11%26rft.spage%253D599%26rft.epage%253D607%26rft.atitle%253DPlant%2Bconservation%2Bgenetics%2Bin%2Ba%2Bchanging%2Bworld.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2009.08.005%26rft_id%253Dinfo%253Apmid%252F19748300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2009.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19748300&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272501700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.46" data-doi="10.1111/j.1365-294X.2012.05678.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leinonen</span>  <span class="cit-name-given-names">PH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Remington</span>  <span class="cit-name-given-names">DL</span></span>, <span class="cit-auth"><span class="cit-name-surname">LeppÄLÄ</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Genetic basis of local adaptation and flowering time variation in <em>Arabidopsis lyrata</em></span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">22</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">709</span>–<span class="cit-lpage">723</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D22%26rft.spage%253D709%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05678.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05678.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000313726300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.47" data-doi="10.1093/mp/ssr036"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luan</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">The cyclophilin AtCYP71 interacts with CAF-1 and LHP1 and functions in multiple chromatin remodeling processes</span>. <abbr class="cit-jnl-abbrev">Mol Plant</abbr> <span class="cit-vol">4</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">748</span>–<span class="cit-lpage">758</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fssr036%26rft_id%253Dinfo%253Apmid%252F21596687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/ssr036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21596687&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.48" data-doi="10.1016/j.ympev.2011.01.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">S-N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">Y-X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">X-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Comes</span>  <span class="cit-name-given-names">HP</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Glacial survival east and west of the ‘Mekong–Salween Divide’in the Himalaya–Hengduan Mountains region as revealed by AFLPs and cpDNA sequence variation in <em>Sinopodophyllum hexandrum</em> (Berberidaceae)</span>. <abbr class="cit-jnl-abbrev">Molecular Phylogenetics and Evolution</abbr> <span class="cit-vol">59</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">412</span>–<span class="cit-lpage">424</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%26rft.volume%253D59%26rft.spage%253D412%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2011.01.009%26rft_id%253Dinfo%253Apmid%252F21296173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2011.01.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21296173&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.49" data-doi="10.1111/nph.12336"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Möller</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Provan</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">L-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poudel</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D-Z</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Geological and ecological factors drive cryptic speciation of yews in a biodiversity hotspot</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">199</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">1093</span>–<span class="cit-lpage">1108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D199%26rft.spage%253D1093%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12336%26rft_id%253Dinfo%253Apmid%252F23718262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12336&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23718262&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.50" data-doi="10.1073/pnas.1821543116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lovell</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonnette</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lloyd-Reilley</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boe</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rouquette</span>  <span class="cit-name-given-names">FM</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">QTL × environment interactions underlie adaptive divergence in switchgrass across a large latitudinal gradient</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">116</span> (<span class="cit-issue">26</span>): <span class="cit-fpage">12933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1821543116%26rft_id%253Dinfo%253Apmid%252F31182579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzI2LzEyOTMzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNC45MDI2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.51" data-doi="10.1016/j.ympev.2011.12.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McCormack</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hird</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zellmer</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carstens</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brumfield</span>  <span class="cit-name-given-names">RT</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Applications of next-generation sequencing to phylogeography and phylogenetics</span>. <abbr class="cit-jnl-abbrev">Molecular Phylogenetics and Evolution</abbr> <span class="cit-vol">66</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">526</span>–<span class="cit-lpage">538</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%26rft.volume%253D66%26rft.spage%253D526%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2011.12.007%26rft_id%253Dinfo%253Apmid%252F22197804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2011.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22197804&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.52" data-doi="10.1073/pnas.0706568104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McRae</span>  <span class="cit-name-given-names">BH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beier</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Circuit theory predicts gene flow in plant and animal populations</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">104</span> (<span class="cit-issue">50</span>): <span class="cit-fpage">19885</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0706568104%26rft_id%253Dinfo%253Apmid%252F18056641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzUwLzE5ODg1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNC45MDI2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.53" data-doi="10.1038/nrg2207"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">JH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">DB</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Which evolutionary processes influence natural genetic variation for phenotypic traits?</span> <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr> <span class="cit-vol">8</span> : <span class="cit-fpage">845</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMitchell-Olds%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D845%26rft.epage%253D856%26rft.atitle%253DWhich%2Bevolutionary%2Bprocesses%2Binfluence%2Bnatural%2Bgenetic%2Bvariation%2Bfor%2Bphenotypic%2Btraits%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2207%26rft_id%253Dinfo%253Apmid%252F17943192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943192&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250281000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.54" data-doi="10.1105/tpc.007906"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mustilli</span>  <span class="cit-name-given-names">AC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merlot</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vavasseur</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fenzi</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giraudat</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Arabidopsis OST1 protein kinase mediates the regulation of stomatal aperture by abscisic acid and acts upstream of reactive oxygen species production</span>. <abbr class="cit-jnl-abbrev">The Plant cell</abbr> <span class="cit-vol">14</span> (<span class="cit-issue">12</span>): <span class="cit-fpage">3089</span>–<span class="cit-lpage">3099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bcell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.007906%26rft_id%253Dinfo%253Apmid%252F12468729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMi8zMDg5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNC45MDI2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nevado</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">ELY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osborne</span>  <span class="cit-name-given-names">OG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Filatov</span>  <span class="cit-name-given-names">DA</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Adaptive Evolution Is Common in Rapid Evolutionary Radiations</span>. <abbr class="cit-jnl-abbrev">Current Biology</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.56" data-doi="10.1146/annurev.genet.39.073003.112420"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2005</span>. <span class="cit-article-title">Molecular Signatures of Natural Selection</span>. <abbr class="cit-jnl-abbrev">Annual Review of Genetics</abbr> <span class="cit-vol">39</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">197</span>–<span class="cit-lpage">218</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D39%26rft.spage%253D197%26rft.epage%253D218%26rft.atitle%253DMolecular%2Bsignatures%2Bof%2Bnatural%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.39.073003.112420%26rft_id%253Dinfo%253Apmid%252F16285858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.39.073003.112420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16285858&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234685200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nowakowski</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeWoody</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fagan</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willoughby</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Mechanistic insights into landscape genetic structure of two tropical amphibians using field-derived resistance surfaces</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">580</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D24%26rft.spage%253D580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-web" id="cit-2020.01.14.902643v2.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oksanen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanchet</span>  <span class="cit-name-given-names">FG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Friendly</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kindt</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGlinn</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minchin</span>  <span class="cit-name-given-names">PR</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Hara</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">GL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solymos</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Vegan: Community Ecology Package</span>. <a href="https://CRAN.R-project.org/package=vegan">https://CRAN.R-project.org/package=vegan</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.59" data-doi="10.1093/bioinformatics/bts460"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Peakall</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smouse</span>  <span class="cit-name-given-names">PE</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">GenAlEx 6.5: genetic analysis in Excel. Population genetic software for teaching and research--an update</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span> (<span class="cit-issue">19</span>): <span class="cit-fpage">2537</span>–<span class="cit-lpage">2539</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts460%26rft_id%253Dinfo%253Apmid%252F22820204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820204&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309687500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.60" data-doi="10.1111/j.1469-8137.2011.04033.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">MH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mansfield</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Desikan</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Arabidopsis histidine kinase 5 regulates salt sensitivity and resistance against bacterial and fungal infection</span>. <abbr class="cit-jnl-abbrev">New Phytol</abbr> <span class="cit-vol">194</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">168</span>–<span class="cit-lpage">180</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DPham%26rft.auinit1%253DJ.%26rft.volume%253D194%26rft.issue%253D1%26rft.spage%253D168%26rft.epage%253D180%26rft.atitle%253DArabidopsis%2Bhistidine%2Bkinase%2B5%2Bregulates%2Bsalt%2Bsensitivity%2Band%2Bresistance%2Bagainst%2Bbacterial%2Band%2Bfungal%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.04033.x%26rft_id%253Dinfo%253Apmid%252F22256998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.04033.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22256998&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300719400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.61" data-doi="10.1111/j.0906-7590.2008.5203.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudik</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Modeling of species distributions with Maxent: new extensions and a comprehensive evaluation</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">31</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">161</span>–<span class="cit-lpage">175</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D31%26rft.spage%253D161%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0906-7590.2008.5203.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0906-7590.2008.5203.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254499200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plummer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowles</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vines</span>  <span class="cit-name-given-names">K</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">CODA: convergence diagnosis and output analysis for MCMC</span>. <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">6</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">7</span>–<span class="cit-lpage">11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D6%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Poudel</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Möller</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">L-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baral</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D-Z</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Low genetic diversity and high inbreeding of the endangered yews in Central Himalaya: implications for conservation of their highly fragmented populations</span>. <abbr class="cit-jnl-abbrev">Diversity and Distributions</abbr> <span class="cit-vol">20</span> (<span class="cit-issue">11</span>): <span class="cit-fpage">1270</span>–<span class="cit-lpage">1284</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2Band%2BDistributions%26rft.volume%253D20%26rft.spage%253D1270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.64" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P</span></span>. <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNC45MDI2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">XJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">DZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">TS</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">PGA: a software package for rapid, accurate, and flexible batch annotation of plastomes</span>. <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">15</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D15%26rft.spage%253D50%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.14.902643v2.66"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Team C</span> <span class="cit-pub-date">2014</span>. <span class="cit-chapter-title">R: A language and environment for statistical computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>. <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-book" id="cit-2020.01.14.902643v2.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-chapter-title">TreeAnnotator version 1.6</span>. <span class="cit-vol">1</span>. <span class="cit-publ-name">University of Edinburgh</span>, <span class="cit-publ-loc">Edinburgh, UK</span>. Available at: <a href="http://beast.bio.ed.ac.uk">http://beast.bio.ed.ac.uk</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.68" data-doi="10.1093/sysbio/syy032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baele</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Posterior Summarization in Bayesian Phylogenetics Using Tracer 1.7</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">67</span> (<span class="cit-issue">5</span>): <span class="cit-fpage">901</span>–<span class="cit-lpage">904</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft.volume%253D67%26rft.spage%253D901%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyy032%26rft_id%253Dinfo%253Apmid%252F29718447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syy032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29718447&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.69" data-doi="10.1080/10635150701883881"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ree</span>  <span class="cit-name-given-names">RH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">SA</span></span>. <span class="cit-pub-date">2008</span>. <span class="cit-article-title">Maximum likelihood inference of geographic range evolution by dispersal, local extinction, and cladogenesis</span>. <abbr class="cit-jnl-abbrev">Syst Biol</abbr> <span class="cit-vol">57</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">4</span>–<span class="cit-lpage">14</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701883881%26rft_id%253Dinfo%253Apmid%252F18253896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150701883881&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2009007777&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=18253896&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253657800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.70" data-doi="10.1111/mec.13322"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rellstab</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gugerli</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holderegger</span>  <span class="cit-name-given-names">R</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">A practical guide to environmental association analysis in landscape genomics</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">24</span> (<span class="cit-issue">17</span>): <span class="cit-fpage">4348</span>–<span class="cit-lpage">4370</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D24%26rft.spage%253D4348%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mateo</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guisan</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conti</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salamin</span>  <span class="cit-name-given-names">N</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Genetic consequences of Quaternary climatic oscillations in the Himalayas: <em>Primula tibetica</em> as a case study based on restriction site-associated DNA sequencing</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">213</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">1500</span>–<span class="cit-lpage">1512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D213%26rft.spage%253D1500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Approaches to the systematic position of <em>Circaeaster</em> based on the morphological data</span>. <abbr class="cit-jnl-abbrev">Acta Botanica Boreali-occidentalia Sinica</abbr> <span class="cit-vol">23</span> (<span class="cit-issue">7</span>): <span class="cit-fpage">1091</span>–<span class="cit-lpage">1097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BBotanica%2BBoreali-occidentalia%2BSinica%26rft.volume%253D23%26rft.spage%253D1091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rochette</span>  <span class="cit-name-given-names">NC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Deriving genotypes from RAD-seq short-read data using Stacks</span>. <abbr class="cit-jnl-abbrev">Nature Protocols</abbr> <span class="cit-vol">12</span> : <span class="cit-fpage">2640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BProtocols%26rft.volume%253D12%26rft.spage%253D2640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.74" data-doi="10.1046/j.1471-8286.2003.00566.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">NA</span></span>. <span class="cit-pub-date">2004</span>. <span class="cit-article-title">Distruct: a program for the graphical display of population structure</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Notes</abbr> <span class="cit-vol">4</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">137</span>–<span class="cit-lpage">138</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BNotes%26rft.volume%253D4%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1471-8286.2003.00566.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1471-8286.2003.00566.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000189159500042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.75" data-doi="10.1093/molbev/msx248"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrer-Mata</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanchez-DelBarrio</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guirao-Rico</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Librado</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanchez-Gracia</span>  <span class="cit-name-given-names">A</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">DnaSP 6: DNA Sequence Polymorphism Analysis of Large Data Sets</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">34</span> (<span class="cit-issue">12</span>): <span class="cit-fpage">3299</span>–<span class="cit-lpage">3302</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D34%26rft.spage%253D3299%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx248%26rft_id%253Dinfo%253Apmid%252F29029172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29029172&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruiz-Sanchez</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rodriguez-Gomez</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sosa</span>  <span class="cit-name-given-names">V</span></span>. <span class="cit-pub-date">2012</span>. <span class="cit-article-title">Refugia and geographic barriers of populations of the desert poppy, <em>Hunnemannia fumariifolia</em> (Papaveraceae)</span>. <abbr class="cit-jnl-abbrev">Organisms Diversity &amp; Evolution</abbr> <span class="cit-vol">12</span> (<span class="cit-issue">2</span>): <span class="cit-fpage">133</span>–<span class="cit-lpage">143</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOrganisms%2BDiversity%2B%2526%2BEvolution%26rft.volume%253D12%26rft.spage%253D133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.77" data-doi="10.1038/nrg3522"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Ecological genomics of local adaptation</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr> <span class="cit-vol">14</span> : <span class="cit-fpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D14%26rft.spage%253D807%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3522%26rft_id%253Dinfo%253Apmid%252F24136507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24136507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.78" data-doi="10.1146/annurev.ecolsys.38.091206.095646"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knürr</span>  <span class="cit-name-given-names">T</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Gene Flow and Local Adaptation in Trees</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">38</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">595</span>–<span class="cit-lpage">619</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D38%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.38.091206.095646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.38.091206.095646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252359800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.79" data-doi="10.1038/nrg3644"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seehausen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butlin</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boughman</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peichel</span>  <span class="cit-name-given-names">CL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saetre</span>  <span class="cit-name-given-names">G-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bank</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brännström</span>  <span class="cit-name-given-names">Å</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Genomics and the origin of species</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr> <span class="cit-vol">15</span> : <span class="cit-fpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D15%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3644%26rft_id%253Dinfo%253Apmid%252F24535286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24535286&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.80" data-doi="10.1105/tpc.18.00441"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sierla</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horak</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Overmyer</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waszczak</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yarmolinsky</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maierhofer</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vainonen</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salojarvi</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denessiouk</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laanemets</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">The Receptor-like Pseudokinase GHR1 Is Required for Stomatal Closure</span>. <abbr class="cit-jnl-abbrev">The Plant cell</abbr> <span class="cit-vol">30</span> (<span class="cit-issue">11</span>): <span class="cit-fpage">2813</span>–<span class="cit-lpage">2837</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bcell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.18.00441%26rft_id%253Dinfo%253Apmid%252F30361234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xMS8yODEzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNC45MDI2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spieth</span>  <span class="cit-name-given-names">PT</span></span>. <span class="cit-pub-date">1974</span>. <span class="cit-article-title">Gene flow and genetic differentiation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSpieth%26rft.auinit1%253DP.%2BT.%26rft.volume%253D78%26rft.issue%253D3%26rft.spage%253D961%26rft.epage%253D965%26rft.atitle%253DGENE%2BFLOW%2BAND%2BGENETIC%2BDIFFERENTIATION%26rft_id%253Dinfo%253Apmid%252F4452477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMy85NjEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi8yMDIwLjAxLjE0LjkwMjY0My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.82" data-doi="10.1038/nature19754"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stroud</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Surgenor</span>  <span class="cit-name-given-names">EE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Formosa</span>  <span class="cit-name-given-names">LE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reljic</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frazier</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dibley</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osellame</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stait</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beilharz</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thorburn</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Accessory subunits are integral for assembly and function of human mitochondrial complex I</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">538</span> (<span class="cit-issue">7623</span>): <span class="cit-fpage">123</span>–<span class="cit-lpage">126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D538%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19754%26rft_id%253Dinfo%253Apmid%252F27626371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27626371&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.83" data-doi="10.1016/j.palaeo.2010.09.023"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">B-N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J-Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y-S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">S-T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">S-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">D-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z-C</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Reconstructing Neogene vegetation and climates to infer tectonic uplift in western Yunnan, China</span>. <abbr class="cit-jnl-abbrev">Palaeogeography, Palaeoclimatology, Palaeoecology</abbr> <span class="cit-vol">304</span> (<span class="cit-issue">3</span>): <span class="cit-fpage">328</span>–<span class="cit-lpage">336</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPalaeogeography%252C%2BPalaeoclimatology%252C%2BPalaeoecology%26rft.stitle%253DPalaeogeography%252C%2BPalaeoclimatology%252C%2BPalaeoecology%26rft.volume%253D304%26rft.issue%253D3-4%26rft.spage%253D328%26rft.epage%253D336%26rft.atitle%253DReconstructing%2BNeogene%2Bvegetation%2Band%2Bclimates%2Bto%2Binfer%2Btectonic%2Buplift%2Bin%2Bwestern%2BYunnan%252C%2BChina%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.palaeo.2010.09.023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.palaeo.2010.09.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2011070789&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000291190300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.84"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boufford</span>  <span class="cit-name-given-names">DE</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Origins and evolution of plant diversity in the Hengduan Mountains, China</span>. <abbr class="cit-jnl-abbrev">Plant Diversity</abbr> <span class="cit-vol">39</span> (<span class="cit-issue">4</span>): <span class="cit-fpage">161</span>–<span class="cit-lpage">166</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDiversity%26rft.volume%253D39%26rft.spage%253D161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landis</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2020</span>. <span class="cit-article-title">The draft genome of the endangered, relictual plant <em>Kingdonia uniflora</em> (Circaeasteraceae, Ranunculales) reveals potential mechanisms and perils of evolutionary specialization</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: 2020.2001.2008.898460.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.86"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adelalu</span>  <span class="cit-name-given-names">KF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jian</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Complete plastome sequencing of both living species of Circaeasteraceae (Ranunculales) reveals unusual rearrangements and the loss of the <em>ndh</em> gene family</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">592</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.87" data-doi="10.1105/tpc.109.071316"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tognetti</span>,  <span class="cit-name-given-names">V. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Aken</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morreel</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vandenbroucke</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van de Cotte</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Clercq</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Perturbation of indole-3-butyric acid homeostasis by the UDP-glucosyltransferase UGT74E2 modulates Arabidopsis architecture and water stress tolerance</span>. <abbr class="cit-jnl-abbrev">The Plant cell</abbr>, <span class="cit-vol">22</span> (<span class="cit-issue">8</span>), <span class="cit-fpage">2660</span>–<span class="cit-lpage">2679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bcell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.071316%26rft_id%253Dinfo%253Apmid%252F20798329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzgvMjY2MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2LzIwMjAuMDEuMTQuOTAyNjQzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.88" data-doi="10.1146/annurev-genet-120213-092110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Population Genomics for Understanding Adaptation in Wild Plant Species</span>. <abbr class="cit-jnl-abbrev">Annual Review of Genetics</abbr> <span class="cit-vol">49</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">315</span>–<span class="cit-lpage">338</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BGenetics%26rft.volume%253D49%26rft.spage%253D315%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-120213-092110%26rft_id%253Dinfo%253Apmid%252F26436459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-120213-092110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26436459&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.89" data-doi="10.2307/2408641"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">BS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">CC</span></span>. <span class="cit-pub-date">1984</span>. <span class="cit-article-title">Estimating F-statistics for the analysis of population structure</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span> (<span class="cit-issue">6</span>): <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft_id%253Dinfo%253Apmid%252F28563791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28563791&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.90" data-doi="10.3389/fgene.2014.00004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">JQ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nie</span>  <span class="cit-name-given-names">ZL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">H</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Evolutionary diversifications of plants on the Qinghai-Tibetan Plateau</span>. <abbr class="cit-jnl-abbrev">Front Genet</abbr> <span class="cit-vol">5</span> : <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront%2BGenet%26rft.volume%253D5%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.91"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S</span></span>. <span class="cit-pub-date">1946</span>. <span class="cit-article-title">Isolation by distance under diverse systems of mating</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">31</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">39</span>–<span class="cit-lpage">59</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D39%26rft.epage%253D59%26rft.atitle%253DISOLATION%2BBY%2BDISTANCE%2BUNDER%2BDIVERSE%2BSYSTEMS%2BOF%2BMATING%26rft_id%253Dinfo%253Apmid%252F17247184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIzMS8xLzM5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvMjAyMC4wMS4xNC45MDI2NDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">DZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">HT</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Highly effective sequencing whole chloroplast genomes of angiosperms by nine novel universal primer pairs</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Resour</abbr> <span class="cit-vol">14</span> (<span class="cit-issue">5</span>): <span class="cit-fpage">1024</span>–<span class="cit-lpage">1031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BResour%26rft.volume%253D14%26rft.spage%253D1024%26rft_id%253Dinfo%253Apmid%252F24620934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24620934&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.93" data-doi="10.1016/j.ympev.2015.03.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blair</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">RASP (Reconstruct Ancestral State in Phylogenies): a tool for historical biogeography</span>. <abbr class="cit-jnl-abbrev">Mol Phylogenet Evol</abbr> <span class="cit-vol">87</span> : <span class="cit-fpage">46</span>–<span class="cit-lpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BPhylogenet%2BEvol%26rft.volume%253D87%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2015.03.008%26rft_id%253Dinfo%253Apmid%252F25819445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2015.03.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25819445&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F2020.01.14.902643.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.94" data-doi="10.1111/1755-0998.13096"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jakovlic</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">WX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">GT</span></span>. <span class="cit-pub-date">2019</span>. <span class="cit-article-title">PhyloSuite: An integrated and scalable desktop platform for streamlined molecular sequence data management and evolutionary phylogenetics studies</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Resour</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/1755-0998.13096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%2BResour%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.13096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.13096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-2020.01.14.902643v2.95"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">Y-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">X-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J-F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z-W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">T-S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">X-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y-C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Adaptation of <em>Arabidopsis thaliana</em> to the Yangtze River basin</span>. <abbr class="cit-jnl-abbrev">Genome Biology</abbr> <span class="cit-vol">18</span> (<span class="cit-issue">1</span>): <span class="cit-fpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D18%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2020.01.14.902643v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.14.902643v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/667493v3" title="Status of aquatic and riparian biodiversity in artificial lake ecosystems with and without management for recreational fisheries: implications for conservation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 16, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2020.01.14.902643v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2020.01.14.902643v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2020.01.14.902643v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2020.01.14.902643v2" class="active" data-icon-position="" data-hide-link-title="0">Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual Circaeaster agrestis (Circaeasteraceae, Ranunculales)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1107107" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-EQugYqqOHgczGz2QOqmjmG_znd-Pu4Efik30B1HiDLc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107107" id="node1107107--2" data-pisa="biorxiv;2020.01.14.902643v2" data-pisa-master="biorxiv;2020.01.14.902643" data-seqnum="1107107" data-apath="/biorxiv/early/2020/01/16/2020.01.14.902643.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual <em>Circaeaster agrestis</em> (Circaeasteraceae, Ranunculales)</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Xu</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yanxia</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jacob B.</span> <span class="nlm-surname">Landis</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianwen</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Linsen</span> <span class="nlm-surname">Yang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Nan</span> <span class="nlm-surname">Lin</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Huajie</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Rui</span> <span class="nlm-surname">Guo</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Lijuan</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Yonghong</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Deng</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Hang</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Hengchang</span> <span class="nlm-surname">Wang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.14.902643; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.14.902643 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.14.902643v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2&amp;title=Genome%20wide%20sequencing%20provides%20evidence%20of%20adaptation%20to%20heterogeneous%20environments%20for%20the%20ancient%20relictual%20Circaeaster%20agrestis%20%28Circaeasteraceae%2C%20Ranunculales%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2&amp;title=Genome%20wide%20sequencing%20provides%20evidence%20of%20adaptation%20to%20heterogeneous%20environments%20for%20the%20ancient%20relictual%20Circaeaster%20agrestis%20%28Circaeasteraceae%2C%20Ranunculales%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2&amp;text=Genome%20wide%20sequencing%20provides%20evidence%20of%20adaptation%20to%20heterogeneous%20environments%20for%20the%20ancient%20relictual%20Circaeaster%20agrestis%20%28Circaeasteraceae%2C%20Ranunculales%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2&amp;title=Genome%20wide%20sequencing%20provides%20evidence%20of%20adaptation%20to%20heterogeneous%20environments%20for%20the%20ancient%20relictual%20Circaeaster%20agrestis%20%28Circaeasteraceae%2C%20Ranunculales%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2&amp;t=Genome%20wide%20sequencing%20provides%20evidence%20of%20adaptation%20to%20heterogeneous%20environments%20for%20the%20ancient%20relictual%20Circaeaster%20agrestis%20%28Circaeasteraceae%2C%20Ranunculales%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2&amp;title=Genome%20wide%20sequencing%20provides%20evidence%20of%20adaptation%20to%20heterogeneous%20environments%20for%20the%20ancient%20relictual%20Circaeaster%20agrestis%20%28Circaeasteraceae%2C%20Ranunculales%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2&amp;title=Genome%20wide%20sequencing%20provides%20evidence%20of%20adaptation%20to%20heterogeneous%20environments%20for%20the%20ancient%20relictual%20Circaeaster%20agrestis%20%28Circaeasteraceae%2C%20Ranunculales%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107107" id="node1107107--3" data-pisa="biorxiv;2020.01.14.902643v2" data-pisa-master="biorxiv;2020.01.14.902643" data-seqnum="1107107" data-apath="/biorxiv/early/2020/01/16/2020.01.14.902643.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Genome wide sequencing provides evidence of adaptation to heterogeneous environments for the ancient relictual <em>Circaeaster agrestis</em> (Circaeasteraceae, Ranunculales)</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Xu</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yanxia</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jacob B.</span> <span class="nlm-surname">Landis</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jianwen</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Linsen</span> <span class="nlm-surname">Yang</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Nan</span> <span class="nlm-surname">Lin</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Huajie</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Rui</span> <span class="nlm-surname">Guo</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Lijuan</span> <span class="nlm-surname">Li</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Yonghong</span> <span class="nlm-surname">Zhang</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Tao</span> <span class="nlm-surname">Deng</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Hang</span> <span class="nlm-surname">Sun</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Hengchang</span> <span class="nlm-surname">Wang</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2020.01.14.902643; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2020.01.14.902643 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Genome%20wide%20sequencing%20provides%20evidence%20of%20adaptation%20to%20heterogeneous%20environments%20for%20the%20ancient%20relictual%20Circaeaster%20agrestis%20%28Circaeasteraceae%2C%20Ranunculales%29&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Zhang%2C+Xu&amp;rft.au=Sun%2C+Yanxia&amp;rft.au=Landis%2C+Jacob+B.&amp;rft.au=Zhang%2C+Jianwen&amp;rft.au=Yang%2C+Linsen&amp;rft.au=Lin%2C+Nan&amp;rft.au=Zhang%2C+Huajie&amp;rft.au=Guo%2C+Rui&amp;rft.au=Li%2C+Lijuan&amp;rft.au=Zhang%2C+Yonghong&amp;rft.au=Deng%2C+Tao&amp;rft.au=Sun%2C+Hang&amp;rft.au=Wang%2C+Hengchang"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1107107/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1107107/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1107107/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1107107/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1107107/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1107107/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1107107/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1107107/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1107107/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1107107/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1107107/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1107107/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2&amp;count=horizontal&amp;via=&amp;text=Genome%20wide%20sequencing%20provides%20evidence%20of%20adaptation%20to%20heterogeneous%20environments%20for%20the%20ancient%20relictual%20Circaeaster%20agrestis%20%28Circaeasteraceae%2C%20Ranunculales%29&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.14.902643v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2020.01.14.902643v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first odd"><span class="highwire-article-collection-term"><a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">Genomics<i class="icon-caret-right"></i>
</a></span></li><li class="last even"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
