<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2016/07/09/063032.atom" />
<meta name="HW.pisa" content="biorxiv;063032v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Past climate changes, population dynamics and the origin of Bison in Europe" />
<meta name="DC.Identifier" content="10.1101/063032" />
<meta name="DC.Date" content="2016-07-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2016, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Climatic and environmental fluctuations as well as anthropogenic pressure have led to the extinction of much of Europe’s megafauna. Here we show that the emblematic European bison has experienced several waves of population expansion, contraction and extinction during the last 50,000 years in Europe, culminating in a major reduction of genetic diversity during the Holocene. Fifty-seven complete and partial ancient mitogenomes from throughout Europe, the Caucausus and Siberia reveal that three populations of wisent ( Bison bonasus ) and steppe bison ( B. priscus ) alternated in Western Europe correlating with climate-induced environmental changes. The Late Pleistocene European steppe bison originated from northern Eurasia whereas the modern wisent population emerged from a refuge in the southern Caucasus after the last glacial maximum. A population overlap in a transition period is reflected in ca. 36,000 year-old paintings in the French Chauvet cave. Bayesian analyses of these complete ancient mitogenomes yielded new dates of the various branching events during the evolution of Bison and its radiation with Bos that lead us to propose that the genetic affiliation between the wisent and cattle mitogenomes result from incomplete lineage sorting rather than post-speciation gene flow.

Significance Climatic fluctuations during the Pleistocene had a major impact on the environment and led to multiple megafaunal extinctions. Through ancient DNA analyses we decipher these processes for one of the largest megafauna of Eurasia, the bison. We show that Western Europe was successively populated during the Late Pleistocene by three different bison clades or species originating from the Caucasus and North-Eastern Europe that can be correlated to major climatic fluctuations and environmental changes. Aurignacian cave artists were witnesses to the first replacement of bison species ~35,000 years ago. All of these populations went extinct except for one that survived into the Holocene where it experienced severe reductions of its genetic diversity due to anthropogenic pressure." />
<meta name="DC.Contributor" content="Diyendo Massilani" />
<meta name="DC.Contributor" content="Silvia Guimaraes" />
<meta name="DC.Contributor" content="Jean-Philip Brugal" />
<meta name="DC.Contributor" content="E. Andrew Bennett" />
<meta name="DC.Contributor" content="Malgorzata Tokarska" />
<meta name="DC.Contributor" content="Rose-Marie Arbogast" />
<meta name="DC.Contributor" content="Gennady Baryshnikov" />
<meta name="DC.Contributor" content="Gennady Boeskorov" />
<meta name="DC.Contributor" content="Jean-Christophe Castel" />
<meta name="DC.Contributor" content="Sergey Davydov" />
<meta name="DC.Contributor" content="Stephane Madelaine" />
<meta name="DC.Contributor" content="Olivier Putelat" />
<meta name="DC.Contributor" content="Natalia Spasskaya" />
<meta name="DC.Contributor" content="Hans-Peter Uerpmann" />
<meta name="DC.Contributor" content="Thierry Grange" />
<meta name="DC.Contributor" content="Eva-Maria Geigl" />
<meta name="article:published_time" content="2016-07-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Past climate changes, population dynamics and the origin of Bison in Europe" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Climatic and environmental fluctuations as well as anthropogenic pressure have led to the extinction of much of Europe’s megafauna. Here we show that the emblematic European bison has experienced several waves of population expansion, contraction and extinction during the last 50,000 years in Europe, culminating in a major reduction of genetic diversity during the Holocene. Fifty-seven complete and partial ancient mitogenomes from throughout Europe, the Caucausus and Siberia reveal that three populations of wisent (&lt;i&gt;Bison bonasus&lt;/i&gt;) and steppe bison (&lt;i&gt;B. priscus&lt;/i&gt;) alternated in Western Europe correlating with climate-induced environmental changes. The Late Pleistocene European steppe bison originated from northern Eurasia whereas the modern wisent population emerged from a refuge in the southern Caucasus after the last glacial maximum. A population overlap in a transition period is reflected in ca. 36,000 year-old paintings in the French Chauvet cave. Bayesian analyses of these complete ancient mitogenomes yielded new dates of the various branching events during the evolution of Bison and its radiation with Bos that lead us to propose that the genetic affiliation between the wisent and cattle mitogenomes result from incomplete lineage sorting rather than post-speciation gene flow.&lt;/p&gt;&lt;h3&gt;Significance&lt;/h3&gt;
&lt;p&gt;Climatic fluctuations during the Pleistocene had a major impact on the environment and led to multiple megafaunal extinctions. Through ancient DNA analyses we decipher these processes for one of the largest megafauna of Eurasia, the bison. We show that Western Europe was successively populated during the Late Pleistocene by three different bison clades or species originating from the Caucasus and North-Eastern Europe that can be correlated to major climatic fluctuations and environmental changes. Aurignacian cave artists were witnesses to the first replacement of bison species ~35,000 years ago. All of these populations went extinct except for one that survived into the Holocene where it experienced severe reductions of its genetic diversity due to anthropogenic pressure.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="biorxiv;063032v1" />
<meta name="citation_id" content="063032v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/063032v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/063032v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/063032v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032.full.pdf" />
<meta name="citation_doi" content="10.1101/063032" />
<meta name="citation_num_pages" content="36" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="063032" />
<meta name="citation_author" content="Diyendo Massilani" />
<meta name="citation_author_institution" content="Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot" />
<meta name="citation_author" content="Silvia Guimaraes" />
<meta name="citation_author_institution" content="Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot" />
<meta name="citation_author" content="Jean-Philip Brugal" />
<meta name="citation_author_institution" content="CNRS, USR 3336 IFRA (Institut Frangais de Recherche en Afrique; Nairobi, Kenya) &amp; Aix-Marseille Universite" />
<meta name="citation_author" content="E. Andrew Bennett" />
<meta name="citation_author_institution" content="Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot" />
<meta name="citation_author" content="Malgorzata Tokarska" />
<meta name="citation_author_institution" content="Mammal Research Institute Polish Academy of Sciences, Genetics and Evolution Department" />
<meta name="citation_author" content="Rose-Marie Arbogast" />
<meta name="citation_author_institution" content="CNRS/UMR 7044/MISHA" />
<meta name="citation_author" content="Gennady Baryshnikov" />
<meta name="citation_author_institution" content="Zoological Institute, Russian Academy of Sciences" />
<meta name="citation_author" content="Gennady Boeskorov" />
<meta name="citation_author_institution" content="Diamond and Precious Metal Geology Institute of the Siberian Branch of the RAS" />
<meta name="citation_author" content="Jean-Christophe Castel" />
<meta name="citation_author_institution" content="Museum d’histoire naturelle de Geneve (MHN), Departement d’Archeozoologie" />
<meta name="citation_author" content="Sergey Davydov" />
<meta name="citation_author_institution" content="North-East Science Station, Pacific Institute of Geography" />
<meta name="citation_author" content="Stephane Madelaine" />
<meta name="citation_author_institution" content="Musee national de Prehistoire" />
<meta name="citation_author" content="Olivier Putelat" />
<meta name="citation_author_institution" content="Pole d’Archeologie Interdepartmental Rhenan, 67600 Selestat" />
<meta name="citation_author" content="Natalia Spasskaya" />
<meta name="citation_author_institution" content="Zoological Museum of Moscow Lomonosow, State University" />
<meta name="citation_author" content="Hans-Peter Uerpmann" />
<meta name="citation_author_institution" content="Institut fur Ur-und Fruhgeschichte und Archaologie des Mittelalters, Abteilung Altere Urgeschichte und Quartarokologie" />
<meta name="citation_author" content="Thierry Grange" />
<meta name="citation_author_institution" content="Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot" />
<meta name="citation_author" content="Eva-Maria Geigl" />
<meta name="citation_author_institution" content="Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot" />
<meta name="citation_reference" content="Hewitt GM (2004) Genetic consequences of climatic oscillations in the Quaternary. Phil. Trans. RL. Soc. Lond. B 359:183–195." />
<meta name="citation_reference" content="Markova AK, et al. (2015) Changes in the Eurasian distribution of the musk ox (Ovibos moschatus) and the extinct bison (Bison priscus) during the last 50 ka BP. Quatern Int 378:99110." />
<meta name="citation_reference" content="Guthrie RD (1990) Frozen fauna of the Mammoth Steppe. The story of Blue Babe. (The University of Chicago Press, Chicago and London) p 323." />
<meta name="citation_reference" content="Brugal JP (1994-1995) Le Bison (Bovinae, Artiodactyla) du gisement Pleistocene moyen ancien de Durfort (Gard, France). Bull.Mus.nat.Hist.Nat., Paris 4e ser. 16, sect.C, n°2–4:349– 381." />
<meta name="citation_reference" content="Brugal JP, David F, Enloe J, &amp; Jaubert J (1999) Le Bison, gibier et moyen de subsistance des hommes du Paleolithique aux Paleoindiens des grandes plaines (Antibes)." />
<meta name="citation_reference" content="citation_title=European Bison-Morphology, systematics, evolution, ecology (in Russian);citation_pages=9-127;citation_year=1979" />
<meta name="citation_reference" content="Shapiro B, et al. (2004) Rise and fall of the Beringian steppe bison. Science 306(5701):1561– 1565." />
<meta name="citation_reference" content="Groves CP (1981) Systematic relationship in the Bovini (Artiodactyla, Bovidae). Z.Zool.Syst.Evolutionforsch. 19(4):264–278." />
<meta name="citation_reference" content="Hassanin A, An J, Ropiquet A, Nguyen TT, &amp; Couloux A (2013) Combining multiple autosomal introns for studying shallow phylogeny and taxonomy of Laurasiatherian mammals: Application to the tribe Bovini (Cetartiodactyla, Bovidae). Molecular phylogenetics and evolution 66(3):766–775." />
<meta name="citation_reference" content="Kowalski K (1967) The evolution and fossil remains of the European Bison. Acta Theriologica 12(21):335–338." />
<meta name="citation_reference" content="Krasinska M &amp; Krasinski ZA (2013) European Bison: The Nature Monograph. 2nd Edition (Springer, Heidelberg)." />
<meta name="citation_reference" content="Cooper A, et al. (2015) PALEOECOLOGY. Abrupt warming events drove Late Pleistocene Holarctic megafaunal turnover. Science 349(6248):602–606." />
<meta name="citation_reference" content="Boeskorov GG, et al. (2014) Preliminary analyses of the frozen mummies of mammoth (Mammuthus primigenius), bison (Bison priscus) and horse (Equus sp.) from the Yana-Indigirka Lowland, Yakutia, Russia. Integrative zoology 9(4):471–480." />
<meta name="citation_reference" content="Kirillova IV, et al. (2015) An ancient bison from the mouth of the Rauchua River (Chukotka, Russia). Quatern. Res. 84(2):232–245." />
<meta name="citation_reference" content="Degerbol M &amp; Iversen J (1945) The Bison in Denmark. A zoological and geological investigation of the finds in Danish Pleistocene deposits (Danm.Geol.Unders0gelse, Copenhagen)." />
<meta name="citation_reference" content="Geist V (1971) The relation of social evolution and dispersal in ungulates during the Pleistocene, with emphasis on the old world Deer and the genus Bison. Quaternary Research 1(3):283–221." />
<meta name="citation_reference" content="Rautian GS, Kalabushkin BA, &amp; Nemtsev AS (2000) A New Subspecies of the European Bison, Bison bonasus montanus ssp. nov. (Bovidae, Artiodactyla). Doklady Biological Sciences 375 636–640." />
<meta name="citation_reference" content="Bocherens H, Hofman-Kaminska E, Drucker DG, Schmolcke U, &amp; Kowalczyk R (2015) European bison as a refugee species? Evidence from isotopic data on Early Holocene bison and other large herbivores in northern Europe. PLoS One 10(2):e0115090." />
<meta name="citation_reference" content="Spassov N &amp; Stovtechev T (2003) On the origin of the wisent, Bison bonasus (Linnaeus, 1758): presence of the wisent in the upper Palaeolithic art of Eurasia. Advances in Vertebrate Paleontology « Hen to Panta »:125–130." />
<meta name="citation_reference" content="citation_title=History of Bison in North America. American Bison: Status survey and conservation guidelines 2010;citation_pages=13-18;citation_year=2010" />
<meta name="citation_reference" content="Verkaar EL, Nijman IJ, Beeke M, Hanekamp E, &amp; Lenstra JA (2004) Maternal and paternal lineages in cross-breeding bovine species. Has wisent a hybrid origin? Mol Biol Evol 21(7):1165–1170." />
<meta name="citation_reference" content="Zeyland J, et al. (2012) Tracking of wisent-bison-yak mitochondrial evolution. Journal of applied genetics 53(3):317–322." />
<meta name="citation_reference" content="Marsolier-Kergoat MC, et al. (2015) Hunting the Extinct Steppe Bison (Bison priscus) Mitochondrial Genome in the Trois-Freres Paleolithic Painted Cave. PLoS One 10(6):e0128267." />
<meta name="citation_reference" content="Lisiecki LE &amp; Raymo ME (2005) A Pliocene-Pleistocene stack of 57 globally distributed benthic 518O records. Paleoceanography 20(1):1–17." />
<meta name="citation_reference" content="Andersen KK, et al. (2004) High-resolution record of Northern Hemisphere climate extending into the last interglacial period. Nature 431(7005):147–151." />
<meta name="citation_reference" content="Van Andel TH &amp; Tzedakis PC (1996) Palaeolithic landscapes of Europe and environs, 150,000 25,0 years ago: An overview. Quat Sci Rev 15(5–6):481–500." />
<meta name="citation_reference" content="Guiot J, Pons A, de Beaulieu JL, &amp; Reille M (1989) A 140,000 year climatic reconstruction from two European pollen records. Nature 338: 309–313." />
<meta name="citation_reference" content="Palombo MR (2014) Deconstructing mammal dispersals and faunal dynamics in SW Europe during the Quaternary. Quat Sci Rev 96:50–71." />
<meta name="citation_reference" content="Julien M-F, et al. (2012) Were European steppe bison migratory? 18O, 13C and Sr intra-tooth isotopic variations applied to a pallaeoethological reconstruction. Quarternary International 271:106–119." />
<meta name="citation_reference" content="Quiles A, et al. (2016) A high-precision chronological model for the decorated Upper Paleolithic cave of Chauvet-Pont d’Arc, Ardeche, France. Proc Natl Acad Sci U S A 113(17):4670–4675." />
<meta name="citation_reference" content="citation_journal_title=American Bison: Status survey and conservation guidelines 2010;citation_title=Taxonomy and nomenclature;citation_author=IUCN, Gland, Switzerland;citation_pages=13-18;citation_year=2010" />
<meta name="citation_reference" content="Yudin NS, et al. (2012) Detection of mitochondrial DNA from domestic cattle in European bison (Bison bonasus) from the Altai Republic in Russia. Anim Genet 43(3):362." />
<meta name="citation_reference" content="Halbert ND &amp; Derr JN (2007) A comprehensive evaluation of cattle introgression into US federal bison herds. The Journal of heredity 98(1):1–12." />
<meta name="citation_reference" content="Martfnez-Navarro B, Rook L, Papini M, &amp; Libsekal Y (2010) A new species of bull from the Early Pleistocene paleoanthropological site of Buia (Eritrea): Parallelism on the dispersal of the genus Bos and the Acheulian culture. Quatern Int 212(2):169–175." />
<meta name="citation_reference" content="Gautier M, et al. (2016) Deciphering the wisent demographic and adaptive histories from individual whole-genome sequences. Mol Biol Evol In press, biorxiv http://dx.doi.org/10.1101/058446." />
<meta name="citation_reference" content="Wecek K, et al. (2016) Complex admixture preceded and followed the extinction of wisent in the wild. bioRxiv 059527:doi: http://dx.doi.org/10.1101/059527" />
<meta name="citation_reference" content="Bennett EA, et al. (2014) Library construction for ancient genomics: single strand or double strand? BioTechniques 56(6):289–290, 292-286, 298, passim." />
<meta name="citation_reference" content="Cote NM, et al. (2016) A New High-Throughput Approach to Genotype Ancient Human Gastrointestinal Parasites. PLoS One 11(1):e0146230." />
<meta name="citation_reference" content="Guimaraes S, et al. (submitted) A cost-effective high-throughput metabarcoding approach powerful enough to genotype ~44,000 year-old rodent remains from Northern Africa. Molecular Ecology Resources." />
<meta name="citation_reference" content="Pruvost M, Grange T, &amp; Geigl E-M (2005) Minimizing DNA contamination by using UNG-coupled quantitative real-time PCR on degraded DNA samples: application to ancient DNA studies. BioTechniques 38(4):569–575." />
<meta name="citation_reference" content="Champlot S, et al. (2010) An efficient multistrategy DNA decontamination procedure of PCR reagents for hypersensitive PCR applications. PLoS ONE 5(9)." />
<meta name="citation_reference" content="Pruvost M, et al. (2007) Freshly excavated fossil bones are best for amplification of ancient DNA. Proc. Natl. Acad. Sci. USA 104(3):739–744." />
<meta name="citation_reference" content="Kearse M, et al. (2012) Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics 28(12):1647– 1649." />
<meta name="citation_reference" content="Gorge O, et al. (2016) Analysis of Ancient DNA in Microbial Ecology. Methods in molecular biology 1399:289–315." />
<meta name="citation_reference" content="Rueckert A &amp; Morgan HW (2007) Removal of contaminating DNA from polymerase chain reaction using ethidium monoazide. Journal of microbiological methods 68(3):596–600." />
<meta name="citation_reference" content="Gnirke A, et al. (2009) Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing. Nature biotechnology 27(2):182–189." />
<meta name="citation_reference" content="Renaud G, Stenzel U, &amp; Kelso J (2014) leeHom: adaptor trimming and merging for Illumina sequencing reads. Nucleic Acids Res 42(18):e141." />
<meta name="citation_reference" content="Guindon S, et al. (2010) New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Systematic biology 59(3):307–321." />
<meta name="citation_reference" content="Stamatakis A (2014) RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30(9):1312–1313." />
<meta name="citation_reference" content="Drummond AJ &amp; Rambaut A (2007) BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evolutionary Biology 7:214." />
<meta name="citation_reference" content="Darriba D, Taboada GL, Doallo R, &amp; Posada D (2012) jModelTest 2: more models, new heuristics and parallel computing. Nature methods 9(8):772." />
<meta name="citation_reference" content="Soares P, et al. (2009) Correcting for purifying selection: an improved human mitochondrial molecular clock. Am J Hum Genet 84(6):740–759." />
<meta name="citation_reference" content="Martfnez-Navarro B, Ros-Montoya S, Espigares MP, &amp; Palmqvist P (2011) Presence of the Asian origin Bovini, Hemibos sp. aff. Hemibos gracilis and Bison sp., at the early Pleistocene site of Venta Micena (Orce, Spain). Quatern Int 243(1):54–60." />
<meta name="citation_reference" content="Qui Z, Deng T, &amp; Wang B (2004) Early Pleistocene mammalian fauna from Longdan, Dongxian, Gansu, China. Paleontologia Sinica 191(C27):1–198." />
<meta name="twitter:title" content="Past climate changes, population dynamics and the origin of Bison in Europe" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Climatic and environmental fluctuations as well as anthropogenic pressure have led to the extinction of much of Europe’s megafauna. Here we show that the emblematic European bison has experienced several waves of population expansion, contraction and extinction during the last 50,000 years in Europe, culminating in a major reduction of genetic diversity during the Holocene. Fifty-seven complete and partial ancient mitogenomes from throughout Europe, the Caucausus and Siberia reveal that three populations of wisent ( Bison bonasus ) and steppe bison ( B. priscus ) alternated in Western Europe correlating with climate-induced environmental changes. The Late Pleistocene European steppe bison originated from northern Eurasia whereas the modern wisent population emerged from a refuge in the southern Caucasus after the last glacial maximum. A population overlap in a transition period is reflected in ca. 36,000 year-old paintings in the French Chauvet cave. Bayesian analyses of these complete ancient mitogenomes yielded new dates of the various branching events during the evolution of Bison and its radiation with Bos that lead us to propose that the genetic affiliation between the wisent and cattle mitogenomes result from incomplete lineage sorting rather than post-speciation gene flow.

Significance Climatic fluctuations during the Pleistocene had a major impact on the environment and led to multiple megafaunal extinctions. Through ancient DNA analyses we decipher these processes for one of the largest megafauna of Eurasia, the bison. We show that Western Europe was successively populated during the Late Pleistocene by three different bison clades or species originating from the Caucasus and North-Eastern Europe that can be correlated to major climatic fluctuations and environmental changes. Aurignacian cave artists were witnesses to the first replacement of bison species ~35,000 years ago. All of these populations went extinct except for one that survived into the Holocene where it experienced severe reductions of its genetic diversity due to anthropogenic pressure." />
<meta name="og-title" property="og:title" content="Past climate changes, population dynamics and the origin of Bison in Europe" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/063032v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Climatic and environmental fluctuations as well as anthropogenic pressure have led to the extinction of much of Europe’s megafauna. Here we show that the emblematic European bison has experienced several waves of population expansion, contraction and extinction during the last 50,000 years in Europe, culminating in a major reduction of genetic diversity during the Holocene. Fifty-seven complete and partial ancient mitogenomes from throughout Europe, the Caucausus and Siberia reveal that three populations of wisent ( Bison bonasus ) and steppe bison ( B. priscus ) alternated in Western Europe correlating with climate-induced environmental changes. The Late Pleistocene European steppe bison originated from northern Eurasia whereas the modern wisent population emerged from a refuge in the southern Caucasus after the last glacial maximum. A population overlap in a transition period is reflected in ca. 36,000 year-old paintings in the French Chauvet cave. Bayesian analyses of these complete ancient mitogenomes yielded new dates of the various branching events during the evolution of Bison and its radiation with Bos that lead us to propose that the genetic affiliation between the wisent and cattle mitogenomes result from incomplete lineage sorting rather than post-speciation gene flow.

Significance Climatic fluctuations during the Pleistocene had a major impact on the environment and led to multiple megafaunal extinctions. Through ancient DNA analyses we decipher these processes for one of the largest megafauna of Eurasia, the bison. We show that Western Europe was successively populated during the Late Pleistocene by three different bison clades or species originating from the Caucasus and North-Eastern Europe that can be correlated to major climatic fluctuations and environmental changes. Aurignacian cave artists were witnesses to the first replacement of bison species ~35,000 years ago. All of these populations went extinct except for one that survived into the Holocene where it experienced severe reductions of its genetic diversity due to anthropogenic pressure." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2016-07-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/063032v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/063032v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/063032v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/063032v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/17292" />
    <title>Past climate changes, population dynamics and the origin of Bison in Europe | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__e8NuQlin4bcnMtI2zOEVRblohOk3xZwfnFAFdibS6pw__XVRsOpcO2N5B6_yYKvNFz_CihNSGO6Yy3SOy-5iyzfE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"bJHZnS5DIsDGB0KBDdDa2OkUwhKSq_Jn2DSEt6LhY1M","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;063032v1"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;063032v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;063032\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;063032\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;063032v1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;063032\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;063032\/DC1"}],"nid":"17292","apath":"\/biorxiv\/early\/2016\/07\/09\/063032.atom","pisa":"biorxiv;063032v1","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/063032v1","title":"Past climate changes, population dynamics and the origin of Bison in Europe","identifier":"node\/17292"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DV5RmErCiLUcEMsXY0Li56O+44uINCw9NylvOld+e889mEtaBkSJzAyPkw6s7EHcP3ELOsU9553t1p9MRt2DYXHFxhwu5saBeAPcgYE8mJ48fo1QKrikUdzE52+xerRzaWk3Z+q+GODbv\\\\\\\/FG+k71oQiRecHru5sDxTJUkFnAzopNNRRX6iZ5ruhCMcomcDFb75mPQTGIQYiyh596NoOS7BwTOa1RmgrY9U\\\\\\\/hPxn2LXiPJC8gR12KPnVVZC0QRGS2vGvZ4Y3pngUjJrP5rvDlTo24IVs9u\\\\\\\/fHEKl1thpg3R7s0hyvimgN2fw\\\\\\\/Vai\\\\\\\/D\\\\\\\/OvOOhY1sxOofr0HvJ+e6bMCrUqPLfogw1AIoq7uTJqI+fRuEoY+SZtW6vF5\\\\\\\/pBeh5kwZxVWux8faDONGSNUFAEuc0EhbrMaBJz\\\\\\\/Y+XLm2yB9wGr7Y74tGvu+O8Lew6u6zw5oQ++gY6NQoO7rqsB1w6jx9zU1VL9m9XbhEx9KQJxlMhwU9\\\\\\\/bsED\\\\\\\/DaxUsF3DayHPznsGScVMJs\\\\\\\/ICo8zn678q6nFaAHYJqosmdeS6m2x4kWVXlC8+r9PEGkOWmC7Ar2MkNiD\\\\\\\/G5X4wWxkUyOnbLP0clX8r5\\\\\\\/v6LqVTtYeV1Ern3MQVcol5XYQ7JubM4TVptXURnRfJxSzBMD63EetO9mh60ds+gYZj5hvuRpE24iqJFOz64tVpw6zKZjCk6RDvIDUBW8h8m4LmHMQhmzZk35SKLJUnQUZU+nRnHP\\\\\\\/WCTLVe\\\\\\\/8PlisFEU9rSvxecRhNsEAZDAhI3xgMoexLoK5y3DEW81OxV0WIPIaTrmjCwWD\\\\\\\/tSh60SG6064Wq3MS1RYTU6TgW4SquacP6KuctiZxNppaaanfcZJn8gEY+ZgvEigmqhCR\\\\\\\/+ZHS7ozIj69YwGZ+2RVrcq3OkfftouECc4ZWiGRIRpBNumvDGDAGVKrn79mYqab9s1fAJNsB8Cw9DqeEPE91bFzg1GrgCnr4Xx1fclMH5QCMrT5W6SQEBA1\\\\\\\/PpOpfqt9NIsQuBHVCOInokqjTp+4nj8Qn+TCtSXT10U+zXlo42U86oY+CTMbjnBbSP8NdUoQmJfrm3ogNRmTAcawwNoJS1tez1ylNDrQtfKQXzCyhLiqZqYdYsefj8Br2TqxzLAfts\\\\\\\/gWiZSkQ5LUCQQ1os\\\\\\\/Xc17Y\\\\\\\/kCR72nOGPwgwmsC1vYbZHiYA\\\\\\\/EE7SF8uVE59GZ51ON4YTNHC2ErJCrrwv7VY1Rvt8y+Dxtw9ulU+bphBw8xvPZuW0lhTrwpqQi+Mpr+prJ8nuGHYOi\\\\\\\/zRNsyAfk0zbrHwpjvu3HwibT322ygwt4Pi2dbNGZRjLRG8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022UPOYhdRtdpm29bF\\\\\\\/sGrxsw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022c9e442ea6b358f1e9fcce316fc6a254544d3fc41206431c13a661d295cb344d7\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/063032v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-17292 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/063032v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mj9BFP64N6frzCTvERKDIiuL0_ICEw0RlqUURa2eGhg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="17292" id="node17292" data-pisa="biorxiv;063032v1" data-pisa-master="biorxiv;063032" data-apath="/biorxiv/early/2016/07/09/063032.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Past climate changes, population dynamics and the origin of Bison in Europe</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Diyendo</span> <span class="nlm-surname">Massilani</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Silvia</span> <span class="nlm-surname">Guimaraes</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jean-Philip</span> <span class="nlm-surname">Brugal</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">E. Andrew</span> <span class="nlm-surname">Bennett</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Malgorzata</span> <span class="nlm-surname">Tokarska</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Rose-Marie</span> <span class="nlm-surname">Arbogast</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Gennady</span> <span class="nlm-surname">Baryshnikov</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Gennady</span> <span class="nlm-surname">Boeskorov</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jean-Christophe</span> <span class="nlm-surname">Castel</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Sergey</span> <span class="nlm-surname">Davydov</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Stephane</span> <span class="nlm-surname">Madelaine</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Olivier</span> <span class="nlm-surname">Putelat</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Natalia</span> <span class="nlm-surname">Spasskaya</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Hans-Peter</span> <span class="nlm-surname">Uerpmann</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Thierry</span> <span class="nlm-surname">Grange</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Eva-Maria</span> <span class="nlm-surname">Geigl</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/063032 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node17292" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Diyendo Massilani </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot</span>, Epigenome and Paleogenome group, 15 rue Helene Brion, 75013 Paris, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diyendo%2BMassilani%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Massilani%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiyendo%2BMassilani%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Silvia Guimaraes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot</span>, Epigenome and Paleogenome group, 15 rue Helene Brion, 75013 Paris, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Silvia%2BGuimaraes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimaraes%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASilvia%2BGuimaraes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jean-Philip Brugal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CNRS, USR 3336 IFRA (Institut Frangais de Recherche en Afrique; Nairobi, Kenya) &#38; Aix-Marseille Universite</span>, UMR 7269 LAMPEA (Labo.Med.de Prehistoire, Europe-Afrique; Aix-en-Provence, France) Maison Mediterraneenne des Sciences de l’Homme, BP 674, 13094, Aix-en-Provence cedex 2, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Philip%2BBrugal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brugal%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Philip%2BBrugal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">E. Andrew Bennett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot</span>, Epigenome and Paleogenome group, 15 rue Helene Brion, 75013 Paris, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=E.%2BAndrew%2BBennett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bennett%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AE.%2BAndrew%2BBennett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Malgorzata Tokarska </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Mammal Research Institute Polish Academy of Sciences, Genetics and Evolution Department</span>, Waszkiewicza 1, 17-230 Bialowieza, <span class='nlm-country'>Poland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malgorzata%2BTokarska%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tokarska%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalgorzata%2BTokarska%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rose-Marie Arbogast </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CNRS/UMR 7044/MISHA</span>, 5 allee du General Rouvillois, 67083 STRASBOURG Cedex</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rose-Marie%2BArbogast%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Arbogast%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARose-Marie%2BArbogast%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gennady Baryshnikov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Zoological Institute, Russian Academy of Sciences</span>, 199034 Saint Petersburg, <span class='nlm-country'>Russia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gennady%2BBaryshnikov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baryshnikov%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGennady%2BBaryshnikov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Gennady Boeskorov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Diamond and Precious Metal Geology Institute of the Siberian Branch of the RAS</span>, Yakutsk, <span class='nlm-country'>Russia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gennady%2BBoeskorov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boeskorov%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGennady%2BBoeskorov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jean-Christophe Castel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Museum d’histoire naturelle de Geneve (MHN), Departement d’Archeozoologie</span>, route de Malagnou 1, 1208 Geneve, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Christophe%2BCastel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Castel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Christophe%2BCastel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Sergey Davydov </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>North-East Science Station, Pacific Institute of Geography</span>, Far East Branch, Russ. Ac. Sci., 678830 Cherskiy, <span class='nlm-country'>Russia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sergey%2BDavydov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davydov%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASergey%2BDavydov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Stephane Madelaine </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Musee national de Prehistoire</span>, 24620 Les Eyzies de Tayac-Sireuil, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephane%2BMadelaine%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Madelaine%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephane%2BMadelaine%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Olivier Putelat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Pole d’Archeologie Interdepartmental Rhenan, 67600 Selestat</span>, France and UMR 7041 ArScan-Archeologies environnementales-Maison de l’Archeologie et de l’Ethnologie, 92023 Nanterre, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BPutelat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Putelat%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlivier%2BPutelat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Natalia Spasskaya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Zoological Museum of Moscow Lomonosow, State University</span>, Bolshaya Nikitskaya Str., 6, Moscow 125009 <span class='nlm-country'>Russia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2BSpasskaya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spasskaya%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANatalia%2BSpasskaya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Hans-Peter Uerpmann </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institut fur Ur-und Fruhgeschichte und Archaologie des Mittelalters, Abteilung Altere Urgeschichte und Quartarokologie</span>, Zentrum fur Naturwissenschaftliche Archaologie, Rumelinstr. 23, 72070 Tubingen, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans-Peter%2BUerpmann%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Uerpmann%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans-Peter%2BUerpmann%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Thierry Grange </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot</span>, Epigenome and Paleogenome group, 15 rue Helene Brion, 75013 Paris, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thierry%2BGrange%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Grange%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThierry%2BGrange%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Eva-Maria Geigl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institut Jacques Monod, UMR7592, CNRS, University Paris Diderot</span>, Epigenome and Paleogenome group, 15 rue Helene Brion, 75013 Paris, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eva-Maria%2BGeigl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Geigl%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEva-Maria%2BGeigl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/063032v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:17292" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:17292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/063032v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:17292" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:17292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/063032v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:17292" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:17292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/063032v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:17292" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:17292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/063032v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:17292" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:17292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/063032v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:17292" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:17292/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-5">Climatic and environmental fluctuations as well as anthropogenic pressure have led to the extinction of much of Europe’s megafauna. Here we show that the emblematic European bison has experienced several waves of population expansion, contraction and extinction during the last 50,000 years in Europe, culminating in a major reduction of genetic diversity during the Holocene. Fifty-seven complete and partial ancient mitogenomes from throughout Europe, the Caucausus and Siberia reveal that three populations of wisent (<em>Bison bonasus</em>) and steppe bison (<em>B. priscus</em>) alternated in Western Europe correlating with climate-induced environmental changes. The Late Pleistocene European steppe bison originated from northern Eurasia whereas the modern wisent population emerged from a refuge in the southern Caucasus after the last glacial maximum. A population overlap in a transition period is reflected in ca. 36,000 year-old paintings in the French Chauvet cave. Bayesian analyses of these complete ancient mitogenomes yielded new dates of the various branching events during the evolution of Bison and its radiation with Bos that lead us to propose that the genetic affiliation between the wisent and cattle mitogenomes result from incomplete lineage sorting rather than post-speciation gene flow.</p><div id="sec-1" class="subsection"><p id="p-6"><strong>Significance</strong> Climatic fluctuations during the Pleistocene had a major impact on the environment and led to multiple megafaunal extinctions. Through ancient DNA analyses we decipher these processes for one of the largest megafauna of Eurasia, the bison. We show that Western Europe was successively populated during the Late Pleistocene by three different bison clades or species originating from the Caucasus and North-Eastern Europe that can be correlated to major climatic fluctuations and environmental changes. Aurignacian cave artists were witnesses to the first replacement of bison species ~35,000 years ago. All of these populations went extinct except for one that survived into the Holocene where it experienced severe reductions of its genetic diversity due to anthropogenic pressure.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-7">Drastic climatic fluctuations during the Pleistocene in the northern hemisphere led to population contractions, extinctions, reexpansions and colonizations of fauna and flora (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>). Bison, along with other large ungulates thrived during the Middle and Late Pleistocene (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). Numerous cave paintings and engravings in France and Spain, such as those in the caves of Chauvet, Lascaux and Altamira, attest to the important role this impressive animal played for the late Paleolithic hunter-gatherers. The steppe bison (<em>Bison priscus</em> (Bojanus, 1827)) appears in the fossil record during the Early Middle Pleistocene, replacing another archaic but smaller forest-adapted bison (B. <em>schoetensacki(<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>)</em>), which went extinct ca. 700 kiloyears ago (kya) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>). Since <em>B. priscus</em> was adapted to the cold tundra-steppe, occurrence of its remains is considered indicative of open environments (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). It roamed over Europe and Asia, and also crossed the Bering Strait during the Middle Pleistocene to populate North America where it evolved into the American Bison <em>B. bison</em> (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). The numerous fossil remains display a pronounced sexual dimorphism, and a large initial body size, gradually decreasing throughout the Pleistocene (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>). Differences in morphology related to climatic, environmental and topographic conditions have led several authors to propose a high diversity for the Pleistocene cold-steppe bison expressed as subspecies or ecomorphotypes (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>).</p><p id="p-8">The taxonomy, evolutionary history and paleobiogeography of the genus <em>Bison</em> in Eurasia, and of the European bison or wisent <em>B. bonasus</em> (Linnaeus, 1758) in particular, is still patchy, despite a rich fossil record and its current endangered status (e.g., (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>)). Indeed, two opposing hypotheses on the evolution of bison in Eurasia coexist (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>). Traditionally, it has been considered that bison developed within one single phylogenetic line (B. <em>schoetensacki-B. priscus-B. bonasus</em>), but it has also been proposed that at least two parallel lines of bison existed, one being the line of forest bison from <em>B. schoetensacki</em> (Freudenberg, 1910) to the recent <em>B. bonasus</em> and the other being the line of the steppe bison <em>B. priscus</em> (for a review see (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>)). Thus, the phyletic relationships between <em>B. shoetensacki, B. priscus</em> and <em>B. bonasus</em>, as well as the approximate date and geographical origin of the wisent remain elusive, due in part to the limited power of paleontological studies to resolve species-level mammalian taxonomy issues or detect broad-scale genetic transitions at the population level (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>).</p><p id="p-9"><em>B. priscus</em> disappeared from the fossil record of Western Europe at the end of the Pleistocene, around 12-10 kya, and relict populations of <em>B. priscus</em> seem to have survived until the beginning of the Middle Holocene (7-6 kya) in Siberia (i.e., (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>)). In Europe, <em>B. priscus</em> is believed to have been replaced at the end of the Pleistocene or during the Holocene by the morphologically (eidonomically) distinguishable wisent, <em>B. bonasus</em> (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>). At least two sub-species are recognized: (i) <em>B. b. bonasus</em> Linnaeus, 1758 from the Lithuanian lowland and the Polish Biatowieza ecosystem, and (ii) the Caucasian highland <em>B. b. caucasicus</em> (Turkin and Satunin, 1904). (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). <em>B. priscus</em> was adapted to forest-steppe and steppe, and <em>B. bonasus</em> to forest and mountain-forest environments. <em>B. priscus</em> and <em>B. bonasus</em> are anatomically much closer to each other than to other more ancient bison, such as <em>B. schoetensacki. B. bonasus</em> has a relatively more massive rear quarter and shorter horns compared to <em>B. priscus</em> which has longer and slightly curved horns and a smooth double-humped appearance (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>). <em>B. priscus</em> and <em>B. bison</em> (Linnaeus, 1758), which are grazers, have a lower head position than <em>B. bonasus</em>, which is a mixed feeder (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>). It is, however, very difficult to assign fossil bison bones to either species (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">2</a>). Cave paintings of bison from caves in France and Spain are often classified as belonging to either <em>B. priscus</em> or to <em>B. bonasus</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). The diversity of the cave art depictions and the large range of their occurrence is interpreted as indicating an origin of <em>B. bonasus</em> in the area between southern Europe and the Middle East and of its existence well before the end of the Late Pleistocene at a time when <em>B. priscus</em> was still present (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>).</p><p id="p-10">Both extant bison species narrowly escaped extinction. The American <em>B. bison</em> was almost wiped out during the 19<sup>th</sup> century through commercial hunting and slaughter, but also introduced bovine diseases and competition with domestic livestock (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). The wisent as well almost went extinct at the beginning of 20<sup>th</sup> century. Indeed, similar to other large herbivores, such as the aurochs, intensification of agriculture since the Neolithic pushed the wisent into the forests of Eastern Europe (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>) where it was strictly protected for a few centuries as a royal game animal (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>). During the First World War, however, diminished population size followed by poaching led to its extinction in the wild (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>). Of the 54 remaining wisents living in captivity at the beginning of the 1920s, the descendants of just 12 animals constitute the entire extant population (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>).</p><p id="p-11">The wisent is still poorly characterized genetically. While genetic markers from the autosomes and Y chromosomes of American bison and wisent are closer to each other than to the other members of the genus <em>Bos</em> and they can reproduce and give rise to fertile offspring, their mitochondrial genomes are phylogenetically separated (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>). Indeed, mitochondrial sequences of the American bison and the yak <em>Poephagus mutus</em> f. <em>grunniens</em> (Linnaeus, 1758) form a distinct cluster while the wisent occupies a phylogenetic position closer to <em>Bos primigenius</em> f. <em>Taurus</em> (Linnaeus, 1758), a phenomenon that has been explained by incomplete lineage sorting or ancient hybridization (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>). European, Siberian and American <em>B. priscus</em> mitogenomes were shown to be phylogenetically closer to <em>B. bison</em> than to <em>B. bonasus</em> (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>).</p><p id="p-12">Ancient DNA (aDNA) studies have the potential to better resolve taxonomy than paleontological studies, in particular at the species level, and have revealed a far more dynamic picture of megafaunal communities, biogeography and ecology, including repeated localized extinctions, migrations, and replacements (e.g., (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>) and citations therein). To better understand the phylogeography and evolution of the wisent and to reconstruct its origin, we performed a study of the mitochondrial genome over the past 50,000 years (50 kyrs) across Europe and Asia. Using Pleistocene skeletal remains from Siberia (Yakutia), the Caucasus and Western Europe (France, Switzerland) as well as Neolithic and Medieval samples from France, and also pre-bottleneck <em>B. bonasus</em> samples from the early 20<sup>th</sup> century from the Polish lowland and the Caucasian highland lines, we constructed a phylogenetic framework based on both the hypervariable region of the mitochondrial DNA as well as complete mitochondrial genomes of selected specimens. Our results give new insights into the climate-driven dynamics of the bison in Europe.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>The evolution of the mitochondrial hypervariable region</h3><p id="p-13">We analyzed 57 paleontological and 25 historical specimens (see SI Table 1) targeting the mitochondrial hypervariable regions with four fragments of at most 150 bp. We obtained a 367 bp-long sequence from 43 specimens, and smaller sequences from 13 additional ones (SI Fig. 1). A Maximum Likelihood phylogenetic tree was constructed using these sequences (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). Three clades can be clearly distinguished. The first clade <em>Bp</em>, which is more divergent from the other two, comprises samples from Pleistocene Yakutia (Siberia) and from southern France (Berbie and lower stratigraphic layers of Gral), dating from 44 to 26 kya and from 39 to 15 kya, respectively. Clade <em>Bp</em> corresponds to the <em>B. priscus</em> lineage previously described for Siberia, North America and Europe (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>). The French and Siberian sequences of this clade are phylogenetically close and lack a phylogeographic structure. This reveals that a relatively homogeneous population of steppe bison was distributed during the Late Pleistocene not only in Siberia and northern America but also throughout the entire northern part of the Eurasian continent up to its most-western part, France.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maximum Likelihood phylogeny of the ancient Bison hypervariable region ML analyses of the HVR produced in this study using PHYML, a HKY+I+G substitution model and 500 bootstraps. The bootstrap support of the nodes is indicated in red. The geolocalization of the analyzed samples is represented using a color code to distinguish five origins and time periods as represented on the Eurasiatic continent map (see SI. Fig.4 for a map of the distribution of the Western European sites). Three clades can be clearly distinguished, the Bison priscus (Bp) clade and two Bison bonasus clades (Bb1 and Bb2). The scale corresponds to the number of nucleotide substitutions per site. The samples that have allowed amplification of all 4 PCR fragments as represented in SI fig. 1 are indicated by the suffix _C whereas those for which one or more fragments were missing are indicated by the suffix _NC. The stars indicate the samples that were used for the full mitogenome analysis presented in Fig. 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905689778" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Maximum Likelihood phylogeny of the ancient Bison hypervariable region&lt;/span&gt; ML analyses of the HVR produced in this study using PHYML, a HKY+I+G substitution model and 500 bootstraps. The bootstrap support of the nodes is indicated in red. The geolocalization of the analyzed samples is represented using a color code to distinguish five origins and time periods as represented on the Eurasiatic continent map (see SI. Fig.4 for a map of the distribution of the Western European sites). Three clades can be clearly distinguished, the &lt;em&gt;Bison priscus (Bp&lt;/em&gt;) clade and two &lt;em&gt;Bison bonasus&lt;/em&gt; clades (Bb1 and Bb2). The scale corresponds to the number of nucleotide substitutions per site. The samples that have allowed amplification of all 4 PCR fragments as represented in SI fig. 1 are indicated by the suffix _C whereas those for which one or more fragments were missing are indicated by the suffix _NC. The stars indicate the samples that were used for the full mitogenome analysis presented in Fig. 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F1.medium.gif" width="440" height="313"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1" src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F1.medium.gif" width="440" height="313"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1</span> <span class="caption-title">Maximum Likelihood phylogeny of the ancient Bison hypervariable region</span><p id="p-14" class="first-child">ML analyses of the HVR produced in this study using PHYML, a HKY+I+G substitution model and 500 bootstraps. The bootstrap support of the nodes is indicated in red. The geolocalization of the analyzed samples is represented using a color code to distinguish five origins and time periods as represented on the Eurasiatic continent map (see SI. Fig.4 for a map of the distribution of the Western European sites). Three clades can be clearly distinguished, the <em>Bison priscus (Bp</em>) clade and two <em>Bison bonasus</em> clades (Bb1 and Bb2). The scale corresponds to the number of nucleotide substitutions per site. The samples that have allowed amplification of all 4 PCR fragments as represented in SI fig. 1 are indicated by the suffix _C whereas those for which one or more fragments were missing are indicated by the suffix _NC. The stars indicate the samples that were used for the full mitogenome analysis presented in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">The two other clades, <em>Bb1</em> and <em>Bb2</em>, are more closely related to the extant wisent <em>B. bonasus.</em> The <em>Bb1</em> clade, significantly divergent from <em>Bb2</em>, contains most of the specimens from the Mezmaiskaya cave Northwest of the Caucasus in Russia, dating from ca. 48 to 43 kya, as well as seven specimens from southern France (Arquet, Plumettes), dating from ca. 47 to 34 kya. Thus, the geographical range of the <em>Bb1</em> clade stretched from the northern Caucasus to the south of France at its latest, from 48 to 34 kya during the overall mild Marine Isotope Stage (MIS) 3 (ranging from 57 to 29 kya (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>), <a href="http://www.lorraine-lisiecki.com/LR04MISboundaries.txt">http://www.lorraine-lisiecki.com/LR04MISboundaries.txt</a>). None of the haplotypes of the <em>Bb1</em> clade was found in the more recent specimens, suggesting that the corresponding population may have been the first to disappear. This <em>Bb1</em> population in France was apparently replaced by the steppe bison of the <em>Bp</em> clade around 30 kya, the latter remaining there throughout the glacial period of MIS2 (ranging from 29 to 14 kya (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>), <a href="http://www.lorraine-lisiecki.com/LR04MISboundaries.txt">http://www.lorraine-lisiecki.com/LR04MISboundaries.txt</a>).</p><p id="p-16">Clade <em>Bb2</em> includes both ancient specimens from the Caucasus and Western Europe as well as all recent and extant <em>B. bonasus</em> specimens. Nearly all ancient samples belonging to this clade are distinct from the more recent populations and include a ca. 49 ky-old specimen from the Mezmaiskaya cave, two specimens from the Kudaro cave in the central part of the southern slope of the Greater Caucasus dated at 38 and 22 kya, and specimens from Western Europe: one specimen from the Kesslerloch cave (Switzerland) dated at 14 kya, and, in France, two bison from the upper stratigraphic sequence of the Gral dated at 12 kya, two 5.2kyr-old bison samples from the Neolithic site of Chalain, as well as three medieval (7<sup>th</sup>-8<sup>th</sup> century CE) specimens from Alsace. The members of this clade represent the western European Pleistocene-Holocene lineage of <em>B. bonasus</em> and display a high mitochondrial diversity. This lineage appears to have replaced the <em>B. priscus</em> lineage, at least in France, at the end of the Upper Pleistocene between 15 and 12 kya, coinciding with the onset of a more temperate climate, and to have persisted in France up to the Middle Ages. Apart from the sequence found in the sample from Kesslerloch (12.2 kya), none of the ancient Bb2 sequences are present in the extant mitochondrial gene pool.</p><p id="p-17">Within the <em>Bb2</em> clade, a compact group of closely related sequences, comprise the Upper Pleistocene Kesslerloch specimen and the 1898-1917 pre-bottleneck wisents from both Poland and the Caucasus that almost became extinct at the end of the First World War. Five out of 24 of these prebottleneck bison have a hypervariable region (HVR) sequence identical to that of extant <em>B. bonasus</em>, whereas the rest differ from the extant sequence by only one or two single nucleotide polymorphisms (SNPs). Thus, the pre-bottleneck mitochondrial diversity appears only slightly higher than at present and much lower than that observed in older samples. The 14 kyr-old Kesslerloch specimen reveals the first occurrence of the mitogenome lineage of the extant <em>B. bonasus</em> population. Thus, the modern population corresponds to a minor fraction of the diversity that was present in Europe during the Late Pleistocene when <em>B. bonasus</em> replaced <em>B. priscus.</em></p><p id="p-18">A major reduction in the intrapopulation diversity is apparent from the Late Pleistocene to the early 20<sup>th</sup> century (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The Pleistocene populations of Siberia, the Caucasus and Europe are characterized by a high diversity at both the haplotype (H=1.00) and nucleotide levels (as estimated with Pi and various Theta estimators), the nucleotide diversity of the European <em>B. priscus</em> being lower than that of its Siberian population. The Western European Holocene population of <em>B. bonasus</em> experienced a major reduction of its diversity between the Middle Ages and the beginning of the 20<sup>th</sup> century.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/218499/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/218499/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/218499" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <div class="sb-div caption-clear"></div></div></div><p id="p-19">Calculations relied only on the samples that yielded the four PCR fragments</p><p id="p-20">We performed a phylogenetic analysis of the HVR under the Bayesian framework estimating mutation and population history parameters from temporally spaced sequence data using all dated ancient and modern bison HVR sequences present in GenBank in 2015 in combination with the data from the current study (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). The resulting tree shows a bifurcation between the <em>Bison bonasus</em> and <em>Bison priscus/bison</em> mitogenome lineages about 1.0 Mya [95% highest posterior density interval (HPD): 1. 5−0.7] (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig 2</a>). The most recent common ancestor (MRCA) of the Eurasiatic and the American <em>B. priscus</em> clades is estimated here at 151 kya [193–119], which is in agreement with a previous estimate of 136 ky [164-111] that was also based on the HVR (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>). <em>B. priscus</em> from America and Siberia group in distinct clades, and the modern American bison descended from a small subgroup of a once diverse American population, as previously proposed (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>). The Late Pleistocene European <em>B. priscus</em> corresponds to a subset of the Siberian population.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bayesian phylogeny of Bison hypervariable region All dated and complete ancient Bison sequences produced here and in a previous B. priscus analysis (7) were aligned and reduced to the 367 bp sequence targeted herein alongside modern B. bonasus and B. bison sequences. A Bayesian phylogenetic analysis was performed using Beast to estimate the age of the nodes from temporally spaced sequence data. The age of the nodes (in kya) is indicated in red, whereas the blue bars represent the 95% HPD of these ages. The color code representing the origin of the various samples is as indicated. The inset on the lower part of the figure represents a magnified view of the B. bonasus branches of the tree. The posterior probability of the nodes is indicated in blue and the thickness of the branches is proportional to this posterior probability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905689778" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Bayesian phylogeny of Bison hypervariable region&lt;/span&gt; All dated and complete ancient Bison sequences produced here and in a previous &lt;em&gt;B. priscus&lt;/em&gt; analysis (7) were aligned and reduced to the 367 bp sequence targeted herein alongside modern &lt;em&gt;B. bonasus&lt;/em&gt; and &lt;em&gt;B. bison&lt;/em&gt; sequences. A Bayesian phylogenetic analysis was performed using Beast to estimate the age of the nodes from temporally spaced sequence data. The age of the nodes (in kya) is indicated in red, whereas the blue bars represent the 95% HPD of these ages. The color code representing the origin of the various samples is as indicated. The inset on the lower part of the figure represents a magnified view of the &lt;em&gt;B. bonasus&lt;/em&gt; branches of the tree. The posterior probability of the nodes is indicated in blue and the thickness of the branches is proportional to this posterior probability.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F2.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2" src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F2.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2</span> <span class="caption-title">Bayesian phylogeny of Bison hypervariable region</span><p id="p-21" class="first-child">All dated and complete ancient Bison sequences produced here and in a previous <em>B. priscus</em> analysis (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">7</a>) were aligned and reduced to the 367 bp sequence targeted herein alongside modern <em>B. bonasus</em> and <em>B. bison</em> sequences. A Bayesian phylogenetic analysis was performed using Beast to estimate the age of the nodes from temporally spaced sequence data. The age of the nodes (in kya) is indicated in red, whereas the blue bars represent the 95% HPD of these ages. The color code representing the origin of the various samples is as indicated. The inset on the lower part of the figure represents a magnified view of the <em>B. bonasus</em> branches of the tree. The posterior probability of the nodes is indicated in blue and the thickness of the branches is proportional to this posterior probability.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/218404/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/218404/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/218404" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <div class="sb-div caption-clear"></div></div></div><p id="p-22">The phylogenetic analyses of the <em>B. bonasus</em> haplotypes reveal that the ancient Caucasian population had a deep root and was highly diverse. Based on the HVR, the age of the MRCA of the <em>Bb1</em> and <em>Bb2</em> clades is estimated at 438 [643–284] kya, which is significantly older than the MRCA of <em>B. priscus</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2</a> and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). This observation indicates that the Upper Pleistocene Caucasian population had a deeper root than the contemporaneous and more northern <em>B. priscus</em> population, even though the latter one may have occupied in the past a much larger territory from Western Europe to the American continent. The <em>Bb2</em> clade encompasses several branching events, the separation of a branch represented by a ca. 50 kyr-old northern Caucasus specimen from Mezmaiskaya occurring first, i. e., 105 [158–75] kya, followed, 61 [89–43] kya, by the separation of a branch represented by a ca. 37.8 kyr-old specimen from the Kudaro cave in the southern Caucasus. Then, 42 [60–26] kya, the subgroup comprising an 22.2 ky-old specimen from the Kudaro cave as well as all French specimens between 12.4 to ca. 1.2 kya separated from the haplogroup encompassing the modern <em>B. bonasus</em> sequences. Finally, in this later haplogroup, the 14 ky-old Kesslerloch specimen and the early 20<sup>th</sup> century Central European and Caucasus specimens have a MRCA estimated at 16 [33-14] kya. This sequential order of the radiation events and the previously mentioned reduction of the population diversity indicates that at the transition between the Pleistocene and the Holocene, Western and Central Europe were populated by a subset of a wisent population established in an area of southern Asia including the Caucasus, and that only a part of this population survived into the 20<sup>th</sup> century.</p></div><div id="sec-5" class="subsection"><h3>The evolution of the Bison clades based on entire mitogenomes</h3><p id="p-23">To render this phylogeny and dating scenario more robust, we used biotinylated RNA probes synthesized from the complete mitogenome of <em>B. p. taurus</em>, and optimized a sequence capture approach to recover complete mitogenome sequences from seven well-preserved 50 to 12 kyr-old specimens representative of the various clades and radiation events defined above. We performed a Bayesian phylogenetic analysis of these mitogenomes and of two recently published <em>B. priscus</em> mitogenomes (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">23</a>) together with those of modern <em>B. bison</em> and <em>B. bonasus, B. p. taurus</em>, and <em>P. grunniens</em> complete mitogenomes present in GenBank in 2015 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>). As previously observed with modern sequences (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">22</a>), the <em>B. bonasus</em> mitogenome lineage is more closely related to the <em>Bos p. taurus</em> lineage than to the <em>B. priscus-B. bison</em> lineages. The Bayesian analysis reveals, however, that there is a significant overlap (35 to 40%) of the 95% HPD intervals of the dates estimated for the node separating the <em>Bos p. taurus-B. bonasus</em> and the <em>B. priscus-B. bison</em> lineages, estimated here at 944 [1090-810] kya, and the node separating the <em>B. p. taurus</em> and <em>B. bonasus</em> lineages, estimated here at 786 [908-668] kya. Such an overlap suggests that the two bifurcation events may have occurred within a relatively short evolutionary period, thus increasing the likelihood that these two events preceded the major separation of the <em>B. p. taurus</em> and <em>Bison</em> species. This peculiar affiliation pattern of mitogenomes renders the incomplete lineage sorting hypothesis a parsimonious interpretation (see SI Fig. 2). The <em>Bison priscus/bison</em> and yak (<em>P. grunniens</em>) mitogenome lineages separated from a common ancestor dated at ca. 328 kya [378-272]. For the <em>Bison</em> mitogenomes, the dates of the nodes estimated with complete mitogenomes are often younger than those estimated using only the HVR (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). For instance, the common ancestor of the <em>Bb1</em> and <em>Bb2</em> clades is estimated at 250 kya [287211] when comparing full mitochondrial genomes, rather than 438 kya [643-284] calculated from only HVR sequences. Similarly, the common ancestor of the Eurasiatic <em>B. priscus</em> and the modern American bison is estimated at 115 [135-95] kya rather than 151 [193-119] kya. In contrast, the various branching events within the <em>Bb2</em> clade show non-significant differences (given the overlap of the HPD) between the two series of date estimations: 87 [97-75] kya instead of 105 [158-75] kya for the ancestor of the North and South Caucasian specimens, 63 [70-52] kya instead of 61 [89-43] kya for the ancestor of the South Caucasus and Holocene European specimens, and 37 [42-29] kya instead of 42 [60-26] kya for the ancestor of the Holocene European specimens. The discrepancies between the date estimates for several nodes depending on the genetic region used may be partly due to the differences in the number of individual sequences studied in the two series. The major source of discrepancy, however, appears to be due to an irregular rate of evolution of the HVR. Indeed, even though the clock rates estimated for the HVR are similar when either the HVR alone or as a partition of the complete mitogenome is used (respectively 5.4 [3.9−6.9] and 4.6 [3.7−5.6] E-7 per site × year), the individual HVRs show distinct evolutionary rates. When the distribution throughout the mitogenome of the SNPs distinguishing individual bison mitogenomes from the <em>B. p. taurus</em> mitogenome as an outgroup are compared, the number of SNPs accumulated in the HVR can vary up to two-fold between individual sequences whereas the rest of the mitogenome is equally distant to the outgroup (SI Fig. 3). For example, there are twice as many mutations accumulated in the HVR subregion (15900-16100) of the Mez 128, Gral 232 and Yaku 118 mitogenomes than in the ones of the Mez130 and Gral 125 specimens. Whatever the underlying mechanism responsible for these differences of the evolutionary rate of this particular region of the mitogenome, this phenomenon limits the reliability of the dating estimations based solely on the HVR.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bayesian phylogeny of complete mitogenomes of Bos and Bison. We used the complete mitogenomes of ancient Bison obtained herein as well as the two published B. priscus mitogenomes, and all modern Bison bison, Bison bonasus, Poephagus grunniens, and Bos primigenius taurus mitogenomes available in Genbank in 2015 totaling 439 sequences. The B. bison, P. grunniens and B. p. taurus sequences have been collapsed to preserve only their subclade structure. The estimate of the age of the nodes, in kya, are indicated in red, with the 95% HPD indicated by blue bars. The statistical supports of the nodes are indicated in blue (Bayesian posterior probability) and in green (bootstrap values of a ML phylogeny performed using RaXML)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905689778" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Bayesian phylogeny of complete mitogenomes of &lt;em&gt;Bos&lt;/em&gt; and &lt;em&gt;Bison&lt;/em&gt;.&lt;/span&gt; We used the complete mitogenomes of ancient &lt;em&gt;Bison&lt;/em&gt; obtained herein as well as the two published &lt;em&gt;B. priscus&lt;/em&gt; mitogenomes, and all modern &lt;em&gt;Bison bison, Bison bonasus, Poephagus grunniens&lt;/em&gt;, and &lt;em&gt;Bos primigenius taurus&lt;/em&gt; mitogenomes available in Genbank in 2015 totaling 439 sequences. The &lt;em&gt;B. bison&lt;/em&gt;, &lt;em&gt;P. grunniens&lt;/em&gt; and &lt;em&gt;B. p. taurus&lt;/em&gt; sequences have been collapsed to preserve only their subclade structure. The estimate of the age of the nodes, in kya, are indicated in red, with the 95% HPD indicated by blue bars. The statistical supports of the nodes are indicated in blue (Bayesian posterior probability) and in green (bootstrap values of a ML phylogeny performed using RaXML).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F3.medium.gif" width="440" height="197"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3" src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F3.medium.gif" width="440" height="197"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3</span> <span class="caption-title">Bayesian phylogeny of complete mitogenomes of <em>Bos</em> and <em>Bison</em>.</span><p id="p-24" class="first-child">We used the complete mitogenomes of ancient <em>Bison</em> obtained herein as well as the two published <em>B. priscus</em> mitogenomes, and all modern <em>Bison bison, Bison bonasus, Poephagus grunniens</em>, and <em>Bos primigenius taurus</em> mitogenomes available in Genbank in 2015 totaling 439 sequences. The <em>B. bison</em>, <em>P. grunniens</em> and <em>B. p. taurus</em> sequences have been collapsed to preserve only their subclade structure. The estimate of the age of the nodes, in kya, are indicated in red, with the 95% HPD indicated by blue bars. The statistical supports of the nodes are indicated in blue (Bayesian posterior probability) and in green (bootstrap values of a ML phylogeny performed using RaXML).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Complete mitogenomes confirm the observation made with the HVR sequences that the European <em>B. priscus</em> specimens from the Late Pleistocene were more homogeneous genetically than the contemporaneous Siberian population. Indeed, the two French mitogenomes from the Gral (Gral232) and Trois-Freres cave (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">23</a>) are very similar, with only 19 SNPs distinguishing them, whereas the two Siberian mitogenomes (Yaku118 and Rauchua (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">14</a>)) are much more divergent with eight-fold more SNPs distinguishing them (153 SNPs). Strikingly, the age of the MRCA of the 26.7 kyr-old Yakutian sample and the 20 to 15 kyr-old French <em>B. priscus</em> samples is estimated at 58 [66-47] kya. Since the MRCA of these sequences is also the MRCA of the group of HVR sequences comprising two additional Yakutian and all other French <em>B. priscus</em> samples (see <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>), this indicates that the steppe bison inhabiting France between 39 and 15 kya originated from a migration from North-East Eurasia that occurred not earlier than 58 [66-47] kya. Presumably, the low genetic diversity of the Western European <em>B. priscus</em> population is due to a founder effect during this migration.</p><p id="p-26">The phylogenetic relationships between the <em>B. bonasus</em> mitogenomes indicate that modern wisent in central Europe are more closely related to the Late Pleistocene/Holocene Western European population than to the Late Pleistocene Caucasian population. Indeed, the 14 kyr-old specimen from the site of Kesslerloch at the Swiss-German border is closely related to modern wisent, with only 15 SNPs distinguishing the ancient and modern mitogenome sequences. This demonstrates that the range of the ancestral population of the modern wisent encompassed Middle Europe 14 kya. The Late Pleistocene Caucasian population was highly diverse with a deep root separating most of the specimens from the Mezmaiskaya cave in the northern Caucasus, belonging to the <em>Bb1</em> clade, from the specimens from the Kudaro cave in the southern Caucasus, belonging to the <em>Bb2</em> clade, and including a specimen from the northern Caucasus: 342 SNPs distinguished the Mez128 (Bb1) and the Mez130 (Bb2) mitogenomes and 105 SNPs distinguished the northern (Mez 130) and southern Caucasian (Kud136) mitogenomes of the <em>Bb2</em> clade. In contrast, the members of the <em>Bb2</em> clade in Western Europe were more similar, with only 39 SNPs distinguishing the French (Gral125) and Swiss (KSL) specimens, in agreement with the reduction of diversity that we observed when comparing the HVR of the Western European and Caucasian <em>B. bonasus</em> populations. Thus, complete mitogenomes confirm the observations from the HVR analyses of a larger sample size while providing a more accurate phylogenetic analysis, in particular with respect to the dating of the MRCA of the various mitogenomes.</p></div></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><div id="sec-7" class="subsection"><h3>European bison populations turnover during the late Pleistocene and the Holocene</h3><p id="p-27">Our results enable us to propose a scenario for the evolutionary history of the bison in Europe that is related to the climatic fluctuations and the resulting environmental changes of the Late Pleistocene and the Pleistocene/Holocene transition as summarized in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. We observe striking regional and temporal differences in the major clades and distinguish three periods, particularly in France. The first period, from at least 47 kya to about 34 kya, was characterized by the dominance of a divergent <em>B. bonasus</em> lineage belonging to the <em>Bb1</em> clade in both southern France (Arquet and Plumettes, 7 <em>Bb1</em> out of 7 samples) and the northern Caucasus (Mezmaiskaya, 5 <em>Bb1</em> and 1 <em>Bb2</em> out of 6 samples). This lineage was absent from the samples from later periods indicating that the corresponding population was the first to disappear. For the same time period, the steppe bison mitotype <em>Bp</em> was the sole mitotype found in Siberia and northern Eurasia (27/27 samples dated from 66 to 34 kya (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">7</a>)). This period, encompassing most of MIS3, is characterized by oscillating shorter glacial and longer interstadial periods, the latter lasting more than one thousand years (glacial interstadial GI15 to GI8 (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>) and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>). During the warmer periods, southern France and the northern Mediterranean coast were covered by deciduous coniferous non-continuous forests, and central France and central Europe by coniferous open woodland (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>). The steppe bison <em>B. priscus</em> was dominant in France during the following period that lasted until the end of the Last Glacial Maximum (LGM) at 14.7 kya (7 out of 7 samples, plus 1 out of 1 in (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">23</a>)). These bison were closely related to the Siberian sample from Yakutia from the same period, the whole mitogenome of which we analyzed (99.4% identity between the Yaku128 and Gral232 mitogenomes). HVR comparisons between the French and Siberian and North American samples (<a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">7</a>) reveal that the French <em>B. priscus</em> mitotypes are closely related to those of the Siberian samples (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2</a>), indicating that the Western European territory was colonized by a subpopulation of the steppe bison from the northern Eurasian continent. The age estimated from full mitogenomes for the MRCA of the French and Siberian samples is 56 [64-46] kya, indicating that this colonization involved a population that separated from the Siberian population more recently than ca. 56 kya and suggesting a lower limit for the arrival in France of these steppe bison. In contrast, specimens older than 60 kya assigned to <em>B. priscus</em> in the fossil record of Western Europe presumably must have belonged to a distinct population that was not the direct ancestor of this <em>B. priscus</em> population that occupied Western Europe during the cold spells of MIS2. At the end of MIS3, around 32 kya, the climate became colder on average and the warmer interstadials were shorter, lasting only a few hundred years. Then, between 27 and 14.7 kya, a second, long glacial period followed that comprised two phases. In the first phase, the tree cover was patchy and incomplete, with a high proportion of steppe vegetation, whereas the second, a full glacial phase, was characterized by sparse grassland and open steppe tundra in southern and northern Europe, respectively (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>). While during this period the steppe bison <em>B. priscus</em> occupied the territory previously occupied by <em>B. bonasus</em> in Western Europe, <em>B. bonasus</em> remained nevertheless present in the southern Caucasus, even during the LGM. Indeed, in our samples, two out of two specimens from the southern Caucasus belonged to the <em>Bb2</em> clade, which we found to be present at low frequency at an earlier period in the northern Caucasus. Finally, during the third period, starting at the end of the MIS2 and lasting up to the present, <em>B. bonasus</em> of the <em>Bb2</em> clade expanded again into Western Europe, as we detected it in the 14 kyr-old specimen from Switzerland at the beginning of the Bølling-Allerød interstadial period (14.7 to 12.7 kya). The more recent specimens from France belonged without exception to the <em>Bb2</em> clade (7 out of 7, dated between 12 kya to the Middle Ages). Strikingly, the sedimentary sequence of the French site Gral recorded a population replacement: all specimens older than ca. 15 kya, before the B0lling-Aller0d interstadial, belong to the <em>B. priscus Bp</em> mitotype (5 out of 5), whereas all more recent ones, coinciding with the onset of a more temperate climate at the end of the last glacial event (end of MIS2), belong to the <em>B. bonasus Bb2</em> mitotype (2 out of 2, Pval&lt;0.05 with Fisher’s test). The ancient French <em>Bb2</em> population is significantly different from the one comprising the modern wisent and the ancient Kesslerloch specimen (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1-3</a>). Moreover, this subclade was not detected later than the Middle Ages, suggesting that it went extinct in Western Europe with the disappearing local wisent. In contrast, the distinct <em>Bb2</em> mitotype of the 14 kyr-old sample from Kesslerloch continued to exist up to present time. It is the only remaining mitotype detected in both present-day wisent as well as in the specimens from Poland and the Caucasus from the beginning of the 20<sup>th</sup> century prior to the last major bottleneck of WWI.</p><p id="p-28">The marked climatic variations that occurred during the Pleistocene implied drastic environmental changes that triggered the profound reorganization of floral and faunal biomes at different geographic and temporal scales. Response of biota to climatic stimuli was regionally/continentally distinct, sometimes occurring synchronously but most often diachronously (e.g., (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>)). The repeated cyclical climate changes progressively promoted landscape renewal, locally modifying, obliterating and creating specific and varied ecological niches (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>). The various expansions and contractions of bison populations in response to these climatic fluctuations led to apparent alternating occupations of Western Europe by either <em>B. priscus</em> or <em>B. bonasus</em> during the last 50 kyrs, each presumably originating respectively from either a northeastern territory including Siberia, or a southern territory including the Caucasus. We hypothesize that these alternating occurrences result from climate-driven changes of two different habitats to which <em>B. priscus</em>, a grazer, and <em>B. bonasus</em>, a mixed feeder, were adapted, i.e., the open tundra-steppe for the first and open woodlands for the second. Indeed, the diet of <em>B. priscus</em> during the LGM included typical steppe and grassland (C<sub>3</sub>) vegetation and lichens (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>), whereas the wisent’s diet in the Holocene was more flexible and included a higher content of shrubs (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">18</a>). Similarly, on the American continent, <em>B. priscus</em> adapted to the climatic and environmental changes of the Holocene and evolved in two recently divergent forms, the plain bison thriving on the grasslands of the Great Plains and the wood bison inhabiting the boreal forest in North America. This suggests that on the Eurasian continent the competition with <em>B. bonasus</em>, a species seemingly better adapter to a more temperate environment, may have prevented a similar adaptation of <em>B. priscus</em> to habitat changes during the warmer periods. Finally, in Western Europe, local variations in ecological and physical barriers could have affected the speed of bison population turnover as a reaction to climatic shifts. In the future, a higher resolution genetic study involving a much higher sample number with denser time and space sampling may provide a more accurate and nuanced view of these population turnovers.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of the distribution through time and space of the various mitogenome clades The geographic regions are represented on the abscissa and the time scale on the ordinate. The ascertained presence of the various mitochondrial haplogroups are represented by solid boxes, whereas the dotted lines indicate possible temporal extension of the presence of these clades. The left side shows the climatic fluctuations as inferred from the North Greenland Ice Core Project (NGRIP) (25) and the combined Carabean Cariaco basin and NGRIP data as shown in (12), as well as the Marine Isotope Stage (MIS) as defined by Lisiecki and Raymo 24 (http://www.lorraine-lisiecki.com/LR04_MISboundaries.txt). The proposed migrations are indicated by solid arrows. The hatched arrow indicates a possible migration of the Bb1 clade that populated Western Europe from a southern refugee before the time period analyzed herein. The genetic identity of the bison that, according to the fossil record, populated Western Europe before 60 kya is not known, but climatic fluctuations may have triggered additional expansions and contractions of different populations of B. priscus and B. bonasus. (Drawings: E.-M. Geigl)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905689778" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Schematic representation of the distribution through time and space of the various mitogenome clades&lt;/span&gt; The geographic regions are represented on the abscissa and the time scale on the ordinate. The ascertained presence of the various mitochondrial haplogroups are represented by solid boxes, whereas the dotted lines indicate possible temporal extension of the presence of these clades. The left side shows the climatic fluctuations as inferred from the North Greenland Ice Core Project (NGRIP) (25) and the combined Carabean Cariaco basin and NGRIP data as shown in (12), as well as the Marine Isotope Stage (MIS) as defined by Lisiecki and Raymo &lt;sup&gt;24&lt;/sup&gt; (http://www.lorraine-lisiecki.com/LR04_MISboundaries.txt). The proposed migrations are indicated by solid arrows. The hatched arrow indicates a possible migration of the &lt;em&gt;Bb1&lt;/em&gt; clade that populated Western Europe from a southern refugee before the time period analyzed herein. The genetic identity of the bison that, according to the fossil record, populated Western Europe before 60 kya is not known, but climatic fluctuations may have triggered additional expansions and contractions of different populations of &lt;em&gt;B. priscus&lt;/em&gt; and &lt;em&gt;B. bonasus.&lt;/em&gt; (Drawings: E.-M. Geigl)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F4.medium.gif" width="440" height="333"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4" src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F4.medium.gif" width="440" height="333"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4</span> <span class="caption-title">Schematic representation of the distribution through time and space of the various mitogenome clades</span><p id="p-29" class="first-child">The geographic regions are represented on the abscissa and the time scale on the ordinate. The ascertained presence of the various mitochondrial haplogroups are represented by solid boxes, whereas the dotted lines indicate possible temporal extension of the presence of these clades. The left side shows the climatic fluctuations as inferred from the North Greenland Ice Core Project (NGRIP) (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>) and the combined Carabean Cariaco basin and NGRIP data as shown in (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>), as well as the Marine Isotope Stage (MIS) as defined by Lisiecki and Raymo <sup>24</sup> (<a href="http://creativecommons.org/licenses/by/4.0/">http://www.lorraine-lisiecki.com/LR04_MISboundaries.txt</a>). The proposed migrations are indicated by solid arrows. The hatched arrow indicates a possible migration of the <em>Bb1</em> clade that populated Western Europe from a southern refugee before the time period analyzed herein. The genetic identity of the bison that, according to the fossil record, populated Western Europe before 60 kya is not known, but climatic fluctuations may have triggered additional expansions and contractions of different populations of <em>B. priscus</em> and <em>B. bonasus.</em> (Drawings: E.-M. Geigl)</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The mitochondrial lineages of <em>B. bonasus</em> that were present 40 kya have an older root than those of <em>B. priscus</em> (250 [287-211] vs 115 [135-95] kya). This indicates that the <em>B. bonasus</em> did not experience as severe a bottleneck prior to 100 kya as the population reduction the steppe Bison experienced between 150 and 100 kya, before later thriving in northern Eurasia, Beringia, and in North America between 80 and 20 kya. All the various population expansions and contractions that occurred in response to the climatic and environmental changes characterizing the late Pleistocene and the transition to the Holocene gave rise to a reduction of the population diversity and to the extinction of lineages, like the <em>Bb1</em> lineages, which apparently was the first to disappeared, and the <em>Bp</em> lineage, which disappeared at the beginning of the Holocene on the Eurasian continent. Finally, the reduction of the diversity of the <em>Bb2</em> lineage seems to involve both migrations and local extinctions with the major and most severe reductions occurring between the Early Holocene and historic times, most likely owing to human impact through hunting pressure and habitat fragmentation.</p><p id="p-31">Numerous cave art representation in southern France and northern Spain, and also in the Caucasus suggest realistic depictions of both the steppe bison and the wisent (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">19</a>). We consider likely that the paintings of bison, the so-called “bison of the pillar”, in the cave Chauvet-Pont d’Arc in France depict the two types of bison distinguished by the shape of horns and back lines (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>). The upper image on the pillar, dated to 38.5-34.1 kya (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>), could represent a wisent, and the lower image, dated to 36.2 to 34.6 kya (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>), a steppe bison. These dates coincide with the period (39 to 34 kya) in which the two bison forms overlap in our dataset from southern France, in the vicinity of these paintings. Within this time frame, wisent population would have been in decline and steppe bison population would have been expanding.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prehistoric painting of bison in the cave of Chauvet-Pont d&#x2019;Arc, Ardeche, France The paintings are the so-called &#x201C;Bison of the pillar&#x201D; in the &#x201C;End Chamber&#x201D; of the Chauvet cave. The charcoal of both drawings have been radiocarbon dated at 38.5-34.1 kya for the upper bison, and at 36.3-34.6 kya for the lower bison (30). We consider, based on criteria stated by Spassov (19) that the &#x201C;great bison&#x201D; in the upper part represents B. bonasus with a highly positioned head, curved horns, a moderately large hump and a weak mane and rather equilibrated body proportions between the front and the rear. The lower part would represent B. priscus with its large hump, its low head position, its abundant mane, crescent-shaped horns, and, although somewhat faded in the image, the steep incline of the back-line and stronger hindquarters can be made out. (Copyright: French Ministry of culture, archeologie.culture.fr/chauvet. Cl. Arnaud Frich. CNP/MCC)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-905689778" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Prehistoric painting of bison in the cave of Chauvet-Pont d&#x2019;Arc, Ardeche, France&lt;/span&gt; The paintings are the so-called &#x201C;Bison of the pillar&#x201D; in the &#x201C;End Chamber&#x201D; of the Chauvet cave. The charcoal of both drawings have been radiocarbon dated at 38.5-34.1 kya for the upper bison, and at 36.3-34.6 kya for the lower bison (30). We consider, based on criteria stated by Spassov (19) that the &#x201C;great bison&#x201D; in the upper part represents &lt;em&gt;B. bonasus&lt;/em&gt; with a highly positioned head, curved horns, a moderately large hump and a weak mane and rather equilibrated body proportions between the front and the rear. The lower part would represent &lt;em&gt;B. priscus&lt;/em&gt; with its large hump, its low head position, its abundant mane, crescent-shaped horns, and, although somewhat faded in the image, the steep incline of the back-line and stronger hindquarters can be made out. (Copyright: French Ministry of culture, archeologie.culture.fr/chauvet. Cl. Arnaud Frich. CNP/MCC)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F5.medium.gif" width="440" height="418"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5" src="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F5.medium.gif" width="440" height="418"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2016/07/09/063032/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5</span> <span class="caption-title">Prehistoric painting of bison in the cave of Chauvet-Pont d’Arc, Ardeche, France</span><p id="p-32" class="first-child">The paintings are the so-called “Bison of the pillar” in the “End Chamber” of the Chauvet cave. The charcoal of both drawings have been radiocarbon dated at 38.5-34.1 kya for the upper bison, and at 36.3-34.6 kya for the lower bison (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>). We consider, based on criteria stated by Spassov (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">19</a>) that the “great bison” in the upper part represents <em>B. bonasus</em> with a highly positioned head, curved horns, a moderately large hump and a weak mane and rather equilibrated body proportions between the front and the rear. The lower part would represent <em>B. priscus</em> with its large hump, its low head position, its abundant mane, crescent-shaped horns, and, although somewhat faded in the image, the steep incline of the back-line and stronger hindquarters can be made out. (Copyright: French Ministry of culture, archeologie.culture.fr/chauvet. Cl. Arnaud Frich. CNP/MCC)</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Radiation of the <em>Bos</em> and <em>Bison</em> lineages</h3><p id="p-33">Since the initial classification of Linnaeus in 1758 of <em>Bos</em> and <em>Bison</em> within a single genus, there has been debate about whether they should be placed in separate genera (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). Indeed, the members of these genera can still be crossed, despite reduced fertility in some combinations of crosses, and <em>B. p.taurus</em> genetic material can be found in a number of present-day American bison and wisent (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). Thus, gene flow could have occurred between the various lineages, in particular in the early phases of their differentiation. Such gene flow between separated populations has already been considered as an alternative possibility to incomplete lineage sorting to explain that the mitogenome of <em>B p. taurus</em> was closer to that of <em>B. bonasus</em> than to that of <em>B. bison</em> (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">22</a>). The phylogenetic estimates of the age of the various common ancestors of these two lineages indicate, however, that the most parsimonious interpretation of incomplete lineage sorting suffices to account for the relative affiliations of these mitogenomes (see SI Fig. 2). Indeed, as mentioned above, the age estimates indicate that the MRCA of the <em>Bos</em> and <em>Bison</em> mitogenome lineages is not much older than that of the <em>B. p. taurus</em> and <em>B. bonasus</em> lineages, respectively 946 [1090-810] kya, and 787 [908-670] kya, with a 35 to 40% overlap of the confidence interval (95% HPD). This indicates that the two bifurcation events have occurred within a relatively short evolutionary period. Rapid speciation during this short evolutionary period appears unlikely since these species have still not yet totally lost interfertility almost a million years later. Our radiation date estimates are within the same range of the earliest fossils that were clearly attributed to the <em>Bos</em> genus and that are dated at 1 Mya (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). They are also consistent with the analysis of the complete genome of a modern wisent that estimated that the wisent and bovine species diverged between 1.7 to 0.85 Mya through a speciation process involving an extended period of limited gene flow with some more recent secondary contacts posterior to 150 kya (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Thus, incomplete lineage sorting of mitogenomes in a metapopulation of the <em>Bos</em> and <em>Bison</em> ancestors during the period of divergence of these species could account for the affiliation patterns of these mitogenomes without the need to postulate a more recent post-speciation gene flow. It is interesting to note that this radiation event could be coincident with the onset and intensification of high-latitude glacial cycles (100 kyr-periodicity) around 1.2-0.8 Mya. Incomplete lineage sorting, however, does not preclude later sporadic introgression of nuclear DNA at any point up to the present owing to the persistent interfertility between these species, as evidenced by the detection in the wisent genome of ancient gene flow from <em>B. p. taurus</em> (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>).</p></div></div><div class="section" id="sec-9"><h2 class="">Conclusion</h2><p id="p-34">The analysis of DNA preserved in ancient bison remains from Eurasia covering the last 50,000 years allowed us to retrace some of the population dynamics that took place during the Late Pleistocene and the Holocene, including population migrations, extinctions and replacements. We could trace back the origin of the wisent to Eastern Europe including the Caucasus. We observed several alternating expansion waves of wisent and steppe bison in Western Europe that were related with climatic fluctuations, the wisent being prevalent when the climate was more temperate leading to a more forested vegetation whereas the steppe bison population originating from northern Eurasia was predominant in Western Europe during the colder periods of the Late Pleistocene with their open environments. These fluctuations may have been recorded in Paleolithic cave paintings, in particular in the cave of Chauvet that had been occupied by humans over a long period and where two distinct bison types are depicted.</p></div><div class="section" id="sec-10"><h2 class="">Materials &amp; Methods</h2><p id="p-35">Details of samples are described in the Supplementary Information (SI Table 1 and SI Fig. 4). All pre-PCR procedures were carried out in the high containment facility of the Jacques Monod Institute physically separated from areas where modern samples are analyzed and dedicated exclusively to ancient DNA analysis using the strict procedures for contamination prevention previously described (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>).</p><div id="sec-11" class="subsection"><h3>DNA Extraction</h3><p id="p-36">The external surface of the specimens was removed with a sterile blade to minimize the environmental contamination. For each bone sample, roughly 0.2 g was ground to a fine powder in a freezer mill (Freezer Mill 6750, Spex Certiprep, Metuchen, NJ), which was then suspended in 2mL of extraction buffer containing 0.5M EDTA, 0.25M Di-sodium Hydrogen Phosphate (Na<sub>2</sub>HPO<sub>4</sub>), pH 8.0, 0.14M 2-mercaptoethanol and 0.25 mg/mL of proteinase K and incubated under agitation at 37°C for 48 hours. Blank extractions were carried out for each extraction series, eight in total.</p><p id="p-37">Samples were then centrifuged and the supernatant was purified with the Qiaquick PCR Purification Kit (Qiagen, Hilden, Germany), as described (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>).</p></div><div id="sec-12" class="subsection"><h3>PCR Amplification of the HVR</h3><p id="p-38">Primers were designed to target conserved regions with minimal primer dimer propensity against a multiple alignment of all the Bison mitochondrial sequences present in Genbank in 2012 using the software Oligo 7 as described (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>), and were then tested for efficiency and dimer formation using quantitative real-time PCR (qPCR) (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">39</a>). Four primer pairs amplifying short (118 to 152 bp) subregions of the mitochondrial Hyper Variable Region (HVR) were selected (SI Table 2 and SI Fig. 1). To protect against cross-contamination between samples, we used the UNG-coupled quantitative realtime PCR system (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>) and, to avoid the production of erroneous data due to the presence of bovine DNA in reagents, we decontaminated reagents as previously described (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>). Amplifications were performed in a final volume of 10 μL containing 2mM MgCl<sub>2</sub>, 1μM primers, 0.04mM of dA/G/CTPs and 0. 08mM of dUTP, 0,01U/μL of UNG, 1U/μL of FastStart Taq (Roche Applied Science) and 1 × qPCR home-made reaction buffer (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>). Blank amplification controls were included for each amplification. In total, 415 amplification blanks were carried out during the various amplifications analyses of 85 samples. No products were observed in any of the amplification blanks. Amplifications were performed using a LightCycler 1.5 (Roche Applied Science) using the following cycling program:15 minutes at 37°C (carry-over contamination prevention through digestion by UNG of dUTP-containing amplicons), 10 minutes at 95°C (inactivation of UNG and activation of the Fast Start DNA polymerase) followed by 60 cycles at 95°C for 15 seconds and 60°C for 40 seconds (for primer pairs Bon1, 2 and 3) and 95°C for 15 seconds, or 56°C for 15 seconds and 67°C for 20 seconds (for primer pair BB3r4m), followed by a final melting curve analysis step. All extracts were tested for inhibition as previously described (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). Each sequence was determined on both strands from at least two independent amplifications using capillary electrophoresis sequencing. Sequencing data analyses were performed using the software Geneious 6.1.8 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>).</p></div><div id="sec-13" class="subsection"><h3>Capture and sequencing of whole mitogenomes</h3><p id="p-39">Dual barcoded libraries for Illumina sequencing were constructed using a double-stranded procedure previously described ((<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>), see also Supplementary Methods). The End-repair and adapter ligation mixtures were decontaminated with Ethidium monoazide (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>) to inactivate bovine DNA associated with BSA present in these reagents that could interfere with the final results. For the production of complete mitogenome sequences, we developed a sequence capture approach using biotinylated RNA baits (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>), further detailed in the supplementary information. Briefly, we amplified the <em>Bos taurus</em> mitogenome by PCR targeting eleven 1.5 kb-long fragments using primer pairs, one of which contained the T3 promoter as a 5’extension. Strand-specific biotinylated RNA probes were synthesized, pooled and submitted to mild, controlled heat-induced hydrolysis to generate RNA fragments ranging from 100 to 600 nucleotides with an average size of 300. Each library was individually hybridized to the biotinylated probes in the presence of RNA competitors complementary to the Illumina adapters for 48 hours at 62°C. After three 10-min 0.1X SSC washes at 62°C, captured DNA was eluted by alkaline hydrolysis of the RNA probes, amplified an optimal number of PCR cycles as predetermined by qPCR, and subjected to a second round of sequence capture in identical conditions. The eluted DNA was then amplified minimally, purified and sequenced on a MiSeq sequencer (Illumina) using paired-end sequencing for 2×75 cycles.</p><p id="p-40">Paired-end reads were merged using leeHom using the—ancientdna parameter (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>). Merged reads were then mapped to modern <em>Bison bison</em> and <em>Bison bonasus</em> mitogenomes using bwa as described in (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">37</a>). The first 100 nt of each linearized reference mitogenome used for mapping was duplicated at the 3′end to allow mapping of fragments overlapping the junction. Mapped read duplicates were then removed using samtools rmdup as described in (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">37</a>). The resulting bam files were then imported into Geneious 6.1.8 (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">43</a>) and remapped onto the appropriate mitogenome sequences without the 100 nt duplication (B. <em>bison</em> for <em>B. priscus</em> sequences, <em>B. bonasus</em> for the Bb1 and Bb2 sequences). Consensus sequences were generated in Geneious and verified by visual inspection of the aligned reads.</p></div><div id="sec-14" class="subsection"><h3>Phylogenetic analyses</h3><p id="p-41">Sequence alignments were performed using the Muscle algorithm and were visually inspected and adjusted using Geneious 6.1.8 (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">43</a>) The maximum likelihood (ML) analyses presented in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a> were computed using PHYML 3.0, using an HKY substitution model with a gamma-distributed rate of variation among sites (+G) and invariant sites (+I) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>). Robustness of the nodes was estimated using 500 bootstraps. RaXML 8.2.3 was used to generate the ML bootstrap support values for the complete mitogenome alignment shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3</a> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>).</p><p id="p-42">Phylogenetic analyses conducted under the Bayesian framework were performed using the program BEAST v. 1.8.2, which allows estimation of mutation and population history parameters simultaneously from temporally spaced sequence data (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>). Nucleotide substitution models were chosen following comparisons performed with jModelTest 2.1.7 using the Bayesian Information Criteria (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>). The HVR analysis presented in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2</a> was performed considering a TN93 model for the nucleotide substitution model, a gamma-distributed rate of variation among sites (+G) with four rate categories and invariant sites (i.e., TN93+I+G model). For the complete mitogenome analysis presented in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3</a>, we used four partitions, the HVR, the 1<sup>st</sup> and 2<sup>nd</sup> positions of the codons within the coding region, the 3<sup>rd</sup> position, and the RNA genes. We considered the HKY+I+G model for the first two partitions, and the TN93+G for the last two. Default priors were used for all parameters of the nucleotide substitution model. For the analysis of <a id="xref-fig-2-6" class="xref-fig" href="#F2">figure 2</a>, we used a strict molecular clock with a lognormal prior for the substitution rate (mean=−15.0, stdev=1.4) corresponding to a median of 2E-7 substitutions per site and per year [95%HPD 1.3E-8, 3.2E-6] based on the estimation for Bison HVR substitution rate (<a id="xref-ref-7-9" class="xref-bibr" href="#ref-7">7</a>). For the various partitions of the mitogenome of <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3</a>, we used estimates for the human mitogenome substitution rate to set the priors (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>): HVR, mean=−16.1 stdev=2.0, corresponding to a median of 1.0E-7 [2E-9-5E-6]; RNA, mean=−18.65 stdev=2.0, corresponding to a median of 8.0E-9 [1.6E-10-4E-7]; 1<sup>st</sup> &amp; 2<sup>nd</sup> positions, mean=−18.5 stdev=2.0, corresponding to a median of 9.0E-9 [1.8E-10-4.7E-7]; 3<sup>rd</sup> position, mean=−17.7 stdev=2.0 corresponding to a median of 2.0E-8 [4E-10-1E-6]. Finally, a standard coalescent model was considered for the tree prior with a Bayesian skyline plot to model populations (5 and 10 populations with default parameters for the HVR and the mitogenomes respectively). The prior for the tree height followed a log-normal distribution, mean=14.6 stdev=0.6, truncate to 8.0E6 and 1.0E5, corresponding to a median of 2.2E6 and a 95%HPD of [6.3E6-6.7E5], which integrates the various fossil finds assumed to correspond to ancestors of cattle and bison (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>).</p><p id="p-43">To estimate the posterior distribution of each parameter of interest, we used the Markov Chain Monte Carlo algorithm implemented in the BEAST software. We ran five independent chains with initial values sampled as described above and an input UPGMA tree constructed using a Juke-Cantor distance matrix. Each of these chains was run for 50,000,000 iterations and for each parameter of interest, 18,0 samples (one every 2,500 generated ones) were drawn after discarding a 10% burn-in period. The BEAST output was analyzed with the software Tracer v. 1.6 <a href="http://tree.bio.ed.ac.uk/software/tracer/">http://tree.bio.ed.ac.uk/software/tracer/</a>. Visual inspection of the traces and the estimated posterior distributions suggested that each MCMC had converged on its stationary distribution. Using Logcombiner v. 1.8.2, we further combined all the results from the 5 independent chains. The maximum clade credibility tree with the median height of the nodes was finally calculated using TreeAnnotator v. 1.8.2 and visualized using FigTree v. 1.4.2 <a href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>.</p></div></div><div class="section" id="sec-15"><h2 class="">Author Contributions</h2><p id="p-45">EMG and TG designed and supervised the overall research with an initial input from MT. DM developed sequence capture and produced the mitogenome data. SG produced the HVR data. TG, EMG, DM and SG analyzed and interpreted the data. TG, EMG and EAB wrote the manuscript with inputs from DM and SG. JPB provided samples, discussed the data and corrected the manuscript. MT, RMA, GBa, GBo, JCC, SM, OP, NS and HPU provided samples and feedbacks on the manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-44">The study was supported by the French national research center CNRS. The paleogenomic facility obtained support from the University Paris Diderot within the program “Actions de recherche structurantes”. The sequencing facility of the Institut Jacques Monod, Paris, is supported by grants from the University Paris Diderot, the Fondation pour la Recherche Medicale (DGE20111123014), and the Region Ile-de-France (11015901). We thank Lou Saier for the production of HVR data for some samples. We acknowledge the contribution to sampling of Myriam Boudadi-Maligne, Jean-Baptiste Mallye, Pierre Petrequin, Sylvie Lourdaux, and Rene Remond. We thank G. Baldacci for continuous support. M. T. also acknowledges financial support through the “Biodiversity of East-European and Siberian large mammals on the level of genetic variation of populations (BIOGEAST)” project within the 7th Framework Programme, Marie Curie Actions, that allowed sampling to take place in Russia. We like to thank Marie-Helene Moncel and Marie-Anne Julien for providing samples that did not yield genetic results, and the French ministry of culture for providing the image of the wall paintings of the Chauvet cave.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">+</span> alphabetical order</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">Bibliography</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hewitt</span>  <span class="cit-name-given-names">GM</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Genetic consequences of climatic oscillations in the Quaternary</span>. <abbr class="cit-jnl-abbrev">Phil. Trans. RL. Soc. Lond. B</abbr> <span class="cit-vol">359</span>:<span class="cit-fpage">183</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BRL.%2BSoc.%2BLond.%2BB%26rft.volume%253D359%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Markova</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Changes in the Eurasian distribution of the musk ox (Ovibos moschatus) and the extinct bison (Bison priscus) during the last 50 ka BP</span>. <abbr class="cit-jnl-abbrev">Quatern Int</abbr> <span class="cit-vol">378</span>:<span class="cit-fpage">99110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuatern%2BInt%26rft.volume%253D378%26rft.spage%253D99110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-063032v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guthrie</span>  <span class="cit-name-given-names">RD</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-chapter-title">Frozen fauna of the Mammoth Steppe</span>. <span class="cit-source">The story of Blue Babe</span>. (<span class="cit-publ-name">The University of Chicago Press</span>, <span class="cit-publ-loc">Chicago and London</span>) p <span class="cit-fpage">323</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brugal</span>  <span class="cit-name-given-names">JP</span></span> (<span class="cit-pub-date">1994-1995</span>) <span class="cit-article-title">Le Bison (Bovinae, Artiodactyla) du gisement Pleistocene moyen ancien de Durfort (Gard, France)</span>. <abbr class="cit-jnl-abbrev">Bull.Mus.nat.Hist.Nat., Paris 4e ser. 16, sect.C, n°2–4</abbr>:<span class="cit-fpage">349</span>– <span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brugal</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Enloe</span>  <span class="cit-name-given-names">J</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Jaubert</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">1999</span>) <abbr class="cit-jnl-abbrev">Le Bison, gibier et moyen de subsistance des hommes du Paleolithique aux Paleoindiens des grandes plaines (Antibes)</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-063032v1.6"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Sokolov</span>  <span class="cit-name-given-names">VE</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Flerow</span>  <span class="cit-name-given-names">CC</span></span> (<span class="cit-pub-date">1979</span>) <span class="cit-chapter-title">Systematics and Evolution</span>. <span class="cit-source">European Bison-Morphology, systematics, evolution, ecology (in Russian)</span>, ed <span class="cit-ed"><span class="cit-name-surname">Sokolov</span>  <span class="cit-name-given-names">VE</span></span> (<span class="cit-publ-name">Nauka (USSR Ac. of Sc.)</span>, <span class="cit-publ-loc">Moscou</span>), pp <span class="cit-fpage">9</span>–<span class="cit-lpage">127</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.7" data-doi="10.1126/science.1101074"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Rise and fall of the Beringian steppe bison</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">306</span>(<span class="cit-issue">5701</span>):<span class="cit-fpage">1561</span>– <span class="cit-lpage">1565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DShapiro%26rft.auinit1%253DB.%26rft.volume%253D306%26rft.issue%253D5701%26rft.spage%253D1561%26rft.epage%253D1565%26rft.atitle%253DRise%2Band%2BFall%2Bof%2Bthe%2BBeringian%2BSteppe%2BBison%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1101074%26rft_id%253Dinfo%253Apmid%252F15567864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDYvNTcwMS8xNTYxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTYvMDcvMDkvMDYzMDMyLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Groves</span>  <span class="cit-name-given-names">CP</span></span> (<span class="cit-pub-date">1981</span>) <span class="cit-article-title">Systematic relationship in the Bovini (Artiodactyla, Bovidae)</span>. <abbr class="cit-jnl-abbrev">Z.Zool.Syst.Evolutionforsch.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">264</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZ.Zool.Syst.Evolutionforsch.%26rft.volume%253D19%26rft.spage%253D264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.9" data-doi="10.1016/j.ympev.2012.11.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hassanin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ropiquet</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">TT</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Couloux</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Combining multiple autosomal introns for studying shallow phylogeny and taxonomy of Laurasiatherian mammals: Application to the tribe Bovini (Cetartiodactyla, Bovidae)</span>. <abbr class="cit-jnl-abbrev">Molecular phylogenetics and evolution</abbr> <span class="cit-vol">66</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">766</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.volume%253D66%26rft.spage%253D766%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2012.11.003%26rft_id%253Dinfo%253Apmid%252F23159894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2012.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23159894&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kowalski</span>  <span class="cit-name-given-names">K</span></span> (<span class="cit-pub-date">1967</span>) <span class="cit-article-title">The evolution and fossil remains of the European Bison</span>. <abbr class="cit-jnl-abbrev">Acta Theriologica</abbr> <span class="cit-vol">12</span>(<span class="cit-issue">21</span>):<span class="cit-fpage">335</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BTheriologica%26rft.volume%253D12%26rft.spage%253D335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-063032v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Krasinska</span>  <span class="cit-name-given-names">M</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Krasinski</span>  <span class="cit-name-given-names">ZA</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-source">European Bison: The Nature Monograph</span>. <span class="cit-edition">2<sup>nd</sup> Edition</span> (<span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Heidelberg</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.12" data-doi="10.1126/science.aac4315"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">PALEOECOLOGY. Abrupt warming events drove Late Pleistocene Holarctic megafaunal turnover</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">349</span>(<span class="cit-issue">6248</span>):<span class="cit-fpage">602</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aac4315%26rft_id%253Dinfo%253Apmid%252F26250679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDkvNjI0OC82MDIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNi8wNy8wOS8wNjMwMzIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boeskorov</span>  <span class="cit-name-given-names">GG</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Preliminary analyses of the frozen mummies of mammoth (Mammuthus primigenius), bison (Bison priscus) and horse (Equus sp.) from the Yana-Indigirka Lowland, Yakutia, Russia</span>. <abbr class="cit-jnl-abbrev">Integrative zoology</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">471</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegrative%2Bzoology%26rft.volume%253D9%26rft.spage%253D471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kirillova</span>  <span class="cit-name-given-names">IV</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">An ancient bison from the mouth of the Rauchua River (Chukotka, Russia)</span>. <abbr class="cit-jnl-abbrev">Quatern. Res.</abbr> <span class="cit-vol">84</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">232</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuatern.%2BRes.%26rft.volume%253D84%26rft.spage%253D232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Degerbol</span>  <span class="cit-name-given-names">M</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Iversen</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">1945</span>) <span class="cit-article-title">The Bison in Denmark</span>. <abbr class="cit-jnl-abbrev">A zoological and geological investigation of the finds in Danish Pleistocene deposits (Danm.Geol.Unders0gelse, Copenhagen)</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Geist</span>  <span class="cit-name-given-names">V</span></span> (<span class="cit-pub-date">1971</span>) <span class="cit-article-title">The relation of social evolution and dispersal in ungulates during the Pleistocene, with emphasis on the old world Deer and the genus Bison</span>. <abbr class="cit-jnl-abbrev">Quaternary Research</abbr> <span class="cit-vol">1</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">283</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BResearch%26rft.volume%253D1%26rft.spage%253D283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rautian</span>  <span class="cit-name-given-names">GS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kalabushkin</span>  <span class="cit-name-given-names">BA</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Nemtsev</span>  <span class="cit-name-given-names">AS</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">A New Subspecies of the European Bison, Bison bonasus montanus ssp. nov. (Bovidae, Artiodactyla)</span>. <abbr class="cit-jnl-abbrev">Doklady Biological Sciences</abbr> <span class="cit-vol">375</span> <span class="cit-fpage">636</span>–<span class="cit-lpage">640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDoklady%2BBiological%2BSciences%26rft.volume%253D375%26rft.spage%253D636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.18" data-doi="10.1371/journal.pone.0115090"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bocherens</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hofman-Kaminska</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drucker</span>  <span class="cit-name-given-names">DG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmolcke</span>  <span class="cit-name-given-names">U</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kowalczyk</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">European bison as a refugee species? Evidence from isotopic data on Early Holocene bison and other large herbivores in northern Europe</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">e0115090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0115090%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0115090%26rft_id%253Dinfo%253Apmid%252F25671634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0115090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25671634&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spassov</span>  <span class="cit-name-given-names">N</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Stovtechev</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">On the origin of the wisent, Bison bonasus (Linnaeus, 1758): presence of the wisent in the upper Palaeolithic art of Eurasia</span>. <abbr class="cit-jnl-abbrev">Advances in Vertebrate Paleontology « Hen to Panta »</abbr>:<span class="cit-fpage">125</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-063032v1.20"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Gates</span>  <span class="cit-name-given-names">CC</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Freese</span>  <span class="cit-name-given-names">CH</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Gogan</span>  <span class="cit-name-given-names">PJP</span></span>, &amp; </li><li><span class="cit-ed"><span class="cit-name-surname">Kotzman</span>  <span class="cit-name-given-names">M</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Potter</span>  <span class="cit-name-given-names">BA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-source">History of Bison in North America. American Bison: Status survey and conservation guidelines 2010</span>, eds <span class="cit-ed"><span class="cit-name-surname">Gates</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-ed"><span class="cit-name-surname">Freese</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-ed"><span class="cit-name-surname">Gogan</span>  <span class="cit-name-given-names">PJP</span></span>, &amp; <span class="cit-ed"><span class="cit-name-surname">Kotzman</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-publ-name">IUCN</span>, <span class="cit-publ-loc">Gland, Switzerland</span>), pp <span class="cit-fpage">13</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Verkaar</span>  <span class="cit-name-given-names">EL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nijman</span>  <span class="cit-name-given-names">IJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beeke</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanekamp</span>  <span class="cit-name-given-names">E</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lenstra</span>  <span class="cit-name-given-names">JA</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Maternal and paternal lineages in cross-breeding bovine species</span>. <abbr class="cit-jnl-abbrev">Has wisent a hybrid origin? Mol Biol Evol</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1165</span>–<span class="cit-lpage">1170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHas%2Bwisent%2Ba%2Bhybrid%2Borigin%253F%2BMol%2BBiol%2BEvol%26rft.volume%253D21%26rft.spage%253D1165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeyland</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Tracking of wisent-bison-yak mitochondrial evolution</span>. <abbr class="cit-jnl-abbrev">Journal of applied genetics</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">317</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bapplied%2Bgenetics%26rft.stitle%253DJ%2BAppl%2BGenet%26rft.aulast%253DZeyland%26rft.auinit1%253DJ.%26rft.volume%253D53%26rft.issue%253D3%26rft.spage%253D317%26rft.epage%253D322%26rft.atitle%253DTracking%2Bof%2Bwisent-bison-yak%2Bmitochondrial%2Bevolution.%26rft_id%253Dinfo%253Apmid%252F22415349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22415349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marsolier-Kergoat</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Hunting the Extinct Steppe Bison (Bison priscus) Mitochondrial Genome in the Trois-Freres Paleolithic Painted Cave</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">e0128267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0128267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.24" data-doi="10.1029/2004PA001071"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lisiecki</span>  <span class="cit-name-given-names">LE</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Raymo</span>  <span class="cit-name-given-names">ME</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">A Pliocene-Pleistocene stack of 57 globally distributed benthic 518O records</span>. <abbr class="cit-jnl-abbrev">Paleoceanography</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPaleoceanography%26rft.volume%253D20%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1029%252F2004PA001071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1029/2004PA001071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.25" data-doi="10.1038/nature02805"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">KK</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">High-resolution record of Northern Hemisphere climate extending into the last interglacial period</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">431</span>(<span class="cit-issue">7005</span>):<span class="cit-fpage">147</span>–<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D431%26rft.issue%253D7005%26rft.spage%253D147%26rft.epage%253D151%26rft.atitle%253DHigh-resolution%2Brecord%2Bof%2BNorthern%2BHemisphere%2Bclimate%2Bextending%2Binto%2Bthe%2Blast%2Binterglacial%2Bperiod%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02805%26rft_id%253Dinfo%253Apmid%252F15356621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02805&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2004083761&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=15356621&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223746000034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.26" data-doi="10.1016/0277-3791(96)00028-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Andel</span>  <span class="cit-name-given-names">TH</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tzedakis</span>  <span class="cit-name-given-names">PC</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Palaeolithic landscapes of Europe and environs, 150,000 25,0 years ago: An overview</span>. <abbr class="cit-jnl-abbrev">Quat Sci Rev</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">5–6</span>):<span class="cit-fpage">481</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat%2BSci%2BRev%26rft.volume%253D15%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0277-3791%252896%252900028-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0277-3791(96)00028-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VL59200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.27" data-doi="10.1038/338309a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guiot</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pons</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Beaulieu</span>  <span class="cit-name-given-names">JL</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Reille</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">A 140,000 year climatic reconstruction from two European pollen records</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">338</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D338%26rft.issue%253D6213%26rft.spage%253D309%26rft.epage%253D313%26rft.atitle%253DA%2B140%252C000-year%2Bcontinental%2Bclimate%2Breconstruction%2Bfrom%2Btwo%2BEuropean%2Bpollen%2Brecords%26rft_id%253Dinfo%253Adoi%252F10.1038%252F338309a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/338309a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1989032003&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Palombo</span>  <span class="cit-name-given-names">MR</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Deconstructing mammal dispersals and faunal dynamics in SW Europe during the Quaternary</span>. <abbr class="cit-jnl-abbrev">Quat Sci Rev</abbr> <span class="cit-vol">96</span>:<span class="cit-fpage">50</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat%2BSci%2BRev%26rft.volume%253D96%26rft.spage%253D50%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Julien</span>  <span class="cit-name-given-names">M-F</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Were European steppe bison migratory? 18O, 13C and Sr intra-tooth isotopic variations applied to a pallaeoethological reconstruction</span>. <abbr class="cit-jnl-abbrev">Quarternary International</abbr> <span class="cit-vol">271</span>:<span class="cit-fpage">106</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuarternary%2BInternational%26rft.volume%253D271%26rft.spage%253D106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-confproc" id="cit-063032v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quiles</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-conf-date">2016</span>) <span class="cit-article-title">A high-precision chronological model for the decorated Upper Paleolithic cave of Chauvet-Pont d’Arc, Ardeche, France</span>. <span class="cit-conf-name">Proc Natl Acad Sci</span> <span class="cit-conf-loc">U S A</span> <span class="cit-vol">113</span>(<span class="cit-issue">17</span>):<span class="cit-fpage">4670</span>–<span class="cit-lpage">4675</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.31"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Gates</span>  <span class="cit-name-given-names">CC</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Freese</span>  <span class="cit-name-given-names">CH</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Gogan</span>  <span class="cit-name-given-names">PJP</span></span>, &amp; </li><li><span class="cit-ed"><span class="cit-name-surname">Kotzman</span>  <span class="cit-name-given-names">M</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">GA</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Gates</span>  <span class="cit-name-given-names">CC</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Taxonomy and nomenclature</span>. <abbr class="cit-jnl-abbrev">American Bison: Status survey and conservation guidelines 2010</abbr>, eds <span class="cit-ed"><span class="cit-name-surname">Gates</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-ed"><span class="cit-name-surname">Freese</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-ed"><span class="cit-name-surname">Gogan</span>  <span class="cit-name-given-names">PJP</span></span>, &amp; <span class="cit-ed"><span class="cit-name-surname">Kotzman</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-auth cit-collab">IUCN, Gland, Switzerland</span>), pp <span class="cit-fpage">13</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yudin</span>  <span class="cit-name-given-names">NS</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Detection of mitochondrial DNA from domestic cattle in European bison (Bison bonasus) from the Altai Republic in Russia</span>. <abbr class="cit-jnl-abbrev">Anim Genet</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DYudin%26rft.auinit1%253DN.%2BS.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D362%26rft.epage%253D362%26rft.atitle%253DDetection%2Bof%2Bmitochondrial%2BDNA%2Bfrom%2Bdomestic%2Bcattle%2Bin%2BEuropean%2Bbison%2B%2528Bison%2Bbonasus%2529%2Bfrom%2Bthe%2BAltai%2BRepublic%2Bin%2BRussia.%26rft_id%253Dinfo%253Apmid%252F22486514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22486514&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.33" data-doi="10.1093/jhered/esl051"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Halbert</span>  <span class="cit-name-given-names">ND</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Derr</span>  <span class="cit-name-given-names">JN</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">A comprehensive evaluation of cattle introgression into US federal bison herds</span>. <abbr class="cit-jnl-abbrev">The Journal of heredity</abbr> <span class="cit-vol">98</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bheredity%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesl051%26rft_id%253Dinfo%253Apmid%252F17172633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esl051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17172633&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243585900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martfnez-Navarro</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rook</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papini</span>  <span class="cit-name-given-names">M</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Libsekal</span>  <span class="cit-name-given-names">Y</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">A new species of bull from the Early Pleistocene paleoanthropological site of Buia (Eritrea): Parallelism on the dispersal of the genus Bos and the Acheulian culture</span>. <abbr class="cit-jnl-abbrev">Quatern Int</abbr> <span class="cit-vol">212</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">169</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuatern%2BInt%26rft.volume%253D212%26rft.spage%253D169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Deciphering the wisent demographic and adaptive histories from individual whole-genome sequences</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol In press</abbr>, biorxiv <a href="http://dx.doi.org/10.1101/058446">http://dx.doi.org/10.1101/058446</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wecek</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Complex admixture preceded and followed the extinction of wisent in the wild</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">059527</span>:doi: <a href="http://dx.doi.org/10.1101/059527">http://dx.doi.org/10.1101/059527</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Library construction for ancient genomics: single strand or double strand?</span> <abbr class="cit-jnl-abbrev">BioTechniques</abbr> <span class="cit-vol">56</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">289</span>–<span class="cit-lpage">290</span>, <span class="cit-auth cit-collab">292-286, 298, passim</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.volume%253D56%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.38" data-doi="10.1371/journal.pone.0146230"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cote</span>  <span class="cit-name-given-names">NM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">A New High-Throughput Approach to Genotype Ancient Human Gastrointestinal Parasites</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">e0146230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0146230%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0146230%26rft_id%253Dinfo%253Apmid%252F26752051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0146230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26752051&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guimaraes</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (submitted) <span class="cit-article-title">A cost-effective high-throughput metabarcoding approach powerful enough to genotype ~44,000 year-old rodent remains from Northern Africa</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.40" data-doi="10.2144/05384ST03"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pruvost</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grange</span>  <span class="cit-name-given-names">T</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Geigl</span>  <span class="cit-name-given-names">E-M</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Minimizing DNA contamination by using UNG-coupled quantitative real-time PCR on degraded DNA samples: application to ancient DNA studies</span>. <abbr class="cit-jnl-abbrev">BioTechniques</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">569</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DPruvost%26rft.auinit1%253DM.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D569%26rft.epage%253D575%26rft.atitle%253DMinimizing%2BDNA%2Bcontamination%2Bby%2Busing%2BUNG-coupled%2Bquantitative%2Breal-time%2BPCR%2Bon%2Bdegraded%2BDNA%2Bsamples%253A%2Bapplication%2Bto%2Bancient%2BDNA%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F05384ST03%26rft_id%253Dinfo%253Apmid%252F15884675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/05384ST03&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15884675&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228375700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Champlot</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">An efficient multistrategy DNA decontamination procedure of PCR reagents for hypersensitive PCR applications</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">9</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-confproc" id="cit-063032v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pruvost</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-conf-date">2007</span>) <span class="cit-article-title">Freshly excavated fossil bones are best for amplification of ancient DNA</span>. <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. <span class="cit-conf-loc">USA</span> <span class="cit-vol">104</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">739</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.43" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1647</span>– <span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gorge</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Analysis of Ancient DNA in Microbial Ecology</span>. <abbr class="cit-jnl-abbrev">Methods in molecular biology</abbr> <span class="cit-vol">1399</span>:<span class="cit-fpage">289</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%26rft.volume%253D1399%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.45" data-doi="10.1016/j.mimet.2006.11.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rueckert</span>  <span class="cit-name-given-names">A</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">HW</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Removal of contaminating DNA from polymerase chain reaction using ethidium monoazide</span>. <abbr class="cit-jnl-abbrev">Journal of microbiological methods</abbr> <span class="cit-vol">68</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">596</span>–<span class="cit-lpage">600</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMicrobiological%2BMethods%26rft.stitle%253DJournal%2Bof%2BMicrobiological%2BMethods%26rft.aulast%253DRueckert%26rft.auinit1%253DA.%26rft.volume%253D68%26rft.issue%253D3%26rft.spage%253D596%26rft.epage%253D600%26rft.atitle%253DRemoval%2Bof%2Bcontaminating%2BDNA%2Bfrom%2Bpolymerase%2Bchain%2Breaction%2Busing%2Bethidium%2Bmonoazide.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mimet.2006.11.006%26rft_id%253Dinfo%253Apmid%252F17187883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mimet.2006.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17187883&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245147900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.46" data-doi="10.1038/nbt.1523"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gnirke</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing</span>. <abbr class="cit-jnl-abbrev">Nature biotechnology</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">182</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGnirke%26rft.auinit1%253DA.%26rft.volume%253D27%26rft.issue%253D2%26rft.spage%253D182%26rft.epage%253D189%26rft.atitle%253DSolution%2Bhybrid%2Bselection%2Bwith%2Bultra-long%2Boligonucleotides%2Bfor%2Bmassively%2Bparallel%2Btargeted%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1523%26rft_id%253Dinfo%253Apmid%252F19182786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19182786&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263209000025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.47" data-doi="10.1093/nar/gku699"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Renaud</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenzel</span>  <span class="cit-name-given-names">U</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kelso</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">leeHom: adaptor trimming and merging for Illumina sequencing reads</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">18</span>):<span class="cit-fpage">e141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku699%26rft_id%253Dinfo%253Apmid%252F25100869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku699&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25100869&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.48" data-doi="10.1093/sysbio/syq010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr> <span class="cit-vol">59</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">307</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq010%26rft_id%253Dinfo%253Apmid%252F20525638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525638&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276528300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.49" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.50" data-doi="10.1186/1471-2148-7-214"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. <abbr class="cit-jnl-abbrev">BMC Evolutionary Biology</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvolutionary%2BBiology%26rft.volume%253D7%26rft.spage%253D214%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-7-214%26rft_id%253Dinfo%253Apmid%252F17996036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-7-214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17996036&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darriba</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">GL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doallo</span>  <span class="cit-name-given-names">R</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">jModelTest 2: more models, new heuristics and parallel computing</span>. <abbr class="cit-jnl-abbrev">Nature methods</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D9%26rft.spage%253D772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.52" data-doi="10.1016/j.ajhg.2009.05.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Soares</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Correcting for purifying selection: an improved human mitochondrial molecular clock</span>. <abbr class="cit-jnl-abbrev">Am J Hum Genet</abbr> <span class="cit-vol">84</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">740</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSoares%26rft.auinit1%253DP.%26rft.volume%253D84%26rft.issue%253D6%26rft.spage%253D740%26rft.epage%253D759%26rft.atitle%253DCorrecting%2Bfor%2Bpurifying%2Bselection%253A%2Ban%2Bimproved%2Bhuman%2Bmitochondrial%2Bmolecular%2Bclock.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.05.001%26rft_id%253Dinfo%253Apmid%252F19500773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19500773&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2016%2F07%2F09%2F063032.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267042800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martfnez-Navarro</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ros-Montoya</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Espigares</span>  <span class="cit-name-given-names">MP</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Palmqvist</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Presence of the Asian origin Bovini, Hemibos sp. aff. Hemibos gracilis and Bison sp., at the early Pleistocene site of Venta Micena (Orce, Spain)</span>. <abbr class="cit-jnl-abbrev">Quatern Int</abbr> <span class="cit-vol">243</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">54</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuatern%2BInt%26rft.volume%253D243%26rft.spage%253D54%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-063032v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qui</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">T</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Early Pleistocene mammalian fauna from Longdan, Dongxian, Gansu, China</span>. <abbr class="cit-jnl-abbrev">Paleontologia Sinica</abbr> <span class="cit-vol">191</span>(<span class="cit-issue">C27</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPaleontologia%2BSinica%26rft.volume%253D191%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/063032v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F063032v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/062844v1" title="Do Dams Also Stop Frogs? Assessing Population Connectivity of Coastal Tailed Frogs (Ascaphus truei) in the North Cascades National Park Service Complex" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/041509v2" title="How domain growth is implemented determines the long term behaviour of a cell population through its effect on spatial correlations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;July 09, 2016.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/063032v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/063032v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/063032v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/063032v1" class="active" data-icon-position="" data-hide-link-title="0">Past climate changes, population dynamics and the origin of Bison in Europe</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/17292" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-rKF62I2oJNfX2ccXMus9wiE98rnGJoip2LZwmGlqm2w" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="17292" id="node17292--2" data-pisa="biorxiv;063032v1" data-pisa-master="biorxiv;063032" data-seqnum="17292" data-apath="/biorxiv/early/2016/07/09/063032.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Past climate changes, population dynamics and the origin of Bison in Europe</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Diyendo</span> <span class="nlm-surname">Massilani</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Silvia</span> <span class="nlm-surname">Guimaraes</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jean-Philip</span> <span class="nlm-surname">Brugal</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">E. Andrew</span> <span class="nlm-surname">Bennett</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Malgorzata</span> <span class="nlm-surname">Tokarska</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Rose-Marie</span> <span class="nlm-surname">Arbogast</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Gennady</span> <span class="nlm-surname">Baryshnikov</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Gennady</span> <span class="nlm-surname">Boeskorov</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jean-Christophe</span> <span class="nlm-surname">Castel</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Sergey</span> <span class="nlm-surname">Davydov</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Stephane</span> <span class="nlm-surname">Madelaine</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Olivier</span> <span class="nlm-surname">Putelat</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Natalia</span> <span class="nlm-surname">Spasskaya</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Hans-Peter</span> <span class="nlm-surname">Uerpmann</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Thierry</span> <span class="nlm-surname">Grange</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Eva-Maria</span> <span class="nlm-surname">Geigl</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">063032; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/063032 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/063032v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/063032v1&amp;title=Past%20climate%20changes%2C%20population%20dynamics%20and%20the%20origin%20of%20Bison%20in%20Europe" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/063032v1&amp;title=Past%20climate%20changes%2C%20population%20dynamics%20and%20the%20origin%20of%20Bison%20in%20Europe" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/063032v1&amp;text=Past%20climate%20changes%2C%20population%20dynamics%20and%20the%20origin%20of%20Bison%20in%20Europe" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/063032v1&amp;title=Past%20climate%20changes%2C%20population%20dynamics%20and%20the%20origin%20of%20Bison%20in%20Europe" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/063032v1&amp;t=Past%20climate%20changes%2C%20population%20dynamics%20and%20the%20origin%20of%20Bison%20in%20Europe" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/063032v1&amp;title=Past%20climate%20changes%2C%20population%20dynamics%20and%20the%20origin%20of%20Bison%20in%20Europe" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/063032v1&amp;title=Past%20climate%20changes%2C%20population%20dynamics%20and%20the%20origin%20of%20Bison%20in%20Europe" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="17292" id="node17292--3" data-pisa="biorxiv;063032v1" data-pisa-master="biorxiv;063032" data-seqnum="17292" data-apath="/biorxiv/early/2016/07/09/063032.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Past climate changes, population dynamics and the origin of Bison in Europe</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Diyendo</span> <span class="nlm-surname">Massilani</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Silvia</span> <span class="nlm-surname">Guimaraes</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jean-Philip</span> <span class="nlm-surname">Brugal</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">E. Andrew</span> <span class="nlm-surname">Bennett</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Malgorzata</span> <span class="nlm-surname">Tokarska</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Rose-Marie</span> <span class="nlm-surname">Arbogast</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Gennady</span> <span class="nlm-surname">Baryshnikov</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Gennady</span> <span class="nlm-surname">Boeskorov</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Jean-Christophe</span> <span class="nlm-surname">Castel</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Sergey</span> <span class="nlm-surname">Davydov</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Stephane</span> <span class="nlm-surname">Madelaine</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Olivier</span> <span class="nlm-surname">Putelat</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Natalia</span> <span class="nlm-surname">Spasskaya</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Hans-Peter</span> <span class="nlm-surname">Uerpmann</span></span>, <span class="highwire-citation-author" data-delta="14"><span class="nlm-given-names">Thierry</span> <span class="nlm-surname">Grange</span></span>, <span class="highwire-citation-author" data-delta="15"><span class="nlm-given-names">Eva-Maria</span> <span class="nlm-surname">Geigl</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">063032; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/063032 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Past%20climate%20changes%2C%20population%20dynamics%20and%20the%20origin%20of%20Bison%20in%20Europe&amp;rft.volume&amp;rft.issue&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Massilani%2C+Diyendo&amp;rft.au=Guimaraes%2C+Silvia&amp;rft.au=Brugal%2C+Jean-Philip&amp;rft.au=Bennett%2C+E.+Andrew&amp;rft.au=Tokarska%2C+Malgorzata&amp;rft.au=Arbogast%2C+Rose-Marie&amp;rft.au=Baryshnikov%2C+Gennady&amp;rft.au=Boeskorov%2C+Gennady&amp;rft.au=Castel%2C+Jean-Christophe&amp;rft.au=Davydov%2C+Sergey&amp;rft.au=Madelaine%2C+Stephane&amp;rft.au=Putelat%2C+Olivier&amp;rft.au=Spasskaya%2C+Natalia&amp;rft.au=Uerpmann%2C+Hans-Peter&amp;rft.au=Grange%2C+Thierry&amp;rft.au=Geigl%2C+Eva-Maria"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/17292/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/17292/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/17292/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/17292/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/17292/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/17292/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/17292/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/17292/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/17292/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/17292/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/17292/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/17292/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/063032v1&amp;count=horizontal&amp;via=&amp;text=Past%20climate%20changes%2C%20population%20dynamics%20and%20the%20origin%20of%20Bison%20in%20Europe&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/063032v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/063032v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/063032v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
