<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/01/31/534636.atom" />
<meta name="HW.pisa" content="biorxiv;534636v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling" />
<meta name="DC.Identifier" content="10.1101/534636" />
<meta name="DC.Date" content="2019-01-31" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Aim Phenology, the temporal response of a population to its climate, is a crucial behavioral trait shared across life on earth. How species adapt their phenologies to climate change is poorly understood but critical in understanding how species will respond to future change. We use a group of flies ( Rhaphiomidas ) endemic to the North American deserts to understand how species adapt to changing climatic conditions. Here we explore a novel approach for taxa with constrained phenologies aimed to accurately model their environmental niche and relate this to phenological and morphological adaptations in a phylogenetic context.

Taxon Insecta, Diptera, Mydidae, Rhaphiomidas 

Location North America, Mojave, Sonoran and Chihuahuan Deserts.

Methods We gathered geographical and phenological occurrence data for the entire genus Rhaphiomidas , and, estimated a time calibrated phylogeny. We compared Daymet derived temperature values for a species adult occurrence period (phenology) with those derived from WorldClim data that is partitioned by month or quarter to examine what effect using more precise data has on capturing a species’ environmental niche. We then examined to what extent phylogenetic signal in phenological traits, climate tolerance and morphology can inform us about how species adapt to different environmental regimes.

Results We found that the Bioclim temperature data, which are averages across monthly intervals, poorly represent the climate windows to which adult flies are actually adapted. Using temporally-relevant climate data, we show that many species use a combination of morphological and phenological changes to adapt to different climate regimes. There are also instances where species changed only phenology to track a climate type or only morphology to adapt to different environments.

Main Conclusions Without using a fine-scale phenological data approach, identifying environmental adaptations could be misleading because the data do not represent the conditions the animals are actually experiencing. We find that fine-scale phenological niche models are needed when assessing taxa that have a discrete phenological window that is key to their survival, accurately linking environment to morphology and phenology. Using this approach, we show that Rhaphiomidas use a combination of niche tracking and adaptation to persist in new niches. Modeling the effect of phenology on such species’ niches will be critical for better predictions of how these species might respond to future climate change." />
<meta name="DC.Contributor" content="Matthew H. Van Dam" />
<meta name="DC.Contributor" content="Andrew J. Rominger" />
<meta name="DC.Contributor" content="Michael S. Brewer" />
<meta name="article:published_time" content="2019-01-31" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;h3&gt;Aim&lt;/h3&gt;
&lt;p&gt;Phenology, the temporal response of a population to its climate, is a crucial behavioral trait shared across life on earth. How species adapt their phenologies to climate change is poorly understood but critical in understanding how species will respond to future change. We use a group of flies (&lt;i&gt;Rhaphiomidas&lt;/i&gt;) endemic to the North American deserts to understand how species adapt to changing climatic conditions. Here we explore a novel approach for taxa with constrained phenologies aimed to accurately model their environmental niche and relate this to phenological and morphological adaptations in a phylogenetic context.&lt;/p&gt;&lt;h3&gt;Taxon&lt;/h3&gt;
&lt;p&gt;Insecta, Diptera, Mydidae, &lt;i&gt;Rhaphiomidas&lt;/i&gt;&lt;/p&gt;&lt;h3&gt;Location&lt;/h3&gt;
&lt;p&gt;North America, Mojave, Sonoran and Chihuahuan Deserts.&lt;/p&gt;&lt;h3&gt;Methods&lt;/h3&gt;
&lt;p&gt;We gathered geographical and phenological occurrence data for the entire genus &lt;i&gt;Rhaphiomidas&lt;/i&gt;, and, estimated a time calibrated phylogeny. We compared Daymet derived temperature values for a species adult occurrence period (phenology) with those derived from WorldClim data that is partitioned by month or quarter to examine what effect using more precise data has on capturing a species’ environmental niche. We then examined to what extent phylogenetic signal in phenological traits, climate tolerance and morphology can inform us about how species adapt to different environmental regimes.&lt;/p&gt;&lt;h3&gt;Results&lt;/h3&gt;
&lt;p&gt;We found that the Bioclim temperature data, which are averages across monthly intervals, poorly represent the climate windows to which adult flies are actually adapted. Using temporally-relevant climate data, we show that many species use a combination of morphological and phenological changes to adapt to different climate regimes. There are also instances where species changed only phenology to track a climate type or only morphology to adapt to different environments.&lt;/p&gt;&lt;h3&gt;Main Conclusions&lt;/h3&gt;
&lt;p&gt;Without using a fine-scale phenological data approach, identifying environmental adaptations could be misleading because the data do not represent the conditions the animals are actually experiencing. We find that fine-scale phenological niche models are needed when assessing taxa that have a discrete phenological window that is key to their survival, accurately linking environment to morphology and phenology. Using this approach, we show that &lt;i&gt;Rhaphiomidas&lt;/i&gt; use a combination of niche tracking and adaptation to persist in new niches. Modeling the effect of phenology on such species’ niches will be critical for better predictions of how these species might respond to future climate change.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;534636v1" />
<meta name="citation_id" content="534636v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/534636v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/534636v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/534636v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636.full.pdf" />
<meta name="citation_doi" content="10.1101/534636" />
<meta name="citation_num_pages" content="34" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="534636" />
<meta name="citation_author" content="Matthew H. Van Dam" />
<meta name="citation_author_institution" content="Department of Environmental Science" />
<meta name="citation_author_email" content="matthewhvandam@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7473-9727" />
<meta name="citation_author" content="Andrew J. Rominger" />
<meta name="citation_author_institution" content="Department of Environmental Science" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3755-4480" />
<meta name="citation_author" content="Michael S. Brewer" />
<meta name="citation_author_institution" content="Department of Environmental Science" />
<meta name="citation_reference" content="van Asch, M., van Tienderen, P.H., Holleman, L.J.M., &amp; Visser, M.E. (2007) Predicting adaptation of phenology in response to climate change, an insect herbivore example. Global Change Biology, 13, 1596–1604." />
<meta name="citation_reference" content="citation_title=Life edge a guide to California’s endangered natural resources: Wildlife;citation_pages=416-417;citation_year=1994" />
<meta name="citation_reference" content="Banta, J.A., Ehrenreich, I.M., Gerard, S., Chou, L., Wilczek, A., Schmitt, J., Kover, P.X., &amp; Purugganan, M.D. (2012) Climate envelope modelling reveals intraspecific relationships among flowering phenology, niche breadth and potential range size in Arabidopsis thaliana. Ecology Letters, 15, 769–777." />
<meta name="citation_reference" content="Becker, R.A., Wilks, A.R.R., Brownrigg, R., Minka, T.P., &amp; Deckmyn, A. (2017) maps: Draw Geographical Maps. R package version 3.2.0." />
<meta name="citation_reference" content="Bittner, T.D., King, R.B., &amp; Kerfin, J.M. (2002) Effects of Body Size and Melanism on the Thermal Biology of Garter Snakes (Thamnophis sirtalis). Copeia, 2002, 477–482." />
<meta name="citation_reference" content="Bradshaw, W.E. &amp; Holzapfel, C.M. (2008) Genetic response to rapid climate change: it’s seasonal timing that matters. Molecular Ecology, 17, 157–166." />
<meta name="citation_reference" content="Cazier, M.A. (1985) A revision of the North American flies belonging to the genus Rhaphiomidas (Diptera, Apioceridae). Bulletin of the AMNH; v. 182, article 2." />
<meta name="citation_reference" content="Clusella Trullas, S., van Wyk, J.H., &amp; Spotila, J.R. (2007) Thermal melanism in ectotherms. Journal of Thermal Biology, 32, 235–245." />
<meta name="citation_reference" content="Van Dam, M.H. (2010) Zootaxa, A new species and key for Rhaphiomidas Osten Sacken (Diptera: Mydidae). Zootaxa, 2622, 49–60." />
<meta name="citation_reference" content="Van Dam, M.H. &amp; Matzke, N.J. (2016) Evaluating the influence of connectivity and 18 distance on biogeographical patterns in the south-western deserts of North America. Journal of Biogeography, 43, 1514–1532." />
<meta name="citation_reference" content="Drummond, A. &amp; Bouckaert, R. (2015) Bayesian evolutionary analysis: with BEAST." />
<meta name="citation_reference" content="Drummond, A.J., Ho, S.Y.W., Phillips, M.J., &amp; Rambaut, A. (2006) Relaxed Phylogenetics and Dating with Confidence. PLoS Biology, 4, e88." />
<meta name="citation_reference" content="Drummond, A.J., Suchard, M.A., Xie, D., &amp; Rambaut, A. (2012) Bayesian Phylogenetics with BEAUti and the BEAST 1.7. Molecular Biology and Evolution, 29, 1969–1973." />
<meta name="citation_reference" content="Elith, J., Phillips, S.J., Hastie, T., Dudík, M., Chee, Y.E., &amp; Yates, C.J. (2011) A statistical explanation of MaxEnt for ecologists. Diversity and Distributions, 17, 43–57." />
<meta name="citation_reference" content="Ellwood, E.R., Temple, S.A., Primack, R.B., Bradley, N.L., &amp; Davis, C.C. (2013) Record-Breaking Early Flowering in the Eastern United States. PLoS ONE, 8, e53788." />
<meta name="citation_reference" content="Evans, M.E.K., Smith, S.A., Flynn, R.S., &amp; Donoghue, M.J. (2009) Climate, Niche Evolution, and Diversification of the “Bird-Cage” Evening Primroses (Oenothera, Sections Anogra and Kleinia). The American Naturalist, 173, 225–240." />
<meta name="citation_reference" content="Everill, P.H., Primack, R.B., Ellwood, E.R., &amp; Melaas, E.K. (2014) Determining past leaf-out times of New England’s deciduous forests from herbarium specimens. American Journal of Botany, 101, 1293–1300." />
<meta name="citation_reference" content="Felsenstein, J. Phylogenies and the Comparative Method. The American Naturalist, 125, 1–15." />
<meta name="citation_reference" content="Felsenstein, J. (2012) A Comparative Method for Both Discrete and Continuous Characters Using the Threshold Model. The American Naturalist, 179, 145–156." />
<meta name="citation_reference" content="Gerst, K.L., Rossington, N.L., &amp; Mazer, S.J. (2017) Phenological responsiveness to climate differs among four species of Quercus in North America. Journal of Ecology, 105, 1610–1622." />
<meta name="citation_reference" content="Harmon, L.J., Weir, J.T., Brock, C.D., Glor, R.E., &amp; Challenger, W. (2008) GEIGER: investigating evolutionary radiations. Bioinformatics, 24, 129–131." />
<meta name="citation_reference" content="Heibl, C. &amp; Calenge, C. (2013) phyloclim: Integrating Phylogenetics and Climatic Niche Modeling. R package version 0.9-4." />
<meta name="citation_reference" content="Hernandez, P.A., Graham, C.H., Master, L.L., &amp; Albert, D.L. (2006) The effect of sample size and species characteristics on performance of different species distribution modeling methods. Ecography, 29, 773–785." />
<meta name="citation_reference" content="Hijmans, R.J., Cameron, S.E., Parra, J.L., Jones, P.G., &amp; Jarvis, A. (2005) Very high resolution interpolated climate surfaces for global land areas. International Journal of Climatology, 25, 1965–1978." />
<meta name="citation_reference" content="Hijmans, R.J. &amp; van Etten, J. (2012) Geographic analysis and modeling with raster data. R package version 2.0-12." />
<meta name="citation_reference" content="Hijmans, R.J., Phillips, S., Leathwick, J., &amp; Elith, J. (2017) dismo: Species Distribution Modeling. R package version 1.1-4." />
<meta name="citation_reference" content="Hof, C., Araújo, M.B., Jetz, W., &amp; Rahbek, C. (2011) Additive threats from pathogens, climate and land-use change for global amphibian diversity. Nature, 480, 516–9." />
<meta name="citation_reference" content="Hoffmann, A.A. &amp; Sgrò, C.M. (2011) Climate change and evolutionary adaptation. Nature, 470, 479–485." />
<meta name="citation_reference" content="Holt, R.D. (1990) The microevolutionary consequences of climate change. Trends in Ecology &amp; Evolution, 5, 311–5." />
<meta name="citation_reference" content="Huelsenbeck, J.P., Larget, B., &amp; Alfaro, M.E. (2004) Bayesian Phylogenetic Model Selection Using Reversible Jump Markov Chain Monte Carlo. Molecular Biology and Evolution, 21, 1123–1133." />
<meta name="citation_reference" content="Kingsley, K.J. (1996) Behavior of the Delhi Sands Flower-Loving Fly (Diptera: Mydidae), a Little-Known Endangered Species. Annals of the Entomological Society of America, 89, 883–891." />
<meta name="citation_reference" content="Kingsley, K.J. (2002) Population Dynamics, Resource Use, and Conservation Needs of the Delhi Sands Flower-loving Fly (Rhaphiomidas Terminatus Abdominalis Cazier) (Diptera: Mydidae), an Endangered Species. Journal of Insect Conservation, 6, 93–101." />
<meta name="citation_reference" content="Kingsolver, J. &amp; Wiernasz, D. (1991) Seasonal polyphenism in wing-melanin pattern and thermoregulatory adaptation in Pieris butterflies. American Naturalist,." />
<meta name="citation_reference" content="Lanfear, R., Calcott, B., Ho, S.Y.W., &amp; Guindon, S. (2012) PartitionFinder: Combined Selection of Partitioning Schemes and Substitution Models for Phylogenetic Analyses. Molecular Biology and Evolution, 29, 1695–1701." />
<meta name="citation_reference" content="Larkin, M.A., Blackshields, G., Brown, N.P., Chenna, R., McGettigan, P.A., McWilliam, H., Valentin, F., Wallace, I.M., Wilm, A., Lopez, R., Thompson, J.D., Gibson, T.J., &amp; Higgins, D.G. (2007) Clustal W and Clustal X version 2.0. Bioinformatics, 23, 2947–2948." />
<meta name="citation_reference" content="Maddison, W. &amp; Maddison, D. (2009) Mesquite: a modular system for evolutionary analysis. Version 2.6. Computer program and documentation distributed by the author." />
<meta name="citation_reference" content="Martill, D.M., Bechly, G., &amp; Loveridge, R.F. (2007) The Crato fossil beds of Brazil: window into an ancient world. Cambridge University Press," />
<meta name="citation_reference" content="Matzke, N.J., (2014) Model Selection in Historical Biogeography Reveals that Founder-event Speciation is a Crucial Process in Island Clades. Systematic Biology. 63, 951–970." />
<meta name="citation_reference" content="McGuire, J.A., Linkem, C.W., Koo, M.S., Hutchison, D.W., Lappin, A.K., Orange, D.I., Lemos-Espinal, J., Riddle, B.R., &amp; Jaeger, J.R. (2007) Mitochondrial introgression and incomplete lineage sorting through space and time: phylogenetics of crotaphytid lizards. Evolution, 61, 2879–2897." />
<meta name="citation_reference" content="Miller-Rushing, A.J. &amp; Primack, R.B. (2008) Global warming and flowering times in Thoreau’s Concord: a community perspective. Ecology, 89, 332–41." />
<meta name="citation_reference" content="Mitchell, N.J., Kearney, M.R., Nelson, N.J., &amp; Porter, W.P. (2008) Predicting the fate of a living fossil: how will global warming affect sex determination and hatching phenology in tuatara? Proceedings. Biological sciences, 275, 2185–93." />
<meta name="citation_reference" content="Nagy, K.A. &amp; Medica, P.A. (1986) Physiological ecology of desert tortoises in southern Nevada. Herpetologica 42, 73–92." />
<meta name="citation_reference" content="Parmesan, C. (2006) Ecological and Evolutionary Responses to Recent Climate Change. Annual Review of Ecology, Evolution, and Systematics, 37, 637–669." />
<meta name="citation_reference" content="Phillips, S.J., Anderson, R.P., &amp; Schapire, R.E. (2006) Maximum entropy modeling of species geographic distributions. Ecological Modelling, 190, 231–259." />
<meta name="citation_reference" content="Pierce, D. (2017) Interface to Unidata netCDF (Version 4 or Earlier) Format Data Files. R package version 1.16." />
<meta name="citation_reference" content="van Proosdij, A.S., Sosef, M.S., Wieringa, J.J. &amp; Raes, N. (2016) Minimum required number of specimen records to develop accurate species distribution models. Ecography. 39, 542–552." />
<meta name="citation_reference" content="Quintero, I. &amp; Wiens, J.J. (2013) Rates of projected climate change dramatically exceed past rates of climatic niche evolution among vertebrate species. Ecology letters, 16, 1095–103." />
<meta name="citation_reference" content="Revell, L.J. (2012) phytools: an R package for phylogenetic comparative biology (and other things). Methods in Ecology and Evolution, 3, 217–223." />
<meta name="citation_reference" content="Rogers, R. &amp; Mattoni, R. (1993) Observations on the natural history and conservation biology of the giant flower loving flies, Rhaphiomidas (Diptera: Apioceridae). Dipterological Research, 4, 21–34." />
<meta name="citation_reference" content="Rogers, R. &amp; Van Dam, M.H. (2007) Two new species of Rhaphiomidas (Diptera: Mydidae). Zootaxa, 1664, 61–68." />
<meta name="citation_reference" content="Ronquist, F., Teslenko, M., van der Mark, P., Ayres, D.L., Darling, A., Höhna, S., Larget, B., Liu, L., Suchard, M.A., &amp; Huelsenbeck, J.P. (2012) MrBayes 3.2: Efficient Bayesian Phylogenetic Inference and Model Choice Across a Large Model Space. Systematic Biology, 61, 539–542." />
<meta name="citation_reference" content="Rosenblum, E.B. (2005) The Role of Phenotypic Plasticity in Color Variation of Tularosa Basin Lizards. Copeia, 2005, 586–596." />
<meta name="citation_reference" content="Sheldon, K.S., Yang, S., &amp; Tewksbury, J.J. (2011) Climate change and community disassembly: impacts of warming on tropical and temperate montane community structure. Ecology Letters, 14, 1191–1200." />
<meta name="citation_reference" content="Sukumaran, J. &amp; Holder, M.T. (2010) DendroPy: a Python library for phylogenetic computing. Bioinformatics, 26, 1569–1571." />
<meta name="citation_reference" content="Svensson, E.I. &amp; Waller, J.T. (2013) Ecology and Sexual Selection: Evolution of Wing Pigmentation in Calopterygid Damselflies in Relation to Latitude, Sexual Dimorphism, and Speciation. The American Naturalist, 182, E174–E195." />
<meta name="citation_reference" content="Thomas, C.D., Cameron, A., Green, R.E., Bakkenes, M., Beaumont, L.J., Collingham, Y.C., Erasmus, B.F.N., De Siqueira, M.F., Grainger, A., Hannah, L., Hughes, L., Huntley, B., Van Jaarsveld, A.S., Midgley, G.F., Miles, L., Ortega-Huerta, M.A., Peterson, A.T., Phillips, O.L., &amp; Williams, S.E. (2004) Extinction risk from climate change. Nature, 427, 145–148." />
<meta name="citation_reference" content="Thornton, P.E., M.M. Thornton, B.W. Mayer, Y. Wei, R. Devarakonda, R.S. Vose, and R.B.C. (2014) Available at: https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=1219." />
<meta name="citation_reference" content="Toft, C. &amp; Kimsey, L.S. (1982) Habitat and Behavior of Selected Apiocera and Rhaphiomidas (Diptera, Apioceridae), and Descriptions of Immature Stages of A. hispida. Journal of the Kansas Entomological Society, 55, 177–186." />
<meta name="citation_reference" content="Visser, M.E. (2008) Keeping up with a warming world; assessing the rate of adaptation to climate change. Proceedings. Biological sciences / The Royal Society, 275, 649–59." />
<meta name="citation_reference" content="Warren, D.L., Glor, R.E., &amp; Turelli, M. (2010) ENMTools: a toolbox for comparative studies of environmental niche models. Ecography, 33, 607–611." />
<meta name="citation_reference" content="Warren, D.L., Cardillo, M., Rosauer, D.F., &amp; Bolnick, D.I. (2014) Mistaking geography for biology: inferring processes from species distributions. Trends Ecology Evolution, 29, 572–80." />
<meta name="citation_reference" content="Warren, D.L., Beaumont, L., Dinnage, R., &amp; Baumgartner, J. (2018) New methods for measuring ENM breadth and overlap in environmental space. Ecography. doi:10.1111/ecog.03900" />
<meta name="citation_reference" content="Wright, S. (1934) An Analysis of Variability in Number of Digits in an Inbred Strain of Guinea Pigs. Genetics, 19, 506–36." />
<meta name="citation_reference" content="Zhang, Y. &amp; Hepner, G.F. (2017) Short-Term Phenological Predictions of Vegetation Abundance Using Multivariate Adaptive Regression Splines in the Upper Colorado River Basin. Earth Interactions, 21, 1–26." />
<meta name="citation_reference" content="Zohner, C.M. &amp; Renner, S.S. (2014) Common garden comparison of the leaf-out phenology of woody species from different native climates, combined with herbarium records, forecasts long-term change. Ecology Letters, 17, 1016–1025." />
<meta name="twitter:title" content="Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Aim Phenology, the temporal response of a population to its climate, is a crucial behavioral trait shared across life on earth. How species adapt their phenologies to climate change is poorly understood but critical in understanding how species will respond to future change. We use a group of flies ( Rhaphiomidas ) endemic to the North American deserts to understand how species adapt to changing climatic conditions. Here we explore a novel approach for taxa with constrained phenologies aimed to accurately model their environmental niche and relate this to phenological and morphological adaptations in a phylogenetic context.

Taxon Insecta, Diptera, Mydidae, Rhaphiomidas 

Location North America, Mojave, Sonoran and Chihuahuan Deserts.

Methods We gathered geographical and phenological occurrence data for the entire genus Rhaphiomidas , and, estimated a time calibrated phylogeny. We compared Daymet derived temperature values for a species adult occurrence period (phenology) with those derived from WorldClim data that is partitioned by month or quarter to examine what effect using more precise data has on capturing a species’ environmental niche. We then examined to what extent phylogenetic signal in phenological traits, climate tolerance and morphology can inform us about how species adapt to different environmental regimes.

Results We found that the Bioclim temperature data, which are averages across monthly intervals, poorly represent the climate windows to which adult flies are actually adapted. Using temporally-relevant climate data, we show that many species use a combination of morphological and phenological changes to adapt to different climate regimes. There are also instances where species changed only phenology to track a climate type or only morphology to adapt to different environments.

Main Conclusions Without using a fine-scale phenological data approach, identifying environmental adaptations could be misleading because the data do not represent the conditions the animals are actually experiencing. We find that fine-scale phenological niche models are needed when assessing taxa that have a discrete phenological window that is key to their survival, accurately linking environment to morphology and phenology. Using this approach, we show that Rhaphiomidas use a combination of niche tracking and adaptation to persist in new niches. Modeling the effect of phenology on such species’ niches will be critical for better predictions of how these species might respond to future climate change." />
<meta name="og-title" property="og:title" content="Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/534636v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Aim Phenology, the temporal response of a population to its climate, is a crucial behavioral trait shared across life on earth. How species adapt their phenologies to climate change is poorly understood but critical in understanding how species will respond to future change. We use a group of flies ( Rhaphiomidas ) endemic to the North American deserts to understand how species adapt to changing climatic conditions. Here we explore a novel approach for taxa with constrained phenologies aimed to accurately model their environmental niche and relate this to phenological and morphological adaptations in a phylogenetic context.

Taxon Insecta, Diptera, Mydidae, Rhaphiomidas 

Location North America, Mojave, Sonoran and Chihuahuan Deserts.

Methods We gathered geographical and phenological occurrence data for the entire genus Rhaphiomidas , and, estimated a time calibrated phylogeny. We compared Daymet derived temperature values for a species adult occurrence period (phenology) with those derived from WorldClim data that is partitioned by month or quarter to examine what effect using more precise data has on capturing a species’ environmental niche. We then examined to what extent phylogenetic signal in phenological traits, climate tolerance and morphology can inform us about how species adapt to different environmental regimes.

Results We found that the Bioclim temperature data, which are averages across monthly intervals, poorly represent the climate windows to which adult flies are actually adapted. Using temporally-relevant climate data, we show that many species use a combination of morphological and phenological changes to adapt to different climate regimes. There are also instances where species changed only phenology to track a climate type or only morphology to adapt to different environments.

Main Conclusions Without using a fine-scale phenological data approach, identifying environmental adaptations could be misleading because the data do not represent the conditions the animals are actually experiencing. We find that fine-scale phenological niche models are needed when assessing taxa that have a discrete phenological window that is key to their survival, accurately linking environment to morphology and phenology. Using this approach, we show that Rhaphiomidas use a combination of niche tracking and adaptation to persist in new niches. Modeling the effect of phenology on such species’ niches will be critical for better predictions of how these species might respond to future climate change." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-01-31" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/534636v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/534636v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/534636v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/534636v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/157565" />
    <title>Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"kpqV-QObootPvyPl6rlxA2X5vOf_94dJJaTyNmI-79c","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;534636v1"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;534636v1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;534636v1"}],"nid":"157565","apath":"\/biorxiv\/early\/2019\/01\/31\/534636.atom","pisa":"biorxiv;534636v1","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/534636v1","title":"Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling","identifier":"node\/157565"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eCPvrGM5WTtre8WBcVZqjgGRHgwvGLZQn5mrqepBr6x7NzbqXcp39+YvpRspdWpEk0CJaGVjAe7gmGXNQ0OiGD8aTJ0p3GNmjnHuUHfS2dNtXwHCekSObat9EdyoEJvnThv7Bf2jQc5SkGRDp0uQWZ6W27ksbA6QTr+nuuFINpFdHsI3C6SzE\\\\\\\/DL6ocV+u\\\\\\\/O7LRI1xZHUi8gE8tnhT5FBP+FbNBaWQ57zTxHf\\\\\\\/yu\\\\\\\/e+Ng\\\\\\\/5U4b6GI\\\\\\\/LsCosGUbl\\\\\\\/PhNY\\\\\\\/0KuMfsOpiTUDUHzbMurbRkPoSHCgIR59XeH1pkYgokZomyBzo17oSDGHG6q\\\\\\\/NzlXvWp2xCM7z9eBwLAiTg\\\\\\\/uLkIoS2yxxfm7AmmQmmXQrsWkbl7IsEw6UFI3fgfS7XJV51gyy72c+FYHZWLEUD0MiRSjLLV6qkoVgChDPhJSwZRlSPm2cyfbvB3HpgGURy4b7afm\\\\\\\/0iapLOxlbnmEMDFPmB1aKIEZbPLKhdSz0u0FArvAhZbZ6NEF2Q5KmePeoA\\\\\\\/N1L\\\\\\\/j1xq4Wv\\\\\\\/e3GhH6ATTHX7PAqk\\\\\\\/1Fzk8bPgjYJuI8\\\\\\\/ojlpDQg5Ajj\\\\\\\/MBiJZyXtTA1S68rclw5amfEBbyF0laOarzv2SG+uyrlT5\\\\\\\/THNs9LNF26P\\\\\\\/wBmtxusEOUa34qQU1wbY4HPSHYLobBBQxJzO1N2vGGxFvvxABx16D+iEYBy19SPtiiMzQ00828vIlLvu3fmroMGfQFmZv2LfKdjP8C1YOKDu2w+M\\\\\\\/dj4fKWa5rmTDFdD3hGuCFJkccYYoFy39\\\\\\\/pxKqPmLhm24RYz\\\\\\\/lovuElL6EaTuz0xpcta22svACC+I6hEOrhSgb8GqiLh9tcy94uGmwl0UIwKaEIVY97BeWugbUggK9MgWarJ43uUa\\\\\\\/b66W2\\\\\\\/zdq27x5w+oJuPBCEUzRHXSu+NXquVDb1wEkmuRTajVszjp2LLWcDfeMTXcJFYNW+1l8f11c+8XYrmL2RF\\\\\\\/SSBHPv\\\\\\\/\\\\\\\/dATRz2rjfifOvExJsm+fz2SuXRsTjfnbEzeBUM1lpE6x6ZM93FOEiAeLEFotK+H7RkcbmyYBpbqHi3EFz4HqB\\\\\\\/nV4IUrm2t1cAvpMskdGJljvjO1tgDJZpQMLigWHw+tO7RktihmrQlA5QGX6YyYdSBfUYTqRkNeAZDjaCcGNpoNkWIgbtR\\\\\\\/P6QBnEZO3SxLMa+v8C+GqPXGH2BM0gzfDKFHhXJjGg9Djx300hFuj\\\\\\\/ev4ae2gDpo1f9Fl235rlYps5DA8gG+SfHTUlcGIRZ8E6jLvXNOxprbY4N4P7fa1FXVRC3Vnpzo9X7d3qBYnMYzs9DMzwMTN33BZPYa6T0lHHAi\\\\\\\/U=\\\u0022,\\\u0022iv\\\u0022:\\\u0022SU+g4a2Y+sxrFLYvhNTWZg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022c81c964e601d798d7766fc5a1dc2d637ba4ea4ba8a7047d4a26c428608884eb7\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/534636v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-157565 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/534636v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nFBTVIqCPyhOb2WPys_ViwojbdLiN0pZqkv76EUrCbk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="157565" id="node157565" data-pisa="biorxiv;534636v1" data-pisa-master="biorxiv;534636" data-apath="/biorxiv/early/2019/01/31/534636.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7473-9727" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Matthew H. Van</span> <span class="nlm-surname">Dam</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3755-4480" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Andrew J.</span> <span class="nlm-surname">Rominger</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael S.</span> <span class="nlm-surname">Brewer</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/534636 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node157565" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew H. Van Dam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Environmental Science</span>, Policy, and Management, University of California, Berkeley, CA, 94720-3140 USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BH.%2BVan%2BDam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dam%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BH.%2BVan%2BDam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7473-9727" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew H. Van Dam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:matthewhvandam@gmail.com" class="" data-icon-position="" data-hide-link-title="0">matthewhvandam@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Andrew J. Rominger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Environmental Science</span>, Policy, and Management, University of California, Berkeley, CA, 94720-3140 USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrew%2BJ.%2BRominger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rominger%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAndrew%2BJ.%2BRominger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3755-4480" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Andrew J. Rominger</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael S. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Environmental Science</span>, Policy, and Management, University of California, Berkeley, CA, 94720-3140 USA</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BS.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20MS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BS.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/534636v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:157565" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:157565/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/534636v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:157565" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:157565/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/534636v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:157565" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:157565/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/534636v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:157565" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:157565/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/534636v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:157565" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:157565/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><div id="sec-1" class="subsection"><p id="p-5"><strong>Aim</strong> Phenology, the temporal response of a population to its climate, is a crucial behavioral trait shared across life on earth. How species adapt their phenologies to climate change is poorly understood but critical in understanding how species will respond to future change. We use a group of flies (<em>Rhaphiomidas</em>) endemic to the North American deserts to understand how species adapt to changing climatic conditions. Here we explore a novel approach for taxa with constrained phenologies aimed to accurately model their environmental niche and relate this to phenological and morphological adaptations in a phylogenetic context.</p></div><div id="sec-2" class="subsection"><p id="p-6"><strong>Taxon</strong> Insecta, Diptera, Mydidae, <em>Rhaphiomidas</em></p></div><div id="sec-3" class="subsection"><p id="p-7"><strong>Location</strong> North America, Mojave, Sonoran and Chihuahuan Deserts.</p></div><div id="sec-4" class="subsection"><p id="p-8"><strong>Methods</strong> We gathered geographical and phenological occurrence data for the entire genus <em>Rhaphiomidas</em>, and, estimated a time calibrated phylogeny. We compared Daymet derived temperature values for a species adult occurrence period (phenology) with those derived from WorldClim data that is partitioned by month or quarter to examine what effect using more precise data has on capturing a species’ environmental niche. We then examined to what extent phylogenetic signal in phenological traits, climate tolerance and morphology can inform us about how species adapt to different environmental regimes.</p></div><div id="sec-5" class="subsection"><p id="p-9"><strong>Results</strong> We found that the Bioclim temperature data, which are averages across monthly intervals, poorly represent the climate windows to which adult flies are actually adapted. Using temporally-relevant climate data, we show that many species use a combination of morphological and phenological changes to adapt to different climate regimes. There are also instances where species changed only phenology to track a climate type or only morphology to adapt to different environments.</p></div><div id="sec-6" class="subsection"><p id="p-10"><strong>Main Conclusions</strong> Without using a fine-scale phenological data approach, identifying environmental adaptations could be misleading because the data do not represent the conditions the animals are actually experiencing. We find that fine-scale phenological niche models are needed when assessing taxa that have a discrete phenological window that is key to their survival, accurately linking environment to morphology and phenology. Using this approach, we show that <em>Rhaphiomidas</em> use a combination of niche tracking and adaptation to persist in new niches. Modeling the effect of phenology on such species’ niches will be critical for better predictions of how these species might respond to future climate change.</p></div></div><div class="section" id="sec-7"><h2 class="">INTRODUCTION</h2><p id="p-11">Evaluating species response to climate change and predicting future distributions is of concern to ecologists, evolutionary biologists and policy makers interested in preserving biodiversity (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Parmesan, 2006</a>). In addition, identifying whether a particular species or population adapts to climate change by adjusting its phenology or morphological/physiological traits, as opposed to changing its range via dispersal, is key to predicting future distributions, extinctions (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Holt, 1990</a>) and community disassembly (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sheldon et al., 2011</a>). Dispersal is often assumed to be the primary response of populations and species to climate change (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Thomas et al., 2004</a>); however, this might not be the case (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Visser, 2008</a>). Dispersal limitation and heterogeneous or fragmented landscapes could preclude spatial tracking of climatic niches (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hof et al., 2011</a>). In such cases, populations must adapt their phenology and/or morphology in order to persist (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hoffmann &amp; Sgrò, 2011</a>). Such adaptation, however, is often not accounted for in models of species’ responses to climate change (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bradshaw &amp; Holzapfel 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">van Asch et al. 2007</a>).</p><p id="p-12">Phenology plays a critical role in the reproductive cycles and developmental processes of many organisms (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Mitchell et al., 2008</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Banta et al., 2012</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zohner &amp; Renner, 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gerst et al., 2017</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhang &amp; Hepner, 2017</a>). The past century of climate change has already produced measurable changes in phenology, especially in plants (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ellwood et al., 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Everill et al., 2014</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Zohner &amp; Renner, 2014</a>). Therefore, finding a better way to model critical environmental conditions linked to phenological cycles and examining how different species adapt their phenologies is important for understanding species’ ecologies and their conservation. Phenological responses have been identified and tested through long-term ecological studies (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Miller-Rushing &amp; Primack, 2008</a>) but can also be understood in a phylogenetic context along with morphological evolution.</p><p id="p-13">Here, we use a group of flies (Diptera: Mydidae: <em>Rhaphiomidas</em> Osten Sacken, 1877; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>) endemic to the North American deserts to understand how species adapt to changing climatic conditions. Specifically, we are interested in whether the flies change their emergence time to track the same basic seasonal climate window, or whether the adult flies adapt physiologically/morphologically to changing climate during that window.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Upper left, Rhaphiomidas pachyrhynchus adult male, R. sp. &#x201C;pseudonigricaudis&#x201D; adult male. Central panel, Activity duration of Rhaphiomidas species. Upper right, daily values over a species&#x2019; adult activity length. Lower left, resulting raster layers for each environmental variable averaged or summed in the case of precipitation over the activity length period of a species. Lower right, resulting environmental niche model from custom raster layers for a species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790020158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;Upper left&lt;/strong&gt;, &lt;em&gt;Rhaphiomidas pachyrhynchus&lt;/em&gt; adult male, &lt;em&gt;R. sp. &#x201C;pseudonigricaudis&#x201D;&lt;/em&gt; adult male. &lt;strong&gt;Central panel&lt;/strong&gt;, Activity duration of &lt;em&gt;Rhaphiomidas&lt;/em&gt; species. &lt;strong&gt;Upper right&lt;/strong&gt;, daily values over a species&#x2019; adult activity length. &lt;strong&gt;Lower left&lt;/strong&gt;, resulting raster layers for each environmental variable averaged or summed in the case of precipitation over the activity length period of a species. &lt;strong&gt;Lower right&lt;/strong&gt;, resulting environmental niche model from custom raster layers for a species.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F1.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F1.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-14" class="first-child"><strong>Upper left</strong>, <em>Rhaphiomidas pachyrhynchus</em> adult male, <em>R. sp. “pseudonigricaudis”</em> adult male. <strong>Central panel</strong>, Activity duration of <em>Rhaphiomidas</em> species. <strong>Upper right</strong>, daily values over a species’ adult activity length. <strong>Lower left</strong>, resulting raster layers for each environmental variable averaged or summed in the case of precipitation over the activity length period of a species. <strong>Lower right</strong>, resulting environmental niche model from custom raster layers for a species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">The genus <em>Rhaphiomidas</em> is distributed throughout the deserts of the Southwestern United States and Northern Mexico (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cazier, 1985</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Van Dam, 2010</a>). Most species of <em>Rhaphiomidas</em> feed on floral nectar as adults and thus phenological matching with their plant resources may be critical for both mutualists. Species are either restricted to aeolian (wind-blown sediments) sand dunes (15/27 species) or loose alluvial sands (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Van Dam, 2010</a>), with many species (9/27) endemic to a single dune system. <em>Rhaphiomidas</em> adults fly in the spring and fall and are most active during spring and fall blooms. Some species do not appear to feed as adults, making the synchronization of their activity all the more important for reproductive success. Multiple studies have been conducted on the behavior of adult flies, noting the temperatures requirements for adult flight activity (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Toft &amp; Kimsey 1982</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ballmer <em>et al.</em> 1994</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rogers &amp; Mattoni 1993</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kingsley 1996</a>). Larvae and pupae of <em>Rhaphiomidas</em> are entirely subterranean at depths of up to 1.5 m (M.H.V.D pers. obs.) and thus only experience the ambient climate for the short duration of their lives spent above ground. This dramatically discrete phenology makes them ideal for studying phenological response to climate.</p><p id="p-16">Studies of climatic niches typically depend on species distribution models to understand the correlation between a species’ occurrences and underlying climatic variables (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Warren et al., 2018</a>). Such studies rarely incorporate phenology when reconstructing niche occupancy, often using only monthly averaged WorldClim data (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hijmans et al., 2005</a>). Constraining climate data to monthly averages may misrepresent the climate that species are actually experiencing. Although the WorldClim variables are appropriate for capturing realized niches based on abiotic environmental factors experienced year-round, such as for perennial plants and some mammals, reliance on these variables could lead to biased predictions for an entire suite of organisms that have discrete phenologies.</p><p id="p-17">To solve this problem, we used climate data that were averaged by day at a 1km square scale available from Daymet (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Thornton et al. 2014</a>). To measure how species adapt to different climate conditions, the Daymet-based approach was put in a phylogenetic framework to test whether phenological adaptation (niche tracking) or physiological/morphological adaptation (niche adapting) has occurred. A phylogenetic framework allows us to account for the role of shared evolutionary history (Felsenstein 1985) in producing the observed correlations between traits and niche preference. We can also infer the evolutionary history of adult niche preference and measure the rate at which changes have occurred.</p><p id="p-18">Understanding how species adapt to climate change in different climate regimes by adapting their morphology and physiology is also important for mitigating extinction risk due to climate change (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Visser, 2008</a>). For many species that rely on ambient temperature for thermoregulation, climate change is predicted to have a negative effect on their survival (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Quintero &amp; Wiens, 2013</a>). In general, species or populations that are poikilotherms (commonly known as cold-blooded) have a darker coloration in colder environments (Shapiro, 1976; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kingsolver &amp; Wiernasz 1991</a>). For example, reptiles become darker in colder conditions to maintain activity (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bittner et al., 2002</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Rosenblum, 2005</a>). This adaptation helps them achieve metabolic activity more quickly. We propose that the dark coloration seen in some <em>Rhaphiomidas</em> species is an adaptation facilitating absorption of more solar energy, thereby allowing increased activity in cooler climates.</p></div><div class="section" id="sec-8"><h2 class="">METHODS</h2><div id="sec-9" class="subsection"><h3>Analysis Pipeline</h3><p id="p-19">In order to test for phenological changes that track a niche and/or if species adapt to environmental conditions, we analyzed a diverse set of data (environmental and phylogenetic). The methods can be broken down into five parts:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-20">We processed environmental variables and constructed niche models.</p></li><li id="list-item-2"><p id="p-21">We created predicted niche occupancy profiles (PNO, in particular thermal maximum) to relate the physical temperature corresponding to probability densities of the niche model (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elith et al., 2011</a>).</p></li><li id="list-item-3"><p id="p-22">We constructed a time-calibrated phylogeny for <em>Rhaphiomidas</em>.</p></li><li id="list-item-4"><p id="p-23">Using the phylogeny, we tested whether species possessing dark coloration are correlated with colder thermal maxima, indicating an adaptive response to colder temperatures and examined ancestral state reconstruction using phylogenetic comparative methods to ensure this wasn’t an artifact of inheritance.</p></li><li id="list-item-5"><p id="p-24">We combined phylogenetic reconstructions of traits (morphology, phenology and temperature) to see if sister species changed their phenology relative to changes in temperature and morphology.</p></li></ol></div><div id="sec-10" class="subsection"><h3>DNA Data and Alignment</h3><p id="p-25"><em>Rhaphiomidas</em> sequences were taken from <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Van Dam &amp; Matzke (2016)</a>. Contigs were assembled and edited in Geneious Pro v. 4.6.4 (Biomatters Ltd.). Sequences were aligned by ClustalW-2.0.10 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Larkin et al., 2007</a>) with parameters set to GAPOPEN=90.0, GAPEXT=10. Sequences were colored by amino acid in Mesquite version 2.71 (Build 514) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Maddison &amp; Maddison 2009</a>) to check for stop codons.</p></div><div id="sec-11" class="subsection"><h3>Phylogenetic Analyses</h3><p id="p-26">We obtained data for 219 individuals, including 183 <em>Rhaphiomidas</em> exemplars and 36 outgroup samples. These data comprised 2904 bp of mtDNA (COI, COII, and 16S genes), and 3720 bp of nDNA (EF1alpha, PGD, snf, Wg, and CAD). The best partitioning scheme was determined in PartitionFinder 1.1.1 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lanfear et al., 2012</a>) using the ‘greedy’ algorithm. The partitioning scheme was selected using the Akaike information criterion corrected for sample size (AICc). Relationships among <em>Rhaphiomidas</em> species were reconstructed using Bayesian Inference (BI) implemented in MrBayes 3.2 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ronquist et al., 2012</a>). Reversible-jump MCMC was used to explore the entire model space of the general time reversible (GTR) substitution models for our different character sets (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Huelsenbeck et al., 2004</a>). We ran two independent runs comprising eight total Markov Chains for 70 million generations, sampling every 1000th generation. Split-frequencies and log-likelihood curves were examined in Tracer 1.5.3 (Rambaut &amp; Drummond 2018).</p><p id="p-27">The first 35,000 of the 70,000 trees sampled were removed as burnin. Standard deviation of split-frequencies &lt;0.05 between the two MrBayes runs suggested reasonable convergence. A maximum clade credibility tree was constructed with <em>DendroPy SumTrees</em> function (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sukumaran &amp; Holder, 2010</a>) using median edge lengths. This tree was inspected for evidence of mitochondrial introgression as in <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">McGuire et al., (2007)</a>. The dataset was then pruned to a single specimen per species, preferring specimens with the most complete sequence that did not show signs of mitochondrial introgression. This was done because Yule and Birth Death (BD) tree priors in BEAST treat every tip as a species; a tree with multiple specimens for each species can result in a biased date inference (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Drummond &amp; Bouckaert, 2015</a>). The MrBayes tree was pruned to the same specimens, and the pruned dataset was analyzed in BEAST version 1.8.2 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Drummond et al., 2012</a>) with the pruned MrBayes tree serving as the starting topology for phylogenetic dating analyses. We used stepping stone (SS) sampling (Drummond et al., 2016) to select the tree prior (Birth-Death chosen over Yule) and clock model (a lognormal relaxed clock chosen over strict clock). The MCMC process was run for 4×10<sup>7</sup> generations sampling every 1000 generations. Stationarity was assessed using Tracer version 1.5.3. The effective sample size (ESS) for model parameters was also examined to assess the adequacy of the post-burnin samples. The tree was calibrated using a fossil calibration point for the origin of the Mydinae in the late Cretaceous (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Martill et al., 2007</a>) using a normal distribution with a mean of 120 Ma and a standard deviation of ±10 Ma.</p></div><div id="sec-12" class="subsection"><h3>Data Acquisition/Niche Modeling</h3><p id="p-28">Specimen data for <em>Rhaphiomidas</em> were acquired from museum collections (EMEC, CAS, UCR, LACM) and previously published literature (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cazier 1985</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Rogers and Mattoni 1993</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rogers and Van Dam 2007</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Van Dam 2010</a>), as well as from the personal collection of M.H.V.D. Locations were georeferenced in ArcGIS (ESRI 2011) and were recorded in both WGS_84 (degrees-minutes-seconds) and Lambert Conformal Conic projection required for Daymet georeferencing (<a href="http://daymet.ornl.gov/datasupport">http://daymet.ornl.gov/datasupport</a>). A total of 409 occurrence points were included, with an average of 15 per species, median 13 and a range of 10–40. Several species are endemic to a single dune system, so 10 points were randomly scattered across the spatial extent of such dune fields. This number was selected because it has been demonstrated to be a reasonable minimum number of points needed for species distribution models using Maxent (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hernandez et al., 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Evans et al., 2009</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">van Proosdij et al. 2016</a>). To obtain the most accurate possible temperature estimates when adult flies are active, we extracted climate data from the Daymet climate data base (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Thornton et al. 2014</a>). This allowed us to partition climate data by day and average over a species’ occurrence time (typically weeks) rather than using monthly averages. A list of the Daymet tiles is provided in the Supporting Information. The 1 km by 1 km resolution tiles were downloaded from years 1980–2011. A total of six variables were used from the Daymet database (vapor pressure <em>vp</em>, day length <em>dayl</em>, precipitation <em>prcp</em>, solar radiation <em>srad</em>, temperature max <em>tmax</em> and temperature minimum <em>tmin</em>). The data totaled just over 2 terabytes of information. The first and last occurrence times were recorded from the specimen data (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>) and used to define the time slices to extract from the Daymet climate data.</p><p id="p-29">Extracted environmental variables were averaged over the 30 years. Precipitation data were summed by time slice and then averaged across years. This procedure was performed in R statistical software with a custom script (see Supporting Information). We utilized the R packages <em>ncdf4</em>, <em>raster</em>, <em>maps</em> and <em>dismo</em> for this process (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hijmans &amp; van Etten, 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Becker et al., 2017</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hijmans et al., 2017</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pierce, 2017</a>). Niche models were constructed in MaxEnt (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Phillips et al., 2006</a>) using the R package <em>dismo</em>. We examined collinearity of predictors using <em>SDMTools</em> (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Warren et al., 2018</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Warren et al., 2010</a>).</p><p id="p-30">To evaluate the extent to which Daymet and BioClim representations of the same geographic point diverge, we extracted temperature values from each respective raster layer for the same point. Because <em>Rhaphiomidas</em> are late spring and summer active species, we compared the values of the tmax (Daymet) data with Bioclim BIO5 (Max Temperature of Warmest Month) and BIO10 (Mean Temperature of Warmest Quarter).</p></div><div id="sec-13" class="subsection"><h3>Predicted Niche Occupancy</h3><p id="p-31">To identify how <em>Rhaphiomidas</em> lineages adapted to change in different climate regimes, we constructed each species' predicted niche occupancy (PNO) profile from the cumulative probability of occurrence against the environmental data (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Evans et al., 2009</a>). We then used the PNO to calculate the weighted mean for each species. PNO profiles were constructed in <em>phyloclim</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Heibl &amp; Calenge, 2013</a>), and 100 random samples were drawn from the PNO profile as in <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Evans et al. (2009)</a> to calculate the weighted mean. Calculations for PNOs from the Daymet data were derived using a slightly different method than those for the Bioclim data. Because each one of our Daymet environmental layers is unique to each taxon, some values were not found among layers. Thus, we first calculated the range across all individual taxa layers for a variable and then merged the PNO files by taking the floor to the nearest °C. For instance, 32.5456845°C in species-A PNO and 32.59975962°C in species-B PNO were changed to 32°C in the two PNOs, so they can then be used for calculating the weighted means as above.</p></div><div id="sec-14" class="subsection"><h3>Evaluating the correlation between temperature and body color of Rhaphiomidas</h3><p id="p-32">We categorized fly coloration according to the following rules:1) if the first 3 abdominal tergites were almost entirely black, they were coded as dark, 2) if the first 3 abdominal tergites were orange or silver they were coded as pale. Because there were no intermediate states, we treated these as discrete.</p><p id="p-33">To examine whether colder daytime temperatures are correlated with a darker body color while accounting for shared evolutionary history, we used a threshold model from quantitative genetics (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wright, 1934</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Felsenstein, 2012</a>), using MCMC to sample the unknown liabilities in a postulated continuous trait underlying a discrete character. We used the R package <em>phytools</em> 0.3-72 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Revell, 2012</a>), which implements this model in the function <em>threshBayes</em>. To account for differences in branch lengths and topologies from our posterior distribution of trees, we sampled 10% of the posterior trees resulting in 3900 sampled trees. We then measured the correlation between the two characters over this set of trees. We ran the MCMC chain for 5×10<sup>6</sup> generations sampling every 1,000 generations for each tree. After examining the trace plot of the posterior, we set a burn-in of 10%.</p></div><div id="sec-15" class="subsection"><h3>Identifying Coloration Shifts</h3><p id="p-34">To identify shifts in coloration, we reconstructed ancestral states for coloration on the phylogeny. First, we sorted the trees from the BEAST posterior (minus the burnin) according to their topologies (grouping identical topologies together). We used the <em>DendroPy SumTrees</em> function (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Sukumaran &amp; Holder, 2010</a>), taking median edge lengths, to create a representative strict consensus tree for each set of unique topologies. We then used a Bayesian threshold model implemented in <em>phytools</em> with the function <em>ancThresh</em>, running 2.5×10<sup>5</sup> MCMC generations sampling every 1,000 with a burnin of 50 over each unique topology.</p></div><div id="sec-16" class="subsection"><h3>Ancestral State Reconstructions and Evaluating Climate Disparity Through Time</h3><p id="p-35">To reconstruct ancestral states for the continuous traits of temperature and occurrence time (phenology), we first determined the best evolutionary model, Brownian Motion (BM) or Ornstein−Uhlenbeck (OU), given our tree and data using the <em>fitContinuous</em> function in <em>geiger</em> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Harmon et al., 2008</a>). In instances where the OU model was selected, we used a modified version of the <em>phytools</em> function <em>contMap</em> where the internal <em>anc.ML</em> function was set to the OU model.</p><p id="p-36">Ancestral state estimates were used to measure the relative amount of disparity between sister nodes using <em>geiger</em>'s disparity function to calculate the average squared Euclidean distance in each clade. To visualize the amount of change between nodes on the tree, we plotted a phenogram for both traits using the <em>phenogram</em> function of <em>phytools</em>. We also calculated the difference between parent and daughter node values (normalized temperature and phenology) from our <em>contMap</em> reconstructions using a custom R script utilizing functions from BioGeoBEARS (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Matzke 2014</a>). These values were used to color the branches and help visualize the change in trait values along the tree.</p></div></div><div class="section" id="sec-17"><h2 class="">RESULTS</h2><div id="sec-18" class="subsection"><h3>Molecular Data and Phylogenetics</h3><p id="p-37">The BEAST analysis indicated that <em>Rhaphiomidas</em> is sister to the remaining Mydidae, and these groups together are sister to the Apioceridae (see Supporting Information). All model parameters had an ESS (effective sample size) &gt;200, suggesting sufficient sampling of the posterior distribution. The root node of <em>Rhaphiomidas</em> is estimated to have diverged 70 ± 42Ma (node height 95% HPD), with most of the species diversifying in the last 18.5 ± 11Ma. Terminals showing evidence of introgression were removed, leaving a topology that was also congruent with species concepts. From the Stepping Stone sampling results, we rejected the strict clock prior and Yule tree prior in favor of the relaxed lognormal clock prior and the Birth-Death tree model (Supporting Information).</p></div><div id="sec-19" class="subsection"><h3>Niche Modeling Results</h3><p id="p-38">First, we compared the distributions of point values for temperature extracted from raster layers between the Daymet tmax data and Bioclim BIO5 and BIO10. We found that the Bioclim data did not adequately reflect the conditions that these species were experiencing as adults (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). Many of the Bioclim values were outside of the species’ temperature range and further did not show differentiation between allochronic species, especially in the case of BIO5. Because the measurements of these two data sets are otherwise congruent (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hijmans et al. 2005</a>), this difference is mostly due to the time periods binned in making the rasters. As the maximum temperature is key to understanding further hypotheses of morphological adaptations, we proceeded using the Daymet raster layers because they are more representative of the environmental conditions that these species experience.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box plots of temperature values extracted from occurrence points: Mean daily maximum temperature (tmax) from Daymet data (yellow), Bioclim Bio10 mean temperature warmest quarter (blue), Bioclim Bio5 maximum temperature warmest month (green). R. &#x201C;arenagena&#x201D; and R. &#x201C;rex&#x201D; are partially sympatric but emerge at different times (allochronic). R. hasbroucki and R. parkeri are also partially sympatric but are allochronic. R. tarsalis is the sister taxon to R. hasbroucki, they are allopatric and allochronic." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790020158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Box plots of temperature values extracted from occurrence points: Mean daily maximum temperature (tmax) from Daymet data (yellow), Bioclim Bio10 mean temperature warmest quarter (blue), Bioclim Bio5 maximum temperature warmest month (green). &lt;em&gt;R. &#x201C;arenagena&#x201D;&lt;/em&gt; and &lt;em&gt;R. &#x201C;rex&#x201D;&lt;/em&gt; are partially sympatric but emerge at different times (allochronic). &lt;em&gt;R. hasbroucki&lt;/em&gt; and &lt;em&gt;R. parkeri&lt;/em&gt; are also partially sympatric but are allochronic. &lt;em&gt;R. tarsalis&lt;/em&gt; is the sister taxon to &lt;em&gt;R. hasbroucki&lt;/em&gt;, they are allopatric and allochronic.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F2.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F2.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-39" class="first-child">Box plots of temperature values extracted from occurrence points: Mean daily maximum temperature (tmax) from Daymet data (yellow), Bioclim Bio10 mean temperature warmest quarter (blue), Bioclim Bio5 maximum temperature warmest month (green). <em>R. “arenagena”</em> and <em>R. “rex”</em> are partially sympatric but emerge at different times (allochronic). <em>R. hasbroucki</em> and <em>R. parkeri</em> are also partially sympatric but are allochronic. <em>R. tarsalis</em> is the sister taxon to <em>R. hasbroucki</em>, they are allopatric and allochronic.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The collinearity between the predictor variables varied between strongly correlated (tmax and tmin) to highly uncorrelated (vapor pressure and solar radiation) (see Supporting Information). Given the small number of predictors and the biological importance of tmax and tmin we left both of these predictor variables in further analyses of species’ niches. The contribution of the six bioclimatic variables varied considerably between species. Variables, such as precipitation, consistently contributed to large percentages of the model probability – 90% in the case of <em>R. undulatus</em>. Other variables such as DayLength contributed modestly to the model predictions, ranging up to 30% for <em>R. xanthos</em>. AUCs varied from 0.65 to 1 (see Supporting Information). The weighted mean for tmax ranged from 25.1 – 38.5° C across species.</p></div><div id="sec-20" class="subsection"><h3>Evaluating the Correlation between Temperature and Body Color</h3><p id="p-41">The mean correlation coefficient between color and temperature is −0.70 over all trees, indicating that as temperature decreases, there is a trait change from pale to dark (pale coded as 0 and dark coded as 1; <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). The 95% HPD varied from −0.93 to −0.25, and none of the individual trees examined had a 95% HPD that overlapped with zero (Supporting Information). Because this is a Bayesian analysis, significance is assessed through the 95% credible interval and its overlap with 0. Our 95% credible interval does not overlap with zero (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="threshBayes posterior density distribution of the correlation between Rhaphiomidas body color (dark or light) and Daymet mean daily maximum temperature. The histogram shows the posterior distributions of 3900 trees (10% of the sampled trees), the solid red line indicates the grand mean, the dashed lines indicate the mean of the 95% confidence intervals. None of the individual 95% confidence intervals from the set of trees overlap with zero, indicating a significant correlation, robust to topology and branch lengths." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790020158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;threshBayes&lt;/em&gt; posterior density distribution of the correlation between &lt;em&gt;Rhaphiomidas&lt;/em&gt; body color (dark or light) and Daymet mean daily maximum temperature. The histogram shows the posterior distributions of 3900 trees (10% of the sampled trees), the solid red line indicates the grand mean, the dashed lines indicate the mean of the 95% confidence intervals. None of the individual 95% confidence intervals from the set of trees overlap with zero, indicating a significant correlation, robust to topology and branch lengths.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F3.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F3.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-42" class="first-child"><em>threshBayes</em> posterior density distribution of the correlation between <em>Rhaphiomidas</em> body color (dark or light) and Daymet mean daily maximum temperature. The histogram shows the posterior distributions of 3900 trees (10% of the sampled trees), the solid red line indicates the grand mean, the dashed lines indicate the mean of the 95% confidence intervals. None of the individual 95% confidence intervals from the set of trees overlap with zero, indicating a significant correlation, robust to topology and branch lengths.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Identifying Coloration Shifts</h3><p id="p-43">The common ancestor of <em>Rhaphiomidas pachyrhynchus</em> and <em>R. episcopus</em> was estimated to have dark coloration in all tree topologies. Only 2/48 topologies (representing only 3/3900 trees) had any other nodes reconstructed as dark in coloration. Thus, the ancestral states are likely pale for all but one node in the tree. The results of the maximum clade credibility tree are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rhaphiomidas maximum clade credibility tree, with ancestral state reconstructions for body coloration. Ancestral light or dark coloration was reconstructed via the ancThresh function in the phytools R package (Revell 2012). The analysis was run for 2.5&#xD7;106 MCMC generations, sampling every 1000, with a burnin of 50." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790020158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Rhaphiomidas&lt;/em&gt; maximum clade credibility tree, with ancestral state reconstructions for body coloration. Ancestral light or dark coloration was reconstructed via the &lt;em&gt;ancThresh&lt;/em&gt; function in the &lt;em&gt;phytools&lt;/em&gt; R package (Revell 2012). The analysis was run for 2.5&#xD7;10&lt;sup&gt;6&lt;/sup&gt; MCMC generations, sampling every 1000, with a burnin of 50.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F4.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F4.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-44" class="first-child"><em>Rhaphiomidas</em> maximum clade credibility tree, with ancestral state reconstructions for body coloration. Ancestral light or dark coloration was reconstructed via the <em>ancThresh</em> function in the <em>phytools</em> R package (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Revell 2012</a>). The analysis was run for 2.5×10<sup>6</sup> MCMC generations, sampling every 1000, with a burnin of 50.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Ancestral State Reconstructions and Evaluating Climate Disparity Through Time</h3><p id="p-45">The OU model was an overwhelmingly better fit than BM for weighted mean tmax dAICc [OU:0, BM:32.6278], but BM was slightly preferred for phenology dAICc [OU:0.441541, BM:0].</p><p id="p-46">The results of the <em>contMap</em> reconstructions suggest most species of <em>Rhaphiomidas</em> tend to occupy a relatively warm climate space, and most species occur in late spring with only a few phenological shifts from spring to fall (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a>). We find relatively few nodes where there is more disparity in temperature than in phenology (6 of 26), with 4 of 26 nodes showing approximately equal disparity between temperature and phenology and 16 of 26 nodes showing more disparity in phenology than in temperature (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6</a>). In addition, the largest shifts in temperature occur at the terminal nodes in the tree, whereas large shifts in phenology occur at both internal and terminal nodes (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the contMap ancestral state reconstructions. Left tree, ancestral state reconstructions for mean daily temperature maximum value, warmer colors along branches indicate warmer maximum temperatures. Right tree, ancestral state reconstructions for mid-occurrence times, warmer colors along branches indicate occurrence times later in the year. Orange rectangles indicate species with a light coloration, black boxes indicate species with a dark coloration." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790020158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results of the &lt;em&gt;contMap&lt;/em&gt; ancestral state reconstructions. Left tree, ancestral state reconstructions for mean daily temperature maximum value, warmer colors along branches indicate warmer maximum temperatures. Right tree, ancestral state reconstructions for mid-occurrence times, warmer colors along branches indicate occurrence times later in the year. Orange rectangles indicate species with a light coloration, black boxes indicate species with a dark coloration.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F5.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F5.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-47" class="first-child">Results of the <em>contMap</em> ancestral state reconstructions. Left tree, ancestral state reconstructions for mean daily temperature maximum value, warmer colors along branches indicate warmer maximum temperatures. Right tree, ancestral state reconstructions for mid-occurrence times, warmer colors along branches indicate occurrence times later in the year. Orange rectangles indicate species with a light coloration, black boxes indicate species with a dark coloration.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cladewise disparity in phenology and temperature. Top, Rhaphiomidas chronogram with nodes colored by age, correlating to colors used in the panels below. Middle, temperature and phenological cladewise disparity of each subtree measured as average squared Euclidean distance. The Y-axis represents the cladewise disparity, and the X-axis is measured in millions of years. Bottom, temperature and phenological cladewise disparity of each subtree measured as average squared Euclidean distance. The Y-axis represents the cladewise disparity, and the X-axis is node chronological order, oldest to youngest. Arrows indicate nodes that are followed by a transition from light to dark coloration along one of their descendant branches." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790020158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cladewise disparity in phenology and temperature. &lt;strong&gt;Top&lt;/strong&gt;, &lt;em&gt;Rhaphiomidas&lt;/em&gt; chronogram with nodes colored by age, correlating to colors used in the panels below. &lt;strong&gt;Middle&lt;/strong&gt;, temperature and phenological cladewise disparity of each subtree measured as average squared Euclidean distance. The Y-axis represents the cladewise disparity, and the X-axis is measured in millions of years. &lt;strong&gt;Bottom&lt;/strong&gt;, temperature and phenological cladewise disparity of each subtree measured as average squared Euclidean distance. The Y-axis represents the cladewise disparity, and the X-axis is node chronological order, oldest to youngest. Arrows indicate nodes that are followed by a transition from light to dark coloration along one of their descendant branches.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F6.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F6.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-48" class="first-child">Cladewise disparity in phenology and temperature. <strong>Top</strong>, <em>Rhaphiomidas</em> chronogram with nodes colored by age, correlating to colors used in the panels below. <strong>Middle</strong>, temperature and phenological cladewise disparity of each subtree measured as average squared Euclidean distance. The Y-axis represents the cladewise disparity, and the X-axis is measured in millions of years. <strong>Bottom</strong>, temperature and phenological cladewise disparity of each subtree measured as average squared Euclidean distance. The Y-axis represents the cladewise disparity, and the X-axis is node chronological order, oldest to youngest. Arrows indicate nodes that are followed by a transition from light to dark coloration along one of their descendant branches.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Color phenogram for mean daily maximum temperature (right half) and phenology (left half). Branch colors represent relative differences from parent to daughter nodes, similar to differences in the placement of parent to daughter nodes along the Y-axis as represented in the phenograms below, but phenogram nodes represent values not relative differences. Left half colors represent shifts to warmer or cooler temperatures between nodes (warmer colors = warmer shifts in temp., cooler colors = colder shifts in temp. relative to parent node), right half branch colors represent shifts in phenology (shifts later in year = warmer colors, earlier in year = cooler colors) relative to parent node. Orange rectangles indicate species with a light coloration, and black boxes indicate species with a dark coloration." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1790020158" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Color phenogram for mean daily maximum temperature (right half) and phenology (left half). Branch colors represent relative differences from parent to daughter nodes, similar to differences in the placement of parent to daughter nodes along the Y-axis as represented in the phenograms below, but phenogram nodes represent values not relative differences. Left half colors represent shifts to warmer or cooler temperatures between nodes (warmer colors = warmer shifts in temp., cooler colors = colder shifts in temp. relative to parent node), right half branch colors represent shifts in phenology (shifts later in year = warmer colors, earlier in year = cooler colors) relative to parent node. Orange rectangles indicate species with a light coloration, and black boxes indicate species with a dark coloration.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F7.medium.gif" width="417" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F7.medium.gif" width="417" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/31/534636/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7.</span> <p id="p-49" class="first-child">Color phenogram for mean daily maximum temperature (right half) and phenology (left half). Branch colors represent relative differences from parent to daughter nodes, similar to differences in the placement of parent to daughter nodes along the Y-axis as represented in the phenograms below, but phenogram nodes represent values not relative differences. Left half colors represent shifts to warmer or cooler temperatures between nodes (warmer colors = warmer shifts in temp., cooler colors = colder shifts in temp. relative to parent node), right half branch colors represent shifts in phenology (shifts later in year = warmer colors, earlier in year = cooler colors) relative to parent node. Orange rectangles indicate species with a light coloration, and black boxes indicate species with a dark coloration.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-23"><h2 class="">DISCUSSION</h2><p id="p-50">Our results indicate that extracting climate data to match species’ phenological activity better exemplifies environmental conditions than the traditional Bioclim variables. Understanding the basic life history of an organism can go a long way toward producing more realistic niche models. Many animals, especially desert-adapted species, partition their life histories in response to local conditions and so share similar phenologies. For example, desert tortoises, <em>Gopherus agassizii,</em> spend the majority of their life underground in burrows, only active above ground for 153 hours per year (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nagy &amp; Medica 1986</a>). Modeling the effects of phenology on such species’ niches will be critical for better predictions of how these species might respond to future climate change. Additionally, our results show that <em>Rhaphiomidas</em> species adapt to different environments by evolving darker coloration (in colder conditions) and shifting phenologies.</p><div id="sec-24" class="subsection"><h3>Using Accurate Temporal Data Improves Biological Insight</h3><p id="p-51">Our approach can be tailored to produce species specific environmental layers to best capture conditions that species are actually experiencing. This can be seen by examining the raw values obtained from the raster layers (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>). For example, when we look at a set of species that are partially sympatric but are separated temporally, the Bioclim data does not recover many differences between the species. By contrast, the Daymet data shows temperature values as completely separate or partially overlapping for both species. For example, <em>Rhaphiomidas sp. “arenagena”</em> is active earlier in the year compared to <em>R. sp. “rex”</em>. Their Bioclim BIO5 profiles overlap in temperature, whereas they do not overlap in the Daymet inference. This is seen with all the species that are partially sympatric but separated in time, as well as for some of the allopatric species that are also separated in time.</p><p id="p-52">This result demonstrates a larger point that ecological inferences, such as testing for phylogenetic niche conservation of a trait related to temperature or another environmental variable, should not be made based on data that do not accurately link environmental data to the traits in question. By using environmental layers that are tailored to a species’ phenology in space and time, this will lessen misinterpretations of evolutionary trends, such as niche conservatism, due to the autocorrelation between geography and environment for species in allopatry (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Warren et al. 2014</a>). An abundance of data sources, such as Daymet, NDVI, and other MODIS data that are binned in daily or bi-weekly intervals, provide the much-needed data to make more accurate ecological and evolutionary inferences.</p></div><div id="sec-25" class="subsection"><h3>Niche Shifts Through Color Evolution and Niche Tracking Through Changes in Phenology</h3><p id="p-53"><em>Rhaphiomidas</em> adapt to cooler conditions by evolving large areas of maculation (i.e. becoming darker), allowing them to emerge at times when conditions are less than optimal for them to fly efficiently or at all. Combining our <em>threshBayes</em> and <em>ancThresh</em> analyses, and the evidence from behavioral observations of activity (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Cazier 1985</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kingsley 1996</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2002</a>), we interpret the dark coloration of the cooler temperature species as an adaptation to warm themselves in order to take flight. This adaptation for dealing with cooler temperatures has allowed <em>Rhaphiomidas</em> to deviate from their thermal mean (as determined from ancestral state reconstructions) and allowed them to expand to new habitats.</p><p id="p-54">Results from <em>threshBayes</em> analyses recover a significant and relatively strong correlation between temperature and coloration. This correlation is expected as <em>Rhaphiomidas</em> adults are usually only active above 26.6 ºC (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Kingsley, 1996</a>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">2002</a>). Efficient warming for adequate flight is likely especially important for larger flies, such as <em>Rhaphiomidas</em> species, as their greater mass would require more time and energy to heat up. This phenomenon of darker coloration allowing ectotherms to warm themselves has been demonstrated in many taxa (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clusella Trullas et al., 2007</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Svensson &amp; Waller, 2013</a>).</p><p id="p-55">Our <em>ancThresh</em> results consistently recovered one ancestral node as dark in coloration – the <em>R. pachyrhynchus</em> and <em>R. episcopus</em> parent node. All other transitions in the tree were from pale to dark coloration, occurring only along the terminal branches, indicating that the observed dark coloration is not simply due to inheritance but instead a response to cooler environments. However, there are two dark <em>Rhaphiomidas</em> species that occur in relatively hot environments – <em>R. episcopus</em> and <em>R. trochilus</em>. In the case of <em>R. episcopus,</em> its dark coloration appears to be because the transition to this coloration occurred in the ancestor (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>). The reason the dark coloration is maintained in these species is not entirely clear and requires further investigation.</p><p id="p-56">The <em>ancThresh</em> reconstructions in concert with the results from <em>contMap</em> can facilitate the identification of environmental niche tracking and/or adaptation to niches via phenology or coloration change. We found that all species were most likely pale in coloration ancestrally, indicating that changes are likely influenced by environment and not solely inheritance. We identified that some species evolved phenologies that are divergent relative to their sister taxon (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In addition, only one species experienced a shift in coloration solely as a response to cooler temperatures – <em>R. acton maculatus</em>. The other dark species changed both phenology and coloration, save <em>R. pachyrhynchus</em> and <em>R. episcopus,</em> which share a dark coloration ancestor. All other dark species, except <em>R. episcopus,</em> have denser setation (hairs), perhaps serving as insulation and providing additional morphological evidence of adaptation to a cool environment.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1061607/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1061607/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1061607" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-57" class="first-child">Summary of results identifying whether species adapt by tracking a niche by changing their phenology (adaptive niche tracking) or by strictly adapting to their niche through changing their coloration (niche adapting) or a combination of both. The factor(s) that evolved is/are listed after the adaptive response (niche tracking or niche adapting).</p><div class="sb-div caption-clear"></div></div></div><p id="p-58">Our findings demonstrate that most <em>Rhaphiomidas</em> species tend to change their phenology as opposed to adapting their coloration to fit the climate, suggesting that phenology is a more labile trait. For example, <em>R. sp. “rex”</em> emerges about a month earlier than its sister taxon, <em>R. acton</em>. For both species, their phenologies coincide with late spring blooms in their different ranges (personal obs., M.H.V.D.). A phenology much later or earlier would render them unable to take advantage of these nectar resources. Their phenology should be a balance between resources availability and optimizing temperature. However, some species do not feed as adults, such as <em>R. hirsuticaudus</em> (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Cazier 1985</a>, personal obs. M.H.V.D.), so phenology should play an even more important role for such species to synchronize their emergence for reproduction.</p></div></div><div class="section" id="sec-26"><h2 class="">SUPPORTING INFORMATION</h2><p id="p-60">Supporting Information link: <a href="https://figshare.com/s/0db26157b3b8c4cb3804">https://figshare.com/s/0db26157b3b8c4cb3804</a>, <a href="https://figshare.com/s/a680df82acaa692ecd79">https://figshare.com/s/a680df82acaa692ecd79</a></p></div><div class="section" id="sec-27"><h2 class="">DATA ACCESSIBILITY</h2><p id="p-61">We have permanently uploaded all relevant data to figshare to make the research fully reproducible: Supporting Information link: <a href="https://figshare.com/s/0db26157b3b8c4cb3804">https://figshare.com/s/0db26157b3b8c4cb3804</a> <a href="https://figshare.com/s/a680df82acaa692ecd79">https://figshare.com/s/a680df82acaa692ecd79</a></p></div><div class="section" id="sec-28"><h2 class="">AUTHOR CONTRIBUTIONS</h2><p id="p-62">M.H.V.D. performed sampling, sequencing, phylogenetic dating, and wrote the first paper draft. A.J.R. wrote R scripts for Daymet data processing and PNO calculations, and M.H.V.D. and M.S.B. contributed to additional R scripts and informatics pipelines. All authors contributed to experimental design, analyses of the data, and final paper draft.</p></div><div class="section" id="sec-29"><h2 class="">BIOSKETCHES</h2><p id="p-63"><strong>Matthew H. Van Dam</strong> is interested in the interaction of geological processes and natural history of organisms in shaping biogeographical patterns; he is also interested in weevil systematics, including their evolution and improving phylogenomic methods.</p><p id="p-64"><strong>Andrew Rominger</strong> is interested in how and why evolutionary history drives contemporary ecological dynamics. He approaches this challenge both from a theoretical perspective, building and testing synthetic theories of biodiversity based on statistical mechanics, and from an empirical perspective, collecting and analyzing new data about the evolution and ecology of arthropod diversity in island-like systems.</p><p id="p-65"><strong>Michael S. Brewer</strong> uses genomics tools, bioinformatics, and arthropods as model organisms to answer broad evolutionary questions, especially those concerning the mechanisms that create and maintain biodiversity. Specific interests include taxonomy, biogeography, phylogenetics, and the evolution of complex traits (e.g., venomics, color evolution, and complex trait evolution).</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-59">The authors would like to thank Reviewer1, Reviewer2 and Sarah Crews for their careful comments that greatly improved the quality of the manuscript. The authors would also like to thank Jun Ying Lim for his help with statistical analyses and discussion of the paper. We also thank Greg Ballmer for his photos of <em>Rhaphiomidas</em> in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>. We also thank Michelle Trautwein for her help in the writing of this manuscript and George Roderick, Rosemary Gillespie and Michael Balke for use of servers to perform the analyses. M.H.V.D. was funded by the following: Desert Research Fund, from The Community Foundation Serving Riverside and San Bernardino Counties, CA, USA; UC MEXUS Dissertation Research Grant; Theodore Roosevelt Memorial Fund from the American Museum of Natural History; Anza-Borrego Foundation and Institute (ABFI) Entomology; Committee for Research and Exploration of the National Geographic Society; Walker Fund in Entomology UCB; Robert L. Usinger Memorial Award UCB; NSF award DBI #1402102.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Asch</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Tienderen</span>,  <span class="cit-name-given-names">P.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holleman</span>,  <span class="cit-name-given-names">L.J.M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Visser</span>,  <span class="cit-name-given-names">M.E.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Predicting adaptation of phenology in response to climate change, an insect herbivore example</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr>, <span class="cit-vol">13</span>, <span class="cit-fpage">1596</span>–<span class="cit-lpage">1604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D13%26rft.spage%253D1596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-534636v1.2"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Helander</span>  <span class="cit-name-given-names">C.G.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Crabtree</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Ballmer</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cazier</span>,  <span class="cit-name-given-names">M.A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Osborne</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">1994</span>) <span class="cit-chapter-title">Delhi Sands Flower-Loving Fly</span>. In: <span class="cit-source">Life edge a guide to California’s endangered natural resources: Wildlife</span> (eds. <span class="cit-ed"><span class="cit-name-surname">Helander</span>  <span class="cit-name-given-names">C.G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Crabtree</span>  <span class="cit-name-given-names">M.</span></span>). <span class="cit-publ-name">BioSystems Books</span>. pp. <span class="cit-fpage">416</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.3" data-doi="10.1111/j.1461-0248.2012.01796.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Banta</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>,  <span class="cit-name-given-names">I.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chou</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilczek</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitt</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kover</span>,  <span class="cit-name-given-names">P.X.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Purugganan</span>,  <span class="cit-name-given-names">M.D.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Climate envelope modelling reveals intraspecific relationships among flowering phenology, niche breadth and potential range size in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">15</span>, <span class="cit-fpage">769</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DBanta%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D769%26rft.epage%253D777%26rft.atitle%253DClimate%2Benvelope%2Bmodelling%2Breveals%2Bintraspecific%2Brelationships%2Bamong%2Bflowering%2Bphenology%252C%2Bniche%2Bbreadth%2Band%2Bpotential%2Brange%2Bsize%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2012.01796.x%26rft_id%253Dinfo%253Apmid%252F22583905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2012.01796.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22583905&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Becker</span>,  <span class="cit-name-given-names">R.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilks</span>,  <span class="cit-name-given-names">A.R.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brownrigg</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minka</span>,  <span class="cit-name-given-names">T.P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Deckmyn</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">maps: Draw Geographical Maps</span>. R package version 3.2.0.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.5" data-doi="10.1643/0045-8511(2002)002[0477:EOBSAM]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bittner</span>,  <span class="cit-name-given-names">T.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">R.B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kerfin</span>,  <span class="cit-name-given-names">J.M.</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Effects of Body Size and Melanism on the Thermal Biology of Garter Snakes (Thamnophis sirtalis)</span>. <abbr class="cit-jnl-abbrev">Copeia</abbr>, <span class="cit-vol">2002</span>, <span class="cit-fpage">477</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCopeia%26rft.volume%253D2002%26rft.spage%253D477%26rft_id%253Dinfo%253Adoi%252F10.1643%252F0045-8511%25282002%2529002%255B0477%253AEOBSAM%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1643/0045-8511(2002)002[0477:EOBSAM]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.6" data-doi="10.1111/j.1365-294X.2007.03509.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>,  <span class="cit-name-given-names">W.E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holzapfel</span>,  <span class="cit-name-given-names">C.M.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Genetic response to rapid climate change: it’s seasonal timing that matters</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">157</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBradshaw%26rft.auinit1%253DW.%2BE.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D157%26rft.epage%253D166%26rft.atitle%253DGenetic%2Bresponse%2Bto%2Brapid%2Bclimate%2Bchange%253A%2Bit%2527s%2Bseasonal%2Btiming%2Bthat%2Bmatters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03509.x%26rft_id%253Dinfo%253Apmid%252F17850269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03509.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17850269&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251740500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cazier</span>,  <span class="cit-name-given-names">M.A.</span></span> (<span class="cit-pub-date">1985</span>) <span class="cit-article-title">A revision of the North American flies belonging to the genus Rhaphiomidas (Diptera, Apioceridae)</span>. <abbr class="cit-jnl-abbrev">Bulletin of the AMNH; v.</abbr> <span class="cit-vol">182</span>, article 2.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.8" data-doi="10.1016/j.jtherbio.2007.01.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clusella Trullas</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Wyk</span>,  <span class="cit-name-given-names">J.H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Spotila</span>,  <span class="cit-name-given-names">J.R.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Thermal melanism in ectotherms</span>. <abbr class="cit-jnl-abbrev">Journal of Thermal Biology</abbr>, <span class="cit-vol">32</span>, <span class="cit-fpage">235</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BThermal%2BBiology%26rft.volume%253D32%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtherbio.2007.01.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtherbio.2007.01.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247666700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Dam</span>,  <span class="cit-name-given-names">M.H.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Zootaxa, A new species and key for Rhaphiomidas Osten Sacken (Diptera: Mydidae)</span>. <abbr class="cit-jnl-abbrev">Zootaxa</abbr>, <span class="cit-vol">2622</span>, <span class="cit-fpage">49</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZootaxa%26rft.volume%253D2622%26rft.spage%253D49%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van Dam</span>,  <span class="cit-name-given-names">M.H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Matzke</span>,  <span class="cit-name-given-names">N.J.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Evaluating the influence of connectivity and 18 distance on biogeographical patterns in the south-western deserts of North America</span>. <abbr class="cit-jnl-abbrev">Journal of Biogeography</abbr>, <span class="cit-vol">43</span>, <span class="cit-fpage">1514</span>–<span class="cit-lpage">1532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiogeography%26rft.volume%253D43%26rft.spage%253D1514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2015</span>) <abbr class="cit-jnl-abbrev">Bayesian evolutionary analysis: with BEAST</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-534636v1.12" data-doi="10.1371/journal.pbio.0040088"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">S.Y.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">M.J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Relaxed Phylogenetics and Dating with Confidence</span>. <abbr class="cit-jnl-abbrev">PLoS Biology</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">e88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDrummond%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De88%26rft.epage%253De88%26rft.atitle%253DRelaxed%2Bphylogenetics%2Band%2Bdating%2Bwith%2Bconfidence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040088%26rft_id%253Dinfo%253Apmid%252F16683862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16683862&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.13" data-doi="10.1093/molbev/mss075"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Drummond</span>,  <span class="cit-name-given-names">A.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">D.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Bayesian Phylogenetics with BEAUti and the BEAST 1.7</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">29</span>, <span class="cit-fpage">1969</span>–<span class="cit-lpage">1973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss075%26rft_id%253Dinfo%253Apmid%252F22367748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22367748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307171300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elith</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">S.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hastie</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudík</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chee</span>,  <span class="cit-name-given-names">Y.E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Yates</span>,  <span class="cit-name-given-names">C.J.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">A statistical explanation of MaxEnt for ecologists</span>. <abbr class="cit-jnl-abbrev">Diversity and Distributions</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">43</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2Band%2BDistributions%26rft.volume%253D17%26rft.spage%253D43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.15" data-doi="10.1371/journal.pone.0053788"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellwood</span>,  <span class="cit-name-given-names">E.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Temple</span>,  <span class="cit-name-given-names">S.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Primack</span>,  <span class="cit-name-given-names">R.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradley</span>,  <span class="cit-name-given-names">N.L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">C.C.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Record-Breaking Early Flowering in the Eastern United States</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">e53788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De53788%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0053788%26rft_id%253Dinfo%253Apmid%252F23342001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0053788&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23342001&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.16" data-doi="10.1086/595757"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Evans</span>,  <span class="cit-name-given-names">M.E.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">S.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flynn</span>,  <span class="cit-name-given-names">R.S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Donoghue</span>,  <span class="cit-name-given-names">M.J.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Climate, Niche Evolution, and Diversification of the “Bird-Cage” Evening Primroses (Oenothera, Sections Anogra and Kleinia)</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">173</span>, <span class="cit-fpage">225</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DEvans%26rft.auinit1%253DM.%2BE.%26rft.volume%253D173%26rft.issue%253D2%26rft.spage%253D225%26rft.epage%253D240%26rft.atitle%253DClimate%252C%2Bniche%2Bevolution%252C%2Band%2Bdiversification%2Bof%2Bthe%2B%2522bird-cage%2522%2Bevening%2Bprimroses%2B%2528Oenothera%252C%2Bsections%2BAnogra%2Band%2BKleinia%2529.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F595757%26rft_id%253Dinfo%253Apmid%252F19072708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/595757&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19072708&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262704600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.17" data-doi="10.3732/ajb.1400045"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Everill</span>,  <span class="cit-name-given-names">P.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Primack</span>,  <span class="cit-name-given-names">R.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellwood</span>,  <span class="cit-name-given-names">E.R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Melaas</span>,  <span class="cit-name-given-names">E.K.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Determining past leaf-out times of New England’s deciduous forests from herbarium specimens</span>. <abbr class="cit-jnl-abbrev">American Journal of Botany</abbr>, <span class="cit-vol">101</span>, <span class="cit-fpage">1293</span>–<span class="cit-lpage">1300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1400045%26rft_id%253Dinfo%253Apmid%252F25156979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMDEvOC8xMjkzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDEvMzEvNTM0NjM2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-534636v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Phylogenies and the Comparative Method</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">125</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.19" data-doi="10.1086/663681"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">A Comparative Method for Both Discrete and Continuous Characters Using the Threshold Model</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">179</span>, <span class="cit-fpage">145</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D179%26rft.issue%253D2%26rft.spage%253D145%26rft.epage%253D156%26rft.atitle%253DA%2Bcomparative%2Bmethod%2Bfor%2Bboth%2Bdiscrete%2Band%2Bcontinuous%2Bcharacters%2Busing%2Bthe%2Bthreshold%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F663681%26rft_id%253Dinfo%253Apmid%252F22218305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/663681&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22218305&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299000400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gerst</span>,  <span class="cit-name-given-names">K.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossington</span>,  <span class="cit-name-given-names">N.L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Mazer</span>,  <span class="cit-name-given-names">S.J.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Phenological responsiveness to climate differs among four species of Quercus in North America</span>. <abbr class="cit-jnl-abbrev">Journal of Ecology</abbr>, <span class="cit-vol">105</span>, <span class="cit-fpage">1610</span>–<span class="cit-lpage">1622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEcology%26rft.volume%253D105%26rft.spage%253D1610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.21" data-doi="10.1093/bioinformatics/btm538"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harmon</span>,  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brock</span>,  <span class="cit-name-given-names">C.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glor</span>,  <span class="cit-name-given-names">R.E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Challenger</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">GEIGER: investigating evolutionary radiations</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">129</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm538%26rft_id%253Dinfo%253Apmid%252F18006550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18006550&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251865000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Heibl</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Calenge</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">phyloclim: Integrating Phylogenetics and Climatic Niche Modeling</span>. R package version 0.9-4.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.23" data-doi="10.1111/j.0906-7590.2006.04700.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>,  <span class="cit-name-given-names">P.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>,  <span class="cit-name-given-names">C.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Master</span>,  <span class="cit-name-given-names">L.L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Albert</span>,  <span class="cit-name-given-names">D.L.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">The effect of sample size and species characteristics on performance of different species distribution modeling methods</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr>, <span class="cit-vol">29</span>, <span class="cit-fpage">773</span>–<span class="cit-lpage">785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D29%26rft.spage%253D773%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0906-7590.2006.04700.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0906-7590.2006.04700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000241507100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.24" data-doi="10.1002/joc.1276"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>,  <span class="cit-name-given-names">S.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parra</span>,  <span class="cit-name-given-names">J.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">P.G.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Jarvis</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Very high resolution interpolated climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D25%26rft.spage%253D1965%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.1276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.1276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234519700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R.J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">van Etten</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Geographic analysis and modeling with raster data</span>. R package version 2.0-12.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leathwick</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Elith</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">dismo: Species Distribution Modeling</span>. R package version 1.1-4.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.27" data-doi="10.1038/nature10650"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hof</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Araújo</span>,  <span class="cit-name-given-names">M.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jetz</span>,  <span class="cit-name-given-names">W.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Rahbek</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Additive threats from pathogens, climate and land-use change for global amphibian diversity</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">480</span>, <span class="cit-fpage">516</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHof%26rft.auinit1%253DC.%26rft.volume%253D480%26rft.issue%253D7378%26rft.spage%253D516%26rft.epage%253D519%26rft.atitle%253DAdditive%2Bthreats%2Bfrom%2Bpathogens%252C%2Bclimate%2Band%2Bland-use%2Bchange%2Bfor%2Bglobal%2Bamphibian%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10650%26rft_id%253Dinfo%253Apmid%252F22089134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10650&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22089134&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298318000059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.28" data-doi="10.1038/nature09670"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">A.A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sgrò</span>,  <span class="cit-name-given-names">C.M.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Climate change and evolutionary adaptation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">470</span>, <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D470%26rft.issue%253D7335%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DClimate%2Bchange%2Band%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09670%26rft_id%253Dinfo%253Apmid%252F21350480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21350480&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287652900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">R.D.</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-article-title">The microevolutionary consequences of climate change</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr>, <span class="cit-vol">5</span>, <span class="cit-fpage">311</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D5%26rft.spage%253D311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.30" data-doi="10.1093/molbev/msh123"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>,  <span class="cit-name-given-names">J.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larget</span>,  <span class="cit-name-given-names">B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Alfaro</span>,  <span class="cit-name-given-names">M.E.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Bayesian Phylogenetic Model Selection Using Reversible Jump Markov Chain Monte Carlo</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">21</span>, <span class="cit-fpage">1123</span>–<span class="cit-lpage">1133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsh123%26rft_id%253Dinfo%253Apmid%252F15034130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msh123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034130&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221599300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.31" data-doi="10.1093/aesa/89.6.883"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kingsley</span>,  <span class="cit-name-given-names">K.J.</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Behavior of the Delhi Sands Flower-Loving Fly (Diptera: Mydidae), a Little-Known Endangered Species</span>. <abbr class="cit-jnl-abbrev">Annals of the Entomological Society of America</abbr>, <span class="cit-vol">89</span>, <span class="cit-fpage">883</span>–<span class="cit-lpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BEntomological%2BSociety%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faesa%252F89.6.883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/aesa/89.6.883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kingsley</span>,  <span class="cit-name-given-names">K.J.</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Population Dynamics, Resource Use, and Conservation Needs of the Delhi Sands Flower-loving Fly (Rhaphiomidas Terminatus Abdominalis Cazier) (Diptera: Mydidae), an Endangered Species</span>. <abbr class="cit-jnl-abbrev">Journal of Insect Conservation</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">93</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInsect%2BConservation%26rft.volume%253D6%26rft.spage%253D93%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kingsolver</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wiernasz</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">1991</span>) <span class="cit-article-title">Seasonal polyphenism in wing-melanin pattern and thermoregulatory adaptation in Pieris butterflies</span>. <abbr class="cit-jnl-abbrev">American Naturalist</abbr>,.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.34" data-doi="10.1093/molbev/mss020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lanfear</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Calcott</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">S.Y.W.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Guindon</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">PartitionFinder: Combined Selection of Partitioning Schemes and Substitution Models for Phylogenetic Analyses</span>. <abbr class="cit-jnl-abbrev">Molecular Biology and Evolution</abbr>, <span class="cit-vol">29</span>, <span class="cit-fpage">1695</span>–<span class="cit-lpage">1701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss020%26rft_id%253Dinfo%253Apmid%252F22319168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22319168&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304200600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.35" data-doi="10.1093/bioinformatics/btm404"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Larkin</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blackshields</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">N.P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chenna</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McGettigan</span>,  <span class="cit-name-given-names">P.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McWilliam</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valentin</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wallace</span>,  <span class="cit-name-given-names">I.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilm</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">J.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gibson</span>,  <span class="cit-name-given-names">T.J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Higgins</span>,  <span class="cit-name-given-names">D.G.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Clustal W and Clustal X version 2.0</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">23</span>, <span class="cit-fpage">2947</span>–<span class="cit-lpage">2948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm404%26rft_id%253Dinfo%253Apmid%252F17846036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846036&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251197700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maddison</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Maddison</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Mesquite: a modular system for evolutionary analysis</span>. <abbr class="cit-jnl-abbrev">Version 2.6. Computer program and documentation distributed by the author</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-534636v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martill</span>,  <span class="cit-name-given-names">D.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bechly</span>,  <span class="cit-name-given-names">G.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Loveridge</span>,  <span class="cit-name-given-names">R.F.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-source">The Crato fossil beds of Brazil: window into an ancient world</span>. <span class="cit-publ-name">Cambridge University Press</span>,</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.38" data-doi="10.1093/sysbio/syu056"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matzke</span>,  <span class="cit-name-given-names">N.J.</span></span>, (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Model Selection in Historical Biogeography Reveals that Founder-event Speciation is a Crucial Process in Island Clades</span>. <abbr class="cit-jnl-abbrev">Systematic Biology.</abbr> <span class="cit-vol">63</span>, <span class="cit-fpage">951</span>–<span class="cit-lpage">970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyu056%26rft_id%253Dinfo%253Apmid%252F25123369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syu056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25123369&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.39" data-doi="10.1111/j.1558-5646.2007.00239.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McGuire</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linkem</span>,  <span class="cit-name-given-names">C.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koo</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hutchison</span>,  <span class="cit-name-given-names">D.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lappin</span>,  <span class="cit-name-given-names">A.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orange</span>,  <span class="cit-name-given-names">D.I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemos-Espinal</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riddle</span>,  <span class="cit-name-given-names">B.R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Jaeger</span>,  <span class="cit-name-given-names">J.R.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Mitochondrial introgression and incomplete lineage sorting through space and time: phylogenetics of crotaphytid lizards</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">61</span>, <span class="cit-fpage">2879</span>–<span class="cit-lpage">2897</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DMcGuire%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D61%26rft.issue%253D12%26rft.spage%253D2879%26rft.epage%253D2897%26rft.atitle%253DMitochondrial%2Bintrogression%2Band%2Bincomplete%2Blineage%2Bsorting%2Bthrough%2Bspace%2Band%2Btime%253A%2Bphylogenetics%2Bof%2Bcrotaphytid%2Blizards.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00239.x%26rft_id%253Dinfo%253Apmid%252F17941840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00239.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17941840&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251250600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.40" data-doi="10.1890/07-0068.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller-Rushing</span>,  <span class="cit-name-given-names">A.J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Primack</span>,  <span class="cit-name-given-names">R.B.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Global warming and flowering times in Thoreau’s Concord: a community perspective</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-vol">89</span>, <span class="cit-fpage">332</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DMiller-Rushing%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D89%26rft.issue%253D2%26rft.spage%253D332%26rft.epage%253D341%26rft.atitle%253DGlobal%2Bwarming%2Band%2Bflowering%2Btimes%2Bin%2BThoreau%2527s%2BConcord%253A%2Ba%2Bcommunity%2Bperspective.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F07-0068.1%26rft_id%253Dinfo%253Apmid%252F18409423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/07-0068.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18409423&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254124200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.41" data-doi="10.1098/rspb.2008.0438"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>,  <span class="cit-name-given-names">N.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kearney</span>,  <span class="cit-name-given-names">M.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">N.J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Porter</span>,  <span class="cit-name-given-names">W.P.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Predicting the fate of a living fossil: how will global warming affect sex determination and hatching phenology in tuatara</span>? <abbr class="cit-jnl-abbrev">Proceedings. Biological sciences</abbr>, <span class="cit-vol">275</span>, <span class="cit-fpage">2185</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DMitchell%26rft.auinit1%253DN.%2BJ%26rft.volume%253D275%26rft.issue%253D1648%26rft.spage%253D2185%26rft.epage%253D2193%26rft.atitle%253DPredicting%2Bthe%2Bfate%2Bof%2Ba%2Bliving%2Bfossil%253A%2Bhow%2Bwill%2Bglobal%2Bwarming%2Baffect%2Bsex%2Bdetermination%2Band%2Bhatching%2Bphenology%2Bin%2Btuatara%253F%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2008.0438%26rft_id%253Dinfo%253Apmid%252F18595840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2008.0438&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18595840&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258440500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nagy</span>,  <span class="cit-name-given-names">K.A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Medica</span>,  <span class="cit-name-given-names">P.A.</span></span> (<span class="cit-pub-date">1986</span>) <span class="cit-article-title">Physiological ecology of desert tortoises in southern Nevada</span>. <abbr class="cit-jnl-abbrev">Herpetologica</abbr> <span class="cit-vol">42</span>, <span class="cit-fpage">73</span>–<span class="cit-lpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHerpetologica%26rft.volume%253D42%26rft.spage%253D73%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1986D394600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.43" data-doi="10.1146/annurev.ecolsys.37.091305.110100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Parmesan</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Ecological and Evolutionary Responses to Recent Climate Change</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">37</span>, <span class="cit-fpage">637</span>–<span class="cit-lpage">669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D37%26rft.spage%253D637%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.37.091305.110100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.37.091305.110100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000243038500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.44" data-doi="10.1016/j.ecolmodel.2005.03.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">S.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">R.P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Schapire</span>,  <span class="cit-name-given-names">R.E.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Maximum entropy modeling of species geographic distributions</span>. <abbr class="cit-jnl-abbrev">Ecological Modelling</abbr>, <span class="cit-vol">190</span>, <span class="cit-fpage">231</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BModelling%26rft.volume%253D190%26rft.spage%253D231%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ecolmodel.2005.03.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ecolmodel.2005.03.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233859600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pierce</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Interface to Unidata netCDF (Version 4 or Earlier) Format Data Files</span>. R package version 1.16..</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Proosdij</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sosef</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wieringa</span>,  <span class="cit-name-given-names">J.J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Raes</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Minimum required number of specimen records to develop accurate species distribution models</span>. <abbr class="cit-jnl-abbrev">Ecography.</abbr> <span class="cit-vol">39</span>, <span class="cit-fpage">542</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography.%26rft.volume%253D39%26rft.spage%253D542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.47" data-doi="10.1111/ele.12144"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Quintero</span>,  <span class="cit-name-given-names">I.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wiens</span>,  <span class="cit-name-given-names">J.J.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Rates of projected climate change dramatically exceed past rates of climatic niche evolution among vertebrate species</span>. <abbr class="cit-jnl-abbrev">Ecology letters</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">1095</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2Bletters%26rft.volume%253D16%26rft.spage%253D1095%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12144%26rft_id%253Dinfo%253Apmid%252F23800223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23800223&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Revell</span>,  <span class="cit-name-given-names">L.J.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">phytools: an R package for phylogenetic comparative biology (and other things)</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">217</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rogers</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mattoni</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">1993</span>) <span class="cit-article-title">Observations on the natural history and conservation biology of the giant flower loving flies, Rhaphiomidas (Diptera: Apioceridae)</span>. <abbr class="cit-jnl-abbrev">Dipterological Research</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">21</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDipterological%2BResearch%26rft.volume%253D4%26rft.spage%253D21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rogers</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Van Dam</span>,  <span class="cit-name-given-names">M.H.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Two new species of Rhaphiomidas (Diptera: Mydidae)</span>. <abbr class="cit-jnl-abbrev">Zootaxa</abbr>, <span class="cit-vol">1664</span>, <span class="cit-fpage">61</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZootaxa%26rft.volume%253D1664%26rft.spage%253D61%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.51" data-doi="10.1093/sysbio/sys029"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ronquist</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Teslenko</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Mark</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ayres</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Darling</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Höhna</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Larget</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>,  <span class="cit-name-given-names">M.A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Huelsenbeck</span>,  <span class="cit-name-given-names">J.P.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">MrBayes 3.2: Efficient Bayesian Phylogenetic Inference and Model Choice Across a Large Model Space</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr>, <span class="cit-vol">61</span>, <span class="cit-fpage">539</span>–<span class="cit-lpage">542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsys029%26rft_id%253Dinfo%253Apmid%252F22357727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/sys029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22357727&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.52" data-doi="10.1643/CP-04-154R1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenblum</span>,  <span class="cit-name-given-names">E.B.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">The Role of Phenotypic Plasticity in Color Variation of Tularosa Basin Lizards</span>. <abbr class="cit-jnl-abbrev">Copeia</abbr>, <span class="cit-vol">2005</span>, <span class="cit-fpage">586</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCopeia%26rft.volume%253D2005%26rft.spage%253D586%26rft_id%253Dinfo%253Adoi%252F10.1643%252FCP-04-154R1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1643/CP-04-154R1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.53" data-doi="10.1111/j.1461-0248.2011.01689.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sheldon</span>,  <span class="cit-name-given-names">K.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Tewksbury</span>,  <span class="cit-name-given-names">J.J.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Climate change and community disassembly: impacts of warming on tropical and temperate montane community structure</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">1191</span>–<span class="cit-lpage">1200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DSheldon%26rft.auinit1%253DK.%2BS.%26rft.volume%253D14%26rft.issue%253D12%26rft.spage%253D1191%26rft.epage%253D1200%26rft.atitle%253DClimate%2Bchange%2Band%2Bcommunity%2Bdisassembly%253A%2Bimpacts%2Bof%2Bwarming%2Bon%2Btropical%2Band%2Btemperate%2Bmontane%2Bcommunity%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2011.01689.x%26rft_id%253Dinfo%253Apmid%252F21978234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2011.01689.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21978234&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.54" data-doi="10.1093/bioinformatics/btq228"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holder</span>,  <span class="cit-name-given-names">M.T.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">DendroPy: a Python library for phylogenetic computing</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">26</span>, <span class="cit-fpage">1569</span>–<span class="cit-lpage">1571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq228%26rft_id%253Dinfo%253Apmid%252F20421198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20421198&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278689000066&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.55" data-doi="10.1086/673206"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Svensson</span>,  <span class="cit-name-given-names">E.I.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Waller</span>,  <span class="cit-name-given-names">J.T.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Ecology and Sexual Selection: Evolution of Wing Pigmentation in Calopterygid Damselflies in Relation to Latitude, Sexual Dimorphism, and Speciation</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">182</span>, <span class="cit-fpage">E174</span>–<span class="cit-lpage">E195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D182%26rft.spage%253DE174%26rft_id%253Dinfo%253Adoi%252F10.1086%252F673206%26rft_id%253Dinfo%253Apmid%252F24107378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/673206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24107378&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325415100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.56" data-doi="10.1038/nature02121"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">C.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">R.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bakkenes</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beaumont</span>,  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Collingham</span>,  <span class="cit-name-given-names">Y.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erasmus</span>,  <span class="cit-name-given-names">B.F.N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Siqueira</span>,  <span class="cit-name-given-names">M.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grainger</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hannah</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hughes</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huntley</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Jaarsveld</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Midgley</span>,  <span class="cit-name-given-names">G.F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miles</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ortega-Huerta</span>,  <span class="cit-name-given-names">M.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peterson</span>,  <span class="cit-name-given-names">A.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">O.L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">S.E.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Extinction risk from climate change</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">427</span>, <span class="cit-fpage">145</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DThomas%26rft.auinit1%253DC.%2BD.%26rft.volume%253D427%26rft.issue%253D6970%26rft.spage%253D145%26rft.epage%253D148%26rft.atitle%253DExtinction%2Brisk%2Bfrom%2Bclimate%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02121%26rft_id%253Dinfo%253Apmid%252F14712274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14712274&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187863900033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thornton</span>,  <span class="cit-name-given-names">P.E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.M.</span> <span class="cit-name-surname">Thornton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.W.</span> <span class="cit-name-surname">Mayer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Devarakonda</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.S.</span> <span class="cit-name-surname">Vose</span></span>, and <abbr class="cit-jnl-abbrev">R.B.C.</abbr> (<span class="cit-pub-date">2014</span>) Available at: <a href="https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=1219">https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=1219</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Toft</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kimsey</span>,  <span class="cit-name-given-names">L.S.</span></span> (<span class="cit-pub-date">1982</span>) <span class="cit-article-title">Habitat and Behavior of Selected Apiocera and Rhaphiomidas (Diptera, Apioceridae), and Descriptions of Immature Stages of A. hispida</span>. <abbr class="cit-jnl-abbrev">Journal of the Kansas Entomological Society</abbr>, <span class="cit-vol">55</span>, <span class="cit-fpage">177</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BKansas%2BEntomological%2BSociety%26rft.volume%253D55%26rft.spage%253D177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Visser</span>,  <span class="cit-name-given-names">M.E.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Keeping up with a warming world; assessing the rate of adaptation to climate change</span>. <abbr class="cit-jnl-abbrev">Proceedings. Biological sciences / The Royal Society</abbr>, <span class="cit-vol">275</span>, <span class="cit-fpage">649</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings.%2BBiological%2Bsciences%2B%252F%2BThe%2BRoyal%2BSociety%26rft.volume%253D275%26rft.spage%253D649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.60" data-doi="10.1111/j.1600-0587.2009.06142.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warren</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glor</span>,  <span class="cit-name-given-names">R.E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Turelli</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">ENMTools: a toolbox for comparative studies of environmental niche models</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr>, <span class="cit-vol">33</span>, <span class="cit-fpage">607</span>–<span class="cit-lpage">611</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D33%26rft.spage%253D607%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1600-0587.2009.06142.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1600-0587.2009.06142.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000280457500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warren</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cardillo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosauer</span>,  <span class="cit-name-given-names">D.F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Bolnick</span>,  <span class="cit-name-given-names">D.I.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Mistaking geography for biology: inferring processes from species distributions</span>. <abbr class="cit-jnl-abbrev">Trends Ecology Evolution</abbr>, <span class="cit-vol">29</span>, <span class="cit-fpage">572</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcology%2BEvolution%26rft.volume%253D29%26rft.spage%253D572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warren</span>,  <span class="cit-name-given-names">D.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beaumont</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dinnage</span>,  <span class="cit-name-given-names">R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Baumgartner</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">New methods for measuring ENM breadth and overlap in environmental space</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr>. doi:10.1111/ecog.03900</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1934</span>) <span class="cit-article-title">An Analysis of Variability in Number of Digits in an Inbred Strain of Guinea Pigs</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">506</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D6%26rft.spage%253D506%26rft.epage%253D536%26rft.atitle%253DAN%2BANALYSIS%2BOF%2BVARIABILITY%2BIN%2BNUMBER%2BOF%2BDIGITS%2BIN%2BAN%2BINBRED%2BSTRAIN%2BOF%2BGUINEA%2BPIGS%26rft_id%253Dinfo%253Apmid%252F17246735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxOS82LzUwNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzAxLzMxLzUzNDYzNi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hepner</span>,  <span class="cit-name-given-names">G.F.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Short-Term Phenological Predictions of Vegetation Abundance Using Multivariate Adaptive Regression Splines in the Upper Colorado River Basin</span>. <abbr class="cit-jnl-abbrev">Earth Interactions</abbr>, <span class="cit-vol">21</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEarth%2BInteractions%26rft.volume%253D21%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-534636v1.65" data-doi="10.1111/ele.12308"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zohner</span>,  <span class="cit-name-given-names">C.M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Renner</span>,  <span class="cit-name-given-names">S.S.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Common garden comparison of the leaf-out phenology of woody species from different native climates, combined with herbarium records, forecasts long-term change</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">1016</span>–<span class="cit-lpage">1025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D17%26rft.spage%253D1016%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12308%26rft_id%253Dinfo%253Apmid%252F24943497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24943497&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F01%2F31%2F534636.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/534636v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F534636v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/536938v1" title="A Polygenic Score for Body Mass Index is Associated with Depressive Symptoms via Early Life Stress: Evidence for gene-environment correlation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 31, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/534636v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/534636v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/534636v1" class="active" data-icon-position="" data-hide-link-title="0">Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/157565" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LeWVoyjHbEXOl08OfTf9n9ihDPgDynwHfHp1pmB2ERY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="157565" id="node157565--2" data-pisa="biorxiv;534636v1" data-pisa-master="biorxiv;534636" data-seqnum="157565" data-apath="/biorxiv/early/2019/01/31/534636.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Matthew H. Van</span> <span class="nlm-surname">Dam</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Andrew J.</span> <span class="nlm-surname">Rominger</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael S.</span> <span class="nlm-surname">Brewer</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">534636; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/534636 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/534636v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/534636v1&amp;title=Environmental%20Niche%20Adaptation%20Revealed%20Through%20Fine%20Scale%20Phenological%20Niche%20Modeling" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/534636v1&amp;title=Environmental%20Niche%20Adaptation%20Revealed%20Through%20Fine%20Scale%20Phenological%20Niche%20Modeling" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/534636v1&amp;text=Environmental%20Niche%20Adaptation%20Revealed%20Through%20Fine%20Scale%20Phenological%20Niche%20Modeling" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/534636v1&amp;title=Environmental%20Niche%20Adaptation%20Revealed%20Through%20Fine%20Scale%20Phenological%20Niche%20Modeling" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/534636v1&amp;t=Environmental%20Niche%20Adaptation%20Revealed%20Through%20Fine%20Scale%20Phenological%20Niche%20Modeling" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/534636v1&amp;title=Environmental%20Niche%20Adaptation%20Revealed%20Through%20Fine%20Scale%20Phenological%20Niche%20Modeling" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/534636v1&amp;title=Environmental%20Niche%20Adaptation%20Revealed%20Through%20Fine%20Scale%20Phenological%20Niche%20Modeling" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="157565" id="node157565--3" data-pisa="biorxiv;534636v1" data-pisa-master="biorxiv;534636" data-seqnum="157565" data-apath="/biorxiv/early/2019/01/31/534636.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Environmental Niche Adaptation Revealed Through Fine Scale Phenological Niche Modeling</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Matthew H. Van</span> <span class="nlm-surname">Dam</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Andrew J.</span> <span class="nlm-surname">Rominger</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael S.</span> <span class="nlm-surname">Brewer</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">534636; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/534636 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Environmental%20Niche%20Adaptation%20Revealed%20Through%20Fine%20Scale%20Phenological%20Niche%20Modeling&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Dam%2C+Matthew+H.+Van&amp;rft.au=Rominger%2C+Andrew+J.&amp;rft.au=Brewer%2C+Michael+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/157565/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/157565/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/157565/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/157565/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/157565/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/157565/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/157565/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/157565/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/157565/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/157565/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/157565/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/157565/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/534636v1&amp;count=horizontal&amp;via=&amp;text=Environmental%20Niche%20Adaptation%20Revealed%20Through%20Fine%20Scale%20Phenological%20Niche%20Modeling&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/534636v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/534636v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/534636v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
