<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/embed/graphic-5.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/11/27/272898.atom" />
<meta name="HW.pisa" content="biorxiv;272898v5" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="All is not decline across global vertebrate populations" />
<meta name="DC.Identifier" content="10.1101/272898" />
<meta name="DC.Date" content="2019-11-27" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The populations of Earth’s species are changing over time in complex ways, creating a mixture of winners and losers in a time of accelerating global change. A critical research challenge is to test if there are specific biomes, taxa or types of species that are experiencing rapid alterations in abundance over time. We conducted an analysis of the Living Planet Database using a state-space modelling approach including nearly 10,000 abundance time-series from over 2,000 vertebrate species. We integrated the population abundance data with information on geographic range, habitat preference, taxonomic and phylogenetic relationships, conservation status and threats from occurrence, phylogenetic and conservation assessment data. We found that 15% of populations were declining, 18% were increasing, and 67% showed no net changes in abundance over time. Against a backdrop of no biogeographic and phylogenetic patterning in population change, we uncovered a distinct taxonomic signal. Amphibians were the only taxa that experienced net declines in the analyzed data, while birds, mammals and reptiles on average became more abundant over time. The continuum of abundance changes over time was poorly captured by species’ rarity and global-scale threats. Capturing the full spectrum of abundance change of species around the planet will inform conservation efforts and improve projections of biodiversity and ecosystem function change under the accelerating influence of the Anthropocene." />
<meta name="DC.Contributor" content="Gergana N. Daskalova" />
<meta name="DC.Contributor" content="Isla H. Myers-Smith" />
<meta name="DC.Contributor" content="John L. Godlee" />
<meta name="article:published_time" content="2019-11-27" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="All is not decline across global vertebrate populations" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;The populations of Earth’s species are changing over time in complex ways, creating a mixture of winners and losers in a time of accelerating global change. A critical research challenge is to test if there are specific biomes, taxa or types of species that are experiencing rapid alterations in abundance over time. We conducted an analysis of the Living Planet Database using a state-space modelling approach including nearly 10,000 abundance time-series from over 2,000 vertebrate species. We integrated the population abundance data with information on geographic range, habitat preference, taxonomic and phylogenetic relationships, conservation status and threats from occurrence, phylogenetic and conservation assessment data. We found that 15% of populations were declining, 18% were increasing, and 67% showed no net changes in abundance over time. Against a backdrop of no biogeographic and phylogenetic patterning in population change, we uncovered a distinct taxonomic signal. Amphibians were the only taxa that experienced net declines in the analyzed data, while birds, mammals and reptiles on average became more abundant over time. The continuum of abundance changes over time was poorly captured by species’ rarity and global-scale threats. Capturing the full spectrum of abundance change of species around the planet will inform conservation efforts and improve projections of biodiversity and ecosystem function change under the accelerating influence of the Anthropocene.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;272898v5" />
<meta name="citation_id" content="272898v5" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/272898v5" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/272898v5.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/272898v5.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898.full.pdf" />
<meta name="citation_doi" content="10.1101/272898" />
<meta name="citation_num_pages" content="44" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="272898" />
<meta name="citation_author" content="Gergana N. Daskalova" />
<meta name="citation_author_institution" content="School of GeoSciences, University of Edinburgh" />
<meta name="citation_author_email" content="gndaskalova@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5674-5322" />
<meta name="citation_author" content="Isla H. Myers-Smith" />
<meta name="citation_author_institution" content="School of GeoSciences, University of Edinburgh" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8417-6112" />
<meta name="citation_author" content="John L. Godlee" />
<meta name="citation_author_institution" content="School of GeoSciences, University of Edinburgh" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5595-255X" />
<meta name="citation_reference" content="Ehrlén, J. &amp; Morris, W. F. Predicting changes in the distribution and abundance of species under environmental change. Ecol. Lett. 18, 303–314 (2015)." />
<meta name="citation_reference" content="Hefley, T. J., Hooten, M. B., Drake, J. M., Russell, R. E. &amp; Walsh, D. P. When can the cause of a population decline be determined? Ecol. Lett. 19, 1353–1362 (2016)." />
<meta name="citation_reference" content="IPBES. Summary for policymakers of the regional assessment report on biodiversity and ecosystem services for Europe and Central Asia of the Intergovernmental Science-Policy Platform on Biodiversity and Ecosystem Services. M. Fischer, M. Rounsevell, A. Torre-Marin Rando, A. Mader, A. Church, M. Elbakidze, V. Elias, T. Hahn. P.A. Harrison, J. Hauck, B. Martín-López, I. Ring, C. Sandström, I. Sousa Pinto, P. Visconti and N.E Zimmermann (eds.). IPBES secretariat, Bonn, Germany. (2018)." />
<meta name="citation_reference" content="Gilroy, J. J., Gill, J. A., Butchart, S. H. M., Jones, V. R. &amp; Franco, A. M. A. Migratory diversity predicts population declines in birds. Ecol. Lett. 19, 308–317 (2016)." />
<meta name="citation_reference" content="Ceballos, G., Ehrlich, P. R. &amp; Dirzo, R. Biological annihilation via the ongoing sixth mass extinction signaled by vertebrate population losses and declines. Proc. Natl. Acad. Sci. 201704949 (2017). doi:10.1073/pnas.1704949114" />
<meta name="citation_reference" content="Tonini, J. F. R., Beard, K. H., Ferreira, R. B., Jetz, W. &amp; Pyron, R. A. Fully-sampled phylogenies of squamates reveal evolutionary patterns in threat status. Biol. Conserv. 204, 23–31 (2016)." />
<meta name="citation_reference" content="Jetz, W. &amp; Pyron, R. A. The interplay of past diversification and evolutionary isolation with present imperilment across the amphibian tree of life. Nat. Ecol. Evol. (2018). doi:10.1038/s41559-018-0515-5" />
<meta name="citation_reference" content="Jetz, W., Thomas, G. H., Joy, J. B., Hartmann, K. &amp; Mooers, A. O. The global diversity of birds in space and time. Nature 491, 444–448 (2012)." />
<meta name="citation_reference" content="Batt, R. D., Morley, J. W., Selden, R. L., Tingley, M. W. &amp; Pinsky, M. L. Gradual changes in range size accompany long-term trends in species richness. Ecol. Lett. 20, 1148–1157 (2017)." />
<meta name="citation_reference" content="Oliver, T. H. et al. Biodiversity and Resilience of Ecosystem Functions. Trends Ecol. Evol. 30, 673–684 (2015)." />
<meta name="citation_reference" content="Hansen, M. C. et al. High-resolution global maps of 21st-century forest cover change. Science 342, 850–853 (2013)." />
<meta name="citation_reference" content="Bowler, D. et al. The geography of the Anthropocene differs between the land and the sea. (2018). doi:10.1101/432880" />
<meta name="citation_reference" content="Halpern, B. S. et al. Spatial and temporal changes in cumulative human impacts on the world’s ocean. Nat. Commun. 6, (2015)." />
<meta name="citation_reference" content="Barlow, J. et al. Quantifying the biodiversity value of tropical primary, secondary, and plantation forests. Proc. Natl. Acad. Sci. 104, 18555–18560 (2007)." />
<meta name="citation_reference" content="Blowes, S., Supp, S. R. et al. Biodiversity trends are stronger in marine than terrestrial assemblages. (2018). doi:10.1101/457424" />
<meta name="citation_reference" content="Isaac, N. J. B. &amp; Cowlishaw, G. How species respond to multiple extinction threats. Proc. R. Soc. Lond. B Biol. Sci. 271, 1135–1141 (2004)." />
<meta name="citation_reference" content="Khaliq, I., Hof, C., Prinzinger, R., Bohning-Gaese, K. &amp; Pfenninger, M. Global variation in thermal tolerances and vulnerability of endotherms to climate change. Proc. R. Soc. B Biol. Sci. 281, 20141097–20141097 (2014)." />
<meta name="citation_reference" content="Morrison, L., Estrada, A. &amp; Early, R. Species traits suggest European mammals facing the greatest climate change are also least able to colonize new locations. Divers. Distrib. 24, 1321–1332 (2018)." />
<meta name="citation_reference" content="Pigott, C. D. &amp; Walters, S. M. The scientific basis of practical conservation: aims and methods of conservation. in Proc. R. Soc. Lond. B 197, 59–68 (The Royal Society, 1977)." />
<meta name="citation_reference" content="Longton, R. E. &amp; Hedderson, T. A. What are rare species and why conserve them? Lindbergia 25, 53–61 (2000)." />
<meta name="citation_reference" content="Gaston, K. &amp; Fuller, R. Commonness, population depletion and conservation biology. Trends Ecol. Evol. 23, 14–19 (2008)." />
<meta name="citation_reference" content="Lande, R. Risks of population extinction from demographic and environmental stochasticity and random catastrophes. Am. Nat. 142, 911–927 (1993)." />
<meta name="citation_reference" content="Melbourne, B. A. &amp; Hastings, A. Extinction risk depends strongly on factors contributing to stochasticity. Nature 454, 100–103 (2008)." />
<meta name="citation_reference" content="Kilpatrick, A. M. &amp; Ives, A. R. Species interactions can explain Taylor’s power law for ecological time series. Nature 422, 65–68 (2003)." />
<meta name="citation_reference" content="Kareiva, P. Population dynamics in spatially complex environments: theory and data. Phil Trans R Soc Lond B 330, 175–190 (1990)." />
<meta name="citation_reference" content="Hanski, I. Metapopulation dynamics. Nature 396, 41 (1998)." />
<meta name="citation_reference" content="Dennis, B., Assas, L., Elaydi, S., Kwessi, E. &amp; Livadiotis, G. Allee effects and resilience in stochastic populations. Theor. Ecol. 9, 323–335 (2016)." />
<meta name="citation_reference" content="Sun, G.-Q. Mathematical modeling of population dynamics with Allee effect. Nonlinear Dyn. 85, 1–12 (2016)." />
<meta name="citation_reference" content="Bowler, D. E., Heldbjerg, H., Fox, A. D., O’Hara, R. B. &amp; Böhning-Gaese, K. Disentangling the effects of multiple environmental drivers on population changes within communities. J. Anim. Ecol. 87, 1034–1045 (2018)." />
<meta name="citation_reference" content="Harnik, P. G., Simpson, C. &amp; Payne, J. L. Long-term differences in extinction risk among the seven forms of rarity. Proc. R. Soc. B Biol. Sci. 279, 4969–4976 (2012)." />
<meta name="citation_reference" content="Steffen, W., Crutzen, P. J. &amp; McNeill, J. R. The Anthropocene: Are Humans Now Overwhelming the Great Forces of Nature. AMBIO J. Hum. Environ. 36, 614–621 (2007)." />
<meta name="citation_reference" content="Maxwell, S. L., Fuller, R. A., Brooks, T. M. &amp; Watson, J. E. Biodiversity: The ravages of guns, nets and bulldozers. Nature 536, 143–145 (2016)." />
<meta name="citation_reference" content="van Strien, A. J. et al. Modest recovery of biodiversity in a western European country: The Living Planet Index for the Netherlands. Biol. Conserv. 200, 44–50 (2016)." />
<meta name="citation_reference" content="Mace, G. M. et al. Quantification of Extinction Risk: IUCN’s System for Classifying Threatened Species. Conserv. Biol. 22, 1424–1442 (2008)." />
<meta name="citation_reference" content="Akçakaya, H. R. et al. Quantifying species recovery and conservation success to develop an IUCN Green List of Species: Species Recovery and Conservation Success. Conserv. Biol. 32, 1128–1138 (2018)." />
<meta name="citation_reference" content="Humbert, J.-Y., Scott Mills, L., Horne, J. S. &amp; Dennis, B. A better way to estimate population trends. Oikos 118, 1940–1946 (2009)." />
<meta name="citation_reference" content="Leung, B., Greenberg, D. A. &amp; Green, D. M. Trends in mean growth and stability in temperate vertebrate populations. Divers. Distrib. 23, 1372–1380 (2017)." />
<meta name="citation_reference" content="Fritz, S. A., Bininda-Emonds, O. R. P. &amp; Purvis, A. Geographical variation in predictors of mammalian extinction risk: big is bad, but only in the tropics. Ecol. Lett. 12, 538–549 (2009)." />
<meta name="citation_reference" content="Brook, B. W., Traill, L. W. &amp; Bradshaw, C. J. A. Minimum viable population sizes and global extinction risk are unrelated: MVP is unrelated to global extinction risk. Ecol. Lett. 9, 375– 382 (2006)." />
<meta name="citation_reference" content="Pereira, H. M., Navarro, L. M. &amp; Martins, I. S. Global Biodiversity Change: The Bad, the Good, and the Unknown. Annu. Rev. Environ. Resour. 37, 25–50 (2012)." />
<meta name="citation_reference" content="Daskalova, G. N. et al. Forest loss as a catalyst of population and biodiversity change: bioRxiv (2018). doi:10.1101/473645" />
<meta name="citation_reference" content="CBD. Strategic plan for biodiversity 2011–2020 and the Aichi targets. In Report of the Tenth Meeting of the Conference of the Parties to the Convention on Biological Diversity. (2010)." />
<meta name="citation_reference" content="Purvis, A., Gittleman, J. L., Cowlishaw, G. &amp; Mace, G. M. Predicting extinction risk in declining species. Proc. R. Soc. B Biol. Sci. 267, 1947–1952 (2000)." />
<meta name="citation_reference" content="Cardillo, M. et al. Human Population Density and Extinction Risk in the World’s Carnivores. PLoS Biol. 2, e197 (2004)." />
<meta name="citation_reference" content="Pounds, A. J. et al. Widespread amphibian extinctions from epidemic disease driven by global warming. Nature 439, 161–167 (2006)." />
<meta name="citation_reference" content="Green, D. M. The ecology of extinction: population fluctuation and decline in amphibians. Biol. Conserv. 111, 331–343 (2003)." />
<meta name="citation_reference" content="Moller, A. P., Rubolini, D. &amp; Lehikoinen, E. Populations of migratory bird species that did not show a phenological response to climate change are declining. Proc. Natl. Acad. Sci. 105, 16195–16200 (2008)." />
<meta name="citation_reference" content="Dornelas, M. et al. A balance of winners and losers in the Anthropocene. Ecol. Lett. [in press] (2019)." />
<meta name="citation_reference" content="Caughley, G. Directions in Conservation Biology. J. Anim. Ecol. 63, 215 (1994)." />
<meta name="citation_reference" content="Robbirt, K. M., Roberts, D. L. &amp; Hawkins, J. A. Comparing IUCN and Probabilistic Assessments of Threat: Do IUCN Red List Criteria Conflate Rarity and Threat? Biodivers. Conserv. 15, 1903–1912 (2006)." />
<meta name="citation_reference" content="Harrison, S., Viers, J. H., Thorne, J. H. &amp; Grace, J. B. Favorable environments and the persistence of naturally rare species. Conserv. Lett. 1, 65–74 (2008)." />
<meta name="citation_reference" content="Mihoub, J.-B. et al. Setting temporal baselines for biodiversity: the limits of available monitoring data for capturing the full impact of anthropogenic pressures. Sci. Rep. 7, 41591 (2017)." />
<meta name="citation_reference" content="Vellend, M. et al. Extinction debt of forest plants persists for more than a century following habitat fragmentation. Ecology 87, 542–548 (2006)." />
<meta name="citation_reference" content="Fagan, W. F. &amp; Holmes, E. E. Quantifying the extinction vortex. Ecol. Lett. 0, 051109031307004 (2005)." />
<meta name="citation_reference" content="WWF. Living Planet Report - 2018: Aiming Higher. Grooten, M. and Almond, R.E.A.(Eds). WWF, Gland, Switzerland. (2018)." />
<meta name="citation_reference" content="McRae, L., Freeman, R. &amp; Marconi, V. ‘The Living Planet Index’ in: Living Planet Report 2016: Risk and resilience in a new era (ed. Oerlemans N). WWF International, Gland, Switzerland. (2016)." />
<meta name="citation_reference" content="McRae, L. et al. The Living Planet Index in: Living Planet Report 2012 (ed. Almond R). WWF, Gland, Switzerland. (2012)." />
<meta name="citation_reference" content="WWF. Living Planet Report 2018 Technical Supplement: Living Planet Index. (2018)." />
<meta name="citation_reference" content="McRae, L., Deinet, S. &amp; Freeman, R. The Diversity-Weighted Living Planet Index: Controlling for Taxonomic Bias in a Global Biodiversity Indicator. PLOS ONE 12, e0169156 (2017)." />
<meta name="citation_reference" content="Davidson, A. D. et al. Geography of current and future global mammal extinction risk. PLOS ONE 12, e0186934 (2017)." />
<meta name="citation_reference" content="Spooner, F. E. B., Pearson, R. G. &amp; Freeman, R. Rapid warming is associated with population decline among terrestrial birds and mammals globally. Glob. Change Biol. (2018). doi:10.1111/gcb.14361" />
<meta name="citation_reference" content="Keogan, K. et al. Global phenological insensitivity to shifting ocean temperatures among seabirds. Nat. Clim. Change 8, 313–318 (2018)." />
<meta name="citation_reference" content="Fournier, A. M. V., White, E. R. &amp; Heard, S. B. Site-selection bias can drive apparent population declines in long-term studies. doi:10.7287/peerj.preprints.27507v1" />
<meta name="citation_reference" content="Dunic, J. C. et al. Attributing the variability in direction and magnitude of local-scale marine biodiversity change to human activities. bioRxiv (2017). doi:10.1101/162362" />
<meta name="citation_reference" content="D. Vinebrooke, R. et al. Impacts of multiple stressors on biodiversity and ecosystem functioning: the role of species co-tolerance. Oikos 104, 451–457 (2004)." />
<meta name="citation_reference" content="R Core Team. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. (2017)." />
<meta name="citation_reference" content="LPI. Living Planet Index database, 2016. www.livingplanetindex.org/ Downloaded February 2016. (2016)." />
<meta name="citation_reference" content="Wauchope, H. S., Johnston, A., Amano, T. &amp; Sutherland, W. J. When can we trust population trends? A method for quantifying the effects of sampling interval and duration. bioRxiv (2019). doi:10.1101/498170" />
<meta name="citation_reference" content="van de Pol, M. &amp; Wright, J. A simple method for distinguishing within-versus between-subject effects using mixed models. Anim. Behav. 77, 753–758 (2009)." />
<meta name="citation_reference" content="Foley, N. M., Springer, M. S. &amp; Teeling, E. C. Mammal madness: is the mammal tree of life not yet resolved? Philos. Trans. R. Soc. B Biol. Sci. 371, 20150140 (2016)." />
<meta name="citation_reference" content="Tarver, J. E. et al. The Interrelationships of Placental Mammals and the Limits of Phylogenetic Inference. Genome Biol. Evol. 8, 330–344 (2016)." />
<meta name="citation_reference" content="Rabinowitz, D. Seven forms of rarity. in The biological aspects of rare plants conservation 205–217 (1981)." />
<meta name="citation_reference" content="GBIF. Biodiversity occurrence data. http://www.gbif.org/ Downloaded Feb 2016 - Apr 2017. (2017)." />
<meta name="citation_reference" content="BirdLife International. IUCN Red List for birds. Downloaded from http://www.birdlife.org in March 2018. (2018)." />
<meta name="citation_reference" content="IUCN. The IUCN Red List of Threatened Species. Version 2017-3. (2017)." />
<meta name="citation_reference" content="Di Marco, M., Venter, O., Possingham, H. P. &amp; Watson, J. E. M. Changes in human footprint drive changes in species extinction risk. Nat. Commun. 9, (2018)." />
<meta name="citation_reference" content="Dennis, B., Ponciano, J. M., Lele, S. R., Taper, M. L. &amp; Staples, D. F. Estimating density dependence, process noise, and observation error. Ecol. Monogr. 76, 323–341 (2006)." />
<meta name="citation_reference" content="Knape, J., Jonzén, N. &amp; Sköld, M. On observation distributions for state space models of population survey data: Observation models for population data. J. Anim. Ecol. 80, 1269– 1277 (2011)." />
<meta name="citation_reference" content="Pedersen, M. W., Berg, C. W., Thygesen, U. H., Nielsen, A. &amp; Madsen, H. Estimation methods for nonlinear state-space models in ecology. Ecol. Model. 222, 1394–1400 (2011)." />
<meta name="citation_reference" content="Rueda-Cediel, P., Anderson, K. E., Regan, T. J. &amp; Regan, H. M. Effects of uncertainty and variability on population declines and IUCN Red List classifications. Conserv. Biol. (2018). doi:10.1111/cobi.13081" />
<meta name="citation_reference" content="Jones, K. E. et al. PanTHERIA: a species-level database of life history, ecology, and geography of extant and recently extinct mammals: Ecological Archives E090-184. Ecology 90, 2648–2648 (2009)." />
<meta name="citation_reference" content="Phillips, H. R. P., Newbold, T. &amp; Purvis, A. Land-use effects on local biodiversity in tropical forests vary between continents. Biodivers. Conserv. 26, 2251–2270 (2017)." />
<meta name="citation_reference" content="Zizka, A. et al. CoordinateCleaner: Standardized cleaning of occurrence records from biological collection databases. Methods Ecol. Evol. (2019). doi:10.1111/2041-210X.13152" />
<meta name="citation_reference" content="Hijmans, R. J. geosphere: Spherical Trigonometry. R package version 1.5-5. https://CRAN.R-project.org/package=geosphere. (2016)." />
<meta name="citation_reference" content="Chamberlain, S. rredlist: ‘IUCN’ Red List Client. R package version 0.4.0. https://CRAN.R-project.org/package=rredlist. (2017)." />
<meta name="citation_reference" content="Hadfield, J. D. MCMC Methods for Multi-Response Generalized Linear Mixed Models: The MCMCglmm R Package. J. Stat. Softw. 33, 1–22 (2010)." />
<meta name="citation_reference" content="Olson, D. M. &amp; Dinerstein, E. The Global 200: Priority Ecoregions for Global Conservation. Ann. Mo. Bot. Gard. 89, 199 (2002)." />
<meta name="citation_reference" content="Magurran, A. E. &amp; Henderson, P. A. Explaining the excess of rare species in natural species abundance distributions. Nature 422, 714 (2003)." />
<meta name="twitter:title" content="All is not decline across global vertebrate populations" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The populations of Earth’s species are changing over time in complex ways, creating a mixture of winners and losers in a time of accelerating global change. A critical research challenge is to test if there are specific biomes, taxa or types of species that are experiencing rapid alterations in abundance over time. We conducted an analysis of the Living Planet Database using a state-space modelling approach including nearly 10,000 abundance time-series from over 2,000 vertebrate species. We integrated the population abundance data with information on geographic range, habitat preference, taxonomic and phylogenetic relationships, conservation status and threats from occurrence, phylogenetic and conservation assessment data. We found that 15% of populations were declining, 18% were increasing, and 67% showed no net changes in abundance over time. Against a backdrop of no biogeographic and phylogenetic patterning in population change, we uncovered a distinct taxonomic signal. Amphibians were the only taxa that experienced net declines in the analyzed data, while birds, mammals and reptiles on average became more abundant over time. The continuum of abundance changes over time was poorly captured by species’ rarity and global-scale threats. Capturing the full spectrum of abundance change of species around the planet will inform conservation efforts and improve projections of biodiversity and ecosystem function change under the accelerating influence of the Anthropocene." />
<meta name="og-title" property="og:title" content="All is not decline across global vertebrate populations" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/272898v5" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The populations of Earth’s species are changing over time in complex ways, creating a mixture of winners and losers in a time of accelerating global change. A critical research challenge is to test if there are specific biomes, taxa or types of species that are experiencing rapid alterations in abundance over time. We conducted an analysis of the Living Planet Database using a state-space modelling approach including nearly 10,000 abundance time-series from over 2,000 vertebrate species. We integrated the population abundance data with information on geographic range, habitat preference, taxonomic and phylogenetic relationships, conservation status and threats from occurrence, phylogenetic and conservation assessment data. We found that 15% of populations were declining, 18% were increasing, and 67% showed no net changes in abundance over time. Against a backdrop of no biogeographic and phylogenetic patterning in population change, we uncovered a distinct taxonomic signal. Amphibians were the only taxa that experienced net declines in the analyzed data, while birds, mammals and reptiles on average became more abundant over time. The continuum of abundance changes over time was poorly captured by species’ rarity and global-scale threats. Capturing the full spectrum of abundance change of species around the planet will inform conservation efforts and improve projections of biodiversity and ecosystem function change under the accelerating influence of the Anthropocene." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-11-27" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/272898v5.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/272898v5.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/272898v5.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/272898v5" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1011534" />
    <title>All is not decline across global vertebrate populations | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"PUt-QIzAIOwXL75k5Bd9euO4N_Z3kT7lq3b15x5izh8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1011534","apath":"\/biorxiv\/early\/2019\/11\/27\/272898.atom","pisa":"biorxiv;272898v5","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;272898v5"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/272898v5","title":"All is not decline across global vertebrate populations","identifier":"node\/1011534"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xBC64D2GtJEK2YoFPJWclrQ77mK40aL90vdR8hBt+EniSpojbscQIS1sddWQaUS1XPzqJegbOWTJn4rGSn8ycayYtB7\\\\\\\/MBGSqlQanBaRbuzOeiRa30MXkkO\\\\\\\/Tz+N5nFtFThR3KY+qe1KbJ+tqi2NAh1KKfnm6mdeMho6qkWIRMtu35uETq42+8kbENbhgZKrzwder1bmbDfmIovqSWtm7\\\\\\\/LQgQY9pxug6R7mMG3wEK43hXNOyZ10CoAnA8cEOFpufM8\\\\\\\/YHskdqb41VAgAee9lnvyYm+795WBFStFHqEjZ1TbT1Fw1h7G7akfOa\\\\\\\/DrSgM5puEfcUrcevICeHwD+1qGkL\\\\\\\/P9KvXOiYbFv40lO+YglGPHFHhkIAWl4n8VFmSdSbzBIBoTiMf81YFlkx1xF50CadB7VEmFKLzCnkb+tpQQkz8IhxzJ4ZNNTapMxbhkqPHJvzomfarixfrz45+\\\\\\\/QXvwPN3y6ITZa3WN15Zbgw70e8dzZp8oWrDLdzF+DTqtaXTiHr2l1hFHwHXrtvDCUQXZcoVDg2JIG9dXhulCxZYP6Z5H3QTv2SMFPUYlIuoZUv0Fe\\\\\\\/IDD+XRWoe3Y\\\\\\\/uAd4I3J0iQkX6PkchphsBPN3O\\\\\\\/9kkLjnAFwtIdNgUV+oU0Izh05zerfXo9Vw2ixpYcwQcC42FXnl0FcQ1sSt8lsesmLNqr\\\\\\\/Vke+pbsJMVjudSt9Hh9l3jQ2Cz64RbQUtHYaSzqPR\\\\\\\/ICgyeU3f39rcFJy+wUMyYR6wvc1\\\\\\\/cYvqklUW+x+cPMPKpO8djcjIOFV24ffZQ1X8fA8fxZLFdD5fZRwgb\\\\\\\/TO4JhwB53DDjPChoGQ1\\\\\\\/2ir0L8E\\\\\\\/sk49RoE9N0lA72ZWtJuNjE2AWMHLr90jsT55WLtc5NMy4eGNp1UdV\\\\\\\/G5AZsxgqmB1zYzrHQlCJvI1MfJDMlqG5hifo\\\\\\\/P2nmvYW04oLrobSqwRK6bH+LeB6CP37JW5k970xuTDtqDC2aLpdBJry7EhyBJK1uTC9Nup86jz83MIkUvvY+izq3LJirWUBGFzx8oaEeVhW8CPfkJs1SaVT5o8WL6AJ7lF+gKrZzmee9I0JhnUPdmdI3erUIcGs5ePceSK8cnS4lUzZTtQy7XlOj84Q0PcT3NuKWyDyDNgUYaEsfe0w16UONSL3GJmqCfZ2uXkMJIL0R3QXX7ZEI7tAwTHO2vuLwTUJJ4eG4UceBOQulXn63EOljfGNAhxS2Tbb8F0KgsA4vDEiGK0jjzN3RJF8LY2PfTVSkksjegO\\\\\\\/OdYpNxrRbC9lOtMcTYCQsyql4hMDwUPlybFvt8fRkBa4N\\\\\\\/m\\\\\\\/vf2zwbnMD3h3p9Gv2\\\\\\\/qBrcNtlQ02\\\\\\\/Rqrur4\\\u0022,\\\u0022iv\\\u0022:\\\u0022Gwfmhm7sevMBPe62KMvsbQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022cf0cf526d576c705ca4abfa0c350b230795587c83d47b5a058c9d736ba1cb657\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/272898v5.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1011534 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/272898v5.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W5E2kv1mH_r_o84ASLyOajDhf_rqPwBiPmZMqW6bzrE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1011534" id="node1011534" data-pisa="biorxiv;272898v5" data-pisa-master="biorxiv;272898" data-apath="/biorxiv/early/2019/11/27/272898.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">All is not decline across global vertebrate populations</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5674-5322" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Gergana N.</span> <span class="nlm-surname">Daskalova</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8417-6112" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Isla H.</span> <span class="nlm-surname">Myers-Smith</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5595-255X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">John L.</span> <span class="nlm-surname">Godlee</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/272898 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1011534" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Gergana N. Daskalova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of GeoSciences, University of Edinburgh</span>, Edinburgh EH9 3FF, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gergana%2BN.%2BDaskalova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Daskalova%20GN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGergana%2BN.%2BDaskalova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5674-5322" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gergana N. Daskalova</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gndaskalova@gmail.com" class="" data-icon-position="" data-hide-link-title="0">gndaskalova@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Isla H. Myers-Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of GeoSciences, University of Edinburgh</span>, Edinburgh EH9 3FF, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Isla%2BH.%2BMyers-Smith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Myers-Smith%20IH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIsla%2BH.%2BMyers-Smith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8417-6112" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Isla H. Myers-Smith</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John L. Godlee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of GeoSciences, University of Edinburgh</span>, Edinburgh EH9 3FF, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BL.%2BGodlee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Godlee%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BL.%2BGodlee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5595-255X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for John L. Godlee</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/272898v5" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1011534" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1011534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/272898v5.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1011534" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1011534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/272898v5.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1011534" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1011534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/272898v5.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1011534" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1011534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/272898v5.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1011534" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1011534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/272898v5.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1011534" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1011534/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">The populations of Earth’s species are changing over time in complex ways, creating a mixture of winners and losers in a time of accelerating global change. A critical research challenge is to test if there are specific biomes, taxa or types of species that are experiencing rapid alterations in abundance over time. We conducted an analysis of the Living Planet Database using a state-space modelling approach including nearly 10,000 abundance time-series from over 2,000 vertebrate species. We integrated the population abundance data with information on geographic range, habitat preference, taxonomic and phylogenetic relationships, conservation status and threats from occurrence, phylogenetic and conservation assessment data. We found that 15% of populations were declining, 18% were increasing, and 67% showed no net changes in abundance over time. Against a backdrop of no biogeographic and phylogenetic patterning in population change, we uncovered a distinct taxonomic signal. Amphibians were the only taxa that experienced net declines in the analyzed data, while birds, mammals and reptiles on average became more abundant over time. The continuum of abundance changes over time was poorly captured by species’ rarity and global-scale threats. Capturing the full spectrum of abundance change of species around the planet will inform conservation efforts and improve projections of biodiversity and ecosystem function change under the accelerating influence of the Anthropocene.</p></div><div class="section" id="sec-1"><h2 class="">Main text</h2><p id="p-5">Alterations to ecosystems currently unfolding all around the globe are modifying the abundances of the different species forming Earth’s biota. As global change continues to accelerate<sup><a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a></sup>, the need for comprehensive assessments of the factors explaining the variety of ecological changes observed across taxa and biomes is also growing<sup><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a></sup>. However, existing empirical studies of the predictors of population change mostly focus on either specific taxa<sup><a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a></sup> or on population declines alone<sup><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a></sup>. A critical research challenge is to disentangle the sources of heterogeneity across the full spectrum of population change. Recent compilations of long-term population time-series, extensive occurrence, phylogenetic, habitat preference and conservation status data<sup><a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>–<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a></sup> provide a unique opportunity to quantify population trends and fluctuations among the world’s well-monitored vertebrate species and test which species- and population-level attributes explain variation in population change. Population change is the underlying process leading to community reassembly<sup><a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a></sup> and the resulting changes to biodiversity are vitally important for ecosystem functions and services<sup><a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a></sup>.</p><p id="p-6">The distribution of global change drivers such as such as land-use change, habitat change, pollution, invasion by non-native species and climate change<sup><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>–<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a></sup> shows distinct clustering across space. Thus, we predicted biogeographic patterning of population trends and fluctuations across the planet’s realms and biomes, in line with particular regions of the world experiencing high rates of environmental change (e.g., tropical forests<sup><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a></sup>). Spatial clustering has also been documented for biodiversity trends derived from assemblage time-series, with the marine realm emerging as a hotspot for rapid changes in community composition<sup><a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a></sup>. Since assemblages are made up of populations, the biogeographic patterns at the assemblage level suggest similar clustering might occur at the population level as well. In addition to geographic patterns in exposure to anthropogenic activities, species’ vulnerability can also influence population abundance over time<sup><a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a></sup>. Species traits can moderate population responses to natural and anthropogenic environmental change<sup><a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a></sup>, and across evolutionary time, certain clades have been more vulnerable to declines and extinctions<sup><a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>–<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a></sup>. Building on known variability in species’ vulnerability<sup><a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>–<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a></sup>, we expected taxonomic and phylogenetic signals in population trends and fluctuations (e.g. greater declines, increases or fluctuations in abundance for specific taxa and among specific clades). Understanding which biomes and taxa are experiencing the most acute changes in abundance over time could provide key insights for conservation prioritization.</p><p id="p-7">Conservation efforts often focus on protecting rare species, those with restricted geographic extents, small population sizes or high habitat specificity, as they are assumed to be more likely to decline and ultimately go extinct<sup><a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>–<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a></sup>. Thus, we predicted that the populations of rare species are more likely to decline and fluctuate than the populations of common species. Species with a smaller geographic range might have more concentrated exposure to environmental change, with less opportunities to find refugia or disperse, thus increasing the likelihood of declines<sup><a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a></sup>. As per population dynamics theory<sup><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a></sup> and Taylor’s power law<sup><a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a></sup>, species with small populations are more likely to undergo stochastic fluctuations that could lead to pronounced declines, local extinction and eventually global species extinction<sup><a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a></sup>. Small populations are also more likely to decline due to inbreeding<sup><a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a></sup>. Allee effects, the relationship between individual fitness and population density, further increase the likelihood of declines due to lack of potential mates and low reproductive output once populations reach a critically low density<sup><a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a></sup>. Furthermore, a high habitat specificity means that environmental change might have disproportionately large effects on the populations of rare species, as their persistence and colonization of new areas are limited by strict habitat preferences<sup><a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a></sup>. The fossil record indicates that on millennial time scales, rare species are more likely to decline and ultimately go extinct<sup><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a></sup>, but human actions have pushed Earth away from traditional geological trajectories<sup><a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a></sup>, and the relationships between rarity and population change across the planet have yet to be tested across the Anthropocene.</p><p id="p-8">On a global scale, species are exposed to a variety of threats, among which habitat change, resource exploitation and hunting dominate as key predictors of extinction risk across the planet’s species<sup><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a></sup>. We predicted that the populations of species with more threatened conservation status and more species-level threats are more likely to decline and fluctuate than the populations of least concern species and those exposed to a smaller number of threats. Species’ IUCN conservation status levels are often used in conservation prioritization and threatened species tend to be the focus of conservation initiatives, but on the local scale, there might be variation in how species abundance is changing over time in different locations<sup><a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a></sup>. Testing population change across species’ IUCN conservation status allows us to link contemporary changes in abundance with long-term probability of extinction<sup><a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a></sup>. Determining how local-scale population trends vary across species’ IUCN conservation status has practical applications for assessing species’ recovery which is useful for the proposed IUCN Green List of Species<sup><a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a></sup>.</p><p id="p-9">Here, we asked how the trends and fluctuations of vertebrate populations vary with biogeography, taxa, phylogenetic relationships and across species’ rarity metrics and IUCN conservation and threat categories. We quantified differences in population trends and fluctuations across latitudes and biomes within the freshwater, marine and terrestrial realms to test the presence of distinct hotspots of declines and increases. Additionally, we assessed taxonomic and phylogenetic signals in the patterns of population change to determine if specific groups experience pronounced changes over time. We investigated whether the heterogeneity in population change globally is explained by differences in species’ rarity and IUCN conservation status. We measured rarity using three separate metrics – geographic range, mean population size (number of animals that were recorded by monitoring) and habitat specificity. In a <em>post-hoc</em> analysis, we categorized each population based on the global IUCN threat categories to determine how the number and type of threats influences local-scale population change. Using the largest currently available compilation of population records over time, we conducted a global synthesis of population trends and fluctuations to provide key empirical evidence for the management, conservation and prediction of ecological changes across the Anthropocene.</p><p id="p-10">We analyzed vertebrate population time-series from the Living Planet Database (133,092 records) covering the period between 1970 and 2014. These time-series represent repeated monitoring surveys of the number of individuals in a given area (species’ abundance over time), hereafter called “populations”. We focus on two aspects of population change – overall changes in abundance over time (population trend) and abundance variability over time (population fluctuations). In the first stage of our analyses, we quantified trends and fluctuations for each population using state-space models that account for observation error and random fluctuations<sup><a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a></sup> (Figure S1). In the second stage, we modelled the population trend and fluctuation estimates from the first stage across latitude, realm, biome, taxa, rarity metrics, phylogenetic relatedness, species’ conservation status and threat type using a Bayesian modelling framework (Figure S2). We included a species random intercept effect to account for the possible correlation between the trends of populations from the same species (see table Table S1 for sample sizes). As sensitivity analyses, we additionally used variance weighting of the population trend estimates (μ) by the observation/measurement error around them (τ) and population trend estimates from linear model fits (slopes instead of μ) as the input variables in the second stage models, as well as several different fluctuations estimates. We also repeated our analyses on a single-country scale, using only populations within the United Kingdom, where monitoring efforts are particularly rigorous and extensive. All different analytical approaches yielded very similar results and are described in further detail in the methods.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Biogeographic patterns of population trends and fluctuations</h3><p id="p-11">Globally, population increases, declines and fluctuations over time occurred across all latitudes and biomes within the freshwater, marine and terrestrial realms, with no strong biogeographic patterning and no specific hotspots of population declines (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, Table S2). Across all time-series, 15% of populations were declining, 18% were increasing, and 67% showed no net changes in abundance over time. Across realms, monitored vertebrate populations experienced net population increases (freshwater slope = 0.005, CI = 0.002 to 0.01; marine slope = 0.004, CI = 0.002 to 0.01; terrestrial slope = 0.003, CI = 0.001 to 0.005, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1d-e</a>). In the freshwater and terrestrial realms, there was a bimodal distribution of population trends (Hartigans’ dip test, D = 0.04, p &lt; 0.01). Across biomes, populations in Mediterranean forests, montane grasslands, polar freshwaters, temperate wetlands, tropical forests and tropical coral biomes were more likely to increase, whereas populations from the remaining studied biomes experienced no net changes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1h</a>, Table S2). Population fluctuations were less pronounced in the terrestrial realm (slope = 0.02, CI = 0.018 to 0.021, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1f-g</a>), but those populations were also monitored for the longest duration across systems (average duration – 28 years for terrestrial, 18 years for marine and 21 years for freshwater populations, Figure S1, Table S2).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population declines, increases and fluctuations over time occur across all latitudes and biomes within the freshwater, marine and terrestrial realms. Results include 9286 populations from 2084 species. The lack of biogeographic patterning in vertebrate population trends was also apparent on a UK scale (Figure S6 and Table S2). The numbers in the legend for plots d-g and on the x axis in plot c show the sample sizes for realms and biomes, respectively. The &#x3BC; values of population trend (plots a-b, d-e, h) and the &#x3C3;2 values of population fluctuation (plots c, f-g) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. Plots d and f show the density distribution of population trends across realms, the raw values (points) and boxplots with the mean, first and third quartiles and boxplot whiskers that indicate the distance that covers 1.5 times the interquartile range. Plots e, g and h show the effect sizes and the 95% credible intervals of population trends (e, h) across realms and biomes, and fluctuations across realms (g). For variation in fluctuations across biomes, see SI Figure S9. The three estimates for each realm/system in plots e and h refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (&#x3BC;, triangles), and population trends (&#x3BC;) weighted by &#x3C4;2, the observation error estimate from the state-space models (squares). The five estimates in plot g refer to different analytical approaches, where the response variables in the models were: 1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the &#x3BC; value of population change (triangles), 3) half of the 95% confidence interval around &#x3BC; weighted by &#x3C4;2, (full squares), 4) the process noise (&#x3C3;2) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). Effect sizes were standardized by dividing the effect size by the standard deviation of the corresponding input data. The process noise is the total variance around the population trend minus the variance attributed to observation error. Error bars in plots e, g and h show 95% credible intervals. See Table S2 for model outputs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-375678661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Population declines, increases and fluctuations over time occur across all latitudes and biomes within the freshwater, marine and terrestrial realms.&lt;/span&gt; Results include 9286 populations from 2084 species. The lack of biogeographic patterning in vertebrate population trends was also apparent on a UK scale (Figure S6 and Table S2). The numbers in the legend for plots &lt;strong&gt;d-g&lt;/strong&gt; and on the x axis in plot &lt;strong&gt;c&lt;/strong&gt; show the sample sizes for realms and biomes, respectively. The &lt;em&gt;&#x3BC;&lt;/em&gt; values of population trend (plots &lt;strong&gt;a-b&lt;/strong&gt;, &lt;strong&gt;d-e&lt;/strong&gt;, &lt;strong&gt;h&lt;/strong&gt;) and the &lt;em&gt;&#x3C3;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; values of population fluctuation (plots &lt;strong&gt;c&lt;/strong&gt;, &lt;strong&gt;f-g&lt;/strong&gt;) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. Plots &lt;strong&gt;d&lt;/strong&gt; and &lt;strong&gt;f&lt;/strong&gt; show the density distribution of population trends across realms, the raw values (points) and boxplots with the mean, first and third quartiles and boxplot whiskers that indicate the distance that covers 1.5 times the interquartile range. Plots &lt;strong&gt;e&lt;/strong&gt;, &lt;strong&gt;g&lt;/strong&gt; and &lt;strong&gt;h&lt;/strong&gt; show the effect sizes and the 95% credible intervals of population trends (&lt;strong&gt;e&lt;/strong&gt;, &lt;strong&gt;h&lt;/strong&gt;) across realms and biomes, and fluctuations across realms (&lt;strong&gt;g&lt;/strong&gt;). For variation in fluctuations across biomes, see SI Figure S9. The three estimates for each realm/system in plots &lt;strong&gt;e&lt;/strong&gt; and &lt;strong&gt;h&lt;/strong&gt; refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (&lt;em&gt;&#x3BC;&lt;/em&gt;, triangles), and population trends (&lt;em&gt;&#x3BC;&lt;/em&gt;) weighted by &#x3C4;&lt;sup&gt;2&lt;/sup&gt;, the observation error estimate from the state-space models (squares). The five estimates in plot &lt;strong&gt;g&lt;/strong&gt; refer to different analytical approaches, where the response variables in the models were: 1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the &lt;em&gt;&#x3BC;&lt;/em&gt; value of population change (triangles), 3) half of the 95% confidence interval around &lt;em&gt;&#x3BC;&lt;/em&gt; weighted by &#x3C4;&lt;sup&gt;2&lt;/sup&gt;, (full squares), 4) the process noise (&lt;em&gt;&#x3C3;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). Effect sizes were standardized by dividing the effect size by the standard deviation of the corresponding input data. The process noise is the total variance around the population trend minus the variance attributed to observation error. Error bars in plots &lt;strong&gt;e&lt;/strong&gt;, &lt;strong&gt;g&lt;/strong&gt; and &lt;strong&gt;h&lt;/strong&gt; show 95% credible intervals. See Table S2 for model outputs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F1.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F1.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Population declines, increases and fluctuations over time occur across all latitudes and biomes within the freshwater, marine and terrestrial realms.</span><p id="p-12" class="first-child">Results include 9286 populations from 2084 species. The lack of biogeographic patterning in vertebrate population trends was also apparent on a UK scale (Figure S6 and Table S2). The numbers in the legend for plots <strong>d-g</strong> and on the x axis in plot <strong>c</strong> show the sample sizes for realms and biomes, respectively. The <em>μ</em> values of population trend (plots <strong>a-b</strong>, <strong>d-e</strong>, <strong>h</strong>) and the <em>σ<sup>2</sup></em> values of population fluctuation (plots <strong>c</strong>, <strong>f-g</strong>) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. Plots <strong>d</strong> and <strong>f</strong> show the density distribution of population trends across realms, the raw values (points) and boxplots with the mean, first and third quartiles and boxplot whiskers that indicate the distance that covers 1.5 times the interquartile range. Plots <strong>e</strong>, <strong>g</strong> and <strong>h</strong> show the effect sizes and the 95% credible intervals of population trends (<strong>e</strong>, <strong>h</strong>) across realms and biomes, and fluctuations across realms (<strong>g</strong>). For variation in fluctuations across biomes, see SI Figure S9. The three estimates for each realm/system in plots <strong>e</strong> and <strong>h</strong> refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (<em>μ</em>, triangles), and population trends (<em>μ</em>) weighted by τ<sup>2</sup>, the observation error estimate from the state-space models (squares). The five estimates in plot <strong>g</strong> refer to different analytical approaches, where the response variables in the models were: 1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the <em>μ</em> value of population change (triangles), 3) half of the 95% confidence interval around <em>μ</em> weighted by τ<sup>2</sup>, (full squares), 4) the process noise (<em>σ<sup>2</sup></em>) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). Effect sizes were standardized by dividing the effect size by the standard deviation of the corresponding input data. The process noise is the total variance around the population trend minus the variance attributed to observation error. Error bars in plots <strong>e</strong>, <strong>g</strong> and <strong>h</strong> show 95% credible intervals. See Table S2 for model outputs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Taxonomic and phylogenetic patterns of population trends and fluctuations</h3><p id="p-13">We found taxonomic, but not phylogenetic patterns, in population trends and fluctuations over time among nearly 10,000 populations from over 2,000 vertebrate species, with amphibians emerging as the taxa experiencing pronounced declines (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Table S2). Amphibians experienced net declines over time (slope = -0.01, CI = -0.02 to -0.005), whereas birds, mammals and reptiles experienced net increases (slope = 0.004, CI = 0.003 to 0.01; slope = 0.01, CI = 0.01 to 0.01; slope = 0.02, CI = 0.01 to 0.02), with birds having a bimodal trend distribution (Hartigans’ dip test, D = 0.04, p &lt; 0.01, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1a</a>). Bony fish population trends were centered on zero (slope = -0.001, CI = -0.004 to 0.002, <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1a-b</a>) and sharks and rays showed net declines, but the credible intervals overlapped zero (slope = -0.01, CI = -0.02 to 0.01). Fluctuations were most common for amphibian populations (slope = 0.04, CI = 0.036 to 0.049, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2d</a>), which were monitored for the shortest time period on average (11 years, Figure S1, Table S2). We did not detect finer scale species-level phylogenetic clustering of population change (both trends and fluctuations) within amphibian, bird and reptile classes (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figures 2</a> and S16, Table S4). Similarly, species identity within amphibian, bird and reptile classes did not explain variation in population trends or fluctuations (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2</a> and S16, Table S4). There were no distinct clusters of specific clades that were more likely undergo increases, decreases or fluctuations in population abundance (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population trends and fluctuations varied more among, rather than within, taxa, with amphibians being the only group showing pronounced declines over time. There were no distinct phylogenetic patterns in population trends and fluctuations (plots e-j). For details on phylogenetic models, see methods. Grey color in the heatmap in plot h shows species for which no population trend data were available. The numbers in the legend for plots a-d show sample size for each taxon. The &#x3BC; values of population trend (plots a-b, e-g) and the &#x3C3;2 values of population fluctuation (plots c-d, h-j) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. Plots a and c show the density distribution of population trends across taxa, the raw values (points) and boxplots with the mean, first and third quartiles and boxplot whiskers that indicate the distance that covers 1.5 times the interquartile range. Plots b and d show the effect sizes and the 95% credible intervals of population trends (b) and fluctuations (d) across the five studied taxa. The three estimates for each metric in plot b refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (&#x3BC;, triangles), and population trends (&#x3BC;) weighted by &#x3C4;2, the observation error estimate from the state-space models (squares). The five estimates in plot d refer to different analytical approaches, where the response variables in the models were:1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the &#x3BC; value of population change (triangles), 3) half of the 95% confidence interval around &#x3BC; weighted by &#x3C4;2, (full squares), 4) the process noise (&#x3C3;2) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). Effect sizes (plots b and d) were standardized by dividing the effect size by the standard deviation of the corresponding input data. The process noise is the total variance around the population trend minus the variance attributed to observation error. Error bars in plots b and d show 95% credible intervals. See Tables S2 and S4 for model outputs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-375678661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Population trends and fluctuations varied more among, rather than within, taxa, with amphibians being the only group showing pronounced declines over time.&lt;/span&gt; There were no distinct phylogenetic patterns in population trends and fluctuations (plots &lt;strong&gt;e-j&lt;/strong&gt;). For details on phylogenetic models, see methods. Grey color in the heatmap in plot h shows species for which no population trend data were available. The numbers in the legend for plots &lt;strong&gt;a-d&lt;/strong&gt; show sample size for each taxon. The &lt;em&gt;&#x3BC;&lt;/em&gt; values of population trend (plots &lt;strong&gt;a-b&lt;/strong&gt;, &lt;strong&gt;e-g&lt;/strong&gt;) and the &lt;em&gt;&#x3C3;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; values of population fluctuation (plots &lt;strong&gt;c-d&lt;/strong&gt;, &lt;strong&gt;h-j&lt;/strong&gt;) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. Plots &lt;strong&gt;a&lt;/strong&gt; and &lt;strong&gt;c&lt;/strong&gt; show the density distribution of population trends across taxa, the raw values (points) and boxplots with the mean, first and third quartiles and boxplot whiskers that indicate the distance that covers 1.5 times the interquartile range. Plots &lt;strong&gt;b&lt;/strong&gt; and &lt;strong&gt;d&lt;/strong&gt; show the effect sizes and the 95% credible intervals of population trends (&lt;strong&gt;b&lt;/strong&gt;) and fluctuations (&lt;strong&gt;d&lt;/strong&gt;) across the five studied taxa. The three estimates for each metric in plot &lt;strong&gt;b&lt;/strong&gt; refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (&lt;em&gt;&#x3BC;&lt;/em&gt;, triangles), and population trends (&lt;em&gt;&#x3BC;&lt;/em&gt;) weighted by &#x3C4;&lt;sup&gt;2&lt;/sup&gt;, the observation error estimate from the state-space models (squares). The five estimates in plot &lt;strong&gt;d&lt;/strong&gt; refer to different analytical approaches, where the response variables in the models were:1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the &lt;em&gt;&#x3BC;&lt;/em&gt; value of population change (triangles), 3) half of the 95% confidence interval around &lt;em&gt;&#x3BC;&lt;/em&gt; weighted by &#x3C4;&lt;sup&gt;2&lt;/sup&gt;, (full squares), 4) the process noise (&lt;em&gt;&#x3C3;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). Effect sizes (plots &lt;strong&gt;b&lt;/strong&gt; and &lt;strong&gt;d&lt;/strong&gt;) were standardized by dividing the effect size by the standard deviation of the corresponding input data. The process noise is the total variance around the population trend minus the variance attributed to observation error. Error bars in plots &lt;strong&gt;b&lt;/strong&gt; and &lt;strong&gt;d&lt;/strong&gt; show 95% credible intervals. See Tables S2 and S4 for model outputs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F2.medium.gif" width="391" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F2.medium.gif" width="391" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Population trends and fluctuations varied more among, rather than within, taxa, with amphibians being the only group showing pronounced declines over time.</span><p id="p-14" class="first-child">There were no distinct phylogenetic patterns in population trends and fluctuations (plots <strong>e-j</strong>). For details on phylogenetic models, see methods. Grey color in the heatmap in plot h shows species for which no population trend data were available. The numbers in the legend for plots <strong>a-d</strong> show sample size for each taxon. The <em>μ</em> values of population trend (plots <strong>a-b</strong>, <strong>e-g</strong>) and the <em>σ<sup>2</sup></em> values of population fluctuation (plots <strong>c-d</strong>, <strong>h-j</strong>) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. Plots <strong>a</strong> and <strong>c</strong> show the density distribution of population trends across taxa, the raw values (points) and boxplots with the mean, first and third quartiles and boxplot whiskers that indicate the distance that covers 1.5 times the interquartile range. Plots <strong>b</strong> and <strong>d</strong> show the effect sizes and the 95% credible intervals of population trends (<strong>b</strong>) and fluctuations (<strong>d</strong>) across the five studied taxa. The three estimates for each metric in plot <strong>b</strong> refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (<em>μ</em>, triangles), and population trends (<em>μ</em>) weighted by τ<sup>2</sup>, the observation error estimate from the state-space models (squares). The five estimates in plot <strong>d</strong> refer to different analytical approaches, where the response variables in the models were:1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the <em>μ</em> value of population change (triangles), 3) half of the 95% confidence interval around <em>μ</em> weighted by τ<sup>2</sup>, (full squares), 4) the process noise (<em>σ<sup>2</sup></em>) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). Effect sizes (plots <strong>b</strong> and <strong>d</strong>) were standardized by dividing the effect size by the standard deviation of the corresponding input data. The process noise is the total variance around the population trend minus the variance attributed to observation error. Error bars in plots <strong>b</strong> and <strong>d</strong> show 95% credible intervals. See Tables S2 and S4 for model outputs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Population trends and fluctuations across rarity metrics, conservation status and threats</h3><p id="p-15">Species-level metrics, such as rarity and global conservation status, did not explain the heterogeneity in trends of monitored populations in the Living Planet Database, and both rare and common species experienced declines, increases and fluctuations in population abundance over time (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a>). Across monitored vertebrate populations globally, species with smaller ranges, smaller population sizes, or narrower habitat specificity (i.e., rare species) were not more prone to population declines than common species (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, Table S2). Populations that experienced more fluctuations had smaller mean population sizes on average (slope = -0.001, CI = -0.001 to -0.001, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3f</a>). We found increasing, decreasing and stable populations across all Red List categories of conservation status (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4a</a>). For example, a population of the Least concern species red deer (<em>Cervus elaphus</em>) in Canada declined by 68% over seven years going from 606 to 194 individuals and a population of the Critically-endangered Hawksbill sea turtle (<em>Eretmochelys imbricate</em>) from Barbados increased by 269% over seven years going from 89 to 328 individuals. We found more fluctuations (Least concern: slope = 0.022, CI = 0.021 to 0.023; Critically endangered: slope = 0.035, CI = 0.028 to 0.041), but not more population declines, with increasing conservation status (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, Table S2). Populations from species that were exposed to a larger number of threats on a global scale based on the species’ IUCN threat categorizations did not experience greater declines when compared to those categorized with a smaller number of threats (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4f</a>). There were no distinct signatures of global species threats which were associated with predominantly declining local trends of monitored populations (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4e</a>) and there were increasing, decreasing and stable trends across all threat types.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rarity metrics do not explain heterogeneity in local population trends, and both rare and common species experienced declines and increases over time, whereas smaller populations fluctuated more over time. Numbers on plots show sample size for each metric. Rarity metrics were calculated for all species for which information was available and cover all taxa represented in the Living Planet Database, with the exception of geographic range, which refers to the global range of only bird and mammal species in the global Living Planet Database (plots a-e). The &#x3BC; values of population trend (plots a-d) and the &#x3C3;2 values of population fluctuation (plots e-h) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. Plots d and h show the effect sizes and the 95% credible intervals of three rarity metrics on population trends (d) and fluctuations (h). The three estimates for each metric in plot d refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (&#x3BC;, triangles), and population trends (&#x3BC;) weighted by &#x3C4;2, the observation error estimate from the state-space models (squares). The five estimates in plot h refer to different analytical approaches, where the response variables in the models were: 1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the &#x3BC; value of population change (triangles), 3) half of the 95% confidence interval around &#x3BC; weighted by &#x3C4;2, (full squares), 4) the process noise (&#x3C3;2) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). Effect sizes (plots d and h) were standardized by dividing the effect size by the standard deviation of the corresponding input data. The process noise is the total variance around the population trend minus the variance attributed to observation error. Error bars in plots d and h show 95% credible intervals. Lines on a-c and e-g show model fits and 95% credible intervals. See Table S2 for model outputs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-375678661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Rarity metrics do not explain heterogeneity in local population trends, and both rare and common species experienced declines and increases over time, whereas smaller populations fluctuated more over time.&lt;/span&gt; Numbers on plots show sample size for each metric. Rarity metrics were calculated for all species for which information was available and cover all taxa represented in the Living Planet Database, with the exception of geographic range, which refers to the global range of only bird and mammal species in the global Living Planet Database (plots &lt;strong&gt;a-e&lt;/strong&gt;). The &lt;em&gt;&#x3BC;&lt;/em&gt; values of population trend (plots &lt;strong&gt;a-d&lt;/strong&gt;) and the &lt;em&gt;&#x3C3;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; values of population fluctuation (plots &lt;strong&gt;e-h&lt;/strong&gt;) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. Plots &lt;strong&gt;d&lt;/strong&gt; and &lt;strong&gt;h&lt;/strong&gt; show the effect sizes and the 95% credible intervals of three rarity metrics on population trends (&lt;strong&gt;d&lt;/strong&gt;) and fluctuations (&lt;strong&gt;h&lt;/strong&gt;). The three estimates for each metric in plot &lt;strong&gt;d&lt;/strong&gt; refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (&lt;em&gt;&#x3BC;&lt;/em&gt;, triangles), and population trends (&lt;em&gt;&#x3BC;&lt;/em&gt;) weighted by &#x3C4;&lt;sup&gt;2&lt;/sup&gt;, the observation error estimate from the state-space models (squares). The five estimates in plot &lt;strong&gt;h&lt;/strong&gt; refer to different analytical approaches, where the response variables in the models were: 1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the &lt;em&gt;&#x3BC;&lt;/em&gt; value of population change (triangles), 3) half of the 95% confidence interval around &lt;em&gt;&#x3BC;&lt;/em&gt; weighted by &#x3C4;&lt;sup&gt;2&lt;/sup&gt;, (full squares), 4) the process noise (&lt;em&gt;&#x3C3;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). Effect sizes (plots &lt;strong&gt;d&lt;/strong&gt; and &lt;strong&gt;h&lt;/strong&gt;) were standardized by dividing the effect size by the standard deviation of the corresponding input data. The process noise is the total variance around the population trend minus the variance attributed to observation error. Error bars in plots &lt;strong&gt;d&lt;/strong&gt; and &lt;strong&gt;h&lt;/strong&gt; show 95% credible intervals. Lines on &lt;strong&gt;a-c&lt;/strong&gt; and &lt;strong&gt;e-g&lt;/strong&gt; show model fits and 95% credible intervals. See Table S2 for model outputs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F3.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F3.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Rarity metrics do not explain heterogeneity in local population trends, and both rare and common species experienced declines and increases over time, whereas smaller populations fluctuated more over time.</span><p id="p-16" class="first-child">Numbers on plots show sample size for each metric. Rarity metrics were calculated for all species for which information was available and cover all taxa represented in the Living Planet Database, with the exception of geographic range, which refers to the global range of only bird and mammal species in the global Living Planet Database (plots <strong>a-e</strong>). The <em>μ</em> values of population trend (plots <strong>a-d</strong>) and the <em>σ<sup>2</sup></em> values of population fluctuation (plots <strong>e-h</strong>) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. Plots <strong>d</strong> and <strong>h</strong> show the effect sizes and the 95% credible intervals of three rarity metrics on population trends (<strong>d</strong>) and fluctuations (<strong>h</strong>). The three estimates for each metric in plot <strong>d</strong> refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (<em>μ</em>, triangles), and population trends (<em>μ</em>) weighted by τ<sup>2</sup>, the observation error estimate from the state-space models (squares). The five estimates in plot <strong>h</strong> refer to different analytical approaches, where the response variables in the models were: 1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the <em>μ</em> value of population change (triangles), 3) half of the 95% confidence interval around <em>μ</em> weighted by τ<sup>2</sup>, (full squares), 4) the process noise (<em>σ<sup>2</sup></em>) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). Effect sizes (plots <strong>d</strong> and <strong>h</strong>) were standardized by dividing the effect size by the standard deviation of the corresponding input data. The process noise is the total variance around the population trend minus the variance attributed to observation error. Error bars in plots <strong>d</strong> and <strong>h</strong> show 95% credible intervals. Lines on <strong>a-c</strong> and <strong>e-g</strong> show model fits and 95% credible intervals. See Table S2 for model outputs.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="On local scales, there are increasing, decreasing and stable populations across the full spectrum of the globally-determined species&#x2019; conservation status and anthropogenic threats (IUCN Red List Index). Numbers in the legend for plots a-d and in plots e-f show sample size for each metric. Plots a and c show the density distribution of population trends across Red List status, the raw values (points) and boxplots with the mean, first and third quartiles and boxplot whiskers that indicate the distance that covers 1.5 times the interquartile range. Plots b and d show the effect sizes and the 95% credible intervals of population trends (b) and fluctuations (d) across Red List status categories. The &#x3BC; values of population trend (plots a, e-f) and the &#x3C3;2 values of population fluctuation (plots c) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. For the relationships between type and number of threats and population fluctuations, see Figure S19. Plots b and d show the effect sizes and the 95% credible intervals for population trends (b) and fluctuations (d) across conservation status across the IUCN Red List categories. The three estimates for each metric in plot b refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (&#x3BC;, triangles), and population trends (&#x3BC;) weighted by &#x3C4;2, the observation error estimate from the state-space models (squares). The five estimates in plot d refer to different analytical approaches, where the response variables in the models were: 1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the &#x3BC; value of population change (triangles), 3) half of the 95% confidence interval around &#x3BC; weighted by &#x3C4;2, (full squares), 4) the process noise (&#x3C3;2) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). The process noise is the total variance around the population trend minus the variance attributed to observation error. Effect sizes (plots b and d) were standardized by dividing the effect size by the standard deviation of the corresponding input data. Error bars in plots b and d show 95% credible intervals. Plot e shows the distributions of population trends across different threats that the species face globally, with the central tendencies of all distributions overlapping with zero. Lines in plot f show model fit and 95% credible intervals, where &#x201C;number of threats&#x201D; refers to the number of different threats that each species, whose populations are locally monitored, are exposed to on a global scale. See Methods for details on deriving types of threats and Table S2 for model outputs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-375678661" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;On local scales, there are increasing, decreasing and stable populations across the full spectrum of the globally-determined species&#x2019; conservation status and anthropogenic threats (IUCN Red List Index).&lt;/span&gt; Numbers in the legend for plots &lt;strong&gt;a-d&lt;/strong&gt; and in plots &lt;strong&gt;e-f&lt;/strong&gt; show sample size for each metric. Plots &lt;strong&gt;a&lt;/strong&gt; and &lt;strong&gt;c&lt;/strong&gt; show the density distribution of population trends across Red List status, the raw values (points) and boxplots with the mean, first and third quartiles and boxplot whiskers that indicate the distance that covers 1.5 times the interquartile range. Plots &lt;strong&gt;b&lt;/strong&gt; and &lt;strong&gt;d&lt;/strong&gt; show the effect sizes and the 95% credible intervals of population trends (&lt;strong&gt;b&lt;/strong&gt;) and fluctuations (&lt;strong&gt;d&lt;/strong&gt;) across Red List status categories. The &lt;em&gt;&#x3BC;&lt;/em&gt; values of population trend (plots &lt;strong&gt;a&lt;/strong&gt;, &lt;strong&gt;e-f&lt;/strong&gt;) and the &lt;em&gt;&#x3C3;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; values of population fluctuation (plots &lt;strong&gt;c&lt;/strong&gt;) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. For the relationships between type and number of threats and population fluctuations, see Figure S19. Plots &lt;strong&gt;b&lt;/strong&gt; and &lt;strong&gt;d&lt;/strong&gt; show the effect sizes and the 95% credible intervals for population trends (&lt;strong&gt;b&lt;/strong&gt;) and fluctuations (&lt;strong&gt;d&lt;/strong&gt;) across conservation status across the IUCN Red List categories. The three estimates for each metric in plot &lt;strong&gt;b&lt;/strong&gt; refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (&lt;em&gt;&#x3BC;&lt;/em&gt;, triangles), and population trends (&lt;em&gt;&#x3BC;&lt;/em&gt;) weighted by &#x3C4;&lt;sup&gt;2&lt;/sup&gt;, the observation error estimate from the state-space models (squares). The five estimates in plot &lt;strong&gt;d&lt;/strong&gt; refer to different analytical approaches, where the response variables in the models were: 1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the &lt;em&gt;&#x3BC;&lt;/em&gt; value of population change (triangles), 3) half of the 95% confidence interval around &lt;em&gt;&#x3BC;&lt;/em&gt; weighted by &#x3C4;&lt;sup&gt;2&lt;/sup&gt;, (full squares), 4) the process noise (&lt;em&gt;&#x3C3;&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). The process noise is the total variance around the population trend minus the variance attributed to observation error. Effect sizes (plots b and d) were standardized by dividing the effect size by the standard deviation of the corresponding input data. Error bars in plots &lt;strong&gt;b&lt;/strong&gt; and &lt;strong&gt;d&lt;/strong&gt; show 95% credible intervals. Plot &lt;strong&gt;e&lt;/strong&gt; shows the distributions of population trends across different threats that the species face globally, with the central tendencies of all distributions overlapping with zero. Lines in plot f show model fit and 95% credible intervals, where &#x201C;number of threats&#x201D; refers to the number of different threats that each species, whose populations are locally monitored, are exposed to on &lt;strong&gt;a&lt;/strong&gt; global scale. See Methods for details on deriving types of threats and Table S2 for model outputs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F4.medium.gif" width="440" height="258"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F4.medium.gif" width="440" height="258"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/27/272898/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">On local scales, there are increasing, decreasing and stable populations across the full spectrum of the globally-determined species’ conservation status and anthropogenic threats (IUCN Red List Index).</span><p id="p-17" class="first-child">Numbers in the legend for plots <strong>a-d</strong> and in plots <strong>e-f</strong> show sample size for each metric. Plots <strong>a</strong> and <strong>c</strong> show the density distribution of population trends across Red List status, the raw values (points) and boxplots with the mean, first and third quartiles and boxplot whiskers that indicate the distance that covers 1.5 times the interquartile range. Plots <strong>b</strong> and <strong>d</strong> show the effect sizes and the 95% credible intervals of population trends (<strong>b</strong>) and fluctuations (<strong>d</strong>) across Red List status categories. The <em>μ</em> values of population trend (plots <strong>a</strong>, <strong>e-f</strong>) and the <em>σ<sup>2</sup></em> values of population fluctuation (plots <strong>c</strong>) were derived from state-space model fits of changes in abundance over the monitoring duration for each population. For the relationships between type and number of threats and population fluctuations, see Figure S19. Plots <strong>b</strong> and <strong>d</strong> show the effect sizes and the 95% credible intervals for population trends (<strong>b</strong>) and fluctuations (<strong>d</strong>) across conservation status across the IUCN Red List categories. The three estimates for each metric in plot <strong>b</strong> refer to different analytical approaches: population trends calculated using linear models (circles), state-space models (<em>μ</em>, triangles), and population trends (<em>μ</em>) weighted by τ<sup>2</sup>, the observation error estimate from the state-space models (squares). The five estimates in plot <strong>d</strong> refer to different analytical approaches, where the response variables in the models were: 1) the standard error around the slope estimates of the linear model fits of abundance versus year (circles), 2) half of the 95% confidence interval around the <em>μ</em> value of population change (triangles), 3) half of the 95% confidence interval around <em>μ</em> weighted by τ<sup>2</sup>, (full squares), 4) the process noise (<em>σ<sup>2</sup></em>) from the state-space models, and 5) the standard deviation of the raw data for each population time-series (empty squares). The process noise is the total variance around the population trend minus the variance attributed to observation error. Effect sizes (plots b and d) were standardized by dividing the effect size by the standard deviation of the corresponding input data. Error bars in plots <strong>b</strong> and <strong>d</strong> show 95% credible intervals. Plot <strong>e</strong> shows the distributions of population trends across different threats that the species face globally, with the central tendencies of all distributions overlapping with zero. Lines in plot f show model fit and 95% credible intervals, where “number of threats” refers to the number of different threats that each species, whose populations are locally monitored, are exposed to on <strong>a</strong> global scale. See Methods for details on deriving types of threats and Table S2 for model outputs.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-6"><h2 class="">Discussion</h2><p id="p-18">Taken together, our analysis of the Living Planet Database using a state-space modelling approach demonstrated ubiquitous alterations in vertebrate abundance of monitored populations over time across all biomes on Earth. We revealed that population change includes both increasing and decreasing populations and spans a wide spectrum of magnitudes, and while anthropogenic impacts have accelerated in recent decades, our results highlight that all is not declines across vertebrate species. Against a backdrop of no biogeographic patterning of population trends and fluctuations (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>), we uncovered distinct taxonomic signals, with amphibians representing the only taxa that exhibited pronounced net declines, while birds, mammals and reptiles on average became more abundant over time (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). Within amphibian, bird and reptile taxa, there was no distinct phylogenetic clustering of closely-related species experiencing similar population trends or fluctuations (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). We found that both rare and common species experienced the full spectrum of population change, from declines to no net changes and increases. Species’ geographic range, mean population size and habitat specificity did not explain variation in population trends, but species with smaller population sizes were nevertheless more likely to fluctuate, potentially increasing their extinction risk (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). There was no consistent pattern of greater population declines with increasing conservation status (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>). On a global scale, the vertebrate species in the Living Planet Database are exposed to a variety of threats, but on more local scales, none of the threats were characterized by predominantly declining populations (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>), testifying to the diverse ways in which populations are changing across the Anthropocene.</p><p id="p-19">Contrary to our initial predictions, we did not find a distinct geographic patterning of population change around the world, nor a consistent trend of increasing declines in population abundance with increasing threat level (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-4-8" class="xref-fig" href="#F4">4</a>). Similar lack of biogeographic signal has been documented in regional studies of population change from the Netherlands<sup><a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a></sup> and in temperate North America and Europe<sup><a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a></sup>. Coarsely represented biogeographic regions and global-scale species’ IUCN status and threat categories might not capture the drivers acting in the locations of the specific populations we studied<sup><a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>–<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a></sup>. Furthermore, the same driver can have opposing effects on population abundance at different sites<sup><a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a></sup>. A lack of of biome-specific directional trends in population change, despite a spatial clustering of human pressure around the world<sup><a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a></sup>, can also arise due to differences in species traits and vulnerability to environmental change within biomes<sup><a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a>–<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a></sup>. Accounting for divergent responses of species to global change is key when translating global policy, such as the upcoming post-2020 planetary biodiversity strategy<sup><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a></sup>, into conservation actions implemented on scales much finer than biogeographic realms.</p><p id="p-20">Our results highlight variation in population change among taxa, with amphibians emerging as the taxa experiencing the most pronounced declines in the Living Planet Database. The remaining taxa showed either stable or increasing net changes in abundance over time (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). Such taxonomic patterns could be driven by different taxon-specific factors including reproductive strategy, trophic level, generation time and life history traits<sup><a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a></sup>. For amphibians, population declines have been linked to the spread of a fungal disease (chytrid fungus, <em>Batrachochytrium dendrobatidis</em>), facilitated by global warming<sup><a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a></sup>, as well as habitat loss and Allee effects in small populations<sup><a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a></sup>. Within bird, amphibian and reptile taxa, phylogenetic relatedness and species-level taxonomic classification did not explain variation in population trends and fluctuations. A similar lack of phylogenetic dependencies has been detected for the population growth rates of migratory birds<sup><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a></sup>. While phylogenetic clustering might be lacking in contemporary trends, there is evidence that phylogenetic relatedness predicts extinction, a process occurring over much longer time scales<sup><a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a></sup>. Over shorter time periods, species’ traits and ability to persist, reproduce and disperse in ever changing landscapes might be influencing local abundance<sup><a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">16</a></sup>, which has created a mix of winners and losers across all taxa<sup><a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a></sup>. We demonstrate ongoing alterations in the abundances of six vertebrate taxa which over time, may lead to shifts in community composition and ultimately alter ecosystem function as some species become locally extinct whilst others become more abundant<sup><a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">9</a>, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a></sup>.</p><p id="p-21">Surprisingly, our results indicate that despite decades of conservation focus on rare species<sup><a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>–<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">21</a></sup>, both rare and common species experienced declines and increases in population abundance over the period of monitoring. The lack of rarity effects on population trends can be explained by theory and empirical evidence demonstrating that small populations do not necessarily have a higher likelihood of experiencing declines and some species are able to persist in small, but stable populations<sup><a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a></sup>. The power of rarity metrics to predict population trends could also be mediated by whether species are naturally rare, or have become rare due to external drivers in recent years<sup><a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a></sup>. Naturally rare species might be more likely to persist over time, whereas species that have more recently become rare might be more likely to decline in response to environmental disturbance. Furthermore, the timing and magnitude of past and current disturbance events influence population trends<sup><a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a></sup> and there could be temporal lags in both positive and negative abundance changes over time<sup><a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a></sup>. However, disentangling the processes leading to rarity over time remains challenging, and across the 2084 species we studied, there are likely cases of both natural and human-driven population change. We found that species with small populations were, nevertheless, more likely to fluctuate (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3f</a>), which may increase their probability of extinction, a process that could play out over longer time-scales than found for most population monitoring time-series<sup><a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a></sup>. Our results highlight that rarity metrics alone do not capture the heterogeneity in local population change over time, and common species should not be overlooked in conservation prioritization decisions as they could be as likely to decrease in abundance over time as rare species.</p><p id="p-22">Our finding that all is not declines for vertebrate populations monitored for longer than five years contrasts with reports of the overall decline in the Living Planet Index (LPI)<sup><a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a></sup>, a weighted summary of population change across all abundance time-series in the Living Planet Database. The LPI reports<sup><a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">55</a>–<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a></sup> also document that the numbers of declining and increasing species are similar across this dataset but that the magnitude of declines is larger than the magnitude of increases. The calculation of the index used involves differential weighting of population trends derived using logged abundance data, geometric means and generalized additive models<sup><a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a></sup>. The LPI is hierarchically averaged from populations to species, taxa and realm and is also weighted by the estimated and relative number of species within biomes, which influences the direction and magnitude of the index<sup><a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">58</a>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a></sup>. Our analysis explores the heterogeneity in local trends and fluctuations of monitored species from the raw population abundance data, and thus, we did not use an index with weightings and we did not aggregate population trends to a species-level. We detected net population declines at local scales over time only in the amphibian taxa, in contrast with the overall negative trend of the aggregate weightings of the LPI<sup><a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">55</a></sup>. Our results were similar when we weighted population trends by the state-space model derived observation error (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figures 1</a>-<a id="xref-fig-4-9" class="xref-fig" href="#F4">4</a> and Tables S2-3). We caution that distilling the heterogeneity of local population change at sites around the world into a simple metric may hide diverging trends at local scales, where we found both increases and declines among species.</p><p id="p-23">To attribute population trends and fluctuations to site-specific anthropogenic drivers, we need to go beyond previous studies that have focused exclusively on declines and extinctions<sup><a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a></sup>. Attribution analyses statistically test the links between observed changes in ecosystems and the experienced extrinsic pressure<sup><a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">3</a></sup>. Through attribution studies encompassing the full spectrum of population change, including positive, negative and stable trends<sup><a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a></sup>, we can better understand the variety of ways in which climate change, land-use change and other drivers are altering biodiversity. For a subset of the bird populations in the Living Planet Database, higher temperature warming corresponded with a higher likelihood of population declines over time<sup><a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">61</a></sup>, which could be caused by worldwide and cross-biome phenological mismatches between breeding and resource availability<sup><a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a></sup>. Across terrestrial species represented in the Living Planet Database, peak forest loss was associated with accelerations in both population increases and decreases in the period following habitat alteration<sup><a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">41</a></sup>. The strength of documented relationships between population alterations and global change could be influenced by how well-monitored populations capture the full range of variation in driver intensity as well as whether monitoring began during a population peak or a population trough<sup><a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a></sup>. Additionally, there is evidence from the marine realm that when species are simultaneously exposed to multiple drivers, the resulting biodiversity effects are antagonistic and could produce patterns of no net biodiversity changes<sup><a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a></sup>. The next critical step is to test how multiple global change drivers together<sup><a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a></sup> influence populations across both terrestrial and marine realms and determine how these relationships are mediated by species’ traits and vulnerability to extrinsic threats<sup><a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a></sup>.</p><p id="p-24">In summary, our global analysis reveals the ubiquitous nature of population change over time across monitored vertebrate species. We show that in a time of accelerating global change, there were as many increases as there are decreases in population abundance over time. Among this heterogeneity, we uncover a pronounced net decline in amphibian abundance as well as net abundance increases for birds, mammals and reptiles. The taxonomic patterning of population change highlights amphibians as a conservation priority, especially as their declines can have further cascading effects across trophic levels within ecosystems. Rarity metrics, specifically geographic range, mean population size and habitat specificity, as well as conservation status and evolutionary history, did not explain the heterogeneity in population change across the data analyzed in this study. Our findings caution the use of rarity metrics for the prediction of future global population trends, but suggest that such metrics, in particular mean population size, are nevertheless indicators of population fluctuations, which might ultimately link to increased species extinction risk. On a global scale, both rare and common vertebrate species face numerous threats due to resource exploitation and habitat change. As human activities continue to accelerate, the next key step is to determine how intrinsic factors, such as rarity attributes and threats, interact with extrinsic global change drivers and together influence the persistence of Earth’s biota. The ongoing alterations in the abundances of species around the world may lead to shifts in community composition and ultimately alter ecosystem functions and services.</p></div><div class="section" id="sec-7"><h2 class="">Methods</h2><p id="p-25">All data syntheses, visualization and statistical analyses were conducted using R version 3.5.1<sup><a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a></sup>. For conceptual diagrams of our workflow, see Figures S1 and S2.</p><div id="sec-8" class="subsection"><h3>Data sources</h3><div id="sec-9" class="subsection"><h4>Population data</h4><p id="p-26">To quantify vertebrate population change (trends and fluctuations), we extracted the abundance data for 9284 population time-series from 2084 species from the publicly available Living Planet Database<sup><a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a></sup> (<a href="http://www.livingplanetindex.org/data_portal">http://www.livingplanetindex.org/data_portal</a>) that covered the period between 1970 and 2014 (Table S1). These time-series represent repeated monitoring surveys of the number of individuals in a given area, hereafter called “populations”. Monitoring duration differed among populations (Figure S1, see SI Figures S6 and S7 for effects of monitoring duration on detected trends). In our analysis, we retained only populations that had at least five time points of monitoring data, as that is a considered a long enough period to detect a directional population trend (if one exists) in 80-90% of cases<sup><a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a></sup>. Populations were monitored using different metrics of abundance (e.g., population indices vs. number of individuals). Before analysis, we scaled the abundance of each population to a common magnitude between zero and one to analyze within-population relationships to prevent conflating within-population relationships and between-population relationships<sup><a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a></sup>. This allowed us to explore trends among populations relative to the variation experienced across each time series.</p></div><div id="sec-10" class="subsection"><h4>Phylogenetic data</h4><p id="p-27">We obtained phylogenies for amphibian species from <a href="https://vertlife.org">https://vertlife.org</a><sup><a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">4</a></sup>, for bird species from <a href="https://birdtree.org">https://birdtree.org</a><sup><a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a></sup>, and for reptile species from <a href="https://vertlife.org">https://vertlife.org</a><sup><a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">6</a></sup>. For each of the three classes (<em>Amphibia</em>, <em>Aves</em> and <em>Reptilia</em>), we downloaded 100 trees and randomly chose 10 for analysis (30 trees in total). Species-level phylogenies for the classes <em>Actinopterygii</em> and <em>Mammalia</em> have not yet been resolved with high confidence<sup><a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">71</a></sup>.</p></div><div id="sec-11" class="subsection"><h4>Rarity metrics, conservation status and threat data</h4><p id="p-28">We defined rarity following a simplified version of the ‘seven forms of rarity’ model<sup><a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">72</a></sup>, and thus consider rarity to be the state in which species exist when they have a small geographic range, low population size, or narrow habitat specificity. We combined publicly available data from three sources: 1) population records for vertebrate species from the Living Planet Database to calculate mean population size, 2) occurrence data from the Global Biodiversity Information Facility<sup><a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a></sup> (<a href="https://www.gbif.org">https://www.gbif.org</a>) and range data from BirdLife<sup><a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a></sup> (<a href="http://datazone.birdlife.org">http://datazone.birdlife.org</a>) to estimate geographic range size, and 3) habitat specificity and Red List classification data for each species from the International Union for Conservation<sup><a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a></sup> (<a href="https://www.iucnredlist.org">https://www.iucnredlist.org</a>). The populations in the Living Planet Database<sup><a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">67</a></sup> do not include species that have gone extinct on a global scale, and we use species’ Red List conservation status as a proxy for extinction risk<sup><a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">75</a></sup>, an approach used recently in other studies<sup><a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a></sup>. We extracted the number and types of threats that each species is exposed to globally from their respective IUCN profiles<sup><a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">75</a></sup>.</p></div></div><div id="sec-12" class="subsection"><h3>Quantifying population trends and fluctuations over time</h3><p id="p-29">In the first stage of our analysis, we used state-space models that model abundance (scaled to a common magnitude between zero and one) over time to calculate the amount of overall abundance change experienced by each population (<em>μ</em>, <sup><a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>, <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a></sup>). State-space models account for process noise (<em>σ<sup>2</sup>)</em> and observation error (τ<sup>2</sup>) and thus deliver robust estimates of population change when working with large datasets where records were collected using different approaches, such as the Living Planet Database<sup><a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>, <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a></sup>. Previous studies have found that not accounting for process noise and measurement error could lead to over-estimation of population declines<sup><a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a></sup>, but in our analyses, we found that population trends derived from state-space models were similar to those derived from linear models. Positive <em>μ</em> values indicate population increase and negative <em>μ</em> values indicate population decline. State-space models partition the variance in abundance estimates into estimated process noise (<em>σ<sup>2</sup></em>) and observation or measurement error (τ<sup>2</sup>) and population trends (<em>μ</em>):
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-5.gif"/></noscript></span>

</span>
where <em>X<sub>t</sub></em> and <em>X<sub>t-1</sub></em> are the scaled (observed) abundance estimates (between 0 and 1) in the present and past year, with process noise represented by <em>εt ∼ gaussian(0, σ<sup>2</sup>)</em>. We included measurement error following:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-6.gif"/></noscript></span>

</span>
where <em>Yt</em> is the estimate of the true (unobserved) population abundance with measurement error:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-7.gif"/></noscript></span>

</span>
We substituted the estimate of population abundance (<em>Y<sub>t</sub></em>) into <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equation 1</a>:
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-8.gif"/></noscript></span>

</span>
Given <em>X</em><sub><em>t</em> − 1</sub> = <em>Y</em><sub><em>t</em> − 1</sub> − <em>F</em><sub><em>t</em> − 1</sub>, then:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-9.gif"/></noscript></span>

</span>
For comparisons of different approaches to modelling population change, see <em>Comparison of modelling approaches</em> section.</p></div><div id="sec-13" class="subsection"><h3>Quantifying rarity metrics</h3><p id="p-30">We tested how population change varied across rarity metrics – geographic range, mean population size and habitat specificity – on two different but complementary scales. In the main text, we presented the results of our global scale analyses, whereas in the SI we included the results when using only populations from the UK – a country with high monitoring intensity, Thus, we quantified rarity metrics for species monitoring globally and in the UK.</p><div id="sec-14" class="subsection"><h4>Geographic range</h4><p id="p-31">To estimate geographic range for bird species monitored globally, we extracted the area of occurrence in km<sup>2</sup> for all bird species in the Living Planet Database that had records in the BirdLife Data Zone<sup><a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">74</a></sup>. For mammal species’ geographic range, we used the PanTHERIA database<sup><a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a></sup> (<a href="http://esapubs.org/archive/ecol/E090/184/">http://esapubs.org/archive/ecol/E090/184/</a>). To estimate geographic range for bony fish, birds, amphibians, mammals and reptiles monitored in the UK (see Table S5 for species list), we calculated a km<sup>2</sup> occurrence area based on species occurrence data from GBIF<sup><a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">73</a></sup>. Extracting and filtering GBIF data and calculating range was computationally-intensive and occurrence data availability was lower for certain species; thus, we did not estimate geographic range from GBIF data for all species part of the Living Planet Database. Instead, we focused on analyzing range effects for birds and mammals globally, as they are a very well-studied taxon and for species monitored in the UK, a country with intensive and detailed biodiversity monitoring of vertebrate species. We did not use IUCN range maps, as they were not available for all of our study species, and previous studies using GBIF occurrences to estimate range have found a positive correlation between GBIF-derived and IUCN-derived geographic ranges<sup><a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a></sup>.</p><p id="p-32">For the geographic ranges of species monitored in the UK, we calculated range extent using a minimal convex hull approach based on GBIF<sup><a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">73</a></sup> occurrence data. We filtered the GBIF data to remove invalid records and outliers using the <em>CoordinateCleaner</em> package<sup><a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a></sup>. We excluded records with no decimal places in the decimal latitude or longitude values, with equal latitude or longitude, within a one-degree radius of the GBIF headquarters in Copenhagen, within 0.0001 degrees of various biodiversity institutions and within 0.1 degrees of capital cities. For each species, we excluded the lower 0.02 and upper 0.98 quantile intervals of the latitude and longitude records to account for outlier points that are records from zoos or other non-wild populations. We drew a convex hull to most parsimoniously encompass all remaining occurrence records using the <em>chull</em> function, and we calculated the area of the resulting polygon using <em>areaPolygon</em> from the <em>geosphere</em> package.</p></div><div id="sec-15" class="subsection"><h4>Mean size of monitored populations</h4><p id="p-33">We calculated mean size of the monitored population, referred to as population size, across the monitoring duration using the raw abundance data, and we excluded populations which were not monitored using population counts (i.e., we excluded indexes).</p></div><div id="sec-16" class="subsection"><h4>Habitat specificity</h4><p id="p-34">To create an index of habitat specificity, we extracted the number of distinct habitats a species occupies based on the IUCN habitat category for each species, accessed through the package <em>rredlist</em><sup><a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a></sup>. We also quantified habitat specificity by surveying the number of breeding and non-breeding habitats for each species from its online IUCN profile (the ‘habitat and ecology’ section). The two approaches yielded similar results (Figure S3, Table S2, key for the profiling method is presented in Table S4). We obtained global conservation status and threat data for all study species through the IUCN Red List classification<sup><a id="xref-ref-75-4" class="xref-bibr" href="#ref-75">75</a></sup>.</p></div></div><div id="sec-17" class="subsection"><h3>Testing the sources of variation in population trends and fluctuations</h3><p id="p-35">In the second stage of our analyses, we modelled the trend and fluctuation estimates from the first stage across latitude, realm, biome, taxa, rarity metrics, phylogenetic relatedness, species’ conservation status and threat type using a Bayesian modelling framework through the package <em>MCMCglmm</em><sup><a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a></sup>. We included a species random intercept effect in the Bayesian models to account for the possible correlation between the trends of populations from the same species (see Table S1 for sample sizes). The models ran for 120 000 iterations with a thinning factor of ten and a burn-in period of 20 000 iterations. We assessed model convergence by visually examining trace plots. We used weakly informative priors for all coefficients (an inverse Wishart prior for the variances and a normal prior for the fixed effects):
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-10.gif"/></noscript></span>

</span>
</p><div id="sec-18" class="subsection"><h4>Population trends and fluctuations across latitude, biomes, realms and taxa</h4><p id="p-36">To investigate the geographic and taxonomic patterns of population trends and fluctuations, we modelled population trends (<em>μ</em>) and population fluctuations (<em>σ<sup>2</sup></em>), derived from the first stage of our analyses (state-space models), as a function of 1) latitude, 2) realm (freshwater, marine, terrestrial), 3) biome (as defined by the ‘biome’ category in the Living Planet Database, e.g., ‘temperate broadleaf forest’<sup><a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a></sup> and 4) taxa (<em>Actinopterygii</em>, bony fish; <em>Elasmobranchii</em>, sharks and rays; <em>Amphibia</em>, amphibians; <em>Aves</em>, birds; <em>Mammalia</em>, mammals; <em>Reptilia</em>, reptiles). We used separate models for each variable, resulting in four models testing the sources of variation in trends and four additional models focusing on fluctuations. Each categorical model from this second stage of our analyses was fitted with a zero intercept to allow us to determine if net population trends differed from zero for each of the categories under investigation. The model structures for all models with a categorical fixed effect were identical with the exception of the identity of the fixed effect, and below we describe the taxa model:
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-11.gif"/></noscript></span>

</span>
where <em>taxa<sub>i,j,k</sub></em> is the taxa of the <em>ith</em> time-series from the <em>jth</em> species; <em>β<sub>0</sub></em> and <em>β<sub>1</sub></em> are the global intercept (in categorical models, <em>β<sub>0</sub></em> = 1) and the slope estimate for the categorical taxa effect (fixed effect), <em>β<sub>0j</sub></em> is the species-level departure from <em>β<sub>0</sub></em> (species-level random effect); <em>yi,j,k</em> is the estimate for change in population abundance for the <em>ith</em> population time-series from the <em>jth</em> species from the <em>kth</em> taxa.</p></div><div id="sec-19" class="subsection"><h4>Population trends and fluctuations across amphibian, bird and reptile phylogenies</h4><p id="p-37">To determine if there is a phylogenetic signal in the patterning of population change within amphibian, bird and reptile taxa, we modelled population trends (<em>μ</em>) and fluctuations (<em>σ<sup>2</sup></em>) across phylogenetic and species-level taxonomic relatedness. We conducted one model per taxa per population change variable – trends or fluctuations using Bayesian linear mixed effects models using the package <em>MCMCglmm</em><sup><a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">86</a></sup>. We included phylogeny and taxa as random effects. The models did not include fixed effects. We assessed the magnitude of the random effects (phylogeny and species) by inspecting their posterior distributions, with a distribution pushed up against zero indicating lack of effect, since these distributions are always bounded by zero and have only positive values. We used parameter-expanded priors, with a variance-covariance structure that allows the slopes of population trend (the <em>mu</em> values from the first stage analysis using state-space models) to covary for each random effect. The prior and model structure were as follows:
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-12.gif"/></noscript></span>

</span>
where <em>β<sub>0</sub></em> is the global intercept (<em>β<sub>0</sub></em> = 1), <em>β<sub>0l</sub></em> is the phylogeny-level departure from <em>β<sub>0</sub></em> (phylogeny random effect); <em>yi,k,m</em> is the estimate for change in population abundance for the <em>ith</em> population time-series for the <em>kth</em> species with the <em>mth</em> phylogenetic distance.</p><p id="p-38">To account for phylogenetic uncertainty, for each class, we ran ten models with identical structures but based on different randomly selected phylogenetic trees. We report the mean estimates and their range for each class.</p></div><div id="sec-20" class="subsection"><h4>Population trends and fluctuations across rarity metrics</h4><p id="p-39">To test the influence of rarity metrics (geographic range, mean population size and habitat specificity) on variation in population trends and fluctuations, we modelled population trends (<em>μ</em>) and fluctuations (<em>σ<sup>2</sup></em>) across all rarity metrics. We conducted one Bayesian linear models per rarity metric per model per scale (for both global and UK analyses) per population change variable – trends or fluctuations. The response variable was population trend (<em>μ</em> values from state-space models) or population fluctuation (<em>σ<sup>2</sup> values from state-space models</em>), and the fixed effects were geographic range (log transformed), mean population size (log transformed) and habitat specificity (number of distinct habitats occupied). The model structures were identical across the different rarity metrics and below we outline the equations for population trends and geographic range:
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/27/272898/embed/graphic-13.gif"/></noscript></span>

</span>
where <em>geographic range<sub>i,k,n</sub></em> is the logged geographic range of the <em>kth</em> species in the <em>ith</em> time-series; <em>β<sub>0</sub></em> and <em>β<sub>1</sub></em> are the global intercept and slope estimate for the geographic range effect (fixed effect), <em>β<sub>0j</sub></em> is the species-level departure from <em>β<sub>0</sub></em> (species-level random effect); <em>yi,k,n</em> is the estimate for change in population abundance for the <em>ith</em> population time-series from the <em>jth</em> species with the <em>nth</em> geographic range.</p></div><div id="sec-21" class="subsection"><h4>Population trends across species’ IUCN conservation status</h4><p id="p-40">To investigate the relationship between population change and species’ Red List status, we modelled population trends (<em>μ</em>) and fluctuations (<em>σ<sup>2</sup></em>) as a function of Red List status (categorical variable). We conducted one Bayesian linear model per population change metric per scale (for both global and UK analyses). To test variation in population trends and fluctuations across the types and number of threats to which species are exposed, we conducted a post-hoc analysis of trends and fluctuations across threat type (categorical effect, determined by species IUCN threat categorization) and number of threats that each species is exposed to across its range (in separate models). The model structures were identical to those presented above, except for the fixed effect which was a categorical IUCN conservation status variable. The analytical workflow of our analyses is summarized in conceptual diagrams (Figures S1 and S2) and all code is available on GitHub (link to public repository to be added at time of publication).</p></div></div><div id="sec-22" class="subsection"><h3>Data limitations</h3><div id="sec-23" class="subsection"><h4>Taxonomic and geographic gaps</h4><p id="p-41">Our analysis is based on 9286 monitored populations from 2084 species from the largest currently available public database of population time-series, the Living Planet Database<sup><a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">67</a></sup>. Nevertheless, the data are characterized by both taxonomic and geographic gaps that can influence our findings. For example, there are very few population records from the Amazon and Siberia (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1b</a>) – two regions currently undergoing rapid environmental changes due to land-use change and climate change respectively. Additionally, birds represent 63% of all population time-series in the Living Planet Database, whilst taxa such as amphibians and sharks where we find declines are included with fewer records (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figures 2</a> and S5). On a larger scale, the Living Planet Database under-represents populations outside of Europe and North America and over-represents common and well-studied species<sup><a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">59</a></sup>. We found that for the populations and species represented by current monitoring, rarity does not explain variation in population trends, but we note that the relationship between population change and rarity metrics could differ for highly endemic specialist species that are not included in the Living Planet Database. As ongoing and future monitoring begins to fill in the taxonomic and geographic gaps in existing datasets, we will be able to re-assess and test the generality of the patterns of population change across biomes, taxa, phylogenies, species traits and threats.</p></div><div id="sec-24" class="subsection"><h4>Monitoring extent and survey techniques</h4><p id="p-42">The Living Planet Database combines population time-series where survey methods were consistent within time-series but varied among time-series. Thus, among populations, abundance was measured using different units and over varying spatial extents. There are no estimates of error around the raw population abundance values available and detection probability likely varies among species. We used state-space models to estimate trends and fluctuations to account for these limitations as this modelling framework is particularly appropriate for analyses of data collected using disparate methods<sup><a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">78</a>, <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">79</a></sup>. Because the precise coordinates of the polygons where the individual populations were monitored are not available, we were not able to test for the potential confounding effect of monitoring extent, but our sensitivity analysis indicated that survey units do not explain variation in the detected trends (Figure S13).</p></div><div id="sec-25" class="subsection"><h4>Temporal gaps</h4><p id="p-43">The population time-series we studied cover the period between 1970 and 2014, with both duration of monitoring and the frequency of surveys varying across time-series. We only included populations with more than five survey points in time in our analyses, because this amount of data has been found to be enough to detect directional trends in 80-90% of cases<sup><a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">68</a></sup>. In a separate analysis, we found significant lags in population change following disturbances (forest loss) and that population monitoring often begins decades to centuries after peak forest loss has occurred at a given site<sup><a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">41</a></sup>. The findings of this related study suggest that the temporal span of the population monitoring does not always capture the period of intense environmental change and lags suggest that there might be abundance changes that have not yet manifested themselves. Thus, the detected trends and the baseline across which trends are compared might be influenced by when monitoring takes place and at what temporal frequency. Nevertheless, the Living Planet Database represents the most comprehensive compilation of temporal population records to date, allowing for the most comprehensive analyses possible into the patterns of vertebrate trends and fluctuations around the world.</p></div><div id="sec-26" class="subsection"><h4>Challenges in estimating geographic range</h4><p id="p-44">Estimating geographic range across taxa, specifically species that are not birds or mammals, remains challenging due to data limitations. We used a static measure of geographic range, which does not account for changes in species distributions over time. Furthermore, species could naturally have a small range or the small range size could be due to historic habitat loss<sup><a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a></sup>. The UK populations included in the Living Planet Database are predominantly from species with wide geographic ranges (Table S3), and our global scale analysis of the relationship between population change and geographic range is based on mammal and bird data. As data availability improves, future research will allow us to test the effect of geographic range on the trends of other taxa, such as amphibians and sharks.</p></div></div><div id="sec-27" class="subsection"><h3>Sensitivity analyses</h3><div id="sec-28" class="subsection"><h4>Monitoring duration, sampling methods and site-selection bias</h4><p id="p-45">To assess the influence of monitoring duration on population trends, we used a Bayesian linear model (Figures S6). We modelled population trend (<em>mu</em>) as a function of monitoring duration (years) for each population, fitted with a zero intercept, as when duration is zero, no population change has occurred. Monitoring duration was weakly positively related to vertebrate population trends, with slightly greater population increases found for longer duration studies (Figure S9, Table S2). Variation in population trends was not explained by sampling method across the five most commonly used abundance metrics (population index, number of individuals, number of pairs, number of nests and population estimate, Figure S13). Following Fournier <em>et al.</em> 2019<sup><a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">63</a></sup>, we tested the time-series that we analyzed for site-selection bias. Removing the first five survey points reduces the bias stemming from starting population surveys at points when individual density is high, whereas removing the last five years reduces the bias of starting surveys when species are very rare. The distribution of population trend values across time-series was not sensitive to the omission of the first five (left-truncation) or the last five years (right-truncation) of population records (Figure S8). Overall, our sensitivity analyses suggest that our findings are robust to the potential confounding effects of differences in monitoring duration, sampling method and site-selection.</p></div><div id="sec-29" class="subsection"><h4>Comparison of modelling approaches</h4><p id="p-46">We conducted the following supplementary analyses: in the second-stage Bayesian models estimating population trends across systems, biomes, taxa and rarity metrics, 1) we weighed <em>mu</em> values by the square of <em>tau</em>, the observation error estimate derived from the state-space models<sup><a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a></sup>, 1) we used slopes of linear model fits of abundance (scaled at the population level, centered on zero and with a standard deviation of one)<sup><a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">69</a></sup> instead of the <em>mu</em> estimates from state-space models, 2) we modelled the standard error around the slope values of the linear models, the error around <em>mu</em> (half of the 95% confidence interval) and the standard deviation of the raw population data for each time-series as additional metrics of population variability. To allow comparison, we scaled the different metrics of population variability to be centered on zero and with a standard deviation of one before they were used as response variables in models. All different analytical approaches yielded very similar results (see main text and Figures S6, S7 and S10, Table S2).</p></div></div><div id="sec-30" class="subsection"><h3>Data availability</h3><p id="p-47">Code for all data processing and analyses and summary datasets are publicly available on GitHub (link to be added at time of publication). Raw data are available from the following websites: for population time-series<sup><a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">67</a></sup> - <a href="http://www.livingplanetindex.org/data_portal">http://www.livingplanetindex.org/data_portal</a>, GBIF occurrences<sup><a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">73</a></sup> - <a href="https://www.gbif.org">https://www.gbif.org</a>, bird geographic ranges<sup><a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">74</a></sup> - <a href="http://datazone.birdlife.org">http://datazone.birdlife.org</a>, mammal geographic ranges<sup><a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">81</a></sup> - <a href="http://esapubs.org/archive/ecol/E090/184/">http://esapubs.org/archive/ecol/E090/184/</a>, species’ habitat preferences, threats and conservation status<sup><a id="xref-ref-75-5" class="xref-bibr" href="#ref-75">75</a></sup> - <a href="https://www.iucnredlist.org">https://www.iucnredlist.org</a>, and phylogenies<sup><a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">6</a>–<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">8</a></sup> - <a href="https://vertlife.org">https://vertlife.org</a> and <a href="https://birdtree.org">https://birdtree.org</a>.</p></div></div><div class="section" id="sec-32"><h2 class="">Author contributions</h2><p id="p-49">GND and IMS conceived the idea and conducted the statistical analyses. JLG contributed to the calculation of geographic range estimates. All authors contributed to the integration of the LPI, GBIF and IUCN databases, which GND led. GND created all figures with input from IMS. GND wrote the first draft of the manuscript and all authors contributed to writing. IMS supervised the research as a senior author.</p></div><div class="section" id="sec-33"><h2 class="">Competing interests</h2><p id="p-50">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-48">We thank the teams behind the open-source databases we used for our study (Living Planet Team, GBIF, BirdLife, PanTHERIA, VertLife and BirdTree. We are grateful to the WWF and the RZSL for compiling and maintaining the Living Planet Database, specifically Louise McRae and Robin Freeman, and to Dmitry Schigel and Jan Legind at GBIF for providing access to a direct download link for the occurrence data. We thank the Critical Thinking in Ecology and Environmental Sciences 2015-16 tutorial group at the University of Edinburgh including Hannah Louise Stevens, Ashlie Nithsdale, Lisa Kopsieker and Claudia Ardrey who helped us conceive the idea for the analysis. We thank Dan Greenberg for advice on state-space models and Kat Keogan and Albert Phillimore for advice on phylogenetic models. We thank Anne Magurran, Albert Phillimore and Mark Vellend for providing very helpful feedback on the manuscript. Funding for this research was provided by the NERC doctoral training partnership grant (NE/L002558/1) at the University of Edinburgh and the Carnegie Trust for the Universities of Scotland.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><span class="em-link"><span class="em-addr">johngodlee{at}gmail.com</span></span>, <span class="em-link"><span class="em-addr">isla.myers-smith{at}ed.ac.uk</span></span></p></li><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-4">We have clarified a sentence about the IUCN Green List.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.1" data-doi="10.1111/ele.12410"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ehrlén</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">W. F</span></span>. <span class="cit-article-title">Predicting changes in the distribution and abundance of species under environmental change</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">18</span>, <span class="cit-fpage">303</span>–<span class="cit-lpage">314</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett%26rft.volume%253D18%26rft.spage%253D303%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12410%26rft_id%253Dinfo%253Apmid%252F25611188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25611188&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hefley</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hooten</span>,  <span class="cit-name-given-names">M. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drake</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">R. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Walsh</span>,  <span class="cit-name-given-names">D. P</span></span>. <span class="cit-article-title">When can the cause of a population decline be determined?</span> <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">19</span>, <span class="cit-fpage">1353</span>–<span class="cit-lpage">1362</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett%26rft.volume%253D19%26rft.spage%253D1353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-272898v5.3"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">IPBES</span>. <span class="cit-source">Summary for policymakers of the regional assessment report on biodiversity and ecosystem services for Europe and Central Asia of the Intergovernmental Science-Policy Platform on Biodiversity and Ecosystem Services</span>. <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fischer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rounsevell</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Torre-Marin Rando</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mader</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Church</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Elbakidze</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Elias</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hahn</span></span>. <span class="cit-auth"> <span class="cit-name-given-names">P.A.</span> <span class="cit-name-surname">Harrison</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hauck</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Martín-López</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Ring</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Sandström</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Sousa Pinto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Visconti</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">N.E</span> <span class="cit-name-surname">Zimmermann</span></span> (eds.). <span class="cit-publ-name">IPBES secretariat</span>, <span class="cit-publ-loc">Bonn, Germany</span>. (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.4" data-doi="10.1111/ele.12569"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilroy</span>,  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gill</span>,  <span class="cit-name-given-names">J. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butchart</span>,  <span class="cit-name-given-names">S. H. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">V. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Franco</span>,  <span class="cit-name-given-names">A. M. A</span></span>. <span class="cit-article-title">Migratory diversity predicts population declines in birds</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">19</span>, <span class="cit-fpage">308</span>–<span class="cit-lpage">317</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett%26rft.volume%253D19%26rft.spage%253D308%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12569%26rft_id%253Dinfo%253Apmid%252F26807694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12569&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26807694&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.5" data-doi="10.1073/pnas.1704949114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ceballos</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehrlich</span>,  <span class="cit-name-given-names">P. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dirzo</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">Biological annihilation via the ongoing sixth mass extinction signaled by vertebrate population losses and declines</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>. <span class="cit-fpage">201704949</span> (<span class="cit-pub-date">2017</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1704949114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1704949114%26rft_id%253Dinfo%253Apmid%252F28696295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzMwL0U2MDg5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjcvMjcyODk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tonini</span>,  <span class="cit-name-given-names">J. F. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beard</span>,  <span class="cit-name-given-names">K. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferreira</span>,  <span class="cit-name-given-names">R. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jetz</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pyron</span>,  <span class="cit-name-given-names">R. A</span></span>. <span class="cit-article-title">Fully-sampled phylogenies of squamates reveal evolutionary patterns in threat status</span>. <abbr class="cit-jnl-abbrev">Biol. Conserv</abbr>. <span class="cit-vol">204</span>, <span class="cit-fpage">23</span>–<span class="cit-lpage">31</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BConserv%26rft.volume%253D204%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.7" data-doi="10.1038/s41559-018-0515-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jetz</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pyron</span>,  <span class="cit-name-given-names">R. A</span></span>. <span class="cit-article-title">The interplay of past diversification and evolutionary isolation with present imperilment across the amphibian tree of life. <em>Nat</em></span>. <abbr class="cit-jnl-abbrev">Ecol. Evol</abbr>. (<span class="cit-pub-date">2018</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41559-018-0515-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-018-0515-5%26rft_id%253Dinfo%253Apmid%252F29581588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-018-0515-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29581588&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.8" data-doi="10.1038/nature11631"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jetz</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">G. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joy</span>,  <span class="cit-name-given-names">J. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartmann</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mooers</span>,  <span class="cit-name-given-names">A. O</span></span>. <span class="cit-article-title">The global diversity of birds in space and time</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>, <span class="cit-fpage">444</span>–<span class="cit-lpage">448</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJetz%26rft.auinit1%253DW.%26rft.volume%253D491%26rft.issue%253D7424%26rft.spage%253D444%26rft.epage%253D448%26rft.atitle%253DThe%2Bglobal%2Bdiversity%2Bof%2Bbirds%2Bin%2Bspace%2Band%2Btime.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11631%26rft_id%253Dinfo%253Apmid%252F23123857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23123857&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311031600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Batt</span>,  <span class="cit-name-given-names">R. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morley</span>,  <span class="cit-name-given-names">J. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Selden</span>,  <span class="cit-name-given-names">R. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tingley</span>,  <span class="cit-name-given-names">M. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pinsky</span>,  <span class="cit-name-given-names">M. L</span></span>. <span class="cit-article-title">Gradual changes in range size accompany long-term trends in species richness</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">20</span>, <span class="cit-fpage">1148</span>–<span class="cit-lpage">1157</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett%26rft.volume%253D20%26rft.spage%253D1148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.10" data-doi="10.1016/j.tree.2015.08.009"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oliver</span>,  <span class="cit-name-given-names">T. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Biodiversity and Resilience of Ecosystem Functions</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol</abbr>. <span class="cit-vol">30</span>, <span class="cit-fpage">673</span>–<span class="cit-lpage">684</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol%26rft.volume%253D30%26rft.spage%253D673%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2015.08.009%26rft_id%253Dinfo%253Apmid%252F26437633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2015.08.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26437633&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.11" data-doi="10.1126/science.1244693"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hansen</span>,  <span class="cit-name-given-names">M. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">High-resolution global maps of 21st-century forest cover change</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>, <span class="cit-fpage">850</span>–<span class="cit-lpage">853</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHansen%26rft.auinit1%253DM.%2BC.%26rft.volume%253D342%26rft.issue%253D6160%26rft.spage%253D850%26rft.epage%253D853%26rft.atitle%253DHigh-Resolution%2BGlobal%2BMaps%2Bof%2B21st-Century%2BForest%2BCover%2BChange%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1244693%26rft_id%253Dinfo%253Apmid%252F24233722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDIvNjE2MC84NTAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8yNy8yNzI4OTguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.12" data-doi="10.1101/432880"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bowler</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">The geography of the Anthropocene differs between the land and the sea</abbr>. (<span class="cit-pub-date">2018</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/432880</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2Bgeography%2Bof%2Bthe%2BAnthropocene%2Bdiffers%2Bbetween%2Bthe%2Bland%2Band%2Bthe%2Bsea%26rft_id%253Dinfo%253Adoi%252F10.1101%252F432880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0MzI4ODB2MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzI3LzI3Mjg5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Halpern</span>,  <span class="cit-name-given-names">B. S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Spatial and temporal changes in cumulative human impacts on the world’s ocean</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">6</span>, (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.14" data-doi="10.1073/pnas.0703333104"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barlow</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Quantifying the biodiversity value of tropical primary, secondary, and plantation forests</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>. <span class="cit-vol">104</span>, <span class="cit-fpage">18555</span>–<span class="cit-lpage">18560</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0703333104%26rft_id%253Dinfo%253Apmid%252F18003934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ3LzE4NTU1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjcvMjcyODk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.15" data-doi="10.1101/457424"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blowes</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Supp</span>,  <span class="cit-name-given-names">S. R.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Biodiversity trends are stronger in marine than terrestrial assemblages</abbr>. (<span class="cit-pub-date">2018</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/457424</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiodiversity%2Btrends%2Bare%2Bstronger%2Bin%2Bmarine%2Bthan%2Bterrestrial%2Bassemblages%26rft_id%253Dinfo%253Adoi%252F10.1101%252F457424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0NTc0MjR2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzI3LzI3Mjg5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.16" data-doi="10.1098/rspb.2004.2724"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Isaac</span>,  <span class="cit-name-given-names">N. J. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Cowlishaw</span>,  <span class="cit-name-given-names">G</span></span>. <span class="cit-article-title">How species respond to multiple extinction threats</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci</abbr>. <span class="cit-vol">271</span>, <span class="cit-fpage">1135</span>–<span class="cit-lpage">1141</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DIsaac%26rft.auinit1%253DN.%2BJ.%2BB.%26rft.volume%253D271%26rft.issue%253D1544%26rft.spage%253D1135%26rft.epage%253D1141%26rft.atitle%253DHow%2Bspecies%2Brespond%2Bto%2Bmultiple%2Bextinction%2Bthreats%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2004.2724%26rft_id%253Dinfo%253Apmid%252F15306363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2004.2724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15306363&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223994100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-272898v5.17" data-doi="10.1098/rspb.2014.1097"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khaliq</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hof</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prinzinger</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bohning-Gaese</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pfenninger</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">Global variation in thermal tolerances and vulnerability of endotherms to climate change</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. B Biol. Sci</abbr>. <span class="cit-vol">281</span>, <span class="cit-fpage">20141097</span>–<span class="cit-lpage">20141097</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BB%2BBiol.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2014.1097%26rft_id%253Dinfo%253Apmid%252F25009066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2014.1097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25009066&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morrison</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estrada</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Early</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">Species traits suggest European mammals facing the greatest climate change are also least able to colonize new locations</span>. <abbr class="cit-jnl-abbrev">Divers. Distrib</abbr>. <span class="cit-vol">24</span>, <span class="cit-fpage">1321</span>–<span class="cit-lpage">1332</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDivers.%2BDistrib%26rft.volume%253D24%26rft.spage%253D1321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.19" data-doi="10.1098/rspb.1977.0056"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pigott</span>,  <span class="cit-name-given-names">C. D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Walters</span>,  <span class="cit-name-given-names">S. M</span></span>. <span class="cit-article-title">The scientific basis of practical conservation: aims and methods of conservation. in <em>Proc</em></span>. <abbr class="cit-jnl-abbrev">R. Soc. Lond. B</abbr> <span class="cit-vol">197</span>, <span class="cit-fpage">59</span>–<span class="cit-lpage">68</span> (The Royal Society, <span class="cit-pub-date">1977</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR.%2BSoc.%2BLond.%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1977.0056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.1977.0056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-272898v5.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Longton</span>,  <span class="cit-name-given-names">R. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hedderson</span>,  <span class="cit-name-given-names">T. A</span></span>. <span class="cit-article-title">What are rare species and why conserve them?</span> <abbr class="cit-jnl-abbrev">Lindbergia</abbr> <span class="cit-vol">25</span>, <span class="cit-fpage">53</span>–<span class="cit-lpage">61</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLindbergia%26rft.volume%253D25%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.21" data-doi="10.1016/j.tree.2007.11.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gaston</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fuller</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">Commonness, population depletion and conservation biology</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol</abbr>. <span class="cit-vol">23</span>, <span class="cit-fpage">14</span>–<span class="cit-lpage">19</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DGaston%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D14%26rft.epage%253D19%26rft.atitle%253DCommonness%252C%2Bpopulation%2Bdepletion%2Band%2Bconservation%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2007.11.001%26rft_id%253Dinfo%253Apmid%252F18037531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2007.11.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18037531&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252767300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.22" data-doi="10.1086/285580"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lande</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">Risks of population extinction from demographic and environmental stochasticity and random catastrophes</span>. <abbr class="cit-jnl-abbrev">Am. Nat</abbr>. <span class="cit-vol">142</span>, <span class="cit-fpage">911</span>–<span class="cit-lpage">927</span> (<span class="cit-pub-date">1993</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat%26rft.volume%253D142%26rft.spage%253D911%26rft_id%253Dinfo%253Adoi%252F10.1086%252F285580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/285580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993MQ51700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.23" data-doi="10.1038/nature06922"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Melbourne</span>,  <span class="cit-name-given-names">B. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hastings</span>,  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">Extinction risk depends strongly on factors contributing to stochasticity</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>, <span class="cit-fpage">100</span>–<span class="cit-lpage">103</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMelbourne%26rft.auinit1%253DB.%2BA.%26rft.volume%253D454%26rft.issue%253D7200%26rft.spage%253D100%26rft.epage%253D103%26rft.atitle%253DExtinction%2Brisk%2Bdepends%2Bstrongly%2Bon%2Bfactors%2Bcontributing%2Bto%2Bstochasticity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06922%26rft_id%253Dinfo%253Apmid%252F18596809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06922&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18596809&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257308300045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.24" data-doi="10.1038/nature01471"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kilpatrick</span>,  <span class="cit-name-given-names">A. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ives</span>,  <span class="cit-name-given-names">A. R</span></span>. <span class="cit-article-title">Species interactions can explain Taylor’s power law for ecological time series</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>, <span class="cit-fpage">65</span>–<span class="cit-lpage">68</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKilpatrick%26rft.auinit1%253DA.%2BM.%26rft.volume%253D422%26rft.issue%253D6927%26rft.spage%253D65%26rft.epage%253D68%26rft.atitle%253DSpecies%2Binteractions%2Bcan%2Bexplain%2BTaylor%2527s%2Bpower%2Blaw%2Bfor%2Becological%2Btime%2Bseries.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01471%26rft_id%253Dinfo%253Apmid%252F12621433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12621433&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.25" data-doi="10.1098/rstb.1990.0191"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kareiva</span>,  <span class="cit-name-given-names">P</span></span>. <span class="cit-article-title">Population dynamics in spatially complex environments: theory and data</span>. <abbr class="cit-jnl-abbrev">Phil Trans R Soc Lond B</abbr> <span class="cit-vol">330</span>, <span class="cit-fpage">175</span>–<span class="cit-lpage">190</span> (<span class="cit-pub-date">1990</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil%2BTrans%2BR%2BSoc%2BLond%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1990.0191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.1990.0191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.26" data-doi="10.1038/23876"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hanski</span>,  <span class="cit-name-given-names">I</span></span>. <span class="cit-article-title">Metapopulation dynamics</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">396</span>, <span class="cit-fpage">41</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D396%26rft.issue%253D6706%26rft.spage%253D41%26rft.epage%253D49%26rft.atitle%253DMetapopulation%2Bdynamics%26rft_id%253Dinfo%253Adoi%252F10.1038%252F23876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/23876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1999002870&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000076852700046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dennis</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Assas</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elaydi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kwessi</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Livadiotis</span>,  <span class="cit-name-given-names">G</span></span>. <span class="cit-article-title">Allee effects and resilience in stochastic populations</span>. <abbr class="cit-jnl-abbrev">Theor. Ecol</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">323</span>–<span class="cit-lpage">335</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BEcol%26rft.volume%253D9%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.28" data-doi="10.1007/s11071-016-2671-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">G.-Q</span></span>. <span class="cit-article-title">Mathematical modeling of population dynamics with Allee effect</span>. <abbr class="cit-jnl-abbrev">Nonlinear Dyn</abbr>. <span class="cit-vol">85</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNonlinear%2BDyn%26rft.volume%253D85%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11071-016-2671-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11071-016-2671-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bowler</span>,  <span class="cit-name-given-names">D. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heldbjerg</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fox</span>,  <span class="cit-name-given-names">A. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Hara</span>,  <span class="cit-name-given-names">R. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Böhning-Gaese</span>,  <span class="cit-name-given-names">K</span></span>. <span class="cit-article-title">Disentangling the effects of multiple environmental drivers on population changes within communities</span>. <abbr class="cit-jnl-abbrev">J. Anim. Ecol</abbr>. <span class="cit-vol">87</span>, <span class="cit-fpage">1034</span>–<span class="cit-lpage">1045</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BEcol%26rft.volume%253D87%26rft.spage%253D1034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.30" data-doi="10.1098/rspb.2012.1902"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harnik</span>,  <span class="cit-name-given-names">P. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">C.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Payne</span>,  <span class="cit-name-given-names">J. L</span></span>. <span class="cit-article-title">Long-term differences in extinction risk among the seven forms of rarity</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. B Biol. Sci</abbr>. <span class="cit-vol">279</span>, <span class="cit-fpage">4969</span>–<span class="cit-lpage">4976</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BB%2BBiol.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2012.1902%26rft_id%253Dinfo%253Apmid%252F23097507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2012.1902&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23097507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.31" data-doi="10.1579/0044-7447(2007)36[614:TAAHNO]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Steffen</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crutzen</span>,  <span class="cit-name-given-names">P. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">McNeill</span>,  <span class="cit-name-given-names">J. R</span></span>. <span class="cit-article-title">The Anthropocene: Are Humans Now Overwhelming the Great Forces of Nature</span>. <abbr class="cit-jnl-abbrev">AMBIO J. Hum. Environ</abbr>. <span class="cit-vol">36</span>, <span class="cit-fpage">614</span>–<span class="cit-lpage">621</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAMBIO%2BJ.%2BHum.%2BEnviron%26rft.volume%253D36%26rft.spage%253D614%26rft_id%253Dinfo%253Adoi%252F10.1579%252F0044-7447%25282007%252936%255B614%253ATAAHNO%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1579/0044-7447(2007)36[614:TAAHNO]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.32" data-doi="10.1038/536143a"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maxwell</span>,  <span class="cit-name-given-names">S. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fuller</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brooks</span>,  <span class="cit-name-given-names">T. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Watson</span>,  <span class="cit-name-given-names">J. E</span></span>. <span class="cit-article-title">Biodiversity: The ravages of guns, nets and bulldozers</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">536</span>, <span class="cit-fpage">143</span>–<span class="cit-lpage">145</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.1038%252F536143a%26rft_id%253Dinfo%253Apmid%252F27510207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/536143a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27510207&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Strien</span>,  <span class="cit-name-given-names">A. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Modest recovery of biodiversity in a western European country: The Living Planet Index for the Netherlands</span>. <abbr class="cit-jnl-abbrev">Biol. Conserv</abbr>. <span class="cit-vol">200</span>, <span class="cit-fpage">44</span>–<span class="cit-lpage">50</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BConserv%26rft.volume%253D200%26rft.spage%253D44%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.34" data-doi="10.1111/j.1523-1739.2008.01044.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mace</span>,  <span class="cit-name-given-names">G. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Quantification of Extinction Risk: IUCN’s System for Classifying Threatened Species</span>. <abbr class="cit-jnl-abbrev">Conserv. Biol</abbr>. <span class="cit-vol">22</span>, <span class="cit-fpage">1424</span>–<span class="cit-lpage">1442</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DConserv%2BBiol%26rft.aulast%253DMace%26rft.auinit1%253DG.%2BM.%26rft.volume%253D22%26rft.issue%253D6%26rft.spage%253D1424%26rft.epage%253D1442%26rft.atitle%253DQuantification%2Bof%2Bextinction%2Brisk%253A%2BIUCN%2527s%2Bsystem%2Bfor%2Bclassifying%2Bthreatened%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1523-1739.2008.01044.x%26rft_id%253Dinfo%253Apmid%252F18847444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1523-1739.2008.01044.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18847444&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261395700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.35" data-doi="10.1111/cobi.13112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Akçakaya</span>,  <span class="cit-name-given-names">H. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Quantifying species recovery and conservation success to develop an IUCN Green List of Species: Species Recovery and Conservation Success</span>. <abbr class="cit-jnl-abbrev">Conserv. Biol</abbr>. <span class="cit-vol">32</span>, <span class="cit-fpage">1128</span>–<span class="cit-lpage">1138</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BBiol%26rft.volume%253D32%26rft.spage%253D1128%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fcobi.13112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/cobi.13112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Humbert</span>,  <span class="cit-name-given-names">J.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scott Mills</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horne</span>,  <span class="cit-name-given-names">J. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dennis</span>,  <span class="cit-name-given-names">B</span></span>. <span class="cit-article-title">A better way to estimate population trends</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr> <span class="cit-vol">118</span>, <span class="cit-fpage">1940</span>–<span class="cit-lpage">1946</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D118%26rft.spage%253D1940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leung</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenberg</span>,  <span class="cit-name-given-names">D. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">D. M</span></span>. <span class="cit-article-title">Trends in mean growth and stability in temperate vertebrate populations</span>. <abbr class="cit-jnl-abbrev">Divers. Distrib</abbr>. <span class="cit-vol">23</span>, <span class="cit-fpage">1372</span>–<span class="cit-lpage">1380</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDivers.%2BDistrib%26rft.volume%253D23%26rft.spage%253D1372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.38" data-doi="10.1111/j.1461-0248.2009.01307.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fritz</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bininda-Emonds</span>,  <span class="cit-name-given-names">O. R. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Purvis</span>,  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">Geographical variation in predictors of mammalian extinction risk: big is bad, but only in the tropics</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">12</span>, <span class="cit-fpage">538</span>–<span class="cit-lpage">549</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DFritz%26rft.auinit1%253DS.%2BA.%26rft.volume%253D12%26rft.issue%253D6%26rft.spage%253D538%26rft.epage%253D549%26rft.atitle%253DGeographical%2Bvariation%2Bin%2Bpredictors%2Bof%2Bmammalian%2Bextinction%2Brisk%253A%2Bbig%2Bis%2Bbad%252C%2Bbut%2Bonly%2Bin%2Bthe%2Btropics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2009.01307.x%26rft_id%253Dinfo%253Apmid%252F19392714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2009.01307.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19392714&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266111200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-272898v5.39" data-doi="10.1111/j.1461-0248.2006.00883.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brook</span>,  <span class="cit-name-given-names">B. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Traill</span>,  <span class="cit-name-given-names">L. W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>,  <span class="cit-name-given-names">C. J. A</span></span>. <span class="cit-article-title">Minimum viable population sizes and global extinction risk are unrelated: MVP is unrelated to global extinction risk</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">9</span>, <span class="cit-fpage">375</span>– <span class="cit-lpage">382</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DBrook%26rft.auinit1%253DB.%2BW.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D375%26rft.epage%253D382%26rft.atitle%253DMinimum%2Bviable%2Bpopulation%2Bsizes%2Band%2Bglobal%2Bextinction%2Brisk%2Bare%2Bunrelated.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2006.00883.x%26rft_id%253Dinfo%253Apmid%252F16623722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2006.00883.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16623722&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236384100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.40" data-doi="10.1146/annurev-environ-042911-093511"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pereira</span>,  <span class="cit-name-given-names">H. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Navarro</span>,  <span class="cit-name-given-names">L. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Martins</span>,  <span class="cit-name-given-names">I. S</span></span>. <span class="cit-article-title">Global Biodiversity Change: The Bad, the Good, and the Unknown</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Environ. Resour</abbr>. <span class="cit-vol">37</span>, <span class="cit-fpage">25</span>–<span class="cit-lpage">50</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEnviron.%2BResour%26rft.volume%253D37%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-environ-042911-093511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-environ-042911-093511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.41" data-doi="10.1101/473645"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Daskalova</span>,  <span class="cit-name-given-names">G. N.</span></span> <span class="cit-etal">et al.</span> <abbr class="cit-jnl-abbrev">Forest loss as a catalyst of population and biodiversity change: bioRxiv</abbr> (<span class="cit-pub-date">2018</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/473645</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2Bloss%2Bas%2Ba%2Bcatalyst%2Bof%2Bpopulation%2Band%2Bbiodiversity%2Bchange%253A%2BbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F473645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0NzM2NDV2NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzI3LzI3Mjg5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.42"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">CBD</span>. <span class="cit-article-title">Strategic plan for biodiversity 2011–2020 and the Aichi targets</span>. In <abbr class="cit-jnl-abbrev">Report of the Tenth Meeting of the Conference of the Parties to the Convention on Biological Diversity</abbr>. (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.43" data-doi="10.1098/rspb.2000.1234"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Purvis</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gittleman</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cowlishaw</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Mace</span>,  <span class="cit-name-given-names">G. M</span></span>. <span class="cit-article-title">Predicting extinction risk in declining species</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. B Biol. Sci</abbr>. <span class="cit-vol">267</span>, <span class="cit-fpage">1947</span>–<span class="cit-lpage">1952</span> (<span class="cit-pub-date">2000</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DPurvis%26rft.auinit1%253DA.%26rft.volume%253D267%26rft.issue%253D1456%26rft.spage%253D1947%26rft.epage%253D1952%26rft.atitle%253DPredicting%2Bextinction%2Brisk%2Bin%2Bdeclining%2Bspecies%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2000.1234%26rft_id%253Dinfo%253Apmid%252F11075706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2000.1234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11075706&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089721600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.44" data-doi="10.1371/journal.pbio.0020197"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cardillo</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Human Population Density and Extinction Risk in the World’s Carnivores</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>. <span class="cit-vol">2</span>, <span class="cit-fpage">e197</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D2%26rft.spage%253De197%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020197%26rft_id%253Dinfo%253Apmid%252F15252445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020197&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15252445&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.45" data-doi="10.1038/nature04246"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pounds</span>,  <span class="cit-name-given-names">A. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Widespread amphibian extinctions from epidemic disease driven by global warming</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">439</span>, <span class="cit-fpage">161</span>–<span class="cit-lpage">167</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAlan%2BPounds%26rft.auinit1%253DJ.%26rft.volume%253D439%26rft.issue%253D7073%26rft.spage%253D161%26rft.epage%253D167%26rft.atitle%253DWidespread%2Bamphibian%2Bextinctions%2Bfrom%2Bepidemic%2Bdisease%2Bdriven%2Bby%2Bglobal%2Bwarming.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04246%26rft_id%253Dinfo%253Apmid%252F16407945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16407945&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234538400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Green</span>,  <span class="cit-name-given-names">D. M</span></span>. <span class="cit-article-title">The ecology of extinction: population fluctuation and decline in amphibians</span>. <abbr class="cit-jnl-abbrev">Biol. Conserv</abbr>. <span class="cit-vol">111</span>, <span class="cit-fpage">331</span>–<span class="cit-lpage">343</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BConserv%26rft.volume%253D111%26rft.spage%253D331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.47" data-doi="10.1073/pnas.0803825105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moller</span>,  <span class="cit-name-given-names">A. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rubolini</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lehikoinen</span>,  <span class="cit-name-given-names">E</span></span>. <span class="cit-article-title">Populations of migratory bird species that did not show a phenological response to climate change are declining</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>. <span class="cit-vol">105</span>, <span class="cit-fpage">16195</span>–<span class="cit-lpage">16200</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803825105%26rft_id%253Dinfo%253Apmid%252F18849475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzQyLzE2MTk1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjcvMjcyODk4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dornelas</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A balance of winners and losers in the Anthropocene</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. [in press] (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.49" data-doi="10.2307/5542"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caughley</span>,  <span class="cit-name-given-names">G</span></span>. <span class="cit-article-title">Directions in Conservation Biology</span>. <abbr class="cit-jnl-abbrev">J. Anim. Ecol</abbr>. <span class="cit-vol">63</span>, <span class="cit-fpage">215</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BEcol%26rft.volume%253D63%26rft.spage%253D215%26rft_id%253Dinfo%253Adoi%252F10.2307%252F5542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/5542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994MZ05700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.50" data-doi="10.1007/s10531-005-4307-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Robbirt</span>,  <span class="cit-name-given-names">K. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roberts</span>,  <span class="cit-name-given-names">D. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hawkins</span>,  <span class="cit-name-given-names">J. A</span></span>. <span class="cit-article-title">Comparing IUCN and Probabilistic Assessments of Threat: Do IUCN Red List Criteria Conflate Rarity and Threat?</span> <abbr class="cit-jnl-abbrev">Biodivers. Conserv</abbr>. <span class="cit-vol">15</span>, <span class="cit-fpage">1903</span>–<span class="cit-lpage">1912</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiodivers.%2BConserv%26rft.volume%253D15%26rft.spage%253D1903%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10531-005-4307-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10531-005-4307-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Viers</span>,  <span class="cit-name-given-names">J. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thorne</span>,  <span class="cit-name-given-names">J. H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Grace</span>,  <span class="cit-name-given-names">J. B</span></span>. <span class="cit-article-title">Favorable environments and the persistence of naturally rare species</span>. <abbr class="cit-jnl-abbrev">Conserv. Lett</abbr>. <span class="cit-vol">1</span>, <span class="cit-fpage">65</span>–<span class="cit-lpage">74</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BLett%26rft.volume%253D1%26rft.spage%253D65%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mihoub</span>,  <span class="cit-name-given-names">J.-B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Setting temporal baselines for biodiversity: the limits of available monitoring data for capturing the full impact of anthropogenic pressures</span>. <abbr class="cit-jnl-abbrev">Sci. Rep</abbr>. <span class="cit-vol">7</span>, <span class="cit-fpage">41591</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D7%26rft.spage%253D41591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.53" data-doi="10.1890/05-1182"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vellend</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Extinction debt of forest plants persists for more than a century following habitat fragmentation</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">87</span>, <span class="cit-fpage">542</span>–<span class="cit-lpage">548</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DVellend%26rft.auinit1%253DM.%26rft.volume%253D87%26rft.issue%253D3%26rft.spage%253D542%26rft.epage%253D548%26rft.atitle%253DExtinction%2Bdebt%2Bof%2Bforest%2Bplants%2Bpersists%2Bfor%2Bmore%2Bthan%2Ba%2Bcentury%2Bfollowing%2Bhabitat%2Bfragmentation.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F05-1182%26rft_id%253Dinfo%253Apmid%252F16602283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/05-1182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16602283&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236289600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fagan</span>,  <span class="cit-name-given-names">W. F.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Holmes</span>,  <span class="cit-name-given-names">E. E</span></span>. <span class="cit-article-title">Quantifying the extinction vortex</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">0</span>, <span class="cit-fpage">051109031307004</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett%26rft.volume%253D0%26rft.spage%253D051109031307004%26rft.atitle%253DQUANTIFYING%2BTHE%2BEXTINCTION%2BVORTEX%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-book" id="cit-272898v5.55"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WWF</span>. <span class="cit-source">Living Planet Report - 2018: Aiming Higher</span>. <span class="cit-auth"><span class="cit-name-surname">Grooten</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Almond</span>,  <span class="cit-name-given-names">R.E.A.</span></span>(Eds). <span class="cit-publ-name">WWF</span>, <span class="cit-publ-loc">Gland, Switzerland</span>. (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">56.</span><div class="cit ref-cit ref-book no-rev-xref" id="cit-272898v5.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McRae</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freeman</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Marconi</span>,  <span class="cit-name-given-names">V</span></span>. <span class="cit-source">‘The Living Planet Index’ in: Living Planet Report 2016: Risk and resilience in a new era</span> (ed. <span class="cit-auth"><span class="cit-name-surname">Oerlemans</span>  <span class="cit-name-given-names">N</span></span>). <span class="cit-publ-name">WWF International</span>, <span class="cit-publ-loc">Gland, Switzerland</span>. (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-272898v5.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McRae</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-source">The Living Planet Index in: Living Planet Report 2012</span> (ed. <span class="cit-auth"><span class="cit-name-surname">Almond</span>  <span class="cit-name-given-names">R</span></span>). <span class="cit-publ-name">WWF</span>, <span class="cit-publ-name">Gland, Switzerland</span>. (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.58"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">WWF</span>. <abbr class="cit-jnl-abbrev">Living Planet Report 2018 Technical Supplement: Living Planet Index</abbr>. (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McRae</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deinet</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Freeman</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">The Diversity-Weighted Living Planet Index: Controlling for Taxonomic Bias in a Global Biodiversity Indicator</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e0169156</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D12%26rft.spage%253De0169156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davidson</span>,  <span class="cit-name-given-names">A. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Geography of current and future global mammal extinction risk</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">e0186934</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D12%26rft.spage%253De0186934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.61" data-doi="10.1111/gcb.14361"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spooner</span>,  <span class="cit-name-given-names">F. E. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pearson</span>,  <span class="cit-name-given-names">R. G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Freeman</span>,  <span class="cit-name-given-names">R</span></span>. <span class="cit-article-title">Rapid warming is associated with population decline among terrestrial birds and mammals globally</span>. <abbr class="cit-jnl-abbrev">Glob. Change Biol</abbr>. (<span class="cit-pub-date">2018</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/gcb.14361</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob.%2BChange%2BBiol%26rft.volume%253D24%26rft.spage%253D4521%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.14361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.14361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Keogan</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Global phenological insensitivity to shifting ocean temperatures among seabirds. <em>Nat</em></span>. <abbr class="cit-jnl-abbrev">Clim. Change</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">313</span>–<span class="cit-lpage">318</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClim.%2BChange%26rft.volume%253D8%26rft.spage%253D313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.63" data-doi="10.7287/peerj.preprints.27507v1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fournier</span>,  <span class="cit-name-given-names">A. M. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">E. R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Heard</span>,  <span class="cit-name-given-names">S. B</span></span>. <abbr class="cit-jnl-abbrev">Site-selection bias can drive apparent population declines in long-term studies</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.7287/peerj.preprints.27507v1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSite-selection%2Bbias%2Bcan%2Bdrive%2Bapparent%2Bpopulation%2Bdeclines%2Bin%2Blong-term%2Bstudies%26rft_id%253Dinfo%253Adoi%252F10.7287%252Fpeerj.preprints.27507v1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7287/peerj.preprints.27507v1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.64" data-doi="10.1101/162362"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dunic</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Attributing the variability in direction and magnitude of local-scale marine biodiversity change to human activities</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (<span class="cit-pub-date">2017</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/162362</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F162362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIxNjIzNjJ2MiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzI3LzI3Mjg5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.65" data-doi="10.1111/j.0030-1299.2004.13255.x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Vinebrooke</span></span>, R. <span class="cit-etal">et al.</span> <span class="cit-article-title">Impacts of multiple stressors on biodiversity and ecosystem functioning: the role of species co-tolerance</span>. <abbr class="cit-jnl-abbrev">Oikos</abbr> <span class="cit-vol">104</span>, <span class="cit-fpage">451</span>–<span class="cit-lpage">457</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D104%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0030-1299.2004.13255.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0030-1299.2004.13255.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000189023900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-book" id="cit-272898v5.66"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team. R</span>: <span class="cit-source">A language and environment for statistical computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>. (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.67"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">LPI. Living Planet Index database</span>, <span class="cit-pub-date">2016</span>. <a href="http://www.livingplanetindex.org">www.livingplanetindex.org</a>/ Downloaded February 2016. (2016).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">68.</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference 68. in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.68" data-doi="10.1101/498170"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wauchope</span>,  <span class="cit-name-given-names">H. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johnston</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amano</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sutherland</span>,  <span class="cit-name-given-names">W. J</span></span>. <span class="cit-article-title">When can we trust population trends? A method for quantifying the effects of sampling interval and duration</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> (<span class="cit-pub-date">2019</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/498170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F498170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0OTgxNzB2NCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzI3LzI3Mjg5OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">69.</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference 69. in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.69" data-doi="10.1016/j.anbehav.2008.11.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van de Pol</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">J</span></span>. <span class="cit-article-title">A simple method for distinguishing within-versus between-subject effects using mixed models</span>. <abbr class="cit-jnl-abbrev">Anim. Behav</abbr>. <span class="cit-vol">77</span>, <span class="cit-fpage">753</span>–<span class="cit-lpage">758</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BBehav%26rft.volume%253D77%26rft.spage%253D753%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.anbehav.2008.11.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.anbehav.2008.11.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263366100023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">70.</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference 70. in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.70" data-doi="10.1098/rstb.2015.0140"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foley</span>,  <span class="cit-name-given-names">N. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Springer</span>,  <span class="cit-name-given-names">M. S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Teeling</span>,  <span class="cit-name-given-names">E. C</span></span>. <span class="cit-article-title">Mammal madness: is the mammal tree of life not yet resolved?</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. B Biol. Sci</abbr>. <span class="cit-vol">371</span>, <span class="cit-fpage">20150140</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BB%2BBiol.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2015.0140%26rft_id%253Dinfo%253Apmid%252F27325836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2015.0140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27325836&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">71.</span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference 71. in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.71" data-doi="10.1093/gbe/evv261"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tarver</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The Interrelationships of Placental Mammals and the Limits of Phylogenetic Inference</span>. <abbr class="cit-jnl-abbrev">Genome Biol. Evol</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">330</span>–<span class="cit-lpage">344</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv261%26rft_id%253Dinfo%253Apmid%252F26733575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26733575&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">72.</span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference 72. in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rabinowitz</span>,  <span class="cit-name-given-names">D</span></span>. <span class="cit-article-title">Seven forms of rarity</span>. in <abbr class="cit-jnl-abbrev">The biological aspects of rare plants conservation</abbr> <span class="cit-fpage">205</span>–<span class="cit-lpage">217</span> (<span class="cit-pub-date">1981</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">73.</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference 73. in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.73"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">GBIF</span>. <abbr class="cit-jnl-abbrev">Biodiversity occurrence data</abbr>. <a href="http://www.gbif.org/">http://www.gbif.org/</a> Downloaded Feb 2016 - Apr 2017. (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">74.</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference 74. in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.74"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">BirdLife International</span>. <abbr class="cit-jnl-abbrev">IUCN Red List for birds</abbr>. Downloaded from <a href="http://www.birdlife.org">http://www.birdlife.org</a> in March 2018. (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">75.</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference 75. in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.75"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">IUCN</span>. <abbr class="cit-jnl-abbrev">The IUCN Red List of Threatened Species</abbr>. Version 2017-3. (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">76.</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference 76. in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Di Marco</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Venter</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Possingham</span>,  <span class="cit-name-given-names">H. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Watson</span>,  <span class="cit-name-given-names">J. E. M</span></span>. <span class="cit-article-title">Changes in human footprint drive changes in species extinction risk</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">9</span>, (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">77.</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference 77. in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.77" data-doi="10.1890/0012-9615(2006)76[323:EDDPNA]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dennis</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ponciano</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lele</span>,  <span class="cit-name-given-names">S. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taper</span>,  <span class="cit-name-given-names">M. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Staples</span>,  <span class="cit-name-given-names">D. F</span></span>. <span class="cit-article-title">Estimating density dependence, process noise, and observation error</span>. <abbr class="cit-jnl-abbrev">Ecol. Monogr</abbr>. <span class="cit-vol">76</span>, <span class="cit-fpage">323</span>–<span class="cit-lpage">341</span> (<span class="cit-pub-date">2006</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BMonogr%26rft.volume%253D76%26rft.spage%253D323%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9615%25282006%252976%255B323%253AEDDPNA%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9615(2006)76[323:EDDPNA]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239845400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">78.</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference 78. in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Knape</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jonzén</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sköld</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-article-title">On observation distributions for state space models of population survey data: Observation models for population data</span>. <abbr class="cit-jnl-abbrev">J. Anim. Ecol</abbr>. <span class="cit-vol">80</span>, <span class="cit-fpage">1269</span>– <span class="cit-lpage">1277</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BEcol%26rft.aulast%253DKnape%26rft.auinit1%253DJ.%26rft.volume%253D80%26rft.issue%253D6%26rft.spage%253D1269%26rft.epage%253D1277%26rft.atitle%253DOn%2Bobservation%2Bdistributions%2Bfor%2Bstate%2Bspace%2Bmodels%2Bof%2Bpopulation%2Bsurvey%2Bdata.%26rft_id%253Dinfo%253Apmid%252F21635251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21635251&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">79.</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference 79. in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.79" data-doi="10.1016/j.ecolmodel.2011.01.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">M. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berg</span>,  <span class="cit-name-given-names">C. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thygesen</span>,  <span class="cit-name-given-names">U. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Madsen</span>,  <span class="cit-name-given-names">H</span></span>. <span class="cit-article-title">Estimation methods for nonlinear state-space models in ecology</span>. <abbr class="cit-jnl-abbrev">Ecol. Model</abbr>. <span class="cit-vol">222</span>, <span class="cit-fpage">1394</span>–<span class="cit-lpage">1400</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BModel%26rft.volume%253D222%26rft.spage%253D1394%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ecolmodel.2011.01.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ecolmodel.2011.01.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000289827500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">80.</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference 80. in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.80" data-doi="10.1111/cobi.13081"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rueda-Cediel</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">K. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regan</span>,  <span class="cit-name-given-names">T. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Regan</span>,  <span class="cit-name-given-names">H. M</span></span>. <span class="cit-article-title">Effects of uncertainty and variability on population declines and IUCN Red List classifications</span>. <abbr class="cit-jnl-abbrev">Conserv. Biol</abbr>. (<span class="cit-pub-date">2018</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/cobi.13081</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fcobi.13081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/cobi.13081&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">81.</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference 81. in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.81" data-doi="10.1890/08-1494.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">K. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">PanTHERIA: a species-level database of life history, ecology, and geography of extant and recently extinct mammals: <em>Ecological Archives</em> E090-184</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">90</span>, <span class="cit-fpage">2648</span>–<span class="cit-lpage">2648</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D90%26rft.spage%253D2648%26rft_id%253Dinfo%253Adoi%252F10.1890%252F08-1494.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/08-1494.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">82.</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference 82. in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.82" data-doi="10.1007/s10531-017-1356-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillips</span>,  <span class="cit-name-given-names">H. R. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newbold</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Purvis</span>,  <span class="cit-name-given-names">A</span></span>. <span class="cit-article-title">Land-use effects on local biodiversity in tropical forests vary between continents</span>. <abbr class="cit-jnl-abbrev">Biodivers. Conserv</abbr>. <span class="cit-vol">26</span>, <span class="cit-fpage">2251</span>–<span class="cit-lpage">2270</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiodivers.%2BConserv%26rft.volume%253D26%26rft.spage%253D2251%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10531-017-1356-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10531-017-1356-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">83.</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference 83. in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.83" data-doi="10.1111/2041-210X.13152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zizka</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">CoordinateCleaner: Standardized cleaning of occurrence records from biological collection databases</span>. <abbr class="cit-jnl-abbrev">Methods Ecol. Evol</abbr>. (<span class="cit-pub-date">2019</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/2041-210X.13152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.13152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.13152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">84.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-272898v5.84"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R. J</span></span>. <span class="cit-article-title">geosphere: Spherical Trigonometry</span>. R package version 1.5-5. <a href="https://CRAN.R-project.org/package=geosphere">https://CRAN.R-project.org/package=geosphere</a>. (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">85.</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference 85. in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>,  <span class="cit-name-given-names">S</span></span>. <span class="cit-article-title">rredlist: ‘IUCN’ Red List Client</span>. R package version 0.4.0. <a href="https://CRAN.R-project.org/package=rredlist">https://CRAN.R-project.org/package=rredlist</a>. (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">86.</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference 86. in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.86" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hadfield</span>,  <span class="cit-name-given-names">J. D</span></span>. <span class="cit-article-title">MCMC Methods for Multi-Response Generalized Linear Mixed Models: The MCMCglmm R Package</span>. <abbr class="cit-jnl-abbrev">J. Stat. Softw</abbr>. <span class="cit-vol">33</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span> (<span class="cit-pub-date">2010</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">87.</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference 87. in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.87" data-doi="10.2307/3298564"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olson</span>,  <span class="cit-name-given-names">D. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Dinerstein</span>,  <span class="cit-name-given-names">E</span></span>. <span class="cit-article-title">The Global 200: Priority Ecoregions for Global Conservation</span>. <abbr class="cit-jnl-abbrev">Ann. Mo. Bot. Gard</abbr>. <span class="cit-vol">89</span>, <span class="cit-fpage">199</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMo.%2BBot.%2BGard%26rft.volume%253D89%26rft.spage%253D199%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3298564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3298564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000176821200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">88.</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference 88. in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-272898v5.88" data-doi="10.1038/nature01547"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Magurran</span>,  <span class="cit-name-given-names">A. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Henderson</span>,  <span class="cit-name-given-names">P. A</span></span>. <span class="cit-article-title">Explaining the excess of rare species in natural species abundance distributions</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>, <span class="cit-fpage">714</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMagurran%26rft.auinit1%253DA.%2BE.%26rft.volume%253D422%26rft.issue%253D6933%26rft.spage%253D714%26rft.epage%253D716%26rft.atitle%253DExplaining%2Bthe%2Bexcess%2Bof%2Brare%2Bspecies%2Bin%2Bnatural%2Bspecies%2Babundance%2Bdistributions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01547%26rft_id%253Dinfo%253Apmid%252F12700760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01547&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12700760&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F27%2F272898.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182272300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/272898v5.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F272898v5" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/856153v1" title="Modelling the Insect Navigation Toolkit: How the Mushroom Bodies and Central Complex Coordinate Guidance Strategies" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;November 27, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/272898v5.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/272898v5.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/272898v5.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/272898v5" class="active" data-icon-position="" data-hide-link-title="0">All is not decline across global vertebrate populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1011534" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yOaQGrc29oUDQEB44y7aVZJ9F7sud2jKx_yTetIQDZ8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1011534" id="node1011534--2" data-pisa="biorxiv;272898v5" data-pisa-master="biorxiv;272898" data-seqnum="1011534" data-apath="/biorxiv/early/2019/11/27/272898.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">All is not decline across global vertebrate populations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gergana N.</span> <span class="nlm-surname">Daskalova</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Isla H.</span> <span class="nlm-surname">Myers-Smith</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">John L.</span> <span class="nlm-surname">Godlee</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">272898; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/272898 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/272898v5" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/272898v5&amp;title=All%20is%20not%20decline%20across%20global%20vertebrate%20populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/272898v5&amp;title=All%20is%20not%20decline%20across%20global%20vertebrate%20populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/272898v5&amp;text=All%20is%20not%20decline%20across%20global%20vertebrate%20populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/272898v5&amp;title=All%20is%20not%20decline%20across%20global%20vertebrate%20populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/272898v5&amp;t=All%20is%20not%20decline%20across%20global%20vertebrate%20populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/272898v5&amp;title=All%20is%20not%20decline%20across%20global%20vertebrate%20populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/272898v5&amp;title=All%20is%20not%20decline%20across%20global%20vertebrate%20populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1011534" id="node1011534--3" data-pisa="biorxiv;272898v5" data-pisa-master="biorxiv;272898" data-seqnum="1011534" data-apath="/biorxiv/early/2019/11/27/272898.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">All is not decline across global vertebrate populations</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Gergana N.</span> <span class="nlm-surname">Daskalova</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Isla H.</span> <span class="nlm-surname">Myers-Smith</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">John L.</span> <span class="nlm-surname">Godlee</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">272898; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/272898 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=All%20is%20not%20decline%20across%20global%20vertebrate%20populations&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Daskalova%2C+Gergana+N.&amp;rft.au=Myers-Smith%2C+Isla+H.&amp;rft.au=Godlee%2C+John+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1011534/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1011534/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1011534/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1011534/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1011534/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1011534/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1011534/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1011534/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1011534/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1011534/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1011534/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1011534/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/272898v5&amp;count=horizontal&amp;via=&amp;text=All%20is%20not%20decline%20across%20global%20vertebrate%20populations&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/272898v5" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/272898v5&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/272898v5" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
