<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/11/14/563635.atom" />
<meta name="HW.pisa" content="biorxiv;563635v3" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae" />
<meta name="DC.Identifier" content="10.1101/563635" />
<meta name="DC.Date" content="2019-11-14" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Macroevolutionary and biogeography studies commonly apply multiple models to test state-dependent diversification. These models track the association between states of interest along a phylogeny, but many of them do not consider whether different clades might be evolving under different evolutionary drivers. Yet, they are still commonly applied to empirical studies without careful consideration of possible lineage diversification heterogeneity along the phylogenetic tree. A recent biogeographic study has suggested that orogenic uplift of the southern Andes has acted as a species pump, driving diversification of the lizard family Liolaemidae (307 described species), native to temperate southern South America. Here, we argue against the main findings of this study that attributes the Andean uplift as main driver of the evolution of this group. We show that there is a clear pattern of heterogeneous diversification in the Liolaemidae, which biased the state- and environment-dependent analyses performed in the GeoSSE and RPANDA programs, respectively. We show here that there are two shifts to accelerated speciation rates involving two clades that were both classified as having “Andean” distributions. We incorporated the Geographic Hidden-State Speciation and Extinction model (GeoHiSSE) to accommodate unrelated diversification shifts, and also re-analyzed the data in RPANDA program after splitting biologically distinct clades for separate analyses, as well as including a more appropriate set of models. We demonstrate that the “Andean uplift” hypothesis is not supported when the heterogeneous diversification histories among these lizards is considered. We use the Liolaemidae as an ideal system to demonstrate potential risks of ignoring clade-specific differences in diversification patterns in macroevolutionary studies. We also implemented simulations to show that, in agreement with previous findings, the HiSSE approach can effectively and substantially reduce the level of distribution-dependent models receiving the highest AIC weights in such scenarios. However, we still find a relatively high rate of distribution-dependent models receiving the highest AIC weights (= 15%), and provide recommendations related to the set of models included in the analyses that reduce these rates by half. Finally, we demonstrate that trees including clades following different dependent-drivers affect RPANDA analyses by producing different outcomes, ranging from partially correct models to completely misleading results. We provide recommendations for the implementation of both programs." />
<meta name="DC.Contributor" content="Olave Melisa" />
<meta name="DC.Contributor" content="Luciano J. Avila" />
<meta name="DC.Contributor" content="Jack W. Sites" />
<meta name="DC.Contributor" content="Morando Mariana" />
<meta name="article:published_time" content="2019-11-14" />
<meta name="article:section" content="Contradictory Results" />
<meta name="citation_title" content="How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Macroevolutionary and biogeography studies commonly apply multiple models to test state-dependent diversification. These models track the association between states of interest along a phylogeny, but many of them do not consider whether different clades might be evolving under different evolutionary drivers. Yet, they are still commonly applied to empirical studies without careful consideration of possible lineage diversification heterogeneity along the phylogenetic tree. A recent biogeographic study has suggested that orogenic uplift of the southern Andes has acted as a species pump, driving diversification of the lizard family Liolaemidae (307 described species), native to temperate southern South America. Here, we argue against the main findings of this study that attributes the Andean uplift as main driver of the evolution of this group. We show that there is a clear pattern of heterogeneous diversification in the Liolaemidae, which biased the state- and environment-dependent analyses performed in the GeoSSE and RPANDA programs, respectively. We show here that there are two shifts to accelerated speciation rates involving two clades that were both classified as having “Andean” distributions. We incorporated the Geographic Hidden-State Speciation and Extinction model (GeoHiSSE) to accommodate unrelated diversification shifts, and also re-analyzed the data in RPANDA program after splitting biologically distinct clades for separate analyses, as well as including a more appropriate set of models. We demonstrate that the “Andean uplift” hypothesis is not supported when the heterogeneous diversification histories among these lizards is considered. We use the Liolaemidae as an ideal system to demonstrate potential risks of ignoring clade-specific differences in diversification patterns in macroevolutionary studies. We also implemented simulations to show that, in agreement with previous findings, the HiSSE approach can effectively and substantially reduce the level of distribution-dependent models receiving the highest AIC weights in such scenarios. However, we still find a relatively high rate of distribution-dependent models receiving the highest AIC weights (= 15%), and provide recommendations related to the set of models included in the analyses that reduce these rates by half. Finally, we demonstrate that trees including clades following different dependent-drivers affect RPANDA analyses by producing different outcomes, ranging from partially correct models to completely misleading results. We provide recommendations for the implementation of both programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;563635v3" />
<meta name="citation_id" content="563635v3" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/563635v3" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/563635v3.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/563635v3.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635.full.pdf" />
<meta name="citation_doi" content="10.1101/563635" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="Contradictory Results" />
<meta name="citation_firstpage" content="563635" />
<meta name="citation_author" content="Olave Melisa" />
<meta name="citation_author_institution" content="Department of Biology, University of Konstanz" />
<meta name="citation_author_institution" content="Instituto Argentino de Investigaciones de Zonas Arídas, Consejo Nacional de Investigaciones Cientificas y Técnicas (IADIZA-CONICET)" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7773-0189" />
<meta name="citation_author" content="Luciano J. Avila" />
<meta name="citation_author_institution" content="Instituto Patagónico para el Estudio de los Ecosistemas Continentales, Consejo Nacional de Investigaciones Científicas y Técnicas (IPEEC-CONICET)" />
<meta name="citation_author" content="Jack W. Sites" />
<meta name="citation_author_institution" content="Department of Biology and M.L. Bean Life Science Museum, Brigham Young University (BYU)" />
<meta name="citation_author_institution" content="Department of Biology, Austin Peay State University" />
<meta name="citation_author" content="Morando Mariana" />
<meta name="citation_author_institution" content="Instituto Patagónico para el Estudio de los Ecosistemas Continentales, Consejo Nacional de Investigaciones Científicas y Técnicas (IPEEC-CONICET)" />
<meta name="citation_author_institution" content="Universidad Nacional de la Patagonia San Juan Bosco, Sede Puerto Madryn" />
<meta name="citation_reference" content="Agnarsson, I., Avilés, L., Coddington, J. A., &amp; Maddison, W. P. (2006). Sociality in theridiid spiders: repeated origins of an evolutionary dead end. Evolution, 60(11), 2342–2351." />
<meta name="citation_reference" content="Armbruster, W. S. (2014). Floral specialization and angiosperm diversity: phenotypic divergence, fitness tradeoffs and realized pollination accuracy. AoB Plants, 6." />
<meta name="citation_reference" content="Anacker, B. L., Whittall, J. B., Goldberg, E. E., &amp; Harrison, S. P. (2011). Origins and consequences of serpentine endemism in the California flora. Evolution: International Journal of Organic Evolution, 65(2), 365–376." />
<meta name="citation_reference" content="Bacon, C. D., Velásquez-Puentes, F. J., Hoorn, C., &amp; Antonelli, A. (2018). Iriarteeae palms tracked the uplift of Andean Cordilleras. Journal of biogeography, 45(7), 1653–1663." />
<meta name="citation_reference" content="Beaulieu, J. M., &amp; O’Meara, B. C. (2016). Detecting hidden diversification shifts in models of trait-dependent speciation and extinction. Systematic biology, 65(4), 583–601." />
<meta name="citation_reference" content="Blisniuk, P. M., Stern, L. A., Chamberlain, C. P., Zeitler, P. K., Ramos, V. A., Sobel, E. R., … &amp; Warkus, F. (2006). Links between mountain uplift, climate, and surface processes in the southern Patagonian Andes. In The Andes (pp. 429–440). Springer, Berlin, Heidelberg." />
<meta name="citation_reference" content="Caetano, D. S., O’Meara, B. C., &amp; Beaulieu, J. M. (2018). Hidden state models improve state-dependent diversification approaches, including biogeographical models. Evolution, 72(11), 2308–2324." />
<meta name="citation_reference" content="Canal, D., Köster, N., Celis, M., Croat, T. B., Borsch, T., &amp; Jones, K. E. (2019). Out of Amazonia and Back again: Historical Biogeography of the Species-Rich Neotropical Genus Philodendron (Araceae) 1. Annals of the Missouri Botanical Garden, 104(1), 49–68." />
<meta name="citation_reference" content="Cei, J. M. (1986). Reptiles del centro, centro-oeste y sur de la Argentina: Herpetofauna de las zonas áridas y semiáridas. Torino: Museo Regionale di Scienze Naturali, Monografie IV: 1–527." />
<meta name="citation_reference" content="Condamine, F. L., Rolland, J., &amp; Morlon, H. (2013). Macroevolutionary perspectives to environmental change. Ecology letters, 16, 72–85." />
<meta name="citation_reference" content="Condamine, F. L., Sperling, F. A., Wahlberg, N., Rasplus, J. Y., &amp; Kergoat, G. J. (2012). What causes latitudinal gradients in species diversity? Evolutionary processes and ecological constraints on swallowtail biodiversity. Ecology Letters, 15(3), 267–277." />
<meta name="citation_reference" content="Cruz, F. B., Belver, L., Acosta, J. C., Villavicencio, H. J., Blanco, G., &amp; Cánovas, M. G. (2009). Thermal biology of Phymaturus lizards: evolutionary constraints or lack of environmental variation?. Zoology, 112(6), 425–432." />
<meta name="citation_reference" content="Esquerré, D., Brennan, I. G., Catullo, R. A., Torres-Pérez, F., &amp; Keogh, J. S. (2019). How mountains shape biodiversity: The role of the Andes in biogeography, diversification, and reproductive biology in South America’s most species-rich lizard radiation (Squamata: Liolaemidae). Evolution, 73, 214–230." />
<meta name="citation_reference" content="FitzJohn, R. G. (2010). Quantitative traits and diversification. Systematic biology, 59(6), 619–633." />
<meta name="citation_reference" content="FitzJohn, R. G. (2012). Diversitree: comparative phylogenetic analyses of diversification in R. Methods in Ecology and Evolution, 3(6), 1084–1092." />
<meta name="citation_reference" content="Forister, M. L., Dyer, L. A., Singer, M. S., Stireman III, J. O., &amp; Lill, J. T. (2012). Revisiting the evolution of ecological specialization, with emphasis on insect–plant interactions. Ecology, 93(5), 981–991." />
<meta name="citation_reference" content="Futuyma, D. J., &amp; Moreno, G. (1988). The evolution of ecological specialization. Annual review of Ecology and Systematics, 19(1), 207–233." />
<meta name="citation_reference" content="Garreaud, R. D., Molina, A., &amp; Farias, M. (2010). Andean uplift, ocean cooling and Atacama hyperaridity: A climate modeling perspective. Earth and Planetary Science Letters, 292(1-2), 39–50." />
<meta name="citation_reference" content="Gregory-Wodzicki, K. M. (2000). Uplift history of the Central and Northern Andes: a review. Geological society of America bulletin, 112(7), 1091–1105." />
<meta name="citation_reference" content="Goldberg, E. E., &amp; Igić, B. (2012). Tempo and mode in plant breeding system evolution. Evolution: International Journal of Organic Evolution, 66(12), 3701–3709." />
<meta name="citation_reference" content="Goldberg, E. E., Lancaster, L. T., &amp; Ree, R. H. (2011). Phylogenetic inference of reciprocal effects between geographic range evolution and diversification. Systematic Biology, 60(4), 451–465." />
<meta name="citation_reference" content="Harrington, S., &amp; Reeder, T. W. (2017). Rate heterogeneity across Squamata, misleading ancestral state reconstruction and the importance of proper null model specification. Journal of evolutionary biology, 30(2), 313–325." />
<meta name="citation_reference" content="Hartley, A. (2003). Andean uplift and climate change. Journal of the Geological Society, 160(1), 7–10." />
<meta name="citation_reference" content="Herrera-Alsina, L., van Els, P., &amp; Etienne, R. S. (2019). Detecting the dependence of diversification on multiple traits from phylogenetic trees and trait data. Systematic biology, 68(2), 317–328." />
<meta name="citation_reference" content="Hipp, A. L., Manos, P. S., González-Rodríguez, A., Hahn, M., Kaproth, M., McVay, J. D., … &amp; Cavender-Bares, J. (2018). Sympatric parallel diversification of major oak clades in the Americas and the origins of Mexican species diversity. New Phytologist, 217(1), 439–452." />
<meta name="citation_reference" content="Ibargüengoytía, N. R., Acosta, J. C., Boretto, J. M., Villavicencio, H. J., Marinero, J. A., &amp; Krenz, J. D. (2008). Field thermal biology in Phymaturus lizards: comparisons from the Andes to the Patagonian steppe in Argentina. Journal of Arid Environments, 72(9), 1620–1630." />
<meta name="citation_reference" content="Jablonski, D. (2008). Species selection: theory and data. Annual Review of Ecology, Evolution, and Systematics, 39, 501–524." />
<meta name="citation_reference" content="Lamb, S., &amp; Davis, P. (2003). Cenozoic climate change as a possible cause for the rise of the Andes. Nature, 425(6960), 792." />
<meta name="citation_reference" content="Lewitus, E., Bittner, L., Malviya, S., Bowler, C., &amp; Morlon, H. (2018). Clade-specific diversification dynamics of marine diatoms since the Jurassic. Nature ecology &amp; evolution, 2(11), 1715." />
<meta name="citation_reference" content="Lewitus, E., &amp; Morlon, H. (2018). Detecting environment-dependent diversification from phylogenies: a simulation study and some empirical illustrations. Systematic biology, 67(4), 576–593." />
<meta name="citation_reference" content="Li, J., Huang, J. P., Sukumaran, J., &amp; Knowles, L. L. (2018). Microevolutionary processes impact macroevolutionary patterns. BMC evolutionary biology, 18(1), 123." />
<meta name="citation_reference" content="Maddison, W. P. (2006). Confounding asymmetries in evolutionary diversification and character change. Evolution, 60(8), 1743–1746." />
<meta name="citation_reference" content="Maddison, W. P., &amp; FitzJohn, R. G. (2014). The unsolved challenge to phylogenetic correlation tests for categorical characters. Systematic biology, 64(1), 127–136." />
<meta name="citation_reference" content="Maddison, W. P., Midford, P. E., &amp; Otto, S. P. (2007). Estimating a binary character’s effect on speciation and extinction. Systematic biology, 56(5), 701–710." />
<meta name="citation_reference" content="Magnuson-Ford, K., &amp; Otto, S. P. (2012). Linking the investigations of character evolution and species diversification. The American Naturalist, 180(2), 225–245." />
<meta name="citation_reference" content="Maliet, O., Hartig, F., &amp; Morlon, H. (2019). A model with many small shifts for estimating species-specific diversification rates. Nature ecology &amp; evolution, 1." />
<meta name="citation_reference" content="Marín, A. G., Olave, M., Avila, L. J., Sites Jr, J. W., &amp; Morando, M. (2018). Evidence of body size and shape stasis driven by selection in Patagonian lizards of the Phymaturus patagonicus clade (Squamata: Liolae-mini). Molecular phylogenetics and evolution, 129, 226–241." />
<meta name="citation_reference" content="Matzke, N. J. (2016). Trait-dependent dispersal models for phylogenetic biogeography, in the R package BioGeoBEARS. Integrative and Comparative Biology, 56, E330." />
<meta name="citation_reference" content="Morlon, H., Parsons, T. L., &amp; Plotkin, J. B. (2011). Reconciling molecular phylogenies with the fossil record. Proceedings of the National Academy of Sciences, 108(39), 16327–16332." />
<meta name="citation_reference" content="Morlon, H., Lewitus, E., Condamine, F. L., Manceau, M., Clavel, J., &amp; Drury, J. (2016). RPANDA: an R package for macroevolutionary analyses on phylogenetic trees. Methods in Ecology and Evolution, 7(5), 589–597." />
<meta name="citation_reference" content="Ng, J., &amp; Smith, S. D. (2014). How traits shape trees: new approaches for detecting character state-dependent lineage diversification. Journal of evolutionary biology, 27(10), 2035–2045." />
<meta name="citation_reference" content="Olave, M., Avila, L. J., Sites Jr, J. W., &amp; Morando, M. (2018). Hybridization could be a common phenomenon within the highly diverse lizard genus Liolaemus. Journal of evolutionary biology, 31(6), 893–903." />
<meta name="citation_reference" content="Olave, M., Gonzalez Marin, A., Avila, L.J., Sites, J.W. Jr. &amp; Morando, M. Disparate patterns of diversification within Liolaemini lizards. In Neotropical diversification, edited by Rull &amp; Carnaval. Springer Nature (in press)." />
<meta name="citation_reference" content="Rabosky, D. L., &amp; Glor, R. E. (2010). Equilibrium speciation dynamics in a model adaptive radiation of island lizards. Proceedings of the National Academy of Sciences, 107(51), 22178–22183." />
<meta name="citation_reference" content="Rabosky, D. L., &amp; Goldberg, E. E. (2015). Model inadequacy and mistaken inferences of trait-dependent speciation. Systematic biology, 64(2), 340–355." />
<meta name="citation_reference" content="Rabosky, D. L., &amp; Goldberg, E. E. (2017). FiSSE: A simple nonparametric test for the effects of a binary character on lineage diversification rates. Evolution, 71(6), 1432–1442." />
<meta name="citation_reference" content="Rabosky, D. L., Grundler, M., Anderson, C., Title, P., Shi, J. J., Brown, J. W., … &amp; Larson, J. G. (2014). BAMM tools: an R package for the analysis of evolutionary dynamics on phylogenetic trees. Methods in Ecology and Evolution, 5(7), 701–707." />
<meta name="citation_reference" content="Rabosky, D. L., &amp; McCune, A. R. (2010). Reinventing species selection with molecular phylogenies. Trends in Ecology &amp; Evolution, 25(2), 68–74." />
<meta name="citation_reference" content="Reaney, A. M., Saldarriaga-Córdoba, M., &amp; Pincheira-Donoso, D. (2018). Macroevolutionary diversification with limited niche disparity in a species-rich lineage of cold-climate lizards. BMC evolutionary biology, 18(1), 16." />
<meta name="citation_reference" content="Román-Palacios, C., &amp; Wiens, J. J. (2018). The Tortoise and the Finch: Testing for island effects on diversification using two iconic Galápagos radiations. Journal of biogeography, 45(8), 1701–1712." />
<meta name="citation_reference" content="Schluter, D. (2016). Speciation, Ecological Opportunity, and Latitude: (American Society of Naturalists Address). The American Naturalist, 187(1), 1–18." />
<meta name="citation_reference" content="Steeman, M. E., Hebsgaard, M. B., Fordyce, R. E., Ho, S. Y., Rabosky, D. L., Nielsen, R., … &amp; Willerslev, E. (2009). Radiation of extant cetaceans driven by restructuring of the oceans. Systematic biology, 58(6), 573–585." />
<meta name="citation_reference" content="Templeton, A. R., Robertson, R. J., Brisson, J., &amp; Strasburg, J. (2001). Disrupting evolutionary processes: the effect of habitat fragmentation on collared lizards in the Missouri Ozarks. Proceedings of the National Academy of Sciences, 98(10), 5426–5432." />
<meta name="citation_reference" content="Thompson, K. A., Rieseberg, L. H., &amp; Schluter, D. (2018). Speciation and the city. Trends in ecology &amp; evolution." />
<meta name="citation_reference" content="Varga, T., Krizsán, K., Földi, C., Dima, B., Sánchez-García, M., Sánchez-Ramírez, S., … &amp; Andreopoulos, W. (2019). Megaphylogeny resolves global patterns of mushroom evolution. Nature ecology &amp; evolution, 1." />
<meta name="citation_reference" content="Vicenzi, N., Corbalán, V., Miles, D., Sinervo, B., &amp; Ibargüengoytía, N. (2017). Range increment or range detriment? Predicting potential changes in distribution caused by climate change for the endemic high-Andean lizard Phymaturus palluma. Biological conservation, 206, 151–160." />
<meta name="citation_reference" content="Winkler, A. M., Kochunov, P., Blangero, J., Almasy, L., Zilles, K., Fox, P. T., … &amp; Glahn, D. C. (2010). Cortical thickness or grey matter volume? The importance of selecting the phenotype for imaging genetics studies. Neuroimage, 53(3), 1135–1146." />
<meta name="citation_reference" content="Zachos, J., Pagani, M., Sloan, L., Thomas, E., &amp; Billups, K. (2001). Trends, rhythms, and aberrations in global climate 65 Ma to present. science, 292(5517), 686–693." />
<meta name="citation_reference" content="Zachos, J. C., Dickens, G. R., &amp; Zeebe, R. E. (2008). An early Cenozoic perspective on greenhouse warming and carbon-cycle dynamics. Nature, 451(7176), 279." />
<meta name="twitter:title" content="How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Macroevolutionary and biogeography studies commonly apply multiple models to test state-dependent diversification. These models track the association between states of interest along a phylogeny, but many of them do not consider whether different clades might be evolving under different evolutionary drivers. Yet, they are still commonly applied to empirical studies without careful consideration of possible lineage diversification heterogeneity along the phylogenetic tree. A recent biogeographic study has suggested that orogenic uplift of the southern Andes has acted as a species pump, driving diversification of the lizard family Liolaemidae (307 described species), native to temperate southern South America. Here, we argue against the main findings of this study that attributes the Andean uplift as main driver of the evolution of this group. We show that there is a clear pattern of heterogeneous diversification in the Liolaemidae, which biased the state- and environment-dependent analyses performed in the GeoSSE and RPANDA programs, respectively. We show here that there are two shifts to accelerated speciation rates involving two clades that were both classified as having “Andean” distributions. We incorporated the Geographic Hidden-State Speciation and Extinction model (GeoHiSSE) to accommodate unrelated diversification shifts, and also re-analyzed the data in RPANDA program after splitting biologically distinct clades for separate analyses, as well as including a more appropriate set of models. We demonstrate that the “Andean uplift” hypothesis is not supported when the heterogeneous diversification histories among these lizards is considered. We use the Liolaemidae as an ideal system to demonstrate potential risks of ignoring clade-specific differences in diversification patterns in macroevolutionary studies. We also implemented simulations to show that, in agreement with previous findings, the HiSSE approach can effectively and substantially reduce the level of distribution-dependent models receiving the highest AIC weights in such scenarios. However, we still find a relatively high rate of distribution-dependent models receiving the highest AIC weights (= 15%), and provide recommendations related to the set of models included in the analyses that reduce these rates by half. Finally, we demonstrate that trees including clades following different dependent-drivers affect RPANDA analyses by producing different outcomes, ranging from partially correct models to completely misleading results. We provide recommendations for the implementation of both programs." />
<meta name="og-title" property="og:title" content="How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/563635v3" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Macroevolutionary and biogeography studies commonly apply multiple models to test state-dependent diversification. These models track the association between states of interest along a phylogeny, but many of them do not consider whether different clades might be evolving under different evolutionary drivers. Yet, they are still commonly applied to empirical studies without careful consideration of possible lineage diversification heterogeneity along the phylogenetic tree. A recent biogeographic study has suggested that orogenic uplift of the southern Andes has acted as a species pump, driving diversification of the lizard family Liolaemidae (307 described species), native to temperate southern South America. Here, we argue against the main findings of this study that attributes the Andean uplift as main driver of the evolution of this group. We show that there is a clear pattern of heterogeneous diversification in the Liolaemidae, which biased the state- and environment-dependent analyses performed in the GeoSSE and RPANDA programs, respectively. We show here that there are two shifts to accelerated speciation rates involving two clades that were both classified as having “Andean” distributions. We incorporated the Geographic Hidden-State Speciation and Extinction model (GeoHiSSE) to accommodate unrelated diversification shifts, and also re-analyzed the data in RPANDA program after splitting biologically distinct clades for separate analyses, as well as including a more appropriate set of models. We demonstrate that the “Andean uplift” hypothesis is not supported when the heterogeneous diversification histories among these lizards is considered. We use the Liolaemidae as an ideal system to demonstrate potential risks of ignoring clade-specific differences in diversification patterns in macroevolutionary studies. We also implemented simulations to show that, in agreement with previous findings, the HiSSE approach can effectively and substantially reduce the level of distribution-dependent models receiving the highest AIC weights in such scenarios. However, we still find a relatively high rate of distribution-dependent models receiving the highest AIC weights (= 15%), and provide recommendations related to the set of models included in the analyses that reduce these rates by half. Finally, we demonstrate that trees including clades following different dependent-drivers affect RPANDA analyses by producing different outcomes, ranging from partially correct models to completely misleading results. We provide recommendations for the implementation of both programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-11-14" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/563635v3.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/563635v3.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/563635v3.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/563635v3" />
<link rel="shortlink" href="https://www.biorxiv.org/node/993719" />
    <title>How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"V8_B_jiFlrEkMjkWAloFgt8lZArgepC0XJoM6AeQOWY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"993719","apath":"\/biorxiv\/early\/2019\/11\/14\/563635.atom","pisa":"biorxiv;563635v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;563635v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/563635v3","title":"How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae","identifier":"node\/993719"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Iwx1aWeZrbZ\\\\\\\/chIL5dyDyKQ9eiNGWkNeioJmRf8zaopoyIFwQyo1LVg7SIgwIjVpgM+iwGLsVvIDwvSzxrBtv4Pi\\\\\\\/bExP+itWkmo3SswPledOYRb1gbEiu7JpTZ4BM0nQUrfGBd6mlIXIvflmnf8My8\\\\\\\/x1vlUgGMx5PFfgeUF3qKCDLf3AWllzjBrw3Jq1lOaobrs2ld1uEmKbTIf3RGhQnswAp5KveELOKW2dkA+XnhGhPNU0VDYtQ3ylHMxPnjlANTrY24w\\\\\\\/Tb5KsMUFayLiaqJR2M0eiwJqnv+rjq5JxSu912tmJRQdbb76Dw15os+JJNqjoxRuMJ89IFsO2u2NuXFQaYLimoLNCmuL4YILKO1PMMhAu7D4H\\\\\\\/Z4zIC8Nzkv3ZYuyBhpDHYRba7GWZmppsSaZl8EcRJJbZAwa94AYifNr65nsgY4NXpHO7XbqNxyLquQT+XX7RkD0TdFwizY8VqUTVFDtlifTryKMe2nf\\\\\\\/1\\\\\\\/hYr\\\\\\\/oclz0\\\\\\\/ph7yL3WCUxHu\\\\\\\/GJfpYMVKqVzYDMoXjINAHH\\\\\\\/3AHBi8Bjsf5Mog7EU+KDR1njItY47JAmC0GV19TsNpIk7jbftogKHRFPhRI9Lg8oNYe8\\\\\\\/kgmeiY1q305erh3S7kx62LQiSf7TXUgr6NoP1EsPBN6+bBFv\\\\\\\/PMdyW5i79qW+DP+MqHMShc4pXCPgffK3EJm6yXli8p8DJd8Dcdw36RFew7N1hIHNleiKij7ZA0KO+4qx67NPImeVkNNvr+5CXjb1JTHY5Fu8GVk\\\\\\\/+EOvtiJ4wi\\\\\\\/z29ZAv\\\\\\\/G8oJwCfcw8rFN78c3GUuBy796kSqvNYGQ2uV39XJtoxYeaxdqmqonAKuX8wPADgF6vqfLsvdMHLUQNGRgfhyTw+ZEQqOwYCxtZtkE+S\\\\\\\/eCADJl8UUZuQEWEdijWAWodIAZZ\\\\\\\/xU5SVlDqeLuIKi6Rl6V3wEEco3kWSoC4nzdbme22ZmdtlSDH5sr2ocCHvXqXsMVHvZeZ78YMkHeZgf6\\\\\\\/2bI23+hi00I+bkbATchvs5ESA\\\\\\\/4miuLKQbceuOFK81hRbD8x1VO4WYUNLsUhVFVAE1FTEqSgUxMd3v7024pBMhUegmkZD24qYeLhKewoZCXqPLBQz659OAbVPk62BJH29yO3mQDRFZi9iX98VrBZIRIFzSrGHQQZywaMaDb9K5MNd7LuWyTMceixoO\\\\\\\/4oJIuycIJLgS3U8zfAUnvApfyhwUT8BuVsQ\\\\\\\/E8eW3W0BZKDyd5PFtI0avVfBBX67JLFUZte4yzrDEoeyeOgCumQE1s5\\\\\\\/vD8YlnkOGBtyL2H\\\\\\\/DWyaSW5voyT15ZkqEkRKRGz3cb+c=\\\u0022,\\\u0022iv\\\u0022:\\\u0022KAnPGPbjPgB0oAuilgA\\\\\\\/pg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f78d7f86d0a111e2dd3a81be5222cf92cbeea3593db13b8b74c1bffe4a357631\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/563635v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-993719 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-contradictory-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/563635v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-b6Nk-lj6m4567oTui0ivhLnZGjTBpIsgc0nfmBzZwnU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="993719" id="node993719" data-pisa="biorxiv;563635v3" data-pisa-master="biorxiv;563635" data-apath="/biorxiv/early/2019/11/14/563635.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        Contradictory Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7773-0189" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Olave</span> <span class="nlm-surname">Melisa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Luciano J.</span> <span class="nlm-surname">Avila</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jack W.</span> <span class="nlm-surname">Sites</span> <span class="nlm-suffix">Jr.</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Morando</span> <span class="nlm-surname">Mariana</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/563635 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node993719" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Olave Melisa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, University of Konstanz</span>, Konstanz, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-institution'>Instituto Argentino de Investigaciones de Zonas Arídas, Consejo Nacional de Investigaciones Cientificas y Técnicas (IADIZA-CONICET)</span>, 5500 Mendoza, <span class='nlm-country'>Argentina</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olave%2BMelisa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Melisa%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOlave%2BMelisa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7773-0189" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Olave Melisa</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Luciano J. Avila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Instituto Patagónico para el Estudio de los Ecosistemas Continentales, Consejo Nacional de Investigaciones Científicas y Técnicas (IPEEC-CONICET)</span>, Boulevard Almirante Brown 2915, U9120ACD, Puerto Madryn, Chubut, <span class='nlm-country'>Argentina</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luciano%2BJ.%2BAvila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Avila%20LJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuciano%2BJ.%2BAvila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jack W. Sites <span class="nlm-suffix">Jr.</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology and M.L. Bean Life Science Museum, Brigham Young University (BYU)</span>, Provo, UT 84602, <span class='nlm-country'>USA</span>; <span class='nlm-institution'>Department of Biology, Austin Peay State University</span>, Clarksville, Tennessee, 37044</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jack%2BW.%2BSites%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EJr.%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sites%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJack%2BW.%2BSites%2BJr." rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Morando Mariana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Instituto Patagónico para el Estudio de los Ecosistemas Continentales, Consejo Nacional de Investigaciones Científicas y Técnicas (IPEEC-CONICET)</span>, Boulevard Almirante Brown 2915, U9120ACD, Puerto Madryn, Chubut, <span class='nlm-country'>Argentina</span></div><div class='author-affiliation'><span class='nlm-institution'>Universidad Nacional de la Patagonia San Juan Bosco, Sede Puerto Madryn</span>, Boulevard Almirante Brown 3700, U9120ACD, Puerto Madryn, Chubut, <span class='nlm-country'>Argentina</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Morando%2BMariana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mariana%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMorando%2BMariana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/563635v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:993719" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:993719/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/563635v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:993719" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:993719/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/563635v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:993719" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:993719/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/563635v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:993719" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:993719/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/563635v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:993719" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:993719/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/563635v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:993719" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:993719/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Macroevolutionary and biogeography studies commonly apply multiple models to test state-dependent diversification. These models track the association between states of interest along a phylogeny, but many of them do not consider whether different clades might be evolving under different evolutionary drivers. Yet, they are still commonly applied to empirical studies without careful consideration of possible lineage diversification heterogeneity along the phylogenetic tree. A recent biogeographic study has suggested that orogenic uplift of the southern Andes has acted as a species pump, driving diversification of the lizard family Liolaemidae (307 described species), native to temperate southern South America. Here, we argue against the main findings of this study that attributes the Andean uplift as main driver of the evolution of this group. We show that there is a clear pattern of heterogeneous diversification in the Liolaemidae, which biased the state- and environment-dependent analyses performed in the GeoSSE and RPANDA programs, respectively. We show here that there are two shifts to accelerated speciation rates involving two clades that were both classified as having “Andean” distributions. We incorporated the Geographic Hidden-State Speciation and Extinction model (GeoHiSSE) to accommodate unrelated diversification shifts, and also re-analyzed the data in RPANDA program after splitting biologically distinct clades for separate analyses, as well as including a more appropriate set of models. We demonstrate that the “Andean uplift” hypothesis is not supported when the heterogeneous diversification histories among these lizards is considered. We use the Liolaemidae as an ideal system to demonstrate potential risks of ignoring clade-specific differences in diversification patterns in macroevolutionary studies. We also implemented simulations to show that, in agreement with previous findings, the HiSSE approach can effectively and substantially reduce the level of distribution-dependent models receiving the highest AIC weights in such scenarios. However, we still find a relatively high rate of distribution-dependent models receiving the highest AIC weights (= 15%), and provide recommendations related to the set of models included in the analyses that reduce these rates by half. Finally, we demonstrate that trees including clades following different dependent-drivers affect RPANDA analyses by producing different outcomes, ranging from partially correct models to completely misleading results. We provide recommendations for the implementation of both programs.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">Macroevolutionary modeling of diversification plays an important role in inferring large-scale biodiversity patterns (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Schluter 2016</a>). Several studies have focused on quantifying differences in macroevolutionary patterns linked to geographic, ecological, life-history and other traits, based on the variation in speciation and extinction rates (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jablonski 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rabosky and McCune 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Condamine et al. 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ng and Smith 2014</a>). Given that mechanisms underlying the correlations between characters and diversification are generally poorly understood (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rabosky and Goldberg 2015</a>), a family of State-Dependent Speciation and Extinction (SSE) models have been developed. The earliest models included binary traits (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Maddison 2006</a>), quantitative traits (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">FitzJohn 2010</a>), geographic character states (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Goldberg et al. 2011</a>), multiple character states (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">FitzJohn 2012</a>), punctuated trait changes (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Goldberg and Igic 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Magnuson-Ford and Otto 2012</a>), and time-dependent macroevolutionary rates (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rabosky and Glor 2010</a>). These models track associations between the states of interest and speciation and extinction rates along a phylogenetic tree, and they have been implemented in hundreds of empirical studies (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Rabosky and Goldberg 2015</a>). In addition, interest has also increased in the application of correlative algorithms (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Steeman et al. 2009</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Winkler et al. 2010</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Condamine et al. 2012</a>; 2013; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Morlon et al. 2016</a>), as implemented in the software RPANDA (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Morlon et al. 2016</a>), to use environment-dependent models to test whether gradual changes in paleo-environments have significantly influenced speciation and extinction rates. These state and environment-dependent models have become popular, but important concerns have been raised (at least) for the SSE family of models that do not consider whether unrelated traits are associated with shifts in diversification rates (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Maddison and FitzJohn 2014</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Rabosky and Goldberg 2015</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">2017</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beaulieu and O’Meara 2016</a>). For example, strong correlations with diversification are sometimes inferred from rate shifts, leading to dramatically high rates of false positives even if the shift is unrelated to the targeted state (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Maddison et al. 2007</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">FitzJohn 2010</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Maddison and FitzJohn 2014</a>; <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Rabosky and Goldberg 2015</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">2017</a>). These issues may also be affecting environment-dependent correlative models in such scenarios (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lewitus and Morlon 2018</a>). Consequently, while larger trees are preferred due to the presumed increase in power, this also increases the risk of including clades that differ in factors that can affect diversification rates along a tree (ecological requirements, dispersal abilities, and life history traits [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li et al. 2018</a>]). Hence, at least for the case of SSE models, new algorithms that include “hidden states” (HiSSE) have been proposed for binary traits (BiHiSSE; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Beaulieu and O’Meara 2016</a>), and more recently geographic-dependent diversification hypotheses (GeoHiSSE; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Caetano et al. 2018</a>) and multiple states (SecSSE; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Herrera-Alsina et al. 2019</a>). Hidden states refer to unsampled traits that are related to diversification shifts in a phylogenetic tree, thus incorporating heterogeneous diversification into the original SSE.</p><p id="p-6">The HiSSE has improved the SSE model family and potentially resolved the problem of unrelated shifts in the phylogeny when appropriate sets of models are tested (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Beaulieu and O’Meara 2016</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Caetano et al. 2018</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Herrera-Alsina et al. 2019</a>). On the other hand, there are no currently available solutions for environment-dependent models (i.e. hidden states have not been implemented so far), besides splitting clades into separate analyses (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lewitus et al. 2018</a>). Even though models that do not include hidden states are not suitable for trees including groups with heterogeneous diversification patterns, they are still commonly applied in macroevolutionary studies without careful assessment. For instance, the Geographic State Speciation and Extinction model (GeoSSE; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Goldberg et al. 2011</a>) was recently used in biogeographic studies of lizards (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Esquerré et al. 2019</a>), tanagers and tortoises (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Román-Palacios and Wiens 2018</a>), plants (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Canal et al. 2019</a>), including palms (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bacon et al. 2018</a>) and oaks (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hipp et al. 2018</a>), and a recent mega-phylogeny of mushrooms (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Varga et al. 2019</a>). Given the above-noted limitations of this model, these empirical cases should be revisited (e.g. <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Harrington and Reeder 2017</a>). Here, we re-analyze the recently published phylogeny of the lizard family Liolaemidae (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Esquerré et al. 2019</a>), and show how the main conclusion can change when considering heterogeneous diversification in state- and environment-dependent hypotheses tests.</p><p id="p-7">The lizard family Liolaemidae is the most species-rich lizard clade in southern South America (307 species; Reptile Database 11 February 2019). The clade includes three genera: <em>Ctenoblepharys, Liolaemus</em> and <em>Phymaturus</em> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). <em>Ctenoblepharys</em> is a monotypic genus restricted to the coastal desert of Peru (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), whereas <em>Liolaemus</em> is the world’s richest temperate zone genus of extant amniotes (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Olave et al. 2018</a>), with 262 described species (Reptile Database 2 February 2019). This clade includes a highly diverse group of species inhabiting a wide range of different environments (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), and the sister genus <em>Phymaturus</em> (44 species; Reptile Database 2 February 2019) is distributed along both the eastern and western Andean slopes in Argentina and Chile (<em>palluma</em> clade), and through Patagonia (<em>patagonicus</em> clade). <em>Phymaturus</em> are highly specialized lizards, strictly saxicolous and largely restricted to volcanic plateaus and peaks (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cei 1986</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/995859/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/995859/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/995859" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-8" class="first-child">Summary of distribution, habitat use, diet and reproductive mode among the three Liolaemidae genera.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">The three genera have clear differences in species richness, ecological requirements, behaviors, and life histories (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). A recent macroevolutionary study found disparate patterns of diversification among the three genera (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Olave et al.</a> <em>in press</em>), while another recent biogeographic study focused on the entire clade, overlooking these striking differences between <em>Phymaturus</em> and <em>Liolaemus</em>, and the diversification rate shifts along the tree (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Esquerré et al. 2019</a>). The Esquerré et al. study represents a major contribution to evolutionary biology and herpetology in that it: (i) presents the largest Liolaemidae time-calibrated phylogeny to date (258 taxa), (ii) the most extensive compilation of habitats, altitudes, and temperature data for all taxa, and (iii) documents shifts in parity modes, including reversals from viviparity to oviparity, a transition believed to be rare. However, Esquerré et al. approach the state-dependent hypothesis using the GeoSSE to test for differences in speciation rates in Andean vs non-Andean (low elevation) species, and detected higher speciation rates in the Andean areas. They also performed correlative environment-dependent analyses in RPANDA program and found a significant exponential relationship of speciation rate with Andean uplift. Combining these results with ancestral area reconstructions, the authors infer that the Andean orogeny has acted as a “species pump” and that it is the main driver of diversification in the evolution of Liolaemidae.</p><p id="p-10">Here, we show that clade-specific differences within the Liolaemidae have affected the conclusions drawn from GeoSSE and RPANDA. We re-analyzed this dataset and show that Andean uplift-dependency is not supported as the main driver of Liolaemidae diversification, when considering heterogeneous diversification across the clade’s phylogenetic history. In this study, we (i) present an alternative explanation to Esquerré et al., to describe the diversification patterns within the Liolaemidae; and (ii) based on simulation studies, suggest recommendations that should be considered in empirical studies using the state- (HiSSE) and environment-dependent (RPANDA) models.</p></div><div class="section" id="sec-2"><h2 class="">Materials and methods</h2><p id="p-11">Here we briefly describe our methods, but for further details, see the Extended Materials and Methods in the Supplementary Material.</p><div id="sec-3" class="subsection"><h3>Identifying possible diversification shifts along the tree and exploration of the relationship between speciation rates vs. altitude</h3><p id="p-12">We explored the association between speciation (and extinction) rates with the maximum altitude of occurrence known for any species (phylogentic tree and altitude data taken from <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Esquerré et al. 2019</a>). The phylogenetic tree includes the monotypic <em>Ctenoblepharys</em>, 188 described + 11 undescribed species of <em>Liolaemus</em>, and 35 described + 23 undescribed species of <em>Phymaturus</em> (73% species coverage of all recognized Liolaemidae). We estimated speciation and extinction rates using BAMM 2.5 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rabosky et al. 2014</a>) because the method estimates rates per branch, allowing us to compare changes of these rates among the clades and species (i.e., tips) of interest. We also compared BAMM results with the recently introduced ClaDS program (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Maliet et al. 2019</a>).</p></div><div id="sec-4" class="subsection"><h3>Hypothesis testing: role of the Andean orogeny in diversification of the Liolaemidae</h3><div id="sec-5" class="subsection"><h4>Geographic Hidden-State Speciation and Extinction (GeoHiSSE)</h4><p id="p-13">We then implemented the GeoHiSSE model (Caetano et al. [2018]) using the <em>hisse</em> package (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Beaulieu and O’Meara 2016</a>). Given that we do not fully agree with the original classification of Andean and non-Andean species by <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Esquerré et al. (2019)</a>, here we propose an alternative classification, and performed these analyses using both (Table S8). As an example, the “Patagonia” group is distributed across a huge area that was assumed to be “Andean” in <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Esquerré et al. (2019)</a>, which we consider a poor classification for many species. For example, both the <em>P. patagonicus</em> and the <em>L. lineomaculatus</em> clades are restricted mainly to the lowland Patagonian steppe. Generally, the Esquerré et al. (here after the “original classification”) classification recognizes 194 Andean species, 39 non-Andean species, and 25 widespread species; here our preferred classification includes 118 Andean species, 121 non-Andean species, and 19 widespread species.</p><p id="p-14">We first evaluated the set of 35 different models proposed by <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Caetano et al. (2018)</a>, and performed simulations based on 200 random permutations of geographic distributions (Rabosky and Golberg 2015) to assess power (for details in simulations see Extended Materials and Methods). Permutations were performed by keeping the Liolaemidae phylogenetic tree, thus maintaining the complexity of the empirical scenario. This strategy allows us to assess the expected rate of distribution-dependent models receiving the highest AIC weights given the specific phylogeny of Liolaemidae. We performed all simulations twice, based on the original and our preferred geographical classification. The models include a wide range of possible scenarios, including 25 area-independent models (CID) indicating no relationship between diversification rates and geographic areas, and 10 area-dependent diversification models (all models listed in Table S7). Given the relatively high rates distribution-dependent models receiving the highest AIC weights found in our simulations (see results), and issues of overparametrization (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Caetano et al. 2018</a>), we removed some of the models from the original 35. Specifically, since some of the proposed state-dependent models have a greater number of free parameters than the most complex null models that <em>hisse</em> can currently implement (15 free parameters in the most complex CID model [M11 and M17, both including 5 hidden classes]), we suggest that excluding models with &gt;15 free parameters is important to reduce these rates. Thus, we consider only 32 models, excluding the most complex models (M10, M16 and M32, with 19, 21 and 17 free parameters, respectively).</p></div><div id="sec-6" class="subsection"><h4>Correlative environment-dependent models in RPANDA</h4><p id="p-15">Esquerré et al. tested whether speciation and extinction rates are correlated with the Andean uplift using the R package RPANDA (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Morlon et al. 2016</a>). Similar to the description above, they implemented this model on the complete phylogeny of Liolaemidae and did not analyze the two different genera into separate analyses. They fitted a total of 10 models, including eight for Andean uplift-dependency vs. two null models where rates are constant through time. We believe that this set of models is lacking in alternative scenarios, and that Andean uplift-dependency could have been selected simply because null constant rate models are unrealistic. Specifically, the study fails to include time-dependent models (see <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Morlon et al. 2011</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lewitus and Morlon 2018</a>). Time-dependent models represent scenarios of rates changing in time (not constant), but not associated to a specific environmental variable. Thus, here we corrected this analysis in two ways: (i) we separated <em>Phymaturus</em> and <em>Liolaemus</em> clades for independent analyses, because RPANDA does not consider possible rate shifts along the phylogeny, or potentially different rates across clades; and (ii) we expanded comparisons among a total of 42 models (details in Table S9), including time-dependency, as well as an alternative environmental variable based on global temperature variation during the past 67 mya (from <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zachos et al. 2001</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">2008</a>). Ambient temperature is an important environmental parameter in reptiles, given that they are ectotherms. These additional models provide a wider range of alternative scenarios beyond simply constant rates, since we incorporated other alternatives to Andean uplift-dependency beyond a simple null constant model. See extended Materials and Methods and Table S9 in Supplementary Material for further details about these analyses.</p><p>Given that our empirical results suggest that <em>Liolaemus</em> and <em>Phymaturus</em> have different diversification drivers (see results), and the fact that RPANDA is not designed to deal with such scenarios, we conducted a simulation study to explore the impact of ignoring such conditions. Here, we wanted to explore the possible outcomes of RPANDA when large trees include clades influenced by different evolutionary drivers. Given that the program is not designed to deal with such scenarios, it is interesting to know whether: (i) the two true models receive the highest AIC weights, (ii) only one true model receives the highest AIC weights, or (iii) other unrelated models are supported by high AIC weights. We simulated a total of 100 replicated trees under a pure birth model for two different clades (C1 and C2), where the C1 speciation rate is exponentially correlated with temperature (lambda = 0.2; alpha = −0.05), and C2 diversification was influenced as follows:
</p><ol class="list-romanlower " id="list-1"><li id="list-item-1"><p id="p-17">C2 with constant speciation rate (lambda = 0.12);</p></li><li id="list-item-2"><p id="p-18">C2 under time-dependent exponentially correlated speciation rate (lambda = 0.2, alpha = −0.025);</p></li><li id="list-item-3"><p id="p-19">C2 under Andean uplift-dependent exponentially correlated speciation rate (lambda = 0.045, alpha = 0.0005);</p><p id="p-20">Parameter values used for simulations were selected in order to produce approximately similar tip numbers between C1 and C2 (average ~ 220-250 each). Both C1 and C2 are set to be 40 mya old.</p></li></ol></div></div></div><div class="section" id="sec-7"><h2 class="">Results and discussion</h2><p id="p-21">See also extended results in Supplementary Materials.</p><div id="sec-8" class="subsection"><h3>Disparate patterns of diversification in the lizard family Liolaemidae: further analyses contradict previous study by <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Esquerré et al. (2019)</a></h3><div id="sec-9" class="subsection"><h4>Heterogeneous diversification within the lizard family Liolaemidae</h4><p id="p-22">BAMM estimation of speciation and extinction rates in the Liolaemidae phylogeny (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1a-b</a>) displays two shifts (PP = 0.4; Table S2), including the origin of the genus <em>Phymaturus</em> (red), and the <em>Liolaemus elongatus</em> clade (light blue). There are significant differences in speciation and extinction rates between genera (p &lt; 0.001), as clearly shown by the distributions of parameter estimations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). Specifically, the genus <em>Phymaturus</em> has the highest speciation rate that is also associated with a high extinction rate. This result is concordant with another study using a different phylogenetic tree (Olave et al. <em>in press</em>), and also supported by similar results recovered using ClaDS (Fig. S2). We note that both programs find two shifts to accelerated speciation rates involving clades that were classified entirely as “Andean” by Esquerré et al. (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Color-coded phylogenetic trees for the speciation (A) and extinction (B) rates through time for the Liolaemidae inferred by BAMM. Our preferred geographic classification of species is shown on the left tree and the original geographic classification on the right tree; corresponding to Andean, non-Andean, and widespread, shown at each tip in white, black, and gray, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1770212756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Color-coded phylogenetic trees for the speciation (A) and extinction (B) rates through time for the Liolaemidae inferred by BAMM. Our preferred geographic classification of species is shown on the left tree and the original geographic classification on the right tree; corresponding to Andean, non-Andean, and widespread, shown at each tip in white, black, and gray, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F1.medium.gif" width="431" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F1.medium.gif" width="431" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-23" class="first-child">Color-coded phylogenetic trees for the speciation (A) and extinction (B) rates through time for the Liolaemidae inferred by BAMM. Our preferred geographic classification of species is shown on the left tree and the original geographic classification on the right tree; corresponding to Andean, non-Andean, and widespread, shown at each tip in white, black, and gray, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Speciation and extinction rates obtained for Ctenoblepharys (green), Liolaemus (blue) and Phymaturus (red). The density plots are constructed using the mean obtained from each of the last 500 trees of the MCMC runs for phylogenetic estimation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1770212756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Speciation and extinction rates obtained for &lt;em&gt;Ctenoblepharys&lt;/em&gt; (green), &lt;em&gt;Liolaemus&lt;/em&gt; (blue) and &lt;em&gt;Phymaturus&lt;/em&gt; (red). The density plots are constructed using the mean obtained from each of the last 500 trees of the MCMC runs for phylogenetic estimation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F2.medium.gif" width="267" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F2.medium.gif" width="267" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-24" class="first-child">Speciation and extinction rates obtained for <em>Ctenoblepharys</em> (green), <em>Liolaemus</em> (blue) and <em>Phymaturus</em> (red). The density plots are constructed using the mean obtained from each of the last 500 trees of the MCMC runs for phylogenetic estimation.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/995853/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/995853/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/995853" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-25" class="first-child">Species counts for the geographic classification from <a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">Esquerré et al. (2019)</a>; taken from their supplementary material. Note that some species were classified as both Andean and non-Andean as “widespread” (classification details in Table S8).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Exploration of the relationship between speciation and extinction rates vs. maximum altitude</h4><p id="p-26">We constructed linear models between the maximum altitude (MA) of species occurrence records, and the species-specific speciation and extinction rates. Linear models reveal nonsignificant correlations of the speciation/extinction rates with the MA for all target clades (Table S5). Scatter plots show no relationship between rates and MA (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>; Figs. S4-S10).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetically controlled linear regressions of the log transformed speciation/extinction rates as a function of the maximum altitude of the species occurrence, for different target clades: Phymaturus genus, Eulaemus subgenus, Liolaemus sensu stricto (s.s.) subgenus when excluding the L. elongatus clade, and with the L. elongatus clade. See also the Figures S3-S9 for more regressions, and Table S5 for full results of the linear models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1770212756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetically controlled linear regressions of the log transformed speciation/extinction rates as a function of the maximum altitude of the species occurrence, for different target clades: &lt;em&gt;Phymaturus&lt;/em&gt; genus, &lt;em&gt;Eulaemus&lt;/em&gt; subgenus, &lt;em&gt;Liolaemus&lt;/em&gt; sensu stricto (s.s.) subgenus when excluding the &lt;em&gt;L. elongatus&lt;/em&gt; clade, and with the &lt;em&gt;L. elongatus&lt;/em&gt; clade. See also the Figures S3-S9 for more regressions, and Table S5 for full results of the linear models.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F3.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F3.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-27" class="first-child">Phylogenetically controlled linear regressions of the log transformed speciation/extinction rates as a function of the maximum altitude of the species occurrence, for different target clades: <em>Phymaturus</em> genus, <em>Eulaemus</em> subgenus, <em>Liolaemus</em> sensu stricto (s.s.) subgenus when excluding the <em>L. elongatus</em> clade, and with the <em>L. elongatus</em> clade. See also the Figures S3-S9 for more regressions, and Table S5 for full results of the linear models.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h4>Incorporating heterogeneous diversification: GeoHiSSE results</h4><p id="p-28">Speciation and extinction rate estimates generated by GeoHiSSE are highly congruent with BAMM and ClaDS estimates, with higher rates in <em>Phymaturus</em> than <em>Liolaemus</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">4</a> and S2). Note that GeoHiSSE model selection was different when considering the two classifications (Table S7). Following our preferred classification (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>), the M6 (CID—GeoHiSSE, two hidden rate classes) was associated to the greatest AIC weight (=0.5), followed by M12 (CID—GeoHiSSE extirpation, two hidden rate classes; AIC weight = 0.29). Other CID models showed lower support (M3, M19, M9, M20, M5 and M11; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4</a>; Table S7). On the other hand, results based on the original geographic classification by Esquerré et al. (Table S7), the M3 (CID—GeoHiSSE, three hidden rate classes) are associated to the highest weight (= 0.47), followed by M18 (CID—anagenetic GeoHiSSE, two hidden rate classes; weight = 0.22) and M19 (CID - GeoHiSSE, four rate classes; weight = 0.17). Other models received little support (weight &lt; 0.05; M4, M5, M9 and M20).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Left, AIC weights of models fitted to the empirical dataset. Right, color-coded phylogenetic tree reflecting the speciation rates (blue-red scale) and geographic-states (whiteblack scale) obtained after averaging parameters of the GeoHiSSE analyses. Results here correspond to those obtained based on our preferred classification. See Table S7 for further result details." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1770212756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Left, AIC weights of models fitted to the empirical dataset. Right, color-coded phylogenetic tree reflecting the speciation rates (blue-red scale) and geographic-states (whiteblack scale) obtained after averaging parameters of the GeoHiSSE analyses. Results here correspond to those obtained based on our preferred classification. See Table S7 for further result details.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F4.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F4.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-29" class="first-child">Left, AIC weights of models fitted to the empirical dataset. Right, color-coded phylogenetic tree reflecting the speciation rates (blue-red scale) and geographic-states (whiteblack scale) obtained after averaging parameters of the GeoHiSSE analyses. Results here correspond to those obtained based on our preferred classification. See Table S7 for further result details.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">We have demonstrated that the genera <em>Phymaturus</em> and <em>Liolaemus</em> display clear disparate patterns of diversification with two clear shifts along the phylogeny (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>, <a id="xref-fig-4-4" class="xref-fig" href="#F4">4</a> and S2) and that, when incorporating the HiSSE into the models, results show no apparent signal of Andean uplift increasing speciation rates in the Liolaemidae (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4</a>; Table S7). This is true in both cases, when considering either our preferred geographic classification or the original classification (Table S7). These results show that Esquerré et al. have confounded clade-specific rate accelerations in their distribution-dependent diversification results.</p><p id="p-31">In addition, the GeoHiSSE analyses also return estimates of ancestral distributions (white-black color gradient in phylogenetic tree in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4</a>). Given the fact that we have changed the geographic classification, these results show that our preferred classification contradicts previous findings by Esquerré et al., using BioGeoBEARS (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Matzke 2016</a>) for ancestral reconstructions. This program returned results showing that ancestral lineages in main clades of Liolaemidae were predominantly Andean, leading Esquerré et al. to conclude that the Andes have acted as a “species pump”, and reinforced the idea that Andean uplift has been a key driver in Liolaemidae diversification. In comparison, our preferred classification now shows that the ancestral <em>Liolaemus</em> was most likely “widespread” (i.e., both Andean and non-Andean), and that <em>Phymaturus</em> is inferred to have a non-Andean ancestor (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4</a>). The ancestors of the other subgenera are also inferred to be non-Andean or widespread (<em>Eulaemus:</em> widespread, <em>Liolaemus s.s.:</em> widespread, <em>P. palluma:</em> widespread; and <em>P. patagonicus</em>: non-Andean; <a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4</a>). These results also argue against the Andes acting as “species pump”.</p></div><div id="sec-12" class="subsection"><h4>Correlative environment-dependent models in RPANDA applied to study the lizard family Liolaemidae</h4><p id="p-32">We fitted 42 models to each genus within Liolaemidae using RPANDA. This algorithm does not incorporate “hidden states”, so the only proper way to approach such heterogeneous scenarios is by running separate analyses for each genus (but see extended results in Supplementary Materials and Table S9 for analyses of Liolaemidae as a whole). The best model to describe evolutionary rates in <em>Liolaemus</em> is a birth-death model in which speciation and extinction rates are linearly correlated with time (i.e. our model 18; Table S9; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>). This model received an AIC weight average of 0.26 ±0.17 (lambda = 0.098, alpha = 0.14; Table S9), followed by a model in which speciation and extinction rates are linearly correlated with global temperature changes (AIC weight mean = 0.08 ± 0.07; model 15; see Table S9 for details of other models that received little support). In contrast, the best model to describe evolutionary rates in <em>Phymaturus</em> corresponds to speciation rates exponentially correlated with the interaction between time and global temperature changes (model 19; AIC weight mean = 0.11 ± 0.03; lambda = 1.03, alpha = −0.6; <a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5B</a>), followed by another model including an exponential dependency of the interaction between time and global temperature changes (AIC weight mean = 0.06 ± 0.04; model 25). In addition, other models received support and a higher dispersion of estimated AIC weights, reflecting increased uncertainty in <em>Phymaturus</em> relative to <em>Liolaemus</em>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A-B) Top 5 models (highest AIC weight means) estimated by RPANDA among 42 models for (A) Liolaemus genus and (B) Phymaturus genus. All detailed results are presented in Table S9. (C) Global temperature changes through time (gray dots) and a smooth spline in black. Speciation rates estimated by BAMM were log transformed for Liolaemus (blue) and Phymaturus (red)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1770212756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A-B) Top 5 models (highest AIC weight means) estimated by RPANDA among 42 models for (A) &lt;em&gt;Liolaemus&lt;/em&gt; genus and (B) &lt;em&gt;Phymaturus&lt;/em&gt; genus. All detailed results are presented in Table S9. (C) Global temperature changes through time (gray dots) and a smooth spline in black. Speciation rates estimated by BAMM were log transformed for &lt;em&gt;Liolaemus&lt;/em&gt; (blue) and &lt;em&gt;Phymaturus&lt;/em&gt; (red).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F5.medium.gif" width="440" height="190"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F5.medium.gif" width="440" height="190"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-33" class="first-child">(A-B) Top 5 models (highest AIC weight means) estimated by RPANDA among 42 models for (A) <em>Liolaemus</em> genus and (B) <em>Phymaturus</em> genus. All detailed results are presented in Table S9. (C) Global temperature changes through time (gray dots) and a smooth spline in black. Speciation rates estimated by BAMM were log transformed for <em>Liolaemus</em> (blue) and <em>Phymaturus</em> (red).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h4>Disparate patterns of diversification in the lizard family Liolaemidae</h4><p id="p-34">Interestingly, even though <em>Liolaemus</em> has extraordinary species richness (262 described species), our analyses show that its sister group <em>Phymaturus</em> evolved under higher speciation rates (44 described species + 23 candidate species included here). Increased speciation rates in <em>Phymaturus</em> are best explained by decreasing global temperatures (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5b</a>). Previous studies have shown that the thermal biology of the cold-adapted <em>Phymaturus</em> genus is remarkably similar across species, suggesting that it may be evolutionarily or ecologically constrained (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ibargüengoytía et al. 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cruz et al. 2009</a>). Thus, it is biologically plausible that once the temperature conditions became more favorable for this group, then <em>Phymaturus’</em> speciation rates increased (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5b; c</a>). Different abiotic factors have been suggested to affect South American climates, including Andean uplift as one of the possible promotors (e.g. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gregory-Wodzicki 2000</a>; Blisniuk et al. 2005). Is it possible that the Andes uplift has indirectly driven the increase in speciation rates in <em>Phymaturus</em> by promoting local climate change? There has been considerable debate over the role of the Andes relative to South American climate change, including whether the Andes promoted climate change, or global climate change promoted the Andean uplift (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lamb and Davis 2003</a>). However, post-Cenozoic temperature decrease was a global phenomenon (Zachos et al. 2011; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">2008</a>), and a combination of relevant factors also acted as locally cooling promotors, such as the Humboldt Current generated by the closure of the Central America Seaway (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hartley 2003</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Garreaud et al. 2010</a>). We emphasize that multiple factors have likely driven <em>Phymaturus</em> evolution, and assuming a temperature-only dependent diversification is unnecessarily reductionist. We also cannot discard other important aspects in the peculiar biology of this genus. For example, all &gt; 40 cold-adapted species only occur in isolated patches of rock outcrops; therefore, migration between populations is likely severely limited relative to <em>Liolaemus</em> (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Vicenzi et al. 2017</a>). The strong fidelity of <em>Phymaturus</em> species to specific microhabitats, “islands” of big boulders with deep crevices in volcanic cliffs, peaks and plateaus (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cei 1986</a>), might have promoted speciation over short periods of time, as is the case in other lizard species in both wild and urban populations (e.g. <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Templeton et al. 2001</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Thompson et al. 2018</a>).</p><p id="p-35">Specialization in <em>Phymaturus</em> has been discussed in other studies (e.g. <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Reaney et al. 2018</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Marín-Gonzalez et al. 2018</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Olave et al.</a> <em>in press</em>); specialization may be advantageous if it results in efficient selection for adaptation to a stable and narrow niche, reducing the cost of trade-offs by abandoning traits needed to utilize a wider range of resources (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Futuyma and Moreno 1988</a>). However, these shorter term microevolutionary benefits may come at the cost of longer term macroevolutionary success (e.g. <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agnarsson et al. 2006</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anacker et al. 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Forister et al. 2012</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Armbruster 2014</a>). Thus, it is possible that the same factors promoting speciation in <em>Phymaturus</em> might also explain its high extinction rates as a trade-off (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">2</a>). The contrasting biology between the sister groups <em>Liolaemus</em> and <em>Phymaturus</em> (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) makes the Liolaemidae an extraordinarily rich clade in which to explore the cost-benefits of lineages becoming generalists vs. specialists, and their roles shaping macroevolutionary dynamics (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Olave et al</a>. <em>in press</em>).</p></div></div><div id="sec-14" class="subsection"><h3>The importance of considering lineage diversification heterogeneity in macroevolutionary studies: warnings in implementation of GeoHiSSE and RPANDA programs</h3><div id="sec-15" class="subsection"><h4>Assessing the power of GeoHiSSE models: are hidden states improving state-dependent diversification models?</h4><p id="p-36">Previously, it has been shown that including hidden states to the model is a plausible solution to account for diversification heterogeneity and improve the accuracy of the original SSE models (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Caetano et al. 2018</a>). We assessed the power of GeoHiSSE analyses by simulating random permutations of geographic areas, and show that most of the CID models received the highest weights over distribution-dependent models (~85%). This is undoubtedly an improvement over the high level of false positives reported for the original SSE models (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Rabosky and Goldberg 2015</a>; 2017). However, we still found a relatively high rate of distribution-dependent models receiving the highest weights (= 15%) when considering all 35 models and permutations based on the original geographic classifications (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6a</a>), and 16% with simulations based on our preferred classification (Table S7). Interestingly, all state-dependent models receiving the highest weights are equal or greater than the maximum number of free parameters (= 15 - 21) in the most complex CID model (M11 and M17). Further, most of these models are supported by high AIC weights (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6b</a>). We show that excluding models with &gt;15 free parameters (i.e. selection among 32 models) reduced these rates to 7% (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Fig. 6c</a>) following the original classification, and to a ~10% rate given the permutations on our preferred classification (Table S7; see also extended results in Supplementary Materials for further details).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AIC weights among replicated simulations based on the original classification. First row corresponds to analyses performed among 35 models, while the second row corresponds to analyses after removing distribution-dependent models with more than 15 free parameters (32 models total). Left column displays AIC weight per replicate for each model. Right column displays the AIC weight of the different models per each replicate, for which a distribution-dependent model received the highest AIC weights. Red triangles represent geographic-dependent models and black dots geographic independent models (CID). See Table S7 for further result details." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1770212756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;AIC weights among replicated simulations based on the original classification. First row corresponds to analyses performed among 35 models, while the second row corresponds to analyses after removing distribution-dependent models with more than 15 free parameters (32 models total). Left column displays AIC weight per replicate for each model. Right column displays the AIC weight of the different models per each replicate, for which a distribution-dependent model received the highest AIC weights. Red triangles represent geographic-dependent models and black dots geographic independent models (CID). See Table S7 for further result details.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F6.medium.gif" width="440" height="355"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F6.medium.gif" width="440" height="355"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <p id="p-37" class="first-child">AIC weights among replicated simulations based on the original classification. First row corresponds to analyses performed among 35 models, while the second row corresponds to analyses after removing distribution-dependent models with more than 15 free parameters (32 models total). Left column displays AIC weight per replicate for each model. Right column displays the AIC weight of the different models per each replicate, for which a distribution-dependent model received the highest AIC weights. Red triangles represent geographic-dependent models and black dots geographic independent models (CID). See Table S7 for further result details.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Given the recent documentation that a proper set of state-independent models for HiSSE methods should, necessarily, have the same number of free diversification parameters than the state-dependent models for a “fair” comparison (<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Caetano et al. 2018</a>), we think that the original set of 35 models should be taken with caution. This issue was paradoxically shown in <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Caetano et al. (2018)</a> for BiHiSSE implementation, but not corrected among the 35 proposed models for GeoHiSSE in the same paper. Our study does not mean that the same power estimations will necessarily apply to another empirical phylogeny, but we have demonstrated the importance of assessing power before applying HiSSE models, as well as reinforced the notion of potential risks related to the set of models (and number of free parameters) implemented for HiSSE hypothesis tests.</p></div><div id="sec-16" class="subsection"><h4>Impact of heterogeneous trees in RPANDA: trees including clades following different dependent-drivers affect results</h4><p id="p-39">Inspired in our empirical results, we performed a simulation study to address how RPANDA analyses could be affected when two different clades in a phylogenetic tree evolve following different drivers. An extensive simulation study to test the performance of RPANDA was performed by <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Lewitus and Morlon (2018)</a>, including exploration of different proportions of missing tips, accurate model selection, and also rate shifts along the phylogeny. While the authors show that RPANDA seem to be robust when rate shifts are present along the phylogeny under their simulated conditions, here we focused on a different type of heterogeneous scenario.</p><p id="p-40">Our results show that, even though RPANDA can accurately recover the true scenario when there is a single driver for speciation rates in a phylogenetic tree (Fig. S13; Table S10), there is a range of different outcomes when each clade evolves in response to different drivers of diversification, including selecting partially correct models to completely misleading results (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a>). On the one hand, in a scenario where one clade follows a temperature-dependent evolution, while the other clade has constant rates (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Fig. 7</a>, left; Table S10), the models with highest AIC weights are two temperature-dependent models (model 4 and 2; model 2 is the true model), followed by a constant rate model (model 1; also a true model). Here, recovering the true models with high support might represent the best possible outcome. However, our second set of simulations included a temperature-dependent + time-dependent trees, and the best-supported models are two unrelated Andean uplift-dependent scenarios (model 2b and 4b; <a id="xref-fig-7-3" class="xref-fig" href="#F7">Fig. 7</a>, middle; Table S10). Further, our 3<sup>rd</sup> set of simulations included trees evolving following two different environmental variables (temperature + Andean uplift; <a id="xref-fig-7-4" class="xref-fig" href="#F7">Fig. 7</a>, right; Table S10), and also returns the highest support for an unrelated model (time-dependent; model 3), followed by a temperature-dependent model (model 4; however not the true model).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AIC weight obtained for the simulation studies including heterogeneous scenarios analyzed in RPANDA program. Top 5 models (highest AIC weight means) are shown for the different simulated scenarios, including trees comprising one clade (C1) evolving under temperature-dependency plus a second clade (C2) evolving either (i) under constant speciation rates (left), (ii) with speciation rates exponentially correlated with time (middle) and (iii) with speciation rates exponentially correlated with the Andes uplift as a second environmental variable (right). Further results are shown in Table S10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1770212756" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;AIC weight obtained for the simulation studies including heterogeneous scenarios analyzed in RPANDA program. Top 5 models (highest AIC weight means) are shown for the different simulated scenarios, including trees comprising one clade (C1) evolving under temperature-dependency plus a second clade (C2) evolving either (i) under constant speciation rates (left), (ii) with speciation rates exponentially correlated with time (middle) and (iii) with speciation rates exponentially correlated with the Andes uplift as a second environmental variable (right). Further results are shown in Table S10.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F7.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F7.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/14/563635/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7:</span> <p id="p-41" class="first-child">AIC weight obtained for the simulation studies including heterogeneous scenarios analyzed in RPANDA program. Top 5 models (highest AIC weight means) are shown for the different simulated scenarios, including trees comprising one clade (C1) evolving under temperature-dependency plus a second clade (C2) evolving either (i) under constant speciation rates (left), (ii) with speciation rates exponentially correlated with time (middle) and (iii) with speciation rates exponentially correlated with the Andes uplift as a second environmental variable (right). Further results are shown in Table S10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Our results highlight clear potential risks of ignoring such heterogeneous diversification histories in RPANDA. Currently, the only way to circumvent such an issue is to recognize potential differences <em>a priori</em>, and separate clades into different analyses. However, while hidden states are not incorporated in RPANDA, it is clearly difficult in practice to recognize which clades should be separated, since identifying speciation rate shifts might not be sufficient. Specifically, two clades with similar rates might follow different environmental drivers. Thus, recognizing clades with important biological differences is key to assess potential differences <em>a priori</em>. As we have shown here, the two contrasting groups of <em>Liolaemus</em> and <em>Phymaturus</em> not only differ in speciation rates, but also in several aspects of their biology (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). Such biological differences are expected to affect the macroevolutionary patterns (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Li et al. 2018</a>). Empirical studies must explore such possibilities when using models that do not allow heterogeneous trees, as is it the case of the current version of RPANDA.</p></div></div></div><div class="section" id="sec-17"><h2 class="">Conclusions</h2><p id="p-43">Incorporating large trees for macroevolutionary studies has the advantage of providing larger datasets, and presumably more power. However, it is important to keep in mind the value of incorporating heterogeneous diversification rates into models for state- and environmentdependent hypothesis testing. We disagree with the main conclusion of Esquerré et al. that the Andes uplift was the main driver of the diversification of these lizards, by showing that: (i) there is no support for increased speciation rate in Andean species (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig. 4</a>, S7), (ii) a timedependent model better explains diversification of the genus <em>Liolaemus</em> than Andean uplift (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5</a>; Table S9), and (iii) a model considering the interaction between time and global temperature changes is a better fit for the genus <em>Phymaturus</em> than Andean uplift (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5</a>; Table S9). As a final remark, our preferred classification of species distributions has now changed inferences in ancestral-range reconstructions, and suggests that ancestors of both genera and all four subgenera were non-Andean or widespread (i.e. both Andean and non-Andean; <a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig. 4</a>), thus challenging the Andean “species pump” hypothesis. However, we do not attempt to invalidate the whole work presented by Esquerré et al. As mentioned before, they have presented many other relevant findings about the diversification history of Liolaemidae.</p><p id="p-44">Finally, our study calls attention to the importance of considering heterogenous diversification when implementing state- and environment-dependent hypotheses tests. Specifically, we have demonstrated that trees including clades following different dependent-drivers affect RPANDA results (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Fig. 7</a>), and the only way to circumvent such problem is to separate clades for independent analyses, given that this method does not account for hidden states. We also provide recommendations for the implementation of GeoHiSSE models to account for fair comparisons among models included in the analysis. We reinforce the previously known case of removing the distribution-dependent over-parameterized models in GeoHiSSE (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Fig. 6</a>), to prevent exceeding the number of free parameters in the most complex null model.</p><p id="p-45">We note that, while here we focused on the recently published Liolaemidae phylogeny as a model system, it is very likely that many earlier empirical studies may have limitations similar to those we report here, and these should be re-assessed.</p></div><div class="section" id="sec-19"><h2 class="">Author contributions</h2><p id="p-47">MO designed the study and carried out the analyses. LJA and MM provided recommendations based on the biology of the focal organism. MO, MM and JWS wrote and edited the manuscript. All authors read and approved the final manuscript.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">We thank Richard Ree and two anonymous reviewers for useful comments made on earlier versions of this article. We also thank D. Esquerré for providing clarification on how they performed their analyses, and useful comments made on earlier versions of this manuscript. We thank all members of the Grupo de Herpetología Patagónica (IPEEC-CONICET) for continuing support. Financial support was provided by ANPCYT-FONCYT 1252/2015 (MM), and a postdoctoral fellowship (MO) from the Alexander von Humboldt Foundation at the University of Konstanz, Germany.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><span class="em-link"><span class="em-addr">avila{at}cenpat-conicet.gob.ar</span></span> (LJA); <span class="em-link"><span class="em-addr">jack_sites{at}byu.edu</span></span> (JWS); <span class="em-link"><span class="em-addr">morando{at}cenpat-conicet.gob.ar</span></span> (MM).</p></li><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-4">Major changes: We have now incorporated RPANDA analyses</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.1" data-doi="10.1111/j.0014-3820.2006.tb01869.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Agnarsson</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avilés</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coddington</span>,  <span class="cit-name-given-names">J. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Maddison</span>,  <span class="cit-name-given-names">W. P</span></span>. (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Sociality in theridiid spiders: repeated origins of an evolutionary dead end</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">60</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">2342</span>–<span class="cit-lpage">2351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DAgnarsson%26rft.auinit1%253DI.%26rft.volume%253D60%26rft.issue%253D11%26rft.spage%253D2342%26rft.epage%253D2351%26rft.atitle%253DSociality%2Bin%2Btheridiid%2Bspiders%253A%2Brepeated%2Borigins%2Bof%2Ban%2Bevolutionary%2Bdead%2Bend.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2006.tb01869.x%26rft_id%253Dinfo%253Apmid%252F17236425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2006.tb01869.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17236425&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243031000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Armbruster</span>,  <span class="cit-name-given-names">W. S.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Floral specialization and angiosperm diversity: phenotypic divergence, fitness tradeoffs and realized pollination accuracy</span>. <abbr class="cit-jnl-abbrev">AoB Plants</abbr>, <span class="cit-vol">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.3" data-doi="10.1111/j.1558-5646.2010.01114.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anacker</span>,  <span class="cit-name-given-names">B. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whittall</span>,  <span class="cit-name-given-names">J. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldberg</span>,  <span class="cit-name-given-names">E. E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">S. P.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Origins and consequences of serpentine endemism in the California flora</span>. <abbr class="cit-jnl-abbrev">Evolution: International Journal of Organic Evolution</abbr>, <span class="cit-vol">65</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">365</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DAnacker%26rft.auinit1%253DB.%2BL.%26rft.volume%253D65%26rft.issue%253D2%26rft.spage%253D365%26rft.epage%253D376%26rft.atitle%253DOrigins%2Band%2Bconsequences%2Bof%2Bserpentine%2Bendemism%2Bin%2Bthe%2BCalifornia%2Bflora.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2010.01114.x%26rft_id%253Dinfo%253Apmid%252F20812977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2010.01114.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20812977&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286629100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bacon</span>,  <span class="cit-name-given-names">C. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Velásquez-Puentes</span>,  <span class="cit-name-given-names">F. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoorn</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Antonelli</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Iriarteeae palms tracked the uplift of Andean Cordilleras</span>. <abbr class="cit-jnl-abbrev">Journal of biogeography</abbr>, <span class="cit-vol">45</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1653</span>–<span class="cit-lpage">1663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiogeography%26rft.volume%253D45%26rft.spage%253D1653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.5" data-doi="10.1093/sysbio/syw022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>,  <span class="cit-name-given-names">J. M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">O’Meara</span>,  <span class="cit-name-given-names">B. C.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Detecting hidden diversification shifts in models of trait-dependent speciation and extinction</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, <span class="cit-vol">65</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">583</span>–<span class="cit-lpage">601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyw022%26rft_id%253Dinfo%253Apmid%252F27016728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syw022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27016728&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-563635v3.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blisniuk</span>,  <span class="cit-name-given-names">P. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stern</span>,  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chamberlain</span>,  <span class="cit-name-given-names">C. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeitler</span>,  <span class="cit-name-given-names">P. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramos</span>,  <span class="cit-name-given-names">V. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sobel</span>,  <span class="cit-name-given-names">E. R.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Warkus</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-chapter-title">Links between mountain uplift, climate, and surface processes in the southern Patagonian Andes</span>. In <span class="cit-source">The Andes</span> (pp. <span class="cit-fpage">429</span>–<span class="cit-lpage">440</span>). <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Caetano</span>,  <span class="cit-name-given-names">D. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Meara</span>,  <span class="cit-name-given-names">B. C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>,  <span class="cit-name-given-names">J. M.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Hidden state models improve state-dependent diversification approaches, including biogeographical models</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">72</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">2308</span>–<span class="cit-lpage">2324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D72%26rft.spage%253D2308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Canal</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Köster</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Celis</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Croat</span>,  <span class="cit-name-given-names">T. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borsch</span>,  <span class="cit-name-given-names">T.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">K. E.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Out of Amazonia and Back again: Historical Biogeography of the Species-Rich Neotropical Genus Philodendron (Araceae) 1</span>. <abbr class="cit-jnl-abbrev">Annals of the Missouri Botanical Garden</abbr>, <span class="cit-vol">104</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">49</span>–<span class="cit-lpage">68</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BMissouri%2BBotanical%2BGarden%26rft.volume%253D104%26rft.spage%253D49%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cei</span>,  <span class="cit-name-given-names">J. M.</span></span> (<span class="cit-pub-date">1986</span>). <span class="cit-article-title">Reptiles del centro, centro-oeste y sur de la Argentina: Herpetofauna de las zonas áridas y semiáridas</span>. <abbr class="cit-jnl-abbrev">Torino: Museo Regionale di Scienze Naturali, Monografie IV</abbr>: <span class="cit-fpage">1</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Condamine</span>,  <span class="cit-name-given-names">F. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rolland</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Morlon</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Macroevolutionary perspectives to environmental change</span>. <abbr class="cit-jnl-abbrev">Ecology letters</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">72</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2Bletters%26rft.volume%253D16%26rft.spage%253D72%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.11" data-doi="10.1111/j.1461-0248.2011.01737.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Condamine</span>,  <span class="cit-name-given-names">F. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperling</span>,  <span class="cit-name-given-names">F. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wahlberg</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasplus</span>,  <span class="cit-name-given-names">J. Y.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Kergoat</span>,  <span class="cit-name-given-names">G. J.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">What causes latitudinal gradients in species diversity? Evolutionary processes and ecological constraints on swallowtail biodiversity</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">15</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">267</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DCondamine%26rft.auinit1%253DF.%2BL.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D267%26rft.epage%253D277%26rft.atitle%253DWhat%2Bcauses%2Blatitudinal%2Bgradients%2Bin%2Bspecies%2Bdiversity%253F%2BEvolutionary%2Bprocesses%2Band%2Becological%2Bconstraints%2Bon%2Bswallowtail%2Bbiodiversity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2011.01737.x%26rft_id%253Dinfo%253Apmid%252F22251895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2011.01737.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22251895&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cruz</span>,  <span class="cit-name-given-names">F. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belver</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acosta</span>,  <span class="cit-name-given-names">J. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Villavicencio</span>,  <span class="cit-name-given-names">H. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanco</span>,  <span class="cit-name-given-names">G.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Cánovas</span>,  <span class="cit-name-given-names">M. G.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Thermal biology of Phymaturus lizards: evolutionary constraints or lack of environmental variation?</span>. <abbr class="cit-jnl-abbrev">Zoology</abbr>, <span class="cit-vol">112</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">425</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoology%26rft.volume%253D112%26rft.spage%253D425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Esquerré</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brennan</span>,  <span class="cit-name-given-names">I. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Catullo</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torres-Pérez</span>,  <span class="cit-name-given-names">F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Keogh</span>,  <span class="cit-name-given-names">J. S.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">How mountains shape biodiversity: The role of the Andes in biogeography, diversification, and reproductive biology in South America’s most species-rich lizard radiation (Squamata: Liolaemidae)</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">73</span>, <span class="cit-fpage">214</span>–<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D73%26rft.spage%253D214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.14" data-doi="10.1093/sysbio/syq053"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">FitzJohn</span>,  <span class="cit-name-given-names">R. G.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Quantitative traits and diversification</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, <span class="cit-vol">59</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">619</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyq053%26rft_id%253Dinfo%253Apmid%252F20884813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syq053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20884813&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283686000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">FitzJohn</span>,  <span class="cit-name-given-names">R. G.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Diversitree: comparative phylogenetic analyses of diversification in R</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1084</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D1084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.16" data-doi="10.1890/11-0650.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forister</span>,  <span class="cit-name-given-names">M. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dyer</span>,  <span class="cit-name-given-names">L. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singer</span>,  <span class="cit-name-given-names">M. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stireman</span>   <span class="cit-name-suffix">III</span>,  <span class="cit-name-given-names">J. O.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Lill</span>,  <span class="cit-name-given-names">J. T.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Revisiting the evolution of ecological specialization, with emphasis on insect–plant interactions</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-vol">93</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">981</span>–<span class="cit-lpage">991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DForister%26rft.auinit1%253DM.%2BL.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D981%26rft.epage%253D991%26rft.atitle%253DRevisiting%2Bthe%2Bevolution%2Bof%2Becological%2Bspecialization%252C%2Bwith%2Bemphasis%2Bon%2Binsect-plant%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F11-0650.1%26rft_id%253Dinfo%253Apmid%252F22764485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/11-0650.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22764485&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304368100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.17" data-doi="10.1146/annurev.es.19.110188.001231"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Futuyma</span>,  <span class="cit-name-given-names">D. J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Moreno</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">1988</span>). <span class="cit-article-title">The evolution of ecological specialization</span>. <abbr class="cit-jnl-abbrev">Annual review of Ecology and Systematics</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">207</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DFutuyma%26rft.auinit1%253DD.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D207%26rft.epage%253D233%26rft.atitle%253DThe%2BEvolution%2BOf%2BEcological%2BSpecialization%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.19.110188.001231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.es.19.110188.001231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988R137700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.18" data-doi="10.1016/j.epsl.2010.01.017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garreaud</span>,  <span class="cit-name-given-names">R. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Molina</span>,  <span class="cit-name-given-names">A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Farias</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Andean uplift, ocean cooling and Atacama hyperaridity: A climate modeling perspective</span>. <abbr class="cit-jnl-abbrev">Earth and Planetary Science Letters</abbr>, <span class="cit-vol">292</span>(<span class="cit-issue">1-2</span>), <span class="cit-fpage">39</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEarth%2Band%2BPlanetary%2BScience%2BLetters%26rft.stitle%253DEarth%2Band%2BPlanetary%2BScience%2BLetters%26rft.volume%253D292%26rft.issue%253D1-2%26rft.spage%253D39%26rft.epage%253D50%26rft.atitle%253DAndean%2Buplift%252C%2Bocean%2Bcooling%2Band%2BAtacama%2Bhyperaridity%253B%2Ba%2Bclimate%2Bmodeling%2Bperspective%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.epsl.2010.01.017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.epsl.2010.01.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2010057338&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000276138900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.19" data-doi="10.1130/0016-7606(2000)112&lt;1091:UHOTCA&gt;2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gregory-Wodzicki</span>,  <span class="cit-name-given-names">K. M.</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Uplift history of the Central and Northern Andes: a review</span>. <abbr class="cit-jnl-abbrev">Geological society of America bulletin</abbr>, <span class="cit-vol">112</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1091</span>–<span class="cit-lpage">1105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeological%2BSociety%2Bof%2BAmerica%2BBulletin%26rft.stitle%253DGeological%2BSociety%2Bof%2BAmerica%2BBulletin%26rft.aulast%253DGregory-Wodzicki%26rft.auinit1%253DK.%2BM.%26rft.volume%253D112%26rft.issue%253D7%26rft.spage%253D1091%26rft.epage%253D1105%26rft.atitle%253DUplift%2Bhistory%2Bof%2Bthe%2BCentral%2Band%2BNorthern%2BAndes%253A%2BA%2Breview%26rft_id%253Dinfo%253Adoi%252F10.1130%252F0016-7606%25282000%2529112%253C1091%253AUHOTCA%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZ3NhYnVsbCI7czo1OiJyZXNpZCI7czoxMDoiMTEyLzcvMTA5MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzE0LzU2MzYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.20" data-doi="10.1111/j.1558-5646.2012.01730.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>,  <span class="cit-name-given-names">E. E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Igić</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Tempo and mode in plant breeding system evolution</span>. <abbr class="cit-jnl-abbrev">Evolution: International Journal of Organic Evolution</abbr>, <span class="cit-vol">66</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">3701</span>–<span class="cit-lpage">3709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DGoldberg%26rft.auinit1%253DE.%2BE.%26rft.volume%253D66%26rft.issue%253D12%26rft.spage%253D3701%26rft.epage%253D3709%26rft.atitle%253DTempo%2Band%2Bmode%2Bin%2Bplant%2Bbreeding%2Bsystem%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01730.x%26rft_id%253Dinfo%253Apmid%252F23206129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01730.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23206129&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312218200005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.21" data-doi="10.1093/sysbio/syr046"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldberg</span>,  <span class="cit-name-given-names">E. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lancaster</span>,  <span class="cit-name-given-names">L. T.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ree</span>,  <span class="cit-name-given-names">R. H.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Phylogenetic inference of reciprocal effects between geographic range evolution and diversification</span>. <abbr class="cit-jnl-abbrev">Systematic Biology</abbr>, <span class="cit-vol">60</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">451</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyr046%26rft_id%253Dinfo%253Apmid%252F21551125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syr046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21551125&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harrington</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Reeder</span>,  <span class="cit-name-given-names">T. W.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Rate heterogeneity across Squamata, misleading ancestral state reconstruction and the importance of proper null model specification</span>. <abbr class="cit-jnl-abbrev">Journal of evolutionary biology</abbr>, <span class="cit-vol">30</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">313</span>–<span class="cit-lpage">325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.volume%253D30%26rft.spage%253D313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.23" data-doi="10.1144/0016-764902-083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hartley</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Andean uplift and climate change</span>. <abbr class="cit-jnl-abbrev">Journal of the Geological Society</abbr>, <span class="cit-vol">160</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">7</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BGeological%2BSociety%26rft.stitle%253DJournal%2Bof%2Bthe%2BGeological%2BSociety%26rft.aulast%253DHARTLEY%26rft.auinit1%253DA.%26rft.volume%253D160%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D10%26rft.atitle%253DAndean%2Buplift%2Band%2Bclimate%2Bchange%26rft_id%253Dinfo%253Adoi%252F10.1144%252F0016-764902-083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamdzIjtzOjU6InJlc2lkIjtzOjc6IjE2MC8xLzciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8xNC81NjM2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Herrera-Alsina</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Els</span>,  <span class="cit-name-given-names">P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Etienne</span>,  <span class="cit-name-given-names">R. S.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Detecting the dependence of diversification on multiple traits from phylogenetic trees and trait data</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, <span class="cit-vol">68</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">317</span>–<span class="cit-lpage">328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft.volume%253D68%26rft.spage%253D317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hipp</span>,  <span class="cit-name-given-names">A. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manos</span>,  <span class="cit-name-given-names">P. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">González-Rodríguez</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaproth</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McVay</span>,  <span class="cit-name-given-names">J. D.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Cavender-Bares</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Sympatric parallel diversification of major oak clades in the Americas and the origins of Mexican species diversity</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr>, <span class="cit-vol">217</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">439</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D217%26rft.spage%253D439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ibargüengoytía</span>,  <span class="cit-name-given-names">N. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acosta</span>,  <span class="cit-name-given-names">J. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boretto</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Villavicencio</span>,  <span class="cit-name-given-names">H. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marinero</span>,  <span class="cit-name-given-names">J. A.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Krenz</span>,  <span class="cit-name-given-names">J. D.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Field thermal biology in Phymaturus lizards: comparisons from the Andes to the Patagonian steppe in Argentina</span>. <abbr class="cit-jnl-abbrev">Journal of Arid Environments</abbr>, <span class="cit-vol">72</span>(<span class="cit-issue">9</span>), <span class="cit-fpage">1620</span>–<span class="cit-lpage">1630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BArid%2BEnvironments%26rft.volume%253D72%26rft.spage%253D1620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.27" data-doi="10.1146/annurev.ecolsys.39.110707.173510"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jablonski</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Species selection: theory and data</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">39</span>, <span class="cit-fpage">501</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D39%26rft.spage%253D501%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000261725500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.28" data-doi="10.1038/nature02049"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lamb</span>,  <span class="cit-name-given-names">S.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Cenozoic climate change as a possible cause for the rise of the Andes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">425</span>(<span class="cit-issue">6960</span>), <span class="cit-fpage">792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D425%26rft.issue%253D6960%26rft.spage%253D792%26rft.epage%253D797%26rft.atitle%253DCenozoic%2Bclimate%2Bchange%2Bas%2Ba%2Bpossible%2Bcause%2Bfor%2Bthe%2Brise%2Bof%2Bthe%2BAndes%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02049%26rft_id%253Dinfo%253Apmid%252F14574402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2004065339&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=14574402&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186118500035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lewitus</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bittner</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Malviya</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bowler</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Morlon</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Clade-specific diversification dynamics of marine diatoms since the Jurassic</span>. <abbr class="cit-jnl-abbrev">Nature ecology &amp; evolution</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">1715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Becology%2B%2526%2Bevolution%26rft.volume%253D2%26rft.spage%253D1715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lewitus</span>,  <span class="cit-name-given-names">E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Morlon</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Detecting environment-dependent diversification from phylogenies: a simulation study and some empirical illustrations</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, <span class="cit-vol">67</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">576</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft.volume%253D67%26rft.spage%253D576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">J. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Knowles</span>,  <span class="cit-name-given-names">L. L.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Microevolutionary processes impact macroevolutionary patterns</span>. <abbr class="cit-jnl-abbrev">BMC evolutionary biology</abbr>, <span class="cit-vol">18</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%26rft.volume%253D18%26rft.spage%253D123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.32" data-doi="10.1111/j.0014-3820.2006.tb00517.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maddison</span>,  <span class="cit-name-given-names">W. P.</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Confounding asymmetries in evolutionary diversification and character change</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">60</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">1743</span>–<span class="cit-lpage">1746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DMaddison%26rft.auinit1%253DW.%2BP.%26rft.volume%253D60%26rft.issue%253D8%26rft.spage%253D1743%26rft.epage%253D1746%26rft.atitle%253DConfounding%2Basymmetries%2Bin%2Bevolutionary%2Bdiversification%2Band%2Bcharacter%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2006.tb00517.x%26rft_id%253Dinfo%253Apmid%252F17017073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2006.tb00517.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17017073&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240530300020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maddison</span>,  <span class="cit-name-given-names">W. P.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">FitzJohn</span>,  <span class="cit-name-given-names">R. G.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">The unsolved challenge to phylogenetic correlation tests for categorical characters</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, <span class="cit-vol">64</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">127</span>–<span class="cit-lpage">136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft.volume%253D64%26rft.spage%253D127%26rft_id%253Dinfo%253Apmid%252F25209222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25209222&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.34" data-doi="10.1080/10635150701607033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maddison</span>,  <span class="cit-name-given-names">W. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Midford</span>,  <span class="cit-name-given-names">P. E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Otto</span>,  <span class="cit-name-given-names">S. P.</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Estimating a binary character’s effect on speciation and extinction</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, <span class="cit-vol">56</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">701</span>–<span class="cit-lpage">710</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.stitle%253DSyst%2BBiol%26rft.aulast%253DMaddison%26rft.auinit1%253DW.%2BP.%26rft.volume%253D56%26rft.issue%253D5%26rft.spage%253D701%26rft.epage%253D710%26rft.atitle%253DEstimating%2Ba%2Bbinary%2Bcharacter%2527s%2Beffect%2Bon%2Bspeciation%2Band%2Bextinction.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150701607033%26rft_id%253Dinfo%253Apmid%252F17849325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150701607033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17849325&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250959000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.35" data-doi="10.1086/666649"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Magnuson-Ford</span>,  <span class="cit-name-given-names">K.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Otto</span>,  <span class="cit-name-given-names">S. P.</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Linking the investigations of character evolution and species diversification</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">180</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">225</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DMagnuson-Ford%26rft.auinit1%253DK.%26rft.volume%253D180%26rft.issue%253D2%26rft.spage%253D225%26rft.epage%253D245%26rft.atitle%253DLinking%2Bthe%2Binvestigations%2Bof%2Bcharacter%2Bevolution%2Band%2Bspecies%2Bdiversification.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F666649%26rft_id%253Dinfo%253Apmid%252F22766933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/666649&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22766933&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306110300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maliet</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hartig</span>,  <span class="cit-name-given-names">F.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Morlon</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">A model with many small shifts for estimating species-specific diversification rates</span>. <abbr class="cit-jnl-abbrev">Nature ecology &amp; evolution</abbr>, <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marín</span>,  <span class="cit-name-given-names">A. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Olave</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avila</span>,  <span class="cit-name-given-names">L. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sites</span>   <span class="cit-name-suffix">Jr</span>,  <span class="cit-name-given-names">J. W.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Morando</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Evidence of body size and shape stasis driven by selection in Patagonian lizards of the Phymaturus patagonicus clade (Squamata: Liolae-mini)</span>. <abbr class="cit-jnl-abbrev">Molecular phylogenetics and evolution</abbr>, <span class="cit-vol">129</span>, <span class="cit-fpage">226</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.volume%253D129%26rft.spage%253D226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matzke</span>,  <span class="cit-name-given-names">N. J.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Trait-dependent dispersal models for phylogenetic biogeography, in the R package BioGeoBEARS</span>. <abbr class="cit-jnl-abbrev">Integrative and Comparative Biology</abbr>, <span class="cit-vol">56</span>, <span class="cit-fpage">E330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegrative%2Band%2BComparative%2BBiology%26rft.volume%253D56%26rft.spage%253DE330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.39" data-doi="10.1073/pnas.1102543108"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morlon</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parsons</span>,  <span class="cit-name-given-names">T. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Plotkin</span>,  <span class="cit-name-given-names">J. B.</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Reconciling molecular phylogenies with the fossil record</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, <span class="cit-vol">108</span>(<span class="cit-issue">39</span>), <span class="cit-fpage">16327</span>–<span class="cit-lpage">16332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102543108%26rft_id%253Dinfo%253Apmid%252F21930899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM5LzE2MzI3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMTQvNTYzNjM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morlon</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lewitus</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Condamine</span>,  <span class="cit-name-given-names">F. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manceau</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clavel</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Drury</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">RPANDA: an R package for macroevolutionary analyses on phylogenetic trees</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>, <span class="cit-vol">7</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">589</span>–<span class="cit-lpage">597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D7%26rft.spage%253D589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.41" data-doi="10.1111/jeb.12460"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ng</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">S. D.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">How traits shape trees: new approaches for detecting character state-dependent lineage diversification</span>. <abbr class="cit-jnl-abbrev">Journal of evolutionary biology</abbr>, <span class="cit-vol">27</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">2035</span>–<span class="cit-lpage">2045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.volume%253D27%26rft.spage%253D2035%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12460%26rft_id%253Dinfo%253Apmid%252F25066512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25066512&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olave</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avila</span>,  <span class="cit-name-given-names">L. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sites</span>   <span class="cit-name-suffix">Jr</span>,  <span class="cit-name-given-names">J. W.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Morando</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Hybridization could be a common phenomenon within the highly diverse lizard genus Liolaemus</span>. <abbr class="cit-jnl-abbrev">Journal of evolutionary biology</abbr>, <span class="cit-vol">31</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">893</span>–<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.volume%253D31%26rft.spage%253D893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-563635v3.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olave</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gonzalez Marin</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avila</span>,  <span class="cit-name-given-names">L.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sites</span>,  <span class="cit-name-given-names">J.W.</span>   <span class="cit-name-suffix">Jr</span>.</span>. &amp; <span class="cit-auth"><span class="cit-name-surname">Morando</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-chapter-title">Disparate patterns of diversification within Liolaemini lizards</span>. In <span class="cit-source">Neotropical diversification</span>, edited by <span class="cit-auth"><span class="cit-name-surname">Rull</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Carnaval</span></span>. <span class="cit-publ-name">Springer Nature</span> (in press).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.44" data-doi="10.1073/pnas.1007606107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rabosky</span>,  <span class="cit-name-given-names">D. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Glor</span>,  <span class="cit-name-given-names">R. E.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Equilibrium speciation dynamics in a model adaptive radiation of island lizards</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, <span class="cit-vol">107</span>(<span class="cit-issue">51</span>), <span class="cit-fpage">22178</span>–<span class="cit-lpage">22183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1007606107%26rft_id%253Dinfo%253Apmid%252F21135239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzUxLzIyMTc4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMTQvNTYzNjM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.45" data-doi="10.1093/sysbio/syu131"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rabosky</span>,  <span class="cit-name-given-names">D. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Goldberg</span>,  <span class="cit-name-given-names">E. E.</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Model inadequacy and mistaken inferences of trait-dependent speciation</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, <span class="cit-vol">64</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">340</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyu131%26rft_id%253Dinfo%253Apmid%252F25601943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syu131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25601943&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rabosky</span>,  <span class="cit-name-given-names">D. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Goldberg</span>,  <span class="cit-name-given-names">E. E.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">FiSSE: A simple nonparametric test for the effects of a binary character on lineage diversification rates</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">71</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">1432</span>–<span class="cit-lpage">1442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D71%26rft.spage%253D1432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rabosky</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grundler</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Title</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">J. W.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Larson</span>,  <span class="cit-name-given-names">J. G.</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">BAMM tools: an R package for the analysis of evolutionary dynamics on phylogenetic trees</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>, <span class="cit-vol">5</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">701</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D5%26rft.spage%253D701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rabosky</span>,  <span class="cit-name-given-names">D. L.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">McCune</span>,  <span class="cit-name-given-names">A. R.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Reinventing species selection with molecular phylogenies</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr>, <span class="cit-vol">25</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">68</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D25%26rft.spage%253D68%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reaney</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saldarriaga-Córdoba</span>,  <span class="cit-name-given-names">M.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Pincheira-Donoso</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Macroevolutionary diversification with limited niche disparity in a species-rich lineage of cold-climate lizards</span>. <abbr class="cit-jnl-abbrev">BMC evolutionary biology</abbr>, <span class="cit-vol">18</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%26rft.volume%253D18%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Román-Palacios</span>,  <span class="cit-name-given-names">C.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Wiens</span>,  <span class="cit-name-given-names">J. J.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">The Tortoise and the Finch: Testing for island effects on diversification using two iconic Galápagos radiations</span>. <abbr class="cit-jnl-abbrev">Journal of biogeography</abbr>, <span class="cit-vol">45</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">1701</span>–<span class="cit-lpage">1712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiogeography%26rft.volume%253D45%26rft.spage%253D1701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.51" data-doi="10.1086/684193"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schluter</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Speciation, Ecological Opportunity, and Latitude: (American Society of Naturalists Address)</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>, <span class="cit-vol">187</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D187%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1086%252F684193%26rft_id%253Dinfo%253Apmid%252F26814593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/684193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26814593&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.52" data-doi="10.1093/sysbio/syp060"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Steeman</span>,  <span class="cit-name-given-names">M. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hebsgaard</span>,  <span class="cit-name-given-names">M. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fordyce</span>,  <span class="cit-name-given-names">R. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">S. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabosky</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Willerslev</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Radiation of extant cetaceans driven by restructuring of the oceans</span>. <abbr class="cit-jnl-abbrev">Systematic biology</abbr>, <span class="cit-vol">58</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">573</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Bbiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyp060%26rft_id%253Dinfo%253Apmid%252F20525610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/sysbio/syp060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20525610&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272072700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.53" data-doi="10.1073/pnas.091093098"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Templeton</span>,  <span class="cit-name-given-names">A. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robertson</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brisson</span>,  <span class="cit-name-given-names">J.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Strasburg</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Disrupting evolutionary processes: the effect of habitat fragmentation on collared lizards in the Missouri Ozarks</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, <span class="cit-vol">98</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">5426</span>–<span class="cit-lpage">5432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DTempleton%26rft.auinit1%253DA.%2BR.%26rft.volume%253D98%26rft.issue%253D10%26rft.spage%253D5426%26rft.epage%253D5432%26rft.atitle%253DDisrupting%2Bevolutionary%2Bprocesses%253A%2BThe%2Beffect%2Bof%2Bhabitat%2Bfragmentation%2Bon%2Bcollared%2Blizards%2Bin%2Bthe%2BMissouri%2BOzarks%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.091093098%26rft_id%253Dinfo%253Apmid%252F11344289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTAvNTQyNiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzE0LzU2MzYzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">K. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>,  <span class="cit-name-given-names">L. H.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Schluter</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Speciation and the city</span>. <abbr class="cit-jnl-abbrev">Trends in ecology &amp; evolution</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Varga</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krizsán</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Földi</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dima</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sánchez-García</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sánchez-Ramírez</span>,  <span class="cit-name-given-names">S.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Andreopoulos</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Megaphylogeny resolves global patterns of mushroom evolution</span>. <abbr class="cit-jnl-abbrev">Nature ecology &amp; evolution</abbr>, <span class="cit-fpage">1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vicenzi</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Corbalán</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miles</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sinervo</span>,  <span class="cit-name-given-names">B.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Ibargüengoytía</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Range increment or range detriment? Predicting potential changes in distribution caused by climate change for the endemic high-Andean lizard Phymaturus palluma</span>. <abbr class="cit-jnl-abbrev">Biological conservation</abbr>, <span class="cit-vol">206</span>, <span class="cit-fpage">151</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2Bconservation%26rft.volume%253D206%26rft.spage%253D151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.57" data-doi="10.1016/j.neuroimage.2009.12.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Winkler</span>,  <span class="cit-name-given-names">A. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kochunov</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blangero</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Almasy</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zilles</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fox</span>,  <span class="cit-name-given-names">P. T.</span></span>, … &amp; <span class="cit-auth"><span class="cit-name-surname">Glahn</span>,  <span class="cit-name-given-names">D. C.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Cortical thickness or grey matter volume? The importance of selecting the phenotype for imaging genetics studies</span>. <abbr class="cit-jnl-abbrev">Neuroimage</abbr>, <span class="cit-vol">53</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">1135</span>–<span class="cit-lpage">1146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuroimage%26rft.volume%253D53%26rft.spage%253D1135%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroimage.2009.12.028%26rft_id%253Dinfo%253Apmid%252F20006715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuroimage.2009.12.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20006715&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282039300040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.58" data-doi="10.1126/science.1059412"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zachos</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pagani</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sloan</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>,  <span class="cit-name-given-names">E.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Billups</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Trends, rhythms, and aberrations in global climate 65 Ma to present</span>. <abbr class="cit-jnl-abbrev">science</abbr>, <span class="cit-vol">292</span>(<span class="cit-issue">5517</span>), <span class="cit-fpage">686</span>–<span class="cit-lpage">693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DZachos%26rft.auinit1%253DJ.%26rft.volume%253D292%26rft.issue%253D5517%26rft.spage%253D686%26rft.epage%253D693%26rft.atitle%253DTrends%252C%2BRhythms%252C%2Band%2BAberrations%2Bin%2BGlobal%2BClimate%2B65%2BMa%2Bto%2BPresent%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1059412%26rft_id%253Dinfo%253Apmid%252F11326091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTIvNTUxNy82ODYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8xNC81NjM2MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-563635v3.59" data-doi="10.1038/nature06588"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zachos</span>,  <span class="cit-name-given-names">J. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dickens</span>,  <span class="cit-name-given-names">G. R.</span></span>, &amp; <span class="cit-auth"><span class="cit-name-surname">Zeebe</span>,  <span class="cit-name-given-names">R. E.</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">An early Cenozoic perspective on greenhouse warming and carbon-cycle dynamics</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">451</span>(<span class="cit-issue">7176</span>), <span class="cit-fpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZachos%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D451%26rft.issue%253D7176%26rft.spage%253D279%26rft.epage%253D283%26rft.atitle%253DAn%2Bearly%2BCenozoic%2Bperspective%2Bon%2Bgreenhouse%2Bwarming%2Band%2Bcarbon-cycle%2Bdynamics%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06588%26rft_id%253Dinfo%253Apmid%252F18202643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06588&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2013050985&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=18202643&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F14%2F563635.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252432100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/563635v3.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F563635v3" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/840660v2" title="Trophoblast paracrine signaling regulates placental hematoendothelial niche" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;November 14, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/563635v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/563635v3.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/563635v3.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/563635v3" class="active" data-icon-position="" data-hide-link-title="0">How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/993719" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-m1GvPp0tiRV_2j0AlIwem8cd_riSu9J-jGLOs88FJ7g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="993719" id="node993719--2" data-pisa="biorxiv;563635v3" data-pisa-master="biorxiv;563635" data-seqnum="993719" data-apath="/biorxiv/early/2019/11/14/563635.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Olave</span> <span class="nlm-surname">Melisa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Luciano J.</span> <span class="nlm-surname">Avila</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jack W.</span> <span class="nlm-surname">Sites</span> <span class="nlm-suffix">Jr.</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Morando</span> <span class="nlm-surname">Mariana</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">563635; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/563635 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/563635v3" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/563635v3&amp;title=How%20important%20is%20it%20to%20consider%20lineage%20diversification%20heterogeneity%20in%20macroevolutionary%20studies%3A%20lessons%20from%20the%20lizard%20family%20Liolaemidae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/563635v3&amp;title=How%20important%20is%20it%20to%20consider%20lineage%20diversification%20heterogeneity%20in%20macroevolutionary%20studies%3A%20lessons%20from%20the%20lizard%20family%20Liolaemidae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/563635v3&amp;text=How%20important%20is%20it%20to%20consider%20lineage%20diversification%20heterogeneity%20in%20macroevolutionary%20studies%3A%20lessons%20from%20the%20lizard%20family%20Liolaemidae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/563635v3&amp;title=How%20important%20is%20it%20to%20consider%20lineage%20diversification%20heterogeneity%20in%20macroevolutionary%20studies%3A%20lessons%20from%20the%20lizard%20family%20Liolaemidae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/563635v3&amp;t=How%20important%20is%20it%20to%20consider%20lineage%20diversification%20heterogeneity%20in%20macroevolutionary%20studies%3A%20lessons%20from%20the%20lizard%20family%20Liolaemidae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/563635v3&amp;title=How%20important%20is%20it%20to%20consider%20lineage%20diversification%20heterogeneity%20in%20macroevolutionary%20studies%3A%20lessons%20from%20the%20lizard%20family%20Liolaemidae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/563635v3&amp;title=How%20important%20is%20it%20to%20consider%20lineage%20diversification%20heterogeneity%20in%20macroevolutionary%20studies%3A%20lessons%20from%20the%20lizard%20family%20Liolaemidae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="993719" id="node993719--3" data-pisa="biorxiv;563635v3" data-pisa-master="biorxiv;563635" data-seqnum="993719" data-apath="/biorxiv/early/2019/11/14/563635.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">How important is it to consider lineage diversification heterogeneity in macroevolutionary studies: lessons from the lizard family Liolaemidae</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Olave</span> <span class="nlm-surname">Melisa</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Luciano J.</span> <span class="nlm-surname">Avila</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jack W.</span> <span class="nlm-surname">Sites</span> <span class="nlm-suffix">Jr.</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Morando</span> <span class="nlm-surname">Mariana</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">563635; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/563635 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=How%20important%20is%20it%20to%20consider%20lineage%20diversification%20heterogeneity%20in%20macroevolutionary%20studies%3A%20lessons%20from%20the%20lizard%20family%20Liolaemidae&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Melisa%2C+Olave&amp;rft.au=Avila%2C+Luciano+J.&amp;rft.au=Sites%2C+Jack+W.&amp;rft.au=Mariana%2C+Morando"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/993719/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/993719/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/993719/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/993719/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/993719/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/993719/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/993719/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/993719/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/993719/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/993719/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/993719/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/993719/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/563635v3&amp;count=horizontal&amp;via=&amp;text=How%20important%20is%20it%20to%20consider%20lineage%20diversification%20heterogeneity%20in%20macroevolutionary%20studies%3A%20lessons%20from%20the%20lizard%20family%20Liolaemidae&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/563635v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/563635v3&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/563635v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
