<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/16/671867.atom" />
<meta name="HW.pisa" content="biorxiv;671867v4" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A young age of subspecific divergence in the desert locust Schistocerca gregaria, inferred by ABC Random Forest" />
<meta name="DC.Identifier" content="10.1101/671867" />
<meta name="DC.Date" content="2020-01-16" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Dating population divergence within species from molecular data and relating such dating to climatic and biogeographic changes is not trivial. Yet it can help formulating evolutionary hypotheses regarding local adaptation and future responses to changing environments. Key issues include statistical selection of a demographic and historical scenario among a set of possible scenarios, and estimation of the parameter(s) of interest under the chosen scenario. Such inferences greatly benefit from new statistical approaches including approximate Bayesian computation - Random Forest (ABC-RF), the latter providing reliable inference at a low computational cost, with the possibility to take into account prior knowledge on both biogeographical history and genetic markers. Here, we used ABC-RF, including independent information on evolutionary rate and pattern at microsatellite markers, to decipher the evolutionary history of the African arid-adapted pest locust, Schistocerca gregaria . We found that the evolutionary processes that have shaped the present geographical distribution of the species in two disjoint northern and southern regions of Africa were recent, dating back 2.6 Ky (90% CI: 0.9 – 6.6 Ky). ABC-RF inferences also supported a southern colonization of Africa from a low number of founders of northern origin. The inferred divergence history is better explained by the peculiar biology of S. gregaria , which involves a density-dependent swarming phase with some exceptional spectacular migrations, rather than a continuous colonization resulting from the continental expansion of open vegetation habitats during more ancient Quaternary glacial climatic episodes." />
<meta name="DC.Contributor" content="Marie-Pierre Chapuis" />
<meta name="DC.Contributor" content="Louis Raynal" />
<meta name="DC.Contributor" content="Christophe Plantamp" />
<meta name="DC.Contributor" content="Christine N. Meynard" />
<meta name="DC.Contributor" content="Laurence Blondin" />
<meta name="DC.Contributor" content="Jean-Michel Marin" />
<meta name="DC.Contributor" content="Arnaud Estoup" />
<meta name="article:published_time" content="2020-01-16" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A young age of subspecific divergence in the desert locust Schistocerca gregaria, inferred by ABC Random Forest" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Dating population divergence within species from molecular data and relating such dating to climatic and biogeographic changes is not trivial. Yet it can help formulating evolutionary hypotheses regarding local adaptation and future responses to changing environments. Key issues include statistical selection of a demographic and historical scenario among a set of possible scenarios, and estimation of the parameter(s) of interest under the chosen scenario. Such inferences greatly benefit from new statistical approaches including approximate Bayesian computation - Random Forest (ABC-RF), the latter providing reliable inference at a low computational cost, with the possibility to take into account prior knowledge on both biogeographical history and genetic markers. Here, we used ABC-RF, including independent information on evolutionary rate and pattern at microsatellite markers, to decipher the evolutionary history of the African arid-adapted pest locust, Schistocerca gregaria. We found that the evolutionary processes that have shaped the present geographical distribution of the species in two disjoint northern and southern regions of Africa were recent, dating back 2.6 Ky (90% CI: 0.9 - 6.6 Ky). ABC-RF inferences also supported a southern colonization of Africa from a low number of founders of northern origin. The inferred divergence history is better explained by the peculiar biology of S. gregaria, which involves a density-dependent swarming phase with some exceptional spectacular migrations, rather than a continuous colonization resulting from the continental expansion of open vegetation habitats during more ancient Quaternary glacial climatic episodes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;671867v4" />
<meta name="citation_id" content="671867v4" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/671867v4" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/671867v4.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/671867v4.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867.full.pdf" />
<meta name="citation_doi" content="10.1101/671867" />
<meta name="citation_num_pages" content="66" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="671867" />
<meta name="citation_author" content="Marie-Pierre Chapuis" />
<meta name="citation_author_institution" content="CIRAD" />
<meta name="citation_author_institution" content="CBGP, CIRAD, INRAE, IRD, Montpellier SupAgro, Univ Montpellier" />
<meta name="citation_author_email" content="marie-pierre.chapuis@cirad.fr" />
<meta name="citation_author" content="Louis Raynal" />
<meta name="citation_author_institution" content="IMAG, Univ de Montpellier" />
<meta name="citation_author" content="Christophe Plantamp" />
<meta name="citation_author_institution" content="ANSES, Laboratoire de Lyon" />
<meta name="citation_author" content="Christine N. Meynard" />
<meta name="citation_author_institution" content="CBGP, INRAE, CIRAD, IRD, Montpellier SupAgro, Univ Montpellier" />
<meta name="citation_author" content="Laurence Blondin" />
<meta name="citation_author_institution" content="CIRAD" />
<meta name="citation_author" content="Jean-Michel Marin" />
<meta name="citation_author_institution" content="IMAG, Univ de Montpellier" />
<meta name="citation_author" content="Arnaud Estoup" />
<meta name="citation_author_institution" content="CBGP, INRAE, CIRAD, IRD, Montpellier SupAgro, Univ Montpellier" />
<meta name="citation_reference" content="Adams JM, Faure H (1997) (eds), QEN members. Review and Atlas of Palaeovegetation: Preliminary land ecosystem maps of the world since the Last Glacial Maximum. Oak Ridge National Laboratory, TN, USA." />
<meta name="citation_reference" content="van Andel TH, Tzedakis PC (1996). Palaeolithic landscapes of Europe and environs: 150,000-25,000 years ago: an overview. Quat Sci Rev, 15, 481–500." />
<meta name="citation_reference" content="Atickem A, Stenseth NC, Drouilly M, Bock S, Roos C, Zinner D (2018) Deep divergence among mitochondrial lineages in African jackals. Zool Scripta, 47, 1–8." />
<meta name="citation_reference" content="Aumassip G (2002) L’algérie des premiers hommes. Ibis Press, 230 p." />
<meta name="citation_reference" content="Blondin L, Badisco L, Pagès C, Foucart A, Risterucci AM, Bazelet CS, Vanden Broeck J, Song H, Ould Ely S, Chapuis M-P (2013) Characterization and comparison of microsatellite markers derived from genomic and expressed libraries for the desert locust. J Appl Entomol, 137, 673–683." />
<meta name="citation_reference" content="Bond G et al. (1997) A pervasive millenial-scale cycle in North Atlantic Holocene and Glacial climates. Science, 278, 1257–1265." />
<meta name="citation_reference" content="Bostoen K, Clist B, Doumenge C, Grollemund R, Hombert JM, Muluwa JK, Maley J (2015) Middle to Late Holocene paleoclimatic change and early Bantu expansion in the rain forests of Western Central Africa. Curr Anthropol, 56, 354–384." />
<meta name="citation_reference" content="Breiman, L. (2001) Random Forests. Machine Learning, 45(1), 5–32." />
<meta name="citation_reference" content="Chakraborty R, Jin L (1993) A unified approach to study hypervariable polymorphisms: statistical considerations of determining relatedness and population distances. EXS, 67, 153–175." />
<meta name="citation_reference" content="Chapuis M-P, Bazelet CS, Blondin L, Foucart A, Vitalis R, Samways MJ. (2016) Subspecific taxonomy of the desert locust, Schistocerca gregaria (Orthoptera: Acrididae), based on molecular and morphological characters. Syst Entomol, 41, 516–530." />
<meta name="citation_reference" content="Chapuis M-P, Foucart A, Plantamp P, Blondin L, Leménager N, Benoit L, Gay P-E, Bazelet CS (2017) Genetic and morphological variation in non-polyphenic southern African populations of the desert locust. Afr Entomol, 25, 13–23." />
<meta name="citation_reference" content="Chapuis M.-P, Plantamp P, Blondin B, Pagès C, Vassal J.-M., Lecoq M (2014) Demographic processes shaping genetic variation of the solitarious phase of the desert locust. Mol Ecol, 23, 1749–1763." />
<meta name="citation_reference" content="Chapuis M-P, Plantamp C, Streiff R, Blondin L, Piou C (2015) Microsatellite evolutionary rate and pattern in Schistocerca gregaria inferred from direct observation of germline mutations. Mol Ecol, 24, 6107–6119." />
<meta name="citation_reference" content="Chase BM, Meadows ME (2007) Late Quaternary dynamics of southern Africa’s winter rainfall zone. Earth Sci Rev, 84, 103–138." />
<meta name="citation_reference" content="Cockcroft MJ, Wilkinson MJ, Tyson PD (1987) The application of a present-day climatic model to the Late Quaternary in southern Africa. Climate Change, 10, 161–181." />
<meta name="citation_reference" content="Cornuet J-M, Ravigne V, Estoup A (2010) Inference on population history and model checking using DNA sequence and microsatellite data with the software DIYABC (v1.0). BMC Bioinformatics, 11, 401." />
<meta name="citation_reference" content="Cornuet J-M, Pudlo P, Veyssier J, Dehne-Garcia A, Gautier M, Leblois R, Marin J-M, Estoup A (2014) DIYABC v2.0: a software to make Approximate Bayesian Computation inferences about population history using Single Nucleotide Polymorphism, DNA sequence and microsatellite data. Bioinformatics, 30, 1187–1189." />
<meta name="citation_reference" content="deMenocal PB (1995) Plio-Pleistocene African climate, Science, 270, 53–59." />
<meta name="citation_reference" content="deMenocal PB, Ortiz J, Guilderson T, Adkins J, Sarnthein M, Baker L, Yarusinsky M (2000) Abrupt onset and termination of the African Humid Period: Rapid climate response to gradual insolation forcing. Quat Sci Rev, 19, 347–361." />
<meta name="citation_reference" content="deMenocal PB (2004) African climate change and faunal evolution during the Pliocene-Pleistocene. Earth and Planetary Science Letters, 220, 3–24." />
<meta name="citation_reference" content="Duranton JF, Foucart A, Gay P-E (2012) Florule des biotopes du criquet pèlerin en Afrique de l’Ouest et du Nord-Ouest à l’usage des prospecteurs de la lutte antiacridienne. Rome : FAO, 487 p." />
<meta name="citation_reference" content="Edmunds WM, Fellman E, Goni IB (1999) Lakes, groundwater and palaeohydrology in the Sahel of NE Nigeria: evidence from hydrogeochemistry. J Geol Soc Lond, 156, 345–355." />
<meta name="citation_reference" content="Ellegren H (2000) Microsatellite mutations in the germline: implications for evolutionary inference. Trends Genet, 16, 551–558." />
<meta name="citation_reference" content="Estoup A, Jarne P, Cornuet J-M (2002) Homoplasy and mutation model at microsatellite loci and their consequences for population genetics analysis. Mol Ecol, 11, 1591–1604." />
<meta name="citation_reference" content="Estoup A, Raynal L, Verdu P, Marin J-M (2018a) Model choice using Approximate Bayesian Computation and Random Forests: analyzes based on model grouping to make inferences about the genetic history of Pygmy human populations. J Soc Fr Statistique, 159, 167–190." />
<meta name="citation_reference" content="Estoup A, Verdu P, Marin J-M, Robert C, Dehne-Garcia A, Cornuet J-M, Pudlo P (2018b) Application of approximate Bayesian computation to infer the genetic history of Pygmy hunter-gatherers populations from Western Central Africa. Handbook of Approximate Bayesian Computation. Chapman and Hall/CRC." />
<meta name="citation_reference" content="Estoup A, Wilson IJ, Sullivan C, Cornuet J-M, Moritz C (2001) Inferring population history from microsatellite et enzyme data in serially introduced cane toads, Bufo marinus. Genetics, 159, 1671–1687." />
<meta name="citation_reference" content="Feldman MW, Bergman A, Pollock DD, Goldstein DB (1997) Microsatellite genetic distances with range constraints: analytic description and problems of estimation. Genetics, 145, 207–216." />
<meta name="citation_reference" content="Fraimout A, Debat V, Fellous S, Hufbauer RA, Foucaud J, Pudlo P, Marin J-M, Price DK, Cattel J et al. (2017). Deciphering the routes of invasion of Drosophila suzukii by means of ABC random forest. Mol Biol Evol, 34(4), 980–996." />
<meta name="citation_reference" content="Ganopolski A, Rahmstorf S, Petoukohov V, Claussen M (1998) Simulation of modern and glacial climates with a coupled global model of intermediate complexity. Nature, 391, 351–356." />
<meta name="citation_reference" content="Garza JC, Williamson E (2001) Detection of reduction in population size using data from microsatellite DNA. Mol Ecol, 10, 305–318." />
<meta name="citation_reference" content="Gasse F (2000) Hydrological changes in the African tropics since the Last Glacial Maximum. Quat Sci Rev, 19, 189–211." />
<meta name="citation_reference" content="Goldstein DB, Linares AR, Cavalli-Sforza LL, Feldman MW (1995) An evaluation of genetic distances for use with microsatellite loci. Genetics, 139, 463–471." />
<meta name="citation_reference" content="Guo Z, Petit-Maire N, Kroepelin S (2000) Holocene non-orbital climatic events in present-day arid areas of northern Africa and China. Global Planet Change v.26 p.97–103." />
<meta name="citation_reference" content="Ho SY, Saarma U, Barnett R, Haile J, Shapiro B (2008) The Effect of Inappropriate Calibration: Three Case Studies in Molecular Ecology. PLoS One, 3, e1615." />
<meta name="citation_reference" content="Jürgens N (1997) Floristic biodiversity and history of African arid regions. Biodiv Conserv, 6, 495–514." />
<meta name="citation_reference" content="Kröpelin S, Verschuren D, Lézine A-M, Eggermont H, Cocquyt C, Francus P, Cazet JP, Fagot M, Rumes B, Russel JM, et al. (2008). Climate-driven ecosystem succession in the Sahara: The past 6000 years. Science, 320, 765–8." />
<meta name="citation_reference" content="Lebrun JP (2001) Introduction à la flore d’Afrique. 156 pp. CIRAD and Ibis Press." />
<meta name="citation_reference" content="Le Gall P, Silvain JF, Nel A, Lachaise D (2010) Les insectes actuels témoins des passés de l’Afrique : essai sur l’origine et la singularité de l’entomofaune de la région afrotropicale. Ann Soc Entomol Fr, 46, 297–343." />
<meta name="citation_reference" content="Leroy T, Roux C, Villate L, Bodénès C, Romiguier J, Paiva JAP, Dossat C, Aury J-M, Plomion C, Kremer A (2017) Extensive recent secondary contacts between four European white oak species. New Phytol 214:865–878." />
<meta name="citation_reference" content="Lorenz MW (2009) Migration and trans-Atlantic flight of locusts. Quaternary International, 196, 4–12." />
<meta name="citation_reference" content="Lorenzen ED, Heller R, Siegismund HR (2012) Comparative phylogeography of African savannah ungulates. Mol Ecol, 21, 3656–3670." />
<meta name="citation_reference" content="Lovejoy NR, Mullen SP, Sword GA, Chapman RF, Harrison RG (2006) Ancient trans-Atlantic flight explains locust biogeography: molecular phylogenetics of Schistocerca. Proc R Soc B, 273, 767–774." />
<meta name="citation_reference" content="citation_title=Chief of Seers: Egyptian Studies in Memory of Cyril Aldred;citation_pages=207-219;citation_year=1997" />
<meta name="citation_reference" content="Maley J, Doumenge C, Giresse P, Mahé G, Philippon N, Hubau W, Lokonda MO, Tshibamba JM, Chepstow-Lusty A (2018) Late Holocene forest contraction and fragmentation in central Africa. Quat Res, 89, 43–59." />
<meta name="citation_reference" content="Marin JM, Pudlo P, Estoup A, Robert CP (2018) Likelihood-free Model Choice. Handbook of Approximate Bayesian Computation. Chapman and Hall/CRC." />
<meta name="citation_reference" content="Meinzingen WF (1993) A guide to migrant pest management in Africa. FAO, Rome, Italy." />
<meta name="citation_reference" content="Miller JM, Hallager S, Monfort S, Newby J, Bishop K, Tidmus S, Black P, Houston B, Matthee C, Robinson J, Fleischer RC (2011). Phylogeographic analysis of nuclear and mtDNA supports subspecies designations in the Ostrich (Struthio camelus). Conserv Genet, 12, 423–431." />
<meta name="citation_reference" content="Monod T (1971) Remarques sur les symetries floristiques des zones sèches nord et sud en Afrique. Mitteilungen der Botanischen Staatssammlung München, 10, 375–423." />
<meta name="citation_reference" content="Moodley Y, Russo I-RM, Robovsky J, Dalton D, Kotzé A, Smith S, Stejskal J, Ryder OA, Hermes R, Walzer C, Bruford MW (2018) Contrasting evolutionary history, anthropogenic declines and genetic contact in the northern and southern white rhinoceros (Ceratotherium simum). Proc R Soc B, 285, 1–9." />
<meta name="citation_reference" content="citation_title=The Limnology, Climatology and Paleoclimatology of the East African Lakes;citation_pages=25-56;citation_year=1996" />
<meta name="citation_reference" content="Partridge TC (1997) Cainozoic environmental change in southern Africa, with special emphasis on the last 200 000 years. Progr Phys Geog, 21, 3–22." />
<meta name="citation_reference" content="Pascual M, Chapuis MP, Mestres F, Balanyà J, Huey RB, Gilchrist GW, Serra L, Estoup A (2007) Introduction history of Drosophila subobscura in the New World: a microsatellite based survey using ABC methods. Mol Ecol, 16, 3069–3083." />
<meta name="citation_reference" content="citation_title=Insect Migration;citation_pages=3-29;citation_year=1995" />
<meta name="citation_reference" content="Pollock DD, Bergman A, Feldman MW, Goldstein DB (1998) Microsatellite behaviour with range constraints: parameter estimation and improved distances for use in phylogenetic reconstruction. Theor Popul Biol, 53, 256–271." />
<meta name="citation_reference" content="Pritchard JK, Stephens M, Donnelly P (2000) Inference of population structure using multilocus genotype data. Genetics, 155, 945–959." />
<meta name="citation_reference" content="Pudlo P, Marin J-M, Estoup A, Cornuet J-M, Gautier M, Robert CP (2016) Reliable ABC model choice via random forests. Bioinformatics, 32, 859–866." />
<meta name="citation_reference" content="Rainey RC, Waloff Z (1951) Flying locusts and convection currents. Anti-Locust Bull, 9, 51–70." />
<meta name="citation_reference" content="Rannala B, Mountain JL (1997) Detecting immigration by using multilocus genotypes. Proc Natl Acad Sci USA, 94, 9197–9201." />
<meta name="citation_reference" content="Raynal L, Marin J-M, Pudlo P, Ribatet M, Robert CP, Estoup A (2019) ABC random forests for Bayesian parameter inference. Bioinformatics, 35, 1720–1728." />
<meta name="citation_reference" content="Richardson, CH, Nemeth, DJ (1991) Hurricane-borne African locusts (Schistocerca gregaria) on the Windward Islands. GeoJournal, 23, 349–357." />
<meta name="citation_reference" content="Roberts N, Taieb M, Baker P, Damnati B, Icole M, Williamson D (1993) Timing of the Younger Dryas event in East Africa from lake-level changes. Nature, 366, 146–8." />
<meta name="citation_reference" content="Roffey J, Magor JI (2003) Desert Locust population parameters. Desert Locust Field Research Stations, Technical Series, 30, 29 p. FAO, Rome, Italy." />
<meta name="citation_reference" content="Rougemont Q, Roux C, Neuenschwander S, Goudet J, Launey S, Evanno G (2016) Reconstructing the demographic history of divergence between European river and brook lampreys using approximate Bayesian computations. PeerJ, 4, e1910." />
<meta name="citation_reference" content="Rousset F (2008) GenePop’007: a complete re-implementation of the GenePop software for Windows and Linux. Mol Ecol Resour, 8, 103–106." />
<meta name="citation_reference" content="Roux C, Fraïsse C, Romiguier J, Anciaux Y, Galtier N, Bierne N (2016) Shedding light on the grey zone of speciation along a continuum of genomic divergence. PLOS Biol, 14(12), e2000234." />
<meta name="citation_reference" content="Schwartz D (1992) Assèchement climatique vers 3000 B.P. et expansion Bantu en Afrique centrale atlantique: quelques réflexions. B Soc Geol Fr, 163, 353–61." />
<meta name="citation_reference" content="Scott L (1993) Palynological evidence for late Quaternary warming episodes in Southern Africa. Palaeogeogr Palaeocl, 101, 229–235." />
<meta name="citation_reference" content="Shi N, Dupont LM, Beug H-J, Schneider R (1998) Vegetation and climate changes during the last 21 000 years in S.W. Africa based on a marine pollen record. Veg Hist Archaeobot, 7, 127–140." />
<meta name="citation_reference" content="Stokes S, Thomas DSG, Washington R (1997) Multiple episodes of aridity in southern Africa since the last interglacial period. Nature, 388, 154–158." />
<meta name="citation_reference" content="Stute M, Talma AS (1997) Glacial temperatures and moisture transport regimes reconstructed from noble gases and 18O, Stampriet aquifer, Namibia. Proceedings of International Symposium on Isotope Techniques in the Study of Past and Current Environmental Changes in the Hydrosphere and the Atmosphere, Vienna, International Atomic Energy Agency." />
<meta name="citation_reference" content="Sun JX, Helgason A, Masson G et al. (2012) A direct characterization of human mutation based on microsatellites. Nat Genet, 44, 1161–1165." />
<meta name="citation_reference" content="Sword GA, Lecoq M, Simpson SJ (2010). Phase polyphenism and preventative locust management. J Insect Physiol, 56, 949–957." />
<meta name="citation_reference" content="Takezaki N, Nei M (1996) Genetic distances and reconstruction of phylogenetic trees from microsatellite DNA. Genetics, 144, 389–399." />
<meta name="citation_reference" content="Talma AS, Vogel JC (1992) Late Quaternary paleotemperatures derived from a speleothem from Cango Caves. Cape Province, South Africa. Quat. Res., 37, 203–13." />
<meta name="citation_reference" content="Uvarov BP (1977) Grasshoppers and Locusts, vol. 2. Centre for Overseas Pest Research, London, UK." />
<meta name="citation_reference" content="Vansina J (1995) New Linguistic Evidence and the Bantu expansion. J Afr Hist, 36, 173–195." />
<meta name="citation_reference" content="de Vivo M, Carmignotto AP (2004) Holocene vegetation change and the mammal faunas of South America and Africa. J Biogeogr, 31, 943–957." />
<meta name="citation_reference" content="Walker MJC, Berkelhammer M, Björck S, Cwynar LC, Fisher DA, Long AJ, Lowe JJ, Newnham RM, Rasmussen SO, Weiss H (2012) Formal subdivision of the Holocene Series/Epoch: a Discussion Paper by a Working Group of INTIMATE (Integration of ice-core, marine and terrestrial records) and the Subcommission on Quaternary Stratigraphy (International Commission on Stratigraphy). J Quat Sci, 27, 649–659." />
<meta name="citation_reference" content="Waloff Z (1966) The upsurges and recessions of the desert locust plague: an historical survey. Anti-Locust Memoir, 8, 111 p." />
<meta name="citation_reference" content="Waloff Z (1976) Some temporal characteristics of desert locust plagues. Anti-Locust Memoir, 13, 1–36." />
<meta name="citation_reference" content="Waloff Z, Pedgley DE (1986) Comparative biogeography and biology of the South American locust, Schistocerca cancellata (Serville), and the South African desert locust, S. gregaria flaviventris (Burmeister) (Orthoptera: Acrididae): a review. Bull Entomol Res, 76, 1–20." />
<meta name="citation_reference" content="Weir BS (1996) Genetic Data Analysis II. Sinauer Associates, Sunderland, Massachusetts." />
<meta name="citation_reference" content="Zhivotovsky LA, Feldman MW, Grishechkin SA (1997) Biased mutations and microsatellite variation. Mol Biol Evol, 14, 926–933." />
<meta name="citation_reference" content="Barbet-Massin M, Jiguet F, Albert CH, Thuiller W (2012) Selecting pseudo-absences for species distribution models: how, where and how many? Methods Ecol Evol, 3, 327–338." />
<meta name="citation_reference" content="Elith J, Leathwick JR, Hastie T (2008) A working guide to boosted regression trees. J Anim Ecol, 77, 802–813." />
<meta name="citation_reference" content="Elith J, Phillips SJ, Hastie T, Dudik M, Chee YE, Yates CJ (2011) A statistical explanation of MaxEnt for ecologists. Divers Distrib, 17, 43–57." />
<meta name="citation_reference" content="Hijmans RJ, Cameron SE, Parra JL, Jones PG, Jarvis A (2005) Very High Resolution interpolated climate surfaces for global land areas. Int J Climatol, 25, 1965–1978." />
<meta name="citation_reference" content="Kriticos DJ, Webber BL, Leriche A, Ota N, Macadam I, Bathols J, Scott JK (2012) CliMond: global high-resolution historical and future scenario climate surfaces for bioclimatic modelling. Methods Ecol Evol, 3, 53–64." />
<meta name="citation_reference" content="Liu C, Berry PM, Dawson TP, Pearson RG (2005) Selecting thresholds of occurrence in the prediction of species distributions. Ecography, 28, 385–393." />
<meta name="citation_reference" content="Meynard CN, Gay P-E, Lecoq M, Foucart A, Piou C, Chapuis M-P (2017) Climate-driven geographic distribution of the desert locust during recession periods: Subspecies’ niche differentiation and relative risks under scenarios of climate change. Glob Change Biol, 23, 4739–4749." />
<meta name="citation_reference" content="Rowell DP, Senior CA, Vellinga M, Graham RJ (2016) Can climate projection uncertainty be constrained over Africa using metrics of contemporary performance? Clim Change, 134, 621–633." />
<meta name="citation_reference" content="Wood SN (2006) Generalized Additive Models: an introduction with R. Boca Raton, Florida: Chapman &amp; Hall/CRC." />
<meta name="citation_reference" content="Beaumont, M. A. (2010). Approximate Bayesian Computation in Evolution and Ecology. Annual Review of Ecology, Evolution, and Systematics, 41:379–406." />
<meta name="citation_reference" content="Beaumont, M. A., Zhang, W., and Balding, D. (2002). Approximate Bayesian Computation in Population Genetics. GENETICS, 162(4):2025–2035." />
<meta name="citation_reference" content="Estoup, A., Raynal, L., Verdu, P., and Marin, J.-M. (2018). Model choice using Approximate Bayesian Computation and Random Forests: analyses based on model grouping to make inferences about the genetic history of Pygmy human populations. Journal de la Société Française de Statistique, 159(3):167–190." />
<meta name="citation_reference" content="Kingman, J. F. C. (1982). On the Genealogy of Large Populations. Journal of Applied Probability, 19(A):27–43." />
<meta name="citation_reference" content="Lippens, C., Estoup, A., Hima, M. K., Loiseau, A., Tatard, C., Dalecky, A., Bâ, K., Kane, M., Diallo, M., Sow, A., Niang, Y., Piry, S., Berthier, K., Leblois, R., Duplantier, J. M., and Brouat, C. (2017). Genetic structure and invasion history of the house mouse (Mus musculus domesticus) in Senegal, West Africa: a legacy of colonial and contemporary times. Heredity, 119(2):64–75." />
<meta name="citation_reference" content="Pudlo, P., Marin, J.-M., Estoup, A., Cornuet, J.-M., Gautier, M., and Robert, C. P. (2016). Reliable ABC model choice via random forests. Bioinformatics, 32(6):859–866." />
<meta name="citation_reference" content="Raynal, L., Marin, J.-M., Pudlo, P., Ribatet, M., Robert, C. P., and Estoup, A. (2019). ABC random forests for Bayesian parameter inference. Bioinformatics, 35(10):1625–1633." />
<meta name="twitter:title" content="A young age of subspecific divergence in the desert locust Schistocerca gregaria, inferred by ABC Random Forest" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Dating population divergence within species from molecular data and relating such dating to climatic and biogeographic changes is not trivial. Yet it can help formulating evolutionary hypotheses regarding local adaptation and future responses to changing environments. Key issues include statistical selection of a demographic and historical scenario among a set of possible scenarios, and estimation of the parameter(s) of interest under the chosen scenario. Such inferences greatly benefit from new statistical approaches including approximate Bayesian computation - Random Forest (ABC-RF), the latter providing reliable inference at a low computational cost, with the possibility to take into account prior knowledge on both biogeographical history and genetic markers. Here, we used ABC-RF, including independent information on evolutionary rate and pattern at microsatellite markers, to decipher the evolutionary history of the African arid-adapted pest locust, Schistocerca gregaria . We found that the evolutionary processes that have shaped the present geographical distribution of the species in two disjoint northern and southern regions of Africa were recent, dating back 2.6 Ky (90% CI: 0.9 – 6.6 Ky). ABC-RF inferences also supported a southern colonization of Africa from a low number of founders of northern origin. The inferred divergence history is better explained by the peculiar biology of S. gregaria , which involves a density-dependent swarming phase with some exceptional spectacular migrations, rather than a continuous colonization resulting from the continental expansion of open vegetation habitats during more ancient Quaternary glacial climatic episodes." />
<meta name="og-title" property="og:title" content="A young age of subspecific divergence in the desert locust Schistocerca gregaria, inferred by ABC Random Forest" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/671867v4" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Dating population divergence within species from molecular data and relating such dating to climatic and biogeographic changes is not trivial. Yet it can help formulating evolutionary hypotheses regarding local adaptation and future responses to changing environments. Key issues include statistical selection of a demographic and historical scenario among a set of possible scenarios, and estimation of the parameter(s) of interest under the chosen scenario. Such inferences greatly benefit from new statistical approaches including approximate Bayesian computation - Random Forest (ABC-RF), the latter providing reliable inference at a low computational cost, with the possibility to take into account prior knowledge on both biogeographical history and genetic markers. Here, we used ABC-RF, including independent information on evolutionary rate and pattern at microsatellite markers, to decipher the evolutionary history of the African arid-adapted pest locust, Schistocerca gregaria . We found that the evolutionary processes that have shaped the present geographical distribution of the species in two disjoint northern and southern regions of Africa were recent, dating back 2.6 Ky (90% CI: 0.9 – 6.6 Ky). ABC-RF inferences also supported a southern colonization of Africa from a low number of founders of northern origin. The inferred divergence history is better explained by the peculiar biology of S. gregaria , which involves a density-dependent swarming phase with some exceptional spectacular migrations, rather than a continuous colonization resulting from the continental expansion of open vegetation habitats during more ancient Quaternary glacial climatic episodes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-16" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/671867v4.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/671867v4.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/671867v4.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/671867v4" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1107113" />
    <title>A young age of subspecific divergence in the desert locust Schistocerca gregaria, inferred by ABC Random Forest | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__AQm9HArb2Y_D7UzgraiLIwAB7088fIq2iZD0Uigie_s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"uZqJdDrwfj8re4w3RZFp3zXT9RXbkljPnySdWc40grY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1107113","apath":"\/biorxiv\/early\/2020\/01\/16\/671867.atom","pisa":"biorxiv;671867v4","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;671867v4"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/671867v4","title":"A young age of subspecific divergence in the desert locust Schistocerca gregaria, inferred by ABC Random Forest","identifier":"node\/1107113"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qgLBGTg4U3rHnz2PSAl1pLSjgv1QfLx830Cs8XmRfQN5ESZrTdKBMRJS+EorpUVCe+JrAygwxrcRiGfueOMmA0GKe4fUNTpxdfd1bXm5b6E8\\\\\\\/sycKm3\\\\\\\/Sy7+I5UZ+fc\\\\\\\/DWAQyiWc9iUXP0DhMhEQtAc+y1GtS2psmuOIKpoc0wZu\\\\\\\/dx2Trj7NFG88gxCeqij8pTbiwUEu7xySUeuYiuH+m\\\\\\\/foVaDz4T6o\\\\\\\/8yaKKSEipKqVI+5HNzLYTX+NbtGBqQ99bNk25wjCR0\\\\\\\/lD5nV1FTYHWBu0GtK3gQE7RR5nXtjjQd0e55rFwzXAslV9lsJ8qxFtr0Nlgh5WVerE+vG8z7fTtnz87qtlxLN\\\\\\\/hKOVmwHYhPxL+OoCkl17EVLMHH6Z45yPm47Z92SWm4L2EbJ05QdR6ZlDxpaackolH7dSWNLud9JqN44s8KCb2Hj8fcUzM5bx4EhL2mK1JdCnLKBLsKnjCVY6g1\\\\\\\/h+HSdyU7lwSHZ1xI0B9jJuwLqrQ4d0zXocU8IGfbLFg8aLtWbJ8OkRfsVbpsPHPGlIDVvMGVWEdLHZyjsiBLi0C686M+WV3wvcsQWTARo2+Tj3Dum6SYbvNEQ2T4lOE7Lz2LgMsaH1P4Jv1F0HxkLWku7t1B5hbkYvOpbdtrjIECh9N0FNvmG3daBc0qR2MOPFa4JfM2yydN7yh1afE1df7w1Z0gB6dkUY6\\\\\\\/LJXuopSBn0yxDFxDF7SBaHe\\\\\\\/83O5v245L+joQOxHAROVDzJ2o\\\\\\\/Nbg88DdccxmRjS24FVfIsxUF7lbeRij12OEAuJCxEd30tydt7djOBTszWhmh5nAAHZWidTx2PB86tnd77+t5AkIC\\\\\\\/wmmIKtqZ9A9IWGBfnxYqw0oi\\\\\\\/9loSoC48nPwe5qO7K4jGbqmSCkvaP84O2yHeCHl0u4Fi1l7bm3TatmQ3Y+jsJ+rE1z\\\\\\\/3Dc8NABsixPogFr4u7AO1duNM1Nza5PUZqs9SAfVdrdRm4zBE762aMk5MMUgJ9gIKZPyPcQR9akat+G9gXSruYo0rBPBQEOsrgP5d\\\\\\\/9a0IkMInLFDdT2o5X9mGlfPIEcu8YtGZr9hg\\\\\\\/CZlcISCFI0YArXgalg6U1C3KAd\\\\\\\/dgCezjfrCJsQ25PGvLlWreIuCe0xTXLUePR+eK8hFiswEaOYD7ReS1M9M8uXqxrYqwU+XeLU67S78WvI0Y\\\\\\\/Aw24WTH9MNFcv4fBT+G\\\\\\\/Kms73qByqdJRnoOopW+4W\\\\\\\/JClL9GeNcYxKpp7zIzQttm9o4AUqb4tJ6ZR0AQ3EdqGZ7nVmeaMdeWmLb\\\\\\\/S10ufaDhqBYKN7aZjhNhtMidkqMGwoFlNBr+M2RqBkTEqZiTID1Dxx6xxy\\\u0022,\\\u0022iv\\\u0022:\\\u0022Srw19H1l4wo+cA8c9ouPIw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00225d434dce21fa2838f5b80d758753d3982c2bec80df8e7e11cdab19f1b90e2e82\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/671867v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1107113 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/671867v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-NrjUmYfpkJspV4G4inL9MoTpzrVyYnOz8tRUFUe2Z34" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107113" id="node1107113" data-pisa="biorxiv;671867v4" data-pisa-master="biorxiv;671867" data-apath="/biorxiv/early/2020/01/16/671867.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A young age of subspecific divergence in the desert locust <em>Schistocerca gregaria</em>, inferred by ABC Random Forest</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Marie-Pierre</span> <span class="nlm-surname">Chapuis</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Louis</span> <span class="nlm-surname">Raynal</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Christophe</span> <span class="nlm-surname">Plantamp</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Christine N.</span> <span class="nlm-surname">Meynard</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Laurence</span> <span class="nlm-surname">Blondin</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jean-Michel</span> <span class="nlm-surname">Marin</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Arnaud</span> <span class="nlm-surname">Estoup</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/671867 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1107113" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marie-Pierre Chapuis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CIRAD</span>, CBGP, Montpellier, <span class='nlm-country'>France</span></div><div class='author-affiliation'><span class='nlm-institution'>CBGP, CIRAD, INRAE, IRD, Montpellier SupAgro, Univ Montpellier</span>, Montpellier, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-Pierre%2BChapuis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chapuis%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarie-Pierre%2BChapuis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:marie-pierre.chapuis@cirad.fr" class="" data-icon-position="" data-hide-link-title="0">marie-pierre.chapuis@cirad.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Louis Raynal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>IMAG, Univ de Montpellier</span>, CNRS, Montpellier, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Louis%2BRaynal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raynal%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALouis%2BRaynal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christophe Plantamp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>ANSES, Laboratoire de Lyon</span>, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BPlantamp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Plantamp%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristophe%2BPlantamp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christine N. Meynard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CBGP, INRAE, CIRAD, IRD, Montpellier SupAgro, Univ Montpellier</span>, Montpellier, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christine%2BN.%2BMeynard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Meynard%20CN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristine%2BN.%2BMeynard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Laurence Blondin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CIRAD</span>, BGPI, Montpellier, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurence%2BBlondin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blondin%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaurence%2BBlondin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jean-Michel Marin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>IMAG, Univ de Montpellier</span>, CNRS, Montpellier, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Michel%2BMarin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marin%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Michel%2BMarin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Arnaud Estoup </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>CBGP, INRAE, CIRAD, IRD, Montpellier SupAgro, Univ Montpellier</span>, Montpellier, <span class='nlm-country'>France</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arnaud%2BEstoup%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Estoup%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArnaud%2BEstoup%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/671867v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1107113" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1107113/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/671867v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1107113" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1107113/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/671867v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1107113" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1107113/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/671867v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1107113" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1107113/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/671867v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1107113" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1107113/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">A<span class="sc">bstract</span></h2><p id="p-2">Dating population divergence within species from molecular data and relating such dating to climatic and biogeographic changes is not trivial. Yet it can help formulating evolutionary hypotheses regarding local adaptation and future responses to changing environments. Key issues include statistical selection of a demographic and historical scenario among a set of possible scenarios, and estimation of the parameter(s) of interest under the chosen scenario. Such inferences greatly benefit from new statistical approaches including approximate Bayesian computation - Random Forest (ABC-RF), the latter providing reliable inference at a low computational cost, with the possibility to take into account prior knowledge on both biogeographical history and genetic markers. Here, we used ABC-RF, including independent information on evolutionary rate and pattern at microsatellite markers, to decipher the evolutionary history of the African arid-adapted pest locust, <em>Schistocerca gregaria</em>. We found that the evolutionary processes that have shaped the present geographical distribution of the species in two disjoint northern and southern regions of Africa were recent, dating back 2.6 Ky (90% CI: 0.9 – 6.6 Ky). ABC-RF inferences also supported a southern colonization of Africa from a low number of founders of northern origin. The inferred divergence history is better explained by the peculiar biology of <em>S. gregaria</em>, which involves a density-dependent swarming phase with some exceptional spectacular migrations, rather than a continuous colonization resulting from the continental expansion of open vegetation habitats during more ancient Quaternary glacial climatic episodes.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">As in other regions of the world, Africa has gone through several major episodes of climate change since the early Pleistocene (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">deMenocal 1995</a> and <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2004</a>). The prevalent climate was colder and drier than nowadays during glaciation periods, and became more humid during warmer interglacial periods. These climatic phases resulted in shifts of vegetation (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">de Vivo and Carmignotto 2004</a>) and are most likely at the origin of the current isolation between northern and southern distributions of arid-adapted species (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Monod 1971</a>). In Africa, at least fifty-six plant species show disjoint geographical distributions in southern and northern arid areas (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Monod 1971</a>; Jurgens 1997; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lebrun 2001</a>). Similarly, a number of animal vertebrate species show meridian disjoint distributions on this continent, including eight mammals and 29 birds (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Monod 1971</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">de Vivo and Carmignotto 2004</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lorenzen <em>et al</em>. 2012</a>). The desert locust, <em>Schistocerca gregaria</em>, is among the few examples of insect species distributed in two distinct regions along the north-south axis of Africa. Other known disjunctions in insects are interspecific and concern species of the families Charilaidae (Orthoptera) and Mythicomyiidae (Diptera), and of the genus <em>Fidelia</em> (Hymenoptera) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Le Gall <em>et al</em>. 2010</a>). Similarities in extant distributions of African arid-adapted species across divergent taxonomic groups point to a common climatic history and an important role of environmental factors. Yet, to our knowledge, studies relating evolutionary history and climatic history have rarely been carried out in this continent; but see mitochondrial studies by <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Miller <em>et al</em>. 2011</a> on the ostrich, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Atickem <em>et al</em>. 2018</a> on the black-backed jackal, and <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Moodley <em>et al</em>. 2018</a> on the white rhinoceros.</p><p id="p-5">Relating evolutionary and climatic histories often requires dating population differentiation events so that species or subspecies divergence can be understood in a broad biogeographic context. However, finding a reliable calibration to convert measures of genetic divergence into units of absolute time is challenging, especially so for recent evolutionary events (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Ho <em>et al</em>. 2008</a>). Internal fossil records are often lacking and extra-specific fossil calibration may lead to considerable overestimates of divergence times (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Ho <em>et al</em>. 2008</a>). A sensible approach is to use an evolutionary rate estimated from sequence data of a related species for which internal fossil calibration is available (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Ho <em>et al</em>. 2008</a>). Unfortunately, on the African continent, fossils, such as radiocarbon-dated ancient samples, remain relatively rare and are often not representative of modern lineages (<em>e</em>.<em>g</em>., <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Le Gall <em>et al</em>. 2010</a> for insects). The lack of paleontological and archaeological records is partly due to their fragility under the aridity conditions of the Sahara. The end-result is that the options to relate population divergence to biogeographic events in this region are very limited. Finding a dating strategy that does not rely on fossils would therefore constitute a key advance in understanding the region’s biogeography.</p><p id="p-6">In this context, the use of versatile molecular markers, such as microsatellite loci, for which evolutionary rates can be obtained from direct observation of germline mutations in the species of interest, represents a useful alternative. Microsatellite mutation rates exceed by several orders of magnitude that of point mutation in DNA sequences, ranging from 10<sup>−6</sup> to 10<sup>−2</sup> events per locus and per generation (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ellegren 2000</a>). Such rates allow one to both observe mutation events in parent-offspring segregation data of realistic sample size and to reconstruct the recent history of related populations. However, the use of microsatellite loci to estimate divergence times at recent evolutionary time-scales still needs to overcome significant challenges. Since microsatellite allele sizes result from the insertion or deletion of single or multiple repeat units and are tightly constrained, these markers can be characterized by high levels of homoplasy that can obscure inferences about gene history (<em>e</em>.<em>g</em>., <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Estoup <em>et al</em>. 2002</a>). In particular, at large time scales (<em>i</em>.<em>e</em>., for distantly related populations), genetic distance values no longer follow a linear relationship with time. Rather, they reach a plateau and therefore provide biased and hence unreliable estimation of divergence time over a certain time threshold (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Takezaki and Nei 1996</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Feldman <em>et al</em>. 1997</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pollock <em>et al</em>. 1998</a>). Microsatellites remain informative with respect to divergence time only if the population split occurs within the period of linearity with time (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Feldman <em>et al</em>. 1997</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Pollock <em>et al</em>. 1998</a>). The exact value of the differentiation threshold above which microsatellite markers would no longer accurately reflect divergence times will depend on constraints on allele sizes and population-scaled mutation rates (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Feldman <em>et al</em>. 1997</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Pollock <em>et al</em>. 1998</a>). In this context, the approximate Bayesian computation - random forest (ABC-RF) approach recently proposed by <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Raynal <em>et al</em>. (2019)</a> is a singular statistical advance, as it allows both to take into account prior knowledge on genetic markers (including mutation rate and pattern) and to compute accuracy of parameter estimation at a local (<em>i</em>.<em>e</em>., posterior) scale. Using this methodological framework, one can envisage to evaluate the divergence time threshold above which posterior estimates of divergence time would become biased under a given evolutionary scenario, and hence in this way thoroughly evaluate the robustness of inferences about divergence time events.</p><p id="p-7">The desert locust, <em>S. gregaria</em>, is a generalist herbivore that can be found in arid grasslands and deserts in both northern and southern Africa (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). In its northern range, the desert locust is one of the most widespread and harmful agricultural pest species, with a very large potential outbreak area, spanning from West Africa to Southwest Asia. The desert locust is also present in the south-western arid zone (SWA) of Africa, which includes South-Africa, Namibia, Botswana and south-western Angola. The southern populations of the desert locust are termed <em>S. g. flaviventris</em> and are geographically separated by nearly 2,500 km from populations of the nominal subspecies from northern Africa, <em>S. g. gregaria</em> (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Uvarov 1977</a>). The isolation of <em>S. g. flaviventris</em> and <em>S. g. gregaria</em> lineages was recently supported by highlighting distinctive mitochondrial DNA haplotypes and male genitalia morphologies (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chapuis <em>et al</em>. 2016</a>). Yet, the precise history of divergence remains elusive.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Present time distribution range of Schistocerca gregaria in Africa under remission periods with winds in August a) and January b), and vegetation habitats suitable for the species during the present period c), the Holocene Climatic Optimum (HCO, 9 to 6 Ky ago) d), the Younger Dryas (YD, 12.9 to 11.7 Ky ago) e) and the Last Glacial Maximum (LGM, 26 to 14.8 Ky ago) f). (a-b) Distribution range and winds are adapted from Sword et al. (2010) and Nicholson (1996), respectively. (c-f) Vegetation habitats are adapted from Adams and Faure (1997). Open vegetation habitats suitable for the desert locust correspond to deserts (dark orange), xeric shrublands (light orange) and tropical - Mediterranean grasslands (pink). Other unsuitable habitat classes (white) are forests, woodlands and temperate shrublands and savannas." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Present time distribution range of Schistocerca gregaria in Africa under remission periods with winds in August a) and January b), and vegetation habitats suitable for the species during the present period c), the Holocene Climatic Optimum (HCO, 9 to 6 Ky ago) d), the Younger Dryas (YD, 12.9 to 11.7 Ky ago) e) and the Last Glacial Maximum (LGM, 26 to 14.8 Ky ago) f).&lt;/span&gt; (a-b) Distribution range and winds are adapted from Sword et al. (2010) and Nicholson (1996), respectively. (c-f) Vegetation habitats are adapted from Adams and Faure (1997). Open vegetation habitats suitable for the desert locust correspond to deserts (dark orange), xeric shrublands (light orange) and tropical - Mediterranean grasslands (pink). Other unsuitable habitat classes (white) are forests, woodlands and temperate shrublands and savannas.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F1.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F1.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Present time distribution range of Schistocerca gregaria in Africa under remission periods with winds in August a) and January b), and vegetation habitats suitable for the species during the present period c), the Holocene Climatic Optimum (HCO, 9 to 6 Ky ago) d), the Younger Dryas (YD, 12.9 to 11.7 Ky ago) e) and the Last Glacial Maximum (LGM, 26 to 14.8 Ky ago) f).</span><p id="p-8" class="first-child">(a-b) Distribution range and winds are adapted from <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Sword et al. (2010)</a> and <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nicholson (1996)</a>, respectively. (c-f) Vegetation habitats are adapted from <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams and Faure (1997)</a>. Open vegetation habitats suitable for the desert locust correspond to deserts (dark orange), xeric shrublands (light orange) and tropical - Mediterranean grasslands (pink). Other unsuitable habitat classes (white) are forests, woodlands and temperate shrublands and savannas.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">The main objective of the present study is to unravel the historical and evolutionary processes that have shaped the present disjoint geographical distribution of the desert locust and the genetic variation observed both within and between populations of its two subspecies. To this aim, we applied an ABC-RF approach and show its full potential to help discriminate between alternative biogeographic scenarios. We first start by identifying a set of evolutionary alternatives relevant to the species from African paleo-vegetation maps that reflect potential past distributions of the desert locust. We then used molecular data obtained from microsatellite markers for which we could obtain independent information on evolutionary rates and allele size constraints in the species of interest from direct observation of germline mutations (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chapuis <em>et al</em>. 2015</a>). We applied recently available algorithms of the ABC-RF on our microsatellite population genetic data to compare a set of thoroughly formalized and justified evolutionary scenarios and estimate the divergence time between <em>S. g. gregaria</em> and <em>S. g. flaviventris</em> under the most likely of our scenarios. Finally, we interpret our results in light of past vegetation cover and desert locust biology.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-10"><a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows the values of the summary statistics obtained from the observed population dataset consisting in two unstructured pooled samples of the subspecies <em>S. g. gregaria</em> and <em>S. g. flaviventris</em>. A total of 170 individuals (<em>i</em>.<em>e</em>., 80 and 90 individuals for <em>S. g. gregaria</em> and <em>S. g. flaviventris</em>, respectively) were genotyped at 23 microsatellite markers derived from either genomic DNA (14 loci) or messenger RNA (9 loci) resources (hereafter referred to as untranscribed and transcribed microsatellite markers, respectively). The level of differentiation between the two subspecies (as measured by the parameter <em>F</em><sub>ST</sub>) was 0.04 and 0.12 for untranscribed and transcribed microsatellite markers, respectively. The level of genetic diversity was higher within the northern subspecies <em>S. g. gregaria</em> (+7% and +14% for the mean number of alleles and expected heterozygosity, respectively).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111599/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111599/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111599" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-11" class="first-child">Summary statistics provided by DIYABC with corresponding values computed from the observed microsatellite dataset.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-3" class="subsection"><h3>Formalization and choice of evolutionary scenarios</h3><p id="p-12">Using a rich corpus of vegetation data, we reconstructed the present time (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1c</a>) and past time (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figs. 1d-f</a>) distribution ranges of <em>S. gregaria</em> in Africa, going back to the Last Glacial Maximum period (LGM, 26 to 14.8 Ky ago). Maps of vegetation cover for glacial arid maximums (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figs. 1e</a> and <a id="xref-fig-1-5" class="xref-fig" href="#F1">1f</a>) showed an expansion of open vegetation habitats sufficient to make the potential range of the species continuous from the Horn of Africa in the north-west to the Cape of Good Hope in the south. Maps of vegetation cover for interglacial humid maximums (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1d</a>) showed a severe contraction of deserts. These maps helped us formalize twelve competing evolutionary scenarios (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>), as well as bounds of prior distributions for various parameters (see the section <em>Prior setting for historical and demographical parameters</em> in Materials and methods). The twelve competing scenarios were generated by the combinations of presence <em>vs</em>. absence of three key evolutionary events that we identified as having potentially played a role in setting up the observed disjoint distribution of the two locust subspecies: (i) a long population size contraction in the ancestral population, due to the reduction of open vegetation habitats during the interglacial periods, (ii) a bottleneck in the southern subspecies <em>S. g. flaviventris</em> right after divergence reflecting a single long-distance migration event of a small fraction of the ancestral population, and (iii) a discrete genetic admixture event either unidirectional from the ancestral northern subspecies <em>S. g. gregaria</em> into <em>S. g. flaviventris</em>, or bidirectional, in order to consider the many climatic transitions of the last Quaternary.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolutionary scenarios compared using ABC-RF. The subscripts g, f and a refer to the subspecies S. g. gregaria, S. g. flaviventris and their unsampled common ancestor, respectively. Twelve scenarios are considered and identified by an acronym. Such scenarios differ by the presence or absence of three evolutionary events: a bottleneck in S. g. flaviventris (B) right after divergence between the two subspecies, a population size contraction in the ancestral population (C) and a discrete genetic admixture event either unidirectional from S. g. gregaria into S. g. flaviventris (AU) or bidirectional (AB). For convenience, parameters associated with an evolutionary event are represented graphically only in the first shown scenario that include this event. Looking forward in time, time periods are tC, the time of ancestral population size contraction, tS, the time of split between the two subspecies, and tA, the time of the genetic admixture between subspecies (with tC &gt; tS &gt; tA). rAU is the unidirectional admixture rate, i.e. the proportion of genes from the S. g. gregaria lineage entering the S. g. flaviventris population at time tA. rAB is the bidirectional admixture rate, i.e. the proportion of genes exchanged between the S. g. gregaria lineage and the S. g. flaviventris lineage at time tA. Ng, Nf and Na are the stable effective population sizes of S. g. gregaria, S. g. flaviventris and the ancestor, respectively. NC is the effective population size during the contraction event of duration dC in the ancestor. NB is the effective population size during the bottleneck event of duration dB." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Evolutionary scenarios compared using ABC-RF.&lt;/span&gt; The subscripts g, f and a refer to the subspecies S. g. gregaria, S. g. flaviventris and their unsampled common ancestor, respectively. Twelve scenarios are considered and identified by an acronym. Such scenarios differ by the presence or absence of three evolutionary events: a bottleneck in S. g. flaviventris (B) right after divergence between the two subspecies, a population size contraction in the ancestral population (C) and a discrete genetic admixture event either unidirectional from S. g. gregaria into S. g. flaviventris (AU) or bidirectional (AB). For convenience, parameters associated with an evolutionary event are represented graphically only in the first shown scenario that include this event. Looking forward in time, time periods are tC, the time of ancestral population size contraction, tS, the time of split between the two subspecies, and tA, the time of the genetic admixture between subspecies (with tC &gt; tS &gt; tA). rAU is the unidirectional admixture rate, i.e. the proportion of genes from the S. g. gregaria lineage entering the S. g. flaviventris population at time tA. rAB is the bidirectional admixture rate, i.e. the proportion of genes exchanged between the S. g. gregaria lineage and the S. g. flaviventris lineage at time tA. Ng, Nf and Na are the stable effective population sizes of S. g. gregaria, S. g. flaviventris and the ancestor, respectively. NC is the effective population size during the contraction event of duration dC in the ancestor. NB is the effective population size during the bottleneck event of duration dB.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F2.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F2.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Evolutionary scenarios compared using ABC-RF.</span><p id="p-13" class="first-child">The subscripts g, f and a refer to the subspecies S. g. gregaria, S. g. flaviventris and their unsampled common ancestor, respectively. Twelve scenarios are considered and identified by an acronym. Such scenarios differ by the presence or absence of three evolutionary events: a bottleneck in S. g. flaviventris (B) right after divergence between the two subspecies, a population size contraction in the ancestral population (C) and a discrete genetic admixture event either unidirectional from S. g. gregaria into S. g. flaviventris (AU) or bidirectional (AB). For convenience, parameters associated with an evolutionary event are represented graphically only in the first shown scenario that include this event. Looking forward in time, time periods are tC, the time of ancestral population size contraction, tS, the time of split between the two subspecies, and tA, the time of the genetic admixture between subspecies (with tC &gt; tS &gt; tA). rAU is the unidirectional admixture rate, i.e. the proportion of genes from the S. g. gregaria lineage entering the S. g. flaviventris population at time tA. rAB is the bidirectional admixture rate, i.e. the proportion of genes exchanged between the S. g. gregaria lineage and the S. g. flaviventris lineage at time tA. Ng, Nf and Na are the stable effective population sizes of S. g. gregaria, S. g. flaviventris and the ancestor, respectively. NC is the effective population size during the contraction event of duration dC in the ancestor. NB is the effective population size during the bottleneck event of duration dB.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">ABC-RF analyses supported the same group of scenarios or the same best individual scenario for all ten replicate analyses (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The classification votes and posterior probabilities estimated for the observed microsatellite dataset were the highest for the groups of scenarios in which (i) <em>S. g. flaviventris</em> experienced a bottleneck event at the time of the split (scenarios SB+SCB+SBA<sub>U</sub>+SBA<sub>B</sub>+SCBA<sub>U</sub>+SCBA<sub>B</sub> in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>; average of 2,829 votes out of 3,000 RF-trees; posterior probability = 0.926), (ii) the ancestral population experienced a population size contraction (scenarios SC+SCB+SCA<sub>U</sub>+SCA<sub>B</sub>+SCBA<sub>U</sub>+SCBA<sub>B</sub>; 2,035 of 3,000 RF-trees; posterior probability = 0.682), and (iii) no admixture event occurred between populations after the split (scenarios S+SC+SB+SCB; 2,013 of 3,000 RF-trees; posterior probability = 0.700). When considering the twelve scenarios separately, the highest classification vote was for scenario SCB (1,521 of 3,000 RF-trees), which congruently excludes a genetic admixture event and includes a population size contraction in the ancestral population as well as a bottleneck event at the time of divergence in the <em>S. g. flaviventris</em> subspecies. The posterior probability of scenario SCB averaged 0.564 over the ten replicate analyses (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table S1.1</a> (Supplementary Material S1) shows that only two other scenarios (SB and SCBA<sub>U</sub>) obtained at least 5% of the votes. The scenario SB included only a single bottleneck event in <em>S. g. flaviventris</em> (mean of 15.7% of votes) and the scenario SCBA<sub>U</sub> included a bottleneck event in <em>S. g. flaviventris</em>, a population size contraction in the ancestral population and an unidirectional genetic admixture event from <em>S. g. gregaria</em> into <em>S. g. flaviventris</em> (mean of 10.8% of votes). All other scenarios obtained less than 5% of the votes and were hence even more weakly supported.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111605/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111605/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111605" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-15" class="first-child">Scenario choice when analyzing groups of scenarios or scenarios separately.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">We found that the posterior error rates (<em>i</em>.<em>e</em>., 1 minus the posterior probabilities) were lower than the prior error rates for the analyses considering either groups of scenarios based on the presence (or not) of a bottleneck in <em>S. g. flaviventris</em> (<em>i</em>.<em>e</em>., 7.4% <em>versus</em> 19.0%) or the scenarios separately (<em>i</em>.<em>e</em>., 43.6% <em>versus</em> 58.4%). For other groups of scenarios, the discrimination power was similar at both the global (prior error rates) and local (posterior error rates) scales, with values ranging from 26.1% to 32.1% (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Altogether, these results indicate that the observed dataset belongs to a region of the data space where the power to discriminate among scenarios is higher than the global power computed over the whole prior data space, and that the presence or absence of a bottleneck in <em>S. g. flaviventris</em> is the demographic event with the most robust prediction in our ABC-RF treatments. These results can be visually illustrated by the projection of the reference table datasets and the observed one on a single (when analyzing pairwise groups of scenarios) or on the first two linear discriminant analysis (LDA) axes (when analyzing the twelve scenarios considered separately) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure S1.1</a>, Supplementary Material S1).</p><p id="p-17"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S1.2</a>, Supplementary Material S1, illustrates how RFs automatically rank the summary statistics according to their level of information. It shows that the number and set of most informative statistics is different depending on the comparisons (groups of scenarios or individual scenarios). Two sample statistics that measure the amount of genetic variation shared between populations (<em>F</em><sub>ST</sub>, DM2 and LIK) were among the most informative when discriminating among groups of scenarios including or not an admixture event. For groups of scenarios differing by population size variation events, statistics summarizing variation between the two subspecies samples (<em>F</em><sub>ST</sub> and DM2 for the bottleneck event in <em>S. g. flaviventris</em>; DAS and LIK for the population size contraction in the ancestral population) and statistics summarizing genetic variation within subspecies samples (mean expected heterozygosity and mean number of alleles for both population size variation events) were among the most discriminative ones. Only fifteen single sample statistics were not informative (according to their position relatively to the noise statistics added to our treatments) when considering the twelve individual scenarios separately. Most of those non informative statistics were associated to the set of transcribed microsatellites (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure S1.3</a>, Supplementary Material S1).</p></div><div id="sec-4" class="subsection"><h3>Parameter estimation under the best evolutionary scenario</h3><p id="p-18"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a</a> shows point estimates with 90% credibility intervals of the posterior distribution of the divergence time between the two subspecies under the best supported scenario SCB. Our estimations point to a young age of subspecies divergence, with a median divergence time of 2.6 Ky and a 90% credibility interval of 0.9 to 6.6 Ky, when assuming an average of three generations per year (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Roffey and Magor 2003</a>; see <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table S1.2</a>, Supplementary Material S1, for details). The accuracy of divergence time estimation was almost similar at both the global and local scales (<em>i</em>.<em>e</em>., normalized mean absolute errors of 0.369 and 0.359, respectively; <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Constraints on allele sizes in conjunction with high population-scaled mutation rates potentially strongly affect the linearity of the relationship between mutation accumulation and time of divergence estimated from microsatellite data. We thus evaluated the accuracy of ABC-RF estimation of the population divergence time as a function of the time scale, under scenario SCB. Analyses of simulated pseudo-observed datasets showed that the ABC-RF median estimate of divergence time reached a plateau for time scales ≥ 100,000 generations (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Thus, the divergence time between <em>S. g. flaviventris</em> and <em>S. g. gregaria</em> estimated on our real microsatellite dataset (∼10,000 generations) is positioned within the period of linearity with time, well before reaching a plateau reflecting a saturation of genetic information at microsatellite markers. It is hence expected to represent a sensible estimation of the actual divergence time.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111594/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111594/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111594" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <span class="caption-title">Parameter estimation under the best supported scenario (scenario SCB)</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Divergence time between S. g. gregaria and S. g. flaviventris inferred under the best supported scenario (scenario SCB) a) in relation to bioclimatic changes in Northern b) and Southern Africa c). a) Dashed and solid lines represent the formal subdivision of the Holocene and Pleistocene epochs (Walker et al. 2012). Dotted lines with labels on the right side are the median value and 90% credibility interval of the posterior density distributions of the divergence time (assuming an average of three generations per year; Roffey and Magor 2003). Asterisks refer to earliest archeological records of the desert locust. In the Algerian Sahara, remains of locusts were found in a special oven dating back to about 6Ky ago, in the rock shelter of Tin Hanakaten (Aumassip 2002). In Egypt, locusts were depicted on daggers of the pharaoh Ahmose, founder of the Eighteenth Dynasty (about 3.5 Ky ago) (Malek 1997) and, at Saqqara, on tombs of the Sixth Dynasty (about 4.2 to 4.4 Ky ago) that is thought to have felt with the impact of severe droughts (Meinzingen 1993). b-c) Climatic episodes include major cycles and additional transitions of aridity (sandy brown) and humidity (steel blue). The grey coloration means that there is debate on the climatic status of the period (arid versus humid). HCO: Holocene Climatic Optimum; YD: Younger Dryas; LGM: Last Glacial Maximum; LIG: Last Inter Glacial. Delimitations of climatic periods were based on published paleoclimatic inferences from geological sediment sequences (e.g., eolian deposition, oxygen isotope data) and biological records (e.g., pollen or insect fossils assemblages) from marine cores or terrestrial lakes. References are Bond et al. (1997), Guo et al.(2000), Kr&#xF6;pelin et al. (2008), Roberts et al. (1993) and van Andel and Tzedakis (1996) for northern Africa, and Talma and Vogel (1992), Stokes et al. (1997), and Shi et al. (1998) for southern Africa. See also Gasse (2000) for a review." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Divergence time between S. g. gregaria and S. g. flaviventris inferred under the best supported scenario (scenario SCB) a) in relation to bioclimatic changes in Northern b) and Southern Africa c).&lt;/span&gt; a) Dashed and solid lines represent the formal subdivision of the Holocene and Pleistocene epochs (Walker et al. 2012). Dotted lines with labels on the right side are the median value and 90% credibility interval of the posterior density distributions of the divergence time (assuming an average of three generations per year; Roffey and Magor 2003). Asterisks refer to earliest archeological records of the desert locust. In the Algerian Sahara, remains of locusts were found in a special oven dating back to about 6Ky ago, in the rock shelter of Tin Hanakaten (Aumassip 2002). In Egypt, locusts were depicted on daggers of the pharaoh Ahmose, founder of the Eighteenth Dynasty (about 3.5 Ky ago) (Malek 1997) and, at Saqqara, on tombs of the Sixth Dynasty (about 4.2 to 4.4 Ky ago) that is thought to have felt with the impact of severe droughts (Meinzingen 1993). b-c) Climatic episodes include major cycles and additional transitions of aridity (sandy brown) and humidity (steel blue). The grey coloration means that there is debate on the climatic status of the period (arid versus humid). HCO: Holocene Climatic Optimum; YD: Younger Dryas; LGM: Last Glacial Maximum; LIG: Last Inter Glacial. Delimitations of climatic periods were based on published paleoclimatic inferences from geological sediment sequences (e.g., eolian deposition, oxygen isotope data) and biological records (e.g., pollen or insect fossils assemblages) from marine cores or terrestrial lakes. References are Bond et al. (1997), Guo et al.(2000), Kr&#xF6;pelin et al. (2008), Roberts et al. (1993) and van Andel and Tzedakis (1996) for northern Africa, and Talma and Vogel (1992), Stokes et al. (1997), and Shi et al. (1998) for southern Africa. See also Gasse (2000) for a review.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F3.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F3.medium.gif" width="440" height="294"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Divergence time between S. g. gregaria and S. g. flaviventris inferred under the best supported scenario (scenario SCB) a) in relation to bioclimatic changes in Northern b) and Southern Africa c).</span><p id="p-19" class="first-child">a) Dashed and solid lines represent the formal subdivision of the Holocene and Pleistocene epochs (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Walker et al. 2012</a>). Dotted lines with labels on the right side are the median value and 90% credibility interval of the posterior density distributions of the divergence time (assuming an average of three generations per year; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Roffey and Magor 2003</a>). Asterisks refer to earliest archeological records of the desert locust. In the Algerian Sahara, remains of locusts were found in a special oven dating back to about 6Ky ago, in the rock shelter of Tin Hanakaten (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Aumassip 2002</a>). In Egypt, locusts were depicted on daggers of the pharaoh Ahmose, founder of the Eighteenth Dynasty (about 3.5 Ky ago) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Malek 1997</a>) and, at Saqqara, on tombs of the Sixth Dynasty (about 4.2 to 4.4 Ky ago) that is thought to have felt with the impact of severe droughts (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Meinzingen 1993</a>). b-c) Climatic episodes include major cycles and additional transitions of aridity (sandy brown) and humidity (steel blue). The grey coloration means that there is debate on the climatic status of the period (arid versus humid). HCO: Holocene Climatic Optimum; YD: Younger Dryas; LGM: Last Glacial Maximum; LIG: Last Inter Glacial. Delimitations of climatic periods were based on published paleoclimatic inferences from geological sediment sequences (e.g., eolian deposition, oxygen isotope data) and biological records (e.g., pollen or insect fossils assemblages) from marine cores or terrestrial lakes. References are <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bond et al. (1997)</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Guo et al.(2000)</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kröpelin et al. (2008)</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Roberts et al. (1993)</a> and <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">van Andel and Tzedakis (1996)</a> for northern Africa, and <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Talma and Vogel (1992)</a>, <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Stokes et al. (1997)</a>, and <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Shi et al. (1998)</a> for southern Africa. See also <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gasse (2000)</a> for a review.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Median and 90% credibility interval a) and local accuracy b) of ABC-RF posterior distributions of the divergence time under the best supported scenario (scenario SCB) as a function of the time scale. Simulated pseudo-observed datasets (5,000 per divergence time) were generated for fixed divergence time values of 100; 250; 500; 1,000; 2,500; 5,000; 10,000; 25,000; 50,000; 100,000; and 250,000 generations (cf. x-axis with a log-scale). a) The estimated median (plain lines) and 90% credibility interval (90% CI; dashed lines), averaged over the 5,000 datasets, are represented (y-axis). b) The local accuracy is measured using out-of-bag predictions and the normalized mean absolute error (NMAE). See Supplementary Material S3 for details." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Median and 90% credibility interval a) and local accuracy b) of ABC-RF posterior distributions of the divergence time under the best supported scenario (scenario SCB) as a function of the time scale.&lt;/span&gt; Simulated pseudo-observed datasets (5,000 per divergence time) were generated for fixed divergence time values of 100; 250; 500; 1,000; 2,500; 5,000; 10,000; 25,000; 50,000; 100,000; and 250,000 generations (cf. x-axis with a log-scale). a) The estimated median (plain lines) and 90% credibility interval (90% CI; dashed lines), averaged over the 5,000 datasets, are represented (y-axis). b) The local accuracy is measured using out-of-bag predictions and the normalized mean absolute error (NMAE). See Supplementary Material S3 for details.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F4.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F4.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Median and 90% credibility interval a) and local accuracy b) of ABC-RF posterior distributions of the divergence time under the best supported scenario (scenario SCB) as a function of the time scale.</span><p id="p-20" class="first-child">Simulated pseudo-observed datasets (5,000 per divergence time) were generated for fixed divergence time values of 100; 250; 500; 1,000; 2,500; 5,000; 10,000; 25,000; 50,000; 100,000; and 250,000 generations (cf. x-axis with a log-scale). a) The estimated median (plain lines) and 90% credibility interval (90% CI; dashed lines), averaged over the 5,000 datasets, are represented (y-axis). b) The local accuracy is measured using out-of-bag predictions and the normalized mean absolute error (NMAE). See Supplementary Material S3 for details.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Using the median as a point estimate, we estimated that the population size contraction in the ancestor could have occurred at a time approximately three times older than the divergence time between the subspecies (cf. estimation of the parameter ratio <em>t</em><sub><em>C</em></sub> <em>/ t</em><sub><em>S</em></sub> in <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Estimations of the ratio of stable effective sizes of the <em>S. g. gregaria</em> and <em>S. g. flaviventris</em> populations (<em>i</em>.<em>e</em>., <em>N</em><sub><em>f</em></sub> / <em>N</em><sub><em>g</em></sub>) showed large 90% credibility intervals and include the rate value of 1 (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). Accuracy analysis indicates that our genetic data withhold little information on the composite parameter <em>N</em><sub><em>f</em></sub> / <em>N</em><sub><em>g</em></sub> (cf. the high associated NMAE values in <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). The bottleneck intensity during the colonization of south-western Africa (corresponding to the composite parameter <em>d</em><sub><em>B</em></sub> / <em>N</em><sub><em>B</em></sub>) shows the highest accuracy of estimation among the parameters of interest (cf. the lowest associated NMAE values in <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>). The median of 1 and the 90% credibility interval of 0.5 to 2.4 support a strong to moderate bottleneck event (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-22">For the time since divergence between the two subspecies, the most informative statistics corresponded to the expected heterozygosity computed within the <em>S. g. flaviventris</em> sample and the mean index of classification from <em>S</em>. <em>g. flaviventris</em> to <em>S. g. gregaria</em> (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure S1.4</a>, Supplementary Material S1). The addition of noise variables in our treatments showed that most statistics characterizing genetic variation within the <em>S. g. gregaria</em> sample were not informative. The most informative summary statistics were different depending on the parameter of interest (results not shown).</p></div></div><div class="section" id="sec-5"><h2 class="">Discussion</h2><div id="sec-6" class="subsection"><h3>A young age of subspecific divergence</h3><p id="p-23">With a 90% credibility interval of the posterior density distribution of the divergence time at 0.9 to 6.6 Ky, our ABC-RF analyses clearly point to a divergence of the two desert locust subspecies occurring during the present Holocene geological epoch (0 to 11.7 Ky ago; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3a</a>). The posterior median estimate (2.6 Ky) and interquartile range (1.8 to 3.7 Ky) postdated the middle-late Holocene boundary (4.2 Ky). The latter past time boundary corresponds to the last transition from humid to arid conditions in the African continent (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3b</a>). This increasing aridity was shown to be a progressive change, with a concomitant maximum in northern and southern Africa at around 4.0 to 4.2 Ky ago, where aridity caused a contraction of the forest at its northern and southern peripheries without affecting its core region (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Guo <em>et al</em>. 2000</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Maley <em>et al</em>. 2018</a>). Interestingly, the earliest archeological records of the desert locust found in Tin Hanakaten (Algeria) and Saqqara (Egypt) archaeological sites date back to this period (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3b</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Meinzingen 1993</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Malek 1997</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Aumassip 2002</a>). Pollen records also showed that during this period the plant community was dominated by the desert and semi-desert taxa found today, including some species of prime importance for the current ecology of the desert locust (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Kröpelin <em>et al</em>. 2008</a>, <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Shi <em>et al</em>. 1998</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Duranton <em>et al</em>. 2012</a>). Then, the past 4 Ky are thought to have been under environmental stability and as dry as at present. One can therefore reasonably assume that, at the inferred divergence time between the two locust subspecies, the connectivity between the two African hemispheres was still limited by the moist equator, in particular at the west, and by the savannahs and woodlands of the eastern coast (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1c</a>). Consequently, contrary to most phylogeographic studies on other African arid-adapted species (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Atickem <em>et al</em>. 2018</a>, <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Moodley <em>et al</em>. 2018</a>), it is unlikely that the rather ancient Quaternary climatic history explained the Southern range extension of the desert locust.</p><p id="p-24">Recent geological and palynological research has shown that a brief fragmentation of the African primary forest occurred during the Holocene interglacial from 2.5 Ky to 2.0 Ky ago (reviewed in <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Maley <em>et al</em>. 2018</a>). This forest fragmentation period is characterized by relatively warm temperatures and a lengthening of the dry season rather than an arid climate. Although this period does not correspond to a phase of general expansion of savannas and grasslands, it led to the opening of the Sangha River Interval (SRI) in the core of the tropical forest in Central Africa (see <a id="xref-fig-1-8" class="xref-fig" href="#F1">Fig. 1</a> in <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Maley <em>et al</em>. 2018</a>). The SRI corresponds to a 400 km wide (14–18° E) open strip composed of savannas and grasslands dividing the rainforest in a north-south direction. The SRI corridor is thought to have facilitated the southern migration of Bantu-speaking pastoralists, along with cultivation of the semi-arid sub-Saharan cereal, pearl millet, <em>Pennisetum glaucum</em> (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Schwartz 1992</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bostoen <em>et al</em>. 2015</a>). The Bantu expansion took place between approximately 5 and 1.5 Ky ago and reached the southern range of the desert locust, including northern Namibia for the Western Bantu branch and southern Botswana and eastern South Africa for the Eastern Bantu branch (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Vansina 1995</a>). We cannot exclude that the recent subspecific distribution of the desert locust has been mediated by this recent ecological disturbance, which included a north-south corridor of open vegetation habitats and the diffusion of agricultural landscapes through the Bantu expansion. The progressive reappearance of forest vegetation 2 Ky ago would have then led to the present-day isolation and subsequent genetic differentiation of the new southern populations from northern parental populations. The level of climatic and habitat favorability of the SRI environmental disturbance to the species, and thereby the likelihood of a south-eastern colonization through the SRI corridor, remains however to be evaluated in line with further data on this period.</p></div><div id="sec-7" class="subsection"><h3>On the role of dispersal on subspecific divergence</h3><p id="p-25">Our ABC-RF results indicate that a demographic bottleneck (<em>i</em>.<em>e</em>., a strong transitory reduction of effective population size) occurred in the nascent southern subspecies of the desert locust. The high posterior probability value (92.6%) shows that this evolutionary event could be inferred with strong confidence. This result is compatible with the abovementioned colonization hypothesis if the proportion of suitable habitats for the desert locust in the SRI corridor was low, strongly limiting the carrying capacity during the time for range expansion. This scenario reduces, at the same time, the likelihood of a successful colonization through the SRI corridor. A more plausible alternative for a bottleneck event in <em>S. g. flaviventris</em> is a southern colonization of Africa through an exceptional long-distance migration event. In winged migratory species, movements are assisted by high velocity winds and may ascend to high altitudes (2000 m in the desert locust; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Uvarov 1977</a>) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Pedgley <em>et al</em>. 1995</a>). Although largest insects exert some control over their direction of migration by flying actively, or at least gliding within (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Pedgley <em>et al</em>. 1995</a>), accidental displacements in wind directions markedly different from that of seasonal dominant winds that assist migrations are likely. Such accidental displacements at long-distance (<em>i</em>.<em>e</em>., a few to several thousand km) were recorded in a number of migratory species, such as cicadas, butterflies, moths and locusts (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Pedgley <em>et al</em>. 1995</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lorenz, 2009</a>).</p><p id="p-26">To survive in its erratic arid and semi-arid habitat, the desert locust migrates downwind to reach areas where rain has recently fallen and exploit newly available resources. The dynamics of prevailing winds and pressure over Africa predicts the likelihood of a south-eastern transport of locusts (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Nicholson 1996</a>, <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Waloff and Pedgley 1986</a>). In northern Africa, at least since 2.7Ky, the strong northeast trade winds bring desert locust swarms equatorward in the moist intertropical convergence zone (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figures 1a-b</a>; Kröpelin <em>et al</em>. 1998). Most winds are westerlies (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figures 1a-b</a>), nevertheless easterly winds flow parallel to the eastern coast of Africa in northern winter (<em>e</em>.<em>g</em>., January; <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1b</a>). In southern Africa, winds blow mostly from the north-east toward the extant south-western distribution of the desert locust in southern winter (<em>e</em>.<em>g</em>., August; <a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1a</a>). In agreement with this, southward movements of desert locust have been documented along the eastern coast of Africa, in southern Tanzania during the plagues of 1926-1934, 1940-1948 and 1949-1963 (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Waloff 1976</a>), and even in Mozambique in January 1945 during the peak of the major plague of 1940-1948 (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Waloff 1966</a>).</p><p id="p-27">Furthermore, most travels off the range listed in the history of the desert locust were associated with plague events (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Waloff 1976</a>), with other records including Portugal, the British Isles and the famous trans-Atlantic crossing observed in October 1988, where large numbers of locusts landed in the Caribbean islands (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Richardson and Nemeth, 1991</a>). Plagues typically culminate several years of above average rainfall, resulting in abundant vegetation that supports both cumulative locust population growth and full development of gregarious characteristics (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Richardson and Nemeth, 1991</a>). The gigantic numbers (over a billion) of swarming locusts may facilitate the success of long-distance migrations, in spite of high mortality. In addition, flight capacity and endurance of the gregarious phase are remarkable, with swarms of winged adults regularly travelling up to hundreds km in a day, in case of sudden lack of resources (<a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Roffey and Magor 2003</a>). Therefore, out-of-range displacements of the desert locust may be explained by the synergistic effects of exceptional environmental conditions (<em>e</em>.<em>g</em>., unusual winds in strength or direction, rain favorable to plague) and peculiar biology of the gregarious phase of this species (<em>e</em>.<em>g</em>., swarming behavior, huge numbers of dispersers).</p><p id="p-28">In conclusion, dynamics of dominant winds in subtropical desert, historical records of exceptional migrations in the desert locust, and the very robust prediction for the presence of a bottleneck in <em>S. g. flaviventris</em> right after divergence by our ABC-RF treatments altogether support that a single or a few swarm(s) from the central region of the desert locust range sourced during a plague the colonization of south-western Africa, suggesting a role of dispersal in the disjoint distribution and divergence of desert locust populations. Interestingly, an exceptional trans-Atlantic flight of the desert locust from Africa to South America was revealed to give rise to the radiation of some 50 <em>Schistocerca</em> species in the western hemisphere (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lovejoy <em>et al</em>., 2006</a>).</p></div><div id="sec-8" class="subsection"><h3>On the influence of climatic cycles</h3><p id="p-29">It may appear surprising, at least at first sight, that the southern colonization of the desert locust did not occur during one of the major glacial episodes of the last Quaternary cycle, since these periods are characterized by a more continuous range of the desert locust (see paleo-vegetation maps in <a id="xref-fig-1-13" class="xref-fig" href="#F1">Figures 1e-f</a>). In particular, during the last glacial maximum (LGM, −14.8 Ky to −26 Ky), the Sahara desert extended hundreds of km further South than at present and annual precipitation were lower (i.e. ∼200–1,000 mm/year). Several hypotheses explain why our evolutionary scenario choice procedure provided low support to the possibility of a birth of the locust subspecies <em>S. g. flaviventris</em> at older periods. First, we cannot exclude that our microsatellite genetic data allow making inferences about the last colonization event only. Error rates at both local and global scales for the choice of scenario groups including or not a genetic admixture event after the split indicated that our discrimination power to infer this specific evolutionary event was poor (<em>i</em>.<em>e</em>., local and global error rates of 32.1% and 30.0%, respectively). The recent north-to-south colonization event selected by our ABC-RF treatment may hence have blurred traces of older colonization events.</p><p id="p-30">Second, while there is large evidence that much of Africa was drier during the last glacial phase, this remains debated for south-western Africa (see the gray coloration in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3b</a>). Some climate models show that at least some parts of this region, such as the Kalahari Desert, may have experienced higher rainfall than at present (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cockcroft <em>et al</em>. 1987</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ganopolski <em>et al</em>. 1998</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chase and Meadows 2007</a>). Such regional responses to glacial cycles may have prolonged until the middle Holocene. In particular, the northern Younger Dryas (<em>i</em>.<em>e</em>., −12.9 to −11.7 Ky) can be correlated only partly with an arid period in the southern hemisphere (<em>i</em>.<em>e</em>., −14.4 to −12.5 Ky). Such older climate episodes in antiphase between hemispheres (see the sandy brown coloration in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3b</a>) may have prevented from either a successful north-to-south migration event or a successful establishment and spread in the new southern range.</p><p id="p-31">Third, although semi-desert and desert biomes were more expanded than at present during the LGM, extreme aridity and lowered temperatures may have actually been unfavorable to the species. For instance, mean temperatures lowered by 5 to 6°C in both southern-western Africa (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Stute and Talma 1997</a>) and Central Sahara (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Edmunds <em>et al</em>. 1999</a>). The maintenance of desert locust populations depends on the proximity of areas with rainfalls at different seasons or with the capacity to capture and release water. For instance, in the African northern range, breeding success of locust populations relies on seasonal movements between the Sahel-Saharan zones of inter-tropical convergence, where the incidence of rain is high in summer, and the Mediterranean-Saharan transition zone, with a winter rainfall regime (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rainey and Waloff 1951</a>). In addition, adult migration and nymphal growth of the desert locust are dependent upon high temperature (<a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Roffey and Magor 2003</a>). It is hence possible then that the conjunction of hyper-aridity with intense cold could not easily support populations of the desert locust, despite the high extent of their migrations. Interestingly, climatic reconstructions during the Last Glacial Maximum (LGM) showed dramatic decreases of mean temperature throughout all of Africa and of precipitation in the Sahelian region (see <a id="xref-fig-13-1" class="xref-fig" href="#F13">Figs. S2.3</a> and <a id="xref-fig-14-1" class="xref-fig" href="#F14">S2.4</a>, Supplementary Material S2). Accordingly, species distribution modelling showed a LGM distribution for the desert locust very similar to its current distribution (<a id="xref-fig-15-1" class="xref-fig" href="#F15">Figs. S2.5</a>, <a id="xref-fig-16-1" class="xref-fig" href="#F16">S2.6</a> and <a id="xref-fig-17-1" class="xref-fig" href="#F17">S2.7</a>, Supplementary Material S2), excluding the hypothesis of a more continuous range of the species at this time.</p><p id="p-32">While ABC-RF analyses did not support that the Quaternary climatic history explained the subspecific divergence in the desert locust, they provided evidence for the occurrence of a large contraction of the size of the ancestral population preceding the divergence. Using the median as a point estimate, we estimated that the population size contraction in the ancestor could have occurred at a time about three fold older than the divergence time between the subspecies. This corresponds to the African humid period in the early and middle stages of the Holocene, though the large credibility interval also included the last interglacial period of the Pleistocene (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3b</a>). Such population size contraction was likely induced by the severe(s) contraction(s) of deserts that prevailed prior the estimated divergence between the two subspecies. Interestingly, these humid periods were more intense and prolonged in northern Africa, which corresponded to the presumed center of origin of the most recent common desert locust ancestor (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Scott 1993</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Partridge 1997</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Shi <em>et al</em>. 1998</a>).</p></div><div id="sec-9" class="subsection"><h3>Statistical advances by means of ABC Random Forest</h3><p id="p-33">To our knowledge, the present study is the first one using recently developed ABC-RF algorithms (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Raynal <em>et al</em>. 2019</a>) to carry out inferences about parameter estimation on a real multi-locus microsatellite dataset (for scenario choice see for example <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rougemont <em>et al</em>. 2016</a> and <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Fraimout <em>et al</em>. 2017</a>). When compared with various ABC solutions, this new RF method offers several advantages: a significant gain in terms of robustness to the choice of the summary statistics, independence from any type of tolerance level, and a good trade-off in terms of quality of point estimator precision of parameters and credible interval estimations for a given computing time (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Raynal <em>et al</em>. 2019</a>).</p><p id="p-34">The RF approach that we used here includes three novelties in statistical analyses that were particularly useful for reconstructing the evolutionary history of the divergence between <em>S. g. gregaria</em> and <em>S. g. flaviventris</em> subspecies. First, our ABC-RF statistical treatments benefited from the incorporation of previous estimations of mutation rates and allele size constraints for the microsatellite loci used in this study (for details see the Materials and methods section <em>Microsatellite dataset, mutation rate and mutational model</em>). Microsatellite mutation rate and pattern of most eukaryotes remains to a large extent unknown, and, to our knowledge, the present study is a rare one where independent information on mutational features was incorporated into the microsatellite prior distributions. Second, given the low RF computing time, we could simulate large pseudo-observed datasets to compute error measures conditionally to a subset of fixed divergence time values chosen to cover the entire prior interval. In this way, we showed that posterior distributions for estimation of divergence time between the two subspecies accurately reflected true divergence time values. Third, because error levels may differ depending on the location of an observed dataset in the prior data space (<em>e</em>.<em>g</em>., <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pudlo <em>et al</em>. 2016</a>), prior-based indicators are poorly relevant, aside from their use to select the best classification method and set of predictors, here our summary statistics. Therefore, in addition to global prior errors, we computed local posterior errors, conditionally to the observed dataset. The latter errors measure prediction quality exactly at the position of the observed dataset. For parameter estimation accuracy, we propose an innovative way to approximate local posterior errors, relying partly on out-of-bag predictions and implemented in a new version of the R library abcrf (version 1.8) available on R CRAN (see the Supplementary Material S3 for a detailed description). Such statistical advances are of general interest and will be useful for any statistical treatments of massive simulation data, including for inferences using single nucleotide polymorphisms (<em>i</em>.<em>e</em>., SNPs) obtained from new generation sequencing technologies.</p></div></div><div class="section" id="sec-10"><h2 class="">Materials &amp; Methods</h2><div id="sec-11" class="subsection"><h3>Formalization of evolutionary scenarios</h3><p id="p-35">To help formalize the evolutionary scenarios to be compared, we relied on maps of vegetation cover in Africa from the Quaternary Environment Network Atlas (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adams and Faure 1997</a>). We considered the periods representative of arid maximums (YD and LGM; <a id="xref-fig-1-14" class="xref-fig" href="#F1">Figs. 1e-f</a>), humid maximums (HCO; <a id="xref-fig-1-15" class="xref-fig" href="#F1">Fig. 1d</a>), and present-day arid conditions (<a id="xref-fig-1-16" class="xref-fig" href="#F1">Fig. 1c</a>), for which reliable vegetation reconstructions have been published. Desert and xeric shrubland cover fits well with the present-day species range during remission periods. Tropical and Mediterranean grasslands were added separately since the species inhabits such environments during outbreak periods only. The congruence between present maps of species distribution (<a id="xref-fig-1-17" class="xref-fig" href="#F1">Fig. 1a</a>) and of open vegetation habitats (<a id="xref-fig-1-18" class="xref-fig" href="#F1">Fig. 1c</a>) suggests that vegetation maps for more ancient periods could be considered as good approximations of the potential range of the desert locust in the past (but see section <em>On the influence of climatic cycles</em> in Discussion). Maps of vegetation cover during ice ages (<a id="xref-fig-1-19" class="xref-fig" href="#F1">Figs. 1e-f</a>) show an expansion of open vegetation habitats (<em>i</em>.<em>e</em>., grasslands in the tropics and deserts in both the North and South of Africa) sufficient to make the potential range of the species continuous from the Horn of Africa in North-West to the Cape of Good Hope in the South. It is worth stressing that we also explored species distribution modelling for the HCO and LGM periods as an alternative to using vegetation as a surrogate for the locust range (detailed in Supplementary Material S2). However, distribution modelling provided a narrower set of alternative hypotheses than the vegetation-based scenarios mentioned above and are therefore not discussed any further.</p><p id="p-36">Based on the above paleo-vegetation map reconstructions, we considered a set of alternative biogeographic hypotheses formulated into different types of evolutionary scenarios. First, we considered scenarios involving a more or less continuous colonization of southern Africa by the ancestral population from a northern origin. In this type of scenario, effective population sizes were allowed to change after the divergence event, without requiring any bottleneck event (<em>i</em>.<em>e</em>., without any abrupt and strong reduction of population size) right after divergence. Second, we considered the situation where the colonization of Southern Africa occurred through a single (or a few) long-distance migration event(s) of a small fraction of the ancestral population. This situation was formalized through scenarios that differed from a continuous colonization scenario by the occurrence right after divergence of a bottleneck event in the newly founded population (<em>i</em>.<em>e</em>., <em>S. g. flaviventris</em>), which was modelled through a limited number of founders during a short period.</p><p id="p-37">Because the last Quaternary cycle includes several arid climatic periods, including the intense punctuation of the Younger Dryas (YD) and the last glacial maximum (LGM), we also considered scenarios that incorporated the possibility of a discrete genetic admixture event, either bidirectional or unidirectional from <em>S. g. gregaria</em> into <em>S. g. flaviventris</em>. Since previous tests based on simulated data showed a poor power to discriminate between a single <em>versus</em> several admixture events (results not shown), we considered only scenarios including a single admixture event.</p><p id="p-38">Finally, at interglacial humid maximums, the map of vegetation cover showed a severe contraction of deserts, which were nearly completely vegetated with annual grasses and shrubs and supported numerous perennial lakes (<a id="xref-fig-1-20" class="xref-fig" href="#F1">Fig. 1d</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">deMenocal <em>et al</em>. 2000</a>). We thus considered the possibility that vegetation-induced contractions of population sizes have pre-dated the separation of the two subspecies. Hence, whereas so far scenarios involved a constant effective population size in the ancestral population, we formalized alternative scenarios in which we assumed that a long population size contraction event occurred into the ancestral population.</p><p id="p-39">Combining the presence or absence of the three above-mentioned key evolutionary events (a bottleneck in <em>S. g. flaviventris</em>, a bidirectional or unidirectional genetic admixture from <em>S. g. gregaria</em> into <em>S. g. flaviventris</em>, and a population size contraction in the ancestral population) allowed defining a total of twelve scenarios, that we compared using ABC-RF. The twelve scenarios with their historical and demographic parameters are graphically depicted in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>. All scenarios assumed a northern origin for the common ancestor of the two subspecies and a subsequent southern colonization of Africa. This assumption is supported by recent mitochondrial DNA data showing that <em>S. g. gregaria</em> have higher levels of genetic diversity and diagnostic bases shared with outgroup and congeneric species, whereas <em>S. g. flaviventris</em> clade was placed at the apical tip within the species tree (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chapuis <em>et al</em>. 2016</a>). In agreement with this assumption, preliminary analyses based on observed data showed a low support for a southern origin for the common ancestor of the two subspecies and a subsequent northern colonization of Africa (results not shown).</p><p id="p-40">All scenarios considered three populations of stable effective population sizes <em>N</em><sub><em>f</em></sub> for <em>S. g. flaviventris, N</em><sub><em>g</em></sub> for <em>S. g. gregaria</em>, and <em>N</em><sub><em>a</em></sub> for the ancestral population, with <em>S</em>. <em>g. flaviventris</em> and <em>S. g. gregaria</em> diverging <em>t</em><sub><em>S</em></sub> generations ago from the ancestral population. The bottleneck event which potentially occurred into <em>S. g. flaviventris</em> was modelled through a limited number of founders <em>N</em><sub><em>B</em></sub> during a short period <em>d</em><sub><em>B</em></sub>. The potential unidirectional genetic admixture into <em>S. g. flaviventris</em> occurred at a time <em>t</em><sub><em>A</em></sub>, with a proportion <em>r</em><sub><em>AU</em></sub> of genes of <em>S. g. gregaria</em> origin. In the case of a bidirectional genetic admixture, still occurring at a time <em>t</em><sub><em>A</em></sub>, the proportion of <em>S. g. gregaria</em> genes entering into the <em>S. g. flaviventris</em> population was <em>r</em><sub><em>AB</em></sub> and the proportion of <em>S. g. flaviventris</em> genes entering into the <em>S. g. gregaria</em> population was 1-<em>r</em><sub><em>AB</em></sub>. The potential population size contraction event occurred into the ancestral population at a time <em>t</em><sub><em>C</em></sub>, with an effective population size <em>N</em><sub><em>C</em></sub> during a duration <em>d</em><sub><em>C</em></sub>.</p></div><div id="sec-12" class="subsection"><h3>Prior setting for historical and demographical parameters</h3><p id="p-41">Prior values for time periods between sampling and admixture, divergence and/or ancestral population size contraction events (<em>t</em><sub>C</sub>, <em>t</em><sub>S</sub> and <em>t</em><sub>A</sub>, respectively) were drawn from log-uniform distributions bounded between 100 and 500,000 generations, with <em>t</em><sub>C</sub> &gt; <em>t</em><sub>S</sub> &gt; <em>t</em><sub>A</sub>. Assuming an average of three generations per year (<a id="xref-ref-63-5" class="xref-bibr" href="#ref-63">Roffey and Magor 2003</a>), this prior setting corresponds to a time period that goes back to the second-to-latest glacial maximum (150 Ky ago) (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">de Vivo and Carmignotto 2004</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">deMenocal <em>et al</em>. 2000</a>). Preliminary analyses showed that assuming a uniform prior shape for all time periods (instead of log-uniform distributions) do not change scenario choice results, with posterior probabilities only moderately affected, and this despite a substantial increase of out-of-bag prior error rates (<em>e</em>.<em>g</em>., + 50% when considering the twelve scenarios separately; <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table S4.1</a>, Supplementary Material S4). Analyses of simulated pseudo-observed datasets (pods) showed that assuming a uniform prior rather than a log-uniform prior for time period parameters would have also biased positively the median estimate of the divergence time and substantially increased its 90% credibility interval (<a id="xref-fig-22-1" class="xref-fig" href="#F22">Figure S4.1</a> and <a id="xref-table-wrap-9-1" class="xref-table" href="#T9">Table S4.2</a>, Supplementary Material S4). Using a log-uniform distribution remains a sensible choice for parameters with ranges of values covering several if not many log-intervals, as doing so allows assigning equal probabilities to each of the log-intervals.</p><p id="p-42">We used uniform prior distributions bounded between 1−10<sup>4</sup> and 1−10<sup>6</sup> diploid individuals for the different stable effective population sizes <em>N</em><sub><em>f</em></sub>, <em>N</em><sub><em>g</em></sub> <em>and N</em><sub><em>a</em></sub> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chapuis <em>et al</em>. 2014</a>). The admixture rate (<em>i</em>.<em>e</em>., the proportion of <em>S. g. gregaria</em> genes entering into the <em>S. g. flaviventris</em> population), was drawn from a uniform prior distribution bounded between 0.05 and 0.50 for a unidirectional event (<em>r</em><sub><em>AU</em></sub>) and between 0.05 and 0.95 for a bidirectional event (<em>r</em><sub><em>AB</em></sub>). We used uniform prior distributions bounded between 2 and 100 for both the numbers of founders (in diploid individuals) and durations of bottleneck events (in number of generations). For the contraction event, we used uniform prior distributions bounded between 100 and 10,000 for both the population size <em>N</em><sub><em>C</em></sub> (in diploid individuals) and duration <em>d</em><sub><em>C</em></sub> (in number of generations). Assuming an average of three generations per year (<a id="xref-ref-63-6" class="xref-bibr" href="#ref-63">Roffey and Magor 2003</a>), such prior choice allowed a reduction in population size for a short to a relatively long period, similar for instance to the whole duration of the HCO (from 9 to 5.5 Ky ago) which was characterized by a severe contraction of deserts.</p></div><div id="sec-13" class="subsection"><h3>Microsatellite dataset, mutation rate and mutational model</h3><p id="p-43">We carried out our statistical inference on the microsatellite dataset previously published in <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Chapuis <em>et al</em>. (2016)</a>. The 23 microsatellite loci genotyped in that dataset were derived from either genomic DNA (14 loci) or messenger RNA (9 loci) resources, and were hereafter referred to as untranscribed and transcribed microsatellite markers (following <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Blondin <em>et al</em>. 2013</a>). These microsatellites were shown to be genetically independent, free of null alleles and at selective neutrality (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Chapuis <em>et al</em>. 2016</a>). Previous levels of <em>F</em><sub>ST</sub> (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Weir 1996</a>) and Bayesian clustering analyses (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Pritchard <em>et al</em>. 2000</a>) among populations showed a weak genetic structuring within each subspecies (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chapuis <em>et al</em>. 2014</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2017</a>). For each subspecies, we selected and pooled three population samples in order to ensure both a large sample size (<em>i</em>.<em>e</em>., 80 and 90 individuals for <em>S. g. gregaria</em> and <em>S. g. flaviventris</em>, respectively), while ensuring a non-significant genetic structure within each subspecies pooled sample, as indicated by non-significant Fisher’s exact tests of genotypic differentiation among the three initial population samples within subspecies and exact tests of Hardy-Weinberg equilibrium for each subspecies pooled sample (<em>i</em>.<em>e</em>., <em>P</em>-value &gt; 0.05 when using Genepop 4.0; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Rousset 2008</a>). More precisely, the <em>S. g. gregaria</em> sample consisted in pooling the population samples 8, 15 and 22 of <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Chapuis <em>et al</em>. (2014)</a> and the <em>S. g. flaviventris</em> sample included the population samples 1, 2 and 6 of <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chapuis <em>et al</em>. (2017)</a>.</p><p id="p-44">Mutations occurring in the repeat region of each microsatellite locus were assumed to follow a symmetric generalized stepwise mutation model (GSM; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zhivotovsky <em>et al</em>. 1997</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Estoup <em>et al</em>. 2002</a>). Prior values for any mutation model settings were drawn independently for untranscribed and transcribed microsatellites in specific distributions. Because allele size constraints exist at microsatellite markers, we informed for each microsatellite locus their lower and upper allele size bounds using values estimated in <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chapuis <em>et al</em>. (2015)</a>, following the approach of <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Pollock <em>et al</em>. (1998)</a> and microsatellite data from several species closely related to <em>S. gregaria</em> (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Blondin <em>et al</em>. (2013)</a>. Prior values for the mean mutation rates <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-1.gif"/></noscript></span></span> were set to the empirical estimates inferred from observation of germline mutations in <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Chapuis <em>et al</em>. (2015)</a>, <em>i</em>.<em>e</em>., 2.8−10<sup>−4</sup> and 9.1−10<sup>−5</sup> for untranscribed and transcribed microsatellites, respectively. The mutation rates for individual microsatellites were then drawn from a Gamma distribution with <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-2.gif"/></noscript></span></span> and shape = 0.7 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Estoup <em>et al</em>. 2001</a>) for both types of microsatellites. We ensured that the chosen value of shape parameter generated the same inter-loci variance as estimated in <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Sun <em>et al</em>. (2012)</a> from direct observations of thousands of human microsatellites. Prior values for the mean parameters of the geometric distributions of the length in number of repeats of mutation events <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-3.gif"/></noscript></span></span> were set to the proportions of multistep germline mutations observed in <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Chapuis <em>et al</em>. (2015)</a>, <em>i</em>.<em>e</em>., 0.14 and 0.67 for untranscribed and transcribed microsatellites, respectively. The <em>P</em> parameters for individual loci were then standardly drawn from a Gamma distribution (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-4.gif"/></noscript></span></span> and shape = 2). We also considered mutations that insert or delete a single nucleotide to the microsatellite sequence. To model this mutational feature, we used the DIYABC default setting values (<em>i</em>.<em>e</em>., a uniform distribution bounded between [10<sup>−8</sup>, 10<sup>−5</sup>] for the mean parameter <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-5.gif"/></noscript></span></span> and a Gamma distribution (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-6.gif"/></noscript></span></span> and shape = 2) for individual loci parameters; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cornuet <em>et al</em>. 2010</a>; see also DIYABC user manual p. 13, <a href="http://www1.montpellier.inra.fr/CBGP/diyabc/">http://www1.montpellier.inra.fr/CBGP/diyabc/</a>).</p></div><div id="sec-14" class="subsection"><h3>Analyses using ABC Random Forest</h3><p id="p-45">We used the software DIYABC v.2.1.0 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cornuet <em>et al</em>. 2014</a>) to simulate datasets constituting the so-called reference tables (<em>i</em>.<em>e</em>., records of a given number of datasets simulated using the scenario ID and the evolutionary parameter values sampled from prior distributions and summarized with a pool of statistics). Random-forest computations were then performed using a new version of the R library abcrf (version 1.8) available on the CRAN. This version includes all ABC-RF algorithms detailed in <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Pudlo <em>et al</em>. (2016)</a>, <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Raynal <em>et al</em>. (2019)</a> and <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Estoup <em>et al</em>. (2018a)</a> for scenario choice and parameter estimation, as well as several statistical novelties allowing to compute error rates in scenario choice and accuracy measures for parameter estimation (see details below). An overview of the ABC-RF methods used in the present paper is provided in Supplementary Material S3. Readers can also consult <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Pudlo <em>et al</em>. (2016)</a>, <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Rougemont <em>et al</em>. (2016)</a>, <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Fraimout <em>et al</em>. (2017)</a>, <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Estoup <em>et al</em>. (2018a</a>,<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">b</a>) and <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Marin <em>et al</em>. (2018)</a> for scenario choice, and <a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Raynal <em>et al</em>. (2019)</a> for parameter estimation to access to further detailed statistical descriptions, testing and applications of ABC-RF algorithms.</p><p id="p-46">For scenario choice, the outcome of the first step of the ABC-RF statistical treatment applied to a given target dataset is a classification vote for each scenario which represents the number of times a scenario is selected in a forest of <em>n</em> trees. The scenario with the highest classification vote corresponds to the scenario best suited to the target dataset among the set of compared scenarios. This step also provides an error rate relevant to the entire prior sampling space, the global prior error. See the section <em>Global prior errors</em> in Supplementary Material S3 for details. The second RF analytical step provides a reliable estimation of the posterior probability of the best supported scenario. One minus such posterior probability yields the local posterior error associated to the observed dataset (see the section <em>Local posterior errors</em> in Supplementary Material S3). In practice, ABC-RF analyses were processed by drawing parameter values into the prior distributions described in the two previous sections and by summarizing microsatellite data using a set of 32 statistics (see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> for details about such summary statistics) and the one LDA axis or eleven LDA axes (i.e. number of scenarios minus 1; <a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Pudlo <em>et al</em>. 2016</a>) computed when considering pairwise groups of scenarios or individual scenarios, respectively. We processed ABC-RF treatments on reference tables including 150,000 simulated datasets (<em>i</em>.<em>e</em>., 12,500 per scenario). Following <a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Pudlo et al. (2016)</a>, we checked that 150,000 datasets was sufficient by evaluating the stability of prior error rates and posterior probabilities estimations of the best scenario on 50,000, 80,000 and 100,000 and 130,000 simulated datasets (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table S1.3</a>, Supplementary Material S1). The number of trees in the constructed random forests was fixed to <em>n</em> = 3,000, as this number turned out to be large enough to ensure a stable estimation of the prior error rate (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure S1.5</a>, Supplementary Material S1). We predicted the best scenario and estimated its posterior probability and prior error rate over ten replicate analyses based on ten different reference tables.</p><p id="p-47">In order to assess the power of our approach to infer each specific evolutionary event of interest, we first processed ABC-RF analyses by grouping scenarios based on the presence or absence of each type of evolutionary event that we identified as having potentially played a role in setting up the disjoint distribution of the two locust subspecies (<em>e</em>.<em>g</em>., <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Roux <em>et al</em>. 2016</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Leroy <em>et al</em>. 2017</a> and <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Estoup <em>et al</em>. 2018a</a>). We conducted ABC-RF treatments on three pairwise groups of scenario: groups of scenarios with <em>vs</em>. without a bottleneck in <em>S. g. flaviventris</em>, groups with <em>vs</em>. without a population size contraction in the ancestral population, and groups with <em>vs</em>. without an asymmetrical genetic admixture event from <em>S. g. gregaria</em> into <em>S. g. flaviventris</em>. We then conducted ABC-RF treatments on the twelve scenarios considered separately.</p><p id="p-48">For parameter estimation, we conducted ten independent replicate RF treatments based on ten different reference tables for each parameter of interest (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Raynal <em>et al</em>. 2019</a>): the time since divergence, the ratio of the time of the contraction event into the ancestral population on the time since divergence, the intensity of the bottleneck event in the sampled <em>S. g. flaviventris</em> population (defined as the ratio of the bottleneck event of duration <em>d</em><sub><em>B</em></sub> on the effective population size <em>N</em><sub><em>B</em></sub>) and the ratio of the stable effective population size of the two sampled populations. For each RF treatment, we simulated a total of 100,000 datasets for the selected scenario (drawing parameter values into the prior distributions described in the two previous sections and using the same 32 summary statistics). Following <a id="xref-ref-60-7" class="xref-bibr" href="#ref-60">Raynal <em>et al</em>. (2019)</a>, we checked that 100,000 datasets was sufficient by evaluating the stability of the measure of accuracy on divergence time estimation using 50,000, 80,000 and 90,000 simulated datasets (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table S1.4</a>, Supplementary Material S1). The number of trees in the constructed random forests was fixed to <em>n</em> = 2,000, as such number turned out to be large enough to ensure a stable estimation of the measure of divergence time estimation accuracy (<a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure S1.6</a>, Supplementary Material S1). For each RF treatment, we estimated the median value and the 5% and 95% quantiles of the posterior distributions. It is worth noting that we considered median values as the later provided more accurate estimations (according to out-of-bag predictions) than when considering mean values (results not shown). Accuracy of parameter estimation was measured using out-of-bag predictions and the normalized mean absolute error (NMAE). NMAE corresponds to the mean of the absolute difference between the point estimate (here the median) and the (true) simulated value divided by the simulated value (formula detailed in Supplementary Material S3).</p><p id="p-49">Finally, because microsatellite markers tend to underestimate divergence time for large time scales due to allele size constraints, we evaluated how the accuracy of ABC-RF estimation of the time of divergence between the two subspecies was sensitive to the time scale. To this aim, we used DIYABC to produce simulated pseudo-observed datasets assuming fixed divergence time values chosen to cover the prior interval (100; 250; 500; 1,000; 2,500; 5,000; 10,000; 25,000; 50,000; 100,000; 250,000 generations) and using the best scenario. We simulated 5,000 of such test datasets for each of the eleven divergence time values. Each of these test dataset was treated using ABC-RF in the same way as the above target observed dataset.</p></div></div><div class="section" id="sec-15"><h2 class="">Data accessibility</h2><p id="p-50">Microsatellite genotypic data and R scripts for ABC-RF analyses are publicly available on the CIRAD Dataverse repository (<a href="https://dataverse.cirad.fr/">https://dataverse.cirad.fr/</a>) at <a href="https://doi.org/10.18167/DVN1/ZNSKGK">https://doi.org/10.18167/DVN1/ZNSKGK</a>.</p></div><div class="section" id="sec-16"><h2 class="">Supplementary material</h2><p id="p-51">Additional supporting information may be found in the online version of this article.</p></div><div class="section" id="sec-17"><h2 class="">Conflict of interest disclosure</h2><p id="p-53">The authors of this preprint declare that they have no financial conflict of interest with the content of this article. Arnaud Estoup is one of the <em>PCI Evolutionary Biology</em> recommenders.</p></div><div class="section" id="sec-18"><h2 class="">Supplementary material online for the manuscript “A young age of subspecific divergence in the desert locust <em>Schistocerca gregaria</em> inferred by ABC Random Forest”</h2><div class="section" id="sec-19"><h3>Supplementary Material S1. Details on results from ABC-RF treatments</h3><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111591/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111591/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111591" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1.1.</span> <span class="caption-title">Scenario choice for each of the ten replicate analyses.</span><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111614/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111614/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111614" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1.2.</span> <p id="p-54" class="first-child">Estimation of the divergence time between <em>S. g. gregaria</em> and <em>S. g. flaviventris</em> for the ten replicate analyses processed under the best supported scenario (scenario SCB).</p><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111606/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111606/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111606" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1.3.</span> <span class="caption-title">Effect of the number of simulated datasets in the reference table on scenario choice.</span><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111615/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111615/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111615" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S1.4.</span> <p id="p-55" class="first-child">Effect of the number of simulated datasets in the reference table on posterior point estimation values a) and estimation accuracy b) of the divergence time between <em>S. g. gregaria</em> and <em>S. g. f laviventris</em> under the best supported scenario (scenario SCB).</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Projection on a single (when analyzing pairwise groups of scenarios) or on the first two LDA axes (when analyzing the twelve scenarios separately) of the observed dataset and the simulated datasets recorded in the reference table. Colors correspond to group of scenarios or individual scenarios. The location of the desert locust observed dataset is indicated by a vertical black line or a star. Scenarios were grouped based on the presence or not of a bottleneck in S. g. flaviventris (B or no B), a population size contraction in ancestor (C or no C) and a genetic admixture either unidirectional from S. g. gregaria in to S. g. flaviventris or bidirectional (AU or AB or no A). When considering the whole set of twelve scenarios separately d), the projected points substantially overlapped for at least some of the scenarios. This suggests an overall low power to discriminate among scenarios considered. Conversely, considering pairwise groups of scenarios, one can observe a weaker overlap of projected points (at least for a) and b)) suggesting a stronger power to discriminate among groups of scenarios of interest than when considering all scenarios separately. One can note that the location of the observed dataset (indicated by a vertical line) suggests an association with the scenario group with a bottleneck event in S. g. flaviventris and with the scenario group with a population size contraction in the ancestral population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Projection on a single (when analyzing pairwise groups of scenarios) or on the first two LDA axes (when analyzing the twelve scenarios separately) of the observed dataset and the simulated datasets recorded in the reference table.&lt;/span&gt; Colors correspond to group of scenarios or individual scenarios. The location of the desert locust observed dataset is indicated by a vertical black line or a star. Scenarios were grouped based on the presence or not of a bottleneck in &lt;em&gt;S. g. flaviventris&lt;/em&gt; (B or no B), a population size contraction in ancestor (C or no C) and a genetic admixture either unidirectional from &lt;em&gt;S. g. gregaria in&lt;/em&gt; to &lt;em&gt;S. g. flaviventris&lt;/em&gt; or bidirectional (A&lt;sub&gt;U&lt;/sub&gt; or A&lt;sub&gt;B&lt;/sub&gt; or no A). When considering the whole set of twelve scenarios separately d), the projected points substantially overlapped for at least some of the scenarios. This suggests an overall low power to discriminate among scenarios considered. Conversely, considering pairwise groups of scenarios, one can observe a weaker overlap of projected points (at least for a) and b)) suggesting a stronger power to discriminate among groups of scenarios of interest than when considering all scenarios separately. One can note that the location of the observed dataset (indicated by a vertical line) suggests an association with the scenario group with a bottleneck event in &lt;em&gt;S. g. flaviventris&lt;/em&gt; and with the scenario group with a population size contraction in the ancestral population.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-12.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-12.medium.gif" width="440" height="200"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Projection on a single (when analyzing pairwise groups of scenarios) or on the first two LDA axes (when analyzing the twelve scenarios separately) of the observed dataset and the simulated datasets recorded in the reference table. Colors correspond to group of scenarios or individual scenarios. The location of the desert locust observed dataset is indicated by a vertical black line or a star. Scenarios were grouped based on the presence or not of a bottleneck in S. g. flaviventris (B or no B), a population size contraction in ancestor (C or no C) and a genetic admixture either unidirectional from S. g. gregaria in to S. g. flaviventris or bidirectional (AU or AB or no A). When considering the whole set of twelve scenarios separately d), the projected points substantially overlapped for at least some of the scenarios. This suggests an overall low power to discriminate among scenarios considered. Conversely, considering pairwise groups of scenarios, one can observe a weaker overlap of projected points (at least for a) and b)) suggesting a stronger power to discriminate among groups of scenarios of interest than when considering all scenarios separately. One can note that the location of the observed dataset (indicated by a vertical line) suggests an association with the scenario group with a bottleneck event in S. g. flaviventris and with the scenario group with a population size contraction in the ancestral population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Projection on a single (when analyzing pairwise groups of scenarios) or on the first two LDA axes (when analyzing the twelve scenarios separately) of the observed dataset and the simulated datasets recorded in the reference table.&lt;/span&gt; Colors correspond to group of scenarios or individual scenarios. The location of the desert locust observed dataset is indicated by a vertical black line or a star. Scenarios were grouped based on the presence or not of a bottleneck in &lt;em&gt;S. g. flaviventris&lt;/em&gt; (B or no B), a population size contraction in ancestor (C or no C) and a genetic admixture either unidirectional from &lt;em&gt;S. g. gregaria in&lt;/em&gt; to &lt;em&gt;S. g. flaviventris&lt;/em&gt; or bidirectional (A&lt;sub&gt;U&lt;/sub&gt; or A&lt;sub&gt;B&lt;/sub&gt; or no A). When considering the whole set of twelve scenarios separately d), the projected points substantially overlapped for at least some of the scenarios. This suggests an overall low power to discriminate among scenarios considered. Conversely, considering pairwise groups of scenarios, one can observe a weaker overlap of projected points (at least for a) and b)) suggesting a stronger power to discriminate among groups of scenarios of interest than when considering all scenarios separately. One can note that the location of the observed dataset (indicated by a vertical line) suggests an association with the scenario group with a bottleneck event in &lt;em&gt;S. g. flaviventris&lt;/em&gt; and with the scenario group with a population size contraction in the ancestral population.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-13.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-13.medium.gif" width="440" height="215"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F5/graphic-13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure S1.1.</span> <span class="caption-title">Projection on a single (when analyzing pairwise groups of scenarios) or on the first two LDA axes (when analyzing the twelve scenarios separately) of the observed dataset and the simulated datasets recorded in the reference table.</span><p id="p-56" class="first-child">Colors correspond to group of scenarios or individual scenarios. The location of the desert locust observed dataset is indicated by a vertical black line or a star. Scenarios were grouped based on the presence or not of a bottleneck in <em>S. g. flaviventris</em> (B or no B), a population size contraction in ancestor (C or no C) and a genetic admixture either unidirectional from <em>S. g. gregaria in</em> to <em>S. g. flaviventris</em> or bidirectional (A<sub>U</sub> or A<sub>B</sub> or no A). When considering the whole set of twelve scenarios separately d), the projected points substantially overlapped for at least some of the scenarios. This suggests an overall low power to discriminate among scenarios considered. Conversely, considering pairwise groups of scenarios, one can observe a weaker overlap of projected points (at least for a) and b)) suggesting a stronger power to discriminate among groups of scenarios of interest than when considering all scenarios separately. One can note that the location of the observed dataset (indicated by a vertical line) suggests an association with the scenario group with a bottleneck event in <em>S. g. flaviventris</em> and with the scenario group with a population size contraction in the ancestral population.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contributions of ABC-RF summary statistics when choosing between groups of scenarios. The contribution of each 32 summary statistics and one LDA axis (or two LDA axes) is evaluated as the total amount of decrease in the Gini criterion (variable importance on the x-axis). The higher the contribution of the statistics, the more informative it is in the inferential process. The microsatellite set and subspecies sample are indicated at the end of each statistics by indices k_i for single population statistics and k_i.j for two population statistics, with k=1 for the set of untranscribed microsatellites or k=2 for the set of transcribed microsatellites, and i(j)=1 for the S. g. flaviventris subspecies or and i(j)=2 for the S. g. gregaria subspecies. See Table S1 for details on the summary statistics abbreviations. Five noise variables, randomly drawn into uniform distributions bounded between 0 and 1, and denoted NOISE1 to NOISE5 were added to the set of summary statistics processed by RF, in order to evaluate from which amount of decrease in the Gini criterion the summary statistics computed from our genetic datasets were not informative anymore (indicated by a red star). a) B = demographic bottleneck event, b) C = demographic contraction event and c) AU or AB or no A = unidirectional from S. g. gregaria into S. g. flaviventris or bidirectional genetic admixture event." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Contributions of ABC-RF summary statistics when choosing between groups of scenarios.&lt;/span&gt; The contribution of each 32 summary statistics and one LDA axis (or two LDA axes) is evaluated as the total amount of decrease in the Gini criterion (variable importance on the x-axis). The higher the contribution of the statistics, the more informative it is in the inferential process. The microsatellite set and subspecies sample are indicated at the end of each statistics by indices &lt;em&gt;k_i&lt;/em&gt; for single population statistics and &lt;em&gt;k_i&lt;/em&gt;.&lt;em&gt;j&lt;/em&gt; for two population statistics, with &lt;em&gt;k&lt;/em&gt;=1 for the set of untranscribed microsatellites or &lt;em&gt;k&lt;/em&gt;=2 for the set of transcribed microsatellites, and &lt;em&gt;i&lt;/em&gt;(&lt;em&gt;j&lt;/em&gt;)=1 for the &lt;em&gt;S. g. flaviventris&lt;/em&gt; subspecies or and &lt;em&gt;i&lt;/em&gt;(&lt;em&gt;j&lt;/em&gt;)=2 for the &lt;em&gt;S. g. gregaria&lt;/em&gt; subspecies. See Table S1 for details on the summary statistics abbreviations. Five noise variables, randomly drawn into uniform distributions bounded between 0 and 1, and denoted NOISE1 to NOISE5 were added to the set of summary statistics processed by RF, in order to evaluate from which amount of decrease in the Gini criterion the summary statistics computed from our genetic datasets were not informative anymore (indicated by a red star). a) B = demographic bottleneck event, b) C = demographic contraction event and c) AU or AB or no A = unidirectional from &lt;em&gt;S. g. gregaria&lt;/em&gt; into &lt;em&gt;S. g. flaviventris&lt;/em&gt; or bidirectional genetic admixture event.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1.2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F6.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1.2." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F6.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1.2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1.2.</span> <span class="caption-title">Contributions of ABC-RF summary statistics when choosing between groups of scenarios.</span><p id="p-57" class="first-child">The contribution of each 32 summary statistics and one LDA axis (or two LDA axes) is evaluated as the total amount of decrease in the Gini criterion (variable importance on the x-axis). The higher the contribution of the statistics, the more informative it is in the inferential process. The microsatellite set and subspecies sample are indicated at the end of each statistics by indices <em>k_i</em> for single population statistics and <em>k_i</em>.<em>j</em> for two population statistics, with <em>k</em>=1 for the set of untranscribed microsatellites or <em>k</em>=2 for the set of transcribed microsatellites, and <em>i</em>(<em>j</em>)=1 for the <em>S. g. flaviventris</em> subspecies or and <em>i</em>(<em>j</em>)=2 for the <em>S. g. gregaria</em> subspecies. See Table S1 for details on the summary statistics abbreviations. Five noise variables, randomly drawn into uniform distributions bounded between 0 and 1, and denoted NOISE1 to NOISE5 were added to the set of summary statistics processed by RF, in order to evaluate from which amount of decrease in the Gini criterion the summary statistics computed from our genetic datasets were not informative anymore (indicated by a red star). a) B = demographic bottleneck event, b) C = demographic contraction event and c) AU or AB or no A = unidirectional from <em>S. g. gregaria</em> into <em>S. g. flaviventris</em> or bidirectional genetic admixture event.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contributions of ABC-RF summary statistics when choosing among the twelve individual scenarios. The contribution of each 32 summary statistics and eleven LDA axes is evaluated as the total amount of decrease in the Gini criterion (variable importance on the x-axis). The higher the contribution of the statistics, the more informative it is in the inferential process. The microsatellite set and subspecies sample are indicated at the end of each statistics by indices k_i for single population statistics and k_i.j for two population statistics, with k=1 for the set of untranscribed microsatellites or k=2 for the set of transcribed microsatellites, and i(j)=1 for the S. g. flaviventris subspecies or and i(j)=2 for the S. g. gregaria subspecies. See Table 1 for details on the summary statistics abbreviations. Five noise variables, randomly drawn into uniform distributions bounded between 0 and 1, and denoted NOISE1 to NOISE5 were added to the set of summary statistics processed by RF, in order to evaluate from which amount of decrease in the Gini criterion the summary statistics computed from our genetic datasets were not informative anymore (indicated by a red star)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Contributions of ABC-RF summary statistics when choosing among the twelve individual scenarios.&lt;/span&gt; The contribution of each 32 summary statistics and eleven LDA axes is evaluated as the total amount of decrease in the Gini criterion (variable importance on the x-axis). The higher the contribution of the statistics, the more informative it is in the inferential process. The microsatellite set and subspecies sample are indicated at the end of each statistics by indices &lt;em&gt;k_i&lt;/em&gt; for single population statistics and &lt;em&gt;k_i&lt;/em&gt;.&lt;em&gt;j&lt;/em&gt; for two population statistics, with &lt;em&gt;k&lt;/em&gt;=1 for the set of untranscribed microsatellites or &lt;em&gt;k&lt;/em&gt;=2 for the set of transcribed microsatellites, and &lt;em&gt;i&lt;/em&gt;(&lt;em&gt;j&lt;/em&gt;)=1 for the &lt;em&gt;S. g. flaviventris&lt;/em&gt; subspecies or and &lt;em&gt;i&lt;/em&gt;(&lt;em&gt;j&lt;/em&gt;)=2 for the &lt;em&gt;S. g. gregaria&lt;/em&gt; subspecies. See Table 1 for details on the summary statistics abbreviations. Five noise variables, randomly drawn into uniform distributions bounded between 0 and 1, and denoted NOISE1 to NOISE5 were added to the set of summary statistics processed by RF, in order to evaluate from which amount of decrease in the Gini criterion the summary statistics computed from our genetic datasets were not informative anymore (indicated by a red star).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1.3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F7.medium.gif" width="265" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1.3." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F7.medium.gif" width="265" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1.3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1.3.</span> <span class="caption-title">Contributions of ABC-RF summary statistics when choosing among the twelve individual scenarios.</span><p id="p-58" class="first-child">The contribution of each 32 summary statistics and eleven LDA axes is evaluated as the total amount of decrease in the Gini criterion (variable importance on the x-axis). The higher the contribution of the statistics, the more informative it is in the inferential process. The microsatellite set and subspecies sample are indicated at the end of each statistics by indices <em>k_i</em> for single population statistics and <em>k_i</em>.<em>j</em> for two population statistics, with <em>k</em>=1 for the set of untranscribed microsatellites or <em>k</em>=2 for the set of transcribed microsatellites, and <em>i</em>(<em>j</em>)=1 for the <em>S. g. flaviventris</em> subspecies or and <em>i</em>(<em>j</em>)=2 for the <em>S. g. gregaria</em> subspecies. See <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> for details on the summary statistics abbreviations. Five noise variables, randomly drawn into uniform distributions bounded between 0 and 1, and denoted NOISE1 to NOISE5 were added to the set of summary statistics processed by RF, in order to evaluate from which amount of decrease in the Gini criterion the summary statistics computed from our genetic datasets were not informative anymore (indicated by a red star).</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contributions of ABC-RF summary statistics when estimating the divergence time between the two desert locust subspecies under the best supported scenario (scenario SCB). The contribution of each 32 summary statistics is evaluated as the total amount of decrease of the residual sum of squares, divided by the number of trees, (variable importance on the x-axis). The higher the contribution of the statistics, the more informative it is in the inferential process. The microsatellite set and subspecies sample are indicated at the end of each statistics by indices k_i for single population statistics and k_i.j for two population statistics, with k=1 for the set of untranscribed microsatellites or k=2 for the set of transcribed microsatellites, and i(j)=1 for the S. g. flaviventris subspecies or and i(j)=2 for the S. g. gregaria subspecies. See Table S6.1 for details on the summary statistics abbreviations. Five noise variables, randomly drawn into uniform distributions bounded between 0 and 1, and denoted NOISE1 to NOISE5 were added to the set of summary statistics processed by RF, in order to evaluate from which amount of decrease in the variable importance criterion the summary statistics computed from our genetic datasets were not informative anymore (indicated by a red star)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Contributions of ABC-RF summary statistics when estimating the divergence time between the two desert locust subspecies under the best supported scenario (scenario SCB).&lt;/span&gt; The contribution of each 32 summary statistics is evaluated as the total amount of decrease of the residual sum of squares, divided by the number of trees, (variable importance on the x-axis). The higher the contribution of the statistics, the more informative it is in the inferential process. The microsatellite set and subspecies sample are indicated at the end of each statistics by indices &lt;em&gt;k_i&lt;/em&gt; for single population statistics and &lt;em&gt;k_i&lt;/em&gt;.&lt;em&gt;j&lt;/em&gt; for two population statistics, with &lt;em&gt;k&lt;/em&gt;=1 for the set of untranscribed microsatellites or &lt;em&gt;k&lt;/em&gt;=2 for the set of transcribed microsatellites, and &lt;em&gt;i&lt;/em&gt;(&lt;em&gt;j&lt;/em&gt;)=1 for the &lt;em&gt;S. g. flaviventris&lt;/em&gt; subspecies or and &lt;em&gt;i&lt;/em&gt;(&lt;em&gt;j&lt;/em&gt;)=2 for the &lt;em&gt;S. g. gregaria&lt;/em&gt; subspecies. See Table S6.1 for details on the summary statistics abbreviations. Five noise variables, randomly drawn into uniform distributions bounded between 0 and 1, and denoted NOISE1 to NOISE5 were added to the set of summary statistics processed by RF, in order to evaluate from which amount of decrease in the variable importance criterion the summary statistics computed from our genetic datasets were not informative anymore (indicated by a red star).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1.4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F8.medium.gif" width="285" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1.4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F8.medium.gif" width="285" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1.4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1.4.</span> <span class="caption-title">Contributions of ABC-RF summary statistics when estimating the divergence time between the two desert locust subspecies under the best supported scenario (scenario SCB).</span><p id="p-59" class="first-child">The contribution of each 32 summary statistics is evaluated as the total amount of decrease of the residual sum of squares, divided by the number of trees, (variable importance on the x-axis). The higher the contribution of the statistics, the more informative it is in the inferential process. The microsatellite set and subspecies sample are indicated at the end of each statistics by indices <em>k_i</em> for single population statistics and <em>k_i</em>.<em>j</em> for two population statistics, with <em>k</em>=1 for the set of untranscribed microsatellites or <em>k</em>=2 for the set of transcribed microsatellites, and <em>i</em>(<em>j</em>)=1 for the <em>S. g. flaviventris</em> subspecies or and <em>i</em>(<em>j</em>)=2 for the <em>S. g. gregaria</em> subspecies. See Table S6.1 for details on the summary statistics abbreviations. Five noise variables, randomly drawn into uniform distributions bounded between 0 and 1, and denoted NOISE1 to NOISE5 were added to the set of summary statistics processed by RF, in order to evaluate from which amount of decrease in the variable importance criterion the summary statistics computed from our genetic datasets were not informative anymore (indicated by a red star).</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of the number of RF-trees for scenario choice. We here illustrate the effect of the number of trees in the forest on the prior error rate when comparing the twelve scenarios separately. The number of datasets in the reference table simulated using DIYABC was 150,000. The shape of the curve shows that the prior error rate stabilizes for a number of RF-trees &gt; 2,000." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Effect of the number of RF-trees for scenario choice.&lt;/span&gt; We here illustrate the effect of the number of trees in the forest on the prior error rate when comparing the twelve scenarios separately. The number of datasets in the reference table simulated using DIYABC was 150,000. The shape of the curve shows that the prior error rate stabilizes for a number of RF-trees &gt; 2,000.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1.5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F9.medium.gif" width="440" height="411"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1.5." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F9.medium.gif" width="440" height="411"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1.5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1.5.</span> <span class="caption-title">Effect of the number of RF-trees for scenario choice.</span><p id="p-60" class="first-child">We here illustrate the effect of the number of trees in the forest on the prior error rate when comparing the twelve scenarios separately. The number of datasets in the reference table simulated using DIYABC was 150,000. The shape of the curve shows that the prior error rate stabilizes for a number of RF-trees &gt; 2,000.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of the number of RF-trees for parameter estimation. We here illustrate the effect of the number of trees in the forest on the out-of-bag mean square error of the divergence time between S. g. gregaria and S. g. flaviventris under the selected scenario SCB. The number of datasets in the reference table simulated using DIYABC was 100,000. The shape of the curve shows that the prior error rate stabilizes for a number of RF-trees &gt; 1,500." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Effect of the number of RF-trees for parameter estimation.&lt;/span&gt; We here illustrate the effect of the number of trees in the forest on the out-of-bag mean square error of the divergence time between &lt;em&gt;S. g. gregaria&lt;/em&gt; and &lt;em&gt;S. g. flaviventris&lt;/em&gt; under the selected scenario SCB. The number of datasets in the reference table simulated using DIYABC was 100,000. The shape of the curve shows that the prior error rate stabilizes for a number of RF-trees &gt; 1,500.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1.6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F10.medium.gif" width="419" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1.6." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F10.medium.gif" width="419" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1.6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1.6.</span> <span class="caption-title">Effect of the number of RF-trees for parameter estimation.</span><p id="p-61" class="first-child">We here illustrate the effect of the number of trees in the forest on the out-of-bag mean square error of the divergence time between <em>S. g. gregaria</em> and <em>S. g. flaviventris</em> under the selected scenario SCB. The number of datasets in the reference table simulated using DIYABC was 100,000. The shape of the curve shows that the prior error rate stabilizes for a number of RF-trees &gt; 1,500.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-20"><h3>Supplementary Material S2. Hindcasting the desert locust potential distribution into the mid-Holocene (HCO) and the last glacial maximum (LGM) using distribution modelling</h3><div class="section" id="sec-21"><h4>Motivation</h4><p id="p-62">To complement distribution projections based on vegetation during the mid-Holocene (HCO; ∼6,000 years ago) and Last Glacial Maximum (LGM; ∼120,000 years ago) periods, we used species distribution modelling along with climatic reconstructions of the past. However, results from this modelling effort do not fully match vegetation reconstructions presented in the main text. Given that most Global Climate Models (GCMs) have been calibrated and tested largely with northern hemisphere data, uncertainties related to current and past extrapolations of climate over large areas of Africa are often unknown (<em>e</em>.<em>g</em>., <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Rowell <em>et al</em>. 2016</a>). In this appendix, we present the results of the distribution modelling effort, while in the main text we base our formalization of evolutionary scenarios on paleo-vegetation reconstructions. Notice that the difference between the two approaches does not affect our major conclusions. Indeed, since paleo-vegetation maps show more severe changes of open vegetation distributions during glacial and interglacial periods, their use make us consider a wider spectrum of key biogeographic events than we would have identified based on climatic modelling (e.g. continuous colonization of southern Africa, population size contraction), and thus a larger set of possible scenarios. In addition, our inference of a most likely scenario including a long-distance migration event, and our estimation of a short time scale of divergence between the two clades, do not conflict with the results of any of these approaches (see also the section <em>On the influence of climatic cycles</em> in the Discussion of the main text).</p></div><div class="section" id="sec-22"><h4>Methods</h4><p id="p-63">In a previous study (<a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Meynard <em>et al</em>. 2017</a>), we used Climond data (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Kriticos <em>et al</em>. 2012</a>) to characterize the current distribution of the two desert locust subspecies, and project their potential fate under different climate change scenarios into 2070. Here, we could not use the same models because the climatic variables used to calibrate those models, in particular radiation and wetness indices, are not available into the mid-Holocene (HCO) and Last Glacial Maximum (LGM) periods. However, we apply here a similar modelling procedure with a different subset of environmental predictors. Instead of Climond, here we downloaded Worldclim v1.4 data (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Hijmans <em>et al</em>. 2005</a>) at a 5 min resolution (approximately 9 km at the equator).</p><p id="p-64">For hindcasting, we first need to calibrate a model under current conditions, which can then be used to draw maps of potential ranges under HCO and LGM conditions. To do so, we used the occurrence data published in <a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Meynard <em>et al</em>. (2017)</a>, which included decades of monitoring of <em>S. g. gregaria</em> in northern Africa, the middle east and southern Asia during remission periods, as well as field records and literature records of <em>S. g. flaviventris</em> in southern Africa. However, rather than modelling each clade independently as we did before, here we decided to group presence records for the two clades as a single unit for the following reasons: (1) here the emphasis is in paleo-climates, at a time scale where the two clades were not yet supposed to be differentiated; (2) a niche analysis in <a id="xref-ref-91-3" class="xref-bibr" href="#ref-91">Meynard <em>et al</em>. (2017)</a> did not show evidence of niche differentiation between <em>S. g. gregaria</em> and <em>S. g. flaviventris</em>, suggesting that the environment occupied by the southern clade is a subset of that occupied by the northern one; and (3) the biogeographic hypothesis supported by the molecular analysis presented here involves the colonization of the southern tip of south Africa by an isolated population of the northern sub-species, which is more generalist in terms of its environmental niche. We therefore considered that grouping occurrence records regardless of the subspecies would provide for a more accurate hindcasting scenario. However, preliminary analysis using separate modelling for each subspecies showed qualitatively similar projections when using <em>S. g. gregaria</em> alone (results not shown).</p><p id="p-65">Finally, since here the emphasis is on Africa, we included only occurrence records in this continent, and used 10,000 pseudo-absences drawn at random from outside the combined current ranges of the two clades by using the distribution maps proposed in <a id="xref-ref-91-4" class="xref-bibr" href="#ref-91">Meynard <em>et al</em>. (2017)</a> to mask out areas available for pseudo-absence selection. We also assumed here that most of the African continent was accessible to the species during this geological time-scale, given the long known dispersal distances during outbreak periods, the wide distribution of the northern clade, and the fact that the species is present in both extremes of the continent. Consequently, here we do not delimit an accessible area, but we limit our modelling to continental Africa.</p><p id="p-66">To model the species distribution, we used four climatic variables to represent mean and variability in temperature and precipitation, while minimizing correlation between them. The four variables selected using these criteria were annual mean temperature (BIO1), annual precipitation (BIO12), annual temperature range (BIO7) and precipitation seasonality (BIO15). We also tested the same forward variable selection procedure presented in <a id="xref-ref-91-5" class="xref-bibr" href="#ref-91">Meynard <em>et al</em>. (2017)</a>, which provided very similar predictions but, for simplicity, we will not present those results here. Using these four climatic predictors, we applied three statistical models in Rv3.6.1 to draw a consensus map for all predictions: Generalized Additive Model (GAM) using library <em>mgcv</em>, and limiting the maximum model complexity to k=4 (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Wood 2006</a>); boosted regression trees (BRT) using the library <em>gbm</em>, with 2000 trees and no interactions (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Elith <em>et al</em>. 2008</a>); and MAXENT, using the library <em>dismo</em> and all default parameters <em>(Elith et al</em>. 2011). As mentioned above, background data was sampled at random in Africa from outside the species current distribution. Pseudo-absences received a combined weight of 50% when using GAM or BRT, in order to balance the large number of pseudo-absences with respect to presence records in the modelling process while representing the range of values of the environmental predictors in the study area (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Barbet-Massin <em>et al</em>. 2012</a>). The consensus was drawn as the median predicted probability between the three models, and four different threshold values were calculated from this consensus: the MST threshold (the threshold that maximizes the sum of sensitivity + specificity), often recommended as a good strategy to optimize presence and absence classification rates (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Liu et al. 2005</a>), was calculated to delimit the predicted range of the species and calculate classification rate statistics; and 3 different thresholds that represent the highest predicted values that include a fixed percentage of presences (50%, 75% and 90%). These thresholds were used to visualize on the maps the core (50% threshold) versus the more marginal (between 75% and 90% thresholds) habitats within the range. In this context, the 50% threshold represents the core distribution, i.e. the highest predicted values that contain more than half of the occurrences, the 90% threshold represents the area that includes most of the occurrences, and the area outside the 90% threshold represents marginal or no habitat, where the species is unlikely to occur. The area between the 75% and 90% threshold can be considered as habitat that is rather marginal but that is still part of the predicted range.</p><p id="p-67">Classification success rates (AUC, TSS, sensitivity and specificity) were calculated for the consensus on all the dataset, using an MST threshold when needed. The thresholds calculated using current climate and occurrence data were then projected into past scenarios (HCO and LGM) using 3 different global circulation models at the same resolution: CCSM4, MIROC-ESM and MPI-ESM-P. We chose these three GCMs because their projections were available for both time steps of interest. Although the three GCMs are based on climatological principles, each one works in slightly different ways, and therefore produces different results. Consensus between GCMs presented below represents median values of predictions between the three GCMs.</p></div><div class="section" id="sec-23"><h4>Results</h4><p id="p-68">The current distribution of <em>S. gregaria</em>, as modelled using both subclades as a single unit and limited to continental Africa, is quite similar to the one already published in <a id="xref-ref-91-6" class="xref-bibr" href="#ref-91">Meynard et al. (2017)</a> (see <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure S2.1</a>). There are, however, differences in the outline of the distributions, but the overall predicted area is largely overlapping and classification rates were excellent for this model under current conditions (% correctly classified = 87%; AUC = 0.960 ± 0.003; sensitivity = 0.911 ± 0.014; specificity = 0.870 ± 0.002; TSS = 0.788 ± 0.016). As expected, the species is more likely to occur in regions with mid to high temperatures, wide annual temperature range, low precipitation and high precipitation seasonality (<a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure S2.2</a>).</p><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Current range as predicted by a model combining both subspecies&#x2019; occurrences and using four climatic predictors. The range predicted by this model (black contour) is compared to the one published in Meynard et al. (2017) using a different set of predictors and separating occurrences by subspecies (blue contour)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Current range as predicted by a model combining both subspecies&#x2019; occurrences and using four climatic predictors. The range predicted by this model (black contour) is compared to the one published in Meynard et al. (2017) using a different set of predictors and separating occurrences by subspecies (blue contour).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2.1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F11.medium.gif" width="440" height="383"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2.1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F11.medium.gif" width="440" height="383"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2.1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.1:</span> <p id="p-69" class="first-child">Current range as predicted by a model combining both subspecies’ occurrences and using four climatic predictors. The range predicted by this model (black contour) is compared to the one published in <a id="xref-ref-91-7" class="xref-bibr" href="#ref-91">Meynard et al. (2017)</a> using a different set of predictors and separating occurrences by subspecies (blue contour).</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Response curves of S. gregaria against the four environmental predictors used in the modelling process, as derived from the GAM model. Notice that temperature is expressed as the real temperature x 10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Response curves of S. gregaria against the four environmental predictors used in the modelling process, as derived from the GAM model. Notice that temperature is expressed as the real temperature x 10.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2.2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F12.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2.2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F12.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2.2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.2:</span> <p id="p-70" class="first-child">Response curves of S. gregaria against the four environmental predictors used in the modelling process, as derived from the GAM model. Notice that temperature is expressed as the real temperature x 10.</p><div class="sb-div caption-clear"></div></div></div><p id="p-71">To better understand the results in terms of distributional changes, <a id="xref-fig-13-2" class="xref-fig" href="#F13">Figure S2.3</a> shows a comparison of the climatic variables used in the modelling process in the study region during the present, the HCO and the LGM periods across Africa. Although only one GCM scenario is shown (CCSM4), climatic conditions for the other two GCMs present similar trends. Notice that, although temperatures are thought to have been warmer than present during the HCO globally, those changes are not uniform across regions and are not reflected in these HCO scenarios for Africa. In all the scenarios in <a id="xref-fig-13-3" class="xref-fig" href="#F13">Figure S2.3</a>, mean temperature showed a slight decrease across Africa, while precipitation increased slightly.</p><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of climates across times (current climate, mid-Holocene, Last Glacial Maximum=LGM). Variables shown are those used in the modelling process. BIO1 = Mean Annual Temperature; BIO7 = Annual Temperature Range; BIO12 = Annual Precipitation; BIO15 = Precipitation Seasonality (Coefficient of Variation). Vertical color lines indicate mean values for each period: blue = current mean; red = HCO mean; green = LGM mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of climates across times (current climate, mid-Holocene, Last Glacial Maximum=LGM). Variables shown are those used in the modelling process. BIO1 = Mean Annual Temperature; BIO7 = Annual Temperature Range; BIO12 = Annual Precipitation; BIO15 = Precipitation Seasonality (Coefficient of Variation). Vertical color lines indicate mean values for each period: blue = current mean; red = HCO mean; green = LGM mean.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2.3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F13.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2.3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F13.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2.3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.3:</span> <p id="p-72" class="first-child">Comparison of climates across times (current climate, mid-Holocene, Last Glacial Maximum=LGM). Variables shown are those used in the modelling process. BIO1 = Mean Annual Temperature; BIO7 = Annual Temperature Range; BIO12 = Annual Precipitation; BIO15 = Precipitation Seasonality (Coefficient of Variation). Vertical color lines indicate mean values for each period: blue = current mean; red = HCO mean; green = LGM mean.</p><div class="sb-div caption-clear"></div></div></div><p id="p-73">However, when looking at the distribution of those changes (<a id="xref-fig-14-2" class="xref-fig" href="#F14">Figure S2.4</a>) some regions showed a decrease and others an increase in temperatures during the same period. Most of Africa shows changes that represent less than 10% of their current mean temperature values (<a id="xref-fig-14-3" class="xref-fig" href="#F14">Figure S2.4</a>, upper left). Overall, conditions under HCO in Africa were therefore similar to current climatic conditions in terms of temperature but were moister in many regions that are relevant to the potential distribution of the desert locust (<a id="xref-fig-14-4" class="xref-fig" href="#F14">Figure S2.4</a>, upper row).</p><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative changes in annual mean temperature (BIO1) and annual precipitation (BIO12) during the HCO and LGM, expressed as a percent of current values per grid cell. Positive values therefore indicate an increase whereas negative values represent a decrease with respect to current climatic conditions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relative changes in annual mean temperature (BIO1) and annual precipitation (BIO12) during the HCO and LGM, expressed as a percent of current values per grid cell. Positive values therefore indicate an increase whereas negative values represent a decrease with respect to current climatic conditions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2.4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F14.medium.gif" width="440" height="311"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2.4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F14.medium.gif" width="440" height="311"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2.4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.4:</span> <p id="p-74" class="first-child">Relative changes in annual mean temperature (BIO1) and annual precipitation (BIO12) during the HCO and LGM, expressed as a percent of current values per grid cell. Positive values therefore indicate an increase whereas negative values represent a decrease with respect to current climatic conditions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-75">During the LGM (<a id="xref-fig-13-4" class="xref-fig" href="#F13">Figure S2.3</a>, right column), mean temperature decreased more dramatically throughout all of Africa, while precipitation increased in some regions but decreased in others (<a id="xref-fig-14-5" class="xref-fig" href="#F14">Figure S2.4</a>, lower row). While the entire continent experienced a decrease in temperature, changes in precipitation were more heterogeneous (<a id="xref-fig-14-6" class="xref-fig" href="#F14">Figure S2.4</a>, lower right). Given that these changes are heterogeneous across regions, changes in the potential distribution of <em>S. gregaria</em> are less dramatic than one would expect given global simplifications of these changes.</p><p id="p-76">Indeed, the predicted distribution during the HCO and the LGM are very similar to the current distribution (<a id="xref-fig-15-2" class="xref-fig" href="#F15">Figure S2.5</a>). There is an overall shrinkage of the core habitat (in red in all maps), but the contour of the distribution (black contour) remains large and mostly overlapping with the current distribution.</p><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of current versus past potential distribution of S. gregaria during the HCO and LGM periods. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Comparison of current versus past potential distribution of S. gregaria during the HCO and LGM periods.&lt;/span&gt; &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2.5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F15.medium.gif" width="440" height="162"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2.5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F15.medium.gif" width="440" height="162"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2.5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.5:</span> <span class="caption-title">Comparison of current versus past potential distribution of S. gregaria during the HCO and LGM periods.</span><div class="sb-div caption-clear"></div></div></div><p id="p-77">Although each GCM resulted in differences in terms of the projected distribution maps of <em>S. gregaria</em>, in the consensus distribution is projected to be very similar to the current distribution during the HCO, and slightly reduced during the LGM, with a shift of the core favorable climatic conditions towards a much reduced area, especially during the LGM (<a id="xref-fig-15-3" class="xref-fig" href="#F15">Figure S2.5</a>).</p><p id="p-78">Although climatic projections for each GCM result in different projected distributions during the HCO (<a id="xref-fig-16-2" class="xref-fig" href="#F16">Figure S2.6</a>) and the LGM (<a id="xref-fig-17-2" class="xref-fig" href="#F17">Figure S2.7</a>), overall the tendencies are the same: a similar distribution when comparing present and HCO conditions, and a more restricted potential distribution when considering LGM conditions.</p><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Projected potential distribution of S. gregaria during the mid-Holocene (&#x223C;6000 years BP) using three different Global Circulation Models (GCMs): CCRM4, MIROC-ESM and MPI-ESM-P." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Projected potential distribution of S. gregaria during the mid-Holocene (&#x223C;6000 years BP) using three different Global Circulation Models (GCMs): CCRM4, MIROC-ESM and MPI-ESM-P.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2.6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F16.medium.gif" width="440" height="148"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2.6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F16.medium.gif" width="440" height="148"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2.6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.6:</span> <p id="p-79" class="first-child">Projected potential distribution of S. gregaria during the mid-Holocene (∼6000 years BP) using three different Global Circulation Models (GCMs): CCRM4, MIROC-ESM and MPI-ESM-P.</p><div class="sb-div caption-clear"></div></div></div><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Projected potential distribution of S. gregaria during the LGM (&#x223C;20,000 years BP) using three different Global Circulation Models (GCMs): CCRM4, MIROC-ESM and MPI-ESM-P." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Projected potential distribution of S. gregaria during the LGM (&#x223C;20,000 years BP) using three different Global Circulation Models (GCMs): CCRM4, MIROC-ESM and MPI-ESM-P.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2.7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F17.medium.gif" width="440" height="148"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2.7:" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F17.medium.gif" width="440" height="148"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2.7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.7:</span> <p id="p-80" class="first-child">Projected potential distribution of S. gregaria during the LGM (∼20,000 years BP) using three different Global Circulation Models (GCMs): CCRM4, MIROC-ESM and MPI-ESM-P.</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section" id="sec-24"><h2 class="">Supplementary Material S3: Overview of the used ABC Random Forest (ABC-RF) methods</h2><p id="p-81">In this supplementary material, we provide readers with an overview of the Approximate Bayesian Computation Random Forest (hereafter ABC-RF) methods used in the present paper. We invite the reader to consult <a id="xref-ref-57-6" class="xref-bibr" href="#ref-57">Pudlo et al. (2016)</a>, <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Estoup et al. (2018)</a>, and <a id="xref-ref-60-8" class="xref-bibr" href="#ref-60">Raynal et al. (2019)</a> for more in-depth explanations.</p><div class="section" id="sec-25"><h3>ABC framework</h3><p id="p-82">Let <strong>y</strong> denote the observed data and <strong><em>θ</em></strong> a vector of parameters associated to a statistical model whose likelihood is <em>f</em> (. | <strong><em>θ</em></strong>). Under the Bayesian parametric paradigm the posterior distribution
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-26.gif"/></noscript></span>

</span>
is of prime interest. It characterizes the distribution of <strong><em>θ</em></strong> given the observation <strong>y</strong> and can be interpreted as an update of the prior distribution <em>π</em>(<strong><em>θ</em></strong>) by the likelihood of <strong>y</strong>. The likelihood is hence pivotal, but unfortunately intractable in the evolutionary scenarios (models) we consider in the present study, as well as in many other evolutionary studies. As a matter of fact, the underlying Kingman’s coalescent process (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Kingman, 1982</a>) does not allow a close expression for the likelihood because all the possible genealogies and mutational process yielding <strong>y</strong> should be considered. To solve this issue, some likelihood-free methods have been developed using the fact that, even though the likelihood is not available, generating artificial (i.e. simulated) data for a given value of <strong><em>θ</em></strong> is much easier if not feasible (e.g. <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Beaumont (2010)</a>. Approximate Bayesian computation (ABC) is one of them (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Beaumont et al., 2002</a>).</p><p id="p-83">In a nutshell, ABC consists in generating parameters <strong><em>θ</em></strong>′ and associated pseudo-data <strong>z</strong> from the scenario, and accepting <strong><em>θ</em></strong>′ as a realization from an <em>approximated</em> posterior if <strong>z</strong> is similar to <strong>y</strong>. In standard ABC treatments, the notion of similarity is defined through the use of a distance <em>ρ</em> to compare <em>η</em>(<strong>z</strong>) and <em>η</em>(<strong>y</strong>), where <em>η</em>(·) is a projection of the data in a lower dimensional space of summary statistics. Only pseudo-data providing distance lower than a threshold <em>ϵ</em> are retained. The choice of <em>ρ, η</em>(·) and <em>ϵ</em> is a major issue in ABC (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Beaumont, 2010</a>).</p><p id="p-84">ABC-RF is a recently derived ABC approach based on the supervised machine learning tool named Random Forest (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Breiman, 2001</a>), which has as major advantage to avoid the three above-mentioned difficulties. Initially introduced in <a id="xref-ref-57-7" class="xref-bibr" href="#ref-57">Pudlo et al. (2016)</a> for model choice and then extended to parameter inference in <a id="xref-ref-60-9" class="xref-bibr" href="#ref-60">Raynal et al. (2019)</a>, ABC-RF relies on the use of random forests on a set of simulated pseudo-data according to the generative Bayesian models under consideration. Let consider <em>M</em> Bayesian parametric models. For a given model index <em>m</em> ∈ {1, …, <em>M</em>}, a prior probability ℙ(ℳ = <em>m</em>) is defined, with <strong><em>θ</em></strong><sub><em>m</em></sub> its associated parameters and <em>f</em><sub><em>m</em></sub>(<strong>y</strong> | <strong><em>θ</em></strong><sub><em>m</em></sub>) its likelihood.</p><p id="p-85">The generation process of a reference table made of <em>H</em> elements is described in <a id="xref-fig-18-1" class="xref-statement" href="#F18">Algorithm 1</a>.</p><div id="statement-1" class="statement"><span class="statement-label">Algorithm 1:</span><p id="p-86">Generation of a reference table with <em>H</em> elements
</p><div id="F18" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F18.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure18" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F18.medium.gif" width="440" height="94"/><noscript><img class="highwire-fragment fragment-image" alt="Figure18" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F18.medium.gif" width="440" height="94"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure18" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><p id="p-87">The output takes the form of a matrix containing simulated model indexes, parameters and summary statistics, as described below
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-28.gif"/></noscript></span>

</span>
</p></div><div class="section" id="sec-26"><h3>ABC-RF for model choice</h3><p id="p-88">The ABC-RF strategy for model choice is described in <a id="xref-fig-19-1" class="xref-statement" href="#F19">Algorithm 2</a>. The output is the affectation of <strong>y</strong> to a model (scenario), this decision being made based on the majority class of the RF tree votes.</p><div id="statement-2" class="statement"><span class="statement-label">Algorithm 2:</span><p id="p-89">ABC-RF for model choice
</p><div id="F19" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F19.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure19" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F19.medium.gif" width="440" height="93"/><noscript><img class="highwire-fragment fragment-image" alt="Figure19" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F19.medium.gif" width="440" height="93"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure19" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><p id="p-90">The selected scenario is the one with the highest number of votes in his favor. In addition to this majority vote, the posterior probability of the selected scenario can be computed as described in <a id="xref-fig-20-1" class="xref-statement" href="#F20">Algorithm 3</a>.</p><p id="p-91">Such posterior probability provides a confidence measure of the previous prediction at the point of interest <em>η</em>(<strong>y</strong>). It relies on the building of a regression random forest designed to explain the model prediction error. More specifically, and as a first step, posterior probability computation makes use of out-of-bag predictions of the training dataset. Because each tree of the random forest is built on a bootstrap sampling of the <em>H</em> elements of the reference table (i.e. the training dataset), there is about one third of the reference table that remains unused per tree, and this ensemble of left aside datasets corresponds to the out-of-bag. Thus, for each pseudo-data of the reference table, one can obtain an out-of-bag prediction by aggregating all the classification trees in which the pseudo-data was out-of-bag. In a second step, the out-of-bag predictions <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-7.gif"/></noscript></span></span> are used to compute the indicators <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-8.gif"/></noscript></span></span>. These 0 - 1 values are used as response variables for the regression random forest training, for which the explanatory variables are the summary statistics of the reference table. Predicting the observed data thanks to this forest allows the derivation of the posterior probability of the selected model (<a id="xref-fig-20-2" class="xref-statement" href="#F20">Algorithm 3</a>). Note that using the out-of-bag procedure prevents over-fitting issues and is computationally parsimonious as it avoids the generation of a second reference table for the regression random forest training.</p><div id="statement-3" class="statement"><span class="statement-label">Algorithm 3:</span><p id="p-92">ABC-RF computation of the posterior probability of the selected scenario
</p><div id="F20" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F20.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure20" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F20.medium.gif" width="440" height="99"/><noscript><img class="highwire-fragment fragment-image" alt="Figure20" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F20.medium.gif" width="440" height="99"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure20" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><div class="section" id="sec-27"><h4>Model grouping</h4><p id="p-93">A recent useful add-on to ABC-RF has been the model-grouping approach developed in <a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Estoup et al. (2018)</a>, where pre-defined groups of scenarios are analysed using <a id="xref-fig-19-2" class="xref-statement" href="#F19">Algorithm 2</a> and <a id="xref-fig-20-3" class="xref-statement" href="#F20">3</a>. The model indexes used in the training reference table are modified in a preliminary step to match the corresponding groups, which are then used during learning phase. When appropriate, unused scenarios are discarded from the reference table. This improvement is particularly useful when a high number of individual scenarios are considered and have been formalized through the absence or presence of some key evolutionary events (e.g. admixture, bottleneck, …). Such key evolutionary events allow defining and further considering groups of scenarios including or not such events. This grouping approach allows to evaluate the power of ABC-RF to make inferences about evolutionary event(s) of interest over the entire prior space and assess (and quantify) whether or not a particular evolutionary event is of prime importance to explain the observed dataset (see <a id="xref-ref-96-3" class="xref-bibr" href="#ref-96">Estoup et al. (2018)</a> for details and illustrations).</p></div></div><div class="section" id="sec-28"><h3>ABC-RF for parameter estimation</h3><p id="p-94">Once the selected (i.e. best) scenario has been identified, the next step is the estimation of its parameters of interest under this scenario. The ABC-RF parameter estimation strategy is described in <a id="xref-fig-21-1" class="xref-statement" href="#F21">Algorithm 4</a> and takes a similar structure to <a id="xref-fig-19-3" class="xref-statement" href="#F19">Algorithm 2</a>. The idea is to use a regression random forest for each dimension of the parameter space (i.e. for each parameter). For a given parameter of interest, the output of the algorithm is a vector of weights <strong>w</strong><sub><strong>y</strong></sub> that can be used to compute posterior quantities of interest such as expectation, variance and quantiles. <strong>w</strong><sub><strong>y</strong></sub> provides an empirical posterior distribution for <em>θ</em><sub><em>m,k</em></sub>; see <a id="xref-ref-60-10" class="xref-bibr" href="#ref-60">Raynal et al. (2019)</a> for more details.</p></div><div class="section" id="sec-29"><h3>Global prior errors</h3><p id="p-95">In both contexts, model choice or parameter estimation, a global quality of the predictor can be computed, which does not take the observed dataset (about which one wants to make inferences) into account. Random forests make it possible the computation of errors on the training reference table, using the out-of-bag predictions.</p><div id="statement-4" class="statement"><span class="statement-label">Algorithm 4:</span><p id="p-96">ABC-RF for parameter estimation
</p><div id="F21" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F21.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure21" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F21.medium.gif" width="440" height="137"/><noscript><img class="highwire-fragment fragment-image" alt="Figure21" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F21.medium.gif" width="440" height="137"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure21" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div></div><p id="p-97">For model choice, this type of error is called the prior error rate, which is the mis-classification error rate computed over the entire multidimensional prior space. It can be computed as
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-32.gif"/></noscript></span>

</span>
</p><p id="p-98">For parameter estimation, the equivalent is the prior mean squared error (MSE) or the normalised mean absolute error (NMAE), the latter being less sensitive to extreme values. These errors are computed as
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-33.gif"/></noscript></span>

</span>
<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-9.gif"/></noscript></span></span> is the out-of-bag estimate of <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-10.gif"/></noscript></span></span>. They can be perceived as Monte Carlo approximation of expectations with respect to the prior distribution.</p></div><div class="section" id="sec-30"><h3>Local posterior errors</h3><p id="p-99">In the present paper, we propose some posterior versions of errors, which target the quality of prediction with respect to the posterior distribution. As such errors take the observed dataset <em>η</em>(<strong>y</strong>) into account, we mention them as local posterior errors.</p><p id="p-100">For model choice, the posterior probability provided by <a id="xref-fig-20-4" class="xref-statement" href="#F20">Algorithm 3</a> is a confidence measure of the selected scenario given the observation. Therefore
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-34.gif"/></noscript></span>

</span>
directly yields the posterior error associated to <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-11.gif"/></noscript></span></span>.</p><p id="p-101">For parameter estimation, when trying to infer on <em>θ</em><sub><em>m,k</em></sub>, a point-wise analogous measure of a local error can be computed as the posterior expectations
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-35.gif"/></noscript></span>

</span>
</p><p id="p-102">We approximate these expectations by
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-36.gif"/></noscript></span>

</span>
</p><p id="p-103">We again uses the out-of-bag information to compute <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-12.gif"/></noscript></span></span>, hence avoiding the (time consuming) production of a second reference table, and assume that the weights <strong>w</strong><sub><strong>y</strong></sub> from the regression random forest are good enough to approximate any posterior expectations of functions of <em>θ</em><sub><em>m,k</em></sub>: 𝔼(<em>g</em>(<em>θ</em><sub><em>m,k</em></sub>) | <em>η</em>(<strong>y</strong>)).</p><p id="p-104">Another more expensive strategy to evaluate the posterior expectations (1) is to construct new regression random forests using the out-of-bag vector of values
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/graphic-37.gif"/></noscript></span>

</span>
depending on the targeted error. The observation <em>η</em>(<strong>y</strong>) is then given to the forests, targeting the expectations (1).</p><p id="p-105">Note that the values <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-13.gif"/></noscript></span></span> in the previous formulas can be replaced by either the approximated posterior expectations <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-14.gif"/></noscript></span></span> or the posterior medians <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/16/671867/embed/inline-graphic-15.gif"/></noscript></span></span>, again using the out-of-bag information, to provide the local posterior errors. We found that both in the present paper (see main text, Materials and Methods section) and for various tests that we carried out on different inferential setups and datasets (results not shown), the posterior median provides a better accuracy of parameter estimation than the posterior expectation (aka posterior mean). This trends also holds for global prior errors that can be computed using either the mean or the median as point estimates.</p><p id="p-106">As final comment, it is worth noting that so far a common practice consisted in evaluating the quality of prediction (for model choice or parameter estimation) in the neighborhood of the observed dataset, that is around <em>η</em>(<strong>y</strong>) and not exactly for <em>η</em>(<strong>y</strong>). For model choice, <a id="xref-ref-96-4" class="xref-bibr" href="#ref-96">Estoup et al. (2018)</a> use the so called posterior predictive error rate which is an error of this type. In this case, some simulated datasets of the reference table close to the observation are selected thanks to an Euclidean distance, then new pseudo-observed datasets are simulated using similar parameters, on which is computed the error (see also <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Lippens et al., 2017</a>, for a similar approach in a standard ABC framework). However, the main problem of processing this way is the difficulty to specify the size of the area around the observation, especially when the number of summary statistics is large. We therefore do not recommend the use of such a “neighborhood” error anymore, but rather to compute the local posterior errors detailed above as the latter measured prediction quality exactly at the position of interest <em>η</em>(<strong>y</strong>).</p></div></div><div class="section" id="sec-31"><h2 class="">Supplementary Material S4. Details on results from ABC-RF treatments when using uniform priors for the three time period parameters of the studied scenarios</h2><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111610/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111610/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111610" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S4.1.</span> <p id="p-107" class="first-child">Scenario choice for each of the ten replicate analyses using uniform priors for the three time period parameters of the studied scenarios.</p><div class="sb-div caption-clear"></div></div></div><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1111596/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1111596/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1111596" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table S4.2.</span> <p id="p-108" class="first-child">Estimation of the divergence time between <em>S. g. gregaria</em> and <em>S. g. flaviventris</em> for ten replicate analyses using uniform prior distributions for the three time period parameters under the best supported scenario (scenario SCB).</p><div class="sb-div caption-clear"></div></div></div><div id="F22" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F22.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimation of the time since divergence between the two desert locust subspecies as a function of time scales using uniform prior distributions for the three time period parameters under the best supported scenario (scenario SCB). Simulated datasets (5,000 par divergence time) were generated for fixed divergence time values of 100; 250; 500; 1,000; 2,500; 5,000; 10,000; 25,000; 50,000; 100,000; and 250,000 generations. The median (plain lines) and 90% credibility interval (90% CI; dashed lines), averaged over the 5,000 datasets, are represented. Divergence time values are in number of generations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1645467595" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Estimation of the time since divergence between the two desert locust subspecies as a function of time scales using uniform prior distributions for the three time period parameters under the best supported scenario (scenario SCB).&lt;/span&gt; Simulated datasets (5,000 par divergence time) were generated for fixed divergence time values of 100; 250; 500; 1,000; 2,500; 5,000; 10,000; 25,000; 50,000; 100,000; and 250,000 generations. The median (plain lines) and 90% credibility interval (90% CI; dashed lines), averaged over the 5,000 datasets, are represented. Divergence time values are in number of generations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4.1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F22.medium.gif" width="440" height="386"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4.1." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F22.medium.gif" width="440" height="386"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4.1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/16/671867/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4.1.</span> <span class="caption-title">Estimation of the time since divergence between the two desert locust subspecies as a function of time scales using uniform prior distributions for the three time period parameters under the best supported scenario (scenario SCB).</span><p id="p-109" class="first-child">Simulated datasets (5,000 par divergence time) were generated for fixed divergence time values of 100; 250; 500; 1,000; 2,500; 5,000; 10,000; 25,000; 50,000; 100,000; and 250,000 generations. The median (plain lines) and 90% credibility interval (90% CI; dashed lines), averaged over the 5,000 datasets, are represented. Divergence time values are in number of generations.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-52">This work was supported by research funds from the French Agricultural Research Centre for International Development (CIRAD), the project ANR-16-CE02-0015-01 (SWING), the INRA scientific department SPE (AAP-SPE 2016), and the Labex NUMEV (NUMEV, ANR10-LABX-20). The data used in this work were partly produced through the technical facilities of the Centre Méditerranéen Environnement Biodiversité, Montpellier. We are grateful to Pierre-Emmanuel Gay for assistance with maps, and Antoine Foucart, Gauthier Dobigny and Jean-Yves Rasplus for fruitful discussions. We are extremely grateful to the <em>Peer Community In Evolutionary Biology</em> recommenders (Concetta Burgarella and Takeshi Kawakami) and reviewers (Michael D Greenfield and two anonymous ones) for their wise and helpful comments on two earlier versions of the manuscript. The Peer Community In Evolutionary Biology recommendation of the version 4 of this preprint (along with peer-reviews and authors’ replies) is available at <a href="https://doi.org/10.24072/pci.evolbiol.100091">https://doi.org/10.24072/pci.evolbiol.100091</a>.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-update fn-group-summary-of-updates" id="fn-1"><p id="p-3">The Version 4 of this preprint has been peer-reviewed (by three reviewers), recommended by Peer Community In Evolutionary Biology (https://doi.org/10.24072/pci.evolbiol.100091), and formatted following the PCI Evol Biol template. Typos in some mathematical formula of the Supplementary Material S3 have been also corrected.</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-source">(eds), QEN members. Review and Atlas of Palaeovegetation: Preliminary land ecosystem maps of the world since the Last Glacial Maximum</span>. <span class="cit-publ-name">Oak Ridge National Laboratory</span>, <span class="cit-publ-loc">TN, USA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.2" data-doi="10.1016/0277-3791(96)00028-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van Andel</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tzedakis</span>  <span class="cit-name-given-names">PC</span></span> (<span class="cit-pub-date">1996</span>). <span class="cit-article-title">Palaeolithic landscapes of Europe and environs: 150,000-25,000 years ago: an overview</span>. <abbr class="cit-jnl-abbrev">Quat Sci Rev</abbr>, <span class="cit-vol">15</span>, <span class="cit-fpage">481</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat%2BSci%2BRev%26rft.volume%253D15%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0277-3791%252896%252900028-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0277-3791(96)00028-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VL59200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Atickem</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stenseth</span>  <span class="cit-name-given-names">NC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drouilly</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bock</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zinner</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Deep divergence among mitochondrial lineages in African jackals</span>. <abbr class="cit-jnl-abbrev">Zool Scripta</abbr>, <span class="cit-vol">47</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZool%2BScripta%26rft.volume%253D47%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aumassip</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-source">L’algérie des premiers hommes</span>. <span class="cit-publ-name">Ibis Press</span>, <span class="cit-fpage">230</span> p.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Badisco</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pagès</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foucart</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Risterucci</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bazelet</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vanden Broeck</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ould Ely</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chapuis</span>  <span class="cit-name-given-names">M-P</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Characterization and comparison of microsatellite markers derived from genomic and expressed libraries for the desert locust</span>. <abbr class="cit-jnl-abbrev">J Appl Entomol</abbr>, <span class="cit-vol">137</span>, <span class="cit-fpage">673</span>–<span class="cit-lpage">683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAppl%2BEntomol%26rft.volume%253D137%26rft.spage%253D673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.6" data-doi="10.1126/science.278.5341.1257"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bond</span>  <span class="cit-name-given-names">G</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">A pervasive millenial-scale cycle in North Atlantic Holocene and Glacial climates</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">278</span>, <span class="cit-fpage">1257</span>–<span class="cit-lpage">1265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBond%26rft.auinit1%253DG.%26rft.volume%253D278%26rft.issue%253D5341%26rft.spage%253D1257%26rft.epage%253D1266%26rft.atitle%253DA%2BPervasive%2BMillennial-Scale%2BCycle%2Bin%2BNorth%2BAtlantic%2BHolocene%2Band%2BGlacial%2BClimates%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.278.5341.1257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzgvNTM0MS8xMjU3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNjcxODY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.7" data-doi="10.1086/681436"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bostoen</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clist</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doumenge</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grollemund</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hombert</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muluwa</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maley</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Middle to Late Holocene paleoclimatic change and early Bantu expansion in the rain forests of Western Central Africa</span>. <abbr class="cit-jnl-abbrev">Curr Anthropol</abbr>, <span class="cit-vol">56</span>, <span class="cit-fpage">354</span>–<span class="cit-lpage">384</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BAnthropol%26rft.volume%253D56%26rft.spage%253D354%26rft_id%253Dinfo%253Adoi%252F10.1086%252F681436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/681436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.8" data-doi="10.1017/CBO9781107415324.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Breiman</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Random Forests</span>. <abbr class="cit-jnl-abbrev">Machine Learning</abbr>, <span class="cit-vol">45</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMachine%2BLearning%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1017%252FCBO9781107415324.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/CBO9781107415324.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-671867v4.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">L</span></span> (<span class="cit-pub-date">1993</span>) <span class="cit-article-title">A unified approach to study hypervariable polymorphisms: statistical considerations of determining relatedness and population distances</span>. <abbr class="cit-jnl-abbrev">EXS</abbr>, <span class="cit-vol">67</span>, <span class="cit-fpage">153</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEXS%26rft.stitle%253DEXS%26rft.aulast%253DChakraborty%26rft.auinit1%253DR.%26rft.volume%253D67%26rft.spage%253D153%26rft.epage%253D175%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bstudy%2Bhypervariable%2Bpolymorphisms%253A%2Bstatistical%2Bconsiderations%2Bof%2Bdetermining%2Brelatedness%2Band%2Bpopulation%2Bdistances.%26rft_id%253Dinfo%253Apmid%252F8400687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8400687&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993BY75H00014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chapuis</span>  <span class="cit-name-given-names">M-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bazelet</span>  <span class="cit-name-given-names">CS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foucart</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vitalis</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Samways</span>  <span class="cit-name-given-names">MJ</span></span>. (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Subspecific taxonomy of the desert locust, <em>Schistocerca gregaria</em> (Orthoptera: Acrididae), based on molecular and morphological characters</span>. <abbr class="cit-jnl-abbrev">Syst Entomol</abbr>, <span class="cit-vol">41</span>, <span class="cit-fpage">516</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst%2BEntomol%26rft.volume%253D41%26rft.spage%253D516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chapuis</span>  <span class="cit-name-given-names">M-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foucart</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plantamp</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leménager</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benoit</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">P-E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bazelet</span>  <span class="cit-name-given-names">CS</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Genetic and morphological variation in non-polyphenic southern African populations of the desert locust</span>. <abbr class="cit-jnl-abbrev">Afr Entomol</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">13</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAfr%2BEntomol%26rft.volume%253D25%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chapuis</span>  <span class="cit-name-given-names">M.-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plantamp</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pagès</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vassal</span>  <span class="cit-name-given-names">J.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lecoq</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Demographic processes shaping genetic variation of the solitarious phase of the desert locust</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>, <span class="cit-vol">23</span>, <span class="cit-fpage">1749</span>–<span class="cit-lpage">1763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D23%26rft.spage%253D1749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chapuis</span>  <span class="cit-name-given-names">M-P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plantamp</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Streiff</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blondin</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piou</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Microsatellite evolutionary rate and pattern in <em>Schistocerca gregaria</em> inferred from direct observation of germline mutations</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">6107</span>–<span class="cit-lpage">6119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D24%26rft.spage%253D6107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chase</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meadows</span>  <span class="cit-name-given-names">ME</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Late Quaternary dynamics of southern Africa’s winter rainfall zone</span>. <abbr class="cit-jnl-abbrev">Earth Sci Rev</abbr>, <span class="cit-vol">84</span>, <span class="cit-fpage">103</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEarth%2BSci%2BRev%26rft.volume%253D84%26rft.spage%253D103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cockcroft</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tyson</span>  <span class="cit-name-given-names">PD</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">The application of a present-day climatic model to the Late Quaternary in southern Africa</span>. <abbr class="cit-jnl-abbrev">Climate Change</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">161</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClimate%2BChange%26rft.volume%253D10%26rft.spage%253D161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.16" data-doi="10.1186/1471-2105-11-401"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cornuet</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ravigne</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Inference on population history and model checking using DNA sequence and microsatellite data with the software DIYABC (v1.0)</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCornuet%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D11%26rft.spage%253D401%26rft.epage%253D401%26rft.atitle%253DInference%2Bon%2Bpopulation%2Bhistory%2Band%2Bmodel%2Bchecking%2Busing%2BDNA%2Bsequence%2Band%2Bmicrosatellite%2Bdata%2Bwith%2Bthe%2Bsoftware%2BDIYABC%2B%2528v1.0%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-401%26rft_id%253Dinfo%253Apmid%252F20667077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20667077&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.17" data-doi="10.1093/bioinformatics/btt763"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cornuet</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pudlo</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Veyssier</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dehne-Garcia</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leblois</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marin</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">DIYABC v2.0: a software to make Approximate Bayesian Computation inferences about population history using Single Nucleotide Polymorphism, DNA sequence and microsatellite data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">30</span>, <span class="cit-fpage">1187</span>–<span class="cit-lpage">1189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt763%26rft_id%253Dinfo%253Apmid%252F24389659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24389659&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335001000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.18" data-doi="10.1126/science.270.5233.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">deMenocal</span>  <span class="cit-name-given-names">PB</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">Plio-Pleistocene African climate</span>, <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">270</span>, <span class="cit-fpage">53</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DdeMenocal%26rft.auinit1%253DP.%2BB.%26rft.volume%253D270%26rft.issue%253D5233%26rft.spage%253D53%26rft.epage%253D59%26rft.atitle%253DPlio-Pleistocene%2BAfrican%2BClimate%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.270.5233.53%26rft_id%253Dinfo%253Apmid%252F7569951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyNzAvNTIzMy81MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2LzY3MTg2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.19" data-doi="10.1016/S0277-3791(99)00081-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">deMenocal</span>  <span class="cit-name-given-names">PB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guilderson</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adkins</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarnthein</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yarusinsky</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Abrupt onset and termination of the African Humid Period: Rapid climate response to gradual insolation forcing</span>. <abbr class="cit-jnl-abbrev">Quat Sci Rev</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">347</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat%2BSci%2BRev%26rft.volume%253D19%26rft.spage%253D347%26rft.atitle%253DQUAT%2BSCI%2BREV%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0277-3791%252899%252900081-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0277-3791(99)00081-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.20" data-doi="10.1016/S0012-821X(04)00003-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">deMenocal</span>  <span class="cit-name-given-names">PB</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">African climate change and faunal evolution during the Pliocene-Pleistocene</span>. <abbr class="cit-jnl-abbrev">Earth and Planetary Science Letters</abbr>, <span class="cit-vol">220</span>, <span class="cit-fpage">3</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEarth%2Band%2BPlanetary%2BScience%2BLetters%26rft.stitle%253DEarth%2Band%2BPlanetary%2BScience%2BLetters%26rft.volume%253D220%26rft.issue%253D1-2%26rft.spage%253D3%26rft.epage%253D24%26rft.atitle%253DAfrican%2Bclimate%2Bchange%2Band%2Bfaunal%2Bevolution%2Bduring%2Bthe%2BPliocene-Pleistocene%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0012-821X%252804%252900003-2%26rft_id%253Dinfo%253Apmid%252F18711123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0012-821X(04)00003-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2005071197&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=18711123&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220556000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duranton</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foucart</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">P-E</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-source">Florule des biotopes du criquet pèlerin en Afrique de l’Ouest et du Nord-Ouest à l’usage des prospecteurs de la lutte antiacridienne</span>. <span class="cit-publ-loc">Rome</span> : <span class="cit-publ-name">FAO</span>, <span class="cit-fpage">487</span> p.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Edmunds</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fellman</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goni</span>  <span class="cit-name-given-names">IB</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Lakes, groundwater and palaeohydrology in the Sahel of NE Nigeria: evidence from hydrogeochemistry</span>. <abbr class="cit-jnl-abbrev">J Geol Soc Lond</abbr>, <span class="cit-vol">156</span>, <span class="cit-fpage">345</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BGeol%2BSoc%2BLond%26rft.volume%253D156%26rft.spage%253D345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.23" data-doi="10.1016/S0168-9525(00)02139-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellegren</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Microsatellite mutations in the germline: implications for evolutionary inference</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">551</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DEllegren%26rft.auinit1%253DH.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D551%26rft.epage%253D558%26rft.atitle%253DMicrosatellite%2Bmutations%2Bin%2Bthe%2Bgermline%253A%2Bimplications%2Bfor%2Bevolutionary%2Binference.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252800%252902139-9%26rft_id%253Dinfo%253Apmid%252F11102705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(00)02139-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11102705&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165861000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.24" data-doi="10.1046/j.1365-294X.2002.01576.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarne</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornuet</span>  <span class="cit-name-given-names">J-M</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Homoplasy and mutation model at microsatellite loci and their consequences for population genetics analysis</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">1591</span>–<span class="cit-lpage">1604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEstoup%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D1591%26rft.epage%253D1604%26rft.atitle%253DHomoplasy%2Band%2Bmutation%2Bmodel%2Bat%2Bmicrosatellite%2Bloci%2Band%2Btheir%2Bconsequences%2Bfor%2Bpopulation%2Bgenetics%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2002.01576.x%26rft_id%253Dinfo%253Apmid%252F12207711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2002.01576.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12207711&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177775600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raynal</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verdu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marin</span>  <span class="cit-name-given-names">J-M</span></span> (<span class="cit-pub-date">2018a</span>) <span class="cit-article-title">Model choice using Approximate Bayesian Computation and Random Forests: analyzes based on model grouping to make inferences about the genetic history of Pygmy human populations</span>. <abbr class="cit-jnl-abbrev">J Soc Fr Statistique</abbr>, <span class="cit-vol">159</span>, <span class="cit-fpage">167</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BSoc%2BFr%2BStatistique%26rft.volume%253D159%26rft.spage%253D167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verdu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marin</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dehne-Garcia</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornuet</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pudlo</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2018b</span>) <span class="cit-chapter-title">Application of approximate Bayesian computation to infer the genetic history of Pygmy hunter-gatherers populations from Western Central Africa</span>. <span class="cit-source">Handbook of Approximate Bayesian Computation</span>. <span class="cit-publ-name">Chapman and Hall/CRC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">IJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornuet</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moritz</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Inferring population history from microsatellite et enzyme data in serially introduced cane toads, <em>Bufo marinus</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">159</span>, <span class="cit-fpage">1671</span>–<span class="cit-lpage">1687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DEstoup%26rft.auinit1%253DA.%26rft.volume%253D159%26rft.issue%253D4%26rft.spage%253D1671%26rft.epage%253D1687%26rft.atitle%253DInferring%2BPopulation%2BHistory%2BFrom%2BMicrosatellite%2Band%2BEnzyme%2BData%2Bin%2BSerially%2BIntroduced%2BCane%2BToads%252C%2BBufo%2Bmarinus%26rft_id%253Dinfo%253Apmid%252F11779806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS80LzE2NzEiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi82NzE4NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pollock</span>  <span class="cit-name-given-names">DD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Microsatellite genetic distances with range constraints: analytic description and problems of estimation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">145</span>, <span class="cit-fpage">207</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFeldman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D145%26rft.issue%253D1%26rft.spage%253D207%26rft.epage%253D216%26rft.atitle%253DMicrosatellite%2BGenetic%2BDistances%2BWith%2BRange%2BConstraints%253A%2BAnalytic%2BDescription%2Band%2BProblems%2Bof%2BEstimation%26rft_id%253Dinfo%253Apmid%252F9017402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ1LzEvMjA3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNjcxODY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fraimout</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Debat</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fellous</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hufbauer</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foucaud</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pudlo</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marin</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">DK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cattel</span>  <span class="cit-name-given-names">J</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Deciphering the routes of invasion of <em>Drosophila suzukii</em> by means of ABC random forest</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>, <span class="cit-vol">34</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">980</span>–<span class="cit-lpage">996</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft.volume%253D34%26rft.spage%253D980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.30" data-doi="10.1038/34839"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ganopolski</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rahmstorf</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petoukohov</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claussen</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Simulation of modern and glacial climates with a coupled global model of intermediate complexity</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">391</span>, <span class="cit-fpage">351</span>–<span class="cit-lpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D391%26rft.issue%253D6665%26rft.spage%253D351%26rft.epage%253D356%26rft.atitle%253DSimulation%2Bof%2Bmodern%2Band%2Bglacial%2Bclimates%2Bwith%2Ba%2Bcoupled%2Bglobal%2Bmodel%2Bof%2Bintermediate%2Bcomplexity%26rft_id%253Dinfo%253Adoi%252F10.1038%252F34839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/34839&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1998013507&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-671867v4.31" data-doi="10.1046/j.1365-294x.2001.01190.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garza</span>  <span class="cit-name-given-names">JC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">E</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Detection of reduction in population size using data from microsatellite DNA</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">305</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGarza%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D305%26rft.epage%253D318%26rft.atitle%253DDetection%2Bof%2Breduction%2Bin%2Bpopulation%2Bsize%2Busing%2Bdata%2Bfrom%2Bmicrosatellite%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294x.2001.01190.x%26rft_id%253Dinfo%253Apmid%252F11298947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294x.2001.01190.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11298947&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167779400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.32" data-doi="10.1016/S0277-3791(99)00061-X"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gasse</span>  <span class="cit-name-given-names">F</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Hydrological changes in the African tropics since the Last Glacial Maximum</span>. <abbr class="cit-jnl-abbrev">Quat Sci Rev</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">189</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat%2BSci%2BRev%26rft.volume%253D19%26rft.spage%253D189%26rft.atitle%253DQUAT%2BSCI%2BREV%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0277-3791%252899%252900061-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0277-3791(99)00061-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084425500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-671867v4.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linares</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">MW</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">An evaluation of genetic distances for use with microsatellite loci</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">139</span>, <span class="cit-fpage">463</span>–<span class="cit-lpage">471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGoldstein%26rft.auinit1%253DD.%2BB.%26rft.volume%253D139%26rft.issue%253D1%26rft.spage%253D463%26rft.epage%253D471%26rft.atitle%253DAn%2Bevaluation%2Bof%2Bgenetic%2Bdistances%2Bfor%2Buse%2Bwith%2Bmicrosatellite%2Bloci.%26rft_id%253Dinfo%253Apmid%252F7705647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM5LzEvNDYzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNjcxODY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petit-Maire</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kroepelin</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Holocene non-orbital climatic events in present-day arid areas of northern Africa and China</span>. <abbr class="cit-jnl-abbrev">Global Planet Change</abbr> v.<span class="cit-vol">26</span> p.<span class="cit-fpage">97</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BPlanet%2BChange%26rft.volume%253D26%26rft.spage%253D97%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.35" data-doi="10.1371/journal.pone.0001615"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">SY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saarma</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barnett</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haile</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">The Effect of Inappropriate Calibration: Three Case Studies in Molecular Ecology</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">e1615</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D3%26rft.spage%253De1615%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001615%26rft_id%253Dinfo%253Apmid%252F18286172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18286172&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-671867v4.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jürgens</span>  <span class="cit-name-given-names">N</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Floristic biodiversity and history of African arid regions</span>. <abbr class="cit-jnl-abbrev">Biodiv Conserv</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">495</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiodiv%2BConserv%26rft.volume%253D6%26rft.spage%253D495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.37" data-doi="10.1126/science.1154913"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kröpelin</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verschuren</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lézine</span>  <span class="cit-name-given-names">A-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eggermont</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cocquyt</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Francus</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cazet</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fagot</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rumes</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russel</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Climate-driven ecosystem succession in the Sahara: The past 6000 years</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">320</span>, <span class="cit-fpage">765</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKro%2526%2523x0308%253Bpelin%26rft.auinit1%253DS.%26rft.volume%253D320%26rft.issue%253D5877%26rft.spage%253D765%26rft.epage%253D768%26rft.atitle%253DClimate-Driven%2BEcosystem%2BSuccession%2Bin%2Bthe%2BSahara%253A%2BThe%2BPast%2B6000%2BYears%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1154913%26rft_id%253Dinfo%253Apmid%252F18467583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3Ny83NjUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi82NzE4NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lebrun</span>  <span class="cit-name-given-names">JP</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-source">Introduction à la flore d’Afrique</span>. <span class="cit-fpage">156</span> pp. <span class="cit-publ-name">CIRAD and Ibis Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Le Gall</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silvain</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nel</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lachaise</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Les insectes actuels témoins des passés de l’Afrique : essai sur l’origine et la singularité de l’entomofaune de la région afrotropicale</span>. <abbr class="cit-jnl-abbrev">Ann Soc Entomol Fr</abbr>, <span class="cit-vol">46</span>, <span class="cit-fpage">297</span>–<span class="cit-lpage">343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn%2BSoc%2BEntomol%2BFr%26rft.volume%253D46%26rft.spage%253D297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.40" data-doi="10.1111/nph.14413"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Villate</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bodénès</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Romiguier</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Paiva</span>  <span class="cit-name-given-names">JAP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dossat</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aury</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plomion</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Extensive recent secondary contacts between four European white oak species</span>. <abbr class="cit-jnl-abbrev">New Phytol</abbr> <span class="cit-vol">214</span>:<span class="cit-fpage">865</span>–<span class="cit-lpage">878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol%26rft.volume%253D214%26rft.spage%253D865%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.14413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.14413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">MW</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Migration and trans-Atlantic flight of locusts</span>. <abbr class="cit-jnl-abbrev">Quaternary International</abbr>, <span class="cit-vol">196</span>, <span class="cit-fpage">4</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BInternational%26rft.stitle%253DQuaternary%2BInternational%26rft.volume%253D196%26rft.issue%253D1-2%26rft.spage%253D4%26rft.epage%253D12%26rft.atitle%253DMigration%2Band%2Btrans-Atlantic%2Bflight%2Bof%2Blocusts%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.42" data-doi="10.1111/j.1365-294X.2012.05650.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lorenzen</span>  <span class="cit-name-given-names">ED</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heller</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Siegismund</span>  <span class="cit-name-given-names">HR</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Comparative phylogeography of African savannah ungulates</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>, <span class="cit-vol">21</span>, <span class="cit-fpage">3656</span>–<span class="cit-lpage">3670</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLorenzen%26rft.auinit1%253DE.%2BD.%26rft.volume%253D21%26rft.issue%253D15%26rft.spage%253D3656%26rft.epage%253D3670%26rft.atitle%253DComparative%2Bphylogeography%2Bof%2BAfrican%2Bsavannah%2Bungulates.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05650.x%26rft_id%253Dinfo%253Apmid%252F22702960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05650.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22702960&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.43" data-doi="10.1098/rspb.2005.3381"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lovejoy</span>  <span class="cit-name-given-names">NR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mullen</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sword</span>  <span class="cit-name-given-names">GA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">RF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">RG</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Ancient trans-Atlantic flight explains locust biogeography: molecular phylogenetics of <em>Schistocerca</em></span>. <abbr class="cit-jnl-abbrev">Proc R Soc B</abbr>, <span class="cit-vol">273</span>, <span class="cit-fpage">767</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DLovejoy%26rft.auinit1%253DN.R%26rft.volume%253D273%26rft.issue%253D1588%26rft.spage%253D767%26rft.epage%253D774%26rft.atitle%253DAncient%2Btrans-Atlantic%2Bflight%2Bexplains%2Blocust%2Bbiogeography%253A%2Bmolecular%2Bphylogenetics%2Bof%2BSchistocerca%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2005.3381%26rft_id%253Dinfo%253Apmid%252F16618668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2005.3381&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2013069908&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=16618668&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236279800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.44"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Goring</span></span>, </li><li><span class="cit-ed"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Reeves</span></span> and </li><li><span class="cit-ed"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Riffle</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Malek</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-chapter-title">The locusts on the daggers of Ahmose</span>. In: <span class="cit-ed"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Goring</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Reeves</span></span> and <span class="cit-ed"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Riffle</span></span> (eds.), <span class="cit-source">Chief of Seers: Egyptian Studies in Memory of Cyril Aldred</span>, <span class="cit-publ-loc">London</span>, <span class="cit-fpage">207</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Maley</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Doumenge</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Giresse</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mahé</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Philippon</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hubau</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lokonda</span>  <span class="cit-name-given-names">MO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tshibamba</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chepstow-Lusty</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Late Holocene forest contraction and fragmentation in central Africa</span>. <abbr class="cit-jnl-abbrev">Quat Res</abbr>, <span class="cit-vol">89</span>, <span class="cit-fpage">43</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat%2BRes%26rft.volume%253D89%26rft.spage%253D43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marin</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pudlo</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">CP</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-chapter-title">Likelihood-free Model Choice</span>. <span class="cit-source">Handbook of Approximate Bayesian Computation</span>. <span class="cit-publ-name">Chapman and Hall/CRC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meinzingen</span>  <span class="cit-name-given-names">WF</span></span> (<span class="cit-pub-date">1993</span>) <span class="cit-source">A guide to migrant pest management in Africa</span>. <span class="cit-publ-name">FAO</span>, <span class="cit-publ-loc">Rome, Italy</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hallager</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Monfort</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newby</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tidmus</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Black</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matthee</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleischer</span>  <span class="cit-name-given-names">RC</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Phylogeographic analysis of nuclear and mtDNA supports subspecies designations in the Ostrich (<em>Struthio camelus</em>)</span>. <abbr class="cit-jnl-abbrev">Conserv Genet</abbr>, <span class="cit-vol">12</span>, <span class="cit-fpage">423</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv%2BGenet%26rft.volume%253D12%26rft.spage%253D423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Monod</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">1971</span>) <span class="cit-article-title">Remarques sur les symetries floristiques des zones sèches nord et sud en Afrique</span>. <abbr class="cit-jnl-abbrev">Mitteilungen der Botanischen Staatssammlung München</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">375</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMitteilungen%2Bder%2BBotanischen%2BStaatssammlung%2BM%25FCnchen%26rft.volume%253D10%26rft.spage%253D375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moodley</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Russo</span>  <span class="cit-name-given-names">I-RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robovsky</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalton</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kotzé</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stejskal</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryder</span>  <span class="cit-name-given-names">OA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hermes</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walzer</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruford</span>  <span class="cit-name-given-names">MW</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Contrasting evolutionary history, anthropogenic declines and genetic contact in the northern and southern white rhinoceros (<em>Ceratotherium simum</em>)</span>. <abbr class="cit-jnl-abbrev">Proc R Soc B</abbr>, <span class="cit-vol">285</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BR%2BSoc%2BB%26rft.volume%253D285%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.51"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">TC</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Odada</span>  <span class="cit-name-given-names">EO</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Nicholson</span>  <span class="cit-name-given-names">SE</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-chapter-title">A review of climate dynamics and climate variability in Eastern Africa</span>. In: <span class="cit-ed"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-ed"><span class="cit-name-surname">Odada</span>  <span class="cit-name-given-names">EO</span></span> (Eds.), <span class="cit-source">The Limnology, Climatology and Paleoclimatology of the East African Lakes</span>. <span class="cit-publ-name">Gordon and Breach</span>, <span class="cit-publ-loc">Amsterdam</span>, pp. <span class="cit-fpage">25</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.52" data-doi="10.1177/030913339702100102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Partridge</span>  <span class="cit-name-given-names">TC</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Cainozoic environmental change in southern Africa, with special emphasis on the last 200 000 years</span>. <abbr class="cit-jnl-abbrev">Progr Phys Geog</abbr>, <span class="cit-vol">21</span>, <span class="cit-fpage">3</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2BPhysical%2BGeography%26rft.stitle%253DProgress%2Bin%2BPhysical%2BGeography%26rft.aulast%253DPartridge%26rft.auinit1%253DT.%2BC.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D22%26rft.atitle%253DCainozoic%2Benvironmental%2Bchange%2Bin%2Bsouthern%2BAfrica%252C%2Bwith%2Bspecial%2Bemphasis%2Bon%2Bthe%2Blast%2B200%2B000%2Byears%26rft_id%253Dinfo%253Adoi%252F10.1177%252F030913339702100102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/030913339702100102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1998022490&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=A1997XA44600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-671867v4.53" data-doi="10.1111/j.1365-294X.2007.03336.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pascual</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chapuis</span>  <span class="cit-name-given-names">MP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mestres</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balanyà</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huey</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gilchrist</span>  <span class="cit-name-given-names">GW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Serra</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Introduction history of <em>Drosophila subobscura</em> in the New World: a microsatellite based survey using ABC methods</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">3069</span>–<span class="cit-lpage">3083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPascual%26rft.auinit1%253DM.%26rft.volume%253D16%26rft.issue%253D15%26rft.spage%253D3069%26rft.epage%253D3083%26rft.atitle%253DIntroduction%2Bhistory%2Bof%2BDrosophila%2Bsubobscura%2Bin%2Bthe%2BNew%2BWorld%253A%2Ba%2Bmicrosatellite-based%2Bsurvey%2Busing%2BABC%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03336.x%26rft_id%253Dinfo%253Apmid%252F17651188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03336.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17651188&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248847600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.54"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">VA</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Gatehouse</span>  <span class="cit-name-given-names">AG</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Pedgley</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tatchell</span>  <span class="cit-name-given-names">GM</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-chapter-title">Long-range migration in relation to climate and weather: Africa and Europe</span>. In: <span class="cit-ed"><span class="cit-name-surname">Drake</span>  <span class="cit-name-given-names">VA</span></span>, <span class="cit-ed"><span class="cit-name-surname">Gatehouse</span>  <span class="cit-name-given-names">AG</span></span> (Eds.), <span class="cit-source">Insect Migration</span>. <span class="cit-publ-loc">Cambridge University Press, Cambridge</span>, pp. <span class="cit-fpage">3</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.55" data-doi="10.1006/tpbi.1998.1363"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pollock</span>  <span class="cit-name-given-names">DD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Microsatellite behaviour with range constraints: parameter estimation and improved distances for use in phylogenetic reconstruction</span>. <abbr class="cit-jnl-abbrev">Theor Popul Biol</abbr>, <span class="cit-vol">53</span>, <span class="cit-fpage">256</span>–<span class="cit-lpage">271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DPollock%26rft.auinit1%253DD.%2BD.%26rft.volume%253D53%26rft.issue%253D3%26rft.spage%253D256%26rft.epage%253D271%26rft.atitle%253DMicrosatellite%2Bbehavior%2Bwith%2Brange%2Bconstraints%253A%2Bparameter%2Bestimation%2Band%2Bimproved%2Bdistances%2Bfor%2Buse%2Bin%2Bphylogenetic%2Breconstruction.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.1998.1363%26rft_id%253Dinfo%253Apmid%252F9679321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.1998.1363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9679321&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000074831600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.56" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">155</span>, <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNjcxODY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.57" data-doi="10.1093/bioinformatics/btv684"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pudlo</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marin</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornuet</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">CP</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Reliable ABC model choice via random forests</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">32</span>, <span class="cit-fpage">859</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv684%26rft_id%253Dinfo%253Apmid%252F26589278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26589278&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rainey</span>  <span class="cit-name-given-names">RC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waloff</span>  <span class="cit-name-given-names">Z</span></span> (<span class="cit-pub-date">1951</span>) <span class="cit-article-title">Flying locusts and convection currents</span>. <abbr class="cit-jnl-abbrev">Anti-Locust Bull</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">51</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnti-Locust%2BBull%26rft.volume%253D9%26rft.spage%253D51%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-671867v4.59" data-doi="10.1073/pnas.94.17.9197"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rannala</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mountain</span>  <span class="cit-name-given-names">JL</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Detecting immigration by using multilocus genotypes</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci USA</abbr>, <span class="cit-vol">94</span>, <span class="cit-fpage">9197</span>–<span class="cit-lpage">9201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRannala%26rft.auinit1%253DB.%26rft.volume%253D94%26rft.issue%253D17%26rft.spage%253D9197%26rft.epage%253D9201%26rft.atitle%253DDetecting%2Bimmigration%2Bby%2Busing%2Bmultilocus%2Bgenotypes%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.17.9197%26rft_id%253Dinfo%253Apmid%252F9256459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTcvOTE5NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzE2LzY3MTg2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.60" data-doi="10.1093/bioinformatics/bty867"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raynal</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marin</span>  <span class="cit-name-given-names">J-M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pudlo</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ribatet</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">CP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estoup</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">ABC random forests for Bayesian parameter inference</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">35</span>, <span class="cit-fpage">1720</span>–<span class="cit-lpage">1728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D1720%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Richardson</span>,  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nemeth</span>,  <span class="cit-name-given-names">DJ</span></span> (<span class="cit-pub-date">1991</span>) <span class="cit-article-title">Hurricane-borne African locusts (<em>Schistocerca gregaria</em>) on the Windward Islands</span>. <abbr class="cit-jnl-abbrev">GeoJournal</abbr>, <span class="cit-vol">23</span>, <span class="cit-fpage">349</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeoJournal%26rft.volume%253D23%26rft.spage%253D349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.62" data-doi="10.1038/366146a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taieb</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Damnati</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Icole</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">1993</span>) <span class="cit-article-title">Timing of the Younger Dryas event in East Africa from lake-level changes</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">366</span>, <span class="cit-fpage">146</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D366%26rft.issue%253D6451%26rft.spage%253D146%26rft.epage%253D148%26rft.atitle%253DTiming%2Bof%2Bthe%2BYounger%2BDryas%2Bevent%2Bin%2BEast%2BAfrica%2Bfrom%2Blake-level%2Bchanges%26rft_id%253Dinfo%253Adoi%252F10.1038%252F366146a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/366146a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1994001445&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roffey</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Magor</span>  <span class="cit-name-given-names">JI</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-chapter-title">Desert Locust population parameters</span>. <span class="cit-source">Desert Locust Field Research Stations, Technical Series</span>, <span class="cit-vol">30</span>, <span class="cit-fpage">29</span> p. <span class="cit-publ-name">FAO</span>, <span class="cit-publ-loc">Rome, Italy</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDesert%2BLocust%2BField%2BResearch%2BStations%252C%2BTechnical%2BSeries%26rft.volume%253D30%26rft.spage%253D29%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rougemont</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neuenschwander</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Launey</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Reconstructing the demographic history of divergence between European river and brook lampreys using approximate Bayesian computations</span>. <abbr class="cit-jnl-abbrev">PeerJ</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">e1910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D4%26rft.spage%253De1910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.65" data-doi="10.1111/j.1471-8286.2007.01931.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rousset</span>  <span class="cit-name-given-names">F</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">GenePop’007: a complete re-implementation of the GenePop software for Windows and Linux</span>. <abbr class="cit-jnl-abbrev">Mol Ecol Resour</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">103</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DRousset%26rft.auinit1%253DF.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D106%26rft.atitle%253Dgenepop%2527007%253A%2Ba%2Bcomplete%2Bre-implementation%2Bof%2Bthe%2Bgenepop%2Bsoftware%2Bfor%2BWindows%2Band%2BLinux.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.01931.x%26rft_id%253Dinfo%253Apmid%252F21585727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.01931.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21585727&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253827100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.66" data-doi="10.1371/journal.pbio.2000234"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fraïsse</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Romiguier</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anciaux</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bierne</span>  <span class="cit-name-given-names">N</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Shedding light on the grey zone of speciation along a continuum of genomic divergence</span>. <abbr class="cit-jnl-abbrev">PLOS Biol</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">e2000234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BBiol%26rft.volume%253D14%26rft.spage%253De2000234%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.2000234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.2000234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">1992</span>) <span class="cit-article-title">Assèchement climatique vers 3000 B.P. et expansion Bantu en Afrique centrale atlantique: quelques réflexions</span>. <abbr class="cit-jnl-abbrev">B Soc Geol Fr</abbr>, <span class="cit-vol">163</span>, <span class="cit-fpage">353</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DB%2BSoc%2BGeol%2BFr%26rft.volume%253D163%26rft.spage%253D353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.68" data-doi="10.1016/0031-0182(93)90015-B"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">L</span></span> (<span class="cit-pub-date">1993</span>) <span class="cit-article-title">Palynological evidence for late Quaternary warming episodes in Southern Africa</span>. <abbr class="cit-jnl-abbrev">Palaeogeogr Palaeocl</abbr>, <span class="cit-vol">101</span>, <span class="cit-fpage">229</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPalaeogeogr%2BPalaeocl%26rft.volume%253D101%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0031-0182%252893%252990015-B%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0031-0182(93)90015-B&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dupont</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beug</span>  <span class="cit-name-given-names">H-J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Vegetation and climate changes during the last 21 000 years in S.W. Africa based on a marine pollen record</span>. <abbr class="cit-jnl-abbrev">Veg Hist Archaeobot</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">127</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeg%2BHist%2BArchaeobot%26rft.volume%253D7%26rft.spage%253D127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.70" data-doi="10.1038/40596"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stokes</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">DSG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Washington</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Multiple episodes of aridity in southern Africa since the last interglacial period</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">388</span>, <span class="cit-fpage">154</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D388%26rft.issue%253D6638%26rft.spage%253D154%26rft.epage%253D158%26rft.atitle%253DMultiple%2Bepisodes%2Bof%2Baridity%2Bin%2BSouthern%2BAfrica%2Bsince%2Bthe%2Blast%2Binterglacial%2Bperiod%26rft_id%253Dinfo%253Adoi%252F10.1038%252F40596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/40596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1997052074&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=A1997XK10900044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stute</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Talma</span>  <span class="cit-name-given-names">AS</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Glacial temperatures and moisture transport regimes reconstructed from noble gases and <sup>18</sup>O, Stampriet aquifer, Namibia</span>. <abbr class="cit-jnl-abbrev">Proceedings of International Symposium on Isotope Techniques in the Study of Past and Current Environmental Changes in the Hydrosphere and the Atmosphere, Vienna, International Atomic Energy Agency</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.72" data-doi="10.1038/ng.2398"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">JX</span></span>, <span class="cit-auth"><span class="cit-name-surname">Helgason</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masson</span>  <span class="cit-name-given-names">G</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">A direct characterization of human mutation based on microsatellites</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr>, <span class="cit-vol">44</span>, <span class="cit-fpage">1161</span>–<span class="cit-lpage">1165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSun%26rft.auinit1%253DJ.%2BX.%26rft.volume%253D44%26rft.issue%253D10%26rft.spage%253D1161%26rft.epage%253D1165%26rft.atitle%253DA%2Bdirect%2Bcharacterization%2Bof%2Bhuman%2Bmutation%2Bbased%2Bon%2Bmicrosatellites.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2398%26rft_id%253Dinfo%253Apmid%252F22922873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22922873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.73" data-doi="10.1016/j.jinsphys.2010.05.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sword</span>  <span class="cit-name-given-names">GA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lecoq</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">SJ</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Phase polyphenism and preventative locust management</span>. <abbr class="cit-jnl-abbrev">J Insect Physiol</abbr>, <span class="cit-vol">56</span>, <span class="cit-fpage">949</span>–<span class="cit-lpage">957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BInsect%2BPhysiology%26rft.stitle%253DJournal%2Bof%2BInsect%2BPhysiology%26rft.aulast%253DSword%26rft.auinit1%253DG.%2BA.%26rft.volume%253D56%26rft.issue%253D8%26rft.spage%253D949%26rft.epage%253D957%26rft.atitle%253DPhase%2Bpolyphenism%2Band%2Bpreventative%2Blocust%2Bmanagement.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jinsphys.2010.05.005%26rft_id%253Dinfo%253Apmid%252F20493192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jinsphys.2010.05.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20493192&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280376300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takezaki</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Genetic distances and reconstruction of phylogenetic trees from microsatellite DNA</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">144</span>, <span class="cit-fpage">389</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTakezaki%26rft.auinit1%253DN.%26rft.volume%253D144%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D399%26rft.atitle%253DGenetic%2BDistances%2Band%2BReconstruction%2Bof%2BPhylogenetic%2BTrees%2BFrom%2BMicrosatellite%2BDNA%26rft_id%253Dinfo%253Apmid%252F8878702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ0LzEvMzg5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTYvNjcxODY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Talma</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">JC</span></span> (<span class="cit-pub-date">1992</span>) <span class="cit-article-title">Late Quaternary paleotemperatures derived from a speleothem from Cango Caves. Cape Province, South Africa</span>. <abbr class="cit-jnl-abbrev">Quat. Res</abbr>., <span class="cit-vol">37</span>, <span class="cit-fpage">203</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuat.%2BRes%26rft.volume%253D37%26rft.spage%253D203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Uvarov</span>  <span class="cit-name-given-names">BP</span></span> (<span class="cit-pub-date">1977</span>) <span class="cit-source">Grasshoppers and Locusts</span>, vol. <span class="cit-vol">2</span>. <span class="cit-publ-name">Centre for Overseas Pest Research</span>, <span class="cit-publ-loc">London, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.77" data-doi="10.1017/S0021853700034101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vansina</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">1995</span>) <span class="cit-article-title">New Linguistic Evidence and the Bantu expansion</span>. <abbr class="cit-jnl-abbrev">J Afr Hist</abbr>, <span class="cit-vol">36</span>, <span class="cit-fpage">173</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BAfr%2BHist%26rft.volume%253D36%26rft.spage%253D173%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0021853700034101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0021853700034101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RL85200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.78" data-doi="10.1111/j.1365-2699.2004.01068.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">de Vivo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carmignotto</span>  <span class="cit-name-given-names">AP</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Holocene vegetation change and the mammal faunas of South America and Africa</span>. <abbr class="cit-jnl-abbrev">J Biogeogr</abbr>, <span class="cit-vol">31</span>, <span class="cit-fpage">943</span>–<span class="cit-lpage">957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBiogeogr%26rft.volume%253D31%26rft.spage%253D943%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2699.2004.01068.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2699.2004.01068.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000221300100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.79" data-doi="10.1002/jqs.2565"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">MJC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berkelhammer</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Björck</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cwynar</span>  <span class="cit-name-given-names">LC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Newnham</span>  <span class="cit-name-given-names">RM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">SO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Formal subdivision of the Holocene Series/Epoch: a Discussion Paper by a Working Group of INTIMATE (Integration of ice-core, marine and terrestrial records) and the Subcommission on Quaternary Stratigraphy (International Commission on Stratigraphy)</span>. <abbr class="cit-jnl-abbrev">J Quat Sci</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">649</span>–<span class="cit-lpage">659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BQuat%2BSci%26rft.volume%253D27%26rft.spage%253D649%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjqs.2565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jqs.2565&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waloff</span>  <span class="cit-name-given-names">Z</span></span> (<span class="cit-pub-date">1966</span>) <span class="cit-article-title">The upsurges and recessions of the desert locust plague: an historical survey</span>. <abbr class="cit-jnl-abbrev">Anti-Locust Memoir</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">111</span> p.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnti-Locust%2BMemoir%26rft.volume%253D8%26rft.spage%253D111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waloff</span>  <span class="cit-name-given-names">Z</span></span> (<span class="cit-pub-date">1976</span>) <span class="cit-article-title">Some temporal characteristics of desert locust plagues</span>. <abbr class="cit-jnl-abbrev">Anti-Locust Memoir</abbr>, <span class="cit-vol">13</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnti-Locust%2BMemoir%26rft.volume%253D13%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.82"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waloff</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pedgley</span>  <span class="cit-name-given-names">DE</span></span> (<span class="cit-pub-date">1986</span>) <span class="cit-article-title">Comparative biogeography and biology of the South American locust, <em>Schistocerca cancellata</em> (Serville), and the South African desert locust, <em>S. gregaria flaviventris</em> (Burmeister) (Orthoptera: Acrididae): a review</span>. <abbr class="cit-jnl-abbrev">Bull Entomol Res</abbr>, <span class="cit-vol">76</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBull%2BEntomol%2BRes%26rft.volume%253D76%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.83"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">BS</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-source">Genetic Data Analysis II</span>. <span class="cit-publ-loc">Sinauer Associates, Sunderland, Massachusetts</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.84" data-doi="10.1093/oxfordjournals.molbev.a025835"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhivotovsky</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grishechkin</span>  <span class="cit-name-given-names">SA</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Biased mutations and microsatellite variation</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">926</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025835%26rft_id%253Dinfo%253Apmid%252F9287425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a025835&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9287425&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><div class="section ref-list" id="ref-list-2"><h2 class="">References cited in Supplementary material S2</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.85" data-doi="10.1111/j.2041-210X.2011.00172.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barbet-Massin</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiguet</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">CH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thuiller</span>  <span class="cit-name-given-names">W</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Selecting pseudo-absences for species distribution models: how, where and how many?</span> <abbr class="cit-jnl-abbrev">Methods Ecol Evol</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">327</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol%2BEvol%26rft.volume%253D3%26rft.spage%253D327%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210X.2011.00172.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210X.2011.00172.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.86" data-doi="10.1111/j.1365-2656.2008.01390.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leathwick</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">A working guide to boosted regression trees</span>. <abbr class="cit-jnl-abbrev">J Anim Ecol</abbr>, <span class="cit-vol">77</span>, <span class="cit-fpage">802</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BEcol%26rft.aulast%253DElith%26rft.auinit1%253DJ.%26rft.volume%253D77%26rft.issue%253D4%26rft.spage%253D802%26rft.epage%253D813%26rft.atitle%253DA%2Bworking%2Bguide%2Bto%2Bboosted%2Bregression%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2656.2008.01390.x%26rft_id%253Dinfo%253Apmid%252F18397250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2656.2008.01390.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397250&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256539800020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-671867v4.87"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dudik</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chee</span>  <span class="cit-name-given-names">YE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">CJ</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">A statistical explanation of MaxEnt for ecologists</span>. <abbr class="cit-jnl-abbrev">Divers Distrib</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">43</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDivers%2BDistrib%26rft.volume%253D17%26rft.spage%253D43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.88" data-doi="10.1002/joc.1276"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Very High Resolution interpolated climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">Int J Climatol</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt%2BJ%2BClimatol%26rft.volume%253D25%26rft.spage%253D1965%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.1276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.1276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.89" data-doi="10.1111/j.2041-210X.2011.00134.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kriticos</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Webber</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leriche</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ota</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Macadam</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bathols</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">JK</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">CliMond: global high-resolution historical and future scenario climate surfaces for bioclimatic modelling</span>. <abbr class="cit-jnl-abbrev">Methods Ecol Evol</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">53</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol%2BEvol%26rft.volume%253D3%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2041-210X.2011.00134.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2041-210X.2011.00134.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.90" data-doi="10.1111/j.0906-7590.2005.03957.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">TP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">RG</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Selecting thresholds of occurrence in the prediction of species distributions</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr>, <span class="cit-vol">28</span>, <span class="cit-fpage">385</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D28%26rft.spage%253D385%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0906-7590.2005.03957.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0906-7590.2005.03957.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000229428800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.91"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Meynard</span>  <span class="cit-name-given-names">CN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gay</span>  <span class="cit-name-given-names">P-E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lecoq</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foucart</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piou</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chapuis</span>  <span class="cit-name-given-names">M-P</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Climate-driven geographic distribution of the desert locust during recession periods: Subspecies’ niche differentiation and relative risks under scenarios of climate change</span>. <abbr class="cit-jnl-abbrev">Glob Change Biol</abbr>, <span class="cit-vol">23</span>, <span class="cit-fpage">4739</span>–<span class="cit-lpage">4749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlob%2BChange%2BBiol%26rft.volume%253D23%26rft.spage%253D4739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.92"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rowell</span>  <span class="cit-name-given-names">DP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Senior</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vellinga</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">RJ</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Can climate projection uncertainty be constrained over Africa using metrics of contemporary performance?</span> <abbr class="cit-jnl-abbrev">Clim Change</abbr>, <span class="cit-vol">134</span>, <span class="cit-fpage">621</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClim%2BChange%26rft.volume%253D134%26rft.spage%253D621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-book" id="cit-671867v4.93"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">SN</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-source">Generalized Additive Models: an introduction with R</span>. <span class="cit-publ-loc">Boca Raton, Florida</span>: <span class="cit-publ-name">Chapman &amp; Hall/CRC</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><div class="section ref-list" id="ref-list-3"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.94" data-doi="10.1146/annurev-ecolsys-102209-144621"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>,  <span class="cit-name-given-names">M. A.</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Approximate Bayesian Computation in Evolution and Ecology</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">41</span>:<span class="cit-fpage">379</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D41%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-102209-144621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-102209-144621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000284978200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.95"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Balding</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2002</span>). <span class="cit-article-title">Approximate Bayesian Computation in Population Genetics</span>. <abbr class="cit-jnl-abbrev">GENETICS</abbr>, <span class="cit-vol">162</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">2025</span>–<span class="cit-lpage">2035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBeaumont%26rft.auinit1%253DM.%2BA.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D2025%26rft.epage%253D2035%26rft.atitle%253DApproximate%2BBayesian%2BComputation%2Bin%2BPopulation%2BGenetics%26rft_id%253Dinfo%253Apmid%252F12524368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzIwMjUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xNi82NzE4NjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.96"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Estoup</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raynal</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verdu</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Marin</span>,  <span class="cit-name-given-names">J.-M.</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Model choice using Approximate Bayesian Computation and Random Forests: analyses based on model grouping to make inferences about the genetic history of Pygmy human populations</span>. <abbr class="cit-jnl-abbrev">Journal de la Société Française de Statistique</abbr>, <span class="cit-vol">159</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">167</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bde%2Bla%2BSoci%25E9t%25E9%2BFran%25E7aise%2Bde%2BStatistique%26rft.volume%253D159%26rft.spage%253D167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.97" data-doi="10.2307/3213548"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kingman</span>,  <span class="cit-name-given-names">J. F. C.</span></span> (<span class="cit-pub-date">1982</span>). <span class="cit-article-title">On the Genealogy of Large Populations</span>. <abbr class="cit-jnl-abbrev">Journal of Applied Probability</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">A</span>):<span class="cit-fpage">27</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BProbability%26rft.volume%253D19%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3213548%26rft_id%253Dinfo%253Apmid%252F11338422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3213548&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11338422&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-671867v4.98" data-doi="10.1038/hdy.2017.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lippens</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estoup</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hima</span>,  <span class="cit-name-given-names">M. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Loiseau</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tatard</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalecky</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bâ</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kane</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diallo</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sow</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Niang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Piry</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berthier</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leblois</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duplantier</span>,  <span class="cit-name-given-names">J. M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Brouat</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Genetic structure and invasion history of the house mouse (Mus musculus domesticus) in Senegal, West Africa: a legacy of colonial and contemporary times</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr>, <span class="cit-vol">119</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">64</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D119%26rft.spage%253D64%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2017.18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2017.18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-671867v4.99" data-doi="10.1093/bioinformatics/btv684"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pudlo</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marin</span>,  <span class="cit-name-given-names">J.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Estoup</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cornuet</span>,  <span class="cit-name-given-names">J.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gautier</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Robert</span>,  <span class="cit-name-given-names">C. P.</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Reliable ABC model choice via random forests</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">32</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">859</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv684%26rft_id%253Dinfo%253Apmid%252F26589278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv684&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26589278&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F16%2F671867.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-671867v4.100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raynal</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marin</span>,  <span class="cit-name-given-names">J.-M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pudlo</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ribatet</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robert</span>,  <span class="cit-name-given-names">C. P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Estoup</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">ABC random forests for Bayesian parameter inference</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">35</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">1625</span>–<span class="cit-lpage">1633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D1625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/671867v4.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F671867v4" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2020.01.13.901538v2" title="Two-step mechanism of J-domain action in driving Hsp70 function" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 16, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/671867v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/671867v4.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/671867v4" class="active" data-icon-position="" data-hide-link-title="0">A young age of subspecific divergence in the desert locust Schistocerca gregaria, inferred by ABC Random Forest</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1107113" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-PnEDXMGiB5oDcQbzn9DLvN9t9Cw8Qe4WIN5iGlXgJSw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107113" id="node1107113--2" data-pisa="biorxiv;671867v4" data-pisa-master="biorxiv;671867" data-seqnum="1107113" data-apath="/biorxiv/early/2020/01/16/671867.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A young age of subspecific divergence in the desert locust <em>Schistocerca gregaria</em>, inferred by ABC Random Forest</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Marie-Pierre</span> <span class="nlm-surname">Chapuis</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Louis</span> <span class="nlm-surname">Raynal</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Christophe</span> <span class="nlm-surname">Plantamp</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Christine N.</span> <span class="nlm-surname">Meynard</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Laurence</span> <span class="nlm-surname">Blondin</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jean-Michel</span> <span class="nlm-surname">Marin</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Arnaud</span> <span class="nlm-surname">Estoup</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">671867; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/671867 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/671867v4" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/671867v4&amp;title=A%20young%20age%20of%20subspecific%20divergence%20in%20the%20desert%20locust%20Schistocerca%20gregaria%2C%20inferred%20by%20ABC%20Random%20Forest" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/671867v4&amp;title=A%20young%20age%20of%20subspecific%20divergence%20in%20the%20desert%20locust%20Schistocerca%20gregaria%2C%20inferred%20by%20ABC%20Random%20Forest" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/671867v4&amp;text=A%20young%20age%20of%20subspecific%20divergence%20in%20the%20desert%20locust%20Schistocerca%20gregaria%2C%20inferred%20by%20ABC%20Random%20Forest" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/671867v4&amp;title=A%20young%20age%20of%20subspecific%20divergence%20in%20the%20desert%20locust%20Schistocerca%20gregaria%2C%20inferred%20by%20ABC%20Random%20Forest" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/671867v4&amp;t=A%20young%20age%20of%20subspecific%20divergence%20in%20the%20desert%20locust%20Schistocerca%20gregaria%2C%20inferred%20by%20ABC%20Random%20Forest" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/671867v4&amp;title=A%20young%20age%20of%20subspecific%20divergence%20in%20the%20desert%20locust%20Schistocerca%20gregaria%2C%20inferred%20by%20ABC%20Random%20Forest" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/671867v4&amp;title=A%20young%20age%20of%20subspecific%20divergence%20in%20the%20desert%20locust%20Schistocerca%20gregaria%2C%20inferred%20by%20ABC%20Random%20Forest" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1107113" id="node1107113--3" data-pisa="biorxiv;671867v4" data-pisa-master="biorxiv;671867" data-seqnum="1107113" data-apath="/biorxiv/early/2020/01/16/671867.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A young age of subspecific divergence in the desert locust <em>Schistocerca gregaria</em>, inferred by ABC Random Forest</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Marie-Pierre</span> <span class="nlm-surname">Chapuis</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Louis</span> <span class="nlm-surname">Raynal</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Christophe</span> <span class="nlm-surname">Plantamp</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Christine N.</span> <span class="nlm-surname">Meynard</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Laurence</span> <span class="nlm-surname">Blondin</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Jean-Michel</span> <span class="nlm-surname">Marin</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Arnaud</span> <span class="nlm-surname">Estoup</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">671867; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/671867 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20young%20age%20of%20subspecific%20divergence%20in%20the%20desert%20locust%20Schistocerca%20gregaria%2C%20inferred%20by%20ABC%20Random%20Forest&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Chapuis%2C+Marie-Pierre&amp;rft.au=Raynal%2C+Louis&amp;rft.au=Plantamp%2C+Christophe&amp;rft.au=Meynard%2C+Christine+N.&amp;rft.au=Blondin%2C+Laurence&amp;rft.au=Marin%2C+Jean-Michel&amp;rft.au=Estoup%2C+Arnaud"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1107113/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1107113/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1107113/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1107113/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1107113/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1107113/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1107113/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1107113/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1107113/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1107113/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1107113/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1107113/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/671867v4&amp;count=horizontal&amp;via=&amp;text=A%20young%20age%20of%20subspecific%20divergence%20in%20the%20desert%20locust%20Schistocerca%20gregaria%2C%20inferred%20by%20ABC%20Random%20Forest&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/671867v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/671867v4&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/671867v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
