<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/embed/graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/11/20/848978.atom" />
<meta name="HW.pisa" content="biorxiv;848978v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding" />
<meta name="DC.Identifier" content="10.1101/848978" />
<meta name="DC.Date" content="2019-11-20" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Environmental adaptation of crops is essential for reliable agricultural production and an important breeding objectives. Genbanks provide genetic variation for the improvement of modern varieties, but the selection of suitable germplasm is frequently impeded by incomplete phenotypic data. We address this bottleneck by combining a Focused Identification of Germplasm Strategy (FIGS) with core collection methodology to select soybean ( Glycine max ) germplasm for Central European breeding from a collection of &gt;17,000 accessions. By focussing on environmental adaptation to high-latitude cold regions, we selected an ‘environmental precore’ of 3,663 accessions using environmental data and compared the Donor Population of Environments (DPE) in Asia and the Target Population of Environments (TPE) in Central Europe in the present and in 2070. Using SNP genotypes we reduced the precore into two diverse core collections of 183 and 366 of accessions as diversity panels for evaluation in high-latitude cold regions. Tests of genetic differentiation between precore and core collections revealed differentiation signatures in genomic regions that control maturity, and novel candidate loci for environmental adaptation demonstrating the potential of diversity panels for studying environmental adaptation. Objective-driven core collections increase germplasm utilization for abiotic adaptation by breeding for a rapidly changing climate, or de novo adaptation of crop species to expand cultivation ranges." />
<meta name="DC.Contributor" content="Max Haupt" />
<meta name="DC.Contributor" content="Karl Schmid" />
<meta name="article:published_time" content="2019-11-20" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;Environmental adaptation of crops is essential for reliable agricultural production and an important breeding objectives. Genbanks provide genetic variation for the improvement of modern varieties, but the selection of suitable germplasm is frequently impeded by incomplete phenotypic data. We address this bottleneck by combining a &lt;i&gt;Focused Identification of Germplasm Strategy&lt;/i&gt; (FIGS) with core collection methodology to select soybean (&lt;i&gt;Glycine max&lt;/i&gt;) germplasm for Central European breeding from a collection of &amp;gt;17,000 accessions. By focussing on environmental adaptation to high-latitude cold regions, we selected an ‘environmental precore’ of 3,663 accessions using environmental data and compared the Donor Population of Environments (DPE) in Asia and the Target Population of Environments (TPE) in Central Europe in the present and in 2070. Using SNP genotypes we reduced the precore into two diverse core collections of 183 and 366 of accessions as diversity panels for evaluation in high-latitude cold regions. Tests of genetic differentiation between precore and core collections revealed differentiation signatures in genomic regions that control maturity, and novel candidate loci for environmental adaptation demonstrating the potential of diversity panels for studying environmental adaptation. Objective-driven core collections increase germplasm utilization for abiotic adaptation by breeding for a rapidly changing climate, or &lt;i&gt;de novo&lt;/i&gt; adaptation of crop species to expand cultivation ranges.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;848978v1" />
<meta name="citation_id" content="848978v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/848978v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/848978v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/848978v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978.full.pdf" />
<meta name="citation_doi" content="10.1101/848978" />
<meta name="citation_num_pages" content="38" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="848978" />
<meta name="citation_author" content="Max Haupt" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim" />
<meta name="citation_author" content="Karl Schmid" />
<meta name="citation_author_institution" content="Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim" />
<meta name="citation_author_email" content="karl.schmid@uni-hohenheim.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5129-895X" />
<meta name="citation_reference" content="Araus, J. L. and Cairns, J. E. (2014). “Field high-throughput phenotyping: The new crop breeding frontier.” Trends in Plant Science, 19(1), 52–61." />
<meta name="citation_reference" content="Balko, C., Hahn, V., and Ordon, F. (2014). “Chilling tolerance in soybeans (Glycine max) – a prerequisite for soybean cultivation in Germany.” Journal of Cultivated Plants, 66(11), 378–388." />
<meta name="citation_reference" content="Bari, A., Street, K., Mackay, M., Endresen, D. T. F., de Pauw, E., and Amri, A. (2012). “Focused identification of germplasm strategy (FIGS) detects wheat stem rust resistance linked to environmental variables.” Genetic Resources and Crop Evolution, 59(7), 1465–1481." />
<meta name="citation_reference" content="Bemer, M., Van Mourik, H., Muiño, J. M., Ferrándiz, C., Kaufmann, K., and Angenent, G. C. (2017). “FRUITFULL controls SAUR10 expression and regulates Arabidopsis growth and architecture.” Journal of Experimental Botany, 68(13), 3391–3403." />
<meta name="citation_reference" content="Bernard, R. L. (1971). “Two Major Genes for Time of Flowering and Maturity in Soybeans.” Crop Science, 11, 242–244." />
<meta name="citation_reference" content="Beukelaer, H. D., Davenport, G. F., De Meyer, G., and Fack, V. (2017a). “JAMES: An object-oriented Java framework for discrete optimization using local search metaheuristics.” Software - Practice and Experience, 47(6), 921–938." />
<meta name="citation_reference" content="Beukelaer, H. D., Davenport, G. F., and Fack, V. (2017b). “Multi-Purpose Core Subset Selection, &lt;www.corehunter.org&gt;." />
<meta name="citation_reference" content="Beukelaer, H. D., Smýkal, P., Davenport, G. F., and Fack, V. (2012). “Core Hunter II: fast core subset selection based on multiple genetic diversity measures using Mixed Replica search.” BMC Bioinformatics, 13(1), 312." />
<meta name="citation_reference" content="Bhullar, N. K., Street, K., Mackay, M., Yahiaoui, N., and Keller, B. (2009). “Unlocking wheat genetic resources for the molecular identification of previously undescribed functional alleles at the Pm3 resistance locus.” Proceedings of the National Academy of Sciences of the United States of America, 106(23), 9519–9524." />
<meta name="citation_reference" content="BMEL (2016). “Ackerbohne, Erbse &amp; Co., &lt;https://www.bmel.de/SharedDocs/Downloads/Broschueren/EiweisspflanzenstrategieBMEL.pdf?{_}{_}blob=publicationFile&gt;." />
<meta name="citation_reference" content="Bootsma, A., McKenney, D. W., Anderson, D., and Papadopol, P. (2007). “A re-evaluation of crop heat units in the maritime provinces of Canada.” Canadian Journal of Plant Science, 87(2), 281–287." />
<meta name="citation_reference" content="Brown, D. and Bootsma, A. (1993). “Crop Heat Units for Corn and Other Warm Season Crops in Ontario.” Report No. Factsheet No. 93-119, Ministry of Agriculture and Food Ontario, &lt;https://www.sojafoerderring.de/wp-content/uploads/2014/02/Berechnung-CHU-Uni-Guelph-Ontario.pdf&gt;." />
<meta name="citation_reference" content="Browning, B. L. and Browning, S. R. (2016). “Genotype Imputation with Millions of Reference Samples.” American Journal of Human Genetics, 98(1), 116–126." />
<meta name="citation_reference" content="Buzzell, R. I. (1971). “Inheritance of a soybean flowering response to fluorescent-daylength conditions.” Canadian Journal of Genetics and Cytology, 13(4), 703–707." />
<meta name="citation_reference" content="Cober, E. R., Molnar, S. J., Rai, S., Soper, J. F., and Voldeng, H. D. (2013). “Selection for cold tolerance during flowering in short-season soybean.” Crop Science, 53(4), 1356–1365." />
<meta name="citation_reference" content="Cockram, J. and Mackay, I. (2018). “Genetic Mapping Populations for Conducting High-Resolution Trait Mapping in Plants.” Plant Genetics and Molecular Biology, R. K. Varshney, M. K. Pandey, and A. Chitikineni, eds., Springer International Publishing, Cham, 109–138." />
<meta name="citation_reference" content="Diamond, J. (2002). “Evolution, consequences and future of plant and animal domestication.” Nature, 418(6898), 700–707." />
<meta name="citation_reference" content="Dray, S. and Dufour, A.-B. (2007). “The ade4 Package: Implementing the Duality Diagram for Ecologists.” Journal of Statistical Software, 22(4)." />
<meta name="citation_reference" content="Du, Q. L., Cui, W. Z., Zhang, C. H., and Yu, D. Y. (2010). “GmRFP1 encodes a previously unknown RING-type E3 ubiquitin ligase in Soybean (Glycine max).” Molecular Biology Reports, 37(2), 685–693." />
<meta name="citation_reference" content="Endresen, D. T. F. (2010). “Predictive association between trait data and ecogeographic data for Nordic barley landraces.” Crop Science, 50(6), 2418–2430." />
<meta name="citation_reference" content="European Soya Declaration (2017). “Common Declaration of Austria, Croatia, Finland, France, Germany, Greece, Hungary, Italy, Luxemburg, the Netherlands, Poland, Romania, Slovakia and Slovenia European Soya Declaration – Enhancing soya and other legumes cultivation, &lt;https://www.bmel.de/SharedDocs/Downloads/Landwirtschaft/Pflanze/SojaErklaerung.pdf?{_}{_}blob=publicationFile&gt;." />
<meta name="citation_reference" content="Fick, S. and Hijmans, R. (2017). “Worldclim 2: New 1-km spatial resolution climate surfaces for global land areas.” International Journal of Climatology, 37(12), 4302–4315." />
<meta name="citation_reference" content="citation_title=“Genetic Perspectives of Germplasm Conservation.” Genetic Manipulation: Impact on Man and Society;citation_pages=161-170;citation_year=1984" />
<meta name="citation_reference" content="Funatsuki, H., Kawaguchi, K., Matsuba, S., Sato, Y., and Ishimoto, M. (2005). “Mapping of QTL associated with chilling tolerance during reproductive growth in soybean.” Theoretical and Applied Genetics, 111(5), 851–861." />
<meta name="citation_reference" content="Funatsuki, H., Matsuba, S., Kawaguchi, K., Murakami, T., and Sato, Y. (2004). “Methods for evaluation of soybean chilling tolerance at the reproductive stage under artificial climatic conditions.” Plant Breeding, 123(6), 558–563." />
<meta name="citation_reference" content="Funatsuki, H., Suzuki, M., Hirose, A., Inaba, H., Yamada, T., Hajika, M., Komatsu, K., Katayama, T., Sayama, T., Ishimoto, M., and Fujino, K. (2014). “Molecular basis of a shattering resistance boosting global dissemination of soybean.” Proceedings of the National Academy of Sciences, 111(50), 17797–17802." />
<meta name="citation_reference" content="Furbank, R. T. and Tester, M. (2011). “Phenomics - technologies to relieve the phenotyping bottleneck.” Trends in Plant Science, 16(12), 635–644." />
<meta name="citation_reference" content="Gass, T., Schori, A., Fossati, A., Soldati, A., and Stamp, P. (1996). “Cold tolerance of soybean (Glycine max (L.) Merr.) during the reproductive phase.” European Journal of Agronomy, 5(1-2), 71–88." />
<meta name="citation_reference" content="Gautier, M. (2015). “Genome-wide scan for adaptive divergence and association with population-specific covariates.” Genetics, 201(4), 1555–1579." />
<meta name="citation_reference" content="Gizlice, Z., Carter, T. E., and Burton, J. W. (1994). “Genetic Base for North American Public Soybean Cultivars Released between 1947 and 1988.” Crop Science, 34(5), 1143–1151." />
<meta name="citation_reference" content="Grant, D., Nelson, R. T., Cannon, S. B., Shoemaker, R. C., and (SoyBase) (2010). “SoyBase, the USDA-ARS soybean genetics and genomics database, &lt;http://soybase.org&gt;." />
<meta name="citation_reference" content="Günther, T. and Coop, G. (2013). “Robust identification of local adaptation from allele frequencies.” Genetics, 195(1), 205–220." />
<meta name="citation_reference" content="Hahn, V. and Würschum, T. (2014). “Molecular genetic characterization of Central European soybean breeding germplasm.” Plant Breeding, 133(6), 748–755." />
<meta name="citation_reference" content="Harlan, J. R. (1992). “Space, Time, and Variation.” Crops &amp; Man, American Society of Agronomy, Crop Science Society of America, Madison, WI, Chapter 7, 135 – 155." />
<meta name="citation_reference" content="Hijmans, R. J., Cameron, S. E., Parra, J. L., Jones, P. G., and Jarvis, A. (2005). “Very high resolution interpolated climate surfaces for global land areas.” International Journal of Climatology, 25(15), 1965–1978." />
<meta name="citation_reference" content="Hui, W., Gel, Y., and Gastwirth, J. (2008). “lawstat: an R package for law, public policy and biostatistics.” Journal of Statistical Software, 28(3), 1–26." />
<meta name="citation_reference" content="Hyten, D. L., Song, Q., Zhu, Y., Choi, I.-Y., Nelson, R. L., Costa, J. M., Specht, J. E., Shoemaker, C., and Cregan, P. B. (2006). “Impacts of genetic bottlenecks on soybean genome diversity.” PNAS, 103(45), 16666–16671." />
<meta name="citation_reference" content="Iuchi, S., Kobayashi, Y., Koyama, H., and Kobayashi, M. (2014). “STOP1, a Cys2/His2 type zinc-finger protein, plays critical role in acid soil tolerance in Arabidopsis.” Plant Signaling &amp; Behavior, 3(2), 128–130." />
<meta name="citation_reference" content="Iwasaki, K., Kamauchi, S., Wadahama, H., Ishimoto, M., Kawada, T., and Urade, R. (2009). “Molecular cloning and characterization of soybean protein disulfide isomerase family proteins with nonclassic active center motifs.” FEBS Journal, 276(15), 4130–4141." />
<meta name="citation_reference" content="Jarquin, D., Specht, J., and Lorenz, A. (2016). “Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions.” G3, 6(8), 2329–2341." />
<meta name="citation_reference" content="Jiang, B., Nan, H., Gao, Y., Tang, L., Yue, Y., Lu, S., Ma, L., Cao, D., Sun, S., Wang, J., Wu, C., Yuan, X., Hou, W., Kong, F., Han, T., and Liu, B. (2014). “Allelic combinations of soybean maturity Loci E1, E2, E3 and E4 result in diversity of maturity and adaptation to different latitudes.” PlOS ONE, 9(8), e106042." />
<meta name="citation_reference" content="Khazaei, H., Street, K., Bari, A., Mackay, M., and Stoddard, F. L. (2013). “The FIGS (Focused Identification of Germplasm Strategy) approach identifies traits related to drought adaptation in Vicia faba genetic resources.” PLOS ONE, 8(5)." />
<meta name="citation_reference" content="Kongrit, D., Jisaka, M., Iwanga, C., Yokomichi, H., Katsube, T., Nishimura, K., Nagaya, T., and Yokota, K. (2007). “Molecular Cloning and Functional Expression of Soybean Allene Oxide Synthases.” Bioscience, Biotechnology and Biochemistry, 71(2), 491–498." />
<meta name="citation_reference" content="Kovinich, N., Saleem, A., Arnason, J. T., and Miki, B. (2012). “Identification of two anthocyanidin reductase genes and three red-brown soybean accessions with reduced anthocyanidin reductase 1 mRNA, activity, and seed coat proanthocyanidin amounts.” Journal of Agricultural and Food Chemistry, 60(2), 574–584." />
<meta name="citation_reference" content="Kurasch, A. K., Hahn, V., Leiser, W. L., Vollmann, J., Schori, A., Bétrix, C.-A., Mayr, B., Winkler, J., Mechtler, K., Aper, J., Sudaric, A., Pejic, I., Sarcevic, H., Jeanson, P., Balko, C., Signor, M., Miceli, F., Strijk, P., Rietman, H., Muresanu, E., Djordjevic, V., Pospiöil, A., Barion, G., Weigold, P., Streng, S., Krön, M., and Würschum, T. (2017). “Identification of mega-environments in Europe and effect of allelic variations at maturity E loci on adaptation of European soybean.” Plant, Cell &amp; Environment, 0000, 765–778." />
<meta name="citation_reference" content="Langridge, P. and Waugh, R. (2019). “Harnessing the potential of germplasm collections.” Nature Genetics, 51(2), 200–201." />
<meta name="citation_reference" content="Liao, Y., Zou, H. F., Wei, W., Hao, Y. J., Tian, A. G., Huang, J., Liu, Y. F., Zhang, J. S., and Chen, S. Y. (2008). “Soybean GmbZIP44, GmbZIP62 and GmbZIP78 genes function as negative regulator of ABA signaling and confer salt and freezing tolerance in transgenic Arabidopsis.” Planta, 228(2), 225–240." />
<meta name="citation_reference" content="Littlejohns, D. a. and Tanner, J. W. (1976). “Preliminary Studies on the Cold Tolerance of Soybean Seedlings.” Canadian Journal of Plant Science, 56(2), 371–375." />
<meta name="citation_reference" content="Liu, B., Watanabe, S., Uchiyama, T., Kong, F., Kanazawa, A., Xia, Z., Nagamatsu, A., Arai, M., Yamada, T., Kitamura, K., Masuta, C., Harada, K., and Abe, J. (2010). “The Soybean Stem Growth Habit Gene Dt1 Is an Ortholog of Arabidopsis TERMINAL FLOWER1.” Plant Physiology, 153(1), 198–210." />
<meta name="citation_reference" content="Ma, Y. Y., Zhang, L. W., Li, P. L., Gan, R., Li, X. P., Zhang, R., Wang, Y., and Wang, N. N. (2006). “Cloning and preliminary characterization of three receptor-like kinase genes in soybean.” Journal of Integrative Plant Biology, 48(11), 1338–1347." />
<meta name="citation_reference" content="Malomane, D. K., Reimer, C., Weigend, S., Weigend, A., Sharifi, A. R., and Simianer, H. (2018). “Efficiency of different strategies to mitigate ascertainment bias when using SNP panels in diversity studies.” BMC genomics, 19(1), 22." />
<meta name="citation_reference" content="McGonigle, B., Keeler, S. J., Lau, S.-M. C., Koeppe, M. K., and O’Keefe, D. P. (2000). “A Genomics Approach to the Comprehensive Analysis of the Glutathione S -Transferase Gene Family in Soybean and Maize.” Plant Physiology, 124(3), 1105–1120." />
<meta name="citation_reference" content="Mizoi, J., Ohori, T., Moriwaki, T., Kidokoro, S., Todaka, D., Maruyama, K., Kusakabe, K., Os-akabe, Y., Shinozaki, K., and Yamaguchi-Shinozaki, K. (2013). “GmDREB2A;2, a Canonical DEHYDRATION-RESPONSIVE ELEMENT-BINDING PROTEIN2-Type Transcription Fac-tor in Soybean, Is Posttranslationally Regulated and Mediates Dehydration-Responsive Element-Dependent Gene Expression.” Plant Physiology, 161(1), 346–361." />
<meta name="citation_reference" content="Nelson, R. L. (2011). “Managing self-pollinated germplasm collections to maximize utilization.” Plant Genetic Resources, 9(01), 123–133." />
<meta name="citation_reference" content="Odong, T. L., Jansen, J., van Eeuwijk, F. A., and van Hintum, T. J. (2013). “Quality of core collections for effective utilisation of genetic resources review, discussion and interpretation.” Theoretical and Applied Genetics, 126(2), 289–305." />
<meta name="citation_reference" content="Oliveira, M. F., Nelson, R. L., Geraldi, I. O., Cruz, C. D., and de Toledo, J. F. F. (2010). “Establishing a soybean germplasm core collection.” Field Crops Research, 119(2-3), 277–289." />
<meta name="citation_reference" content="Olsson, A. S., Engström, P., and Söderman, E. (2004). “The homeobox genes ATHB12 and ATHB7 encode potential regulators of growth in response to water deficit in Arabidopsis.” Plant Molecular Biology, 55(5), 663–677." />
<meta name="citation_reference" content="Park, S.-Y., Yu, J.-W., Park, J.-S., Li, J., Yoo, S.-C., Lee, N.-Y., Lee, S.-K., Jeong, S.-W., Seo, H. S., Koh, H.-J., Jeon, J.-S., Park, Y.-I., and Paek, N.-C. (2007). “The Senescence-Induced Staygreen Protein Regulates Chlorophyll Degradation.” the Plant Cell Online, 19(5), 1649–1664." />
<meta name="citation_reference" content="Recknagel, J. D. S. (2015). “Sorten für Deutschland, &lt;https://www.sojafoerderring.de/anbauratgeber/sortenratgeber/deutschland/&gt;." />
<meta name="citation_reference" content="Sanders, R., Bari, A., Street, K., and Devlin, M. (2013). “A new approach to mining agricultural gene banks – to speed the pace of research innovation for food security. ‘FIGS’ - the Focused Identification of Germplasm Strategy.” Report No. 3, International Center for Agricultural Research in the Dry Areas (ICARDA), Beirut." />
<meta name="citation_reference" content="Sasaki, K. and Imai, R. (2012). “Pleiotropic Roles of Cold Shock Domain Proteins in Plants.” Frontiers in Plant Science, 2(January), 1–6." />
<meta name="citation_reference" content="Schmutz, J., Cannon, S. B., Schlueter, J., Ma, J., Mitros, T., Nelson, W., Hyten, D. L., Song, Q., Thelen, J. J., Cheng, J., Xu, D., Hellsten, U., May, G. D., Yu, Y., Sakurai, T., Umezawa, T., Bhattacharyya, M. K., Sandhu, D., Valliyodan, B., Lindquist, E., Peto, M., Grant, D., Shu, S., Goodstein, D., Barry, K., Futrell-Griggs, M., Abernathy, B., Du, J., Tian, Z., Zhu, L., Gill, N., Joshi, T., Libault, M., Sethuraman, A., Zhang, X.-C., Shinozaki, K., Nguyen, H. T., Wing, R. a., Cregan, P., Specht, J., Grimwood, J., Rokhsar, D., Stacey, G., Shoemaker, R. C., and Jackson, R. a. (2010). “Genome sequence of the palaeopolyploid soybean.” Nature, 463(7278), 178–183." />
<meta name="citation_reference" content="Schnable, P. S., Wu, Y., Wang, M. L., Guo, T., Yu, J., Tesso, T. T., Bernardo, R., Roozeboom, K. L., Wang, D., Yu, X., Li, X., Mitchell, S. E., Pederson, G. A., and Zhu, C. (2016). “Genomic prediction contributing to a promising global strategy to turbocharge gene banks.” Nature Plants, 2(10), 16150." />
<meta name="citation_reference" content="Silva, P. A., Silva, J. C. F., Caetano, H. D., Machado, J. P. B., Mendes, G. C., Reis, P. A., Brustolini, O. J., Dal-Bianco, M., and Fontes, E. P. (2015). “Comprehensive analysis of the endoplasmic reticulum stress response in the soybean genome: Conserved and plant-specific features.” BMC Genomics, 16(1), 1–20." />
<meta name="citation_reference" content="Somers, D. E., Schultz, T. F., Milnamow, M., and Kay, S. A. (2000). “ZEITLUPE encodes a novel clock-associated PAS protein from Arabidopsis.” Cell, 101(3), 319–329." />
<meta name="citation_reference" content="Song, Q., Hyten, D. L., Jia, G., Quigley, C. V., Fickus, E. W., Nelson, R. L., and Cregan, P. B. (2013). “Development and Evaluation of SoySNP50K, a High-Density Genotyping Array for Soybean.” PLoS ONE, 8(1), 1–12." />
<meta name="citation_reference" content="Song, Q., Hyten, D. L., Jia, G., Quigley, C. V., Fickus, E. W., Nelson, R. L., and Cregan, P. B. (2015). “Fingerprinting Soybean Germplasm and Its Utility in Genomic Research.” G3, 5(10), 1999–2006." />
<meta name="citation_reference" content="Song, W., Solimeo, H., Rupert, R. A., Yadav, N. S., and Zhu, Q. (2002). “Functional dissection of a Rice Dr1/DrAp1 transcriptional repression complex.” Plant Cell, 14(January), 181–195." />
<meta name="citation_reference" content="Spencer, C. C., Su, Z., Donnelly, P., and Marchini, J. (2009). “Designing genome-wide association studies: Sample size, power, imputation, and the choice of genotyping chip.” PLoS Genetics, 5(5)." />
<meta name="citation_reference" content="Street, K., Mackay, M., Zuev, E., Kaul, N., El Bouhssini, M., Konopka, J., and Mitrofanova, O. (2008). “Diving into the genepool: a rational system to access specific traits from large germplasm collections.” Proceedings of the 11th International wheat genetics symposium, 28–31." />
<meta name="citation_reference" content="Su, L. T., Li, J. W., Liu, D. Q., Zhai, Y., Zhang, H. J., Li, X. W., Zhang, Q. L., Wang, Y., and Wang, Q. Y. (2014). “A novel MYB transcription factor, GmMYBJ1, from soybean confers drought and cold tolerance in Arabidopsis thaliana.” Gene, 538(1), 46–55." />
<meta name="citation_reference" content="Sun, H., Jia, Z., Cao, D., Jiang, B., Wu, C., Hou, W., Liu, Y., Fei, Z., Zhao, D., and Han, T. (2011). “GmFT2a, a soybean homolog of flowering locus T, is involved in flowering transition and maintenance.” PLOS ONE, 6(12), 18–20." />
<meta name="citation_reference" content="Tanksley, S. D. and McCouch, S. R. (1997). “Seed Banks and Molecular Maps: Unlocking Genetic Potential from the Wild.” Science, 277(5329), 1063–1066." />
<meta name="citation_reference" content="Thachuk, C., Crossa, J., Franco, J., Dreisigacker, S., Warburton, M., and Davenport, G. F. (2009). “Core Hunter: an algorithm for sampling genetic resources based on multiple genetic measures.” BMC bioinformatics, 10(243), 1–13." />
<meta name="citation_reference" content="Varshney, R. K., Singh, V. K., Kumar, A., Powell, W., and Sorrells, M. E. (2018). “Can genomics deliver climate-change ready crops?.” Current Opinion in Plant Biology, 45, 205–211." />
<meta name="citation_reference" content="Wang, C., Hu, S., Gardner, C., and Lübberstedt, T. (2017a). “Emerging Avenues for Utilization of Exotic Germplasm.” Trends in Plant Science, 22(7), 624–637." />
<meta name="citation_reference" content="Wang, H., Zhao, S., Gao, Y., and Yang, J. (2017b). “Characterization of dof transcription factors and their responses to osmotic stress in poplar (Populus trichocarpa).” PLoS ONE, 12(1), 1–19." />
<meta name="citation_reference" content="Wang, Y., Chai, C., Valliyodan, B., Maupin, C., Annen, B., and Nguyen, H. T. (2015). “Genome-wide analysis and expression profiling of the PIN auxin transporter gene family in soybean (Glycine max).” BMC Genomics, 16(1), 1–13." />
<meta name="citation_reference" content="Watanabe, S., Xia, Z., Hideshima, R., Tsubokura, Y., Sato, S., Yamanaka, N., Takahashi, R., Anai, T., Tabata, S., Kitamura, K., and Harada, K. (2011). “A map-based cloning strategy employing a residual heterozygous line reveals that the GIGANTEA gene is involved in soybean maturity and flowering.” Genetics, 188(2), 395–407." />
<meta name="citation_reference" content="Watson, A., Ghosh, S., Williams, M. J., Cuddy, W. S., Simmonds, J., Rey, M. D., Asyraf Md Hatta, M., Hinchliffe, A., Steed, A., Reynolds, D., Adamski, N. M., Breakspear, A., Korolev, A., Rayner, T., Dixon, L. E., Riaz, A., Martin, W., Ryan, M., Edwards, D., Batley, J., Raman, H., Carter, J., Rogers, C., Domoney, C., Moore, G., Harwood, W., Nicholson, P., Dieters, M. J., Delacy, I. H., Zhou, J., Uauy, C., Boden, S. A., Park, R. F., Wulff, B. B., and Hickey, L. T. (2018). “Speed breeding is a powerful tool to accelerate crop research and breeding.” Nature Plants, 4(1), 23–29." />
<meta name="citation_reference" content="Xavier, A., Thapa, R., Muir, W. M., and Rainey, K. M. (2018). “Population and quantitative genomic properties of the USDA soybean germplasm collection.” Plant Genetic Resources: Characterisation and Utilisation, 16(6), 513–523." />
<meta name="citation_reference" content="Yamaguchi, N., Kurosaki, H., Ishimoto, M., and Kawasaki, M. (2015). “Early-Maturing and Chilling-Tolerant Soybean Lines Derived from Crosses between Japanese and Polish Cultivars.” Plant Production Science, 18(August 2014), 234–239." />
<meta name="citation_reference" content="Yu, G. H., Jiang, L. L., Ma, X. F., Xu, Z. S., Liu, M. M., Shan, S. G., and Cheng, X. G. (2014). “A soybean C2H2-type zinc finger gene GmZF1 enhanced cold tolerance in transgenic Arabidopsis.” PLOS ONE, 9(10)." />
<meta name="citation_reference" content="Zeng, X., Liu, H., Du, H., Wang, S., Yang, W., Chi, Y., Wang, J., Huang, F., and Yu, D. (2018). “Soybean MADS-box gene GmAGL1 promotes flowering via the photoperiod pathway.” BMC Genomics, 19(1), 1–17." />
<meta name="citation_reference" content="Zhao, C., Takeshima, R., Zhu, J., Xu, M., Sato, M., Watanabe, S., Kanazawa, A., Liu, B., Kong, F., Yamada, T., and Abe, J. (2016). “A recessive allele for delayed flowering at the soybean maturity locus E9 is a leaky allele of FT2a, a FLOWERING LOCUS T ortholog.” BMC Plant Biology, 16(1), 1–15." />
<meta name="citation_reference" content="Zhou, Q. Y., Tian, A. G., Zou, H. F., Xie, Z. M., Lei, G., Huang, J., Wang, C. M., Wang, H. W., Zhang, J. S., and Chen, S. Y. (2008). “Soybean WRKY-type transcription factor genes, GmWRKY13, GmWRKY21, and GmWRKY54, confer differential tolerance to abiotic stresses in transgenic Arabidopsis plants.” Plant Biotechnology Journal, 6(5), 486–503." />
<meta name="twitter:title" content="Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Environmental adaptation of crops is essential for reliable agricultural production and an important breeding objectives. Genbanks provide genetic variation for the improvement of modern varieties, but the selection of suitable germplasm is frequently impeded by incomplete phenotypic data. We address this bottleneck by combining a Focused Identification of Germplasm Strategy (FIGS) with core collection methodology to select soybean ( Glycine max ) germplasm for Central European breeding from a collection of &gt;17,000 accessions. By focussing on environmental adaptation to high-latitude cold regions, we selected an ‘environmental precore’ of 3,663 accessions using environmental data and compared the Donor Population of Environments (DPE) in Asia and the Target Population of Environments (TPE) in Central Europe in the present and in 2070. Using SNP genotypes we reduced the precore into two diverse core collections of 183 and 366 of accessions as diversity panels for evaluation in high-latitude cold regions. Tests of genetic differentiation between precore and core collections revealed differentiation signatures in genomic regions that control maturity, and novel candidate loci for environmental adaptation demonstrating the potential of diversity panels for studying environmental adaptation. Objective-driven core collections increase germplasm utilization for abiotic adaptation by breeding for a rapidly changing climate, or de novo adaptation of crop species to expand cultivation ranges." />
<meta name="og-title" property="og:title" content="Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/848978v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Environmental adaptation of crops is essential for reliable agricultural production and an important breeding objectives. Genbanks provide genetic variation for the improvement of modern varieties, but the selection of suitable germplasm is frequently impeded by incomplete phenotypic data. We address this bottleneck by combining a Focused Identification of Germplasm Strategy (FIGS) with core collection methodology to select soybean ( Glycine max ) germplasm for Central European breeding from a collection of &gt;17,000 accessions. By focussing on environmental adaptation to high-latitude cold regions, we selected an ‘environmental precore’ of 3,663 accessions using environmental data and compared the Donor Population of Environments (DPE) in Asia and the Target Population of Environments (TPE) in Central Europe in the present and in 2070. Using SNP genotypes we reduced the precore into two diverse core collections of 183 and 366 of accessions as diversity panels for evaluation in high-latitude cold regions. Tests of genetic differentiation between precore and core collections revealed differentiation signatures in genomic regions that control maturity, and novel candidate loci for environmental adaptation demonstrating the potential of diversity panels for studying environmental adaptation. Objective-driven core collections increase germplasm utilization for abiotic adaptation by breeding for a rapidly changing climate, or de novo adaptation of crop species to expand cultivation ranges." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-11-20" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/848978v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/848978v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/848978v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/848978v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1002231" />
    <title>Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WLdw-MrL7tPQHR_VwjM5Y0dgxGw0Niy2tAv55tG2tcU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1002231","apath":"\/biorxiv\/early\/2019\/11\/20\/848978.atom","pisa":"biorxiv;848978v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;848978v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/848978v1","title":"Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding","identifier":"node\/1002231"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226ecY+xdeTSMLLQICnD2V8L+RbvKJ0luT7Kq0JDSZkuF\\\\\\\/Pg\\\\\\\/cp2jS5KylMpouJ3GojRiXQAxJt1vUX6RFp2drx9470qKlnERopAbUTqvw3glf6abbRz\\\\\\\/eQxYeDNiBCLx9Zd0UKDECqOisIn177Ft7Xxy\\\\\\\/HuNwXouXp\\\\\\\/542fatBwATZeFCKqS2zVKz\\\\\\\/VzFOfcwij9s5\\\\\\\/AZAOz3Fziq4kYYM4kT8VpbOcdfRdkzMnilGS\\\\\\\/pJaqNDBhMFzToBqj4Kb8d8Vz7ndQ218LXVCrYpHA0AKd9rXAWQHd83NFaz8JVvgbcUbr1oDlEkzvjZ1mdK3JcIWS9MhgY7ba+tKbF9jISloWTDVATdaIUX3+SQag2milabFj+\\\\\\\/rJnMjaGbk8HWLGGPEbv6IszzxUX5m8pCZf0mL+\\\\\\\/pgtQqsnKfE+Z6yQ3fb1LpTOFTUm6PwsMCHHXgXOoOxtIeUNrw0u61oDyZFzBa3nEDMIqCJElv3Zwn+1SCf7UisQy9UDFizhuz+2uAkhPzkLs4zqB1IyDKt2hUHQs6h2i9uJM1i5Smt9JvxXOjqMCOeWc+owkb5M1tygGc42e2im14d0Mwe0NkhN1uCQFcrJsW+DeFQcKaIfsbCIBlGYJGfGHexJzOPWxNIwzsH1Ayqj7nckow8Txcs+iB1LAA307Sha7B0uD\\\\\\\/PjP+6c5QAV\\\\\\\/jT8ZlQrEarFQ8ILpaWCw9S1+sHR+l5tlZw270NgxCZfBSF22JVk+5zDn7WKEUe7V62var4lQIj4ctHv00LewCYubgBAIytQ7HV3YTMTVxzCGXMYzoJrqH2mUmEGirWrtXFxdupZWuVt\\\\\\\/LDHc7vkYwJNjYOEAoje0Qt9S4RrNaey2hiQ5S0wnYSOLyYK+alceU2+bIuCZUmvx33FJZclCLccdfMR2vM9dxlfSZQJJBveai1CeiMbxhS2yDSvkwfWB9J1HkpIHKZzVhB4Adkp9x2wSxE72b9ptTaGPYETF8HmvgBPfBeLfDikGPTAnMsOR\\\\\\\/BNHrD9RE\\\\\\\/JzExn0xXKfdjqdGP0zH+HJmSgz9qR5mxgKM+alystkIhljnJgMc3d+gESGqhG6iJwm7K5MU20MXkW9aSV1rVTOgDRZSyj\\\\\\\/b\\\\\\\/WsTOuMiv3LCbcEEcmfucAXJ0Le3HOeE\\\\\\\/o44B4wnZjjmptSN1juU0A7XMkIcMkjCtZ8nIytBy+AgDl\\\\\\\/IHWzzg60y6S40FjO0QLl6ZETcVBNC1ofpuvYq2Ypi2QT0UE6VgVNZkCmFeMrq2ZdQ\\\\\\\/OcS6NLZrfP057Vg2tUl8nP\\\\\\\/VDrKdgylIWE\\\\\\\/DfmD\\\\\\\/orYbWpDlCF6L62B6HdJlBZR4N8eMe9dhr7kAOHJ5NNEU8h1ZVK\\\u0022,\\\u0022iv\\\u0022:\\\u0022rhIeVPawEWqeLpwvG2O62Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022c223a6d283d0dba31cc5ad87ecb760fd897a555a1aaf2fc4ab8511b7e34c0b3d\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/848978v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1002231 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/848978v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C4v3zlGdgxAhXdwqI6zDw80cFyagoIzQfDPVR9tw_U4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1002231" id="node1002231" data-pisa="biorxiv;848978v1" data-pisa-master="biorxiv;848978" data-apath="/biorxiv/early/2019/11/20/848978.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Max</span> <span class="nlm-surname">Haupt</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5129-895X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Karl</span> <span class="nlm-surname">Schmid</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/848978 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1002231" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Max Haupt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim</span>, Stuttgart, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Max%2BHaupt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haupt%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMax%2BHaupt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Karl Schmid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute of Plant Breeding, Seed Science and Population Genetics, University of Hohenheim</span>, Stuttgart, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl%2BSchmid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmid%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKarl%2BSchmid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5129-895X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Karl Schmid</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:karl.schmid@uni-hohenheim.de" class="" data-icon-position="" data-hide-link-title="0">karl.schmid@uni-hohenheim.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/848978v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1002231" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1002231/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/848978v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1002231" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1002231/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/848978v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1002231" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1002231/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/848978v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1002231" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1002231/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/848978v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1002231" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1002231/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/848978v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1002231" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1002231/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-3">Environmental adaptation of crops is essential for reliable agricultural production and an important breeding objectives. Genbanks provide genetic variation for the improvement of modern varieties, but the selection of suitable germplasm is frequently impeded by incomplete phenotypic data. We address this bottleneck by combining a <em>Focused Identification of Germplasm Strategy</em> (FIGS) with core collection methodology to select soybean (<em>Glycine max</em>) germplasm for Central European breeding from a collection of &gt;17,000 accessions. By focussing on environmental adaptation to high-latitude cold regions, we selected an ‘environmental precore’ of 3,663 accessions using environmental data and compared the Donor Population of Environments (DPE) in Asia and the Target Population of Environments (TPE) in Central Europe in the present and in 2070. Using SNP genotypes we reduced the precore into two diverse core collections of 183 and 366 of accessions as diversity panels for evaluation in high-latitude cold regions. Tests of genetic differentiation between precore and core collections revealed differentiation signatures in genomic regions that control maturity, and novel candidate loci for environmental adaptation demonstrating the potential of diversity panels for studying environmental adaptation. Objective-driven core collections increase germplasm utilization for abiotic adaptation by breeding for a rapidly changing climate, or <em>de novo</em> adaptation of crop species to expand cultivation ranges.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-4">Many crop species are cultivated outside their center of domestication (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Diamond 2002</a>). An expansion of the cultivation range requires adaptation to local biotic and abiotic environmental conditions, which can be limited by reduced genetic variation as a result of variation ‘left behind’ in the center of origin due to founder effects (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Tanksley and McCouch 1997</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hyten et al. 2006</a>). Nevertheless, numerous crops were successfully adapted to novel environments by natural and human selection that contributed to the rapid expansion of farming beyond centers of domestication (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Harlan 1992</a>). In current plant breeding programs, reduced levels of genetic variation in elite breeding populations limit breeding progress and the adaptation to rapidly changing environmental conditions. Plant genetic resources stored in large <em>ex situ</em> germplasm collections are an important source of novel, potentially useful genetic variation to achieve these goals (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wang et al. 2017a</a>).</p><p id="p-5">The efficient utilization of plant genetic resources by breeders and researchers is frequently limited by incomplete phenotypic data for traits of interest, which makes the targeted selection of putative useful accessions from large germplasm collections very challenging. The <em>Focused Identification of Germplasm Strategy</em> (FIGS) promises a solution to this phenotyping bottleneck for traits associated with environmental adaptation. FIGS is based on the premise that native landraces evolved during centuries of cultivation in response to local eco-geographic conditions which led to local adaptation to biotic and abiotic conditions. Genomic footprints of this adaptation should be detectable if environmental parameters describing plant germplasm collection sites are used as selection criteria to ‘focus in’ on a subset of accessions that potentially harbor phenotypic variation in a target trait. The comprehensive phenotypic and genomic evaluation of focused subsets is more resource efficient than of large collections (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Street et al. 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sanders et al. 2013</a>). FIGS has been successfully used in a number of crops including <em>Hordeum vulgare</em> for agro-morphological characteristics and time of heading and maturity (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Endresen 2010</a>), <em>Triticum aestivum</em> for resistances to powdery mildew (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bhullar et al. 2009</a>) and stem rust (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bari et al. 2012</a>), and <em>Vicia faba</em> for drought tolerance (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Khazaei et al. 2013</a>). A second tool in the characterization of germplasm collections are core collections. They are small subsets of larger collections that maximize genetic and phenotypic diversity (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Frankel 1984</a>). Similar to trait-focused subsets, core collections provide a resource efficient means for further evaluation and utilization and have been established for many crops based on passport information, agronomic and morphological data as well as genetic diversity to serve varying objectives (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Odong et al. 2013</a>).</p><p id="p-6">The soybean (<em>Glycine max</em>) is a typical example of a crop whose cultivation expanded outside its center of origin. It was originally domesticated in China and is still the most important legume crop throughout East Asia until today. It was first introduced to Europe and the US in the 18th century. During the 20th century it became a major crop in the Americas, and is currently the most widely cultivated legume crop of the world. In Central Europe (CEU), the area of soybean cultivation has been constantly increasing over the last decade and has revived the interest in breeding improved varieties for this production environment. This development is driven by the motivation to reduce dependencies from soy imports, to re-establish legumes in crop rotations for a more sustainable agriculture, and to mitigate the consequences of climate change by cultivating thermotolerant crops (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">BMEL 2016</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">European Soya Declaration 2017</a>).</p><p id="p-7">The adaptation of soybean to Central European production environments is an ongoing effort. So far, breeding research mainly focused on characterizing adaptive phenotypic variation of elite breeding germplasm by investigating the genetic basis of time to flowering and maturity (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kurasch et al. 2017</a>). A portfolio of CEU cultivars that are classified into maturity groups (MGs) exists. In analogy to the system used in Northern America, MGs express the suitability of a variety for cultivation within a certain latitudinal range to ensure a full utilization of the vegetation period and timely maturation. Most cultivars suitable for production in Central Europe belong to the earliest MGs <em>000-0</em> and are mainly derived from North American material (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hahn and Würschum 2014</a>). This narrow genetic basis (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gizlice et al. 1994</a>) can limit long-term breeding progress and the utilization of genetic resources may alleviate this situation.</p><p id="p-8">In addition to photoperiod, temperature and daily irradiation influence soybean development. Cold tolerance ensures that the vegetation period in high-latitude cold regions is fully exploited (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Kurasch et al. 2017</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Balko et al. 2014</a>), but the trait has received little attention in soybean research so far. Flowering is the most susceptible developmental stage for low temperature, which causes flower and pod abscissions (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gass et al. 1996</a>). Poor growth in early development and insufficient grain filling during pod development also reduce grain yield (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Littlejohns and Tanner 1976</a>). Phenotyping for cold tolerance, especially at flowering stage is laborious and a reliable exposure to cold stress requires controlled conditions, which limited the diversity of germplasm that has been evaluated for this trait (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Funatsuki et al. 2004</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cober et al. 2013</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Balko et al. 2014</a>). Previous studies indicated a polygenic basis of cold tolerance and a potential overlap with the <em>E</em> series loci (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Funatsuki et al. 2005</a>) that are responsible for adaptation to photoperiod and soybean diffusion across latitudes (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Jiang et al. 2014</a>). Central European varieties exhibit variation in cold tolerance (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Balko et al. 2014</a>) and are used for improving cold tolerance in breeding of varieties for Northern Japan (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Yamaguchi et al. 2015</a>). Additional useful variation for cold tolerance and adaptation to high-latitude regions may be present in Asian landraces that are conserved in <em>ex situ</em> germplasm collections.</p><p id="p-9">In this study, we present the development of two core collections of the USDA Soybean Germplasm Collection (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Nelson 2011</a>) with a focus on environmental adaptation to high-latitude cold regions to support the breeding of varieties for cultivation in Central Europe. Following the FIGS approach, we used environmental data characterizing the geographic origin of landraces and other germplasm in combination with current and future environmental conditions of the Central European <em>Target Population of Environments</em>. First, a precore collection consisting of accessions potentially adapted to CEU was formed based on environmental data. A scan for genetic differentiation between precore accessions and non-selected germplasm identified selection signals in genomic regions known to be involved in environmental adaptation and thus confirmed that FIGS enriched the precore for adaptive genetic variation. From the environmentally defined precore, we constructed genetically diverse, but substantially smaller core collections based on marker information. These may be used by researchers and breeders for comprehensive phenotypic characterization to facilitate trait discovery for Central European production environments. We show that a combination of FIGS and core collection methodology to define core subsets driven by targeted objectives will likely contribute to a more rapid and efficient utilization of soybean genetic resources.</p></div><div class="section" id="sec-2"><h2 class="">MATERIALS &amp; METHODS</h2><div id="sec-3" class="subsection"><h3>Plant Material and the Donor Population of Environments</h3><p id="p-10">The <em>G. max</em> accessions originated from the USDA Soybean Germplasm Collection and only the <em>Introduced G. max</em> sub-collection (<em>N</em> &gt; 17, 000) was considered, because it includes landraces (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Nelson 2011</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Song et al. 2015</a>). Out of 6,180 accessions with georeferences we retained 5,886 Asian accessions by excluding material collected west of 60°E to remove accessions outside the original soybean cultivation range. The collection sites of the remaining accessions constitute the <em>Donor Population of Environments</em> (DPE) of Asian landraces, which may include potentially useful germplasm for CEU soybean breeding.</p></div><div id="sec-4" class="subsection"><h3>Target Population of Environments</h3><p id="p-11">The <em>Target Population of Environments</em> (TPE) is the set of environments to which germplasm needs to be adapted for a successful cultivation. Our TPE consists of the soybean production environments in Central Europe (CEU) and includes Germany, Austria, Switzerland, Poland, Czech Republic and Slovakia. To characterize the TPE, we georeferenced locations that previously hosted soybean variety trials (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Recknagel 2015</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Kurasch et al. 2017</a>). These served as proxies for the geographic extent of soybean cultivation in CEU since no detailed records regarding soybean cultivation exist for this region.</p></div><div id="sec-5" class="subsection"><h3>Environmental Data</h3><p id="p-12">Based on the georeferences of the curated collection, environmental data was retrieved from the <kbd>WorldClim</kbd> (version 2) database (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fick and Hijmans 2017</a>) at a resolution of 30 arc-seconds (≈ 1 km<sup>2</sup>). We used variables that are informative for the soybean cropping season in the northern hemisphere which lasts from May to September. Additionally, we modified the bioclimatic variables to render them informative for the soybean cropping season: Instead of the <em>annual mean temperature</em> (<kbd>BIO1</kbd>) we computed the <em>seasonal mean temperature</em> as the average mean temperature from May to September. Likewise we proceeded with <kbd>BIO2</kbd> to <kbd>BIO4</kbd>, <kbd>BIO6</kbd>, <kbd>BIO7</kbd> and <kbd>BIO12</kbd> to <kbd>BIO15</kbd>. <kbd>BIO8</kbd> to <kbd>BIO11</kbd> (<em>mean temperature of wettest / driest / warmest / coldest quarter</em>) were replaced by <em>mean temperatures of wettest / driest / warmest / coldest month</em> between May and September. <kbd>BIO16</kbd> to <kbd>BIO19</kbd> (<em>precipitation of wettest / driest / warmest / coldest quarter</em>) were replaced by monthly substitutes. An overview of the data used is provided in Table S1.</p><p id="p-13">We also calculated monthly sums of <em>Crop Heat Units</em> (CHU) to quantify the accumulation of temperature over the growing season. CHUs are commonly used in Canada to rate the regional suitability of warm-season crop varieties with differing thermal requirements (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bootsma et al. 2007</a>) and were also adopted by Central European soybean growers. Average daily CHU were computed from monthly averages of daily maximum and minimum temperatures according to <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Brown and Bootsma (1993)</a>:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/graphic-1.gif"/></noscript></span>

</span>
</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/graphic-2.gif"/></noscript></span>

</span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/graphic-3.gif"/></noscript></span>

</span><p id="p-14">where <em>T<sub>max</sub></em> and <em>T<sub>min</sub></em> refer to the monthly averages of daily maximum and minimum temperatures, respectively. Negative <em>Y<sub>max</sub></em> and <em>Y<sub>min</sub></em> values were set to zero. A similar dataset was assembled for the TPE considering climate data for both current and future conditions. Climate projections for the year 2070 were retrieved from <kbd>WorldClim</kbd> version 1.4 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hijmans et al. 2005</a>) for the greenhouse gas scenario RCP8.5 as modeled by the <em>Community Climate System Model</em> (CCSM4.0). Available projections included average monthly climate data for minimum and maximum temperature, precipitation and the bioclimatic variables, and were used to compute derived variables as outlined above. Environmental qualities without projections for the year 2070 were substituted by their current equivalents to preserve the dimensional consistency of the datasets. Latitude information for the collection sites was included as additional environmental variable to account for the rather narrow adaptation of soybean germplasm to photoperiod.</p></div><div id="sec-6" class="subsection"><h3>Environmental Characterization and Selection of Precore Accessions</h3><p id="p-15">Principal component analysis (PCA) was performed to summarize the high dimensional datasets of the DPE and the TPE separately using the <kbd>dudi.pca()</kbd> function from <kbd>R ade4</kbd> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dray and Dufour 2007</a>). Variables were mean centered and normalized. Correlations (i.e. the loadings) and squared loadings between the original variables and the principal components were used to quantify the amount of shared information and the contribution of single variables to the components. Environmental data for the TPE (current and projected for 2070 climate) was used to project Central European soybean growing environments as illustrative observations into the DPE’s multi-environmental space. The position of a collection site in the multivariate space was then used as ‘environmental profile’ associated with accessions of corresponding descent. Accessions with an environmental origin similar to environments of the TPE along the first two principal components were subsequently included in the precore to form a group of promising accessions with potential abiotic adaptation to the TPE.</p></div><div id="sec-7" class="subsection"><h3>Genotypic Data</h3><p id="p-16">The USDA Soybean Germplasm Collection has been previously genotyped with the SoySNP50K array (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Song et al. 2013</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Song et al. 2015</a>) and genotypes were downloaded from <em>SoyBase</em> (<a href="https://soybase.org/snps/">https://soybase.org/snps/</a>; May 16, 2017) in the version mapped to the second <em>G. max</em> genome assembly “Glyma.Wm82.a2” (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Schmutz et al. 2010</a>). Non-chromosomal SNPs were removed from the genotype dataset and missing data was imputed using <kbd>BEAGLE</kbd> version 4.1 with default settings (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Browning and Browning 2016</a>).</p></div><div id="sec-8" class="subsection"><h3>Phenotypic Data</h3><p id="p-17">Phenotypic data for the <em>Introduced G. max</em> sub-collection of the USDA Soybean Germplasm collection (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Oliveira et al. 2010</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Nelson 2011</a>) was retrieved from <a href="https://npgsweb.ars-grin.gov/gringlobal/search.aspx">https://npgsweb.ars-grin.gov/gringlobal/search.aspx</a>. Data included qualitative and quantitative trait data like maturity group, 100 seed weight, seed yield, protein and oil contents. We used the quantitative trait data to assess changes in phenotypic performance and variance in the selection of the core collection. Homogeneity of variances in groups was tested using the modified robust Brown-Forsythe Levene-type test based on the absolute deviations from the median as implemented in <kbd>R lawstat</kbd> version 3.2 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hui et al. 2008</a>). Significance of differences of the means between groups was assessed with Welch’s unequal variances t-test implemented in <kbd>R</kbd> version 3.4.3 with Bonferroni correction for multiple testing.</p></div><div id="sec-9" class="subsection"><h3>Core Sampling</h3><p id="p-18">Core sampling is based on reducing the number of available accessions by means of redundancy reduction through penalizing core subset solutions with too many too similar accessions. The large-scale genotyping of germplasm collections revealed the magnitude of identical and highly similar accessions in the USDA Soybean Germplasm Collection (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Song et al. 2015</a>) and other crops.</p><p id="p-19">Core collections were assembled by analysing genotype data with <kbd>R Core Hunter</kbd> version 3.2 (<a href="http://www.corehunter.org/">http://www.corehunter.org/</a>). All accessions that qualified after the environmental characterization and/or had a maturity group rating from 000 to I were considered for core sampling. We used <kbd>Core Hunter</kbd> in default execution mode, i.e. using the advanced parallel tempering search algorithm, and fixed the maximum number of steps without improvement to 100 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Beukelaer et al. 2017a</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beukelaer et al. 2017b</a>) to efficiently select core subsets from the population of all possible cores (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Thachuk et al. 2009</a>). We explored different optimization strategies to identify the approach most suitable to our data and objectives: Sampling objectives regarding (1) optimization (i.e. maximization) of allelic diversity included the parameters expected heterozygosity and Shannon’s diversity index; (2) maximization of genetic distance based on the average entry-to-nearest-entry distance (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Beukelaer et al. 2012</a>) using the Modified Roger’s distance (MR) between accessions; (3) optimization of the auxiliary parameter allele coverage was performed the proportion of alleles retainable in core collections. For the definition of these measures the reader is referred to <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Thachuk et al. (2009)</a>. Core sizes of 5%, 10%, 15% and 20% of the input number of accessions were cross-evaluated for all three optimization objectives. Additionally, the simultaneous optimization of expected heterozygosity and MR was examined with varying weights to estimate the trade-off between both approaches. The measures expected heterozygosity, MR and allele coverage were also assessed for all sets used in this study to evaluate the success of the sampling process.</p><p id="p-20">We also explored the effect of different sampling strategies on the subsequent utilization of the core collection: (1) Cores were sampled without any stratification; (2) they were sampled by means of classic stratification according to maturity, i.e. cores were sampled separately within three groups of accessions corresponding to maturity group ratings 000 to 0, I and II to X. A final core was then obtained by merging the result from these three groups; (3) a more refined 2-fold pseudostratification sampling strategy was devised and consisted in first sampling within accessions of maturity groups 000 to 0 and adopting the result of this first run as fixed in a second and final sampling run from all accessions, and (4) a 3-fold pseudostratification strategy first sampled within maturity groups 000 to 0, too. The second run then was limited to maturity groups 000 to I while fixing the result of the first run. In a third and final run, sampling was performed from all accessions while fixing the result from the second run. The general rationale behind all stratification strategies was primarily to retain accessions in the cores relative to their input group sizes by mitigating the effect of varying levels of genetic similarities within groups on the core sampling process. Table S4 provides a more detailed overview of the four core sampling strategies. For all reported cores the random seed was fixed to one prior to sampling for the sake of reproducibility of the core subset selection. To validate the stability of the results, re-sampling was performed for five additional independent runs with random seeds to rule out convergence effects.</p></div><div id="sec-10" class="subsection"><h3>Screening for Signals of Adaptation</h3><p id="p-21">To identify genomic regions that reflect adaptive genetic differentiation between precore and non-selected accessions we used the basic model of <kbd>BAYPASS</kbd> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gautier 2015</a>) for allele count data. Only marker data for loci with minor allele frequencies ≥0.01 in the <em>Introduced G. max</em> subcollection was considered. <kbd>BAYPASS</kbd> calculates the <em>Xt X</em> statistic that may be seen as a SNP-specific <em>F<sub>ST</sub></em> and accounts for the variance–covariance structure of the groups for population structure correction (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Günther and Coop 2013</a>). Significance of the observed <em>Xt X</em> signals was assessed from allele count data for 1,000 SNPs sampled randomly from a pseudo-observed dataset (POD). The POD was constructed with the R function <kbd>simulate.baypass()</kbd> based on the covariance matrix of population allele frequencies (Ω) and other properties of the original data (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gautier 2015</a>). <kbd>BAYPASS</kbd> was then run on the simulated data to produce an empirical distribution of <em>Xt X</em> estimates and the 99% quantile value was used as threshold to distinguish selection from neutrality.</p></div></div><div class="section" id="sec-11"><h2 class="">RESULTS</h2><div id="sec-12" class="subsection"><h3>Curation of Collection Sites</h3><p id="p-22">Of 6,180 USDA soybean accessions with georeferenced collection sites and <kbd>WorldClim</kbd> data, 5,886 fell within our definition of the DPE. These originated from 699 unique collection sites, constituting an average contribution of ≈8 accessions per site. The distribution of accession numbers per collection site (Fig. S1) revealed several locations that contributed a large number (up to 619) of accessions to the collection, suggesting that multiple accessions from a region were aggregated into a single georeference. To purge the data set of accessions with unreliable geographic information, we checked locations with ≥20 accessions and found that georeferences were partly assigned based on provenance from cities, districts or even provinces (<a href="https://npgsweb.ars-grin.gov/gringlobal/search.aspx?">https://npgsweb.ars-grin.gov/gringlobal/search.aspx?</a>). We removed 1,704 accessions (29%) from ten collection sites that were considered too large for a reliable inference of environmental adaptation (Fig. S1).</p></div><div id="sec-13" class="subsection"><h3>Identification of Candidate Accessions for the TPE</h3><p id="p-23">By analyzing climate data from the geographic origin of accessions, we aimed at selecting accessions pre-adapted to cool and high latitude environments of Central Europe, and to identify the selective environmental factors acting during the soybean cropping season from May to September that characterize the DPE and TPE. Most environmental variables were strongly correlated with each other (Fig. S2 - S4). A PCA summarized 71.8% of the total environmental variation among collection sites in the first two principal components (PCs) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). Latitude and temperature based variables were strongly correlated with the first PC, while precipitation based variables related to the first and the second PC (Fig. S5 and S6). The first PC separated cooler from warmer regions of origin. To identify locations with accessions suitable for cultivation in Europe, we projected the European soybean cultivation environments onto the multivariate DPE to relate the TPE to the DPE (Polygons in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>). The projection shows that current CEU environments cluster in a narrow environmental range of less than 3,000 CHUs that includes a small proportion of provenances. Climate projections for 2070 indicate an average increase of more than 500 CHUs during the soybean cropping season in the TPE (Fig. S7). Even these conditions in the TPE exclude the majority of accessions because they originate from warmer regions with &gt;3,500 CHUs. But the overlap of DPE and TPE with respect to temperature and other environmental variables increased and included North-Eastern China, Northern Japan, the Himalayas and Russia. The latter regions also contributed accessions from areas that appear to be highly unfavorable for soybean cultivation. To select potentially suitable accessions for cultivation in CEU, we included 123 environments with a position of ≥ 5 on the first PC (grey dashed line in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>) that provided 523 accessions with potential beneficial variation for abiotic adaptation to CEU (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis of environmental variables obtained from G. max collection sites to characterize the donor population of environments (DPE). The color of collection sites corresponds to the sum of Crop Heat Units (CHUs) of the soybean cultivation season from May to mid September as a site-specific estimate of available temperatures. Contour lines represent bins of 400 CHUs. Current (grey) and future (white) climatic conditions in Central European soybean growing environments are included as illustrative observations. The environmental scopes of Central European scenarios (today and in 2070) are indicated with polygons." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-334829606" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Principal component analysis of environmental variables obtained from &lt;em&gt;G. max&lt;/em&gt; collection sites to characterize the donor population of environments (DPE). The color of collection sites corresponds to the sum of Crop Heat Units (CHUs) of the soybean cultivation season from May to mid September as a site-specific estimate of available temperatures. Contour lines represent bins of 400 CHUs. Current (grey) and future (white) climatic conditions in Central European soybean growing environments are included as illustrative observations. The environmental scopes of Central European scenarios (today and in 2070) are indicated with polygons.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F1.medium.gif" width="440" height="230"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F1.medium.gif" width="440" height="230"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <p id="p-24" class="first-child">Principal component analysis of environmental variables obtained from <em>G. max</em> collection sites to characterize the donor population of environments (DPE). The color of collection sites corresponds to the sum of Crop Heat Units (CHUs) of the soybean cultivation season from May to mid September as a site-specific estimate of available temperatures. Contour lines represent bins of 400 CHUs. Current (grey) and future (white) climatic conditions in Central European soybean growing environments are included as illustrative observations. The environmental scopes of Central European scenarios (today and in 2070) are indicated with polygons.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Collection sites of G. max germplasm in the Donor Population of Environments: Filled dots represent sites that were identified as donors of candidate accessions for CEU. Vertical lines represent sites which were not selected to contribute germplasm. Colouring represents estimates of available termperatures during the soybean season in Crop Heat Units. The inset shows the Target Population of Environments (current situation) with triangles indicating the extent of soybean cultivation in CEU based on soybean variety testing locations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-334829606" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Collection sites of &lt;em&gt;G. max&lt;/em&gt; germplasm in the &lt;em&gt;Donor Population of Environments&lt;/em&gt;: Filled dots represent sites that were identified as donors of candidate accessions for CEU. Vertical lines represent sites which were not selected to contribute germplasm. Colouring represents estimates of available termperatures during the soybean season in Crop Heat Units. The inset shows the &lt;em&gt;Target Population of Environments&lt;/em&gt; (current situation) with triangles indicating the extent of soybean cultivation in CEU based on soybean variety testing locations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F2.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F2.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <p id="p-25" class="first-child">Collection sites of <em>G. max</em> germplasm in the <em>Donor Population of Environments</em>: Filled dots represent sites that were identified as donors of candidate accessions for CEU. Vertical lines represent sites which were not selected to contribute germplasm. Colouring represents estimates of available termperatures during the soybean season in Crop Heat Units. The inset shows the <em>Target Population of Environments</em> (current situation) with triangles indicating the extent of soybean cultivation in CEU based on soybean variety testing locations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">Only ≈20% of accessions from the USDA <em>Introduced G. max</em> collection were georeferenced and included in the above analysis of environmental variation in the DPE. However, the complete collection was categorized for maturity group, which is a proxy for the photo-thermal requirements of accessions and georeferenced accessions show a strong correlation between CHUs and their maturity group classification (Fig. S11). Accessions from early maturity groups originate from environments that resemble CEU e.g. in terms of temperature while late maturing accessions originate from warmer regions. We also included accessions without georeferences and no environmental data, but were classified as early maturing (maturity groups 000-I) and potentially suitable for cultivation in CEU. By combining accessions selected based on environmental data or maturity group classifications, we constructed an ‘environmental precore collection’ of 3,663 accessions (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Tables 3</a> and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">4</a>).</p></div><div id="sec-14" class="subsection"><h3>Genomic footprints of environmental adaptation</h3><p id="p-27">To test whether soybean accessions are locally adapted to their original cultivation environment, we searched for genomic regions with allele frequency differences between the 3,663 environmental precore accessions and the remaining 13,354 USDA <em>Introduced G. max</em> accessions. The <kbd>BAYPASS</kbd> program identified a total of 52 genomic regions with a stronger genetic differentiation (i.e. higher <em>XtX</em> values) than expected given the population structure (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). These regions likely reflect genetic differentiation due to local adaption because they harbor several well characterized maturity genes of soybean, including <em>E1 - E3</em> on chromosomes 6, 10 and 19, which are three out of four cloned genes known to be major determinants of soybean adaptation to latitudinal zones (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Jiang et al. 2014</a>). On chromosome 19, a strongly differentiated region harbors <em>Dt1</em>, which is an ortholog of the <em>Arabidopsis thaliana TFL1</em> gene. In soybean, this gene controls the agronomically important trait indeterminacy and affects the length of flowering and reproductive period, which impacts plant height and maturity (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Liu et al. 2010</a>). The strongest differentiation occurs in a genomic region that contain the <em>Pdh1</em> and <em>GmFT2a</em> genes. The <em>pdh1</em> allele conveys shattering resistance and has been hypothesized crucial for soybean adaptation to Northern and semi-arid environments as opposed to humid South Asian environments where selection on pod dehiscence was more relaxed (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Funatsuki et al. 2014</a>). <em>GmFT2a</em> is a homolog of <em>Arabidopsis FLOWERING LOCUS T</em> (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Sun et al. 2011</a>) that has been identified as soybean maturity locus <em>E9</em> (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zhao et al. 2016</a>). Additional significantly differentiated genomic regions harbor genes for which a role in abiotic adaptation of soybean has been demonstrated or postulated (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Tab. 1</a> and S2). Phenotypic traits controlled by these genes include adaptation to photoperiod, heat, drought and cold stress. The limited marker density of the SoySNP50K array did not allow a fine-scaled mapping of differentiation signals, but in summary provides evidence that the environmental precore collection reflects such an adaptation. It also illustrates the possibility for mining the precore accessions for further traits that might not yet have been introduced into CEU breeding but could benefit soybean cultivation in cold and high-latitude environments. We used loci known to control environmental adaptation (i.e. <em>Dt1</em>, <em>E1-E3</em>, <em>E9</em> and <em>Pdh1</em>) to test whether genetic differentiation caused by local adaptation is better identified by environmental parameters or maturity group ratings. In general, genome scans comparing early maturing accessions (MG 000 to I) to late material (MG II to X) and comparing accessions of CEU-like origin to accessions of non CEU-like origin produced very similar results (Fig. S16). One exception was the <em>E1</em> locus region that exhibited a weaker differentiation signal in the <kbd>BAYPASS</kbd> run with accessions grouped solely based on environmental data as accessions of CEU-like origin were not necessarily of early maturity. This may indicate imprecise or incorrect georeferencing, which puts late maturing accessions in the wrong environments. On the other hand, these accessions might have acquired unique adaptation strategies that would not be identified by solely focusing on early maturing accessions. Therefore we decided to retain the respective accessions in the precore.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide differentiation between precore and non-candidate accessions estimated with BAYPASS. Significantly differentiated regions are located above the dotted horizontal line which represents the POD 99% quantile of XtX values. Labels indicate genes that are located within significantly differentiated regions and known or hypothesized to be involved in abiotic adaptation and which are located within significantly differentiated regions. Chromosome-level plots are provided in Fig. S12 - S15. Candidate genes and their function are listed in Tab. 1 and Tab. S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-334829606" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide differentiation between precore and non-candidate accessions estimated with &lt;kbd&gt;BAYPASS&lt;/kbd&gt;. Significantly differentiated regions are located above the dotted horizontal line which represents the POD 99% quantile of &lt;em&gt;XtX&lt;/em&gt; values. Labels indicate genes that are located within significantly differentiated regions and known or hypothesized to be involved in abiotic adaptation and which are located within significantly differentiated regions. Chromosome-level plots are provided in Fig. S12 - S15. Candidate genes and their function are listed in Tab. 1 and Tab. S2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F3.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F3.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 3.</span> <p id="p-28" class="first-child">Genome-wide differentiation between precore and non-candidate accessions estimated with <kbd>BAYPASS</kbd>. Significantly differentiated regions are located above the dotted horizontal line which represents the POD 99% quantile of <em>XtX</em> values. Labels indicate genes that are located within significantly differentiated regions and known or hypothesized to be involved in abiotic adaptation and which are located within significantly differentiated regions. Chromosome-level plots are provided in Fig. S12 - S15. Candidate genes and their function are listed in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Tab. 1</a> and Tab. S2.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1005121/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1005121/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1005121" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 1.</span> <p id="p-29" class="first-child">Overview of genomic regions significantly differentiated between the environmental precore and remaining accessions of the USDA <em>Introduced G. max</em> collection with proximity to known or hypothetical genes for abiotic adaptation. Gene functions are provided in Tab. S2.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Construction of core collections</h3><p id="p-30">Since the environmentally defined precore comprised 3,663 accessions, we constructed core collections comprising 5% and 10% of the precore accessions to obtain manageable subsets for further evaluation by breeders and researchers.</p><div id="sec-16" class="subsection"><h4>Evaluation of Core Subset Optimization Strategies</h4><p id="p-31">We compared strategies that maximized allelic diversity within core collection accessions or maximized genetic distance between core entries, as well as a joint maximization of both parameters. As shown before (<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Thachuk et al. 2009</a>), a combination of both parameters revealed a trade-off because the maximization of one parameter reduced the other (Fig. S17). Therefore, maximizing each parameter separately is the best approach for a given sampling objective (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic distance and allelic diversity preserved in cores of varying size with varying optimization objectives. Core size &#x2018;100&#x2019; denotes the 3,663 precore accessions. Values on the y-axis refer to the respective facet label and all variables are defined in [0,1]. Core subset optimization based on Modified Rogers distance was most effective in forming cores consisting of distinct accessions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-334829606" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic distance and allelic diversity preserved in cores of varying size with varying optimization objectives. Core size &#x2018;100&#x2019; denotes the 3,663 precore accessions. Values on the y-axis refer to the respective facet label and all variables are defined in [0,1]. Core subset optimization based on Modified Rogers distance was most effective in forming cores consisting of distinct accessions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F4.medium.gif" width="440" height="149"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F4.medium.gif" width="440" height="149"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 4.</span> <p id="p-32" class="first-child">Genetic distance and allelic diversity preserved in cores of varying size with varying optimization objectives. Core size ‘100’ denotes the 3,663 precore accessions. Values on the y-axis refer to the respective facet label and all variables are defined in [0,1]. Core subset optimization based on Modified Rogers distance was most effective in forming cores consisting of distinct accessions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Importantly, both approaches retained &gt;99% of SNP alleles in all core collections. The loss of &lt;1% of alleles resulted from the removal of <em>G. soja</em> specific alleles and the environmental stratification (Fig. S18). We compared two measures of allelic diversity, <em>H<sub>exp</sub></em> and Shannon’s diversity index. Both gave very similar results with respect to diversity within and distance between accessions, and were increased in core collections (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>). The genetic distance between accessions increased more strongly in core collections than allelic diversity and this behavior was independent of the optimization strategy. For example, in the most successful scenario the average minimum Modified Rogers distance increased 1.5-fold in the 5% core compared to the precore whereas <em>H<sub>exp</sub></em> differed only marginally (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4</a>). Our estimates of allelic diversity (Table S3) might show an ascertainment bias caused by the design the SoySNP50K array if it includes SNPs that were preselected for high minor allele frequencies to guarantee high informativeness (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Song et al. 2013</a>). This most likely contributed to the modest increase of <em>H<sub>exp</sub></em> in core subset optimization (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Malomane et al. 2018</a>). By contrast, core subset optimization based on Modified Rogers distance purged highly similar accessions from subsets and was effective in forming cores consisting of distinct accessions (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4</a>). We therefore based the core subset optimization exclusively on the maximization of genetic distance between core accessions.</p></div><div id="sec-17" class="subsection"><h4>Evaluation of Core Subset Sampling Strategies</h4><p id="p-34">The construction of core collections resulted in an underrepresentation of early maturing accessions (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a> and S19) because average redundancy levels were higher in early than within late maturing accessions (Fig. S20). Since early maturity is an important trait for CEU soybean breeding we explored sampling strategies based on a stratification by maturity group designation that favored the selection of early accessions. Any stratification prior to optimization represents a trade-off between accepting higher levels of redundancy in exchange for gaining improved representation in the stratification criterion. The best relative representation across all maturity groups was therefore observed for subsets sampled according to a classic stratification strategy (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5A</a>) of independent sampling within three groups of accessions and aggregating selected accessions into a final core (Table S4). But it also resulted in the lowest level of realized genetic distance between accessions (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5B</a>) and reflects that accessions in one subcore can be similar to accessions in another, which reduces the overall diversity of the complete set. To increase genetic distance we adopted two pseudostratification sampling strategies that allowed to coordinate sub-core subset selection in a consecutive, step-wise fashion that ensured the selection of complementary sub-cores (Table S4). Sampling with pseudostratification resulted in cores that in terms of genetic dissimilarity between accessions were comparable to the no stratification benchmark. The 2-fold pseudostratification however failed in ensuring a good level of representation for accessions of maturity group <em>I</em>, presumably because accessions of this group on average are more similar to accessions of maturity groups <em>000 - 0</em> than to later accessions. The 3-fold pseudostratification resulted in a high average minimum genetic distance between accessions in the final set and also maintained acceptable levels of representation over all maturity groups. We therefore used the 3-fold pseudostratification to construct the final core collections with 5% and 10% of individuals from the precore collection.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: Evaluation of different core sampling strategies with regards to the preservation of MG fractions in 5% cores relative to the precore. B: Increase in genetic distance between core accessions measured by Modified Roger&#x2019;s distance. Among different sampling strategies, the 3-fold pseudostratification* showed good results in terms of favouring early maturing material while maintaining high average minimum genetic distance between accessions. *Consecutive sampling within groups of maturity categories 000 - 0, 000 - I, 000 - X with fixation of already picked accessions (Tab. S4)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-334829606" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;strong&gt;A&lt;/strong&gt;: Evaluation of different core sampling strategies with regards to the preservation of MG fractions in 5% cores relative to the precore. &lt;strong&gt;B&lt;/strong&gt;: Increase in genetic distance between core accessions measured by Modified Roger&#x2019;s distance. Among different sampling strategies, the 3-fold pseudostratification&lt;sup&gt;*&lt;/sup&gt; showed good results in terms of favouring early maturing material while maintaining high average minimum genetic distance between accessions. &lt;sup&gt;*&lt;/sup&gt;Consecutive sampling within groups of maturity categories &lt;em&gt;000 - 0&lt;/em&gt;, &lt;em&gt;000 - I&lt;/em&gt;, &lt;em&gt;000 - X&lt;/em&gt; with fixation of already picked accessions (Tab. S4).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F5.medium.gif" width="440" height="192"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 5." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F5.medium.gif" width="440" height="192"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 5.</span> <p id="p-35" class="first-child"><strong>A</strong>: Evaluation of different core sampling strategies with regards to the preservation of MG fractions in 5% cores relative to the precore. <strong>B</strong>: Increase in genetic distance between core accessions measured by Modified Roger’s distance. Among different sampling strategies, the 3-fold pseudostratification<sup>*</sup> showed good results in terms of favouring early maturing material while maintaining high average minimum genetic distance between accessions.</p><p id="p-36"><sup>*</sup>Consecutive sampling within groups of maturity categories <em>000 - 0</em>, <em>000 - I</em>, <em>000 - X</em> with fixation of already picked accessions (Tab. S4).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h4>Further Efects of Core Sampling: Shifts in Germplasm Composition and Allele Frequencies</h4><p id="p-37">The reduction of the number of accessions through our environmental stratification and the selection of core subsets maximized for genetic diversity was naturally accompanied by changes in the geographic and genomic composition of the respective accession population at each stage of the core formation process (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">6</a> and S21). The reduction of genetic redundancy among core entries furthermore resulted in genome wide changes in population-wise allele frequencies (Fig. S22). These mainly affected loci with low minor allele frequencies in the environmental precore by elevating the frequencies of the respective alleles in the cores (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Fig. 8</a>). Changes in MAFs between the 3,663 precore accessions and the core collection entries however never exceeded 0.25 (Fig. 22) and the groups showed a comparable distribution along the first two components of a PCA (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Fig. 7</a>). Also the genetic basis of the presumed environmental adaptation was unaffected by the core formation process and well preserved in the cores as can be observed in a comparison of genetic differentiation between core, precore and non-precore accessions (Fig. S16).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis summarizing the genetic structure in &gt; 17.000 G. max accessions conserved in the Introduced G. max subcollection of the USDA Soybean Germplasm Collection. Each dot represents one accession. A: Accessions with collection site information available in their passport data are highlighted according to temperature supply at origin. B: 3,663 accessions selected for the environmental precore based on environmental data (coloured) and / or early maturity group ratings (white) are highlighted. C: 366 entries of the 10% core are highlighted. D: 183 entries of the 5% core are highlighted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-334829606" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Principal component analysis summarizing the genetic structure in &gt; 17.000 &lt;em&gt;G. max&lt;/em&gt; accessions conserved in the &lt;em&gt;Introduced G. max&lt;/em&gt; subcollection of the USDA Soybean Germplasm Collection. Each dot represents one accession. &lt;strong&gt;A&lt;/strong&gt;: Accessions with collection site information available in their passport data are highlighted according to temperature supply at origin. &lt;strong&gt;B&lt;/strong&gt;: 3,663 accessions selected for the environmental precore based on environmental data (coloured) and / or early maturity group ratings (white) are highlighted. &lt;strong&gt;C&lt;/strong&gt;: 366 entries of the 10% core are highlighted. &lt;strong&gt;D&lt;/strong&gt;: 183 entries of the 5% core are highlighted.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F6.medium.gif" width="440" height="312"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 6." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F6.medium.gif" width="440" height="312"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 6.</span> <p id="p-38" class="first-child">Principal component analysis summarizing the genetic structure in &gt; 17.000 <em>G. max</em> accessions conserved in the <em>Introduced G. max</em> subcollection of the USDA Soybean Germplasm Collection. Each dot represents one accession. <strong>A</strong>: Accessions with collection site information available in their passport data are highlighted according to temperature supply at origin. <strong>B</strong>: 3,663 accessions selected for the environmental precore based on environmental data (coloured) and / or early maturity group ratings (white) are highlighted. <strong>C</strong>: 366 entries of the 10% core are highlighted. <strong>D</strong>: 183 entries of the 5% core are highlighted.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis summarizing the genetic structure among 3,663 precore accessions shows an even spatial distribution of the core entries: core accessions are highlighted according to maturity group ratings. A: 366 entries of the 10% core. B: 183 entries of the 5% core." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-334829606" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Principal component analysis summarizing the genetic structure among 3,663 precore accessions shows an even spatial distribution of the core entries: core accessions are highlighted according to maturity group ratings. &lt;strong&gt;A&lt;/strong&gt;: 366 entries of the 10% core. &lt;strong&gt;B&lt;/strong&gt;: 183 entries of the 5% core.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F7.medium.gif" width="440" height="200"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 7." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F7.medium.gif" width="440" height="200"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 7.</span> <p id="p-39" class="first-child">Principal component analysis summarizing the genetic structure among 3,663 precore accessions shows an even spatial distribution of the core entries: core accessions are highlighted according to maturity group ratings. <strong>A</strong>: 366 entries of the 10% core. <strong>B</strong>: 183 entries of the 5% core.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise comparison of MAFs in the precore versus the MAF change from precore to 5% core: Core subset optimization resulted in genome wide MAF changes through enrichment of low frequency variants in the core. Fig. S22 provides a more detailed overview on MAF changes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-334829606" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pairwise comparison of MAFs in the precore versus the MAF change from precore to 5% core: Core subset optimization resulted in genome wide MAF changes through enrichment of low frequency variants in the core. Fig. S22 provides a more detailed overview on MAF changes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Fig. 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F8.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Fig. 8." src="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F8.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/20/848978/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 8.</span> <p id="p-40" class="first-child">Pairwise comparison of MAFs in the precore versus the MAF change from precore to 5% core: Core subset optimization resulted in genome wide MAF changes through enrichment of low frequency variants in the core. Fig. S22 provides a more detailed overview on MAF changes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>Robustness of Core Sampling</h4><p id="p-41">With 3,663 precore accessions, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/inline-graphic-1.gif"/></noscript></span></span> core subsets with 5% (<em>n</em> = 183) and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/11/20/848978/embed/inline-graphic-2.gif"/></noscript></span></span> accessions are possible. To find the best core, stochastic algorithms to approximate the optimal solution as implemented in <kbd>Core Hunter</kbd> need to be used. Independent runs of these search algorithms may produce different results, but using a fixed seed to define the starting point ensures the reproducibility of the optimization solution as long as the termination criteria are kept constant. All results presented so far regarded core subset sampling solutions derived from optimization runs with a fixed seed of one. We verified the stability of our results from runs with fixed seeds by performing re-sampling with random seeds for five additional runs. Average levels of genetic distances between accessions in these runs were highly similar to the previous runs. The intersection of all 5% cores included the same 82 out of 183 (45%) and 223 out of 366 10%-core accessions (61%). Only a small proportion of accessions (9% / 5% in 5%/10% cores, respectively) was restricted to a single core subset and seemingly interchangeable through genetically similar entries (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Core subsets differing in sampling intensities were comparable on an individual level too, with 134 accessions of the 5% core having been also selected for the 10% core solution.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1005122/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1005122/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1005122" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 2.</span> <p id="p-42" class="first-child">Comparison of core collection sampling alternatives: Sampling intensity, core size, averaged minimum Modified Rogers distance between core accessions sampled with fixed seed, mean and standard deviation of averaged minimum Modified Rogers distance between core accessions of all sampled cores (including cores sampled with random seed), proportion of accessions that jointly occur in all sampled cores, proportion of accessions private to one core.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">We followed <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Odong et al. (2013)</a> and investigated how the formation of core collections depends on the marker set used by randomly dividing our marker set in a training set and an evaluation set of equal size (split-half analysis). The training set was used to sample cores with varying sampling intensities and the average minimum genetic distance between core entries was subsequently evaluated independently based on both marker sets and based on all available markers. Estimates of genetic distance were highly similar in all comparisons of the same sampling intensity and did not differ between marker sets (Δ<em>MR</em> 0.00032, data not shown). This suggests that our SNP marker set was sufficiently dense to warrant for the robust estimation of genetic distance and the presented core solutions can thus be considered stable within the disparities discussed above.</p></div><div id="sec-20" class="subsection"><h4>Evaluation of phenotypic properties</h4><p id="p-44">We assessed changes in major phenotypic characteristics of accession sets throughout the core collection formation process, i.e. from the <em>Introduced G. max</em> subcollection over the environmental precore to the actual core subsets. The largest phenotypic changes were caused by the environmental stratification that removed most accessions of maturity group ≥ <em>II</em> and therefore the majority of the original collection (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). As a consequence, variation in phenotypic traits of the precore and core subsets differed substantially from the <em>Introduced G. max</em> collection while the smaller core collections preserved the phenotypic variation of the precore (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Regarding the average phenotypic performance, seed weight was reduced in the cores compared to the precore while the seed yields did not differ significantly and seed components (protein and oil content) varied partially. A reduction in seed weight may result from the marker-based selection of more diverse exotic accessions instead of more recently adapted material with larger seeds. Cores sampled with random seeds produced similar results (Table S5).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1005134/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1005134/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1005134" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 3.</span> <p id="p-45" class="first-child">Composition of the <em>Introduced G. max</em> sub-collection with regards to maturity group ratings and the shift towards earlier maturity in precore accessions and core entries.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1005130/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1005130/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1005130" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 4.</span> <p id="p-46" class="first-child">Selected phenotypic properties of the <em>Introduced G. max</em> sub-collection, the environmental precore and the core collections. Differences between groups are reported as significant at a p-value &lt; 0.05.</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section" id="sec-21"><h2 class="">DISCUSSION</h2><div id="sec-22" class="subsection"><h3>Creation of Core Collections</h3><p id="p-47">By combining the <em>Focused Identification of Germplasm Strategy</em> (FIGS) with core collection methodology, we identified subsets of genebank accessions from a large soybean germplasm collection (N &gt; 17,000) that are likely pre-adapted to cultivation in Central Europe while simultaneously conserving a high level of genetic diversity. To establish these core collections, we first defined a precore of 3,663 accessions with potential abiotic adaptation to CEU based on environmental data. This precore was subsequently reduced to two core collections of 183 and 366 accessions (5% and 10% of the precore, please consult the supplementary information for further core accession details) by limiting genetic redundancies among core entries. We quantified the success of this approach at each step of the process and found strong evidence for the enrichment of adaptive genetic variation for abiotic adaptation to high-latitude cold regions throughout the genome (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>, S12 to S16, <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tab. 1</a> and Tab. S2).</p></div><div id="sec-23" class="subsection"><h3>Limitations and Evidence for Enrichment of Environmental Adaptation</h3><p id="p-48">By inferring local adaptation from environmental data we followed FIGS methodology that is based on the assumption that landraces adapted to the environment in which they were originally cultivated by natural and artificial selection (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Street et al. 2008</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Sanders et al. 2013</a>). Knowledge about the geographic origin of germplasm is therefore a key parameter in the successful application of FIGS for abiotic traits. For our implementation of FIGS in the USDA Soybean Germplasm Collection, the origin of accessions was approximated by the georeference for the respective collection sites recorded in the passport data. This information was only available for parts of the collection and in many cases seemed to be an approximate reconstruction of the historic collection sites, sometimes with limited accuracy. Inaccurate georeference information lead to false associations of accessions with environmental data and even though we removed the least reliable collection site - origin misspecifications, this certainly led to the inclusion of some non-adapted accessions into the environmental precore. Another limitation concerns the granularity of the environmental data: The spatial resolution and categorical scope of WorldClim data is sufficiently detailed, but the temporal resolution is restricted to monthly averages. Since records on local cropping practices such as the period of cultivation of landraces in their native environment are lacking too, a detailed reconstruction of the agro-ecological conditions that influenced local adaptation was not possible. Missing georeferences for the majority of accessions were even more limiting, which required us to use maturity group assignments of these accessions as sole proxy for environmental adaptation. Although field trials for the maturity group classification were conducted in North America, we consider using these information suitable for this purpose because of the high heritability of this trait (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Xavier et al. 2018</a>).</p><p id="p-49">To test whether the selection of the precore accessions led to an enrichment of abiotic adaptation to CEU environments, we conducted a selection scan for genetic differentiation between precore accessions and non-precore accessions. Although the marker density of the SoySNP50K array was too low to pinpoint differentiation at the level of single genes, we found evidence that the selection based on climate data and maturity group assignments targets genomic regions involved in abiotic adaptation and enriches genetic variants which are advantageous for cultivation in CEU in the pre-core. Highly differentiated regions mainly indicated genomic regions that harbor well-known genes responsible for photoperiodic adaptation in soybean. This result is expected because the selection of adapted accessions was partially based on maturity groups that are conditioned by photoperiod genes. In addition, also new and promising candidate genes for adaptation to abiotic factors like heat, drought and cold stress were highlighted (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Tab. 1</a> and Tab. S2). Most importantly, these signals of adaptation were preserved in the core collections (Fig. S16) which indicates that the genetic variation responsible for this presumed adaptation can be introgressed from core accessions into CEU elite breeding germplasm.</p></div><div id="sec-24" class="subsection"><h3>Towards a Characterization of Core Collections</h3><p id="p-50">To unlock the full potential of the core accessions, long term dedication of breeders and researchers for the detailed genotypic and phenotypic characterization of the cores will be necessary. A list of core entries is included as supplementary information for further reference. Environmental adaptation is complex with possibly manifold ways to adapt to the same stress which in a real world scenario will not be present as a single factor but as a combination of multiple factors. Therefore, multi-location field trials will be necessary to accurately estimate the level of abiotic adaptation to CEU environments, complemented by trials with controlled conditions in which the most promising accessions can be confronted with selected stresses. It should be noted that no accession is likely to outcompete current CEU varieties per se, but may contribute to improve breeding germplasm. Detailed phenotyping will identify beneficial variation in landraces for use in soybean prebreeding. The increasing availability of high-throughput aerial and field-based phenomics technologies holds great potential in this respect (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Furbank and Tester 2011</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Araus and Cairns 2014</a>).</p><p id="p-51">The efficient introduction of beneficial abiotic adaptation into breeding germplasm via marker assisted selection or genome editing requires the identification of causal genetic variants. The allele frequency of causal variants is a major determinant for the detection power in association-mapping panels (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Spencer et al. 2009</a>) and variants with MAF below 5% are usually not considered because of their low statistical power. Our optimization strategy for the selection of core accessions resulted in elevated MAF for rare alleles (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Fig. 8</a>), which will aid in the discovery of rare traits using modern phenotyping approaches and provides an increased power to map causal variants despite the fairly small core collection sizes. Accessions that harbor favorable alleles can be subsequently included in large multiparent populations for genetic fine-mapping and gene identification (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cockram and Mackay 2018</a>).</p></div><div id="sec-25" class="subsection"><h3>Core Collections for a Future Climate</h3><p id="p-52">The time span required for the development of new varieties requires long-term planning in an era of climate change. We therefore selected precore accessions not only using current climate parameters of the TPE, but also considered climate projections for the year 2070, in which lower precipitation and higher temperatures during the soybean cropping season are expected for CEU (Fig. S7 and S8). These projections suggest that future varieties may not require cold adaptation, but will have to be highly drought tolerant. It is reasonable to argue that both types of adaptation will remain relevant because climate estimates are based on long-term averages, whereas short-term extreme events during the cropping season are frequently limiting crop yields. The future climate is expected to become more volatile and extreme events more frequent, which will include low temperatures in the early and high temperatures in the later growing season. The improvement of crop abiotic adaptation is therefore pivotal to equip agricultural systems with crop varieties for the future. The novel combination of FIGS and core collection methodology is a promising strategy to assemble relevant objective driven core collections and to increase the efficiency of germplasm characterization. Both will aid the identification of alleles for abiotic stress tolerance in order to complement modern breeding germplasm. Upon discovery, targeted crosses and combinations of marker-assisted selection and speed breeding or genome engineering can be employed to incorporate beneficial alleles into modern backgrounds in real-time (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Varshney et al. 2018</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Watson et al. 2018</a>). Genomic selection has emerged as a strategy to predict complex traits also in genebank populations (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Jarquin et al. 2016</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Schnable et al. 2016</a>), but its successful application is challenging in genetically diverse germplasm collections that are frequently lacking phenotypic characterization data for traits of interest (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Langridge and Waugh 2019</a>). Here, upon their sufficient characterization, objective driven core collections from FIGS selected germplasm groups have the potential to complement prediction efforts in genebank germplasm for abiotic adaptation by serving as training and validation populations.</p></div></div><div class="section" id="sec-27"><h2 class="">AUTHOR CONTRIBUTION STATEMENT</h2><p id="p-53">MH and KS designed the study, MH analyzed the data and wrote the first version of the manuscript. MH and KS wrote the final version of the manuscript.</p></div><div class="section" id="sec-28"><h2 class="">CONFLICT OF INTEREST STATEMENT</h2><p id="p-55">On behalf of all authors, the corresponding author states that there is no conflict of interest.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGEMENTS</h2><p id="p-54">The authors thank the United States Department of Agriculture (USDA) and the USDA Soybean Germplasm Collection for their open data and free distribution of germplasm policies that made this work possible. This work was funded by BMEL Project Sojagenopath (FKZ 2814EPS012).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><strong>Funding source:</strong> BMEL Grant number FKZ 2814EPS012</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.1" data-doi="10.1016/j.tplants.2013.09.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Araus</span>,  <span class="cit-name-given-names">J. L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Cairns</span>,  <span class="cit-name-given-names">J. E</span></span>. (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Field high-throughput phenotyping: The new crop breeding frontier</span>.” <abbr class="cit-jnl-abbrev">Trends in Plant Science</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BPlant%2BScience%26rft.volume%253D19%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2013.09.008%26rft_id%253Dinfo%253Apmid%252F24139902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2013.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24139902&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330013700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Balko</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">V.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ordon</span>,  <span class="cit-name-given-names">F</span></span>. (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Chilling tolerance in soybeans (<em>Glycine max</em>) – a prerequisite for soybean cultivation in Germany</span>.” <abbr class="cit-jnl-abbrev">Journal of Cultivated Plants</abbr>, <span class="cit-vol">66</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">378</span>–<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCultivated%2BPlants%26rft.volume%253D66%26rft.spage%253D378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.3" data-doi="10.1007/s10722-011-9775-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bari</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Street</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Endresen</span>,  <span class="cit-name-given-names">D. T. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Pauw</span>,  <span class="cit-name-given-names">E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Amri</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Focused identification of germplasm strategy (FIGS) detects wheat stem rust resistance linked to environmental variables</span>.” <abbr class="cit-jnl-abbrev">Genetic Resources and Crop Evolution</abbr>, <span class="cit-vol">59</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">1465</span>–<span class="cit-lpage">1481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2BResources%2Band%2BCrop%2BEvolution%26rft.volume%253D59%26rft.spage%253D1465%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-011-9775-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-011-9775-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.4" data-doi="10.1093/jxb/erx184"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bemer</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Mourik</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muiño</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrándiz</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaufmann</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Angenent</span>,  <span class="cit-name-given-names">G. C.</span></span> (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title"><em>FRUITFULL</em> controls <em>SAUR10</em> expression and regulates Arabidopsis growth and architecture</span>.” <abbr class="cit-jnl-abbrev">Journal of Experimental Botany</abbr>, <span class="cit-vol">68</span>(<span class="cit-issue">13</span>), <span class="cit-fpage">3391</span>–<span class="cit-lpage">3403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBotany%26rft.volume%253D68%26rft.spage%253D3391%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferx184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erx184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.5" data-doi="10.2135/cropsci1971.0011183X001100020022x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bernard</span>,  <span class="cit-name-given-names">R. L</span></span>. (<span class="cit-pub-date">1971</span>). “<span class="cit-article-title">Two Major Genes for Time of Flowering and Maturity in Soybeans</span>.” <abbr class="cit-jnl-abbrev">Crop Science</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">242</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernard%26rft.auinit1%253DR.%2BL.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D242%26rft.epage%253D244%26rft.atitle%253DTwo%2BMajor%2BGenes%2Bfor%2BTime%2Bof%2BFlowering%2Band%2BMaturity%2Bin%2BSoybeans%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1971.0011183X001100020022x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1971.0011183X001100020022x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971J140000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beukelaer</span>,  <span class="cit-name-given-names">H. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davenport</span>,  <span class="cit-name-given-names">G. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Meyer</span>,  <span class="cit-name-given-names">G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fack</span>,  <span class="cit-name-given-names">V.</span></span> (<span class="cit-pub-date">2017a</span>). “<span class="cit-article-title">JAMES: An object-oriented Java framework for discrete optimization using local search metaheuristics</span>.” <abbr class="cit-jnl-abbrev">Software - Practice and Experience</abbr>, <span class="cit-vol">47</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">921</span>–<span class="cit-lpage">938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSoftware%2B-%2BPractice%2Band%2BExperience%26rft.volume%253D47%26rft.spage%253D921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beukelaer</span>,  <span class="cit-name-given-names">H. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davenport</span>,  <span class="cit-name-given-names">G. F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fack</span>,  <span class="cit-name-given-names">V.</span></span> (<span class="cit-pub-date">2017b</span>). “<abbr class="cit-jnl-abbrev">Multi-Purpose Core Subset Selection</abbr>, &lt;<a href="http://www.corehunter.org">www.corehunter.org</a>&gt;.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beukelaer</span>,  <span class="cit-name-given-names">H. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Smýkal</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davenport</span>,  <span class="cit-name-given-names">G. F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fack</span>,  <span class="cit-name-given-names">V.</span></span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Core Hunter II: fast core subset selection based on multiple genetic diversity measures using Mixed Replica search</span>.” <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DDe%2BBeukelaer%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D312%26rft.epage%253D312%26rft.atitle%253DCore%2BHunter%2BII%253A%2BFast%2Bcore%2Bsubset%2Bselection%2Bbased%2Bon%2Bmultiple%2Bgenetic%2Bdiversity%2Bmeasures%2Busing%2BMixed%2BReplica%2BSearch.%26rft_id%253Dinfo%253Apmid%252F23174036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23174036&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.9" data-doi="10.1073/pnas.0904152106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bhullar</span>,  <span class="cit-name-given-names">N. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Street</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yahiaoui</span>,  <span class="cit-name-given-names">N.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Keller</span>,  <span class="cit-name-given-names">B</span></span>. (<span class="cit-pub-date">2009</span>). “<span class="cit-article-title">Unlocking wheat genetic resources for the molecular identification of previously undescribed functional alleles at the <em>Pm3</em> resistance locus</span>.” <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences of the United States of America</abbr>, <span class="cit-vol">106</span>(<span class="cit-issue">23</span>), <span class="cit-fpage">9519</span>–<span class="cit-lpage">9524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%2Bof%2Bthe%2BUnited%2BStates%2Bof%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904152106%26rft_id%253Dinfo%253Apmid%252F19470492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzIzLzk1MTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8yMC84NDg5NzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.10"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">BMEL</span> (<span class="cit-pub-date">2016</span>). “<abbr class="cit-jnl-abbrev">Ackerbohne, Erbse &amp; Co.</abbr>, &lt;<a href="https://www.bmel.de/SharedDocs/Downloads/Broschueren/EiweisspflanzenstrategieBMEL.pdf?__blob=publicationFile">https://www.bmel.de/SharedDocs/Downloads/Broschueren/EiweisspflanzenstrategieBMEL.pdf?{_}{_}blob=publicationFile</a>&gt;.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bootsma</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKenney</span>,  <span class="cit-name-given-names">D. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Papadopol</span>,  <span class="cit-name-given-names">P</span></span>. (<span class="cit-pub-date">2007</span>). “<span class="cit-article-title">A re-evaluation of crop heat units in the maritime provinces of Canada</span>.” <abbr class="cit-jnl-abbrev">Canadian Journal of Plant Science</abbr>, <span class="cit-vol">87</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">281</span>–<span class="cit-lpage">287</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2BJournal%2Bof%2BPlant%2BScience%26rft.volume%253D87%26rft.spage%253D281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bootsma</span>,  <span class="cit-name-given-names">A</span></span>. (<span class="cit-pub-date">1993</span>). “<span class="cit-article-title">Crop Heat Units for Corn and Other Warm Season Crops in Ontario</span>.” <abbr class="cit-jnl-abbrev">Report No. Factsheet No. 93-119, Ministry of Agriculture and Food Ontario</abbr>, &lt;<a href="https://www.sojafoerderring.de/wp-content/uploads/2014/02/Berechnung-CHU-Uni-Guelph-Ontario.pdf">https://www.sojafoerderring.de/wp-content/uploads/2014/02/Berechnung-CHU-Uni-Guelph-Ontario.pdf</a>&gt;.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.13" data-doi="10.1016/j.ajhg.2015.11.020"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">B. L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">S. R</span></span>. (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">Genotype Imputation with Millions of Reference Samples</span>.” <abbr class="cit-jnl-abbrev">American Journal of Human Genetics</abbr>, <span class="cit-vol">98</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">116</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BHuman%2BGenetics%26rft.volume%253D98%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.11.020%26rft_id%253Dinfo%253Apmid%252F26748515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.11.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26748515&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Buzzell</span>,  <span class="cit-name-given-names">R. I</span></span>. (<span class="cit-pub-date">1971</span>). “<span class="cit-article-title">Inheritance of a soybean flowering response to fluorescent-daylength conditions</span>.” <abbr class="cit-jnl-abbrev">Canadian Journal of Genetics and Cytology</abbr>, <span class="cit-vol">13</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">703</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2BJournal%2Bof%2BGenetics%2Band%2BCytology%26rft.volume%253D13%26rft.spage%253D703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1971L704700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cober</span>,  <span class="cit-name-given-names">E. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Molnar</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rai</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soper</span>,  <span class="cit-name-given-names">J. F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Voldeng</span>,  <span class="cit-name-given-names">H. D</span></span>. (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">Selection for cold tolerance during flowering in short-season soybean</span>.” <abbr class="cit-jnl-abbrev">Crop Science</abbr>, <span class="cit-vol">53</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">1356</span>–<span class="cit-lpage">1365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.volume%253D53%26rft.spage%253D1356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-848978v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cockram</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-chapter-title">Genetic Mapping Populations for Conducting High-Resolution Trait Mapping in Plants</span>.” <span class="cit-source">Plant Genetics and Molecular Biology</span>, <span class="cit-auth"> <span class="cit-name-given-names">R. K.</span> <span class="cit-name-surname">Varshney</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. K.</span> <span class="cit-name-surname">Pandey</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chitikineni</span></span>, eds., <span class="cit-publ-name">Springer International Publishing, Cham</span>, <span class="cit-fpage">109</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.17" data-doi="10.1038/nature01019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Diamond</span>,  <span class="cit-name-given-names">J</span></span>. (<span class="cit-pub-date">2002</span>). “<span class="cit-article-title">Evolution, consequences and future of plant and animal domestication</span>.” <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">418</span>(<span class="cit-issue">6898</span>), <span class="cit-fpage">700</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDiamond%26rft.auinit1%253DJ.%26rft.volume%253D418%26rft.issue%253D6898%26rft.spage%253D700%26rft.epage%253D707%26rft.atitle%253DEvolution%252C%2Bconsequences%2Band%2Bfuture%2Bof%2Bplant%2Band%2Banimal%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01019%26rft_id%253Dinfo%253Apmid%252F12167878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12167878&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177305600058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dray</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Dufour</span>,  <span class="cit-name-given-names">A.-B</span></span>. (<span class="cit-pub-date">2007</span>). “<span class="cit-article-title">The ade4 Package: Implementing the Duality Diagram for Ecologists</span>.” <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">22</span>(<span class="cit-fpage">4</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D22%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">Q. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">W. Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">C. H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">D. Y</span></span>. (<span class="cit-pub-date">2010</span>). “<span class="cit-article-title"><em>GmRFP1</em> encodes a previously unknown RING-type E3 ubiquitin ligase in Soybean (<em>Glycine max</em>)</span>.” <abbr class="cit-jnl-abbrev">Molecular Biology Reports</abbr>, <span class="cit-vol">37</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">685</span>–<span class="cit-lpage">693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Breports%26rft.stitle%253DMol%2BBiol%2BRep%26rft.aulast%253DDu%26rft.auinit1%253DQ.%2BL.%26rft.volume%253D37%26rft.issue%253D2%26rft.spage%253D685%26rft.epage%253D693%26rft.atitle%253DGmRFP1%2Bencodes%2Ba%2Bpreviously%2Bunknown%2BRING-type%2BE3%2Bubiquitin%2Bligase%2Bin%2BSoybean%2B%2528Glycine%2Bmax%2529.%26rft_id%253Dinfo%253Apmid%252F19373563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19373563&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Endresen</span>,  <span class="cit-name-given-names">D. T. F</span></span>. (<span class="cit-pub-date">2010</span>). “<span class="cit-article-title">Predictive association between trait data and ecogeographic data for Nordic barley landraces</span>.” <abbr class="cit-jnl-abbrev">Crop Science</abbr>, <span class="cit-vol">50</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">2418</span>–<span class="cit-lpage">2430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.volume%253D50%26rft.spage%253D2418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.21"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">European Soya Declaration</span> (<span class="cit-pub-date">2017</span>). “<abbr class="cit-jnl-abbrev">Common Declaration of Austria, Croatia, Finland, France, Germany, Greece, Hungary, Italy, Luxemburg, the Netherlands, Poland, Romania, Slovakia and Slovenia European Soya Declaration – Enhancing soya and other legumes cultivation</abbr>, &lt;<a href="https://www.bmel.de/SharedDocs/Downloads/Landwirtschaft/Pflanze/SojaErklaerung.pdf?__blob=publicationFile">https://www.bmel.de/SharedDocs/Downloads/Landwirtschaft/Pflanze/SojaErklaerung.pdf?{_}{_}blob=publicationFile</a>&gt;.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.22" data-doi="https://doi.org/10.1002/joc.5086"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fick</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title">Worldclim 2: New 1-km spatial resolution climate surfaces for global land areas</span>.” <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr>, <span class="cit-vol">37</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">4302</span>–<span class="cit-lpage">4315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D37%26rft.spage%253D4302%26rft_id%253Dinfo%253Adoi%252Fhttps%253A%252F%252Fdoi.org%252F10.1002%252Fjoc.5086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=https://doi.org/10.1002/joc.5086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-848978v1.23"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Arber</span></span>, </li><li><span class="cit-ed"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Llimensee</span></span>, </li><li><span class="cit-ed"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Peacock</span></span>, and </li><li><span class="cit-ed"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Stralinger</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Frankel</span>,  <span class="cit-name-given-names">O</span></span>. (<span class="cit-pub-date">1984</span>). <span class="cit-source">“Genetic Perspectives of Germplasm Conservation.” Genetic Manipulation: Impact on Man and Society</span>, <span class="cit-ed"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Arber</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Llimensee</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Peacock</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Stralinger</span></span>, eds., <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge</span>, <span class="cit-fpage">161</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.24" data-doi="10.1007/s00122-005-0007-2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Funatsuki</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsuba</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Ishimoto</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">2005</span>). “<span class="cit-article-title">Mapping of QTL associated with chilling tolerance during reproductive growth in soybean</span>.” <abbr class="cit-jnl-abbrev">Theoretical and Applied Genetics</abbr>, <span class="cit-vol">111</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFunatsuki%26rft.auinit1%253DH.%26rft.volume%253D111%26rft.issue%253D5%26rft.spage%253D851%26rft.epage%253D861%26rft.atitle%253DMapping%2Bof%2BQTL%2Bassociated%2Bwith%2Bchilling%2Btolerance%2Bduring%2Breproductive%2Bgrowth%2Bin%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0007-2%26rft_id%253Dinfo%253Apmid%252F16059730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0007-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16059730&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232598600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Funatsuki</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Matsuba</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murakami</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">Y</span></span>. (<span class="cit-pub-date">2004</span>). “<span class="cit-article-title">Methods for evaluation of soybean chilling tolerance at the reproductive stage under artificial climatic conditions</span>.” <abbr class="cit-jnl-abbrev">Plant Breeding</abbr>, <span class="cit-vol">123</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">558</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreeding%26rft.volume%253D123%26rft.spage%253D558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.26" data-doi="10.1073/pnas.1417282111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Funatsuki</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suzuki</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hirose</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Inaba</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamada</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hajika</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Komatsu</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katayama</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sayama</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishimoto</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fujino</span>,  <span class="cit-name-given-names">K</span></span>. (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Molecular basis of a shattering resistance boosting global dissemination of soybean</span>.” <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr>, <span class="cit-vol">111</span>(<span class="cit-issue">50</span>), <span class="cit-fpage">17797</span>–<span class="cit-lpage">17802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1417282111%26rft_id%253Dinfo%253Apmid%252F25468966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzUwLzE3Nzk3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjAvODQ4OTc4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.27" data-doi="10.1016/j.tplants.2011.09.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Furbank</span>,  <span class="cit-name-given-names">R. T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tester</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">2011</span>). “<span class="cit-article-title">Phenomics - technologies to relieve the phenotyping bottleneck</span>.” <abbr class="cit-jnl-abbrev">Trends in Plant Science</abbr>, <span class="cit-vol">16</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">635</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DFurbank%26rft.auinit1%253DR.%2BT.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D635%26rft.epage%253D644%26rft.atitle%253DPhenomics--technologies%2Bto%2Brelieve%2Bthe%2Bphenotyping%2Bbottleneck.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2011.09.005%26rft_id%253Dinfo%253Apmid%252F22074787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2011.09.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22074787&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298724900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gass</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schori</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fossati</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Soldati</span>,  <span class="cit-name-given-names">A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stamp</span>,  <span class="cit-name-given-names">P</span></span>. (<span class="cit-pub-date">1996</span>). “<span class="cit-article-title">Cold tolerance of soybean (<em>Glycine max</em> (L.) Merr.) during the reproductive phase</span>.” <abbr class="cit-jnl-abbrev">European Journal of Agronomy</abbr>, <span class="cit-vol">5</span>(<span class="cit-issue">1-2</span>), <span class="cit-fpage">71</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2BJournal%2Bof%2BAgronomy%26rft.volume%253D5%26rft.spage%253D71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.29" data-doi="10.1534/genetics.115.181453"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gautier</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Genome-wide scan for adaptive divergence and association with population-specific covariates</span>.” <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">201</span>(<span class="cit-issue">4</span>), <span class="cit-fpage">1555</span>–<span class="cit-lpage">1579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.181453%26rft_id%253Dinfo%253Apmid%252F26482796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS80LzE1NTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8yMC84NDg5NzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.30" data-doi="10.2135/cropsci1994.0011183X003400050001x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gizlice</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carter</span>,  <span class="cit-name-given-names">T. E.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Burton</span>,  <span class="cit-name-given-names">J. W</span></span>. (<span class="cit-pub-date">1994</span>). “<span class="cit-article-title">Genetic Base for North American Public Soybean Cultivars Released between 1947 and 1988</span>.” <abbr class="cit-jnl-abbrev">Crop Science</abbr>, <span class="cit-vol">34</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">1143</span>–<span class="cit-lpage">1151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DGizlice%26rft.auinit1%253DZ.%26rft.volume%253D34%26rft.issue%253D5%26rft.spage%253D1143%26rft.epage%253D1151%26rft.atitle%253DGenetic%2BBase%2Bfor%2BNorth%2BAmerican%2BPublic%2BSoybean%2BCultivars%2BReleased%2Bbetween%2B1947%2Band%2B1988%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1994.0011183X003400050001x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1994.0011183X003400050001x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994PF01600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">R. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cannon</span>,  <span class="cit-name-given-names">S. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>,  <span class="cit-name-given-names">R. C.</span></span>, and (SoyBase) (<span class="cit-pub-date">2010</span>). “<abbr class="cit-jnl-abbrev">SoyBase, the USDA-ARS soybean genetics and genomics database</abbr>, &lt;<a href="http://soybase.org">http://soybase.org</a>&gt;.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.32" data-doi="10.1534/genetics.113.152462"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Günther</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Coop</span>,  <span class="cit-name-given-names">G</span></span>. (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">Robust identification of local adaptation from allele frequencies</span>.” <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">195</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">205</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152462%26rft_id%253Dinfo%253Apmid%252F23821598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzEvMjA1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjAvODQ4OTc4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">V.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Würschum</span>,  <span class="cit-name-given-names">T</span></span>. (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Molecular genetic characterization of Central European soybean breeding germplasm</span>.” <abbr class="cit-jnl-abbrev">Plant Breeding</abbr>, <span class="cit-vol">133</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">748</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreeding%26rft.volume%253D133%26rft.spage%253D748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harlan</span>,  <span class="cit-name-given-names">J. R</span></span>. (<span class="cit-pub-date">1992</span>). <span class="cit-article-title">“Space, Time, and Variation.”</span> <abbr class="cit-jnl-abbrev">Crops &amp; Man, American Society of Agronomy, Crop Science Society of America, Madison, WI, Chapter 7</abbr>, <span class="cit-fpage">135</span> – <span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.35" data-doi="10.1002/joc.1276"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>,  <span class="cit-name-given-names">S. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parra</span>,  <span class="cit-name-given-names">J. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">P. G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Jarvis</span>,  <span class="cit-name-given-names">A</span></span>. (<span class="cit-pub-date">2005</span>). “<span class="cit-article-title">Very high resolution interpolated climate surfaces for global land areas</span>.” <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr>, <span class="cit-vol">25</span>(<span class="cit-issue">15</span>), <span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D25%26rft.spage%253D1965%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.1276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.1276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234519700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hui</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gel</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Gastwirth</span>,  <span class="cit-name-given-names">J</span></span>. (<span class="cit-pub-date">2008</span>). “<span class="cit-article-title">lawstat: an R package for law, public policy and biostatistics</span>.” <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr>, <span class="cit-vol">28</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D28%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.37" data-doi="10.1073/pnas.0604379103"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hyten</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Choi</span>,  <span class="cit-name-given-names">I.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">R. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Costa</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Specht</span>,  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cregan</span>,  <span class="cit-name-given-names">P. B</span></span>. (<span class="cit-pub-date">2006</span>). “<span class="cit-article-title">Impacts of genetic bottlenecks on soybean genome diversity</span>..” <abbr class="cit-jnl-abbrev">PNAS</abbr>, <span class="cit-vol">103</span>(<span class="cit-issue">45</span>), <span class="cit-fpage">16666</span>–<span class="cit-lpage">16671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0604379103%26rft_id%253Dinfo%253Apmid%252F17068128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzQ1LzE2NjY2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjAvODQ4OTc4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iuchi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koyama</span>,  <span class="cit-name-given-names">H.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title"><em>STOP1</em>, a Cys2/His2 type zinc-finger protein, plays critical role in acid soil tolerance in Arabidopsis</span>.” <abbr class="cit-jnl-abbrev">Plant Signaling &amp; Behavior</abbr>, <span class="cit-vol">3</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">128</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSignaling%2B%2526%2BBehavior%26rft.volume%253D3%26rft.spage%253D128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iwasaki</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kamauchi</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wadahama</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishimoto</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kawada</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Urade</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2009</span>). “<span class="cit-article-title">Molecular cloning and characterization of soybean protein disulfide isomerase family proteins with nonclassic active center motifs</span>.” <abbr class="cit-jnl-abbrev">FEBS Journal</abbr>, <span class="cit-vol">276</span>(<span class="cit-issue">15</span>), <span class="cit-fpage">4130</span>–<span class="cit-lpage">4141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.volume%253D276%26rft.spage%253D4130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Specht</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">A</span></span>. (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">Prospects of Genomic Prediction in the USDA Soybean Germplasm Collection: Historical Data Creates Robust Models for Enhancing Selection of Accessions</span>.” <abbr class="cit-jnl-abbrev">G3</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">2329</span>–<span class="cit-lpage">2341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D6%26rft.spage%253D2329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.41" data-doi="10.1371/journal.pone.0106042"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nan</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yue</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">B</span></span>. (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Allelic combinations of soybean maturity Loci E1, E2, E3 and E4 result in diversity of maturity and adaptation to different latitudes</span>..” <abbr class="cit-jnl-abbrev">PlOS ONE</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">8</span>), <span class="cit-fpage">e106042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlOS%2BONE%26rft.volume%253D9%26rft.spage%253De106042%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0106042%26rft_id%253Dinfo%253Apmid%252F25162675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0106042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25162675&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.42" data-doi="10.1371/journal.pone.0064638"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Khazaei</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Street</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bari</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Stoddard</span>,  <span class="cit-name-given-names">F. L</span></span>. (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">The FIGS (Focused Identification of Germplasm Strategy) approach identifies traits related to drought adaptation in <em>Vicia faba</em> genetic resources</span>.” <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>, <span class="cit-vol">8</span>(<span class="cit-fpage">5</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D8%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0064638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0064638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kongrit</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jisaka</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iwanga</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yokomichi</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Katsube</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishimura</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagaya</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yokota</span>,  <span class="cit-name-given-names">K</span></span>. (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">“Molecular Cloning and Functional Expression of Soybean Allene Oxide Synthases.”</span> <abbr class="cit-jnl-abbrev">Bioscience, Biotechnology and Biochemistry</abbr>, <span class="cit-vol">71</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.stitle%253DBioscience%252C%2BBiotechnology%252C%2Band%2BBiochemistry%26rft.aulast%253DKongrit%26rft.auinit1%253DD.%26rft.volume%253D71%26rft.issue%253D2%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DMolecular%2Bcloning%2Band%2Bfunctional%2Bexpression%2Bof%2Bsoybean%2Ballene%2Boxide%2Bsynthases.%26rft_id%253Dinfo%253Apmid%252F17284840%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17284840&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.44" data-doi="10.1021/jf2033939"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kovinich</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saleem</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arnason</span>,  <span class="cit-name-given-names">J. T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Miki</span>,  <span class="cit-name-given-names">B</span></span>. (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Identification of two anthocyanidin reductase genes and three red-brown soybean accessions with reduced anthocyanidin reductase 1 mRNA, activity, and seed coat proanthocyanidin amounts</span>.” <abbr class="cit-jnl-abbrev">Journal of Agricultural and Food Chemistry</abbr>, <span class="cit-vol">60</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">574</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.stitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.aulast%253DKovinich%26rft.auinit1%253DN.%26rft.volume%253D60%26rft.issue%253D2%26rft.spage%253D574%26rft.epage%253D584%26rft.atitle%253DIdentification%2Bof%2Btwo%2Banthocyanidin%2Breductase%2Bgenes%2Band%2Bthree%2Bred-brown%2Bsoybean%2Baccessions%2Bwith%2Breduced%2Banthocyanidin%2Breductase%2B1%2BmRNA%252C%2Bactivity%252C%2Band%2Bseed%2Bcoat%2Bproanthocyanidin%2Bamounts.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjf2033939%26rft_id%253Dinfo%253Apmid%252F22107112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jf2033939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22107112&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299584600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kurasch</span>,  <span class="cit-name-given-names">A. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leiser</span>,  <span class="cit-name-given-names">W. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vollmann</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schori</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bétrix</span>,  <span class="cit-name-given-names">C.-A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mayr</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Winkler</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mechtler</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aper</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sudaric</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pejic</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sarcevic</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeanson</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balko</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Signor</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miceli</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Strijk</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rietman</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muresanu</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Djordjevic</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pospiöil</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barion</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weigold</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Streng</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krön</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Würschum</span>,  <span class="cit-name-given-names">T</span></span>. (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title">Identification of mega-environments in Europe and effect of allelic variations at maturity <em>E</em> loci on adaptation of European soybean</span>.” <abbr class="cit-jnl-abbrev">Plant, Cell &amp; Environment</abbr>, <span class="cit-vol">0000</span>, <span class="cit-fpage">765</span>–<span class="cit-lpage">778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%2B%2526%2BEnvironment%26rft.volume%253D0000%26rft.spage%253D765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.46" data-doi="10.1038/s41588-018-0340-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Langridge</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Waugh</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Harnessing the potential of germplasm collections</span>.” <abbr class="cit-jnl-abbrev">Nature Genetics</abbr>, <span class="cit-vol">51</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">200</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D51%26rft.spage%253D200%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-018-0340-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-018-0340-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.47" data-doi="10.1007/s00425-008-0731-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">H. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wei</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>,  <span class="cit-name-given-names">Y. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">A. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J. S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S. Y</span></span>. (<span class="cit-pub-date">2008</span>). “<span class="cit-article-title">Soybean <em>GmbZIP44</em>, <em>GmbZIP62</em> and <em>GmbZIP78</em> genes function as negative regulator of ABA signaling and confer salt and freezing tolerance in transgenic Arabidopsis</span>.” <abbr class="cit-jnl-abbrev">Planta</abbr>, <span class="cit-vol">228</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">225</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.stitle%253DPlanta%26rft.aulast%253DLiao%26rft.auinit1%253DY.%26rft.volume%253D228%26rft.issue%253D2%26rft.spage%253D225%26rft.epage%253D240%26rft.atitle%253DSoybean%2BGmbZIP44%252C%2BGmbZIP62%2Band%2BGmbZIP78%2Bgenes%2Bfunction%2Bas%2Bnegative%2Bregulator%2Bof%2BABA%2Bsignaling%2Band%2Bconfer%2Bsalt%2Band%2Bfreezing%2Btolerance%2Bin%2Btransgenic%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-008-0731-3%26rft_id%253Dinfo%253Apmid%252F18365246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-008-0731-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18365246&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256522400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Littlejohns</span>,  <span class="cit-name-given-names">D. a.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tanner</span>,  <span class="cit-name-given-names">J. W.</span></span> (<span class="cit-pub-date">1976</span>). “<span class="cit-article-title">Preliminary Studies on the Cold Tolerance of Soybean Seedlings</span>.” <abbr class="cit-jnl-abbrev">Canadian Journal of Plant Science</abbr>, <span class="cit-vol">56</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">371</span>–<span class="cit-lpage">375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2BJournal%2Bof%2BPlant%2BScience%26rft.volume%253D56%26rft.spage%253D371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.49" data-doi="10.1104/pp.109.150607"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uchiyama</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanazawa</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nagamatsu</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arai</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamada</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitamura</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masuta</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harada</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Abe</span>,  <span class="cit-name-given-names">J</span></span>. (<span class="cit-pub-date">2010</span>). “<span class="cit-article-title">The Soybean Stem Growth Habit Gene <em>Dt1</em> Is an Ortholog of Arabidopsis <em>TERMINAL FLOWER1</em></span>.” <abbr class="cit-jnl-abbrev">Plant Physiology</abbr>, <span class="cit-vol">153</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">198</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.109.150607%26rft_id%253Dinfo%253Apmid%252F20219831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTMvMS8xOTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8yMC84NDg5NzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">Y. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">L. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">P. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X. P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">N. N</span></span>. (<span class="cit-pub-date">2006</span>). “<span class="cit-article-title">Cloning and preliminary characterization of three receptor-like kinase genes in soybean</span>.” <abbr class="cit-jnl-abbrev">Journal of Integrative Plant Biology</abbr>, <span class="cit-vol">48</span>(<span class="cit-issue">11</span>), <span class="cit-fpage">1338</span>–<span class="cit-lpage">1347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BIntegrative%2BPlant%2BBiology%26rft.volume%253D48%26rft.spage%253D1338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.51" data-doi="10.1186/s12864-017-4416-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Malomane</span>,  <span class="cit-name-given-names">D. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reimer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weigend</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weigend</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sharifi</span>,  <span class="cit-name-given-names">A. R.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Simianer</span>,  <span class="cit-name-given-names">H</span></span>. (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Efficiency of different strategies to mitigate ascertainment bias when using SNP panels in diversity studies</span>.” <abbr class="cit-jnl-abbrev">BMC genomics</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%26rft.volume%253D19%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-4416-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-4416-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.52" data-doi="10.1104/pp.124.3.1105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McGonigle</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keeler</span>,  <span class="cit-name-given-names">S. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>,  <span class="cit-name-given-names">S.-M. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koeppe</span>,  <span class="cit-name-given-names">M. K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">O’Keefe</span>,  <span class="cit-name-given-names">D. P</span></span>. (<span class="cit-pub-date">2000</span>). “<span class="cit-article-title">A Genomics Approach to the Comprehensive Analysis of the Glutathione <em>S</em> -Transferase Gene Family in Soybean and Maize</span>.” <abbr class="cit-jnl-abbrev">Plant Physiology</abbr>, <span class="cit-vol">124</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">1105</span>–<span class="cit-lpage">1120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DMcGonigle%26rft.auinit1%253DB.%26rft.volume%253D124%26rft.issue%253D3%26rft.spage%253D1105%26rft.epage%253D1120%26rft.atitle%253DA%2BGenomics%2BApproach%2Bto%2Bthe%2BComprehensive%2BAnalysis%2Bof%2Bthe%2BGlutathione%2BS-Transferase%2BGene%2BFamily%2Bin%2BSoybean%2Band%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.124.3.1105%26rft_id%253Dinfo%253Apmid%252F11080288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTI0LzMvMTEwNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzExLzIwLzg0ODk3OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.53" data-doi="10.1104/pp.112.204875"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mizoi</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ohori</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moriwaki</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kidokoro</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Todaka</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maruyama</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kusakabe</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Os-akabe</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shinozaki</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yamaguchi-Shinozaki</span>,  <span class="cit-name-given-names">K</span></span>. (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title"><em>GmDREB2A;2</em>, a Canonical DEHYDRATION-RESPONSIVE ELEMENT-BINDING PROTEIN2-Type Transcription Fac-tor in Soybean, Is Posttranslationally Regulated and Mediates Dehydration-Responsive Element-Dependent Gene Expression</span>.” <abbr class="cit-jnl-abbrev">Plant Physiology</abbr>, <span class="cit-vol">161</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">346</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.112.204875%26rft_id%253Dinfo%253Apmid%252F23151346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjEvMS8zNDYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMS8yMC84NDg5NzguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">R. L</span></span>. (<span class="cit-pub-date">2011</span>). “<span class="cit-article-title">Managing self-pollinated germplasm collections to maximize utilization</span>.” <abbr class="cit-jnl-abbrev">Plant Genetic Resources</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">01</span>), <span class="cit-fpage">123</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenetic%2BResources%26rft.volume%253D9%26rft.spage%253D123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.55" data-doi="10.1007/s00122-012-1971-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Odong</span>,  <span class="cit-name-given-names">T. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jansen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>,  <span class="cit-name-given-names">F. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">van Hintum</span>,  <span class="cit-name-given-names">T. J.</span></span> (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">Quality of core collections for effective utilisation of genetic resources review, discussion and interpretation</span>.” <abbr class="cit-jnl-abbrev">Theoretical and Applied Genetics</abbr>, <span class="cit-vol">126</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">289</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Band%2BApplied%2BGenetics%26rft.volume%253D126%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1971-y%26rft_id%253Dinfo%253Apmid%252F22983567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1971-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22983567&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>,  <span class="cit-name-given-names">M. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">R. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geraldi</span>,  <span class="cit-name-given-names">I. O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cruz</span>,  <span class="cit-name-given-names">C. D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">de Toledo</span>,  <span class="cit-name-given-names">J. F. F.</span></span> (<span class="cit-pub-date">2010</span>). “<span class="cit-article-title">Establishing a soybean germplasm core collection</span>.” <abbr class="cit-jnl-abbrev">Field Crops Research</abbr>, <span class="cit-vol">119</span>(<span class="cit-issue">2-3</span>), <span class="cit-fpage">277</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BResearch%26rft.volume%253D119%26rft.spage%253D277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.57" data-doi="10.1007/s11103-004-1581-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olsson</span>,  <span class="cit-name-given-names">A. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Engström</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Söderman</span>,  <span class="cit-name-given-names">E</span></span>. (<span class="cit-pub-date">2004</span>). “<span class="cit-article-title">The homeobox genes <em>ATHB12</em> and <em>ATHB7</em> encode potential regulators of growth in response to water deficit in Arabidopsis</span>.” <abbr class="cit-jnl-abbrev">Plant Molecular Biology</abbr>, <span class="cit-vol">55</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">663</span>–<span class="cit-lpage">677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DOlsson%26rft.auinit1%253DA.%2BS.%26rft.volume%253D55%26rft.issue%253D5%26rft.spage%253D663%26rft.epage%253D677%26rft.atitle%253DThe%2Bhomeobox%2Bgenes%2BATHB12%2Band%2BATHB7%2Bencode%2Bpotential%2Bregulators%2Bof%2Bgrowth%2Bin%2Bresponse%2Bto%2Bwater%2Bdeficit%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-004-1581-4%26rft_id%253Dinfo%253Apmid%252F15604708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-004-1581-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15604708&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225690500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">S.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">J.-W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">J.-S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoo</span>,  <span class="cit-name-given-names">S.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">N.-Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S.-K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeong</span>,  <span class="cit-name-given-names">S.-W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seo</span>,  <span class="cit-name-given-names">H. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koh</span>,  <span class="cit-name-given-names">H.-J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeon</span>,  <span class="cit-name-given-names">J.-S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">Y.-I.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Paek</span>,  <span class="cit-name-given-names">N.-C</span></span>. (<span class="cit-pub-date">2007</span>). “<span class="cit-article-title">The Senescence-Induced Staygreen Protein Regulates Chlorophyll Degradation</span>.” <abbr class="cit-jnl-abbrev">the Plant Cell Online</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">1649</span>–<span class="cit-lpage">1664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253Dthe%2BPlant%2BCell%2BOnline%26rft.volume%253D19%26rft.spage%253D1649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Recknagel</span>,  <span class="cit-name-given-names">J. D. S.</span></span> (<span class="cit-pub-date">2015</span>). “<abbr class="cit-jnl-abbrev">Sorten für Deutschland</abbr>, &lt;<a href="https://www.sojafoerderring.de/anbauratgeber/sortenratgeber/deutschland/">https://www.sojafoerderring.de/anbauratgeber/sortenratgeber/deutschland/</a>&gt;.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanders</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bari</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Street</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Devlin</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">A new approach to mining agricultural gene banks – to speed the pace of research innovation for food security. ‘FIGS’ - the Focused Identification of Germplasm Strategy</span>.” <abbr class="cit-jnl-abbrev">Report No. 3, International Center for Agricultural Research in the Dry Areas (ICARDA), Beirut</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>,  <span class="cit-name-given-names">K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Imai</span>,  <span class="cit-name-given-names">R</span></span>. (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Pleiotropic Roles of Cold Shock Domain Proteins in Plants</span>.” <abbr class="cit-jnl-abbrev">Frontiers in Plant Science</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">January</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BPlant%2BScience%26rft.volume%253D2%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.62" data-doi="10.1038/nature08670"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cannon</span>,  <span class="cit-name-given-names">S. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlueter</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitros</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hyten</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thelen</span>,  <span class="cit-name-given-names">J. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hellsten</span>,  <span class="cit-name-given-names">U.</span></span>, <span class="cit-auth"><span class="cit-name-surname">May</span>,  <span class="cit-name-given-names">G. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sakurai</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Umezawa</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bhattacharyya</span>,  <span class="cit-name-given-names">M. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandhu</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valliyodan</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindquist</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peto</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goodstein</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barry</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futrell-Griggs</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abernathy</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gill</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joshi</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Libault</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sethuraman</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.-C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shinozaki</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">H. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wing</span>,  <span class="cit-name-given-names">R. a.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cregan</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Specht</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grimwood</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rokhsar</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stacey</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>,  <span class="cit-name-given-names">R. C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">R. a.</span></span> (<span class="cit-pub-date">2010</span>). “<span class="cit-article-title">Genome sequence of the palaeopolyploid soybean</span>..” <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">463</span>(<span class="cit-issue">7278</span>), <span class="cit-fpage">178</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchmutz%26rft.auinit1%253DJ.%26rft.volume%253D463%26rft.issue%253D7278%26rft.spage%253D178%26rft.epage%253D183%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpalaeopolyploid%2Bsoybean.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08670%26rft_id%253Dinfo%253Apmid%252F20075913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20075913&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273582700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schnable</span>,  <span class="cit-name-given-names">P. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">M. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tesso</span>,  <span class="cit-name-given-names">T. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernardo</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roozeboom</span>,  <span class="cit-name-given-names">K. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>,  <span class="cit-name-given-names">S. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pederson</span>,  <span class="cit-name-given-names">G. A.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">Genomic prediction contributing to a promising global strategy to turbocharge gene banks</span>.” <abbr class="cit-jnl-abbrev">Nature Plants</abbr>, <span class="cit-vol">2</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">16150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BPlants%26rft.volume%253D2%26rft.spage%253D16150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.64" data-doi="10.1186/1471-2164-16-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silva</span>,  <span class="cit-name-given-names">P. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silva</span>,  <span class="cit-name-given-names">J. C. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Caetano</span>,  <span class="cit-name-given-names">H. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Machado</span>,  <span class="cit-name-given-names">J. P. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendes</span>,  <span class="cit-name-given-names">G. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reis</span>,  <span class="cit-name-given-names">P. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brustolini</span>,  <span class="cit-name-given-names">O. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dal-Bianco</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Fontes</span>,  <span class="cit-name-given-names">E. P</span></span>. (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Comprehensive analysis of the endoplasmic reticulum stress response in the soybean genome: Conserved and plant-specific features</span>.” <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-vol">16</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-16-1%26rft_id%253Dinfo%253Apmid%252F25553907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-16-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25553907&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.65" data-doi="10.1016/S0092-8674(00)80841-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Somers</span>,  <span class="cit-name-given-names">D. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schultz</span>,  <span class="cit-name-given-names">T. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milnamow</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kay</span>,  <span class="cit-name-given-names">S. A</span></span>. (<span class="cit-pub-date">2000</span>). “<span class="cit-article-title"><em>ZEITLUPE</em> encodes a novel clock-associated PAS protein from Arabidopsis</span>.” <abbr class="cit-jnl-abbrev">Cell</abbr>, <span class="cit-vol">101</span>(<span class="cit-issue">3</span>), <span class="cit-fpage">319</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSomers%26rft.auinit1%253DD.%2BE.%26rft.volume%253D101%26rft.issue%253D3%26rft.spage%253D319%26rft.epage%253D329%26rft.atitle%253DZEITLUPE%2Bencodes%2Ba%2Bnovel%2Bclock-associated%2BPAS%2Bprotein%2Bfrom%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980841-7%26rft_id%253Dinfo%253Apmid%252F10847686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80841-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10847686&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086741900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.66" data-doi="10.1371/journal.pone.0054583"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hyten</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quigley</span>,  <span class="cit-name-given-names">C. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fickus</span>,  <span class="cit-name-given-names">E. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">R. L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cregan</span>,  <span class="cit-name-given-names">P. B</span></span>. (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">Development and Evaluation of SoySNP50K, a High-Density Genotyping Array for Soybean</span>.” <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, <span class="cit-vol">8</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054583%26rft_id%253Dinfo%253Apmid%252F23620825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23620825&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hyten</span>,  <span class="cit-name-given-names">D. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Quigley</span>,  <span class="cit-name-given-names">C. V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fickus</span>,  <span class="cit-name-given-names">E. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">R. L.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cregan</span>,  <span class="cit-name-given-names">P. B</span></span>. (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Fingerprinting Soybean Germplasm and Its Utility in Genomic Research</span>..” <abbr class="cit-jnl-abbrev">G3</abbr>, <span class="cit-vol">5</span>(<span class="cit-issue">10</span>), <span class="cit-fpage">1999</span>–<span class="cit-lpage">2006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D1999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.68" data-doi="10.1105/tpc.010320"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solimeo</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rupert</span>,  <span class="cit-name-given-names">R. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yadav</span>,  <span class="cit-name-given-names">N. S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Q</span></span>. (<span class="cit-pub-date">2002</span>). “<span class="cit-article-title">Functional dissection of a Rice Dr1/DrAp1 transcriptional repression complex</span>.” <abbr class="cit-jnl-abbrev">Plant Cell</abbr>, <span class="cit-vol">14</span>(<span class="cit-issue">January</span>), <span class="cit-fpage">181</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DSong%26rft.auinit1%253DW.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D181%26rft.epage%253D195%26rft.atitle%253DFunctional%2BDissection%2Bof%2Ba%2BRice%2BDr1%252FDrAp1%2BTranscriptional%2BRepression%2BComplex%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.010320%26rft_id%253Dinfo%253Apmid%252F11826307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE0LzEvMTgxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjAvODQ4OTc4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Spencer</span>,  <span class="cit-name-given-names">C. C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donnelly</span>,  <span class="cit-name-given-names">P.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Marchini</span>,  <span class="cit-name-given-names">J</span></span>. (<span class="cit-pub-date">2009</span>). “<span class="cit-article-title">Designing genome-wide association studies: Sample size, power, imputation, and the choice of genotyping chip</span>.” <abbr class="cit-jnl-abbrev">PLoS Genetics</abbr>, <span class="cit-vol">5</span>(<span class="cit-fpage">5</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D5%26rft.spage%253D5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Street</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zuev</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaul</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">El Bouhssini</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Konopka</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Mitrofanova</span>,  <span class="cit-name-given-names">O</span></span>. (<span class="cit-pub-date">2008</span>). “<span class="cit-article-title">Diving into the genepool: a rational system to access specific traits from large germplasm collections</span>.” <abbr class="cit-jnl-abbrev">Proceedings of the 11th International wheat genetics symposium</abbr>, <span class="cit-fpage">28</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.71" data-doi="10.1016/j.gene.2014.01.024"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Su</span>,  <span class="cit-name-given-names">L. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">D. Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhai</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Q. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Q. Y</span></span>. (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">A novel MYB transcription factor, <em>GmMYBJ1</em>, from soybean confers drought and cold tolerance in <em>Arabidopsis thaliana</em></span>.” <abbr class="cit-jnl-abbrev">Gene</abbr>, <span class="cit-vol">538</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">46</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D538%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2014.01.024%26rft_id%253Dinfo%253Apmid%252F24440241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2014.01.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24440241&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332815600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hou</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fei</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">D.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">T</span></span>. (<span class="cit-pub-date">2011</span>). “<span class="cit-article-title"><em>GmFT2a</em>, a soybean homolog of flowering locus T, is involved in flowering transition and maintenance</span>.” <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">12</span>), <span class="cit-fpage">18</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D6%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.73" data-doi="10.1126/science.277.5329.1063"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>,  <span class="cit-name-given-names">S. D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">McCouch</span>,  <span class="cit-name-given-names">S. R</span></span>. (<span class="cit-pub-date">1997</span>). “<span class="cit-article-title">Seed Banks and Molecular Maps: Unlocking Genetic Potential from the Wild</span>.” <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">277</span>(<span class="cit-issue">5329</span>), <span class="cit-fpage">1063</span>–<span class="cit-lpage">1066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTanksley%26rft.auinit1%253DS.%2BD.%26rft.volume%253D277%26rft.issue%253D5329%26rft.spage%253D1063%26rft.epage%253D1066%26rft.atitle%253DSeed%2BBanks%2Band%2BMolecular%2BMaps%253A%2BUnlocking%2BGenetic%2BPotential%2Bfrom%2Bthe%2BWild%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.277.5329.1063%26rft_id%253Dinfo%253Apmid%252F9262467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzcvNTMyOS8xMDYzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjAvODQ4OTc4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.74" data-doi="10.1186/1471-2105-10-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thachuk</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Franco</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warburton</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Davenport</span>,  <span class="cit-name-given-names">G. F</span></span>. (<span class="cit-pub-date">2009</span>). “<span class="cit-article-title">Core Hunter: an algorithm for sampling genetic resources based on multiple genetic measures</span>..” <abbr class="cit-jnl-abbrev">BMC bioinformatics</abbr>, <span class="cit-vol">10</span>(<span class="cit-issue">243</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DMa%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DRegularized%2Bgene%2Bselection%2Bin%2Bcancer%2Bmicroarray%2Bmeta-analysis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-1%26rft_id%253Dinfo%253Apmid%252F19118496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19118496&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Varshney</span>,  <span class="cit-name-given-names">R. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">V. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Powell</span>,  <span class="cit-name-given-names">W.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Sorrells</span>,  <span class="cit-name-given-names">M. E</span></span>. (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Can genomics deliver climate-change ready crops?</span>.” <abbr class="cit-jnl-abbrev">Current Opinion in Plant Biology</abbr>, <span class="cit-vol">45</span>, <span class="cit-fpage">205</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BPlant%2BBiology%26rft.volume%253D45%26rft.spage%253D205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.76" data-doi="10.1016/j.tplants.2017.04.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardner</span>,  <span class="cit-name-given-names">C.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Lübberstedt</span>,  <span class="cit-name-given-names">T</span></span>. (<span class="cit-pub-date">2017a</span>). “<span class="cit-article-title">Emerging Avenues for Utilization of Exotic Germplasm</span>.” <abbr class="cit-jnl-abbrev">Trends in Plant Science</abbr>, <span class="cit-vol">22</span>(<span class="cit-issue">7</span>), <span class="cit-fpage">624</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BPlant%2BScience%26rft.volume%253D22%26rft.spage%253D624%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.04.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.77" data-doi="10.1371/journal.pone.0179159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gao</span>,  <span class="cit-name-given-names">Y.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J</span></span>. (<span class="cit-pub-date">2017b</span>). “<span class="cit-article-title">Characterization of dof transcription factors and their responses to osmotic stress in poplar (<em>Populus trichocarpa</em>)</span>.” <abbr class="cit-jnl-abbrev">PLoS ONE</abbr>, <span class="cit-vol">12</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D12%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0179159%26rft_id%253Dinfo%253Apmid%252F28957441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0179159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28957441&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.78" data-doi="10.1186/1471-2164-16-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chai</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valliyodan</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maupin</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Annen</span>,  <span class="cit-name-given-names">B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">H. T</span></span>. (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Genome-wide analysis and expression profiling of the PIN auxin transporter gene family in soybean (Glycine max)</span>.” <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-vol">16</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-16-1%26rft_id%253Dinfo%253Apmid%252F25553907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-16-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25553907&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F11%2F20%2F848978.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.79" data-doi="10.1534/genetics.110.125062"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hideshima</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tsubokura</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamanaka</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takahashi</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anai</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tabata</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kitamura</span>,  <span class="cit-name-given-names">K.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Harada</span>,  <span class="cit-name-given-names">K</span></span>. (<span class="cit-pub-date">2011</span>). “<span class="cit-article-title">A map-based cloning strategy employing a residual heterozygous line reveals that the <em>GIGANTEA</em> gene is involved in soybean maturity and flowering</span>.” <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">188</span>(<span class="cit-issue">2</span>), <span class="cit-fpage">395</span>–<span class="cit-lpage">407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.125062%26rft_id%253Dinfo%253Apmid%252F21406680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvMzk1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTEvMjAvODQ4OTc4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Watson</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghosh</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cuddy</span>,  <span class="cit-name-given-names">W. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simmonds</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rey</span>,  <span class="cit-name-given-names">M. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Asyraf Md Hatta</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hinchliffe</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steed</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reynolds</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adamski</span>,  <span class="cit-name-given-names">N. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breakspear</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korolev</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rayner</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dixon</span>,  <span class="cit-name-given-names">L. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Riaz</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ryan</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Batley</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Raman</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carter</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rogers</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domoney</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moore</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harwood</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicholson</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dieters</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Delacy</span>,  <span class="cit-name-given-names">I. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uauy</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boden</span>,  <span class="cit-name-given-names">S. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">R. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wulff</span>,  <span class="cit-name-given-names">B. B.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Hickey</span>,  <span class="cit-name-given-names">L. T.</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Speed breeding is a powerful tool to accelerate crop research and breeding</span>.” <abbr class="cit-jnl-abbrev">Nature Plants</abbr>, <span class="cit-vol">4</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BPlants%26rft.volume%253D4%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Thapa</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muir</span>,  <span class="cit-name-given-names">W. M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Rainey</span>,  <span class="cit-name-given-names">K. M.</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Population and quantitative genomic properties of the USDA soybean germplasm collection</span>.” <abbr class="cit-jnl-abbrev">Plant Genetic Resources: Characterisation and Utilisation</abbr>, <span class="cit-vol">16</span>(<span class="cit-issue">6</span>), <span class="cit-fpage">513</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenetic%2BResources%253A%2BCharacterisation%2Band%2BUtilisation%26rft.volume%253D16%26rft.spage%253D513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.82"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yamaguchi</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kurosaki</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishimoto</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Kawasaki</span>,  <span class="cit-name-given-names">M</span></span>. (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Early-Maturing and Chilling-Tolerant Soybean Lines Derived from Crosses between Japanese and Polish Cultivars</span>.” <abbr class="cit-jnl-abbrev">Plant Production Science</abbr>, <span class="cit-vol">18</span>(August 2014), <span class="cit-fpage">234</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BProduction%2BScience%26rft.volume%253D18%26rft.spage%253D234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.83"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">G. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">L. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">X. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Z. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">M. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shan</span>,  <span class="cit-name-given-names">S. G.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">X. G.</span></span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">A soybean C2H2-type zinc finger gene <em>GmZF1</em> enhanced cold tolerance in transgenic <em>Arabidopsis</em></span>.” <abbr class="cit-jnl-abbrev">PLOS ONE</abbr>, <span class="cit-vol">9</span>(<span class="cit-issue">10</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.84" data-doi="10.1186/s12864-017-4368-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chi</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">F.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">D</span></span>. (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Soybean MADS-box gene <em>GmAGL1</em> promotes flowering via the photoperiod pathway</span>.” <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-vol">19</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-4368-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-4368-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-848978v1.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Takeshima</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sato</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Watanabe</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kanazawa</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kong</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamada</span>,  <span class="cit-name-given-names">T.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Abe</span>,  <span class="cit-name-given-names">J</span></span>. (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">A recessive allele for delayed flowering at the soybean maturity locus <em>E9</em> is a leaky allele of <em>FT2a</em>, a <em>FLOWERING LOCUS T</em> ortholog</span>.” <abbr class="cit-jnl-abbrev">BMC Plant Biology</abbr>, <span class="cit-vol">16</span>(<span class="cit-issue">1</span>), <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiology%26rft.volume%253D16%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-848978v1.86"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Q. Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">A. G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">H. F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">Z. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lei</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">C. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J. S.</span></span>, and <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S. Y</span></span>. (<span class="cit-pub-date">2008</span>). “<span class="cit-article-title">Soybean WRKY-type transcription factor genes, <em>GmWRKY13</em>, <em>GmWRKY21</em>, and <em>GmWRKY54</em>, confer differential tolerance to abiotic stresses in transgenic <em>Arabidopsis</em> plants</span>.” <abbr class="cit-jnl-abbrev">Plant Biotechnology Journal</abbr>, <span class="cit-vol">6</span>(<span class="cit-issue">5</span>), <span class="cit-fpage">486</span>–<span class="cit-lpage">503</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnology%2BJournal%26rft.volume%253D6%26rft.spage%253D486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/848978v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F848978v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/848135v1" title="Determination of Cas9/dCas9 associated toxicity in microbes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;November 20, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/848978v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/848978v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/848978v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/848978v1" class="active" data-icon-position="" data-hide-link-title="0">Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1002231" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-XxOOBXedU8DP5-om3rYObTxdGUhvEKhGHfhIXEoDmEk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1002231" id="node1002231--2" data-pisa="biorxiv;848978v1" data-pisa-master="biorxiv;848978" data-seqnum="1002231" data-apath="/biorxiv/early/2019/11/20/848978.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Max</span> <span class="nlm-surname">Haupt</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Karl</span> <span class="nlm-surname">Schmid</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">848978; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/848978 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/848978v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/848978v1&amp;title=Combining%20Focused%20Identification%20of%20Germplasm%20and%20Core%20Collection%20Strategies%20to%20Identify%20Genebank%20Accessions%20for%20Central%20European%20Soybean%20Breeding" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/848978v1&amp;title=Combining%20Focused%20Identification%20of%20Germplasm%20and%20Core%20Collection%20Strategies%20to%20Identify%20Genebank%20Accessions%20for%20Central%20European%20Soybean%20Breeding" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/848978v1&amp;text=Combining%20Focused%20Identification%20of%20Germplasm%20and%20Core%20Collection%20Strategies%20to%20Identify%20Genebank%20Accessions%20for%20Central%20European%20Soybean%20Breeding" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/848978v1&amp;title=Combining%20Focused%20Identification%20of%20Germplasm%20and%20Core%20Collection%20Strategies%20to%20Identify%20Genebank%20Accessions%20for%20Central%20European%20Soybean%20Breeding" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/848978v1&amp;t=Combining%20Focused%20Identification%20of%20Germplasm%20and%20Core%20Collection%20Strategies%20to%20Identify%20Genebank%20Accessions%20for%20Central%20European%20Soybean%20Breeding" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/848978v1&amp;title=Combining%20Focused%20Identification%20of%20Germplasm%20and%20Core%20Collection%20Strategies%20to%20Identify%20Genebank%20Accessions%20for%20Central%20European%20Soybean%20Breeding" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/848978v1&amp;title=Combining%20Focused%20Identification%20of%20Germplasm%20and%20Core%20Collection%20Strategies%20to%20Identify%20Genebank%20Accessions%20for%20Central%20European%20Soybean%20Breeding" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1002231" id="node1002231--3" data-pisa="biorxiv;848978v1" data-pisa-master="biorxiv;848978" data-seqnum="1002231" data-apath="/biorxiv/early/2019/11/20/848978.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Combining Focused Identification of Germplasm and Core Collection Strategies to Identify Genebank Accessions for Central European Soybean Breeding</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Max</span> <span class="nlm-surname">Haupt</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Karl</span> <span class="nlm-surname">Schmid</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">848978; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/848978 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Combining%20Focused%20Identification%20of%20Germplasm%20and%20Core%20Collection%20Strategies%20to%20Identify%20Genebank%20Accessions%20for%20Central%20European%20Soybean%20Breeding&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Haupt%2C+Max&amp;rft.au=Schmid%2C+Karl"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1002231/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1002231/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1002231/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1002231/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1002231/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1002231/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1002231/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1002231/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1002231/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1002231/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1002231/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1002231/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/848978v1&amp;count=horizontal&amp;via=&amp;text=Combining%20Focused%20Identification%20of%20Germplasm%20and%20Core%20Collection%20Strategies%20to%20Identify%20Genebank%20Accessions%20for%20Central%20European%20Soybean%20Breeding&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/848978v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/848978v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/848978v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/plant-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Plant Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
