<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/13/807057.atom" />
<meta name="HW.pisa" content="biorxiv;807057v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Matching the forecast horizon with the relevant spatial and temporal processes and data sources" />
<meta name="DC.Identifier" content="10.1101/807057" />
<meta name="DC.Date" content="2019-12-13" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Most phenomenological, statistical models used to generate ecological forecasts take either a time-series approach, based on long-term data from one location, or a space-for-time approach, based on data describing spatial patterns across environmental gradients. Here we consider how the forecast horizon determines whether more accurate predictions come from the time-series approach, the space-for-time approach, or a combination of the two. We use two simulated case studies to show that forecasts for short and long forecast horizons need to focus on different ecological processes, which are reflected in different kinds of data. In the short-term, dynamics reflect initial conditions and fast processes such as birth and death, and the time-series approach makes the best predictions. In the long-term, dynamics reflect the additional influence of slower processes such as evolutionary and ecological selection, colonization and extinction, which the space-for-time approach can effectively capture. At intermediate time-scales, a weighted average of the two approaches shows promise. However, making this weighted model operational will require new research to predict the rate at which slow processes begin to influence dynamics." />
<meta name="DC.Contributor" content="Peter B. Adler" />
<meta name="DC.Contributor" content="Ethan P. White" />
<meta name="DC.Contributor" content="Michael H. Cortez" />
<meta name="article:published_time" content="2019-12-13" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Matching the forecast horizon with the relevant spatial and temporal processes and data sources" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Most phenomenological, statistical models used to generate ecological forecasts take either a time-series approach, based on long-term data from one location, or a space-for-time approach, based on data describing spatial patterns across environmental gradients. Here we consider how the forecast horizon determines whether more accurate predictions come from the time-series approach, the space-for-time approach, or a combination of the two. We use two simulated case studies to show that forecasts for short and long forecast horizons need to focus on different ecological processes, which are reflected in different kinds of data. In the short-term, dynamics reflect initial conditions and fast processes such as birth and death, and the time-series approach makes the best predictions. In the long-term, dynamics reflect the additional influence of slower processes such as evolutionary and ecological selection, colonization and extinction, which the space-for-time approach can effectively capture. At intermediate time-scales, a weighted average of the two approaches shows promise. However, making this weighted model operational will require new research to predict the rate at which slow processes begin to influence dynamics.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;807057v2" />
<meta name="citation_id" content="807057v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/807057v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/807057v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/807057v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057.full.pdf" />
<meta name="citation_doi" content="10.1101/807057" />
<meta name="citation_num_pages" content="27" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="807057" />
<meta name="citation_author" content="Peter B. Adler" />
<meta name="citation_author_institution" content="Department of Wildland Resources and the Ecology Center, Utah State University" />
<meta name="citation_author_email" content="peter.adler@usu.edu" />
<meta name="citation_author" content="Ethan P. White" />
<meta name="citation_author_institution" content="Department of Wildlife Ecology and Conservation, University of Florida" />
<meta name="citation_author_institution" content="Informatics Institute, University of Florida" />
<meta name="citation_author_institution" content="Biodiversity Institute, University of Florida" />
<meta name="citation_author" content="Michael H. Cortez" />
<meta name="citation_author_institution" content="Department of Biological Science, Florida State University" />
<meta name="citation_reference" content="Alexander, J. M. et al. 2018. Lags in the response of mountain plant communities to climate change. – Global Change Biology 24: 563–579." />
<meta name="citation_reference" content="Blois, J. L. et al. 2013. Space can substitute for time in predicting climate-change effects on biodiversity. – Proceedings of the National Academy of Sciences 110: 9374–9379." />
<meta name="citation_reference" content="Brun, P. et al. 2016. The predictive skill of species distribution models for plankton in a changing climate. – Global Change Biology 22: 3170–3181." />
<meta name="citation_reference" content="Chevalier, M. and Knape, J. 2020. The cost of complexity in forecasts of population abundances is reduced but not eliminated by borrowing information across space using a hierarchical 9 approach. – Oikos in press." />
<meta name="citation_reference" content="Clark, J. et al. 2001. Ecological forecasts: An emerging imperative. – Science 293: 657–660." />
<meta name="citation_reference" content="Dalgleish, H. J. et al. 2011. Climate influences the demography of three dominant sagebrush steppe plants. – Ecology 92: 75–85." />
<meta name="citation_reference" content="Davis, E. B. et al. 2014. Ecological niche models of mammalian glacial refugia show consistent bias. – Ecography 37: 1133–1138." />
<meta name="citation_reference" content="Dietze, M. C. 2017. Prediction in ecology: a first-principles framework. – Ecological Applications 27: 2048–2060." />
<meta name="citation_reference" content="Dietze, M. C. et al. 2018. Iterative near-term ecological forecasting: Needs, opportunities, and challenges. – Proceedings of the National Academy of Sciences 115: 1424–1432." />
<meta name="citation_reference" content="Elith, J. and Leathwick, J. R. 2009. Species distribution models: ecological explanation and prediction across space and time. – Annual Review of Ecology, Evolution, and Systematics 40: 677–697." />
<meta name="citation_reference" content="Good, B. H. et al. 2017. The dynamics of molecular evolution over 60,000 generations. – Nature 551: 45." />
<meta name="citation_reference" content="Harris, D. J. et al. 2018. Forecasting biodiversity in breeding birds using best practices. – PeerJ 6: e4278." />
<meta name="citation_reference" content="Hefley, T. J. et al. 2017. When mechanism matters: Bayesian forecasting using models of ecological diffusion. – Ecology Letters 20: 640–650." />
<meta name="citation_reference" content="Hoffmann, A. A. and Sgro, C. M. 2011. Climate change and evolutionary adaptation. – Nature 470: 479–485." />
<meta name="citation_reference" content="Houlahan, J. E. et al. 2017. The priority of prediction in ecological understanding. – Oikos 126: 1–7." />
<meta name="citation_reference" content="Hyndman, R. and Athanasopoulos, G. 2018. Forecasting: Principles and Practice. – OTexts." />
<meta name="citation_reference" content="Illán, J. G. et al. 2014. Precipitation and winter temperature predict long-term range-scale abundance changes in Western North American birds. – Global Change Biology 20: 3351–3364." />
<meta name="citation_reference" content="Kleinhesselink, A. R. and Adler, P. B. 2018. The response of big sagebrush (Artemisia tridentata) to interannual climate variation changes across its range. – Ecology 99: 1139–1149." />
<meta name="citation_reference" content="Lauenroth, W. K. and Sala, O. E. 1992. Long-term forage production of North American short-grass steppe. – Ecological Applications 2: 397–403." />
<meta name="citation_reference" content="Levin, S. A. 1992. The problem of pattern and scale in ecology: the robert h. macarthur award lecture. – Ecology 73: 1943–1967." />
<meta name="citation_reference" content="McDermott, P. L. and Wikle, C. K. 2017. An ensemble quadratic echo state network for non-linear spatio-temporal forecasting. – Stat 6: 315–330." />
<meta name="citation_reference" content="Mouquet, N. et al. 2015. REVIEW: Predictive ecology in a changing world. – Journal of Applied Ecology 52: 1293–1310." />
<meta name="citation_reference" content="Nice, C. C. et al. 2019. Extreme heterogeneity of population response to climatic variation and the limits of prediction. – Global Change Biology 25: 2127–2136." />
<meta name="citation_reference" content="Norberg, A. et al. 2019. A comprehensive evaluation of predictive performance of 33 species distribution models at species and community levels. – Ecological Monographs 89: e01370." />
<meta name="citation_reference" content="Renwick, K. M. et al. 2018. Multi-model comparison highlights consistency in predicted effect of warming on a semi-arid shrub. – Global Change Biology 24: 424–438." />
<meta name="citation_reference" content="Rosenzweig, M. L. et al. 1995. Species diversity in space and time. – Cambridge University Press." />
<meta name="citation_reference" content="Sala, O. E. et al. 1988. Primary production of the central grassland region of the United States. – Ecology 69: 40–45." />
<meta name="citation_reference" content="Schliep, E. M. et al. 2018. Joint species distribution modelling for spatio-temporal occurrence and ordinal abundance data. – Global Ecology and Biogeography 27: 142–155." />
<meta name="citation_reference" content="Silvertown, J. et al. 2006. The Park Grass Experiment 1856-2006: Its contribution to ecology. – Journal of Ecology 94: 814." />
<meta name="citation_reference" content="Urban, M. C. et al. 2012. On a collision course: competition and dispersal differences create no-analogue communities and cause extinctions during climate change. – Proceedings of the Royal Society of London B: Biological Sciences 279: 2072–2080." />
<meta name="citation_reference" content="Veloz, S. D. et al. 2012. No-analog climates and shifting realized niches during the late quaternary: implications for 21st-century predictions by species distribution models. – Global Change Biology 18: 1698–1713." />
<meta name="citation_reference" content="Ward, E. J. et al. 2014. Complexity is costly: a meta-analysis of parametric and non-parametric methods for short-term population forecasting. – Oikos 123: 652–661." />
<meta name="citation_reference" content="Wikle, C. K. 2003. Hierarchical Bayesian Models for Predicting the Spread of Ecological Processes. – Ecology 84: 1382–1394." />
<meta name="citation_reference" content="Williams, J. W. and Jackson, S. T. 2007. Novel climates, no-analog communities, and ecological surprises. – Frontiers in Ecology and the Environment 5: 475–482." />
<meta name="citation_reference" content="Williams, P. J. et al. 2017. An integrated data model to estimate spatiotemporal occupancy, abundance, and colonization dynamics. – Ecology 98: 328–336." />
<meta name="citation_reference" content="Worth, J. R. P. et al. 2014. Environmental niche modelling fails to predict Last Glacial Maximum refugia: niche shifts, microrefugia or incorrect palaeoclimate estimates? – Global Ecology and Biogeography 23: 1186–1197." />
<meta name="twitter:title" content="Matching the forecast horizon with the relevant spatial and temporal processes and data sources" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Most phenomenological, statistical models used to generate ecological forecasts take either a time-series approach, based on long-term data from one location, or a space-for-time approach, based on data describing spatial patterns across environmental gradients. Here we consider how the forecast horizon determines whether more accurate predictions come from the time-series approach, the space-for-time approach, or a combination of the two. We use two simulated case studies to show that forecasts for short and long forecast horizons need to focus on different ecological processes, which are reflected in different kinds of data. In the short-term, dynamics reflect initial conditions and fast processes such as birth and death, and the time-series approach makes the best predictions. In the long-term, dynamics reflect the additional influence of slower processes such as evolutionary and ecological selection, colonization and extinction, which the space-for-time approach can effectively capture. At intermediate time-scales, a weighted average of the two approaches shows promise. However, making this weighted model operational will require new research to predict the rate at which slow processes begin to influence dynamics." />
<meta name="og-title" property="og:title" content="Matching the forecast horizon with the relevant spatial and temporal processes and data sources" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/807057v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Most phenomenological, statistical models used to generate ecological forecasts take either a time-series approach, based on long-term data from one location, or a space-for-time approach, based on data describing spatial patterns across environmental gradients. Here we consider how the forecast horizon determines whether more accurate predictions come from the time-series approach, the space-for-time approach, or a combination of the two. We use two simulated case studies to show that forecasts for short and long forecast horizons need to focus on different ecological processes, which are reflected in different kinds of data. In the short-term, dynamics reflect initial conditions and fast processes such as birth and death, and the time-series approach makes the best predictions. In the long-term, dynamics reflect the additional influence of slower processes such as evolutionary and ecological selection, colonization and extinction, which the space-for-time approach can effectively capture. At intermediate time-scales, a weighted average of the two approaches shows promise. However, making this weighted model operational will require new research to predict the rate at which slow processes begin to influence dynamics." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-13" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/807057v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/807057v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/807057v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/807057v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1036436" />
    <title>Matching the forecast horizon with the relevant spatial and temporal processes and data sources | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__AQm9HArb2Y_D7UzgraiLIwAB7088fIq2iZD0Uigie_s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"pp8NyHI1WBmw7DzOfjboH7sA_qtfXEyP9EF7Q_fnNYY","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;807057v2"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;807057v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;807057v2"}],"nid":"1036436","apath":"\/biorxiv\/early\/2019\/12\/13\/807057.atom","pisa":"biorxiv;807057v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/807057v2","title":"Matching the forecast horizon with the relevant spatial and temporal processes and data sources","identifier":"node\/1036436"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XrnVHN5Vt1nWopnrPLplGFibxr5G4\\\\\\\/Es4HVybT\\\\\\\/2zsTwn0Ys0CfyoO6ifT6PZwVMfY32XCtBo3de9pJXYV03tofnblj42GZwTmua2TgSaU\\\\\\\/tKfzOgJ6HLYuB50kiU\\\\\\\/hYutJAkiVqoxkcZ30m1\\\\\\\/eSXVVWw+EfC\\\\\\\/xPf6VikuCajwNQ2O0sCtVaTjNDHIOMgCLOrr9Am+1\\\\\\\/fmhc2wJgd6XvkijHVlstPOxk\\\\\\\/yxAmCwGLZhWGhUUZGUWHKoivUmZnAE5qVF+JX\\\\\\\/6mQMaMvvS8LAtgN5Qnw3QPOb8c1k\\\\\\\/L6sdfBggNgvp\\\\\\\/9cyArEwWni+9qaF613l3fjmW7C8zFHi\\\\\\\/UKgKVhL5YW6YPHHUtzS9GrGp7hl2BBZMtQ8OpbKrZtaDf5TwojPTn05BFZP7CKk9h5TuwfvP3vl6ZSfkxcI6nYMUixo9tfJdiW0di7dL8pFAePxpDQ2Z3FFjARwuS1RICvfvnDIPFx0jM4rW0lolNQhZqiIkDg1IvNfRQgl5GkhcdLyaxDezHb9g3swkvpaWwrbMPqxsZxue2AzbrS2vYIZRIsyWTZYdxGIx60QW+rEMR7bapVnGxxvM05oDxyow7CDKJB0LsqwJHM0fhK\\\\\\\/rz3piIURHB\\\\\\\/WbYMRJzAnpnZ\\\\\\\/4ogDF\\\\\\\/+GXOZ9v4\\\\\\\/ttjM2svYEHDJvWOZPDEi+TT8Hxl327q3tmJlxPbpaKlQ1b2niuyxQVXNpGSD9CiXkIMgfoSCEMSX1zELy15EWvgV\\\\\\\/Pt06s7EJ3Lw\\\\\\\/x7fdO6WXsTIwSCEiKB0o8yVPtg2H9ZuKsXRM96H+jOZx\\\\\\\/9HhMuWuIXWR6WN0sfQEbgYBdQhcdyfJ44p72BwovojIYafsmYvdLZ9Gar4dw6\\\\\\\/JpEdIqh+KNB3ANAk1JFhchvtebacpGFxEzPToCjcpumpns+Q9epd9RiPgwl8LUAZLwSKYZlBczE89lHMLEFDcfH4Jl0B4EDRE214X3RqML3nJg8fTSDZpZJN\\\\\\\/QpGjZvivuAQ7W6cjeLiSQDptnjDPBKocpD8hSPUOka1x4crvlrkELdhRkFUYfIBqLz8U0WvAGxj+xQ5foXiQmaXOaLRLkIggCqI3fDKO9tdAa27clYBhhlWcrPp5z\\\\\\\/aB78C9v5tEekWPApC39rqkvchN8rw\\\\\\\/FDCOHL+gtiX5eLqs4rwyGlDCYM+l6PIagIVhTj4eggu5isemvu0BlV0p58x84v75TaLq0vm9XVwZSrKiiP+FwSBPPTAWm9SHTyBNBPMKrNn1KAK+p9Ei\\\\\\\/TQzBaHU2GoV+oQXUv\\\\\\\/zgMbL8BsYLwf4rjDzrrXfh811x2+N69hueRfCxCpnC4UK+4PneVSaD7ZfW4KDZa\\\\\\\/wusB3ZDcb\\\u0022,\\\u0022iv\\\u0022:\\\u00221Rosv+iDkv3E5cDsStULmA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00223c598ad3ea6f92cc190fc2eaad43d3c4c97acc4e5c1fe2e9bfca3273c658c92b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/807057v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1036436 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/807057v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-I1g0VLtEtTMi37-NppWPNHqPmuVJAa_02vxOm4GYMvw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1036436" id="node1036436" data-pisa="biorxiv;807057v2" data-pisa-master="biorxiv;807057" data-apath="/biorxiv/early/2019/12/13/807057.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Matching the forecast horizon with the relevant spatial and temporal processes and data sources</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Peter B.</span> <span class="nlm-surname">Adler</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ethan P.</span> <span class="nlm-surname">White</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael H.</span> <span class="nlm-surname">Cortez</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/807057 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1036436" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Peter B. Adler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Wildland Resources and the Ecology Center, Utah State University</span>, Logan, Utah</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BB.%2BAdler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Adler%20PB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BB.%2BAdler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:peter.adler@usu.edu" class="" data-icon-position="" data-hide-link-title="0">peter.adler@usu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Ethan P. White </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Wildlife Ecology and Conservation, University of Florida</span>, Gainesville, Florida</div><div class='author-affiliation'><span class='nlm-institution'>Informatics Institute, University of Florida</span>, Gainesville, Florida</div><div class='author-affiliation'><span class='nlm-institution'>Biodiversity Institute, University of Florida</span>, Gainesville, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ethan%2BP.%2BWhite%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=White%20EP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEthan%2BP.%2BWhite%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael H. Cortez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biological Science, Florida State University</span>, Tallahasee, Florida</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BH.%2BCortez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cortez%20MH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BH.%2BCortez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/807057v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1036436" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1036436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/807057v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1036436" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1036436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/807057v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1036436" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1036436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/807057v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1036436" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1036436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/807057v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1036436" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1036436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Most phenomenological, statistical models used to generate ecological forecasts take either a time-series approach, based on long-term data from one location, or a space-for-time approach, based on data describing spatial patterns across environmental gradients. Here we consider how the forecast horizon determines whether more accurate predictions come from the time-series approach, the space-for-time approach, or a combination of the two. We use two simulated case studies to show that forecasts for short and long forecast horizons need to focus on different ecological processes, which are reflected in different kinds of data. In the short-term, dynamics reflect initial conditions and fast processes such as birth and death, and the time-series approach makes the best predictions. In the long-term, dynamics reflect the additional influence of slower processes such as evolutionary and ecological selection, colonization and extinction, which the space-for-time approach can effectively capture. At intermediate time-scales, a weighted average of the two approaches shows promise. However, making this weighted model operational will require new research to predict the rate at which slow processes begin to influence dynamics.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-5">Forecasting is increasingly recognized as important to the application and advancement of ecological research. Forecasts are necessary to guide environmental policy and management decisions about mitigation and adaption to global change (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Clark et al., 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Mouquet et al., 2015</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dietze et al., 2018</a>). But forecasts can also advance understanding of the processes governing ecological systems by providing rigorous tests of model predictions (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Houlahan et al., 2017</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dietze, 2017</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dietze et al., 2018</a>). The dual benefits of informing management and advancing basic knowledge makes forecasting an important priority for ecological research.</p><p id="p-6">Statistical models used for ecological forecasting generally rely on either time-series approaches or space-for-time substitutions. The time-series approach involves fitting models to long-term datasets to describe the temporal dynamics of a system. We then use those dynamic models to make predictions about what will happen in the future. This approach is often used to study population or vital rate fluctuations as a function of weather (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dalgleish et al., 2011</a>), or primary production as a function of annual precipitation (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lauenroth and Sala, 1992</a>). When time-series models are fit with typically short ecological data sets, they capture “fast processes” operating on interannual time-scales, such as birth, death, individual growth, small-scale dispersal events, and short-term responses to environmental conditions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>). Statistical models built using this approach normally cover a limited spatial extent (but see <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hefley et al. 2017</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kleinhesselink and Adler 2018</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chevalier and Knape 2020</a>), and ignore slower processes, such as evolutionary adaptation or turnover in community composition, that could influence dynamics at longer time scales (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Clark et al., 2001</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fast and slow processes operate at different time scales, and are reflected in different kinds of datasets. Fast processes, such as births, deaths, and individual growth, operate at all time scales, but are the exclusive drivers of the short-term dynamics captured in most time series datasets. Slower processes, such as evolutionary selection on genotype frequencies, ecological selection on species abundances, and colonization and extinction, interact with fast processes to drive dynamics over the long-term. The influence of these slow processes is seen in very long time series, or in spatial gradients. Understanding dynamics at intermediate time scales requires integrating information from spatial and temporal data sources. We propose a model weighting approach; mechanistic spatiotemporal modeling is another alternative. The time scales shown here were chosen with vascular plants in mind, but the same concepts would apply for much shorter-lived organisms but at shorter time scales." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1000258793" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Fast and slow processes operate at different time scales, and are reflected in different kinds of datasets. Fast processes, such as births, deaths, and individual growth, operate at all time scales, but are the exclusive drivers of the short-term dynamics captured in most time series datasets. Slower processes, such as evolutionary selection on genotype frequencies, ecological selection on species abundances, and colonization and extinction, interact with fast processes to drive dynamics over the long-term. The influence of these slow processes is seen in very long time series, or in spatial gradients. Understanding dynamics at intermediate time scales requires integrating information from spatial and temporal data sources. We propose a model weighting approach; mechanistic spatiotemporal modeling is another alternative. The time scales shown here were chosen with vascular plants in mind, but the same concepts would apply for much shorter-lived organisms but at shorter time scales.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F1.medium.gif" width="440" height="360"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F1.medium.gif" width="440" height="360"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-7" class="first-child">Fast and slow processes operate at different time scales, and are reflected in different kinds of datasets. Fast processes, such as births, deaths, and individual growth, operate at all time scales, but are the exclusive drivers of the short-term dynamics captured in most time series datasets. Slower processes, such as evolutionary selection on genotype frequencies, ecological selection on species abundances, and colonization and extinction, interact with fast processes to drive dynamics over the long-term. The influence of these slow processes is seen in very long time series, or in spatial gradients. Understanding dynamics at intermediate time scales requires integrating information from spatial and temporal data sources. We propose a model weighting approach; mechanistic spatiotemporal modeling is another alternative. The time scales shown here were chosen with vascular plants in mind, but the same concepts would apply for much shorter-lived organisms but at shorter time scales.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Space-for-time substitution approaches begin by describing how an ecological variable of interest, such as occupancy or productivity, varies across sites experiencing different environmental conditions. These spatial relationships between environment and ecological response are assumed to also hold for changes at a site through time. To make a forecast, we first predict the future environmental conditions and then determine the associated ecological response, based on the observed spatial relationship. This is the approach commonly used to predict population distribution or abundance as a function of climate (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elith and Leathwick, 2009</a>) or mean primary production as a function of mean precipitation (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Sala et al., 1988</a>). Space-for-time models capture the outcome of interactions between fast processes and slower processes operating over long time periods, such as immigration, extinction, and responses to large or prolonged environmental changes (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>). However, space-for-time models provide no information about how quickly the system will move from the current state to the predicted, future state. In fact, transient dynamics could prevent the system from ever reaching the predicted steady state (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Urban et al., 2012</a>). Although both time-series and space-for-time approaches are widely used, there has been little discussion of their advantages and disadvantages for guiding policy decisions or advancing our understanding of ecological dynamics (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Harris et al., 2018</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Renwick et al., 2018</a>).</p><p id="p-9">Whether historical dynamics, contemporary spatial patterns, or some combination of the two will serve as the best source of information for forecasting may depend on how far into the future we are attempting to forecast (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Harris et al., 2018</a>). This potential dependency on the “forecast horizon” (<em>sensu</em> <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hyndman and Athanasopoulos 2018</a>) reflects lags in the response of ecological conditions to environmental change, shifts in the importance of ecological processes with time scale (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Levin, 1992</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Rosenzweig et al., 1995</a>), and differences between time-series and spatial gradients in the range of environmental conditions represented in observed data (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a>). At short forecast horizons (days to years), dynamics will reflect physiological and demographic responses and interactions among the organisms present at a site more than temporal turnover of genotypes or species; environmental conditions are likely to stay within the range of historical variation; and the current state of the system is likely to capture the influence of unmeasured processes. As a result, for near-term forecasts time-series approaches may capture the key dynamics and provide accurate predictions.</p><p id="p-10">In contrast, at long forecast horizons (decades to centuries), environmental conditions that have not been historically observed are likely to not only occur but to persist long enough to drive significant turnover of genotypes and species through colonization and extinction as well as changes in the flux of energy and nutrients. At these long forecast horizons, the state of the system at the time the forecast is issued may be little help in predicting the future state. For the century-scale forecasts often featured in biodiversity and species-distribution modeling, space-for-time approaches may effectively capture the response of ecosystems to major shifts in climate over long periods, producing better long-term forecasts than time-series approaches. Using different modeling approaches for different forecast horizons is common in other disciplines. For example, meteorological models for short-term weather forecasts differ substantially in spatial and temporal resolution and extent from the global circulation models used to predict long-term changes in climate.</p><p id="p-11">Why not simply use process-based models to avoid the difficulties posed by phenomenological time-series and space-for-time modeling approaches? If we could accurately characterize all of the processes governing a system, then a model based on that understanding should make accurate predictions at all forecast horizons. Process-based models should also be more robust for making predictions outside of historically observed conditions and even beyond the conditions observed across spatial gradients, which will be especially important in a future with increasingly novel combinations of environment and species interactions (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Williams and Jackson, 2007</a>). Unfortunately, in most cases this approach is not currently feasible because we lack a detailed knowledge of all the complex and interacting processes influencing the dynamics of real ecological systems. Even if the general form of the models were known, estimating the high number of parameters and quantifying how they vary across ecosystems typically requires more data than is currently available even for well-studied systems. Furthermore, the high complexity and corresponding parameter uncertainty of such models can increase predictive errors; simpler time-series models may actually perform better (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ward et al., 2014</a>), though spatial replication can reduce the cost of complexity (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chevalier and Knape, 2020</a>). As a result, models used for ecological forecasting will include at least some phenomenological components. But that does not mean that phenomenological forecast models cannot benefit from process-based understanding. Even if process-level understanding does not enable a fully mechanistic model, it can improve the specification of phenomenological models. Our hypothesis is that different processes may be relevant for different forecast horizons, and that we can act on this knowledge by fitting models to different kinds of datasets.</p><p>Here we use two simulated case studies to 1) demonstrate that the best model-building approaches for ecological forecasting depend on the time horizon of the forecast, and 2) explore how time-series and space-for-time approaches might be combined via weighted averaging to make better forecasts at intermediate time scales. The first case study focuses on how interspecific interactions affect the population dynamics of a focal species, and the second focuses on an eco-evolutionary scenario. Our analyses show that:
</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-13">For short-term forecasts, phenomenological time-series approaches are hard to beat, whereas longer-term forecasts require accounting for the influence of slow processes such as evolutionary and ecological selection as well as dispersal.</p></li><li id="list-item-2"><p id="p-14">Different kinds of data reflect the operation of different processes: longitudinal data capture autocorrelation and fast responses of current assemblages to interannual environmental variation, while data spanning spatial gradients capture the long-term outcome of interactions between fast and slow processes. Whether predictive models should be trained using longitudinal or spatial data sets, or both, depends on the time-scale of the desired forecast.</p></li><li id="list-item-3"><p id="p-15">A key challenge for future research is determining the rate at which slow processes begin to influence dynamics.</p></li></ol></div><div class="section" id="sec-2"><h2 class="">Modeling approach</h2><p id="p-16">In each case study, we simulated the effects of an increase in temperature on simple systems with known dynamics. The truth was represented by a simulation model that was mechanistic for at least one important process, but we treated the data-generating model as unknown when analyzing the data and we assumed that perfectly recovering the mechanisms it contains would not be possible in practice. We began each simulation under a stationary distribution of annual temperatures, allowing the system to equilibrate; we call this the baseline phase. We then increased temperature progressively over a period of time, followed by a second period of stationary, now elevated, temperature. The objective was to forecast the response of the system to the temperature increase based on spatial and/or temporal data “sampled” from the simulation during the baseline period.</p><p id="p-17">We made forecasts based on two phenomenological statistical models, each representing processes operating at different time scales. One statistical model represents the time-series or “temporal approach.” We correlated interannual variation in an ecological response with interannual variation in temperature at just one site. The other statistical model relies on a space-for-time substitution, which we call the “spatial approach” for brevity. We correlated the mean temperature with the mean of an ecological state or rate across many sites. We compared forecasts from both statistical models to the simulated dynamics to determine how well the two approaches performed at different forecast horizons. We also assessed the potential for combining the information available in temporal and spatial patterns by using a weighted average of the forecasts from the temporal and spatial approaches optimized to best match the (simulated) observations. We then studied how the optimal model weights changed over time. We expected the temporal approach to best predict short-term dynamics, the spatial approach to best predict long-term dynamics, while the weighted model would show potential to provide the best forecasts at transitional, intermediate time scales. The three statistical models are described in Supporting Information (<a id="xref-sec-7-1" class="xref-app" href="#sec-7">Appendix A</a>). Computer code for both case studies is available for reviewers online (<a href="https://drive.google.com/open?id=1aju04qtQvJmZG1mcpN-6hYBZILBhDfl2">https://drive.google.com/open?id=1aju04qtQvJmZG1mcpN-6hYBZILBhDfl2</a>) and will be archived at Zenodo upon acceptance.</p></div><div class="section" id="sec-3"><h2 class="">Community turnover example</h2><p id="p-18">Conservation biologists and natural resource managers often need to anticipate the impact of environmental change on the abundance of endangered species, biological invaders, and harvested species. Although the managers may be primarily interested in just one focal species, skillful prediction might require considering interactions with many other species, greatly complicating the problem. But at what forecast horizon do altered species interactions become impossible to ignore? We explored this question using a metacommunity simulation model developed by <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexander et al. (2018)</a> to study how community responses to increasing temperature depend on the interplay between within-site demography and competitive interactions and the movement of species across sites. The model features Lotka-Volterra competitive interactions among plants within sites that are arrayed along an elevation and temperature gradient. Composition varies along the gradient because of a trade-off between growth rate and cold tolerance: cold sites are dominated by slow-growing species that can tolerate low temperatures, while warm sites are dominated by fast-growing species that are cold intolerant. Multiple species can coexist within sites because all species experience stronger competition from conspecifics than from heterospecifics. Sites are linked by dispersal: a specified fraction of each species’ offspring leaves the site where they were produced and reaches all other sites with equal probability. We provide a more detailed description of the simulation model in SI <a id="xref-sec-8-1" class="xref-app" href="#sec-8">Appendix B</a>.</p><p id="p-19">We first simulated a baseline period with variable but stationary temperature, followed by a period of rapid temperature increase, and then a final period of stationary temperature. Interannual variation in temperature is the same at all sites, but mean temperature varies among sites. All sites experienced the same absolute increase in mean temperature. We focused on the biomass dynamics of one focal species that dominated the central site during the baseline period.</p><p id="p-20">During the baseline period there were strong spatial patterns across the mean temperature gradient. Individual species, including our focal species, showed classic, unimodal “Whittaker” patterns of abundances across the gradient (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). These spatial patterns are the basis for our spatial statistical model of the temperature-biomass relationship for our focal species (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2A</a>). In contrast to the strong spatial patterns, population and community responses to interannual variation in temperature within sites were weak. At our focal site in the center of the gradient, the biomass of the focal species was quite insensitive to interannnual variation in temperature, but showed strong temporal autocorrelation (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2B</a>). Our temporal statistical model estimates this weak, linear temperature effect, along with the strong lag effect of biomass in the previous year.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Mean biomass by species (colors) across the temperature gradient during the baseline period. The focal species, dominant at the site in the center of the gradient (vertical gray line), is shown in dark blue. The dashed blue line shows predictions from the spatial statistical model. (B) Annual biomass of the focal species at the central site during the baseline period. The dashed line shows predictions from the temporal statistical model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1000258793" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Mean biomass by species (colors) across the temperature gradient during the baseline period. The focal species, dominant at the site in the center of the gradient (vertical gray line), is shown in dark blue. The dashed blue line shows predictions from the spatial statistical model. (B) Annual biomass of the focal species at the central site during the baseline period. The dashed line shows predictions from the temporal statistical model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F2.medium.gif" width="440" height="174"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F2.medium.gif" width="440" height="174"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-21" class="first-child">(A) Mean biomass by species (colors) across the temperature gradient during the baseline period. The focal species, dominant at the site in the center of the gradient (vertical gray line), is shown in dark blue. The dashed blue line shows predictions from the spatial statistical model. (B) Annual biomass of the focal species at the central site during the baseline period. The dashed line shows predictions from the temporal statistical model.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">We used both the temporal and spatial statistical models to forecast the effect of a temperature increase (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A</a>) on the focal species’ biomass at one location in the center of the temperature gradient. The predictions from these two models contrasted markedly, with the temporal statistical model predicting a large increase in biomass and the spatial statistical model predicting a decrease. Initially, the simulated abundances followed the increase predicted by the temporal model, but as faster-growing species colonized and increased in abundance at the focal site, the biomass of the focal species decreased, eventually falling below its baseline level (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3B</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Simulated annual temperatures (grey) and expected temperature (black), which was used to make forecasts, at the focal site. (B) Simulated focal species biomass and forecasts from the spatial, temporal and weighted statistical models at the focal site in the metacommunity model. (C) Simulated changes in biomass of the focal species (black) and all other species (grey), and the weight given to the temporal statistical model for focal species biomass (blue). Year 1000 in each panel corresponds to the start of the temperature increase." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1000258793" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Simulated annual temperatures (grey) and expected temperature (black), which was used to make forecasts, at the focal site. (B) Simulated focal species biomass and forecasts from the spatial, temporal and weighted statistical models at the focal site in the metacommunity model. (C) Simulated changes in biomass of the focal species (black) and all other species (grey), and the weight given to the temporal statistical model for focal species biomass (blue). Year 1000 in each panel corresponds to the start of the temperature increase.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F3.medium.gif" width="246" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F3.medium.gif" width="246" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-23" class="first-child">(A) Simulated annual temperatures (grey) and expected temperature (black), which was used to make forecasts, at the focal site. (B) Simulated focal species biomass and forecasts from the spatial, temporal and weighted statistical models at the focal site in the metacommunity model. (C) Simulated changes in biomass of the focal species (black) and all other species (grey), and the weight given to the temporal statistical model for focal species biomass (blue). Year 1000 in each panel corresponds to the start of the temperature increase.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To combine information from the temporal and spatial statistical models into a single prediction, we fit a weighting parameter, <em>ω</em>, which varies over time and is bounded between 0 and 1. At any time point, <em>t</em>, this weighted forecast is <em>ω</em> · <em>T</em>(<em>N</em><sub><em>t</em>–1</sub>, <em>K<sub>t</sub></em>) + (1 – <em>ω</em>) · <em>S</em>(<em>K<sub>t</sub></em>) where <em>T</em> is the temporal statistical model, which depends on population size, <em>N</em>, and expected temperature, <em>K</em>, and <em>S</em> is the spatial statistical model, which depends only on <em>K</em> (see SI <a id="xref-sec-7-2" class="xref-app" href="#sec-7">Appendix A</a> for a full description of the approach). The weighted model accurately predicts the simulated dynamics across the full forecast horizon (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3B</a>). It also shows that the most rapid shifts in the model weights occurred during the period when warm-adapted, faster growing species were increasing most rapidly in abundance (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3C</a>). However, the reason the weighted models works so well is that the weights were determined by fitting directly to the data. Unlike the forecasts from the spatial and temporal statistical models, we did not generate out-of-sample predictions from the weighted model; it merely provides a convenient way to quantify how rapidly dynamics shift from being dominated by interannual variation captured in the temporal model (time <em>t</em> = 0 to <em>t</em> ≈ 1250 in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3B</a>) to being dominated by the steady-state equilibrium captured by the spatial model (time <em>t</em> ≥ 2500). A true forecast from the weighted model would require a method to determine the model weights <em>a priori</em>.</p><p id="p-25">The compositional turnover affecting our focal species also influences total biomass, linking community and ecosystem dynamics. We repeated our focal species analysis for total community biomass, and the results were similar: the temporal statistical model initially made the best forecasts immediately following the onset of the temperature increase, but as the identity and abundances of species at the study site changed, the model weights rapidly shifted to the spatial statistical model (SI <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figs. S-1</a> and <a id="xref-fig-8-1" class="xref-fig" href="#F8">S-2</a>).</p></div><div class="section" id="sec-4"><h2 class="">Eco-evolutionary example</h2><p id="p-26">Evolutionary adaptation is a key uncertainty in predicting how environmental change will impact a focal population at a given location (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hoffmann and Sgro, 2011</a>). Like the shifts in species composition illustrated in the previous example, shifts in genotype frequencies can also influence dynamics and forecasts at different time scales. Although shifts in genotype frequencies at the population level are analogous to changes in species composition at the community level, the mechanisms are distinct: heterozygosity and genetic recombination have no analogue at the community level. We demonstrate how these processes influence short and long-term forecasts with a standard eco-evolutionary simulation model for a hypothetical annual plant population in which fecundity is temperature dependent, and different genotypes have different temperature optima (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4A</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Reaction norms of the three genotypes. (B) The spatial pattern of individual genotypes (colors) and total population abundance (black) at sites arrayed across a gradient of mean annual temperature. The dashed black line (almost entirely hidden by the slid black line) shows predictions from an empirical, spatial statistical model, a linear regression that describes mean population size as a function of mean temperature. (C) The relationship between annual temperature and per capita growth rate at a location with a mean temperature that favors the cold-adapted genotype. Colors show population size (the green to brown gradient depicting low to high population density), which influences the population growth rate through density dependence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1000258793" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Reaction norms of the three genotypes. (B) The spatial pattern of individual genotypes (colors) and total population abundance (black) at sites arrayed across a gradient of mean annual temperature. The dashed black line (almost entirely hidden by the slid black line) shows predictions from an empirical, spatial statistical model, a linear regression that describes mean population size as a function of mean temperature. (C) The relationship between annual temperature and per capita growth rate at a location with a mean temperature that favors the cold-adapted genotype. Colors show population size (the green to brown gradient depicting low to high population density), which influences the population growth rate through density dependence.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F4.medium.gif" width="160" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F4.medium.gif" width="160" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-27" class="first-child">(A) Reaction norms of the three genotypes. (B) The spatial pattern of individual genotypes (colors) and total population abundance (black) at sites arrayed across a gradient of mean annual temperature. The dashed black line (almost entirely hidden by the slid black line) shows predictions from an empirical, spatial statistical model, a linear regression that describes mean population size as a function of mean temperature. (C) The relationship between annual temperature and per capita growth rate at a location with a mean temperature that favors the cold-adapted genotype. Colors show population size (the green to brown gradient depicting low to high population density), which influences the population growth rate through density dependence.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Our model describes how the local density of each genotype changes between years, which depends on temperature and genotype densities in the previous year. Transient temporal dynamics are computed directly from the model; these dynamics are the basis for the temporal statistical model. To create a spatial gradient, we simulated the equilibrium density of each genotype in a series of local populations experiencing different mean temperatures. The pattern of equilibrium densities across the mean annual temperature gradient is the basis for our spatial statistical model: cold sites will be dominated by the cold-adapted homozygous genotype, warm sites will be dominated by the heat-adapted homozygous genotype, and intermediate sites will be dominated by the heterozygous genotype (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4B</a>). The full description of the eco-evolutionary simulation model is provided in SI <a id="xref-sec-9-1" class="xref-app" href="#sec-9">Appendix C</a>.</p><p id="p-29">The spatial pattern shown in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4B</a> is the outcome of steady-state conditions. But at any one site, the population’s short-term response to temperature will be determined by the dominant genotype’s reaction norm (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4A</a>). For example, at a cold site dominated by the cold-adapted homozygous genotype, a warmer than average year would cause a decrease in population size due to decreases in fecundity (blue line in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4A</a>), even though the heat-adapted homozygote might perform optimally at that temperature. However, if warmer than normal conditions persist for many years, then genotype frequencies should shift, and the heat-adapted homozygote will compensate for the decreases of the cold-adapted genotype.</p><p id="p-30">To demonstrate these dynamics, we simulated a diploid annual plant population at a colder than average site. During the baseline period, the population is dominated by the cold-adapted genotype. We used the simulated data from this baseline period to fit a temporal statistical model (<a id="xref-sec-7-3" class="xref-app" href="#sec-7">Appendix A</a>) that predicts population growth rate as a function of annual temperature and population size (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4C</a>), assuming no knowledge of the underlying eco-evolutionary dynamics. We then imposed a period of warming, followed by a final period of higher stationary temperature (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5</a> top).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(Top) Simulated annual temperatures (grey) and expected temperature (black), which was used to make forecasts. (Bottom) Simulated population size and forecasts from the spatial, temporal and weighted statistical models." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1000258793" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(Top) Simulated annual temperatures (grey) and expected temperature (black), which was used to make forecasts. (Bottom) Simulated population size and forecasts from the spatial, temporal and weighted statistical models.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F5.medium.gif" width="421" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F5.medium.gif" width="421" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-31" class="first-child">(Top) Simulated annual temperatures (grey) and expected temperature (black), which was used to make forecasts. (Bottom) Simulated population size and forecasts from the spatial, temporal and weighted statistical models.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">With the onset of warming, the population crashed as the cold-adapted genotype decreased in abundance. Eventually, frequencies of the heterozygous genotype and the warm-adapted homozygous genotype began to increase and the population recovered (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5</a> bottom). The temporal statistical model (solid blue line in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5</a>) accurately predicted the impact of the initial warming trend, but eventually became too pessimistic, while the spatial statistical model (solid red line in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5</a>) did not handle the initial trend but accurately predicted the eventual, new steady state (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5</a> bottom).</p><p id="p-33">As in the community turnover example, we also fit a weighted average of predictions from the spatial and temporal statistical models, with the weights changing over time. This weighted model initially reflected the temporal model (decrease from <em>t</em> = 500 to <em>t</em> = 600), but then rapidly transitioned to reflect the spatial model (<em>t</em> ≥ 700). The rapid transition in the weighting term, <em>ω</em>, occurred during the period of most rapid change in genotype frequencies (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Fig. S-3</a>). The weighted model’s predictions look impressively accurate, but, as in the community turnover example, that is because we used the full, simulated time series to fit the weighting term. A true forecast would require an independent method to predict how the model weights shift over time.</p></div><div class="section" id="sec-5"><h2 class="">Discussion</h2><p id="p-34">Ecological forecasts are typically made using either a space-for-time substitution approach based on models fit to spatial data or using dynamic models fit to time-series data. Our results demonstrate that these two approaches can make very different predictions about the future state of ecological systems. Which approach provides the most accurate forecasts depends on the forecast-horizon. In our simulations, time-series approaches performed best for short forecast horizons, whereas models based on spatial data made more accurate forecasts at long horizons. In addition, our simulations demonstrate extended transitional periods during which neither the time-series or the spatial approach was effective on its own. The challenge is determining what is “short-term,” what is “long-term,” and how to handle the many forecasts we need in ecology which fall in between. We have proposed that a weighted combination of the time-series and space-for-time approaches may produce better forecasts at these intermediate forecast horizons.</p><p id="p-35">We designed our simulation studies to illustrate how the change in statistical model performance with increasing forecast horizon reflects differences in the types and scales of processes captured by spatial and temporal data sets. How could these hypotheses be tested with empirical data? The hypothesis that time-series models will be most effective for near-term forecasts already has empirical support, in the form of recent analyses of biodiversity forecasts at time ·, scales from one to ten years (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Harris et al., 2018</a>). The result should not be surprising, since local time-series data capture demographic processes, lagged effects, and responses of current assemblages to small changes in environmental conditions. In addition, the state of the system in the near future depends heavily on the current state. Since short-term forecasts do not typically require extrapolating into novel conditions, a model based on the historical range of variation which incorporates lags and accurate initial conditions is likely to be successful.</p><p id="p-36">Space-for-time modeling approaches for predicting long-term, steady-state outcomes of ecological change have also been tested empirically, primarily via hind-casting. Overall, the results are mixed: some tests show reasonable prediction of changes in community composition (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blois et al., 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Illán et al., 2014</a>) or species distributions (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Norberg et al., 2019</a>), supporting the hypothesis that datasets spanning spatial gradients capture the long-term outcome of interactions between fast processes and slower processes such as ecological and evolutionary selection, dispersal, and responses to large changes in the environment. Other attempts to validate predictions from space-for-time models have been discouraging (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Worth et al., 2014</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Illán et al., 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Davis et al., 2014</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brun et al., 2016</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Veloz et al., 2012</a>), indicating violations of model assumptions or effects of transient dynamics. However, predictions from the space-for-time approaches are rarely compared directly to predictions from time-series models (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Harris et al. 2018</a> but see <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Renwick et al. 2018</a>). We need more such comparisons to identify the appropriate modeling approach for different forecast horizons.</p><p id="p-37">The greatest empirical challenge will be testing our hypothesis that a weighted average of spatial and temporal statistical models will make the best forecasts at intermediate time scales. There are two problems: finding appropriate data and determining the model weights <em>a priori</em>. Many data sets have both a longitudinal and spatial dimension, but we could not think of one which also featured a clear ecological response to significant environmental change. Surely such datasets exist, and we hope researchers who work with them will test our proposed weighted model. Determining model weights may be more difficult. In our simulations, we fit the weights directly to the simulated data, which is impossible to do for actual forecasting when the future is unknown. We need new theory or empirical case studies in order to assign these weights <em>a priori</em>.</p><p id="p-38">Theory could explore the influence of different parameters on the rate at which slow processes begin to influence dynamics. The effects of some parameters are intuitive: in the community turnover example, increasing the fraction of dispersing individuals caused a more rapid shift in species composition and in model weights (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Fig. 6A</a>). Other parameters have less intuitive effects: we expected that increasing the temperature tolerance of genotypes in the evo-evolutionary example would accelerate the shift in model weights by maintaining higher genetic diversity. Our simulations showed the opposite effect, with wider tolerances slowing the shift in model weights (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Fig. 6B</a>), presumably by decreasing the strength of selection. Additional factors to consider include organism lifespans and the magnitude of directional environmental change relative to historical interannual variation.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The rate of change in the weight of the temporal forecast (y-axis) depends on (A) the fraction of propagules dispersing in the community turnover example and (B) on the temperature tolerance of genotypes, given by &#x3C3;T (larger values indicate wider thermal niches) in the eco-evolutionary example. Year 0 in these figures corresponds to the start of the temperature increase." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1000258793" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The rate of change in the weight of the temporal forecast (y-axis) depends on (A) the fraction of propagules dispersing in the community turnover example and (B) on the temperature tolerance of genotypes, given by &lt;em&gt;&#x3C3;&lt;sub&gt;T&lt;/sub&gt;&lt;/em&gt; (larger values indicate wider thermal niches) in the eco-evolutionary example. Year 0 in these figures corresponds to the start of the temperature increase.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F6.medium.gif" width="440" height="184"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F6.medium.gif" width="440" height="184"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <p id="p-39" class="first-child">The rate of change in the weight of the temporal forecast (y-axis) depends on (A) the fraction of propagules dispersing in the community turnover example and (B) on the temperature tolerance of genotypes, given by <em>σ<sub>T</sub></em> (larger values indicate wider thermal niches) in the eco-evolutionary example. Year 0 in these figures corresponds to the start of the temperature increase.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Empirical research could inform model weights by accumulating enough case studies to infer patterns in the weighting functions and guide applications in new systems. Developing rules of thumb would require testing many forecasts from both time-series and spatial models across a range of time-horizons. This effort may require a novel integration of typically disparate approaches, such as analyses of paleoecological data (e.g., <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Worth et al. 2014</a>), long-term observational (e.g., <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nice et al. 2019</a>) or experimental data (e.g., <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Silvertown et al. 2006</a>), and model systems with short-generation times (e.g., <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Good et al. 2017</a>).</p><p id="p-41">Given the challenges of determining model weights <em>a priori</em>, we should also pursue alternatives for intermediate forecast horizons. In the Introduction, we argued that fully process-based models are not feasible. However, a new class of statistical models offers a compromise between mechanistic detail and phenomenological feasibility. Spatiotemporal statistical modeling approaches are being developed to study patterns and processes of interest to ecological forecasters, such the spread of an invasive species or population status of a threatened species (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wikle, 2003</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Williams et al., 2017</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Schliep et al., 2018</a>). Because these models include both fast processes, such as births and deaths, and slower processes, such as colonization and extinction dynamics, they have the potential to make better predictions at intermediate forecast horizons than purely spatial or temporal models. However, these spatiotemporal models have rarely been used in a forecasting context, due to a combination of data limitation and computational challenges. Many data sources contain either spatial or temporal variation, but not both, and when spatiotemporal datasets are available they often involve irregular sampling, creating challenges for modeling. Fitting and generating predictions from spatiotemporal models is also computationally intensive, especially with large datasets (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">McDermott and Wikle, 2017</a>). Fortunately, thanks to large-scale monitoring efforts from remote sensing platforms, the National Ecological Observatory Network (<a href="https://www.neonscience.org/">https://www.neonscience.org/</a>), and community science projects (e.g., eBird), large scale spatiotemporal data is increasingly available. In addition, new methods for spatiotemporal forecasting are being developed that address existing computational challenges (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">McDermott and Wikle, 2017</a>), and access to high performance computing resources is increasingly common. Given these developments, future ecological forecasting efforts should explore spatiotemporal approaches and assess whether they improve predictions at intermediate time scales relative to traditional time-series or space-for-time approaches.</p><p id="p-42">Our results have important implications for the emerging field of ecological forecasting. First, they suggest that evaluating model performance at both short and long forecsat horizons will be essential as research on forecasting methods accelerates. Second, while single approaches may perform reasonably well for either short or long horizons, skillful predictions at intermediate forecast horizons may require a combination of information from spatial and temporal statistical models. Intermediate time horizons pose challenges in other forecasting contexts as well. Weather forecasts based on regional-scale meteorological models are very effective for forecasting a week to ten days in advance, but then become largely uninformative. Forecasting these intermediate scales has been challenging in meteorology and will likely be challenging in ecology as well. While the recent emphasis on near-term iterative forecasting (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Dietze et al., 2018</a>) is the logical and tractable starting point, we also need to build understanding and capacity for forecasting ecological dynamics across all forecast horizons of interest.</p><div id="sec-6" class="subsection"><h3>Data accessibility statement</h3><p id="p-43">The manuscript contains no original data. All computer code is available for reviewers in a zip archive that can be downloaded from Google Drive: <a href="https://drive.google.com/open?id=1aju04qtQvJmZG1mcpN-6hYBZILBhDfl2">https://drive.google.com/open?id=1aju04qtQvJmZG1mcpN-6hYBZILBhDfl2</a>. Upon acceptance of the manuscript, code will be archived at Zenodo.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-44">We thank Heather Lynch and Juan Manuel Morales for comments that improved early drafts of the paper. PBA was supported by the National Science Foundation (DEB-1927282) and the Utah Agriculture Experiment Station (grant 1322). EPW was supported by the Gordon and Betty Moore Foundation’s Data-Driven Discovery Initiative (GBMF4563).</p></div><div class="section app" id="app-1"><h2>Appendices</h2><div class="section" id="sec-7"><h3>A Spatial, temporal and spatial-temporal-weighted models</h3><p id="p-45">The two simulation models in the main text describe how population size, <em>N</em>(<em>x</em>, <em>t</em>), at location <em>x</em> changes over time (<em>t</em>). We assume that the temperature, <em>K</em>(<em>x</em>, <em>t</em>), at each location can vary in time and space. To forecast the dynamics generated by these simulations models, we fit a series of statistical models.</p><p id="p-46">The spatial model, which we refer to as <em>S</em>, is a quadratic regression of the mean long-term population density at a location <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-1.gif"/></noscript></span></span> against the mean temperature at that location <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-2.gif"/></noscript></span></span>. The quadratic term describes the unimodal relationship between <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-3.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-4.gif"/></noscript></span></span>. The spatial statistical model is
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-7.gif"/></noscript></span></span>
</p><p id="p-47">The temporal model, which we call <em>T</em>, starts with a time-series of “observed” population sizes, or total biomasses, at one location, <em>N</em>(<em>t</em>), for <em>t</em> = 1…<em>n</em> (the spatial index is suppressed because we only focus on one location at a time). In the community turnover example, we fit the following regression, which predicts biomass at time <em>t</em> + 1 as a function of biomass (<em>N</em>(<em>t</em>)) and annual temperature (<em>K</em>(<em>t</em>)) at time <em>t</em>,
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-8.gif"/></noscript></span></span>
</p><p id="p-48">In the eco-evolutionary example, the response variable is the log of the population growth rate. The regression is
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-9.gif"/></noscript></span></span>
</p><p id="p-49">This version of the temporal model returns a per capita growth rate on the log scale. To predict population size at the next time step, we exponentiate the growth rate and multiply it by the current population size: exp(<em>T</em>(<em>N</em>(<em>t</em>), <em>K</em>(<em>t</em>)))<em>N</em>(<em>t</em>).</p><p id="p-50">The weighted model is a weighted average of predictions from the spatial and temporal models, with the weights changing as a function of time, here expressed as the forecast horizon. The weights change as a function of the square root of the forecast horizon, to allow rapid shifts in the model weights.
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-10.gif"/></noscript></span></span>
</p><p id="p-51">For the community turnover example, the predicted biomass from the weighted model is:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-11.gif"/></noscript></span></span>
</p><p id="p-52">Again, we suppress the spatial subscript (<em>x</em>) here because we are focused on densities at just one location. For the eco-evolutionary example, the predicted population size from the weighted model is:
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-12.gif"/></noscript></span></span>
</p><p id="p-53">We used the <kbd>optim</kbd> function to estimate the <em>β<sup>W</sup></em>s that minimize the sum of squared errors, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/inline-graphic-5.gif"/></noscript></span></span>.</p><p id="p-54">In the main text, we show the point forecasts but not the uncertainty around the forecasts. After exploring that uncertainty, we decided that presenting it would be misleading. For the spatial and, especially, the temporal statistical models, the uncertainty is unrealistically low, because the models are estimated with very large samples sizes from the simulations. Furthermore, the simulations do not include noise; the only reason there is any uncertainty is because the statistical models are slightly mis-specified with respect to the process models. Showing uncertainty for the weighted model would be even less meaningful, because it is not a true, out-of-sample forecast (parameters are fit directly to the observations for which we make predictions). The R code to compute uncertainties for the spatial and temporal forecasts is available on our Github repository (<a href="https://github.com/pbadler/space-time-forecast">https://github.com/pbadler/space-time-forecast</a>), but is commented out.</p></div><div class="section" id="sec-8"><h3>B Description of the meta-community model</h3><p id="p-55">Alexander et al. (2018) developed a meta-community model to represent dynamics of local communities arrayed along a one-dimensional elevation gradient, as influenced by three main processes: temperature-dependent growth, competition, and dispersal. Here we adapt their notation to be consistent our own.</p><p id="p-56">The population size of species <em>i</em> in cell <em>x</em> at time <em>t</em> + 1, <em>N<sub>i</sub></em> (<em>x</em>, <em>t</em> + 1), is computed in two steps. The first step accounts for changes in local population sizes due to dispersal. In each local community, all species export a fraction (<em>d</em>) of their local population to the two adjacent communities in the 1-dimensional landscape:
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-13.gif"/></noscript></span></span>
</p><p id="p-57">Here <em>N</em>′ distinguishes the post-dispersal population size from the pre-dispersal population size. The second step computes population growth, taking into account competition:
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-14.gif"/></noscript></span></span>
</p><p id="p-58">In the absence of competition, the growth rate (<em>g<sub>i</sub></em>) is determined by the difference between the temperature at site <em>x</em> (<em>K</em>(<em>x</em>)) and the focal species’ minimum temperature tolerance, <em>Kmin<sub>i</sub></em>, the lowest temperature at which a species can maintain a positive growth rate. Growth is further reduced by intraspecific and interspecific competition, parameterized by <em>c<sub>i</sub></em> and <em>l<sub>i</sub></em>. All species are assigned the same value of <em>c<sub>i</sub></em>, which represents an additional effect of intraspecific competition on top of interspecific competition. This stabilizes coexistence, since every species will exert stronger intra- than interspecific competition. However, values of <em>l</em> vary among species to create a trade-off between growth rates and competitive ability versus low temperature tolerance: fast-growing species (high <em>g<sub>i</sub></em>) are more tolerant of interspecific competition (low <em>l<sub>i</sub></em>) but are more limited by temperature (high <em>Kmin<sub>i</sub></em>).</p></div><div class="section" id="sec-9"><h3>C Description of the eco-evolutionary annual plant model</h3><div id="sec-10" class="subsection"><h4>Haploid Model</h4><p id="p-59">Begin with a haploid model that describes the number of seeds present in a seed bank. <em>N<sub>i,t</sub></em> is the number of seeds of species <em>i</em> at time <em>t</em>. The model is
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-15.gif"/></noscript></span></span>
where <em>g<sub>i</sub></em>(<em>K</em>(<em>t</em>)) is the probability of germination, <em>K</em>(<em>t</em>) is the temperature at time <em>t</em>, <em>s<sub>i</sub></em> is the seed survival probability for species <em>i</em>, and <em>λ<sub>i</sub></em> is the seed production rate per plant. Below we refer to the <em>α<sub>ij</sub></em> as intra- and inter-genotype competition coefficients.</p></div><div id="sec-11" class="subsection"><h4>Diploid Model</h4><p id="p-60">Consider a one-species diploid model. The genotypes are denoted by <em>AA</em>, <em>Aa</em>, and <em>aa</em>. The number of each genotypes at time <em>t</em> is <em>N<sub>AA</sub></em>(<em>t</em>), <em>N<sub>Aa</sub></em>(<em>t</em>), and <em>N<sub>aa</sub></em>(<em>t</em>). The germination rates for each genotype are <em>g<sub>AA</sub></em>(<em>K</em>(<em>t</em>)), <em>g<sub>Aa</sub></em>(<em>K</em>(<em>t</em>)), and <em>g<sub>aa</sub></em>(<em>K</em>(<em>t</em>)). The seed survival probability and seed production rate for genotype <em>AA</em> are <em>s<sub>AA</sub></em> and <em>λ<sub>AA</sub></em>, respectively. The analogous parameters for the other genotypes are similarly denoted. The competition coefficients are denoted by <em>α<sub>i,j</sub></em>, e.g., <em>α<sub>AA,AA</sub></em> or <em>α<sub>AA,Aa</sub></em>. Throughout we assume that gametes mix randomly in the population.</p><p id="p-61">First consider the case where the competition coefficients are zero (<em>α<sub>i,j</sub></em> = 0). Let <em>T</em> denote the total number of gamete-pairs produced in a given year,
<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-16.gif"/></noscript></span></span>
</p><p id="p-62">The first term is the number of gamete-pairs produced by <em>AA</em> individuals. The second and third terms are the numbers of gamete-pairs produced by <em>Aa</em> and <em>aa</em> individuals, respectively. The proportion of <em>A</em> gametes (<em>ϕ<sub>A</sub></em>) and the proportion of <em>a</em> gametes (<em>ϕ<sub>a</sub></em>) are given by
<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-17.gif"/></noscript></span></span>
</p><p id="p-63">Note that the <em>T</em> in the denominator of <em>ϕ<sub>A</sub></em> shows up because we are computing proportions. Combining all of these we get the dynamics for each genotype,
<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-18.gif"/></noscript></span></span>
</p><p id="p-64">Now consider the case where the competition coefficients are non-zero (<em>α<sub>i,j</sub></em> ≠ 0). Including competition changes the way in which we compute <em>T</em>, <em>ϕ<sub>A</sub></em>, and <em>ϕ<sub>a</sub></em>. Specifically, because the total number of seeds produced per year by each genotypes is reduced based on intra- and intergenotype competition, the total number of gamete-pairs becomes
<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-19.gif"/></noscript></span></span>
</p><p id="p-65">The first line is the number of gamete-pairs produced by <em>AA</em> individuals after accounting for the effects of competition. The second and third lines are the numbers of gamete-pairs produced by <em>Aa</em> and <em>aa</em> individuals, respectively. The proportions of <em>A</em> gametes and <em>a</em> gametes are
<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-20.gif"/></noscript></span></span>
</p><p id="p-66">Combining all of this results in the same model as above,
<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/13/807057/embed/graphic-21.gif"/></noscript></span></span>
but the definitions of <em>T</em>, <em>ϕ<sub>A</sub></em>, and <em>ϕ<sub>a</sub></em> are given by <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">equations (13)</a> and <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">(14)</a>.</p></div></div><div class="section" id="sec-12"><h3>D Supplementary Figures</h3><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(Results for total biomass from the community turnover model. Blue points show mean total biomass during the baseline period at locations across the temperature gradient, and the blue line shows predictions from the spatial model. Red points show annual total biomass during the baseline period as a function of annual temperature at the central site on the gradient. The red line shows predictions from the temporal model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1000258793" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(Results for total biomass from the community turnover model. Blue points show mean total biomass during the baseline period at locations across the temperature gradient, and the blue line shows predictions from the spatial model. Red points show annual total biomass during the baseline period as a function of annual temperature at the central site on the gradient. The red line shows predictions from the temporal model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S-1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F7.medium.gif" width="440" height="407"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S-1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F7.medium.gif" width="440" height="407"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S-1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S-1:</span> <p id="p-67" class="first-child">(Results for total biomass from the community turnover model. Blue points show mean total biomass during the baseline period at locations across the temperature gradient, and the blue line shows predictions from the spatial model. Red points show annual total biomass during the baseline period as a function of annual temperature at the central site on the gradient. The red line shows predictions from the temporal model.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results for total biomass from the community turnover model. (A) Simulated annual temperatures (grey) and expected temperature (black), which was used to make forecasts, at the focal site. (B) Simulated total biomass and forecasts from the spatial, temporal and weighted models. (C) Simulated changes in biomass of all species (grey) at the focal site in the metacommunity model, and the weight given to the temporal model for total biomass (blue). Year 1000 in this figure corresponds to the start of the temperature increase." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1000258793" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Results for total biomass from the community turnover model. (A) Simulated annual temperatures (grey) and expected temperature (black), which was used to make forecasts, at the focal site. (B) Simulated total biomass and forecasts from the spatial, temporal and weighted models. (C) Simulated changes in biomass of all species (grey) at the focal site in the metacommunity model, and the weight given to the temporal model for total biomass (blue). Year 1000 in this figure corresponds to the start of the temperature increase.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S-2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F8.medium.gif" width="246" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S-2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F8.medium.gif" width="246" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S-2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S-2:</span> <p id="p-68" class="first-child">Results for total biomass from the community turnover model. (A) Simulated annual temperatures (grey) and expected temperature (black), which was used to make forecasts, at the focal site. (B) Simulated total biomass and forecasts from the spatial, temporal and weighted models. (C) Simulated changes in biomass of all species (grey) at the focal site in the metacommunity model, and the weight given to the temporal model for total biomass (blue). Year 1000 in this figure corresponds to the start of the temperature increase.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulated shifts in genotype abundances, and the model weighting term, &#x3C9;, during the warming phase and the following stationary temperature phase. Year 0 in this figure corresponds to the start of the temperature increase." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1000258793" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Simulated shifts in genotype abundances, and the model weighting term, &lt;em&gt;&#x3C9;&lt;/em&gt;, during the warming phase and the following stationary temperature phase. Year 0 in this figure corresponds to the start of the temperature increase.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S-3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F9.medium.gif" width="440" height="345"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S-3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F9.medium.gif" width="440" height="345"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S-3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/13/807057/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S-3:</span> <p id="p-69" class="first-child">Simulated shifts in genotype abundances, and the model weighting term, <em>ω</em>, during the warming phase and the following stationary temperature phase. Year 0 in this figure corresponds to the start of the temperature increase.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><em>Statement of authorship:</em> PBA and EPW designed the study, PBA and MHC built the models, and PBA analyzed the models. PBA wrote the first draft of the manuscript, and all authors contributed substantially to revisions.</p></li><li class="fn-update fn-group-summary-of-updates" id="fn-2"><p id="p-4">Minor changes in terminology, some restructuring of Introduction and Discussion</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Lags in the response of mountain plant communities to climate change</span>. – <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D24%26rft.spage%253D563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.2" data-doi="10.1073/pnas.1220228110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blois</span>,  <span class="cit-name-given-names">J. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Space can substitute for time in predicting climate-change effects on biodiversity</span>. – <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">9374</span>–<span class="cit-lpage">9379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1220228110%26rft_id%253Dinfo%253Apmid%252F23690569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIzLzkzNzQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xMy84MDcwNTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brun</span>,  <span class="cit-name-given-names">P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2016</span>. <span class="cit-article-title">The predictive skill of species distribution models for plankton in a changing climate</span>. – <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3170</span>–<span class="cit-lpage">3181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D22%26rft.spage%253D3170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chevalier</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Knape</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2020</span>. <span class="cit-article-title">The cost of complexity in forecasts of population abundances is reduced but not eliminated by borrowing information across space using a hierarchical 9 approach</span>. – <abbr class="cit-jnl-abbrev">Oikos in press</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.5" data-doi="10.1126/science.293.5530.657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clark</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2001</span>. <span class="cit-article-title">Ecological forecasts: An emerging imperative</span>. – <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">293</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">660</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DClark%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D293%26rft.issue%253D5530%26rft.spage%253D657%26rft.epage%253D660%26rft.atitle%253DEcological%2BForecasts%253A%2BAn%2BEmerging%2BImperative%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.293.5530.657%26rft_id%253Dinfo%253Apmid%252F11474103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTMvNTUzMC82NTciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMi8xMy84MDcwNTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.6" data-doi="10.1890/10-0780.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dalgleish</span>,  <span class="cit-name-given-names">H. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Climate influences the demography of three dominant sagebrush steppe plants</span>. – <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DDalgleish%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D85%26rft.atitle%253DClimate%2Binfluences%2Bthe%2Bdemography%2Bof%2Bthree%2Bdominant%2Bsagebrush%2Bsteppe%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F10-0780.1%26rft_id%253Dinfo%253Apmid%252F21560678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/10-0780.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21560678&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F807057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">E. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Ecological niche models of mammalian glacial refugia show consistent bias</span>. – <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D37%26rft.spage%253D1133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dietze</span>,  <span class="cit-name-given-names">M. C.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Prediction in ecology: a first-principles framework</span>. – <abbr class="cit-jnl-abbrev">Ecological Applications</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2048</span>–<span class="cit-lpage">2060</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.volume%253D27%26rft.spage%253D2048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.9" data-doi="10.1073/pnas.1710231115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dietze</span>,  <span class="cit-name-given-names">M. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Iterative near-term ecological forecasting: Needs, opportunities, and challenges</span>. – <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">1424</span>–<span class="cit-lpage">1432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1710231115%26rft_id%253Dinfo%253Apmid%252F29382745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE1LzcvMTQyNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzEzLzgwNzA1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.10" data-doi="10.1146/annurev.ecolsys.110308.120159"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elith</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Leathwick</span>,  <span class="cit-name-given-names">J. R.</span></span> <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Species distribution models: ecological explanation and prediction across space and time</span>. – <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D40%26rft.spage%253D677%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.110308.120159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.110308.120159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000272455700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.11" data-doi="10.1038/nature24287"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Good</span>,  <span class="cit-name-given-names">B. H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">The dynamics of molecular evolution over 60,000 generations</span>. – <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">551</span>: <span class="cit-fpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24287%26rft_id%253Dinfo%253Apmid%252F29045390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature24287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29045390&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F807057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Forecasting biodiversity in breeding birds using best practices</span>. – <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e4278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D6%26rft.spage%253De4278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hefley</span>,  <span class="cit-name-given-names">T. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">When mechanism matters: Bayesian forecasting using models of ecological diffusion</span>. – <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D20%26rft.spage%253D640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.14" data-doi="10.1038/nature09670"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>,  <span class="cit-name-given-names">A. A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sgro</span>,  <span class="cit-name-given-names">C. M.</span></span> <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Climate change and evolutionary adaptation</span>. – <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D470%26rft.issue%253D7335%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DClimate%2Bchange%2Band%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09670%26rft_id%253Dinfo%253Apmid%252F21350480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21350480&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F807057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287652900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Houlahan</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">The priority of prediction in ecological understanding</span>. – <abbr class="cit-jnl-abbrev">Oikos</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D126%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hyndman</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Athanasopoulos</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Forecasting: Principles and Practice</span>. – <abbr class="cit-jnl-abbrev">OTexts</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Illán</span>,  <span class="cit-name-given-names">J. G.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Precipitation and winter temperature predict long-term range-scale abundance changes in Western North American birds</span>. – <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3351</span>–<span class="cit-lpage">3364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D20%26rft.spage%253D3351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kleinhesselink</span>,  <span class="cit-name-given-names">A. R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Adler</span>,  <span class="cit-name-given-names">P. B.</span></span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">The response of big sagebrush (Artemisia tridentata) to interannual climate variation changes across its range</span>. – <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">1139</span>–<span class="cit-lpage">1149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D99%26rft.spage%253D1139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.19" data-doi="10.2307/1941874"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauenroth</span>,  <span class="cit-name-given-names">W. K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sala</span>,  <span class="cit-name-given-names">O. E.</span></span> <span class="cit-pub-date">1992</span>. <span class="cit-article-title">Long-term forage production of North American short-grass steppe</span>. – <abbr class="cit-jnl-abbrev">Ecological Applications</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.volume%253D2%26rft.spage%253D397%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1941874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1941874&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992JV60500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.20" data-doi="10.2307/1941447"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Levin</span>,  <span class="cit-name-given-names">S. A.</span></span> <span class="cit-pub-date">1992</span>. <span class="cit-article-title">The problem of pattern and scale in ecology: the robert h. macarthur award lecture</span>. – <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1943</span>–<span class="cit-lpage">1967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D73%26rft.spage%253D1943%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1941447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1941447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1992KB84300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McDermott</span>,  <span class="cit-name-given-names">P. L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wikle</span>,  <span class="cit-name-given-names">C. K.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">An ensemble quadratic echo state network for non-linear spatio-temporal forecasting</span>. – <abbr class="cit-jnl-abbrev">Stat</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStat%26rft.volume%253D6%26rft.spage%253D315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mouquet</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">REVIEW: Predictive ecology in a changing world</span>. – <abbr class="cit-jnl-abbrev">Journal of Applied Ecology</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1293</span>–<span class="cit-lpage">1310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BEcology%26rft.volume%253D52%26rft.spage%253D1293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nice</span>,  <span class="cit-name-given-names">C. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">Extreme heterogeneity of population response to climatic variation and the limits of prediction</span>. – <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2127</span>–<span class="cit-lpage">2136</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D25%26rft.spage%253D2127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Norberg</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span>. <span class="cit-article-title">A comprehensive evaluation of predictive performance of 33 species distribution models at species and community levels</span>. – <abbr class="cit-jnl-abbrev">Ecological Monographs</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">e01370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BMonographs%26rft.volume%253D89%26rft.spage%253De01370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Renwick</span>,  <span class="cit-name-given-names">K. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Multi-model comparison highlights consistency in predicted effect of warming on a semi-arid shrub</span>. – <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D24%26rft.spage%253D424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-807057v2.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenzweig</span>,  <span class="cit-name-given-names">M. L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">1995</span>. <span class="cit-source">Species diversity in space and time</span>. – <span class="cit-publ-name">Cambridge University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.27" data-doi="10.2307/1943158"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sala</span>,  <span class="cit-name-given-names">O. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">1988</span>. <span class="cit-article-title">Primary production of the central grassland region of the United States</span>. – <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D69%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1943158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1943158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1988L798500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.28" data-doi="10.1111/geb.12666"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schliep</span>,  <span class="cit-name-given-names">E. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Joint species distribution modelling for spatio-temporal occurrence and ordinal abundance data</span>. – <abbr class="cit-jnl-abbrev">Global Ecology and Biogeography</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BEcology%2Band%2BBiogeography%26rft.volume%253D27%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgeb.12666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/geb.12666&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silvertown</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2006</span>. <span class="cit-article-title">The Park Grass Experiment 1856-2006: Its contribution to ecology</span>. – <abbr class="cit-jnl-abbrev">Journal of Ecology</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEcology%26rft.volume%253D94%26rft.spage%253D814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.30" data-doi="10.1098/rspb.2011.2367"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Urban</span>,  <span class="cit-name-given-names">M. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">On a collision course: competition and dispersal differences create no-analogue communities and cause extinctions during climate change</span>. – <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society of London B: Biological Sciences</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">2072</span>–<span class="cit-lpage">2080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BLondon%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2011.2367%26rft_id%253Dinfo%253Apmid%252F22217718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2011.2367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22217718&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F13%2F807057.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.31" data-doi="10.1111/j.1365-2486.2011.02635.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Veloz</span>,  <span class="cit-name-given-names">S. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">No-analog climates and shifting realized niches during the late quaternary: implications for 21st-century predictions by species distribution models</span>. – <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1698</span>–<span class="cit-lpage">1713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D18%26rft.spage%253D1698%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2486.2011.02635.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2486.2011.02635.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000302543500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.32" data-doi="10.1111/j.1600-0706.2014.00916.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">E. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Complexity is costly: a meta-analysis of parametric and non-parametric methods for short-term population forecasting</span>. – <abbr class="cit-jnl-abbrev">Oikos</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">661</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D123%26rft.spage%253D652%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1600-0706.2014.00916.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1600-0706.2014.00916.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000335721400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.33" data-doi="10.1890/0012-9658(2003)084[1382:HBMFPT]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wikle</span>,  <span class="cit-name-given-names">C. K.</span></span> <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Hierarchical Bayesian Models for Predicting the Spread of Ecological Processes</span>. – <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">1382</span>–<span class="cit-lpage">1394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D84%26rft.spage%253D1382%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9658%25282003%2529084%255B1382%253AHBMFPT%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9658(2003)084[1382:HBMFPT]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000184451900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.34" data-doi="10.1890/1540-9295(2007)5[475:NCNCAE]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">J. W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">S. T.</span></span> <span class="cit-pub-date">2007</span>. <span class="cit-article-title">Novel climates, no-analog communities, and ecological surprises</span>. – <abbr class="cit-jnl-abbrev">Frontiers in Ecology and the Environment</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BEcology%2Band%2Bthe%2BEnvironment%26rft.volume%253D5%26rft.spage%253D475%26rft_id%253Dinfo%253Adoi%252F10.1890%252F1540-9295%25282007%25295%255B475%253ANCNCAE%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/1540-9295(2007)5[475:NCNCAE]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000250659900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">P. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">An integrated data model to estimate spatiotemporal occupancy, abundance, and colonization dynamics</span>. – <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">328</span>–<span class="cit-lpage">336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D98%26rft.spage%253D328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-807057v2.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Worth</span>,  <span class="cit-name-given-names">J. R. P.</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Environmental niche modelling fails to predict Last Glacial Maximum refugia: niche shifts, microrefugia or incorrect palaeoclimate estimates?</span> – <abbr class="cit-jnl-abbrev">Global Ecology and Biogeography</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1186</span>–<span class="cit-lpage">1197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BEcology%2Band%2BBiogeography%26rft.volume%253D23%26rft.spage%253D1186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/807057v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F807057v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/779561v2" title="Quantifying geographic accessibility to improve efficiency of entomological monitoring" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 13, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/807057v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/807057v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/807057v2" class="active" data-icon-position="" data-hide-link-title="0">Matching the forecast horizon with the relevant spatial and temporal processes and data sources</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1036436" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form--T0j6t7UK1TOXCLnCnxDaGIrkad-vwT9DNSkb2OZe54" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1036436" id="node1036436--2" data-pisa="biorxiv;807057v2" data-pisa-master="biorxiv;807057" data-seqnum="1036436" data-apath="/biorxiv/early/2019/12/13/807057.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Matching the forecast horizon with the relevant spatial and temporal processes and data sources</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Peter B.</span> <span class="nlm-surname">Adler</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ethan P.</span> <span class="nlm-surname">White</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael H.</span> <span class="nlm-surname">Cortez</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">807057; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/807057 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/807057v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/807057v2&amp;title=Matching%20the%20forecast%20horizon%20with%20the%20relevant%20spatial%20and%20temporal%20processes%20and%20data%20sources" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/807057v2&amp;title=Matching%20the%20forecast%20horizon%20with%20the%20relevant%20spatial%20and%20temporal%20processes%20and%20data%20sources" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/807057v2&amp;text=Matching%20the%20forecast%20horizon%20with%20the%20relevant%20spatial%20and%20temporal%20processes%20and%20data%20sources" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/807057v2&amp;title=Matching%20the%20forecast%20horizon%20with%20the%20relevant%20spatial%20and%20temporal%20processes%20and%20data%20sources" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/807057v2&amp;t=Matching%20the%20forecast%20horizon%20with%20the%20relevant%20spatial%20and%20temporal%20processes%20and%20data%20sources" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/807057v2&amp;title=Matching%20the%20forecast%20horizon%20with%20the%20relevant%20spatial%20and%20temporal%20processes%20and%20data%20sources" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/807057v2&amp;title=Matching%20the%20forecast%20horizon%20with%20the%20relevant%20spatial%20and%20temporal%20processes%20and%20data%20sources" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1036436" id="node1036436--3" data-pisa="biorxiv;807057v2" data-pisa-master="biorxiv;807057" data-seqnum="1036436" data-apath="/biorxiv/early/2019/12/13/807057.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Matching the forecast horizon with the relevant spatial and temporal processes and data sources</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Peter B.</span> <span class="nlm-surname">Adler</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Ethan P.</span> <span class="nlm-surname">White</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Michael H.</span> <span class="nlm-surname">Cortez</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">807057; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/807057 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Matching%20the%20forecast%20horizon%20with%20the%20relevant%20spatial%20and%20temporal%20processes%20and%20data%20sources&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Adler%2C+Peter+B.&amp;rft.au=White%2C+Ethan+P.&amp;rft.au=Cortez%2C+Michael+H."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1036436/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1036436/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1036436/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1036436/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1036436/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1036436/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1036436/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1036436/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1036436/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1036436/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1036436/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1036436/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/807057v2&amp;count=horizontal&amp;via=&amp;text=Matching%20the%20forecast%20horizon%20with%20the%20relevant%20spatial%20and%20temporal%20processes%20and%20data%20sources&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/807057v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/807057v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/807057v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
