<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/embed/graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/03/28/586909.atom" />
<meta name="HW.pisa" content="biorxiv;586909v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Estimating spatially and temporally complex range dynamics when detection is imperfect" />
<meta name="DC.Identifier" content="10.1101/586909" />
<meta name="DC.Date" content="2019-03-28" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This article is a US Government work. It is not subject to copyright under 17 USC 105 and is also made available for use under a CC0 license" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Species distributions are determined by the interaction of multiple biotic and abiotic factors, which produces complex spatial and temporal patterns of occurrence. As habitats and climate change due to anthropogenic activities, there is a need to develop species distribution models that can quantify these complex range dynamics. In this paper, we develop a dynamic occupancy model that uses a spatial generalized additive model to estimate non-linear spatial variation in occupancy not accounted for by environmental covariates. The model is flexible and can accommodate data from a range of sampling designs that provide information about both occupancy and detection probability. Output from the model can be used to create distribution maps and to estimate indices of temporal range dynamics. We demonstrate the utility of this approach by modeling long-term range dynamics of 10 eastern North American birds using data from the North American Breeding Bird Survey. We anticipate this framework will be particularly useful for modeling species’ distributions over large spatial scales and for quantifying range dynamics over long temporal scales." />
<meta name="DC.Contributor" content="Clark S. Rushing" />
<meta name="DC.Contributor" content="J. Andrew Royle" />
<meta name="DC.Contributor" content="David J. Ziolkowski" />
<meta name="DC.Contributor" content="Keith L. Pardieck" />
<meta name="article:published_time" content="2019-03-28" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Estimating spatially and temporally complex range dynamics when detection is imperfect" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Species distributions are determined by the interaction of multiple biotic and abiotic factors, which produces complex spatial and temporal patterns of occurrence. As habitats and climate change due to anthropogenic activities, there is a need to develop species distribution models that can quantify these complex range dynamics. In this paper, we develop a dynamic occupancy model that uses a spatial generalized additive model to estimate non-linear spatial variation in occupancy not accounted for by environmental covariates. The model is flexible and can accommodate data from a range of sampling designs that provide information about both occupancy and detection probability. Output from the model can be used to create distribution maps and to estimate indices of temporal range dynamics. We demonstrate the utility of this approach by modeling long-term range dynamics of 10 eastern North American birds using data from the North American Breeding Bird Survey. We anticipate this framework will be particularly useful for modeling species’ distributions over large spatial scales and for quantifying range dynamics over long temporal scales.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;586909v1" />
<meta name="citation_id" content="586909v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/586909v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/586909v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/586909v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909.full.pdf" />
<meta name="citation_doi" content="10.1101/586909" />
<meta name="citation_num_pages" content="25" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="586909" />
<meta name="citation_author" content="Clark S. Rushing" />
<meta name="citation_author_institution" content="U.S. Geological Survey, Patuxent Wildlife Research Center" />
<meta name="citation_author_email" content="crushing@usgs.gov" />
<meta name="citation_author" content="J. Andrew Royle" />
<meta name="citation_author_institution" content="U.S. Geological Survey, Patuxent Wildlife Research Center" />
<meta name="citation_author" content="David J. Ziolkowski" />
<meta name="citation_author_institution" content="U.S. Geological Survey, Patuxent Wildlife Research Center" />
<meta name="citation_author" content="Keith L. Pardieck" />
<meta name="citation_author_institution" content="U.S. Geological Survey, Patuxent Wildlife Research Center" />
<meta name="citation_reference" content="Araújo, M. B., and M. Luoto. 2007. The importance of biotic interactions for modelling species distributions under climate change. Global Ecology and Biogeography 16:743–753." />
<meta name="citation_reference" content="Barbet-Massin, M., and W. Jetz. 2014. A 40-year, continent-wide, multispecies assessment of relevant climate predictors for species distribution modelling. Diversity and Distributions 20:1285–1295." />
<meta name="citation_reference" content="Bled, F., J. D. Nichols, and R. Altwegg. 2013. Dynamic occupancy models for analyzing species’ range dynamics across large geographic scales. Ecology and Evolution 3:4896–4909." />
<meta name="citation_reference" content="Bled, F., J. A. Royle, and E. Cam. 2011. Hierarchical modeling of an invasive spread: The Eurasian Collared-Dove streptopelia decaocto in the United States. Ecological Applications 21:290–302." />
<meta name="citation_reference" content="Brooks, S. P., E. A. Catchpole, and B. J. Morgan. 2000. Bayesian animal survival estimation. Statistical Science:357–376." />
<meta name="citation_reference" content="Carpenter, B., A. Gelman, M. D. Hoffman, D. Lee, B. Goodrich, M. Betancourt, M. Brubaker, J. Guo, P. Li, and A. Riddell. 2017. Stan: A probabilistic programming language. Journal of Statistical Software 76." />
<meta name="citation_reference" content="Clement, M. J., J. E. Hines, J. D. Nichols, K. L. Pardieck, and D. J. Ziolkowski. 2016. Estimating indices of range shifts in birds using dynamic models when detection is imperfect. Global Change Biology 22:3273–3285." />
<meta name="citation_reference" content="Dellaportas, P., J. J. Forster, and I. Ntzoufras. 2002. On Bayesian model and variable selection using MCMC. Statistics and Computing 12:27–36." />
<meta name="citation_reference" content="Elith, J., M. Kearney, and S. Phillips. 2010. The art of modelling range-shifting species. Methods in Ecology and Evolution 1:330–342." />
<meta name="citation_reference" content="Fiske, I., and R. Chandler. 2011. Ummarked: An R package for fitting hierarchical models of wildlife occurrence and abundance. Journal of Statistical Software 43:1–23." />
<meta name="citation_reference" content="Guélat, J., and M. Kéry. 2018. Effects of spatial autocorrelation and imperfect detection on species distribution models. Methods in Ecology and Evolution:n/a–n/a." />
<meta name="citation_reference" content="Guillera-Arroita, G. 2017. Modelling of species distributions, range dynamics and communities under imperfect detection: Advances, challenges and opportunities. Ecography 40." />
<meta name="citation_reference" content="Harris, I., P. Jones, T. Osborn, and D. Lister. 2014. Updated high-resolution grids of monthly climatic observations–the CRU TS3. 10 Dataset. International Journal of Climatology 34:623–642." />
<meta name="citation_reference" content="Hefley, T. J., K. M. Broms, B. M. Brost, F. E. Buderman, S. L. Kay, H. R. Scharf, J. R. Tipton, P. J. Williams, and M. B. Hooten. 2017. The basis function approach for modeling autocorrelation in ecological data. Ecology 98:632–646." />
<meta name="citation_reference" content="Hijmans, R., S. Phillips, J. Leathwick, and J. Elith. 2016. Dismo: Species distribution modeling. R package ver. 1.0-15." />
<meta name="citation_reference" content="Hill, J. K., C. D. Thomas, and B. Huntley. 1999. Climate and habitat availability determine 20th century changes in a butterfly’s range margin. Proceedings of the Royal Society of London B: Biological Sciences 266:1197–1206." />
<meta name="citation_reference" content="Hines, J. E. 2006. Program presence. See http://www.mbrpwrc.usgs.gov/software/doc/presence/presence.html." />
<meta name="citation_reference" content="Hines, J. E., J. D. Nichols, and J. A. Collazo. 2014. Multiseason occupancy models for correlated replicate surveys. Methods in Ecology and Evolution 5:583–591." />
<meta name="citation_reference" content="Hines, J. E., J. D. Nichols, J. A. Royle, D. I. MacKenzie, A. Gopalaswamy, N. Kumar, and K. Karanth. 2010. Tigers on trails: Occupancy modeling for cluster sampling. Ecological Applications 20:1456–1466." />
<meta name="citation_reference" content="Johnson, D. S., P. B. Conn, M. B. Hooten, J. C. Ray, and B. A. Pond. 2013. Spatial occupancy models for large data sets. Ecology 94:801–808." />
<meta name="citation_reference" content="Kearney, M., and W. Porter. 2009. Mechanistic niche modelling: Combining physiological and spatial data to predict species ranges. Ecology Letters 12:334–350." />
<meta name="citation_reference" content="Kellner, K. 2015. jagsUI: A wrapper around rjags to streamline JAGS analyses. R package version 1." />
<meta name="citation_reference" content="Kéry, M. 2011. Towards the modelling of true species distributions. Journal of Biogeography 38:617–618." />
<meta name="citation_reference" content="Kéry, M., and J. A. Royle. 2015. Applied hierarchical modeling in ecology: Analysis of distribution, abundance and species richness in R and BUGS: Volume 1: Prelude and static models. Academic Press." />
<meta name="citation_reference" content="Kuo, L., and B. Mallick. 1998. Variable selection for regression models. The Indian Journal of Statistics, Series B 60:65–81." />
<meta name="citation_reference" content="Link, W. A., and J. R. Sauer. 2002. A hierarchical analysis of population change with application to Cerulean Warblers. Ecology 83:2832–2840." />
<meta name="citation_reference" content="Lunn, D. J., A. Thomas, N. Best, and D. Spiegelhalter. 2000. WinBUGS-a Bayesian modelling framework: Concepts, structure, and extensibility. Statistics and Computing 10:325–337." />
<meta name="citation_reference" content="MacKenzie, D. I., J. D. Nichols, J. E. Hines, M. G. Knutson, and A. B. Franklin. 2003. Estimating site occupancy, colonization, and local extinction when a species is detected imperfectly. Ecology 84:2200–2207." />
<meta name="citation_reference" content="MacKenzie, D. I., J. D. Nichols, J. A. Royle, K. H. Pollock, L. Bailey, and J. E. Hines. 2017. Occupancy estimation and modeling: Inferring patterns and dynamics of species occurrence. Elsevier." />
<meta name="citation_reference" content="Ntzoufras, I. 2002. Gibbs variable selection using BUGS. Journal of statistical software 7:1–19." />
<meta name="citation_reference" content="Pardieck, K. L., D. J. Ziolkowski Jr, M. Lutmerding, K. J. Campbell, and M.-A. R. Hudson. 2016. North american breeding bird survey dataset 1966-2015, version 2015.0. U.S. Geological Survey, Patuxent Wildlife Research Center doi:10.5066/F7W0944J." />
<meta name="citation_reference" content="Plummer, M. 2012. JAGS: Just another Gibbs sampler. Astrophysics Source Code Library." />
<meta name="citation_reference" content="R Core Team. 2016. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria." />
<meta name="citation_reference" content="Reich, B. J., J. S. Hodges, and V. Zadnik. 2006. Effects of residual smoothing on the posterior of the fixed effects in disease-mapping models. Biometrics 62:1197–1206." />
<meta name="citation_reference" content="Rich, J. L., and D. J. Currie. 2018. Are north american bird species’ geographic ranges mainly determined by climate? Global Ecology and Biogeography." />
<meta name="citation_reference" content="Rosenberg, K. V., T. Will, D. A. Buehler, S. B. Swarthout, W. E. Thogmartin, R. E. Bennett, and R. Chandler. 2016. Dynamic distributions and population declines of golden-winged warblers. Studies in Avian Biology 49:3–28." />
<meta name="citation_reference" content="Rushing, C. S., T. B. Ryder, A. L. Scarpignato, J. F. Saracco, and P. P. Marra. 2016. Using demographic attributes from long-term monitoring data to delineate natural population structure. Journal of applied ecology 53:491–500." />
<meta name="citation_reference" content="Sauer, J., J. Hines, J. Fallon, K. Pardieck, D. Ziolkowski, and W. Link. 2015. Breeding Bird Survey Summary and Analysis 1966-2013. Version 01.30.2015. USGS Patuxent Wildlife Research Center Laurel MD:http://www.mbr-pwrc.usgs.gov/bbs/bbs.html." />
<meta name="citation_reference" content="Tingley, M. W., and S. R. Beissinger. 2009. Detecting range shifts from historical species occurrences: New perspectives on old data. Trends in Ecology &amp; Evolution 24:625–633." />
<meta name="citation_reference" content="Twedt, D. J. 2015. Estimating regional landbird populations from enhanced North American Breeding Bird Surveys. Journal of Field Ornithology 86:352–368." />
<meta name="citation_reference" content="Valpine, P.de, D. Turek, C. J. Paciorek, C. Anderson-Bergman, D. T. Lang, and R. Bodik. 2017. Programming with models: Writing statistical algorithms for general model structures with NIMBLE. Journal of Computational and Graphical Statistics 26:403–413." />
<meta name="citation_reference" content="White, G. C., and K. P. Burnham. 1999. Program MARK: Survival estimation from populations of marked animals. Bird Study 46:S120–139." />
<meta name="citation_reference" content="Wood, S. N. 2017. Generalized additive models: An introduction with R. CRC press." />
<meta name="twitter:title" content="Estimating spatially and temporally complex range dynamics when detection is imperfect" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Species distributions are determined by the interaction of multiple biotic and abiotic factors, which produces complex spatial and temporal patterns of occurrence. As habitats and climate change due to anthropogenic activities, there is a need to develop species distribution models that can quantify these complex range dynamics. In this paper, we develop a dynamic occupancy model that uses a spatial generalized additive model to estimate non-linear spatial variation in occupancy not accounted for by environmental covariates. The model is flexible and can accommodate data from a range of sampling designs that provide information about both occupancy and detection probability. Output from the model can be used to create distribution maps and to estimate indices of temporal range dynamics. We demonstrate the utility of this approach by modeling long-term range dynamics of 10 eastern North American birds using data from the North American Breeding Bird Survey. We anticipate this framework will be particularly useful for modeling species’ distributions over large spatial scales and for quantifying range dynamics over long temporal scales." />
<meta name="og-title" property="og:title" content="Estimating spatially and temporally complex range dynamics when detection is imperfect" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/586909v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Species distributions are determined by the interaction of multiple biotic and abiotic factors, which produces complex spatial and temporal patterns of occurrence. As habitats and climate change due to anthropogenic activities, there is a need to develop species distribution models that can quantify these complex range dynamics. In this paper, we develop a dynamic occupancy model that uses a spatial generalized additive model to estimate non-linear spatial variation in occupancy not accounted for by environmental covariates. The model is flexible and can accommodate data from a range of sampling designs that provide information about both occupancy and detection probability. Output from the model can be used to create distribution maps and to estimate indices of temporal range dynamics. We demonstrate the utility of this approach by modeling long-term range dynamics of 10 eastern North American birds using data from the North American Breeding Bird Survey. We anticipate this framework will be particularly useful for modeling species’ distributions over large spatial scales and for quantifying range dynamics over long temporal scales." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-03-28" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/586909v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/586909v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/586909v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/586909v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/386979" />
    <title>Estimating spatially and temporally complex range dynamics when detection is imperfect | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"MzAAZa4bE3jiQHfiMr9k7g0uTh2Jt2Oey7H-_ImEEuw","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;586909v1"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;586909v1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;586909v1"}],"nid":"386979","apath":"\/biorxiv\/early\/2019\/03\/28\/586909.atom","pisa":"biorxiv;586909v1","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/586909v1","title":"Estimating spatially and temporally complex range dynamics when detection is imperfect","identifier":"node\/386979"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022euzEfr0EBGMRZx+NcKJI3JhTqighdpu8o\\\\\\\/aNaMzovtahZZrwCkeweHrbAxkFmgD8u6RMTV0JtH79Dwn3ZZ7Rf03t+ExvJmAGI3xqkQP8tSt0MAx2m+YbNZKsT+WPRu4GNp\\\\\\\/h7+T+87VTFwSgAxsxHmdce3bT4a7h95FRDijd0FJwCO+0RohHHo3j2E9THjoNklv\\\\\\\/EEsKr3Ah9DvaAuWKBwfDPQR\\\\\\\/M8yk5CBA+evi6Y2AIMDLDf91NEnzIDwn3KB8v9rwoPJVyPQPT2yvW1XwWfbNOYS8\\\\\\\/BNZUGeUQ2qM\\\\\\\/6xbJL1Fi614iWSSgDy0cPXIg7MKRSABtUxlTcrYYcdbfIV7mWGVhEY7vTQmY+jZNBUvB8Fs2s1PeBaHRp5xTqfgrPqKH70GYXE\\\\\\\/V1Fe7T2N8s3jYemH\\\\\\\/UyPQJtQ3p\\\\\\\/T2ycIIfaoSE1vRRC5gcWmNyB2So49ZW0+oMMdiW2Of4gxH1Zk27nmncd6+WIS1bSnNPId61kwbv7FHWW9haCVXq2C+ovQ8v\\\\\\\/PWCS45cEgLXYta+b3RR4M5jkyI8P5K+vkLMidi8GBnsvNDEF5h4jdUl\\\\\\\/WmERuyNedXXMy001uh\\\\\\\/TFYky28MGUoCELPwT2m+xHP2DxkCNcTDRahIgEteTjgeCLIwuhClHQOI9h26Dejkuw6tFGqSjbITmU8Zdv1cnW5CWtg2PO0bvABPMt3zkCgALpAC6jFlJ955lNqDvnf2q+8Xsc6n55Vu7MFLuCzazzvU3Acw2OZKH+uu1Qxhjx7mVoCceQKoFchI4BO89n2FYwv0LeVmT8jQPdEFdDFUn7f2Dpc+G6RD20IsPE6zxLKX5r7A8TeFzd0xImf4tiHupEnHr4upDtJ1llTl+YI9wDdwYQ2OUP7GgB7Fa+mDKYpLtSYbr8nDSaM\\\\\\\/tjlo5oj7S4cCHvBn\\\\\\\/9J9nLzJh+RVWYdY3zivEVqjImPZn8A8V47C4y0isDmh2Guyh+96H2B875q\\\\\\\/hhUMvoxI0KGoUykHu2iUDJm2\\\\\\\/XIbSuZskzVH6wUey3wSBVNEqBzLuFwnAUAlv\\\\\\\/wo98LDdOfRPQi0IdnHK\\\\\\\/PPg3viCgcv9xwhTMiF8OdWUlIKMlzHrB\\\\\\\/ylhOBnYW3w+zQkffqxD7uoGi280cFoyuinRWH6IanYVU84AO+RX+tp574LW8b9\\\\\\\/QEyyBL\\\\\\\/pAM3Mv4pPCJs2041g9WPC+6zXvHSWP9drr336JAicMdcZTRpRhwV5G\\\\\\\/6gFngTB7F2n1IGHHPmxZ\\\\\\\/ueBb3ewXLJ8ND0MkVJxCXn0\\\\\\\/CQAZ\\\\\\\/5ijMKOAhsStDlwMoQuhswwAzvxncgiSxHaYYypcJPd0UNirBVMo=\\\u0022,\\\u0022iv\\\u0022:\\\u0022zFFr6TOEHZrrzbRpitcO8w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226e34fd3c7149b6b18ce8e9c640f86cc4ba6a3bd9ed3089f4c9bd02d8979c0385\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/586909v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-386979 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/586909v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-R2b74Xqlwuj1IbNEqcVmnWtWnML11GyweGxU0VWlnNQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="386979" id="node386979" data-pisa="biorxiv;586909v1" data-pisa-master="biorxiv;586909" data-apath="/biorxiv/early/2019/03/28/586909.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Estimating spatially and temporally complex range dynamics when detection is imperfect</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Clark S.</span> <span class="nlm-surname">Rushing</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">J. Andrew</span> <span class="nlm-surname">Royle</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">David J.</span> <span class="nlm-surname">Ziolkowski</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Keith L.</span> <span class="nlm-surname">Pardieck</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/586909 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node386979" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Clark S. Rushing </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>U.S. Geological Survey, Patuxent Wildlife Research Center</span>, 12100 Beech Forest Rd., Laurel, MD 20708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clark%2BS.%2BRushing%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rushing%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClark%2BS.%2BRushing%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:crushing@usgs.gov" class="" data-icon-position="" data-hide-link-title="0">crushing@usgs.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">J. Andrew Royle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>U.S. Geological Survey, Patuxent Wildlife Research Center</span>, 12100 Beech Forest Rd., Laurel, MD 20708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BAndrew%2BRoyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Royle%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BAndrew%2BRoyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David J. Ziolkowski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>U.S. Geological Survey, Patuxent Wildlife Research Center</span>, 12100 Beech Forest Rd., Laurel, MD 20708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BJ.%2BZiolkowski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ziolkowski%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BJ.%2BZiolkowski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Keith L. Pardieck </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>U.S. Geological Survey, Patuxent Wildlife Research Center</span>, 12100 Beech Forest Rd., Laurel, MD 20708</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BL.%2BPardieck%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardieck%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeith%2BL.%2BPardieck%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/586909v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:386979" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:386979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/586909v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:386979" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:386979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/586909v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:386979" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:386979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/586909v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:386979" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:386979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/586909v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:386979" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:386979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Species distributions are determined by the interaction of multiple biotic and abiotic factors, which produces complex spatial and temporal patterns of occurrence. As habitats and climate change due to anthropogenic activities, there is a need to develop species distribution models that can quantify these complex range dynamics. In this paper, we develop a dynamic occupancy model that uses a spatial generalized additive model to estimate non-linear spatial variation in occupancy not accounted for by environmental covariates. The model is flexible and can accommodate data from a range of sampling designs that provide information about both occupancy and detection probability. Output from the model can be used to create distribution maps and to estimate indices of temporal range dynamics. We demonstrate the utility of this approach by modeling long-term range dynamics of 10 eastern North American birds using data from the North American Breeding Bird Survey. We anticipate this framework will be particularly useful for modeling species’ distributions over large spatial scales and for quantifying range dynamics over long temporal scales.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">The distribution of each species is determined by the interaction of multiple biotic and abiotic factors that vary across both space and time, including weather and climate (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barbet-Massin and Jetz 2014</a>), habitat availability (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hill et al. 1999</a>), physiological tolerances (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kearney and Porter 2009</a>), and biotic interactions (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Araújo and Luoto 2007</a>). As a result, most species’ distributions are characterized by complex spatial and temporal patterns of occurrence, which combined with the large scales over which distributions change, present challenges for both the collection and analysis of data to quantify range dynamics (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Elith et al. 2010</a>). As habitats and climate change due to anthropogenic activities, there is an increasingly urgent need to develop species distribution models (SDMs) that can accurately and efficiently quantify complex range dynamics over large spatial and long temporal scales.</p><p id="p-4">In response to this need, researchers have developed a range of SDM approaches that vary in their data requirements and analytical methods (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Elith et al. 2010</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Guillera-Arroita 2017</a>). Although each of these methods has strengths and weaknesses, SDMs designed to quantify range dynamics require several key features. First, SDMs must include sufficient flexibility to quantify highly non-linear spatial patterns in occurrence probability. Although spatial variation in occurrence can in some cases be modeled using environmental covariates, residual spatial variation (which is likely common in most applications of SDMs at large spatial scales) can bias estimates of occurrence probability (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Johnson et al. 2013</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Guélat and Kéry 2018</a>). Second, because occurrence probability at a given point in time is not independent of occurrence probability at earlier points in time, SDMs must explicitly account for temporal auto-correlation in occurrence probability (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">MacKenzie et al. 2003</a>). If temporal auto-correlation is not accounted for within the SDM, occurrence dynamics are likely to appear more variable than they really are, leading to spurious conclusions about temporal variation range dynamics (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Clement et al. 2016</a>). Third, SDMs must uncouple true changes in occupancy from observation errors because the locations experiencing the largest changes in occupancy (e.g., range limits) are also the locations where errors arising from low detection probability are most likely (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Tingley and Beissinger 2009</a>, <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Guillera-Arroita 2017</a>). Although progress has been made on accounting for each of these three issues (complex spatial variation, temporal auto-correlation, and imperfect detection) in SDMs, there are few modeling frameworks that address all three simultaneously.</p><p id="p-5">Dynamic (or multi-season) occupancy models, which jointly estimate temporal change in the probability of occurrence and the probability of false-negative observations (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">MacKenzie et al. 2003</a>), provide a natural framework that potentially meets each of above criteria (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kéry 2011</a>). As a result, the use of occupancy models for species distribution modeling has grown in recent years (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Guillera-Arroita 2017</a>). At present, most occupancy-based SDMs have been implemented using one of several likelihood-based software programs, including programs PRESENCE (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hines 2006</a>) and MARK (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">White and Burnham 1999</a>) and the R package unmarked (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fiske and Chandler 2011</a>). These programs allow users to fit several variations of the standard static or dynamic occupancy models (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">MacKenzie et al. 2003</a>) using generalized linear models (GLMs) to estimate covariate effects on occupancy and detection. Although this GLM-based approach can account for imperfect detection and temporal auto-correlation in occupancy probability, these programs are restricted in their ability to model the complex non-linear patterns that typically characterize species’ distributions. In most cases, GLM-based models assume that spatial variation can be modeled as a linear or quadratic function of latitude and longitude, based on the assumption that species have a single center of high occupancy (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rich and Currie 2018</a>). For many species, this assumption is not warranted and therefore GLM-based occupancy models are unlikely to provide accurate estimates of spatial variation in occupancy.</p><p id="p-6">One alternative to the GLM approach is to estimate non-linear spatial variation in occupancy probability using generalized additive models (GAMs). An extension of GLMs, GAMs estimate the relationship between a response variable and a smoothed non-parametric function of covariates (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wood 2017</a>). Because the shape of the smoothed functions is determined by the data rather than a parametric function, GAMs can estimate complex, nonlinear spatial patterns that are not accounted for by covariates (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hefley et al. 2017</a>). Like GLMs, GAMs use link functions to accommodate response variables with normal or non-normal error distributions (e.g., binomial, Poisson), making it conceptually simple to extend occupancy models to estimate highly nonlinear effects of covariates (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kéry and Royle 2015</a>). Although GAMs have been used to estimate species distributions in a number of contexts (e.g., <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bled et al. 2013</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Guélat and Kéry 2018</a>), this approach has not been widely used in occupancy-based SDMs that account for both temporal dynamics and imperfect detection.</p><p id="p-7">We developed a dynamic occupancy model that uses a spatial GAM to estimate non-linear spatial variation in occupancy not accounted for by environmental covariates. The model is flexible and can accommodate data from a range of sampling designs that provide information about both occupancy and detection. Output from the model can be used to create distribution maps and to estimate intuitive indices of range shifts (range center, range limits). We demonstrate the utility of this approach by modeling long-term (1972-2015) range dynamics of 10 eastern North American birds using data from the North American Breeding Bird Survey. For this application, we also use Gibbs variable selection (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dellaportas et al. 2002</a>) to identify species-specific relationships between climate and occupancy. We anticipate this framework will be particularly useful for modeling species’ distributions over large spatial scales and for quantifying range dynamics over long temporal scales because of the improved fit to complex species distributions.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Model description</h3><p id="p-8">We assume that <em>j</em> = 1, 2,…<em>J</em> temporally or spatially replicated presence/absence surveys are conducted in <em>t</em> = 1, 2,…, <em>T</em> primary periods at <em>i</em> = 1, 2,…, <em>N</em> sampling locations. Further, we assume that the true (but latent) occupancy state of each site, denoted <em>z</em><sub><em>i,t</em></sub>, is closed within each primary period but can change across primary periods. During each survey, the observed occupancy state of the focal species, denoted <em>h</em><sub><em>i,j,t</em></sub>, is recorded (0 = species not observed, 1 = species observed). Our primary aim is to model temporal and spatial variation in the probability of occupancy <em>ψ</em><sub><em>i,t</em></sub> = <em>Pr</em>(<em>z</em><sub><em>i,t</em></sub> = 1) while accounting for imperfect detection. Below, we describe a Bayesian state-space formulation of this model that uses smoothing splines to model complex spatial and temporal auto-correlation in occupancy probability.</p><div id="sec-4" class="subsection"><h4>State model</h4><p id="p-9">In each primary period <em>t</em>, occupancy state is modeled as a Bernoulli random variable with probability <em>ψ</em><sub><em>i,t</em></sub>:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-1.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-2.gif"/></noscript></span>

</span>
where <em>f</em><sub><em>t</em></sub>(<em>lat</em><sub><em>i</em></sub>, <em>lon</em><sub><em>i</em></sub>) is a spatial smoothing function, <strong><em>β</em></strong> is a vector of slope coefficients, and <strong><em>X</em></strong><sub><em>i,t</em></sub> is a matrix containing covariate values for route <em>i</em> in period <em>t</em>. The smooth function <em>f</em><sub><em>t</em></sub> is composed of basis functions <em>g</em><sub><em>k</em></sub> and their corresponding regressions coefficients <em>γ</em><sub><em>k,t</em></sub>:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-3.gif"/></noscript></span>

</span>
</p><p id="p-10">Different smooth functions can be chosen based on the structure of the data (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hefley et al. 2017</a>, <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wood 2017</a>), providing a flexible and efficient means to model complex spatial variation in occupancy probability. The basis dimension <em>K</em> should be large enough to approximate the smooth function (i.e., avoid over-smoothing), though the exact choice of <em>K</em> is not critical because the degree of smoothing is determined primarily by a smoothing penalty term λ which penalizes against over-fitting (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Wood 2017</a>). In a Bayesian context, this penalization can be incorporated by specifying multivariate normal priors for the <em>γ</em><sub><em>k</em></sub> coefficients, with the precision matrix proportional to λ. Larger values of λ produce more constrained priors and thus more similar (i.e., more smooth) estimates of the <em>γ</em><sub><em>k</em></sub> coefficients (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Wood 2017</a>).</p><p id="p-11">In combination with time-varying covariates <strong><em>X</em></strong><sub><em>i,t</em></sub>, allowing the basis function coefficients to vary across primary periods allows occupancy probability at each site to change over time. When <em>t</em> = 1, the <em>γ</em><sub><em>k,1</em></sub> coefficients are estimated using the Bayesian penalization approach described above. To account for temporal auto-correlation in occupancy, the basis function coefficients in periods <em>t</em> &gt; 1 were modeled as temporally-correlated random effects:
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-4.gif"/></noscript></span>

</span>
where <em>σ</em><sup>2</sup> is the variance among primary periods. Initial testing of our model indicated that the temporal random-effect formulation resulted in better mixing of the MCMC chains than the conventional dynamic occupancy model in which occupancy probability in years <em>t</em> &gt; 1 is modeled as a function of extinction and colonization probabilities (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">MacKenzie et al. 2003</a>). This formulation also accounts for temporal auto-correlation in occupancy probability without assuming that long-term dynamics are subject to equilibrium conditions.</p></div><div id="sec-5" class="subsection"><h4>Observation model</h4><p id="p-12">The observed status of each site during each survey (<em>h</em><sub><em>i,j,t</em></sub>) is modeled as a function of both the latent state process (<em>z</em><sub><em>i,t</em></sub>) and detection probability <em>p</em><sub><em>t</em></sub>:
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-5.gif"/></noscript></span>

</span>
</p><p id="p-13">Covariates thought to influence the detection process (observer bias, weather, etc.) can be incorporated into this structure using a logistic link function on <em>p</em><sub><em>t</em></sub> (see below).</p></div></div><div id="sec-6" class="subsection"><h3>Distribution maps and indices of range dynamics</h3><p id="p-14">When latitude, longitude, and annual covariate values are available at unsampled locations within a species’ range, posterior distributions of the predicted occupancy probability at those locations can be easily estimated from the posterior samples of the fitted model (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Kéry 2011</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Clement et al. 2016</a>). Posterior estimates of range-wide occupancy probability can be used to visualize changes in species’ distributions and to quantify indices of range dynamics (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Clement et al. 2016</a>). Although many such indices are possible, we describe four that may be particularly relevant to quantifying range shifts. First, the mean occupancy probability of all map cells provides an index of changes in the proportion of area occupied (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Clement et al. 2016</a>). Second, the mean breeding latitude, estimated as the sum of the cell latitudes weighted by their occupancy probabilities and divided by the total occupancy probability across all cells, can be used to quantify shifts in the center of species range (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Clement et al. 2016</a>). Finally, annual indices of the northern/southern range limits can be estimated by sorting the map cells by latitude and then using a smoothing spline function to predict the latitude below/above which 99.9% of the total occupancy probability is located. Although not an absolute measure of the northern- and southern-most latitudes at which a species was found, this index provides a time series of relative change in the northern and southern range limits, which can be used to determine whether distributions have expanded or contracted over time. We provide code in Appendix S1 for estimation of these indices.</p></div><div id="sec-7" class="subsection"><h3>Application</h3><div id="sec-8" class="subsection"><h4>Data</h4><p id="p-15">We demonstrate the utility of this model by quantifying range dynamics of 10 eastern North American bird species: Red-bellied Woodpecker (<em>Melanerpes carolinus</em>), Fish Crow (<em>Corvus ossifragus</em>), Carolina Chickadee (<em>Poecile carolinensis</em>), Carolina Wren (<em>Thryothorus ludovicianus</em>), Blue-gray Gnatcatcher (<em>Polioptila caerulea</em>), Wood Thrush (<em>Hylocichla mustelina</em>), Golden-winged Warbler (<em>Vermivora chrysoptera</em>), Swainson’s Warbler (<em>Limnothlypis swainsonii</em>), Louisiana Waterthrush (<em>Parkesia motacilla</em>), and Kentucky Warbler (<em>Geothlypis formosa</em>). We selected these species because they exhibit a wide range of spatial and temporal complexity in range dynamics. Data for this analysis came from the North American Breeding Bird Survey (BBS), a large-scale citizen science program consisting of over 5500 roadside survey routes of which approximately 3000 are surveyed each May or June by trained volunteers (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sauer et al. 2015</a>). The BBS was initiated in 1966, though only a small number of routes were surveyed during the early years. For this reason, we chose to use BBS data collected from 1972 to 2015 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pardieck et al. 2016</a>). Trained observers conduct 3-minute point counts at 50 regularly spaced stops along each 39.4km-long route. See <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Sauer et al. (2015)</a> for more details regarding the BBS survey protocol. Prior to analysis, we converted the raw counts to stop-level presence/absence data.</p><p id="p-16">To model spatial/climate relationships across the edge of each species’ occupied range, we subset all BBS routes with at least one detection of the focal species over the study period (i.e., routes occupied in at least one year). Next, we created a 2°-buffered convex hull around the occupied routes and included all routes within the buffered region.</p><p id="p-17">Climate data was obtained from the University of East Anglia Climate Research Unit (CRU; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Harris et al. (2014)</a>). The CRU data contains global estimates of monthly surface climate variables for 0.5° grid cells (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Harris et al. 2014</a>). Following <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Clement et al. (2016)</a>, we converted the monthly temperature and precipitation estimates from the CRU data set into five ‘bioclim’ variables that have low correlation and are effective for modeling species ranges (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barbet-Massin and Jetz 2014</a>). Specifically, for each grid cell we calculated the mean temperature, mean diurnal temperature range, mean temperature of the wettest quarter, annual precipitation, and precipitation of the warmest quarter for the 12 months preceding each BBS survey (i.e., June-May). All estimates were obtained using the ‘biovars’ function in the ‘dismo’ package (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hijmans et al. 2016</a>) in program R (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">R Core Team 2016</a>). Prior to analysis, each variable was scaled to mean = 0 and standard deviation = 1 and we extracted the annual climate values for the grid cell containing the first stop of each BBS route.</p><p id="p-18">Because BBS routes are surveyed a single time each year, conventional methods for estimating detection probability from temporally replicated surveys are not possible. Instead, we adapted an occupancy model that uses the correlation between adjacent spatial replicates to estimate detection probability (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hines et al. 2010</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">2014</a>). In this model, occurrence is estimated at two scales: 1) the route-level (i.e., <em>z</em><sub><em>i,t</em></sub>), and 2) the stop-level (denoted <em>y</em><sub><em>j,i,t</em></sub>|<em>z</em><sub><em>i,t</em></sub>). Hereafter we follow <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Clement et al. (2016)</a> and refer to presence at the route-level as “occupancy” and presence at the stop-level as “availability”. Digital records of the raw 50-stop BBS data are only available from 1997-present. However, 10-stop summaries (sum of counts from stops 1-10, 11-20, 21-30, 31-40, and 41-50) are available for the entire BBS period (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Pardieck et al. 2016</a>). Initial testing indicated that estimates of the route-level occupancy did not differ when the model was fit using the full 50-stop data or the 10-stop summaries. Therefore, we chose to use the 10-stop data so that inferences could be made over the entire BBS time series.</p></div><div id="sec-9" class="subsection"><h4>State model</h4><p id="p-19">Spatial and annual variation in route-level occupancy probability was modeled using a modified version of <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Eq. 1</a>:
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-6.gif"/></noscript></span>

</span>
where <strong><em>ω</em></strong> is a vector of binary indicator variables determining whether each climate predictor is included in the model and <strong><em>X</em></strong><sub><em>i,t</em></sub> is a matrix containing the annual climate values in year <em>t</em> at route <em>i</em>. Estimation of <strong><em>ω</em></strong> is described below. To capture non-linear relationships between climate and occupancy, the matrix <strong><em>X</em></strong> contained both the linear and quadratic terms for each of the 5 climate covariates. For the spatial smooth, we used a thin-plate regression spline of the latitude and longitude of each BBS route. For all species, we chose <em>k</em> = 60, which initial tests indicated was large enough to approximate the smooth function for all species considered.</p><p>At the stop level, availability is modeled as a first-order Markov p rocess with parameters:
</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-21"><em>θ</em><sub><em>i,j,t</em></sub> = Pr(availability at stop <em>j</em>|route <em>i</em> occupied and stop <em>j</em> − 1 unavailable)</p></li><li id="list-item-2"><p id="p-22"><span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/inline-graphic-1.gif"/></noscript></span></span> = Pr(availability at stop <em>j</em>|route <em>i</em> occupied and stop <em>j</em> − 1 available)</p></li></ul><p>
</p><p id="p-23">As noted by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hines et al. (2010)</a>, the first stop on each BBS routes has no predecessor and thus availability at stop 1 cannot be modeled using <em>θ</em> or <em>θ</em>′. Instead, we directly estimated the probability <em>π</em><sub><em>t</em></sub> that the first stop is available in year <em>t</em>:
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-7.gif"/></noscript></span>

</span>
</p><p id="p-24">For the remaining stops, availability was modeled as:
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-8.gif"/></noscript></span>

</span>
</p></div><div id="sec-10" class="subsection"><h4>Observation model</h4><p id="p-25">Numerous factors could influence detection probability in BBS surveys. Observer experience and variation among observers are both known to influence detection of many species (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Link and Sauer 2002</a>). Weather conditions, particularly wind speed, may also influence detectability. For our analysis, we included wind speed scores recorded at the start of each BBS count using the Beaufort wind scale (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Sauer et al. 2015</a>). Between 2009 and 2015, a small number of BBS routes (n = 106) were surveyed using a modified p rotocol (RPID = 501) that incorporated time and distance information (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Twedt 2015</a>). Compared to the standard BBS protocol (RPID = 101), this time-distance protocol resulted in on average 10% fewer observations per survey (Sauer et al. <em>in prep</em>). To account for these effects, we modeled detection conditional on stop-level availability and route-level occupancy as:
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-9.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-10.gif"/></noscript></span>

</span>
where <em>α</em><sub>0</sub> is an intercept term, <em>WIND</em><sub><em>i,t</em></sub> is the wind score, <strong><em>I</em></strong><sub><em>i,t</em></sub> is a binary dummy variable indicating whether year <em>t</em> was an observer’s first year of service, <strong><em>l</em></strong><sub><em>i,t</em></sub> is a dummy variable indicating the survey protocol used (0 = standard BBS survey, 1 = time-distance protocol), and <em>η</em><sub><em>i,t</em></sub> is a random observer effect.</p></div><div id="sec-11" class="subsection"><h4>Model selection</h4><p id="p-26">Given the large number of climate predictors in our model and the lack of <em>a priori</em> hypotheses about which predictors should influence the distribution of each species, each climate variable <em>m</em> was multiplied by a binary latent variable <em>ω</em><sub><em>m</em></sub> which determined whether the variable was included in the linear predictor. The posterior probability <em>Pr</em>(<em>ω</em><sub><em>m</em></sub> = 1) is then a measure of the relative importance of variable <em>m</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kuo and Mallick 1998</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dellaportas et al. 2002</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ntzoufras 2002</a>). For the linear effect of each climate variable, we assumed mutually independent Bernoulli priors:
<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-11.gif"/></noscript></span>

</span>
</p><p id="p-27">For the quadratic terms, we enforced marginality by setting:
<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-12.gif"/></noscript></span>

</span>
where <em>m</em><sup>2</sup> is the quadratic term associated with the linear term <em>m</em>. Thus, quadratic terms could only enter the model if the corresponding linear term is also in the model. To ensure good mixing of the <em>ω</em><sub><em>m</em></sub> parameters, we used Gibbs Variable Selection (GVS; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Dellaportas et al. 2002</a>) to create joint prior distributions for the <em>β</em><sub><em>m</em></sub> parameters conditional on <em>ω</em><sub><em>m</em></sub>:
<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/graphic-13.gif"/></noscript></span>

</span>
where <em>N</em> (0,100) is a non-informative normal prior when <em>ω</em><sub><em>m</em></sub> = 1 and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/inline-graphic-2.gif"/></noscript></span></span> is a pseudo-prior sampled when <em>ω</em><sub><em>m</em></sub> = 0. We estimated <em>µ</em><sub><em>m</em></sub> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/inline-graphic-3.gif"/></noscript></span></span> by running the correlated detection model in PRESENCE (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hines 2006</a>) using the first 10 years of BBS data for each species and including linear and quadratic effects for all five climate predictors.</p></div><div id="sec-12" class="subsection"><h4>Modeling fitting and indices of range shifts</h4><p id="p-28">We fit the models in JAGS (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Plummer 2012</a>) called from R using the jagsUI package (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kellner 2015</a>). As described above, we specified multivariate normal priors for the GAM smooth coefficients, Bernoulli priors for the indicator variables, and vague normal priors for the <em>β</em> coefficients. For all other parameters, we specified appropriate vague priors. See Data S1 for model code and specification details. We assessed goodness-of-fit using posterior predictive checks (PPC). Because conventional PPC metrics are inappropriate for binary occupancy data (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kéry and Royle 2015</a>), we used each posterior estimate from the fitted model to simulate the expected number of routes with each of the 32 possible detection histories in each year. We used a Freeman-Tukey statistic (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brooks et al. 2000</a>) to measure the discrepancy between the observed/simulated and predicted detection history frequencies and we report the Bayesian P-value from these tests.</p><p id="p-29">For each species, we created annual distribution maps and range shift indices using the climate covariate values, latitude, and longitude for each 0.5° raster cell within the same buffered convex hull used to subset BBS routes. Posterior distributions of the predicted annual occupancy probability in each cell were estimated using the posterior samples for each model parameter. From these distributions we then estimated posterior distributions for the four indices described above (proportion of area occupied, mean breeding latitude, and northern/southern range limits) and report the mean and 95% credible interval for each.</p></div></div></div><div class="section" id="sec-13"><h2 class="">Results</h2><p id="p-30">By allowing for complex variation in occupancy while simultaneously penalizing against over-fitting, the spatial GAM was able to model both highly complex and relatively simple distributions. For example, species like Fish Crow and Swainson’s Warbler have complex spatial distributions that do not exhibit simple linear or quadratic relationships with latitude and/or longitude. Occupancy probabilities for the Fish Crow were high along the Atlantic and Gulf coasts as well as in the southern Mississippi River valley, resulting in a U-shaped distribution with no clear range center (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). The distribution of Swainson’s Warbler was also complex, with three distinct areas of high occupancy and low occupancy in between (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>). In contrast, the distributions of some species, including Red-bellied Woodpecker and Carolina Chickadee, were less complex, with a large central area of high occupancy with declining occupancy along the periphery of the range (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1C-D</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of (A) Fish Crow, (B) Swainson&#x2019;s Warbler, (C) Carolina Chickadee, and (D) Eastern Towhee. Maps show the predicted probability of occupancy in 1972 for each species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1862515759" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distributions of (A) Fish Crow, (B) Swainson&#x2019;s Warbler, (C) Carolina Chickadee, and (D) Eastern Towhee. Maps show the predicted probability of occupancy in 1972 for each species.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F1.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F1.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-31" class="first-child">Distributions of (A) Fish Crow, (B) Swainson’s Warbler, (C) Carolina Chickadee, and (D) Eastern Towhee. Maps show the predicted probability of occupancy in 1972 for each species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The model was also able to quantify temporal changes in occupancy probability over the 43 year time period of the BBS data, revealing similarities and differences in regional dynamics in several species. For example, Louisiana Waterthrush, Kentucky Warblers, Wood Thrush, and Golden-winged Warblers all experienced large declines in occupancy probability in the eastern portions of their range, especially in the northeastern United States and Appalachia, but were relatively stable or increasing in the mid-western United States (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2</a>). These species differed, however, in occupancy trends in the southeastern United States, with Louisiana Waterthrush and Kentucky Warblers showing modest increases in occupancy probability and Wood Thrush and Golden-winged Warblers declining in occupancy probability.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Net change in occupancy probability (&#x3A8;2015 &#x2212; &#x3A8;1972) for (A) Louisiana Waterthrush, (B) Brown Thrasher, (C) Kentucky Warbler, and (D) Golden-winged Warbler. Blue indicates areas that have increased in occupancy probability; red indicates areas that have decreased in occupancy probability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1862515759" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Net change in occupancy probability (&lt;em&gt;&#x3A8;&lt;/em&gt;&lt;sub&gt;2015&lt;/sub&gt; &#x2212; &lt;em&gt;&#x3A8;&lt;/em&gt;&lt;sub&gt;1972&lt;/sub&gt;) for (A) Louisiana Waterthrush, (B) Brown Thrasher, (C) Kentucky Warbler, and (D) Golden-winged Warbler. Blue indicates areas that have increased in occupancy probability; red indicates areas that have decreased in occupancy probability.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F2.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F2.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-33" class="first-child">Net change in occupancy probability (<em>Ψ</em><sub>2015</sub> − <em>Ψ</em><sub>1972</sub>) for (A) Louisiana Waterthrush, (B) Brown Thrasher, (C) Kentucky Warbler, and (D) Golden-winged Warbler. Blue indicates areas that have increased in occupancy probability; red indicates areas that have decreased in occupancy probability.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Indices of range shifts from our model indicate that some species have undergone distributional shifts over the past four decades. For example, the northern range limit of Blue-gray Gnatcatchers shifted northward by ~1.2° latitude and the mean breeding latitude shifted northward by ~1° (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3A-B</a>). Interestingly, this species has shown small (~0.2°) southward expansion at the southern edge of its range and as a result, the proportion of area occupied has increased over time (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3C</a>). The indices were also able to capture transient dynamics in distributional shifts. Northern populations of Carolina Wren, for example, experienced large declines in occupancy probability in the late 1970’s, resulting in a contraction of the northern range limit and mean breeding latitude by ~1° latitude (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3E-F</a>). This contraction was temporary, however, with these populations subsequently experiencing a sustained northward expansion extending ~1.7° beyond their initial northern range limit (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3D</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Net change in occupancy probability, latitudinal range indices, and proportion of area occupied for (A-C) Blue-gray Gnatcatcher and (D-F) Carolina Wren. For latitudinal range indices, solid lines indicate the mean breeding latitude and dashed lines indicate the northern and southern range limit. Gray ribbons represent the 95% credible intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1862515759" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Net change in occupancy probability, latitudinal range indices, and proportion of area occupied for (A-C) Blue-gray Gnatcatcher and (D-F) Carolina Wren. For latitudinal range indices, solid lines indicate the mean breeding latitude and dashed lines indicate the northern and southern range limit. Gray ribbons represent the 95% credible intervals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F3.medium.gif" width="440" height="263"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F3.medium.gif" width="440" height="263"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/03/28/586909/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-35" class="first-child">Net change in occupancy probability, latitudinal range indices, and proportion of area occupied for (A-C) Blue-gray Gnatcatcher and (D-F) Carolina Wren. For latitudinal range indices, solid lines indicate the mean breeding latitude and dashed lines indicate the northern and southern range limit. Gray ribbons represent the 95% credible intervals.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-14"><h2 class="">Discussion</h2><p id="p-36">The distributions of most species are characterized by complex and dynamic variation in occurrence. Species distribution modeling seeks to relate this variation to environmental covariates and extrapolate these relationships to unsampled sites and times. Because habitats and species-habitat relationships change across both space and time, conventional GLM-based models rarely capture the inherent complexity of species distributions, especially when inferences are made across large spatial or long temporal scales. Here, we demonstrate a novel occupancy-based SDM that combines environmental predictors with a spatial GAM to model covariate relationships and complex, non-linear spatial variation in occupancy probability while accounting for imperfect detection.</p><p id="p-37">Application of this model to 10 North American bird species demonstrates the utility and flexibility of our approach for species’ distribution modeling. Unlike parametric models that use low-order polynomials to capture spatial variation in occupancy, the spatial GAM derives non-parametric occupancy patterns during estimation while avoiding overfitting through the incorporation of a smoothing penalty term. This balancing of complexity and smoothing ensures that the model allows for, but does not impose, complex spatial variation in occupancy. This formulation provides an efficient and flexible method to model large-scale spatial variation in occupancy probability, for example allowing us to model both the complex distributions of Fish Crow and Swainson’s Warbler and the relativity more simple distributions of Carolina Chickadees and Red-bellied Woodpeckers using a common model structure.</p><p id="p-38">Modeling the GAM coefficients as temporally-correlated random effects also allowed us to explicitly model changes in occupancy probability over time. In our analysis of BBS data, the model uncovered interesting similarities and differences in the range dynamics of several species that inhabit forest habitats in the eastern United States, including Louisiana Waterthrush, Wood Thrush, Kentucky Warblers, and Golden-winged Warblers. In the case of Wood Thrush and Golden-winged Warblers, this regional variation in occupancy dynamics is consistent with differences in demographic rates among the regions (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rosenberg et al. 2016</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Rushing et al. 2016</a>), suggesting that our occupancy model was able to capture spatial variation in population dynamics.</p><p id="p-39">The indices of latitudinal range dynamics from our model also documented range expansions of Blue-gray Gnatcatchers and Carolina Wrens, demonstrating the utility of these metrics for quantifying range shifts over long temporal scales. The ability to document shifts at range margins while accounting for imperfect detection is particularly important given that these locations are likely to experience the largest changes in occupancy but also have the lowest detection probability. Application of this framework to a larger pool of species could indicate whether range shifts provide a consistent fingerprint of habitat or climate change. It may also be possible to quantify range shifts of entire groups of species by creating composite versions of our indices, which would be particularly useful for testing hypotheses about which traits promote or impede the ability of species to respond to habitat and climate change.</p><p id="p-40">Our model differs from the conventional dynamic occupancy model in that we did not directly estimate change in occupancy as the result extinction/colonization processes. Under the extinction/colonization formulation, occupancy probability at a given location <em>i</em> will converge on the stable-state occupancy distribution defined by <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/03/28/586909/embed/inline-graphic-4.gif"/></noscript></span></span>, where <em>γ</em><sub><em>i</em></sub> and <em>ϵ</em><sub><em>i</em></sub> are the colonization and extinction rates at location <em>i</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">MacKenzie et al. 2017</a>). Initial testing of our model using the extinction/colonization formulation indicated that extinction rates were very low relative to colonization rates for most species, resulting in non-negligible occupancy probabilities (&gt; ~8-10%) in the final year of our analysis at locations where the species were never detected. We suspect this may be a common issue when fitting dynamic occupancy models at large spatial and long temporal scales. Because our formulation does not make any equilibrium assumptions, it may be a better approach when the goal is to document range shifts. In some cases, using the conventional dynamic occupancy model with spatial variation in extinction/colonization probabilities may be preferable, particularly when the focus is on mechanistic understanding of range dynamics. Additionally, GAMs generally perform poorly when predicting outside of the data used to fit the model so the conventional occupancy model may be more suitable when the goal is to predict future distributions.</p><p id="p-41">The model presented in this paper builds off of recent advances in the development of occupancy-based SDMs that account for both imperfect detection and temporal auto-correlation in occupancy (<a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Clement et al. 2016</a>). Our model extends this framework to account for complex spatial and temporal variation in occupancy probability through the use of a hierarchical Bayesian model with a spatial GAM. Accounting for complex spatial structure in SDMs is an active area of research (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Reich et al. 2006</a>) and other methods exist for handling spatial structure in occupancy models, particularly through the use of conditional autoregressive (CAR) modeling (e.g., <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bled et al. 2011</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Johnson et al. 2013</a>, <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Guélat and Kéry 2018</a>). Both CAR and GAM approaches have been shown to reduce bias in SDM models (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Guélat and Kéry 2018</a>), though the GAM approach is generally more computationally efficient, produces more precise parameter estimates, and can be fit using most popular Bayesian software programs, including JAGS, WinBUGS (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lunn et al. 2000</a>), NIMBLE (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Valpine et al. 2017</a>), and STAN (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carpenter et al. 2017</a>). These two approaches are not mutually exclusive though and future work integrating GAM and CAR models in occupancy-based SDMs is likely to improve inferences about past and future range dynamics.</p></div><div class="section" id="sec-15"><h2 class="">Author contributions</h2><p id="p-43">CSR, JAR, and KLP concieved of the study with input from DJZ. CSR and JAR developed the methods. CSR carried out all analyses and drafted the manuscript. All authors contributed to subsequent revisions.</p></div><div class="section COI-statement" id="sec-16"><h2 class="">Competing interests</h2><p id="p-44">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-42">The authors thank J. Hines, M. Clement, J. Sauer, and N. Hostetter for assistance with model development and M. Kery for comments that improved earlier drafts of this manuscript. We also thank the thousands of BBS volunteers for collecting the data that made this project possible.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Araújo</span>,  <span class="cit-name-given-names">M. B.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Luoto</span></span>. <span class="cit-pub-date">2007</span>. <span class="cit-article-title">The importance of biotic interactions for modelling species distributions under climate change</span>. <abbr class="cit-jnl-abbrev">Global Ecology and Biogeography</abbr> <span class="cit-vol">16</span>:<span class="cit-fpage">743</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BEcology%2Band%2BBiogeography%26rft.volume%253D16%26rft.spage%253D743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barbet-Massin</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Jetz</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">A 40-year, continent-wide, multispecies assessment of relevant climate predictors for species distribution modelling</span>. <abbr class="cit-jnl-abbrev">Diversity and Distributions</abbr> <span class="cit-vol">20</span>:<span class="cit-fpage">1285</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2Band%2BDistributions%26rft.volume%253D20%26rft.spage%253D1285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bled</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Nichols</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Altwegg</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Dynamic occupancy models for analyzing species’ range dynamics across large geographic scales</span>. <abbr class="cit-jnl-abbrev">Ecology and Evolution</abbr> <span class="cit-vol">3</span>:<span class="cit-fpage">4896</span>–<span class="cit-lpage">4909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D4896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bled</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Royle</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Cam</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Hierarchical modeling of an invasive spread: The Eurasian Collared-Dove <em>streptopelia decaocto</em> in the United States</span>. <abbr class="cit-jnl-abbrev">Ecological Applications</abbr> <span class="cit-vol">21</span>:<span class="cit-fpage">290</span>–<span class="cit-lpage">302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.stitle%253DEcological%2BApplications%26rft.aulast%253DBled%26rft.auinit1%253DF.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D290%26rft.epage%253D302%26rft.atitle%253DHierarchical%2Bmodeling%2Bof%2Ban%2Binvasive%2Bspread%253A%2Bthe%2BEurasian%2BCollared-Dove%2BStreptopelia%2Bdecaocto%2Bin%2Bthe%2BUnited%2BStates.%26rft_id%253Dinfo%253Apmid%252F21516906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21516906&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F28%2F586909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brooks</span>,  <span class="cit-name-given-names">S. P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Catchpole</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Morgan</span></span>. <span class="cit-pub-date">2000</span>. <span class="cit-article-title">Bayesian animal survival estimation</span>. <abbr class="cit-jnl-abbrev">Statistical Science</abbr>:<span class="cit-fpage">357</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gelman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Hoffman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Goodrich</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Betancourt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Brubaker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Li</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Riddell</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Stan: A probabilistic programming language</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr> <span class="cit-fpage">76</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Clement</span>,  <span class="cit-name-given-names">M. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Hines</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Nichols</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Pardieck</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Ziolkowski</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Estimating indices of range shifts in birds using dynamic models when detection is imperfect</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">22</span>:<span class="cit-fpage">3273</span>–<span class="cit-lpage">3285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D22%26rft.spage%253D3273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dellaportas</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Forster</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Ntzoufras</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">On Bayesian model and variable selection using MCMC</span>. <abbr class="cit-jnl-abbrev">Statistics and Computing</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">27</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistics%2Band%2BComputing%26rft.volume%253D12%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elith</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kearney</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Phillips</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">The art of modelling range-shifting species</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">1</span>:<span class="cit-fpage">330</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D1%26rft.spage%253D330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fiske</span>,  <span class="cit-name-given-names">I.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Chandler</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Ummarked: An R package for fitting hierarchical models of wildlife occurrence and abundance</span>. <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr> <span class="cit-vol">43</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D43%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guélat</span>,  <span class="cit-name-given-names">J.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kéry</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Effects of spatial autocorrelation and imperfect detection on species distribution models</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>:<span class="cit-fpage">n/a</span>–<span class="cit-lpage">n/a</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guillera-Arroita</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Modelling of species distributions, range dynamics and communities under imperfect detection: Advances, challenges and opportunities</span>. <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.13" data-doi="10.1002/joc.3711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jones</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Osborn</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lister</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Updated high-resolution grids of monthly climatic observations–the CRU TS3. 10 Dataset</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr> <span class="cit-vol">34</span>:<span class="cit-fpage">623</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D34%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.3711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.3711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000332092900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.14" data-doi="10.1002/ecy.1674"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hefley</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Broms</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Brost</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. E.</span> <span class="cit-name-surname">Buderman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Kay</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">H. R.</span> <span class="cit-name-surname">Scharf</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Tipton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Williams</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Hooten</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">The basis function approach for modeling autocorrelation in ecological data</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">98</span>:<span class="cit-fpage">632</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D98%26rft.spage%253D632%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fecy.1674%26rft_id%253Dinfo%253Apmid%252F27935640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ecy.1674&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27935640&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F28%2F586909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.15"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Phillips</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Leathwick</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Elith</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Dismo: Species distribution modeling</span>. <abbr class="cit-jnl-abbrev">R package ver. 1.0-15</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.16" data-doi="10.1098/rspb.1999.0763"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hill</span>,  <span class="cit-name-given-names">J. K.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Thomas</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Huntley</span></span>. <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Climate and habitat availability determine 20th century changes in a butterfly’s range margin</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society of London B: Biological Sciences</abbr> <span class="cit-vol">266</span>:<span class="cit-fpage">1197</span>–<span class="cit-lpage">1206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DHill%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D266%26rft.issue%253D1425%26rft.spage%253D1197%26rft.epage%253D1206%26rft.atitle%253DClimate%2Band%2Bhabitat%2Bavailability%2Bdetermine%2B20th%2Bcentury%2Bchanges%2Bin%2Ba%2Bbutterfly%2527s%2Brange%2Bmargin%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1999.0763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.1999.0763&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000081155500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-web" id="cit-586909v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hines</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-pub-date">2006</span>. <span class="cit-source">Program presence</span>. See <a href="http://www.mbrpwrc.usgs.gov/software/doc/presence/presence.html">http://www.mbrpwrc.usgs.gov/software/doc/presence/presence.html</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hines</span>,  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Nichols</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Collazo</span></span>. <span class="cit-pub-date">2014</span>. <span class="cit-article-title">Multiseason occupancy models for correlated replicate surveys</span>. <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">583</span>–<span class="cit-lpage">591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D5%26rft.spage%253D583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.19" data-doi="10.1890/09-0321.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hines</span>,  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Nichols</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Royle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. I.</span> <span class="cit-name-surname">MacKenzie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gopalaswamy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kumar</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Karanth</span></span>. <span class="cit-pub-date">2010</span>. <span class="cit-article-title">Tigers on trails: Occupancy modeling for cluster sampling</span>. <abbr class="cit-jnl-abbrev">Ecological Applications</abbr> <span class="cit-vol">20</span>:<span class="cit-fpage">1456</span>–<span class="cit-lpage">1466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.stitle%253DEcological%2BApplications%26rft.aulast%253DHines%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D20%26rft.issue%253D5%26rft.spage%253D1456%26rft.epage%253D1466%26rft.atitle%253DTigers%2Bon%2Btrails%253A%2Boccupancy%2Bmodeling%2Bfor%2Bcluster%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F09-0321.1%26rft_id%253Dinfo%253Apmid%252F20666261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/09-0321.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20666261&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F28%2F586909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.20" data-doi="10.1890/12-0564.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">D. S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. B.</span> <span class="cit-name-surname">Conn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Hooten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Ray</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Pond</span></span>. <span class="cit-pub-date">2013</span>. <span class="cit-article-title">Spatial occupancy models for large data sets</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">94</span>:<span class="cit-fpage">801</span>–<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D94%26rft.spage%253D801%26rft_id%253Dinfo%253Adoi%252F10.1890%252F12-0564.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/12-0564.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.21" data-doi="10.1111/j.1461-0248.2008.01277.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kearney</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Porter</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Mechanistic niche modelling: Combining physiological and spatial data to predict species ranges</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">12</span>:<span class="cit-fpage">334</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DKearney%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D334%26rft.epage%253D350%26rft.atitle%253DMechanistic%2Bniche%2Bmodelling%253A%2Bcombining%2Bphysiological%2Band%2Bspatial%2Bdata%2Bto%2Bpredict%2Bspecies%2527%2Branges.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2008.01277.x%26rft_id%253Dinfo%253Apmid%252F19292794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2008.01277.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19292794&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F28%2F586909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264067600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kellner</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">jagsUI: A wrapper around rjags to streamline JAGS analyses</span>. <abbr class="cit-jnl-abbrev">R package version 1</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kéry</span>,  <span class="cit-name-given-names">M</span></span>. <span class="cit-pub-date">2011</span>. <span class="cit-article-title">Towards the modelling of true species distributions</span>. <abbr class="cit-jnl-abbrev">Journal of Biogeography</abbr> <span class="cit-vol">38</span>:<span class="cit-fpage">617</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiogeography%26rft.volume%253D38%26rft.spage%253D617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-586909v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kéry</span>,  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Royle</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-chapter-title">Applied hierarchical modeling in ecology: Analysis of distribution, abundance and species richness in R and BUGS</span>: Volume <span class="cit-vol">1</span>: <span class="cit-source">Prelude and static models</span>. <span class="cit-publ-name">Academic Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuo</span>,  <span class="cit-name-given-names">L.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Mallick</span></span>. <span class="cit-pub-date">1998</span>. <span class="cit-article-title">Variable selection for regression models</span>. <abbr class="cit-jnl-abbrev">The Indian Journal of Statistics, Series B</abbr> <span class="cit-vol">60</span>:<span class="cit-fpage">65</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BIndian%2BJournal%2Bof%2BStatistics%252C%2BSeries%2BB%26rft.volume%253D60%26rft.spage%253D65%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.26" data-doi="10.1890/0012-9658(2002)083[2832:AHAOPC]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Link</span>,  <span class="cit-name-given-names">W. A.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Sauer</span></span>. <span class="cit-pub-date">2002</span>. <span class="cit-article-title">A hierarchical analysis of population change with application to Cerulean Warblers</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">83</span>:<span class="cit-fpage">2832</span>–<span class="cit-lpage">2840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D83%26rft.spage%253D2832%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9658%25282002%2529083%255B2832%253AAHAOPC%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9658(2002)083[2832:AHAOPC]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000179348800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lunn</span>,  <span class="cit-name-given-names">D. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Thomas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Best</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Spiegelhalter</span></span>. <span class="cit-pub-date">2000</span>. <span class="cit-article-title">WinBUGS-a Bayesian modelling framework: Concepts, structure, and extensibility</span>. <abbr class="cit-jnl-abbrev">Statistics and Computing</abbr> <span class="cit-vol">10</span>:<span class="cit-fpage">325</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistics%2Band%2BComputing%26rft.volume%253D10%26rft.spage%253D325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.28" data-doi="10.1890/02-3090"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>,  <span class="cit-name-given-names">D. I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Nichols</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Hines</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Knutson</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Franklin</span></span>. <span class="cit-pub-date">2003</span>. <span class="cit-article-title">Estimating site occupancy, colonization, and local extinction when a species is detected imperfectly</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">84</span>:<span class="cit-fpage">2200</span>–<span class="cit-lpage">2207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D84%26rft.spage%253D2200%26rft_id%253Dinfo%253Adoi%252F10.1890%252F02-3090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/02-3090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000185073100026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-586909v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>,  <span class="cit-name-given-names">D. I.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Nichols</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Royle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Pollock</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bailey</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Hines</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-source">Occupancy estimation and modeling: Inferring patterns and dynamics of species occurrence</span>. <span class="cit-publ-name">Elsevier</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ntzoufras</span>,  <span class="cit-name-given-names">I.</span></span> <span class="cit-pub-date">2002</span>. <span class="cit-article-title">Gibbs variable selection using BUGS</span>. <abbr class="cit-jnl-abbrev">Journal of statistical software</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstatistical%2Bsoftware%26rft.volume%253D7%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.31" data-doi="10.5066/F7W0944J"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pardieck</span>,  <span class="cit-name-given-names">K. L.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Ziolkowski</span>   <span class="cit-name-suffix">Jr</span>.</span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lutmerding</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Campbell</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">M.-A. R.</span> <span class="cit-name-surname">Hudson</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">North american breeding bird survey dataset 1966-2015, version 2015.0</span>. <abbr class="cit-jnl-abbrev">U.S. Geological Survey, Patuxent Wildlife Research Center</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.5066/F7W0944J</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DU.S.%2BGeological%2BSurvey%252C%2BPatuxent%2BWildlife%2BResearch%2BCenter%26rft_id%253Dinfo%253Adoi%252F10.5066%252FF7W0944J%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.5066/F7W0944J&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Plummer</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2012</span>. <span class="cit-article-title">JAGS: Just another Gibbs sampler</span>. <abbr class="cit-jnl-abbrev">Astrophysics Source Code Library</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-586909v1.33"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>. <span class="cit-pub-date">2016</span>. <span class="cit-source">R: A language and environment for statistical computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.34" data-doi="10.1111/j.1541-0420.2006.00617.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reich</span>,  <span class="cit-name-given-names">B. J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Hodges</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Zadnik</span></span>. <span class="cit-pub-date">2006</span>. <span class="cit-article-title">Effects of residual smoothing on the posterior of the fixed effects in disease-mapping models</span>. <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">62</span>:<span class="cit-fpage">1197</span>–<span class="cit-lpage">1206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DReich%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D62%26rft.issue%253D4%26rft.spage%253D1197%26rft.epage%253D1206%26rft.atitle%253DEffects%2Bof%2Bresidual%2Bsmoothing%2Bon%2Bthe%2Bposterior%2Bof%2Bthe%2Bfixed%2Beffects%2Bin%2Bdisease-mapping%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1541-0420.2006.00617.x%26rft_id%253Dinfo%253Apmid%252F17156295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1541-0420.2006.00617.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17156295&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F03%2F28%2F586909.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242771800027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rich</span>,  <span class="cit-name-given-names">J. L.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Currie</span></span>. <span class="cit-pub-date">2018</span>. <span class="cit-article-title">Are north american bird species’ geographic ranges mainly determined by climate?</span> <abbr class="cit-jnl-abbrev">Global Ecology and Biogeography</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.36"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>,  <span class="cit-name-given-names">K. V.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Will</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Buehler</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. B.</span> <span class="cit-name-surname">Swarthout</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. E.</span> <span class="cit-name-surname">Thogmartin</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Bennett</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Chandler</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Dynamic distributions and population declines of golden-winged warblers</span>. <abbr class="cit-jnl-abbrev">Studies in Avian Biology</abbr> <span class="cit-vol">49</span>:<span class="cit-fpage">3</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStudies%2Bin%2BAvian%2BBiology%26rft.volume%253D49%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rushing</span>,  <span class="cit-name-given-names">C. S.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T. B.</span> <span class="cit-name-surname">Ryder</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Scarpignato</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Saracco</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">P. P.</span> <span class="cit-name-surname">Marra</span></span>. <span class="cit-pub-date">2016</span>. <span class="cit-article-title">Using demographic attributes from long-term monitoring data to delineate natural population structure</span>. <abbr class="cit-jnl-abbrev">Journal of applied ecology</abbr> <span class="cit-vol">53</span>:<span class="cit-fpage">491</span>–<span class="cit-lpage">500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bapplied%2Becology%26rft.volume%253D53%26rft.spage%253D491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sauer</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hines</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fallon</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Pardieck</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ziolkowski</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Link</span></span>. <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Breeding Bird Survey Summary and Analysis 1966-2013</span>. Version 01.30.2015. <abbr class="cit-jnl-abbrev">USGS Patuxent Wildlife Research Center Laurel MD</abbr>:<a href="http://www.mbr-pwrc.usgs.gov/bbs/bbs.html">http://www.mbr-pwrc.usgs.gov/bbs/bbs.html</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tingley</span>,  <span class="cit-name-given-names">M. W.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Beissinger</span></span>. <span class="cit-pub-date">2009</span>. <span class="cit-article-title">Detecting range shifts from historical species occurrences: New perspectives on old data</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">24</span>:<span class="cit-fpage">625</span>–<span class="cit-lpage">633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D24%26rft.spage%253D625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.40"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Twedt</span>,  <span class="cit-name-given-names">D. J.</span></span> <span class="cit-pub-date">2015</span>. <span class="cit-article-title">Estimating regional landbird populations from enhanced North American Breeding Bird Surveys</span>. <abbr class="cit-jnl-abbrev">Journal of Field Ornithology</abbr> <span class="cit-vol">86</span>:<span class="cit-fpage">352</span>–<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BField%2BOrnithology%26rft.volume%253D86%26rft.spage%253D352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Valpine</span>,  <span class="cit-name-given-names">P.de</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Turek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Paciorek</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Anderson-Bergman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. T.</span> <span class="cit-name-surname">Lang</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bodik</span></span>. <span class="cit-pub-date">2017</span>. <span class="cit-article-title">Programming with models: Writing statistical algorithms for general model structures with NIMBLE</span>. <abbr class="cit-jnl-abbrev">Journal of Computational and Graphical Statistics</abbr> <span class="cit-vol">26</span>:<span class="cit-fpage">403</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BComputational%2Band%2BGraphical%2BStatistics%26rft.volume%253D26%26rft.spage%253D403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-586909v1.42" data-doi="10.1080/00063659909477239"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">White</span>,  <span class="cit-name-given-names">G. C.</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Burnham</span></span>. <span class="cit-pub-date">1999</span>. <span class="cit-article-title">Program MARK: Survival estimation from populations of marked animals</span>. <abbr class="cit-jnl-abbrev">Bird Study</abbr> <span class="cit-vol">46</span>:<span class="cit-fpage">S120</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBird%2BStudy%26rft.volume%253D46%26rft.spage%253DS120%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00063659909477239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00063659909477239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-586909v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wood</span>,  <span class="cit-name-given-names">S. N.</span></span> <span class="cit-pub-date">2017</span>. <span class="cit-source">Generalized additive models: An introduction with R</span>. <span class="cit-publ-name">CRC press</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/586909v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F586909v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/592014v1" title="Damage to the Right Insula Disrupts the Perception of Affective Touch" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;March 28, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/586909v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/586909v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/586909v1" class="active" data-icon-position="" data-hide-link-title="0">Estimating spatially and temporally complex range dynamics when detection is imperfect</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/386979" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8Zdi1_9zhAAt0SrHf5dcrVJswXyqBnH3COI-catG_Cs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="386979" id="node386979--2" data-pisa="biorxiv;586909v1" data-pisa-master="biorxiv;586909" data-seqnum="386979" data-apath="/biorxiv/early/2019/03/28/586909.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Estimating spatially and temporally complex range dynamics when detection is imperfect</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Clark S.</span> <span class="nlm-surname">Rushing</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">J. Andrew</span> <span class="nlm-surname">Royle</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">David J.</span> <span class="nlm-surname">Ziolkowski</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Keith L.</span> <span class="nlm-surname">Pardieck</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">586909; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/586909 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/586909v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/586909v1&amp;title=Estimating%20spatially%20and%20temporally%20complex%20range%20dynamics%20when%20detection%20is%20imperfect" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/586909v1&amp;title=Estimating%20spatially%20and%20temporally%20complex%20range%20dynamics%20when%20detection%20is%20imperfect" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/586909v1&amp;text=Estimating%20spatially%20and%20temporally%20complex%20range%20dynamics%20when%20detection%20is%20imperfect" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/586909v1&amp;title=Estimating%20spatially%20and%20temporally%20complex%20range%20dynamics%20when%20detection%20is%20imperfect" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/586909v1&amp;t=Estimating%20spatially%20and%20temporally%20complex%20range%20dynamics%20when%20detection%20is%20imperfect" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/586909v1&amp;title=Estimating%20spatially%20and%20temporally%20complex%20range%20dynamics%20when%20detection%20is%20imperfect" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/586909v1&amp;title=Estimating%20spatially%20and%20temporally%20complex%20range%20dynamics%20when%20detection%20is%20imperfect" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="386979" id="node386979--3" data-pisa="biorxiv;586909v1" data-pisa-master="biorxiv;586909" data-seqnum="386979" data-apath="/biorxiv/early/2019/03/28/586909.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Estimating spatially and temporally complex range dynamics when detection is imperfect</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Clark S.</span> <span class="nlm-surname">Rushing</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">J. Andrew</span> <span class="nlm-surname">Royle</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">David J.</span> <span class="nlm-surname">Ziolkowski</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Keith L.</span> <span class="nlm-surname">Pardieck</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">586909; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/586909 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Estimating%20spatially%20and%20temporally%20complex%20range%20dynamics%20when%20detection%20is%20imperfect&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Rushing%2C+Clark+S.&amp;rft.au=Royle%2C+J.+Andrew&amp;rft.au=Ziolkowski%2C+David+J.&amp;rft.au=Pardieck%2C+Keith+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/386979/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/386979/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/386979/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/386979/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/386979/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/386979/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/386979/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/386979/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/386979/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/386979/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/386979/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/386979/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/586909v1&amp;count=horizontal&amp;via=&amp;text=Estimating%20spatially%20and%20temporally%20complex%20range%20dynamics%20when%20detection%20is%20imperfect&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/586909v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/586909v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/586909v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
