<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/embed/graphic-14.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/10/31/826149.atom" />
<meta name="HW.pisa" content="biorxiv;826149v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA" />
<meta name="DC.Identifier" content="10.1101/826149" />
<meta name="DC.Date" content="2019-10-31" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The European continent was subject to two major migrations of peoples during the Holocene: the northwestward movement of Anatolian farmer populations during the Neolithic and the westward movement of Yamnaya steppe peoples during the Bronze Age. These movements changed the genetic composition of the continent’s inhabitants, via admixture and population replacement processes. The Holocene was also characterized by major changes in vegetation composition, which altered the environment occupied by the original hunter-gatherer populations. Here, we use a combination of paleogenomics and geostatistical modelling to produce detailed maps of the movement of these populations over time and space, and to understand how these movements impacted the European vegetation landscape. We find that the dilution of hunter-gatherer ancestries and the Yamnaya steppe migration coincided with a reduction in the amount of broad-leaf forest and an increase in the amount of pasture lands in the continent. We also show that climate played a role in these vegetational changes. Additionally, we find that the spread of Neolithic farmer ancestry had a two-pronged wavefront, in agreement with similar findings based on patterns of the cultural spread of farming from radiocarbon-dated archaeological sites. With thousands of ancient genomes publicly available and in production, we foresee that the integration of ancient DNA with geostatistical techniques and large-scale archaeological datasets will revolutionize the study of ancient populations movements, and their effects on local fauna and flora." />
<meta name="DC.Contributor" content="Fernando Racimo" />
<meta name="DC.Contributor" content="Jessie Woodbridge" />
<meta name="DC.Contributor" content="Ralph M. Fyfe" />
<meta name="DC.Contributor" content="Martin Sikora" />
<meta name="DC.Contributor" content="Karl-Göran Sjögren" />
<meta name="DC.Contributor" content="Kristian Kristiansen" />
<meta name="DC.Contributor" content="Marc Vander Linden" />
<meta name="article:published_time" content="2019-10-31" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;The European continent was subject to two major migrations of peoples during the Holocene: the northwestward movement of Anatolian farmer populations during the Neolithic and the westward movement of Yamnaya steppe peoples during the Bronze Age. These movements changed the genetic composition of the continent’s inhabitants, via admixture and population replacement processes. The Holocene was also characterized by major changes in vegetation composition, which altered the environment occupied by the original hunter-gatherer populations. Here, we use a combination of paleogenomics and geostatistical modelling to produce detailed maps of the movement of these populations over time and space, and to understand how these movements impacted the European vegetation landscape. We find that the dilution of hunter-gatherer ancestries and the Yamnaya steppe migration coincided with a reduction in the amount of broad-leaf forest and an increase in the amount of pasture lands in the continent. We also show that climate played a role in these vegetational changes. Additionally, we find that the spread of Neolithic farmer ancestry had a two-pronged wavefront, in agreement with similar findings based on patterns of the cultural spread of farming from radiocarbon-dated archaeological sites. With thousands of ancient genomes publicly available and in production, we foresee that the integration of ancient DNA with geostatistical techniques and large-scale archaeological datasets will revolutionize the study of ancient populations movements, and their effects on local fauna and flora.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;826149v1" />
<meta name="citation_id" content="826149v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/826149v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/826149v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/826149v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149.full.pdf" />
<meta name="citation_doi" content="10.1101/826149" />
<meta name="citation_num_pages" content="33" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="826149" />
<meta name="citation_author" content="Fernando Racimo" />
<meta name="citation_author_institution" content="Lundbeck GeoGenetics Centre, The Globe Institute, University of Copenhagen" />
<meta name="citation_author_email" content="fracimo@bio.ku.dk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5025-2607" />
<meta name="citation_author" content="Jessie Woodbridge" />
<meta name="citation_author_institution" content="School of Geography, Earth and Environmental Sciences, University of Plymouth" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0756-3538" />
<meta name="citation_author" content="Ralph M. Fyfe" />
<meta name="citation_author_institution" content="School of Geography, Earth and Environmental Sciences, University of Plymouth" />
<meta name="citation_author" content="Martin Sikora" />
<meta name="citation_author_institution" content="Lundbeck GeoGenetics Centre, The Globe Institute, University of Copenhagen" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2818-8319" />
<meta name="citation_author" content="Karl-Göran Sjögren" />
<meta name="citation_author_institution" content="Department of Historical Studies, University of Gothenburg" />
<meta name="citation_author" content="Kristian Kristiansen" />
<meta name="citation_author_institution" content="Department of Historical Studies, University of Gothenburg" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2423-308X" />
<meta name="citation_author" content="Marc Vander Linden" />
<meta name="citation_author_institution" content="Department of Archaeology, University of Cambridge" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0120-7754" />
<meta name="citation_reference" content="Allentoft, Morten E et al. (2015). “Population genomics of bronze age Eurasia”. In: Nature 522.7555, p. 167." />
<meta name="citation_reference" content="Ammerman, Albert J and Luigi L Cavalli-Sforza (1971). “Measuring the rate of spread of early farming in Europe”. In: Man, pp. 674–688." />
<meta name="citation_reference" content="Anthony, David W (2010). The horse, the wheel, and language: how Bronze-Age riders from the Eurasian steppes shaped the modern world. Princeton University Press." />
<meta name="citation_reference" content="Bakar, Khandoker Shuvo, Sujit K Sahu, et al. (2015). “spTimer: Spatio-temporal bayesian modelling using R”. In: Journal of Statistical Software 63.15, pp. 1–32." />
<meta name="citation_reference" content="Barnett, William K (2000). “Cardial pottery and the agricultural transition in Mediterranean Europe”. In: Europe’s first farmers, pp. 93–116." />
<meta name="citation_reference" content="Battey, CJ, Peter L Ralph, and Andrew D Kern (2019). “Space is the Place: Effects of Continuous Spatial Structure on Analysis of Population Genetic Data”. In: BioRxiv, p. 659235." />
<meta name="citation_reference" content="Bevan, Andrew et al. (2017). “Holocene fluctuations in human population demonstrate repeated links to food production and climate”. In: Proceedings of the National Academy of Sciences 114.49, E10524–E10531." />
<meta name="citation_reference" content="Bickle, Penny and Alasdair Whittle (2013). The first farmers of central Europe: diversity in LBK lifeways. Oxbow Books." />
<meta name="citation_reference" content="Binder, Didier et al. (2018). “Modelling the earliest north-western dispersal of Mediterranean Impressed Wares: new dates and Bayesian chronological model.” In: Documenta praehistorica. 44, pp. 54–77." />
<meta name="citation_reference" content="Bishop, Rosie R, Mike J Church, and Peter A Rowley-Conwy (2015). “Firewood, food and human niche construction: the potential role of Mesolithic hunter–gatherers in actively structuring Scotland’s woodlands”. In: Quaternary Science Reviews 108, pp. 51–75." />
<meta name="citation_reference" content="Borcard, Daniel, François Gillet, and Pierre Legendre (2018). Numerical ecology with R. Springer." />
<meta name="citation_reference" content="Bradburd, Gideon S, Graham M Coop, and Peter L Ralph (2018). “Inferring continuous and discrete population genetic structure across space”. In: Genetics 210.1, pp. 33–52." />
<meta name="citation_reference" content="Brown, Jason L et al. (2018). “PaleoClim, high spatial resolution paleoclimate surfaces for global land areas”. In: Scientific data 5, p. 180254." />
<meta name="citation_reference" content="Cheng, Jade Yu, Thomas Mailund, and Rasmus Nielsen (2017). “Fast admixture analysis and population tree estimation for SNP and NGS data”. In: Bioinformatics 33.14, pp. 2148–2155." />
<meta name="citation_reference" content="Collins, William D et al. (2006). “The community climate system model version 3 (CCSM3)”. In: Journal of Climate 19.11, pp. 2122–2143." />
<meta name="citation_reference" content="Cressie, Noel and Christopher K Wikle (2015). Statistics for spatio-temporal data. John Wiley &amp; Sons." />
<meta name="citation_reference" content="de Barros Damgaard, Peter et al. (2018). “137 ancient human genomes from across the Eurasian steppes”. In: Nature 557.7705, p. 369." />
<meta name="citation_reference" content="Excoffier, Laurent et al. (2013). “Robust demographic inference from genomic and SNP data”. In: PLoS genetics 9.10, e1003905." />
<meta name="citation_reference" content="Fordham, Damien A et al. (2017). “PaleoView: a tool for generating continuous climate projections spanning the last 21 000 years at regional and global scales”. In: Ecography 40.11, pp. 1348–1358." />
<meta name="citation_reference" content="Fort, Joaquim (2015). “Demic and cultural diffusion propagated the Neolithic transition across different regions of Europe”. In: Journal of the Royal Society interface 12.106, p. 20150166." />
<meta name="citation_reference" content="Fort, Joaquim (2018). “The Neolithic Transition: Diffusion of People or Diffusion of Culture?” In: Diffusive spreading in nature, technology and society. Springer, pp. 313–331." />
<meta name="citation_reference" content="Frantz, AC et al. (2009). “Using spatial Bayesian methods to determine the genetic structure of a continuously distributed population: clusters or isolation by distance?” In: Journal of Applied Ecology 46.2, pp. 493–505." />
<meta name="citation_reference" content="Fu, Qiaomei et al. (2016). “The genetic history of ice age Europe”. In: Nature 534.7606, p. 200." />
<meta name="citation_reference" content="Fyfe, Ralph M, Claire Twiddle, et al. (2013). “The Holocene vegetation cover of Britain and Ireland: overcoming problems of scale and discerning patterns of openness”. In: Quaternary Science Reviews 73, pp. 132–148." />
<meta name="citation_reference" content="Fyfe, Ralph M, Jessie Woodbridge, and Neil Roberts (2015). “From forest to farmland: polleninferred land cover change across Europe using the pseudobiomization approach”. In: Global Change Biology 21.3, pp. 1197–1212." />
<meta name="citation_reference" content="Fyfe, Ralph, Neil Roberts, and Jessie Woodbridge (2010). “A pollen-based pseudobiomisation approach to anthropogenic land-cover change”. In: The Holocene 20.7, pp. 1165–1171." />
<meta name="citation_reference" content="Haak, Wolfgang et al. (2015). “Massive migration from the steppe was a source for Indo-European languages in Europe”. In: Nature 522.7555, p. 207." />
<meta name="citation_reference" content="Hellenthal, Garrett et al. (2014). “A genetic atlas of human admixture history”. In: Science 343.6172, pp. 747–751." />
<meta name="citation_reference" content="Hunt, Harriet V et al. (2018). “Genetic evidence for a western Chinese origin of broomcorn millet (Panicum miliaceum)”. In: The Holocene 28.12, pp. 1968–1978." />
<meta name="citation_reference" content="Jacomet, Stefanie (2008). Subsistenz und Landnutzung während des 3. Jahrtausends v. Chr. aufgrund von archäobotanischen Daten aus dem südwestlichen Mitteleuropa. 84. Wachholtz." />
<meta name="citation_reference" content="Janes, Jasmine K et al. (2017). “The K= 2 conundrum”. In: Molecular Ecology 26.14, pp. 3594–3602." />
<meta name="citation_reference" content="Joseph, Tyler A and Itsik Pe’er (2018). “Inference of population structure from ancient DNA”. In: International Conference on Research in Computational Molecular Biology. Springer, pp. 90–104." />
<meta name="citation_reference" content="Kamm, Jack et al. (2019). “Efficiently inferring the demographic history of many populations with allele count data”. In: Journal of the American Statistical Association just-accepted, pp. 1–42." />
<meta name="citation_reference" content="Kelleher, Jerome et al. (2018). “Inferring the ancestry of everyone”. In: BioRxiv, p. 458067." />
<meta name="citation_reference" content="Kolář, Jan et al. (2018). “Population and forest dynamics during the Central European Eneolithic (4500-2000 BC)”. In: Archaeological and anthropological sciences 10.5, pp. 1153–1164." />
<meta name="citation_reference" content="Kristiansen, Kristian, Morten E Allentoft, et al. (2017). “Re-theorising mobility and the formation of culture and language among the Corded Ware Culture in Europe”. In: Antiquity 91.356, pp. 334–347." />
<meta name="citation_reference" content="Kristiansen, Kristian and Thomas B Larsson (2005). The rise of Bronze Age society: travels, transmissions and transformations. Cambridge University Press." />
<meta name="citation_reference" content="Lawson, Daniel J, Lucy Van Dorp, and Daniel Falush (2018). “A tutorial on how not to overinterpret STRUCTURE and ADMIXTURE bar plots”. In: Nature Communications 9.1, p. 3258." />
<meta name="citation_reference" content="Lawson, Daniel John et al. (2012). “Inference of population structure using dense haplotype data”. In: PLoS genetics 8.1, e1002453." />
<meta name="citation_reference" content="Lazaridis, Iosif, Dani Nadel, et al. (2016). “Genomic insights into the origin of farming in the ancient Near East”. In: Nature 536.7617, p. 419." />
<meta name="citation_reference" content="Lazaridis, Iosif, Nick Patterson, et al. (2014). “Ancient human genomes suggest three ancestral populations for present-day Europeans”. In: Nature 513.7518, p. 409." />
<meta name="citation_reference" content="Lechterbeck, Jutta et al. (2014). “Is Neolithic land use correlated with demography? An evaluation of pollen-derived land cover and radiocarbon-inferred demographic change from Central Europe”. In: The Holocene 24.10, pp. 1297–1307." />
<meta name="citation_reference" content="Liu, Zhengyu, Zhengyao Lu, et al. (2014). “Evolution and forcing mechanisms of El Niño over the past 21,000 years”. In: Nature 515.7528, p. 550." />
<meta name="citation_reference" content="Liu, Zhengyu, BL Otto-Bliesner, et al. (2009). “Transient simulation of last deglaciation with a new mechanism for Bølling-Allerød warming”. In: Science 325.5938, pp. 310–314." />
<meta name="citation_reference" content="Manen, Claire et al. (2019). “The Neolithic Transition in the Western Mediterranean: a Complex and Non-Linear Diffusion Process—The Radiocarbon Record Revisited”. In: Radiocarbon 61.2, pp. 531–571." />
<meta name="citation_reference" content="Margaryan, Ashot et al. (2019). “Population genomics of the Viking world”. In: bioRxiv, p. 703405." />
<meta name="citation_reference" content="Marquer, Laurent et al. (2017). “Quantifying the effects of land use and climate on Holocene vegetation in Europe”. In: Quaternary Science Reviews 171, pp. 20–37." />
<meta name="citation_reference" content="Matérn, Bertil (1986). Spatial variation, volume 36 of Lecture Notes in Statistics." />
<meta name="citation_reference" content="Mathieson, Iain, Songül Alpaslan-Roodenberg, et al. (2018). “The genomic history of southeastern Europe”. In: Nature 555.7695, p. 197." />
<meta name="citation_reference" content="Mathieson, Iain, Iosif Lazaridis, et al. (2015). “Genome-wide patterns of selection in 230 ancient Eurasians”. In: Nature 528.7583, p. 499." />
<meta name="citation_reference" content="Mittnik, Alissa et al. (2019). “Kinship-based social inequality in Bronze Age Europe”. In: Science, eaax6219." />
<meta name="citation_reference" content="Müller, Johannes (2015). “Eight Million Neolithic Europeans: Social Demography and Social Archaeology on the Scope of Change–From the Near East to Scandinavia”. In: Paradigm found: archaeological theory present past and future: essays in honour of Evžen Neustupn. Oxford: Oxbow, pp. 200–214." />
<meta name="citation_reference" content="Müller, Johannes et al. (2009). “A revision of Corded Ware settlement pattern–new results from the Central European low mountain range”. In: Proceedings of the Prehistoric Society. Vol. 75. Cambridge University Press, pp. 125–142." />
<meta name="citation_reference" content="Nielsen, Anne Birgitte et al. (2012). “Quantitative reconstructions of changes in regional openness in north-central Europe reveal new insights into old questions”. In: Quaternary Science Reviews 47, pp. 131–149." />
<meta name="citation_reference" content="Olalde, Iñigo et al. (2018). “The Beaker phenomenon and the genomic transformation of northwest Europe”. In: Nature 555.7695, p. 190." />
<meta name="citation_reference" content="Otto-Bliesner, Bette L et al. (2006). “Climate sensitivity of moderate-and low-resolution versions of CCSM3 to preindustrial forcings”. In: Journal of Climate 19.11, pp. 2567–2583." />
<meta name="citation_reference" content="Patterson, Nick et al. (2012). “Ancient admixture in human history”. In: Genetics 192.3, pp. 1065–1093." />
<meta name="citation_reference" content="Pebesma, Edzer et al. (2012). “spacetime: Spatio-temporal data in r”. In: Journal of Statistical Software 51.7, pp. 1–30." />
<meta name="citation_reference" content="Pebesma, Edzer J (2004). “Multivariable geostatistics in S: the gstat package”. In: Computers &amp; Geosciences 30.7, pp. 683–691." />
<meta name="citation_reference" content="Pebesma, Edzer and Gerard Heuvelink (2016). “Spatio-temporal interpolation using gstat”. In: RFID Journal 8.1, pp. 204–218." />
<meta name="citation_reference" content="Pinhasi, Ron, Joaquim Fort, and Albert J Ammerman (2005). “Tracing the origin and spread of agriculture in Europe”. In: PLoS biology 3.12, e410." />
<meta name="citation_reference" content="Roberts, C Neil et al. (2019). “Mediterranean landscape change during the Holocene: Synthesis, comparison and regional trends in population, land cover and climate”. In: The Holocene 29.5, pp. 923–937." />
<meta name="citation_reference" content="Roberts, Neil et al. (2018). “Europe’s lost forests: a pollen-based synthesis for the last 11,000 years”. In: Scientific reports 8.1, p. 716." />
<meta name="citation_reference" content="Sánchez-Quinto, Federico et al. (2012). “Genomic affinities of two 7,000-year-old Iberian huntergatherers”. In: Current biology 22.16, pp. 1494–1499." />
<meta name="citation_reference" content="Schauer, Peter et al. (2019). “Supply and demand in prehistory? Economics of Neolithic mining in northwest Europe”. In: Journal of Anthropological Archaeology 54, pp. 149–160." />
<meta name="citation_reference" content="Seregély, Timo and Johannes Müller (2008). “Endneolithische Siedlungsstrukturen in Oberfranken: Wattendorf-Motzenstein: eine schnurkeramische Siedlung in der nördlichen Frankenalb”. In:" />
<meta name="citation_reference" content="Shennan, Stephen et al. (2013). “Regional population collapse followed initial agriculture booms in mid-Holocene Europe”. In: Nature communications 4, p. 2486." />
<meta name="citation_reference" content="Shishlina, Natalia I (2008). Reconstruction of the Bronze Age of the Caspian steppes: life styles and life ways of pastoral nomads. British Archaeological Reports Ltd." />
<meta name="citation_reference" content="Sikora, Martin et al. (2014). “Population genomic analysis of ancient and modern genomes yields new insights into the genetic ancestry of the Tyrolean Iceman and the genetic structure of Europe”. In: PLoS genetics 10.5, e1004353." />
<meta name="citation_reference" content="Silva, Fabio and James Steele (2014). “New methods for reconstructing geographical effects on dispersal rates and routes from large-scale radiocarbon databases”. In: Journal of Archaeological Science 52, pp. 609–620." />
<meta name="citation_reference" content="Sjögren, Karl-Göran, T Douglas Price, and Kristian Kristiansen (2016). “Diet and mobility in the Corded Ware of Central Europe”. In: PloS one 11.5, e0155083." />
<meta name="citation_reference" content="Skoglund, Pontus, Helena Malmström, Ayça Omrak, et al. (2014). “Genomic diversity and admixture differs for Stone-Age Scandinavian foragers and farmers”. In: Science 344.6185, pp. 747–750." />
<meta name="citation_reference" content="Skoglund, Pontus, Helena Malmström, Maanasa Raghavan, et al. (2012). “Origins and genetic legacy of Neolithic farmers and hunter-gatherers in Europe”. In: Science 336.6080, pp. 466–469." />
<meta name="citation_reference" content="Speidel, Leo et al. (2019). “A method for genome-wide genealogy estimation for thousands of samples”. In: BioRxiv, p. 550558." />
<meta name="citation_reference" content="Vander Linden, Marc and Fabio Silva (2018). “Comparing and modeling the spread of early farming across Europe.” In: PAGES magazine 26.1, pp. 28–29." />
<meta name="citation_reference" content="Vandkilde, Helle (2007). Culture and change in Central European prehistory. ISD LLC." />
<meta name="citation_reference" content="Walvoort, Dennis JJ and Jaap J de Gruijter (2001). “Compositional kriging: a spatial interpolation method for compositional data”. In: Mathematical Geology 33.8, pp. 951–966." />
<meta name="citation_reference" content="Warren, Graeme et al. (2014). “The potential role of humans in structuring the wooded landscapes of Mesolithic Ireland: a review of data and discussion of approaches”. In: Vegetation History and Archaeobotany 23.5, pp. 629–646." />
<meta name="citation_reference" content="Woodbridge, Jessie, Ralph M Fyfe, et al. (2014). “The impact of the Neolithic agricultural transition in Britain: a comparison of pollen-based land-cover and archaeological 14C date-inferred population change”. In: Journal of Archaeological Science 51, pp. 216–224." />
<meta name="citation_reference" content="Woodbridge, Jessie, Neil Roberts, and Ralph Fyfe (2018). “Pan-Mediterranean Holocene vegetation and land-cover dynamics from synthesized pollen data”. In: Journal of Biogeography 45.9, pp. 2159–2174." />
<meta name="citation_reference" content="Yeager, Stephen G et al. (2006). “The low-resolution CCSM3”. In:. Journal of Climate 19.11, pp. 2545–2566." />
<meta name="twitter:title" content="A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The European continent was subject to two major migrations of peoples during the Holocene: the northwestward movement of Anatolian farmer populations during the Neolithic and the westward movement of Yamnaya steppe peoples during the Bronze Age. These movements changed the genetic composition of the continent’s inhabitants, via admixture and population replacement processes. The Holocene was also characterized by major changes in vegetation composition, which altered the environment occupied by the original hunter-gatherer populations. Here, we use a combination of paleogenomics and geostatistical modelling to produce detailed maps of the movement of these populations over time and space, and to understand how these movements impacted the European vegetation landscape. We find that the dilution of hunter-gatherer ancestries and the Yamnaya steppe migration coincided with a reduction in the amount of broad-leaf forest and an increase in the amount of pasture lands in the continent. We also show that climate played a role in these vegetational changes. Additionally, we find that the spread of Neolithic farmer ancestry had a two-pronged wavefront, in agreement with similar findings based on patterns of the cultural spread of farming from radiocarbon-dated archaeological sites. With thousands of ancient genomes publicly available and in production, we foresee that the integration of ancient DNA with geostatistical techniques and large-scale archaeological datasets will revolutionize the study of ancient populations movements, and their effects on local fauna and flora." />
<meta name="og-title" property="og:title" content="A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/826149v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The European continent was subject to two major migrations of peoples during the Holocene: the northwestward movement of Anatolian farmer populations during the Neolithic and the westward movement of Yamnaya steppe peoples during the Bronze Age. These movements changed the genetic composition of the continent’s inhabitants, via admixture and population replacement processes. The Holocene was also characterized by major changes in vegetation composition, which altered the environment occupied by the original hunter-gatherer populations. Here, we use a combination of paleogenomics and geostatistical modelling to produce detailed maps of the movement of these populations over time and space, and to understand how these movements impacted the European vegetation landscape. We find that the dilution of hunter-gatherer ancestries and the Yamnaya steppe migration coincided with a reduction in the amount of broad-leaf forest and an increase in the amount of pasture lands in the continent. We also show that climate played a role in these vegetational changes. Additionally, we find that the spread of Neolithic farmer ancestry had a two-pronged wavefront, in agreement with similar findings based on patterns of the cultural spread of farming from radiocarbon-dated archaeological sites. With thousands of ancient genomes publicly available and in production, we foresee that the integration of ancient DNA with geostatistical techniques and large-scale archaeological datasets will revolutionize the study of ancient populations movements, and their effects on local fauna and flora." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-10-31" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/826149v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/826149v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/826149v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/826149v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/974425" />
    <title>A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__AQm9HArb2Y_D7UzgraiLIwAB7088fIq2iZD0Uigie_s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"q5uuwkKKvOXaIyBbFR0vzJYaps56DPORCXzDuVhyvWI","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"974425","apath":"\/biorxiv\/early\/2019\/10\/31\/826149.atom","pisa":"biorxiv;826149v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;826149v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/826149v1","title":"A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA","identifier":"node\/974425"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uTn21qRtp0IEBvcZV+h1PpBxm+eXO3j+aSt2z+X4+BfaXl5yxE3tnvo16xt4VOMt90g5+Nk\\\\\\\/JSJJDF1l3d1Yze2+K36zj1fXrOYw+v0rwBigfS4pArOnR4GJwCuYVHtAaoUjA8Gd8GrvvyMQqafXNyl8\\\\\\\/t8gDL0QNftItmzf+oRTLFC7aEaAI5AvMY3d4nSa3UCdtP5O3rlKZ+qpdVnLQPg6saZ9N9BPQGgtU3qyna+QZ+hC6xAf8rnSWt8dBqXUeqX8OwSsgC46HD4eZ1HAHe625N90AHZPO7rSkKD1wlEUyoX9S5mWvMsQZA0xX8Xwe6L9BckqoXb2rE7SO+InZIhVcfWUjBlYT4ggYPi7oLB7FinZkoufs\\\\\\\/gCk+LoSnoChe0WymAwLqaWwAJ2GlSY7G+JfXpU\\\\\\\/UnH7GwZEo1cqhHj6drY06Hfy+J1Xa1AMJaZbjMdXbZ0Dg+in5+I7HOk3J9tmO66Jmzm0iEad6NAK2raVKaL5mDzDpMgfSaEsAHJf2fj\\\\\\\/MvAl0tDYU7dGEaxzzb+tNxZ5zDZKfb3x8WaJwLbEKt9lpXQiIPHXndLDXNAJTsJ3UtUXquTe1wY\\\\\\\/8s7TyhnPLeXBTx6D5P5R0A1H641nII3pdqQA1HUAfPGSgEMERyWfuLGgIKFRiseiVPFaVMH7Pvn2uiHhhu+HHQ7ost0BycMmzd3fpWNmNE9Htv09d7S2ThBHkxV7BVctlbg4ju608ordGJdtuubh94o1OOUc\\\\\\\/gxv2PwYWCu5cFriXjlc\\\\\\\/riptygm598joMFq6NPUCzm0OGLRNsbKDbtRdKuwuYF6PTK7b62jzkqd7WJlJzjqORId\\\\\\\/xiXcvLUJUz0AOQNj2JVDQPOuruFrbal8YzkFDlwLLXQDubEi3nzpvv7Hsq26DetpWjq8ioyzndMUjwFIEMbOSP29NQsNSkQsZr6kOXO0LCiLkIbP2rUofWXnOUBCnWAqjPKzEaBl0X5fmwVqBF+FPk5fCWStht1himf4dJ5O0jwmqgoNLYGDzfEqtkHS1yc3fHh8M7ESkGHpzpTZs4sxFhVXLrIZbp0rX+8ij1cD66ra5gaAkx\\\\\\\/WUZ2+cp7CZtwZR3Ln7H7dno+OLLbhK3BCMaVmAg0ENKdPV+\\\\\\\/8XBiuL1mxn6PjgxukR9upZBc2sEj95mcwLJy0PRj46N1SacFJw8Cp9Fmmv0VPbeNlPhBBuHu1FroOrTMfa0mpn8ZGSml4T2wa61\\\\\\\/I0RfGWwaMgSj3khZzxelWV8LvYogr9VKUqF5h7XAuZ9riVEA\\\\\\\/4V\\\\\\\/mW1UlcUY040CvCx9pd+FWb0pMefwQ+lN5rcjtXpctU=\\\u0022,\\\u0022iv\\\u0022:\\\u002290Vlzs7CdJQNCLoXwt9kCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022103b2c28eee49c1a655933f9b27ad837411832a8236e4d6462a0fe52c2b255ba\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/826149v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-974425 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/826149v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XzrhkL8kaBfsCCnBCgotV4BNzDViGetXykwxngjy9fM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="974425" id="node974425" data-pisa="biorxiv;826149v1" data-pisa-master="biorxiv;826149" data-apath="/biorxiv/early/2019/10/31/826149.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5025-2607" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Fernando</span> <span class="nlm-surname">Racimo</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0756-3538" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jessie</span> <span class="nlm-surname">Woodbridge</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Ralph M.</span> <span class="nlm-surname">Fyfe</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-2818-8319" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Sikora</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Karl-Göran</span> <span class="nlm-surname">Sjögren</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2423-308X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Kristian</span> <span class="nlm-surname">Kristiansen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-0120-7754" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Marc Vander</span> <span class="nlm-surname">Linden</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/826149 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node974425" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Fernando Racimo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Lundbeck GeoGenetics Centre, The Globe Institute, University of Copenhagen</span>, <span class='nlm-country'>Denmark</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BRacimo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Racimo%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFernando%2BRacimo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5025-2607" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fernando Racimo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:fracimo@bio.ku.dk" class="" data-icon-position="" data-hide-link-title="0">fracimo@bio.ku.dk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jessie Woodbridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of Geography, Earth and Environmental Sciences, University of Plymouth</span>, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessie%2BWoodbridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Woodbridge%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJessie%2BWoodbridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0756-3538" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jessie Woodbridge</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ralph M. Fyfe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of Geography, Earth and Environmental Sciences, University of Plymouth</span>, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ralph%2BM.%2BFyfe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fyfe%20RM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARalph%2BM.%2BFyfe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Martin Sikora </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Lundbeck GeoGenetics Centre, The Globe Institute, University of Copenhagen</span>, <span class='nlm-country'>Denmark</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BSikora%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sikora%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BSikora%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2818-8319" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin Sikora</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Karl-Göran Sjögren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Historical Studies, University of Gothenburg</span>, <span class='nlm-country'>Sweden</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karl-G%C3%B6ran%2BSj%C3%B6gren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sj%C3%B6gren%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKarl-G%25C3%25B6ran%2BSj%25C3%25B6gren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kristian Kristiansen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Historical Studies, University of Gothenburg</span>, <span class='nlm-country'>Sweden</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kristian%2BKristiansen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kristiansen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKristian%2BKristiansen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2423-308X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kristian Kristiansen</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Marc Vander Linden </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Archaeology, University of Cambridge</span>, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marc%2BVander%2BLinden%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Linden%20MV&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarc%2BVander%2BLinden%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0120-7754" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marc Vander Linden</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/826149v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:974425" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:974425/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/826149v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:974425" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:974425/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/826149v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:974425" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:974425/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/826149v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:974425" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:974425/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/826149v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:974425" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:974425/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/826149v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:974425" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:974425/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">The European continent was subject to two major migrations of peoples during the Holocene: the northwestward movement of Anatolian farmer populations during the Neolithic and the westward movement of Yamnaya steppe peoples during the Bronze Age. These movements changed the genetic composition of the continent’s inhabitants, via admixture and population replacement processes. The Holocene was also characterized by major changes in vegetation composition, which altered the environment occupied by the original hunter-gatherer populations. Here, we use a combination of paleogenomics and geostatistical modelling to produce detailed maps of the movement of these populations over time and space, and to understand how these movements impacted the European vegetation landscape. We find that the dilution of hunter-gatherer ancestries and the Yamnaya steppe migration coincided with a reduction in the amount of broad-leaf forest and an increase in the amount of pasture lands in the continent. We also show that climate played a role in these vegetational changes. Additionally, we find that the spread of Neolithic farmer ancestry had a two-pronged wavefront, in agreement with similar findings based on patterns of the cultural spread of farming from radiocarbon-dated archaeological sites. With thousands of ancient genomes publicly available and in production, we foresee that the integration of ancient DNA with geostatistical techniques and large-scale archaeological datasets will revolutionize the study of ancient populations movements, and their effects on local fauna and flora.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Recent paleogenomic studies - often involving hundreds of individuals - have provided new insights into the nature of large-scale population movements during the Holocene (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allentoft et al., 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Haak et al., 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Mathieson, Lazaridis, et al., 2015</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de Barros Damgaard et al., 2018</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mathieson, Alpaslan-Roodenberg, et al., 2018</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">de Barros Damgaard et al., 2018</a>). Although we cannot directly observe migrations of people, as they occurred in the past, we can observe spatial changes in genomic ancestries over time. Under a latent mixed-membership model, these ancestries represent inferred, unobserved ancestral groups that contributed different amounts of genetic material to each of the individuals analyzed, and are found by fitting allele frequency patterns to the genomes of these individuals. Changes in ancestry can indicate when people from distant lands started producing offspring in new, distant locations from their previous homelands. Nevertheless, considerable care must be taken while interpreting these inferred ancestries as true ancestral populations (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Daniel J Lawson, Van Dorp, and Falush, 2018</a>).</p><p id="p-4">Analyses based on patterns of ancestry as well as various other sources of population genetic evidence based on differential allele sharing indicate a wave of populations from the Middle East entered Europe starting ~8,500 years ago (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Sikora et al., 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lazaridis, Patterson, et al., 2014</a>). Furthermore, studies based on large-scale datasets of radiocarbon-dated domestic plants, animals and finds from associated contexts (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Vander Linden and Silva, 2018</a>) indicate that this migration wave spread farming practices into the region, initiating the Neolithic revolution in Europe (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ammerman and Cavalli-Sforza, 1971</a>). A second massive wave of movement occurred at the beginning of the Bronze Age, when populations associated with the Yamnaya culture in the Pontic steppe entered the continent from the east (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anthony, 2010</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Shishlina, 2008</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kristiansen and Larsson, 2005</a>). These groups might have introduced horse-herding and proto-Indo-European languages as they moved westward, and are associated with the Corded Ware culture in central and northern Europe and, later on, the Bell Beaker complex in northwestern Europe (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Haak et al., 2015</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Allentoft et al., 2015</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Olalde et al., 2018</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Vandkilde, 2007</a>).</p><p id="p-5">In parallel, over the last 10,000 years, the European continent has undergone major changes in its land-cover composition. Recent pollen-based studies suggest a dramatic reduction of broad-leaf forests in Europe occurred from about 6,000 BP until the present (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">N. Roberts et al., 2018</a>). The deforestation intensified at around 2,200 BP, resulting in a replacement of these forests by pasture and arable land throughout the continent (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Marquer et al., 2017</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">R. M. Fyfe, Woodbridge, and N. Roberts, 2015</a>). This process, however, did not occur at the same rate throughout all regions of the continent. For example, while considerable decreases in broad-leaf forests occurred in central Europe starting around 4,000 BP, the Atlantic seaboard was predominantly occupied by semi-open vegetation since well before this occurred, while southern Scandinavia underwent much less considerable reductions in forest cover, at least until the Middle Ages (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">R. M. Fyfe, Woodbridge, and N. Roberts, 2015</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Anne Birgitte Nielsen et al., 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">R. M. Fyfe, Twiddle, et al., 2013</a>). Presumably, this phenomenon was partly effected by new human land use activities involving forest clearance and the establishment of farming and herding practices. Most of these activities were introduced into the continent via migration of foreign populations: up until 8,000 BP, Europe was largely populated by groups of hunter-gatherers living at relatively low densities, with likely limited effects on their surrounding flora and fauna (although see <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bishop, Church, and Rowley-Conwy (2015)</a> and <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Warren et al. (2014</a>)). Additionally, changes in climate patterns may have also played a role. Until now, however, few efforts have been carried out to explicitly link changes in paleo-vegetation to particular human population movements, or to distinguish between climatic and human-based factors (but see <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Marquer et al. (2017)</a>).</p><p id="p-6">Paleogenomic ancestry inferences provide a way to trace these migrations in unprecedented spatial detail. In this study, we aim to model the major Holocene migrations using geostatistical methods - like spatiotemporal kriging and hierarchical Bayesian modeling - which are commonly used to model environmental processes. We use these methods to better study these migrations, their relationship to the spread of farming practices and to Holocene vegetation change. Additionally, we estimate their front speed, to better understand how these may have occurred, and compare our results to reconstructions of cultural dispersal obtained from radiocarbon-dated archaeological sites.</p><p id="p-7">Our modeling approach suggests that the decline in broad-leaf forest was concurrent with a decline in hunter-gatherer ancestry, perhaps as a consequence of outside migrations, starting in the Neolithic. The major conversion from forest to pasture land, however, did not occur until fairly late in the Holocene, about 4,000 years ago. Our results suggest that it may have been the fast movement of steppe peoples that kick-started this major conversion. Natural variations in climate patterns during this period also played a role, however. For example, increases in temperatures seem to be related to increases in scrubland, pasture and arable land. Although our analyses do not include other important variables that might have played important roles - like increases in population density in the last 2,000 years - they do point to important factors that may have affected land-cover change in the past 10,000 years and pave the way for future geostatistical studies integrating paleogenomics with other paleo-disciplines.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-8">We downloaded publicly available ancient and present-day genomic sequences typed at the Human Origins SNP array (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Mathieson, Lazaridis, et al., 2015</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Lazaridis, Patterson, et al., 2014</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Patterson et al., 2012</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Allentoft et al., 2015</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Haak et al., 2015</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Mathieson, Alpaslan-Roodenberg, et al., 2018</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Olalde et al., 2018</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lazaridis, Nadel, et al., 2016</a>). We performed unsupervised latent ancestry estimation on each of these sequences using Ohana (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cheng, Mailund, and R. Nielsen, 2017</a>) with K=4 hidden ancestry clusters. We chose this value of K because, under this scheme, three of the components correspond to the three major ancestral populations that have been previously shown to have resulted - via multiple migration and admixture events - into the present-day European gene pool: the original Mesolithic hunter-gatherers (HG), the Neolithic farmers that migrated from the Near East (NEOL), and the Yamnaya steppe peoples that entered Europe during the Bronze Age (YAM) (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Lazaridis, Patterson, et al., 2014</a>; <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Allentoft et al., 2015</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Haak et al., 2015</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The fourth is an ancestry component that remains largely confined to Northern Africa throughout most of the Holocene (NAF). In the following, we largely focus on the first three components and we note that the specification of a larger number of ancestry components could provide further details into more subtle patterns of migration and population expansion, but may also be confounded by bottlenecks and ghost admixture events (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Daniel J Lawson, Van Dorp, and Falush, 2018</a>), so we do not pursue finer ancestry estimation here.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal maps of ancestry proportions for ancient and present-day genomes in this study. The year in each panel&#x2019;s title is the year of the most ancient sample in each panel, so not all ancient samples in each panel are strictly contemporaneous with each other. ANC:HG = hunter-gatherer ancestry. ANC:NEOL = Neolithic farmer ancestry. ANC:YAM = Yamnaya steppe ancestry." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal maps of ancestry proportions for ancient and present-day genomes in this study. The year in each panel&#x2019;s title is the year of the most ancient sample in each panel, so not all ancient samples in each panel are strictly contemporaneous with each other. ANC:HG = hunter-gatherer ancestry. ANC:NEOL = Neolithic farmer ancestry. ANC:YAM = Yamnaya steppe ancestry.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F1.medium.gif" width="283" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F1.medium.gif" width="283" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-9" class="first-child">Spatiotemporal maps of ancestry proportions for ancient and present-day genomes in this study. The year in each panel’s title is the year of the most ancient sample in each panel, so not all ancient samples in each panel are strictly contemporaneous with each other. ANC:HG = hunter-gatherer ancestry. ANC:NEOL = Neolithic farmer ancestry. ANC:YAM = Yamnaya steppe ancestry.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">As we show below, the YAM and NEOL ancestries closely parallel the Yamnaya and Neolithic farmer cultural horizons. However, ancestry and culture are distinct concepts that do not always overlap in time and space, so we choose to use the acronym nomenclature when referring to ancestries, and the full name when referring to cultures, unless otherwise specified. Furthermore, there were various, quite differentiated hunter-gatherer populations (Eastern, Western and Scandinavian hunter-gatherers) that migrated into Western Eurasia before the Holocene (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Skoglund, Malmström, Omrak, et al., 2014</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Skoglund, Malmström, Raghavan, et al., 2012</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sánchez-Quinto et al., 2012</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lazaridis, Nadel, et al., 2016</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fu et al., 2016</a>). The HG ancestry roughly corresponds to the ancestry referred to as “Western hunter-gatherer" in these publications. However, the data for each of these populations is scarcer than for Bronze Age and Neolithic individuals, and, in this work, we chose only to focus on Holocene ancestry movements.</p><p id="p-11">We first sought to compare the spread of dispersal of NEOL and YAM ancestries over time, using the calibrated C14 dates of each genome. Following <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Pinhasi, Fort, and Ammerman (2005)</a> and <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Silva and Steele (2014)</a>, we regressed time against distance from the presumed origin of the spread of each of these ancestries, using the ranged major axis (RMA) method. This allows us to obtain an estimate for the migration front speed. We first used samples that had at least 50% of the corresponding ancestry we were studying, and then used a higher ancestry cutoff in which we only looked at samples with at least 75% of the corresponding ancestry. For the 50% ancestry approach, we find that the speed of the YAM migration (4.2 km/year; CI: 3.5-5.2) was at least twice as fast as the NEOL migration (1.8 km/year; CI: 1.6-2.1), assuming an origin of the Yamnaya migration at the center of its historical range (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2.A</a>). The higher ancestry cutoff approach yields the same estimate for the NEOL migration, but an even faster estimate for the YAM migration (9.4 km/year; CI: 6.1-20). Given that the original Yamnaya range was quite large (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Anthony, 2010</a>), we also aimed to see how our estimates varied as we varied the point of origin within the Yamnaya range. We obtained estimates of speed assuming a location of origin at the northern-most, eastern-most, western-most and southern-most parts of the Yamnaya range, which yielded similar estimates of speed (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>)). The correlation coefficients between time and distance from origin can also be used to estimate the point of origin (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Pinhasi, Fort, and Ammerman, 2005</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hunt et al., 2018</a>), assuming a range expansion for the YAM and NEOL ancestries. Indeed, when we varied the point of origin, we find that the most negative correlation coefficients correspond to Anatolia and the Middle East for the NEOL ancestry and to the Caspian steppe for the YAM ancestry (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2.B</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/977950/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/977950/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/977950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-12" class="first-child">Estimated speed of the YAM ancestry expansion based on the genomes that have more than 50% of this ancestry, obtained by taking the negative inverse of the slope of an RMA regression of the point of origin against the time of sampling. We tried different points of origin at the center and extremes of the Yamnaya historical range. We also list the correlation coefficients obtained from the RMA analysis.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A. Front speed estimation for the Neolithic farmer (left column) and Yamnaya steppe peoples (right column) population movements. We used an RMA regression on time against distance from the hypothesized origin of the spread, to estimate average migration front speed. We used a lenient (&gt; 50%) ancestry cutoff to define genomes as belong to a particular migration wave (top row) and a more conservative (&gt; 75%) ancestry cutoff (bottom row). B. Point of origin estimation. We computed the correlation coefficient between time of sampling and distance from a hypothesized origin, which should be negative for a range expansion. Each dot in the map represents a different hypothesized origin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A. Front speed estimation for the Neolithic farmer (left column) and Yamnaya steppe peoples (right column) population movements. We used an RMA regression on time against distance from the hypothesized origin of the spread, to estimate average migration front speed. We used a lenient (&lt;em&gt;&gt;&lt;/em&gt; 50%) ancestry cutoff to define genomes as belong to a particular migration wave (top row) and a more conservative (&lt;em&gt;&gt;&lt;/em&gt; 75%) ancestry cutoff (bottom row). B. Point of origin estimation. We computed the correlation coefficient between time of sampling and distance from a hypothesized origin, which should be negative for a range expansion. Each dot in the map represents a different hypothesized origin.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F2.medium.gif" width="418" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F2.medium.gif" width="418" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-13" class="first-child">A. Front speed estimation for the Neolithic farmer (left column) and Yamnaya steppe peoples (right column) population movements. We used an RMA regression on time against distance from the hypothesized origin of the spread, to estimate average migration front speed. We used a lenient (<em>&gt;</em> 50%) ancestry cutoff to define genomes as belong to a particular migration wave (top row) and a more conservative (<em>&gt;</em> 75%) ancestry cutoff (bottom row). B. Point of origin estimation. We computed the correlation coefficient between time of sampling and distance from a hypothesized origin, which should be negative for a range expansion. Each dot in the map represents a different hypothesized origin.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">To be able to compare ancestry across time and space with other variables, we aimed to project our ancestry values to particular times and locations for which we do not necessarily have sampled genomes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). To do so, we computed a spatiotemporal variogram and fitted it to a metric covariance function (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">E. J. Pebesma, 2004</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">E. Pebesma and Heuvelink, 2016</a>) (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figures S1</a>,<a id="xref-fig-10-1" class="xref-fig" href="#F10">S2</a>,<a id="xref-fig-11-1" class="xref-fig" href="#F11">S3</a>,<a id="xref-fig-12-1" class="xref-fig" href="#F12">S4</a>). Then, we performed spatiotemporal kriging of the inferred latent ancestry values on a dense grid of spatial points across Europe, over a 10,800-year span, with intervals of 400 years (<a id="xref-fig-12-2" class="xref-fig" href="#F12">Figures 4</a>, <a id="xref-fig-13-1" class="xref-fig" href="#F13">5</a>, <a id="xref-fig-14-1" class="xref-fig" href="#F14">S6</a>, <a id="xref-fig-13-2" class="xref-fig" href="#F13">S5</a>, Supplementary Animations 1-3). In practice, however, given the sparseness of the data in the distant past, we restrict our discussion to patterns seen after 8,000 years BP.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal semivariogram of YAM ancestry in Europe, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal semivariogram of YAM ancestry in Europe, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F9.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F9.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S1:</span> <p id="p-65" class="first-child">Spatiotemporal semivariogram of YAM ancestry in Europe, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal semivariogram of NEOL ancestry in Europe, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal semivariogram of NEOL ancestry in Europe, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F10.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F10.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2:</span> <p id="p-66" class="first-child">Spatiotemporal semivariogram of NEOL ancestry in Europe, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal semivariogram of HG ancestry in Europe, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal semivariogram of HG ancestry in Europe, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F11.medium.gif" width="440" height="249"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F11.medium.gif" width="440" height="249"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3:</span> <p id="p-67" class="first-child">Spatiotemporal semivariogram of HG ancestry in Europe, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal semivariogram of NAF ancestry, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal semivariogram of NAF ancestry, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F12.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F12.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4:</span> <p id="p-68" class="first-child">Spatiotemporal semivariogram of NAF ancestry, with temporal lags of 60-year increments. Left panel: empirical semivariogram. Right panel: fitted semivariogram model using the metric method. The lags are in 60-year increments.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal kriging of NAF ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal kriging of NAF ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F13.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F13.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5:</span> <p id="p-69" class="first-child">Spatiotemporal kriging of NAF ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid.</p><div class="sb-div caption-clear"></div></div></div><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal kriging of HG ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal kriging of HG ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F14.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F14.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S6:</span> <p id="p-70" class="first-child">Spatiotemporal kriging of HG ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of methodology for spatio-temporal kriging and vegetation modelling. Left panel: We first fitted a latent mixed membership model to the ancient and present-day genomes. The ancestry proportions are then assigned the temporal and spatial meta-data of their respective genomes, which allows us to perform spatio-temporal kriging to any location and time in the European Holocene. Right panel: We used a spatiotemporally-aware model to understand how patterns of human migration and climate relate to patterns of vegetation type changes during the European Holocene, while accounting for spatiotemporal autocorrelation. We used a bootstrapping method to account for biases due to uneven sampling of ancient genomes. Brighter colors represent higher values of each depicted variable." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Schematic of methodology for spatio-temporal kriging and vegetation modelling. Left panel: We first fitted a latent mixed membership model to the ancient and present-day genomes. The ancestry proportions are then assigned the temporal and spatial meta-data of their respective genomes, which allows us to perform spatio-temporal kriging to any location and time in the European Holocene. Right panel: We used a spatiotemporally-aware model to understand how patterns of human migration and climate relate to patterns of vegetation type changes during the European Holocene, while accounting for spatiotemporal autocorrelation. We used a bootstrapping method to account for biases due to uneven sampling of ancient genomes. Brighter colors represent higher values of each depicted variable.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F3.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F3.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-15" class="first-child">Schematic of methodology for spatio-temporal kriging and vegetation modelling. Left panel: We first fitted a latent mixed membership model to the ancient and present-day genomes. The ancestry proportions are then assigned the temporal and spatial meta-data of their respective genomes, which allows us to perform spatio-temporal kriging to any location and time in the European Holocene. Right panel: We used a spatiotemporally-aware model to understand how patterns of human migration and climate relate to patterns of vegetation type changes during the European Holocene, while accounting for spatiotemporal autocorrelation. We used a bootstrapping method to account for biases due to uneven sampling of ancient genomes. Brighter colors represent higher values of each depicted variable.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">We also downloaded land cover class (LCC) maps (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">R. M. Fyfe, Woodbridge, and N. Roberts, 2015</a>) and paleo-climatic variable maps (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Brown et al., 2018</a>) spanning the Holocene, and projected them on the same spatiotemporal grid that we used for our kriged ancestry values (see Methods). We computed correlations between each of the spatiotemporally projected ancestry proportions and vegetation types, and between the climate variables and vegetation types. This was done in three different ways. In the first, we simply obtained the correlation of the raw values of any two variables (<a id="xref-fig-15-1" class="xref-fig" href="#F15">Figure S7</a>). In the second, we obtained the correlation of the differences in these variables at each time slice (<a id="xref-fig-17-1" class="xref-fig" href="#F17">Figure S9</a>). In the third, we obtained the correlations of the variable anomalies, defined as the value of each ancient variable after subtracting the present-day value from the same location (<a id="xref-fig-16-1" class="xref-fig" href="#F16">Figure S8</a>). We note, however, that this approach does not account for the autocorrelation in time and space that necessarily exists for all the compared variables, not only because of real autocorrelation in the processes under study, but also because of enforced autocorrelation from the smoothing techniques that generated the maps.</p><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations between kriged ancestry and vegetation type (top) or between climate variables and vegetation type (bottom), computed across time and space. ANC = ancestry. NEOL = neolithic farmer. HG = hunter-gatherer. YAM = Yamnaya. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO7 = Temperature Annual Range (BIO5-BIO6). BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlations between kriged ancestry and vegetation type (top) or between climate variables and vegetation type (bottom), computed across time and space. ANC = ancestry. NEOL = neolithic farmer. HG = hunter-gatherer. YAM = Yamnaya. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO7 = Temperature Annual Range (BIO5-BIO6). BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F15.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S7:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F15.medium.gif" width="416" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S7:</span> <p id="p-71" class="first-child">Correlations between kriged ancestry and vegetation type (top) or between climate variables and vegetation type (bottom), computed across time and space. ANC = ancestry. NEOL = neolithic farmer. HG = hunter-gatherer. YAM = Yamnaya. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO7 = Temperature Annual Range (BIO5-BIO6). BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter.</p><div class="sb-div caption-clear"></div></div></div><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations between anomalies, defined as the values of a variable in each past time slice before the present, after subtracting the value of that variable at the same location in the present. Top: Correlations between kriged ancestry anomalies and vegetation type anomalies. Bottom: Correlations between climate anomalies and vegetation type anomalies. ANC = ancestry. NEOL = neolithic farmer. HG = hunter-gatherer. YAM = Yamnaya. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO7 = Temperature Annual Range (BIO5-BIO6). BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlations between anomalies, defined as the values of a variable in each past time slice before the present, after subtracting the value of that variable at the same location in the present. Top: Correlations between kriged ancestry anomalies and vegetation type anomalies. Bottom: Correlations between climate anomalies and vegetation type anomalies. ANC = ancestry. NEOL = neolithic farmer. HG = hunter-gatherer. YAM = Yamnaya. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO7 = Temperature Annual Range (BIO5-BIO6). BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F16.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S8:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F16.medium.gif" width="416" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S8:</span> <p id="p-72" class="first-child">Correlations between anomalies, defined as the values of a variable in each past time slice before the present, after subtracting the value of that variable at the same location in the present. Top: Correlations between kriged ancestry anomalies and vegetation type anomalies. Bottom: Correlations between climate anomalies and vegetation type anomalies. ANC = ancestry. NEOL = neolithic farmer. HG = hunter-gatherer. YAM = Yamnaya. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO7 = Temperature Annual Range (BIO5-BIO6). BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal kriging of NEOL ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal kriging of NEOL ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F4.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F4.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-17" class="first-child">Spatiotemporal kriging of NEOL ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal kriging of YAM steppe ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal kriging of YAM steppe ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F5.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F5.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5:</span> <p id="p-18" class="first-child">Spatiotemporal kriging of YAM steppe ancestry during the Holocene, using 5000 spatial grid points. The colors represent the predicted ancestry proportion at each point in the grid.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The raw correlations largely reflect spatially static patterns of co-occurrence (<a id="xref-fig-15-2" class="xref-fig" href="#F15">Figure S7</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). For example, YAM ancestry is largely prevalent in northeast Europe throughout much of the Holocene, and this coincides with the prevalence of needle-leaf forests, which is why there is strong positive correlation between these variables. Conversely, NEOL ancestry is largely prevalent in southern Europe during this period, which is why there is a negative correlation with needle-leaf forest.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/977949/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/977949/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/977949" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-20" class="first-child">Correlations between kriged ancestries and paleo-vegetation variables. Raw = correlation of raw variables corresponding to each spatio-temporal grid point. Differences = correlation of differences in raw variables at the same spatial grid point, between consecutive time slices. Anomalies = correlation in anomalies in variables, computed by subtracting the raw variable from its present-day value at the same spatial grid point.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">In contrast, the correlations in differences and in anomalies reflect spatially dynamic patterns of co-occurrence (<a id="xref-fig-17-2" class="xref-fig" href="#F17">Figures S9</a> and <a id="xref-fig-16-2" class="xref-fig" href="#F16">S8</a>, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Here, temporal increases in one variable that coincide with temporal increases in a second variable at the same location will result in positive correlation, The same will result if there are co-occurring decreases. If, however, a variable decreases while another increases at the same location, this will result in negative correlation. For example, we see that YAM ancestry anomalies are positively correlated with pasture land anomalies, but negatively correlated with broad-leaf forest anomalies.</p><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlations between differences in kriged ancestry and vegetation type at each time slice (top) or between differences in climate variables and in vegetation type at each time slice (bottom), computed across time and space. ANC = ancestry. ANC:NEOL = neolithic farmer ancestry. ANC:HG = hunter-gatherer ancestry. ANC:YAM = Yamnaya steppe ancestry. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO7 = Temperature Annual Range (BIO5-BIO6). BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Correlations between differences in kriged ancestry and vegetation type at each time slice (top) or between differences in climate variables and in vegetation type at each time slice (bottom), computed across time and space. ANC = ancestry. ANC:NEOL = neolithic farmer ancestry. ANC:HG = hunter-gatherer ancestry. ANC:YAM = Yamnaya steppe ancestry. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO7 = Temperature Annual Range (BIO5-BIO6). BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F17.medium.gif" width="416" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S9:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F17.medium.gif" width="416" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S9:</span> <p id="p-73" class="first-child">Correlations between differences in kriged ancestry and vegetation type at each time slice (top) or between differences in climate variables and in vegetation type at each time slice (bottom), computed across time and space. ANC = ancestry. ANC:NEOL = neolithic farmer ancestry. ANC:HG = hunter-gatherer ancestry. ANC:YAM = Yamnaya steppe ancestry. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO7 = Temperature Annual Range (BIO5-BIO6). BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Focusing on particular locations of our map, we can better understand the reason for some of these correlations (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). For example, in central France, we observe that increases in YAM ancestry coincided with decreases in broad leaf forest cover, and, later on, with increases in pasture land-cover. This process was not spatially homogeneous, however. In southeastern and southwestern Europe, forest cover remained stable at low levels, even as YAM ancestry was increasing, perhaps due to the development of tree cropping within the agropastoral system in the Mediterranean (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">C. N. Roberts et al., 2019</a>). Considerable increases in arable land-cover occur fairly late throughout the continent, and much later than the incursion of NEOL ancestry during the Neolithic. Interestingly, we observe a decrease in NEOL ancestry that continues even after the incursion of YAM ancestry into Europe, though our resolution for quantifying changes in this ancestry in the last 2,000 years is limited by the scarcity of ancient genomes from the very recent past.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Timelines of kriged ancestry and vegetation type proportions at different points in Europe. NEOL = Neolithic farmer ancestry. HG = Hunter-gatherer ancestry. YAM = Yamnaya steppe ancestry." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Timelines of kriged ancestry and vegetation type proportions at different points in Europe. NEOL = Neolithic farmer ancestry. HG = Hunter-gatherer ancestry. YAM = Yamnaya steppe ancestry.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F6.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F6.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6:</span> <p id="p-23" class="first-child">Timelines of kriged ancestry and vegetation type proportions at different points in Europe. NEOL = Neolithic farmer ancestry. HG = Hunter-gatherer ancestry. YAM = Yamnaya steppe ancestry.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">While these correlation patterns are interesting, they do not account for the fact that we are projecting the data to lie in a particular set of grid points and time slices, which generate complex auto-correlations in time and space, potentially affecting the correlations we observe between variables. To address this, we used a spatiotemporally explicit hierarchical Bayesian model to better understand the relationships between changes in climate, ancestry and paleo-vegetation, while accounting for their autocorrelation in time and space (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). We used two models, implemented in the R package spTimer (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bakar, Sahu, et al., 2015</a>). One is a Gaussian process (GP) model that incorporates a spatiotemporal nugget that is independent of time and has a distribution that depends on a spatial correlation matrix (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). The other is an extension of this method that incorporates a temporal autoregressive component (AR; <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). We set the kriged ancestry and climate variables to be the explanatory variables, while each of the paleo-vegetation variables was set as a response variable. We fitted five separate models for each paleo-vegetation variable.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/977965/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/977965/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/977965" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3:</span> <p id="p-25" class="first-child">Coefficients of the spatiotemporal Gaussian process model, using the PaleoClim (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Brown et al., 2018</a>) simulation-based paleo-climate variables as covariates, with 95% Bayesian credible intervals.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/977953/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/977953/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/977953" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4:</span> <p id="p-26" class="first-child">Coefficients of the spatiotemporal autoregressive model, using the PaleoClim (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Brown et al., 2018</a>) simulation-based paleo-climate variables as covariates, with 95% Bayesian credible intervals.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Posterior mean coefficients of spatiotemporal model for paleo-vegetation anomalies, using kriged ancestry anomalies and anomalies from simulation-based paleo-climate reconstructions as explanatory variables. Upper left and middle panels: posterior coefficients from Gaussian process model. Upper right and bottom panels: coefficients from autoregressive model. Coefficients whose corresponding posterior distribution has a 95% central probability mass interval that spans the value of 0 are not depicted. ANC:NEOL = neolithic farmer ancestry. ANC:HG = hunter-gatherer ancestry. ANC:YAM = Yamnaya steppe ancestry. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Posterior mean coefficients of spatiotemporal model for paleo-vegetation anomalies, using kriged ancestry anomalies and anomalies from simulation-based paleo-climate reconstructions as explanatory variables. Upper left and middle panels: posterior coefficients from Gaussian process model. Upper right and bottom panels: coefficients from autoregressive model. Coefficients whose corresponding posterior distribution has a 95% central probability mass interval that spans the value of 0 are not depicted. ANC:NEOL = neolithic farmer ancestry. ANC:HG = hunter-gatherer ancestry. ANC:YAM = Yamnaya steppe ancestry. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F7.medium.gif" width="384" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F7.medium.gif" width="384" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7:</span> <p id="p-27" class="first-child">Posterior mean coefficients of spatiotemporal model for paleo-vegetation anomalies, using kriged ancestry anomalies and anomalies from simulation-based paleo-climate reconstructions as explanatory variables. Upper left and middle panels: posterior coefficients from Gaussian process model. Upper right and bottom panels: coefficients from autoregressive model. Coefficients whose corresponding posterior distribution has a 95% central probability mass interval that spans the value of 0 are not depicted. ANC:NEOL = neolithic farmer ancestry. ANC:HG = hunter-gatherer ancestry. ANC:YAM = Yamnaya steppe ancestry. The climate variables follow the WorldClim nomenclature. BIO1 = Annual Mean Temperature. BIO2 = Mean Diurnal Range (Mean of monthly (max temp - min temp)). BIO3 = Isothermality (BIO2/BIO7) (* 100). BIO4 = Temperature Seasonality (standard deviation *100). BIO5 = Max Temperature of Warmest Month. BIO6 = Min Temperature of Coldest Month. BIO8 = Mean Temperature of Wettest Quarter. BIO9 = Mean Temperature of Driest Quarter. BIO10 = Mean Temperature of Warmest Quarter. BIO11 = Mean Temperature of Coldest Quarter. BIO12 = Annual Precipitation. BIO13 = Precipitation of Wettest Month. BIO14 = Precipitation of Driest Month. BIO15 = Precipitation Seasonality (Coefficient of Variation). BIO16 = Precipitation of Wettest Quarter. BIO17 = Precipitation of Driest Quarter. BIO18 = Precipitation of Warmest Quarter. BIO19 = Precipitation of Coldest Quarter.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In comparison to the GP model, the AR model results in a more sparse set of posterior co-efficients whose confidence intervals do not overlap with 0 (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). An evaluation of the root mean squared error of the predictions (see Methods) suggests that the GP model with a uniform prior distribution for the spatial decay parameter of the Matérn correlation function generally has the best predictive accuracy (<a id="xref-fig-19-1" class="xref-fig" href="#F19">Figure S11</a>). We also compared the predictive accuracy of models incorporating ancestry only, climate only, both sets of variables or none of them. Our results suggest that, for almost all the paleo-vegetation variables, there is no observable difference in predictive power in adding climate or ancestry, with the exception of pasture (LCC6), which shows the lowest predictive error when including both climate and ancestry under the GP model (<a id="xref-fig-19-2" class="xref-fig" href="#F19">Figure S11</a>).</p><p id="p-29">Because we are using kriged ancestry as an explanatory variable and our ancient genomes are unevenly sampled across space and time, we were mindful that the Bayesian credible intervals (BCI) obtained from the hierarchical model would not accurately reflect uncertainty in particular regions of space-time. For that reason, we performed nonparametric bootstrapping of the parameter estimates. We randomly sampled ancient and present-day genomes with replacement from among the list of all genomes until we had as many genomes as were in the original dataset, then obtained their ancestry assignments, kriged them on the spatiotemporal grid and inputted them into the Bayesian hierarchical model. We did this 100 times to obtain 100 pseudo-samples, which allowed us to obtain 95% bootstrap-based confidence intervals (BBCI) around the mean posterior estimates (<a id="xref-fig-18-1" class="xref-fig" href="#F18">Figure S10</a>, <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Below, we discuss results that are supported by one or both models, and that are also supported by the bootstrapping approach.</p><div id="F18" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F18.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Spatiotemporal Gaussian process model for paleo-vegetation anomalies, using kriged ancestry anomalies and anomalies from simulation-based paleo-climate reconstructions as explanatory variables. Here, we used bootstrapping to account for uncertainty in the ancestry kriging estimation. The depicted lines denote the mean coefficients from the bootstrap distribution. Co-efficients whose bootstrap distribution has a 95% central probability mass interval that spans the value of 0 are not depicted. ANC = ancestry. NEOL = neolithic farmer. HG = hunter-gatherer. YAM = Yamnaya. CLIM = Climate. The climate variables follow the WorldClim nomenclature." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Spatiotemporal Gaussian process model for paleo-vegetation anomalies, using kriged ancestry anomalies and anomalies from simulation-based paleo-climate reconstructions as explanatory variables. Here, we used bootstrapping to account for uncertainty in the ancestry kriging estimation. The depicted lines denote the mean coefficients from the bootstrap distribution. Co-efficients whose bootstrap distribution has a 95% central probability mass interval that spans the value of 0 are not depicted. ANC = ancestry. NEOL = neolithic farmer. HG = hunter-gatherer. YAM = Yamnaya. CLIM = Climate. The climate variables follow the WorldClim nomenclature.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F18.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S10:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F18.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S10:</span> <p id="p-74" class="first-child">Spatiotemporal Gaussian process model for paleo-vegetation anomalies, using kriged ancestry anomalies and anomalies from simulation-based paleo-climate reconstructions as explanatory variables. Here, we used bootstrapping to account for uncertainty in the ancestry kriging estimation. The depicted lines denote the mean coefficients from the bootstrap distribution. Co-efficients whose bootstrap distribution has a 95% central probability mass interval that spans the value of 0 are not depicted. ANC = ancestry. NEOL = neolithic farmer. HG = hunter-gatherer. YAM = Yamnaya. CLIM = Climate. The climate variables follow the WorldClim nomenclature.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/977968/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/977968/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/977968" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5:</span> <p id="p-30" class="first-child">Coefficients of the spatiotemporal autoregressive model, using the PaleoClim (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Brown et al., 2018</a>) simulation-based paleo-climate variables as covariates, with 95% bootstrap-based confidence intervals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Regardless of the model used, we find that HG ancestry contributes positively to broad-leaf forest anomalies, but negatively to arable land anomalies. YAM ancestry, in turn, contributes positively to pasture (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figures 7</a>, <a id="xref-fig-18-2" class="xref-fig" href="#F18">S10</a>). In the fitted AR model and the bootstrapping approach, we also see a negative contribution of HG ancestry to pasture and scrubland. In the fitted GP model and the bootstrapping approach, we observe a negative contribution of YAM ancestry to forest vegetation, which is strongest for broad-leaf forest. We see weak or non-existent contributions of NEOL ancestry to any vegetation type, suggesting that the population movement associated with NEOL ancestry did not have an immediate effect on vegetation composition, or that the effect was not large enough to be detected. We cannot discard the possibility that we may lack the power to detect some of these changes at our current scale of resolution.</p><p id="p-32">We additionally observe non-zero contributions of different climate variables to the different vegetation anomalies, which become sparser in the AR model (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7</a>). For example, in both the AR and GP models, increases in temperature are related to increases in non-forest vegetation types (scrubland, pasture and arable land). In addition, temperature seasonality may be interpreted as contributing negatively to the proportion of arable land, while precipitation during the driest quarter may be interpreted as contributing positively to heath / scrubland, under the fitted model. Finally, we also built “first arrival” maps (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Pinhasi, Fort, and Ammerman, 2005</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fort, 2015</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Vander Linden and Silva, 2018</a>) for both the NEOL ancestry and the YAM ancestry, given that these ancestry changes can be broadly interpreted as incursions of foreign populations into the European continent during the Neolithic and Bronze Age (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Allentoft et al., 2015</a>; <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Haak et al., 2015</a>). Here, we record the approximate period of first appearance of individuals with more than some particular ancestry proportion cutoff (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). Constructing this map for NEOL ancestry allows us to observe that this ancestry spread closely parallels the inferred cultural spread of farming, which has been inferred from archaeological sites (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fort, 2018</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Fort, 2015</a>; <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Pinhasi, Fort, and Ammerman, 2005</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Silva and Steele, 2014</a>). When performing the same type of reconstruction for the YAM ancestry, we observe that this spread occurs first via north and central Europe, and only much later begins to spread into southern Europe (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>), again reflecting reconstructions from archaeological records for the spread of the Yamnaya, Corded Ware and Bell Beaker cultures.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of inferred spread of farming from archaeological sites and spread of NEOL (upper panel) and YAM ancestries (lower panels). The left panels define first arrival as the first time slice in which a grid point has more than 50% of the ancestry depicted. The center panels are the result of using a more strict cutoff: 75% ancestry. The top right panel is a spatially kriged map of first arrivals of farming practices, based on radiocarbon-dated archaeological sites. White regions are either regions without any genomes included in our data or regions that do not contain genomes passing the assigned cutoff throughout the duration of the Holocene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of inferred spread of farming from archaeological sites and spread of NEOL (upper panel) and YAM ancestries (lower panels). The left panels define first arrival as the first time slice in which a grid point has more than 50% of the ancestry depicted. The center panels are the result of using a more strict cutoff: 75% ancestry. The top right panel is a spatially kriged map of first arrivals of farming practices, based on radiocarbon-dated archaeological sites. White regions are either regions without any genomes included in our data or regions that do not contain genomes passing the assigned cutoff throughout the duration of the Holocene.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F8.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F8.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8:</span> <p id="p-33" class="first-child">Comparison of inferred spread of farming from archaeological sites and spread of NEOL (upper panel) and YAM ancestries (lower panels). The left panels define first arrival as the first time slice in which a grid point has more than 50% of the ancestry depicted. The center panels are the result of using a more strict cutoff: 75% ancestry. The top right panel is a spatially kriged map of first arrivals of farming practices, based on radiocarbon-dated archaeological sites. White regions are either regions without any genomes included in our data or regions that do not contain genomes passing the assigned cutoff throughout the duration of the Holocene.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-3"><h2 class="">Discussion</h2><p id="p-34">An explicitly geostatistical approach allows us to easily visualize how movements of ancestry oc-curred during the Holocene in Europe. We can observe, for example, that the NEOL ancestry expansion followed a two-pronged shape, paralleling the expansion of farming practices estimated from radiocarbon-dated archaeological sites. In particular, we observe two wave fronts, one north-ward across central Europe, and one westward along the Mediterranean coast (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8</a>). In the cultural map, these two wave fronts correspond to the Linear Pottery culture (LBK) (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bickle and Whittle, 2013</a>) and the Impressa / Cardial Pottery culture (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barnett, 2000</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Binder et al., 2018</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Manen et al., 2019</a>). Given their close parallels in the ancestry map, this supports the view that these two cultural expansions may have been driven by migrations of people (<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Allentoft et al., 2015</a>; <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Haak et al., 2015</a>).</p><p id="p-35">We estimate that the expansion of YAM ancestry occurred faster than the expansion of NEOL ancestry. The reasons for this could be numerous, including the use of horses for long-distance travel (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Anthony, 2010</a>). YAM ancestry predominates in individuals associated with the Yamnaya and Corded Ware cultures, and is presumed to have moved into Europe from the Eurasian Steppes (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Kristiansen and Larsson, 2005</a>). Another possibility could be the opening of the landscape previous to the arrival of the Yamnaya people, perhaps due to Neolithic agricultural, grazing and mining practices (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Schauer et al., 2019</a>), which may have facilitated later movements of people. We do not observe a strong decrease in forest vegetation in Northern and Central Europe until the Bronze Age, however. On the other hand, there is limited evidence that Corded Ware people were horse herders, and evidence from settlements in central Europe suggests they may have practiced mixed agriculture (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Müller et al., 2009</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Seregély and Müller, 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jacomet, 2008</a>).</p><p id="p-36">This approach also allows us to model ancestry together with other variables for which there are rich spatio-temporal estimates, namely model-based paleo-climate and pollen-based vegetation landcover types, while accounting for the spatio-temporal autocorrelation among these variables (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7</a>). Our models incorporating both climate and ancestry generally fit HG ancestry as positively contributing to forest-type vegetation, while YAM ancestry as negatively contributing to this form of vegetation, and positively contributing to pasture and arable land.</p><p id="p-37">We do not find that NEOL ancestry had much of a strong contribution to changes in vegetation, suggesting that this expansion did not have an immediate effect on the European landscape, or that this impact was too weak or localized for us to clearly detect an effect in our model. Earlier studies have shown that Neolithic communities did in fact alter their local environments (Mercuri et al 2019 Holocene) and had a local impact on vegetation to a certain extent, at least in northwestern Europe (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Woodbridge, R. M. Fyfe, et al., 2014</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Schauer et al., 2019</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lechterbeck et al., 2014</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Woodbridge, R. M. Fyfe, et al., 2014</a>). <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> shows a very minor decline in broad-leaf forest that coincided with the increase in NEOL ancestry, in particular areas such as northern and northwestern Europe. However, a much more pronounced reduction in broad-leaf forest occurs later on throughout western and northwestern Europe, and coincides with the increase in YAM ancestry. In relation to this point, it is important to note that cultivated tree types (olive, chestnut and walnut) - which are pervasive in the Mediterranean - also fall into the category of broad-leaf forest. Thus, our capacity to infer causes for changes in forest types in regions with this type of cultivar (e.g. the Mediterranean, <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Woodbridge, N. Roberts, and R. Fyfe (2018)</a>) is limited.</p><p id="p-38">The decrease in broad-leaf forest (starting 6,000 years BP) was followed by a minor increase in pasture and disturbed land in some parts of the continent. These vegetation types are naturally present in the Mediterranean and the Black Sea region throughout the earlier part of the Holocene and remain fairly stable until the present (<a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">R. M. Fyfe, Woodbridge, and N. Roberts, 2015</a>). In contrast, in western Europe, these vegetation types only reach intermediate levels during the Bronze Age - as YAM ancestry begins to increase - and they continue to increase after the end of this period. Furthermore, increases in YAM ancestry in southern and eastern Europe do not coincide with increases in pasture and disturbed land.</p><p id="p-39">Pasture is the only land cover type whose predictive power considerably increases as a result of adding ancestry and climate variables into our model. This might be because we currently lack the spatiotemporal resolution to provide much predictive power with the addition of climate or ancestry variables, or because these variables may not be strongly predictive of the other land cover classes. Other unaccounted factors may have had a stronger effect on the vegetated landscape. An obvious candidate is the dramatic increase in population density that occurred over the last 3,000 years (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bevan et al., 2017</a>), which likely led to strong changes in land use practices, consequently disturbing vegetation throughout the continent. Earlier population rises and collapses during the Neolithic and Bronze Age could have also influenced the vegetated landscape in significant ways, although on a smaller scale (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Shennan et al., 2013</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Lechterbeck et al., 2014</a>; <a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">Woodbridge, R. M. Fyfe, et al., 2014</a>). Thus, a future study could aim to incorporate estimates of human population density or other measures of human activity into explanatory models for changes in vegetation, together with population movement. A recent approach using human land use estimates, for example, showed that, on a continental scale, climate changes were the main driver of changes in Holocene vegetation, but the influence of human land use increased in the later stages of the period (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Marquer et al., 2017</a>).</p><p id="p-40">There are a number of caveats and assumptions in our modelling procedure that are important to stress. First, we are assuming that changes in ancient ancestries can be used as a proxy for long-distance movement of people. This may be the case for particular periods of time - especially when peoples of highly divergent ancestries first met each other - but this assumption loses validity as we move closer to the present, and the ancestry components tend to become more homogenized due to later migrations within the area of study (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Margaryan et al., 2019</a>). Tracing relatively high YAM ancestry in the present day is approximately equivalent to tracing people with high Northern European ancestry, who cannot be equated with ancient “steppe peoples”.</p><p id="p-41">Second, we are relying on existing ancient DNA data, which has its own idiosyncrasies, due to environmental and historical biases in sampling. For example, North Africa and eastern Scandinavia are sparsely sampled in our dataset, so our ancestry estimates for those regions are much poorer than for the rest of the European continent. We attempted to prevent this type of biases from affecting our model inference via a bootstrapping approach, to assess how robust these were to accidents of sampling.</p><p id="p-42">Additionally, we are relying on a particular choice of the number of hidden clusters or components (K) under a latent mixed-membership model. We chose this model and parameter setting to be able to discretize patterns of ancestry into three major population clusters (HG, NEOL and YAM), which have been documented via other, more involved, population genetic analyses (<a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Haak et al., 2015</a>; <a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Allentoft et al., 2015</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Lazaridis, Patterson, et al., 2014</a>). We also chose a low number of clusters to have enough data points across extended periods of time, in order to accurately estimate the space-time decay in covariance between ancestries (e.g. <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figures S1</a> and <a id="xref-fig-10-2" class="xref-fig" href="#F10">S2</a>). These clusters are, however, an approximation of a very complex genealogical process. Indeed, the clusters cannot be seen as discrete, originally isolated populations, as there may be both isolation-by-distance and hierarchical population structure within each of these groups (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Frantz et al., 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Janes et al., 2017</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Battey, Ralph, and Kern, 2019</a>). The clusters themselves are also the result of more complex admixture and migration events that occurred before the Holocene (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Fu et al., 2016</a>). Additionally, there is likely a large degree of differentiation in population structure over time and space, even when looking at the same admixture components (e.g. the NEOL component of a present-day Sardinian is differentiated relative to the NEOL component of a Bronze Age central European). These subtle patterns are hard to pick up by simple latent mixed-membership models (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Daniel J Lawson, Van Dorp, and Falush, 2018</a>), although there has recently been some progress in this regard (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Joseph and Pe’er, 2018</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bradburd, Coop, and Ralph, 2018</a>). Other types of population genetic frameworks are able to better detect some of these more subtle signals by, for example, modelling patterns of haplotype sharing (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hellenthal et al., 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Daniel John Lawson et al., 2012</a>), the full site-frequency spectrum (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Excoffier et al., 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kamm et al., 2019</a>) or an approximation to the full ancestral recombination graph (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kelleher et al., 2018</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Speidel et al., 2019</a>). Nevertheless, these also have their own limitations and assumptions. For all these reasons, we advice the reader to consider that the ancestry components used in this study are approximations of the true historical admixture process.</p><p id="p-43">An improvement to our current approach could involve developing a hierarchical dynamical model for explicitly modelling spatiotemporal movements on the genetic data directly, without relying on ancestry assignments estimated from a non-spatiotemporally-aware model (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cressie and Wikle, 2015</a>). This could also help with better dealing with boundary constraints that are not accounted for by the kriging methodology. For example, we currently have to correct kriged estimates that are lower than 0 or higher than 1. A generative model of spatio-temporal ancestry would not allow for these types of parameters in the first place, for example, by placing Bayesian priors on ancestry with 0 probability outside of the 0-1 range. This could also be solved by extending compositional interpolation techniques to a spatiotemporal setting (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Walvoort and Gruijter, 2001</a>).</p><p id="p-44">We believe there is also a lot of potential for new geostatistical approaches that could be deigned to combine other types of datasets in an integrative approach for the study of the past, even at more local scales than we looked at here. This could encompass, for example, the combination of strontium and oxygen isotope analyses together with radiocarbon data and contextual archaeological information (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sjögren, Price, and Kristiansen, 2016</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mittnik et al., 2019</a>), the joint analysis of genetic and linguistic changes over time (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kristiansen, Allentoft, et al., 2017</a>) or the study of the interactions between population density and vegetation (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Müller, 2015</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kolář et al., 2018</a>).</p><p id="p-45">In summary, the methodologies used here rely on several assumptions, and could benefit from a myriad of extensions. However, accounting jointly for space and time in the study of genetic and environmental variables is a necessary first step towards better understanding the impact of major migrations in the past. Without explicitly modelling space and time, researchers might miss local phenomena that may have unravelled in different ways across the area of study. This could lead us to ignore important historical processes that become occluded by taking an overly global perspective. We should not ignore the forest for the trees, but sometimes, the trees themselves might be hidden by the forest.</p></div><div class="section" id="sec-4"><h2 class="">Methods</h2><p id="p-46">All R code used to perform the analyses in this manuscript has been deposited in: <a href="https://github.com/FerRacimo/STAdmix">https://github.com/FerRacimo/STAdmix</a></p><div id="sec-5" class="subsection"><h3>Kriged ancestry maps over time and space</h3><p id="p-47">For our ancestry analyses, we used a combined dataset of 842 ancient and 955 present-day genomic sequences. The present-day genomes were typed at the Human Origins SNP array, while the ancient genomes were either typed at this array or whole-genome sequenced, followed by filtering for SNPs that are in this array (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Mathieson, Lazaridis, et al., 2015</a>; <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Lazaridis, Patterson, et al., 2014</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Patterson et al., 2012</a>; <a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">Allentoft et al., 2015</a>). We restricted our analyses to modern human genomes obtained from human remains located within an area encompassing most of the European continent: north of 30<sup>◦</sup>N, south of 75<sup>◦</sup>N, east of 15<sup>◦</sup>W and west of 45<sup>◦</sup>E. We inferred latent ancestry components on these genomes using Ohana (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cheng, Mailund, and R. Nielsen, 2017</a>).</p><p id="p-48">We performed ordinary global spatio-temporal kriging using the R libraries gstat (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">E. J. Pebesma, 2004</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">E. Pebesma and Heuvelink, 2016</a>) and spacetime (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">E. Pebesma et al., 2012</a>), to obtain un-biased linear predictions of ancestry for unsampled locations and times. Suppose we have a set of noisy observations of a variable distributed unevenly across space and time. In our case, this will be the inferred proportion of a particular ancestry in each of our ancient genomes. Let <em><strong>s</strong></em><sub><em><strong>i</strong></em></sub> be a vector representing the <em>i</em>th site (out of <em>n</em>) in our grid, which is composed of two values: its longitude and latitude. Following the notation by <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Cressie and Wikle (2015)</a>, suppose we have <em>T</em><sub><em>i</em></sub> different temporal samples of a measured variable at site <em><strong>s</strong></em><sub><em><strong>i</strong></em></sub>. A temporal sample obtained at the <em>j</em>th time (<em>t</em><sub><em>ij</em></sub>) from this site will be denoted as <em>Z</em>(<em><strong>s</strong></em><sub><em><strong>i</strong></em></sub>, <em>t</em><sub><em>ij</em></sub>). Suppose these data are equal to the true spatiotemporal process plus some measurement error <em>ϵ</em>:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-14.gif"/></noscript></span>

</span></p><p id="p-49">Let <em><strong>Z</strong></em><sup>(<em>i</em>)</sup> be the vector containing all values that were measured at different time points in location <em>s</em><sub><em>i</em></sub>. Also, let <em><strong>Z</strong></em> = (<em>Z</em><sup>(1)′</sup>, …, <em>Z</em><sup>(<em>m</em>)′</sup>)<sup>′</sup> where <em>m</em> is the number of locations sampled. We can obtain a linear predictor, <em>Y</em> <sup>∗</sup>(<em>s</em><sub>0</sub>; <em>t</em><sub>0</sub>) for a particular unsampled data point at time <em>t</em><sub>0</sub> and location <em>s</em><sub>0</sub>:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-15.gif"/></noscript></span>

</span>
where <em><strong>l</strong></em> and <em>c</em> are parameters than can be optimized. In particular, for the case that the true process <em>Y</em> (;) has a constant unknown mean <em>µ</em>, one can show that the linear unbiased predictor that minimizes the mean squared prediction error - also called the ordinary kriging predictor - is equal to:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-16.gif"/></noscript></span>

</span>
where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/inline-graphic-1.gif"/></noscript></span></span>, <em><strong>c</strong></em><sub>0</sub> = <em>var</em>(<em><strong>Z</strong></em>) and <em><strong>C</strong></em><sub><em>Z</em></sub> = <em>cov</em>(<em>Y</em> (<em><strong>s</strong></em><sub>0</sub>, <em>t</em><sub>0</sub>), <em><strong>Z</strong></em>). The latter can be obtained by fitting a spatio-temporal covariance function for the true process <em><strong>Y</strong></em> to the empirical spatio-temporal variogram of the observed measurements <em><strong>Z</strong></em> (<a id="xref-fig-10-3" class="xref-fig" href="#F10">Figures S2</a>, <a id="xref-fig-9-3" class="xref-fig" href="#F9">S1</a>, <a id="xref-fig-11-2" class="xref-fig" href="#F11">S3</a>, <a id="xref-fig-12-3" class="xref-fig" href="#F12">S4</a>). In our case, the variogram was computed over a range of 3,000 years, with 60-year windows, and we used the “metric" variogram model to fit it (Gräler et al. 2015). For a more extensive explanation of spatiotemporal kriging, we refer the reader to <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Cressie and Wikle (2015)</a>.</p><p id="p-50">As our predicted grid, we used a set of spatial points distributed evenly across Europe. We used two types of spatial grids: one containing a dense set (5000) of points and a sparser set, containing 200 points. We call this our "spatial grid". The dense version of the spatial grid was used for plotting spatiotemporal maps (e.g. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>), while the sparse set was used to fit the Bayesian spatio-temporal model (e.g. <a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7</a>), for ease of computation. We observed that the ancestry-vegetation and climate-vegetation correlations computed under both schemes were almost identical, suggesting that the use of the sparser grid should not affect inference under the Bayesian model. Potential biases arising from particular grid points possessing few nearby ancient genomes are accounted for in the bootstrapping method described below. Unless otherwise stated, our "temporal" grid had a 10,800-year span, with intervals of 400 years until the present, for a total of 28 time slices. Thus, if our spatial grid had <em>a</em> spatial points, our "spatiotemporal grid" had 28<em>a</em> spatiotemporal points. We bounded the kriged ancestry values between 0 and 1, and so kriged values that were negative were set to 0, and those that were larger than 1 were set to 1.</p><p id="p-51">In all analyses below, we did not include a kriged ancestry component that is largely restricted to north Africa (NAF). The reasons for this are two-fold: 1) this ancestry remains largely spatially static throughout the Holocene, at least with respect to the box we defined to bound our analyses; 2) given that all latent ancestries must add up to 1 in each individual genome, this ancestry is equal to 1 minus the sum of the other three ancestries, and is therefore not linearly independent from them.</p></div><div id="sec-6" class="subsection"><h3>Paleo-vegetation maps</h3><p id="p-52">We downloaded inferred Holocene paleo-vegetation spatiotemporal maps from <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">R. M. Fyfe, Wood-bridge, and N. Roberts (2015)</a>. This paleo-vegetation reconstruction was built from 982 pollen records across Europe, using the pseudobiomization method (PBM) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">R. Fyfe, N. Roberts, and Woodbridge, 2010</a>). It has a 10,800-year span, with intervals of 200 years until the present. To ease computation, we sampled every two time windows, resulting in intervals of 400 years until the present, and for each paleo-vegetation time slice, we rasterized the map to have 6,540 points (down from 35,856). Then, for each time slice, we inferred the value of each point in our spatial grid by taking the median of the 5 nearest points in the rasterized map.</p></div><div id="sec-7" class="subsection"><h3>Paleo-climate maps</h3><p id="p-53">We obtained a set of simulation-based Holocene paleo-climate reconstructions for Europe from PaleoClim (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Brown et al., 2018</a>), which includes surface temperature and precipitation estimates for the Early (11.7-8.326 kya), Middle (8.326-4.2 kya) and Late Holocene (4.2-0.3 kya), using snapshot-style climate model simulations. These simulations were accessed through PaleoView (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fordham et al., 2017</a>) and come from the TRaCE21ka experiment (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Liu, B. Otto-Bliesner, et al., 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Liu, Lu, et al., 2014</a>), which used the Community Climate System Model v3 (CCSM3) (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Bette L Otto-Bliesner et al., 2006</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Collins et al., 2006</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Yeager et al., 2006</a>): a general circulation model involving atmosphere, ocean, sea ice and land. The PaleoClim authors refined the simulations from this model, incorporating small-scale topographic nuances of regional climatologies, thus creating high-resolution paleo-climate maps. We projected the three Holocene maps - together with the present-day WorldClim map - onto the previously delineated temporal grid, for each of the 19 climate variables that are present in the PaleoClim database. At each time slice, for each point in the our spatial grid, we inferred the value of each climate variable, by taking a weighted average of the values of the two closest bounding paleo-climate time points (past and future) at that spatial point, weighted by their respective temporal distance to our time slice. These allowed us to obtain a spatiotemporal grid of the climate variables at the same locations and times for which we had kriged ancestry and paleo-vegetation data. In the Bayesian hierarchical model, we excluded one of these variables (temperature annual range) because it is a linear combination of two of the other climate variables.</p></div><div id="sec-8" class="subsection"><h3>Computation of correlations</h3><p id="p-54">We computed Pearson correlations between the kriged ancestry, climate and vegetation variables in three ways. First, we simply took the vector containing the values of one variable across all points in our spatiotemporal grid and computed its correlation with the values of another variable at all the same spatiotemporal points. We call these the “raw" correlations. Second, starting from the second oldest time slice, we took each of the values of a particular variable of a time slice and subtracted from them the values of the same variable at the same location but from the immediately previous time slice. We did this for all variables and then computed their pairwise correlations, which we call the correlations in differences. Finally, we took each of the values of a particular variable of a time slice and subtracted from them the values of the same variable at the same location but from the last (present-day) time slice. We then computed pairwise correlation between the resulting values for each of the variables, excluding the last time slice from the analysis (as it would just contain zeroes). We call these the correlations in “anomalies", in the sense that the resulting values represent anomalies of a variable with respect to its present-day value at a given location.</p></div><div id="sec-9" class="subsection"><h3>Spatiotemporal Bayesian modeling of vegetation anomalies</h3><p id="p-55">We used two types of hierarchical spatio-temporal Bayesian model implemented in the R library spTimer (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bakar, Sahu, et al., 2015</a>), in order to jointly model climate and kriged ancestry anomalies as explanatory variables for vegetation type anomalies. To simplify notation, we will now index time with the variable <em>t</em> and assume all sites have observations at the same time slices, i.e. <em>T</em><sub><em>i</em></sub> = <em>T</em> for all sites <em>s</em><sub><em>i</em></sub>. We will suppose we have <em>n</em> sites, and so <em>nT</em> is the total number of spatiotemporal observations. The first of the spatiotemporal models treats the response variable <em><strong>Z</strong></em>(<em>t</em>) - in our case, containing all vegetation type values in our spatial map at time <em>t</em> - as a noisy observation of a Gaussian process <em><strong>O</strong></em>(<em>t</em>):
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-17.gif"/></noscript></span>

</span>
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-18.gif"/></noscript></span>

</span></p><p id="p-56">Here, <em><strong>β</strong></em> is a <em>p</em> × 1 vector of coefficients, <em><strong>X</strong></em><sub><em><strong>t</strong></em></sub> is a <em>n</em> × <em>p</em> matrix of covariates at time <em>t</em>, <em>ϵ</em><sub><em>t</em></sub> is an error vector that only depends on an unknown pure error variance <em>σ</em><sub><em>ϵ</em></sub>:
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-19.gif"/></noscript></span>

</span>
while <em>η</em><sub><em>t</em></sub> is a spatiotemporal nugget vector that is independent of <em>ϵ</em><sub><em>t</em></sub> and whose distribution depends on a site invariant spatial variance <em>σ</em><sub><em>η</em></sub> and the spatial correlation matrix <em>S</em><sub><em>η</em></sub>:
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-20.gif"/></noscript></span>

</span></p><p id="p-57">The correlation matrix <em>S</em><sub><em>η</em></sub> is obtained from the general Matérn correlation function (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Matérn, 1986</a>), whose shape depends on two unknown parameters - <em>λ</em> and <em>ν</em>. These control the rate of decay of the correlation as the distance between sites increases and the smoothness of the random field, respectively (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bakar, Sahu, et al., 2015</a>).</p><p id="p-58">The second model is a temporal auto-regressive model that works by incorporating a term in <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">equation 7</a> that depends on the previous instance of the <em><strong>O</strong></em>() process and a temporal correlation parameter <em>ρ</em>:
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/10/31/826149/embed/graphic-21.gif"/></noscript></span>

</span></p><p id="p-59">spTimer can fit these models via Gibbs sampling and infer the posterior distribution of the unknown parameters <em><strong>β</strong></em>, <em>ϵ</em><sub><em>t</em></sub>, <em>η</em><sub><em>t</em></sub>, <em>ν</em>, <em>φ</em> and <em>ρ</em>. Unless otherwise stated, we used spTimer’s default prior distributions for these parameters, which are detailed in <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bakar, Sahu, et al. (2015)</a>. Before inputting all explanatory and response variables into either model, we first centered and scaled them to have mean 0 and variance 1. We tried three different types of prior distributions for the spatial decay parameter of the Matérn correlation function (<a id="xref-fig-19-3" class="xref-fig" href="#F19">Figures S11</a>) - a fixed value, a Uniform distribution and a Gamma distribution, each with default hyperparameters - and compared their performance using the root mean squared error of the predictions (see below).</p><div id="F19" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F19.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Root mean squared error for each fitted spatio-temporal model for paleo-vegetation anomalies. Black filled circles: using a fixed spatial decay parameter of the Mat&#xE9;rn correlation function (default value in spTimer). Red circles: using a Uniform prior distribution for the spatial decay parameter (with default hyperparameters in spTimer). Blue squares: using a Gamma prior distribution for the spatial decay parameter (with default hyperparameters in spTimer). GP = Gaussian process model. AR = autoregressive model. BOTH = model incorporating both climate and ancestry explanatory variables. CLIMATE ONLY = model incorporating climate explanatory variables. CLIMATE ONLY = model incorporating ancestry explanatory variables. NONE = model incorporating neither ancestry nor climate explanatory variables." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-290625239" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Root mean squared error for each fitted spatio-temporal model for paleo-vegetation anomalies. Black filled circles: using a fixed spatial decay parameter of the Mat&#xE9;rn correlation function (default value in spTimer). Red circles: using a Uniform prior distribution for the spatial decay parameter (with default hyperparameters in spTimer). Blue squares: using a Gamma prior distribution for the spatial decay parameter (with default hyperparameters in spTimer). GP = Gaussian process model. AR = autoregressive model. BOTH = model incorporating both climate and ancestry explanatory variables. CLIMATE ONLY = model incorporating climate explanatory variables. CLIMATE ONLY = model incorporating ancestry explanatory variables. NONE = model incorporating neither ancestry nor climate explanatory variables.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F19.medium.gif" width="440" height="334"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S11:" src="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F19.medium.gif" width="440" height="334"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S11:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/10/31/826149/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S11:</span> <p id="p-75" class="first-child">Root mean squared error for each fitted spatio-temporal model for paleo-vegetation anomalies. Black filled circles: using a fixed spatial decay parameter of the Matérn correlation function (default value in spTimer). Red circles: using a Uniform prior distribution for the spatial decay parameter (with default hyperparameters in spTimer). Blue squares: using a Gamma prior distribution for the spatial decay parameter (with default hyperparameters in spTimer). GP = Gaussian process model. AR = autoregressive model. BOTH = model incorporating both climate and ancestry explanatory variables. CLIMATE ONLY = model incorporating climate explanatory variables. CLIMATE ONLY = model incorporating ancestry explanatory variables. NONE = model incorporating neither ancestry nor climate explanatory variables.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Assessment of prediction error</h3><p id="p-60">We randomly removed 20% of the grid points in the map and fitted a spatio-temporal model to the remainer of the data. We computed the root mean square error by comparing the predicted values across all temporal slices with the previously removed observed values. We then selected the spatiotemporal model (AR vs GP) and the prior distribution for the spatial decay parameter of the Matérn correlation function (fixed value vs. Uniform vs. Gamma) based on visual comparison of the root mean squared error plots for each of these model choices (<a id="xref-fig-19-4" class="xref-fig" href="#F19">Figure S11</a>) (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bakar, Sahu, et al., 2015</a>).</p></div><div id="sec-11" class="subsection"><h3>Nonparametric bootstrapping of parameter estimates</h3><p id="p-61">The Gibbs sampler allows us to obtain posterior estimates of the <em><strong>β</strong></em> parameters relating the explanatory to the response variables. However, it relies on the kriged ancestry grid-point maps as input, so it does not account for the uncertainty in the estimation of these maps from the ancient genomes that we currently have. To address this, we derived confidence intervals on the Bayesian posterior estimates using a nonparametric bootstrapping approach. We created 100 pseudo-samples, by randomly sampling ancient and present-day genomes 100 times - with replacement - from among the list of all ancient and present-day genomes, then obtaining their ancestry assignments and kriging them on the spatiotemporal grid. We then fitted the Bayesian spatiotemporal model to each pseudosample, and thus obtained a distribution of bootstrapped <em><strong>β</strong></em> parameter estimates, from which we obtained 95% confidence intervals.</p></div><div id="sec-12" class="subsection"><h3>Arrival time maps</h3><p id="p-62">We created ancestry arrival time maps by recording the time in each cell of the spatial grid at which the spatiotemporal surface map reaches a kriged ancestry value higher than X, where X could be 50% or 75%. In this case, we used a spatial grid of 5,000 points and 200-year time intervals. To create the cultural arrival maps for the spread of farming, we overlaid a 50×50km map covering Europe and selected, for each square, the oldest radiocarbon date directly associated with early farming. The dataset we used to obtain these dates came from the EUROFARM database, which contains 1,779 records of archaeological farming sites (Vander Linden in prep.). It was then spatially kriged using the spatstat package in R.</p></div><div id="sec-13" class="subsection"><h3>Front speed estimation</h3><p id="p-63">To estimate the front speed of the spread of NEOL and YAM ancestries, we used a method developed by <a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Pinhasi, Fort, and Ammerman (2005)</a> and <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Silva and Steele (2014)</a>, in which the authors regress great-circle distances of sampled locations to a hypothesized migration origin against the time at which the migration reached those locations. The negative inverse of the slope is then an estimate of the migration front speed. We restricted to genomes older than 5,000 years BP for the NEOL ancestry spread and to genomes older than 3,000 years BP for the YAM ancestry spread. We used Cayönü (37.38N, 40.39E) as the NEOL ancestry origin, based on estimates of the Neolithic farmer expansion origin (<a id="xref-ref-61-6" class="xref-bibr" href="#ref-61">Pinhasi, Fort, and Ammerman, 2005</a>; <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Silva and Steele, 2014</a>). We set various points at the center and extremes of the hypothesized original Yamnaya distribution in the Eurasian steppe as the YAM ancestry origin (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We used a ranged major axis (RMA) regression approach implemented in the R package lmodel2 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Borcard, Gillet, and Legendre, 2018</a>), which assumes a symmetrical distribution of measurement error in both distance and time.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">We thank John Novembre, Rasmus Nielsen, Michael K. Borregaard, Mark G. Thomas and Kurt H. Kjær for helpful advice and discussions. FR was funded by a Villum Young Investigator award (project no. 000253000). The palaeo-vegetation research was funded by the Leverhulme Trust and we gratefully acknowledge contributors to the European Pollen Database.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.1" data-doi="10.1038/nature14507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allentoft</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morten</span>  <span class="cit-name-given-names">E</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Population genomics of bronze age Eurasia</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">522</span>.<span class="cit-issue">7555</span>, p. <span class="cit-fpage">167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D522%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14507%26rft_id%253Dinfo%253Apmid%252F26062507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26062507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ammerman</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">J</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Luigi L</span> <span class="cit-name-surname">Cavalli-Sforza</span></span> (<span class="cit-pub-date">1971</span>). “<span class="cit-article-title">Measuring the rate of spread of early farming in Europe</span>”. In: <abbr class="cit-jnl-abbrev">Man</abbr>, pp. <span class="cit-fpage">674</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anthony</span></span>, <span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">W</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-chapter-title">The horse, the wheel, and language: how Bronze-Age riders from the Eurasian steppes shaped the modern world</span>. <span class="cit-publ-name">Princeton University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bakar</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Khandoker</span> <span class="cit-name-surname">Shuvo</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Sujit K</span> <span class="cit-name-surname">Sahu</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">spTimer: Spatio-temporal bayesian modelling using R</span>”. In: <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr> <span class="cit-vol">63</span>.<span class="cit-issue">15</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D63%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barnett</span></span>, <span class="cit-auth"><span class="cit-name-surname">William</span>  <span class="cit-name-given-names">K</span></span> (<span class="cit-pub-date">2000</span>). “<span class="cit-article-title">Cardial pottery and the agricultural transition in Mediterranean Europe</span>”. In: <abbr class="cit-jnl-abbrev">Europe’s first farmers</abbr>, pp. <span class="cit-fpage">93</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Battey</span>,  <span class="cit-name-given-names">CJ</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Peter L</span> <span class="cit-name-surname">Ralph</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Andrew D</span> <span class="cit-name-surname">Kern</span></span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Space is the Place: Effects of Continuous Spatial Structure on Analysis of Population Genetic Data</span>”. In: <abbr class="cit-jnl-abbrev">BioRxiv</abbr>, p. <span class="cit-fpage">659235</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.7" data-doi="10.1073/pnas.1709190114"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bevan</span></span>, <span class="cit-auth"><span class="cit-name-surname">Andrew</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title">Holocene fluctuations in human population demonstrate repeated links to food production and climate</span>”. In: <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">114</span>.<span class="cit-issue">49</span>, <span class="cit-fpage">E10524</span>–<span class="cit-lpage">E10531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1709190114%26rft_id%253Dinfo%253Apmid%252F29158411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMzoiMTE0LzQ5L0UxMDUyNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEwLzMxLzgyNjE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bickle</span></span>, <span class="cit-auth"><span class="cit-name-surname">Penny</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Alasdair</span> <span class="cit-name-surname">Whittle</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-chapter-title">The first farmers of central Europe: diversity in LBK lifeways</span>. <span class="cit-publ-name">Oxbow Books</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Binder</span></span>, <span class="cit-auth"><span class="cit-name-surname">Didier</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Modelling the earliest north-western dispersal of Mediterranean Impressed Wares: new dates and Bayesian chronological model</span>.” In: <abbr class="cit-jnl-abbrev">Documenta praehistorica</abbr>. <span class="cit-vol">44</span>, pp. <span class="cit-fpage">54</span>–<span class="cit-lpage">77</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDocumenta%2Bpraehistorica%26rft.volume%253D44%26rft.spage%253D54%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.10" data-doi="10.1016/j.quascirev.2014.11.004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bishop</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosie</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Mike J</span> <span class="cit-name-surname">Church</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Peter A</span> <span class="cit-name-surname">Rowley-Conwy</span></span> (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Firewood, food and human niche construction: the potential role of Mesolithic hunter–gatherers in actively structuring Scotland’s woodlands</span>”. In: <abbr class="cit-jnl-abbrev">Quaternary Science Reviews</abbr> <span class="cit-vol">108</span>, pp. <span class="cit-fpage">51</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BScience%2BReviews%26rft.stitle%253DQuaternary%2BScience%2BReviews%26rft.aulast%253DBishop%26rft.auinit1%253DR.%2BR.%26rft.volume%253D108%26rft.spage%253D51%26rft.epage%253D75%26rft.atitle%253DFirewood%252C%2Bfood%2Band%2Bhuman%2Bniche%2Bconstruction%253B%2Bthe%2Bpotential%2Brole%2Bof%2BMesolithic%2Bhunter-gatherers%2Bin%2Bactively%2Bstructuring%2BScotland%2527s%2Bwoodlands%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.quascirev.2014.11.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.quascirev.2014.11.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2016022642&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Borcard</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daniel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">François</span> <span class="cit-name-surname">Gillet</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Pierre</span> <span class="cit-name-surname">Legendre</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-chapter-title">Numerical ecology with R</span>. <span class="cit-publ-name">Springer</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.12" data-doi="10.1534/genetics.118.301333"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bradburd</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gideon</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Graham M</span> <span class="cit-name-surname">Coop</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Peter L</span> <span class="cit-name-surname">Ralph</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Inferring continuous and discrete population genetic structure across space</span>”. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>.<span class="cit-issue">1</span>, pp. <span class="cit-fpage">33</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301333%26rft_id%253Dinfo%253Apmid%252F30026187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjEwLzEvMzMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8zMS84MjYxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jason</span>  <span class="cit-name-given-names">L</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">PaleoClim, high spatial resolution paleoclimate surfaces for global land areas</span>”. In: <abbr class="cit-jnl-abbrev">Scientific data</abbr> <span class="cit-vol">5</span>, p. <span class="cit-fpage">180254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2Bdata%26rft.volume%253D5%26rft.spage%253D180254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.14" data-doi="10.1093/bioinformatics/btx098"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jade</span> <span class="cit-name-surname">Yu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Thomas</span> <span class="cit-name-surname">Mailund</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Rasmus</span> <span class="cit-name-surname">Nielsen</span></span> (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title">Fast admixture analysis and population tree estimation for SNP and NGS data</span>”. In: <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>.<span class="cit-issue">14</span>, pp. <span class="cit-fpage">2148</span>–<span class="cit-lpage">2155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2148%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx098%26rft_id%253Dinfo%253Apmid%252F28334108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28334108&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.15" data-doi="10.1175/JCLI3761.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Collins</span></span>, <span class="cit-auth"><span class="cit-name-surname">William</span>  <span class="cit-name-given-names">D</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2006</span>). “<span class="cit-article-title">The community climate system model version 3 (CCSM3)”</span>. In: <abbr class="cit-jnl-abbrev">Journal of Climate</abbr> <span class="cit-vol">19</span>.<span class="cit-issue">11</span>, pp. <span class="cit-fpage">2122</span>–<span class="cit-lpage">2143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClimate%26rft.volume%253D19%26rft.spage%253D2122%26rft_id%253Dinfo%253Adoi%252F10.1175%252FJCLI3761.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/JCLI3761.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000238148200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cressie</span></span>, <span class="cit-auth"><span class="cit-name-surname">Noel</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Christopher K</span> <span class="cit-name-surname">Wikle</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-chapter-title">Statistics for spatio-temporal data</span>. <span class="cit-publ-name">John Wiley &amp; Sons</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.17" data-doi="10.1038/s41586-018-0094-2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">de Barros</span> <span class="cit-name-surname">Damgaard</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peter</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">137 ancient human genomes from across the Eurasian steppes</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">557</span>.<span class="cit-issue">7705</span>, p. <span class="cit-fpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D557%26rft.spage%253D369%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0094-2%26rft_id%253Dinfo%253Apmid%252F29743675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0094-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29743675&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Excoffier</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laurent</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">Robust demographic inference from genomic and SNP data</span>”. In: <abbr class="cit-jnl-abbrev">PLoS genetics</abbr> <span class="cit-vol">9</span>.<span class="cit-issue">10</span>, <span class="cit-fpage">e1003905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bgenetics%26rft.volume%253D9%26rft.spage%253De1003905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fordham</span></span>, <span class="cit-auth"><span class="cit-name-surname">Damien</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title">PaleoView: a tool for generating continuous climate projections spanning the last 21 000 years at regional and global scales</span>”. In: <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">40</span>.<span class="cit-issue">11</span>, pp. <span class="cit-fpage">1348</span>–<span class="cit-lpage">1358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D40%26rft.spage%253D1348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fort</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joaquim</span></span> (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Demic and cultural diffusion propagated the Neolithic transition across different regions of Europe</span>”. In: <abbr class="cit-jnl-abbrev">Journal of the Royal Society interface</abbr> <span class="cit-vol">12</span>.<span class="cit-issue">106</span>, p. <span class="cit-fpage">20150166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BRoyal%2BSociety%2Binterface%26rft.volume%253D12%26rft.spage%253D20150166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fort</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joaquim</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-chapter-title">The Neolithic Transition: Diffusion of People or Diffusion of Culture</span>?” In: <span class="cit-source">Diffusive spreading in nature, technology and society</span>. <span class="cit-publ-name">Springer</span>, pp. <span class="cit-fpage">313</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.22" data-doi="10.1111/j.1365-2664.2008.01606.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frantz</span></span>, AC <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>). “<span class="cit-article-title">Using spatial Bayesian methods to determine the genetic structure of a continuously distributed population: clusters or isolation by distance</span>?” In: <abbr class="cit-jnl-abbrev">Journal of Applied Ecology</abbr> <span class="cit-vol">46</span>.<span class="cit-issue">2</span>, pp. <span class="cit-fpage">493</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BEcology%26rft.volume%253D46%26rft.spage%253D493%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2664.2008.01606.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2664.2008.01606.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.23" data-doi="10.1038/nature12736"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fu</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qiaomei</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">The genetic history of ice age Europe</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">534</span>.<span class="cit-issue">7606</span>, p. <span class="cit-fpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D534%26rft.spage%253D200%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12736%26rft_id%253Dinfo%253Apmid%252F27135931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12736&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27135931&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.24" data-doi="10.1016/j.quascirev.2013.05.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fyfe</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Claire</span> <span class="cit-name-surname">Twiddle</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">The Holocene vegetation cover of Britain and Ireland: overcoming problems of scale and discerning patterns of openness</span>”. In: <abbr class="cit-jnl-abbrev">Quaternary Science Reviews</abbr> <span class="cit-vol">73</span>, pp. <span class="cit-fpage">132</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BScience%2BReviews%26rft.stitle%253DQuaternary%2BScience%2BReviews%26rft.aulast%253DFyfe%26rft.auinit1%253DR.%2BM.%26rft.volume%253D73%26rft.spage%253D132%26rft.epage%253D148%26rft.atitle%253DThe%2BHolocene%2Bvegetation%2Bcover%2Bof%2BBritain%2Band%2BIreland%253B%2Bovercoming%2Bproblems%2Bof%2Bscale%2Band%2Bdiscerning%2Bpatterns%2Bof%2Bopenness%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.quascirev.2013.05.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.quascirev.2013.05.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2014032701&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000322356400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.25" data-doi="10.1111/gcb.12776"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fyfe</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Jessie</span> <span class="cit-name-surname">Woodbridge</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Neil</span> <span class="cit-name-surname">Roberts</span></span> (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">From forest to farmland: polleninferred land cover change across Europe using the pseudobiomization approach</span>”. In: <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">21</span>.<span class="cit-issue">3</span>, pp. <span class="cit-fpage">1197</span>–<span class="cit-lpage">1212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D21%26rft.spage%253D1197%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.12776%26rft_id%253Dinfo%253Apmid%252F25345850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.12776&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25345850&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.26" data-doi="10.1177/0959683610369509"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fyfe</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ralph</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Neil</span> <span class="cit-name-surname">Roberts</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Jessie</span> <span class="cit-name-surname">Woodbridge</span></span> (<span class="cit-pub-date">2010</span>). “<span class="cit-article-title">A pollen-based pseudobiomisation approach to anthropogenic land-cover change</span>”. In: <abbr class="cit-jnl-abbrev">The Holocene</abbr> <span class="cit-vol">20</span>.<span class="cit-issue">7</span>, pp. <span class="cit-fpage">1165</span>–<span class="cit-lpage">1171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BHolocene%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0959683610369509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/0959683610369509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.27" data-doi="10.1038/nature14317"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haak</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolfgang</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Massive migration from the steppe was a source for Indo-European languages in Europe</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">522</span>.<span class="cit-issue">7555</span>, p. <span class="cit-fpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D522%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14317%26rft_id%253Dinfo%253Apmid%252F25731166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25731166&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.28" data-doi="10.1126/science.1243518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hellenthal</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garrett</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">A genetic atlas of human admixture history</span>”. In: <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>.<span class="cit-issue">6172</span>, pp. <span class="cit-fpage">747</span>–<span class="cit-lpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHellenthal%26rft.auinit1%253DG.%26rft.volume%253D343%26rft.issue%253D6172%26rft.spage%253D747%26rft.epage%253D751%26rft.atitle%253DA%2BGenetic%2BAtlas%2Bof%2BHuman%2BAdmixture%2BHistory%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1243518%26rft_id%253Dinfo%253Apmid%252F24531965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDMvNjE3Mi83NDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8zMS84MjYxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hunt</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harriet</span>  <span class="cit-name-given-names">V</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Genetic evidence for a western Chinese origin of broomcorn millet (Panicum miliaceum)</span>”. In: <abbr class="cit-jnl-abbrev">The Holocene</abbr> <span class="cit-vol">28</span>.<span class="cit-issue">12</span>, pp. <span class="cit-fpage">1968</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BHolocene%26rft.volume%253D28%26rft.spage%253D1968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jacomet</span></span>, Stefanie (<span class="cit-pub-date">2008</span>). <span class="cit-chapter-title">Subsistenz und Landnutzung während des 3. Jahrtausends v. Chr. aufgrund von archäobotanischen Daten aus dem südwestlichen Mitteleuropa</span>. <span class="cit-vol">84</span>. <span class="cit-publ-name">Wachholtz</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Janes</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jasmine</span>  <span class="cit-name-given-names">K</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title">The K= 2 conundrum</span>”. In: <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">26</span>.<span class="cit-issue">14</span>, pp. <span class="cit-fpage">3594</span>–<span class="cit-lpage">3602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D26%26rft.spage%253D3594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-confproc" id="cit-826149v1.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Joseph</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tyler</span>  <span class="cit-name-given-names">A</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Itsik</span> <span class="cit-name-surname">Pe’er</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Inference of population structure from ancient DNA</span>”. In: <span class="cit-conf-name">International Conference on Research in Computational Molecular Biology</span>. <span class="cit-publ-name">Springer</span>, pp. <span class="cit-fpage">90</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kamm</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jack</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Efficiently inferring the demographic history of many populations with allele count data</span>”. In: <abbr class="cit-jnl-abbrev">Journal of the American Statistical Association</abbr> just-accepted, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelleher</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jerome</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Inferring the ancestry of everyone</span>”. In: <abbr class="cit-jnl-abbrev">BioRxiv</abbr>, p. <span class="cit-fpage">458067</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kolář</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jan</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Population and forest dynamics during the Central European Eneolithic (4500-2000 BC)</span>”. In: <abbr class="cit-jnl-abbrev">Archaeological and anthropological sciences</abbr> <span class="cit-vol">10</span>.<span class="cit-issue">5</span>, pp. <span class="cit-fpage">1153</span>–<span class="cit-lpage">1164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArchaeological%2Band%2Banthropological%2Bsciences%26rft.volume%253D10%26rft.spage%253D1153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.36" data-doi="10.15184/aqy.2017.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kristiansen</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kristian</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Morten E</span> <span class="cit-name-surname">Allentoft</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title">Re-theorising mobility and the formation of culture and language among the Corded Ware Culture in Europe</span>”. In: <abbr class="cit-jnl-abbrev">Antiquity</abbr> <span class="cit-vol">91</span>.<span class="cit-issue">356</span>, pp. <span class="cit-fpage">334</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntiquity%26rft.volume%253D91%26rft.spage%253D334%26rft_id%253Dinfo%253Adoi%252F10.15184%252Faqy.2017.17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.15184/aqy.2017.17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kristiansen</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kristian</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Thomas B</span> <span class="cit-name-surname">Larsson</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-chapter-title">The rise of Bronze Age society: travels, transmissions and transformations</span>. <span class="cit-publ-name">Cambridge University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lawson</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daniel</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Lucy</span> <span class="cit-name-surname">Van Dorp</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Daniel</span> <span class="cit-name-surname">Falush</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">A tutorial on how not to overinterpret STRUCTURE and ADMIXTURE bar plots</span>”. In: <abbr class="cit-jnl-abbrev">Nature Communications</abbr> <span class="cit-vol">9</span>.<span class="cit-issue">1</span>, p. <span class="cit-fpage">3258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D9%26rft.spage%253D3258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lawson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Daniel</span> <span class="cit-name-surname">John</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Inference of population structure using dense haplotype data</span>”. In: <abbr class="cit-jnl-abbrev">PLoS genetics</abbr> <span class="cit-vol">8</span>.<span class="cit-issue">1</span>, <span class="cit-fpage">e1002453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bgenetics%26rft.volume%253D8%26rft.spage%253De1002453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.40" data-doi="10.1038/nature19310"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lazaridis</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iosif</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Dani</span> <span class="cit-name-surname">Nadel</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">Genomic insights into the origin of farming in the ancient Near East</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">536</span>.<span class="cit-issue">7617</span>, p. <span class="cit-fpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D536%26rft.spage%253D419%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature19310%26rft_id%253Dinfo%253Apmid%252F27459054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature19310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27459054&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.41" data-doi="10.1038/nature13673"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lazaridis</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iosif</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Nick</span> <span class="cit-name-surname">Patterson</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Ancient human genomes suggest three ancestral populations for present-day Europeans</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">513</span>.<span class="cit-issue">7518</span>, p. <span class="cit-fpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D513%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13673%26rft_id%253Dinfo%253Apmid%252F25230663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25230663&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000341814900058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.42" data-doi="10.1177/0959683614540952"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lechterbeck</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jutta</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Is Neolithic land use correlated with demography? An evaluation of pollen-derived land cover and radiocarbon-inferred demographic change from Central Europe</span>”. In: <abbr class="cit-jnl-abbrev">The Holocene</abbr> <span class="cit-vol">24</span>.<span class="cit-issue">10</span>, pp. <span class="cit-fpage">1297</span>–<span class="cit-lpage">1307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BHolocene%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0959683614540952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1177/0959683614540952&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.43" data-doi="10.1038/nature13963"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhengyu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Zhengyao</span> <span class="cit-name-surname">Lu</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Evolution and forcing mechanisms of El Niño over the past 21,000 years</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">515</span>.<span class="cit-issue">7528</span>, p. <span class="cit-fpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiu%2BZhengyu%26rft.volume%253D515%26rft.issue%253D7528%26rft.spage%253D550%26rft.epage%253D553%26rft.atitle%253DEvolution%2Band%2Bforcing%2Bmechanisms%2Bof%2BEl%2BNino%2Bover%2Bthe%2Bpast%2B21%252C000%2Byears%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13963%26rft_id%253Dinfo%253Apmid%252F25428502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2015047331&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=25428502&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.44" data-doi="10.1126/science.1171041"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhengyu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">BL</span> <span class="cit-name-surname">Otto-Bliesner</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>). “<span class="cit-article-title">Transient simulation of last deglaciation with a new mechanism for Bølling-Allerød warming</span>”. In: <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>.<span class="cit-issue">5938</span>, pp. <span class="cit-fpage">310</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D325%26rft.issue%253D5938%26rft.spage%253D310%26rft.epage%253D314%26rft.atitle%253DTransient%2BSimulation%2Bof%2BLast%2BDeglaciation%2Bwith%2Ba%2BNew%2BMechanism%2Bfor%2BBolling-Allerod%2BWarming%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1171041%26rft_id%253Dinfo%253Apmid%252F19608916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTkzOC8zMTAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8zMS84MjYxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manen</span></span>, <span class="cit-auth"><span class="cit-name-surname">Claire</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">The Neolithic Transition in the Western Mediterranean: a Complex and Non-Linear Diffusion Process—The Radiocarbon Record Revisited</span>”. In: <abbr class="cit-jnl-abbrev">Radiocarbon</abbr> <span class="cit-vol">61</span>.<span class="cit-issue">2</span>, pp. <span class="cit-fpage">531</span>–<span class="cit-lpage">571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRadiocarbon%26rft.volume%253D61%26rft.spage%253D531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Margaryan</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ashot</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Population genomics of the Viking world</span>”. In: <abbr class="cit-jnl-abbrev">bioRxiv</abbr>, p. <span class="cit-fpage">703405</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marquer</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laurent</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). “<span class="cit-article-title">Quantifying the effects of land use and climate on Holocene vegetation in Europe</span>”. In: <abbr class="cit-jnl-abbrev">Quaternary Science Reviews</abbr> <span class="cit-vol">171</span>, pp. <span class="cit-fpage">20</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BScience%2BReviews%26rft.volume%253D171%26rft.spage%253D20%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matérn</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bertil</span></span> (<span class="cit-pub-date">1986</span>). <abbr class="cit-jnl-abbrev">Spatial variation, volume 36 of Lecture Notes in Statistics</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.49" data-doi="10.1038/nature25778"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mathieson</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iain</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Songül</span> <span class="cit-name-surname">Alpaslan-Roodenberg</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">The genomic history of southeastern Europe</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">555</span>.<span class="cit-issue">7695</span>, p. <span class="cit-fpage">197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D555%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature25778%26rft_id%253Dinfo%253Apmid%252F29466330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature25778&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29466330&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.50" data-doi="10.1038/nature16152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mathieson</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iain</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Iosif</span> <span class="cit-name-surname">Lazaridis</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). “<span class="cit-article-title">Genome-wide patterns of selection in 230 ancient Eurasians</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">528</span>.<span class="cit-issue">7583</span>, p. <span class="cit-fpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D528%26rft.spage%253D499%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16152%26rft_id%253Dinfo%253Apmid%252F26595274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26595274&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mittnik</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alissa</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Kinship-based social inequality in Bronze Age Europe</span>”. In: <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-fpage">eaax6219</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Müller</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johannes</span></span> (<span class="cit-pub-date">2015</span>). “<span class="cit-chapter-title">Eight Million Neolithic Europeans: Social Demography and Social Archaeology on the Scope of Change–From the Near East to Scandinavia</span>”. In: <span class="cit-source">Paradigm found: archaeological theory present past and future: essays in honour of Evžen Neustupn</span>. <span class="cit-publ-loc">Oxford: Oxbow</span>, pp. <span class="cit-fpage">200</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Müller</span></span>, <span class="cit-auth"><span class="cit-name-surname">Johannes</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>). “<span class="cit-chapter-title">A revision of Corded Ware settlement pattern–new results from the Central European low mountain range</span>”. In: <span class="cit-source">Proceedings of the Prehistoric Society</span>. Vol. <span class="cit-vol">75</span>. <span class="cit-publ-name">Cambridge University Press</span>, pp. <span class="cit-fpage">125</span>–<span class="cit-lpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BPrehistoric%2BSociety%26rft.volume%253D75%26rft.spage%253D125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.54" data-doi="10.1016/j.quascirev.2012.05.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nielsen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Anne</span> <span class="cit-name-surname">Birgitte</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Quantitative reconstructions of changes in regional openness in north-central Europe reveal new insights into old questions</span>”. In: <abbr class="cit-jnl-abbrev">Quaternary Science Reviews</abbr> <span class="cit-vol">47</span>, pp. <span class="cit-fpage">131</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BScience%2BReviews%26rft.stitle%253DQuaternary%2BScience%2BReviews%26rft.aulast%253DNielsen%26rft.auinit1%253DA.-B.%26rft.volume%253D47%26rft.spage%253D131%26rft.epage%253D149%26rft.atitle%253DQuantitative%2Breconstructions%2Bof%2Bchanges%2Bin%2Bregional%2Bopenness%2Bin%2Bnorth-central%2BEurope%2Breveal%2Bnew%2Binsights%2Binto%2Bold%2Bquestions%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.quascirev.2012.05.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.quascirev.2012.05.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2016004134&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000306729300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.55" data-doi="10.1038/nature25738"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Olalde</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iñigo</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">The Beaker phenomenon and the genomic transformation of northwest Europe</span>”. In: <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">555</span>.<span class="cit-issue">7695</span>, p. <span class="cit-fpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D555%26rft.spage%253D190%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature25738%26rft_id%253Dinfo%253Apmid%252F29466337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature25738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29466337&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.56" data-doi="10.1175/JCLI3754.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Otto-Bliesner</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bette</span>  <span class="cit-name-given-names">L</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2006</span>). “<span class="cit-article-title">Climate sensitivity of moderate-and low-resolution versions of CCSM3 to preindustrial forcings</span>”. In: <abbr class="cit-jnl-abbrev">Journal of Climate</abbr> <span class="cit-vol">19</span>.<span class="cit-issue">11</span>, pp. <span class="cit-fpage">2567</span>–<span class="cit-lpage">2583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClimate%26rft.stitle%253DJournal%2Bof%2BClimate%26rft.volume%253D19%26rft.issue%253D11%26rft.spage%253D2567%26rft.epage%253D2583%26rft.atitle%253DClimate%2Bsensitivity%2Bof%2Bmoderate-%2Band%2Blow-resolution%2Bversions%2Bof%2BCCSM3%2Bto%2Bpreindustrial%2Bforcings%26rft_id%253Dinfo%253Adoi%252F10.1175%252FJCLI3754.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/JCLI3754.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2006058799&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000238148200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.57" data-doi="10.1534/genetics.112.145037"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Patterson</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nick</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Ancient admixture in human history</span>”. In: <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>.<span class="cit-issue">3</span>, pp. <span class="cit-fpage">1065</span>–<span class="cit-lpage">1093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145037%26rft_id%253Dinfo%253Apmid%252F22960212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi8zLzEwNjUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8zMS84MjYxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pebesma</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edzer</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">spacetime: Spatio-temporal data in r</span>”. In: <abbr class="cit-jnl-abbrev">Journal of Statistical Software</abbr> <span class="cit-vol">51</span>.<span class="cit-issue">7</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D51%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pebesma</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edzer</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2004</span>). “<span class="cit-article-title">Multivariable geostatistics in S: the gstat package</span>”. In: <abbr class="cit-jnl-abbrev">Computers &amp; Geosciences</abbr> <span class="cit-vol">30</span>.<span class="cit-issue">7</span>, pp. <span class="cit-fpage">683</span>–<span class="cit-lpage">691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComputers%2B%2526%2BGeosciences%26rft.volume%253D30%26rft.spage%253D683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pebesma</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edzer</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Gerard</span> <span class="cit-name-surname">Heuvelink</span></span> (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">Spatio-temporal interpolation using gstat</span>”. In: <abbr class="cit-jnl-abbrev">RFID Journal</abbr> <span class="cit-vol">8</span>.<span class="cit-issue">1</span>, pp. <span class="cit-fpage">204</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRFID%2BJournal%26rft.volume%253D8%26rft.spage%253D204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.61" data-doi="10.1371/journal.pbio.0030410"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pinhasi</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ron</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Joaquim</span> <span class="cit-name-surname">Fort</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Albert J</span> <span class="cit-name-surname">Ammerman</span></span> (<span class="cit-pub-date">2005</span>). “<span class="cit-article-title">Tracing the origin and spread of agriculture in Europe</span>”. In: <abbr class="cit-jnl-abbrev">PLoS biology</abbr> <span class="cit-vol">3</span>.<span class="cit-issue">12</span>, <span class="cit-fpage">e410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DPinhasi%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253De410%26rft.epage%253De410%26rft.atitle%253DTracing%2Bthe%2Borigin%2Band%2Bspread%2Bof%2Bagriculture%2Bin%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030410%26rft_id%253Dinfo%253Apmid%252F16292981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16292981&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roberts</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C</span> <span class="cit-name-surname">Neil</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Mediterranean landscape change during the Holocene: Synthesis, comparison and regional trends in population, land cover and climate</span>”. In: <abbr class="cit-jnl-abbrev">The Holocene</abbr> <span class="cit-vol">29</span>.<span class="cit-issue">5</span>, pp. <span class="cit-fpage">923</span>–<span class="cit-lpage">937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BHolocene%26rft.volume%253D29%26rft.spage%253D923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roberts</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neil</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Europe’s lost forests: a pollen-based synthesis for the last 11,000 years</span>”. In: <abbr class="cit-jnl-abbrev">Scientific reports</abbr> <span class="cit-vol">8</span>.<span class="cit-issue">1</span>, p. <span class="cit-fpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D8%26rft.spage%253D716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.64" data-doi="10.1016/j.cub.2012.06.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sánchez-Quinto</span></span>, <span class="cit-auth"><span class="cit-name-surname">Federico</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Genomic affinities of two 7,000-year-old Iberian huntergatherers</span>”. In: <abbr class="cit-jnl-abbrev">Current biology</abbr> <span class="cit-vol">22</span>.<span class="cit-issue">16</span>, pp. <span class="cit-fpage">1494</span>–<span class="cit-lpage">1499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSanchez-Quinto%26rft.auinit1%253DF.%26rft.volume%253D22%26rft.issue%253D16%26rft.spage%253D1494%26rft.epage%253D1499%26rft.atitle%253DGenomic%2Baffinities%2Bof%2Btwo%2B7%252C000-year-old%2BIberian%2Bhunter-gatherers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2012.06.005%26rft_id%253Dinfo%253Apmid%252F22748318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2012.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22748318&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F10%2F31%2F826149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schauer</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peter</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">Supply and demand in prehistory? Economics of Neolithic mining in northwest Europe</span>”. In: <abbr class="cit-jnl-abbrev">Journal of Anthropological Archaeology</abbr> <span class="cit-vol">54</span>, pp. <span class="cit-fpage">149</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnthropological%2BArchaeology%26rft.volume%253D54%26rft.spage%253D149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.66"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Seregély</span></span>, <span class="cit-auth"><span class="cit-name-surname">Timo</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Johannes</span> <span class="cit-name-surname">Müller</span></span> (<span class="cit-pub-date">2008</span>). “<span class="cit-article-title">Endneolithische Siedlungsstrukturen in Oberfranken: Wattendorf-Motzenstein: eine schnurkeramische Siedlung in der nördlichen Frankenalb</span>”. In:</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shennan</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephen</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). “<span class="cit-article-title">Regional population collapse followed initial agriculture booms in mid-Holocene Europe</span>”. In: <abbr class="cit-jnl-abbrev">Nature communications</abbr> <span class="cit-vol">4</span>, p. <span class="cit-fpage">2486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D4%26rft.spage%253D2486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.68"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shishlina</span></span>, <span class="cit-auth"><span class="cit-name-surname">Natalia</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-chapter-title">Reconstruction of the Bronze Age of the Caspian steppes: life styles and life ways of pastoral nomads</span>. <span class="cit-publ-name">British Archaeological Reports Ltd</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.69"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sikora</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martin</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Population genomic analysis of ancient and modern genomes yields new insights into the genetic ancestry of the Tyrolean Iceman and the genetic structure of Europe</span>”. In: <abbr class="cit-jnl-abbrev">PLoS genetics</abbr> <span class="cit-vol">10</span>.<span class="cit-issue">5</span>, <span class="cit-fpage">e1004353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bgenetics%26rft.volume%253D10%26rft.spage%253De1004353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.70"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Silva</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fabio</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">James</span> <span class="cit-name-surname">Steele</span></span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">New methods for reconstructing geographical effects on dispersal rates and routes from large-scale radiocarbon databases</span>”. In: <abbr class="cit-jnl-abbrev">Journal of Archaeological Science</abbr> <span class="cit-vol">52</span>, pp. <span class="cit-fpage">609</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BArchaeological%2BScience%26rft.volume%253D52%26rft.spage%253D609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.71" data-doi="10.1371/journal.pone.0155083"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sjögren</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karl-Göran</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T</span> <span class="cit-name-surname">Douglas Price</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Kristian</span> <span class="cit-name-surname">Kristiansen</span></span> (<span class="cit-pub-date">2016</span>). “<span class="cit-article-title">Diet and mobility in the Corded Ware of Central Europe</span>”. In: <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">11</span>.<span class="cit-issue">5</span>, <span class="cit-fpage">e0155083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D11%26rft.spage%253De0155083%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0155083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0155083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.72" data-doi="10.1126/science.1253448"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Skoglund</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pontus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Helena</span> <span class="cit-name-surname">Malmström</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ayça</span> <span class="cit-name-surname">Omrak</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">Genomic diversity and admixture differs for Stone-Age Scandinavian foragers and farmers</span>”. In: <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">344</span>.<span class="cit-issue">6185</span>, pp. <span class="cit-fpage">747</span>–<span class="cit-lpage">750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1253448%26rft_id%253Dinfo%253Apmid%252F24762536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDQvNjE4NS83NDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8zMS84MjYxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.73" data-doi="10.1126/science.1216304"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Skoglund</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pontus</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Helena</span> <span class="cit-name-surname">Malmström</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Maanasa</span> <span class="cit-name-surname">Raghavan</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). “<span class="cit-article-title">Origins and genetic legacy of Neolithic farmers and hunter-gatherers in Europe</span>”. In: <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">336</span>.<span class="cit-issue">6080</span>, pp. <span class="cit-fpage">466</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSkoglund%26rft.auinit1%253DP.%26rft.volume%253D336%26rft.issue%253D6080%26rft.spage%253D466%26rft.epage%253D469%26rft.atitle%253DOrigins%2Band%2BGenetic%2BLegacy%2Bof%2BNeolithic%2BFarmers%2Band%2BHunter-Gatherers%2Bin%2BEurope%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1216304%26rft_id%253Dinfo%253Apmid%252F22539720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzYvNjA4MC80NjYiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8xMC8zMS84MjYxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Speidel</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leo</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). “<span class="cit-article-title">A method for genome-wide genealogy estimation for thousands of samples</span>”. In: <abbr class="cit-jnl-abbrev">BioRxiv</abbr>, p. <span class="cit-fpage">550558</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.75"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Vander</span> <span class="cit-name-surname">Linden</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marc</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Fabio</span> <span class="cit-name-surname">Silva</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Comparing and modeling the spread of early farming across Europe</span>.” In: <abbr class="cit-jnl-abbrev">PAGES magazine</abbr> <span class="cit-vol">26</span>.<span class="cit-issue">1</span>, pp. <span class="cit-fpage">28</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPAGES%2Bmagazine%26rft.volume%253D26%26rft.spage%253D28%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-book" id="cit-826149v1.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vandkilde</span></span>, <span class="cit-auth"><span class="cit-name-surname">Helle</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-chapter-title">Culture and change in Central European prehistory</span>. <span class="cit-publ-name">ISD LLC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.77"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Walvoort</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dennis</span>  <span class="cit-name-given-names">JJ</span></span> and <span class="cit-auth"> <span class="cit-name-given-names">Jaap J</span> <span class="cit-name-surname">de Gruijter</span></span> (<span class="cit-pub-date">2001</span>). “<span class="cit-article-title">Compositional kriging: a spatial interpolation method for compositional data</span>”. In: <abbr class="cit-jnl-abbrev">Mathematical Geology</abbr> <span class="cit-vol">33</span>.<span class="cit-issue">8</span>, pp. <span class="cit-fpage">951</span>–<span class="cit-lpage">966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2BGeology%26rft.stitle%253DMathematical%2BGeology%26rft.volume%253D33%26rft.issue%253D8%26rft.spage%253D951%26rft.epage%253D966%26rft.atitle%253DCompositional%2Bkriging%253B%2Ba%2Bspatial%2Binterpolation%2Bmethod%2Bfor%2Bcompositional%2Bdata%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2002006392&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Warren</span></span>, <span class="cit-auth"><span class="cit-name-surname">Graeme</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">The potential role of humans in structuring the wooded landscapes of Mesolithic Ireland: a review of data and discussion of approaches</span>”. In: <abbr class="cit-jnl-abbrev">Vegetation History and Archaeobotany</abbr> <span class="cit-vol">23</span>.<span class="cit-issue">5</span>, pp. <span class="cit-fpage">629</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVegetation%2BHistory%2Band%2BArchaeobotany%26rft.volume%253D23%26rft.spage%253D629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.79" data-doi="10.1016/j.jas.2012.10.025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woodbridge</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jessie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Ralph M</span> <span class="cit-name-surname">Fyfe</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). “<span class="cit-article-title">The impact of the Neolithic agricultural transition in Britain: a comparison of pollen-based land-cover and archaeological 14C date-inferred population change</span>”. In: <abbr class="cit-jnl-abbrev">Journal of Archaeological Science</abbr> <span class="cit-vol">51</span>, pp. <span class="cit-fpage">216</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BArchaeological%2BScience%26rft.volume%253D51%26rft.spage%253D216%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jas.2012.10.025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jas.2012.10.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Woodbridge</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jessie</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">Neil</span> <span class="cit-name-surname">Roberts</span></span>, and <span class="cit-auth"> <span class="cit-name-given-names">Ralph</span> <span class="cit-name-surname">Fyfe</span></span> (<span class="cit-pub-date">2018</span>). “<span class="cit-article-title">Pan-Mediterranean Holocene vegetation and land-cover dynamics from synthesized pollen data</span>”. In: <abbr class="cit-jnl-abbrev">Journal of Biogeography</abbr> <span class="cit-vol">45</span>.<span class="cit-issue">9</span>, pp. <span class="cit-fpage">2159</span>–<span class="cit-lpage">2174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiogeography%26rft.volume%253D45%26rft.spage%253D2159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-826149v1.81"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeager</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">G</span></span> <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2006</span>). “<span class="cit-article-title">The low-resolution CCSM3</span>”. In:. <abbr class="cit-jnl-abbrev">Journal of Climate</abbr> <span class="cit-vol">19</span>.<span class="cit-issue">11</span>, pp. <span class="cit-fpage">2545</span>–<span class="cit-lpage">2566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClimate%26rft.volume%253D19%26rft.spage%253D2545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/826149v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F826149v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/817619v1" title="CRISPRCasIdentifier: Machine learning for accurate identification and classification of CRISPR-Cas systems" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;October 31, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/826149v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/826149v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/826149v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/826149v1" class="active" data-icon-position="" data-hide-link-title="0">A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/974425" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-WMV6r0Mr2vSITz66MU7zBXcR-LoMabPtKHTnoVzmM20" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="974425" id="node974425--2" data-pisa="biorxiv;826149v1" data-pisa-master="biorxiv;826149" data-seqnum="974425" data-apath="/biorxiv/early/2019/10/31/826149.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Fernando</span> <span class="nlm-surname">Racimo</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jessie</span> <span class="nlm-surname">Woodbridge</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Ralph M.</span> <span class="nlm-surname">Fyfe</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Sikora</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Karl-Göran</span> <span class="nlm-surname">Sjögren</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kristian</span> <span class="nlm-surname">Kristiansen</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Marc Vander</span> <span class="nlm-surname">Linden</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">826149; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/826149 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/826149v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/826149v1&amp;title=A%20geostatistical%20approach%20to%20modelling%20human%20Holocene%20migrations%20in%20Europe%20using%20ancient%20DNA" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/826149v1&amp;title=A%20geostatistical%20approach%20to%20modelling%20human%20Holocene%20migrations%20in%20Europe%20using%20ancient%20DNA" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/826149v1&amp;text=A%20geostatistical%20approach%20to%20modelling%20human%20Holocene%20migrations%20in%20Europe%20using%20ancient%20DNA" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/826149v1&amp;title=A%20geostatistical%20approach%20to%20modelling%20human%20Holocene%20migrations%20in%20Europe%20using%20ancient%20DNA" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/826149v1&amp;t=A%20geostatistical%20approach%20to%20modelling%20human%20Holocene%20migrations%20in%20Europe%20using%20ancient%20DNA" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/826149v1&amp;title=A%20geostatistical%20approach%20to%20modelling%20human%20Holocene%20migrations%20in%20Europe%20using%20ancient%20DNA" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/826149v1&amp;title=A%20geostatistical%20approach%20to%20modelling%20human%20Holocene%20migrations%20in%20Europe%20using%20ancient%20DNA" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="974425" id="node974425--3" data-pisa="biorxiv;826149v1" data-pisa-master="biorxiv;826149" data-seqnum="974425" data-apath="/biorxiv/early/2019/10/31/826149.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A geostatistical approach to modelling human Holocene migrations in Europe using ancient DNA</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Fernando</span> <span class="nlm-surname">Racimo</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Jessie</span> <span class="nlm-surname">Woodbridge</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Ralph M.</span> <span class="nlm-surname">Fyfe</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Martin</span> <span class="nlm-surname">Sikora</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Karl-Göran</span> <span class="nlm-surname">Sjögren</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kristian</span> <span class="nlm-surname">Kristiansen</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Marc Vander</span> <span class="nlm-surname">Linden</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">826149; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/826149 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20geostatistical%20approach%20to%20modelling%20human%20Holocene%20migrations%20in%20Europe%20using%20ancient%20DNA&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Racimo%2C+Fernando&amp;rft.au=Woodbridge%2C+Jessie&amp;rft.au=Fyfe%2C+Ralph+M.&amp;rft.au=Sikora%2C+Martin&amp;rft.au=Sj%C3%B6gren%2C+Karl-G%C3%B6ran&amp;rft.au=Kristiansen%2C+Kristian&amp;rft.au=Linden%2C+Marc+Vander"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/974425/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/974425/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/974425/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/974425/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/974425/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/974425/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/974425/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/974425/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/974425/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/974425/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/974425/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/974425/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/826149v1&amp;count=horizontal&amp;via=&amp;text=A%20geostatistical%20approach%20to%20modelling%20human%20Holocene%20migrations%20in%20Europe%20using%20ancient%20DNA&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/826149v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/826149v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/826149v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
