<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2017/03/24/090894.atom"/>
  <meta name="HW.pisa" content="biorxiv;090894v6"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Modeling Adaptive and Non-adaptive Responses of Populations to Environmental Change"/>
  <meta name="DC.Identifier" content="10.1101/090894"/>
  <meta name="DC.Date" content="2017-03-24"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2017, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Understanding how the natural world will be impacted by environmental change over the coming decades is one of the most pressing challenges facing humanity. Addressing this challenge is difficult because environmental change can generate both population level plastic and evolutionary responses, with plastic responses being either adaptive or non-adaptive. We develop an approach that links quantitative genetic theory with data-driven structured models to allow prediction of population responses to environmental change via plasticity and adaptive evolution. After introducing general new theory, we construct a number of example models to demonstrate that evolutionary responses to environmental change over the short-term will be considerably slower than plastic responses, and that the rate of adaptive evolution to a new environment depends upon whether plastic responses are adaptive or non-adaptive. Parameterization of the models we develop requires information on genetic and phenotypic variation and demography that will not always be available, meaning that simpler models will often be required to predict responses to environmental change. We consequently develop a method to examine whether the full machinery of the evolutionarily explicit models we develop will be needed to predict responses to environmental change, or whether simpler non-evolutionary models that are now widely constructed may be sufficient."/>
  <meta name="DC.Contributor" content="Tim Coulson"/>
  <meta name="DC.Contributor" content="Bruce E Kendall"/>
  <meta name="DC.Contributor" content="Julia Barthold"/>
  <meta name="DC.Contributor" content="Floriane Plard"/>
  <meta name="DC.Contributor" content="Susanne Schindler"/>
  <meta name="DC.Contributor" content="Arpat Ozgul"/>
  <meta name="DC.Contributor" content="Jean-Michel Gaillard"/>
  <meta name="article:published_time" content="2017-03-24"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Modeling Adaptive and Non-adaptive Responses of Populations to Environmental Change"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;Abstract&lt;;/h3&gt;; &lt;;p&gt;;Understanding how the natural world will be impacted by environmental change over the coming decades is one of the most pressing challenges facing humanity. Addressing this challenge is difficult because environmental change can generate both population level plastic and evolutionary responses, with plastic responses being either adaptive or non-adaptive. We develop an approach that links quantitative genetic theory with data-driven structured models to allow prediction of population responses to environmental change via plasticity and adaptive evolution. After introducing general new theory, we construct a number of example models to demonstrate that evolutionary responses to environmental change over the short-term will be considerably slower than plastic responses, and that the rate of adaptive evolution to a new environment depends upon whether plastic responses are adaptive or non-adaptive. Parameterization of the models we develop requires information on genetic and phenotypic variation and demography that will not always be available, meaning that simpler models will often be required to predict responses to environmental change. We consequently develop a method to examine whether the full machinery of the evolutionarily explicit models we develop will be needed to predict responses to environmental change, or whether simpler non-evolutionary models that are now widely constructed may be sufficient.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2017/01/01"/>
  <meta name="citation_mjid" content="biorxiv;090894v6"/>
  <meta name="citation_id" content="090894v6"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/090894v6"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/090894v6.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/090894v6.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894.full.pdf"/>
  <meta name="citation_doi" content="10.1101/090894"/>
  <meta name="citation_num_pages" content="72"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="090894"/>
  <meta name="citation_author" content="Tim Coulson"/>
  <meta name="citation_author_institution" content="Department of Zoology, University of Oxford"/>
  <meta name="citation_author" content="Bruce E Kendall"/>
  <meta name="citation_author_institution" content="Bren School of Environmental Science &amp;; Management, 2400 Bren Hall, University of California"/>
  <meta name="citation_author" content="Julia Barthold"/>
  <meta name="citation_author_institution" content="Max-Planck Odense Center on the Biodemography of Aging, Department of Public Health"/>
  <meta name="citation_author" content="Floriane Plard"/>
  <meta name="citation_author_institution" content="Department of Biology, Stanford University"/>
  <meta name="citation_author" content="Susanne Schindler"/>
  <meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich"/>
  <meta name="citation_author" content="Arpat Ozgul"/>
  <meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zurich"/>
  <meta name="citation_author" content="Jean-Michel Gaillard"/>
  <meta name="citation_author_institution" content="UMR 5558 Biometrie et Biologie Evolutive, Batiment G. Mendel, Universite Claude Bernard Lyon"/>
  <meta name="citation_reference" content="Agrawal, A. F., and J. R. Stinchcombe. 2009. How much do genetic covariances alter the rate of adaptation? Proceedings of the Royal Society of London B 276: 1183–1191."/>
  <meta name="citation_reference" content="Barfield, M., R. D. Holt, and R. Gomulkiewicz. 2011. Evolution in stage-structured populations. The American Naturalist 177: 397–409."/>
  <meta name="citation_reference" content="Bonduriansky, R., A. J. Crean, and T. Day. 2012. The implications of nongenetic inheritance for evolution in changing environments. Evolutionary Applications 5: 192–201."/>
  <meta name="citation_reference" content="Bonenfant, C., J.-M. Gaillard, T. Coulson, M. Festa-Bianchet, A. Loison, M. Garel, L. E. Loe, P. Blanchard, N. Pettorelli, N. Owen-Smith, et al. 2009. Empirical evidence of density-dependence in populations of large herbivores. Advances in Ecological Research 41: 313–357."/>
  <meta name="citation_reference" content="Bossdorf, O., C. L. Richards, and M. Pigliucci. 2008. Epigenetics for ecologists. Ecology Letters 11: 106–115."/>
  <meta name="citation_reference" content="Charlesworth, B. 1994. Evolution in Age-structured Populations. Cambridge University Press Cambridge."/>
  <meta name="citation_reference" content="Chevin, L.-M. 2015. Evolution of adult size depends on genetic variance in growth trajectories: a comment on analyses of evolutionary dynamics using integral projection models. Methods in Ecology and Evolution 6: 981–986."/>
  <meta name="citation_reference" content="Chevin, L.-M., R. Lande, G. M. Mace, et al. 2010. Adaptation, plasticity, and extinction in a changing environment: towards a predictive theory. PLoS Biology 8: e1000357."/>
  <meta name="citation_reference" content="Childs, D. Z., M. Rees, K. E. Rose, P.J. Grubb, and S. P. Ellner. 2003a. Evolution of complex flowering strategies: an age–and size–structured integral projection model. Proceedings of the Royal Society of London B 270: 1829–1838."/>
  <meta name="citation_reference" content="Childs, D. Z., M. Rees, K. E. Rose, P.J. Grubb, and S. P. Ellner. 2003b. Evolution of complex flowering strategies: an age–and size–structured integral projection model. Proceedings of the Royal Society of London B: Biological Sciences 270: 1829–1838."/>
  <meta name="citation_reference" content="Childs, D. Z., B. C. Sheldon, and M. Rees. 2016. The evolution of labile traits in sex-and age-structured populations. Journal of Animal Ecology 85: 329–342."/>
  <meta name="citation_reference" content="Coltman, D. W., P. O’Donoghue, J. T. Jorgenson, J. T. Hogg, C. Strobeck, and M. Festa-Bianchet. 2003. Undesirable evolutionary consequences of trophy hunting. Nature 426: 655–658."/>
  <meta name="citation_reference" content="Cooch, E., R. F. Rockwell, and S. Brault. 2001. Retrospective analysis of demographic responses to environmental change: a lesser snow goose example. Ecological Monographs 71: 377–400."/>
  <meta name="citation_reference" content="Coulson, T. 2012. Integral projections models, their construction and use in posing hypotheses in ecology. Oikos 121: 1337–1350."/>
  <meta name="citation_reference" content="Coulson, T., D. R. MacNulty, D. R. Stahler, R. K. Wayne, D. W. Smith, et al. 2011. Modeling effects of environmental change on wolf population dynamics, trait evolution, and life history. Science 334: 1275–1278."/>
  <meta name="citation_reference" content="Coulson, T., and S. Tuljapurkar. 2008. The dynamics of a quantitative trait in an age-structured population living in a variable environment. The American Naturalist 172: 599–612."/>
  <meta name="citation_reference" content="Coulson, T., S. Tuljapurkar, and D. Z. Childs. 2010. Using evolutionary demography to link life history theory, quantitative genetics and population ecology. Journal of Animal Ecology 79: 1226–1240."/>
  <meta name="citation_reference" content="Dawson, T. P., S. T. Jackson, J. I. House, I. C. Prentice, and G. M. Mace. 2011. Beyond predictions: biodiversity conservation in a changing climate. Science 332: 53–58."/>
  <meta name="citation_reference" content="Diaconis, P., D. Ylvisaker, et al. 1979. Conjugate priors for exponential families. The Annals of statistics 7: 269–281."/>
  <meta name="citation_reference" content="Doebeli, M. 1997. Genetic variation and persistence of predator-prey interactions in the nicholson–bailey model. Journal of Theoretical Biology 188: 109–120."/>
  <meta name="citation_reference" content="Easterling, M. R., S. P. Ellner, and P. M. Dixon. 2000. Size-specific sensitivity: applying a new structured population model. Ecology 81: 694–708."/>
  <meta name="citation_reference" content="Ellner, S. P., and M. Rees. 2006. Integral projection models for species with complex demography. The American Naturalist 167: 410–428."/>
  <meta name="citation_reference" content="Falconer, D. S. 1960. Introduction to Quantitative Genetics. Ronald Press co. NY."/>
  <meta name="citation_reference" content="Feldman, M. W., and L. L. Cavalli-Sforza. 1979. Aspects of variance and covariance analysis with cultural inheritance. Theoretical population biology 15: 276–307."/>
  <meta name="citation_reference" content="Felsenstein, J. 1981. Continuous-genotype models and assortative mating. Theoretical Population Biology 19: 341–357."/>
  <meta name="citation_reference" content="Fisher, R. A. 1930. The genetical theory of natural selection. Oxford University Press."/>
  <meta name="citation_reference" content="Garnett, M. 1981. Body size, its heritability and influence on juvenile survival among great tits, parus major. Ibis 123: 31–41."/>
  <meta name="citation_reference" content="Gavrilets, S., and S. M. Scheiner. 1993. The genetics of phenotypic plasticity. v. evolution of reaction norm shape. Journal of Evolutionary Biology 6: 31–48."/>
  <meta name="citation_reference" content="Ghalambor, C. K., K. L. Hoke, E. W. Ruell, E. K. Fischer, D. N. Reznick, and K. A. Hughes.2015. Non-adaptive plasticity potentiates rapid adaptive evolution of gene expression in nature. Nature 525: 372–375."/>
  <meta name="citation_reference" content="Ghalambor, C. K., J. K. McKay, S. P. Carroll, and D. N. Reznick. 2007. Adaptive versus non-adaptive phenotypic plasticity and the potential for contemporary adaptation in new environments. Functional Ecology 21: 394–407."/>
  <meta name="citation_reference" content="Gienapp, P., C. Teplitsky, J. Alho, J. Mills, and J. Merilä. 2008. Climate change and evolution: disentangling environmental and genetic responses. Molecular Ecology 17: 167–178."/>
  <meta name="citation_reference" content="Gilbert, S. F., and D. Epel. 2009. Ecological developmental biology: integrating epigenetics, medicine, and evolution. Sinauer Associates."/>
  <meta name="citation_reference" content="Hoffmann, A. A., and C. M. Sgrò. 2011. Climate change and evolutionary adaptation. Nature 470: 479–485."/>
  <meta name="citation_reference" content="Ives, A. R. 1995. Predicting the response of populations to environmental change. Ecology 73: 926–941."/>
  <meta name="citation_reference" content="Jablonka, E., and M. J. Lamb. 1989. The inheritance of acquired epigenetic variations. Journal of Theoretical Biology 139: 69–83."/>
  <meta name="citation_reference" content="Janeiro, M. J., M. Festa-Bianchet, F. Pelletier, D. W. Coltman, and M. B. Morrissey. in press. Towards robust evolutionary inference with integral projection models. Journal of Evolutionary Biology."/>
  <meta name="citation_reference" content="Kendall, B. E. 2015. A statistical symphony: Instrumental variables reveal causality and control measurement error. Pages 149–167 in G. A. Fox, S. Negrete-Yankelevich, and V. J. Sosa, eds. Ecological Statistics: Contemporary Theory and Application. Oxford University Press, Oxford, UK."/>
  <meta name="citation_reference" content="Kendall, B. E., and M. E. Wittmann. 2010. A stochastic model for annual reproductive success. The American Naturalist 175: 461–468."/>
  <meta name="citation_reference" content="Kingsolver, J. G., H. E. Hoekstra, J. M. Hoekstra, D. Berrigan, S. N. Vignieri, C. Hill, A. Hoang, P. Gibert, and P. Beerli. 2001. The strength of phenotypic selection in natural populations. The American Naturalist 157: 245–261."/>
  <meta name="citation_reference" content="Kruuk, L. E., J. Slate, and A. J. Wilson. 2008. New answers for old questions: the evolutionary quantitative genetics of wild animal populations. Annual Review of Ecology, Evolution, and Systematics 39: 525–548."/>
  <meta name="citation_reference" content="Lande, R. 1979. Quantitative genetic analysis of multivariate evolution, applied to brain: body size allometry. Evolution 33: 402–416."/>
  <meta name="citation_reference" content="Lande, R. 1982. A quantitative genetic theory of life history evolution. Ecology 63: 607–615."/>
  <meta name="citation_reference" content="Lande, R. 2009. Adaptation to an extraordinary environment by evolution of phenotypic plasticity and genetic assimilation. Journal of Evolutionary Biology 22: 1435–1446."/>
  <meta name="citation_reference" content="Lande, R., S. Engen, and B.-E. Saether. 2003. Stochastic population dynamics in ecology and conservation. Oxford University Press."/>
  <meta name="citation_reference" content="Lavergne, S., N. Mouquet, W. Thuiller, and O. Ronce. 2010. Biodiversity and climate change: integrating evolutionary and ecological responses of species and communities. Annual Review of Ecology, Evolution and Systematics 41: 321–350."/>
  <meta name="citation_reference" content="López-Maury, L., S. Marguerat, and J. Bähler. 2008. Tuning gene expression to changing environments: from rapid responses to evolutionary adaptation. Nature Reviews Genetics 9: 583–593."/>
  <meta name="citation_reference" content="Lynch, M., and B. Walsh. 1998. Genetics and Analysis of Quantitative Traits, vol. 1. Sinauer Sunderland, MA."/>
  <meta name="citation_reference" content="Merilä, J., L. Kruuk, and B. Sheldon. 2001. Cryptic evolution in a wild bird population. Nature 412: 76–79."/>
  <meta name="citation_reference" content="Merow, C., J. P. Dahlgren, C. J. E. Metcalf, D. Z. Childs, M. E. Evans, E. Jongejans, S. Record, M. Rees, R. Salguero-Gómez, and S. M. McMahon. 2014. Advancing population ecology with integral projection models: a practical guide. Methods in Ecology and Evolution 5: 99–110."/>
  <meta name="citation_reference" content="Metz, J. A., S. A. Geritz, G. Meszéna, F. J. Jacobs, J. S. Van Heerwaarden, et al. 1996. Adaptive dynamics, a geometrical study of the consequences of nearly faithful reproduction. Stochastic and spatial structures of dynamical systems 45: 183–231."/>
  <meta name="citation_reference" content="Miner, B. G., S. E. Sultan, S. G. Morgan, D. K. Padilla, and R. A. Relyea. 2005. Ecological consequences of phenotypic plasticity. Trends in Ecology and Evolution 20: 685–692."/>
  <meta name="citation_reference" content="Mousseau, T. A., and C. W. Fox. 1998. The adaptive significance of maternal effects. Trends in Ecology &amp;; Evolution 13: 403–407."/>
  <meta name="citation_reference" content="Nussey, D., A. Wilson, and J. Brommer. 2007. The evolutionary ecology of individual phenotypic plasticity in wild populations. Journal of Evolutionary Biology 20: 831–844."/>
  <meta name="citation_reference" content="Nussey, D. H., T. H. Clutton-Brock, D. A. Elston, S. D. Albon, and L. E. Kruuk. 2005. Phenotypic plasticity in a maternal trait in red deer. Journal of Animal Ecology 74: 387–396."/>
  <meta name="citation_reference" content="Ozgul, A., D. Z. Childs, M. K. Oli, K. B. Armitage, D. T. Blumstein, L. E. Olson, S. Tuljapurkar, and T. Coulson. 2010. Coupled dynamics of body mass and population growth in response to environmental change. Nature 466: 482–485."/>
  <meta name="citation_reference" content="Ozgul, A., S. Tuljapurkar, T. G. Benton, J. M. Pemberton, T. H. Clutton-Brock, and T. Coulson. 2009. The dynamics of phenotypic change and the shrinking sheep of st. kilda. Science 325: 464–467."/>
  <meta name="citation_reference" content="Pelletier, F., and T. Coulson. 2012. A new metric to calculate the opportunity for selection on quantitative characters. Evolutionary Ecology Research 14: 729–742."/>
  <meta name="citation_reference" content="Reed, T. E., R. S. Waples, D. E. Schindler, J. J. Hard, and M. T. Kinnison. 2010. Phenotypic plasticity and population viability: the importance of environmental predictability. Proceedings of the Royal Society of London B: Biological Sciences 277: 3391–3400."/>
  <meta name="citation_reference" content="Rees, M., D. Z. Childs, and S. P. Ellner. 2014. Building integral projection models: a user’s guide. Journal of Animal Ecology 83: 528–545."/>
  <meta name="citation_reference" content="Rees, M., and S. P. Ellner. 2009. Integral projection models for populations in temporally varying environments. Ecological Monographs 79: 575–594."/>
  <meta name="citation_reference" content="Rees, M., and S. P. Ellner. 2016. Evolving integral projection models: evolutionary demography meets eco-evolutionary dynamics. Methods in Ecology and Evolution 7: 157–170."/>
  <meta name="citation_reference" content="Reznick, D. N., F. H. Shaw, F. H. Rodd, and R. G. Shaw. 1997. Evaluation of the rate of evolution in natural populations of guppies (Poecilia reticulata). Science 275: 1934–1937."/>
  <meta name="citation_reference" content="Sax, D. F., J. J. Stachowicz, J. H. Brown, J. F. Bruno, M. N. Dawson, S. D. Gaines, R. K. Grosberg, A. Hastings, R. D. Holt, M. M. Mayfield, et al. 2007. Ecological and evolutionary insights from species invasions. Trends in Ecology and Evolution 22: 465–471."/>
  <meta name="citation_reference" content="Scheiner, S. M. 1993. Genetics and evolution of phenotypic plasticity. Annual Review of Ecology and Systematics 24: 35–68."/>
  <meta name="citation_reference" content="Schindler, S., J. Gaillard, A. Grüning, P. Neuhaus, L. Traill, S. Tuljapurkar, and T. Coulson. 2015. Sex-specific demography and generalization of the Trivers-Willard theory. Nature 526: 249–252."/>
  <meta name="citation_reference" content="Schmalhausen, I. I. 1949. Factors of evolution: the theory of stabilizing selection. Blakiston."/>
  <meta name="citation_reference" content="Sih, A., M. C. Ferrari, and D. J. Harris. 2011. Evolution and behavioural responses to human-induced rapid environmental change. Evol App 4: 367–387."/>
  <meta name="citation_reference" content="Smallegange, I. M., and T. Coulson. 2011. The stochastic demography of two coexisting male morphs. Ecology 92: 755–764."/>
  <meta name="citation_reference" content="Söderström, T. D., and P. G. Stoica. 2002. Instrumental variable methods for system identification, vol. 21. Springer."/>
  <meta name="citation_reference" content="Steiner, U. K., S. Tuljapurkar, and T. Coulson. 2014. Generation time, net reproductive rate, and growth in stage-age-structured populations. The American Naturalist 183: 771–783."/>
  <meta name="citation_reference" content="Steiner, U. K., S. Tuljapurkar, T. Coulson, and C. Horvitz. 2012. Trading stages: life expectancies in structured populations. Experimental gerontology 47: 773–781."/>
  <meta name="citation_reference" content="Traill, L. W., S. Schindler, and T. Coulson. 2014a. Demography, not inheritance, drives phenotypic change in hunted bighorn sheep. Proceedings of the National Academy of Sciences 111: 13223–13228."/>
  <meta name="citation_reference" content="Traill, L. W., S. Schindler, and T. Coulson. 2014b. Reply to hedrick et al.: Trophy hunting influences the distribution of trait values through demographic impacts. Proceedings of the National Academy of Sciences 111:E4811–E4811."/>
  <meta name="citation_reference" content="Tuljapurkar, S. 1990. Population dynamics in variable environments. Springer."/>
  <meta name="citation_reference" content="Tuljapurkar, S., J.-M. Gaillard, and T. Coulson. 2009. From stochastic environments to life histories and back. Philosophical Transactions of the Royal Society B 364: 1499–1509."/>
  <meta name="citation_reference" content="Turelli, M., and N. Barton. 1994. Genetic and statistical analyses of strong selection on polygenic traits: what, me normal? Genetics 138: 913–941."/>
  <meta name="citation_reference" content="Vindenes, Y., and Ø. Langangen. 2015. Individual heterogeneity in life histories and eco-evolutionary dynamics. Ecology letters 18: 417–432."/>
  <meta name="citation_reference" content="Wallace, K., A. Leslie, and T. Coulson. 2013. Re-evaluating the effect of harvesting regimes on nile crocodiles using an integral projection model. Journal of Animal Ecology 82: 155–165."/>
  <meta name="citation_reference" content="Walther, G., E. Post, P. Convey, A. Menzel, C. Parmesan, T. Beebee, et al. 2002. Ecological responses to recent climate change. Nature 416: 389–395."/>
  <meta name="citation_reference" content="Wiens, J. A., D. Stralberg, D. Jongsomjit, C. A. Howell, and M. A. Snyder. 2009. Niches, models, and climate change: assessing the assumptions and uncertainties. Proceedings of the National Academy of Sciences 106: 19729–19736."/>
  <meta name="citation_reference" content="Wilson, A. J., L. E. Kruuk, and D. W. Coltman. 2005. Ontogenetic patterns in heritable variation for body size: using random regression models in a wild ungulate population. The American Naturalist 166:E177–E192."/>
  <meta name="citation_reference" content="Wolf, M., G. S. Van Doorn, O. Leimar, and F. J. Weissing. 2007. Life-history trade-offs favour the evolution of animal personalities. Nature 447: 581–584."/>
  <meta name="citation_reference" content="Yoshida, T., L. E. Jones, S. P. Ellner, G. F. Fussmann, and N. G. Hairston. 2003. Rapid evolution drives ecological dynamics in a predator–prey system. Nature 424:303–306."/>
  <meta name="twitter:title" content="Modeling Adaptive and Non-adaptive Responses of Populations to Environmental Change"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Understanding how the natural world will be impacted by environmental change over the coming decades is one of the most pressing challenges facing humanity. Addressing this challenge is difficult because environmental change can generate both population level plastic and evolutionary responses, with plastic responses being either adaptive or non-adaptive. We develop an approach that links quantitative genetic theory with data-driven structured models to allow prediction of population responses to environmental change via plasticity and adaptive evolution. After introducing general new theory, we construct a number of example models to demonstrate that evolutionary responses to environmental change over the short-term will be considerably slower than plastic responses, and that the rate of adaptive evolution to a new environment depends upon whether plastic responses are adaptive or non-adaptive. Parameterization of the models we develop requires information on genetic and phenotypic variation and demography that will not always be available, meaning that simpler models will often be required to predict responses to environmental change. We consequently develop a method to examine whether the full machinery of the evolutionarily explicit models we develop will be needed to predict responses to environmental change, or whether simpler non-evolutionary models that are now widely constructed may be sufficient."/>
  <meta name="og-title" property="og:title" content="Modeling Adaptive and Non-adaptive Responses of Populations to Environmental Change"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/090894v6"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Understanding how the natural world will be impacted by environmental change over the coming decades is one of the most pressing challenges facing humanity. Addressing this challenge is difficult because environmental change can generate both population level plastic and evolutionary responses, with plastic responses being either adaptive or non-adaptive. We develop an approach that links quantitative genetic theory with data-driven structured models to allow prediction of population responses to environmental change via plasticity and adaptive evolution. After introducing general new theory, we construct a number of example models to demonstrate that evolutionary responses to environmental change over the short-term will be considerably slower than plastic responses, and that the rate of adaptive evolution to a new environment depends upon whether plastic responses are adaptive or non-adaptive. Parameterization of the models we develop requires information on genetic and phenotypic variation and demography that will not always be available, meaning that simpler models will often be required to predict responses to environmental change. We consequently develop a method to examine whether the full machinery of the evolutionarily explicit models we develop will be needed to predict responses to environmental change, or whether simpler non-evolutionary models that are now widely constructed may be sufficient."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2017-03-24"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Modeling Adaptive and Non-adaptive Responses of Populations to Environmental Change | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-35317 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-3">Understanding how the natural world will be impacted by environmental change over the coming decades is one of the most pressing challenges facing humanity. Addressing this challenge is difficult because environmental change can generate both population level plastic and evolutionary responses, with plastic responses being either adaptive or non-adaptive. We develop an approach that links quantitative genetic theory with data-driven structured models to allow prediction of population responses to environmental change via plasticity and adaptive evolution. After introducing general new theory, we construct a number of example models to demonstrate that evolutionary responses to environmental change over the short-term will be considerably slower than plastic responses, and that the rate of adaptive evolution to a new environment depends upon whether plastic responses are adaptive or non-adaptive. Parameterization of the models we develop requires information on genetic and phenotypic variation and demography that will not always be available, meaning that simpler models will often be required to predict responses to environmental change. We consequently develop a method to examine whether the full machinery of the evolutionarily explicit models we develop will be needed to predict responses to environmental change, or whether simpler non-evolutionary models that are now widely constructed may be sufficient.</p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">Introduction</h2>
    <p id="p-4">Ecosystems from the deep ocean to the high arctic, from deserts to tropical forests are responding to environmental change. Understanding and predicting these responses is one of the most pressing issues currently facing humanity. For this reason, in the last quarter of a century, there has been considerable interest in developing ways to understand how the natural world will be affected by environmental change (
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bossdorf et al., 2008</a>; 
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dawson et al., 2011</a>; 
     <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gilbert and Epel, 2009</a>; 
     <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hoffmann and Sgrò, 2011</a>; 
     <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ives, 1995</a>; 
     <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lavergne et al., 2010</a>; 
     <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wiens et al., 2009</a>). We introduce a new, general approach combining insights from structured population modeling and evolutionary genetics that allows us to examine how adaptive evolution and plasticity contribute to the way that populations, and consequently the ecosystems in which they are embedded, respond to environmental change.
    </p>
    <p id="p-5">In order to understand how evolution and plasticity contribute to population responses to environment change it is necessary to appreciate how different levels of biological organization – alleles, genotypes, phenotypes, populations – are linked, as well as feedbacks between the different levels. First, evolution is defined as a change in allele frequencies (
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Charlesworth, 1994</a>). Allele frequencies change as a direct consequence of changes in the frequencies of the genotypes the alleles occur in, and genotype frequencies can change with a change in the distribution of the phenotypes they code for (
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fisher, 1930</a>). The dynamics of phenotypic trait distributions are determined by differential birth, death, development and inheritance rates across phenotypic trait values, where inheritance is defined in the broad sense as the map between parental and offspring phenotypes (
     <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Easterling et al., 2000</a>; 
     <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rees et al., 2014</a>). Given these links between different levels of biological organization, there can be a cascading dynamic at the level of the phenotype, the genotype and the allele caused by differences in the demography of individuals with different phenotypic trait values (
     <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lynch and Walsh, 1998</a>). Another consequence of this variation is the ecology of the system: population dynamics are an emergent property of who lives, who breeds and with whom, as are the dynamics of the community and ecosystem the population is embedded within (?).
    </p>
    <p id="p-6">Although the cascading ecological and evolutionary consequences of variation in demographic rates is relatively straightforward to grasp, the devil is in the detail, and in particular how alleles combine to make genotypes, how genotypes influence phenotypes, and how phenotypes influence demographic rates (
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Coulson et al., 2011</a>). The rate and direction of evolution depends upon how these links influence the relative fitness of each allele within the population (
     <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Charlesworth, 1994</a>). The challenge is these links are often complicated, particularly for complex phenotypic traits like body size that are routinely measured by field biologists. The complexity arises not only because large gene networks and multiple cell types can contribute to the phenotype, but also because the environment makes a contribution too via plasticity defined as change in a phenotypic trait distribution that is not caused by genetic change (
     <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gavrilets and Scheiner, 1993</a>; 
     <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lande, 2009</a>; ?).
    </p>
    <p id="p-7">The environment can be partitioned into biotic and abiotic components (?). The biotic component captures the sizes and structures of the population of the focal species and of all other species with which it interacts. The abiotic environment includes weather, mineral and water available. The biotic and abiotic environments can influence one another, although the influence of the biotic environment on the abiotic environment typically plays out over geological time scales (one exception being manmade climate change).</p>
    <p id="p-8">The biotic and abiotic environment can influence both the map between genotype and phenotype (?), and between phenotype and demographic rates (?). Put another way, demographic rates are a function of phenotype-by-environment interactions, and phenotypic traits are a function of genotype-by-environment interactions. For quantitative phenotypic traits, genotype-by-environment interactions can often usefully be understood by treating the phenotype as consisting of a genetic and an environmental component, with the environmental component determined by aspects of the current and past biotic and abiotic environments (
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Falconer, 1960</a>; 
     <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lande, 1982</a>; ?). The environmental component of the phenotype can capture phenotypic change caused by individuals altering their physiology, metabolism, behavior or levels of gene expression. We use the term epigenetic to refer to any process that does not involve genetic change that is captured by the dynamics of the environmental component of the phenotype. The biotic and abiotic environment can also influence the generation of new alleles via, for example, retroviral insertions into the germline of their hosts, or via ultraviolet radiation (??). In 
     <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1(a)</a> we depict how different levels of biological organization are linked and feedback to influence one another.
    </p>
    <div id="F1" class="fig pos-float type-figure odd">
     <div class="highwire-figure">
      <div class="fig-inline-img-wrapper">
       <div class="fig-inline-img">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="(A) linkages and feedbacks in biology. Evolution is defined as change in allele frequencies but is often inferred from the dynamics of genotypes and phenotypes. Research into links between alleles and genotypes, and particularly between genotypes and phenotypes often focuses on mechanism (red arrows). Differential survival and reproduction and patterns of mating determine (i) the dynamics of phenotypes, genotypes and alleles and (ii) population, community and ecosystem dynamics (purple arrows). Ecological dynamics determine the biotic environment that, along with the abiotic environment, can influence the generation of new alleles as well as the maps between genotype and phenotype and between phenotype and demographic rates. (B) IPMs track the dynamics of the phenotype distribution from t (black line) to t + 1 (blue line). (C) In our approach, we treat the phenotype as a bivariate distribution of an additive genetic (breeding value) and environmental component and iterate this distribution forwards. Dashed gray lines are clines, where each point on a cline denotes the same phenotypic trait value. There are two steps to iterate the phenotype forwards within a cohort. First (D) viability selection. In this example, all individuals with a trait value below a threshold have lower survival than those above the threshold, and second (E) development among survivors. Breeding values do not change within individuals as they age, meaning (F) only selection can generate change in the breeding value distribution within a cohort. In contrast, (G) selection and development can alter the distribution of the environmental component of the phenotype. The dynamics of the two components combine to generate the dynamics of the phenotype. (H) Mechanistic inheritance rules generate the distribution of offspring breeding value given parental breeding values in two steps. First, a distribution of mid-point parental breeding values is generated, before segregation variance is added to create a distribution of offspring breeding values. The inheritance rules for the map between the parental and environmental component of the phenotype are less constrained than genetic inheritance and are not shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-164699597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;(A) linkages and feedbacks in biology. Evolution is defined as change in allele frequencies but is often inferred from the dynamics of genotypes and phenotypes. Research into links between alleles and genotypes, and particularly between genotypes and phenotypes often focuses on mechanism (red arrows). Differential survival and reproduction and patterns of mating determine (i) the dynamics of phenotypes, genotypes and alleles and (ii) population, community and ecosystem dynamics (purple arrows). Ecological dynamics determine the biotic environment that, along with the abiotic environment, can influence the generation of new alleles as well as the maps between genotype and phenotype and between phenotype and demographic rates. (B) IPMs track the dynamics of the phenotype distribution from t (black line) to t + 1 (blue line). (C) In our approach, we treat the phenotype as a bivariate distribution of an additive genetic (breeding value) and environmental component and iterate this distribution forwards. Dashed gray lines are clines, where each point on a cline denotes the same phenotypic trait value. There are two steps to iterate the phenotype forwards within a cohort. First (D) viability selection. In this example, all individuals with a trait value below a threshold have lower survival than those above the threshold, and second (E) development among survivors. Breeding values do not change within individuals as they age, meaning (F) only selection can generate change in the breeding value distribution within a cohort. In contrast, (G) selection and development can alter the distribution of the environmental component of the phenotype. The dynamics of the two components combine to generate the dynamics of the phenotype. (H) Mechanistic inheritance rules generate the distribution of offspring breeding value given parental breeding values in two steps. First, a distribution of mid-point parental breeding values is generated, before segregation variance is added to create a distribution of offspring breeding values. The inheritance rules for the map between the parental and environmental component of the phenotype are less constrained than genetic inheritance and are not shown.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
         <span class="hw-responsive-img">
          <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F1.medium.gif" width="440" height="368"/>
          <noscript>
           <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F1.medium.gif" width="440" height="368"/>
          </noscript>
         </span>
        </a>
       </div>
      </div>
      <ul class="highwire-figure-links inline">
       <li class="download-fig first">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
       </li>
       <li class="new-tab last">
        <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
       </li>
      </ul>
     </div>
     <div class="fig-caption">
      <span class="fig-label">Figure 1.</span>
      <p id="p-9" class="first-child">(A) linkages and feedbacks in biology. Evolution is defined as change in allele frequencies but is often inferred from the dynamics of genotypes and phenotypes. Research into links between alleles and genotypes, and particularly between genotypes and phenotypes often focuses on mechanism (red arrows). Differential survival and reproduction and patterns of mating determine (i) the dynamics of phenotypes, genotypes and alleles and (ii) population, community and ecosystem dynamics (purple arrows). Ecological dynamics determine the biotic environment that, along with the abiotic environment, can influence the generation of new alleles as well as the maps between genotype and phenotype and between phenotype and demographic rates. (B) IPMs track the dynamics of the phenotype distribution from t (black line) to t + 1 (blue line). (C) In our approach, we treat the phenotype as a bivariate distribution of an additive genetic (breeding value) and environmental component and iterate this distribution forwards. Dashed gray lines are clines, where each point on a cline denotes the same phenotypic trait value. There are two steps to iterate the phenotype forwards within a cohort. First (D) viability selection. In this example, all individuals with a trait value below a threshold have lower survival than those above the threshold, and second (E) development among survivors. Breeding values do not change within individuals as they age, meaning (F) only selection can generate change in the breeding value distribution within a cohort. In contrast, (G) selection and development can alter the distribution of the environmental component of the phenotype. The dynamics of the two components combine to generate the dynamics of the phenotype. (H) Mechanistic inheritance rules generate the distribution of offspring breeding value given parental breeding values in two steps. First, a distribution of mid-point parental breeding values is generated, before segregation variance is added to create a distribution of offspring breeding values. The inheritance rules for the map between the parental and environmental component of the phenotype are less constrained than genetic inheritance and are not shown.</p>
      <div class="sb-div caption-clear"/>
     </div>
    </div>
    <p id="p-10">How can this view of biology be used to inform how populations respond to environmental change? Environmental change occurs when the biotic or abiotic environment changes. Biotic changes can result from the arrival of a new species or an extinction within the ecosystem, or from evolution. In order to capture such change, and to model the links between alleles and demographic rates described above and in 
     <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1(a)</a>, it is necessary for models to incorporate (i) the genotype-phenotype map at birth, (ii) how the phenotype develops, (iii) how the phenotype influences survival at each developmental stage, (iv) the population’s mating system and (v) patterns of mate choice based on the phenotype, as well as how these mate choice patterns influence (vi) reproductive success, (vii) the distribution of genotypes among offspring and (viii) how all these processes result in change in allele frequency and population size from one generation to the next. Processes (i) to (vi) (and consequently also (viii)) can be influenced by the biotic or abiotic environment. Integral Projection Models (IPMs) provide a very flexible structured modeling framework that allow each of these processes to be simultaneously modeled (
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Coulson, 2012</a>; 
     <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Easterling et al., 2000</a>; 
     <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Merow et al., 2014</a>).
    </p>
    <p id="p-11">IPMs project the dynamics of phenotype distributions as a function of expected survival and reproduction, the way the phenotype develops and the distribution of offspring phenotypes (
     <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Easterling et al., 2000</a>). Numerous quantities of interest to ecologists and evolutionary biologists describing life history, population dynamic and phenotypic traits can be calculated from IPMs (
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Childs et al., 2003a</a>; 
     <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Coulson et al., 2011</a>, 
     <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">2010</a>; 
     <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ellner and Rees, 2006</a>; 
     <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Rees et al., 2014</a>; 
     <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Steiner et al., 2014</a>, 
     <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">2012</a>; 
     <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Vindenes and Langangen, 2015</a>). They consequently offer great potential to study ecological and evolutionary responses to environmental change (
     <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Coulson et al., 2011</a>). However, most IPMs to date have been restricted to phenotypic variation in that they do not include genotype-phenotype maps (
     <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Merow et al., 2014</a>). A small number of evolutionarily explicit IPMs that do include these maps have been developed. For example, 
     <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Coulson et al. (2011)</a> used IPMs to track the distribution of body size and coat color in wolves, where coat color was determined by genotype at a single bi-allelic locus. 
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> and 
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Childs et al. (2016)</a> developed IPMs of quantitative characters determined by a large number of unlinked loci of small effect. However, none of these models incorporates plasticity, nor different genetic influences on the phenotype at different ages, and these omissions limit their utility in predicting how populations will be influenced by environmental change (
     <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chevin, 2015</a>).
    </p>
    <p id="p-12">The aim of this paper is to introduce a general framework to allow prediction of how populations respond to environmental change. We do this by developing IPMs of the bivariate distribution of a phenotype split into its genetic and environmental components. The models incorporate different development and inheritance rules for each component of the phenotype. We develop and illustrate our framework using simple models. Our models reveal new insights into the way that plasticity can influence evolution, while also allowing us to retrieve key findings from evolutionary genetics that are already known.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">Methods and Results</h2>
    <div id="sec-3" class="subsection">
     <h3>Modeling approach</h3>
     <p id="p-13">We refer to functions as 
      <em>f</em> (…) where the dots inside parentheses define the variables the function 
      <em>f</em> operates on. Parameters of a function are referenced by the same letter as the function, with subscripts defining the variable they influence. For example, a parameter 
      <span class="inline-formula" id="inline-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-1.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-1.gif"/>
        </noscript>
       </span>
      </span> represents a parameter of function 
      <em>f</em> that operates on variable 
      <span class="inline-formula" id="inline-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-2.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-2.gif"/>
        </noscript>
       </span>
      </span>. We reserve 
      <em>I</em> for the intercept of functions and 
      <em>a</em> for age. Age is only included in models for species with overlapping generations. Following standard convention for IPMs (
      <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Coulson, 2012</a>; 
      <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Merow et al., 2014</a>; 
      <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Rees et al., 2014</a>; ?), we use primes to indicate the character value of an individual at the end of a time step. For example, this allows us to show how an individual with phenotype 
      <span class="inline-formula" id="inline-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-3.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-3.gif"/>
        </noscript>
       </span>
      </span> can develop over a time step to a potentially different phenotype 
      <span class="inline-formula" id="inline-formula-4">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-4.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-4.gif"/>
        </noscript>
       </span>
      </span>, or how a parent with genotype 
      <span class="inline-formula" id="inline-formula-5">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-5.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-5.gif"/>
        </noscript>
       </span>
      </span> can produce an offspring with genotype 
      <span class="inline-formula" id="inline-formula-6">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-6.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-6.gif"/>
        </noscript>
       </span>
      </span>. There are, of course, other notational conventions that could achieve the same objective, and we recognize that primes are used differently in evolutionary genetics; our notation is chosen to make clear how evolutionary processes can be included in the IPM framework. The definitions of all variables and functions are summarized in 
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/297857/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup last">
          <a href="/highwire/markup/297857/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1:</span>
       <p id="p-14" class="first-child">Notation used in the paper.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-15">Our starting point is a widely used phenotypic modeling approach that many readers will be familiar with (
      <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Coulson, 2012</a>; 
      <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Merow et al., 2014</a>; 
      <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Rees et al., 2014</a>). We then extend this approach by developing dynamic models of the phenotype decomposed into its genetic and environmental components. We start with a two-sex IPM that captures all demographic processes that can contribute to the dynamics of phenotypes – survival, recruitment, development, inheritance, and mating patterns (
      <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Coulson et al., 2011</a>; 
      <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Schindler et al., 2015</a>; 
      <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Traill et al., 2014a</a>; ?) and which iterates forwards the distribution of the phenotype at time 
      <span class="inline-formula" id="inline-formula-7">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-7.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-7.gif"/>
        </noscript>
       </span>
      </span> (
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1(B)</a>).
     </p>
     <p id="p-16">The model consists of two equations – one for females and one for males – with each equation consisting of two additive components (?). The first component deals with survival and development of individuals already within the population, the second component deals with reproduction and the generation of phenotypes among newborns entering the population. We assume a pre-breeding census such that survival occurs before development and recruitment before inheritance, 
      <span class="disp-formula" id="disp-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-4.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-4.gif"/>
        </noscript>
       </span>
      </span>
      <span class="inline-formula" id="inline-formula-8">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-8.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-8.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-9">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-9.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-9.gif"/>
        </noscript>
       </span>
      </span> are distributions of phenotypes 
      <span class="inline-formula" id="inline-formula-10">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-10.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-10.gif"/>
        </noscript>
       </span>
      </span> in respectively females and males at time 
      <em>t</em> + 1; 
      <span class="inline-formula" id="inline-formula-11">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-11.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-11.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-12">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-12.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-12.gif"/>
        </noscript>
       </span>
      </span> are the probability of the phenotype developing from 
      <span class="inline-formula" id="inline-formula-13">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-13.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-13.gif"/>
        </noscript>
       </span>
      </span> in respectively females and males between 
      <em>t</em> and 
      <em>t</em> + 1 as a function of environmental drivers 
      <em>θ</em>;
      <span class="inline-formula" id="inline-formula-14">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-14.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-14.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-15">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-15.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-15.gif"/>
        </noscript>
       </span>
      </span> are survival functions for females and males from 
      <em>t</em> to 
      <em>t</em> + 1 including effects of phenotype and environmental drivers 
      <em>θ</em>; 
      <em>s</em> is the birth sex ratio measured as the proportion of female offspring produced; 
      <span class="inline-formula" id="inline-formula-16">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-16.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-16.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-17">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-17.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-17.gif"/>
        </noscript>
       </span>
      </span> describe the probabilities of parents with phenotypes 
      <span class="inline-formula" id="inline-formula-18">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-18.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-18.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-19">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-19.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-19.gif"/>
        </noscript>
       </span>
      </span> respectively producing male and female offspring with phenotype 
      <span class="inline-formula" id="inline-formula-20">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-20.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-20.gif"/>
        </noscript>
       </span>
      </span> as a function of environmental drivers 
      <em>θ</em> at time 
      <span class="inline-formula" id="inline-formula-21">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-21.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-21.gif"/>
        </noscript>
       </span>
      </span> captures the rate of mating between a male with phenotype 
      <span class="inline-formula" id="inline-formula-22">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-22.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-22.gif"/>
        </noscript>
       </span>
      </span> and a female with phenotype 
      <span class="inline-formula" id="inline-formula-23">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-23.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-23.gif"/>
        </noscript>
       </span>
      </span> is the expected litter size given a mating between a male and a female with phenotypes 
      <span class="inline-formula" id="inline-formula-24">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-24.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-24.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-25">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-25.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-25.gif"/>
        </noscript>
       </span>
      </span> in environment 
      <em>θ</em> at time 
      <em>t</em>; 
      <em>C
       <sub>N
        <sub>f</sub>N
        <sub>m</sub>
       </sub>
      </em> is a normalization constant that is used to specify the mating system. In theory it could be combined with the mating function, but we follow the notation of ?. 
      <em>C
       <sub>N
        <sub>f</sub>N
        <sub>m</sub>
       </sub>
      </em> can be used to capture a range of mating systems. For example, if we follow ? and write, 
      <span class="disp-formula" id="disp-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-5.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-5.gif"/>
        </noscript>
       </span>
      </span> this adds a minimum size at which females can reproduce 
      <span class="inline-formula" id="inline-formula-26">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-26.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-26.gif"/>
        </noscript>
       </span>
      </span>. Depending on the mating behavior of the species, 
      <em>C
       <sub>N
        <sub>f</sub>N
        <sub>m</sub>
       </sub>
      </em> can be modified in various ways. For example, it can easily be altered such that the number of birth events is determined by the number of the rarer sex, as in monogamous species. Mate choice can be influenced by specifying different functions for 
      <span class="inline-formula" id="inline-formula-27">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-27.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-27.gif"/>
        </noscript>
       </span>
      </span>. ? demonstrate how it can be specified for random mating, assortative mating, disassortative mating and size-selective mating.
     </p>
     <p id="p-17">In phenotypic IPMs, the phenotypic development functions are usually Gaussian probability functions (
      <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Easterling et al., 2000</a>), e.g.: 
      <span class="disp-formula" id="disp-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-6.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-6.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-18">The functions 
      <span class="inline-formula" id="inline-formula-28">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-28.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-28.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-29">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-29.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-29.gif"/>
        </noscript>
       </span>
      </span> respectively describe the expected value of 
      <span class="inline-formula" id="inline-formula-30">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-30.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-30.gif"/>
        </noscript>
       </span>
      </span> given 
      <span class="inline-formula" id="inline-formula-31">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-31.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-31.gif"/>
        </noscript>
       </span>
      </span> and 
      <em>θ</em> at time 
      <em>t</em> and the standard deviation around 
      <span class="inline-formula" id="inline-formula-32">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-32.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-32.gif"/>
        </noscript>
       </span>
      </span>. The Gaussian form can also be used for inheritance functions 
      <span class="inline-formula" id="inline-formula-33">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-33.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-33.gif"/>
        </noscript>
       </span>
      </span> with functions 
      <em>μ
       <sup>Η</sup>
      </em>(…) and 
      <em>V
       <sup>H</sup>
      </em>(…).
     </p>
     <p id="p-19">The two-sex IPM described above is not evolutionarily explicit as it does not include mechanistic rules for genetic inheritance. We now take this phenotypic model and extend it to be evolutionarily explicit. We do this by writing the phenotype as a function of genetic 
      <span class="inline-formula" id="inline-formula-34">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-34.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-34.gif"/>
        </noscript>
       </span>
      </span> and environmental 
      <span class="inline-formula" id="inline-formula-35">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-35.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-35.gif"/>
        </noscript>
       </span>
      </span> components 
      <span class="inline-formula" id="inline-formula-36">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-36.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-36.gif"/>
        </noscript>
       </span>
      </span>. We assume that 
      <span class="inline-formula" id="inline-formula-37">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-37.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-37.gif"/>
        </noscript>
       </span>
      </span> is a quantitative phenotype (i.e. measured in integer or real values). The genotypic value 
      <span class="inline-formula" id="inline-formula-38">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-38.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-38.gif"/>
        </noscript>
       </span>
      </span> and environmental value 
      <span class="inline-formula" id="inline-formula-39">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-39.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-39.gif"/>
        </noscript>
       </span>
      </span> describe the numerical contributions of the genetic and environmental components of the phenotype to an individual’s phenotypic trait value. A simple map can consequently be written 
      <span class="inline-formula" id="inline-formula-40">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-40.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-40.gif"/>
        </noscript>
       </span>
      </span> (
      <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Falconer, 1960</a>).
     </p>
     <p id="p-20">
      <span class="inline-formula" id="inline-formula-41">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-41.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-41.gif"/>
        </noscript>
       </span>
      </span> is determined by genotype, 
      <em>g</em>. When the map between 
      <em>g</em> and 
      <span class="inline-formula" id="inline-formula-42">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-42.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-42.gif"/>
        </noscript>
       </span>
      </span> is additive, the dynamics of 
      <em>g</em> and 
      <span class="inline-formula" id="inline-formula-43">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-43.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-43.gif"/>
        </noscript>
       </span>
      </span> are identical (
      <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Falconer, 1960</a>). This means that the dynamics of alleles are identical to the dynamics of genotypes in which they occur. In contrast, when alleles interact, either at a locus (dominance) or across loci (epistasis) the map between 
      <em>g</em> and 
      <span class="inline-formula" id="inline-formula-44">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-44.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-44.gif"/>
        </noscript>
       </span>
      </span> is not additive, and the dynamics of 
      <span class="inline-formula" id="inline-formula-45">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-45.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-45.gif"/>
        </noscript>
       </span>
      </span> are not identical to the dynamics of 
      <em>g</em> (
      <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Fisher, 1930</a>). In classical quantitative genetics it is assumed that the map between 
      <em>g</em> and 
      <span class="inline-formula" id="inline-formula-46">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-46.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-46.gif"/>
        </noscript>
       </span>
      </span> is additive (
      <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Falconer, 1960</a>). Under these assumptions, it is not necessary to track the dynamics of 
      <em>g</em> but evolution can be investigated by modeling the dynamics of just 
      <span class="inline-formula" id="inline-formula-47">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-47.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-47.gif"/>
        </noscript>
       </span>
      </span>. When the map is additive we refer to the genetic component of the phenotype as a breeding value and denote it 
      <span class="inline-formula" id="inline-formula-48">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-48.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-48.gif"/>
        </noscript>
       </span>
      </span>.
     </p>
     <p id="p-21">In classical population genetics, when the contribution of dominance and epistasis to evolution are often a key focus, it is necessary to track the dynamics of 
      <em>g</em> and calculate 
      <span class="inline-formula" id="inline-formula-49">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-49.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-49.gif"/>
        </noscript>
       </span>
      </span> from each 
      <em>g</em>. The map between 
      <span class="inline-formula" id="inline-formula-50">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-50.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-50.gif"/>
        </noscript>
       </span>
      </span> and the phenotype 
      <span class="inline-formula" id="inline-formula-51">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-51.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-51.gif"/>
        </noscript>
       </span>
      </span> is often assumed to be one-to-one (?). In contrast, in quantitative genetics, the environment can influence the map between 
      <span class="inline-formula" id="inline-formula-52">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-52.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-52.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-53">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-53.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-53.gif"/>
        </noscript>
       </span>
      </span> by influencing the value of the environmental component of the phenotype, 
      <span class="inline-formula" id="inline-formula-54">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-54.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-54.gif"/>
        </noscript>
       </span>
      </span> (
      <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Falconer, 1960</a>). 
      <span class="inline-formula" id="inline-formula-55">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-55.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-55.gif"/>
        </noscript>
       </span>
      </span> can take different values in different individuals and can vary within individuals throughout life. The dynamics of the phenotype may not consequently represent the dynamics of the genotypic value 
      <span class="inline-formula" id="inline-formula-56">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-56.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-56.gif"/>
        </noscript>
       </span>
      </span>. Statistical quantitative genetics is concerned with estimating moments of 
      <span class="inline-formula" id="inline-formula-57">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-57.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-57.gif"/>
        </noscript>
       </span>
      </span> from 
      <span class="inline-formula" id="inline-formula-58">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-58.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-58.gif"/>
        </noscript>
       </span>
      </span> by correcting for environmental and individual variables that determine 
      <span class="inline-formula" id="inline-formula-59">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-59.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-59.gif"/>
        </noscript>
       </span>
      </span> (
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kruuk et al., 2008</a>).
     </p>
     <p id="p-22">The genotype-phenotype map for phenotypic traits measured by biologists in free-living populations is rarely known, and quantitative genetic assumptions are widely adopted (
      <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Kruuk et al., 2008</a>). In particular, the infinitesimal model is assumed in which 
      <span class="inline-formula" id="inline-formula-60">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-60.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-60.gif"/>
        </noscript>
       </span>
      </span> is determined by a large number of unlinked loci of small, additive, effect (
      <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Fisher, 1930</a>). Until we have a better understanding of the genetic architecture of complex traits, this approach is the most powerful available to investigate evolution in the wild (
      <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Kruuk et al., 2008</a>). We consequently adopt it here.
     </p>
     <p id="p-23">We track the joint distribution of the two components 
      <span class="inline-formula" id="inline-formula-61">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-61.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-61.gif"/>
        </noscript>
       </span>
      </span>. The utility of this is we can write expressions to describe the dynamics of each of the components separately, if necessary, before easily combining them to retrieve the dynamics of the phenotype. For 
      <span class="inline-formula" id="inline-formula-62">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-62.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-62.gif"/>
        </noscript>
       </span>
      </span> we can use a convolution (represented by the mathematical operator *) between the two components of the phenotype to construct the phenotype (
      <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>).
     </p>
     <p id="p-24">Phenotypic plasticity and epigenetic inheritance are captured in the dynamics of 
      <span class="inline-formula" id="inline-formula-63">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-63.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-63.gif"/>
        </noscript>
       </span>
      </span>. In previous quantitative genetic IPMs 
      <span class="inline-formula" id="inline-formula-64">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-64.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-64.gif"/>
        </noscript>
       </span>
      </span> is a randomly distributed variable that captures developmental noise (
      <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>; 
      <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Childs et al., 2016</a>). A key contribution of this paper is to show how 
      <span class="inline-formula" id="inline-formula-65">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-65.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-65.gif"/>
        </noscript>
       </span>
      </span> can be extended to also capture the biotic or abiotic environment as well as signatures of parental 
      <span class="inline-formula" id="inline-formula-66">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-66.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-66.gif"/>
        </noscript>
       </span>
      </span> s and 
      <span class="inline-formula" id="inline-formula-67">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-67.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-67.gif"/>
        </noscript>
       </span>
      </span> s. 
      <span class="inline-formula" id="inline-formula-68">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-68.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-68.gif"/>
        </noscript>
       </span>
      </span> is defined as function of these drivers, and we write 
      <span class="inline-formula" id="inline-formula-69">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-69.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-69.gif"/>
        </noscript>
       </span>
      </span> to capture the effects of 
      <span class="inline-formula" id="inline-formula-70">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-70.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-70.gif"/>
        </noscript>
       </span>
      </span>, 
      <span class="inline-formula" id="inline-formula-71">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-71.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-71.gif"/>
        </noscript>
       </span>
      </span> and the environment 
      <em>θ</em> at time 
      <em>t</em> on 
      <span class="inline-formula" id="inline-formula-72">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-72.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-72.gif"/>
        </noscript>
       </span>
      </span>.
     </p>
     <p id="p-25">We now expand terms in our two-sex phenotypic IPM to include the genotype-phenotype map 
      <span class="inline-formula" id="inline-formula-73">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-73.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-73.gif"/>
        </noscript>
       </span>
      </span>. We start with the bivariate distribution of 
      <span class="inline-formula" id="inline-formula-74">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-74.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-74.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-75">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-75.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-75.gif"/>
        </noscript>
       </span>
      </span> at time 
      <em>t</em> among females that are already within the population at time 
      <em>t</em>: 
      <span class="inline-formula" id="inline-formula-76">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-76.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-76.gif"/>
        </noscript>
       </span>
      </span>. Viability selection now operates on this distribution. Viability selection is a simple multiplicative process describing the expected survival from 
      <em>t</em> to 
      <em>t</em> + 1 as a function of the phenotype. We can consequently write, 
      <span class="disp-formula" id="disp-formula-4">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-7.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-7.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-26">When it comes to development, 
      <span class="inline-formula" id="inline-formula-77">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-77.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-77.gif"/>
        </noscript>
       </span>
      </span> remains fixed throughout life while 
      <span class="inline-formula" id="inline-formula-78">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-78.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-78.gif"/>
        </noscript>
       </span>
      </span> may vary, 
      <span class="disp-formula" id="disp-formula-5">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-8.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-8.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-27">Recruitment is dealt with in a similar way to survival in that it is a multiplicative process, 
      <span class="disp-formula" id="disp-formula-6">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-9.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-9.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-28">Note this is a recruitment related term of both male and female offspring that is not yet scaled by the normalization factor 
      <em>C
       <sub>N
        <sub>f</sub>N
        <sub>m</sub>
       </sub>
      </em>.
     </p>
     <p id="p-29">As with development, inheritance of the genetic and environmental components of the phenotype operates in different ways. For example, once mating pairs have formed and the number of offspring from each mating has been determined, the distribution of offspring genotypes is predictable. We can write the inheritance function for the genetic and environmental components of the phenotype as, 
      <span class="disp-formula" id="disp-formula-7">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-10.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-10.gif"/>
        </noscript>
       </span>
      </span> then, 
      <span class="disp-formula" id="disp-formula-8">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-11.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-11.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-30">The same logic applies to the production of male offspring. We can construct the phenotype from the two components 
      <span class="inline-formula" id="inline-formula-79">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-79.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-79.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-80">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-80.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-80.gif"/>
        </noscript>
       </span>
      </span>, e.g. 
      <span class="disp-formula" id="disp-formula-9">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-12.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-12.gif"/>
        </noscript>
       </span>
      </span> where 
      <span class="inline-formula" id="inline-formula-81">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-81.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-81.gif"/>
        </noscript>
       </span>
      </span> is the set of 
      <span class="inline-formula" id="inline-formula-82">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-82.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-82.gif"/>
        </noscript>
       </span>
      </span> values satisfying 
      <span class="inline-formula" id="inline-formula-83">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-83.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-83.gif"/>
        </noscript>
       </span>
      </span>. For the second integral in equation (8) we have 
      <span class="inline-formula" id="inline-formula-84">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-84.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-84.gif"/>
        </noscript>
       </span>
      </span> as the 
      <span class="inline-formula" id="inline-formula-85">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-85.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-85.gif"/>
        </noscript>
       </span>
      </span> does not change within individuals and consequently has no prime.
     </p>
     <p id="p-31">The additivity assumption means that models of clonal inheritance can generate very similar predictions to models of two sexes, particularly if both males and females have similar demography. However, clonal models are simpler than two sex models (
      <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Lande, 1982</a>). We utilize this consequence of the additivity assumption and initially work with clonal reproduction to examine how the dynamics of 
      <span class="inline-formula" id="inline-formula-86">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-86.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-86.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-87">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-87.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-87.gif"/>
        </noscript>
       </span>
      </span> influence population and phenotypic trait dynamics and adaptive evolution. We can write a clonal model, 
      <span class="disp-formula" id="disp-formula-10">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-13.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-13.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="disp-formula" id="disp-formula-11">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-14.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-14.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-32">The above equations describe how the dynamics of a bivariate distribution of the genetic and environmental components of the phenotype. 
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figures 1(B-G)</a> provide graphical examples of how these functions alter the bivariate distribution, and in particular how development and inheritance rules differ between the environmental and additive genetic components. To demonstrate these differences we now focus on developing univariate models of (i) 
      <span class="inline-formula" id="inline-formula-88">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-88.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-88.gif"/>
        </noscript>
       </span>
      </span>, and (ii) 
      <span class="inline-formula" id="inline-formula-89">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-89.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-89.gif"/>
        </noscript>
       </span>
      </span>. These models capture limits where all phenotypic variation among individuals is determined by (i) genetic variation and (ii) variation in the environmental component of the phenotype. We then combine insights from these univariate model and construct models of the bivariate distribution of 
      <span class="inline-formula" id="inline-formula-90">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-90.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-90.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-91">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-91.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-91.gif"/>
        </noscript>
       </span>
      </span>.
     </p>
     <p id="p-33">We primarily work with linear functions for three reasons. First, they are easier to interpret and analyze than non-linear or non-additive forms. Second, when the environment changes impacting populations, responses, at least in the short term, can be well described with linear or linearized additive models (
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cooch et al., 2001</a>). Third, selection, the underpinning of evolution, is often directional and well described with linear or linearized associations between phenotypic traits and components of fitness (
      <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kingsolver et al., 2001</a>). Parameters used for all models are provided in Appendix A (§1.1), as are expressions to calculate key statistics used to show ecological and evolutionary change from model outputs (§1.2). Code to produce each figure is available on GitHub – 
      <a href="https://github.com/tncoulson/QG-meets-IPM-figure-code/tree/master">https://github.com/tncoulson/QG-meets-IPM-figure-code/tree/master</a> and Dryad (?).
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Adaptive Evolution</h3>
     <p id="p-34">In this section we start with a simple clonal model of a univariate distribution of 
      <span class="inline-formula" id="inline-formula-92">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-92.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-92.gif"/>
        </noscript>
       </span>
      </span>. We go on to show how genetic constraints can be imposed to slow, or stop, evolution. We then extend this clonal model in two ways: first, to include a multivariate, age-structured, distribution of 
      <span class="inline-formula" id="inline-formula-93">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-93.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-93.gif"/>
        </noscript>
       </span>
      </span>, and second we relax the clonality assumption and compare the dynamics of clonal and sexual models. Finally, we introduce a new approximation to describe sexual reproduction and compare its performance with our initial approach.
     </p>
     <p id="p-35">Genotypes (and hence 
      <span class="inline-formula" id="inline-formula-94">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-94.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-94.gif"/>
        </noscript>
       </span>
      </span>) are determined at birth and remain fixed throughout life; neither are influenced by the environment. A consequence of this is the development function simplifies to a one-to-one map and can be removed from equation (5). We also start by considering clonal reproduction, which means that the inheritance function can also be removed as offspring genotype is identical to parental genotype. The dynamics of 
      <span class="inline-formula" id="inline-formula-95">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-95.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-95.gif"/>
        </noscript>
       </span>
      </span> are consequently determined by the survival and reproduction functions – selection. In these models, as long as there is genetic variation within a population, and fitness is a monotonic function of genotype, evolution, defined as 
      <span class="inline-formula" id="inline-formula-96">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-96.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-96.gif"/>
        </noscript>
       </span>
      </span> (where 
      <span class="inline-formula" id="inline-formula-97">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-97.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-97.gif"/>
        </noscript>
       </span>
      </span> represents expectations) will occur.
     </p>
     <p id="p-36">In our first models we assume non-overlapping generations, 
      <span class="disp-formula" id="disp-formula-12">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-15.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-15.gif"/>
        </noscript>
       </span>
      </span> and a linear reproduction function 
      <span class="inline-formula" id="inline-formula-98">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-98.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-98.gif"/>
        </noscript>
       </span>
      </span> with expected fitness increasing with the value of 
      <span class="inline-formula" id="inline-formula-99">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-99.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-99.gif"/>
        </noscript>
       </span>
      </span>. Over the course of a simulation of 30 generations (Appendix A§1.1 Model A), the population never achieves an equilibrium structure or growth rate; it grows hyper-exponentially (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2(a)</a>, black line) and the shape of the breeding value distribution continually changes location (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3(b)</a>, black line) and shape (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>(b,d, black lines)). Linear selection only slowly erodes the genetic variance and skew (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2(c,d)</a>) and these changes lead to a slight slowing of the rate of change in the mean breeding value (
      <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2(b)</a>) and the population growth rate (
      <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2(a)</a>) each generation (the black lines are not linear).
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The role of selection on the dynamics of . Dynamics of univariate subject to linear selection and clonal inheritance (a)-(d) (Appendix A[[#xA7]]1.1 Model A). The population does not reach an equilibrium, with (a) population growth, and the (b) mean, (c) variance and (d) skew of the character continually evolving. Imposing a maximum possible character value constrains change (red lines versus black lines (a)-(d)). In the age-structured case we track the dynamics of a bivariate character distribution (e)-(g) (Appendix A[[#xA7]]1.1 models B, C and D). The models in (e) and (f) (Appendix AModels B and C) are identical except the starting distribution at time t = 1 has a covariance of [[#x2212]]0.2 in (f) compared to 0.7 in (e). The parameterisation in (g) is chosen to demonstrate a case where the two traits evolve in different directions. The coloured image plots in figures (e)-(g) represent Gaussian development functions fitted to the bivariate distributions of at the beginning and end of the simulation. Evolution of the bivariate character has resulted in different parameterisations of these phenomenological functions. The lighter the shading, the greater the probability of a transition from character value at age 1 and to age 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-164699597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;The role of selection on the dynamics of . Dynamics of univariate subject to linear selection and clonal inheritance (a)-(d) (Appendix A[[#xA7]]1.1 Model A). The population does not reach an equilibrium, with (a) population growth, and the (b) mean, (c) variance and (d) skew of the character continually evolving. Imposing a maximum possible character value constrains change (red lines versus black lines (a)-(d)). In the age-structured case we track the dynamics of a bivariate character distribution (e)-(g) (Appendix A[[#xA7]]1.1 models B, C and D). The models in (e) and (f) (Appendix AModels B and C) are identical except the starting distribution at time t = 1 has a covariance of [[#x2212]]0.2 in (f) compared to 0.7 in (e). The parameterisation in (g) is chosen to demonstrate a case where the two traits evolve in different directions. The coloured image plots in figures (e)-(g) represent Gaussian development functions fitted to the bivariate distributions of at the beginning and end of the simulation. Evolution of the bivariate character has resulted in different parameterisations of these phenomenological functions. The lighter the shading, the greater the probability of a transition from character value at age 1 and to age 2.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F2.medium.gif" width="440" height="296"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F2.medium.gif" width="440" height="296"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <p id="p-37" class="first-child">The role of selection on the dynamics of 
        <span class="inline-formula" id="inline-formula-100">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-100.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-100.gif"/>
          </noscript>
         </span>
        </span>. Dynamics of univariate 
        <span class="inline-formula" id="inline-formula-101">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-101.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-101.gif"/>
          </noscript>
         </span>
        </span> subject to linear selection and clonal inheritance (a)-(d) (Appendix A§1.1 Model A). The population does not reach an equilibrium, with (a) population growth, and the (b) mean, (c) variance and (d) skew of the character continually evolving. Imposing a maximum possible character value constrains change (red lines versus black lines (a)-(d)). In the age-structured case we track the dynamics of a bivariate character distribution (e)-(g) (Appendix A§1.1 models B, C and D). The models in (e) and (f) (Appendix AModels B and C) are identical except the starting distribution at time t = 1 has a covariance of −0.2 in (f) compared to 0.7 in (e). The parameterisation in (g) is chosen to demonstrate a case where the two traits evolve in different directions. The coloured image plots in figures (e)-(g) represent Gaussian development functions 
        <span class="inline-formula" id="inline-formula-102">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-102.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-102.gif"/>
          </noscript>
         </span>
        </span> fitted to the bivariate distributions of 
        <span class="inline-formula" id="inline-formula-103">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-103.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-103.gif"/>
          </noscript>
         </span>
        </span> at the beginning and end of the simulation. Evolution of the bivariate character has resulted in different parameterisations of these phenomenological functions. The lighter the shading, the greater the probability of a transition from character value 
        <span class="inline-formula" id="inline-formula-104">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-104.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-104.gif"/>
          </noscript>
         </span>
        </span> at age 1 and to 
        <span class="inline-formula" id="inline-formula-105">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-105.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F2/embed/inline-graphic-105.gif"/>
          </noscript>
         </span>
        </span> age 2.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="The dynamics of inheritance (Appendix AModel E). The dynamics of (a) population growth rate (R0), the (b) mean and (c) variance of vary between models with clonal inheritance (black line), the convolution in equation (15) (red line) and the Gaussian inheritance function in equation (16) (blue line). Dynamics predicted from the convolution and the Gaussian inheritance function are indistinguishable in this model. (d) the temporal dynamics of the clonal model versus the other models. The initial distribution at t = 1 is Gaussian. After 100 generations the character distributions predicted by the clonal and sexual models have only diverged slightly. The infinitesimal model of quantitative genetics assumes that the dynamics of alleles can be inferred from the dynamics of genotypes. Under this assumption (e) selection alters genotype and allele frequencies, while inheritance does not. In contrast, (f) when dominance variance operates, both selection and inheritance alter genotype frequency while neither alter allele frequencies. For a Gaussian distributed character, (g) dominance variance acts as an offset, reducing the intercept of a Gaussian inheritance function." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-164699597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;The dynamics of inheritance (Appendix AModel E). The dynamics of (a) population growth rate (R0), the (b) mean and (c) variance of vary between models with clonal inheritance (black line), the convolution in equation (15) (red line) and the Gaussian inheritance function in equation (16) (blue line). Dynamics predicted from the convolution and the Gaussian inheritance function are indistinguishable in this model. (d) the temporal dynamics of the clonal model versus the other models. The initial distribution at t = 1 is Gaussian. After 100 generations the character distributions predicted by the clonal and sexual models have only diverged slightly. The infinitesimal model of quantitative genetics assumes that the dynamics of alleles can be inferred from the dynamics of genotypes. Under this assumption (e) selection alters genotype and allele frequencies, while inheritance does not. In contrast, (f) when dominance variance operates, both selection and inheritance alter genotype frequency while neither alter allele frequencies. For a Gaussian distributed character, (g) dominance variance acts as an offset, reducing the intercept of a Gaussian inheritance function.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F3.medium.gif" width="438" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F3.medium.gif" width="438" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <p id="p-38" class="first-child">The dynamics of inheritance (Appendix AModel E). The dynamics of (a) population growth rate (R0), the (b) mean and (c) variance of 
        <span class="inline-formula" id="inline-formula-106">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F3/embed/inline-graphic-106.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F3/embed/inline-graphic-106.gif"/>
          </noscript>
         </span>
        </span> vary between models with clonal inheritance (black line), the convolution in equation (15) (red line) and the Gaussian inheritance function in equation (16) (blue line). Dynamics predicted from the convolution and the Gaussian inheritance function are indistinguishable in this model. (d) the temporal dynamics of the clonal model versus the other models. The initial distribution at t = 1 is Gaussian. After 100 generations the character distributions predicted by the clonal and sexual models have only diverged slightly. The infinitesimal model of quantitative genetics assumes that the dynamics of alleles can be inferred from the dynamics of genotypes. Under this assumption (e) selection alters genotype and allele frequencies, while inheritance does not. In contrast, (f) when dominance variance operates, both selection and inheritance alter genotype frequency while neither alter allele frequencies. For a Gaussian distributed character, (g) dominance variance acts as an offset, reducing the intercept of a Gaussian inheritance function.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-39">In this model there are two ways to prevent the fitness function from generating change in the location of the distribution. First, the fitness function can take unimodal non-linear forms such as 
      <span class="inline-formula" id="inline-formula-107">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-107.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-107.gif"/>
        </noscript>
       </span>
      </span> with 
      <span class="inline-formula" id="inline-formula-108">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-108.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-108.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-109">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-109.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-109.gif"/>
        </noscript>
       </span>
      </span> constrained to non-negative values. This generates stabilizing selection, with the mean breeding value being maintained at the value that maximizes fitness. Eventually, in this model, the breeding value distribution will achieve a trivial equilibrium – a Dirac delta function at this value. Second, continual change in the location of the distribution can be prevented by defining a maximum possible value for 
      <span class="inline-formula" id="inline-formula-110">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-110.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-110.gif"/>
        </noscript>
       </span>
      </span> that cannot be exceeded. This captures a genetic constraint in the maximum possible character value – i.e. evolution has not evolved a genetic solution to creating a larger breeding value. In our models, this process can be captured by setting the abundance of 
      <em>N</em>(
      <span class="inline-formula" id="inline-formula-111">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-111.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-111.gif"/>
        </noscript>
       </span>
      </span> &gt;; 
      <em>x</em>, 1) where 
      <em>x</em> is the maximum possible trait value that evolution can achieve. Selection now pushes the breeding value distribution up to 
      <em>x</em>, again eventually achieving a trivial equilibrium captured by a Dirac delta function where all mass of the distribution is at 
      <span class="inline-formula" id="inline-formula-112">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-112.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-112.gif"/>
        </noscript>
       </span>
      </span> = 
      <em>x</em>.
     </p>
     <p id="p-40">Genetic constraints can also impact the transient dynamics of the breeding value distribution (
      <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>(a-d, red lines)). When we impose a genetic constraint (Appendix A§1.1 model A with 
      <em>x</em> = 11.5), the genetic variance and skew evolve faster than when no genetic constraint is in place (
      <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2(c)</a> and 
      <a id="xref-fig-2-8" class="xref-fig" href="#F2">(d)</a>). These more rapid changes result in a slowing in the evolution of the mean breeding value (
      <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2(b)</a>), and of the population growth rate (
      <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2(a)</a>).
     </p>
     <p id="p-41">Genetic covariances between traits can also capture genetic constraints and can also influence the outcome of evolution. We demonstrate this by developing an age-structured model. 
      <span class="inline-formula" id="inline-formula-113">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-113.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-113.gif"/>
        </noscript>
       </span>
      </span> now becomes age-structured but is still inherited at birth. We construct a multivariate character 
      <span class="inline-formula" id="inline-formula-114">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-114.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-114.gif"/>
        </noscript>
       </span>
      </span> describing the breeding values that influence a character at each age (e.g. 
      <span class="inline-formula" id="inline-formula-115">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-115.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-115.gif"/>
        </noscript>
       </span>
      </span> for breeding values at ages 
      <em>a</em> = 1,2,…, 
      <em>n</em>). If some of the same loci contribute to the genetic components of the character at different ages there is a genetic covariation across ages. The genetic variances within each age, and the covariances between ages, can be used to construct a 
      <strong>G</strong> matrix (
      <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lande, 1979</a>). Such age-structured 
      <strong>G</strong> matrices underpin the character-state approach of quantitative genetics (
      <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Lynch and Walsh, 1998</a>). In the age-structured model that follows, we define a bivariate normal distribution with a known variance-covariance structure as our starting point and iterate this forwards (Appendix A§1.1 models B-D). We consider a simple case: a monocarpic biennial life cycle where individuals in their first year of life do not reproduce and all age 2 individuals die after reproduction. As with our model for a species with non-overlapping generations we assume clonal inheritance, 
      <span class="disp-formula" id="disp-formula-13">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-18.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-18.gif"/>
        </noscript>
       </span>
      </span> where survival from age 1 to age 2 is specified as 
      <span class="disp-formula" id="disp-formula-14">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-19.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-19.gif"/>
        </noscript>
       </span>
      </span> with expected survival to age 2 being highest for larger values of 
      <span class="inline-formula" id="inline-formula-116">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-116.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-116.gif"/>
        </noscript>
       </span>
      </span>. Although 
      <span class="inline-formula" id="inline-formula-117">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-117.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-117.gif"/>
        </noscript>
       </span>
      </span> is not under direct selection, its distribution is modified by its covariance with 
      <span class="inline-formula" id="inline-formula-118">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-118.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-118.gif"/>
        </noscript>
       </span>
      </span>.
     </p>
     <p id="p-42">
      <span class="inline-formula" id="inline-formula-119">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-119.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-119.gif"/>
        </noscript>
       </span>
      </span>, the genotype at age 2, determines expected reproduction, 
      <span class="disp-formula" id="disp-formula-15">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-20.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-20.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-43">Although 
      <span class="inline-formula" id="inline-formula-120">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-120.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-120.gif"/>
        </noscript>
       </span>
      </span> does not directly influence reproduction, there is an association between it and reproduction via its covariance with 
      <span class="inline-formula" id="inline-formula-121">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-121.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-121.gif"/>
        </noscript>
       </span>
      </span>. All age 2 individuals die following reproduction in this model, although it is possible to extend our approach to any arbitrary number of ages.
     </p>
     <p id="p-44">The evolutionary dynamics that particular parameterizations of the fitness functions 
      <span class="inline-formula" id="inline-formula-122">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-122.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-122.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-123">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-123.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-123.gif"/>
        </noscript>
       </span>
      </span> generate are dependent upon (i) the initial covariance between the characters and (ii) the fitness functions (Appendix A§1.1 models B-D). Many parameterizations and initial covariances are likely to generate evolutionary dynamics that may be biologically unrealistic. We demonstrate this with three contrasting parameterizations, considering size as our trait (
      <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2(e)-(g)</a>). In the first example, (
      <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2(e)</a> Appendix A §1.1 model B), the two characters positively covary and experience selection in the same direction. Over the course of the simulation the average developmental trajectory has evolved with 
      <span class="inline-formula" id="inline-formula-124">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-124.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-124.gif"/>
        </noscript>
       </span>
      </span> evolving to be 1.76 times larger and 
      <span class="inline-formula" id="inline-formula-125">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-125.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-125.gif"/>
        </noscript>
       </span>
      </span> evolving to be 1.52 times larger. For a trait like body size, such a proportional change at different ages may be appropriate. In examples (
      <a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2(f and g)</a>, Appendix A§1.1 models C and D) the bivariate character evolves in contrasting ways. In (F), 
      <span class="inline-formula" id="inline-formula-126">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-126.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-126.gif"/>
        </noscript>
       </span>
      </span> evolves much faster than 
      <span class="inline-formula" id="inline-formula-127">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-127.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-127.gif"/>
        </noscript>
       </span>
      </span> while in (G) 
      <span class="inline-formula" id="inline-formula-128">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-128.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-128.gif"/>
        </noscript>
       </span>
      </span> evolves to be larger, while 
      <span class="inline-formula" id="inline-formula-129">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-129.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-129.gif"/>
        </noscript>
       </span>
      </span> evolves to be smaller. These simulations demonstrate that only a constrained set of fitness functions and genetic covariances will give biologically realistic evolutionary trajectories for the size-related traits that biologists often study.
     </p>
     <p id="p-45">We now return to a univariate model and examine the clonality assumption. How can the clonality assumption be relaxed, and what are the consequences? In sexually reproducing species, offspring inherit a mix of their parent’s genomes. However, genetic segregation means that full siblings do not have the same genotype. When additivity is assumed, the breeding value of offspring is expected to be midway between parental breeding values. However, to obtain the distribution of offspring genotypes, the contribution of genetic segregation to variation among offspring needs to be taken into account. In two sex models, three steps are required to generate the distribution of offspring genotypes or breeding values given parental values. First, a distribution of mating pairs needs to be constructed. Second, the distribution of midpoint parental genotypes or breeding values given the distribution of mating pairs needs to be constructed. Third, segregation variance needs to be added to the distribution (
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Feldman and Cavalli-Sforza, 1979</a>; 
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Felsenstein, 1981</a>; 
      <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Turelli and Barton, 1994</a>). The mating system and the segregation variance are related: when mating is assortative with respect to genotype, the segregation variance is small and siblings closely resemble one another and their parents. In contrast, when mating is disassortative with respect to genotype, siblings can differ markedly from one another, and the segregation variance is large.
     </p>
     <p id="p-46">Expressions have been derived for the segregation variance for the infinitesimal model where it is assumed that traits are determined by a very large number of unlinked loci of small additive effects and mating is random (
      <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Fisher, 1930</a>). The infinitesimal model is assumed in most empirical quantitative genetic analyses (
      <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Kruuk et al., 2008</a>) and in our initial model. For random mating where both sexes have identical demographies, the distribution of offspring breeding values given parental breeding values is (
      <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>): 
      <span class="disp-formula" id="disp-formula-16">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-21.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-21.gif"/>
        </noscript>
       </span>
      </span> where * represents convolution and 
      <span class="inline-formula" id="inline-formula-130">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-130.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-130.gif"/>
        </noscript>
       </span>
      </span> exp 
      <span class="inline-formula" id="inline-formula-131">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-131.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-131.gif"/>
        </noscript>
       </span>
      </span> is a Gaussian function with mean zero and variance 
      <em>σ</em>
      <sup>2</sup> representing the segregation variance.
     </p>
     <p id="p-47">If males and females have different demographies then they will have different distributions of genetic values after selection; we represent these as 
      <span class="inline-formula" id="inline-formula-132">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-132.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-132.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-133">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-133.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-133.gif"/>
        </noscript>
       </span>
      </span>, respectively. In this case, eq. (14) is replaced by 
      <span class="disp-formula" id="disp-formula-17">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-22.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-22.gif"/>
        </noscript>
       </span>
      </span> where 
      <span class="inline-formula" id="inline-formula-134">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-134.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-134.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-135">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-135.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-135.gif"/>
        </noscript>
       </span>
      </span> are variances of the post-recruitment-selection genetic value of males and females. respectively. We do not superscript the 
      <em>rs</em> with 
      <em>σ
       <sup>2</sup>
      </em> to avoid a notation making it appear 
      <em>σ</em> is raised to some quantity 2
      <em>r</em>.
     </p>
     <p id="p-48">The first two terms on the right hand side of equation (15) generates the distribution of expected parental midpoint values; it ensures that the mean breeding value among offspring is midway between the two parental breeding values. However, because the parental distributions are halved, the variance of this distribution is half that of the parental distributions. The third term on the right hand side of equation (15) adds the segregation variance. For random mating, the variance is assumed to be normally distributed with a mean of 0 and a variance of half the additive genetic variance among the entire population when the population is at linkage equilibrium (
      <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Felsenstein, 1981</a>). We approximate this variance as half the additive genetic variance in the parental distribution (
      <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Feldman and Cavalli-Sforza, 1979</a>). This approach has already been incorporated into IPMs (
      <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>; 
      <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Childs et al., 2016</a>).
     </p>
     <p id="p-49">We now run two simulations (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3(a)-(d)</a>) to examine differences in the predictions of clonal and sexual models. The first model assumes clonal inheritance and the second the convolution in Equation (15), with both models assuming a linear function 
      <span class="inline-formula" id="inline-formula-136">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-136.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-136.gif"/>
        </noscript>
       </span>
      </span> (Appendix A§1.1 model E). The two models predict slightly divergent dynamics. The reason for this is that equation (15) results in the skew and kurtosis in 
      <span class="inline-formula" id="inline-formula-137">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-137.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-137.gif"/>
        </noscript>
       </span>
      </span> is reduced at each time step in the sexual model compared to in the clonal model. If selection is exponential (and the starting distribution proportional to a Gaussian distribution) then there will be no difference between the two approaches. This is because a normal distribution multiplied by an exponential fitness function results in a normal distribution with an unchanged variance (Diaconis et al., 1979). These results suggest that insights from clonal models will approximate those from sexual models reasonably well, at least when males and females have similar demography.
     </p>
     <p id="p-50">Some authors have queried the use of Equation (3) as an approximation in IPMs to the inheritance convolution in Equation (15) used in models of sexually reproducing species (
      <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chevin et al., 2010</a>; Janeiro et al., in press). However, being able to construct inheritance functions for 
      <span class="inline-formula" id="inline-formula-138">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-138.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-138.gif"/>
        </noscript>
       </span>
      </span> that are of the form of equation (3) would be useful as it would permit methods developed for two sex phenotypic IPMs to be applied to evolutionarily explicit IPMs (e.g. 
      <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Schindler et al., 2015</a>). Given Gaussian approximations frequently perform well in models of evolution (
      <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Turelli and Barton, 1994</a>) we hypothesize that Gaussian inheritance functions may perform well in evolutionarily explicit IPMs. We consequently constructed a Gaussian inheritance function and compared results with those obtained from the convolution.
     </p>
     <p id="p-51">Equation (15) results in the mean and variance of the parental and offspring breeding value being the same. We can approximate this by ensuring that the function 
      <span class="inline-formula" id="inline-formula-139">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-139.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-139.gif"/>
        </noscript>
       </span>
      </span> passes through the coordinate 
      <span class="inline-formula" id="inline-formula-140">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-140.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-140.gif"/>
        </noscript>
       </span>
      </span> and that the variance 
      <span class="inline-formula" id="inline-formula-141">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-141.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-141.gif"/>
        </noscript>
       </span>
      </span>. When both sexes have the same demography, we can write, 
      <span class="disp-formula" id="disp-formula-18">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-23.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-23.gif"/>
        </noscript>
       </span>
      </span> where 
      <span class="inline-formula" id="inline-formula-142">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-142.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-142.gif"/>
        </noscript>
       </span>
      </span> and 
      <em>σ
       <sup>2</sup>
      </em> represent expectations and variances respectively and 
      <em>η</em> represents the degree of assortative mating. When 
      <em>η</em> mating is entirely assortative, when 
      <em>θ</em> = 0.5 mating is random and when 
      <em>η</em> = 0 mating is completely disassortative. An equation for the case when males and females have different demographies is provided in the Appendix A§1.3. The approximation in Equation (16) will increase in accuracy as the distribution of mid-point parental breeding values becomes more Gaussian.
     </p>
     <p id="p-52">When we compared predictions from equations (15) and (16) with 
      <em>η</em> = 0.5 using the same model used to compare clonal and sexual life histories, results were indistinguishable (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3(a)</a>-(d). This reveals that, for linear selection, Gaussian inheritance functions for 
      <span class="inline-formula" id="inline-formula-143">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-143.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-143.gif"/>
        </noscript>
       </span>
      </span> perform remarkably well.
     </p>
     <p id="p-53">None of our models to date include any form of mutation. We have not incorporated mutation into our models as we are simulating responses to environmental change over a few tens to hundreds of generations (
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figures 1</a>-
      <a id="xref-fig-3-4" class="xref-fig" href="#F3">3</a>), and over that time period mutation is unlikely to play a major role in adaptation. However, for simulations over longer time periods, we can incorporate mutation into our models by slightly increasing the size of the segregation variance (e.g 
      <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Lynch and Walsh, 1998</a>). This will have the effect of increasing the additive genetic variance, partly countering any loss of genetic variance due to selection.
     </p>
     <p id="p-54">Our approximation can be used to examine the dynamical contributions of non-additive genetic processes to population responses to environmental change in a phenomenological manner. 
      <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Fisher (1930)</a> demonstrated that dominance variance can be treated as an offset, and in our models this would lower the intercept of the function 
      <span class="inline-formula" id="inline-formula-144">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-144.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-144.gif"/>
        </noscript>
       </span>
      </span> in equation (16). A consequence of this is that the mean of the offspring genotype is no longer equal to the mean of parental genotype and the dynamics of genotypes no longer exactly match the dynamics of alleles. We demonstrate this with a single locus-two allele model. When the effects of alleles are additive, the dynamics of the genotype captures the dynamics of alleles (
      <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3(e)</a>). In contrast, when the heterozygote has higher fitness, allele frequencies do not change once the equilibrium is achieved. However, selection and inheritance alter genotype frequencies (
      <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3(f)</a>). This effect of dominance variance can be phenomenologically capturing within an IPM by setting the intercept of the inheritance function for the genetic component of the phenotype to be less than 
      <span class="inline-formula" id="inline-formula-145">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-145.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-145.gif"/>
        </noscript>
       </span>
      </span> – this imposes an offset that can reverse gains made by selection (
      <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3(g)</a>). Because this offset is negative when dominance variance is operating, dominance variance will slow, or prevent, rates of evolutionary change. We could easily phenomenologically explore how a particular value of this offset impacts predicted dynamics, however, further work is required to relate different levels of dominance variance to specific values of the offset in our models.
     </p>
     <p id="p-55">Having shown how IPMs can be formulated to project forwards the dynamics of the genetic component of the phenotype under a wide range of circumstances, we now turn our attention to the dynamics of the environmental component of the phenotype.</p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Plasticity</h3>
     <p id="p-56">Plasticity is determined by the dynamics of 
      <span class="inline-formula" id="inline-formula-146">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-146.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-146.gif"/>
        </noscript>
       </span>
      </span> and in particular in how 
      <span class="inline-formula" id="inline-formula-147">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-147.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-147.gif"/>
        </noscript>
       </span>
      </span> is influenced by the ecological environment 
      <em>θ</em>. For this, we require a probability density function. We show in this section how different forms of plasticity can be incorporated into evolutionarily explicit IPMs, and explore the dynamics of some simple cases.
     </p>
     <p id="p-57">To capture plasticity in IPMs we need to model the probability of transition from 
      <span class="inline-formula" id="inline-formula-148">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-148.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-148.gif"/>
        </noscript>
       </span>
      </span> at time 
      <em>t</em> to 
      <span class="inline-formula" id="inline-formula-149">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-149.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-149.gif"/>
        </noscript>
       </span>
      </span> at time 
      <em>t</em> + 1 as a function of the environment 
      <em>θ</em>. For most plastic traits we have a poor mechanistic understanding of development and inheritance patterns, and for that reason we use the Gaussian probability density function in Equation (3).
     </p>
     <p id="p-58">In quantitative genetics it is often assumed that the mean of 
      <span class="inline-formula" id="inline-formula-150">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-150.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-150.gif"/>
        </noscript>
       </span>
      </span> and any individual departures are purely random (
      <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Falconer, 1960</a>). In equation 3 this requires the intercepts and slopes of the functions 
      <em>μ</em>
      <sup>D</sup>(…) and 
      <em>μ</em>
      <sup>Η</sup>(…) to take the following values: 
      <span class="inline-formula" id="inline-formula-151">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-151.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-151.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-152">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-152.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-152.gif"/>
        </noscript>
       </span>
      </span>. We relax this assumption and allow the mean (and variance) of 
      <span class="inline-formula" id="inline-formula-153">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-153.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-153.gif"/>
        </noscript>
       </span>
      </span> to vary with time as θ varies by specifying particular forms for development and inheritance functions of 
      <span class="inline-formula" id="inline-formula-154">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-154.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-154.gif"/>
        </noscript>
       </span>
      </span>.
     </p>
     <p id="p-59">Gaussian transition functions (equation 3) can be formulated to predictably modify moments of the distribution of 
      <span class="inline-formula" id="inline-formula-155">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-155.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-155.gif"/>
        </noscript>
       </span>
      </span> from time 
      <em>t</em> to time 
      <em>t</em> + 1. For example, careful choice of intercepts and slopes of 
      <span class="inline-formula" id="inline-formula-156">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-156.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-156.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-157">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-157.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-157.gif"/>
        </noscript>
       </span>
      </span> can be used to predictable grow, or shrink, the variance of 
      <span class="inline-formula" id="inline-formula-158">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-158.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-158.gif"/>
        </noscript>
       </span>
      </span> via either development or inheritance (Appendix A§1.4). In addition, specific biological processes can be easily incorporated into the dynamics of 
      <span class="inline-formula" id="inline-formula-159">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-159.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-159.gif"/>
        </noscript>
       </span>
      </span>: if the slopes 
      <span class="inline-formula" id="inline-formula-160">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-160.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-160.gif"/>
        </noscript>
       </span>
      </span> then there will be temporal autocorrelation in the value of 
      <span class="inline-formula" id="inline-formula-161">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-161.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-161.gif"/>
        </noscript>
       </span>
      </span> among individuals, and between parents and their offspring. For example, if 
      <span class="inline-formula" id="inline-formula-162">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-162.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-162.gif"/>
        </noscript>
       </span>
      </span> then individuals with a relatively large value of 
      <span class="inline-formula" id="inline-formula-163">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-163.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-163.gif"/>
        </noscript>
       </span>
      </span> at time 
      <em>t</em> will be expected to have a relatively large value of 
      <span class="inline-formula" id="inline-formula-164">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-164.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-164.gif"/>
        </noscript>
       </span>
      </span> at time 
      <em>t</em> + 1. This property of development functions is useful as it allows some memory of 
      <span class="inline-formula" id="inline-formula-165">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-165.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-165.gif"/>
        </noscript>
       </span>
      </span> across ages: if an individual has benefited from a particularly good set of circumstances at one age, any phenotypic consequences can persist to older ages. In a similar vein, if 
      <span class="inline-formula" id="inline-formula-166">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-166.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-166.gif"/>
        </noscript>
       </span>
      </span> then a parent with a relatively large 
      <span class="inline-formula" id="inline-formula-167">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-167.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-167.gif"/>
        </noscript>
       </span>
      </span> at time 
      <em>t</em> will produce offspring with relatively large 
      <span class="inline-formula" id="inline-formula-168">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-168.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-168.gif"/>
        </noscript>
       </span>
      </span> at time t + 1, a form of parental environmental effect (
      <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Nussey et al., 2007</a>).
     </p>
     <p id="p-60">Different formulations of 
      <em>μ</em>
      <sup>Η</sup>(…) and 
      <em>μ
       <sup>D</sup>
      </em>(…) can be used to capture a variety of different forms of plasticity (
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). When 
      <em>θ</em> is incorporated as an additive effect, it acts to shift the intercept of these functions as 
      <em>t</em> changes. This means that the environment influences all values of 
      <span class="inline-formula" id="inline-formula-169">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-169.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-169.gif"/>
        </noscript>
       </span>
      </span> in the same manner. If 
      <span class="inline-formula" id="inline-formula-170">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-170.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-170.gif"/>
        </noscript>
       </span>
      </span> then 
      <span class="inline-formula" id="inline-formula-171">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-171.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-171.gif"/>
        </noscript>
       </span>
      </span> changes as a function of how 
      <em>θ</em> influences 
      <span class="inline-formula" id="inline-formula-172">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-172.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-172.gif"/>
        </noscript>
       </span>
      </span> if 
      <span class="inline-formula" id="inline-formula-173">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-173.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-173.gif"/>
        </noscript>
       </span>
      </span> remains constant. 
      <span class="inline-formula" id="inline-formula-174">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-174.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-174.gif"/>
        </noscript>
       </span>
      </span> remains constant when it does not vary within individuals as they age, or if 
      <span class="inline-formula" id="inline-formula-175">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-175.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-175.gif"/>
        </noscript>
       </span>
      </span> in offspring is the same as 
      <span class="inline-formula" id="inline-formula-176">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-176.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-176.gif"/>
        </noscript>
       </span>
      </span> in parents.
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/297858/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/297858/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/297858" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2:</span>
       <p id="p-61" class="first-child">Different forms of plasticity and their incorporation into IPMs. Each term in the table below can be included in the functions 
        <span class="inline-formula" id="inline-formula-177">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-177.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-177.gif"/>
          </noscript>
         </span>
        </span> or 
        <span class="inline-formula" id="inline-formula-178">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-178.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-178.gif"/>
          </noscript>
         </span>
        </span>. Similar terms could be included in 
        <span class="inline-formula" id="inline-formula-179">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-179.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-179.gif"/>
          </noscript>
         </span>
        </span> or 
        <span class="inline-formula" id="inline-formula-180">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-180.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-180.gif"/>
          </noscript>
         </span>
        </span> if the variance in inheritance or development varied for specific values of 
        <span class="inline-formula" id="inline-formula-181">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-181.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/T2/embed/inline-graphic-181.gif"/>
          </noscript>
         </span>
        </span> in predictable ways. This would capture different forms of bet-hedging.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-62">Interactions between 
      <span class="inline-formula" id="inline-formula-182">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-182.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-182.gif"/>
        </noscript>
       </span>
      </span>,
      <span class="inline-formula" id="inline-formula-183">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-183.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-183.gif"/>
        </noscript>
       </span>
      </span> and θ are listed in 
      <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. Each form describes a different type of reaction norm (
      <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Gavrilets and Scheiner, 1993</a>). These forms allow 
      <span class="inline-formula" id="inline-formula-184">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-184.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-184.gif"/>
        </noscript>
       </span>
      </span> to develop among individuals (phenotypic plasticity) or be inherited (epigenetic inheritance) as a function of an individual’s breeding value 
      <span class="inline-formula" id="inline-formula-185">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-185.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-185.gif"/>
        </noscript>
       </span>
      </span> and the environment θ as well as the value of 
      <span class="inline-formula" id="inline-formula-186">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-186.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-186.gif"/>
        </noscript>
       </span>
      </span> at time t.
     </p>
     <p id="p-63">Plasticity can be either adaptive or non-adaptive (
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ghalambor et al., 2015</a>), and both forms can be captured into our models. Adaptive plasticity enables populations to rapidly respond to an environmental change. For example, if environmental change reduces population size, then adaptive plasticity would result in a change to the mean of the phenotype via either phenotypic plasticity (the development function) or epigenetic inheritance (the inheritance function) that leads to an increase in survival or recruitment rates. In contrast, non-adaptive plasticity does the opposite, potentially exacerbating the detrimental effects of environmental change.
     </p>
     <p id="p-64">We demonstrate this with an example of a simple IPM of a species with non-overlapping generations: 
      <span class="inline-formula" id="inline-formula-187">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-187.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-187.gif"/>
        </noscript>
       </span>
      </span>. The model contains no genetic variation and the phenotype is determined by the density at the time the offspring is born. This means we can remove 
      <span class="inline-formula" id="inline-formula-188">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-188.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-188.gif"/>
        </noscript>
       </span>
      </span> from the model. We assume a linear fitness function and a Gaussian inheritance function,
     </p>
     <span class="disp-formula" id="disp-formula-19">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-25.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-25.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-65">Next, we assume that the phenotypic trait is positively associated with expected recruitment such that 
      <span class="inline-formula" id="inline-formula-189">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-189.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-189.gif"/>
        </noscript>
       </span>
      </span>. We also assume that the environmental driver is positively associated with expected recruitment such that as 
      <em>θ</em> increases in value, fitness increases (
      <em>R
       <sub>θ</sub>
      </em> &gt;; 0). This means that the population growth rate (in a density-independent model) or population size (in a density-dependent model) also increases with 
      <em>θ</em>. Now assume that a negative environmental perturbation decreases 
      <em>θ</em> such that fitness decreases. For adaptive plasticity to counter this, the effect of the decrease in 
      <em>θ</em> on epigenetic inheritance must increase the expected value of 
      <span class="inline-formula" id="inline-formula-190">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-190.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-190.gif"/>
        </noscript>
       </span>
      </span>. In our simple model, this can only occur if 
      <span class="inline-formula" id="inline-formula-191">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-191.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-191.gif"/>
        </noscript>
       </span>
      </span> Then, as 
      <em>θ</em> declines, 
      <span class="inline-formula" id="inline-formula-192">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-192.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-192.gif"/>
        </noscript>
       </span>
      </span> becomes less, and the value of 
      <span class="inline-formula" id="inline-formula-193">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-193.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-193.gif"/>
        </noscript>
       </span>
      </span> becomes larger, increasing the mean of 
      <span class="inline-formula" id="inline-formula-194">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-194.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-194.gif"/>
        </noscript>
       </span>
      </span> and fitness. In general, in additive linear models like this, if 
      <span class="inline-formula" id="inline-formula-195">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-195.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-195.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-196">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-196.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-196.gif"/>
        </noscript>
       </span>
      </span> take opposing signs then plasticity will be adaptive.
     </p>
     <p id="p-66">We develop three density-dependent models of a phenotype in a species with non-overlapping generations. In all models we define the fitness function to be 
      <span class="inline-formula" id="inline-formula-197">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-197.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-197.gif"/>
        </noscript>
       </span>
      </span> where 
      <span class="inline-formula" id="inline-formula-198">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-198.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-198.gif"/>
        </noscript>
       </span>
      </span> and where 
      <em>R
       <sub>n(t)</sub>
      </em> &lt;; 0. In each model we define 
      <span class="inline-formula" id="inline-formula-199">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-199.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-199.gif"/>
        </noscript>
       </span>
      </span>. We set in model 
      <span class="inline-formula" id="inline-formula-200">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-200.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-200.gif"/>
        </noscript>
       </span>
      </span> in model 
      <span class="inline-formula" id="inline-formula-201">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-201.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-201.gif"/>
        </noscript>
       </span>
      </span> and in model 
      <span class="inline-formula" id="inline-formula-202">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-202.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-202.gif"/>
        </noscript>
       </span>
      </span> (Appendix A§1.1).
     </p>
     <p id="p-67">The first model (F) does not include plasticity 
      <span class="inline-formula" id="inline-formula-203">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-203.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-203.gif"/>
        </noscript>
       </span>
      </span> the second (G) captures adaptive plasticity 
      <span class="inline-formula" id="inline-formula-204">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-204.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-204.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-205">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-205.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-205.gif"/>
        </noscript>
       </span>
      </span>, and the third (H) captures non-adaptive plasticity (
      <span class="inline-formula" id="inline-formula-206">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-206.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-206.gif"/>
        </noscript>
       </span>
      </span>) and 
      <span class="inline-formula" id="inline-formula-207">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-207.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-207.gif"/>
        </noscript>
       </span>
      </span>. All three models include temporal autocorrelation in the environmental component of the phenotype (sometimes referred to as phenotypic carryover) when 
      <span class="inline-formula" id="inline-formula-208">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-208.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-208.gif"/>
        </noscript>
       </span>
      </span> (
      <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Because the models are not age-structured and do not include development, plasticity operates via epigenetic inheritance (e.g. maternal environmental effects). The same logic can be extended to the development function in age-structured populations. In our examples, parameterizations are chosen so all models converge to the same value of carrying capacity, 
      <em>K</em>. Once all three models have converged, we initially impose a one off perturbation. Model (G) regains the equilibrium first, followed by model (F), and then model (H) (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4(a)</a>) showing that adaptive plasticity allows the population to recover from a one off environmental perturbation much faster than when there is no plasticity, or plasticity is non-adaptive. Non-adaptivity plasticity significantly slows the rate at which the population can recover from a perturbation, with the initial population size pre-perturbation only re-attained after 80 generations.
     </p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Dynamics of and plasticity. (a) Return times to equilibrium for three inheritance functions (Appendix A[[#xA7]]1.1 models F-H) following a one off perturbation (see main text). There is no plasticity incorporated into model F (black line). Model G (red line) and model H (blue line) respectively incorporate adaptive and non-adaptive phenotypic plasticity. In (b) and (c) we imposed a permanent environmental change by reducing the intercept of the fitness function. (c) Represents the mean phenotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-164699597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Dynamics of and plasticity. (a) Return times to equilibrium for three inheritance functions (Appendix A[[#xA7]]1.1 models F-H) following a one off perturbation (see main text). There is no plasticity incorporated into model F (black line). Model G (red line) and model H (blue line) respectively incorporate adaptive and non-adaptive phenotypic plasticity. In (b) and (c) we imposed a permanent environmental change by reducing the intercept of the fitness function. (c) Represents the mean phenotype.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F4.medium.gif" width="440" height="141"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F4.medium.gif" width="440" height="141"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4.</span>
       <p id="p-68" class="first-child">Dynamics of 
        <span class="inline-formula" id="inline-formula-209">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F4/embed/inline-graphic-209.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F4/embed/inline-graphic-209.gif"/>
          </noscript>
         </span>
        </span> and plasticity. (a) Return times to equilibrium for three inheritance functions (Appendix A§1.1 models F-H) following a one off perturbation (see main text). There is no plasticity incorporated into model F (black line). Model G (red line) and model H (blue line) respectively incorporate adaptive and non-adaptive phenotypic plasticity. In (b) and (c) we imposed a permanent environmental change by reducing the intercept of the fitness function. (c) Represents the mean phenotype.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-69">Adaptive and non-adaptive plasticity also impact the way populations respond to permanent environmental change. We demonstrate this by running the same models (F), (G) and (H), except now we impose a constant change in fitness by permanently changing the intercept of the fitness function 
      <em>R
       <sub>I</sub>
      </em>. When we do this, the three models attain different equilibria population sizes (
      <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4(b)</a>) and different mean phenotypes (
      <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4(c)</a>). Model (G) achieves a larger population size than the two other models. This buffering of the population against environmental change happens because adaptive phenotypic plasticity results in a change in the mean phenotype (
      <a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2(c)</a>) that increases the expected recruitment rate and asymptotic population size (
      <a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2(b)</a>). In contrast, non-adaptive plasticity exacerbates the consequences via a change in the mean phenotype that decreases fitness.
     </p>
     <p id="p-70">In contrast to our example models in the §
      <strong>Adaptive Evolution</strong>, the IPMs we have developed in this section, and indeed all non-genetic IPMs so far published, achieve an asymptotic population growth rate or equilibrium population size and a stable population structure. These IPMs have monotonically increasing or decreasing fitness functions: an increase in the character results in an increase in expected fitness. A consequence of this is that in these models the recruitment function acts to alter the location of the character distribution, and often also alter its shape (
      <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wallace et al., 2013</a>). This is reflected in the means (and often other moments) differ between the distributions of the phenotype pre- and post-selection. In models at equilibrium with monotonic fitness functions, the inheritance function must reverse the locational and shape changes caused by the fitness function. This is because at equilibrium the moments of the phenotype distribution at times t and t + 1 must be equal.
     </p>
     <p id="p-71">In models of species with non-overlapping generations at equilibrium like those above, the inheritance function for 
      <span class="inline-formula" id="inline-formula-210">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-210.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-210.gif"/>
        </noscript>
       </span>
      </span> must exactly reverse the changes to the character distribution generated by the fitness function. This requires moments of parental and offspring characters to differ from one another if 
      <span class="inline-formula" id="inline-formula-211">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-211.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-211.gif"/>
        </noscript>
       </span>
      </span>. When there is a correlation between parental and offspring traits in the inheritance function for 
      <span class="inline-formula" id="inline-formula-212">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-212.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-212.gif"/>
        </noscript>
       </span>
      </span> as in our models, the intercept of the inheritance function must take a value such that offspring characters are smaller than their parent’s were at the same age (
      <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Coulson and Tuljapurkar, 2008</a>).
     </p>
     <p id="p-72">IPMs for species with overlapping generations include development functions 
      <span class="inline-formula" id="inline-formula-213">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-213.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-213.gif"/>
        </noscript>
       </span>
      </span>. These functions can alter the size (population size) and shape of the distribution of 
      <span class="inline-formula" id="inline-formula-214">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-214.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-214.gif"/>
        </noscript>
       </span>
      </span> as individuals age. When generations are overlapping, and at equilibrium, changes to the location of the character distribution via survival, recruitment and development are all exactly countered by the inheritance functions 
      <span class="inline-formula" id="inline-formula-215">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-215.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-215.gif"/>
        </noscript>
       </span>
      </span>.
     </p>
     <p id="p-73">
      <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Coulson and Tuljapurkar (2008)</a> showed that in red deer age-specific effects meant that young and old parents were incapable of producing offspring that had the same body weight as they did at birth. This process reversed the effects of viability selection removing small individuals from the population early in life. The same process was observed in marmots (
      <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Ozgul et al., 2010</a>) and Soay sheep (
      <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Ozgul et al., 2009</a>) and may be general for body size in mammals.
     </p>
     <p id="p-74">The models we have developed do not incorporate the evolution of phenotypic plasticity. However, if genotype-by-environment interactions were included in models, such that different breeding values had different responses to environmental variation, then plasticity could evolve. If this was coupled with a segregation variance that introduced novel genetic variance, this could capture the evolution of novel phenotypic plasticity. However, over the time periods over which our simulations are conducted, the evolution of novel forms of phenotypic plasticity, is unlikely to play a major role in population responses to environmental change.</p>
     <p id="p-75">We have now developed IPMs for (i) 
      <span class="inline-formula" id="inline-formula-216">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-216.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-216.gif"/>
        </noscript>
       </span>
      </span> where we assumed all individuals had the same, constant, 
      <span class="inline-formula" id="inline-formula-217">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-217.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-217.gif"/>
        </noscript>
       </span>
      </span> and (ii) 
      <span class="inline-formula" id="inline-formula-218">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-218.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-218.gif"/>
        </noscript>
       </span>
      </span> where we assumed all individuals had the same, constant, 
      <span class="inline-formula" id="inline-formula-219">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-219.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-219.gif"/>
        </noscript>
       </span>
      </span>. We have shown how IPMs can capture a wide range of biological processes including adaptive and non-adaptive plasticity and correlated characters, and the circumstances when equilibria are achieved. We now link together these advances into models of the joint dynamics of the bivariate distribution 
      <span class="inline-formula" id="inline-formula-220">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-220.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-220.gif"/>
        </noscript>
       </span>
      </span>.
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Models for the phenotype consisting of genetic and environmental components</h3>
     <p id="p-76">In the section we construct models where the character can be determined by a mixture of the genetic and environmental components. These models allow us to explore how adaptive evolution is influenced by plasticity.</p>
     <p id="p-77">We first develop a dynamic univariate version of the Breeders equation (
      <a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Falconer, 1960</a>) for a species with non-overlapping generations in a constant environment. In this case, the environmental component of the phenotype is assumed to be a consequence of developmental noise: individuals achieve their genetic potential, plus or minus a departure. At each generation within each breeding value, the distribution of the environmental component of the phenotype is assumed to be Gaussian with a mean of 0 and a constant variance (Appendix A§1.1 Model I).
     </p>
     <p id="p-78">Our initial conditions are a bivariate Gaussian distribution of 
      <span class="inline-formula" id="inline-formula-221">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-221.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-221.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-222">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-222.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-222.gif"/>
        </noscript>
       </span>
      </span> which we iterate forwards for 300 time steps. Over time, the mean of the genetic component of the phenotype increases. In contrast, the mean of the environmental component is constant. The population grows hyper-exponentially (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5(a)</a>), the mean of the phenotype increases in value due to evolution (
      <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5(a,d)</a>) and the additive genetic variance is slowly eroded (
      <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure A2</a>). Because the additive genetic variance is eroded, while the phenotypic variance remains constant, the heritability declines over time (
      <a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure A2</a>).
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="A dynamic version of the Breeders Equation. The dynamics of the phenotype (red lines) and its genetic (black lines) and environmental (blue lines) components (a)-(c) and (g)-(i), and the dynamics of the population (d)-(f) and (j)-(l). In the first model (a) and (d), both fitness and inheritance of the environmental component of the phenotype are independent of density (Appendix A[[#xA7]]1.1 model I). In the second model (b) and (e) fitness is negatively density-dependent and inheritance of the environmental component of the phenotype is density-independent (Appendix A[[#xA7]]1.1 model J). In the third model, both fitness and inheritance of the environmental component of the phenotype are negative density-dependent (Appendix A[[#xA7]]1.1 Model K). The treatment of plasticity can dramatically influence the dynamics of the phenotype and population size (Appendix A[[#xA7]]1.1 models L-N). Adaptive phenotypic plasticity (h) and (k) leads to the population size and phenotype recovering from a perturbation much faster than non-adaptive plasticity (i)-(l). The absence of a plastic response (g) and (j) results in the population recovering from a perturbation at an intermediate rate between cases where adaptive and non-adaptive plasticity are operating." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-164699597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;A dynamic version of the Breeders Equation. The dynamics of the phenotype (red lines) and its genetic (black lines) and environmental (blue lines) components (a)-(c) and (g)-(i), and the dynamics of the population (d)-(f) and (j)-(l). In the first model (a) and (d), both fitness and inheritance of the environmental component of the phenotype are independent of density (Appendix A[[#xA7]]1.1 model I). In the second model (b) and (e) fitness is negatively density-dependent and inheritance of the environmental component of the phenotype is density-independent (Appendix A[[#xA7]]1.1 model J). In the third model, both fitness and inheritance of the environmental component of the phenotype are negative density-dependent (Appendix A[[#xA7]]1.1 Model K). The treatment of plasticity can dramatically influence the dynamics of the phenotype and population size (Appendix A[[#xA7]]1.1 models L-N). Adaptive phenotypic plasticity (h) and (k) leads to the population size and phenotype recovering from a perturbation much faster than non-adaptive plasticity (i)-(l). The absence of a plastic response (g) and (j) results in the population recovering from a perturbation at an intermediate rate between cases where adaptive and non-adaptive plasticity are operating.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F5.medium.gif" width="439" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F5.medium.gif" width="439" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5.</span>
       <p id="p-79" class="first-child">A dynamic version of the Breeders Equation. The dynamics of the phenotype (red lines) and its genetic (black lines) and environmental (blue lines) components (a)-(c) and (g)-(i), and the dynamics of the population (d)-(f) and (j)-(l). In the first model (a) and (d), both fitness and inheritance of the environmental component of the phenotype are independent of density (Appendix A§1.1 model I). In the second model (b) and (e) fitness is negatively density-dependent and inheritance of the environmental component of the phenotype is density-independent (Appendix A§1.1 model J). In the third model, both fitness and inheritance of the environmental component of the phenotype are negative density-dependent (Appendix A§1.1 Model K). The treatment of plasticity can dramatically influence the dynamics of the phenotype and population size (Appendix A§1.1 models L-N). Adaptive phenotypic plasticity (h) and (k) leads to the population size and phenotype recovering from a perturbation much faster than non-adaptive plasticity (i)-(l). The absence of a plastic response (g) and (j) results in the population recovering from a perturbation at an intermediate rate between cases where adaptive and non-adaptive plasticity are operating.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-80">Our second model (Appendix A§1.1 model J) has a negative density-dependent term in the fitness function. The phenotype evolves faster in this model than in our density-independent model (
      <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5(b)</a>). Population size grows nearly linearly in this model (
      <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5(d)</a>), although the rate of increase does slow slightly each generation as genetic variation is eroded. The difference between the hyper-exponential (density-independent model) and nearly linear increases (density-dependent model) in population size explain the difference in the rates of evolution. This is because the selection differential that determines the rate of evolution (an emergent property from our model (
      <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Wallace et al., 2013</a>)) has the population growth rate in its denominator. The population growth rate is smaller in the density-dependent model (just above unity) than in our density-independent one (it increases with time), and this leads to an increase in the strength of selection and the rate of evolution (see also 
      <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pelletier and Coulson, 2012</a>). A consequence of this is that the additive genetic variation and heritability tend towards zero faster the in density-dependent model than in the density-independent one (
      <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure A2</a>).
     </p>
     <p id="p-81">In our third model (Appendix A§1.1 model K), negative density-dependence is included in the inheritance function for the environmental component of the phenotype as well as in the fitness function. This captures adaptive phenotypic plasticity. This results in a negative change in the mean of the environmental component of the phenotype with time (
      <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5(c)</a>). This decrease is reflected in a change in the mean of the phenotype itself. Adaptive phenotypic plasticity leads to a decline in the population growth rate which results in a slight increase in the rate of evolution compared to the density-dependent model with no plasticity. However, the effect is not large and is only just distinguishable when comparing 
      <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figures 5(b)</a> and 
      <a id="xref-fig-5-7" class="xref-fig" href="#F5">(c)</a>.
     </p>
     <p id="p-82">In our final models (Appendix A§1.1 models L to N) we examine how a one off perturbation influences the mean of the phenotype, its components and the population growth rate (
      <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5(g)-(l)</a>) when there is no plasticity, adaptive plasticity and non-adaptive plasticity. We set the variance in the genetic and environmental component of the phenotype to be equal, giving an initial heritability of 
      <em>h
       <sup>2</sup>
      </em> = 0.5. In each model we allow the population to achieve the same equilibrium population size in the absence of selection 
      <span class="inline-formula" id="inline-formula-223">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-223.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-223.gif"/>
        </noscript>
       </span>
      </span>. We then impose a one off mortality event when 99% of individuals above the mean of the phenotype are killed off. At this point we also impose selection 
      <span class="inline-formula" id="inline-formula-224">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-224.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-224.gif"/>
        </noscript>
       </span>
      </span>. In all three models the mortality event results in a small change in the mean value of the phenotype (Appendix A§1.5 for an explanation) (
      <a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5(g)-(i)</a>, red lines) but a halving of population size (
      <a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5(j)-(l)</a>). Adaptive plasticity results in the environmental component of the phenotype returning to its pre-perturbation value very quickly (
      <a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5(g)-(i)</a> blue lines). In contrast, although the perturbation causes a modest change in the mean of the genetic component of the phenotype, it takes &gt;; 10 generations for evolution to reverse the change (
      <a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5(g)-(i)</a>, black lines). This demonstrates that a strong selective effect can leave a large population dynamic impact, but leave only a small initial signature in the phenotype even when the trait is highly heritable.
     </p>
     <p id="p-83">Over the longer term, the dynamics of the all components of the phenotype, the phenotype itself and the population dynamics all depend upon whether plasticity is adaptive or non-adaptive. Adaptive plasticity allows the population size to initially recover from the perturbation more quickly than when plasticity is absent or non-adaptive (
      <a id="xref-fig-5-13" class="xref-fig" href="#F5">Figure 5(j)-(l)</a>). However, over a longer time period, non-adaptive plasticity results in the population achieving a larger size than when plasticity is absent or adaptive. These differences in population growth rate impact rates of evolution: immediately following the perturbation, the rate of evolution is greatest when plasticity is non-adaptive. However, the rate of evolution then increases when plasticity is adaptive (Figures S2 and S3). As with our previous models, the effects of adaptive and non-adaptive plasticity on rates of evolution are relatively small, but our results demonstrate how the two processes can interact.
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Signatures of evolution in models that are not evolutionarily explicit</h3>
     <p id="p-84">The models in the previous section are quite complex. Do we always need to construct such evolutionarily explicit IPMs to predict population responses to environmental change, or can we rely on simpler, phenotypic IPMs? There are two reasons why it may be preferable to not construct evolutionarily explicit models. First, evolutionarily explicit IPMs are more complicated to construct than those that do not include genotypes or breeding values. Second, when data are unavailable to explicitly include breeding values into models (
      <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Traill et al., 2014
       <em>b</em>
      </a>), the effects of evolution on predicted dynamics can still be explored by examining the consequences of perturbing parameter values (
      <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Traill et al., 2014a</a>).
     </p>
     <p id="p-85">When evolution occurs within a system we would expect parameters in phenomenological inheritance and development functions that are fitted to data to change with time. We can see this in 
      <a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2(e)-(g)</a>). In these age-structured evolutionarily explicit models, the bivariate breeding value distribution (black contours) changes location as evolution occurs. We have fitted Gaussian development functions to these bivariate distributions at the beginning of each simulation and at the end (coloured image plots). The parameters that determine these developments functions have clearly changed as the location of the functions have changed. A similar process occurs for inheritance functions (not shown).
     </p>
     <p id="p-86">Numerous authors have previously noted this phenomenon in models of evolution. For example, in population genetic (
      <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Charlesworth, 1994</a>) and eco-evolutionary models (
      <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Coulson et al., 2011</a>; 
      <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Yoshida et al., 2003</a>) when genotype frequencies change with time, macroscopic, population level quantities like mean survival and recruitment also change; in adaptive dynamic models, as one strategy invades another, population level parameters inevitably change with strategy frequency over time (
      <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Metz et al., 1996</a>); in quantitative genetic predator-prey models population level parameters of both predators and prey vary over time leading to persistence of the interaction (
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Doebeli, 1997</a>); and in evolutionarily explicit IPMs parameters in inheritance functions have been shown to change with time as evolution progresses (
      <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Rees and Ellner, 2016</a>). These insights are useful because if evolution is occurring within a system, then temporal trends in statistical estimates of model parameters would be expected – in other words, the effect of time, either additively or in an interaction with other parameters, would be expected in 
      <span class="inline-formula" id="inline-formula-225">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-225.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-225.gif"/>
        </noscript>
       </span>
      </span> or 
      <span class="inline-formula" id="inline-formula-226">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-226.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-226.gif"/>
        </noscript>
       </span>
      </span>. If marked temporal trends are observed in parameters in development and inheritance functions that cannot be attributed to a changing environmental driver, then evolutionarily explicit IPMs may be required.
     </p>
     <p id="p-87">What about parameters in fitness functions 
      <span class="inline-formula" id="inline-formula-227">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-227.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-227.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-228">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-228.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-228.gif"/>
        </noscript>
       </span>
      </span> Can any inferences from temporal trends in these parameters be made? In our approach, evolution of a focal trait would not be expected to alter statistical estimates of the fitness functions. In our models, evolution simply moves the location and shape of the phenotype distribution, but not its association with survival or recruitment.
     </p>
     <p id="p-88">We have identified one circumstance where evolution will leave a signature in the dynamics of fitness function parameters. Parameters in these functions can evolve in the presence of a genetically unmeasured correlated character that is also evolving. To demonstrate this we construct a model of a bivariate character, examine the dynamics it predicts, before exploring the consequences of failing to measure one of the characters.</p>
     <p id="p-89">We assume clonal inheritance such that dynamics of the characters are solely determined by a bivariate fitness function,</p>
     <span class="disp-formula" id="disp-formula-20">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-28.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-28.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-90">The dynamics this model predicts depend upon the initial covariance between the two characters in a similar way to our age-structured model (equation 11). In our first example the two characters negatively covary, while in the second they positively covary (Appendix A§1.1 for model parameterizations). The initial negative covariation allows rapid evolution, with population growth (
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6(a)</a>), the mean of the characters (
      <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6(b)</a>), their variances (
      <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6(c)</a>)) and the covariance between them (
      <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6(d)</a>) evolving relatively quickly. In contrast, when the two characters positively covary, evolution is much slower, with the character means, variances and covariance changing much more slowly, even though the fitness functions are identical in each model (
      <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6(e)-(h)</a>).
     </p>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Dynamics of bivariate characters and evolution of fitness functions in the presence of an unmeasured, genetically correlated character (Appendix A[[#xA7]]1.1 model P and Q). We construct a simple model with clonal inheritance of two correlated characters that both influence fitness. We explore two initial starting conditions that only differ in their genetic covariance (Appendix A[[#xA7]]1.1 models P and Q). In (a)-(d) the covariance accelerates the rate of evolution compared to (e)-(h). The dynamics of the fitness function for each character when the other character is not measured (i) and (j). Regardless of the covariance between characters, the fitness functions evolve during the course of 120 time step simulation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-164699597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Dynamics of bivariate characters and evolution of fitness functions in the presence of an unmeasured, genetically correlated character (Appendix A[[#xA7]]1.1 model P and Q). We construct a simple model with clonal inheritance of two correlated characters that both influence fitness. We explore two initial starting conditions that only differ in their genetic covariance (Appendix A[[#xA7]]1.1 models P and Q). In (a)-(d) the covariance accelerates the rate of evolution compared to (e)-(h). The dynamics of the fitness function for each character when the other character is not measured (i) and (j). Regardless of the covariance between characters, the fitness functions evolve during the course of 120 time step simulation.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F6.medium.gif" width="440" height="418"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F6.medium.gif" width="440" height="418"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6.</span>
       <p id="p-91" class="first-child">Dynamics of bivariate characters and evolution of fitness functions in the presence of an unmeasured, genetically correlated character (Appendix A§1.1 model P and Q). We construct a simple model with clonal inheritance of two correlated characters that both influence fitness. We explore two initial starting conditions that only differ in their genetic covariance (Appendix A§1.1 models P and Q). In (a)-(d) the covariance accelerates the rate of evolution compared to (e)-(h). The dynamics of the fitness function for each character when the other character is not measured (i) and (j). Regardless of the covariance between characters, the fitness functions evolve during the course of 120 time step simulation.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-92">We now construct a fitness function for 
      <span class="inline-formula" id="inline-formula-229">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-229.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-229.gif"/>
        </noscript>
       </span>
      </span> when 
      <span class="inline-formula" id="inline-formula-230">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-230.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-230.gif"/>
        </noscript>
       </span>
      </span> is not measured. We start by defining mean fitness, an observable, as 
      <span class="inline-formula" id="inline-formula-231">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-231.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-231.gif"/>
        </noscript>
       </span>
      </span>. The slope 
      <span class="inline-formula" id="inline-formula-232">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-232.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-232.gif"/>
        </noscript>
       </span>
      </span> is given by, 
      <span class="disp-formula" id="disp-formula-21">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-30.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-30.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-93">The intercept can be calculated in the usual manner by estimating the means of fitness and 
      <span class="inline-formula" id="inline-formula-233">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-233.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-233.gif"/>
        </noscript>
       </span>
      </span>
      <span class="disp-formula" id="disp-formula-22">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-31.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-31.gif"/>
        </noscript>
       </span>
      </span> giving, 
      <span class="disp-formula" id="disp-formula-23">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-32.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-32.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-94">Equation (20) is what would be estimated from data if 
      <span class="inline-formula" id="inline-formula-234">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-234.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-234.gif"/>
        </noscript>
       </span>
      </span> were not measured and included in analyses (
      <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kendall, 2015</a>; ?). It will correctly describe the consequences of selection on 
      <span class="inline-formula" id="inline-formula-235">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-235.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-235.gif"/>
        </noscript>
       </span>
      </span> even though 
      <span class="inline-formula" id="inline-formula-236">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-236.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-236.gif"/>
        </noscript>
       </span>
      </span> could be correlated with it. This is because the unmeasured correlated character impacts fitness whether it is measured or not, and consequently impacts the association between the focal character and fitness in its absence (?). However, the fitness function cannot provide accurate predictions over multiple generations when it is assumed that the fitness function is constant.
     </p>
     <p id="p-95">Over multiple generations the existence of unmeasured correlated characters will alter parameters in the fitness function in Equation (20) if selection alters genetic variances and covariances of measured and unmeasured correlated characters (
      <a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6(i)-(j)</a>). This is because 
      <span class="inline-formula" id="inline-formula-237">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-237.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-237.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-238">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-238.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-238.gif"/>
        </noscript>
       </span>
      </span> are both functions of the covariance between the two characters (equations 18-20). If selection alters this covariance, parameters 
      <span class="inline-formula" id="inline-formula-239">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-239.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-239.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-240">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-240.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-240.gif"/>
        </noscript>
       </span>
      </span> will evolve with time. It is also why we use the subscript 
      <em>t</em> for 
      <span class="inline-formula" id="inline-formula-241">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-241.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-241.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-242">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-242.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-242.gif"/>
        </noscript>
       </span>
      </span>. Evidence of correlated characters under selection can consequently be inferred if parameters in fitness functions are observed to change with time in a system in the absence of a changing environmental driver. Note that a non-stationary unmeasured environmental driver could also generate trends in parameter values in fitness functions in phenomenological IPMs.
     </p>
    </div>
   </div>
   <div class="section" id="sec-8">
    <h2 class="">Discussion</h2>
    <p id="p-96">In this paper we develop an approach that allows prediction of how populations respond to environmental change via adaptive evolution and plasticity. We do this by incorporating insights from evolutionary genetics into data-driven structured population models. Our approach is to split the phenotype into its genetic and environmental components and to model the dynamics of the genetic component with functions based on understanding of the mechanisms of inheritance. In contrast, the dynamics of the environmental component of the phenotype are modeled with phenomenological functions that can be identified from the analysis of data. Our approach is appropriate for sexually reproducing or clonal species with either overlapping or non-overlapping generations.</p>
    <div id="sec-9" class="subsection">
     <h3>Evolutionarily explicit structured models</h3>
     <p id="p-97">IPMs are now a widely used tool in ecology and evolution because of their versatility and the ease with which they can be parameterized (
      <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Merow et al., 2014</a>). All key statistics routinely estimated in population ecology, quantitative genetics, population genetics and life history describe some aspect of a character distribution or its dynamics (
      <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Coulson et al., 2010</a>). IPMs are so versatile because they describe the dynamics of these distributions. Characterization of the determinants of these statistics gained via sensitivity or elasticity analysis of models have provided insight into how ecological and evolutionary quantities that interest biologists are linked (
      <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Coulson et al., 2011</a>). Although this logic was developed several years ago, there has recently been criticism that IPMs cannot be used to track the dynamics of multivariate breeding values expressed at different ages (
      <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chevin, 2015</a>; Janeiro et al., in press). Our paper addresses this criticism head-on—we show how IPMs can be formulated to capture a mechanistic understanding of inheritance and development. In demonstrating this we develop a general modeling approach to capture population responses to environmental change. Having done this, we are now in a position to construct IPMs of quantitative characters and examine how perturbing the environment will influence not only the dynamics of the phenotype and its genetic and environmental components, but also the life history (
      <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Steiner et al., 2014</a>, 
      <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">2012</a>) and population dynamics (
      <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Easterling et al., 2000</a>).
     </p>
     <p id="p-98">The work we present here adds to a growing literature that explicitly incorporates evolution into structured models, and IPMs in particular. Within the population genetics paradigm, 
      <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Charlesworth (1994)</a> developed structured models with a one-to-one map between genotype and phenotype in age-structured populations. Building on this work, 
      <a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Coulson et al. (2011)</a> showed how simple genetic architectures can be incorporated into IPMs, developing a model to explore how evolution at a single locus would occur simultaneously with phenotypic change of discrete and continuous characters, life history and population dynamics.
     </p>
     <p id="p-99">Working in the quantitative genetic paradigm, Lande (1982) derived age-structured models that tracked the dynamics of the mean of the additive genetic component of the phenotype (
      <span class="inline-formula" id="inline-formula-243">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-243.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-243.gif"/>
        </noscript>
       </span>
      </span> in our notation) and the mean of the phenotype itself 
      <span class="inline-formula" id="inline-formula-244">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-244.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-244.gif"/>
        </noscript>
       </span>
      </span>. He assumed a constant genetic-variance covariance matrix and consequently weak selection and normally distributed character values—assumptions we relax. 
      <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> extended Lande (1982)’s approach to track the dynamics of the entire character distribution and to stage-structured populations. In doing so, they developed a general, flexible approach to track the entire distributions of 
      <span class="inline-formula" id="inline-formula-245">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-245.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-245.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-246">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-246.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-246.gif"/>
        </noscript>
       </span>
      </span>. 
      <a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Childs et al. (2016)</a> extended this approach to two sexes. Because 
      <span class="inline-formula" id="inline-formula-247">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-247.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-247.gif"/>
        </noscript>
       </span>
      </span> is inherited with mechanistic rules that are not impacted by the environment, while inheritance and development of [
      <span class="inline-formula" id="inline-formula-248">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-248.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-248.gif"/>
        </noscript>
       </span>
      </span> are plastic and can be impacted by the ecological environment (
      <a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Falconer, 1960</a>), it is difficult to incorporate the effects of the environment on the dynamics of the phenotype by focusing on 
      <span class="inline-formula" id="inline-formula-249">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-249.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-249.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-250">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-250.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-250.gif"/>
        </noscript>
       </span>
      </span> as Lande (1982), 
      <a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> and 
      <a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Childs et al. (2016)</a> have done. In contrast, our approach (which otherwise has a similar logic to 
      <a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Barfield et al. (2011)</a> and 
      <a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Childs et al. (2016)</a>) tracks the dynamics of 
      <span class="inline-formula" id="inline-formula-251">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-251.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-251.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-252">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-252.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-252.gif"/>
        </noscript>
       </span>
      </span> (or 
      <span class="inline-formula" id="inline-formula-253">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-253.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-253.gif"/>
        </noscript>
       </span>
      </span>—the full genotypic value, including non-additive components—if desired), making incorporation of environmental drivers that influence inheritance and development of [
      <span class="inline-formula" id="inline-formula-254">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-254.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-254.gif"/>
        </noscript>
       </span>
      </span>] more straightforward. We show that it is possible to have selection operating on the phenotype while incorporating modern understanding of genetic inheritance into the dynamics of the genetic component of the phenotype and phenomenological insight into the role of the ecological environment on the dynamics of the environmental component of the phenotype. By doing this, we show how population responses to environmental change via adaptive evolution, phenotypic plasticity and epigenetic inheritance can be simultaneously explored. This opens up the way to provide novel insights into the circumstances when each process is expected to contribute to population responses to environmental change.
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Population responses to environmental change</h3>
     <p id="p-100">Unlike previous evolutionarily explicit IPMs (
      <a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">Barfield et al., 2011</a>; 
      <a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Childs et al., 2016</a>; 
      <a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Rees and Ellner, 2016</a>), our approach requires explicit consideration of the inheritance and development of 
      <span class="inline-formula" id="inline-formula-255">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-255.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-255.gif"/>
        </noscript>
       </span>
      </span>, the environmental component of the phenotype. This allows our models to capture a range of plastic responses to environmental change along with adaptive ones. What do our findings say about the contributions of plasticity, evolution, and their interaction to population responses to environmental change?
     </p>
     <p id="p-101">Detrimental environmental change often causes a decline in population size. When there is an association between a phenotypic trait and survival and recruitment rates, phenotypic change can lead to increased survival and recruitment rates (
      <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Ozgul et al., 2010</a>) and consequently an increase in population growth rate and size. Two processes can lead to phenotypic change - plasticity and adaptive evolution. There has been considerable discussion about the relative roles of each in allowing populations to respond to change (e.g. 
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bonduriansky et al., 2012</a>; 
      <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chevin et al., 2010</a>).
     </p>
     <p id="p-102">Genotypes and breeding values remain fixed within individuals throughout life which means that differential survival and recruitment rates are the processes that alter these distributions and underpin evolution. The strength of differential survival and recruitment can be impacted by environmental variation generating fluctuating selection (?). Environmental variation does not influence genetic inheritance: once mating pairs are formed, inheritance of breeding values, 
      <span class="inline-formula" id="inline-formula-256">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-256.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-256.gif"/>
        </noscript>
       </span>
      </span>, does not alter the mean or variance of breeding value distributions (
      <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Fisher, 1930</a>). In contrast, distributions of the environmental component of the phenotype can be altered via survival, recruitment, development and inheritance with each process potentially impacted by environmental variation (
      <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Reed et al., 2010</a>). Given these differences between the dynamics of 
      <span class="inline-formula" id="inline-formula-257">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-257.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-257.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-258">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-258.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-258.gif"/>
        </noscript>
       </span>
      </span> plasticity can lead to more rapid change than evolution in our models (e.g. 
      <a id="xref-fig-5-14" class="xref-fig" href="#F5">Figure 5</a>). This is because more biological processes can directly alter the distribution of plastic characters than can impact distributions of breeding values. These results are consistent with those of other authors, including Lande (2009) and 
      <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Chevin et al. (2010)</a>, who also concluded that plastic change should be faster than evolutionary change. But how quickly will evolution alter phenotypic trait distributions?
     </p>
     <p id="p-103">Our results on the speed of evolution suggest that claims of detectable rapid evolution in quantitative phenotypes is likely to take a few tens of generations. For example, environmental change increases mortality leading to a decline in population size, but for mortality selection to lead to evolutionary change over the course of a generation, a large proportion of the population needs to be selectively removed and the phenotype needs to be highly heritable. This is seen in our model results (
      <a id="xref-fig-5-15" class="xref-fig" href="#F5">Figure 5(g)-(i)</a>) and with a simple numerical example: when all individuals above the mean of a normally distributed character are removed from the population and the trait has a heritable of 
      <em>h
       <sup>2</sup>
      </em> = 0.5, population size halves in a single time step but the mean of the character will only shift from the 50
      <sup>th</sup> percentile to the 37.5
      <sup>th</sup> percentile. For a standard normal distribution with a mean of 0 and a standard deviation of unity, this means the mean would only shift by 0.319 – i.e. less than 
      <span class="inline-formula" id="inline-formula-259">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-259.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-259.gif"/>
        </noscript>
       </span>
      </span> of a standard deviation – i.e. a long way from statistical significance. In reality, mortality selection resulting from environmental change will likely result in a change to the mean of the distribution that is only a fraction of a standard deviation compared to our example. Given this, reports of rapid evolution due to environmental change increasing mortality selection over a small number of generations (e.g. 
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Coltman et al., 2003</a>) should be treated with extreme caution. It is much more likely that change is a consequence of phenotypic plasticity. Over multiple generations, recruitment selection can also contribute to evolutionary change and our approach allows the role of this to be investigated. However, unless reproduction is restricted to individuals with extreme phenotypic trait values in both sexes, it seems unlikely that evolution can generate statistically demonstrable evolutionary change over a small number of generations (?). This is not to say that evolution is not important over longer time scales. Over tens of generations evolution can shift phenotypic trait means to a greater extent than phenotypic plasticity (
      <a id="xref-fig-5-16" class="xref-fig" href="#F5">Figure 5(g)-(i)</a> blue versus black lines).
     </p>
     <p id="p-104">In order for plasticity to allow populations to rapidly respond to environmental change, a large proportion of individuals within the population must exhibit the same plastic response. A good example of such a dynamic is for size-related traits that are determined by resource availability, particularly when scramble competition is operating. When resources becoming limiting, all individuals will be unable to develop as rapidly as when resources are more common. A consequence of this is that individuals that developed in cohorts when resource were sparse will exhibit smaller body sizes compared to individuals in those cohorts that developed when resources were more abundant. We can capture this form of plasticity in our framework with an additive effect of density in the inheritance or development function for 
      <span class="inline-formula" id="inline-formula-260">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-260.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-260.gif"/>
        </noscript>
       </span>
      </span> (e.g. 
      <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). In contrast, when contest competition operates, larger individuals would acquire more resources than those that are smaller, and would develop faster. We can capture this in our models with interactions between density, 
      <span class="inline-formula" id="inline-formula-261">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-261.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-261.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-262">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-262.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-262.gif"/>
        </noscript>
       </span>
      </span> in either the inheritance or development functions for 
      <span class="inline-formula" id="inline-formula-263">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-263.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-263.gif"/>
        </noscript>
       </span>
      </span>.
     </p>
     <p id="p-105">The above discussion demonstrates how our approach can be used to capture different forms of plasticity. However, for plasticity to help populations respond to environmental change it must be adaptive: plasticity must change the mean trait value in a way that increases fitness (
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ghalambor et al., 2007</a>). We demonstrate that for additive, linear models, adaptive and non-adaptive plasticity can be specified by altering the sign for the effect of the environment in the function specifying the mean dynamics of the inheritance or development functions (
      <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). When interactions are included in these functions specifying general rules for whether plasticity is adaptive or non-adaptive will likely be more challenging. However, our approach provides a way in which to investigate when plasticity is adaptive or non-adaptive, and how different types of plasticity will influence population responses to environmental change.
     </p>
     <p id="p-106">Our results also show how plasticity can influence evolutionary rates. Plasticity, operating via development and inheritance functions for the environmental component of the phenotype, alters the distribution of the phenotype, and this can alter the strength of selection, which can then influence the dynamics of the genetic component of the phenotype (evolution). The effects of plasticity on selection and evolution can be surprisingly complex. We only examined the evolutionary consequences of plasticity following an environmental shock that influenced all individuals in the same way, but even in this simple case we found that adaptive plasticity initially slowed the rate of evolution compared to non-adaptive plasticity, before increasing it (
      <a id="xref-fig-5-17" class="xref-fig" href="#F5">Figure 5</a> and Appendix A). In general in order to understand how plasticity will influence selection, it is necessary to understand how it influences both the numerator and denominator of the selection differential that underpins evolution (
      <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Pelletier and Coulson, 2012</a>). The numerator is the covariance between the phenotype and absolute fitness (
      <a id="xref-ref-23-9" class="xref-bibr" href="#ref-23">Falconer, 1960</a>) and the denominator is mean fitness. In our models of species with non-overlapping generations this is mean recruitment – the population growth rate (
      <a id="xref-ref-26-7" class="xref-bibr" href="#ref-26">Fisher, 1930</a>). Selection is linear in our models where plasticity influences all individuals in the same way via an additive effect of density on inheritance of the environmental component of the phenotype (
      <a id="xref-fig-5-18" class="xref-fig" href="#F5">figure 5</a>), and this means that plasticity influences the population growth rate rather than the numerator of the selection differential. A consequence of this is that it is differences in the population growth rate that generates the differences in evolutionary rates between models when plasticity is adaptive and non-adaptive. In more complex cases when plasticity influences the covariance between the phenotype and fitness via genotype-phenotype interactions within a generation, to understand how selection influences evolution it is necessary to understand how plasticity not only influences mean fitness, but also how it generates differences between the covariance between the genetic component of the phenotype and fitness and the covariance between the phenotype itself and fitness. Because the components of the selection differential can be calculated from IPMs (
      <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Coulson et al., 2010</a>; 
      <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Wallace et al., 2013</a>) the approach we develop here provides a flexible way to examine how different types of plasticity can influence evolution following environmental change.
     </p>
     <p id="p-107">We have not considered bet-hedging in this paper. Bet-hedging is another form of plasticity that can influence the way populations respond to environmental change and it can be incorporated into IPMs (?). Deterministic IPMs incorporate probabilistic transitions when 
      <span class="inline-formula" id="inline-formula-264">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-264.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-264.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-265">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-265.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-265.gif"/>
        </noscript>
       </span>
      </span>. These probabilities do not vary from one time step to the next. In stochastic models these functions can include terms for an environmental driver 
      <em>θ</em>, such that the variation in trajectories changes with the environment. In evolutionarily explicit models, the variance in transition rates among different values of 
      <span class="inline-formula" id="inline-formula-266">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-266.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-266.gif"/>
        </noscript>
       </span>
      </span> can be made to depend upon 
      <em>θ</em>, 
      <span class="inline-formula" id="inline-formula-267">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-267.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-267.gif"/>
        </noscript>
       </span>
      </span> and their interaction (if desired). This means that individuals with specific values of 
      <span class="inline-formula" id="inline-formula-268">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-268.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-268.gif"/>
        </noscript>
       </span>
      </span> can produce offspring with more variable values of 
      <span class="inline-formula" id="inline-formula-269">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-269.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-269.gif"/>
        </noscript>
       </span>
      </span> (and consequently 
      <span class="inline-formula" id="inline-formula-270">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-270.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-270.gif"/>
        </noscript>
       </span>
      </span>) in particular environments than individuals with other values of 
      <span class="inline-formula" id="inline-formula-271">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-271.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-271.gif"/>
        </noscript>
       </span>
      </span>. In this paper we focused on the incorporation of 
      <em>θ</em> into 
      <span class="inline-formula" id="inline-formula-272">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-272.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-272.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-273">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-273.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-273.gif"/>
        </noscript>
       </span>
      </span> but responses to environmental change could also be incorporated into functions for the standard deviation that we use to construct our kernels.
     </p>
     <p id="p-108">In order to explore how the various forms of plasticity influence rates of evolution for real systems it will be necessary to parameterize our models with data.</p>
    </div>
    <div id="sec-11" class="subsection">
     <h3>Parameterizing and analyzing evolutionarily explicit IPMs</h3>
     <p id="p-109">A large literature exists on how to statistically parameterize IPMs (
      <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Easterling et al., 2000</a>; 
      <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Merow et al., 2014</a>; 
      <a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Rees et al., 2014</a>). The vast majority of IPMs have been constructed phenomenologically, using statistical descriptions of observational data. Several authors have shown how fixed and random effects incorporated into these statistical functions can be formulated within IPMs (
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Childs et al., 2003
       <em>a</em>
      </a>; 
      <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Coulson, 2012</a>; 
      <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rees and Ellner, 2009</a>), but additional statistical estimation is required to parameterize the evolutionarily explicit IPMs we have developed.
     </p>
     <p id="p-110">Fitness functions in evolutionarily explicit IPMs can be parameterized using standard general, generalized and additive regression methods that are routinely used to parameterize phenomenological IPMs (
      <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Rees and Ellner, 2009</a>). If relatedness information is available and the infinitesimal model is assumed, genetic and phenotypic variances and covariances can be estimated using the animal model (
      <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Lynch and Walsh, 1998</a>). These quantities can be used to construct the initial distributions of the genetic and environmental components of the phenotype. Parameter estimates of ecological drivers fitted as fixed or random effects in the animal model can be used to parameterize inheritance and development functions for the environmental component of the phenotype. It is consequently possible to parameterize models using our approach with existing methods.
     </p>
     <p id="p-111">There is also a large literature on how to analyze IPMs (
      <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Ellner and Rees, 2006</a>; Steiner et al.,
      <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">2014</a>, 
      <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">2012</a>). The majority of these tools, including sensitivity and elasticity analysis of model predictions to transition rates and function parameters (
      <a id="xref-ref-15-9" class="xref-bibr" href="#ref-15">Coulson et al., 2011</a>, 
      <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">2010</a>; 
      <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Ellner and Rees, 2006</a>; 
      <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Steiner et al., 2014</a>, 
      <a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">2012</a>), are likely sufficiently general to be applicable to evolutionarily explicit IPMs. In future work we plan to parameterize models for bird, mammal and fish species with overlapping generations and to analyze them with existing methods. Once evolutionarily explicit IPMs have been parameterized and analyzed we will be able to explore how populations, phenotypic characters and life histories are predicted to respond to a range of environmental changes via plasticity and adaptation.
     </p>
    </div>
    <div id="sec-12" class="subsection">
     <h3>When should evolutionarily explicit IPMs be used to predict population responses to environmental change?</h3>
     <p id="p-112">
      <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Chevin (2015)</a> and Janeiro et al. (in press) speculated that published IPMs that did not include explicit evolutionary processes could provide spurious insight. Three strands of evidence suggest this speculation may often be unwarranted.
     </p>
     <p id="p-113">First, the signature of evolutionary change in model predictions is a function of the heritability of the trait: when the phenotypic variance is dominated by the environmental component of the phenotype then the dynamics of that component will dominate model predictions. Most IPMs to date have been constructed for body weight (
      <a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Merow et al., 2014</a>), a trait that often has a heritability of less than 0.2 in vertebrates (e.g., blue tits; 
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Garnett, 1981</a>) and often around 0.1 (e.g., bighorn sheep; 
      <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Wilson et al., 2005</a>). This means that model predictions will be dominated by the dynamics of the environmental component of the phenotype and that a phenomenological statistical approach to parameterising these models has the potential to capture observed dynamics well.
     </p>
     <p id="p-114">Second, even when phenotypic traits are heritable, they rarely evolve in the wild as predicted: evolutionary stasis of heritable phenotypic traits in the presence of directional selection is frequently observed in nature (Merilä et al., 
      <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">2001</a>). When fitness functions are monotonic in the phenotypic value and selection is directional (which is typical for body size (
      <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Kingsolver et al., 2001</a>)), then in order to maintain an equilibrium trait distribution the inheritance function must reverse the phenotypic changes caused by selection. 
      <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Coulson and Tuljapurkar (2008)</a> showed this for the mean phenotypic trait. However, when the genotype-phenotype map is additive and there is additive genetic variance for the trait, directional selection is expected to result in evolutionary change and the inheritance function for the genetic component of the phenotype can not reverse genetic changes attributable to selection. Unmeasured genetically correlated characters can prevent evolutionary change in these circumstances, although the cases when this is likely to prevent evolution are restrictive, and evidence for such characters playing a major role in limiting evolution in the wild is lacking (
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agrawal and Stinchcombe, 2009</a>). Assuming selection on the phenotype has been measured appropriately and is directional, this suggests that the assumption of an additive genotype-phenotype map may be violated, and the mean of the parental and offspring breeding value distributions may not be equal. A mechanism such as over-dominance can achieve this (
      <a id="xref-ref-26-8" class="xref-bibr" href="#ref-26">Fisher, 1930</a>). Our approach allows the effects of relaxing assumptions of quantitative genetics on evolutionary change to be approximated through the use of phenomenological inheritance functions for the genetic component of the phenotype.
     </p>
     <p id="p-115">Third, because evolutionary change is rarely observed in the wild when it is predicted, observed phenotype change in natural populations is usually attributable to plasticity (e.g. 
      <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Ozgul et al., 2010</a>, 
      <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">2009</a>). In these cases, standard, non-evolutionarily explicit, IPMs have accurately captured observed dynamics (
      <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Childs et al., 2003
       <em>a</em>
      </a>; 
      <a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">Merow et al., 2014</a>; 
      <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Ozgul et al., 2010</a>).
     </p>
     <p id="p-116">These three strands of evidence suggest that evolutionarily explicit IPMs may frequently not be required to gain useful insight into population responses to environmental change. If there is no statistical evidence of temporal trends in inheritance, development or fitness function parameters once variation in the ecological environment has been corrected for, then the use of evolutionarily explicit IPMs may result in the construction of unnecessarily complex models. There is often a temptation to include ever more complexity into models, but this comes at the cost of analytical tractability: as more mechanisms or processes are incorporated into models, understanding why a model produces the predictions it does becomes increasingly challenging. However, when evolutionary change is convincingly documented (e.g. 
      <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Reznick et al., 1997</a>) or is proposed to be a possible mechanism generating rapid phenotypic change (
      <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Coltman et al., 2003</a>), the construction of evolutionarily explicit IPMs is advised as the models allow separation of the roles of adaptive and plastic responses to environmental change.
     </p>
     <p id="p-117">We have shown how evolutionarily explicit IPMs can be constructed, invalidating the criticisms of 
      <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Chevin (2015)</a> and Janeiro et al. (in press) that IPMs have not been developed to incorporate the character-state approach of quantitative genetics. IPMs that are not evolutionarily explicit have been used to address many questions in ecology and their application has proven insightful (
      <a id="xref-ref-49-9" class="xref-bibr" href="#ref-49">Merow et al., 2014</a>). They are likely to remain widely used and we expect this use to result in important new insights. However, we have extended their utility to cases where evolutionary processes are known, or proposed, to be drivers of phenotypic change.
     </p>
    </div>
   </div>
   <div class="section" id="sec-13">
    <h2 class="">Conclusions</h2>
    <p id="p-118">In this paper we have developed a theoretical modeling approach that links demography and quantitative genetics to explore how populations will respond to environmental change. The approach is general, providing formal links between ecology and evolution. Our work builds upon a growing literature of developing evolutionarily explicit structured population models. This body of literature shows how flexible IPMs are. They provide a powerful tool with the potential to unify ecology and evolution.</p>
   </div>
   <div class="section" id="sec-14">
    <h2 class="">Appendix A: Model parameters and predicted dynamics</h2>
    <p id="p-120">Model Parameterization</p>
    <p id="p-121">
     <strong>Model A</strong>: 
     <span class="disp-formula" id="disp-formula-24">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-33.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-33.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-122">
     <strong>Models B</strong> and 
     <strong>C</strong>: 
     <span class="disp-formula" id="disp-formula-25">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-34.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-34.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-123">Starting conditions at time 
     <em>t</em> = 1 are multivariate normal with the following parameters, Model B: 
     <span class="disp-formula" id="disp-formula-26">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-35.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-35.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-124">
     <strong>Model C</strong>: 
     <span class="disp-formula" id="disp-formula-27">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-36.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-36.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-125">
     <strong>Model D</strong>: 
     <span class="disp-formula" id="disp-formula-28">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-37.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-37.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-126">Starting conditions at time 
     <em>t</em> = 1 for 
     <strong>model D:</strong>
     <span class="disp-formula" id="disp-formula-29">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-38.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-38.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-127">
     <strong>Model E</strong>
     <span class="disp-formula" id="disp-formula-30">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-39.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-39.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-128">
     <strong>Model F</strong>: no plasticity: 
     <span class="disp-formula" id="disp-formula-31">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-40.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-40.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-129">
     <strong>Model G</strong>: Adaptive phenotypic plasticity: 
     <span class="disp-formula" id="disp-formula-32">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-41.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-41.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-130">
     <strong>Model H</strong>: Non-adaptive plasticity: 
     <span class="disp-formula" id="disp-formula-33">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-42.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-42.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-131">
     <strong>Model I</strong>
     <span class="disp-formula" id="disp-formula-34">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-43.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-43.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-132">
     <strong>Model J</strong>
     <span class="disp-formula" id="disp-formula-35">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-44.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-44.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-133">
     <strong>Model K</strong>
     <span class="disp-formula" id="disp-formula-36">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-45.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-45.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-134">Initial starting conditions for 
     <span class="inline-formula" id="inline-formula-274">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-274.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-274.gif"/>
       </noscript>
      </span>
     </span> for 
     <strong>models I to K</strong>: 
     <span class="disp-formula" id="disp-formula-37">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-46.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-46.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-135">
     <strong>Model L</strong>
     <span class="disp-formula" id="disp-formula-38">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-47.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-47.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-136">
     <strong>Model M</strong>
     <span class="disp-formula" id="disp-formula-39">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-48.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-48.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-137">
     <strong>Model N</strong>
     <span class="disp-formula" id="disp-formula-40">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-49.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-49.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <p id="p-138">Initial starting conditions for 
     <span class="inline-formula" id="inline-formula-275">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-275.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-275.gif"/>
       </noscript>
      </span>
     </span> for 
     <strong>models L to N</strong>: 
     <span class="disp-formula" id="disp-formula-41">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-50.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-50.gif"/>
       </noscript>
      </span>
     </span>
     <strong>Models P and Q</strong>: 
     <span class="disp-formula" id="disp-formula-42">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-51.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-51.gif"/>
       </noscript>
      </span>
     </span> Starting conditions at time 
     <em>t</em> + 1 for 
     <strong>model P</strong>: 
     <span class="disp-formula" id="disp-formula-43">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-52.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-52.gif"/>
       </noscript>
      </span>
     </span> Starting conditions at time 
     <em>t</em> + 1 for 
     <strong>model Q</strong>: 
     <span class="disp-formula" id="disp-formula-44">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-53.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-53.gif"/>
       </noscript>
      </span>
     </span>
    </p>
    <div class="section" id="sec-15">
     <h3>Calculating quantities from model outputs</h3>
     <p id="p-139">The expectation of a distribution of 
      <span class="inline-formula" id="inline-formula-276">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-276.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-276.gif"/>
        </noscript>
       </span>
      </span> can be calculated as
     </p>
     <span class="disp-formula" id="disp-formula-45">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-54.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-54.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-140">The variance of a distribution can be calculated as</p>
     <span class="disp-formula" id="disp-formula-46">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-55.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-55.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-141">For a bivariate distribution 
      <span class="inline-formula" id="inline-formula-277">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-277.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-277.gif"/>
        </noscript>
       </span>
      </span> consisting of traits 
      <span class="inline-formula" id="inline-formula-278">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-278.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-278.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-279">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-279.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-279.gif"/>
        </noscript>
       </span>
      </span> then the covariance between these two traits will be,
     </p>
     <span class="disp-formula" id="disp-formula-47">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-56.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-56.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-142">The skew can be calculated as,</p>
     <span class="disp-formula" id="disp-formula-48">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-57.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-57.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-143">The kurtosis can be calculated in the following way. First, we define the n
      <sup>th</sup> non-central moment of a distribution at time 
      <em>t</em> as 
      <span class="inline-formula" id="inline-formula-280">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-280.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-280.gif"/>
        </noscript>
       </span>
      </span> then,
     </p>
     <span class="disp-formula" id="disp-formula-49">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-58.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-58.gif"/>
       </noscript>
      </span>
     </span>
    </div>
    <div class="section" id="sec-16">
     <h3>Gaussian inheritance function when demography differs between males and females</h3>
     <p id="p-144">The distribution of mothers and fathers at time t is respectively defined as 
      <span class="inline-formula" id="inline-formula-281">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-281.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-281.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-282">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-282.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-282.gif"/>
        </noscript>
       </span>
      </span>. These distributions are the same size.
     </p>
     <p id="p-145">We can write 
      <span class="disp-formula" id="disp-formula-50">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-59.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-59.gif"/>
        </noscript>
       </span>
      </span> where the component functions of 
      <span class="inline-formula" id="inline-formula-283">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-283.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-283.gif"/>
        </noscript>
       </span>
      </span> are 
      <span class="disp-formula" id="disp-formula-51">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-60.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-60.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-284">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-284.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-284.gif"/>
        </noscript>
       </span>
      </span> is the variance in 
      <span class="inline-formula" id="inline-formula-285">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-285.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-285.gif"/>
        </noscript>
       </span>
      </span> across all parents.
     </p>
     <p id="p-146">Alternatively, 
      <span class="disp-formula" id="disp-formula-52">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-61.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-61.gif"/>
        </noscript>
       </span>
      </span> where the component functions of 
      <span class="inline-formula" id="inline-formula-286">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-286.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-286.gif"/>
        </noscript>
       </span>
      </span> are 
      <span class="disp-formula" id="disp-formula-53">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-62.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-62.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-147">As the distributions 
      <span class="inline-formula" id="inline-formula-287">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-287.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-287.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-288">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-288.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-288.gif"/>
        </noscript>
       </span>
      </span> depart from normality, the approximations will predict dynamics that diverge from those predicted by the convolution.
     </p>
    </div>
    <div class="section" id="sec-17">
     <h3>How do different functions alter character distributions?</h3>
     <p id="p-148">Assume 
      <span class="inline-formula" id="inline-formula-289">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-289.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-289.gif"/>
        </noscript>
       </span>
      </span> is proportional to a Gaussian distribution. The following parameterizations of a transition functions 
      <span class="inline-formula" id="inline-formula-290">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-290.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-290.gif"/>
        </noscript>
       </span>
      </span> in a model 
      <span class="inline-formula" id="inline-formula-291">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-291.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-291.gif"/>
        </noscript>
       </span>
      </span> will have no effect on the location or shape of the distribution such that 
      <span class="inline-formula" id="inline-formula-292">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-292.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-292.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <span class="disp-formula" id="disp-formula-54">
      <span class="highwire-responsive-lazyload">
       <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-63.gif"/>
       <noscript>
        <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/graphic-63.gif"/>
       </noscript>
      </span>
     </span>
     <p id="p-149">Note that in this model there is no fitness function and no selection.</p>
     <p id="p-150">When the intercept of 
      <span class="inline-formula" id="inline-formula-293">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-293.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-293.gif"/>
        </noscript>
       </span>
      </span> is less than 
      <span class="inline-formula" id="inline-formula-294">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-294.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-294.gif"/>
        </noscript>
       </span>
      </span> then 
      <span class="inline-formula" id="inline-formula-295">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-295.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-295.gif"/>
        </noscript>
       </span>
      </span> and vice versa. A function 
      <span class="inline-formula" id="inline-formula-296">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-296.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-296.gif"/>
        </noscript>
       </span>
      </span> can consequently be parameterized to reduce the mean of a distribution across generations or time steps if desired.
     </p>
     <p id="p-151">The slope 
      <em>β</em> will reduce 
      <span class="inline-formula" id="inline-formula-297">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-297.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-297.gif"/>
        </noscript>
       </span>
      </span> by 
      <em>β</em>
      <sup>2</sup> compared to 
      <span class="inline-formula" id="inline-formula-298">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-298.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-298.gif"/>
        </noscript>
       </span>
      </span>. The intercept of 
      <span class="inline-formula" id="inline-formula-299">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-299.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-299.gif"/>
        </noscript>
       </span>
      </span> injects additional variation. If the intercept is larger than 
      <span class="inline-formula" id="inline-formula-300">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-300.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-300.gif"/>
        </noscript>
       </span>
      </span> then 
      <span class="inline-formula" id="inline-formula-301">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-301.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-301.gif"/>
        </noscript>
       </span>
      </span>. Functions 
      <span class="inline-formula" id="inline-formula-302">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-302.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-302.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-303">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-303.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-303.gif"/>
        </noscript>
       </span>
      </span> can consequently be selected to alter the variance from one time step or age to the next.
     </p>
     <p id="p-152">The further the distribution 
      <span class="inline-formula" id="inline-formula-304">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-304.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-304.gif"/>
        </noscript>
       </span>
      </span> departs from normality, the more approximate these equalities will become. However, large departures from these equalities can be used to increase the mean or variance of any distribution in a desired direction.
     </p>
     <p id="p-153">In 
      <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure A1</a> we show how 
      <span class="inline-formula" id="inline-formula-305">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-305.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-305.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-306">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-306.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-306.gif"/>
        </noscript>
       </span>
      </span> can be parameterized to modify the mean and variance of 
      <span class="inline-formula" id="inline-formula-307">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-307.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-307.gif"/>
        </noscript>
       </span>
      </span> when it is proportional to a normal distribution.
     </p>
     <div id="F7" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F7.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="How parameterizations of transition functions for the environmental component of the phenotype can be used to grow, maintain or shrink the mean and variance of . We start with a normal distribution. The initial distribution is represented with a black line in the main figures. The inset figures in (a) to (c) shows the transition functions, with the black line representing the function that has no effect on the location or shape of . (a) increasing or decreasing the intercept of influences the location, but not the shape of . (b) How altering the slope of influences the shape of . In this example the mean is unaffected as the function passes through the x,y co-ordinate . (c) how altering the intercept of influences the variance. The transition functions in the insets of (b) and (c) generate distributions with the same means and variances (compare blue, red and black distributions in (b) and (c)). A change in variance between and achieved by altering the slope of or the intercept of generates different amounts of mixing. In (d) upper and lower functions impact the variance to the same extend (left hand figures) except the red function simply spreads out the distribution without altering the relative rank of each individual. In contrast, the blue function changes relative ranks (right hand figures)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-164699597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;How parameterizations of transition functions for the environmental component of the phenotype can be used to grow, maintain or shrink the mean and variance of . We start with a normal distribution. The initial distribution is represented with a black line in the main figures. The inset figures in (a) to (c) shows the transition functions, with the black line representing the function that has no effect on the location or shape of . (a) increasing or decreasing the intercept of influences the location, but not the shape of . (b) How altering the slope of influences the shape of . In this example the mean is unaffected as the function passes through the &lt;;em&gt;;x,y&lt;;/em&gt;; co-ordinate . (c) how altering the intercept of influences the variance. The transition functions in the insets of (b) and (c) generate distributions with the same means and variances (compare blue, red and black distributions in (b) and (c)). A change in variance between and achieved by altering the slope of or the intercept of generates different amounts of mixing. In (d) upper and lower functions impact the variance to the same extend (left hand figures) except the red function simply spreads out the distribution without altering the relative rank of each individual. In contrast, the blue function changes relative ranks (right hand figures).&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure A1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F7.medium.gif" width="440" height="429"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure A1." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F7.medium.gif" width="440" height="429"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure A1.</span>
       <p id="p-154" class="first-child">How parameterizations of transition functions for the environmental component of the phenotype 
        <span class="inline-formula" id="inline-formula-308">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-308.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-308.gif"/>
          </noscript>
         </span>
        </span> can be used to grow, maintain or shrink the mean and variance of 
        <span class="inline-formula" id="inline-formula-309">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-309.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-309.gif"/>
          </noscript>
         </span>
        </span>. We start with a normal distribution. The initial distribution is represented with a black line in the main figures. The inset figures in (a) to (c) shows the transition functions, with the black line representing the function that has no effect on the location or shape of 
        <span class="inline-formula" id="inline-formula-310">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-310.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-310.gif"/>
          </noscript>
         </span>
        </span>. (a) increasing or decreasing the intercept of 
        <span class="inline-formula" id="inline-formula-311">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-311.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-311.gif"/>
          </noscript>
         </span>
        </span> influences the location, but not the shape of 
        <span class="inline-formula" id="inline-formula-312">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-312.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-312.gif"/>
          </noscript>
         </span>
        </span>. (b) How altering the slope of 
        <span class="inline-formula" id="inline-formula-313">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-313.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-313.gif"/>
          </noscript>
         </span>
        </span> influences the shape of 
        <span class="inline-formula" id="inline-formula-314">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-314.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-314.gif"/>
          </noscript>
         </span>
        </span>. In this example the mean is unaffected as the function passes through the 
        <em>x,y</em> co-ordinate 
        <span class="inline-formula" id="inline-formula-315">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-315.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-315.gif"/>
          </noscript>
         </span>
        </span>. (c) how altering the intercept of 
        <span class="inline-formula" id="inline-formula-316">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-316.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-316.gif"/>
          </noscript>
         </span>
        </span> influences the variance. The transition functions in the insets of (b) and (c) generate distributions with the same means and variances (compare blue, red and black distributions in (b) and (c)). A change in variance between 
        <span class="inline-formula" id="inline-formula-317">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-317.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-317.gif"/>
          </noscript>
         </span>
        </span> and 
        <span class="inline-formula" id="inline-formula-318">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-318.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-318.gif"/>
          </noscript>
         </span>
        </span> achieved by altering the slope of 
        <span class="inline-formula" id="inline-formula-319">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-319.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-319.gif"/>
          </noscript>
         </span>
        </span> or the intercept of 
        <span class="inline-formula" id="inline-formula-320">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-320.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-320.gif"/>
          </noscript>
         </span>
        </span> generates different amounts of mixing. In (d) upper and lower 
        <span class="inline-formula" id="inline-formula-321">
         <span class="highwire-responsive-lazyload">
          <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-321.gif"/>
          <noscript>
           <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/F7/embed/inline-graphic-321.gif"/>
          </noscript>
         </span>
        </span> functions impact the variance to the same extend (left hand figures) except the red function simply spreads out the distribution without altering the relative rank of each individual. In contrast, the blue function changes relative ranks (right hand figures).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F8" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F8.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Dynamics of the additive genetic variance (a)-(c) and the heritability (d)-(f) in models I to K. Models of the additive genetic (back line) and environmental (red line) variance (g)-(i) and the heritability (j)-(l) in models L to N. See Figure 5 main paper for dynamics of means and population growth." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-164699597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Dynamics of the additive genetic variance (a)-(c) and the heritability (d)-(f) in models I to K. Models of the additive genetic (back line) and environmental (red line) variance (g)-(i) and the heritability (j)-(l) in models L to N. See Figure 5 main paper for dynamics of means and population growth.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure A2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F8.medium.gif" width="440" height="440"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure A2." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F8.medium.gif" width="440" height="440"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure A2.</span>
       <p id="p-155" class="first-child">Dynamics of the additive genetic variance (a)-(c) and the heritability (d)-(f) in models I to K. Models of the additive genetic (back line) and environmental (red line) variance (g)-(i) and the heritability (j)-(l) in models L to N. See 
        <a id="xref-fig-5-19" class="xref-fig" href="#F5">Figure 5</a> main paper for dynamics of means and population growth.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div class="section" id="sec-18">
     <h3>Mortality selection and changes in the mean phenotype</h3>
     <p id="p-156">When a trait is normally distribution, selection needs to be strong in order to substantially shift the mean of a phenotype distribution. Such strong selection inevitably leads to a decrease in population size. In 
      <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure A3</a> we show how killing 25% of the heaviest individuals has only a small effect on the mean for a distribution with a mean of 0 and a standard deviation of unity. The evolutionary response is even less if 
      <span class="inline-formula" id="inline-formula-322">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-322.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-322.gif"/>
        </noscript>
       </span>
      </span> and 
      <span class="inline-formula" id="inline-formula-323">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-323.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2017/03/24/090894/embed/inline-graphic-323.gif"/>
        </noscript>
       </span>
      </span> are uncorrelated. For example, in the example in 
      <a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure A3</a>, the evolutionary response would be half the phenotypic response for 
      <em>h</em>= 0.5. In order to substantially shift the mean of the a normal distribution via mortality selection it is necessary for the majority of the population to die.
     </p>
     <div id="F9" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F9.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="A normal distribution with mean 0 and standard deviation 1 prior to mortality selection (black line). Mortality occurs, killing off the top 25% of individuals (red distribution). The mean changes from 0 (vertical dashed line) to [[#x2212]]0.0324. In other words, even a large highly selective mortality event has a relatively small effect on the mean of a normal distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-164699597" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;A normal distribution with mean 0 and standard deviation 1 prior to mortality selection (black line). Mortality occurs, killing off the top 25% of individuals (red distribution). The mean changes from 0 (vertical dashed line) to [[#x2212]]0.0324. In other words, even a large highly selective mortality event has a relatively small effect on the mean of a normal distribution.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure A3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F9.medium.gif" width="440" height="398"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure A3." src="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F9.medium.gif" width="440" height="398"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure A3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2017/03/24/090894/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure A3.</span>
       <p id="p-157" class="first-child">A normal distribution with mean 0 and standard deviation 1 prior to mortality selection (black line). Mortality occurs, killing off the top 25% of individuals (red distribution). The mean changes from 0 (vertical dashed line) to −0.0324. In other words, even a large highly selective mortality event has a relatively small effect on the mean of a normal distribution.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-119">Bernt-Erik Saether, Luis-Miguel Chevin and Michael Morrissey participated in useful discussion and provided comments on an earlier version of the manuscript. Stephen Proulx, Thomas Reed, and two anonymous reviewers provided critical feedback that greatly improved the first submitted version of the paper. TC acknowledges support from NERC (grant number NE/K014218/1) and support of Centre for Advanced Study in Oslo, Norway, that funded and hosted the research project (Climate effects on harvested large mammal populations) during the Academic year of 2015/16.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Agrawal</span>, 
          <span class="cit-name-given-names">A. F.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. R.</span>
          <span class="cit-name-surname">Stinchcombe</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">How much do genetic covariances alter the rate of adaptation?</span>
         <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society of London</abbr> B 
         <span class="cit-vol">276</span>: 
         <span class="cit-fpage">1183</span>–
         <span class="cit-lpage">1191</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BLondon%26rft.volume%253D276%26rft.spage%253D1183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.2" data-doi="10.1086/658903">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Barfield</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. D.</span>
          <span class="cit-name-surname">Holt</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Gomulkiewicz</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Evolution in stage-structured populations</span>. 
         <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>
         <span class="cit-vol">177</span>: 
         <span class="cit-fpage">397</span>–
         <span class="cit-lpage">409</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DBarfield%26rft.auinit1%253DM.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D397%26rft.epage%253D409%26rft.atitle%253DEvolution%2Bin%2Bstage-structured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F658903%26rft_id%253Dinfo%253Apmid%252F21460563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1086/658903&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21460563&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000289237700002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.3" data-doi="10.1111/j.1752-4571.2011.00213.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bonduriansky</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. J.</span>
          <span class="cit-name-surname">Crean</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Day</span>
         </span>. 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">The implications of nongenetic inheritance for evolution in changing environments</span>. 
         <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>
         <span class="cit-vol">5</span>: 
         <span class="cit-fpage">192</span>–
         <span class="cit-lpage">201</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D5%26rft.spage%253D192%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2011.00213.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2011.00213.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000300875400009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.4" data-doi="10.1016/S0065-2504(09)00405-X">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bonenfant</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-M.</span>
          <span class="cit-name-surname">Gaillard</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Festa-Bianchet</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Loison</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Garel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L. E.</span>
          <span class="cit-name-surname">Loe</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Blanchard</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Pettorelli</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Owen-Smith</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Empirical evidence of density-dependence in populations of large herbivores</span>. 
         <abbr class="cit-jnl-abbrev">Advances in Ecological Research</abbr>
         <span class="cit-vol">41</span>: 
         <span class="cit-fpage">313</span>–
         <span class="cit-lpage">357</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2BEcological%2BResearch%26rft.volume%253D41%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2504%252809%252900405-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0065-2504(09)00405-X&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000268499700005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.5" data-doi="10.1111/j.1461-0248.2007.01130.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bossdorf</span>, 
          <span class="cit-name-given-names">O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. L.</span>
          <span class="cit-name-surname">Richards</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Pigliucci</span>
         </span>. 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-article-title">Epigenetics for ecologists</span>. 
         <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>
         <span class="cit-vol">11</span>: 
         <span class="cit-fpage">106</span>–
         <span class="cit-lpage">115</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DBossdorf%26rft.auinit1%253DO.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D106%26rft.epage%253D115%26rft.atitle%253DEpigenetics%2Bfor%2Becologists.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2007.01130.x%26rft_id%253Dinfo%253Apmid%252F18021243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2007.01130.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18021243&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000252206800002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-book" id="cit-090894v6.6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Charlesworth</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>
         <span class="cit-pub-date">1994</span>. 
         <span class="cit-chapter-title">Evolution in Age-structured Populations</span>. 
         <span class="cit-publ-name">Cambridge University Press Cambridge</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.7">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chevin</span>, 
          <span class="cit-name-given-names">L.-M.</span>
         </span>
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Evolution of adult size depends on genetic variance in growth trajectories: a comment on analyses of evolutionary dynamics using integral projection models</span>. 
         <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>
         <span class="cit-vol">6</span>: 
         <span class="cit-fpage">981</span>–
         <span class="cit-lpage">986</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D6%26rft.spage%253D981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.8" data-doi="10.1371/journal.pbio.1000357">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chevin</span>, 
          <span class="cit-name-given-names">L.-M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Lande</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G. M.</span>
          <span class="cit-name-surname">Mace</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">Adaptation, plasticity, and extinction in a changing environment: towards a predictive theory</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Biology</abbr>
         <span class="cit-vol">8</span>: 
         <span class="cit-fpage">e1000357</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DChevin%26rft.auinit1%253DL.%2BM.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253De1000357%26rft.epage%253De1000357%26rft.atitle%253DAdaptation%252C%2Bplasticity%252C%2Band%2Bextinction%2Bin%2Ba%2Bchanging%2Benvironment%253A%2Btowards%2Ba%2Bpredictive%2Btheory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000357%26rft_id%253Dinfo%253Apmid%252F20463950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000357&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20463950&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Childs</span>, 
          <span class="cit-name-given-names">D. Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Rees</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K. E.</span>
          <span class="cit-name-surname">Rose</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.J.</span>
          <span class="cit-name-surname">Grubb</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. P.</span>
          <span class="cit-name-surname">Ellner</span>
         </span>. 
         <span class="cit-pub-date">2003a</span>. 
         <span class="cit-article-title">Evolution of complex flowering strategies: an age–and size–structured integral projection model</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society of London</abbr> B 
         <span class="cit-vol">270</span>: 
         <span class="cit-fpage">1829</span>–
         <span class="cit-lpage">1838</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BLondon%26rft.volume%253D270%26rft.spage%253D1829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.10" data-doi="10.1098/rspb.2003.2399">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Childs</span>, 
          <span class="cit-name-given-names">D. Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Rees</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K. E.</span>
          <span class="cit-name-surname">Rose</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.J.</span>
          <span class="cit-name-surname">Grubb</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. P.</span>
          <span class="cit-name-surname">Ellner</span>
         </span>. 
         <span class="cit-pub-date">2003b</span>. 
         <span class="cit-article-title">Evolution of complex flowering strategies: an age–and size–structured integral projection model</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society of London B: Biological Sciences</abbr>
         <span class="cit-vol">270</span>: 
         <span class="cit-fpage">1829</span>–
         <span class="cit-lpage">1838</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DChilds%26rft.auinit1%253DD.%2BZ.%26rft.volume%253D270%26rft.issue%253D1526%26rft.spage%253D1829%26rft.epage%253D1838%26rft.atitle%253DEvolution%2Bof%2Bcomplex%2Bflowering%2Bstrategies%253A%2Ban%2Bage-%2Band%2Bsize-structured%2Bintegral%2Bprojection%2Bmodel%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2003.2399%26rft_id%253Dinfo%253Apmid%252F12964986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1098/rspb.2003.2399&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12964986&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000185206500011&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Childs</span>, 
          <span class="cit-name-given-names">D. Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">B. C.</span>
          <span class="cit-name-surname">Sheldon</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Rees</span>
         </span>. 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-article-title">The evolution of labile traits in sex-and age-structured populations</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>
         <span class="cit-vol">85</span>: 
         <span class="cit-fpage">329</span>–
         <span class="cit-lpage">342</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D85%26rft.spage%253D329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.12" data-doi="10.1038/nature02177">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Coltman</span>, 
          <span class="cit-name-given-names">D. W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">O’Donoghue</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. T.</span>
          <span class="cit-name-surname">Jorgenson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. T.</span>
          <span class="cit-name-surname">Hogg</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Strobeck</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Festa-Bianchet</span>
         </span>. 
         <span class="cit-pub-date">2003</span>. 
         <span class="cit-article-title">Undesirable evolutionary consequences of trophy hunting</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">426</span>: 
         <span class="cit-fpage">655</span>–
         <span class="cit-lpage">658</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DColtman%26rft.auinit1%253DD.%2BW.%26rft.volume%253D426%26rft.issue%253D6967%26rft.spage%253D655%26rft.epage%253D658%26rft.atitle%253DUndesirable%2Bevolutionary%2Bconsequences%2Bof%2Btrophy%2Bhunting.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02177%26rft_id%253Dinfo%253Apmid%252F14668862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature02177&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14668862&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000187132800039&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.13" data-doi="10.1890/0012-9615(2001)071[0377:RAODRT]2.0.CO;2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Cooch</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. F.</span>
          <span class="cit-name-surname">Rockwell</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Brault</span>
         </span>. 
         <span class="cit-pub-date">2001</span>. 
         <span class="cit-article-title">Retrospective analysis of demographic responses to environmental change: a lesser snow goose example</span>. 
         <abbr class="cit-jnl-abbrev">Ecological Monographs</abbr>
         <span class="cit-vol">71</span>: 
         <span class="cit-fpage">377</span>–
         <span class="cit-lpage">400</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcological%2BMonographs%26rft.volume%253D71%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9615%25282001%2529071%255B0377%253ARAODRT%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1890/0012-9615(2001)071[0377:RAODRT]2.0.CO;2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000170305800003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.14" data-doi="10.1111/j.1600-0706.2012.00035.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Coulson</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">Integral projections models, their construction and use in posing hypotheses in ecology</span>. 
         <abbr class="cit-jnl-abbrev">Oikos</abbr>
         <span class="cit-vol">121</span>: 
         <span class="cit-fpage">1337</span>–
         <span class="cit-lpage">1350</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DOikos%26rft.volume%253D121%26rft.spage%253D1337%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1600-0706.2012.00035.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1600-0706.2012.00035.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000308091200002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.15" data-doi="10.1126/science.1209441">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Coulson</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. R.</span>
          <span class="cit-name-surname">MacNulty</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. R.</span>
          <span class="cit-name-surname">Stahler</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. K.</span>
          <span class="cit-name-surname">Wayne</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. W.</span>
          <span class="cit-name-surname">Smith</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Modeling effects of environmental change on wolf population dynamics, trait evolution, and life history</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">334</span>: 
         <span class="cit-fpage">1275</span>–
         <span class="cit-lpage">1278</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCoulson%26rft.auinit1%253DT.%26rft.volume%253D334%26rft.issue%253D6060%26rft.spage%253D1275%26rft.epage%253D1278%26rft.atitle%253DModeling%2BEffects%2Bof%2BEnvironmental%2BChange%2Bon%2BWolf%2BPopulation%2BDynamics%252C%2BTrait%2BEvolution%252C%2Band%2BLife%2BHistory%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209441%26rft_id%253Dinfo%253Apmid%252F22144626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzQvNjA2MC8xMjc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDMvMjQvMDkwODk0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.16" data-doi="10.1086/591693">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Coulson</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Tuljapurkar</span>
         </span>. 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-article-title">The dynamics of a quantitative trait in an age-structured population living in a variable environment</span>. 
         <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>
         <span class="cit-vol">172</span>: 
         <span class="cit-fpage">599</span>–
         <span class="cit-lpage">612</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DCoulson%26rft.auinit1%253DT.%26rft.volume%253D172%26rft.issue%253D5%26rft.spage%253D599%26rft.epage%253D612%26rft.atitle%253DThe%2Bdynamics%2Bof%2Ba%2Bquantitative%2Btrait%2Bin%2Ban%2Bage-structured%2Bpopulation%2Bliving%2Bin%2Ba%2Bvariable%2Benvironment.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F591693%26rft_id%253Dinfo%253Apmid%252F18840061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1086/591693&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18840061&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000260186000001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.17">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Coulson</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Tuljapurkar</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. Z.</span>
          <span class="cit-name-surname">Childs</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">Using evolutionary demography to link life history theory, quantitative genetics and population ecology</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>
         <span class="cit-vol">79</span>: 
         <span class="cit-fpage">1226</span>–
         <span class="cit-lpage">1240</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D79%26rft.spage%253D1226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.18" data-doi="10.1126/science.1200303">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Dawson</span>, 
          <span class="cit-name-given-names">T. P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. T.</span>
          <span class="cit-name-surname">Jackson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. I.</span>
          <span class="cit-name-surname">House</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">I. C.</span>
          <span class="cit-name-surname">Prentice</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">G. M.</span>
          <span class="cit-name-surname">Mace</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Beyond predictions: biodiversity conservation in a changing climate</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">332</span>: 
         <span class="cit-fpage">53</span>–
         <span class="cit-lpage">58</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDawson%26rft.auinit1%253DT.%2BP.%26rft.volume%253D332%26rft.issue%253D6025%26rft.spage%253D53%26rft.epage%253D58%26rft.atitle%253DBeyond%2BPredictions%253A%2BBiodiversity%2BConservation%2Bin%2Ba%2BChanging%2BClimate%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1200303%26rft_id%253Dinfo%253Apmid%252F21454781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzIvNjAyNS81MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE3LzAzLzI0LzA5MDg5NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Diaconis</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Ylvisaker</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">1979</span>. 
         <span class="cit-article-title">Conjugate priors for exponential families</span>. 
         <abbr class="cit-jnl-abbrev">The Annals of statistics</abbr>
         <span class="cit-vol">7</span>: 
         <span class="cit-fpage">269</span>–
         <span class="cit-lpage">281</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2Bstatistics%26rft.volume%253D7%26rft.spage%253D269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.20" data-doi="10.1006/jtbi.1997.0454">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Doebeli</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-pub-date">1997</span>. 
         <span class="cit-article-title">Genetic variation and persistence of predator-prey interactions in the nicholson–bailey model</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Theoretical Biology</abbr>
         <span class="cit-vol">188</span>: 
         <span class="cit-fpage">109</span>–
         <span class="cit-lpage">120</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.volume%253D188%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjtbi.1997.0454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1006/jtbi.1997.0454&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1997XV48100010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.21" data-doi="10.1890/0012-9658(2000)081[0694:SSSAAN]2.0.CO;2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Easterling</span>, 
          <span class="cit-name-given-names">M. R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. P.</span>
          <span class="cit-name-surname">Ellner</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. M.</span>
          <span class="cit-name-surname">Dixon</span>
         </span>. 
         <span class="cit-pub-date">2000</span>. 
         <span class="cit-article-title">Size-specific sensitivity: applying a new structured population model</span>. 
         <abbr class="cit-jnl-abbrev">Ecology</abbr>
         <span class="cit-vol">81</span>: 
         <span class="cit-fpage">694</span>–
         <span class="cit-lpage">708</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D81%26rft.spage%253D694%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9658%25282000%2529081%255B0694%253ASSSAAN%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1890/0012-9658(2000)081[0694:SSSAAN]2.0.CO;2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000085611400009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.22" data-doi="10.1086/499438">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ellner</span>, 
          <span class="cit-name-given-names">S. P.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Rees</span>
         </span>. 
         <span class="cit-pub-date">2006</span>. 
         <span class="cit-article-title">Integral projection models for species with complex demography</span>. 
         <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>
         <span class="cit-vol">167</span>: 
         <span class="cit-fpage">410</span>–
         <span class="cit-lpage">428</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DEllner%26rft.auinit1%253DS.%2BP.%26rft.volume%253D167%26rft.issue%253D3%26rft.spage%253D410%26rft.epage%253D428%26rft.atitle%253DIntegral%2Bprojection%2Bmodels%2Bfor%2Bspecies%2Bwith%2Bcomplex%2Bdemography.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F499438%26rft_id%253Dinfo%253Apmid%252F16673349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1086/499438&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16673349&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000236143500010&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-book" id="cit-090894v6.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Falconer</span>, 
          <span class="cit-name-given-names">D. S.</span>
         </span>
         <span class="cit-pub-date">1960</span>. 
         <span class="cit-chapter-title">Introduction to Quantitative Genetics</span>. 
         <span class="cit-publ-name">Ronald Press co</span>. 
         <span class="cit-publ-loc">NY</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.24" data-doi="10.1016/0040-5809(79)90043-1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Feldman</span>, 
          <span class="cit-name-given-names">M. W.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">L. L.</span>
          <span class="cit-name-surname">Cavalli-Sforza</span>
         </span>. 
         <span class="cit-pub-date">1979</span>. 
         <span class="cit-article-title">Aspects of variance and covariance analysis with cultural inheritance</span>. 
         <abbr class="cit-jnl-abbrev">Theoretical population biology</abbr>
         <span class="cit-vol">15</span>: 
         <span class="cit-fpage">276</span>–
         <span class="cit-lpage">307</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DFeldman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D276%26rft.epage%253D307%26rft.atitle%253DAspects%2Bof%2Bvariance%2Band%2Bcovariance%2Banalysis%2Bwith%2Bcultural%2Binheritance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252879%252990043-1%26rft_id%253Dinfo%253Apmid%252F515973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/0040-5809(79)90043-1&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=515973&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1979HN07600001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.25" data-doi="10.1016/0040-5809(81)90025-3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Felsenstein</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>
         <span class="cit-pub-date">1981</span>. 
         <span class="cit-article-title">Continuous-genotype models and assortative mating</span>. 
         <abbr class="cit-jnl-abbrev">Theoretical Population Biology</abbr>
         <span class="cit-vol">19</span>: 
         <span class="cit-fpage">341</span>–
         <span class="cit-lpage">357</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTheoretical%2BPopulation%2BBiology%26rft.volume%253D19%26rft.spage%253D341%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252881%252990025-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/0040-5809(81)90025-3&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1981LY62900004&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-book" id="cit-090894v6.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fisher</span>, 
          <span class="cit-name-given-names">R. A.</span>
         </span>
         <span class="cit-pub-date">1930</span>. 
         <span class="cit-chapter-title">The genetical theory of natural selection</span>. 
         <span class="cit-publ-name">Oxford University Press</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.27" data-doi="10.1111/j.1474-919X.1981.tb00170.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Garnett</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>
         <span class="cit-pub-date">1981</span>. 
         <span class="cit-article-title">Body size, its heritability and influence on juvenile survival among great tits, parus major</span>. 
         <abbr class="cit-jnl-abbrev">Ibis</abbr>
         <span class="cit-vol">123</span>: 
         <span class="cit-fpage">31</span>–
         <span class="cit-lpage">41</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIbis%26rft.volume%253D123%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1474-919X.1981.tb00170.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1474-919X.1981.tb00170.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.28" data-doi="10.1046/j.1420-9101.1993.6010031.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gavrilets</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. M.</span>
          <span class="cit-name-surname">Scheiner</span>
         </span>. 
         <span class="cit-pub-date">1993</span>. 
         <span class="cit-article-title">The genetics of phenotypic plasticity. v. evolution of reaction norm shape</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr>
         <span class="cit-vol">6</span>: 
         <span class="cit-fpage">31</span>–
         <span class="cit-lpage">48</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEvolutionary%2BBiology%26rft.volume%253D6%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1420-9101.1993.6010031.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1046/j.1420-9101.1993.6010031.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.29" data-doi="10.1038/nature15256">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ghalambor</span>, 
          <span class="cit-name-given-names">C. K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K. L.</span>
          <span class="cit-name-surname">Hoke</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E. W.</span>
          <span class="cit-name-surname">Ruell</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E. K.</span>
          <span class="cit-name-surname">Fischer</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. N.</span>
          <span class="cit-name-surname">Reznick</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">K. A.</span>
          <span class="cit-name-surname">Hughes</span>
         </span>.
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Non-adaptive plasticity potentiates rapid adaptive evolution of gene expression in nature</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">525</span>: 
         <span class="cit-fpage">372</span>–
         <span class="cit-lpage">375</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D525%26rft.spage%253D372%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature15256%26rft_id%253Dinfo%253Apmid%252F26331546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature15256&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26331546&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.30" data-doi="10.1111/j.1365-2435.2007.01283.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ghalambor</span>, 
          <span class="cit-name-given-names">C. K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. K.</span>
          <span class="cit-name-surname">McKay</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. P.</span>
          <span class="cit-name-surname">Carroll</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. N.</span>
          <span class="cit-name-surname">Reznick</span>
         </span>. 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-article-title">Adaptive versus non-adaptive phenotypic plasticity and the potential for contemporary adaptation in new environments</span>. 
         <abbr class="cit-jnl-abbrev">Functional Ecology</abbr>
         <span class="cit-vol">21</span>: 
         <span class="cit-fpage">394</span>–
         <span class="cit-lpage">407</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFunctional%2BEcology%26rft.volume%253D21%26rft.spage%253D394%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2435.2007.01283.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1365-2435.2007.01283.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000246708200002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.31" data-doi="10.1111/j.1365-294X.2007.03413.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gienapp</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Teplitsky</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Alho</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Mills</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Merilä</span>
         </span>. 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-article-title">Climate change and evolution: disentangling environmental and genetic responses</span>. 
         <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>
         <span class="cit-vol">17</span>: 
         <span class="cit-fpage">167</span>–
         <span class="cit-lpage">178</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGienapp%26rft.auinit1%253DP.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D167%26rft.epage%253D178%26rft.atitle%253DClimate%2Bchange%2Band%2Bevolution%253A%2Bdisentangling%2Benvironmental%2Band%2Bgenetic%2Bresponses.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03413.x%26rft_id%253Dinfo%253Apmid%252F18173499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03413.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18173499&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000251740500014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gilbert</span>, 
          <span class="cit-name-given-names">S. F.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Epel</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Ecological developmental biology: integrating epigenetics, medicine, and evolution</span>. 
         <abbr class="cit-jnl-abbrev">Sinauer Associates</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.33" data-doi="10.1038/nature09670">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hoffmann</span>, 
          <span class="cit-name-given-names">A. A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. M.</span>
          <span class="cit-name-surname">Sgrò</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Climate change and evolutionary adaptation</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">470</span>: 
         <span class="cit-fpage">479</span>–
         <span class="cit-lpage">485</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D470%26rft.issue%253D7335%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DClimate%2Bchange%2Band%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09670%26rft_id%253Dinfo%253Apmid%252F21350480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature09670&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21350480&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000287652900031&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.34">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ives</span>, 
          <span class="cit-name-given-names">A. R.</span>
         </span>
         <span class="cit-pub-date">1995</span>. 
         <span class="cit-article-title">Predicting the response of populations to environmental change</span>. 
         <abbr class="cit-jnl-abbrev">Ecology</abbr>
         <span class="cit-vol">73</span>: 
         <span class="cit-fpage">926</span>–
         <span class="cit-lpage">941</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D73%26rft.spage%253D926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.35" data-doi="10.1016/S0022-5193(89)80058-X">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Jablonka</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. J.</span>
          <span class="cit-name-surname">Lamb</span>
         </span>. 
         <span class="cit-pub-date">1989</span>. 
         <span class="cit-article-title">The inheritance of acquired epigenetic variations</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Theoretical Biology</abbr>
         <span class="cit-vol">139</span>: 
         <span class="cit-fpage">69</span>–
         <span class="cit-lpage">83</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DJablonka%26rft.auinit1%253DE.%26rft.volume%253D139%26rft.issue%253D1%26rft.spage%253D69%26rft.epage%253D83%26rft.atitle%253DThe%2Binheritance%2Bof%2Bacquired%2Bepigenetic%2Bvariations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-5193%252889%252980058-X%26rft_id%253Dinfo%253Apmid%252F2593687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0022-5193(89)80058-X&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=2593687&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1989AG62100005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Janeiro</span>, 
          <span class="cit-name-given-names">M. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Festa-Bianchet</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F.</span>
          <span class="cit-name-surname">Pelletier</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. W.</span>
          <span class="cit-name-surname">Coltman</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. B.</span>
          <span class="cit-name-surname">Morrissey</span>
         </span>. in press. 
         <span class="cit-article-title">Towards robust evolutionary inference with integral projection models</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-book" id="cit-090894v6.37">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kendall</span>, 
          <span class="cit-name-given-names">B. E.</span>
         </span>
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-chapter-title">A statistical symphony: Instrumental variables reveal causality and control measurement error</span>. Pages 
         <span class="cit-fpage">149</span>–
         <span class="cit-lpage">167</span> in 
         <span class="cit-auth">
          <span class="cit-name-given-names">G. A.</span>
          <span class="cit-name-surname">Fox</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Negrete-Yankelevich</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">V. J.</span>
          <span class="cit-name-surname">Sosa</span>
         </span>, eds. 
         <span class="cit-source">Ecological Statistics: Contemporary Theory and Application</span>. 
         <span class="cit-publ-name">Oxford University Press</span>, 
         <span class="cit-publ-loc">Oxford, UK</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.38" data-doi="10.1086/650724">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kendall</span>, 
          <span class="cit-name-given-names">B. E.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. E.</span>
          <span class="cit-name-surname">Wittmann</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">A stochastic model for annual reproductive success</span>. 
         <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>
         <span class="cit-vol">175</span>: 
         <span class="cit-fpage">461</span>–
         <span class="cit-lpage">468</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DKendall%26rft.auinit1%253DB.%2BE.%26rft.volume%253D175%26rft.issue%253D4%26rft.spage%253D461%26rft.epage%253D468%26rft.atitle%253DA%2Bstochastic%2Bmodel%2Bfor%2Bannual%2Breproductive%2Bsuccess.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F650724%26rft_id%253Dinfo%253Apmid%252F20163244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1086/650724&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20163244&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.39" data-doi="10.1086/319193">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kingsolver</span>, 
          <span class="cit-name-given-names">J. G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">H. E.</span>
          <span class="cit-name-surname">Hoekstra</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. M.</span>
          <span class="cit-name-surname">Hoekstra</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Berrigan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. N.</span>
          <span class="cit-name-surname">Vignieri</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Hill</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Hoang</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Gibert</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Beerli</span>
         </span>. 
         <span class="cit-pub-date">2001</span>. 
         <span class="cit-article-title">The strength of phenotypic selection in natural populations</span>. 
         <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>
         <span class="cit-vol">157</span>: 
         <span class="cit-fpage">245</span>–
         <span class="cit-lpage">261</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DKingsolver%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D157%26rft.issue%253D3%26rft.spage%253D245%26rft.epage%253D261%26rft.atitle%253DThe%2Bstrength%2Bof%2Bphenotypic%2Bselection%2Bin%2Bnatural%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F319193%26rft_id%253Dinfo%253Apmid%252F18707288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1086/319193&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18707288&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000167301000001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.40" data-doi="10.1146/annurev.ecolsys.39.110707.173542">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kruuk</span>, 
          <span class="cit-name-given-names">L. E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Slate</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">A. J.</span>
          <span class="cit-name-surname">Wilson</span>
         </span>. 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-article-title">New answers for old questions: the evolutionary quantitative genetics of wild animal populations</span>. 
         <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>
         <span class="cit-vol">39</span>: 
         <span class="cit-fpage">525</span>–
         <span class="cit-lpage">548</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D39%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173542&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.41" data-doi="10.2307/2407630">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lande</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>
         <span class="cit-pub-date">1979</span>. 
         <span class="cit-article-title">Quantitative genetic analysis of multivariate evolution, applied to brain: body size allometry</span>. 
         <abbr class="cit-jnl-abbrev">Evolution</abbr>
         <span class="cit-vol">33</span>: 
         <span class="cit-fpage">402</span>–
         <span class="cit-lpage">416</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D33%26rft.spage%253D402%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2407630%26rft_id%253Dinfo%253Apmid%252F28568194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2307/2407630&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=28568194&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1979GV98300014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.42" data-doi="10.2307/1936778">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lande</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>. 
         <span class="cit-pub-date">1982</span>. 
         <span class="cit-article-title">A quantitative genetic theory of life history evolution</span>. 
         <abbr class="cit-jnl-abbrev">Ecology</abbr>
         <span class="cit-vol">63</span>: 
         <span class="cit-fpage">607</span>–
         <span class="cit-lpage">615</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D63%26rft.spage%253D607%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1936778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2307/1936778&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1982NV13400001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.43" data-doi="10.1111/j.1420-9101.2009.01754.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lande</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Adaptation to an extraordinary environment by evolution of phenotypic plasticity and genetic assimilation</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr>
         <span class="cit-vol">22</span>: 
         <span class="cit-fpage">1435</span>–
         <span class="cit-lpage">1446</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DLande%26rft.auinit1%253DR.%26rft.volume%253D22%26rft.issue%253D7%26rft.spage%253D1435%26rft.epage%253D1446%26rft.atitle%253DAdaptation%2Bto%2Ban%2Bextraordinary%2Benvironment%2Bby%2Bevolution%2Bof%2Bphenotypic%2Bplasticity%2Band%2Bgenetic%2Bassimilation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2009.01754.x%26rft_id%253Dinfo%253Apmid%252F19467134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2009.01754.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19467134&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000267129500008&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-book no-rev-xref" id="cit-090894v6.44">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lande</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Engen</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.-E.</span>
          <span class="cit-name-surname">Saether</span>
         </span>. 
         <span class="cit-pub-date">2003</span>. 
         <span class="cit-chapter-title">Stochastic population dynamics in ecology and conservation</span>. 
         <span class="cit-publ-name">Oxford University Press</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.45" data-doi="10.1146/annurev-ecolsys-102209-144628">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lavergne</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Mouquet</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">W.</span>
          <span class="cit-name-surname">Thuiller</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">O.</span>
          <span class="cit-name-surname">Ronce</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">Biodiversity and climate change: integrating evolutionary and ecological responses of species and communities</span>. 
         <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution and Systematics</abbr>
         <span class="cit-vol">41</span>: 
         <span class="cit-fpage">321</span>–
         <span class="cit-lpage">350</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%2Band%2BSystematics%26rft.volume%253D41%26rft.spage%253D321%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-102209-144628%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-102209-144628&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000284978200016&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.46" data-doi="10.1038/nrg2398">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">López-Maury</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Marguerat</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Bähler</span>
         </span>. 
         <span class="cit-pub-date">2008</span>. 
         <span class="cit-article-title">Tuning gene expression to changing environments: from rapid responses to evolutionary adaptation</span>. 
         <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr>
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">583</span>–
         <span class="cit-lpage">593</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLopez-Maury%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D583%26rft.epage%253D593%26rft.atitle%253DTuning%2Bgene%2Bexpression%2Bto%2Bchanging%2Benvironments%253A%2Bfrom%2Brapid%2Bresponses%2Bto%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2398%26rft_id%253Dinfo%253Apmid%252F18591982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nrg2398&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18591982&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000257758400009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-book" id="cit-090894v6.47">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lynch</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Walsh</span>
         </span>. 
         <span class="cit-pub-date">1998</span>. 
         <span class="cit-chapter-title">Genetics and Analysis of Quantitative Traits</span>, vol. 
         <span class="cit-vol">1</span>. 
         <span class="cit-publ-name">Sinauer</span>
         <span class="cit-publ-loc">Sunderland, MA</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.48" data-doi="10.1038/35083580">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Merilä</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Kruuk</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">B.</span>
          <span class="cit-name-surname">Sheldon</span>
         </span>. 
         <span class="cit-pub-date">2001</span>. 
         <span class="cit-article-title">Cryptic evolution in a wild bird population</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">412</span>: 
         <span class="cit-fpage">76</span>–
         <span class="cit-lpage">79</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMeril%25C3%25A4%26rft.auinit1%253DJ.%26rft.volume%253D412%26rft.issue%253D6842%26rft.spage%253D76%26rft.epage%253D79%26rft.atitle%253DCryptic%2Bevolution%2Bin%2Ba%2Bwild%2Bbird%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35083580%26rft_id%253Dinfo%253Apmid%252F11452309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/35083580&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11452309&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference in text" id="ref-49">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.49">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Merow</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. P.</span>
          <span class="cit-name-surname">Dahlgren</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. J. E.</span>
          <span class="cit-name-surname">Metcalf</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. Z.</span>
          <span class="cit-name-surname">Childs</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. E.</span>
          <span class="cit-name-surname">Evans</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Jongejans</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Record</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M.</span>
          <span class="cit-name-surname">Rees</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R.</span>
          <span class="cit-name-surname">Salguero-Gómez</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. M.</span>
          <span class="cit-name-surname">McMahon</span>
         </span>. 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Advancing population ecology with integral projection models: a practical guide</span>. 
         <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>
         <span class="cit-vol">5</span>: 
         <span class="cit-fpage">99</span>–
         <span class="cit-lpage">110</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D5%26rft.spage%253D99%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.50">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Metz</span>, 
          <span class="cit-name-given-names">J. A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. A.</span>
          <span class="cit-name-surname">Geritz</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G.</span>
          <span class="cit-name-surname">Meszéna</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F. J.</span>
          <span class="cit-name-surname">Jacobs</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. S.</span>
          <span class="cit-name-surname">Van Heerwaarden</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">1996</span>. 
         <span class="cit-article-title">Adaptive dynamics, a geometrical study of the consequences of nearly faithful reproduction</span>. 
         <abbr class="cit-jnl-abbrev">Stochastic and spatial structures of dynamical systems</abbr>
         <span class="cit-vol">45</span>: 
         <span class="cit-fpage">183</span>–
         <span class="cit-lpage">231</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DStochastic%2Band%2Bspatial%2Bstructures%2Bof%2Bdynamical%2Bsystems%26rft.volume%253D45%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.51" data-doi="10.1016/j.tree.2005.08.002">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Miner</span>, 
          <span class="cit-name-given-names">B. G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. E.</span>
          <span class="cit-name-surname">Sultan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. G.</span>
          <span class="cit-name-surname">Morgan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. K.</span>
          <span class="cit-name-surname">Padilla</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. A.</span>
          <span class="cit-name-surname">Relyea</span>
         </span>. 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-article-title">Ecological consequences of phenotypic plasticity</span>. 
         <abbr class="cit-jnl-abbrev">Trends in Ecology and Evolution</abbr>
         <span class="cit-vol">20</span>: 
         <span class="cit-fpage">685</span>–
         <span class="cit-lpage">692</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D20%26rft.spage%253D685%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2005.08.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.tree.2005.08.002&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000233878700011&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.52" data-doi="10.1016/S0169-5347(98)01472-4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mousseau</span>, 
          <span class="cit-name-given-names">T. A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. W.</span>
          <span class="cit-name-surname">Fox</span>
         </span>. 
         <span class="cit-pub-date">1998</span>. 
         <span class="cit-article-title">The adaptive significance of maternal effects</span>. 
         <abbr class="cit-jnl-abbrev">Trends in Ecology &amp;; Evolution</abbr>
         <span class="cit-vol">13</span>: 
         <span class="cit-fpage">403</span>–
         <span class="cit-lpage">407</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D13%26rft.spage%253D403%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252898%252901472-4%26rft_id%253Dinfo%253Apmid%252F21238360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0169-5347(98)01472-4&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21238360&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000075997500009&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference in text" id="ref-53">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.53" data-doi="10.1111/j.1420-9101.2007.01300.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nussey</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Wilson</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Brommer</span>
         </span>. 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-article-title">The evolutionary ecology of individual phenotypic plasticity in wild populations</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr>
         <span class="cit-vol">20</span>: 
         <span class="cit-fpage">831</span>–
         <span class="cit-lpage">844</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bevolutionary%2Bbiology%26rft.stitle%253DJ%2BEvol%2BBiol%26rft.aulast%253DNussey%26rft.auinit1%253DD.%2BH.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D831%26rft.epage%253D844%26rft.atitle%253DThe%2Bevolutionary%2Becology%2Bof%2Bindividual%2Bphenotypic%2Bplasticity%2Bin%2Bwild%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1420-9101.2007.01300.x%26rft_id%253Dinfo%253Apmid%252F17465894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1420-9101.2007.01300.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17465894&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000245989600001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.54" data-doi="10.1111/j.1365-2656.2005.00941.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Nussey</span>, 
          <span class="cit-name-given-names">D. H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. H.</span>
          <span class="cit-name-surname">Clutton-Brock</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. A.</span>
          <span class="cit-name-surname">Elston</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. D.</span>
          <span class="cit-name-surname">Albon</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">L. E.</span>
          <span class="cit-name-surname">Kruuk</span>
         </span>. 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-article-title">Phenotypic plasticity in a maternal trait in red deer</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>
         <span class="cit-vol">74</span>: 
         <span class="cit-fpage">387</span>–
         <span class="cit-lpage">396</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D74%26rft.spage%253D387%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2656.2005.00941.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1365-2656.2005.00941.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000227593600020&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference in text" id="ref-55">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.55" data-doi="10.1038/nature09210">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ozgul</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. Z.</span>
          <span class="cit-name-surname">Childs</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. K.</span>
          <span class="cit-name-surname">Oli</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">K. B.</span>
          <span class="cit-name-surname">Armitage</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. T.</span>
          <span class="cit-name-surname">Blumstein</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L. E.</span>
          <span class="cit-name-surname">Olson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Tuljapurkar</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">Coupled dynamics of body mass and population growth in response to environmental change</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">466</span>: 
         <span class="cit-fpage">482</span>–
         <span class="cit-lpage">485</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOzgul%26rft.auinit1%253DA.%26rft.volume%253D466%26rft.issue%253D7305%26rft.spage%253D482%26rft.epage%253D485%26rft.atitle%253DCoupled%2Bdynamics%2Bof%2Bbody%2Bmass%2Band%2Bpopulation%2Bgrowth%2Bin%2Bresponse%2Bto%2Benvironmental%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09210%26rft_id%253Dinfo%253Apmid%252F20651690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature09210&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20651690&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000280141200034&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference in text" id="ref-56">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.56" data-doi="10.1126/science.1173668">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ozgul</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Tuljapurkar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. G.</span>
          <span class="cit-name-surname">Benton</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. M.</span>
          <span class="cit-name-surname">Pemberton</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T. H.</span>
          <span class="cit-name-surname">Clutton-Brock</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">The dynamics of phenotypic change and the shrinking sheep of st. kilda</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">325</span>: 
         <span class="cit-fpage">464</span>–
         <span class="cit-lpage">467</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1173668%26rft_id%253Dinfo%253Apmid%252F19574350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTkzOS80NjQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxNy8wMy8yNC8wOTA4OTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference in text" id="ref-57">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.57">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pelletier</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">A new metric to calculate the opportunity for selection on quantitative characters</span>. 
         <abbr class="cit-jnl-abbrev">Evolutionary Ecology Research</abbr>
         <span class="cit-vol">14</span>: 
         <span class="cit-fpage">729</span>–
         <span class="cit-lpage">742</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BEcology%2BResearch%26rft.volume%253D14%26rft.spage%253D729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference in text" id="ref-58">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.58" data-doi="10.1098/rspb.2010.0771">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Reed</span>, 
          <span class="cit-name-given-names">T. E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. S.</span>
          <span class="cit-name-surname">Waples</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. E.</span>
          <span class="cit-name-surname">Schindler</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. J.</span>
          <span class="cit-name-surname">Hard</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. T.</span>
          <span class="cit-name-surname">Kinnison</span>
         </span>. 
         <span class="cit-pub-date">2010</span>. 
         <span class="cit-article-title">Phenotypic plasticity and population viability: the importance of environmental predictability</span>. 
         <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society of London B: Biological Sciences</abbr>
         <span class="cit-vol">277</span>: 
         <span class="cit-fpage">3391</span>–
         <span class="cit-lpage">3400</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BLondon%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2010.0771%26rft_id%253Dinfo%253Apmid%252F20554553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1098/rspb.2010.0771&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20554553&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000283448800003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference in text" id="ref-59">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.59">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rees</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. Z.</span>
          <span class="cit-name-surname">Childs</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. P.</span>
          <span class="cit-name-surname">Ellner</span>
         </span>. 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Building integral projection models: a user’s guide</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>
         <span class="cit-vol">83</span>: 
         <span class="cit-fpage">528</span>–
         <span class="cit-lpage">545</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D83%26rft.spage%253D528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference in text" id="ref-60">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.60" data-doi="10.1890/08-1474.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rees</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. P.</span>
          <span class="cit-name-surname">Ellner</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Integral projection models for populations in temporally varying environments</span>. 
         <abbr class="cit-jnl-abbrev">Ecological Monographs</abbr>
         <span class="cit-vol">79</span>: 
         <span class="cit-fpage">575</span>–
         <span class="cit-lpage">594</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcological%2BMonographs%26rft.volume%253D79%26rft.spage%253D575%26rft_id%253Dinfo%253Adoi%252F10.1890%252F08-1474.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1890/08-1474.1&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000270909900003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.61">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rees</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. P.</span>
          <span class="cit-name-surname">Ellner</span>
         </span>. 
         <span class="cit-pub-date">2016</span>. 
         <span class="cit-article-title">Evolving integral projection models: evolutionary demography meets eco-evolutionary dynamics</span>. 
         <abbr class="cit-jnl-abbrev">Methods in Ecology and Evolution</abbr>
         <span class="cit-vol">7</span>: 
         <span class="cit-fpage">157</span>–
         <span class="cit-lpage">170</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D7%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference in text" id="ref-62">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.62" data-doi="10.1126/science.275.5308.1934">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Reznick</span>, 
          <span class="cit-name-given-names">D. N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F. H.</span>
          <span class="cit-name-surname">Shaw</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">F. H.</span>
          <span class="cit-name-surname">Rodd</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. G.</span>
          <span class="cit-name-surname">Shaw</span>
         </span>. 
         <span class="cit-pub-date">1997</span>. 
         <span class="cit-article-title">Evaluation of the rate of evolution in natural populations of guppies (Poecilia reticulata)</span>. 
         <abbr class="cit-jnl-abbrev">Science</abbr>
         <span class="cit-vol">275</span>: 
         <span class="cit-fpage">1934</span>–
         <span class="cit-lpage">1937</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DReznick%26rft.auinit1%253DD.%2BN.%26rft.volume%253D275%26rft.issue%253D5308%26rft.spage%253D1934%26rft.epage%253D1937%26rft.atitle%253DEvaluation%2Bof%2Bthe%2BRate%2Bof%2BEvolution%2Bin%2BNatural%2BPopulations%2Bof%2BGuppies%2B%2528Poecilia%2Breticulata%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.275.5308.1934%26rft_id%253Dinfo%253Apmid%252F9072971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzUvNTMwOC8xOTM0IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDMvMjQvMDkwODk0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.63" data-doi="10.1016/j.tree.2007.06.009">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sax</span>, 
          <span class="cit-name-given-names">D. F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. J.</span>
          <span class="cit-name-surname">Stachowicz</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. H.</span>
          <span class="cit-name-surname">Brown</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J. F.</span>
          <span class="cit-name-surname">Bruno</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. N.</span>
          <span class="cit-name-surname">Dawson</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. D.</span>
          <span class="cit-name-surname">Gaines</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. K.</span>
          <span class="cit-name-surname">Grosberg</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Hastings</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">R. D.</span>
          <span class="cit-name-surname">Holt</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. M.</span>
          <span class="cit-name-surname">Mayfield</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-article-title">Ecological and evolutionary insights from species invasions</span>. 
         <abbr class="cit-jnl-abbrev">Trends in Ecology and Evolution</abbr>
         <span class="cit-vol">22</span>: 
         <span class="cit-fpage">465</span>–
         <span class="cit-lpage">471</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D22%26rft.spage%253D465%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2007.06.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.tree.2007.06.009&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000249249100007&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.64" data-doi="10.1146/annurev.es.24.110193.000343">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Scheiner</span>, 
          <span class="cit-name-given-names">S. M.</span>
         </span>
         <span class="cit-pub-date">1993</span>. 
         <span class="cit-article-title">Genetics and evolution of phenotypic plasticity</span>. 
         <abbr class="cit-jnl-abbrev">Annual Review of Ecology and Systematics</abbr>
         <span class="cit-vol">24</span>: 
         <span class="cit-fpage">35</span>–
         <span class="cit-lpage">68</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DScheiner%26rft.auinit1%253DS.%2BM.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D68%26rft.atitle%253DGenetics%2Band%2BEvolution%2Bof%2BPhenotypic%2BPlasticity%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.24.110193.000343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1146/annurev.es.24.110193.000343&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1993MJ37100002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference in text" id="ref-65">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.65" data-doi="10.1038/nature14968">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schindler</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.</span>
          <span class="cit-name-surname">Gaillard</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Grüning</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Neuhaus</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L.</span>
          <span class="cit-name-surname">Traill</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Tuljapurkar</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Sex-specific demography and generalization of the Trivers-Willard theory</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">526</span>: 
         <span class="cit-fpage">249</span>–
         <span class="cit-lpage">252</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D526%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14968%26rft_id%253Dinfo%253Apmid%252F26390152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature14968&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26390152&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.66">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schmalhausen</span>, 
          <span class="cit-name-given-names">I. I.</span>
         </span>
         <span class="cit-pub-date">1949</span>. 
         <span class="cit-article-title">Factors of evolution: the theory of stabilizing selection</span>. 
         <abbr class="cit-jnl-abbrev">Blakiston</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.67" data-doi="10.1111/j.1752- 4571.2010.00166.">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sih</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. C.</span>
          <span class="cit-name-surname">Ferrari</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. J.</span>
          <span class="cit-name-surname">Harris</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">Evolution and behavioural responses to human-induced rapid environmental change</span>. 
         <abbr class="cit-jnl-abbrev">Evol App</abbr>
         <span class="cit-vol">4</span>: 
         <span class="cit-fpage">367</span>–
         <span class="cit-lpage">387</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEvol%2BApp%26rft.volume%253D4%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-%2B4571.2010.00166.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1752- 4571.2010.00166.&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.68">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Smallegange</span>, 
          <span class="cit-name-given-names">I. M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2011</span>. 
         <span class="cit-article-title">The stochastic demography of two coexisting male morphs</span>. 
         <abbr class="cit-jnl-abbrev">Ecology</abbr>
         <span class="cit-vol">92</span>: 
         <span class="cit-fpage">755</span>–
         <span class="cit-lpage">764</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DSmallegange%26rft.auinit1%253DI.%2BM.%26rft.volume%253D92%26rft.issue%253D3%26rft.spage%253D755%26rft.epage%253D764%26rft.atitle%253DThe%2Bstochastic%2Bdemography%2Bof%2Btwo%2Bcoexisting%2Bmale%2Bmorphs.%26rft_id%253Dinfo%253Apmid%252F21608483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=21608483&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.69">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Söderström</span>, 
          <span class="cit-name-given-names">T. D.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">P. G.</span>
          <span class="cit-name-surname">Stoica</span>
         </span>. 
         <span class="cit-pub-date">2002</span>. 
         <span class="cit-article-title">Instrumental variable methods for system identification</span>, vol. 
         <span class="cit-vol">21</span>. 
         <abbr class="cit-jnl-abbrev">Springer</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference in text" id="ref-70">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.70">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Steiner</span>, 
          <span class="cit-name-given-names">U. K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Tuljapurkar</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2014</span>. 
         <span class="cit-article-title">Generation time, net reproductive rate, and growth in stage-age-structured populations</span>. 
         <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>
         <span class="cit-vol">183</span>: 
         <span class="cit-fpage">771</span>–
         <span class="cit-lpage">783</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D183%26rft.spage%253D771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference in text" id="ref-71">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.71">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Steiner</span>, 
          <span class="cit-name-given-names">U. K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Tuljapurkar</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Horvitz</span>
         </span>. 
         <span class="cit-pub-date">2012</span>. 
         <span class="cit-article-title">Trading stages: life expectancies in structured populations</span>. 
         <abbr class="cit-jnl-abbrev">Experimental gerontology</abbr>
         <span class="cit-vol">47</span>: 
         <span class="cit-fpage">773</span>–
         <span class="cit-lpage">781</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DExperimental%2Bgerontology%26rft.stitle%253DExp%2BGerontol%26rft.aulast%253DSteiner%26rft.auinit1%253DU.%2BK.%26rft.volume%253D47%26rft.issue%253D10%26rft.spage%253D773%26rft.epage%253D781%26rft.atitle%253DTrading%2Bstages%253A%2Blife%2Bexpectancies%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F22664576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=22664576&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference in text" id="ref-72">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-090894v6.72">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Traill</span>, 
          <span class="cit-name-given-names">L. W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Schindler</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2014a</span>. 
         <span class="cit-article-title">Demography, not inheritance, drives phenotypic change in hunted bighorn sheep</span>. 
         <span class="cit-conf-name">Proceedings of the National Academy of Sciences</span>
         <span class="cit-vol">111</span>: 
         <span class="cit-fpage">13223</span>–
         <span class="cit-lpage">13228</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference in text" id="ref-73">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-090894v6.73">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Traill</span>, 
          <span class="cit-name-given-names">L. W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S.</span>
          <span class="cit-name-surname">Schindler</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2014b</span>. 
         <span class="cit-auth cit-collab">Reply to hedrick</span>
         <span class="cit-etal">et al</span>: 
         <span class="cit-article-title">Trophy hunting influences the distribution of trait values through demographic impacts</span>. 
         <span class="cit-conf-name">Proceedings of the National Academy of Sciences</span>
         <span class="cit-vol">111</span>:
         <span class="cit-fpage">E4811</span>–
         <span class="cit-lpage">E4811</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.74">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tuljapurkar</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>
         <span class="cit-pub-date">1990</span>. 
         <span class="cit-article-title">Population dynamics in variable environments</span>. 
         <abbr class="cit-jnl-abbrev">Springer</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.75" data-doi="10.1098/rstb.2009.0021">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Tuljapurkar</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">J.-M.</span>
          <span class="cit-name-surname">Gaillard</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">From stochastic environments to life histories and back</span>. 
         <abbr class="cit-jnl-abbrev">Philosophical Transactions of the Royal Society B</abbr>
         <span class="cit-vol">364</span>: 
         <span class="cit-fpage">1499</span>–
         <span class="cit-lpage">1509</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DTuljapurkar%26rft.auinit1%253DS.%26rft.volume%253D364%26rft.issue%253D1523%26rft.spage%253D1499%26rft.epage%253D1509%26rft.atitle%253DFrom%2Bstochastic%2Benvironments%2Bto%2Blife%2Bhistories%2Band%2Bback%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2009.0021%26rft_id%253Dinfo%253Apmid%252F19414465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1098/rstb.2009.0021&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19414465&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference in text" id="ref-76">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.76">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Turelli</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">N.</span>
          <span class="cit-name-surname">Barton</span>
         </span>. 
         <span class="cit-pub-date">1994</span>. 
         <span class="cit-article-title">Genetic and statistical analyses of strong selection on polygenic traits: what, me normal?</span>
         <abbr class="cit-jnl-abbrev">Genetics</abbr>
         <span class="cit-vol">138</span>: 
         <span class="cit-fpage">913</span>–
         <span class="cit-lpage">941</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTurelli%26rft.auinit1%253DM.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D913%26rft.epage%253D941%26rft.atitle%253DGenetic%2Band%2Bstatistical%2Banalyses%2Bof%2Bstrong%2Bselection%2Bon%2Bpolygenic%2Btraits%253A%2Bwhat%252C%2Bme%2Bnormal%253F%26rft_id%253Dinfo%253Apmid%252F7851785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTEzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTcvMDMvMjQvMDkwODk0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference in text" id="ref-77">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.77" data-doi="10.1111/ele.12421">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vindenes</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">Ø.</span>
          <span class="cit-name-surname">Langangen</span>
         </span>. 
         <span class="cit-pub-date">2015</span>. 
         <span class="cit-article-title">Individual heterogeneity in life histories and eco-evolutionary dynamics</span>. 
         <abbr class="cit-jnl-abbrev">Ecology letters</abbr>
         <span class="cit-vol">18</span>: 
         <span class="cit-fpage">417</span>–
         <span class="cit-lpage">432</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEcology%2Bletters%26rft.volume%253D18%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12421%26rft_id%253Dinfo%253Apmid%252F25807980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/ele.12421&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25807980&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference in text" id="ref-78">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.78">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wallace</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Leslie</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Coulson</span>
         </span>. 
         <span class="cit-pub-date">2013</span>. 
         <span class="cit-article-title">Re-evaluating the effect of harvesting regimes on nile crocodiles using an integral projection model</span>. 
         <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr>
         <span class="cit-vol">82</span>: 
         <span class="cit-fpage">155</span>–
         <span class="cit-lpage">165</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.volume%253D82%26rft.spage%253D155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.79" data-doi="10.1038/416389a">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Walther</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">E.</span>
          <span class="cit-name-surname">Post</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">P.</span>
          <span class="cit-name-surname">Convey</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">A.</span>
          <span class="cit-name-surname">Menzel</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C.</span>
          <span class="cit-name-surname">Parmesan</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">T.</span>
          <span class="cit-name-surname">Beebee</span>
         </span>, 
         <span class="cit-etal">et al.</span>
         <span class="cit-pub-date">2002</span>. 
         <span class="cit-article-title">Ecological responses to recent climate change</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">416</span>: 
         <span class="cit-fpage">389</span>–
         <span class="cit-lpage">395</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWalther%26rft.auinit1%253DG.%2BR.%26rft.volume%253D416%26rft.issue%253D6879%26rft.spage%253D389%26rft.epage%253D395%26rft.atitle%253DEcological%2Bresponses%2Bto%2Brecent%2Bclimate%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F416389a%26rft_id%253Dinfo%253Apmid%252F11919621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/416389a&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=11919621&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000174607800036&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference in text" id="ref-80">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-090894v6.80">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wiens</span>, 
          <span class="cit-name-given-names">J. A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Stralberg</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">D.</span>
          <span class="cit-name-surname">Jongsomjit</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">C. A.</span>
          <span class="cit-name-surname">Howell</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">M. A.</span>
          <span class="cit-name-surname">Snyder</span>
         </span>. 
         <span class="cit-pub-date">2009</span>. 
         <span class="cit-article-title">Niches, models, and climate change: assessing the assumptions and uncertainties</span>. 
         <span class="cit-conf-name">Proceedings of the National Academy of Sciences</span>
         <span class="cit-vol">106</span>: 
         <span class="cit-fpage">19729</span>–
         <span class="cit-lpage">19736</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference in text" id="ref-81">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.81" data-doi="10.1086/497441">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wilson</span>, 
          <span class="cit-name-given-names">A. J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L. E.</span>
          <span class="cit-name-surname">Kruuk</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">D. W.</span>
          <span class="cit-name-surname">Coltman</span>
         </span>. 
         <span class="cit-pub-date">2005</span>. 
         <span class="cit-article-title">Ontogenetic patterns in heritable variation for body size: using random regression models in a wild ungulate population</span>. 
         <abbr class="cit-jnl-abbrev">The American Naturalist</abbr>
         <span class="cit-vol">166</span>:
         <span class="cit-fpage">E177</span>–
         <span class="cit-lpage">E192</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DWilson%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D166%26rft.issue%253D6%26rft.spage%253DE177%26rft.epage%253DE192%26rft.atitle%253DOntogenetic%2Bpatterns%2Bin%2Bheritable%2Bvariation%2Bfor%2Bbody%2Bsize%253A%2Busing%2Brandom%2Bregression%2Bmodels%2Bin%2Ba%2Bwild%2Bungulate%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F497441%26rft_id%253Dinfo%253Apmid%252F16475080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1086/497441&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=16475080&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000233576600003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <div class="cit ref-cit ref-journal no-rev-xref" id="cit-090894v6.82" data-doi="10.1038/nature05835">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wolf</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G. S.</span>
          <span class="cit-name-surname">Van Doorn</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">O.</span>
          <span class="cit-name-surname">Leimar</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">F. J.</span>
          <span class="cit-name-surname">Weissing</span>
         </span>. 
         <span class="cit-pub-date">2007</span>. 
         <span class="cit-article-title">Life-history trade-offs favour the evolution of animal personalities</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">447</span>: 
         <span class="cit-fpage">581</span>–
         <span class="cit-lpage">584</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWolf%26rft.auinit1%253DM.%26rft.volume%253D447%26rft.issue%253D7144%26rft.spage%253D581%26rft.epage%253D584%26rft.atitle%253DLife-history%2Btrade-offs%2Bfavour%2Bthe%2Bevolution%2Bof%2Banimal%2Bpersonalities.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05835%26rft_id%253Dinfo%253Apmid%252F17538618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature05835&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17538618&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000246842000046&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference in text" id="ref-83">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-090894v6.83" data-doi="10.1038/nature01767">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yoshida</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">L. E.</span>
          <span class="cit-name-surname">Jones</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">S. P.</span>
          <span class="cit-name-surname">Ellner</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-given-names">G. F.</span>
          <span class="cit-name-surname">Fussmann</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-given-names">N. G.</span>
          <span class="cit-name-surname">Hairston</span>
         </span>. 
         <span class="cit-pub-date">2003</span>. 
         <span class="cit-article-title">Rapid evolution drives ecological dynamics in a predator–prey system</span>. 
         <abbr class="cit-jnl-abbrev">Nature</abbr>
         <span class="cit-vol">424</span>:
         <span class="cit-fpage">303</span>–
         <span class="cit-lpage">306</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYoshida%26rft.auinit1%253DT.%26rft.volume%253D424%26rft.issue%253D6946%26rft.spage%253D303%26rft.epage%253D306%26rft.atitle%253DRapid%2Bevolution%2Bdrives%2Becological%2Bdynamics%2Bin%2Ba%2Bpredator-prey%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01767%26rft_id%253Dinfo%253Apmid%252F12867979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nature01767&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12867979&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2017%2F03%2F24%2F090894.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000184183900039&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
