<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/12/20/2019.12.20.884114.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.20.884114v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions" />
<meta name="DC.Identifier" content="10.1101/2019.12.20.884114" />
<meta name="DC.Date" content="2019-12-20" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="In a context of rapid global change, one of the key components for the survival of species is their genetic adaptive potential. Many methods have been developed to identify adaptive genetic variants, but few tools were made available to integrate this knowledge into conservation management. We present here the SPatial Areas of Genotype probability (SPAG), using genotype-environment logistic associations to map the probability of finding beneficial variants in a study area. We define a univariate model predicting the spatial distribution of a single genotype, and three multivariate models allowing the integration of several genotypes, potentially associated with various environmental variables. We then integrate the conditions predicted by climate change scenarios to map the corresponding future spatial distribution of genotypes. The analysis of the mismatch between current and future SPAGs makes it possible to identify a) populations that are better adapted to the future climate through the presence of genetic variants able to cope with future conditions and b) vulnerable populations where genotype(s) of interest are not frequent enough for the individuals to adapt to the future climate. We use the SPAGs to study the potential adaptation of 161 Moroccan and 410 European goats to the bioclimatic conditions. In Morocco, using whole genome sequence data, we identify seven genomic regions strongly associated with the precipitation seasonality (WorldClim database). The predicted shift in SPAGs under strong climate change scenario for 2070 highlights goat’s populations likely to be threatened by the expected increase in precipitation variation in the future. In Europe, we find genomic regions associated with low precipitation, the shift in SPAGs highlighting vulnerable populations not adapted to the very dry conditions expected in 2070. The SPAG methodology is successfully validated using training and test samples and provides an efficient tool to take the adaptive potential into account in general conservation frameworks." />
<meta name="DC.Contributor" content="Estelle Rochat" />
<meta name="DC.Contributor" content="Stéphane Joost" />
<meta name="article:published_time" content="2019-12-20" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;In a context of rapid global change, one of the key components for the survival of species is their genetic adaptive potential. Many methods have been developed to identify adaptive genetic variants, but few tools were made available to integrate this knowledge into conservation management. We present here the SPatial Areas of Genotype probability (SPAG), using genotype-environment logistic associations to map the probability of finding beneficial variants in a study area. We define a univariate model predicting the spatial distribution of a single genotype, and three multivariate models allowing the integration of several genotypes, potentially associated with various environmental variables. We then integrate the conditions predicted by climate change scenarios to map the corresponding future spatial distribution of genotypes. The analysis of the mismatch between current and future SPAGs makes it possible to identify a) populations that are better adapted to the future climate through the presence of genetic variants able to cope with future conditions and b) vulnerable populations where genotype(s) of interest are not frequent enough for the individuals to adapt to the future climate. We use the SPAGs to study the potential adaptation of 161 Moroccan and 410 European goats to the bioclimatic conditions. In Morocco, using whole genome sequence data, we identify seven genomic regions strongly associated with the precipitation seasonality (WorldClim database). The predicted shift in SPAGs under strong climate change scenario for 2070 highlights goat’s populations likely to be threatened by the expected increase in precipitation variation in the future. In Europe, we find genomic regions associated with low precipitation, the shift in SPAGs highlighting vulnerable populations not adapted to the very dry conditions expected in 2070. The SPAG methodology is successfully validated using training and test samples and provides an efficient tool to take the adaptive potential into account in general conservation frameworks.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.20.884114v1" />
<meta name="citation_id" content="2019.12.20.884114v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.20.884114v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.20.884114v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.20.884114v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.20.884114" />
<meta name="citation_num_pages" content="30" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.20.884114" />
<meta name="citation_author" content="Estelle Rochat" />
<meta name="citation_author_institution" content="Laboratory of Geographic Information Systems (LASIG), School of Architecture, Civil and Environmental Engineering (ENAC), Ecole Polytechnique Fédérale de Lausanne (EPFL)" />
<meta name="citation_author_institution" content="The ADAPTMAP Consortium" />
<meta name="citation_author_email" content="estelle.rochat@epfl.ch" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7978-5239" />
<meta name="citation_author" content="Stéphane Joost" />
<meta name="citation_author_institution" content="Laboratory of Geographic Information Systems (LASIG), School of Architecture, Civil and Environmental Engineering (ENAC), Ecole Polytechnique Fédérale de Lausanne (EPFL)" />
<meta name="citation_author_institution" content="The ADAPTMAP Consortium" />
<meta name="citation_author_institution" content="The NEXTGEN Consortium" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1184-7501" />
<meta name="citation_reference" content="Ai H, Yang B, Li J, Xie X, Chen H, Ren J (2014). Population history and genomic signatures for high-altitude adaptation in Tibetan pigs. BMC Genomics 15: 834." />
<meta name="citation_reference" content="Aitken SN, Whitlock MC (2013). Assisted Gene Flow to Facilitate Local Adaptation to Climate Change. Annual Review of Ecology, Evolution, and Systematics 44: 367–388." />
<meta name="citation_reference" content="Alberto FJ, Boyer F, Orozco-terWengel P, Streeter I, Servin B, Villemereuil P de, et al. (2018). Convergent genomic signatures of domestication in sheep and goats. Nat Commun 9: 1–9." />
<meta name="citation_reference" content="Allendorf FW, Hohenlohe PA, Luikart G (2010). Genomics and the future of conservation genetics. Nat Rev Genet 11: 697–709." />
<meta name="citation_reference" content="Allendorf FW, Leary RF (1986). Heterozygosity and fitness in natural populations of animals. Conservation biology: the science of scarcity and diversity: 57–76." />
<meta name="citation_reference" content="Banta JA, Ehrenreich IM, Gerard S, Chou L, Wilczek A, Schmitt J, et al. (2012). Climate envelope modelling reveals intraspecific relationships among flowering phenology, niche breadth and potential range size in Arabidopsis thaliana. Ecology Letters 15: 769–777." />
<meta name="citation_reference" content="Benjamini Y, Hochberg Y (1995). Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. Journal of the Royal Statistical Society: Series B (Methodological) 57: 289–300." />
<meta name="citation_reference" content="Benjelloun B, Alberto FJ, Streeter I, Boyer F, Coissac E, Stucki S, et al. (2015). Characterizing neutral genomic diversity and selection signatures in indigenous populations of Moroccan goats (Capra hircus) using WGS data. Front Genet 6." />
<meta name="citation_reference" content="Bertolini F, Servin B, Talenti A, Rochat E, Kim ES, Oget C, et al. (2018). Signatures of selection and environmental adaptation across the goat genome post-domestication. Genetics Selection Evolution 50: 57." />
<meta name="citation_reference" content="Bickhart DM, Rosen BD, Koren S, Sayre BL, Hastie AR, Chan S, et al. (2017). Single-molecule sequencing and chromatin conformation capture enable de novo reference assembly of the domestic goat genome. Nat Genet 49: 643–650." />
<meta name="citation_reference" content="Bonin A, Nicole F, Pompanon F, Miaud C, Taberlet P (2007). Population Adaptive Index: a New Method to Help Measure Intraspecific Genetic Diversity and Prioritize Populations for Conservation. Conservation Biology 21: 697–708." />
<meta name="citation_reference" content="Boujenane I (2005). Small Ruminant Breeds of Morocco." />
<meta name="citation_reference" content="Brito LF, Kijas JW, Ventura RV, Sargolzaei M, Porto-Neto LR, Cánovas A, et al. (2017). Genetic diversity and signatures of selection in various goat breeds revealed by genome-wide SNP markers. BMC Genomics 18: 229." />
<meta name="citation_reference" content="Burkitt Wright EMM, Spencer HL, Daly SB, Manson FDC, Zeef LAH, Urquhart J, et al. (2011). Mutations in PRDM5 in Brittle Cornea Syndrome Identify a Pathway Regulating Extracellular Matrix Development and Maintenance. The American Journal of Human Genetics 88: 767–777." />
<meta name="citation_reference" content="Cardona A, Pagani L, Antao T, Lawson DJ, Eichstaedt CA, Yngvadottir B, et al. (2014). Genome-Wide Analysis of Cold Adaptation in Indigenous Siberian Populations. PLOS ONE 9: e98076." />
<meta name="citation_reference" content="Chentouf M (2014). Systèmes de production caprine au Nord du Maroc: Contraintes et propositions d’amélioration. Opt Médit A: Médit Semin 108: 25–32." />
<meta name="citation_reference" content="Cousyn C, Meester LD, Colbourne JK, Brendonck L, Verschuren D, Volckaert F (2001). Rapid, local adaptation of zooplankton behavior to changes in predation pressure in the absence of neutral genetic changes. PNAS 98: 6256–6260." />
<meta name="citation_reference" content="Dawson TP, Jackson ST, House JI, Prentice IC, Mace GM (2011). Beyond Predictions: Biodiversity Conservation in a Changing Climate. Science 332: 53–58." />
<meta name="citation_reference" content="Dong Y, Xie M, Jiang Y, Xiao N, Du X, Zhang W, et al. (2013). Sequencing and automated whole-genome optical mapping of the genome of a domestic goat (Capra hircus). Nature Biotechnology 31: 135–141." />
<meta name="citation_reference" content="Duputié A, Rutschmann A, Ronce O, Chuine I (2015). Phenological plasticity will not help all species adapt to climate change. Global Change Biology 21: 3062–3073." />
<meta name="citation_reference" content="Fitzpatrick MC, Keller SR (2015). Ecological genomics meets community-level modelling of biodiversity: mapping the genomic landscape of current and future environmental adaptation. Ecology Letters 18: 1–16." />
<meta name="citation_reference" content="Forester BR, Lasky JR, Wagner HH, Urban DL (2017). Using genotype-environment associations to identify multilocus local adaptation. bioRxiv: 129460." />
<meta name="citation_reference" content="Fournier-Level A, Korte A, Cooper MD, Nordborg M, Schmitt J, Wilczek AM (2011). A Map of Local Adaptation in Arabidopsis thaliana. Science 334: 86–89." />
<meta name="citation_reference" content="Funk WC, Forester BR, Converse SJ, Darst C, Morey S (2019). Improving conservation policy with genomics: a guide to integrating adaptive potential into U.S. Endangered Species Act decisions for conservation practitioners and geneticists. Conserv Genet 20: 115–134." />
<meta name="citation_reference" content="Funk WC, McKay JK, Hohenlohe PA, Allendorf FW (2012). Harnessing genomics for delineating conservation units. Trends in Ecology &amp; Evolution 27: 489–496." />
<meta name="citation_reference" content="Galbraith H (2010). Fundamental hair follicle biology and fine fibre production in animals. animal 4: 1490–1509." />
<meta name="citation_reference" content="Garzón MB, Robson TM, Hampe A (2019). ΔTraitSDMs: species distribution models that account for local adaptation and phenotypic plasticity. New Phytologist 222: 1757–1765." />
<meta name="citation_reference" content="Ge W, Wang S-H, Sun B, Zhang Y-L, Shen W, Khatib H, et al. (2018). Melatonin promotes Cashmere goat (Capra hircus) secondary hair follicle growth: a view from integrated analysis of long non-coding and coding RNAs. Cell Cycle 17: 1255–1267." />
<meta name="citation_reference" content="Godber OF, Laroussi BF, Chentouf M, Wall R (2016). Intensification of Mediterranean Goat Production Systems: A Case Study in Northern Morocco. Agriculture 6: 16." />
<meta name="citation_reference" content="Goto M (1997). Hierarchical deterioration of body systems in Werner’s syndrome: Implications for normal ageing. Mechanisms of Ageing and Development 98: 239–254." />
<meta name="citation_reference" content="Hällfors MH, Liao J, Dzurisin J, Grundel R, Hyvärinen M, Towle K, et al. (2016). Addressing potential local adaptation in species distribution models: implications for conservation under climate change. Ecological Applications 26: 1154–1169." />
<meta name="citation_reference" content="Hancock AM, Brachi B, Faure N, Horton MW, Jarymowycz LB, Sperone FG, et al. (2011). Adaptation to Climate Across the Arabidopsis thaliana Genome. Science 334: 83–86." />
<meta name="citation_reference" content="Harris SE, Munshi-South J (2016). Scans for positive selection reveal candidate genes and local adaptation of Peromyscus leucopus populations to urbanization. bioRxiv: 038141." />
<meta name="citation_reference" content="Harrisson KA, Pavlova A, Telonis-Scott M, Sunnucks P (2014). Using genomics to characterize evolutionary potential for conservation of wild populations. Evol Appl 7: 1008–1025." />
<meta name="citation_reference" content="Hayashi R, Yamato M, Sugiyama H, Sumide T, Yang J, Okano T, et al. (2007). N-Cadherin Is Expressed by Putative Stem/Progenitor Cells and Melanocytes in the Human Limbal Epithelial Stem Cell Niche. STEM CELLS 25: 289–296." />
<meta name="citation_reference" content="Hijmans RJ, Cameron SE, Parra JL, Jones PG, Jarvis A (2005). Very high resolution interpolated climate surfaces for global land areas. International Journal of Climatology 25: 1965–1978." />
<meta name="citation_reference" content="Hijmans RJ, van Etten J (2012). raster: Geographic analysis and modeling with raster data. R package version 2.0-12. http://CRAN.R-project.org/package=raster." />
<meta name="citation_reference" content="Hoban S, Kelley JL, Lotterhos KE, Antolin MF, Bradburd G, Lowry DB, et al. (2016). Finding the Genomic Basis of Local Adaptation: Pitfalls, Practical Solutions, and Future Directions. The American Naturalist 188: 379–397." />
<meta name="citation_reference" content="Hoelzel AR, Bruford MW, Fleischer RC (2019). Conservation of adaptive potential and functional diversity. Conserv Genet 20: 1–5." />
<meta name="citation_reference" content="Hoffmann I (2010). Climate change and the characterization, breeding and conservation of animal genetic resources. Animal Genetics 41: 32–46." />
<meta name="citation_reference" content="Hoffmann AA, Sgrò CM (2011). Climate change and evolutionary adaptation. Nature 470: 479–485." />
<meta name="citation_reference" content="Hoffmann AA, Willi Y (2008). Detecting genetic responses to environmental change. Nat Rev Genet 9: 421–432." />
<meta name="citation_reference" content="Hughes L (2000). Biological consequences of global warming: is the signal already apparent? Trends in Ecology &amp; Evolution 15: 56–61." />
<meta name="citation_reference" content="Ikeda DH, Max TL, Allan GJ, Lau MK, Shuster SM, Whitham TG (2017). Genetically informed ecological niche models improve climate change predictions. Global Change Biology 23: 164–176." />
<meta name="citation_reference" content="IPCC (2014). Climate Change 2014: Synthesis Report. Contribution of Working Groups I, II and III to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change." />
<meta name="citation_reference" content="Jombart T (2008). adegenet: a R package for the multivariate analysis of genetic markers. Bioinformatics 24: 1403–1405." />
<meta name="citation_reference" content="Joost S (2006). The geographical dimension of genetic diversity: a GIScience contribution for the conservation of animal genetic resources." />
<meta name="citation_reference" content="Joost S, Bonin A, Bruford MW, Després L, Conord C, Erhardt G, et al. (2007). A spatial analysis method (SAM) to detect candidate loci for selection: towards a landscape genomics approach to adaptation. Molecular Ecology 16: 3955–3969." />
<meta name="citation_reference" content="Joost S, Vuilleumier S, Jensen JD, Schoville S, Leempoel K, Stucki S, et al. (2013). Uncovering the genetic basis of adaptive change: on the intersection of landscape genomics and theoretical population genetics. Mol Ecol 22: 3659–3665." />
<meta name="citation_reference" content="Kao WW, Liu CY, Converse RL, Shiraishi A, Kao CW, Ishizaki M, et al. (1996). Keratin 12-deficient mice have fragile corneal epithelia. Invest Ophthalmol Vis Sci 37: 2572–2584." />
<meta name="citation_reference" content="Kardos M, Shafer ABA (2018). The Peril of Gene-Targeted Conservation. Trends in Ecology &amp; Evolution 33: 827–839." />
<meta name="citation_reference" content="Karenko L, Hahtola S, Päivinen S, Karhu R, Syrjä S, Kähkönen M, et al. (2005). Primary Cutaneous T-Cell Lymphomas Show a Deletion or Translocation Affecting NAV3, the Human UNC-53 Homologue. Cancer Res 65: 8101–8110." />
<meta name="citation_reference" content="Kelly E, Phillips BL (2016). Targeted gene flow for conservation. Conservation Biology 30: 259–267." />
<meta name="citation_reference" content="Kljuic A, Bazzi H, Sundberg JP, Martinez-Mir A, O’Shaughnessy R, Mahoney MG, et al. (2003). Desmoglein 4 in Hair Follicle Differentiation and Epidermal Adhesion: Evidence from Inherited Hypotrichosis and Acquired Pemphigus Vulgaris. Cell 113: 249–260." />
<meta name="citation_reference" content="Kolmogorov A (1956). Foundations of the Theory of Probability. Chelsea Pub Co." />
<meta name="citation_reference" content="Kutzleb C, Sanders G, Yamamoto R, Wang X, Lichte B, Petrasch-Parwez E, et al. (1998). Paralemmin, a Prenyl-Palmitoyl–anchored Phosphoprotein Abundant in Neurons and Implicated in Plasma Membrane Dynamics and Cell Process Formation. The Journal of Cell Biology 143: 795–813." />
<meta name="citation_reference" content="Lasky JR, Upadhyaya HD, Ramu P, Deshpande S, Hash CT, Bonnette J, et al. (2015). Genome-environment associations in sorghum landraces predict adaptive traits. Science Advances 1: e1400218." />
<meta name="citation_reference" content="Li X, Chen C, Wang F, Huang W, Liang Z, Xiao Y, et al. (2014). KCTD1 Suppresses Canonical Wnt Signaling Pathway by Enhancing β-catenin Degradation. PLOS ONE 9: e94343." />
<meta name="citation_reference" content="Li J, Li H, Jakobsson M, Li S, Sjödin P, Lascoux M (2012). Joint analysis of demography and selection in population genetics: where do we stand and where could we go? Molecular Ecology 21: 28–44." />
<meta name="citation_reference" content="Ling YH, Xiang, H, Zhang G, Ding JP, Zhang ZJ, Zhang YH, et al. (2014). Identification of complete linkage disequilibrium in the DSG4 gene and its association with wool length and crimp in Chinese indigenous sheep. GMR | Genetics and Molecular Research | The Original by FUNPEC-RP 13: 5617–5625." />
<meta name="citation_reference" content="Mable BK (2019). Conservation of adaptive potential and functional diversity: integrating old and new approaches. Conserv Genet 20: 89–100." />
<meta name="citation_reference" content="Mdladla K, Dzomba EF, Muchadeyi FC (2018). Landscape genomics and pathway analysis to understand genetic adaptation of South African indigenous goat populations. Heredity 120: 369–378." />
<meta name="citation_reference" content="Nicotra AB, Beever EA, Robertson AL, Hofmann GE, O’Leary J (2015). Assessing the components of adaptive capacity to improve conservation and management efforts under global change. Conservation Biology 29: 1268–1278." />
<meta name="citation_reference" content="Opdam P, Wascher D (2004). Climate change meets habitat fragmentation: linking landscape and biogeographical scale levels in research and conservation. Biological Conservation 117: 285–297." />
<meta name="citation_reference" content="Orr HA, Unckless RL (2008). Population Extinction and the Genetics of Adaptation. The American Naturalist 172: 160–169." />
<meta name="citation_reference" content="Pariset L, Joost S, Marsan PA, Valentini A, Econogene Consortium (EC) (2009). Landscape genomics and biased FST approaches reveal single nucleotide polymorphisms under selection in goat breeds of North-East Mediterranean. BMC Genet 10: 7." />
<meta name="citation_reference" content="Pauls SU, Nowak C, Bálint M, Pfenninger M (2013). The impact of global climate change on genetic diversity within populations and species. Mol Ecol 22: 925–946." />
<meta name="citation_reference" content="Phillips SJ, Anderson RP, Schapire RE (2006). Maximum entropy modeling of species geographic distributions. Ecological Modelling 190: 231–259." />
<meta name="citation_reference" content="Pritchard JK, Rienzo AD (2010). Adaptation – not by sweeps alone. Nat Rev Genet 11: 665–667." />
<meta name="citation_reference" content="R Development Core Team (2008). R: A language and environment for statistical computing. R Foundation for Statistical Computing." />
<meta name="citation_reference" content="Ramunno L, Cosenza G, Pappalardo M, Longobardi E, Gallo D, Pastore N, et al. (2001). Characterization of two new alleles at the goat CSN1S2 locus. Animal Genetics 32: 264–268." />
<meta name="citation_reference" content="Razgour O, Forester B, Taggart JB, Bekaert M, Juste J, Ibáñez C, et al. (2019). Considering adaptive genetic variation in climate change vulnerability assessment reduces species range loss projections. PNAS 116: 10418–10423." />
<meta name="citation_reference" content="Razgour O, Taggart JB, Manel S, Juste J, Ibáñez C, Rebelo H, et al. (2018). An integrated framework to identify wildlife populations under threat from climate change. Molecular Ecology Resources 18: 18–31." />
<meta name="citation_reference" content="Reed DH, Frankham R (2001). How Closely Correlated Are Molecular and Quantitative Measures of Genetic Variation? A Meta-Analysis. Evolution 55: 1095–1103." />
<meta name="citation_reference" content="Reed TE, Schindler DE, Waples RS (2011). Interacting Effects of Phenotypic Plasticity and Evolution on Population Persistence in a Changing Climate. Conservation Biology 25: 56–63." />
<meta name="citation_reference" content="Ren H, Wang G, Chen L, Jiang J, Liu L, Li N, et al. (2016). Genome-wide analysis of long non-coding RNAs at early stage of skin pigmentation in goats (Capra hircus). BMC Genomics 17: 67." />
<meta name="citation_reference" content="Reside AE, Butt N, Adams VM (2018). Adapting systematic conservation planning for climate change. Biodivers Conserv 27: 1–29." />
<meta name="citation_reference" content="Rischkowsky B, Pilling D (2007). The state of the world’s animal genetic resources for food and agriculture. Food &amp; Agriculture Org." />
<meta name="citation_reference" content="Rochat E, Leempoel K, Vajana E, Colli L, Ajmone-Marsan P, Joost S, et al. (2016). Map of genotype frequency change in autochtonous Moroccan sheep breeds due to global warming." />
<meta name="citation_reference" content="Savage AE, Mulder KP, Torres T, Wells S (2018). Lost but not forgotten: MHC genotypes predict overwinter survival despite depauperate MHC diversity in a declining frog. Conserv Genet 19: 309–322." />
<meta name="citation_reference" content="citation_title=The Future of Drylands;citation_pages=89-100;citation_year=2008" />
<meta name="citation_reference" content="Schoville SD, Bonin A, François O, Lobreaux S, Melodelima C, Manel S (2012). Adaptive Genetic Variation on the Landscape: Methods and Cases. Annual Review of Ecology, Evolution, and Systematics 43: 23–43." />
<meta name="citation_reference" content="Selmoni O, Rochat E, Lecellier G, Berteaux-Lecellier V, Joost S (2019). Seascape genomics as a new tool to empower coral reef conservation strategies: an example on north-western Pacific Acropora digitifera. bioRxiv: 588228." />
<meta name="citation_reference" content="Sgrò CM, Lowe AJ, Hoffmann AA (2011). Building evolutionary resilience for conserving biodiversity under climate change. Evolutionary Applications 4: 326–337." />
<meta name="citation_reference" content="Shafer ABA, Wolf JBW, Alves PC, Bergström L, Bruford MW, Brännström I, et al. (2015). Genomics and the challenging translation into conservation practice. Trends in Ecology &amp; Evolution 30: 78–87." />
<meta name="citation_reference" content="Stella A, Nicolazzi EL, Van Tassell CP, Rothschild MF, Colli L, Rosen BD, et al. (2018). AdaptMap: exploring goat diversity and adaptation. Genet Sel Evol 50: 61." />
<meta name="citation_reference" content="Stucki S, Orozco-terWengel P, Forester BR, Duruz S, Colli L, Masembe C, et al. (2017). High performance computation of landscape genomic models including local indicators of spatial association. Molecular Ecology Resources 17: 1072–1089." />
<meta name="citation_reference" content="Thomassen HA, Fuller T, Buermann W, Milá B, Kieswetter CM, Jarrín-V. P, et al. (2011). Mapping evolutionary process: a multi-taxa approach to conservation prioritization. Evolutionary Applications 4: 397–413." />
<meta name="citation_reference" content="Vajana E, Barbato M, Colli L, Milanesi M, Rochat E, Fabrizi E, et al. (2018). Combining Landscape Genomics and Ecological Modelling to Investigate Local Adaptation of Indigenous Ugandan Cattle to East Coast Fever. Front Genet 9." />
<meta name="citation_reference" content="Vandergast AG, Perry WM, Lugo RV, Hathaway SA (2011). Genetic landscapes GIS Toolbox: tools to map patterns of genetic divergence and diversity. Molecular Ecology Resources 11: 158–161." />
<meta name="citation_reference" content="Vangestel C, Eckert AJ, Wegrzyn JL, St. Clair JB, Neale DB (2018). Linking phenotype, genotype and environment to unravel genetic components underlying cold hardiness in coastal Douglas-fir (Pseudotsuga menziesii var. menziesii). Tree Genetics &amp; Genomes 14: 10." />
<meta name="citation_reference" content="Villemereuil P de, Frichot É, Bazin É, François O, Gaggiotti OE (2014). Genome scan methods against more complex models: when and how much should we trust them? Molecular Ecology 23: 2006–2019." />
<meta name="citation_reference" content="Vitti JJ, Grossman SR, Sabeti PC (2013). Detecting Natural Selection in Genomic Data. Annual Review of Genetics 47: 97–120." />
<meta name="citation_reference" content="Wang S, Ge W, Luo Z, Guo Y, Jiao B, Qu L, et al. (2017). Integrated analysis of coding genes and non-coding RNAs during hair follicle cycle of cashmere goat (Capra hircus). BMC Genomics 18: 767." />
<meta name="citation_reference" content="Weeks AR, Sgro CM, Young AG, Frankham R, Mitchell NJ, Miller KA, et al. (2011). Assessing the benefits and risks of translocations in changing environments: a genetic perspective. Evolutionary Applications 4: 709–725." />
<meta name="citation_reference" content="Williams SE, Shoo LP, Isaac JL, Hoffmann AA, Langham G (2008). Towards an Integrated Framework for Assessing the Vulnerability of Species to Climate Change. PLOS Biol 6: e325." />
<meta name="citation_reference" content="Willoughby JR, Harder AM, Tennessen JA, Scribner KT, Christie MR (2018). Rapid genetic adaptation to a novel environment despite a genome-wide reduction in genetic diversity. Molecular Ecology 27: 4041–4051." />
<meta name="citation_reference" content="Zhang Y, Wu K, Wang L, Wang Z, Han W, Chen D, et al. (2019). Comparative study on seasonal hair follicle cycling by analysis of the transcriptomes from cashmere and milk goats. Genomics." />
<meta name="citation_reference" content="Zheng X, Levine D, Shen J, Gogarten SM, Laurie C, Weir BS (2012). A high-performance computing toolset for relatedness and principal component analysis of SNP data. Bioinformatics 28: 3326–3328." />
<meta name="citation_reference" content="Zhou X, Carbonetto P, Stephens M (2013). Polygenic Modeling with Bayesian Sparse Linear Mixed Models. PLOS Genetics 9: e1003264." />
<meta name="twitter:title" content="Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="In a context of rapid global change, one of the key components for the survival of species is their genetic adaptive potential. Many methods have been developed to identify adaptive genetic variants, but few tools were made available to integrate this knowledge into conservation management. We present here the SPatial Areas of Genotype probability (SPAG), using genotype-environment logistic associations to map the probability of finding beneficial variants in a study area. We define a univariate model predicting the spatial distribution of a single genotype, and three multivariate models allowing the integration of several genotypes, potentially associated with various environmental variables. We then integrate the conditions predicted by climate change scenarios to map the corresponding future spatial distribution of genotypes. The analysis of the mismatch between current and future SPAGs makes it possible to identify a) populations that are better adapted to the future climate through the presence of genetic variants able to cope with future conditions and b) vulnerable populations where genotype(s) of interest are not frequent enough for the individuals to adapt to the future climate. We use the SPAGs to study the potential adaptation of 161 Moroccan and 410 European goats to the bioclimatic conditions. In Morocco, using whole genome sequence data, we identify seven genomic regions strongly associated with the precipitation seasonality (WorldClim database). The predicted shift in SPAGs under strong climate change scenario for 2070 highlights goat’s populations likely to be threatened by the expected increase in precipitation variation in the future. In Europe, we find genomic regions associated with low precipitation, the shift in SPAGs highlighting vulnerable populations not adapted to the very dry conditions expected in 2070. The SPAG methodology is successfully validated using training and test samples and provides an efficient tool to take the adaptive potential into account in general conservation frameworks." />
<meta name="og-title" property="og:title" content="Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.20.884114v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="In a context of rapid global change, one of the key components for the survival of species is their genetic adaptive potential. Many methods have been developed to identify adaptive genetic variants, but few tools were made available to integrate this knowledge into conservation management. We present here the SPatial Areas of Genotype probability (SPAG), using genotype-environment logistic associations to map the probability of finding beneficial variants in a study area. We define a univariate model predicting the spatial distribution of a single genotype, and three multivariate models allowing the integration of several genotypes, potentially associated with various environmental variables. We then integrate the conditions predicted by climate change scenarios to map the corresponding future spatial distribution of genotypes. The analysis of the mismatch between current and future SPAGs makes it possible to identify a) populations that are better adapted to the future climate through the presence of genetic variants able to cope with future conditions and b) vulnerable populations where genotype(s) of interest are not frequent enough for the individuals to adapt to the future climate. We use the SPAGs to study the potential adaptation of 161 Moroccan and 410 European goats to the bioclimatic conditions. In Morocco, using whole genome sequence data, we identify seven genomic regions strongly associated with the precipitation seasonality (WorldClim database). The predicted shift in SPAGs under strong climate change scenario for 2070 highlights goat’s populations likely to be threatened by the expected increase in precipitation variation in the future. In Europe, we find genomic regions associated with low precipitation, the shift in SPAGs highlighting vulnerable populations not adapted to the very dry conditions expected in 2070. The SPAG methodology is successfully validated using training and test samples and provides an efficient tool to take the adaptive potential into account in general conservation frameworks." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-12-20" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.20.884114v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.20.884114v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.20.884114v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.20.884114v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1057159" />
    <title>Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Us0ld0S5h5DktbqAMbU2hGLiWrhQfqWXQ6miDlCAvrs","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1057159","apath":"\/biorxiv\/early\/2019\/12\/20\/2019.12.20.884114.atom","pisa":"biorxiv;2019.12.20.884114v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.20.884114v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.20.884114v1","title":"Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions","identifier":"node\/1057159"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220GI5K1xMKcJ6s04JzndypoTM2jfAgkiavX3SwH1M9XQH\\\\\\\/4xzknz6GJAsA60Ja79hLF2E6oCUhPlWCpQewAG3dhakE94OTE++8MvzE+5V4aiVd+mgMwqy3NMaokGpkfzO0nHdx1E95BenSXlOyzqmvEs9opR1H6btGMVymoJD0SrfC38xEa7gAvn\\\\\\\/vAVj41WEpTWg6V16mGtlZI0DX9rinw+jcJ8vMm8xxANcdApdqB1mU0V59Rkh+oV6hwBXsqw5UEGFJ6Jei\\\\\\\/EhTCKs4z9MOWBTg8b+1pa1tq7eipaWgXcDevBtNc6DFDCEPLMtwws6h0acWt4KIf8LydC+m6zofOqDprpd53M\\\\\\\/ZOs+fgTJ+jpLeg8axO9SSB5ouM4B22418wfm1JSUdE\\\\\\\/sTQ\\\\\\\/SZ5RQMRYfidjjKPnPXWXTD+MGUs2jiNretOlId3L4ILZh09fJ2HM76zphj6GYVlV1Olg4xefUNGwUlejq8fuxTPrs0lTGbG1cCcVLBMk0qffc7EFwqxR671KE+HEhtoRrYw5LgEouEbIrqRZSylFfyuA8ThbfQhpj62vtzcYCd9KN+QxilNYKCktoxNUuUHBMzksTGafyl+1B3sHOfV2BhNuOBqvvFZByYyQbsyx1TV7G3NbrwRO1Cwu0xlKckEzfsCLHkIDsets4K+xb6w3\\\\\\\/5H2PYsAR3i+1VyaUV55aOG4nSUvWgPBzKez8Y5dLmTLOJv4Gfn4N8\\\\\\\/mmHYSBYxci9MKG2dK7oXVZ8Iu+4UPlh3OApf0g+7jdRv86iqCFjnFRN18O04FuJFz5FJJX4tM\\\\\\\/7jyxpUdsUdQ3sjT31oBv43q8A3ShHCDr\\\\\\\/epDMUn0xcgSsVdS2cJYAoepgj+iszfgE6OkVgmMjfOrOdpU2HanoftCabAIRsfCHkV2Pn8h8jhj0yvpEj7ERtJuR0BdT\\\\\\\/ZfupqRUAb4Oapx3EGw9JAbLYdWyJ+FSxRklq5pACsHvW2mWHM7AcLbZyvSL8BG8BKa3pEBhJ\\\\\\\/5\\\\\\\/0aSGxiT2rwGVRynk2GOLrEqJjJDwsfV0dcvGNoqZNTZmmQPj4Q60IXrUZeQ1T8brVrH8zNXr5BIUmT9K4Md1xjyC7qMZWwjIgHRlQUijtF6xqvOtKdRVG+xXHgaEn+kQVQ8O0EMKKO4bnmzakDYfl10H8WdGp1S8jxmWVde5wNYxVeQsdhPkVnzDe3FGVGDIQctfyofdD1siJjEztgWTlomjiKe7dqTwe3znFeR+LRBtYqU8geQdz9tRBiLLMZim0VGn9BkS7XjyQfIRaYhRP\\\\\\\/nePaaOEoDb29R7EBwEiqBk0wh0rq\\\\\\\/YIvYRcIyhOmGxlqFSL7SPeybw2pM\\\\\\\/G7f\\\u0022,\\\u0022iv\\\u0022:\\\u0022EHrKXKWlc2aLL8DEI\\\\\\\/INVQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00224f52cf9d991f037f68f73a499aa31a72e578b267c42f2e01148e84d0967ec209\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.20.884114v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1057159 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.20.884114v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mOfmrmhB1kd4bAPI6y7-MAixKM4MooNzafl1n3osDW0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1057159" id="node1057159" data-pisa="biorxiv;2019.12.20.884114v1" data-pisa-master="biorxiv;2019.12.20.884114" data-apath="/biorxiv/early/2019/12/20/2019.12.20.884114.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-7978-5239" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Estelle</span> <span class="nlm-surname">Rochat</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-1184-7501" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Stéphane</span> <span class="nlm-surname">Joost</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.20.884114 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1057159" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Estelle Rochat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratory of Geographic Information Systems (LASIG), School of Architecture, Civil and Environmental Engineering (ENAC), Ecole Polytechnique Fédérale de Lausanne (EPFL)</span>, Lausanne, <span class='nlm-country'>Switzerland</span></div><div class='author-affiliation'><span class='nlm-institution'>The ADAPTMAP Consortium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Estelle%2BRochat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rochat%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEstelle%2BRochat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7978-5239" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Estelle Rochat</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:estelle.rochat@epfl.ch" class="" data-icon-position="" data-hide-link-title="0">estelle.rochat@epfl.ch</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stéphane Joost </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratory of Geographic Information Systems (LASIG), School of Architecture, Civil and Environmental Engineering (ENAC), Ecole Polytechnique Fédérale de Lausanne (EPFL)</span>, Lausanne, <span class='nlm-country'>Switzerland</span></div><div class='author-affiliation'><span class='nlm-institution'>The ADAPTMAP Consortium</span></div><div class='author-affiliation'><span class='nlm-institution'>The NEXTGEN Consortium</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=St%C3%A9phane%2BJoost%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Joost%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASt%25C3%25A9phane%2BJoost%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1184-7501" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Stéphane Joost</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.20.884114v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1057159" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1057159/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.20.884114v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1057159" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1057159/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.20.884114v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1057159" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1057159/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.20.884114v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1057159" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1057159/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.20.884114v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1057159" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1057159/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.20.884114v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1057159" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1057159/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">In a context of rapid global change, one of the key components for the survival of species is their genetic adaptive potential. Many methods have been developed to identify adaptive genetic variants, but few tools were made available to integrate this knowledge into conservation management. We present here the SPatial Areas of Genotype probability (SPAG), using genotype-environment logistic associations to map the probability of finding beneficial variants in a study area. We define a univariate model predicting the spatial distribution of a single genotype, and three multivariate models allowing the integration of several genotypes, potentially associated with various environmental variables. We then integrate the conditions predicted by climate change scenarios to map the corresponding future spatial distribution of genotypes. The analysis of the mismatch between current and future SPAGs makes it possible to identify a) populations that are better adapted to the future climate through the presence of genetic variants able to cope with future conditions and b) vulnerable populations where genotype(s) of interest are not frequent enough for the individuals to adapt to the future climate. We use the SPAGs to study the potential adaptation of 161 Moroccan and 410 European goats to the bioclimatic conditions. In Morocco, using whole genome sequence data, we identify seven genomic regions strongly associated with the precipitation seasonality (WorldClim database). The predicted shift in SPAGs under strong climate change scenario for 2070 highlights goat’s populations likely to be threatened by the expected increase in precipitation variation in the future. In Europe, we find genomic regions associated with low precipitation, the shift in SPAGs highlighting vulnerable populations not adapted to the very dry conditions expected in 2070. The SPAG methodology is successfully validated using training and test samples and provides an efficient tool to take the adaptive potential into account in general conservation frameworks.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-3">Climate change models predict a rise in the world’s mean temperature and an increase in extreme precipitation events by the end of the 21<sup>st</sup> century, which is likely to cause acidification of oceans, a rise in sea-level and more frequent extreme events like storms, drought and flooding (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">IPCC, 2014</a>). When such important changes occur, many animal and plant species are confronted with a shift away from the favourable conditions necessary for their survival. In such conditions, to avoid extinction, they can either move to more favourable habitats or adapt to their new environment, plastically or genetically (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hughes, 2000</a>). Due to the limits of the dispersal capacity, the loss of favourable habitats and the increase in landscape fragmentation, migration to new areas is often limited (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Opdam and Wascher, 2004</a>). Plastic adaptation can allow species to rapidly evolve by changing their behaviour, physiology or morphology (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Reed <em>et al.</em>, 2011</a>). However, this can potentially lead to a fitness reduction (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Duputié et <em>al.</em>, 2015</a>) and it will not necessarily ensure persistence of the next generations since it does not rely on any genetic heritable variation. In order to preserve biodiversity, it is therefore crucial to promote the conservation of the genetic adaptive potential in populations (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Hoffmann and Sgrò, 2011</a>; Sgrò et <em>al.</em>, 2011; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Nicotra et <em>al.</em>, 2015</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Shafer et <em>al.</em>, 2015</a>).</p><p id="p-4">The conservation of this adaptive potential is also of main importance for livestock management in order to ensure the persistence of herds, particularly in developing countries facing challenging habitat conditions (e.g. heat stress, poor food resources and the presence of parasites and diseases), where the traditional breeds have adapted to the local environment (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hoffmann, 2010</a>). However, due to the increasing demand for food production, local breeds currently tend to be replaced by high-producing commercial breeds imported from developed countries (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Rischkowsky and Pilling, 2007</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Hoffmann, 2010</a>). Such animals lack adaptive qualities and may thus be particularly vulnerable to climate change, itself likely to reinforce the already stressful conditions. It is therefore essential to highlight the adaptive potential of livestock species in order to encourage farmers to conserve local traditional breeds, and to carefully design cross-breeding, translocation or artificial selection (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Scherf et <em>al.</em>, 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Allendorf <em>et al.</em>, 2010</a>).</p><p id="p-5">One of the essential components of the adaptive capacity of populations is genetic diversity (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Allendorf and Leary, 1986</a>). Since mutation rates are generally low, adaptation to rapid environmental changes largely depends on the amount of genetic variants already present in populations, i.e. standing genetic diversity (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Orr and Unckless, 2008</a>). With the recent increase in the availability of genetic data and the development of conservation genomics, various tools were developed to integrate the genetic diversity into conservation frameworks. For example, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bonin et al. (2007)</a> developed an index to integrate the intraspecific genetic diversity into conservation planning, <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Thomassen et al. (2011)</a> proposed a method to summarize phenotypes or genotypes diversity and <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Vandergast et al. (2011)</a> developed a GIS Toolbox to map the genetic diversity and divergence.</p><p id="p-6">However, conserving neutral genetic diversity in populations may not be sufficient to allow rapid adaptation to increasingly stressful conditions (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Reed and Frankham, 2001</a>), and it could be more valuable to specifically preserve the standing genetic variants likely to provide a better adaptation to future conditions, i.e. adaptive genetic diversity (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hoffmann and Willi, 2008</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Sgrò <em>et al.</em>, 2011</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Willoughby <em>et al.</em>, 2018</a>) or to combine both approaches (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Funk <em>et al.</em>, 2012</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Pauls <em>et al.</em>, 2013</a>). An increasing attention is currently being paid to this issue in conservation discussions (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Funk <em>et al.</em>, 2019</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hoelzel <em>et al.</em>, 2019</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Mable, 2019</a>).</p><p id="p-7">Many methods have been developed to identify loci showing signature of local adaptation, based on various assumptions and with different limitations and advantages (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Schoville <em>et al.</em>, 2012</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Vitti <em>et al.</em>, 2013</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Joost <em>et al.</em>, 2013</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hoban <em>et al.</em>, 2016</a>). The results were notably used to establish prediction of future habitat range of species facing climate change (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hällfors <em>et al.</em>, 2016</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ikeda <em>et al.</em>, 2017</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Garzón <em>et al.</em>, 2019</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Razgour <em>et al.</em>, 2019</a>). However, there is currently a need to integrate this knowledge in order to predict the distribution of adapted genetic variants along environmental gradients, estimate the adaptive potential of populations and identify individuals better suited to future environmental conditions. Only few studies presented methods to map the spatial distribution of adaptive alleles. Fournier et al (2011) and <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Banta et al. (2012)</a> addressed this issue using the well-known species distribution modelling software Maxent (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Phillips <em>et al.</em>, 2006</a>). <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fitzpatrick and Keller (2015)</a> proposed a much powerful approach based on two community-level modelling methods (Generalised Dissimilarity Modelling and Gradient Forest) to map the current spatial distribution of adaptive variants and assess the vulnerability of populations under climate change as a function of the mismatch between current distribution of alleles and future predictions.</p><p id="p-8">We propose here a new approach to predict genotype frequencies and map Spatial Areas of Genotypes Probabilities (SPAG) based on logistic genotype-environment associations (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Joost <em>et al.</em>, 2007</a>) and the theory of conditional probabilities. SPAG - whose concept had been sketched out several years ago (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Joost 2006</a>; page 138) - can be used a) to predict the probability of presence of one or many adaptive variants in non-sampled areas b) identify areas where there is a higher probability to find individuals better adapted to future climatic conditions, c) identify vulnerable populations that may be threatened by climate change and d) integrate the results into conservation framework by means of an easy combination with other georeferenced layers. We first introduce the theoretical bases of SPAGs and present an application of the approach to analyse the local adaptive potential of Moroccan and European goats’ populations. We apply the methodology to two independent datasets and validate the result using training and testing samples.</p></div><div class="section" id="sec-2"><h2 class="">Material and Methods</h2><div id="sec-3" class="subsection"><h3>Genetic data</h3><p id="p-9">Two genetic datasets characterising goat individuals <em>(Capra hircus)</em> were used for the analyses presented here. The first one was produced in the context of the NEXTGEN project (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alberto <em>et al.</em>, 2018</a>) and the second was collected by the ADAPTMAP consortium (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Stella <em>et al.</em>, 2018</a>; <a href="http://www.goatadaptmap.org/">http://www.goatadaptmap.org/</a>).</p><p id="p-10">The NEXTGEN consortium produced whole genome sequences data for 161 Moroccan goats from 6 different local breeds. Since goat production system in Morocco is mainly free-range, those goats are living from 8 to 12 months outdoors (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Boujenane, 2005</a>), and are confronted to contrasting environmental conditions, from the Sahara desert to the Atlas Mountains (see Figure A in Supp. File 1). The sequencing method is described by <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Benjelloun et al. (2015)</a> and the final sequences contain 31.8 M of SNPS mapped to the goat’s reference genome CHIR v1.0 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dong <em>et al.</em>, 2013</a>).</p><p id="p-11">The ADAPTMAP consortium gathered genetic data for 4’563 goats from 144 breeds, sequenced worldwide with the CaprineSNP50 BeadChip and mapped on the most recent goat’s reference genome ARS1 (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bickhart <em>et al.</em>, 2017</a>). We used here a subset of these data, constituted of individuals from Switzerland, North of Italy and France. This represented 458 individuals distributed in 196 locations, with 1 to 39 individuals per site. In order to avoid overweighting of some locations, we selected a maximum of five individuals per sampling site. Those five individuals were chosen such as the subset keeping the highest Nei’s genetic distances, computed with the function <em>dist.genpop</em> from the package <em>adegenet</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Jombart, 2008</a>) in the R environment (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">R Development Core Team, 2008</a>). The resulting dataset contains 382 individuals from 196 locations and 11 different breeds (see Figure B in Supp. File 1).</p><p id="p-12">Both genetic datasets were filtered such as to keep only autosome, bi-allelic SNPs, with a maximum missingness per individuals and per site of 0.05, a minimum minor allele frequency of 0.1 and a maximum major genotype frequency of 0.9. The final datasets contain 8’497’971 SNPs for the Moroccan goats and 46’294 SNPs for the European ones.</p></div><div id="sec-4" class="subsection"><h3>Environmental data</h3><p id="p-13">The climatic conditions of the sampling locations were characterised using the 19 bioclimatic variables (Supp. File 2) from the WorldClim database, representative of the period 1960-1990 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hijmans et al., 2005</a>). Each variable was retrieved as a raster layer with a spatial resolution of 30 arc-seconds (approx. 1km2) and values were extracted for all sampling locations using the <em>extract</em> function from the R-package <em>raster</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hijmans and van Etten, 2012</a>).</p><p id="p-14">In order to get similar ranges of values for all bioclimatic variables, which makes it easier to compare the subsequently derived models, all variables were standardised for each dataset, by subtracting the mean and dividing by the standard deviation. Some of the bioclimatic variables are highly correlated. However, we choose to keep all of them to be able to identify a posteriori which variable had the strongest effect. Since no models computed involved more than one environmental variable simultaneously, this collinearity will not impact the results.</p></div><div id="sec-5" class="subsection"><h3>Population Structure</h3><p id="p-15">The genetic population structure was estimated with a Principal Component Analysis (PCA) computed with the function <em>snpgdsPCA</em> from the <em>SNPRelate</em> R-package (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Zheng <em>et al.</em>, 2012</a>). In order to avoid a strong influence of SNP clusters on this analysis, we used here a pruned set of SNPs that are in approximate linkage equilibrium with each other. The pruning was performed with the function <em>snpgdsLDpruning</em> from the <em>SNPRelate</em> package, with a threshold D’=0.2. The resulting datasets contain 59’224 SNPs for the Moroccan goats and 14’571 SNPs for the European ones.</p></div><div id="sec-6" class="subsection"><h3>Logistic regressions</h3><p id="p-16">The SAM method (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Joost <em>et al.</em>, 2007</a>) was used to detect genotypes that are strongly associated with an environmental variable and are therefore potential adaptive variants. This method uses logistic regressions (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">formula 1</a>) to assess the probability of presence of a genotype <em>p(G1)</em> as a function of an environmental variable (x1),
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-1.gif"/></noscript></span>

</span>
where β0 and β1 are the parameters of the regression to be fitted.</p><p id="p-17">Such models were computed for each genotype with the 19 bioclimatic variables. The statistical significance of the model was assessed using Wald test and G score (log-likelihood ratio), both corrected for the false-discovery rate due to multiple comparisons using the procedure proposed by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Benjamini and Hochberg (1995)</a>, under an expected false discovery rate (FDR) of 0.01 or 0.05 (i.e. 1% or 5% of the results expected to be false positives).</p><p id="p-18">In order to lower the number of false positive resulting from demographic processes instead of natural selection (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Li <em>et al.</em>, 2012</a>), logistic models were computed with the addition of covariates corresponding to the coordinates of individuals on the significant components of the PCA (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Stucki <em>et al.</em>, 2017</a>). The significance of the models with population covariates was assessed using a Wald test and a log likelihood ratio which compares the model with environment and covariates to the model with covariates only. An association was considered as significant if both the models without covariates and with population covariates were significant.</p><p id="p-19">Finally, to identify potential functions of the SNPs involved into the significant associations, we used the NCBI Genome Browser to search for the presence of annotated genes in the genomic region of 10kbp surrounding the SNPs of interest.</p><p id="p-20">All analyses were computed using a combination of the Samβada software (<a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Stucki <em>et al.</em>, 2017</a>) and an in-house R-script based on the <em>glm</em> function.</p></div><div id="sec-7" class="subsection"><h3>SPatial Areas of Genotype Probability (SPAG)</h3><p id="p-21">With the β parameters of the logistic regressions previously computed (<a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">formula 1</a>), we can estimate the probability of presence of a genotype for any value of an environmental variable (x1). We consequently used them to estimate and delimit on a map the probability of presence of a genotype over the whole region of interest (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Joost, 2006</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Rochat <em>et al.</em>, 2016</a>). We named such a delimited surface <em>Spatial Area of Genotype Probability (SPAG).</em></p><p id="p-22">As more than one adaptive variant are usually identified, we also developed multivariate models to compute a single map showing the probability of presence of multiple genotypes. Three different multivariate models were developed to date: the Intersection, Union and K-Percentage.</p><p id="p-23">The <strong>Intersection</strong> model (I-SPAG) is used to compute the probability that the variants of interest are all simultaneously present. Following the theory of conditional probability (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Kolmogorov, 1956</a>), the probability of simultaneous presence of <em>n</em> genotypes <em>Gi, i=1:n</em> can be computed using <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">formula 2</a>
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-2.gif"/></noscript></span>

</span>
where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/inline-graphic-1.gif"/></noscript></span></span> is a conditional probability that can be estimated using a logistic regression where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/inline-graphic-2.gif"/></noscript></span></span> is integrated as a covariate (<a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">formula 3</a>)</p><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-3.gif"/></noscript></span>

</span><p id="p-24">However, as we would like to use this model to predict the probability of presence of the genotypes for any point of the region of interest, i.e. also where <em>Gi</em> values are unknown, we suggested to estimate <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/inline-graphic-3.gif"/></noscript></span></span>, which enables the implementation of <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">formula 3</a> with a recursive model using the univariate formula (see Supp. File 3 for more details).</p><p id="p-25">The <strong>union</strong> model (U-SPAG) is used to compute the probability that at least one of the adaptive variants is present. We implemented it with the inclusion-exclusion principle (<a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">formula 4</a>), using an in-house R script based on the intersection model previously described.</p><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-4.gif"/></noscript></span>

</span><p id="p-26">Finally, we developed a <strong>K-percentage</strong> model (K-SPAG) to obtain the probability that an individual carries K% of <em>n</em> adaptive variants. This probability can be computed by combining formulas from the union and intersection models (<a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">formula 5</a>).</p><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/12/20/2019.12.20.884114/embed/graphic-5.gif"/></noscript></span>

</span><p id="p-27">All formulas are explained in more details in Supp. File 3. Note that all multivariate models allow the integration of adaptive genotypes associated with various environmental variables since the environmental variable <em>xi</em> used to compute <em>p(Gi)</em> can be different for each <em>i.</em></p></div><div id="sec-8" class="subsection"><h3>Validation of the models</h3><p id="p-28">To test the validity of the models proposed, we selected only 25% of the individuals to compute the SPAG and used the remaining 75% to test it. In order to capture the whole variability of the environmental variable in the training set and thus increase the predictive power of the logistic regression, the range of environmental values was divided into intervals and training individuals were randomly selected within each interval. This selection method is presented in details in Supp. File 3. The testing dataset was then used to validate the result by comparing the genotype frequency predicted by the SPAG with the observed genotype frequency among the testing individuals. We plot the results with graphs as presented in Box 1. We repeated the validation procedure 10 times, with various random selection of training samples in order to compute 95% confidence intervals.</p><div class="boxed-text" id="boxed-text-1"><div id="sec-9" class="subsection"><h4>Box 1: Validation Procedure</h4><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1258757115" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F1.medium.gif" width="440" height="183"/><noscript><img class="highwire-fragment fragment-image" alt="Figure1" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F1.medium.gif" width="440" height="183"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div></div><p id="p-29">For each validation value V (x-axis) between 0 and 1 with a step of 0.1, we delimited the area where the predicted genotype frequency is higher or equal to V (SPAG≥V). We then calculated the percentage of testing individuals situated within that area (solid grey line) and the observed genotype frequency among them (green line). We computed the same values outside that area, i.e. where SPAG &lt; V (dashed grey line and red line). If the model is valid, the observed frequency inside the SPAG≥V (green line) should effectively be higher or equal to V and it should be lower than V outside (red line). The black line indicates the limit case where the observed genotype frequency is equal to the predicted probability V. The model is therefore valid if the green line remains above the black line and the red line remains below it. In the example above, for a testing value V=0.4, the SPAG is validated since the observed genotype frequency within the SPAG≥0.4 is higher than 0.4 (green line) and is lower than 0.4 outside (red line). Inversely, the model is not validated for V=0.6, since the percentage of presence inside the SPAG≥0.6 is lower than 0.6 (the green line falls under the black line). The model is also not valid for a value V=0.2 since the genotype frequency outside the SPAG≥0.2 is higher than 0.2 (the red line goes above the black line). The green and red areas around the lines indicates the 95% confidence intervals for each line, computed on the basis of the 10 runs with different random selection of training datasets. Finally, the hatched grey areas indicate ranges of testing values where there was less than 5 individuals inside or outside the SPAG, which was therefore considered not to be significant for the validation.</p></div></div></div><div id="sec-10" class="subsection"><h3>Projections under climate change</h3><p id="p-30">In order to predict the genotype frequency optimal for future conditions, we retrieved Worldclim data for the year 2070, corresponding to a strong climate change scenario from the couple Max Planck Institute Earth System Model (MPI-ESM-LR) with a Representative Concentration Pathway equals to 8.5 (RCP 8.5). We then assume that the optimal genotype frequency for future conditions should be close to the genotype frequency currently observed in areas with climatic conditions resembling the future ones. We thus applied the current parameters of the logistic regressions on the future environmental variables in order to derive the future SPAGs for the genotypes of interest. We then study the shift between the current and future SPAG to identify vulnerable populations for which specific genotype frequencies should be much higher so that individuals can adapt to the future conditions.</p></div></div><div class="section" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Population structure</h3><p id="p-31">For the Moroccan dataset, the cumulated variance explained by the 10 first PCA components represents only 8.1% of the total variance and the increase in variance explained is almost proportional to the number of components, which highlights that there is no clear sub-structure. We therefore do not include any population structure on the subsequent analysis and computed only logistic regressions without any covariates.</p><p id="p-32">For the European goats, the first component of the PCA explains 6.2% of the total variance when the second, third and fourth components explain 2.0%, 1.7% and 1.6% respectively. The low variance explained by each PCA component indicates no clear population structure. However, since the variance explained by the first component is much higher than what explained by the next ones, it is possible that the first component is partially related to population structure. We therefore computed logistic regressions without covariates and then logistic regressions with a covariate corresponding to the coordinates of goat individuals on the first component of the PCA.</p></div><div id="sec-13" class="subsection"><h3>Logistic Regressions</h3><p id="p-33">For the Moroccan dataset, more than 483 million of association models were computed. After correction for false discovery rate with a significant threshold of 5%, none models are significant according to the Wald score but seven models are significant according to the G score (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Three of them indicated an association of SNPs on chromosome 1 with the mean diurnal range (bio2), three others relate SNPs on chromosome 6 to the coefficient of precipitation (bio15) and one to the precipitation of the driest quarter (bio17). None of the SNPs identified correspond to any annotated gene on the goat assembly CHIR 1.0.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1071940/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1071940/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1071940" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-34" class="first-child">Significant models obtained for the analysis of Moroccan datasets, considering all bioclimatic variables, after FDR correction with a threshold of 5%. Chr=Chromosome, BP=Position in base pairs, GF=Genotype Frequency, β0 and β1 = parameters of the logistic regression, G=G score (Log Likelihood ratio) of the model, W=Wald score of the model, qG= p-value of G score corrected for FDR</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">We then investigated in more details the adaptation to the precipitation seasonality (bio15), which is a measure of the variation of monthly precipitation over the year. This variable shows a high variability over the sampling area, with values from 25% to 50% in the Atlas Mountains and the northeast of Morocco (i.e. the standard deviation of monthly precipitation is equal to 25%-50% of the mean of monthly precipitation), values higher than 70% in the coastal areas, and a maximum of more than 100% close to the Sahara. When considering only the associations involving this bioclimatic variable (25’447’348 models), 78 models are significant after FDR-correction of G score, with a significant threshold of 5% (Supp. File 4). Those SNPs are located on seven different genomic regions (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), corresponding to four annotated genes on the reference genome CHIR 1.0: DSG4, CDH2, KCTD1 and WRN. The peak on chromosome 6, containing 33 significant models, does not correspond to any annotated gene. However, two SNPs from the CaprineSNP50 BeadChip fall within this genomic region when mapped on CHIR1.0 and are situated within a long intergenic noncoding RNA (LincRNA) when mapped on the new genome assembly ARS1.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1071942/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1071942/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1071942" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-36" class="first-child">Significant models obtained for the analysis of Moroccan datasets with precipitation seasonality (bio15) after FDR correction. Chr=Chromosome, Start=Start in base pairs of the region identified as under selection, End=End in base pairs of the region, Peak SNP = SNP of the most significant model on that region, Geno = corresponding Genotype, GF=corresponding Genotype Frequency, β0 and β1 = parameters of the logistic regression, G=G score (Log Likelihood ratio) of the model, qG=corresponding p-value corrected for FDR, Genes = Annotated genes on the genomic region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">For the European dataset, 2’638’758 associations were computed, among which 44’191 models (1.67%) were significant both without covariate and with the first PCA-component as covariate, according to both G score and Wald score corrected for a false positive rate of 1% (Supp. File 5). The fifteen models with the strongest G scores when computed without covariate correspond to 10 genomic regions (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>), associated with three bioclimatic variables related to precipitation (bio13 = precipitation of wettest month, bio16 = precipitation of wettest quarter, bio18 = precipitation of warmest quarter) and two bioclimatic variables related to temperature (bio3 = isothermality, bio8 = mean temperature of wettest quarter). Seven annotated genes correspond exactly to one of the SNPs identified: KRT12, CSN1S2, CACNB, PRDM5, LOC102174324, PALM and NAV3.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1071943/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1071943/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1071943" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <p id="p-38" class="first-child">Fifteen most significant models (based on G score of the model without covariate) obtained for the analysis of the European dataset, considering all bioclimatic variables, after FDR correction with a threshold of 1% for both G and Wald Score. Chr=Chromosome, BP=Position in base pairs, GENO=Genotype of interest, GF=Genotype Frequency, qGpop (resp. qWpop) = FDR-corrected p-values of Gscore (resp. Wald score) of the model with the first PCA-component as covariate, qG0 (resp. qW0) = FDR-corrected p-values of Gscore (resp. Wald score) of the models without any covariate, β0 and β1 = parameters of the logistic regression without covariate, Genes = Annotated genes on the genomic region.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>SPatial Areas of Genotype Probability</h3><p id="p-39">Spatial Areas of Genotypes probability were computed for the models presented on <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">tables 2</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">3</a>. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 1</a> show examples of results obtained for the Moroccan goats and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 2</a> for the European ones. Univariate results for all the other models from <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">tables 2</a> and <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">3</a> are available in Supp. File 6.</p><p id="p-40">For the Moroccan dataset, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 1A</a> corresponds to the univariate SPAG for the genotype of model M1 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>) that is mainly carried by goats living close to the coast and the Sahara, where high variations of precipitation can be observed. The predicted genotype frequency is the highest in the extreme southwest of the country where the variations of precipitation are the most important and all goats carry this marker. This is not the case in coastal areas, where goats without the variant are living close to goats having it, the predicted genotype frequency in that area being coherently close to 0.5. In the Atlas Mountains and northeast of the country, the predicted variant frequency is much lower (&lt;0.2 in most areas) which corresponds to the observation of an absence of the genotype in most goats sampled in this region. Two other markers positively correlated with bio15 were highlighted by models M3 and M5 (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). However, the simultaneous presence of those two genotypes is very unlikely, as indicated by the I-SPAG (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 1B</a>) showing a predicted frequency lower than 0.1 for most part of the territory. Indeed, although those two markers are similarly associated with the environment, only 11% of the sampled goats carry them both. Nevertheless, both variants are absent in the Atlas Mountains and the northeast of the country where variations of precipitation are the lowest, what explains their similar positive correlation with bio15. When considering the probability to find at least one of the genotypes from the three models M1, M3 and M5 all positively associated with the coefficient of precipitation, the U-SPAG (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 1C</a>) predicts a very high probability in many parts of the territory. Indeed, 78% of all goats carry at least one of those three genotypes and only some goats living in the northeast of the country have none of them. Finally the K-SPAG presented on <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 1D</a> shows the probability that goats carry at least 50% of the four variants positively associated with the coefficient of precipitation (M1, M2, M3, M5), i.e. the probability to find at least two of those genotypes. This map is the most contrasted one, with a very high probability of presence close to the coast and Sahara (&gt; 0.9) and a very low frequency (&lt;0.2) in the centre and northeast of the country.</p><p id="p-41">For the European dataset, the univariate SPAG presented (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 2A</a>) corresponds to a genotype negatively associated with the precipitation of the warmest quarter (bio18). This variable ranges from less than 100 mm in the Mediterranean border to more than 500 mm in the Swiss Alps, with values between 250 and 300 mm in the Jura, the French Alps and the Swiss Plateau, and between 150 and 200 mm in most of other parts of France and of Northern Italy. Consequently, the predicted genotype frequency is the lowest in the Swiss Alps (&lt;0.1), slightly higher in Jura, French Alps and Swiss Plateau (between 0.2 and 0.4) and higher than 0.5 everywhere else, with a maximum around 0.8 in the Mediterranean border. We then computed the I-SPAG (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 2B</a>) of this same variant (model E10) with the model E1d corresponding to a genotype AA associated with low precipitation in the wettest month (bio13). For this last model, the genotype GG of the same SNP shows the strongest positive association with bio18 (model E1a). The two models E1d and E10 may therefore correspond to a similar adaptation to low values of precipitation during the warmest quarter. However, the I-SPAG indicates that their simultaneous presence is not very likely (predicted frequency &lt; 0.6 everywhere). Indeed, 23% of the total goats carry those two genotypes, but they are essentially localised in areas where other goats carry only one of the two markers. The predicted probability is the highest in the centre-north of France (regions Centre, Iles de France, East of Pays de la Loire, Normandie and South of Hauts de France, see Sup. File 1 for regions’ map) and in the southern part (Occitanie and West of Provence), whereas in the Alps, Jura and most part of Switzerland, no goats carry the two genotypes and the predicted frequency is close to 0. The probability to find at least one of those two variants, presented on the U-SPAG in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 2C</a>, indicates a trend similar to the probability of presence of E10 alone (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 2A</a>), but with even stronger contrast between the Alps-Jura-Switzerland area (frequencies &lt; 0.3) and the rest of the territory (frequencies &gt; 0.7). Finally the K-SPAG (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 2D</a>) shows the probability to find at least 50% of five genotypes negatively associated with the precipitation of the warmest quarter (bio18), i.e. the probability to find at least three of them. Note that for the models positively associated with bio18 in <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>, we used the alternative genotype that was the most significantly negatively correlated with bio18 (indicated after the model ID). The resulting SPAG is very close to the I-SPAG of E1d and E10 (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 2B</a>).</p><p id="p-42">For all cases presented, the validation graphs indicate that the SPAGs computed with 25% of the individuals generally correctly predict the genotype frequency of the 75% remaining individuals. However, for the Moroccan dataset, the U-SPAG (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 1C</a>) tends to slightly overestimate the probability of presence since the observed genotype frequency outside the SPAG (red line) tends to be slightly higher than the predicted value (black line).</p></div><div id="sec-15" class="subsection"><h3>Projections under climate change</h3><p id="p-43"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 3</a> shows the differences between the current SPAGs presented in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figures 1</a> and <a id="xref-fig-3-8" class="xref-fig" href="#F3">2</a> and their corresponding projections for 2070. In Morocco, the precipitation seasonality (bio15) is predicted to increase in the northwest of the country, with a maximum increase of 5 to 10 in the extreme northwestern region (Tangier-Tetouan, see region’s map in Supp. File 1) and to decrease in other areas, especially in the Atlas Mountains and close to the Sahara (from −10 to −20). The evolution of the univariate SPAG for model M1 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 3A</a>) consequently indicates the highest risk in the Tangier-Tetouan area, where the genotype frequency should be approximately 20% higher to be well adapted to future conditions. However, many individuals in this area already carry the favourable genotype and the risk for the population may therefore be reduced thanks to natural gene flow. Nevertheless, this is not the case in the southwest of that area (Rabat, Casablanca) where the genotype frequency should be 10-20% higher and none of the goats sampled there currently carry the adaptive variant. Similar observations can be made as regards the I-SPAG of M3 and M5 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 3B</a>), two other markers that potentially confer an adaptation to high variations of precipitation. However, the U-SPAG (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 3C</a>) highlights no vulnerable areas, which indicates that if the presence of at least one of the adaptive variants is sufficient to enable the adaptation to high variations of precipitation, no populations are threatened. Finally the K-SPAG (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 3D</a>) shows once again a risk area in the northwest of the country, where the probability to carry at least 50% of the adaptive variants should be approximately 20% higher. Again, individuals in the most northern part of this risk area may be less threatened thanks to the close presence of goats that already carry the favourable genotypes, whereas the population from the Rabat-Casablanca area may be more threatened due to the current much lower presence of the adapted variants.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Univariate and Multivariate Spatial Areas of Genotypes Probability for the Moroccan dataset. The identifiers of the presented models (M1, M2, M3, M5) refers to Table 2. The maps show the average genotype(s) frequency(ies) based on the 10 runs computed with different random selection of training sets containing 25% of the total number of individuals. Please refer to Box 1 to interpret the validation graphs shown on the right of each map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1258757115" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Univariate and Multivariate Spatial Areas of Genotypes Probability for the Moroccan dataset. The identifiers of the presented models (M1, M2, M3, M5) refers to Table 2. The maps show the average genotype(s) frequency(ies) based on the 10 runs computed with different random selection of training sets containing 25% of the total number of individuals. Please refer to Box 1 to interpret the validation graphs shown on the right of each map.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F2.medium.gif" width="300" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F2.medium.gif" width="300" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-44" class="first-child">Univariate and Multivariate Spatial Areas of Genotypes Probability for the Moroccan dataset. The identifiers of the presented models (M1, M2, M3, M5) refers to <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>. The maps show the average genotype(s) frequency(ies) based on the 10 runs computed with different random selection of training sets containing 25% of the total number of individuals. Please refer to Box 1 to interpret the validation graphs shown on the right of each map.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Univariate and Multivariate Spatial Areas of Genotypes Probability for the European dataset. The identifiers of the presented models (E10, E1d, E2, E3b, E5a, E7) refers to Table 3. The maps show the average genotype(s) frequency(ies) based on the 10 runs computed with different random selection of training sets containing 25% of the total number of individuals. Note that since up to five individuals can be localised on the same site, a black dot indicates a presence if at least 50% of the individuals of the site carry the marker(s). Please refer to Box 1 to interpret the validation graphs shown on the right of each map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1258757115" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Univariate and Multivariate Spatial Areas of Genotypes Probability for the European dataset. The identifiers of the presented models (E10, E1d, E2, E3b, E5a, E7) refers to Table 3. The maps show the average genotype(s) frequency(ies) based on the 10 runs computed with different random selection of training sets containing 25% of the total number of individuals. Note that since up to five individuals can be localised on the same site, a black dot indicates a presence if at least 50% of the individuals of the site carry the marker(s). Please refer to Box 1 to interpret the validation graphs shown on the right of each map.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F3.medium.gif" width="302" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F3.medium.gif" width="302" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-45" class="first-child">Univariate and Multivariate Spatial Areas of Genotypes Probability for the European dataset. The identifiers of the presented models (E10, E1d, E2, E3b, E5a, E7) refers to <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>. The maps show the average genotype(s) frequency(ies) based on the 10 runs computed with different random selection of training sets containing 25% of the total number of individuals. Note that since up to five individuals can be localised on the same site, a black dot indicates a presence if at least 50% of the individuals of the site carry the marker(s). Please refer to Box 1 to interpret the validation graphs shown on the right of each map.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Predicted SPAG difference for 2070 considering the MPI-ESM-LR climate change scenario with RCP 8.5, for the Moroccan (left) and European (right) datasets. The identifiers of the presented models (M1, M2, M3, M5, E10, E1d, E2, E3b, E5a, E7) refer to Table 2 (Moroccan dataset) and Table 3 (European dataset). The maps show the average difference in genotype(s) frequency(ies) based on the 10 runs computed with different random selection of training sets containing 25% of the total number of individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1258757115" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Predicted SPAG difference for 2070 considering the MPI-ESM-LR climate change scenario with RCP 8.5, for the Moroccan (left) and European (right) datasets. The identifiers of the presented models (M1, M2, M3, M5, E10, E1d, E2, E3b, E5a, E7) refer to Table 2 (Moroccan dataset) and Table 3 (European dataset). The maps show the average difference in genotype(s) frequency(ies) based on the 10 runs computed with different random selection of training sets containing 25% of the total number of individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F4.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F4.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/12/20/2019.12.20.884114/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-46" class="first-child">Predicted SPAG difference for 2070 considering the MPI-ESM-LR climate change scenario with RCP 8.5, for the Moroccan (left) and European (right) datasets. The identifiers of the presented models (M1, M2, M3, M5, E10, E1d, E2, E3b, E5a, E7) refer to <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a> (Moroccan dataset) and <a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a> (European dataset). The maps show the average difference in genotype(s) frequency(ies) based on the 10 runs computed with different random selection of training sets containing 25% of the total number of individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">For the European dataset, the model E10 presented in <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 2A</a> was associated with low precipitation of the warmest quarter (bio 18), which is predicted to decrease from 20 to 180 mm over the whole study area until 2070. The predicted precipitation loss is maximum in the Alps (−120 to −180), in the Mediterranean coast (−100 to −130) and in the centre-south of France (West of Auvergne and East of Nouvelle Acquitaine, −90 to −120 mm, see region’s map in Supp. File 1). As a consequence, the evolution of the univariate SPAG for model E10 (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 3E</a>) indicates that all populations may be vulnerable, since the genotype frequency should be 10 to 20% higher everywhere. Even though the potentially adaptive genotype is already present in most French goats (75% carry it), this is not the case for much of the goats populations from the Alps and Switzerland that currently lack the adaptive variant and may be particularly vulnerable. When considering the I-SPAG of E10 and E1d (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 3F</a>), a higher risk is highlighted in the centre-south of France (West of Auvergne and its surrounding), northeast of France (Alsace in East of Grand-Est), Swiss Plateau, and northwest of Italy. Our dataset does not contain any goat sampled in Alsace and northwest of Italy. However, in centre-south of France and Swiss-Plateau, the genotypes are almost never simultaneously present in the goats sampled and the populations may therefore be threatened. The U-SPAG of the same genotypes (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 3G</a>) shows results very similar to the univariate SPAG for E10 alone (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 3E</a>). If the presence of at least one of those two genotypes may be sufficient to allow an adaptation to low precipitation, the region the most threatened will then once again be the Alps and Switzerland, where the two markers are currently absent. Finally, the evolution of the K-SPAG (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 3H</a>) shows a high risk in a large part of France and north of Italy, where the genotype frequency should be more than 20% higher. The risk is particularly high in the centre and south of France (Auvergne, Occitanie) and populations from Auvergne may be particularly threatened due to the current low genotype frequencies on the goats sampled there.</p></div></div><div class="section" id="sec-16"><h2 class="">Discussion</h2><div id="sec-17" class="subsection"><h3>Signature of adaptive potential</h3><p id="p-48">In Morocco, very few models are significant when considering all bioclimatic variables, which may be due to the huge number of associations computed, resulting in a strong FDR-correction. When considering the precipitation seasonality only, we identified seven genomic regions potentially under selection and corresponding to four annotated genes. Among them, DSG4 and KCTD1 may be related to the development of hair properties. <strong>DSG4</strong> is involved in the creation of a desmosomal cadherin found in epidermal and hair follicles cells (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Kljuic <em>et al.</em>, 2003</a>) and has been reported to play a role in the development of wool properties in sheep (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ling et <em>al.</em>, 2014</a>) and in the regulation of keratin in goats (Wang et <em>al.</em>, 2017; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Zhang et <em>al.</em>, 2019</a>). <strong>KCTD1</strong> codes for a protein interacting with the β-catenin involved in the regulation of the WNT signalling pathway (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Li <em>et al.</em>, 2014</a>), which acts in the development of hair follicle in animals (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Galbraith, 2010</a>). Another gene identified, the <strong>N-cadherin</strong> (CDH2), may be related to skin properties since it encodes for a protein which mediates cell-cell adhesion and may interact with melanocytes (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hayashi <em>et al.</em>, 2007</a>). Those results suggest that goats confronted with high variations of precipitation may have developed an adaptation related to hair or skin properties, which could for example ensure a better water repulsion. In addition, the genomic region containing the highest number of significant associations (Supp. File 3) is located on chromosome 6, in a genomic area corresponding to a long intergenic noncoding RNAs (LincRNA) on the new assembly ARS1. Although we do not have any indication regarding the function of this LincRNA, intergenic non-coding RNA have also been reported to be implicated in hair follicle growth or skin pigmentation in goats (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Ren <em>et al.</em>, 2016</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Wang <em>et al.</em>, 2017</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ge <em>et al.</em>, 2018</a>). The potential adaptive function of the last gene identified, <strong>WRN</strong>, is less evident since this gene codes for a protein associated with the Werner syndrome responsible for premature and accelerated aging (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Goto, 1997</a>). Since this disease notably includes an early greying of the hair, the WRN gene could interact with some actors of the hair development, but it may also likely be a false positive.</p><p id="p-49">For the European dataset, the number of significant models was much higher, which may be explained by 1) a less strong FDR-correction due to a lower number of associations computed, 2) the pre¬selection of SNPs with a chip purposely chosen to represent selected traits and 3) a larger study area with a higher number of different breeds analysed. From the fifteen most significant models, seven candidate genes for natural selection were identified, among which one LOC gene of unknown function, associated with the precipitation of warmest quarter and thus potentially conferring an adaptation to drought. Two of the other genes are related to properties of the cornea: <strong>KRT12</strong> is involved in the creation of a protein forming molecules of the corneal epithelium (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Kao <em>et al.</em>, 1996</a>) and <strong>PRDM5</strong> is implicated in the regulation of the fibrillary collagen needed for the maintain of the cornea resistance (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Burkitt Wright <em>et al.</em>, 2011</a>, p. 5). This last gene may also confer an adaptive advantage to pigs living in high altitudes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ai <em>et al.</em>, 2014</a>). The SNPs corresponding to those genes were significantly associated with the precipitation of the warmest quarter and the mean temperature of the wettest quarter respectively. They may therefore highlight an adaptation to drought conditions that can be correlated with higher UV-radiation potentially influencing cornea properties. Among the other genes identified, <strong>CSN1S2</strong> is related to the casein content of the milk (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Ramunno <em>et al.</em>, 2001</a>) and has already been reported as a gene potentially under selection in goats (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Pariset <em>et al.</em>, 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bertolini <em>et al.</em>, 2018</a>). The SNP associated with this gene was correlated with the isothermality (bio3) and the precipitation of the warmest month (bio18) and may therefore relate to a specific property of milk resulting either from an adaptation to drought conditions or to an artificial selection. Another identified gene <strong>CACNB2</strong> was also previously identified as a potential candidate for selection in goats (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Brito <em>et al.</em>, 2017</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Mdladla <em>et al.</em>, 2018</a>). This gene is involved in the calcium channel and energy pathway and may confer an advantage for adaptation to cold temperature in Siberian human populations (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cardona <em>et al.</em>, 2014</a>). It corresponds here to a variant negatively correlated with the mean temperature of the wettest quarter and may therefore also confer an adaptive potential for goats confronted with cold habitat. Finally, the two last genes identified may be related to skin properties. The <strong>PALM</strong> gene codes for a protein involved in the cell formation and potentially related to pigmentation (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Kutzleb <em>et al.</em>, 1998</a>), whereas, <strong>NAV3</strong> may be involved in the development of cutaneous T-cells lymphoma (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Karenko <em>et al.</em>, 2005</a>, p. 3).</p><p id="p-50">Many of the genes highlighted on the two case studies may therefore be associated with a function that can be influenced by climate, which reinforces the potential that they are true signatures of local adaptation. However, although previous studies show the power of genotype-environment associations to predict phenotype (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Lasky <em>et al.</em>, 2015</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Vangestel <em>et al.</em>, 2018</a>) or fitness (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fournier-Level <em>et al.</em>, 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hancock <em>et al.</em>, 2011</a>), more investigations are needed to verify that the variants identified are really conferring an adaptive advantage and that they are heritable (Funk et <em>al.</em>, 2019). Moreover, adaptation process is complex and often involves polygenic traits (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Pritchard and Rienzo, 2010</a>), for which the detection power of the genotype-environment associations may be much reduced (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Villemereuil et <em>al.</em>, 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Harrisson et <em>al.</em>, 2014</a>). In that case, it may be advisable to use multivariate genotype-environment association models (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Forester et <em>al.</em>, 2017</a>) or to integrate other methods to identify SNPs related to polygenic adaptation (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Zhou et <em>al.</em>, 2013</a>; Lasky et <em>al.</em>, 2015).</p></div><div id="sec-18" class="subsection"><h3>From SPAG to conservation</h3><p id="p-51">The first utility of the SPAGs is to quantify the current probability to find beneficial alleles or the expression of favourable traits in plant and animal populations, even in regions where no individuals have been sampled. Our results show that with few training individuals (i.e. 40 goats in Morocco and 120 in Europe), a good estimation of the genotype frequency is possible, for one or many genotypes of interest. The univariate models presented here were already applied to map the genotype frequencies of adaptive variants of the Scandinavian brown bears (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Joost, 2006</a>), Moroccan sheep (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Rochat et al., 2016</a>) and coral reefs from the Ryukyu Archipelago (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Selmoni et al., 2019</a>). The multivariate models are presented here for the first time and according to the validation procedure applied, they appear to be powerful in estimating the combined probability to find many genotypes potentially correlated with different environmental variables. With the I-SPAG, the resulting probabilities may rapidly become very low, but this model could be used when we suspect that the simultaneous presence of some adaptive genotypes is needed to ensure the adaptation or when we would like to highlight the probability to find simultaneously variants that may confer an adaptive response to different environmental variables (for example low precipitation and high temperature). At the other extreme, the U-SPAG may rapidly indicate high probabilities of presence in most parts of the territory, but it can be used when suspecting that the presence of at least one of the variants may be sufficient to confer the adaptive potential. Since it is usually difficult to know if the simultaneous presence of variants is needed or if an union is sufficient, the K-SPAG offers an interesting compromise, allowing the identification of populations that retain a given percentage of variants, which may allow the delimitation of areas where there is the highest probability to find goats with a high adaptive potential.</p><p id="p-52">On the other hand, the study of the shift in SPAG under climate change conditions can help identify 1) well-adapted populations, where individuals currently show adaptive variants that seem to be optimal under future conditions, 2) populations at risk where the current genotype frequency is not optimal, but where the favourable variants are already present in the population, thus potentially allowing a natural increase of the genotype frequency through gene flow and 3) threatened populations where optimal variants are currently missing but would be needed in order to ensure an adaptation to future climate. Those identifications may be of high utility in conservation planning. Indeed, when priorizing areas for conservation, the success may be increased by choosing to preserve preadapted individuals that already carry functional variants conferring a good adaptation to future climate (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Orr and Unckless, 2008</a>). Moreover SPAG can also be used to prevent translocation of individuals that currently do not carry the variants favourable for the future conditions of the target site, which would result in a reduction or loss of the adaptive potential of target populations (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Weeks et <em>al.</em>, 2011</a>). In addition, conservation plans may be developed to increase the survival capacity of threatened populations. This can imply assisted gene flow to import the adaptive variants in a population where they are missing (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aitken and Whitlock, 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Kelly and Phillips, 2016</a>) or artificial selection of individuals already pre¬adapted to future conditions (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Hoffmann, 2010</a>). However, this has to be undertaken carefully since the selection of locally adapted individuals can result in a loss of genetic diversity (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Savage et <em>al.</em>, 2018</a>), which may decrease the potential of populations to adapt to new environmental changes. <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Kardos and Shafer (2018)</a> therefore proposed that gene-targeted conservation measures should be done only with traits affecting vital processes of the species and when phenotypic variations are large enough to ensure a high probability of success. Moreover, since the results of the shift under climate change may be highly dependent on the climate change scenario considered, computations should be made with various scenarios and less weight should be given to the conclusions not consistent within scenarios (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Reside et <em>al.</em>, 2018</a>). Finally, an analysis of connectivity should be done to highlight the potential of natural gene flow to increase the genotype frequency.</p><p id="p-53">The Moroccan case study highlighted that the goats’ populations from the surroundings of Rabat and Casablanca may lack adaptive variants potentially conferring an advantage to face high variation of precipitation. If the adaptive role of those genotypes is validated, the goat populations of this area may be threatened. Due to the high economic and social importance of goats in Morocco, it is crucial to preserve viable populations. Indeed, in this country, agriculture contributes to 12 to 24% of the national GDP and employs 40% of the total active population (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Boujenane, 2005</a>). Livestock, especially small ruminants, is the most important sector in agriculture and goat farms represent 20% of the total number of agricultural farms (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Boujenane, 2005</a>). It is therefore important to consider preserving or introducing the adaptive variant on each vulnerable population. This could be done for example by favouring crossbreeding with individuals from the southern or north-eastern part of the country, where the adaptive genotypes are currently well present and avoiding breeding or translocation with exotic goats or goats from the Atlas or Oriental areas. In the Northern part of Morocco (Tanger-Tetouan regions), goats population represent 12% of the national goats (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chentouf, 2014</a>) and play an important role to preserve food security (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Godber et <em>al.</em>, 2016</a>). In that area, crossbreeding with exotic breeds have been introduced to improve milk production (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Boujenane, 2005</a>; Godber et <em>al.</em>, 2016). However, our result show that the genotype frequency of adaptive variants should increase in the goat populations from this region and that it is therefore of main importance to maintain there local individuals that show the necessary adaptive variants.</p><p id="p-54">Finally, the SPAG maps could be integrated into decision frameworks considering the adaptive potential when defining the vulnerability of species (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bonin <em>et al.</em>, 2007</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Williams <em>et al.</em>, 2008</a>; <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Sgrò <em>et al.</em>, 2011</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dawson <em>et al.</em>, 2011</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Razgour <em>et al.</em>, 2018</a>) or in more global decision frameworks considering other factors of vulnerability such as predation level or habitat loss. Moreover, SPAG could also be used to predict the presence of genotype(s) associated with other pressures showing a spatial distribution, such as the presence of a parasite (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Vajana <em>et al.</em>, 2018</a>) or a predator (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cousyn <em>et al.</em>, 2001</a>) or the urbanization level (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Harris and Munshi-South, 2016</a>). Very similar models can also be derived to predict allele frequencies instead of genotypes frequencies or to integrate other covariates (for example to account for autocorrelation). SPAG therefore constitute a valuable tool to support conservation decisions, especially under current changing climatic conditions.</p></div></div><div class="section" id="sec-20"><h2 class="">Conflict of Interest</h2><p id="p-55">The authors declare that there is no conflict of interest regarding the publication of this paper.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.1" data-doi="10.1186/1471-2164-15-834"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ai</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Population history and genomic signatures for high-altitude adaptation in Tibetan pigs</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D834%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.2" data-doi="10.1146/annurev-ecolsys-110512-135747"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Assisted Gene Flow to Facilitate Local Adaptation to Climate Change</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">388</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D44%26rft.spage%253D367%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110512-135747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110512-135747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000329821800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.3" data-doi="10.1038/s41467-018-02974-x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alberto</span>  <span class="cit-name-given-names">FJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boyer</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Streeter</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Servin</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Villemereuil</span>  <span class="cit-name-given-names">P de</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Convergent genomic signatures of domestication in sheep and goats</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-02974-x%26rft_id%253Dinfo%253Apmid%252F29317637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-02974-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29317637&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.4" data-doi="10.1038/nrg2844"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">FW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Genomics and the future of conservation genetics</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">697</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D697%26rft.epage%253D709%26rft.atitle%253DGenomics%2Band%2Bthe%2Bfuture%2Bof%2Bconservation%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2844%26rft_id%253Dinfo%253Apmid%252F20847747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20847747&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">FW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leary</span>  <span class="cit-name-given-names">RF</span></span> (<span class="cit-pub-date">1986</span>). <span class="cit-article-title">Heterozygosity and fitness in natural populations of animals</span>. <abbr class="cit-jnl-abbrev">Conservation biology: the science of scarcity and diversity</abbr>: <span class="cit-fpage">57</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.6" data-doi="10.1111/j.1461-0248.2012.01796.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Banta</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">IM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerard</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilczek</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Climate envelope modelling reveals intraspecific relationships among flowering phenology, niche breadth and potential range size in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DBanta%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D769%26rft.epage%253D777%26rft.atitle%253DClimate%2Benvelope%2Bmodelling%2Breveals%2Bintraspecific%2Brelationships%2Bamong%2Bflowering%2Bphenology%252C%2Bniche%2Bbreadth%2Band%2Bpotential%2Brange%2Bsize%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2012.01796.x%26rft_id%253Dinfo%253Apmid%252F22583905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2012.01796.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22583905&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.7" data-doi="10.2307/2346101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y</span></span> (<span class="cit-pub-date">1995</span>). <span class="cit-article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</span>. <abbr class="cit-jnl-abbrev">Journal of the Royal Statistical Society: Series B (Methodological)</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BRoyal%2BStatistical%2BSociety%253A%2BSeries%2BB%2B%2528Methodological%2529%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2346101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2346101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QE45300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Benjelloun</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alberto</span>  <span class="cit-name-given-names">FJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Streeter</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boyer</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coissac</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stucki</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Characterizing neutral genomic diversity and selection signatures in indigenous populations of Moroccan goats (Capra hircus) using WGS data</span>. <abbr class="cit-jnl-abbrev">Front Genet</abbr> <span class="cit-vol">6</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bertolini</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Servin</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Talenti</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rochat</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">ES</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oget</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Signatures of selection and environmental adaptation across the goat genome post-domestication</span>. <abbr class="cit-jnl-abbrev">Genetics Selection Evolution</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%2BSelection%2BEvolution%26rft.volume%253D50%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.10" data-doi="doi.org/10.1038/ng.3802"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bickhart</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosen</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sayre</span>  <span class="cit-name-given-names">BL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Single-molecule sequencing and chromatin conformation capture enable de novo reference assembly of the domestic goat genome</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D49%26rft.spage%253D643%26rft_id%253Dinfo%253Adoi%252Fdoi.org%252F10.1038%252Fng.3802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi.org/10.1038/ng.3802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.11" data-doi="10.1111/j.1523-1739.2007.00685.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicole</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pompanon</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miaud</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taberlet</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">Population Adaptive Index: a New Method to Help Measure Intraspecific Genetic Diversity and Prioritize Populations for Conservation</span>. <abbr class="cit-jnl-abbrev">Conservation Biology</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">697</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BBiology%26rft.stitle%253DConservation%2BBiology%26rft.aulast%253DBonin%26rft.auinit1%253DA.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D697%26rft.epage%253D708%26rft.atitle%253DPopulation%2Badaptive%2Bindex%253A%2Ba%2Bnew%2Bmethod%2Bto%2Bhelp%2Bmeasure%2Bintraspecific%2Bgenetic%2Bdiversity%2Band%2Bprioritize%2Bpopulations%2Bfor%2Bconservation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1523-1739.2007.00685.x%26rft_id%253Dinfo%253Apmid%252F17531048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1523-1739.2007.00685.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17531048&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246843200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Boujenane</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Small Ruminant Breeds of Morocco</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brito</span>  <span class="cit-name-given-names">LF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kijas</span>  <span class="cit-name-given-names">JW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ventura</span>  <span class="cit-name-given-names">RV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sargolzaei</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Porto-Neto</span>  <span class="cit-name-given-names">LR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cánovas</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Genetic diversity and signatures of selection in various goat breeds revealed by genome-wide SNP markers</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.14" data-doi="10.1016/j.ajhg.2011.05.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burkitt Wright</span>  <span class="cit-name-given-names">EMM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spencer</span>  <span class="cit-name-given-names">HL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">SB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manson</span>  <span class="cit-name-given-names">FDC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeef</span>  <span class="cit-name-given-names">LAH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urquhart</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Mutations in PRDM5 in Brittle Cornea Syndrome Identify a Pathway Regulating Extracellular Matrix Development and Maintenance</span>. <abbr class="cit-jnl-abbrev">The American Journal of Human Genetics</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBurkitt%2BWright%26rft.auinit1%253DE.%2BM.%26rft.volume%253D88%26rft.issue%253D6%26rft.spage%253D767%26rft.epage%253D777%26rft.atitle%253DMutations%2Bin%2BPRDM5%2Bin%2Bbrittle%2Bcornea%2Bsyndrome%2Bidentify%2Ba%2Bpathway%2Bregulating%2Bextracellular%2Bmatrix%2Bdevelopment%2Band%2Bmaintenance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.05.007%26rft_id%253Dinfo%253Apmid%252F21664999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21664999&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.15" data-doi="10.1371/journal.pone.0098076"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cardona</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pagani</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antao</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eichstaedt</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yngvadottir</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Genome-Wide Analysis of Cold Adaptation in Indigenous Siberian Populations</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e98076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D9%26rft.spage%253De98076%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0098076%26rft_id%253Dinfo%253Apmid%252F24847810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0098076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24847810&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chentouf</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Systèmes de production caprine au Nord du Maroc: Contraintes et propositions d’amélioration</span>. <abbr class="cit-jnl-abbrev">Opt Médit A: Médit Semin</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpt%2BM%25E9dit%2BA%253A%2BM%25E9dit%2BSemin%26rft.volume%253D108%26rft.spage%253D25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.17" data-doi="10.1073/pnas.111606798"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cousyn</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Meester</span>  <span class="cit-name-given-names">LD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colbourne</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brendonck</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Verschuren</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Volckaert</span>  <span class="cit-name-given-names">F</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Rapid, local adaptation of zooplankton behavior to changes in predation pressure in the absence of neutral genetic changes</span>. <abbr class="cit-jnl-abbrev">PNAS</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">6256</span>–<span class="cit-lpage">6260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.111606798%26rft_id%253Dinfo%253Apmid%252F11353872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTEvNjI1NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzIwLzIwMTkuMTIuMjAuODg0MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.18" data-doi="10.1126/science.1200303"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">TP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">ST</span></span>, <span class="cit-auth"><span class="cit-name-surname">House</span>  <span class="cit-name-given-names">JI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prentice</span>  <span class="cit-name-given-names">IC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">GM</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Beyond Predictions: Biodiversity Conservation in a Changing Climate</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">332</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDawson%26rft.auinit1%253DT.%2BP.%26rft.volume%253D332%26rft.issue%253D6025%26rft.spage%253D53%26rft.epage%253D58%26rft.atitle%253DBeyond%2BPredictions%253A%2BBiodiversity%2BConservation%2Bin%2Ba%2BChanging%2BClimate%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1200303%26rft_id%253Dinfo%253Apmid%252F21454781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzIvNjAyNS81MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzIwLzIwMTkuMTIuMjAuODg0MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.19" data-doi="10.1038/nbt.2478"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Sequencing and automated whole-genome optical mapping of the genome of a domestic goat (Capra hircus)</span>. <abbr class="cit-jnl-abbrev">Nature Biotechnology</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BBiotechnology%26rft.volume%253D31%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2478%26rft_id%253Dinfo%253Apmid%252F23263233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23263233&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Duputié</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rutschmann</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ronce</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chuine</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Phenological plasticity will not help all species adapt to climate change</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">3062</span>–<span class="cit-lpage">3073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D21%26rft.spage%253D3062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.21" data-doi="10.1111/ele.12376"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">SR</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Ecological genomics meets community-level modelling of biodiversity: mapping the genomic landscape of current and future environmental adaptation</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12376%26rft_id%253Dinfo%253Apmid%252F25270536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25270536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">HH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Urban</span>  <span class="cit-name-given-names">DL</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Using genotype-environment associations to identify multilocus local adaptation</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: <span class="cit-fpage">129460</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.23" data-doi="10.1126/science.1209271"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fournier-Level</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitt</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilczek</span>  <span class="cit-name-given-names">AM</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">A Map of Local Adaptation in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFournier-Level%26rft.auinit1%253DA.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D86%26rft.epage%253D89%26rft.atitle%253DA%2BMap%2Bof%2BLocal%2BAdaptation%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209271%26rft_id%253Dinfo%253Apmid%252F21980109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzIwLzIwMTkuMTIuMjAuODg0MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Funk</span>  <span class="cit-name-given-names">WC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Converse</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Darst</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morey</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Improving conservation policy with genomics: a guide to integrating adaptive potential into U.S. Endangered Species Act decisions for conservation practitioners and geneticists</span>. <abbr class="cit-jnl-abbrev">Conserv Genet</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv%2BGenet%26rft.volume%253D20%26rft.spage%253D115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Funk</span>  <span class="cit-name-given-names">WC</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">FW</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Harnessing genomics for delineating conservation units</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">489</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D27%26rft.spage%253D489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Galbraith</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Fundamental hair follicle biology and fine fibre production in animals</span>. <abbr class="cit-jnl-abbrev">animal</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1490</span>–<span class="cit-lpage">1509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnimal%26rft.aulast%253DGalbraith%26rft.auinit1%253DH.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253D1490%26rft.epage%253D1509%26rft.atitle%253DFundamental%2Bhair%2Bfollicle%2Bbiology%2Band%2Bfine%2Bfibre%2Bproduction%2Bin%2Banimals.%26rft_id%253Dinfo%253Apmid%252F22444696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22444696&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garzón</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robson</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hampe</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">ΔTraitSDMs: species distribution models that account for local adaptation and phenotypic plasticity</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr> <span class="cit-vol">222</span>: <span class="cit-fpage">1757</span>–<span class="cit-lpage">1765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D222%26rft.spage%253D1757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y-L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Khatib</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Melatonin promotes Cashmere goat (Capra hircus) secondary hair follicle growth: a view from integrated analysis of long non-coding and coding RNAs</span>. <abbr class="cit-jnl-abbrev">Cell Cycle</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BCycle%26rft.volume%253D17%26rft.spage%253D1255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Godber</span>  <span class="cit-name-given-names">OF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laroussi</span>  <span class="cit-name-given-names">BF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chentouf</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Intensification of Mediterranean Goat Production Systems: A Case Study in Northern Morocco</span>. <abbr class="cit-jnl-abbrev">Agriculture</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgriculture%26rft.volume%253D6%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.30" data-doi="10.1016/S0047-6374(97)00111-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">1997</span>). <span class="cit-article-title">Hierarchical deterioration of body systems in Werner’s syndrome: Implications for normal ageing</span>. <abbr class="cit-jnl-abbrev">Mechanisms of Ageing and Development</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMechanisms%2Bof%2Bageing%2Band%2Bdevelopment%26rft.stitle%253DMech%2BAgeing%2BDev%26rft.aulast%253DGoto%26rft.auinit1%253DM.%26rft.volume%253D98%26rft.issue%253D3%26rft.spage%253D239%26rft.epage%253D254%26rft.atitle%253DHierarchical%2Bdeterioration%2Bof%2Bbody%2Bsystems%2Bin%2BWerner%2527s%2Bsyndrome%253A%2Bimplications%2Bfor%2Bnormal%2Bageing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0047-6374%252897%252900111-5%26rft_id%253Dinfo%253Apmid%252F9352493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0047-6374(97)00111-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9352493&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YB78100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hällfors</span>  <span class="cit-name-given-names">MH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dzurisin</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grundel</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hyvärinen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Towle</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Addressing potential local adaptation in species distribution models: implications for conservation under climate change</span>. <abbr class="cit-jnl-abbrev">Ecological Applications</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1154</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.volume%253D26%26rft.spage%253D1154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.32" data-doi="10.1126/science.1209244"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarymowycz</span>  <span class="cit-name-given-names">LB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperone</span>  <span class="cit-name-given-names">FG</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Adaptation to Climate Across the Arabidopsis thaliana Genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzIwLzIwMTkuMTIuMjAuODg0MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.33"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Munshi-South</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Scans for positive selection reveal candidate genes and local adaptation of Peromyscus leucopus populations to urbanization</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: <span class="cit-fpage">038141</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.34" data-doi="10.1111/eva.12149"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harrisson</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pavlova</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Telonis-Scott</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sunnucks</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Using genomics to characterize evolutionary potential for conservation of wild populations</span>. <abbr class="cit-jnl-abbrev">Evol Appl</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1008</span>–<span class="cit-lpage">1025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BAppl%26rft.volume%253D7%26rft.spage%253D1008%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12149%26rft_id%253Dinfo%253Apmid%252F25553064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25553064&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.35" data-doi="10.1634/stemcells.2006-0167"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamato</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sugiyama</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sumide</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Okano</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">N-Cadherin Is Expressed by Putative Stem/Progenitor Cells and Melanocytes in the Human Limbal Epithelial Stem Cell Niche</span>. <abbr class="cit-jnl-abbrev">STEM CELLS</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSTEM%2BCELLS%26rft_id%253Dinfo%253Adoi%252F10.1634%252Fstemcells.2006-0167%26rft_id%253Dinfo%253Apmid%252F17008425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1634/stemcells.2006-0167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17008425&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244070600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.36" data-doi="10.1002/joc.1276"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">PG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Very high resolution interpolated climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1965</span>–<span class="cit-lpage">1978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D25%26rft.spage%253D1965%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.1276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.1276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000234519700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Etten</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">raster: Geographic analysis and modeling with raster data</span>. <abbr class="cit-jnl-abbrev">R package version 2.0-12</abbr>. <a href="http://CRAN.R-project.org/package=raster">http://CRAN.R-project.org/package=raster</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.38" data-doi="10.1086/688018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoban</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Antolin</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bradburd</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">DB</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Finding the Genomic Basis of Local Adaptation: Pitfalls, Practical Solutions, and Future Directions</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D188%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1086%252F688018%26rft_id%253Dinfo%253Apmid%252F27622873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/688018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27622873&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoelzel</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruford</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleischer</span>  <span class="cit-name-given-names">RC</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Conservation of adaptive potential and functional diversity</span>. <abbr class="cit-jnl-abbrev">Conserv Genet</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv%2BGenet%26rft.volume%253D20%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.40" data-doi="10.1111/j.1365-2052.2010.02043.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Climate change and the characterization, breeding and conservation of animal genetic resources</span>. <abbr class="cit-jnl-abbrev">Animal Genetics</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2BGenetics%26rft.volume%253D41%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2010.02043.x%26rft_id%253Dinfo%253Apmid%252F20500754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2010.02043.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20500754&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.41" data-doi="10.1038/nature09670"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sgrò</span>  <span class="cit-name-given-names">CM</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Climate change and evolutionary adaptation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D470%26rft.issue%253D7335%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DClimate%2Bchange%2Band%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09670%26rft_id%253Dinfo%253Apmid%252F21350480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21350480&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287652900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Willi</span>  <span class="cit-name-given-names">Y</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Detecting genetic responses to environmental change</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D421%26rft.epage%253D432%26rft.atitle%253DDetecting%2Bgenetic%2Bresponses%2Bto%2Benvironmental%2Bchange.%26rft_id%253Dinfo%253Apmid%252F18463665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18463665&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255953500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">L</span></span> (<span class="cit-pub-date">2000</span>). <span class="cit-article-title">Biological consequences of global warming: is the signal already apparent</span>? <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D15%26rft.spage%253D56%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.44" data-doi="10.1111/gcb.13470."><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ikeda</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Max</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Allan</span>  <span class="cit-name-given-names">GJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shuster</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitham</span>  <span class="cit-name-given-names">TG</span></span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Genetically informed ecological niche models improve climate change predictions</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">164</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D23%26rft.spage%253D164%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.13470.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.13470.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.45"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">IPCC</span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Climate Change 2014: Synthesis Report</span>. <abbr class="cit-jnl-abbrev">Contribution of Working Groups I, II and III to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.46" data-doi="10.1093/bioinformatics/btn129"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">adegenet: a R package for the multivariate analysis of genetic markers</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn129%26rft_id%253Dinfo%253Apmid%252F18397895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18397895&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256169300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Joost</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">The geographical dimension of genetic diversity: a GIScience contribution for the conservation of animal genetic resources</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.48" data-doi="10.1111/j.1365-294X.2007.03442.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Joost</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruford</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Després</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conord</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Erhardt</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2007</span>). <span class="cit-article-title">A spatial analysis method (SAM) to detect candidate loci for selection: towards a landscape genomics approach to adaptation</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3955</span>–<span class="cit-lpage">3969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DJoost%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D18%26rft.spage%253D3955%26rft.epage%253D3969%26rft.atitle%253DA%2Bspatial%2Banalysis%2Bmethod%2B%2528SAM%2529%2Bto%2Bdetect%2Bcandidate%2Bloci%2Bfor%2Bselection%253A%2Btowards%2Ba%2Blandscape%2Bgenomics%2Bapproach%2Bto%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03442.x%26rft_id%253Dinfo%253Apmid%252F17850556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03442.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17850556&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249433400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.49" data-doi="10.1111/mec.12352"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Joost</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vuilleumier</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoville</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leempoel</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stucki</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Uncovering the genetic basis of adaptive change: on the intersection of landscape genomics and theoretical population genetics</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3659</span>–<span class="cit-lpage">3665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D22%26rft.spage%253D3659%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000321552800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">WW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">CY</span></span>, <span class="cit-auth"><span class="cit-name-surname">Converse</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shiraishi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">CW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ishizaki</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1996</span>). <span class="cit-article-title">Keratin 12-deficient mice have fragile corneal epithelia</span>. <abbr class="cit-jnl-abbrev">Invest Ophthalmol Vis Sci</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">2572</span>–<span class="cit-lpage">2584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInvestigative%2BOphthalmology%2B%2526%2BVisual%2BScience%26rft.stitle%253DIOVS%26rft.aulast%253DKao%26rft.auinit1%253DW.%2BW.%26rft.volume%253D37%26rft.issue%253D13%26rft.spage%253D2572%26rft.epage%253D2584%26rft.atitle%253DKeratin%2B12-deficient%2Bmice%2Bhave%2Bfragile%2Bcorneal%2Bepithelia%26rft_id%253Dinfo%253Apmid%252F8977471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiaW92cyI7czo1OiJyZXNpZCI7czoxMDoiMzcvMTMvMjU3MiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzIwLzIwMTkuMTIuMjAuODg0MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kardos</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">ABA</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">The Peril of Gene-Targeted Conservation</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D33%26rft.spage%253D827%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.52" data-doi="10.1158/0008-5472.CAN-04-0366"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karenko</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahtola</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Päivinen</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karhu</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Syrjä</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kähkönen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2005</span>). <span class="cit-article-title">Primary Cutaneous T-Cell Lymphomas Show a Deletion or Translocation Affecting NAV3, the Human UNC-53 Homologue</span>. <abbr class="cit-jnl-abbrev">Cancer Res</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">8101</span>–<span class="cit-lpage">8110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DKarenko%26rft.auinit1%253DL.%26rft.volume%253D65%26rft.issue%253D18%26rft.spage%253D8101%26rft.epage%253D8110%26rft.atitle%253DPrimary%2BCutaneous%2BT-Cell%2BLymphomas%2BShow%2Ba%2BDeletion%2Bor%2BTranslocation%2BAffecting%2BNAV3%252C%2Bthe%2BHuman%2BUNC-53%2BHomologue%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-04-0366%26rft_id%253Dinfo%253Apmid%252F16166283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2NS8xOC84MTAxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMjAvMjAxOS4xMi4yMC44ODQxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">BL</span></span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Targeted gene flow for conservation</span>. <abbr class="cit-jnl-abbrev">Conservation Biology</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BBiology%26rft.volume%253D30%26rft.spage%253D259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.54" data-doi="10.1016/S0092-8674(03)00273-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kljuic</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bazzi</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sundberg</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martinez-Mir</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Shaughnessy</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mahoney</span>  <span class="cit-name-given-names">MG</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>). <span class="cit-article-title">Desmoglein 4 in Hair Follicle Differentiation and Epidermal Adhesion: Evidence from Inherited Hypotrichosis and Acquired Pemphigus Vulgaris</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKljuic%26rft.auinit1%253DA.%26rft.volume%253D113%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D260%26rft.atitle%253DDesmoglein%2B4%2Bin%2Bhair%2Bfollicle%2Bdifferentiation%2Band%2Bepidermal%2Badhesion%253A%2Bevidence%2Bfrom%2Binherited%2Bhypotrichosis%2Band%2Bacquired%2Bpemphigus%2Bvulgaris.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252803%252900273-3%26rft_id%253Dinfo%253Apmid%252F12705872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(03)00273-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12705872&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182508500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.20.884114v1.55"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kolmogorov</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">1956</span>). <span class="cit-chapter-title">Foundations of the Theory of Probability</span>. <span class="cit-publ-name">Chelsea Pub Co</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.56" data-doi="10.1083/jcb.143.3.795"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kutzleb</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sanders</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lichte</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petrasch-Parwez</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">1998</span>). <span class="cit-article-title">Paralemmin, a Prenyl-Palmitoyl–anchored Phosphoprotein Abundant in Neurons and Implicated in Plasma Membrane Dynamics and Cell Process Formation</span>. <abbr class="cit-jnl-abbrev">The Journal of Cell Biology</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.143.3.795%26rft_id%253Dinfo%253Apmid%252F9813098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE0My8zLzc5NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDE5LzEyLzIwLzIwMTkuMTIuMjAuODg0MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.57" data-doi="10.1126/sciadv.1400218"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">HD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramu</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hash</span>  <span class="cit-name-given-names">CT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonnette</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Genome-environment associations in sorghum landraces predict adaptive traits</span>. <abbr class="cit-jnl-abbrev">Science Advances</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e1400218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DLasky%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253De1400218%26rft.epage%253De1400218%26rft.atitle%253DGenome-environment%2Bassociations%2Bin%2Bsorghum%2Blandraces%2Bpredict%2Badaptive%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1400218%26rft_id%253Dinfo%253Apmid%252F26601206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMS82L2UxNDAwMjE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMjAvMjAxOS4xMi4yMC44ODQxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">KCTD1 Suppresses Canonical Wnt Signaling Pathway by Enhancing β-catenin Degradation</span>. <abbr class="cit-jnl-abbrev">PLOS ONE</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e94343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft.volume%253D9%26rft.spage%253De94343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.59" data-doi="10.1111/j.1365-294X.2011.05308.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sjödin</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Joint analysis of demography and selection in population genetics: where do we stand and where could we go</span>? <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLi%26rft.auinit1%253DJ.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D44%26rft.atitle%253DJoint%2Banalysis%2Bof%2Bdemography%2Band%2Bselection%2Bin%2Bpopulation%2Bgenetics%253A%2Bwhere%2Bdo%2Bwe%2Bstand%2Band%2Bwhere%2Bcould%2Bwe%2Bgo%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05308.x%26rft_id%253Dinfo%253Apmid%252F21999307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05308.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21999307&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">YH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">ZJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">YH</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Identification of complete linkage disequilibrium in the DSG4 gene and its association with wool length and crimp in Chinese indigenous sheep</span>. <abbr class="cit-jnl-abbrev">GMR | Genetics and Molecular Research | The Original by FUNPEC-RP</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">5617</span>–<span class="cit-lpage">5625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGMR%2B%257C%2BGenetics%2Band%2BMolecular%2BResearch%2B%257C%2BThe%2BOriginal%2Bby%2BFUNPEC-RP%26rft.volume%253D13%26rft.spage%253D5617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">BK</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Conservation of adaptive potential and functional diversity: integrating old and new approaches</span>. <abbr class="cit-jnl-abbrev">Conserv Genet</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv%2BGenet%26rft.volume%253D20%26rft.spage%253D89%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.62"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mdladla</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dzomba</span>  <span class="cit-name-given-names">EF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muchadeyi</span>  <span class="cit-name-given-names">FC</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Landscape genomics and pathway analysis to understand genetic adaptation of South African indigenous goat populations</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">369</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D120%26rft.spage%253D369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nicotra</span>  <span class="cit-name-given-names">AB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Beever</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">GE</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Assessing the components of adaptive capacity to improve conservation and management efforts under global change</span>. <abbr class="cit-jnl-abbrev">Conservation Biology</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1268</span>–<span class="cit-lpage">1278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BBiology%26rft.volume%253D29%26rft.spage%253D1268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.64" data-doi="10.1016/j.biocon.2003.12.008"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Opdam</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wascher</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2004</span>). <span class="cit-article-title">Climate change meets habitat fragmentation: linking landscape and biogeographical scale levels in research and conservation</span>. <abbr class="cit-jnl-abbrev">Biological Conservation</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BConservation%26rft.volume%253D117%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biocon.2003.12.008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.biocon.2003.12.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000220702900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.65" data-doi="10.1086/589460"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Unckless</span>  <span class="cit-name-given-names">RL</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Population Extinction and the Genetics of Adaptation</span>. <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">160</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D172%26rft.issue%253D2%26rft.spage%253D160%26rft.epage%253D169%26rft.atitle%253DPopulation%2Bextinction%2Band%2Bthe%2Bgenetics%2Bof%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F589460%26rft_id%253Dinfo%253Apmid%252F18662122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/589460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18662122&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257986200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.66" data-doi="10.1186/1471-2156-10-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pariset</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joost</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marsan</span>  <span class="cit-name-given-names">PA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Valentini</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth cit-collab">Econogene Consortium (EC)</span> (<span class="cit-pub-date">2009</span>). <span class="cit-article-title">Landscape genomics and biased FST approaches reveal single nucleotide polymorphisms under selection in goat breeds of North-East Mediterranean</span>. <abbr class="cit-jnl-abbrev">BMC Genet</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DPariset%26rft.auinit1%253DL.%26rft.volume%253D10%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DLandscape%2Bgenomics%2Band%2Bbiased%2BFST%2Bapproaches%2Breveal%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bunder%2Bselection%2Bin%2Bgoat%2Bbreeds%2Bof%2BNorth-East%2BMediterranean.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-10-7%26rft_id%253Dinfo%253Apmid%252F19228375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-10-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19228375&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.67" data-doi="10.1111/mec.12152"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pauls</span>  <span class="cit-name-given-names">SU</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bálint</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfenninger</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">The impact of global climate change on genetic diversity within populations and species</span>. <abbr class="cit-jnl-abbrev">Mol Ecol</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BEcol%26rft.volume%253D22%26rft.spage%253D925%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.68" data-doi="10.1016/j.ecolmodel.2005.03.026"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">RP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schapire</span>  <span class="cit-name-given-names">RE</span></span> (<span class="cit-pub-date">2006</span>). <span class="cit-article-title">Maximum entropy modeling of species geographic distributions</span>. <abbr class="cit-jnl-abbrev">Ecological Modelling</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BModelling%26rft.volume%253D190%26rft.spage%253D231%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ecolmodel.2005.03.026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ecolmodel.2005.03.026&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233859600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.69" data-doi="10.1038/nrg2880"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rienzo</span>  <span class="cit-name-given-names">AD</span></span> (<span class="cit-pub-date">2010</span>). <span class="cit-article-title">Adaptation – not by sweeps alone</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D665%26rft.epage%253D667%26rft.atitle%253DAdaptation%2B-%2Bnot%2Bby%2Bsweeps%2Balone.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2880%26rft_id%253Dinfo%253Apmid%252F20838407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838407&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281911300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.70"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Development Core Team</span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">R: A language and environment for statistical computing</span>. <abbr class="cit-jnl-abbrev">R Foundation for Statistical Computing</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.71" data-doi="10.1046/j.1365-2052.2001.00786.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramunno</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cosenza</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pappalardo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Longobardi</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gallo</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pastore</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">Characterization of two new alleles at the goat CSN1S2 locus</span>. <abbr class="cit-jnl-abbrev">Animal Genetics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">264</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DRamunno%26rft.auinit1%253DL.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D264%26rft.epage%253D268%26rft.atitle%253DCharacterization%2Bof%2Btwo%2Bnew%2Balleles%2Bat%2Bthe%2Bgoat%2BCSN1S2%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2052.2001.00786.x%26rft_id%253Dinfo%253Apmid%252F11683712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2052.2001.00786.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11683712&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.72" data-doi="10.1073/pnas.1820663116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Razgour</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juste</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ibáñez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Considering adaptive genetic variation in climate change vulnerability assessment reduces species range loss projections</span>. <abbr class="cit-jnl-abbrev">PNAS</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">10418</span>–<span class="cit-lpage">10423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1820663116%26rft_id%253Dinfo%253Apmid%252F31061126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzIxLzEwNDE4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMTIvMjAvMjAxOS4xMi4yMC44ODQxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Razgour</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manel</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juste</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ibáñez</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rebelo</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">An integrated framework to identify wildlife populations under threat from climate change</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D18%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.74" data-doi="10.1554/0014-3820(2001)055[1095:HCCAMA]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frankham</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">2001</span>). <span class="cit-article-title">How Closely Correlated Are Molecular and Quantitative Measures of Genetic Variation? A Meta-Analysis</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1095</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DReed%26rft.auinit1%253DD.%2BH.%26rft.volume%253D55%26rft.issue%253D6%26rft.spage%253D1095%26rft.epage%253D1103%26rft.atitle%253DHow%2Bclosely%2Bcorrelated%2Bare%2Bmolecular%2Band%2Bquantitative%2Bmeasures%2Bof%2Bgenetic%2Bvariation%253F%2BA%2Bmeta-analysis.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282001%2529055%255B1095%253AHCCAMA%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F11475045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2001)055[1095:HCCAMA]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11475045&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169903000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.75" data-doi="10.1111/j.1523-1739.2010.01552.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reed</span>  <span class="cit-name-given-names">TE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schindler</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">RS</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Interacting Effects of Phenotypic Plasticity and Evolution on Population Persistence in a Changing Climate</span>. <abbr class="cit-jnl-abbrev">Conservation Biology</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BBiology%26rft.volume%253D25%26rft.spage%253D56%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1523-1739.2010.01552.x%26rft_id%253Dinfo%253Apmid%252F20646016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1523-1739.2010.01552.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20646016&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286380500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.76" data-doi="10.1186/s12864-016-2365-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Genome-wide analysis of long non-coding RNAs at early stage of skin pigmentation in goats (Capra hircus)</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2365-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2365-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.77"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Reside</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butt</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">VM</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Adapting systematic conservation planning for climate change</span>. <abbr class="cit-jnl-abbrev">Biodivers Conserv</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiodivers%2BConserv%26rft.volume%253D27%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.20.884114v1.78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rischkowsky</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pilling</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2007</span>). <span class="cit-chapter-title">The state of the world’s animal genetic resources for food and agriculture</span>. <span class="cit-publ-name">Food &amp; Agriculture Org</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.79"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rochat</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leempoel</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vajana</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colli</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ajmone-Marsan</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joost</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>). <span class="cit-article-title">Map of genotype frequency change in autochtonous Moroccan sheep breeds due to global warming</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Savage</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">KP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Lost but not forgotten: MHC genotypes predict overwinter survival despite depauperate MHC diversity in a declining frog</span>. <abbr class="cit-jnl-abbrev">Conserv Genet</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv%2BGenet%26rft.volume%253D19%26rft.spage%253D309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-book" id="cit-2019.12.20.884114v1.81"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Schaaf</span>  <span class="cit-name-given-names">T</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Scherf</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rischkowsky</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wieczorek</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montironi</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cardellino</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-chapter-title">Livestock Genetic Diversity in Dry Rangelands</span>. In: <span class="cit-ed"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-ed"><span class="cit-name-surname">Schaaf</span>  <span class="cit-name-given-names">T</span></span> (eds) <span class="cit-source">The Future of Drylands</span>, <span class="cit-publ-name">Springer</span> <span class="cit-publ-loc">Netherlands</span>, pp <span class="cit-fpage">89</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.82"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schoville</span>  <span class="cit-name-given-names">SD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonin</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lobreaux</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melodelima</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manel</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">Adaptive Genetic Variation on the Landscape: Methods and Cases</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D43%26rft.spage%253D23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.83"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Selmoni</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rochat</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lecellier</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Berteaux-Lecellier</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joost</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Seascape genomics as a new tool to empower coral reef conservation strategies: an example on north-western Pacific Acropora digitifera</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: <span class="cit-fpage">588228</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.84"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sgrò</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">AA</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Building evolutionary resilience for conserving biodiversity under climate change</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D4%26rft.spage%253D326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">ABA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">JBW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alves</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bergström</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bruford</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brännström</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>). <span class="cit-article-title">Genomics and the challenging translation into conservation practice</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D30%26rft.spage%253D78%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.86"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nicolazzi</span>  <span class="cit-name-given-names">EL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">CP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rothschild</span>  <span class="cit-name-given-names">MF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colli</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosen</span>  <span class="cit-name-given-names">BD</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">AdaptMap: exploring goat diversity and adaptation</span>. <abbr class="cit-jnl-abbrev">Genet Sel Evol</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet%2BSel%2BEvol%26rft.volume%253D50%26rft.spage%253D61%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.87"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stucki</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Forester</span>  <span class="cit-name-given-names">BR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Duruz</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colli</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masembe</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">High performance computation of landscape genomic models including local indicators of spatial association</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D17%26rft.spage%253D1072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thomassen</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buermann</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milá</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kieswetter</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarrín-V</span>.  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Mapping evolutionary process: a multi-taxa approach to conservation prioritization</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D4%26rft.spage%253D397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.89"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vajana</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barbato</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colli</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milanesi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rochat</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fabrizi</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Combining Landscape Genomics and Ecological Modelling to Investigate Local Adaptation of Indigenous Ugandan Cattle to East Coast Fever</span>. <abbr class="cit-jnl-abbrev">Front Genet</abbr> <span class="cit-vol">9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.90"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vandergast</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">WM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lugo</span>  <span class="cit-name-given-names">RV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hathaway</span>  <span class="cit-name-given-names">SA</span></span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Genetic landscapes GIS Toolbox: tools to map patterns of genetic divergence and diversity</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">158</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D11%26rft.spage%253D158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.91"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vangestel</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">St. Clair</span>  <span class="cit-name-given-names">JB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Linking phenotype, genotype and environment to unravel genetic components underlying cold hardiness in coastal Douglas-fir (Pseudotsuga menziesii var. menziesii)</span>. <abbr class="cit-jnl-abbrev">Tree Genetics &amp; Genomes</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenetics%2B%2526%2BGenomes%26rft.volume%253D14%26rft.spage%253D10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.92" data-doi="10.1111/mec.12705"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Villemereuil</span>  <span class="cit-name-given-names">P de</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frichot</span>  <span class="cit-name-given-names">É</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bazin</span>  <span class="cit-name-given-names">É</span></span>, <span class="cit-auth"><span class="cit-name-surname">François</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">OE</span></span> (<span class="cit-pub-date">2014</span>). <span class="cit-article-title">Genome scan methods against more complex models: when and how much should we trust them</span>? <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2006</span>–<span class="cit-lpage">2019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D23%26rft.spage%253D2006%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12705%26rft_id%253Dinfo%253Apmid%252F24611968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24611968&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.93" data-doi="10.1146/annurev-genet-111212-133526"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vitti</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">PC</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Detecting Natural Selection in Genomic Data</span>. <abbr class="cit-jnl-abbrev">Annual Review of Genetics</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BGenetics%26rft.volume%253D47%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-111212-133526%26rft_id%253Dinfo%253Apmid%252F24274750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-111212-133526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24274750&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329816000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.94"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>). <span class="cit-article-title">Integrated analysis of coding genes and non-coding RNAs during hair follicle cycle of cashmere goat (Capra hircus)</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.95"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Weeks</span>  <span class="cit-name-given-names">AR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sgro</span>  <span class="cit-name-given-names">CM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">AG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frankham</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">NJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>). <span class="cit-article-title">Assessing the benefits and risks of translocations in changing environments: a genetic perspective</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D4%26rft.spage%253D709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.96" data-doi="10.1371/journal.pbio.0060325"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">SE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shoo</span>  <span class="cit-name-given-names">LP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Isaac</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Langham</span>  <span class="cit-name-given-names">G</span></span> (<span class="cit-pub-date">2008</span>). <span class="cit-article-title">Towards an Integrated Framework for Assessing the Vulnerability of Species to Climate Change</span>. <abbr class="cit-jnl-abbrev">PLOS Biol</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BBiol%26rft.volume%253D6%26rft.spage%253De325%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.97"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Willoughby</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harder</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tennessen</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Scribner</span>  <span class="cit-name-given-names">KT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Christie</span>  <span class="cit-name-given-names">MR</span></span> (<span class="cit-pub-date">2018</span>). <span class="cit-article-title">Rapid genetic adaptation to a novel environment despite a genome-wide reduction in genetic diversity</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">4041</span>–<span class="cit-lpage">4051</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D27%26rft.spage%253D4041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.98"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>). <span class="cit-article-title">Comparative study on seasonal hair follicle cycling by analysis of the transcriptomes from cashmere and milk goats</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.99" data-doi="10.1093/bioinformatics/bts606"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gogarten</span>  <span class="cit-name-given-names">SM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">BS</span></span> (<span class="cit-pub-date">2012</span>). <span class="cit-article-title">A high-performance computing toolset for relatedness and principal component analysis of SNP data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">3326</span>–<span class="cit-lpage">3328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts606%26rft_id%253Dinfo%253Apmid%252F23060615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23060615&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F12%2F20%2F2019.12.20.884114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312105300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.20.884114v1.100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carbonetto</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2013</span>). <span class="cit-article-title">Polygenic Modeling with Bayesian Sparse Linear Mixed Models</span>. <abbr class="cit-jnl-abbrev">PLOS Genetics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenetics%26rft.volume%253D9%26rft.spage%253D1003264e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/2019.12.20.884114v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.20.884114v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2019.12.20.883926v1" title="Increasing motor skill acquisition by driving theta-gamma coupling" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;December 20, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.20.884114v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2019.12.20.884114v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.20.884114v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.20.884114v1" class="active" data-icon-position="" data-hide-link-title="0">Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1057159" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-L4cbmpGV-ULrrkHbvPIqvO6SThUVGlzsBLe768GlFvU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1057159" id="node1057159--2" data-pisa="biorxiv;2019.12.20.884114v1" data-pisa-master="biorxiv;2019.12.20.884114" data-seqnum="1057159" data-apath="/biorxiv/early/2019/12/20/2019.12.20.884114.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Estelle</span> <span class="nlm-surname">Rochat</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Stéphane</span> <span class="nlm-surname">Joost</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.20.884114; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.20.884114 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.20.884114v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1&amp;title=Spatial%20Areas%20of%20Genotype%20Probability%20%28SPAG%29%3A%20predicting%20the%20spatial%20distribution%20of%20adaptive%20genetic%20variants%20under%20future%20climatic%20conditions" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1&amp;title=Spatial%20Areas%20of%20Genotype%20Probability%20%28SPAG%29%3A%20predicting%20the%20spatial%20distribution%20of%20adaptive%20genetic%20variants%20under%20future%20climatic%20conditions" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1&amp;text=Spatial%20Areas%20of%20Genotype%20Probability%20%28SPAG%29%3A%20predicting%20the%20spatial%20distribution%20of%20adaptive%20genetic%20variants%20under%20future%20climatic%20conditions" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1&amp;title=Spatial%20Areas%20of%20Genotype%20Probability%20%28SPAG%29%3A%20predicting%20the%20spatial%20distribution%20of%20adaptive%20genetic%20variants%20under%20future%20climatic%20conditions" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1&amp;t=Spatial%20Areas%20of%20Genotype%20Probability%20%28SPAG%29%3A%20predicting%20the%20spatial%20distribution%20of%20adaptive%20genetic%20variants%20under%20future%20climatic%20conditions" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1&amp;title=Spatial%20Areas%20of%20Genotype%20Probability%20%28SPAG%29%3A%20predicting%20the%20spatial%20distribution%20of%20adaptive%20genetic%20variants%20under%20future%20climatic%20conditions" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1&amp;title=Spatial%20Areas%20of%20Genotype%20Probability%20%28SPAG%29%3A%20predicting%20the%20spatial%20distribution%20of%20adaptive%20genetic%20variants%20under%20future%20climatic%20conditions" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1057159" id="node1057159--2" data-pisa="biorxiv;2019.12.20.884114v1" data-pisa-master="biorxiv;2019.12.20.884114" data-seqnum="1057159" data-apath="/biorxiv/early/2019/12/20/2019.12.20.884114.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Spatial Areas of Genotype Probability (SPAG): predicting the spatial distribution of adaptive genetic variants under future climatic conditions</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Estelle</span> <span class="nlm-surname">Rochat</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Stéphane</span> <span class="nlm-surname">Joost</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.20.884114; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.20.884114 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Spatial%20Areas%20of%20Genotype%20Probability%20%28SPAG%29%3A%20predicting%20the%20spatial%20distribution%20of%20adaptive%20genetic%20variants%20under%20future%20climatic%20conditions&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Rochat%2C+Estelle&amp;rft.au=Joost%2C+St%C3%A9phane"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1057159/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1057159/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1057159/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1057159/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1057159/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1057159/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1057159/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1057159/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1057159/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1057159/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1057159/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1057159/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1&amp;count=horizontal&amp;via=&amp;text=Spatial%20Areas%20of%20Genotype%20Probability%20%28SPAG%29%3A%20predicting%20the%20spatial%20distribution%20of%20adaptive%20genetic%20variants%20under%20future%20climatic%20conditions&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.20.884114v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.20.884114v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/genetics" class="highlight" data-icon-position="" data-hide-link-title="0">Genetics<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
