<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/10/744797.atom" />
<meta name="HW.pisa" content="biorxiv;744797v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species" />
<meta name="DC.Identifier" content="10.1101/744797" />
<meta name="DC.Date" content="2020-01-10" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The wild currant tomato Solanum pimpinellifolium inhabits a wide range of abiotic habitats across its native range of Ecuador and Peru. Although it has served as a key genetic resource for the improvement of domestic cultivars, little is known about the genetic basis of traits underlying local adaptation in this species, nor what abiotic variables are most important for driving differentiation. Here we use redundancy analysis (RDA) and other multivariate statistical methods (structural equation modeling (SEM) and generalized dissimilarity modeling (GDM)) to quantify the relationship of genomic variation (6,830 single nucleotide polymorphisms) with climate and geography, among 140 wild accessions. RDA, SEM, and GDM each identified environment as explaining more genomic variation than geography, suggesting that local adaptation to heterogeneous abiotic habitats may be an important source of genetic diversity in this species. Environmental factors describing temporal variation in precipitation and evaporative demand explained the most SNP variation among accessions, indicating that these forces may represent key selective agents. Lastly, by studying how SNP-environment associations vary throughout the genome (44,064 SNPs), we mapped the location and investigated the functions of loci putatively contributing to climatic adaptations. Together our findings indicate an important role for selection imposed by the abiotic environment in driving genomic differentiation between populations." />
<meta name="DC.Contributor" content="Matthew JS Gibson" />
<meta name="DC.Contributor" content="Leonie C Moyle" />
<meta name="article:published_time" content="2020-01-10" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The wild currant tomato &lt;i&gt;Solanum pimpinellifolium&lt;/i&gt; inhabits a wide range of abiotic habitats across its native range of Ecuador and Peru. Although it has served as a key genetic resource for the improvement of domestic cultivars, little is known about the genetic basis of traits underlying local adaptation in this species, nor what abiotic variables are most important for driving differentiation. Here we use redundancy analysis (RDA) and other multivariate statistical methods (structural equation modeling (SEM) and generalized dissimilarity modeling (GDM)) to quantify the relationship of genomic variation (6,830 single nucleotide polymorphisms) with climate and geography, among 140 wild accessions. RDA, SEM, and GDM each identified environment as explaining more genomic variation than geography, suggesting that local adaptation to heterogeneous abiotic habitats may be an important source of genetic diversity in this species. Environmental factors describing temporal variation in precipitation and evaporative demand explained the most SNP variation among accessions, indicating that these forces may represent key selective agents. Lastly, by studying how SNP-environment associations vary throughout the genome (44,064 SNPs), we mapped the location and investigated the functions of loci putatively contributing to climatic adaptations. Together our findings indicate an important role for selection imposed by the abiotic environment in driving genomic differentiation between populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;744797v2" />
<meta name="citation_id" content="744797v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/744797v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/744797v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/744797v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797.full.pdf" />
<meta name="citation_doi" content="10.1101/744797" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="744797" />
<meta name="citation_author" content="Matthew JS Gibson" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University" />
<meta name="citation_author_email" content="gibsomat@indiana.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7855-1628" />
<meta name="citation_author" content="Leonie C Moyle" />
<meta name="citation_author_institution" content="Department of Biology, Indiana University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4960-8001" />
<meta name="citation_reference" content="Adrion, J. R., Hahn, M. W. and Cooper, B. S. (2015) Revisiting classic clines in Drosophila melanogaster in the age of genomics. Trends in Genetics, 31, 434–444. URL: http://dx.doi.org/10.1016/j.tig.2015.05.006." />
<meta name="citation_reference" content="Andrews, S. (2010) FastQC: a quality control tool for high throughput sequence data." />
<meta name="citation_reference" content="Blanca, J., Canizares, J., Cordero, L., Pascual, L., Diez, M. and Nuez, F. (2012) Variation Revealed by SNP Genotyping and Morphology Provides Insight into the Origin of the Tomato. PLoS One, 7, e48198." />
<meta name="citation_reference" content="Blanca, J., Montero-Pau, J., Sauvage, C., Bauchet, G., Illa, E., Diez, N., Francis, D., Causse, M., van der Knaap, E. and Canizares, J. (2015) Genomic variation in tomato, from wild ancestors to contemporary breeding accessions. BMC Genomics, 16, 257." />
<meta name="citation_reference" content="Bolarin, M., Fernandez, F., Cruz, V. and Cuartero, J. (1991) Salinity tolerance in four wild tomato species using vegetative yieldsalinity response curves. Journal of the American Society for Horticultural Science, 116, 286–290." />
<meta name="citation_reference" content="Borcard, D., Gillet, F. and Legendre, P. (2018) Numerical Ecology with R. Springer International, 2 edn." />
<meta name="citation_reference" content="Catchen, J., Hohenlohe, P., Bassham, S., Amores, A. and Cresko, W. (2013) Stacks: an analysis tool set for population genomics. Molecular Ecology, 22, 3124–3140." />
<meta name="citation_reference" content="Chen, S., Zhou, Y. and Gu, J. (2018) fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics, 34, i884–i890." />
<meta name="citation_reference" content="Chitwood, D., Headland, L., Filiault, D., Kumar, R., Jimenez-Gomeez, J., Schrager, A., Park, D., Peng, J., Sinha, N. and Maloof, J. (2012a) Native environment modulates leaf size and response to simulated foliar shade across wild tomato species. PLoS One, 7, e29570." />
<meta name="citation_reference" content="Chitwood, D., Headland, L., Kumar, R., Peng, J., Maloof, J. and Sinha, N. (2012b) The developmental trajectory of leaflet morphology in wild tomato species. Plant Physiology, 158, 1230–1240." />
<meta name="citation_reference" content="Cingolani, P., Platts, A., Wang, L., Coon, M., Nguyem, T., Wang, L., Landm S.J, Lu, X. and Ruden, D. (2012) A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Fly, 6, 1–13." />
<meta name="citation_reference" content="Consortium, T. G. (2012) The tomato genome sequence provides insights into fleshy fruit evolution. Nature, 485, 635–641." />
<meta name="citation_reference" content="Cuartero, J., Yeo, A. and Flowers, T. (1992) Selection of donors for salt-tolerance in tomato using physiological traits. New Phytologist, 121, 63–69." />
<meta name="citation_reference" content="Danecek, P., Auton, A., Abecasis, G., Albers, C. A., Banks, E., DePristo, M. A., Handsaker, R. E., Lunter, G., Marth, G. T., Sherry, S. T., McVean, G., Durbin, R. and Group,. G. P. A. (2011) The variant call format and VCFtools. Bioinformatics, 27, 2156–2158. URL: https://doi.org/10.1093/bioinformatics/btr330." />
<meta name="citation_reference" content="Des Marais, D. and Juenger, T. (2010) Pleiotropy, plasticity, and the evolution of plant abiotic stress tolerance. Annals of the New York Academy of Sciences, 1206, 56–79." />
<meta name="citation_reference" content="Des Marais, D., McKay, J., Richards, J., Sen, S., Wayne, T. and Juenger, T. (2012) Physiological genomics of response to soil drying in diverse Arabidopsis acceessions. The Plant cell, 24, 893–914." />
<meta name="citation_reference" content="Endler, J. (1977) Geographic Variation, Speciation, and Clines. Princeton, NJ.: Princeton University Press." />
<meta name="citation_reference" content="Endler, J. (1986) Natural Selection in the Wild. Princeton, NJ.: Princeton University Press." />
<meta name="citation_reference" content="Ferrier, S., Manion, G., Elith, J. and Richardson, K. (2007) Using generalized dissimilarity modelling to analyse and predict patterns of beta diversity in regional biodiversity assessment. Diversity and Distributions, 13, 252–264." />
<meta name="citation_reference" content="Fick, S. and Hijmans, R. (2017) Worldclim 2: New 1-km spatial resolution climate surfaces for global land areas. Intenational Jounrnal of Climatology." />
<meta name="citation_reference" content="Foolad, M. and Yin, G. (1997) Genetic potential for salt tolerance during germination in Lycopersicon species. HortScience, 32, 296–300." />
<meta name="citation_reference" content="citation_title=Advances in Molecular Breeding Toward Drought and Salt Tolerant Crops;citation_year=2007" />
<meta name="citation_reference" content="Forester, B., Lasky, J., Wagner, H. and Urbal, D. (2018) Comparing methods for detecting multilocus adaptation with multivariate genotype-environment associations. Molecular Ecology, 27, 2215–2233." />
<meta name="citation_reference" content="Fournier-Level, A., Korte, A., Cooper, M., Nordborg, M., Schmitt, J. and Wilczek, A. (2011) A map of local adaptation in Arabidopsis thaliana. Science, 334, 86–89." />
<meta name="citation_reference" content="Griebel, T. and Zeier, J. (2010) A role for beta-sitosterol to stigmasterol conversion in plant-pathogen interactions. Plant Journal, 63, 254–268." />
<meta name="citation_reference" content="Guichard, S., Gary, C., Leonardi, C. and Bertin, N. (2005) Analysis of Growth and Water Relations of Tomato Fruits in Relation to Air Vapor Pressure Deficit and Plant Fruit Load. Journal Of Plant Growth Regulation, 24, 201." />
<meta name="citation_reference" content="Hamann, A., Wang, T., Spittlehouse, D. and Murdock, T. (2013) A comprehensive, high-resolution database of historical and projected climate surfaces for western North America. Bulletin of the American Meteorological Society, 94, 1307–1309." />
<meta name="citation_reference" content="Hancock, A., Brachi, B., Faure, N., Horton, M., Jarymowycz, L., Sperone, G., Toomajian, C., Roux, F. and Bergelson, J. (2011) Adaptation to Climate Across the Arabidopsis thaliana Genome. Science, 334, 83–86." />
<meta name="citation_reference" content="Hengl, T., Mendes de Jesus, J., Heuvelink, G., Ruiperez Gonzalez, M., Kilibarda, M., Blagotic, A., Shangguan, W., Wrightt, M., Geng, X., Bauer-Marschallinger, B., Guevara, M., Vargas, R., MacMillan, R., Batjes, N., Leenaars, J., Ribeiro, E., Wheeler, I., Mantel, S. and Kempen, B. (2017) Soil-Grids250m: Global gridded soil information based on machine learning. PLoS One, 12, e0169748." />
<meta name="citation_reference" content="Hodgins-Davis, A. and Towsend, J. (2009) Evolving gene expression: from G to E to G x E. Trends in Ecology &amp; Evolution, 24, 649–658." />
<meta name="citation_reference" content="Hugly, S., McCourt, P., Browse, J., Patterson, G. and Somrville, C. (1990) A Chilling Sensitive Mutant of Arabidopsis with Altered Steryl-Ester Metabolism. Plant Physiology, 93, 1053–1062." />
<meta name="citation_reference" content="Jiang, J., Ma, S., Ye, N., Jiang, M., Cao, J. and Zhang, J. (2017) WRKY transcription factors in plant responses to stresses. Journal of Integrative Plant Biology, 59, 86–101." />
<meta name="citation_reference" content="Jissoudis, C., Sunarti, S., van de Wiel, C., Visser, R., van der Linden, C. and Bai, Y. (2016) Responses to combined abiotic and biotic stress in tomato are governed by stress intensity and resistance mechanism. Journal of Experimental Botany, 67, 5119–5132." />
<meta name="citation_reference" content="Jombart, T. and Ahmed, I. (2011) Adegenet 1.3-1: new tools for the analysis of genome-wide SNP data. Bioinformatics2, 27, 3070–3071." />
<meta name="citation_reference" content="Kim, J., Nguyen, N., Jeong, C., Nguyen, N., Hong, S. and Lee, H. (2013) Loss of the R2R3 MYB, AtMyb73, causes hyperinduction of the SOS1 and SOS3 genes in response to high salinity in Arabidopsis. Journal of Plant Physiology, 170, 1461–1465." />
<meta name="citation_reference" content="Klepikova, A., Kasianov, A., Gerasimov, E., Logacheva, M. and Penin, A. (2016) A high resolution map of the Arabidopsis thaliana developmental transcriptome based on RNA-seq profiling. Plant Journal, 88, 1058–1070." />
<meta name="citation_reference" content="Kooyers, N., Greenlee, A., Colicchio, J., Oh, M. and Blackman, B. (2015) Replicate altitudinal clines reveal that evolutionary flexibility underlies adaptation to drought stress in annual Mimulus guttatus. New Phytologist, 206, 152–165." />
<meta name="citation_reference" content="Kumar, M., Ali, K., Dahuja, A. and Tyagi, A. (2015) Role of phytosterols in drought stress tolerance in rice. Plant Physiology and Biochemistry, 96, 83–89." />
<meta name="citation_reference" content="Lasky, J., Des Marais, D., McKay, J., Richards, J., Juenger, T. and Keitt, T. (2012) Characterizing genomic variation of Arabidopsis thaliana: the roles of geography and climate. Molecular Ecology, 21, 5512–5529." />
<meta name="citation_reference" content="Lasky, J., Upadhyaya, H., Ramu, P., Deshhhpande, S., Hash, C., Bonnette, J., Juenger, T., Hyma, K., Acharya, C., Mitchell, S., Buckler, E., Brenton, Z., Kresovich, S. and Morris, G. (2015) Genome-environment associations in sorghum landraces predict adaptive traits. Science Advances, 1, e1400218." />
<meta name="citation_reference" content="Lee, C. and Mitchell-Olds, T. (2011) Quantifying effects of environmental and geographical factors on patterns of genetic differentiation. Molecular Ecology, 20, 4631–4642." />
<meta name="citation_reference" content="Legendre, P. and Legendre, L. (1998) Numerical Ecology. New York: Elsevier, 2 edn." />
<meta name="citation_reference" content="Li, H. (2011) A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics, 27, 2987–2993." />
<meta name="citation_reference" content="Li, H. and Durbin, R. (2009) Fast and accurate short read alignment with Burrows-Wheeler Transform. Bioinformatics, 25, 1754–1760." />
<meta name="citation_reference" content="Li, Z., Peng, R., Tian, Y., Han, H., Xu, J. and Yao, Q. (2016) Genome-Wide Identification and Analysis of the MYB Transcription Factor Superfamily in Solanum lycopersicum. Plant Cell and Physiology, 57, 1657–1677." />
<meta name="citation_reference" content="Lin, Y., Liu, C. and Chen, K. (2019) Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers. G3, 9, 1497–1505." />
<meta name="citation_reference" content="Manel, S., Poncet, B., Legendre, P., Gugerli, F. and Holderegger, R. (2010) Common factors drive adaptive genetic variation at different spatial scales in Arabis alpina. Molecular Ecology, 19, 3824–3835." />
<meta name="citation_reference" content="Manion, G., Lisk, M., Ferrier, S., Nieto-Lugilde, D., Mokany, K. and Fitzpatrick, M. (2018) gdm: Generalized Dissimilarity Modeling." />
<meta name="citation_reference" content="Mishra, M., Singh, G., Tiwari, S., Singh, R., Kumari, N. and Misrra, P. (2015) Characterization of Arabidopsis sterol glycosyltransferase TTG15/UGT80B1 role during freeze and heat stress. Plant Signaling &amp; Behavior, 10, e1075682." />
<meta name="citation_reference" content="Nakazato, T., Bogonovich, M. and Moyle, L. (2008) Environmental factors predict adaptive phenotypic differentiation within and between two wild andean tomatoes. Evolution, 62, 774–792." />
<meta name="citation_reference" content="Nakazato, T., Warren, D. and Moyle, L. (2010) Ecological and geographic modes of species divergence in wild tomatoes. American Journal of Botany, 97, 680–693." />
<meta name="citation_reference" content="Oksanen, J., Blanchet, F., Kindt, R., Legendre, P., Minchin, P., O’Hara, R., Simpson, G., Solymos, P., Stevens, M., Wagnerr, H. and Oksanen, M. (2013) Package ‘vegan’. Community ecology package, version 2.9." />
<meta name="citation_reference" content="Pedley, K. and Martin, G. (2003) Molecular basis of Pto-mediated resistance to bacterial speck disease in tomato. Annual Review of Phytopathology, 41, 215–243." />
<meta name="citation_reference" content="Pitblado, R. and Kerr, E. (1979) A source of resistance to bactereial speck–Pseudomonas tomato. Tomato Genetics Cooperative, 29, 30." />
<meta name="citation_reference" content="Raj, A., Stephen, M. and Pritchard, J. (2014) fastSTRUCTURE: Variational Inference of Population Structure in Large SNP Data Sets. Genetics2, 197, 573–589." />
<meta name="citation_reference" content="Ramirez-Estrada, K., Castillo, N., Lara, J., Arro, M., Boronat, A., Ferrer, A. and Altabella, T. (2017) Tomato UDP-Glucose Sterol Glycosyltransferases: A Family of Developmental and Stress Regulated Genes that Encode Cytosolic and Membrane-Associated Forms of the Enzyme. Frontiers in Plant Science, 8." />
<meta name="citation_reference" content="Rao, E., Kadirvel, P., Symonds, R. and Ebert, A. (2013) Relationship between survival and yield related traits in Solanum pimpinellifolium under salt stress. Euphytica, 190, 215–228." />
<meta name="citation_reference" content="Razali, R., Bougouffa, S., Morton, M., Lightfoot, D., Alam, I., Essack, M., Arold, S., Kamau, A., Schmockel, S., Pailles, Y., Shahid, M., Michell, C., Salim, A., Ho, Y., Tester, M., Bajic, V. and Negrao, S. (2018) The Genome Sequence of the Wild Tomato Solanum pimpinellifolium Provides Insights Into Salinity Tolerance. Frontiers in Plant Science, 9." />
<meta name="citation_reference" content="Rellstab, C., Gugerli, F., Eckert, A., Hancock, A. and Holderegger, R. (2015) A practical guide to environmental association analysis in landscape genomics. Molecular Ecology, 24, 4348–4370." />
<meta name="citation_reference" content="Rick, C., Fobes, J. and Holle, M. (1977) Genetic variation in Lycopersicon pimpinellifolium: evidence of evolutionary change in mating systems. Plant Systematics and Evolution, 127, 139–170." />
<meta name="citation_reference" content="Rosseel, Y. (2012) lavaan: An R Package for Structural Equation Modeling. Statistical Software, 48, 1–36." />
<meta name="citation_reference" content="Salathe, R. and Schmid-Hempel, P. (2011) The genotypic structure of a multi-host bumblebee parasite suggests a role for ecological niche overlap. PLoS One, 6, e22054." />
<meta name="citation_reference" content="Senthil-Kumar, M., Wang, K. and Mysore, K. (2013) At-CYP710A1 gene-mediated stigmasterol production plays a role in imparting temperature stress tolerance in Arabidopsis thaliana. Plant Signaling &amp; Behavior, 8, e23142." />
<meta name="citation_reference" content="Sork, V., Davis, F., Westfall, R., Flint, A., Ikegami, M., Wang, H. and Grivet, D. (2010) Gene movement and genetic association with regional climate gradients in California valley oak (Quercus lobata Nee) in the face of climate change. Molecular Ecology, 19, 3806–3823." />
<meta name="citation_reference" content="Tanksley, S. (2004) The genetic, developmental, and molecular bases of fruit size and shape variation in tomato. The Plant cell, 16, 181–189." />
<meta name="citation_reference" content="Tanksley, S., Grandillo, S., Fulton, T., Zamir, D., Eshed, Y., Petiard, V., Lopez, J. and Beck-Bunn, T. (1996) Advanced backcross QTL analysis in a cross between an elite processing line of tomato and its wild relative L. pimpinellifolium. Theoretical and Applied Genetics, 92, 213–224." />
<meta name="citation_reference" content="Vucetich, J. and Walte, T. (2003) Spatial patterns of demography and genetic processes across the species’ range: Null hypotheses for landscape conservation genetics. Conservation Genetics, 4, 639–645." />
<meta name="citation_reference" content="Wang, I., Glor, R. and Losos, J. (2013) Quantifying the roles of ecology and geography in spatial genetic divergence. Ecology Letters, 16, 175–182." />
<meta name="citation_reference" content="Wiens, J. (1989) Spatial Scaling in Ecology. Functional Ecology, 3, 385–397." />
<meta name="citation_reference" content="van den Wollenberg, A. (1977) Redundancy analysis: an alternative for canonical correlation analysis. Psychometrika, 42, 207–219." />
<meta name="citation_reference" content="Wright, S. (1943) Isolation by distance. Genetics, 28, 114–138." />
<meta name="citation_reference" content="Yoder, J., Stanton-Geddes, J., Zhou, P., Briskine, R., Young, N. and Tiffin, P. (2012) Genomic signature of adaptation to climate in Medicago truncatula. Genetics, 196, 1263–1275." />
<meta name="citation_reference" content="Zhang, D., Du, Q., Zhang, Z., Jiao, X., Song, X. and Li, J. (2017) Vapour pressure deficit control in relation to water transport and water productivity in greenhouse tomato production during summer. Scientific Reports, 7, 43461." />
<meta name="citation_reference" content="Zomer, R., Trabucco, A., Bossio, D., van Straaten, O. and Verchot, L. (2008) Climate Change Mitigation: A Spatial Analysis of Global Land Suitability for Clean Development Mechanism Afforestation and Reforestation. Agric. Ecosystems and Envir, 126, 67–80." />
<meta name="citation_reference" content="Zuriaga, E., Blaanca, J., Cordero, L., Sifres, A., Blas-Cerdan, W., Morales, R. and Nuez, F. (2009) Genetic and bioclimatic variation in Solanum pimpinellifolium. Genetic Resources and Crop Evolution, 56, 39–51." />
<meta name="twitter:title" content="Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The wild currant tomato Solanum pimpinellifolium inhabits a wide range of abiotic habitats across its native range of Ecuador and Peru. Although it has served as a key genetic resource for the improvement of domestic cultivars, little is known about the genetic basis of traits underlying local adaptation in this species, nor what abiotic variables are most important for driving differentiation. Here we use redundancy analysis (RDA) and other multivariate statistical methods (structural equation modeling (SEM) and generalized dissimilarity modeling (GDM)) to quantify the relationship of genomic variation (6,830 single nucleotide polymorphisms) with climate and geography, among 140 wild accessions. RDA, SEM, and GDM each identified environment as explaining more genomic variation than geography, suggesting that local adaptation to heterogeneous abiotic habitats may be an important source of genetic diversity in this species. Environmental factors describing temporal variation in precipitation and evaporative demand explained the most SNP variation among accessions, indicating that these forces may represent key selective agents. Lastly, by studying how SNP-environment associations vary throughout the genome (44,064 SNPs), we mapped the location and investigated the functions of loci putatively contributing to climatic adaptations. Together our findings indicate an important role for selection imposed by the abiotic environment in driving genomic differentiation between populations." />
<meta name="og-title" property="og:title" content="Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/744797v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The wild currant tomato Solanum pimpinellifolium inhabits a wide range of abiotic habitats across its native range of Ecuador and Peru. Although it has served as a key genetic resource for the improvement of domestic cultivars, little is known about the genetic basis of traits underlying local adaptation in this species, nor what abiotic variables are most important for driving differentiation. Here we use redundancy analysis (RDA) and other multivariate statistical methods (structural equation modeling (SEM) and generalized dissimilarity modeling (GDM)) to quantify the relationship of genomic variation (6,830 single nucleotide polymorphisms) with climate and geography, among 140 wild accessions. RDA, SEM, and GDM each identified environment as explaining more genomic variation than geography, suggesting that local adaptation to heterogeneous abiotic habitats may be an important source of genetic diversity in this species. Environmental factors describing temporal variation in precipitation and evaporative demand explained the most SNP variation among accessions, indicating that these forces may represent key selective agents. Lastly, by studying how SNP-environment associations vary throughout the genome (44,064 SNPs), we mapped the location and investigated the functions of loci putatively contributing to climatic adaptations. Together our findings indicate an important role for selection imposed by the abiotic environment in driving genomic differentiation between populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-10" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/744797v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/744797v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/744797v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/744797v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1099442" />
    <title>Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__AQm9HArb2Y_D7UzgraiLIwAB7088fIq2iZD0Uigie_s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Gy8lCdnUXIS0r8qhphwN8V-ud0myj1Z996oq5RwAIgg","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1099442","apath":"\/biorxiv\/early\/2020\/01\/10\/744797.atom","pisa":"biorxiv;744797v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;744797v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/744797v2","title":"Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species","identifier":"node\/1099442"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gf924RKhjAKy7U3QI0Zo8zB8Yg+t1k2VhDW8FaXtj34Ez9w9tEf2oWbGObP+V19k4mQQfoB2ilU6l+d3gPWpOKSDHh0MYLqYBce8tu5EYORqop\\\\\\\/MgkM+XmGxoB\\\\\\\/zsKz1PnBlCyE8VcrX6QLontoXnQH3AJY52iZpi2IO8btVKoP+CyRCEli5XsoskyArkF2V3sNg5+KWK6ifH8\\\\\\\/YvQ8Aq5PQeximsOw88cpjIb6IwsY1FAOMdqAaPyPkDb8YYB+fQMjWXhMU\\\\\\\/yUdlaPqAqgLeqcboA96NEEyoKDw9fXt5qp\\\\\\\/+H\\\\\\\/3TSpK6vBIbbkX8Siuiv\\\\\\\/SLHJxBzEJyjGKZyrLwJYBoEo799JS2gz1bhI7xqObZRv\\\\\\\/xDNq\\\\\\\/UuGj5JAk8IFu1SfNR\\\\\\\/Ef9CrZLzd3C8X9w+7kNoCPXzMF6WHwGiyzHUoqm9fBmVPcVY19Na8Cu3unhU4FDcmhVL7ZI5oTTPJtHH1X2HbxFTkxhyLecEStLTThF\\\\\\\/M+Wn8BBUOs\\\\\\\/fmJD0srmmCSF4jmkifMQ8goa5JHrQRRmf8kskachzkQMCjAtzoNOWD3ZpBq\\\\\\\/NWelgI+Z8G2A0oCJmR8LrScdo\\\\\\\/o25C36C1netbT+D\\\\\\\/jq0NtrTukURRR5iIW\\\\\\\/JVpwSrPvvtePAYRYKCAfFAPH8eSjjybNrNzw9jEuFA4k7GWpvCkHXQ7k1n5kflhhFmx5c4Cv3200b3oOVkhp8imau+Nua8vSQW7dSt+jDCrvgvOkpJNVPmOFFRQqVya2rTughw06bW7ZKRZH3ugONLKaVTGphiRKWrwZgfuyODjMgCmS\\\\\\\/qa5BUpk3GrnQcuNSftDdxiaHLVfTFnBaOgz4N\\\\\\\/6wos\\\\\\\/Sn4EGVkf7ON4W6\\\\\\\/j0f0CBfjXjlD4NHbw9UpF1Iev97ihkHPveiippgyR2t15h4e\\\\\\\/xICIT85Gpog9ZLE1y0VKg3wmV+GFZIWF53UnKUdudwSlPITo\\\\\\\/Un0tQ4ErCsKb7G2fIUgApTL1Q7oszajMuvhA2wxZ7vq7C0wl\\\\\\\/oozQaAk+GTX4lkn1h40X5kS94IllCxfVDqooRyUVEOPMLdR4GTrGAwYENWsIhn3vJf5UNlHedPByCzu6qk8mQOGryxSTH8v7uAWpCMovPv1xXki\\\\\\\/3UxZlmalkwXY407XnC0TJwz8i0UZSMzGlzgoSUHnb5eSJ8QBWuRryXf1KYn2C5buHg32ssMG+CbbccjGz27Y5YANns+MWrCLBROOHPZDx2Pt1OsGIELp8M1hI5cQM8fuoiGkd7ClG8MUubjmxkQODztp7AOUWvs+XcNGIKYozeZJLWJr2\\\\\\\/1HoZ4rXMsK1B\\\\\\\/LWT\\\\\\\/icY76uJ0dOMWfoSnj79YggPOVLSCG\\\u0022,\\\u0022iv\\\u0022:\\\u0022kaunw\\\\\\\/hJWyJoSDxDADiDCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022521f0061af724a9ff0efe547bacd271f8b782a7c39da4f83e59c47858f64d39b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/744797v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1099442 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/744797v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Qxl7tPPmH_tTPUg1sj5RRQnSEWB6FIYXGbBk8OaNWXI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1099442" id="node1099442" data-pisa="biorxiv;744797v2" data-pisa-master="biorxiv;744797" data-apath="/biorxiv/early/2020/01/10/744797.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7855-1628" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Matthew JS</span> <span class="nlm-surname">Gibson</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4960-8001" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Leonie C</span> <span class="nlm-surname">Moyle</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/744797 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1099442" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew JS Gibson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, Indiana University</span>, Bloomington, Indiana, 47401, <span class='nlm-country'>United States</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BJS%2BGibson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gibson%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BJS%2BGibson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7855-1628" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew JS Gibson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gibsomat@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">gibsomat@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Leonie C Moyle </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, Indiana University</span>, Bloomington, Indiana, 47401, <span class='nlm-country'>United States</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Leonie%2BC%2BMoyle%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moyle%20LC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALeonie%2BC%2BMoyle%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4960-8001" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Leonie C Moyle</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/744797v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1099442" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1099442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/744797v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1099442" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1099442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/744797v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1099442" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1099442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/744797v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1099442" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1099442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/744797v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1099442" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1099442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/744797v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1099442" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1099442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/744797v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1099442" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1099442/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The wild currant tomato <em>Solanum pimpinellifolium</em> inhabits a wide range of abiotic habitats across its native range of Ecuador and Peru. Although it has served as a key genetic resource for the improvement of domestic cultivars, little is known about the genetic basis of traits underlying local adaptation in this species, nor what abiotic variables are most important for driving differentiation. Here we use redundancy analysis (RDA) and other multivariate statistical methods (structural equation modeling (SEM) and generalized dissimilarity modeling (GDM)) to quantify the relationship of genomic variation (6,830 single nucleotide polymorphisms) with climate and geography, among 140 wild accessions. RDA, SEM, and GDM each identified environment as explaining more genomic variation than geography, suggesting that local adaptation to heterogeneous abiotic habitats may be an important source of genetic diversity in this species. Environmental factors describing temporal variation in precipitation and evaporative demand explained the most SNP variation among accessions, indicating that these forces may represent key selective agents. Lastly, by studying how SNP-environment associations vary throughout the genome (44,064 SNPs), we mapped the location and investigated the functions of loci putatively contributing to climatic adaptations. Together our findings indicate an important role for selection imposed by the abiotic environment in driving genomic differentiation between populations.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-4">Local adaptation is thought to underlie a large proportion of phenotypic variation segregating within species. Despite this, the primary environmental agents driving selection between natural populations are still unknown in most systems. Across landscapes, genetic divergence can result from selection imposed by environmental forces, but also from the effects of restricted gene flow and genetic drift when subpopulations are partially isolated (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wright, 1943</a>). Disentangling the concurrent effects of spatial isolation and natural selection on generating genetic diversity is critical for understanding the relative importance of adaptation in driving diversification, as well as for identifying the environmental factors underlying its occurrence. This remains a major technical challenge, mainly because collinearity is common among spatial and environmental variables (Kissoudis et al., 2016; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lasky et al., 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lee &amp; Mitchell-Olds, 2011</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wiens, 1989</a>). New statistical methods and the increasing availability of fine-scale environmental and genomic data provide one approach to estimate the independent contributions of environment and space to explaining patterns of genetic variation. These strategies can complement traditional tests for local adaptation (i.e., reciprocal transplants) and begin to characterize the nature of selection acting in the wild, as well as its genetic targets (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fournier-Level et al., 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hancock et al., 2011</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lasky et al., 2012</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">2015</a>).</p><p id="p-5"><em>Solanum pimpinellifolium</em> L. (currant tomato) is a diploid, short-lived perennial relative of domesticated tomato (<em>Solanum lycopersicum</em> L.) that has historically served as a key resource for the identification and breeding of agronomically-important traits (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Pedley &amp; Martin, 2003</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Pitblado &amp; Kerr, 1979</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tanksley, 2004</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tanksley et al., 1996</a>). Population germplasm collections (accessions) are diverse, varying substantially in mating system (higher rates of outcrossing in the center of the range; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rick et al., 1977</a>) as well as in tolerance to acute abiotic stressors such as salinity and drought (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolarin et al., 1991</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cuartero et al., 1992</a>; Kissoudis et al., 2016; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Foolad, 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Foolad &amp; Yin, 1997</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Razali et al., 2018</a>). Previous studies have suggested that a large fraction of this phenotypic variation could be attributable to spatially varying selection imposed by the abiotic environment (e.g., <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zuriaga et al., 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blanca et al., 2012</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">2015</a>). <em>S. pimpinellifolium</em> occupies a large geographic range from Ecuador to southern Peru (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>); populations therefore experience environments that vary from cool mid-elevation habitats in the Andes to coastal deserts and tropical rainforests along the pacific ocean, and the associated intraspecific variation in abiotic conditions(climate and soil characteristics; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) is proposed to shape numerous ecologically relevant traits (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chitwood et al., 2012a</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2012b</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nakazato et al., 2008</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nakazato et al., 2010</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Rick et al., 1977</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Zuriaga et al., 2009</a>). For example, using common garden experiments <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Nakazato et al. (2008)</a> identified significant intraspecific variation for drought tolerance between accessions from coastal and highland regions, consistent with a past history of selection that ranges from water restricted (and drought tolerant) on the coast to less restricted (and less tolerant) in the highlands. More broadly, using Mantel tests <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Blanca et al. (2015)</a> showed that major patterns of genetic structure among <em>S. pimpinellifolium</em> accessions correspond strongly with regional bioclimatic variation, and a clear pattern of latitudinal population structure has been demonstrated in this species several times (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Blanca et al. 2012</a>, <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">2015</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lin et al., 2019</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Rick et al., 1977</a>; <a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Zuriaga et al., 2009</a>). Between wild tomato species, distribution models have further suggested that local climatic variation (especially in temperature and water availability), rather than geographic isolation, is the main factor explaining contemporary distributions (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Nakazato et al., 2010</a>). However, while these studies point to the potential importance of abiotic selection in shaping patterns of intraspecific genetic variation, the degree to which population genetic structure is determined by environmental factors rather than demographic history, the key selective variables responsible, and the identity of relevant/causal loci remain unknown. Describing and understanding these factors is essential for uncovering the processes that initiate phenotypic divergence, including the relative role of local adaptation in driving diversification and the number and identity of its genetic targets.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="140 South American accessions included in our analyses. Ecuador (North) and Peru (South) are outlined in black. (A) Accession collection sites overlain on average vapor pressure (in units of kPa). Variation in vapor pressure contributed the most to explaining patterns of spatial genetic variation using RDA. (B) Accessions overlain on precipitation seasonality (coefficient of monthly variation)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-488042217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;140 South American accessions included in our analyses. Ecuador (North) and Peru (South) are outlined in black. (A) Accession collection sites overlain on average vapor pressure (in units of kPa). Variation in vapor pressure contributed the most to explaining patterns of spatial genetic variation using RDA. (B) Accessions overlain on precipitation seasonality (coefficient of monthly variation).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F1.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 1" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F1.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">FIGURE 1</span> <p id="p-6" class="first-child">140 South American accessions included in our analyses. Ecuador (North) and Peru (South) are outlined in black. (A) Accession collection sites overlain on average vapor pressure (in units of kPa). Variation in vapor pressure contributed the most to explaining patterns of spatial genetic variation using RDA. (B) Accessions overlain on precipitation seasonality (coefficient of monthly variation).</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">The main goals of this study were to quantitatively characterize the environmental forces that might impose selection on ecologically relevant traits across the range of <em>S. pimpinellifolium</em>, to determine the degree to which these environmental factors explain genetic variation independent of demographic history, and to identify potential genetic targets (SNPs and genes) of this environmental selection. In the absence of confounding population structure, associations between allele frequencies and environmental variables are considered evidence for local adaptation (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Endler, 1986</a>). These associations can be used to identify both the forces responsible for imposing selection, as well as the specific loci involved in adaptive evolutionary responses. However, since both adaptation and spatially structured gene flow can result in correlations between allele frequencies and the environment (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Endler, 1977</a>), demographic history can cloud such inferences of local adaptation. Moreover, abiotic selection is often the result of a combination of many partially colinear variables, making simple gene-environment correlations incomplete and potentially misleading. To address these issues, multivariate statistical methods commonly used in community ecology—in particular the multivariate regression technique “redundancy analysis” (RDA; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Legendre &amp; Legendre, 1998</a>) and other constrained ordination procedures—have recently been applied to landscape genomics to study how climatic variation affects spatial patterns of genomic variation (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Lasky et al., 2012</a>, 2016; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Forester et al., 2018</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Wang et al., 2013</a>). These methods have been used to identify both the sources of selection imposed by the multivariate environment and their genetic targets (individual loci) (Forrester et al., 2018; <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Lasky et al., 2012</a>, 2016; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Lee &amp; Mitchell-Olds, 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Manel et al., 2010</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Salathe &amp; Schmid-Hempel, 2011</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sork et al., 2010</a>). In doing so, they have begun to address the challenges of disentangling geographical structure from environmental variation in order to quantify the independent contribution of environmental forces to shaping genomic variation across space.</p><p id="p-8">Here we employ RDA to study the relationship between environmental and SNP data matrices while modeling population structure as a covariate, in order to estimate the relative influence of abiotic adaptation on genomic variation across the native range of <em>S. pimpinellifolium</em>. Specifically, we use RDA to 1) estimate the independent quantitative contributions of climate, space, and their colinear portion to explaining patterns of genome-wide SNP variation; 2) characterize the specific abiotic variables with the largest influence on genetic variation; and 3) identify genomic regions showing strong associations with major axes of multivariate climate and investigate their potential functional roles in mediating local adaptation. We complement these RDA analyses with other methods designed to quantify the roles of selection and spatial isolation. Our analyses reveal that models including several key abiotic variables explain more genomic variation among genotypes than models considering only geographic isolation. They also allow us to identify loci that have significant associations with environmental variation, after incorporating population structure via partial regression. Our results suggest a prominent role for local adaptation in driving genetic divergence among populations of this agronomically important crop wild relative, and provide a substantial step towards dissecting the contributions of specific loci to local adaptation.</p></div><div class="section" id="sec-2"><h2 class="">2 METHODS</h2><div id="sec-3" class="subsection"><h3>2.1 Plant material and genotyping</h3><p id="p-9">Seeds from 140 wild georeferenced <em>S. pimpinellifolium</em> accessions (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>; Table S1) spanning the entire species range were obtained from the Charles M. Rick Tomato Genetic Resource Center (TGRC; Davis, Calif.). Seeds were soaked in a 50% bleach solution (as per the TGRC protocol), germinated, and transplanted into 1-gallon pots at the Indiana University greenhouses. Fresh leaf material was collected from one plant per accession at 30 days post-germination. DNA was extracted using DNeasy Plant Mini Kits (Qiagen, Valencia, Calif, USA) and two double-digest restriction site associated sequencing (ddRAD) libraries were prepared using <em>PstI</em> and <em>EcoRI</em> enzymes by the Indiana University Center for Genomics and Bioinformatics. These enzymes were selected to enrich for cut sites in coding regions based on an in silico digestion of the <em>S. lycopersicum</em> reference genome ver. SL3.0 (Tomato Genome <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Consortium, 2012</a>). Libraries were sequenced across two Illumina NextSeq flowcells (150 bp, paired-end, high-output). Raw reads were trimmed of Illumina adapter sequences using <em>fastp</em> (adapter type was automatically detected with fastp; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen et al., 2018</a>), and then demultiplexed by individual using the process_radtags program in Stacks (ver. 2; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Catchen et al., 2013</a>). Reads were processed in sliding windows (15% of read length). Windows with scores below 5 and reads less than 30 bp were discarded. Sequencing error was high at restriction sites due to low base diversity and were changed to the correct restriction overhang (<em>PstI</em>; 5‘-TGCA-3‘) to avoid mapping errors. fastp was also used to correct low quality base calls in overlapping regions of paired-end reads (minimum of 30bp overlap required for correction). Read quality was verified pre- and post-processing using fastqc <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andrews, 2010</a>). After demultiplexing, reads were mapped to the <em>S. lycopersicum</em> reference genome version SL3.0 using default parameters in BWA (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Li &amp; Durbin, 2009</a>). Only uniquely mapped paired-end reads were retained (filtering performed with samtools; Li et al., 2009). Variants were called using the Stacks ref_map pipeline, using alpha of 0.05 to call variant sites and genotypes. Genotype calls made with fewer than 4 reads were changed to missing with vcftools and only bi-allelic SNPs typed in at least 30 percent of individuals (42/140) were retained. Further filtering was applied prior to running specific analyses (see below). All pipeline scripts are available at <a href="http://github.com/gibsonmatt/pimpGEA">http://github.com/gibsonmatt/pimpGEA</a>.</p></div><div id="sec-4" class="subsection"><h3>2.2 Environmental data</h3><p id="p-10">For each accession, data were compiled for 54 abiotic environmental parameters available from public databases. 19 bioclimatic variables as well as monthly solar radiation, water vapor pressure, and wind speed data were obtained directly from the WorldClim2 database (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fick &amp; Hijmans, 2017</a>). Additional transformations of the WorldClim data were obtained from the Climate South America (ClimateSA v1.0; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hamann et al., 2013</a>) database. Global aridity measures were obtained from the Consultative Group on International Agricultural Research (CGIAR; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zomer et al., 2008</a>). Lastly, soil pH, type, density (fine earth fraction), and water capacity at a depth of 5cm were obtained from the SoilGrids initiative (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hengl et al., 2017</a>). All variables were centered, scaled, and tested for normality using Shapiro-Wilks tests. If appropriate, variables were log-transformed. One-hot encoding was used for all factor variables prior to running RDA. The full list of all 54 climate and soil variables can be found in Table S2 and raw environmental data for each accession is provided in Table S12.</p></div><div id="sec-5" class="subsection"><h3>2.3 Spatial data</h3><p id="p-11">We represented spatial (non-genetic) structure in RDA using distance-based Moran Eigenvector Maps (dbMEMs; formally referred to as principle components of neighborhood matrices or PCNMs). Briefly, dbMEMs—much like principle components in a PCA—represent structure in a dataset with orthogonal axes. dbMEMs describe complex patterns of spatial structure in rectangular form suitable for use as a co-factor in downstream statistical analyses such as RDA (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Borcard et al., 2018</a>). Importantly, they are able to model both broad-and fine-scale patterns (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Borcard et al., 2018</a>; see Figure S1), both of which may affect the partitioning of genetic variation across large and highly heterogenous landscapes. dbMEMs were calculated using the quickMEM function included in <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Borcard et al. (2018)</a>. This function uses forward selection and permutation tests to identify the appropriate number of axes needed to explain patterns of spatial genetic variation, with the intention of preventing overfitting. We projected latitude and longitude data for each accession using the SIRGAS 2000 South America datum, which was then used as input to calculate dbMEMs.</p></div><div id="sec-6" class="subsection"><h3>2.4 Population genetic structure</h3><p id="p-12">Both model-based (<em>fastStructure</em>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Raj et al., 2014</a>) and non-model based (PCA) methods were used to quantify genetic structure in the data. Markers in high linkage disequilibrium (r<sup>2</sup> &gt; 0.9) and those containing high levels of missing data (&gt;5%) were pruned from the dataset prior to running PCA and <em>fastStructure</em> using the bcftools ‘prune’ plugin and vcftools, respectively (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Li, 2011</a>; (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Danecek et al., 2011</a>). The resulting dataset contained 7,317 high quality SNPs. PCA was implemented in the R (R Core Team, 2019) package adegenet (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jombart &amp; Ahmed, 2011</a>). Prior to running PCA, missing calls were imputed as the mean numeric value for each marker, to speed up computation. Model complexity in <em>fastStructure</em> was determined by minimizing the marginal likelihood of the data over values of K from one to ten. Clusters in the data were also identified using the K-means algorithm for all PCs by minimizing the Bayesian information criterion (BIC) across values of K from one to 100.</p></div><div id="sec-7" class="subsection"><h3>2.5 Partitioning total SNP variation</h3><p id="p-13">To evaluate the relative contribution of demographic history and the abiotic environment to explaining patterns of spatial genetic variation, the amount of genome-wide SNP diversity attributable to environment and geography was estimated using three methods: variance partitioning by redundancy analysis (RDA), generalized dissimilarity modeling (GDM), and structural equation modeling (SEM). All variance partitioning methods were performed with LD-filtered sites containing no missing data (6,830 SNPs, out of the 7,317 5% set above; Table S5). RDA is a form of constrained ordination first applied in community ecology that allows one to assess the explanatory power of multivariate predictors (in this case, environmental and geographic variables) for multivariate responses (genotypes; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">van den Wollenberg, 1977</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Legendre &amp; Legendre, 1998</a>). RDA identifies linear combinations of explanatory variables (canonical axes) that best explain linear combinations of response variables. As with other forms of eigenanalysis, these canonical axes are orthogonal to each other. Here, these axes represent correlated SNPs that are also correlated with environment. Using multiple partially constrained models, it is possible to partition total genetic variation into that explained independently by climate, space, and their colinear portion (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Borcard et al., 2018</a>). All RDA analyses were performed using the R package vegan (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Oksanen, 2018</a>) and significance was assessed using 1,000 permutations of the genotype matrix. Since RDA requires no missing data, only markers containing no missing calls were included (6,830 SNPs). Distances between collection sites were represented using dbMEMs (described above). Because collinearity among environmental predictors can lead to inflation of variance components in RDA (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Borcard et al., 2018</a>), we performed forward variable selection using the forward.sel function in the R package adespatial to identify predictive and non-redundant environmental variables for partitioning.</p><p id="p-14">In addition to RDA, structural equation modeling(SEM) was performed using the R package <em>lavaan</em> (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Rosseel, 2012</a>). SEM is a statistical framework similar to path analysis that has been applied to estimating the relative contributions of isolation-by-environment (IBE) and isolation-by-distance (IBD) to explaining spatial genetic variation (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Wang et al., 2013</a>). While powerful, this method is not able to estimate the independent contribution of the colinear portion of environment and space on explaining SNP variation, unlike RDA. For SEM, geographic distance was represented using Euclidean distance between accessions (calculated from projections using the SIRGAS 2000 datum). All 54 environmental variables were used to infer the latent environmental distance term (SEM accommodates correlated inputs; <a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Wang et al., 2013</a>). SEM was performed using the same set of markers as used for variance partitioning in RDA.</p><p id="p-15">Lastly, generalized dissimilarity modeling (GDM; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ferrier et al., 2007</a>) was applied to the SNP data. GDM is similar in principle to RDA and SEM but relies on different statistical methods. GDM uses nonlinear matrix regression to accommodate nonlinear statistical relationships that are often observed in ecological datasets; in contrast, RDA and SEM—as implemented here—do not model nonlinear relationships. GDM analyses were performed in R using the <em>gdm</em> package (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Manion et al., 2018</a>).</p></div><div id="sec-8" class="subsection"><h3>2.6 Importance of specific climate variables</h3><p id="p-16">Forward variable selection with RDA was used to infer which specific environmental variables may be most important for generating selective differences across the range of S. pimpinellifolium. We performed this procedure with the forward.sel function in adespatial, using 10,000 permutations to assess significance at each iteration and with RDA models both including and not including spatial structure as a covariate. The relative contribution of each individual variable to the constrained model was calculated as a sum of correlation coefficients across the first four RDA axes, weighted by the eigenvalue for each axis. The contribution of each variable to the full RDA model was then calculated as the ratio of this value and total genetic variance (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Lasky et al., 2012</a>).</p></div><div id="sec-9" class="subsection"><h3>2.7 Identifying outlier loci</h3><p id="p-17">RDA was also used to identify outlier loci showing especially strong relationships with multivariate environmental axes. The ability of RDA to identify loci responding to combinations of environmental predictors, instead of testing and treating each variable independently, is an advantage of RDA outlier analysis over classical mixed-model association methods. Moreover, because it simultaneously evaluates combinations of SNPs, RDA may also be better able to identify more subtle patterns environmental association involving multiple minimally diverged loci (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Forester et al., 2018</a>)—suggestive of polygenic adaptation—that can be obscured by traditional approaches. Here, outliers were defined as the SNPs having loadings along the first RDA axis +/− 4 SDs from the mean for that axis (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Forester et al., 2018</a>; <a id="xref-ref-39-6" class="xref-bibr" href="#ref-39">Lasky et al., 2012</a>). To reduce the potential for spurious associations due to shared ancestry, a partial RDA conditioned on dbMEMs was implemented. Instead of using the reduced set of markers with no missing calls for this analysis, we imputed SNPs at the markers with missing data in the full dataset (44, 064 SNPs) based on the mean allelic value at each site, and used this full dataset for outlier identification. Top candidate SNPs were investigated for their proximity to known gene models, using the ITAG 3.2 tomato genome annotation (Tomato Genome <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Consortium, 2012</a>).</p></div></div><div class="section" id="sec-10"><h2 class="">3 RESULTS</h2><div id="sec-11" class="subsection"><h3>3.1 Genotype Coverage and Distribution</h3><p id="p-18">Our ddRAD procedure tagged 458,945 high coverage loci (effective per-sample coverage = 66.1x [sd = 36.7x]; average insert size = 139.5 bp; Table S3). The average number of loci seen in each individual was 60,044 (Table S4). Our genotyping pipeline identified 366,466 SNPs segregating within <em>S. pimpinellifolium</em> (Table S5). After stringent filtering for depth and missing data, this set of SNPs was reduced to 44,064. A second LD-filtered (r<sup>2</sup> &lt; 0.9) dataset was also generated for specific analyses where these data are preferable (see below); this dataset contained 17,358 SNPs (see Table S5 for details on genotype filtering). One variant occurred roughly every 18 kb. SNPs were enriched (75.6%) for sites in or nearby (within 5 kb) annotated gene models (9.4% within exons), suggesting that our enrichment based on <em>in silico</em> digestion was successful (determined by <em>SnpEff</em>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cingolani et al., 2012</a>). Of annotated SNPs incoding regions, 60.9% resulted in changes to amino acid composition (i.e., were non-synonymous).</p></div><div id="sec-12" class="subsection"><h3>3.2 Population structure follows a latitudinal gradient and defines regional genetic clusters</h3><p id="p-19">The first two axes of the multi-locus PCA based on 7,317 SNPs (LD pruned; &lt;5% missing data) explained 18.03% and 6.53% of genetic variation, respectively. Minimizing BIC in K-means clustering identified five clusters in the data (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> &amp; <a id="xref-fig-2-2" class="xref-fig" href="#F2">2B</a>), which followed a latitudinal gradient and roughly corresponded to variation along the first principal component axis. The model-based analysis conducted in <em>fastStructure</em> identified four groups in the data; these groups largely corresponded to the clusters identified using K-means, except that clusters four and five were not separated and instead were identified as a single group (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>; Figure S5). These patterns agree with other recent genomic analyses of population structure in this species (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Lin et al., 2019</a>). Varying degrees of admixture were inferred. The most heterogeneous individuals were found at the center and northern regions of the species range (clusters 1 [purple; Ecuador] and 3 [light blue; central Peru] in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), which matches general expectations for patterns of genetic diversity across species ranges (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Vucetich &amp; Waite, 2003</a>). Similarly, patterns of heterozygosity (Figure S2) generally support a Northern Peruvian origin for <em>S. pimpinellifolium</em>, in agreement with previous marker-based analyses in this species (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Rick et al., 1977</a>; <a id="xref-ref-75-4" class="xref-bibr" href="#ref-75">Zuriaga et al., 2009</a>; <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Lin et al., 2019</a>). Nonetheless, while some previous analyses have shown heterozygosity to specifically decrease in Northern Ecuador relative to Southern Ecuador and Northern Peru (corresponding to a unimodal cline in outcrossing centered on Northern Peru; <a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Rick et al., 1977</a>; <a id="xref-ref-75-5" class="xref-bibr" href="#ref-75">Zuriaga et al., 2009</a>), our data do not show a clear decline at the northern end of the species range, possibly because North Ecuador localities are more sparsely represented among our focal accessions and may not capture the northern tail of the cline in diversity.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic population structure in S. pimpinellifolium as revealed by principle components (PC) analysis (top panels) and fastStructure (bottom panel). (A) First two PC axes. Points are colored by membership in the five K-means clusters. (B) Accession collection sites colored by K-mean/BIC clin thuster membership. (C) Admixture proportions of all accessions identified with fastStructure, ordered by clusters identified in K-means to show correspondence between methods. BIC clusters 4 and 5 were identified as a single group using fastStructure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-488042217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic population structure in &lt;em&gt;S. pimpinellifolium&lt;/em&gt; as revealed by principle components (PC) analysis (top panels) and &lt;em&gt;fastStructure&lt;/em&gt; (bottom panel). (A) First two PC axes. Points are colored by membership in the five K-means clusters. (B) Accession collection sites colored by K-mean/BIC clin thuster membership. (C) Admixture proportions of all accessions identified with &lt;em&gt;fastStructure&lt;/em&gt;, ordered by clusters identified in K-means to show correspondence between methods. BIC clusters 4 and 5 were identified as a single group using &lt;em&gt;fastStructure&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F2.medium.gif" width="440" height="264"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 2" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F2.medium.gif" width="440" height="264"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 2</span> <p id="p-20" class="first-child">Genetic population structure in <em>S. pimpinellifolium</em> as revealed by principle components (PC) analysis (top panels) and <em>fastStructure</em> (bottom panel). (A) First two PC axes. Points are colored by membership in the five K-means clusters. (B) Accession collection sites colored by K-mean/BIC clin thuster membership. (C) Admixture proportions of all accessions identified with <em>fastStructure</em>, ordered by clusters identified in K-means to show correspondence between methods. BIC clusters 4 and 5 were identified as a single group using <em>fastStructure</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>3.3 RDA identifies climate and spatial variables that predict SNP variation</h3><div id="sec-14" class="subsection"><h4>3.3.1 Spatial variables</h4><p id="p-21">Forward selection of the dbMEM variables identified 10 axes as sufficient to explain geographic structure among accessions. Four of these axes described broad patterns of latitudinal structure (dbMEM1, 2, 3, and 4) and the remaining six described fine-scale patterns (dbMEM6, 7, 8, 9, 11 and 17; Figure S1). The axes describing fine-scale spatial structure cumulatively explained the most SNP variation among accessions (7.01%; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Broad-scale spatial variables explained a total of 5.42% of SNP variation (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). dbMEM2 explained the most variation of any single variable (1.74%).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1104918/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1104918/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1104918" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 1</span> <p id="p-22" class="first-child">List of dbMEM variables and their contribution to SNP variation as determined by forward variable selection with RDA. Cumulatively more variation was explained by variables describing fine-scale geographic structure (0.070) than large-scale structure (0.054). Significance in forward selection was determined by permutation tests.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h4>3.3.2 Climate variables</h4><p id="p-23">When analyzed without accounting for spatial structure using partially constrained ordination, we identified 14 abiotic variables as significantly predictive of genetic variation among accessions (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The most predictive variable was degree days above 18°C, however several variables contributed nearly equally to the RDA model. These include annual maximum solar radiation, average annual wind speed, isothermality (a measure of temperature variability), annual maximum wind speed, and variation in vapor pressure. In contrast, forward selection constrained on dbMEM spatial axes identified 6 environmental variables as predictive of genetic variation (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>); these variables represent gradients predictive of genetic variation after accounting for the proportion explained by geographic distance. Importantly, these will not include environmental gradients that are entirely colinear with space—including some that may contribute to clinal adaptation—and thus represent a conservative subset of the variables potentially contributing to geographical variation in selection. The contribution of these 6 gradients in RDA space is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. The strongest predictor was the coefficient of monthly variation in vapor pressure, followed by precipitation seasonality, soil texture, annual maximum solar radiation, maximum potential evapotranspiration, and minimum potential evapotranspiration (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1104909/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1104909/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1104909" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 2</span> <p id="p-24" class="first-child">Top environmental variables and their contribution to spatial SNP variation as determined by forward variable selection with RDA. Results from both the full RDA model selection (not conditioned on space) and partial RDA model selection (conditioned on space) are reported.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Biplot of site (accession) scores for the first two RDA axes, constraining on space with dbMEMs. Points are colored to correspond with genetic clusters identified in PCA (shown in inset, from Figure 2). Top and right axes (red) show the correlation of each environmental predictor with RDA axes 1 and 2, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-488042217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Biplot of site (accession) scores for the first two RDA axes, constraining on space with dbMEMs. Points are colored to correspond with genetic clusters identified in PCA (shown in inset, from Figure 2). Top and right axes (red) show the correlation of each environmental predictor with RDA axes 1 and 2, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F3.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 3" src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F3.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 3</span> <p id="p-25" class="first-child">Biplot of site (accession) scores for the first two RDA axes, constraining on space with dbMEMs. Points are colored to correspond with genetic clusters identified in PCA (shown in inset, from <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). Top and right axes (red) show the correlation of each environmental predictor with RDA axes 1 and 2, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The correlations of these six variables with the first two RDA axes (loadings in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) suggests that the contribution of each variable to SNP variation differs geographically across the species range, so that the environmental predictors most influential in driving SNP divergence between genetic/geographic clusters varies depending upon the specific clusters being compared. To test for the presence of regional climatic differences that may contribute to divergent selection, we studied how each environmental predictor varied by genetic cluster (Figure S3). Significant differences among genetic clusters were identified for all 6 predictors based on ANOVA tests (P &lt; 1 × 10<sup>−8</sup> for all tests), suggesting that each regional cluster occupies a more or less unique abiotic habitat. This finding is recapitulated when breaking down all 58 environmental variables into orthogonal components (Figure S4); the first two components of climatic variation separate accessions by region.</p></div></div><div id="sec-16" class="subsection"><h3>3.4 Regional climatic differences explain more SNP variation than geography</h3><div id="sec-17" class="subsection"><h4>3.4.1 Variance partitioning by RDA</h4><p id="p-27">Using RDA, the abiotic environment (14 variables identified through forward selection) and spatial structure (10 dbMEMs identified through forward selection) explained 22% of SNP variation among accessions. The majority of this (17%) was attributable to the colinear portion of environment + space (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). This fraction could represent either the contribution of clinal environmental factors or latent spatial population structure that is not captured by the dbMEMs. Considering the independent effects of environment and space, the environment alone explained more SNP variation (3%) than space alone (2%; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The observed proportions of variation explained by climate and space were greater than the proportions explained in all of 1000 permuted data sets (P &lt; 0.001).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the three variance partitioning methods. Note that SEM and GDM are unable to quantify the independent contribution of environment + space." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-488042217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary of the three variance partitioning methods. Note that SEM and GDM are unable to quantify the independent contribution of environment + space.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F4.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F4.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 4.</span> <p id="p-28" class="first-child">Summary of the three variance partitioning methods. Note that SEM and GDM are unable to quantify the independent contribution of environment + space.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h4>3.4.2 Structural equation modeling (SEM)</h4><p id="p-29">Regression components of environmental distance and geographic distance on genetic distance identified using SEM were 0.788 (SE = 0.044) and 0.174 (SE = 0.012), respectively (Table S6). This corresponds to 7% (6.4% 7.7%) and 3% (2.6% 3.5%) of total genetic variance being attributable to environmental distance and geographic distance between accessions, respectively, which is consistent with our findings from RDA (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Only the environmental regression component was higher than those obtained from all of 1000 permuted data sets (P &lt; 0.001). As with RDA, this suggests that environmental differences between accessions are shaping the distribution of genetic variation to a larger degree than spatial isolation.</p></div><div id="sec-19" class="subsection"><h4>3.4.3 Generalized dissimilarity modeling (GDM)</h4><p id="p-30">The percent deviance in genetic variation among accessions explained by the full (environment + space) GDM model was 30.2% (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>; see Table S7 for full model fit and Table S8 &amp; S9 for variable contributions). 29.2% and 23.9% of deviance was explained by models considering only environmental and geographic effects on genetic variation between accessions, respectively (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). All of these models were significant (P &lt; 0.001) based on permutation tests. These results agree with RDA and SEM; a larger fraction of SNP variation is attributable to environmental variation than to spatial isolation.</p></div></div><div id="sec-20" class="subsection"><h3>3.5 RDA identifies outlier loci associated with multivariate climate</h3><p id="p-31">We observed significant variation among SNPs in the degree to which they were associated with the abiotic environment, as measured by their loadings along the first RDA axis (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We identified candidate loci for local adaptation by examining SNPs displaying loadings along the first RDA axis +/− 4 standard deviations from the mean (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Forester et al., 2018</a>; <a id="xref-ref-39-7" class="xref-bibr" href="#ref-39">Lasky et al., 2012</a>). This identified 252 SNPs showing particularly strong associations with multivariate climate. In the model containing spatial structure as a covariate, the strongest signal of association spanned 23.5 Mbp on chromosome four (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). This region contained 333 annotated genes. The top associated SNP in this region was 30 bp upstream of a 60S ribosomal subunit gene (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Other significant associations in this region include intergenic substitutions proximal to DNA glycosylase, exocyst complex component 7, and cellulose synthase; a non-synonymous substitution in MYB transcription factor 73; and intronic substitutions in sterol glusoyltransferase 4 (SGT4), nuclease S1, BY-2 kinesin protein 5, RNA helicase DEAH-Box 13, and translation initiation factor 3, among others (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>; Table S11). No outliers identified in the analysis without spatial covariates were also present in the space + environment constrained RDA analysis. This is consistent with our general observation of a strong degree of collinearity between genetic and geographical variation in this species (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>; Table S10), and reiterates the need to include spatial structure when evaluating the independent contribution of environmental factors to SNP variation.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1104910/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1104910/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1104910" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">TABLE 3</span> <p id="p-32" class="first-child">Annotated gene models located within 10kb of the top 15 outlier SNPs along the first RDA axis, after accounting for spatial structure. We also do not report SNPs within 10kb of SNPs with higher loadings. The full list of outlier SNPs and their annotations can be found in table S8.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Values of squared RDA1 SNP loadings throughout the genome. Top panel: SNP loadings on the first RDA axis of raw SNP variation. Bottom panel: SNP loadings on the first RDA axis accounting for spatial structure among accessions. Note the change in y-axis scale between the two panels. Dashed lines represent +/&#x2212; 4 SD cutoff used to identify outliers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-488042217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Values of squared RDA1 SNP loadings throughout the genome. Top panel: SNP loadings on the first RDA axis of raw SNP variation. Bottom panel: SNP loadings on the first RDA axis accounting for spatial structure among accessions. Note the change in y-axis scale between the two panels. Dashed lines represent +/&#x2212; 4 SD cutoff used to identify outliers.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="FIGURE 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F5.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="FIGURE 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F5.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download FIGURE 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/10/744797/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">FIGURE 5.</span> <p id="p-33" class="first-child">Values of squared RDA1 SNP loadings throughout the genome. Top panel: SNP loadings on the first RDA axis of raw SNP variation. Bottom panel: SNP loadings on the first RDA axis accounting for spatial structure among accessions. Note the change in y-axis scale between the two panels. Dashed lines represent +/− 4 SD cutoff used to identify outliers.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-21"><h2 class="">4 DISCUSSION</h2><p id="p-34">Local adaptation to the abiotic environment is often thought to play a major role in diversification both within and between species, however its contribution relative to other evolutionary forces is rarely quantified. In this study, we investigated the relationship of genome-wide SNPs with fine-scale climatic and spatial data among 140 accessions of the ecologically diverse crop wild relative <em>S. pimpinellifolium</em>. Our study affirms previous work demonstrating broad correlations between genetic and climatic structure in this system (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Blanca et al., 2012</a>, <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">2015</a>; <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Lin et al., 2019</a>; Zuriaga et al., 2012), and builds beyond this to quantify the contributions of individual environmental variables to patterns of genomic variation and dissect the genetic targets of these putative selective gradients in a multivariate statistical context. We found that the abiotic environment—as described by climate and soil characteristics—explained more SNP variation than spatial structure (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>), demonstrating that our evaluated environmental factors are more influential in shaping the geographic distributions of SNPs in this species, compared to spatial isolation. Both GDM and SEM—two methods which have also been applied to quantifying the contributions of space and environment to patterns of genetic variation—agree with our RDA results. From this we conclude that ecological differences among populations are closely associated with the maintenance of contemporary genetic variation across space and that local adaptation is likely a major driver of these patterns.</p><p id="p-35">Our RDA analysis identified that a large (17%) portion of SNP variation among accessions was explainable by the colinear fraction of environment and space (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>). This fraction could represent the contribution of clinal environmental forces that are highly correlated with space (e.g., latitudinal gradients in temperature). Clinal gradients are considered to be important drivers of adaptive evolution in many species (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adrion et al., 2015</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kooyers et al., 2015</a>), and across the range of <em>S. pimpinellifolium</em>, many environmental variables show strong geographic trends and are correlated with both genetic (i.e., PCs) and spatial (i.e., dbMEMs) structure (Table S10). Previous studies in <em>S. pimpinellifolium</em> have demonstrated strong correlations between environmental and genetic distances among accessions of this species, largely associated with latitudinal gradients (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Blanca et al., 2015</a>; <a id="xref-ref-75-6" class="xref-bibr" href="#ref-75">Zuriaga et al., 2009</a>). Accordingly, when we did not correct for spatial structure with constrained ordination, a highly clinal variable—degree days above 18C—explained the most total SNP variation (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Although geographic clines have historically been considered important for understanding the genetics of adaptation (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adrion et al., 2015</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Endler, 1977</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">1986</a>), disentangling geographically structured variation from adaptive variation remains a major challenge—as evidenced here—that will likely require novel statistical methods and experimental studies to resolve. In <em>S. pimpinellifolium</em>, further population sampling across clinal gradients coupled with <em>in situ</em> common garden experiments could be one way of disentangling the independent effects of space and climate in these cases.</p><p id="p-36">Despite the substantial collinearity we observed between space and environment, our analysis discerned several axes of spatial variation that uniquely explained SNP variation. The second spatial dbMEM variable explained the largest portion of SNP variation among accessions (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). This variable describes broad patterns of spatial structure and separates accessions from the center of the range (northern Peru) from those at the northern and southern extremes (Figure S1). This finding is consistent with our genetic analysis of population structure—as well as previous spatial genetic data (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Rick et al., 1977</a>; <a id="xref-ref-75-7" class="xref-bibr" href="#ref-75">Zuriaga et al., 2009</a>; <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Lin et al., 2019</a>)—which demonstrate a strong latitudinal pattern of genetic relatedness. Interestingly, while the second dbMEM axis explained the most variance of any single spatial variable, cumulatively more variance was explained by fine-scale dbMEM axes (dbMEM5, 6, 7, 8, 9, &amp; 17; cumulative R<sup>2</sup> = 0.0701). Such a pattern may be driven by the geographic landscape of the Andes; large and patchy altitudinal clines (sea level to &gt; 6500 m) may limit local dispersal between adjacent populations, amplifying genetic differentiation over small spatial scales.</p><p id="p-37">Apart from spatial variation associated with SNPs, our analysis also identified specific environmental factors that uniquely explained additional SNP variation. Temporal fluctuations in precipitation (seasonality) and vapor pressure (the coefficient of monthly variation) explained the most genome-wide SNP variation when we simultaneously considered the effects of spatial structure in our RDA analysis (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). This reinforces some previous empirical findings—especially the role of precipitation seasonality as a key factor—but also points to additional environmental forces as potential abiotic selective agents in this species. Precipitation was previously implicated in driving ecological adaptation in both this and other tomato species (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Nakazato et al., 2008</a>, <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">2010</a>). Specifically, among accessions of <em>S. pimpinellifolium</em>, variation in days to wilting (a measure of drought tolerance) is strongly associated with differences in annual precipitation between coastal and highland regions (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Nakazato et al., 2008</a>), consistent with selection being imposed by this environmental axis. Interestingly, our findings at the intraspecific scale are also similar to previous inferences of important ecological drivers among wild tomato species. Distribution models have identified precipitation and temperature as the major determinants of differences in contemporary species ranges across the wild tomato clade (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Nakazato et al., 2010</a>). Moreover, these analyses indicate that precipitation seasonality in particular is important for determining the allopatric distributions of <em>S. pimpinellifolium</em> and its sister taxon <em>S. lycopersicum</em> var. <em>cerasiforme</em>.</p><p id="p-38">In contrast to precipitation, the importance of vapor pressure as a selective agent within or among wild tomato species has not been evaluated, although vapor pressure’s role in modulating plant-water relations has been well studied in domesticated tomato (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Guichard et al., 2005</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zhang et al., 2017</a>). Deficits in vapor pressure between the mesophyll and external leaf surface promotes transpiration (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Zhang et al., 2017</a>), and if not met with additional water uptake from the soil, large differentials in vapor pressure will lead to reduced growth and desiccation. Our findings from RDA suggest that regional differences in vapor pressure are a strong selective force in <em>S. pimpinellifolium</em> (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). The importance of variation in vapor pressure—and not a measure of annual or monthly mean values—may point to a role for specific physiological mechanisms which maximize fitness during acute fluctuations in evaporative demand (e.g., rapid stomatal closure responses, induction of root growth, or other strategies for increasing water use efficiency under temporary stress). This hypothesis is testable in the future with direct manipulative experiments across accessions that are predicted to differ at specific SNPs associated with this axis of the environment.</p><p id="p-39">While variation in vapor pressure and precipitation were the most predictive of genomic variation across the range as a whole (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>), regional environmental differences and our RDA analysis suggest that different specific factors are most influential in different regions of the species range (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>; Figure S3). Genetic clusters we identify within <em>S. pimpinellifolium</em> are, in general, significantly diverged from each other for their environments (Figure S3; Figure S4). For example, coastal accessions (genetic clusters 1, 4, and partially 3) are exposed to higher levels of solar radiation compared to the rest of the range (Figure S3, panel 3); northern Peruvian accessions (cluster 2) experience highly seasonal precipitation (Figure S3; panel 2); and Ecuadorian accessions (cluster 5) experience high soil type diversity (i.e., all four soil types that found throughout the species range are also present in this region; Figure S3, panel 6). Together with our RDA results—which identified these and other variables as significantly predictive of SNP variation independent of spatial population structure—these patterns of environmental variation are consistent with regional differences in the primary agents of ecological selection. Unlike studies of genetic variation across precipitation gradients (see above), equivalent analyses of genetic variation for tolerance to these environmental factors (i.e., solar radiation, soil texture, and potential evapotranspiration) is currently limited. Further common garden manipulation experiments would be helpful to understand the exact role these forces have in shaping local climatic adaptation in <em>S. pimpinellifolium</em>. Nonetheless, these observations broadly mirror comparative findings from among wild tomatoes—that detect clear niche differentiation between species (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Nakazato et al., 2010</a>)—by uncovering similar niche variation intraspecifically at the regional level.</p><p id="p-40">In addition to these findings, and just as with other genetic (<a id="xref-ref-39-8" class="xref-bibr" href="#ref-39">Lasky et al., 2012</a>, 2016) and community ecological studies using similar methods (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Borcard et al., 2018</a>), a large portion of variance remained unexplained. This is likely due to the influence of several factors that cannot be fully addressed in this type of experiment. First, although we included a large set of environmental variables in our analyses, many other unmeasured ecological forces could also be acting; most importantly, this includes biotic interactions that are likely only partially reflected in the abiotic factors considered here. Second, axes of population structure may not be fully explained by spatial dbMEM variables, potentially due to unmodeled geographic heterogeneity present in the Andes. Third, other evolutionary forces that maintain local allelic diversity (e.g., balancing selection) may weaken signals of SNP-environment associations. Lastly, as implemented here, both RDA and SEM only model linear associations between climate/space and SNPs (<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Borcard et al., 2018</a>), and thus will not capture non-linear statistical relationships if they exist. Interestingly, our GDM analysis—which does model nonlinear associations—explained more total SNP variation than the other two methods (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a>), suggesting the presence of nonlinear SNP-environment relationships across the range of <em>S. pimpinellifolium</em>. Explicitly modeling and estimating the contribution of nonlinear terms to patterns of SNP-environment associations may be crucial to understanding local adaptation in the context of multivariate environments, but such methods are currently limited (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rellstab et al., 2015</a>).</p><p id="p-41">Our SNP outlier analysis allowed us to map the location of genomic regions showing strong associations with major axes of the environment and identify candidate loci potentially involved in local adaptation (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). In contrast to traditional scans that treat each environmental variable independently (e.g., <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Fournier-Level et al., 2011</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hancock et al., 2011</a>), our analysis considered how loci are associated with linear combinations of environmental gradients. This framework may be more appropriate for identifying ecologically relevant loci; rarely can a single environmental factor be considered the only contributor to selection. We identified 252 signals of putatively adaptive loci in <em>S. pimpinellifolium</em> using genomic scans based on RDA loadings. Of foremost interest are two regions containing known regulators of abiotic stress tolerance: Solyc04g051150—sterol glucosyl transferase 4 (SGT4)—and Solyc04g050080—MYB transcription factor 73 (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). SGT genes have been shown to have functional roles in abiotic stress tolerance in several species (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Griebel &amp; Zeier, 2010</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hugly et al., 1990</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kumar et al., 2015</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Senthil-Kumar et al., 2013</a>). Expression of SGT4 and its paralog SGT2 is rapidly induced upon exposure to cold and salt stress in domesticated tomato (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Ramirez-Estrada et al., 2017</a>). In Arabidopsis thaliana SGT knock-out mutants, plants show reduced tolerance to several abiotic stressors, including cold and salinity (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mishra et al., 2015</a>). MYB 73 acts as a salt stress-specific transcription factor and modulates whole-plant responses to salinity in A. thaliana (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kim et al., 2013</a>). Expression profiles of domesticated tomato under hormone-induced stress suggest a general role for MYB transcription factors in responding to abiotic cues (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Li et al., 2016</a>), although the specific function of MYB 73 has not been evaluated in Solanum. While genotypic variance in salt-stress tolerance has been documented in <em>S. pimpinellifolium</em> (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rao et al., 2013</a>)—suggesting that functional variation in salt-response loci may be prevalent—our analyses were not able to evaluate the contribution or association off this factor to genome-wide SNP variation, since fine-scale data on soil salinity in South America is not yet available. Nonetheless, these scans provide new information on the location of loci relevant for adaptive responses to abiotic stress, generating prime candidates for future functional investigation, as well as for breeding into domesticated lines.</p><p id="p-42">Overall, our study adds to an emerging landscape genomics literature applying an ensemble of multivariate methods to dissect the demographic and environmental features shaping patterns of genomic differentiation. Quantifying the influence of local adaptation to climatic variation has profound importance for predicting how species will respond to future climates, for the breeding of resilient cultivars, and for broadly understanding selection as a mechanism of diversification. Here we generate a strong case that environmentally imposed selection plays a key role in genetic differentiation within a widespread and diverse crop wild relative, and distinguish the climatic factors most likely responsible. All three multivariate methods (RDA, SEM, and GDM) identified that climatic variation among our 140 <em>S. pimpinellifolium</em> accessions explained more genomic variation than does geographic distance, with temporal variation in precipitation and evaporative demand explaining the most SNP variation overall. Our analysis also identified significant geographical differences in other key climatic factors that suggest additional intraspecific niche differentiation between geographic regions. These results complement comparative findings from the wild tomato clade (<a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Nakazato et al., 2010</a>) indicating that spatially varying abiotic selection may be an important source of genetic variability in this species, but also demonstrate the additional qualitative and quantitative insight that can be gained by applying a lens of landscape genomics. This additional power includes uncovering genomic regions with strong environmental associations—potential genomic targets of ecological selection—that can be passed to further studies to understand their exact genetic and phenotypic contributions, for example, by incorporating SNP-environment associations into breeding value prediction frameworks (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lasky et al., 2015</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Yoder et al., 2012</a>). Together, our findings add to a growing literature demonstrating a pervasive role of selection in shaping patterns of genomic differentiation, and the likely multilocus architecture that underlies adaptive differentiation, using landscape genomics. Additional studies in a broader range of species and ecological contexts will be critical for assessing additional generalities about the demographic and environmental features most influential in defining patterns of genetic variation.</p></div><div class="section" id="sec-23"><h2 class="">Funding information</h2><p id="p-43">This research was supported by National Science Foundation award DEB-1136707 to LCM</p></div><div class="section" id="sec-24"><h2 class="">A<span class="sc">uthor</span> C<span class="sc">ontributions</span></h2><p id="p-45">MJSG and LCM designed the experiments; MJSG performed the greenhouse and molecular work and conducted the bioinformatic analyses; MJSG and LCM wrote the manuscript.</p></div><div class="section COI-statement" id="sec-25"><h2 class="">C<span class="sc">onflict of</span> I<span class="sc">nterest</span></h2><p id="p-46">The authors declare no conflicts of interest.</p></div><div class="section ack" id="ack-1"><h2 class="">A<span class="sc">ckowledgements</span></h2><p id="p-44">The authors thank the UC Davis Tomato Genetic Resource Center for providing access to the <em>S. pimpinellifolium</em> germplasm used in this study, the Indiana University Center for Genomics and Bioinformatics for library preparation and sequencing, the Indiana University greenhouse staff, and three anonymous reviewers for comments that improved a previous version of this manuscript. This research was supported by National Science Foundation award DEB1136707 to LCM.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-1"><p id="p-3">
<a href="https://github.com/gibsonMatt/pimpGEA">https://github.com/gibsonMatt/pimpGEA</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">REFERENCES</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.1" data-doi="10.1016/j.tig.2015.05.006"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Adrion</span>,  <span class="cit-name-given-names">J. R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hahn</span>,  <span class="cit-name-given-names">M. W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">B. S.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Revisiting classic clines in Drosophila melanogaster in the age of genomics</span>. <abbr class="cit-jnl-abbrev">Trends in Genetics</abbr>, <span class="cit-vol">31</span>, <span class="cit-fpage">434</span>–<span class="cit-lpage">444</span>. URL: <a href="http://dx.doi.org/10.1016/j.tig.2015.05.006">http://dx.doi.org/10.1016/j.tig.2015.05.006</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BGenetics%26rft.volume%253D31%26rft.spage%253D434%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2015.05.006%26rft_id%253Dinfo%253Apmid%252F26072452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2015.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26072452&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Andrews</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">FastQC: a quality control tool for high throughput sequence data</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.3" data-doi="10.1371/journal.pone.0048198"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blanca</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Canizares</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cordero</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pascual</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diez</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nuez</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Variation Revealed by SNP Genotyping and Morphology Provides Insight into the Origin of the Tomato</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">e48198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBlanca%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De48198%26rft.epage%253De48198%26rft.atitle%253DVariation%2Brevealed%2Bby%2BSNP%2Bgenotyping%2Band%2Bmorphology%2Bprovides%2Binsight%2Binto%2Bthe%2Borigin%2Bof%2Bthe%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0048198%26rft_id%253Dinfo%253Apmid%252F23118951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0048198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23118951&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.4" data-doi="10.1186/s12864-015-1444-1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blanca</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Montero-Pau</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sauvage</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bauchet</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Illa</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diez</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Francis</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Causse</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Knaap</span>,  <span class="cit-name-given-names">E.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Canizares</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Genomic variation in tomato, from wild ancestors to contemporary breeding accessions</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D257%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1444-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1444-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bolarin</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fernandez</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cruz</span>,  <span class="cit-name-given-names">V.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Cuartero</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1991</span>) <span class="cit-article-title">Salinity tolerance in four wild tomato species using vegetative yieldsalinity response curves</span>. <abbr class="cit-jnl-abbrev">Journal of the American Society for Horticultural Science</abbr>, <span class="cit-vol">116</span>, <span class="cit-fpage">286</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BSociety%2Bfor%2BHorticultural%2BScience%26rft.stitle%253Djashs%26rft.aulast%253DBolarin%26rft.auinit1%253DM.C.%26rft.volume%253D116%26rft.issue%253D2%26rft.spage%253D286%26rft.epage%253D290%26rft.atitle%253DSalinity%2BTolerance%2Bin%2BFour%2BWild%2BTomato%2BSpecies%2Busing%2BVegetative%2BYield-Salinity%2BResponse%2BCurves%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiamFzaHMiO3M6NToicmVzaWQiO3M6OToiMTE2LzIvMjg2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTAvNzQ0Nzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-744797v2.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Borcard</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gillet</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Legendre</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-chapter-title">Numerical Ecology with R</span>. <span class="cit-publ-name">Springer International</span>, <span class="cit-edition">2 edn</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.7" data-doi="10.1111/mec.12354"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Catchen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bassham</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amores</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Cresko</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Stacks: an analysis tool set for population genomics</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">22</span>, <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.8" data-doi="10.1093/bioinformatics/bty560"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">Y.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">fastp: an ultra-fast all-in-one FASTQ preprocessor</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">34</span>, <span class="cit-fpage">i884</span>–<span class="cit-lpage">i890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253Di884%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty560%26rft_id%253Dinfo%253Apmid%252F30423086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30423086&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.9" data-doi="10.1371/journal.pone.0029570"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Headland</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Filiault</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jimenez-Gomeez</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schrager</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sinha</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Maloof</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2012a</span>) <span class="cit-article-title">Native environment modulates leaf size and response to simulated foliar shade across wild tomato species</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">e29570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChitwood%26rft.auinit1%253DD.%2BH.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253De29570%26rft.epage%253De29570%26rft.atitle%253DNative%2Benvironment%2Bmodulates%2Bleaf%2Bsize%2Band%2Bresponse%2Bto%2Bsimulated%2Bfoliar%2Bshade%2Bacross%2Bwild%2Btomato%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0029570%26rft_id%253Dinfo%253Apmid%252F22253737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0029570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22253737&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.10" data-doi="10.1104/pp.111.192518"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Headland</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Maloof</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sinha</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2012b</span>) <span class="cit-article-title">The developmental trajectory of leaflet morphology in wild tomato species</span>. <abbr class="cit-jnl-abbrev">Plant Physiology</abbr>, <span class="cit-vol">158</span>, <span class="cit-fpage">1230</span>–<span class="cit-lpage">1240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.192518%26rft_id%253Dinfo%253Apmid%252F22247269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU4LzMvMTIzMCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzEwLzc0NDc5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.11" data-doi="10.4161/fly.19393"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cingolani</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Platts</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coon</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyem</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Landm</span>  <span class="cit-name-given-names">S.J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">X.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ruden</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3</span>. <abbr class="cit-jnl-abbrev">Fly</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFly%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.4161%252Ffly.19393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/fly.19393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.12" data-doi="10.1038/nature11119"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Consortium</span>,  <span class="cit-name-given-names">T. G.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">The tomato genome sequence provides insights into fleshy fruit evolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">485</span>, <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D485%26rft.issue%253D7400%26rft.spage%253D635%26rft.epage%253D641%26rft.atitle%253DThe%2Btomato%2Bgenome%2Bsequence%2Bprovides%2Binsights%2Binto%2Bfleshy%2Bfruit%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11119%26rft_id%253Dinfo%253Apmid%252F22660326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660326&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304608000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.13" data-doi="10.1111/j.1469-8137.1992.tb01093.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cuartero</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeo</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Flowers</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">1992</span>) <span class="cit-article-title">Selection of donors for salt-tolerance in tomato using physiological traits</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr>, <span class="cit-vol">121</span>, <span class="cit-fpage">63</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D121%26rft.spage%253D63%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.1992.tb01093.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.1992.tb01093.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.14" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Auton</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abecasis</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albers</span>,  <span class="cit-name-given-names">C. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Banks</span>,  <span class="cit-name-given-names">E.</span></span>, De<span class="cit-auth"><span class="cit-name-surname">Pristo</span>,  <span class="cit-name-given-names">M. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Handsaker</span>,  <span class="cit-name-given-names">R. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lunter</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marth</span>,  <span class="cit-name-given-names">G. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sherry</span>,  <span class="cit-name-given-names">S. T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth cit-collab">Group,. G. P. A.</span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">The variant call format and VCFtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>. URL: <a href="https://doi.org/10.1093/bioinformatics/btr330">https://doi.org/10.1093/bioinformatics/btr330</a>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-744797v2.15" data-doi="10.1111/j.1749-6632.2010.05703.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Des Marais</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Juenger</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Pleiotropy, plasticity, and the evolution of plant abiotic stress tolerance</span>. <abbr class="cit-jnl-abbrev">Annals of the New York Academy of Sciences</abbr>, <span class="cit-vol">1206</span>, <span class="cit-fpage">56</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DDes%2BMarais%26rft.auinit1%253DD.%2BL.%26rft.volume%253D1206%26rft.spage%253D56%26rft.epage%253D79%26rft.atitle%253DPleiotropy%252C%2Bplasticity%252C%2Band%2Bthe%2Bevolution%2Bof%2Bplant%2Babiotic%2Bstress%2Btolerance.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2010.05703.x%26rft_id%253Dinfo%253Apmid%252F20860683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2010.05703.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20860683&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283626300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-744797v2.16" data-doi="10.1105/tpc.112.096180"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Des Marais</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wayne</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Juenger</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Physiological genomics of response to soil drying in diverse Arabidopsis acceessions</span>. <abbr class="cit-jnl-abbrev">The Plant cell</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">893</span>–<span class="cit-lpage">914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bcell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.112.096180%26rft_id%253Dinfo%253Apmid%252F22408074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI0LzMvODkzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTAvNzQ0Nzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-744797v2.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Endler</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1977</span>) <span class="cit-chapter-title">Geographic Variation, Speciation, and Clines</span>. <span class="cit-publ-loc">Princeton, NJ.</span>: <span class="cit-publ-name">Princeton University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-744797v2.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Endler</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1986</span>) <span class="cit-chapter-title">Natural Selection in the Wild</span>. <span class="cit-publ-loc">Princeton, NJ.</span>: <span class="cit-publ-name">Princeton University Press</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ferrier</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manion</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elith</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Richardson</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Using generalized dissimilarity modelling to analyse and predict patterns of beta diversity in regional biodiversity assessment</span>. <abbr class="cit-jnl-abbrev">Diversity and Distributions</abbr>, <span class="cit-vol">13</span>, <span class="cit-fpage">252</span>–<span class="cit-lpage">264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2Band%2BDistributions%26rft.volume%253D13%26rft.spage%253D252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.20"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fick</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Worldclim 2: New 1-km spatial resolution climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">Intenational Jounrnal of Climatology</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Foolad</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yin</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Genetic potential for salt tolerance during germination in Lycopersicon species</span>. <abbr class="cit-jnl-abbrev">HortScience</abbr>, <span class="cit-vol">32</span>, <span class="cit-fpage">296</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortScience%26rft.stitle%253Dhortsci%26rft.aulast%253DFoolad%26rft.auinit1%253DM.R.%26rft.volume%253D32%26rft.issue%253D2%26rft.spage%253D296%26rft.epage%253D300%26rft.atitle%253DGenetic%2BPotential%2Bfor%2BSalt%2BTolerance%2BDuring%2BGermination%2Bin%2BLycopersicon%2BSpecies%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiaG9ydHNjaSI7czo1OiJyZXNpZCI7czo4OiIzMi8yLzI5NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzEwLzc0NDc5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-744797v2.22"><div class="cit-metadata"><ol class="duplicate"><li><span class="cit-ed"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jenks</span></span>, </li><li><span class="cit-ed"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hasegawa</span></span> and </li><li><span class="cit-ed"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jain</span></span></li></ol><cite><span class="cit-auth"><span class="cit-name-surname">Foolad</span>,  <span class="cit-name-given-names">M. R.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-chapter-title">Current status of breeding tomatoes for salt and drought tolerance</span>. In. <span class="cit-source">Advances in Molecular Breeding Toward Drought and Salt Tolerant Crops</span> (eds. <span class="cit-ed"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jenks</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hasegawa</span></span> and <span class="cit-ed"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jain</span></span>). <span class="cit-publ-name">Springer</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.23" data-doi="10.1111 /mec. 14584"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Forester</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Urbal</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Comparing methods for detecting multilocus adaptation with multivariate genotype-environment associations</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">2215</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D27%26rft.spage%253D2215%26rft_id%253Dinfo%253Adoi%252F10.1111%2B%252Fmec.%2B14584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111 /mec. 14584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.24" data-doi="10.1126/science.1209271"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fournier-Level</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korte</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cooper</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nordborg</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmitt</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wilczek</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">A map of local adaptation in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">334</span>, <span class="cit-fpage">86</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFournier-Level%26rft.auinit1%253DA.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D86%26rft.epage%253D89%26rft.atitle%253DA%2BMap%2Bof%2BLocal%2BAdaptation%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209271%26rft_id%253Dinfo%253Apmid%252F21980109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84NiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzEwLzc0NDc5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.25" data-doi="10.1111/j.1365-313X.2010.04235.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Griebel</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Zeier</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">A role for beta-sitosterol to stigmasterol conversion in plant-pathogen interactions</span>. <abbr class="cit-jnl-abbrev">Plant Journal</abbr>, <span class="cit-vol">63</span>, <span class="cit-fpage">254</span>–<span class="cit-lpage">268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DGriebel%26rft.auinit1%253DT.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D254%26rft.epage%253D268%26rft.atitle%253DA%2Brole%2Bfor%2Bbeta-sitosterol%2Bto%2Bstigmasterol%2Bconversion%2Bin%2Bplant-pathogen%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2010.04235.x%26rft_id%253Dinfo%253Apmid%252F20444228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2010.04235.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20444228&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279750800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.26" data-doi="10.1007/s00344-005-0040-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Guichard</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gary</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leonardi</span>,  <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bertin</span>,  <span class="cit-name-given-names">N.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Analysis of Growth and Water Relations of Tomato Fruits in Relation to Air Vapor Pressure Deficit and Plant Fruit Load</span>. <abbr class="cit-jnl-abbrev">Journal Of Plant Growth Regulation</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2BOf%2BPlant%2BGrowth%2BRegulation%26rft.volume%253D24%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00344-005-0040-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00344-005-0040-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233646800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hamann</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spittlehouse</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Murdock</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">A comprehensive, high-resolution database of historical and projected climate surfaces for western North America</span>. <abbr class="cit-jnl-abbrev">Bulletin of the American Meteorological Society</abbr>, <span class="cit-vol">94</span>, <span class="cit-fpage">1307</span>–<span class="cit-lpage">1309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBulletin%2Bof%2Bthe%2BAmerican%2BMeteorological%2BSociety%26rft.volume%253D94%26rft.spage%253D1307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.28" data-doi="10.1126/science.1209244"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hancock</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brachi</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faure</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Horton</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jarymowycz</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperone</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Toomajian</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roux</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bergelson</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Adaptation to Climate Across the Arabidopsis thaliana Genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">334</span>, <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzEwLzc0NDc5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.29" data-doi="10.1371/journal.pone.0169748"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hengl</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mendes de Jesus</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heuvelink</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruiperez Gonzalez</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kilibarda</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blagotic</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shangguan</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wrightt</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Geng</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bauer-Marschallinger</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guevara</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vargas</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacMillan</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Batjes</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leenaars</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wheeler</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mantel</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Kempen</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Soil-Grids250m: Global gridded soil information based on machine learning</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-vol">12</span>, <span class="cit-fpage">e0169748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0169748%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0169748%26rft_id%253Dinfo%253Apmid%252F28207752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0169748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28207752&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-744797v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hodgins-Davis</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Towsend</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Evolving gene expression: from G to E to G x E</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">649</span>–<span class="cit-lpage">658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D24%26rft.spage%253D649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.31" data-doi="10.1104/pp.93.3.1053"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hugly</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCourt</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Browse</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Somrville</span>,  <span class="cit-name-given-names">C.</span></span> (<span class="cit-pub-date">1990</span>) <span class="cit-article-title">A Chilling Sensitive Mutant of Arabidopsis with Altered Steryl-Ester Metabolism</span>. <abbr class="cit-jnl-abbrev">Plant Physiology</abbr>, <span class="cit-vol">93</span>, <span class="cit-fpage">1053</span>–<span class="cit-lpage">1062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DHugly%26rft.auinit1%253DS.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D1053%26rft.epage%253D1062%26rft.atitle%253DA%2BChilling%2BSensitive%2BMutant%2Bof%2BArabidopsis%2Bwith%2BAltered%2BSteryl-Ester%2BMetabolism%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.93.3.1053%26rft_id%253Dinfo%253Apmid%252F16667557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiI5My8zLzEwNTMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8xMC83NDQ3OTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-744797v2.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ma</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cao</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">WRKY transcription factors in plant responses to stresses</span>. <abbr class="cit-jnl-abbrev">Journal of Integrative Plant Biology</abbr>, <span class="cit-vol">59</span>, <span class="cit-fpage">86</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BIntegrative%2BPlant%2BBiology%26rft.volume%253D59%26rft.spage%253D86%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-744797v2.33" data-doi="10.1093/jxb/erw285"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jissoudis</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sunarti</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van de Wiel</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Visser</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Linden</span>,  <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bai</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Responses to combined abiotic and biotic stress in tomato are governed by stress intensity and resistance mechanism</span>. <abbr class="cit-jnl-abbrev">Journal of Experimental Botany</abbr>, <span class="cit-vol">67</span>, <span class="cit-fpage">5119</span>–<span class="cit-lpage">5132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBotany%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferw285%26rft_id%253Dinfo%253Apmid%252F27436279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erw285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27436279&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.34" data-doi="10.1093/bioinformatics/btr521"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jombart</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ahmed</span>,  <span class="cit-name-given-names">I.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Adegenet 1.3-1: new tools for the analysis of genome-wide SNP data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics2</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">3070</span>–<span class="cit-lpage">3071</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics2%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr521%26rft_id%253Dinfo%253Apmid%252F21926124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21926124&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.35" data-doi="10.1016/j.jplph.2013.05.011"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeong</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hong</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Loss of the R2R3 MYB, AtMyb73, causes hyperinduction of the SOS1 and SOS3 genes in response to high salinity in Arabidopsis</span>. <abbr class="cit-jnl-abbrev">Journal of Plant Physiology</abbr>, <span class="cit-vol">170</span>, <span class="cit-fpage">1461</span>–<span class="cit-lpage">1465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPlant%2BPhysiology%26rft.volume%253D170%26rft.spage%253D1461%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jplph.2013.05.011%26rft_id%253Dinfo%253Apmid%252F23809151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jplph.2013.05.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23809151&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325301500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-744797v2.36" data-doi="10.1111/tpj.13312"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klepikova</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kasianov</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gerasimov</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Logacheva</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Penin</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">A high resolution map of the Arabidopsis thaliana developmental transcriptome based on RNA-seq profiling</span>. <abbr class="cit-jnl-abbrev">Plant Journal</abbr>, <span class="cit-vol">88</span>, <span class="cit-fpage">1058</span>–<span class="cit-lpage">1070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJournal%26rft.volume%253D88%26rft.spage%253D1058%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13312%26rft_id%253Dinfo%253Apmid%252F27549386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27549386&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.37" data-doi="10.1111/nph.13153"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kooyers</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Greenlee</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colicchio</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oh</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Blackman</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Replicate altitudinal clines reveal that evolutionary flexibility underlies adaptation to drought stress in annual Mimulus guttatus</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr>, <span class="cit-vol">206</span>, <span class="cit-fpage">152</span>–<span class="cit-lpage">165</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.volume%253D206%26rft.spage%253D152%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13153%26rft_id%253Dinfo%253Apmid%252F25407964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25407964&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.38" data-doi="10.1016/j.plaphy.2015.07.014"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ali</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dahuja</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tyagi</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Role of phytosterols in drought stress tolerance in rice</span>. <abbr class="cit-jnl-abbrev">Plant Physiology and Biochemistry</abbr>, <span class="cit-vol">96</span>, <span class="cit-fpage">83</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%2Band%2BBiochemistry%26rft.volume%253D96%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plaphy.2015.07.014%26rft_id%253Dinfo%253Apmid%252F26233709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plaphy.2015.07.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26233709&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.39" data-doi="10.1111/j.1365-294X.2012.05709.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lasky</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Des Marais</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">McKay</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richards</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juenger</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Keitt</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Characterizing genomic variation of Arabidopsis thaliana: the roles of geography and climate</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">21</span>, <span class="cit-fpage">5512</span>–<span class="cit-lpage">5529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLasky%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D21%26rft.issue%253D22%26rft.spage%253D5512%26rft.epage%253D5529%26rft.atitle%253DCharacterizing%2Bgenomic%2Bvariation%2Bof%2BArabidopsis%2Bthaliana%253A%2Bthe%2Broles%2Bof%2Bgeography%2Band%2Bclimate.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05709.x%26rft_id%253Dinfo%253Apmid%252F22857709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05709.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22857709&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310583800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.40" data-doi="10.1126/sciadv.1400218"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lasky</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ramu</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Deshhhpande</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hash</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonnette</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Juenger</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hyma</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Acharya</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mitchell</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckler</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brenton</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kresovich</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Morris</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Genome-environment associations in sorghum landraces predict adaptive traits</span>. <abbr class="cit-jnl-abbrev">Science Advances</abbr>, <span class="cit-vol">1</span>, <span class="cit-fpage">e1400218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DLasky%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253De1400218%26rft.epage%253De1400218%26rft.atitle%253DGenome-environment%2Bassociations%2Bin%2Bsorghum%2Blandraces%2Bpredict%2Badaptive%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1400218%26rft_id%253Dinfo%253Apmid%252F26601206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMS82L2UxNDAwMjE4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTAvNzQ0Nzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.41" data-doi="10.1111/j.1365-294X.2011.05310.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Mitchell-Olds</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Quantifying effects of environmental and geographical factors on patterns of genetic differentiation</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">20</span>, <span class="cit-fpage">4631</span>–<span class="cit-lpage">4642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLee%26rft.auinit1%253DC.%2BR.%26rft.volume%253D20%26rft.issue%253D22%26rft.spage%253D4631%26rft.epage%253D4642%26rft.atitle%253DQuantifying%2Beffects%2Bof%2Benvironmental%2Band%2Bgeographical%2Bfactors%2Bon%2Bpatterns%2Bof%2Bgenetic%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05310.x%26rft_id%253Dinfo%253Apmid%252F21999331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05310.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21999331&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297414200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-744797v2.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Legendre</span>,  <span class="cit-name-given-names">P.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Legendre</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-chapter-title">Numerical Ecology</span>. <span class="cit-publ-loc">New York</span>: <span class="cit-publ-name">Elsevier</span>, <span class="cit-edition">2 edn</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.43" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.44" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Durbin</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler Transform</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.45" data-doi="10.1093/pcp/pcw091"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yao</span>,  <span class="cit-name-given-names">Q.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Genome-Wide Identification and Analysis of the MYB Transcription Factor Superfamily in Solanum lycopersicum</span>. <abbr class="cit-jnl-abbrev">Plant Cell and Physiology</abbr>, <span class="cit-vol">57</span>, <span class="cit-fpage">1657</span>–<span class="cit-lpage">1677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2Band%2BPhysiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcw091%26rft_id%253Dinfo%253Apmid%252F27279646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/pcp/pcw091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27279646&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.46" data-doi="10.1534/g3.118.200862"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Assessment of Genetic Differentiation and Linkage Disequilibrium in Solanum pimpinellifolium Using Genome-Wide High-Density SNP Markers</span>. <abbr class="cit-jnl-abbrev">G3</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">1497</span>–<span class="cit-lpage">1505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200862%26rft_id%253Dinfo%253Apmid%252F30858236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjkvNS8xNDk3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTAvNzQ0Nzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.47" data-doi="10.1111/j.1365-294X.2010.04716.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manel</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Poncet</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legendre</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gugerli</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Holderegger</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Common factors drive adaptive genetic variation at different spatial scales in Arabis alpina</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">3824</span>–<span class="cit-lpage">3835</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DManel%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D3824%26rft.epage%253D3835%26rft.atitle%253DCommon%2Bfactors%2Bdrive%2Badaptive%2Bgenetic%2Bvariation%2Bat%2Bdifferent%2Bspatial%2Bscales%2Bin%2BArabis%2Balpina.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04716.x%26rft_id%253Dinfo%253Apmid%252F20723057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04716.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20723057&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281285200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manion</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lisk</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrier</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nieto-Lugilde</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mokany</span>,  <span class="cit-name-given-names">K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">gdm: Generalized Dissimilarity Modeling</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mishra</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tiwari</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumari</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Misrra</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Characterization of Arabidopsis sterol glycosyltransferase TTG15/UGT80B1 role during freeze and heat stress</span>. <abbr class="cit-jnl-abbrev">Plant Signaling &amp; Behavior</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">e1075682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSignaling%2B%2526%2BBehavior%26rft.volume%253D10%26rft.spage%253D1075682e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.50" data-doi="10.1111/j.1558-5646.2008.00332.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nakazato</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bogonovich</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Moyle</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Environmental factors predict adaptive phenotypic differentiation within and between two wild andean tomatoes</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">62</span>, <span class="cit-fpage">774</span>–<span class="cit-lpage">792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DNakazato%26rft.auinit1%253DT.%26rft.volume%253D62%26rft.issue%253D4%26rft.spage%253D774%26rft.epage%253D792%26rft.atitle%253DEnvironmental%2Bfactors%2Bpredict%2Badaptive%2Bphenotypic%2Bdifferentiation%2Bwithin%2Band%2Bbetween%2Btwo%2Bwild%2BAndean%2Btomatoes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00332.x%26rft_id%253Dinfo%253Apmid%252F18221381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00332.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18221381&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254640900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.51" data-doi="10.3732/ajb.0900216"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nakazato</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Warren</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Moyle</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Ecological and geographic modes of species divergence in wild tomatoes</span>. <abbr class="cit-jnl-abbrev">American Journal of Botany</abbr>, <span class="cit-vol">97</span>, <span class="cit-fpage">680</span>–<span class="cit-lpage">693</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.0900216%26rft_id%253Dinfo%253Apmid%252F21622430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk3LzQvNjgwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTAvNzQ0Nzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.52"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Oksanen</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanchet</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kindt</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Legendre</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Minchin</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’Hara</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Simpson</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Solymos</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stevens</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagnerr</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Oksanen</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Package ‘vegan’. Community ecology package, version 2.9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.53" data-doi="10.1146/annurev.phyto.41.121602.143032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pedley</span>,  <span class="cit-name-given-names">K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Molecular basis of Pto-mediated resistance to bacterial speck disease in tomato</span>. <abbr class="cit-jnl-abbrev">Annual Review of Phytopathology</abbr>, <span class="cit-vol">41</span>, <span class="cit-fpage">215</span>–<span class="cit-lpage">243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DPedley%26rft.auinit1%253DK.%2BF.%26rft.volume%253D41%26rft.spage%253D215%26rft.epage%253D243%26rft.atitle%253DMolecular%2Bbasis%2Bof%2BPto-mediated%2Bresistance%2Bto%2Bbacterial%2Bspeck%2Bdisease%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.41.121602.143032%26rft_id%253Dinfo%253Apmid%252F14527329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.41.121602.143032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14527329&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186493900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pitblado</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Kerr</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">1979</span>) <span class="cit-article-title">A source of resistance to bactereial speck–Pseudomonas tomato</span>. <abbr class="cit-jnl-abbrev">Tomato Genetics Cooperative</abbr>, <span class="cit-vol">29</span>, <span class="cit-fpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTomato%2BGenetics%2BCooperative%26rft.volume%253D29%26rft.spage%253D30%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.55" data-doi="10.1534/genetics.114.164350"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raj</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephen</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">fastSTRUCTURE: Variational Inference of Population Structure in Large SNP Data Sets</span>. <abbr class="cit-jnl-abbrev">Genetics2</abbr>, <span class="cit-vol">197</span>, <span class="cit-fpage">573</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics2%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164350%26rft_id%253Dinfo%253Apmid%252F24700103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNTczIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMTAvNzQ0Nzk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ramirez-Estrada</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castillo</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lara</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arro</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boronat</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrer</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Altabella</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Tomato UDP-Glucose Sterol Glycosyltransferases: A Family of Developmental and Stress Regulated Genes that Encode Cytosolic and Membrane-Associated Forms of the Enzyme</span>. <abbr class="cit-jnl-abbrev">Frontiers in Plant Science</abbr>, <span class="cit-vol">8</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.57"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rao</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kadirvel</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Symonds</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ebert</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Relationship between survival and yield related traits in Solanum pimpinellifolium under salt stress</span>. <abbr class="cit-jnl-abbrev">Euphytica</abbr>, <span class="cit-vol">190</span>, <span class="cit-fpage">215</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D190%26rft.spage%253D215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Razali</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bougouffa</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morton</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lightfoot</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alam</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Essack</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arold</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kamau</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmockel</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pailles</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shahid</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Michell</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salim</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ho</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tester</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bajic</span>,  <span class="cit-name-given-names">V.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Negrao</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">The Genome Sequence of the Wild Tomato Solanum pimpinellifolium Provides Insights Into Salinity Tolerance</span>. <abbr class="cit-jnl-abbrev">Frontiers in Plant Science</abbr>, <span class="cit-vol">9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.59" data-doi="10.1111/mec.13322"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rellstab</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gugerli</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckert</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hancock</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Holderegger</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">A practical guide to environmental association analysis in landscape genomics</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">4348</span>–<span class="cit-lpage">4370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D24%26rft.spage%253D4348%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.60" data-doi="10.1007/BF00984147"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rick</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fobes</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Holle</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">1977</span>) <span class="cit-article-title">Genetic variation in Lycopersicon pimpinellifolium: evidence of evolutionary change in mating systems</span>. <abbr class="cit-jnl-abbrev">Plant Systematics and Evolution</abbr>, <span class="cit-vol">127</span>, <span class="cit-fpage">139</span>–<span class="cit-lpage">170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSystematics%2Band%2BEvolution%26rft.volume%253D127%26rft.spage%253D139%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00984147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00984147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977DL82000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosseel</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">lavaan: An R Package for Structural Equation Modeling</span>. <abbr class="cit-jnl-abbrev">Statistical Software</abbr>, <span class="cit-vol">48</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistical%2BSoftware%26rft.volume%253D48%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.62" data-doi="10.1371/journal.pone.0022054"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salathe</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Schmid-Hempel</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">The genotypic structure of a multi-host bumblebee parasite suggests a role for ecological niche overlap</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr>, <span class="cit-vol">6</span>, <span class="cit-fpage">e22054</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSalathe%26rft.auinit1%253DR.%2BM.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De22054%26rft.epage%253De22054%26rft.atitle%253DThe%2Bgenotypic%2Bstructure%2Bof%2Ba%2Bmulti-host%2Bbumblebee%2Bparasite%2Bsuggests%2Ba%2Brole%2Bfor%2Becological%2Bniche%2Boverlap.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0022054%26rft_id%253Dinfo%253Apmid%252F21853023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0022054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21853023&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Senthil-Kumar</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Mysore</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">At-CYP710A1 gene-mediated stigmasterol production plays a role in imparting temperature stress tolerance in Arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Plant Signaling &amp; Behavior</abbr>, <span class="cit-vol">8</span>, <span class="cit-fpage">e23142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSignaling%2B%2526%2BBehavior%26rft.volume%253D8%26rft.spage%253De23142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.64" data-doi="10.1111/j.1365-294X.2010.04726.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sork</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Westfall</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Flint</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ikegami</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Grivet</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Gene movement and genetic association with regional climate gradients in California valley oak (Quercus lobata Nee) in the face of climate change</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">3806</span>–<span class="cit-lpage">3823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSork%26rft.auinit1%253DV.%2BL.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D3806%26rft.epage%253D3823%26rft.atitle%253DGene%2Bmovement%2Band%2Bgenetic%2Bassociation%2Bwith%2Bregional%2Bclimate%2Bgradients%2Bin%2BCalifornia%2Bvalley%2Boak%2B%2528Quercus%2Blobata%2BNee%2529%2Bin%2Bthe%2Bface%2Bof%2Bclimate%2Bchange.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2010.04726.x%26rft_id%253Dinfo%253Apmid%252F20723054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2010.04726.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20723054&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281285200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.65" data-doi="10.1105/tpc.018119"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">The genetic, developmental, and molecular bases of fruit size and shape variation in tomato</span>. <abbr class="cit-jnl-abbrev">The Plant cell</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">181</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bcell%26rft.volume%253D16%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.018119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1105/tpc.018119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.66" data-doi="10.1007/s001220050116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grandillo</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fulton</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zamir</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eshed</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Petiard</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lopez</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Beck-Bunn</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-article-title">Advanced backcross QTL analysis in a cross between an elite processing line of tomato and its wild relative L. pimpinellifolium</span>. <abbr class="cit-jnl-abbrev">Theoretical and Applied Genetics</abbr>, <span class="cit-vol">92</span>, <span class="cit-fpage">213</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Band%2BApplied%2BGenetics%26rft.volume%253D92%26rft.spage%253D213%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050116%26rft_id%253Dinfo%253Apmid%252F24166170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24166170&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TZ82500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vucetich</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Walte</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Spatial patterns of demography and genetic processes across the species’ range: Null hypotheses for landscape conservation genetics</span>. <abbr class="cit-jnl-abbrev">Conservation Genetics</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">639</span>–<span class="cit-lpage">645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BGenetics%26rft.volume%253D4%26rft.spage%253D639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.68" data-doi="10.1111/ele.12025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Glor</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Losos</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Quantifying the roles of ecology and geography in spatial genetic divergence</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">16</span>, <span class="cit-fpage">175</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D16%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12025%26rft_id%253Dinfo%253Apmid%252F23137142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23137142&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F10%2F744797.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.69" data-doi="10.2307/2389612"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wiens</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">1989</span>) <span class="cit-article-title">Spatial Scaling in Ecology</span>. <abbr class="cit-jnl-abbrev">Functional Ecology</abbr>, <span class="cit-vol">3</span>, <span class="cit-fpage">385</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunctional%2BEcology%26rft.volume%253D3%26rft.spage%253D385%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2389612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2389612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1989AP42100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.70" data-doi="10.1007/BF02294050"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">van den Wollenberg</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">1977</span>) <span class="cit-article-title">Redundancy analysis: an alternative for canonical correlation analysis</span>. <abbr class="cit-jnl-abbrev">Psychometrika</abbr>, <span class="cit-vol">42</span>, <span class="cit-fpage">207</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychometrika%26rft.volume%253D42%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02294050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02294050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1977DM59500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">1943</span>) <span class="cit-article-title">Isolation by distance</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">28</span>, <span class="cit-fpage">114</span>–<span class="cit-lpage">138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D114%26rft.epage%253D138%26rft.atitle%253DISOLATION%2BBY%2BDISTANCE%26rft_id%253Dinfo%253Apmid%252F17247074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIyOC8yLzExNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzEwLzc0NDc5Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.72"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoder</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stanton-Geddes</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">P.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Briskine</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Tiffin</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Genomic signature of adaptation to climate in Medicago truncatula</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">196</span>, <span class="cit-fpage">1263</span>–<span class="cit-lpage">1275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D196%26rft.spage%253D1263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.73"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jiao</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">X.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Vapour pressure deficit control in relation to water transport and water productivity in greenhouse tomato production during summer</span>. <abbr class="cit-jnl-abbrev">Scientific Reports</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">43461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D7%26rft.spage%253D43461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.74"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zomer</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trabucco</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bossio</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Straaten</span>,  <span class="cit-name-given-names">O.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Verchot</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Climate Change Mitigation: A Spatial Analysis of Global Land Suitability for Clean Development Mechanism Afforestation and Reforestation</span>. <abbr class="cit-jnl-abbrev">Agric. Ecosystems and Envir</abbr>, <span class="cit-vol">126</span>, <span class="cit-fpage">67</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgric.%2BEcosystems%2Band%2BEnvir%26rft.volume%253D126%26rft.spage%253D67%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-744797v2.75"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zuriaga</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blaanca</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cordero</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sifres</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blas-Cerdan</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Morales</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Nuez</span>,  <span class="cit-name-given-names">F.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Genetic and bioclimatic variation in Solanum pimpinellifolium</span>. <abbr class="cit-jnl-abbrev">Genetic Resources and Crop Evolution</abbr>, <span class="cit-vol">56</span>, <span class="cit-fpage">39</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2BResources%2Band%2BCrop%2BEvolution%26rft.volume%253D56%26rft.spage%253D39%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/744797v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F744797v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/721589v4" title="Heterogeneity in human hippocampal CaMKII transcripts reveals allosteric hub-dependent regulation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 10, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/744797v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/744797v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/01/10/744797.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/744797v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/744797v2" class="active" data-icon-position="" data-hide-link-title="0">Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1099442" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1sMC5UfmwynxcBI0OUkBM90hw_s3MhK5W7altY2jNzM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1099442" id="node1099442--2" data-pisa="biorxiv;744797v2" data-pisa-master="biorxiv;744797" data-seqnum="1099442" data-apath="/biorxiv/early/2020/01/10/744797.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Matthew JS</span> <span class="nlm-surname">Gibson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Leonie C</span> <span class="nlm-surname">Moyle</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">744797; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/744797 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/744797v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/744797v2&amp;title=Regional%20differences%20in%20the%20abiotic%20environment%20contribute%20to%20genomic%20divergence%20within%20a%20wild%20tomato%20species" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/744797v2&amp;title=Regional%20differences%20in%20the%20abiotic%20environment%20contribute%20to%20genomic%20divergence%20within%20a%20wild%20tomato%20species" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/744797v2&amp;text=Regional%20differences%20in%20the%20abiotic%20environment%20contribute%20to%20genomic%20divergence%20within%20a%20wild%20tomato%20species" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/744797v2&amp;title=Regional%20differences%20in%20the%20abiotic%20environment%20contribute%20to%20genomic%20divergence%20within%20a%20wild%20tomato%20species" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/744797v2&amp;t=Regional%20differences%20in%20the%20abiotic%20environment%20contribute%20to%20genomic%20divergence%20within%20a%20wild%20tomato%20species" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/744797v2&amp;title=Regional%20differences%20in%20the%20abiotic%20environment%20contribute%20to%20genomic%20divergence%20within%20a%20wild%20tomato%20species" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/744797v2&amp;title=Regional%20differences%20in%20the%20abiotic%20environment%20contribute%20to%20genomic%20divergence%20within%20a%20wild%20tomato%20species" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1099442" id="node1099442--3" data-pisa="biorxiv;744797v2" data-pisa-master="biorxiv;744797" data-seqnum="1099442" data-apath="/biorxiv/early/2020/01/10/744797.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Matthew JS</span> <span class="nlm-surname">Gibson</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Leonie C</span> <span class="nlm-surname">Moyle</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">744797; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/744797 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Regional%20differences%20in%20the%20abiotic%20environment%20contribute%20to%20genomic%20divergence%20within%20a%20wild%20tomato%20species&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Gibson%2C+Matthew+JS&amp;rft.au=Moyle%2C+Leonie+C"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1099442/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1099442/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1099442/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1099442/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1099442/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1099442/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1099442/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1099442/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1099442/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1099442/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1099442/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1099442/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/744797v2&amp;count=horizontal&amp;via=&amp;text=Regional%20differences%20in%20the%20abiotic%20environment%20contribute%20to%20genomic%20divergence%20within%20a%20wild%20tomato%20species&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/744797v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/744797v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/744797v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
