<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2020/01/09/851766.atom" />
<meta name="HW.pisa" content="biorxiv;851766v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range" />
<meta name="DC.Identifier" content="10.1101/851766" />
<meta name="DC.Date" content="2020-01-09" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Determining how different populations adapt to similar environments is fundamental to understanding the limits of adaptation under changing environments. Snowshoe hares ( Lepus americanus ) typically molt into white winter coats to remain camouflaged against snow. In some warmer climates, hares have evolved brown winter camouflage – an adaptation that may spread under climate change. We used extensive range-wide genomic data to 1) resolve broad-scale patterns of population structure and gene flow and 2) investigate the factors shaping the origins and distribution of winter-brown camouflage variation. In coastal Pacific Northwest (PNW) populations, winter-brown camouflage is known to be determined by a recessive haplotype at the Agouti pigmentation gene. Our phylogeographic analyses revealed deep structure and limited gene flow between PNW and more northern Boreal populations, where winter-brown camouflage is rare along the range edge. Genome sequencing of a winter-brown snowshoe hare from Alaska shows that it lacks the winter-brown PNW haplotype, reflecting a history of convergent phenotypic evolution. However, the PNW haplotype does occur at low frequency in a winter-white population from Montana, consistent with the spread of a locally deleterious recessive variant that is masked from selection when rare. Simulations show that if annual snow cover dramatically declined in the same population, then the predicted selective increase in frequency of the now beneficial winter-brown Agouti allele is likely to be extremely slow due to the same masking effect. Our findings underscore how allelic dominance can shape the geographic extent and rate of convergent adaptation in response to rapidly changing environments." />
<meta name="DC.Contributor" content="Matthew R. Jones" />
<meta name="DC.Contributor" content="L. Scott Mills" />
<meta name="DC.Contributor" content="Jeffrey D. Jensen" />
<meta name="DC.Contributor" content="Jeffrey M. Good" />
<meta name="article:published_time" content="2020-01-09" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Determining how different populations adapt to similar environments is fundamental to understanding the limits of adaptation under changing environments. Snowshoe hares (&lt;i&gt;Lepus americanus&lt;/i&gt;) typically molt into white winter coats to remain camouflaged against snow. In some warmer climates, hares have evolved brown winter camouflage – an adaptation that may spread under climate change. We used extensive range-wide genomic data to 1) resolve broad-scale patterns of population structure and gene flow and 2) investigate the factors shaping the origins and distribution of winter-brown camouflage variation. In coastal Pacific Northwest (PNW) populations, winter-brown camouflage is known to be determined by a recessive haplotype at the &lt;i&gt;Agouti&lt;/i&gt; pigmentation gene. Our phylogeographic analyses revealed deep structure and limited gene flow between PNW and more northern Boreal populations, where winter-brown camouflage is rare along the range edge. Genome sequencing of a winter-brown snowshoe hare from Alaska shows that it lacks the winter-brown PNW haplotype, reflecting a history of convergent phenotypic evolution. However, the PNW haplotype does occur at low frequency in a winter-white population from Montana, consistent with the spread of a locally deleterious recessive variant that is masked from selection when rare. Simulations show that if annual snow cover dramatically declined in the same population, then the predicted selective increase in frequency of the now beneficial winter-brown &lt;i&gt;Agouti&lt;/i&gt; allele is likely to be extremely slow due to the same masking effect. Our findings underscore how allelic dominance can shape the geographic extent and rate of convergent adaptation in response to rapidly changing environments.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2020/01/01" />
<meta name="citation_mjid" content="biorxiv;851766v2" />
<meta name="citation_id" content="851766v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/851766v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/851766v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/851766v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766.full.pdf" />
<meta name="citation_doi" content="10.1101/851766" />
<meta name="citation_num_pages" content="34" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="851766" />
<meta name="citation_author" content="Matthew R. Jones" />
<meta name="citation_author_institution" content="Division of Biological Sciences, University of Montana" />
<meta name="citation_author_institution" content="School of Life Sciences, Arizona State University" />
<meta name="citation_author_email" content="jeffrey.good@umontana.edu" />
<meta name="citation_author_email" content="matthew.r.jones.1@asu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4822-157X" />
<meta name="citation_author" content="L. Scott Mills" />
<meta name="citation_author_institution" content="Wildlife Biology Program, University of Montana" />
<meta name="citation_author_institution" content="Office of Research and Creative Scholarship, University of Montana" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8771-509X" />
<meta name="citation_author" content="Jeffrey D. Jensen" />
<meta name="citation_author_institution" content="School of Life Sciences, Arizona State University" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4786-8064" />
<meta name="citation_author" content="Jeffrey M. Good" />
<meta name="citation_author_institution" content="Division of Biological Sciences, University of Montana" />
<meta name="citation_author_institution" content="Wildlife Biology Program, University of Montana" />
<meta name="citation_author_email" content="jeffrey.good@umontana.edu" />
<meta name="citation_author_email" content="matthew.r.jones.1@asu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0707-5374" />
<meta name="citation_reference" content="Wright S. 1931 Evolution in Mendelian populations. Genetics 16, 97–159." />
<meta name="citation_reference" content="Ralph PL, Coop G. 2015 Convergent evolution during local adaptation to patchy landscapes. PLOS Genet. 11, e1005630. (doi:10.1371/journal.pgen.1005630)" />
<meta name="citation_reference" content="Haldane JBS. 1932 The causes of evolution. London: Longmans, Green and Co." />
<meta name="citation_reference" content="Bridle JR, Vines TH. 2007 Limits to evolution at range margins: when and why does adaptation fail? Trends Ecol. Evol. 22, 140–147. (doi:10.1016/j.tree.2006.11.002)" />
<meta name="citation_reference" content="Hoffmann AA, Sgrò CM. 2011 Climate change and evolutionary adaptation. Nature 470, 479–485. (doi:10.1038/nature09670)" />
<meta name="citation_reference" content="Fisher RA. 1937 The wave of advance of advantageous genes. Ann. Eugen. 7, 355–369. (doi:10.1111/j.1469-1809.1937.tb02153.x)" />
<meta name="citation_reference" content="Bay RA, Rose N, Barrett R, Bernatchez L, Ghalambor CK, Lasky JR, Brem RB, Palumbi SR, Ralph P. 2017 Predicting responses to contemporary environmental change using evolutionary response architectures. Am. Nat. 189, 463–473. (doi:10.1086/691233)" />
<meta name="citation_reference" content="Hoekstra HE, Nachman MW. 2003 Different genes underlie adaptive melanism in different populations of rock pocket mice. Mol. Ecol. 12, 1185–94. (doi:10.1046/j.1365-294X.2003.01788.x)" />
<meta name="citation_reference" content="Rosenblum EB, Römpler H, Schöneberg T, Hoekstra HE. 2010 Molecular and functional basis of phenotypic convergence in white lizards at White Sands. Proc. Natl. Acad. Sci. U. S. A. 107, 2113–7. (doi:10.1073/pnas.0911042107)" />
<meta name="citation_reference" content="Dobler S, Dalla S, Wagschal V, Agrawal AA. 2012 Community-wide convergent evolution in insect adaptation to toxic cardenolides by substitutions in the Na,K-ATPase. Proc. Natl. Acad. Sci. U. S. A. 109, 13040–5. (doi:10.1073/pnas.1202111109)" />
<meta name="citation_reference" content="Steiner CC, Rompler H, Boettger LM, Schoneberg T, Hoekstra HE. 2008 The genetic basis of phenotypic convergence in beach mice: similar pigment patterns but different genes. Mol. Biol. Evol. 26, 35–45. (doi:10.1093/molbev/msn218)" />
<meta name="citation_reference" content="Marques DA, Taylor JS, Jones FC, Di Palma F, Kingsley DM, Reimchen TE. 2017 Convergent evolution of SWS2 opsin facilitates adaptive radiation of threespine stickleback into different light environments. PLOS Biol. 15, e2001627. (doi:10.1371/journal.pbio.2001627)" />
<meta name="citation_reference" content="Hoekstra HE, Krenz JG, Nachman MW. 2005 Local adaptation in the rock pocket mouse (Chaetodipus intermedius): Natural selection and phylogenetic history of populations. Heredity (Edinb). 94, 217–228. (doi:10.1038/sj.hdy.6800600)" />
<meta name="citation_reference" content="Kreiner JM et al. 2019 Multiple modes of convergent adaptation in the spread of glyphosate-resistant Amaranthus tuberculatus. Proc. Natl. Acad. Sci., 201900870. (doi:10.1073/pnas.1900870116)" />
<meta name="citation_reference" content="Ralph PL, Coop G. 2010 Parallel adaptation: One or many waves of advance of an advantageous allele? Genetics 186, 647–668. (doi:10.1534/genetics.110.119594)" />
<meta name="citation_reference" content="Slatkin M. 1973 Gene flow and selection in a cline. Genetics 75, 733–56." />
<meta name="citation_reference" content="Nachman MW, Hoekstra HE, D’Agostino SL. 2003 The genetic basis of adaptive melanism in pocket mice. Proc. Natl. Acad. Sci. U. S. A. 100, 5268–73. (doi:10.1073/pnas.0431157100)" />
<meta name="citation_reference" content="Harris RB et al. 2019 The population genetics of crypsis in vertebrates: recent insights from mice, hares, and lizards. Heredity. 124, 1–14. (doi:10.1038/s41437-019-0257-4)" />
<meta name="citation_reference" content="Hoekstra HE, Drumm KE, Nachman MW. 2004 Ecological genetics of adaptive color polymorphism in pocket mice: Geographic variation in selected and neutral genes. Evolution. 58, 1329–1341. (doi:10.1111/j.0014-3820.2004.tb01711.x)" />
<meta name="citation_reference" content="Pfeifer SP, Laurent S, Sousa VC, Linnen CR, Foll M, Excoffier L, Hoekstra HE, Jensen JD. 2018 The evolutionary history of Nebraska deer mice: local adaptation in the face of strong gene flow. Mol. Biol. Evol. 35, 792–806. (doi:10.1093/molbev/msy004)" />
<meta name="citation_reference" content="Barrett RDH et al. 2019 Linking a mutation to survival in wild mice. Science. 363, 499–504. (doi:10.1126/science.aav3824)" />
<meta name="citation_reference" content="Turner JRG. 1981 Adaptation and evolution in Heliconius: a defense of NeoDarwinism. Annu. Rev. Ecol. Syst. 12, 99–121. (doi:10.1146/annurev.es.12.110181.000531)" />
<meta name="citation_reference" content="Haldane JBS. 1924 A mathematical theory of natural and artificial selection (Part I). Trans. Cambridge Philos. Soc. 23, 19–41." />
<meta name="citation_reference" content="Teshima KM, Przeworski M. 2006 Directional positive selection on an allele of arbitrary dominance. Genetics 172, 713–8. (doi:10.1534/genetics.105.044065)" />
<meta name="citation_reference" content="Orr HA, Betancourt AJ. 2001 Haldane’s sieve and adaptation from the standing genetic variation. Genetics 157, 875–884." />
<meta name="citation_reference" content="Mills LS, Zimova M, Oyler J, Running S, Abatzoglou JT, Lukacs PM. 2013 Camouflage mismatch in seasonal coat color due to decreased snow duration. Proc. Natl. Acad. Sci. U. S. A. 110, 7360–7365. (doi:10.1073/pnas.1222724110)" />
<meta name="citation_reference" content="Zimova M, Hackländer K, Good JM, Melo-Ferreira J, Alves PC, Mills LS. 2018 Function and underlying mechanisms of seasonal colour moulting in mammals and birds: what keeps them changing in a warming world? Biol. Rev. 93, 1478–1498. (doi:10.1111/brv.12405)" />
<meta name="citation_reference" content="Mills LS et al. 2018 Winter color polymorphisms identify global hot spots for evolutionary rescue from climate change. Science. 359, 1033–1036. (doi:10.1126/science.aan8097)" />
<meta name="citation_reference" content="Jones MR et al. 2018 Adaptive introgression underlies polymorphic seasonal camouflage in snowshoe hares. Science. 360, 1355–1358. (doi:10.1126/science.aar5273)" />
<meta name="citation_reference" content="Zimova M, Mills LS, Nowak JJ. 2016 High fitness costs of climate change-induced camouflage mismatch. Ecol. Lett. 19, 299–307. (doi:10.1111/ele.12568)" />
<meta name="citation_reference" content="Brown RD, Mote PW. 2009 The response of northern hemisphere snow cover to a changing climate. J. Clim. 22, 2124–2145. (doi:10.1175/2008JCLI2665.1)" />
<meta name="citation_reference" content="Knowles N, Dettinger MD, Cayan DR, Knowles N, Dettinger MD, Cayan DR. 2006 Trends in snowfall versus rainfall in the western United States. J. Clim. 19, 4545–4559. (doi:10.1175/JCLI3850.1)" />
<meta name="citation_reference" content="Nagorsen DW. 1983 Winter pelage colour in snowshoe hares (Lepus americanus) from the Pacific Northwest. Can. J. Zool. 61, 2313–2318. (doi:10.1139/z83-305)" />
<meta name="citation_reference" content="Jones MR, Mills LS, Jensen JD, Good JM. 2019 The origin and spread of locally adaptive seasonal camouflage in snowshoe hares. bioRxiv (doi:10.1101/847616)" />
<meta name="citation_reference" content="Gigliotti LC, Diefenbach DR, Sheriff MJ. 2017 Geographic variation in winter adaptations of snowshoe hares (Lepus americanus). Can. J. Zool. 95, 539–545. (doi:10.1139/cjz-2016-0165)" />
<meta name="citation_reference" content="Cheng E, Hodges KE, Melo-Ferreira J, Alves PC, Mills LS. 2014 Conservation implications of the evolutionary history and genetic diversity hotspots of the snowshoe hare. Mol. Ecol. 23, 2929–2942. (doi:10.1111/mec.12790)" />
<meta name="citation_reference" content="Melo-Ferreira J, Seixas FA, Cheng E, Mills LS, Alves PC. 2014 The hidden history of the snowshoe hare, Lepus americanus: extensive mitochondrial DNA introgression inferred from multilocus genetic variation. Mol. Ecol. 23, 4617–4630. (doi:10.1111/mec.12886)" />
<meta name="citation_reference" content="Rohland N, Reich D. 2012 Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture. Genome Res. 22, 939–946. (doi:10.1101/gr.128124.111)" />
<meta name="citation_reference" content="Bolger AM, Lohse M, Usadel B. 2014 Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120. (doi:10.1093/bioinformatics/btu170)" />
<meta name="citation_reference" content="Magoc T, Salzberg SL. 2011 FLASH: fast length adjustment of short reads to improve genome assemblies. Bioinformatics 27, 2957–2963. (doi:10.1093/bioinformatics/btr507)" />
<meta name="citation_reference" content="McKenna A et al. 2010 The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297–1303. (doi:10.1101/gr.107524.110)" />
<meta name="citation_reference" content="Danecek P et al. 2011 The variant call format and VCFtools. Bioinformatics 27, 2156–2158. (doi:10.1093/bioinformatics/btr330)" />
<meta name="citation_reference" content="Stamatakis A. 2014 RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312–1313. (doi:10.1093/bioinformatics/btu033)" />
<meta name="citation_reference" content="Bouckaert R, Heled J, Kühnert D, Vaughan T, Wu C-H, Xie D, Suchard MA, Rambaut A, Drummond AJ. 2014 BEAST 2: a software platform for bayesian evolutionary analysis. PLoS Comput. Biol. 10, e1003537. (doi:10.1371/journal.pcbi.1003537)" />
<meta name="citation_reference" content="Matthee CA, Van Vuuren BJ, Bell D, Robinson TJ. 2004 A molecular supermatrix of the rabbits and hares (Leporidae) allows for the identification of five intercontinental exchanges during the Miocene. Syst. Biol. 53, 433–447. (doi:10.1080/10635150490445715)" />
<meta name="citation_reference" content="Zhang C, Rabiee M, Sayyari E, Mirarab S. 2018 ASTRAL-III: Polynomial time species tree reconstruction from partially resolved gene trees. BMC Bioinformatics 19. (doi:10.1186/s12859-018-2129-y)" />
<meta name="citation_reference" content="Alexander DH, Novembre J, Lange K. 2009 Fast model-based estimation of ancestry in unrelated individuals. Genome Res. 19, 1655–1664. (doi:10.1101/gr.094052.109)" />
<meta name="citation_reference" content="Petkova D, Novembre J, Stephens M. 2016 Visualizing spatial population structure with estimated effective migration surfaces. Nat. Genet. 48, 94–100. (doi:10.1038/ng.3464)" />
<meta name="citation_reference" content="Kofler R, Orozco-terWengel P, De Maio N, Pandey RV, Nolte V, Futschik A, Kosiol C, Schlötterer C. 2011 PoPoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals. PLoS One 6, e15925. (doi:10.1371/journal.pone.0015925)" />
<meta name="citation_reference" content="Schlager G, Dickie MM. 1971 Natural mutation rates in the house mouse. Estimates for five specific loci and dominant mutations. Mutat. Res. 11, 89–96. (doi:10.1016/0027-5107(71)90034-0)" />
<meta name="citation_reference" content="Milholland B, Dong X, Zhang L, Hao X, Suh Y, Vijg J. 2017 Differences between germline and somatic mutation rates in humans and mice. Nat. Commun. 8, 15183. (doi:10.1038/ncomms15183)" />
<meta name="citation_reference" content="Carneiro M, Albert FW, Melo-Ferreira J, Galtier N, Gayral P, Blanco-Aguiar JA, Villafuerte R, Nachman MW, Ferrand N. 2012 Evidence for widespread positive and purifying selection across the European rabbit (Oryctolagus cuniculus) genome. Mol. Biol. Evol. 29, 1837–1849. (doi:10.1093/molbev/mss025)" />
<meta name="citation_reference" content="Gillis EA, Krebs CJ. 1999 Natal dispersal of snowshoe hares during a cyclic population increase. J. Mammal. 80, 933–939. (doi:10.2307/1383263)" />
<meta name="citation_reference" content="Haller BC, Messer PW. 2019 SLiM 3: Forward genetic simulations beyond the Wright-Fisher model. Mol. Biol. Evol. 36, 632–637. (doi:10.1101/418657)" />
<meta name="citation_reference" content="Kimura M. 1957 Some Problems of Stochastic Processes in Genetics. Ann. Math. Stat. 28, 882–901. (doi:10.1214/aoms/1177706791)" />
<meta name="citation_reference" content="R Core Team. 2018 R: A language and environment for statistical computing." />
<meta name="citation_reference" content="Nelson TC, Jones MR, Velotta JP, Dhawanjewar AS, Schweizer RM. 2019 UNVEILing connections between genotype, phenotype, and fitness in natural populations. Mol. Ecol. 28, 1866–1876. (doi:10.1111/mec.15067)" />
<meta name="citation_reference" content="Giska I et al. 2019 Introgression drives repeated evolution of winter coat color polymorphism in hares. Proc. Natl. Acad. Sci. U. S. A. (doi:10.1073/pnas.1910471116)" />
<meta name="citation_reference" content="Zheng Y, Peng R, Kuro-o M, Zeng X. 2011 Exploring patterns and extent of bias in estimating divergence time from mitochondrial DNA sequence data in a particular lineage: a case study of salamanders (Order Caudata). Mol. Biol. Evol. 28, 2521–2535. (doi:10.1093/molbev/msr072)" />
<meta name="citation_reference" content="Manceau M, Domingues VS, Mallarino R, Hoekstra HE. 2011 The developmental role of Agouti in color pattern evolution. Science. 331, 1062–1065. (doi:10.1126/science.1200684)" />
<meta name="citation_reference" content="Laurent S, Pfeifer SP, Settles ML, Hunter SS, Hardwick KM, Ormond L, Sousa VC, Jensen JD, Rosenblum EB. 2016 The population genomics of rapid adaptation: disentangling signatures of selection and demography in white sands lizards. Mol. Ecol. 25, 306–323. (doi:10.1111/mec.13385)" />
<meta name="citation_reference" content="Protas ME, Hersey C, Kochanek D, Zhou Y, Wilkens H, Jeffery WR, Zon LI, Borowsky R, Tabin CJ. 2006 Genetic analysis of cavefish reveals molecular convergence in the evolution of albinism. Nat. Genet. 38, 107–111. (doi:10.1038/ng1700)" />
<meta name="citation_reference" content="Kingsley EP, Manceau M, Wiley CD, Hoekstra HE. 2009 Melanism in Peromyscus is caused by independent mutations in Agouti. PLoS One 4, e6435. (doi:10.1371/journal.pone.0006435)" />
<meta name="citation_reference" content="Nachman MW, Hoekstra HE, D’Agostino SL. 2003 The genetic basis of adaptive melanism in pocket mice. Proc. Natl. Acad. Sci. U. S. A. 100, 5268–73. (doi:10.1073/pnas.0431157100)" />
<meta name="twitter:title" content="Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Determining how different populations adapt to similar environments is fundamental to understanding the limits of adaptation under changing environments. Snowshoe hares ( Lepus americanus ) typically molt into white winter coats to remain camouflaged against snow. In some warmer climates, hares have evolved brown winter camouflage – an adaptation that may spread under climate change. We used extensive range-wide genomic data to 1) resolve broad-scale patterns of population structure and gene flow and 2) investigate the factors shaping the origins and distribution of winter-brown camouflage variation. In coastal Pacific Northwest (PNW) populations, winter-brown camouflage is known to be determined by a recessive haplotype at the Agouti pigmentation gene. Our phylogeographic analyses revealed deep structure and limited gene flow between PNW and more northern Boreal populations, where winter-brown camouflage is rare along the range edge. Genome sequencing of a winter-brown snowshoe hare from Alaska shows that it lacks the winter-brown PNW haplotype, reflecting a history of convergent phenotypic evolution. However, the PNW haplotype does occur at low frequency in a winter-white population from Montana, consistent with the spread of a locally deleterious recessive variant that is masked from selection when rare. Simulations show that if annual snow cover dramatically declined in the same population, then the predicted selective increase in frequency of the now beneficial winter-brown Agouti allele is likely to be extremely slow due to the same masking effect. Our findings underscore how allelic dominance can shape the geographic extent and rate of convergent adaptation in response to rapidly changing environments." />
<meta name="og-title" property="og:title" content="Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/851766v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Determining how different populations adapt to similar environments is fundamental to understanding the limits of adaptation under changing environments. Snowshoe hares ( Lepus americanus ) typically molt into white winter coats to remain camouflaged against snow. In some warmer climates, hares have evolved brown winter camouflage – an adaptation that may spread under climate change. We used extensive range-wide genomic data to 1) resolve broad-scale patterns of population structure and gene flow and 2) investigate the factors shaping the origins and distribution of winter-brown camouflage variation. In coastal Pacific Northwest (PNW) populations, winter-brown camouflage is known to be determined by a recessive haplotype at the Agouti pigmentation gene. Our phylogeographic analyses revealed deep structure and limited gene flow between PNW and more northern Boreal populations, where winter-brown camouflage is rare along the range edge. Genome sequencing of a winter-brown snowshoe hare from Alaska shows that it lacks the winter-brown PNW haplotype, reflecting a history of convergent phenotypic evolution. However, the PNW haplotype does occur at low frequency in a winter-white population from Montana, consistent with the spread of a locally deleterious recessive variant that is masked from selection when rare. Simulations show that if annual snow cover dramatically declined in the same population, then the predicted selective increase in frequency of the now beneficial winter-brown Agouti allele is likely to be extremely slow due to the same masking effect. Our findings underscore how allelic dominance can shape the geographic extent and rate of convergent adaptation in response to rapidly changing environments." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2020-01-09" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/851766v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/851766v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/851766v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/851766v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/1097947" />
    <title>Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__AQm9HArb2Y_D7UzgraiLIwAB7088fIq2iZD0Uigie_s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"KU-7ZJDYyTn8vLwJeRMHtDseT75QmqLVTW3SBbBKu6k","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"1097947","apath":"\/biorxiv\/early\/2020\/01\/09\/851766.atom","pisa":"biorxiv;851766v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;851766v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/851766v2","title":"Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range","identifier":"node\/1097947"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iWaY\\\\\\\/AwiGj6hnXXyooN+BJdqtjsLqhsjRBR+4+MTv2\\\\\\\/ND6qXW26yJkWZdlbdzqVj0bpshwdnVEgdlQRN52Y+X7FwZWAa7EwlkvNy1zBmgkJT4VNAbdbBFDvvBFeKtr\\\\\\\/ugk5F23NNuSHp5z1mJPuwyoSJXXIW3sY\\\\\\\/t3rQRlQOWv1840vD+lKSFI6\\\\\\\/EBe9znkDtFkBImYeOHO50EWtEIzYw5jJ6Xo0Uj7bpv0GbgHRWRaEmb2NwMTVUTvgu2CzTAzXdd2VMQAVarJeakXijZJ0S05aTaho4Bq4al4BuPClKOamLmBhZTROj1tJdCYr6Z4SJRs8u5d9wu2z6uJr3OnrfKmSffm+2xWbyA8UJZFE0ILNPUE2TQYOecIbQaqiHM3ruNPuXbqqnCXh7SyLdmuH97NeSY2q+6T4spz7IJ2ja91pDO0nwAzF6SJJ+ypraoC0OxnyxnzNV8glL6vHMQ\\\\\\\/iEXnnzVbxerdYcqdaUkhNMJ5h8lQMkx7IRwVSIHRWhuuqezA7EsamRFCvu6KQ\\\\\\\/h1QsBDCN+AWGqgNK2XS78Ku1X27MsC1mGiui4kw4WXJMangDgGuRhYOw6q2xLQsKq1UcDG9Csp+YI8aXRxkLl8rrZlfdG5ge85qrofHdKMLV1mo6btqND8ybGCJiH0aabUwKubUiyKFYJvVIrUP1BTYqo2XXm9gpzhGz1IaFSGdZYTDPdDS646v3p52ZMdy88K4CvRgQcLuNdXqdjXsq3hMKjN0yCwY5uejRYtafgKi2ZKseX7dx52giT65NXq86+kykuXzrRQsD5Zun9MP0kHXM\\\\\\\/tH4kZPNyU+5cDzc53OufeeTRp0Qs9O5WEYRN5mylE4ide3aSVTAc4mIJUpJg0lHvpmtPHdRHqrfrjbHQkJK4peoAxURLUT3dvFon3XR9vMhhaE94YnpA0I1fW5TAsV+JL3wpMQp3qbiMthjSdFBhac3SUhgb0itJp1WrbqB2YVrrjtxuFKsr9k5rq+dursOPT6qiTp8Wyy6nexy9iqnp\\\\\\\/yy4x5KmNeMwoYnl9CB6PoMGAmqd5ix3XqjQmqdI3InMkq2Z8CrDcH7\\\\\\\/n4aadoU76+S8NJn+uWoFXsbJV6IpIXKd34yhkgBlrnfzRhx4seg08p39wc9Nr6xOIulDbHWERXJbUjgBSGnrSTVCyOXSBptNmilM616y++giSZODKPg\\\\\\\/O0m34AAop7jAint\\\\\\\/7bXq8P7OJ784Z60tbPyGoMf6cK\\\\\\\/NBWUJ69bzj7cFeRa8tSReyY9BKyZFrsVU6SqU7FzYliK5YBQxpI1VTwoVPfjPRruKM1Hbz0GYcNh0eSnQD\\\\\\\/ugHk7wSmIZqWHjbTgiN\\\\\\\/Od41\\\u0022,\\\u0022iv\\\u0022:\\\u002239Wc4cndloWuQQWCP1HblA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00228906712d1f8de2215f5f6a17956708409272d76ece2ca0ff0a824ae1372167a3\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/851766v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-1097947 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/851766v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vBdA65aeSIFNN6q11_7BP_nYG_8UKj7YnvA_G5Z_G7E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1097947" id="node1097947" data-pisa="biorxiv;851766v2" data-pisa-master="biorxiv;851766" data-apath="/biorxiv/early/2020/01/09/851766.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4822-157X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Matthew R.</span> <span class="nlm-surname">Jones</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-8771-509X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">L. Scott</span> <span class="nlm-surname">Mills</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4786-8064" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jeffrey D.</span> <span class="nlm-surname">Jensen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0707-5374" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Jeffrey M.</span> <span class="nlm-surname">Good</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/851766 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node1097947" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew R. Jones </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Division of Biological Sciences, University of Montana</span>, Missoula, MT 59812, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>School of Life Sciences, Arizona State University</span>, Tempe, AZ 85281, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BR.%2BJones%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jones%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BR.%2BJones%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4822-157X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew R. Jones</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jeffrey.good@umontana.edu" class="" data-icon-position="" data-hide-link-title="0">jeffrey.good@umontana.edu</a>
<a href="mailto:matthew.r.jones.1@asu.edu" class="" data-icon-position="" data-hide-link-title="0">matthew.r.jones.1@asu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">L. Scott Mills </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Wildlife Biology Program, University of Montana</span>, Missoula, MT 59812, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>Office of Research and Creative Scholarship, University of Montana</span>, Missoula, MT 59812, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BScott%2BMills%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mills%20LS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AL.%2BScott%2BMills%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8771-509X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for L. Scott Mills</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jeffrey D. Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>School of Life Sciences, Arizona State University</span>, Tempe, AZ 85281, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BD.%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BD.%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4786-8064" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey D. Jensen</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jeffrey M. Good </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Division of Biological Sciences, University of Montana</span>, Missoula, MT 59812, <span class='nlm-country'>USA</span></div><div class='author-affiliation'><span class='nlm-institution'>Wildlife Biology Program, University of Montana</span>, Missoula, MT 59812, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BM.%2BGood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Good%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BM.%2BGood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0707-5374" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey M. Good</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jeffrey.good@umontana.edu" class="" data-icon-position="" data-hide-link-title="0">jeffrey.good@umontana.edu</a>
<a href="mailto:matthew.r.jones.1@asu.edu" class="" data-icon-position="" data-hide-link-title="0">matthew.r.jones.1@asu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/851766v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:1097947" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:1097947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/851766v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:1097947" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:1097947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/851766v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:1097947" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:1097947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/851766v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:1097947" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:1097947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/851766v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:1097947" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:1097947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/851766v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:1097947" data-trigger="external-links" data-url-enabled="1">Data/Code</a><a href="/panels_ajax_tab/article_tab_data_code/node:1097947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/851766v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:1097947" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:1097947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Determining how different populations adapt to similar environments is fundamental to understanding the limits of adaptation under changing environments. Snowshoe hares (<em>Lepus americanus</em>) typically molt into white winter coats to remain camouflaged against snow. In some warmer climates, hares have evolved brown winter camouflage – an adaptation that may spread under climate change. We used extensive range-wide genomic data to 1) resolve broad-scale patterns of population structure and gene flow and 2) investigate the factors shaping the origins and distribution of winter-brown camouflage variation. In coastal Pacific Northwest (PNW) populations, winter-brown camouflage is known to be determined by a recessive haplotype at the <em>Agouti</em> pigmentation gene. Our phylogeographic analyses revealed deep structure and limited gene flow between PNW and more northern Boreal populations, where winter-brown camouflage is rare along the range edge. Genome sequencing of a winter-brown snowshoe hare from Alaska shows that it lacks the winter-brown PNW haplotype, reflecting a history of convergent phenotypic evolution. However, the PNW haplotype does occur at low frequency in a winter-white population from Montana, consistent with the spread of a locally deleterious recessive variant that is masked from selection when rare. Simulations show that if annual snow cover dramatically declined in the same population, then the predicted selective increase in frequency of the now beneficial winter-brown <em>Agouti</em> allele is likely to be extremely slow due to the same masking effect. Our findings underscore how allelic dominance can shape the geographic extent and rate of convergent adaptation in response to rapidly changing environments.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">In response to a shared selection pressure, populations may adapt through the migration of beneficial alleles or through independent mutations that result in the evolution of convergent phenotypes [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. Distinguishing between these scenarios is crucial to understand the capacity of populations to adapt rapidly to environmental change [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. If gene flow between populations is sufficiently high, then beneficial variation may spread quickly [<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>] and potentially allow for rapid adaptive responses to changing environments [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. However, the spread of adaptive variation may be limited at broad geographic scales, and populations may have to rely on independent standing genetic variation or new mutations to evolve convergent traits. While there is considerable evidence for both adaptation through gene flow and independent mutations in nature [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>], few empirical studies have examined the specific factors that influence these outcomes in natural populations [<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>].</p><p id="p-5">When a species range encompasses a mosaic of habitats, the relative probability of adaptation through gene flow versus independent mutation is predicted to be primarily a function of the distance between habitat patches (in units of dispersal distance) and the strength of selection against locally adaptive alleles in intervening habitats [<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">2</a>,<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. In general, as distance between patches and the strength of purifying selection in intervening habitats increases, so does the relative probability of adaptation through independent mutations. Adaptation via independent mutations is therefore predicted to be more common in widespread populations where dispersal distance is short relative to the total range size [<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>]. For example, rock pocket mice (<em>Chaetodipus intermedius</em>) have repeatedly evolved melanistic coats across patchy lava flows in the southwestern United States. Although substantial gene flow between adjacent lava flows has likely resulted in the migration of melanic alleles [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>], melanism is attributed to different mutations across disparate lava flows [<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>,<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>,<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. Theoretical models further predict that the relative probability of adaptation via independent mutations increases rapidly with distance between lava flows (over the scale of tens to hundreds of kilometers) [<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">2</a>], due in large part to strong selection against coat color mismatch [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>–<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. Thus, there is a clear tradeoff between dispersal distance and the strength of purifying selection that strongly dictates the probability of adaptation through convergent evolution or gene flow.</p><p id="p-6">The effect of genetic dominance on the probability of convergent evolution has not yet been thoroughly explored [<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">2</a>]. ‘Haldane’s sieve’ [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>] predicts that <em>de novo</em> dominant mutations enjoy a much greater probability of fixation compared to <em>de novo</em> recessive mutations (probability of fixation ≈2<em>hs</em>), because rare dominant mutations are visible to selection [<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. As <em>de novo</em> beneficial recessives are masked to selection when rare, those that do ultimately reach fixation may spend a longer period of time drifting at low frequency [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>]. Likewise, rare recessive migrant alleles are expected to exhibit the same behavior, which may allow more time for mutation to generate ‘competing’ convergent phenotypes upon which selection can act. However, Orr and Betancourt [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] demonstrated that genetic dominance has no effect on the probability of fixation for alleles in mutation-selection balance because recessive alleles have a higher mutation-selection balance frequency. Thus, for relatively high amounts of gene flow, genetic dominance may have little to no effect on the probability of adaptation via migration versus <em>de novo</em> mutation. In fact, the masking of low frequency recessive alleles may result in weaker purifying selection in intervening habitats therefore facilitating the spread of adaptive variation through gene flow [<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">2</a>], although this hypothesis remains to be tested.</p><p id="p-7">Snowshoe hares (<em>Lepus americanus</em>) are one of at least 21 species that have evolved seasonal molts to white winter coats to maintain camouflage in snowy winter environments. Because color molts are cued by photoperiod, and hence may become mismatched under rapidly changing environments [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>,<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>], seasonal camouflage provides a useful trait to understand adaptation to climate change [<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>–<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>]. Reduction in the extent and duration of snow cover is one of the strongest signatures of climate change in the Northern hemisphere [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>,<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>], suggesting that selection should increasingly favor delayed winter-white molts in snowshoe hares to reduce the total duration of coat color mismatch. However, below some annual snow cover threshold, populations are predicted to maintain brown coloration during the winter [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]. Consistent with this, snowshoe hares maintain brown winter camouflage in some temperate environments with reduced snow cover [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>], a strategy that should be increasingly favored under climate change [<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>].</p><p id="p-8">Winter-brown snowshoe hares are common in coastal regions of the Pacific Northwest (PNW), where this Mendelian trait is determined by a recessive variant of the <em>Agouti</em> pigmentation gene [<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>]. The locally adaptive <em>Agouti</em> variant was introduced into snowshoe hares by hybridization with black-tailed jackrabbits ∼9-18 thousand years ago (kya) and subsequently experienced a local selective sweep within the last few thousand years [<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. Occasional records of winter-brown camouflage also occur in more northern regions along the Pacific coast (e.g., Canada and Alaska) and in eastern North America [<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. Although hares are expected to be more or less continuously distributed along suitable Pacific coast environments, there is a deep split and little evidence of recent gene flow between the northern ‘Boreal’ populations and southern ‘PNW’ populations [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. Given this historic population structure, it remains unclear whether the distribution of winter-brown camouflage across populations from disparate parts of the range reflects independent genetic origins (i.e. trait convergence) or spread of the introgressed PNW <em>Agouti</em> allele.</p><p id="p-9">Here, we use new and previously published genetic data to investigate the roles of gene flow and mutation in shaping the evolution of winter-brown camouflage across populations of snowshoe hares. We first combined previously published microsatellite (<em>n</em>=853 individuals, 8 loci) with new and published whole exome data (<em>n</em>=95 individuals) to resolve range-wide patterns of population history and gene flow in snowshoe hares, which provides crucial context for understanding the historical spread and adaptive potential for winter-brown camouflage to climate change. We then generated whole genome sequence (WGS) data of a winter-brown Alaska (AK) hare to test whether winter-brown camouflage in Boreal snowshoe hares arose independently from PNW populations. Next, to understand the geographic limits of the recessive PNW <em>Agouti</em> allele, we used pooled WGS data to estimate its frequency in a winter-white population from Montana (MT), approximately 600 km in a straight-line from the closest winter-brown PNW population. Finally, we used both theoretical predictions and simulations to understand the factors influencing the geographic scope of the PNW <em>Agouti</em> allele and its potential to contribute to rapid adaptation in response to warming climates.</p></div><div class="section" id="sec-2"><h2 class="">Methods</h2><div id="sec-3" class="subsection"><h3>Samples and genomic data generation</h3><p id="p-10">To resolve patterns of range-wide population structure, we performed targeted whole exome enrichment of 12 snowshoe hares previously sampled from 12 localities across Canada, Alaska, and the eastern United States and three hares from the southern Rocky Mountains (representing the ‘Boreal’ and ‘Rockies’ lineages as defined by Cheng et al. [<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">36</a>]). The targeted whole exome capture was designed by Jones et al. [<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>] to enrich for ∼99% of genic regions annotated in the European rabbit (<em>Oryctolagus cuniculus</em>) genome (61.7-Mb spanning 213,164 intervals; ∼25-Mb protein-coding exons, ∼28-Mb untranslated region, ∼9-Mb intronic or intergenic). We followed the library preparation protocols outlined in Jones et al. [<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">29</a>] and sequenced libraries on one lane of Illumina HiSeq2500 with paired-end 100 bp reads (HudsonAlpha Institute for Biotechnology; Huntsville, AL). Exome sequences for Boreal and Rockies snowshoe hares were combined with published whole exome data [<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">29</a>] from 80 PNW snowshoe hares (<em>n</em>=95 total, Table S1), including a monomorphic winter-brown population from southeast British Columbia (BC1, <em>n</em>=14), a monomorphic winter-white population from Seeley Lake area in western MT (<em>n</em>=14), and two polymorphic coat color populations from Oregon (OR, <em>n</em>=26; two localities) and Washington (WA, <em>n</em>=26) [<a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">29</a>].</p><p id="p-11">To resolve the historical spread of the winter-brown PNW <em>Agouti</em> allele, we performed whole genome sequencing of a winter-brown snowshoe hare museum specimen (AK) from southwest Alaska first noted by Link Olson and later obtained through loan from University of Alaska Museum of the North (UAM 116170, collected on 4 January 2013). We also performed whole genome sequencing on a DNA pool of 81 snowshoe hares from two localities in Glacier National Park, MT (Table S1). We extracted genomic DNA from muscle tissue of the AK hare sample following the Qiagen DNeasy Blood and Tissue kit protocol (Qiagen, Valencia, CA). For the pooled MT whole genome sequencing, we pooled previously extracted DNA samples [<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a>] in approximately equimolar quantities based on Qubit concentrations (Invitrogen Qubit Quantitation system LTI). We then prepared genomic libraries for all samples following the KAPA Hyper prep kit manufacturer’s protocol. We sheared genomic DNA to ∼300 bp using a Covaris E220evolution ultrasonicator and performed a stringent size selection using a custom-prepared carboxyl-coated magnetic bead mix [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>]. We determined indexing PCR cycle number for each library with quantitative PCR (qPCR) on a Stratagene Mx3000P thermocycler (Applied Biosystems) using a DyNAmo Flash SYBR Green qPCR kit (Thermo Fisher Scientific). Final libraries were size-selected again with carboxyl-coated magnetic beads, quantified with a Qubit (Thermo Fisher Scientific), and pooled for sequencing by Novogene (Novogene Corporation Ltd.; Davis, CA) on two lanes of Illumina HiSeq4000 using paired-end 150bp reads. The whole genome sequence from the AK hare was combined with previously published whole genome sequencing of five black-tailed jackrabbits, three winter-brown snowshoe hares from OR, WA, and BC1, and three winter white snowshoe hares from MT, a hare from Utah (UT), and a hare from Pennsylvania (PA, Table S1) [<a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">29</a>].</p></div><div id="sec-4" class="subsection"><h3>Read processing and variant calling</h3><p id="p-12">For all raw Illumina sequence data, we trimmed adapters and low-quality bases (mean phred-scaled quality score &lt;15 across 4 bp window) using Trimmomatic v0.35 [<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>] and merged paired-end reads overlapping more than 10 bp and with lower than 10% mismatched bases using FLASH2 [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>].</p><p id="p-13">Whole genome sequence data were mapped to either a snowshoe hare or black-tailed jackrabbit pseudoreference (see Jones et al. [<a id="xref-ref-29-8" class="xref-bibr" href="#ref-29">29</a>] for details) using default settings in BWA-MEM v0.7.12 [<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>]. We used <em>PicardTools</em> to remove duplicate reads with the MarkDuplicates function and assigned read group information with the AddOrReplaceReadGroups function. Using GATK v3.4.046 [<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>], we then identified poorly aligned genomic regions with RealignerTargetCreator and locally realigned sequence data in these regions with IndelRealigner. We performed population-level multi-sample variant calling using default settings with the GATK UnifiedGenotyper. Here, we called variants separately for each previously defined snowshoe hare population genetic cluster (i.e., Boreal, Rockies, BC1, MT, OR, and WA) and for black-tailed jackrabbits. We performed variant filtration in VCFtools v0.1.14 [<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. For whole exome and whole genome data, we filtered genotypes with individual coverage &lt;5× or &gt;70× or with a phred-scaled quality score &lt;30. Additionally, we removed all indel variants and filtered SNPs with a phred-scaled quality score &lt;30, Hardy-Weinberg <em>P</em>&lt;0.001. We required that sites have no missing data across individuals.</p></div><div id="sec-5" class="subsection"><h3>Range-wide population genetic structure and gene flow</h3><p id="p-14">We inferred a maximum likelihood tree with a general time reversible model in RAxML v8.2.8 [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>] using a subset of the concatenated snowshoe hare exome data (<em>n</em>=12 Boreal hares, <em>n</em>=3 Rockies hares, <em>n</em>=12 PNW hares; 21,167,932 total sites) with European rabbit (<em>Oryctolagus cuniculus</em>) as the outgroup. Using this maximum likelihood tree as a starting tree, we estimated a maximum clade credibility tree and node ages with a constant population size coalescent model in BEAST 2 [<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]. We assumed a strict molecular clock and an HKY substitution model using empirical base frequencies. We specified default priors for the kappa parameter, gamma shape parameter, and population size parameter and used a gamma distribution (alpha=0.0344, beta=1) as a prior for the clock rate parameter. We ran the MCMC for 5 million steps and calibrated divergence times using a log-normal distribution for the <em>Oryctolagus</em>-<em>Lepus</em> node age with a median of 11.8 million years (95% prior density: 9.8–14.3) [<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>]. We also performed a species-tree analysis using gene trees in ASTRAL v5.6.3 [<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>]. Gene trees were generated across 200 kb windows (excluding windows wither fewer than 500 sequenced bases) using RAxML with a GTR+gamma model and rapid bootstrap analysis (-f a -# 10). We collapse nodes on gene trees with low bootstrap support values (&lt;=10) and performed ASTRAL analyses on the collapsed gene trees using default settings.</p><p id="p-15">To test for signatures of population structure and gene flow among lineages, we performed a population cluster analysis in A<span class="sc">DMIXTURE</span> v1.3.0 [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>]. We tested <em>K</em> values from 1-10 (representing the number of population clusters) and selected the <em>K</em> value with the lowest cross-validation error. We also estimated range-wide effective migration and diversity surfaces with EEMS [<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>] using extensive microsatellite data (<em>n</em>=853 individuals, 8 loci) generated by Cheng et al. [<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">36</a>]. Varying the number of demes (50, 100, and 200) had little effect on estimates of effective rates of migration and diversity, therefore we only report results for 200 demes. We used default hyper-parameter values and tuned the proposal variances such that proposals were accepted approximately 30% of the time. We ran EEMS for 2 million iterations with a burn-in of 1 million iterations and thinning iteration of 9999. Runs produced strong correlations between observed and expected genetic dissimilarity both within and between demes, indicating good model fit.</p></div><div id="sec-6" class="subsection"><h3>Geographic distribution of the PNW <em>Agouti</em> allele</h3><p id="p-16">The winter-brown AK snowshoe hare was collected approximately 3000 km (via the Pacific coast) from Vancouver, BC, the northern limit of winter-brown PNW hare populations [<a id="xref-ref-29-9" class="xref-bibr" href="#ref-29">29</a>]. To test whether the introgressed PNW <em>Agouti</em> allele has seeded winter-brown camouflage in southwest AK, we generated a tree of the PNW <em>Agouti</em> haplotype region from our whole genome sequence data of winter-brown and winter-white snowshoe hares (<em>n</em>=7) and black-tailed jackrabbits (<em>n</em>=5). We defined the PNW <em>Agouti</em> haplotype region (chr4:5,340,275 – 5,633,020, oryCun2 coordinates) based on the location of the introgressed black-tailed jackrabbit tract identified by Jones et al. [<a id="xref-ref-29-10" class="xref-bibr" href="#ref-29">29</a>] using a hidden Markov model. We then inferred a maximum likelihood phylogeny for this interval using RAxML [<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>] as above. Node support values were generated from 1000 replicate bootstrap runs.</p><p id="p-17">We also estimated the frequency of the PNW <em>Agouti</em> haplotype among pooled individuals from two localities in MT (<em>n</em>=81) that are 575 km and 627 km from the polymorphic sampling locality in WA, where winter-brown camouflage is relatively common. We used PoPoolation2 [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>] to calculate the frequency of the winter brown-associated alleles at 555 sites across the introgressed <em>Agouti</em> haplotype (chr4: 5,340,275 – 5,633,020). These positions were previously shown to be strongly associated with coat color based on a likelihood ratio test of allele frequency differences between winter-brown and winter-white individuals from low coverage whole genome sequence data [<a id="xref-ref-29-11" class="xref-bibr" href="#ref-29">29</a>]. We summed the counts of winter-white and winter-brown alleles across all positions to estimate both the mean and standard deviation of winter-brown allele frequency. We excluded seven positions with unusually high frequencies of the winter-brown allele (∼45-100%) supported by reads that did not otherwise carry brown-associated alleles (i.e., incongruent haplotype information), as these likely reflect sequencing errors.</p></div><div id="sec-7" class="subsection"><h3>Probability of convergent adaptation through independent mutation</h3><p id="p-18">To compare the observed spread of the PNW <em>Agouti</em> allele to theoretical expectations, we estimated the relative probability of adaptation via independent mutations with distance from a focal habitat patch using the model developed by Ralph and Coop [<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">2</a>] (equation 12) under a range of conditions. Since the relative probability of adaptation through migration or <em>de novo</em> mutation depends strongly on the rate at which mutations generate convergent phenotypes [<a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">2</a>], we tested a range of plausible mutation rates. A previous study using over 7 million house mice estimated a mean rate of spontaneous visible coat color mutations of 11e<sup>-6</sup> per locus/gamete [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>], which is ∼1930× higher than the average genome-wide germline per site mutation rate (5.7e<sup>-9</sup> per site/gamete) [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>]. Given a genome-wide mutation rate of 2.35e<sup>-9</sup> per site/generation for rabbits [<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>], we tested a high mutation rate of <em>μ</em>=4.54e<sup>-6</sup> (i.e., the overall expected rate of visible coat color mutations) and a low mutation rate of <em>μ</em>=4.54e<sup>-8</sup> (i.e., assuming only 1% of coat color mutations would lead to brown winter camouflage). We also tested how strong (<em>s</em><sub><em>m</em></sub>=0.01), moderate (<em>s</em><sub><em>m</em></sub>=0.001), or weak (<em>s</em><sub><em>m</em></sub>=0.0001) purifying selection against the winter-brown allele in intervening (i.e., snow-covered) habitats influences the probability of adaptation through independent mutations. We assumed a relatively high mean dispersal distance of 2 km [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>], which is likely to produce a conservative estimate of the probability of adaptation via independent mutation. We also assumed that the width of the secondary habitat patch was relatively small (<em>w</em>=1 or ∼6-45 km, depending on <em>s</em><sub><em>m</em></sub>), which is consistent with observations of winter-brown camouflage at low frequency along portions of the range edge. Finally, we assumed <em>d</em>=1 (dimension of the habitat), <em>C</em>=1, and <em>γ</em>=0.5.</p></div><div id="sec-8" class="subsection"><h3>Simulations of selection on migrant alleles with genetic dominance</h3><p id="p-19">Theory predicts that allelic dominance should not influence a mutation’s probability of fixation under positive selection if it starts in mutation/migration-selection balance [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]. Nonetheless, rare recessive mutations may increase in frequency more slowly relative to rare dominant mutations because they are initially invisible to selection, allowing more opportunity for competition from convergent phenotypes arising through independent mutations. We used simulations to test how the genetic dominance of alleles at equivalent mutation-selection equilibrium frequencies influences the probability and rate of adaptation to new habitats when environmental conditions change. In SLiM 3.0 [<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>] we performed 100 simulations of the MT population (estimated <em>N</em><sub><em>e</em></sub>=245430) [<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>] with a positively selected recessive mutation (<em>s</em>=0.01, <em>h</em>=0) starting at the inferred frequency of the PNW <em>Agouti</em> allele in MT (<em>p</em>). We assumed that <em>p</em> reflects the equilibrium frequency (<em>x</em>) between the migration rate of the allele into the environment (<em>m</em>) and the selection coefficient against the allele (<em>s</em>), which is given as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/09/851766/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/09/851766/embed/inline-graphic-1.gif"/></noscript></span></span> for a recessive mutation. Given that <em>x</em> for a dominant mutation is simply <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/09/851766/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/01/09/851766/embed/inline-graphic-2.gif"/></noscript></span></span>, we calculated <em>x</em> of an equivalent dominant mutation (same values of <em>m</em> and <em>s</em>) as simple <em>p</em><sup>2</sup>. We then repeated these simulations for a positively selected (<em>s</em>=0.01), completely dominant allele (<em>h</em>=1) starting at its expected mutation-selection balance frequency. For each simulation we tracked the frequency of the selected allele until it was either fixed or lost. To validate these simulations, we compared the probabilities of allelic fixation from simulations to fixation probabilities analytically derived by Kimura [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>]. To test for significant differences in fixation probabilities and rates of adaptation between recessive and dominant mutations we used Student’s t-tests in R [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>].</p></div></div><div class="section" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Snowshoe hare phylogeography</h3><p id="p-20">We combined previously generated whole exome data for 80 PNW snowshoe hares (mean coverage 21× ± 7.6 per individual) with new whole exomes of 15 hares across the range sequenced to a mean coverage of 20.2× ± 8.2 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). Exome-wide phylogenetic analyses show three broad phylogeographic groups of snowshoe hares (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>) with unambiguous ASTRAL support scores using 6582 gene trees, consistent with previous studies [<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">37</a>]. However, ∼12-25% of SNPs within phylogeographic groups appear to be shared across some population boundaries (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), indicating some amount of shared ancestral variation or gene flow. We estimated that snowshoe hares from Canada, Alaska, and the eastern United States— representing the Boreal lineage—diverged approximately 97.2 kya (95% posterior density: 77.4-120.8 kya) from PNW and Rockies snowshoe hares (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1B</a>). PNW and Rockies hare populations are estimated to have split approximately 47.1 kya (95% posterior density: 37.2-58.2 kya).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/1100073/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/1100073/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/1100073" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-21" class="first-child">Counts of the number of private (e.g., Boreal/Boreal) versus shared SNPs (e.g., Rockies/Boreal) for each major snowshoe hare clade. Individuals from WA represent the PNW.</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Range-wide phylogeography of snowshoe hares based on whole exome sequences. The snowshoe hare range is colored brown to gray according to the modelled probability of winter-brown camouflage from Mills et al. (2018). Sizes of sampling locality circles are scaled to sample size and are colored according to their population assignment (see C). (B) A maximum credibility phylogenetic tree estimated with BEAST 2 [45]. All nodes have posterior probabilities &gt;99%. Each major node shows the median node age in thousands of years (95% posterior density in parentheses and gray rectangles) and the ASTRAL support score. (C) The Admixture plot shows the proportion of ancestry across samples based on a K=3 clustering, which had the lowest cross validation error." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969254263" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(&lt;strong&gt;A&lt;/strong&gt;) Range-wide phylogeography of snowshoe hares based on whole exome sequences. The snowshoe hare range is colored brown to gray according to the modelled probability of winter-brown camouflage from Mills et al. (2018). Sizes of sampling locality circles are scaled to sample size and are colored according to their population assignment (see C). (&lt;strong&gt;B&lt;/strong&gt;) A maximum credibility phylogenetic tree estimated with BEAST 2 [45]. All nodes have posterior probabilities &gt;99%. Each major node shows the median node age in thousands of years (95% posterior density in parentheses and gray rectangles) and the ASTRAL support score. (&lt;strong&gt;C&lt;/strong&gt;) The Admixture plot shows the proportion of ancestry across samples based on a &lt;em&gt;K&lt;/em&gt;=3 clustering, which had the lowest cross validation error.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F1.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F1.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-22" class="first-child">(<strong>A</strong>) Range-wide phylogeography of snowshoe hares based on whole exome sequences. The snowshoe hare range is colored brown to gray according to the modelled probability of winter-brown camouflage from Mills et al. (2018). Sizes of sampling locality circles are scaled to sample size and are colored according to their population assignment (see C). (<strong>B</strong>) A maximum credibility phylogenetic tree estimated with BEAST 2 [<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">45</a>]. All nodes have posterior probabilities &gt;99%. Each major node shows the median node age in thousands of years (95% posterior density in parentheses and gray rectangles) and the ASTRAL support score. (<strong>C</strong>) The A<span class="sc">dmixture</span> plot shows the proportion of ancestry across samples based on a <em>K</em>=3 clustering, which had the lowest cross validation error.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Population structure analyses in A<span class="sc">dmixture</span> also found strongest support for three population clusters, although MT snowshoe hares (in the PNW lineage) showed 11-16% of their ancestry assigned to the Rockies lineage, indicating ongoing gene flow or continuous genetic structure (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1C</a>). The A<span class="sc">dmixture</span> analysis also indicated little apparent gene flow between the Boreal lineage and PNW or Rockies lineages. Likewise, microsatellite-based estimates of effective migration surfaces revealed that effective migration rates are approximately 100-fold lower than the range-wide average near the apparent zone of contact between PNW and Boreal lineages in western North America (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>). We also estimated relatively low effective migration rates across the southwestern edge of snowshoe hare range (western US; log(<em>m</em>) ≈-1, or 10-fold lower than the mean). In contrast, effective migration rates in the northwest part of the range (Alaska and western Canada) appeared relatively high (∼10-fold higher than the mean). Effective diversity surfaces show that the highest relative genetic diversity occurs in the eastern extent (Boreal lineage) and southwestern extent (PNW lineage) of the snowshoe hare range (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a>), consistent with previous studies suggesting that these regions were likely glacial refugia [<a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">36</a>]. Relative genetic diversity gradually decreased moving from east to west across the Boreal range and the lowest genetic diversity was observed in the Rocky Mountain lineage, distributed across the western US.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Effective migration rates for snowshoe hares inferred from range-wide microsatellite data set (853 individuals, 8 loci) from Cheng et al. (2014). The sizes of circles are scaled to the number of samples collected at that location. Effective migration rate is measured as the rate of decay in genetic similarity of individuals across space. Regions that are colored white are characterized by isolation-by-distance while regions that are colored blue or red have higher or lower effective migration, respectively. (B) Effective diversity rates based on the same microsatellite data set. Here effective diversity rates measure the genetic dissimilarity between individuals in the same deme, where blue regions have higher than average diversity and red regions have lower than average diversity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969254263" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(&lt;strong&gt;A&lt;/strong&gt;) Effective migration rates for snowshoe hares inferred from range-wide microsatellite data set (853 individuals, 8 loci) from Cheng et al. (2014). The sizes of circles are scaled to the number of samples collected at that location. Effective migration rate is measured as the rate of decay in genetic similarity of individuals across space. Regions that are colored white are characterized by isolation-by-distance while regions that are colored blue or red have higher or lower effective migration, respectively. (B) Effective diversity rates based on the same microsatellite data set. Here effective diversity rates measure the genetic dissimilarity between individuals in the same deme, where blue regions have higher than average diversity and red regions have lower than average diversity.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F2.medium.gif" width="440" height="377"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F2.medium.gif" width="440" height="377"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-24" class="first-child">(<strong>A</strong>) Effective migration rates for snowshoe hares inferred from range-wide microsatellite data set (853 individuals, 8 loci) from Cheng et al. (2014). The sizes of circles are scaled to the number of samples collected at that location. Effective migration rate is measured as the rate of decay in genetic similarity of individuals across space. Regions that are colored white are characterized by isolation-by-distance while regions that are colored blue or red have higher or lower effective migration, respectively. (B) Effective diversity rates based on the same microsatellite data set. Here effective diversity rates measure the genetic dissimilarity between individuals in the same deme, where blue regions have higher than average diversity and red regions have lower than average diversity.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Convergent evolution and the distribution of winter-brown camouflage variation</h3><p id="p-25">Under a single origin of winter brown camouflage, we would expect the winter-brown AK individual to nest within the black-tailed jackrabbit clade with the other winter-brown snowshoe hares from the PNW. However, our phylogenetic analysis indicates that the winter-brown AK individual unambiguously groups with a winter-white Boreal hare from PA (100% bootstrap support), and more broadly with other winter-white hares across the range (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a>). These results demonstrate that the winter-brown phenotype of this AK hare is caused by mutations that are independent from the introgressed <em>Agouti</em> haplotype that encodes winter-brown camouflage in the PNW, approximately 3000 km away. Determining the genetic basis of this independent origin of winter brown camouflage will likely require an independent genetic association study in these populations.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A maximum clade credibility tree of the introgressed Agouti locus (&#x223C;293 kb) based on whole genome sequencing of black-tailed jackrabbits (BTJR; black circles) and snowshoe hares. Values indicate node support based on bootstrapping. Colored stars indicate winter-brown snowshoe hares from Alaska (yellow, UAM 116170 pictured top) or the Pacific Northwest (purple, pictured bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969254263" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A maximum clade credibility tree of the introgressed &lt;em&gt;Agouti&lt;/em&gt; locus (&#x223C;293 kb) based on whole genome sequencing of black-tailed jackrabbits (BTJR; black circles) and snowshoe hares. Values indicate node support based on bootstrapping. Colored stars indicate winter-brown snowshoe hares from Alaska (yellow, UAM 116170 pictured top) or the Pacific Northwest (purple, pictured bottom).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F3.medium.gif" width="440" height="299"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F3.medium.gif" width="440" height="299"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-26" class="first-child">A maximum clade credibility tree of the introgressed <em>Agouti</em> locus (∼293 kb) based on whole genome sequencing of black-tailed jackrabbits (BTJR; black circles) and snowshoe hares. Values indicate node support based on bootstrapping. Colored stars indicate winter-brown snowshoe hares from Alaska (yellow, UAM 116170 pictured top) or the Pacific Northwest (purple, pictured bottom).</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Pooled whole genome sequencing detected that the recessive winter-brown <em>Agouti</em> allele occurs at an estimated frequency of 1.24% (± 0.01%) across predominately winter-white MT localities ∼600 km from the polymorphic zone. Interestingly, long-term live-trapping and radiotelemetry-based field work from this region has resulted in a single observation of a winter-brown hare in approximately 300 winter hare observations (0.33%; Scott Mills pers. obs.), although this observed frequency is slightly higher than the Hardy-Weinberg expectation under the estimated frequency of the winter-brown <em>Agouti</em> allele (<em>p</em><sup><em>2</em></sup>=0.015%, binomial test <em>p</em>-value=0.044).</p><p id="p-28">To understand the factors that have shaped the geographic distribution of the PNW <em>Agouti</em> allele, we modelled the relative probability of adaptation through independent mutation with distance from a focal habitat using the theoretical framework of Ralph and Coop [<a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">2</a>]. The change in relative probability of adaptation through independent mutation depended strongly on both the mutation rate and selection coefficient against the allele in intervening habitats (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>). Stronger selection coefficients produced a more rapid shift in the convergence probability with distance relative to weaker selection (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>). For instance, under a high mutation rate to the winter-brown phenotype (<em>μ</em>=4.54e<sup>-6</sup>) and strong purifying selection (<em>s</em><sub><em>m</em></sub>=0.01) the probability of independent adaptation transitioned sharply from <em>P</em>=0.1 at 78 km to <em>P</em>=0.9 at 140 km from a focal habitat. Under a low mutation rate to the winter-brown phenotype (<em>μ</em>=4.54e<sup>-8</sup>) and weak selection (<em>s</em><sub><em>m</em></sub>=0.0001), independent adaptation was much less likely at close distances, as expected, and the probability increased more gradually (<em>P</em>=0.1 at 778 km to <em>P</em>=0.9 at 1400 km).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The probability of adaptation through independent mutations in snowshoe hares as a function of distance in kilometers from a focal habitat patch harboring a locally adaptive variant. The probability of independent mutation is calculated according to equation 12 in Ralph and Coop (2015). Here we varied the mutation rate to the winter-brown phenotype (&#x3BC;; black=4.54e-8, gray=4.54e-6) and the negative selection coefficient in intervening habitats (sm; solid line=0.0001, dashed line=0.001, dotted line=0.01)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969254263" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The probability of adaptation through independent mutations in snowshoe hares as a function of distance in kilometers from a focal habitat patch harboring a locally adaptive variant. The probability of independent mutation is calculated according to equation 12 in Ralph and Coop (2015). Here we varied the mutation rate to the winter-brown phenotype (&lt;em&gt;&#x3BC;&lt;/em&gt;; black=4.54e&lt;sup&gt;-8&lt;/sup&gt;, gray=4.54e&lt;sup&gt;-6&lt;/sup&gt;) and the negative selection coefficient in intervening habitats (&lt;em&gt;s&lt;/em&gt;&lt;sub&gt;&lt;em&gt;m&lt;/em&gt;&lt;/sub&gt;; solid line=0.0001, dashed line=0.001, dotted line=0.01).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F4.medium.gif" width="440" height="320"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F4.medium.gif" width="440" height="320"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-29" class="first-child">The probability of adaptation through independent mutations in snowshoe hares as a function of distance in kilometers from a focal habitat patch harboring a locally adaptive variant. The probability of independent mutation is calculated according to equation 12 in Ralph and Coop (2015). Here we varied the mutation rate to the winter-brown phenotype (<em>μ</em>; black=4.54e<sup>-8</sup>, gray=4.54e<sup>-6</sup>) and the negative selection coefficient in intervening habitats (<em>s</em><sub><em>m</em></sub>; solid line=0.0001, dashed line=0.001, dotted line=0.01).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Migration-selection balance</h3><p id="p-30">Assuming an environmental shift towards less snow results in strong positive selection (<em>s</em>=0.01) on standing variation of the recessive (winter-brown) <em>Agouti</em> allele in MT, we used simulations to estimate that the corresponding probability of fixation would be ∼81% (95% confidence intervals: 72.2-87.5%; <a id="xref-fig-5-1" class="xref-fig" href="#F5">Fig. 5A</a>), which is consistent with an analytically-derived probability of 78.1%. For an equivalent dominant mutation starting in mutation-selection balance frequency (0.015%) the simulated probability of fixation was approximately 77% using simulations (95% confidence intervals: 67.8-84.2%) or 77.9% using the Kimura [<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a>] equation. Neither estimate was significantly different from the fixation probability of a recessive mutation (<em>p</em>=0.48, two-tailed test of equality of proportions). However, we observed striking differences in the rates of increase, conditional on fixation, of dominant and recessive mutations following a sudden environmental change (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Fig. 5B</a>). The mean time to fixation was significantly faster (<em>p</em>&lt;2.2e-16, Student’s T-test) for a recessive mutation (9645 generations, SD=3609 generations) compared to a dominant mutation (19445 generations, SD=7273 generations, <em>p</em>&lt;2.2e<sup>-16</sup>). However, as expected, we see that initial rates of allele frequency change and phenotypic adaptation are considerably faster for a positively selected dominant mutation (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Fig. 5B</a>). For instance, the mean time for the selected phenotype determined by the recessive mutation to reach a frequency of 75% (<em>p</em>=0.866) was 8837 generations (SD=3630 generations), compared to 1007 generations (SD=116 generations) if determined by the dominant mutation. The striking difference in the rate at which beneficial dominant versus recessive alleles contribute to adaptation is maintained even up to the point at which the selected phenotype reaches a frequency of 99%, which takes approximately 9262 generations (SD=3624 generations) for a recessive mutation and only 2030 generations for dominant mutation (SD=141 generations).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) The simulated probability of fixation of a completely dominant (red, mean=77%, N=100) or recessive (blue, mean=81%, N=100) mutation experiencing positive selection and starting in migration-selection balance frequency (0.015% for dominant, 1.24% for recessive). (B) The simulated allele frequency trajectories of the same dominant (red) and recessive (blue) mutations starting in migration-selection balance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-969254263" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(&lt;strong&gt;A&lt;/strong&gt;) The simulated probability of fixation of a completely dominant (red, mean=77%, &lt;em&gt;N&lt;/em&gt;=100) or recessive (blue, mean=81%, &lt;em&gt;N&lt;/em&gt;=100) mutation experiencing positive selection and starting in migration-selection balance frequency (0.015% for dominant, 1.24% for recessive). (&lt;strong&gt;B&lt;/strong&gt;) The simulated allele frequency trajectories of the same dominant (red) and recessive (blue) mutations starting in migration-selection balance.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F5.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F5.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2020/01/09/851766/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-31" class="first-child">(<strong>A</strong>) The simulated probability of fixation of a completely dominant (red, mean=77%, <em>N</em>=100) or recessive (blue, mean=81%, <em>N</em>=100) mutation experiencing positive selection and starting in migration-selection balance frequency (0.015% for dominant, 1.24% for recessive). (<strong>B</strong>) The simulated allele frequency trajectories of the same dominant (red) and recessive (blue) mutations starting in migration-selection balance.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-13"><h2 class="">Discussion</h2><p id="p-32">A growing number of studies have found evidence for convergent adaptation within and between species [<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">8</a>–<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>,<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>,<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>,<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>], although we often lack an understanding of the forces that determine whether local adaptation occurs through independent <em>de novo</em> mutations or migration of pre-existing alleles from other populations [<a id="xref-ref-2-10" class="xref-bibr" href="#ref-2">2</a>]. Our study provides rare empirical insights into how gene flow, mutation, allelic dominance, and selection interact to shape the spatial scale and pace of local adaptation to new or changing environments.</p><p id="p-33">Understanding the potential for adaptive variation to spread between populations across large spatial scales requires basic insights into range-wide patterns of population structure and gene flow. Winter-brown camouflage in snowshoe hares occurs across only ∼5% of the total range but is broadly-distributed from the western edge of the range along the Pacific coast to the eastern extent of the range in Pennsylvania [<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">28</a>,<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>,<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>]. Previous phylogeographic studies based on limited genetic data suggest that this distribution spans a deep phylogenetic boundary between Boreal and PNW lineages (∼2 Myr divergence) [<a id="xref-ref-36-7" class="xref-bibr" href="#ref-36">36</a>,<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">37</a>] with little evidence for gene flow. With whole exomes (<em>n</em>=95), our phylogenetic analysis also supports a deep split but suggests a substantially more recent divergence time between Boreal and Rockies/PNW lineages than previously estimated (∼97 kya; <a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1B</a>). Our more recent estimates may be attributable to the increased power of whole exomes or heavy reliance on mitochondrial DNA in previous studies [<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">37</a>], which can lead to overestimation of divergence times [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>]. Consistent with previous studies, our A<span class="sc">dmixture</span> analyses and range-wide effective migration surfaces suggest low gene flow between the PNW and Boreal lineages (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1C</a>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2A</a>). The absence of observable gene flow is perhaps striking given their close proximity to each other in western North America. Melo-Ferreira et al. [<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">37</a>] hypothesized that these lineages may exhibit incipient reproductive isolation. The evolution of reproductive isolation between these populations is possible, but, given our more recent divergence time estimates, we suggest that reduced gene flow may reflect quite recent secondary contact between these populations in western North America. Genetic and fossil evidence suggests the common ancestors of PNW and Boreal populations occupied separate refugia in southern and eastern North America, respectively, during the last glacial maximum [<a id="xref-ref-36-8" class="xref-bibr" href="#ref-36">36</a>]. The east-to-west phylogenetic clustering (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1B</a>) and genetic diversity gradient (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2B</a>) of the Boreal population is consistent with a recent range expansion from this eastern refugia [<a id="xref-ref-36-9" class="xref-bibr" href="#ref-36">36</a>], which implies the PNW and Boreal hares have only recently experienced secondary contact. However, more detailed population history modeling and sampling of the putative contact zone is required to test these hypotheses.</p><p id="p-34">Consistent with low historical gene flow, the AK winter-brown hare lacks the PNW winter-brown <em>Agouti</em> haplotype (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3</a>), indicating a role for independent mutation leading to the convergent evolution of brown winter coats. This likely reflects constraints on the migration of adaptive variation between populations. Even in the absence of intrinsic or extrinsic reproductive isolation, the rate at which a positively selected variant spreads across a uniform environment is constrained by dispersal distance and the strength of positive selection [<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>]. Across spatially or temporally heterogenous landscapes, the spread of an adaptive allele between patches is further constrained by the strength or frequency of purifying selection in maladaptive habitats or climatic periods [<a id="xref-ref-2-11" class="xref-bibr" href="#ref-2">2</a>]. Given that previous models suggest a low probability of winter-brown camouflage along the majority of the Pacific coast [<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">28</a>], we suspect that the winter-brown PNW variant would have to traverse snowy habitats where it is strongly disfavored in order to reach Alaska, ∼3000 away via the coast. At this distance, the probability of adaptation through gene flow is virtually zero under a range of model assumptions (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4</a>). In fact, dispersal limitations and patchy range-edge habitats should generally favor independent evolution of winter-brown camouflage at scales beyond 100-1000 km (depending on the assumed values of <em>μ</em> and <em>s</em><sub><em>m</em></sub>; <a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4</a>). Further research is needed to dissect the genetic basis of winter-brown camouflage in the northwestern and eastern edges of the range (e.g., Alaska and Pennsylvania). Subtle phenotypic similarities between observed winter-brown hares in Alaska and Pennsylvania could imply a shared genetic basis (e.g., white feet versus brown feet in the PNW, unpublished data). However, our theoretical modelling would suggest that they likely reflect independent mutations as well (i.e., ∼5900 km between AK and PA sampling sites).</p><p id="p-35">Convergent evolution is thought to be more common for ‘loss-of-function’ traits because they may have a larger mutational target size relative to ‘gain-of-function’ traits [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>]. Convergent color adaptation involving loss-of-function mutations has been shown between different species of lizard [<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">9</a>,<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>] and cavefish [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>] and the repeated evolution of melanism across populations of deer mice involves convergent loss-of-function <em>Agouti</em> mutations [<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. In PNW hares, adaptive introgression of <em>Agouti</em> variation from black-tailed jackrabbits appears to have caused a reversion to the ancestral winter-brown condition in <em>Lepus</em> (i.e., the likely ancestral state before the common ancestor of winter-white <em>Lepus</em> species). As the ancestral winter-brown variant is recessive, this implies that derived winter-white camouflage in snowshoe hares evolved through dominant gain-of-function mutations, consistent with the seasonal upregulation of <em>Agouti</em> during the development of white coats [<a id="xref-ref-29-12" class="xref-bibr" href="#ref-29">29</a>]. Independent origins of winter-brown camouflage across the snowshoe hare range could similarly involve relatively simple loss-of-function mutations that break the molecular pathways contributing to the development of white winter coats. Indeed, the evolution of darkened winter coats in some populations of mountain hares (<em>Lepus timidus</em>) appears to have also involved introgression of a recessive, loss-of-function <em>Agouti</em> variant [<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">59</a>]. Collectively, our results suggest that geographic distance and mutational target size, in addition to population structure and history, should play a crucial role in generating hypotheses about the relative roles of independent mutation and gene flow in adaptation.</p><p id="p-36">Despite the evidence for independent winter-brown mutations at broad spatial scales, we also find that the PNW <em>Agouti</em> allele has traversed a substantial distance (∼600 km) to western Montana – a predominately winter-white locality – where it occurs at a frequency of 1.24%. At this starting frequency, both theory and simulations show that a shift in environmental conditions towards favoring the brown allele results in a ∼80% probability of fixation (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Fig. 5</a>). The high probability of adaptation through gene flow ∼600 km from a focal patch is consistent with a scenario of very weak purifying selection (<em>s</em><sub><em>m</em></sub>=0.0001) against the winter-brown allele in snowy environments under the Ralph and Coop [<a id="xref-ref-2-12" class="xref-bibr" href="#ref-2">2</a>] model (<em>P</em>(independent mutation)=∼3% (<em>μ</em>=4.54e-8) or ∼76% (<em>μ</em>=4.54e-6); <a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4</a>). Although winter-brown camouflage is likely more strongly deleterious in winter-white environments given the known fitness consequences of mismatch [<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>], purifying selection against the winter-brown <em>Agouti</em> allele in winter-white environments may be weak or absent because it is recessive and thus hidden to selection at low frequency. In agreement with Ralph and Coop [<a id="xref-ref-2-13" class="xref-bibr" href="#ref-2">2</a>], we suggest that adaptation between distant populations via gene flow may be relatively more common through recessive variation compared to dominant variation when purifying selection is strong in intervening habitats.</p><p id="p-37">Climate change is expected to reduce winter snow cover across the snowshoe hare range [<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>], which could potentially result in winter-brown camouflage being favored in predominately winter-white populations [<a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">28</a>]. Under this scenario, positive selection could operate on convergent <em>de novo</em> winter-brown mutations or on standing genetic variation for winter-brown camouflage to facilitate rapid adaptation. Using simulations of positive selection, we show that although dominant and recessive mutations in mutation/migration-selection balance share a similar fixation probability (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Fig. 5A</a>), they are likely to experience vastly different frequency change dynamics (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Fig. 5B</a>) [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>] and thus lead to very different rates of adaptation following environmental change. For instance, we show that in MT the initial rate of adaptation from segregating winter-brown <em>Agouti</em> variation is likely to be slow relative to an equivalently beneficial dominant mutation. While recessive mutations tended to fix more quickly than dominant mutations, they also took substantially longer to reach high frequencies in populations (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Fig. 5B</a>; e.g., 8837 vs. 1007 generations for a phenotype determined by recessive vs. dominant mutation to reach a population frequency of 75%, respectively). This pattern can be explained by the different allele frequency phases most strongly affected by genetic drift. For instance, positive selection can readily act on beneficial dominant mutations at low frequency, but at high frequency the recessive wildtype allele is hidden in heterozygotes and genetic drift governs (and generally slows) fixation dynamics. The opposite is true for beneficial recessive mutations, which are governed by genetic drift at low frequency but driven to fixation by selection. Overall, selection on low frequency recessive variation may be a relatively ineffective way to adapt rapidly to changing environments. In fact, the temporal lag for the spread of beneficial recessive variation may be sufficient enough to allow time for dominant independent mutations (e.g., gain-of-function MC1R mutations that result in melanism) [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>] to appear and spread before the recessive allele increases appreciably. These findings highlight the important roles of genetic dominance in shaping both the geographic scope and rate of convergent adaptation and underscores the need for further study.</p><p id="p-38">Collectively, our study provides important insights into long-standing evolutionary questions, as well as the potential for adaptation to climate change in snowshoe hares. A key to understanding the potential of adaptive variation to spread under climate change is revealing how it has spread in the past. We show that adaptive responses to reduced snow cover in snowshoe hares have involved both the local spread of winter-brown camouflage in the PNW as well as convergent evolution elsewhere in the range. In snowshoe hares and other seasonally changing species, regions of winter-camouflage polymorphism may be crucial areas to maintain population connectivity as a conduits for the spread of winter-brown camouflage across broader portions of the range [<a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">28</a>]. However, facilitating gene flow alone may not be sufficient to facilitate rapid adaptive responses, given the broad range of snowshoe hares and the genetic architecture of winter-brown camouflage variation. Rather, we suspect that rapid climate change responses in winter-white populations distant from winter-brown habitat may also likely involve independent origins of winter-brown camouflage and selection on variation in coat color phenology (i.e., shifts in the timing of winter-white molts). Although the genetic basis of variation in camouflage phenology remains unresolved in this system, it is likely quantitative and perhaps more responsive to selection. Regardless, this study represents an important step towards making predictions about evolutionary responses under climate change in this species.</p></div><div class="section" id="sec-15"><h2 class="">Data Accessibility</h2><p id="p-40">Original sequence data are available in the Sequence Read Archive (<a href="http://www.ncbi.nlm.nih.gov/sra">www.ncbi.nlm.nih.gov/sra</a>). Previously generated whole exome and genome sequence data of snowshoe hare (BioProject PRJNA420081, SAMN02782769, SAMN07526959) are also available in the Sequence Read Archive.</p></div><div class="section" id="sec-16"><h2 class="">Author Contributions</h2><p id="p-41">MRJ, JMG, and LSM conceived and designed the study; LSM led all field work and related collection of biological samples; MRJ performed molecular lab work and data analysis, with contributions and guidance from JMG and JDJ; MRJ led the writing with contributions from all authors. All authors gave final approval for publication and agree to be held accountable for the work performed therein.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-39">We thank E. Cheng and K. Garrison for assistance with sample collection. We thank L. Olson for informing us on the recent collection of a winter brown snowshoe hare, and L. Olson and the University of Alaska Museum of the North facilitating a tissue loan of this specimen (UAM 116170). Our research would not be possible without the incredible and irreplaceable support of natural history museums. We thank J. Melo-Ferreira, P. C. Alves, M. S. Ferreira, N. Herrera, E. Kopania, A. Kumar, M. Zimova, K. Garrison, and the UNVEIL network for helpful discussions. Funding and support for this research was provided a National Science Foundation (NSF) Graduate Research Fellowship (DGE-1313190), NSF Doctoral Dissertation Improvement Grant (DGE-1702043), NSF Graduate Research Opportunities Worldwide, NSF EPSCoR (OIA-1736249), and NSF (DEB-0841884; DEB-1907022), the Drollinger-Dial Foundation, American Society of Mammalogists Grant-in-aid of Research, and a Swiss Government Excellence Scholarship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-dataset fn-group-external-links" id="fn-1"><p id="p-3">
<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA420081">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA420081</a>
</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.1" data-doi="10.1007/BF02459575"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">1931</span> <span class="cit-article-title">Evolution in Mendelian populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">97</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D159%26rft.atitle%253DEVOLUTION%2BIN%2BMENDELIAN%2BPOPULATIONS%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02459575%26rft_id%253Dinfo%253Apmid%252F17246615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIxNi8yLzk3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.2" data-doi="10.1371/journal.pgen.1005630)"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2015</span> <span class="cit-article-title">Convergent evolution during local adaptation to patchy landscapes</span>. <abbr class="cit-jnl-abbrev">PLOS Genet</abbr>. <span class="cit-vol">11</span>, <span class="cit-fpage">e1005630</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pgen.1005630)</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenet%26rft.volume%253D11%26rft.spage%253De1005630%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005630%2529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005630)&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-851766v2.3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">JBS</span></span>. <span class="cit-pub-date">1932</span> <span class="cit-source">The causes of evolution</span>. <span class="cit-publ-loc">London</span>: <span class="cit-publ-name">Longmans, Green and Co</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.4" data-doi="10.1016/j.tree.2006.11.002"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bridle</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vines</span>  <span class="cit-name-given-names">TH</span></span>. <span class="cit-pub-date">2007</span> <span class="cit-article-title">Limits to evolution at range margins: when and why does adaptation fail?</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol</abbr>. <span class="cit-vol">22</span>, <span class="cit-fpage">140</span>–<span class="cit-lpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.tree.2006.11.002</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DBridle%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D140%26rft.epage%253D147%26rft.atitle%253DLimits%2Bto%2Bevolution%2Bat%2Brange%2Bmargins%253A%2Bwhen%2Band%2Bwhy%2Bdoes%2Badaptation%2Bfail%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2006.11.002%26rft_id%253Dinfo%253Apmid%252F17113679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2006.11.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17113679&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244947400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.5" data-doi="10.1038/nature09670"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sgrò</span>  <span class="cit-name-given-names">CM.</span></span> <span class="cit-pub-date">2011</span> <span class="cit-article-title">Climate change and evolutionary adaptation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">470</span>, <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nature09670</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D470%26rft.issue%253D7335%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DClimate%2Bchange%2Band%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09670%26rft_id%253Dinfo%253Apmid%252F21350480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21350480&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287652900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.6" data-doi="10.1111/j.1469-1809.1937.tb02153.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">RA</span></span>. <span class="cit-pub-date">1937</span> <span class="cit-article-title">The wave of advance of advantageous genes</span>. <abbr class="cit-jnl-abbrev">Ann. Eugen</abbr>. <span class="cit-vol">7</span>, <span class="cit-fpage">355</span>–<span class="cit-lpage">369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1469-1809.1937.tb02153.x</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen%26rft.volume%253D7%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1937.tb02153.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1937.tb02153.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.7" data-doi="10.1086/691233"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bay</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bernatchez</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ghalambor</span>  <span class="cit-name-given-names">CK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lasky</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palumbi</span>  <span class="cit-name-given-names">SR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">P.</span></span> <span class="cit-pub-date">2017</span> <span class="cit-article-title">Predicting responses to contemporary environmental change using evolutionary response architectures</span>. <abbr class="cit-jnl-abbrev">Am. Nat</abbr>. <span class="cit-vol">189</span>, <span class="cit-fpage">463</span>–<span class="cit-lpage">473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1086/691233</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat%26rft.volume%253D189%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1086%252F691233%26rft_id%253Dinfo%253Apmid%252F28410032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/691233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28410032&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.8" data-doi="10.1046/j.1365-294X.2003.01788.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">MW</span></span>. <span class="cit-pub-date">2003</span> <span class="cit-article-title">Different genes underlie adaptive melanism in different populations of rock pocket mice</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. <span class="cit-vol">12</span>, <span class="cit-fpage">1185</span>–<span class="cit-lpage">94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1046/j.1365-294X.2003.01788.x</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHoekstra%26rft.auinit1%253DH.%2BE.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D1185%26rft.epage%253D1194%26rft.atitle%253DDifferent%2Bgenes%2Bunderlie%2Badaptive%2Bmelanism%2Bin%2Bdifferent%2Bpopulations%2Bof%2Brock%2Bpocket%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2003.01788.x%26rft_id%253Dinfo%253Apmid%252F12694282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2003.01788.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12694282&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182261500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.9" data-doi="10.1073/pnas.0911042107"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenblum</span>  <span class="cit-name-given-names">EB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Römpler</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schöneberg</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>. <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular and functional basis of phenotypic convergence in white lizards at White Sands</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">107</span>, <span class="cit-fpage">2113</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0911042107</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0911042107%26rft_id%253Dinfo%253Apmid%252F20080544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzUvMjExMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzA5Lzg1MTc2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">10.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-851766v2.10" data-doi="10.1073/pnas.1202111109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dobler</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalla</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagschal</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Agrawal</span>  <span class="cit-name-given-names">AA</span></span>. <span class="cit-pub-date">2012</span> <span class="cit-article-title">Community-wide convergent evolution in insect adaptation to toxic cardenolides by substitutions in the Na,K-ATPase</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">109</span>, <span class="cit-fpage">13040</span>–<span class="cit-lpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1202111109</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1202111109%26rft_id%253Dinfo%253Apmid%252F22826239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMyLzEzMDQwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">11.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-851766v2.11" data-doi="10.1093/molbev/msn218"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">CC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rompler</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boettger</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schoneberg</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>. <span class="cit-pub-date">2008</span> <span class="cit-article-title">The genetic basis of phenotypic convergence in beach mice: similar pigment patterns but different genes</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">26</span>, <span class="cit-fpage">35</span>–<span class="cit-lpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msn218</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn218%26rft_id%253Dinfo%253Apmid%252F18832078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msn218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18832078&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261681900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.12" data-doi="10.1371/journal.pbio.2001627"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Marques</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">JS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">FC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Palma</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kingsley</span>  <span class="cit-name-given-names">DM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reimchen</span>  <span class="cit-name-given-names">TE</span></span>. <span class="cit-pub-date">2017</span> <span class="cit-article-title">Convergent evolution of SWS2 opsin facilitates adaptive radiation of threespine stickleback into different light environments</span>. <abbr class="cit-jnl-abbrev">PLOS Biol</abbr>. <span class="cit-vol">15</span>, <span class="cit-fpage">e2001627</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pbio.2001627</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BBiol%26rft.volume%253D15%26rft.spage%253De2001627%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.2001627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.2001627&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.13" data-doi="10.1038/sj.hdy.6800600"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krenz</span>  <span class="cit-name-given-names">JG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">MW</span></span>. <span class="cit-pub-date">2005</span> <span class="cit-article-title">Local adaptation in the rock pocket mouse (Chaetodipus intermedius): Natural selection and phylogenetic history of populations</span>. <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr>. <span class="cit-vol">94</span>, <span class="cit-fpage">217</span>–<span class="cit-lpage">228</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/sj.hdy.6800600</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHoekstra%26rft.auinit1%253DH.%2BE.%26rft.volume%253D94%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D228%26rft.atitle%253DLocal%2Badaptation%2Bin%2Bthe%2Brock%2Bpocket%2Bmouse%2B%2528Chaetodipus%2Bintermedius%2529%253A%2Bnatural%2Bselection%2Band%2Bphylogenetic%2Bhistory%2Bof%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800600%26rft_id%253Dinfo%253Apmid%252F15523507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800600&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15523507&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226577300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.14" data-doi="10.1073/pnas.1900870116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kreiner</span>  <span class="cit-name-given-names">JM</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span> <span class="cit-article-title">Multiple modes of convergent adaptation in the spread of glyphosate-resistant <em>Amaranthus tuberculatus</em></span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>., 201900870.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1900870116</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1900870116%26rft_id%253Dinfo%253Apmid%252F31570613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzQyLzIxMDc2IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.15" data-doi="10.1534/genetics.110.119594"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">PL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span> <span class="cit-pub-date">2010</span> <span class="cit-article-title">Parallel adaptation: One or many waves of advance of an advantageous allele?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>, <span class="cit-fpage">647</span>–<span class="cit-lpage">668</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.110.119594</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119594%26rft_id%253Dinfo%253Apmid%252F20660645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNjQ3IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">1973</span> <span class="cit-article-title">Gene flow and selection in a cline</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">75</span>, <span class="cit-fpage">733</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D75%26rft.issue%253D4%26rft.spage%253D733%26rft.epage%253D756%26rft.atitle%253DGENE%2BFLOW%2BAND%2BSELECTION%2BIN%2BA%2BCLINE%26rft_id%253Dinfo%253Apmid%252F4778791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzUvNC83MzMiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8wOS84NTE3NjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.17" data-doi="10.1073/pnas.0431157100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">D’Agostino</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-pub-date">2003</span> <span class="cit-article-title">The genetic basis of adaptive melanism in pocket mice</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">100</span>, <span class="cit-fpage">5268</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0431157100</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0431157100%26rft_id%253Dinfo%253Apmid%252F12704245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzkvNTI2OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzA5Lzg1MTc2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.18" data-doi="10.1038/s41437-019-0257-4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">RB</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span> <span class="cit-article-title">The population genetics of crypsis in vertebrates: recent insights from mice, hares, and lizards</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr>. <span class="cit-vol">124</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/s41437-019-0257-4</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41437-019-0257-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41437-019-0257-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.19" data-doi="10.1554/03-418"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drumm</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">MW</span></span>. <span class="cit-pub-date">2004</span> <span class="cit-article-title">Ecological genetics of adaptive color polymorphism in pocket mice: Geographic variation in selected and neutral genes</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>. <span class="cit-vol">58</span>, <span class="cit-fpage">1329</span>–<span class="cit-lpage">1341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.0014-3820.2004.tb01711.x</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHoekstra%26rft.auinit1%253DH.%2BE.%26rft.volume%253D58%26rft.issue%253D6%26rft.spage%253D1329%26rft.epage%253D1341%26rft.atitle%253DEcological%2Bgenetics%2Bof%2Badaptive%2Bcolor%2Bpolymorphism%2Bin%2Bpocket%2Bmice%253A%2Bgeographic%2Bvariation%2Bin%2Bselected%2Band%2Bneutral%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F03-418%26rft_id%253Dinfo%253Apmid%252F15266981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/03-418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15266981&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222513900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-851766v2.20" data-doi="10.1093/molbev/msy004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pfeifer</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">VC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Linnen</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">JD</span></span>. <span class="cit-pub-date">2018</span> <span class="cit-article-title">The evolutionary history of Nebraska deer mice: local adaptation in the face of strong gene flow</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">35</span>, <span class="cit-fpage">792</span>–<span class="cit-lpage">806</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msy004</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft.volume%253D35%26rft.spage%253D792%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy004%26rft_id%253Dinfo%253Apmid%252F29346646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29346646&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.21" data-doi="10.1126/science.aav3824"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">RDH</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span> <span class="cit-article-title">Linking a mutation to survival in wild mice</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">363</span>, <span class="cit-fpage">499</span>–<span class="cit-lpage">504</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aav3824</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBarrett%26rft.auinit1%253DR.%2BD.%2BH.%26rft.volume%253D363%26rft.issue%253D6426%26rft.spage%253D499%26rft.epage%253D504%26rft.atitle%253DLinking%2Ba%2Bmutation%2Bto%2Bsurvival%2Bin%2Bwild%2Bmice%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aav3824%26rft_id%253Dinfo%253Apmid%252F30705186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjMvNjQyNi80OTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8wOS84NTE3NjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.22" data-doi="10.1146/annurev.es.12.110181.000531"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">JRG</span></span>. <span class="cit-pub-date">1981</span> <span class="cit-article-title">Adaptation and evolution in Heliconius: a defense of NeoDarwinism</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst</abbr>. <span class="cit-vol">12</span>, <span class="cit-fpage">99</span>–<span class="cit-lpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev.es.12.110181.000531</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DTurner%26rft.auinit1%253DJ.%26rft.volume%253D12%26rft.spage%253D99%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.12.110181.000531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.es.12.110181.000531&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1981MQ93600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">JBS</span></span>. <span class="cit-pub-date">1924</span> <span class="cit-article-title">A mathematical theory of natural and artificial selection (Part I)</span>. <abbr class="cit-jnl-abbrev">Trans. Cambridge Philos. Soc</abbr>. <span class="cit-vol">23</span>, <span class="cit-fpage">19</span>–<span class="cit-lpage">41</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BCambridge%2BPhilos.%2BSoc%26rft.volume%253D23%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.24" data-doi="10.1534/genetics.105.044065"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teshima</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2006</span> <span class="cit-article-title">Directional positive selection on an allele of arbitrary dominance</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>, <span class="cit-fpage">713</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.105.044065</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044065%26rft_id%253Dinfo%253Apmid%252F16219788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvNzEzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Betancourt</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-pub-date">2001</span> <span class="cit-article-title">Haldane’s sieve and adaptation from the standing genetic variation</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>, <span class="cit-fpage">875</span>–<span class="cit-lpage">884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D157%26rft.issue%253D2%26rft.spage%253D875%26rft.epage%253D884%26rft.atitle%253DHaldane%2527s%2BSieve%2Band%2BAdaptation%2BFrom%2Bthe%2BStanding%2BGenetic%2BVariation%26rft_id%253Dinfo%253Apmid%252F11157004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU3LzIvODc1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.26" data-doi="10.1073/pnas.1222724110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">LS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zimova</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oyler</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Running</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Abatzoglou</span>  <span class="cit-name-given-names">JT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lukacs</span>  <span class="cit-name-given-names">PM</span></span>. <span class="cit-pub-date">2013</span> <span class="cit-article-title">Camouflage mismatch in seasonal coat color due to decreased snow duration</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">110</span>, <span class="cit-fpage">7360</span>–<span class="cit-lpage">7365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1222724110</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1222724110%26rft_id%253Dinfo%253Apmid%252F23589881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE4LzczNjAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMS8wOS84NTE3NjYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.27" data-doi="10.1111/brv.12405"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zimova</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hackländer</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melo-Ferreira</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alves</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">LS</span></span>. <span class="cit-pub-date">2018</span> <span class="cit-article-title">Function and underlying mechanisms of seasonal colour moulting in mammals and birds: what keeps them changing in a warming world?</span> <abbr class="cit-jnl-abbrev">Biol. Rev</abbr>. <span class="cit-vol">93</span>, <span class="cit-fpage">1478</span>–<span class="cit-lpage">1498</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/brv.12405</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BRev%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fbrv.12405%26rft_id%253Dinfo%253Apmid%252F29504224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/brv.12405&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29504224&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.28" data-doi="10.1126/science.aan8097"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">LS</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span> <span class="cit-article-title">Winter color polymorphisms identify global hot spots for evolutionary rescue from climate change</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">359</span>, <span class="cit-fpage">1033</span>–<span class="cit-lpage">1036</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aan8097</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aan8097%26rft_id%253Dinfo%253Apmid%252F29449510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTkvNjM3OS8xMDMzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.29" data-doi="10.1126/science.aar5273"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">MR</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2018</span> <span class="cit-article-title">Adaptive introgression underlies polymorphic seasonal camouflage in snowshoe hares</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">360</span>, <span class="cit-fpage">1355</span>–<span class="cit-lpage">1358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.aar5273</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJones%26rft.auinit1%253DM.%2BR.%26rft.volume%253D360%26rft.issue%253D6395%26rft.spage%253D1355%26rft.epage%253D1358%26rft.atitle%253DAdaptive%2Bintrogression%2Bunderlies%2Bpolymorphic%2Bseasonal%2Bcamouflage%2Bin%2Bsnowshoe%2Bhares%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aar5273%26rft_id%253Dinfo%253Apmid%252F29930138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNjAvNjM5NS8xMzU1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.30" data-doi="10.1111/ele.12568"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zimova</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">LS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nowak</span>  <span class="cit-name-given-names">JJ</span></span>. <span class="cit-pub-date">2016</span> <span class="cit-article-title">High fitness costs of climate change-induced camouflage mismatch</span>. <abbr class="cit-jnl-abbrev">Ecol. Lett</abbr>. <span class="cit-vol">19</span>, <span class="cit-fpage">299</span>–<span class="cit-lpage">307</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/ele.12568</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett%26rft.volume%253D19%26rft.spage%253D299%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12568%26rft_id%253Dinfo%253Apmid%252F26799459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12568&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26799459&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.31" data-doi="10.1175/2008JCLI2665.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">RD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mote</span>  <span class="cit-name-given-names">PW</span></span>. <span class="cit-pub-date">2009</span> <span class="cit-article-title">The response of northern hemisphere snow cover to a changing climate</span>. <abbr class="cit-jnl-abbrev">J. Clim</abbr>. <span class="cit-vol">22</span>, <span class="cit-fpage">2124</span>–<span class="cit-lpage">2145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1175/2008JCLI2665.1</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClim%26rft.volume%253D22%26rft.spage%253D2124%26rft_id%253Dinfo%253Adoi%252F10.1175%252F2008JCLI2665.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/2008JCLI2665.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000266002800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.32" data-doi="10.1175/JCLI3850.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dettinger</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cayan</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knowles</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dettinger</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cayan</span>  <span class="cit-name-given-names">DR</span></span>. <span class="cit-pub-date">2006</span> <span class="cit-article-title">Trends in snowfall versus rainfall in the western United States</span>. <abbr class="cit-jnl-abbrev">J. Clim</abbr>. <span class="cit-vol">19</span>, <span class="cit-fpage">4545</span>–<span class="cit-lpage">4559</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1175/JCLI3850.1</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClimate%26rft.stitle%253DJournal%2Bof%2BClimate%26rft.volume%253D19%26rft.issue%253D18%26rft.spage%253D4545%26rft.epage%253D4559%26rft.atitle%253DTrends%2Bin%2Bsnowfall%2Bversus%2Brainfall%2Bin%2Bthe%2BWestern%2BUnited%2BStates%26rft_id%253Dinfo%253Adoi%252F10.1175%252FJCLI3850.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/JCLI3850.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2008002717&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000240987000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.33" data-doi="10.1139/z83-305"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nagorsen</span>  <span class="cit-name-given-names">DW</span></span>. <span class="cit-pub-date">1983</span> <span class="cit-article-title">Winter pelage colour in snowshoe hares (<em>Lepus americanus</em>) from the Pacific Northwest</span>. <abbr class="cit-jnl-abbrev">Can. J. Zool</abbr>. <span class="cit-vol">61</span>, <span class="cit-fpage">2313</span>–<span class="cit-lpage">2318</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1139/z83-305</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BZool%26rft.volume%253D61%26rft.spage%253D2313%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fz83-305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/z83-305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1983RR31100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.34" data-doi="10.1101/847616"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">LS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Good</span>  <span class="cit-name-given-names">JM</span></span>. <span class="cit-pub-date">2019</span> <span class="cit-article-title">The origin and spread of locally adaptive seasonal camouflage in snowshoe hares</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/847616</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F847616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI4NDc2MTZ2MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzA5Lzg1MTc2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.35" data-doi="10.1139/cjz-2016-0165"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gigliotti</span>  <span class="cit-name-given-names">LC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diefenbach</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sheriff</span>  <span class="cit-name-given-names">MJ</span></span>. <span class="cit-pub-date">2017</span> <span class="cit-article-title">Geographic variation in winter adaptations of snowshoe hares (<em>Lepus americanus</em>)</span>. <abbr class="cit-jnl-abbrev">Can. J. Zool</abbr>. <span class="cit-vol">95</span>, <span class="cit-fpage">539</span>–<span class="cit-lpage">545</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1139/cjz-2016-0165</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BZool%26rft.volume%253D95%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fcjz-2016-0165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/cjz-2016-0165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.36" data-doi="10.1111/mec.12790"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melo-Ferreira</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alves</span>  <span class="cit-name-given-names">PC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">LS</span></span>. <span class="cit-pub-date">2014</span> <span class="cit-article-title">Conservation implications of the evolutionary history and genetic diversity hotspots of the snowshoe hare</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. <span class="cit-vol">23</span>, <span class="cit-fpage">2929</span>–<span class="cit-lpage">2942</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/mec.12790</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol%26rft.volume%253D23%26rft.spage%253D2929%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12790%26rft_id%253Dinfo%253Apmid%252F24814937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12790&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24814937&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.37" data-doi="10.1111/mec.12886"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Melo-Ferreira</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Seixas</span>  <span class="cit-name-given-names">FA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">LS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alves</span>  <span class="cit-name-given-names">PC</span></span>. <span class="cit-pub-date">2014</span> <span class="cit-article-title">The hidden history of the snowshoe hare, <em>Lepus americanus</em>: extensive mitochondrial DNA introgression inferred from multilocus genetic variation</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. <span class="cit-vol">23</span>, <span class="cit-fpage">4617</span>–<span class="cit-lpage">4630</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/mec.12886</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol%26rft.volume%253D23%26rft.spage%253D4617%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12886%26rft_id%253Dinfo%253Apmid%252F25113393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12886&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25113393&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.38" data-doi="10.1101/gr.128124.111"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rohland</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">22</span>, <span class="cit-fpage">939</span>–<span class="cit-lpage">946</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.128124.111</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.128124.111%26rft_id%253Dinfo%253Apmid%252F22267522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzUvOTM5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.39" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span> <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btu170</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.40" data-doi="10.1093/bioinformatics/btr507"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Magoc</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-pub-date">2011</span> <span class="cit-article-title">FLASH: fast length adjustment of short reads to improve genome assemblies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">2957</span>–<span class="cit-lpage">2963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btr507</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr507%26rft_id%253Dinfo%253Apmid%252F21903629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903629&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span> <span class="cit-pub-date">2013</span> <span class="cit-article-title">Aligning sequence reads, clone sequences and assembly contigs with</span> <abbr class="cit-jnl-abbrev">BWA-MEM</abbr>.<span class="cit-pub-id-sep cit-pub-id-arxiv-sep"> </span><span class="cit-pub-id cit-pub-id-arxiv"><span class="cit-pub-id-scheme-arxiv">arxiv:</span>1303.3997</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.42" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">20</span>, <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.107524.110</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzA5Lzg1MTc2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.43" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>, <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btr330</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.44" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span> <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btu033</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.45" data-doi="10.1371/journal.pcbi.1003537"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouckaert</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heled</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kühnert</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C-H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">MA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">AJ</span></span>. <span class="cit-pub-date">2014</span> <span class="cit-article-title">BEAST 2: a software platform for bayesian evolutionary analysis</span>. <abbr class="cit-jnl-abbrev">PLoS Comput. Biol</abbr>. <span class="cit-vol">10</span>, <span class="cit-fpage">e1003537</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pcbi.1003537</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput.%2BBiol%26rft.volume%253D10%26rft.spage%253D1003537e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003537%26rft_id%253Dinfo%253Apmid%252F24722319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003537&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24722319&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.46" data-doi="10.1080/10635150490445715"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Matthee</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Van Vuuren</span>  <span class="cit-name-given-names">BJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">TJ</span></span>. <span class="cit-pub-date">2004</span> <span class="cit-article-title">A molecular supermatrix of the rabbits and hares (Leporidae) allows for the identification of five intercontinental exchanges during the Miocene</span>. <abbr class="cit-jnl-abbrev">Syst. Biol</abbr>. <span class="cit-vol">53</span>, <span class="cit-fpage">433</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1080/10635150490445715</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150490445715%26rft_id%253Dinfo%253Apmid%252F15503672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150490445715&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2007011348&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=15503672&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222351000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.47" data-doi="10.1186/s12859-018-2129-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rabiee</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sayyari</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mirarab</span>  <span class="cit-name-given-names">S.</span></span> <span class="cit-pub-date">2018</span> <span class="cit-article-title">ASTRAL-III: Polynomial time species tree reconstruction from partially resolved gene trees</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1186/s12859-018-2129-y</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D19%26rft.spage%253D153%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2129-y%26rft_id%253Dinfo%253Apmid%252F29745866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-018-2129-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29745866&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.48" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">DH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span> <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr>. <span class="cit-vol">19</span>, <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/gr.094052.109</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzA5Lzg1MTc2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.49" data-doi="10.1038/ng.3464"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petkova</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">2016</span> <span class="cit-article-title">Visualizing spatial population structure with estimated effective migration surfaces</span>. <abbr class="cit-jnl-abbrev">Nat. Genet</abbr>. <span class="cit-vol">48</span>, <span class="cit-fpage">94</span>–<span class="cit-lpage">100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/ng.3464</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D48%26rft.spage%253D94%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3464%26rft_id%253Dinfo%253Apmid%252F26642242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3464&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26642242&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.50" data-doi="10.1371/journal.pone.0015925"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kofler</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">De Maio</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">RV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Futschik</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kosiol</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schlötterer</span>  <span class="cit-name-given-names">C.</span></span> <span class="cit-pub-date">2011</span> <span class="cit-article-title">PoPoolation: a toolbox for population genetic analysis of next generation sequencing data from pooled individuals</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e15925</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0015925</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKofler%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De15925%26rft.epage%253De15925%26rft.atitle%253DPoPoolation%253A%2Ba%2Btoolbox%2Bfor%2Bpopulation%2Bgenetic%2Banalysis%2Bof%2Bnext%2Bgeneration%2Bsequencing%2Bdata%2Bfrom%2Bpooled%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015925%26rft_id%253Dinfo%253Apmid%252F21253599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21253599&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.51" data-doi="10.1016/0027-5107(71)90034-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schlager</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dickie</span>  <span class="cit-name-given-names">MM</span></span>. <span class="cit-pub-date">1971</span> <span class="cit-article-title">Natural mutation rates in the house mouse. Estimates for five specific loci and dominant mutations</span>. <abbr class="cit-jnl-abbrev">Mutat. Res</abbr>. <span class="cit-vol">11</span>, <span class="cit-fpage">89</span>–<span class="cit-lpage">96</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/0027-5107(71)90034-0</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutation%2BResearch%26rft.stitle%253DMutation%2BResearch%26rft.aulast%253DSchlager%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D96%26rft.atitle%253DNatural%2Bmutation%2Brates%2Bin%2Bthe%2Bhouse%2Bmouse.%2BEstimates%2Bfor%2Bfive%2Bspecific%2Bloci%2Band%2Bdominant%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252871%252990034-0%26rft_id%253Dinfo%253Apmid%252F5104863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(71)90034-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5104863&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1971I367100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.52" data-doi="10.1038/ncomms15183"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Milholland</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Suh</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vijg</span>  <span class="cit-name-given-names">J.</span></span> <span class="cit-pub-date">2017</span> <span class="cit-article-title">Differences between germline and somatic mutation rates in humans and mice</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>. <span class="cit-vol">8</span>, <span class="cit-fpage">15183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/ncomms15183</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D8%26rft.spage%253D15183%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms15183%26rft_id%253Dinfo%253Apmid%252F28485371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms15183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28485371&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.53" data-doi="10.1093/molbev/mss025"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">FW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melo-Ferreira</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galtier</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gayral</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanco-Aguiar</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Villafuerte</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ferrand</span>  <span class="cit-name-given-names">N.</span></span> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evidence for widespread positive and purifying selection across the European rabbit (<em>Oryctolagus cuniculus</em>) genome</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">29</span>, <span class="cit-fpage">1837</span>–<span class="cit-lpage">1849</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/mss025</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss025%26rft_id%253Dinfo%253Apmid%252F22319161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22319161&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305409000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.54" data-doi="10.2307/1383263"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gillis</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krebs</span>  <span class="cit-name-given-names">CJ</span></span>. <span class="cit-pub-date">1999</span> <span class="cit-article-title">Natal dispersal of snowshoe hares during a cyclic population increase</span>. <abbr class="cit-jnl-abbrev">J. Mammal</abbr>. <span class="cit-vol">80</span>, <span class="cit-fpage">933</span>–<span class="cit-lpage">939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.2307/1383263</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMammal%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1383263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1383263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.55" data-doi="10.1101/418657"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">BC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">PW</span></span>. <span class="cit-pub-date">2019</span> <span class="cit-article-title">SLiM 3: Forward genetic simulations beyond the Wright-Fisher model</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">36</span>, <span class="cit-fpage">632</span>–<span class="cit-lpage">637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1101/418657</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft.volume%253D36%26rft.spage%253D632%26rft_id%253Dinfo%253Adoi%252F10.1101%252F418657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1101/418657&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.56" data-doi="10.1214/aoms/1177706791"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span> <span class="cit-pub-date">1957</span> <span class="cit-article-title">Some Problems of Stochastic Processes in Genetics</span>. <abbr class="cit-jnl-abbrev">Ann. Math. Stat</abbr>. <span class="cit-vol">28</span>, <span class="cit-fpage">882</span>–<span class="cit-lpage">901</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1214/aoms/1177706791</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMath.%2BStat%26rft.volume%253D28%26rft.spage%253D882%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faoms%252F1177706791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aoms/1177706791&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.57"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">R Core Team</span>. <span class="cit-pub-date">2018</span> <abbr class="cit-jnl-abbrev">R: A language and environment for statistical computing</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.58" data-doi="10.1111/mec.15067"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">TC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Velotta</span>  <span class="cit-name-given-names">JP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dhawanjewar</span>  <span class="cit-name-given-names">AS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schweizer</span>  <span class="cit-name-given-names">RM</span></span>. <span class="cit-pub-date">2019</span> <span class="cit-article-title">UNVEILing connections between genotype, phenotype, and fitness in natural populations</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. <span class="cit-vol">28</span>, <span class="cit-fpage">1866</span>–<span class="cit-lpage">1876</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/mec.15067</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol%26rft.volume%253D28%26rft.spage%253D1866%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.15067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.15067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.59" data-doi="10.1073/pnas.1910471116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Giska</span>  <span class="cit-name-given-names">I</span></span> <span class="cit-etal">et al.</span> <span class="cit-pub-date">2019</span> <span class="cit-article-title">Introgression drives repeated evolution of winter coat color polymorphism in hares</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1910471116</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1910471116%26rft_id%253Dinfo%253Apmid%252F31712446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE2LzQ4LzI0MTUwIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.60" data-doi="10.1093/molbev/msr072"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuro-o</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">X.</span></span> <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exploring patterns and extent of bias in estimating divergence time from mitochondrial DNA sequence data in a particular lineage: a case study of salamanders (Order Caudata)</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">28</span>, <span class="cit-fpage">2521</span>–<span class="cit-lpage">2535</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/molbev/msr072</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr072%26rft_id%253Dinfo%253Apmid%252F21422243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21422243&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294552700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.61" data-doi="10.1126/science.1200684"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Manceau</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingues</span>  <span class="cit-name-given-names">VS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mallarino</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>. <span class="cit-pub-date">2011</span> <span class="cit-article-title">The developmental role of <em>Agouti</em> in color pattern evolution</span>. <abbr class="cit-jnl-abbrev">Science</abbr>. <span class="cit-vol">331</span>, <span class="cit-fpage">1062</span>–<span class="cit-lpage">1065</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1126/science.1200684</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DManceau%26rft.auinit1%253DM.%26rft.volume%253D331%26rft.issue%253D6020%26rft.spage%253D1062%26rft.epage%253D1065%26rft.atitle%253DThe%2BDevelopmental%2BRole%2Bof%2BAgouti%2Bin%2BColor%2BPattern%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1200684%26rft_id%253Dinfo%253Apmid%252F21350176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzEvNjAyMC8xMDYyIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDEvMDkvODUxNzY2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.62" data-doi="10.1111/mec.13385"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pfeifer</span>  <span class="cit-name-given-names">SP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Settles</span>  <span class="cit-name-given-names">ML</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">SS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hardwick</span>  <span class="cit-name-given-names">KM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ormond</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">VC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenblum</span>  <span class="cit-name-given-names">EB</span></span>. <span class="cit-pub-date">2016</span> <span class="cit-article-title">The population genomics of rapid adaptation: disentangling signatures of selection and demography in white sands lizards</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol</abbr>. <span class="cit-vol">25</span>, <span class="cit-fpage">306</span>–<span class="cit-lpage">323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/mec.13385</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol%26rft.volume%253D25%26rft.spage%253D306%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13385&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.63" data-doi="10.1038/nm1334"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Protas</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hersey</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kochanek</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilkens</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">WR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zon</span>  <span class="cit-name-given-names">LI</span></span>, <span class="cit-auth"><span class="cit-name-surname">Borowsky</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tabin</span>  <span class="cit-name-given-names">CJ</span></span>. <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic analysis of cavefish reveals molecular convergence in the evolution of albinism</span>. <abbr class="cit-jnl-abbrev">Nat. Genet</abbr>. <span class="cit-vol">38</span>, <span class="cit-fpage">107</span>–<span class="cit-lpage">111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/ng1700</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DProtas%26rft.auinit1%253DM.%2BE.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D107%26rft.epage%253D111%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bcavefish%2Breveals%2Bmolecular%2Bconvergence%2Bin%2Bthe%2Bevolution%2Bof%2Balbinism.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1334%26rft_id%253Dinfo%253Apmid%252F16341223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16341223&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234227200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.64" data-doi="10.1371/journal.pone.0006435"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kingsley</span>  <span class="cit-name-given-names">EP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manceau</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wiley</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>. <span class="cit-pub-date">2009</span> <span class="cit-article-title">Melanism in <em>Peromyscus</em> is caused by independent mutations in <em>Agouti</em></span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">e6435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0006435</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKingsley%26rft.auinit1%253DE.%2BP.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De6435%26rft.epage%253De6435%26rft.atitle%253DMelanism%2Bin%2Bperomyscus%2Bis%2Bcaused%2Bby%2Bindependent%2Bmutations%2Bin%2Bagouti.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0006435%26rft_id%253Dinfo%253Apmid%252F19649329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0006435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19649329&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2020%2F01%2F09%2F851766.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-851766v2.65" data-doi="10.1073/pnas.0431157100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">MW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">HE</span></span>, <span class="cit-auth"><span class="cit-name-surname">D’Agostino</span>  <span class="cit-name-given-names">SL</span></span>. <span class="cit-pub-date">2003</span> <span class="cit-article-title">The genetic basis of adaptive melanism in pocket mice</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>. <span class="cit-vol">100</span>, <span class="cit-fpage">5268</span>–<span class="cit-lpage">73</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> (</span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.0431157100</span>)</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0431157100%26rft_id%253Dinfo%253Apmid%252F12704245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzkvNTI2OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAxLzA5Lzg1MTc2Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/851766v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F851766v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/736645v2" title="Marked Neurospora crassa strains for competition experiments and Bayesian methods for fitness estimates" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 09, 2020.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/851766v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/851766v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/early/2020/01/09/851766.external-links" target="_self" class="link-icon"><i class="icon-file"></i> <span class="title">Data/Code</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/851766v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/851766v2" class="active" data-icon-position="" data-hide-link-title="0">Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/1097947" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-m4pzaLuDbY3K6RjADqIe6F1o_CLro-l1QP3JxnThq3o" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1097947" id="node1097947--2" data-pisa="biorxiv;851766v2" data-pisa-master="biorxiv;851766" data-seqnum="1097947" data-apath="/biorxiv/early/2020/01/09/851766.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Matthew R.</span> <span class="nlm-surname">Jones</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">L. Scott</span> <span class="nlm-surname">Mills</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jeffrey D.</span> <span class="nlm-surname">Jensen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jeffrey M.</span> <span class="nlm-surname">Good</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">851766; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/851766 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/851766v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/851766v2&amp;title=Convergent%20evolution%20of%20seasonal%20camouflage%20in%20response%20to%20reduced%20snow%20cover%20across%20the%20snowshoe%20hare%20range" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/851766v2&amp;title=Convergent%20evolution%20of%20seasonal%20camouflage%20in%20response%20to%20reduced%20snow%20cover%20across%20the%20snowshoe%20hare%20range" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/851766v2&amp;text=Convergent%20evolution%20of%20seasonal%20camouflage%20in%20response%20to%20reduced%20snow%20cover%20across%20the%20snowshoe%20hare%20range" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/851766v2&amp;title=Convergent%20evolution%20of%20seasonal%20camouflage%20in%20response%20to%20reduced%20snow%20cover%20across%20the%20snowshoe%20hare%20range" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/851766v2&amp;t=Convergent%20evolution%20of%20seasonal%20camouflage%20in%20response%20to%20reduced%20snow%20cover%20across%20the%20snowshoe%20hare%20range" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/851766v2&amp;title=Convergent%20evolution%20of%20seasonal%20camouflage%20in%20response%20to%20reduced%20snow%20cover%20across%20the%20snowshoe%20hare%20range" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/851766v2&amp;title=Convergent%20evolution%20of%20seasonal%20camouflage%20in%20response%20to%20reduced%20snow%20cover%20across%20the%20snowshoe%20hare%20range" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="1097947" id="node1097947--3" data-pisa="biorxiv;851766v2" data-pisa-master="biorxiv;851766" data-seqnum="1097947" data-apath="/biorxiv/early/2020/01/09/851766.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Convergent evolution of seasonal camouflage in response to reduced snow cover across the snowshoe hare range</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Matthew R.</span> <span class="nlm-surname">Jones</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">L. Scott</span> <span class="nlm-surname">Mills</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Jeffrey D.</span> <span class="nlm-surname">Jensen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Jeffrey M.</span> <span class="nlm-surname">Good</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">851766; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/851766 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Convergent%20evolution%20of%20seasonal%20camouflage%20in%20response%20to%20reduced%20snow%20cover%20across%20the%20snowshoe%20hare%20range&amp;rft.volume&amp;rft.issue&amp;rft.date=2020-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Jones%2C+Matthew+R.&amp;rft.au=Mills%2C+L.+Scott&amp;rft.au=Jensen%2C+Jeffrey+D.&amp;rft.au=Good%2C+Jeffrey+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/1097947/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/1097947/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/1097947/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/1097947/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/1097947/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/1097947/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/1097947/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/1097947/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/1097947/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/1097947/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/1097947/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/1097947/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/851766v2&amp;count=horizontal&amp;via=&amp;text=Convergent%20evolution%20of%20seasonal%20camouflage%20in%20response%20to%20reduced%20snow%20cover%20across%20the%20snowshoe%20hare%20range&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/851766v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/851766v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/851766v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
