<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/08/08/726240.atom" />
<meta name="HW.pisa" content="biorxiv;726240v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint" />
<meta name="DC.Identifier" content="10.1101/726240" />
<meta name="DC.Date" content="2019-08-08" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Understanding the genomic basis of adaptation to different abiotic environments is important for understanding organismal responses to current short-term environmental fluctuations. Using functional and comparative genomics approaches, we here investigated whether genomic adaptation to a set of environmental parameters is contingent across vertebrate genomes or, alternatively, contains an element of evolutionary constraint that would be evident through recurrent involvement of specific subsets of genes and functions in adaptation to similar environments. We first identified 200 genes with signatures of selection from transcriptomes of 24 species of lacertid lizards with known adaptations in preferred temperature, correlated with thermal environment experienced by these lizards in their range. In order to discern genes adapting to climate from other selective factors, we then performed a meta-analysis of 1100 genes with signatures of selection obtained from-omics studies in vertebrate species adapted to different abiotic environments. We found that this gene set formed a tightly connected interactome which was to 23% enriched in predicted functions of adaptation to climate and to 18% involved in organismal stress response. We found a much higher degree of recurrent use of identical genes (43.6%) and functional similarity than expected by chance, and no clear division between genes used in ectotherm and endotherm physiological strategies. 171 out of 200 genes of Lacertidae were part of this network, indicating that a comparative genomic approach can help to disentangle genes functionally related to adaptation to different abiotic environments from other selective factors. These results furthermore highlight an important role of genomic constraint in adaptation to the abiotic environment, and narrows the set of candidate markers to be used in future research on environmental adaptability related to climate change.

Significance Statement / Nontechnical summary While the convergent evolution of phenotypes in similar environments is a well-studied phenomenon, the genomic basis of such common phenotypes and physiologies is still enigmatic. The prevalent notion is that re-use of the same genes to adapt to similar environments in different species is about as likely as winning the lottery – but organismal systems are also, to some extent, comparable between different species such as man and fruit fly through shared genes and gene functions. In this paper, we test whether constraint or contingency is more prevalent in genomic adaptation of vertebrates to aspects of their abiotic environment. We find evidence for strong functional constraint and stress responsiveness of the genes involved, which might help understand how currently experienced stress under changing climates may result in future adaptation." />
<meta name="DC.Contributor" content="Katharina C. Wollenberg Valero" />
<meta name="DC.Contributor" content="Joan Garcia-Porta" />
<meta name="DC.Contributor" content="Iker Irisarri" />
<meta name="DC.Contributor" content="Lauric Feugere" />
<meta name="DC.Contributor" content="Adam Bates" />
<meta name="DC.Contributor" content="Sebastian Kirchhof" />
<meta name="DC.Contributor" content="Olga Jovanović Glavaš" />
<meta name="DC.Contributor" content="Panayiotis Pafilis" />
<meta name="DC.Contributor" content="Sabrina F. Samuel" />
<meta name="DC.Contributor" content="Johannes Müller" />
<meta name="DC.Contributor" content="Miguel Vences" />
<meta name="DC.Contributor" content="Alexander P. Turner" />
<meta name="DC.Contributor" content="Pedro Beltran-Alvarez" />
<meta name="DC.Contributor" content="Kenneth B. Storey" />
<meta name="article:published_time" content="2019-08-08" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Understanding the genomic basis of adaptation to different abiotic environments is important for understanding organismal responses to current short-term environmental fluctuations. Using functional and comparative genomics approaches, we here investigated whether genomic adaptation to a set of environmental parameters is contingent across vertebrate genomes or, alternatively, contains an element of evolutionary constraint that would be evident through recurrent involvement of specific subsets of genes and functions in adaptation to similar environments. We first identified 200 genes with signatures of selection from transcriptomes of 24 species of lacertid lizards with known adaptations in preferred temperature, correlated with thermal environment experienced by these lizards in their range. In order to discern genes adapting to climate from other selective factors, we then performed a meta-analysis of 1100 genes with signatures of selection obtained from-omics studies in vertebrate species adapted to different abiotic environments. We found that this gene set formed a tightly connected interactome which was to 23% enriched in predicted functions of adaptation to climate and to 18% involved in organismal stress response. We found a much higher degree of recurrent use of identical genes (43.6%) and functional similarity than expected by chance, and no clear division between genes used in ectotherm and endotherm physiological strategies. 171 out of 200 genes of Lacertidae were part of this network, indicating that a comparative genomic approach can help to disentangle genes functionally related to adaptation to different abiotic environments from other selective factors. These results furthermore highlight an important role of genomic constraint in adaptation to the abiotic environment, and narrows the set of candidate markers to be used in future research on environmental adaptability related to climate change.&lt;/p&gt;&lt;h3&gt;Significance Statement / Nontechnical summary&lt;/h3&gt;
&lt;p&gt;While the convergent evolution of phenotypes in similar environments is a well-studied phenomenon, the genomic basis of such common phenotypes and physiologies is still enigmatic. The prevalent notion is that re-use of the same genes to adapt to similar environments in different species is about as likely as winning the lottery – but organismal systems are also, to some extent, comparable between different species such as man and fruit fly through shared genes and gene functions. In this paper, we test whether constraint or contingency is more prevalent in genomic adaptation of vertebrates to aspects of their abiotic environment. We find evidence for strong functional constraint and stress responsiveness of the genes involved, which might help understand how currently experienced stress under changing climates may result in future adaptation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;726240v1" />
<meta name="citation_id" content="726240v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/726240v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/726240v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/726240v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240.full.pdf" />
<meta name="citation_doi" content="10.1101/726240" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="726240" />
<meta name="citation_author" content="Katharina C. Wollenberg Valero" />
<meta name="citation_author_institution" content="Department of Biological and Marine Sciences, University of Hull" />
<meta name="citation_author_email" content="k.wollenberg-valero@hull.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8858-1804" />
<meta name="citation_author" content="Joan Garcia-Porta" />
<meta name="citation_author_institution" content="Department of Biology, Washington University in Saint Louis" />
<meta name="citation_author" content="Iker Irisarri" />
<meta name="citation_author_institution" content="Department of Organismal Biology, Uppsala University" />
<meta name="citation_author_institution" content="Dept. Biodiversity and Evolutionary Biology, Museo Nacional de Ciencias Naturales" />
<meta name="citation_author" content="Lauric Feugere" />
<meta name="citation_author_institution" content="Department of Biological and Marine Sciences, University of Hull" />
<meta name="citation_author" content="Adam Bates" />
<meta name="citation_author_institution" content="Department of Biological and Marine Sciences, University of Hull" />
<meta name="citation_author" content="Sebastian Kirchhof" />
<meta name="citation_author_institution" content="Museum für Naturkunde, Leibniz Institute for Evolution and Biodiversity Science" />
<meta name="citation_author_institution" content="New York University Abu Dhabi" />
<meta name="citation_author" content="Olga Jovanović Glavaš" />
<meta name="citation_author_institution" content="Department of Biology, University of Osijek" />
<meta name="citation_author" content="Panayiotis Pafilis" />
<meta name="citation_author_institution" content="Section of Zoology and Marine Biology, Department of Biology, National and Kapodistrian University of Athens" />
<meta name="citation_author" content="Sabrina F. Samuel" />
<meta name="citation_author_institution" content="Department of Biomedical Sciences, University of Hull" />
<meta name="citation_author" content="Johannes Müller" />
<meta name="citation_author_institution" content="Museum für Naturkunde, Leibniz Institute for Evolution and Biodiversity Science" />
<meta name="citation_author" content="Miguel Vences" />
<meta name="citation_author_institution" content="Zoological Institute, Braunschweig University of Technology" />
<meta name="citation_author" content="Alexander P. Turner" />
<meta name="citation_author_institution" content="Department of Biological and Marine Sciences, University of Hull" />
<meta name="citation_author" content="Pedro Beltran-Alvarez" />
<meta name="citation_author_institution" content="Department of Biomedical Sciences, University of Hull" />
<meta name="citation_author" content="Kenneth B. Storey" />
<meta name="citation_author_institution" content="Department of Biology, Carleton University" />
<meta name="citation_reference" content="Futuyma DJ (2010) Evolutionary constraint and ecological consequences. Evolution 64(7):1865–1884." />
<meta name="citation_reference" content="Turelli M (1985) Effects of pleiotropy on predictions concerning mutation-selection balance for polygenic traits. Genetics 111(1):165–195." />
<meta name="citation_reference" content="Salverda MLM, et al. (2011) Initial mutations direct alternative pathways of protein evolution. PLoS Genet 7(3):e1001321." />
<meta name="citation_reference" content="Blount ZD, Lenski RE, Losos JB (2018) Contingency and determinism in evolution: Replaying life’s tape. Science 362(6415):eaam5979." />
<meta name="citation_reference" content="Therkildsen NO, et al. (2019) Contrasting genomic shifts underlie parallel phenotypic evolution in response to fishing. Science 365(6452):487–490." />
<meta name="citation_reference" content="Pavlicev M, Wagner GP (2012) A model of developmental evolution: selection, pleiotropy and compensation. Trends Ecol Evol 27(6):316–322." />
<meta name="citation_reference" content="Han J-DJ, et al. (2004) Evidence for dynamically organized modularity in the yeast protein-protein interaction network. Nature 430(6995):88–93." />
<meta name="citation_reference" content="Wollenberg Valero KC (2018) Aligning functional network constraint to evolutionary outcomes. bioRxiv:278663." />
<meta name="citation_reference" content="Losos JB (2011) Convergence, adaptation, and constraint. Evolution 65(7):1827–1840." />
<meta name="citation_reference" content="Babonis LS, Martindale MQ (2017) Phylogenetic evidence for the modular evolution of metazoan signalling pathways. Philos Trans R Soc Lond B Biol Sci 372(1713). doi:10.1098/rstb.2015.0477." />
<meta name="citation_reference" content="Porcelli D, Butlin RK, Gaston KJ, Joly D, Snook RR (2015) The environmental genomics of metazoan thermal adaptation. Heredity 114(5):502–514." />
<meta name="citation_reference" content="Rodríguez A, et al. (2017) Genomic and phenotypic signatures of climate adaptation in an Anolis lizard. Ecol Evol 7:6390–6403." />
<meta name="citation_reference" content="Denver RJ (2009) Structural and functional evolution of vertebrate neuroendocrine stress systems. Ann N Y Acad Sci 1163:1–16." />
<meta name="citation_reference" content="Logan CA, Buckley BA (2015) Transcriptomic responses to environmental temperature in eurythermal and stenothermal fishes. J Exp Biol 218(Pt 12):1915–1924." />
<meta name="citation_reference" content="Buttke TM, Sandstrom PA (1994) Oxidative stress as a mediator of apoptosis. Immunol Today 15(1):7–10." />
<meta name="citation_reference" content="Pozhitkov AE, et al. (2017) Tracing the dynamics of gene transcripts after organismal death. Open Biol 7(1). doi:10.1098/rsob.160267." />
<meta name="citation_reference" content="Jensen LJ, et al. (2002) Prediction of human protein function from post-translational modifications and localization features. J Mol Biol 319(5):1257–1265." />
<meta name="citation_reference" content="Qiu Q, et al. (2012) The yak genome and adaptation to life at high altitude. Nat Genet 44(8):946–949." />
<meta name="citation_reference" content="Simonson TS (2015) Altitude Adaptation: A Glimpse Through Various Lenses. High Alt Med Biol 16(2):125–137." />
<meta name="citation_reference" content="Chen L, DeVries AL, Cheng CH (1997) Convergent evolution of antifreeze glycoproteins in Antarctic notothenioid fish and Arctic cod. Proc Natl Acad Sci U S A 94(8):3817–3822." />
<meta name="citation_reference" content="Zhuang X, Yang C, Murphy KR, Cheng C-HC (2019) Molecular mechanism and history of non-sense to sense evolution of antifreeze glycoprotein gene in northern gadids. Proc Natl Acad Sci U S A. doi:10.1073/pnas.1817138116." />
<meta name="citation_reference" content="Feiner N, Rago A, While GM, Uller T (2018) Signatures of selection in embryonic transcriptomes of lizards adapting in parallel to cool climate. Evolution 72(1):67–81." />
<meta name="citation_reference" content="Sanger TJ, Kyrkos J, Lachance DJ, Czesny B, Stroud JT (2018) The effects of thermal stress on the early development of the lizard Anolis sagrei. J Exp Zool A Ecol Integr Physiol. doi:10.1002/jez.2185." />
<meta name="citation_reference" content="Hancock AM, et al. (2008) Adaptations to climate in candidate genes for common metabolic disorders. PLoS Genet 4(2):e32." />
<meta name="citation_reference" content="Diele-Viegas LM, Rocha CFD (2018) Unraveling the influences of Climate Change in Lepidosauria (Reptilia). J Therm Biol. doi:10.1016/j.jtherbio.2018.11.005." />
<meta name="citation_reference" content="Campbell-Staton SC, et al. (2017) Winter storms drive rapid phenotypic, regulatory, and genomic shifts in the green anole lizard. Science 357(6350):495–498." />
<meta name="citation_reference" content="Yang W, Qi Y, Fu J (2014) Exploring the genetic basis of adaptation to high elevations in reptiles: a comparative transcriptome analysis of two toad-headed agamas (genus Phrynocephalus). PLoS One 9(11):e112218." />
<meta name="citation_reference" content="Sun Y-B, et al. (2018) Species groups distributed across elevational gradients reveal convergent and continuous genetic adaptation to high elevations. Proc Natl Acad Sci U S A 115(45):E10634–E10641." />
<meta name="citation_reference" content="Garcia-Porta J, et al. (2019 in press) Environmental temperatures shape thermal physiology, diversification and genome-wide substitution rates in lizards. Nat Commun." />
<meta name="citation_reference" content="Araújo MB, et al. (2013) Heat freezes niche evolution. Ecol Lett 16:1206–1219." />
<meta name="citation_reference" content="Wollenberg Valero KC, et al. (2014) A candidate multimodal functional genetic network for thermal adaptation. PeerJ 2:e578." />
<meta name="citation_reference" content="White CR, Alton LA, Frappell PB (2012) Metabolic cold adaptation in fishes occurs at the level of whole animal, mitochondria and enzyme. Proc Roy Soc B 279(1734):1740–1747." />
<meta name="citation_reference" content="Sokolova I (2018) Mitochondrial adaptations to variable environments and their role in animals’ stress tolerance. Integr Comp Biol. doi:10.1093/icb/icy017." />
<meta name="citation_reference" content="Munoz MM, et al. (2014) Evolutionary stasis and lability in thermal physiology in a group of tropical lizards. Proceedings of the Royal Society of London B: Biological Sciences 281(1778):20132433." />
<meta name="citation_reference" content="Bourgeois Y, Ruggiero RP, Manthey JD, Boissinot S (2019) Recent secondary contacts, linked selection and variable recombination rates shape genomic diversity in the model species Anolis carolinensis. Genome Biol Evol. doi:10.1093/gbe/evz110." />
<meta name="citation_reference" content="Yu G, et al. (2010) GOSemSim: an R package for measuring semantic similarity among GO terms and gene products. Bioinformatics 26(7):976–978." />
<meta name="citation_reference" content="Dalle-Donne I, Rossi R, Milzani A, Di Simplicio P, Colombo R (2001) The actin cytoskeleton response to oxidants: from small heat shock protein phosphorylation to changes in the redox state of actin itself. Free Radic Biol Med 31(12):1624–1632." />
<meta name="citation_reference" content="Klose MK, Robertson RM (2004) Stress-induced thermoprotection of neuromuscular transmission. Integr Comp Biol 44(1):14–20." />
<meta name="citation_reference" content="Pedersen SF, Hoffmann EK, Mills JW (2001) The cytoskeleton and cell volume regulation. Comp Biochem Physiol A Mol Integr Physiol 130(3):385–399." />
<meta name="citation_reference" content="Morin P Jr, Storey KB (2009) Mammalian hibernation: differential gene expression and novel application of epigenetic controls. Int J Dev Biol 53(2–3):433–442." />
<meta name="citation_reference" content="Bond MR, Hanover JA (2015) A little sugar goes a long way: the cell biology of O-GlcNAc. J Cell Biol 208(7):869–880." />
<meta name="citation_reference" content="Niu Y, DesMarais TL, Tong Z, Yao Y, Costa M (2015) Oxidative stress alters global histone modification and DNA methylation. Free Radic Biol Med 82:22–28." />
<meta name="citation_reference" content="Xu H, et al. (2016) Lysine Acetylation and Succinylation in HeLa Cells and their Essential Roles in Response to UV-induced Stress. Sci Rep 6:30212." />
<meta name="citation_reference" content="Eisenberg E, Levanon EY (2013) Human housekeeping genes, revisited. Trends Genet 29(10):569–574." />
<meta name="citation_reference" content="Pancaldi V, et al. (2012) Predicting the fission yeast protein interaction network. G3 2(4):453–467." />
<meta name="citation_reference" content="López-Maury L, Marguerat S, Bähler J (2008) Tuning gene expression to changing environments: from rapid responses to evolutionary adaptation. Nat Rev Genet 9(8):583–593." />
<meta name="citation_reference" content="Feder ME, Hofmann GE (1999) Heat-shock proteins, molecular chaperones, and the stress response: evolutionary and ecological physiology. Annu Rev Physiol 61:243–282." />
<meta name="citation_reference" content="Ortega Z, Mencía A, Pérez-Mellado V (2016) The peak of thermoregulation effectiveness: Thermal biology of the Pyrenean rock lizard, Iberolacerta bonnali (Squamata, Lacertidae). J Therm Biol 56:77–83." />
<meta name="citation_reference" content="Sannolo M, Barroso FM, Carretero MA (2018) Physiological differences in preferred temperatures and evaporative water loss rates in two sympatric lacertid species. Zoology 126:58–64." />
<meta name="citation_reference" content="Carneiro D, Garcia-Munoz E, Zagar A, Pafilis P, Carretero MA (2017) Is ecophysiology congruent with the present-day relictual distribution of a lizard group? Evidence from preferred temperatures and water loss rates. Herpetol J 27(1):47–56." />
<meta name="citation_reference" content="Romero-Diaz C, Breedveld MC, Fitze PS (2017) Climate effects on growth, body condition, and survival depend on the genetic characteristics of the population. Am Nat 190(5):649–662." />
<meta name="citation_reference" content="Somero GN (2010) The physiology of climate change: how potentials for acclimatization and genetic adaptation will determine “winners” and “losers.” J Exp Biol 213(6):912–920." />
<meta name="citation_reference" content="Bay RA, Rose NH, Logan CA, Palumbi SR (2017) Genomic models predict successful coral adaptation if future ocean warming rates are reduced. Sci Adv 3(11):e1701413." />
<meta name="citation_reference" content="Smith MD, et al. (2015) Less is more: an adaptive branch-site random effects model for efficient detection of episodic diversifying selection. Mol Biol Evol 32(5):1342–1353." />
<meta name="citation_reference" content="Kosakovsky Pond SL, Frost SDW, Muse SV (2005) HyPhy: hypothesis testing using phylogenies. Bioinformatics 21(5):676–679." />
<meta name="citation_reference" content="Shannon P, et al. (2003) Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res 13(11):2498–2504." />
<meta name="citation_reference" content="Bindea G, et al. (2009) ClueGO: a Cytoscape plug-in to decipher functionally grouped gene ontology and pathway annotation networks. Bioinformatics 25(8):1091–1093." />
<meta name="citation_reference" content="Bindea G, Galon J, Mlecnik B (2013) CluePedia Cytoscape plugin: pathway insights using integrated experimental and in silico data. Bioinformatics 29(5):661–663." />
<meta name="citation_reference" content="Altenhoff AM, Dessimoz C (2009) Phylogenetic and functional assessment of orthologs inference projects and methods. PLoS Comput Biol 5:e1000262." />
<meta name="citation_reference" content="Hsiao LL, et al. (2001) A compendium of gene expression in normal human tissues. Physiol Genomics 7(2):97–104." />
<meta name="citation_reference" content="Wang JZ, Du Z, Payattakool R, Yu PS, Chen C-F (2007) A new method to measure the semantic similarity of GO terms. Bioinformatics 23(10):1274–1281." />
<meta name="citation_reference" content="Lukk M, et al. (2010) A global map of human gene expression. Nat Biotechnol 28(4):322–324." />
<meta name="citation_reference" content="White RJ, et al. (2017) A high-resolution mRNA expression time course of embryonic development in zebrafish. Elife 6. doi:10.7554/eLife.30860." />
<meta name="citation_reference" content="Yi H, et al. (2010) Gene expression atlas for human embryogenesis. FASEB J 24(9):3341–3350." />
<meta name="citation_reference" content="Lê S, Josse J, Husson F, Others (2008) FactoMineR: an R package for multivariate analysis. J Stat Softw 25(1):1–18." />
<meta name="citation_reference" content="Suzuki R, Shimodaira H (2006) Pvclust: an R package for assessing the uncertainty in hierarchical clustering. Bioinformatics 22(12):1540–1542." />
<meta name="citation_reference" content="Suzuki R, Shimodaira H (2013) pvclust: An R package for hierarchical clustering with p-values. Bioinformatics 22:1–7." />
<meta name="twitter:title" content="Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Understanding the genomic basis of adaptation to different abiotic environments is important for understanding organismal responses to current short-term environmental fluctuations. Using functional and comparative genomics approaches, we here investigated whether genomic adaptation to a set of environmental parameters is contingent across vertebrate genomes or, alternatively, contains an element of evolutionary constraint that would be evident through recurrent involvement of specific subsets of genes and functions in adaptation to similar environments. We first identified 200 genes with signatures of selection from transcriptomes of 24 species of lacertid lizards with known adaptations in preferred temperature, correlated with thermal environment experienced by these lizards in their range. In order to discern genes adapting to climate from other selective factors, we then performed a meta-analysis of 1100 genes with signatures of selection obtained from-omics studies in vertebrate species adapted to different abiotic environments. We found that this gene set formed a tightly connected interactome which was to 23% enriched in predicted functions of adaptation to climate and to 18% involved in organismal stress response. We found a much higher degree of recurrent use of identical genes (43.6%) and functional similarity than expected by chance, and no clear division between genes used in ectotherm and endotherm physiological strategies. 171 out of 200 genes of Lacertidae were part of this network, indicating that a comparative genomic approach can help to disentangle genes functionally related to adaptation to different abiotic environments from other selective factors. These results furthermore highlight an important role of genomic constraint in adaptation to the abiotic environment, and narrows the set of candidate markers to be used in future research on environmental adaptability related to climate change.

Significance Statement / Nontechnical summary While the convergent evolution of phenotypes in similar environments is a well-studied phenomenon, the genomic basis of such common phenotypes and physiologies is still enigmatic. The prevalent notion is that re-use of the same genes to adapt to similar environments in different species is about as likely as winning the lottery – but organismal systems are also, to some extent, comparable between different species such as man and fruit fly through shared genes and gene functions. In this paper, we test whether constraint or contingency is more prevalent in genomic adaptation of vertebrates to aspects of their abiotic environment. We find evidence for strong functional constraint and stress responsiveness of the genes involved, which might help understand how currently experienced stress under changing climates may result in future adaptation." />
<meta name="og-title" property="og:title" content="Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/726240v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Understanding the genomic basis of adaptation to different abiotic environments is important for understanding organismal responses to current short-term environmental fluctuations. Using functional and comparative genomics approaches, we here investigated whether genomic adaptation to a set of environmental parameters is contingent across vertebrate genomes or, alternatively, contains an element of evolutionary constraint that would be evident through recurrent involvement of specific subsets of genes and functions in adaptation to similar environments. We first identified 200 genes with signatures of selection from transcriptomes of 24 species of lacertid lizards with known adaptations in preferred temperature, correlated with thermal environment experienced by these lizards in their range. In order to discern genes adapting to climate from other selective factors, we then performed a meta-analysis of 1100 genes with signatures of selection obtained from-omics studies in vertebrate species adapted to different abiotic environments. We found that this gene set formed a tightly connected interactome which was to 23% enriched in predicted functions of adaptation to climate and to 18% involved in organismal stress response. We found a much higher degree of recurrent use of identical genes (43.6%) and functional similarity than expected by chance, and no clear division between genes used in ectotherm and endotherm physiological strategies. 171 out of 200 genes of Lacertidae were part of this network, indicating that a comparative genomic approach can help to disentangle genes functionally related to adaptation to different abiotic environments from other selective factors. These results furthermore highlight an important role of genomic constraint in adaptation to the abiotic environment, and narrows the set of candidate markers to be used in future research on environmental adaptability related to climate change.

Significance Statement / Nontechnical summary While the convergent evolution of phenotypes in similar environments is a well-studied phenomenon, the genomic basis of such common phenotypes and physiologies is still enigmatic. The prevalent notion is that re-use of the same genes to adapt to similar environments in different species is about as likely as winning the lottery – but organismal systems are also, to some extent, comparable between different species such as man and fruit fly through shared genes and gene functions. In this paper, we test whether constraint or contingency is more prevalent in genomic adaptation of vertebrates to aspects of their abiotic environment. We find evidence for strong functional constraint and stress responsiveness of the genes involved, which might help understand how currently experienced stress under changing climates may result in future adaptation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-08-08" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/726240v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/726240v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/726240v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/726240v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/861192" />
    <title>Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"HB83uTxxTq_zdHwZK1xaGEF0FPi3ayaA_U4jw5h7ERE","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"861192","apath":"\/biorxiv\/early\/2019\/08\/08\/726240.atom","pisa":"biorxiv;726240v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;726240v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/726240v1","title":"Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint","identifier":"node\/861192"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VFPerhNfCcG5cgnsE4yKLbOUZS3PG9P0KUzNxMepQgs3tv3wcrFF0Q+PXIYWmgSmTgvWtw83fGnuNz0d+rUZQ5LR7o2XyVP7G2e1w5sIfa1rWaj50doRifceUlYUZnwfLBuXze8Bumimy5UMYupiY8LQ6flzKaip0YGc\\\\\\\/bmlho0DPOMPN+Fy4gc3JumUBzpyOs6ytr7t+40ggfkahBHQk\\\\\\\/jVT56\\\\\\\/R7htbr7Ew0tOmZrmMbLL4YOAj\\\\\\\/WsUV9pONhVmcVwGvZygTmp6x+1NtgG4zHgJOQRcASGGmQOMl2X3k+kiDDhsckMBnZ0XUSiNxuCZBmbF\\\\\\\/N1l89PfE7MAE6ox7ib4tA86+wTvqDdZ1T6XlWMvtIU7761AgBfVgxZ5YaBg+mre4E2Gpu2uQ\\\\\\\/ixzzCZH37QiGxNgQXf6TNsSXHf3Gjt7NFG9Wxeb+iACtwA7ngzTp\\\\\\\/7v4Gp+krMoJd4VpPzTDnw\\\\\\\/YjA2cNpsRcjSrbRCz+fyAE\\\\\\\/scT\\\\\\\/84vd0qQCTPsYf1Iaxsa1nr12hnxBeAfZCxfH3w3spQZI1ctpjAkLW7aFYMFn\\\\\\\/v5l9DziTznDMlc2ZoCn0bKgnzJRhYMb3CWElt7lw\\\\\\\/ubaj8hOf1tqrwKUZWyxNJYPaFljiehZ8Sx0urCs7IsQDK0Z7McRbyB1yVBEsGUfdwCsuLZ5OYgQ2h3t9MGZqDTBfegn+RdN6rC5caVbLtYpPYO6Y6gKBzbw\\\\\\\/53tqUGPiuoVJ5uRsDqw1j4kShmxoHKt1zrxPfEkxLOXHBmLPYL3pDdrFno9brNOyufBJ5kWHRkLo9yY0e7HELNjGcT5dLCgCoCmGq8UkGq6BwycXK6KWhc1nNjjB5p5KqhECDmmNAzImCWqUFp7j+RyHvc9VvipelpTSXuA8iF6Itko9Hern94XNXiThwb5Zu9T38N2NwTj5so5ZtLUNF2fVVt8RD7bAuGyMBDCkDb2gzjDola+fHoOfxEDHbutKhEErowH0iUbfVjLWX0aBCaylT8xQ7fP1kpKXGlg66W6+vHAihL6JnCpjf17UywNTttUEBsqn2p2BmHYXEzljsjF6AmmKm8nfwLu5T1j4so5MKEw7qgUHuuh+RS\\\\\\\/KMSQWz8stEPSpn800VOP0sc00gdnSX85Bv\\\\\\\/1WkZtJ3890jHYYjq\\\\\\\/UQGaqap0B7eWyjeteiaiMfnyc\\\\\\\/9KWgPlfK2m5caKF6GNXoNEaKPzfX4dqNE5JbTK4vIEtnsN9jSF13UeLoi9ah3Rs6i0ONHVnpQGmf4v9VCMuf0FofHa1o+RJIb9k9HnI3yvbq8BaDs6GEu+XrdtdQ5AZ3kb218xP4DxqHqqE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022nlB0CkeFDgQJ\\\\\\\/NcU3DY2Hg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002283a2d84663c30e13819b61682d1c8c9ae823bc9e0eb91d81694571a8f48ab744\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/726240v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-861192 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/726240v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-q8-ticXDFAuexCcxBkYC8Ftkpg4rSEYSdVMLiXMXZbY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="861192" id="node861192" data-pisa="biorxiv;726240v1" data-pisa-master="biorxiv;726240" data-apath="/biorxiv/early/2019/08/08/726240.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8858-1804" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Katharina C. Wollenberg</span> <span class="nlm-surname">Valero</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joan</span> <span class="nlm-surname">Garcia-Porta</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Iker</span> <span class="nlm-surname">Irisarri</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Lauric</span> <span class="nlm-surname">Feugere</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Adam</span> <span class="nlm-surname">Bates</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sebastian</span> <span class="nlm-surname">Kirchhof</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Olga</span> <span class="nlm-surname">Jovanović Glavaš</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Panayiotis</span> <span class="nlm-surname">Pafilis</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Sabrina F.</span> <span class="nlm-surname">Samuel</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Johannes</span> <span class="nlm-surname">Müller</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Miguel</span> <span class="nlm-surname">Vences</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Alexander P.</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Pedro</span> <span class="nlm-surname">Beltran-Alvarez</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Kenneth B.</span> <span class="nlm-surname">Storey</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/726240 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node861192" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katharina C. Wollenberg Valero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biological and Marine Sciences, University of Hull</span>, Cottingham Road HU6 7RX, Kingston-Upon-Hull, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katharina%2BC.%2BWollenberg%2BValero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Valero%20KC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatharina%2BC.%2BWollenberg%2BValero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8858-1804" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katharina C. Wollenberg Valero</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:k.wollenberg-valero@hull.ac.uk" class="" data-icon-position="" data-hide-link-title="0">k.wollenberg-valero@hull.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Joan Garcia-Porta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, Washington University in Saint Louis</span>, St. Louis, MO 63130, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joan%2BGarcia-Porta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia-Porta%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoan%2BGarcia-Porta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Iker Irisarri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Organismal Biology, Uppsala University</span>, Norbyvägen 18D, 752 36 Uppsala, <span class='nlm-country'>Sweden</span></div><div class='author-affiliation'><span class='nlm-institution'>Dept. Biodiversity and Evolutionary Biology, Museo Nacional de Ciencias Naturales</span>, José Gutiérrez Abascal 2, 28006 Madrid, <span class='nlm-country'>Spain</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iker%2BIrisarri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Irisarri%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIker%2BIrisarri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lauric Feugere </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biological and Marine Sciences, University of Hull</span>, Cottingham Road HU6 7RX, Kingston-Upon-Hull, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauric%2BFeugere%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feugere%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALauric%2BFeugere%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adam Bates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biological and Marine Sciences, University of Hull</span>, Cottingham Road HU6 7RX, Kingston-Upon-Hull, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BBates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bates%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BBates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Sebastian Kirchhof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Museum für Naturkunde, Leibniz Institute for Evolution and Biodiversity Science</span>, Invalidenstr. 43, 10115 Berlin, <span class='nlm-country'>Germany</span></div><div class='author-affiliation'><span class='nlm-institution'>New York University Abu Dhabi</span>, Saadiyat Island Campus, Abu Dhabi, United Arab Emirates</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastian%2BKirchhof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirchhof%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastian%2BKirchhof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Olga Jovanović Glavaš </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, University of Osijek</span>, Cara Hadrijana 8A, Osijek, <span class='nlm-country'>Croatia</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olga%2BJovanovi%C4%87%2BGlava%C5%A1%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jovanovi%C4%87%20Glava%C5%A1%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlga%2BJovanovi%25C4%2587%2BGlava%25C5%25A1%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Panayiotis Pafilis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Section of Zoology and Marine Biology, Department of Biology, National and Kapodistrian University of Athens</span>, Panepistimioupolis, Ilissia, Athens 157-84, <span class='nlm-country'>Greece</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Panayiotis%2BPafilis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pafilis%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APanayiotis%2BPafilis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sabrina F. Samuel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biomedical Sciences, University of Hull</span>, Cottingham Road HU6 7RX, Kingston-Upon-Hull, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sabrina%2BF.%2BSamuel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samuel%20SF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASabrina%2BF.%2BSamuel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Johannes Müller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Museum für Naturkunde, Leibniz Institute for Evolution and Biodiversity Science</span>, Invalidenstr. 43, 10115 Berlin, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BM%C3%BCller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=M%C3%BCller%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohannes%2BM%25C3%25BCller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Miguel Vences </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Zoological Institute, Braunschweig University of Technology</span>, Mendelssohnstr. 4, 38106 Braunschweig, <span class='nlm-country'>Germany</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miguel%2BVences%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vences%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiguel%2BVences%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Alexander P. Turner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biological and Marine Sciences, University of Hull</span>, Cottingham Road HU6 7RX, Kingston-Upon-Hull, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BP.%2BTurner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Turner%20AP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexander%2BP.%2BTurner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Pedro Beltran-Alvarez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biomedical Sciences, University of Hull</span>, Cottingham Road HU6 7RX, Kingston-Upon-Hull, <span class='nlm-country'>United Kingdom</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pedro%2BBeltran-Alvarez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beltran-Alvarez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APedro%2BBeltran-Alvarez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Kenneth B. Storey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, Carleton University</span>, 1125 Colonel By Drive, K1S 5B6 Ottawa, <span class='nlm-country'>Canada</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kenneth%2BB.%2BStorey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Storey%20KB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKenneth%2BB.%2BStorey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/726240v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:861192" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:861192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/726240v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:861192" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:861192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/726240v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:861192" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:861192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/726240v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:861192" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:861192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/726240v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:861192" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:861192/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Understanding the genomic basis of adaptation to different abiotic environments is important for understanding organismal responses to current short-term environmental fluctuations. Using functional and comparative genomics approaches, we here investigated whether genomic adaptation to a set of environmental parameters is contingent across vertebrate genomes or, alternatively, contains an element of evolutionary constraint that would be evident through recurrent involvement of specific subsets of genes and functions in adaptation to similar environments. We first identified 200 genes with signatures of selection from transcriptomes of 24 species of lacertid lizards with known adaptations in preferred temperature, correlated with thermal environment experienced by these lizards in their range. In order to discern genes adapting to climate from other selective factors, we then performed a meta-analysis of 1100 genes with signatures of selection obtained from-omics studies in vertebrate species adapted to different abiotic environments. We found that this gene set formed a tightly connected interactome which was to 23% enriched in predicted functions of adaptation to climate and to 18% involved in organismal stress response. We found a much higher degree of recurrent use of identical genes (43.6%) and functional similarity than expected by chance, and no clear division between genes used in ectotherm and endotherm physiological strategies. 171 out of 200 genes of Lacertidae were part of this network, indicating that a comparative genomic approach can help to disentangle genes functionally related to adaptation to different abiotic environments from other selective factors. These results furthermore highlight an important role of genomic constraint in adaptation to the abiotic environment, and narrows the set of candidate markers to be used in future research on environmental adaptability related to climate change.</p><div id="sec-1" class="subsection"><p id="p-4"><strong>Significance Statement / Nontechnical summary</strong> While the convergent evolution of phenotypes in similar environments is a well-studied phenomenon, the genomic basis of such common phenotypes and physiologies is still enigmatic. The prevalent notion is that re-use of the same genes to adapt to similar environments in different species is about as likely as winning the lottery – but organismal systems are also, to some extent, comparable between different species such as man and fruit fly through shared genes and gene functions. In this paper, we test whether constraint or contingency is more prevalent in genomic adaptation of vertebrates to aspects of their abiotic environment. We find evidence for strong functional constraint and stress responsiveness of the genes involved, which might help understand how currently experienced stress under changing climates may result in future adaptation.</p></div></div><div class="section" id="sec-2"><h2 class="">Introduction</h2><p id="p-5">The sheer number of genes and alleles (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>) within a genome, the fact that many traits are polygenic (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>) and that genomes can evolve via alternative pathways (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>) theoretically provide infinite possibilities for adaptations to evolve. As a result of this, genomes are expected to reflect a high degree of contingency (evolutionary unpredictability (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>)). However, many studies have noted that there are less realized adaptations than expected, which means that there could be constraints at the genome level that could lead to evolutionary determinism (more predictable outcomes (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>)) (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>). One such element of constraint could be explained through the inertia generated by the interaction of genes that perform a common function upon which selection acts (gene functional constraint) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>–<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>). In this paper, we address the question of how prevalent such constraint is at the genomic level, on the example of adaptation to aspects of the abiotic environment.</p><p id="p-6">First, constraint in adaptation to a common selective pressure (e.g., the abiotic environment) could be detected by comparing the similarity of gene functions involved in adaptation across different taxa. Adaptation to different or novel abiotic environments involves changes in traits which, in similar environments, often converge (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). Traits of the phenotype or physiology are based on groups of gene products operating within defined functions (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>). Through common descent, this relationship between specific genes, their functions, and traits may be conserved across different lineages. A high degree of conservation can constrain the number of genes performing a function. As a result, the number of genes that may be modified by selection on that function to adapt across these different lineages may be constrained (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1</a>) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). For example, adaptation to climate in vertebrate ectotherms such as squamate reptiles, is directed through a specific set of gene functions with evidence that similar genes are involved (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). In vertebrate ectotherms, these functions contain many genes involved in the conserved stress response (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>), e.g., heat shock proteins (HSPs) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>). Other candidates for environmental adaptation are genes active within a cancer microenvironment which likewise represents aspects of a stressful environment such as hypoxia and acidic pH. Furthermore, cellular stress such as oxidative stress often ends in apoptosis (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>) and consequently, genes related to the cellular pathways of apoptosis, stress and inflammation (dubbed “Zombie genes” (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>)) could also be candidates for environmental adaptation. Functionally similar genes can also be recognized through similar epiproteomic signals, such as protein post-translational modifications (PTMs (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>)).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Contingency versus gene functional constraint at the gene product level. Organisms use gene products (shapes) to perform functions (arrows) at the molecular level. Expectation of constraint: Since they share a common ancestor, it is likely that these molecular functional systems likewise share common ancestry. If a specific function (such as temperature homeostasis) is under strong balancing selection, it is unlikely that over time genes with this function get exapted to other functions, or that new genes get recruited into this function (shown here as black symbols). In different lineages (a-c), the molecular function will therefore be performed by the same gene products. If adaptation then occurs due to a novel environment (e.g., a different environmental temperature), it is likely that the adaptation (in the form of changed gene sequence or expression level) will be constrained to the set of genes which has evolved to perform that function. If several lineages are compared, we expect to find genes being recruited multiple times to adapt to this change in thermal environment, i.e., recurrently. Expectation of contingency: In contrast, if the function of temperature homeostasis is under relaxed constraint (shown as grey gene symbols), the system will be modified through evolution to recruit and lose different gene products. A novel abiotic environment will lead to adaptation in different EAGs in different lineages. Consequently, the degree of genes recurrently involved in adaptation will be very low and more comparable to the likelihood of drawing genes from the genome by chance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1891019830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Contingency versus gene functional constraint at the gene product level.&lt;/span&gt; Organisms use gene products (shapes) to perform functions (arrows) at the molecular level. &lt;strong&gt;Expectation of constraint:&lt;/strong&gt; Since they share a common ancestor, it is likely that these molecular functional systems likewise share common ancestry. If a specific function (such as temperature homeostasis) is under strong balancing selection, it is unlikely that over time genes with this function get exapted to other functions, or that new genes get recruited into this function (shown here as black symbols). In different lineages (a-c), the molecular function will therefore be performed by the same gene products. If adaptation then occurs due to a novel environment (e.g., a different environmental temperature), it is likely that the adaptation (in the form of changed gene sequence or expression level) will be constrained to the set of genes which has evolved to perform that function. If several lineages are compared, we expect to find genes being recruited multiple times to adapt to this change in thermal environment, i.e., recurrently. &lt;strong&gt;Expectation of contingency:&lt;/strong&gt; In contrast, if the function of temperature homeostasis is under relaxed constraint (shown as grey gene symbols), the system will be modified through evolution to recruit and lose different gene products. A novel abiotic environment will lead to adaptation in different EAGs in different lineages. Consequently, the degree of genes recurrently involved in adaptation will be very low and more comparable to the likelihood of drawing genes from the genome by chance.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F1.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F1.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">Contingency versus gene functional constraint at the gene product level.</span><p id="p-7" class="first-child">Organisms use gene products (shapes) to perform functions (arrows) at the molecular level. <strong>Expectation of constraint:</strong> Since they share a common ancestor, it is likely that these molecular functional systems likewise share common ancestry. If a specific function (such as temperature homeostasis) is under strong balancing selection, it is unlikely that over time genes with this function get exapted to other functions, or that new genes get recruited into this function (shown here as black symbols). In different lineages (a-c), the molecular function will therefore be performed by the same gene products. If adaptation then occurs due to a novel environment (e.g., a different environmental temperature), it is likely that the adaptation (in the form of changed gene sequence or expression level) will be constrained to the set of genes which has evolved to perform that function. If several lineages are compared, we expect to find genes being recruited multiple times to adapt to this change in thermal environment, i.e., recurrently. <strong>Expectation of contingency:</strong> In contrast, if the function of temperature homeostasis is under relaxed constraint (shown as grey gene symbols), the system will be modified through evolution to recruit and lose different gene products. A novel abiotic environment will lead to adaptation in different EAGs in different lineages. Consequently, the degree of genes recurrently involved in adaptation will be very low and more comparable to the likelihood of drawing genes from the genome by chance.</p><div class="sb-div caption-clear"></div></div></div><p id="p-8">Second, evidence for constraint can be estimated through the number of identical genes that are involved in the functions related to specific aspects of adaptation. Under gene functional constraint, we expect that genes cannot easily be recruited or lost from their function, and thus we would expect to find adaptations to similar selective pressure to be concentrated in a subset of genes performing these functions across different organisms. Under strong constraint, or with just a small number of genes performing these relevant functions, this could even lead to the re-use of the same genes in adaptation to similar environments. We here call this process recurrence, or recurrent evolution in cases where the same genes are recruited or modified for adaptation in non-sister lineages. One of the best-known examples of recurrent evolution based on modification of identical genes in different species is altitude adaptation in the Himalayas. High-altitude yak, Tibetan humans and their dogs all show adaptations in the ADAM17 gene related to the physiological function of hypoxia tolerance (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>) compared to their low-altitude relatives. An example for recurrent evolution independent of common descent are the antifreeze glycoprotein genes in Antarctic notothenioid fish and Arctic cod. Antarctic fishes re-purposed the glycoprotein sequence from a different ancestral function (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>), whereas the same functional gene sequence in Arctic cod evolved <em>de novo</em> from a non-coding genomic region (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>). Alternatively, through modification, different genes could perform these functions in different organisms, which would indicate the constraint on genes contributing to adaptation is more relaxed. Complete relaxation of the gene functional constraint would mean that any gene could perform any function in different lineages and genes adapting to different environments across lineages will be different across lineages, and will have less functional similarity (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>).</p><p id="p-9">Third, constraint can be identified through comparing organisms with obvious differences in physiological strategy. Strong constraint will keep gene function and gene identity similar even across divergent physiologies and evolutionary lineages, whereas relaxed constraint will lead to more pronounced genetic differences across these groups. Especially in those vertebrates where development in the majority of taxa occurs outside the parental body, climate can be a strong selective force for both cold (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>) and hot (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>) climate stress and -adaptation. While endothermic vertebrates (including humans) have been well studied from an -omics perspective (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>), the genomic basis of environmental adaptation in vertebrate ectotherms is generally understudied (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>), with only a few studies incorporating genome-wide scans for environmental adaptation-relevant genes to date (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>–<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>).</p><p id="p-10">The first aim of this contribution is therefore to better understand the genomic basis of environmental adaptation in vertebrate ectotherms in one such group, squamate reptiles belonging to the Western Palearctic lacertid lizards (family Lacertidae). We use an RNA sequencing data set we previously generated for phylogenomic analysis of this group (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>) to newly identify genes with signatures of selection across 24 species representing the lacertid diversity. Informed by the physiological adaptations in preferred body temperature (T<sub>pref</sub>) with bioclimate identified by us in a previous study (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>), we hypothesize that at least some of these genes will harbor genomic adaptations to cold across the climatic gradient they inhabit (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>).</p><p id="p-11">To identify the extent of constraint in adaptation to the abiotic environment across vertebrates, we then compare the functions of genes putatively involved in environmental adaptation (in the following called environmental adaptation genes, EAG) across vertebrates, mined from 25 -omics studies of adaptation and 75 -omics studies of stress. Under the premise of gene functional constraint, we expect that EAGs identified in different lineages are tightly functionally connected, fall within predefined functions for climate adaptation (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>) and are expressed or modified in cellular functions related to stressful conditions (here: environmental stress, apoptosis, cancer). We also expect functional similarity of both lacertid and other vertebrate EAGs to be higher than expected by chance. Lastly, we aim to understand the extent of genomic recurrence across endotherm and ectotherm vertebrates including Lacertidae. We quantify the genes that are identified in more than one species, and compare their frequency of occurrence against a random draw simulation representing the absence of constraint as null hypothesis. Here, we expect to find both functional similarity as well as evidence for constraint in the modification of EAGs across different taxa.</p></div><div class="section" id="sec-3"><h2 class="">Results</h2><p id="p-12">Among 5,498 ortholog genes compiled from the transcriptomes of 24 lacertid taxa (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>), we identified 200 genes to have evolved under episodic positive diversifying selection. The estimated median ω (dN/dS ratio) per branch for these 200 genes was plotted onto a maximum likelihood phylogeny (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2a</a>). Branches splitting from basal nodes, except <em>Podarcis</em>, showed the highest median values of ω. The majority of 10&gt;ω&gt;1 genes were located on the (long) terminal branches. In contrast, ω ≥ 10 genes were concentrated both along the phylogeny backbone and on a set of terminal branches. Based on previous literature meta-analyses and experimental confirmation of genes adapting to climate, a set of candidate functions related to climate adaptation was defined (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>). We tested whether newly identified genes were also functionally enriched in these functions. The Lacertid genes were significantly enriched (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2b</a>) in several previously defined candidate functions potentially related to climate adaptation (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>), and in cytoskeletal processes as additional, not predicted functional category (Tables S1-S2).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic and functional properties of lacertid and vertebrate EAGs. (a) Visualization of median selection coefficient (&#x3C9;) on RNAseq maximum likelihood phylogram of 24 lacertid species for 200 genes under episodic positive diversifying selection. Genes with 10&gt;&#x3C9;&gt;1 and &#x3C9;&gt;10 are shown along the branches as circles. (b) Function network of significantly enriched functions for these genes in Lacertidae (circles - biological process; diamond - molecular function). (c) Functional genomic network for environmental adaptation generated from 902 genes across endotherm and ectotherm vertebrates (blue &#x2013;genes that recurrently adapted to environmental parameters in different species; ochre &#x2013;genes that are stress responsive, maroon &#x2013;genes that evolved both recurrently and are stress responsive). (d) Graph of functional categories of significantly overrepresented ClueGo Gene Ontology groups, for the 902 functionally connected environmental adaptation genes, categorized by candidate functions (Cf) and additional functions. Numbers in labels refer to numbers of Gene Ontology terms within each category." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1891019830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Phylogenetic and functional properties of lacertid and vertebrate EAGs.&lt;/span&gt; (a) Visualization of median selection coefficient (&#x3C9;) on RNAseq maximum likelihood phylogram of 24 lacertid species for 200 genes under episodic positive diversifying selection. Genes with 10&gt;&#x3C9;&gt;1 and &#x3C9;&gt;10 are shown along the branches as circles. (b) Function network of significantly enriched functions for these genes in Lacertidae (circles - biological process; diamond - molecular function). (c) Functional genomic network for environmental adaptation generated from 902 genes across endotherm and ectotherm vertebrates (blue &#x2013;genes that recurrently adapted to environmental parameters in different species; ochre &#x2013;genes that are stress responsive, maroon &#x2013;genes that evolved both recurrently and are stress responsive). (d) Graph of functional categories of significantly overrepresented ClueGo Gene Ontology groups, for the 902 functionally connected environmental adaptation genes, categorized by candidate functions (Cf) and additional functions. Numbers in labels refer to numbers of Gene Ontology terms within each category.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F2.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F2.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Phylogenetic and functional properties of lacertid and vertebrate EAGs.</span><p id="p-13" class="first-child">(a) Visualization of median selection coefficient (ω) on RNAseq maximum likelihood phylogram of 24 lacertid species for 200 genes under episodic positive diversifying selection. Genes with 10&gt;ω&gt;1 and ω&gt;10 are shown along the branches as circles. (b) Function network of significantly enriched functions for these genes in Lacertidae (circles - biological process; diamond - molecular function). (c) Functional genomic network for environmental adaptation generated from 902 genes across endotherm and ectotherm vertebrates (blue –genes that recurrently adapted to environmental parameters in different species; ochre –genes that are stress responsive, maroon –genes that evolved both recurrently and are stress responsive). (d) Graph of functional categories of significantly overrepresented ClueGo Gene Ontology groups, for the 902 functionally connected environmental adaptation genes, categorized by candidate functions (Cf) and additional functions. Numbers in labels refer to numbers of Gene Ontology terms within each category.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">A total of 1100 putative EAGs across ectotherm and endotherm vertebrates (12 ectotherm and 17 endotherm species) were compiled from -omics studies of adaptations to different or novel abiotic environments. These genes resulted from statistical comparisons among sister clades in phylogenies, among populations, or across generational timepoints. Including the 200 Lacertidae genes, the set of 1100 genes were involved in adaptation to different environmental conditions comprised of thermal differences in latitude, altitude or season, sulfidic water in streams, osmotic differences, desiccation, hypoxia, or wet-dry environmental gradients (Tables S3). Of the 1100 putative EAGs, 902 or 83% were part of a large, tightly connected protein-protein interaction (PPI) network (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2c</a>). Among these, 480 genes or 43.6% aligned to our definition of recurrent evolution as they were involved in environmental adaptation in multiple, not closely related vertebrate clades (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2c</a>, Table S4). The Gene Ontology (GO) terms for this network covered all except one <em>a priori</em> predicted function potentially related to climate adaptation, which was muscle use and -development (Table S1, S5). Additional significantly enriched GO terms within this network were related to the cytoskeleton, apoptosis, and localization and transport (Table S1, S5, <a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2d</a>). The latter function encompassed 68 related GO terms and 459 genes (45%). 86% or 171 of the 200 selected genes in Lacertidae were also part of this functional network and 14 of these adapted recurrently with other taxa (Fig. S1, Table S1). As outlined above, we consider it likely that at least these 902 genes (171 of Lacertidae) connected to each other by climate adaptation-relevant functions and evidence for recurrence represent true EAGs, while this is less likely for the 29 Lacertidae genes not connected to this network. To investigate the extent of constraint through gene re-use in environmental adaptation, we tested whether the number of 480 recurrently evolving genes out of 1,843 genes across all data sets (some of the 1100 individual genes were identified in more than two species) was higher than expected by chance. In 100,000 simulations, we found that in order to obtain an identical number of recurrent genes solely by chance, on average 4736.7 (± 93.4) genes would have to be drawn from a hypothetical genome of 20,000 genes. This value is significantly higher than the empirical value obtained here (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a>, Z= −30.98, p ≤ 0.00001).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of lacertid genes under selection and vertebrate recurrently evolving EAG to randomized and simulated data. (a) Number of genes that was drawn in this study to find 480 recurrent genes (red bar) is significantly lower than the number of genes obtained in N=100,000 simulations of random draws from a hypothetical genome of 20,000 genes to yield 480 recurrent genes, as expected under no constraint (histogram, ***). (b) Comparisons of functional (semantic) similarity of GO terms between genes under selection in Lacertidae and random networks (A-F), and between 902 vertebrate EAG and 20 sets of 902 randomized genes (G, H). Boxes show medians, 25-75 quartiles, outliers, and jittered raw data. Transparent bands indicate modelled &#xB1;95% prediction intervals. The plot shows that the 200 genes under selection (A) have the highest functional similarity. Their semantic similarity is higher than that of 29 genes under selection not part of the network (B), of 20 random replicates of each 200 genes not under selection (C,**); of 20 random replicates of 200 genes drawn from 5,498 annotated ortholog genes with incomplete taxon sampling obtained from RNAseq (D, ***), of 20 replicates of 200 random human genes (E, ***), but not higher than 10 replicates of 451 human housekeeping genes (F); 902 vertebrate EAGs part of a functional genomic network (G) have significantly higher functional similarity than 20 random samples of 902 genes (H, ***). p values are denoted as * &#x2264; 0.05; ** &#x2264; 0.01; *** &#x2264; 0.001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1891019830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Comparison of lacertid genes under selection and vertebrate recurrently evolving EAG to randomized and simulated data.&lt;/span&gt; (a) Number of genes that was drawn in this study to find 480 recurrent genes (red bar) is significantly lower than the number of genes obtained in N=100,000 simulations of random draws from a hypothetical genome of 20,000 genes to yield 480 recurrent genes, as expected under no constraint (histogram, ***). &lt;strong&gt;(b)&lt;/strong&gt; Comparisons of functional (semantic) similarity of GO terms between genes under selection in Lacertidae and random networks (A-F), and between 902 vertebrate EAG and 20 sets of 902 randomized genes (G, H). Boxes show medians, 25-75 quartiles, outliers, and jittered raw data. Transparent bands indicate modelled &#xB1;95% prediction intervals. The plot shows that the 200 genes under selection (A) have the highest functional similarity. Their semantic similarity is higher than that of 29 genes under selection not part of the network (B), of 20 random replicates of each 200 genes not under selection (C,**); of 20 random replicates of 200 genes drawn from 5,498 annotated ortholog genes with incomplete taxon sampling obtained from RNAseq (D, ***), of 20 replicates of 200 random human genes (E, ***), but not higher than 10 replicates of 451 human housekeeping genes (F); 902 vertebrate EAGs part of a functional genomic network (G) have significantly higher functional similarity than 20 random samples of 902 genes (H, ***). p values are denoted as * &#x2264; 0.05; ** &#x2264; 0.01; *** &#x2264; 0.001.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F3.medium.gif" width="440" height="226"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F3.medium.gif" width="440" height="226"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Comparison of lacertid genes under selection and vertebrate recurrently evolving EAG to randomized and simulated data.</span><p id="p-15" class="first-child">(a) Number of genes that was drawn in this study to find 480 recurrent genes (red bar) is significantly lower than the number of genes obtained in N=100,000 simulations of random draws from a hypothetical genome of 20,000 genes to yield 480 recurrent genes, as expected under no constraint (histogram, ***). <strong>(b)</strong> Comparisons of functional (semantic) similarity of GO terms between genes under selection in Lacertidae and random networks (A-F), and between 902 vertebrate EAG and 20 sets of 902 randomized genes (G, H). Boxes show medians, 25-75 quartiles, outliers, and jittered raw data. Transparent bands indicate modelled ±95% prediction intervals. The plot shows that the 200 genes under selection (A) have the highest functional similarity. Their semantic similarity is higher than that of 29 genes under selection not part of the network (B), of 20 random replicates of each 200 genes not under selection (C,**); of 20 random replicates of 200 genes drawn from 5,498 annotated ortholog genes with incomplete taxon sampling obtained from RNAseq (D, ***), of 20 replicates of 200 random human genes (E, ***), but not higher than 10 replicates of 451 human housekeeping genes (F); 902 vertebrate EAGs part of a functional genomic network (G) have significantly higher functional similarity than 20 random samples of 902 genes (H, ***). p values are denoted as * ≤ 0.05; ** ≤ 0.01; *** ≤ 0.001.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">We then tested whether the functions and functional connectedness of vertebrate EAGs were more similar than expected by chance or properties of the sequenced transcriptomes. The vertebrate EAG network had significantly more interactions between the genes than expected if they were randomly drawn from the genome, with a PPI enrichment p-value of 2e<sup>−13</sup>. It also had significantly higher functional similarity, measured via the semantic similarity of its GO terms than random networks of comparable size (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3b</a>, Z = 4.57, p = 0.00001). When likely lacertid EAGs were separately subjected to GO enrichment analysis, two additional candidate functions for climate adaptation were recovered (cell signalling and muscle-related processes; Table S1). For the 29 Lacertidae genes under selection that were not part of the network (and less likely constituting EAGs), no functional GO term enrichment could be detected (Table S1). Lacertid EAGs also had higher semantic similarity of GO terms than those 29 not part of the network, and higher semantic similarity than randomized replicates of all other comparison groups (29 genes not under selection Z = 2.5, p = 0.012, all other annotated orthologs in transcriptomes Z = 5, p ≤ 0.00001, randomly drawn genes Z = 3.6, p = 0.0003, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3b</a>), but not compared to human “housekeeping” genes. (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3b</a>, Z = 1, p = 0.159).</p><p id="p-17">We subsequently tested whether putative EAGs, and those with evidence for recurrent adaptation, could be grouped by a priori defined candidate functions for climate adaptation, or related to stress (evidence for stress responsiveness, cancer and apoptosis), as well as PTMs. Evidence for human gene PTMs was present in 19% of EAGs, and 21% genes furthermore had the function “gene regulation - protein modification” (Table S6). EAGs overall had significantly more PTMs than randomly drawn genes (Fig. S2, Mann-Whitney W = 19572, p ≤ 2.2e<sup>−16</sup>), and significantly less PTMs than randomly drawn human housekeeping genes (W = 8599, p = 2.57e<sup>−15</sup>). These results remain significant after comparison with a randomized distribution of W (EAG to random genes Z = 443.726, p ≤ 0.001; EAG to housekeeping genes Z = 192.44, p ≤ 0.001, Fig. S2). 198 EAGs or 18% had “stress response” as a significant GO term (Table S4). Furthermore, we found empirical evidence in the literature for 329 (29.9%) genes being differentially expressed in response to abiotic environmental stress (including cold stress, heat stress, hypoxia stress, and others including hibernation, desiccation and detoxification, Table S6, the complete data is provided in Table S7). Of these, 125 (38.0%) were upregulated and 74 (22.5%) were downregulated (Fig. S3). The top three candidate functions for climate adaptation represented by these genes were “cellular component biogenesis”, “morphogenesis”, and “response to oxidative stress / stress response (incl. thermal stress)” (Table S8-S9). All of these genes were part of the functional network. Several of them were heat shock proteins; however, the two most recorded genes responding to all analyzed aspects of environmental stress across different species were the less-studied BAG3 and AHSG. AHSG furthermore responded to all four groups of stressors, together with PKLR and SQRDL (Fig. S4a). With regards to cellular pathways active in cancer as possible EAG candidate function, we found that recurrent EAGs have lower expression values in both cancer and healthy tissues (Fig. S4b), whilst ratios of gene expression between cancer and reference conditions did not differ (Table S10). Five genes of the data set were, however, involved in more than 80 cancer types. (Fig. S5, Table S11). To visually align EAGs to candidate functions, six functional dimensions of EAGs were extracted by Multiple Correspondence Analysis. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a> shows the dendrogram obtained by <em>pvclust</em> following 100,000 bootstrap replicates. Over ten hierarchical functional clusters were supported with bootstrap values &gt;95, with functional dimensions not clearly partitioning genes into clusters that could be identified as clearly ectotherm or endotherm (for more details see Supplementary Results).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of Multiple Correspondence Analysis (MCA), represented as a dendrogram, to scale EAGs by different functional and evolutionary aspects. Statistical support was obtained via multiscale bootstrap (N=100,000 replicates (67)). 11 functional clusters were obtained with combined bootstrap support (alpha&gt;0.9, red numbers on nodes), and standard error of estimate " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1891019830" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;span xmlns=&quot;http://www.w3.org/1999/xhtml&quot; class=&quot;caption-title&quot;&gt;Results of Multiple Correspondence Analysis (MCA), represented as a dendrogram, to scale EAGs by different functional and evolutionary aspects.&lt;/span&gt; Statistical support was obtained via multiscale bootstrap (N=100,000 replicates (67)). 11 functional clusters were obtained with combined bootstrap support (alpha&gt;0.9, red numbers on nodes), and standard error of estimate &lt;0.01 (not shown).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F4.medium.gif" width="290" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F4.medium.gif" width="290" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/08/08/726240/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">Results of Multiple Correspondence Analysis (MCA), represented as a dendrogram, to scale EAGs by different functional and evolutionary aspects.</span><p id="p-18" class="first-child">Statistical support was obtained via multiscale bootstrap (N=100,000 replicates (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>)). 11 functional clusters were obtained with combined bootstrap support (alpha&gt;0.9, red numbers on nodes), and standard error of estimate &lt;0.01 (not shown).</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-4"><h2 class="">Discussion</h2><p id="p-19">In this contribution, we investigated the relative importance of genomic constraint vs contingency in shaping adaptation to the abiotic environment at the genome level. We expected that the existence of functional genomic constraint in adaptation could be proven through higher than random levels of identity and of functional similarity among genes that have adapted to similar selective regimes among different vertebrate taxa and physiological strategies. We here found evidence that this type of constraint plays an important role in generating environmental adaptation.</p><p id="p-20">We found that EAGs of different species form a tight network with more functional connections among the genes than expected by chance. Across the different species and environmental factors considered, almost half of all genes in this functional network (43.6%) evolved recurrently, which was significantly higher than simulated in the absence of any constraint. Such events of recurrent evolution span a wide range of both ectotherm and endotherm species, as also a range of environmental factors. For example, 14 EAGs of Lacertidae that are related to a bioclimatic gradient have been independently recruited for environmental adaptation in other vertebrates. Ectotherms use these to adapt to desiccation and anoxia (killifish <em>Austrofundulus limnaeus)</em>, altitude (Himalayan <em>Nanorana</em> frogs, Himalayan <em>Phrynocephalus</em> lizards), and endotherms use them to adapt to latitude (Yakutian horse, woolly mammoth, minke whale, polar bear), deserts (camel and dromedary), and altitude (Himalayan marmot). Aspects of environmental adaptation were not clearly separated between processes in vertebrate endotherms and ectotherms, which indicates that the genomic mechanisms and functions of adaptation to abiotic parameters seem to be constrained even among these two, very different physiological strategies. This constitutes another indicator for the presence of genomic constraint in environmental adaptation. For example, Lacertidae shared in total five genes under selection with the extinct woolly mammoth (ETFA, MRPS31, PFAS, PHKB, and SENP5).</p><p id="p-21">In lacertid lizards, terminal branches with very high values of episodic positive selection were identified mainly in species inhabiting cooler, more seasonal and partially montane environments such as <em>Podarcis muralis</em> and <em>Zootoca vivipara</em>. In contrast, they were absent in hot-arid adapted species such as <em>Mesalina olivieri</em> and species living in tropical environments (<em>Takydromus sexlineatus, Holaspis guentheri</em>). Adaptation in genes linked to mitochondrial cell respiration and oxidative stress response (proteasome component genes and the chaperone HSP90B1), could facilitate adaptation to cold environments via influencing the metabolic rate (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>). This corroborates previous findings that lower preferred temperatures are more likely to evolve than higher thermal preferences in lizards inhabiting climatic gradients (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>). Lacertidae genes under selection but not part of the vertebrate functional network did not have any functional enrichment, and could have evolved in response to selection pressures other than abiotic climate-related factors. Our comparative genomic approach thus may help disentangle environmental adaptation from covarying factors such as locomotor performance or reproduction (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>).</p><p id="p-22">The results with regards to gene function are in alignment with the hypothesis of constraint as well. High functional connectedness, and comparatively higher values of GO semantic similarity of the EAGs analyzed in our study indicated that they were drawn from a nonrandom set of very similar functions (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>). Many EAGs fell within predicted candidate functions for climate adaptation (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">31</a>). Involvement in cytoskeletal and apoptotic processes, as well as localization and transport emerged as novel environmental adaptation-related function. Both apoptosis and transport-related genes have previously been associated with death-related processes, during which transport genes may become activated attempting to restore homeostasis (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>). The cytoskeleton is remodelled through stress and involved in HSP phosphorylation (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>) which in turn protect cytoskeletal integrity (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>). Stress furthermore induces changes in cytoskeleton-regulated cell volume (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>).</p><p id="p-23">PTMs have previously been related to regulation of torpor as environment-induced metabolic process (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>). Surprisingly, we found that specific PTMs were present in almost all functional clusters associated with environmental adaptation. Genes that were involved in latitudinal adaptation were related to various types of PTMs such as serine O-GlcNAc (serine-β-linked N-acetylglucosamine modification) which are associated with transcription, metabolism, apoptosis, organelle biogenesis, and transport (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>) and disease including metabolic syndrome and cancer (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>). 89% of all EAGs were involved in at least one human cancer, and a subset of them had cancer involvement as predominant function (e.g., FCGBP and SYT11 adapting to latitude in polar bear and woolly mammoth), which reinforces a relationship between abiotic environmental adaptation and homeostasis disturbance via disease (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">16</a>). “Zombie genes” evolve recurrently, and are associated with altitude adaptation. These genes are both activated in dying cells, and involve localization and transport processes, which supports these functions as emerging cellular candidate processes for environmental adaptation. Adaptation to hypoxia and desiccation is furthermore associated with the PTM lysine trimethylation (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>). Lastly, lysine succinylation was linked to genes involved in environmental adaptation in lacertids and other ectotherms, possibly related to UV radiation (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>).</p><p id="p-24">Our transcriptome data set on Lacertidae genes could be biased for genes with high expression (that are more likely recovered from all species in an RNA-based alignment), and further environmental adaptations could exist in other genes that we did not analyze; however, across all vertebrates, EAGs had higher expression values, higher numbers of PTM modification sites, and higher functional similarity than other partitions of the transcriptomes, but random sets of housekeeping genes had similar properties.EAGs have a higher expression level than expected by chance, which is similar to housekeeping genes, but are not identical with them. Only eight housekeeping genes were identified in our data set. A further 277 genes (or 25%) were, however, included in 3804 “wider definition” housekeeping genes, which are expressed at a more baseline or mid-level across all tissues and conditions (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>). Genes with frequent expression in several tissues could expand the range of life stages and situations where natural selection can modify allele frequencies in response to abiotic environmental selection. However, as the dendrogram shows (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>), genes that are only expressed during embryonic development are aligned with thermal adaptation across latitude, elevation, and recurrent adaptation, pinpointing early development as a vulnerable stage for environmental selection (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>).</p><p id="p-25">Whilst 18% of environmental adaptation genes were involved in the response to stress in experimental studies, we did not find evidence for an alignment between specific axes of environmental adaptation to specific types of stress response such as desert adaptation to heat stress response. This result supports the idea that functional genetic networks are partitioned into different types of functions covered by distinct gene sets, which then interlink with others for information exchange (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>), and supports the necessity of a generality of the organismal stress response to ensure viability in the face of multiple stressors (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>). As expected, HSPs were differentially expressed under all stressors (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>), and so were other genes functionally involved in the heat shock response. We identified little-studied EAGs, that ubiquitously responded to all analyzed stressors (PKLR, AHSG, SQRDL). Such genes may deserve further attention as possible biomarkers for environmental stress-related adaptation, as will be required under climate change conditions.</p></div><div class="section" id="sec-5"><h2 class="">Conclusions</h2><p id="p-26">Here we have shown that adaptations to abiotic environmental parameters are characterized by a high degree of genomic recurrence and functional similarity, in line with the hypothesis of genomic constraint. Although Lacertidae are thought to be excellent thermoregulators (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>), many lacertid lizard populations, especially those in humid montane localities, are currently undergoing population declines linked to climate change (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>). Studies are already underway that assess the future adaptive potential or “evolvability” of extant populations based on physiological parameters and distribution areas (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>). The genes identified in this study deserve concerted focus in future research aimed at evaluating current climate stress on populations at the molecular level, and whether it may lead to evolutionary adaptation in the future. Bay and colleagues (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>) pioneered such an approach to determine the likelihood of heat adaptation in corals. We recommend that similar efforts in vertebrates focus on the genes and functions outlined in this contribution, and on the specific functional mutations of different alleles within these candidate genes.</p></div><div class="section" id="sec-6"><h2 class="">Materials &amp; Methods</h2><p id="p-27">Transcriptomic (RNAseq) data of 24 lacertid taxa were obtained from (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">29</a>) (collection and ethics permits listed therein). The final alignment contained 6,269 gene sequences, which could be annotated to 5,498 unique gene symbols. Gene sequences obtained from RNAseq with no missing taxa were further checked for alignment errors including HMMCleaner for false negatives to pre-empt including artefacts in nucleotide-level analyses. Ten sequences were found to contain such potential artifacts and were removed from this data set. A maximum likelihood tree was generated from the concatenated alignment which is also published in (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">29</a>). From this final dataset for 24 lacertids (22 species; two species with two divergent populations each) and one outgroup, we selected a subset of 695 genes with no missing sequences to analyze for signatures of selection. The aBSREL (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>) random-effects branch site model as implemented in HyPhy (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>) was used to find lineages subject to episodic positive diversifying selection. The full model was run in exploratory mode for all branches with Holm-Bonferroni correction. Summary statistics of dN/dS (ω) were computed across all branches where episodic positive diversifying selection was identified. A functional genetic network of all genes found to harbor signatures of selection was generated with Cytoscape v.3 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>) using the STRING app against the <em>Anolis carolinensis</em> genome as background, the closest related lizard with a sequenced and comprehensively annotated genome. The network was then tested for functional enrichment in Cytoscape using the ClueGO app v.2.5.0 (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>), with the CluePedia plugin (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>).</p><p id="p-28">We compiled a matrix of putative EAGs in other vertebrate endotherms and ectotherms from 27 studies, adding the 200 genes obtained for Lacertidae. Support for this cross-taxon comparative functional genomics approach comes from the fact that orthologs across taxa are, despite their phylogenetic definition, most commonly identified by conserved function (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>). The STRING database plugin within the Cytoscape (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a>) software was used to generate a functional genomic PPI network from these genes, using human PPI databases (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">59</a>). We recorded and visualized whether single EAGs (nodes) within this network additionally adapted recurrently (i.e., in more than one species, whether they were responsive to stress, and recorded which of the Lacertidae genes under selection were part of this network. Gene functions in form of Gene Ontologies (GO), for EAGs that were additionally part of the resulting network (N=902 genes), were generated and grouped in ClueGo within Cytoscape (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>). We then sorted, where appropriate, the resulting grouped GO terms into the functional categories for climate adaptation that we and others have previously defined (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">31</a>). Additional GO functions were also recorded. We subsequently repeated this analysis for only those genes under selection in Lacertidae that were additionally part of the functional network (putative EAGs, N=171), and for those genes that were under selection in Lacertidae, but were not part of the functional network (less likely to be EAGs, N=29).</p><p id="p-29">The 695 genes in our RNAseq alignment of Lacertidae represent a subset of genes of the entire aligned transcriptomes that were selected for analysis of selection based on the absence of paralogs and presence of transcripts in all 24 lacertid taxa, as explained above. Consequently, the 200 genes under selection found within this dataset might represent a nonrandom subset of the transcriptome dataset consisting of 5,498 annotated genes, with genes related to fundamental metabolic cell processes possibly being overrepresented as they would more likely to be found in all studied lacertid species. To test to which extent the functional properties of Lacertidae genes under selection is determined by a sampling bias and whether they are different from random genes, we therefore generated comparison groups in form of replicates of 200 randomly drawn genes. These were drawn from the total sequenced exomes (20 sets), from the genes not found under selection (20 sets), from 451 human housekeeping genes expressed in different types of healthy cells (brain, kidney, prostate, liver, muscle, lung, vulva, 10 sets) (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>), and from the human genome (20 sets, obtained via <a href="http://www.molbiotools.com">www.molbiotools.com</a>). GO terms are a hierarchical vocabulary-based classification method for gene functions (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>). Functional similarity of gene products can therefore be quantitatively estimated through the semantic similarity of GO terms (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">61</a>). We compared the mean semantic similarity of genes among the lacertid genes under selection, and the other comparison groups using ±95% prediction intervals with Z-tests. This analysis was repeated also for the semantic similarity of the 902 (lacertid + other vertebrate) EAGs that were part of the functional network, compared to 20 sets of 902 genes randomly drawn from the human genome as described above.</p><p id="p-30">The question of constraint by gene identity was addressed by identifying the proportion of genes that were found to have adapted to differences in the abiotic environment in more than one species (=adapted recurrently). We simulated the number of genes that have to be drawn from a hypothetical genome to yield the same number of recurrent EAG, and tested for differences between observed number of genes and the simulated distribution using ± 95% prediction intervals and Z-test.</p><p id="p-31">Additionally, data on candidate functions for environmental adaptation were collected for all EAGs. Genes were binary coded for their association with pre-defined candidate functions for environmental adaptation. Significance was based on FDR-corrected p-values of overrepresented GOs obtained with ClueGo. To code involvement in cellular pathways active in cancer as a candidate function for EAGs, gene expression data for 813 EAGs were obtained from microarray experiments of 89 cancers (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>) and universal reference tissue. With respect to involvement in the candidate function apoptosis, “zombie genes” were identified in our data matrix (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">16</a>). We binary-coded involvement in the acute response to stress caused by environmental factors from published information from 75-omics studies on the environmental stress response. Data on presence and type of experimentally confirmed PTMs was obtained from the PhosphoSite database (<a href="http://www.phosphosite.org">www.phosphosite.org</a>, last accessed February 2019), and compared counts against the previously described 20 replicates of 200 randomly drawn genes from the human genome using Mann-Whitney U test and Z-statistic of randomized Mann-Whitney W statistic. To investigate differences between ectotherm and endotherm species, genes expressed during embryonic development for zebrafish as an ectotherm model species (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>) and humans as an endotherm model species (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>) were also binary coded. The resulting gene/ function matrix was subsequently analyzed via MCA in R (package FactoMineR (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>) to align genes into functional clusters. Statistical support for MCA dimension groupings were obtained from 100,000 multiscale bootstrap replicates in the R package <em>pvclust (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>).</em> For details see Supplementary Methods.</p></div><div class="section" id="sec-7"><h2 class="">Data deposition statement</h2><p id="p-33">Additional figures, tables and code is provided in Supplementary Materials. RNA-Seq data are available from the Sequence Read Archive (SRA Bioproject PRJNA543749). Binary data table for environmental adaptation genes and functions is provided as supplemental file in. csv format (Table S7).</p></div><div class="section" id="sec-8"><h2 class="">Author contributions</h2><p id="p-34">KWV, MV, JM, KBS designed research, KWV, JGP, II, SK performed research, OJG, PP, APT contributed new reagents or analytic tools, KWV, JGP, II, LF, AB, SFS, APT, PBA analyzed data, KWV and all co-authors wrote the paper</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-32">The Deutsche Forschungsgemeinschaft (DFG) supported M.V. and J.M. (VE 247/11-1 / MU 1760/9-1). The University of Hull is thanked for funding the Molecular Stress in changing aquatic environments (MolStressH<sub>2</sub>O) research cluster (supporting K.W.V., P.B.A., and L.F.); other cluster members are thanked for helpful discussions. Hervé Philippe (Station d’Ecologie Expérimentale CNRS, Moulis, France) is thanked for help with sequence analysis. We acknowledge the Viper High Performance Computing facility of the University of Hull and its support team. Computations were partly carried out at the Swedish National Infrastructure for Computing (SNIC) at Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under project SNIC 2017/7-275. S.S. acknowledges an Allam PhD studentship and A.B. acknowledges a PhD scholarship within the Endothelial cells in chronic disease research cluster at UoH. I.I. was supported by a postdoctoral Juan de la Cierva-Incorporación postdoctoral fellowship (IJCI-2016-29566) from Spanish Ministry of Economy (MINECO).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Co-author e-mails: Joan Garcia-Porta (<span class="em-link"><span class="em-addr">j.garcia-porta{at}wustl.edu</span></span>); Iker Irisarri (<span class="em-link"><span class="em-addr">irisarri.iker{at}gmail.com</span></span>); Lauric Feugere (<span class="em-link"><span class="em-addr">L.Feugere-2018{at}hull.ac.uk</span></span>); Adam Bates (<span class="em-link"><span class="em-addr">Adam.Bates-2018{at}hull.ac.uk</span></span>); Sebastian Kirchhof (<span class="em-link"><span class="em-addr">Sebastian.Kirchhof{at}mfn.berlin</span></span>); Olga Jovanović Glavaš (<span class="em-link"><span class="em-addr">jovanovic.olga{at}gmail.com</span></span>); Panayiotis Pafilis (<span class="em-link"><span class="em-addr">ppafil{at}biol.uoa.gr</span></span>); Sabrina F. Samuel (<span class="em-link"><span class="em-addr">S.Samuel{at}2017.hull.ac.uk</span></span>); Johannes Müller (<span class="em-link"><span class="em-addr">Johannes.Mueller{at}mfn.berlin</span></span>); Miguel Vences (<span class="em-link"><span class="em-addr">m.vences{at}tu-bs.de</span></span>); Alexander P. Turner (<span class="em-link"><span class="em-addr">Alexander.Turner{at}hull.ac.uk</span></span>); Pedro Beltran-Alvarez (<span class="em-link"><span class="em-addr">P.Beltran-Alvarez{at}hull.ac.uk</span></span>); Kenneth B. Storey (<span class="em-link"><span class="em-addr">KenStorey{at}cunet.carleton.ca</span></span>)</p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.1" data-doi="10.1111/j.1558-5646.2010.00960.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Futuyma</span>  <span class="cit-name-given-names">DJ</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Evolutionary constraint and ecological consequences</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">64</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1865</span>–<span class="cit-lpage">1884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DFutuyma%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D64%26rft.issue%253D7%26rft.spage%253D1865%26rft.epage%253D1884%26rft.atitle%253DEvolutionary%2Bconstraint%2Band%2Becological%2Bconsequences.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2010.00960.x%26rft_id%253Dinfo%253Apmid%252F20659157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2010.00960.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20659157&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279617800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">2.</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Turelli</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">1985</span>) <span class="cit-article-title">Effects of pleiotropy on predictions concerning mutation-selection balance for polygenic traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">111</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">165</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTurelli%26rft.auinit1%253DM.%26rft.volume%253D111%26rft.issue%253D1%26rft.spage%253D165%26rft.epage%253D195%26rft.atitle%253DEFFECTS%2BOF%2BPLEIOTROPY%2BON%2BPREDICTIONS%2BCONCERNING%2BMUTATION-SELECTION%2BBALANCE%2BFOR%2BPOLYGENIC%2BTRAITS%26rft_id%253Dinfo%253Apmid%252F4029610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTExLzEvMTY1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMDgvNzI2MjQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.3" data-doi="10.1371/journal.pgen.1001321"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salverda</span>  <span class="cit-name-given-names">MLM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Initial mutations direct alternative pathways of protein evolution</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">e1001321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSalverda%26rft.auinit1%253DM.%2BL.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253De1001321%26rft.epage%253De1001321%26rft.atitle%253DInitial%2Bmutations%2Bdirect%2Balternative%2Bpathways%2Bof%2Bprotein%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001321%26rft_id%253Dinfo%253Apmid%252F21408208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21408208&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.4" data-doi="10.1126/science.aam5979"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Blount</span>  <span class="cit-name-given-names">ZD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lenski</span>  <span class="cit-name-given-names">RE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Losos</span>  <span class="cit-name-given-names">JB</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Contingency and determinism in evolution: Replaying life’s tape</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">362</span>(<span class="cit-issue">6415</span>):<span class="cit-fpage">eaam5979</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBlount%26rft.auinit1%253DZ.%2BD.%26rft.volume%253D362%26rft.issue%253D6415%26rft.spage%253Deaam5979%26rft.epage%253Deaam5979%26rft.atitle%253DContingency%2Band%2Bdeterminism%2Bin%2Bevolution%253A%2BReplaying%2Blifes%2Btape%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam5979%26rft_id%253Dinfo%253Apmid%252F30409860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNjIvNjQxNS9lYWFtNTk3OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzA4LzcyNjI0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.5" data-doi="10.1126/science.aaw7271"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Therkildsen</span>  <span class="cit-name-given-names">NO</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Contrasting genomic shifts underlie parallel phenotypic evolution in response to fishing</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">365</span>(<span class="cit-issue">6452</span>):<span class="cit-fpage">487</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTherkildsen%26rft.auinit1%253DN.%2BO.%26rft.volume%253D365%26rft.issue%253D6452%26rft.spage%253D487%26rft.epage%253D490%26rft.atitle%253DContrasting%2Bgenomic%2Bshifts%2Bunderlie%2Bparallel%2Bphenotypic%2Bevolution%2Bin%2Bresponse%2Bto%2Bfishing%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaw7271%26rft_id%253Dinfo%253Apmid%252F31371613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjUvNjQ1Mi80ODciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOC8wOC83MjYyNDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.6" data-doi="10.1016/j.tree.2012.01.016"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pavlicev</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">GP</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">A model of developmental evolution: selection, pleiotropy and compensation</span>. <abbr class="cit-jnl-abbrev">Trends Ecol Evol</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">316</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DPavlicev%26rft.auinit1%253DM.%26rft.volume%253D27%26rft.issue%253D6%26rft.spage%253D316%26rft.epage%253D322%26rft.atitle%253DA%2Bmodel%2Bof%2Bdevelopmental%2Bevolution%253A%2Bselection%252C%2Bpleiotropy%2Band%2Bcompensation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2012.01.016%26rft_id%253Dinfo%253Apmid%252F22385978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2012.01.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22385978&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305105000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.7" data-doi="10.1038/nature02555"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">J-DJ</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Evidence for dynamically organized modularity in the yeast protein-protein interaction network</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>(<span class="cit-issue">6995</span>):<span class="cit-fpage">88</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHan%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D430%26rft.issue%253D6995%26rft.spage%253D88%26rft.epage%253D93%26rft.atitle%253DEvidence%2Bfor%2Bdynamically%2Borganized%2Bmodularity%2Bin%2Bthe%2Byeast%2Bprotein-protein%2Binteraction%2Bnetwork.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02555%26rft_id%253Dinfo%253Apmid%252F15190252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02555&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15190252&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222356800049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">8.</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wollenberg Valero</span>  <span class="cit-name-given-names">KC</span></span> (<span class="cit-pub-date">2018</span>) <abbr class="cit-jnl-abbrev">Aligning functional network constraint to evolutionary outcomes</abbr>. bioRxiv:278663.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">9.</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.9" data-doi="10.1111/j.1558-5646.2011.01289.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Losos</span>  <span class="cit-name-given-names">JB</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Convergence, adaptation, and constraint</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">65</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">1827</span>–<span class="cit-lpage">1840</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DLosos%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D65%26rft.issue%253D7%26rft.spage%253D1827%26rft.epage%253D1840%26rft.atitle%253DConvergence%252C%2Badaptation%252C%2Band%2Bconstraint.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2011.01289.x%26rft_id%253Dinfo%253Apmid%252F21729041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2011.01289.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729041&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292447000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.10" data-doi="10.1098/rstb.2015.0477"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Babonis</span>  <span class="cit-name-given-names">LS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martindale</span>  <span class="cit-name-given-names">MQ</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Phylogenetic evidence for the modular evolution of metazoan signalling pathways</span>. <abbr class="cit-jnl-abbrev">Philos Trans R Soc Lond B Biol Sci</abbr> <span class="cit-vol">372</span>(<span class="cit-issue">1713</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1098/rstb.2015.0477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos%2BTrans%2BR%2BSoc%2BLond%2BB%2BBiol%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2015.0477%26rft_id%253Dinfo%253Apmid%252F27994120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2015.0477&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27994120&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.11" data-doi="10.1038/hdy.2014.119"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Porcelli</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Butlin</span>  <span class="cit-name-given-names">RK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaston</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joly</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Snook</span>  <span class="cit-name-given-names">RR</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">The environmental genomics of metazoan thermal adaptation</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">502</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D502%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rodríguez</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Genomic and phenotypic signatures of climate adaptation in an Anolis lizard</span>. <abbr class="cit-jnl-abbrev">Ecol Evol</abbr> <span class="cit-vol">7</span>:<span class="cit-fpage">6390</span>–<span class="cit-lpage">6403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BEvol%26rft.volume%253D7%26rft.spage%253D6390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.13" data-doi="10.1111/j.1749-6632.2009.04433.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">RJ</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Structural and functional evolution of vertebrate neuroendocrine stress systems</span>. <abbr class="cit-jnl-abbrev">Ann N Y Acad Sci</abbr> <span class="cit-vol">1163</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DDenver%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D1163%26rft.spage%253D1%26rft.epage%253D16%26rft.atitle%253DStructural%2Band%2Bfunctional%2Bevolution%2Bof%2Bvertebrate%2Bneuroendocrine%2Bstress%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2009.04433.x%26rft_id%253Dinfo%253Apmid%252F19456324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2009.04433.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19456324&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266493400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.14" data-doi="10.1242/jeb.114397"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Logan</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckley</span>  <span class="cit-name-given-names">BA</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Transcriptomic responses to environmental temperature in eurythermal and stenothermal fishes</span>. <abbr class="cit-jnl-abbrev">J Exp Biol</abbr> <span class="cit-vol">218</span>(<span class="cit-issue">Pt 12</span>):<span class="cit-fpage">1915</span>–<span class="cit-lpage">1924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DLogan%26rft.auinit1%253DC.%2BA.%26rft.volume%253D218%26rft.issue%253D12%26rft.spage%253D1915%26rft.epage%253D1924%26rft.atitle%253DTranscriptomic%2Bresponses%2Bto%2Benvironmental%2Btemperature%2Bin%2Beurythermal%2Band%2Bstenothermal%2Bfishes%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.114397%26rft_id%253Dinfo%253Apmid%252F26085668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjExOiIyMTgvMTIvMTkxNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzA4LzcyNjI0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.15" data-doi="10.1016/0167-5699(94)90018-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Buttke</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sandstrom</span>  <span class="cit-name-given-names">PA</span></span> (<span class="cit-pub-date">1994</span>) <span class="cit-article-title">Oxidative stress as a mediator of apoptosis</span>. <abbr class="cit-jnl-abbrev">Immunol Today</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">7</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunology%2Btoday%26rft.stitle%253DImmunol%2BToday%26rft.aulast%253DButtke%26rft.auinit1%253DT.%2BM.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D10%26rft.atitle%253DOxidative%2Bstress%2Bas%2Ba%2Bmediator%2Bof%2Bapoptosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0167-5699%252894%252990018-3%26rft_id%253Dinfo%253Apmid%252F8136014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0167-5699(94)90018-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8136014&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994MT96700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.16" data-doi="10.1098/rsob.160267"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pozhitkov</span>  <span class="cit-name-given-names">AE</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Tracing the dynamics of gene transcripts after organismal death</span>. <abbr class="cit-jnl-abbrev">Open Biol</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">1</span>).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1098/rsob.160267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOpen%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsob.160267%26rft_id%253Dinfo%253Apmid%252F28123054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsob.160267&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28123054&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.17" data-doi="10.1016/S0022-2836(02)00379-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">LJ</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Prediction of human protein function from post-translational modifications and localization features</span>. <abbr class="cit-jnl-abbrev">J Mol Biol</abbr> <span class="cit-vol">319</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1257</span>–<span class="cit-lpage">1265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DJensen%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D319%26rft.issue%253D5%26rft.spage%253D1257%26rft.epage%253D1265%26rft.atitle%253DPrediction%2Bof%2Bhuman%2Bprotein%2Bfunction%2Bfrom%2Bpost-translational%2Bmodifications%2Band%2Blocalization%2Bfeatures.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252802%252900379-0%26rft_id%253Dinfo%253Apmid%252F12079362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(02)00379-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12079362&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176687400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.18" data-doi="10.1038/ng.2343"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">Q</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">The yak genome and adaptation to life at high altitude</span>. <abbr class="cit-jnl-abbrev">Nat Genet</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">946</span>–<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DQiu%26rft.auinit1%253DQ.%26rft.volume%253D44%26rft.issue%253D8%26rft.spage%253D946%26rft.epage%253D949%26rft.atitle%253DThe%2Byak%2Bgenome%2Band%2Badaptation%2Bto%2Blife%2Bat%2Bhigh%2Baltitude.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2343%26rft_id%253Dinfo%253Apmid%252F22751099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22751099&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.19" data-doi="10.1089/ham.2015.0033"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Simonson</span>  <span class="cit-name-given-names">TS</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Altitude Adaptation: A Glimpse Through Various Lenses</span>. <abbr class="cit-jnl-abbrev">High Alt Med Biol</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">125</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHigh%2BAlt%2BMed%2BBiol%26rft.volume%253D16%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fham.2015.0033%26rft_id%253Dinfo%253Apmid%252F26070057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/ham.2015.0033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26070057&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.20" data-doi="10.1073/pnas.94.8.3817"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">DeVries</span>  <span class="cit-name-given-names">AL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">CH</span></span> (<span class="cit-pub-date">1997</span>) <span class="cit-article-title">Convergent evolution of antifreeze glycoproteins in Antarctic notothenioid fish and Arctic cod</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">3817</span>–<span class="cit-lpage">3822</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DChen%26rft.auinit1%253DL.%26rft.volume%253D94%26rft.issue%253D8%26rft.spage%253D3817%26rft.epage%253D3822%26rft.atitle%253DConvergent%2Bevolution%2Bof%2Bantifreeze%2Bglycoproteins%2Bin%2BAntarctic%2Bnotothenioid%2Bfish%2Band%2BArctic%2Bcod%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.8.3817%26rft_id%253Dinfo%253Apmid%252F9108061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NC84LzM4MTciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOC8wOC83MjYyNDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">21.</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.21" data-doi="10.1073/pnas.1817138116"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">X</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">KR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C-HC</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Molecular mechanism and history of non-sense to sense evolution of antifreeze glycoprotein gene in northern gadids</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1073/pnas.1817138116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1817138116%26rft_id%253Dinfo%253Apmid%252F30765531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE2LzEwLzQ0MDAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOC8wOC83MjYyNDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">22.</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feiner</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rago</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">While</span>  <span class="cit-name-given-names">GM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Uller</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Signatures of selection in embryonic transcriptomes of lizards adapting in parallel to cool climate</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">72</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">67</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D72%26rft.spage%253D67%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.23" data-doi="10.1002/jez.2185"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanger</span>  <span class="cit-name-given-names">TJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kyrkos</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lachance</span>  <span class="cit-name-given-names">DJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Czesny</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stroud</span>  <span class="cit-name-given-names">JT</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">The effects of thermal stress on the early development of the lizard Anolis sagrei</span>. <abbr class="cit-jnl-abbrev">J Exp Zool A Ecol Integr Physiol</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/jez.2185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BZool%2BA%2BEcol%2BIntegr%2BPhysiol%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.2185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.2185&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.24" data-doi="10.1371/journal.pgen.0040032"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Adaptations to climate in candidate genes for common metabolic disorders</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">e32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253De32%26rft.epage%253De32%26rft.atitle%253DAdaptations%2Bto%2Bclimate%2Bin%2Bcandidate%2Bgenes%2Bfor%2Bcommon%2Bmetabolic%2Bdisorders.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0040032%26rft_id%253Dinfo%253Apmid%252F18282109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0040032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18282109&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.25" data-doi="10.1016/j.jtherbio.2018.11.005"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Diele-Viegas</span>  <span class="cit-name-given-names">LM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rocha</span>  <span class="cit-name-given-names">CFD</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Unraveling the influences of Climate Change in Lepidosauria (Reptilia)</span>. <abbr class="cit-jnl-abbrev">J Therm Biol</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.jtherbio.2018.11.005</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BTherm%2BBiol%26rft.volume%253D78%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtherbio.2018.11.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtherbio.2018.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.26" data-doi="10.1126/science.aam5512"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Campbell-Staton</span>  <span class="cit-name-given-names">SC</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Winter storms drive rapid phenotypic, regulatory, and genomic shifts in the green anole lizard</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">357</span>(<span class="cit-issue">6350</span>):<span class="cit-fpage">495</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCampbell-Staton%26rft.auinit1%253DS.%2BC.%26rft.volume%253D357%26rft.issue%253D6350%26rft.spage%253D495%26rft.epage%253D498%26rft.atitle%253DWinter%2Bstorms%2Bdrive%2Brapid%2Bphenotypic%252C%2Bregulatory%252C%2Band%2Bgenomic%2Bshifts%2Bin%2Bthe%2Bgreen%2Banole%2Blizard%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam5512%26rft_id%253Dinfo%253Apmid%252F28774927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTcvNjM1MC80OTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wOC8wOC83MjYyNDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">27.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-726240v1.27"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Exploring the genetic basis of adaptation to high elevations in reptiles: a comparative transcriptome analysis of two toad-headed agamas (genus Phrynocephalus)</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">e112218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.28" data-doi="10.1073/pnas.1813593115"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y-B</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Species groups distributed across elevational gradients reveal convergent and continuous genetic adaptation to high elevations</span>. <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci U S A</abbr> <span class="cit-vol">115</span>(<span class="cit-issue">45</span>):<span class="cit-fpage">E10634</span>–<span class="cit-lpage">E10641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1813593115%26rft_id%253Dinfo%253Apmid%252F30348757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMzoiMTE1LzQ1L0UxMDYzNCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzA4LzcyNjI0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Garcia-Porta</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2019</span> in press) <span class="cit-article-title">Environmental temperatures shape thermal physiology, diversification and genome-wide substitution rates in lizards</span>. <abbr class="cit-jnl-abbrev">Nat Commun</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.30" data-doi="10.1111/ele.12155"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Araújo</span>  <span class="cit-name-given-names">MB</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Heat freezes niche evolution</span>. <abbr class="cit-jnl-abbrev">Ecol Lett</abbr> <span class="cit-vol">16</span>:<span class="cit-fpage">1206</span>–<span class="cit-lpage">1219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol%2BLett%26rft.volume%253D16%26rft.spage%253D1206%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12155%26rft_id%253Dinfo%253Apmid%252F23869696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23869696&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wollenberg Valero</span>  <span class="cit-name-given-names">KC</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">A candidate multimodal functional genetic network for thermal adaptation</span>. <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">2</span>:<span class="cit-fpage">e578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D2%26rft.spage%253De578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.32" data-doi="10.1098/rspb.2011.2060"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">CR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alton</span>  <span class="cit-name-given-names">LA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frappell</span>  <span class="cit-name-given-names">PB</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Metabolic cold adaptation in fishes occurs at the level of whole animal, mitochondria and enzyme</span>. <abbr class="cit-jnl-abbrev">Proc Roy Soc B</abbr> <span class="cit-vol">279</span>(<span class="cit-issue">1734</span>):<span class="cit-fpage">1740</span>–<span class="cit-lpage">1747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BRoy%2BSoc%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2011.2060%26rft_id%253Dinfo%253Apmid%252F22158960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2011.2060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22158960&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.33" data-doi="10.1093/icb/icy017"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sokolova</span>  <span class="cit-name-given-names">I</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Mitochondrial adaptations to variable environments and their role in animals’ stress tolerance</span>. <abbr class="cit-jnl-abbrev">Integr Comp Biol</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/icb/icy017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegr%2BComp%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ficb%252Ficy017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/icb/icy017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.34" data-doi="10.1098/rspb.2013.2433"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">MM</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Evolutionary stasis and lability in thermal physiology in a group of tropical lizards</span>. <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society of London B: Biological Sciences</abbr> <span class="cit-vol">281</span>(<span class="cit-issue">1778</span>):<span class="cit-fpage">20132433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2Bof%2BLondon%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.2433%26rft_id%253Dinfo%253Apmid%252F24430845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2013.2433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24430845&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.35" data-doi="10.1093/gbe/evz110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bourgeois</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ruggiero</span>  <span class="cit-name-given-names">RP</span></span>, <span class="cit-auth"><span class="cit-name-surname">Manthey</span>  <span class="cit-name-given-names">JD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Boissinot</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Recent secondary contacts, linked selection and variable recombination rates shape genomic diversity in the model species Anolis carolinensis</span>. <abbr class="cit-jnl-abbrev">Genome Biol Evol</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/gbe/evz110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevz110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evz110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.36" data-doi="10.1093/bioinformatics/btq064"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">GOSemSim: an R package for measuring semantic similarity among GO terms and gene products</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">976</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq064%26rft_id%253Dinfo%253Apmid%252F20179076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20179076&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276045800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.37" data-doi="10.1016/S0891-5849(01)00749-3"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dalle-Donne</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rossi</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Milzani</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Simplicio</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Colombo</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">The actin cytoskeleton response to oxidants: from small heat shock protein phosphorylation to changes in the redox state of actin itself</span>. <abbr class="cit-jnl-abbrev">Free Radic Biol Med</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1624</span>–<span class="cit-lpage">1632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFree%2Bradical%2Bbiology%2B%2526%2Bmedicine%26rft.stitle%253DFree%2BRadic%2BBiol%2BMed%26rft.aulast%253DDalle-Donne%26rft.auinit1%253DI.%26rft.volume%253D31%26rft.issue%253D12%26rft.spage%253D1624%26rft.epage%253D1632%26rft.atitle%253DThe%2Bactin%2Bcytoskeleton%2Bresponse%2Bto%2Boxidants%253A%2Bfrom%2Bsmall%2Bheat%2Bshock%2Bprotein%2Bphosphorylation%2Bto%2Bchanges%2Bin%2Bthe%2Bredox%2Bstate%2Bof%2Bactin%2Bitself.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0891-5849%252801%252900749-3%26rft_id%253Dinfo%253Apmid%252F11744337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0891-5849(01)00749-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11744337&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172796300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.38" data-doi="10.1093/icb/44.1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Klose</span>  <span class="cit-name-given-names">MK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">RM</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">Stress-induced thermoprotection of neuromuscular transmission</span>. <abbr class="cit-jnl-abbrev">Integr Comp Biol</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">14</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegrative%2Band%2BComparative%2BBiology%26rft.stitle%253DIntegr.%2BComp.%2BBiol.%26rft.aulast%253DKlose%26rft.auinit1%253DM.%2BK.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D14%26rft.epage%253D20%26rft.atitle%253DStress-Induced%2BThermoprotection%2Bof%2BNeuromuscular%2BTransmission%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ficb%252F44.1.14%26rft_id%253Dinfo%253Apmid%252F21680481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/icb/44.1.14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21680481&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222323000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.39" data-doi="10.1016/S1095-6433(01)00429-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">EK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">JW</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">The cytoskeleton and cell volume regulation</span>. <abbr class="cit-jnl-abbrev">Comp Biochem Physiol A Mol Integr Physiol</abbr> <span class="cit-vol">130</span>(<span class="cit-issue">3</span>):<span class="cit-fpage">385</span>–<span class="cit-lpage">399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BA%252C%2BMolecular%2B%2526%2BIntegrative%2BPhysiology%26rft.stitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BA%252C%2BMolecular%2B%2526%2BIntegrative%2BPhysiology%26rft.aulast%253DPedersen%26rft.auinit1%253DS.%2BF.%26rft.volume%253D130%26rft.issue%253D3%26rft.spage%253D385%26rft.epage%253D399%26rft.atitle%253DThe%2Bcytoskeleton%2Band%2Bcell%2Bvolume%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1095-6433%252801%252900429-9%26rft_id%253Dinfo%253Apmid%252F11913452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1095-6433(01)00429-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11913452&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.40" data-doi="10.1387/ijdb.082643pm"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morin</span>  <span class="cit-name-given-names">P</span>   <span class="cit-name-suffix">Jr</span>.</span>, <span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">KB</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Mammalian hibernation: differential gene expression and novel application of epigenetic controls</span>. <abbr class="cit-jnl-abbrev">Int J Dev Biol</abbr> <span class="cit-vol">53</span>(<span class="cit-issue">2–3</span>):<span class="cit-fpage">433</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BInternational%2Bjournal%2Bof%2Bdevelopmental%2Bbiology%26rft.stitle%253DInt%2BJ%2BDev%2BBiol%26rft.aulast%253DMorin%26rft.auinit1%253DP.%26rft.volume%253D53%26rft.issue%253D2-3%26rft.spage%253D433%26rft.epage%253D442%26rft.atitle%253DMammalian%2Bhibernation%253A%2Bdifferential%2Bgene%2Bexpression%2Band%2Bnovel%2Bapplication%2Bof%2Bepigenetic%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1387%252Fijdb.082643pm%26rft_id%253Dinfo%253Apmid%252F19412897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1387/ijdb.082643pm&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19412897&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267306900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.41" data-doi="10.1083/jcb.201501101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bond</span>  <span class="cit-name-given-names">MR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hanover</span>  <span class="cit-name-given-names">JA</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">A little sugar goes a long way: the cell biology of O-GlcNAc</span>. <abbr class="cit-jnl-abbrev">J Cell Biol</abbr> <span class="cit-vol">208</span>(<span class="cit-issue">7</span>):<span class="cit-fpage">869</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BCell%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201501101%26rft_id%253Dinfo%253Apmid%252F25825515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIwOC83Lzg2OSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzA4LzcyNjI0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.42" data-doi="10.1016/j.freeradbiomed.2015.01.028"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">DesMarais</span>  <span class="cit-name-given-names">TL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">Y</span></span>, <span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">M</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Oxidative stress alters global histone modification and DNA methylation</span>. <abbr class="cit-jnl-abbrev">Free Radic Biol Med</abbr> <span class="cit-vol">82</span>:<span class="cit-fpage">22</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFree%2BRadic%2BBiol%2BMed%26rft.volume%253D82%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.freeradbiomed.2015.01.028%26rft_id%253Dinfo%253Apmid%252F25656994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.freeradbiomed.2015.01.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25656994&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Lysine Acetylation and Succinylation in HeLa Cells and their Essential Roles in Response to UV-induced Stress</span>. <abbr class="cit-jnl-abbrev">Sci Rep</abbr> <span class="cit-vol">6</span>:<span class="cit-fpage">30212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D6%26rft.spage%253D30212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.44" data-doi="10.1016/j.tig.2013.05.010"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eisenberg</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levanon</span>  <span class="cit-name-given-names">EY</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Human housekeeping genes, revisited</span>. <abbr class="cit-jnl-abbrev">Trends Genet</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">569</span>–<span class="cit-lpage">574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet%26rft.volume%253D29%26rft.spage%253D569%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.05.010%26rft_id%253Dinfo%253Apmid%252F23810203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.05.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23810203&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325300800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.45" data-doi="10.1534/g3.111.001560"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pancaldi</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Predicting the fission yeast protein interaction network</span>. <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">453</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D2%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.001560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.46" data-doi="10.1038/nrg2398"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">López-Maury</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Marguerat</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bähler</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Tuning gene expression to changing environments: from rapid responses to evolutionary adaptation</span>. <abbr class="cit-jnl-abbrev">Nat Rev Genet</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">583</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLopez-Maury%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D583%26rft.epage%253D593%26rft.atitle%253DTuning%2Bgene%2Bexpression%2Bto%2Bchanging%2Benvironments%253A%2Bfrom%2Brapid%2Bresponses%2Bto%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2398%26rft_id%253Dinfo%253Apmid%252F18591982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2398&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18591982&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257758400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.47" data-doi="10.1146/annurev.physiol.61.1.243"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Feder</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">GE</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Heat-shock proteins, molecular chaperones, and the stress response: evolutionary and ecological physiology</span>. <abbr class="cit-jnl-abbrev">Annu Rev Physiol</abbr> <span class="cit-vol">61</span>:<span class="cit-fpage">243</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BPhysiology%26rft.stitle%253DAnnual%2BReview%2Bof%2BPhysiology%26rft.aulast%253DFeder%26rft.auinit1%253DM.%2BE.%26rft.volume%253D61%26rft.issue%253D1%26rft.spage%253D243%26rft.epage%253D282%26rft.atitle%253DHEAT-SHOCK%2BPROTEINS%252C%2BMOLECULAR%2BCHAPERONES%252C%2BAND%2BTHE%2BSTRESS%2BRESPONSE%253A%2BEvolutionary%2Band%2BEcological%2BPhysiology%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.physiol.61.1.243%26rft_id%253Dinfo%253Apmid%252F10099689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.physiol.61.1.243&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10099689&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079229000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ortega</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mencía</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pérez-Mellado</span>  <span class="cit-name-given-names">V</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">The peak of thermoregulation effectiveness: Thermal biology of the Pyrenean rock lizard, Iberolacerta bonnali (Squamata, Lacertidae)</span>. <abbr class="cit-jnl-abbrev">J Therm Biol</abbr> <span class="cit-vol">56</span>:<span class="cit-fpage">77</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BTherm%2BBiol%26rft.volume%253D56%26rft.spage%253D77%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">49.</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sannolo</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barroso</span>  <span class="cit-name-given-names">FM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carretero</span>  <span class="cit-name-given-names">MA</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Physiological differences in preferred temperatures and evaporative water loss rates in two sympatric lacertid species</span>. <abbr class="cit-jnl-abbrev">Zoology</abbr> <span class="cit-vol">126</span>:<span class="cit-fpage">58</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoology%26rft.volume%253D126%26rft.spage%253D58%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">50.</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.50"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Carneiro</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Garcia-Munoz</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zagar</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pafilis</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carretero</span>  <span class="cit-name-given-names">MA</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Is ecophysiology congruent with the present-day relictual distribution of a lizard group? Evidence from preferred temperatures and water loss rates</span>. <abbr class="cit-jnl-abbrev">Herpetol J</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">47</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHerpetol%2BJ%26rft.volume%253D27%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.51"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Romero-Diaz</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Breedveld</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fitze</span>  <span class="cit-name-given-names">PS</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Climate effects on growth, body condition, and survival depend on the genetic characteristics of the population</span>. <abbr class="cit-jnl-abbrev">Am Nat</abbr> <span class="cit-vol">190</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">649</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm%2BNat%26rft.volume%253D190%26rft.spage%253D649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.52" data-doi="10.1242/jeb.037473"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Somero</span>  <span class="cit-name-given-names">GN</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">The physiology of climate change: how potentials for acclimatization and genetic adaptation will determine “winners” and “losers.”</span> <abbr class="cit-jnl-abbrev">J Exp Biol</abbr> <span class="cit-vol">213</span>(<span class="cit-issue">6</span>):<span class="cit-fpage">912</span>–<span class="cit-lpage">920</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBiology%26rft.stitle%253DJ.%2BExp.%2BBiol.%26rft.aulast%253DSomero%26rft.auinit1%253DG.%2BN.%26rft.volume%253D213%26rft.issue%253D6%26rft.spage%253D912%26rft.epage%253D920%26rft.atitle%253DThe%2Bphysiology%2Bof%2Bclimate%2Bchange%253A%2Bhow%2Bpotentials%2Bfor%2Bacclimatization%2Band%2Bgenetic%2Badaptation%2Bwill%2Bdetermine%2B%2527winners%2527%2Band%2B%2527losers%2527%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.037473%26rft_id%253Dinfo%253Apmid%252F20190116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjk6IjIxMy82LzkxMiI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzA4LzcyNjI0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.53" data-doi="10.1126/sciadv.1701413"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bay</span>  <span class="cit-name-given-names">RA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">NH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Logan</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palumbi</span>  <span class="cit-name-given-names">SR</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Genomic models predict successful coral adaptation if future ocean warming rates are reduced</span>. <abbr class="cit-jnl-abbrev">Sci Adv</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">e1701413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DBay%26rft.auinit1%253DR.%2BA.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253De1701413%26rft.epage%253De1701413%26rft.atitle%253DGenomic%2Bmodels%2Bpredict%2Bsuccessful%2Bcoral%2Badaptation%2Bif%2Bfuture%2Bocean%2Bwarming%2Brates%2Bare%2Breduced%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1701413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMzoiMy8xMS9lMTcwMTQxMyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA4LzA4LzcyNjI0MC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.54" data-doi="10.1093/molbev/msv022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">MD</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Less is more: an adaptive branch-site random effects model for efficient detection of episodic diversifying selection</span>. <abbr class="cit-jnl-abbrev">Mol Biol Evol</abbr> <span class="cit-vol">32</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">1342</span>–<span class="cit-lpage">1353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol%2BBiol%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv022%26rft_id%253Dinfo%253Apmid%252F25697341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25697341&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.55" data-doi="10.1093/bioinformatics/bti079"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>  <span class="cit-name-given-names">SL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">SDW</span></span>, <span class="cit-auth"><span class="cit-name-surname">Muse</span>  <span class="cit-name-given-names">SV</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">HyPhy: hypothesis testing using phylogenies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">676</span>–<span class="cit-lpage">679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti079%26rft_id%253Dinfo%253Apmid%252F15509596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15509596&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227241200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.56" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</span>. <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">11</span>):<span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDgvMDgvNzI2MjQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.57" data-doi="10.1093/bioinformatics/btp101"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bindea</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">ClueGO: a Cytoscape plug-in to decipher functionally grouped gene ontology and pathway annotation networks</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">8</span>):<span class="cit-fpage">1091</span>–<span class="cit-lpage">1093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp101%26rft_id%253Dinfo%253Apmid%252F19237447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19237447&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265094400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.58" data-doi="10.1093/bioinformatics/btt019"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bindea</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Galon</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mlecnik</span>  <span class="cit-name-given-names">B</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">CluePedia Cytoscape plugin: pathway insights using integrated experimental and in silico data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">5</span>):<span class="cit-fpage">661</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt019%26rft_id%253Dinfo%253Apmid%252F23325622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23325622&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315623000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.59" data-doi="10.1371/journal.pcbi.1000262"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Altenhoff</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dessimoz</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Phylogenetic and functional assessment of orthologs inference projects and methods</span>. <abbr class="cit-jnl-abbrev">PLoS Comput Biol</abbr> <span class="cit-vol">5</span>:<span class="cit-fpage">e1000262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DAltenhoff%26rft.auinit1%253DA.%2BM.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000262%26rft.epage%253De1000262%26rft.atitle%253DPhylogenetic%2Band%2Bfunctional%2Bassessment%2Bof%2Borthologs%2Binference%2Bprojects%2Band%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000262%26rft_id%253Dinfo%253Apmid%252F19148271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19148271&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.60" data-doi="10.1152/physiolgenomics.00040.2001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hsiao</span>  <span class="cit-name-given-names">LL</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">A compendium of gene expression in normal human tissues</span>. <abbr class="cit-jnl-abbrev">Physiol Genomics</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">2</span>):<span class="cit-fpage">97</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00040.2001%26rft_id%253Dinfo%253Apmid%252F11773596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physiolgenomics.00040.2001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11773596&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172906700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">61.</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.61" data-doi="10.1093/bioinformatics/btm087"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">JZ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Z</span></span>, <span class="cit-auth"><span class="cit-name-surname">Payattakool</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">PS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C-F</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">A new method to measure the semantic similarity of GO terms</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">10</span>):<span class="cit-fpage">1274</span>–<span class="cit-lpage">1281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm087%26rft_id%253Dinfo%253Apmid%252F17344234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17344234&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247348300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">62.</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.62" data-doi="10.1038/nbt0410-322"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lukk</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">A global map of human gene expression</span>. <abbr class="cit-jnl-abbrev">Nat Biotechnol</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">4</span>):<span class="cit-fpage">322</span>–<span class="cit-lpage">324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DLukk%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D322%26rft.epage%253D324%26rft.atitle%253DA%2Bglobal%2Bmap%2Bof%2Bhuman%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt0410-322%26rft_id%253Dinfo%253Apmid%252F20379172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt0410-322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20379172&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276462400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.63" data-doi="10.7554/eLife.30860"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">A high-resolution mRNA expression time course of embryonic development in zebrafish</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.7554/eLife.30860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D6%26rft.spage%253D1328%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.30860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.30860&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.64" data-doi="10.1096/fj.10-158782"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Gene expression atlas for human embryogenesis</span>. <abbr class="cit-jnl-abbrev">FASEB J</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">9</span>):<span class="cit-fpage">3341</span>–<span class="cit-lpage">3350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFASEB%2BJ%26rft_id%253Dinfo%253Adoi%252F10.1096%252Ffj.10-158782%26rft_id%253Dinfo%253Apmid%252F20430792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1096/fj.10-158782&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20430792&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">65.</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference 65. in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.65" data-doi="10.18637/jss.v025.i14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lê</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Josse</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Husson</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth cit-collab">Others</span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">FactoMineR: an R package for multivariate analysis</span>. <abbr class="cit-jnl-abbrev">J Stat Softw</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">1</span>):<span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BStat%2BSoftw%26rft.volume%253D25%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v025.i14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v025.i14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">66.</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference 66. in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.66" data-doi="10.1093/bioinformatics/btl117"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimodaira</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Pvclust: an R package for assessing the uncertainty in hierarchical clustering</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">12</span>):<span class="cit-fpage">1540</span>–<span class="cit-lpage">1542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl117%26rft_id%253Dinfo%253Apmid%252F16595560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16595560&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F08%2F08%2F726240.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238768500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">67.</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference 67. in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-726240v1.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shimodaira</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">pvclust: An R package for hierarchical clustering with p-values</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>:<span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D22%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/726240v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F726240v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/720318v1" title="A VP2/3-derived peptide exhibits potent antiviral activity against BK and JC polyomaviruses by targeting a novel VP1 binding site" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/569319v4" title="Neonatal morphometric similarity mapping for predicting brain age and characterizing neuroanatomic variation associated with preterm birth" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;August 08, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/726240v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/726240v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/726240v1" class="active" data-icon-position="" data-hide-link-title="0">Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/861192" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-scyQ5YwWrPu6UPk0_fG3T7Phut90XyVqDdfaMy9SMtE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="861192" id="node861192--2" data-pisa="biorxiv;726240v1" data-pisa-master="biorxiv;726240" data-seqnum="861192" data-apath="/biorxiv/early/2019/08/08/726240.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Katharina C. Wollenberg</span> <span class="nlm-surname">Valero</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joan</span> <span class="nlm-surname">Garcia-Porta</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Iker</span> <span class="nlm-surname">Irisarri</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Lauric</span> <span class="nlm-surname">Feugere</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Adam</span> <span class="nlm-surname">Bates</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sebastian</span> <span class="nlm-surname">Kirchhof</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Olga</span> <span class="nlm-surname">Jovanović Glavaš</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Panayiotis</span> <span class="nlm-surname">Pafilis</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Sabrina F.</span> <span class="nlm-surname">Samuel</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Johannes</span> <span class="nlm-surname">Müller</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Miguel</span> <span class="nlm-surname">Vences</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Alexander P.</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Pedro</span> <span class="nlm-surname">Beltran-Alvarez</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Kenneth B.</span> <span class="nlm-surname">Storey</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">726240; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/726240 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/726240v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/726240v1&amp;title=Abiotic%20environmental%20adaptation%20in%20vertebrates%20is%20characterized%20by%20functional%20genomic%20constraint" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/726240v1&amp;title=Abiotic%20environmental%20adaptation%20in%20vertebrates%20is%20characterized%20by%20functional%20genomic%20constraint" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/726240v1&amp;text=Abiotic%20environmental%20adaptation%20in%20vertebrates%20is%20characterized%20by%20functional%20genomic%20constraint" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/726240v1&amp;title=Abiotic%20environmental%20adaptation%20in%20vertebrates%20is%20characterized%20by%20functional%20genomic%20constraint" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/726240v1&amp;t=Abiotic%20environmental%20adaptation%20in%20vertebrates%20is%20characterized%20by%20functional%20genomic%20constraint" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/726240v1&amp;title=Abiotic%20environmental%20adaptation%20in%20vertebrates%20is%20characterized%20by%20functional%20genomic%20constraint" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/726240v1&amp;title=Abiotic%20environmental%20adaptation%20in%20vertebrates%20is%20characterized%20by%20functional%20genomic%20constraint" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="861192" id="node861192--3" data-pisa="biorxiv;726240v1" data-pisa-master="biorxiv;726240" data-seqnum="861192" data-apath="/biorxiv/early/2019/08/08/726240.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Abiotic environmental adaptation in vertebrates is characterized by functional genomic constraint</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Katharina C. Wollenberg</span> <span class="nlm-surname">Valero</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Joan</span> <span class="nlm-surname">Garcia-Porta</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Iker</span> <span class="nlm-surname">Irisarri</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Lauric</span> <span class="nlm-surname">Feugere</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Adam</span> <span class="nlm-surname">Bates</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Sebastian</span> <span class="nlm-surname">Kirchhof</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Olga</span> <span class="nlm-surname">Jovanović Glavaš</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Panayiotis</span> <span class="nlm-surname">Pafilis</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Sabrina F.</span> <span class="nlm-surname">Samuel</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Johannes</span> <span class="nlm-surname">Müller</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Miguel</span> <span class="nlm-surname">Vences</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Alexander P.</span> <span class="nlm-surname">Turner</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Pedro</span> <span class="nlm-surname">Beltran-Alvarez</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Kenneth B.</span> <span class="nlm-surname">Storey</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">726240; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/726240 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Abiotic%20environmental%20adaptation%20in%20vertebrates%20is%20characterized%20by%20functional%20genomic%20constraint&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Valero%2C+Katharina+C.+Wollenberg&amp;rft.au=Garcia-Porta%2C+Joan&amp;rft.au=Irisarri%2C+Iker&amp;rft.au=Feugere%2C+Lauric&amp;rft.au=Bates%2C+Adam&amp;rft.au=Kirchhof%2C+Sebastian&amp;rft.au=Jovanovi%C4%87+Glava%C5%A1%2C+Olga&amp;rft.au=Pafilis%2C+Panayiotis&amp;rft.au=Samuel%2C+Sabrina+F.&amp;rft.au=M%C3%BCller%2C+Johannes&amp;rft.au=Vences%2C+Miguel&amp;rft.au=Turner%2C+Alexander+P.&amp;rft.au=Beltran-Alvarez%2C+Pedro&amp;rft.au=Storey%2C+Kenneth+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/861192/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/861192/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/861192/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/861192/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/861192/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/861192/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/861192/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/861192/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/861192/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/861192/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/861192/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/861192/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/726240v1&amp;count=horizontal&amp;via=&amp;text=Abiotic%20environmental%20adaptation%20in%20vertebrates%20is%20characterized%20by%20functional%20genomic%20constraint&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/726240v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/726240v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/726240v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
