<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2016/06/15/059261.atom"/>
  <meta name="HW.pisa" content="biorxiv;059261v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Genome resources for climate-resilient cowpea, an essential crop for food security"/>
  <meta name="DC.Identifier" content="10.1101/059261"/>
  <meta name="DC.Date" content="2016-06-15"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2016, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Cowpea ( Vigna unguiculata L. Walp.) is a legume crop that is resilient to hot and drought-prone climates, and a primary source of protein in sub-Saharan Africa and other parts of the developing world. However, genome resources for cowpea have lagged behind most other major crop plants. Here we describe foundational genome resources and their application to analysis of germplasm currently in use in West African breeding programs. Resources developed from the African cultivar IT97K-499-35 include bacterial artificial chromosome (BAC) libraries and a BAC-based physical map, assembled sequences from 4,355 BACs, as well as a whole-genome shotgun (WGS) assembly. These resources and WGS sequences of an additional 36 diverse cowpea accessions supported the development of a genotyping assay for over 50,000 SNPs, which was then applied to five biparental RIL populations to produce a consensus genetic map containing 37,372 SNPs. This genetic map enabled the anchoring of 100 Mb of WGS and 420 Mb of BAC sequences, an exploration of genetic diversity along each linkage group, and clarification of macrosynteny between cowpea and common bean. The genomes of West African breeding lines and landraces have regions of marked depletion of diversity, some of which coincide with QTL that may be the result of artificial selection or environmental adaptation. The new publicly available resources and knowledge help to define goals and accelerate the breeding of improved varieties to address food security issues related to limited-input small-holder farming and climate stress."/>
  <meta name="DC.Contributor" content="María Muñoz-Amatriaín"/>
  <meta name="DC.Contributor" content="Hamid Mirebrahim"/>
  <meta name="DC.Contributor" content="Pei Xu"/>
  <meta name="DC.Contributor" content="Steve I. Wanamaker"/>
  <meta name="DC.Contributor" content="MingCheng Luo"/>
  <meta name="DC.Contributor" content="Hind Alhakami"/>
  <meta name="DC.Contributor" content="Matthew Alpert"/>
  <meta name="DC.Contributor" content="Ibrahim Atokple"/>
  <meta name="DC.Contributor" content="Benoit Joseph Batieno"/>
  <meta name="DC.Contributor" content="Ousmane Boukar"/>
  <meta name="DC.Contributor" content="Serdar Bozdag"/>
  <meta name="DC.Contributor" content="Ndiaga Cisse"/>
  <meta name="DC.Contributor" content="Issa Drabo"/>
  <meta name="DC.Contributor" content="Jeffrey D. Ehlers"/>
  <meta name="DC.Contributor" content="Andrew Farmer"/>
  <meta name="DC.Contributor" content="Christian Fatokun"/>
  <meta name="DC.Contributor" content="Yong Q. Gu"/>
  <meta name="DC.Contributor" content="Yi-Ning Guo"/>
  <meta name="DC.Contributor" content="Bao-Lam Huynh"/>
  <meta name="DC.Contributor" content="Scott A. Jackson"/>
  <meta name="DC.Contributor" content="Francis Kusi"/>
  <meta name="DC.Contributor" content="Cynthia T. Lawley"/>
  <meta name="DC.Contributor" content="Mitchell R. Lucas"/>
  <meta name="DC.Contributor" content="Yaqin Ma"/>
  <meta name="DC.Contributor" content="Michael P. Timko"/>
  <meta name="DC.Contributor" content="Jiajie Wu"/>
  <meta name="DC.Contributor" content="Frank You"/>
  <meta name="DC.Contributor" content="Philip A. Roberts"/>
  <meta name="DC.Contributor" content="Stefano Lonardi"/>
  <meta name="DC.Contributor" content="Timothy J. Close"/>
  <meta name="article:published_time" content="2016-06-15"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Genome resources for climate-resilient cowpea, an essential crop for food security"/>
  <meta name="citation_abstract" lang="en" content="&lt;;h3&gt;;SUMMARY&lt;;/h3&gt;; &lt;;p&gt;;Cowpea (&lt;;i&gt;;Vigna unguiculata&lt;;/i&gt;; L. Walp.) is a legume crop that is resilient to hot and drought-prone climates, and a primary source of protein in sub-Saharan Africa and other parts of the developing world. However, genome resources for cowpea have lagged behind most other major crop plants. Here we describe foundational genome resources and their application to analysis of germplasm currently in use in West African breeding programs. Resources developed from the African cultivar IT97K-499-35 include bacterial artificial chromosome (BAC) libraries and a BAC-based physical map, assembled sequences from 4,355 BACs, as well as a whole-genome shotgun (WGS) assembly. These resources and WGS sequences of an additional 36 diverse cowpea accessions supported the development of a genotyping assay for over 50,000 SNPs, which was then applied to five biparental RIL populations to produce a consensus genetic map containing 37,372 SNPs. This genetic map enabled the anchoring of 100 Mb of WGS and 420 Mb of BAC sequences, an exploration of genetic diversity along each linkage group, and clarification of macrosynteny between cowpea and common bean. The genomes of West African breeding lines and landraces have regions of marked depletion of diversity, some of which coincide with QTL that may be the result of artificial selection or environmental adaptation. The new publicly available resources and knowledge help to define goals and accelerate the breeding of improved varieties to address food security issues related to limited-input small-holder farming and climate stress.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2016/01/01"/>
  <meta name="citation_mjid" content="biorxiv;059261v1"/>
  <meta name="citation_id" content="059261v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/059261v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/059261v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/059261v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261.full.pdf"/>
  <meta name="citation_doi" content="10.1101/059261"/>
  <meta name="citation_num_pages" content="40"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="059261"/>
  <meta name="citation_author" content="María Muñoz-Amatriaín"/>
  <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California"/>
  <meta name="citation_author_email" content="maria.munoz-amatriain@ucr.edu"/>
  <meta name="citation_author_email" content="timothy.close@ucr.edu"/>
  <meta name="citation_author" content="Hamid Mirebrahim"/>
  <meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of California"/>
  <meta name="citation_author" content="Pei Xu"/>
  <meta name="citation_author_institution" content="Institute of Vegetables, Zhejiang Academy of Agricultural Sciences"/>
  <meta name="citation_author" content="Steve I. Wanamaker"/>
  <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California"/>
  <meta name="citation_author" content="MingCheng Luo"/>
  <meta name="citation_author_institution" content="Department of Plant Sciences, University of California"/>
  <meta name="citation_author" content="Hind Alhakami"/>
  <meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of California"/>
  <meta name="citation_author" content="Matthew Alpert"/>
  <meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of California"/>
  <meta name="citation_author" content="Ibrahim Atokple"/>
  <meta name="citation_author_institution" content="Council for Scientific and Industrial Research, Savanna Agricultural Research Institute"/>
  <meta name="citation_author" content="Benoit Joseph Batieno"/>
  <meta name="citation_author_institution" content="Institut de l’Environnement et de Recherches Agricoles"/>
  <meta name="citation_author" content="Ousmane Boukar"/>
  <meta name="citation_author_institution" content="International Institute of Tropical Agriculture"/>
  <meta name="citation_author" content="Serdar Bozdag"/>
  <meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of California"/>
  <meta name="citation_author_institution" content="Department of Mathematics, Statistics and Computer Science, Marquette University"/>
  <meta name="citation_author" content="Ndiaga Cisse"/>
  <meta name="citation_author_institution" content="Institut Sénégalais de Recherches Agricoles"/>
  <meta name="citation_author" content="Issa Drabo"/>
  <meta name="citation_author_institution" content="Institut de l’Environnement et de Recherches Agricoles"/>
  <meta name="citation_author" content="Jeffrey D. Ehlers"/>
  <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California"/>
  <meta name="citation_author_institution" content="The Bill &amp;; Melinda Gates Foundation"/>
  <meta name="citation_author" content="Andrew Farmer"/>
  <meta name="citation_author_institution" content="National Center for Genome Resources"/>
  <meta name="citation_author" content="Christian Fatokun"/>
  <meta name="citation_author_institution" content="International Institute of Tropical Agriculture"/>
  <meta name="citation_author" content="Yong Q. Gu"/>
  <meta name="citation_author_institution" content="USDA-ARS Western Regional Research Center"/>
  <meta name="citation_author" content="Yi-Ning Guo"/>
  <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California"/>
  <meta name="citation_author" content="Bao-Lam Huynh"/>
  <meta name="citation_author_institution" content="Department of Nematology, University of California"/>
  <meta name="citation_author" content="Scott A. Jackson"/>
  <meta name="citation_author_institution" content="Center for Applied Genetic Technologies, University of Georgia"/>
  <meta name="citation_author" content="Francis Kusi"/>
  <meta name="citation_author_institution" content="Council for Scientific and Industrial Research, Savanna Agricultural Research Institute"/>
  <meta name="citation_author" content="Cynthia T. Lawley"/>
  <meta name="citation_author_institution" content="Illumina, Inc."/>
  <meta name="citation_author" content="Mitchell R. Lucas"/>
  <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California"/>
  <meta name="citation_author" content="Yaqin Ma"/>
  <meta name="citation_author_institution" content="Department of Plant Sciences, University of California"/>
  <meta name="citation_author" content="Michael P. Timko"/>
  <meta name="citation_author_institution" content="Department of Biology, University of Virginia"/>
  <meta name="citation_author" content="Jiajie Wu"/>
  <meta name="citation_author_institution" content="Department of Plant Sciences, University of California"/>
  <meta name="citation_author" content="Frank You"/>
  <meta name="citation_author_institution" content="Department of Plant Sciences, University of California"/>
  <meta name="citation_author_institution" content="Agriculture and Agri-Food Canada"/>
  <meta name="citation_author" content="Philip A. Roberts"/>
  <meta name="citation_author_institution" content="Department of Nematology, University of California"/>
  <meta name="citation_author" content="Stefano Lonardi"/>
  <meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of California"/>
  <meta name="citation_author" content="Timothy J. Close"/>
  <meta name="citation_author_institution" content="Department of Botany and Plant Sciences, University of California"/>
  <meta name="citation_author_email" content="maria.munoz-amatriain@ucr.edu"/>
  <meta name="citation_author_email" content="timothy.close@ucr.edu"/>
  <meta name="citation_reference" content="Bankevich, A., Nurk, S., Antipov, D., Gurevich, A.A., Dvorkin, M., Kulikov, A.S., et al. (2012) SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comp. Biol. 19, 455–477."/>
  <meta name="citation_reference" content="Bapna, C.S., Joshi, S.N., and Kabria, M.M. (1972) Correlation studies on yield and agronomic characters in cowpea. Indian J. Agron. 17, 321–324."/>
  <meta name="citation_reference" content="Botstein, D., White, R.L., Skolnick, M., and Davis, R.W. (1980) Construction of a genetic linkage map in man using restriction fragment length polymorphisms. Am. J. Hum. Genet. 32, 314–331."/>
  <meta name="citation_reference" content="Bozdag, S., Close, T., and Lonardi, S. (2013) A graph-theoretical approach to the selection of the minimum tiling path from a physical map. IEEE/ACM Trans. Comput. Biol. Bioinform. 10, 352–360."/>
  <meta name="citation_reference" content="Bradbury, P.J., Zhang, Z., Kroon, D.E., Casstevens, T.M., Ramdoss, Y., and Buckler, E.S. (2007) TASSEL: software for association mapping of complex traits in diverse samples. Bioinformatics 23, 2633–5."/>
  <meta name="citation_reference" content="Chen, X., Laudeman, T.W., Rushton, P.J., Spraggins, T.A., and Timko, M.P. (2007) CGKB: an annotation knowledge base for cowpea (Vigna unguiculata L.) methylation filtered genomic genespace sequences. BMC Bioinformatics 8, 129."/>
  <meta name="citation_reference" content="Ewing, B., and Green, F. (1998) Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res. 8, 186–94."/>
  <meta name="citation_reference" content="Fang, J., Chao, C-C.T., Roberts, P.A., and Ehlers, J.D. (2007) Genetic diversity of cowpea [Vigna unguiculata (L.) Walp.] in four West African and USA breeding programs as determined by AFLP analysis. Genet. Resour. Crop Evol. 54, 1197–1209."/>
  <meta name="citation_reference" content="Garrison, E., and Marth, G. (2012) Haplotype-based variant detection from short-read sequencing. arXiv preprint arXiv:1207.3907."/>
  <meta name="citation_reference" content="Gu, Y.Q., Ma, Y., Huo, N., Vogel, J.P., You, F.M., Lazo, G.R., et al. (2009) A BAC-based physical map of Brachypodium distachyon and its comparative analysis with rice and wheat. BMC Genomics 10, 496."/>
  <meta name="citation_reference" content="Hamblin, M.T., Buckler, E.S., and Jannink, J.L. (2011) Population genetics of genomics-based crop improvement methods. Trends Genet. 27, 98–106."/>
  <meta name="citation_reference" content="Huynh, B.L. Close, T.J., Roberts, P.A., Hubc, Z., Wanamaker, S., Lucas, M.R., et al. (2013) Gene pools and the genetic architecture of domesticated cowpea. Plant Genome 6."/>
  <meta name="citation_reference" content="Iwata-Otsubo, A., Lin, J.Y., Gill, N., and Jackson, S.A. (2016) Highly distinct chromosomal structures in cowpea (Vigna unguiculata), as revealed by molecular cytogenetic analysis. Chromosome Res. 24, 197–216."/>
  <meta name="citation_reference" content="Korte, A., and Farlow, A. (2013) The advantages and limitations of trait analysis with GWAS: a review. Plant Methods 9, 29."/>
  <meta name="citation_reference" content="Krzywinsk, M., Schein, J., Birol, I., Connors, J., Gascoyne, R., Horsman, D., et al. (2009) Circos: an information aesthetic for comparative genomics. Genome Res. 19, 1639–45."/>
  <meta name="citation_reference" content="Kujur, A., Bajaj, D., Upadhyaya, H.D., Das, S., Ranjan, R., Shree, T., et al. (2015) A genome-wide SNP scan accelerates trait-regulatory genomic loci identification in chickpea. Sci. Rep. 5, 11166."/>
  <meta name="citation_reference" content="Kurtz, S., Phillippy, A., Delcher, A.L., Smoot, M., Shumway, M., Antonescu, C., and Salzberg, S.L. (2004) Versatile and open software for comparing large genomes. Genome Biol. 5, R12."/>
  <meta name="citation_reference" content="Li, H. and Durbin, R. (2009a) Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25, 1754–60."/>
  <meta name="citation_reference" content="Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., et al. (2009b) The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–9."/>
  <meta name="citation_reference" content="Lonardi, S., Duma, D., Alpert, M., Cordero, F., Beccuti, M., Bhat, P.R., et al. (2013) Combinatorial pooling enables selective sequencing of the barley gene space. PLoS Comput Biol. 9, e1003010."/>
  <meta name="citation_reference" content="Lonardi, S., Mirebrahim, H., Wanamaker, S., Alpert, M., Ciardo, G., Duma, D., and Close, T.J. (2015) When less is more: ‘slicing’ sequencing data improves read decoding accuracy and de novo assembly quality. Bioinformatics 31, 2972–80."/>
  <meta name="citation_reference" content="Lorenc, M.T., Hayashi, S., Stiller, J., Lee, H., Manoli, S., Ruperao, P., et al. (2012) Discovery of single nucleotide polymorphisms in complex genomes using SGSautoSNP. Biology (Basel) 1, 370–382."/>
  <meta name="citation_reference" content="Lucas, M.R., Diop, N.N., Wanamaker, S., Ehlers, J.E., Roberts, P. A., and Close, T.J. (2011) Cowpea-soybean synteny clarified through an improved genetic map. Plant Genome 4, 218–225."/>
  <meta name="citation_reference" content="Lucas, M.R., Ehlers, J.D., Huynh, B.L., Diop, N.N., Roberts, P.A., and Close, T.J. (2013a) Markers for breeding heat-tolerant cowpea. Mol. Breeding 31, 529–536."/>
  <meta name="citation_reference" content="Lucas, M.R., Huynh, B.L., da Silva Vinholes, P., Cisse, N., Drabo, I., Ehlers, J.D., et al. (2013b) Association studies and legume synteny reveal haplotypes determining seed size in Vigna unguiculata. Front. Plant Sci. 4, 95."/>
  <meta name="citation_reference" content="Luo M.C., Thomas, C., You, F.M., Hsiao, J., Ouyang, S., Buell, C.R., et al. (2003) High-throughput fingerprinting of bacterial artificial chromosomes using the snapshot labeling kit and sizing of restriction fragments by capillary electrophoresis. Genomics 82, 378–389."/>
  <meta name="citation_reference" content="Luo, R., Liu, B., Xie, Y., Li, Z., Huang, W., Yuan, J., et al. (2012) SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler. GigaScience 1, 18."/>
  <meta name="citation_reference" content="McKenna, A. Hanna, M., Banks, E., Sivachenko, A., Cibulskis, K., Kernytsky, A., et al. (2010) The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297–303."/>
  <meta name="citation_reference" content="Muchero, W., Diop, N.N., Bhat, P.R., Fenton, R.D., Wanamaker, S., Pottorff, M., et al. (2009) A consensus genetic map of cowpea Vigna unguiculata (L) Walp‥ and synteny based on EST-derived SNPs. Proc. Natl. Acad. Sci. U.S.A. 106, 18159–18164."/>
  <meta name="citation_reference" content="Muchero, W., Roberts, P.A., Diop, N.N., Drabo, I., Cisse, N., Close, T.J., et al. (2013) Genetic architecture of delayed senescence, biomass, and grain yield under drought stress in cowpea. PLoS One 8, e70041."/>
  <meta name="citation_reference" content="Pottorff, M.O., Li, G., Ehlers, J.D., Close, T.J., and Roberts, P.A. (2014) Genetic mapping, synteny, and physical location of two loci for Fusarium oxysporum f. sp. tracheiphilum race 4 resistance in cowpea Vigna unguiculata (L.) Walp‥ Mol. Breeding 33, 779– 791."/>
  <meta name="citation_reference" content="Ray, J.D., Dhanapal, A.P., Singh, S.K., Hoyos-Villegas, V., Smith, J.R., Purcell, L.C., et al. (2015) Genome-wide association study of ureide concentration in diverse maturity group IV soybean [Glycine max (L.) Merr.] accessions. G3 5, 2391–403."/>
  <meta name="citation_reference" content="Sakai, H., Naito, K., Takahashi, Y., Sato, T., Yamamoto, T., Muto, I., et al. (2016) The Vigna Genome Server, ‘VigGS’: a genomic knowledge base of the genus Vigna gased on high-quality, annotated genome sequence of the Azuki Bean, Vigna angularis (Willd.) Ohwi &amp;; Ohashi. Plant Cell Physiol. 57, e2."/>
  <meta name="citation_reference" content="Schmutz, J., McClean, P.E., Mamidi, S., Wu, G.A., Cannon, S.B., Grimwood, J., et al. (2014) A reference genome for common bean and genome-wide analysis of dual domestications. Nat. Genet. 46, 707–713."/>
  <meta name="citation_reference" content="Singh, B.B. (2014) Cowpea: the food legume of the 21st century. Crop Science Society of America, Inc. DOI:10.2135/2014.cowpea."/>
  <meta name="citation_reference" content="Soderlund, C. Longden, I., and Mott, R. (2000) FPC: a system for building contigs from restriction fingerprinted clones. Comput. Appl. Biosci. 13, 523–535."/>
  <meta name="citation_reference" content="Thomson, J.A. The role of biotechnology for agricultural sustainability in Africa (2008) Philos. Trans. R. Soc. Lond. B Biol. Sci. 363, 905–913."/>
  <meta name="citation_reference" content="Thornton, P.K., Ericksen, P.J., Herrero, M., and Challinor, A.J. (2014) Climate variability and vulnerability to climate change: a review. Glob. Chang. Biol. 20, 3313–3328."/>
  <meta name="citation_reference" content="Timko, M.P., Rushton, P.J., Laudeman, T.W., Bokowiec, M.T., Chipumuro, E., Cheung, F., et al. (2008) Sequencing and analysis of the gene-rich space of cowpea. BMC Genomics 9, 103."/>
  <meta name="citation_reference" content="Varshney, R.K., Chen, W., Li, Y., Bharti, A.K., Saxena, R.K., Schlueter, J.A., et al. (2012) Draft genome sequence of pigeonpea (Cajanus cajan), an orphan legume crop of resource-poor farmers. Nat. Biotechnol. 30, 83–89."/>
  <meta name="citation_reference" content="Vasconcelos, E.V., de Andrade Fonseca, A.F., Pedrosa-Harand, A., de Andrade Bortoleti, K.C., Benko-Iseppon, A.M., da Costa, A.F., and Brasileiro-Vidal, A.C. (2015) Intra-and interchromosomal rearrangements between cowpea Vigna unguiculata (L.) Walp‥ and common bean (Phaseolus vulgaris L.) revealed by BAC-FISH. Chromosome Res. 23, 253266."/>
  <meta name="citation_reference" content="Wu, Y., Bhat, P.R., Close, T.J., and Lonardi, S. (2008) Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph. PLoS Genet. 4, e1000212."/>
  <meta name="citation_reference" content="Wu, Y., Close, T.J., and Lonardi, S. (2011) Accurate construction of consensus genetic maps via integer linear programming. IEEE/ACM Trans. Comput. Biol. Bioinform. 8, 381–94."/>
  <meta name="citation_reference" content="Xu, P., Wu, X., Munoz-Amatriain, M., Wang, B., Wu, X., Hu, Y., et al. (2016) Discovery of genome regions associated with pod length provides insights into grain-vegetable cowpea diversification. Submitted."/>
  <meta name="citation_reference" content="Yang, K. Tian, Z., Chen, C., Luo, L., Zhao, B., Wang, Z., et al. (2015) Genome sequencing of adzuki bean (Vigna angularis) provides insight into high starch and low fat accumulation and domestication. Proc. Natl. Acad. Sci. U.S.A. 112, 13213–8."/>
  <meta name="twitter:title" content="Genome resources for climate-resilient cowpea, an essential crop for food security"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Cowpea ( Vigna unguiculata L. Walp.) is a legume crop that is resilient to hot and drought-prone climates, and a primary source of protein in sub-Saharan Africa and other parts of the developing world. However, genome resources for cowpea have lagged behind most other major crop plants. Here we describe foundational genome resources and their application to analysis of germplasm currently in use in West African breeding programs. Resources developed from the African cultivar IT97K-499-35 include bacterial artificial chromosome (BAC) libraries and a BAC-based physical map, assembled sequences from 4,355 BACs, as well as a whole-genome shotgun (WGS) assembly. These resources and WGS sequences of an additional 36 diverse cowpea accessions supported the development of a genotyping assay for over 50,000 SNPs, which was then applied to five biparental RIL populations to produce a consensus genetic map containing 37,372 SNPs. This genetic map enabled the anchoring of 100 Mb of WGS and 420 Mb of BAC sequences, an exploration of genetic diversity along each linkage group, and clarification of macrosynteny between cowpea and common bean. The genomes of West African breeding lines and landraces have regions of marked depletion of diversity, some of which coincide with QTL that may be the result of artificial selection or environmental adaptation. The new publicly available resources and knowledge help to define goals and accelerate the breeding of improved varieties to address food security issues related to limited-input small-holder farming and climate stress."/>
  <meta name="og-title" property="og:title" content="Genome resources for climate-resilient cowpea, an essential crop for food security"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/059261v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Cowpea ( Vigna unguiculata L. Walp.) is a legume crop that is resilient to hot and drought-prone climates, and a primary source of protein in sub-Saharan Africa and other parts of the developing world. However, genome resources for cowpea have lagged behind most other major crop plants. Here we describe foundational genome resources and their application to analysis of germplasm currently in use in West African breeding programs. Resources developed from the African cultivar IT97K-499-35 include bacterial artificial chromosome (BAC) libraries and a BAC-based physical map, assembled sequences from 4,355 BACs, as well as a whole-genome shotgun (WGS) assembly. These resources and WGS sequences of an additional 36 diverse cowpea accessions supported the development of a genotyping assay for over 50,000 SNPs, which was then applied to five biparental RIL populations to produce a consensus genetic map containing 37,372 SNPs. This genetic map enabled the anchoring of 100 Mb of WGS and 420 Mb of BAC sequences, an exploration of genetic diversity along each linkage group, and clarification of macrosynteny between cowpea and common bean. The genomes of West African breeding lines and landraces have regions of marked depletion of diversity, some of which coincide with QTL that may be the result of artificial selection or environmental adaptation. The new publicly available resources and knowledge help to define goals and accelerate the breeding of improved varieties to address food security issues related to limited-input small-holder farming and climate stress."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2016-06-15"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Genome resources for climate-resilient cowpea, an essential crop for food security | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-16324 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">SUMMARY</h2>
    <p id="p-3">Cowpea (
     <em>Vigna unguiculata</em> L. Walp.) is a legume crop that is resilient to hot and drought-prone climates, and a primary source of protein in sub-Saharan Africa and other parts of the developing world. However, genome resources for cowpea have lagged behind most other major crop plants. Here we describe foundational genome resources and their application to analysis of germplasm currently in use in West African breeding programs. Resources developed from the African cultivar IT97K-499-35 include bacterial artificial chromosome (BAC) libraries and a BAC-based physical map, assembled sequences from 4,355 BACs, as well as a whole-genome shotgun (WGS) assembly. These resources and WGS sequences of an additional 36 diverse cowpea accessions supported the development of a genotyping assay for over 50,000 SNPs, which was then applied to five biparental RIL populations to produce a consensus genetic map containing 37,372 SNPs. This genetic map enabled the anchoring of 100 Mb of WGS and 420 Mb of BAC sequences, an exploration of genetic diversity along each linkage group, and clarification of macrosynteny between cowpea and common bean. The genomes of West African breeding lines and landraces have regions of marked depletion of diversity, some of which coincide with QTL that may be the result of artificial selection or environmental adaptation. The new publicly available resources and knowledge help to define goals and accelerate the breeding of improved varieties to address food security issues related to limited-input small-holder farming and climate stress.
    </p>
   </div>
   <div class="section" id="sec-1">
    <h2 class="">INTRODUCTION</h2>
    <p id="p-4">Cowpea 
     <em>(Vigna unguiculata</em> (L.) Walp.), native to Africa and a member of the Fabaceae family, is a primary source of protein in sub-Saharan Africa, where it is grown for fresh and dry grains, foliage, and forage. Cowpea is also an important crop in parts of Asia, South America, and the USA (
     <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Singh 2014</a>). Because of its adaptability to harsh conditions, cowpea is a successful crop in arid and semiarid regions where few other crops perform well. Cowpea is important to the nutrition and income of smallholder farmers in Africa, while also contributing to sustainability of the cropping system through fixation of atmospheric nitrogen and prevention of soil erosion. Despite its relevance to agriculture in the developing world and its stress resilience, actual yields of cowpea are much lower than the known yield potential, and cowpea genome resources have lagged behind those developed for other major crop plants.
    </p>
    <p id="p-5">Cowpea is a diploid with a chromosome number 2n = 22 and an estimated genome size of 620 Mb (
     <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen et al. 2007</a>). Its genome shares a high degree of collinearity with other warm season legumes, especially common bean 
     <em>(Phaseolus vulgaris</em> L.) (
     <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Vasconcelos et al. 2015</a>). Diverse cowpea germplasm is available from collections in Africa (International Institute of Tropical Agriculture [IITA], Nigeria), the USDA repository in Griffin, GA (USA), the University of California, Riverside, CA (USA), and India (National Bureau of Plant Genetic Resources [NBPGR] in New Delhi). These collections contain diversity relevant to pests, pathogens, plant architecture, seed characteristics and adaptation to marginal environments. Resources that were developed previously to support adoption of markers for breeding include a 1,536-SNP GoldenGate assay (
     <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Muchero et al. 2009</a>), which has enabled linkage mapping and QTL analysis (e.g. 
     <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lucas et al. 2011</a>; 
     <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Muchero et al. 2013</a>; 
     <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pottorff et al. 2014</a>) as well as an assessment of the diversity of landraces throughout Africa (
     <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huynh et al. 2013</a>).
    </p>
    <p id="p-6">IT97K-499-35, developed at IITA, was released in Nigeria in 2008 as a line that is resistant to most races of the parasitic weed 
     <em>Striga gesnerioides</em> that are prevalent in West Africa. This black-eyed variety has also been released as a cultivar in Mali and Ghana under the names “Djiguiya” and “Songotra”, respectively. Gene space sequences accounting for ~160 Mb of the IT97K-499-35 genome were previously published (
     <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Timko et al. 2008</a>). In addition, 29,728 “unigene” consensus sequences, derived from 183,118 ESTs from cDNA libraries of 17 different cowpea accessions are available in the software HarvEST:Cowpea (harvest.ucr.edu) (
     <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Muchero et al. 2009</a>).
    </p>
    <p id="p-7">Here we present additional resources including a physical map from BACs of IT97K-499-35, sequence assemblies for minimal tiling path (MTP) BACs and from 65x coverage whole genome shotgun (WGS) short-reads, more than 1 million SNPs discovered from sequences of 36 diverse accessions, and an Illumina Cowpea iSelect Consortium Array which represents a publicly accessible resource for screening 51,128 SNPs. These resources have been leveraged for linkage mapping and synteny analysis, used to investigate frequency and distribution of cowpea genome diversity, and used to evaluate breeding materials from four West African breeding programs, which serve one of the most food insecure regions of the world. These genomic resources do not constitute a complete sequence of the cowpea genome, yet they have enabled advances in genetic marker development, assessment of the diversity within breeding programs and comparative genomics.</p>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">RESULTS</h2>
    <div id="sec-3" class="subsection">
     <h3>Physical map and BAC sequencing</h3>
     <p id="p-8">Two BAC libraries were constructed from cowpea accession IT97K-499-35 using restriction enzymes 
      <em>HindIII</em> and 
      <em>MboI</em> (36,864 clones each with 150 kb and 130 kb average clone insert size, respectively). High-quality BAC-end sequences (BES) were generated from 30,343 BACs using the Sanger method. BES had an average read length of 674 bp, a GC content of 37.2%, and accounted for 20.5 Mb. They were included in the WGS assembly described below. For physical mapping, 59,408 BACs (97.9% from 
      <em>HindIII</em> and 63.2% from 
      <em>MboI</em>) were fingerprinted using the method of 
      <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Luo et al. (2003)</a>. After quality filtering, 43,717 clones were assembled into 829 contigs (40,952 BACs) and 2,765 singletons using FPC (
      <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Soderlund et al. 2000</a>). The total number of fingerprints in the physical map represents an equivalent of 11-fold haploid genome coverage. The resulting cowpea physical map is available at 
      <a href="http://phymap.ucdavis.edu/cowpea">http://phymap.ucdavis.edu/cowpea</a>.
     </p>
     <p id="p-9">A total of 4,355 MTP clones were sequenced in combinatorial pools (
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lonardi et al. 2013</a>) using Illumina HiSeq2000. Reads were assigned to individual BACs and then assembled using SPAdes (
      <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bankevich et al. 2012</a>). BAC assemblies had an average N50 of 18.5 kb, an average L50 of 5.7 contigs, and a total length of 496.9 Mb (
      <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The GC content was 34.05%. Analysis of overlap between sequenced BACs provided an estimate of non-redundant genome coverage at 372.8 Mb (~60.1% of the cowpea genome; see Experimental procedures for more details). Sequence comparison revealed that the BAC assemblies contain 17,216 (57.9%) of 29,728 cowpea EST-derived “unigene” consensus sequences available from HarvEST:Cowpea (
      <a href="http://harvest.ucr.edu">http://harvest.ucr.edu</a>). In addition, the BAC sequences had high homology with 15,617 (57.4%) of the 27,197 protein-coding gene models in common bean (
      <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schmutz et al. 2014</a>). These analyses suggest that ~40% of the cowpea genome is missing from the BAC assemblies.
     </p>
     <div id="T1" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/219084/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/219084/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/219084" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 1.</span>
       <span class="caption-title">
        <strong>BAC assembly characteristics and anchoring to the genetic map.</strong>
       </span>
       <p id="p-10" class="first-child">Values for N50 through L90 are averages of all BAC assemblies.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-11">N50: length for which the collection of contigs of that length or longer contains at least half of the sum of the lengths of all contigs in the BAC assembly.</p>
     <p id="p-12">N90: length for which the collection of contigs of that length or longer contains at least 90% of the sum of the lengths of all contigs in the BAC assembly.</p>
     <p id="p-13">L50: minimum number of contigs accounting for more than 50% of the BAC assembly.</p>
     <p id="p-14">L90: minimum number of contigs accounting for more than 90% of the BAC assembly.</p>
     <p id="p-15">Pv.: 
      <em>Phaseolus vulgaris</em>.
     </p>
    </div>
    <div id="sec-4" class="subsection">
     <h3>Whole-genome shotgun sequencing and assembly</h3>
     <p id="p-16">A WGS approach using short-read sequencing was followed to assemble sequences of the cowpea genome. WGS data from IT97K-499-35 included 394 million paired-end short reads for a total of 40.6 Gb of sequence data (~65x coverage) from Illumina GAII, and Illumina HiSeq sequences from one 5 kb long-insert paired end (LIPE) library. These two datasets were assembled using SOAPdenovo (
      <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Luo et al. 2012</a>) together with the Sanger BES described above and the “gene-space” sequences available from 
      <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Timko et al. (2008)</a>. The resulting assembly has over 600,000 scaffolds (97,777 of 1 kb or longer), accounting for 323 Mb of the cowpea genome (724 Mb of total scaffold length including Ns; 
      <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). This highly-fragmented assembly reflects the short length of the reads and the expected highly-repetitive genome; its close-relatives common bean (
      <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Schmutz et al. 2014</a>) and adzuki bean (
      <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yang et al. 2015</a>) are ~45% repetitive. Despite the fragmentation, the assembly yielded high BLAST hits to 97.2% of the available EST-derived cowpea unigenes (
      <a href="http://harvest.ucr.edu">http://harvest.ucr.edu</a>). This number may be an underestimate of the representation of genes in IT97K-499-35 because the 17 cowpea accessions used for the EST libraries may contain genes not present in IT97K-499-35. The WGS assembly also produced BLAST hits to 24,712 common bean gene models, which is 90.9% of the total number of predicted protein-coding loci (
      <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Schmutz et al. 2014</a>). The average GC content of the WGS assembly was 35.96%, similar to other sequenced legumes (
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Varshney et al. 2012</a>; 
      <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Schmutz et al. 2014</a>; 
      <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Yang et al. 2015</a>).
     </p>
     <div id="T2" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/218781/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/218781/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/218781" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 2.</span>
       <p id="p-17" class="first-child">
        <strong>WGS assembly characteristics and anchoring to the genetic map.</strong>
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-18">N50: length for which the collection of scaffolds of that length or longer contains at least half of the sum of the lengths of all scaffolds in the WGS assembly.</p>
     <p id="p-19">N90: length for which the collection of scaffolds of that length or longer contains at least 90% of the sum of the lengths of all scaffolds in the WGS assembly.</p>
     <p id="p-20">L50: minimum number of scaffolds accounting for more than 50% of the WGS assembly.</p>
     <p id="p-21">L90: minimum number of scaffolds accounting for more than 90% of the WGS assembly.</p>
     <p id="p-22">Pv.: 
      <em>Phaseolus vulgaris</em>.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Development of the Cowpea iSelect Consortium Array</h3>
     <p id="p-23">A total of 36 additional cowpea accessions relevant to Africa, China and the USA were shotgun sequenced using Illumina HiSeq 2500 (12.5x average coverage) and aligned to the WGS assembly of IT97K-499-35 to discover SNPs. These accessions were chosen to represent the geographic, phenotypic and genetic diversity of cultivated cowpea (Figure S1; Table S1). An additional set of 12.5x HiSeq data was also produced from IT97K-499-35 and included as a control against spurious SNP calls. The reads were mapped to the reference sequence (Data S1) using BWA (
      <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li et al. 2009a</a>) to generate a. bam file. Then, SAMtools (
      <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li et al. 2009b</a>), SGSautoSNP (
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lorenc et al. 2012</a>) and FreeBayes (
      <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Garrison et al. 2012</a>) were used to generate three overlapping sets of candidate SNPs, from which the intersection yielded about 1 million SNPs. No accession contributed substantially more SNPs than any other, highlighting the broad coverage of diversity within the set of accessions used for SNP discovery. The most distant accession from IT97K-499-35 was UCR779 (differing at 25% of SNP loci) followed closely by the four Chinese accessions (22%-24%). The accessions most closely related to IT97K-499-35 were the IITA breeding lines IT89KD-288, IT93K-503-1 and IT84S-2246 (12%-13%). The set of ~1 million SNPs was filtered to 55,496 SNPs for the design of an Illumina iSelect Consortium Array (see Esperimental procedures for details on filtering criteria). The design also included 1,163 SNPs from the prior GoldenGate Assay (
      <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Muchero et al. 2009</a>) and 60 presumed organelle SNPs, for a total of 56,719 intended SNPs (60,000 assays). From those, 51,128 SNPs (90.1%) were represented in the final product manifest (Data S2). The cowpea iSelect Consortium Array is available from Illumina (Illumina Inc., San Diego, CA; 
      <a href="http://www.illumina.com/areas-of-interest/agrigenomics/consortia.html">http://www.illumina.com/areas-of-interest/agrigenomics/consortia.html</a>).
     </p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Construction of a consensus genetic map for cowpea</h3>
     <p id="p-24">Five bi-parental RIL populations were used to develop a consensus genetic map (Table S2). Monomorphic SNPs and those with an excessive number of missing and/or heterozygous calls were eliminated, as well as individuals that were duplicated or highly heterozygous. The number of lines per population used for mapping ranged from 94 to 135 (Table S2) for a total of 575 RILs. A genetic map was constructed using MSTmap (
      <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wu et al. 2008</a>; 
      <a href="http://mstmap.org/">http://mstmap.org/</a>) at LOD 10 for each RIL population. Linkage groups (LGs) were numbered and oriented based on a previous cowpea consensus map (
      <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lucas et al. 2011</a>). Individual maps and the genotype data used for their construction can be found in Data S3. Two maps (Sanzi x Vita7 and CB27 x IT82E-18) each had two chromosomes separated into two LGs (Table S2 and Data S3) due to regions where parents lack polymorphisms. One region of identity between CB27 and IT82E-18 on LG4 impacted the number of polymorphisms and marker bins, and the total size of that LG in the specific genetic map (
      <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Genetic map sizes varied among the five populations, from 803.4 cM in ZN016 x Zhijiang282 to 917.1 cM in Sanzi x Vita7 (
      <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).
     </p>
     <div id="T3" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/218800/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/218800/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/218800" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 3.</span>
       <p id="p-25" class="first-child">Distribution of SNPs in the individual component maps and the consensus map.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-26">Individual maps were merged into a consensus map using MergeMap (
      <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wu et al. 2011</a>; 
      <a href="http://mergemap.org/">http://mergemap.org/</a>). Equal weight was given to each individual map. MergeMap identified a few conflicts in marker order, which were resolved by deleting a few conflicted markers with priority given to the map with the highest resolution in the particular LG (i.e. more bins). No SNP was placed on different LGs between maps. Since MergeMap’s coordinate calculations for a consensus map are inflated relative to cM distances in individual maps, consensus LG lengths were normalized to the mean cM length from the individual maps. The resulting consensus map contains 37,372 SNP loci mapped to 3,280 bins (
      <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>; Data S4). This is a 34-fold increase in marker density and a four-fold increase in resolution (number of bins) over the consensus map of 
      <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Lucas et al. (2011)</a>. The new consensus map includes 757 SNPs that were included in the prior GoldenGate assay (
      <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Muchero et al. 2009</a>). The map spans 837.11 cM at an average density of one bin per 0.26 cM and 11.4 SNPs per bin. The new consensus map has dense coverage of all eleven cowpea LGs, with 1.85 cM on LG1 being the largest gap (Figure S2; Data S4).
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Syntenic relationships between cowpea and common bean</h3>
     <p id="p-27">Similar to cowpea, common bean is a diploid member of the Phaseoleae tribe with 2n = 22 chromosomes. The iSelect SNP design sequences were compared to 
      <em>P. vulgaris</em> gene models (
      <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Schmutz et al. 2014</a>) to clarify the syntenic relationships of cowpea with this closely related species. The 26,550 SNPs that were mapped in 
      <em>V. unguiculata</em> and matched a 
      <em>P. vulgaris</em> gene model provided a view of synteny (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Six cowpea LGs (VuLG2, VuLG6, VuLG8, VuLG9, VuLG10 and VuLG11) are largely collinear with six common bean pseudomolecules (Pv7, Pv6, Pv9, Pv11, Pv10 and Pv4, respectively), while the rest have synteny mainly with two common bean pseudomolecules (
      <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>; Table S3). From these five cowpea LGs with one-to-two relationships, three (VuLG3, VuLG4, and VuLG7) have a higher number of BLAST hits, and over a longer genome interval, with one 
      <em>P. vulgaris</em> chromosome (Pv3, Pv1 and Pv2, respectively; 
      <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and Table S3). The other two cowpea LGs, VuLG1 and VuLG5, both have their largest block of homologous synteny with Pv8, followed by Pv5 and Pv1, respectively (
      <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>; Table S3).
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Circos illustration of synteny between cowpea linkage groups (VuLG) and common bean pseudomolecules (Pv). SNP frequencies calculated for 2 cM windows and normalized to the total anchored scaffold size (in kb) are shown for the 11 cowpea LG. P. vulgaris gene densities are also shown for 500 kb windows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1065830936" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;div xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;;&gt;;&lt;;span class=&quot;;caption-title&quot;;&gt;;Circos illustration of synteny between cowpea linkage groups (VuLG) and common bean pseudomolecules (Pv).&lt;;/span&gt;; SNP frequencies calculated for 2 cM windows and normalized to the total anchored scaffold size (in kb) are shown for the 11 cowpea LG. &lt;;em&gt;;P. vulgaris&lt;;/em&gt;; gene densities are also shown for 500 kb windows.&lt;;/div&gt;;&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F1.medium.gif" width="440" height="439"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F1.medium.gif" width="440" height="439"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1.</span>
       <span class="caption-title">Circos illustration of synteny between cowpea linkage groups (VuLG) and common bean pseudomolecules (Pv).</span>
       <p id="p-28" class="first-child">SNP frequencies calculated for 2 cM windows and normalized to the total anchored scaffold size (in kb) are shown for the 11 cowpea LG. 
        <em>P. vulgaris</em> gene densities are also shown for 500 kb windows.
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-29">The same numbering scheme for common bean and cowpea chromosomes would facilitate comparative studies between the two species. Adoption of the chromosome numbers of 
      <em>P. vulgaris</em> according to synteny relationships with LGs of cowpea seems sensible, but additional cowpea sequence information will be needed to clarify the relationships between VuLG1 and VuLG5 with Pv1, Pv5 and Pv8. The BAC-FISH analysis by 
      <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Iwata-Otsubo et al. (2016)</a> that correlates the genetic and chromosome maps in cowpea can be used to orient the cowpea genetic map so that it meets the convention of displaying the short arm on top.
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Genetic anchoring of BACs and WGS scaffolds</h3>
     <p id="p-30">The 37,372-SNP consensus map was used to anchor WGS and BAC assemblies to genetic map positions. The iSelect SNP design sequences were used as BLAST queries to search against WGS and BAC sequences, and matches with an e-value = 1e
      <sup>−50</sup> or better were tallied. Assembled sequences were considered anchored to the genetic map if 100% of the matching SNPs mapped to the same LG, and were at most 5 cM apart (Data S5 and S6). The anchored sequences contain 420 Mb of BAC assemblies (
      <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and Data S5) and 100 Mb of the WGS assembly (237 Mb scaffold size including Ns; 
      <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> and Data S6). For BACs, this is an overestimate of the actual genome coverage because BAC sequences have ~23% overlap (see “Physical map and BAC sequencing”), resulting in a reduced estimate of 323 Mb of unique sequences within anchored BACs. Also, observe in 
      <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> that 95.3% of the anchored WGS scaffolds are larger than 1 kb and they comprise 99.1% of the anchored non-N sequence. Thus, the anchored portion of the WGS assembly, which is comprised mainly of 24,342 scaffolds larger than 1 kb among 25,537 anchored scaffolds, contains many fewer fragments than the entire WGS assembly (644,126 scaffolds). This is the outcome of having selected SNPs in the largest WGS contigs as a final criterion in SNP selection (see Experimental procedures).
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Distribution of genetic variation</h3>
     <p id="p-31">The anchoring of WGS scaffolds to the genetic map enabled investigation of the frequency and positional distribution of genetic diversity in the cowpea genetic map. Nearly half of the 1,036,981 SNPs discovered from the 36 diverse cowpea accessions were anchored to the genetic map based on the anchoring of 25,537 WGS scaffolds using mapped iSelect SNPs. This information was used to examine the SNP frequency and distribution across the eleven cowpea LGs. Frequencies were calculated for 2 cM intervals and normalized to the total anchored scaffold size. SNP frequencies were not uniformly distributed across the genetic map (
      <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). LG11 and LG10 had significantly higher SNP frequencies than all other cowpea linkage groups. Relatively higher SNP frequencies were also observed in the distal ends of LG5 and LG9, in the centromeric region of LG7, and toward the ends of LG1. In contrast, LG8 had relatively low SNP diversity (
      <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). There was no clear relationship between the most diverse cowpea genomic regions and the gene-dense syntenic regions of common bean (
      <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>).
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Genetic diversity in West African breeding programs</h3>
     <p id="p-32">Evaluation of the extent and distribution of genetic diversity has important implications for breeding programs and for the conservation of genetic resources. The level of genetic diversity within West African cowpea breeding programs was examined by comparing the diversity among cultivars/breeding lines with the diversity among locally adapted landraces. A total of 146 unique cultivated cowpea accessions were genotyped, including 105 cultivars or breeding lines from the breeding programs of IITA (Nigeria), INERA (Institut de l'Environnement et de Recherches Agricoles, Burkina Faso), ISRA (Institut Senegalais de Recherches Agricoles, Senegal), and CSIR-SARI (Council for Scientific and Industrial Research, Savanna Agricultural Research Institute, Ghana), and 41 landraces collected from these same countries (Table S4). A total of 46,620 SNPs were polymorphic among these accessions and were used for the diversity analysis.</p>
     <p id="p-33">Principal Component Analysis (PCA) was performed to clarify the genetic relationships between accessions. PCA clearly shows that cultivars and breeding lines do not cluster by breeding program, but rather are scattered in the PC plot (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). This indicates that the four West African breeding programs are working with very similar materials, except for somewhat narrower diversity within the Ghana program than in the other three West African programs. Also, landraces were not differentiated from the improved breeding materials, indicating limited impact of breeding on the overall allelic composition. Landraces were less dispersed than cultivars/breeding lines, mostly distributed along the first PC. The landraces from Senegal were the most diverse within the landrace group. Thirteen landraces that were collected in the same geographical area of Burkina Faso clustered together (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), indicating high genetic similarity between them. Ghanaian and Nigerian landraces have sparse representation in this analysis.
     </p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Principal Component Analysis (PCA) of 146 cultivated cowpea accessions from West Africa. The plot on the left (A) displays accessions colored by the type of material (breeding vs. landrace), while the plot on the right (B) show accessions colored by their country of origin." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1065830936" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Principal Component Analysis (PCA) of 146 cultivated cowpea accessions from West Africa.&lt;;/span&gt;; The plot on the left (A) displays accessions colored by the type of material (breeding vs. landrace), while the plot on the right (B) show accessions colored by their country of origin.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F2.medium.gif" width="440" height="165"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F2.medium.gif" width="440" height="165"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2.</span>
       <span class="caption-title">Principal Component Analysis (PCA) of 146 cultivated cowpea accessions from West Africa.</span>
       <p id="p-34" class="first-child">The plot on the left (A) displays accessions colored by the type of material (breeding vs. landrace), while the plot on the right (B) show accessions colored by their country of origin.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-35">Polymorphism information content (PIC; 
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Botstein et al. 1980</a>) was used as an evaluation of genetic diversity within the 146 West African accessions, and between breeding materials and landraces. PIC values ranged from 0.007 to 0.375 (0.375 is the maximum PIC for bialellic markers), with an average of 0.247 when considering all 146 accessions. The average PIC value among cultivars and breeding lines was slightly higher than that in landraces (PIC = 0.242 and 0.234, respectively). However, given the small sample size of local landraces, this may be an inaccurate estimate of the diversity in West African landrace germplasm. The spatial patterns of genetic diversity across the eleven LGs were explored by plotting PIC values for landraces and improved breeding lines against the consensus genetic map (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>; Figure S3). Regions of very low average PIC values (&gt;;0.05) were identified for linkage groups 1, 3, 4, 7 and 8. Many alleles in those regions were fixed (PIC = 0) in breeding materials, landraces, or both (
      <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>; Data S7). Three of those regions coincide with the positions of loci for traits involved in subspecies differentiation [pod length (
      <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Xu et al. 2016</a>), LG1 and LG4; 
      <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>] or environmental adaptation [heat tolerance (
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lucas et al. 2013a</a>), LG3; 
      <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>], whereas traits associated with other low PIC regions are unknown (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>; Figure S3; 
      <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). A moderate reduction of diversity was observed in a region on LG5 where a domestication-related QTL, 
      <em>Css-1</em>, involved in seed size (
      <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lucas et al. 2013b</a>) has been mapped (Figure S3).
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Diversity in West African cultivated cowpeas across linkage groups 1, 3, and 8. Blue lines represent polymorphism information content (PIC) values calculated in cultivars and breeding lines, while orange lines represent PIC values in local landraces from Burkina Faso, Ghana, Nigeria and Senegal. PIC was averaged across a sliding window of 5 genetic bins with a step of one bin. Arrows indicate regions with a markedly depletion of genetic diversity in one or both types of materials." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1065830936" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;&lt;;span xmlns=&quot;;http://www.w3.org/1999/xhtml&quot;; class=&quot;;caption-title&quot;;&gt;;Diversity in West African cultivated cowpeas across linkage groups 1, 3, and 8.&lt;;/span&gt;; Blue lines represent polymorphism information content (PIC) values calculated in cultivars and breeding lines, while orange lines represent PIC values in local landraces from Burkina Faso, Ghana, Nigeria and Senegal. PIC was averaged across a sliding window of 5 genetic bins with a step of one bin. Arrows indicate regions with a markedly depletion of genetic diversity in one or both types of materials.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F3.medium.gif" width="440" height="314"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F3.medium.gif" width="440" height="314"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2016/06/15/059261/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3.</span>
       <span class="caption-title">Diversity in West African cultivated cowpeas across linkage groups 1, 3, and 8.</span>
       <p id="p-36" class="first-child">Blue lines represent polymorphism information content (PIC) values calculated in cultivars and breeding lines, while orange lines represent PIC values in local landraces from Burkina Faso, Ghana, Nigeria and Senegal. PIC was averaged across a sliding window of 5 genetic bins with a step of one bin. Arrows indicate regions with a markedly depletion of genetic diversity in one or both types of materials.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="T4" class="table pos-float">
      <div class="table-inline table-callout-links">
       <div class="callout">
        <span>View this table:</span>
        <ul class="callout-links">
         <li class="view-inline first">
          <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/218884/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a>
         </li>
         <li class="view-popup">
          <a href="/highwire/markup/218884/expansion?width=1000&amp;;height=500&amp;;iframe=true&amp;;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a>
         </li>
         <li class="download-ppt last">
          <a href="/highwire/powerpoint/218884" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a>
         </li>
        </ul>
       </div>
      </div>
      <div class="table-caption">
       <span class="table-label">Table 4.</span>
       <span class="caption-title">Regions showing the lowest genetic diversity in West African improved breeding lines, local landraces, or both.</span>
       <p id="p-37" class="first-child">Minimum Polymorphism Information Content (PIC) values per bin are shown for the breeding and landrace group, as well as the number of SNPs contained in those regions and any QTL previously identified in those regions.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-38">Patterns of polymorphism in breeding materials were similar to those in landraces throughout most of the genetic map. This is consistent with their close genetic relationships illustrated by the PCA analysis (
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). However, there were some notable differences. The most distinct difference was in LG8, where West African landraces showed very low levels of genetic diversity while improved breeding materials exhibited very high PIC values (
      <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), suggesting that additional diversity may have been introduced into the breeding programs for one or more traits in this region.
     </p>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">DISCUSSION</h2>
    <p id="p-39">Increase in climate variability is projected to have the greatest negative consequences on agricultural and human systems in the tropical and subtropical developing world, aggravating food insecurity in already vulnerable populations (
     <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Thornton et al. 2014</a>). Cowpea is a relatively drought and heat tolerant crop that provides protein to nearly 200 million Africans and cash income to smallholder farmers (
     <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Thomson et al. 2008</a>). The limited availability of genome resources for cowpea has contributed to the relatively slow development of higher yielding varieties adapted to tolerate abiotic and biotic stresses. This report presents 323 Mb of WGS and 497 Mb of BAC sequence information, a tool to simultaneously test 51,128 single nucleotide variants, and a high-density genetic map providing coordinates for most of those sequences and variants. Application of these resources can be made for genome-wide association studies (GWAS) of cowpea germplasm to discover favorable alleles for simple and complex traits, as already being conducted in other legume crops (e.g. 
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kujur et al. 2015</a>; 
     <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ray et al. 2015</a>). Useful variation can then be connected to assembled genome sequences-including BACs-annotated for 
     <em>P. vulgaris</em> syntenic gene models, thereby increasing the precision and speed of cowpea improvement.
    </p>
    <p id="p-40">One of the biggest obstacles in comparing and using results obtained by different research groups is the lack of a common nomenclature for cowpea linkage groups. With a high SNP coverage of the genome and connections to cowpea genome sequences, this study provides the basis for a unified chromosome nomenclature for the cowpea research community. Such common nomenclature could adopt the 
     <em>P. vulgaris</em> chromosome numbering on the basis of synteny comparisons between both species as well as cytogenetic studies in cowpea (
     <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Iwata-Otsubo et al. 2016</a>) and between cowpea and common bean (
     <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Vasconcelos et al. 2015</a>). While several cowpea LGs are largely syntenic with one 
     <em>P. vulgaris</em> chromosome, further resolution is needed to satisfy a single nomenclature for those LGs whose syntenic relationships with common bean are less clear. The goal would be to extend a standard chromosome numbering to other diploid 
     <em>Vigna</em> species whose genomes have been sequenced and are integrated into a genome database (
     <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sakai et al. 2016</a>). This would facilitate the transfer of genomic information on target traits from one Fabaceae species to another.
    </p>
    <p id="p-41">West Africa is the region with the largest production and consumption of cowpea in the world (FAOSTAT 2012, 
     <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Singh 2014</a>). Evaluating the wealth of genetic diversity present in the West African breeding germplasm is important to manage breeding programs and assure future genetic gains. By applying the Cowpea i Select Consortium Array to 146 breeding lines and landraces, we provided a useful overview of genetic variability in West African cultivated germplasm. We found moderately high levels of genetic diversity in breeding materials (PIC ~ 0.25), which are similar to those present in regional landraces. This indicates that, at the genome scale, the 40+ years of breeding practices in the region did not deplete diversity available in the West African breeding germplasm. Introgression of Asian and US germplasm into West African breeding lines was reported previously (
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fang et al. 2007</a>). This would explain the relatively high levels of diversity in breeding materials and their more extended distribution along PC2 (
     <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). The common agro-ecological zones which extend across cowpea production areas of the four included countries of Burkina Faso, Ghana, Nigeria and Senegal facilitates exchange of germplasm between breeding programs and supports their genetic relatedness. The influence of the IITA, Nigeria breeding program, in being a regional distributor of new breeding materials during the last few decades is difficult to quantify, but this has no doubt contributed significantly to the overall similarity of the breeding materials across these West African national breeding programs. Even though the sample size is relatively small, the lack of strong population structure in this West African dataset would be beneficial for identifying true marker-trait associations via GWAS, provided that phenotypic diversity is present for the traits of interest (
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hamblin et al. 2011</a>; 
     <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Korte and Farlow 2013</a>).
    </p>
    <p id="p-42">Average PIC values should be interpreted cautiously because patterns of polymorphism vary across LGs. In fact, although the overall genetic diversity within the West African breeding population is relatively high, we identified genomic regions of diversity depletion. Those regions may contain favorable alleles for important traits that became fixed during domestication and breeding selection. The lowest PIC values in LG1 coincide with the position of SNPs associated with pod length in Chinese germplasm of 
     <em>V. unguiculata</em> subspecies 
     <em>sesquipedalis</em> (
     <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Xu et al. 2016</a>). One interpretation could be that there has been selection for shorter-podded forms in West Africa, a trait which correlates with bushy plant architecture and increased pod number per plant in cultivated dry grain cowpeas (
     <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bapna et al. 1972</a>). Also, a previously reported QTL for heat tolerance 
     <em>(Cht-5</em>) coincides with a very low-diversity region of LG3 (
     <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lucas et al. 2013a</a>). Favorable alleles at this QTL were donated by the line IT82E-18, the African parent of the RIL population (
     <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Lucas et al. 2013a</a>; 
     <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). The low diversity in this region of LG3 may reflect selection at this genomic region for better yield performance of West African cowpeas under higher growing season temperatures. A more complete analysis of diversity in cultivated and wild cowpea germplasm is required to further test these hypotheses of trait association with genome regions of low genetic diversity, and to reveal if other low-diversity regions contain traits involved in domestication or crop improvement.
    </p>
    <p id="p-43">The release of PIC values for each SNP (Supplementary Table 9) is another valuable resource stemming from this work. PIC values can be used as criteria for selecting efficient subsets of markers for conversion to other platforms. Customized, maximally informative subsets of markers have numerous applications including routine tests of seed purity, validation of germplasm fidelity, verification of successful crosses and guidance of progeny selection in later generations during trait introgression into preferred backgrounds via backcrossing.</p>
    <p id="p-44">The new cowpea genome resources must be easily accessed if they are to be widely utilized for basic research and agricultural development. All information presented in this manuscript, including SNPs, BAC and WGS sequences, 
     <em>Phaseolus vulgaris</em> gene annotations, as well as genetic anchoring information are available through the online interface HarvEST:Web (
     <a href="http://harvest-web.org/hweb/utilmenu.wc">http://harvest-web.org/hweb/utilmenu.wc</a>) or in the latest version of HarvEST:Cowpea (
     <a href="http://harvest.ucr.edu">http://harvest.ucr.edu</a>). In addition, a cowpea-common bean synteny viewer has been implemented in HarvEST:Cowpea, enabling facile comparisons between these two closely related legume crops.
    </p>
   </div>
   <div class="section" id="sec-12">
    <h2 class="">EXPERIMENTAL PROCEDURES</h2>
    <div id="sec-13" class="subsection">
     <h3>Physical mapping and BAC-end sequencing</h3>
     <p id="p-45">Cowpea accession IT97K-499-35 was grown for three generations by single seed descent and then increased to provide a supply of seed for DNA isolation. The material was screened with the Illumina GoldenGate assgay (
      <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Muchero et al. 2009</a>) to establish that homozygosity was attained. Young seedling leaves were harvested at UCR and shipped on dry ice to Amplicon Express (Pullman, WA) for purification of nuclei and extraction of mainly nuclear DNA. Two BAC libraries were then constructed by Amplicon Express (Pullman, WA) from high molecular weight DNA of cowpea accession IT97K-499-35 using restriction enzymes 
      <em>HindIII</em> and 
      <em>MboI</em>. After partial digestion with restriction enzymes, high MW cowpea DNA fragments were ligated with 
      <em>HindIII</em> or 
      <em>BamHI</em> linearized BAC vector pCC1. Ligated DNA molecules were introduced into 
      <em>Escherichia coli</em> DH10B cells by electroporation and plated on LB agar containing 12.5 mg/ml chloramphenicol, 0.5 mM IPTG and 40 mg/ml X-Gal and cultured overnight. White colonies were picked and inoculated into 384-well plates containing LB freezing buffer. Cultures were incubated at 37°C for 24 h with aerations, and then stored at-80°C.The libraries contained 36,864 clones each, with average insert sizes of 150 kb for the 
      <em>HindIII</em> library and 130 kb for the 
      <em>Mbo</em>I library.
     </p>
     <p id="p-46">BAC clones from the two libraries (36,096 from 
      <em>HindIII</em> and 23,312 from 
      <em>MboI</em>) were fingerprinted using the SNaPshot-based fingerprinting procedure (
      <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Luo et al. 2003</a>). BAC DNAs were simultaneously digested with five restriction enzymes (BamH1, EcoRI, 
      <em>XbaI, XhoI</em>, and HaeIII), and then labeled with the SNaPshot labeling kit (
      <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Luo et al. 2003</a>). The fragments were sized on an ABI3730XL instrument with the GS1200Liz size-standard (
      <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gu et al. 2009</a>). Fragment sizes in the range of 100-1000 bp were compiled for computational assembly. After removing substandard fingerprints, potential cross contamination and clones with less than 40 total fragments, fingerprints from 43,717 clones (73.6%) were used for an initial contig assembly using the FPC software (
      <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Soderlund et al. 2000</a>). This initial assembly was performed with a relatively high stringency (1×10
      <sup>−45</sup>) to minimize co-assembly of clones from unrelated regions of the genome. The “DQer” function of the FPC software was used for second stage assembly by disassembling contigs containing more than 15% questionable clones. The “Single-to-End” and “End-to-End” merging function of FPC was used for a final, third stage assembly by stepwise decreases of assembly stringency based on Sulston score cutoff values (down to 1×10
      <sup>−35</sup>). Finally, the 10% largest contigs were subjected to manual editing, examining with CB map analysis and disjoining contigs with CB analysis results at 1×10
      <sup>−30</sup>.
     </p>
     <p id="p-47">The same BAC DNA used for fingerprinting was also used for BES. BAC clones were sequenced using pIndigoBAC5 Reverse End-Sequencing primer (5′ TACGCCAAGCTATTTAGGTGAGA 3′) and BigDye terminator chemistry (Applied Biosystems) on an ABI3730XL automated sequencer (Applied Biosystems). Raw sequence reads were trimmed with Phred program using a quality score of 20 (
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Ewing and Green, 1998</a>). BES from vector sequences, E 
      <em>coli</em>, mitochondria and chloroplasts were identified using BLASTN. The chloroplast sequences of common bean (DQ886273.1), soybean (DQ317523), 
      <em>Medicago truncatula</em> (AC093544), 
      <em>Lotus japonicus</em> (AP002983), and mitochondrial DNA sequences of Arabidopsis (Y08501.2) and rice (DQ167399.1) were used to identify organelle contaminations. The resulting high quality BES were then processed with the RepeatMasker program (
      <a href="http://www.repeatmasker.org">www.repeatmasker.org</a>) to identify characterized repeats. Cowpea BES with more than 80% of the sequence length showing homology to known repeats were removed, otherwise the BES were kept but the repetitive region was marked using letter N. Self-comparisons were conducted with the RepeatMasker processed sequences to further filter the cowpea-specific repeat elements.
     </p>
    </div>
    <div id="sec-14" class="subsection">
     <h3>MTP sequencing and BAC assembly</h3>
     <p id="p-48">A set of minimal tiling path BACs was chosen using the FMTP method of 
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bozdag et al. (2013)</a>. MTP BACs were paired-end sequenced (2 × 100 bases) using Illumina HiSeq2000 (Illumina, Inc, San Diego, CA, USA). Sequencing was done in two sets of 2,197 BACs (Vu1 and Vu2) applying a combinatorial pooling design (
      <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lonardi et al. 2013</a>). After quality-trimming, reads in each pool were “sliced” into smaller samples of optimal size, deconvoluted, and then assembled BAC-by-BAC using SPAdes (
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bankevich et al. 2012</a>), as explained in detailed by 
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lonardi et al. (2015)</a>. From the 4,394 intended BACs, 4,355 produced sufficient reads to generate an assembly. Raw reads for these cowpea BACs were deposited in NCBI SRA under accession numbers SRA052227 and SRA052228.
     </p>
     <p id="p-49">To estimate the percentage of overlapping BAC sequences, 19-mers occurring at least 4 times were identified and used for repeat-masking of sequences. Repeat-masked sequences were then BLASTed against themselves using an e-value cutoff of e
      <sup>−40</sup>. Only overlapping sequences &gt;; 300 bp were considered to be overlaps. To estimate the gene content of the BAC assemblies, BAC sequences were compared to cowpea EST-derived “unigenes” (
      <a href="http://harvest.ucr.edu">http://harvest.ucr.edu</a>) and 
      <em>Phaseolus vulgaris</em> gene models (
      <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Schmutz et al. 2014</a>) using BLAST (e-value cutoffs of e
      <sup>−40</sup> and e
      <sup>−25</sup>, respectively).
     </p>
    </div>
    <div id="sec-15" class="subsection">
     <h3>Whole-genome shotgun sequencing and assembly</h3>
     <p id="p-50">The same batch of IT97K-499-35 nuclear DNA that was used for BAC library construction was used for whole-genome shotgun sequencing. About 394 M paired-end reads (equivalent to ~65x coverage) with an average read length of ~100 bases after quality-trimming were produced at the National Center for Genome Resources (NCGR; Santa Fe, NM) on an Illumina GAII sequencing instrument. An additional ~90 M Illumina reads were produced using an Illumina Hi-Seq sequencing instrument at NCGR from one 5 kb long-insert paired end (LIPE) library made from the same batch of nuclear DNA.</p>
     <p id="p-51">For the assembly, two additional sets of Sanger sequences were included. One set of Sanger sequences were the basis of a prior publication on “gene-space sequences” (GSS; 
      <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Timko et al. 2008</a>), comprised of ~250,000 reads from methyl filtered fragments of IT97K-499-35. The other set of Sanger sequences included the BAC-end sequences described above. The assembly combined the paired-end short reads, LIPE, GSS, and BES data using SOAPdenovo with k=31 (
      <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Luo et al. 2012</a>). To estimate the gene content of the WGS assembly, sequences were BLASTed against cowpea EST-derived “unigenes” (
      <a href="http://harvest.ucr.edu">http://harvest.ucr.edu</a>) and 
      <em>Phaseolus vulgaris</em> gene models (
      <a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Schmutz et al. 2014</a>), using e-value cutoffs of e
      <sup>−40</sup> and e
      <sup>−25</sup>, respectively.
     </p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>SNP discovery and design of the Cowpea iSelect Consortium Array</h3>
     <p id="p-52">A total of 32 accessions were sequenced to 12.5x coverage by the Beijing Genomics Institute (BGI) using Illumina HiSeq 2500 (Illumina, Inc., California, USA). Four additional accessions from China (see Table S1) were sequenced at the Majorbio Pharm Technology Co. Ltd (Shanghai). Additional sequences of IT97K-499-35 were produced in the Genomics Core Facility at the University of California, Riverside.</p>
     <p id="p-53">The WGS assembly from IT97K-499-35 described above was used as the reference to map each of these 36 sets of reads, and one new set of HiSeq sequences from the reference genotype sequenced at the University of California Riverside (Genomics Core Facility). This 37
      <sup>th</sup> set was used as a control (i.e. SNPs call in this accession were considered false positives). BWA (
      <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Li et al. 2009a</a>) was used to uniquely map each set of reads (BWA mem with-M option to mark shorter split hits as secondary). Reads which mapped to multiple locations were excluded from further analysis. Alignment files were merged with the software tool Picard to a single “sam” file. Reads that “hanged off” the end of the contigs in the reference sequence were clipped with Picard. Also, to avoid skewed variant calling, duplicated reads were marked with Picard.
     </p>
     <p id="p-54">To filter putative SNPs to a shorter list of highest confidence variants, three software packages were used, namely SAMtools (
      <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Li et al. 2009b</a>), SGSautoSNP (
      <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lorenc et al. 2012</a>), and FreeBayes (
      <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Garrison et al. 2012</a>). It was not possible to utilize GATK (
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McKenna et al. 2010</a>) because it requires a relatively large set of confirmed training SNPs for the base quality score recalibration phase, and no such set of SNPs was available for cowpea. In total, SAMtools discovered 5,108,787 SNPs using mpileup with default parameters, SGSautoSNP detected 2,488,797 SNPs and FreeBayes called a total of 8,269,140 SNPs. Finally, we used the haplotype-based variant detection tool FreeBayes to independently call the cowpea SNPs. FreeBayes called a total of 8,269,140 SNPs on the cowpea genome. An intersection set of SNPs was then identified, leading to 1,036,981 SNPs that were identified by all three methods. Additional filtering was required to reduce the number of SNPs to the target density of 60,000 SNP assays designed as a community resource for future germplasm characterization. These filtering steps included: 1) designability score based upon Illumina’s Assay Design Tool; 2) avoidance of a SNP whose adjacent sequences occurred frequently in the genome assembly; 3) consideration of allele frequency, generally avoiding SNPs with only one accession carrying the minor allele; 4) selection of two SNPs in or near each inferred cowpea gene based on MUMmer sequence alignment with 
      <em>Phaseolus vulgaris</em> gene models (
      <a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Schmutz et al. 2014</a>); 5) requirement for a minimum distance from a SNP that had already been selected, 6) preference against an A/T or C/G SNP since these require two beadtypes (assay space), and 7) location within a relatively larger WGS contig to maximize the amount of WGS contigs that could subsequently be anchored to a SNP-based genetic map.
     </p>
     <p id="p-55">In addition to SNPs discovered by WGS sequencing of diverse accessions, a total of 1,163 SNPs previously validated on the GoldenGate platform (
      <a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">Muchero et al. 2009</a>) were included in the design to facilitate comparisons with prior genotyping research. A total of 56,719 SNPs were submitted for assay design using 60,000 beadtypes with 51,128 assays included in the final manifest for the publicly available Cowpea iSelect Consortium Array.
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Consensus genetic map construction</h3>
     <p id="p-56">Five bi-parental RIL populations developed previously (
      <a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">Muchero et al. 2009</a>; 
      <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Lucas et al. 2011</a>) were genotyped with the Cowpea iSelect Consortium Array at the University of Southern California. SNPs were called using the GenomeStudio software (Illumina, Inc.). To meet assumptions of the clustering algorithm, “synthetic heterozygotes” were constructed and included in the initial set of 96 genotyped samples by creating 1:1 mixtures of DNA samples from individuals known from prior work to be most genetically distant from each other. The data from these individuals provided the signal needed for the algorithm to place a cluster position for heterozygotes. SNPs with low GenTrain scores were visually inspected based upon manufacturers published best practice for optimizing accuracy in genotyping projects (
      <a href="http://www.illumina.com/documents/products/technotes/technote_infinium_genotyping_data_analysis.pdf">http://www.illumina.com/documents/products/technotes/technote_infinium_genotyping_data_analysis.pdf</a>). The resulting cluster file is available upon request.
     </p>
     <p id="p-57">SNP data from each population were exported from GenomeStudio and curated to eliminate 1) monomorphic SNP loci, 2) SNPs with &gt;;20% missing or heterozygous calls, and 3) segregation-distorted markers (MAF&lt;;0.25). RILs were also curated to remove individuals with &gt;;10% heterozygous loci or those carrying many non-parental alleles. Identical individuals were also thinned to one such individual prior to mapping. Genetic maps for each RIL population were constructed at LOD 10 using MSTmap (
      <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wu et al. 2008</a>; 
      <a href="http://mstmap.org/">http://mstmap.org/</a>). Because the level of residual heterozygosity varied among populations, different population type options were chosen for map construction in MSTmap (RIL 7 for Tvu-14676 x IT84S-2246-4; RIL 6 for Sanzi x Vita7 and ZN016 x Zhijiang282; and RIL5 for CB46 x IT93K-503-1 and CB27 x IT82E-18). Other parameters for MSTmap included: grouping LOD criteria = 10; no mapping size threshold = 2; no mapping distance threshold = 10 cM; try to detect genotyping errors = no; and genetic mapping function = kosambi. Output maps were inspected to identify and remove data that would result in presumably spurious double recombination events, unless supported by several markers or moderate to large genetic distances.
     </p>
     <p id="p-58">LGs from each population were numbered and oriented based on the previous cowpea consensus map (
      <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Lucas et al. 2011</a>) and then merged into a consensus map using MergeMap (
      <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wu et al. 2011</a>; 
      <a href="http://mergemap.org/">http://mergemap.org/</a>). Equal weight was given to each individual map (weight = 1.0). MergeMap identified a few conflicts in marker order, which were resolved by deleting a few conflicted markers with priority given to the map with the highest resolution in the particular LG (i.e. more bins). Since MergeMap’s coordinate calculations for a consensus map are inflated relative to cM distances in individual maps, consensus LG lengths were normalized to the mean cM length from the individual maps.
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Synteny with 
      <em>Phaseolus vulgaris</em>
     </h3>
     <p id="p-59">The cowpea genome assembly described above was compared to 
      <em>P. vulgaris</em> pseudomolecules and unanchored scaffolds (from 
      <a href="https://phytozome.jgi.doe.gov/pz/portal.html">https://phytozome.jgi.doe.gov/pz/portal.html</a>) using MUMmer (
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kurtz et al. 2004</a>). Alignments that were further used had a minimum identity of 55.11% and a mean identity of 89.24%. The positions of 
      <em>P. vulgaris</em> gene models within the aligned regions was used to position each cowpea SNP relative to 
      <em>P. vulgaris</em> gene models. A synteny plot was constructed based on SNPs that had a cM position in the cowpea consensus map and fell within the region of the cowpea sequence that was aligned with a common bean gene model. Circos v.67-7 (
      <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Krzywinsk et al. 2009</a>) was used to illustrate the synteny between each cowpea linkage group and common bean chromosome that shared 50 or more SNPs. Cowpea LGs were plotted according to cM lengths, while common bean chromosomes were plotted as physical length.
     </p>
     <p id="p-60">Cowpea SNP frequencies were based on the number of discovered SNPs per genetic bin and the total size of the WGS scaffolds allocated into the corresponding bin. Then, for every 2 cM window the number of SNPs allocated within that window was divided by the sum of the corresponding WGS scaffold sizes in kb. Two outlying values were replacing by a maximum value so that all of the other calculated values could be easily visualized. 
      <em>Phaseolus vulgaris</em> gene densities were calculated as number of genes available from 
      <a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Schmutz et al. (2014)</a> per 500 kb windows.
     </p>
    </div>
    <div id="sec-19" class="subsection">
     <h3>Genetic analysis of West African accessions</h3>
     <p id="p-61">A total of 146 accessions were genotyped with the Cowpea i Select Consortium Array. Monomorphic loci were eliminated, as were SNPs with missing or heterozygous calls in more than 20% of the samples. PCA was conducted in TASSEL v5.0 (
      <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bradbury et al. 2007</a>; 
      <a href="http://www.maizegenetics.net">http://www.maizegenetics.net</a>) using SNPs with MAF&gt;;0.05, and results were displayed using TIBCO Spotfire
      <sup>®</sup> 6.5.0. PIC values were calculated using the method of 
      <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Botstein et al. (1980)</a> for all 46,620 SNPs in the entire set of samples, and then separately in the cultivar/breeding lines and landraces. PIC values were plotted along the consensus genetic map by averaging PIC values across a sliding window of 5 bins in 1 bin steps.
     </p>
    </div>
   </div>
   <div class="section" id="sec-20">
    <h2 class="">ACCESSION NUMBERS</h2>
    <p id="p-62">The BAC sequence reads supporting the results of this article are available in NCBI SRA under accession numbers SRA052227 and SRA052228.</p>
   </div>
   <div class="section" id="sec-21">
    <h2 class="">SUPPORTING INFORMATION</h2>
    <p id="p-64">
     <strong>Figure S1.</strong> Principal component analysis (PCA) of 729 samples representing the diversity of cultivated cowpea (in blue) and distribution of 34 out of the 36 accessions included in the SNP discovery panel (in red). The reference genome IT97K-499-35 is represented in green. PCA was performed using previous information from 1,535 GoldenGate SNPs.
    </p>
    <p id="p-65">
     <strong>Figure S2.</strong> Graphical representation of the iSelect SNP consensus genetic map for cowpea. Each horizontal line is a bin.
    </p>
    <p id="p-66">
     <strong>Figure S3.</strong> Polymorphism information content (PIC) values along linkage groups 2, 4, 5, 6, 7, 9, 10 and 11. Blue lines PIC values calculated in cultivars/breeding lines from four breeding programs, while orange lines represent PIC values in local landraces from Burkina Faso, Ghana, Nigeria and Senegal. PIC was averaged across a sliding window of 5 genetic bins with a step of one bin. Wide arrows indicate regions with a markedly depletion of genetic diversity in one or both types of materials, while narrow arrows represents reduced diversity regions containing a known QTL.
    </p>
    <p id="p-67">
     <strong>Table S1.</strong> Information of the cowpea accessions used for SNP discovery. The reference genome is underlined. UCR = University of California Riverside.
    </p>
    <p id="p-68">
     <strong>Table S2.</strong> Information on the individual mapping population data used for consensus map construction.
    </p>
    <p id="p-69">
     <strong>Table S3.</strong> Pairwise counts of the number of links between cowpea linkage groups (VuLG) and common bean pseudomolecules (Pv). Clear relationships between VuLGs and Pvs are marked in bold, while conflicting relationships are marked in red.
    </p>
    <p id="p-70">
     <strong>Table S4.</strong> Information of West African accession used in the study. IITA = the International Institute of Tropical Agriculture; INERA = Institut de l'Environnement et de Recherches Agricoles; ISRA = Institut Senegalais de Recherches Agricoles; SARI = the Savanna Agricultural Research Institute.
    </p>
    <p id="p-71">
     <strong>Data S1.</strong> Mapping statistics for 37 cowpea accessions.
    </p>
    <p id="p-72">
     <strong>Data S2.</strong> Information of SNPs included in the final Cowpea iSelect Consortium Assay. Pv = 
     <em>Phaseolus vulgaris</em>, Ath = 
     <em>Arabidopsis thaliana</em>.
    </p>
    <p id="p-73">
     <strong>Data S3.</strong> Five individual genetic maps (each sheet) and the genotype dataset used for their construction. “A” and “B” represent Parent 1 and Parent 2 in the cross, respectively. Lower case letters mean that the genotype calls were reversed based on the parental alleles.
    </p>
    <p id="p-74">
     <strong>Data S4.</strong> iSelect SNP consensus genetic map for cowpea.
    </p>
    <p id="p-75">
     <strong>Data S5.</strong> List of sequenced BACs and their genetic anchoring information.
    </p>
    <p id="p-76">
     <strong>Data S6.</strong> List of WGS scaffolds and their genetic anchoring information.
    </p>
    <p id="p-77">
     <strong>Data S7.</strong> Polymorphism Information Content (PIC) values considering the entire sample set (146 West African accessions; PICall), only cultivar/breeding lines (105 accessions; PICbreeding), and only landraces (41 accessions; PIClandraces).
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">ACKNOWLEDGMENTS</h2>
    <p id="p-63">Authors thank John Weger (Genome Core Facility, UC Riverside) and Greg D. May (NCGR) for sequencing services, and Sassoum Lo and Savanah St. Clair (UC Riverside) for DNA isolation. Development of the cowpea iSelect was supported by the Feed the Future Innovation Lab for Climate Resilient Cowpea (USAID Cooperative Agreement AID-OAA-A-13-00070) and the 2014 Illumina Agricultural Greater Good Initiative. BAC library construction, physical mapping and BAC-end sequencing was supported by the Generation Challenge Program “Tropical Legumes 1” project. Minimal tiling path BAC sequencing was supported by NSF IIS-1526742 (“III: Small: Algorithms for Genome Assembly”). Partial support was also provided by Hatch Project CA-R-BPS-5306-H. “GSS” sequencing (
     <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Timko et al. 2008</a>) was supported by Kirkhouse Trust.
    </p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">REFERENCES</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.1" data-doi="10.1089/cmb.2012.0021">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bankevich</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nurk</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Antipov</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gurevich</span>, 
          <span class="cit-name-given-names">A.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dvorkin</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kulikov</span>, 
          <span class="cit-name-given-names">A.S.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</span>. 
         <abbr class="cit-jnl-abbrev">J Comp. Biol.</abbr>
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">455</span>–
         <span class="cit-lpage">477</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BComp.%2BBiol.%26rft.volume%253D19%26rft.spage%253D455%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bapna</span>, 
          <span class="cit-name-given-names">C.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Joshi</span>, 
          <span class="cit-name-given-names">S.N.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Kabria</span>, 
          <span class="cit-name-given-names">M.M.</span>
         </span> (
         <span class="cit-pub-date">1972</span>) 
         <span class="cit-article-title">Correlation studies on yield and agronomic characters in cowpea</span>. 
         <abbr class="cit-jnl-abbrev">Indian J. Agron.</abbr>
         <span class="cit-vol">17</span>, 
         <span class="cit-fpage">321</span>–
         <span class="cit-lpage">324</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIndian%2BJ.%2BAgron.%26rft.volume%253D17%26rft.spage%253D321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.3">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Botstein</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">White</span>, 
          <span class="cit-name-given-names">R.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Skolnick</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Davis</span>, 
          <span class="cit-name-given-names">R.W.</span>
         </span> (
         <span class="cit-pub-date">1980</span>) 
         <span class="cit-article-title">Construction of a genetic linkage map in man using restriction fragment length polymorphisms</span>. 
         <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr>
         <span class="cit-vol">32</span>, 
         <span class="cit-fpage">314</span>–
         <span class="cit-lpage">331</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBotstein%26rft.auinit1%253DD.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D314%26rft.epage%253D331%26rft.atitle%253DConstruction%2Bof%2Ba%2Bgenetic%2Blinkage%2Bmap%2Bin%2Bman%2Busing%2Brestriction%2Bfragment%2Blength%2Bpolymorphisms.%26rft_id%253Dinfo%253Apmid%252F6247908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=6247908&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1980JX05900002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.4">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bozdag</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Close</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Lonardi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">A graph-theoretical approach to the selection of the minimum tiling path from a physical map</span>. 
         <abbr class="cit-jnl-abbrev">IEEE/ACM Trans. Comput. Biol. Bioinform.</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">352</span>–
         <span class="cit-lpage">360</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DIEEE%252FACM%2BTrans.%2BComput.%2BBiol.%2BBioinform.%26rft.volume%253D10%26rft.spage%253D352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.5" data-doi="10.1093/bioinformatics/btm308">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bradbury</span>, 
          <span class="cit-name-given-names">P.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kroon</span>, 
          <span class="cit-name-given-names">D.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Casstevens</span>, 
          <span class="cit-name-given-names">T.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ramdoss</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Buckler</span>, 
          <span class="cit-name-given-names">E.S.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">23</span>, 
         <span class="cit-fpage">2633</span>–
         <span class="cit-lpage">5</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17586829&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000250673800021&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.6" data-doi="10.1186/1471-2105-8-129">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Laudeman</span>, 
          <span class="cit-name-given-names">T.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rushton</span>, 
          <span class="cit-name-given-names">P.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Spraggins</span>, 
          <span class="cit-name-given-names">T.A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Timko</span>, 
          <span class="cit-name-given-names">M.P.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">CGKB: an annotation knowledge base for cowpea (Vigna unguiculata L.) methylation filtered genomic genespace sequences</span>. 
         <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">129</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DChen%26rft.auinit1%253DX.%26rft.volume%253D8%26rft.spage%253D129%26rft.epage%253D129%26rft.atitle%253DCGKB%253A%2Ban%2Bannotation%2Bknowledge%2Bbase%2Bfor%2Bcowpea%2B%2528Vigna%2Bunguiculata%2BL.%2529%2Bmethylation%2Bfiltered%2Bgenomic%2Bgenespace%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-129%26rft_id%253Dinfo%253Apmid%252F17445272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-129&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17445272&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.7" data-doi="10.1101/gr.8.3.186">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ewing</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Green</span>, 
          <span class="cit-name-given-names">F.</span>
         </span> (
         <span class="cit-pub-date">1998</span>) 
         <span class="cit-article-title">Base-calling of automated sequencer traces using phred. II. Error probabilities</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res.</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">186</span>–
         <span class="cit-lpage">94</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.8.3.186%26rft_id%253Dinfo%253Apmid%252F9521922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1101/gr.8.3.186&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=9521922&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Fang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chao</span>, 
          <span class="cit-name-given-names">C-C.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>, 
          <span class="cit-name-given-names">P.A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Ehlers</span>, 
          <span class="cit-name-given-names">J.D.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">Genetic diversity of cowpea [Vigna unguiculata (L.) Walp.] in four West African and USA breeding programs as determined by AFLP analysis</span>. 
         <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr>
         <span class="cit-vol">54</span>, 
         <span class="cit-fpage">1197</span>–
         <span class="cit-lpage">1209</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D54%26rft.spage%253D1197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Garrison</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Marth</span>, 
          <span class="cit-name-given-names">G.</span>
         </span> (
         <span class="cit-pub-date">2012</span>) 
         <abbr class="cit-jnl-abbrev">Haplotype-based variant detection from short-read sequencing</abbr>. arXiv preprint 
         <a href="http://arxiv.org/abs/arXiv:1207.3907">arXiv:1207.3907</a>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.10" data-doi="10.1186/1471-2164-10-496">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gu</span>, 
          <span class="cit-name-given-names">Y.Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huo</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vogel</span>, 
          <span class="cit-name-given-names">J.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">You</span>, 
          <span class="cit-name-given-names">F.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lazo</span>, 
          <span class="cit-name-given-names">G.R.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">A BAC-based physical map of Brachypodium distachyon and its comparative analysis with rice and wheat</span>. 
         <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>
         <span class="cit-vol">10</span>, 
         <span class="cit-fpage">496</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGu%26rft.auinit1%253DY.%2BQ.%26rft.volume%253D10%26rft.spage%253D496%26rft.epage%253D496%26rft.atitle%253DA%2BBAC-based%2Bphysical%2Bmap%2Bof%2BBrachypodium%2Bdistachyon%2Band%2Bits%2Bcomparative%2Banalysis%2Bwith%2Brice%2Band%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-496%26rft_id%253Dinfo%253Apmid%252F19860896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-496&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19860896&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000271882900001&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.11" data-doi="10.1016/j.tig.2010.12.003">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Hamblin</span>, 
          <span class="cit-name-given-names">M.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Buckler</span>, 
          <span class="cit-name-given-names">E.S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Jannink</span>, 
          <span class="cit-name-given-names">J.L.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Population genetics of genomics-based crop improvement methods</span>. 
         <abbr class="cit-jnl-abbrev">Trends Genet.</abbr>
         <span class="cit-vol">27</span>, 
         <span class="cit-fpage">98</span>–
         <span class="cit-lpage">106</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DHamblin%26rft.auinit1%253DM.%2BT.%26rft.volume%253D27%26rft.issue%253D3%26rft.spage%253D98%26rft.epage%253D106%26rft.atitle%253DPopulation%2Bgenetics%2Bof%2Bgenomics-based%2Bcrop%2Bimprovement%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2010.12.003%26rft_id%253Dinfo%253Apmid%252F21227531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.tig.2010.12.003&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21227531&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000288685900003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huynh</span>, 
          <span class="cit-name-given-names">B.L.</span>
         </span>
         <span class="cit-auth">
          <span class="cit-name-surname">Close</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>, 
          <span class="cit-name-given-names">P.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hubc</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wanamaker</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lucas</span>, 
          <span class="cit-name-given-names">M.R.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Gene pools and the genetic architecture of domesticated cowpea</span>. 
         <abbr class="cit-jnl-abbrev">Plant Genome</abbr>
         <span class="cit-fpage">6</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Iwata-Otsubo</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">J.Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gill</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Jackson</span>, 
          <span class="cit-name-given-names">S.A.</span>
         </span> (
         <span class="cit-pub-date">2016</span>) 
         <span class="cit-article-title">Highly distinct chromosomal structures in cowpea (Vigna unguiculata), as revealed by molecular cytogenetic analysis</span>. 
         <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr>
         <span class="cit-vol">24</span>, 
         <span class="cit-fpage">197</span>–
         <span class="cit-lpage">216</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChromosome%2BRes.%26rft.volume%253D24%26rft.spage%253D197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.14" data-doi="10.1186/1746-4811-9-29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Korte</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Farlow</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">The advantages and limitations of trait analysis with GWAS: a review</span>. 
         <abbr class="cit-jnl-abbrev">Plant Methods</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">29</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D9%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-9-29%26rft_id%253Dinfo%253Apmid%252F23876160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1746-4811-9-29&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23876160&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.15" data-doi="10.1101/gr.092759.109">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Krzywinsk</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schein</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Birol</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Connors</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gascoyne</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Horsman</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Circos: an information aesthetic for comparative genomics</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res.</abbr>
         <span class="cit-vol">19</span>, 
         <span class="cit-fpage">1639</span>–
         <span class="cit-lpage">45</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE2LzA2LzE1LzA1OTI2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.16" data-doi="10.1038/srep11166">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kujur</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bajaj</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Upadhyaya</span>, 
          <span class="cit-name-given-names">H.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Das</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ranjan</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shree</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">A genome-wide SNP scan accelerates trait-regulatory genomic loci identification in chickpea</span>. 
         <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr>
         <span class="cit-vol">5</span>, 
         <span class="cit-fpage">11166</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D11166%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep11166%26rft_id%253Dinfo%253Apmid%252F26058368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/srep11166&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26058368&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.17" data-doi="10.1186/gb-2004-5-2-r12">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kurtz</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Phillippy</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Delcher</span>, 
          <span class="cit-name-given-names">A.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smoot</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shumway</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Antonescu</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Salzberg</span>, 
          <span class="cit-name-given-names">S.L.</span>
         </span> (
         <span class="cit-pub-date">2004</span>) 
         <span class="cit-article-title">Versatile and open software for comparing large genomes</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol.</abbr>
         <span class="cit-vol">5</span>, 
         <span class="cit-fpage">R12</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DVersatile%2Band%2Bopen%2Bsoftware%2Bfor%2Bcomparing%2Blarge%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-2-r12%26rft_id%253Dinfo%253Apmid%252F14759262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-2-r12&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=14759262&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.18" data-doi="10.1093/bioinformatics/btp324">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.</span>
         </span> and 
         <span class="cit-auth">
          <span class="cit-name-surname">Durbin</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2009a</span>) 
         <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">1754</span>–
         <span class="cit-lpage">60</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19451168&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000267665900006&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.19" data-doi="10.1093/bioinformatics/btp352">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Handsaker</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wysoker</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fennell</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruan</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Homer</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2009b</span>) 
         <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">25</span>, 
         <span class="cit-fpage">2078</span>–
         <span class="cit-lpage">9</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19505943&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000268808600014&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.20" data-doi="10.1371/journal.pcbi.1003010">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lonardi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duma</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alpert</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cordero</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Beccuti</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bhat</span>, 
          <span class="cit-name-given-names">P.R.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Combinatorial pooling enables selective sequencing of the barley gene space</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Comput Biol.</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">e1003010</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol.%26rft.volume%253D9%26rft.spage%253De1003010%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003010%26rft_id%253Dinfo%253Apmid%252F23592960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003010&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23592960&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.21" data-doi="10.1093/bioinformatics/btv311">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lonardi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mirebrahim</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wanamaker</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alpert</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ciardo</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duma</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Close</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">When less is more: ‘slicing’ sequencing data improves read decoding accuracy and de novo assembly quality</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>
         <span class="cit-vol">31</span>, 
         <span class="cit-fpage">2972</span>–
         <span class="cit-lpage">80</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv311%26rft_id%253Dinfo%253Apmid%252F25995232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv311&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25995232&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.22">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lorenc</span>, 
          <span class="cit-name-given-names">M.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hayashi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stiller</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lee</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Manoli</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ruperao</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">Discovery of single nucleotide polymorphisms in complex genomes using SGSautoSNP</span>. 
         <abbr class="cit-jnl-abbrev">Biology (Basel)</abbr>
         <span class="cit-vol">1</span>, 
         <span class="cit-fpage">370</span>–
         <span class="cit-lpage">382</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBiology%2B%2528Basel%2529%26rft.volume%253D1%26rft.spage%253D370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lucas</span>, 
          <span class="cit-name-given-names">M.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Diop</span>, 
          <span class="cit-name-given-names">N.N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wanamaker</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ehlers</span>, 
          <span class="cit-name-given-names">J.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>, 
          <span class="cit-name-given-names">P. A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Close</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Cowpea-soybean synteny clarified through an improved genetic map</span>. 
         <abbr class="cit-jnl-abbrev">Plant Genome</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">218</span>–
         <span class="cit-lpage">225</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.24">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lucas</span>, 
          <span class="cit-name-given-names">M.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ehlers</span>, 
          <span class="cit-name-given-names">J.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huynh</span>, 
          <span class="cit-name-given-names">B.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Diop</span>, 
          <span class="cit-name-given-names">N.N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>, 
          <span class="cit-name-given-names">P.A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Close</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span> (
         <span class="cit-pub-date">2013a</span>) 
         <span class="cit-article-title">Markers for breeding heat-tolerant cowpea</span>. 
         <abbr class="cit-jnl-abbrev">Mol. Breeding</abbr>
         <span class="cit-vol">31</span>, 
         <span class="cit-fpage">529</span>–
         <span class="cit-lpage">536</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBreeding%26rft.volume%253D31%26rft.spage%253D529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lucas</span>, 
          <span class="cit-name-given-names">M.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huynh</span>, 
          <span class="cit-name-given-names">B.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">da Silva Vinholes</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cisse</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drabo</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ehlers</span>, 
          <span class="cit-name-given-names">J.D.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2013b</span>) 
         <span class="cit-article-title">Association studies and legume synteny reveal haplotypes determining seed size in Vigna unguiculata</span>. 
         <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">95</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D4%26rft.spage%253D95%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.26" data-doi="10.1016/S0888-7543(03)00128-9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>
          <span class="cit-name-given-names">M.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Thomas</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">You</span>, 
          <span class="cit-name-given-names">F.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hsiao</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ouyang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Buell</span>, 
          <span class="cit-name-given-names">C.R.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2003</span>) 
         <span class="cit-article-title">High-throughput fingerprinting of bacterial artificial chromosomes using the snapshot labeling kit and sizing of restriction fragments by capillary electrophoresis</span>. 
         <abbr class="cit-jnl-abbrev">Genomics</abbr>
         <span class="cit-vol">82</span>, 
         <span class="cit-fpage">378</span>–
         <span class="cit-lpage">389</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLuo%26rft.auinit1%253DM.%2BC.%26rft.volume%253D82%26rft.issue%253D3%26rft.spage%253D378%26rft.epage%253D389%26rft.atitle%253DHigh-throughput%2Bfingerprinting%2Bof%2Bbacterial%2Bartificial%2Bchromosomes%2Busing%2Bthe%2Bsnapshot%2Blabeling%2Bkit%2Band%2Bsizing%2Bof%2Brestriction%2Bfragments%2Bby%2Bcapillary%2Belectrophoresis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0888-7543%252803%252900128-9%26rft_id%253Dinfo%253Apmid%252F12906862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0888-7543(03)00128-9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=12906862&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000184833100013&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.27" data-doi="10.1186/2047-217X-1-18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xie</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yuan</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</span>. 
         <abbr class="cit-jnl-abbrev">GigaScience</abbr>
         <span class="cit-vol">1</span>, 
         <span class="cit-fpage">18</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGigaScience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23587118&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.28" data-doi="10.1101/gr.107524.110">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">McKenna</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>
         <span class="cit-auth">
          <span class="cit-name-surname">Hanna</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Banks</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sivachenko</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cibulskis</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kernytsky</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2010</span>) 
         <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. 
         <abbr class="cit-jnl-abbrev">Genome Res.</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">1297</span>–
         <span class="cit-lpage">303</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE2LzA2LzE1LzA1OTI2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-059261v1.29">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Muchero</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Diop</span>, 
          <span class="cit-name-given-names">N.N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bhat</span>, 
          <span class="cit-name-given-names">P.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Fenton</span>, 
          <span class="cit-name-given-names">R.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wanamaker</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pottorff</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-conf-date">2009</span>) 
         <span class="cit-article-title">A consensus genetic map of cowpea Vigna unguiculata (L) Walp‥ and synteny based on EST-derived SNPs</span>. 
         <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. 
         <span class="cit-conf-loc">U.S.A</span>. 
         <span class="cit-vol">106</span>, 
         <span class="cit-fpage">18159</span>–
         <span class="cit-lpage">18164</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.30" data-doi="10.1371/journal.pone.0070041">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Muchero</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>, 
          <span class="cit-name-given-names">P.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Diop</span>, 
          <span class="cit-name-given-names">N.N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Drabo</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cisse</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Close</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Genetic architecture of delayed senescence, biomass, and grain yield under drought stress in cowpea</span>. 
         <abbr class="cit-jnl-abbrev">PLoS One</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">e70041</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De70041%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0070041%26rft_id%253Dinfo%253Apmid%252F23936140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0070041&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23936140&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pottorff</span>, 
          <span class="cit-name-given-names">M.O.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ehlers</span>, 
          <span class="cit-name-given-names">J.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Close</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Roberts</span>, 
          <span class="cit-name-given-names">P.A.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Genetic mapping, synteny, and physical location of two loci for Fusarium oxysporum f. sp. tracheiphilum race 4 resistance in cowpea Vigna unguiculata (L.) Walp</span>‥ 
         <abbr class="cit-jnl-abbrev">Mol. Breeding</abbr>
         <span class="cit-vol">33</span>, 
         <span class="cit-fpage">779</span>– 
         <span class="cit-lpage">791</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol.%2BBreeding%26rft.volume%253D33%26rft.spage%253D779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.32">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ray</span>, 
          <span class="cit-name-given-names">J.D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Dhanapal</span>, 
          <span class="cit-name-given-names">A.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Singh</span>, 
          <span class="cit-name-given-names">S.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hoyos-Villegas</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Smith</span>, 
          <span class="cit-name-given-names">J.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Purcell</span>, 
          <span class="cit-name-given-names">L.C.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Genome-wide association study of ureide concentration in diverse maturity group IV soybean [Glycine max (L.) Merr.] accessions</span>. 
         <abbr class="cit-jnl-abbrev">G3</abbr>
         <span class="cit-vol">5</span>, 
         <span class="cit-fpage">2391</span>–
         <span class="cit-lpage">403</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D5%26rft.spage%253D2391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.33" data-doi="10.1093/pcp/pcv189">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Sakai</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Naito</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Takahashi</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sato</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yamamoto</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Muto</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2016</span>) 
         <span class="cit-article-title">The Vigna Genome Server, ‘VigGS’: a genomic knowledge base of the genus Vigna gased on high-quality, annotated genome sequence of the Azuki Bean, Vigna angularis (Willd.) Ohwi &amp;; Ohashi</span>. 
         <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr>
         <span class="cit-vol">57</span>, 
         <span class="cit-fpage">e2</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcv189%26rft_id%253Dinfo%253Apmid%252F26644460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/pcp/pcv189&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26644460&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.34" data-doi="10.1038/ng.3008">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Schmutz</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">McClean</span>, 
          <span class="cit-name-given-names">P.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mamidi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">G.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cannon</span>, 
          <span class="cit-name-given-names">S.B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Grimwood</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">A reference genome for common bean and genome-wide analysis of dual domestications</span>. 
         <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr>
         <span class="cit-vol">46</span>, 
         <span class="cit-fpage">707</span>–
         <span class="cit-lpage">713</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3008%26rft_id%253Dinfo%253Apmid%252F24908249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ng.3008&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24908249&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.35" data-doi="10.2135/2014.cowpea">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Singh</span>, 
          <span class="cit-name-given-names">B.B.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Cowpea: the food legume of the 21
          <sup>st</sup> century
         </span>. 
         <abbr class="cit-jnl-abbrev">Crop Science Society of America, Inc.</abbr>
         <span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span>
         <span class="cit-pub-id-scheme">DOI:</span>
         <span class="cit-pub-id cit-pub-id-doi">10.2135/2014.cowpea</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%2BSociety%2Bof%2BAmerica%252C%2BInc.%26rft_id%253Dinfo%253Adoi%252F10.2135%252F2014.cowpea%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2135/2014.cowpea&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.36">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Soderlund</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>
         <span class="cit-auth">
          <span class="cit-name-surname">Longden</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Mott</span>, 
          <span class="cit-name-given-names">R.</span>
         </span> (
         <span class="cit-pub-date">2000</span>) 
         <span class="cit-article-title">FPC: a system for building contigs from restriction fingerprinted clones</span>. 
         <abbr class="cit-jnl-abbrev">Comput. Appl. Biosci.</abbr>
         <span class="cit-vol">13</span>, 
         <span class="cit-fpage">523</span>–
         <span class="cit-lpage">535</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DComput.%2BAppl.%2BBiosci.%26rft.volume%253D13%26rft.spage%253D523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.37" data-doi="10.1098/rstb.2007.2191">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thomson</span>, 
          <span class="cit-name-given-names">J.A.</span>
         </span>
         <span class="cit-article-title">The role of biotechnology for agricultural sustainability in Africa</span> (
         <span class="cit-pub-date">2008</span>) 
         <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr>
         <span class="cit-vol">363</span>, 
         <span class="cit-fpage">905</span>–
         <span class="cit-lpage">913</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DThomson%26rft.auinit1%253DJ.%2BA%26rft.volume%253D363%26rft.issue%253D1492%26rft.spage%253D905%26rft.epage%253D913%26rft.atitle%253DThe%2Brole%2Bof%2Bbiotechnology%2Bfor%2Bagricultural%2Bsustainability%2Bin%2BAfrica%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2007.2191%26rft_id%253Dinfo%253Apmid%252F17761472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1098/rstb.2007.2191&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17761472&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Thornton</span>, 
          <span class="cit-name-given-names">P.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ericksen</span>, 
          <span class="cit-name-given-names">P.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Herrero</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Challinor</span>, 
          <span class="cit-name-given-names">A.J.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Climate variability and vulnerability to climate change: a review</span>. 
         <abbr class="cit-jnl-abbrev">Glob. Chang. Biol.</abbr>
         <span class="cit-vol">20</span>, 
         <span class="cit-fpage">3313</span>–
         <span class="cit-lpage">3328</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGlob.%2BChang.%2BBiol.%26rft.volume%253D20%26rft.spage%253D3313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.39" data-doi="10.1186/1471-2164-9-103">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Timko</span>, 
          <span class="cit-name-given-names">M.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rushton</span>, 
          <span class="cit-name-given-names">P.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Laudeman</span>, 
          <span class="cit-name-given-names">T.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bokowiec</span>, 
          <span class="cit-name-given-names">M.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chipumuro</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cheung</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2008</span>) 
         <span class="cit-article-title">Sequencing and analysis of the gene-rich space of cowpea</span>. 
         <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>
         <span class="cit-vol">9</span>, 
         <span class="cit-fpage">103</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DTimko%26rft.auinit1%253DM.%2BP.%26rft.volume%253D9%26rft.spage%253D103%26rft.epage%253D103%26rft.atitle%253DSequencing%2Band%2Banalysis%2Bof%2Bthe%2Bgene-rich%2Bspace%2Bof%2Bcowpea.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-103%26rft_id%253Dinfo%253Apmid%252F18304330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-103&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18304330&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.40" data-doi="10.1038/nbt.2022">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Varshney</span>, 
          <span class="cit-name-given-names">R.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Li</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bharti</span>, 
          <span class="cit-name-given-names">A.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Saxena</span>, 
          <span class="cit-name-given-names">R.K.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schlueter</span>, 
          <span class="cit-name-given-names">J.A.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">Draft genome sequence of pigeonpea (Cajanus cajan), an orphan legume crop of resource-poor farmers</span>. 
         <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr>
         <span class="cit-vol">30</span>, 
         <span class="cit-fpage">83</span>–
         <span class="cit-lpage">89</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DVarshney%26rft.auinit1%253DR.%2BK.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D83%26rft.epage%253D89%26rft.atitle%253DDraft%2Bgenome%2Bsequence%2Bof%2Bpigeonpea%2B%2528Cajanus%2Bcajan%2529%252C%2Ban%2Borphan%2Blegume%2Bcrop%2Bof%2Bresource-poor%2Bfarmers.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2022%26rft_id%253Dinfo%253Apmid%252F22057054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/nbt.2022&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22057054&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.41">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Vasconcelos</span>, 
          <span class="cit-name-given-names">E.V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Andrade Fonseca</span>, 
          <span class="cit-name-given-names">A.F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pedrosa-Harand</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Andrade Bortoleti</span>, 
          <span class="cit-name-given-names">K.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Benko-Iseppon</span>, 
          <span class="cit-name-given-names">A.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">da Costa</span>, 
          <span class="cit-name-given-names">A.F.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Brasileiro-Vidal</span>, 
          <span class="cit-name-given-names">A.C.</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Intra-and interchromosomal rearrangements between cowpea Vigna unguiculata (L.) Walp‥ and common bean (Phaseolus vulgaris L.) revealed by BAC-FISH</span>. 
         <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr>
         <span class="cit-vol">23</span>, 
         <span class="cit-fpage">253266</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DChromosome%2BRes.%26rft.volume%253D23%26rft.spage%253D253266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.42" data-doi="10.1371/journal.pgen.1000212">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bhat</span>, 
          <span class="cit-name-given-names">P.R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Close</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Lonardi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">2008</span>) 
         <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr>
         <span class="cit-vol">4</span>, 
         <span class="cit-fpage">e1000212</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18846212&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.43" data-doi="10.1109/TCBB.2010.35">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Close</span>, 
          <span class="cit-name-given-names">T.J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Lonardi</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">Accurate construction of consensus genetic maps via integer linear programming</span>. 
         <abbr class="cit-jnl-abbrev">IEEE/ACM Trans. Comput. Biol. Bioinform.</abbr>
         <span class="cit-vol">8</span>, 
         <span class="cit-fpage">381</span>–
         <span class="cit-lpage">94</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DIEEE%252FACM%2BTrans%2BComput%2BBiol%2BBioinform%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D381%26rft.epage%253D394%26rft.atitle%253DAccurate%2Bconstruction%2Bof%2Bconsensus%2Bgenetic%2Bmaps%2Bvia%2Binteger%2Blinear%2Bprogramming.%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2010.35%26rft_id%253Dinfo%253Apmid%252F20479505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1109/TCBB.2010.35&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=20479505&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2016%2F06%2F15%2F059261.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-059261v1.44">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Munoz-Amatriain</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hu</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2016</span>) 
         <span class="cit-article-title">Discovery of genome regions associated with pod length provides insights into grain-vegetable cowpea diversification</span>. 
         <abbr class="cit-jnl-abbrev">Submitted</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-confproc" id="cit-059261v1.45">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">K.</span>
         </span>
         <span class="cit-auth">
          <span class="cit-name-surname">Tian</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-conf-date">2015</span>) 
         <span class="cit-article-title">Genome sequencing of adzuki bean (Vigna angularis) provides insight into high starch and low fat accumulation and domestication</span>. 
         <span class="cit-conf-name">Proc. Natl. Acad. Sci</span>. 
         <span class="cit-conf-loc">U.S.A</span>. 
         <span class="cit-vol">112</span>, 
         <span class="cit-fpage">13213</span>–
         <span class="cit-lpage">8</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
