<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/04/19/292540.atom" />
<meta name="HW.pisa" content="biorxiv;292540v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Adaptation to local climate in a multi-trait space: evidence from silver fir (Abies alba Mill.) populations across a heterogeneous environment" />
<meta name="DC.Identifier" content="10.1101/292540" />
<meta name="DC.Date" content="2018-04-19" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Attempts to identify and understand selection pressures responsible for local adaptation are central to evolutionary research. We tested whether populations of silver fir ( Abies alba Mill.), sampled across a heterogeneous environment, have more strongly diverged at quantitative traits than expected from genetic drift. We genotyped 387 trees from 19 Swiss populations at 374 single-nucleotide polymorphisms (SNPs) to estimate their demographic distances, and used these to generate a null expectation for divergence in a multi-trait space including morphology and life-history traits obtained from a published common garden trial. Local soil and historical climate data were used to identify the selective environment of the source populations. Our results revealed a strong selection on height driven by temperature: trees from warm sites evolved to become taller than those from cooler sites. The evolution of growth rate, growth duration and bud break were correlated, and populations evolved towards two extreme strategies, “start early and grow slow” or “start late and grow fast”, driven by precipitation seasonality. We conclude that local climate has shaped the morphology and life-history of silver fir populations since they recolonized the Alps. Our methodology provides a show-case for empirical evaluation of adaptive evolutionary strategies combining genetic data and common gardens.

### Author contributions

KC designed the study and obtained the SNP data with help from AW and FG. KC performed all analysis with advice from OO. CS was involved in the published common garden experiment and provided help with data interpretation. KC wrote the manuscript with help from all other authors.

We thank all members of the ADAPT project that provided the phenotypic data for this study, especially Aline Frank, Caroline Heiri and Peter Brang. Additionally, Andrea Kupferschmid helped with correcting some errors in the phenotypic data thanks to her follow-up project (EADAPT). Part of the sampling was conducted by Catherine Folly. The project was funded by a research grant from the Center for Adaptation to a Changing Environment (ACE) at the ETH Zürich. KC was supported by ACE fellowship while obtaining and analyzing the data, and by a Marie Sklodowska-Curie fellowship (FORGENET), while working on the manuscript. We thank Kristian Ullrich for help with the SNP selection, and Dirk Krager with extracting the climatic data from the data bases. The contribution of several field and lab workers was necessary for this project, in particular, we thank René Graf and Olivier Charlandie." />
<meta name="DC.Contributor" content="Katalin Csilléry" />
<meta name="DC.Contributor" content="Otso Ovaskainen" />
<meta name="DC.Contributor" content="Christoph Sperisen" />
<meta name="DC.Contributor" content="Alex Widmer" />
<meta name="DC.Contributor" content="Felix Gugerli" />
<meta name="article:published_time" content="2018-04-19" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Adaptation to local climate in a multi-trait space: evidence from silver fir (Abies alba Mill.) populations across a heterogeneous environment" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Attempts to identify and understand selection pressures responsible for local adaptation are central to evolutionary research. We tested whether populations of silver fir (&lt;i&gt;Abies alba&lt;/i&gt; Mill.), sampled across a heterogeneous environment, have more strongly diverged at quantitative traits than expected from genetic drift. We genotyped 387 trees from 19 Swiss populations at 374 single-nucleotide polymorphisms (SNPs) to estimate their demographic distances, and used these to generate a null expectation for divergence in a multi-trait space including morphology and life-history traits obtained from a published common garden trial. Local soil and historical climate data were used to identify the selective environment of the source populations. Our results revealed a strong selection on height driven by temperature: trees from warm sites evolved to become taller than those from cooler sites. The evolution of growth rate, growth duration and bud break were correlated, and populations evolved towards two extreme strategies, “start early and grow slow” or “start late and grow fast”, driven by precipitation seasonality. We conclude that local climate has shaped the morphology and life-history of silver fir populations since they recolonized the Alps. Our methodology provides a show-case for empirical evaluation of adaptive evolutionary strategies combining genetic data and common gardens.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;292540v2" />
<meta name="citation_id" content="292540v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/292540v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/292540v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/292540v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540.full.pdf" />
<meta name="citation_doi" content="10.1101/292540" />
<meta name="citation_num_pages" content="39" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="292540" />
<meta name="citation_author" content="Katalin Csilléry" />
<meta name="citation_author_institution" content="Department of Evolutionary Biology and Environmental Studies, University of Zürich, Zürich" />
<meta name="citation_author_institution" content="Swiss Federal Research Institute WSL, Birmensdorf" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0039-9296" />
<meta name="citation_author" content="Otso Ovaskainen" />
<meta name="citation_author_institution" content="Faculty of Biological and Environmental Sciences, University of Helsinki, Helsinki" />
<meta name="citation_author_institution" content="Centre for Biodiversity Dynamics, Department of Biology, Norwegian University of Science and Technology, Trondheim" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9750-4421" />
<meta name="citation_author" content="Christoph Sperisen" />
<meta name="citation_author_institution" content="Swiss Federal Research Institute WSL, Birmensdorf" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1241-5636" />
<meta name="citation_author" content="Alex Widmer" />
<meta name="citation_author_institution" content="Institute of Integrative Biology, ETH Zürich, Zürich" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8253-5137" />
<meta name="citation_author" content="Felix Gugerli" />
<meta name="citation_author_institution" content="Swiss Federal Research Institute WSL, Birmensdorf" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3878-1845" />
<meta name="citation_reference" content="Aitken SN, Bemmels JB (2016) Time to get moving: assisted gene flow of forest trees. Evolutionary applications, 9, 271–290. doi:10.1111/eva.12293." />
<meta name="citation_reference" content="Aitken SN, Yeaman S, Holliday JA, Wang T, Curtis-McLane S (2008) Adaptation, migration or extirpation: climate change outcomes for tree populations. Evolutionary Applications, 1, 95–111. doi:10.1111/j.1752-4571.2007.00013.x." />
<meta name="citation_reference" content="Alberto FJ, Aitken SN, Alía R, et al. (2013) Potential for evolutionary responses to climate change - evidence from tree populations. Global Change Biology, 19, 1645–1661. doi:10. 1111/gcb.12181." />
<meta name="citation_reference" content="Allen CD, Macalady AK, Chenchouni H, et al. (2010) A global overview of drought and heat-induced tree mortality reveals emerging climate change risks for forests. Forest Ecology and Management, 259, 660–684. doi:10.1016/j.foreco.2009.09.001." />
<meta name="citation_reference" content="Baranzelli MC, Sersic AN, Cocucci AA (2014) The search for pleiades in trait constellations: functional integration and phenotypic selection in the complex flowers of Morrenia brachystephana (apocynaceae). Journal of Evolutionary Biology, 27, 724–736." />
<meta name="citation_reference" content="Beguería S, Vicente-Serrano SM (2017) SPEI: calculation of the standardised precipitation-evapotranspiration index. R package version 1.7, 1." />
<meta name="citation_reference" content="Berg R (1960) The ecological significance of correlation pleiades. Evolution, 14, 171–180." />
<meta name="citation_reference" content="Booth TH, Nix HA, Busby JR, Hutchinson MF (2014) Bioclim: the first species distribution modelling package, its early applications and relevance to most current maxent studies. Diversity and Distributions, 20, 1–9." />
<meta name="citation_reference" content="Burga CA, Hussendörfer E (2001) Vegetation history of Abies alba Mill. (silver fir) in Switzerland-pollen analytical and genetic surveys related to aspects of vegetation history of Picea abies (L.) H. Karsten (Norway spruce). Vegetation History and Archaeobotany, 10, 151–159." />
<meta name="citation_reference" content="Bürgi M, Schuler A (2003) Driving forces of forest management - an analysis of regeneration practices in the forests of the Swiss Central Plateau during the 19th and 20th century. Forest Ecology and Management, 176, 173–183." />
<meta name="citation_reference" content="Cailleret M, Nourtier M, Amm A, Durand-Gillmann M, Davi H (2014) Drought-induced decline and mortality of silver fir differ among three sites in Southern France. Annals of Forest Science, 71, 1–15. doi:10.1007/s13595-013-0265-0." />
<meta name="citation_reference" content="Chave J, Réjou-Méchain M, Burquez A, et al. (2014) Improved allometric models to estimate the aboveground biomass of tropical trees. Global Change Biology, 20, 3177–3190." />
<meta name="citation_reference" content="Chenoweth SF, Blows MW (2008) Qst meets the G matrix: The dimensionality of adaptive divergence in multiple correlated quantitative traits. Evolution, 62, 1437–1449. doi:10.1111/j.1558-5646.2008.00374.x." />
<meta name="citation_reference" content="Cooke JE, Eriksson ME, Junttila O (2012) The dynamic nature of bud dormancy in trees: environmental control and molecular mechanisms. Plant, Cell &amp; Environment, 35, 1707–1728." />
<meta name="citation_reference" content="Cornelissen JHC, Cerabolini B, Castro-Diez P, et al. (2003) Functional traits of woody plants: correspondence of species rankings between field adults and laboratory-grown seedlings? Journal of Vegetation Science, 14, 311–322." />
<meta name="citation_reference" content="Csilléry K, Kunstler G, Courbaud B, Allard D, Lassègues P, Haslinger K, Gardiner B (2017) Coupled effects of wind-storms and drought on tree mortality across 115 forest stands from the Western Alps and the Jura mountains. Global Change Biology, 23, 5092–5107. doi:10.1111/gcb.13773." />
<meta name="citation_reference" content="De Kort H, Vander Mijnsbrugge K, Vandepitte K, Mergeay J, Ovaskainen O, Honnay O (2016) Evolution, plasticity and evolving plasticity of phenology in the tree species Alnus glutinosa. Journal of Evolutionary Biology, 29, 253–264. doi:10.1111/jeb.12777." />
<meta name="citation_reference" content="Earl DA, et al. (2012) STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method. Conservation Genetics Resources, 4, 359–361." />
<meta name="citation_reference" content="Ellenberg H (1988) Vegetation ecology of central Europe. Cambridge University Press, Cambridge, UK." />
<meta name="citation_reference" content="Evanno G, Regnaut S, Goudet J (2005) Detecting the number of clusters of individuals using the software structure: a simulation study. Molecular Ecology, 14, 2611–2620." />
<meta name="citation_reference" content="Falconer DS, Mackay TFC (1996) Introduction to Quantitative Genetics. Longmans Green, Harlow, Essex, UK, 4 edn." />
<meta name="citation_reference" content="Falush D, Stephens M, Pritchard JK (2003) Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. Genetics, 164, 1567–1587." />
<meta name="citation_reference" content="Finkeldey R, Mátyás G, Sperisen C, Bonfils P (2000) Strategien zur Auswahl forstlicher Genreservate in der Schweiz. Forest Snow and Landscape Research, 75, 137–152." />
<meta name="citation_reference" content="Frank A, Howe GT, Sperisen C, Brang P, Clair JBS, Schmatz DR, Heiri C (2017a) Risk of genetic maladaptation due to climate change in three major European tree species. Global Change Biology, pp. 1–14. doi:10.1111/gcb.13802." />
<meta name="citation_reference" content="Frank A, Sperisen C, Howe GT, Brang P, Walthert L, Clair JBS, Heiri C (2017b) Distinct genecological patterns in seedlings of Norway spruce and silver fir from a mountainous landscape. Ecology, 98, 211–227. doi:10.1002/ecy.1632." />
<meta name="citation_reference" content="Franks SJ, Weber JJ, Aitken SN (2014) Evolutionary and plastic responses to climate change in terrestrial plant populations. Evolutionary Applications, 7, 123–139. doi:10.1111/eva.12112." />
<meta name="citation_reference" content="Galili T (2015) dendextend: an r package for visualizing, adjusting, and comparing trees of hierarchical clustering. Bioinformatics. doi:10.1093/bioinformatics/btv428." />
<meta name="citation_reference" content="Gazol A, Camarero JJ, Gutiérrez E, et al. (2015) Distinct effects of climate warming on populations of silver fir (Abies alba) across Europe. Journal of Biogeography, 42, 1150–1162." />
<meta name="citation_reference" content="Gilbert KJ, Whitlock MC (2015) QST – FST comparisons with unbalanced half-sib designs. Molecular Ecology Resources, 15, 262–267." />
<meta name="citation_reference" content="Hampe A, Jump AS (2011) Climate relicts: past, present, future. Annual Review of Ecology, Evolution, and Systematics, 42." />
<meta name="citation_reference" content="Harris I, Jones P, Osborn T, Lister D (2014) Updated high-resolution grids of monthly climatic observations-the cruts3. 10 dataset. International Journal of Climatology, 34, 623–642." />
<meta name="citation_reference" content="Hay LE, Wilby RL, Leavesley GH (2000) A comparison of delta change and downscaled GCM scenarios for three mountainous basins in the United States. JAWRA Journal of the American Water Resources Association, 36, 387–397." />
<meta name="citation_reference" content="Hengl T, de Jesus JM, Heuvelink GB, et al. (2017) SoilGrids250m: Global gridded soil information based on machine learning. PLoS one, 12, e0169748." />
<meta name="citation_reference" content="Hijmans RJ, Phillips S, Leathwick J, Elith J, Hijmans MRJ (2017) Package ‘dismo’. Circles, 9." />
<meta name="citation_reference" content="Hoffmann AA, Sgrò CM (2011) Climate change and evolutionary adaptation. Nature, 470, 479–485." />
<meta name="citation_reference" content="Hubisz MJ, Falush D, Stephens M, Pritchard JK (2009) Inferring weak population structure with the assistance of sample group information. Molecular Ecology Resources, 9, 1322–1332. doi:10.1111/j.1755-0998.2009.02591.x." />
<meta name="citation_reference" content="Iseli C, Jongeneel CV, Bucher P (1999) ESTScan: a program for detecting, evaluating, and reconstructing potential coding regions in EST sequences. In: ISMB: International Conference on Intelligent Systems for Molecular Biology, vol. 99, pp. 138–148." />
<meta name="citation_reference" content="Jakobsson M, Rosenberg NA (2007) CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure. Bioinformatics, 23, 1801–1806." />
<meta name="citation_reference" content="Karger DN, Conrad O, Böhner J, et al. (2017) Climatologies at high resolution for the earth’s land surface areas. Scientific Data, 4, 170122." />
<meta name="citation_reference" content="Karhunen M, Ovaskainen O (2012) Estimating population-level coancestry coefficients by an admixture F model. Genetics, 192, 609–617. doi:10.1534/genetics.112.140871." />
<meta name="citation_reference" content="Karhunen M, Ovaskainen O, Herczeg G, Merilä J (2014) Bringing habitat information into statistical tests of local adaptation in quantitative traits: A case study of nine-spined sticklebacks. Evolution, 68, 559–568. doi:10.1111/evo.12268." />
<meta name="citation_reference" content="Kremer A, Ronce O, Robledo-Arnuncio JJ, et al. (2012) Long-distance gene flow and adaptation of forest trees to rapid climate change. Ecology Letters, 15, 378–392. doi:10.1111/j.1461-0248.2012.01746.x." />
<meta name="citation_reference" content="Legendre P, Fortin MJ (2010) Comparison of the mantel test and alternative approaches for detecting complex multivariate relationships in the spatial analysis of genetic data. Molecular Ecology Resources, 10, 831–844." />
<meta name="citation_reference" content="Leinonen T, McCairns RS, O’hara RB, Merilä J (2013) QST – FST comparisons: evolutionary and ecological insights from genomic heterogeneity. Nature Reviews Genetics, 14, 179. doi:10.1038/nrg3395." />
<meta name="citation_reference" content="Lenormand T (2002) Gene flow and the limits to natural selection. Trends in Ecology &amp; Evolution, 17, 183–189." />
<meta name="citation_reference" content="Liepelt S, Bialozyt R, Ziegenhagen B (2002) Wind-dispersed pollen mediates postglacial gene flow among refugia. Proceedings of the National Academy of Sciences, 99, 14590–14594." />
<meta name="citation_reference" content="Liepelt S, Cheddadi R, de Beaulieu JL, et al. (2009) Postglacial range expansion and its genetic imprints in Abies alba (Mill.) - A synthesis from palaeobotanic and genetic data. Review of Palaeobotany and Palynology, 153, 139–149. doi:10.1016/j.revpalbo.2008.07.007." />
<meta name="citation_reference" content="Lind BM, Menon M, Bolte CE, Faske TM, Eckert AJ (2018) The genomics of local adaptation in trees: Are we out of the woods yet? Tree Genetics &amp; Genomes, 14, 29 doi:10.1007/s11295-017-1224-y." />
<meta name="citation_reference" content="Major JE, Mosseler A, Barsi DC, Campbell M, Rajora OP (2003) Morphometric, allometric, and developmentally adaptive traits in red spruce and black spruce. ii. seedling and mature tree assessment of controlled intra-and inter-specific hybrids. Canadian journal of forest research, 33, 897–909." />
<meta name="citation_reference" content="Martin G, Chapuis E, Goudet J (2008) Multivariate QST-Fst comparisons: A neutrality test for the evolution of the G matrix in structured populations. Genetics, 180, 2135–2149. doi:10.1534/genetics.107.080820." />
<meta name="citation_reference" content="Mosca E, Eckert AJ, Di Pierro EA, Rocchini D, La Porta N, Belletti P, Neale DB (2012) The geographical and environmental determinants of genetic diversity for four alpine conifers of the European Alps. Molecular Ecology, 21, 5530–5545." />
<meta name="citation_reference" content="Mousseau TA, Roff DA (1987) Natural selection and the heritability of fitness components. Heredity, 59, 181." />
<meta name="citation_reference" content="Ovaskainen O, Karhunen M, Zheng C, Arias JMC, Merila J (2011) A new method to uncover signatures of divergent and stabilizing selection in quantitative traits. Genetics, 189, 621632. doi:10.1534/genetics.111.129387." />
<meta name="citation_reference" content="Petit RJ, Aguinagalde I, de Beaulieu JL, et al. (2003) Glacial refugia: hotspots but not melting pots of genetic diversity. science, 300, 1563–1565." />
<meta name="citation_reference" content="Petit RJ, Hampe A (2006) Some evolutionary consequences of being a tree. Annual Review of Ecology, Evolution, and Systematics, 37, 187–214." />
<meta name="citation_reference" content="Pigliucci M, Preston KA (2004) Phenotypic integration: studying the ecology and evolution of complex phenotypes. Oxford University Press, Oxford, UK." />
<meta name="citation_reference" content="Pritchard JK, Stephens M, Donnelly PJ (2000) Inference of population structure using multilocus genotype data. Genetics, 155, 945–959." />
<meta name="citation_reference" content="Roschanski AM, Csillery K, Liepelt S, et al. (2016) Evidence of divergent selection for drought and cold tolerance at landscape and local scales in Abies alba Mill. in the French Mediterranean Alps. Molecular Ecology, 25, 776–794." />
<meta name="citation_reference" content="Roschanski AM, Fady B, Ziegenhagen B, Liepelt S (2013) Annotation and re-sequencing of genes from de novo transcriptome assembly of Abies alba (Pinaceae). Applications in Plant Sciences, 1, 1200179. doi:10.3732/apps.1200179." />
<meta name="citation_reference" content="Ruosch M, Spahni R, Joos F, Henne PD, van der Knaap WO, Tinner W (2016) Past and future evolution of Abies alba forests in Europe - comparison of a dynamic vegetation model with palaeo data and observations. Global Change Biology, 22, 727–740. doi:10.1111/gcb.13075." />
<meta name="citation_reference" content="Savolainen O, Pyhäjärvi T, Knurr T (2007) Gene flow and local adaptation in trees. Annual Review of Ecology, Evolution, and Systematics, 38, 595–619. doi:10.1146/annurev.ecolsys.38.091206.095646." />
<meta name="citation_reference" content="Shaw RG, Etterson JR (2012) Rapid climate change and the rate of adaptation: insight from experimental quantitative genetics. New Phytologist, 195, 752–765." />
<meta name="citation_reference" content="Stocker TF, Qin D, Plattner GK, et al. (2014) Climate change 2013: The physical science basis. Cambridge University Press Cambridge, UK, and New York." />
<meta name="citation_reference" content="Thornton K (2003) libsequence: A C++ class library for evolutionary genetic analysis. Bioinformatics, 19, 2325–2327. doi:10.1093/bioinformatics/btg316." />
<meta name="citation_reference" content="Van der Knaap W, van Leeuwen JF, Finsinger W, et al. (2005) Migration and population expansion of Abies, Fagus, Picea, and Quercus since 15000 years in and across the Alps, based on pollen-percentage threshold values. Quaternary Science Reviews, 24, 645–680." />
<meta name="citation_reference" content="Vilella AJ, Blanco-Garcia A, Hutter S, Rozas J (2005) VariScan: analysis of evolutionary patterns from large-scale DNA sequence polymorphism data. Bioinformatics, 21, 2791–2793." />
<meta name="citation_reference" content="Visscher PM, Hill WG, Wray NR (2008) Heritability in the genomics era - concepts and misconceptions. Nature Reviews Genetics, 9, 255–266. doi:10.1038/nrg2322." />
<meta name="citation_reference" content="Vitali V, Buntgen U, Bauhus J (2017) Silver fir and Douglas fir are more tolerant to extreme droughts than Norway spruce in south-western Germany. Global Change Biology. doi:10.1111/gcb.13774." />
<meta name="citation_reference" content="Waitt DE, Levin DA (1998) Genetic and phenotypic correlations in plants: a botanical test of Cheverud’s conjecture. Heredity, 80, 310–319." />
<meta name="citation_reference" content="Wells N, Goddard S, Hayes MJ (2004) A self-calibrating Palmer drought severity index. Journal of Climate, 17, 2335–2351. doi:10.1175/1520-0442(2004)017&lt;2335:ASPDSI&gt;2.0.CO;2." />
<meta name="citation_reference" content="Whitlock MC (2008) Evolutionary inference from QST. Molecular Ecology, 17, 1885–1896." />
<meta name="citation_reference" content="Whitlock MC, Guillaume F (2009) Testing for spatially divergent selection: comparing QST to FST. Genetics, 183, 1055–1063." />
<meta name="citation_reference" content="Yeaman S, Hodgins KA, Lotterhos KE, et al. (2016) Convergent local adaptation to climate in distantly related conifers. Science, 353, 1431–1433." />
<meta name="twitter:title" content="Adaptation to local climate in a multi-trait space: evidence from silver fir (Abies alba Mill.) populations across a heterogeneous environment" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Attempts to identify and understand selection pressures responsible for local adaptation are central to evolutionary research. We tested whether populations of silver fir ( Abies alba Mill.), sampled across a heterogeneous environment, have more strongly diverged at quantitative traits than expected from genetic drift. We genotyped 387 trees from 19 Swiss populations at 374 single-nucleotide polymorphisms (SNPs) to estimate their demographic distances, and used these to generate a null expectation for divergence in a multi-trait space including morphology and life-history traits obtained from a published common garden trial. Local soil and historical climate data were used to identify the selective environment of the source populations. Our results revealed a strong selection on height driven by temperature: trees from warm sites evolved to become taller than those from cooler sites. The evolution of growth rate, growth duration and bud break were correlated, and populations evolved towards two extreme strategies, “start early and grow slow” or “start late and grow fast”, driven by precipitation seasonality. We conclude that local climate has shaped the morphology and life-history of silver fir populations since they recolonized the Alps. Our methodology provides a show-case for empirical evaluation of adaptive evolutionary strategies combining genetic data and common gardens.

### Author contributions

KC designed the study and obtained the SNP data with help from AW and FG. KC performed all analysis with advice from OO. CS was involved in the published common garden experiment and provided help with data interpretation. KC wrote the manuscript with help from all other authors.

We thank all members of the ADAPT project that provided the phenotypic data for this study, especially Aline Frank, Caroline Heiri and Peter Brang. Additionally, Andrea Kupferschmid helped with correcting some errors in the phenotypic data thanks to her follow-up project (EADAPT). Part of the sampling was conducted by Catherine Folly. The project was funded by a research grant from the Center for Adaptation to a Changing Environment (ACE) at the ETH Zürich. KC was supported by ACE fellowship while obtaining and analyzing the data, and by a Marie Sklodowska-Curie fellowship (FORGENET), while working on the manuscript. We thank Kristian Ullrich for help with the SNP selection, and Dirk Krager with extracting the climatic data from the data bases. The contribution of several field and lab workers was necessary for this project, in particular, we thank René Graf and Olivier Charlandie." />
<meta name="og-title" property="og:title" content="Adaptation to local climate in a multi-trait space: evidence from silver fir (Abies alba Mill.) populations across a heterogeneous environment" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/292540v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Attempts to identify and understand selection pressures responsible for local adaptation are central to evolutionary research. We tested whether populations of silver fir ( Abies alba Mill.), sampled across a heterogeneous environment, have more strongly diverged at quantitative traits than expected from genetic drift. We genotyped 387 trees from 19 Swiss populations at 374 single-nucleotide polymorphisms (SNPs) to estimate their demographic distances, and used these to generate a null expectation for divergence in a multi-trait space including morphology and life-history traits obtained from a published common garden trial. Local soil and historical climate data were used to identify the selective environment of the source populations. Our results revealed a strong selection on height driven by temperature: trees from warm sites evolved to become taller than those from cooler sites. The evolution of growth rate, growth duration and bud break were correlated, and populations evolved towards two extreme strategies, “start early and grow slow” or “start late and grow fast”, driven by precipitation seasonality. We conclude that local climate has shaped the morphology and life-history of silver fir populations since they recolonized the Alps. Our methodology provides a show-case for empirical evaluation of adaptive evolutionary strategies combining genetic data and common gardens.

### Author contributions

KC designed the study and obtained the SNP data with help from AW and FG. KC performed all analysis with advice from OO. CS was involved in the published common garden experiment and provided help with data interpretation. KC wrote the manuscript with help from all other authors.

We thank all members of the ADAPT project that provided the phenotypic data for this study, especially Aline Frank, Caroline Heiri and Peter Brang. Additionally, Andrea Kupferschmid helped with correcting some errors in the phenotypic data thanks to her follow-up project (EADAPT). Part of the sampling was conducted by Catherine Folly. The project was funded by a research grant from the Center for Adaptation to a Changing Environment (ACE) at the ETH Zürich. KC was supported by ACE fellowship while obtaining and analyzing the data, and by a Marie Sklodowska-Curie fellowship (FORGENET), while working on the manuscript. We thank Kristian Ullrich for help with the SNP selection, and Dirk Krager with extracting the climatic data from the data bases. The contribution of several field and lab workers was necessary for this project, in particular, we thank René Graf and Olivier Charlandie." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-04-19" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/292540v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/292540v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/292540v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/292540v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/93581" />
    <title>Adaptation to local climate in a multi-trait space: evidence from silver fir (Abies alba Mill.) populations across a heterogeneous environment | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__vup9YXfxWOBOGbthJCUtxdPS6xRY2OQ_vXqbojfxPAk__bSlZZj-vtmxJTtkr5S_WmHsMReVxViXzXXCAOZ2J91g__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"es8515BV2ORcK1D6hmlxoHizo3BDwMw5Zim-_fPqiTA","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"93581","apath":"\/biorxiv\/early\/2018\/04\/19\/292540.atom","pisa":"biorxiv;292540v2","markup":[{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;292540\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;292540\/DC1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;292540v2"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;292540\/DC1"},{"requested":"index","variant":"index","view":"full","pisa":"biorxiv;292540\/DC1"}],"processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/292540v2","title":"Adaptation to local climate in a multi-trait space: evidence from silver fir (Abies alba Mill.) populations across a heterogeneous environment","identifier":"node\/93581"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002222VZ+0rQkoKOL9qCdFe7zdhUNwCooxUylqIM46A99zC1z6X1LSNGplxkuWXEcE2M+KhmaEfXa+HtCmD9Ak9l0rwnUxXu95GJXj0yNZwdcgD1vjA95Sx60owZdvq5\\\\\\\/DhxPX9CB6cSSXp8RJvSpadw8j8cDHx1kXHx6nDu501ikK3fNTvq0yGqihuLNNyo8YHFcdEzM3tAOAHoO3eTI5qCmPIva6G9mAVnrrcIJEREJv0bUN\\\\\\\/dwHzI+oiIOvZjEN3KK11t10xNYrRe36FqhOCUFPBX3YdFImmTnL5cvu72SGwqhuPZyOfSy1xWus6VArUostISAxfRbw7U+sN\\\\\\\/pmF599lDgIy+K6NsPfPn7P3lomOyGySweXxmBWmp\\\\\\\/HOxlMBVJQMY1+3nd+ogobLiP53ni++Q9dPkQ4eHXYCAWfV5fcaZsedWsfRyknUXG123\\\\\\\/vQjU61iTZctT2UWGrQrTK4WOaQo8H2pHi31rVBPgEErdrlzwjZOPvRtJDXhnA0+o4znqnMxZaJyd46kguRfp+glxumeb5vk83py2wRDz0sM\\\\\\\/D3HSfEosdMcDOyG\\\\\\\/r9kx5ko79zIXIx5+MqBk0SQ2LABj22BPlIpdwrxDL5p5eghe0kAVBqh9\\\\\\\/L1YBFrYf20m68\\\\\\\/TMAGvXncs7ebg0ybKzf62j4v\\\\\\\/pRb5TV4tVetW2tmNUXgpWTEOjjK9x3vHZ6wd+c6sShqGZO0GCxLczyBtVJnjo1KbDQe6nSrNitEw\\\\\\\/\\\\\\\/kaggYG3iH6Yh2ezxjTOEDbSWu95dB61Jwc\\\\\\\/NoYGEl6ZJQdtJd1B8oZY6jIUoXNtrOsWU2g4Qsxw7\\\\\\\/TgsVKmpzEdq4Slaawg8nUxXwNGh\\\\\\\/ltDUDIjxkK7filbwq1Y4jN4WLzJKr8nwgpYUyjSvRth4x0exXUQ2B+wja0o+15FzFP0CUAkbgNTRH5TnySk54FSrM9ecixswvLS83z0sr85QkzfACuhsXu9nz04jWQvGt+\\\\\\\/ZvZw4DHLrENR8\\\\\\\/saV80JaYJYmC8p5mEa2imXuzwxTazYFm1mLp4bvLA4KVEo+HGwtX45Ln+uaSw+Mu9BK7mj849w8ehK74bjUC0T+rvMJ51LtwWZ6gw+9nfhReh+ssS5BUcjkqqp3+DVWzDUDvLoN2L8dcnfkbmhCdS\\\\\\\/i2tyldTlWyHvz3O2S980lNcQTQUOJr07bc\\\\\\\/4iitwUss2NohB9sMKK+kkIG7L5HVxvgYmAXdBBfr2RFqvtNkUvlS67I0vNCmeWxgUuBFtPJR1GnbxpcIGBHkE4BimWI5WHf2bBb2+Or+70Gt0XipGKa8rbNbKOPbQDiyxbIrSSpVEOQpiSX5Q=\\\u0022,\\\u0022iv\\\u0022:\\\u0022AeDsIONg5MBbeD3eXqBg+w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00224ec8b0c7dfc72ba653f77ee6d4fbea13e92dbb790da8e202270efa8b080c0d44\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/292540v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-93581 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/292540v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-26_cSCtY9UXpxBW6Bk8H2yuhZRO1hIRlGbRPElS9Y9w" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="93581" id="node93581" data-pisa="biorxiv;292540v2" data-pisa-master="biorxiv;292540" data-apath="/biorxiv/early/2018/04/19/292540.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Adaptation to local climate in a multi-trait space: evidence from silver fir (<em>Abies alba</em> Mill.) populations across a heterogeneous environment</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0039-9296" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Katalin</span> <span class="nlm-surname">Csilléry</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9750-4421" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Otso</span> <span class="nlm-surname">Ovaskainen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1241-5636" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Christoph</span> <span class="nlm-surname">Sperisen</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-8253-5137" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Alex</span> <span class="nlm-surname">Widmer</span></span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-3878-1845" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">Felix</span> <span class="nlm-surname">Gugerli</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/292540 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node93581" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katalin Csilléry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Evolutionary Biology and Environmental Studies, University of Zürich, Zürich</span>, <span class='nlm-country'>Switzerland</span></div><div class='author-affiliation'><span class='nlm-institution'>Swiss Federal Research Institute WSL, Birmensdorf</span>, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katalin%2BCsill%C3%A9ry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Csill%C3%A9ry%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatalin%2BCsill%25C3%25A9ry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0039-9296" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katalin Csilléry</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Otso Ovaskainen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Faculty of Biological and Environmental Sciences, University of Helsinki, Helsinki</span>, <span class='nlm-country'>Finland</span></div><div class='author-affiliation'><span class='nlm-institution'>Centre for Biodiversity Dynamics, Department of Biology, Norwegian University of Science and Technology, Trondheim</span>, <span class='nlm-country'>Norway</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Otso%2BOvaskainen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ovaskainen%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOtso%2BOvaskainen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9750-4421" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Otso Ovaskainen</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Christoph Sperisen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Swiss Federal Research Institute WSL, Birmensdorf</span>, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christoph%2BSperisen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sperisen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristoph%2BSperisen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1241-5636" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christoph Sperisen</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alex Widmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute of Integrative Biology, ETH Zürich, Zürich</span>, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BWidmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Widmer%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlex%2BWidmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8253-5137" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alex Widmer</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Felix Gugerli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Swiss Federal Research Institute WSL, Birmensdorf</span>, <span class='nlm-country'>Switzerland</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Felix%2BGugerli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gugerli%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFelix%2BGugerli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3878-1845" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Felix Gugerli</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/292540v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:93581" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:93581/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/292540v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:93581" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:93581/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/292540v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:93581" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:93581/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/292540v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:93581" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:93581/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/292540v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:93581" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:93581/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/292540v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:93581" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:93581/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-4">Attempts to identify and understand selection pressures responsible for local adaptation are central to evolutionary research. We tested whether populations of silver fir (<em>Abies alba</em> Mill.), sampled across a heterogeneous environment, have more strongly diverged at quantitative traits than expected from genetic drift. We genotyped 387 trees from 19 Swiss populations at 374 single-nucleotide polymorphisms (SNPs) to estimate their demographic distances, and used these to generate a null expectation for divergence in a multi-trait space including morphology and life-history traits obtained from a published common garden trial. Local soil and historical climate data were used to identify the selective environment of the source populations. Our results revealed a strong selection on height driven by temperature: trees from warm sites evolved to become taller than those from cooler sites. The evolution of growth rate, growth duration and bud break were correlated, and populations evolved towards two extreme strategies, “start early and grow slow” or “start late and grow fast”, driven by precipitation seasonality. We conclude that local climate has shaped the morphology and life-history of silver fir populations since they recolonized the Alps. Our methodology provides a show-case for empirical evaluation of adaptive evolutionary strategies combining genetic data and common gardens.</p></div><div class="bio" id="bio-1"><h3>Author contributions</h3><p id="p-5">KC designed the study and obtained the SNP data with help from AW and FG. KC performed all analysis with advice from OO. CS was involved in the published common garden experiment and provided help with data interpretation. KC wrote the manuscript with help from all other authors.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-6">We thank all members of the ADAPT project that provided the phenotypic data for this study, especially Aline Frank, Caroline Heiri and Peter Brang. Additionally, Andrea Kupferschmid helped with correcting some errors in the phenotypic data thanks to her follow-up project (EADAPT). Part of the sampling was conducted by Catherine Folly. The project was funded by a research grant from the Center for Adaptation to a Changing Environment (ACE) at the ETH Zürich. KC was supported by ACE fellowship while obtaining and analyzing the data, and by a Marie Sklodowska-Curie fellowship (FORGENET), while working on the manuscript. We thank Kristian Ullrich for help with the SNP selection, and Dirk Krager with extracting the climatic data from the data bases. The contribution of several field and lab workers was necessary for this project, in particular, we thank René Graf and Olivier Charlandie.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-7">With increasing concerns about climate change, estimating the adaptive potential of forest trees has high societal and economic relevance. The capacity of many forest tree species to survive and reproduce across a broad geographical range is, at least partly, due to adaptation to local conditions (<em>e.g</em>. <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aitken et <em>al</em>., 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Franks et <em>al</em>., 2014</a>). There is growing evidence that adaptation can be important for enabling populations to persist under changing conditions, including climate change (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hoffmann &amp; Sgrò, 2011</a>). However, many populations are increasingly exposed to novel environmental conditions that are beyond the range to which they are adapted and/or outside of their physiological tolerance (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Shaw &amp; Etterson, 2012</a>), and it is unclear how commonly and rapidly adaptations or plastic responses would arise in order to keep up with climate change (e.g. <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Franks et <em>al</em>., 2014</a>).</p><p id="p-8">Many forest tree species have distribution ranges that expand over large geographic regions, which is, at least in the Northern Hemisphere, a result of recent population expansions after the Last Glacial Maximum (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hampe &amp; Jump, 2011</a>). Given the large effective population sizes, predominant outcrossing mode of reproduction, and moderate to high levels of genetic diversity, forest tree species have been able to adapt to local conditions, as reflected by ample empirical evidence of trait divergence patterns that are congruent with latitudinal and altitudinal climatic gradients (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Savolainen et <em>al</em>., 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alberto et <em>al</em>., 2013</a>). However, since many of these gradients also coincide with recent population expansion, it can be difficult to tease apart neutral and selective processes underlying trait variation (e.g. <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yeaman et <em>al</em>., 2016</a>).</p><p id="p-9">Attempts to identify adaptive trait differentiation in forest trees have often relied on the long tradition of provenance trials, in which different populations from across the geographic range are tested in a common environment. Phenotypic data is often exploited using so-called <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> tests, which compare quantitative trait differentiation (<em>Q<sub>ST</sub></em>) with genetic divergence inferred from neutral markers (<em>F<sub>ST</sub></em>) (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Whitlock, 2008</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Whitlock &amp; Guillaume, 2009</a>). A recent literature survey of forest trees suggests that most tree traits exhibit considerable heritable trait variation and that <em>Q<sub>ST</sub></em> most often (69% of the studies)exceeds <em>F<sub>ST</sub></em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lind <em>et al</em>., 2018</a>). However, a simple comparison between <em>Q<sub>ST</sub></em> and <em>F<sub>ST</sub></em> has many shortcomings that have been criticized (e.g. reviewed in <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Leinonen et <em>al</em>. (2013)</a>), and alternative solutions have been suggested (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chenoweth &amp; Blows, 2008</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Martin et <em>al</em>., 2008</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ovaskainen et <em>al</em>., 2011</a>). The fundamental issue is that both <em>Q<sub>ST</sub></em> and <em>F<sub>ST</sub></em> are random variables, i.e. the observed values are just a single realization of the evolutionary process. Even though <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Whitlock &amp; Guillaume (2009)</a> proposed an improved <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> test where <em>Q<sub>ST</sub></em> is compared to a simulated distribution of <em>F<sub>ST</sub></em>, this test still does not fully account for the demographic relationships among the populations, because it is reduced to a single summary of the data, the <em>F<sub>ST</sub></em>. Further, the <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> test may also give misleading results because each trait is considered in isolation, and the signature of adaptive trait differentiation can be the result of selection at another trait if they are genetically correlated (e.g. <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chenoweth &amp; Blows, 2008</a>). <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Ovaskainen et <em>al</em>. (2011)</a> proposed an alternative method, which uses a statistically more powerful null hypothesis than the <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> tests: it accounts for drift distances among all populations (i.e. the coancestry matrix) to derive the null expectation of trait divergence. So far, however, only a handful of applications of this method have been published (e.g. <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">De Kort et <em>al</em>., 2016</a>).</p><p id="p-10">Here, we assess adaptive divergence at size, growth and phenology traits in silver fir (<em>Abies alba</em> Mill.), an ecologically and economically important European forest tree species. Indeed, recent dendrological (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Vitali et <em>al</em>., 2017</a>) and genecological (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Frank et <em>al</em>., 2017a</a>) evidence suggests silver fir to potentially be better adapted to future climatic conditions, notably to drought, than the other dominant conifer of European forests, Norway spruce (<em>Picea abies</em> Karst.), which often co-occurs with silver fir. In this study, we concentrate on populations from ecologically diverse sites across Switzerland, including populations from core ranges (Pre-Alps, Central Plateau and Jura Mountains), but also marginal populations isolated by high mountain ranges. These alpine populations encompass the wet outer Alpine chain to the dry Central Alps across diverse soil types (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ellenberg, 1988</a>). Silver fir populations in Switzerland are rather young, palynological and dynamic vegetation modeling suggest that the Swiss Alps were colonized by silver fir from south to north after the Last Glacial Maximum. The species most likely reached the southern slopes of the Alps between 10 and 9 kyr BP and the northern slopes between 8 and 5 kyr BP (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Van der Knaap <em>et al</em>., 2005</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Liepelt <em>et al</em>., 2009</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ruosch <em>et al</em>., 2016</a>). Range-wide patterns of chloroplast and mitochondrial DNA variation (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Liepelt et <em>al</em>., 2002</a>) and isozyme data (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Burga &amp; Hussendörfer, 2001</a>) from extant silver fir populations suggest that the Swiss Alps were colonized from a single ancestral refugial population situated in the Central and/or Northern Apennines, even though the potential contribution of eastern refugial populations cannot be excluded.</p><p id="p-11">The potential for adaption can be the greatest in isolated populations and poor environmental conditions, however, gene flow from the core areas can easily counteract selection (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lenormand, 2002</a>). Many other wind-pollinated conifer species have high levels of pollen-mediated gene flow (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Petit &amp; Hampe, 2006</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kremer et <em>al</em>., 2012</a>). Thus, regardless of the heterogeneous environment across Switzerland, little adaptive trait differentiation is expected among the studied populations. Indeed, results of a large-scale common-garden experiment involving seedlings from 90 silver fir populations across Switzerland show that levels of population divergence are rather low, particularly when compared to those of Norway spruce from the same area (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Frank et <em>al</em>., 2017b</a>). For example, population divergence(<em>Q<sub>ST</sub></em>) in seedling height for silver fir was 0.22 and in phenological traits ≤ 0.11. Nevertheless, <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Frank et <em>al</em>. (2017b)</a> also found that <em>Q<sub>ST</sub></em> values of these traits are much higher than population differentiation at isozyme loci (<em>F<sub>ST</sub></em> = 0.034) in a similar set of populations (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Finkeldey et <em>al</em>., 2000</a>) and speculated that population divergence might reflect adaptation to local environmental conditions. A further evidence for possible adaptation was that population differentiation in seedling height showed a regional pattern, i.e. seedlings from the Central Plateau were taller than those from the Alps, and environmental variables explained a large proportion of the trait differentiation (49%) (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Frank et <em>al</em>., 2017b</a>).</p><p id="p-12">Here, we revisit the phenotypic data obtained by <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Frank <em>et al</em>. (2017b)</a> for a subset of 19 populations, complement it with modern genetic data, and assess to what extent the observed trait differentiation is due to selection. The goal of this study is three-fold. First, we test for adaptive trait divergence using the multi-trait test developed by <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Ovaskainen <em>et al</em>. (2011)</a>, thus we account for the full demography in devising the null expectation for trait divergence. Since, we have data from a relatively large number of populations and from a highly heterogeneous environment separated by potential barriers to gene flow, accounting for the likely complex and hierarchical population structure is essential. Further, the studied set of size, growth and phenology traits most likely do not evolve independently, thus the multi-trait approach is highly desirable to identify any correlated responses to selection. Second, we develop a set of downscaled historical climate variables and adopt soil data from <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Frank et <em>al</em>. (2017b)</a> to test which one of them drives the adaptive divergence patterns using the methodology described in <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Karhunen et <em>al</em>. (2014)</a>. Third, we wish to advocate the methodology developed herein to leverage information that can be obtained from a common-garden trial for testing hypothesis about adaptive evolution. In particular, we contrast our results with the simple <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em>, and propose two methodological improvements to evaluate the signature of selection in each population and a more powerful version of the H-test proposed by <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Karhunen et <em>al</em>. (2014)</a>.</p></div><div class="section" id="sec-2"><h2 class="">Material and Methods</h2><div id="sec-3" class="subsection"><h3>Study sites and sampling</h3><p id="p-13">In 2009, 90 silver fir stands were selected across Switzerland in the framework of a genecological study (<a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Frank et <em>al</em>., 2017b</a>). From each population, three mother trees were selected from an area with a relief as uniform as possible and with at least 100m distance between them. From this study, we selected a subset of 19 presumably allochthonous populations that represented the main geographic regions of Switzerland (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), and re-visited them in 2013 and 2016. In spring 2013, needles were sampled from nine to ten adult trees, including the three mother trees, if they were found. In spring 2016, needles were sampled from an additional ten to twelve trees, always respecting a minimum distance of 100m between the newly or previously sampled trees. In total, 387 adult trees were used in this study, with at least 20 individuals per population. Needles were stored in plastic bags for a maximum of one week at 5°<em>C</em> before being freeze-dried and prepared for DNA extraction.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: Geographic location of the sampling sites. B: Drift distances between populations as estimated with the admixture F-model (AFM). Coancestry between populations is the mean of the posterior means from 10 independent Markov chains. Distances were calculated from the coancestry matrix to draw the dendrogram." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1564583975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A: Geographic location of the sampling sites. B: Drift distances between populations as estimated with the admixture F-model (AFM). Coancestry between populations is the mean of the posterior means from 10 independent Markov chains. Distances were calculated from the coancestry matrix to draw the dendrogram.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F1.medium.gif" width="440" height="395"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F1.medium.gif" width="440" height="395"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <p id="p-14" class="first-child">A: Geographic location of the sampling sites. B: Drift distances between populations as estimated with the admixture F-model (AFM). Coancestry between populations is the mean of the posterior means from 10 independent Markov chains. Distances were calculated from the coancestry matrix to draw the dendrogram.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Genetic data</h3><p id="p-15">All adult trees were genotyped at 374 single-nucleotide polymorphism (SNP) loci originating from three different sources. First, we used SNPs discovered from 13 <em>A. alba</em> trees originating from Mont Ventoux (France) and the Black Forest (District Oberharmersbach, Germany). Transcriptomes were sequenced on an Illumina HiSeq2000 at IGA Technology Services (Udine, Italy). Transcriptome assembly and SNP detection was carried out with an in-house pipeline at IGA; see details in <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Roschanski <em>et al</em>. (2013)</a>. Previous SNP genotyping using KASP assays manufactured by LGC Genomics (Middlesex, United Kingdom) in French populations yielded 267 polymorphic loci situated in 175 different candidate genes (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Roschanski et <em>al</em>., 2016</a>). In this study, 220 of these loci were successfully genotyped using the same KASP assays. Second, we selected 110 new putatively neutral SNPs from the transcriptome assembly based on their Tajima’s D being between 2 and −2 and dN/dS between 0.9 and 1.1, low LD with the existing 224 SNPs (r2 &lt; 0.1 and p-values &gt; 0.05). ESTScan version 3.0.3 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Iseli et <em>al</em>., 1999</a>) for <em>Arabidopsis thaliana</em> was used to predict coding DNA sequences for each contig from the transcriptome (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Roschanski <em>et al</em>., 2013</a>), then the SNPs detected by IGA were used to create pseudo-contigs on which the neutrality statistics were calculated. dN/dS was calculated with ‘polydNdS’<sup><a id="xref-fn-3-1" class="xref-fn" href="#fn-3">1</a></sup> based on (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Thornton, 2003</a>), and Tajima’s D with VariScan v2.0.3 (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Vilella <em>et al</em>., 2005</a>). Although all 110 SNPs passed initial in silico design tests, only 25 of these SNPs were successfully genotyped using KASP technology, most likely because the primer sequences were not specific enough (results not shown). Third, we selected 149 SNPs from <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mosca <em>et al</em>. (2012)</a>, who discovered SNPs by resequencing 800 amplicons using PCR primer pairs from <em>Pinus taeda</em> in 12 <em>Abies alba</em> trees, and genotyped on a Golden Gate platform (Illumina, San Diego, CA, USA). We selected SNPs from this study that had less than 5% missing data. One hundred and twenty-nine of these SNPs were successfully genotyped with KASP. Both DNA extraction and all genotyping was performed using the all-inclusive service from LGC Genomics (Middlesex, United Kingdom).</p></div><div id="sec-5" class="subsection"><h3>Common garden experiment</h3><p id="p-16">We used published seedling phenotype data from the descendants of three mother trees from each of the 19 populations (<a id="xref-ref-25-7" class="xref-bibr" href="#ref-25">Frank <em>et al</em>., 2017b</a>), while correcting inconsistencies and outliers based on measurements taken in subsequent years (unpublished data of Andrea Kupferschmid). Here, we briefly summarize the protocol of the experiment relevant for silver fir. In April 2010, from each mother tree approximately 2000 seeds (subsequently called family) were sown directly into nursery beds and grown for two years at the Swiss Federal Research Institute WSL in Birmensdorf, Switzerland (47°21′42″<em>N</em>, 8°27′22″<em>E</em>, 550 m a.s.l.). Families and populations were not replicated or randomized in the nursery. In spring 2012, at least 12 randomly selected viable seedlings per family with present terminal buds were transplanted to the open experimental field site. The field site was located at Brunnersberg, a former pasture on a south facing slope (20-24% incline) in the Swiss Jura Mountains (47°19′35″<em>N</em>, 7°36′42"<em>E</em>, 1090 m a.s.l.). Seedlings were planted at 30 × 40 cm spacing in 10 blocks. Provenances, families and seedlings were randomized across blocks. Mortality during 2012 in the field was minor. Seedling height and diameter were measured in the autumn of 2012 and used in the analysis as a covariate to account for possible growth differences in the nursery.</p><p id="p-17">Phenotypic measurements used herein were performed during the growing season of 2013. Measurements of seedling Height (variable names capitalized hereafter) and Terminal and Lateral Bud Break started on 7 May 2013, when the first seedlings started to break buds, i.e. the membrane below bud scales was broken and the first green needles became visible. Observations of bud break dates continued twice a week until Growth Cessation, defined as the date when 95% of terminal leader height growth was achieved. Five to 17 height measures were recorded per seedling from which a Maximum Growth Rate was calculated as the first derivative of the growth curve (for simplicity, subsequently called Growth Rate). Growth Duration was calculated as time from Terminal Bud Break to Growth Cessation. Final seedling Height from ground surface to the uppermost bud base and stem Diameter at 2cm above ground surface were measured in autumn 2013. Finally, we also calculated the ratio Height/Diameter to test for trait divergence in growth allometry. For clarity, we call Height, Diameter, Height/Diameter size traits, Growth Rate and Duration growth traits, and Terminal/Lateral Bud Break and Growth Cessation (equivalent to bud set) phenology traits. All traits were normally distributed and correlated with one another to a varying extent (Figure S1).</p></div><div id="sec-6" class="subsection"><h3>Environmental data</h3><p id="p-18">We used soil characteristics and downscaled historical climatic data to characterize environmental conditions in each population. Soil and geographic variables were adopted from <a id="xref-ref-25-8" class="xref-bibr" href="#ref-25">Frank et <em>al</em>. (2017b)</a>, who established soil pits within a few meters of one of the mother trees (see list of variables used in Table S1). In order to obtain the closest representation of the climate of the period when the current populations were established, we used data from 1 January 1901 to 31 December 1978. The choice of this period was justified by two facts: (i) no observation-based climate data go back further in time, and (ii) starting from approximately 1980, the temperature time series are overwhelmed by the effect of global warming (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Harris et <em>al</em>., 2014</a>). We used statistical downscaling using the delta method (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hay et <em>al</em>., 2000</a>) to obtain 1 km grid scale mean monthly temperature and precipitation fields for this period. The reference climatic data set was the 0.5° resolution CRU TS v. 4.01 data (20 September 2017 release, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Harris et <em>al</em>. (2014)</a>) available for the 1 January 1901 - 31 December 2016 period, while the downscaling was based on the overlapping period of 1 January 1979 - 31 December 2016 with the 1 km resolution CHELSA data (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Karger et <em>al</em>., 2017</a>).</p><p id="p-19">We calculated several bioclimatic indices to characterize the environment for each site, including the 19 bioclimatic variables (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Booth et <em>al</em>., 2014</a>) implemented in the R package dismo (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hijmans et <em>al</em>., 2017</a>), two potential evapotranspiration (PET) indices and four standardized precipitation - evapotranspiration index (SPEI) variables using the R package SPEI (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Beguería &amp; Vicente-Serrano, 2017</a>), and two indicators of late frost (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Soil available water capacity (AWC) was obtained at a 250 m resolution from the Soilgrids data base (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hengl et <em>al</em>., 2017</a>). Using temperature, precipitation and AWC, we calculated the self-calibrated Palmer’s drought severity index or scPDSI (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wells et <em>al</em>., 2004</a>), and summarized to measures of drought severity and frequency across the full time series (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). All climatic variables were considered as raw values or as deviations from the common garden environment in Brunnersberg (based on the CHELSA data for the period of 1 January 1979 - 31 December 2013). However, the two ways of calculating the climate led to the same conclusions (results not shown), so we present results with the raw variables only for ease of interpretation. To visualize the environmental differences between sites, we conducted a Principal Component Analysis (PCA) of all environmental variables of <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Tables 1</a> and S1, scaled and centered to make them comparable. The first PCA explained 32.6% of the variance and was dominated by temperature variables, and the second PCA axis explained 18.8% of the variance and was dominated by precipitation variables (Figure S2).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/476081/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/476081/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/476081" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1:</span> <p id="p-20" class="first-child">Climatic variables developed in this study using statistical downscaling from CHELSA (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Karger et <em>al</em>., 2017</a>) and the CRU TS v. 4.01 data (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Harris et <em>al</em>., 2014</a>). Monthly mean, minimum and maximum temperature and total precipitation were used to calculate the bioclimatic, PET, SPEI and frost variables. Soil AWC was extracted from the SoilGrids data base (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Hengl et <em>al</em>., 2017</a>), and used for the calculation of the scPDSI variables.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Statistical analysis</h3><p id="p-21">We used both single-trait and multi-trait approaches to test if the phenotypic divergence between populations is due to selection. First, we used a single-trait <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> test using the bootstrap procedure described in <a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Whitlock &amp; Guillaume (2009)</a> implemented in the R package QstFstComp (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gilbert &amp; Whitlock, 2015</a>)<sup><a id="xref-fn-4-1" class="xref-fn" href="#fn-4">2</a></sup>. We considered a one-tailed test, because we were interested in testing for adaptive divergence only, thus <em>Q<sub>ST</sub></em> being significantly greater than <em>F<sub>ST</sub></em>. Second, we used single- and multi-trait versions of the test of adaptive divergence (S-test) developed by <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Ovaskainen et <em>al</em>. (2011)</a> and implemented in the R package driftsel<sup><a id="xref-fn-5-1" class="xref-fn" href="#fn-5">3</a></sup>. We estimated the coancestry matrix between all pairs of populations from variation in SNP allele frequencies based on the admixture F-model (AFM) using a Metropolis-Hastings algorithm implemented in the R package RAFM<sup><a id="xref-fn-6-1" class="xref-fn" href="#fn-6">4</a></sup> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Karhunen &amp; Ovaskainen, 2012</a>). Thus, we assumed that contemporary populations evolved from a single ancestral population, which is likely a reasonable assumption for silver fir in Switzerland (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Liepelt et <em>al</em>., 2002</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Burga &amp; Hussendörfer, 2001</a>). Ten independent Markov chains of the AFM model reached slightly different local optima using a burn-in of 20,000 iterations followed by 10,000 iterations for estimation with a thinning interval of ten. Running longer chains (up to 50,000 iterations tested, results not shown) did not improve convergence to a common optimum, so we took the mean of the posterior means across the ten chains. Further, to gain more confidence in the estimated coancestry matrix, we compared the posterior mean coancestry matrix against that estimated using the Bayesian clustering algorithm implemented in the software Structure v.2.3.4 <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pritchard et <em>al</em>. (2000)</a> (see Supporting Information for more details). Third, we used the H-test, developed by <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Karhunen et <em>al</em>. (2014)</a>, and a standardized version of it, subsequently referred to <em>H</em>*, to identify the potential environmental drivers of local adaptation. In the following paragraphs, we briefly summarize the rationale of the <em>S</em>-test, and its population-wise version, proposed herein, and the rationale of the proposed <em>H</em>* statistic.</p><p id="p-22">Following the notation of <a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">Ovaskainen et <em>al</em>. (2011)</a>, if we consider only additive effects, the vector of additive values of all traits for individual <em>i</em> is <strong>a</strong><em><sub>i</sub></em>, and the matrix of additive vectors for all individuals is <strong>A</strong> = (<strong>a</strong><em><sub>i</sub></em>)<em><sub>i</sub></em>. Then, the mean additive value of population <em>X</em>, <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/inline-graphic-1.gif"/></noscript></span></span>, can be obtained as the mean of the additive values of all individuals in population X, and the matrix of additive vectors for all populations is <strong>A</strong><sup><em>P</em></sup> = <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/inline-graphic-2.gif"/></noscript></span></span><em><sub>X</sub></em>. When populations are derived from a common ancestral population and the trait values are normally distributed, under drift, the matrix of population-level effects, <strong>A</strong><em><sup>P</sup></em>, is expected to follow the multivariate normal distribution as
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/graphic-3.gif"/></noscript></span>

</span>
where <em><strong>μ</strong></em> is the vector of expected additive trait means determined by the allele frequencies in the ancestral population, <em>n<sub>P</sub></em> is the number of populations and <strong>I</strong><em><sub>np</sub></em> is an <em>n<sub>P</sub> × n<sub>P</sub></em> identity matrix, <em><strong>G</strong><sup>A</sup></em> is the ancestral variance-covariance matrix, <em><strong>θ</strong><sup>P</sup></em> is the population-to-population coancestry matrix, and ⊗ is the Kroenecker product. <em><strong>θ</strong><sup>P</sup></em> can be estimated assuming the admixture F-model, while <em><strong>μ</strong></em>, <strong>A</strong><em><sup>p</sup></em>, and <em><strong>G</strong><sup>A</sup></em> can be co-estimated using the Bayesian mixed-effects animal model accounting for the family structure of the common garden (i.e. the pedigree) and <em><strong>θ</strong><sup>P</sup></em>(<a id="xref-ref-53-6" class="xref-bibr" href="#ref-53">Ovaskainen et <em>al</em>., 2011</a>). Then, the additive genetic variance-covariance matrix of the contemporary populations assuming no selection, <strong>G</strong>, can be estimated as <strong>G</strong> = 2<strong>G</strong><em><sup>A</sup></em>(1 – <em>θ<sup>S</sup></em>), where <em>θ<sup>S</sup></em> is the mean within-population (or self) coancestry of all populations, thus the drift distance of the contemporary populations from the ancestral population. Here, we estimated the heritability of traits and the genetic correlation between trait pairs as the proportion of the observed phenotypic variance and covariances that are additive (i.e. using <strong>G</strong>) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Falconer &amp; Mackay, 1996</a>). The evidence for selection can be summarized using the S-statistic calculated as the Mahalanobis distance between <strong>A</strong><sup>P</sup> and the distribution of equation 1. <em>S</em> = 0.5 indicates consistency with neutrality, S = 0 implies a match with purifying, and S = 1 with diversifying selection. Thus, S measures the overall signature of selection across all populations. In this study, we also test if any particular populations show a signature of selection. We propose that adaptive trait divergence at population X may be concluded if the 95% of the posterior distribution of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/inline-graphic-3.gif"/></noscript></span></span> is outside of the neutral envelop defined as <em><strong>μ</strong></em> ± <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/04/19/292540/embed/inline-graphic-4.gif"/></noscript></span></span>. The neutral envelop can be uni- or multi-variate depending if one or multiple traits are studied. Our motivation for this population-wise evaluation of divergence is that a single <em>S</em>-statistic cannot distinguish between the two scenarios of many populations that are slightly diverged or a single/few populations that are diverged to a great extent.</p><p id="p-23">The <em>H</em>-statistics measure if the distance between the populations’ mean additive trait values is more similar to the environmental distances than expected based on drift (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Karhunen <em>et al</em>., 2014</a>). The original H-test is based on the Mantel test statistic, i.e. the product moment between the distance matrices of environment and traits. As the covariance is not only influenced by the correlation between the matrices but also by the absolute values of trait differences, the original H-test may yield false positive results. In particular, in cases with strong evidence for selection, the <em>H</em> statistic can be high (i.e. close to 1) even when selection is uncorrelated with the tested environmental driver. For this reason, we propose the use of the <em>H</em>* statistic, which is the Pearson or standardized Mantel statistic, thus the <em>H</em>*-test can be viewed as a standardized version of the H-test. We note that the superiority of a standardized Mantel statistic has already been pointed out in the context of spatial distance matrices (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Legendre &amp; Fortin, 2010</a>). The population-wise divergence test and the <em>H</em>*-test are now implemented in the most recent version of the R package driftsel<sup><a id="xref-fn-7-1" class="xref-fn" href="#fn-7">5</a></sup>.</p></div></div><div class="section" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Population structure and drift distances</h3><p id="p-24"><em>F<sub>ST</sub></em> using the <a id="xref-ref-72-4" class="xref-bibr" href="#ref-72">Whitlock &amp; Guillaume (2009)</a> approach was 0.0056 (95% confidence interval: 0.0051, 0.0061). The drift distances estimated with AFM indicated the presence of two main clusters that correspond to eastern and western populations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The population POS did not belong to either of these groups, which is plausible given that it is situated south of a high mountain pass (the Bernina pass) close to the Italian border (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). The coancestry matrix can also be used to obtain a coancestry-based estimate of <em>F<sub>ST</sub></em>, which was 0.0184 (95% credible interval: 0.0167, 0.0202) for the 19 populations. The comparison between AFM and Structure revealed similar results (see Supporting Information for more details).</p></div><div id="sec-10" class="subsection"><h3>Single-trait analysis</h3><p id="p-25">Single-trait tests of adaptive divergence in terms of <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> comparisons using the <a id="xref-ref-72-5" class="xref-bibr" href="#ref-72">Whitlock &amp; Guillaume (2009)</a> approach and in terms of the <em>S</em> statistic from the <a id="xref-ref-53-7" class="xref-bibr" href="#ref-53">Ovaskainen <em>et al</em>. (2011)</a> method gave similar results (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Using either of the methods, the strongest signature of adaptive divergence was observed for seedling Height followed by four other traits with a strong evidence of selection; in decreasing order of evidence based on the <em>S</em> statistic: Lateral Bud Break, Terminal Bud Break, Diameter, and Growth Duration. The remaining three traits showed no or a weak signal of unusual trait differentiation using either of the approaches. For Growth Cessation and Height/Diameter the <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> test was marginally significant, while the S statistic was close to 0.5, consist with neutrality. For Growth Rate, there was no evidence of selection using either approach.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/475958/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/475958/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/475958" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2:</span> <p id="p-26" class="first-child">Summary of single-trait tests of adaptive divergence. <em>Q<sub>ST</sub></em>, <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> and p-value correspond to the <a id="xref-ref-72-6" class="xref-bibr" href="#ref-72">Whitlock &amp; Guillaume (2009)</a> approach, <em>S</em> is the summary of evidence of selection using the <a id="xref-ref-53-8" class="xref-bibr" href="#ref-53">Ovaskainen et <em>al</em>. (2011)</a> method, <em>H</em> indicates the role of the environment in adaptation using the <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Karhunen et <em>al</em>. (2014)</a> test, and <em>H</em>* is a standardized version of <em>H</em> proposed in the study. Both <em>H</em> and <em>H</em>* were calculated with all environmental variables listed in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Tables 1</a> and S1. Values of <em>S</em>, <em>H</em>, and <em>H</em>* close to 1 indicate high evidence of divergent selection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">The population-wise <em>S</em>-tests revealed a signature of selection in several populations. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows, for each trait, how much each population diverged from the ancestral mean and if this divergence is more than expected by drift. These population-level estimates confirmed that Height diverged the most among populations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>): eight out of 19 populations deviated from the neutral expectation with at least 95% posterior probability. All these eight populations evolved towards a higher mean height and no populations have been selected for reduced height. Although the deviation from the neutral expectation was not as strong for Diameter as for Height, the first seven tallest populations also diverged towards the largest Diameter (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>), suggesting co-evolution between Height and Diameter, and explaining the lack of unusual differentiation at Height/Diameter (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). Evidence for trait divergence was also observed for a single population, SIR, for Growth Duration (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2E</a>) and Lateral Bud Break (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2H</a>). The SIR population evolved towards earlier bud break and longer growth duration than expected by drift alone. At the opposite end of the trait space, population VRG evolved towards late Terminal Bud Break and shorter Growth Duration, yet the evidence for divergence was not very strong in this single-trait test.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the single-trait test of adaptive divergence using the Ovaskainen et al. (2011) method for the eight studied traits shown across panels A to H. Each panel shows the estimated ancestral additive mean trait value (horizontal line), the amount of trait divergence from this mean that is expected based on drift (gray envelop), and the estimated posterior distribution of the additive trait values for each population (boxes). Stars above the boxes indicate the signature of selection at the particular population. Population order is different on each panel because populations are ordered according to their additive trait values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1564583975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of the single-trait test of adaptive divergence using the Ovaskainen et &lt;em&gt;al&lt;/em&gt;. (2011) method for the eight studied traits shown across panels A to H. Each panel shows the estimated ancestral additive mean trait value (horizontal line), the amount of trait divergence from this mean that is expected based on drift (gray envelop), and the estimated posterior distribution of the additive trait values for each population (boxes). Stars above the boxes indicate the signature of selection at the particular population. Population order is different on each panel because populations are ordered according to their additive trait values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F2.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F2.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-28" class="first-child">Summary of the single-trait test of adaptive divergence using the <a id="xref-ref-53-9" class="xref-bibr" href="#ref-53">Ovaskainen et <em>al</em>. (2011)</a> method for the eight studied traits shown across panels A to H. Each panel shows the estimated ancestral additive mean trait value (horizontal line), the amount of trait divergence from this mean that is expected based on drift (gray envelop), and the estimated posterior distribution of the additive trait values for each population (boxes). Stars above the boxes indicate the signature of selection at the particular population. Population order is different on each panel because populations are ordered according to their additive trait values.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Multi-trait analysis</h3><p id="p-29">We performed only two-trait analyses due to convergence problems with three or more traits (results not shown). We considered only one of the bud break traits in this pairwise analysis, Terminal Bud Break, because the two traits were very strongly correlated. The heritability values for a given trait were slightly different depending on the particular pairwise analysis because <strong>G<sup>A</sup></strong> is different, and also <em>θ</em><sup><strong>P</strong></sup> is updated in the Bayesian animal model (Table S2). For most trait pairs the genetic correlations were small or negligible (Table S2) and agreed well with phenotypic correlations (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). The highest phenotypic correlation (0.64) was observed between two size traits, Height and Diameter, yet their genetic correlation was half as much (0.32, Table S1). Ten populations showed evidence for adaptive co-divergence in Height and Diameter (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). In contrast, some unusually high genetic correlations (i.e. higher than the phenotypic equivalents) were observed between traits of different kind: between growth and phenology (Growth Duration and Terminal Bud Break, Growth Rate and Terminal Bud Break), and between size and phenology (Height/Diameter and Growth Cessation; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). The S-statistics was high only for the first two trait pairs (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>), suggesting a correlated evolution at the three traits (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). In particular, two populations reached an important level of divergence in the opposite directions. These patters can be interpreted as contrasting life history strategies. SIR showed adaptive divergence for all three trait combinations, while MGY for Growth Duration vs Growth Rate (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Thus, we can say that SIR and MGY follow a “start early and grow slow” strategy, i.e. they burst buds early and then grow for a long time at a low rate. At the other end of trait space, population VRG follows a or “start late and grow fast” strategy, i.e. bursts buds late, but then grows fast for a short period of time (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the two-trait analysis of adaptive divergence using the Ovaskainen et al. (2011) method. Panel A shows the phenotypic (rp) and genetic (rg) correlations between trait pairs (see main text for formula). The top three trait pairs with rp &#x2265; rg are highlighted in gray, and those with rg &#x2265; rp in black. Panel B shows the S-statistic for the two-trait selection test as a function of rg. The three trait pairs with rg &gt; rp highlighted in panel A are also highlighted here." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1564583975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of the two-trait analysis of adaptive divergence using the Ovaskainen et &lt;em&gt;al&lt;/em&gt;. (2011) method. Panel A shows the phenotypic (&lt;em&gt;r&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt;) and genetic (&lt;em&gt;r&lt;sub&gt;g&lt;/sub&gt;&lt;/em&gt;) correlations between trait pairs (see main text for formula). The top three trait pairs with &lt;em&gt;r&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt; &#x2265; &lt;em&gt;r&lt;sub&gt;g&lt;/sub&gt;&lt;/em&gt; are highlighted in gray, and those with &lt;em&gt;r&lt;sub&gt;g&lt;/sub&gt;&lt;/em&gt; &#x2265; &lt;em&gt;r&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt; in black. Panel B shows the S-statistic for the two-trait selection test as a function of &lt;em&gt;r&lt;sub&gt;g&lt;/sub&gt;&lt;/em&gt;. The three trait pairs with &lt;em&gt;r&lt;sub&gt;g&lt;/sub&gt;&lt;/em&gt; &gt; &lt;em&gt;r&lt;sub&gt;p&lt;/sub&gt;&lt;/em&gt; highlighted in panel A are also highlighted here.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F3.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F3.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-30" class="first-child">Summary of the two-trait analysis of adaptive divergence using the <a id="xref-ref-53-10" class="xref-bibr" href="#ref-53">Ovaskainen et <em>al</em>. (2011)</a> method. Panel A shows the phenotypic (<em>r<sub>p</sub></em>) and genetic (<em>r<sub>g</sub></em>) correlations between trait pairs (see main text for formula). The top three trait pairs with <em>r<sub>p</sub></em> ≥ <em>r<sub>g</sub></em> are highlighted in gray, and those with <em>r<sub>g</sub></em> ≥ <em>r<sub>p</sub></em> in black. Panel B shows the S-statistic for the two-trait selection test as a function of <em>r<sub>g</sub></em>. The three trait pairs with <em>r<sub>g</sub></em> &gt; <em>r<sub>p</sub></em> highlighted in panel A are also highlighted here.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlated adaptive divergence in Height and Diameter using the Ovaskainen et al. (2011) method. A is the estimated ancestral additive mean trait value. Ellipses represent the median amount of trait divergence that is expected based on drift for each population. Population codes (3 letters) represent the median of the posterior distribution of the additive trait values for each population. Ellipses and population codes with evidence of trait divergence are shown in black, while populations that do not deviate from what is expected based on drift are in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1564583975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlated adaptive divergence in Height and Diameter using the Ovaskainen et &lt;em&gt;al&lt;/em&gt;. (2011) method. &lt;em&gt;A&lt;/em&gt; is the estimated ancestral additive mean trait value. Ellipses represent the median amount of trait divergence that is expected based on drift for each population. Population codes (3 letters) represent the median of the posterior distribution of the additive trait values for each population. Ellipses and population codes with evidence of trait divergence are shown in black, while populations that do not deviate from what is expected based on drift are in gray.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F4.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F4.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-31" class="first-child">Correlated adaptive divergence in Height and Diameter using the <a id="xref-ref-53-11" class="xref-bibr" href="#ref-53">Ovaskainen et <em>al</em>. (2011)</a> method. <em>A</em> is the estimated ancestral additive mean trait value. Ellipses represent the median amount of trait divergence that is expected based on drift for each population. Population codes (3 letters) represent the median of the posterior distribution of the additive trait values for each population. Ellipses and population codes with evidence of trait divergence are shown in black, while populations that do not deviate from what is expected based on drift are in gray.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlated adaptive divergence at Terminal Bud Break, Growth Duration and Growth Rate using the Ovaskainen et al. (2011) method. Each panel shows the evidence for unusual trait divergence in a two-trait space. A is the estimated ancestral additive mean trait value. Ellipses represent the median amount of trait divergence that is expected based on drift for each population. Population codes (3 letters) represent the median of the posterior distribution of the additive trait values for each population. Ellipses and population codes with evidence of trait divergence are shown in black, while populations that do not deviate from what is expected based on drift are in gray." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1564583975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlated adaptive divergence at Terminal Bud Break, Growth Duration and Growth Rate using the Ovaskainen &lt;em&gt;et al&lt;/em&gt;. (2011) method. Each panel shows the evidence for unusual trait divergence in a two-trait space. A is the estimated ancestral additive mean trait value. Ellipses represent the median amount of trait divergence that is expected based on drift for each population. Population codes (3 letters) represent the median of the posterior distribution of the additive trait values for each population. Ellipses and population codes with evidence of trait divergence are shown in black, while populations that do not deviate from what is expected based on drift are in gray.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F5.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F5.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5.</span> <p id="p-32" class="first-child">Correlated adaptive divergence at Terminal Bud Break, Growth Duration and Growth Rate using the <a id="xref-ref-53-12" class="xref-bibr" href="#ref-53">Ovaskainen <em>et al</em>. (2011)</a> method. Each panel shows the evidence for unusual trait divergence in a two-trait space. A is the estimated ancestral additive mean trait value. Ellipses represent the median amount of trait divergence that is expected based on drift for each population. Population codes (3 letters) represent the median of the posterior distribution of the additive trait values for each population. Ellipses and population codes with evidence of trait divergence are shown in black, while populations that do not deviate from what is expected based on drift are in gray.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Environmental drivers</h3><p id="p-33"><em>H</em> and <em>H</em>* assess how much each environmental variable moves the Mantel statistic away from the neutral expectation (i.e. random association with the environment conditional on the drift distances). First, we tested the effect of the “global” environment, i.e. used all environmental variables listed in <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Tables 1</a> and S1 to calculate the environmental distance matrix. We repeated this analysis on the scaled and centered environmental variables, but they led to similar results, so only the analysis using the raw variables is reported. The “global” environment explained trait divergence for Diameter, Height and Growth Rate (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). For traits for which S was close to 0.5, such as Height/Diameter and Growth Cessation, there were no associations with the environmental variables either. Similarly, for Bud Break traits and Growth Duration, <em>H</em> was also slightly smaller than <em>S</em> and <em>H</em>* was close to 0.5 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). However, using the “global” environment, we assume that all aspects of the environment are important for adaptation. Further such global environmental distance matrix is dominated by the effects of temperature, which explains the most variation in the environmental data (Figure S2). In reality, it is likely that for each particular trait a particular aspect of the environment matters more than other aspects, so we applied the <em>H</em>- and <em>H</em>* -tests for each environmental variable listed in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Tables 1</a> and S1 separately. Since our objective was to prioritize the environmental variables, we did not correct for multiple testing. The <em>H</em> - and <em>H</em>*-statistics generally led to similar conclusions concerning the role of each environmental variable, but the <em>H</em>-statistic was inflated under a generally strong signal of selection, such as for Height (Figure S3).</p><p id="p-34">The role of the environment in the divergence of Height and Diameter was most important among all traits (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Additionally, the environmental variables that had an <em>H</em>*-statistic higher than 0.95 for Height included all variables that fulfilled the same criterion for Diameter, suggesting that the same environmental cues matter for the evolution of both traits. Remarkably, all of these environmental variables were related to temperature and drought, but not precipitation per se. The most important environmental variables in the order of decreasing <em>H</em>*-statistic for Height were Thornthwaite’s mean annual evapotranspiration (PET.thorn), mean temperature of the warmest quarter (bio.10), isothermality (bio.3), annual mean temperature (bio.1), available water capacity (AWC), elevation, mean temperature of the coldest quarter (bio.11), maximum temperature of the warmest month (bio.5), minimum temperature of the coldest month (bio.6), mean diurnal range (bio.2), latitude and drought severity measured as the 1% quantile of the self-calibrated Palmer’s Drought Severity Index across the whole time series. To support the interpretation of these results, we performed a PCA of these environmental variables. We found that all populations that diverged towards a larger size in terms of Height and Diameter were situated at the warm, stable and humid side of the environmental space (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). Finally, note that the precipitation-related variables (bio.12, bio.13, bio.14, bio.15, bio.18) had the lowest correlation with adaptive divergence for tree size (<em>H</em>*-statistics were below 0.5).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A: The first two principal components of the environmental variables that had a H* -statistics higher than 0.95 for Height and Diameter. Populations that showed a signature of adaptive divergence for Height in the population-wise S-test are situated in the &#x201C;warm&#x201D; and &#x201C;stable&#x201D; left side of the environmental space and are indicated by larger letter sizes for their population codes (i.e. VWD, BEI, COR, LUT, SIG, GRB, MGY, MUO). B: The first two principal components of the environmental variables that had and H*-statistics higher than 0.85 for Growth Rate and Duration, and Terminal and Lateral Bud Break. Populations that showed a signature of adaptive divergence for these traits in the single- or multi-trait population-wise S-test are indicated by larger letter sizes for their population codes (i.e. SIR, MGY, VRG). Explications of the environmental axes (in gray) can be found in Table 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1564583975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A: The first two principal components of the environmental variables that had a &lt;em&gt;H&lt;/em&gt;* -statistics higher than 0.95 for Height and Diameter. Populations that showed a signature of adaptive divergence for Height in the population-wise &lt;em&gt;S&lt;/em&gt;-test are situated in the &#x201C;warm&#x201D; and &#x201C;stable&#x201D; left side of the environmental space and are indicated by larger letter sizes for their population codes (i.e. VWD, BEI, COR, LUT, SIG, GRB, MGY, MUO). B: The first two principal components of the environmental variables that had and &lt;em&gt;H&lt;/em&gt;*-statistics higher than 0.85 for Growth Rate and Duration, and Terminal and Lateral Bud Break. Populations that showed a signature of adaptive divergence for these traits in the single- or multi-trait population-wise &lt;em&gt;S&lt;/em&gt;-test are indicated by larger letter sizes for their population codes (i.e. SIR, MGY, VRG). Explications of the environmental axes (in gray) can be found in Table 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F6.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F6.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/04/19/292540/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6.</span> <p id="p-35" class="first-child">A: The first two principal components of the environmental variables that had a <em>H</em>* -statistics higher than 0.95 for Height and Diameter. Populations that showed a signature of adaptive divergence for Height in the population-wise <em>S</em>-test are situated in the “warm” and “stable” left side of the environmental space and are indicated by larger letter sizes for their population codes (i.e. VWD, BEI, COR, LUT, SIG, GRB, MGY, MUO). B: The first two principal components of the environmental variables that had and <em>H</em>*-statistics higher than 0.85 for Growth Rate and Duration, and Terminal and Lateral Bud Break. Populations that showed a signature of adaptive divergence for these traits in the single- or multi-trait population-wise <em>S</em>-test are indicated by larger letter sizes for their population codes (i.e. SIR, MGY, VRG). Explications of the environmental axes (in gray) can be found in <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Divergence at the two Growth traits and the two Bud Break traits also correlated with habitat differences: The most important environmental variables (<em>H</em>*≥0.85) largely overlapped for these traits and were most often related to precipitation (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). Mean temperature of the wettest quarter (bio.8), precipitation seasonality (bio.15), precipitation of warmest quarter (bio.18), and mean diurnal range (bio.2) were omnipresent as environmental variables with the highest <em>H</em>*-statistics for the four traits. Again, the common habitat variables that explain adaptation for these traits is not surprising given the evidence for their correlated evolution (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). The PCA of these environmental variables illustrates that the drivers of the “start early and grow slow” strategy of SIR, and to some extent that of MGY and GRY, are low average precipitation, low of precipitation seasonality, and high temperature amplitude both at the annual and monthly levels (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>), reflecting the cold winters and hot and dry summers of the Rhône Valley. At the other end of the trait space, the climate of population VRG is characterized by ample winter precipitation, late snow and late frost, and low soil silt content, which could explain the late bud break and short but fast growth of this population (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>).</p></div></div><div class="section" id="sec-13"><h2 class="">Discussion</h2><p id="p-37">In this study, we found clear evidence of local adaptation at size, growth and phenology traits of seedlings originating from 19 silver fir populations from across Switzerland. Thus, our study relies on the common, and often supported (<em>e.g</em>. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cornelissen et <em>al</em>., 2003</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Major et <em>al</em>., 2003</a>), assumption that traits measured at seedlings predicts those at adult trees. Tree size (Height and Diameter) has been under strong selection in warm and humid areas of Switzerland (Central Plateau), where populations evolved towards a taller and larger stature (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-6-5" class="xref-fig" href="#F6">6</a>). The role of selection on Height is further supported by the fact that the selection gradient is South-North (northern populations being taller), while the main demographic axis is East-West (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). The annual development cycle of temperate trees likely results from an evolutionary trade-off between two opposing forces: maximizing the length of the vegetative season while avoiding late frost and summer drought. This life-history trade-off is particularly crucial in mountainous environments, where the length of the growing season is often limited. In this study, the timing, the rate, and the duration of growth have been under selection that allowed populations to evolve towards two extreme strategies that reflect adaptation to the amount and timing of precipitation (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-6" class="xref-fig" href="#F6">6</a>). Populations of the Rhône Valley, especially SIR, but also MGY and GRY (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>), seem to persevere the conditions of this notoriously dry inner Alpine valley of Switzerland. Here, annual precipitation is low and falls mainly in winter, while high temperatures and drought are common during spring and summer. These populations are adapted to such climatic conditions through early bud break and slow growth during the dry but long vegetative season (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>). We observed the opposite strategy in VRG, and to some extent in MUO. Both populations are situated in high-altitude mountain valleys (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>), where late snow delays the start of the vegetative season, but then growth is fastest in these populations (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>).</p><p id="p-38">Quantitative genetic theory suggests that fitness-related traits are under strong selective pressure with stabilizing selection reducing genetic variation among individuals (e.g. <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Mousseau &amp; Roff, 1987</a>). As a result, it is often observed that heritability estimates are lower on morphological than on life-history traits (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Visscher et <em>al</em>., 2008</a>). In contrast, estimates of heritability from <a id="xref-ref-25-9" class="xref-bibr" href="#ref-25">Frank et <em>al</em>. (2017b)</a> using 90 Swiss populations of silver fir revealed the highest heritability for phenology and growth, and lowest for size. Further, our estimates of the heritability derived from the ancestral G-matrix also revealed the highest values for the two Bud Break traits, while the lowest for Height (Table S2). We may speculate that height has been under strong selection, thus its additive genetic variance has been reduced. The fitness advantage of being tall for trees are clear: Taller trees have access to more light, thus can have enhanced growth, and high stature facilitates the pollen and grain dispersion (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Petit &amp; Hampe, 2006</a>). Further, artificial selection for height may also have affected these populations, further reducing the additive genetic variance. The populations that have been selected for being taller in this study (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>) are located in the geographic region that are the most easily accessible for foresters due to their low elevation and often flat terrain. Since tree height is a key trait from an economical point of view, and there has been a long history of management in this region (<em>e.g</em>. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bürgi &amp; Schuler, 2003</a>), we cannot exclude the possibility that there has been artificial selection in these populations.</p><p id="p-39">Interestingly, two traits, Height/Diameter and Growth Cessation, did not show evidence of adaptive divergence, but they were genetically correlated (0.25), while lacking any phenotypic correlation (0.04, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, Table S1). The allometric relationship between Height and Diameter has been extensively studied in forest trees because it determines the volume of the harvestable stem and carbon storage, and it is also associated with wood quality such as lignin and cellulose content (e.g. <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chave et <em>al</em>., 2014</a>). The fact that Growth Cessation did not show a signature of selection may seem surprising given the overwhelming evidence of adaptive clines for bud set (a proxy for growth cessation) in many species, including conifers (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Alberto et <em>al</em>., 2013</a>). The explanation may lie in the deterministic bud development of the <em>Abies</em> species (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cooke et <em>al</em>., 2012</a>). They produce terminal buds during the summer at the tip of each leading branch shoot and remain dormant during the following winter. Each bud contains a preformed miniature of the terminal and lateral shoots with all the needles that will grow to branches and photosynthesizing needles, respectively, during the following growing season. Thus, given that the timing of the start (bud burst) and the rate (maximum growth rate and growth duration) of this developmental trait have been under selection (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>), the timing of the end cannot be under selection. The positive genetic and negligible phenotypic correlation between Height/Diameter and Growth Cessation may indicate that the trees can only increase their height in relation to their diameter at the price of growing longer; however, the evolution of this developmental process is extremely constrained due to canalization.</p><p id="p-40">A long-standing hypothesis in evolutionary biology is that traits belonging to the same functional and/or developmental group are genetically more integrated than traits with different functions or developmental origins (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Berg, 1960</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Pigliucci &amp; Preston, 2004</a>). Several empirical studies found evidence that there is greater genetic and phenotypic character integration within suites of functionally or developmentally related traits than between them, e.g. between and within floral vs. vegetative traits in plants (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Waitt &amp; Levin, 1998</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baranzelli <em>et al</em>., 2014</a>). In this study, we measured exclusively vegetative traits, but three kinds, related to size, growth, and phenology. Indeed, we found that the genetic correlations tended to be higher than phenotypic correlations between different trait kinds than within (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3A</a>), and in particular, between growth and phenology. These results suggest that at the physiological and molecular level, growth and phenology are strongly linked.</p><p id="p-41">Overall, our methodology proved more informative than classical <em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> comparisons, even though the high number of populations leveraged the power of the classical test and led to similar global conclusions (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). First, we were able to pinpoint which populations diverged from a supposed ancestral trait value (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). Second, the multi-trait analysis allowed us to identify life-history strategies that are under selection, and third, we were able to identify the selective environment. In particular, we found that the same environmental variables explained best the single-trait adaptive population differentiation at traits that were genetically correlated in the multi-trait analysis, which provides particularly strong evidence for the role of temperature and humidity in the evolution of tree size (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6A</a>) and the role of precipitation seasonality in the timing and mode of growth (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6B</a>).</p><p id="p-42">Recent evidence suggests a global increase in climate-driven forest decline that is primarily driven by a combination of rising temperatures and drought stress, so-called hot-drought, e.g. <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Allen et <em>al</em>. (2010)</a>. Such conditions foster detrimental agents such as pests (pathogens, herbivores) or physical disturbances such as storms (e.g. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Csilléry et <em>al</em>., 2017</a>). For Europe, the most recent IPCC report predicts an increase in temperature, along with a decrease in summer precipitation and an increase in winter precipitation (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Stocker et <em>al</em>., 2014</a>). Although there is abundant evidence of the past evolution of plants in response to climate, such as past range shifts during glaciations (e.g. <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Petit et <em>al</em>., 2003</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aitken et <em>al</em>., 2008</a>), the question is the extent to which adaptive evolution can compensate the effects of rapid climate change. Silver fir may already be in danger in some areas of the Mediterranean, where die-back events have been documented (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cailleret et <em>al</em>., 2014</a>), or in south-western Europe, where reduced growth patterns have been reported (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gazol et <em>al</em>., 2015</a>). Yet, positive effects of climate warming have also been observed in temperate areas, where warming enhanced growth (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Gazol et <em>al</em>., 2015</a>). The predicted pace of climate change is much faster than it has been during the time since post-glacial expansion/re-colonization, thus assisted migration can provide a practical solution (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aitken &amp; Bemmels, 2016</a>). Based on our results, populations of the Rhône Valley may provide dry tolerant seed sources for future plantations in other parts of Switzerland.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-conflict" id="fn-1"><p id="p-1"><strong>Conflict of Interest Statement:</strong> The authors declare no conflicts of interest.</p></li><li class="fn" id="fn-2"><p id="p-2"><strong>Data archival:</strong> SNP data will be archived on Dryad.</p></li><li class="fn" id="fn-3"><p id="p-43"><a class="rev-xref" href="#xref-fn-3-1">↵</a><span class="fn-label">1</span> <a href="https://github.com/molpopgen/analysis">https://github.com/molpopgen/analysis</a></p></li><li class="fn" id="fn-4"><p id="p-44"><a class="rev-xref" href="#xref-fn-4-1">↵</a><span class="fn-label">2</span> <a href="https://github.com/kjgilbert/QstFstComp">https://github.com/kjgilbert/QstFstComp</a></p></li><li class="fn" id="fn-5"><p id="p-45"><a class="rev-xref" href="#xref-fn-5-1">↵</a><span class="fn-label">3</span> <a href="https://www.helsinki.fi/en/researchgroups/metapopulation-research-centre/rafm-and-driftsel">https://www.helsinki.fi/en/researchgroups/metapopulation-research-centre/rafm-and-driftsel</a></p></li><li class="fn" id="fn-6"><p id="p-46"><a class="rev-xref" href="#xref-fn-6-1">↵</a><span class="fn-label">4</span> <a href="https://www.helsinki.fi/en/researchgroups/metapopulation-research-centre/rafm-and-driftsel">https://www.helsinki.fi/en/researchgroups/metapopulation-research-centre/rafm-and-driftsel</a></p></li><li class="fn" id="fn-7"><p id="p-47"><a class="rev-xref" href="#xref-fn-7-1">↵</a><span class="fn-label">5</span> <a href="https://github.com/kcsillery/driftsel">https://github.com/kcsillery/driftsel</a></p></li></ul></div><div class="bio" id="bio-1"><h3>Author contributions</h3><p id="p-5">KC designed the study and obtained the SNP data with help from AW and FG. KC performed all analysis with advice from OO. CS was involved in the published common garden experiment and provided help with data interpretation. KC wrote the manuscript with help from all other authors.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.1" data-doi="10.1111/eva.12293"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bemmels</span>  <span class="cit-name-given-names">JB</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Time to get moving: assisted gene flow of forest trees</span>. <abbr class="cit-jnl-abbrev">Evolutionary applications</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">271</span>–<span class="cit-lpage">290</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/eva.12293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2Bapplications%26rft.volume%253D9%26rft.spage%253D271%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.2" data-doi="10.1111/j.1752-4571.2007.00013.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">JA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Curtis-McLane</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Adaptation, migration or extirpation: climate change outcomes for tree populations</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>, <span class="cit-vol">1</span>, <span class="cit-fpage">95</span>–<span class="cit-lpage">111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1752-4571.2007.00013.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D1%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2007.00013.x%26rft_id%253Dinfo%253Apmid%252F25567494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2007.00013.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25567494&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.3" data-doi="10.1111/gcb.12181"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Alberto</span>  <span class="cit-name-given-names">FJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">SN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alía</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Potential for evolutionary responses to climate change - evidence from tree populations</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">1645</span>–<span class="cit-lpage">1661</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10. 1111/gcb.12181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D19%26rft.spage%253D1645%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.12181%26rft_id%253Dinfo%253Apmid%252F23505261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.12181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23505261&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.4" data-doi="10.1016/j.foreco.2009.09.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">CD</span></span>, <span class="cit-auth"><span class="cit-name-surname">Macalady</span>  <span class="cit-name-given-names">AK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chenchouni</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">A global overview of drought and heat-induced tree mortality reveals emerging climate change risks for forests</span>. <abbr class="cit-jnl-abbrev">Forest Ecology and Management</abbr>, <span class="cit-vol">259</span>, <span class="cit-fpage">660</span>–<span class="cit-lpage">684</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.foreco.2009.09.001</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2BEcology%2Band%2BManagement%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foreco.2009.09.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foreco.2009.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.5" data-doi="10.1111/jeb.12341"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Baranzelli</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sersic</span>  <span class="cit-name-given-names">AN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cocucci</span>  <span class="cit-name-given-names">AA</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">The search for pleiades in trait constellations: functional integration and phenotypic selection in the complex flowers of <em>Morrenia brachystephana</em> (apocynaceae)</span>. <abbr class="cit-jnl-abbrev">Journal of Evolutionary Biology</abbr>, <span class="cit-vol">27</span>, <span class="cit-fpage">724</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BEvolutionary%2BBiology%26rft.volume%253D27%26rft.spage%253D724%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12341%26rft_id%253Dinfo%253Apmid%252F24528334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24528334&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beguería</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vicente-Serrano</span>  <span class="cit-name-given-names">SM</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">SPEI: calculation of the standardised precipitation-evapotranspiration index</span>. <abbr class="cit-jnl-abbrev">R package version 1.7</abbr>, <span class="cit-vol">1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.7" data-doi="10.2307/2405824"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">R</span></span> (<span class="cit-pub-date">1960</span>) <span class="cit-article-title">The ecological significance of correlation pleiades</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">171</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D14%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2405824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2405824&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1960XE16900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Booth</span>  <span class="cit-name-given-names">TH</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nix</span>  <span class="cit-name-given-names">HA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Busby</span>  <span class="cit-name-given-names">JR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hutchinson</span>  <span class="cit-name-given-names">MF</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Bioclim: the first species distribution modelling package, its early applications and relevance to most current maxent studies</span>. <abbr class="cit-jnl-abbrev">Diversity and Distributions</abbr>, <span class="cit-vol">20</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiversity%2Band%2BDistributions%26rft.volume%253D20%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.9" data-doi="10.1007/PL00006927"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Burga</span>  <span class="cit-name-given-names">CA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hussendörfer</span>  <span class="cit-name-given-names">E</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Vegetation history of <em>Abies alba</em> Mill. (silver fir) in Switzerland-pollen analytical and genetic surveys related to aspects of vegetation history of Picea abies (L.) H. Karsten (Norway spruce)</span>. <abbr class="cit-jnl-abbrev">Vegetation History and Archaeobotany</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">151</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVegetation%2BHistory%2Band%2BArchaeobotany%26rft.volume%253D10%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006927&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000172555900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.10" data-doi="10.1016/S0378-1127(02)00270-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bürgi</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schuler</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Driving forces of forest management - an analysis of regeneration practices in the forests of the Swiss Central Plateau during the 19th and 20th century</span>. <abbr class="cit-jnl-abbrev">Forest Ecology and Management</abbr>, <span class="cit-vol">176</span>, <span class="cit-fpage">173</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2BEcology%2Band%2BManagement%26rft.volume%253D176%26rft.spage%253D173%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1127%252802%252900270-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1127(02)00270-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.11" data-doi="10.1007/s13595-013-0265-0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cailleret</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nourtier</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Amm</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Durand-Gillmann</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davi</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Drought-induced decline and mortality of silver fir differ among three sites in Southern France</span>. <abbr class="cit-jnl-abbrev">Annals of Forest Science</abbr>, <span class="cit-vol">71</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s13595-013-0265-0</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BForest%2BScience%26rft.volume%253D71%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13595-013-0265-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13595-013-0265-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chave</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Réjou-Méchain</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Burquez</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Improved allometric models to estimate the aboveground biomass of tropical trees</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr>, <span class="cit-vol">20</span>, <span class="cit-fpage">3177</span>–<span class="cit-lpage">3190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D20%26rft.spage%253D3177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.13" data-doi="10.1111/j.1558-5646.2008.00374.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chenoweth</span>  <span class="cit-name-given-names">SF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blows</span>  <span class="cit-name-given-names">MW</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title"><span class="sc">Q<sub>st</sub></span> meets the G matrix: The dimensionality of adaptive divergence in multiple correlated quantitative traits</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">62</span>, <span class="cit-fpage">1437</span>–<span class="cit-lpage">1449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1558-5646.2008.00374.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DChenoweth%26rft.auinit1%253DS.%2BF.%26rft.volume%253D62%26rft.issue%253D6%26rft.spage%253D1437%26rft.epage%253D1449%26rft.atitle%253DQ%2528St%2529%2Bmeets%2Bthe%2BG%2Bmatrix%253A%2Bthe%2Bdimensionality%2Bof%2Badaptive%2Bdivergence%2Bin%2Bmultiple%2Bcorrelated%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00374.x%26rft_id%253Dinfo%253Apmid%252F18346219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00374.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18346219&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256356300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.14" data-doi="10.1111/j.1365-3040.2012.02552.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">ME</span></span>, <span class="cit-auth"><span class="cit-name-surname">Junttila</span>  <span class="cit-name-given-names">O</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">The dynamic nature of bud dormancy in trees: environmental control and molecular mechanisms</span>. <abbr class="cit-jnl-abbrev">Plant, Cell &amp; Environment</abbr>, <span class="cit-vol">35</span>, <span class="cit-fpage">1707</span>–<span class="cit-lpage">1728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.stitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.aulast%253DCooke%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D35%26rft.issue%253D10%26rft.spage%253D1707%26rft.epage%253D1728%26rft.atitle%253DThe%2Bdynamic%2Bnature%2Bof%2Bbud%2Bdormancy%2Bin%2Btrees%253A%2Benvironmental%2Bcontrol%2Band%2Bmolecular%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3040.2012.02552.x%26rft_id%253Dinfo%253Apmid%252F22670814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3040.2012.02552.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22670814&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308395600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.15" data-doi="10.1111/j.1654-1103.2003.tb02157.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cornelissen</span>  <span class="cit-name-given-names">JHC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cerabolini</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Castro-Diez</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Functional traits of woody plants: correspondence of species rankings between field adults and laboratory-grown seedlings?</span> <abbr class="cit-jnl-abbrev">Journal of Vegetation Science</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">311</span>–<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVegetation%2BScience%26rft.volume%253D14%26rft.spage%253D311%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1654-1103.2003.tb02157.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1654-1103.2003.tb02157.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000184495400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.16" data-doi="10.1111/gcb.13773"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Csilléry</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kunstler</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Courbaud</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Allard</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lassègues</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haslinger</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gardiner</span>  <span class="cit-name-given-names">B</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Coupled effects of wind-storms and drought on tree mortality across 115 forest stands from the Western Alps and the Jura mountains</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr>, <span class="cit-vol">23</span>, <span class="cit-fpage">5092</span>–<span class="cit-lpage">5107</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/gcb.13773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.13773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.13773&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.17" data-doi="10.1111/jeb.12777"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">De Kort</span>  <span class="cit-name-given-names">H</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vander Mijnsbrugge</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vandepitte</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mergeay</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ovaskainen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Honnay</span>  <span class="cit-name-given-names">O</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Evolution, plasticity and evolving plasticity of phenology in the tree species</span> <abbr class="cit-jnl-abbrev">Alnus glutinosa. Journal of Evolutionary Biology</abbr>, <span class="cit-vol">29</span>, <span class="cit-fpage">253</span>–<span class="cit-lpage">264</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/jeb.12777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAlnus%2Bglutinosa.%2BJournal%2Bof%2BEvolutionary%2BBiology%26rft.volume%253D29%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-292540v2.18" data-doi="10.1007/s12686-011-9548-7"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Earl</span>  <span class="cit-name-given-names">DA</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method</span>. <abbr class="cit-jnl-abbrev">Conservation Genetics Resources</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">359</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BGenetics%2BResources%26rft.volume%253D4%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12686-011-9548-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12686-011-9548-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000303536400036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-292540v2.19"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellenberg</span>  <span class="cit-name-given-names">H</span></span> (<span class="cit-pub-date">1988</span>) <span class="cit-source">Vegetation ecology of central Europe</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-292540v2.20" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Detecting the number of clusters of individuals using the software structure: a simulation study</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-292540v2.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">DS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">TFC</span></span> (<span class="cit-pub-date">1996</span>) <span class="cit-source">Introduction to Quantitative Genetics</span>. <span class="cit-publ-name">Longmans Green, Harlow, Essex</span>, <span class="cit-publ-loc">UK</span>, <span class="cit-edition">4 edn</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-292540v2.22" data-doi="PubMed PMID: 88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">164</span>, <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Adoi%252FPubMed%2BPMID%253A%2B88%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8xOS8yOTI1NDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Finkeldey</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mátyás</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperisen</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bonfils</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Strategien zur Auswahl forstlicher Genreservate in der Schweiz</span>. <abbr class="cit-jnl-abbrev">Forest Snow and Landscape Research</abbr>, <span class="cit-vol">75</span>, <span class="cit-fpage">137</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForest%2BSnow%2Band%2BLandscape%2BResearch%26rft.volume%253D75%26rft.spage%253D137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.24" data-doi="10.1111/gcb.13802"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">GT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperisen</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brang</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clair</span>  <span class="cit-name-given-names">JBS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Schmatz</span>  <span class="cit-name-given-names">DR</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heiri</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2017a</span>) <span class="cit-article-title">Risk of genetic maladaptation due to climate change in three major European tree species</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/gcb.13802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.13802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.13802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.25" data-doi="10.1002/ecy.1632"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Frank</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sperisen</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">GT</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brang</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walthert</span>  <span class="cit-name-given-names">L</span></span>, <span class="cit-auth"><span class="cit-name-surname">Clair</span>  <span class="cit-name-given-names">JBS</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heiri</span>  <span class="cit-name-given-names">C</span></span> (<span class="cit-pub-date">2017b</span>) <span class="cit-article-title">Distinct genecological patterns in seedlings of Norway spruce and silver fir from a mountainous landscape</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr>, <span class="cit-vol">98</span>, <span class="cit-fpage">211</span>–<span class="cit-lpage">227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1002/ecy.1632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D98%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fecy.1632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ecy.1632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.26" data-doi="10.1111/eva.12112"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Franks</span>  <span class="cit-name-given-names">SJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">SN</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Evolutionary and plastic responses to climate change in terrestrial plant populations</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">123</span>–<span class="cit-lpage">139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/eva.12112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D7%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12112%26rft_id%253Dinfo%253Apmid%252F24454552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24454552&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-292540v2.27" data-doi="10.1093/bioinformatics/btv428"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Galili</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">dendextend: an r package for visualizing, adjusting, and comparing trees of hierarchical clustering</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btv428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv428%26rft_id%253Dinfo%253Apmid%252F26209431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26209431&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.28"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gazol</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Camarero</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Distinct effects of climate warming on populations of silver fir (<em>Abies alba</em>) across Europe</span>. <abbr class="cit-jnl-abbrev">Journal of Biogeography</abbr>, <span class="cit-vol">42</span>, <span class="cit-fpage">1150</span>–<span class="cit-lpage">1162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiogeography%26rft.volume%253D42%26rft.spage%253D1150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">KJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title"><em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> comparisons with unbalanced half-sib designs</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr>, <span class="cit-vol">15</span>, <span class="cit-fpage">262</span>–<span class="cit-lpage">267</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D15%26rft.spage%253D262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.30"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hampe</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jump</span>  <span class="cit-name-given-names">AS</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Climate relicts: past, present, future</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">42</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.31" data-doi="10.1002/joc.3711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">D</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Updated high-resolution grids of monthly climatic observations-the cruts3. 10 dataset</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr>, <span class="cit-vol">34</span>, <span class="cit-fpage">623</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D34%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.3711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.3711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000332092900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.32" data-doi="10.1111/j.1752-1688.2000.tb04276.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hay</span>  <span class="cit-name-given-names">LE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wilby</span>  <span class="cit-name-given-names">RL</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leavesley</span>  <span class="cit-name-given-names">GH</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">A comparison of delta change and downscaled GCM scenarios for three mountainous basins in the United States</span>. <abbr class="cit-jnl-abbrev">JAWRA Journal of the American Water Resources Association</abbr>, <span class="cit-vol">36</span>, <span class="cit-fpage">387</span>–<span class="cit-lpage">397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAWRA%2BJournal%2Bof%2Bthe%2BAmerican%2BWater%2BResources%2BAssociation%26rft.volume%253D36%26rft.spage%253D387%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-1688.2000.tb04276.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-1688.2000.tb04276.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.33" data-doi="10.1371/journal.pone.0169748"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hengl</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Jesus</span>  <span class="cit-name-given-names">JM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heuvelink</span>  <span class="cit-name-given-names">GB</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">SoilGrids250m: Global gridded soil information based on machine learning</span>. <abbr class="cit-jnl-abbrev">PLoS one</abbr>, <span class="cit-vol">12</span>, <span class="cit-fpage">e0169748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bone%26rft.volume%253D12%26rft.spage%253De0169748%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0169748%26rft_id%253Dinfo%253Apmid%252F28207752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0169748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28207752&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Leathwick</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Elith</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hijmans</span>  <span class="cit-name-given-names">MRJ</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Package ‘dismo’</span>. <abbr class="cit-jnl-abbrev">Circles</abbr>, <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.35" data-doi="10.1038/nature09670"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">AA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sgrò</span>  <span class="cit-name-given-names">CM</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Climate change and evolutionary adaptation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">470</span>, <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHoffmann%26rft.auinit1%253DA.%2BA.%26rft.volume%253D470%26rft.issue%253D7335%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DClimate%2Bchange%2Band%2Bevolutionary%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09670%26rft_id%253Dinfo%253Apmid%252F21350480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21350480&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287652900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-292540v2.36" data-doi="10.1111/j.1755-0998.2009.02591.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">MJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Inferring weak population structure with the assistance of sample group information</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">1322</span>–<span class="cit-lpage">1332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1755-0998.2009.02591.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BEcol%2BResour%26rft.aulast%253DHubisz%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D1322%26rft.epage%253D1332%26rft.atitle%253DInferring%2Bweak%2Bpopulation%2Bstructure%2Bwith%2Bthe%2Bassistance%2Bof%2Bsample%2Bgroup%2Binformation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2009.02591.x%26rft_id%253Dinfo%253Apmid%252F21564903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2009.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21564903&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Iseli</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jongeneel</span>  <span class="cit-name-given-names">CV</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bucher</span>  <span class="cit-name-given-names">P</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">ESTScan: a program for detecting, evaluating, and reconstructing potential coding regions in EST sequences</span>. In: <abbr class="cit-jnl-abbrev">ISMB: International Conference on Intelligent Systems for Molecular Biology</abbr>, vol. <span class="cit-vol">99</span>, pp. <span class="cit-fpage">138</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISMB%253A%2BInternational%2BConference%2Bon%2BIntelligent%2BSystems%2Bfor%2BMolecular%2BBiology%26rft.volume%253D99%26rft.spage%253D138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-292540v2.38" data-doi="10.1093/bioinformatics/btm233"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">NA</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">CLUMPP: a cluster matching and permutation program for dealing with label switching and multimodality in analysis of population structure</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">23</span>, <span class="cit-fpage">1801</span>–<span class="cit-lpage">1806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm233%26rft_id%253Dinfo%253Apmid%252F17485429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17485429&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249248300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karger</span>  <span class="cit-name-given-names">DN</span></span>, <span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Böhner</span>  <span class="cit-name-given-names">J</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Climatologies at high resolution for the earth’s land surface areas</span>. <abbr class="cit-jnl-abbrev">Scientific Data</abbr>, <span class="cit-vol">4</span>, <span class="cit-fpage">170122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2BData%26rft.volume%253D4%26rft.spage%253D170122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.40" data-doi="10.1534/genetics.112.140871"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karhunen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ovaskainen</span>  <span class="cit-name-given-names">O</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Estimating population-level coancestry coefficients by an admixture F model</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">192</span>, <span class="cit-fpage">609</span>–<span class="cit-lpage">617</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.112.140871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140871%26rft_id%253Dinfo%253Apmid%252F22798486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNjA5IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMTkvMjkyNTQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.41" data-doi="10.1111/evo.12268"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Karhunen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ovaskainen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herczeg</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Bringing habitat information into statistical tests of local adaptation in quantitative traits: A case study of nine-spined sticklebacks</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr>, <span class="cit-vol">68</span>, <span class="cit-fpage">559</span>–<span class="cit-lpage">568</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/evo.12268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D68%26rft.spage%253D559%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.12268%26rft_id%253Dinfo%253Apmid%252F24117061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.12268&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24117061&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.42" data-doi="10.1111/j.1461-0248.2012.01746.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ronce</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Robledo-Arnuncio</span>  <span class="cit-name-given-names">JJ</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Long-distance gene flow and adaptation of forest trees to rapid climate change</span>. <abbr class="cit-jnl-abbrev">Ecology Letters</abbr>, <span class="cit-vol">15</span>, <span class="cit-fpage">378</span>–<span class="cit-lpage">392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/j.1461-0248.2012.01746.x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D15%26rft.spage%253D378%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2012.01746.x%26rft_id%253Dinfo%253Apmid%252F22372546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2012.01746.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22372546&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.43"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fortin</span>  <span class="cit-name-given-names">MJ</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Comparison of the mantel test and alternative approaches for detecting complex multivariate relationships in the spatial analysis of genetic data</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr>, <span class="cit-vol">10</span>, <span class="cit-fpage">831</span>–<span class="cit-lpage">844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D10%26rft.spage%253D831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.44" data-doi="10.1038/nrg3395"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Leinonen</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">McCairns</span>  <span class="cit-name-given-names">RS</span></span>, <span class="cit-auth"><span class="cit-name-surname">O’hara</span>  <span class="cit-name-given-names">RB</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title"><em>Q<sub>ST</sub></em> – <em>F<sub>ST</sub></em> comparisons: evolutionary and ecological insights from genomic heterogeneity</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrg3395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D14%26rft.spage%253D179%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3395%26rft_id%253Dinfo%253Apmid%252F23381120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23381120&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lenormand</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Gene flow and the limits to natural selection</span>. <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">183</span>–<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D17%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-confproc" id="cit-292540v2.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liepelt</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bialozyt</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziegenhagen</span>  <span class="cit-name-given-names">B</span></span> (<span class="cit-pub-date">2002</span>) <span class="cit-article-title">Wind-dispersed pollen mediates postglacial gene flow among refugia</span>. <span class="cit-conf-name">Proceedings of the National Academy of Sciences</span>, <span class="cit-vol">99</span>, <span class="cit-fpage">14590</span>–<span class="cit-lpage">14594</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.47" data-doi="10.1016/j.revpalbo.2008.07.007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liepelt</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheddadi</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Beaulieu</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Postglacial range expansion and its genetic imprints in <em>Abies alba</em> (Mill.) - A synthesis from palaeobotanic and genetic data</span>. <abbr class="cit-jnl-abbrev">Review of Palaeobotany and Palynology</abbr>, <span class="cit-vol">153</span>, <span class="cit-fpage">139</span>–<span class="cit-lpage">149</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1016/j.revpalbo.2008.07.007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReview%2Bof%2BPalaeobotany%2Band%2BPalynology%26rft.stitle%253DReview%2Bof%2BPalaeobotany%2Band%2BPalynology%26rft.volume%253D153%26rft.issue%253D1-2%26rft.spage%253D139%26rft.epage%253D149%26rft.atitle%253DPostglacial%2Brange%2Bexpansion%2Band%2Bits%2Bgenetic%2Bimprints%2Bin%2BAbies%2Balba%2B%2528Mill.%2529%253B%2Ba%2Bsynthesis%2Bfrom%2Bpalaeobotanic%2Band%2Bgenetic%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.revpalbo.2008.07.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.revpalbo.2008.07.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2012019131&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000261990000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.48" data-doi="10.1007/s11295-017-1224-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lind</span>  <span class="cit-name-given-names">BM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Menon</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bolte</span>  <span class="cit-name-given-names">CE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Faske</span>  <span class="cit-name-given-names">TM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">AJ</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">The genomics of local adaptation in trees: Are we out of the woods yet?</span> <abbr class="cit-jnl-abbrev">Tree Genetics &amp; Genomes</abbr>, <span class="cit-vol">14</span>, <span class="cit-fpage">29</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1007/s11295-017-1224-y</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenetics%2B%2526%2BGenomes%26rft.volume%253D14%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-017-1224-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-017-1224-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Major</span>  <span class="cit-name-given-names">JE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mosseler</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Barsi</span>  <span class="cit-name-given-names">DC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rajora</span>  <span class="cit-name-given-names">OP</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Morphometric, allometric, and developmentally adaptive traits in red spruce and black spruce. ii. seedling and mature tree assessment of controlled intra-and inter-specific hybrids</span>. <abbr class="cit-jnl-abbrev">Canadian journal of forest research</abbr>, <span class="cit-vol">33</span>, <span class="cit-fpage">897</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCanadian%2Bjournal%2Bof%2Bforest%2Bresearch%26rft.volume%253D33%26rft.spage%253D897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.50" data-doi="10.1534/genetics.107.080820"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">G</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chapuis</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Multivariate <em>Q<sub>ST</sub>-F<sub>st</sub></em> comparisons: A neutrality test for the evolution of the G matrix in structured populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">180</span>, <span class="cit-fpage">2135</span>–<span class="cit-lpage">2149</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.107.080820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080820%26rft_id%253Dinfo%253Apmid%252F18245845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzIxMzUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8xOS8yOTI1NDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.51" data-doi="10.1111/mec.12043"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mosca</span>  <span class="cit-name-given-names">E</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Di Pierro</span>  <span class="cit-name-given-names">EA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rocchini</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">La Porta</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Belletti</span>  <span class="cit-name-given-names">P</span></span>, <span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">DB</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">The geographical and environmental determinants of genetic diversity for four alpine conifers of the European Alps</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">21</span>, <span class="cit-fpage">5530</span>–<span class="cit-lpage">5545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMosca%26rft.auinit1%253DE.%26rft.volume%253D21%26rft.issue%253D22%26rft.spage%253D5530%26rft.epage%253D5545%26rft.atitle%253DThe%2Bgeographical%2Band%2Benvironmental%2Bdeterminants%2Bof%2Bgenetic%2Bdiversity%2Bfor%2Bfour%2Balpine%2Bconifers%2Bof%2Bthe%2BEuropean%2BAlps.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12043%26rft_id%253Dinfo%253Apmid%252F23058000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23058000&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.52" data-doi="10.1038/hdy.1987.113"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mousseau</span>  <span class="cit-name-given-names">TA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Roff</span>  <span class="cit-name-given-names">DA</span></span> (<span class="cit-pub-date">1987</span>) <span class="cit-article-title">Natural selection and the heritability of fitness components</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr>, <span class="cit-vol">59</span>, <span class="cit-fpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D59%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1987.113%26rft_id%253Dinfo%253Apmid%252F3316130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1987.113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3316130&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K315700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.53" data-doi="10.1534/genetics.111.129387"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ovaskainen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Karhunen</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C</span></span>, <span class="cit-auth"><span class="cit-name-surname">Arias</span>  <span class="cit-name-given-names">JMC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Merila</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">A new method to uncover signatures of divergent and stabilizing selection in quantitative traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">189</span>, <span class="cit-fpage">621632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1534/genetics.111.129387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.129387%26rft_id%253Dinfo%253Apmid%252F21840853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzIvNjIxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMTkvMjkyNTQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.54" data-doi="10.1126/science.1083264"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Aguinagalde</span>  <span class="cit-name-given-names">I</span></span>, <span class="cit-auth"><span class="cit-name-surname">de Beaulieu</span>  <span class="cit-name-given-names">JL</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">Glacial refugia: hotspots but not melting pots of genetic diversity</span>. <abbr class="cit-jnl-abbrev">science</abbr>, <span class="cit-vol">300</span>, <span class="cit-fpage">1563</span>–<span class="cit-lpage">1565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPetit%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D300%26rft.issue%253D5625%26rft.spage%253D1563%26rft.epage%253D1565%26rft.atitle%253DGlacial%2BRefugia%253A%2BHotspots%2BBut%2BNot%2BMelting%2BPots%2Bof%2BGenetic%2BDiversity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1083264%26rft_id%253Dinfo%253Apmid%252F12791991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNS8xNTYzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMTkvMjkyNTQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.55" data-doi="10.1146/annurev.ecolsys.37.091305.110215"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">RJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hampe</span>  <span class="cit-name-given-names">A</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Some evolutionary consequences of being a tree</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">37</span>, <span class="cit-fpage">187</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D37%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.37.091305.110215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.37.091305.110215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000243038500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-292540v2.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pigliucci</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Preston</span>  <span class="cit-name-given-names">KA</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-source">Phenotypic integration: studying the ecology and evolution of complex phenotypes</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.57" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">JK</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">PJ</span></span> (<span class="cit-pub-date">2000</span>) <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">155</span>, <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMTkvMjkyNTQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.58" data-doi="10.1111/mec.13516"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roschanski</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csillery</span>  <span class="cit-name-given-names">K</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liepelt</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Evidence of divergent selection for drought and cold tolerance at landscape and local scales in <em>Abies alba</em> Mill. in the French Mediterranean Alps</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">776</span>–<span class="cit-lpage">794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%26rft.volume%253D25%26rft.spage%253D776%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.59" data-doi="10.3732/apps.1200179"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Roschanski</span>  <span class="cit-name-given-names">AM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fady</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ziegenhagen</span>  <span class="cit-name-given-names">B</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liepelt</span>  <span class="cit-name-given-names">S</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Annotation and re-sequencing of genes from de novo transcriptome assembly of <em>Abies alba</em> (Pinaceae)</span>. <abbr class="cit-jnl-abbrev">Applications in Plant Sciences</abbr>, <span class="cit-vol">1</span>, <span class="cit-fpage">1200179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.3732/apps.1200179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplications%2Bin%2BPlant%2BSciences%26rft.volume%253D1%26rft.spage%253D1200179%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fapps.1200179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3732/apps.1200179&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.60" data-doi="10.1111/gcb.13075"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ruosch</span>  <span class="cit-name-given-names">M</span></span>, <span class="cit-auth"><span class="cit-name-surname">Spahni</span>  <span class="cit-name-given-names">R</span></span>, <span class="cit-auth"><span class="cit-name-surname">Joos</span>  <span class="cit-name-given-names">F</span></span>, <span class="cit-auth"><span class="cit-name-surname">Henne</span>  <span class="cit-name-given-names">PD</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Knaap</span>  <span class="cit-name-given-names">WO</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tinner</span>  <span class="cit-name-given-names">W</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Past and future evolution of <em>Abies alba</em> forests in Europe - comparison of a dynamic vegetation model with palaeo data and observations</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr>, <span class="cit-vol">22</span>, <span class="cit-fpage">727</span>–<span class="cit-lpage">740</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/gcb.13075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft.volume%253D22%26rft.spage%253D727%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.13075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.13075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.61" data-doi="10.1146/annurev.ecolsys.38.091206.095646"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T</span></span>, <span class="cit-auth"><span class="cit-name-surname">Knurr</span>  <span class="cit-name-given-names">T</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Gene flow and local adaptation in trees</span>. <abbr class="cit-jnl-abbrev">Annual Review of Ecology, Evolution, and Systematics</abbr>, <span class="cit-vol">38</span>, <span class="cit-fpage">595</span>–<span class="cit-lpage">619</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1146/annurev.ecolsys.38.091206.095646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D38%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.38.091206.095646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.38.091206.095646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252359800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.62" data-doi="10.1111/j.1469-8137.2012.04230.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">RG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Etterson</span>  <span class="cit-name-given-names">JR</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Rapid climate change and the rate of adaptation: insight from experimental quantitative genetics</span>. <abbr class="cit-jnl-abbrev">New Phytologist</abbr>, <span class="cit-vol">195</span>, <span class="cit-fpage">752</span>–<span class="cit-lpage">765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DShaw%26rft.auinit1%253DR.%2BG.%26rft.volume%253D195%26rft.issue%253D4%26rft.spage%253D752%26rft.epage%253D765%26rft.atitle%253DRapid%2Bclimate%2Bchange%2Band%2Bthe%2Brate%2Bof%2Badaptation%253A%2Binsight%2Bfrom%2Bexperimental%2Bquantitative%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2012.04230.x%26rft_id%253Dinfo%253Apmid%252F22816320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2012.04230.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22816320&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307077800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-292540v2.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Stocker</span>  <span class="cit-name-given-names">TF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">D</span></span>, <span class="cit-auth"><span class="cit-name-surname">Plattner</span>  <span class="cit-name-given-names">GK</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2014</span>) <span class="cit-source">Climate change 2013: The physical science basis</span>. <span class="cit-publ-name">Cambridge University Press Cambridge</span>, <span class="cit-publ-loc">UK, and New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.64" data-doi="10.1093/bioinformatics/btg316"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K</span></span> (<span class="cit-pub-date">2003</span>) <span class="cit-article-title">libsequence: A C++ class library for evolutionary genetic analysis</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">19</span>, <span class="cit-fpage">2325</span>–<span class="cit-lpage">2327</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1093/bioinformatics/btg316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThornton%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2325%26rft.epage%253D2327%26rft.atitle%253Dlibsequence%253A%2Ba%2BC%252B%252B%2Bclass%2Blibrary%2Bfor%2Bevolutionary%2Bgenetic%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg316%26rft_id%253Dinfo%253Apmid%252F14630667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14630667&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186919200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.65" data-doi="10.1016/j.quascirev.2004.06.013"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Van der Knaap</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">JF</span></span>, <span class="cit-auth"><span class="cit-name-surname">Finsinger</span>  <span class="cit-name-given-names">W</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Migration and population expansion of <em>Abies, Fagus, Picea</em>, and <em>Quercus</em> since 15000 years in and across the Alps, based on pollen-percentage threshold values</span>. <abbr class="cit-jnl-abbrev">Quaternary Science Reviews</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">645</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuaternary%2BScience%2BReviews%26rft.stitle%253DQuaternary%2BScience%2BReviews%26rft.volume%253D24%26rft.issue%253D5-6%26rft.spage%253D645%26rft.epage%253D680%26rft.atitle%253DMigration%2Band%2Bpopulation%2Bexpansion%2Bof%2BAbies%252C%2BFagus%252C%2BPicea%252C%2Band%2BQuercus%2Bsince%2B15%252C000%2Byears%2Bin%2Band%2Bacross%2Bthe%2BAlps%252C%2Bbased%2Bon%2Bpollen-percentage%2Bthreshold%2Bvalues%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.quascirev.2004.06.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.quascirev.2004.06.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2006006104&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000227777800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.66" data-doi="10.1093/bioinformatics/bti403"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vilella</span>  <span class="cit-name-given-names">AJ</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blanco-Garcia</span>  <span class="cit-name-given-names">A</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hutter</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">VariScan: analysis of evolutionary patterns from large-scale DNA sequence polymorphism data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">21</span>, <span class="cit-fpage">2791</span>–<span class="cit-lpage">2793</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti403%26rft_id%253Dinfo%253Apmid%252F15814564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15814564&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229441500033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.67" data-doi="10.1038/nrg2322"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">PM</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">WG</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">NR</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Heritability in the genomics era - concepts and misconceptions</span>. <abbr class="cit-jnl-abbrev">Nature Reviews Genetics</abbr>, <span class="cit-vol">9</span>, <span class="cit-fpage">255</span>–<span class="cit-lpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1038/nrg2322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D255%26rft.epage%253D266%26rft.atitle%253DHeritability%2Bin%2Bthe%2Bgenomics%2Bera--concepts%2Band%2Bmisconceptions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2322%26rft_id%253Dinfo%253Apmid%252F18319743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319743&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.68" data-doi="10.1111/gcb.13774"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vitali</span>  <span class="cit-name-given-names">V</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buntgen</span>  <span class="cit-name-given-names">U</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bauhus</span>  <span class="cit-name-given-names">J</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Silver fir and Douglas fir are more tolerant to extreme droughts than Norway spruce in south-western Germany</span>. <abbr class="cit-jnl-abbrev">Global Change Biology</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/gcb.13774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGlobal%2BChange%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgcb.13774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gcb.13774&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.69" data-doi="10.1046/j.1365-2540.1998.00298.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waitt</span>  <span class="cit-name-given-names">DE</span></span>, <span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">DA</span></span> (<span class="cit-pub-date">1998</span>) <span class="cit-article-title">Genetic and phenotypic correlations in plants: a botanical test of Cheverud’s conjecture</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr>, <span class="cit-vol">80</span>, <span class="cit-fpage">310</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D80%26rft.spage%253D310%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2540.1998.00298.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2540.1998.00298.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000072783500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.70" data-doi="10.1175/1520-0442(2004)017&amp;lt;2335:ASPDSI&amp;gt;2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">N</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">MJ</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">A self-calibrating Palmer drought severity index</span>. <abbr class="cit-jnl-abbrev">Journal of Climate</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">2335</span>–<span class="cit-lpage">2351</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1175/1520-0442(2004)017&lt;2335:ASPDSI&gt;2.0.CO;2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClimate%26rft.volume%253D17%26rft.spage%253D2335%26rft_id%253Dinfo%253Adoi%252F10.1175%252F1520-0442%25282004%2529017%2526lt%253B2335%253AASPDSI%2526gt%253B2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/1520-0442(2004)017&amp;lt;2335:ASPDSI&amp;gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000222081800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.71" data-doi="10.1111/j.1365-294X.2008.03712.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">Evolutionary inference from <em>Q<sub>ST</sub></em></span>. <abbr class="cit-jnl-abbrev">Molecular Ecology</abbr>, <span class="cit-vol">17</span>, <span class="cit-fpage">1885</span>–<span class="cit-lpage">1896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DWhitlock%26rft.auinit1%253DM.%2BC.%26rft.volume%253D17%26rft.issue%253D8%26rft.spage%253D1885%26rft.epage%253D1896%26rft.atitle%253DEvolutionary%2Binference%2Bfrom%2BQST.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03712.x%26rft_id%253Dinfo%253Apmid%252F18363667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03712.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18363667&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F04%2F19%2F292540.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254953900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.72" data-doi="10.1534/genetics.108.099812"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">MC</span></span>, <span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Testing for spatially divergent selection: comparing <em>Q<sub>ST</sub></em> to <em>F<sub>ST</sub></em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr>, <span class="cit-vol">183</span>, <span class="cit-fpage">1055</span>–<span class="cit-lpage">1063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.099812%26rft_id%253Dinfo%253Apmid%252F19687138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My8zLzEwNTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wNC8xOS8yOTI1NDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-292540v2.73" data-doi="10.1126/science.aaf7812"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hodgins</span>  <span class="cit-name-given-names">KA</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lotterhos</span>  <span class="cit-name-given-names">KE</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Convergent local adaptation to climate in distantly related conifers</span>. <abbr class="cit-jnl-abbrev">Science</abbr>, <span class="cit-vol">353</span>, <span class="cit-fpage">1431</span>–<span class="cit-lpage">1433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYeaman%26rft.auinit1%253DS.%26rft.volume%253D353%26rft.issue%253D6306%26rft.spage%253D1431%26rft.epage%253D1433%26rft.atitle%253DConvergent%2Blocal%2Badaptation%2Bto%2Bclimate%2Bin%2Bdistantly%2Brelated%2Bconifers%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aaf7812%26rft_id%253Dinfo%253Apmid%252F27708038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTMvNjMwNi8xNDMxIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTgvMDQvMTkvMjkyNTQwLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/292540v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F292540v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/304394v2" title="Human cytomegalovirus tropism modulator UL148 interacts with SEL1L, a cellular factor that governs ER-associated degradation of the viral envelope glycoprotein, gO" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;April 19, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/292540v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/292540v2.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/292540v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/292540v2" class="active" data-icon-position="" data-hide-link-title="0">Adaptation to local climate in a multi-trait space: evidence from silver fir (Abies alba Mill.) populations across a heterogeneous environment</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/93581" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-GKCuUZe-o027ekbEkmVW5WCBSpRwHwhL7DiSPQvR_iQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="93581" id="node93581--2" data-pisa="biorxiv;292540v2" data-pisa-master="biorxiv;292540" data-seqnum="93581" data-apath="/biorxiv/early/2018/04/19/292540.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Adaptation to local climate in a multi-trait space: evidence from silver fir (<em>Abies alba</em> Mill.) populations across a heterogeneous environment</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Katalin</span> <span class="nlm-surname">Csilléry</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Otso</span> <span class="nlm-surname">Ovaskainen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Christoph</span> <span class="nlm-surname">Sperisen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Alex</span> <span class="nlm-surname">Widmer</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Felix</span> <span class="nlm-surname">Gugerli</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">292540; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/292540 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/292540v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/292540v2&amp;title=Adaptation%20to%20local%20climate%20in%20a%20multi-trait%20space%3A%20evidence%20from%20silver%20fir%20%28Abies%20alba%20Mill.%29%20populations%20across%20a%20heterogeneous%20environment" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/292540v2&amp;title=Adaptation%20to%20local%20climate%20in%20a%20multi-trait%20space%3A%20evidence%20from%20silver%20fir%20%28Abies%20alba%20Mill.%29%20populations%20across%20a%20heterogeneous%20environment" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/292540v2&amp;text=Adaptation%20to%20local%20climate%20in%20a%20multi-trait%20space%3A%20evidence%20from%20silver%20fir%20%28Abies%20alba%20Mill.%29%20populations%20across%20a%20heterogeneous%20environment" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/292540v2&amp;title=Adaptation%20to%20local%20climate%20in%20a%20multi-trait%20space%3A%20evidence%20from%20silver%20fir%20%28Abies%20alba%20Mill.%29%20populations%20across%20a%20heterogeneous%20environment" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/292540v2&amp;t=Adaptation%20to%20local%20climate%20in%20a%20multi-trait%20space%3A%20evidence%20from%20silver%20fir%20%28Abies%20alba%20Mill.%29%20populations%20across%20a%20heterogeneous%20environment" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/292540v2&amp;title=Adaptation%20to%20local%20climate%20in%20a%20multi-trait%20space%3A%20evidence%20from%20silver%20fir%20%28Abies%20alba%20Mill.%29%20populations%20across%20a%20heterogeneous%20environment" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/292540v2&amp;title=Adaptation%20to%20local%20climate%20in%20a%20multi-trait%20space%3A%20evidence%20from%20silver%20fir%20%28Abies%20alba%20Mill.%29%20populations%20across%20a%20heterogeneous%20environment" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="93581" id="node93581--3" data-pisa="biorxiv;292540v2" data-pisa-master="biorxiv;292540" data-seqnum="93581" data-apath="/biorxiv/early/2018/04/19/292540.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Adaptation to local climate in a multi-trait space: evidence from silver fir (<em>Abies alba</em> Mill.) populations across a heterogeneous environment</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Katalin</span> <span class="nlm-surname">Csilléry</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Otso</span> <span class="nlm-surname">Ovaskainen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Christoph</span> <span class="nlm-surname">Sperisen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Alex</span> <span class="nlm-surname">Widmer</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Felix</span> <span class="nlm-surname">Gugerli</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">292540; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/292540 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Adaptation%20to%20local%20climate%20in%20a%20multi-trait%20space%3A%20evidence%20from%20silver%20fir%20%28Abies%20alba%20Mill.%29%20populations%20across%20a%20heterogeneous%20environment&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Csill%C3%A9ry%2C+Katalin&amp;rft.au=Ovaskainen%2C+Otso&amp;rft.au=Sperisen%2C+Christoph&amp;rft.au=Widmer%2C+Alex&amp;rft.au=Gugerli%2C+Felix"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/93581/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/93581/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/93581/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/93581/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/93581/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/93581/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/93581/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/93581/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/93581/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/93581/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/93581/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/93581/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/292540v2&amp;count=horizontal&amp;via=&amp;text=Adaptation%20to%20local%20climate%20in%20a%20multi-trait%20space%3A%20evidence%20from%20silver%20fir%20%28Abies%20alba%20Mill.%29%20populations%20across%20a%20heterogeneous%20environment&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/292540v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/292540v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/292540v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
