<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/embed/graphic-5.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2019/07/18/706739.atom" />
<meta name="HW.pisa" content="biorxiv;706739v1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Single-gene resolution of locally adaptive genetic variation in Mexican maize" />
<meta name="DC.Identifier" content="10.1101/706739" />
<meta name="DC.Date" content="2019-07-18" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Threats to crop production due to climate change are one of the greatest challenges facing plant breeders today. While considerable adaptive variation exists in traditional landraces, natural populations of crop wild relatives, and ex situ germplasm collections, separating adaptive alleles from linked deleterious variants that impact agronomic traits is challenging and has limited the utility of these diverse germplasm resources. Modern genome editing techniques such as CRISPR offer a potential solution by targeting specific alleles for transfer to new backgrounds, but such methods require a higher degree of precision than traditional mapping approaches can achieve. Here we present a high-resolution genome-wide association analysis to identify loci exhibiting adaptive patterns in a large panel of more than 4500 traditional maize landraces representing the breadth of genetic diversity of maize in Mexico. We evaluate associations between genotype and plant performance in 13 common gardens across a range of environments, identifying hundreds of candidate genes underlying genotype by environment interaction. We further identify genetic associations with environment across Mexico and show that such loci are associated with variation in yield and flowering time in our field trials and predict performance in independent drought trials. Our results indicate that the variation necessary to adapt crops to changing climate exists in traditional landraces that have been subject to ongoing environmental adaptation and can be identified by both phenotypic and environmental association." />
<meta name="DC.Contributor" content="Daniel J Gates" />
<meta name="DC.Contributor" content="Dan Runcie" />
<meta name="DC.Contributor" content="Garrett M. Janzen" />
<meta name="DC.Contributor" content="Alberto Romero Navarro" />
<meta name="DC.Contributor" content="Martha Willcox" />
<meta name="DC.Contributor" content="Kai Sonder" />
<meta name="DC.Contributor" content="Samantha J. Snodgrass" />
<meta name="DC.Contributor" content="Fausto Rodríguez-Zapata" />
<meta name="DC.Contributor" content="Ruairidh J. H. Sawers" />
<meta name="DC.Contributor" content="Rubén Rellán-Álvarez" />
<meta name="DC.Contributor" content="Edward S. Buckler" />
<meta name="DC.Contributor" content="Sarah Hearne" />
<meta name="DC.Contributor" content="Matthew B. Hufford" />
<meta name="DC.Contributor" content="Jeffrey Ross-Ibarra" />
<meta name="article:published_time" content="2019-07-18" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Single-gene resolution of locally adaptive genetic variation in Mexican maize" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Threats to crop production due to climate change are one of the greatest challenges facing plant breeders today. While considerable adaptive variation exists in traditional landraces, natural populations of crop wild relatives, and &lt;i&gt;ex situ&lt;/i&gt; germplasm collections, separating adaptive alleles from linked deleterious variants that impact agronomic traits is challenging and has limited the utility of these diverse germplasm resources. Modern genome editing techniques such as CRISPR offer a potential solution by targeting specific alleles for transfer to new backgrounds, but such methods require a higher degree of precision than traditional mapping approaches can achieve. Here we present a high-resolution genome-wide association analysis to identify loci exhibiting adaptive patterns in a large panel of more than 4500 traditional maize landraces representing the breadth of genetic diversity of maize in Mexico. We evaluate associations between genotype and plant performance in 13 common gardens across a range of environments, identifying hundreds of candidate genes underlying genotype by environment interaction. We further identify genetic associations with environment across Mexico and show that such loci are associated with variation in yield and flowering time in our field trials and predict performance in independent drought trials. Our results indicate that the variation necessary to adapt crops to changing climate exists in traditional landraces that have been subject to ongoing environmental adaptation and can be identified by both phenotypic and environmental association.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2019/01/01" />
<meta name="citation_mjid" content="biorxiv;706739v1" />
<meta name="citation_id" content="706739v1" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/706739v1" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/706739v1.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/706739v1.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739.full.pdf" />
<meta name="citation_doi" content="10.1101/706739" />
<meta name="citation_num_pages" content="29" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="706739" />
<meta name="citation_author" content="Daniel J Gates" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology and Center for Population Biology, University of California Davis" />
<meta name="citation_author_email" content="danjgates@gmail.com" />
<meta name="citation_author" content="Dan Runcie" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology and Center for Population Biology, University of California Davis" />
<meta name="citation_author" content="Garrett M. Janzen" />
<meta name="citation_author_institution" content="Department of Ecology, Evolution, and Organismal Biology, Iowa State University" />
<meta name="citation_author" content="Alberto Romero Navarro" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University" />
<meta name="citation_author" content="Martha Willcox" />
<meta name="citation_author_institution" content="Centro International de Mejoramiento de Maiz y Trigo (CIMMYT), Carretera Mexico-Veracruz" />
<meta name="citation_author" content="Kai Sonder" />
<meta name="citation_author_institution" content="Centro International de Mejoramiento de Maiz y Trigo (CIMMYT), Carretera Mexico-Veracruz" />
<meta name="citation_author" content="Samantha J. Snodgrass" />
<meta name="citation_author_institution" content="Department of Ecology, Evolution, and Organismal Biology, Iowa State University" />
<meta name="citation_author" content="Fausto Rodríguez-Zapata" />
<meta name="citation_author_institution" content="Department of Molecular and Structural Biochemistry, North Carolina State University" />
<meta name="citation_author_institution" content="Laboratorio Nacional de Genómica para la Biodiversidad/Unidad de Genómica Avanzada" />
<meta name="citation_author_institution" content="Program in Genetics, North Carolina State University" />
<meta name="citation_author" content="Ruairidh J. H. Sawers" />
<meta name="citation_author_institution" content="Laboratorio Nacional de Genómica para la Biodiversidad/Unidad de Genómica Avanzada" />
<meta name="citation_author_institution" content="Department of Plant Science, The Pennsylvania State University" />
<meta name="citation_author" content="Rubén Rellán-Álvarez" />
<meta name="citation_author_institution" content="Laboratorio Nacional de Genómica para la Biodiversidad/Unidad de Genómica Avanzada" />
<meta name="citation_author_institution" content="Department of Molecular and Structural Biochemistry, North Carolina State University" />
<meta name="citation_author" content="Edward S. Buckler" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University" />
<meta name="citation_author" content="Sarah Hearne" />
<meta name="citation_author_institution" content="Centro International de Mejoramiento de Maiz y Trigo (CIMMYT), Carretera Mexico-Veracruz" />
<meta name="citation_author" content="Matthew B. Hufford" />
<meta name="citation_author_institution" content="Department of Ecology, Evolution, and Organismal Biology, Iowa State University" />
<meta name="citation_author" content="Jeffrey Ross-Ibarra" />
<meta name="citation_author_institution" content="Department of Evolution and Ecology and Center for Population Biology, University of California Davis" />
<meta name="citation_author_institution" content="Genome Center, University of California Davis" />
<meta name="citation_reference" content="Hulme, M. 1.5 c and climate research after the paris agreement. Nature Climate Change 6, 222 (2016)." />
<meta name="citation_reference" content="Team, C. W., Pachauri, R. K. &amp; Meyer, L. Ipcc, 2014: climate change 2014: synthesis report. contribution of working groups i. II and III to the Fifth Assessment Report of the intergovernmental panel on Climate Change. IPCC, Geneva, Switzerland 151 (2014)." />
<meta name="citation_reference" content="Lesk, C., Rowhani, P. &amp; Ramankutty, N. Influence of extreme weather disasters on global crop production. Nature 529, 84 (2016)." />
<meta name="citation_reference" content="Challinor, A. J. et al. A meta-analysis of crop yield under climate change and adaptation. Nature Climate Change 4, 287 (2014)." />
<meta name="citation_reference" content="Varshney, R. K., Singh, V. K., Kumar, A., Powell, W. &amp; Sorrells, M. E. Can genomics deliver climate-change ready crops? Current opinion in plant biology (2018)." />
<meta name="citation_reference" content="Bellon, M. R. et al. Evolutionary and food supply implications of ongoing maize domestication by mexican campesinos. Proc. R. Soc. B 285, 20181049 (2018)." />
<meta name="citation_reference" content="Brush, S. B. In situ conservation of landraces in centers of crop diversity. Crop science 35, 346–354 (1995)." />
<meta name="citation_reference" content="McCouch, S. Diversifying selection in plant breeding. PLoS biology 2, e347 (2004)." />
<meta name="citation_reference" content="Young, N. &amp; Tanksley, S. Rflp analysis of the size of chromosomal segments retained around the tm-2 locus of tomato during backcross breeding. Theoretical and Applied Genetics 77, 353–359 (1989)." />
<meta name="citation_reference" content="Chilcoat, D., Liu, Z.-B. &amp; Sander, J. Use of crispr/cas9 for crop improvement in maize and soybean. In Progress in molecular biology and translational science, vol. 149, 27–46 (Elsevier, 2017)." />
<meta name="citation_reference" content="Li, H., Rasheed, A., Hickey, L. T. &amp; He, Z. Fast-forwarding genetic gain. Trends in plant science 23, 184–186 (2018)." />
<meta name="citation_reference" content="Wambugu, P. W., Ndjiondjop, M.-N. &amp; Henry, R. J. Role of genomics in promoting the utilization of plant genetic resources in genebanks. Briefings in functional genomics 17, 198–206 (2018)." />
<meta name="citation_reference" content="Mercer, K. L. &amp; Perales, H. Structure of local adaptation across the landscape: flowering time and fitness in mexican maize (zea mays l. subsp. mays) landraces. Genetic Resources and Crop Evolution 66, 27–45 (2019)." />
<meta name="citation_reference" content="Lei, L. et al. Discovery of barley gene candidates for low temperature and drought tolerance via environmental association. bioRxiv 405399 (2018)." />
<meta name="citation_reference" content="Lasky, J. R. et al. Genome-environment associations in sorghum landraces predict adaptive traits. Science Advances 1, e1400218 (2015)." />
<meta name="citation_reference" content="Anderson, J. E., Kono, T. J., Stupar, R. M., Kantar, M. B. &amp; Morrell, P. L. Environmental association analyses identify candidates for abiotic stress tolerance in glycine soja, the wild progenitor of cultivated soybeans. G3: Genes, Genomes, Genetics 6, 835–843 (2016)." />
<meta name="citation_reference" content="Bandillo, N. B. et al. Dissecting the genetic basis of local adaptation in soybean. Scientific reports 7, 17195 (2017)." />
<meta name="citation_reference" content="Yoder, J. B. et al. Genomic signature of adaptation to climate in medicago truncatula. Genetics genetics–113 (2014)." />
<meta name="citation_reference" content="Hake, S. &amp; Ross-Ibarra, J. The natural history of model organisms: genetic, evolutionary and plant breeding insights from the domestication of maize. Elife 4, e05861 (2015)." />
<meta name="citation_reference" content="Rosenzweig, C. et al. Assessing agricultural risks of climate change in the 21st century in a global gridded crop model intercomparison. Proceedings of the National Academy of Sciences 111, 3268–3273 (2014)." />
<meta name="citation_reference" content="Eagles, H. &amp; Lothrop, J. Highland maize from central mexico—its origin, characteristics, and use in breeding programs. Crop Science 34, 11–19 (1994)." />
<meta name="citation_reference" content="Liu, Z. et al. Expanding maize genetic resources with predomestication alleles: Maize–teosinte introgression populations. The plant genome 9 (2016)." />
<meta name="citation_reference" content="Prasanna, B. Diversity in global maize germplasm: characterization and utilization. Journal of biosciences 37, 843–855 (2012)." />
<meta name="citation_reference" content="Lafitte, H., Edmeades, G. &amp; Johnson, E. Temperature responses of tropical maize cultivars selected for broad adaptation. Field Crops Research 49, 215–229 (1997)." />
<meta name="citation_reference" content="Lafitte, H. &amp; Edmeades, G. Temperature effects on radiation use and biomass partitioning in diverse tropical maize cultivars. Field Crops Research 49, 231–247 (1997)." />
<meta name="citation_reference" content="Runcie, D. E. &amp; Crawford, L. Fast and flexible linear mixed models for genome-wide genetics. bioRxiv 373902 (2019)." />
<meta name="citation_reference" content="Ross-Ibarra, J., Tenaillon, M. &amp; Gaut, B. S. Historical divergence and gene flow in the genus zea. Genetics 181, 1399–1413 (2009)." />
<meta name="citation_reference" content="Hufford, M. B., Martínez-Meyer, E., Gaut, B. S., Eguiarte, L. E. &amp; Tenaillon, M. I. Inferences from the historical distribution of wild and domesticated maize provide ecological and evolutionary insight. PLoS One 7, e47659 (2012)." />
<meta name="citation_reference" content="Hufford, M. B. et al. The genomic signature of crop-wild introgression in maize. PLoS Genetics 9, e1003477 (2013)." />
<meta name="citation_reference" content="Navarro, J. A. R. et al. A study of allelic diversity underlying flowering-time adaptation in maize landraces. Nature genetics 49, 476 (2017)." />
<meta name="citation_reference" content="Takuno, S. et al. Independent molecular basis of convergent highland adaptation in maize. Genetics genetics-115 (2015)." />
<meta name="citation_reference" content="Degenkolbe, T. et al. Differential remodeling of the lipidome during cold acclimation in natural accessions of arabidopsis thaliana. The Plant Journal 72, 972–982 (2012)." />
<meta name="citation_reference" content="Welti, R. et al. Profiling membrane lipids in plant stress responses: Role of phospholipase dα in freezing-induced lipid changes in Arabidopsis. Journal of Biological Chemistry 277, 31994–32002 (2002)." />
<meta name="citation_reference" content="Kisko, M. et al. controls phosphate homeostasis in a zinc-dependent manner. Elife 7 (2018)." />
<meta name="citation_reference" content="Rodríguez-Zapata, F., Sawers, R. &amp; Rellán-Álvarez, R. SoilP: A package to extract phosphorus retention data from soil maps and study local adaptation to phosphorus availability (2018)." />
<meta name="citation_reference" content="Kuppusamy, T. et al. Lipid biosynthesis and protein concentration respond uniquely to phosphate supply during leaf development in highly phosphorus-efficient Hakea prostrata. Plant Physiol. 166, 1891–1911 (2014)." />
<meta name="citation_reference" content="Borrego, E. J. &amp; Kolomiets, M. V. Synthesis and functions of jasmonates in maize. Plants 5 (2016)." />
<meta name="citation_reference" content="Riedelsheimer, C., Brotman, Y., Méret, M., Melchinger, A. E. &amp; Willmitzer, L. The maize leaf lipidome shows multilevel genetic control and high predictive value for agronomic traits. Sci. Rep. 3, 2479 (2013)." />
<meta name="citation_reference" content="Rodríguez-Zapata, F. G x E QTL mapping of a PT x B73 biparental population (2019). URL https://github.com/sawers-rellan-labs/PTxB73_lipid_QTL_GxE." />
<meta name="citation_reference" content="Waters, M. I. N. J. B. L. H. C. H. C., A. &amp; Springer, N. Natural variation for gene expression responses to abiotic stress in maize. The Plant Journal 89, 706–717 (2017)." />
<meta name="citation_reference" content="Fick, S. E. &amp; Hijmans, R. J. Worldclim 2: new 1-km spatial resolution climate surfaces for global land areas. International Journal of Climatology 37, 4302–4315 (2017)." />
<meta name="citation_reference" content="Mercer, K., Martínez-Vásquez, Á. &amp; Perales, H. R. Asymmetrical local adaptation of maize landraces along an altitudinal gradient. Evolutionary Applications 1, 489–500 (2008)." />
<meta name="citation_reference" content="Hung, H.-Y. et al. Zmcct and the genetic basis of day-length adaptation underlying the postdomestication spread of maize. Proceedings of the National Academy of Sciences 109, E1913–E1921 (2012)." />
<meta name="citation_reference" content="Kakumanu, A. et al. Effects of drought on gene expression in maize reproductive and leaf meristem tissue revealed by rna-seq. Plant Physiology pp–112 (2012)." />
<meta name="citation_reference" content="Hu, B. et al. Nitrate-nrt1. 1b–spx4 cascade integrates nitrogen and phosphorus signalling networks in plants. Nature plants 1 (2019)." />
<meta name="citation_reference" content="Dröge-Laser, W. &amp; Weiste, C. The c/s1 bzip network: a regulatory hub orchestrating plant energy homeostasis. Trends in plant science 23, 422–433 (2018)." />
<meta name="citation_reference" content="Opitz, N. et al. Extensive tissue-specific transcriptomic plasticity in maize primary roots upon water deficit. Journal of Experimental Botany 67, 1095–1107 (2015)." />
<meta name="citation_reference" content="Zhang, X., Wang, B., Zhao, Y., Zhang, J. &amp; Li, Z. Auxin and ga signaling play important roles in the maize response to phosphate deficiency. Plant Science (2019)." />
<meta name="citation_reference" content="Xue, R.-G., Zhang, B. &amp; Xie, H.-F. Overexpression of a ntr1 in transgenic soybean confers tolerance to water stress. Plant cell, tissue and organ culture 89, 177–183 (2007)." />
<meta name="citation_reference" content="Moose, S. P. &amp; Sisco, P. H. Glossy15 controls the epidermal juvenile-to-adult phase transition in maize. The Plant Cell 6, 1343–1355 (1994)." />
<meta name="citation_reference" content="Moose, S. P. &amp; Sisco, P. H. Glossy15, an apetala2-like gene from maize that regulates leaf epidermal cell identity. Genes &amp; development 10, 3018–3027 (1996)." />
<meta name="citation_reference" content="Lauter, N., Kampani, A., Carlson, S., Goebel, M. &amp; Moose, S. P. microrna172 down-regulates glossy15 to promote vegetative phase change in maize. Proceedings of the National Academy of Sciences 102, 9412–9417 (2005)." />
<meta name="citation_reference" content="Bianchi, G., Avato, P. &amp; Salamini, F. Glossy mutants of maize. Heredity 42, 391 (1979)." />
<meta name="citation_reference" content="Li, L. et al. The maize glossy6 gene is involved in cuticular wax deposition and drought tolerance. Journal of experimental botany (2019)." />
<meta name="citation_reference" content="Cairns, J. E. et al. Identification of drought, heat, and combined drought and heat tolerant donors in maize. Crop Science 53, 1335–1346 (2013)." />
<meta name="citation_reference" content="Schöffl, F., Prändl, R. &amp; Reindl, A. Regulation of the heat-shock response. Plant physiology 117, 1135–1141 (1998)." />
<meta name="citation_reference" content="Lohmann, C., Eggers-Schumacher, G., Wunderlich, M. &amp; Schöffl, F. Two different heat shock transcription factors regulate immediate early expression of stress genes in arabidopsis. Molecular Genetics and Genomics 271, 11–21 (2004)." />
<meta name="citation_reference" content="Mishra, S. K. et al. In the complex family of heat stress transcription factors, hsfa1 has a unique role as master regulator of thermotolerance in tomato. Genes &amp; Development 16, 1555–1567 (2002)." />
<meta name="citation_reference" content="Jiang, L., Wang, W., Lian, T. &amp; Zhang, C. Manipulation of metabolic pathways to develop vitamin-enriched crops for human health. Frontiers in plant science 8, 937 (2017)." />
<meta name="citation_reference" content="Kremling, K., Diepenbrock, C., Gore, M., Buckler, E. &amp; Bandillo, N. Transcriptome-wide association supplements genome-wide association in zea mays. bioRxiv 363242 (2018)." />
<meta name="citation_reference" content="Hasanuzzaman, M., Nahar, K. &amp; Fujita, M. Role of tocopherol (vitamin e) in plants: Abiotic stress tolerance and beyond. In Emerging technologies and management of crop stress tolerance, 267–289 (Elsevier, 2014)." />
<meta name="citation_reference" content="Bouchet, S. et al. Adaptation of maize to temperate climates: mid-density genome-wide association genetics and diversity patterns reveal key genomic regions, with a major contribution of the vgt2 (zcn8) locus. PLoS One 8, e71377 (2013)." />
<meta name="citation_reference" content="Ku, L. et al. Dual functions of the zmcct-associated quantitative trait locus in flowering and stress responses under long-day conditions. BMC plant biology 16, 239 (2016)." />
<meta name="citation_reference" content="Minow, M. A. et al. Distinct gene networks modulate floral induction of autonomous maize and photoperiod-dependent teosinte. Journal of experimental botany 69, 2937–2952 (2018)." />
<meta name="citation_reference" content="Moutiq, R. Identification of quantitative trait loci associated with flowering, vegetative traits and photoperiod response in maize (zea mays l) (2002)." />
<meta name="citation_reference" content="Beavis, W., Grant, D., Albertsen, M. &amp; Fincher, R. Quantitative trait loci for plant height in four maize populations and their associations with qualitative genetic loci. Theoretical and Applied Genetics 83, 141–145 (1991)." />
<meta name="citation_reference" content="Almeida, G. D. et al. Qtl mapping in three tropical maize populations reveals a set of constitutive and adaptive genomic regions for drought tolerance. Theoretical and Applied Genetics 126, 583–600 (2013)." />
<meta name="citation_reference" content="Zhu, J., Kaeppler, S. M. &amp; Lynch, J. P. Mapping of qtl controlling root hair length in maize (zea mays l.) under phosphorus deficiency. Plant and Soil 270, 299–310 (2005)." />
<meta name="citation_reference" content="Hancock, A. M. et al. Adaptation to climate across the arabidopsis thaliana genome. Science 334, 83–86 (2011)." />
<meta name="citation_reference" content="Russell, J. et al. Exome sequencing of geographically diverse barley landraces and wild relatives gives insights into environmental adaptation. Nature Genetics 48, 1024 (2016)." />
<meta name="citation_reference" content="Scheben, A. &amp; Edwards, D. Towards a more predictable plant breeding pipeline with crispr/cas-induced allelic series to optimize quantitative and qualitative traits. Current opinion in plant biology (2018)." />
<meta name="citation_reference" content="Rodríguez-Leal, D., Lemmon, Z. H., Man, J., Bartlett, M. E. &amp; Lippman, Z. B. Engineering quantitative trait variation for crop improvement by genome editing. Cell 171, 470–480 (2017)." />
<meta name="citation_reference" content="Elshire, R. J. et al. A robust, simple genotyping-by-sequencing (gbs) approach for high diversity species. PloS one 6, e19379 (2011)." />
<meta name="citation_reference" content="Glaubitz, J. C. et al. Tassel-gbs: a high capacity genotyping by sequencing analysis pipeline. PloS one 9, e90346 (2014)." />
<meta name="citation_reference" content="Swarts, K. et al. Genomic estimation of complex traits reveals ancient maize adaptation to temperate north america. Science 357, 512–515 (2017)." />
<meta name="citation_reference" content="Jiao, Y. et al. Improved maize reference genome with single molecule technologies. bioRxiv 079004 (2017)." />
<meta name="citation_reference" content="Zhao, H. et al. Crossmap: a versatile tool for coordinate conversion between genome assemblies. Bioinformatics 30, 1006–1007 (2013)." />
<meta name="citation_reference" content="Jarvis, A., Reuter, H., Nelson, A. &amp; Guevara, E. Hole-filled seamless srtm data v4: International centre for tropical agriculture (ciat): http. srtm. csi. cgiar. org, accessed 31 (2008)." />
<meta name="citation_reference" content="Harris, I., Jones, P. D., Osborn, T. J. &amp; Lister, D. H. Updated high-resolution grids of monthly climatic observations-the cru ts3. 10 dataset. International journal of climatology 34, 623–642 (2014)." />
<meta name="citation_reference" content="Trabucco, A. &amp; Zomer, R.Global aridity index and potential evapotranspiration (et0) climate database v2 (2019). URL https://figshare.com/articles/Global_Aridity_Index_and_Potential_Evapotranspiration_ET0_Climate_Database_v2/7504448." />
<meta name="citation_reference" content="INIFAP &amp; CONABIO. Edafologia mexico 1:1000000. esri shapefile. (1995). URL “http://www.conabio.gob.mx/informacion/metadata/gis/eda251mgw.xml?_httpcache=yes&amp;_xsl=/db/metadata/xsl/fgdc_html.xsl&amp;_indent=no”." />
<meta name="citation_reference" content="Sanchez, P. A., Palm, C. A. &amp; Buol, S. W. Fertility capability soil classification: a tool to help assess soil quality in the tropics. Geoderma 114, 157–185 (2003)." />
<meta name="citation_reference" content="Ahamed, S., Palm, C. A. &amp; Sanchez, P. A. Global aridity index and potential evapotranspiration (et0) climate database v2 (2010). URL http://harvestchoice.org/node/1435." />
<meta name="citation_reference" content="Beissinger, T. M., Rosa, G. J., Kaeppler, S. M., Gianola, D. &amp; De Leon, N. Defining window-boundaries for genomic analyses using smoothing spline techniques. Genetics Selection Evolution 47, 30 (2015)." />
<meta name="citation_reference" content="Lawrence, M. et al. Software for computing and annotating genomic ranges. PLoS computational biology 9, e1003118 (2013)." />
<meta name="citation_reference" content="Yuan, Y. et al. Genome-wide association mapping and genomic prediction analyses reveal the genetic architecture of grain yield and flowering time under drought and heat stress conditions in maize. Frontiers in plant science 9 (2018)." />
<meta name="citation_reference" content="Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic rna-seq quantification. Nature biotechnology 34, 525 (2016)." />
<meta name="citation_reference" content="Pimentel, H., Bray, N. L., Puente, S., Melsted, P. &amp; Pachter, L. Differential analysis of rna-seq incorporating quantification uncertainty. Nature methods 14, 687 (2017)." />
<meta name="twitter:title" content="Single-gene resolution of locally adaptive genetic variation in Mexican maize" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Threats to crop production due to climate change are one of the greatest challenges facing plant breeders today. While considerable adaptive variation exists in traditional landraces, natural populations of crop wild relatives, and ex situ germplasm collections, separating adaptive alleles from linked deleterious variants that impact agronomic traits is challenging and has limited the utility of these diverse germplasm resources. Modern genome editing techniques such as CRISPR offer a potential solution by targeting specific alleles for transfer to new backgrounds, but such methods require a higher degree of precision than traditional mapping approaches can achieve. Here we present a high-resolution genome-wide association analysis to identify loci exhibiting adaptive patterns in a large panel of more than 4500 traditional maize landraces representing the breadth of genetic diversity of maize in Mexico. We evaluate associations between genotype and plant performance in 13 common gardens across a range of environments, identifying hundreds of candidate genes underlying genotype by environment interaction. We further identify genetic associations with environment across Mexico and show that such loci are associated with variation in yield and flowering time in our field trials and predict performance in independent drought trials. Our results indicate that the variation necessary to adapt crops to changing climate exists in traditional landraces that have been subject to ongoing environmental adaptation and can be identified by both phenotypic and environmental association." />
<meta name="og-title" property="og:title" content="Single-gene resolution of locally adaptive genetic variation in Mexican maize" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/706739v1" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Threats to crop production due to climate change are one of the greatest challenges facing plant breeders today. While considerable adaptive variation exists in traditional landraces, natural populations of crop wild relatives, and ex situ germplasm collections, separating adaptive alleles from linked deleterious variants that impact agronomic traits is challenging and has limited the utility of these diverse germplasm resources. Modern genome editing techniques such as CRISPR offer a potential solution by targeting specific alleles for transfer to new backgrounds, but such methods require a higher degree of precision than traditional mapping approaches can achieve. Here we present a high-resolution genome-wide association analysis to identify loci exhibiting adaptive patterns in a large panel of more than 4500 traditional maize landraces representing the breadth of genetic diversity of maize in Mexico. We evaluate associations between genotype and plant performance in 13 common gardens across a range of environments, identifying hundreds of candidate genes underlying genotype by environment interaction. We further identify genetic associations with environment across Mexico and show that such loci are associated with variation in yield and flowering time in our field trials and predict performance in independent drought trials. Our results indicate that the variation necessary to adapt crops to changing climate exists in traditional landraces that have been subject to ongoing environmental adaptation and can be identified by both phenotypic and environmental association." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2019-07-18" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/706739v1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/706739v1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/706739v1.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/706739v1" />
<link rel="shortlink" href="https://www.biorxiv.org/node/823424" />
    <title>Single-gene resolution of locally adaptive genetic variation in Mexican maize | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__icrP-Y0qw2NP73rJxxkUtrJta0z3VLAABtNjfSdEghA__rlN3rx0wOoswGNHZMeZM96bNn-3cLWHXTztGDoThNlw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__q48SDxVDWut_cH6dTPyQe7HjADuSLLmJVFztuiJEFrU__DJqddQGjm5dXCHsaiwNriGytKN5eevIQqyjdq65TZtE__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8LhnbR2iAmVW0ZyXI6QuFVYFRORa5XxglXs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bOnTzo4vaQ-hrWzaG6oD19l-R9CQ1YQ-o__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__W6wAjopaALytcBam7oT3WhEzllkWaB9R-8hazHyvShM__rK5YBIqBy9qt0di0Sn4DriBx9TJupdmv6uGmLJyZvDs__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"N50czApbkQBiHTuf1rw3e0cXE-HTRi2lYholQ9dMLIU","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"823424","apath":"\/biorxiv\/early\/2019\/07\/18\/706739.atom","pisa":"biorxiv;706739v1","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;706739v1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/706739v1","title":"Single-gene resolution of locally adaptive genetic variation in Mexican maize","identifier":"node\/823424"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wrXRehnRQ7Hb914dtwr95WNaD7ekLHGqfujK2kxvE42BaosB2I1xAA2\\\\\\\/gQz9XvaeBYpLjFfgnyykMD7kGTuPvBT+x7nSZr3tbepLjkgUXpylYlYMkby+JtjtWVfr4IE8oX6O78+NsUSpA1nwsFg0xuEKSEHVRp8MLLt7nL9GTxg32E5MU2kw\\\\\\\/csiRD9QdiJJZF4VWzzyVCrov5oESgWAx7+HoBofwW5VNar6lHPtpB1H+Askzz9hPvmyEjgb4dflT0b2McDxg+GzOQADugvtqiBVGvkYvrkrD5cFAwB\\\\\\\/ka+3Fpes32UZYEeleH4E9btF+rbjs5nGoZ8o1+v40aH\\\\\\\/kF9Kd\\\\\\\/1OacJ4bM\\\\\\\/5Dswojx1FegsRPVkjte15XGx0oMXixQNDNNQE2RH+QZZ6GDkkrk2BcNGjG6H1YZkKH1qdJblGDZSCzUydzt\\\\\\\/HYtqhUl\\\\\\\/3uE++XBawHxlxmgZs1\\\\\\\/TSGxnVNXtZTGQYPU\\\\\\\/3SwSFLqtC87rkc3I2bM9368IcgO9LmfrEGL80Uefbuz8hnPUxdKtMuoE6sWjTZ2pRlkJ6w9xBxDCoU7\\\\\\\/pB9WFH7\\\\\\\/6SUsTuSZpuqFInfXmYFfJB4LIgkDYW3YuYqSc+FLjn1pzyXoCeUeCVLiXqGyfUB+H8vbY156ZpZi1faBoPLdbp2nO+sv+n04rTK4MiPW6LkVnOV48zoCZCVp3SEVLphEaZunGt01l8vi67tQXMGdDdSchN2l\\\\\\\/cyvU5lcwq1GRXNy8JRvwrTR24TC6yCZz8V1lu3cwOZJPFYf+\\\\\\\/Ud0OJrgaRp7HqOvO1H2Gz7JMCZRWyT6pr5tgRQb\\\\\\\/U67cLfqAqYFT08AkQIdNn7i3H75JCQsUJrrzhzW6h\\\\\\\/ByxQ9qOeviN44F8+CRx2miEMwvNToaU2E9BBILhCGbOWHoFwafIX7wFBNKxNRvaORNsb62X5XGk2ex9TNEZ5GzHwR5rFsLCqJQh2fsK5vl0AmG48ce1zXdEsyDp4f7YwQbLjUw0JG\\\\\\\/w8QIqrEYLbhbPipyKdDFvm2rKnaBkeo8x\\\\\\\/I5lssHqAs8NZFGpT6+nxU61ov8+AnIW6QHdRbDq6Cpt5hwaXoARy7uiND8ysBXZPoVt0CvuhewT+ZIj9a72JIvp2v+hNDml1KT6RZ3V4l5a7vzRco8\\\\\\\/Bx\\\\\\\/QLdiq9\\\\\\\/itc5q1SDJBMqKBuCtrgbMMf5j3f1O7lyno1DXGwpa86qyblm\\\\\\\/9mN\\\\\\\/N7o5YQYCRbtLdcbVkve+4kffh2wvhUZA00HklhwysHvLvsbDHKuj5lPrsiJPEN0ITWlUkCjDPAasr7EyIBT\\\\\\\/iNyNRhV1E34T9\\\\\\\/GQO5My7BqdGbgpng=\\\u0022,\\\u0022iv\\\u0022:\\\u0022uzYLgpGAcZrQDPFCo1wcOQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00224b708b688a431e0b4414858f4be8bb8bb083645add16a4b325eb8a4d6e480dbb\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/706739v1.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-823424 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/706739v1.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tn706KiBF8KYKqa6F2PRIm_FeJoiGqIIXo9gjzRX0Y0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="823424" id="node823424" data-pisa="biorxiv;706739v1" data-pisa-master="biorxiv;706739" data-apath="/biorxiv/early/2019/07/18/706739.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Single-gene resolution of locally adaptive genetic variation in Mexican maize</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Daniel J</span> <span class="nlm-surname">Gates</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dan</span> <span class="nlm-surname">Runcie</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Garrett M.</span> <span class="nlm-surname">Janzen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Alberto Romero</span> <span class="nlm-surname">Navarro</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Martha</span> <span class="nlm-surname">Willcox</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kai</span> <span class="nlm-surname">Sonder</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Samantha J.</span> <span class="nlm-surname">Snodgrass</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Fausto</span> <span class="nlm-surname">Rodríguez-Zapata</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Ruairidh J. H.</span> <span class="nlm-surname">Sawers</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Rubén</span> <span class="nlm-surname">Rellán-Álvarez</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Edward S.</span> <span class="nlm-surname">Buckler</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Sarah</span> <span class="nlm-surname">Hearne</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Matthew B.</span> <span class="nlm-surname">Hufford</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Jeffrey</span> <span class="nlm-surname">Ross-Ibarra</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/706739 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node823424" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel J Gates </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Evolution and Ecology and Center for Population Biology, University of California Davis</span>, Davis, CA 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJ%2BGates%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gates%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BJ%2BGates%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:danjgates@gmail.com" class="" data-icon-position="" data-hide-link-title="0">danjgates@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dan Runcie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Evolution and Ecology and Center for Population Biology, University of California Davis</span>, Davis, CA 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BRuncie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Runcie%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADan%2BRuncie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Garrett M. Janzen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology, Evolution, and Organismal Biology, Iowa State University</span>, Ames, Ia 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Garrett%2BM.%2BJanzen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Janzen%20GM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGarrett%2BM.%2BJanzen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alberto Romero Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute for Genomic Diversity, Cornell University</span>, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alberto%2BRomero%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlberto%2BRomero%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Martha Willcox </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centro International de Mejoramiento de Maiz y Trigo (CIMMYT), Carretera Mexico-Veracruz</span>, El Batan, 56130, <span class='nlm-country'>Mexico</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martha%2BWillcox%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Willcox%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartha%2BWillcox%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kai Sonder </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centro International de Mejoramiento de Maiz y Trigo (CIMMYT), Carretera Mexico-Veracruz</span>, El Batan, 56130, <span class='nlm-country'>Mexico</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BSonder%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sonder%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKai%2BSonder%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Samantha J. Snodgrass </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology, Evolution, and Organismal Biology, Iowa State University</span>, Ames, Ia 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samantha%2BJ.%2BSnodgrass%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snodgrass%20SJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASamantha%2BJ.%2BSnodgrass%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Fausto Rodríguez-Zapata </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Molecular and Structural Biochemistry, North Carolina State University</span>, Raleigh, NC</div><div class='author-affiliation'><span class='nlm-institution'>Laboratorio Nacional de Genómica para la Biodiversidad/Unidad de Genómica Avanzada</span>, Cinvestav, Irapuato, <span class='nlm-country'>México</span></div><div class='author-affiliation'><span class='nlm-institution'>Program in Genetics, North Carolina State University</span>, Raleigh, NC 27695-7614, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fausto%2BRodr%C3%ADguez-Zapata%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodr%C3%ADguez-Zapata%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFausto%2BRodr%25C3%25ADguez-Zapata%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Ruairidh J. H. Sawers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratorio Nacional de Genómica para la Biodiversidad/Unidad de Genómica Avanzada</span>, Cinvestav, Irapuato, <span class='nlm-country'>México</span></div><div class='author-affiliation'><span class='nlm-institution'>Department of Plant Science, The Pennsylvania State University</span>, State College, PA, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ruairidh%2BJ.%2BH.%2BSawers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sawers%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARuairidh%2BJ.%2BH.%2BSawers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Rubén Rellán-Álvarez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Laboratorio Nacional de Genómica para la Biodiversidad/Unidad de Genómica Avanzada</span>, Cinvestav, Irapuato, <span class='nlm-country'>México</span></div><div class='author-affiliation'><span class='nlm-institution'>Department of Molecular and Structural Biochemistry, North Carolina State University</span>, Raleigh, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rub%C3%A9n%2BRell%C3%A1n-%C3%81lvarez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rell%C3%A1n-%C3%81lvarez%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARub%25C3%25A9n%2BRell%25C3%25A1n-%25C3%2581lvarez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Institute for Genomic Diversity, Cornell University</span>, Ithaca, NY 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Sarah Hearne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Centro International de Mejoramiento de Maiz y Trigo (CIMMYT), Carretera Mexico-Veracruz</span>, El Batan, 56130, <span class='nlm-country'>Mexico</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sarah%2BHearne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hearne%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASarah%2BHearne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Matthew B. Hufford </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology, Evolution, and Organismal Biology, Iowa State University</span>, Ames, Ia 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BB.%2BHufford%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hufford%20MB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BB.%2BHufford%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Jeffrey Ross-Ibarra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Evolution and Ecology and Center for Population Biology, University of California Davis</span>, Davis, CA 95616</div><div class='author-affiliation'><span class='nlm-institution'>Genome Center, University of California Davis</span>, Davis, Ca 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BRoss-Ibarra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ross-Ibarra%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BRoss-Ibarra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/706739v1" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:823424" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:823424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/706739v1.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:823424" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:823424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/706739v1.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:823424" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:823424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/706739v1.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:823424" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:823424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/706739v1.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:823424" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:823424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/706739v1.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:823424" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:823424/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-3">Threats to crop production due to climate change are one of the greatest challenges facing plant breeders today. While considerable adaptive variation exists in traditional landraces, natural populations of crop wild relatives, and <em>ex situ</em> germplasm collections, separating adaptive alleles from linked deleterious variants that impact agronomic traits is challenging and has limited the utility of these diverse germplasm resources. Modern genome editing techniques such as CRISPR offer a potential solution by targeting specific alleles for transfer to new backgrounds, but such methods require a higher degree of precision than traditional mapping approaches can achieve. Here we present a high-resolution genome-wide association analysis to identify loci exhibiting adaptive patterns in a large panel of more than 4500 traditional maize landraces representing the breadth of genetic diversity of maize in Mexico. We evaluate associations between genotype and plant performance in 13 common gardens across a range of environments, identifying hundreds of candidate genes underlying genotype by environment interaction. We further identify genetic associations with environment across Mexico and show that such loci are associated with variation in yield and flowering time in our field trials and predict performance in independent drought trials. Our results indicate that the variation necessary to adapt crops to changing climate exists in traditional landraces that have been subject to ongoing environmental adaptation and can be identified by both phenotypic and environmental association.</p></div><div class="section" id="sec-1"><h2 class="">Introduction</h2><p id="p-4">Protecting crop and wild plant populations against the harmful effects of climate change is one of the most important goals of plant genetic research today. Temperature increases of 1.5°C above preindustrial levels are already occurring (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>) with further increases of 4-7°C likely in the next 100 years (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>). Agricultural systems have already been impacted by warming (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>), and models under future warming scenarios indicate this trend will be exacerbated, with yield losses across major staple commodities in the range of 25-50% by 2100 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>).</p><p id="p-5">One promising strategy for maintaining crop productivity under climate change is to harness existing genetic variation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). Traditional domesticated varieties (<em>i.e</em>., landraces) and the wild relatives of many crops have been evolving in diverse environments for thousands of years (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>), resulting in a broad base of genetic variation and adaptation to a wide range of environmental niches. Breeders have nonetheless been reluctant to use landrace and wild relative resources because incorporation of novel diversity requires many cycles of backcrossing and phenotyping to minimize linkage drag (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>). But recent advances in breeding techniques such as rapid cycle genomic selection and molecular approaches including CRISPR/Cas9 now allow targeted transfer of alleles, saving both time and cost in breeding (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>) and present novel opportunities to take advantage of novel adaptive alleles. Transfer in this manner, however, requires detailed knowledge of candidate adaptive loci with precision to the gene or even nucleotide level. A significant impediment to this approach, then, is our limited understanding of the genetic basis of adaptation in most crops and their wild relatives.</p><p id="p-6">Adaptive variation can be identified in crops by assembling germplasm sampled across gradients of interest, evaluating field trials across environmental extremes, and identifying population genetic signatures consistent with adaptation. Farmer seed networks and <em>ex situ</em> conservation efforts have preserved extensive collections of wild and landrace germplasm (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>). Adaptation in such materials can be identified through evaluation in observation trials (common gardens) and reciprocal transplant experiments that evaluate variation in fitness components across environments. Maize landraces grown across in common gardens across elevational gradients in Mexico, for instance, demonstrated substantially higher fitness when grown closer to their natal environments (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>). Population genetic signatures of adaptation can also be identified using correlations between environmental gradients and allele frequencies (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>). For example, (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">15</a>) showed that Sorghum alleles associated with drought and low pH can be used to predict plant phenotype, and (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>) predicted <em>Medicago</em> growth in controlled environments using environmentally associated SNPs. While there is great promise for identifying adaptive variation for crop improvement using these combined approaches, comprehensive efforts are lacking.</p><p id="p-7">Maize is the most productive crop globally and has adapted to grow in nearly every environment that humans inhabit (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>), but yield in modern maize lines has already been impacted by climate change (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>). Maize landraces and its wild relatives the teosintes are well adapted to their distinct local environments (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">6</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>) and constitute an extensive but largely untapped well of genetic diversity (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). CIMMYT (the international maize and wheat improvement center), which is part of the CGIAR network, curates the world’s largest collection of maize landrace accessions (&gt;24,000), a resource that could play a crucial breeding role in mitigating the negative effects of climate change on maize.</p><p id="p-8">In this study, we identify adaptive genetic variation in a core set of CIMMYT’s global collection of maize landraces based on the results of field trials replicated across a range of environments and a population genetic analysis that identifies loci associated with environmental variation across the landscape. We demonstrate that alleles exhibiting genotype by environment (<em>G</em> × <em>E</em>) interactions for fitness across trials are predictably associated with climatic variables spanning the range of landraces in Mexico. Likewise, we show that environmental association (eGWAS) can identify candidate genetic loci which are linked to phenotypic and fitness differences across trials. We show substantial overlap in the loci identified by these analyses; alleles at loci associated with fitness difference between trials tend to be found in their beneficial environments. We validate a set of loci associated with precipitation by demonstrating their effect on fitness in independent drought trials, and identify a list of candidate genes from both approaches that also show population genetic evidence of recent selection.</p></div><div class="section" id="sec-2"><h2 class="">Results</h2><p id="p-9">To identify patterns of local adaptation in maize landraces we assayed plant phenotypes across 23 trials in 13 different common garden field locations in Mexico (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Fig. 1A</a>). Briefly, each of more than 2,200 landrace individuals was crossed to a locally adapted hybrid tester and the F1 progeny were grown at each site, resulting in a total of approximately 280,000 plants with phenotypic data (see Methods). Maize plants in these common gardens exhibit a classic phenotypic pattern of local adaptation: plants are taller, flower earlier, and exhibit higher yield (measured as field weight, bare cob weight, or grain weight per hectare) and less stress (lower anthesis-silking interval) when grown at elevations closer to their original collection locality (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1B</a>; <em>P</em> &lt; 1e<sup>-06</sup> for each phenotype). We observe similar patterns for other environmental variables including mean temperature, precipitation, isothermality, and the number of wet days an experimental trial receives (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Supplemental Figs. S4</a>-<a id="xref-fig-11-1" class="xref-fig" href="#F11">S7</a>), suggesting broad adaptation to local environments, consistent with previous transplant experiments (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average elevation of origin of plants grown in elevations at different trials." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average elevation of origin of plants grown in elevations at different trials.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F5.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F5.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure S1.</span> <p id="p-39" class="first-child">Average elevation of origin of plants grown in elevations at different trials.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of phenotypes collected at different elevational classes" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of phenotypes collected at different elevational classes&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F6.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F6.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S2.</span> <p id="p-41" class="first-child">Number of phenotypes collected at different elevational classes</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Yield performance in drought trials of precipitation outlier alleles categorized by environmental preference" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Yield performance in drought trials of precipitation outlier alleles categorized by environmental preference&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F7.medium.gif" width="440" height="290"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F7.medium.gif" width="440" height="290"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S3.</span> <p id="p-42" class="first-child">Yield performance in drought trials of precipitation outlier alleles categorized by environmental preference</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plant phenotypes by difference in home and experimental trial temperature" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plant phenotypes by difference in home and experimental trial temperature&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F8.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F8.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S4.</span> <p id="p-43" class="first-child">Plant phenotypes by difference in home and experimental trial temperature</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plant phenotypes by difference in home and experimental trial precipitation" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plant phenotypes by difference in home and experimental trial precipitation&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F9.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F9.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S5.</span> <p id="p-44" class="first-child">Plant phenotypes by difference in home and experimental trial precipitation</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plant phenotypes by difference in home and experimental trial isothermality" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plant phenotypes by difference in home and experimental trial isothermality&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F10.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F10.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S6.</span> <p id="p-45" class="first-child">Plant phenotypes by difference in home and experimental trial isothermality</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plant phenotypes by difference in home and experimental trial wet days" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plant phenotypes by difference in home and experimental trial wet days&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F11.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F11.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S7.</span> <p id="p-46" class="first-child">Plant phenotypes by difference in home and experimental trial wet days</p><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Maize plants grown in experimental gardens are adapted to diverse environments. A) Temperature, elevation, and precipitation across Mexico (Z-standardized). The bottom plot shows locations that landraces originated from (black) and the location of phenotyping trials (red). B) Fitness decreases as elevational distance between common garden and collection locality increases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Maize plants grown in experimental gardens are adapted to diverse environments. A) Temperature, elevation, and precipitation across Mexico (Z-standardized). The bottom plot shows locations that landraces originated from (black) and the location of phenotyping trials (red). B) Fitness decreases as elevational distance between common garden and collection locality increases.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F1.medium.gif" width="440" height="403"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F1.medium.gif" width="440" height="403"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 1.</span> <p id="p-10" class="first-child">Maize plants grown in experimental gardens are adapted to diverse environments. A) Temperature, elevation, and precipitation across Mexico (Z-standardized). The bottom plot shows locations that landraces originated from (black) and the location of phenotyping trials (red). B) Fitness decreases as elevational distance between common garden and collection locality increases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">To identify loci underpinning this strong genotype by environment (<em>G</em> × <em>E</em>) effect on plant fitness, we implemented a mixed linear model in GridLMM (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>) to perform an association analysis of fitness-related phenotypes (field weight, grain weight per hectare, bare cob weight) across our study sites (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Fig. 2A</a>; see Methods). For each of the top 1000 SNPs, showing significant <em>G</em> × <em>E</em>, we polarize the major allele by the field trial environment in which it increases fitness. This polarization accurately predicts where alleles are naturally found across the landscape: alleles that increase fitness in high elevation trials, for example, are found roughly 100-400 meters higher than random set of frequency-matched alleles and 400-600 meters higher than alleles that increase fitness at low elevations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2B</a>). In addition to predicting the modern geography of maize variants, our top <em>G</em> × <em>E</em> loci also predict allele frequency differences between distinct locally adapted teosinte subspecies. Lowland (<em>Zea mays</em> ssp. <em>parviglumis</em>) and highland (<em>Zea mays</em> ssp. <em>mexicana</em>) teosinte diverged 60,000 years ago (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>) and subsequently adaptated to distinct ecological niches (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>). Yet we show that long-term ecological divergence between these wild subspecies is predictable from the <em>G</em> × <em>E</em> effects of an allele in maize (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2C</a>), suggesting either selection has maintained such alleles as standing variation over relatively long periods or gene flow has transferred adaptive alleles into maize (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotype by environment effects predict SNP association with the environment and frequency in wild populations. A) Genome-wide association of G &#xD7; E effects for three fitness-related traits across all common gardens. B) Alleles that are favored in high elevation trials exist at higher elevations across Mexico (top) and show a larger frequency difference between the highland teosinte Zea mays ssp. mexicana and its lowland counterpart Zea mays ssp. parviglumis (bottom). Colors represent the phenotypes in part A. Lines extending above and below points represent 95% confidence intervals. In both plots, SNPs showing significant GWAS effects are compared to a control set with matched allele frequencies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotype by environment effects predict SNP association with the environment and frequency in wild populations. A) Genome-wide association of G &#xD7; E effects for three fitness-related traits across all common gardens. B) Alleles that are favored in high elevation trials exist at higher elevations across Mexico (top) and show a larger frequency difference between the highland teosinte &lt;em&gt;Zea mays&lt;/em&gt; ssp. &lt;em&gt;mexicana&lt;/em&gt; and its lowland counterpart &lt;em&gt;Zea mays&lt;/em&gt; ssp. &lt;em&gt;parviglumis&lt;/em&gt; (bottom). Colors represent the phenotypes in part A. Lines extending above and below points represent 95% confidence intervals. In both plots, SNPs showing significant GWAS effects are compared to a control set with matched allele frequencies.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F2.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F2.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-12" class="first-child">Genotype by environment effects predict SNP association with the environment and frequency in wild populations. A) Genome-wide association of G × E effects for three fitness-related traits across all common gardens. B) Alleles that are favored in high elevation trials exist at higher elevations across Mexico (top) and show a larger frequency difference between the highland teosinte <em>Zea mays</em> ssp. <em>mexicana</em> and its lowland counterpart <em>Zea mays</em> ssp. <em>parviglumis</em> (bottom). Colors represent the phenotypes in part A. Lines extending above and below points represent 95% confidence intervals. In both plots, SNPs showing significant GWAS effects are compared to a control set with matched allele frequencies.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">One of the clearest features of our <em>G</em> × <em>E</em> association is a broad peak on the long arm of chromosome 4 that is identified for several distinct phenotypes (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figs. 2A</a> and <a id="xref-fig-12-1" class="xref-fig" href="#F12">S8</a>). This region harbors the well-characterized inversion polymorphism <em>Inv4m</em> which spans ≈ 15Mb. The non-reference haplotype at <em>Inv4m</em> appears to have arisen via introgression from the highland teosinte subspecies <em>Z. mays</em> ssp. <em>mexicana</em> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>), is associated with a number of adaptive phenotypes (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>), and shows strong population genetic signals of selection (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). Although large-scale polymorphisms like <em>Inv4m</em> often stand out, a key advantage of our association approach is its high resolution — 41% contain a single gene, and nearly 80% contain fewer than three genes. A detailed description of association peaks and potential candidate genes can be found in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Supplemental Tables S1</a>-S6.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/825991/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup"><a href="/highwire/markup/825991/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/825991" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-40" class="first-child">Number of accessions and trials of different traits</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots for GxE GWAS analysis" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Manhattan plots for GxE GWAS analysis&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F12.medium.gif" width="293" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S8." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F12.medium.gif" width="293" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S8.</span> <p id="p-47" class="first-child">Manhattan plots for GxE GWAS analysis</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Variation in lipid composition has been linked with multiple environmental factors (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>), is associated with variation in flowering time (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>), and population genetic analysis suggests that lipid metabolic pathways may have been important in local adaptation to highland regions in Mexico and South America (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>). We thus compared our results to a QTL analysis of <em>G</em> × <em>E</em> for phospholipid metabolites in Mexican highland maize (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>). We find a number of specific lipid <em>G</em> × <em>E</em> QTLs that overlap with loci we identify as showing <em>G</em> × <em>E</em> flowering time and yield across elevation (Supplemental Table S7). Among the overlapping regions, we found three genes (Zm00001d043228, Zm00001d043207 and Zm00001d039542) involved in the synthesis and degradation of phospholipids. All three are up-regulated after cold stress in temperate inbreds (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>) (<a id="xref-fig-13-1" class="xref-fig" href="#F13">Supplemental Fig. S9</a>).</p><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression values of lipid QTLxE candidate genes overlapping with GxE outliers. Data at each condition shows expression values of B73, Mo17, Oh43 and PH207 inbreds. Data was taken from (40)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Expression values of lipid QTLxE candidate genes overlapping with GxE outliers. Data at each condition shows expression values of B73, Mo17, Oh43 and PH207 inbreds. Data was taken from (40).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F13.medium.gif" width="337" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S9." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F13.medium.gif" width="337" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S9.</span> <p id="p-48" class="first-child">Expression values of lipid QTLxE candidate genes overlapping with GxE outliers. Data at each condition shows expression values of B73, Mo17, Oh43 and PH207 inbreds. Data was taken from (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">40</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-15">While our <em>G</em> × <em>E</em> associations directly test the link between fitness and genotype across many trial environments, environmental association offers even greater climatic precision. Using genotype data from nearly 4,000 geo-referenced maize landraces across the Americas (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1A</a> 30), we evaluated the association between SNP genotype and both 50-year average climate (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>) and fine-scale measurements from local weather stations over the course of the local growing season (see Methods). As with our previous phenotypic <em>G</em> × <em>E</em> results, our environmental associations (eGWAS) highlight <em>Inv4m</em> but nonetheless exhibit a similar fine-scale resolution (41% of association peaks contain a single gene, while only 20% contain more than two genes).</p><p id="p-16">Loci identified in our environmental association (<a id="xref-fig-14-1" class="xref-fig" href="#F14">Supplemental Figs. S10</a>, <a id="xref-fig-15-1" class="xref-fig" href="#F15">S11</a> show strong <em>G</em> × <em>E</em> effects on yield, increasing plant survival and reproduction in trials most similar to their associated environment (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3B</a>); this effect also holds for a number of relevant phenotypes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3C</a>). But while eGWAS SNPs show strong <em>G</em> × <em>E</em> for fitness, we see little difference in their average main effect S13, as expected if such alleles are favorable only in a limited number of environments. In contrast, we see only a very weak <em>G</em> × <em>E</em> effect of eGWAS SNPs on flowering time (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3D</a>), but a large main effect. While selection will quickly fix or remove variants with large average effects on fitness, variants with large effects on flowering time or other phenotypes can segregate at appreciable frequency (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>) if the phenotypic optimum varies across the species’ range (e.g. 42; 43).</p><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots for environmental GWAS analysis" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Manhattan plots for environmental GWAS analysis&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F14.medium.gif" width="293" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S10." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F14.medium.gif" width="293" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S10.</span> <p id="p-49" class="first-child">Manhattan plots for environmental GWAS analysis</p><div class="sb-div caption-clear"></div></div></div><div id="F15" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F15.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots for environmental GWAS analysis (continued)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Manhattan plots for environmental GWAS analysis (continued)&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F15.medium.gif" width="251" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S11." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F15.medium.gif" width="251" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S11." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S11.</span> <p id="p-50" class="first-child">Manhattan plots for environmental GWAS analysis (continued)</p><div class="sb-div caption-clear"></div></div></div><div id="F16" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F16.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise comparisons of outliers calculated through different GWAS methods TASSEL, lm(R function), LFMM. All analyses use the first two PCs (TASSEL,lm) or two latent factors (LFMM) as a representative of population structure. TASSEL and lm are most similar likely reflecting differences in how latent factors and principal components account for population structure. All correlations are significant P &lt; 2e-16" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pairwise comparisons of outliers calculated through different GWAS methods TASSEL, lm(R function), LFMM. All analyses use the first two PCs (TASSEL,lm) or two latent factors (LFMM) as a representative of population structure. TASSEL and lm are most similar likely reflecting differences in how latent factors and principal components account for population structure. All correlations are significant &lt;em&gt;P&lt;/em&gt; &lt; 2e&lt;sup&gt;-16&lt;/sup&gt;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F16.medium.gif" width="440" height="285"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S12." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F16.medium.gif" width="440" height="285"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S12." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S12.</span> <p id="p-51" class="first-child">Pairwise comparisons of outliers calculated through different GWAS methods TASSEL, lm(R function), LFMM. All analyses use the first two PCs (TASSEL,lm) or two latent factors (LFMM) as a representative of population structure. TASSEL and lm are most similar likely reflecting differences in how latent factors and principal components account for population structure. All correlations are significant <em>P</em> &lt; 2e<sup>-16</sup></p><div class="sb-div caption-clear"></div></div></div><div id="F17" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F17.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The main effect estimates of elevation associated alleles on phenotypes of the six different traits." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The main effect estimates of elevation associated alleles on phenotypes of the six different traits.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure S13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F17.medium.gif" width="440" height="235"/><noscript><img class="highwire-fragment fragment-image" alt="Figure S13." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F17.medium.gif" width="440" height="235"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S13." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure S13.</span> <p id="p-52" class="first-child">The main effect estimates of elevation associated alleles on phenotypes of the six different traits.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Adaptive variation from environmental GWAS. A) Manhattan plots showing associations with temperature, precipitation, isothermality, and the number of wet days in the growing season. B) SNPs identified via environmental association show strong G &#xD7; E effects on grain weight per hectare, increasing fitness in common gardens more similar to their environment of origin. C) Environmental GWAS hits across elevation predict beneficial G &#xD7; E of DA: days to anthesis, ASI: anthesis-silking interval, BCW: bare cob weight, and GWH: grain weight per hectare (top). D) Additional levels of adaptive flowering time are influenced by main effect as environmental GWAS alleles originating from lowlands flower later than their highland counterparts (and vice versa)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Adaptive variation from environmental GWAS. A) Manhattan plots showing associations with temperature, precipitation, isothermality, and the number of wet days in the growing season. B) SNPs identified via environmental association show strong &lt;em&gt;G&lt;/em&gt; &#xD7; &lt;em&gt;E&lt;/em&gt; effects on grain weight per hectare, increasing fitness in common gardens more similar to their environment of origin. C) Environmental GWAS hits across elevation predict beneficial &lt;em&gt;G&lt;/em&gt; &#xD7; &lt;em&gt;E&lt;/em&gt; of DA: days to anthesis, ASI: anthesis-silking interval, BCW: bare cob weight, and GWH: grain weight per hectare (top). D) Additional levels of adaptive flowering time are influenced by main effect as environmental GWAS alleles originating from lowlands flower later than their highland counterparts (and vice versa).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F3.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F3.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-17" class="first-child">Adaptive variation from environmental GWAS. A) Manhattan plots showing associations with temperature, precipitation, isothermality, and the number of wet days in the growing season. B) SNPs identified via environmental association show strong <em>G</em> × <em>E</em> effects on grain weight per hectare, increasing fitness in common gardens more similar to their environment of origin. C) Environmental GWAS hits across elevation predict beneficial <em>G</em> × <em>E</em> of DA: days to anthesis, ASI: anthesis-silking interval, BCW: bare cob weight, and GWH: grain weight per hectare (top). D) Additional levels of adaptive flowering time are influenced by main effect as environmental GWAS alleles originating from lowlands flower later than their highland counterparts (and vice versa).</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">To validate our eGWAS analyses, we tested whether alleles associated with annual precipitation would impact yield in a pair of independent drought trials including more than 1,000 of our genotyped accessions (see Methods). We find that alleles associated with low precipitation across the landscape are more likely to increase yield in drought conditions than either high precipitation alleles or alleles from randomly sampled frequency-matched loci (<em>P</em> &lt; 1e<sup>-7</sup>, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Supplemental Figure S3</a>. We combined information from this drought experiment with expression data of leaf and ovary response to drought (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>) in order to further refine candidate drought adaptation genes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>; a full list is in Supplemental Tables S8-S10). For example, 6 SNPs in Zm00001d003176 (Nodule inception protein-like protein 3; <em>nlp3</em>) are associated with precipitation across Mexico and 3 are associated with yield in our drought trials. Interrogation of the expression data shows that <em>nlp3</em> also shows significant gene expression differences between well watered and drought conditions (<em>P</em> = 7.3e<sup>-05</sup>). <em>nlp3</em> acts as a central signaling protein to control nitrate and phosphate responsive genes (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>), is responsive to multiple environmental stress conditions, and <em>nlp3</em> responses change carbon, nitrogen, and phosphorous metabolism (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>). Another candidate is Zm00001d002999 (textitga2ox2), which functions in giberellin catabolism. Consistent with our expression analysis (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), textitga2ox2 is upregulated in areas of active growth during drought stress (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>), leading to a reduction in gibberellic acid that slows elongation and in turn allows for resources to be allocated to other drought dependent processes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>). Zm00001d004621 (<em>ntr1</em>) is a peptide transporter and another clear for drought adaptation — when overexpressed in soybean it increased seedling adaptation to drought (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>). Finally, candidate gene Zm00001d046621 is <em>glossy15</em>, an apetala2 type transcription factor. Variation at textitglossy15 hastens or delays the transition from juvenile to adult stage (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>), which will influence drought avoidance ability as the transition from juvenile to adult stage is required for flowering in maize. Additionally, <em>glossy15</em> variation influences the composition and amount of epicuticular waxes on young maize leaves (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>) which can reduce drought tolerance in maize seedlings (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). Finally, we note that drought expression data further strengthens candidates identified previously — all three of the candidates highlighted as showing <em>G</em> × <em>E</em> for both phospholipid content and phenotypes additionally show differential expression under drought as well as cold conditions (<a id="xref-fig-13-2" class="xref-fig" href="#F13">Supplemental Fig. S9</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Environmental GWAS identifies candidate drought improvement genes. Candidate SNPs (columns) have low precipitation associated alleles that increase yield in drought trials (top row) and show differential expression under drought (middle row). Significant environmental GWAS (precipitation) p-values for each gene are highly localized and do not extend to nearby genes. Manhattan plots (bottom row) show the precision of environmental GWAS around each candidate SNP (broken red line) and 5KB upstream and downstream of each candidate. Each candidate has no other gene models present within the extent of the windows." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1310042272" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Environmental GWAS identifies candidate drought improvement genes. Candidate SNPs (columns) have low precipitation associated alleles that increase yield in drought trials (top row) and show differential expression under drought (middle row). Significant environmental GWAS (precipitation) p-values for each gene are highly localized and do not extend to nearby genes. Manhattan plots (bottom row) show the precision of environmental GWAS around each candidate SNP (broken red line) and 5KB upstream and downstream of each candidate. Each candidate has no other gene models present within the extent of the windows.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F4.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F4.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2019/07/18/706739/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <p id="p-19" class="first-child">Environmental GWAS identifies candidate drought improvement genes. Candidate SNPs (columns) have low precipitation associated alleles that increase yield in drought trials (top row) and show differential expression under drought (middle row). Significant environmental GWAS (precipitation) p-values for each gene are highly localized and do not extend to nearby genes. Manhattan plots (bottom row) show the precision of environmental GWAS around each candidate SNP (broken red line) and 5KB upstream and downstream of each candidate. Each candidate has no other gene models present within the extent of the windows.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">We also compared out results to published yield data from drought and heat stress trials in the “Drought Tolerant Maize of Africa” (DTMA) panel (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>). We compared our top 5000 eGWAS hits to <em>G</em> × <em>E</em> associations with yield across environments in the DTMA, finding that 142 of our eGWAS hits are associated with significant (<em>P</em> &lt; 1e<sup>-3</sup>) drought-specific yield effects in one of the three measured traits (Supplemental Tables S11-S13). At the end of chromosome 9, for example, is the strongest eGWAS hit for precipitation and wet days during the growing season (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>) as well as diurnal temperature range and isothermality S11. Although this association peak includes roughly 7 different genes, an annotated heat-shock-related transcription factor (hsftf9; Zm00001d048041) is a compelling candidate, as its <em>Arabidopsis</em> ortholog (AtHSF1) is expressed under heat stress and is responsible for activation of heat-shock-protein-based thermotolerance (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>), while its tomato homolog HSFA1 acts as a master regulator of thermotolerance (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>). Indeed, a SNP in the 5’ UTR of hsftf9 showed a strong G × E association for yield between drought and well watered treatments (p=0.00012) in the DTMA drought panel. Another candidate gene identified from our eGWAS (Zm00001d007709; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>) is from a family of proteins containing In Between Ring (IBR) domains, and likely functions in protein degradation through protein-protein interaction. Little is known about the function of this gene in maize or of its most similar relative (ATARI7) in <em>Arabidopsis</em>. But despite the lack of functional information around this gene, the SNPs tested within the coding sequence of this gene show at least one significant (<em>P</em> &lt; 0.01) hit for all three traits (DA, ASI, and Yield) tested in the DTMA drought trials.</p><p id="p-21">To highlight a combined set of candidates for characterization, we complemented our <em>G</em> × <em>E</em> and environmental associations with a phenotype- and environment-independent scan for SNPs showing extreme patterns of allele frequency differentiation across the landscape (see Methods). We focused our assessment of gene function on the set of 39 genes (Supplemental Table S14) identified as candidates in all three approaches (<em>G</em> × <em>E</em>, eGWAS, and PCAdapt), many of which have functions that consistent with adaption to abiotic and biotic stress. Zm00001d046909 (homogentisate phytyltransferase <em>hpt1</em>), for example, is active in Vitamin E biosynthesis (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>), known to play a role in tolerance to salinity, drought, cold temperature, metal toxicity, ozone, and UV radiation stress (reviewed in (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>)). SNPs in Zm00001d046909 are associated with daily temperature variability during the growing season, and show <em>G</em> × <em>E</em> effects for ear mass and temperature, precipitation, and elevation. The candidates Zm00001d024909 (CCT domain transcription factor) and Zm00001d010752 (<em>pebp8, zcn8</em>) are known to be involved in photoperiod sensitivity, flowering time, and long-day stress response (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>). The former shows environmental association with mean temperature and is a candidate in our <em>G</em> × <em>E</em> analysis between ear mass and temperature, precipitation, and elevation, and the latter is a <em>G</em> × <em>E</em> candidate for flowering time and temperature as well as annual precipitation and wet days). Finally, we identify Zm00001d045972 (<em>umc81</em>) as a candidate for <em>G</em> × <em>E</em> between yield and elevation, and as showing allele frequency associations with temperature and water vapor pressure, but it has been previously highlighted as a candidate for days to anthesis (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>), plant height (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>), drought stress tolerance (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">67</a>), and root hair length plasticity (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">68</a>).</p></div><div class="section" id="sec-3"><h2 class="">Conclusions</h2><p id="p-22">Developing diverse crops that can deliver increased yields in the face of a changing global climate is one of the greatest challenges facing modern agricultural research. Understanding and characterizing adaptive loci will allow for the incorporation of novel variation into breeding efforts to adapt crops to changing climates. Although <em>in situ</em> breeding remains an important source of adaptive variation (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">6</a>), our work highlights the utility of leveraging <em>ex situ</em> germplasm collections to evaluate phenotype across a diverse set of environments. Such collections typically have location information which can further increase the value of genomic data by information about the environments to which taxa have been locally adapted (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">69</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">15</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">70</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">14</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>). We use these data to identify loci underlying phenotypic and fitness differences in maize across environments and demonstrate that our environment-specific effect estimates successfully predict where alleles are found across the landscape. We then show that approaching the problem from the other direction — identifying loci that associate with environmental variables — can precisely delineate individual genes that show predictable phenotypic effects in field trials. In sum, we have validated the utility of phenotypic and environmental association approaches for identifying relevant adaptive variation and provided a high-resolution list of hundreds of candidate genes showing environmental and phenotypic associations that can be directly tested in targeted experiments (e.g. CRISPR-mediated knockout 71; 72) to improve climate adaptation in modern breeding populations.</p></div><div class="section" id="sec-4"><h2 class="">Methods</h2><div id="sec-5" class="subsection"><h3>Samples and Genotyping</h3><p id="p-23">The Seeds of Discovery project (SeeD) has genotypically characterized the CIMMYT International Germplasm Bank maize collection containing over 24,000 landrace accessions primarily from North, Central, and South America. Here, we used data generated across a core collection of ≈ 4, 500 individual accessions from this project representing the breadth of environmental variation across the Americas as described in (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">30</a>). Briefly, each individual was sequenced using genotype-by-sequencing (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">73</a>) to a median 2X coverage. Genotypes were called using TASSEL (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">74</a>), resulting in an initial set of 955,120 SNPs reported in (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">30</a>). In addition to these data, we used GBS genotypes from the wild teosintes <em>Zea mays</em> ssp. <em>parviglumis</em> and <em>Zea mays</em> ssp. <em>mexicana</em> from (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">75</a>). All genotype data were uplifted to coordinates on the B73 v4 reference genome (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">76</a>) using the crossmap liftover tool (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">77</a>).</p></div><div id="sec-6" class="subsection"><h3>Multi-Environment Estimation of Phenotypes</h3><p id="p-24">As part of the SeeD evaluation of the landrace core collection, offspring of each of the genotyped individuals were planted in multiple environments under a replicated F1 crossing design (see 30, for design details). Two important features of the crossing experiment were added to ensure that phenotype data was not overly biased by elevational adaptation. First, plants were preferentially grown in locations that were of similar adaptation (highland tropical, sub-tropical or lowland tropical) to their home environment. While F1s with a highland accession parent were grown in low elevation and vice versa, on average, more plants from highland accessions were grown in high elevation than low elevation trials (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure S2</a>). Second, each plant was crossed to a test plant that was adapted to the environment that the F1 seeds were grown in. Both of these design features facilitates comparison of a larger sample but reduces adaptive differences among landraces and make our estimates of adaptive differences more conservative. The resulting F1 offspring were then grown in replicate in ambient field conditions across a total of 23 trials in 13 locations in 2 years. Trials used in the study are detailed in Supplemental Table S15, data is available from <a href="http://hdl.handle.net/11529/10548233">http://hdl.handle.net/11529/10548233</a>.</p><p id="p-25">Although the collections are from across the Americas, the overwhelming majority were sampled from the center of maize diversity in Mexico. Because these samples had the best climate data, and to minimize complications arising from large differences in plant environment of origin, trial environments, and population structure (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">31</a>), we restricted our analysis of field phenotype data to the 2433 landrace accessions of Mexican origin. A range of 9-26 (average of 16) offspring per landrace cross were measured in an augmented row-column design to provide additional control for random stochastic variation; <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure S2</a> shows the number of accessions with phenotypes across the elevational range of the trials.</p><p id="p-26">Across the 13 sites plants were phenotyped for a variety of traits. We used data from plant height (PlH), field weight (FW), bare cob weight (BCW), and moisture adjusted grain weight per hectare (GWH). In addition to these phenotypes, we added previously published data on flowering traits (days to anthesis (DA), days to silking (DS), and anthesis-silking interval (ASI)) from these same trials (<a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">30</a>).</p></div><div id="sec-7" class="subsection"><h3>Phenotypic Analysis</h3><p id="p-27">Breeding values for each landrace were estimated as described in Navarro <em>et. al</em>. (<a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">30</a>), controlling for tester, checks, and field position in a complete nested model. We then removed any plots that had a reported disease score and also removed trials and accessions with outlying phenotype scores (typically &gt; 4 standard deviations from the mean). Overall, there were 23 distinct trials between the 13 sites over 2 years, each with breeding values calculated for an average of approximately 850 accessions. The GxE phenotype datasets that we used for calculating GxE outliers ranged from 11,762 to 4,851 measurements depending on what trait was measured 1.</p><p id="p-28">To test for associations between markers and responses to environmental variation across the trials (i.e. GxE effects) for each phenotype, we used the following mixed-effects GWAS model for each marker <em>i</em> = 1…<em>p</em>:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/18/706739/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/18/706739/embed/graphic-5.gif"/></noscript></span>

</span></p><p id="p-29">Here, <strong>y</strong> is a vector of <em>n</em> phenotypes (breeding values) for each landrace across all trials that it was tested in; <strong>X</strong> is a <em>n</em> × 8 design matrix specifying fixed effects for year (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2</a>) and testers (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>) with coefficient vector <em>α</em>; <strong>e</strong> is a vector representing the environmental value, i.e. the elevation, mean temperature or precipitation of the trial, with mean effect <em>β</em><sub>0</sub>; <strong>x<sub>i</sub></strong> is a vector or genotypes at marker <em>i</em> for each observation and <em>β</em><sub>1</sub> and <em>β</em><sub>2</sub> are marker effects on the intercept and response to the environment, respectively; <strong>Z</strong> is a <em>n</em> × <em>r</em> design matrix for landraces, and <strong>u<sub>m</sub>, g<sub>m</sub>, u<sub>e</sub></strong>, and <strong>u<sub>e</sub></strong> are random effects accounting for the expected correlations of intercepts (<em>m</em>) and responses to the environment (<em>e</em>) among observations due to shared background additive (<em>u</em>) and non-additive (<em>g</em>) genetic variation for the multiple observations of the same landrace across different trials; and <strong>e</strong> is a vector of the environmental variable (i.e. mean phenotype value of the trial). The random effects have distributions:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/18/706739/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/18/706739/embed/graphic-6.gif"/></noscript></span>

</span>
where <strong>K</strong> is the <em>r</em> × <em>r</em> kinship matrix for the <em>r</em> ≈ 2200 landraces computed in Tassel 5 (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">74</a>) using a set of ≈ 6,500 unimputed loci that contain over 95% complete genotype data, <strong>I<sub>r</sub></strong> is the <em>r</em> × <em>r</em> identity matrix and <strong>I<sub>n</sub></strong> is the <em>n</em> × <em>n</em> identity matrix.</p><p id="p-30">We used the <kbd>GridLMM_GWAS</kbd> function (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>) to perform a GWAS scan across all markers, re-estimating the four variance components by REML separately at each marker (<kbd>method = “REML”</kbd>) using GridLMM-fast heuristic (<kbd>algorithm = “Fast”</kbd>) with <kbd>h2_step = 0.1</kbd>, and tested the significance of the marker main effects <em>β</em><sub>1</sub> and plasticity effects <em>β</em><sub>2</sub> independent by Wald tests.</p></div><div id="sec-8" class="subsection"><h3>Environmental associations</h3><p id="p-31">In order to identify loci important for environmental adaptation, we performed an additional association analysis between allele frequency and environmental variables across the landscape. Here, to take advantages of a wider array of environmental variation, we expanded our sampling of individuals to all 4,127 genotyped accessions with latitude and longitude coordinates, including samples collected outside of Mexico. For general environmental layers (such as elevation, mean annual temperature, diurnal temperature range, mean isothermality, mean annual precipitation, and precipitation seasonality) we used latitude and longitude coordinates to infer values using the WorldClim 2 (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">41</a>) layers at 0.5 minute resolution (<a href="https://data.cimmyt.org/dataset.xhtml?persistentId=hdl:11529/10548079">https://data.cimmyt.org/dataset.xhtml?persistentId=hdl:11529/10548079</a>) based on longterm averages for 1970-2000 for the climatic parameters as well as elevation data at 90 m resolution from the global elevation model from NASA and CIAT (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">78</a>). In addition to these broader climate measurements, we included detailed weather station measurement based gridded datasets of the frequency of wet and frost-free days, cloud cover, vapor pressure covering the period 1950-2012 (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">79</a>) and a longterm aridity index (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">80</a>), each averaged over the approximate six month local growing season of accessions. Additional data sets representing soil pH (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">81</a>), and soil waterlogging percentage (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">82</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">83</a>) were used to extract values to represent soil-based abiotic stresses in the environments where accessions originated. Due to differences in collection specificity, some individuals in the panel have unverified latitude longitude coordinates, and we restrict our detailed climate analyses to a curated subset of individuals (roughly 2,900 individuals) with verified location data.</p><p id="p-32">For each different environmental layer we fit a model to measure the association between SNP genotype <strong>x<sub>i</sub></strong> and the given environmental values for each accession <strong>p</strong>, correcting for population structure using the first five principal components <strong>Q</strong>:
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/18/706739/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2019/07/18/706739/embed/graphic-7.gif"/></noscript></span>

</span></p><p id="p-33">We then tested the significance of <em>β</em> with an F-test. In this analysis we only use loci with a minor allele frequency greater than 10%.</p><p id="p-34">To determine if environmentally associated loci are contributing to adaption we test whether outlier loci contribute more to adaptive phenotypic differences in our experimental trials. We first selected 1,000 SNPs with the strongest association (lowest p-value) with elevation and a set of randomly selected SNPs that match the frequency deciles of associated SNPs. For each SNP in both sets, we used our field evaluation to estimate the standardized genetic value (in z-scores) of the main effect and the <em>G</em> × <em>E</em> term and tested if environmentally associated loci had a larger phenotypic effect than randomly matched loci. To ameliorate issue with sampling nonindependent SNPs, we excluded SNPs inside the large inversion <em>Inv4m</em> from this analysis.</p></div><div id="sec-9" class="subsection"><h3>Gene Identification</h3><p id="p-35">To identify genes in our GWAS signal, we first smoothed the per-SNP association p-values across each chromosome using a spline analysis implemented in the R package GenWin (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">84</a>). We used the maize reference B73 v4 annotation (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">76</a>) to define gene positions and used the ‘countoverlaps’ function from the GenomicRanges R package (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">85</a>) to determine the number and identity of genes that overlap with outlier SNP windows.</p></div><div id="sec-10" class="subsection"><h3>Drought Trials</h3><p id="p-36">We also use drought trials to validate adaptive drought tolerance loci. The first drought experiment dataset we use was generated as part of the SeeDs of discovery phenotyping data discussed earlier and consists of 1,057 different accessions grown in drought and well watered trials in the Iguala and Obregon localities. These accessions were phenotyped for dry grain weight in drought and well-watered conditions at both locations, for a total of 2,047 yield phenotypes. We used GridLMM to fit a model with tester and location as covariates and measured the strength of the <em>G</em> × <em>E</em> yield response as above. In order to determine the effect of environmental GWAS loci on phenotypes in drought we ran an association analysis on the top 1,000 environmental association hits from annual precipitation and 1,000 randomly chosen loci matched for allele frequency. As above, we tested whether alleles associated with low precipitation would increase yield in drought trials in comparison to the alternate high precipitation alleles by making a prediction of the low and high precipitation genotypes from the model for both the environmentally associated and randomly paired loci.</p></div><div id="sec-11" class="subsection"><h3>Drought Tolerant Maize in Africa</h3><p id="p-37">Three hundred maize lines from CIMMYT IITA (the International Institute of Tropical Agriculture) were previously genotyped and tested for drought tolerance in yield trials of well watered, drought, high heat, or combined conditions in the Drought Tolerant Maize for Africa (DTMA) GWAS panel (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">55</a>). We used GBS genotypes for the DTMA from (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">86</a>), allowing us to test associations between individual genes and phenotypes. We used GridLMM (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">26</a>) to fit a <em>G</em> × <em>E</em> model analogous to that used previously in our phenotypic trials. In this model we fit grain weight per hectare of the as explained by genotype, the treatment the phenotype was collected in, and the interaction term (the change in yield across different trials).</p></div><div id="sec-12" class="subsection"><h3>Expression analyses</h3><p id="p-38">To test for regulatory variation in candidate drought genes we downloaded raw RNA-seq data from (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">44</a>). This experiment consists of replicated individuals of the inbred line B73 grown in drought and well watered conditions, with expression measured in the meristem of the three youngest leaves and ovary tissue at 1 day after pollination. We used kallisto (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">87</a>) to estimate expression based upon kmer abundance of RNA-seq data at the gene models from the maize v4 genome. We used sleuth (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">88</a>) to normalize and fit a factorial model to test the strength of the effect of growth condition and the growth condition by tissue interaction effect compared to a null model with only the main effect of tissue specific differences.</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">↵</a><span class="fn-label">*</span> <span class="em-link"><span class="em-addr">dangates{at}ucdavis.edu</span></span>; <span class="em-link"><span class="em-addr">rossibarra{at}ucdavis.edu</span></span></p></li></ul></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.1"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hulme</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-article-title">1.5 c and climate research after the paris agreement</span>. <abbr class="cit-jnl-abbrev">Nature Climate Change</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">222</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BClimate%2BChange%26rft.volume%253D6%26rft.spage%253D222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-706739v1.2"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Team</span>,  <span class="cit-name-given-names">C. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pachauri</span>,  <span class="cit-name-given-names">R. K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Meyer</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-chapter-title">Ipcc, 2014: climate change 2014: synthesis report. contribution of working groups i</span>. <span class="cit-source">II and III to the Fifth Assessment Report of the intergovernmental panel on Climate Change</span>. <span class="cit-publ-name">IPCC</span>, <span class="cit-publ-loc">Geneva, Switzerland</span> <span class="cit-fpage">151</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.3" data-doi="10.1038/nature16467"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lesk</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rowhani</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ramankutty</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Influence of extreme weather disasters on global crop production</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">529</span>, <span class="cit-fpage">84</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D529%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature16467%26rft_id%253Dinfo%253Apmid%252F26738594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature16467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26738594&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Challinor</span>,  <span class="cit-name-given-names">A. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A meta-analysis of crop yield under climate change and adaptation</span>. <abbr class="cit-jnl-abbrev">Nature Climate Change</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">287</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BClimate%2BChange%26rft.volume%253D4%26rft.spage%253D287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Varshney</span>,  <span class="cit-name-given-names">R. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">V. K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Powell</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sorrells</span>,  <span class="cit-name-given-names">M. E.</span></span> <span class="cit-article-title">Can genomics deliver climate-change ready crops?</span> <abbr class="cit-jnl-abbrev">Current opinion in plant biology</abbr> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.6" data-doi="10.1098/rspb.2018.1049"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bellon</span>,  <span class="cit-name-given-names">M. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Evolutionary and food supply implications of ongoing maize domestication by mexican campesinos</span>. <abbr class="cit-jnl-abbrev">Proc. R. Soc. B</abbr> <span class="cit-vol">285</span>, <span class="cit-fpage">20181049</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BB%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2018.1049%26rft_id%253Dinfo%253Apmid%252F30158306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2018.1049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30158306&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.7" data-doi="10.2135/cropsci1995.0011183X003500020009x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brush</span>,  <span class="cit-name-given-names">S. B.</span></span> <span class="cit-article-title">In situ conservation of landraces in centers of crop diversity</span>. <abbr class="cit-jnl-abbrev">Crop science</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">346</span>–<span class="cit-lpage">354</span> (<span class="cit-pub-date">1995</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBrush%26rft.auinit1%253DS.%2BB.%26rft.volume%253D35%26rft.issue%253D2%26rft.spage%253D346%26rft.epage%253D354%26rft.atitle%253DIn%2BSitu%2BConservation%2Bof%2BLandraces%2Bin%2BCenters%2Bof%2BCrop%2BDiversity%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1995.0011183X003500020009x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1995.0011183X003500020009x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995QP34600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.8" data-doi="10.1371/journal.pbio.0020347"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">McCouch</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Diversifying selection in plant breeding</span>. <abbr class="cit-jnl-abbrev">PLoS biology</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">e347</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DMcCouch%26rft.auinit1%253DS.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253De347%26rft.epage%253De347%26rft.atitle%253DDiversifying%2Bselection%2Bin%2Bplant%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020347%26rft_id%253Dinfo%253Apmid%252F15486582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15486582&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.9" data-doi="10.1007/BF00305828"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tanksley</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-article-title">Rflp analysis of the size of chromosomal segments retained around the tm-2 locus of tomato during backcross breeding</span>. <abbr class="cit-jnl-abbrev">Theoretical and Applied Genetics</abbr> <span class="cit-vol">77</span>, <span class="cit-fpage">353</span>–<span class="cit-lpage">359</span> (<span class="cit-pub-date">1989</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Band%2BApplied%2BGenetics%26rft.volume%253D77%26rft.spage%253D353%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00305828%26rft_id%253Dinfo%253Apmid%252F24232612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00305828&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24232612&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T895900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-706739v1.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chilcoat</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Z.-B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sander</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-chapter-title">Use of crispr/cas9 for crop improvement in maize and soybean</span>. In <span class="cit-source">Progress in molecular biology and translational science</span>, vol. <span class="cit-vol">149</span>, <span class="cit-fpage">27</span>–<span class="cit-lpage">46</span> (<span class="cit-publ-name">Elsevier</span>, <span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2Bmolecular%2Bbiology%2Band%2Btranslational%2Bscience%26rft.volume%253D149%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rasheed</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hickey</span>,  <span class="cit-name-given-names">L. T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">He</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">Fast-forwarding genetic gain</span>. <abbr class="cit-jnl-abbrev">Trends in plant science</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">184</span>–<span class="cit-lpage">186</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.volume%253D23%26rft.spage%253D184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.12"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wambugu</span>,  <span class="cit-name-given-names">P. W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ndjiondjop</span>,  <span class="cit-name-given-names">M.-N.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Henry</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-article-title">Role of genomics in promoting the utilization of plant genetic resources in genebanks</span>. <abbr class="cit-jnl-abbrev">Briefings in functional genomics</abbr> <span class="cit-vol">17</span>, <span class="cit-fpage">198</span>–<span class="cit-lpage">206</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBriefings%2Bin%2Bfunctional%2Bgenomics%26rft.volume%253D17%26rft.spage%253D198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mercer</span>,  <span class="cit-name-given-names">K. L.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Perales</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-article-title">Structure of local adaptation across the landscape: flowering time and fitness in mexican maize (zea mays l. subsp. mays) landraces</span>. <abbr class="cit-jnl-abbrev">Genetic Resources and Crop Evolution</abbr> <span class="cit-vol">66</span>, <span class="cit-fpage">27</span>–<span class="cit-lpage">45</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2BResources%2Band%2BCrop%2BEvolution%26rft.volume%253D66%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.14"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lei</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Discovery of barley gene candidates for low temperature and drought tolerance via environmental association</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">405399</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.15" data-doi="10.1126/sciadv.1400218"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lasky</span>,  <span class="cit-name-given-names">J. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome-environment associations in sorghum landraces predict adaptive traits</span>. <abbr class="cit-jnl-abbrev">Science Advances</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">e1400218</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DLasky%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D1%26rft.issue%253D6%26rft.spage%253De1400218%26rft.epage%253De1400218%26rft.atitle%253DGenome-environment%2Bassociations%2Bin%2Bsorghum%2Blandraces%2Bpredict%2Badaptive%2Btraits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1400218%26rft_id%253Dinfo%253Apmid%252F26601206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMS82L2UxNDAwMjE4IjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDcvMTgvNzA2NzM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.16"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">J. E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kono</span>,  <span class="cit-name-given-names">T. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stupar</span>,  <span class="cit-name-given-names">R. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kantar</span>,  <span class="cit-name-given-names">M. B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Morrell</span>,  <span class="cit-name-given-names">P. L.</span></span> <span class="cit-article-title">Environmental association analyses identify candidates for abiotic stress tolerance in glycine soja, the wild progenitor of cultivated soybeans</span>. <abbr class="cit-jnl-abbrev">G3: Genes, Genomes, Genetics</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">835</span>–<span class="cit-lpage">843</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%253A%2BGenes%252C%2BGenomes%252C%2BGenetics%26rft.volume%253D6%26rft.spage%253D835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.17"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bandillo</span>,  <span class="cit-name-given-names">N. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dissecting the genetic basis of local adaptation in soybean</span>. <abbr class="cit-jnl-abbrev">Scientific reports</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">17195</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D7%26rft.spage%253D17195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.18"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoder</span>,  <span class="cit-name-given-names">J. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic signature of adaptation to climate in medicago truncatula</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> genetics–<span class="cit-fpage">113</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.19" data-doi="10.7554/eLife.05861"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hake</span>,  <span class="cit-name-given-names">S.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">The natural history of model organisms: genetic, evolutionary and plant breeding insights from the domestication of maize</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">4</span>, <span class="cit-fpage">e05861</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.stitle%253Delife%26rft.volume%253D4%26rft.issue%253D0%26rft.spage%253De05861%26rft.epage%253De05861%26rft.atitle%253DGenetic%252C%2Bevolutionary%2Band%2Bplant%2Bbreeding%2Binsights%2Bfrom%2Bthe%2Bdomestication%2Bof%2Bmaize%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.05861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.05861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.20" data-doi="10.1073/pnas.1222463110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rosenzweig</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Assessing agricultural risks of climate change in the 21st century in a global gridded crop model intercomparison</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">111</span>, <span class="cit-fpage">3268</span>–<span class="cit-lpage">3273</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1222463110%26rft_id%253Dinfo%253Apmid%252F24344314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTExLzkvMzI2OCI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA3LzE4LzcwNjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.21"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Eagles</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lothrop</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-article-title">Highland maize from central mexico—its origin, characteristics, and use in breeding programs</span>. <abbr class="cit-jnl-abbrev">Crop Science</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">11</span>–<span class="cit-lpage">19</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DEagles%26rft.auinit1%253DH.%2BA.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D19%26rft.atitle%253DHighland%2BMaize%2Bfrom%2BCentral%2BMexico%2BIts%2BOrigin%252C%2BCharacteristics%252C%2Band%2BUse%2Bin%2BBreeding%2BPrograms%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1994MV45600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.22"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Expanding maize genetic resources with predomestication alleles: Maize–teosinte introgression populations</span>. <abbr class="cit-jnl-abbrev">The plant genome</abbr> <span class="cit-vol">9</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.23"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Prasanna</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-article-title">Diversity in global maize germplasm: characterization and utilization</span>. <abbr class="cit-jnl-abbrev">Journal of biosciences</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">843</span>–<span class="cit-lpage">855</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiosciences%26rft.stitle%253DJ%2BBiosci%26rft.aulast%253DPrasanna%26rft.auinit1%253DB.%2BM.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D843%26rft.epage%253D855%26rft.atitle%253DDiversity%2Bin%2Bglobal%2Bmaize%2Bgermplasm%253A%2BCharacterization%2Band%2Butilization.%26rft_id%253Dinfo%253Apmid%252F23107920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23107920&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lafitte</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Edmeades</span>,  <span class="cit-name-given-names">G.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Temperature responses of tropical maize cultivars selected for broad adaptation</span>. <abbr class="cit-jnl-abbrev">Field Crops Research</abbr> <span class="cit-vol">49</span>, <span class="cit-fpage">215</span>–<span class="cit-lpage">229</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BResearch%26rft.volume%253D49%26rft.spage%253D215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.25"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lafitte</span>,  <span class="cit-name-given-names">H.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Edmeades</span>,  <span class="cit-name-given-names">G.</span></span> <span class="cit-article-title">Temperature effects on radiation use and biomass partitioning in diverse tropical maize cultivars</span>. <abbr class="cit-jnl-abbrev">Field Crops Research</abbr> <span class="cit-vol">49</span>, <span class="cit-fpage">231</span>–<span class="cit-lpage">247</span> (<span class="cit-pub-date">1997</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BResearch%26rft.volume%253D49%26rft.spage%253D231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.26"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Runcie</span>,  <span class="cit-name-given-names">D. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Crawford</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Fast and flexible linear mixed models for genome-wide genetics</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">373902</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.27" data-doi="10.1534/genetics.108.097238"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Gaut</span>,  <span class="cit-name-given-names">B. S.</span></span> <span class="cit-article-title">Historical divergence and gene flow in the genus zea</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>, <span class="cit-fpage">1399</span>–<span class="cit-lpage">1413</span> (<span class="cit-pub-date">2009</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.097238%26rft_id%253Dinfo%253Apmid%252F19153259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS80LzEzOTkiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xOC83MDY3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.28" data-doi="10.1371/journal.pone.0047659"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hufford</span>,  <span class="cit-name-given-names">M. B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez-Meyer</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gaut</span>,  <span class="cit-name-given-names">B. S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eguiarte</span>,  <span class="cit-name-given-names">L. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Tenaillon</span>,  <span class="cit-name-given-names">M. I.</span></span> <span class="cit-article-title">Inferences from the historical distribution of wild and domesticated maize provide ecological and evolutionary insight</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">e47659</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De47659%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047659%26rft_id%253Dinfo%253Apmid%252F23155371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23155371&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.29"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hufford</span>,  <span class="cit-name-given-names">M. B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The genomic signature of crop-wild introgression in maize</span>. <abbr class="cit-jnl-abbrev">PLoS Genetics</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">e1003477</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D9%26rft.spage%253D1003477e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.30" data-doi="10.1038/ng.3784"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Navarro</span>,  <span class="cit-name-given-names">J. A. R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A study of allelic diversity underlying flowering-time adaptation in maize landraces</span>. <abbr class="cit-jnl-abbrev">Nature genetics</abbr> <span class="cit-vol">49</span>, <span class="cit-fpage">476</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.volume%253D49%26rft.spage%253D476%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3784%26rft_id%253Dinfo%253Apmid%252F28166212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3784&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28166212&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.31"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Takuno</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Independent molecular basis of convergent highland adaptation in maize</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> genetics-<span class="cit-fpage">115</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.32" data-doi="10.1111/tpj.12007"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Degenkolbe</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Differential remodeling of the lipidome during cold acclimation in natural accessions of arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">The Plant Journal</abbr> <span class="cit-vol">72</span>, <span class="cit-fpage">972</span>–<span class="cit-lpage">982</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BJournal%26rft.volume%253D72%26rft.spage%253D972%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12007%26rft_id%253Dinfo%253Apmid%252F23061922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23061922&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000314182000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.33" data-doi="10.1074/jbc.M205375200"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Welti</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Profiling membrane lipids in plant stress responses: Role of phospholipase d<em>α</em> in freezing-induced lipid changes in <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Journal of Biological Chemistry</abbr> <span class="cit-vol">277</span>, <span class="cit-fpage">31994</span>–<span class="cit-lpage">32002</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M205375200%26rft_id%253Dinfo%253Apmid%252F12077151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzcvMzUvMzE5OTQiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xOC83MDY3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.34"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kisko</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">controls phosphate homeostasis in a zinc-dependent manner</span>. <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">7</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.35"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Zapata</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sawers</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Rellán-Álvarez</span>,  <span class="cit-name-given-names">R.</span></span> <abbr class="cit-jnl-abbrev">SoilP: A package to extract phosphorus retention data from soil maps and study local adaptation to phosphorus availability</abbr> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[36].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.36" data-doi="10.1104/pp.114.248930"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kuppusamy</span>,  <span class="cit-name-given-names">T.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Lipid biosynthesis and protein concentration respond uniquely to phosphate supply during leaf development in highly phosphorus-efficient Hakea prostrata</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">166</span>, <span class="cit-fpage">1891</span>–<span class="cit-lpage">1911</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.114.248930%26rft_id%253Dinfo%253Apmid%252F25315604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTY2LzQvMTg5MSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA3LzE4LzcwNjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Borrego</span>,  <span class="cit-name-given-names">E. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Kolomiets</span>,  <span class="cit-name-given-names">M. V.</span></span> <span class="cit-article-title">Synthesis and functions of jasmonates in maize</span>. <abbr class="cit-jnl-abbrev">Plants</abbr> <span class="cit-vol">5</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Brotman</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Méret</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melchinger</span>,  <span class="cit-name-given-names">A. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Willmitzer</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">The maize leaf lipidome shows multilevel genetic control and high predictive value for agronomic traits</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">2479</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D3%26rft.spage%253D2479%26rft_id%253Dinfo%253Apmid%252F23963398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23963398&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-web" id="cit-706739v1.39"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Zapata</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">G x E QTL mapping of a PT x B73 biparental population</span> (<span class="cit-pub-date">2019</span>). URL <a href="https://github.com/sawers-rellan-labs/PTxB73_lipid_QTL_GxE">https://github.com/sawers-rellan-labs/PTxB73_lipid_QTL_GxE</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[40].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.40" data-doi="10.1111/tpj.13414"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Waters</span>,  <span class="cit-name-given-names">M. I. N. J. B. L. H. C. H. C.</span></span>, A. &amp; <span class="cit-auth"><span class="cit-name-surname">Springer</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Natural variation for gene expression responses to abiotic stress in maize</span>. <abbr class="cit-jnl-abbrev">The Plant Journal</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">706</span>–<span class="cit-lpage">717</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BJournal%26rft.volume%253D89%26rft.spage%253D706%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.41"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Fick</span>,  <span class="cit-name-given-names">S. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Hijmans</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-article-title">Worldclim 2: new 1-km spatial resolution climate surfaces for global land areas</span>. <abbr class="cit-jnl-abbrev">International Journal of Climatology</abbr> <span class="cit-vol">37</span>, <span class="cit-fpage">4302</span>–<span class="cit-lpage">4315</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BClimatology%26rft.volume%253D37%26rft.spage%253D4302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[42].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-706739v1.42"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mercer</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Martínez-Vásquez</span>,  <span class="cit-name-given-names">Á.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Perales</span>,  <span class="cit-name-given-names">H. R.</span></span> <span class="cit-article-title">Asymmetrical local adaptation of maize landraces along an altitudinal gradient</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <span class="cit-vol">1</span>, <span class="cit-fpage">489</span>–<span class="cit-lpage">500</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D1%26rft.spage%253D489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[43].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-706739v1.43" data-doi="10.1073/pnas.1203189109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hung</span>,  <span class="cit-name-given-names">H.-Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Zmcct and the genetic basis of day-length adaptation underlying the postdomestication spread of maize</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">109</span>, <span class="cit-fpage">E1913</span>–<span class="cit-lpage">E1921</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1203189109%26rft_id%253Dinfo%253Apmid%252F22711828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzI4L0UxOTEzIjtzOjQ6ImF0b20iO3M6Mzc6Ii9iaW9yeGl2L2Vhcmx5LzIwMTkvMDcvMTgvNzA2NzM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.44"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kakumanu</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Effects of drought on gene expression in maize reproductive and leaf meristem tissue revealed by rna-seq</span>. <abbr class="cit-jnl-abbrev">Plant Physiology</abbr> pp–<span class="cit-fpage">112</span> (<span class="cit-pub-date">2012</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hu</span>,  <span class="cit-name-given-names">B.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Nitrate-nrt1. 1b–spx4 cascade integrates nitrogen and phosphorus signalling networks in plants</span>. <abbr class="cit-jnl-abbrev">Nature plants</abbr> <span class="cit-vol">1</span> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[46].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.46" data-doi="10.1016/j.tplants.2018.02.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Dröge-Laser</span>,  <span class="cit-name-given-names">W.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Weiste</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">The c/s1 bzip network: a regulatory hub orchestrating plant energy homeostasis</span>. <abbr class="cit-jnl-abbrev">Trends in plant science</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">422</span>–<span class="cit-lpage">433</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.volume%253D23%26rft.spage%253D422%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2018.02.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2018.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[47].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.47"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Opitz</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Extensive tissue-specific transcriptomic plasticity in maize primary roots upon water deficit</span>. <abbr class="cit-jnl-abbrev">Journal of Experimental Botany</abbr> <span class="cit-vol">67</span>, <span class="cit-fpage">1095</span>–<span class="cit-lpage">1107</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BBotany%26rft.volume%253D67%26rft.spage%253D1095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[48].</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.48"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Z.</span></span> <span class="cit-article-title">Auxin and ga signaling play important roles in the maize response to phosphate deficiency</span>. <abbr class="cit-jnl-abbrev">Plant Science</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[49].</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.49"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xue</span>,  <span class="cit-name-given-names">R.-G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">B.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Xie</span>,  <span class="cit-name-given-names">H.-F.</span></span> <span class="cit-article-title">Overexpression of a ntr1 in transgenic soybean confers tolerance to water stress</span>. <abbr class="cit-jnl-abbrev">Plant cell, tissue and organ culture</abbr> <span class="cit-vol">89</span>, <span class="cit-fpage">177</span>–<span class="cit-lpage">183</span> (<span class="cit-pub-date">2007</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bcell%252C%2Btissue%2Band%2Borgan%2Bculture%26rft.volume%253D89%26rft.spage%253D177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[50].</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.50" data-doi="10.1105/tpc.6.10.1343"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moose</span>,  <span class="cit-name-given-names">S. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sisco</span>,  <span class="cit-name-given-names">P. H.</span></span> <span class="cit-article-title">Glossy15 controls the epidermal juvenile-to-adult phase transition in maize</span>. <abbr class="cit-jnl-abbrev">The Plant Cell</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">1343</span>–<span class="cit-lpage">1355</span> (<span class="cit-pub-date">1994</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DMoose%26rft.auinit1%253DS.%2BP.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253D1343%26rft.epage%253D1355%26rft.atitle%253DGlossy15%2BControls%2Bthe%2BEpidermal%2BJuvenile-to-Adult%2BPhase%2BTransition%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.6.10.1343%26rft_id%253Dinfo%253Apmid%252F12244224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjYvMTAvMTM0MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA3LzE4LzcwNjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[51].</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.51" data-doi="10.1101/gad.10.23.3018"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moose</span>,  <span class="cit-name-given-names">S. P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sisco</span>,  <span class="cit-name-given-names">P. H.</span></span> <span class="cit-article-title">Glossy15, an apetala2-like gene from maize that regulates leaf epidermal cell identity</span>. <abbr class="cit-jnl-abbrev">Genes &amp; development</abbr> <span class="cit-vol">10</span>, <span class="cit-fpage">3018</span>–<span class="cit-lpage">3027</span> (<span class="cit-pub-date">1996</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMoose%26rft.auinit1%253DS%2BP%26rft.volume%253D10%26rft.issue%253D23%26rft.spage%253D3018%26rft.epage%253D3027%26rft.atitle%253DGlossy15%252C%2Ban%2BAPETALA2-like%2Bgene%2Bfrom%2Bmaize%2Bthat%2Bregulates%2Bleaf%2Bepidermal%2Bcell%2Bidentity.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.23.3018%26rft_id%253Dinfo%253Apmid%252F8957002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEwLzIzLzMwMTgiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xOC83MDY3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[52].</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.52" data-doi="10.1073/pnas.0503927102"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lauter</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kampani</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Carlson</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Goebel</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Moose</span>,  <span class="cit-name-given-names">S. P.</span></span> <span class="cit-article-title">microrna172 down-regulates glossy15 to promote vegetative phase change in maize</span>. <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences</abbr> <span class="cit-vol">102</span>, <span class="cit-fpage">9412</span>–<span class="cit-lpage">9417</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0503927102%26rft_id%253Dinfo%253Apmid%252F15958531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzI2Lzk0MTIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xOC83MDY3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[53].</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.53"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bianchi</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Avato</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Salamini</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Glossy mutants of maize</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">42</span>, <span class="cit-fpage">391</span> (<span class="cit-pub-date">1979</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D42%26rft.spage%253D391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[54].</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.54"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">The maize glossy6 gene is involved in cuticular wax deposition and drought tolerance</span>. <abbr class="cit-jnl-abbrev">Journal of experimental botany</abbr> (<span class="cit-pub-date">2019</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[55].</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.55" data-doi="10.2135/cropsci2012.09.0545"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cairns</span>,  <span class="cit-name-given-names">J. E.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Identification of drought, heat, and combined drought and heat tolerant donors in maize</span>. <abbr class="cit-jnl-abbrev">Crop Science</abbr> <span class="cit-vol">53</span>, <span class="cit-fpage">1335</span>–<span class="cit-lpage">1346</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.volume%253D53%26rft.spage%253D1335%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.09.0545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.09.0545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[56].</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.56" data-doi="10.1104/pp.117.4.1135"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Schöffl</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Prändl</span>,  <span class="cit-name-given-names">R.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Reindl</span>,  <span class="cit-name-given-names">A.</span></span> <span class="cit-article-title">Regulation of the heat-shock response</span>. <abbr class="cit-jnl-abbrev">Plant physiology</abbr> <span class="cit-vol">117</span>, <span class="cit-fpage">1135</span>–<span class="cit-lpage">1141</span> (<span class="cit-pub-date">1998</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DSchoffl%26rft.auinit1%253DF.%26rft.volume%253D117%26rft.issue%253D4%26rft.spage%253D1135%26rft.epage%253D1141%26rft.atitle%253DRegulation%2Bof%2Bthe%2BHeat-Shock%2BResponse%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.117.4.1135%26rft_id%253Dinfo%253Apmid%252F9701569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTE3LzQvMTEzNSI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA3LzE4LzcwNjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[57].</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.57" data-doi="10.1007/s00438-003-0954-8"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lohmann</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eggers-Schumacher</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wunderlich</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Schöffl</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-article-title">Two different heat shock transcription factors regulate immediate early expression of stress genes in arabidopsis</span>. <abbr class="cit-jnl-abbrev">Molecular Genetics and Genomics</abbr> <span class="cit-vol">271</span>, <span class="cit-fpage">11</span>–<span class="cit-lpage">21</span> (<span class="cit-pub-date">2004</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DLohmann%26rft.auinit1%253DC.%26rft.volume%253D271%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D21%26rft.atitle%253DTwo%2Bdifferent%2Bheat%2Bshock%2Btranscription%2Bfactors%2Bregulate%2Bimmediate%2Bearly%2Bexpression%2Bof%2Bstress%2Bgenes%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-003-0954-8%26rft_id%253Dinfo%253Apmid%252F14655047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-003-0954-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14655047&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189000800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[58].</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.58" data-doi="10.1101/gad.228802"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mishra</span>,  <span class="cit-name-given-names">S. K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">In the complex family of heat stress transcription factors, hsfa1 has a unique role as master regulator of thermotolerance in tomato</span>. <abbr class="cit-jnl-abbrev">Genes &amp; Development</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">1555</span>–<span class="cit-lpage">1567</span> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMishra%26rft.auinit1%253DS.%2BK.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D1555%26rft.epage%253D1567%26rft.atitle%253DIn%2Bthe%2Bcomplex%2Bfamily%2Bof%2Bheat%2Bstress%2Btranscription%2Bfactors%252C%2BHsfA1%2Bhas%2Ba%2Bunique%2Brole%2Bas%2Bmaster%2Bregulator%2Bof%2Bthermotolerance%2Bin%2Btomato%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.228802%26rft_id%253Dinfo%253Apmid%252F12080093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE2LzEyLzE1NTUiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xOC83MDY3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[59].</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.59"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lian</span>,  <span class="cit-name-given-names">T.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">C.</span></span> <span class="cit-article-title">Manipulation of metabolic pathways to develop vitamin-enriched crops for human health</span>. <abbr class="cit-jnl-abbrev">Frontiers in plant science</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">937</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bplant%2Bscience%26rft.volume%253D8%26rft.spage%253D937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[60].</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference [60] in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.60"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kremling</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diepenbrock</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gore</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buckler</span>,  <span class="cit-name-given-names">E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Bandillo</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Transcriptome-wide association supplements genome-wide association in zea mays</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">363242</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[61].</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference [61] in text" id="ref-61">↵</a><div class="cit ref-cit ref-book" id="cit-706739v1.61"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hasanuzzaman</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nahar</span>,  <span class="cit-name-given-names">K.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fujita</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-chapter-title">Role of tocopherol (vitamin e) in plants: Abiotic stress tolerance and beyond</span>. In <span class="cit-source">Emerging technologies and management of crop stress tolerance</span>, <span class="cit-fpage">267</span>–<span class="cit-lpage">289</span> (<span class="cit-publ-name">Elsevier</span>, <span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[62].</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference [62] in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.62" data-doi="10.1371/journal.pone.0071377"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bouchet</span>,  <span class="cit-name-given-names">S.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Adaptation of maize to temperate climates: mid-density genome-wide association genetics and diversity patterns reveal key genomic regions, with a major contribution of the vgt2 (zcn8) locus</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>, <span class="cit-fpage">e71377</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De71377%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0071377%26rft_id%253Dinfo%253Apmid%252F24023610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0071377&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24023610&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[63].</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference [63] in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.63"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ku</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Dual functions of the zmcct-associated quantitative trait locus in flowering and stress responses under long-day conditions</span>. <abbr class="cit-jnl-abbrev">BMC plant biology</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">239</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%26rft.volume%253D16%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[64].</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference [64] in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.64"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Minow</span>,  <span class="cit-name-given-names">M. A.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Distinct gene networks modulate floral induction of autonomous maize and photoperiod-dependent teosinte</span>. <abbr class="cit-jnl-abbrev">Journal of experimental botany</abbr> <span class="cit-vol">69</span>, <span class="cit-fpage">2937</span>–<span class="cit-lpage">2952</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bexperimental%2Bbotany%26rft.volume%253D69%26rft.spage%253D2937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[65].</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference [65] in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.65"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moutiq</span>,  <span class="cit-name-given-names">R.</span></span> <abbr class="cit-jnl-abbrev">Identification of quantitative trait loci associated with flowering, vegetative traits and photoperiod response in maize (zea mays l)</abbr> (<span class="cit-pub-date">2002</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[66].</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference [66] in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.66" data-doi="10.1007/BF00215714"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beavis</span>,  <span class="cit-name-given-names">W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Grant</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Albertsen</span>,  <span class="cit-name-given-names">M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Fincher</span>,  <span class="cit-name-given-names">R.</span></span> <span class="cit-article-title">Quantitative trait loci for plant height in four maize populations and their associations with qualitative genetic loci</span>. <abbr class="cit-jnl-abbrev">Theoretical and Applied Genetics</abbr> <span class="cit-vol">83</span>, <span class="cit-fpage">141</span>–<span class="cit-lpage">145</span> (<span class="cit-pub-date">1991</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Band%2BApplied%2BGenetics%26rft.volume%253D83%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00215714%26rft_id%253Dinfo%253Apmid%252F24202349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00215714&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24202349&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[67].</span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference [67] in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.67"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Almeida</span>,  <span class="cit-name-given-names">G. D.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Qtl mapping in three tropical maize populations reveals a set of constitutive and adaptive genomic regions for drought tolerance</span>. <abbr class="cit-jnl-abbrev">Theoretical and Applied Genetics</abbr> <span class="cit-vol">126</span>, <span class="cit-fpage">583</span>–<span class="cit-lpage">600</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Band%2BApplied%2BGenetics%26rft.volume%253D126%26rft.spage%253D583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[68].</span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference [68] in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.68" data-doi="10.1007/s11104-004-1697-y"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>,  <span class="cit-name-given-names">S. M.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">J. P.</span></span> <span class="cit-article-title">Mapping of qtl controlling root hair length in maize (zea mays l.) under phosphorus deficiency</span>. <abbr class="cit-jnl-abbrev">Plant and Soil</abbr> <span class="cit-vol">270</span>, <span class="cit-fpage">299</span>–<span class="cit-lpage">310</span> (<span class="cit-pub-date">2005</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Band%2BSoil%26rft.volume%253D270%26rft.spage%253D299%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11104-004-1697-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11104-004-1697-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000230397000028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[69].</span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference [69] in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.69" data-doi="10.1126/science.1209244"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hancock</span>,  <span class="cit-name-given-names">A. M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Adaptation to climate across the arabidopsis thaliana genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>, <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjM3OiIvYmlvcnhpdi9lYXJseS8yMDE5LzA3LzE4LzcwNjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[70].</span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference [70] in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.70" data-doi="10.1038/ng.3612"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Russell</span>,  <span class="cit-name-given-names">J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Exome sequencing of geographically diverse barley landraces and wild relatives gives insights into environmental adaptation</span>. <abbr class="cit-jnl-abbrev">Nature Genetics</abbr> <span class="cit-vol">48</span>, <span class="cit-fpage">1024</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D48%26rft.spage%253D1024%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3612%26rft_id%253Dinfo%253Apmid%252F27428750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27428750&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[71].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-706739v1.71"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Scheben</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Edwards</span>,  <span class="cit-name-given-names">D.</span></span> <span class="cit-article-title">Towards a more predictable plant breeding pipeline with crispr/cas-induced allelic series to optimize quantitative and qualitative traits</span>. <abbr class="cit-jnl-abbrev">Current opinion in plant biology</abbr> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[72].</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-706739v1.72" data-doi="10.1016/j.cell.2017.08.030"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Leal</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lemmon</span>,  <span class="cit-name-given-names">Z. H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Man</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bartlett</span>,  <span class="cit-name-given-names">M. E.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lippman</span>,  <span class="cit-name-given-names">Z. B.</span></span> <span class="cit-article-title">Engineering quantitative trait variation for crop improvement by genome editing</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">171</span>, <span class="cit-fpage">470</span>–<span class="cit-lpage">480</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D171%26rft.spage%253D470%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.08.030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.08.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[73].</span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference [73] in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.73" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (gbs) approach for high diversity species</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">6</span>, <span class="cit-fpage">e19379</span> (<span class="cit-pub-date">2011</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[74].</span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference [74] in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.74" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>,  <span class="cit-name-given-names">J. C.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Tassel-gbs: a high capacity genotyping by sequencing analysis pipeline</span>. <abbr class="cit-jnl-abbrev">PloS one</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">e90346</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[75].</span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference [75] in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.75" data-doi="10.1126/science.aam9425"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Swarts</span>,  <span class="cit-name-given-names">K.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genomic estimation of complex traits reveals ancient maize adaptation to temperate north america</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">357</span>, <span class="cit-fpage">512</span>–<span class="cit-lpage">515</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSwarts%26rft.auinit1%253DK.%26rft.volume%253D357%26rft.issue%253D6350%26rft.spage%253D512%26rft.epage%253D515%26rft.atitle%253DGenomic%2Bestimation%2Bof%2Bcomplex%2Btraits%2Breveals%2Bancient%2Bmaize%2Badaptation%2Bto%2Btemperate%2BNorth%2BAmerica%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aam9425%26rft_id%253Dinfo%253Apmid%252F28774930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTcvNjM1MC81MTIiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOS8wNy8xOC83MDY3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[76].</span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference [76] in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.76"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jiao</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Improved maize reference genome with single molecule technologies</span>. <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-fpage">079004</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[77].</span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference [77] in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.77"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">H.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Crossmap: a versatile tool for coordinate conversion between genome assemblies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">1006</span>–<span class="cit-lpage">1007</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D30%26rft.spage%253D1006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[78].</span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference [78] in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.78"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Reuter</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Guevara</span>,  <span class="cit-name-given-names">E.</span></span> <span class="cit-article-title">Hole-filled seamless srtm data v4: International centre for tropical agriculture (ciat): http</span>. <abbr class="cit-jnl-abbrev">srtm. csi. cgiar. org, accessed</abbr> <span class="cit-vol">31</span> (<span class="cit-pub-date">2008</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[79].</span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference [79] in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.79" data-doi="10.1002/joc.3711"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">P. D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Osborn</span>,  <span class="cit-name-given-names">T. J.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Lister</span>,  <span class="cit-name-given-names">D. H.</span></span> <span class="cit-article-title">Updated high-resolution grids of monthly climatic observations-the cru ts3. 10 dataset</span>. <abbr class="cit-jnl-abbrev">International journal of climatology</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">623</span>–<span class="cit-lpage">642</span> (<span class="cit-pub-date">2014</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bclimatology%26rft.volume%253D34%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjoc.3711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/joc.3711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000332092900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[80].</span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference [80] in text" id="ref-80">↵</a><div class="cit ref-cit ref-web" id="cit-706739v1.80"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Trabucco</span>,  <span class="cit-name-given-names">A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Zomer</span>,  <span class="cit-name-given-names">R.</span></span><span class="cit-article-title">Global aridity index and potential evapotranspiration (et0) climate database v2</span> (<span class="cit-pub-date">2019</span>). URL <a href="https://figshare.com/articles/Global_Aridity_Index_and_Potential_Evapotranspiration_ET0_Climate_Database_v2/7504448">https://figshare.com/articles/Global_Aridity_Index_and_Potential_Evapotranspiration_ET0_Climate_Database_v2/7504448</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[81].</span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference [81] in text" id="ref-81">↵</a><div class="cit ref-cit ref-web" id="cit-706739v1.81"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">INIFAP &amp; CONABIO</span>. <span class="cit-source">Edafologia mexico 1:1000000. esri shapefile</span>. (<span class="cit-pub-date">1995</span>). URL “<a href="http://www.conabio.gob.mx/informacion/metadata/gis/eda251mgw.xml?_httpcache=yes&amp;_xsl=/db/metadata/xsl/fgdc_html.xsl&amp;_indent=no">http://www.conabio.gob.mx/informacion/metadata/gis/eda251mgw.xml?_httpcache=yes&amp;_xsl=/db/metadata/xsl/fgdc_html.xsl&amp;_indent=no</a>”.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[82].</span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference [82] in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.82" data-doi="10.1016/S0016-7061(03)00040-5"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>,  <span class="cit-name-given-names">P. A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palm</span>,  <span class="cit-name-given-names">C. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Buol</span>,  <span class="cit-name-given-names">S. W.</span></span> <span class="cit-article-title">Fertility capability soil classification: a tool to help assess soil quality in the tropics</span>. <abbr class="cit-jnl-abbrev">Geoderma</abbr> <span class="cit-vol">114</span>, <span class="cit-fpage">157</span>–<span class="cit-lpage">185</span> (<span class="cit-pub-date">2003</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeoderma%26rft.stitle%253DGeoderma%26rft.volume%253D114%26rft.issue%253D3-4%26rft.spage%253D157%26rft.epage%253D185%26rft.atitle%253DFertility%2Bcapability%2Bsoil%2Bclassification%253B%2Ba%2Btool%2Bto%2Bhelp%2Bassess%2Bsoil%2Bquality%2Bin%2Bthe%2Btropics%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0016-7061%252803%252900040-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0016-7061(03)00040-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2003040681&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=000182558000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[83].</span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference [83] in text" id="ref-83">↵</a><div class="cit ref-cit ref-web" id="cit-706739v1.83"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ahamed</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Palm</span>,  <span class="cit-name-given-names">C. A.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Sanchez</span>,  <span class="cit-name-given-names">P. A.</span></span> <span class="cit-article-title">Global aridity index and potential evapotranspiration (et0) climate database v2</span> (<span class="cit-pub-date">2010</span>). URL <a href="http://harvestchoice.org/node/1435">http://harvestchoice.org/node/1435</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[84].</span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference [84] in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.84" data-doi="10.1186/s12711-015-0105-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>,  <span class="cit-name-given-names">T. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rosa</span>,  <span class="cit-name-given-names">G. J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>,  <span class="cit-name-given-names">S. M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">De Leon</span>,  <span class="cit-name-given-names">N.</span></span> <span class="cit-article-title">Defining window-boundaries for genomic analyses using smoothing spline techniques</span>. <abbr class="cit-jnl-abbrev">Genetics Selection Evolution</abbr> <span class="cit-vol">47</span>, <span class="cit-fpage">30</span> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%2BSelection%2BEvolution%26rft.volume%253D47%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0105-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0105-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[85].</span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference [85] in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.85"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>,  <span class="cit-name-given-names">M.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Software for computing and annotating genomic ranges</span>. <abbr class="cit-jnl-abbrev">PLoS computational biology</abbr> <span class="cit-vol">9</span>, <span class="cit-fpage">e1003118</span> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D9%26rft.spage%253D1003118e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[86].</span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference [86] in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.86"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yuan</span>,  <span class="cit-name-given-names">Y.</span></span> <span class="cit-etal">et al.</span> <span class="cit-article-title">Genome-wide association mapping and genomic prediction analyses reveal the genetic architecture of grain yield and flowering time under drought and heat stress conditions in maize</span>. <abbr class="cit-jnl-abbrev">Frontiers in plant science</abbr> <span class="cit-vol">9</span> (<span class="cit-pub-date">2018</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[87].</span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference [87] in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.87" data-doi="10.1038/nbt.3519"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bray</span>,  <span class="cit-name-given-names">N. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pimentel</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melsted</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pachter</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Near-optimal probabilistic rna-seq quantification</span>. <abbr class="cit-jnl-abbrev">Nature biotechnology</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">525</span> (<span class="cit-pub-date">2016</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.volume%253D34%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3519%26rft_id%253Dinfo%253Apmid%252F27043002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27043002&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2019%2F07%2F18%2F706739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[88].</span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference [88] in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-706739v1.88"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bray</span>,  <span class="cit-name-given-names">N. L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Puente</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Melsted</span>,  <span class="cit-name-given-names">P.</span></span> &amp; <span class="cit-auth"><span class="cit-name-surname">Pachter</span>,  <span class="cit-name-given-names">L.</span></span> <span class="cit-article-title">Differential analysis of rna-seq incorporating quantification uncertainty</span>. <abbr class="cit-jnl-abbrev">Nature methods</abbr> <span class="cit-vol">14</span>, <span class="cit-fpage">687</span> (<span class="cit-pub-date">2017</span>).</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D14%26rft.spage%253D687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/706739v1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F706739v1" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/706614v1" title="A selective and rapid cell-permeable inhibitor of human caspase-3" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/283929v2-0" title="Cell-type heterogeneity in adipose tissue is associated with complex traits and reveals disease-relevant cell-specific eQTLs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;July 18, 2019.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/706739v1.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/706739v1.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/706739v1.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/706739v1" class="active" data-icon-position="" data-hide-link-title="0">Single-gene resolution of locally adaptive genetic variation in Mexican maize</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/823424" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-E7iaSwQb2EgODKEijGMhiYg2txUY6AmSKaVrKDnbe04" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="823424" id="node823424--2" data-pisa="biorxiv;706739v1" data-pisa-master="biorxiv;706739" data-seqnum="823424" data-apath="/biorxiv/early/2019/07/18/706739.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Single-gene resolution of locally adaptive genetic variation in Mexican maize</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Daniel J</span> <span class="nlm-surname">Gates</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dan</span> <span class="nlm-surname">Runcie</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Garrett M.</span> <span class="nlm-surname">Janzen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Alberto Romero</span> <span class="nlm-surname">Navarro</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Martha</span> <span class="nlm-surname">Willcox</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kai</span> <span class="nlm-surname">Sonder</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Samantha J.</span> <span class="nlm-surname">Snodgrass</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Fausto</span> <span class="nlm-surname">Rodríguez-Zapata</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Ruairidh J. H.</span> <span class="nlm-surname">Sawers</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Rubén</span> <span class="nlm-surname">Rellán-Álvarez</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Edward S.</span> <span class="nlm-surname">Buckler</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Sarah</span> <span class="nlm-surname">Hearne</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Matthew B.</span> <span class="nlm-surname">Hufford</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Jeffrey</span> <span class="nlm-surname">Ross-Ibarra</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">706739; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/706739 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/706739v1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/706739v1&amp;title=Single-gene%20resolution%20of%20locally%20adaptive%20genetic%20variation%20in%20Mexican%20maize" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/706739v1&amp;title=Single-gene%20resolution%20of%20locally%20adaptive%20genetic%20variation%20in%20Mexican%20maize" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/706739v1&amp;text=Single-gene%20resolution%20of%20locally%20adaptive%20genetic%20variation%20in%20Mexican%20maize" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/706739v1&amp;title=Single-gene%20resolution%20of%20locally%20adaptive%20genetic%20variation%20in%20Mexican%20maize" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/706739v1&amp;t=Single-gene%20resolution%20of%20locally%20adaptive%20genetic%20variation%20in%20Mexican%20maize" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/706739v1&amp;title=Single-gene%20resolution%20of%20locally%20adaptive%20genetic%20variation%20in%20Mexican%20maize" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/706739v1&amp;title=Single-gene%20resolution%20of%20locally%20adaptive%20genetic%20variation%20in%20Mexican%20maize" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="823424" id="node823424--3" data-pisa="biorxiv;706739v1" data-pisa-master="biorxiv;706739" data-seqnum="823424" data-apath="/biorxiv/early/2019/07/18/706739.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Single-gene resolution of locally adaptive genetic variation in Mexican maize</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Daniel J</span> <span class="nlm-surname">Gates</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Dan</span> <span class="nlm-surname">Runcie</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Garrett M.</span> <span class="nlm-surname">Janzen</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Alberto Romero</span> <span class="nlm-surname">Navarro</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Martha</span> <span class="nlm-surname">Willcox</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Kai</span> <span class="nlm-surname">Sonder</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Samantha J.</span> <span class="nlm-surname">Snodgrass</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Fausto</span> <span class="nlm-surname">Rodríguez-Zapata</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Ruairidh J. H.</span> <span class="nlm-surname">Sawers</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Rubén</span> <span class="nlm-surname">Rellán-Álvarez</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Edward S.</span> <span class="nlm-surname">Buckler</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Sarah</span> <span class="nlm-surname">Hearne</span></span>, <span class="highwire-citation-author" data-delta="12"><span class="nlm-given-names">Matthew B.</span> <span class="nlm-surname">Hufford</span></span>, <span class="highwire-citation-author" data-delta="13"><span class="nlm-given-names">Jeffrey</span> <span class="nlm-surname">Ross-Ibarra</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">706739; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/706739 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Single-gene%20resolution%20of%20locally%20adaptive%20genetic%20variation%20in%20Mexican%20maize&amp;rft.volume&amp;rft.issue&amp;rft.date=2019-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Gates%2C+Daniel+J&amp;rft.au=Runcie%2C+Dan&amp;rft.au=Janzen%2C+Garrett+M.&amp;rft.au=Navarro%2C+Alberto+Romero&amp;rft.au=Willcox%2C+Martha&amp;rft.au=Sonder%2C+Kai&amp;rft.au=Snodgrass%2C+Samantha+J.&amp;rft.au=Rodr%C3%ADguez-Zapata%2C+Fausto&amp;rft.au=Sawers%2C+Ruairidh+J.+H.&amp;rft.au=Rell%C3%A1n-%C3%81lvarez%2C+Rub%C3%A9n&amp;rft.au=Buckler%2C+Edward+S.&amp;rft.au=Hearne%2C+Sarah&amp;rft.au=Hufford%2C+Matthew+B.&amp;rft.au=Ross-Ibarra%2C+Jeffrey"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/823424/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/823424/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/823424/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/823424/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/823424/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/823424/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/823424/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/823424/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/823424/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/823424/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/823424/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/823424/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/706739v1&amp;count=horizontal&amp;via=&amp;text=Single-gene%20resolution%20of%20locally%20adaptive%20genetic%20variation%20in%20Mexican%20maize&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/706739v1" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/706739v1&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/706739v1" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Evolutionary Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__bLlcwhzZhT-IwoQ_dvm1_xPxU7E7gecFLC6Fyeqtv8s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
