<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/embed/inline-graphic-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2018/01/05/224352.atom" />
<meta name="HW.pisa" content="biorxiv;224352v2" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Ecological change points: The strength of density dependence and the loss of history" />
<meta name="DC.Identifier" content="10.1101/224352" />
<meta name="DC.Date" content="2018-01-05" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/" />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="Change points in the dynamics of animal abundances have extensively been recorded in historical time series records. Little attention has been paid to the theoretical dynamic consequences of such change-points. Here we propose a change-point model of stochastic population dynamics. This investigation embodies a shift of attention from the problem of detecting when a change will occur, to another non-trivial puzzle: using ecological theory to understand and predict the post-breakpoint behavior of the population dynamics. The proposed model and the explicit expressions derived here predict and quantify how density dependence modulates the influence of the pre-breakpoint parameters into the post-breakpoint dynamics. Time series transitioning from one stationary distribution to another contain information about where the process was before the change-point, where is it heading and how long it will take to transition, and here this information is explicitly stated. Importantly, our results provide a direct connection of the strength of density dependence with theoretical properties of dynamic systems, such as the concept of resilience. Finally, we illustrate how to harness such information through maximum likelihood estimation for state-space models, and test the model robustness to widely different forms of compensatory dynamics. The model can be used to estimate important quantities in the theory and practice of population recovery." />
<meta name="DC.Contributor" content="José M. Ponciano" />
<meta name="DC.Contributor" content="Mark L. Taper" />
<meta name="DC.Contributor" content="Brian Dennis" />
<meta name="article:published_time" content="2018-01-05" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="Ecological change points: The strength of density dependence and the loss of history" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;Abstract&lt;/h3&gt;
&lt;p&gt;Change points in the dynamics of animal abundances have extensively been recorded in historical time series records. Little attention has been paid to the theoretical dynamic consequences of such change-points. Here we propose a change-point model of stochastic population dynamics. This investigation embodies a shift of attention from the problem of detecting when a change will occur, to another non-trivial puzzle: using ecological theory to understand and predict the post-breakpoint behavior of the population dynamics. The proposed model and the explicit expressions derived here predict and quantify how density dependence modulates the influence of the pre-breakpoint parameters into the post-breakpoint dynamics. Time series transitioning from one stationary distribution to another contain information about where the process was before the change-point, where is it heading and how long it will take to transition, and here this information is explicitly stated. Importantly, our results provide a direct connection of the strength of density dependence with theoretical properties of dynamic systems, such as the concept of resilience. Finally, we illustrate how to harness such information through maximum likelihood estimation for state-space models, and test the model robustness to widely different forms of compensatory dynamics. The model can be used to estimate important quantities in the theory and practice of population recovery.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="biorxiv;224352v2" />
<meta name="citation_id" content="224352v2" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/224352v2" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/224352v2.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/224352v2.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352.full.pdf" />
<meta name="citation_doi" content="10.1101/224352" />
<meta name="citation_num_pages" content="34" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="224352" />
<meta name="citation_author" content="José M. Ponciano" />
<meta name="citation_author_institution" content="Department of Biology, University of Florida" />
<meta name="citation_author_email" content="josemi@ufl.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8457-7840" />
<meta name="citation_author" content="Mark L. Taper" />
<meta name="citation_author_institution" content="Department of Ecology, Montana State University" />
<meta name="citation_author" content="Brian Dennis" />
<meta name="citation_author_institution" content="Department of Fish and Wildlife Sciences and Department of Statistical Science, University of Idaho" />
<meta name="citation_reference" content="M. Bulmer, The statistical analysis of density dependence, Biometrics (1975) 901–911." />
<meta name="citation_reference" content="M. Scheffer, Alternative stable states and regime shifts in ecosystems, The Princeton guide to ecology, Princeton, Princeton university press 809 (2009) 395–406." />
<meta name="citation_reference" content="D. F. Staples, M. L. Taper, B. Dennis, Estimating population trend and process variation for pva in the presence of sampling error, Ecology 85 (2004) 923–929." />
<meta name="citation_reference" content="D. F. Staples, M. L. Taper, B. B. Shepard, Risk-based viable population monitoring, Conservation Biology 19 (2005) 1908–1916." />
<meta name="citation_reference" content="D. F. Staples, et al., Viable population monitoring: Risk-based population monitoring for threatened and endangered species with application to bull trout, Salvelinus confluentus, Ph.D. thesis, Montana State University-Bozeman, College of Letters &amp; Science, 2006." />
<meta name="citation_reference" content="C. C. Muhlfeld, M. L. Taper, D. F. Staples, B. B. Shepard, Observer error structure in bull trout redd counts in montana streams: implications for inference on true redd numbers, Transactions of the American Fisheries Society 135 (2006) 643–654." />
<meta name="citation_reference" content="B. Dennis, J. M. Ponciano, S. R. Lele, M. L. Taper, D. F. Staples, Estimating density dependence, process noise, and observation error, Ecological Monographs 76 (2006) 323–341." />
<meta name="citation_reference" content="M. L. Taper, D. F. Staples, B. B. Shepard, Model structure adequacy analysis: selecting models on the basis of their ability to answer scientific questions, Synthese 163 (2008) 357–370." />
<meta name="citation_reference" content="D. F. Staples, M. L. Taper, B. Dennis, R. J. Boik, Effects of sampling error and temporal correlations in population growth on process variance estimators, Environmental and ecological statistics 16 (2009) 547–560." />
<meta name="citation_reference" content="R. D. Holt, Density-independent mortality, non-linear competitive interactions, and species coexistence, Journal of Theoretical Biology 116 (1985) 479–493." />
<meta name="citation_reference" content="J.-D. Lebreton, O. Gimenez, Detecting and estimating density dependence in wildlife populations, The Journal of Wildlife Management 77 (2013) 12–23." />
<meta name="citation_reference" content="B. Dennis, M. L. Taper, Density dependence in time series observations of natural populations: estimation and testing, Ecological Monographs 64 (1994) 205–224." />
<meta name="citation_reference" content="B. Dennis, W. Kemp, M. Taper, Joint density dependence, Ecology 79 (1998) 426–441." />
<meta name="citation_reference" content="R. P. Freckleton, A. R. Watkinson, R. E. Green, W. J. Sutherland, Census error and the detection of density dependence, Journal of Animal Ecology 75 (2006) 837–851." />
<meta name="citation_reference" content="J. R. Thomson, W. J. Kimmerer, L. R. Brown, K. B. Newman, R. M. Nally, W. A. Bennett, F. Feyrer, E. Fleishman, Bayesian change point analysis of abundance trends for pelagic fishes in the upper san francisco estuary, Ecological Applications 20 (2010) 1431–1448." />
<meta name="citation_reference" content="C. Packer, R. Hilborn, A. Mosser, B. Kissui, M. Borner, G. Hopcraft, J. Wilmshurst, S. Mduma, A. R. Sinclair, Ecological change, group territoriality, and population dynamics in serengeti lions, Science 307 (2005) 390–393." />
<meta name="citation_reference" content="M. B. Hooten, C. K. Wikle, Shifts in the spatio-temporal growth dynamics of shortleaf pine, Environmental and Ecological Statistics 14 (2007) 207–227." />
<meta name="citation_reference" content="T. Andersen, J. Carstensen, E. Hernandez-Garcia, C. M. Duarte, Ecological thresholds and regime shifts: approaches to identification, Trends in Ecology &amp; Evolution 24 (2009) 49–57." />
<meta name="citation_reference" content="A. R. Ives, V. Dakos, Detecting dynamical changes in nonlinear time series using locally linear state-space models, Ecosphere 3 (2012) art58." />
<meta name="citation_reference" content="B. E. Hansen, The likelihood ratio test under nonstandard conditions: testing the markov switching model of gnp, Journal of Applied Econometrics 7 (1992) S61–S82." />
<meta name="citation_reference" content="R. Garcia, Asymptotic null distribution of the likelihood ratio test in markov switching models, International Economic Review (1998) 763–788." />
<meta name="citation_reference" content="D. H. Timmer, J. J. Pignatiello, Change point estimates for the parameters of an ar (1) process, Quality and Reliability Engineering International 19 (2003) 355–369." />
<meta name="citation_reference" content="V. Jandhyala, S. Fotopoulos, I. MacNeill, P. Liu, Inference for single and multiple change-points in time series, Journal of Time Series Analysis (2013)." />
<meta name="citation_reference" content="D. Picard, Testing and estimating change-points in time series, Advances in Applied Probability (1985) 841–867." />
<meta name="citation_reference" content="D. W. Andrews, Tests for parameter instability and structural change with unknown change point, Econometrica: Journal of the Econometric Society (1993) 821–856." />
<meta name="citation_reference" content="J. Bai, P. Perron, Estimating and testing linear models with multiple structural changes, Econometrica (1998) 47–78." />
<meta name="citation_reference" content="M. Lavielle, E. Moulines, Least-squares estimation of an unknown number of shifts in a time series, Journal of Time Series Analysis 21 (2000) 33–59." />
<meta name="citation_reference" content="L. Perreault, J. Bernier, B. Bobee, E. Parent, Bayesian change-point analysis in hydrometeorological time series. part 1. the normal model revisited, Journal of Hydrology 235 (2000) 221–241." />
<meta name="citation_reference" content="F. Yuan, Tests for Change-Point of the AR (1) Model, Ph.D. thesis, University of Minnesota, 2013." />
<meta name="citation_reference" content="M. B. Perry, Identifying the time of polynomial drift in the mean of autocorrelated processes, Quality and Reliability Engineering International 26 (2010) 399–415." />
<meta name="citation_reference" content="A. Amiri, S. Allahyari, Change point estimation methods for control chart postsignal diagnostics: a literature review, Quality and Reliability Engineering International 28 (2012) 673–685." />
<meta name="citation_reference" content="T. Zhou, Bayesian approach to Phase II statistical process control for time series, Ph.D. thesis, University of Minnesota, 2013." />
<meta name="citation_reference" content="B. Dennis, G. Patil, The gamma distribution and weighted multimodal gamma distributions as models of population abundance, Mathematical Biosciences 68 (1984) 187–212." />
<meta name="citation_reference" content="Z. Zeng, R. M. Nowierski, M. L. Taper, B. Dennis, W. P. Kemp, Complex population dynamics in the real world: modeling the influence of time-varying parameters and time lags, Ecology 79 (1998) 2193–2209." />
<meta name="citation_reference" content="P. de Valpine, A. Hastings, Fitting population models incorporating process noise and observation error, Ecological Monographs 72 (2002) 57–76." />
<meta name="citation_reference" content="A. Ives, B. Dennis, K. Cottingham, S. Carpenter, Estimating community stability and ecological interactions from time-series data, Ecological Monographs 73 (2003) 301–330." />
<meta name="citation_reference" content="J. S. Clark, O. N. Bjørnstad, Population time series: process variability, observation errors, missing values, lags, and hidden states, Ecology 85 (2004) 3140–3150." />
<meta name="citation_reference" content="S. R. Lele, B. Dennis, F. Lutscher, Data cloning: easy maximum likelihood estimation for complex ecological models using bayesian markov chain monte carlo methods, Ecology Letters 10 (2007) 551–563." />
<meta name="citation_reference" content="J. M. Ponciano, M. L. Taper, B. Dennis, S. R. Lele, Hierarchical models in ecology: confidence intervals, hypothesis testing, and model selection using data cloning, Ecology 90 (2009) 356–362." />
<meta name="citation_reference" content="L. Polansky, P. De Valpine, J. O. Lloyd-Smith, W. M. Getz, Likelihood ridges and multimodality in population growth rate models, Ecology 90 (2009) 2313–2320." />
<meta name="citation_reference" content="B. Dennis, J. M. Ponciano, M. L. Taper, Replicated sampling increases efficiency in monitoring biological populations, Ecology 91 (2010) 610–620." />
<meta name="citation_reference" content="P. de Valpine, Frequentist analysis of hierarchical models for population dynamics and demographic data, Journal of Ornithology 152 (2012) 393–408." />
<meta name="citation_reference" content="B.-E. Sæther, T. Coulson, V. Grøtan, S. Engen, R. Altwegg, K. B. Armitage, C. Barbraud, P. H. Becker, D. T. Blumstein, F. S. Dobson, et al., How life history influences population dynamics in fluctuating environments, The American Naturalist 182 (2013) 743–759." />
<meta name="citation_reference" content="S. R. Lele, M. L. Taper, Statistical inference in ecology, in: International Encyclopedia of Statistical Science, Springer, 2011, pp. 1430–1433." />
<meta name="citation_reference" content="J. Knape, Estimability of density dependence in models of time series data, Ecology 89 (2008) 2994–3000." />
<meta name="citation_reference" content="R. Holt, M. Barfield, Trait-mediated effects, density dependence and the dynamic stability of ecological systems, Trait-Mediated Indirect Interactions: Ecological and Evolutionary Perspectives (2012) 89." />
<meta name="citation_reference" content="R. Lande, S. Engen, B.-E. Sæther, Estimating density dependence in time-series of age-structured populations, Philosophical Transactions of the Royal Society of London. Series B: Biological Sciences 357 (2002) 1179–1184." />
<meta name="citation_reference" content="M. Kot, Elements of mathematical ecology, Cambridge University Press, 2001." />
<meta name="citation_reference" content="S. Herrando-Perez, S. Delean, B. W. Brook, C. J. Bradshaw, Density dependence: an ecological tower of babel, Oecologia 170 (2012) 585–603." />
<meta name="citation_reference" content="B. Dennis, G. Patil, Applications in ecology, Lognormal Distributions: Theory and Applications (1988) 303–330." />
<meta name="citation_reference" content="H. Rue, L. Held, Gaussian Markov random fields: theory and applications, CRC Press, 2005." />
<meta name="citation_reference" content="A. Berryman, M. Lima, Deciphering the effects of climate on animal populations: diagnostic analysis provides new interpretation of soay sheep dynamics, The American Naturalist 168 (2006) 784–795." />
<meta name="citation_reference" content="J. M. Ferguson, J. M. Ponciano, Evidence and implications of higher order scaling in the environmental variation of animal population growth, Proceedings of the National Academy of Sciences In press (2015)." />
<meta name="citation_reference" content="J. M. Cushing, R. F. Costantino, B. Dennis, R. Desharnais, S. M. Henson, Chaos in ecology: experimental nonlinear dynamics, volume 1, Elsevier, 2002." />
<meta name="citation_reference" content="B. A. Melbourne, A. Hastings, Extinction risk depends strongly on factors contributing to stochasticity, Nature 454 (2008) 100–103." />
<meta name="citation_reference" content="J. M. Ferguson, J. M. Ponciano, Predicting the process of extinction in experimental microcosms and accounting for interspecific interactions in single-species time series, Ecology Letters 17 (2014) 251–259." />
<meta name="citation_reference" content="M. Schwager, K. Johst, F. Jeltsch, Does red noise increase or decrease extinction risk? single extreme events versus series of unfavorable conditions, The American Naturalist 167 (2006) 879–888." />
<meta name="citation_reference" content="B. Dennis, L. Assas, S. Elaydi, E. Kwessi, G. Livadiotis, Allee effects and resilience in stochastic populations, Theoretical Ecology 9 (2016) 323–335." />
<meta name="citation_reference" content="T. J. Case, Illustrated guide to theoretical ecology, Ecology 80 (2000) 2848–2848." />
<meta name="citation_reference" content="R. M. Sibly, D. Barker, M. C. Denham, J. Hone, M. Pagel, On the regulation of populations of mammals, birds, fish, and insects, Science 309 (2005) 607–610." />
<meta name="citation_reference" content="C. E. Cattau, R. J. Fletcher Jr, R. T. Kimball, C. W. Miller, W. M. Kitchens, Rapid morphological change of a top predator with the invasion of a novel prey, Nature ecology &amp; evolution 2 (2018) 108." />
<meta name="citation_reference" content="B. Connors, A. Cooper, R. Peterman, N. Dulvy, The false classification of extinction risk in noisy environments, Proceedings of the Royal Society B: Biological Sciences 281 (2014) 20132935." />
<meta name="citation_reference" content="J. Ravel, R. M. Brotman, P. Gajer, B. Ma, M. Nandy, D. W. Fadrosh, J. Sakamoto, S. S. Koenig, L. Fu, X. Zhou, et al., Daily temporal dynamics of vaginal microbiota before, during and after episodes of bacterial vaginosis, Microbiome 1 (2013) 29." />
<meta name="citation_reference" content="J. M. Drake, Early warning signals of stochastic switching, Proceedings of the Royal Society B: Biological Sciences 280 (2013) 20130686." />
<meta name="citation_reference" content="C. Boettiger, A. Hastings, Quantifying limits to detection of early warning for critical transitions, Journal of The Royal Society Interface 9 (2012) 2527–2539." />
<meta name="citation_reference" content="P. Chesson, Aedt: A new concept for ecological dynamics in the ever-changing world, PLoS biology 15 (2017) e2002634." />
<meta name="twitter:title" content="Ecological change points: The strength of density dependence and the loss of history" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="Change points in the dynamics of animal abundances have extensively been recorded in historical time series records. Little attention has been paid to the theoretical dynamic consequences of such change-points. Here we propose a change-point model of stochastic population dynamics. This investigation embodies a shift of attention from the problem of detecting when a change will occur, to another non-trivial puzzle: using ecological theory to understand and predict the post-breakpoint behavior of the population dynamics. The proposed model and the explicit expressions derived here predict and quantify how density dependence modulates the influence of the pre-breakpoint parameters into the post-breakpoint dynamics. Time series transitioning from one stationary distribution to another contain information about where the process was before the change-point, where is it heading and how long it will take to transition, and here this information is explicitly stated. Importantly, our results provide a direct connection of the strength of density dependence with theoretical properties of dynamic systems, such as the concept of resilience. Finally, we illustrate how to harness such information through maximum likelihood estimation for state-space models, and test the model robustness to widely different forms of compensatory dynamics. The model can be used to estimate important quantities in the theory and practice of population recovery." />
<meta name="og-title" property="og:title" content="Ecological change points: The strength of density dependence and the loss of history" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/224352v2" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="Change points in the dynamics of animal abundances have extensively been recorded in historical time series records. Little attention has been paid to the theoretical dynamic consequences of such change-points. Here we propose a change-point model of stochastic population dynamics. This investigation embodies a shift of attention from the problem of detecting when a change will occur, to another non-trivial puzzle: using ecological theory to understand and predict the post-breakpoint behavior of the population dynamics. The proposed model and the explicit expressions derived here predict and quantify how density dependence modulates the influence of the pre-breakpoint parameters into the post-breakpoint dynamics. Time series transitioning from one stationary distribution to another contain information about where the process was before the change-point, where is it heading and how long it will take to transition, and here this information is explicitly stated. Importantly, our results provide a direct connection of the strength of density dependence with theoretical properties of dynamic systems, such as the concept of resilience. Finally, we illustrate how to harness such information through maximum likelihood estimation for state-space models, and test the model robustness to widely different forms of compensatory dynamics. The model can be used to estimate important quantities in the theory and practice of population recovery." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2018-01-05" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/224352v2.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/224352v2.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/224352v2.ppt" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/224352v2" />
<link rel="shortlink" href="https://www.biorxiv.org/node/74472" />
    <title>Ecological change points: The strength of density dependence and the loss of history | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__jMRAK66KMC1e4TQlwUNn3KiWVDC5AjueUAYEm1xBY_U__KSUjdT4jdcJ4qJz6fKY1K9WkYh1a5EcaWZxt_-zbTis__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q5e781" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__qlYh2rDEXmsOlJKnxoaEiN2PoZU7YzDXSSTk6K1VAQE__XkaAZGAgxyDmOmv0mrGL0sA-jzEgAbgA12amvVPlIf0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__GFpBMW7A95ZNl4oKhiI_0u7WD_8tFHiDfbKIVNkYgis__AQm9HArb2Y_D7UzgraiLIwAB7088fIq2iZD0Uigie_s__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2GA36xO4909042UvLkzdUfDlC8goNvX-K-IFzM-zzeA__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__lnFWsMpGZ7GjgbuXw29vPE0f_bgKAMdegQ-1zHT0M2g__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?q5e781"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js___KyvuEvfmewWxdOLBXTYsSUo8Pfm_DijV6fWtGvSL_k__HSKkTgIr7nDcdvQYewipf6v4ak6ZLUOQkwPj44wKouA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__OGPr3wJNGw2BAY3Tj6TjIBwKmjQh5i-7muOaPrr0MFA__fCWiU_pDm6xztXUSccH7zrf5422-kgOkuLdWyafrH_0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__hqoKSXYZqkcZkXP--ltzsy0Y3txW5TJBw6HT1khr30Y__6_HqNYoBVMxlkWZwdscqCLCtCxpOhl9_rIOpTDdaI0k__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__lrItkPGmrIuH-E_sdUWEK2i9eTqLar16BAHVxNTWqrc__FhtnljBrjuzYWWrT_f4OYV4jtMBituhMpusb3h0m0rY__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__1PtAOZ8x7l6QAi8D7X6Kj_Y7HcUOFzvWRvvYHUcCT98__U8IB92u_CnnGwJ-X_xaUa9lzsC8OzUfN0WkK2JQfirg__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TtICw43d9WDd9bOsbZX5z-FUhsaFrl7brvhopNk4UaI","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"modules\/forum\/forum.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/default\/modules\/biorxiv\/css\/biorxiv.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/flexible\/flexible.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_collection\/css\/highwire_subject_collection.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire.highwire-list-expand.css":1,"public:\/\/ctools\/css\/bab59090150caf0c64e27afa1b36eef1.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/css\/eu_cookie_compliance.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"0":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_biorxiv_styles\/css\/biorxiv-styles.css.less":1,"https:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.css":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"http:\/\/d33xdlntwy0kbs.cloudfront.net\/cshl_custom.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/eu_cookie_compliance.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/adv-search-collection-toggle.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/eu_cookie_compliance\/js\/jquery.cookie-1.4.1.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/default\/modules\/jnl_biorxiv_styles\/js\/biorxiv-scripts.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.highwire-list-expand.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"74472","apath":"\/biorxiv\/early\/2018\/01\/05\/224352.atom","pisa":"biorxiv;224352v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;224352v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/224352v2","title":"Ecological change points: The strength of density dependence and the loss of history","identifier":"node\/74472"},"panels_ajax_pane":{"new-41":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220fosnls7GLIN2mRGr6hqIHqdHg4LxCjD3quWhtmXgEsoHMqunbuIJkEtLqhg6nmF4vAQj1lIeWZ\\\\\\\/SXRBdwZVqn4UJIHT5sFezyEofCPxGP3M5vgg7anJC+Pzi7lFphwyzW1HgfVaRup5Db9WDNJIvf7uKyg0\\\\\\\/jkxPw\\\\\\\/ff+QbpcI2140KmWIHqrRWPGkojJep7Pr+vE7MxIy+qfjbUKG8N2of3Zk5gMudE8L4Au1QO9Re8oh57yCfeG+t5ueC+b7HldaOWghet9SyMHlRhDERG6R\\\\\\\/hGhzm0QP3VViN6rEwKbvjuHVNskXrc69H2dKHJ9+1IfeSOf8uYh\\\\\\\/TaZHKMzncDo4HDaP6fpodkarGdUlmu8mOi8mMk2oV9R26\\\\\\\/0jVhw9ughe7E2TGKboaW5YQfI2XpoAVZsgbVcbbwRRHMMvhXxQ08BLzlnb2Tfi4T6gnkOAEuUflguvNPwdjWj0P9rJpEFPTRAfNH2G9J1YxWy+\\\\\\\/U4qkUHnCUBcKb2tV1zSD068cKBcaNV4IJ8fohRjZpgspsPNqd6jCdElwaROHecuvHHXPxdCHIqtYidnIWqyJ\\\\\\\/54Vv6LvogfL2zxS712GThik87F4UbIh4VovMR74uR1hjh0fUiDXHrwyVMzdkjWQvy0+FJM+oiNWZqrenXUikMye+8jrDgXumjkYHPml3IaItc6GmRhnL30IWc5kFzg1XsrwXH5Jn6Oct53bpgFRLSqf4OjLy1q6XwhpgYJJ0zmrNDYhqYXM6uFqvC68Jet9tr0T9KiQpOCyw0H5EtPNWI9TzsPD\\\\\\\/yrwCgSpkfQnmskylkoyMsC8oyiu8AqhFBAGZ+P2OIAFfIUkVmTIN2u6lgGIcFixP1a1fxNzeKTVn9k5XMLUcIwwIiJNJmLHV67y2fmKOGLp6bn5K+7ftKPgvY16bDdY\\\\\\\/L7wUmLIrCOEOkcMrD7jCI\\\\\\\/wiw5vQ5w1NfUWFO\\\\\\\/kpr20Bj4RBkyNzenrKiWW9riwbifE+x0Cq0CTHojXixukwLpGE1fPqm2elA1gu633vUsvDWZJ+oR1QGvCIIFUHHe2k41\\\\\\\/Jgh5LVMuFzpq\\\\\\\/b1RXMVNalGbC13iSgx2ItVBroIssthz8zjiViQC+\\\\\\\/qwalCwww4mFXgGMYwlURx0d0fj\\\\\\\/DGkOV75Mwh9qAD0KulGgNC18cHYPknpMtWSnVnzn9te2xPVAtLPzHYHbqMNikqDMwtc3+crc6gBDMuYvkFU3d68oB76g7Y1\\\\\\\/4MrB8ohzxy+qWAE5H3APxESV\\\\\\\/cL8\\\\\\\/aMQakgBdthyWSClWSB3JkOG2G9ACpgt0N86fkE0eFj0bPgoCyuBCozdvhvgTqvTg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Mv3RzyZnxh23l4XU2j5t5g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00227f80ccacc59a07b3429f6d20fc12e0e592db3cf50b73e2dc716ecccf99bd4196\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/224352v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-74472 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/224352v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--2" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-g1CSYP3SjaqqBBw4YrY-wJTohnicgBOI_2L5HvozedA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="74472" id="node74472" data-pisa="biorxiv;224352v2" data-pisa-master="biorxiv;224352" data-apath="/biorxiv/early/2018/01/05/224352.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">Ecological change points: The strength of density dependence and the loss of history</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-8457-7840" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a><span class="nlm-given-names">José M.</span> <span class="nlm-surname">Ponciano</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mark L.</span> <span class="nlm-surname">Taper</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Dennis</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/224352 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node74472" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">José M. Ponciano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Biology, University of Florida</span>, Gainesville, FL. 32611.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BM.%2BPonciano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ponciano%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BM.%2BPonciano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8457-7840" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José M. Ponciano</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:josemi@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">josemi@ufl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mark L. Taper </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Ecology, Montana State University</span>, Bozeman, MT, 59717</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BL.%2BTaper%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taper%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BL.%2BTaper%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Brian Dennis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-institution'>Department of Fish and Wildlife Sciences and Department of Statistical Science, University of Idaho</span>, Moscow ID 83844-1136, <span class='nlm-country'>USA</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BDennis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dennis%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BDennis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/224352v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:74472" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:74472/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/224352v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:74472" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:74472/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/224352v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:74472" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:74472/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/224352v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:74472" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:74472/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/224352v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:74472" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:74472/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">Abstract</h2><p id="p-2">Change points in the dynamics of animal abundances have extensively been recorded in historical time series records. Little attention has been paid to the theoretical dynamic consequences of such change-points. Here we propose a change-point model of stochastic population dynamics. This investigation embodies a shift of attention from the problem of detecting when a change will occur, to another non-trivial puzzle: using ecological theory to understand and predict the post-breakpoint behavior of the population dynamics. The proposed model and the explicit expressions derived here predict and quantify how density dependence modulates the influence of the pre-breakpoint parameters into the post-breakpoint dynamics. Time series transitioning from one stationary distribution to another contain information about where the process was before the change-point, where is it heading and how long it will take to transition, and here this information is explicitly stated. Importantly, our results provide a direct connection of the strength of density dependence with theoretical properties of dynamic systems, such as the concept of resilience. Finally, we illustrate how to harness such information through maximum likelihood estimation for state-space models, and test the model robustness to widely different forms of compensatory dynamics. The model can be used to estimate important quantities in the theory and practice of population recovery.</p></div><div class="section" id="sec-1"><h2 class="">1. Introduction</h2><p id="p-3">Change more than stasis has perplexed both theoretical and empirical students of ecological time-series. Questions like: “when is a change in dynamics going to occur?”, or “why does a change in dynamics occur?” have preoccupied ecologists working in population dynamics for decades now [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Yet, relatively missing in the theoretical population 5 ecology literature (but see [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]) is an in-depth exploration of how a change in a ecological process, such as density dependence, can drive change in the statistical properties of a (inherently stochastic) dynamic system. How these in turn result in changes with direct management implications is a question of paramount relevance in conservation biology.</p><p id="p-4">The origin of this paper can be traced to a theoretical and empirical study of the population dynamics of a salmonid species by members of the Taper laboratory and colleagues [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. In particular, we were then aiming at modeling the reaction of the population dynamics of a Bull Trout population (Salvelinus confluentus) to a drastic change in the community composition. Such change was noticeable by eye in the trends of the annual counts of adult females (data from the Montana Fish and Wildlife Service): around 1991, a marked drop in abundances consistently occurred in different tributaries of the affected river and lake system. This scenario led us to specify a natural model candidate: a population dynamics model with a change-point. After all, that very same statistical model had long been used in statistical time series modeling. Managers were and are eager for answers to questions like: how were the dynamics before the change affecting the dynamics after the change, and the extent of the change? After so many years fluctuating at low abundances, is the population expected to recover? If so, how long does the recovery process would take?</p><p id="p-5">The strength and effect of density dependence has long been a focus of theoretical and applied population ecology [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>]. Although the dependence of the per capita growth rate of a species on its own density or abundance is now widely regarded as a main driver of population dynamics, such proposition still instills rich debates, theoretical problems and practical dilemmas (see citations in [<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>], [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>], [<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]). Our approach directly links a change in dynamics with this central concept in theoretical ecology.</p><p id="p-6">The purpose of this paper is to show that progress in understanding population trends with a change point does not reside per se in the implementation of statistical methodologies aiming at detecting and quantifying change. Rather (through the analysis of these Bull Trout population trends) we found that hidden within the numerical evaluation of the standard matrix model specification of the change-point auto-regressive moving average model, was an ecologically meaningful mechanistic process. Clear patterns and coherent algebraic structures appeared that showed how the strength of density dependence shapes the post change-point dynamics. These expressions partitioned the variance components of the stochastic population process in a way that it allowed us to directly answer those important managerial questions through changes in the strength of density-dependence. Surprisingly then, parameter estimation turned out to be a simple by-product of the process of scientific inquiry of what a stochastic model was really implying.</p><p id="p-7">We strongly believe that the results presented in this paper are generally important for two reasons: first, these results have been all along within the reach of any statistical analysis of population time series data but have so far been overlooked, hidden within the associated matrix algebra calculations of any statistical time series analysis. Second and most importantly, to our knowledge no other work extracts from routine matrix algebra calculations like we do, the explicit and readily interpretable algebraic forms that allow the analyst to go back and forth between the statistical and numerical phrasing of a change and its ecological interpretation.</p><p id="p-8">Statistical time series models with change points in population dynamics have long been used [<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1</a>] but are nowadays rarely routinely considered in ecological analyses despite being extensively studied in the statistical literature (but see [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]). Asymptotics of maximum likelihood parameter estimates have been derived [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>]. Hypotheses tests for the presence of change points have been proposed [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>, <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. Quality control theory features studies of how to incorporate change point warnings in control charts [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>]. These statistical methods usually test whether an observed time series of population abundances (or densities) depart from a fixed deterministic equilibrium, or from a stochastic stationary distribution. In the first approach, the observed population abundances are observations with measurement error (observation error) added to a deterministic equilibrium or trend. In the second approach, stochastic perturbations are included in the dynamic model to describe the natural fluctuations in abundance. These natural fluctuations are also known as “process noise” [<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>]. With process noise, concepts from deterministic ecological accounts of population regulation take on new forms. For instance, a stable equilibrium of population abundances becomes a stationary probability distribution in stochastic population models [<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>]. Although these statistical studies provide substantial underutilized opportunities for ecological data analysis, missing to date is an ecological understanding of the dynamical properties of populations undergoing a change point.</p><p id="p-9">The last twenty years of research in statistical population dynamics concentrated on parameter estimation, model selection and hypothesis testing of models with and without process and observation error [<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>, <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>]. This research program is warranted because the effective statistical coupling of empirical observations with mathematical models is critical for testing hypotheses concerning population regulation [<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>]. However, the ecological investigation of populations undergoing change has lagged behind the attention to statistical issues [<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>]. This paper seeks to understand what are the ecological consequences of population processes that are not static, but undergo change. To do that, we reveal how the analytical phrasing of a change in a dynamic model translates into readily interpretable and closed-form mathematical changes in the stochastic properties of the population process.</p><p id="p-10">The key idea of this paper is to represent an ecological change point as a saltational change in the parameters of the stochastic population model. Such a change produces both a shift in the stationary distribution of population abundance and a transition to a new equilibrium distribution. We obtain simple formulae characterizing the statistical distribution of the population process during the transition between stationary distributions of abundance. Our study demonstrates that certain historical properties of a population’s growth dynamics are determined by the strength of density-dependent processes. In particular, a time series transitioning from one stationary distribution to another contains information about where the process was before, where is it heading, and how long it is going to take to get there. We illustrate how to harness the information in time series containing transitionary portions via maximum likelihood estimation for state-space models. The model proposed here provides new insights into the role of density dependence in shifting environments. This model also represents a practical tool for detecting and predicting change events in population monitoring.</p></div><div class="section" id="sec-2"><h2 class="">2. The strength of density dependence in discrete time models</h2><p id="p-11">Consider the general discrete time population growth model <em>n</em><sub><em>t</em>+1</sub> = <em>f</em>(<em>n<sub>t</sub></em>) = λ(<em>n<sub>t</sub></em>)<em>n<sub>t</sub></em>, where <em>n<sub>t</sub></em> is the population density or abundance at time <em>t</em> and λ(<em>n<sub>t</sub></em>) is the (density-dependent) per capita growth rate. Assume that <em>f</em>(<em>n</em>), the recruitment map, is continuously differentiable and that <em>n*</em> is its non-trivial equilibrium abundance (<em>i.e.</em> its satisfies f (<em>n*</em>) = <em>n*</em>).</p><p id="p-12">Three measures of the strength of density dependence have been suggested. The first measure is motivated by thinking of the strength of density dependence simply as the marginal effect on the per capita growth rate of an increase in density [<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">10</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>], which according to our general setting, corresponds to <em>∂</em>λ(<em>n<sub>t</sub></em>)/<em>∂n<sub>t</sub></em>. This is a measure of the effects of density at the individual-not population-level that has been used, among other things, to phrase an evolutionary perspective of intra-specific competition as the way individual, demographic traits respond to an increase in density [<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">46</a>]. Because for some discrete, density-dependent maps the per capita growth rate λ(<em>n<sub>t</sub></em>) is written as an exponential function (as in the Ricker or in the Gompertz equations, see [<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">12</a>, <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">11</a>]), the marginal effect of an increase in density can be conveniently measured and plotted in the log-scale of the per capita growth rate by computing <em>∂</em> ln λ(<em>n<sub>t</sub></em>)/<em>∂n<sub>t</sub></em>.</p><p id="p-13">The second measure corresponds to the derivative of the recruitment map <em>f</em>(<em>n</em>) at equilibrium. Such measure can be directly read from the graph of <em>f</em>(<em>n</em>) as a function of n as the value of the slope when <em>f</em>(<em>n</em>) crosses the 1:1 line. It is important to note that a consideration of both an individual and a population wide measure of density dependence is of obvious importance in population ecology to explain persistence, average abundance, bounds on temporal stability and in general, for a careful understanding of the unfolding of population dynamics [<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">46</a>].</p><p id="p-14">A third measure of the strength of density dependence in a simple model with no age structure was given by <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lande et al (2002)</a>, who defined it as the negative elasticity at equilibrium of the per capita population growth rate with respect to change in the population. This amounts to computing <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-1.gif"/></noscript></span></span>.</p><p id="p-15">This definition of the strength of density-dependence is insightful and relevant in population dynamics modeling, not only because it provides a scale-free measure of the change in the per capita growth rate as the (log) population size varies, but also because it is readily extendable to scenarios dealing with more complex life histories [<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a>].</p></div><div class="section" id="sec-3"><h2 class="">3. Methods</h2><div id="sec-4" class="subsection"><h3>3.1. The discrete Gompertz model</h3><p id="p-16">The deterministic skeleton of our stochastic model is the discrete-time Gompertz model [<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">11</a>] given by:
<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-1.gif"/></noscript></span>

</span>
where <em>n<sub>t</sub></em> is population abundance at time <em>t</em> and <em>a</em> and <em>b</em> are constants. Note that many formulations of the Gompertz model explicitly include a minus sign before b and thus can look superficially different.</p><p id="p-17">The non-trivial equilibrium density, or carrying capacity is given by <em>n*</em> = <em>e<sup>−a/b</sup></em>. This equilibrium is asymptotically stable, provided the absolute value of the derivative of the recruitment map (<a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">eq. 1</a>) evaluated at <em>n*</em> is less than 1 [<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>], i.e., if |1 + b| &lt; 1. For this model, the strength of density dependence measured using the per capita growth rate is <em>∂</em> λ(<em>n<sub>t</sub></em>)/<em>∂n<sub>t</sub></em> = <em>be</em><sup><em>a+bln n<sub>t</sub></em>-1</sup> /<em>n<sub>t</sub></em> which is equal to <em>b</em>/ exp{-<em>a/b</em>} at equilibrium. The measure of the strength of density dependence using the logarithm of the per capita growth rate, written as <em>∂</em>ln λ(<em>n<sub>t</sub></em>)/<em>∂n<sub>t</sub></em> = <em>b/n<sub>t</sub></em> is again equal to <em>b</em>/ exp{‒<em>a/b</em>} at equilibrium. Finally, Lande et al.’s negative elasticity is simply given by −<em>b</em> for all population abundances. Then, the density dependent coefficient <em>b</em>, or its carrying capacity-scaled version <em>b</em>/ exp{−<em>a/b</em>}, is directly involved in the definition of the strength of density dependence. So following [<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">47</a>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">49</a>, <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">11</a>], heretofore we identify <em>b</em> as the strength of density dependence for the Gompertz model, due to its role in the rate of approach to a locally stable equilibrium.</p><p id="p-18">Let <em>x<sub>t</sub></em> = ln(<em>n<sub>t</sub></em>) and <em>c</em> = <em>b</em> + 1. Then, Equation (1) becomes the first order difference equation:
<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-2.gif"/></noscript></span>

</span>
whose solution can be found by induction
<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-3.gif"/></noscript></span>

</span>
provided <em>c</em> ≠ 1. If <em>c</em> = 1 then <em>b</em> = 0 and population growth is density independent. The solution in that case is <em>x<sub>t</sub></em> = <em>x</em><sub>0</sub> + <em>at</em>. If density-dependent processes lead to a stable equilibrium density, <em>i.e,</em> if |c| &lt; 1 then we may distinguish two cases: 0 &lt; <em>c</em> &lt; +1 and −1 &lt; <em>c</em> &lt; 0. In both cases, as <em>t</em> → ∞, the log population density (abundance) <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-2.gif"/></noscript></span></span>. In the first case, <em>x<sub>t</sub></em> approaches <em>x*</em> directly, either from above or below, depending on initialization. In the second case, the deterministic map exhibits damped oscillatory behavior while approaching the equilibrium.</p></div><div id="sec-5" class="subsection"><h3>3.2. The stochastic Gompertz model with environmental noise</h3><p id="p-19">A model of Gompertz population growth with environmental stochasticity may be written as
<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-4.gif"/></noscript></span>

</span>
where the population size <em>N<sub>t</sub></em> is now modeled with a random variable representing environmental variation. A useful model for the noise takes the <em>W<sub>t</sub></em> to be independent and identically distributed (<em>iid</em>) log-normal random variables describing heavy-tailed temporal perturbations of the per-capita growth rate [<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">50</a>]. Accordingly, we set <em>W<sub>t</sub></em> = <em>e<sup>E<sub>t</sub></sup></em> where <em>E<sub>t</sub></em> ~ N(0, <em>σ</em><sup>2</sup>) to get
<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-5.gif"/></noscript></span>

</span></p><p id="p-20">On the logarithmic scale, with <em>X<sub>t</sub></em> = ln <em>N<sub>t</sub></em>, this stochastic Gompertz model with environmental noise becomes the well known AR(1) process
<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-6.gif"/></noscript></span>

</span></p><p id="p-21">Provided |<em>c</em>| &lt; 1, this stochastic process has a normal stationary distribution with mean and variance given by
<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-7.gif"/></noscript></span>

</span></p><p id="p-22">Thus, both, the process stationary mean (equivalent to the deterministic stable equilibrium) and its associated variance critically depend on the strength of density dependence. Simply put, although external, environmental variability induces a variability in the growth rate, the magnitude and intensity of the population fluctuations are also influenced by the strength of density dependence. This feature led <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Ives et al (2003)</a> to a detailed study of the concept of ecological stability applied to stochastic community dynamics models. Before moving on to the breakpoint model, we note that through this article, random variables will denoted with capital letters and realizations of these random variables with lower case letters. Also, random vectors like the multivariate random vector <em><strong>X</strong></em> = [<em>X</em><sub>0</sub>, <em>X</em><sub>1;</sub> <em>X</em><sub>2</sub>,…, <em>X<sub>q</sub></em>]′, where the <em>X<sub>i</sub></em> denote the log-population abundance at time <em>i</em>, will be denoted with capital, boldface letters. Realizations of the random vectors will be written with lower case, bold face letters. Finally note that here, as in all the vectors we define, the length of the vector is the last index (<em>q</em> in this case) plus one.</p></div><div id="sec-6" class="subsection"><h3>3.3. The breakpoint Gompertz model</h3><p id="p-23">We consider the case where the stochastic process model undergoes a change after a given time time step, <em>τ</em>. Operationally, such change is implemented by assuming that the value of one or more of the stochastic Gompertz model parameters changes after time <em>τ</em>. A general model, from which all of the possible changes can be specified as sub-models, specifies that there are two different sets of parameters for the stochastic Gompertz model before and after the change-point. If <em>Z<sub>t</sub></em> denote <em>iid</em> N(0,1) random variables, a general change-point model is written as
<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-8.gif"/></noscript></span>

</span></p><p id="p-24">In the log-scale, the model becomes
<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-9.gif"/></noscript></span>

</span></p><p id="p-25">Here, the environmental noise process is written as <em>σ<sub>i</sub>Z<sub>t</sub></em> to show model parameter changes after the breakpoint, as opposed to <em>E<sub>t</sub></em> as above. Also, whether or not <em>t</em> &lt; <em>τ</em> will determine if <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-3.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-4.gif"/></noscript></span></span>, so no additional subscript or notation will be used from now on to distinguish between the environmental noise random variables before and after the breakpoint. Also, as before, we assume that |<em>c</em><sub>2</sub>| &lt; 1. Writing the breakpoint model as in equation 6 is a common practice, for instance, among studies of applications of Random Markov Fields in Biology [<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>]. However, as we show below, writing the breakpoint model in such form, and/or its equivalent matrix representation falls short of a full analysis of the role of the strength of density dependence in shaping the speed of the change and all statistical properties (means, variances and covariances) of the process. After writing the model in its matrix format, we did the explicit matrix algebra multiplications, simplified the expressions and sought out to write the moments of the process as a function of the strength of density dependence. The resulting formulae as well as their ecological interpretation are the main results of this paper. To begin these calculations, by induction (see <a id="xref-app-1-1" class="xref-app" href="#app-1">Appendix 1</a>), we arrive at a matrix equation for the entire random vector of the log abundances before and after the breakpoint:
<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-10.gif"/></noscript></span>

</span>
or
<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-11.gif"/></noscript></span>

</span></p><p id="p-26">Note that vector <strong>Z</strong> is Multivariate Normal (<em>MVN</em>) with mean
<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-12.gif"/></noscript></span>

</span>
and variance-covariance matrix
<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-13.gif"/></noscript></span>

</span>
where the changes in the diagonal occur in the 2<sup>nd</sup> and in the (<em>τ</em> + 2)<sup>th</sup> elements. Because a linear transformation of a multivariate normal distribution is again multivariate normal (Theorem 3.3.5 p. 101 Graybill 1976), it immediately follows that
<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-14.gif"/></noscript></span>

</span>
where the subscript <em>τ</em> indexes the mean and the variance of the process with a change-point, to explicitly differentiate them from the mean <em>μ</em> and variance <em>Σ</em> of the process without a change-point (see <a id="xref-app-1-2" class="xref-app" href="#app-1">Appendix 1</a>).</p></div><div id="sec-7" class="subsection"><h3>3.4. Parameter estimation</h3><p id="p-27">A state space model where sampling error is compounded to the process error can be easily specified, and that is useful for parameter estimation. We call this model the Break-Point Gompertz State-Space (BPGSS) model. Using equation 9 and a lognormal sampling error process (see the discussion of [<a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">7</a>] for a justification of such model), the multivariate distribution of the log-observations with added sampling error <strong>Y = X + F</strong> is
<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-15.gif"/></noscript></span>

</span></p><p id="p-28">This <em>MVN</em> distribution for the observations follows since both, the pdf for the process <em>X</em> and for the observation error <em>F</em> are both <em>MVN</em>, just as in the model of [<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">7</a>]. Parameter estimation via Maximum Likelihood (ML) for this state space model then proceeds by maximizing this multivariate pdf over the model parameters. This pdf can be readily programmed as it is written above, but the computations can be rendered much more efficient when the matrix multiplications in expression (10) are carefully simplified (see Appendices 1 and 2). Sample code for parameter estimation is uploaded as supplementary material and available upon request from JMP.</p></div><div id="sec-8" class="subsection"><h3>3.5. Testing the robustness of the BPGSS model</h3><p id="p-29">Clearly, the BPGSS model is a simplified representation of a population dynamics undergoing a drastic change. One of the most important questions for practicing ecologists is to what extent these simple population dynamics models can be used to infer intrinsic properties of the system that can be of immediate utility. Therefore, we performed a validation effort via simulations that aimed at testing the robustness of the quantitative and qualitative model inferences derived from this model. To do that, we simulated breakpoint dynamics by relaxing most of the major simplifying assumptions made by the BPGSS model and adding biological complexity. Then, the simulated dynamics <em>(i.e.</em> time series) were saved and used to confront the BPGSS model with different ecological scenarios of interest.</p><p id="p-30">The discrete time, discrete state stochastic process used to simulate population dynamics was a Markov process that, besides incorporating sampling error, accounted for demographic stochasticity and different forms of environmental variability. Most of the environmental noise models, like the BPGSS model, incorporate stochasticity in the maximum growth rate. Such noise is multiplicative in the scale of population abundances, and additive in the scale of the logarithmic per capita growth rate. Another possibility, however, is to assume temporal randomness in the strength of density dependence. This approach assumes that the environmental forces shaping the population dynamics affect the intensity of density-dependence rather than the maximum growth rate and to date, remains a rarely explored model alternative (but see [<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>, <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>]).</p><p id="p-31">Our discrete-time Markov model of reproduction and survival followed the general discrete time model construction presented by [<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>] and later revisited and expanded by [<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>] and [<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">53</a>]. A detailed explanation of such construction is given in the supplemental material of [<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">56</a>]. Briefly, population growth from one generation to the other is modeled as a two-stage, stochastic process: at time <em>t</em> the <em>n<sub>t</sub></em> adults present in the population give birth each to a random number of offspring (<em>i.e.</em> 0,1,2,3, …). Then, we assume that both adults and offspring survive to time <em>t</em> + 1 according to a density dependent survival probability, which takes on one of the well-known discrete-time density-dependent maps (Ricker, Hassell, Gompertz, Below, etc…). This reproduction and survival process considers only demographic stochasticity. To include environmental stochasticity, [<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>] and [<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">55</a>] have previously assumed before that the mean of the offspring distribution varies randomly from one time step to the next according to a stochastic environmental process. Such variability mimics the random, temporal fluctuation in the quality of the environment that leads to temporal fluctuations of the average number of offspring produced by every adult. As recently shown by [<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">53</a>], this is but one way of incorporating environmental variability. Here, we assumed that the environment randomly perturbs both the mean of the offspring distribution and the density dependent survival probability. For our simulations, we assumed that the random (over time) environmental process, denoted Λ, had independent and identically distributed realizations from a Gamma(<em>k, α</em>) distribution. The results we present here use the value of the scale parameter α set to 1000 and the shape parameter <em>k</em> set to 1500, but hold for other parameter values as well.</p><p id="p-32">Accordingly, conditional on the realized value λ of the random environmental process, each individual, independently from each other, was assumed to have a Poisson (λ) offspring distribution. As a result, from time <em>t</em> to time <em>t</em> +1, the (conditional) number of potential recruits was distributed Poisson (<em>n<sub>t</sub></em>λ). We then modeled offspring survival with a density dependent, binomial thinning process. The survival probability of the offspring (potential recruits) was taken to be an explicit function of the population size (density) at time <em>t</em>, denoted as <em>n<sub>t</sub></em> by writing it as
<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-16.gif"/></noscript></span>

</span></p><p id="p-33">This function for the offspring survival probability <em>p<sub>t</sub></em> corresponds to Below’s recruitment map [<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>]. The adoption of this function is important because it provides a means to test the generality of our results under different forms of density dependence. Indeed, in this function <em>p<sub>t</sub></em> the parameter <em>β</em> controls the type of compensatory dynamics (under compensatory (<em>β</em> &lt; 1), compensatory (<em>β</em> = 1) and over compensatory (<em>β</em> &gt; 1). Thus, tuning the value of the parameter <em>β</em> effectively changes the shape and curvature of the recruitment map and can recapitulate the behavior of an entire variety of discrete recruitment maps (see [<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">57</a>] for details).</p><p id="p-34">Accounting for binomial survival when the number of potential recruits is Poisson distributed is well known to be a special case of a “randomly stopped sum” and results in a Poisson distributed total number of recruits from one time to the next [<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">54</a>]. Here, the (random) total number of recruits N for the next generation, is denoted as N<sub>t+1</sub>. From the previous assumptions, it follows that N<sub>t+1</sub>|(N<sub>t</sub> = n<sub>t</sub>, Λ = λ) ~ Poisson (λn<sub>t</sub>p<sub>t</sub>). Note again that because the realized value of the environmental process λ modifies the density dependent term in Below’s model, the effect of the stochastic environmental process we specified is qualitatively different from the environmental noise formulation proposed in our BPGSS model. This stochastic model formulation where the stochastic environmental process affects the density-dependent coefficient was recently studied by [<a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">53</a>]. Introducing this type of variability changes the well-known scaling relation between the population variance and population abundances in stochastic growth models, and may be important in many animal populations [<a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">53</a>]. Averaging the environmental process over time results in the marginal transition pdf of the process. This transition, however, does not result in the well-known demographic and environmental negative binomial process [<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">54</a>], because of the complex environmental process effects.</p><p id="p-35">To incorporate different density-dependent scenarios, we used three values for <em>β</em>: 0.5, 1 and 1.2. These values represent respectively under-compensatory, compensatory and over-compensatory dynamics. This axis of variation allowed us to test the robustness to different forms of density dependence. The simulations were started at the value of the deterministic skeleton’s carrying capacity in the log-scale (14) and a change point of dynamics was introduced at time <em>t</em> = 40. The change consisted of a change in the deterministic skeleton carrying capacity, from 14 to 4.48 (in the log-scale). Although we introduced a single change at a single time point, for each value of <em>β</em> it took some time for the time series to “stabilize” around the second deterministic carrying capacity. The time it takes for the average of the process to reach the arithmetic average of the two (log) deterministic carrying capacities is heretofore referred to as the “half-life of the change”. The resulting simulated population dynamics was sampled 1000 times per value of <em>β</em> using a multiplicative lognormal sampling error with parameters μ = 0 and ν<sup>2</sup> = 0.2315. This sampling resulted in 1000 time series of length 200 per value of <em>β</em> that were taken as data sets to which the BPGSS model was fitted. Furthermore, for each value of <em>β</em>, the “true” half-life of the change point was computed numerically as the average half-life in 50000 simulated time series. We then computed the relative bias of our half-life estimator for each one of the 1000 simulated time series to which the BPGSS model was fitted.</p></div></div><div class="section" id="sec-9"><h2 class="">4. Results</h2><div id="sec-10" class="subsection"><h3>Understanding the breakpoint dynamics</h3><p id="p-36">The vector expressions in the multivariate normal distribution of the log-normal abundances (<a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">eq. 9</a>) conceal explicit, simple expressions linking the dynamics before and after the change. In particular, immediately after the change, there is a transitional period. The time-varying distribution of this transition contains information about both, the past stationary mean and variance, as well as the future stationary mean and variance. This information is exposed by partitioning the vector X of log-population abundances over time, into a vector of pre-breakpoint log-abundances <strong>X<sub>1</sub></strong> = [<em>X</em><sub>0</sub> <em>X</em><sub>1</sub> <em>X</em><sub>2</sub> … <em>X</em><sub>τ</sub>]′ and another of post-breakpoint log-abundances <strong>X<sub>2</sub></strong> = [<em>X</em><sub><em>τ</em>+1</sub> <em>X</em><sub><em>τ</em>+2</sub> … <em>X</em><sub>τ+q</sub>]′. In what follows, we state six main results along with three corollaries that follow from this partitioned structure. These results follow from a detailed algebraic examination of the vector and matrix calculations resulting from the model formulation in equations 7-9. These algebraic examinations, necessary to fully understand the results, are presented in appendices 1 and 2. Each one of these results brings a different dimension to the understanding of the transitional dynamics in ecological terms. At the end, we also list the result regarding the robustness tests of the BPGSS model.</p><div id="statement-1" class="statement"><p id="p-37"><strong>Result 1:</strong> <em>In transition, the mean of the (log) population process becomes a weighted average of the stationary mean before the breakpoint, and the stationary mean predicted by the new set of dynamical parameters. The weights of this weighted average are given by <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-5.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-6.gif"/></noscript></span></span> respectively, where t is the number of time steps after the break.</em></p></div><p id="p-38">The result follows directly from the expression for the mean of the multivariate distribution in <a id="xref-disp-formula-14-2" class="xref-disp-formula" href="#disp-formula-14">eq. 9</a>. By developing and explicitly writing the entries of the vector and matrix calculations in this expression, we found that the mean vector of the first partition <strong>X<sub>1</sub></strong> is given by <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-7.gif"/></noscript></span></span>, where <strong>j</strong><sub>(τ+1)×1</sub> is a vector of ones, whereas the mean of the second partition is given by the time-varying weighted average:
<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-17.gif"/></noscript></span>

</span></p><p id="p-39">The first element in the vector above (<a id="xref-disp-formula-17-1" class="xref-disp-formula" href="#disp-formula-17">eq. 11</a>) is the mean of the transition process one time step after the breakpoint. This element is a weighted average of the first stationary mean <em>μ</em><sub>1</sub> = <em>a</em><sub>1</sub>/(1 – <em>c</em><sub>1</sub>) and the second stationary mean <em>μ</em><sub>2</sub> = <em>a</em><sub>2</sub>/(1 – <em>c</em><sub>2</sub>). The weights are given by the strength of density-dependence after the breakpoint, <em>c</em><sub>2</sub>. Thus, this coefficient completely determines how important the mean of the first phase of the process is right after the change. Two time-steps after the breakpoint, the weights are given by <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-8.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-9.gif"/></noscript></span></span> respectively. Again, because <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-10.gif"/></noscript></span></span> and hence, the influence of the first stationary means decays two time steps after the breakpoint. <em>t</em> time units after the breakpoint, the weights become <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-12.gif"/></noscript></span></span>. As a result, as the number of time steps <em>t</em> after the breakpoint grows large, the weight for the first stationary mean goes to 0 while the weight of the second stationary mean converges to 1. It then follows that the mean of the post-breakpoint process truly represents a transitionary trend.</p><div id="statement-2" class="statement"><p id="p-40"><strong>Result 2:</strong> <em>The speed of the change of the mean of the log-population process (<a id="xref-disp-formula-14-3" class="xref-disp-formula" href="#disp-formula-14">eq. 9</a>) is determined by the strength of density dependence. The half life of the change in means, or the average time <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-13.gif"/></noscript></span></span> it takes for the average of the process to reach (<em>μ</em><sub>1</sub> + <em>μ</em><sub>2</sub>)/2 from <em>μ</em><sub>1</sub>, is a quantity that directly depends on the strength of density dependence post breakpoint. Thus, any property of this dynamic system that depends on the waiting time to complete the transition, also depends on the strength ofdensity dependence.</em></p></div><p id="p-41">An explicit expression for the half life of the transition in means is readily found by setting
<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-18.gif"/></noscript></span>

</span>
and solving for <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-14.gif"/></noscript></span></span>, which yields
<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-19.gif"/></noscript></span>

</span></p><p id="p-42">Thus, the speed of the change from one stationary mean to the other one is determined by the strength of density dependence after the breakpoint. For example, if <em>c</em><sub>2</sub> is close to 0, then the transition weights <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-16.gif"/></noscript></span></span> (see <a id="xref-disp-formula-17-2" class="xref-disp-formula" href="#disp-formula-17">eq. 11</a>) converge fast to 0 and 1, respectively and the process approaches the second stationary mean faster. A detailed characterization of the speed of the change as a function of <em>c</em><sub>2</sub> is given in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figs. 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">2</a>. From this characterization it is readily apparent that when 1/2 &lt; <em>c</em><sub>2</sub> &lt; 1 the strength of density dependence can be thought as weak; strong when −1/2 &lt; <em>c</em><sub>2</sub> &lt; 1/2 and as very strong when −1 &lt; <em>c</em><sub>2</sub> &lt; −1/2 (see legend in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Fig. 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The stochastic Gompertz breakpoint process and the half life of the change in the mean of the dynamical process, . Plotted are 5 realizations of the Stochastic Gompertz breakpoint process. Dotted lines mark the process mean before the breakpoint (&#x3BC;1 = a1/(1 &#x2013; c1), the mean after the breakpoint (&#x3BC;2 = a2/(1 &#x2013; c2), and the arithmetic average of both means . The time at which such arithmetic average is reached is the half life of the process, . The upper panel shows a case where the mean changes to a smaller size whereas the lower panel depicts the change from a lower equilibrium size to a higher one. This second case could depict a population recovery scenario, in which case the half life is half the mean time to recovery." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1608624292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The stochastic Gompertz breakpoint process and the half life of the change in the mean of the dynamical process, . Plotted are 5 realizations of the Stochastic Gompertz breakpoint process. Dotted lines mark the process mean before the breakpoint (&lt;em&gt;&#x3BC;&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; = &lt;em&gt;a&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;/(1 &#x2013; &lt;em&gt;c&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;), the mean after the breakpoint (&lt;em&gt;&#x3BC;&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = &lt;em&gt;a&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/(1 &#x2013; &lt;em&gt;c&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;), and the arithmetic average of both means . The time at which such arithmetic average is reached is the half life of the process, . The upper panel shows a case where the mean changes to a smaller size whereas the lower panel depicts the change from a lower equilibrium size to a higher one. This second case could depict a population recovery scenario, in which case the half life is half the mean time to recovery.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F1.medium.gif" width="326" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F1.medium.gif" width="326" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1:</span> <p id="p-43" class="first-child">The stochastic Gompertz breakpoint process and the half life of the change in the mean of the dynamical process, <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F1/embed/inline-graphic-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F1/embed/inline-graphic-17.gif"/></noscript></span></span>. Plotted are 5 realizations of the Stochastic Gompertz breakpoint process. Dotted lines mark the process mean before the breakpoint (<em>μ</em><sub>1</sub> = <em>a</em><sub>1</sub>/(1 – <em>c</em><sub>1</sub>), the mean after the breakpoint (<em>μ</em><sub>2</sub> = <em>a</em><sub>2</sub>/(1 – <em>c</em><sub>2</sub>), and the arithmetic average of both means <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F1/embed/inline-graphic-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F1/embed/inline-graphic-18.gif"/></noscript></span></span>. The time at which such arithmetic average is reached is the half life of the process, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F1/embed/inline-graphic-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F1/embed/inline-graphic-19.gif"/></noscript></span></span>. The upper panel shows a case where the mean changes to a smaller size whereas the lower panel depicts the change from a lower equilibrium size to a higher one. This second case could depict a population recovery scenario, in which case the half life is half the mean time to recovery.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Half life of the change in the mean of the dynamics, , plotted as a function of the strength of density dependence c2. If the strength of density-dependence after the breakpoint c2 is close to 1 (i.e. b is close to 0 and hence growth is close to density independence), then change occurs slowly. As c2 approaches 0.5 from the right, the speed of the change increases. When c2 = 1/2, the change occurs within one time step (e.g. a year). If c2 is exactly 0, then the change is immediate. Between 0 and &#x2013;1/2 the change still occurs within a single time step, but weak damped oscillations towards the equilibrium are produced. Finally, when &#x2013;1 &lt; c2 &lt; &#x2013;1/2, the speed of the change is slow again, with the added complexity that such change is accompanied with strong, damped oscillations towards the equilibrium. Thus, the non-stationary trend contains information about a) the location of the past stationary mean, b) the location of the future stationary mean, and c) the duration of the transitionary period." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1608624292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Half life of the change in the mean of the dynamics, , plotted as a function of the strength of density dependence &lt;em&gt;c&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;. If the strength of density-dependence after the breakpoint &lt;em&gt;c&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; is close to 1 (&lt;em&gt;i.e. b&lt;/em&gt; is close to 0 and hence growth is close to density independence), then change occurs slowly. As &lt;em&gt;c&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; approaches 0.5 from the right, the speed of the change increases. When &lt;em&gt;c&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; = 1/2, the change occurs within one time step (&lt;em&gt;e.g.&lt;/em&gt; a year). If &lt;em&gt;c&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; is exactly 0, then the change is immediate. Between 0 and &#x2013;1/2 the change still occurs within a single time step, but weak damped oscillations towards the equilibrium are produced. Finally, when &#x2013;1 &lt; &lt;em&gt;c&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; &lt; &#x2013;1/2, the speed of the change is slow again, with the added complexity that such change is accompanied with strong, damped oscillations towards the equilibrium. Thus, the non-stationary trend contains information about &lt;em&gt;a)&lt;/em&gt; the location of the past stationary mean, &lt;em&gt;b)&lt;/em&gt; the location of the future stationary mean, and &lt;em&gt;c)&lt;/em&gt; the duration of the transitionary period.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F2.medium.gif" width="308" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F2.medium.gif" width="308" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2:</span> <p id="p-44" class="first-child">Half life of the change in the mean of the dynamics, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F2/embed/inline-graphic-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F2/embed/inline-graphic-20.gif"/></noscript></span></span>, plotted as a function of the strength of density dependence <em>c</em><sub>2</sub>. If the strength of density-dependence after the breakpoint <em>c</em><sub>2</sub> is close to 1 (<em>i.e. b</em> is close to 0 and hence growth is close to density independence), then change occurs slowly. As <em>c</em><sub>2</sub> approaches 0.5 from the right, the speed of the change increases. When <em>c</em><sub>2</sub> = 1/2, the change occurs within one time step (<em>e.g.</em> a year). If <em>c</em><sub>2</sub> is exactly 0, then the change is immediate. Between 0 and –1/2 the change still occurs within a single time step, but weak damped oscillations towards the equilibrium are produced. Finally, when –1 &lt; <em>c</em><sub>2</sub> &lt; –1/2, the speed of the change is slow again, with the added complexity that such change is accompanied with strong, damped oscillations towards the equilibrium. Thus, the non-stationary trend contains information about <em>a)</em> the location of the past stationary mean, <em>b)</em> the location of the future stationary mean, and <em>c)</em> the duration of the transitionary period.</p><div class="sb-div caption-clear"></div></div></div><div id="statement-3" class="statement"><p id="p-45"><strong>Result 3:</strong> <em>The covariance between a population abundance before the breakpoint and after the breakpoint decays geometrically over time in powers of c<sub>2</sub> to the right of the breakpoint and in powers of c<sub>1</sub> to the left of the breakpoint. The intensity of such covariance is always proportional to the variance of the stationary distribution of the first partition.</em></p></div><p id="p-46">The partitioned representation of the log-population process X is completed by writing its variance covariance matrix as a block matrix:
<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-22.gif"/></noscript></span>

</span>
where the diagonal matrices <strong>Σ<sub><em>ii</em></sub></strong>, <em>i</em> = 1, 2 are the covariance matrices for the pre and post-breakpoint elements of <strong>X</strong> respectively. Accordingly, the sub-matrix <strong>Σ<sub>11</sub></strong> specifies the variance of the process at every time point before the breakpoint and the covariances among these points. Likewise, the sub-matrix <strong>Σ<sub>22</sub></strong> specifies the variances at every point after the breakpoint, along with the covariances among these points. As we will see below, such variances and covariances also bear information about the process before the breakpoint. Finally, the sub-matrix <strong>Σ<sub>21</sub></strong>, which is equal to <strong>Σ<sub>12</sub>′</strong>, specifies the covariance Cov(<strong>X<sub>2</sub></strong>, <strong>X<sub>1</sub></strong>), between any time point before the breakpoint and any point after it. Remarkably, these unwieldy expressions reduce to simple mathematical forms that can be easily interpreted. To find these explicit expressions, we first recall that <strong>Σ<sub><em>τ</em></sub> = ΓΦΓ′</strong>, and thus that it is convenient to also partition the matrices <strong>Γ</strong> and <strong>Φ</strong> into four sub-matrices matching the dimensions of the Σ<sub><em>τ</em></sub> blocks. That is,
<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-23.gif"/></noscript></span>

</span></p><p id="p-47">Next, noting that <strong>Γ<sub>12</sub> = 0</strong>, the variance-covariance of the process can be re-written as
<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-24.gif"/></noscript></span>

</span></p><p id="p-48">Elementary matrix multiplications and an examination of the power patterns (see <a id="xref-app-2-1" class="xref-app" href="#app-2">Appendix 2</a>) leads to a simple expression of the elements of the matrix <strong>Σ<sub>12</sub> = Γ<sub>11</sub>Φ<sub>11</sub>Γ<sub>21</sub>′</strong>, which represents the covariance between any process realization before the breakpoint <em>X</em><sub>0</sub>, <em>X</em><sub>1</sub>,…, <em>X</em><sub>τ</sub>, and any process realization after the breakpoint, <em>X</em><sub>τ+1</sub>, <em>X</em><sub><em>τ</em>+2</sub>,…, <em>X<sub>τ+q</sub></em>:
<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-25.gif"/></noscript></span>

</span></p><p id="p-49">Thus, these covariances are proportional to the variance of the stationary distribution of the first partition and their strength increases from the left of the breakpoint in powers of <em>c</em><sub>1</sub>, and to the right of such point, it decreases in powers of <em>c</em><sub>2</sub>.</p><div id="statement-4" class="statement"><p id="p-50"><strong>Result 4:</strong> <em>The variance-covariance matrix <strong>Σ22</strong> can be decomposed into two variance component matrices. An examination of these components reveals exact analytical expressions detailing information about the past and the future stationary distribution of the population process concealed in the post-breakpoint time series.</em></p></div><p id="p-51">While the sub-matrix <strong>Σ<sub>11</sub></strong> is identical in form to the variance covariance matrix of the process without a breakpoint (<a id="xref-disp-formula-38-1" class="xref-disp-formula" href="#disp-formula-38">eq. A.6</a>, except that with dimensions (<em>τ</em> + 1) × (<em>τ</em> + 1)), the sub-matrix <strong>Σ<sub>22</sub></strong> is the sum of two matrices, <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-21.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-22.gif"/></noscript></span></span>.</p><div id="statement-5" class="statement"><p id="p-52"><strong>Result 4.1</strong> <em>The first variance component of the matrix <strong>Σ<sub>22</sub></strong>, given by <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-23.gif"/></noscript></span></span>, is proportional to the variance of the stationary distribution in the first partition. The weight of such stationary variance decays over time, in powers of <em>c</em><sub>2</sub>.</em></p></div><p id="p-53">After performing the matrix multiplications in the right hand side (RHS) of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-24.gif"/></noscript></span></span> and simplifying the resulting expressions (see <a id="xref-app-2-2" class="xref-app" href="#app-2">Appendix 2</a>) the elements of <strong>Σ<sub>22</sub><sup>(1)</sup></strong> are found to be:
<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-26.gif"/></noscript></span>

</span></p><p id="p-54">Thus, the first variance component of the second time series partition is proportional to the variance of the stationary distribution in the first partition, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-25.gif"/></noscript></span></span>. The influence of such stationary variance decays over time, in powers of the second dynamics’ density-dependent effect, <em>c</em><sub>2</sub>.</p><div id="statement-6" class="statement"><p id="p-55"><strong>Result 4.2</strong> <em>The second variance component is proportional to the stationary variance of the second partition. These covariances measure the departure of the variance-covariance matrix of the second partition from its limiting form at stationarity. The magnitude of the departure from the limiting covariances is controlled by the strength of density dependence in the second partition, modulated by the positioning of the observations relative to each other and to the location of the breakpoint.</em></p></div><p id="p-56">The elements of the second matrix <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-26.gif"/></noscript></span></span> are given by
<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-27.gif"/></noscript></span>

</span></p><p id="p-57">To understand this formula, note that <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-27.gif"/></noscript></span></span> is the covariance between two observations |<em>i</em> – <em>j</em>| time units apart, had the second partition arisen directly from the stationary distribution of the process parameterized with the second set of parameters. Thus, this second matrix is a measure of the departure of the variance-covariance matrix of the second partition from what the variance-covariance matrix would be if the process, right after the breakpoint, had been drawn from the second stationary distribution. Thus, it follows that the matrix <strong>Σ<sub>22</sub></strong> can be expressed as a sum of two variance components, one directly related with the stationary variance of the process from the first partition, and another measuring the divergence from stationarity in the second partition in the variance-covariance space. In the particular case when <em>i = j</em> (<em>i.e.</em> for the diagonal elements of <strong>Σ<sub>22</sub></strong>), this sum gives the variance of the elements after the second partition. This brings us to our result:</p><div id="statement-7" class="statement"><p id="p-58"><strong>Result 4.3</strong> <em>The variances of the log-observations after the breakpoint (the diagonal elements of the variance-covariance matrix <strong>Σ<sub>22</sub></strong>) can be written as a weighted average of the stationary distribution before and after the breakpoint.</em></p></div><p id="p-59">The developed representation of the transitional variances using the variance decomposition from “Result 4.1 and 4.2” turns out to be analogous to the the equation for the transtional means (<a id="xref-disp-formula-17-3" class="xref-disp-formula" href="#disp-formula-17">eq. 11</a>). Specifically, these variances can be written as a weighted average of the variance of the stationary distribution before and after the breakpoint:
<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-28.gif"/></noscript></span>

</span></p><p id="p-60">The half life of the change in variance, denoted <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-28.gif"/></noscript></span></span> and found by setting
<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-29.gif"/></noscript></span>

</span>
and solving for <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-29.gif"/></noscript></span></span> is given by
<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-30.gif"/></noscript></span>

</span></p><p id="p-61">Therefore, the variance of the process transitions twice as fast as the mean of the process from one dynamical scenario to the next. A depiction of these formulae accompanied with matching empirical trends is shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3</a></p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The theoretical and empirical means and variances of the breakpoint Stochastic Gompertz model. In the upper left and lower left panels, the theoretical mean and variance (grey) of the breakpoint process is depicted. Overlaid in the upper panel are 5 simulated trajectories and in blue is the empirical average of 10000 simulated trajectories. The empirical average of the variances of the same 10000 trajectories is depicted in blue in the lower left panel. As stated in the results, both the mean and the variance of the process after a change are a weighted average of the stationary means and variances before and after the change. The upper and lower right panels show how the weights in these weighted averages depend on the strength of density-dependence after the change (See result 1). In particular, as stated in Result 4.3, note that the weights of the variance change twice as fast as the changes in the mean (see for instance [36])" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1608624292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The theoretical and empirical means and variances of the breakpoint Stochastic Gompertz model. In the upper left and lower left panels, the theoretical mean and variance (grey) of the breakpoint process is depicted. Overlaid in the upper panel are 5 simulated trajectories and in blue is the empirical average of 10000 simulated trajectories. The empirical average of the variances of the same 10000 trajectories is depicted in blue in the lower left panel. As stated in the results, both the mean and the variance of the process after a change are a weighted average of the stationary means and variances before and after the change. The upper and lower right panels show how the weights in these weighted averages depend on the strength of density-dependence after the change (See result 1). In particular, as stated in Result 4.3, note that the weights of the variance change twice as fast as the changes in the mean (see for instance [36])&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F3.medium.gif" width="322" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F3.medium.gif" width="322" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3:</span> <p id="p-62" class="first-child">The theoretical and empirical means and variances of the breakpoint Stochastic Gompertz model. In the upper left and lower left panels, the theoretical mean and variance (grey) of the breakpoint process is depicted. Overlaid in the upper panel are 5 simulated trajectories and in blue is the empirical average of 10000 simulated trajectories. The empirical average of the variances of the same 10000 trajectories is depicted in blue in the lower left panel. As stated in the results, both the mean and the variance of the process after a change are a weighted average of the stationary means and variances before and after the change. The upper and lower right panels show how the weights in these weighted averages depend on the strength of density-dependence after the change (See result 1). In particular, as stated in Result 4.3, note that the weights of the variance change twice as fast as the changes in the mean (see for instance [<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">36</a>])</p><div class="sb-div caption-clear"></div></div></div><div id="statement-8" class="statement"><p id="p-63"><strong>Result 5:</strong> <em>The variance-covariance matrix of the observations <strong>Y</strong>, the process with added sampling error, is written as the sum of the variance-covariance of the process and the variance-covariance matrix of the observation error model.</em></p></div><p id="p-64">According to the sampling error setting (see <em>Parameter Estimation</em> subsection above),
<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-32.gif"/></noscript></span>

</span></p><p id="p-65">From this development, it follows that the log-likelihood needed for parameter estimation accounting for a change-point in the dynamics occurring between times <em>τ</em> and <em>τ</em> + 1 is simply the probability density function (pdf) of such MVN distribution evaluated at the observations:
<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-33.gif"/></noscript></span>

</span></p><p id="p-66">A by-product of this multivariate normal formulation is that the problem of estimation in the presence of missing observations in the time series is readily solved (see Dennis and Ponciano 2014): suppose that a subset <em>s</em> of elements of the vector <strong>Y</strong> is missing. Then, the likelihood function of the remaining observations is simply written as in <a id="xref-disp-formula-30-1" class="xref-disp-formula" href="#disp-formula-30">eq. 21</a> but by removing the <em>s</em><sup>th</sup> elements in the mean vector <strong><em>μ<sub>τ</sub></em></strong> and the <em>s</em><sup>th</sup> rows and columns of the matrix <strong>V<sub>τ</sub></strong>.</p><div id="statement-9" class="statement"><p id="p-67"><strong>Result 6:</strong> <em>Despite our model being a simplified representation of a change in population dynamics, statistical inferences from this model are robust to biologically realistic departures of the model assumptions.</em></p></div><p id="p-68">After simulating from a model with both, demographic and environmental stochasticities as well as sampling error, the half-life estimates obtained from fitting the BPGSS model appear reliable and with almost no bias, even at the most extreme level of simulated compensatory dynamics (see <a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4</a>). The computer code in <strong>R</strong> used to do the simulations, parameter estimation and plotting of each one of the figures is provided in the first author (<strong>JMP</strong>)’s laboratory web page.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Testing the robustness of the breakpoint Stochastic Gompertz model. A biologically realistic change-point stochastic process was used to test the robustness of the Gompertz model to unaccounted biological complexities. Accordingly, for three different types of compensatory dynamics, 1000 simulations from a population growth model with demographic variability, demographic noise, sampling error and a change-point were used to estimate the parameters of the BPSG model with added sampling error, along with the half life of the change. In the figure, each row, from left to right, shows the type of compensatory dynamics used in the simulations (left panels), 10 samples of the simulated time series of length 200 with the true half life  (center panels and solid vertical line) and the estimated half life (dotted vertical line), and a box plot of the 1000 half life estimates (right panels). For the compensatory and over compensatory dynamics, only the first 75 time steps of the simulated time series are shown for clarity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1608624292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Testing the robustness of the breakpoint Stochastic Gompertz model. A biologically realistic change-point stochastic process was used to test the robustness of the Gompertz model to unaccounted biological complexities. Accordingly, for three different types of compensatory dynamics, 1000 simulations from a population growth model with demographic variability, demographic noise, sampling error and a change-point were used to estimate the parameters of the BPSG model with added sampling error, along with the half life of the change. In the figure, each row, from left to right, shows the type of compensatory dynamics used in the simulations (left panels), 10 samples of the simulated time series of length 200 with the true half life  (center panels and solid vertical line) and the estimated half life (dotted vertical line), and a box plot of the 1000 half life estimates (right panels). For the compensatory and over compensatory dynamics, only the first 75 time steps of the simulated time series are shown for clarity.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F4.medium.gif" width="360" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F4.medium.gif" width="360" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2018/01/05/224352/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4:</span> <p id="p-69" class="first-child">Testing the robustness of the breakpoint Stochastic Gompertz model. A biologically realistic change-point stochastic process was used to test the robustness of the Gompertz model to unaccounted biological complexities. Accordingly, for three different types of compensatory dynamics, 1000 simulations from a population growth model with demographic variability, demographic noise, sampling error and a change-point were used to estimate the parameters of the BPSG model with added sampling error, along with the half life of the change. In the figure, each row, from left to right, shows the type of compensatory dynamics used in the simulations (left panels), 10 samples of the simulated time series of length 200 with the true half life <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F4/embed/inline-graphic-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/F4/embed/inline-graphic-30.gif"/></noscript></span></span> (center panels and solid vertical line) and the estimated half life (dotted vertical line), and a box plot of the 1000 half life estimates (right panels). For the compensatory and over compensatory dynamics, only the first 75 time steps of the simulated time series are shown for clarity.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-11"><h2 class="">5. Discussion</h2><p id="p-70">In this study, we show that after a saltational change in population dynamics’ parameters triggered by a change in environmental conditions, the transition to new equilibrial conditions is governed by the strength of density dependence. In response to a change in environmental conditions, the stationary probability distribution of a species’ population size undergoes a transition and asymptotically approaches a new equilibrium distribution. The statistical properties, means, variances and covariances, respond to change in different ways (Results 1 through 6 above). Remarkably, in all of them it is density dependence what governs the speed of change. In all of these statistical properties, the history of the past persists and the future is anticipated as an explicit function of the strength of density dependence.</p><p id="p-71">According to our second result, the strength of density dependence controls the average of a first passage time, the average time it takes to complete the transition from one stationary mean to the other. This result automatically connects the strength of density dependence with any theoretical dynamic concept. Take for instance the concept of “potential functions”. For continuous time, deterministic population dynamics models of the form dn/dt = m(n), the potential function is defined as the function <em>u</em>(<em>n</em>) satisfying <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-31.gif"/></noscript></span></span> [<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>]. This function is deeply connected with deterministic waiting times. As Dennis et al [<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">58</a>] point out, the waiting time <em>t</em>(<em>n</em>) needed to reach abundance n from an initial abundance <em>x</em> can be written as a function of the potential function as follows
<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-35.gif"/></noscript></span>

</span></p><p id="p-72">Thus, the formulation in equation 22 aids ecological interpretation because it gives to the potential function a direct connection with <em>t</em>(<em>n</em>).</p><p id="p-73">The famous “marble in a bowl” conceptual image has been extremely fruitful in Ecology to explain classical dynamic ideas like resilience [<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>]. The potential function <em>u</em>(<em>n</em>) is the mathematical representation of the “bowl” in such image. Just as the potential function is connected to deterministic waiting times (<a id="xref-disp-formula-31-1" class="xref-disp-formula" href="#disp-formula-31">equation 22</a>), one could expect our average first passage times, that depend on the strength of density dependence, to be connected with a stochastic formulation of the potential well. In fact, as depicted in ecological textbooks, the “bowl” is often shaken (<em>e.g.</em> Figure 6.15 in [<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">59</a>]). This stochastic formulation would then be the mathematical representation of dynamics under such perturbations. Importantly, following our results this representation of a stochastic well will not only depend on the strength of density-dependence but also on the variance properties of the system, which we have concisely described in this paper. We are developing a full treatment of the stochastic potential well in a separate manuscript.</p><p id="p-74">Despite our model being a simplified representation of a change in population dynamics, statistical inferences from this model are robust to biologically realistic departures of the model assumptions. Indeed, here we show that the inference via ML seems to be robust to the inclusion of an atypical yet realistic form of environmental noise: temporal variability in the strength of density dependence, recently shown to be important in animal populations [<a id="xref-ref-53-6" class="xref-bibr" href="#ref-53">53</a>] (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4</a>).This robustness confers the model the generality that is needed for sound hypothesis-driven learning in population dynamics modeling. After simulating from a model with both, demographic and environmental stochasticities as well as sampling error and notably, multiple forms of density-dependence, the half-life estimates obtained from fitting the BPGSS model appear reliable and with almost no bias, even at the most extreme level of simulated compensatory dynamics (see <a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4</a>). These results seem to support the early result of [<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>] who found that the growth rate parameterization of the theta-logistic model that was fairly common across a wide taxonomic spectrum was basically equivalent to the Gompertz model parameterization. Providing a mechanistic support for this pattern is an interesting topic for further research using ecological and evolutionary arguments (see for instance [<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">61</a>]).</p><p id="p-75">Bias in ML estimates of population dynamic models has been previously reported and remedied using REML estimation. In our applied conservation biology work (not shown here) using the GSS model, we have successfully applied the REML bias reduction technique (for an introduction to this topic in population dynamics models see [<a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">7</a>]). A full exploration of this bias-reduction issue warrants an independent research study. Therefore, we believe that after sufficient investigation and testing our model may turn out to be sufficient for understanding, management and importantly, prediction. The latter is the focus of our ongoing Population Viability Monitoring (VPM) programatic research [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>].</p><p id="p-76">A more complete characterization through simulation of the statistical properties of the estimators of this model is in progress and will be reported elsewhere. In particular, a computational study of the properties of the change point parameter estimate and of the biological parameters as a function of the length of the time series before and after the break are still needed. Given the covariance structure of the model’s second partition <em>(i.e.</em> the time series post-change point), we expect the length of the first partition, as well as the size of the change to be critical to be able to correctly estimate the covariance structure in the second partition. In that respect, the location of the change point within the time series is expected to play a critical role. We also expect that these statistical studies will reveal a suitable approximation for the usually less biased restricted maximum likelihood (REML) estimators. In any case, however, if a change point is suspected to be present in a particular data set, extensive statistical tests should be carried to confront our change-point model to a null model of no change, and possibly, to other types of dynamics (see for instance [<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">62</a>]). Interestingly, because the covariance structure of the second partition also contains information about the parameters in the first partition, one could envision estimating the pre-breakpoint parameters even if only post-breakpoint time series data immediately after the breakpoint is available. An immediate application of our results and observations is the study of the recovery process, or the case where the change is from a lower stationary distribution to a (potentially) higher stationary density (see lower panel of <a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1</a>). The explicit changes in variances and covariances of the process could then be directly linked to changes in, for instance, pseude-extinction dynamics.</p><p id="p-77">The principles driving the inferences from these models are applicable to a wide array of systems, from macroscopic to microscopic scales. The advent of DNA-based technology, for instance, has made it possible to explore the population dynamics of entire microbial communities, whose signature trends exhibit sudden changes. Being able to achieve a process-based understanding of such changes is at the center of very active research in microbial ecology [<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>], because different microbial community compositions and structures are associated with different pathological states of the hosts. It follows that understanding the drivers of the “stability” of the community composition is key to understand how to manage changes in a microbial community. Our model can be readily extended to a multi-species autoregressive format (e.g. [<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">36</a>]) that can serve as the basis to move the research from the study of patterns of change in abundance to the study of patterns of change in the nature and intensity of ecological interactions. Such move would make readily accessible an ecological, process-based understanding of change. Much recent work in ecology and other scientific fields aims at developing statistical tests to anticipate changes of state [<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>]. One of the latest examinations of the problem [<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">65</a>] emphasizes the importance of model-based approaches to find reliable indicators of impending regime shifts. In this paper, we shift the attention from the problem of detecting when a change will occur, to another non-trivial puzzle: understanding and predicting the post-breakpoint behavior of the population dynamics. In this work we show that population time series transitioning from one stationary distribution to another contain explicit information about where the process was before the change-point, where is it heading and how long it will take to transition. We view the current contribution’s explicit modeling as a step in understanding the relationship between stochasticity, density dependence, population dynamics shifts, and the influence of the past dynamics on future dynamics.</p><p id="p-78">This paper has begun the study of the effects on natural population of environmentally induced changes from the simplest manifestation of change −a single saltational break in parameter values. Our experience with many wildlife time-series have led us to believe that, while such sharp changes are not uncommon, they are certainly not universal. This observation raises several questions. How can we investigate the effects of more gradual change? And, how can we distinguish saltational parameter change from more gradual shifts?</p><p id="p-79">Fortunately, the formulations of this paper can be immediately extended to by considering every time step as a potential saltational change as in [<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">66</a>]. As full likelihood estimates will be available for any modelable transition of parameter values, identifying the best model of change can be accomplished using information criteria. This also is material for upcoming research.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-80">JMP and MLT were funded by both NIH R01-GM103604 and by a Montana Fish, Wildlife, and Parks contract 060327 to M. L. Taper.</p></div><h2 class="">AppendixA</h2><div class="section app" id="app-1"><h2 class="">AppendixA.1. Matrix formulation of the breakpoint process</h2><p id="p-81">Here we derive by induction a matrix formulation for the (random) discrete map with a breakpoint. To do that, we derive first the multivariate distribution of the process without a breakpoint, initialized at time 0 and stopped at time <em>q</em>. Although this derivation has been shown before (<a id="xref-ref-7-9" class="xref-bibr" href="#ref-7">Dennis et al 2006</a>), presenting it here illuminates the formulation of the multivariate distribution with a change-point. To find the joint distribution of [<em>X</em><sub>0</sub>, <em>X</em><sub>1</sub>, <em>X</em><sub>2</sub>,… <em>X<sub>q</sub></em>]′ we first iterate the stochastic Gompertz map in <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">eq. (4)</a>:
<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-36.gif"/></noscript></span>

</span></p><p id="p-82">Then, continuing in this fashion and using vector notation we get
<span class="disp-formula" id="disp-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-37.gif"/></noscript></span>

</span>
where we used the well known formula for the partial sum of a geometric series. Upon examination of this iteration, it is seen that the vector of log population sizes through time, <strong>X</strong> = [<em>X</em><sub>0</sub> <em>X</em><sub>1</sub> <em>X</em><sub>2</sub> … <em>X<sub>q</sub></em>]′ can be written as:
<span class="disp-formula" id="disp-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-38.gif"/></noscript></span>

</span></p><p id="p-83">Assuming that the first observation arises from the Gaussian stationary distribution with mean and variance <em>a</em>/(1 – <em>c</em>) and <em>σ</em><sup>2</sup>/(1 – <em>c</em><sup>2</sup>) respectively, we then have that:
<span class="disp-formula" id="disp-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-39.gif"/></noscript></span>

</span>
and
<span class="disp-formula" id="disp-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-40.gif"/></noscript></span>

</span></p><p id="p-84">Using this multivariate mean and variance and the properties of the Multivariate Normal (<em>MVN</em>) distribution (Johnson and Wichern 2002, chapter 4), it follows that
<span class="disp-formula" id="disp-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-41.gif"/></noscript></span>

</span>
where
<span class="disp-formula" id="disp-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-42.gif"/></noscript></span>

</span>
and <strong>j</strong><sub>(<em>q</em>+1)×1</sub> is a vector of ones of size <em>q</em> + 1.</p><p id="p-85">Now, to derive the multivariate distribution of the process with a breakpoint, we start by writing expressions for the log-population process, from time <em>τ</em> (right before the breakpoint) and onwards:
<span class="disp-formula" id="disp-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-43.gif"/></noscript></span>

</span>
and continuing in this fashion, a geometric series for <em>c</em><sub>2</sub> appears in the solution:
<span class="disp-formula" id="disp-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-44.gif"/></noscript></span>

</span></p><p id="p-86">The induction patterns revealed by this iterated map can then be readily expressed in the matrix form shown in <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">eq. (7)</a> in the main text.</p></div><div class="section app" id="app-2"><h2 class="">AppendixA.2. Derivation of the variance-covariance matrix of the breakpoint process</h2><p id="p-87">Here we derive the four sub-matrices of <strong>Σ<sub><em>τ</em></sub></strong>. We recall that we also partitioned the matrices <strong>Γ</strong> and <strong>Φ</strong> into four sub-matrices matching the dimensions of the <strong>Σ<sub><em>τ</em></sub></strong> blocks. For Γ these matrices are
<span class="disp-formula" id="disp-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-45.gif"/></noscript></span>

</span>
and for <strong>Φ</strong> these are
<span class="disp-formula" id="disp-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-46.gif"/></noscript></span>

</span>
and
<span class="disp-formula" id="disp-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-47.gif"/></noscript></span>

</span></p><p id="p-88">With these matrices in place, the matrices in eqs. (16) and (17) are obtained by simple matrix multiplication and elementary algebraic manipulations:
<span class="disp-formula" id="disp-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-48.gif"/></noscript></span>

</span>
whereas
<span class="disp-formula" id="disp-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-49.gif"/></noscript></span>

</span></p><p id="p-89">Noting that
<span class="disp-formula" id="disp-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-50.gif"/></noscript></span>

</span>
is identical to the variance covariance of the second partition, had it arisen directly from the stationary distribution of the second set of parameters, and solving in that equation for <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/inline-graphic-32.gif"/></noscript></span></span> leads to the expression in <a id="xref-disp-formula-25-1" class="xref-disp-formula" href="#disp-formula-25">eq. (17)</a>.</p><p id="p-90">Next, we compute <strong>Σ<sub>12</sub></strong> = <em>Cov</em>(<strong>X<sub>1</sub>, X<sub>2</sub></strong>), which is equal to the transpose of Σ<sub>21</sub>. Straightforward matrix multiplications yield
<span class="disp-formula" id="disp-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-51.gif"/></noscript></span>

</span>
where
<span class="disp-formula" id="disp-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2018/01/05/224352/embed/graphic-52.gif"/></noscript></span>

</span></p><p id="p-91">A simple examination of <a id="xref-disp-formula-47-1" class="xref-disp-formula" href="#disp-formula-47">eq. (A.10)</a> and <a id="xref-disp-formula-17-4" class="xref-disp-formula" href="#disp-formula-17">eq. (A.11)</a> then leads to eq. 15 in the main text.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">[1].</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bulmer</span></span>, <span class="cit-article-title">The statistical analysis of density dependence</span>, <abbr class="cit-jnl-abbrev">Biometrics</abbr> (<span class="cit-pub-date">1975</span>) <span class="cit-fpage">901</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[2].</span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-224352v2.2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Scheffer</span></span>, <span class="cit-chapter-title">Alternative stable states and regime shifts in ecosystems</span>, <span class="cit-source">The Princeton guide to ecology</span>, <span class="cit-publ-loc">Princeton</span>, <span class="cit-publ-name">Princeton university press</span> <span class="cit-vol">809</span> (<span class="cit-pub-date">2009</span>) <span class="cit-fpage">395</span>–<span class="cit-lpage">406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPrinceton%2Bguide%2Bto%2Becology%26rft.volume%253D809%26rft.spage%253D395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[3].</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.3" data-doi="10.1890/03-3101"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Staples</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-article-title">Estimating population trend and process variation for pva in the presence of sampling error</span>, <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">85</span> (<span class="cit-pub-date">2004</span>) <span class="cit-fpage">923</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D85%26rft.spage%253D923%26rft_id%253Dinfo%253Adoi%252F10.1890%252F03-3101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/03-3101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[4].</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.4"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Staples</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. B.</span> <span class="cit-name-surname">Shepard</span></span>, <span class="cit-article-title">Risk-based viable population monitoring</span>, <abbr class="cit-jnl-abbrev">Conservation Biology</abbr> <span class="cit-vol">19</span> (<span class="cit-pub-date">2005</span>) <span class="cit-fpage">1908</span>–<span class="cit-lpage">1916</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConservation%2BBiology%26rft.volume%253D19%26rft.spage%253D1908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[5].</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-224352v2.5"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Staples</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-chapter-title">Viable population monitoring: Risk-based population monitoring for threatened and endangered species with application to bull trout, Salvelinus confluentus</span>, <span class="cit-source">Ph.D. thesis</span>, <span class="cit-publ-name">Montana State University-Bozeman, College of Letters &amp; Science</span>, <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[6].</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.6"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Muhlfeld</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Staples</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. B.</span> <span class="cit-name-surname">Shepard</span></span>, <span class="cit-article-title">Observer error structure in bull trout redd counts in montana streams: implications for inference on true redd numbers</span>, <abbr class="cit-jnl-abbrev">Transactions of the American Fisheries Society</abbr> <span class="cit-vol">135</span> (<span class="cit-pub-date">2006</span>) <span class="cit-fpage">643</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransactions%2Bof%2Bthe%2BAmerican%2BFisheries%2BSociety%26rft.volume%253D135%26rft.spage%253D643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[7].</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.7" data-doi="10.1890/0012-9615(2006)76[323:EDDPNA]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Ponciano</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Lele</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Staples</span></span>, <span class="cit-article-title">Estimating density dependence, process noise, and observation error</span>, <abbr class="cit-jnl-abbrev">Ecological Monographs</abbr> <span class="cit-vol">76</span> (<span class="cit-pub-date">2006</span>) <span class="cit-fpage">323</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BMonographs%26rft.volume%253D76%26rft.spage%253D323%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9615%25282006%252976%255B323%253AEDDPNA%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9615(2006)76[323:EDDPNA]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239845400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[8].</span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.8" data-doi="10.1007/s11229-007-9299-x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Staples</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. B.</span> <span class="cit-name-surname">Shepard</span></span>, <span class="cit-article-title">Model structure adequacy analysis: selecting models on the basis of their ability to answer scientific questions</span>, <abbr class="cit-jnl-abbrev">Synthese</abbr> <span class="cit-vol">163</span> (<span class="cit-pub-date">2008</span>) <span class="cit-fpage">357</span>–<span class="cit-lpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSynthese%26rft.volume%253D163%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11229-007-9299-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11229-007-9299-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[9].</span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Staples</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Boik</span></span>, <span class="cit-article-title">Effects of sampling error and temporal correlations in population growth on process variance estimators</span>, <abbr class="cit-jnl-abbrev">Environmental and ecological statistics</abbr> <span class="cit-vol">16</span> (<span class="cit-pub-date">2009</span>) <span class="cit-fpage">547</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2Band%2Becological%2Bstatistics%26rft.volume%253D16%26rft.spage%253D547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[10].</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.10" data-doi="10.1016/S0022-5193(85)80084-9"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Holt</span></span>, <span class="cit-article-title">Density-independent mortality, non-linear competitive interactions, and species coexistence</span>, <abbr class="cit-jnl-abbrev">Journal of Theoretical Biology</abbr> <span class="cit-vol">116</span> (<span class="cit-pub-date">1985</span>) <span class="cit-fpage">479</span>–<span class="cit-lpage">493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.volume%253D116%26rft.spage%253D479%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-5193%252885%252980084-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-5193(85)80084-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[11].</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.11"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.-D.</span> <span class="cit-name-surname">Lebreton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Gimenez</span></span>, <span class="cit-article-title">Detecting and estimating density dependence in wildlife populations</span>, <abbr class="cit-jnl-abbrev">The Journal of Wildlife Management</abbr> <span class="cit-vol">77</span> (<span class="cit-pub-date">2013</span>) <span class="cit-fpage">12</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BWildlife%2BManagement%26rft.volume%253D77%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[12].</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.12" data-doi="10.2307/2937041"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-article-title">Density dependence in time series observations of natural populations: estimation and testing</span>, <abbr class="cit-jnl-abbrev">Ecological Monographs</abbr> <span class="cit-vol">64</span> (<span class="cit-pub-date">1994</span>) <span class="cit-fpage">205</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BMonographs%26rft.volume%253D64%26rft.spage%253D205%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2937041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2937041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NG26300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[13].</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.13" data-doi="10.1890/0012-9658(1998)079[0426:JDD]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Kemp</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-article-title">Joint density dependence</span>, <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">79</span> (<span class="cit-pub-date">1998</span>) <span class="cit-fpage">426</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D79%26rft.spage%253D426%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9658%25281998%2529079%255B0426%253AJDD%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9658(1998)079[0426:JDD]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[14].</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.14" data-doi="10.1111/j.1365-2656.2006.01121.x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. P.</span> <span class="cit-name-surname">Freckleton</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Watkinson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Green</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Sutherland</span></span>, <span class="cit-article-title">Census error and the detection of density dependence</span>, <abbr class="cit-jnl-abbrev">Journal of Animal Ecology</abbr> <span class="cit-vol">75</span> (<span class="cit-pub-date">2006</span>) <span class="cit-fpage">837</span>–<span class="cit-lpage">851</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.stitle%253DJournal%2Bof%2BAnimal%2BEcology%26rft.aulast%253DFreckleton%26rft.auinit1%253DR.%2BP.%26rft.volume%253D75%26rft.issue%253D4%26rft.spage%253D837%26rft.epage%253D851%26rft.atitle%253DCensus%2Berror%2Band%2Bthe%2Bdetection%2Bof%2Bdensity%2Bdependence.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2656.2006.01121.x%26rft_id%253Dinfo%253Apmid%252F17009748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2656.2006.01121.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17009748&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[15].</span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.15" data-doi="10.1890/09-0998.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Thomson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. J.</span> <span class="cit-name-surname">Kimmerer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L. R.</span> <span class="cit-name-surname">Brown</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. B.</span> <span class="cit-name-surname">Newman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Nally</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. A.</span> <span class="cit-name-surname">Bennett</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Feyrer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Fleishman</span></span>, <span class="cit-article-title">Bayesian change point analysis of abundance trends for pelagic fishes in the upper san francisco estuary</span>, <abbr class="cit-jnl-abbrev">Ecological Applications</abbr> <span class="cit-vol">20</span> (<span class="cit-pub-date">2010</span>) <span class="cit-fpage">1431</span>–<span class="cit-lpage">1448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BApplications%26rft.stitle%253DEcological%2BApplications%26rft.aulast%253DThomson%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D20%26rft.issue%253D5%26rft.spage%253D1431%26rft.epage%253D1448%26rft.atitle%253DBayesian%2Bchange%2Bpoint%2Banalysis%2Bof%2Babundance%2Btrends%2Bfor%2Bpelagic%2Bfishes%2Bin%2Bthe%2Bupper%2BSan%2BFrancisco%2BEstuary.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F09-0998.1%26rft_id%253Dinfo%253Apmid%252F20666259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/09-0998.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20666259&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279047400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[16].</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.16" data-doi="10.1126/science.1105122"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Packer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hilborn</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mosser</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Kissui</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Borner</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hopcraft</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wilmshurst</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mduma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Sinclair</span></span>, <span class="cit-article-title">Ecological change, group territoriality, and population dynamics in serengeti lions</span>, <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">307</span> (<span class="cit-pub-date">2005</span>) <span class="cit-fpage">390</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DPacker%26rft.auinit1%253DC.%26rft.volume%253D307%26rft.issue%253D5708%26rft.spage%253D390%26rft.epage%253D393%26rft.atitle%253DEcological%2BChange%252C%2BGroup%2BTerritoriality%252C%2Band%2BPopulation%2BDynamics%2Bin%2BSerengeti%2BLions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1105122%26rft_id%253Dinfo%253Apmid%252F15662005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDcvNTcwOC8zOTAiO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMS8wNS8yMjQzNTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[17].</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.17"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Hooten</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Wikle</span></span>, <span class="cit-article-title">Shifts in the spatio-temporal growth dynamics of shortleaf pine</span>, <abbr class="cit-jnl-abbrev">Environmental and Ecological Statistics</abbr> <span class="cit-vol">14</span> (<span class="cit-pub-date">2007</span>) <span class="cit-fpage">207</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2Band%2BEcological%2BStatistics%26rft.volume%253D14%26rft.spage%253D207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[18].</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.18"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Andersen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Carstensen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hernandez-Garcia</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Duarte</span></span>, <span class="cit-article-title">Ecological thresholds and regime shifts: approaches to identification</span>, <abbr class="cit-jnl-abbrev">Trends in Ecology &amp; Evolution</abbr> <span class="cit-vol">24</span> (<span class="cit-pub-date">2009</span>) <span class="cit-fpage">49</span>–<span class="cit-lpage">57</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.volume%253D24%26rft.spage%253D49%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[19].</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.19"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Ives</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Dakos</span></span>, <span class="cit-article-title">Detecting dynamical changes in nonlinear time series using locally linear state-space models</span>, <abbr class="cit-jnl-abbrev">Ecosphere</abbr> <span class="cit-vol">3</span> (<span class="cit-pub-date">2012</span>) <span class="cit-fpage">art58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcosphere%26rft.volume%253D3%26rft.spage%253D58art%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[20].</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.20" data-doi="10.1002/jae.3950070506"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B. E.</span> <span class="cit-name-surname">Hansen</span></span>, <span class="cit-article-title">The likelihood ratio test under nonstandard conditions: testing the markov switching model of gnp</span>, <abbr class="cit-jnl-abbrev">Journal of Applied Econometrics</abbr> <span class="cit-vol">7</span> (<span class="cit-pub-date">1992</span>) <span class="cit-fpage">S61</span>–<span class="cit-lpage">S82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BEconometrics%26rft.volume%253D7%26rft.spage%253DS61%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjae.3950070506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jae.3950070506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[21].</span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.21"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Garcia</span></span>, <span class="cit-article-title">Asymptotic null distribution of the likelihood ratio test in markov switching models</span>, <abbr class="cit-jnl-abbrev">International Economic Review</abbr> (<span class="cit-pub-date">1998</span>) <span class="cit-fpage">763</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[22].</span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.22"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Timmer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Pignatiello</span></span>, <span class="cit-article-title">Change point estimates for the parameters of an ar (1) process</span>, <abbr class="cit-jnl-abbrev">Quality and Reliability Engineering International</abbr> <span class="cit-vol">19</span> (<span class="cit-pub-date">2003</span>) <span class="cit-fpage">355</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuality%2Band%2BReliability%2BEngineering%2BInternational%26rft.volume%253D19%26rft.spage%253D355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[23].</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.23"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Jandhyala</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Fotopoulos</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">MacNeill</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-article-title">Inference for single and multiple change-points in time series</span>, <abbr class="cit-jnl-abbrev">Journal of Time Series Analysis</abbr> (<span class="cit-pub-date">2013</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[24].</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.24"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Picard</span></span>, <span class="cit-article-title">Testing and estimating change-points in time series</span>, <abbr class="cit-jnl-abbrev">Advances in Applied Probability</abbr> (<span class="cit-pub-date">1985</span>) <span class="cit-fpage">841</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[25].</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.25"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Andrews</span></span>, <span class="cit-article-title">Tests for parameter instability and structural change with unknown change point</span>, <abbr class="cit-jnl-abbrev">Econometrica: Journal of the Econometric Society</abbr> (<span class="cit-pub-date">1993</span>) <span class="cit-fpage">821</span>–<span class="cit-lpage">856</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[26].</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.26"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bai</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Perron</span></span>, <span class="cit-article-title">Estimating and testing linear models with multiple structural changes</span>, <abbr class="cit-jnl-abbrev">Econometrica</abbr> (<span class="cit-pub-date">1998</span>) <span class="cit-fpage">47</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[27].</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.27"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lavielle</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Moulines</span></span>, <span class="cit-article-title">Least-squares estimation of an unknown number of shifts in a time series</span>, <abbr class="cit-jnl-abbrev">Journal of Time Series Analysis</abbr> <span class="cit-vol">21</span> (<span class="cit-pub-date">2000</span>) <span class="cit-fpage">33</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTime%2BSeries%2BAnalysis%26rft.volume%253D21%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[28].</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.28"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Perreault</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bernier</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Bobee</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Parent</span></span>, <span class="cit-article-title">Bayesian change-point analysis in hydrometeorological time series. part 1. the normal model revisited</span>, <abbr class="cit-jnl-abbrev">Journal of Hydrology</abbr> <span class="cit-vol">235</span> (<span class="cit-pub-date">2000</span>) <span class="cit-fpage">221</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHydrology%26rft.stitle%253DJournal%2Bof%2BHydrology%26rft.volume%253D235%26rft.issue%253D3-4%26rft.spage%253D221%26rft.epage%253D241%26rft.atitle%253DBayesian%2Bchange-point%2Banalysis%2Bin%2Bhydrometeorological%2Btime%2Bseries%253B%2BPart%2B1%252C%2BThe%2Bnormal%2Bmodel%2Brevisited%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2000068898&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a></div></div></li><li><span class="ref-label">[29].</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-224352v2.29"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Yuan</span></span>, <span class="cit-chapter-title">Tests for Change-Point of the AR (1) Model</span>, <span class="cit-source">Ph.D. thesis</span>, <span class="cit-publ-name">University of Minnesota</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[30].</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.30"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Perry</span></span>, <span class="cit-article-title">Identifying the time of polynomial drift in the mean of autocorrelated processes</span>, <abbr class="cit-jnl-abbrev">Quality and Reliability Engineering International</abbr> <span class="cit-vol">26</span> (<span class="cit-pub-date">2010</span>) <span class="cit-fpage">399</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuality%2Band%2BReliability%2BEngineering%2BInternational%26rft.volume%253D26%26rft.spage%253D399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[31].</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.31"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Amiri</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Allahyari</span></span>, <span class="cit-article-title">Change point estimation methods for control chart postsignal diagnostics: a literature review</span>, <abbr class="cit-jnl-abbrev">Quality and Reliability Engineering International</abbr> <span class="cit-vol">28</span> (<span class="cit-pub-date">2012</span>) <span class="cit-fpage">673</span>–<span class="cit-lpage">685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQuality%2Band%2BReliability%2BEngineering%2BInternational%26rft.volume%253D28%26rft.spage%253D673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[32].</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-224352v2.32"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-chapter-title">Bayesian approach to Phase II statistical process control for time series</span>, <span class="cit-source">Ph.D. thesis</span>, <span class="cit-publ-name">University of Minnesota</span>, <span class="cit-pub-date">2013</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[33].</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.33" data-doi="10.1016/0025-5564(84)90031-2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Patil</span></span>, <span class="cit-article-title">The gamma distribution and weighted multimodal gamma distributions as models of population abundance</span>, <abbr class="cit-jnl-abbrev">Mathematical Biosciences</abbr> <span class="cit-vol">68</span> (<span class="cit-pub-date">1984</span>) <span class="cit-fpage">187</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2BBiosciences%26rft.volume%253D68%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0025-5564%252884%252990031-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0025-5564(84)90031-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">[34].</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.34"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zeng</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Nowierski</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. P.</span> <span class="cit-name-surname">Kemp</span></span>, <span class="cit-article-title">Complex population dynamics in the real world: modeling the influence of time-varying parameters and time lags</span>, <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">79</span> (<span class="cit-pub-date">1998</span>) <span class="cit-fpage">2193</span>–<span class="cit-lpage">2209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D79%26rft.spage%253D2193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[35].</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.35" data-doi="10.1890/0012-9615(2002)072[0057:FPMIPN]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">de Valpine</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hastings</span></span>, <span class="cit-article-title">Fitting population models incorporating process noise and observation error</span>, <abbr class="cit-jnl-abbrev">Ecological Monographs</abbr> <span class="cit-vol">72</span> (<span class="cit-pub-date">2002</span>) <span class="cit-fpage">57</span>–<span class="cit-lpage">76</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BMonographs%26rft.volume%253D72%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9615%25282002%2529072%255B0057%253AFPMIPN%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9615(2002)072[0057:FPMIPN]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000173435500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[36].</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.36" data-doi="10.1890/0012-9615(2003)073[0301:ECSAEI]2.0.CO;2"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ives</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Cottingham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Carpenter</span></span>, <span class="cit-article-title">Estimating community stability and ecological interactions from time-series data</span>, <abbr class="cit-jnl-abbrev">Ecological Monographs</abbr> <span class="cit-vol">73</span> (<span class="cit-pub-date">2003</span>) <span class="cit-fpage">301</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcological%2BMonographs%26rft.volume%253D73%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9615%25282003%2529073%255B0301%253AECSAEI%255D2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9615(2003)073[0301:ECSAEI]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000183500200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[37].</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.37" data-doi="10.1890/03-0520"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Clark</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">O. N.</span> <span class="cit-name-surname">Bjørnstad</span></span>, <span class="cit-article-title">Population time series: process variability, observation errors, missing values, lags, and hidden states</span>, <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">85</span> (<span class="cit-pub-date">2004</span>) <span class="cit-fpage">3140</span>–<span class="cit-lpage">3150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D85%26rft.spage%253D3140%26rft_id%253Dinfo%253Adoi%252F10.1890%252F03-0520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/03-0520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225263500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[38].</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.38" data-doi="10.1111/j.1461-0248.2007.01047.x"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Lele</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lutscher</span></span>, <span class="cit-article-title">Data cloning: easy maximum likelihood estimation for complex ecological models using bayesian markov chain monte carlo methods</span>, <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">10</span> (<span class="cit-pub-date">2007</span>) <span class="cit-fpage">551</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEcol%2BLett%26rft.aulast%253DLele%26rft.auinit1%253DS.%2BR.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D551%26rft.epage%253D563%26rft.atitle%253DData%2Bcloning%253A%2Beasy%2Bmaximum%2Blikelihood%2Bestimation%2Bfor%2Bcomplex%2Becological%2Bmodels%2Busing%2BBayesian%2BMarkov%2Bchain%2BMonte%2BCarlo%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2007.01047.x%26rft_id%253Dinfo%253Apmid%252F17542934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2007.01047.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17542934&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247123900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[39].</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.39" data-doi="10.1890/08-0967.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Ponciano</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Lele</span></span>, <span class="cit-article-title">Hierarchical models in ecology: confidence intervals, hypothesis testing, and model selection using data cloning</span>, <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">90</span> (<span class="cit-pub-date">2009</span>) <span class="cit-fpage">356</span>–<span class="cit-lpage">362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DPonciano%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D90%26rft.issue%253D2%26rft.spage%253D356%26rft.epage%253D362%26rft.atitle%253DHierarchical%2Bmodels%2Bin%2Becology%253A%2Bconfidence%2Bintervals%252C%2Bhypothesis%2Btesting%252C%2Band%2Bmodel%2Bselection%2Busing%2Bdata%2Bcloning.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F08-0967.1%26rft_id%253Dinfo%253Apmid%252F19323219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/08-0967.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19323219&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263570800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[40].</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.40" data-doi="10.1890/08-1461.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Polansky</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">De Valpine</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Lloyd-Smith</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Getz</span></span>, <span class="cit-article-title">Likelihood ridges and multimodality in population growth rate models</span>, <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">90</span> (<span class="cit-pub-date">2009</span>) <span class="cit-fpage">2313</span>–<span class="cit-lpage">2320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DPolansky%26rft.auinit1%253DL.%26rft.volume%253D90%26rft.issue%253D8%26rft.spage%253D2313%26rft.epage%253D2320%26rft.atitle%253DLikelihood%2Bridges%2Band%2Bmultimodality%2Bin%2Bpopulation%2Bgrowth%2Brate%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F08-1461.1%26rft_id%253Dinfo%253Apmid%252F19739392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/08-1461.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19739392&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268293000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[41].</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.41"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Ponciano</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-article-title">Replicated sampling increases efficiency in monitoring biological populations</span>, <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">91</span> (<span class="cit-pub-date">2010</span>) <span class="cit-fpage">610</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DDennis%26rft.auinit1%253DB.%26rft.volume%253D91%26rft.issue%253D2%26rft.spage%253D610%26rft.epage%253D620%26rft.atitle%253DReplicated%2Bsampling%2Bincreases%2Befficiency%2Bin%2Bmonitoring%2Bbiological%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F20392025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20392025&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[42].</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.42"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">de Valpine</span></span>, <span class="cit-article-title">Frequentist analysis of hierarchical models for population dynamics and demographic data</span>, <abbr class="cit-jnl-abbrev">Journal of Ornithology</abbr> <span class="cit-vol">152</span> (<span class="cit-pub-date">2012</span>) <span class="cit-fpage">393</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BOrnithology%26rft.volume%253D152%26rft.spage%253D393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[43].</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.43" data-doi="10.1086/673497"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.-E.</span> <span class="cit-name-surname">Sæther</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Coulson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Grøtan</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Engen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Altwegg</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K. B.</span> <span class="cit-name-surname">Armitage</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Barbraud</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">Becker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. T.</span> <span class="cit-name-surname">Blumstein</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F. S.</span> <span class="cit-name-surname">Dobson</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">How life history influences population dynamics in fluctuating environments</span>, <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">182</span> (<span class="cit-pub-date">2013</span>) <span class="cit-fpage">743</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BAmerican%2BNaturalist%26rft.volume%253D182%26rft.spage%253D743%26rft_id%253Dinfo%253Adoi%252F10.1086%252F673497%26rft_id%253Dinfo%253Apmid%252F24231536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/673497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24231536&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326907300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[44].</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-224352v2.44"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Lele</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Taper</span></span>, <span class="cit-chapter-title">Statistical inference in ecology</span>, in: <span class="cit-source">International Encyclopedia of Statistical Science</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-pub-date">2011</span>, pp. <span class="cit-fpage">1430</span>–<span class="cit-lpage">1433</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[45].</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.45" data-doi="10.1890/08-0071.1"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Knape</span></span>, <span class="cit-article-title">Estimability of density dependence in models of time series data</span>, <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">89</span> (<span class="cit-pub-date">2008</span>) <span class="cit-fpage">2994</span>–<span class="cit-lpage">3000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D89%26rft.spage%253D2994%26rft_id%253Dinfo%253Adoi%252F10.1890%252F08-0071.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/08-0071.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000261053500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[46].</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.46"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Holt</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Barfield</span></span>, <span class="cit-article-title">Trait-mediated effects, density dependence and the dynamic stability of ecological systems</span>, <abbr class="cit-jnl-abbrev">Trait-Mediated Indirect Interactions: Ecological and Evolutionary Perspectives</abbr> (<span class="cit-pub-date">2012</span>) <span class="cit-fpage">89</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[47].</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.47"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Lande</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Engen</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.-E.</span> <span class="cit-name-surname">Sæther</span></span>, <span class="cit-article-title">Estimating density dependence in time-series of age-structured populations, Philosophical Transactions of the Royal Society of London</span>. <abbr class="cit-jnl-abbrev">Series B: Biological Sciences</abbr> <span class="cit-vol">357</span> (<span class="cit-pub-date">2002</span>) <span class="cit-fpage">1179</span>–<span class="cit-lpage">1184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeries%2BB%253A%2BBiological%2BSciences%26rft.volume%253D357%26rft.spage%253D1179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[48].</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-224352v2.48"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kot</span></span>, <span class="cit-source">Elements of mathematical ecology</span>, <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-pub-date">2001</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[49].</span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.49" data-doi="10.1007/s00442-012-2347-3"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Herrando-Perez</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Delean</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B. W.</span> <span class="cit-name-surname">Brook</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Bradshaw</span></span>, <span class="cit-article-title">Density dependence: an ecological tower of babel</span>, <abbr class="cit-jnl-abbrev">Oecologia</abbr> <span class="cit-vol">170</span> (<span class="cit-pub-date">2012</span>) <span class="cit-fpage">585</span>–<span class="cit-lpage">603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOecologia%26rft.volume%253D170%26rft.spage%253D585%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00442-012-2347-3%26rft_id%253Dinfo%253Apmid%252F22648068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00442-012-2347-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22648068&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309866200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[50].</span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.50"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Patil</span></span>, <span class="cit-article-title">Applications in ecology</span>, <abbr class="cit-jnl-abbrev">Lognormal Distributions: Theory and Applications</abbr> (<span class="cit-pub-date">1988</span>) <span class="cit-fpage">303</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[51].</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference [51] in text" id="ref-51">↵</a><div class="cit ref-cit ref-book" id="cit-224352v2.51"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Rue</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Held</span></span>, <span class="cit-source">Gaussian Markov random fields: theory and applications</span>, <span class="cit-publ-name">CRC Press</span>, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[52].</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference [52] in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.52" data-doi="10.1086/508670"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Berryman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lima</span></span>, <span class="cit-article-title">Deciphering the effects of climate on animal populations: diagnostic analysis provides new interpretation of soay sheep dynamics</span>, <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">168</span> (<span class="cit-pub-date">2006</span>) <span class="cit-fpage">784</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DBerryman%26rft.auinit1%253DA.%26rft.volume%253D168%26rft.issue%253D6%26rft.spage%253D784%26rft.epage%253D795%26rft.atitle%253DDeciphering%2Bthe%2Beffects%2Bof%2Bclimate%2Bon%2Banimal%2Bpopulations%253A%2Bdiagnostic%2Banalysis%2Bprovides%2Bnew%2Binterpretation%2Bof%2BSoay%2Bsheep%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F508670%26rft_id%253Dinfo%253Apmid%252F17109320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/508670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17109320&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242126900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[53].</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference [53] in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.53"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Ferguson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Ponciano</span></span>, <span class="cit-article-title">Evidence and implications of higher order scaling in the environmental variation of animal population growth</span>, <abbr class="cit-jnl-abbrev">Proceedings of the National Academy of Sciences In press</abbr> (<span class="cit-pub-date">2015</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[54].</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference [54] in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-224352v2.54"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Cushing</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Costantino</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Desharnais</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Henson</span></span>, <span class="cit-source">Chaos in ecology: experimental nonlinear dynamics</span>, volume <span class="cit-vol">1</span>, <span class="cit-publ-name">Elsevier</span>, <span class="cit-pub-date">2002</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">[55].</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference [55] in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.55" data-doi="10.1038/nature06922"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Melbourne</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hastings</span></span>, <span class="cit-article-title">Extinction risk depends strongly on factors contributing to stochasticity</span>, <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span> (<span class="cit-pub-date">2008</span>) <span class="cit-fpage">100</span>–<span class="cit-lpage">103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMelbourne%26rft.auinit1%253DB.%2BA.%26rft.volume%253D454%26rft.issue%253D7200%26rft.spage%253D100%26rft.epage%253D103%26rft.atitle%253DExtinction%2Brisk%2Bdepends%2Bstrongly%2Bon%2Bfactors%2Bcontributing%2Bto%2Bstochasticity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06922%26rft_id%253Dinfo%253Apmid%252F18596809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06922&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18596809&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257308300045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[56].</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference [56] in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.56" data-doi="10.1111/ele.12227"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Ferguson</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Ponciano</span></span>, <span class="cit-article-title">Predicting the process of extinction in experimental microcosms and accounting for interspecific interactions in single-species time series</span>, <abbr class="cit-jnl-abbrev">Ecology Letters</abbr> <span class="cit-vol">17</span> (<span class="cit-pub-date">2014</span>) <span class="cit-fpage">251</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%2BLetters%26rft.volume%253D17%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12227%26rft_id%253Dinfo%253Apmid%252F24304946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24304946&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[57].</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference [57] in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.57"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schwager</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Johst</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Jeltsch</span></span>, <span class="cit-article-title">Does red noise increase or decrease extinction risk? single extreme events versus series of unfavorable conditions</span>, <abbr class="cit-jnl-abbrev">The American Naturalist</abbr> <span class="cit-vol">167</span> (<span class="cit-pub-date">2006</span>) <span class="cit-fpage">879</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAm%2BNat%26rft.aulast%253DSchwager%26rft.auinit1%253DM.%26rft.volume%253D167%26rft.issue%253D6%26rft.spage%253D879%26rft.epage%253D888%26rft.atitle%253DDoes%2Bred%2Bnoise%2Bincrease%2Bor%2Bdecrease%2Bextinction%2Brisk%253F%2BSingle%2Bextreme%2Bevents%2Bversus%2Bseries%2Bof%2Bunfavorable%2Bconditions.%26rft_id%253Dinfo%253Apmid%252F16615033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16615033&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[58].</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference [58] in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.58"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dennis</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Assas</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Elaydi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Kwessi</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Livadiotis</span></span>, <span class="cit-article-title">Allee effects and resilience in stochastic populations</span>, <abbr class="cit-jnl-abbrev">Theoretical Ecology</abbr> <span class="cit-vol">9</span> (<span class="cit-pub-date">2016</span>) <span class="cit-fpage">323</span>–<span class="cit-lpage">335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2BEcology%26rft.volume%253D9%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[59].</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference [59] in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.59"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">T. J.</span> <span class="cit-name-surname">Case</span></span>, <span class="cit-article-title">Illustrated guide to theoretical ecology</span>, <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">80</span> (<span class="cit-pub-date">2000</span>) <span class="cit-fpage">2848</span>–<span class="cit-lpage">2848</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D80%26rft.spage%253D2848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[60].</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference [60] in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.60" data-doi="10.1126/science.1110760"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Sibly</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Barker</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Denham</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hone</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pagel</span></span>, <span class="cit-article-title">On the regulation of populations of mammals, birds, fish, and insects</span>, <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">309</span> (<span class="cit-pub-date">2005</span>) <span class="cit-fpage">607</span>–<span class="cit-lpage">610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSibly%26rft.auinit1%253DR.%2BM.%26rft.volume%253D309%26rft.issue%253D5734%26rft.spage%253D607%26rft.epage%253D610%26rft.atitle%253DOn%2Bthe%2BRegulation%2Bof%2BPopulations%2Bof%2BMammals%252C%2BBirds%252C%2BFish%252C%2Band%2BInsects%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1110760%26rft_id%253Dinfo%253Apmid%252F16040705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDkvNTczNC82MDciO3M6NDoiYXRvbSI7czozNzoiL2Jpb3J4aXYvZWFybHkvMjAxOC8wMS8wNS8yMjQzNTIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">[61].</span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference [61] in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.61"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Cattau</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Fletcher</span>   <span class="cit-name-suffix">Jr</span>.</span>, <span class="cit-auth"> <span class="cit-name-given-names">R. T.</span> <span class="cit-name-surname">Kimball</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Miller</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Kitchens</span></span>, <span class="cit-article-title">Rapid morphological change of a top predator with the invasion of a novel prey</span>, <abbr class="cit-jnl-abbrev">Nature ecology &amp; evolution</abbr> <span class="cit-vol">2</span> (<span class="cit-pub-date">2018</span>) <span class="cit-fpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Becology%2B%2526%2Bevolution%26rft.volume%253D2%26rft.spage%253D108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">[62].</span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference [62] in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.62" data-doi="10.1098/rspb.2013.2935"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Connors</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cooper</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Peterman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Dulvy</span></span>, <span class="cit-article-title">The false classification of extinction risk in noisy environments</span>, <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr> <span class="cit-vol">281</span> (<span class="cit-pub-date">2014</span>) <span class="cit-fpage">20132935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.2935%26rft_id%253Dinfo%253Apmid%252F24898368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2013.2935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24898368&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[63].</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference [63] in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.63" data-doi="10.1186/2049-2618-1-29"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ravel</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Brotman</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gajer</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ma</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nandy</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Fadrosh</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sakamoto</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Koenig</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Fu</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhou</span></span>, <span class="cit-etal">et al.</span>, <span class="cit-article-title">Daily temporal dynamics of vaginal microbiota before, during and after episodes of bacterial vaginosis</span>, <abbr class="cit-jnl-abbrev">Microbiome</abbr> <span class="cit-vol">1</span> (<span class="cit-pub-date">2013</span>) <span class="cit-fpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiome%26rft.volume%253D1%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2049-2618-1-29%26rft_id%253Dinfo%253Apmid%252F24451163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2049-2618-1-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451163&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[64].</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference [64] in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.64" data-doi="10.1098/rspb.2013.0686"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Drake</span></span>, <span class="cit-article-title">Early warning signals of stochastic switching</span>, <abbr class="cit-jnl-abbrev">Proceedings of the Royal Society B: Biological Sciences</abbr> <span class="cit-vol">280</span> (<span class="cit-pub-date">2013</span>) <span class="cit-fpage">20130686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.0686%26rft_id%253Dinfo%253Apmid%252F23843385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2013.0686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23843385&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2018%2F01%2F05%2F224352.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">[65].</span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference [65] in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.65" data-doi="10.1098/rsif.2012.0125"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Boettiger</span></span>, <span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Hastings</span></span>, <span class="cit-article-title">Quantifying limits to detection of early warning for critical transitions</span>, <abbr class="cit-jnl-abbrev">Journal of The Royal Society Interface</abbr> <span class="cit-vol">9</span> (<span class="cit-pub-date">2012</span>) <span class="cit-fpage">2527</span>–<span class="cit-lpage">2539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BThe%2BRoyal%2BSociety%2BInterface%26rft.volume%253D9%26rft.spage%253D2527%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsif.2012.0125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsif.2012.0125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000308240400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">[66].</span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference [66] in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-224352v2.66"><div class="cit-metadata"><cite><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Chesson</span></span>, <span class="cit-article-title">Aedt: A new concept for ecological dynamics in the ever-changing world</span>, <abbr class="cit-jnl-abbrev">PLoS biology</abbr> <span class="cit-vol">15</span> (<span class="cit-pub-date">2017</span>) <span class="cit-fpage">e2002634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.volume%253D15%26rft.spage%253De2002634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.biorxiv.org/content/10.1101/224352v2.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F224352v2" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/243410v1" title="The brown alga Saccharina japonica features distinct vanadium-dependent bromoperoxidases and iodoperoxidases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/243683v1" title="Efficacy and dynamics of self-targeting CRISPR/Cas constructs for gene editing in the retina" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 05, 2018.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/224352v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><i class="icon-external-link-sign"></i> <span class="title">Download PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/224352v2.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/224352v2" class="active" data-icon-position="" data-hide-link-title="0">Ecological change points: The strength of density dependence and the loss of history</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/74472" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-rYZKwvVjPoIWbqhQ0xxMk7Mg9kYdNFQTvvXyqdPgJH8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="74472" id="node74472--2" data-pisa="biorxiv;224352v2" data-pisa-master="biorxiv;224352" data-seqnum="74472" data-apath="/biorxiv/early/2018/01/05/224352.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Ecological change points: The strength of density dependence and the loss of history</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">José M.</span> <span class="nlm-surname">Ponciano</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mark L.</span> <span class="nlm-surname">Taper</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Dennis</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">224352; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/224352 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/224352v2" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/224352v2&amp;title=Ecological%20change%20points%3A%20The%20strength%20of%20density%20dependence%20and%20the%20loss%20of%20history" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/224352v2&amp;title=Ecological%20change%20points%3A%20The%20strength%20of%20density%20dependence%20and%20the%20loss%20of%20history" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/224352v2&amp;text=Ecological%20change%20points%3A%20The%20strength%20of%20density%20dependence%20and%20the%20loss%20of%20history" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.biorxiv.org/content/10.1101/224352v2&amp;title=Ecological%20change%20points%3A%20The%20strength%20of%20density%20dependence%20and%20the%20loss%20of%20history" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/224352v2&amp;t=Ecological%20change%20points%3A%20The%20strength%20of%20density%20dependence%20and%20the%20loss%20of%20history" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/224352v2&amp;title=Ecological%20change%20points%3A%20The%20strength%20of%20density%20dependence%20and%20the%20loss%20of%20history" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/224352v2&amp;title=Ecological%20change%20points%3A%20The%20strength%20of%20density%20dependence%20and%20the%20loss%20of%20history" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="74472" id="node74472--3" data-pisa="biorxiv;224352v2" data-pisa-master="biorxiv;224352" data-seqnum="74472" data-apath="/biorxiv/early/2018/01/05/224352.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">Ecological change points: The strength of density dependence and the loss of history</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">José M.</span> <span class="nlm-surname">Ponciano</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Mark L.</span> <span class="nlm-surname">Taper</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Brian</span> <span class="nlm-surname">Dennis</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">224352; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/224352 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=Ecological%20change%20points%3A%20The%20strength%20of%20density%20dependence%20and%20the%20loss%20of%20history&amp;rft.volume&amp;rft.issue&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Ponciano%2C+Jos%C3%A9+M.&amp;rft.au=Taper%2C+Mark+L.&amp;rft.au=Dennis%2C+Brian"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/74472/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/74472/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/74472/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/74472/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/74472/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/74472/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/74472/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/74472/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/74472/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/74472/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/74472/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/74472/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/224352v2&amp;count=horizontal&amp;via=&amp;text=Ecological%20change%20points%3A%20The%20strength%20of%20density%20dependence%20and%20the%20loss%20of%20history&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/224352v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/224352v2&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/224352v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/ecology" class="highlight" data-icon-position="" data-hide-link-title="0">Ecology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-8 clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-8-inside">
<div class="panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(1651)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(2761)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(1919)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(10321)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(4210)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(3243)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(4583)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(135)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(2858)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(4501)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2041)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(7254)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(5504)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(6835)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(2409)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(7535)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(3014)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(18714)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(140)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(476)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(787)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(1167)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(2735)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(684)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(894)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(2862)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(470)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__t6UjIjIKIAD6nKc6BAU9qmlVV2VR4CBTNQbxlbJPgN0__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__QJacVmveFyLIBSPkp21xL0ZvuDW9DMBrD2HB1P4Ry5I.js"></script>
  </body>
</html>
