<?xml version="1.0" encoding="UTF-8"?>
<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"> 
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> 
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2020/02/07/2020.02.07.938456.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.07.938456v1"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Genetic basis of phenotypic plasticity and genotype x environment interaction in a multi-parental population"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.07.938456"/>
  <meta name="DC.Date" content="2020-02-07"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Deciphering the genetic basis of phenotypic plasticity and genotype x environment interaction (GxE) is of primary importance for plant breeding in the context of global climate change. Tomato is a widely cultivated crop that can grow in different geographical habitats and which evinces a great capacity of expressing phenotypic plasticity. We used a multi-parental advanced generation intercross (MAGIC) tomato population to explore GxE and plasticity for multiple traits measured in a multi-environment trial (MET) design comprising optimal cultural conditions and water deficit, salinity and heat stress over 12 environments. Substantial GxE was observed for all the traits measured. Different plasticity parameters were estimated through the Finlay-Wilkinson and factorial regression models and used together with the genotypic means for quantitative trait loci (QTL) mapping analyses. Mixed linear models were further used to investigate the presence of interactive QTLs (QEI). The results highlighted a complex genetic architecture of tomato plasticity and GxE. Candidate genes that might be involved in the occurrence of GxE were proposed, paving the way for functional characterization of stress response genes in tomato and breeding for climate-adapted crop. Highlight The genetic architecture of tomato response to several abiotic stresses is deciphered. QTL for plasticity and QTL x Environment were identified in a highly recombinant MAGIC population."/>
  <meta name="DC.Contributor" content="Isidore Diouf"/>
  <meta name="DC.Contributor" content="Laurent Derivot"/>
  <meta name="DC.Contributor" content="Shai Koussevitzky"/>
  <meta name="DC.Contributor" content="Yolande Carretero"/>
  <meta name="DC.Contributor" content="Frédérique Bitton"/>
  <meta name="DC.Contributor" content="Laurence Moreau"/>
  <meta name="DC.Contributor" content="Mathilde Causse"/>
  <meta name="article:published_time" content="2020-02-07"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Genetic basis of phenotypic plasticity and genotype x environment interaction in a multi-parental population"/>
  <meta name="citation_abstract" lang="en" content="&lt;;p&gt;;Deciphering the genetic basis of phenotypic plasticity and genotype x environment interaction (GxE) is of primary importance for plant breeding in the context of global climate change. Tomato is a widely cultivated crop that can grow in different geographical habitats and which evinces a great capacity of expressing phenotypic plasticity. We used a multi-parental advanced generation intercross (MAGIC) tomato population to explore GxE and plasticity for multiple traits measured in a multi-environment trial (MET) design comprising optimal cultural conditions and water deficit, salinity and heat stress over 12 environments. Substantial GxE was observed for all the traits measured. Different plasticity parameters were estimated through the Finlay-Wilkinson and factorial regression models and used together with the genotypic means for quantitative trait loci (QTL) mapping analyses. Mixed linear models were further used to investigate the presence of interactive QTLs (QEI). The results highlighted a complex genetic architecture of tomato plasticity and GxE. Candidate genes that might be involved in the occurrence of GxE were proposed, paving the way for functional characterization of stress response genes in tomato and breeding for climate-adapted crop.&lt;;/p&gt;;"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2020/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.07.938456v1"/>
  <meta name="citation_id" content="2020.02.07.938456v1"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.938456v1"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.938456v1.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.938456v1.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.07.938456"/>
  <meta name="citation_num_pages" content="35"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.07.938456"/>
  <meta name="citation_author" content="Isidore Diouf"/>
  <meta name="citation_author_institution" content="INRAE, GAFL"/>
  <meta name="citation_author" content="Laurent Derivot"/>
  <meta name="citation_author_institution" content="GAUTIER Semences, route d’Avignon"/>
  <meta name="citation_author" content="Shai Koussevitzky"/>
  <meta name="citation_author_institution" content="Hazera – Seeds of Growth"/>
  <meta name="citation_author" content="Yolande Carretero"/>
  <meta name="citation_author_institution" content="INRAE, GAFL"/>
  <meta name="citation_author" content="Frédérique Bitton"/>
  <meta name="citation_author_institution" content="INRAE, GAFL"/>
  <meta name="citation_author" content="Laurence Moreau"/>
  <meta name="citation_author_institution" content="UMR GQE-Le Moulon, INRA, CNRS, AgroParisTech, Université Paris-Saclay"/>
  <meta name="citation_author" content="Mathilde Causse"/>
  <meta name="citation_author_institution" content="INRAE, GAFL"/>
  <meta name="citation_author_email" content="mathilde.causse@inra.fr"/>
  <meta name="citation_reference" content="Albert, E., Duboscq, R., Latreille, M., Santoni, S., Beukers, M., Bouchet, J.-P., et al. (2018) Allele specific expression and genetic determinants of transcriptomic variations in response to mild water deficit in tomato. Plant J."/>
  <meta name="citation_reference" content="Albert, E., Segura, V., Gricourt, J., Bonnefoi, J., Derivot, L., and Causse, M. (2016) Association mapping reveals the genetic architecture of tomato response to water deficit: focus on major fruit quality traits. J Exp Bot. 67: 6413–6430."/>
  <meta name="citation_reference" content="Aubert, D., Chen, L., Moon, Y.H., Martin, D., Castle, L.A., Yang, C.H., et al. (2001) EMF1, a novel protein involved in the control of shoot architecture and flowering in Arabidopsis. Plant Cell. 13: 1865–75."/>
  <meta name="citation_reference" content="Barrero, J.M., Cavanagh, C., Verbyla, K.L., Tibbits, J.F.G., Verbyla, A.P., Huang, B.E., et al. (2015) Transcriptomic analysis of wheat near-isogenic lines identifies PM19-A1 and A2 as candidates for a major dormancy QTL. Genome Biol. 16: 93."/>
  <meta name="citation_reference" content="Bradshaw, A.D. (1965) Evolutionary Significance of Phenotypic Plasticity in Plants. Adv Genet. 13: 115–155."/>
  <meta name="citation_reference" content="Broman, K.W., Gatti, D.M., Simecek, P., Furlotte, N.A., Prins, P., Sen, S., et al. (2019) R/qtl2: Software for Mapping Quantitative Trait Loci with High-Dimensional Data and Multiparent Populations. Genetics. 211: 495–502."/>
  <meta name="citation_reference" content="Causse, M., Desplat, N., Pascual, L., Le Paslier, M.-C., Sauvage, C., Bauchet, G., et al. (2013) Whole genome resequencing in tomato reveals variation associated with introgression and breeding events. BMC Genomics. 14: 791."/>
  <meta name="citation_reference" content="Costa, J.M., Ortuño, M.F., and Chaves, M.M. (2007) Deficit Irrigation as a Strategy to Save Water: Physiology and Potential Application to Horticulture. J Integr Plant Biol. 49: 1421–1434."/>
  <meta name="citation_reference" content="Diouf, I.A., Derivot, L., Bitton, F., Pascual, L., and Causse, M. (2018) Water Deficit and Salinity Stress Reveal Many Specific QTL for Plant Growth and Fruit Quality Traits in Tomato. Front Plant Sci. 9: 279."/>
  <meta name="citation_reference" content="Driedonks, N., Wolters-Arts, M., Huber, H., de Boer, G.-J., Vriezen, W., Mariani, C., et al. (2018) Exploring the natural variation for reproductive thermotolerance in wild tomato species. Euphytica. 214: 67."/>
  <meta name="citation_reference" content="El-Soda, M., Malosetti, M., Zwaan, B.J., Koornneef, M., and Aarts, M.G.M. (2014). Trends Plant Sci."/>
  <meta name="citation_reference" content="Estañ, M.T., Villalta, I., Bolarín, M.C., Carbonell, E.A., and Asins, M.J. (2009) Identification of fruit yield loci controlling the salt tolerance conferred by solanum rootstocks. Theor Appl Genet. 118: 305–312."/>
  <meta name="citation_reference" content="Finlay, B.K.W., and Wilkinson, G.N. (1963) THE ANALYSIS OF ADAPTATION IN A PLANT-BREEDING PROGRAMME The ability of some crop varieties to perform well over a wide range of environmental conditions has lcng been appreciated by the agronomist and plant breeder. In the cereal belts of southern Au."/>
  <meta name="citation_reference" content="Foolad, M.R. (2007) Genome mapping and molecular breeding of tomato. Int J Plant Genomics. 2007: 64358."/>
  <meta name="citation_reference" content="Gage, J.L., Jarquin, D., Romay, C., Lorenz, A., Buckler, E.S., Kaeppler, S., et al. (2017) The effect of artificial selection on phenotypic plasticity in maize. Nat Commun. 8: 1348."/>
  <meta name="citation_reference" content="Gillespie, J.H., and Turelli, M. (1989) Genotype-environment interactions and the maintenance of polygenic variation. Genetics. 121."/>
  <meta name="citation_reference" content="Giraud, H., Bauland, C., Falque, M., Madur, D., Combes, V., Jamin, P., et al. (2017) Reciprocal Genetics: Identifying QTL for General and Specific Combining Abilities in Hybrids Between Multiparental Populations from Two Maize (Zea mays L.) Heterotic Groups. Genetics. 207: 1167–1180."/>
  <meta name="citation_reference" content="Grandillo, S., Termolino, P., and van der Knaap, E. (2013) Molecular Mapping of Complex Traits in Tomato. In Genetics, Genomics, and Breeding of Tomato. pp. 150–227 Science Publishers."/>
  <meta name="citation_reference" content="Grilli, G., Trevizan Braz, L., Gertrudes, E., and Lemos, M. (2007) QTL identification for tolerance to fruit set in tomatoby fAFLP markers. Crop Breed Appl Biotechnol. 7: 234–241."/>
  <meta name="citation_reference" content="Herath, V. (2019) The architecture of the GhD7 promoter reveals the roles of GhD7 in growth, development and the abiotic stress response in rice. Comput Biol Chem. 82: 1–8."/>
  <meta name="citation_reference" content="Huang, B.E., and George, A.W. (2011) R/mpMap: A computational platform for the genetic analysis of multiparent recombinant inbred lines. Bioinformatics. 27: 727–729."/>
  <meta name="citation_reference" content="Huang, B.E., George, A.W., Forrest, K.L., Kilian, A., Hayden, M.J., Morell, M.K., et al. (2012) A multiparent advanced generation inter-cross population for genetic analysis in wheat. Plant Biotechnol J. 10: 826–839."/>
  <meta name="citation_reference" content="Huang, H., Gehan, M.A., Huss, S.E., Alvarez, S., Lizarraga, C., Gruebbling, E.L., et al. (2017) Cross-species complementation reveals conserved functions for EARLY FLOWERING 3 between monocots and dicots. Plant Direct. 1: e00018."/>
  <meta name="citation_reference" content="Kover, P.X., Valdar, W., Trakalo, J., Scarcelli, N., Ehrenreich, I.M., Purugganan, M.D., et al. (2009) A multiparent advanced generation inter-cross to fine-map quantitative traits in Arabidopsis thaliana. PLoS Genet. 5: e1000551."/>
  <meta name="citation_reference" content="Kusmec, A., Srinivasan, S., Nettleton, D., and Schnable, P.S. (2017) Distinct genetic architectures for phenotype means and plasticities in Zea mays. Nat Plants. 3: 715–723."/>
  <meta name="citation_reference" content="Lacaze, X., Hayes, P.M., and Korol, A. (2009) Genetics of phenotypic plasticity: QTL analysis in barley, Hordeum vulgare. Heredity (Edinb). 102: 163–173."/>
  <meta name="citation_reference" content="Laitinen, R.A.E., and Nikoloski, Z. (2019) Genetic basis of plasticity in plants. J Exp Bot. 70: 739–745."/>
  <meta name="citation_reference" content="Lê, S., Josse, J., and Husson, F. (2008) FactoMineR: An R package for multivariate analysis. J Stat Softw. 25: 1–18."/>
  <meta name="citation_reference" content="Lin, C.S., Binns, M.R., and Lefkovitch, L.P. (1986) Stability Analysis: Where Do We Stand?1. Crop Sci. 26: 894."/>
  <meta name="citation_reference" content="Lin, T., Zhu, G., Zhang, J., Xu, X., Yu, Q., Zheng, Z., et al. (2014) Genomic analyses provide insights into the history of tomato breeding. Nat Genet. 46: 1220–1226."/>
  <meta name="citation_reference" content="Luo, X., Gao, Z., Wang, Y., Chen, Z., Zhang, W., Huang, J., et al. (2018) The NUCLEAR FACTOR-CONSTANS complex antagonizes Polycomb repression to de-repress FLOWERING LOCUS T expression in response to inductive long days in Arabidopsis. Plant J. 95: 17–29."/>
  <meta name="citation_reference" content="Malosetti, M., Ribaut, J.-M., and van Eeuwijk, F.A. (2013) The statistical analysis of multi-environment data: modeling genotype-by-environment interaction and its genetic basis. Front Physiol. 4: 44."/>
  <meta name="citation_reference" content="Mangin, B., Casadebaig, P., Cadic, E., Blanchet, N., Boniface, M.-C., Carrère, S., et al. (2017) Genetic control of plasticity of oil yield for combined abiotic stresses using a joint approach of crop modelling and genome-wide association. Plant Cell Environ. 40: 2276–2291."/>
  <meta name="citation_reference" content="Mitchell, J., Shennan, C., and Grattan, S. (1991) Developmental-Changes in Tomato Fruit Composition in Response To Water Deficit and Salinity. Physiol Plant. 83: 177–185."/>
  <meta name="citation_reference" content="Munns, R., and Gilliham, M. (2015) Salinity tolerance of crops - what is the cost? New Phytol. 208: 668–673."/>
  <meta name="citation_reference" content="Pascual, L., Desplat, N., Huang, B.E., Desgroux, A., Bruguier, L., Bouchet, J.-P.P., et al. (2015) Potential of a tomato MAGIC population to decipher the genetic control of quantitative traits and detect causal variants in the resequencing era. Plant Biotechnol J. 13: 565–577."/>
  <meta name="citation_reference" content="Ripoll, J., Urban, L., Staudt, M., Lopez-Lauri, F., Bidel, L.P.R., and Bertin, N. (2014) Water shortage and quality of fleshy fruits—making the most of the unavoidable. J Exp Bot. 65: 4097–4117."/>
  <meta name="citation_reference" content="Rothan, C., Diouf, I., and Causse, M. (2019) Trait discovery and editing in tomato. Plant J. 97: 73–90."/>
  <meta name="citation_reference" content="Scheiner, S.M. (1993) Genetics and Evolution of Phenotypic Plasticity. Annu Rev Ecol Syst. 24: 35–68."/>
  <meta name="citation_reference" content="Septiani, P., Lanubile, A., Stagnati, L., Busconi, M., Nelissen, H., Pè, M.E., et al. (2019) Unravelling the genetic basis of fusarium seedling rot resistance in the MAGIC maize population: Novel targets for breeding. Sci Rep. 9: 4–13."/>
  <meta name="citation_reference" content="Ungerer, M.C., Halldorsdottir, S.S., Purugganan, M.D., and Mackay, T.F.C. (2003) Genotype-environment interactions at quantitative trait loci affecting inflorescence development in Arabidopsis thaliana. Genetics. 165: 353–365."/>
  <meta name="citation_reference" content="Valladares, F., Sanchez-Gomez, D., and Zavala, M.A. (2006) Quantitative estimation of phenotypic plasticity: bridging the gap between the evolutionary concept and its ecological applications. J Ecol. 94: 1103–1116."/>
  <meta name="citation_reference" content="Verbyla, A.P., Cavanagh, C.R., and Verbyla, K.L. (2014) Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC. G3 Genes, Genomes, Genet. 4."/>
  <meta name="citation_reference" content="Via, S., Gomulkiewicz, R., De Jong, G., Scheiner, S.M., Schlichting, C.D., and Van Tienderen, P.H. (1995) Adaptive phenotypic plasticity: consensus and controversy. Trends Ecol Evol. 10: 212–7."/>
  <meta name="citation_reference" content="Villalta, I., Bernet, G.P., Carbonell, E.A., and Asins, M.J. (2007) Comparative QTL analysis of salinity tolerance in terms of fruit yield using two solanum populations of F7 lines. Theor Appl Genet. 114: 1001–1017."/>
  <meta name="citation_reference" content="Xavier, A., Jarquin, D., Howard, R., Ramasubramanian, V., Specht, J.E., Graef, G.L., et al. (2018) Genome-Wide Analysis of Grain Yield Stability and Environmental Interactions in a Multiparental Soybean Population. G3 (Bethesda). 8: 519–529."/>
  <meta name="citation_reference" content="Xu, J., Driedonks, N., Rutten, M.J.M., Vriezen, W.H., de Boer, G.-J., and Rieu, I. (2017a) Mapping quantitative trait loci for heat tolerance of reproductive traits in tomato (Solanum lycopersicum). Mol Breed. 37: 58."/>
  <meta name="citation_reference" content="Xu, J., Wolters-Arts, M., Mariani, C., Huber, H., and Rieu, I. (2017b) Heat stress affects vegetative and reproductive performance and trait correlations in tomato (Solanum lycopersicum). Euphytica. 213: 156."/>
  <meta name="citation_reference" content="Yan, W., Kang, M.S., Ma, B., Woods, S., and Cornelius, P.L. (2007) GGE Biplot vs. AMMI Analysis of Genotype-by-Environment Data. Crop Sci. 47: 643."/>
  <meta name="citation_reference" content="Zhao, C., Liu, B., Piao, S., Wang, X., Lobell, D.B., Huang, Y., et al. (2017) Temperature increase reduces global yields of major crops in four independent estimates. Proc Natl Acad Sci. 114: 9326–9331."/>
  <meta name="citation_reference" content="Zhu, G., Wang, S., Huang, Z., Zhang, S., Liao, Q., Zhang, C., et al. (2018) Rewiring of the Fruit Metabolome in Tomato Breeding. Cell. 172: 249-261.e12."/>
  <meta name="twitter:title" content="Genetic basis of phenotypic plasticity and genotype x environment interaction in a multi-parental population"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Deciphering the genetic basis of phenotypic plasticity and genotype x environment interaction (GxE) is of primary importance for plant breeding in the context of global climate change. Tomato is a widely cultivated crop that can grow in different geographical habitats and which evinces a great capacity of expressing phenotypic plasticity. We used a multi-parental advanced generation intercross (MAGIC) tomato population to explore GxE and plasticity for multiple traits measured in a multi-environment trial (MET) design comprising optimal cultural conditions and water deficit, salinity and heat stress over 12 environments. Substantial GxE was observed for all the traits measured. Different plasticity parameters were estimated through the Finlay-Wilkinson and factorial regression models and used together with the genotypic means for quantitative trait loci (QTL) mapping analyses. Mixed linear models were further used to investigate the presence of interactive QTLs (QEI). The results highlighted a complex genetic architecture of tomato plasticity and GxE. Candidate genes that might be involved in the occurrence of GxE were proposed, paving the way for functional characterization of stress response genes in tomato and breeding for climate-adapted crop. Highlight The genetic architecture of tomato response to several abiotic stresses is deciphered. QTL for plasticity and QTL x Environment were identified in a highly recombinant MAGIC population."/>
  <meta name="og-title" property="og:title" content="Genetic basis of phenotypic plasticity and genotype x environment interaction in a multi-parental population"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.07.938456v1"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Deciphering the genetic basis of phenotypic plasticity and genotype x environment interaction (GxE) is of primary importance for plant breeding in the context of global climate change. Tomato is a widely cultivated crop that can grow in different geographical habitats and which evinces a great capacity of expressing phenotypic plasticity. We used a multi-parental advanced generation intercross (MAGIC) tomato population to explore GxE and plasticity for multiple traits measured in a multi-environment trial (MET) design comprising optimal cultural conditions and water deficit, salinity and heat stress over 12 environments. Substantial GxE was observed for all the traits measured. Different plasticity parameters were estimated through the Finlay-Wilkinson and factorial regression models and used together with the genotypic means for quantitative trait loci (QTL) mapping analyses. Mixed linear models were further used to investigate the presence of interactive QTLs (QEI). The results highlighted a complex genetic architecture of tomato plasticity and GxE. Candidate genes that might be involved in the occurrence of GxE were proposed, paving the way for functional characterization of stress response genes in tomato and breeding for climate-adapted crop. Highlight The genetic architecture of tomato response to several abiotic stresses is deciphered. QTL for plasticity and QTL x Environment were identified in a highly recombinant MAGIC population."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2020-02-07"/> 
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/> 
  <title>Genetic basis of phenotypic plasticity and genotype x environment interaction in a multi-parental population | bioRxiv</title> 
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-1140390 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results"> 
  <div class="article fulltext-view ">
   <span class="highwire-journal-article-marker-start"/>
   <div class="section abstract" id="abstract-1">
    <h2 class="">Abstract</h2>
    <p id="p-2">Deciphering the genetic basis of phenotypic plasticity and genotype x environment interaction (GxE) is of primary importance for plant breeding in the context of global climate change. Tomato is a widely cultivated crop that can grow in different geographical habitats and which evinces a great capacity of expressing phenotypic plasticity. We used a multi-parental advanced generation intercross (MAGIC) tomato population to explore GxE and plasticity for multiple traits measured in a multi-environment trial (MET) design comprising optimal cultural conditions and water deficit, salinity and heat stress over 12 environments. Substantial GxE was observed for all the traits measured. Different plasticity parameters were estimated through the Finlay-Wilkinson and factorial regression models and used together with the genotypic means for quantitative trait loci (QTL) mapping analyses. Mixed linear models were further used to investigate the presence of interactive QTLs (QEI). The results highlighted a complex genetic architecture of tomato plasticity and GxE. Candidate genes that might be involved in the occurrence of GxE were proposed, paving the way for functional characterization of stress response genes in tomato and breeding for climate-adapted crop.</p>
    <div id="sec-1" class="subsection">
     <p id="p-3">
      <strong>Highlight</strong> The genetic architecture of tomato response to several abiotic stresses is deciphered. QTL for plasticity and QTL x Environment were identified in a highly recombinant MAGIC population.
     </p>
    </div>
   </div>
   <div class="section" id="sec-2">
    <h2 class="">INTRODUCTION</h2>
    <p id="p-4">Plants are sessile organisms which have to cope with environmental fluctuations to ensure species reproduction for persistence in nature. For a given genotype, the expression of different phenotypes according to the growing environment is commonly called phenotypic plasticity (PP) (
     <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bradshaw, 1965</a>). It offers the possibility to plants to adapt to new environments, notably new locations, changes in climatic conditions or seasonal variations. In agriculture, the range of environmental variation for crop cultivation may also include different cultural practices or growing conditions, leading to the expression of PP on agronomic traits and unstable performance. When different genotypes/accessions are examined for PP within a species, inter-individual variations in their responses usually lead to the common phenomenon of genotype-environment (GxE) interaction (
     <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">El-Soda et al., 2014</a>). Understanding the genetic mechanisms driving PP and GxE in plants is a crucial step for being able to predict yield performance of crop cultivars and to adapt breeding strategies according to the targeted environments.
    </p>
    <p id="p-5">In plants, the genetic basis of PP has been investigated to assess whether PP has its own genetic regulation and thus could be directly selected. Three main genetic models, widely known as the over-dominance, allelic-sensitivity and gene-regulatory models were proposed in the literature as underlying plant PP (
     <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Scheiner, 1993</a>; 
     <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Via et al., 1995</a>). The over-dominance model suggests that PP is negatively correlated to the number of heterozygous loci (
     <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gillespie and Turelli, 1989</a>). The heterozygous status is favored by allele’s complementarity in this case. Allelic-sensitivity and gene-regulatory models are assumed to arise from the differential expression of an allele according to the environment and epistatic interactions between structural and regulatory alleles, respectively. The latter assumes an independent genetic control of mean phenotype and plasticity of a trait. Using a wide range of environmental conditions, the prevalence of the allelic-sensitivity or gene-regulatory model in explaining the genetic architecture of PP was explored in different crop species including barley (
     <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lacaze et al. 2009</a>), maize (
     <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gage et al., 2017</a>; 
     <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kusmec et al., 2017</a>), soybean (
     <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Xavier et al., 2018</a>) and sunflower (
     <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mangin et al., 2017</a>).
    </p>
    <p id="p-6">Quantification of PP is however a common question when analyzing the genetic architecture of plasticity since different parameters for PP estimation are available as reviewed by 
     <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Valladares et al. (2006)</a>. At a population level, when multiple genotypes are screened in different environments, different approaches can be used to assess plasticity (
     <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Laitinen and Nikoloski, 2019</a>). The most common of these approaches is the joint regression model (
     <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Finlay and Wilkinson, 1963</a>) that uses the average performance of the set of tested genotypes in each environment as an index on which the individual phenotypes are regressed. This model, commonly known as the Finlay-Wilkinson regression model, allows to estimate linear (slopes) and non-linear plasticity parameters (from the residual errors) that presumably have different genetic basis (
     <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kusmec et al., 2017</a>). If the detailed description of the environments is available, the environmental index used in the Finlay-Wilkinson regression model can be replaced by environmental covariates such as stress indexes through factorial regression models (
     <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Malosetti et al. 2013</a>). Thus plasticity could be estimated as the degree of sensitivity to a given stress continuum (
     <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Mangin et al., 2017</a>).
    </p>
    <p id="p-7">Climate change is predicted to increase the frequency and intensity of abiotic stresses with a high and negative impact on crop yield (
     <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhao et al., 2017</a>). Plants respond to abiotic stresses by altering their morphology and physiology, reallocating the energy for growth to defense against stress (
     <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Munns and Gilliham, 2015</a>). Consequences on agronomic performances are apparent and detrimental to productivity. The most common abiotic stresses studied across species are water deficit (WD), salinity stress (SS) and high temperature stress (HT). The negative impact of these stresses on yield have been underlined for major cultivated crops; however, positive effects of WD and SS on fruit quality have been observed in fruit trees and some vegetables notably in tomato (
     <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Costa et al. 2007</a>; 
     <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mitchell et al. 1991</a>; 
     <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ripoll et al. 2014</a>).
    </p>
    <p id="p-8">Tomato is an economically important crop and a plant model species which led to numerous studies that contributed much in understanding the genetic architecture of the crop and its response to environmental variation. However, most of the studies that addressed the genetic architecture of tomato response to environment were conducted on experimental populations exposed to two conditions (
     <em>i</em>.
     <em>e</em>. control 
     <em>vs</em> stress). 
     <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert et al. (2018)</a> for example identified different WD-response quantitative trait loci (QTL) in a bi-parental population derived from a cross of large and cherry tomato accessions. Tomato heat-response QTLs were also identified in different experimental populations including interspecific and intraspecific populations (
     <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grilli et al., 2007</a>; 
     <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Xu et al., 2017a</a>; 
     <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Driedonks et al., 2018</a>). These studies investigated heat-response QTLs using mostly reproductive traits screened under heat stress condition. 
     <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Villalta et al. (2007)</a> and 
     <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Diouf et al. (2018)</a> investigated the genetic architecture of tomato response to SS and identified different QTLs for physiological and agronomic traits, involved in salinity tolerance. However, no QTL study has yet been conducted on tomato plasticity assessed under a multiple stress design, although the coincidence of different stresses is a more realistic scenario in crop cultivation, especially with the climate change.
    </p>
    <p id="p-9">Tomato benefits of a large panel of genetic resources that have been used in multiple genetic mapping analyses (
     <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grandillo et al. 2013</a>). Bi-parental populations were first used in QTL mapping and permitted the characterization of plenty of QTLs related to yield, disease resistance and fruit quality. In the genomic era, new experimental populations were developed offering higher power and advantages for QTL detection. These include mutant collections, BIL-populations and multi-parent advanced generation intercross (MAGIC) as described in 
     <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Rothan et al. (2019)</a>. The first tomato MAGIC population was developed at INRA-Avignon in France and is composed of about 400 lines derived from an 8-way cross (
     <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pascual et al. 2015</a>). This population showed a wide intra-specific genetic variation under control and stress environments and is highly suitable for mapping QTLs (
     <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Diouf et al., 2018</a>).
    </p>
    <p id="p-10">In the present study, we used the 8-way tomato MAGIC population described above and evaluated its response in a multi-environment trial (MET) design. The population was grown in 12 environments including control and several stress conditions (WD, SS and HT), and agronomic traits related to yield, fruit quality, plant growth and phenology were measured. Different plasticity parameters were computed and used together with mean phenotypes to decipher the genetic control of response to environmental variation. Multi-environment QTL analysis was performed in addition to detection of interactive QTLs (QEI) along with QTL mapping for plasticity traits.</p>
   </div>
   <div class="section" id="sec-3">
    <h2 class="">MATERIALS AND METHODS</h2>
    <div id="sec-4" class="subsection">
     <h3>Plant material and phenotyping</h3>
     <p id="p-11">The MAGIC population was derived from a cross between eight parental lines that belong to 
      <em>Solanum. lycopersicum</em> and 
      <em>Solanum lycopersicum cerasiforme</em> groups. More details about the population development can be found in 
      <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Pascual et al. (2015)</a>. Briefly, the population was composed of about 400 8-way MAGIC lines that underwent three generations of selfing before greenhouse evaluations were carried out. In this study, a subset of 241 to 397 lines was grown in each environment (Supplemental Table 1).
     </p>
     <p id="p-12">The full genome of each parental line was re-sequenced and their comparison with the reference tomato genome (‘Heinz 1706’) yielded 4 millions SNPs (
      <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Causse et al., 2013</a>). From these polymorphisms, a genetic map of 1345 discriminant SNPs was developed (
      <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Pascual et al., 2015</a>) and used in the present study for the QTL analysis.
     </p>
    </div>
    <div id="sec-5" class="subsection">
     <h3>Experimental design</h3>
     <p id="p-13">The MAGIC population was grown in three different geographical regions (France, Israel and Morocco) and four specific stress treatments were applied. Trials were conducted in order that in a given trial any stress treatment was applied aside a control trial (Supplemental Table 1). Treatments consisted in water deficit (WD), two levels of salinity – considered here as low salinity (LS) and high salinity (HS) – and high temperature (HT) stress. Water deficit was applied by reducing the water irrigation of about 70% and 30% according to the reference evapotranspiration in Israel in 2014 and 2015, respectively and by 50% in Morocco in 2015. Salinity treatment was managed as described in 
      <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Diouf et al. (2018)</a> and the average electrical conductivity of the substrate (Ec) in Morocco 2016 was 3.76 and 6.50 dS.m
      <sup>-1</sup> for LS and HS, respectively; while the Ec in the control condition in Morocco 2015 was about 1.79 dS.m
      <sup>-1</sup>. For HT stress, plants were sown during the late spring and phenotyped in the summer 2014 in Israel (HIs14) and summer 2017 in France (HAvi17). During HT treatments, greenhouse vent opening was managed all along the entire growing season, with opening the vent only when temperatures rose up to 25°C. Average mean (respectively maximal) temperatures calculated on daily (24 hours) measurements were of 26°C (respectively 34°C) for HAvi17 and 33°C (respectively 48°C) for HIs14. Besides stress treatments, local conventional cultural conditions were applied for control treatments as described in 
      <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Diouf et al., (2018)</a>.
     </p>
     <p id="p-14">Environments were considered as any combination of a geographical region, a year of trial and an applied treatment (Supplemental Table 1). Climatic sensors were installed in the greenhouses and climatic parameters recorded hourly in all environments. From the climatic parameters, seven environmental covariates were defined (Supplemental Figure 1) including temperature parameters (mean, minimal and maximal daily temperatures and thermal amplitude), the sum of degree-day (SDD), the vapour-pressure-deficit (Vpd in kPa) and the relative humidity (RH) within the greenhouse. To characterize the environments, every covariate was calculated during the period covering flowering time of the population on the fourth truss. Indeed, phenotypic data analyzed here were mostly recorded on the fourth and fifth trusses (Supplemental Table 2). Hierarchical clustering was performed with ‘FactoMineR’ R package (
      <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lê et al., 2008</a>) using the environmental parameters to group environments according to their similarity regarding the within-greenhouse climatic conditions.
     </p>
     <div id="F1" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F1.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Clustering environments according to seven environmental covariates, measured during the vegetative and flowering stage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174045846" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Clustering environments according to seven environmental covariates, measured during the vegetative and flowering stage.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F1.medium.gif" width="440" height="346"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F1.medium.gif" width="440" height="346"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 1:</span>
       <p id="p-15" class="first-child">Clustering environments according to seven environmental covariates, measured during the vegetative and flowering stage.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-16">The MAGIC population, the eight parental lines and the four first generation hybrids (one hybrid per two-way cross) were evaluated for fruit weight (FW) by measuring the average FW of the third and/or fourth plant truss in each environment. Phenotypic data were recorded across the different environments for nine supplemental traits related to fruit quality – fruit fruit firmness (firm) and soluble solid content (SSC); plant phenology – flowering time (flw), number of flowers (nflw) and fruit setting (fset); plant development – stem diameter (diam), leaf length (leaf) and plant height (height) and fruit number (nfr). Details about the phenotyping measurements are in Supplemental Table 2. At least two plants per MAGIC line were replicated in each environment except in Avi17 (control condition) where the average phenotype was recorded from single plant measurements. Parents and hybrids had more replicates per genotype (at least two) and served as control lines to measure within-environment heterogeneity.</p>
    </div>
    <div id="sec-6" class="subsection">
     <h3>Evaluation of GxE and heritability</h3>
     <p id="p-17">Data were first analyzed separately in each environment to remove outliers and correct for spatial heterogeneity within the environment. The model (1) below was applied to test for micro-environmental variation within the greenhouse where 
      <em>y</em>
      <sub>
       <em>ijjk</em>
      </sub> represents the phenotype of the individual 
      <em>i</em>, located in row 
      <em>j</em> and position 
      <em>k</em> in the greenhouse; 
      <em>µ</em> is the overall mean; 
      <em>C</em>
      <sub>
       <em>i</em>
      </sub> and 
      <em>L</em>
      <sub>
       <em>i</em>
      </sub> represent the fixed effect of control lines and the random effect of the MAGIC lines, respectively. In this model, 
      <em>t</em>
      <sub>
       <em>i</em>
      </sub> is an index of 0 or 1, defined to distinguish between control and MAGIC lines; 
      <em>c</em>
      <sub>
       <em>ijk</em>
      </sub> is the random residual error. 
      <span class="disp-formula" id="disp-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-2.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-2.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-18">For every trait where row (
      <em>R</em>
      <sub>
       <em>j</em>
      </sub>) and/or position (
      <em>P</em>
      <sub>
       <em>k</em>
      </sub>) effects were significant, required corrections were applied by removing the BLUP of the significant effects from the raw data. Corrected data were gathered and used in model (2) in order to estimate the broad-sense heritability (
      <em>H</em>
      <sup>2</sup>) and the proportion of variance associated to the GxE (
      <em>prop. σ</em>
      <sup>2</sup>
      <sub>
       <em>GxE</em>
      </sub>). 
      <span class="disp-formula" id="disp-formula-2">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-3.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-3.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-19">In model (2), 
      <em>y</em>
      <sub>
       <em>ij</em>
      </sub> represents the phenotype of the individual 
      <em>i</em>, in environment 
      <em>j</em>; 
      <em>µ, C</em>
      <sub>
       <em>i</em>
      </sub>, 
      <em>L</em>
      <sub>
       <em>i</em>
      </sub> and the 
      <em>t</em>
      <sub>
       <em>i</em>
      </sub> index are as described in model (1); 
      <em>CxE</em>
      <sub>
       <em>ij</em>
      </sub> and 
      <em>LxE</em>
      <sub>
       <em>ij</em>
      </sub> are the fixed control lines x environment interaction effect and the random MAGIC lines x environment interaction effect, respectively. Within a given environment, random residuals error terms were assumed to be independent and identically distributed with a variance specific to each environment. From this model, the proportion of the total genotypic and GxE variance explained by the model was calculated as the following formula: 
      <em>prop. σ</em>
      <sup>2</sup>
      <sub>
       <em>GxE</em>
      </sub>
      <em>= σ</em>
      <sup>2</sup>
      <sub>
       <em>LxE</em>
      </sub>
      <em>/</em>(
      <em>σ</em>
      <sup>2</sup>
      <sub>
       <em>L</em>
      </sub> + 
      <em>σ</em>
      <sup>2</sup>
      <sub>
       <em>LxE</em>
      </sub>). The significance of GxE was tested with a likelihood ratio test (at 5% level) between the models with and without GxE. The broad-sense heritability at the whole design level (
      <em>H</em>
      <sup>2</sup>) was derived from variance components of model (2) and calculated as the following: 
      <span class="inline-formula" id="inline-formula-1">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/inline-graphic-1.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/inline-graphic-1.gif"/>
        </noscript>
       </span>
      </span>, where 
      <em>σ</em>
      <sup>2</sup>
      <sub>
       <em>L</em>
      </sub>and 
      <em>σ</em>
      <sup>2</sup>
      <sub>
       <em>LxE</em>
      </sub> are the variance components associated to the MAGIC lines and MAGIC lines x environment interaction effects, respectively. Here 
      <em>nb. E</em> and 
      <em>nb. R</em> represent the number of environments (
      <em>e</em>.
      <em>g</em>. 12 for FW) and the average number of replicates over the whole design; 
      <em>σ</em>
      <sup>2</sup>
      <sub>
       <em>E</em>
      </sub> is the average environmental variance (
      <em>i</em>.
      <em>e</em>. ∑ 
      <em>σ</em>
      <sup>2</sup>
      <sub>
       <em>Ej</em>
      </sub>
      <em>/ nb. E</em>).
     </p>
    </div>
    <div id="sec-7" class="subsection">
     <h3>Phenotypic plasticity</h3>
     <p id="p-20">Three different parameters of plasticity were estimated using the Finlay-Wilkinson regression (3) and a factorial regression (4) models.</p>
     <p id="p-21">In model (3), 
      <em>y</em>
      <sub>
       <em>ij</em>
      </sub> is the phenotype (average values per environment and genotype) and 
      <em>µ</em> the general intercept. 
      <em>G</em>
      <sub>
       <em>i</em>
      </sub> and 
      <em>E</em>
      <sub>
       <em>j</em>
      </sub> are the effects of the MAGIC line 
      <em>i</em> and environment 
      <em>j</em>, respectively and 
      <em>β</em>
      <sub>
       <em>i</em>
      </sub> represents the regression coefficient of the model. It measures individual genotypic sensitivity to the environment. 
      <span class="disp-formula" id="disp-formula-3">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-4.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-4.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-22">Environments are described here as an index that represents the ‘quality’ of the environment (
      <em>i</em>.
      <em>e</em>. the average performance of all genotypes in a given environment). The 
      <em>ε</em>
      <sub>
       <em>ij</em>
      </sub> are the error terms including the GxE and 
      <em>ε</em>
      <sub>
       <em>ij</em>
      </sub> ∼ N (0, σ
      <sup>2</sup>R). From model (3), three parameters were estimated: (i) the genotypic means that is equivalent to the sum (
      <em>µ</em> + 
      <em>G</em>
      <sub>
       <em>i</em>
      </sub>) representing the average performance of a genotype considering all environments; (ii) he 
      <em>β</em>
      <sub>
       <em>i</em>
      </sub> terms (slope), corresponding to genotypic responses to the environments and the variance (VAR) of the 
      <em>ε</em>
      <sub>
       <em>ij</em>
      </sub> terms that is a measurement of non-linear plasticity (
      <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Kusmec et al., 2017</a>). All these parameters were used then to characterize the genotypes according to their individual performance and their stability in the MAGIC-MET design. For every trait, reaction norms were then computed from the model (3).
     </p>
     <p id="p-23">The factorial regression model (4) was further applied to describe the GxE through the genotypic response to the different environmental covariates (Tmin°, Tmax°, Tm°, Amp.Th°, Vpd, RH and SDD). The environmental covariates defined from the daily recorded climatic variables in the greenhouses were used for this purpose. For each trait, the most significant environmental covariate (p-value significant at α = 5%) was first identified – by testing successively the significance of each single covariate – and used as an explanatory variable represented by 
      <em>Cv</em>
      <sub>
       <em>j</em>
      </sub> in model (4). 
      <span class="disp-formula" id="disp-formula-4">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-5.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-5.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-24">The 
      <em>α</em>
      <sub>
       <em>i</em>
      </sub> terms of the model were extracted and considered as a third plasticity parameter (SCv). They represent genotypic sensitivities to the most impacting environmental covariate for each trait. This measurement of plasticity is of interest as it allows identifying the direction and the intensity of each MAGIC line’s sensitivity to a meaningful environmental covariate. Throughout the rest of the document, the ‘slope’ and ‘VAR’ estimated from the Finlay-Wilkinson model and the ‘SCv’ from the factorial regression model will be considered as plasticity phenotypes – all of these parameters being trait-specific.
     </p>
    </div>
    <div id="sec-8" class="subsection">
     <h3>Linkage mapping on the genotypic mean and plasticity phenotypes</h3>
     <p id="p-25">Linkage mapping was carried out with a set of 1345 SNP markers selected from the genome resequencing of the eight parental lines. All the MAGIC lines were genotyped for those SNPs and at each SNP position, the founder haplotype probability was predicted with the function 
      <em>calc_genoprob</em> from R/qtl2 package (
      <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Broman et al., 2019</a>). Founder probabilities were then used with the Haley-Knott regression model implemented in R/qtl2 for QTL detection. The response variables were the genotypic means, slope, VAR and SCv for each trait. To attest for significance, the threshold for all phenotypes was set to a LOD threshold of −log10 (α/number of SNPs) where α was fixed at 5% risk level. The VAR plasticity parameter was log transformed for all traits except fset (sqrt transformation) to meet normality assumption before QTL analysis. The function 
      <em>find_peaks ()</em> of R/qtl2 package was used to detect all peaks exceeding the defined threshold and the LOD score was dropped of two and one units to separate two significant peaks as distinct QTLs and to define the confidence interval of the QTLs, respectively.
     </p>
    </div>
    <div id="sec-9" class="subsection">
     <h3>Multi-environment QTL analysis (QEI)</h3>
     <p id="p-26">The strength of QTL dependence on the environment was tested afterward in a second step by identifying QTLs that significantly interact with the environment (QEI). Two multi-environment forward-backward models (5 &amp;; 6) were used to test at each marker position the effect of the marker x environment interaction. 
      <span class="disp-formula" id="disp-formula-5">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-6.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-6.gif"/>
        </noscript>
       </span>
      </span>
      <span class="disp-formula" id="disp-formula-6">
       <span class="highwire-responsive-lazyload">
        <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-7.gif"/>
        <noscript>
         <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2020/02/07/2020.02.07.938456/embed/graphic-7.gif"/>
        </noscript>
       </span>
      </span>
     </p>
     <p id="p-27">For model (5) and (6), 
      <em>y</em>
      <sub>
       <em>ij</em>
      </sub> represents the phenotype (mean value per genotype and per environment), 
      <em>E</em>
      <sub>
       <em>j</em>
      </sub> reflects the fixed environment effect; 
      <em>α</em>
      <sub>
       <em>kp</em>
      </sub> and 
      <em>β</em>
      <sub>
       <em>kpj</em>
      </sub> represent the main and interactive parental allelic effects (
      <em>p</em>)at marker 
      <em>k</em> and in environment 
      <em>j</em> for 
      <em>β</em>
      <sub>
       <em>kpj</em>
      </sub>; 
      <em>x</em>
      <sub>
       <em>ikp</em>
      </sub> is the probability of the parental allele’s origin for the MAGIC line 
      <em>i</em>; 
      <em>G</em>
      <sub>
       <em>i</em>
      </sub> stands for a random genotype effect and the residual errors including a part of the GxE that is not explained by the detected QTLs are specific to each environment, 
      <em>ε</em>
      <sub>
       <em>ij</em>
      </sub> ∼ N (0, 
      <em>σ</em>
      <sup>2</sup>
      <em>Rj</em>).
     </p>
     <p id="p-28">Significant QEI were declared in a two-step procedure. First, the main QTL and the QEI effects were tested separately in model (5). The QTL detection process was adapted from the script proposed by 
      <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Giraud et al., (2017)</a>. Every marker showing a significant main QTL or QEI was added as a fixed cofactor and the significance of the remaining markers tested again until no more significant marker was found. All markers selected as cofactors were then jointly tested in the backward procedure and only significant QEI after the backward selection are reported. The second procedure used in model (6) to declare QEI consisted in a slight modification of model (5) where 
      <em>β</em>
      <sub>
       <em>kpj</em>
      </sub> represents this time the global (main + interactive) effect of the marker. It allowed the detection of markers that had a main QTL effect or QEI just below the threshold detection but whose global effect is significant when the two components are jointly tested. To determine the threshold level for QEI detection, permutation test were performed 1000 times on the adjusted means with the function 
      <em>sim</em>.
      <em>sightr</em> of mpMap 2.0 R package (
      <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Huang and George, 2011</a>).
     </p>
    </div>
    <div id="sec-10" class="subsection">
     <h3>Data availability</h3>
     <p id="p-29">The phenotypic data, average climatic parameters and genotypic information described in the present study are available at 
      <a href="https://doi.org/10.15454/UVZTAV">https://doi.org/10.15454/UVZTAV</a>. The custom scripts used for the two-stage analysis and QEI modelling are also provided.
     </p>
    </div>
   </div>
   <div class="section" id="sec-11">
    <h2 class="">RESULTS</h2>
    <div id="sec-12" class="subsection">
     <h3>Environment description</h3>
     <p id="p-30">The 12 environmental conditions were described by the daily climatic parameters recorded until the end of flowering of the 4
      <sup>th</sup> truss. Seven environmental covariates were selected, and the environments clustered according to these covariates in four groups (
      <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The first group included all trials from Morocco that were characterized by high thermal amplitude and low Vpd. The control environments in France (Avi12 and Avi17) clustered together in the 2
      <sup>nd</sup> group, defined by low maximal temperatures and high relative humidity. HIs14 clustered alone in the 4
      <sup>th</sup> group and formed the most extreme environment showing very high temperatures and dry climate with low relative humidity. The remaining environments clustered together in the 3
      <sup>rd</sup> and most disparate group.
     </p>
     <p id="p-31">Phenotypic distributions were plotted for each trait regarding the environments where it was evaluated (Supplemental Figure 2) showing a distribution in accordance with the clustering of the environments for some traits (firm, height, nflw and leaf). Other traits such as FW, nfr, SSC and fset showed a distribution pattern with relatively high within-group variability, notably for environments clustering in group 1 from Morocco.</p>
     <div id="F2" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F2.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Pearson[[#x2019]]s correlation between mean and plasticity parameters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174045846" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Pearson[[#x2019]]s correlation between mean and plasticity parameters.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F2.medium.gif" width="440" height="302"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F2.medium.gif" width="440" height="302"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 2:</span>
       <p id="p-32" class="first-child">Pearson’s correlation between mean and plasticity parameters.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-13" class="subsection">
     <h3>GxE in the MAGIC population</h3>
     <p id="p-33">Genotype x environment interaction analysis was carried out after correcting data for micro-environmental heterogeneity and removing outliers. As a first step, variance analysis was conducted with ASReml-R package and the variance components from model (2) used to estimate the proportion of GxE variance (
      <em>prop. σ</em>
      <sup>2</sup>
      <sub>
       <em>GxE</em>
      </sub>) and heritability at the whole design level (
      <em>H</em>
      <sup>2</sup>). Significant GxE was found for every trait and the 
      <em>prop. σ</em>
      <sup>2</sup>
      <sub>
       <em>GxE</em>
      </sub> varied from 0.15 (for nflw) to 0.68 (for leaf). Although GxE was significant, seven out of the ten measured traits showed a higher proportion of genotypic variance compared to GxE (Supplemental Table 3). The broad-sense heritability of the whole design 
      <em>H</em>
      <sup>2</sup> was largely variable according to the trait, varying from 0.18 (nfr) to 0.77 (flw). Its calculation took into account the residual environment-specific variance which showed different range according to the trait, lowering heritability of traits such as nfr and fset (Supplemental Table 3). Furthermore, 
      <em>H</em>
      <sup>2</sup> at the whole design level was lower than the heritability computed in single environment (Supplemental Figure 3).
     </p>
     <div id="F3" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F3.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Representation of plasticity QTLs along the genome. Numbers above the square represent the different chromosomes and the colors distinguished the different traits. The x-axis represents the physical distances in mega base pair (Mbp)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174045846" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Representation of plasticity QTLs along the genome. Numbers above the square represent the different chromosomes and the colors distinguished the different traits. The x-axis represents the physical distances in mega base pair (Mbp).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F3.medium.gif" width="440" height="353"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F3.medium.gif" width="440" height="353"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 3:</span>
       <p id="p-34" class="first-child">Representation of plasticity QTLs along the genome. Numbers above the square represent the different chromosomes and the colors distinguished the different traits. The x-axis represents the physical distances in mega base pair (Mbp).</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <p id="p-35">Afterwards, the proportion of the GxE that could be predicted by the environmental covariates was assessed following the factorial regression model (4). Across traits, different environmental covariates significantly explained the GxE #(Supplemental Figure 4). Considering only the most significant covariate, from 18% (FW) to 47% (fset) of the GxE (proportion of the sum of squares) could be reliably attributed to the responses of genotypes to climatic parameters measured within the greenhouses. To perform the factorial regression model (4), the most important environmental covariate was first identified for each trait (Supplemental Figure 4). Growth traits, height and leaf were for example mostly affected by the thermal amplitude and maximal temperature, respectively, while yield component traits, FW and nfr were particularly sensitive to the sum of degree day. The vapour pressure deficit (Vpd, kPa) was the most important environmental factor affecting firm, fset and SSC. Flowering time (flw) and nflw were mostly affected by minimal temperatures and relative humidity, respectively. Stem diameter was the only trait for which none of the environmental covariates significantly affected the trait.</p>
     <div id="F4" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F4.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Number of QTLs identified specifically on mean, plasticity or QEI and QTLs that were common to at least two of them." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174045846" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Number of QTLs identified specifically on mean, plasticity or QEI and QTLs that were common to at least two of them.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F4.medium.gif" width="440" height="381"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F4.medium.gif" width="440" height="381"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 4:</span>
       <p id="p-36" class="first-child">Number of QTLs identified specifically on mean, plasticity or QEI and QTLs that were common to at least two of them.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-14" class="subsection">
     <h3>Phenotypic plasticity</h3>
     <p id="p-37">Three different parameters were used to quantify phenotypic plasticity in the MAGIC-MET design. For each trait, the slope and VAR from the Finlay-Wilkinson regression model and the genotypic sensitivity to the most important environmental covariate (SCv) from the factorial regression model were extracted. A large genetic variability was observed for plasticity of all traits (Supplemental Figure 5 and Supplemental Figure 6). Besides, significant correlations were found between the mean phenotypes and plasticity parameters (
      <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) for most of the traits. The best average-performing genotypes were usually the most responsive to environmental variation as highlighted by the positive correlation between the genotypic means and slope from the Finlay-Wilkinson regression model. The majority of the MAGIC lines responded in the same direction to the environmental quality and only a few genotypes (none in the case of height) showed negative reaction norms; however, more divergent shapes of reaction norms were observed from the factorial regression model (Supplemental Figure 5).
     </p>
     <div id="F5" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F5.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Physical positions of the MAGIC-MET QTLs for fruit weight and flowering time. The following circle with black bars represents the different domestication/improvement sweep regions identified in (Zhu et al. 2018). The other circles plot the CI of QTLs identified on mean (green), plasticity (orange) or with QEI analysis (purple)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174045846" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Physical positions of the MAGIC-MET QTLs for fruit weight and flowering time. The following circle with black bars represents the different domestication/improvement sweep regions identified in (Zhu et al. 2018). The other circles plot the CI of QTLs identified on mean (green), plasticity (orange) or with QEI analysis (purple).&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F5.medium.gif" width="440" height="248"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F5.medium.gif" width="440" height="248"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 5:</span>
       <p id="p-38" class="first-child">Physical positions of the MAGIC-MET QTLs for fruit weight and flowering time. The following circle with black bars represents the different domestication/improvement sweep regions identified in (
        <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhu et al. 2018</a>). The other circles plot the CI of QTLs identified on mean (green), plasticity (orange) or with QEI analysis (purple).
       </p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
     <div id="F6" class="fig pos-float type-figure odd">
      <div class="highwire-figure">
       <div class="fig-inline-img-wrapper">
        <div class="fig-inline-img">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F6.large.jpg?width=800&amp;;height=600&amp;;carousel=1" title="Correlation of the estimated allelic effect for consistent QTLs between mean and plasticity phenotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1174045846" data-figure-caption="&lt;;div class=&quot;;highwire-markup&quot;;&gt;;Correlation of the estimated allelic effect for consistent QTLs between mean and plasticity phenotypes.&lt;;/div&gt;;" data-icon-position="" data-hide-link-title="0">
          <span class="hw-responsive-img">
           <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F6.medium.gif" width="440" height="278"/>
           <noscript>
            <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F6.medium.gif" width="440" height="278"/>
           </noscript>
          </span>
         </a>
        </div>
       </div>
       <ul class="highwire-figure-links inline">
        <li class="download-fig first">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">Download figure</a>
        </li>
        <li class="new-tab last">
         <a href="https://www.biorxiv.org/content/biorxiv/early/2020/02/07/2020.02.07.938456/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a>
        </li>
       </ul>
      </div>
      <div class="fig-caption">
       <span class="fig-label">Figure 6:</span>
       <p id="p-39" class="first-child">Correlation of the estimated allelic effect for consistent QTLs between mean and plasticity phenotypes.</p>
       <div class="sb-div caption-clear"/>
      </div>
     </div>
    </div>
    <div id="sec-15" class="subsection">
     <h3>QTL mapping</h3>
     <p id="p-40">We used genotypic means and plasticity measurements for every trait as input phenotypes to decipher the genetic architecture of tomato response to abiotic stresses. Considering the 10 traits evaluated, a total of 104 unique QTLs were identified for genotypic means and the plasticity parameters (Supplemental Table 4). The proportion of QTLs shared between mean and plasticity was about 21%, lower than QTLs that were plasticity or mean specific (79%). Considering only the 63 plasticity QTLs, 11 and 7 QTLs were specifically detected with the SCv and VAR plasticity parameters. Plasticity QTLs were detected on every chromosome (
      <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>); however, the chromosome 1 showed the highest number with 12 plasticity QTLs. In this chromosome, plasticity QTLs were detected at least once for every trait. The chromosome 11 carried a total of 11 plasticity QTLs and interestingly all these QTL (except ppnflw11.1) co-localized in a short region of the chromosome between 52 and 55 Mbp. The chromosomes 5, 6 and 10 showed the lowest number (only 3) of plasticity QTLs. For QTLs detected on genotypic means, the number of QTLs per chromosome varied from 2 QTLs on chromosomes 6 and 10 to one QTL on chromosome 1.
     </p>
    </div>
    <div id="sec-16" class="subsection">
     <h3>QTL-by-environment analysis (QEI)</h3>
     <p id="p-41">Multi-environment forward-backward models were used to assess the significance and the strength of the QTL effects across environments. The QEI analysis was conducted in two steps using the same set of 1345 SNP markers that were also used for linkage mapping analysis. This analysis yielded 28 QEI (only those showing significant interaction) for the 10 traits (Supplemental Table 4). The number of QEI varied from 0 QEI for nfr to 6 QEI for flw. These two traits also demonstrated the lowest and highest 
      <em>H</em>
      <sup>2</sup>.
     </p>
     <p id="p-42">All QEI identified in this step were confronted to the plasticity and genotypic means QTLs using the physical positions of the QTLs and their confidence intervals. Interestingly, this comparison revealed that all the detected QEI were also identified using either genotypic means or plasticity parameters, in the linkage mapping analysis, except two QEI located on the same region of chromosome 6 (flw6.1 and firm6.1). Among the 106 unique QTLs identified on genotypic means, PP and QEI, a notable number of QTLs were specific representing 30 and 32% for plasticity and genotypic means, respectively (
      <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Eight QTLs involving five different traits (flw1.1, fw2.1, fw2.2, fw11.2, leaf6.1, nflw11.2, SSC1.2 and SSC9.1) were identified with all the three approaches highlighting their robustness and susceptibility to environmental variation.
     </p>
    </div>
    <div id="sec-17" class="subsection">
     <h3>Genetic location of the MAGIC-MET QTLs</h3>
     <p id="p-43">The physical positions based on the SL2.50 version of the reference genome, were used to compare the position of the different QTL category (genotypic means, plasticity or QEI). Indeed, a recent study has identified different tomato regions (Sweep regions) that were selected during domestication and improvement events (
      <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Zhu et al., 2018</a>). These regions were cross checked against the positions of our QTLs. Some QTLs detected in the MAGIC-MET design were located in large regions thus colocating with a high number of Sweep regions (
      <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> &amp;; Supplemental Figure 7). Thus, considering only the QTLs with CI lower than 2Mbp intervals and all QEI, a total of 61 QTLs were selected and compared with the Sweep regions. Plasticity QTLs appeared to be in majority located within the Sweep regions and only 6% of the selected plasticity QTLs were outside the domestication/improvement selective sweeps (Supplemental Figure 8). Interestingly, the Sweep region SW75 located in chromosome 3 (between 64.76 and 65.01 Mbp) carried a total of five QTLs (ht3.1, fset3.1, flw3.2, leaf3.1, fset3.1). The Supplemental Table 5 presents all the Sweep regions holding at least one MAGIC-MET QTL. Chromosome 11 was highlighted as holding a number of plasticity QTLs for different traits (
      <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Indeed, seven different QTLs all identified with plasticity parameters, were located within the Sweep regions SW254 and SW255, from 53.81 – 55.62 Mbp on chromosome 11 (Supplemental Figure 9). Among the ten QTLs that were outside the Sweep regions, one QTL was identified for mean FW and located on chromosome 5 (fw5.1) in position 4.52 Mbp. This QTL was mapped in a region holding other QTLs segregating in the MAGIC population for fruit size, fruit width and fruit length (Supplemental Table 6; data from the experiment in 
      <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Pascual et al. (2015)</a>).
     </p>
    </div>
    <div id="sec-18" class="subsection">
     <h3>Candidate genes</h3>
     <p id="p-44">Confidence intervals (CI) of the MAGIC-MET QTLs varied from 0.45Mbp to 87Mbp including a variable number of genes. We thus focused on QTLs presenting CI regions smaller than 2Mbp for CG screening. From 49 (nflw12.1) to 256 (diam4.1) genes were within the regions of the selected QTLs. Taking advantage of the parental allelic effect, the CG were narrowed for each QTL by contrasting the allelic effect of the eight parental lines. The selected candidates after the filtering procedure are presented in Supplemental Table 7, highlighting interesting candidates for further studies. Flowering time QTLs for instance included some CG with consistent matching regarding their functional annotation. For example, the CI of the QTL ppflw11.1 on chromosome 11 included two CG: Solyc11g070100 and Solyc11g071250 corresponding to “Early flowering protein” (ELF) and “EMBYO FLOWERING 1-like protein” (EMF1), respectively. Among other potential flowering candidates, we noticed Solyc12g010490 (AP2-like ERF) for the QTL flw12.1 and Solyc03g114890 and Solyc03g114900 (COBRA-like proteins) for the QTL flw3.2. Aside flowering time, the selected candidate genes for the QTLs diam4.1 and ppSSC1.1 included the Solyc04g081870 (annotated as an Expansin gene) and Solyc01g006740 (annotated as Sucrose phosphate phosphatase) genes, respectively.</p>
     <p id="p-45">We could identify some plasticity QTLs showing sensitivity to the environmental conditions, notably the QTLs detected using the Scv plasticity parameter. Candidate genes were screened for some QTLs falling into this category. The ppfw9.1 QTL CI for example, showing susceptibility to the sum of degree day (SDD), carried a chaperone candidate (solyc09g091180) which might be involved in regulating fruit weight depending on the SDD variation. Similarly, the QTL ppleaf11.1 is affected by the maximal temperature (Supplemental Table 4). Three CG (Solyc11g071830, Solyc11g071930 and Solyc11g071710) belonging to the Chaperone J-domain family, were retained after the filtering procedure in the region of this QTL. Interestingly, the DnaJ-like zinc-finger gene (Solyc11g071710) was among the candidates corresponding to several plasticity QTL including ppflw11.1, ppleaf11.1, ppnflw11.1, ppht11.1 and ppdiam11.2. This gene presented a total of 122 polymorphisms across the eight parental lines among which 35 and 68 are in the up-stream and down-stream gene region. Further investigation regarding this gene is needed to state its potential pleiotropic effect.</p>
    </div>
   </div>
   <div class="section" id="sec-19">
    <h2 class="">DISCUSSION</h2>
    <div id="sec-20" class="subsection">
     <h3>Genetic variability in tomato response to environmental variation</h3>
     <p id="p-46">Genotype x environment interaction is a long-standing challenge for breeders and the predicted climate change has encouraged plant geneticists to devote more attention into understanding its genetic basis. Tomato is a widely cultivated crop adapted to a variety of environmental conditions (
      <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Rothan et al. 2019</a>). However, important incidences of abiotic stress in the final productivity, fruit quality and reproductive performance have been noticed (
      <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Albert et al. 2016</a>; 
      <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Estañ et al. 2009</a>; 
      <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mitchell et al. 1991</a>; 
      <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Xu et al. 2017</a>). We quantified the level of GxE and the subjacent phenotypic plasticity in a multi-environment and multi-stress trial – involving induced water-deficit, salinity and heat stresses – in a highly recombinant tomato population. An important genetic variability was observed for the plasticity traits related to yield, fruit quality, plant growth and phenology (Supplemental Figure 6). This highlights the interest of the MAGIC population as a valuable resource for tomato breeding in dynamic changing environments. Tomato wild species have been also characterized as an important reservoir for abiotic stress tolerance genes (
      <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Foolad, 2007</a>). However, their effective use in breeding programs could be difficult due to undesirable linkage drag, notably for fruit quality. Unlikely, the MAGIC population characterized here is an intra-specific population with high diversity regarding fruit quality components, which provides a great advantage as a breeding resource compared to wild populations.
     </p>
     <p id="p-47">Several statistical models are available to explore, describe and predict GxE in plants (
      <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Yan et al., 2007</a>; 
      <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Malosetti et al., 2013</a>). Factorial regression model is among the most attractive as it allows to describe the observed GxE regarding relevant environmental information. We used the factorial regression model with different environmental covariates that are readily accessible from year to year, which allowed us to predict a variable proportion of the observed GxE (Supplemental Figure 4). Besides, each MAGIC line was characterized for its sensitivity to the growing climatic conditions opening avenues to effectively select the most interesting genotypes for further evaluation in breeding programs targeting stressful environments.
     </p>
     <p id="p-48">Interestingly we found significant correlation between the genotypic sensitivities to the different environmental covariates and slopes from the Finlay-Wilkinson regression model (Supplemental Figure 10). This emphasizes the adequacy of the selected environmental covariates to explain differences observed in the average performance of the genotypes across environments. Conversely, slope and VAR showed less significant correlations, although they were both correlated to mean phenotypes in the same direction – except for SSC (
      <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). This may be induced by distinct genetic regulation of these two plasticity parameters which reflect different types of agronomic stability (
      <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lin et al. 1986</a>). Indeed, we identified 7 and 14 plasticity QTLs that were specific to VAR and slope, respectively (Supplemental Table 4). The correlation pattern of the different plasticity parameters evokes a complex regulation of plasticity which besides is seemingly trait specific.
     </p>
     <p id="p-49">Significant correlation at phenotypic level might result from the action of pleiotropic genes. The 
      <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> displays the correlations between genotypic means and plasticity which were significant for almost every trait at variable degree. These correlations were reflected at the genetic level by 22 QTLs overlapping between genotypic mean and plasticity parameters, representing about 21% of all identified QTLs. However, a high proportion of the QTLs were specific either to genotypic means or plasticity parameters (Supplemental Figure 11), hence suggesting the action of both common and distinct genetic loci in the control of mean phenotype and plasticity variation in tomato.
     </p>
    </div>
    <div id="sec-21" class="subsection">
     <h3>Genomic location of the MAGIC-MET QTLs</h3>
     <p id="p-50">The availability of substantial genomic information in tomato enabled the identification of different genomic regions which have undergone selective sweeps which were strongly selected during the domestication and improvement process (
      <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lin et al. 2014</a>; 
      <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Zhu et al. 2018</a>). When projected on the physical positions of the tomato reference genome (SL2.50 version), most of the plasticity QTLs we identified were located within the sweep regions defined by 
      <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Zhu et al. (2018)</a>. It therefore suggests that plasticity might have been selected together with other interesting agronomic traits during tomato domestication and improvement. For instance, this is corroborated by the positive correlation between slope (from the Finlay-Wilkinson regression model) and mean fruit weight variation. Indeed, genotypes with higher FW slope are characterized by good adaptability in high quality environments and will likely be intended to selection. Co-selection of allelic variants leading to higher performance in optimal condition together with plasticity alleles is a realistic assumption that would explain the significant correlation that we observed between the genotypic means and plasticity. In rice for instance, 
      <em>GhD7</em> has been described as a key high-yield gene simultaneously involved in the regulation of plasticity of panicle and tiller branching and involved in abiotic stress response (
      <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Herath 2019</a>). This example highlights a gene carrying different allelic variants affecting together plasticity and mean phenotypes. Further investigations are needed to assess how domestication and breeding have affected plasticity in tomato and other crop species.
     </p>
     <p id="p-51">An important genomic region involved in the genetic regulation of plasticity for six different traits was identified in chromosome 11 (Supplemental Figure 9). This region is obviously a regulatory hub carrying interesting plasticity genes. It remains to determine if the co-localization of the different plasticity QTLs in this region is due to the action of a pleiotropic gene or different linked genes. Nevertheless, the chromosome 11 region highlighted here is an interesting target for breeding as well as for understanding the functional mechanisms of plasticity genes.</p>
    </div>
    <div id="sec-22" class="subsection">
     <h3>Allelic-sensitivity vs gene-regulatory model</h3>
     <p id="p-52">Sixty-three plasticity QTLs were identified among which 22 (35%) were also identified when using the genotypic means; and 41 (65%) were specific to plasticity. 
      <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Via et al. (1995)</a> proposed two genetic models – the allelic-sensitivity and gene-regulatory models – among the mechanisms involved in the genetic control of phenotypic plasticity. These two models are distinguishable through QTL analysis (
      <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ungerer et al., 2003</a>) with the expectation that allelic-sensitivity model will lead to co-localization of genotypic means and plasticity QTLs, while a distinct location of QTLs affecting mean and plasticity will likely correspond to the gene-regulatory model (
      <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Kusmec et al., 2017</a>). Regarding our results, both models are suspected to regulate tomato plasticity, even though the gene-regulatory model is predominant with 65% of the plasticity QTLs that did not co-localize with genotypic means QTLs for the same trait. In maize, using a larger number of environments and traits, 
      <a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Kusmec et al. (2017)</a> found similar results and even a higher rate of distinct locations of plasticity and mean QTLs. Studying plasticity as a trait 
      <em>per se</em> is therefore of a major interest since breeding in both direction (considering the mean phenotype and its plasticity) is achievable. Through transcriptomic analyses, 
      <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Albert et al. (2018)</a> observed that genotype x water deficit interaction was mostly associated to 
      <em>trans-</em>acting genes which could be assimilated to the gene-regulatory model in agreement with our results.
     </p>
     <p id="p-53">Although the distinct location of QTLs detected on plasticity and genotypic means could be confidently assigned to the action of genes in interaction, their co-localization is not necessarily a case of allelic-sensitivity regulation, especially if the QTL is in a large region. Indeed, the allelic-sensitivity model assumes that a constitutive gene is directly sensitive to the environment regulating its expression across different environmental conditions, inducing hence phenotypic plasticity. This is a very strong hypothesis regarding the QTLs since the overlapping region between genotypic means and plasticity could carry different causal variants in strong linkage disequilibrium affecting either mean phenotype or plasticity. Thus, co-locating mean and plasticity QTLs should be not automatically imputed to the allelic-sensitivity model. We found a total of 22 constitutive QTLs between genotypic means and plasticity for all 10 measured traits (Supplemental Table 4). Considering the estimated QTL effects, the variation patterns of the eight parental allelic classes were compared between mean and PP QTL of the same trait. Only ten QTL showed consistent allelic effects (Spearman correlation significant at 0.05 threshold level) strengthening the hypothesis of the allelic-sensitivity model for these QTLs (
      <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Further studies should help to elucidate and validate the candidate plasticity genes and to clarify their functional mechanism.
     </p>
    </div>
    <div id="sec-23" class="subsection">
     <h3>Complementary methods to identify environment-responsive QTLs</h3>
     <p id="p-54">Different approaches have been proposed in the literature to dissect GxE into its genetic components (
      <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Malosetti et al., 2013</a>; 
      <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">El-Soda et al., 2014</a>). We used a mixed linear model with a random genetic effect accounting for the correlation structure of the MAGIC-MET design to identify the QEI. Extending the use of mixed linear models to MAGIC populations in the framework of MET analysis has been very rarely applied in crops. To our knowledge, only 
      <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Verbyla et al., (2014)</a> applied such approach in wheat and identified QEI for flowering time. Our model was adequate to account for the complex mating design of the MAGIC population by using the haplotype probabilities. Indeed, it allows estimating the QTL effect for each parental allelic class and for each environment at every SNP marker. Overall, 28 QEI were detected showing significant marker x environment interaction for ten traits.
     </p>
     <p id="p-55">Methods using plasticity as a trait 
      <em>per se</em> are also attractive to identify environmentally sensitive QTLs. This strategy was applied in maize, sunflower, barley and soybean to detect the loci governing GxE (
      <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Lacaze et al., 2009</a>; 
      <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gage et al., 2017</a>; 
      <a id="xref-ref-25-6" class="xref-bibr" href="#ref-25">Kusmec et al., 2017</a>; 
      <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Mangin et al., 2017</a>; 
      <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Xavier et al., 2018</a>). With different plasticity parameters, we identified a total of 63 plasticity QTLs and only 24% were also identified with the QEI models. Thus, both methods, using plasticity or mixed linear models, are complementary approaches to study the genetic component of GxE.
     </p>
    </div>
    <div id="sec-24" class="subsection">
     <h3>Candidate genes</h3>
     <p id="p-56">Multi-parental populations are powerful for QTL mapping studies (
      <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Huang et al. 2012</a>; 
      <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kover et al. 2009</a>) and are besides interesting for fine mapping and candidate gene screening. 
      <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrero et al. (2015)</a> for instance considered the variation of the QTL effect estimated for the different parental lines, combined with transcriptomic analyses to efficiently identify candidate genes. Similarly, 
      <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Septiani et al. (2019)</a> narrowed candidate genes for Fusarium resistance in a maize MAGIC population using allelic effect of the MAGIC parents.
     </p>
     <p id="p-57">A number of candidate genes were proposed in our study, affecting both genotypic means and plasticity variation. These candidate genes were selected based on the parental allelic effect and represent valuable targets for future studies attempting to characterize the molecular mechanisms underlying plasticity in tomato. Indeed, relevant candidate genes were proposed for plasticity of flowering time including the Solyc11g071250 which corresponds to an “EMBYO FLOWERING 1-like protein” (EMF1). The implication of EMF1 in flowering time has been observed in Arabidopsis by 
      <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aubert et al., (2001)</a> who highlighted an indirect effect of EMF1 on flowering time and inflorescence. More recently, 
      <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Luo et al., (2018)</a> outlined the role of EMF1 interacting with CONSTANS proteins in a complex pathway to regulate the expression of flowering time genes in Arabidopsis. Solyc11g070100 which is annotated as “Early flowering protein” (ELF) gene is also an interesting candidate for flowering time regulation. It was observed across species that a consistent expression of ELF3 can extend the rapid transition to flowering (
      <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang et al., 2017</a>). ELF3 loss of function is therefore expected to trigger early flowering according to these authors. Interestingly, Solyc11g070100 was affected by 69 SNPs and 14 INDELs polymorphisms, among which only one SNP showed polymorphism variation in line with the estimated allelic effect for the eight parental lines at this QTL. This SNP was localized at the position 54,632,225 bp in chromosome 11, upstream the gene Solyc11g070100. The parent LA1420 carried the reference allele at this SNP while the remaining parents held the alternative allele. Considering the estimated allelic effects at this QTL, we could assume that the LA1420 allele variant might induce an early flowering phenotype comparatively to the other parents.
     </p>
    </div>
    <div id="sec-25" class="subsection">
     <h3>Conclusion</h3>
     <p id="p-58">We aimed to dissect the genetic architecture of tomato response to different environments involving control and stress growing conditions. The MAGIC population demonstrated a large genetic variability in response to abiotic stresses which was reflected by the identification of 63 plasticity QTLs. This was achieved through the use of different plasticity parameter highlighting the importance of plasticity quantification for deciphering its genetic basis. The plasticity QTLs were in majority (65% of the plasticity QTLs) located in distinct regions than the QTLs detected for the mean phenotypes, suggesting a specific genetic control of mean trait variation and plasticity at some extent. Using plasticity as a trait 
      <em>per se</em> in mapping analysis turned out to be a good method for identifying genetic regions underlying GxE. Almost all the QEI were also identified for at least one of the plasticity parameters. Overall, this study presents the MAGIC population as a powerful resource for tomato breeding under abiotic stress conditions, as well as for understanding the genetic mechanisms regulating tomato response to environmental variation.
     </p>
    </div>
   </div>
   <div class="section" id="sec-26">
    <h2 class="">Supplemental files</h2>
    <p id="p-60">
     <strong>Supplemental Figure 1:</strong> Selection of 7 environmental covariates for the factorial regression model. Three periods – each of 20 days – were defined from planting to the end of flowering on the 4
     <sup>th</sup> truss. The period from 20 to 60 days after planting (DAP) covered vegetative growth and flowering on the 4
     <sup>th</sup> truss and the measured climatic variables averaged during this period. The different environmental covariates are described
    </p>
    <p id="p-61">
     <strong>Supplemental Figure 2:</strong> Boxplot distribution of the traits across environments. The colors of the boxplot are according to the groups defined by clustering of the environments
    </p>
    <p id="p-62">
     <strong>Supplemental Figure 3:</strong> Heritability in the MAGIC-MET design. For each trait, heritability was computed at every environment and plotted with heritability of the full design 
     <strong>
      <em>H</em>
     </strong>
     <sup>
      <strong>2</strong>
     </sup> (in green)
    </p>
    <p id="p-63">
     <strong>Supplemental Figure 4:</strong> Proportion of the sum of square attributed to the different factors in the factorial regression model. For each trait, the orange and green stacked bars represent the proportion of the SSq explained by the Genotype and Environment factors in model (4). The remaining colors represent the effect part of the GxE that could be explained by the different environmental covariates. Only significant covariates were highlighted within the bars.
    </p>
    <p id="p-64">
     <strong>Supplemental Figure 5:</strong> Reaction norms from the Finlay-Wilkinson regression model (A) and the factorial regression model (B). In 
     <a id="xref-fig-5-2" class="xref-fig" href="#F5">figure 5 A</a>, the blue and orange lines represent the positive and negative reaction norms. In 
     <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a> B, the green and purple lines represent the positive and negative reaction norms
    </p>
    <p id="p-65">
     <strong>Supplemental Figure 6:</strong> Histogram distribution of mean and all plasticity parameters for each trait
    </p>
    <p id="p-66">
     <strong>Supplemental Figure 7:</strong> Physical positions of the MAGIC-MET QTLs for diam, leaf, height, fset, nflw, nfr, firm and SSC. The outer circle with gray font represents the known and cloned QTL/gene for each trait. The following circle with black bars represents the different domestication/improvement sweep regions identified in (
     <a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Zhu et al. 2018</a>). The other circles plot the CI of QTLs identified on mean (green), plasticity (orange) or with QEI analysis (purple)
    </p>
    <p id="p-67">
     <strong>Supplemental Figure 8:</strong> Number of the MAGIC-MET QTLs identified within or outside the domesticated/improved regions. Only the MAGIC-MET QTLs within short CI (lower than 2Mbp) were considered. The response specific category included QEI and plasticity specific QTLs; the common category correspond to QTLs that were commonly identified on mean, plasticity and QEI or at least two of them
    </p>
    <p id="p-68">
     <strong>Supplemental Figure 9:</strong> Zoom plot on Chromosome 11 region from 53-57 Mbp. Each color represents a different QTL located in this region and the top black bars are the Sweep regions SW254 and SW255
    </p>
    <p id="p-69">
     <strong>Supplemental Figure 10:</strong> Correlation between the genotypic sensitivities to environmental covariates from the factorial regression model and slopes from the Finlay-Wilkinson regression model
    </p>
    <p id="p-70">
     <strong>Supplemental Figure 11:</strong> Venn diagram of the number of QTL specific or commonly detected with mean, PP or using the QEI models.
    </p>
    <p id="p-71">
     <strong>Supplemental Table 1:</strong> Description of the MAGIC-MET design with the 12 environments and their respective names
    </p>
    <p id="p-72">
     <strong>Supplemental Table 2:</strong> Description of the phenotypic traits evaluated in the MAGIC-MET design
    </p>
    <p id="p-73">
     <strong>Supplemental Table 3:</strong> Estimates of the variance components from model (2)
    </p>
    <p id="p-74">
     <strong>Supplemental Table 4:</strong> Results of QTL and QEI analysis in the MAGIC-MET design
    </p>
    <p id="p-75">
     <strong>Supplemental Table 5:</strong> Genetic location of the MAGIC-MET QTLs overlapping with the Sweep (domestication/improvement) regions.
    </p>
    <p id="p-76">
     <strong>Supplemental Table 6:</strong> QTLs identified for fruit size, fruit width and fruit length in the MAGIC population
    </p>
    <p id="p-77">
     <strong>Supplemental Table 7:</strong> Selected candidate genes for all the mean and plasticity QTLs located within 2Mbp CI region
    </p>
   </div>
   <div class="section ack" id="ack-1">
    <h2 class="">Acknowledgements</h2>
    <p id="p-59">We acknowledge the greenhouse staff of INRA GAFL, Gautier SEMENCES and Hazera seeds for the trial management. The ANR project Adaptom (ANR-13-ADAP-0013) and TomEpiSet (ANR-16-CE20-0014) supported this work. ID was supported by the WAAPP (West Africa Agricultural Productivity Project) fellowship and hosted as Ph.D. student in the INRA GAFL.</p>
   </div>
   <div class="section ref-list" id="ref-list-1">
    <h2 class="">References</h2>
    <ol class="cit-list ref-use-labels">
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.1">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Albert</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Duboscq</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Latreille</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Santoni</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Beukers</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bouchet</span>, 
          <span class="cit-name-given-names">J.-P.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">Allele specific expression and genetic determinants of transcriptomic variations in response to mild water deficit in tomato</span>. 
         <abbr class="cit-jnl-abbrev">Plant J</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.2" data-doi="10.1093/jxb/erw411">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Albert</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Segura</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gricourt</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bonnefoi</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Derivot</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Causse</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2016</span>) 
         <span class="cit-article-title">Association mapping reveals the genetic architecture of tomato response to water deficit: focus on major fruit quality traits</span>. 
         <abbr class="cit-jnl-abbrev">J Exp Bot</abbr>. 
         <span class="cit-vol">67</span>: 
         <span class="cit-fpage">6413</span>–
         <span class="cit-lpage">6430</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BBot%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferw411%26rft_id%253Dinfo%253Apmid%252F27856709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/jxb/erw411&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=27856709&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.3" data-doi="10.1105/tpc.13.8.1865">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Aubert</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Moon</span>, 
          <span class="cit-name-given-names">Y.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Martin</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Castle</span>, 
          <span class="cit-name-given-names">L.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yang</span>, 
          <span class="cit-name-given-names">C.H.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2001</span>) 
         <span class="cit-article-title">EMF1, a novel protein involved in the control of shoot architecture and flowering in Arabidopsis</span>. 
         <abbr class="cit-jnl-abbrev">Plant Cell</abbr>. 
         <span class="cit-vol">13</span>: 
         <span class="cit-fpage">1865</span>–
         <span class="cit-lpage">75</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DAubert%26rft.auinit1%253DD.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D1865%26rft.epage%253D1875%26rft.atitle%253DEMF1%252C%2BA%2BNovel%2BProtein%2BInvolved%2Bin%2Bthe%2BControl%2Bof%2BShoot%2BArchitecture%2Band%2BFlowering%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.13.8.1865%26rft_id%253Dinfo%253Apmid%252F11487698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjEzLzgvMTg2NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIwLzAyLzA3LzIwMjAuMDIuMDcuOTM4NDU2LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.4" data-doi="10.1186/s13059-015-0665-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Barrero</span>, 
          <span class="cit-name-given-names">J.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cavanagh</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Verbyla</span>, 
          <span class="cit-name-given-names">K.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Tibbits</span>, 
          <span class="cit-name-given-names">J.F.G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Verbyla</span>, 
          <span class="cit-name-given-names">A.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">B.E.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Transcriptomic analysis of wheat near-isogenic lines identifies PM19-A1 and A2 as candidates for a major dormancy QTL</span>. 
         <abbr class="cit-jnl-abbrev">Genome Biol</abbr>. 
         <span class="cit-vol">16</span>: 
         <span class="cit-fpage">93</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D16%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0665-6%26rft_id%253Dinfo%253Apmid%252F25962727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/s13059-015-0665-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25962727&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.5" data-doi="10.1016/S0065-2660(08)60048-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Bradshaw</span>, 
          <span class="cit-name-given-names">A.D.</span>
         </span> (
         <span class="cit-pub-date">1965</span>) 
         <span class="cit-article-title">Evolutionary Significance of Phenotypic Plasticity in Plants</span>. 
         <abbr class="cit-jnl-abbrev">Adv Genet</abbr>. 
         <span class="cit-vol">13</span>: 
         <span class="cit-fpage">115</span>–
         <span class="cit-lpage">155</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAdv%2BGenet%26rft.volume%253D13%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252808%252960048-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0065-2660(08)60048-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.6" data-doi="10.1534/genetics.118.301595">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Broman</span>, 
          <span class="cit-name-given-names">K.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gatti</span>, 
          <span class="cit-name-given-names">D.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Simecek</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Furlotte</span>, 
          <span class="cit-name-given-names">N.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Prins</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sen</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2019</span>) 
         <span class="cit-article-title">R/qtl2: Software for Mapping Quantitative Trait Loci with High-Dimensional Data and Multiparent Populations</span>. 
         <abbr class="cit-jnl-abbrev">Genetics</abbr>. 
         <span class="cit-vol">211</span>: 
         <span class="cit-fpage">495</span>–
         <span class="cit-lpage">502</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301595%26rft_id%253Dinfo%253Apmid%252F30591514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjExLzIvNDk1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDcvMjAyMC4wMi4wNy45Mzg0NTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.7" data-doi="10.1186/1471-2164-14-791">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Causse</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Desplat</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pascual</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Le Paslier</span>, 
          <span class="cit-name-given-names">M.-C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sauvage</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bauchet</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Whole genome resequencing in tomato reveals variation associated with introgression and breeding events</span>. 
         <abbr class="cit-jnl-abbrev">BMC Genomics</abbr>. 
         <span class="cit-vol">14</span>: 
         <span class="cit-fpage">791</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D791%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-791%26rft_id%253Dinfo%253Apmid%252F24228636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-791&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24228636&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.8" data-doi="10.1111/j.1672-9072.2007.00556.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Costa</span>, 
          <span class="cit-name-given-names">J.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ortuño</span>, 
          <span class="cit-name-given-names">M.F.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Chaves</span>, 
          <span class="cit-name-given-names">M.M.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">Deficit Irrigation as a Strategy to Save Water: Physiology and Potential Application to Horticulture</span>. 
         <abbr class="cit-jnl-abbrev">J Integr Plant Biol</abbr>. 
         <span class="cit-vol">49</span>: 
         <span class="cit-fpage">1421</span>–
         <span class="cit-lpage">1434</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.volume%253D49%26rft.spage%253D1421%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1672-9072.2007.00556.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1672-9072.2007.00556.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000249898500003&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Diouf</span>, 
          <span class="cit-name-given-names">I.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Derivot</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bitton</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pascual</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Causse</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">Water Deficit and Salinity Stress Reveal Many Specific QTL for Plant Growth and Fruit Quality Traits in Tomato</span>. 
         <abbr class="cit-jnl-abbrev">Front Plant Sci</abbr>. 
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">279</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront%2BPlant%2BSci%26rft.volume%253D9%26rft.spage%253D279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.10">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Driedonks</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wolters-Arts</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huber</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Boer</span>, 
          <span class="cit-name-given-names">G.-J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vriezen</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mariani</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">Exploring the natural variation for reproductive thermotolerance in wild tomato species</span>. 
         <abbr class="cit-jnl-abbrev">Euphytica</abbr>. 
         <span class="cit-vol">214</span>: 
         <span class="cit-fpage">67</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D214%26rft.spage%253D67%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.11">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">El-Soda</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Malosetti</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zwaan</span>, 
          <span class="cit-name-given-names">B.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Koornneef</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Aarts</span>, 
          <span class="cit-name-given-names">M.G.M.</span>
         </span> (
         <span class="cit-pub-date">2014</span>). 
         <abbr class="cit-jnl-abbrev">Trends Plant Sci</abbr>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.12" data-doi="10.1007/s00122-008-0900-6">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Estañ</span>, 
          <span class="cit-name-given-names">M.T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Villalta</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bolarín</span>, 
          <span class="cit-name-given-names">M.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carbonell</span>, 
          <span class="cit-name-given-names">E.A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Asins</span>, 
          <span class="cit-name-given-names">M.J.</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Identification of fruit yield loci controlling the salt tolerance conferred by solanum rootstocks</span>. 
         <abbr class="cit-jnl-abbrev">Theor Appl Genet</abbr>. 
         <span class="cit-vol">118</span>: 
         <span class="cit-fpage">305</span>–
         <span class="cit-lpage">312</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DEstan%26rft.auinit1%253DM.%2BT.%26rft.volume%253D118%26rft.issue%253D2%26rft.spage%253D305%26rft.epage%253D312%26rft.atitle%253DIdentification%2Bof%2Bfruit%2Byield%2Bloci%2Bcontrolling%2Bthe%2Bsalt%2Btolerance%2Bconferred%2Bby%2Bsolanum%2Brootstocks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0900-6%26rft_id%253Dinfo%253Apmid%252F18846361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s00122-008-0900-6&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=18846361&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.13">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Finlay</span>, 
          <span class="cit-name-given-names">B.K.W.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Wilkinson</span>, 
          <span class="cit-name-given-names">G.N.</span>
         </span> (
         <span class="cit-pub-date">1963</span>) 
         <span class="cit-article-title">THE ANALYSIS OF ADAPTATION IN A PLANT-BREEDING PROGRAMME The ability of some crop varieties to perform well over a wide range of environmental conditions has lcng been appreciated by the agronomist and plant breeder</span>. 
         <abbr class="cit-jnl-abbrev">In the cereal belts of southern Au</abbr>..
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Foolad</span>, 
          <span class="cit-name-given-names">M.R.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">Genome mapping and molecular breeding of tomato</span>. 
         <abbr class="cit-jnl-abbrev">Int J Plant Genomics</abbr>. 
         <span class="cit-vol">2007</span>: 
         <span class="cit-fpage">64358</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BPlant%2BGenomics%26rft.aulast%253DFoolad%26rft.auinit1%253DM.%2BR.%26rft.volume%253D2007%26rft.spage%253D64358%26rft.epage%253D64358%26rft.atitle%253DGenome%2Bmapping%2Band%2Bmolecular%2Bbreeding%2Bof%2Btomato.%26rft_id%253Dinfo%253Apmid%252F18364989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=18364989&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.15" data-doi="10.1038/s41467-017-01450-2">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gage</span>, 
          <span class="cit-name-given-names">J.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jarquin</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Romay</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lorenz</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Buckler</span>, 
          <span class="cit-name-given-names">E.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kaeppler</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">The effect of artificial selection on phenotypic plasticity in maize</span>. 
         <abbr class="cit-jnl-abbrev">Nat Commun</abbr>. 
         <span class="cit-vol">8</span>: 
         <span class="cit-fpage">1348</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D1348%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-01450-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/s41467-017-01450-2&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.16">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Gillespie</span>, 
          <span class="cit-name-given-names">J.H.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Turelli</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">1989</span>) 
         <span class="cit-article-title">Genotype-environment interactions and the maintenance of polygenic variation</span>. 
         <abbr class="cit-jnl-abbrev">Genetics</abbr>. 
         <span class="cit-vol">121</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.17" data-doi="10.1534/genetics.117.300305">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Giraud</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bauland</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Falque</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Madur</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Combes</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jamin</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Reciprocal Genetics: Identifying QTL for General and Specific Combining Abilities in Hybrids Between Multiparental Populations from Two Maize (Zea mays L.) Heterotic Groups</span>. 
         <abbr class="cit-jnl-abbrev">Genetics</abbr>. 
         <span class="cit-vol">207</span>: 
         <span class="cit-fpage">1167</span>–
         <span class="cit-lpage">1180</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300305%26rft_id%253Dinfo%253Apmid%252F28971957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNy8zLzExNjciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wNy8yMDIwLjAyLjA3LjkzODQ1Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.18">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grandillo</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Termolino</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">van der Knaap</span>, 
          <span class="cit-name-given-names">E.</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">Molecular Mapping of Complex Traits in Tomato</span>. 
         <abbr class="cit-jnl-abbrev">In Genetics, Genomics, and Breeding of Tomato</abbr>. pp. 
         <span class="cit-fpage">150</span>–
         <span class="cit-lpage">227 Science Publishers</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.19">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Grilli</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Trevizan Braz</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gertrudes</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Lemos</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">QTL identification for tolerance to fruit set in tomatoby fAFLP markers</span>. 
         <abbr class="cit-jnl-abbrev">Crop Breed Appl Biotechnol</abbr>. 
         <span class="cit-vol">7</span>: 
         <span class="cit-fpage">234</span>–
         <span class="cit-lpage">241</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCrop%2BBreed%2BAppl%2BBiotechnol%26rft.volume%253D7%26rft.spage%253D234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.20">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Herath</span>, 
          <span class="cit-name-given-names">V.</span>
         </span> (
         <span class="cit-pub-date">2019</span>) 
         <span class="cit-article-title">The architecture of the GhD7 promoter reveals the roles of GhD7 in growth, development and the abiotic stress response in rice</span>. 
         <abbr class="cit-jnl-abbrev">Comput Biol Chem</abbr>. 
         <span class="cit-vol">82</span>: 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">8</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DComput%2BBiol%2BChem%26rft.volume%253D82%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.21" data-doi="10.1093/bioinformatics/btq719">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">B.E.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">George</span>, 
          <span class="cit-name-given-names">A.W.</span>
         </span> (
         <span class="cit-pub-date">2011</span>) 
         <span class="cit-article-title">R/mpMap: A computational platform for the genetic analysis of multiparent recombinant inbred lines</span>. 
         <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>. 
         <span class="cit-vol">27</span>: 
         <span class="cit-fpage">727</span>–
         <span class="cit-lpage">729</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq719%26rft_id%253Dinfo%253Apmid%252F21217121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq719&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21217121&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000287758400023&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.22" data-doi="10.1111/j.1467-7652.2012.00702.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">B.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">George</span>, 
          <span class="cit-name-given-names">A.W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Forrest</span>, 
          <span class="cit-name-given-names">K.L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kilian</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hayden</span>, 
          <span class="cit-name-given-names">M.J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Morell</span>, 
          <span class="cit-name-given-names">M.K.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2012</span>) 
         <span class="cit-article-title">A multiparent advanced generation inter-cross population for genetic analysis in wheat</span>. 
         <abbr class="cit-jnl-abbrev">Plant Biotechnol J</abbr>. 
         <span class="cit-vol">10</span>: 
         <span class="cit-fpage">826</span>–
         <span class="cit-lpage">839</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DHuang%26rft.auinit1%253DB.%2BE.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D826%26rft.epage%253D839%26rft.atitle%253DA%2Bmultiparent%2Badvanced%2Bgeneration%2Binter-cross%2Bpopulation%2Bfor%2Bgenetic%2Banalysis%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00702.x%26rft_id%253Dinfo%253Apmid%252F22594629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00702.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=22594629&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000307051700007&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.23">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gehan</span>, 
          <span class="cit-name-given-names">M.A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huss</span>, 
          <span class="cit-name-given-names">S.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Alvarez</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lizarraga</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gruebbling</span>, 
          <span class="cit-name-given-names">E.L.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Cross-species complementation reveals conserved functions for EARLY FLOWERING 3 between monocots and dicots</span>. 
         <abbr class="cit-jnl-abbrev">Plant Direct</abbr>. 
         <span class="cit-vol">1</span>: 
         <span class="cit-fpage">e00018</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BDirect%26rft.volume%253D1%26rft.spage%253D00018e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.24" data-doi="10.1371/journal.pgen.1000551">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kover</span>, 
          <span class="cit-name-given-names">P.X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Valdar</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Trakalo</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scarcelli</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ehrenreich</span>, 
          <span class="cit-name-given-names">I.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Purugganan</span>, 
          <span class="cit-name-given-names">M.D.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in Arabidopsis thaliana</span>. 
         <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. 
         <span class="cit-vol">5</span>: 
         <span class="cit-fpage">e1000551</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=19593375&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.25">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Kusmec</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Srinivasan</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nettleton</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Schnable</span>, 
          <span class="cit-name-given-names">P.S.</span>
         </span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Distinct genetic architectures for phenotype means and plasticities in Zea mays</span>. 
         <abbr class="cit-jnl-abbrev">Nat Plants</abbr>. 
         <span class="cit-vol">3</span>: 
         <span class="cit-fpage">715</span>–
         <span class="cit-lpage">723</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BPlants%26rft.volume%253D3%26rft.spage%253D715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.26">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lacaze</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Hayes</span>, 
          <span class="cit-name-given-names">P.M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Korol</span>, 
          <span class="cit-name-given-names">A.</span>
         </span> (
         <span class="cit-pub-date">2009</span>) 
         <span class="cit-article-title">Genetics of phenotypic plasticity: QTL analysis in barley, Hordeum vulgare</span>. 
         <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr>. 
         <span class="cit-vol">102</span>: 
         <span class="cit-fpage">163</span>–
         <span class="cit-lpage">173</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D102%26rft.spage%253D163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.27">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Laitinen</span>, 
          <span class="cit-name-given-names">R.A.E.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Nikoloski</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span> (
         <span class="cit-pub-date">2019</span>) 
         <span class="cit-article-title">Genetic basis of plasticity in plants</span>. 
         <abbr class="cit-jnl-abbrev">J Exp Bot</abbr>. 
         <span class="cit-vol">70</span>: 
         <span class="cit-fpage">739</span>–
         <span class="cit-lpage">745</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BBot%26rft.volume%253D70%26rft.spage%253D739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.28" data-doi="10.18637/jss.v025.i14">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lê</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Josse</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Husson</span>, 
          <span class="cit-name-given-names">F.</span>
         </span> (
         <span class="cit-pub-date">2008</span>) 
         <span class="cit-article-title">FactoMineR: An R package for multivariate analysis</span>. 
         <abbr class="cit-jnl-abbrev">J Stat Softw</abbr>. 
         <span class="cit-vol">25</span>: 
         <span class="cit-fpage">1</span>–
         <span class="cit-lpage">18</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BStat%2BSoftw%26rft.volume%253D25%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v025.i14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.18637/jss.v025.i14&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.29" data-doi="10.2135/cropsci1986.0011183X002600050012x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">C.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Binns</span>, 
          <span class="cit-name-given-names">M.R.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Lefkovitch</span>, 
          <span class="cit-name-given-names">L.P.</span>
         </span> (
         <span class="cit-pub-date">1986</span>) 
         <span class="cit-article-title">Stability Analysis: Where Do We Stand?1</span>. 
         <abbr class="cit-jnl-abbrev">Crop Sci</abbr>. 
         <span class="cit-vol">26</span>: 
         <span class="cit-fpage">894</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DLin%26rft.auinit1%253DC.%2BS.%26rft.volume%253D26%26rft.issue%253D5%26rft.spage%253D894%26rft.epage%253D900%26rft.atitle%253DStability%2BAnalysis%253A%2BWhere%2BDo%2BWe%2BStand%253F%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1986.0011183X002600050012x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2135/cropsci1986.0011183X002600050012x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.30" data-doi="10.1038/ng.3117">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Lin</span>, 
          <span class="cit-name-given-names">T.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Yu</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zheng</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Genomic analyses provide insights into the history of tomato breeding</span>. 
         <abbr class="cit-jnl-abbrev">Nat Genet</abbr>. 
         <span class="cit-vol">46</span>: 
         <span class="cit-fpage">1220</span>–
         <span class="cit-lpage">1226</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNat%2BGenet%26rft.volume%253D46%26rft.spage%253D1220%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3117%26rft_id%253Dinfo%253Apmid%252F25305757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1038/ng.3117&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25305757&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.31">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Luo</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gao</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Chen</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">The NUCLEAR FACTOR-CONSTANS complex antagonizes Polycomb repression to de-repress FLOWERING LOCUS T expression in response to inductive long days in Arabidopsis</span>. 
         <abbr class="cit-jnl-abbrev">Plant J</abbr>. 
         <span class="cit-vol">95</span>: 
         <span class="cit-fpage">17</span>–
         <span class="cit-lpage">29</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BJ%26rft.volume%253D95%26rft.spage%253D17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.32" data-doi="10.3389/fphys.2013.00044">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Malosetti</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ribaut</span>, 
          <span class="cit-name-given-names">J.-M.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">van Eeuwijk</span>, 
          <span class="cit-name-given-names">F.A.</span>
         </span> (
         <span class="cit-pub-date">2013</span>) 
         <span class="cit-article-title">The statistical analysis of multi-environment data: modeling genotype-by-environment interaction and its genetic basis</span>. 
         <abbr class="cit-jnl-abbrev">Front Physiol</abbr>. 
         <span class="cit-vol">4</span>: 
         <span class="cit-fpage">44</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DFront%2BPhysiol%26rft.volume%253D4%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffphys.2013.00044%26rft_id%253Dinfo%253Apmid%252F23487515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.3389/fphys.2013.00044&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=23487515&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.33">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mangin</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Casadebaig</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cadic</span>, 
          <span class="cit-name-given-names">E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Blanchet</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Boniface</span>, 
          <span class="cit-name-given-names">M.-C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carrère</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Genetic control of plasticity of oil yield for combined abiotic stresses using a joint approach of crop modelling and genome-wide association</span>. 
         <abbr class="cit-jnl-abbrev">Plant Cell Environ</abbr>. 
         <span class="cit-vol">40</span>: 
         <span class="cit-fpage">2276</span>–
         <span class="cit-lpage">2291</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron%26rft.volume%253D40%26rft.spage%253D2276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.34" data-doi="10.1111/j.1399-3054.1991.tb01299.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Mitchell</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Shennan</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Grattan</span>, 
          <span class="cit-name-given-names">S.</span>
         </span> (
         <span class="cit-pub-date">1991</span>) 
         <span class="cit-article-title">Developmental-Changes in Tomato Fruit Composition in Response To Water Deficit and Salinity</span>. 
         <abbr class="cit-jnl-abbrev">Physiol Plant</abbr>. 
         <span class="cit-vol">83</span>: 
         <span class="cit-fpage">177</span>–
         <span class="cit-lpage">185</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPhysiol%2BPlant%26rft.volume%253D83%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1399-3054.1991.tb01299.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1399-3054.1991.tb01299.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.35" data-doi="10.1111/nph.13519">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Munns</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Gilliham</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Salinity tolerance of crops - what is the cost?</span>
         <abbr class="cit-jnl-abbrev">New Phytol</abbr>. 
         <span class="cit-vol">208</span>: 
         <span class="cit-fpage">668</span>–
         <span class="cit-lpage">673</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol%26rft.volume%253D208%26rft.spage%253D668%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13519%26rft_id%253Dinfo%253Apmid%252F26108441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/nph.13519&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=26108441&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference in text" id="ref-36">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.36" data-doi="10.1111/pbi.12282">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Pascual</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Desplat</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">B.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Desgroux</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bruguier</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bouchet</span>, 
          <span class="cit-name-given-names">J.-P.P.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2015</span>) 
         <span class="cit-article-title">Potential of a tomato MAGIC population to decipher the genetic control of quantitative traits and detect causal variants in the resequencing era</span>. 
         <abbr class="cit-jnl-abbrev">Plant Biotechnol J</abbr>. 
         <span class="cit-vol">13</span>: 
         <span class="cit-fpage">565</span>–
         <span class="cit-lpage">577</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol%2BJ%26rft.volume%253D13%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12282%26rft_id%253Dinfo%253Apmid%252F25382275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/pbi.12282&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=25382275&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.37" data-doi="10.1093/jxb/eru197">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ripoll</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Urban</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Staudt</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lopez-Lauri</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bidel</span>, 
          <span class="cit-name-given-names">L.P.R.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Bertin</span>, 
          <span class="cit-name-given-names">N.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Water shortage and quality of fleshy fruits—making the most of the unavoidable</span>. 
         <abbr class="cit-jnl-abbrev">J Exp Bot</abbr>. 
         <span class="cit-vol">65</span>: 
         <span class="cit-fpage">4097</span>–
         <span class="cit-lpage">4117</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BExp%2BBot%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Feru197%26rft_id%253Dinfo%253Apmid%252F24821951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1093/jxb/eru197&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=24821951&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.38">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Rothan</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Diouf</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Causse</span>, 
          <span class="cit-name-given-names">M.</span>
         </span> (
         <span class="cit-pub-date">2019</span>) 
         <span class="cit-article-title">Trait discovery and editing in tomato</span>. 
         <abbr class="cit-jnl-abbrev">Plant J</abbr>. 
         <span class="cit-vol">97</span>: 
         <span class="cit-fpage">73</span>–
         <span class="cit-lpage">90</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DPlant%2BJ%26rft.volume%253D97%26rft.spage%253D73%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.39" data-doi="10.1146/annurev.es.24.110193.000343">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Scheiner</span>, 
          <span class="cit-name-given-names">S.M.</span>
         </span> (
         <span class="cit-pub-date">1993</span>) 
         <span class="cit-article-title">Genetics and Evolution of Phenotypic Plasticity</span>. 
         <abbr class="cit-jnl-abbrev">Annu Rev Ecol Syst</abbr>. 
         <span class="cit-vol">24</span>: 
         <span class="cit-fpage">35</span>–
         <span class="cit-lpage">68</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DScheiner%26rft.auinit1%253DS.%2BM.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D68%26rft.atitle%253DGenetics%2Band%2BEvolution%2Bof%2BPhenotypic%2BPlasticity%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.24.110193.000343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1146/annurev.es.24.110193.000343&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1993MJ37100002&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.40">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Septiani</span>, 
          <span class="cit-name-given-names">P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lanubile</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Stagnati</span>, 
          <span class="cit-name-given-names">L.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Busconi</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Nelissen</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Pè</span>, 
          <span class="cit-name-given-names">M.E.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2019</span>) 
         <span class="cit-article-title">Unravelling the genetic basis of fusarium seedling rot resistance in the MAGIC maize population: Novel targets for breeding</span>. 
         <abbr class="cit-jnl-abbrev">Sci Rep</abbr>. 
         <span class="cit-vol">9</span>: 
         <span class="cit-fpage">4</span>–
         <span class="cit-lpage">13</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D9%26rft.spage%253D4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.41">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Ungerer</span>, 
          <span class="cit-name-given-names">M.C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Halldorsdottir</span>, 
          <span class="cit-name-given-names">S.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Purugganan</span>, 
          <span class="cit-name-given-names">M.D.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Mackay</span>, 
          <span class="cit-name-given-names">T.F.C.</span>
         </span> (
         <span class="cit-pub-date">2003</span>) 
         <span class="cit-article-title">Genotype-environment interactions at quantitative trait loci affecting inflorescence development in Arabidopsis thaliana</span>. 
         <abbr class="cit-jnl-abbrev">Genetics</abbr>. 
         <span class="cit-vol">165</span>: 
         <span class="cit-fpage">353</span>–
         <span class="cit-lpage">365</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DUngerer%26rft.auinit1%253DM.%2BC.%26rft.volume%253D165%26rft.issue%253D1%26rft.spage%253D353%26rft.epage%253D365%26rft.atitle%253DGenotype-Environment%2BInteractions%2Bat%2BQuantitative%2BTrait%2BLoci%2BAffecting%2BInflorescence%2BDevelopment%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Apmid%252F14504242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzEvMzUzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjAvMDIvMDcvMjAyMC4wMi4wNy45Mzg0NTYuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference in text" id="ref-42">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.42" data-doi="10.1111/j.1365-2745.2006.01176.x">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Valladares</span>, 
          <span class="cit-name-given-names">F.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Sanchez-Gomez</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Zavala</span>, 
          <span class="cit-name-given-names">M.A.</span>
         </span> (
         <span class="cit-pub-date">2006</span>) 
         <span class="cit-article-title">Quantitative estimation of phenotypic plasticity: bridging the gap between the evolutionary concept and its ecological applications</span>. 
         <abbr class="cit-jnl-abbrev">J Ecol</abbr>. 
         <span class="cit-vol">94</span>: 
         <span class="cit-fpage">1103</span>–
         <span class="cit-lpage">1116</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DJ%2BEcol%26rft.volume%253D94%26rft.spage%253D1103%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2745.2006.01176.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1111/j.1365-2745.2006.01176.x&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000240920400007&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.43">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Verbyla</span>, 
          <span class="cit-name-given-names">A.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Cavanagh</span>, 
          <span class="cit-name-given-names">C.R.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Verbyla</span>, 
          <span class="cit-name-given-names">K.L.</span>
         </span> (
         <span class="cit-pub-date">2014</span>) 
         <span class="cit-article-title">Whole-Genome Analysis of Multienvironment or Multitrait QTL in MAGIC</span>. 
         <abbr class="cit-jnl-abbrev">G3 Genes, Genomes, Genet</abbr>. 
         <span class="cit-vol">4</span>.
        </cite>
       </div>
       <div class="cit-extra"/>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference in text" id="ref-44">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.44" data-doi="10.1016/S0169-5347(00)89061-8">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Via</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Gomulkiewicz</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">De Jong</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Scheiner</span>, 
          <span class="cit-name-given-names">S.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Schlichting</span>, 
          <span class="cit-name-given-names">C.D.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Van Tienderen</span>, 
          <span class="cit-name-given-names">P.H.</span>
         </span> (
         <span class="cit-pub-date">1995</span>) 
         <span class="cit-article-title">Adaptive phenotypic plasticity: consensus and controversy</span>. 
         <abbr class="cit-jnl-abbrev">Trends Ecol Evol</abbr>. 
         <span class="cit-vol">10</span>: 
         <span class="cit-fpage">212</span>–
         <span class="cit-lpage">7</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DVia%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D5%26rft.spage%253D212%26rft.epage%253D217%26rft.atitle%253DAdaptive%2Bphenotypic%2Bplasticity%253A%2Bconsensus%2Band%2Bcontroversy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252800%252989061-8%26rft_id%253Dinfo%253Apmid%252F21237012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/S0169-5347(00)89061-8&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=21237012&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=A1995QU19400018&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.45" data-doi="10.1007/s00122-006-0494-9">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Villalta</span>, 
          <span class="cit-name-given-names">I.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Bernet</span>, 
          <span class="cit-name-given-names">G.P.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Carbonell</span>, 
          <span class="cit-name-given-names">E.A.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Asins</span>, 
          <span class="cit-name-given-names">M.J.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">Comparative QTL analysis of salinity tolerance in terms of fruit yield using two solanum populations of F7 lines</span>. 
         <abbr class="cit-jnl-abbrev">Theor Appl Genet</abbr>. 
         <span class="cit-vol">114</span>: 
         <span class="cit-fpage">1001</span>–
         <span class="cit-lpage">1017</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVillalta%26rft.auinit1%253DI.%26rft.volume%253D114%26rft.issue%253D6%26rft.spage%253D1001%26rft.epage%253D1017%26rft.atitle%253DComparative%2BQTL%2Banalysis%2Bof%2Bsalinity%2Btolerance%2Bin%2Bterms%2Bof%2Bfruit%2Byield%2Busing%2Btwo%2BSolanum%2Bpopulations%2Bof%2BF7%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0494-9%26rft_id%253Dinfo%253Apmid%252F17394031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1007/s00122-006-0494-9&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=17394031&amp;;link_type=MED&amp;;atom=%2Fbiorxiv%2Fearly%2F2020%2F02%2F07%2F2020.02.07.938456.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
         <span>PubMed</span>
        </a>
        <a href="/lookup/external-ref?access_num=000245362000005&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference in text" id="ref-46">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.46" data-doi="10.1534/g3.117.300300">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xavier</span>, 
          <span class="cit-name-given-names">A.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Jarquin</span>, 
          <span class="cit-name-given-names">D.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Howard</span>, 
          <span class="cit-name-given-names">R.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ramasubramanian</span>, 
          <span class="cit-name-given-names">V.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Specht</span>, 
          <span class="cit-name-given-names">J.E.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Graef</span>, 
          <span class="cit-name-given-names">G.L.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">Genome-Wide Analysis of Grain Yield Stability and Environmental Interactions in a Multiparental Soybean Population</span>. 
         <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr>. 
         <span class="cit-vol">8</span>: 
         <span class="cit-fpage">519</span>–
         <span class="cit-lpage">529</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.300300%26rft_id%253Dinfo%253Apmid%252F29217731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjgvMi81MTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wNy8yMDIwLjAyLjA3LjkzODQ1Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference in text" id="ref-47">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.47">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Driedonks</span>, 
          <span class="cit-name-given-names">N.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Rutten</span>, 
          <span class="cit-name-given-names">M.J.M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Vriezen</span>, 
          <span class="cit-name-given-names">W.H.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">de Boer</span>, 
          <span class="cit-name-given-names">G.-J.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Rieu</span>, 
          <span class="cit-name-given-names">I.</span>
         </span> (
         <span class="cit-pub-date">2017a</span>) 
         <span class="cit-article-title">Mapping quantitative trait loci for heat tolerance of reproductive traits in tomato (Solanum lycopersicum)</span>. 
         <abbr class="cit-jnl-abbrev">Mol Breed</abbr>. 
         <span class="cit-vol">37</span>: 
         <span class="cit-fpage">58</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DMol%2BBreed%26rft.volume%253D37%26rft.spage%253D58%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference in text" id="ref-48">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.48">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Xu</span>, 
          <span class="cit-name-given-names">J.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wolters-Arts</span>, 
          <span class="cit-name-given-names">M.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Mariani</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huber</span>, 
          <span class="cit-name-given-names">H.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Rieu</span>, 
          <span class="cit-name-given-names">I.</span>
         </span> (
         <span class="cit-pub-date">2017b</span>) 
         <span class="cit-article-title">Heat stress affects vegetative and reproductive performance and trait correlations in tomato (Solanum lycopersicum)</span>. 
         <abbr class="cit-jnl-abbrev">Euphytica</abbr>. 
         <span class="cit-vol">213</span>: 
         <span class="cit-fpage">156</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D213%26rft.spage%253D156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference in text" id="ref-49">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.49" data-doi="10.2135/cropsci2006.06.0374">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Yan</span>, 
          <span class="cit-name-given-names">W.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Kang</span>, 
          <span class="cit-name-given-names">M.S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Ma</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Woods</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, and 
         <span class="cit-auth">
          <span class="cit-name-surname">Cornelius</span>, 
          <span class="cit-name-given-names">P.L.</span>
         </span> (
         <span class="cit-pub-date">2007</span>) 
         <span class="cit-article-title">GGE Biplot vs. AMMI Analysis of Genotype-by-Environment Data</span>. 
         <abbr class="cit-jnl-abbrev">Crop Sci</abbr>. 
         <span class="cit-vol">47</span>: 
         <span class="cit-fpage">643</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DYan%26rft.auinit1%253DW.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D643%26rft.epage%253D653%26rft.atitle%253DGGE%2BBiplot%2Bvs.%2BAMMI%2BAnalysis%2Bof%2BGenotype-by-Environment%2BData%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.06.0374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.2135/cropsci2006.06.0374&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
        <a href="/lookup/external-ref?access_num=000245735800020&amp;;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
         <span>Web of Science</span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference in text" id="ref-50">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.50" data-doi="10.1073/pnas.1701762114">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhao</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liu</span>, 
          <span class="cit-name-given-names">B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Piao</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">X.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Lobell</span>, 
          <span class="cit-name-given-names">D.B.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">Y.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2017</span>) 
         <span class="cit-article-title">Temperature increase reduces global yields of major crops in four independent estimates</span>. 
         <abbr class="cit-jnl-abbrev">Proc Natl Acad Sci</abbr>. 
         <span class="cit-vol">114</span>: 
         <span class="cit-fpage">9326</span>–
         <span class="cit-lpage">9331</span>.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1701762114%26rft_id%253Dinfo%253Apmid%252F28811375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE0LzM1LzkzMjYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMC8wMi8wNy8yMDIwLjAyLjA3LjkzODQ1Ni5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
         <span>
          <span class="cit-reflinks-abstract">Abstract</span>
          <span class="cit-sep cit-reflinks-variant-name-sep">/</span>
          <span class="cit-reflinks-full-text">
           <span class="free-full-text">FREE </span>Full Text
          </span>
         </span>
        </a>
       </div>
      </div>
     </li>
     <li>
      <span class="ref-label ref-label-empty"/>
      <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference in text" id="ref-51">↵</a>
      <div class="cit ref-cit ref-journal" id="cit-2020.02.07.938456v1.51" data-doi="10.1016/j.cell.2017.12.019">
       <div class="cit-metadata">
        <cite>
         <span class="cit-auth">
          <span class="cit-name-surname">Zhu</span>, 
          <span class="cit-name-given-names">G.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Wang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Huang</span>, 
          <span class="cit-name-given-names">Z.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">S.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Liao</span>, 
          <span class="cit-name-given-names">Q.</span>
         </span>, 
         <span class="cit-auth">
          <span class="cit-name-surname">Zhang</span>, 
          <span class="cit-name-given-names">C.</span>
         </span>, 
         <span class="cit-etal">et al.</span> (
         <span class="cit-pub-date">2018</span>) 
         <span class="cit-article-title">Rewiring of the Fruit Metabolome in Tomato Breeding</span>. 
         <abbr class="cit-jnl-abbrev">Cell</abbr>. 
         <span class="cit-vol">172</span>: 
         <span class="cit-fpage">249</span>-
         <span class="cit-lpage">261</span>.e12.
        </cite>
       </div>
       <div class="cit-extra">
        <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D172%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2017.12.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
         <span>OpenUrl</span>
        </a>
        <a href="/lookup/external-ref?access_num=10.1016/j.cell.2017.12.019&amp;;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
         <span>CrossRef</span>
        </a>
       </div>
      </div>
     </li>
    </ol>
   </div>
   <span class="highwire-journal-article-marker-end"/>
  </div>
 </body>
</html>
